

<!DOCTYPE html>
<html lang="en">

<head><title>
	mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/1/53/2462457","@type":"ScholarlyArticle","name":"mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea","datePublished":"2015-12-19","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2016-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv036","keywords":["chickpea","mQTL-seq","pod number","SNP","wild accessions"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/1/53/2462457","author":[{"name":"Das, Shouvik","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Singh, Mohar","affiliation":"2  National Bureau of Plant Genetic Resources Regional Station ,  Shimla, Himachal Pradesh  171004 ,  India","@type":"Person"},{"name":"Srivastava, Rishi","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Bajaj, Deepak","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Saxena, Maneesha S.","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Rana, Jai C.","affiliation":"3  National Bureau of Plant Genetic Resources (NBPGR) ,  New Delhi  110012 ,  India","@type":"Person"},{"name":"Bansal, Kailash C.","affiliation":"3  National Bureau of Plant Genetic Resources (NBPGR) ,  New Delhi  110012 ,  India","@type":"Person"},{"name":"Tyagi, Akhilesh K.","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  New Delhi 110067 ,  India","@type":"Person"},{"name":"Parida, Swarup K.","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  New Delhi 110067 ,  India","@type":"Person"}],"description":"The present study used a whole-genome, NGS resequencing-based mQTL-seq (multiple QTL-seq) strategy in two inter-specific mapping populations (Pusa 1103 × ILWC 46 and Pusa 256 × ILWC 46) to scan the major genomic region(s) underlying QTL(s) governing pod number trait in chickpea. Essentially, the whole-genome resequencing of low and high pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of these two mapping populations discovered &gt;8 million high-quality homozygous SNPs with respect to the reference kabuli chickpea. The functional significance of the physically mapped SNPs was apparent from the identified 2,264 non-synonymous and 23,550 regulatory SNPs, with 8–10% of these SNPs-carrying genes corresponding to transcription factors and disease resistance-related proteins. The utilization of these mined SNPs in Δ (SNP index)-led QTL-seq analysis and their correlation between two mapping populations based on mQTL-seq, narrowed down two (CaqaPN4.1: 867.8 kb and CaqaPN4.2: 1.8 Mb) major genomic regions harbouring robust pod number QTLs into the high-resolution short QTL intervals (CaqbPN4.1: 637.5 kb and CaqbPN4.2: 1.28 Mb) on chickpea chromosome 4. The integration of mQTL-seq-derived one novel robust QTL with QTL region-specific association analysis delineated the regulatory (C/T) and coding (C/A) SNPs-containing one pentatricopeptide repeat (PPR) gene at a major QTL region regulating pod number in chickpea. This target gene exhibited anther, mature pollen and pod-specific expression, including pronounced higher up-regulated (∼3.5-folds) transcript expression in high pod number-containing parental accessions and homozygous individuals of two mapping populations especially during pollen and pod development. The proposed mQTL-seq-driven combinatorial strategy has profound efficacy in rapid genome-wide scanning of potential candidate gene(s) underlying trait-associated high-resolution robust QTL(s), thereby expediting genomics-assisted breeding and genetic enhancement of crop plants, including chickpea.","pageStart":"53","pageEnd":"65","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=1572217438&Signature=eqktcNUYMxj5hP3dR6phoB4ivPDoPZjUiIt8nZiTYXe7uDv4obBddKhrdOz4kooqhhM8BROfxXytwrxlQYtlgH1lcVS84eYRfYGoDUsStGMZs3dob1UEEInJhWEhfz4bxxZKIGmJWgoEZ9b2IyuvG9ypGZr-aac-x0Yfl3nI2C5bIks3wF1NGeGGLp0bEqT4O3i7WkqbbX~Wt0U0vnBJHhsL34mwlEdUCPdwK334DG0GNPPdtbe8U~9kbIjWBGKZKTBe7IzPxnQ5zZIZ5lQnPuQythaXQPfPkMEFSQ~WKSB36-s7jKQIjdAqUWxlCgueWuoPNZ~sWBu8IkDvcPKoRw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=1572217438&Signature=eqktcNUYMxj5hP3dR6phoB4ivPDoPZjUiIt8nZiTYXe7uDv4obBddKhrdOz4kooqhhM8BROfxXytwrxlQYtlgH1lcVS84eYRfYGoDUsStGMZs3dob1UEEInJhWEhfz4bxxZKIGmJWgoEZ9b2IyuvG9ypGZr-aac-x0Yfl3nI2C5bIks3wF1NGeGGLp0bEqT4O3i7WkqbbX~Wt0U0vnBJHhsL34mwlEdUCPdwK334DG0GNPPdtbe8U~9kbIjWBGKZKTBe7IzPxnQ5zZIZ5lQnPuQythaXQPfPkMEFSQ~WKSB36-s7jKQIjdAqUWxlCgueWuoPNZ~sWBu8IkDvcPKoRw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea | DNA Research | Oxford Academic" />
    <meta property="og:description" content="The present study used a whole-genome, NGS resequencing-based mQTL-seq (multiple QTL-seq) strategy in two inter-specific mapping populations (Pusa 1103 &#215; ILWC 46 and Pusa 256 &#215; ILWC 46) to scan the major genomic region(s) underlying QTL(s) governing pod number trait in chickpea. Essentially, the whole-genome resequencing of low and high pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of these two mapping populations discovered &amp;gt;8 million high-quality homozygous SNPs with respect to the reference kabuli chickpea. The functional significance of the physically mapped SNPs was apparent from the identified 2,264 non-synonymous and 23,550 regulatory SNPs, with 8–10% of these SNPs-carrying genes corresponding to transcription factors and disease resistance-related proteins. The utilization of these mined SNPs in Δ (SNP index)-led QTL-seq analysis and their correlation between two mapping populations based on mQTL-seq, narrowed down two (CaqaPN4.1: 867.8 kb and CaqaPN4.2: 1.8 Mb) major genomic regions harbouring robust pod number QTLs into the high-resolution short QTL intervals (CaqbPN4.1: 637.5 kb and CaqbPN4.2: 1.28 Mb) on chickpea chromosome 4. The integration of mQTL-seq-derived one novel robust QTL with QTL region-specific association analysis delineated the regulatory (C/T) and coding (C/A) SNPs-containing one pentatricopeptide repeat (PPR) gene at a major QTL region regulating pod number in chickpea. This target gene exhibited anther, mature pollen and pod-specific expression, including pronounced higher up-regulated (∼3.5-folds) transcript expression in high pod number-containing parental accessions and homozygous individuals of two mapping populations especially during pollen and pod development. The proposed mQTL-seq-driven combinatorial strategy has profound efficacy in rapid genome-wide scanning of potential candidate gene(s) underlying trait-associated high-resolution robust QTL(s), thereby expediting genomics-assisted breeding and genetic enhancement of crop plants, including chickpea." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/1/53/2462457" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217438&amp;Signature=ShG8JIECIbbVntWKCGrq6JIL4V4DXCd4-04Mzbbzwv6dbmonwebBH46uBl-ME54ZlIO72~pq574NkFm-1TUmUa9XC12WQ3WqpH8jJOcNu7sJvHIoPLNyqei71dAJVcNUMEwAsTJ~JNNSmT-JYjEY2XmI6OmxPbL2TzYOjKbMzq~ofDFBa9TZXbI9gsd4QqJ6xcHgiRaOGfGY71yAli29dez8Z-OGX2bVejxsPe-rpW0Cj-3Qq~htO41zZ1tcFQKE2Qsn9KfekMdK7z4sCT7VIOOdW-LsOWdItNaqW7alfFBgM5x31gsIShIhx59-HtVlkwtlAXX1e43G3cooZmh-HQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2015-12-19" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Das, Shouvik" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), New Delhi 110067, India" /><meta name="citation_author" content="Singh, Mohar" /><meta name="citation_author_institution" content="National Bureau of Plant Genetic Resources Regional Station, Shimla, Himachal Pradesh171004, India" /><meta name="citation_author" content="Srivastava, Rishi" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), New Delhi 110067, India" /><meta name="citation_author" content="Bajaj, Deepak" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), New Delhi 110067, India" /><meta name="citation_author" content="Saxena, Maneesha S." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), New Delhi 110067, India" /><meta name="citation_author" content="Rana, Jai C." /><meta name="citation_author_institution" content="National Bureau of Plant Genetic Resources (NBPGR), New Delhi110012, India" /><meta name="citation_author" content="Bansal, Kailash C." /><meta name="citation_author_institution" content="National Bureau of Plant Genetic Resources (NBPGR), New Delhi110012, India" /><meta name="citation_author" content="Tyagi, Akhilesh K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), New Delhi 110067, India" /><meta name="citation_author" content="Parida, Swarup K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), New Delhi 110067, India" /><meta name="citation_title" content="mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea" /><meta name="citation_firstpage" content="53" /><meta name="citation_lastpage" content="65" /><meta name="citation_doi" content="10.1093/dnares/dsv036" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2016/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Towards marker-assisted selection in pulses: a review;  citation_journal_title=Plant Breed.;  citation_year=2011;  citation_volume=130;  citation_pages=297-313; " /><meta name="citation_reference" content="citation_title=Achievements and prospects of genomics-assisted breeding in three legume crops of the semi-arid tropics;  citation_journal_title=Biotechnol. Adv.;  citation_year=2013;  citation_volume=31;  citation_pages=1120-34; " /><meta name="citation_reference" content="citation_title=Potential effects of individual versus simultaneous climate change factors on growth and water use in chickpea;  citation_journal_title=Int. J. Plant Prod.;  citation_year=2007;  citation_volume=1;  citation_pages=189-204; " /><meta name="citation_reference" content="citation_title=Climate change, adaptation, and phenotypic plasticity: the problem and the evidence;  citation_journal_title=Evol. Appl.;  citation_year=2014;  citation_volume=7;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Abiotic stresses, constraints and improvement strategies in chickpea;  citation_journal_title=Plant Breed.;  citation_year=2014;  citation_volume=133;  citation_pages=163-78; " /><meta name="citation_reference" content="citation_title=Fast-track introgression of “QTL-hotspot” for root traits and other drought tolerance trait in JG 11, an elite and leading variety of chickpea (Cicer arietinum L.);  citation_journal_title=Plant Genome;  citation_year=2013;  citation_volume=6; " /><meta name="citation_reference" content="citation_title=Marker-assisted backcrossing to introgress resistance to Fusarium wilt race 1 and Ascochyta blight in C214, an elite cultivar of chickpea;  citation_journal_title=Plant Genome;  citation_year=2014;  citation_volume=7; " /><meta name="citation_reference" content="citation_title=Mapping and identification of a Cicer arietinum NSP2 gene involved in nodulation pathway;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2014;  citation_volume=127;  citation_pages=481-8; " /><meta name="citation_reference" content="citation_title=An integrated genomic approach for rapid delineation of candidate genes regulating agro-morphological traits in chickpea;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=695-710; " /><meta name="citation_reference" content="citation_title=Functionally relevant microsatellite markers from chickpea transcription factor genes for efficient genotyping applications and trait association mapping;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=355-74; " /><meta name="citation_reference" content="citation_title=An efficient and cost-effective approach for genic microsatellite marker-based large-scale trait association mapping: identification of candidate genes for seed weight in chickpea;  citation_journal_title=Mol. Breed.;  citation_year=2014;  citation_volume=34;  citation_pages=241-65; " /><meta name="citation_reference" content="citation_title=Ultra-high density intra-specific genetic linkage maps accelerate identification of functionally relevant molecular tags governing important agronomic traits in chickpea;  citation_journal_title=Sci. Rep.;  citation_year=2015;  citation_volume=5;  citation_pages=9468" /><meta name="citation_reference" content="citation_title=A genome-wide SNP scan accelerates trait-regulatory genomic loci identification in chickpea;  citation_journal_title=Sci. Rep.;  citation_year=2015;  citation_volume=5;  citation_pages=11166" /><meta name="citation_reference" content="citation_title=Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci;  citation_journal_title=Theor. Appl. Genet.;  citation_year=1997;  citation_volume=95;  citation_pages=1005-11; " /><meta name="citation_reference" content="citation_title=Development of a set of near isogenic and backcross recombinant inbred lines containing most of the Lycopersicon hirsutum genome in a L. esculentum genetic background: a tool for gene mapping and gene discovery;  citation_journal_title=Genome;  citation_year=2000;  citation_volume=43;  citation_pages=803-13; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci controlling root growth and architecture in Arabidopsis thaliana confirmed by heterogeneous inbred family;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=742-53; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (Cicer arietinum L.);  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=357-73; " /><meta name="citation_reference" content="citation_title=Large-scale development of cost-effective SNP marker assays for diversity assessment and genetic mapping in chickpea and comparative mapping in legumes;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=716-32; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism genotyping for breeding and genetics applications in chickpea and pigeonpea using the BeadXpress platform;  citation_journal_title=Plant Genome;  citation_year=2013;  citation_volume=6; " /><meta name="citation_reference" content="citation_title=Genome wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=708" /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing based intra-specific genetic map refines a “QTL-hotspot” region for drought tolerance in chickpea;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2015;  citation_volume=290;  citation_pages=559-71; " /><meta name="citation_reference" content="citation_title=Genome-wide conserved non-coding microsatellite (CNMS) marker-based integrative genetical genomics for quantitative dissection of seed weight in chickpea;  citation_journal_title=J. Exp. Bot.;  citation_year=2015;  citation_volume=66;  citation_pages=1271-90; " /><meta name="citation_reference" content="citation_title=A combinatorial approach of comprehensive QTL-based comparative genome mapping and transcript profiling identified a seed weight-regulating candidate gene in chickpea;  citation_journal_title=Sci. Rep.;  citation_year=2015;  citation_volume=5;  citation_pages=9264" /><meta name="citation_reference" content="citation_title=Mapping genes for double podding and other morphological traits in chickpea;  citation_journal_title=Euphytica;  citation_year=2002;  citation_volume=125;  citation_pages=285-92; " /><meta name="citation_reference" content="citation_title=DAF marker tightly linked to a major locus for Ascochyta blight resistance in chickpea (Cicer arietinum L.);  citation_journal_title=Euphytica;  citation_year=2003;  citation_volume=132;  citation_pages=23-30; " /><meta name="citation_reference" content="citation_title=Linkage map of chickpea (Cicer arietinum L.) based on populations from Kabuli × Desi crosses: location of genes for resistance to Fusarium wilt race 0;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1347-53; " /><meta name="citation_reference" content="citation_title=A new QTL for Ascochyta blight resistance in an RIL population derived from an interspecific cross in chickpea;  citation_journal_title=Euphytica;  citation_year=2006;  citation_volume=149;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Genetic analysis of agronomic traits in a wide cross of chickpea;  citation_journal_title=Field Crops Res.;  citation_year=2009;  citation_volume=111;  citation_pages=130-6; " /><meta name="citation_reference" content="citation_title=Mapping quantitative trait loci in chickpea associated with time to flowering and resistance to Didymella rabiei the causal agent of Ascochyta blight;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=1357-69; " /><meta name="citation_reference" content="citation_title=Genetic mapping of Ascochyta blight resistance in chickpea (Cicer arietinum) using a simple sequence repeat linkage map;  citation_journal_title=Genome;  citation_year=2007;  citation_volume=50;  citation_pages=26-34; " /><meta name="citation_reference" content="citation_title=Development of an integrated intraspecific map of chickpea (Cicer arietinum L.) using two recombinant inbred line populations;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=209-16; " /><meta name="citation_reference" content="citation_title=Genetic analyses and conservation of QTL for Ascochyta blight resistance in chickpea (Cicer arietinum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2009;  citation_volume=119;  citation_pages=757-65; " /><meta name="citation_reference" content="citation_title=Molecular mapping of wilt resistance genes in chickpea;  citation_journal_title=Mol. Breed.;  citation_year=2009;  citation_volume=24;  citation_pages=177-83; " /><meta name="citation_reference" content="citation_title=Mapping QTL for resistance to Botrytis grey mould in chickpea;  citation_journal_title=Euphytica;  citation_year=2011;  citation_volume=182;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Mapping a major gene for growth habit and QTLs for Ascochyta blight resistance and flowering time in a population between chickpea and Cicer reticulatum;  citation_journal_title=Euphytica;  citation_year=2010;  citation_volume=173;  citation_pages=307-19; " /><meta name="citation_reference" content="citation_title=Identification of large-seeded high-yielding stable kabuli chickpea germplasm lines for use in crop improvement;  citation_journal_title=Crop Sci.;  citation_year=2011;  citation_volume=5;  citation_pages=198-209; " /><meta name="citation_reference" content="citation_title=Mapping QTL associated with traits affecting grain yield in chickpea (Cicer arietinum L.) under terminal drought stress;  citation_journal_title=Crop Sci.;  citation_year=2011;  citation_volume=51;  citation_pages=450-63; " /><meta name="citation_reference" content="citation_title=Assessment of ICCV2 × JG62 chickpea progenies shows sensitivity of reproduction to salt stress and reveals QTLs for seed yield and yield components;  citation_journal_title=Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=9-21; " /><meta name="citation_reference" content="citation_title=Molecular mapping of QTLs for resistance to Fusarium wilt (race 1) and Ascochyta blight in chickpea (Cicer arietinum L.);  citation_journal_title=Euphytica;  citation_year=2013;  citation_volume=193;  citation_pages=121-33; " /><meta name="citation_reference" content="citation_title=Genetic marker discovery, intraspecific linkage map construction and quantitative trait locus analysis of Ascochyta blight resistance in chickpea (Cicer arietinum L.);  citation_journal_title=Mol. Breed.;  citation_year=2014;  citation_volume=33;  citation_pages=297-313; " /><meta name="citation_reference" content="citation_title=Genetic dissection of drought tolerance in chickpea (Cicer arietinum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2014;  citation_volume=127;  citation_pages=445-62; " /><meta name="citation_reference" content="citation_title=QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations;  citation_journal_title=Plant J.;  citation_year=2013;  citation_volume=74;  citation_pages=174-83; " /><meta name="citation_reference" content="citation_title=QTL-seq identifies an early flowering QTL located near flowering locus T in cucumber;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2014;  citation_volume=127;  citation_pages=1491-9; " /><meta name="citation_reference" content="citation_title=Deploying QTL-seq for rapid delineation of a potential candidate gene underlying major trait-associated QTL in chickpea;  citation_journal_title=DNA Res.;  citation_year=2015;  citation_volume=22;  citation_pages=193-203; " /><meta name="citation_reference" content="citation_title=Rapid and reliable identification of tomato fruit weight and locule number loci by QTL-seq;  citation_journal_title=Theor. App. Genet.;  citation_year=2015;  citation_volume=128;  citation_pages=1329-42; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of chickpea (Cicer arietinum) provides a resource for trait improvement;  citation_journal_title=Nat. Biotechnol.;  citation_year=2013;  citation_volume=31;  citation_pages=240-6; " /><meta name="citation_reference" content="citation_title=Genome sequencing reveals agronomically important loci in rice using MutMap;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=174-8; " /><meta name="citation_reference" content="citation_title=Employing genome-wide SNP discovery and genotyping strategy to extrapolate the natural allelic diversity and domestication patterns in chickpea;  citation_journal_title=Front. Plant. Sci.;  citation_year=2015;  citation_volume=6;  citation_pages=162" /><meta name="citation_reference" content="citation_title=Efficient control of population structure in model organism association mapping;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=178;  citation_pages=1709-23; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping of salinity tolerance in rice (Oryza sativa);  citation_journal_title=DNA Res.;  citation_year=2015;  citation_volume=22;  citation_pages=133-45; " /><meta name="citation_reference" content="citation_title=A chromosomal genomics approach to assess and validate the desi and kabuli draft chickpea genome assemblies;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2014;  citation_volume=12;  citation_pages=778-86; " /><meta name="citation_reference" content="citation_title=Arabidopsis emb175 and other ppr knockout mutants reveal essential roles for pentatricopeptide repeat (PPR) proteins in plant embryogenesis;  citation_journal_title=Planta;  citation_year=2005;  citation_volume=221;  citation_pages=424-36; " /><meta name="citation_reference" content="citation_title=Identification and analysis of differentially expressed genes during seed development using suppression subtractive hybridization (SSH) in Phaseolus vulgaris;  citation_journal_title=Plant Mol. Biol. Rep.;  citation_year=2012;  citation_volume=30;  citation_pages=719-30; " /><meta name="citation_reference" content="citation_title=A pentatricopeptide repeat-containing gene restores fertility to cytoplasmic male-sterile plants;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=10887-92; " /><meta name="citation_reference" content="citation_title=A pentatricopeptide repeat-containing gene that promotes the processing of aberrant atp6 RNA of cytoplasmic male-sterile rice;  citation_journal_title=FEBS Lett.;  citation_year=2003;  citation_volume=544;  citation_pages=99-102; " /><meta name="citation_reference" content="citation_title=Genetic characterization of a pentatricopeptide repeat protein gene, orf687, that restores fertility in the cytoplasmic male-sterile Kosena radish;  citation_journal_title=Plant J.;  citation_year=2003;  citation_volume=34;  citation_pages=407-15; " /><meta name="citation_reference" content="citation_title=Pentatricopeptide repeat proteins and their emerging roles in plants;  citation_journal_title=Plant Physiol. Biochem.;  citation_year=2007;  citation_volume=45;  citation_pages=521-34; " /><meta name="citation_reference" content="citation_title=Pentatricopeptide repeat proteins: a socket set for organelle gene expression;  citation_journal_title=Trends Plant Sci.;  citation_year=2008;  citation_volume=13;  citation_pages=663-70; " /><meta name="citation_reference" content="citation_title=Genic markers for WA cytoplasm based male sterility and its fertility restoration in rice;  citation_journal_title=Mol. Breed.;  citation_year=2010;  citation_volume=26;  citation_pages=275-92; " /><meta name="citation_reference" content="citation_title=Disruption of a rice pentatricopeptide repeat protein causes a seedling-specific albino phenotype and its utilization to enhance seed purity in hybrid rice production;  citation_journal_title=Plant Physiol.;  citation_year=2012;  citation_volume=159;  citation_pages=227-38; " /><meta name="citation_reference" content="citation_title=Pentatricopeptide repeat proteins in plants;  citation_journal_title=Annu. Rev. Plant Biol.;  citation_year=2014;  citation_volume=65;  citation_pages=415-42; " /><meta name="citation_reference" content="citation_title=An overview of pentatricopeptide repeat proteins and their applications;  citation_journal_title=Biochimie;  citation_year=2015;  citation_volume=113;  citation_pages=93-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/1/53/17463153/dsv036.pdf" /><meta name="description" content="Abstract.  The present study used a whole-genome, NGS resequencing-based mQTL-seq (multiple QTL-seq) strategy in two inter-specific mapping populations (Pusa 11" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/1/53/2462457" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/1/53/2462457" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsv036';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462457" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF+50BPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEYXMsIFNob3V2aWsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2luZ2gsIE1vaGFyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQnVyZWF1IG9mIFBsYW50IEdlbmV0aWMgUmVzb3VyY2VzIFJlZ2lvbmFsIFN0YXRpb24sIFNoaW1sYSwgSGltYWNoYWwgUHJhZGVzaDE3MTAwNCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTcml2YXN0YXZhLCBSaXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYWphaiwgRGVlcGFrIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNheGVuYSwgTWFuZWVzaGEgUy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFuYSwgSmFpIEMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQnVyZWF1IG9mIFBsYW50IEdlbmV0aWMgUmVzb3VyY2VzIChOQlBHUiksIE5ldyBEZWxoaTExMDAxMiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYW5zYWwsIEthaWxhc2ggQy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBCdXJlYXUgb2YgUGxhbnQgR2VuZXRpYyBSZXNvdXJjZXMgKE5CUEdSKSwgTmV3IERlbGhpMTEwMDEyLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlR5YWdpLCBBa2hpbGVzaCBLLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJpZGEsIFN3YXJ1cCBLLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Im1RVEwtc2VxIGRlbGluZWF0ZXMgZnVuY3Rpb25hbGx5IHJlbGV2YW50IGNhbmRpZGF0ZSBnZW5lIGhhcmJvdXJpbmcgYSBtYWpvciBRVEwgcmVndWxhdGluZyBwb2QgbnVtYmVyIGluIGNoaWNrcGVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmRzIG1hcmtlci1hc3Npc3RlZCBzZWxlY3Rpb24gaW4gcHVsc2VzOiBhIHJldmlldzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEzMDsgIGNpdGF0aW9uX3BhZ2VzPTI5Ny0zMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjaGlldmVtZW50cyBhbmQgcHJvc3BlY3RzIG9mIGdlbm9taWNzLWFzc2lzdGVkIGJyZWVkaW5nIGluIHRocmVlIGxlZ3VtZSBjcm9wcyBvZiB0aGUgc2VtaS1hcmlkIHRyb3BpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEFkdi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xMTIwLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3RlbnRpYWwgZWZmZWN0cyBvZiBpbmRpdmlkdWFsIHZlcnN1cyBzaW11bHRhbmVvdXMgY2xpbWF0ZSBjaGFuZ2UgZmFjdG9ycyBvbiBncm93dGggYW5kIHdhdGVyIHVzZSBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBQbGFudCBQcm9kLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTg5LTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xpbWF0ZSBjaGFuZ2UsIGFkYXB0YXRpb24sIGFuZCBwaGVub3R5cGljIHBsYXN0aWNpdHk6IHRoZSBwcm9ibGVtIGFuZCB0aGUgZXZpZGVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2wuIEFwcGwuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYmlvdGljIHN0cmVzc2VzLCBjb25zdHJhaW50cyBhbmQgaW1wcm92ZW1lbnQgc3RyYXRlZ2llcyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTE2My03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdC10cmFjayBpbnRyb2dyZXNzaW9uIG9mIOKAnFFUTC1ob3RzcG904oCdIGZvciByb290IHRyYWl0cyBhbmQgb3RoZXIgZHJvdWdodCB0b2xlcmFuY2UgdHJhaXQgaW4gSkcgMTEsIGFuIGVsaXRlIGFuZCBsZWFkaW5nIHZhcmlldHkgb2YgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXJrZXItYXNzaXN0ZWQgYmFja2Nyb3NzaW5nIHRvIGludHJvZ3Jlc3MgcmVzaXN0YW5jZSB0byBGdXNhcml1bSB3aWx0IHJhY2UgMSBhbmQgQXNjb2NoeXRhIGJsaWdodCBpbiBDMjE0LCBhbiBlbGl0ZSBjdWx0aXZhciBvZiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYW5kIGlkZW50aWZpY2F0aW9uIG9mIGEgQ2ljZXIgYXJpZXRpbnVtIE5TUDIgZ2VuZSBpbnZvbHZlZCBpbiBub2R1bGF0aW9uIHBhdGh3YXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI3OyAgY2l0YXRpb25fcGFnZXM9NDgxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVncmF0ZWQgZ2Vub21pYyBhcHByb2FjaCBmb3IgcmFwaWQgZGVsaW5lYXRpb24gb2YgY2FuZGlkYXRlIGdlbmVzIHJlZ3VsYXRpbmcgYWdyby1tb3JwaG9sb2dpY2FsIHRyYWl0cyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02OTUtNzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsbHkgcmVsZXZhbnQgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmcm9tIGNoaWNrcGVhIHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmVzIGZvciBlZmZpY2llbnQgZ2Vub3R5cGluZyBhcHBsaWNhdGlvbnMgYW5kIHRyYWl0IGFzc29jaWF0aW9uIG1hcHBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzU1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlZmZpY2llbnQgYW5kIGNvc3QtZWZmZWN0aXZlIGFwcHJvYWNoIGZvciBnZW5pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXItYmFzZWQgbGFyZ2Utc2NhbGUgdHJhaXQgYXNzb2NpYXRpb24gbWFwcGluZzogaWRlbnRpZmljYXRpb24gb2YgY2FuZGlkYXRlIGdlbmVzIGZvciBzZWVkIHdlaWdodCBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0yNDEtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhLWhpZ2ggZGVuc2l0eSBpbnRyYS1zcGVjaWZpYyBnZW5ldGljIGxpbmthZ2UgbWFwcyBhY2NlbGVyYXRlIGlkZW50aWZpY2F0aW9uIG9mIGZ1bmN0aW9uYWxseSByZWxldmFudCBtb2xlY3VsYXIgdGFncyBnb3Zlcm5pbmcgaW1wb3J0YW50IGFncm9ub21pYyB0cmFpdHMgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTQ2OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21lLXdpZGUgU05QIHNjYW4gYWNjZWxlcmF0ZXMgdHJhaXQtcmVndWxhdG9yeSBnZW5vbWljIGxvY2kgaWRlbnRpZmljYXRpb24gaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTExNjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm9nZW5lb3VzIGluYnJlZCBmYW1pbHkgKEhJRikgYW5hbHlzaXM6IGEgbWV0aG9kIGZvciBkZXZlbG9waW5nIG5lYXItaXNvZ2VuaWMgbGluZXMgdGhhdCBkaWZmZXIgYXQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xMDA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIHNldCBvZiBuZWFyIGlzb2dlbmljIGFuZCBiYWNrY3Jvc3MgcmVjb21iaW5hbnQgaW5icmVkIGxpbmVzIGNvbnRhaW5pbmcgbW9zdCBvZiB0aGUgTHljb3BlcnNpY29uIGhpcnN1dHVtIGdlbm9tZSBpbiBhIEwuIGVzY3VsZW50dW0gZ2VuZXRpYyBiYWNrZ3JvdW5kOiBhIHRvb2wgZm9yIGdlbmUgbWFwcGluZyBhbmQgZ2VuZSBkaXNjb3Zlcnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTgwMy0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgY29udHJvbGxpbmcgcm9vdCBncm93dGggYW5kIGFyY2hpdGVjdHVyZSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBjb25maXJtZWQgYnkgaGV0ZXJvZ2VuZW91cyBpbmJyZWQgZmFtaWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTc0Mi01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgZm9yIGNvbnN0cnVjdGluZyBhIHNhdHVyYXRlZCBsaW5rYWdlIG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGRldmVsb3BtZW50IG9mIGNvc3QtZWZmZWN0aXZlIFNOUCBtYXJrZXIgYXNzYXlzIGZvciBkaXZlcnNpdHkgYXNzZXNzbWVudCBhbmQgZ2VuZXRpYyBtYXBwaW5nIGluIGNoaWNrcGVhIGFuZCBjb21wYXJhdGl2ZSBtYXBwaW5nIGluIGxlZ3VtZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzE2LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZ2Vub3R5cGluZyBmb3IgYnJlZWRpbmcgYW5kIGdlbmV0aWNzIGFwcGxpY2F0aW9ucyBpbiBjaGlja3BlYSBhbmQgcGlnZW9ucGVhIHVzaW5nIHRoZSBCZWFkWHByZXNzIHBsYXRmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHdpZGUgU05QIGlkZW50aWZpY2F0aW9uIGluIGNoaWNrcGVhIGZvciB1c2UgaW4gZGV2ZWxvcG1lbnQgb2YgYSBoaWdoIGRlbnNpdHkgZ2VuZXRpYyBtYXAgYW5kIGltcHJvdmVtZW50IG9mIGNoaWNrcGVhIHJlZmVyZW5jZSBnZW5vbWUgYXNzZW1ibHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTcwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBpbmctYnktc2VxdWVuY2luZyBiYXNlZCBpbnRyYS1zcGVjaWZpYyBnZW5ldGljIG1hcCByZWZpbmVzIGEg4oCcUVRMLWhvdHNwb3TigJ0gcmVnaW9uIGZvciBkcm91Z2h0IHRvbGVyYW5jZSBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjkwOyAgY2l0YXRpb25fcGFnZXM9NTU5LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBjb25zZXJ2ZWQgbm9uLWNvZGluZyBtaWNyb3NhdGVsbGl0ZSAoQ05NUykgbWFya2VyLWJhc2VkIGludGVncmF0aXZlIGdlbmV0aWNhbCBnZW5vbWljcyBmb3IgcXVhbnRpdGF0aXZlIGRpc3NlY3Rpb24gb2Ygc2VlZCB3ZWlnaHQgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTEyNzEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tYmluYXRvcmlhbCBhcHByb2FjaCBvZiBjb21wcmVoZW5zaXZlIFFUTC1iYXNlZCBjb21wYXJhdGl2ZSBnZW5vbWUgbWFwcGluZyBhbmQgdHJhbnNjcmlwdCBwcm9maWxpbmcgaWRlbnRpZmllZCBhIHNlZWQgd2VpZ2h0LXJlZ3VsYXRpbmcgY2FuZGlkYXRlIGdlbmUgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTI2NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgZ2VuZXMgZm9yIGRvdWJsZSBwb2RkaW5nIGFuZCBvdGhlciBtb3JwaG9sb2dpY2FsIHRyYWl0cyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTI4NS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9REFGIG1hcmtlciB0aWdodGx5IGxpbmtlZCB0byBhIG1ham9yIGxvY3VzIGZvciBBc2NvY2h5dGEgYmxpZ2h0IHJlc2lzdGFuY2UgaW4gY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0yMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBtYXAgb2YgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLikgYmFzZWQgb24gcG9wdWxhdGlvbnMgZnJvbSBLYWJ1bGkgw5cgRGVzaSBjcm9zc2VzOiBsb2NhdGlvbiBvZiBnZW5lcyBmb3IgcmVzaXN0YW5jZSB0byBGdXNhcml1bSB3aWx0IHJhY2UgMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xMzQ3LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBRVEwgZm9yIEFzY29jaHl0YSBibGlnaHQgcmVzaXN0YW5jZSBpbiBhbiBSSUwgcG9wdWxhdGlvbiBkZXJpdmVkIGZyb20gYW4gaW50ZXJzcGVjaWZpYyBjcm9zcyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmFseXNpcyBvZiBhZ3Jvbm9taWMgdHJhaXRzIGluIGEgd2lkZSBjcm9zcyBvZiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RmllbGQgQ3JvcHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xMzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBpbiBjaGlja3BlYSBhc3NvY2lhdGVkIHdpdGggdGltZSB0byBmbG93ZXJpbmcgYW5kIHJlc2lzdGFuY2UgdG8gRGlkeW1lbGxhIHJhYmllaSB0aGUgY2F1c2FsIGFnZW50IG9mIEFzY29jaHl0YSBibGlnaHQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9MTM1Ny02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBtYXBwaW5nIG9mIEFzY29jaHl0YSBibGlnaHQgcmVzaXN0YW5jZSBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtKSB1c2luZyBhIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbGlua2FnZSBtYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTI2LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhbiBpbnRlZ3JhdGVkIGludHJhc3BlY2lmaWMgbWFwIG9mIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pIHVzaW5nIHR3byByZWNvbWJpbmFudCBpbmJyZWQgbGluZSBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz0yMDktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzZXMgYW5kIGNvbnNlcnZhdGlvbiBvZiBRVEwgZm9yIEFzY29jaHl0YSBibGlnaHQgcmVzaXN0YW5jZSBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz03NTctNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBtYXBwaW5nIG9mIHdpbHQgcmVzaXN0YW5jZSBnZW5lcyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNzctODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgUVRMIGZvciByZXNpc3RhbmNlIHRvIEJvdHJ5dGlzIGdyZXkgbW91bGQgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODI7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYSBtYWpvciBnZW5lIGZvciBncm93dGggaGFiaXQgYW5kIFFUTHMgZm9yIEFzY29jaHl0YSBibGlnaHQgcmVzaXN0YW5jZSBhbmQgZmxvd2VyaW5nIHRpbWUgaW4gYSBwb3B1bGF0aW9uIGJldHdlZW4gY2hpY2twZWEgYW5kIENpY2VyIHJldGljdWxhdHVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTczOyAgY2l0YXRpb25fcGFnZXM9MzA3LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBsYXJnZS1zZWVkZWQgaGlnaC15aWVsZGluZyBzdGFibGUga2FidWxpIGNoaWNrcGVhIGdlcm1wbGFzbSBsaW5lcyBmb3IgdXNlIGluIGNyb3AgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTk4LTIwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBRVEwgYXNzb2NpYXRlZCB3aXRoIHRyYWl0cyBhZmZlY3RpbmcgZ3JhaW4geWllbGQgaW4gY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLikgdW5kZXIgdGVybWluYWwgZHJvdWdodCBzdHJlc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTQ1MC02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiBJQ0NWMiDDlyBKRzYyIGNoaWNrcGVhIHByb2dlbmllcyBzaG93cyBzZW5zaXRpdml0eSBvZiByZXByb2R1Y3Rpb24gdG8gc2FsdCBzdHJlc3MgYW5kIHJldmVhbHMgUVRMcyBmb3Igc2VlZCB5aWVsZCBhbmQgeWllbGQgY29tcG9uZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz05LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWFwcGluZyBvZiBRVExzIGZvciByZXNpc3RhbmNlIHRvIEZ1c2FyaXVtIHdpbHQgKHJhY2UgMSkgYW5kIEFzY29jaHl0YSBibGlnaHQgaW4gY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz0xMjEtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFya2VyIGRpc2NvdmVyeSwgaW50cmFzcGVjaWZpYyBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gYW5kIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBhbmFseXNpcyBvZiBBc2NvY2h5dGEgYmxpZ2h0IHJlc2lzdGFuY2UgaW4gY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9Mjk3LTMxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXNzZWN0aW9uIG9mIGRyb3VnaHQgdG9sZXJhbmNlIGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyNzsgIGNpdGF0aW9uX3BhZ2VzPTQ0NS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMLXNlcTogcmFwaWQgbWFwcGluZyBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBpbiByaWNlIGJ5IHdob2xlIGdlbm9tZSByZXNlcXVlbmNpbmcgb2YgRE5BIGZyb20gdHdvIGJ1bGtlZCBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0xNzQtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTC1zZXEgaWRlbnRpZmllcyBhbiBlYXJseSBmbG93ZXJpbmcgUVRMIGxvY2F0ZWQgbmVhciBmbG93ZXJpbmcgbG9jdXMgVCBpbiBjdWN1bWJlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz0xNDkxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlcGxveWluZyBRVEwtc2VxIGZvciByYXBpZCBkZWxpbmVhdGlvbiBvZiBhIHBvdGVudGlhbCBjYW5kaWRhdGUgZ2VuZSB1bmRlcmx5aW5nIG1ham9yIHRyYWl0LWFzc29jaWF0ZWQgUVRMIGluIGNoaWNrcGVhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE5My0yMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGFuZCByZWxpYWJsZSBpZGVudGlmaWNhdGlvbiBvZiB0b21hdG8gZnJ1aXQgd2VpZ2h0IGFuZCBsb2N1bGUgbnVtYmVyIGxvY2kgYnkgUVRMLXNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTEzMjktNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtKSBwcm92aWRlcyBhIHJlc291cmNlIGZvciB0cmFpdCBpbXByb3ZlbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI0MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2luZyByZXZlYWxzIGFncm9ub21pY2FsbHkgaW1wb3J0YW50IGxvY2kgaW4gcmljZSB1c2luZyBNdXRNYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNzQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1wbG95aW5nIGdlbm9tZS13aWRlIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgc3RyYXRlZ3kgdG8gZXh0cmFwb2xhdGUgdGhlIG5hdHVyYWwgYWxsZWxpYyBkaXZlcnNpdHkgYW5kIGRvbWVzdGljYXRpb24gcGF0dGVybnMgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGNvbnRyb2wgb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgaW4gbW9kZWwgb3JnYW5pc20gYXNzb2NpYXRpb24gbWFwcGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc4OyAgY2l0YXRpb25fcGFnZXM9MTcwOS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gbWFwcGluZyBvZiBzYWxpbml0eSB0b2xlcmFuY2UgaW4gcmljZSAoT3J5emEgc2F0aXZhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xMzMtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hyb21vc29tYWwgZ2Vub21pY3MgYXBwcm9hY2ggdG8gYXNzZXNzIGFuZCB2YWxpZGF0ZSB0aGUgZGVzaSBhbmQga2FidWxpIGRyYWZ0IGNoaWNrcGVhIGdlbm9tZSBhc3NlbWJsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTc3OC04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJhYmlkb3BzaXMgZW1iMTc1IGFuZCBvdGhlciBwcHIga25vY2tvdXQgbXV0YW50cyByZXZlYWwgZXNzZW50aWFsIHJvbGVzIGZvciBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgKFBQUikgcHJvdGVpbnMgaW4gcGxhbnQgZW1icnlvZ2VuZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnRhOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyMTsgIGNpdGF0aW9uX3BhZ2VzPTQyNC0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGFuYWx5c2lzIG9mIGRpZmZlcmVudGlhbGx5IGV4cHJlc3NlZCBnZW5lcyBkdXJpbmcgc2VlZCBkZXZlbG9wbWVudCB1c2luZyBzdXBwcmVzc2lvbiBzdWJ0cmFjdGl2ZSBoeWJyaWRpemF0aW9uIChTU0gpIGluIFBoYXNlb2x1cyB2dWxnYXJpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NzE5LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBlbnRhdHJpY29wZXB0aWRlIHJlcGVhdC1jb250YWluaW5nIGdlbmUgcmVzdG9yZXMgZmVydGlsaXR5IHRvIGN5dG9wbGFzbWljIG1hbGUtc3RlcmlsZSBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTA4ODctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGVudGF0cmljb3BlcHRpZGUgcmVwZWF0LWNvbnRhaW5pbmcgZ2VuZSB0aGF0IHByb21vdGVzIHRoZSBwcm9jZXNzaW5nIG9mIGFiZXJyYW50IGF0cDYgUk5BIG9mIGN5dG9wbGFzbWljIG1hbGUtc3RlcmlsZSByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU0NDsgIGNpdGF0aW9uX3BhZ2VzPTk5LTEwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgcGVudGF0cmljb3BlcHRpZGUgcmVwZWF0IHByb3RlaW4gZ2VuZSwgb3JmNjg3LCB0aGF0IHJlc3RvcmVzIGZlcnRpbGl0eSBpbiB0aGUgY3l0b3BsYXNtaWMgbWFsZS1zdGVyaWxlIEtvc2VuYSByYWRpc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NDA3LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgcHJvdGVpbnMgYW5kIHRoZWlyIGVtZXJnaW5nIHJvbGVzIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01MjEtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlbnRhdHJpY29wZXB0aWRlIHJlcGVhdCBwcm90ZWluczogYSBzb2NrZXQgc2V0IGZvciBvcmdhbmVsbGUgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTY2My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuaWMgbWFya2VycyBmb3IgV0EgY3l0b3BsYXNtIGJhc2VkIG1hbGUgc3RlcmlsaXR5IGFuZCBpdHMgZmVydGlsaXR5IHJlc3RvcmF0aW9uIGluIHJpY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Mjc1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNydXB0aW9uIG9mIGEgcmljZSBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgcHJvdGVpbiBjYXVzZXMgYSBzZWVkbGluZy1zcGVjaWZpYyBhbGJpbm8gcGhlbm90eXBlIGFuZCBpdHMgdXRpbGl6YXRpb24gdG8gZW5oYW5jZSBzZWVkIHB1cml0eSBpbiBoeWJyaWQgcmljZSBwcm9kdWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTk7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlbnRhdHJpY29wZXB0aWRlIHJlcGVhdCBwcm90ZWlucyBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz00MTUtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIG92ZXJ2aWV3IG9mIHBlbnRhdHJpY29wZXB0aWRlIHJlcGVhdCBwcm90ZWlucyBhbmQgdGhlaXIgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9OTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzEvNTMvMTc0NjMxNTMvZHN2MDM2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgcHJlc2VudCBzdHVkeSB1c2VkIGEgd2hvbGUtZ2Vub21lLCBOR1MgcmVzZXF1ZW5jaW5nLWJhc2VkIG1RVEwtc2VxIChtdWx0aXBsZSBRVEwtc2VxKSBzdHJhdGVneSBpbiB0d28gaW50ZXItc3BlY2lmaWMgbWFwcGluZyBwb3B1bGF0aW9ucyAoUHVzYSAxMSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjQ2MjQ1NyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGRLIxWp9OpNO5SRQVocIlSCdJCjcQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f53%2f2462457" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f53%2f2462457" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=1509239038&amp;Signature=F7u4laj4ABQ5sYoFVBpFOGdyf7FoQoG7bQWH7dJ4DXYNgkpcNxZQ2EpU7YfkoDO6F-lGI3ZfA5VL7koxn4o5CDgLMqzKy5v8yybxzfzSehFl4ldGvQrT5UqopkQKyYBPvChHggRddAjeO8zp79HTQYw9M5JaeSzYrlFA2JT7tXQbkVN1l3O-tP~Sq04zQJHwWyxMQ1jj6I3k~hRY707hxXS5Hf5UVCHtDXsJJACSekCcXfw1L0p8Ko1uSu7EsODqjHz0um0M8j5eKsHjtvJMA0mwbiBeXdqNJ7pAGULMg5Qnptm4NbvuEJylVLXTCdQ26DG9roIAGdzb0Fea9bjXTQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/1">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199560">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199560">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199562">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199562">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199572">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199572">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199590">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199590">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199649">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199649">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199652">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199652">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199655">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199655">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/1/43/2462442">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/1/67/2462472">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Shouvik Das</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shouvik Das</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shouvik+Das">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Das S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Das author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Mohar Singh</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mohar Singh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">National Bureau of Plant Genetic Resources Regional Station</div>, <div class="addr-line">Shimla, Himachal Pradesh</div><div class="addr-line">171004</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mohar+Singh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Singh M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Singh author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Rishi Srivastava</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rishi Srivastava</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Rishi+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Srivastava author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Deepak Bajaj</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deepak Bajaj</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Deepak+Bajaj">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bajaj D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Bajaj author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Maneesha S. Saxena</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maneesha S. Saxena</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Maneesha+S.+Saxena">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saxena M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Saxena author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jai C. Rana</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jai C. Rana</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Bureau of Plant Genetic Resources (NBPGR)</div>, <div class="addr-line">New Delhi</div><div class="addr-line">110012</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jai+C.+Rana">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rana J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Rana author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Kailash C. Bansal</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kailash C. Bansal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Bureau of Plant Genetic Resources (NBPGR)</div>, <div class="addr-line">New Delhi</div><div class="addr-line">110012</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kailash+C.+Bansal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bansal K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Bansal author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Akhilesh K. Tyagi</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh K. Tyagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Akhilesh+K.+Tyagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tyagi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Tyagi author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Swarup K. Parida</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swarup K. Parida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 11-26735228. Fax. +91 11-26741658. E-mail: <a href="mailto:swarup@nipgr.ac.in" target="_blank">swarup@nipgr.ac.in</a>; <a href="mailto:swarupdbt@gmail.com" target="_blank">swarupdbt@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Swarup+K.+Parida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parida S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Parida author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 53–65, <a href='https://doi.org/10.1093/dnares/dsv036'>https://doi.org/10.1093/dnares/dsv036</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">19 December 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 July 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 November 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462457" href="/dnaresearch/article-pdf/23/1/53/17463153/dsv036.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shouvik Das, Mohar Singh, Rishi Srivastava, Deepak Bajaj, Maneesha S. Saxena, Jai C. Rana, Kailash C. Bansal, Akhilesh K. Tyagi, Swarup K. Parida;  mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea, <em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 53–65, <a href='https://doi.org/10.1093/dnares/dsv036'>https://doi.org/10.1093/dnares/dsv036</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462457?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=mQTL-seq%20delineates%20functionally%20relevant%20candidate%20gene%20harbouring%20a%20major%20QTL%20regulating%20pod%20number%20in%20chickpea&amp;publicationDate=2015-12-19&amp;volumeNum=23&amp;issueNum=1&amp;author=Das%2C%20Shouvik%3B%20Singh%2C%20Mohar&amp;startPage=53&amp;endPage=65&amp;contentId=10.1093%2Fdnares%2Fdsv036&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f53%2f2462457" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199560 id="87199560" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The present study used a whole-genome, NGS resequencing-based mQTL-seq (multiple QTL-seq) strategy in two inter-specific mapping populations (Pusa 1103 × ILWC 46 and Pusa 256 × ILWC 46) to scan the major genomic region(s) underlying QTL(s) governing pod number trait in chickpea. Essentially, the whole-genome resequencing of low and high pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of these two mapping populations discovered &gt;8 million high-quality homozygous SNPs with respect to the reference <em>kabuli</em> chickpea. The functional significance of the physically mapped SNPs was apparent from the identified 2,264 non-synonymous and 23,550 regulatory SNPs, with 8–10% of these SNPs-carrying genes corresponding to transcription factors and disease resistance-related proteins. The utilization of these mined SNPs in Δ (SNP index)-led QTL-seq analysis and their correlation between two mapping populations based on mQTL-seq, narrowed down two (<em>Caq<sup>a</sup>PN4.1</em>: 867.8 kb and <em>Caq<sup>a</sup>PN4.2</em>: 1.8 Mb) major genomic regions harbouring robust pod number QTLs into the high-resolution short QTL intervals (<em>Caq<sup>b</sup>PN4.1</em>: 637.5 kb and <em>Caq<sup>b</sup>PN4.2</em>: 1.28 Mb) on chickpea chromosome 4. The integration of mQTL-seq-derived one novel robust QTL with QTL region-specific association analysis delineated the regulatory (C/T) and coding (C/A) SNPs-containing one pentatricopeptide repeat (PPR) gene at a major QTL region regulating pod number in chickpea. This target gene exhibited anther, mature pollen and pod-specific expression, including pronounced higher up-regulated (∼3.5-folds) transcript expression in high pod number-containing parental accessions and homozygous individuals of two mapping populations especially during pollen and pod development. The proposed mQTL-seq-driven combinatorial strategy has profound efficacy in rapid genome-wide scanning of potential candidate gene(s) underlying trait-associated high-resolution robust QTL(s), thereby expediting genomics-assisted breeding and genetic enhancement of crop plants, including chickpea.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="chickpea">chickpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mQTL-seq">mQTL-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;pod number&quot;">pod number</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;wild accessions&quot;">wild accessions</a></div>                            <h2 scrollto-destination=87199562 id="87199562" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Chickpea (<em>Cicer arietinum</em> L.) is the third most vital food legume with a genome size of ∼720 Mb.<sup><a reveal-id="DSV036C1" data-open="DSV036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV036C2" data-open="DSV036C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The current climatic conditions are adversely affecting the world-wide chickpea productivity to a large extent by highly variable phenotypic plasticity in conjunction with multiple abiotic and biotic stresses, including drought, terminal heat stress, salinity, <em>Fusarium</em> wilt and <em>Ascochyta</em> blight.<sup><a reveal-id="DSV036C2  DSV036C5" data-open="DSV036C2  DSV036C5" class="link link-ref link-reveal xref-bibr">2–5</a></sup> Therefore, the primary goal of present chickpea genomics is focused towards minimizing the effect of climate change and biotic/abiotic stresses upon chickpea yield and productivity by developing genetically tailored high-yielding stress-tolerant chickpea cultivars. However, most of the stress tolerance and yield component traits targeted for chickpea genetic enhancement usually have complex genetic architecture and are regulated by multiple major and/or minor genes/QTLs (quantitative trait loci). It is thus imperative to delineate the potential genes/alleles underlying QTLs associated with various traits of interest (yield contributing and stress-tolerant traits) by fine mapping/map-based cloning prior to their effective deployment in marker-assisted genetic enhancement of chickpea. So far, very limited attention has been paid towards fine mapping of trait-regulatory QTLs and their subsequent utilization in marker-assisted genetic improvement of chickpea.<sup><a reveal-id="DSV036C6  DSV036C13" data-open="DSV036C6  DSV036C13" class="link link-ref link-reveal xref-bibr">6–13</a></sup> The narrow genetic base including low marker genetic polymorphism especially between parental accessions of diverse intra-specific mapping populations along with inadequate accessibility of high-density, intra-specific genetic linkage maps are the major bottlenecks in identification and fine mapping of QTLs in chickpea. In this perspective, an alternative genome-wide strategy is crucial for rapid molecular mapping of trait-associated high-resolution QTLs/genes, which in turn will benefit marker-assisted breeding in chickpea.</p><p>To accelerate molecular mapping of QTLs in chickpea, traditional QTL mapping strategy<sup><a reveal-id="DSV036C14  DSV036C16" data-open="DSV036C14  DSV036C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> that primarily relies on genome-wide discovery and large-scale genotyping of simple sequence repeat (SSR) and single-nucleotide polymorphism (SNP) markers in individuals of diverse inter-/intra-specific mapping populations (recombinant inbred and near-isogenic lines) by using multiple high-throughput genotyping assays, including NGS (next-generation sequencing)-based GBS (genotyping-by-sequencing) assay, are found to be most fruitful in chickpea.<sup><a reveal-id="DSV036C9  DSV036C13" data-open="DSV036C9  DSV036C13" class="link link-ref link-reveal xref-bibr">9–13</a>, <a reveal-id="DSV036C17  DSV036C23" data-open="DSV036C17  DSV036C23" class="link link-ref link-reveal xref-bibr">17–23</a></sup> This approach successfully identified a number of low-resolution major QTLs governing various yield contributing (i.e. flowering and maturation time, seed size/100-seed weight, double podding and seed/pod number per plant) and abiotic/biotic stress tolerance traits (viz. salinity and drought tolerance root traits, <em>Fusarium</em> wilt, <em>Ascochyta</em> blight and <em>Botrytis</em> gray mold) in chickpea.<sup><a reveal-id="DSV036C1" data-open="DSV036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV036C2" data-open="DSV036C2" class="link link-ref link-reveal xref-bibr">2</a>, <a reveal-id="DSV036C8  DSV036C13" data-open="DSV036C8  DSV036C13" class="link link-ref link-reveal xref-bibr">8–13</a>, <a reveal-id="DSV036C22  DSV036C41" data-open="DSV036C22  DSV036C41" class="link link-ref link-reveal xref-bibr">22–41</a></sup> In light of the added advantages and its broader applicability vis-à-vis the diverse aforesaid traditional QTL mapping approaches, the high-throughput NGS-based QTL-seq has recently been used fruitfully as a cost-effective strategy for rapid genome-wide scanning and mapping of major QTLs governing multiple qualitative and quantitative traits such as seedling vigour and blast resistance in rice, flowering time in cucumber, fruit weight and locule number in tomato, and 100-seed weight in chickpea.<sup><a reveal-id="DSV036C42  DSV036C45" data-open="DSV036C42  DSV036C45" class="link link-ref link-reveal xref-bibr">42–45</a></sup> Nonetheless, most of the major QTLs identified through QTL-seq are being localized within sizeable long genetic marker intervals of chromosomes, which requires large-scale validation in multiple mapping populations and subsequent fine mapping through traditional QTL mapping. This will ensure the effective practical utilization of these well-validated robust QTLs/genes in marker-assisted genetic improvement of chickpea.</p><p>Simultaneously to expedite the above-mentioned, a multiple QTL-seq (mQTL-seq) approach that involves QTL-seq analysis in multiple mapping populations derived from the common parental accessions could be an attractive genome-wide strategy. Essentially, mQTL-seq deals with NGS-based whole-genome resequencing of DNA bulks (showing two extreme contrasting phenotypic trait values) of progenies that are derived from each of the multiple segregating mapping population with at least single common parental accession (detailed strategy illustrated in Fig. <a reveal-id="DSV036F1" data-open="DSV036F1" class="link link-reveal link-table xref-fig">1</a>). Collectively, the utility of this strategy is evident from the large-scale validation of major QTLs derived from QTL-seq of each preliminary/advanced generation mapping population in diverse genetic backgrounds (multiple mapping populations) along with its efficiency to scale down each QTL-seq originated QTLs into potential candidate genes regulating diverse agronomic traits in chickpea. Henceforth, implementing this mQTL-seq approach can be beneficial for quick genome-wide scanning and fine mapping of trait-associated major genes harbouring robust QTLs (validated in diverse mapping populations) with optimal expense of resources in chickpea. This will provide much needed inputs for molecular dissection of complex quantitative traits culminating into genomics-assisted crop improvement in chickpea at a faster pace. </p>                        <a id="87199566" scrollto-destination="87199566"></a>
<div id="DSV036F1" data-id="DSV036F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03601.png?Expires=1509239038&Signature=Pegb5-VOTlbZfp1PkdAHi7dSgJz2rzGp7EoVXiqP7HrNQmkRgSalrQwxxj4~fPdGTAVl8cgBLdO8SpdvleW41P-3pRUx9qIbmhTTL3HBDPOY0e3jfHBYpgKelhnIjKswgLhPiZeyiLUAG7DGvqqm7iUXURKm~pZlpLXSfEyvlGvoPo19mqS1ISOCgypcD8l6l6XD5SC~n-f7URdaqvtvvklDiWpnsFOX0gBwWi~MiQaZ5ptKDRMUa-adP0SA57Ro8rxf3iVPHXyKi-zjuFm49MH6i1tBsGcdyPcRgo3G7y~auHOaCKJb3-~7S-XFB-4N2zK61Miw681vPWgNMZO4KQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(A) Development of multiple mapping populations by inter-crossing of parental accessions (with at least one common parent) contrasting for a particular agronomic trait such as pod number. LPNP: low pod number mapping parent (green colour) and HPNP1 and 2: high pod number mapping parents 1 (red colour) and 2 (blue colour). LPNP is considered the common parent. (B) Genetic inheritance study on normal frequency distribution of target trait (such as pod number) in mapping populations for selection of homozygous individuals (constituting the bulks) from each mapping population exhibiting two extreme contrasting phenotypic trait values (like low and high pod number). LPNB and HPNB: low and high pod number bulks. (C) The Δ (SNP index)-based QTL-seq analysis (following detailed strategies of Takagi et al.42 and Das et al.44) using NGS-genome resequences of parents and bulks from each of the multiple mapping populations to identify major genomic regions that underlie QTLs governing particular agronomic trait in chickpea. The major QTLs identified from each of the multiple mapping populations are localized at varying physical sequence intervals (Mb) spanned by flanking SNP markers on the chromosomes. For instance, QTL-seq analysis in two contrasting pod number mapping populations identified two genomic regions harbouring two major pod number QTLs (PNQTL1 and PNQTL2) that spanned X and Y-Mb sequence intervals between M1 and M2 as well as M3 and M4 flanking SNP markers, respectively, on the same chromosomes. (D) Integration of QTL-seq-derived individual QTL outcomes from multiple mapping populations (with at least one common parent) called ‘mQTL-seq (multiple QTL-seq)’ for large-scale validation of these identified major QTLs in multiple genetic backgrounds to scan the robust QTLs, and further to narrow down the longer robust QTL intervals into shorter genomic intervals for delineation of candidate gene(s) regulating the trait under study. The integration of QTL-seq-derived two pod number QTLs (PNQTL1 and PNQTL2) identified from two mapping populations enabled to validate these QTLs in two diverse genetic backgrounds to ascertain their robustness for pod number regulation in chickpea. Further, the mQTL-seq analysis scaled-down a longer QTL interval (spanned X Mb) into a shorter pod number robust QTL interval (spanned Y Mb) flanking within M3 and M4 SNP markers on the chromosome; this can serve as a potential target genomic region for rapid identification and delineation of candidate gene(s) regulating pod number in chickpea. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199566/dsv03601.tiff" path-from-xml="dsv03601.tiff" target="_blank">View large</a><a section="87199566" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03601.png?Expires=1509239038&Signature=UXKcZixYIzA~Hoj2FYP7Jvr8ziPTJpg9PTT7VhyBizFzXjxzyTvzN1Rqujrjhmj7uJvQvUHX~LEYBtknj281uLwmI-ox6saOuCsM-Xm7zbSV2FefdSj8BRwMY1c4uoiSK67eDfr6b0q2TGnxo3b25LFJAYbBhJY0vjrWmq0R6CWHJ-yhonMn4nzARD-bBYm70OrJ8cMkKheYsiv1OWWaGv-Y2CWKPbAkWsFF1vglSGtCheqBK3daNxOBWLx0VkKSfm6x-SvgYr~IlXdo0McO3UNdGNlCEWSBSzLFJhMMb1eurDPqVfrNt-GhBturQ2HxqmeuRzpUbNPZsolO5Pl1vQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199566&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Development of multiple mapping populations by inter-crossing of parental accessions (with at least one common parent) contrasting for a particular agronomic trait such as pod number. LPNP: low pod number mapping parent (green colour) and HPNP1 and 2: high pod number mapping parents 1 (red colour) and 2 (blue colour). LPNP is considered the common parent. (B) Genetic inheritance study on normal frequency distribution of target trait (such as pod number) in mapping populations for selection of homozygous individuals (constituting the bulks) from each mapping population exhibiting two extreme contrasting phenotypic trait values (like low and high pod number). LPNB and HPNB: low and high pod number bulks. (C) The Δ (SNP index)-based QTL-seq analysis (following detailed strategies of Takagi et al.<sup><a reveal-id="DSV036C42" data-open="DSV036C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and Das et al.<sup><a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup>) using NGS-genome resequences of parents and bulks from each of the multiple mapping populations to identify major genomic regions that underlie QTLs governing particular agronomic trait in chickpea. The major QTLs identified from each of the multiple mapping populations are localized at varying physical sequence intervals (Mb) spanned by flanking SNP markers on the chromosomes. For instance, QTL-seq analysis in two contrasting pod number mapping populations identified two genomic regions harbouring two major pod number QTLs (<em>PNQTL1</em> and <em>PNQTL2</em>) that spanned X and Y-Mb sequence intervals between M1 and M2 as well as M3 and M4 flanking SNP markers, respectively, on the same chromosomes. (D) Integration of QTL-seq-derived individual QTL outcomes from multiple mapping populations (with at least one common parent) called ‘mQTL-seq (multiple QTL-seq)’ for large-scale validation of these identified major QTLs in multiple genetic backgrounds to scan the robust QTLs, and further to narrow down the longer robust QTL intervals into shorter genomic intervals for delineation of candidate gene(s) regulating the trait under study. The integration of QTL-seq-derived two pod number QTLs (<em>PNQTL1</em> and <em>PNQTL2</em>) identified from two mapping populations enabled to validate these QTLs in two diverse genetic backgrounds to ascertain their robustness for pod number regulation in chickpea. Further, the mQTL-seq analysis scaled-down a longer QTL interval (spanned X Mb) into a shorter pod number robust QTL interval (spanned Y Mb) flanking within M3 and M4 SNP markers on the chromosome; this can serve as a potential target genomic region for rapid identification and delineation of candidate gene(s) regulating pod number in chickpea. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV036F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03601.png?Expires=1509239038&Signature=Pegb5-VOTlbZfp1PkdAHi7dSgJz2rzGp7EoVXiqP7HrNQmkRgSalrQwxxj4~fPdGTAVl8cgBLdO8SpdvleW41P-3pRUx9qIbmhTTL3HBDPOY0e3jfHBYpgKelhnIjKswgLhPiZeyiLUAG7DGvqqm7iUXURKm~pZlpLXSfEyvlGvoPo19mqS1ISOCgypcD8l6l6XD5SC~n-f7URdaqvtvvklDiWpnsFOX0gBwWi~MiQaZ5ptKDRMUa-adP0SA57Ro8rxf3iVPHXyKi-zjuFm49MH6i1tBsGcdyPcRgo3G7y~auHOaCKJb3-~7S-XFB-4N2zK61Miw681vPWgNMZO4KQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(A) Development of multiple mapping populations by inter-crossing of parental accessions (with at least one common parent) contrasting for a particular agronomic trait such as pod number. LPNP: low pod number mapping parent (green colour) and HPNP1 and 2: high pod number mapping parents 1 (red colour) and 2 (blue colour). LPNP is considered the common parent. (B) Genetic inheritance study on normal frequency distribution of target trait (such as pod number) in mapping populations for selection of homozygous individuals (constituting the bulks) from each mapping population exhibiting two extreme contrasting phenotypic trait values (like low and high pod number). LPNB and HPNB: low and high pod number bulks. (C) The Δ (SNP index)-based QTL-seq analysis (following detailed strategies of Takagi et al.42 and Das et al.44) using NGS-genome resequences of parents and bulks from each of the multiple mapping populations to identify major genomic regions that underlie QTLs governing particular agronomic trait in chickpea. The major QTLs identified from each of the multiple mapping populations are localized at varying physical sequence intervals (Mb) spanned by flanking SNP markers on the chromosomes. For instance, QTL-seq analysis in two contrasting pod number mapping populations identified two genomic regions harbouring two major pod number QTLs (PNQTL1 and PNQTL2) that spanned X and Y-Mb sequence intervals between M1 and M2 as well as M3 and M4 flanking SNP markers, respectively, on the same chromosomes. (D) Integration of QTL-seq-derived individual QTL outcomes from multiple mapping populations (with at least one common parent) called ‘mQTL-seq (multiple QTL-seq)’ for large-scale validation of these identified major QTLs in multiple genetic backgrounds to scan the robust QTLs, and further to narrow down the longer robust QTL intervals into shorter genomic intervals for delineation of candidate gene(s) regulating the trait under study. The integration of QTL-seq-derived two pod number QTLs (PNQTL1 and PNQTL2) identified from two mapping populations enabled to validate these QTLs in two diverse genetic backgrounds to ascertain their robustness for pod number regulation in chickpea. Further, the mQTL-seq analysis scaled-down a longer QTL interval (spanned X Mb) into a shorter pod number robust QTL interval (spanned Y Mb) flanking within M3 and M4 SNP markers on the chromosome; this can serve as a potential target genomic region for rapid identification and delineation of candidate gene(s) regulating pod number in chickpea. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199566/dsv03601.tiff" path-from-xml="dsv03601.tiff" target="_blank">View large</a><a section="87199566" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03601.png?Expires=1509239038&Signature=UXKcZixYIzA~Hoj2FYP7Jvr8ziPTJpg9PTT7VhyBizFzXjxzyTvzN1Rqujrjhmj7uJvQvUHX~LEYBtknj281uLwmI-ox6saOuCsM-Xm7zbSV2FefdSj8BRwMY1c4uoiSK67eDfr6b0q2TGnxo3b25LFJAYbBhJY0vjrWmq0R6CWHJ-yhonMn4nzARD-bBYm70OrJ8cMkKheYsiv1OWWaGv-Y2CWKPbAkWsFF1vglSGtCheqBK3daNxOBWLx0VkKSfm6x-SvgYr~IlXdo0McO3UNdGNlCEWSBSzLFJhMMb1eurDPqVfrNt-GhBturQ2HxqmeuRzpUbNPZsolO5Pl1vQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199566&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Development of multiple mapping populations by inter-crossing of parental accessions (with at least one common parent) contrasting for a particular agronomic trait such as pod number. LPNP: low pod number mapping parent (green colour) and HPNP1 and 2: high pod number mapping parents 1 (red colour) and 2 (blue colour). LPNP is considered the common parent. (B) Genetic inheritance study on normal frequency distribution of target trait (such as pod number) in mapping populations for selection of homozygous individuals (constituting the bulks) from each mapping population exhibiting two extreme contrasting phenotypic trait values (like low and high pod number). LPNB and HPNB: low and high pod number bulks. (C) The Δ (SNP index)-based QTL-seq analysis (following detailed strategies of Takagi et al.<sup><a reveal-id="DSV036C42" data-open="DSV036C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and Das et al.<sup><a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup>) using NGS-genome resequences of parents and bulks from each of the multiple mapping populations to identify major genomic regions that underlie QTLs governing particular agronomic trait in chickpea. The major QTLs identified from each of the multiple mapping populations are localized at varying physical sequence intervals (Mb) spanned by flanking SNP markers on the chromosomes. For instance, QTL-seq analysis in two contrasting pod number mapping populations identified two genomic regions harbouring two major pod number QTLs (<em>PNQTL1</em> and <em>PNQTL2</em>) that spanned X and Y-Mb sequence intervals between M1 and M2 as well as M3 and M4 flanking SNP markers, respectively, on the same chromosomes. (D) Integration of QTL-seq-derived individual QTL outcomes from multiple mapping populations (with at least one common parent) called ‘mQTL-seq (multiple QTL-seq)’ for large-scale validation of these identified major QTLs in multiple genetic backgrounds to scan the robust QTLs, and further to narrow down the longer robust QTL intervals into shorter genomic intervals for delineation of candidate gene(s) regulating the trait under study. The integration of QTL-seq-derived two pod number QTLs (<em>PNQTL1</em> and <em>PNQTL2</em>) identified from two mapping populations enabled to validate these QTLs in two diverse genetic backgrounds to ascertain their robustness for pod number regulation in chickpea. Further, the mQTL-seq analysis scaled-down a longer QTL interval (spanned X Mb) into a shorter pod number robust QTL interval (spanned Y Mb) flanking within M3 and M4 SNP markers on the chromosome; this can serve as a potential target genomic region for rapid identification and delineation of candidate gene(s) regulating pod number in chickpea. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>As a proof of concept, we made an effort to implement an NGS-based genome-wide high-throughput mQTL-seq approach in two F<sub>5</sub> mapping populations of <em>C. arietinum</em> and <em>C. reticulatum</em> inter-specific crosses (Pusa 1103 × ILWC 46 and Pusa 256 × ILWC 46) for identifying the major genomic regions harbouring the robust QTLs associated with pod number per plant in chickpea. The integration of mQTL-seq with QTL region-specific association analysis and differential expression profiling delineated a potential novel candidate gene and natural allelic variants at the major QTL interval governing pod number in chickpea.</p>                            <h2 scrollto-destination=87199572 id="87199572" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199574 id="87199574" class="section-title" data-legacy-id=s2a>2.1. Development of inter-specific mapping populations and their phenotyping for pod number</h3>
<p>Two inter-specific F<sub>5</sub> mapping populations of Pusa 1103× ILWC 46 (population size, 102) and Pusa 256× ILWC 46 (population size, 98) derived from high (cultivated <em>C. arietinum desi</em> accessions Pusa 1103 and Pusa 256 with 129 and 120 pods/plant, respectively) and low (wild <em>C. reticulatum</em> accession ILWC 46 with 10 pods/plant) pod number-containing parental accessions were developed using single seed descent method. The wild accession ILWC 46 serves as a common parent for both the populations generated. The mapping individuals and parental accessions of both populations were grown (following random complete block design with two replications) and phenotyped in the real field conditions at two diverse geographical locations of India (CSKHPKV, Palampur: latitude 32.1°N and longitude 76.5°E and NBPGR, New Delhi: 28.6°N and 77.2°E) for two consecutive years. A set of 15–20 representative plants were selected from each mapping individual and parental accession of both populations, and pod number (PN) was measured by counting the average number of fully developed pods per plant at maturity stage. The mean, standard deviation, coefficient of variation (CV%), broad-sense heritability (H<sup>2</sup>) and frequency distribution of PN (based on ANOVA) were measured in both populations individually as per Kujur et al.<sup><a reveal-id="DSV036C10" data-open="DSV036C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV036C11" data-open="DSV036C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and Bajaj et al.<sup><a reveal-id="DSV036C23" data-open="DSV036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> To determine the trait inheritance pattern, the effects/interactions among accessions/genotypes (G) and environments (E) (i.e. experimental years and geographical locations) were measured. The broad-sense heritability among accessions [<em>H</em><sup>2</sup> = <em>σ</em><sup>2</sup>g/(<em>σ</em><sup>2</sup>g + <em>σ</em><sup>2</sup>ge/<em>n</em> + <em>σ</em><sup>2</sup>e/nr)] was measured based on <em>σ</em><sup>2</sup>g (genetic), <em>σ</em><sup>2</sup>ge (G x E) and <em>σ</em><sup>2</sup>e (error) variance with <em>n</em> (number of experimental years/environments) = 2 and <em>r</em> (number of replicates) = 2. Additionally, 92 <em>desi</em> and <em>kabuli</em> chickpea accessions representing a PN-specific association panel were also grown in the field and phenotyped for pod number following aforesaid methods.</p>                            <h3 scrollto-destination=87199576 id="87199576" class="section-title" data-legacy-id=s2b>2.2. Whole-genome resequencing and mQTL-seq analysis</h3>
<p>According to the preliminary clues obtained from our traditional QTL mapping analysis, 10 of each high and low pod number-containing homozygous mapping individuals (a total of 20 individuals) representing two utmost ends of PN normal frequency distribution curve were selected individually from the mapping populations of Pusa 1103 × ILWC 46 and Pusa 256 × ILWC 46 for QTL-seq analysis. The homozygous genetic constitution of these selected 20 individuals from each of the two mapping populations for either of the high and low pod number trait was ascertained by utilizing their PN field phenotyping information and genotyping data of 96 genome-wide SSR markers in QTL mapping. The isolation and quantification of genomic DNA, pooling of equal concentration DNA to constitute low pod number bulk (LPNB) and high pod number bulk (HPNB), construction and sequencing of pair-end sequencing libraries (100-bp read length) using HiSeq2000 (Illumina Inc., San Diego, CA, USA) NGS platform and generation of high-quality genomic sequences were performed according to Das et al.<sup><a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The BWA with default parameters was used to align and map the high-quality sequence reads onto the reference <em>kabuli</em> genome.<sup><a reveal-id="DSV036C46" data-open="DSV036C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Subsequently, normalization of the mappable sequence reads based on depth of read coverage among mapping parents and individuals constituting LPNB and HPNB was performed. The high-quality homozygous SNPs (minimum sequence read depth: 10 with SNP base quality ≥20) between two parental accessions as well as among mapping individuals constituting the LPNB and HPNB were discovered and further structurally and functionally annotated with respect to reference genome as per Kujur et al.<sup><a reveal-id="DSV036C12" data-open="DSV036C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV036C13" data-open="DSV036C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>A SNP index and Δ (SNP index)-based QTL-seq approach was deployed individually in two mapping populations to scan major PN QTLs in chickpea following the earlier defined recommended parameters.<sup><a reveal-id="DSV036C42  DSV036C45" data-open="DSV036C42  DSV036C45" class="link link-ref link-reveal xref-bibr">42–45</a>, <a reveal-id="DSV036C47" data-open="DSV036C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The Δ (SNP index) was estimated according to subtraction of SNP index (proportion of sequence reads supported the SNPs, which are entirely different from the reference <em>kabuli</em> genome sequences) between LPNB and HPNB. The SNP index was estimated as ‘0’ and ‘1’ based on the representation of genomic fragments derived from Pusa 1103/Pusa 256 and ILWC 46, respectively, in entire high-quality sequence reads generated. A sliding window approach with 1-Mb window size and 10-kb increment was utilized to measure the average distribution of Δ (SNP index) of SNPs physically mapped across eight <em>kabuli</em> chromosomes in a given genomic interval. The SNP index plots were generated for two mapping populations individually by plotting the Δ (SNP index) of LPNB and HPNB and their corresponding SNP index within the specified window size in the graphs. The statistical confidence intervals of Δ (SNP index) with a given sequence read depth under the null hypothesis of no QTLs were calculated to assure the accuracy of QTLs derived from QTL-seq as per Takagi et al.,<sup><a reveal-id="DSV036C42" data-open="DSV036C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Lu et al.<sup><a reveal-id="DSV036C43" data-open="DSV036C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and Das et al.<sup><a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                            <h3 scrollto-destination=87199580 id="87199580" class="section-title" data-legacy-id=s2c>2.3. QTL region-specific association analysis</h3>
<p>The major genomic regions underlying PN QTLs identified from two mapping populations individually using QTL-seq were compared/correlated to identify the consensus robust QTLs that are well-validated across these diverse populations. One of the selected novel genomic region harbouring robust PN QTL was sequenced in low and high pod number-containing homozygous mapping individuals (constituting LPNB and HPNB in QTL-seq analysis) and parental accessions from each of the two mapping populations using the multiplexed amplicons sequencing protocol (as per manufacturer's instructions) of TruSeq Custom Amplicon v1.5 in Illumina MiSeq next-generation sequencer. The mapping of high-quality amplicon sequence reads onto the reference <em>kabuli</em> genome and detection of high-quality SNPs among parental accessions and mapping individuals was performed as per Saxena et al.<sup><a reveal-id="DSV036C9" data-open="DSV036C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Kujur et al.<sup><a reveal-id="DSV036C12" data-open="DSV036C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV036C48" data-open="DSV036C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The SNPs discovered at the sequenced genomic region harbouring robust PN QTL was structurally and functionally annotated following Kujur et al.<sup><a reveal-id="DSV036C12" data-open="DSV036C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV036C48" data-open="DSV036C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The physically mapped SNPs revealing differentiation between high and low pod number-containing parental accessions and individuals of two mapping populations at QTL region of interest were genotyped in the genomic DNA of 92 <em>desi</em> and <em>kabuli</em> chickpea accessions belonging to a PN-specific association panel using Illumina GoldenGate assay as per Saxena et al.<sup><a reveal-id="DSV036C9" data-open="DSV036C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Bajaj et al.<sup><a reveal-id="DSV036C23" data-open="DSV036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The replicated multi-location/years PN field phenotyping data, population structure (<em>K</em> = 2) statistics, principal component analysis (PCA) and kinship matrix of 92 accessions were obtained from Kujur et al.<sup><a reveal-id="DSV036C13" data-open="DSV036C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The association analysis was performed by correlating the genotyping information of SNPs at robust PN QTL interval with aforesaid phenotyping and diversity statistics using general linear model (GLM), mixed linear model (MLM at optimum level of compression with P3D method) and mixed model approach of EMMA<sup><a reveal-id="DSV036C49" data-open="DSV036C49" class="link link-ref link-reveal xref-bibr">49</a></sup> following the methods of Kujur et al.<sup><a reveal-id="DSV036C13" data-open="DSV036C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and Kumar et al.<sup><a reveal-id="DSV036C50" data-open="DSV036C50" class="link link-ref link-reveal xref-bibr">50</a></sup> The Bonferroni correction of <em>P</em>-value was used for each trait-associated SNPs at 5% significance level to eliminate the confounding effect of population structure and correct the false discovery rate (FDR) based on multiple comparisons. The informative SNPs-carrying candidate gene associated (<em>R</em><sup>2</sup> = correlation potential of significant SNPs with traits) with PN at significant cut-off <em>P</em> ≤ 10<sup>−5</sup> was screened by combining the outcomes of GLM and MLM with EMMA and FDR correction.</p>                            <h3 scrollto-destination=87199586 id="87199586" class="section-title" data-legacy-id=s2d>2.4. Differential gene expression profiling</h3>
<p>To infer the gene regulation, differential expression profiling of SNPs-containing gene annotated at the major genomic interval harbouring a novel robust PN QTL (validated by both QTL-seq and QTL region-specific association analysis) was performed. The vegetative (leaf and root) and reproductive (flower bud, ovary, anther and mature pollen) tissues and two-pod (∼5 mm beginning pod size at 5–10 days after anthesis/DAA and ∼2 cm full pod at 15–20 DAA) developmental stages were collected (at least with three biological replicates) from 40–60 days old healthy plants of low and high pod number-containing homozygous mapping individuals (constituting LPNB and HPNB in QTL-seq analysis) and parental accessions from each of the two mapping populations under study. The RNA isolated from all the tissues/stages of mapping individuals and parents were amplified with desirable gene-specific primers and internal control elongation factor 1-alpha (EF1α) using semi-quantitative and quantitative RT–PCR assays. The differential expression level of gene observed in diverse tissues/developmental stages of high and low pod number-containing mapping individuals and parents was compared/correlated following Bajaj et al.<sup><a reveal-id="DSV036C22" data-open="DSV036C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h2 scrollto-destination=87199590 id="87199590" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87199592 id="87199592" class="section-title" data-legacy-id=s3a>3.1. Genetic inheritance pattern of pod number in two inter-specific mapping populations</h3>
<p>We observed a significant difference of PN (5-237) with 13% CV and 81% <em>H</em><sup>2</sup> in 102 individuals and parental accessions of an inter-specific F<sub>5</sub> mapping population of Pusa 1103 × ILWC 46 (Table <a reveal-id="DSV036TB1" data-open="DSV036TB1" class="link link-reveal link-table xref-fig">1</a>). A wider phenotypic variation for PN (5-229) with 14.8% CV and 80% <em>H</em><sup>2</sup> was detected in another 98 individuals and parents of an inter-specific F<sub>5</sub> mapping population of Pusa 256 × ILWC 46 (Table <a reveal-id="DSV036TB1" data-open="DSV036TB1" class="link link-reveal link-table xref-fig">1</a>). The continuous variation as well as normal frequency distribution of PN trait was observed in these two mapping populations, which indicates the quantitative genetic inheritance pattern of target trait under study. The pod number is a complex yield component quantitative trait in chickpea, which is known to be governed by multiple abiotic (drought, heat, cold and salinity) and biotic (pod bores and other diseases) stress factors and often influenced by climate/growing conditions. To overcome these intricacies, present study selectively used three contrasting parental accessions-derived inter-specific mapping populations exhibiting wider phenotypic variability and higher heritability (consistent phenotypic expression) for pod number across geographical locations/years for molecular mapping of major QTLs regulating pod number in chickpea through mQTL-seq. </p>                        <a id="87199597" scrollto-destination="87199597"></a>
<div class="table-wrap"><div id="DSV036TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Statistical measures of pod number estimated in parental accessions and individuals of two inter-specific F<sub>5</sub> mapping populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Mapping populations </th><th colspan="2">Parental accessions<hr /> </th><th colspan="3">F<sub>5</sub> mapping individuals<hr /> </th><th rowspan="2">Broad-sense heritability (<em>H</em><sup>2</sup>%) </th></tr><tr><th>Pusa 1103 (Mean ± S.D.) </th><th>ILWC 46 (Mean ± S.D.) </th><th>Mean ± S.D. </th><th>Range </th><th>Coefficient of variation (CV%) </th></tr></thead><tbody><tr><td>Pusa 1103 × ILWC 46 </td><td>129 ± 2.2 </td><td>29 ± 1.9 </td><td>145.5 ± 18.9 </td><td>5.0–237.0 </td><td>13.0 </td><td>81 </td></tr><tr><td> </td><td>Pusa 256 (Mean ± S.D.) </td><td>ILWC 46 (Mean ± S.D.) </td><td>Mean ± S.D. </td><td>Range </td><td>Coefficient of variation (CV%) </td><td> </td></tr><tr><td>Pusa 256 × ILWC 46 </td><td>125 ± 2.4 </td><td>29 ± 1.5 </td><td>129.1 ± 19.1 </td><td>5.0–229.0 </td><td>14.8 </td><td>80 </td></tr></tbody></table></div><div content-id="DSV036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Mapping populations </th><th colspan="2">Parental accessions<hr /> </th><th colspan="3">F<sub>5</sub> mapping individuals<hr /> </th><th rowspan="2">Broad-sense heritability (<em>H</em><sup>2</sup>%) </th></tr><tr><th>Pusa 1103 (Mean ± S.D.) </th><th>ILWC 46 (Mean ± S.D.) </th><th>Mean ± S.D. </th><th>Range </th><th>Coefficient of variation (CV%) </th></tr></thead><tbody><tr><td>Pusa 1103 × ILWC 46 </td><td>129 ± 2.2 </td><td>29 ± 1.9 </td><td>145.5 ± 18.9 </td><td>5.0–237.0 </td><td>13.0 </td><td>81 </td></tr><tr><td> </td><td>Pusa 256 (Mean ± S.D.) </td><td>ILWC 46 (Mean ± S.D.) </td><td>Mean ± S.D. </td><td>Range </td><td>Coefficient of variation (CV%) </td><td> </td></tr><tr><td>Pusa 256 × ILWC 46 </td><td>125 ± 2.4 </td><td>29 ± 1.5 </td><td>129.1 ± 19.1 </td><td>5.0–229.0 </td><td>14.8 </td><td>80 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199597" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87199599 id="87199599" class="section-title" data-legacy-id=s3b>3.2. NGS-based whole-genome resequencing for QTL-seq</h3>
<p>The NGS-based high-throughput whole-genome resequencing of two high and low pod number-containing parental accessions as well as bulks (LPNB and HPNB) of two inter-specific mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)] were generated on an average of 81.5 million high-quality sequence reads (ranging from 80.7 to 83.5 million reads) with a ∼11.6-fold sequencing depth coverage. Notably, 82.1% (varying from 82.3 to 83.6%) sequence reads of these were mapped (minimum mapping quality: 30) to unique physical locations of <em>kabuli</em> reference genome with a 69% mean coverage. The high-quality uniquely mapped sequence reads generated from parental accessions and bulks (LPNB and HPNB) of two mapping populations were normalized based on depth of read coverage to reduce the potential biasness of read depth in the studied samples. Subsequently, the sequencing depth coverage (fold) and genome coverage (%) of uniquely mapped non-redundant sequence reads in mapping parents and bulks were estimated individually, with the average coverage of ∼11.6-fold and 64.1% (474.2 Mb) of <em>kabuli</em> chickpea genome (with an estimated genome size of ∼740 Mb), respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>). The normalized individual sequence reads of mapping parents and bulks were compared with reference <em>kabuli</em> genomic sequences (pseudomolecules and scaffolds) to mine the valid homozygous SNPs for QTL-seq analysis. The sequencing data generated in the present investigation was submitted to NCBI-sequence read archive (SRA) database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) with accession number SRR2228974 under BioProject ID: PRJNA294404 (submission ID: SUB1081829). The comparative assessment on <em>desi</em> and <em>kabuli</em> draft chickpea genome assemblies provided clues regarding high-quality and uniform whole-genome sequence assembly of <em>kabuli</em>, including its large size (Mb) chromosome pseudomolecule and scaffolds than <em>desi</em> chickpea genome.<sup><a reveal-id="DSV036C51" data-open="DSV036C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Henceforth, <em>kabuli</em> rather than the <em>desi</em> genome sequence was preferably utilized as a reference for whole-genome resequencing-based SNPs mining and QTL-seq analyses in mapping parents and bulks of chickpea.</p>                            <h3 scrollto-destination=87199603 id="87199603" class="section-title" data-legacy-id=s3c>3.3. Discovery and annotation of genome-wide SNPs in a mapping population of Pusa 1103 × ILWC 46</h3>
<p>The comparative genome sequence data analysis of high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing mapping parents and bulks with reference genomic sequence (pseudomolecule) of <em>kabuli</em> accession (CDC Frontier) identified 243,896 and 234,921 high-quality homozygous SNPs (with read depth ≥10 and SNP base quality ≥20) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Tables S2 and S3</a></span>, Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>A and B). Of these, 108,170 SNPs irrespective of monomorphic/polymorphic allele types were found to be common between high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing mapping parents and bulks based on their congruent physical positions (bp) on the reference <em>kabuli</em> genome (with a mean map density of 4.8 kb) (Table <a reveal-id="DSV036TB2" data-open="DSV036TB2" class="link link-reveal link-table xref-fig">2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S4</a></span>, Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>A). These selected SNPs were further used for QTL-seq analysis. The 54,731 (50.6% of identified 108 170 SNPs) SNPs were physically mapped on eight chromosomes of <em>kabuli</em> genome with a mean map density of 6.3 kb that varied from 4.3 (chromosome 7) to 9.1 (chromosome 6) kb (Table <a reveal-id="DSV036TB2" data-open="DSV036TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>B). At a genome-wide level, the density of physically mapped SNPs was maximum on chromosome 7 (235.1 SNPs/Mb) followed by chromosome 4 (213.2 SNPs/Mb) and minimum on chromosome 6 (109.9 SNPs/Mb), with a mean of 157.6 SNPs/Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1A</a></span>). The remaining 53,439 (49.4%) SNPs were mapped on the unanchored scaffolds of <em>kabuli</em> genome with an average map density of 3.2 kb (Table <a reveal-id="DSV036TB2" data-open="DSV036TB2" class="link link-reveal link-table xref-fig">2</a>). All the 243,896 and 234,921 high-quality homozygous SNPs were submitted to NCBI dbSNP (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/SNP/snp_viewTable.cgi?handle=NIPGR">http://www.ncbi.nlm.nih.gov/SNP/snp_viewTable.cgi?handle=NIPGR</a>) with SNP submission (SS) accession numbers (1940124817-1945872952; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Tables S2 and S3</a></span>) for unrestricted public access. </p>                        <a id="87199607" scrollto-destination="87199607"></a>
<div class="table-wrap"><div id="DSV036TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Genomic distribution of SNPs physically mapped on eight chromosomes and unanchored scaffolds of <em>kabuli</em> chickpea genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosomes </th><th rowspan="2">Size (Mb) of <em>kabuli</em> chromosomes (pseudomolecules) </th><th colspan="2">Number (%) of common SNPs mapped<hr /> </th><th colspan="2">Average map density (kb)<hr /> </th></tr><tr><th>(Pusa 1103 and HPNB) vs. (ILWC 46 and LPNB) </th><th>(Pusa 256 and HPNB) vs. (ILWC 46 and LPNB) </th><th>(Pusa 1103 and HPNB) vs. (ILWC 46 and LPNB) </th><th>(Pusa 256 and HPNB) vs. (ILWC 46 and LPNB) </th></tr></thead><tbody><tr><td><em>Ca_Kabuli_Chr01</em> </td><td>48.4 </td><td>6,191 (11.3) </td><td>5,643 (16.2) </td><td>7.8 </td><td>8.6 </td></tr><tr><td><em>Ca_Kabuli_Chr02</em> </td><td>36.6 </td><td>5,232 (9.5) </td><td>3,475 (10.0) </td><td>7.0 </td><td>10.5 </td></tr><tr><td><em>Ca_Kabuli_Chr03</em> </td><td>40.0 </td><td>5,712 (10.4) </td><td>3,505 (10.1) </td><td>7.0 </td><td>11.4 </td></tr><tr><td><em>Ca_Kabuli_Chr04</em> </td><td>49.2 </td><td>10,491 (19.2) </td><td>7,331 (21.1) </td><td>4.7 </td><td>6.7 </td></tr><tr><td><em>Ca_Kabuli_Chr05</em> </td><td>48.2 </td><td>6,770 (12.4) </td><td>3,659 (10.5) </td><td>7.1 </td><td>13.2 </td></tr><tr><td><em>Ca_Kabuli_Chr06</em> </td><td>59.5 </td><td>6,540 (11.9) </td><td>4,761 (13.7) </td><td>9.1 </td><td>12.5 </td></tr><tr><td><em>Ca_Kabuli_Chr07</em> </td><td>49.0 </td><td>11,507 (21.0) </td><td>4,564 (13.1) </td><td>4.3 </td><td>10.7 </td></tr><tr><td><em>Ca_Kabuli_Chr08</em> </td><td>16.5 </td><td>2,288 (4.2) </td><td>1,775 (5.1) </td><td>7.2 </td><td>9.3 </td></tr><tr><td>Total </td><td>347.2 </td><td>54,731 (50.6) </td><td>34,713 (45.0) </td><td>6.3 </td><td>10.0 </td></tr><tr><td>Unanchored scaffolds </td><td>171.5 </td><td>53,439 (49.4) </td><td>42,384 (55.0) </td><td>3.2 </td><td>4.0 </td></tr><tr><td>Total </td><td>518.7 </td><td>108,170 </td><td>77,097 </td><td>4.8 </td><td>6.7 </td></tr></tbody></table></div><div content-id="DSV036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosomes </th><th rowspan="2">Size (Mb) of <em>kabuli</em> chromosomes (pseudomolecules) </th><th colspan="2">Number (%) of common SNPs mapped<hr /> </th><th colspan="2">Average map density (kb)<hr /> </th></tr><tr><th>(Pusa 1103 and HPNB) vs. (ILWC 46 and LPNB) </th><th>(Pusa 256 and HPNB) vs. (ILWC 46 and LPNB) </th><th>(Pusa 1103 and HPNB) vs. (ILWC 46 and LPNB) </th><th>(Pusa 256 and HPNB) vs. (ILWC 46 and LPNB) </th></tr></thead><tbody><tr><td><em>Ca_Kabuli_Chr01</em> </td><td>48.4 </td><td>6,191 (11.3) </td><td>5,643 (16.2) </td><td>7.8 </td><td>8.6 </td></tr><tr><td><em>Ca_Kabuli_Chr02</em> </td><td>36.6 </td><td>5,232 (9.5) </td><td>3,475 (10.0) </td><td>7.0 </td><td>10.5 </td></tr><tr><td><em>Ca_Kabuli_Chr03</em> </td><td>40.0 </td><td>5,712 (10.4) </td><td>3,505 (10.1) </td><td>7.0 </td><td>11.4 </td></tr><tr><td><em>Ca_Kabuli_Chr04</em> </td><td>49.2 </td><td>10,491 (19.2) </td><td>7,331 (21.1) </td><td>4.7 </td><td>6.7 </td></tr><tr><td><em>Ca_Kabuli_Chr05</em> </td><td>48.2 </td><td>6,770 (12.4) </td><td>3,659 (10.5) </td><td>7.1 </td><td>13.2 </td></tr><tr><td><em>Ca_Kabuli_Chr06</em> </td><td>59.5 </td><td>6,540 (11.9) </td><td>4,761 (13.7) </td><td>9.1 </td><td>12.5 </td></tr><tr><td><em>Ca_Kabuli_Chr07</em> </td><td>49.0 </td><td>11,507 (21.0) </td><td>4,564 (13.1) </td><td>4.3 </td><td>10.7 </td></tr><tr><td><em>Ca_Kabuli_Chr08</em> </td><td>16.5 </td><td>2,288 (4.2) </td><td>1,775 (5.1) </td><td>7.2 </td><td>9.3 </td></tr><tr><td>Total </td><td>347.2 </td><td>54,731 (50.6) </td><td>34,713 (45.0) </td><td>6.3 </td><td>10.0 </td></tr><tr><td>Unanchored scaffolds </td><td>171.5 </td><td>53,439 (49.4) </td><td>42,384 (55.0) </td><td>3.2 </td><td>4.0 </td></tr><tr><td>Total </td><td>518.7 </td><td>108,170 </td><td>77,097 </td><td>4.8 </td><td>6.7 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199607" target="_blank">View Large</a></div></div>                        <a id="87199609" scrollto-destination="87199609"></a>
<div id="DSV036F2" data-id="DSV036F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03602.png?Expires=1509239038&Signature=PHWPIsAR0Wvvmws1C6i-yO8PqijYbMuuPqnGrq-KWd7ZSy9jzrAI7YxfEzGv3xWGnI46wJP~h9vBrlzfwBZXjDik~IbgLbA7RaXhZfn2ABpcxyR7hdBUR24RdJgpoYfb80nO4axWXuZaob4MSaAfmgfZEnWBxk1NP-3VPVH7eDfR4C~JrDjp-ea2bu~NMGWVtmdPDUNcfpcO0XTXRQsSXCWGPG0MUrAznpyVpH5ipqc7ZIjJzox9umW8QdML7O0m-BgpkywdPYTWAI9iX2qOt0SFVST~WL7xLQTjw7DXW2mbvdfly2ihvKEITQSnb8XY1CT0M6B3hVmO01inkrGmXg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide distribution pattern of SNPs identified by comparing the whole-genome resequencing data of high and low pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)] with respect to kabuli chickpea (CDC Frontier). SNPs differentiating high and low-pod number-containing parental accessions and homozygous bulks of two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] from CDC Frontier are illustrated by venn diagrams. HPNB: high pod number bulk and LPNB: low pod number bulk. (C) The relative genomic distribution of SNPs physically mapped on eight chromosomes of kabuli chickpea genome are depicted by a Circos circular ideogram. The outermost circles denote the different physical size (Mb) of eight chromosomes coded with multiple colours as per the pseudomolecule size reported in kabuli chickpea genome.46 SNPs (Pusa 1103 and HPNB) vs. CDC Frontier (a), (ILWC 46 and LPNB) vs. CDC Frontier (b), (Pusa 256 and HPNB) vs. CDC Frontier (c), and (ILWC 46 and LPNB) vs. CDC Frontier (d) are indicated. (e) Total SNPs, including non-synonymous (f) and regulatory SNPs (g) common between high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. (h) Total SNPs, including non-synonymous (i) and regulatory SNPs (j) common between high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. Frequency of SNPs discovered from the intergenic and various coding (synonymous and non-synonymous) and non-coding (introns and regulatory regions) sequence components of protein-coding genes (D, E and F) annotated on kabuli chickpea genome. Parenthesis designates the number of SNPs-containing genes. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of genes were defined as per the gene annotation of kabuli genome. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199609/dsv03602.tiff" path-from-xml="dsv03602.tiff" target="_blank">View large</a><a section="87199609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03602.png?Expires=1509239038&Signature=arVFUhtxVRNEnepkR-Pv5WiQk8MQDKxmmrgCA~4xv3wVvpPFXq-6qhDKL2xNtqmf~OzXzdTCHNSm7aWxhooT4jri4qc7mfHEU6rt4J1lsN5hJyZFmf7yLa~70WqTtvUkIZMSnHEQbZy7MWaL1cJvAmO62OTgznE2Mr66-jdG7xnWnbtfOJiV0pMQLX2qDjEoR~hWypphjENb15rpAIODyWFERxYjRu3m0pxqnLb6eGOB-gzJv8SL1iC5tnK4rC8dL-Yrn1yFSZ~0mGqBkyDwZyfInqwxwV5yShqwrQxp~WyJkP745BEuRpDf5H6rXbpSMb5ueVJKdnurQ0YRPYx-3w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199609&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide distribution pattern of SNPs identified by comparing the whole-genome resequencing data of high and low pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)] with respect to <em>kabuli</em> chickpea (CDC Frontier). SNPs differentiating high and low-pod number-containing parental accessions and homozygous bulks of two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] from CDC Frontier are illustrated by venn diagrams. HPNB: high pod number bulk and LPNB: low pod number bulk. (C) The relative genomic distribution of SNPs physically mapped on eight chromosomes of <em>kabuli</em> chickpea genome are depicted by a Circos circular ideogram. The outermost circles denote the different physical size (Mb) of eight chromosomes coded with multiple colours as per the pseudomolecule size reported in <em>kabuli</em> chickpea genome.<sup><a reveal-id="DSV036C46" data-open="DSV036C46" class="link link-ref link-reveal xref-bibr">46</a></sup> SNPs (Pusa 1103 and HPNB) vs. CDC Frontier (a), (ILWC 46 and LPNB) vs. CDC Frontier (b), (Pusa 256 and HPNB) vs. CDC Frontier (c), and (ILWC 46 and LPNB) vs. CDC Frontier (d) are indicated. (e) Total SNPs, including non-synonymous (f) and regulatory SNPs (g) common between high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. (h) Total SNPs, including non-synonymous (i) and regulatory SNPs (j) common between high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. Frequency of SNPs discovered from the intergenic and various coding (synonymous and non-synonymous) and non-coding (introns and regulatory regions) sequence components of protein-coding genes (D, E and F) annotated on <em>kabuli</em> chickpea genome. Parenthesis designates the number of SNPs-containing genes. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of genes were defined as per the gene annotation of <em>kabuli</em> genome. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV036F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03602.png?Expires=1509239038&Signature=PHWPIsAR0Wvvmws1C6i-yO8PqijYbMuuPqnGrq-KWd7ZSy9jzrAI7YxfEzGv3xWGnI46wJP~h9vBrlzfwBZXjDik~IbgLbA7RaXhZfn2ABpcxyR7hdBUR24RdJgpoYfb80nO4axWXuZaob4MSaAfmgfZEnWBxk1NP-3VPVH7eDfR4C~JrDjp-ea2bu~NMGWVtmdPDUNcfpcO0XTXRQsSXCWGPG0MUrAznpyVpH5ipqc7ZIjJzox9umW8QdML7O0m-BgpkywdPYTWAI9iX2qOt0SFVST~WL7xLQTjw7DXW2mbvdfly2ihvKEITQSnb8XY1CT0M6B3hVmO01inkrGmXg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide distribution pattern of SNPs identified by comparing the whole-genome resequencing data of high and low pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)] with respect to kabuli chickpea (CDC Frontier). SNPs differentiating high and low-pod number-containing parental accessions and homozygous bulks of two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] from CDC Frontier are illustrated by venn diagrams. HPNB: high pod number bulk and LPNB: low pod number bulk. (C) The relative genomic distribution of SNPs physically mapped on eight chromosomes of kabuli chickpea genome are depicted by a Circos circular ideogram. The outermost circles denote the different physical size (Mb) of eight chromosomes coded with multiple colours as per the pseudomolecule size reported in kabuli chickpea genome.46 SNPs (Pusa 1103 and HPNB) vs. CDC Frontier (a), (ILWC 46 and LPNB) vs. CDC Frontier (b), (Pusa 256 and HPNB) vs. CDC Frontier (c), and (ILWC 46 and LPNB) vs. CDC Frontier (d) are indicated. (e) Total SNPs, including non-synonymous (f) and regulatory SNPs (g) common between high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. (h) Total SNPs, including non-synonymous (i) and regulatory SNPs (j) common between high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. Frequency of SNPs discovered from the intergenic and various coding (synonymous and non-synonymous) and non-coding (introns and regulatory regions) sequence components of protein-coding genes (D, E and F) annotated on kabuli chickpea genome. Parenthesis designates the number of SNPs-containing genes. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of genes were defined as per the gene annotation of kabuli genome. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199609/dsv03602.tiff" path-from-xml="dsv03602.tiff" target="_blank">View large</a><a section="87199609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03602.png?Expires=1509239038&Signature=arVFUhtxVRNEnepkR-Pv5WiQk8MQDKxmmrgCA~4xv3wVvpPFXq-6qhDKL2xNtqmf~OzXzdTCHNSm7aWxhooT4jri4qc7mfHEU6rt4J1lsN5hJyZFmf7yLa~70WqTtvUkIZMSnHEQbZy7MWaL1cJvAmO62OTgznE2Mr66-jdG7xnWnbtfOJiV0pMQLX2qDjEoR~hWypphjENb15rpAIODyWFERxYjRu3m0pxqnLb6eGOB-gzJv8SL1iC5tnK4rC8dL-Yrn1yFSZ~0mGqBkyDwZyfInqwxwV5yShqwrQxp~WyJkP745BEuRpDf5H6rXbpSMb5ueVJKdnurQ0YRPYx-3w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199609&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide distribution pattern of SNPs identified by comparing the whole-genome resequencing data of high and low pod number-containing parental accessions and homozygous individuals (constituting bulks) from each of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)] with respect to <em>kabuli</em> chickpea (CDC Frontier). SNPs differentiating high and low-pod number-containing parental accessions and homozygous bulks of two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] from CDC Frontier are illustrated by venn diagrams. HPNB: high pod number bulk and LPNB: low pod number bulk. (C) The relative genomic distribution of SNPs physically mapped on eight chromosomes of <em>kabuli</em> chickpea genome are depicted by a Circos circular ideogram. The outermost circles denote the different physical size (Mb) of eight chromosomes coded with multiple colours as per the pseudomolecule size reported in <em>kabuli</em> chickpea genome.<sup><a reveal-id="DSV036C46" data-open="DSV036C46" class="link link-ref link-reveal xref-bibr">46</a></sup> SNPs (Pusa 1103 and HPNB) vs. CDC Frontier (a), (ILWC 46 and LPNB) vs. CDC Frontier (b), (Pusa 256 and HPNB) vs. CDC Frontier (c), and (ILWC 46 and LPNB) vs. CDC Frontier (d) are indicated. (e) Total SNPs, including non-synonymous (f) and regulatory SNPs (g) common between high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. (h) Total SNPs, including non-synonymous (i) and regulatory SNPs (j) common between high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing parental accessions and homozygous mapping individuals. Frequency of SNPs discovered from the intergenic and various coding (synonymous and non-synonymous) and non-coding (introns and regulatory regions) sequence components of protein-coding genes (D, E and F) annotated on <em>kabuli</em> chickpea genome. Parenthesis designates the number of SNPs-containing genes. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of genes were defined as per the gene annotation of <em>kabuli</em> genome. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The structural annotation of SNPs exhibited the occurrence of 84,529 (78.1% of total mined 108,170 SNPs) and 23,641 (21.9%) SNPs in the intergenic and different sequence components of 6,179 genes, respectively (Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>D). A maximum (9,280 SNPs, 39.2%) and minimum (1,948 SNPs, 8.2%) frequency of SNPs were observed in the upstream regulatory regions (URRs) and CDS of genes, respectively (Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>E). Notably, 445 and 706 coding SNPs-containing genes exhibited synonymous (666 SNPs) and non-synonymous (1,282 SNPs) (missense and nonsense) substitutions, respectively (Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>F). The non-synonymous SNPs comprise 1,194 missense and 88 nonsense SNPs in the 625 and 81 genes, respectively. The average frequency of SNPs within genes was estimated as 3.8 SNPs/gene. The functional annotation of 6,179 SNPs-carrying genes revealed correspondence of ∼9.6 and 8.4% of SNPs to transcription factors and disease resistance-related proteins, respectively. The numerous informative SNPs differentiating <em>desi</em> (Pusa 1103), <em>kabuli</em> (CDC Frontier) and wild (ILWC 46) accessions, structurally and functionally annotated at a genome-wide scale, can be deployed for multi-dimensional high-throughput genotyping applications in chickpea.</p>                            <h3 scrollto-destination=87199615 id="87199615" class="section-title" data-legacy-id=s3d>3.4. Discovery and annotation of genome-wide SNPs in a mapping population of Pusa 256 × ILWC 46</h3>
<p>We identified 170,032 and 175,796 high-quality genome-wide homozygous SNPs differentiating the high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing mapping parents and bulks from reference <em>kabuli</em> accession (CDC Frontier) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S5 and S6</a></span>, Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>B and C). Of these, 77,097 SNPs identified to be common between high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing mapping parents and bulks following the aforesaid criteria were further used in QTL-seq analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S7</a></span>, Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>C). All these 77,097 SNPs were mapped on the <em>kabuli</em> genome with an average map density of 6.7 kb (Table <a reveal-id="DSV036TB2" data-open="DSV036TB2" class="link link-reveal link-table xref-fig">2</a>). The 34,713 (45% of identified 770,987 SNPs) SNPs mined were physically mapped on eight chromosomes of <em>kabuli</em> genome with a mean map density of 10 kb that ranged from 6.7 (chromosome 4) to 13.2 (chromosome 5) kb (Table <a reveal-id="DSV036TB2" data-open="DSV036TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>B). At a whole-genome level, the density of physically mapped SNPs was highest on chromosome 4 (149.1 SNPs/Mb) followed by chromosome 1 (116.7 SNPs/Mb) and minimum on chromosome 5 (75.9 SNPs/Mb), with a mean of 99.9 SNPs/Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1B</a></span>). The rest 42,384 (55%) SNPs were mapped on the unanchored scaffolds of <em>kabuli</em> genome with an average map density of 4.0 kb (Table <a reveal-id="DSV036TB2" data-open="DSV036TB2" class="link link-reveal link-table xref-fig">2</a>). All these 170,032 and 175,796 high-quality homozygous SNPs were submitted to NCBI dbSNP (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/SNP/snp_viewTable.cgi?handle=NIPGR">http://www.ncbi.nlm.nih.gov/SNP/snp_viewTable.cgi?handle=NIPGR</a>) with SNP submission (SS) accession numbers (1945872954–1947070701, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Tables S5 and S6</a></span>) for unrestricted use.</p><p>The structural annotation of SNPs showed the presence of 62,174 (80.6% of total mined 77,097 SNPs) and 14,923 (19.4%) SNPs in the intergenic and various sequence components of 4,175 genes, respectively (Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>D). The URRs and CDS of genes contained maximum (5,557 SNPs, 37.2%) and minimum (1,467 SNPs, 9.8%) frequency of SNPs, respectively (Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>E). Notably, 319 and 475 coding SNPs-carrying genes revealed synonymous (485 SNPs) and non-synonymous (982 SNPs) substitutions, respectively (Fig. <a reveal-id="DSV036F2" data-open="DSV036F2" class="link link-reveal link-table xref-fig">2</a>F). The non-synonymous SNPs contained 909 missense and 73 nonsense SNPs in the 408 and 67 genes, respectively. The average frequency of SNPs within genes was estimated to be 3.6 SNPs/gene. The functional annotation of 4,175 SNPs-carrying genes exhibited that ∼7.3 and 5.3% of SNPs were belonging to transcription factors and disease resistance-related proteins, respectively. These functionally relevant SNPs, discriminating <em>desi</em> (Pusa 256), <em>kabuli</em> (CDC Frontier) and wild (ILWC 46) accessions annotated in diverse sequence components of genome/genes, have tremendous practical utility towards establishing efficient marker-trait association and rapid detection of potential genes/QTLs governing important agronomic traits in chickpea.</p>                            <h3 scrollto-destination=87199620 id="87199620" class="section-title" data-legacy-id=s3e>3.5. Molecular mapping of QTL-seq-derived major PN QTLs in a mapping population of Pusa 1103 × ILWC 46</h3>
<p>The SNP index of individual SNPs showing differentiation between high (Pusa 1103 and HPNB) and low (ILWC 46 and LPNB) pod number-containing mapping parents and bulks was assessed. We estimated an average SNP index across a 1-Mb genomic interval individually in Pusa 1103 and HPNB as well as ILWC 46 and LPNB using a 10-kb sliding window approach. Further, these SNP indexes were plotted against eight chromosomes of <em>kabuli</em> reference genome. The Δ (SNP index) was measured through combining the SNP index information of HPNB and LPNB, and plotted against the genomic positions (Mb) of <em>kabuli</em> reference genome (Fig. <a reveal-id="DSV036F3" data-open="DSV036F3" class="link link-reveal link-table xref-fig">3</a>A). Two major genomic regions (<em>Caq<sup>a</sup>PN4.1</em>: 13690423 to 14558233 bp and <em>Caq<sup>a</sup>PN4.2</em>: 31916540 to 33716608 bp) on chromosome 4 demonstrating the average SNP index of higher than 0.8 in HPNB and lower than 0.2 in LPNB were detected (Fig. <a reveal-id="DSV036F3" data-open="DSV036F3" class="link link-reveal link-table xref-fig">3</a>A) following the SNP index estimation criteria as defined in QTL-seq analysis.<sup><a reveal-id="DSV036C42" data-open="DSV036C42" class="link link-ref link-reveal xref-bibr">42</a>–<a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The major genomic regions harbouring PN QTLs identified by QTL-seq were assured by Δ (SNP index) value that is significantly different from 0 at 99% significance level. Our comprehensive analysis of these target genomic regions inferred that high and low pod number-containing mapping individuals constituting the HPNB and LPNB comprised majority of the SNP alleles from high (Pusa 1103) and low (ILWC 46) pod number-containing mapping parental accessions, respectively. Taken together, our findings by QTL-seq in an inter-specific mapping population (Pusa 1103 × ILWC 46) ascertained the presence of two major QTLs (<em>Caq<sup>a</sup>PN4.1</em> and <em>Caq<sup>a</sup>PN4.2</em>) governing PN at the 867.8 Kb [13690423 (SNP_20990A) to 14558233 (SNP_21161A) bp] and 1.80 Mb [31916540 (SNP_27349A) to 33716608 (SNP_28364A) bp] genomic intervals on chromosome 4 of chickpea (Fig. <a reveal-id="DSV036F3" data-open="DSV036F3" class="link link-reveal link-table xref-fig">3</a>A). The SNPs flanking the QTL-seq-derived major PN QTL (<em>Caq<sup>a</sup>PN4.1</em> and <em>Caq<sup>a</sup>PN4.2</em>) were validated in accordance with their expected allelic discrimination by resequencing of PCR amplicons amplified from the parental accessions (Pusa 1103 and ILWC 46) and mapping individuals constituting the HPNB and LPNB. </p>                        <a id="87199624" scrollto-destination="87199624"></a>
<div id="DSV036F3" data-id="DSV036F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03603.png?Expires=1509239038&Signature=Tr1jf2Voft0LM8daPYwb0vyY122n7t0D9vGKVFk~5uzd1Tnvm0zQvtdBEK2BV2bq5qRqxD8fICqzCSdnfwSqDjczkWsU9-TdSISrdvqcy85CV8NuVDDpEH1dBiffVNDrfiXHVEepfQ8UCncBpcJt-CEkFT8nwPMCrAWELNtDsWajy5~b8Wxk~Mnknc7DP82s~VS~lwqKEhJra-6SmlypJrSwmm8X3cWh7Z3BprZiAABEgyK0y9FGHideBFmCip4dLnSp~xz9QAmq12~LzB7AJqqN~y2hlXn7ptQoco8yi6vHhZFQsmm3dj0gmZpADEJ~VNPf9CohmmiANrSJSQ6bJA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Graphs illustrating the SNP index and Δ (SNP index) of HPNB (high pod number bulk), LPNB (low pod number bulk) generated from mQTL-seq analysis in two inter-specific chickpea mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)]. The X-axis designates the physical positions (Mb) of kabuli chickpea chromosome 4. The Y-axis denotes the SNP indexes that are measured according to 1-Mb physical interval with a 10-kb sliding window. The Δ (SNP index) was plotted using the statistical confidence intervals under null hypothesis of no QTL (P &lt; 0.01) (indicated by a black dotted line) as per Takagi et al.42 and Das et al.44 Two candidate major genomic intervals underlying the robust PN QTLs (CaqaPN4.1, CaqaPN4.2, CaqbPN4.1 and CaqbPN4.2) identified from each of the two mapping populations using mQTL-seq were defined following the criteria of SNP index near to 1 and 0 in HPNB and LPNB, respectively, and the confidence value of significant Δ (SNP index) &gt;0.5 (significance level at P &lt; 0.01). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199624/dsv03603.tiff" path-from-xml="dsv03603.tiff" target="_blank">View large</a><a section="87199624" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03603.png?Expires=1509239038&Signature=A1pdvtRm6720dzoj-rHvI7EasiCK2msgRgvg9k8eO848d9qh-6YkmWxmbynYOqSjRCGLKBgK3WG6cdyKFsguZ70~i4txS8Jdqhbcgkm08T~bbU14vjWspSIc1KIw4ZnVECqAefZsv28YcvQYpeaRrSd02Hhk-zc9~PS2R~OXjzMIlYroD5b2Ce-7i-aD1P1F2CqU46jOxB6YHFl9FvNCoWNeYTKIHpu5IjSSj4KiHiR4R-HVkXJUy0hzulaT7koLqsLyTFM7PuzXro~4wJqQPYCWWJdrMTlDlVaSh5t8yVFYvg95bKFdpDn9~SPrFV3ODpIUkuKAHmoycL~nm6cZXA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199624&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphs illustrating the SNP index and Δ (SNP index) of HPNB (high pod number bulk), LPNB (low pod number bulk) generated from mQTL-seq analysis in two inter-specific chickpea mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)]. The <em>X</em>-axis designates the physical positions (Mb) of <em>kabuli</em> chickpea chromosome 4. The <em>Y</em>-axis denotes the SNP indexes that are measured according to 1-Mb physical interval with a 10-kb sliding window. The Δ (SNP index) was plotted using the statistical confidence intervals under null hypothesis of no QTL (<em>P</em> &lt; 0.01) (indicated by a black dotted line) as per Takagi et al.<sup><a reveal-id="DSV036C42" data-open="DSV036C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and Das et al.<sup><a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Two candidate major genomic intervals underlying the robust PN QTLs (<em>Caq<sup>a</sup>PN4.1</em>, <em>Caq<sup>a</sup>PN4.2, Caq<sup>b</sup>PN4.1 and Caq<sup>b</sup>PN4.2</em>) identified from each of the two mapping populations using mQTL-seq were defined following the criteria of SNP index near to 1 and 0 in HPNB and LPNB, respectively, and the confidence value of significant Δ (SNP index) &gt;0.5 (significance level at <em>P</em> &lt; 0.01). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV036F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03603.png?Expires=1509239038&Signature=Tr1jf2Voft0LM8daPYwb0vyY122n7t0D9vGKVFk~5uzd1Tnvm0zQvtdBEK2BV2bq5qRqxD8fICqzCSdnfwSqDjczkWsU9-TdSISrdvqcy85CV8NuVDDpEH1dBiffVNDrfiXHVEepfQ8UCncBpcJt-CEkFT8nwPMCrAWELNtDsWajy5~b8Wxk~Mnknc7DP82s~VS~lwqKEhJra-6SmlypJrSwmm8X3cWh7Z3BprZiAABEgyK0y9FGHideBFmCip4dLnSp~xz9QAmq12~LzB7AJqqN~y2hlXn7ptQoco8yi6vHhZFQsmm3dj0gmZpADEJ~VNPf9CohmmiANrSJSQ6bJA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Graphs illustrating the SNP index and Δ (SNP index) of HPNB (high pod number bulk), LPNB (low pod number bulk) generated from mQTL-seq analysis in two inter-specific chickpea mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)]. The X-axis designates the physical positions (Mb) of kabuli chickpea chromosome 4. The Y-axis denotes the SNP indexes that are measured according to 1-Mb physical interval with a 10-kb sliding window. The Δ (SNP index) was plotted using the statistical confidence intervals under null hypothesis of no QTL (P &lt; 0.01) (indicated by a black dotted line) as per Takagi et al.42 and Das et al.44 Two candidate major genomic intervals underlying the robust PN QTLs (CaqaPN4.1, CaqaPN4.2, CaqbPN4.1 and CaqbPN4.2) identified from each of the two mapping populations using mQTL-seq were defined following the criteria of SNP index near to 1 and 0 in HPNB and LPNB, respectively, and the confidence value of significant Δ (SNP index) &gt;0.5 (significance level at P &lt; 0.01). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199624/dsv03603.tiff" path-from-xml="dsv03603.tiff" target="_blank">View large</a><a section="87199624" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03603.png?Expires=1509239038&Signature=A1pdvtRm6720dzoj-rHvI7EasiCK2msgRgvg9k8eO848d9qh-6YkmWxmbynYOqSjRCGLKBgK3WG6cdyKFsguZ70~i4txS8Jdqhbcgkm08T~bbU14vjWspSIc1KIw4ZnVECqAefZsv28YcvQYpeaRrSd02Hhk-zc9~PS2R~OXjzMIlYroD5b2Ce-7i-aD1P1F2CqU46jOxB6YHFl9FvNCoWNeYTKIHpu5IjSSj4KiHiR4R-HVkXJUy0hzulaT7koLqsLyTFM7PuzXro~4wJqQPYCWWJdrMTlDlVaSh5t8yVFYvg95bKFdpDn9~SPrFV3ODpIUkuKAHmoycL~nm6cZXA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199624&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphs illustrating the SNP index and Δ (SNP index) of HPNB (high pod number bulk), LPNB (low pod number bulk) generated from mQTL-seq analysis in two inter-specific chickpea mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)]. The <em>X</em>-axis designates the physical positions (Mb) of <em>kabuli</em> chickpea chromosome 4. The <em>Y</em>-axis denotes the SNP indexes that are measured according to 1-Mb physical interval with a 10-kb sliding window. The Δ (SNP index) was plotted using the statistical confidence intervals under null hypothesis of no QTL (<em>P</em> &lt; 0.01) (indicated by a black dotted line) as per Takagi et al.<sup><a reveal-id="DSV036C42" data-open="DSV036C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and Das et al.<sup><a reveal-id="DSV036C44" data-open="DSV036C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Two candidate major genomic intervals underlying the robust PN QTLs (<em>Caq<sup>a</sup>PN4.1</em>, <em>Caq<sup>a</sup>PN4.2, Caq<sup>b</sup>PN4.1 and Caq<sup>b</sup>PN4.2</em>) identified from each of the two mapping populations using mQTL-seq were defined following the criteria of SNP index near to 1 and 0 in HPNB and LPNB, respectively, and the confidence value of significant Δ (SNP index) &gt;0.5 (significance level at <em>P</em> &lt; 0.01). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199627 id="87199627" class="section-title" data-legacy-id=s3f>3.6. Molecular mapping of QTL-seq-derived major PN QTLs in a mapping population of Pusa 256 × ILWC 46</h3>
<p>The SNP index of individual SNPs revealing differentiation between high (Pusa 256 and HPNB) and low (ILWC 46 and LPNB) pod number-containing mapping parents and bulks was measured. The estimation of average SNP index (across a 5-Mb genomic interval of a 10-kb sliding window) and Δ (SNP index) of HPNB and LPNB, and their plotting across eight <em>kabuli</em> chromosomes (following aforementioned strategies) identified two major genomic regions (<em>Caq<sup>b</sup>PN4.1</em>: 13770030 to 14407570 bp and <em>Caq<sup>b</sup>PN4.2</em>: 31916540 to 33195624 bp) on chromosome 4 exhibiting the average SNP index of higher than 0.9 in HPNB and lower than 0.1 in LPNB (Fig. <a reveal-id="DSV036F3" data-open="DSV036F3" class="link link-reveal link-table xref-fig">3</a>B). The authenticity of these identified major genomic regions harbouring PN QTLs was confirmed by valid significant Δ (SNP index) value (at 99% significance level). The detail analysis of these target genomic regions indicated the presence of most of the SNP alleles from Pusa 256 and ILWC 46 in high and low pod number-containing mapping individuals constituting the HPNB and LPNB, respectively. Collectively, the QTL-seq in an inter-specific mapping population (Pusa 256 × ILWC 46) identified two major QTLs (<em>Caq<sup>b</sup>PN4.1</em> and <em>Caq<sup>b</sup>PN4.2</em>) regulating PN at the 637.5 kb [13770030 (SNP_15561B) to 14407570 (SNP_15696B) bp] and 1.28 Mb [31916540 (SNP_18917B) to 33195624 (SNP_19254B) bp] genomic intervals on chromosome 4 of chickpea (Fig. <a reveal-id="DSV036F3" data-open="DSV036F3" class="link link-reveal link-table xref-fig">3</a>B). The SNPs flanking the QTL-seq-derived major PN QTL (<em>Caq<sup>b</sup>PN4.1</em> and <em>Caq<sup>b</sup>PN4.2</em>) were validated according to their expected allelic discrimination by resequencing of PCR amplicons amplified from the parental accessions (Pusa 256 and ILWC 46) and mapping individuals constituting the HPNB and LPNB.</p>                            <h3 scrollto-destination=87199631 id="87199631" class="section-title" data-legacy-id=s3g>3.7. QTL region-specific trait association mapping</h3>
<p>The four major genomic regions underlying PN QTLs (<em>Caq<sup>a</sup>PN4.1</em>, <em>Caq<sup>a</sup>PN4.2</em>, <em>Caq<sup>b</sup>PN4.1</em> and <em>Caq<sup>b</sup>PN4.2</em>) detected in two mapping populations of Pusa 1103 × ILWC 46 and Pusa 256 × ILWC 46 using QTL-seq were compared and correlated (Fig. <a reveal-id="DSV036F4" data-open="DSV036F4" class="link link-reveal link-table xref-fig">4</a>A and B). This led to identification of two consensus major genomic regions with short physical intervals of 637.5 kb [<em>Caq<sup>b</sup>PN4.1</em>: 13770030 (SNP_15561B) to 14407570 (SNP_15696B) bp] and 1.28 Mb [<em>Caq<sup>b</sup>PN4.2</em>: 31916540 (SNP_18917B) to 33195624 (SNP_19254B) bp] on chromosome 4 harbouring PN QTLs (Fig. <a reveal-id="DSV036F4" data-open="DSV036F4" class="link link-reveal link-table xref-fig">4</a>A and B). These QTLs, being well-validated across two diverse chickpea mapping populations under study, were considered as robust QTLs. The scale-down of longer major PN QTL intervals identified by QTL-seq in individual mapping population into shorter QTL regions by mQTL-seq implicates the significance of mQTL-seq over QTL-seq approach for high-resolution molecular mapping along with fine mapping of major potential genomic regions underlying robust trait-regulatory QTLs in chickpea. Based on congruent physical position, one of the robust QTL (<em>Caq<sup>b</sup>PN4.1</em>) of these exhibited correspondence with a priorly identified known major QTL regulating pod number/plant was mapped on an intra-specific genetic linkage map of chickpea (Fig. <a reveal-id="DSV036F4" data-open="DSV036F4" class="link link-reveal link-table xref-fig">4</a>A and B).<sup><a reveal-id="DSV036C41" data-open="DSV036C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The remaining one robust PN QTL (<em>Caq<sup>b</sup>PN4.2</em>) was considered to be novel (not previously identified) and thus selected by us to delineate potential candidate gene-regulating pod number through QTL region-specific high-resolution trait association mapping. A wider practical applicability of mQTL-seq strategy for rapid detection of major as well as robust high-resolution PN QTLs using two F<sub>5</sub> inter-specific chickpea mapping populations (with at least one common parental accession) was evident. More recently, such QTL-seq approach has been deployed in multiple mapping populations for scanning the major QTLs controlling fruit weight and locule number in tomato.<sup><a reveal-id="DSV036C45" data-open="DSV036C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Collectively, this suggests the accuracy and robustness of mQTL-seq as an approach for rapid delineation of high-resolution major QTLs governing traits of agronomic importance at a genome-wide scale in chickpea. </p>                        <a id="87199634" scrollto-destination="87199634"></a>

  <div id="DSV036F4" data-id="DSV036F4" class="fig fig-section">
    <div class="label fig-label">Figure 4.</div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03604.png?Expires=1509239038&amp;Signature=PeXbaWMwt9gYWB-fHgI2zkU-PeA24FAPwS8nSDZEOwptLN8hp-WZTItnk8MF5iYaMMA6A-m8lGky10ptj8kGqppR3Mk5ClSI2iTs5RcM3FhI1VLwgMu9F7BdlDb-VFSCs-SqRPDgJ4aGVhTS~0c1IPQyH5hJHJppca0CRffzeJrZ1CsK8qm2Z7LnPBp6cXxIlbwfLgOjGXVrtzCoeYy87Pl-8SClk-LHnxQXy0tt6gSBkU9asKVNUckbVpih4r8OkFq9sGpfWQCBMHCzxnFJucSkGMKKbzqPDPUcXNYRFZEh~DAzFbxzGQbDIzRkOzyjNi-rS-YEHyux56PrRbZHIQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The integration of Δ (SNP index)-based QTL-seq-derived four pod number major QTLs in two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] narrowed down one novel major genomic region harbouring a robust pod number QTL (CaqbPN4.2) into 1.28 Mb sequence interval [between flanking SNP (SNP_18917B and SNP_19254B) markers] indicated by green font on kabuli chickpea chromosome 4. This QTL region was subsequently scaled down into the regulatory (C/T) and coding (C/A) SNPs-containing one candidate PPR (pentatricopeptide repeat) gene associated strongly with pod number by combining mQTL-seq with QTL region-specific association analysis (C). *Known pod number QTLs (CaqaPN4.1 and CaqbPN4.1) documented previously by Varshney et al.41 The details regarding SNPs flanking/localized at the major PN QTLs and SNPs in a PPR gene associated with PN are mentioned in the Supplementary Tables S4 and S7. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03604.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199634/dsv03604.tiff" path-from-xml="dsv03604.tiff" target="_blank">View large</a>
        <a section="87199634" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03604.png?Expires=1509239038&amp;Signature=eLJ1RQ0ptacJs5fQRRlgO4Wkw118AWQ37xEM-Cn3t-nFjWDzeE~LfY9IC-FeQudVuAbqYpNNbBXpoMkqDSgAN2~YTrBElO3wBnEu8clXsvvXnFVag3rO~zUWvHptGr8pVnWBaS1Yc9tPjFPfhLLkarflz7-nz4YhvA5ra2xmnfwY-RFe8TLP1GeEG2o3h50HovjrTiHM1EUPOLTTEa5MV9Xe~WbXeLn4QqgifMIn2JztufyozYoBkrV3jjQWjyYf8teFRRgt7OStDxbr-WLK65dFb2UqO-hgK0BKi-qumHNlCGYb47iZ5MOfxRdE6FRaI4h0thbEfWZPes5eMV82Ig__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=87199634&amp;ar=2462457&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv03604.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="caption fig-caption">
      <p>The integration of Δ (SNP index)-based QTL-seq-derived four pod number major QTLs in two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] narrowed down one novel major genomic region harbouring a robust pod number QTL (<em>Caq<sup>b</sup>PN4.2</em>) into 1.28 Mb sequence interval [between flanking SNP (SNP_18917B and SNP_19254B) markers] indicated by green font on <em>kabuli</em> chickpea chromosome 4. This QTL region was subsequently scaled down into the regulatory (C/T) and coding (C/A) SNPs-containing one candidate PPR (pentatricopeptide repeat) gene associated strongly with pod number by combining mQTL-seq with QTL region-specific association analysis (C). *Known pod number QTLs (<em>Caq<sup>a</sup>PN4.1</em> and <em>Caq<sup>b</sup>PN4.1</em>) documented previously by Varshney et al.<sup><a reveal-id="DSV036C41" data-open="DSV036C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The details regarding SNPs flanking/localized at the major PN QTLs and SNPs in a PPR gene associated with PN are mentioned in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&amp;Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Tables S4 and S7</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p>
    </div>
  </div>
  <div content-id="DSV036F4" class="fig fig-modal reveal-modal">
    <div class="label fig-label">Figure 4.</div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03604.png?Expires=1509239038&amp;Signature=PeXbaWMwt9gYWB-fHgI2zkU-PeA24FAPwS8nSDZEOwptLN8hp-WZTItnk8MF5iYaMMA6A-m8lGky10ptj8kGqppR3Mk5ClSI2iTs5RcM3FhI1VLwgMu9F7BdlDb-VFSCs-SqRPDgJ4aGVhTS~0c1IPQyH5hJHJppca0CRffzeJrZ1CsK8qm2Z7LnPBp6cXxIlbwfLgOjGXVrtzCoeYy87Pl-8SClk-LHnxQXy0tt6gSBkU9asKVNUckbVpih4r8OkFq9sGpfWQCBMHCzxnFJucSkGMKKbzqPDPUcXNYRFZEh~DAzFbxzGQbDIzRkOzyjNi-rS-YEHyux56PrRbZHIQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The integration of Δ (SNP index)-based QTL-seq-derived four pod number major QTLs in two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] narrowed down one novel major genomic region harbouring a robust pod number QTL (CaqbPN4.2) into 1.28 Mb sequence interval [between flanking SNP (SNP_18917B and SNP_19254B) markers] indicated by green font on kabuli chickpea chromosome 4. This QTL region was subsequently scaled down into the regulatory (C/T) and coding (C/A) SNPs-containing one candidate PPR (pentatricopeptide repeat) gene associated strongly with pod number by combining mQTL-seq with QTL region-specific association analysis (C). *Known pod number QTLs (CaqaPN4.1 and CaqbPN4.1) documented previously by Varshney et al.41 The details regarding SNPs flanking/localized at the major PN QTLs and SNPs in a PPR gene associated with PN are mentioned in the Supplementary Tables S4 and S7. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03604.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199634/dsv03604.tiff" path-from-xml="dsv03604.tiff" target="_blank">View large</a>
        <a section="87199634" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03604.png?Expires=1509239038&amp;Signature=eLJ1RQ0ptacJs5fQRRlgO4Wkw118AWQ37xEM-Cn3t-nFjWDzeE~LfY9IC-FeQudVuAbqYpNNbBXpoMkqDSgAN2~YTrBElO3wBnEu8clXsvvXnFVag3rO~zUWvHptGr8pVnWBaS1Yc9tPjFPfhLLkarflz7-nz4YhvA5ra2xmnfwY-RFe8TLP1GeEG2o3h50HovjrTiHM1EUPOLTTEa5MV9Xe~WbXeLn4QqgifMIn2JztufyozYoBkrV3jjQWjyYf8teFRRgt7OStDxbr-WLK65dFb2UqO-hgK0BKi-qumHNlCGYb47iZ5MOfxRdE6FRaI4h0thbEfWZPes5eMV82Ig__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=87199634&amp;ar=2462457&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv03604.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="caption fig-caption">
      <p>The integration of Δ (SNP index)-based QTL-seq-derived four pod number major QTLs in two mapping populations [A: (Pusa 1103 × ILWC 46) and B: (Pusa 256 × ILWC 46)] narrowed down one novel major genomic region harbouring a robust pod number QTL (<em>Caq<sup>b</sup>PN4.2</em>) into 1.28 Mb sequence interval [between flanking SNP (SNP_18917B and SNP_19254B) markers] indicated by green font on <em>kabuli</em> chickpea chromosome 4. This QTL region was subsequently scaled down into the regulatory (C/T) and coding (C/A) SNPs-containing one candidate PPR (pentatricopeptide repeat) gene associated strongly with pod number by combining mQTL-seq with QTL region-specific association analysis (C). *Known pod number QTLs (<em>Caq<sup>a</sup>PN4.1</em> and <em>Caq<sup>b</sup>PN4.1</em>) documented previously by Varshney et al.<sup><a reveal-id="DSV036C41" data-open="DSV036C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The details regarding SNPs flanking/localized at the major PN QTLs and SNPs in a PPR gene associated with PN are mentioned in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&amp;Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Tables S4 and S7</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p>
    </div>
  </div>
<p>The targeted resequencing of this 1.28 Mb delineated novel <em>Caq<sup>b</sup>PN4.2</em> robust QTL region in high and low pod number-containing mapping parents (Pusa 1103 × ILWC 46) and bulks (HPNB and LPNB) detected 2,913 high-quality SNPs with an average SNP density of 1/439.4 bp. This contained 1,543 intergenic SNPs and 1,370 SNPs derived from the coding and non-coding sequence components of 448 genes. For QTL region-specific trait association analysis, the genotyping information of 2,913 SNPs mined and mapped on a 1.28 Mb (<em>Caq<sup>b</sup>PN4.2</em>) novel QTL region was correlated with PN field phenotyping data (PN: 21.9–204.5 with 33% CV and 70% <em>H</em><sup>2</sup>) and diversity statistics (population structure, PCA, kinship) of 92 <em>desi</em> and <em>kabuli</em> chickpea accessions (PN-specific association panel). The comprehensive association analysis by effective integration of GLM and MLM outcomes with EMMA and FDR based on multiple-comparisons detected two SNPs (SNP_1 at 32636428 bp: C/A and SNP_19172B at 32640058 bp: C/T) in the CDS and URR of a pentatricopeptide repeat (PPR) protein-coding gene revealing strong association (<em>P</em>: 1.5–2.0 × 10<sup>−7</sup> and <em>R</em><sup>2</sup> = 25–28%) with PN in chickpea (Fig. <a reveal-id="DSV036F4" data-open="DSV036F4" class="link link-reveal link-table xref-fig">4</a>B and C). This is further ascertained by identification of similar PPR gene-derived regulatory SNP (SNP_19172B at 32640058 bp: C/T) with high Δ (SNP index) (0.8) at <em>Caq<sup>b</sup>PN4.2</em> QTL region governing pod number based on our mQTL-seq analysis in two inter-specific mapping populations of chickpea (Fig. <a reveal-id="DSV036F3" data-open="DSV036F3" class="link link-reveal link-table xref-fig">3</a>A and B). Therefore, this strong PN-associated PPR gene localized at a major PN novel QTL interval (<em>Caq<sup>b</sup>PN4.2</em>) was selected as one of the potential candidate for understanding its efficacy in pod number regulation through differential expression profiling.</p>                            <h3 scrollto-destination=87199638 id="87199638" class="section-title" data-legacy-id=s3h>3.8. Validation of a PN-associated PPR gene through differential expression profiling</h3>
<p>One strong PN-associated regulatory and coding SNPs-containing PPR gene delineated at 1.28 Mb major genomic region harbouring a novel <em>Caq<sup>b</sup>PN4.2</em> robust QTL (validated by both mQTL-seq- and QTL region-specific association analysis) was selected for differential expression profiling. The primer-pair designed targeting this gene was used for amplification with the RNA isolated from the vegetative (leaf and root) and reproductive (flower bud, ovary, anther and mature pollen) tissues and two pod (∼5 mm beginning pod size at 5–10 days after anthesis/DAA and ∼2 cm full pod at 15–20 DAA) developmental stages of low and high pod number-containing homozygous mapping individuals (used in QTL-seq analysis) and parental accessions of two mapping populations using semi-quantitative and quantitative RT–PCR assays. A strong PN-associated PPR gene revealed tissue-specific expression in the anthers, mature pollens and pods of parental accessions and mapping individuals compared with their respective vegetative (leaf and root) and reproductive (flower bud and ovary) tissues (Fig. <a reveal-id="DSV036F5" data-open="DSV036F5" class="link link-reveal link-table xref-fig">5</a>). This gene also exhibited pronounced differential up-regulation (∼6.7-folds, <em>P</em> ≤ 10<sup>−3</sup>) in anther, mature pollen and two pod developmental stages (compared with vegetative tissues) of all high and low pod number-containing parental accessions and individuals of two mapping populations under study (Fig. <a reveal-id="DSV036F5" data-open="DSV036F5" class="link link-reveal link-table xref-fig">5</a>). Interestingly, a significant higher (∼3.5-fold, <em>P</em> ≤ 10<sup>−3</sup>) differential up-regulation of PPR gene in anther, mature pollen and pod developmental stages of high pod number-containing parental accessions and mapping individuals compared with low pod number-containing parental accessions and mapping individuals was evident (Fig. <a reveal-id="DSV036F5" data-open="DSV036F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="87199640" scrollto-destination="87199640"></a>
<div id="DSV036F5" data-id="DSV036F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03605.png?Expires=1509239038&Signature=cUDFhhiBtH9CVXkUwxosxizvEbYmG-DzioGAd21pHq72BUZQ5-KOB9OogRYj0TjdLPs9gSRLPtc2jKvHYqzu0jGmExDFHWsre-te3YRWAO2qbkuh08sAopr4VBelyAcjy7Y4Gmog5gnSGkabbYZEL8094Kcw~cNDFXRQKsKlxyDn0H9BmP6VJaLDSN75kxGGOFgMXw45dzO7GgPmqTOEwxgoMqWm~riXvKvszJxZEI8Ai16JSL7~zYTn3kqr3jMgENoVrxcgE7wxt0rwu5PmQgz1pz2LY-uvY7H0unqIM4arvTgeO6dnqa3YkKvGwyqq~MWxNWVpfaDuS2gZ9ywSsw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Differential expression profiles of one PPR gene delineated at a novel 1.28 Mb major genomic region underlying the robust pod number QTL (CaqbPN4.2) (detected by integrating mQTL-seq with QTL region-specific association analysis) in diverse vegetative (leaf and root) and reproductive (flower bud, ovary, anther and mature pollen) tissues and two pod developmental stages of low and high pod number-containing parental accessions and homozygous individuals of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)]. The average log signal expression value of PPR gene in various tissues and developmental stages was specified at the top with a colour scale; in which green, black and red colours represent low, medium and high level of expression, respectively. The tissues/stages as well as homozygous mapping individuals and parental accessions utilized for expression profiling are indicated on the top and right sides of expression map, respectively. An endogenous control elongation factor-1 alpha was used in quantitative RT–PCR assay for normalization of the expression value across different tissues/developmental stages of parents and mapping individuals. HPNB: high pod number bulk and LPNB: low pod number bulk. Pod I: Pod development stage 1 (∼5 mm beginning pod size at 5–10 days after anthesis/DAA) and Pod II: Pod development stage 2 (∼2 cm full pod at 15–20 DAA). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03605.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199640/dsv03605.tiff" path-from-xml="dsv03605.tiff" target="_blank">View large</a><a section="87199640" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03605.png?Expires=1509239038&Signature=TobRBUhtRG0lOONv8Xhy6Q29h0YbgMUYCTbiiycJSaB-hRvn8rzAd1ZGVchqazLS2OBv4-lNdjPbpyDM2zD0OKo0P0tezHRKsUKaARdghaGHntZeO1PZk2T20y04CJAG-FXKpl33ihnj5HiHKihbvnvT1e-9j6Ftg-zeWKypg0qOuDF-bEw3sWX-GbUtBOI4DyeRpG0qHhxr2a0ZR-hP1x40fEpbu0EGf2fYdfx4dGT278RQZa1rJluSSxA6x8AIUzAk~9JLVpKn9hvp9adTQNmXyzHD2JQOhcKLe29NyJyeVvKmsVT6~rp-2hYQZ1tY-I3AN9eHzkOGdAqJcQvJWQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199640&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03605.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differential expression profiles of one PPR gene delineated at a novel 1.28 Mb major genomic region underlying the robust pod number QTL (<em>Caq<sup>b</sup>PN4.2</em>) (detected by integrating mQTL-seq with QTL region-specific association analysis) in diverse vegetative (leaf and root) and reproductive (flower bud, ovary, anther and mature pollen) tissues and two pod developmental stages of low and high pod number-containing parental accessions and homozygous individuals of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)]. The average log signal expression value of PPR gene in various tissues and developmental stages was specified at the top with a colour scale; in which green, black and red colours represent low, medium and high level of expression, respectively. The tissues/stages as well as homozygous mapping individuals and parental accessions utilized for expression profiling are indicated on the top and right sides of expression map, respectively. An endogenous control elongation factor-1 alpha was used in quantitative RT–PCR assay for normalization of the expression value across different tissues/developmental stages of parents and mapping individuals. HPNB: high pod number bulk and LPNB: low pod number bulk. Pod I: Pod development stage 1 (∼5 mm beginning pod size at 5–10 days after anthesis/DAA) and Pod II: Pod development stage 2 (∼2 cm full pod at 15–20 DAA). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV036F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/m_dsv03605.png?Expires=1509239038&Signature=cUDFhhiBtH9CVXkUwxosxizvEbYmG-DzioGAd21pHq72BUZQ5-KOB9OogRYj0TjdLPs9gSRLPtc2jKvHYqzu0jGmExDFHWsre-te3YRWAO2qbkuh08sAopr4VBelyAcjy7Y4Gmog5gnSGkabbYZEL8094Kcw~cNDFXRQKsKlxyDn0H9BmP6VJaLDSN75kxGGOFgMXw45dzO7GgPmqTOEwxgoMqWm~riXvKvszJxZEI8Ai16JSL7~zYTn3kqr3jMgENoVrxcgE7wxt0rwu5PmQgz1pz2LY-uvY7H0unqIM4arvTgeO6dnqa3YkKvGwyqq~MWxNWVpfaDuS2gZ9ywSsw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Differential expression profiles of one PPR gene delineated at a novel 1.28 Mb major genomic region underlying the robust pod number QTL (CaqbPN4.2) (detected by integrating mQTL-seq with QTL region-specific association analysis) in diverse vegetative (leaf and root) and reproductive (flower bud, ovary, anther and mature pollen) tissues and two pod developmental stages of low and high pod number-containing parental accessions and homozygous individuals of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)]. The average log signal expression value of PPR gene in various tissues and developmental stages was specified at the top with a colour scale; in which green, black and red colours represent low, medium and high level of expression, respectively. The tissues/stages as well as homozygous mapping individuals and parental accessions utilized for expression profiling are indicated on the top and right sides of expression map, respectively. An endogenous control elongation factor-1 alpha was used in quantitative RT–PCR assay for normalization of the expression value across different tissues/developmental stages of parents and mapping individuals. HPNB: high pod number bulk and LPNB: low pod number bulk. Pod I: Pod development stage 1 (∼5 mm beginning pod size at 5–10 days after anthesis/DAA) and Pod II: Pod development stage 2 (∼2 cm full pod at 15–20 DAA). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03605.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199640/dsv03605.tiff" path-from-xml="dsv03605.tiff" target="_blank">View large</a><a section="87199640" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv03605.png?Expires=1509239038&Signature=TobRBUhtRG0lOONv8Xhy6Q29h0YbgMUYCTbiiycJSaB-hRvn8rzAd1ZGVchqazLS2OBv4-lNdjPbpyDM2zD0OKo0P0tezHRKsUKaARdghaGHntZeO1PZk2T20y04CJAG-FXKpl33ihnj5HiHKihbvnvT1e-9j6Ftg-zeWKypg0qOuDF-bEw3sWX-GbUtBOI4DyeRpG0qHhxr2a0ZR-hP1x40fEpbu0EGf2fYdfx4dGT278RQZa1rJluSSxA6x8AIUzAk~9JLVpKn9hvp9adTQNmXyzHD2JQOhcKLe29NyJyeVvKmsVT6~rp-2hYQZ1tY-I3AN9eHzkOGdAqJcQvJWQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87199640&ar=2462457&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03605.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differential expression profiles of one PPR gene delineated at a novel 1.28 Mb major genomic region underlying the robust pod number QTL (<em>Caq<sup>b</sup>PN4.2</em>) (detected by integrating mQTL-seq with QTL region-specific association analysis) in diverse vegetative (leaf and root) and reproductive (flower bud, ovary, anther and mature pollen) tissues and two pod developmental stages of low and high pod number-containing parental accessions and homozygous individuals of two mapping populations [(Pusa 1103 × ILWC 46) and (Pusa 256 × ILWC 46)]. The average log signal expression value of PPR gene in various tissues and developmental stages was specified at the top with a colour scale; in which green, black and red colours represent low, medium and high level of expression, respectively. The tissues/stages as well as homozygous mapping individuals and parental accessions utilized for expression profiling are indicated on the top and right sides of expression map, respectively. An endogenous control elongation factor-1 alpha was used in quantitative RT–PCR assay for normalization of the expression value across different tissues/developmental stages of parents and mapping individuals. HPNB: high pod number bulk and LPNB: low pod number bulk. Pod I: Pod development stage 1 (∼5 mm beginning pod size at 5–10 days after anthesis/DAA) and Pod II: Pod development stage 2 (∼2 cm full pod at 15–20 DAA). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The integration of mQTL-seq (two inter-specific mapping populations, Pusa 1103 × ILWC 46 and Pusa 256 × ILWC 46) with QTL region-specific association analysis (92 association panel) and differential expression profiling (anther, mature pollen and pod developmental stages) has got functional relevance to identify potential novel regulatory and coding SNP allelic variants in one of the major PPR gene delineated at a novel robust QTL region (<em>Caq<sup>b</sup>PN4.2</em>) associated strongly with pod number in chickpea. The efficacy of the combinatorial approach integrating genetic and association mapping with expression profiling to scale down potential candidate gene and alleles at a major QTL region governing diverse agronomic traits, including seed weight and pod/seed number is well established in chickpea.<sup><a reveal-id="DSV036C9  DSV036C13" data-open="DSV036C9  DSV036C13" class="link link-ref link-reveal xref-bibr">9–13</a>, <a reveal-id="DSV036C22" data-open="DSV036C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSV036C23" data-open="DSV036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> A pod number-regulating PPR gene identified in our study, showing &gt;80% sequence conservation with <em>Arabidopsis</em> gene orthologue (<em>At1g52620</em>), is reportedly involved in controlling growth and development-related traits (pollination and embryogenesis) in multiple crop plants, including <em>Arabidopsis</em> and <em>Phaseolus</em>.<sup><a reveal-id="DSV036C52" data-open="DSV036C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSV036C53" data-open="DSV036C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Specifically, PPR gene family proteins have definite functional role in regulating proper pollen development by higher accumulation of transcripts in the mature pollen and efficient nuclear-organelle interactions to produce higher fertile seeds (embryogenesis) and pods in diverse plant species.<sup><a reveal-id="DSV036C54  DSV036C62" data-open="DSV036C54  DSV036C62" class="link link-ref link-reveal xref-bibr">54–62</a></sup> Accordingly, we observed a higher expression of PPR gene transcripts in the mature pollen and two pod developmental stages of high pod number-containing parental accessions and homozygous mapping individuals vis-a-vis low pod number accessions and mapping individuals in chickpea. However, a detail molecular characterization and functional validation of this gene is essential to decipher its regulation specifically during pollen and pod development causing high pod number in chickpea. The PPR gene once validated in a large scale could be deployed for genomics-assisted crop improvement to develop genetically tailored varieties with higher pod/seed number and yield in chickpea.</p>                            <h2 scrollto-destination=87199649 id="87199649" class="section-title" data-legacy-id=s5>Funding</h2>
<p>Funding to pay the Open Access publication charges for this article was provided by the National Institute of Plant Genome Research (NIPGR).</p>                            <h2 scrollto-destination=87199652 id="87199652" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors gratefully acknowledge the financial support for this study provided by a research grant from the Department of Biotechnology (DBT), Government of India (102/IFD/SAN/2161/2013-14). S.D. acknowledges the DBT for Junior Research Fellowship award. The authors also acknowledge the financial support provided by the Department of Agriculture and Co-operation (DAC), Government of India for developing wide cross populations and their precise phenotyping under the National Food Security Mission. We are thankful to the Editor and reviewers for critically evaluating the manuscript and providing constructive comments.</p>                            <h2 scrollto-destination=87199655 id="87199655" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsv036c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Choudhary</div> <div class="given-names">A.K.</div></div>, <div class="name string-name"><div class="surname">Solanki</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Pratap</div> <div class="given-names">A.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Towards marker-assisted selection in pulses: a review</div>, <div class="source">Plant Breed.</div> , <div class="volume">130</div>, <div class="fpage">297</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20marker-assisted%20selection%20in%20pulses%3A%20a%20review&amp;author=A.%20Kumar&amp;author=A.K.%20Choudhary&amp;author=R.K.%20Solanki&amp;author=A.%20Pratap&amp;publication_year=2011&amp;journal=Plant%20Breed.&amp;volume=130&amp;pages=297-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2011.130.issue-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2011.130.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2011.130.issue-3"> </span></div> </div></div></div></div></div><div content-id="dsv036c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Murali Mohan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">P.M.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Achievements and prospects of genomics-assisted breeding in three legume crops of the semi-arid tropics</div>, <div class="source">Biotechnol. Adv.</div> , <div class="volume">31</div>, <div class="fpage">1120</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Achievements%20and%20prospects%20of%20genomics-assisted%20breeding%20in%20three%20legume%20crops%20of%20the%20semi-arid%20tropics&amp;author=R.K.%20Varshney&amp;author=S.%20Murali%20Mohan&amp;author=P.M.%20Gaur&amp;publication_year=2013&amp;journal=Biotechnol.%20Adv.&amp;volume=31&amp;pages=1120-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2013.01.001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2013.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2013.01.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23313999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gholipoor</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2007</div>, <div class="article-title">Potential effects of individual versus simultaneous climate change factors on growth and water use in chickpea</div>, <div class="source">Int. J. Plant Prod.</div> , <div class="volume">1</div>, <div class="fpage">189</div>–<div class="lpage">204</div>.</div></div></div></div><div content-id="dsv036c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Merila</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Hendry</div> <div class="given-names">A.P.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Climate change, adaptation, and phenotypic plasticity: the problem and the evidence</div>, <div class="source">Evol. Appl.</div> , <div class="volume">7</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Climate%20change%2C%20adaptation%2C%20and%20phenotypic%20plasticity%3A%20the%20problem%20and%20the%20evidence&amp;author=J.%20Merila&amp;author=A.P.%20Hendry&amp;publication_year=2014&amp;journal=Evol.%20Appl.&amp;volume=7&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/eva.12137" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Feva.12137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Feva.12137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24454544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jha</div>
               <div class="given-names">U.C.</div>
            </div>, <div class="name string-name"><div class="surname">Chaturvedi</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Bohra</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Basu</div> <div class="given-names">P.S.</div></div>, <div class="name string-name"><div class="surname">Khan</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Barh</div> <div class="given-names">D.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Abiotic stresses, constraints and improvement strategies in chickpea</div>, <div class="source">Plant Breed.</div> , <div class="volume">133</div>, <div class="fpage">163</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abiotic%20stresses%2C%20constraints%20and%20improvement%20strategies%20in%20chickpea&amp;author=U.C.%20Jha&amp;author=S.K.%20Chaturvedi&amp;author=A.%20Bohra&amp;author=P.S.%20Basu&amp;author=M.S.%20Khan&amp;author=D.%20Barh&amp;publication_year=2014&amp;journal=Plant%20Breed.&amp;volume=133&amp;pages=163-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2014.133.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2014.133.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2014.133.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsv036c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Chamarthi</div> <div class="given-names">S.K.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Fast-track introgression of “QTL-hotspot” for root traits and other drought tolerance trait in JG 11, an elite and leading variety of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Plant Genome</div> , <div class="volume">6</div>, <div class="pub-id-doi"><a href="https://doi.org/10.3835/plantgenome2013.07.0022" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.3835/plantgenome2013.07.0022</a></div>.</div></div></div></div><div content-id="dsv036c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Mohan</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">P.M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Marker-assisted backcrossing to introgress resistance to <em>Fusarium</em> wilt race 1 and <em>Ascochyta</em> blight in C214, an elite cultivar of chickpea</div>, <div class="source">Plant Genome</div> , <div class="volume">7</div>, <div class="pub-id-doi"><a href="https://doi.org/10.3835/plantgenome2013.10.0035" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.3835/plantgenome2013.10.0035</a></div>.</div></div></div></div><div content-id="dsv036c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ali</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Madrid</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Mapping and identification of a <em>Cicer arietinum NSP2</em> gene involved in nodulation pathway</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">127</div>, <div class="fpage">481</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20identification%20of%20a%20Cicer%20arietinum%20NSP2%20gene%20involved%20in%20nodulation%20pathway&amp;author=L.%20Ali&amp;author=E.%20Madrid&amp;author=R.K.%20Varshney&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=127&amp;pages=481-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2233-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2233-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2233-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24247237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saxena</div>
               <div class="given-names">M.S.</div>
            </div>, <div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Das</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">An integrated genomic approach for rapid delineation of candidate genes regulating agro-morphological traits in chickpea</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">695</div>–<div class="lpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genomic%20approach%20for%20rapid%20delineation%20of%20candidate%20genes%20regulating%20agro-morphological%20traits%20in%20chickpea&amp;author=M.S.%20Saxena&amp;author=D.%20Bajaj&amp;author=S.%20Das&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=695-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu031" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25335477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kujur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">M.S.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Functionally relevant microsatellite markers from chickpea transcription factor genes for efficient genotyping applications and trait association mapping</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">355</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functionally%20relevant%20microsatellite%20markers%20from%20chickpea%20transcription%20factor%20genes%20for%20efficient%20genotyping%20applications%20and%20trait%20association%20mapping&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=M.S.%20Saxena&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=355-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst015" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23633531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kujur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">M.S.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">An efficient and cost-effective approach for genic microsatellite marker-based large-scale trait association mapping: identification of candidate genes for seed weight in chickpea</div>, <div class="source">Mol. Breed.</div> , <div class="volume">34</div>, <div class="fpage">241</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20and%20cost-effective%20approach%20for%20genic%20microsatellite%20marker-based%20large-scale%20trait%20association%20mapping%3A%20identification%20of%20candidate%20genes%20for%20seed%20weight%20in%20chickpea&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=M.S.%20Saxena&amp;publication_year=2014&amp;journal=Mol.%20Breed.&amp;volume=34&amp;pages=241-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-014-0033-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-014-0033-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-014-0033-3"> </span></div> </div></div></div></div></div><div content-id="dsv036c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kujur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Shree</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Ultra-high density intra-specific genetic linkage maps accelerate identification of functionally relevant molecular tags governing important agronomic traits in chickpea</div>, <div class="source">Sci. Rep.</div> , <div class="volume">5</div>, <div class="fpage">9468</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultra-high%20density%20intra-specific%20genetic%20linkage%20maps%20accelerate%20identification%20of%20functionally%20relevant%20molecular%20tags%20governing%20important%20agronomic%20traits%20in%20chickpea&amp;author=A.%20Kujur&amp;author=H.D.%20Upadhyaya&amp;author=T.%20Shree&amp;publication_year=2015&amp;journal=Sci.%20Rep.&amp;volume=5&amp;pages=9468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep09468" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep09468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep09468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25942004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kujur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">A genome-wide SNP scan accelerates trait-regulatory genomic loci identification in chickpea</div>, <div class="source">Sci. Rep.</div> , <div class="volume">5</div>, <div class="fpage">11166</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20SNP%20scan%20accelerates%20trait-regulatory%20genomic%20loci%20identification%20in%20chickpea&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=H.D.%20Upadhyaya&amp;publication_year=2015&amp;journal=Sci.%20Rep.&amp;volume=5&amp;pages=11166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep11166" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep11166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep11166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26058368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuinstra</div>
               <div class="given-names">M.R.</div>
            </div>, <div class="name string-name"><div class="surname">Ejeta</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Goldsbrough</div> <div class="given-names">P.B.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">95</div>, <div class="fpage">1005</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneous%20inbred%20family%20(HIF)%20analysis%3A%20a%20method%20for%20developing%20near-isogenic%20lines%20that%20differ%20at%20quantitative%20trait%20loci&amp;author=M.R.%20Tuinstra&amp;author=G.%20Ejeta&amp;author=P.B.%20Goldsbrough&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=95&amp;pages=1005-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050654" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050654"> </span></div> </div></div></div></div></div><div content-id="dsv036c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Monforte</div>
               <div class="given-names">A.J.</div>
            </div>, <div class="name string-name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Development of a set of near isogenic and backcross recombinant inbred lines containing most of the <em>Lycopersicon hirsutum</em> genome in a <em>L. esculentum</em> genetic background: a tool for gene mapping and gene discovery</div>, <div class="source">Genome</div> , <div class="volume">43</div>, <div class="fpage">803</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20set%20of%20near%20isogenic%20and%20backcross%20recombinant%20inbred%20lines%20containing%20most%20of%20the%20Lycopersicon%20hirsutum%20genome%20in%20a%20L.%20esculentum%20genetic%20background%3A%20a%20tool%20for%20gene%20mapping%20and%20gene%20discovery&amp;author=A.J.%20Monforte&amp;author=S.D.%20Tanksley&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=803-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-43-5-803" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-43-5-803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-43-5-803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11081970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Loudet</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Gaudon</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Trubuil</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Daniel-Vedele</div> <div class="given-names">F.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Quantitative trait loci controlling root growth and architecture in <em>Arabidopsis thaliana</em> confirmed by heterogeneous inbred family</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">110</div>, <div class="fpage">742</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20controlling%20root%20growth%20and%20architecture%20in%20Arabidopsis%20thaliana%20confirmed%20by%20heterogeneous%20inbred%20family&amp;author=O.%20Loudet&amp;author=V.%20Gaudon&amp;author=A.%20Trubuil&amp;author=F.%20Daniel-Vedele&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=742-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1900-9" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1900-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1900-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15678326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gaur</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Azam</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jeena</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">357</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20genotyping%20for%20constructing%20a%20saturated%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Gaur&amp;author=S.%20Azam&amp;author=G.%20Jeena&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=357-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss018" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hiremath</div>
               <div class="given-names">P.J.</div>
            </div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Penmetsa</div> <div class="given-names">R.V.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Large-scale development of cost-effective SNP marker assays for diversity assessment and genetic mapping in chickpea and comparative mapping in legumes</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">10</div>, <div class="fpage">716</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20development%20of%20cost-effective%20SNP%20marker%20assays%20for%20diversity%20assessment%20and%20genetic%20mapping%20in%20chickpea%20and%20comparative%20mapping%20in%20legumes&amp;author=P.J.%20Hiremath&amp;author=A.%20Kumar&amp;author=R.V.%20Penmetsa&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=716-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00710.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00710.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00710.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22703242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Roorkiwal</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Sawargaonkar</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Chitikineni</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Single nucleotide polymorphism genotyping for breeding and genetics applications in chickpea and pigeonpea using the BeadXpress platform</div>, <div class="source">Plant Genome</div> , <div class="volume">6</div>, <div class="pub-id-doi"><a href="https://doi.org/10.3835/plantgenome2013.05.0017" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.3835/plantgenome2013.05.0017</a></div>.</div></div></div></div><div content-id="dsv036c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Deokar</div>
               <div class="given-names">A.A.</div>
            </div>, <div class="name string-name"><div class="surname">Ramsay</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Sharpe</div> <div class="given-names">A.G.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">708</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20wide%20SNP%20identification%20in%20chickpea%20for%20use%20in%20development%20of%20a%20high%20density%20genetic%20map%20and%20improvement%20of%20chickpea%20reference%20genome%20assembly&amp;author=A.A.%20Deokar&amp;author=L.%20Ramsay&amp;author=A.G.%20Sharpe&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=708" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-708" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25150411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jaganathan</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Thudi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kale</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genotyping-by-sequencing based intra-specific genetic map refines a “QTL-hotspot” region for drought tolerance in chickpea</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">290</div>, <div class="fpage">559</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20based%20intra-specific%20genetic%20map%20refines%20a%20%E2%80%9CQTL-hotspot%E2%80%9D%20region%20for%20drought%20tolerance%20in%20chickpea&amp;author=D.%20Jaganathan&amp;author=M.%20Thudi&amp;author=S.%20Kale&amp;publication_year=2015&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=290&amp;pages=559-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-014-0932-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-014-0932-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-014-0932-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25344290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bajaj</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Kujur</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genome-wide conserved non-coding microsatellite (CNMS) marker-based integrative genetical genomics for quantitative dissection of seed weight in chickpea</div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">66</div>, <div class="fpage">1271</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20conserved%20non-coding%20microsatellite%20(CNMS)%20marker-based%20integrative%20genetical%20genomics%20for%20quantitative%20dissection%20of%20seed%20weight%20in%20chickpea&amp;author=D.%20Bajaj&amp;author=M.S.%20Saxena&amp;author=A.%20Kujur&amp;publication_year=2015&amp;journal=J.%20Exp.%20Bot.&amp;volume=66&amp;pages=1271-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/eru478" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Feru478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Feru478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25504138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bajaj</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Khan</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">A combinatorial approach of comprehensive QTL-based comparative genome mapping and transcript profiling identified a seed weight-regulating candidate gene in chickpea</div>, <div class="source">Sci. Rep.</div> , <div class="volume">5</div>, <div class="fpage">9264</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20combinatorial%20approach%20of%20comprehensive%20QTL-based%20comparative%20genome%20mapping%20and%20transcript%20profiling%20identified%20a%20seed%20weight-regulating%20candidate%20gene%20in%20chickpea&amp;author=D.%20Bajaj&amp;author=H.D.%20Upadhyaya&amp;author=Y.%20Khan&amp;publication_year=2015&amp;journal=Sci.%20Rep.&amp;volume=5&amp;pages=9264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep09264" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep09264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep09264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25786576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cho</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Shultz</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Anupama</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tefera</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Muehlbauer</div> <div class="given-names">F.J.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Mapping genes for double podding and other morphological traits in chickpea</div>, <div class="source">Euphytica</div> , <div class="volume">125</div>, <div class="fpage">285</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20genes%20for%20double%20podding%20and%20other%20morphological%20traits%20in%20chickpea&amp;author=S.%20Cho&amp;author=J.%20Kumar&amp;author=J.F.%20Shultz&amp;author=K.%20Anupama&amp;author=F.%20Tefera&amp;author=F.J.%20Muehlbauer&amp;publication_year=2002&amp;journal=Euphytica&amp;volume=125&amp;pages=285-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1020872009306" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1020872009306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1020872009306"> </span></div> </div></div></div></div></div><div content-id="dsv036c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rakshit</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Tekeoglu</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">DAF marker tightly linked to a major locus for <em>Ascochyta</em> blight resistance in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Euphytica</div> , <div class="volume">132</div>, <div class="fpage">23</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DAF%20marker%20tightly%20linked%20to%20a%20major%20locus%20for%20Ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=S.%20Rakshit&amp;author=P.%20Winter&amp;author=M.%20Tekeoglu&amp;publication_year=2003&amp;journal=Euphytica&amp;volume=132&amp;pages=23-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1024681422799" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1024681422799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1024681422799"> </span></div> </div></div></div></div></div><div content-id="dsv036c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cobos</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Fernandez</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rubio</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Linkage map of chickpea (<em>Cicer arietinum</em> L.) based on populations from <em>Kabuli</em> × <em>Desi</em> crosses: location of genes for resistance to <em>Fusarium</em> wilt race 0</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">110</div>, <div class="fpage">1347</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)%20based%20on%20populations%20from%20Kabuli%20%C3%97%20Desi%20crosses%3A%20location%20of%20genes%20for%20resistance%20to%20Fusarium%20wilt%20race%200&amp;author=M.J.%20Cobos&amp;author=M.%20Fernandez&amp;author=J.%20Rubio&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=1347-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1980-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1980-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1980-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806343" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cobos</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Rubio</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Strange</div> <div class="given-names">R.N.</div></div>, <div class="name string-name"><div class="surname">Moreno</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Gil</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Millán</div> <div class="given-names">T.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">A new QTL for <em>Ascochyta</em> blight resistance in an RIL population derived from an interspecific cross in chickpea</div>, <div class="source">Euphytica</div> , <div class="volume">149</div>, <div class="fpage">105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20QTL%20for%20Ascochyta%20blight%20resistance%20in%20an%20RIL%20population%20derived%20from%20an%20interspecific%20cross%20in%20chickpea&amp;author=M.%20Cobos&amp;author=J.%20Rubio&amp;author=R.N.%20Strange&amp;author=M.T.%20Moreno&amp;author=J.%20Gil&amp;author=T.%20Mill%C3%A1n&amp;publication_year=2006&amp;journal=Euphytica&amp;volume=149&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-005-9058-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-005-9058-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-005-9058-3"> </span></div> </div></div></div></div></div><div content-id="dsv036c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cobos</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Winter</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kharrat</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Genetic analysis of agronomic traits in a wide cross of chickpea</div>, <div class="source">Field Crops Res.</div> , <div class="volume">111</div>, <div class="fpage">130</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20agronomic%20traits%20in%20a%20wide%20cross%20of%20chickpea&amp;author=M.J.%20Cobos&amp;author=P.%20Winter&amp;author=M.%20Kharrat&amp;publication_year=2009&amp;journal=Field%20Crops%20Res.&amp;volume=111&amp;pages=130-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fcr.2008.11.006" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fcr.2008.11.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fcr.2008.11.006"> </span></div> </div></div></div></div></div><div content-id="dsv036c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lichtenzveig</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Bonfil</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Shtienberg</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Mapping quantitative trait loci in chickpea associated with time to flowering and resistance to <em>Didymella rabiei</em> the causal agent of <em>Ascochyta</em> blight</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">113</div>, <div class="fpage">1357</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20quantitative%20trait%20loci%20in%20chickpea%20associated%20with%20time%20to%20flowering%20and%20resistance%20to%20Didymella%20rabiei%20the%20causal%20agent%20of%20Ascochyta%20blight&amp;author=J.%20Lichtenzveig&amp;author=D.J.%20Bonfil&amp;author=H.B.%20Zhang&amp;author=D.%20Shtienberg&amp;author=S.%20Abbo&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=1357-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0390-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0390-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0390-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17016689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tar'an</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Warkentin</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Tullu</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vanderberg</div> <div class="given-names">A.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Genetic mapping of <em>Ascochyta</em> blight resistance in chickpea (<em>Cicer arietinum</em>) using a simple sequence repeat linkage map</div>, <div class="source">Genome</div> , <div class="volume">50</div>, <div class="fpage">26</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20Ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum)%20using%20a%20simple%20sequence%20repeat%20linkage%20map&amp;author=B.%20Tar'an&amp;author=T.D.%20Warkentin&amp;author=A.%20Tullu&amp;author=A.%20Vanderberg&amp;publication_year=2007&amp;journal=Genome&amp;volume=50&amp;pages=26-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g06-137" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg06-137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg06-137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17546068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Radhika</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Gowda</div> <div class="given-names">S.J.M.</div></div>, <div class="name string-name"><div class="surname">Kadoo</div> <div class="given-names">N.Y.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Development of an integrated intraspecific map of chickpea (<em>Cicer arietinum</em> L.) using two recombinant inbred line populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">115</div>, <div class="fpage">209</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20an%20integrated%20intraspecific%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)%20using%20two%20recombinant%20inbred%20line%20populations&amp;author=P.%20Radhika&amp;author=S.J.M.%20Gowda&amp;author=N.Y.%20Kadoo&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=209-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0556-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0556-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0556-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17503013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Anbessa</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Tar'an</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Warkentin</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Tullu</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vandenberg</div> <div class="given-names">A.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Genetic analyses and conservation of QTL for <em>Ascochyta</em> blight resistance in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">119</div>, <div class="fpage">757</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analyses%20and%20conservation%20of%20QTL%20for%20Ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=Y.%20Anbessa&amp;author=B.%20Tar'an&amp;author=T.D.%20Warkentin&amp;author=A.%20Tullu&amp;author=A.%20Vandenberg&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=119&amp;pages=757-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1086-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1086-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1086-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19517090" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gowda</div>
               <div class="given-names">S.J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Radhika</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kadoo</div> <div class="given-names">N.Y.</div></div>, <div class="name string-name"><div class="surname">Mhase</div> <div class="given-names">L.B.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">V.S.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Molecular mapping of wilt resistance genes in chickpea</div>, <div class="source">Mol. Breed.</div> , <div class="volume">24</div>, <div class="fpage">177</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20of%20wilt%20resistance%20genes%20in%20chickpea&amp;author=S.J.M.%20Gowda&amp;author=P.%20Radhika&amp;author=N.Y.%20Kadoo&amp;author=L.B.%20Mhase&amp;author=V.S.%20Gupta&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=24&amp;pages=177-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9282-y" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9282-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9282-y"> </span></div> </div></div></div></div></div><div content-id="dsv036c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Anuradha</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Pande</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Mapping QTL for resistance to <em>Botrytis</em> grey mould in chickpea</div>, <div class="source">Euphytica</div> , <div class="volume">182</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20QTL%20for%20resistance%20to%20Botrytis%20grey%20mould%20in%20chickpea&amp;author=C.%20Anuradha&amp;author=P.M.%20Gaur&amp;author=S.%20Pande&amp;publication_year=2011&amp;journal=Euphytica&amp;volume=182&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-011-0394-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-011-0394-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-011-0394-1"> </span></div> </div></div></div></div></div><div content-id="dsv036c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Aryamanesh</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Clarke</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Siddique</div> <div class="given-names">K.H.M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Mapping a major gene for growth habit and QTLs for <em>Ascochyta</em> blight resistance and flowering time in a population between chickpea and <em>Cicer reticulatum</em></div>, <div class="source">Euphytica</div> , <div class="volume">173</div>, <div class="fpage">307</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20a%20major%20gene%20for%20growth%20habit%20and%20QTLs%20for%20Ascochyta%20blight%20resistance%20and%20flowering%20time%20in%20a%20population%20between%20chickpea%20and%20Cicer%20reticulatum&amp;author=N.%20Aryamanesh&amp;author=M.N.%20Nelson&amp;author=G.%20Yan&amp;author=H.J.%20Clarke&amp;author=K.H.M.%20Siddique&amp;publication_year=2010&amp;journal=Euphytica&amp;volume=173&amp;pages=307-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-009-0086-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-009-0086-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-009-0086-2"> </span></div> </div></div></div></div></div><div content-id="dsv036c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gowda</div>
               <div class="given-names">C.L.L.</div>
            </div>, <div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Dronavalli</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Identification of large-seeded high-yielding stable <em>kabuli</em> chickpea germplasm lines for use in crop improvement</div>, <div class="source">Crop Sci.</div> , <div class="volume">5</div>, <div class="fpage">198</div>–<div class="lpage">209</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20large-seeded%20high-yielding%20stable%20kabuli%20chickpea%20germplasm%20lines%20for%20use%20in%20crop%20improvement&amp;author=C.L.L.%20Gowda&amp;author=H.D.%20Upadhyaya&amp;author=N.%20Dronavalli&amp;author=S.%20Singh&amp;publication_year=2011&amp;journal=Crop%20Sci.&amp;volume=5&amp;pages=198-209" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2010.01.0078" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2010.01.0078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2010.01.0078"> </span></div> </div></div></div></div></div><div content-id="dsv036c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rehman</div>
               <div class="given-names">A.U.</div>
            </div>, <div class="name string-name"><div class="surname">Malhotra</div> <div class="given-names">R.S.</div></div>, <div class="name string-name"><div class="surname">Bett</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tar'an</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bueckert</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Warkentin</div> <div class="given-names">T.D.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Mapping QTL associated with traits affecting grain yield in chickpea (<em>Cicer arietinum</em> L.) under terminal drought stress</div>, <div class="source">Crop Sci.</div> , <div class="volume">51</div>, <div class="fpage">450</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20QTL%20associated%20with%20traits%20affecting%20grain%20yield%20in%20chickpea%20(Cicer%20arietinum%20L.)%20under%20terminal%20drought%20stress&amp;author=A.U.%20Rehman&amp;author=R.S.%20Malhotra&amp;author=K.%20Bett&amp;author=B.%20Tar'an&amp;author=R.%20Bueckert&amp;author=T.D.%20Warkentin&amp;publication_year=2011&amp;journal=Crop%20Sci.&amp;volume=51&amp;pages=450-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2010.03.0129" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2010.03.0129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2010.03.0129"> </span></div> </div></div></div></div></div><div content-id="dsv036c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vadez</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Krishnamurthy</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Thudi</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Assessment of ICCV2 × JG62 chickpea progenies shows sensitivity of reproduction to salt stress and reveals QTLs for seed yield and yield components</div>, <div class="source">Mol. Breed.</div> , <div class="volume">30</div>, <div class="fpage">9</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20ICCV2%20%C3%97%20JG62%20chickpea%20progenies%20shows%20sensitivity%20of%20reproduction%20to%20salt%20stress%20and%20reveals%20QTLs%20for%20seed%20yield%20and%20yield%20components&amp;author=V.%20Vadez&amp;author=L.%20Krishnamurthy&amp;author=M.%20Thudi&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=9-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9594-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9594-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9594-6"> </span></div> </div></div></div></div></div><div content-id="dsv036c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sabbavarapu</div>
               <div class="given-names">M.M.</div>
            </div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chamarthi</div> <div class="given-names">S.K.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Molecular mapping of QTLs for resistance to <em>Fusarium</em> wilt (race 1) and <em>Ascochyta</em> blight in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Euphytica</div> , <div class="volume">193</div>, <div class="fpage">121</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20of%20QTLs%20for%20resistance%20to%20Fusarium%20wilt%20(race%201)%20and%20Ascochyta%20blight%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=M.M.%20Sabbavarapu&amp;author=M.%20Sharma&amp;author=S.K.%20Chamarthi&amp;publication_year=2013&amp;journal=Euphytica&amp;volume=193&amp;pages=121-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-013-0959-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-013-0959-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-013-0959-2"> </span></div> </div></div></div></div></div><div content-id="dsv036c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stephens</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Lombardi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cogan</div> <div class="given-names">N.O.I.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genetic marker discovery, intraspecific linkage map construction and quantitative trait locus analysis of <em>Ascochyta</em> blight resistance in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="volume">33</div>, <div class="fpage">297</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20marker%20discovery%2C%20intraspecific%20linkage%20map%20construction%20and%20quantitative%20trait%20locus%20analysis%20of%20Ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=A.%20Stephens&amp;author=M.%20Lombardi&amp;author=N.O.I.%20Cogan&amp;publication_year=2014&amp;journal=Mol.%20Breed.&amp;volume=33&amp;pages=297-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-013-9950-9" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-013-9950-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-013-9950-9"> </span></div> </div></div></div></div></div><div content-id="dsv036c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Thudi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nayak</div> <div class="given-names">S.N.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genetic dissection of drought tolerance in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">127</div>, <div class="fpage">445</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20drought%20tolerance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.K.%20Varshney&amp;author=M.%20Thudi&amp;author=S.N.%20Nayak&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=127&amp;pages=445-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2230-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2230-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2230-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24326458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Takagi</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Abe</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations</div>, <div class="source">Plant J.</div> , <div class="volume">74</div>, <div class="fpage">174</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL-seq%3A%20rapid%20mapping%20of%20quantitative%20trait%20loci%20in%20rice%20by%20whole%20genome%20resequencing%20of%20DNA%20from%20two%20bulked%20populations&amp;author=H.%20Takagi&amp;author=A.%20Abe&amp;author=K.%20Yoshida&amp;publication_year=2013&amp;journal=Plant%20J.&amp;volume=74&amp;pages=174-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.74.issue-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.74.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.74.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23289725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Klein</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">QTL-seq identifies an early flowering QTL located near <em>flowering locus T</em> in cucumber</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">127</div>, <div class="fpage">1491</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL-seq%20identifies%20an%20early%20flowering%20QTL%20located%20near%20flowering%20locus%20T%20in%20cucumber&amp;author=H.%20Lu&amp;author=T.%20Lin&amp;author=J.%20Klein&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=127&amp;pages=1491-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-014-2313-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-014-2313-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-014-2313-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24845123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Das</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Deploying QTL-seq for rapid delineation of a potential candidate gene underlying major trait-associated QTL in chickpea</div>, <div class="source">DNA Res.</div> , <div class="volume">22</div>, <div class="fpage">193</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deploying%20QTL-seq%20for%20rapid%20delineation%20of%20a%20potential%20candidate%20gene%20underlying%20major%20trait-associated%20QTL%20in%20chickpea&amp;author=S.%20Das&amp;author=H.D.%20Upadhyaya&amp;author=D.%20Bajaj&amp;publication_year=2015&amp;journal=DNA%20Res.&amp;volume=22&amp;pages=193-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25922536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Illa-Berenguer</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Van Houten</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">van der Knaap</div> <div class="given-names">E.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Rapid and reliable identification of tomato fruit weight and locule number loci by QTL-seq</div>, <div class="source">Theor. App. Genet.</div> , <div class="volume">128</div>, <div class="fpage">1329</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20reliable%20identification%20of%20tomato%20fruit%20weight%20and%20locule%20number%20loci%20by%20QTL-seq&amp;author=E.%20Illa-Berenguer&amp;author=J.%20Van%20Houten&amp;author=Z.%20Huang&amp;author=E.%20van%20der%20Knaap&amp;publication_year=2015&amp;journal=Theor.%20App.%20Genet.&amp;volume=128&amp;pages=1329-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-015-2509-x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-015-2509-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-015-2509-x"> </span></div> </div></div></div></div></div><div content-id="dsv036c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Draft genome sequence of chickpea (<em>Cicer arietinum</em>) provides a resource for trait improvement</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">31</div>, <div class="fpage">240</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.K.%20Varshney&amp;author=C.%20Song&amp;author=R.K.%20Saxena&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol.&amp;volume=31&amp;pages=240-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2491" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23354103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Abe</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genome sequencing reveals agronomically important loci in rice using MutMap</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">174</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20reveals%20agronomically%20important%20loci%20in%20rice%20using%20MutMap&amp;author=A.%20Abe&amp;author=S.%20Kosugi&amp;author=K.%20Yoshida&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=174-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2095" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22267009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kujur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Employing genome-wide SNP discovery and genotyping strategy to extrapolate the natural allelic diversity and domestication patterns in chickpea</div>, <div class="source">Front. Plant. Sci.</div> , <div class="volume">6</div>, <div class="fpage">162</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Employing%20genome-wide%20SNP%20discovery%20and%20genotyping%20strategy%20to%20extrapolate%20the%20natural%20allelic%20diversity%20and%20domestication%20patterns%20in%20chickpea&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=H.D.%20Upadhyaya&amp;publication_year=2015&amp;journal=Front.%20Plant.%20Sci.&amp;volume=6&amp;pages=162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2015.00162" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2015.00162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25873920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kang</div>
               <div class="given-names">H.M.</div>
            </div>, <div class="name string-name"><div class="surname">Zaitlen</div> <div class="given-names">N.A.</div></div>, <div class="name string-name"><div class="surname">Wade</div> <div class="given-names">C.M.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Efficient control of population structure in model organism association mapping</div>, <div class="source">Genetics</div> , <div class="volume">178</div>, <div class="fpage">1709</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20control%20of%20population%20structure%20in%20model%20organism%20association%20mapping&amp;author=H.M.%20Kang&amp;author=N.A.%20Zaitlen&amp;author=C.M.%20Wade&amp;publication_year=2008&amp;journal=Genetics&amp;volume=178&amp;pages=1709-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.080101" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.080101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.080101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18385116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Amitha Mithra</div> <div class="given-names">S.V.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genome-wide association mapping of salinity tolerance in rice (<em>Oryza sativa</em>)</div>, <div class="source">DNA Res.</div> , <div class="volume">22</div>, <div class="fpage">133</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20of%20salinity%20tolerance%20in%20rice%20(Oryza%20sativa)&amp;author=V.%20Kumar&amp;author=A.%20Singh&amp;author=S.V.%20Amitha%20Mithra&amp;publication_year=2015&amp;journal=DNA%20Res.&amp;volume=22&amp;pages=133-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu046" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25627243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ruperao</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Chan</div> <div class="given-names">C.K.</div></div>, <div class="name string-name"><div class="surname">Azam</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">A chromosomal genomics approach to assess and validate the <em>desi</em> and <em>kabuli</em> draft chickpea genome assemblies</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">12</div>, <div class="fpage">778</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chromosomal%20genomics%20approach%20to%20assess%20and%20validate%20the%20desi%20and%20kabuli%20draft%20chickpea%20genome%20assemblies&amp;author=P.%20Ruperao&amp;author=C.K.%20Chan&amp;author=S.%20Azam&amp;publication_year=2014&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=12&amp;pages=778-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2014.12.issue-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2014.12.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2014.12.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24702794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cushing</div>
               <div class="given-names">D.A.</div>
            </div>, <div class="name string-name"><div class="surname">Forsthoefel</div> <div class="given-names">N.R.</div></div>, <div class="name string-name"><div class="surname">Gestaut</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Vernon</div> <div class="given-names">D.M.</div></div></div>
          <div class="year">2005</div>, <div class="article-title"><em>Arabidopsis</em> emb175 and other ppr knockout mutants reveal essential roles for pentatricopeptide repeat (PPR) proteins in plant embryogenesis</div>, <div class="source">Planta</div> , <div class="volume">221</div>, <div class="fpage">424</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20emb175%20and%20other%20ppr%20knockout%20mutants%20reveal%20essential%20roles%20for%20pentatricopeptide%20repeat%20(PPR)%20proteins%20in%20plant%20embryogenesis&amp;author=D.A.%20Cushing&amp;author=N.R.%20Forsthoefel&amp;author=D.R.%20Gestaut&amp;author=D.M.%20Vernon&amp;publication_year=2005&amp;journal=Planta&amp;volume=221&amp;pages=424-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-004-1452-x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-004-1452-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-004-1452-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15647901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Abid</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Sassi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Muhovski</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Identification and analysis of differentially expressed genes during seed development using suppression subtractive hybridization (SSH) in <em>Phaseolus vulgaris</em></div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="volume">30</div>, <div class="fpage">719</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20differentially%20expressed%20genes%20during%20seed%20development%20using%20suppression%20subtractive%20hybridization%20(SSH)%20in%20Phaseolus%20vulgaris&amp;author=G.%20Abid&amp;author=K.%20Sassi&amp;author=Y.%20Muhovski&amp;publication_year=2012&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=30&amp;pages=719-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-011-0381-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-011-0381-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-011-0381-7"> </span></div> </div></div></div></div></div><div content-id="dsv036c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bentolila</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Alfonso</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Hanson</div> <div class="given-names">M.R.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">A pentatricopeptide repeat-containing gene restores fertility to cytoplasmic male-sterile plants</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">99</div>, <div class="fpage">10887</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20pentatricopeptide%20repeat-containing%20gene%20restores%20fertility%20to%20cytoplasmic%20male-sterile%20plants&amp;author=S.%20Bentolila&amp;author=A.A.%20Alfonso&amp;author=M.R.%20Hanson&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=10887-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.102301599" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.102301599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.102301599"> </span></div> </div></div></div></div></div><div content-id="dsv036c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kazama</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Toriyama</div> <div class="given-names">K.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">A pentatricopeptide repeat-containing gene that promotes the processing of aberrant atp6 RNA of cytoplasmic male-sterile rice</div>, <div class="source">FEBS Lett.</div> , <div class="volume">544</div>, <div class="fpage">99</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20pentatricopeptide%20repeat-containing%20gene%20that%20promotes%20the%20processing%20of%20aberrant%20atp6%20RNA%20of%20cytoplasmic%20male-sterile%20rice&amp;author=T.%20Kazama&amp;author=K.%20Toriyama&amp;publication_year=2003&amp;journal=FEBS%20Lett.&amp;volume=544&amp;pages=99-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(03)00480-0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(03)00480-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(03)00480-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12782297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Koizuka</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Imai</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Genetic characterization of a pentatricopeptide repeat protein gene, <em>orf687</em>, that restores fertility in the cytoplasmic male-sterile Kosena radish</div>, <div class="source">Plant J.</div> , <div class="volume">34</div>, <div class="fpage">407</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20characterization%20of%20a%20pentatricopeptide%20repeat%20protein%20gene%2C%20orf687%2C%20that%20restores%20fertility%20in%20the%20cytoplasmic%20male-sterile%20Kosena%20radish&amp;author=N.%20Koizuka&amp;author=R.%20Imai&amp;author=H.%20Fujimoto&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=34&amp;pages=407-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01735.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01735.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01735.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12753581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saha</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Prasad</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Srinivasan</div> <div class="given-names">R.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Pentatricopeptide repeat proteins and their emerging roles in plants</div>, <div class="source">Plant Physiol. Biochem.</div> , <div class="volume">45</div>, <div class="fpage">521</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pentatricopeptide%20repeat%20proteins%20and%20their%20emerging%20roles%20in%20plants&amp;author=D.%20Saha&amp;author=A.M.%20Prasad&amp;author=R.%20Srinivasan&amp;publication_year=2007&amp;journal=Plant%20Physiol.%20Biochem.&amp;volume=45&amp;pages=521-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plaphy.2007.03.026" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plaphy.2007.03.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plaphy.2007.03.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17560114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmitz-Linneweber</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Small</div> <div class="given-names">I.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Pentatricopeptide repeat proteins: a socket set for organelle gene expression</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">13</div>, <div class="fpage">663</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pentatricopeptide%20repeat%20proteins%3A%20a%20socket%20set%20for%20organelle%20gene%20expression&amp;author=C.%20Schmitz-Linneweber&amp;author=I.%20Small&amp;publication_year=2008&amp;journal=Trends%20Plant%20Sci.&amp;volume=13&amp;pages=663-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2008.10.001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2008.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2008.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19004664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ngangkham</div>
               <div class="given-names">U.</div>
            </div>, <div class="name string-name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Dey</div> <div class="given-names">S.K.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Genic markers for WA cytoplasm based male sterility and its fertility restoration in rice</div>, <div class="source">Mol. Breed.</div> , <div class="volume">26</div>, <div class="fpage">275</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20markers%20for%20WA%20cytoplasm%20based%20male%20sterility%20and%20its%20fertility%20restoration%20in%20rice&amp;author=U.%20Ngangkham&amp;author=S.K.%20Parida&amp;author=S.K.%20Dey&amp;publication_year=2010&amp;journal=Mol.%20Breed.&amp;volume=26&amp;pages=275-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9397-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9397-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9397-1"> </span></div> </div></div></div></div></div><div content-id="dsv036c60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Su</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">M.L.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">D.X.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Disruption of a rice pentatricopeptide repeat protein causes a seedling-specific albino phenotype and its utilization to enhance seed purity in hybrid rice production</div>, <div class="source">Plant Physiol.</div> , <div class="volume">159</div>, <div class="fpage">227</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disruption%20of%20a%20rice%20pentatricopeptide%20repeat%20protein%20causes%20a%20seedling-specific%20albino%20phenotype%20and%20its%20utilization%20to%20enhance%20seed%20purity%20in%20hybrid%20rice%20production&amp;author=N.%20Su&amp;author=M.L.%20Hu&amp;author=D.X.%20Wu&amp;publication_year=2012&amp;journal=Plant%20Physiol.&amp;volume=159&amp;pages=227-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.195081" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.195081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.195081"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22430843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barkan</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Small</div> <div class="given-names">I.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Pentatricopeptide repeat proteins in plants</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="volume">65</div>, <div class="fpage">415</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pentatricopeptide%20repeat%20proteins%20in%20plants&amp;author=A.%20Barkan&amp;author=I.%20Small&amp;publication_year=2014&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=65&amp;pages=415-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-050213-040159" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-050213-040159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-050213-040159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24471833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsv036c62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsv036c62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Manna</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2015</div>, <div class="article-title">An overview of pentatricopeptide repeat proteins and their applications</div>, <div class="source">Biochimie</div> , <div class="volume">113</div>, <div class="fpage">93</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20overview%20of%20pentatricopeptide%20repeat%20proteins%20and%20their%20applications&amp;author=S.%20Manna&amp;publication_year=2015&amp;journal=Biochimie&amp;volume=113&amp;pages=93-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biochi.2015.04.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biochi.2015.04.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biochi.2015.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25882680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462457&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv036/1/dsv036_Supplementary_Data.zip?Expires=1509239038&amp;Signature=crw4StCTUBTYSvVfNGGUqOtO8LunTkJoJeFjg2WJSxw3qY7pNjZB7dJd8sVwSn41aYzM2PjTEWs~RGQCYiz1lnFWcQu8~LFQqcy78PPH4jysTTFQL2qQzTvORXJR7FgpA5FQfH8benVBg0nr3RTeeIYi5NJh4Ka5O4LAszSzqmUft2LguHb51XKj5vxk55Wb93tb8~Zp~dmS1ryboK~5b913axoezD78YXz1K1hcEJNb-AwWliAqj1-3hw9mwGgUX2kpJy7aP1r0-BE~JT9boWWLO6jV668IEI2XoPpyKMazosKGMpJWGlvv3ZUYlQ1YH~b7DNjWeYOPKSBC8Yhc2Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">131</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">9</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsv036"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462457"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f1%2f53%2f2462457">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000370973100006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (9)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F1%2F53%2F2462457" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462457'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="mQTL-seq delineates functionally relevant candidate gene harbouring a major QTL regulating pod number in chickpea" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f53%2f2462457">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

