

<!DOCTYPE html>
<html lang="en">

<head><title>
	Draft genome sequence of Cicer reticulatum L., the wild progenitor of chickpea provides a resource for agronomic trait improvement | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/1/2669389","@type":"ScholarlyArticle","name":"Draft genome sequence of  Cicer reticulatum  L., the wild progenitor of chickpea provides a resource for agronomic trait improvement","datePublished":"2016-08-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw042","keywords":["<italic>Cicer reticulatum</italic> L. PI489777","wild chickpea","genome sequence","annotation","diversity"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/1/2669389","author":[{"name":"Gupta, Sonal","affiliation":"National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi 110067, India","@type":"Person"},{"name":"Nawaz, Kashif","affiliation":"National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi 110067, India","@type":"Person"},{"name":"Parween, Sabiha","@type":"Person"},{"name":"Roy, Riti","@type":"Person"},{"name":"Sahu, Kamlesh","@type":"Person"},{"name":"Kumar Pole, Anil","@type":"Person"},{"name":"Khandal, Hitaishi","@type":"Person"},{"name":"Srivastava, Rishi","@type":"Person"},{"name":"Kumar Parida, Swarup","@type":"Person"},{"name":"Chattopadhyay, Debasis","@type":"Person"}],"description":"Cicer reticulatum L. is the wild progenitor of the fourth most important legume crop chickpea (C. arietinum L.). We assembled short-read sequences into 416 Mb draft genome of C. reticulatum and anchored 78% (327 Mb) of this assembly to eight linkage groups. Genome annotation predicted 25,680 protein-coding genes covering more than 90% of predicted gene space. The genome assembly shared a substantial synteny and conservation of gene orders with the genome of the model legume Medicago truncatula. Resistance gene homologs of wild and domesticated chickpeas showed high sequence homology and conserved synteny. Comparison of gene sequences and nucleotide diversity using 66 wild and domesticated chickpea accessions suggested that the desi type chickpea was genetically closer to the wild species than the kabuli type. Comparative analyses predicted gene flow between the wild and the cultivated species during domestication. Molecular diversity and population genetic structure determination using 15,096 genome-wide single nucleotide polymorphisms revealed an admixed domestication pattern among cultivated (desi and kabuli) and wild chickpea accessions belonging to three population groups reflecting significant influence of parentage or geographical origin for their cultivar-specific population classification. The assembly and the polymorphic sequence resources presented here would facilitate the study of chickpea domestication and targeted use of wild Cicer germplasms for agronomic trait improvement in chickpea.","pageStart":"1","pageEnd":"10","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217474&Signature=cpNhkC15jim8nlRTSeR5fJslMPxH-7sfPV95-h9~P9OzftaZS8EDV7EjUzNK2IxXsNOnpiOMEKdUx1YmrulEisdSQpvKVbN2GM35HvMIiWcUDk59fx-2i-yuBf8ZbENQOntO-NRSs9rGPndMFUg7pPAuT0nrGBT0Z1mdT-CdJ19GbsQXJSZl42Cg7f9VS4hthrNPJ0Tx-dFZgZPEypgHzJFopKNiBnMgjVdr5Zugdxu~pB~CQZxWsAK8CpnjqNTloshjMQivPG-gfZCFMdTUW2CvnJY8KgnuGChTrjPKYMQjx0wqu~8K57Yq-ub~i~-XFbvmV73oA7RgA4zYQyF80Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Draft genome sequence of  Cicer reticulatum  L., the wild progenitor of chickpea provides a resource for agronomic trait improvement","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217474&Signature=cpNhkC15jim8nlRTSeR5fJslMPxH-7sfPV95-h9~P9OzftaZS8EDV7EjUzNK2IxXsNOnpiOMEKdUx1YmrulEisdSQpvKVbN2GM35HvMIiWcUDk59fx-2i-yuBf8ZbENQOntO-NRSs9rGPndMFUg7pPAuT0nrGBT0Z1mdT-CdJ19GbsQXJSZl42Cg7f9VS4hthrNPJ0Tx-dFZgZPEypgHzJFopKNiBnMgjVdr5Zugdxu~pB~CQZxWsAK8CpnjqNTloshjMQivPG-gfZCFMdTUW2CvnJY8KgnuGChTrjPKYMQjx0wqu~8K57Yq-ub~i~-XFbvmV73oA7RgA4zYQyF80Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Draft genome sequence of  Cicer reticulatum  L., the wild progenitor of chickpea provides a resource for agronomic trait improvement | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Cicer reticulatum L. is the wild progenitor of the fourth most important legume crop chickpea (C. arietinum L.). We assembled short-read sequences into 416 Mb draft genome of C. reticulatum and anchored 78% (327 Mb) of this assembly to eight linkage groups. Genome annotation predicted 25,680 protein-coding genes covering more than 90% of predicted gene space. The genome assembly shared a substantial synteny and conservation of gene orders with the genome of the model legume Medicago truncatula. Resistance gene homologs of wild and domesticated chickpeas showed high sequence homology and conserved synteny. Comparison of gene sequences and nucleotide diversity using 66 wild and domesticated chickpea accessions suggested that the desi type chickpea was genetically closer to the wild species than the kabuli type. Comparative analyses predicted gene flow between the wild and the cultivated species during domestication. Molecular diversity and population genetic structure determination using 15,096 genome-wide single nucleotide polymorphisms revealed an admixed domestication pattern among cultivated (desi and kabuli) and wild chickpea accessions belonging to three population groups reflecting significant influence of parentage or geographical origin for their cultivar-specific population classification. The assembly and the polymorphic sequence resources presented here would facilitate the study of chickpea domestication and targeted use of wild Cicer germplasms for agronomic trait improvement in chickpea." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/1/2669389" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217474&amp;Signature=IodWGbH69Td6dutByxS9HZGrJMqVC34MA9rmCFh2lKvtybRTLfcQMEHF4SxC0LUXG-WbUlq6bu7cPcW91AEGQ4ybAFVr1Q0tSfqc6ahqSl0Mdvfn7ZaUy5EMKoYKcCLHt~o~1inLBkwZnQuYFIBWOs0kHZiJ4I8zgUlUlyPMIGhYqI9-vjXqdWmLJz496g2pbdB4BZm-sAT4w8Es5zASVv9lOG8ClYNkQo6T8Oz1VZnKvgn-bSpiEMJTfGCeOP6xcjBDC2xCexhGybRPw6RLQtc9TcWh6GaTLk7Vv9rgZn0WbiUATG5hpwobPxTd3gsNVsF9rPpTM-Y0XC2USe053g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2016-08-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Gupta, Sonal" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Nawaz, Kashif" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Parween, Sabiha" /><meta name="citation_author" content="Roy, Riti" /><meta name="citation_author" content="Sahu, Kamlesh" /><meta name="citation_author" content="Kumar Pole, Anil" /><meta name="citation_author" content="Khandal, Hitaishi" /><meta name="citation_author" content="Srivastava, Rishi" /><meta name="citation_author" content="Kumar Parida, Swarup" /><meta name="citation_author" content="Chattopadhyay, Debasis" /><meta name="citation_title" content="Draft genome sequence of Cicer reticulatum L., the wild progenitor of chickpea provides a resource for agronomic trait improvement" /><meta name="citation_firstpage" content="1" /><meta name="citation_lastpage" content="10" /><meta name="citation_doi" content="10.1093/dnares/dsw042" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Draft genome sequence of chickpea (Cicer arietinum) provides a resource for trait improvement,;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=240-246; " /><meta name="citation_reference" content="citation_title=A draft genome sequence of the pulse crop chickpea (Cicer arietinum L.);  citation_journal_title=Plant J;  citation_year=2013;  citation_volume=74;  citation_pages=715-729; " /><meta name="citation_reference" content="citation_title=An advanced draft genome assembly of a desi type chickpea (Cicer arietinum L.),;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=12806" /><meta name="citation_reference" content="citation_title=Evolution of cultivated chickpea: four bottlenecks limit diversity and constrain adaptation;  citation_journal_title=Funct. Plant Biol;  citation_year=2003;  citation_volume=30;  citation_pages=1081-1087; " /><meta name="citation_reference" content="citation_title=Agricultural origins: centers and noncenters,;  citation_journal_title=Science;  citation_year=1971;  citation_volume=174;  citation_pages=468-474; " /><meta name="citation_reference" content="citation_title=The origin of chickpea Cicer arietinum L;  citation_journal_title=Euphytica;  citation_year=1976;  citation_volume=25;  citation_pages=211-217; " /><meta name="citation_reference" content="citation_title=The chickpea, summer cropping, and a new model for pulse domestication in the ancient near East,;  citation_journal_title=Quant. Rev. Biol;  citation_year=2003;  citation_volume=78;  citation_pages=435-448; " /><meta name="citation_reference" content="citation_publisher=CAB International, Wallingford;  citation_title=Chickpea Breeding and Management;  citation_year=2007;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Improving plant breeding with exotic genetic libraries,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2001;  citation_volume=2;  citation_pages=983-989; " /><meta name="citation_reference" content="citation_title=Genome sequence of the polymyxin-producing plant-probiotic rhizobacterium Paenibacillus polymyxa E681,;  citation_journal_title=J. Bacteriol;  citation_year=2010;  citation_volume=192;  citation_pages=6103-6104; " /><meta name="citation_reference" content="citation_title=The genome of the stress-tolerant wild tomato species Solanum pennellii,;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=1034-1038; " /><meta name="citation_reference" content="citation_title=Characterization and mapping of sequence-tagged microsatellite sites in the chickpea (Cicer arietinum L.) genome;  citation_journal_title=Mol. Gen Genet;  citation_year=1999;  citation_volume=262;  citation_pages=90-101; " /><meta name="citation_reference" content="citation_title=Genetic transformation of crops for insect resistance: potential and limitations,;  citation_journal_title=Crit. Rev. Plant Sci;  citation_year=2004;  citation_volume=23;  citation_pages=47-72; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (Cicer arietinum L.),;  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=357-373; " /><meta name="citation_reference" content="citation_title=Development and integration of genome-wide polymorphic microsatellite markers onto a reference linkage map for constructing a high-density genetic map of chickpea,;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0125583" /><meta name="citation_reference" content="citation_title=NGS QC Toolkit: a toolkit for quality control of next generation sequencing data,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30619" /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,;  citation_journal_title=GigaScience;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=The MaSuRCA genome assembler;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=2669-2677; " /><meta name="citation_reference" content="citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=1513-1518; " /><meta name="citation_reference" content="citation_title=High density linkage mapping of genomic and transcriptomic SNPs for synteny analysis and anchoring the genome sequence of chickpea,;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=13387" /><meta name="citation_reference" content="citation_title=BLAT-the BLAST-like alignment tool,;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-664; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-1760; " /><meta name="citation_reference" content="citation_title=Using RepeatMasker to identify repetitive elements in genomic sequences;  citation_journal_title=Curr. Protoc. Bioinform;  citation_year=2004;  citation_volume=25;  citation_pages=4.10.1-4.10.14; " /><meta name="citation_reference" content="citation_title=PILER: identification and classification of genomic repeats,;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=52-58; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes,;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=351-358; " /><meta name="citation_reference" content="citation_title=LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons,;  citation_journal_title=Nucl. Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=265-268; " /><meta name="citation_reference" content="citation_title=Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis,;  citation_journal_title=BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=327" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of wild chickpea as a rich resource for marker development,;  citation_journal_title=Plant Biotechnol. J;  citation_year=2012;  citation_volume=10;  citation_pages=690-702; " /><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments,;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7" /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics,;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1639-1645; " /><meta name="citation_reference" content="citation_title=An efficient algorithm for large-scale detection of protein families,;  citation_journal_title=Nucl. Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=1575-1584; " /><meta name="citation_reference" content="citation_title=SyMAP v3.4: a turnkey synteny system with application to plant genomes,;  citation_journal_title=Nucl. Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=e68" /><meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity,;  citation_journal_title=Nucl. Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49" /><meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs,;  citation_journal_title=Nucl. Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=3497-3500; " /><meta name="citation_reference" content="citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments;  citation_journal_title=Nucl. Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W609-W612; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-1591; " /><meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes,;  citation_journal_title=Science;  citation_year=2000;  citation_volume=290;  citation_pages=1151-1155; " /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping loci de novo from short-read sequences,;  citation_journal_title=Genes Genomes Genet;  citation_year=2011;  citation_volume=1;  citation_pages=171-182; " /><meta name="citation_reference" content="citation_title=Employing genome-wide SNP discovery and genotyping strategy to extrapolate the natural allelic diversity and domestication patterns in chickpea,;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=162" /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-770; " /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-1067; " /><meta name="citation_reference" content="citation_title=The NB-LRR proteins RGA4 and RGA5 interact functionally and physically to confer disease resistance,;  citation_journal_title=EMBO J;  citation_year=2014;  citation_volume=33;  citation_pages=1941-1959; " /><meta name="citation_reference" content="citation_title=Pseudomonas syringae pv. maculicola in Australia: pathogenic, phenotypic and genetic diversity,;  citation_journal_title=Plant Pathol;  citation_year=2004;  citation_volume=53;  citation_pages=73-79; " /><meta name="citation_reference" content="citation_title=The product of the tobacco mosaic virus resistance gene N: similarity to toll and the interleukin-1 receptor,;  citation_journal_title=Cell;  citation_year=1994;  citation_volume=78;  citation_pages=1101-1115; " /><meta name="citation_reference" content="citation_title=The powdery mildew resistance protein RPW8.2 is carried on VAMP721/722 vesicles to the extrahaustorial membrane of haustorial complexes,;  citation_journal_title=Plant J;  citation_year=2014;  citation_volume=79;  citation_pages=835-847; " /><meta name="citation_reference" content="citation_title=Comparative analysis of kabuli chickpea transcriptome with desi and wild chickpea provides a rich resource for development of functional markers,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e52443" /><meta name="citation_reference" content="citation_title=Vernalization response of wild chickpea,;  citation_journal_title=New Phytol;  citation_year=2002;  citation_volume=154;  citation_pages=695-701; " /><meta name="citation_reference" content="citation_title=Vernalization in Arabidopsis thaliana is mediated by the PHD finger protein VIN3,;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=427;  citation_pages=159-164; " /><meta name="citation_reference" content="citation_title=The VERNALIZATION 2 gene mediates the epigenetic regulation of vernalization in Arabidopsis,;  citation_journal_title=Cell;  citation_year=2001;  citation_volume=107;  citation_pages=525-535; " /><meta name="citation_reference" content="citation_title=Variation in Cicer arietinum L.,;  citation_journal_title=Euphytica;  citation_year=1978;  citation_volume=27;  citation_pages=465-485; " /><meta name="citation_reference" content="citation_title=Ecogeography of annual wild Cicer species: the parlous state of the world collection,;  citation_journal_title=Crop Sci;  citation_year=2003;  citation_volume=43;  citation_pages=1076" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/1/10731765/dsw042.pdf" /><meta name="description" content="Abstract.  Cicer reticulatum L. is the wild progenitor of the fourth most important legume crop chickpea (C. arietinum L.). We assembled short-read sequences in" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/1/2669389" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/1/2669389" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsw042';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2669389" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFmHI8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1cHRhLCBTb25hbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2gsIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmF3YXosIEthc2hpZiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2gsIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyd2VlbiwgU2FiaWhhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm95LCBSaXRpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FodSwgS2FtbGVzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1bWFyIFBvbGUsIEFuaWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaGFuZGFsLCBIaXRhaXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNyaXZhc3RhdmEsIFJpc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VtYXIgUGFyaWRhLCBTd2FydXAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGF0dG9wYWRoeWF5LCBEZWJhc2lzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgQ2ljZXIgcmV0aWN1bGF0dW0gTC4sIHRoZSB3aWxkIHByb2dlbml0b3Igb2YgY2hpY2twZWEgcHJvdmlkZXMgYSByZXNvdXJjZSBmb3IgYWdyb25vbWljIHRyYWl0IGltcHJvdmVtZW50IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtKSBwcm92aWRlcyBhIHJlc291cmNlIGZvciB0cmFpdCBpbXByb3ZlbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI0MC0yNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBwdWxzZSBjcm9wIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9NzE1LTcyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWR2YW5jZWQgZHJhZnQgZ2Vub21lIGFzc2VtYmx5IG9mIGEgZGVzaSB0eXBlIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEyODA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGN1bHRpdmF0ZWQgY2hpY2twZWE6IGZvdXIgYm90dGxlbmVja3MgbGltaXQgZGl2ZXJzaXR5IGFuZCBjb25zdHJhaW4gYWRhcHRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuY3QuIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMDgxLTEwODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFncmljdWx0dXJhbCBvcmlnaW5zOiBjZW50ZXJzIGFuZCBub25jZW50ZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk3MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzQ7ICBjaXRhdGlvbl9wYWdlcz00NjgtNDc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIG9mIGNoaWNrcGVhIENpY2VyIGFyaWV0aW51bSBMOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5NzY7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMTEtMjE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY2hpY2twZWEsIHN1bW1lciBjcm9wcGluZywgYW5kIGEgbmV3IG1vZGVsIGZvciBwdWxzZSBkb21lc3RpY2F0aW9uIGluIHRoZSBhbmNpZW50IG5lYXIgRWFzdCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVF1YW50LiBSZXYuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz00MzUtNDQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q0FCIEludGVybmF0aW9uYWwsIFdhbGxpbmdmb3JkOyAgY2l0YXRpb25fdGl0bGU9Q2hpY2twZWEgQnJlZWRpbmcgYW5kIE1hbmFnZW1lbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgcGxhbnQgYnJlZWRpbmcgd2l0aCBleG90aWMgZ2VuZXRpYyBsaWJyYXJpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTk4My05ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcG9seW15eGluLXByb2R1Y2luZyBwbGFudC1wcm9iaW90aWMgcmhpem9iYWN0ZXJpdW0gUGFlbmliYWNpbGx1cyBwb2x5bXl4YSBFNjgxLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE5MjsgIGNpdGF0aW9uX3BhZ2VzPTYxMDMtNjEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgc3RyZXNzLXRvbGVyYW50IHdpbGQgdG9tYXRvIHNwZWNpZXMgU29sYW51bSBwZW5uZWxsaWksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTAzNC0xMDM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIGFuZCBtYXBwaW5nIG9mIHNlcXVlbmNlLXRhZ2dlZCBtaWNyb3NhdGVsbGl0ZSBzaXRlcyBpbiB0aGUgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLikgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjI7ICBjaXRhdGlvbl9wYWdlcz05MC0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdHJhbnNmb3JtYXRpb24gb2YgY3JvcHMgZm9yIGluc2VjdCByZXNpc3RhbmNlOiBwb3RlbnRpYWwgYW5kIGxpbWl0YXRpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JpdC4gUmV2LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz00Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgZm9yIGNvbnN0cnVjdGluZyBhIHNhdHVyYXRlZCBsaW5rYWdlIG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzU3LTM3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGludGVncmF0aW9uIG9mIGdlbm9tZS13aWRlIHBvbHltb3JwaGljIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgb250byBhIHJlZmVyZW5jZSBsaW5rYWdlIG1hcCBmb3IgY29uc3RydWN0aW5nIGEgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwIG9mIGNoaWNrcGVhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDEyNTU4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5HUyBRQyBUb29sa2l0OiBhIHRvb2xraXQgZm9yIHF1YWxpdHkgY29udHJvbCBvZiBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMDYxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNPQVBkZW5vdm8yOiBhbiBlbXBpcmljYWxseSBpbXByb3ZlZCBtZW1vcnktZWZmaWNpZW50IHNob3J0LXJlYWQgZGUgbm92byBhc3NlbWJsZXIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HaWdhU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTWFTdVJDQSBnZW5vbWUgYXNzZW1ibGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI2NjktMjY3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1xdWFsaXR5IGRyYWZ0IGFzc2VtYmxpZXMgb2YgbWFtbWFsaWFuIGdlbm9tZXMgZnJvbSBtYXNzaXZlbHkgcGFyYWxsZWwgc2VxdWVuY2UgZGF0YSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTE1MTMtMTUxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCBkZW5zaXR5IGxpbmthZ2UgbWFwcGluZyBvZiBnZW5vbWljIGFuZCB0cmFuc2NyaXB0b21pYyBTTlBzIGZvciBzeW50ZW55IGFuYWx5c2lzIGFuZCBhbmNob3JpbmcgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBjaGlja3BlYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMzM4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVQtdGhlIEJMQVNULWxpa2UgYWxpZ25tZW50IHRvb2wsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjU2LTY2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTE3NjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIFJlcGVhdE1hc2tlciB0byBpZGVudGlmeSByZXBldGl0aXZlIGVsZW1lbnRzIGluIGdlbm9taWMgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBQcm90b2MuIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTQuMTAuMS00LjEwLjE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QSUxFUjogaWRlbnRpZmljYXRpb24gYW5kIGNsYXNzaWZpY2F0aW9uIG9mIGdlbm9taWMgcmVwZWF0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTItNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gaWRlbnRpZmljYXRpb24gb2YgcmVwZWF0IGZhbWlsaWVzIGluIGxhcmdlIGdlbm9tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM1MS0zNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxUUl9GSU5ERVI6IGFuIGVmZmljaWVudCB0b29sIGZvciB0aGUgcHJlZGljdGlvbiBvZiBmdWxsLWxlbmd0aCBMVFIgcmV0cm90cmFuc3Bvc29ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTI2NS0yNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgYW5hbHlzaXMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gcmljZSBhbmQgY29tcGFyYXRpdmUgYW5hbHlzZXMgd2l0aCBBcmFiaWRvcHNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIG9mIHdpbGQgY2hpY2twZWEgYXMgYSByaWNoIHJlc291cmNlIGZvciBtYXJrZXIgZGV2ZWxvcG1lbnQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NjkwLTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGV1a2FyeW90aWMgZ2VuZSBzdHJ1Y3R1cmUgYW5ub3RhdGlvbiB1c2luZyBFVmlkZW5jZU1vZGVsZXIgYW5kIHRoZSBwcm9ncmFtIHRvIGFzc2VtYmxlIHNwbGljZWQgYWxpZ25tZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmNvczogYW4gaW5mb3JtYXRpb24gYWVzdGhldGljIGZvciBjb21wYXJhdGl2ZSBnZW5vbWljcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjM5LTE2NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVmZmljaWVudCBhbGdvcml0aG0gZm9yIGxhcmdlLXNjYWxlIGRldGVjdGlvbiBvZiBwcm90ZWluIGZhbWlsaWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTU3NS0xNTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeU1BUCB2My40OiBhIHR1cm5rZXkgc3ludGVueSBzeXN0ZW0gd2l0aCBhcHBsaWNhdGlvbiB0byBwbGFudCBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9ZTY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUNTY2FuWDogYSB0b29sa2l0IGZvciBkZXRlY3Rpb24gYW5kIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiBnZW5lIHN5bnRlbnkgYW5kIGNvbGxpbmVhcml0eSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPWU0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIHRoZSBDbHVzdGFsIHNlcmllcyBvZiBwcm9ncmFtcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM0OTctMzUwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFMMk5BTDogcm9idXN0IGNvbnZlcnNpb24gb2YgcHJvdGVpbiBzZXF1ZW5jZSBhbGlnbm1lbnRzIGludG8gdGhlIGNvcnJlc3BvbmRpbmcgY29kb24gYWxpZ25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9VzYwOS1XNjEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQU1MIDQ6IHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBieSBtYXhpbXVtIGxpa2VsaWhvb2QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTg2LTE1OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcnkgZmF0ZSBhbmQgY29uc2VxdWVuY2VzIG9mIGR1cGxpY2F0ZSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjkwOyAgY2l0YXRpb25fcGFnZXM9MTE1MS0xMTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFja3M6IGJ1aWxkaW5nIGFuZCBnZW5vdHlwaW5nIGxvY2kgZGUgbm92byBmcm9tIHNob3J0LXJlYWQgc2VxdWVuY2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgR2Vub21lcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTcxLTE4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1wbG95aW5nIGdlbm9tZS13aWRlIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgc3RyYXRlZ3kgdG8gZXh0cmFwb2xhdGUgdGhlIG5hdHVyYWwgYWxsZWxpYyBkaXZlcnNpdHkgYW5kIGRvbWVzdGljYXRpb24gcGF0dGVybnMgaW4gY2hpY2twZWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhc3QsIGxvY2stZnJlZSBhcHByb2FjaCBmb3IgZWZmaWNpZW50IHBhcmFsbGVsIGNvdW50aW5nIG9mIG9jY3VycmVuY2VzIG9mIGstbWVycyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NzY0LTc3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0VHTUE6IGEgcGlwZWxpbmUgdG8gYWNjdXJhdGVseSBhbm5vdGF0ZSBjb3JlIGdlbmVzIGluIGV1a2FyeW90aWMgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMDYxLTEwNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBOQi1MUlIgcHJvdGVpbnMgUkdBNCBhbmQgUkdBNSBpbnRlcmFjdCBmdW5jdGlvbmFsbHkgYW5kIHBoeXNpY2FsbHkgdG8gY29uZmVyIGRpc2Vhc2UgcmVzaXN0YW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gSjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTE5NDEtMTk1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHNldWRvbW9uYXMgc3lyaW5nYWUgcHYuIG1hY3VsaWNvbGEgaW4gQXVzdHJhbGlhOiBwYXRob2dlbmljLCBwaGVub3R5cGljIGFuZCBnZW5ldGljIGRpdmVyc2l0eSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBhdGhvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTczLTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcHJvZHVjdCBvZiB0aGUgdG9iYWNjbyBtb3NhaWMgdmlydXMgcmVzaXN0YW5jZSBnZW5lIE46IHNpbWlsYXJpdHkgdG8gdG9sbCBhbmQgdGhlIGludGVybGV1a2luLTEgcmVjZXB0b3IsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9MTEwMS0xMTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcG93ZGVyeSBtaWxkZXcgcmVzaXN0YW5jZSBwcm90ZWluIFJQVzguMiBpcyBjYXJyaWVkIG9uIFZBTVA3MjEvNzIyIHZlc2ljbGVzIHRvIHRoZSBleHRyYWhhdXN0b3JpYWwgbWVtYnJhbmUgb2YgaGF1c3RvcmlhbCBjb21wbGV4ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9ODM1LTg0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2Yga2FidWxpIGNoaWNrcGVhIHRyYW5zY3JpcHRvbWUgd2l0aCBkZXNpIGFuZCB3aWxkIGNoaWNrcGVhIHByb3ZpZGVzIGEgcmljaCByZXNvdXJjZSBmb3IgZGV2ZWxvcG1lbnQgb2YgZnVuY3Rpb25hbCBtYXJrZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MjQ0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlcm5hbGl6YXRpb24gcmVzcG9uc2Ugb2Ygd2lsZCBjaGlja3BlYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9Njk1LTcwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVybmFsaXphdGlvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBpcyBtZWRpYXRlZCBieSB0aGUgUEhEIGZpbmdlciBwcm90ZWluIFZJTjMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDI3OyAgY2l0YXRpb25fcGFnZXM9MTU5LTE2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFZFUk5BTElaQVRJT04gMiBnZW5lIG1lZGlhdGVzIHRoZSBlcGlnZW5ldGljIHJlZ3VsYXRpb24gb2YgdmVybmFsaXphdGlvbiBpbiBBcmFiaWRvcHNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NTI1LTUzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWF0aW9uIGluIENpY2VyIGFyaWV0aW51bSBMLiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MTk3ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTQ2NS00ODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVjb2dlb2dyYXBoeSBvZiBhbm51YWwgd2lsZCBDaWNlciBzcGVjaWVzOiB0aGUgcGFybG91cyBzdGF0ZSBvZiB0aGUgd29ybGQgY29sbGVjdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTA3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8xLzEvMTA3MzE3NjUvZHN3MDQyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBDaWNlciByZXRpY3VsYXR1bSBMLiBpcyB0aGUgd2lsZCBwcm9nZW5pdG9yIG9mIHRoZSBmb3VydGggbW9zdCBpbXBvcnRhbnQgbGVndW1lIGNyb3AgY2hpY2twZWEgKEMuIGFyaWV0aW51bSBMLikuIFdlIGFzc2VtYmxlZCBzaG9ydC1yZWFkIHNlcXVlbmNlcyBpbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjY2OTM4OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGSP0/PzVnMMYNWYazEXnIdbkww9gA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f1%2f2669389" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f1%2f2669389" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1509239074&amp;Signature=O1ktQDiYhTUgGieNSsOKPFybP-S7tYfAswt0bxoH45Hs6KtTkwlKTIXj7HtsfpwtW5e89EUxGVRDuHYaSI-h8UuAjXonrh-WQGVTkq0-SyHw8EOYc3ayhn8zOXCJ4JWadZuNdTbXX0tz6vVk~RAdSlY9zN~MAH3SGv4wudm8MtEzo~KKEeOE35da95PawCUvVw6GMnE4B-W9KG3eZeHDMHj5nuklo4~KK0DhCtLGliZufHwOx-QXfjSOLKc5wQK7R36lythtym~27E~WnuYueyABkyTN0xQM~YXUdiX4lAIycK~rBMrPbJLluWASx4e046RJ0oDybOV2a5Fhki6FSA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61462649">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462649">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462651">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462651">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462654">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462654">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462666">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462666">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462681">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462681">DISCUSSION</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462684">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462684">ACKNOWLEDGEMENTS</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462694">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462694">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/11/2670108">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Draft genome sequence of <em>Cicer reticulatum</em> L., the wild progenitor of chickpea provides a resource for agronomic trait improvement
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Sonal Gupta</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sonal Gupta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi 110067, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Sonal+Gupta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gupta S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Gupta author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Kashif Nawaz</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kashif Nawaz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi 110067, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kashif+Nawaz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nawaz K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nawaz author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Sabiha Parween</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sabiha Parween</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Sabiha+Parween">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parween S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Parween author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Riti Roy</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Riti Roy</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Riti+Roy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Roy R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Roy author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Kamlesh Sahu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kamlesh Sahu</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kamlesh+Sahu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sahu K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Sahu author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Anil Kumar Pole</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anil Kumar Pole</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Anil+Kumar+Pole">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar Pole A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kumar Pole author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Hitaishi Khandal</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hitaishi Khandal</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hitaishi+Khandal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Khandal H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Khandal author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Rishi Srivastava</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rishi Srivastava</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Rishi+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Srivastava author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Swarup Kumar Parida</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swarup Kumar Parida</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Swarup+Kumar+Parida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar Parida S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kumar Parida author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Debasis Chattopadhyay</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Debasis Chattopadhyay</div>
    </div>


                <div class="info-card-note">
                    <a reveal-id="dsw042-cor1" data-open="dsw042-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +91-11-26735189. Fax: +91-11-26741658. Email: <a href="mailto:debasis@nipgr.ac.in" target="_blank">debasis@nipgr.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Debasis+Chattopadhyay">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chattopadhyay D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Chattopadhyay author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw042-FM2"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors equally contributed to this study.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw042-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Masahiro Yano</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 1–10, <a href='https://doi.org/10.1093/dnares/dsw042'>https://doi.org/10.1093/dnares/dsw042</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 August 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 April 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 July 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2669389" href="/dnaresearch/article-pdf/24/1/1/10731765/dsw042.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sonal Gupta, Kashif Nawaz, Sabiha Parween, Riti Roy, Kamlesh Sahu, Anil Kumar Pole, Hitaishi Khandal, Rishi Srivastava, Swarup Kumar Parida, Debasis Chattopadhyay;  Draft genome sequence of <em>Cicer reticulatum</em> L., the wild progenitor of chickpea provides a resource for agronomic trait improvement, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 1–10, <a href='https://doi.org/10.1093/dnares/dsw042'>https://doi.org/10.1093/dnares/dsw042</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2669389?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Draft%20genome%20sequence%20of%20%26lt%3Bem%26gt%3BCicer%20reticulatum%26lt%3B%2Fem%26gt%3B%20L.%2C%20the%20wild%20progenitor%20of%20chickpea%20provides%20a%20resource%20for%20agronomic%20trait%20improvement&amp;publicationDate=2016-08-26&amp;volumeNum=24&amp;issueNum=1&amp;author=Gupta%2C%20Sonal%3B%20Nawaz%2C%20Kashif&amp;startPage=1&amp;endPage=10&amp;contentId=10.1093%2Fdnares%2Fdsw042&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f1%2f2669389" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61462649 id="61462649" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Cicer reticulatum</em> L. is the wild progenitor of the fourth most important legume crop chickpea (<em>C. arietinum</em> L.). We assembled short-read sequences into 416 Mb draft genome of <em>C. reticulatum</em> and anchored 78% (327 Mb) of this assembly to eight linkage groups. Genome annotation predicted 25,680 protein-coding genes covering more than 90% of predicted gene space. The genome assembly shared a substantial synteny and conservation of gene orders with the genome of the model legume Medicago truncatula. Resistance gene homologs of wild and domesticated chickpeas showed high sequence homology and conserved synteny. Comparison of gene sequences and nucleotide diversity using 66 wild and domesticated chickpea accessions suggested that the <em>desi</em> type chickpea was genetically closer to the wild species than the <em>kabuli</em> type. Comparative analyses predicted gene flow between the wild and the cultivated species during domestication. Molecular diversity and population genetic structure determination using 15,096 genome-wide single nucleotide polymorphisms revealed an admixed domestication pattern among cultivated (<em>desi</em> and <em>kabuli</em>) and wild chickpea accessions belonging to three population groups reflecting significant influence of parentage or geographical origin for their cultivar-specific population classification. The assembly and the polymorphic sequence resources presented here would facilitate the study of chickpea domestication and targeted use of wild <em>Cicer</em> germplasms for agronomic trait improvement in chickpea.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Cicer reticulatum L. PI489777&quot;"><em>Cicer reticulatum</em> L. PI489777</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;wild chickpea&quot;">wild chickpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence&quot;">genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="annotation">annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="diversity">diversity</a></div>                            <h2 scrollto-destination=61462651 id="61462651" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Chickpea (<em>Cicer arietinum</em> L.) is the only cultivated species in the genus <em>Cicer</em>. This is the fourth important legume crop after soybean (Glycine max), common bean (Phaseolus vulgaris) and common pea (Pisum sativum). According to the seed morphology, chickpea is of two types; <em>desi</em> type with small dark brown seeds and <em>kabuli</em> with bold pale brown seeds. Recently, draft genome assemblies of both the chickpea types have been reported.<a reveal-id="dsw042-B1 dsw042-B2 dsw042-B3" data-open="dsw042-B1 dsw042-B2 dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> Chickpea was domesticated with wheat, barley, peas and lentil as a member of West Asian Neolithic crops during the origin of agriculture around 10,000 years ago with the oldest archaeological evidence from 7500 B.C.<a reveal-id="dsw042-B4" data-open="dsw042-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw042-B5" data-open="dsw042-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The annual species <em>C. reticulatum</em> L. is considered to be the wild progenitor of the cultivated chickpea and both the species are interfertile, suggesting the wild as the primary gene pool of the cultigen. <em>C. echinospermum</em> D. is considered as the secondary gene pool and is crossable with both the wild progenitor and the cultivated chickpea, but shows a strong barrier to genic flux and the hybrids tend to be sterile. The other annual <em>Cicer</em> species may form the tertiary gene pool.<a reveal-id="dsw042-B6" data-open="dsw042-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> In contrast to the wild progenitors of other Neolithic crops, <em>C. reticulatum</em> shows a narrow eco-geographic distribution indicating its limited adaptive variation, which caused the first bottleneck of genetic base of cultivated chickpea.<a reveal-id="dsw042-B4" data-open="dsw042-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Domestication of many crops occurred with a limited number of founding genotypes followed by subsequent selection. Selection of phenotypes included loss of dormancy, reduced pod dehiscence, larger plant and seed size with more erect habit (<a reveal-id="dsw042-F1" data-open="dsw042-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). However, the key feature of chickpea domestication was the change in the cropping season. While most of the crops with West Asian origin are autumn-sown, chickpea was transformed from an autumn-sown crop to a spring-sown crop to avoid <em>Ascochyta</em> blight caused by the fungus <em>Ascochyta rabiei</em>.<a reveal-id="dsw042-B7" data-open="dsw042-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> This change in cropping season resulted in the separation of the reproductive phases of the cultivated species and the wild progenitor and caused another bottleneck in the genetic base. The major genetic difference that was selected for the change in cropping season was the loss of response to vernalization in the cultivated species.<a reveal-id="dsw042-B8" data-open="dsw042-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><div data-id="dsw042-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f1.png?Expires=1509239074&Signature=E-FmNk72ZTux5VZrfkhs~~OpDJZNpM9uGY7ig0t1O0~BT2cUdr0h8nyopFvtVseYcokeOj2BTX5o8IeqKgKvlmof8dJ2yIIlKufREqGkVh7yF3THBmqRYibYi2LCfGPLG79vUOPiPmrI1Rlf0OcGQAdVnVHzl25UtzwWGTDEqPmbDIu4nGQhty6rN6bGgqPDbWCaHcwma4yWSDjftoMmcUt0mUwHhAV7U5Ht5YaVXLb4PwFYzrTC0wEcOS1733PZAYhQ4ghNVpKI38s~MDIK4~4xhJ2w4vkERLcY2K8q5gCrT5qf1UlbM-r8AXSy0ILwzgkfj5xDLhZIrOVFEzjdVg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Images of field-grown wild (C. reticulatum L.) (left) and cultivated (C. arietinum L.) chickpea (right) plants and seeds. The colour version of the figure is available online." path-from-xml="dsw042f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462652/dsw042f1.tif" path-from-xml="dsw042f1.tif" target="_blank">View large</a><a section="61462652" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f1.png?Expires=1509239074&Signature=aShh56vN5I20Q7fRreKcGKhRKicBnPEYmhT3rACsm1PGPX7Tz8Mj1eBCW6iYMCF1NRKKYPuA3jc5ReLBG7HzFJxq6QW~EKW0FzqD5POeZyBOY~uVm7FQKBz6qjKO2Z6tADO9DqNVCYfgGp2nYjnOGqt0g43xS6WulOPBU-zD-Y5OQZNZFR2HPJhqzw0xtYLKuoNs5oZwr-4-OMDSqJfe6ilB00qRq5p8zhtKXbCsHTGaGsjnbcMGMdJAu8j9OVVVCu5Cznd1qIwMo2aN1Iv7udSpofyaXMRZ~z9q8aw7uda8~5Bivsskac7lOVqJgCfsyMNXVXOeH5-3GErOq8siyQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61462652&ar=2669389&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw042f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Images of field-grown wild (<em>C. reticulatum</em> L.) (left) and cultivated (<em>C. arietinum</em> L.) chickpea (right) plants and seeds. The colour version of the figure is available online.</p></div></div><div content-id="dsw042-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f1.png?Expires=1509239074&Signature=E-FmNk72ZTux5VZrfkhs~~OpDJZNpM9uGY7ig0t1O0~BT2cUdr0h8nyopFvtVseYcokeOj2BTX5o8IeqKgKvlmof8dJ2yIIlKufREqGkVh7yF3THBmqRYibYi2LCfGPLG79vUOPiPmrI1Rlf0OcGQAdVnVHzl25UtzwWGTDEqPmbDIu4nGQhty6rN6bGgqPDbWCaHcwma4yWSDjftoMmcUt0mUwHhAV7U5Ht5YaVXLb4PwFYzrTC0wEcOS1733PZAYhQ4ghNVpKI38s~MDIK4~4xhJ2w4vkERLcY2K8q5gCrT5qf1UlbM-r8AXSy0ILwzgkfj5xDLhZIrOVFEzjdVg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Images of field-grown wild (C. reticulatum L.) (left) and cultivated (C. arietinum L.) chickpea (right) plants and seeds. The colour version of the figure is available online." path-from-xml="dsw042f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462652/dsw042f1.tif" path-from-xml="dsw042f1.tif" target="_blank">View large</a><a section="61462652" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f1.png?Expires=1509239074&Signature=aShh56vN5I20Q7fRreKcGKhRKicBnPEYmhT3rACsm1PGPX7Tz8Mj1eBCW6iYMCF1NRKKYPuA3jc5ReLBG7HzFJxq6QW~EKW0FzqD5POeZyBOY~uVm7FQKBz6qjKO2Z6tADO9DqNVCYfgGp2nYjnOGqt0g43xS6WulOPBU-zD-Y5OQZNZFR2HPJhqzw0xtYLKuoNs5oZwr-4-OMDSqJfe6ilB00qRq5p8zhtKXbCsHTGaGsjnbcMGMdJAu8j9OVVVCu5Cznd1qIwMo2aN1Iv7udSpofyaXMRZ~z9q8aw7uda8~5Bivsskac7lOVqJgCfsyMNXVXOeH5-3GErOq8siyQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61462652&ar=2669389&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw042f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Images of field-grown wild (<em>C. reticulatum</em> L.) (left) and cultivated (<em>C. arietinum</em> L.) chickpea (right) plants and seeds. The colour version of the figure is available online.</p></div></div></p><p>Domestication process of most of the crops focused on improved yield, product size and organoleptic quality, however, tolerance to abiotic stresses and pathogen infection were compromised in this process. Naturally occurring resistant alleles in the wild relatives of the cultivated crops are an underexploited resource for plant breeding and can be introgressed into the elite cultivated genotypes to improve agricultural performance.<a reveal-id="dsw042-B9" data-open="dsw042-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Wild relatives are the vital source of higher natural allelic diversity, particularly for the improvement of cultivated crops with narrow genetic bases. They are also important for construction of high-density inter-specific genetic linkage maps and mapping of genes/QTLs (quantitative trait loci) governing agronomically important traits because of higher genomic polymorphism. Previously, genomes of the wild progenitors of some cultivated legume crops have been sequenced to decipher the genetic constitution of coding and non-coding components of their genomes and the process of domestication.<a reveal-id="dsw042-B10" data-open="dsw042-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw042-B11" data-open="dsw042-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> An interspecies mapping population, generated by crossing a chickpea cultivar ICC4958 and its wild progenitor <em>C. reticulatum</em> PI489777, is being widely used as a reference to construct high-resolution genetic linkage maps for identification and mapping of genes/QTLs especially controlling important abiotic/biotic stress tolerance and domestication traits.<a reveal-id="dsw042-B12 dsw042-B13 dsw042-B14 dsw042-B15" data-open="dsw042-B12 dsw042-B13 dsw042-B14 dsw042-B15" class="link link-ref link-reveal xref-bibr"><sup>12–15</sup></a> Here, we report a draft genome assembly of the wild progenitor of cultivated chickpea <em>C. reticulatum</em> L. PI489777. Comparative analysis of wild and cultivated chickpea genome assemblies and the genomic resources would accelerate the understanding of molecular basis of selection during the domestication process and genetic enhancement of chickpea.</p>                            <h2 scrollto-destination=61462654 id="61462654" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=61462655 id="61462655" class="section-title" >2.1. Sequencing and assembly</h3>
<p><em>Cicer</em><em>reticulatum</em> L. (accession PI489777) plants were grown in the plant growth chamber under controlled condition for 4 weeks before harvesting the fresh leaves for isolating genomic DNA using DNeasy Plant Maxi Kit (Qiagen, Cat no. 68163, Hilden, Germany) following manufacturer’s protocol. The sequence data was generated by Illumina HiSeq1000 platform (Illumina, San Diego, CA). Two individual whole genome shotgun (WGS) libraries with average insert size 180 base pair (bp), one WGS library with average insert size 500 bp and one mate-pair (MP) library each of 3,000 and 10,000 bp average insert sizes were prepared and sequenced in 100 baseX2 paired-end (PE) format following manufacturer’s protocol for the assembly. The raw sequence reads were filtered using NGS toolkit with default parameter for obtaining high-quality reads.<a reveal-id="dsw042-B16" data-open="dsw042-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Filtered reads were assembled by three tools namely, SOAPdenovo2, MaSuRCA and ALLPATHS-LG using a 512 GB RAM and 48 core server.<a reveal-id="dsw042-B17 dsw042-B18 dsw042-B19" data-open="dsw042-B17 dsw042-B18 dsw042-B19" class="link link-ref link-reveal xref-bibr"><sup>17–19</sup></a> Finally, the best assembly with ALLPATHS-LG version SL 2.0 with ploidy 2 and default parameters was used for analysis. The minimum scaffold size was kept as 1 kb. Scaffolds were ordered in the linkage groups following the marker sequences of a linkage map constructed using a bi-parental RIL (recombinant inbred line) mapping population (<em>C. arietinum</em> acc. ICC 4958 x <em>C. reticulatum</em> acc. PI 489777).<a reveal-id="dsw042-B20" data-open="dsw042-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The gaps of unknown sizes were joined with 100 N as per National Centre for Biotechnology Information (NCBI) guideline. Transcriptome coverage in the genome assembly was verified using BLAT.<a reveal-id="dsw042-B21" data-open="dsw042-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The assembly was submitted to NCBI under the Bioproject PRJNA263699. The sequence reads were submitted to the short read archive (SRA) of NCBI under the IDs SRR160365, SRR1640444, SRR1640451 and SRR1640455. Reference-based genome assemblies were generated by mapping the <em>C. reticulatum</em> WGS reads on ICC4958 (<em>desi</em>) and CDC Frontier (<em>kabuli</em>) genome assemblies using bwa mapping tool with default parameter and producing consensus sequence.<a reveal-id="dsw042-B1" data-open="dsw042-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw042-B22" data-open="dsw042-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p>                            <h3 scrollto-destination=61462657 id="61462657" class="section-title" >2.2. Gene annotation</h3>
<p>Gene annotation was performed following the pipeline described previously. Briefly, repeat sequences were identified and masked by the software packages REPEATMODELLER, PILER, REPEATSCOUT, LTR_FINDER REPEATMASKER (<a class="link link-uri" href="http://repeatmasker.org" target="_blank">http://repeatmasker.org</a>) and REPEATPROTEIN MASK.<a reveal-id="dsw042-B23 dsw042-B24 dsw042-B25 dsw042-B26" data-open="dsw042-B23 dsw042-B24 dsw042-B25 dsw042-B26" class="link link-ref link-reveal xref-bibr"><sup>23–26</sup></a> Repeat-masked assembly was used for annotating protein-coding genes by <em>ab initio</em>, homology-based and EST-based approaches. Augustus and GENESCAN tools trained with Arabidopsis data were used for <em>ab initio</em> prediction. EXONERATE (<a class="link link-uri" href="http://www.ebi.ac.uk/&amp;sim;guy/exonerate" target="_blank">http://www.ebi.ac.uk/∼guy/exonerate</a><span class="underline">/</span>) and PASA<a reveal-id="dsw042-B27" data-open="dsw042-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> tools were used for the homology-based and EST-based annotations, respectively, using predicted proteome sequences of <em>desi</em> and <em>kabuli</em> chickpeas<a reveal-id="dsw042-B1" data-open="dsw042-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw042-B2" data-open="dsw042-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> and assembled transcriptome of <em>C.</em> reticulatum.<a reveal-id="dsw042-B28" data-open="dsw042-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Outputs from these three approaches were integrated by EVIDENCEMODELLER (EVM)<a reveal-id="dsw042-B29" data-open="dsw042-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> to generate consensus gene sets and the EVM output was run through PASA for prediction of spliced variants and various elements like untranslated regions. Filtering parameters of minimum coding sequence (CDS) of 150 base pairs (bp) and low N-content (&lt;10%) were applied to arrive at the final predicted gene set. To resolve annotation differences between the orthologous gene pairs of wild and cultivated chickpeas during comparative analysis, corresponding gene sequences of both the genotypes were re-annotated using Augustus only for a better comparison. Gene annotation of the reference-based assembly was done by following the annotation files of the corresponding assemblies.</p><p>For functional annotation, the predicted protein sequences were searched with various databases such as, ntdb of NCBI, TrEMBL, UniRef100, SwissProt, TAIR and KEGG with cut off E value 1e−5. Resistance gene homologues (RGH) were identified following a previously described method.<a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Briefly, Hidden Markov Model (HMM) profiles were created for different R gene families using sequences present in the plant resistance gene database. These HMM profiles were used to identify the primary RGH set. These sequences were further screened for the presence of specific RGH-protein domains using InterPro scan and a search in the NCBI conserved domain database.</p>                            <h3 scrollto-destination=61462660 id="61462660" class="section-title" >2.3. Mining of single nucleotide polymorphisms and insertion/deletions</h3>
<p>Sequence reads were filtered with NGS tool kit.<a reveal-id="dsw042-B16" data-open="dsw042-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Filtered reads were mapped on the reference sequences using bwa mapping tool. Only uniquely mapped reads were considered for further analysis. The sam files were converted to bam files using samtools, and then to vcf format to identify single nucleotide polymorphisms (SNPs) and insertion/deletions (InDels). High quality SNPs were identified using only unique non-reference alleles supported by five or more reads (in case of <em>C. reticulatum</em> WGS reads) and/or three or more reads (in case of restriction-associated genotyping-by-sequencing reads). SNPs on pseudomolecules of eight linkage groups were plotted and visualized by Circos.<a reveal-id="dsw042-B30" data-open="dsw042-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Reference-based genome assemblies, reference-based gene annotation and the vcf files of the SNPs/InDels are curated at the website nipgr.res.in/WCGAP/.</p>                            <h3 scrollto-destination=61462662 id="61462662" class="section-title" >2.4. Pair-wise comparison of genomes and genes</h3>
<p>Gene cluster analysis was performed by all-against-all BLASTP search of the proteomes of all the concerned plants followed by orthoMCL.<a reveal-id="dsw042-B31" data-open="dsw042-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Pair-wise sequence comparison between the pseudomolecules of wild and cultivated chickpea was performed using default parameters of SyMap v4.0.<a reveal-id="dsw042-B32" data-open="dsw042-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> The whole genome dot plot with eight linkage groups of wild and <em>desi</em> chickpeas was generated by selecting hits with &gt;90% identity. Inter-species gene order colinearity was detected by MCScanX<a reveal-id="dsw042-B33" data-open="dsw042-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with default parameter considering BLASTP ≤1e<sup>−</sup><sup>5</sup>. Orthologous genes between the wild and cultivated chickpea were identified by BLASTP. Links of collinear gene blocks between the pseudomolecules of two species were shown by dual synteny or circle plot. Protein sequences of orthologous gene-pairs were aligned by ClustalW<a reveal-id="dsw042-B34" data-open="dsw042-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a>. The output file and CDS sequences of the corresponding genes were used to determine synonymous substitution rate (Ks) by PAL2NAL and CODEML in PAML4.5.<a reveal-id="dsw042-B35" data-open="dsw042-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw042-B36" data-open="dsw042-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Species divergence was calculated using the equation T = Ks/2r. The value of r was taken as 6.1 × 10 <sup>−</sup> <a reveal-id="dsw042-B9" data-open="dsw042-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> per year.<a reveal-id="dsw042-B37" data-open="dsw042-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a></p>                            <h3 scrollto-destination=61462664 id="61462664" class="section-title" >2.5. Diversity analysis</h3>
<p>Stacks pipeline was used to build loci for restriction site associated DNA (RAD) sequence reads.<a reveal-id="dsw042-B38" data-open="dsw042-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> The SNP-based genetic diversity and phylogenetic relationships among cultivated and wild chickpea accessions were determined using cladogram interface of TASSEL5.0 (<a class="link link-uri" href="http://www.maizegenetics.net/#&amp;excl;tassel/c17q9" target="_blank">http://www.maizegenetics.net/#!tassel/c17q9</a>). For assessment of population structure among chickpea accessions, the SNP genotyping data were analyzed in STRUCTURE following the methods of Kujur et al.<a reveal-id="dsw042-B39" data-open="dsw042-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> For determining nucleotide diversity in the intergenic regions, 2 kb up- and downstream regions of gene sequences according to direction of their annotation in the genome were considered. Significance analysis was performed by Wilcoxon rank sum test and paired <em>t</em>-test using R stats package (<a class="link link-uri" href="https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html" target="_blank">https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html</a>).</p>                            <h2 scrollto-destination=61462666 id="61462666" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=61462667 id="61462667" class="section-title" >3.1. Genome sequencing and assembly</h3>
<p>We prepared 3 paired-end libraries; 2 with 180 bp and 1 with 500 bp insert sizes, along with 2 mate-pair (MP) libraries with 3 kb and 10 kb insert sizes and sequenced using Illumina HiSeq1000 platform to generate 100 × 2 bp short sequence reads. The filtered high quality 1,198,234,588 reads (∼119 Gb) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>) were used for <em>de novo</em> assembly using ALLPATHS-LG, SOAPdenovo2 and MaSuRCA. ALLPATHS-LG produced a better output in comparison to other assembly tools in terms of number and lengths of the scaffolds, generating total scaffold length of 416,562,993 bp at 49X estimated genome coverage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span>). ALLPATHS-LG predicted an estimated genome size of 817.64 Mb at k-mer 25 and 63% of the genome was estimated to be repetitive, while distribution frequency of k-mer 17 by Jellyfish predicted an estimated genome size of 802.003 Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1</a></span>)<a reveal-id="dsw042-B40" data-open="dsw042-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>. The assembly tool generated total 7,383 scaffolds with N50 and average sizes of 216.8 and 56.534 kb, respectively, from 35,261 contigs with N50 and minimum sizes of 23.2 and 1 kb, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S3</a></span>). Seventeen small scaffolds showing significant similarity with bacterial genome sequences were removed by us and NCBI. Approximately 1,651 scaffolds were anchored to eight linkage groups (LGs) using marker sequences of a linkage map constructed using a RIL mapping population of <em>C. arietinum</em> ICC4958 X <em>C. reticulatum</em> PI489777<sup>18</sup> to generate eight pseudomolecules of total 327.07 Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S4</a></span>) within the total assembly of 416,658,930 bp distributed in 5,723 fragments. N50 length of the whole assembly was 39.84 Mb and the largest assembled pseudomolecule was LG6 with 58.23 Mb in length (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S5</a></span>). Out of 1,651 anchored scaffolds, 1,081 scaffolds (∼65%) covering 283.74 Mb (86.77% of total pseudomolecule length) were oriented. Less than 1,000 fragments contributed to more than 90% of the assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S2</a></span>). Approximately 74.64 and 48.33% reads of WGS and MP libraries, respectively, were mapped back and covered the whole genome assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S6</a></span>). Paired reads of the 3 and 10 kb MP libraries were mapped to the average distances of 2662 ± 496 bp and 9002 ± 1226 bp, respectively, as a validation of the assembly. Mapping of an earlier reported transcriptome assembly of <em>C. reticulatum</em><a reveal-id="dsw042-B28" data-open="dsw042-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> showed alignment of 90.74% (33,815 of 37,265 tentative contigs) assembled transcript on the genome assembly with ≥90% identity with ≥80% coverage. About 97% (1,136,248 of 1,171,453 reads) of high-quality transcriptome sequence reads could be mapped on the genome assembly. Core eukaryotic genes mapping approach (CEGMA)<a reveal-id="dsw042-B41" data-open="dsw042-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> analysis was able to map 319 (∼70%) of total 458 core eukaryotic genes with full coverage and 426 (∼93%) genes with more than 90% coverage and <em>E</em>-value 1e−10. Both these data suggested that more than 90% of <em>C. reticulatum</em> gene space has been covered in the present genome assembly. For a better comparative analysis with the available chickpea genome assemblies,<a reveal-id="dsw042-B1" data-open="dsw042-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> sequence reads of <em>C. reticulatum</em> were mapped on both the chickpea genome assemblies and two reference-based consensus genome assemblies of <em>C. reticulatum</em> were produced. The chickpea var. ICC4958 (<em>desi</em>) reference-based assembly was of 440,184,772 bp and the same for chickpea var. CDC Frontier (<em>kabuli</em>) was of 421,850,125 bp in lengths (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S7</a></span>). Polymorphic simple sequence repeats (SSR) between the wild and two domesticated chickpea draft assemblies and the flanking sequences were listed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S8</a></span>) as a resource for developing genetic markers. Salient features of the assembly are mentioned in <a reveal-id="dsw042-T1" data-open="dsw042-T1" class="link link-reveal link-table xref-fig">Table 1</a>. <div class="table-wrap"><div id="dsw042-T1" data-id="dsw042-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly and annotation statistics of <em>C. reticulatum</em> genome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Total span </td><td char=".">416,658,930 bp </td></tr><tr><td>No. Scaffolds </td><td char=".">5,723 </td></tr><tr><td>Max. Scaffold length </td><td char=".">58,232,078 bp </td></tr><tr><td>Min. Scaffold length </td><td char=".">1,000 bp </td></tr><tr><td>N50 length </td><td char=".">39,846,785 bp </td></tr><tr><td>N50 index </td><td char=".">5 </td></tr><tr><td>Number of annotated gene </td><td char=".">25,680 </td></tr><tr><td>Average gene length (bp) </td><td char=".">3,340.89 </td></tr><tr><td>Average coding sequence length (bp) </td><td char=".">1,192.195 </td></tr><tr><td>Functionally annotated gene </td><td char=".">25,075 </td></tr><tr><td>Genome assembly </td><td> </td></tr><tr><td>Based on ICC4958 </td><td char=".">440,184,772 bp </td></tr><tr><td>Genome assembly </td><td> </td></tr><tr><td>Based on CDC Frontier </td><td char=".">421,850,125 bp </td></tr><tr><td>%GC </td><td char=".">27.55 </td></tr></tbody></table></div><div content-id="dsw042-T1" class="table-modal"><table><tbody><tr><td>Total span </td><td char=".">416,658,930 bp </td></tr><tr><td>No. Scaffolds </td><td char=".">5,723 </td></tr><tr><td>Max. Scaffold length </td><td char=".">58,232,078 bp </td></tr><tr><td>Min. Scaffold length </td><td char=".">1,000 bp </td></tr><tr><td>N50 length </td><td char=".">39,846,785 bp </td></tr><tr><td>N50 index </td><td char=".">5 </td></tr><tr><td>Number of annotated gene </td><td char=".">25,680 </td></tr><tr><td>Average gene length (bp) </td><td char=".">3,340.89 </td></tr><tr><td>Average coding sequence length (bp) </td><td char=".">1,192.195 </td></tr><tr><td>Functionally annotated gene </td><td char=".">25,075 </td></tr><tr><td>Genome assembly </td><td> </td></tr><tr><td>Based on ICC4958 </td><td char=".">440,184,772 bp </td></tr><tr><td>Genome assembly </td><td> </td></tr><tr><td>Based on CDC Frontier </td><td char=".">421,850,125 bp </td></tr><tr><td>%GC </td><td char=".">27.55 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=61462669 id="61462669" class="section-title" >3.2. Genome annotation and comparative analysis of protein-coding genes</h3>
<p>Using repeat-masked assembly, the protein-coding genes were predicted by three approaches (<em>ab initio</em>, homology-based and EST-based), followed by generating a high confidence non-redundant set of consensus genes by merging their results. A total of 25,680 non-redundant consensus genes spanning 85.786 Mb with a gene density of 6.1 genes per 100 kb was predicted. A sum of 666 genes was predicted to have average 2.08 transcript isoforms resulting in total 26,404 protein coding DNA sequences (CDS). Average gene length was 3.34 kb with 5.76 exons per gene and the total CDS length was 30.622 Mb with an average length 1.192 kb. A total of 22,259 genes were located on the pseudomolecules (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S9</a></span>). Annotation of the reference-based assemblies following the annotation coordinates of the reference assemblies resulted in identification of 27,722 and 23,656 genes for ICC4958- and CDC Frontier-based assemblies. The number of predicted genes by <em>de novo</em> annotation was less because of less availability of wild chickpea ESTs in the public databases. The <em>ab initio</em> and EST-based approaches predicted 27,151 and 25,946 gene models, respectively, resulting in annotation of 25,680 high confidence consensus gene models. A clustering analysis of the <em>de novo</em> protein sets showed that a total of 64,904 proteins belonging to 15,068 families were shared by wild, <em>desi</em> (ICC4958) and <em>kabuli</em> (CDC Frontier) chickpeas, while 17,314 and 15,695 protein families were shared by the wild and <em>desi</em>, and the wild and <em>kabuli</em> chickpeas, respectively. A sum of 131 proteins belonging to 59 families of the wild chickpea were not shared with either <em>desi</em> or <em>kabuli</em> chickpea (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S3</a></span>). Notably, 97.64% (25,075) predicted proteins displayed significant sequence similarity (≤1e<sup>−</sup><sup>5</sup> by BLAST search) with at least one of the public protein databases analyzed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S10</a></span>). 15.70% (4033) of the total wild chickpea CDSs did not show any sequence variation with those of <em>desi</em> chickpea, while sequences of 12.02% (3209) wild chickpea CDSs were identical with those of <em>kabuli</em> chickpea. Approximately 83.62% (21,475) of wild chickpea CDSs showed significant sequence similarity (≥90% identity with ≥80% coverage) with the <em>desi</em> chickpea CDSs in contrast to 74.65% (19,170) with <em>kabuli</em> chickpea CDSs. In case of protein sequence, 73.50% (18,876) and 63.46% (16,296) of predicted proteins of <em>C. reticulatum</em> showed significant sequence similarity (≥95% identity with ≥80% coverage) with the predicted protein sets of <em>desi</em> and <em>kabuli</em> chickpea, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S11</a></span>). All these observations emphasized that the <em>desi</em> type chickpea was closer to the wild progenitor <em>C. reticulatum</em> in evolution as compared to the <em>kabuli</em> type chickpea.</p>                            <h3 scrollto-destination=61462671 id="61462671" class="section-title" >3.3. Genomic features</h3>

  <p>Putative pericentromeric region in each linkage group of <em>C. reticulatum</em> assembly was delineated by comparing genetic and physical distances and mapping the repeat and gene densities (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S4</a></span> and Table S12). Gene density in the pericentromeric region was 5.1 genes/100 kb, in contrast to 9.3 genes/100 kb in the euchromatic region. Average recombination rate in the pericentromeric region (1,921 kb/cM) was similar to that in the <em>desi</em> chickpea assembly (1,691 kb/cM).<a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> This high pericentromeric recombination rate (4,350 kb/cM in Phaseolus vulgaris) indicated less anchoring of sequence scaffolds in the pericentromeric region due to lack of markers and scarcity of longer scaffolds. Poor gene density (3.8 genes/100 kb) in the un-anchored scaffolds indicated that those mostly belonged to the pericentromeric region. <em>C. reticulatum</em> genome shared a substantial synteny and conservation of gene orders with the genome of the model legume <em>Medicago truncatula</em> (Medicago) (<a reveal-id="dsw042-F2" data-open="dsw042-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Total 17,154 genes of Medicago and <em>C. reticulatum</em> existed in 849 colinear blocks of 5 or more genes. The highest synteny was observed between Medicago chromosome 1 (<em>Mt</em>1) and <em>C. reticulatum</em> LG 4 (<em>Cr</em>4) with 82 colinear blocks comprising of 2,037 genes, followed by the synteny between <em>Mt</em>3 and <em>Cr</em>5 with 52 blocks of 1,475 genes and between <em>Mt</em>2 and <em>Cr</em>1 with 41 blocks of 1,445 genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S13</a></span>). <em>C. reticulatum</em> genome assembly was compared with that of the <em>desi</em> chickpea (ICC4958). The same genetic map was used for both the assemblies and, therefore, pseudomolecules in both the assemblies shared an overall synteny at the sequence level (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5</a></span>) and gene order level (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S6</a></span>) for LGs 2, 6 and 7. The other pseudomolecules showed a few disagreements. Although, extensive local gene collinearity was observed in all the pseudomolecules, incongruity in the positions and orientations of a few gene blocks between the corresponding pseudomolecules of LG1, 3, 5 and 8 were observed. The most disagreement was observed in the upper arm of LG4. These disagreements might have resulted from relatively more dependence on the genetic map for anchoring a large number of scaffolds in case of ICC4958. Comparatively larger scaffolds were anchored in case of <em>C. reticulatum</em> using a fewer genetic markers and most of them are oriented. <div data-id="dsw042-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f2.png?Expires=1509239074&amp;Signature=NdVxWmfuvZ1gssDIReWl-afS1B3K4K0U5n~AveooQ7AIBIFRbbkNH8mrvIyZb5YO0gA2-bEqVu6hdGWAZl7dXlRr7jvHjXDUwVJnrhtNCpY2vvMLuIV~0VHQTZZ5PXKAufuIx4IADjf76MiN4SkSrEL~Zyfy86ByjAkrhjoVIqC9OSfC8sAcUUjhTXQw8ol-dem0Ok1i4yfOzWDSSJchMz94d7iLDI4wsXylH8DeAf0Ifmtc1O7TiDfnvlZpMznDtT~V9gjdpu401yyEqvzE6DR-RsFim5BmoLSvaJvT3-QcMlF9z-ftoMaEEIzvO8-rtywPawaifEv1JD-0BPF-Kg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Syntenic relationship between C. reticulatum (Cr) and Medicago truncatula (Mt) pseudomolecules. Mt pseudomolecules are labelled as Mt1-8. C. reticulatum pseudomolecules are labelled as Cr1-8. Collinear blocks are shown according to the shades of the corresponding Cr pseudomolecules. The colour version of the figure is available online." path-from-xml="dsw042f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462672/dsw042f2.tif" path-from-xml="dsw042f2.tif" target="_blank">View large</a><a section="61462672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f2.png?Expires=1509239074&amp;Signature=MTIcuXr4z~KerhQmzhVIvWvhMzY0mx49Iooic~~ox1ON2-9HsbPB0ttklv3JC3iEdAKostli4rzeJZnYM8M122W8SnbWdh1qlI50bJH-EDWs0Q59xv5VyKHFQ0QWsduSKonC9IcCx~xr4goaezeDLSZMbYgi2pSHTgiEGap3~vBe6KkgVZdJHtwYHAcU1hEisAkTAVn0VZbGUSBCQjSaD5DYhk8sm4Ka-kUU3ptxpqiZoVc~MPTDVbl5PeIGKrDhNl6Z7-ajuHeQasuBaGCeHKvH7bkTy8k1Km3fQ5Yy~Hdwe4KzTgvVIAQdCAYi7gdZWIFz4yS0r3~e0tE20u6JyA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61462672&amp;ar=2669389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw042f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Syntenic relationship between <em>C. reticulatum</em> (Cr) and <em>Medicago truncatula</em> (Mt) pseudomolecules. Mt pseudomolecules are labelled as Mt1-8. <em>C. reticulatum</em> pseudomolecules are labelled as Cr1-8. Collinear blocks are shown according to the shades of the corresponding Cr pseudomolecules. The colour version of the figure is available online.</p></div></div><div content-id="dsw042-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f2.png?Expires=1509239074&amp;Signature=NdVxWmfuvZ1gssDIReWl-afS1B3K4K0U5n~AveooQ7AIBIFRbbkNH8mrvIyZb5YO0gA2-bEqVu6hdGWAZl7dXlRr7jvHjXDUwVJnrhtNCpY2vvMLuIV~0VHQTZZ5PXKAufuIx4IADjf76MiN4SkSrEL~Zyfy86ByjAkrhjoVIqC9OSfC8sAcUUjhTXQw8ol-dem0Ok1i4yfOzWDSSJchMz94d7iLDI4wsXylH8DeAf0Ifmtc1O7TiDfnvlZpMznDtT~V9gjdpu401yyEqvzE6DR-RsFim5BmoLSvaJvT3-QcMlF9z-ftoMaEEIzvO8-rtywPawaifEv1JD-0BPF-Kg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Syntenic relationship between C. reticulatum (Cr) and Medicago truncatula (Mt) pseudomolecules. Mt pseudomolecules are labelled as Mt1-8. C. reticulatum pseudomolecules are labelled as Cr1-8. Collinear blocks are shown according to the shades of the corresponding Cr pseudomolecules. The colour version of the figure is available online." path-from-xml="dsw042f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462672/dsw042f2.tif" path-from-xml="dsw042f2.tif" target="_blank">View large</a><a section="61462672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f2.png?Expires=1509239074&amp;Signature=MTIcuXr4z~KerhQmzhVIvWvhMzY0mx49Iooic~~ox1ON2-9HsbPB0ttklv3JC3iEdAKostli4rzeJZnYM8M122W8SnbWdh1qlI50bJH-EDWs0Q59xv5VyKHFQ0QWsduSKonC9IcCx~xr4goaezeDLSZMbYgi2pSHTgiEGap3~vBe6KkgVZdJHtwYHAcU1hEisAkTAVn0VZbGUSBCQjSaD5DYhk8sm4Ka-kUU3ptxpqiZoVc~MPTDVbl5PeIGKrDhNl6Z7-ajuHeQasuBaGCeHKvH7bkTy8k1Km3fQ5Yy~Hdwe4KzTgvVIAQdCAYi7gdZWIFz4yS0r3~e0tE20u6JyA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61462672&amp;ar=2669389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw042f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Syntenic relationship between <em>C. reticulatum</em> (Cr) and <em>Medicago truncatula</em> (Mt) pseudomolecules. Mt pseudomolecules are labelled as Mt1-8. <em>C. reticulatum</em> pseudomolecules are labelled as Cr1-8. Collinear blocks are shown according to the shades of the corresponding Cr pseudomolecules. The colour version of the figure is available online.</p></div></div></p>
                            <h3 scrollto-destination=61462673 id="61462673" class="section-title" >3.4. Comparison of resistance (R) genes</h3>

  <p>Attenuation of resistance against diseases is a common effect of domestication. Therefore, the resistance gene homologues (RGH) of <em>C. reticulatum</em> were analyzed in detail in comparison to those of the domesticated chickpea. Total 116 RGH were identified in the wild chickpea assembly following stringent criteria described previously.<a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, when the gene sequences of 133 RGH, identified in the advanced draft genome of chickpea ICC4958, were mapped on the wild chickpea genome, four more RGH were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Text S1</a></span>), which were not annotated before. Three clusters of R genes were observed in the wild chickpea genome each in LG3, 5 and 6. Among all identified RGH, 22 belong to TIR-NBS-LRR and 28 belong to CC-NBS-LRR families (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S14</a></span>). One hundred four RGH of wild chickpea were orthologous to those of ICC4958, of which coding sequences of 80 orthologous pairs were of same length including 24 identical pairs. The remaining 24 orthologous pairs showed length differences due to InDels in the exons. Experimental validation of most of their sequence was not possible due to lack of the corresponding transcript sequences in the databases. Only two of them were verified by comparing the transcript sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Text S2</a></span>). One of those (Cr_03218) belonged to CC-NBS family and is orthologous to Ca_03824 of ICC4958 and XM_004490520.2 of CDC Frontier and showed 69 base-deletion in the exon 3. Cr_03218 is orthologous to RGA4 known for resistance against Magnaporthe oryzae.<a reveal-id="dsw042-B42" data-open="dsw042-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The other (Cr_10687) belonged to NBS-LRR family and is orthologous to Ca_09490 of ICC4958 and XM_012715315.1 of CDC Frontier showed 9 base deletion in the exon1 of the gene. Cr_10687 is orthologous to <em>RPM1</em> gene, well known for conferring resistance against bacterial leaf spot caused by <em>Pseudomonus</em> syringae pv. maculicola.<a reveal-id="dsw042-B43" data-open="dsw042-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Of the remaining 16 genes of total 120 RGHs, 3 genes were not found in the cultivated chickpea assemblies, 3 genes showed large deletions in the wild species as compared to the corresponding genes in the cultivated species and 10 genes showed deletion in the wild chickpea assembly due to the presence of ambiguous bases (N). Total 34 RGHs are present in 9 clusters (3 or more RGH within 2 Mb) in <em>C. reticulatum</em> genome assembly (<a reveal-id="dsw042-F3" data-open="dsw042-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). LG6 showed the presence of total 9 RGHs in 2 clusters. Six RGHs were found present in LG1 in a cluster. Each of LG2, 3, 4, 5, 7 and 8 showed the presence of 1 RGH cluster. Seven of these clusters were collinear with the CDC Frontier genome assembly. Colinearity of other two clusters could not be confirmed due to the presence of the orthologous genes in unanchored scaffolds of the CDC Frontier assembly. A cluster analysis of the predicted RGHs in the wild and <em>desi</em> chickpea assemblies showed high sequence conservation without any outlier group (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S7</a></span>). Total 54 RGH of Medicago showed colinearity with <em>C. reticulatum</em> RGHs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S15</a></span>). Orthologues of some well-known RGH, such as RGA4, TMV resistance gene <em>N</em><a reveal-id="dsw042-B44" data-open="dsw042-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> and powdery mildew resistance gene <em>RPW8</em><a reveal-id="dsw042-B45" data-open="dsw042-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> can be traced comparing <em>C. reticulatum</em> and Medicago genomes through syntenic relationship (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S8</a></span>). <div data-id="dsw042-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f3.png?Expires=1509239074&amp;Signature=bG~sV9vbMv3X~DE0ebZGAewutNMHcSdhOlYwwqAl3iHQ4VTYi~hBNoI9iYaBKxPLkUTGCbhOsNxMMxNuhT0YpgyQIKVw5xUX2PyD3hlfx2D-9Aetb~hd2~uhBix4HMgn8TVffEzzLgg55fNyEboF69ZaZ8-nkKMouwZVwTy0-d2CpDzDQIGeO-5B8rxJN7X5p9Jz81kkKYvyO-8f17phuy9A36QRxYWTETD6rm1-UlHkyVPfjXMvSmVhoBnA3V2Q01Pn8sxHrYCsyFNW0KWMs9ACTUnVlAlpSJbvViOoDUCFy-4UKwAF0v1eOwr1PqVbtPKw6ys~8sNYWOIFaB2vvg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide sequence diversity between wild and cultivated chickpea. (a) Genome-wide distribution of nucleotide diversity (θπ) within the wild, desi and kabuli chickpea genotypes. RAD sequence reads of 10 accessions of C. reticulatum, 28 accessions each of desi and kabuli chickpeas were mapped on the pseudomolecules of 8 LGs of C. reticulatum. Circular maps show (from periphery to centre) distribution of gene density (in 0.5 Mb) with R gene loci are shown by vertical lines, nucleotide diversities within C. reticulatum, desi and kabuli accessions. A 500 kb bin size with a 50 kb sliding window and the maximum value of 1 for the Y-axis was used to plot nucleotide diversity. (b) Density plot showing distribution of Ks values of the orthologous gene pairs between wild and cultivated (desi-light line, kabuli-dark line) chickpeas. The colour version of the figure is available online (desi-green line, kabuli-red line)." path-from-xml="dsw042f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462674/dsw042f3.tif" path-from-xml="dsw042f3.tif" target="_blank">View large</a><a section="61462674" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f3.png?Expires=1509239074&amp;Signature=UCbrI-tilbKuHg6o9IXWInxc8I~hqgFLQcEIH1~Fg7gLCusI-~vpKIeXQnZNKbzFVSyiU5dgVLRZvzyyJuxmBV~bv1vronk3OmF5THebOCd1Abd5YOZ6uOlwsCNCA~wb4hJnza8y5E-~zlqmle5nCFJiwef9SOCTVuDtnTRPm95lHN0eP-oHXeD10aA3DvqIM9Gz1ZNqqCNwBNvNKYQTr1cR1Wnd3DwSRIYeC3N315oYacNacBNOvOaKNEKAMIQ3ECsfgBxVVfEZZf0Hsryt71l55aIokBb9hqhsB-YcIvpNeYDQ7SChswqUgvFW2ZAozfBC6v5gzDRtFScEfOKbxQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61462674&amp;ar=2669389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw042f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide sequence diversity between wild and cultivated chickpea. (a) Genome-wide distribution of nucleotide diversity (θπ) within the wild, <em>desi</em> and <em>kabuli</em> chickpea genotypes. RAD sequence reads of 10 accessions of <em>C. reticulatum</em>, 28 accessions each of <em>desi</em> and <em>kabuli</em> chickpeas were mapped on the pseudomolecules of 8 LGs of <em>C. reticulatum</em>. Circular maps show (from periphery to centre) distribution of gene density (in 0.5 Mb) with R gene loci are shown by vertical lines, nucleotide diversities within <em>C. reticulatum</em>, <em>desi</em> and <em>kabuli</em> accessions. A 500 kb bin size with a 50 kb sliding window and the maximum value of 1 for the <em>Y</em>-axis was used to plot nucleotide diversity. (b) Density plot showing distribution of Ks values of the orthologous gene pairs between wild and cultivated (<em>desi</em>-light line, <em>kabuli</em>-dark line) chickpeas. The colour version of the figure is available online (<em>desi</em>-green line, <em>kabuli</em>-red line).</p></div></div><div content-id="dsw042-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f3.png?Expires=1509239074&amp;Signature=bG~sV9vbMv3X~DE0ebZGAewutNMHcSdhOlYwwqAl3iHQ4VTYi~hBNoI9iYaBKxPLkUTGCbhOsNxMMxNuhT0YpgyQIKVw5xUX2PyD3hlfx2D-9Aetb~hd2~uhBix4HMgn8TVffEzzLgg55fNyEboF69ZaZ8-nkKMouwZVwTy0-d2CpDzDQIGeO-5B8rxJN7X5p9Jz81kkKYvyO-8f17phuy9A36QRxYWTETD6rm1-UlHkyVPfjXMvSmVhoBnA3V2Q01Pn8sxHrYCsyFNW0KWMs9ACTUnVlAlpSJbvViOoDUCFy-4UKwAF0v1eOwr1PqVbtPKw6ys~8sNYWOIFaB2vvg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide sequence diversity between wild and cultivated chickpea. (a) Genome-wide distribution of nucleotide diversity (θπ) within the wild, desi and kabuli chickpea genotypes. RAD sequence reads of 10 accessions of C. reticulatum, 28 accessions each of desi and kabuli chickpeas were mapped on the pseudomolecules of 8 LGs of C. reticulatum. Circular maps show (from periphery to centre) distribution of gene density (in 0.5 Mb) with R gene loci are shown by vertical lines, nucleotide diversities within C. reticulatum, desi and kabuli accessions. A 500 kb bin size with a 50 kb sliding window and the maximum value of 1 for the Y-axis was used to plot nucleotide diversity. (b) Density plot showing distribution of Ks values of the orthologous gene pairs between wild and cultivated (desi-light line, kabuli-dark line) chickpeas. The colour version of the figure is available online (desi-green line, kabuli-red line)." path-from-xml="dsw042f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462674/dsw042f3.tif" path-from-xml="dsw042f3.tif" target="_blank">View large</a><a section="61462674" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f3.png?Expires=1509239074&amp;Signature=UCbrI-tilbKuHg6o9IXWInxc8I~hqgFLQcEIH1~Fg7gLCusI-~vpKIeXQnZNKbzFVSyiU5dgVLRZvzyyJuxmBV~bv1vronk3OmF5THebOCd1Abd5YOZ6uOlwsCNCA~wb4hJnza8y5E-~zlqmle5nCFJiwef9SOCTVuDtnTRPm95lHN0eP-oHXeD10aA3DvqIM9Gz1ZNqqCNwBNvNKYQTr1cR1Wnd3DwSRIYeC3N315oYacNacBNOvOaKNEKAMIQ3ECsfgBxVVfEZZf0Hsryt71l55aIokBb9hqhsB-YcIvpNeYDQ7SChswqUgvFW2ZAozfBC6v5gzDRtFScEfOKbxQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61462674&amp;ar=2669389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw042f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide sequence diversity between wild and cultivated chickpea. (a) Genome-wide distribution of nucleotide diversity (θπ) within the wild, <em>desi</em> and <em>kabuli</em> chickpea genotypes. RAD sequence reads of 10 accessions of <em>C. reticulatum</em>, 28 accessions each of <em>desi</em> and <em>kabuli</em> chickpeas were mapped on the pseudomolecules of 8 LGs of <em>C. reticulatum</em>. Circular maps show (from periphery to centre) distribution of gene density (in 0.5 Mb) with R gene loci are shown by vertical lines, nucleotide diversities within <em>C. reticulatum</em>, <em>desi</em> and <em>kabuli</em> accessions. A 500 kb bin size with a 50 kb sliding window and the maximum value of 1 for the <em>Y</em>-axis was used to plot nucleotide diversity. (b) Density plot showing distribution of Ks values of the orthologous gene pairs between wild and cultivated (<em>desi</em>-light line, <em>kabuli</em>-dark line) chickpeas. The colour version of the figure is available online (<em>desi</em>-green line, <em>kabuli</em>-red line).</p></div></div></p>
                            <h3 scrollto-destination=61462675 id="61462675" class="section-title" >3.5. Sequence diversity between the wild and cultivated chickpea</h3>
<p>Sequence reads of <em>C. reticulatum</em> PI489777 were mapped on pseudomolecules of eight assembled linkage groups of chickpea cultivars CDC Frontier and ICC4958 to analyze genome-wide distribution of SNPs and InDels (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S9 and Table S16</a></span>). LG4 of both the assemblies showed the presence of the highest number of SNPs and InDels supporting the previous observation that LG4 is the most variable among all the linkage groups.<a reveal-id="dsw042-B1" data-open="dsw042-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The rate of synonymous substitutions (Ks) were calculated for 15,403 and 12,780 orthologous gene pairs between wild-<em>desi</em> and wild-<em>kabuli</em>, respectively, to ascertain diversification periods of the domesticated species from the wild species. The plot of Ks values against the number of gene pairs showed two peaks of diversification, one sharp peak at 0.0001 and another diffused peak around 0.006 (<a reveal-id="dsw042-F3" data-open="dsw042-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). Similar pattern was observed when the transcriptome sequences of the domesticated and wild chickpeas were compared.<a reveal-id="dsw042-B46" data-open="dsw042-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The diffused peak around Ks 0.006 corresponds to speciation around 0.5 million years ago, while the peak at 0.0001 possibly suggests recent gene flow from the wild during the domestication process as both the wild and domesticated species are interfertile. To investigate that, SNP frequencies were compared between the intergenic regions of the orthologous gene pairs constituting these peaks. The SNP frequencies in the intergenic regions of the corresponding gene pairs of the younger peak (0.0055 bp<sup>−</sup><sup>1</sup>) was significantly lower (<em>P</em>-value &lt; 0.0001) than that of the older peak (0.0090 bp<sup>−</sup><sup>1</sup>) suggesting gene flow from the wild to cultivated chickpea. Linkage groups of the present wild chickpea genome assembly was used as the reference for mapping restriction site associated DNA (RAD) sequence reads generated from 10 <em>C. reticulatum</em>, and 56 (28 each of <em>desi</em> and <em>kabuli</em>) cultivated chickpea accessions to assess sequence diversities. Total 15,096 SNPs discovered revealed high average pair-wise nucleotide diversity within the wild (θ<sub>Π </sub>= 0.338) as compared to that within the cultivated chickpeas (0.296). Nucleotide diversities were compared between the wild and domesticated accessions in the intergenic regions of the orthologous gene pairs representing two aforementioned younger and older peaks. The nucleotide diversity in the intergenic regions of the genes belonging to younger peak (θ<sub>Π</sub> = 0.33) was significantly lower (<em>P</em>-value &lt; 0.0063) than that of the older peak (θ<sub>Π</sub> = 0.37) further implicating the occurrence of gene flow from the wild to the domesticated chickpea. Nucleotide diversities of the wild and domesticated chickpeas were plotted along the linkage groups (<a reveal-id="dsw042-F3" data-open="dsw042-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S17</a></span>). Expectedly, LG 4 showed the highest average nucleotide variation within the wild (θ<sub>Π</sub> = 0.389) and the domesticated chickpeas (θ<sub>Π</sub> = 0.320). Out of total 7,185 SNPs within the wild chickpea accessions, 3,615 SNPs were found to be present in all 10 <em>C.</em><em>reticulatum</em> accessions, and 1,083 and 611 of these were shared with all the <em>desi</em> and <em>kabuli</em> accessions, respectively. Four hundred and thirty-nine SNPs were found common in all the wild and domesticated chickpeas used in this study. Flanking sequences of the unique SNPs present within the wild, <em>desi</em> and <em>kabuli</em> chickpeas were provided as marker resource (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S18</a></span>).</p><p>Although, there are many contrasting phenotypic differences between wild and cultivated chickpeas and also between the <em>desi</em> and <em>kabuli</em> landraces such as, erectness, seed coat and flower colour, sizes of seed, leaf and whole plant, in contrast to other grain legumes, the major trait for chickpea domestication was the change in the cropping season and associated loss of vernalization response.<a reveal-id="dsw042-B47" data-open="dsw042-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> In Arabidopsis, functional <em>VRN1</em> and <em>VRN2</em> genes are required to maintain vernalization response.<a reveal-id="dsw042-B48" data-open="dsw042-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsw042-B49" data-open="dsw042-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><em>VRN1</em> and <em>VRN2</em> orthologues in the wild and domesticated chickpea genome assemblies did not show any sequence variation in the CDS. However, 5′-upstream activating sequences (5′-UAS) of <em>VRN1</em> orthologues in the <em>desi</em> and <em>kabuli</em> genome assemblies showed a 35 base insertion at around 1.1 kb upstream of the translation start codon as compared to their orthologue in the wild chickpea. Ten accessions each of <em>desi</em> and <em>kabuli</em> chickpea and 4 accessions of <em>C. reticulatum</em> were used to verify this sequence variation by PCR amplification followed by sequencing. All the 20 chickpea cultivars used showed 35 base larger amplification products than the 4 accessions of wild chickpea (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S10</a></span>). On the other hand, in case of <em>VRN2</em> orthologue located at LG8, a conversion of a dinucleotide repeat (TC)<sub>15</sub>, present around 200 bp upstream to the translation start codon in the wild genotype, to (TC)<sub>49</sub> and (TC)<sub>25</sub> was observed in the <em>desi</em> and the <em>kabuli</em> genome assemblies, respectively. This sequence variation was verified in 10 accessions each of <em>desi</em> and <em>kabuli</em> chickpea and 4 accessions of <em>C. reticulatum</em>. The extent of this repeat varied in the domesticated and wild accessions when investigated by PCR amplification. However, all the used wild accessions displayed shorter repeat sequence than the cultivated accessions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S11</a></span>). Further investigations are required to associate these structural variations with the vernalization response in chickpea.</p>                            <h3 scrollto-destination=61462678 id="61462678" class="section-title" >3.6 Molecular diversity, phylogeny and population genetic structure</h3>

  <p>The estimation of pair-wise genetic distance among 28 accessions each of <em>desi</em> and <em>kabuli</em> chickpea and 10 accessions of wild <em>C. reticulatum</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&amp;Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S19</a></span>) using 15,096 SNPs physically mapped over eight pseudomolecules of LGs of the present assembly depicted a wide range of distance coefficient from 0.13 to 0.56 with an average of 0.35. The genetic distance between the <em>desi</em> and <em>kabuli</em> cultivar groups (distance coefficient 0.35) was least as compared to that of <em>desi</em> versus wild (0.42) and <em>kabuli</em> versus wild (0.45). The genetic relationship among 56 cultivated and 10 wild chickpea accessions was depicted in an unrooted dendrogram (<a reveal-id="dsw042-F4" data-open="dsw042-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>). The SNPs clearly discriminated all the 66 accessions from each other resulting in a definite grouping among <em>desi</em> and <em>kabuli</em> and wild chickpeas. The accessions belonging to wild <em>C. reticulatum</em> were clustered together in a group distinctly from the <em>desi</em> and <em>kabuli</em> cultivars. <div data-id="dsw042-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f4.png?Expires=1509239074&amp;Signature=CV2WZkgLLljdyFJJfRWx3HG~EUxWZC~5QL7AAx1DTuLjyBE4kzjO0XYEY3xpiVoUNVrDE9aoK~dOtcgncCLoL6pxUHt9vuuSzqhj1Q0J2F-O5JPBcdKmOMcxPtJgSiz5hJZlYDUxvnEtBgnaTCMTerk0IOhEEOTWky-MCd-PuRPM28C28SH-N3ILwywzSrYEAq19HNqIZU~3shXu6KeRPWb1OOSqtEJPbGc-AAi~r~AfPYqj0W~Ax~9NcG4h~dYPzCONrIH2k88E1LOTmnVbifi44YINbSP4yC-rPqKRQJ20PBebph4Ht37psyCaanVC0Z2e8Z1k3KS7OKPmWQpR6A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide SNP-based molecular diversity among 66 wild and domesticated chickpea accessions. (a) Unrooted phylogram depicting the genetic relationships (Nei’s genetic distance) among 66 wild (R1-10), desi (D1-28) and kabuli (K1-28) chickpea accessions based on genome-wide SNP mapped on C. reticulatum genome assembly. The phylogenetic tree differentiated 66 accessions into three diverse groups. (b) The population genetic structure of the wild and domesticated chickpea accessions. The mapped genetic markers assigned to three distinct desi, kabuli and wild population groups at population number (K = 3). The accessions represented by vertical bars along the horizontal axis were classified into K colour segments based on their estimated membership fraction in each K cluster. The colour version of the figure is available online. " path-from-xml="dsw042f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462679/dsw042f4.tif" path-from-xml="dsw042f4.tif" target="_blank">View large</a><a section="61462679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f4.png?Expires=1509239074&amp;Signature=GYJpcHo~F0Y3M62txKbr83CD4ashMBb19KjFipAOCJrHv1~lkbg-XSF~IB938NtjOs9B1ZOgD2drpatZzfRsWwoKGk0h7ouPaaWobGRnWdKgHPva1EJzehcnZHwD-K6OCTaKCkQL~kJzenrni4nFIxdL~~B6HvqHYi0U1F1WTJHJ3yz6dTbBAsM-NfE0LoLyY8or0r5wpj7XDbHv2epbziKLgpJKeopGQZ~d5PaUpn33yqQ4262grszUKsBddPGb4wYKmvmTH3NMz~FQcWvN5O7BYScXSq5fgrjzdGEO~GJiKh5icCkCrM~CFmin0vvXJA7mWo93NqBu8PjmHwPNAw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61462679&amp;ar=2669389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw042f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide SNP-based molecular diversity among 66 wild and domesticated chickpea accessions. (a) Unrooted phylogram depicting the genetic relationships (Nei’s genetic distance) among 66 wild (R1-10), <em>desi</em> (D1-28) and <em>kabuli</em> (K1-28) chickpea accessions based on genome-wide SNP mapped on <em>C. reticulatum</em> genome assembly. The phylogenetic tree differentiated 66 accessions into three diverse groups. (b) The population genetic structure of the wild and domesticated chickpea accessions. The mapped genetic markers assigned to three distinct <em>desi</em>, <em>kabuli</em> and wild population groups at population number (K = 3). The accessions represented by vertical bars along the horizontal axis were classified into K colour segments based on their estimated membership fraction in each K cluster. The colour version of the figure is available online. </p></div></div><div content-id="dsw042-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/m_dsw042f4.png?Expires=1509239074&amp;Signature=CV2WZkgLLljdyFJJfRWx3HG~EUxWZC~5QL7AAx1DTuLjyBE4kzjO0XYEY3xpiVoUNVrDE9aoK~dOtcgncCLoL6pxUHt9vuuSzqhj1Q0J2F-O5JPBcdKmOMcxPtJgSiz5hJZlYDUxvnEtBgnaTCMTerk0IOhEEOTWky-MCd-PuRPM28C28SH-N3ILwywzSrYEAq19HNqIZU~3shXu6KeRPWb1OOSqtEJPbGc-AAi~r~AfPYqj0W~Ax~9NcG4h~dYPzCONrIH2k88E1LOTmnVbifi44YINbSP4yC-rPqKRQJ20PBebph4Ht37psyCaanVC0Z2e8Z1k3KS7OKPmWQpR6A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide SNP-based molecular diversity among 66 wild and domesticated chickpea accessions. (a) Unrooted phylogram depicting the genetic relationships (Nei’s genetic distance) among 66 wild (R1-10), desi (D1-28) and kabuli (K1-28) chickpea accessions based on genome-wide SNP mapped on C. reticulatum genome assembly. The phylogenetic tree differentiated 66 accessions into three diverse groups. (b) The population genetic structure of the wild and domesticated chickpea accessions. The mapped genetic markers assigned to three distinct desi, kabuli and wild population groups at population number (K = 3). The accessions represented by vertical bars along the horizontal axis were classified into K colour segments based on their estimated membership fraction in each K cluster. The colour version of the figure is available online. " path-from-xml="dsw042f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462679/dsw042f4.tif" path-from-xml="dsw042f4.tif" target="_blank">View large</a><a section="61462679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042f4.png?Expires=1509239074&amp;Signature=GYJpcHo~F0Y3M62txKbr83CD4ashMBb19KjFipAOCJrHv1~lkbg-XSF~IB938NtjOs9B1ZOgD2drpatZzfRsWwoKGk0h7ouPaaWobGRnWdKgHPva1EJzehcnZHwD-K6OCTaKCkQL~kJzenrni4nFIxdL~~B6HvqHYi0U1F1WTJHJ3yz6dTbBAsM-NfE0LoLyY8or0r5wpj7XDbHv2epbziKLgpJKeopGQZ~d5PaUpn33yqQ4262grszUKsBddPGb4wYKmvmTH3NMz~FQcWvN5O7BYScXSq5fgrjzdGEO~GJiKh5icCkCrM~CFmin0vvXJA7mWo93NqBu8PjmHwPNAw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61462679&amp;ar=2669389&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw042f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide SNP-based molecular diversity among 66 wild and domesticated chickpea accessions. (a) Unrooted phylogram depicting the genetic relationships (Nei’s genetic distance) among 66 wild (R1-10), <em>desi</em> (D1-28) and <em>kabuli</em> (K1-28) chickpea accessions based on genome-wide SNP mapped on <em>C. reticulatum</em> genome assembly. The phylogenetic tree differentiated 66 accessions into three diverse groups. (b) The population genetic structure of the wild and domesticated chickpea accessions. The mapped genetic markers assigned to three distinct <em>desi</em>, <em>kabuli</em> and wild population groups at population number (K = 3). The accessions represented by vertical bars along the horizontal axis were classified into K colour segments based on their estimated membership fraction in each K cluster. The colour version of the figure is available online. </p></div></div></p>
<p>The population genetic structure was determined among the 66 cultivated and wild accessions using the physically mapped SNPs. The optimization of population structure at varying levels of K (K = 2–10) with 20 replications revealed that at K value of 3, all the 66 cultivated and wild accessions were classified majorly into three distinct populations, wild, <em>kabuli</em> and <em>desi</em> as per expected pedigree relationships and parentage (<a reveal-id="dsw042-F4" data-open="dsw042-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>). This was agreed-well with the clustering pattern as observed among 66 chickpea accessions by the phylogenetic tree analysis using pair-wise genetic distances (<a reveal-id="dsw042-F4" data-open="dsw042-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>). Based on population structure analysis, the population group I consisting of 10 accessions of wild <em>C. reticulatum</em>, group II with 26 and 4 accessions of <em>kabuli</em> and <em>desi</em>, respectively and group III having 24 and 2 accessions of <em>desi</em> and <em>kabuli</em>, respectively. The molecular genetic variation among and within the three populations based on the SNPs exhibited a wider level of quantitative genetic differentiation (F<sub>ST</sub> varied from 0.17 to 0.54 with an average of 0.38) among three population groups. Among the three population groups, the divergence was maximum between group II (<em>kabuli</em>) and group I (wild) (F<sub>ST</sub> = 0.47) and minimum between group III (<em>desi</em>) and group II (<em>kabuli</em>) (F<sub>ST</sub> = 0.15). The proportion of F<sub>ST</sub> and thus diversity between the population groups was higher as compared to that estimated within populations. Higher genetic differentiation was observed in <em>kabuli</em> than that of <em>desi</em>. All the 6 cultivated chickpea and 10 wild accessions clearly belonged to a single population in which about 81.3% of their inferred ancestry was derived from one of the model-based population and remaining ∼18.7% contained admixed ancestry. Maximum admixture (∼12%) was observed between group II (<em>kabuli</em>) and group III (<em>desi</em>) populations followed by group III (<em>desi</em>) and group I (wild) (∼4%) and group II (<em>kabuli</em>) and wild (∼2%) populations (<a reveal-id="dsw042-F4" data-open="dsw042-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>).</p>                            <h2 scrollto-destination=61462681 id="61462681" class="section-title" >DISCUSSION</h2>
<p>Wild relatives of chickpea are underexploited resource for introducing genomic variation in the domesticated species, which is suffering from a narrow genetic base. The draft genome assembly of the inter-crossable wild progenitor <em>C. reticulatum</em> would provide an excellent resource for studying allelic variation and domestication of chickpea. The assembled pseudomolecules of eight linkage groups in the present draft assembly are equivalent in length to those in the published assemblies of the cultivated species<a reveal-id="dsw042-B1" data-open="dsw042-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw042-B3" data-open="dsw042-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and were constructed with larger sequence scaffolds. Almost the whole gene spaces of the respective species have been covered in the corresponding assemblies and, therefore, the analyses presented in this report including the resource of polymorphic SSRs and SNPs between the wild and the domesticated species would provide a significant resource for genome-wide large-scale genotyping applications. The reference-based assemblies produced by mapping <em>C. reticulatum</em> reads on two chickpea draft genome assemblies could be used for direct comparison between the wild and the domesticated species. Domestication process of chickpea followed a different course from the other crops domesticated in Fertile Crescent.<a reveal-id="dsw042-B4" data-open="dsw042-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Geographical distributions of two domesticated chickpea types are distinctly separate. While the <em>kabuli</em> type is restricted mostly in the Western Mediterranean, the <em>desi</em> type is mostly grown in the Central Asia and Indian subcontinent. It is possible that the domestication traits were fixed before the evolution of agriculture and subsequent selection based on local adaptation occurred during domestication. Comparative analysis showed <em>desi</em> type chickpea genome was relatively closer to the wild as compared to the <em>kabuli</em> type. However, at present, selection of <em>kabuli</em> type from <em>desi</em> type chickpea<a reveal-id="dsw042-B50" data-open="dsw042-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> appears unlikely because of extent of genomic variations between this two types considering separation of only a few thousand years. Additionally, molecular diversity analysis clearly separates these two types. Therefore, an early separation predating agriculture followed by domestication-associated selection according to local adaptability may provide an alternative hypothesis of chickpea domestication process.</p><p>Detection of a wider molecular diversity and genetic base using genome-wide SNPs would be much relevant in establishing distinctness and selection of desirable plant types for varietal improvement in chickpea. As expected, the correspondence of clustering pattern obtained among the accessions belonging to <em>desi</em>, <em>kabuli</em> and wild chickpea groups with the pedigree relationship and parentage further suggests that the molecular diversity assessed by genome-wide SNPs is realistic and thus would be useful in chickpea genomics-assisted breeding. The admixed ancestry among three populations might be due to their complex breeding history involving inter-crossing and introgression among <em>desi</em>, <em>kabuli</em> and wild chickpea accessions along with strong selection pressure and evolutionary bottleneck during chickpea domestication.<a reveal-id="dsw042-B4" data-open="dsw042-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw042-B51" data-open="dsw042-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Higher admixture (∼5%) among 2 wild <em>C. reticulatum</em> accessions (ILWC 247 and ILWC 242) and 1 each of <em>desi</em> (ICC 16374) and <em>kabuli</em> (ICC 11847) accessions originated from Syria is expected due to involvement of <em>C. reticulatum</em> as one of the parents in cross-breeding program for their varietal improvement. This reflects more influence of parentage/geographical origination rather than cultivar-specific classification especially in higher admixed 6 <em>desi</em> and <em>kabuli</em> and 3 wild <em>C. reticulatum</em> accessions. Collectively, the genome assembly and annotation of the wild progenitor of chickpea, and the comparative analysis with the domesticated species would increase opportunities for more strategic use of the wild germplasms for improved crop management.</p>                            <h2 scrollto-destination=61462684 id="61462684" class="backacknowledgements-title" >ACKNOWLEDGEMENTS</h2>
<p>We acknowledge funding from the National Institute of Plant Genome Research, New Delhi, India. Technical suggestion from Dr. Saulo Aflitos is highly acknowledged.</p>                            <h3 scrollto-destination=61462686 id="61462686" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=61462688 id="61462688" class="section-title" >Accession numbers</h3>
<p>Genome assembly is available at National Centre for Biotechnology Information (NCBI) as Bioproject ID PRJNA263699 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/bioproject?term=" target="_blank">http://www.ncbi.nlm.nih.gov/bioproject?term=</a>PRJNA263699). The sequence data are available in Sequence Read Archive under accession numbers SRR160365, SRR1640444, SRR1640451 and SRR1640455. Genome assembly, annotation data and all the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary figures, tables</a></span>, data are available for viewing and downloading at nipgr.res.in/WCGAP.</p>                            <h3 scrollto-destination=61462690 id="61462690" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239074&Signature=BZDupAAA4HHC~oLW4nb0YvUlyW1Gahc2wdT-oF0RNamkYe3Q6Kxm~5qtRwKIL6OXCaEJ2-mUv~CNb60IZPsFw2ne-6I53QNTkb7TXRVKa3rH-Be0Y7zuqzMZnFsZJWAaXEzkgBjSz9pPpbUhKhV3-V82kDgeSqx1Wv98p9BDtXaL5wU-uggC-SdgrYLwS7WQ~RRl7C4xISB031Vz66Spe9Uu8KDMpAsWGLmrENwRpCfDV9iGQOXSjZXsO0UQVWyHr2FEyQdTUmycGLZNUb56jH9y5DGSBfUT1BIfAqk1tqXqAk98tmE2lv3EKN9Z24WvvooxU1A1ygSHIFC5QkR4dQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=61462692 id="61462692" class="section-title" >Funding</h3>
<p>This study was financially supported with a core grant from the National Institute of Plant Genome Research, New Delhi, India.</p>                            <h2 scrollto-destination=61462694 id="61462694" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw042-b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">R. K.</div></div><div class="name string-name"><div class="surname">Song</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">R. K.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence of chickpea (<em>Cicer arietinum</em>) provides a resource for trait improvement,</div> <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">240</div>–<div class="lpage">246</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement%2C&amp;author=R.%20K.%20Varshney&amp;author=C.%20Song&amp;author=R.%20K.%20Saxena&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=240-246" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2491" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23354103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jain</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Misra</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Patel</div> <div class="given-names">R. K.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">A draft genome sequence of the pulse crop chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Plant J</div> ., <div class="volume">74</div>, <div class="fpage">715</div>–<div class="lpage">729</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20genome%20sequence%20of%20the%20pulse%20crop%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=M.%20Jain&amp;author=G.%20Misra&amp;author=R.%20K.%20Patel&amp;publication_year=2013&amp;journal=Plant%20J&amp;volume=74&amp;pages=715-729" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.74.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.74.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.74.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23489434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parween</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Nawaz</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Roy</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">An advanced draft genome assembly of a desi type chickpea (<em>Cicer arietinum</em> L.),</div> <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">12806</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20advanced%20draft%20genome%20assembly%20of%20a%20desi%20type%20chickpea%20(Cicer%20arietinum%20L.)%2C&amp;author=S.%20Parween&amp;author=K.%20Nawaz&amp;author=R.%20Roy&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=12806" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep12806" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep12806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep12806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26259924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Berger</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Turner</div> <div class="given-names">N.C.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Evolution of cultivated chickpea: four bottlenecks limit diversity and constrain adaptation</div>, <div class="source">Funct. Plant Biol</div> ., <div class="volume">30</div>, <div class="fpage">1081</div>–<div class="lpage">1087</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20cultivated%20chickpea%3A%20four%20bottlenecks%20limit%20diversity%20and%20constrain%20adaptation&amp;author=S.%20Abbo&amp;author=J.%20Berger&amp;author=N.C.%20Turner&amp;publication_year=2003&amp;journal=Funct.%20Plant%20Biol&amp;volume=30&amp;pages=1081-1087" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/FP03084" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FFP03084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FFP03084"> </span></div> </div></div></div></div></div><div content-id="dsw042-b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harlan</div> <div class="given-names">J. R.</div></div></div> <div class="year">1971</div>, 
<div class="article-title">Agricultural origins: centers and noncenters,</div> <div class="source">Science</div> , <div class="volume">174</div>, <div class="fpage">468</div>–<div class="lpage">474</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Agricultural%20origins%3A%20centers%20and%20noncenters%2C&amp;author=J.%20R.%20Harlan&amp;publication_year=1971&amp;journal=Science&amp;volume=174&amp;pages=468-474" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.174.4008.468" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.174.4008.468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.174.4008.468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17745730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ladinsky</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Adler</div> <div class="given-names">A.</div></div></div> <div class="year">1976</div>, 
<div class="article-title">The origin of chickpea <em>Cicer arietinum</em> L</div>., <div class="source">Euphytica</div> , <div class="volume">25</div>, <div class="fpage">211</div>–<div class="lpage">217</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20chickpea%20Cicer%20arietinum%20L&amp;author=G.%20Ladinsky&amp;author=A.%20Adler&amp;publication_year=1976&amp;journal=Euphytica&amp;volume=25&amp;pages=211-217" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00041547" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00041547" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00041547"> </span></div> </div></div></div></div></div><div content-id="dsw042-b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Shtienberg</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Lichtenzveig</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Lev-Yadun</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Gopher</div> <div class="given-names">A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">The chickpea, summer cropping, and a new model for pulse domestication in the ancient near East,</div> <div class="source">Quant. Rev. Biol</div> ., <div class="volume">78</div>, <div class="fpage">435</div>–<div class="lpage">448</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chickpea%2C%20summer%20cropping%2C%20and%20a%20new%20model%20for%20pulse%20domestication%20in%20the%20ancient%20near%20East%2C&amp;author=S.%20Abbo&amp;author=D.%20Shtienberg&amp;author=J.%20Lichtenzveig&amp;author=S.%20Lev-Yadun&amp;author=A.%20Gopher&amp;publication_year=2003&amp;journal=Quant.%20Rev.%20Biol&amp;volume=78&amp;pages=435-448" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/378927" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F378927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F378927"> </span></div> </div></div></div></div></div><div content-id="dsw042-b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Redden</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Berger</div> <div class="given-names">J.</div></div></div> <div class="year">2007</div>, History and origion of chickpea, In: <div class="source">Chickpea Breeding and Management</div>  (<div class="person-group"><div class="name string-name"><div class="surname">Yadav</div> <div class="given-names">S. S.</div></div><div class="name string-name"><div class="surname">Redden</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">B.</div></div></div>, eds.). 
<div class="publisher-name">CAB International</div>, 
<div class="publisher-loc">Wallingford</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.</div></div></div></div><div content-id="dsw042-b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Improving plant breeding with exotic genetic libraries,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">2</div>, <div class="fpage">983</div>–<div class="lpage">989</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20plant%20breeding%20with%20exotic%20genetic%20libraries%2C&amp;author=D.%20Zamir&amp;publication_year=2001&amp;journal=Nat.%20Rev.%20Genet&amp;volume=2&amp;pages=983-989" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35103590" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35103590" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35103590"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11733751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J. F.</div></div><div class="name string-name"><div class="surname">Jeong</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Park</div> <div class="given-names">S. Y.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Genome sequence of the polymyxin-producing plant-probiotic rhizobacterium <em>Paenibacillus polymyxa</em> E681,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">192</div>, <div class="fpage">6103</div>–<div class="lpage">6104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20polymyxin-producing%20plant-probiotic%20rhizobacterium%20Paenibacillus%20polymyxa%20E681%2C&amp;author=J.%20F.%20Kim&amp;author=H.%20Jeong&amp;author=S.%20Y.%20Park&amp;publication_year=2010&amp;journal=J.%20Bacteriol&amp;volume=192&amp;pages=6103-6104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00983-10" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00983-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00983-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20851896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Scossa</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">M. E.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">The genome of the stress-tolerant wild tomato species <em>Solanum pennellii</em>,</div> <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">1034</div>–<div class="lpage">1038</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20stress-tolerant%20wild%20tomato%20species%20Solanum%20pennellii%2C&amp;author=A.%20Bolger&amp;author=F.%20Scossa&amp;author=M.%20E.%20Bolger&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=1034-1038" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3046" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25064008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winter</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Pfaff</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Udupa</div> <div class="given-names">S. M.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Characterization and mapping of sequence-tagged microsatellite sites in the chickpea (<em>Cicer arietinum</em> L.) genome</div>. <div class="source">Mol. Gen Genet</div> ., <div class="volume">262</div>, <div class="fpage">90</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20mapping%20of%20sequence-tagged%20microsatellite%20sites%20in%20the%20chickpea%20(Cicer%20arietinum%20L.)%20genome&amp;author=P.%20Winter&amp;author=T.%20Pfaff&amp;author=S.%20M.%20Udupa&amp;publication_year=1999&amp;journal=Mol.%20Gen%20Genet&amp;volume=262&amp;pages=90-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380051063" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380051063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380051063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10503540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">H. C.</div></div><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">K.K.</div></div><div class="name string-name"><div class="surname">Crouch</div> <div class="given-names">J.H.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Genetic transformation of crops for insect resistance: potential and limitations,</div> <div class="source">Crit. Rev. Plant Sci</div> ., <div class="volume">23</div>, <div class="fpage">47</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20transformation%20of%20crops%20for%20insect%20resistance%3A%20potential%20and%20limitations%2C&amp;author=H.%20C.%20Sharma&amp;author=K.K.%20Sharma&amp;author=J.H.%20Crouch&amp;publication_year=2004&amp;journal=Crit.%20Rev.%20Plant%20Sci&amp;volume=23&amp;pages=47-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07352680490273400" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07352680490273400" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07352680490273400"> </span></div> </div></div></div></div></div><div content-id="dsw042-b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Azam</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Jeena</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (<em>Cicer arietinum</em> L.),</div> <div class="source">DNA Res.</div>  <div class="volume">19</div>, <div class="fpage">357</div>–<div class="lpage">373</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20genotyping%20for%20constructing%20a%20saturated%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)%2C&amp;author=R.%20Gaur&amp;author=S.%20Azam&amp;author=G.%20Jeena&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=357-373" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss018" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khajuria</div> <div class="given-names">Y. P.</div></div><div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">M. S.</div></div><div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Development and integration of genome-wide polymorphic microsatellite markers onto a reference linkage map for constructing a high-density genetic map of chickpea,</div> <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0125583</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20integration%20of%20genome-wide%20polymorphic%20microsatellite%20markers%20onto%20a%20reference%20linkage%20map%20for%20constructing%20a%20high-density%20genetic%20map%20of%20chickpea%2C&amp;author=Y.%20P.%20Khajuria&amp;author=M.%20S.%20Saxena&amp;author=R.%20Gaur&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0125583" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0125583" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0125583" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0125583"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25974327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Patel</div> <div class="given-names">R. K.</div></div><div class="name string-name"><div class="surname">Jain</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">NGS QC Toolkit: a toolkit for quality control of next generation sequencing data,</div> <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e30619</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NGS%20QC%20Toolkit%3A%20a%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data%2C&amp;author=R.%20K.%20Patel&amp;author=M.%20Jain&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30619" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030619" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22312429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,</div> <div class="source">GigaScience</div> , <div class="volume">1</div>:<div class="fpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler%2C&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=GigaScience&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zimin</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Puiu</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Yorke</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The MaSuRCA genome assembler</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">2669</div>–<div class="lpage">2677</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MaSuRCA%20genome%20assembler&amp;author=A.%20Zimin&amp;author=G.%20Mar%C3%A7ais&amp;author=D.%20Puiu&amp;author=M.%20Roberts&amp;author=S.%20Salzberg&amp;author=J.%20Yorke&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=2669-2677" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt476" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt476" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt476"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23990416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gnerre</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Maccallum</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Przybylski</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">1513</div>–<div class="lpage">1518</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-quality%20draft%20assemblies%20of%20mammalian%20genomes%20from%20massively%20parallel%20sequence%20data%2C&amp;author=S.%20Gnerre&amp;author=I.%20Maccallum&amp;author=D.%20Przybylski&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=1513-1518" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1017351108" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1017351108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1017351108"> </span></div> </div></div></div></div></div><div content-id="dsw042-b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Jeena</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Shah</div> <div class="given-names">N.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">High density linkage mapping of genomic and transcriptomic SNPs for synteny analysis and anchoring the genome sequence of chickpea,</div> <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">13387</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20density%20linkage%20mapping%20of%20genomic%20and%20transcriptomic%20SNPs%20for%20synteny%20analysis%20and%20anchoring%20the%20genome%20sequence%20of%20chickpea%2C&amp;author=R.%20Gaur&amp;author=G.%20Jeena&amp;author=N.%20Shah&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=13387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep13387" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep13387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep13387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26303721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W. J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT-the BLAST-like alignment tool,</div> <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">664</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT-the%20BLAST-like%20alignment%20tool%2C&amp;author=W.%20J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">1760</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform%2C&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-1760" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">N.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Using RepeatMasker to identify repetitive elements in genomic sequences</div>. <div class="source">Curr. Protoc. Bioinform</div> ., <div class="volume">25</div>, <div class="fpage">4.10.1</div>–<div class="lpage">4.10.14</div>.</div></div></div></div><div content-id="dsw042-b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R. C.</div></div><div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">PILER: identification and classification of genomic repeats,</div> <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">52</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PILER%3A%20identification%20and%20classification%20of%20genomic%20repeats%2C&amp;author=R.%20C.%20Edgar&amp;author=E.%20W.%20Myers&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=52-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1003"> </span></div> </div></div></div></div></div><div content-id="dsw042-b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Price</div> <div class="given-names">A. L.</div></div><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N. C.</div></div><div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P. A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title"><em>De novo</em> identification of repeat families in large genomes,</div> <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">351</div>–<div class="lpage">358</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes%2C&amp;author=A.%20L.%20Price&amp;author=N.%20C.%20Jones&amp;author=P.%20A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=351-358" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div> </div></div></div></div></div><div content-id="dsw042-b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons,</div> <div class="source">Nucl. Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">265</div>–<div class="lpage">268</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LTR_FINDER%3A%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons%2C&amp;author=Z.%20Xu&amp;author=H.%20Wang&amp;publication_year=2007&amp;journal=Nucl.%20Acids%20Res&amp;volume=35&amp;pages=265-268" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm286" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm286"> </span></div> </div></div></div></div></div><div content-id="dsw042-b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">M. A.</div></div><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div><div class="name string-name"><div class="surname">Hamilton</div> <div class="given-names">J. P.</div></div><div class="name string-name"><div class="surname">Mount</div> <div class="given-names">S. M.</div></div><div class="name string-name"><div class="surname">Buell</div> <div class="given-names">C. R.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Comprehensive analysis of alternative splicing in rice and comparative analyses with Arabidopsis,</div> <div class="source">BMC Genomics</div> , <div class="volume">7</div>, <div class="fpage">327</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20analysis%20of%20alternative%20splicing%20in%20rice%20and%20comparative%20analyses%20with%20Arabidopsis%2C&amp;author=M.%20A.%20Campbell&amp;author=B.%20J.%20Haas&amp;author=J.%20P.%20Hamilton&amp;author=S.%20M.%20Mount&amp;author=C.%20R.%20Buell&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-327" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17194304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Priya</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Garg</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Parida</div> <div class="given-names">S. K.</div></div><div class="name string-name"><div class="surname">Tyagi</div> <div class="given-names">A. K.</div></div><div class="name string-name"><div class="surname">Jain</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Transcriptome sequencing of wild chickpea as a rich resource for marker development,</div> <div class="source">Plant Biotechnol. J</div> ., <div class="volume">10</div>, <div class="fpage">690</div>–<div class="lpage">702</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20wild%20chickpea%20as%20a%20rich%20resource%20for%20marker%20development%2C&amp;author=S.%20Jhanwar&amp;author=P.%20Priya&amp;author=R.%20Garg&amp;author=S.%20K.%20Parida&amp;author=A.%20K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J&amp;volume=10&amp;pages=690-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00712.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00712.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00712.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22672127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments,</div> <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20program%20to%20assemble%20spliced%20alignments%2C&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Schein</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Birol</div> <div class="given-names">I.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Circos: an information aesthetic for comparative genomics,</div> <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1639</div>–<div class="lpage">1645</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics%2C&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1639-1645" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Enright</div> <div class="given-names">A. J.</div></div><div class="name string-name"><div class="surname">Van Dongen</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ouzounis</div> <div class="given-names">C. A.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">An efficient algorithm for large-scale detection of protein families,</div> <div class="source">Nucl. Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">1575</div>–<div class="lpage">1584</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20algorithm%20for%20large-scale%20detection%20of%20protein%20families%2C&amp;author=A.%20J.%20Enright&amp;author=S.%20Van%20Dongen&amp;author=C.%20A.%20Ouzounis&amp;publication_year=2002&amp;journal=Nucl.%20Acids%20Res&amp;volume=30&amp;pages=1575-1584" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.7.1575" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.7.1575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.7.1575"> </span></div> </div></div></div></div></div><div content-id="dsw042-b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soderlund</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Bomhoff</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">W. M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">SyMAP v3.4: a turnkey synteny system with application to plant genomes,</div> <div class="source">Nucl. Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">e68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SyMAP%20v3.4%3A%20a%20turnkey%20synteny%20system%20with%20application%20to%20plant%20genomes%2C&amp;author=C.%20Soderlund&amp;author=M.%20Bomhoff&amp;author=W.%20M.%20Nelson&amp;publication_year=2011&amp;journal=Nucl.%20Acids%20Res&amp;volume=39&amp;pages=e68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr123" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr123"> </span></div> </div></div></div></div></div><div content-id="dsw042-b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Debarry</div> <div class="given-names">J. D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity,</div> <div class="source">Nucl. Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity%2C&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20Debarry&amp;publication_year=2012&amp;journal=Nucl.%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1293"> </span></div> </div></div></div></div></div><div content-id="dsw042-b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chenna</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Koike</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">Multiple sequence alignment with the Clustal series of programs,</div> <div class="source">Nucl. Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">3497</div>–<div class="lpage">3500</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20sequence%20alignment%20with%20the%20Clustal%20series%20of%20programs%2C&amp;author=R.%20Chenna&amp;author=H.%20Sugawara&amp;author=T.%20Koike&amp;publication_year=2003&amp;journal=Nucl.%20Acids%20Res&amp;volume=31&amp;pages=3497-3500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg500"> </span></div> </div></div></div></div></div><div content-id="dsw042-b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suyama</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Torrents</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments</div>. <div class="source">Nucl. Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W609</div>–<div class="lpage">W612</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAL2NAL%3A%20robust%20conversion%20of%20protein%20sequence%20alignments%20into%20the%20corresponding%20codon%20alignments&amp;author=M.%20Suyama&amp;author=D.%20Torrents&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Nucl.%20Acids%20Res&amp;volume=34&amp;pages=W609-W612" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl315" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl315"> </span></div> </div></div></div></div></div><div content-id="dsw042-b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">1591</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood%2C&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-1591" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Conery</div> <div class="given-names">J. S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The evolutionary fate and consequences of duplicate genes,</div> <div class="source">Science</div> , <div class="volume">290</div>, <div class="fpage">1151</div>–<div class="lpage">1155</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20fate%20and%20consequences%20of%20duplicate%20genes%2C&amp;author=M.%20Lynch&amp;author=J.%20S.%20Conery&amp;publication_year=2000&amp;journal=Science&amp;volume=290&amp;pages=1151-1155" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.290.5494.1151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.290.5494.1151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11073452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Stacks: building and genotyping loci de novo from short-read sequences,</div> <div class="source">Genes Genomes Genet</div> ., <div class="volume">1</div>, <div class="fpage">171</div>–<div class="lpage">182</div>.</div></div></div></div><div content-id="dsw042-b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kujur</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bajaj</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Upadhyaya</div> <div class="given-names">H. D.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Employing genome-wide SNP discovery and genotyping strategy to extrapolate the natural allelic diversity and domestication patterns in chickpea,</div> <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">162</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Employing%20genome-wide%20SNP%20discovery%20and%20genotyping%20strategy%20to%20extrapolate%20the%20natural%20allelic%20diversity%20and%20domestication%20patterns%20in%20chickpea%2C&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=H.%20D.%20Upadhyaya&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2015.00162" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2015.00162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25873920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcais</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">770</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers%2C&amp;author=G.%20Marcais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-770" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>. <div class="source">Bioinformatics</div> ,<div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">1067</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-1067" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cesari</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Kanzaki</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Fujiwara</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">The NB-LRR proteins RGA4 and RGA5 interact functionally and physically to confer disease resistance,</div> <div class="source">EMBO J</div> ., <div class="volume">33</div>, <div class="fpage">1941</div>–<div class="lpage">1959</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20NB-LRR%20proteins%20RGA4%20and%20RGA5%20interact%20functionally%20and%20physically%20to%20confer%20disease%20resistance%2C&amp;author=S.%20Cesari&amp;author=H.%20Kanzaki&amp;author=T.%20Fujiwara&amp;publication_year=2014&amp;journal=EMBO%20J&amp;volume=33&amp;pages=1941-1959" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.15252/embj.201487923" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.15252%2Fembj.201487923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.15252%2Fembj.201487923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25024433" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peters</div> <div class="given-names">B. J.</div></div><div class="name string-name"><div class="surname">Ash</div> <div class="given-names">G. J.</div></div><div class="name string-name"><div class="surname">Cother</div> <div class="given-names">E. J.</div></div><div class="name string-name"><div class="surname">Hailstones</div> <div class="given-names">D. L.</div></div><div class="name string-name"><div class="surname">Noble</div> <div class="given-names">D. H.</div></div><div class="name string-name"><div class="surname">Urwin</div> <div class="given-names">N. A. R.</div></div></div> <div class="year">2004</div>, 
<div class="article-title"><em>Pseudomonas syringae</em> pv. <em>maculicola</em> in Australia: pathogenic, phenotypic and genetic diversity,</div> <div class="source">Plant Pathol</div> ., <div class="volume">53</div>, <div class="fpage">73</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pseudomonas%20syringae%20pv.%20maculicola%20in%20Australia%3A%20pathogenic%2C%20phenotypic%20and%20genetic%20diversity%2C&amp;author=B.%20J.%20Peters&amp;author=G.%20J.%20Ash&amp;author=E.%20J.%20Cother&amp;author=D.%20L.%20Hailstones&amp;author=D.%20H.%20Noble&amp;author=N.%20A.%20R.%20Urwin&amp;publication_year=2004&amp;journal=Plant%20Pathol&amp;volume=53&amp;pages=73-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppa.2004.53.issue-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppa.2004.53.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppa.2004.53.issue-1"> </span></div> </div></div></div></div></div><div content-id="dsw042-b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Whitham</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Dinesh-Kumar</div> <div class="given-names">S. P.</div></div><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Hehl</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Corr</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Baker</div> <div class="given-names">B.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">The product of the tobacco mosaic virus resistance gene N: similarity to toll and the interleukin-1 receptor,</div> <div class="source">Cell</div> , <div class="volume">78</div>, <div class="fpage">1101</div>–<div class="lpage">1115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20product%20of%20the%20tobacco%20mosaic%20virus%20resistance%20gene%20N%3A%20similarity%20to%20toll%20and%20the%20interleukin-1%20receptor%2C&amp;author=S.%20Whitham&amp;author=S.%20P.%20Dinesh-Kumar&amp;author=D.%20Choi&amp;author=R.%20Hehl&amp;author=C.%20Corr&amp;author=B.%20Baker&amp;publication_year=1994&amp;journal=Cell&amp;volume=78&amp;pages=1101-1115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(94)90283-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(94)90283-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(94)90283-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7923359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">O'Connell</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Maekawa-Yoshikawa</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Uemura</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Neumann</div> <div class="given-names">U.</div></div><div class="name string-name"><div class="surname">Schulze-Lefert</div> <div class="given-names">P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The powdery mildew resistance protein RPW8.2 is carried on VAMP721/722 vesicles to the extrahaustorial membrane of haustorial complexes,</div> <div class="source">Plant J</div> ., <div class="volume">79</div>, <div class="fpage">835</div>–<div class="lpage">847</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20powdery%20mildew%20resistance%20protein%20RPW8.2%20is%20carried%20on%20VAMP721%2F722%20vesicles%20to%20the%20extrahaustorial%20membrane%20of%20haustorial%20complexes%2C&amp;author=H.%20Kim&amp;author=R.%20O'Connell&amp;author=M.%20Maekawa-Yoshikawa&amp;author=T.%20Uemura&amp;author=U.%20Neumann&amp;author=P.%20Schulze-Lefert&amp;publication_year=2014&amp;journal=Plant%20J&amp;volume=79&amp;pages=835-847" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2014.79.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2014.79.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2014.79.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24941879" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Agarwal</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Priya</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Comparative analysis of kabuli chickpea transcriptome with desi and wild chickpea provides a rich resource for development of functional markers,</div> <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e52443</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20kabuli%20chickpea%20transcriptome%20with%20desi%20and%20wild%20chickpea%20provides%20a%20rich%20resource%20for%20development%20of%20functional%20markers%2C&amp;author=G.%20Agarwal&amp;author=S.%20Jhanwar&amp;author=P.%20Priya&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e52443" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0052443" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0052443"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Lev-Yadun</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Galwey</div> <div class="given-names">N.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Vernalization response of wild chickpea,</div> <div class="source">New Phytol</div> ., <div class="volume">154</div>, <div class="fpage">695</div>–<div class="lpage">701</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vernalization%20response%20of%20wild%20chickpea%2C&amp;author=S.%20Abbo&amp;author=S.%20Lev-Yadun&amp;author=N.%20Galwey&amp;publication_year=2002&amp;journal=New%20Phytol&amp;volume=154&amp;pages=695-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1469-8137.2002.00405.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1469-8137.2002.00405.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1469-8137.2002.00405.x"> </span></div> </div></div></div></div></div><div content-id="dsw042-b48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sung</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Amasino</div> <div class="given-names">R. M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Vernalization in <em>Arabidopsis thaliana</em> is mediated by the PHD finger protein VIN3,</div> <div class="source">Nature</div> , <div class="volume">427</div>, <div class="fpage">159</div>–<div class="lpage">164</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vernalization%20in%20Arabidopsis%20thaliana%20is%20mediated%20by%20the%20PHD%20finger%20protein%20VIN3%2C&amp;author=S.%20Sung&amp;author=R.%20M.%20Amasino&amp;publication_year=2004&amp;journal=Nature&amp;volume=427&amp;pages=159-164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02195" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14712276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gendall</div> <div class="given-names">A. R.</div></div><div class="name string-name"><div class="surname">Levy</div> <div class="given-names">Y. Y.</div></div><div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Dean</div> <div class="given-names">C.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">The VERNALIZATION 2 gene mediates the epigenetic regulation of vernalization in Arabidopsis,</div> <div class="source">Cell</div> , <div class="volume">107</div>, <div class="fpage">525</div>–<div class="lpage">535</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20VERNALIZATION%202%20gene%20mediates%20the%20epigenetic%20regulation%20of%20vernalization%20in%20Arabidopsis%2C&amp;author=A.%20R.%20Gendall&amp;author=Y.%20Y.%20Levy&amp;author=A.%20Wilson&amp;author=C.%20Dean&amp;publication_year=2001&amp;journal=Cell&amp;volume=107&amp;pages=525-535" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(01)00573-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(01)00573-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(01)00573-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11719192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw042-b50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moreno</div> <div class="given-names">M. T.</div></div><div class="name string-name"><div class="surname">Cubero</div> <div class="given-names">C. I.</div></div></div> <div class="year">1978</div>, 
<div class="article-title">Variation in <em>Cicer arietinum</em> L.,</div> <div class="source">Euphytica</div> , <div class="volume">27</div>, <div class="fpage">465</div>–<div class="lpage">485</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20Cicer%20arietinum%20L.%2C&amp;author=M.%20T.%20Moreno&amp;author=C.%20I.%20Cubero&amp;publication_year=1978&amp;journal=Euphytica&amp;volume=27&amp;pages=465-485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00043173" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00043173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00043173"> </span></div> </div></div></div></div></div><div content-id="dsw042-b51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw042-b51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berger</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Turner</div> <div class="given-names">N. C.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Ecogeography of annual wild <em>Cicer</em> species: the parlous state of the world collection,</div> <div class="source">Crop Sci</div> ., <div class="volume">43</div>, <div class="fpage">1076</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ecogeography%20of%20annual%20wild%20Cicer%20species%3A%20the%20parlous%20state%20of%20the%20world%20collection%2C&amp;author=J.%20Berger&amp;author=S.%20Abbo&amp;author=N.%20C.%20Turner&amp;publication_year=2003&amp;journal=Crop%20Sci&amp;volume=43&amp;pages=1076" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2003.1076" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2003.1076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2003.1076"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2669389&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw042/1/dsw042_Supp.pdf?Expires=1509239075&amp;Signature=O06nOPCGeQ05DY0XKRRbXMu~VNxPFfZakcCbSpeUs~z-MhK5QSYY2Eaxawj9q~uv4Cfulf3Hn0gkzGR70qxKJOiX3KzXhA61IdQeydQzcgiJZeRPCJp3PdxM0agNlWAjKCdAQ4W2jEiTOL478j5RsoMIk2uR-h~PkVPpv~UcWgQVtHv5rx24uiyBdwYYV~bx47l5ToiSCzG1S6xXbHcyh5CKaTzmm9pvvb01GZvu4DtnJUibBufo90MCxIRqK0CUtV2zaaLtNfmW7YG-UcPCTowmJWtfWiqja1knk4nZK08mFWN08jr8RFl3sNDFWzJxdCKQZ-N5v1HCqBRp2fSzGQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">569</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsw042"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2669389"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f1%2f2669389">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000397051500001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F1%2F2669389" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2669389'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Draft genome sequence of Cicer reticulatum L., the wild progenitor of chickpea provides a resource for agronomic trait improvement" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f1%2f2669389">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

