

<!DOCTYPE html>
<html lang="en">

<head><title>
	Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/59/2709598","@type":"ScholarlyArticle","name":"Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines","datePublished":"2016-12-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw048","keywords":["SINE","mammalian-wide interspersed repeats","RNA polymerase III","RNA-Seq","ENCODE"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/59/2709598","author":[{"name":"Carnevali, Davide","affiliation":"1 Department of Life Sciences, University of Parma, Parma, Italy","@type":"Person"},{"name":"Conti, Anastasia","affiliation":"1 Department of Life Sciences, University of Parma, Parma, Italy","@type":"Person"},{"name":"Pellegrini, Matteo","affiliation":"2 Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, CA 90095 723, USA","@type":"Person"},{"name":"Dieci, Giorgio","affiliation":"1 Department of Life Sciences, University of Parma, Parma, Italy","@type":"Person"}],"description":"With more than 500,000 copies, mammalian-wide interspersed repeats (MIRs), a sub-group of SINEs, represent ∼2.5% of the human genome and one of the most numerous family of potential targets for the RNA polymerase (Pol) III transcription machinery. Since MIR elements ceased to amplify ∼130 myr ago, previous studies primarily focused on their genomic impact, while the issue of their expression has not been extensively addressed. We applied a dedicated bioinformatic pipeline to ENCODE RNA-Seq datasets of seven human cell lines and, for the first time, we were able to define the Pol III-driven MIR transcriptome at single-locus resolution. While the majority of Pol III-transcribed MIR elements are cell-specific, we discovered a small set of ubiquitously transcribed MIRs mapping within Pol II-transcribed genes in antisense orientation that could influence the expression of the overlapping gene. We also identified novel Pol III-transcribed ncRNAs, deriving from transcription of annotated MIR fragments flanked by unique MIR-unrelated sequences, and confirmed the role of Pol III-specific internal promoter elements in MIR transcription. Besides demonstrating widespread transcription at these retrotranspositionally inactive elements in human cells, the ability to profile MIR expression at single-locus resolution will facilitate their study in different cell types and states including pathological alterations.","pageStart":"59","pageEnd":"69","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217484&Signature=bU0BzLZTxVF80OE~WGR0tKqWvE3HDuzXO3RUN7E~RROg1L--vmrCicTT6wJGf2c~hV0VY2G02v9EBioDtRup3NLvq-85J0ffqAgsD9njPndQRNuwWXHpM6W~Bd6f3j9H2uvddEfAOfeMefXOd1-gfNKU2gfRY5BsyYoMKtL05nx9roKLteV-OdDBurPBe9QnnRNeIPWAc4AH4dPAEAiyXPcI~rm3rOWlX3OKP05wAQR~iKyQKSOpyv5UYWictUR2utQf0wlA1k7G19xL2X0hKcaEoAtKJwOi6SGiuPrA-HhOfb7aABp8VmrKqeTd465i1s3FcUFgJX9LgQ0qsDFlWg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217484&Signature=bU0BzLZTxVF80OE~WGR0tKqWvE3HDuzXO3RUN7E~RROg1L--vmrCicTT6wJGf2c~hV0VY2G02v9EBioDtRup3NLvq-85J0ffqAgsD9njPndQRNuwWXHpM6W~Bd6f3j9H2uvddEfAOfeMefXOd1-gfNKU2gfRY5BsyYoMKtL05nx9roKLteV-OdDBurPBe9QnnRNeIPWAc4AH4dPAEAiyXPcI~rm3rOWlX3OKP05wAQR~iKyQKSOpyv5UYWictUR2utQf0wlA1k7G19xL2X0hKcaEoAtKJwOi6SGiuPrA-HhOfb7aABp8VmrKqeTd465i1s3FcUFgJX9LgQ0qsDFlWg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines | DNA Research | Oxford Academic" />
    <meta property="og:description" content="With more than 500,000 copies, mammalian-wide interspersed repeats (MIRs), a sub-group of SINEs, represent ∼2.5% of the human genome and one of the most numerous family of potential targets for the RNA polymerase (Pol) III transcription machinery. Since MIR elements ceased to amplify ∼130 myr ago, previous studies primarily focused on their genomic impact, while the issue of their expression has not been extensively addressed. We applied a dedicated bioinformatic pipeline to ENCODE RNA-Seq datasets of seven human cell lines and, for the first time, we were able to define the Pol III-driven MIR transcriptome at single-locus resolution. While the majority of Pol III-transcribed MIR elements are cell-specific, we discovered a small set of ubiquitously transcribed MIRs mapping within Pol II-transcribed genes in antisense orientation that could influence the expression of the overlapping gene. We also identified novel Pol III-transcribed ncRNAs, deriving from transcription of annotated MIR fragments flanked by unique MIR-unrelated sequences, and confirmed the role of Pol III-specific internal promoter elements in MIR transcription. Besides demonstrating widespread transcription at these retrotranspositionally inactive elements in human cells, the ability to profile MIR expression at single-locus resolution will facilitate their study in different cell types and states including pathological alterations." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/59/2709598" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217484&amp;Signature=faH6oWPUS6M4UXfd~19CBrIKe6qf8KS5qRwhJAptMp~HBRuGJpqgQkAN5ig4J7ZmpP3jMors1PEQAH3W9UcZeiv4QuJ9QTzrrvtFS4xHoBegyLvIy9RfktGaYR3gfFzKoE3UqFzckKqFvBIKlSPHElcn06nGUxsyvZ-hZWVjOg6LRMCRvkP8OZkNKTmqWLXqKW1AUQrYwMJtPL77cAR79yzB0eoNiQHvZaiZQStPqYlpTwGh1igVtg-S9n6vP-YeOJQoaiq-Fa8mSDLisuuIcQBlmrRv0Tufqt1AoLeazSKV1hc6YL5WUQPFwIX0dMfIE7NljzH7-En1IVrvGT5eow__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2016-12-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Carnevali, Davide" /><meta name="citation_author_institution" content="Department of Life Sciences, University of Parma, Parma, Italy" /><meta name="citation_author" content="Conti, Anastasia" /><meta name="citation_author_institution" content="Department of Life Sciences, University of Parma, Parma, Italy" /><meta name="citation_author" content="Pellegrini, Matteo" /><meta name="citation_author_institution" content="Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, CA 90095 723, USA" /><meta name="citation_author" content="Dieci, Giorgio" /><meta name="citation_author_institution" content="Department of Life Sciences, University of Parma, Parma, Italy" /><meta name="citation_title" content="Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines" /><meta name="citation_firstpage" content="59" /><meta name="citation_lastpage" content="69" /><meta name="citation_doi" content="10.1093/dnares/dsw048" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Ubiquitous mammalian-wide interspersed repeats (MIRs) are molecular fossils from the mesozoic era,;  citation_journal_title=Nucleic Acids Res;  citation_year=1995;  citation_volume=23;  citation_pages=170-5; " /><meta name="citation_reference" content="citation_title=MIRs are classic, tRNA-derived SINEs that amplified before the mammalian radiation,;  citation_journal_title=Nucleic Acids Res;  citation_year=1995;  citation_volume=23;  citation_pages=98-102; " /><meta name="citation_reference" content="citation_title=SINEBase: a database and tool for SINE analysis;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D83-9; " /><meta name="citation_reference" content="citation_title=Alu repeats and human genomic diversity,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2002;  citation_volume=3;  citation_pages=370-9; " /><meta name="citation_reference" content="citation_title=SINE cousins: the 3'-end tails of the two oldest and distantly related families of SINEs are descended from the 3' ends of LINEs with the same genealogical origin,;  citation_journal_title=Mol. Biol. Evol;  citation_year=1998;  citation_volume=15;  citation_pages=1460-71; " /><meta name="citation_reference" content="citation_title=CORE-SINEs: eukaryotic short interspersed retroposing elements with common sequence motifs,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=2869-74; " /><meta name="citation_reference" content="citation_title=Evolution of MIR Elements Located in the Coding Regions of Human Genome;  citation_journal_title=Mol Biol (Mosk);  citation_year=2001;  citation_volume=35;  citation_pages=1023-31; " /><meta name="citation_reference" content="citation_title=Gene function and expression level influence the insertion/fixation dynamics of distinct transposon families in mammalian introns;  citation_journal_title=Genome Biol;  citation_year=2006;  citation_volume=7;  citation_pages=R120" /><meta name="citation_reference" content="citation_title=Mammalian-wide interspersed repeat (MIR)-derived enhancers and the regulation of human gene expression,;  citation_journal_title=Mob. DNA;  citation_year=2014;  citation_volume=5;  citation_pages=14" /><meta name="citation_reference" content="citation_title=Recruitment of RNA polymerase III to its target promoters,;  citation_journal_title=Genes Dev;  citation_year=2002;  citation_volume=16;  citation_pages=2593-620; " /><meta name="citation_reference" content="citation_title=Defining the RNA polymerase III transcriptome: Genome-wide localization of the RNA polymerase III transcription machinery in human cells,;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=710-21; " /><meta name="citation_reference" content="citation_title=Human RNA polymerase III transcriptomes and relationships to Pol II promoter chromatin and enhancer-binding factors,;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2010;  citation_volume=17;  citation_pages=620-8; " /><meta name="citation_reference" content="citation_title=Genomic binding of Pol III transcription machinery and relationship with TFIIS transcription factor distribution in mouse embryonic stem cells,;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=270-83; " /><meta name="citation_reference" content="citation_title=A multiplicity of factors contributes to selective RNA polymerase III occupancy of a subset of RNA polymerase III genes in mouse liver,;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=666-80; " /><meta name="citation_reference" content="citation_title=MIR retrotransposon sequences provide insulators to the human genome,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=E4428-37; " /><meta name="citation_reference" content="citation_title=Landscape of transcription in human cells,;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=489;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=Identification of RNA polymerase III-transcribed Alu loci by computational screening of RNA-Seq data,;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=817-35; " /><meta name="citation_reference" content="citation_title=Accurate transcription initiation by RNA polymerase II in a soluble extract from isolated mammalian nuclei,;  citation_journal_title=Nucleic Acids Res;  citation_year=1983;  citation_volume=11;  citation_pages=1475-89; " /><meta name="citation_reference" content="citation_title=Widespread occurrence of non-canonical transcription termination by human RNA polymerase III,;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=5499-12; " /><meta name="citation_reference" content="citation_title=Genomic binding profiles of functionally distinct RNA polymerase III transcription complexes in human cells,;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2010;  citation_volume=17;  citation_pages=635-40; " /><meta name="citation_reference" content="citation_title=Pol II and its associated epigenetic marks are present at Pol III-transcribed noncoding RNA genes,;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2010;  citation_volume=17;  citation_pages=629-34; " /><meta name="citation_reference" content="citation_title=RNA polymerase III transcription control elements: themes and variations,;  citation_journal_title=Gene;  citation_year=2012;  citation_volume=493;  citation_pages=185-94; " /><meta name="citation_reference" content="citation_title=A minimal promoter for TFIIIC-dependent in vitro transcription of snoRNA and tRNA genes by RNA polymerase III,;  citation_journal_title=J. Biol. Chem;  citation_year=2006;  citation_volume=281;  citation_pages=23945-57; " /><meta name="citation_reference" content="citation_title=Close association of RNA polymerase II and many transcription factors with Pol III genes,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=3639-44; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of transcription and possible functions of mammalian short interspersed elements, SINEs,;  citation_journal_title=Genes Genet Syst;  citation_year=2013;  citation_volume=88;  citation_pages=19-9; " /><meta name="citation_reference" content="citation_title=LINEs, SINEs and other retroelements: do birds of a feather flock together?,;  citation_journal_title=Front Biosci. (Landmark Ed);  citation_year=2012;  citation_volume=17;  citation_pages=1345-61; " /><meta name="citation_reference" content="citation_title=SINE transcription by RNA polymerase III is suppressed by histone methylation but not by DNA methylation,;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=6569" /><meta name="citation_reference" content="citation_title=Alu expression profiles as a novel RNA signature in biology and disease;  citation_journal_title=RNA Dis;  citation_year=2015;  citation_volume=2;  citation_pages=e735" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/59/10731831/dsw048.pdf" /><meta name="description" content="Abstract.  With more than 500,000 copies, mammalian-wide interspersed repeats (MIRs), a sub-group of SINEs, represent ∼2.5% of the human genome and one of the m" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/59/2709598" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/59/2709598" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsw048';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2709598" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF+Uc8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhcm5ldmFsaSwgRGF2aWRlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFBhcm1hLCBQYXJtYSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb250aSwgQW5hc3Rhc2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFBhcm1hLCBQYXJtYSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZWxsZWdyaW5pLCBNYXR0ZW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciwgQ2VsbCwgYW5kIERldmVsb3BtZW50YWwgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBDYWxpZm9ybmlhLCBMb3MgQW5nZWxlcywgQ0EgOTAwOTUgNzIzLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEaWVjaSwgR2lvcmdpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBQYXJtYSwgUGFybWEsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJXaG9sZS1nZW5vbWUgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiBtYW1tYWxpYW4td2lkZSBpbnRlcnNwZXJzZWQgcmVwZWF0IGVsZW1lbnRzIGluIGh1bWFuIGNlbGwgbGluZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVViaXF1aXRvdXMgbWFtbWFsaWFuLXdpZGUgaW50ZXJzcGVyc2VkIHJlcGVhdHMgKE1JUnMpIGFyZSBtb2xlY3VsYXIgZm9zc2lscyBmcm9tIHRoZSBtZXNvem9pYyBlcmEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE3MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NSVJzIGFyZSBjbGFzc2ljLCB0Uk5BLWRlcml2ZWQgU0lORXMgdGhhdCBhbXBsaWZpZWQgYmVmb3JlIHRoZSBtYW1tYWxpYW4gcmFkaWF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz05OC0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNJTkVCYXNlOiBhIGRhdGFiYXNlIGFuZCB0b29sIGZvciBTSU5FIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQ4My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHUgcmVwZWF0cyBhbmQgaHVtYW4gZ2Vub21pYyBkaXZlcnNpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTM3MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TSU5FIGNvdXNpbnM6IHRoZSAzJy1lbmQgdGFpbHMgb2YgdGhlIHR3byBvbGRlc3QgYW5kIGRpc3RhbnRseSByZWxhdGVkIGZhbWlsaWVzIG9mIFNJTkVzIGFyZSBkZXNjZW5kZWQgZnJvbSB0aGUgMycgZW5kcyBvZiBMSU5FcyB3aXRoIHRoZSBzYW1lIGdlbmVhbG9naWNhbCBvcmlnaW4sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNDYwLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DT1JFLVNJTkVzOiBldWthcnlvdGljIHNob3J0IGludGVyc3BlcnNlZCByZXRyb3Bvc2luZyBlbGVtZW50cyB3aXRoIGNvbW1vbiBzZXF1ZW5jZSBtb3RpZnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTI4NjktNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBNSVIgRWxlbWVudHMgTG9jYXRlZCBpbiB0aGUgQ29kaW5nIFJlZ2lvbnMgb2YgSHVtYW4gR2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wgQmlvbCAoTW9zayk7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xMDIzLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGZ1bmN0aW9uIGFuZCBleHByZXNzaW9uIGxldmVsIGluZmx1ZW5jZSB0aGUgaW5zZXJ0aW9uL2ZpeGF0aW9uIGR5bmFtaWNzIG9mIGRpc3RpbmN0IHRyYW5zcG9zb24gZmFtaWxpZXMgaW4gbWFtbWFsaWFuIGludHJvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1SMTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFtbWFsaWFuLXdpZGUgaW50ZXJzcGVyc2VkIHJlcGVhdCAoTUlSKS1kZXJpdmVkIGVuaGFuY2VycyBhbmQgdGhlIHJlZ3VsYXRpb24gb2YgaHVtYW4gZ2VuZSBleHByZXNzaW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9iLiBETkE7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjcnVpdG1lbnQgb2YgUk5BIHBvbHltZXJhc2UgSUlJIHRvIGl0cyB0YXJnZXQgcHJvbW90ZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MjU5My02MjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmluaW5nIHRoZSBSTkEgcG9seW1lcmFzZSBJSUkgdHJhbnNjcmlwdG9tZTogR2Vub21lLXdpZGUgbG9jYWxpemF0aW9uIG9mIHRoZSBSTkEgcG9seW1lcmFzZSBJSUkgdHJhbnNjcmlwdGlvbiBtYWNoaW5lcnkgaW4gaHVtYW4gY2VsbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NzEwLTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBSTkEgcG9seW1lcmFzZSBJSUkgdHJhbnNjcmlwdG9tZXMgYW5kIHJlbGF0aW9uc2hpcHMgdG8gUG9sIElJIHByb21vdGVyIGNocm9tYXRpbiBhbmQgZW5oYW5jZXItYmluZGluZyBmYWN0b3JzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTYyMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGJpbmRpbmcgb2YgUG9sIElJSSB0cmFuc2NyaXB0aW9uIG1hY2hpbmVyeSBhbmQgcmVsYXRpb25zaGlwIHdpdGggVEZJSVMgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZGlzdHJpYnV0aW9uIGluIG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0yNzAtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbXVsdGlwbGljaXR5IG9mIGZhY3RvcnMgY29udHJpYnV0ZXMgdG8gc2VsZWN0aXZlIFJOQSBwb2x5bWVyYXNlIElJSSBvY2N1cGFuY3kgb2YgYSBzdWJzZXQgb2YgUk5BIHBvbHltZXJhc2UgSUlJIGdlbmVzIGluIG1vdXNlIGxpdmVyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTY2Ni04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUlSIHJldHJvdHJhbnNwb3NvbiBzZXF1ZW5jZXMgcHJvdmlkZSBpbnN1bGF0b3JzIHRvIHRoZSBodW1hbiBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz1FNDQyOC0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFuZHNjYXBlIG9mIHRyYW5zY3JpcHRpb24gaW4gaHVtYW4gY2VsbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg5OyAgY2l0YXRpb25fcGFnZXM9MTAxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFJOQSBwb2x5bWVyYXNlIElJSS10cmFuc2NyaWJlZCBBbHUgbG9jaSBieSBjb21wdXRhdGlvbmFsIHNjcmVlbmluZyBvZiBSTkEtU2VxIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTgxNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgdHJhbnNjcmlwdGlvbiBpbml0aWF0aW9uIGJ5IFJOQSBwb2x5bWVyYXNlIElJIGluIGEgc29sdWJsZSBleHRyYWN0IGZyb20gaXNvbGF0ZWQgbWFtbWFsaWFuIG51Y2xlaSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTQ3NS04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBvY2N1cnJlbmNlIG9mIG5vbi1jYW5vbmljYWwgdHJhbnNjcmlwdGlvbiB0ZXJtaW5hdGlvbiBieSBodW1hbiBSTkEgcG9seW1lcmFzZSBJSUksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTU0OTktMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgYmluZGluZyBwcm9maWxlcyBvZiBmdW5jdGlvbmFsbHkgZGlzdGluY3QgUk5BIHBvbHltZXJhc2UgSUlJIHRyYW5zY3JpcHRpb24gY29tcGxleGVzIGluIGh1bWFuIGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTYzNS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9sIElJIGFuZCBpdHMgYXNzb2NpYXRlZCBlcGlnZW5ldGljIG1hcmtzIGFyZSBwcmVzZW50IGF0IFBvbCBJSUktdHJhbnNjcmliZWQgbm9uY29kaW5nIFJOQSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz02MjktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBwb2x5bWVyYXNlIElJSSB0cmFuc2NyaXB0aW9uIGNvbnRyb2wgZWxlbWVudHM6IHRoZW1lcyBhbmQgdmFyaWF0aW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkzOyAgY2l0YXRpb25fcGFnZXM9MTg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1pbmltYWwgcHJvbW90ZXIgZm9yIFRGSUlJQy1kZXBlbmRlbnQgaW4gdml0cm8gdHJhbnNjcmlwdGlvbiBvZiBzbm9STkEgYW5kIHRSTkEgZ2VuZXMgYnkgUk5BIHBvbHltZXJhc2UgSUlJLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yODE7ICBjaXRhdGlvbl9wYWdlcz0yMzk0NS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvc2UgYXNzb2NpYXRpb24gb2YgUk5BIHBvbHltZXJhc2UgSUkgYW5kIG1hbnkgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIHdpdGggUG9sIElJSSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTM2MzktNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiB0cmFuc2NyaXB0aW9uIGFuZCBwb3NzaWJsZSBmdW5jdGlvbnMgb2YgbWFtbWFsaWFuIHNob3J0IGludGVyc3BlcnNlZCBlbGVtZW50cywgU0lORXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lcyBHZW5ldCBTeXN0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9MTktOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TElORXMsIFNJTkVzIGFuZCBvdGhlciByZXRyb2VsZW1lbnRzOiBkbyBiaXJkcyBvZiBhIGZlYXRoZXIgZmxvY2sgdG9nZXRoZXI/LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQgQmlvc2NpLiAoTGFuZG1hcmsgRWQpOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTM0NS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U0lORSB0cmFuc2NyaXB0aW9uIGJ5IFJOQSBwb2x5bWVyYXNlIElJSSBpcyBzdXBwcmVzc2VkIGJ5IGhpc3RvbmUgbWV0aHlsYXRpb24gYnV0IG5vdCBieSBETkEgbWV0aHlsYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NjU2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdSBleHByZXNzaW9uIHByb2ZpbGVzIGFzIGEgbm92ZWwgUk5BIHNpZ25hdHVyZSBpbiBiaW9sb2d5IGFuZCBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEgRGlzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lNzM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzEvNTkvMTA3MzE4MzEvZHN3MDQ4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXaXRoIG1vcmUgdGhhbiA1MDAsMDAwIGNvcGllcywgbWFtbWFsaWFuLXdpZGUgaW50ZXJzcGVyc2VkIHJlcGVhdHMgKE1JUnMpLCBhIHN1Yi1ncm91cCBvZiBTSU5FcywgcmVwcmVzZW50IOKIvDIuNSUgb2YgdGhlIGh1bWFuIGdlbm9tZSBhbmQgb25lIG9mIHRoZSBtIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNzA5NTk4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgQWxsIEZpZ3VyZXNkZE8HyzONB5htU15ONXVLbdmKZBGg" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f59%2f2709598" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f59%2f2709598" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1509239084&amp;Signature=aLWDBixemHXKeu3p6McZmXZgXFz7xf7WLelwCCjdY~8udFvL8S~eId77UuZY5VBw0ysRWKx2B5waz613KnE0Z0gblzm28j5V8p0Rb5arIDU-Eg7sHtDEcIWGicykl4m08WcbPfF7JSQFpmh0HbcvUonRgVzC~0gC5kCp6~TugOdweXMyLwAHqyibS5VZcuMPcIrUQc4cDTH~8oG6PnCU-Mq~x~p00FnOgNgmaRDk-zmyhqKKpDJIyYQF4aHN5Y3HQxlNd3BcCwsGzk9VcEjR~sy2wigEIqZpfWD-1o0dCoy9PT8p-WiwywssxMqIeRDLdy~Qq6m8CeqNRKZYRvq4ug__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61463427">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463427">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463429">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463429">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463435">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463435">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463447">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463447">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463477">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463477">Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463488">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463488">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463494">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463494">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/51/2709597">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/71/2726856">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Davide Carnevali</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Davide Carnevali</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Life Sciences, University of Parma, Parma, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Davide+Carnevali">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Carnevali D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Carnevali author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Anastasia Conti</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anastasia Conti</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Life Sciences, University of Parma, Parma, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Anastasia+Conti">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Conti A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Conti author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Matteo Pellegrini</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matteo Pellegrini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Molecular, Cell, and Developmental Biology, University of California, Los Angeles, CA 90095 723, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Matteo+Pellegrini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pellegrini M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Pellegrini author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Giorgio Dieci</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Giorgio Dieci</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Life Sciences, University of Parma, Parma, Italy</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw048-cor1" data-open="dsw048-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +39 0521 905649. Fax. +39 0521 905151. Email: <a href="mailto:giorgio.dieci@unipr.it" target="_blank">giorgio.dieci@unipr.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Giorgio+Dieci">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dieci G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Dieci author:G.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw048-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: The San Raffaele Telethon Institute for Gene Therapy (SR-TIGET), Milano, Italy</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw048-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Minoru Ko</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 59–69, <a href='https://doi.org/10.1093/dnares/dsw048'>https://doi.org/10.1093/dnares/dsw048</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 December 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 August 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 October 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2709598" href="/dnaresearch/article-pdf/24/1/59/10731831/dsw048.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Davide Carnevali, Anastasia Conti, Matteo Pellegrini, Giorgio Dieci;  Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 59–69, <a href='https://doi.org/10.1093/dnares/dsw048'>https://doi.org/10.1093/dnares/dsw048</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2709598?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Whole-genome%20expression%20analysis%20of%20mammalian-wide%20interspersed%20repeat%20elements%20in%20human%20cell%20lines&amp;publicationDate=2016-12-17&amp;volumeNum=24&amp;issueNum=1&amp;author=Carnevali%2C%20Davide%3B%20Conti%2C%20Anastasia&amp;startPage=59&amp;endPage=69&amp;contentId=10.1093%2Fdnares%2Fdsw048&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f59%2f2709598" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61463427 id="61463427" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>With more than 500,000 copies, mammalian-wide interspersed repeats (MIRs), a sub-group of SINEs, represent ∼2.5% of the human genome and one of the most numerous family of potential targets for the RNA polymerase (Pol) III transcription machinery. Since MIR elements ceased to amplify ∼130 myr ago, previous studies primarily focused on their genomic impact, while the issue of their expression has not been extensively addressed. We applied a dedicated bioinformatic pipeline to ENCODE RNA-Seq datasets of seven human cell lines and, for the first time, we were able to define the Pol III-driven MIR transcriptome at single-locus resolution. While the majority of Pol III-transcribed MIR elements are cell-specific, we discovered a small set of ubiquitously transcribed MIRs mapping within Pol II-transcribed genes in antisense orientation that could influence the expression of the overlapping gene. We also identified novel Pol III-transcribed ncRNAs, deriving from transcription of annotated MIR fragments flanked by unique MIR-unrelated sequences, and confirmed the role of Pol III-specific internal promoter elements in MIR transcription. Besides demonstrating widespread transcription at these retrotranspositionally inactive elements in human cells, the ability to profile MIR expression at single-locus resolution will facilitate their study in different cell types and states including pathological alterations.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="SINE">SINE</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mammalian-wide interspersed repeats&quot;">mammalian-wide interspersed repeats</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA polymerase III&quot;">RNA polymerase III</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-Seq">RNA-Seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ENCODE">ENCODE</a></div>                            <h2 scrollto-destination=61463429 id="61463429" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Mammalian-wide Interspersed Repeats (MIRs) represent an ancient family of tRNA-derived Short INterspersed Elements (SINEs) found in all mammalian genomes, whose amplification seems to have ceased in the ancestors of placental mammals.<a reveal-id="dsw048-B1 dsw048-B2 dsw048-B3" data-open="dsw048-B1 dsw048-B2 dsw048-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> This is in contrast with <em>Alu</em> elements, the most abundant SINEs, which are still transpositionally active.<a reveal-id="dsw048-B4" data-open="dsw048-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> It is thought that MIRs may have arisen following the fusion of a tRNA molecule with the 3′-end of an existing Long INterspersed Element (LINE).<a reveal-id="dsw048-B5" data-open="dsw048-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The complete MIR element is about 260 bp in length and comprises a tRNA-related 5′ head, a 70-bp conserved central domain containing a 15-bp core sequence, and a LINE-related sequence located at the 3′-end<a reveal-id="dsw048-B5" data-open="dsw048-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> (<a reveal-id="dsw048-F1" data-open="dsw048-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). In an earlier study first describing MIRs, two segments overlapping with the LINE-related region were described as separate interspersed repeats MER24 and DBR.<a reveal-id="dsw048-B2" data-open="dsw048-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><div data-id="dsw048-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f1.png?Expires=1509239085&Signature=L6p7TjTPFURmqXmefIKILPnFjEaMk1qSqfVv50VG54f~~TErcQlqPuDy6bXnHLmz5QuU6gtBANwmykZ68quB0phxUjjSX6LJYdKmEXQP2-5yzXdPS8g3BhdNaUQWlVrHNUcJRuUoKT7fiIXTqtlbqWRPJfXxamjDiluO7495NodCuX65lt0MIaZ~J2gb0cR0peb2kyRw1rl6BBpM2TefNSm1eO~i4nl0LYxXSEq0L7pwwz3gUwNZnDr~JqmDyIuKhmYtxT2SkityANJhc2biIlunP2aGpMpHEtDTON4q4A-JjsGnGFLsUYBZXO6THeEb3YSOOYPOzCmyPb4jqIU-YA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Representation of the structure of a mammalian-wide interspersed repeat (MIR). A tRNA-related region contains A- and B-box promoter elements driving Pol III transcription by being recognized by TFIIIC. Core-SINE indicates a highly conserved central sequence, followed by a LINE-related region. Pol III is expected to terminate at the first encountered termination signal (Tn) which may be located at varying distances from the end of the MIR body." path-from-xml="dsw048f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463430/dsw048f1.tif" path-from-xml="dsw048f1.tif" target="_blank">View large</a><a section="61463430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f1.png?Expires=1509239085&Signature=JtRmKuU8212e7Q7WO~90pspN8FJ11Ue5UzzwxgAEZ4DHFRepJu3Huayl2hPR~nRttZ8LMvotTAMvbCom3Cmxhkz37vHyo5J5IpbggAYkiFji0W4P84MHVRkWG1DSu4FJDxb6UeyUlylerPNZckTI2mOwtuQhsxGbE4NiNZZqBCKYCMoxF08pd9fGOwXoeFcKJ9oymPqp2SQEIftdKtTKTEM1PMH7kltceRhztT7MiErl8l47~9QupVOXn2xelKHYTub5l70Hf7-hF2y5hy4n2Qcmb~u1IvpBJomMgMQDil5-sfeezdZdeh8kjzi6n~h2CH6AOCzvk6PQYkvGJPpbLQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463430&ar=2709598&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw048f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation of the structure of a mammalian-wide interspersed repeat (MIR). A tRNA-related region contains A- and B-box promoter elements driving Pol III transcription by being recognized by TFIIIC. Core-SINE indicates a highly conserved central sequence, followed by a LINE-related region. Pol III is expected to terminate at the first encountered termination signal (T<em>n</em>) which may be located at varying distances from the end of the MIR body.</p></div></div><div content-id="dsw048-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f1.png?Expires=1509239085&Signature=L6p7TjTPFURmqXmefIKILPnFjEaMk1qSqfVv50VG54f~~TErcQlqPuDy6bXnHLmz5QuU6gtBANwmykZ68quB0phxUjjSX6LJYdKmEXQP2-5yzXdPS8g3BhdNaUQWlVrHNUcJRuUoKT7fiIXTqtlbqWRPJfXxamjDiluO7495NodCuX65lt0MIaZ~J2gb0cR0peb2kyRw1rl6BBpM2TefNSm1eO~i4nl0LYxXSEq0L7pwwz3gUwNZnDr~JqmDyIuKhmYtxT2SkityANJhc2biIlunP2aGpMpHEtDTON4q4A-JjsGnGFLsUYBZXO6THeEb3YSOOYPOzCmyPb4jqIU-YA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Representation of the structure of a mammalian-wide interspersed repeat (MIR). A tRNA-related region contains A- and B-box promoter elements driving Pol III transcription by being recognized by TFIIIC. Core-SINE indicates a highly conserved central sequence, followed by a LINE-related region. Pol III is expected to terminate at the first encountered termination signal (Tn) which may be located at varying distances from the end of the MIR body." path-from-xml="dsw048f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463430/dsw048f1.tif" path-from-xml="dsw048f1.tif" target="_blank">View large</a><a section="61463430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f1.png?Expires=1509239085&Signature=JtRmKuU8212e7Q7WO~90pspN8FJ11Ue5UzzwxgAEZ4DHFRepJu3Huayl2hPR~nRttZ8LMvotTAMvbCom3Cmxhkz37vHyo5J5IpbggAYkiFji0W4P84MHVRkWG1DSu4FJDxb6UeyUlylerPNZckTI2mOwtuQhsxGbE4NiNZZqBCKYCMoxF08pd9fGOwXoeFcKJ9oymPqp2SQEIftdKtTKTEM1PMH7kltceRhztT7MiErl8l47~9QupVOXn2xelKHYTub5l70Hf7-hF2y5hy4n2Qcmb~u1IvpBJomMgMQDil5-sfeezdZdeh8kjzi6n~h2CH6AOCzvk6PQYkvGJPpbLQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463430&ar=2709598&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw048f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation of the structure of a mammalian-wide interspersed repeat (MIR). A tRNA-related region contains A- and B-box promoter elements driving Pol III transcription by being recognized by TFIIIC. Core-SINE indicates a highly conserved central sequence, followed by a LINE-related region. Pol III is expected to terminate at the first encountered termination signal (T<em>n</em>) which may be located at varying distances from the end of the MIR body.</p></div></div></p><p>MIR elements were actively propagating prior to the radiation of mammals and before placental mammals separated. For this reason their age was originally estimated to be ∼130 million years (myr) even if it has been suggested that the CORE-SINE<a reveal-id="dsw048-B6" data-open="dsw048-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> may have originated ∼550 myr ago due to the remarkable similarity between Ther-1 (the MIR consensus in placental mammals which corresponds to that revealed earlier in humans) and the OR2 SINE of octopuses.<a reveal-id="dsw048-B7" data-open="dsw048-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Intriguingly, there are observations suggesting that the core region may serve some general function in mammalian genomes, because the level of sequence conservation is higher than the 3′ and 5′-flanking sequences.<a reveal-id="dsw048-B8" data-open="dsw048-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p><p>In the human genome, there are more than 500,000 annotated MIRs.<a reveal-id="dsw048-B9" data-open="dsw048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Based on their sequence similarity, they have been grouped into 4 subfamilies named MIR, MIRb, MIRc, and MIR3. Like all SINEs, MIRs are thought to be transcribed by the RNA Polymerase III (Pol III) machinery, with the assembly factor TFIIIC recognizing the A- and B-box internal control regions within the tRNA-derived portion of the element.<a reveal-id="dsw048-B2" data-open="dsw048-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> As well established for tRNA gene promoters, once bound TFIIIC recruits TFIIIB [composed of Brf1, Bdp1 and the TATA box-binding protein (TBP)], which in turn recruits Pol III.<a reveal-id="dsw048-B10" data-open="dsw048-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The first experimental verifications of MIRs as Pol III targets in the human genome have come from the results of genome-wide location analyses of the Pol III machinery in human<a reveal-id="dsw048-B11" data-open="dsw048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw048-B12" data-open="dsw048-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and mouse cells.<a reveal-id="dsw048-B13" data-open="dsw048-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Interestingly, one of these studies revealed that, in human immortalized fibroblasts, the Pol III machinery is consistently associated with a MIR located in the first intron of the POLR3E gene, coding for a specific subunit of Pol III (RPC5). A significant enrichment of components of the Pol III machinery was also observed at a dozen other MIRs, thus supporting the notion that MIRs, although transpositionally inactive, can undergo autonomous transcription.<a reveal-id="dsw048-B11" data-open="dsw048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw048-B13" data-open="dsw048-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>Unlike <em>Alu</em>s that, although non-autonomous in retrotransposition, nevertheless exploit the LINE-encoded machinery to amplify in the genome, MIRs have ceased to amplify by retrotransposition ∼130 Myr ago. This could be due to many reasons, including for example the inactivation of L2 (the partner LINE for MIR elements) or mutations in LINE-related regions of MIRs. Nevertheless, it can be speculated that since part of these elements are still transcriptionally active at least in the human and mouse genome,<a reveal-id="dsw048-B11" data-open="dsw048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> changes in the 3′ tail due to sequence mutations could potentially rescue the retrotransposition potential of some of them. Recent studies have shown that enhancers are a platform from which MIRs can exert a regulatory function in the human genome<a reveal-id="dsw048-B9" data-open="dsw048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and that MIRs can also serve as insulators acting as chromatin barriers or enhancer-blocking elements.<a reveal-id="dsw048-B15" data-open="dsw048-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Nothing is known, however, about the possible involvement of MIR expression in their role at enhancers/insulators. In general, in spite of some advancement in our knowledge of how the massive presence of MIRs impacts genome biology, their contribution to the human transcriptome is still largely unexplored, as are their transcription mechanism and regulation. In particular, the <em>cis</em>-acting elements controlling MIR transcription have never been systematically studied.</p><p>In this study, by applying a recently developed bioinformatics pipeline to ENCODE RNA-seq data, we define for the first time the MIR expression profiles of human cells, and we investigate on the <em>in cis</em> requirements for MIR transcription by biochemical analysis <em>in vitro</em>.</p>                            <h2 scrollto-destination=61463435 id="61463435" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=61463436 id="61463436" class="section-title" >2.1. Bioinformatic pipeline for MIR expression</h3>
<p>For MIR RNA identification we used the Cold Spring Harbor Laboratory (CSHL) long RNA-seq data within ENCODE<a reveal-id="dsw048-B16" data-open="dsw048-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> of seven human cell lines (GM12878, H1-hESC, HeLa-S3, HepG2, HUVEC, K562, and NHEK). The RNA-Seq reads were aligned to the human reference genome GRCh37/hg19 as previously described.<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> PolyA+ and PolyA- datasets (bam files) were merged to streamline the analyses. We were not interested in the differences between these two features because MIR RNAs usually do not have a poly(A) tail.</p>
  <p>The bam files from each dataset, containing RNA-Seq reads aligned to the reference genome (GRCh37/hg19) using TopHat aligner, along with the annotated MIRs, were submitted to a Python script that performs the identification of individual MIR transcripts. The script first builds stranded coverage vectors for the whole genome, using the bam file supplied and only considering uniquely mapped reads (tag NH:i:1 in the bam file). Then, for each annotated MIR having an expression coverage value over a calculated background noise threshold, the script calculates the coordinates of the corresponding expected full-length consensus element (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span>), to take into account the fact that many of the annotated MIR elements are truncated. Finally a flanking region filter is applied to the identified expected full-length element, as described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span>, in order to exclude false positives arising from MIR elements embedded in Pol II transcripts. The filter is designed to do this by imposing a significantly lower expression coverage value to the flanking regions immediately upstream and downstream of the expected full-length SINE, thus discriminating between genuine SINE RNAs and those that are ‘passengers’ of longer Pol II transcripts or part of their trailers extending downstream of their annotated 3′UTRs. <a reveal-id="dsw048-F2" data-open="dsw048-F2" class="link link-reveal link-table xref-fig">Figure 2</a> shows a schematic representation of the pipeline, which is improved with respect to the one previously employed to distinguish genuine from ‘passenger’ <em>Alu</em> RNAs<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> allowing us to control each step and to identify both <em>Alu</em> and MIR transcripts of the SINE class of retrotransposons. <div data-id="dsw048-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f2.png?Expires=1509239085&amp;Signature=Lx6R8ZknQ9tSp862ZXbAUSEHorepxxYHHWKPxXnXHQIcncMTNHourtsZZAkumqPEpfGL7EUHwj~J-8Dw2feXy2RojMc6MMETqRPuxcsrWSZqsayapHW1xx03bhpmvdxu1FoL~2yoq39-eHyJDgvo~C43ZHWpDEPNTDMa-ecz1mj9eG6nCnEs~NLvLoF-3SPObAPMdQGE224DDGs1fRGDc4aiMwAHxWbtXMZnvHtx3wz~NMv1OTIfgMT48O~X5pSZfVAnmkHwp7pmFp9Kim3dPwxexXKeiK1-Dqo8bg5sAeoTaawthP6n7F8dL-uA5m1tzaNbOuwy597FTluJkNHYvQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Bioinformatic pipeline flowchart. Shown is a flow diagram of the improved bioinformatic pipeline for the identification of autonomously expressed SINE loci from RNA-seq data sets. See text and Supplementary Methods for details." path-from-xml="dsw048f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463438/dsw048f2.tif" path-from-xml="dsw048f2.tif" target="_blank">View large</a><a section="61463438" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f2.png?Expires=1509239085&amp;Signature=CGUKU-iSzn-flVUG87h0asUwZfLsHJ5EpRf1PA9JJhcLt2VFyKdW~3cPdFg6qHpSQyWadhvCJ7pHGtKI5vlq4k56tbrmQIvhuI~uPh0x1nrhV5L10Xg0VrYxXBPEiXGEsadptgwDpbSSBC95TFxwIpFBIXc90PJj4BerdmG5x4l1r9XOvAMKL33rJdIg6I53swM9W6WqPSAJpmDLna7~bPtrs~xdaDfMegqVPZjVof-f~CgmIgScQPtDJapv3NJikh8-cmizTxo8KVwHKHD2BIsdF~i0JFJNPnVy~LNE4DuDs1hvA6FE01OU~Ei-VVFFWD80Bf8Jc~JX50BK5ZaTBg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463438&amp;ar=2709598&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw048f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bioinformatic pipeline flowchart. Shown is a flow diagram of the improved bioinformatic pipeline for the identification of autonomously expressed SINE loci from RNA-seq data sets. See text and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span> for details.</p></div></div><div content-id="dsw048-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f2.png?Expires=1509239085&amp;Signature=Lx6R8ZknQ9tSp862ZXbAUSEHorepxxYHHWKPxXnXHQIcncMTNHourtsZZAkumqPEpfGL7EUHwj~J-8Dw2feXy2RojMc6MMETqRPuxcsrWSZqsayapHW1xx03bhpmvdxu1FoL~2yoq39-eHyJDgvo~C43ZHWpDEPNTDMa-ecz1mj9eG6nCnEs~NLvLoF-3SPObAPMdQGE224DDGs1fRGDc4aiMwAHxWbtXMZnvHtx3wz~NMv1OTIfgMT48O~X5pSZfVAnmkHwp7pmFp9Kim3dPwxexXKeiK1-Dqo8bg5sAeoTaawthP6n7F8dL-uA5m1tzaNbOuwy597FTluJkNHYvQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Bioinformatic pipeline flowchart. Shown is a flow diagram of the improved bioinformatic pipeline for the identification of autonomously expressed SINE loci from RNA-seq data sets. See text and Supplementary Methods for details." path-from-xml="dsw048f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463438/dsw048f2.tif" path-from-xml="dsw048f2.tif" target="_blank">View large</a><a section="61463438" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f2.png?Expires=1509239085&amp;Signature=CGUKU-iSzn-flVUG87h0asUwZfLsHJ5EpRf1PA9JJhcLt2VFyKdW~3cPdFg6qHpSQyWadhvCJ7pHGtKI5vlq4k56tbrmQIvhuI~uPh0x1nrhV5L10Xg0VrYxXBPEiXGEsadptgwDpbSSBC95TFxwIpFBIXc90PJj4BerdmG5x4l1r9XOvAMKL33rJdIg6I53swM9W6WqPSAJpmDLna7~bPtrs~xdaDfMegqVPZjVof-f~CgmIgScQPtDJapv3NJikh8-cmizTxo8KVwHKHD2BIsdF~i0JFJNPnVy~LNE4DuDs1hvA6FE01OU~Ei-VVFFWD80Bf8Jc~JX50BK5ZaTBg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463438&amp;ar=2709598&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw048f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bioinformatic pipeline flowchart. Shown is a flow diagram of the improved bioinformatic pipeline for the identification of autonomously expressed SINE loci from RNA-seq data sets. See text and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span> for details.</p></div></div></p>
<p>Only MIRs which passed the final filter of the pipeline in both ENCODE RNA-Seq replicates were considered to represent autonomously expressed MIR loci and will be referred to as ‘expression-positive’. The complete list of expression-positive MIRs are reported in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>.</p><p>To further support the identification of unique MIR transcripts found in Hela-S3 and K562 cells we intersected the ChIP-seq peak data from ENCODE/Stanford/Yale/USC/Harvard (<a class="link link-uri" href="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeSydhTfbs/" target="_blank">http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeSydhTfbs/)</a> and Transcription Factor Binding Sites (TFBS) from ENCODE data uniformly processed by the ENCODE Analysis Working Group (<a class="link link-uri" href="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeRegTfbsClustered/wgEncodeRegTfbsClusteredWithCellsV3.bed.gz" target="_blank">http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wg EncodeRegTfbsClustered/wgEncodeReg TfbsClustered WithCells V3.bed.gz</a>) for some Pol III transcription factors (TFs) binding sites with the coordinates of the expected full-length expression-positive MIRs, extended to 200 bp upstream. To identify other Pol II TFs associated with expression-positive MIR elements, we intersected, for each cell line, the 500 bp upstream of the expected full-length MIRs with the coordinates of the TF binding sites from ENCODE data, also uniformly processed by the ENCODE Analysis Working Group. The lists of Pol III-associated expression-positive MIRs and of those associated with other TFs are reported in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span>.</p>                            <h3 scrollto-destination=61463441 id="61463441" class="section-title" >2.2. Plasmid construction</h3>
<p>Four human MIR loci (whose chromosome coordinates are reported in <a reveal-id="dsw048-T3" data-open="dsw048-T3" class="link link-reveal link-table xref-fig">Table 3</a>), together with their 5′- and 3′-flanking regions, were PCR-amplified and cloned into pGEM®-T Easy vector (Promega) using the oligonucleotides listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S4</a></span>. Constructs containing a targeted mutation of the B box internal control element were generated as described previously.<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Upstream deletion constructs employed forward PCR primers generating amplicons truncated at position −25 with respect to MIR A box at the 5′-end. Truncated amplicons were inserted into pGEM®-T Easy and the constructs selected for <em>in vitro</em> transcription contained the 5′-truncated insert in the same orientation as their wild-type MIR counterpart, to minimize the influence of vector sequence on transcription efficiency. </p>                            <h3 scrollto-destination=61463443 id="61463443" class="section-title" >2.3. <em>In vitro</em> transcription</h3>
<p>All recombinant plasmids for <em>in vitro</em> transcription reactions were purified with the Qiagen Plasmid Mini kit (Qiagen). Reaction mixtures (final volume: 25 µl) contained 500 ng of template DNA, 70 mM KCl, 5 mM MgCl<sub>2</sub>, 1 mM DTT, 2.5% glycerol, 20 mM Tris–HCl pH 8, 5 mM phosphocreatine, 2 µg<em>/</em>ml alpha-amanitin, 0.4 U<em>/</em>µl SUPERase-In (Ambion) as RNase inhibitor, 40 µg of HeLa cell nuclear extract,<a reveal-id="dsw048-B18" data-open="dsw048-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> 0.5 mM each of ATP, CTP, and GTP, 0.025 mM UTP and 5µCi of [α-32P]UTP (Perkin-Elmer). Reactions were allowed to proceed for 60 min at 30 °C before being stopped by the addition of 75 µl of nuclease-free water and 100 µl of phenol:chloroform (1:1) pH 5.5. Purified labeled RNA products were resolved on a 6% polyacrylamide, 7 M urea gel and visualized, and quantified with a Cyclone Phosphor Imager (PerkinElmer) and the Quantity One software (Bio-Rad).</p>                            <h3 scrollto-destination=61463445 id="61463445" class="section-title" >2.3. Availability</h3>
<p>The Python script allowing to identify individual MIR transcripts is available at the following URL: <a class="link link-uri" href="https://github.com/davidecarnevali/SINEsFind" target="_blank">https://github.com/davidecarnevali/SINEsFind</a>.</p>                            <h2 scrollto-destination=61463447 id="61463447" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=61463448 id="61463448" class="section-title" >3.1. A bioinformatic pipeline for the identification of transcriptionally active MIR loci from RNA-Seq datasets</h3>
<p>To leverage the potential of our previous work,<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> we developed a pipeline in order to improve the identification of Pol III SINE transcripts (see Materials and Methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span>). We focused on the identification of MIR transcripts by applying the improved pipeline to the aligned Long RNA-Seq reads from ENCODE,<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> considering for analysis a subset of the annotated MIR elements from the GRCh37/hg19 UCSC RepeatMasker track classified according to their genomic location: (i) MIRs mapping in intergenic regions (i.e. outside RefSeq, Ensembl, and lincRNA genes), (ii) MIRs mapping within RefSeq, Ensembl, and lincRNA genes but in antisense orientation, and (iii) MIRs fully contained within introns of RefSeq, Ensembl, and lincRNA genes in sense orientation and not overlapping with any exon. We refer to the first two groups together as ‘intergenic/antisense’.</p><p><a reveal-id="dsw048-F2" data-open="dsw048-F2" class="link link-reveal link-table xref-fig">Figure 2</a> provides a schematic representation of our improved pipeline. The main improvements with respect to the previously described pipeline<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> consist in the estimation of the ‘background’ coverage signal for each sample (instead using a fixed value for all the samples) and the definition of the flanking regions based on the global alignment of each annotated element with its consensus sequence (See Materials and Methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span>).</p>                            <h3 scrollto-destination=61463451 id="61463451" class="section-title" >3.2. General features of MIR transcriptomes emerging from ENCODE RNA-seq data analysis</h3>
<p>A preliminary survey of the expression coverage of the MIR loci identified as expression-positive showed us both convincing MIR transcription profiles and less clearcut ones with noise background signals upstream and downstream of the MIR element probably deriving from the sequencing of introns or unknown Pol II transcripts. Even the latter cases, however, displayed expression coverage enrichment in the region of the annotated MIR element, suggesting the possibility of autonomous Pol III transcription as a pathway for their biogenesis, perhaps occurring in parallel with Pol II transcription of the host gene with intron-containing MIR RNAs generated by processing from longer Pol II transcripts. Since the tendency to high noise-to-background signals is more frequently observed in MIR element mapping within introns of Pol II genes in sense orientation, which constitute 79% of the expression-positive MIRs (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>), we decided to mainly focus on the expression of intergenic/antisense MIRs, while a few examples of gene-hosted sense-oriented MIRs will be addressed later in the Results section.</p><p>The full list of MIRs identified as expression-positive by our search strategy is reported in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>. As summarized in <a reveal-id="dsw048-T1" data-open="dsw048-T1" class="link link-reveal link-table xref-fig">Table 1</a> each of the cell lines expressed a limited number of MIR elements (ranging from 145 in the case of GM12878 cells to 435 in the case of HUVEC cells). Of the whole set of 1301 expression-positive MIR loci a small percentage (∼21%) were intergenic/antisense, which is in contrast with the global distribution of all the annotated MIRs regardless of their expression where the percentage of the intergenic/antisense MIRs is much higher (∼68%) (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>). A significant fraction (∼39%) of intergenic/antisense MIRs mapped in antisense orientation to annotated Pol II-transcribed genes. For comparison, the fraction of intergenic/antisense expression-positive <em>Alus</em> mapping in antisense orientation to the overlapping genes in the same cell lines was only 22%,<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> suggesting that MIRs antisense to Pol II genes could have a more specific role, possibly correlated with the regulation of the overlapping gene. A small fraction (∼20%) of all the expression-positive MIRs were found to be expressed in more than one cell line (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>), while this percentage decreased to 12% considering only the intergenic/antisense MIRs (<a reveal-id="dsw048-T1" data-open="dsw048-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This suggests a marked cell line specificity in MIR expression (even though few MIRs seem to be ubiquitously expressed as will be further reported below) greater than the one found for <em>Alu</em> expression where the fraction of expression-positive <em>Alus</em> expressed in more than one cell line was higher (∼24%).<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><div class="table-wrap"><div id="dsw048-T1" data-id="dsw048-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Statistic of expression-positive MIR elements in selected cell lines</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Cell line </th><th>Total MIRs<sup>a</sup> </th><th>Intergenic/ antisense </th><th>Intergenic/ antisense shared<sup>b</sup> </th><th>Antisense<sup>c</sup> </th></tr></thead><tbody><tr><td>GM12878 </td><td>145 </td><td>39 </td><td>12 </td><td>16 </td></tr><tr><td>H1-hESC </td><td>280 </td><td>52 </td><td>18 </td><td>23 </td></tr><tr><td>HeLa-S3 </td><td>188 </td><td>32 </td><td>6 </td><td>15 </td></tr><tr><td>HepG2 </td><td>348 </td><td>46 </td><td>16 </td><td>23 </td></tr><tr><td>HUVEC </td><td>435 </td><td>59 </td><td>7 </td><td>18 </td></tr><tr><td>K562 </td><td>161 </td><td>42 </td><td>13 </td><td>18 </td></tr><tr><td>NHEK </td><td>158 </td><td>55 </td><td>15 </td><td>25 </td></tr><tr><td>ALL<sup>d</sup> </td><td>1301 </td><td>271 </td><td>33 </td><td>105 </td></tr></tbody></table></div><div content-id="dsw048-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Cell line </th><th>Total MIRs<sup>a</sup> </th><th>Intergenic/ antisense </th><th>Intergenic/ antisense shared<sup>b</sup> </th><th>Antisense<sup>c</sup> </th></tr></thead><tbody><tr><td>GM12878 </td><td>145 </td><td>39 </td><td>12 </td><td>16 </td></tr><tr><td>H1-hESC </td><td>280 </td><td>52 </td><td>18 </td><td>23 </td></tr><tr><td>HeLa-S3 </td><td>188 </td><td>32 </td><td>6 </td><td>15 </td></tr><tr><td>HepG2 </td><td>348 </td><td>46 </td><td>16 </td><td>23 </td></tr><tr><td>HUVEC </td><td>435 </td><td>59 </td><td>7 </td><td>18 </td></tr><tr><td>K562 </td><td>161 </td><td>42 </td><td>13 </td><td>18 </td></tr><tr><td>NHEK </td><td>158 </td><td>55 </td><td>15 </td><td>25 </td></tr><tr><td>ALL<sup>d</sup> </td><td>1301 </td><td>271 </td><td>33 </td><td>105 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>For each cell line, the column reports the number of MIRs considered as autonomously expressed in both ENCODE RNA-seq replicates.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>For each cell line, the column reports the number of intergenic/antisense MIRs that are also expressed in one or more different cell lines.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>Reported in this column are the numbers of intergenic MIRs mapping with an antisense orientation to either protein-coding, non protein-coding or lincRNA genes.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>d</sup>The numbers in this raw refer to individual MIRs expressed in one or more cell lines.</p></div></div></div></div></p><p>As summarized in <a reveal-id="dsw048-T2" data-open="dsw048-T2" class="link link-reveal link-table xref-fig">Table 2</a>, no significant under- or over-representation of any particular MIR subfamily within the set of expressed MIRs was observed (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>). <div class="table-wrap"><div id="dsw048-T2" data-id="dsw048-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Subfamily distribution of expression-positive MIRs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>MIR subfamily </th><th>Total genomic<sup>a</sup> </th><th>Expressed genomic<sup>a</sup> </th><th>Total intergenic/ antisense<sup>a</sup> </th><th>Expressed intergenic/ antisense<sup>a</sup> </th></tr></thead><tbody><tr><td>MIR </td><td>171148 (30%) </td><td>436 (33%) </td><td>115860 (30%) </td><td>101 (37%) </td></tr><tr><td>MIRb </td><td>219279 (38%) </td><td>487 (37%) </td><td>148567 (38%) </td><td>104 (38%) </td></tr><tr><td>MIRc </td><td>100252 (17%) </td><td>202 (16%) </td><td>67888 (17%) </td><td>34 (13%) </td></tr><tr><td>MIR3 </td><td>87763 (15%) </td><td>176 (14%) </td><td>59078 (15%) </td><td>32 (12%) </td></tr></tbody></table></div><div content-id="dsw048-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>MIR subfamily </th><th>Total genomic<sup>a</sup> </th><th>Expressed genomic<sup>a</sup> </th><th>Total intergenic/ antisense<sup>a</sup> </th><th>Expressed intergenic/ antisense<sup>a</sup> </th></tr></thead><tbody><tr><td>MIR </td><td>171148 (30%) </td><td>436 (33%) </td><td>115860 (30%) </td><td>101 (37%) </td></tr><tr><td>MIRb </td><td>219279 (38%) </td><td>487 (37%) </td><td>148567 (38%) </td><td>104 (38%) </td></tr><tr><td>MIRc </td><td>100252 (17%) </td><td>202 (16%) </td><td>67888 (17%) </td><td>34 (13%) </td></tr><tr><td>MIR3 </td><td>87763 (15%) </td><td>176 (14%) </td><td>59078 (15%) </td><td>32 (12%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>Reported are the absolute copy numbers and (in parentheses) the percentages of MIRs of each sub-family considered relative to (from left to right): the total set of genomic MIRs (intergenic/antisense plus intronic sense MIRs); the set of MIRs found to be expression-positive in one or more cell line (intergenic/antisense plus intronic sense ones); the genomic set of intergenic/antisense MIRs; the set of intergenic/antisense MIRs found to be expression-positive in one or more cell lines.</p></div></div></div></div></p>                            <h3 scrollto-destination=61463455 id="61463455" class="section-title" >3.3. Survey of expressed MIRs according to location and base-resolution expression profile</h3>
<p>The ∼6 × 10<sup>5</sup> annotated MIRs are not all complete in sequence, and represent only a portion of the canonical full-length MIR element. Among expression-positive MIRs, we found both complete and incomplete elements and, correspondingly, three main types of base-resolution expression profiles: (i) full-length or almost full-length MIRs (<a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3A and B</a>), covered by sequence reads along all their extension; (ii) incomplete MIRs representing either the left or the right portion of the canonical full-length MIR but whose transcript coverage tends to correspond to the one of a fully transcribed canonical MIR [more specifically, transcript coverage tends to extend into the downstream MIR-unrelated region for incomplete MIRs lacking the 3′ moiety (<a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>), while it tends to start in an upstream MIR-unrelated region, possessing functional A and B-boxes, for incomplete MIRs lacking the 5′ moiety (<a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3E</a>)]; (iii) incomplete MIRs lacking the 3′ moiety (thus containing A and B boxes) and producing transcripts that do not extend outside of the MIR sequence (<a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>). <div data-id="dsw048-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f3.png?Expires=1509239085&Signature=T2EAUEOZaR0Bib9dRqdG1qjKkxNi7NrytHyE1-K5kAwhq1qeDv8X3DjMMqD6HE4UZWFIO-z~-2JVQD6WKvrcZ-dE0rWRlCDWaJ9SUtPIKQQEZmOFXuckqaO3Ei4Tkq3eJKpbECZs2mFzgE~cxpxCxc9wewVNO2ZIzI6-aGjdY7Cah9PUcmF~Um1zQ8jpR61bCrxXesrUxsuypteKbMTB40pPfmywxQW69HKYH3IUgfhB2~N1vuc6qMgnctAyboJGMIycrJ5mHA7TasvD-rczw8TouHUq0Wlz9RRm6H2idlNYaGXwkjUbwf0edhOBsOalwBLpThbX2sY0GaQEXETVqQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Base-resolution expression profiles for five representative MIRs. See text for descriptions. Bars labeled with the names of Pol III transcription components (RPC155, TFIIIC110, BDP1, BRF1) indicate regions of enrichment of the corresponding proteins according to publicly available ChIP-seq data; upper arrowed bars represent the annotated MIR elements while the lower arrowed bars represent the corresponding expected full-length MIR elements which does not correspond to annotated elements and are reported merely to locate the alignment position of the annotated MIRs within the corresponding consensus sequence. Arrows in panel C point to the positions of strong Pol III terminators. (A) MIR_dup717 chr14:34206132-34206363; (B) MIR_dup2691 chr11:35548054-35548257 which resides within an intron of the PAMR1 gene in sense orientation; (C) MIR_dup2285 chr16:22309780-22309939; (D) MIRb_dup1281 chr17:17863550-17863651; (E) MIRc_dup2189 chr14:89445565-89445634" path-from-xml="dsw048f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463456/dsw048f3.tif" path-from-xml="dsw048f3.tif" target="_blank">View large</a><a section="61463456" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f3.png?Expires=1509239085&Signature=O1zEOHvnk08MhSoiE~53IsIHkGCLGGXdr8oj0dj68D-paKyWZC6Ck6Z6mXgZ9rTRFJfqbGQOeBxpUPY3s3j~5n6YxGYxUh~2WOCVYxkTYqywMYHjdX5ns91oCrOvi0D33mrxZ6oGq8pUeiQN9WXXHLdTd~4w7f66qEGMshSXxLkjtObnabKIRlteEVYFsPNevwaHkaVgO3guys27j4gqtBQz5KjIwvRXdCHeRLRAciAsCvQuvVBiazaQ~vTBMDPaM3n7n05gkLHP6JZQfBGX2XpR5kfRZUSqilZ~8NGdSI1hNhjBA11zQo3qGpJlxqls1wS3-1etPPSwlSpeKIcdpA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463456&ar=2709598&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw048f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Base-resolution expression profiles for five representative MIRs. See text for descriptions. Bars labeled with the names of Pol III transcription components (RPC155, TFIIIC110, BDP1, BRF1) indicate regions of enrichment of the corresponding proteins according to publicly available ChIP-seq data; upper arrowed bars represent the annotated MIR elements while the lower arrowed bars represent the corresponding expected full-length MIR elements which does not correspond to annotated elements and are reported merely to locate the alignment position of the annotated MIRs within the corresponding consensus sequence. Arrows in panel C point to the positions of strong Pol III terminators. (A) MIR_dup717 chr14:34206132-34206363; (B) MIR_dup2691 chr11:35548054-35548257 which resides within an intron of the PAMR1 gene in sense orientation; (C) MIR_dup2285 chr16:22309780-22309939; (D) MIRb_dup1281 chr17:17863550-17863651; (E) MIRc_dup2189 chr14:89445565-89445634</p></div></div><div content-id="dsw048-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f3.png?Expires=1509239085&Signature=T2EAUEOZaR0Bib9dRqdG1qjKkxNi7NrytHyE1-K5kAwhq1qeDv8X3DjMMqD6HE4UZWFIO-z~-2JVQD6WKvrcZ-dE0rWRlCDWaJ9SUtPIKQQEZmOFXuckqaO3Ei4Tkq3eJKpbECZs2mFzgE~cxpxCxc9wewVNO2ZIzI6-aGjdY7Cah9PUcmF~Um1zQ8jpR61bCrxXesrUxsuypteKbMTB40pPfmywxQW69HKYH3IUgfhB2~N1vuc6qMgnctAyboJGMIycrJ5mHA7TasvD-rczw8TouHUq0Wlz9RRm6H2idlNYaGXwkjUbwf0edhOBsOalwBLpThbX2sY0GaQEXETVqQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Base-resolution expression profiles for five representative MIRs. See text for descriptions. Bars labeled with the names of Pol III transcription components (RPC155, TFIIIC110, BDP1, BRF1) indicate regions of enrichment of the corresponding proteins according to publicly available ChIP-seq data; upper arrowed bars represent the annotated MIR elements while the lower arrowed bars represent the corresponding expected full-length MIR elements which does not correspond to annotated elements and are reported merely to locate the alignment position of the annotated MIRs within the corresponding consensus sequence. Arrows in panel C point to the positions of strong Pol III terminators. (A) MIR_dup717 chr14:34206132-34206363; (B) MIR_dup2691 chr11:35548054-35548257 which resides within an intron of the PAMR1 gene in sense orientation; (C) MIR_dup2285 chr16:22309780-22309939; (D) MIRb_dup1281 chr17:17863550-17863651; (E) MIRc_dup2189 chr14:89445565-89445634" path-from-xml="dsw048f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463456/dsw048f3.tif" path-from-xml="dsw048f3.tif" target="_blank">View large</a><a section="61463456" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f3.png?Expires=1509239085&Signature=O1zEOHvnk08MhSoiE~53IsIHkGCLGGXdr8oj0dj68D-paKyWZC6Ck6Z6mXgZ9rTRFJfqbGQOeBxpUPY3s3j~5n6YxGYxUh~2WOCVYxkTYqywMYHjdX5ns91oCrOvi0D33mrxZ6oGq8pUeiQN9WXXHLdTd~4w7f66qEGMshSXxLkjtObnabKIRlteEVYFsPNevwaHkaVgO3guys27j4gqtBQz5KjIwvRXdCHeRLRAciAsCvQuvVBiazaQ~vTBMDPaM3n7n05gkLHP6JZQfBGX2XpR5kfRZUSqilZ~8NGdSI1hNhjBA11zQo3qGpJlxqls1wS3-1etPPSwlSpeKIcdpA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463456&ar=2709598&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw048f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Base-resolution expression profiles for five representative MIRs. See text for descriptions. Bars labeled with the names of Pol III transcription components (RPC155, TFIIIC110, BDP1, BRF1) indicate regions of enrichment of the corresponding proteins according to publicly available ChIP-seq data; upper arrowed bars represent the annotated MIR elements while the lower arrowed bars represent the corresponding expected full-length MIR elements which does not correspond to annotated elements and are reported merely to locate the alignment position of the annotated MIRs within the corresponding consensus sequence. Arrows in panel C point to the positions of strong Pol III terminators. (A) MIR_dup717 chr14:34206132-34206363; (B) MIR_dup2691 chr11:35548054-35548257 which resides within an intron of the PAMR1 gene in sense orientation; (C) MIR_dup2285 chr16:22309780-22309939; (D) MIRb_dup1281 chr17:17863550-17863651; (E) MIRc_dup2189 chr14:89445565-89445634</p></div></div></p><p>The MIR reported in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3A</a> (chr14:34206132-34206363 MIR_dup717) was scored as expression-positive in Gm12878 and K562 cells. It also displayed a specific, yet very low coverage, below the threshold for a positive score, in H1-hESC, HeLa-S3, HepG2, and NHEK cells. This representative MIR is full-length and maps to intron 6 or 7 (depending on which transcript isoform is considered) of the NPAS3 gene, in antisense orientation. It has an expression profile that almost entirely covers the annotated element with uniquely mapped sequence reads. The double-humped expression coverage profiles shown in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3</a> are due to cDNA size selection during library preparation and sequencing specifications, as previously reported.<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> A- and B- boxes are conserved at 13 and 50 bp downstream of the TSS, respectively. Unusually, the annotated MIR contains a strong termination signal (T<sub>4</sub>) in its sequence 104 nt from the beginning of the element, which is clearly skipped by the Pol III machinery, given the coverage signal in the downstream moiety of the element. We investigated the possibility that the genomes of the cell lines analyzed had a sequence variation with respect to the reference GRCh37/hg19 sequence, in correspondence of these Pol III strong terminators. To this end, we reconstructed the corresponding consensus genome sequences from RNA sequence reads (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span>). We did not find any sequence variant, thus indicating the presence of a termination signal skipped by the Pol III machinery. Another terminator of 4Ts is present in the downstream moiety of the MIR, leading to a transcript of ∼220 nt. Strong support for the genuine nature of this Pol III-derived MIR RNA comes from the association with this locus of the Pol III subunit RPC155 in K562 cells (<a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>).</p><p><a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3B</a> shows a MIR fully contained within an intron of the PAMR1 gene in the sense orientation that has been found to be expressed in 5 cell lines. The Pol III-dependent character of this transcription unit is supported by its association with the Pol III subunit RPC155 and the Pol III transcription factor BDP1 in K562 and HeLa cell lines, also confirming the ability of Pol III machinery complex to access and transcribe Pol II genomic loci.</p><p>Reported in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3C</a> is the transcript coverage profile of an incomplete MIR element of 159 bp (chr16:22309780-22309939 MIR_dup2285), which was expressed in all 7 cell lines. This element aligns to the left portion of the MIR consensus sequence, but its transcription continues downstream of the region annotated as MIR (in orange in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) for ∼200 nt before encountering a non-canonical termination signal (T<sub>3</sub>CT). This previously characterized MIR<a reveal-id="dsw048-B11" data-open="dsw048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> is located in antisense orientation in the first intron of the POLR3E gene, encoding a subunit (RPC5) of human RNA polymerase III. Its transcription by Pol III is supported by the presence of canonical A- and B-boxes as well as by association with components of the Pol III machinery in HeLa-S3 and K562 cell lines. Here, again, the annotated MIR portion contains a strong termination signal (T<sub>4</sub>) early in its sequence, 121 nt downstream the 5′ end of the element. Another potential terminator is located 82 nt downstream of the 3′ end of the annotated element. In this case, the coverage signal tends to decrease temporarily but slightly increases again downstream, until a non-canonical terminator is encountered. The skipping of the first terminator by Pol III is supported by the expression coverage levels upstream and downstream, which are almost the same, strongly suggesting that they arise from the same transcript. In contrast, lower coverage downstream of the second terminator suggests the occurrence of only limited read through of this termination signal by Pol III. We also considered the possibility of a mutation in the strong terminator sequences in the DNA of these cells (see above), but we did not find any sequence variant, thus confirming the ability of Pol III to skip strong terminator signals within certain sequence contexts.<a reveal-id="dsw048-B19" data-open="dsw048-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> This result was partially confirmed by <em>in vitro</em> transcription analysis (see <a reveal-id="dsw048-F5" data-open="dsw048-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>).</p><p>In <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3D</a>, we report a MIR element (chr17:17863550-17863651 MIRb_dup1281), found to be expressed at low levels in in H1-hESC cell line, originating from an incomplete MIR element antisense to the TOM1L2 gene. By inspecting the coverage profile in other cell lines, we could find signals of much lower expression in at least one replicate of each of the analyzed cell lines. The source element of this transcript corresponds to a MIRb left fragment carrying in its sequence functional A- and B- boxes. The transcript coverage precisely spans the length of the annotated element up to a strong terminator (T<sub>10</sub>) right at the end of it, leading to a transcript of ∼100 nt.</p><p><a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3E</a> shows a MIR, found as expression-positive in H1-hESC, K562 and NHEK cell lines, whose transcription initiates in an upstream MIR-unrelated region (chr14:89445565-89445634 MIRc_dup2189) and ends ∼50 nt downstream of the annotated element corresponding with a strong termination signal (T<sub>5</sub>). By inspecting the MIR-unrelated upstream region we found canonical A- and B- boxes, indicating that this MIR element, lacking Pol III promoters in its sequence, is transcribed from an upstream MIR-unrelated region providing functional control elements. Also in this case we noted a termination signal (T<sub>3</sub>AT<sub>4</sub>) at the beginning of the annotated element which seems to be (partially) skipped <em>in vivo</em> by Pol III (as for MIR_dup2285 in POLR3E gene in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) and is located right at the end of the first coverage hump. Indeed the second hump has a slightly lower coverage signal (at least in H1-hESC and NHEK cells) suggesting a shorter primary transcript as confirmed by <em>in vitro</em> transcription (see below). The binding of Pol III TFs throughout this region in K562 cells, together with the transcript coverage profile, strongly support the Pol III-dependence of this MIR transcript.</p>
  <p>As a further note on the MIR in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Figure 3C, a</a> Polr3e antisense-oriented MIR with the same location and similar sequence of MIR_dup2285 is also present in the mouse genome, where it has also been shown to be transcriptionally active, and to be associated with an unusual second point accumulation of RNA Pol II, in addition to the expected one in the Polr3e promoter, which abuts precisely on the Pol III occupancy peak reflecting MIR transcription on the opposite strand.<a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> These observations, along with the fact that this MIR is conserved in human and mouse, led the authors to hypothesize a functional role of this MIR in POLR3E gene regulation. We therefore asked if this second RNA Pol II accumulation point also occurs in the ENCODE cell lines, and found Pol II peaks for all 7 cell types (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S5</a></span>) thus strengthening the hypothesis that this transcriptionally active MIR plays a regulatory role in Pol II transcription of POLR3E. <a reveal-id="dsw048-F4" data-open="dsw048-F4" class="link link-reveal link-table xref-fig">Figure 4</a> shows Pol II in three of the seven cell lines along with the Pol III signals. <div data-id="dsw048-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f4.png?Expires=1509239085&amp;Signature=ChBhVVRfKmU9m2hLz88daQoJd4Ru1r0hR-STGZrXe2bcE~Kf2Ebc78kktDou9GndXmcPm-ZxBH8wGy9ROfwRciSLU9j9iGhN~vHwrCjOiJKnSQMHQAEPfBR5uKZibHdw4nIB-x7BzX3v~vr8hkg4EF3s0-RsRi6ZDQnRas0Sgls6Hh07IeMKXrHenjibN1mrOdCJt01q5V-niF0~fvo-eyBbF0JrSbLNE7DdFe77sTMJ3TxpnY~Wnhpuh7FpqGNGCavmfufSx3KkSi7W0siI96NBBeZosmYJCVbftmz3WwkMDBldbkPJ7~HYAp~7skpNuek~M9y1z6JLYP4Ld5rdkg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Simultaneous Pol III and Pol II accumulation signals at MIR_dup2285. Shown are the ChIP-seq signals of Pol II (POLR2A) (blue) and Pol III (RPC155) (red) in HeLa-S3 and K562 cell lines across the expression-positive MIR located within the first intron of POLR3E gene in antisense orientation (cf. Fig. 3C)." path-from-xml="dsw048f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463462/dsw048f4.tif" path-from-xml="dsw048f4.tif" target="_blank">View large</a><a section="61463462" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f4.png?Expires=1509239085&amp;Signature=geU8YX2mvVUfKtaDLKhZF~zVKL1rM-tKP8XvU0BUXvTgHhpwx990Kcc1LZw1jgzDzn7zvcqviz27~AsyGh4wRJ0NdjkSo781m1MF537B3vpjve-QKPMcnTrBapCYqNtnQh9K9UNp1p6v9EDLufQZZU6--lo-QcZjiYqWyVksZGUTPy3MidL0U2XL3joROMZRDxkls677p0Gl~xk9uQ0V-dUAsdhbQ8gDKnTscholuwWSwjX48xju4AQZiJIooHrHcZSFGScZw5U5RqCyh58cmUElUjGSGWtICCHBk~g3GCqHkA4lfAmZnUuJUitQkZTCp5~vGDr1h8O00-kSU8Gvqw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463462&amp;ar=2709598&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw048f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Simultaneous Pol III and Pol II accumulation signals at MIR_dup2285. Shown are the ChIP-seq signals of Pol II (POLR2A) (blue) and Pol III (RPC155) (red) in HeLa-S3 and K562 cell lines across the expression-positive MIR located within the first intron of POLR3E gene in antisense orientation (cf. <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>).</p></div></div><div content-id="dsw048-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f4.png?Expires=1509239085&amp;Signature=ChBhVVRfKmU9m2hLz88daQoJd4Ru1r0hR-STGZrXe2bcE~Kf2Ebc78kktDou9GndXmcPm-ZxBH8wGy9ROfwRciSLU9j9iGhN~vHwrCjOiJKnSQMHQAEPfBR5uKZibHdw4nIB-x7BzX3v~vr8hkg4EF3s0-RsRi6ZDQnRas0Sgls6Hh07IeMKXrHenjibN1mrOdCJt01q5V-niF0~fvo-eyBbF0JrSbLNE7DdFe77sTMJ3TxpnY~Wnhpuh7FpqGNGCavmfufSx3KkSi7W0siI96NBBeZosmYJCVbftmz3WwkMDBldbkPJ7~HYAp~7skpNuek~M9y1z6JLYP4Ld5rdkg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Simultaneous Pol III and Pol II accumulation signals at MIR_dup2285. Shown are the ChIP-seq signals of Pol II (POLR2A) (blue) and Pol III (RPC155) (red) in HeLa-S3 and K562 cell lines across the expression-positive MIR located within the first intron of POLR3E gene in antisense orientation (cf. Fig. 3C)." path-from-xml="dsw048f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463462/dsw048f4.tif" path-from-xml="dsw048f4.tif" target="_blank">View large</a><a section="61463462" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f4.png?Expires=1509239085&amp;Signature=geU8YX2mvVUfKtaDLKhZF~zVKL1rM-tKP8XvU0BUXvTgHhpwx990Kcc1LZw1jgzDzn7zvcqviz27~AsyGh4wRJ0NdjkSo781m1MF537B3vpjve-QKPMcnTrBapCYqNtnQh9K9UNp1p6v9EDLufQZZU6--lo-QcZjiYqWyVksZGUTPy3MidL0U2XL3joROMZRDxkls677p0Gl~xk9uQ0V-dUAsdhbQ8gDKnTscholuwWSwjX48xju4AQZiJIooHrHcZSFGScZw5U5RqCyh58cmUElUjGSGWtICCHBk~g3GCqHkA4lfAmZnUuJUitQkZTCp5~vGDr1h8O00-kSU8Gvqw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463462&amp;ar=2709598&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw048f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Simultaneous Pol III and Pol II accumulation signals at MIR_dup2285. Shown are the ChIP-seq signals of Pol II (POLR2A) (blue) and Pol III (RPC155) (red) in HeLa-S3 and K562 cell lines across the expression-positive MIR located within the first intron of POLR3E gene in antisense orientation (cf. <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>).</p></div></div></p>
<p>Finally, we asked if there was a correlation between the expression of the 78 MIRs overlapping with Pol II protein-coding genes in antisense orientation and the expression of the overlapping genes themselves. To this end, we compared normalized read counts of the MIRs in each replicate with those of the overlapping Pol II genes using the Pearson correlation coefficient (See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Methods</a></span>) and found only positive correlation (<em>r</em> &gt; 0.7 with <em>P</em>  &lt; 10 <sup>−</sup> <a reveal-id="dsw048-B5" data-open="dsw048-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a>) for few genes (ARHGEF3, DSCAM, GNA15, JAKMIP2, LRP2, NBPF10, PLXDC2, TMEM177) (data not shown). We also asked whether the protein-coding genes characterized by the presence of intronic MIRs (mapping in either antisense or sense orientation) are enriched in any particular gene ontology functional category. Interestingly, we found a strong enrichment (<em>P</em> &lt; 10 <sup>−</sup> <sup>139</sup>) for genes producing multiple proteins due to alternative splicing, suggesting a possible influence of intronic MIRs on this process. When expression-positive MIR only were considered, such a GO enrichment was less marked, although still remarkable (<em>P</em> value 1.8 × 10 <sup>−</sup> <a reveal-id="dsw048-B24" data-open="dsw048-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a>).</p>                            <h3 scrollto-destination=61463464 id="61463464" class="section-title" >3.4. Association of the Pol III machinery with expression-positive MIRs</h3>
<p>Before the use of RNA-Seq data to help identify expression-positive SINEs, genome-wide studies based on ChIP-Seq approaches were used in humans and mouse models with the aim of producing inventories of loci inferred to be transcribed by Pol III from their association with one or more component of the Pol III machinery.<a reveal-id="dsw048-B11" data-open="dsw048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw048-B12" data-open="dsw048-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw048-B20" data-open="dsw048-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsw048-B21" data-open="dsw048-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Such studies revealed a limited number of Pol III-associated SINE elements, and most of them were <em>Alus</em>. The availability of genome-wide ChIP-Seq data from ENCODE/Stanford/Yale/USC/Harvard (SYDH) and from ENCODE data uniformly processed by the ENCODE Analysis Working Group for key components of Pol III transcription machinery (BDP1, BRF1/2, RPC155, and TFIIIC110) allowed us to readdress this issue, by investigating whether a significant enrichment of these TFs could be found in the expression-positive intergenic/antisense MIRs of HeLa and K562 cells. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span> lists the expression positive MIRs found to be associated with Pol III components in HeLa and K562 cells. In HeLa cells, of the 32 intergenic/antisense expression-positive MIRs, only 5 were found associated with one or more Pol III TFs. The three transcription components whose association was found to be statistically significant, with a <em>P</em>  &lt; 5.5x10 <sup>−</sup> <a reveal-id="dsw048-B5" data-open="dsw048-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> (calculated using Fisher’s exact test), were BDP1, TFIIIC-110 and RPC155 when considering the fraction of the intergenic/antisense expressed MIRs against the total intergenic/antisense annotated ones. When the Pol III component association with the 156 intron-hosted sense oriented expression positive MIRs was investigated, only one (MIR_dup2691 reported in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>) was found associated with the Pol III subunit RPC155. When K562 cells were considered, a higher percentage (36%, 15 MIRs) of intergenic/antisense expression-positive MIRs where found bound by one or more Pol III components, while the percentage drop down to 3% (4 MIRs) when considering the 119 MIRs fully contained within introns of Pol II genes in sense orientation (data not shown). In K562 cells we found significant enrichment for BDP1, TFIIIC110 and RPC155 (<em>P</em> values 5.2 × 10 <sup>−</sup> <a reveal-id="dsw048-B9" data-open="dsw048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a>, 9.7 × 10 <sup>−</sup> <a reveal-id="dsw048-B6" data-open="dsw048-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and &lt;2.2 × 10 <sup>−</sup> <a reveal-id="dsw048-B16" data-open="dsw048-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a>, respectively) in intergenic/antisense expression-positive MIRs. It is interesting to compare these results with those previously reported for <em>Alus</em>.<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> K562 cells were found to have a higher number of <em>Alu</em> loci bound to components of the Pol III machinery complex than HeLa cells, thus suggesting a more permissive environment for Pol III association in K562 cells. The lower percentage of Pol III-associated elements among intronic sense-oriented expression-positive MIRs suggests that their expression is Pol II-dependent in most cases.</p><p>Interestingly the MIR antisense to POLR3E, expressed in all 7 cell lines, is the one bound by the highest number of Pol III TFs in both HeLa and K562 cells (BDP1, BRF1, TFIIIC-110, RPC155) thus further confirming its genuine character of Pol III transcription unit.</p>                            <h3 scrollto-destination=61463467 id="61463467" class="section-title" >3.5. Association of expression-positive MIRs with TFs</h3>
<p>In order to assess whether the upstream region of MIR elements could influence their transcription through TFs specifically interacting with it, we took advantage of the availability of ChIP-Seq data for several TFs within ENCODE datasets. We asked if intergenic/antisense MIRs identified as expressed through our pipeline tend to be associated with one or more Pol II TFs, in addition to the known components of the Pol III machinery. The results of this analysis are reported in detail in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span>. Highly variable TFs associations were observed among the 7 cell lines, both in terms of the total number of TF-bound MIRs (ranging from 7 to 21) and in terms of the number of TFs associated to each MIR, mostly depending on the number of available TF ChIP-seq data for each cell line (ranging from 3 in NHEK cells to 100 in K562 cells). Among the Pol II-related transcription proteins most significantly associated with expression-positive intergenic/antisense MIRs we found RNA polymerase II itself (POLR2A), TBP, MAX, MAZ, and YY1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S3</a></span>). HeLa, HUVEC and NHEK cell lines are those with the lowest number of TFs being measured by ChIP-seq and therefore have very low numbers of intergenic/antisense expression-positive MIRs bound by TFs and with no significant enrichment.</p>                            <h3 scrollto-destination=61463469 id="61463469" class="section-title" >3.6. Expression-positive MIRs and chromatin states</h3>
<p>A recent study, showing that MIR elements are highly enriched in the enhancer state of chromatin in K562 and HeLa cell lines<a reveal-id="dsw048-B9" data-open="dsw048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> prompted us to investigate whether our intergenic/antisense expression-positive MIRs found in the same cell lines were among those found by this study. We intersected the coordinates of our expression-positive MIRs with those of the MIR elements found to be enriched in enhancers (lifted over from NCBI36/hg18 to GRCh37/hg19), but we found only one overlapping MIR among the fully intronic sense-oriented dataset. We also tested for enrichment in the enhancer states of expression-positive intergenic/antisense MIRs by using the ENCODE Chromatin State Segmentation data. A Fisher’s exact test was performed against all the other annotated intergenic/antisense MIRs used in the pipeline. We found significant enrichment only in Gm12878 and HUVEC cell line in strong enhancer state (<em>P</em> values 6.35e <sup>−</sup> <sup>06</sup> and 1.08e <sup>−</sup> <sup>05</sup>, respectively) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S5</a></span>). Extending the test to the fully intronic sense-oriented expression-positive MIRs, we did not find any significant enrichment in any of the seven cell lines.</p><p>Considering the results from another study<a reveal-id="dsw048-B15" data-open="dsw048-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> which shows that MIR sequences provide insulators in the genome of CD4<sup>+ </sup>T cells, we verified if any of our expression-positive MIRs found in all the seven cell lines analysed were among the MIR-derived insulators, but we found only three overlaps. Again, we decided to check if our expression-positive MIRs were enriched in the insulator state of chromatin but we found very few not statistically significant (using Fisher’s exact test) overlaps between the coordinates of our expression-positive MIRs, both intergenic/antisense (no overlaps) and fully intronic sense-oriented (one in GM12878, two in H1-hESC, three in HepG2, and two in NHEK cell lines), and those of genomic regions marked as insulators in ENCODE Chromatin State Segmentation by HMM data (data not shown).</p>                            <h3 scrollto-destination=61463472 id="61463472" class="section-title" >3.7. <em>In vitro</em> transcription analysis of expressed MIR elements</h3>
<p>Our bioinformatic pipeline permitted us to detect <em>in vivo</em> expression of individual MIR elements that could be transcribed by the Pol III machinery. To confirm Pol III transcription and make a precise promoter characterization of these transcriptional units, we conducted their <em>in vitro</em> transcription in a HeLa nuclear extract. We focused our attention on a small subset of MIR loci, which are expressed in at least three cell types (listed in <a reveal-id="dsw048-T3" data-open="dsw048-T3" class="link link-reveal link-table xref-fig">Table 3</a>). One of them is MIR_dup2285 (chr16:22309780-22309939), expressed in all the seven investigated ENCODE cell lines and associated with at least 3 components of the Pol III machinery in both HeLa-S3 and K562 cells (see <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span>). Another locus subjected to <em>in vitro</em> transcription analysis, MIR_dup3493 (chr1: 34943459-34943727), was found to be expressed in four cells lines (GM12878, H1-hESC, K562 and NHEK) and has A- and B- boxes perfectly matching the consensus.<a reveal-id="dsw048-B22" data-open="dsw048-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The remaining two <em>in vitro</em> tested loci (MIRb_dup5848 chr2:71762977-71763215 and MIRc_dup2189 chr14:89445565-89445634) were found to be expressed in at least three cell types (GM12878, H1-hESC, HepG2, NHEK and H1-hESC, K562, NHEK respectively). Based on ENCODE ChIP-seq data, only MIRc_dup2189 is associated with two components of the Pol III machinery (RPC155 and BDP1). <div class="table-wrap"><div id="dsw048-T3" data-id="dsw048-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>MIRs subjected to <em>in vitro</em> transcription analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>MIR </th><th>Expression in cell lines<sup>a</sup> </th><th>Predicted length of primary transcript(s)<sup>b</sup> </th></tr></thead><tbody><tr><td>MIR_dup2285 (chr16:22309780-22309939) </td><td>GM12878, H1-hESC, HeLa-S3, HepG2, HUVEC, K562, NHEK </td><td>124/131 (T<sub>4</sub>), 223/230 (T<sub>3</sub>GT<sub>2</sub>), 243/250 (T<sub>4</sub>), 354/361 (T<sub>3</sub>CT, downstream of annotated MIR but within the cloned sequence) </td></tr><tr><td>MIR_dup3493 (chr1:34943459-34943727) </td><td>GM12878, H1-hESC, K562, NHEK </td><td>177 (TAT<sub>3</sub>), 213 (TAT<sub>3</sub>), 277 (T<sub>2</sub>AT<sub>2</sub>). Expected transcripts originating from terminators more downstream of the annotated MIR but within the cloned sequence: 305 (T<sub>2</sub>AT<sub>2</sub>), 365 (T<sub>2</sub>AT<sub>3</sub>), 393 (T<sub>4</sub>) </td></tr><tr><td>MIRb_dup5848 (chr2:71762977-71763215) </td><td>H1-hESC, HepG2, NHEK </td><td>119 (TCT<sub>3</sub>), 256 (T<sub>3</sub>CT), 358 (T<sub>5</sub>) </td></tr><tr><td>MIRc_dup2189 (chr14:89445565-89445634) </td><td>H1-hESC, K562, NHEK </td><td>137 (T<sub>3</sub>AT) and 140 (T<sub>4</sub>), 207 (T<sub>2</sub>GT<sub>3</sub>), 250 (T<sub>5</sub>) </td></tr></tbody></table></div><div content-id="dsw048-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>MIR </th><th>Expression in cell lines<sup>a</sup> </th><th>Predicted length of primary transcript(s)<sup>b</sup> </th></tr></thead><tbody><tr><td>MIR_dup2285 (chr16:22309780-22309939) </td><td>GM12878, H1-hESC, HeLa-S3, HepG2, HUVEC, K562, NHEK </td><td>124/131 (T<sub>4</sub>), 223/230 (T<sub>3</sub>GT<sub>2</sub>), 243/250 (T<sub>4</sub>), 354/361 (T<sub>3</sub>CT, downstream of annotated MIR but within the cloned sequence) </td></tr><tr><td>MIR_dup3493 (chr1:34943459-34943727) </td><td>GM12878, H1-hESC, K562, NHEK </td><td>177 (TAT<sub>3</sub>), 213 (TAT<sub>3</sub>), 277 (T<sub>2</sub>AT<sub>2</sub>). Expected transcripts originating from terminators more downstream of the annotated MIR but within the cloned sequence: 305 (T<sub>2</sub>AT<sub>2</sub>), 365 (T<sub>2</sub>AT<sub>3</sub>), 393 (T<sub>4</sub>) </td></tr><tr><td>MIRb_dup5848 (chr2:71762977-71763215) </td><td>H1-hESC, HepG2, NHEK </td><td>119 (TCT<sub>3</sub>), 256 (T<sub>3</sub>CT), 358 (T<sub>5</sub>) </td></tr><tr><td>MIRc_dup2189 (chr14:89445565-89445634) </td><td>H1-hESC, K562, NHEK </td><td>137 (T<sub>3</sub>AT) and 140 (T<sub>4</sub>), 207 (T<sub>2</sub>GT<sub>3</sub>), 250 (T<sub>5</sub>) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p><sup>a</sup>The column lists, for each MIR element, the cell lines in which it was found to be expressed by ENCODE RNA-seq data analysis.</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>b</sup>The reported transcript lengths were calculated by assuming as TSS the A or G residue closest to the position 12 bp upstream of the A box. To estimate the 3’ end of the transcript, both canonical (Tn with <em>n</em> ≥ 4) and non-canonical T-rich Pol III terminators [17] were considered both within and downstream of MIR body sequence (indicated in parentheses after the transcript length). For canonical terminators, the four Us corresponding to the first four Ts of the termination signal were considered as part of the transcripts; for non-canonical terminators, all the nucleotides of the terminator were considered as incorporated into the RNA. In the case of MIR_dup2285, for which two possible A boxes could drive transcription, the expected lengths of both putative alternative transcripts are indicated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1</a></span>).</p></div></div></div></div></p>
  <p>The above MIRs were transcribed <em>in vitro</em> using a HeLa cell nuclear extract in the presence of α-amanitin (2 µg<em>/</em>ml) to completely inhibit RNA polymerase II activity (<a reveal-id="dsw048-F5" data-open="dsw048-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Transcription reactions were also programmed in parallel with two mutant versions of each MIR element: one in which the B-box internal promoter element was inactivated by site-specific mutagenesis, and the other in which the upstream flanking region was deleted. Control transcription reactions were set with empty pGEM-T-Easy plasmid (<a reveal-id="dsw048-F5" data-open="dsw048-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, lanes 1 and 10) and the same vector carrying either a previously characterized <em>Alu</em> (<em>Alu</em>Sq2 chr1:61523296–61523586)<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> (lanes 2 and 11) or a tRNA<sup>Val</sup> (AAC) gene (TRNAV18, chr6) (lanes 3 and 12) whose transcription produces three different primary transcripts (of 87, 112 and 142 nt) because of heterogeneous termination at one of three consecutive termination signals.<a reveal-id="dsw048-B19" data-open="dsw048-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Each of the tested MIR elements produced a distinct pattern of transcription, in which the sizes of the most abundant transcripts agreed with those predicted on the basis of sequence inspection of the Pol III termination signals, either canonical (a run of at least four Ts) or non-canonical, both internal and in the 3′-flanking region (see <a reveal-id="dsw048-T3" data-open="dsw048-T3" class="link link-reveal link-table xref-fig">Table 3</a>). In particular, the MIR_dup2285 transcription unit has a predicted non-canonical terminator (T<sub>3</sub>CT) ∼200 bp downstream the 3′ end of the annotated element, as well as two strong early terminators, along with another non-canonical terminator, that seems to be skipped by the Pol III machinery (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1</a></span>). The first strong terminator (T<sub>4</sub>) is located within the annotated element 121 nt from the start coordinate, while the other one (T<sub>4</sub>), located 240 nt downstream of the 5′ end, is downstream of the annotated element but still within the corresponding expected full-length MIR. As shown in <a reveal-id="dsw048-F5" data-open="dsw048-F5" class="link link-reveal link-table xref-fig">Figure 5</a> (lane 4) all four transcripts, corresponding to the four termination signals, are identified during <em>in vitro</em> transcription, supporting the hypothesis of terminator skipping by Pol III. However it is worth noting that the most abundant signal arises from the transcript ending at the first strong Pol III terminator, suggesting a reduced tendency of the Pol III machinery complex to skip terminators of this MIR <em>in vitro</em> than <em>in vivo</em> (see profile in <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). A fifth transcript, migrating between 112 nt and 142 nt markers, is also identified as likely corresponding to a transcript ending at the first strong terminator but using an alternative A-box (see <a reveal-id="dsw048-T3" data-open="dsw048-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <div data-id="dsw048-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f5.png?Expires=1509239085&amp;Signature=JqBjUXQU7H7yHPJcLurHV5RvZYxVgjb0WYzuAvNqeLPv5DyVwWtBrLsl~JmR2CGbYligopUhCXq2OQ88SJc5NNV5ArW60VTwzvqyq~IefU-PoBpivKsv9zdYa7FgZn4SW5nXBEwcEnMcBcrbNBu5aUX9IVbsQPE-MQx-WufmCgSnH~7ZZPzKuF5ZN-Luat1uUXV5lNAKlkCrOk1m9S0KkOrklQtKzPNa52CMS6ybCoKPVUMzrqms9bNkUki0D4gu4Znm1QU9C0scLEv3nIwm~x2s1cXEK1174LJoV8S8tgCCOACA8vymwQ1k-SywCULyGJw9N0HhBFe6ProFx3f7Cg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="In vitro transcription for selected expression-positive MIRs. In vitro transcription reactions were performed in HeLa nuclear extract using 0.5 μg of the indicated MIR templates (lanes 4–9, 13–18). A previously characterized Alu producing a 355-nt RNA (lanes 2, 11) and a human tRNAVal gene producing a known transcript pattern due to heterogeneous transcription termination (lanes 3, 12)19 were used as positive controls for in vitro transcription and, at the same time, as a source of RNA size markers. Negative control reactions contained empty pGEM®-T Easy vector (lanes 1, 10). For each MIR, both the wild type, B box-mutated (Bmut) and 5′-flanking region (5′ del) version were tested. Indicated by arrows on the gel image are the migration positions (with lengths) of bands corresponding to the expected transcripts in Table 3." path-from-xml="dsw048f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463474/dsw048f5.tif" path-from-xml="dsw048f5.tif" target="_blank">View large</a><a section="61463474" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f5.png?Expires=1509239085&amp;Signature=ItLZPdOqapNAuDHFDyTArcb70qFPCcI~V-6JidRNwmL2tT9KiYQtzLnvA8XIy3hdDUQP53-8RmOqKVizW02u0s9iI9GteXLxSZndiy7BC1RAZs2zcGHA5v-shlpc0daUSudGpZYrMDO7qxPyHbNNaDBvdRp1I7N3KFq95~WDnBvpKtmuaYInSe~8xvAghclP91BYx-5VwXNMkTgC77YqSrhQENF7M5eSZhYG0sLiqgP530zWRpBnxnYOncrWKXxRDofxSEyqOWosBj5QIBZs9SUesQICyim5as-lHrXV0P5vex-sIoyIu5~OfBQPAtzYS1gnyYd0wemt~VcSa5UCqQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463474&amp;ar=2709598&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw048f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> transcription for selected expression-positive MIRs. <em>In vitro</em> transcription reactions were performed in HeLa nuclear extract using 0.5 μg of the indicated MIR templates (lanes 4–9, 13–18). A previously characterized <em>Alu</em> producing a 355-nt RNA (lanes 2, 11) and a human tRNA<sup>Val</sup> gene producing a known transcript pattern due to heterogeneous transcription termination (lanes 3, 12)<a reveal-id="dsw048-B19" data-open="dsw048-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> were used as positive controls for <em>in vitro</em> transcription and, at the same time, as a source of RNA size markers. Negative control reactions contained empty pGEM®-T Easy vector (lanes 1, 10). For each MIR, both the wild type, B box-mutated (<em>Bmut</em>) and 5′-flanking region (<em>5′ del</em>) version were tested. Indicated by arrows on the gel image are the migration positions (with lengths) of bands corresponding to the expected transcripts in <a reveal-id="dsw048-T3" data-open="dsw048-T3" class="link link-reveal link-table xref-fig">Table 3</a>.</p></div></div><div content-id="dsw048-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/m_dsw048f5.png?Expires=1509239085&amp;Signature=JqBjUXQU7H7yHPJcLurHV5RvZYxVgjb0WYzuAvNqeLPv5DyVwWtBrLsl~JmR2CGbYligopUhCXq2OQ88SJc5NNV5ArW60VTwzvqyq~IefU-PoBpivKsv9zdYa7FgZn4SW5nXBEwcEnMcBcrbNBu5aUX9IVbsQPE-MQx-WufmCgSnH~7ZZPzKuF5ZN-Luat1uUXV5lNAKlkCrOk1m9S0KkOrklQtKzPNa52CMS6ybCoKPVUMzrqms9bNkUki0D4gu4Znm1QU9C0scLEv3nIwm~x2s1cXEK1174LJoV8S8tgCCOACA8vymwQ1k-SywCULyGJw9N0HhBFe6ProFx3f7Cg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="In vitro transcription for selected expression-positive MIRs. In vitro transcription reactions were performed in HeLa nuclear extract using 0.5 μg of the indicated MIR templates (lanes 4–9, 13–18). A previously characterized Alu producing a 355-nt RNA (lanes 2, 11) and a human tRNAVal gene producing a known transcript pattern due to heterogeneous transcription termination (lanes 3, 12)19 were used as positive controls for in vitro transcription and, at the same time, as a source of RNA size markers. Negative control reactions contained empty pGEM®-T Easy vector (lanes 1, 10). For each MIR, both the wild type, B box-mutated (Bmut) and 5′-flanking region (5′ del) version were tested. Indicated by arrows on the gel image are the migration positions (with lengths) of bands corresponding to the expected transcripts in Table 3." path-from-xml="dsw048f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463474/dsw048f5.tif" path-from-xml="dsw048f5.tif" target="_blank">View large</a><a section="61463474" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048f5.png?Expires=1509239085&amp;Signature=ItLZPdOqapNAuDHFDyTArcb70qFPCcI~V-6JidRNwmL2tT9KiYQtzLnvA8XIy3hdDUQP53-8RmOqKVizW02u0s9iI9GteXLxSZndiy7BC1RAZs2zcGHA5v-shlpc0daUSudGpZYrMDO7qxPyHbNNaDBvdRp1I7N3KFq95~WDnBvpKtmuaYInSe~8xvAghclP91BYx-5VwXNMkTgC77YqSrhQENF7M5eSZhYG0sLiqgP530zWRpBnxnYOncrWKXxRDofxSEyqOWosBj5QIBZs9SUesQICyim5as-lHrXV0P5vex-sIoyIu5~OfBQPAtzYS1gnyYd0wemt~VcSa5UCqQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463474&amp;ar=2709598&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw048f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> transcription for selected expression-positive MIRs. <em>In vitro</em> transcription reactions were performed in HeLa nuclear extract using 0.5 μg of the indicated MIR templates (lanes 4–9, 13–18). A previously characterized <em>Alu</em> producing a 355-nt RNA (lanes 2, 11) and a human tRNA<sup>Val</sup> gene producing a known transcript pattern due to heterogeneous transcription termination (lanes 3, 12)<a reveal-id="dsw048-B19" data-open="dsw048-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> were used as positive controls for <em>in vitro</em> transcription and, at the same time, as a source of RNA size markers. Negative control reactions contained empty pGEM®-T Easy vector (lanes 1, 10). For each MIR, both the wild type, B box-mutated (<em>Bmut</em>) and 5′-flanking region (<em>5′ del</em>) version were tested. Indicated by arrows on the gel image are the migration positions (with lengths) of bands corresponding to the expected transcripts in <a reveal-id="dsw048-T3" data-open="dsw048-T3" class="link link-reveal link-table xref-fig">Table 3</a>.</p></div></div></p>
<p>The global transcription outputs of the four MIRs were roughly comparable (cf. lanes 4, 7, 13, and 16) indicating that their varying expression levels in cultured cells are not due to differences in <em>cis</em>-acting elements recognized by the basal Pol III transcription machinery. When the MIR B-box was mutationally inactivated (by substituting CG for the invariant TC dinucleotide of the B box consensus sequence GWTCRAnnC), a clear reduction in MIR transcription efficiency was observed in each case, thus confirming the importance of this element for MIR transcription. Remarkably, however, in no case was transcription abolished by B box inactivation. This was most evident for MIR_dup2285, whose B box-independent transcription was only 2.4-fold less than full transcription (cf. lanes 4 and 16). These data demonstrate that MIRs are efficiently transcribed by Pol III and confirms a key role for B box recognition by TFIIIC, even though the appreciable levels of residual transcription observed with B box-mutated MIRs provide evidence of a key role of A box (and possibly of 5′-flanking region) as a core promoter element in the assembly of the basal transcription machinery, even in the absence of a B box, as previously observed for other Pol III-transcribed genes in yeast.<a reveal-id="dsw048-B23" data-open="dsw048-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p><p>To understand if upstream regions influence MIR transcription as they do in the case of <em>Alu</em>s,<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> we compared the <em>in vitro</em> transcriptional activity of the four isolated MIRs with that of the corresponding 5′-deletion mutants. As shown in <a reveal-id="dsw048-F5" data-open="dsw048-F5" class="link link-reveal link-table xref-fig">Figure 5</a>, upstream sequence deletion negatively affected transcription to different extents for the different MIRs. Transcription of upstream deleted MIRs was reduced by 1.8- to 2.5-fold in the case of MIR_dup2285, MIR_dup3493 and MIRb_dup5848 (cf. lanes 4, 7, and 13 with lanes 6, 9 and 15, respectively), while it was only moderately reduced (∼1.3-fold) in the case of MIRc_dup2189 (cf. lanes 16 with 18). Overall these data reveal that MIR transcription is generally influenced by the upstream region, albeit at a lower extent than <em>Alu</em> transcription.</p>                            <h2 scrollto-destination=61463477 id="61463477" class="section-title" >Discussion</h2>
<p>This work provides, for the first time, a comprehensive account of transcriptionally active MIR loci in human cells and identifies MIR-derived transcripts representing novel ncRNAs in which a MIR fragment is fused with a MIR-unrelated, unique sequence. It also confirms the existence of ubiquitously transcribed MIR elements that could play a role in the regulation of their Pol II-transcribed overlapping genes.</p><p>The analysis of publicly available ENCODE RNA-seq datasets using an improved bioinformatic pipeline derived from our previous work<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> allowed us to define the Pol III-driven MIR transcriptome at single locus resolution using a search strategy that worked well especially for intergenic/antisense MIRs, whose coverage expression profiles where less biased than in the case of intronic sense MIRs by signals arising from sequencing of unknown ncRNA or hnRNA introns.</p><p>While a few previous studies on MIR elements focused on their genomic role as regulatory sequences, none of them tried to asses MIR expression <em>in vivo</em>, presumably because the only known role of MIR transcripts was that of retrotransposition and they ceased to amplify ∼130 myr ago.<a reveal-id="dsw048-B2" data-open="dsw048-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The only previous experimental evidence of MIR transcription by Pol III in mammals consists of the results of genome-wide localization analyses of the Pol III machinery in human cells<a reveal-id="dsw048-B11" data-open="dsw048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw048-B12" data-open="dsw048-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and in mouse.<a reveal-id="dsw048-B13" data-open="dsw048-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Our results reveal that MIR elements, although retrotranspositionally inactive, are widely transcribed and thus contribute to the ncRNA transcriptome of human cells.</p><p>Pol III transcription of MIRs occurs extensively both in intergenic regions and within introns of Pol II genes both in antisense and sense orientation. Pol III thus generates a MIR transcriptome composed mainly of cell-specific transcripts and, to a lesser extent, of a tiny subset of MIR-derived RNAs expressed in more than one cell line. Many of the observed MIR-derived transcripts arise from complete MIR elements that are transcribed along their entire length. However, we also found frequent cases of expressed MIRs corresponding to the left or right portion of the canonical full-length MIR whose transcription gives rise to chimeric RNAs composed of a MIR portion fused with a MIR-unrelated unique sequence from the genomic moiety downstream or upstream of the annotated MIR fragment. The genuine, Pol III-dependent character of expression-positive intergenic/antisense MIRs identified by our analysis is supported by their statistically significant enrichment of associated Pol III components within ENCODE ChIP-seq datasets for HeLa-S3 and K562 cells.</p><p>As previously observed for <em>Alu</em>s, we confirm the ability of the Pol III machinery to access SINE elements located in genomic regions transcribed by Pol II, both in the sense and antisense orientation, suggesting the possibility that Pol III-dependent MIR transcription might contribute to the regulation of the overlapping Pol II-transcribed genes. On the other hand, the high percentage of expression-positive MIRs within introns of both protein-coding and noncoding Pol II genes in antisense orientation (see <a reveal-id="dsw048-T1" data-open="dsw048-T1" class="link link-reveal link-table xref-fig">Table 1</a>) might explain their significant association with components of the Pol II machinery (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary Tables S2–S3</a></span>) and strengthens the notion of a close, functionally relevant association of Pol II and its TFs with Pol III genes.<a reveal-id="dsw048-B24" data-open="dsw048-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a></p><p>Particularly intriguing in this respect is the MIR element located within the first intron of POLR3E gene. This MIR was found to be expression-positive in all the seven cell lines we analyzed, and its intronic location within this gene appears to be conserved in mouse and other mammals (data not shown). Furthermore, its Pol III transcription is conserved at least in mouse.<a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> This MIR produces a chimeric transcript containing a MIR-unrelated downstream moiety, and its Pol III transcription is confirmed by the high number of associated Pol III components (see <a reveal-id="dsw048-F3" data-open="dsw048-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). Since the POLR3E gene codes for a subunit of Pol III (RPC5), it is tempting to speculate that a MIR-mediated negative autoregulation occurs at this locus, whereby Pol III transcribes the MIR element whose transcription in turn downregulates Pol III abundance through reduced Pol II-dependent expression of its RPC5 subunit.<a reveal-id="dsw048-B14" data-open="dsw048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>In an effort to elucidate basic features of MIR transcription mechanism and control, we studied the <em>in vitro</em> transcription properties of four expression-positive MIR loci, and we could establish three distinctive features of Pol III-dependent MIR transcription: (i) the importance of B-box promoter recognition by TFIIIC, but also the possibility of some levels of B box-independent MIR transcription; (ii) the influence of 5′-flanking sequence on MIR transcription; (iii) the strong epigenetic control of MIR expression <em>in vivo</em>, allowing for cell type-specific expression of MIRs containing equally strong basal promoters. Of the four MIRs, whose transcription properties were analyzed <em>in vitro</em> in this study, all showed a marked reduction of transcription ranging from 2.4 to 5.2-fold following mutation of the B-box promoter. Moreover three of them exhibited a 1.8 to 2.5-fold reduction of transcription, while only one was almost unaffected, upon deletion of the 5′-flanking region. The impact of the 5′-flanking region on MIR transcription efficiency is less marked than in the case of <em>Alu</em>s<a reveal-id="dsw048-B17" data-open="dsw048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> thus suggesting the possibility that the MIR internal promoter is intrinsically stronger.</p><p>The appreciable levels of residual transcription upon B box disruption, together with the modest reduction of transcription upon deletion of the 5′-flanking sequence, suggests that the assembly of the Pol III transcription machinery on MIRs might take place through alternative pathways, involving either the 5′ flank-A box pair or the A box-B box pair of <em>cis</em>-acting elements. We also cannot exclude that other factors, such as Pol II, TBP, and YY1, which were found to be enriched at expression-positive MIR loci, could take part in Pol III transcription of MIR elements <em>in vivo</em>.</p><p>The idea of an epigenetic control used by the cell to domesticate transposable elements has been widely accepted and used to explain their overall low expression levels despite their genomic abundance (reviewed in<a reveal-id="dsw048-B25" data-open="dsw048-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw048-B26" data-open="dsw048-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a>). Recently, SINE silencing has been proposed to more strictly depend on histone methylation rather than DNA methylation.<a reveal-id="dsw048-B27" data-open="dsw048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> This <em>in vivo</em> epigenetic silencing can be deduced from the same <em>in vitro</em> transcription levels of all the four cloned MIRs, whose expression profiles completely differ from each other in cell lines. Epigenetic control could also be responsible for the higher rate of strong terminator signals skipping by Pol III whose ability is only partially maintained <em>in vitro</em>.</p><p>Until now, MIRs have been studied from a genomic perspective while no efforts have been carried out to obtain insight into their actual in vivo expression and the role of Pol III machinery in their transcription. The present work reveals that these ancient transposable elements that have lost their retrotranspositional activity million years ago, are still widely transcribed in human cells. Their genome-wide expression profiling at single locus resolution represents another step toward the understanding of the mechanisms through which the cells have domesticated them, as well as of the possible roles played by both the act and the products of MIR transcription. The differential expression in different cell types of a large fraction of expression-positive MIR elements is suggestive of a strong epigenetic control which is specific to each cell type and condition, thus opening the possibility to exploit MIR expression profiling to monitor and discover patterns of epigenomic alterations that may accompany pathological states. The ever-increasing availability of RNA-seq datasets for many cell types in different physiological and pathological states represents a valuable source of information for our bioinformatic pipeline to assess, as anticipated for <em>Alu</em> RNAs,<a reveal-id="dsw048-B28" data-open="dsw048-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> MIR RNA profiles and their possible roles as a novel type of highly specific molecular signature of diseases.</p>                            <h2 scrollto-destination=61463488 id="61463488" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work used computational and storage services associated with the Hoffman2 Shared Cluster provided by UCLA Institute for Digital Research and Education’s Research Technology Group. We thank Martin Teichmann (IECB, Pessac, France) for HeLa cell nuclear extract. This work was supported by Italian Association for Cancer Research (AIRC, Grant IG16877 to G.D.).</p>                            <h3 scrollto-destination=61463490 id="61463490" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=61463492 id="61463492" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=61463494 id="61463494" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw048-b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Zietkiewicz</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Labuda</div> <div class="given-names">D.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Ubiquitous mammalian-wide interspersed repeats (MIRs) are molecular fossils from the mesozoic era,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">23</div>, <div class="fpage">170</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ubiquitous%20mammalian-wide%20interspersed%20repeats%20(MIRs)%20are%20molecular%20fossils%20from%20the%20mesozoic%20era%2C&amp;author=J.%20Jurka&amp;author=E.%20Zietkiewicz&amp;author=D.%20Labuda&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res&amp;volume=23&amp;pages=170-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.1.170" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.1.170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.1.170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7870583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smit</div> <div class="given-names">A.F.</div></div><div class="name string-name"><div class="surname">Riggs</div> <div class="given-names">A.D.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">MIRs are classic, tRNA-derived SINEs that amplified before the mammalian radiation,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">23</div>, <div class="fpage">98</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MIRs%20are%20classic%2C%20tRNA-derived%20SINEs%20that%20amplified%20before%20the%20mammalian%20radiation%2C&amp;author=A.F.%20Smit&amp;author=A.D.%20Riggs&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res&amp;volume=23&amp;pages=98-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.1.98" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.1.98" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.1.98"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7870595" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vassetzky</div> <div class="given-names">N.S.</div></div><div class="name string-name"><div class="surname">Kramerov</div> <div class="given-names">D.A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">SINEBase: a database and tool for SINE analysis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D83</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SINEBase%3A%20a%20database%20and%20tool%20for%20SINE%20analysis&amp;author=N.S.%20Vassetzky&amp;author=D.A.%20Kramerov&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1263" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23203982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Batzer</div> <div class="given-names">M.A.</div></div><div class="name string-name"><div class="surname">Deininger</div> <div class="given-names">P.L.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Alu repeats and human genomic diversity,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">3</div>, <div class="fpage">370</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alu%20repeats%20and%20human%20genomic%20diversity%2C&amp;author=M.A.%20Batzer&amp;author=P.L.%20Deininger&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet&amp;volume=3&amp;pages=370-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg798" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg798"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Terai</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Okada</div> <div class="given-names">N.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">SINE cousins: the 3'-end tails of the two oldest and distantly related families of SINEs are descended from the 3' ends of LINEs with the same genealogical origin,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">15</div>, <div class="fpage">1460</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SINE%20cousins%3A%20the%203'-end%20tails%20of%20the%20two%20oldest%20and%20distantly%20related%20families%20of%20SINEs%20are%20descended%20from%20the%203'%20ends%20of%20LINEs%20with%20the%20same%20genealogical%20origin%2C&amp;author=Y.%20Terai&amp;author=K.%20Takahashi&amp;author=N.%20Okada&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol&amp;volume=15&amp;pages=1460-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025873" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025873" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025873"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12572609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Labuda</div> <div class="given-names">D.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">CORE-SINEs: eukaryotic short interspersed retroposing elements with common sequence motifs,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">96</div>, <div class="fpage">2869</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CORE-SINEs%3A%20eukaryotic%20short%20interspersed%20retroposing%20elements%20with%20common%20sequence%20motifs%2C&amp;author=N.%20Gilbert&amp;author=D.%20Labuda&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=2869-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.6.2869" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.6.2869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.6.2869"> </span></div> </div></div></div></div></div><div content-id="dsw048-b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chalei</div> <div class="given-names">M. B.</div></div><div class="name string-name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Evolution of MIR Elements Located in the Coding Regions of Human Genome</div>, <div class="source">Mol Biol (Mosk)</div> , <div class="volume">35</div>, <div class="fpage">1023</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20MIR%20Elements%20Located%20in%20the%20Coding%20Regions%20of%20Human%20Genome&amp;author=M.%20B.%20Chalei&amp;author=E.%20V.%20Korotkov&amp;publication_year=2001&amp;journal=Mol%20Biol%20(Mosk)&amp;volume=35&amp;pages=1023-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11771126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sironi</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Menozzi</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Comi</div> <div class="given-names">G.P.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Gene function and expression level influence the insertion/fixation dynamics of distinct transposon families in mammalian introns</div>, <div class="source">Genome Biol</div> ., <div class="volume">7</div>, <div class="fpage">R120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20function%20and%20expression%20level%20influence%20the%20insertion%2Ffixation%20dynamics%20of%20distinct%20transposon%20families%20in%20mammalian%20introns&amp;author=M.%20Sironi&amp;author=G.%20Menozzi&amp;author=G.P.%20Comi&amp;publication_year=2006&amp;journal=Genome%20Biol&amp;volume=7&amp;pages=R120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-12-r120" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-12-r120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-12-r120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17181857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jjingo</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Conley</div> <div class="given-names">A. B.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Marino-Ramirez</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Lunyak</div> <div class="given-names">V.V.</div></div><div class="name string-name"><div class="surname">Jordan</div> <div class="given-names">I.K.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Mammalian-wide interspersed repeat (MIR)-derived enhancers and the regulation of human gene expression,</div> <div class="source">Mob. DNA</div> , <div class="volume">5</div>, <div class="fpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian-wide%20interspersed%20repeat%20(MIR)-derived%20enhancers%20and%20the%20regulation%20of%20human%20gene%20expression%2C&amp;author=D.%20Jjingo&amp;author=A.%20B.%20Conley&amp;author=J.%20Wang&amp;author=L.%20Marino-Ramirez&amp;author=V.V.%20Lunyak&amp;author=I.K.%20Jordan&amp;publication_year=2014&amp;journal=Mob.%20DNA&amp;volume=5&amp;pages=14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1759-8753-5-14" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1759-8753-5-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1759-8753-5-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25018785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schramm</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">N.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Recruitment of RNA polymerase III to its target promoters,</div> <div class="source">Genes Dev</div> ., <div class="volume">16</div>, <div class="fpage">2593</div>–<div class="lpage">620</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recruitment%20of%20RNA%20polymerase%20III%20to%20its%20target%20promoters%2C&amp;author=L.%20Schramm&amp;author=N.%20Hernandez&amp;publication_year=2002&amp;journal=Genes%20Dev&amp;volume=16&amp;pages=2593-620" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1018902" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1018902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1018902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12381659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Canella</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Praz</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Reina</div> <div class="given-names">J.H.</div></div><div class="name string-name"><div class="surname">Cousin</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">N.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Defining the RNA polymerase III transcriptome: Genome-wide localization of the RNA polymerase III transcription machinery in human cells,</div> <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">710</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20RNA%20polymerase%20III%20transcriptome%3A%20Genome-wide%20localization%20of%20the%20RNA%20polymerase%20III%20transcription%20machinery%20in%20human%20cells%2C&amp;author=D.%20Canella&amp;author=V.%20Praz&amp;author=J.H.%20Reina&amp;author=P.%20Cousin&amp;author=N.%20Hernandez&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=710-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.101337.109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.101337.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.101337.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20413673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oler</div> <div class="given-names">A. J.</div></div><div class="name string-name"><div class="surname">Alla</div> <div class="given-names">R. K.</div></div><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">D.N.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Human RNA polymerase III transcriptomes and relationships to Pol II promoter chromatin and enhancer-binding factors,</div> <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">17</div>, <div class="fpage">620</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20RNA%20polymerase%20III%20transcriptomes%20and%20relationships%20to%20Pol%20II%20promoter%20chromatin%20and%20enhancer-binding%20factors%2C&amp;author=A.%20J.%20Oler&amp;author=R.%20K.%20Alla&amp;author=D.N.%20Roberts&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=17&amp;pages=620-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1801" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1801"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20418882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carriere</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Graziani</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Alibert</div> <div class="given-names">O.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Genomic binding of Pol III transcription machinery and relationship with TFIIS transcription factor distribution in mouse embryonic stem cells,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">270</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20binding%20of%20Pol%20III%20transcription%20machinery%20and%20relationship%20with%20TFIIS%20transcription%20factor%20distribution%20in%20mouse%20embryonic%20stem%20cells%2C&amp;author=L.%20Carriere&amp;author=S.%20Graziani&amp;author=O.%20Alibert&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=270-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr737" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21911356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Canella</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Bernasconi</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Gilardi</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">A multiplicity of factors contributes to selective RNA polymerase III occupancy of a subset of RNA polymerase III genes in mouse liver,</div> <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">666</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20multiplicity%20of%20factors%20contributes%20to%20selective%20RNA%20polymerase%20III%20occupancy%20of%20a%20subset%20of%20RNA%20polymerase%20III%20genes%20in%20mouse%20liver%2C&amp;author=D.%20Canella&amp;author=D.%20Bernasconi&amp;author=F.%20Gilardi&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=666-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.130286.111" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.130286.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.130286.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22287103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Vicente-Garcia</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Seruggia</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">MIR retrotransposon sequences provide insulators to the human genome,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">E4428</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MIR%20retrotransposon%20sequences%20provide%20insulators%20to%20the%20human%20genome%2C&amp;author=J.%20Wang&amp;author=C.%20Vicente-Garcia&amp;author=D.%20Seruggia&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=E4428-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1507253112" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1507253112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1507253112"> </span></div> </div></div></div></div></div><div content-id="dsw048-b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Djebali</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Davis</div> <div class="given-names">C.A.</div></div><div class="name string-name"><div class="surname">Merkel</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Landscape of transcription in human cells,</div> <div class="source">Nature</div> , <div class="volume">489</div>, <div class="fpage">101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Landscape%20of%20transcription%20in%20human%20cells%2C&amp;author=S.%20Djebali&amp;author=C.A.%20Davis&amp;author=A.%20Merkel&amp;publication_year=2012&amp;journal=Nature&amp;volume=489&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11233" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conti</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Carnevali</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Bollati</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Fustinoni</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Pellegrini</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Dieci</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Identification of RNA polymerase III-transcribed Alu loci by computational screening of RNA-Seq data,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">817</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20RNA%20polymerase%20III-transcribed%20Alu%20loci%20by%20computational%20screening%20of%20RNA-Seq%20data%2C&amp;author=A.%20Conti&amp;author=D.%20Carnevali&amp;author=V.%20Bollati&amp;author=S.%20Fustinoni&amp;author=M.%20Pellegrini&amp;author=G.%20Dieci&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=817-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1361" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25550429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dignam</div> <div class="given-names">J.D.</div></div><div class="name string-name"><div class="surname">Lebovitz</div> <div class="given-names">R.M.</div></div><div class="name string-name"><div class="surname">Roeder</div> <div class="given-names">R.G.</div></div></div> <div class="year">1983</div>, 
<div class="article-title">Accurate transcription initiation by RNA polymerase II in a soluble extract from isolated mammalian nuclei,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">11</div>, <div class="fpage">1475</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20transcription%20initiation%20by%20RNA%20polymerase%20II%20in%20a%20soluble%20extract%20from%20isolated%20mammalian%20nuclei%2C&amp;author=J.D.%20Dignam&amp;author=R.M.%20Lebovitz&amp;author=R.G.%20Roeder&amp;publication_year=1983&amp;journal=Nucleic%20Acids%20Res&amp;volume=11&amp;pages=1475-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/11.5.1475" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F11.5.1475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F11.5.1475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6828386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Orioli</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Pascali</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Quartararo</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Widespread occurrence of non-canonical transcription termination by human RNA polymerase III,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">5499</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20non-canonical%20transcription%20termination%20by%20human%20RNA%20polymerase%20III%2C&amp;author=A.%20Orioli&amp;author=C.%20Pascali&amp;author=J.%20Quartararo&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=5499-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr074" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21421562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moqtaderi</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Raha</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Genomic binding profiles of functionally distinct RNA polymerase III transcription complexes in human cells,</div> <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">17</div>, <div class="fpage">635</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20binding%20profiles%20of%20functionally%20distinct%20RNA%20polymerase%20III%20transcription%20complexes%20in%20human%20cells%2C&amp;author=Z.%20Moqtaderi&amp;author=J.%20Wang&amp;author=D.%20Raha&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=17&amp;pages=635-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1794" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1794" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1794"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20418883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barski</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Chepelev</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Liko</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Pol II and its associated epigenetic marks are present at Pol III-transcribed noncoding RNA genes,</div> <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">17</div>, <div class="fpage">629</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pol%20II%20and%20its%20associated%20epigenetic%20marks%20are%20present%20at%20Pol%20III-transcribed%20noncoding%20RNA%20genes%2C&amp;author=A.%20Barski&amp;author=I.%20Chepelev&amp;author=D.%20Liko&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=17&amp;pages=629-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1806" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20418881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Orioli</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Pascali</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Pagano</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Teichmann</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Dieci</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">RNA polymerase III transcription control elements: themes and variations,</div> <div class="source">Gene</div> , <div class="volume">493</div>, <div class="fpage">185</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20polymerase%20III%20transcription%20control%20elements%3A%20themes%20and%20variations%2C&amp;author=A.%20Orioli&amp;author=C.%20Pascali&amp;author=A.%20Pagano&amp;author=M.%20Teichmann&amp;author=G.%20Dieci&amp;publication_year=2012&amp;journal=Gene&amp;volume=493&amp;pages=185-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2011.06.015" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2011.06.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2011.06.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21712079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guffanti</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Ferrari</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Preti</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">A minimal promoter for TFIIIC-dependent in vitro transcription of snoRNA and tRNA genes by RNA polymerase III,</div> <div class="source">J. Biol. Chem</div> ., <div class="volume">281</div>, <div class="fpage">23945</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20minimal%20promoter%20for%20TFIIIC-dependent%20in%20vitro%20transcription%20of%20snoRNA%20and%20tRNA%20genes%20by%20RNA%20polymerase%20III%2C&amp;author=E.%20Guffanti&amp;author=R.%20Ferrari&amp;author=M.%20Preti&amp;publication_year=2006&amp;journal=J.%20Biol.%20Chem&amp;volume=281&amp;pages=23945-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M513814200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M513814200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M513814200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16787917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Raha</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Moqtaderi</div> <div class="given-names">Z.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Close association of RNA polymerase II and many transcription factors with Pol III genes,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">3639</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Close%20association%20of%20RNA%20polymerase%20II%20and%20many%20transcription%20factors%20with%20Pol%20III%20genes%2C&amp;author=D.%20Raha&amp;author=Z.%20Wang&amp;author=Z.%20Moqtaderi&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=3639-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0911315106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0911315106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0911315106"> </span></div> </div></div></div></div></div><div content-id="dsw048-b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ichiyanagi</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Epigenetic regulation of transcription and possible functions of mammalian short interspersed elements, SINEs,</div> <div class="source">Genes Genet Syst</div> ., <div class="volume">88</div>, <div class="fpage">19</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20transcription%20and%20possible%20functions%20of%20mammalian%20short%20interspersed%20elements%2C%20SINEs%2C&amp;author=K.%20Ichiyanagi&amp;publication_year=2013&amp;journal=Genes%20Genet%20Syst&amp;volume=88&amp;pages=19-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.88.19" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.88.19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.88.19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23676707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roy-Engel</div> <div class="given-names">A.M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">LINEs, SINEs and other retroelements: do birds of a feather flock together?,</div> <div class="source">Front Biosci. (Landmark Ed)</div> , <div class="volume">17</div>, <div class="fpage">1345</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LINEs%2C%20SINEs%20and%20other%20retroelements%3A%20do%20birds%20of%20a%20feather%20flock%20together%3F%2C&amp;author=A.M.%20Roy-Engel&amp;publication_year=2012&amp;journal=Front%20Biosci.%20(Landmark%20Ed)&amp;volume=17&amp;pages=1345-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2741/3991" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2741%2F3991" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2741%2F3991"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22201808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Vavrova-Anderson</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Oler</div> <div class="given-names">A.J.</div></div><div class="name string-name"><div class="surname">Cowling</div> <div class="given-names">V.H.</div></div><div class="name string-name"><div class="surname">Cairns</div> <div class="given-names">B.R.</div></div><div class="name string-name"><div class="surname">White</div> <div class="given-names">R.J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">SINE transcription by RNA polymerase III is suppressed by histone methylation but not by DNA methylation,</div> <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">6569</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SINE%20transcription%20by%20RNA%20polymerase%20III%20is%20suppressed%20by%20histone%20methylation%20but%20not%20by%20DNA%20methylation%2C&amp;author=D.%20Varshney&amp;author=J.%20Vavrova-Anderson&amp;author=A.J.%20Oler&amp;author=V.H.%20Cowling&amp;author=B.R.%20Cairns&amp;author=R.J.%20White&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=6569" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms7569" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms7569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms7569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25798578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw048-b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw048-b28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carnevali</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Dieci</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Alu expression profiles as a novel RNA signature in biology and disease</div>, <div class="source">RNA Dis</div> ., <div class="volume">2</div>:<div class="fpage">e735</div>.</div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2709598&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw048/1/dsw048_Supp.zip?Expires=1509239085&amp;Signature=Pho8ybyk76hDgxCXCXmCidfpCUeSGP17bKG2n99SpPDKdHobTbhDBmWJ04gMfMr16mKmfkiMi5hF30aVh4w4k4GLcOtqVlCDI3vDxM5uWWZhP8~X-RkFjfua9UHBOU330mE08pWgNwfY3F4JoT6fyC5Ae48rJkwti5NymcN63g4xQtFmBFLFzzAipn2W7ZdDsHDynz7zaYt0l7LYtyzSjnTApJZM7xjZrVDi5~wn8RlkOpsFTk-SdQCuoT9D3y1eJWPEHX1UVy3U5tZnglZiUDyLcxMHuQ4kCCDQIfyK9WpnDOfO1LI~N6NESAT6LcD6IaMTQSdhsDQpYDOZT87~Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">240</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsw048"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2709598"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f59%2f2709598">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F59%2F2709598" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2709598'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Whole-genome expression analysis of mammalian-wide interspersed repeat elements in human cell lines" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f59%2f2709598">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

