

<!DOCTYPE html>
<html lang="en">

<head><title>
	Genome analysis of Hibiscus syriacus provides insights of polyploidization and indeterminate flowering in woody plants | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/71/2726856","@type":"ScholarlyArticle","name":"Genome analysis of  Hibiscus syriacus  provides insights of polyploidization and indeterminate flowering in woody plants","datePublished":"2016-12-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw049","keywords":["<italic>Hibiscus syriacus</italic>","Whole Genome Duplication","Diploidization","Multivoltinism","Homeolog"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/71/2726856","author":[{"name":"Kim, Yong-Min","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Kim, Seungill","affiliation":"2 Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea","@type":"Person"},{"name":"Koo, Namjin","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Shin, Ah-Young","affiliation":"3 Plant Systems Engineering Research Center, Korea Research Institute of Bioscience and Biotechnology (KRIBB), Daejeon 34141, Korea","@type":"Person"},{"name":"Yeom, Seon-In","affiliation":"4 Department of Agricultural Plant Science, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju 52828, Korea","@type":"Person"},{"name":"Seo, Eunyoung","affiliation":"2 Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea","@type":"Person"},{"name":"Park, Seong-Jin","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Kang, Won-Hee","affiliation":"4 Department of Agricultural Plant Science, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju 52828, Korea","@type":"Person"},{"name":"Kim, Myung-Shin","affiliation":"2 Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea","@type":"Person"},{"name":"Park, Jieun","affiliation":"2 Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea","@type":"Person"},{"name":"Jang, Insu","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Kim, Pan-Gyu","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Byeon, Iksu","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Kim, Min-Seo","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Choi, JinHyuk","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Ko, Gunhwan","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"},{"name":"Hwang, JiHye","affiliation":"5 National Institute of Agricultural Sciences, Rural Development Administration, Jeonju 54875, Korea","@type":"Person"},{"name":"Yang, Tae-Jin","affiliation":"2 Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea","@type":"Person"},{"name":"Choi, Sang-Bong","affiliation":"6 Division of Bioscience and Bioinformatics, Myongji University, Yongin 17058, Korea","@type":"Person"},{"name":"Lee, Je Min","affiliation":"7 Department of Horticultural Science, College of Agriculture and Life Science, Kyungpook National University, Daegu 41566, Korea","@type":"Person"},{"name":"Lim, Ki-Byung","affiliation":"7 Department of Horticultural Science, College of Agriculture and Life Science, Kyungpook National University, Daegu 41566, Korea","@type":"Person"},{"name":"Lee, Jungho","affiliation":"8 Green Plant Institute, Yongin 446-908, Korea","@type":"Person"},{"name":"Choi, Ik-Young","affiliation":"9 Department of Agriculture and Life Industry, Kangwon National University, Chuncheon 24341, Korea","@type":"Person"},{"name":"Park, Beom-Seok","affiliation":"5 National Institute of Agricultural Sciences, Rural Development Administration, Jeonju 54875, Korea","@type":"Person"},{"name":"Kwon, Suk-Yoon","affiliation":"3 Plant Systems Engineering Research Center, Korea Research Institute of Bioscience and Biotechnology (KRIBB), Daejeon 34141, Korea","@type":"Person"},{"name":"Choi, Doil","affiliation":"2 Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea","@type":"Person"},{"name":"Kim, Ryan W.","affiliation":"1 Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea","@type":"Person"}],"description":"Hibiscus syriacus (L.) (rose of Sharon) is one of the most widespread garden shrubs in the world. We report a draft of the H. syriacus genome comprised of a 1.75 Gb assembly that covers 92% of the genome with only 1.7% (33 Mb) gap sequences. Predicted gene modeling detected 87,603 genes, mostly supported by deep RNA sequencing data. To define gene family distribution among relatives of H. syriacus, orthologous gene sets containing 164,660 genes in 21,472 clusters were identified by OrthoMCL analysis of five plant species, including H. syriacus, Arabidopsis thaliana, Gossypium raimondii, Theobroma cacao and Amborella trichopoda. We inferred their evolutionary relationships based on divergence times among Malvaceae plant genes and found that gene families involved in flowering regulation and disease resistance were more highly divergent and expanded in H. syriacus than in its close relatives, G. raimondii (DD) and T. cacao. Clustered gene families and gene collinearity analysis revealed that two recent rounds of whole-genome duplication were followed by diploidization of the H. syriacus genome after speciation. Copy number variation and phylogenetic divergence indicates that WGDs and subsequent diploidization led to unequal duplication and deletion of flowering-related genes in H. syriacus and may affect its unique floral morphology.","pageStart":"71","pageEnd":"80","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217486&Signature=PAyXwnc2Kt4np0IZfiNll2Cr8OaCloRLf7UBGi8GW2U-6qEdvudzV3W0cQJgocI~lMvPF9MDGNaxqTQ-zLH4YQukjO9XZW6WXMrgJi2RjgvIQRE24CPdnt3R-aqsLLFh7RvOnEhuKbhMOtf64~PyFC05k6YpaAj3KXra~C64O-Dp9MzAw5DaRiu6vUHsxxgYHX2HYRd3wnJ9AD5y0Xg5TI4lqvIsRDy5nXM7nMVAUTQA11lJrj0b76SwJBID79Y9GUwXmgess1t-k97q4o2Vsvll0KS43UfCLt2b-7gOcZ-nWr~wTlt87ALcCzS7fFQVfUNq5WENnDLTPd3pR~OLcQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Genome analysis of  Hibiscus syriacus  provides insights of polyploidization and indeterminate flowering in woody plants","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217486&Signature=PAyXwnc2Kt4np0IZfiNll2Cr8OaCloRLf7UBGi8GW2U-6qEdvudzV3W0cQJgocI~lMvPF9MDGNaxqTQ-zLH4YQukjO9XZW6WXMrgJi2RjgvIQRE24CPdnt3R-aqsLLFh7RvOnEhuKbhMOtf64~PyFC05k6YpaAj3KXra~C64O-Dp9MzAw5DaRiu6vUHsxxgYHX2HYRd3wnJ9AD5y0Xg5TI4lqvIsRDy5nXM7nMVAUTQA11lJrj0b76SwJBID79Y9GUwXmgess1t-k97q4o2Vsvll0KS43UfCLt2b-7gOcZ-nWr~wTlt87ALcCzS7fFQVfUNq5WENnDLTPd3pR~OLcQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome analysis of  Hibiscus syriacus  provides insights of polyploidization and indeterminate flowering in woody plants | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Hibiscus syriacus (L.) (rose of Sharon) is one of the most widespread garden shrubs in the world. We report a draft of the H. syriacus genome comprised of a 1.75 Gb assembly that covers 92% of the genome with only 1.7% (33 Mb) gap sequences. Predicted gene modeling detected 87,603 genes, mostly supported by deep RNA sequencing data. To define gene family distribution among relatives of H. syriacus, orthologous gene sets containing 164,660 genes in 21,472 clusters were identified by OrthoMCL analysis of five plant species, including H. syriacus, Arabidopsis thaliana, Gossypium raimondii, Theobroma cacao and Amborella trichopoda. We inferred their evolutionary relationships based on divergence times among Malvaceae plant genes and found that gene families involved in flowering regulation and disease resistance were more highly divergent and expanded in H. syriacus than in its close relatives, G. raimondii (DD) and T. cacao. Clustered gene families and gene collinearity analysis revealed that two recent rounds of whole-genome duplication were followed by diploidization of the H. syriacus genome after speciation. Copy number variation and phylogenetic divergence indicates that WGDs and subsequent diploidization led to unequal duplication and deletion of flowering-related genes in H. syriacus and may affect its unique floral morphology." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/71/2726856" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217486&amp;Signature=GQwupm~jUN5ealQVAmkeWQm-Z2SfWIQgyng-1Zs-H5RKXoRb4huTy1DJT624lJX-o-Q0W3AzrEjTKYOSGCmkTIRpfK8lNVLOn-CmZ1Bt1Dd7Uazv8hqK~rBOpnRbnNyZ8gzn-8GG65FeHBa6ZctfRS51pO-Hg42V~HF-rrs0cveiOeJSM1GmNUIw94WaULLAJODnK8zPmdJPG3vczcBYzFkTeQqBx2aOO~L4z527GWccx3vdJh-7zn5tCL6nH53AhNHzgzIZK8LP~Dp73Y-ThpEXOouGg0tmy1X~qQf~AyN5c-OMfnshh8FlfyLKa0FDdp~rdV9XgsZD5DYlGOY4Eg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2016-12-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kim, Yong-Min" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Kim, Seungill" /><meta name="citation_author_institution" content="Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea" /><meta name="citation_author" content="Koo, Namjin" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Shin, Ah-Young" /><meta name="citation_author_institution" content="Plant Systems Engineering Research Center, Korea Research Institute of Bioscience and Biotechnology (KRIBB), Daejeon 34141, Korea" /><meta name="citation_author" content="Yeom, Seon-In" /><meta name="citation_author_institution" content="Department of Agricultural Plant Science, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju 52828, Korea" /><meta name="citation_author" content="Seo, Eunyoung" /><meta name="citation_author_institution" content="Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea" /><meta name="citation_author" content="Park, Seong-Jin" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Kang, Won-Hee" /><meta name="citation_author_institution" content="Department of Agricultural Plant Science, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju 52828, Korea" /><meta name="citation_author" content="Kim, Myung-Shin" /><meta name="citation_author_institution" content="Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea" /><meta name="citation_author" content="Park, Jieun" /><meta name="citation_author_institution" content="Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea" /><meta name="citation_author" content="Jang, Insu" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Kim, Pan-Gyu" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Byeon, Iksu" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Kim, Min-Seo" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Choi, JinHyuk" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Ko, Gunhwan" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_author" content="Hwang, JiHye" /><meta name="citation_author_institution" content="National Institute of Agricultural Sciences, Rural Development Administration, Jeonju 54875, Korea" /><meta name="citation_author" content="Yang, Tae-Jin" /><meta name="citation_author_institution" content="Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea" /><meta name="citation_author" content="Choi, Sang-Bong" /><meta name="citation_author_institution" content="Division of Bioscience and Bioinformatics, Myongji University, Yongin 17058, Korea" /><meta name="citation_author" content="Lee, Je Min" /><meta name="citation_author_institution" content="Department of Horticultural Science, College of Agriculture and Life Science, Kyungpook National University, Daegu 41566, Korea" /><meta name="citation_author" content="Lim, Ki-Byung" /><meta name="citation_author_institution" content="Department of Horticultural Science, College of Agriculture and Life Science, Kyungpook National University, Daegu 41566, Korea" /><meta name="citation_author" content="Lee, Jungho" /><meta name="citation_author_institution" content="Green Plant Institute, Yongin 446-908, Korea" /><meta name="citation_author" content="Choi, Ik-Young" /><meta name="citation_author_institution" content="Department of Agriculture and Life Industry, Kangwon National University, Chuncheon 24341, Korea" /><meta name="citation_author" content="Park, Beom-Seok" /><meta name="citation_author_institution" content="National Institute of Agricultural Sciences, Rural Development Administration, Jeonju 54875, Korea" /><meta name="citation_author" content="Kwon, Suk-Yoon" /><meta name="citation_author_institution" content="Plant Systems Engineering Research Center, Korea Research Institute of Bioscience and Biotechnology (KRIBB), Daejeon 34141, Korea" /><meta name="citation_author" content="Choi, Doil" /><meta name="citation_author_institution" content="Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea" /><meta name="citation_author" content="Kim, Ryan W." /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea" /><meta name="citation_title" content="Genome analysis of Hibiscus syriacus provides insights of polyploidization and indeterminate flowering in woody plants" /><meta name="citation_firstpage" content="71" /><meta name="citation_lastpage" content="80" /><meta name="citation_doi" content="10.1093/dnares/dsw049" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Deciduous shrubs for ozone bioindication: Hibiscus syriacus as an example;  citation_journal_title=Environ. Pollut;  citation_year=2009;  citation_volume=157;  citation_pages=865-70; " /><meta name="citation_reference" content="citation_title=Hibispeptin A, a novel cyclic peptide from Hibiscus syriacus;  citation_journal_title=Tetrahedron Lett;  citation_year=1998;  citation_volume=39;  citation_pages=993-6; " /><meta name="citation_reference" content="citation_title=Three naphthalenes from root bark of Hibiscus syriacus;  citation_journal_title=Phytochemistry;  citation_year=1998;  citation_volume=47;  citation_pages=799-802; " /><meta name="citation_reference" content="citation_title=Consequences of genome duplication;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2007;  citation_volume=17;  citation_pages=505-12; " /><meta name="citation_reference" content="citation_title=Polyploidy and genome evolution in plants;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2015;  citation_volume=35;  citation_pages=119-25; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=Two rounds of whole genome duplication in the ancestral vertebrate;  citation_journal_title=PLoS Biol;  citation_year=2005;  citation_volume=3;  citation_pages=e314" /><meta name="citation_reference" content="citation_title=Polyploid incidence and evolution;  citation_journal_title=Annu. Rev. Genet;  citation_year=2000;  citation_volume=34;  citation_pages=401-37; " /><meta name="citation_reference" content="citation_title=Plants with double genomes might have had a better chance to survive the Cretaceous-Tertiary extinction event;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=2009;  citation_volume=106;  citation_pages=5737-42; " /><meta name="citation_reference" content="citation_title=The draft genome of a diploid cotton Gossypium raimondii;  citation_journal_title=Nat. Genet;  citation_year=2012;  citation_volume=44;  citation_pages=1098-03; " /><meta name="citation_reference" content="citation_title=Genome sequence of the cultivated cotton Gossypium arboreum;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=567-72; " /><meta name="citation_reference" content="citation_title=Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=524-30; " /><meta name="citation_reference" content="citation_publisher=Allen &amp; Unwin, London;  citation_title=Chromosome Atlas of Flowering Plants;  citation_year=1955; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=FLASH: fast length adjustment of short reads to improve genome assemblies;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2957-63; " /><meta name="citation_reference" content="citation_title=Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=270-8; " /><meta name="citation_reference" content="citation_title=AUGUSTUS: ab initio prediction of alternative transcripts;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W435-9; " /><meta name="citation_reference" content="citation_title=The genome of Theobroma cacao;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=GeneWise and Genomewise;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=988-95; " /><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7" /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D130-135; " /><meta name="citation_reference" content="citation_title=Update on activities at the Universal Protein Resource (UniProt) in 2013; citation_author=UniProt, C;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D43-7; " /><meta name="citation_reference" content="citation_title=InterProScan–an integration platform for the signature-recognition methods in InterPro;  citation_journal_title=Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=847-8; " /><meta name="citation_reference" content="citation_title=SolexaQA: At-a-glance quality assessment of Illumina second-generation sequencing data;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=485" /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49" /><meta name="citation_reference" content="citation_title=Phylogeny-aware alignment with PRANK;  citation_journal_title=Methods Mol. Biol;  citation_year=2014;  citation_volume=1079;  citation_pages=155-70; " /><meta name="citation_reference" content="citation_title=Bayesian phylogenetics with BEAUti and the BEAST 1.7;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=1969-73; " /><meta name="citation_reference" content="citation_title=Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences;  citation_journal_title=Mol. Biol. Evol;  citation_year=2006;  citation_volume=23;  citation_pages=7-9; " /><meta name="citation_reference" content="citation_title=The conditioned reconstructed process;  citation_journal_title=J. Theor. Biol;  citation_year=2008;  citation_volume=253;  citation_pages=769-78; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D290-301; " /><meta name="citation_reference" content="citation_title=PlnTFDB: updated content and new features of the plant transcription factor database;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=D822-7; " /><meta name="citation_reference" content="citation_title=HMMER web server: interactive sequence similarity searching;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W29-37; " /><meta name="citation_reference" content="citation_title=Tracker: continuous HMMER and BLAST searching;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=388-9; " /><meta name="citation_reference" content="citation_title=SMART, a simple modular architecture research tool: identification of signaling domains;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=1998;  citation_volume=95;  citation_pages=5857-64; " /><meta name="citation_reference" content="citation_title=Pfam: the protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=Predicting coiled coils from protein sequences;  citation_journal_title=Science;  citation_year=1991;  citation_volume=252;  citation_pages=1162-4; " /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the tuber crop potato;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=189-95; " /><meta name="citation_reference" content="citation_title=Whole genome duplication events in plant evolution reconstructed and predicted using myosin motor proteins;  citation_journal_title=BMC Evol. Biol;  citation_year=2013;  citation_volume=13;  citation_pages=202" /><meta name="citation_reference" content="citation_title=Genome-wide analysis of the RNA helicase gene family in Gossypium raimondii;  citation_journal_title=Int. J. Mol. Sci;  citation_year=2014;  citation_volume=15;  citation_pages=4635-56; " /><meta name="citation_reference" content="citation_title=Capturing sequence variation among flowering-time regulatory gene homologues in the allopolyploid crop species Brassica napus;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5;  citation_pages=404" /><meta name="citation_reference" content="citation_title=Candidate gene identification of flowering time genes in cotton;  citation_journal_title=Plant Genome;  citation_year=2015;  citation_volume=8; " /><meta name="citation_reference" content="citation_title=Plant pathogens and integrated defence responses to infection;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=411;  citation_pages=826-33; " /><meta name="citation_reference" content="citation_title=Plant disease resistance genes: recent insights and potential applications;  citation_journal_title=Trends Biotechnol;  citation_year=2003;  citation_volume=21;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of NBS-LRR-encoding genes in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=809-34; " /><meta name="citation_reference" content="citation_title=Plant NB-LRR proteins: tightly regulated sensors in a complex manner;  citation_journal_title=Brief. Funct. Genomics;  citation_year=2015;  citation_volume=14;  citation_pages=233-42; " /><meta name="citation_reference" content="citation_title=Genome-wide comparative analyses reveal the dynamic evolution of nucleotide-binding leucine-rich repeat gene family among Solanaceae plants;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7; " /><meta name="citation_reference" content="citation_title=Fractionation and subfunctionalization following genome duplications: mechanisms that drive gene content and their consequences;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2015;  citation_volume=35;  citation_pages=110-8; " /><meta name="citation_reference" content="citation_title=Trends, rhythms, and aberrations in global climate 65 Ma to present;  citation_journal_title=Science;  citation_year=2001;  citation_volume=292;  citation_pages=686-93; " /><meta name="citation_reference" content="citation_title=Broad-spectrum mildew resistance in Arabidopsis thaliana mediated by RPW8;  citation_journal_title=Science;  citation_year=2001;  citation_volume=291;  citation_pages=118-20; " /><meta name="citation_reference" content="citation_title=Divergent evolution of plant NBS-LRR resistance gene homologues in dicot and cereal genomes;  citation_journal_title=J. Mol. Evol;  citation_year=2000;  citation_volume=50;  citation_pages=203-13; " /><meta name="citation_reference" content="citation_title=TIR-NBS-LRR genes are rare in monocots: evidence from diverse monocot orders;  citation_journal_title=BMC Res. Notes;  citation_year=2009;  citation_volume=2;  citation_pages=197" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/71/10731839/dsw049.pdf" /><meta name="description" content="Abstract.  Hibiscus syriacus (L.) (rose of Sharon) is one of the most widespread garden shrubs in the world. We report a draft of the H. syriacus genome compris" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/71/2726856" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/71/2726856" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsw049';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2726856" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFr54BPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIFlvbmctTWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzQxNDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBTZXVuZ2lsbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgU2NpZW5jZSwgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwgMDg4MjYsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29vLCBOYW1qaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLb3JlYW4gQmlvaW5mb3JtYXRpb24gQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgRGFlamVvbiAzNDE0MSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGluLCBBaC1Zb3VuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBsYW50IFN5c3RlbXMgRW5naW5lZXJpbmcgUmVzZWFyY2ggQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSAoS1JJQkIpLCBEYWVqZW9uIDM0MTQxLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illlb20sIFNlb24tSW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCBQbGFudCBTY2llbmNlLCBJbnN0aXR1dGUgb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZSwgR3llb25nc2FuZyBOYXRpb25hbCBVbml2ZXJzaXR5LCBKaW5qdSA1MjgyOCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTZW8sIEV1bnlvdW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAwODgyNiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJrLCBTZW9uZy1KaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLb3JlYW4gQmlvaW5mb3JtYXRpb24gQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgRGFlamVvbiAzNDE0MSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5nLCBXb24tSGVlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyYWwgUGxhbnQgU2NpZW5jZSwgSW5zdGl0dXRlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2UsIEd5ZW9uZ3NhbmcgTmF0aW9uYWwgVW5pdmVyc2l0eSwgSmluanUgNTI4MjgsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBNeXVuZy1TaGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAwODgyNiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJrLCBKaWV1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgU2NpZW5jZSwgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwgMDg4MjYsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmFuZywgSW5zdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhbiBCaW9pbmZvcm1hdGlvbiBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBEYWVqZW9uIDM0MTQxLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpbSwgUGFuLUd5dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhbiBCaW9pbmZvcm1hdGlvbiBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBEYWVqZW9uIDM0MTQxLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJ5ZW9uLCBJa3N1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzQxNDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBNaW4tU2VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzQxNDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvaSwgSmluSHl1ayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhbiBCaW9pbmZvcm1hdGlvbiBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBEYWVqZW9uIDM0MTQxLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvLCBHdW5od2FuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzQxNDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHdhbmcsIEppSHllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIEplb25qdSA1NDg3NSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW5nLCBUYWUtSmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAwODgyNiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaG9pLCBTYW5nLUJvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9zY2llbmNlIGFuZCBCaW9pbmZvcm1hdGljcywgTXlvbmdqaSBVbml2ZXJzaXR5LCBZb25naW4gMTcwNTgsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVlLCBKZSBNaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyYWwgU2NpZW5jZSwgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlLCBLeXVuZ3Bvb2sgTmF0aW9uYWwgVW5pdmVyc2l0eSwgRGFlZ3UgNDE1NjYsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGltLCBLaS1CeXVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgSG9ydGljdWx0dXJhbCBTY2llbmNlLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2UsIEt5dW5ncG9vayBOYXRpb25hbCBVbml2ZXJzaXR5LCBEYWVndSA0MTU2NiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIEp1bmdobyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyZWVuIFBsYW50IEluc3RpdHV0ZSwgWW9uZ2luIDQ0Ni05MDgsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvaSwgSWstWW91bmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIEluZHVzdHJ5LCBLYW5nd29uIE5hdGlvbmFsIFVuaXZlcnNpdHksIENodW5jaGVvbiAyNDM0MSwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJrLCBCZW9tLVNlb2siIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBSdXJhbCBEZXZlbG9wbWVudCBBZG1pbmlzdHJhdGlvbiwgSmVvbmp1IDU0ODc1LCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt3b24sIFN1ay1Zb29uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgU3lzdGVtcyBFbmdpbmVlcmluZyBSZXNlYXJjaCBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5IChLUklCQiksIERhZWplb24gMzQxNDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvaSwgRG9pbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgU2NpZW5jZSwgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwgMDg4MjYsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBSeWFuIFcuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzQxNDEsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUgYW5hbHlzaXMgb2YgSGliaXNjdXMgc3lyaWFjdXMgcHJvdmlkZXMgaW5zaWdodHMgb2YgcG9seXBsb2lkaXphdGlvbiBhbmQgaW5kZXRlcm1pbmF0ZSBmbG93ZXJpbmcgaW4gd29vZHkgcGxhbnRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNpZHVvdXMgc2hydWJzIGZvciBvem9uZSBiaW9pbmRpY2F0aW9uOiBIaWJpc2N1cyBzeXJpYWN1cyBhcyBhbiBleGFtcGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBQb2xsdXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTU3OyAgY2l0YXRpb25fcGFnZXM9ODY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWJpc3BlcHRpbiBBLCBhIG5vdmVsIGN5Y2xpYyBwZXB0aWRlIGZyb20gSGliaXNjdXMgc3lyaWFjdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRldHJhaGVkcm9uIExldHQ7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz05OTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhyZWUgbmFwaHRoYWxlbmVzIGZyb20gcm9vdCBiYXJrIG9mIEhpYmlzY3VzIHN5cmlhY3VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHl0b2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTc5OS04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcXVlbmNlcyBvZiBnZW5vbWUgZHVwbGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIEdlbmV0LiBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz01MDUtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHlwbG9pZHkgYW5kIGdlbm9tZSBldm9sdXRpb24gaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MTE5LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3JhcGV2aW5lIGdlbm9tZSBzZXF1ZW5jZSBzdWdnZXN0cyBhbmNlc3RyYWwgaGV4YXBsb2lkaXphdGlvbiBpbiBtYWpvciBhbmdpb3NwZXJtIHBoeWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDQ5OyAgY2l0YXRpb25fcGFnZXM9NDYzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byByb3VuZHMgb2Ygd2hvbGUgZ2Vub21lIGR1cGxpY2F0aW9uIGluIHRoZSBhbmNlc3RyYWwgdmVydGVicmF0ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seXBsb2lkIGluY2lkZW5jZSBhbmQgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NDAxLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudHMgd2l0aCBkb3VibGUgZ2Vub21lcyBtaWdodCBoYXZlIGhhZCBhIGJldHRlciBjaGFuY2UgdG8gc3Vydml2ZSB0aGUgQ3JldGFjZW91cy1UZXJ0aWFyeSBleHRpbmN0aW9uIGV2ZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTU3MzctNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgb2YgYSBkaXBsb2lkIGNvdHRvbiBHb3NzeXBpdW0gcmFpbW9uZGlpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTA5OC0wMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBjdWx0aXZhdGVkIGNvdHRvbiBHb3NzeXBpdW0gYXJib3JldW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz01NjctNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBjdWx0aXZhdGVkIFVwbGFuZCBjb3R0b24gKEdvc3N5cGl1bSBoaXJzdXR1bSBUTS0xKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGdlbm9tZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTUyNC0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFsbGVuICZhbXA7IFVud2luLCBMb25kb247ICBjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21lIEF0bGFzIG9mIEZsb3dlcmluZyBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTE5NTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBkZSBub3ZvIGFzc2VtYmx5IG9mIGhpZ2hseSBoZXRlcm96eWdvdXMgZ2Vub21lcyBmcm9tIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNob3J0IHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTM4NC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgcHJlLWFzc2VtYmxlZCBjb250aWdzIHVzaW5nIFNTUEFDRTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RkxBU0g6IGZhc3QgbGVuZ3RoIGFkanVzdG1lbnQgb2Ygc2hvcnQgcmVhZHMgdG8gaW1wcm92ZSBnZW5vbWUgYXNzZW1ibGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yOTU3LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGhvdCBwZXBwZXIgcHJvdmlkZXMgaW5zaWdodHMgaW50byB0aGUgZXZvbHV0aW9uIG9mIHB1bmdlbmN5IGluIENhcHNpY3VtIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0yNzAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVVHVVNUVVM6IGFiIGluaXRpbyBwcmVkaWN0aW9uIG9mIGFsdGVybmF0aXZlIHRyYW5zY3JpcHRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPVc0MzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBUaGVvYnJvbWEgY2FjYW87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZVdpc2UgYW5kIEdlbm9tZXdpc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz05ODgtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBldWthcnlvdGljIGdlbmUgc3RydWN0dXJlIGFubm90YXRpb24gdXNpbmcgRVZpZGVuY2VNb2RlbGVyIGFuZCB0aGUgUHJvZ3JhbSB0byBBc3NlbWJsZSBTcGxpY2VkIEFsaWdubWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbiA1OiBnZW5vbWUtc2NhbGUgcHJvdGVpbiBmdW5jdGlvbiBjbGFzc2lmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMjM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIFJlZmVyZW5jZSBTZXF1ZW5jZXMgKFJlZlNlcSk6IGN1cnJlbnQgc3RhdHVzLCBuZXcgZmVhdHVyZXMgYW5kIGdlbm9tZSBhbm5vdGF0aW9uIHBvbGljeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMTMwLTEzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXBkYXRlIG9uIGFjdGl2aXRpZXMgYXQgdGhlIFVuaXZlcnNhbCBQcm90ZWluIFJlc291cmNlIChVbmlQcm90KSBpbiAyMDEzOyBjaXRhdGlvbl9hdXRob3I9VW5pUHJvdCwgQzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1ENDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm9TY2Fu4oCTYW4gaW50ZWdyYXRpb24gcGxhdGZvcm0gZm9yIHRoZSBzaWduYXR1cmUtcmVjb2duaXRpb24gbWV0aG9kcyBpbiBJbnRlclBybzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz04NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29sZXhhUUE6IEF0LWEtZ2xhbmNlIHF1YWxpdHkgYXNzZXNzbWVudCBvZiBJbGx1bWluYSBzZWNvbmQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNFR01BOiBhIHBpcGVsaW5lIHRvIGFjY3VyYXRlbHkgYW5ub3RhdGUgY29yZSBnZW5lcyBpbiBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTA2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CVVNDTzogYXNzZXNzaW5nIGdlbm9tZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBjb21wbGV0ZW5lc3Mgd2l0aCBzaW5nbGUtY29weSBvcnRob2xvZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzIxMC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcnRob01DTDogaWRlbnRpZmljYXRpb24gb2Ygb3J0aG9sb2cgZ3JvdXBzIGZvciBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMTc4LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQ1NjYW5YOiBhIHRvb2xraXQgZm9yIGRldGVjdGlvbiBhbmQgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIGdlbmUgc3ludGVueSBhbmQgY29sbGluZWFyaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPWU0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VueS1hd2FyZSBhbGlnbm1lbnQgd2l0aCBQUkFOSzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA3OTsgIGNpdGF0aW9uX3BhZ2VzPTE1NS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmF5ZXNpYW4gcGh5bG9nZW5ldGljcyB3aXRoIEJFQVV0aSBhbmQgdGhlIEJFQVNUIDEuNzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTk2OS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hvb3NpbmcgYXBwcm9wcmlhdGUgc3Vic3RpdHV0aW9uIG1vZGVscyBmb3IgdGhlIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBwcm90ZWluLWNvZGluZyBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbmRpdGlvbmVkIHJlY29uc3RydWN0ZWQgcHJvY2VzczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gVGhlb3IuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjUzOyAgY2l0YXRpb25fcGFnZXM9NzY5LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQyOTAtMzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbG5URkRCOiB1cGRhdGVkIGNvbnRlbnQgYW5kIG5ldyBmZWF0dXJlcyBvZiB0aGUgcGxhbnQgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9RDgyMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ITU1FUiB3ZWIgc2VydmVyOiBpbnRlcmFjdGl2ZSBzZXF1ZW5jZSBzaW1pbGFyaXR5IHNlYXJjaGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMjktMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYWNrZXI6IGNvbnRpbnVvdXMgSE1NRVIgYW5kIEJMQVNUIHNlYXJjaGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zODgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U01BUlQsIGEgc2ltcGxlIG1vZHVsYXIgYXJjaGl0ZWN0dXJlIHJlc2VhcmNoIHRvb2w6IGlkZW50aWZpY2F0aW9uIG9mIHNpZ25hbGluZyBkb21haW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9NTg1Ny02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGZhbTogdGhlIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDIyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyBjb2lsZWQgY29pbHMgZnJvbSBwcm90ZWluIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTI7ICBjaXRhdGlvbl9wYWdlcz0xMTYyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBhbmQgYW5hbHlzaXMgb2YgdGhlIHR1YmVyIGNyb3AgcG90YXRvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc1OyAgY2l0YXRpb25fcGFnZXM9MTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBnZW5vbWUgZHVwbGljYXRpb24gZXZlbnRzIGluIHBsYW50IGV2b2x1dGlvbiByZWNvbnN0cnVjdGVkIGFuZCBwcmVkaWN0ZWQgdXNpbmcgbXlvc2luIG1vdG9yIHByb3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgRXZvbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIHRoZSBSTkEgaGVsaWNhc2UgZ2VuZSBmYW1pbHkgaW4gR29zc3lwaXVtIHJhaW1vbmRpaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBNb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTQ2MzUtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcHR1cmluZyBzZXF1ZW5jZSB2YXJpYXRpb24gYW1vbmcgZmxvd2VyaW5nLXRpbWUgcmVndWxhdG9yeSBnZW5lIGhvbW9sb2d1ZXMgaW4gdGhlIGFsbG9wb2x5cGxvaWQgY3JvcCBzcGVjaWVzIEJyYXNzaWNhIG5hcHVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz00MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYW5kaWRhdGUgZ2VuZSBpZGVudGlmaWNhdGlvbiBvZiBmbG93ZXJpbmcgdGltZSBnZW5lcyBpbiBjb3R0b247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBwYXRob2dlbnMgYW5kIGludGVncmF0ZWQgZGVmZW5jZSByZXNwb25zZXMgdG8gaW5mZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDExOyAgY2l0YXRpb25fcGFnZXM9ODI2LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBkaXNlYXNlIHJlc2lzdGFuY2UgZ2VuZXM6IHJlY2VudCBpbnNpZ2h0cyBhbmQgcG90ZW50aWFsIGFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIE5CUy1MUlItZW5jb2RpbmcgZ2VuZXMgaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz04MDktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IE5CLUxSUiBwcm90ZWluczogdGlnaHRseSByZWd1bGF0ZWQgc2Vuc29ycyBpbiBhIGNvbXBsZXggbWFubmVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmllZi4gRnVuY3QuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjMzLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBjb21wYXJhdGl2ZSBhbmFseXNlcyByZXZlYWwgdGhlIGR5bmFtaWMgZXZvbHV0aW9uIG9mIG51Y2xlb3RpZGUtYmluZGluZyBsZXVjaW5lLXJpY2ggcmVwZWF0IGdlbmUgZmFtaWx5IGFtb25nIFNvbGFuYWNlYWUgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyYWN0aW9uYXRpb24gYW5kIHN1YmZ1bmN0aW9uYWxpemF0aW9uIGZvbGxvd2luZyBnZW5vbWUgZHVwbGljYXRpb25zOiBtZWNoYW5pc21zIHRoYXQgZHJpdmUgZ2VuZSBjb250ZW50IGFuZCB0aGVpciBjb25zZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIEdlbmV0LiBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xMTAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlbmRzLCByaHl0aG1zLCBhbmQgYWJlcnJhdGlvbnMgaW4gZ2xvYmFsIGNsaW1hdGUgNjXigIlNYSB0byBwcmVzZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5MjsgIGNpdGF0aW9uX3BhZ2VzPTY4Ni05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJvYWQtc3BlY3RydW0gbWlsZGV3IHJlc2lzdGFuY2UgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgbWVkaWF0ZWQgYnkgUlBXODsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTE7ICBjaXRhdGlvbl9wYWdlcz0xMTgtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyZ2VudCBldm9sdXRpb24gb2YgcGxhbnQgTkJTLUxSUiByZXNpc3RhbmNlIGdlbmUgaG9tb2xvZ3VlcyBpbiBkaWNvdCBhbmQgY2VyZWFsIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTIwMy0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VElSLU5CUy1MUlIgZ2VuZXMgYXJlIHJhcmUgaW4gbW9ub2NvdHM6IGV2aWRlbmNlIGZyb20gZGl2ZXJzZSBtb25vY290IG9yZGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFJlcy4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8xLzcxLzEwNzMxODM5L2RzdzA0OS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSGliaXNjdXMgc3lyaWFjdXMgKEwuKSAocm9zZSBvZiBTaGFyb24pIGlzIG9uZSBvZiB0aGUgbW9zdCB3aWRlc3ByZWFkIGdhcmRlbiBzaHJ1YnMgaW4gdGhlIHdvcmxkLiBXZSByZXBvcnQgYSBkcmFmdCBvZiB0aGUgSC4gc3lyaWFjdXMgZ2Vub21lIGNvbXByaXMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI3MjY4NTYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2RkWWZacruUVNN62yyhhzUv0KVv8PY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f71%2f2726856" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f71%2f2726856" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1509239074&amp;Signature=O1ktQDiYhTUgGieNSsOKPFybP-S7tYfAswt0bxoH45Hs6KtTkwlKTIXj7HtsfpwtW5e89EUxGVRDuHYaSI-h8UuAjXonrh-WQGVTkq0-SyHw8EOYc3ayhn8zOXCJ4JWadZuNdTbXX0tz6vVk~RAdSlY9zN~MAH3SGv4wudm8MtEzo~KKEeOE35da95PawCUvVw6GMnE4B-W9KG3eZeHDMHj5nuklo4~KK0DhCtLGliZufHwOx-QXfjSOLKc5wQK7R36lythtym~27E~WnuYueyABkyTN0xQM~YXUdiX4lAIycK~rBMrPbJLluWASx4e046RJ0oDybOV2a5Fhki6FSA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61463501">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463501">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463503">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463503">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463507">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463507">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463528">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463528">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463544">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463544">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463550">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463550">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463552">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463552">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463554">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463556">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463556">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463559">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/59/2709598">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/81/2656160">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome analysis of <em>Hibiscus syriacus</em> provides insights of polyploidization and indeterminate flowering in woody plants
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Yong-Min Kim</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong-Min Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yong-Min+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kim author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Seungill Kim</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seungill Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Seungill+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kim author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Namjin Koo</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Namjin Koo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Namjin+Koo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koo N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Koo author:N.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Ah-Young Shin</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ah-Young Shin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Plant Systems Engineering Research Center, Korea Research Institute of Bioscience and Biotechnology (KRIBB), Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ah-Young+Shin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shin A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Shin author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Seon-In Yeom</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seon-In Yeom</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Department of Agricultural Plant Science, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju 52828, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Seon-In+Yeom">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yeom S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yeom author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Eunyoung Seo</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eunyoung Seo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Eunyoung+Seo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seo E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Seo author:E.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Seong-Jin Park</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seong-Jin Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Seong-Jin+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Park author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Won-Hee Kang</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Won-Hee Kang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Department of Agricultural Plant Science, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju 52828, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Won-Hee+Kang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kang author:W.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Myung-Shin Kim</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Myung-Shin Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Myung-Shin+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kim author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Jieun Park</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jieun Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jieun+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Park author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Insu Jang</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Insu Jang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Insu+Jang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jang I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Jang author:I.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Pan-Gyu Kim</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pan-Gyu Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Pan-Gyu+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kim author:P.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Iksu Byeon</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Iksu Byeon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Iksu+Byeon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Byeon I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Byeon author:I.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Min-Seo Kim</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Min-Seo Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Min-Seo+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kim author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">JinHyuk Choi</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">JinHyuk Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=JinHyuk+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Choi author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Gunhwan Ko</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gunhwan Ko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Gunhwan+Ko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ko G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Ko author:G.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">JiHye Hwang</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">JiHye Hwang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>National Institute of Agricultural Sciences, Rural Development Administration, Jeonju 54875, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=JiHye+Hwang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hwang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hwang author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Tae-Jin Yang</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tae-Jin Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Tae-Jin+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yang author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Sang-Bong Choi</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sang-Bong Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div>Division of Bioscience and Bioinformatics, Myongji University, Yongin 17058, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Sang-Bong+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Choi author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Je Min Lee</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Je Min Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div>Department of Horticultural Science, College of Agriculture and Life Science, Kyungpook National University, Daegu 41566, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Je+Min+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Lee author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Ki-Byung Lim</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ki-Byung Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div>Department of Horticultural Science, College of Agriculture and Life Science, Kyungpook National University, Daegu 41566, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ki-Byung+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Lim author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Jungho Lee</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jungho Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div>Green Plant Institute, Yongin 446-908, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jungho+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Lee author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Ik-Young Choi</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ik-Young Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div>Department of Agriculture and Life Industry, Kangwon National University, Chuncheon 24341, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ik-Young+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Choi author:I.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Beom-Seok Park</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Beom-Seok Park</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>National Institute of Agricultural Sciences, Rural Development Administration, Jeonju 54875, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Beom-Seok+Park">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Park B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Park author:B.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Suk-Yoon Kwon</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suk-Yoon Kwon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Plant Systems Engineering Research Center, Korea Research Institute of Bioscience and Biotechnology (KRIBB), Daejeon 34141, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Suk-Yoon+Kwon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kwon S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kwon author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Doil Choi</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Doil Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Plant Science, College of Agriculture and Life Sciences, Seoul National University, Seoul 08826, Korea</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Doil+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Choi author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Ryan W. Kim</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryan W. Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 34141, Korea</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw049-cor1" data-open="dsw049-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: 82-42-870-8500. Fax: 82-42-870-8519. Email: <a href="mailto:rwkim@kirbb.re.kr" target="_blank">rwkim@kirbb.re.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ryan+W.+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kim author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw049-FM1"><div class="fn"><div class="footnote-content"><sup>†</sup>These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 71–80, <a href='https://doi.org/10.1093/dnares/dsw049'>https://doi.org/10.1093/dnares/dsw049</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 December 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 October 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2726856" href="/dnaresearch/article-pdf/24/1/71/10731839/dsw049.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yong-Min Kim, Seungill Kim, Namjin Koo, Ah-Young Shin, Seon-In Yeom, Eunyoung Seo, Seong-Jin Park, Won-Hee Kang, Myung-Shin Kim, Jieun Park, Insu Jang, Pan-Gyu Kim, Iksu Byeon, Min-Seo Kim, JinHyuk Choi, Gunhwan Ko, JiHye Hwang, Tae-Jin Yang, Sang-Bong Choi, Je Min Lee, Ki-Byung Lim, Jungho Lee, Ik-Young Choi, Beom-Seok Park, Suk-Yoon Kwon, Doil Choi, Ryan W. Kim;  Genome analysis of <em>Hibiscus syriacus</em> provides insights of polyploidization and indeterminate flowering in woody plants, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 71–80, <a href='https://doi.org/10.1093/dnares/dsw049'>https://doi.org/10.1093/dnares/dsw049</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2726856?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20analysis%20of%20%26lt%3Bem%26gt%3BHibiscus%20syriacus%26lt%3B%2Fem%26gt%3B%20provides%20insights%20of%20polyploidization%20and%20indeterminate%20flowering%20in%20woody%20plants&amp;publicationDate=2016-12-21&amp;volumeNum=24&amp;issueNum=1&amp;author=Kim%2C%20Yong-Min%3B%20Kim%2C%20Seungill&amp;startPage=71&amp;endPage=80&amp;contentId=10.1093%2Fdnares%2Fdsw049&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f71%2f2726856" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61463501 id="61463501" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Hibiscus syriacus</em> (L.) (rose of Sharon) is one of the most widespread garden shrubs in the world. We report a draft of the <em>H. syriacus</em> genome comprised of a 1.75 Gb assembly that covers 92% of the genome with only 1.7% (33 Mb) gap sequences. Predicted gene modeling detected 87,603 genes, mostly supported by deep RNA sequencing data. To define gene family distribution among relatives of <em>H. syriacus</em>, orthologous gene sets containing 164,660 genes in 21,472 clusters were identified by OrthoMCL analysis of five plant species, including <em>H. syriacus</em>, <em>Arabidopsis thaliana, Gossypium raimondii, Theobroma cacao</em> and <em>Amborella trichopoda.</em> We inferred their evolutionary relationships based on divergence times among Malvaceae plant genes and found that gene families involved in flowering regulation and disease resistance were more highly divergent and expanded in <em>H. syriacus</em> than in its close relatives, <em>G. raimondii</em> (DD) and <em>T. cacao</em>. Clustered gene families and gene collinearity analysis revealed that two recent rounds of whole-genome duplication were followed by diploidization of the <em>H. syriacus</em> genome after speciation. Copy number variation and phylogenetic divergence indicates that WGDs and subsequent diploidization led to unequal duplication and deletion of flowering-related genes in <em>H. syriacus</em> and may affect its unique floral morphology.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Hibiscus syriacus&quot;"><em>Hibiscus syriacus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Whole Genome Duplication&quot;">Whole Genome Duplication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Diploidization">Diploidization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Multivoltinism">Multivoltinism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Homeolog">Homeolog</a></div>                            <h2 scrollto-destination=61463503 id="61463503" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p><em>Hibiscus syriacus</em> (L.) (rose of Sharon) is a fast-growing deciduous shrub of the Malvaceae family, which includes species such as <em>Gossypium raimondii</em> and <em>Theobroma cacao</em>. Although its’ name indicates this species was first identified in Syria, <em>H. syriacus</em> likely originated from the Korean peninsula and southern China and has since spread to Western countries. In temperate zones, <em>H. syriacus</em> is a commonly grown ornamental species with attractive white, pink, red, lavender, or purple flowers displayed over a long blooming period, though individual flowers last only a day. Its Korean name, <em>Mugunghwa</em>, literally means ‘flowering forever’. In addition to its ornamental value, <em>H. syriacus</em> acts as an ozone bioindicator,<a reveal-id="dsw049-B1" data-open="dsw049-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and its dried flowers and root bark are used in Oriental herbal medicines. Specifically, a novel cyclic peptide (Hibispeptin A) and three naphthalene compounds (syriacusins A-C) isolated from the plant’s root bark have been used as anti-pyretic, anti-helminthic and anti-fungal agents.<a reveal-id="dsw049-B2" data-open="dsw049-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw049-B3" data-open="dsw049-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a></p><p>Polyploidy is a well-established influence on plant genome evolution but is now recognized as a common phenomenon in diverse eukaryotes,<a reveal-id="dsw049-B4" data-open="dsw049-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> as signs of whole-genome duplication (WGD) have been detected in many sequenced genomes. Recent genome analysis demonstrated that most eudicot plants descended from an ancient hexaploid ancestor and followed lineage-specific polyploidization<a reveal-id="dsw049-B6" data-open="dsw049-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and that two rounds of WGD occurred in ancestral vertebrates.<a reveal-id="dsw049-B7" data-open="dsw049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In general, changes in ploidy are expected to be deleterious and an ‘evolutionary dead end’ for many species.<a reveal-id="dsw049-B8" data-open="dsw049-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> However, polyploidization of plants mediated their survival during the Cretaceous-Tertiary extinction event by increasing their genetic diversity.<a reveal-id="dsw049-B9" data-open="dsw049-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Each round of polyploidization was followed by many gene deletions (homeolog gene loss), interchromosomal rearrangements, neofunctionalization, and subfunctionalization.<a reveal-id="dsw049-B4" data-open="dsw049-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In Malvaceae plants, <em>Gossypium</em> includes five tetraploid taxa (AD<sub>1</sub> to AD<sub>5</sub>, 2<em>n</em><em>=</em> 4<em>x</em>) and 45 diploid taxa (2<em>n</em><em>=</em> 2<em>x</em>).<a reveal-id="dsw049-B10" data-open="dsw049-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Among them, <em>G. raimondii</em><a reveal-id="dsw049-B10" data-open="dsw049-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> (DD, D-genome), <em>G. arboreum</em><a reveal-id="dsw049-B11" data-open="dsw049-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> (AA, A-genome) and <em>G. hirsutum</em><a reveal-id="dsw049-B12" data-open="dsw049-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> (A<sub>t</sub>D<sub>t</sub>) genomes were reported. <em>Hibiscus</em> also includes many polyploid species, such as <em>H. syriacus</em> (2<em>n</em><em>=</em> 4<em>x</em><em>=</em> 80)<em>, H. aspera</em> (2<em>n</em><em>=</em> 8<em>x</em><em>=</em> 72), and <em>H. rosa-sinensis</em> (2<em>n</em><em>=</em> 16<em>x</em><em>=</em> 144) and diploid species [<em>H. pedunculatus</em> (2<em>n</em><em>=</em> 2<em>x</em><em>=</em> 30) and <em>H. phoeniceus</em> (2<em>n</em><em>=</em> 2<em>x</em><em>=</em> 22)].<a reveal-id="dsw049-B13" data-open="dsw049-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p><p>Here, we report the genome sequence of <em>H. syriacus</em> and the possible correlation between polyploidization and its phenotypes. Comparative genomic analysis of Malvaceae species, including <em>H. syriacus</em>, <em>T. cacao</em>, and <em>G. raimondii</em> (DD), provides clues of the recent polyploidization in <em>H. syriacus</em> by WGDs and unequal regulation of gene dosage by subsequent paleopolyploidy. Our investigation of copy number variations of floral regulators in Malvaceae plants also offers insight into the evolution of flowering phenotypes in <em>H. syriacus</em>. Moreover, the reference genome of <em>H. syriacus</em> is an important resource for identifying relationships between polyploidization and gene diversity. To our knowledge, this is the first report on whole genome sequence analysis of polyploidy woody plants and the effects of WGD on their unique phenotypes.</p>                            <h2 scrollto-destination=61463507 id="61463507" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=61463508 id="61463508" class="section-title" >2.1. Plant materials and whole genome sequencing</h3>
<p>Leaves of <em>H. syriacus</em> plants &gt;100-years-old and nominated as National Monument of Korea trees (serial number 520) were harvested and frozen immediately in liquid nitrogen. Genomic DNA for Paired-end (PE) and Mate-pair (MP) libraries was extracted, and libraries for next-generation sequencing were constructed according to the manufacturer’s instructions (Illumina, San Diego, CA, USA). The quality of each library was validated using the KAPA SYBR FAST Universal 2× qPCR Master Mix (Kapa Biosystems, Boston, MA, USA). Each library was sequenced with the Illumina HiSeq 2000 platform.</p>                            <h3 scrollto-destination=61463510 id="61463510" class="section-title" >2.2. Genome assembly, scaffolding and gap-closing</h3>
<p>Genome assembly was performed using both Platanus v1.2.1<a reveal-id="dsw049-B14" data-open="dsw049-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and SSPACE v2.0.<a reveal-id="dsw049-B15" data-open="dsw049-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> To generate longer initial contigs, single reads merged using FLASH v1.2.2<a reveal-id="dsw049-B16" data-open="dsw049-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> and reads from the PE libraries were assembled using Platanus with parameters to resolve heterozygosity in the <em>H. syriacus</em> genome (-u 0.2 -c 5 -d 0.3 -m 460). The scaffolding process was performed with Platanus and SSPACE. We first determined mapping seed length for scaffolding and then generated longer scaffolds using optimized Platanus parameters (-l 5 -s 41 -u 0.3). To extend scaffold length, SSPACE fulfilled serial scaffolding with hash parameters for the scaffolds generated by Platanus. Lastly, remaining gaps were filled with Platanus and GapCloser version 1.10 (<a class="link link-uri" href="http://soap.genomics.org.cn/down/GapCloser_release_2011.tar.gz" target="_blank">http://soap.genomics.org.cn/down/GapCloser_release_2011.tar.gz</a>) using reads from the PE and MP libraries.</p>                            <h3 scrollto-destination=61463512 id="61463512" class="section-title" >2.3. Genome annotation</h3>
<p>Annotation of the <em>H. syriacus</em> genome was performed using the KOBIC annotation pipeline (modified PGA pipeline<a reveal-id="dsw049-B17" data-open="dsw049-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>) consisting of repeat masking, mapping of different protein sequence sets, and <em>ab initio</em> prediction performed by AUGUSTUS v3.0.3.<a reveal-id="dsw049-B18" data-open="dsw049-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The protein sequences of <em>A. thaliana</em> (TAIR10, <a class="link link-uri" href="http://www.arabidopsis.org" target="_blank">http://www.arabidopsis.org</a>), <em>T. cacao</em><a reveal-id="dsw049-B19" data-open="dsw049-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and <em>G. raimondii</em><a reveal-id="dsw049-B10" data-open="dsw049-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> were mapped using GeneWise v2.1<a reveal-id="dsw049-B20" data-open="dsw049-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> to generate protein-based gene models for consensus modeling. AUGUSTUS was used for gene prediction in <em>H. syriacus.</em> Then predicted gene models from AUGUSTUS were validated using BLASTp with protein sequences from the three genomes (<em>T. cacao, G. raimondii and A. thaliana</em>) as queries and erratic gene models were filtered with a BLASTp cut-off value of query coverage ≥ 0.3. The predicted gene models from GeneWise were also filtered using query coverage ≥ 0.3. Remaining gene models of GeneWise depicted as GeneWise format were reformatted as GFF3 data and used to determine the consensus gene model via EVidenceModeler (EVM),<a reveal-id="dsw049-B21" data-open="dsw049-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> which combines <em>ab initio</em> gene predictions with protein alignments into weighted consensus gene structures (<em>ab initio</em> predictions <em>=</em> 1, protein alignment <em>=</em> 5, transcript alignment assemblies <em>=</em> 7). Biological functions of the final gene models were assigned using InterPro,<a reveal-id="dsw049-B22" data-open="dsw049-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> plant protein sequences in the RefSeq<a reveal-id="dsw049-B23" data-open="dsw049-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and UniProt databases,<a reveal-id="dsw049-B24" data-open="dsw049-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> which includes SWISS-PROT and TrEMBL data as described in previous study.<a reveal-id="dsw049-B17" data-open="dsw049-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> For functional annotation, three quality criteria were concerned: (i) bit score of the BLAST result is &gt;50 and <em>e</em>-value is &lt;<em>e</em>−10; (ii) subject coverage of the BLAST result is &gt;60%; and (iii) top token score from lexical analysis is &gt;0.5. To infer function for the protein-coding genes, we used InterProScan<a reveal-id="dsw049-B25" data-open="dsw049-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> version 5.4 to scan protein sequences against the protein signatures from InterPro.</p>                            <h3 scrollto-destination=61463514 id="61463514" class="section-title" >2.4. RNA sequencing and <em>de novo</em> transcriptome assembly</h3>
<p>Total RNA was extracted from plant leaves, petals, ovaries, and roots using TRIzol reagent (Invitrogen, CA, USA) following the manufacturer’s instructions. RNA-Seq libraries were generated using purified total RNA and sequenced using an Illumina HiSeq 2000 system. Thirty-six gigabases of raw reads were generated and preprocessed using DynamicTrim and LengthSort in SolexaQA.<a reveal-id="dsw049-B26" data-open="dsw049-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> The preprocessed raw reads were then used for transcriptome assembly and DEG analysis. Velvet v1.2.07 was used to assess <em>k</em>-mer sizes and assembled contigs, which were then merged using Oases v0.2.08. Assembled transcripts were validated using BLASTx (<em>e</em>-value &lt; <em>e</em>−10, best hit) against 1,917,424 protein sequences from 39 plant genomes selected from each family including <em>Arabidopsis thaliana, Brassica rapa, Solanum lycopersicum, Solanum phureja, T</em><em>.</em><em>cacao, G</em><em>.</em><em>raimondii, Oryza sativa, Zea mays, Cucumis sativus, Vitis vinifera</em>.</p>                            <h3 scrollto-destination=61463516 id="61463516" class="section-title" >2.5. Evaluation of genome assembly</h3>
<p>For validation of the assembled genome sequence, CEGMA (Core Eukaryotic Genes Mapping Apporoach) v2.5<a reveal-id="dsw049-B27" data-open="dsw049-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and BUSCO (Benchmarking Universal Single-Copy Orthologs) v1.22<sup>28</sup> were used in <em>H. syriacus</em> genome using default parameters. The CEGMA mapped a gene structures to new genomic sequence using a set of highly conserved protein family in eukaryotes by Hidden Markov Model. We evaluated 248 core eukaryotic genes defied by CEGMA to our genome sequence. The BUSCO provides completeness assessment of assembled genome based on orthologous group with single copy from OrthoDB (<a class="link link-uri" href="http://www.orthodb.org" target="_blank">http://www.orthodb.org</a>) using hidden Markov model for profile of amino acid alignments. For BUSCO assessments, we used 429 gene sets of conserved orthologs in eukaryotes.</p>                            <h3 scrollto-destination=61463518 id="61463518" class="section-title" >2.6. Detection of gene families in the <em>H. syriacus</em> genome</h3>
<p>OrthoMCL v2.0.2<a reveal-id="dsw049-B29" data-open="dsw049-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> was used to identify gene family clusters in <em>H. syriacus</em> and the other four sequenced genomes which are <em>G. raimondii, T. cacao, A. trichopoda</em><em>and A. thaliana</em> (In the first step, a set of high quality of gene models was obtained by rejecting low-quality sequences based on default parameters in OrthoMCL. The default parameters of rejecting low quality protein sequences were (i) shorter than 10 amino acids (ii) &gt;20% stop-codons (iii) &gt;20% non-standard amino acids. Pairwise sequence similarities between all input protein sequences were calculated by all-by-all BLASTp with an <em>e</em>-value cut-off of 1<em>e</em>−05 and a minimum match length of 50%. To define ortholog cluster structure, a Markov clustering algorithm was applied with an inflation value (−I) of 1.5 (default value in OrthoMCL). Putative splice variants were removed from the data set; longest protein sequences were kept and subsequently filtered for premature stop codons and incompatible sequences.</p>                            <h3 scrollto-destination=61463520 id="61463520" class="section-title" >2.7. Detection of collinearity blocks in Malvaceae plants</h3>
<p>MCScanX<a reveal-id="dsw049-B30" data-open="dsw049-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> was used to construct synteny and collinearity blocks between <em>H. syriacus</em> and <em>G. raimondii</em> against <em>T. cacao</em><em>.</em><a reveal-id="dsw049-B19" data-open="dsw049-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> First, homologous gene pairs were identified using protein sequences from the three genomes and then scanned inter- and intra-species by BLASTp (options with −e 1e−10 –b 5 –v 5). The BLASTp output was used with merged GFFs of three species to perform MCscanX with default parameters. We generated gene synteny and collinearity data to align proteins of the two species against reference chromosome of <em>T. cacao</em>. Collinearity blocks containing fewer than five proteins were excluded. To search a candidate of duplicated regions, we made the groups of collinear block from multiple collinear blocks which have similar protein members (&gt;80%), and the same chromosome in <em>T. cacao</em>. Then, each block in <em>H. syriacus</em> and <em>G. raimondii</em> was counted by overlap against the cluster blocks of <em>T. cacao</em>. The duplicated regions in <em>H. syriacus</em> and <em>G. raimondii w</em>ere identified, if the number of blocks was more than two.</p>                            <h3 scrollto-destination=61463522 id="61463522" class="section-title" >2.8. Estimation of speciation time in Malvaceae plants</h3>
<p>To construct a phylogenetic tree of the five species (<em>A. trichopoda</em>, <em>A. thaliana</em>, <em>G. raimondii</em>, <em>T. cacao</em> and <em>H. syriacus</em>), we extracted 941 single-copy gene sets from all genomes in the OrthoMCL clusters. We performed multiple alignments of the CDSs of each gene set using Prank (−f <em>=</em> nexus -codon).<a reveal-id="dsw049-B31" data-open="dsw049-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The alignment file was used to construct a phylogenetic tree based on calculations of divergence time for the five species.<a reveal-id="dsw049-B32" data-open="dsw049-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> For accurate tree construction, we assigned taxon sets based on previously calculated speciation of <em>A. thaliana</em>, <em>G. raimondii</em> and <em>T. cacao</em>. The Bayesian software package BEAST<a reveal-id="dsw049-B32" data-open="dsw049-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> (v1.8.2) was used to estimate divergence times and construct the final tree. The Markov chain Monte Carlo (MCMC) analyses in BEAST was conducted for 10 million generations with samples every 1,000 steps and the effective sampling size was over 150 for all of parameters. We used SRD06<a reveal-id="dsw049-B33" data-open="dsw049-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> as a substitution model and the Yule process<a reveal-id="dsw049-B34" data-open="dsw049-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> as a traditional speciation model.</p>                            <h3 scrollto-destination=61463524 id="61463524" class="section-title" >2.9. Identification of TF candidates</h3>
<p>We identified TF candidates as previously described.<a reveal-id="dsw049-B17" data-open="dsw049-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Briefly, predicted proteins containing TF domains were screened by InterProScan<a reveal-id="dsw049-B22" data-open="dsw049-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> search against Pfam<a reveal-id="dsw049-B35" data-open="dsw049-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> databases. The TF candidates were classified based on rules as indicated at PlnTFDB (<a class="link link-uri" href="http://plntfdb.bio.uni-potsdam.de/v3.0/rules.php" target="_blank">http://plntfdb.bio.uni-potsdam.de/v3.0/rules.php</a>, Rules for the classification of TFs).<a reveal-id="dsw049-B36" data-open="dsw049-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In the case of TF gene families that don’t have Pfam ID, domain alignments as Clustal format were downloaded from PlnTFDB and Hidden Markov Model profiles were built and screened using HMMER.<a reveal-id="dsw049-B37" data-open="dsw049-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> The assigned TF candidates were confirmed by BLASTp against plant TF protein sequences downloaded from PlnTFDB (<a class="link link-uri" href="http://plntfdb.bio.uni-potsdam.de/v3.0/downloads.php" target="_blank">http://plntfdb.bio.uni-potsdam.de/v3.0/downloads.php</a>).</p>                            <h3 scrollto-destination=61463526 id="61463526" class="section-title" >2.10. Identification of genes encoding nucleotide-binding site proteins</h3>
<p>To identify nucleotide-binding site (NBS)-encoding genes, representative genes from each plant genome were screened using the raw Hidden Markov Model (HMMER3.0)<a reveal-id="dsw049-B38" data-open="dsw049-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> to search for the Pfam NBS family PF00931 domain (<em>e</em>-value cut-off of 1.0). All putative NBS protein sequences were analysed and manually curated by a BLASTp search against known <em>R</em> gene sequences in GenBank. To further identify TIR homologs and sequences encoding CC and LRR motifs, candidate NBS-LRR protein sequences were characterized using SMART,<a reveal-id="dsw049-B39" data-open="dsw049-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> the Pfam database<a reveal-id="dsw049-B40" data-open="dsw049-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and the COILS programme<a reveal-id="dsw049-B41" data-open="dsw049-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> with a threshold of 0.9 to detect CC domain specifically.</p>                            <h2 scrollto-destination=61463528 id="61463528" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=61463529 id="61463529" class="section-title" >3.1. Genome sequencing and assembly</h3>
<p><em>H. syriacus</em> plants over 100-years-old were selected for genome sequencing. Illumina whole-genome shotgun sequencing generated 233.3 Gb (122.8× coverage) of genomic sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>). PE libraries (250–500 bp) were generated, and 2 kb and 5 kb MP libraries were sequenced with a read length of 101 or 151 bp. Pre-processing analysis of raw sequences was performed to remove extraneous sequences for accurate genome assembly as described in the previous report.<a reveal-id="dsw049-B17" data-open="dsw049-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> After filtering, 156.6 Gb (82.4× coverage) of <em>H. syriacus</em> genome sequences were used for further analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span>). <em>K</em>-mer distribution analysis, which provides information related to low frequencies, sequencing depth, level of heterozygosity, and genome size<a reveal-id="dsw049-B42" data-open="dsw049-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> was then applied using Jellyfish (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1</a></span>). The estimated genome size of 1,901 Mb was calculated by dividing the total volume by the peak of distribution as described previously.<a reveal-id="dsw049-B17" data-open="dsw049-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p><p>Validation of the assembled genome was performed using 128,888 representative transcripts derived from <em>de novo</em> assembly of a combined transcriptome from all libraries (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S3</a></span>). To confirm sequence alignment between transcriptome assembly and scaffolds, we performed BLAST comparisons for the transcriptome assembly and scaffolds as queries and subjects, respectively. We found that 117,431 (91.1%) assembled transcripts as query sequences matched to scaffolds based with 98% identity. In addition, 93,688 (79.8%) transcripts matched to genome sequence with query coverage over 80%, and 82,394 (70.2%) transcripts matched over 90% coverage by the assembled scaffolds (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. 2</a></span>). The quality of the assembly was also evaluating using CEGMA<a reveal-id="dsw049-B27" data-open="dsw049-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> (Core Eukaryotic Genes Mapping Approach) and BUSCO<a reveal-id="dsw049-B28" data-open="dsw049-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> (Benchmarking Universal Single-Copy Orthologs). These analyses showed 92.74% of completeness (230 of 248 CEGs) from CEGMA and 92% of complete BUSCOs. These results suggested that <em>H. syriacus</em> genome assembly was high quality. As a result, 1,748 Mb (91.9% of 1,901 Mb) of genomic sequences were assembled into 77,492 scaffolds. The assembled genome was comprised of 33 Mb (1.9%) gap sequences and 1,715 Mb of contigs with N50 <em>=</em> 30 kb (<a reveal-id="dsw049-T1" data-open="dsw049-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S4</a></span>). <div class="table-wrap"><div id="dsw049-T1" data-id="dsw049-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of <em>H. syriacus</em> genome assembly</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of scaffolds </td><td>77,492 </td></tr><tr><td>Total length of scaffolds </td><td>1,748 Mb </td></tr><tr><td>N50 of scaffolds </td><td>140 kb </td></tr><tr><td>Longest (shortest) length of scaffolds </td><td>1.54 Mb (500 bp) </td></tr><tr><td>Number of contigs </td><td>172,672 </td></tr><tr><td>Total length of contigs </td><td>1,715 Mb </td></tr><tr><td>N50 of contigs </td><td>30.0 kb </td></tr><tr><td>Longest (shortest) length of contigs </td><td>643 kb (87 bp) </td></tr><tr><td>Number of gap sequences </td><td>33 Mb (1.9%) </td></tr><tr><td>GC content </td><td>34.04% </td></tr><tr><td>Total size of TEs </td><td>1,095 Mb (57.6%) </td></tr></tbody></table></div><div content-id="dsw049-T1" class="table-modal"><table><tbody><tr><td>Number of scaffolds </td><td>77,492 </td></tr><tr><td>Total length of scaffolds </td><td>1,748 Mb </td></tr><tr><td>N50 of scaffolds </td><td>140 kb </td></tr><tr><td>Longest (shortest) length of scaffolds </td><td>1.54 Mb (500 bp) </td></tr><tr><td>Number of contigs </td><td>172,672 </td></tr><tr><td>Total length of contigs </td><td>1,715 Mb </td></tr><tr><td>N50 of contigs </td><td>30.0 kb </td></tr><tr><td>Longest (shortest) length of contigs </td><td>643 kb (87 bp) </td></tr><tr><td>Number of gap sequences </td><td>33 Mb (1.9%) </td></tr><tr><td>GC content </td><td>34.04% </td></tr><tr><td>Total size of TEs </td><td>1,095 Mb (57.6%) </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=61463532 id="61463532" class="section-title" >3.2. Genome annotation</h3>
<p>Annotation of the <em>H. syriacus</em> genome was performed using the KOBIC Genome Annotation pipeline (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S3</a></span>), including masking repetitive sequences, transcriptome mapping, reference protein mapping using GeneWise, <em>ab initio</em> gene prediction, and determination of consensus gene models using EVM.</p><p>Before masking repetitive sequences, repeat annotation was performed by RepeatModeler and RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a>) for the assembled genome. Due to a lack of repeat sequence information for this genome, we constructed a <em>de novo</em> repeat library using RepeatModeler, and RepeatMasker was applied for annotation of the constructed repeat library. Repeat sequences, except for unknown transposable elements (TEs), were masked so we could identify essential gene families, such as those that encode receptor-like kinases and nucleotide-binding proteins. TEs comprised 1,095.8 Mb (57.6%) of the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S5</a></span>) and mostly included long terminal repeats (LTRs), which accounted for approximately 30% of total TEs. Gypsy and Copia retrotransposons were the most common LTRs detected.</p><p>Transcripts mapping was performed using TopHat and Cufflinks, and protein alignment was performed by GeneWise. The protein sequences of <em>A</em><em>.</em><em>thaliana</em> (TAIR10), <em>T. cacao</em> and <em>G. raimondii</em> were mapped to generate protein-based gene models. For annotation of duplicated genes or gene families, mapping regions of a reference protein in the <em>H. syriacus</em> genome were determined from tBLASTn (default <em>e</em>-value 10) results using custom Perl scripts. These steps prevented mis-annotation of duplicated genes due to lack of mapping data for reference proteins from parsing single best-matched regions in the <em>H. syriacus</em> genome. We annotated 87,603 genes using KOBIC annotation prediction with an average CDS length of 1,188 bp, similar to that for <em>G. raimondii</em> (<a reveal-id="dsw049-T2" data-open="dsw049-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Consensus gene models were evaluated using 88.4 Gb of Illumina-derived RNA-Seq data. Overall, 91.76% of the predicted coding sequences were supported by Illumina data, demonstrating the high accuracy of KOBIC annotation prediction. The <em>H. syriacus</em> genome contains two times more genes than <em>G. raimondii</em> and three times more genes than <em>T. cacao</em> (<a reveal-id="dsw049-T2" data-open="dsw049-T2" class="link link-reveal link-table xref-fig">Table 2</a>), suggesting a polyploid genome as first indicated in a previous report.<a reveal-id="dsw049-B13" data-open="dsw049-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><div class="table-wrap"><div id="dsw049-T2" data-id="dsw049-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Statistics of <em>H. syriacus</em> gene models</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Protein-coding loci </th><th>Total CDS length (bp) </th><th>Avg. CDS length (bp) </th><th>Avg. Exon length (bp) </th><th>Avg. Intron length (bp) </th></tr></thead><tbody><tr><td><em>H. syriacus</em> </td><td>87,603 </td><td>104,087,809 </td><td>1,188 </td><td>239 </td><td>383 </td></tr><tr><td><em>T. cacao</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>28,798 </td><td>33,494,538 </td><td>1,857 </td><td>231 </td><td>502 </td></tr><tr><td><em>G. raimondii</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>40,976 </td><td>45,237,504 </td><td>1,104 </td><td>244 </td><td>339 </td></tr><tr><td><em>A. thaliana</em><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>27,206 </td><td>24,861,465 </td><td>1,212 </td><td>265 </td><td>164 </td></tr></tbody></table></div><div content-id="dsw049-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Protein-coding loci </th><th>Total CDS length (bp) </th><th>Avg. CDS length (bp) </th><th>Avg. Exon length (bp) </th><th>Avg. Intron length (bp) </th></tr></thead><tbody><tr><td><em>H. syriacus</em> </td><td>87,603 </td><td>104,087,809 </td><td>1,188 </td><td>239 </td><td>383 </td></tr><tr><td><em>T. cacao</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>28,798 </td><td>33,494,538 </td><td>1,857 </td><td>231 </td><td>502 </td></tr><tr><td><em>G. raimondii</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>40,976 </td><td>45,237,504 </td><td>1,104 </td><td>244 </td><td>339 </td></tr><tr><td><em>A. thaliana</em><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td><td>27,206 </td><td>24,861,465 </td><td>1,212 </td><td>265 </td><td>164 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Cacao genome paper<sup>19</sup></p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Cotton genome paper<sup>10</sup></p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>TAIR10 annotation (<a class="link link-uri" href="http://www.arabidopsis.org" target="_blank">http://www.arabidopsis.org</a>)</p></div></div></div></div></p><p>We also performed OrthoMCL analysis to detect orthologous genes among Malvaceae plants, <em>A. thaliana</em>, and <em>Amborella trichopod</em>a. We identified 21,472 orthologous gene sets containing 164,660 genes, 5,300 of which were <em>H. syriacus</em>-specific (<a reveal-id="dsw049-F1" data-open="dsw049-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Interestingly, these genes were the number of gene was three times larger than those of <em>G. raimondii</em> and <em>T. cacao</em>, further indicating <em>H. syriacus</em>’ polyploidy. In addition, relatively large numbers of singletons in the genomes of <em>A. thaliana</em> and <em>A. trichopoda</em> suggest that the Malvaceae lineage diverged long ago and now shows a high degree of evolutionary distance from other eudicots. For further analysis, estimation of speciation time and comparison of genome structures among Malvaceae plants were performed using paired gene sets. <div data-id="dsw049-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f1.png?Expires=1509239087&Signature=VfjqT2TWePlhIW61TEeIAlrz9bexNJImLrVd3ik3B385I4~svzA3hB6zS4urnQ2g4MuSnrQjbBDg94lcRFyD93DC4l0E4Sdn6W2CQQLC0iX~GK~wBl7ZuhzoLXdXvFnpZKRJgBSCFaKDfRNL3LkXC6QfHShzIE7hBHHEp~D1qkgAuT0HSTt3lTpYqDzKgwUe~GDtn1VtmhWDYUrws9EdG6rFvfVtOPv5rxtB4t0cS-GT34fYQXVXYb1iYuTkZ5~Aol8ZkrhIH8gXE6sjq3~Y-apxI~MyiAVJuU7Z-OZyElUiNnyUR9VkruHeEfNlmih8sZi3vc3y8WqSaAYvKT8u6w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of orthologous gene families of H. syriacus, G. raimondii, T. cacao, A. trichopoda and A. thaliana, from which 169,570 sequences were clustered into 9,076 groups. The number of clustered groups and genes in each species are shown on the left and center, and total gene numbers are shown on the right." path-from-xml="dsw049f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463536/dsw049f1.tif" path-from-xml="dsw049f1.tif" target="_blank">View large</a><a section="61463536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f1.png?Expires=1509239087&Signature=CZQstR2udmaf3ucsJ76YNEVN0abCfV9PZizVptgxWzDB~esXIjJ1B7KxeWT8adKyik3zpMmorclypK7vqV-y8LZU4CMBCkE1363lPhBXEkJk8FkyizbLlAgQHrbZ7PNo9toKQczNEI-JVkUaNbSZN2O1s9nsDYoBR6bM0Hej2LSZ8HOZFxlzy-njgO6DYFJ1-ebpfGZR6LGUJ9XbJ2L7k9LygCXKGefFhT8Jx8UJd~klzS5NxFme9ESla5dgB3POrRnTDvnxe2TFyGsEvKXCqh13M6LZFsSAcp-8OGiq4FMmL~pqOnOp6F7XJLJylfhPFvQU34T7PAaYLB3-abu9iA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463536&ar=2726856&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw049f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of orthologous gene families of <em>H. syriacus, G. raimondii, T. cacao, A. trichopoda</em> and <em>A. thaliana</em>, from which 169,570 sequences were clustered into 9,076 groups. The number of clustered groups and genes in each species are shown on the left and center, and total gene numbers are shown on the right.</p></div></div><div content-id="dsw049-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f1.png?Expires=1509239087&Signature=VfjqT2TWePlhIW61TEeIAlrz9bexNJImLrVd3ik3B385I4~svzA3hB6zS4urnQ2g4MuSnrQjbBDg94lcRFyD93DC4l0E4Sdn6W2CQQLC0iX~GK~wBl7ZuhzoLXdXvFnpZKRJgBSCFaKDfRNL3LkXC6QfHShzIE7hBHHEp~D1qkgAuT0HSTt3lTpYqDzKgwUe~GDtn1VtmhWDYUrws9EdG6rFvfVtOPv5rxtB4t0cS-GT34fYQXVXYb1iYuTkZ5~Aol8ZkrhIH8gXE6sjq3~Y-apxI~MyiAVJuU7Z-OZyElUiNnyUR9VkruHeEfNlmih8sZi3vc3y8WqSaAYvKT8u6w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of orthologous gene families of H. syriacus, G. raimondii, T. cacao, A. trichopoda and A. thaliana, from which 169,570 sequences were clustered into 9,076 groups. The number of clustered groups and genes in each species are shown on the left and center, and total gene numbers are shown on the right." path-from-xml="dsw049f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463536/dsw049f1.tif" path-from-xml="dsw049f1.tif" target="_blank">View large</a><a section="61463536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f1.png?Expires=1509239087&Signature=CZQstR2udmaf3ucsJ76YNEVN0abCfV9PZizVptgxWzDB~esXIjJ1B7KxeWT8adKyik3zpMmorclypK7vqV-y8LZU4CMBCkE1363lPhBXEkJk8FkyizbLlAgQHrbZ7PNo9toKQczNEI-JVkUaNbSZN2O1s9nsDYoBR6bM0Hej2LSZ8HOZFxlzy-njgO6DYFJ1-ebpfGZR6LGUJ9XbJ2L7k9LygCXKGefFhT8Jx8UJd~klzS5NxFme9ESla5dgB3POrRnTDvnxe2TFyGsEvKXCqh13M6LZFsSAcp-8OGiq4FMmL~pqOnOp6F7XJLJylfhPFvQU34T7PAaYLB3-abu9iA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463536&ar=2726856&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw049f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of orthologous gene families of <em>H. syriacus, G. raimondii, T. cacao, A. trichopoda</em> and <em>A. thaliana</em>, from which 169,570 sequences were clustered into 9,076 groups. The number of clustered groups and genes in each species are shown on the left and center, and total gene numbers are shown on the right.</p></div></div></p>                            <h3 scrollto-destination=61463537 id="61463537" class="section-title" >3.3. Genome structure and polyploidization of <em>H. syriacu</em>s</h3>

  <p>To compare genome structures among Malvaceae plants, collinearity blocks were detected using MCScanX.<a reveal-id="dsw049-B30" data-open="dsw049-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Two WGDs or a triplication event have occurred in <em>G. raimondii</em> (DD),<a reveal-id="dsw049-B43" data-open="dsw049-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsw049-B44" data-open="dsw049-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> while none have occurred in <em>T. cacao</em>. Therefore, the <em>T. cacao</em> genome was used as a template to detect collinearity blocks in <em>G. raimondii</em> and <em>H. syriacus</em> (<a reveal-id="dsw049-F2" data-open="dsw049-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). We detected <em>T. cacao</em> collinearity blocks in <em>G. raimondii</em> and <em>H. syriacus</em> with frequencies ranging from 2 to 7. The <em>H. syriacus</em> genome contains four times as many collinearity blocks than <em>G. raimondii</em> and blocks two times larger, indicating WGD events in <em>H. syriacus</em>. Duplication patterns were identified using phylogenetic analyses, which revealed single-copy flowering regulator genes in the diploid genomes of <em>A. thaliana, A. trichopoda</em> and <em>T. cacao</em> (<a reveal-id="dsw049-F2" data-open="dsw049-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S4</a></span>). Duplication of the <em>GIGANTEA</em> (GI) gene indicated WGDs occurred three times in <em>H. syriacus</em> but that many descendant genes since the first WGD have been lost (<a reveal-id="dsw049-F2" data-open="dsw049-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>), such as the <em>CONSTANS</em> and <em>SOC1</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S4</a></span>). Thus, diploidization and homeolog loss in <em>H. syriacus</em>, first proposed in previous studies,<a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> included duplication of distinct, individual gene families stemming from random homeolog gene loss after each WGD. Paleohexaploidy has occurred in the <em>G. raimondii</em> genome,<a reveal-id="dsw049-B10" data-open="dsw049-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw049-B43" data-open="dsw049-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsw049-B44" data-open="dsw049-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> and duplication patterns we observed were consistent with these previous results. <div data-id="dsw049-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f2.png?Expires=1509239087&amp;Signature=E3IPdjUF0~DT1VHEboDQSM8jQdrjxR~wV-QcUXh7QLoxdES4q0~37ICEcARiKsAHbV0cw2XaqE1rNmgInvdkwpUlb88sSYPm-yEKrteC1yUqfYy9DUV15h9LBi9~h7QnRH1PleEXmZ2cOhS2~oWGvAjnuPm6emPiEq2UoBpEZvSk3vGChmhGhe1JwwRVrCvcz1CUojmbJPxtHa2ydeujSi2imgajd8QElcFks5e3PEIC8rN12o6yMKzLgQRo3D7fb-DUuAtOLq0UZsxO9nBWUY4bsbkmK3OoeY8M6YlPV9LkjdCjKq~XH5mNfxfVKFPLJM1hGu1ohAbo5IReOwy~Pg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Collinearity block detection and calculation of gene duplication times. (A) Collinearity blocks of the T. cacao genome were detected in G. raimondii and H. syriacus. (B) Calculation of divergence times of individual gene families. Circles and triangles indicate H. syriacus and G. raimondii, respectively, and shade boxes indicate each WGD. (C) Divergence time of Malvales plants. H. syriacus diverged from the H. syriacus-G. raimondii common ancestor 22.28 MYA. Red (H. syriacus) and green (G. raimondii) stars indicate WGD and blue circles indicate diploidization events." path-from-xml="dsw049f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463538/dsw049f2.tif" path-from-xml="dsw049f2.tif" target="_blank">View large</a><a section="61463538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f2.png?Expires=1509239087&amp;Signature=f8KIMrHYpClzde-aPtViR6~nseNGN2F1mFqHgFxo~JU7z8AAdJ90Eo8VXkAS9kOrUNZ0Ymp4WiK2fD-QuMC75OjTV69jnKU4e04-9zKKYRP3gvmNbbCNMT4f~YC4rkEYqxIyIVHAwegd5W8giHCm-406wZStoK7M5AC9m4VAdeF3AoM67XuAADwcQS3HmHiqDAoXS0tQdRCiNw2RsPZ~NlJWsn7t~IN17blLaE3Tk021bLq7qeE8dPy6A7Gek~UhSv5Ogd7B0rFLqlMN9zMwt~COCYIR~cBm-x8twWG7zijCiUw-4m82i7Pi083583T68YYZl0uOCFXzXxyC0bk7cw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463538&amp;ar=2726856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw049f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity block detection and calculation of gene duplication times. <strong>(A)</strong> Collinearity blocks of the <em>T. cacao</em> genome were detected in <em>G. raimondii</em> and <em>H. syriacus</em>. <strong>(B)</strong> Calculation of divergence times of individual gene families. Circles and triangles indicate <em>H. syriacus</em> and <em>G. raimondii</em>, respectively, and shade boxes indicate each WGD. <strong>(C)</strong> Divergence time of Malvales plants. <em>H. syriacus</em> diverged from the <em>H. syriacus-G. raimondii</em> common ancestor 22.28 MYA. Red (<em>H. syriacus</em>) and green (<em>G. raimondii</em>) stars indicate WGD and blue circles indicate diploidization events.</p></div></div><div content-id="dsw049-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f2.png?Expires=1509239087&amp;Signature=E3IPdjUF0~DT1VHEboDQSM8jQdrjxR~wV-QcUXh7QLoxdES4q0~37ICEcARiKsAHbV0cw2XaqE1rNmgInvdkwpUlb88sSYPm-yEKrteC1yUqfYy9DUV15h9LBi9~h7QnRH1PleEXmZ2cOhS2~oWGvAjnuPm6emPiEq2UoBpEZvSk3vGChmhGhe1JwwRVrCvcz1CUojmbJPxtHa2ydeujSi2imgajd8QElcFks5e3PEIC8rN12o6yMKzLgQRo3D7fb-DUuAtOLq0UZsxO9nBWUY4bsbkmK3OoeY8M6YlPV9LkjdCjKq~XH5mNfxfVKFPLJM1hGu1ohAbo5IReOwy~Pg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Collinearity block detection and calculation of gene duplication times. (A) Collinearity blocks of the T. cacao genome were detected in G. raimondii and H. syriacus. (B) Calculation of divergence times of individual gene families. Circles and triangles indicate H. syriacus and G. raimondii, respectively, and shade boxes indicate each WGD. (C) Divergence time of Malvales plants. H. syriacus diverged from the H. syriacus-G. raimondii common ancestor 22.28 MYA. Red (H. syriacus) and green (G. raimondii) stars indicate WGD and blue circles indicate diploidization events." path-from-xml="dsw049f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463538/dsw049f2.tif" path-from-xml="dsw049f2.tif" target="_blank">View large</a><a section="61463538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f2.png?Expires=1509239087&amp;Signature=f8KIMrHYpClzde-aPtViR6~nseNGN2F1mFqHgFxo~JU7z8AAdJ90Eo8VXkAS9kOrUNZ0Ymp4WiK2fD-QuMC75OjTV69jnKU4e04-9zKKYRP3gvmNbbCNMT4f~YC4rkEYqxIyIVHAwegd5W8giHCm-406wZStoK7M5AC9m4VAdeF3AoM67XuAADwcQS3HmHiqDAoXS0tQdRCiNw2RsPZ~NlJWsn7t~IN17blLaE3Tk021bLq7qeE8dPy6A7Gek~UhSv5Ogd7B0rFLqlMN9zMwt~COCYIR~cBm-x8twWG7zijCiUw-4m82i7Pi083583T68YYZl0uOCFXzXxyC0bk7cw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463538&amp;ar=2726856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw049f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity block detection and calculation of gene duplication times. <strong>(A)</strong> Collinearity blocks of the <em>T. cacao</em> genome were detected in <em>G. raimondii</em> and <em>H. syriacus</em>. <strong>(B)</strong> Calculation of divergence times of individual gene families. Circles and triangles indicate <em>H. syriacus</em> and <em>G. raimondii</em>, respectively, and shade boxes indicate each WGD. <strong>(C)</strong> Divergence time of Malvales plants. <em>H. syriacus</em> diverged from the <em>H. syriacus-G. raimondii</em> common ancestor 22.28 MYA. Red (<em>H. syriacus</em>) and green (<em>G. raimondii</em>) stars indicate WGD and blue circles indicate diploidization events.</p></div></div></p>
<p>To estimate divergence time among Malvaceae plants, we calculated synonymous substitution rates (Ks) and constructed phylogenetic trees via the BEAST package using single-copy genes in OrthoMCL clusters. The trees revealed that the Malvaceae family diverged from a Brassicae-Malvaceae common ancestor approximately 91.91 MYA (<a reveal-id="dsw049-F2" data-open="dsw049-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>) and that <em>H. syriacus, G. raimondii</em> and <em>T. cacao</em> belong to a common subclade that diverged from a common ancestor approximately 30.88 MYA, which corroborates earlier studies.<a reveal-id="dsw049-B10" data-open="dsw049-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Occurrence of duplications in <em>G. raimondii</em> genes ranged from 24.46 to 45.46 MYA (<a reveal-id="dsw049-F2" data-open="dsw049-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>), while <em>H. syriacus</em> individual gene duplications before speciation and WGD events ranged from 25.23 to 48.23 MYA and from 4.61 to 21.15 MYA, respectively (<a reveal-id="dsw049-F2" data-open="dsw049-F2" class="link link-reveal link-table xref-fig">Fig. 2B and C</a>). These results suggest that one WGD occurred in <em>H. syriacus</em> before speciation and two WGDs occurred after speciation.</p><p>Previous reports indicate transcriptional factors (TFs) were retained as duplicated genes, while other genes remained singletons.<a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> We investigated the duplication status of TFs in <em>H. syriacus</em>, and other Malvaceae plants and identified 9,642 TFs and transcriptional regulators in 81 families in the <em>H. syriacus</em> genome. Eighteen <em>H. syriacus</em> TF gene families, including <em>AP2-ERF, AUX/IAA</em>, and <em>FAR1</em>, contained more genes than those in diploid genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S6</a></span>). In particular, the <em>H. syriacus</em> genome contains 10 times more <em>FAR1</em> family genes than the other genomes we analysed, although 19 TF genes showed convergent evolution patterns, and the proportions of other major TF family genes were similar across species. Thus, complex WGD events followed by diploidization led to unequal regulation of gene dosage and caused gene family copy number variations in <em>H. syriacu</em>s.</p>                            <h3 scrollto-destination=61463541 id="61463541" class="section-title" >3.4. Flowering-time and disease-resistance genes in <em>H. syriacus</em></h3>

  <p>Genetic and molecular mechanisms of floral development in different plant species is highly conserved<a reveal-id="dsw049-B45" data-open="dsw049-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and include four major flowering pathways (photoperiod, autonomous, vernalization and gibberellin) well-characterized in <em>A. thaliana.</em> Main flowering signals are regulated by the <em>FLOWERING LOCUS T (FT)</em> in the photoperiod pathway, while the vernalization pathway acts via removal of an <em>FT</em> repressor after exposure to certain stimuli. <em>H. syriacus</em> is a long-day flowering plant with a long blooming period and can express a multivoltinism phenotype with 20–30 blossoms per day. However, the flowers of <em>H. syriacus</em> open daily and last for only one day. To uncover the genetic mechanisms controlling these phenotypes, we investigated genes involved in the four major flowering pathways of <em>A. trichopoda, A. thaliana, T. cacao, G. raimondii</em> and <em>H. syriacus</em> and their expression pattern in <em>H. syriacus</em> tissues (flowers, ovaries, roots and leaves) (<a reveal-id="dsw049-F3" data-open="dsw049-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Phylogenetic analysis of flowering-time genes identified <em>H. syriacus</em>-specific clusters (<a reveal-id="dsw049-F3" data-open="dsw049-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). Because flowering time is frequently dependent on gene copy number,<a reveal-id="dsw049-B46" data-open="dsw049-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> we used reference genes from <em>Arabidopsis</em> species to determine copy number variation of these genes among diverse plant genomes (<a reveal-id="dsw049-T3" data-open="dsw049-T3" class="link link-reveal link-table xref-fig">Table 3</a>) and found that copy numbers of <em>H. syriacus</em> were two to seven times greater than those in the other four genomes analysed. Among the flowering-time regulatory genes examined in our study, numbers of genes involved in circadian rhythm regulation (CO, ELF4, FKF1, GI, LHY, PHYs) and flower initiation (FCA, FLK, FT, LFY, VIN3, SOC1, TFL, SVP) were significantly higher in <em>H. syriacus</em> than in the other genomes. Moreover, the copy number of FAR1 family genes that modulate phytochrome A signaling showed high copy numbers in <em>H. syriacus</em> compared to other genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S6</a></span>). Plants with spike inflorescence, such as barley, rice, and wheat, also contain high copy numbers of <em>FAR1</em> genes; thus, high copy numbers of <em>FAR1</em> may also affect the flowering phenotype of <em>H. syriacus</em>. <div data-id="dsw049-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f3.png?Expires=1509239087&amp;Signature=GCQQejybB-M2rOn2y5atTFj9AYnIX2CHs3qQ2dpnrUOhoZCd5UmQMPODdmUP4GqyjKTbK8TcxnC3lscpk-5Tslo7mfNvwukA01IX7rEJvC5rtddxVKFItMEewmUj3kShfRugtx6RiD7xB8ZXGSjJpEU2dG-D4UTfib~tprdnQY3KDHf~cXGS~KzFRRwElSSUIve~LrziceE54aDnjyvS5LHgxH2awOYc3uIKYFp6A8BzdXcSkRaLHaCOXjuRB25aROos7cOLeYSup-po5wEhByfwqmWj55tVkNGs21aqVIGXG0rutyWoBZ4s5vIyQiy7pAfZ1eYTHQleZb8g92R5uQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree of photoperiod/circadian clock genes. (A) The evolutionary history of these genes was inferred using the minimum evolution method. Blue (A. trichopoda), green (A. thaliana), pink (G. raimondii), orange (T. cacao) and red (H. syriacus) indicate genes from each species. (B) Expression patterns of photoperiod/circadian clock genes in petals, ovaries, roots, and leaves." path-from-xml="dsw049f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463542/dsw049f3.tif" path-from-xml="dsw049f3.tif" target="_blank">View large</a><a section="61463542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f3.png?Expires=1509239087&amp;Signature=G8HS-v2ZvY9zBPCYyELJbHAdpVs2i-fmON5NxSs~mPljpwXlJDPElk2VpLC41OA3GjUwuInY6HQzhFeA97M5lod84B2nVivsyWpWNQAdi7tN5gBGwqeFk69Caca2N3WNaYsdOsTzHNOjMmXr075ttRv7XHWlbTW9lHxDq1qWwlF8cZGttkb3XB9c5vS7YI1xFg17jlNcsHVGu3Pr5pjBhJCBTjof7VjDY0qNTD9KcoOEGtYaySxHTEl0Sg7KziKp8T0G2r-lHEKJw3UptTW8snprnKr8knbgW9KUpoGUgjhYrLyjFNd8eFe5ZSXiYggAWcQEgPjitH76RNR5aBZvEg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463542&amp;ar=2726856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw049f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of photoperiod/circadian clock genes. <strong>(A)</strong> The evolutionary history of these genes was inferred using the minimum evolution method. Blue (<em>A. trichopoda</em>), green (<em>A. thaliana</em>), pink (<em>G. raimondii</em>), orange (<em>T. cacao</em>) and red (<em>H. syriacus</em>) indicate genes from each species. <strong>(B)</strong> Expression patterns of photoperiod/circadian clock genes in petals, ovaries, roots, and leaves.</p></div></div><div content-id="dsw049-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f3.png?Expires=1509239087&amp;Signature=GCQQejybB-M2rOn2y5atTFj9AYnIX2CHs3qQ2dpnrUOhoZCd5UmQMPODdmUP4GqyjKTbK8TcxnC3lscpk-5Tslo7mfNvwukA01IX7rEJvC5rtddxVKFItMEewmUj3kShfRugtx6RiD7xB8ZXGSjJpEU2dG-D4UTfib~tprdnQY3KDHf~cXGS~KzFRRwElSSUIve~LrziceE54aDnjyvS5LHgxH2awOYc3uIKYFp6A8BzdXcSkRaLHaCOXjuRB25aROos7cOLeYSup-po5wEhByfwqmWj55tVkNGs21aqVIGXG0rutyWoBZ4s5vIyQiy7pAfZ1eYTHQleZb8g92R5uQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree of photoperiod/circadian clock genes. (A) The evolutionary history of these genes was inferred using the minimum evolution method. Blue (A. trichopoda), green (A. thaliana), pink (G. raimondii), orange (T. cacao) and red (H. syriacus) indicate genes from each species. (B) Expression patterns of photoperiod/circadian clock genes in petals, ovaries, roots, and leaves." path-from-xml="dsw049f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463542/dsw049f3.tif" path-from-xml="dsw049f3.tif" target="_blank">View large</a><a section="61463542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f3.png?Expires=1509239087&amp;Signature=G8HS-v2ZvY9zBPCYyELJbHAdpVs2i-fmON5NxSs~mPljpwXlJDPElk2VpLC41OA3GjUwuInY6HQzhFeA97M5lod84B2nVivsyWpWNQAdi7tN5gBGwqeFk69Caca2N3WNaYsdOsTzHNOjMmXr075ttRv7XHWlbTW9lHxDq1qWwlF8cZGttkb3XB9c5vS7YI1xFg17jlNcsHVGu3Pr5pjBhJCBTjof7VjDY0qNTD9KcoOEGtYaySxHTEl0Sg7KziKp8T0G2r-lHEKJw3UptTW8snprnKr8knbgW9KUpoGUgjhYrLyjFNd8eFe5ZSXiYggAWcQEgPjitH76RNR5aBZvEg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463542&amp;ar=2726856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw049f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of photoperiod/circadian clock genes. <strong>(A)</strong> The evolutionary history of these genes was inferred using the minimum evolution method. Blue (<em>A. trichopoda</em>), green (<em>A. thaliana</em>), pink (<em>G. raimondii</em>), orange (<em>T. cacao</em>) and red (<em>H. syriacus</em>) indicate genes from each species. <strong>(B)</strong> Expression patterns of photoperiod/circadian clock genes in petals, ovaries, roots, and leaves.</p></div></div><div class="table-wrap"><div id="dsw049-T3" data-id="dsw049-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of flowering-time gene copy numbers</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Regulators </th><th rowspan="2"><em>Arabidopsis</em> locus </th><th colspan="4">Copy number<hr /> </th></tr><tr><th><em>H. syriacus</em> </th><th><em>T. cacao</em> </th><th><em>G. raimondii</em> </th><th><em>A. trichopoda</em> </th></tr></thead><tbody><tr><td><em>CO</em> </td><td>AT5G15840 </td><td>6 </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td><em>ELF4</em> </td><td>AT2G40080 </td><td>7 </td><td>1 </td><td>4 </td><td>1 </td></tr><tr><td><em>FCA</em> </td><td>AT4G16280 </td><td>2 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>FKF1</em> </td><td>AT1G68050 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>FLK</em> </td><td>AT3G04610 </td><td>3 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>FT</em> </td><td>AT1G65480 </td><td>2 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>GI</em> </td><td>AT1G22770 </td><td>5 </td><td>1 </td><td>2 </td><td>0 </td></tr><tr><td><em>LFY</em> </td><td>AT5G61850 </td><td>4 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>LHY</em> </td><td>AT1G01060 </td><td>7 </td><td>1 </td><td>3 </td><td>1 </td></tr><tr><td><em>VIN3</em> </td><td>AT5G57380 </td><td>7 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>SOC1</em> </td><td>AT2G45660 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>TFL</em> </td><td>AT5G03840 </td><td>4 </td><td>2 </td><td>2 </td><td>0 </td></tr><tr><td><em>SVP</em> </td><td>AT1G24260 </td><td>3 </td><td>2 </td><td>2 </td><td>0 </td></tr><tr><td><em>PHYA</em> </td><td>AT1G09570 </td><td>4 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>PHYB</em> </td><td>AT2G18790 </td><td>4 </td><td>0 </td><td>1 </td><td>1 </td></tr><tr><td><em>PHYC</em> </td><td>AT5G35840 </td><td>2 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>PHYE</em> </td><td>AT4G18130 </td><td>4 </td><td>1 </td><td>0 </td><td>0 </td></tr></tbody></table></div><div content-id="dsw049-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Regulators </th><th rowspan="2"><em>Arabidopsis</em> locus </th><th colspan="4">Copy number<hr /> </th></tr><tr><th><em>H. syriacus</em> </th><th><em>T. cacao</em> </th><th><em>G. raimondii</em> </th><th><em>A. trichopoda</em> </th></tr></thead><tbody><tr><td><em>CO</em> </td><td>AT5G15840 </td><td>6 </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td><em>ELF4</em> </td><td>AT2G40080 </td><td>7 </td><td>1 </td><td>4 </td><td>1 </td></tr><tr><td><em>FCA</em> </td><td>AT4G16280 </td><td>2 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>FKF1</em> </td><td>AT1G68050 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>FLK</em> </td><td>AT3G04610 </td><td>3 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>FT</em> </td><td>AT1G65480 </td><td>2 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>GI</em> </td><td>AT1G22770 </td><td>5 </td><td>1 </td><td>2 </td><td>0 </td></tr><tr><td><em>LFY</em> </td><td>AT5G61850 </td><td>4 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>LHY</em> </td><td>AT1G01060 </td><td>7 </td><td>1 </td><td>3 </td><td>1 </td></tr><tr><td><em>VIN3</em> </td><td>AT5G57380 </td><td>7 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>SOC1</em> </td><td>AT2G45660 </td><td>4 </td><td>1 </td><td>2 </td><td>1 </td></tr><tr><td><em>TFL</em> </td><td>AT5G03840 </td><td>4 </td><td>2 </td><td>2 </td><td>0 </td></tr><tr><td><em>SVP</em> </td><td>AT1G24260 </td><td>3 </td><td>2 </td><td>2 </td><td>0 </td></tr><tr><td><em>PHYA</em> </td><td>AT1G09570 </td><td>4 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>PHYB</em> </td><td>AT2G18790 </td><td>4 </td><td>0 </td><td>1 </td><td>1 </td></tr><tr><td><em>PHYC</em> </td><td>AT5G35840 </td><td>2 </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td><em>PHYE</em> </td><td>AT4G18130 </td><td>4 </td><td>1 </td><td>0 </td><td>0 </td></tr></tbody></table></div></div></p>

  <p>Most disease-resistance (<em>R</em>) family genes encode intracellular proteins with a NBS and leucine-rich repeats (LRR).<a reveal-id="dsw049-B47 dsw049-B48 dsw049-B49 dsw049-B50 dsw049-B51" data-open="dsw049-B47 dsw049-B48 dsw049-B49 dsw049-B50 dsw049-B51" class="link link-ref link-reveal xref-bibr"><sup>47–51</sup></a> The NBS-encoding <em>R</em> gene family is one of the largest in the <em>H. syriacus</em> genome, with 472 genes, approximately three times greater than <em>A. trichopoda</em> and <em>A. thaliana</em>. These genes are divided into two clades based on presence of the distinct toll interleukin receptor (TIR) domain.<a reveal-id="dsw049-B50" data-open="dsw049-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> TIR genes in <em>H. syriacus</em> (<em>n</em><em>=</em> 76, 17%) are markedly over-represented compared to those of <em>S</em><em>.</em><em>lycopersicum</em> (25 genes, 9%), <em>G. raimondii</em> (27 genes, 9%) and <em>T. cacao</em> (17 genes, 6%) (<a reveal-id="dsw049-T4" data-open="dsw049-T4" class="link link-reveal link-table xref-fig">Table 4</a> and <a reveal-id="dsw049-F4" data-open="dsw049-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). More than 70% of NBS-encoding genes in Malvaceae plants (<em>H. syriacus</em>, <em>T. cacao</em>, and <em>G. raimondii</em>) are shared among 26 subclasses, indicating that most <em>R</em> genes are derived from a common ancestor (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S7</a></span>). In addition, 125 NBS-encoding genes in <em>H. syriacus</em> from four subclasses are expanded approximately five time more than other Malvaceae and 18 NBS-encoding genes from seven subclasses are unique to <em>H. syriacus</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S8</a></span>). Notably, genes in TIR and RPW8 motif-encoding subclasses (NBS cluster 11 and NBS cluster 20, respectively) exhibited extensive expansion in the <em>H. syriacus</em> genome, underwent unequal duplication events, and displayed great diversity among plant genomes (<a reveal-id="dsw049-F4" data-open="dsw049-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S8</a></span>). The different <em>R</em> gene repertoires in the <em>H. syriacus</em> genome suggest that expansion and diversity of clustered <em>R</em> genes might involve lineage-specific gene duplication events, eventually leading to divergent evolution in close relatives. These results provide useful preliminary information to support further comparative analysis of flowering-time and disease-resistance genes in other perennial plant species. <div data-id="dsw049-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f4.png?Expires=1509239087&amp;Signature=gxjuKVnbMkTQYRGeeB43ePYXSMnSTKDGzDCUAcnATa0rcq~ZMNJ6zjVNRKq~aHSc1DSFTNS6rLqEvasseW-bfeanLzXhTd6ZfiPL8cBYjqPyCeCyt0MhbOA7WdTnU34diiIj6mhadPPC~NqAUJHCTheYmBn3zc5hrakh~GWDxy6t6EeE4JIMCc5WwNsiHRT4AZB2ZpxzEB2e24-H9cBvKLC6~-V~WX7DTI1mHjTF4QCeI0oxoVP2ez1VoNlJaxAPTR9HJIYh0has1cVYkNV7c9QE7ZvsbGOFstGs7vlaGnWXMV-pFcA5QR8fPdnGDKhq6MzMayJu1igRvyvZfObzVQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic relationships of NBS-LRR genes with &gt;80% bootstrap values. (A) Phylogenetic relationships of predicted NBS-LRR genes in H. syriacus. Red (TNL-A, TNL-B, CNL-5 and RPW-CNL subgroups) indicates expanded subgroups of the H. syriacus genome compared to other plant genomes (Supplementary Table S8). (B) Detailed phylogenetic relationships of expanded TNL subgroups are shown. Intact NB-ARC domains of H. syriacus (red), G. raimondii (green), T. cacao (blue), S. lycopersicum (orange), A. thaliana (light blue) and V. vinifera (purple) were used in the phylogenetic construction. " path-from-xml="dsw049f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463543/dsw049f4.tif" path-from-xml="dsw049f4.tif" target="_blank">View large</a><a section="61463543" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f4.png?Expires=1509239087&amp;Signature=QsF~VYlkfljPmfQxP0i2v5ithZY2skRxM8yNPYL9eEVqCdM6UmpDIYssgK12FEnoWrHlfrjSdgEkDsKjEILX-KT8cGRdonQ4lZI9VU8q9UvjQoO4mOpW04czzxNDKXshbUN3gOCqb8w4~-w66msGXT9K6Eaj~tWMabyK77wdeNAVCnHnloeUHFH6UYOY2GqYxSBMdRHVbJ6jYQnUGOCoTNiW9fzGFdWqI3-3140dTYRqKZsNg~~ORNnwW5UHv8x5t2wEJJ1aGTbv0DlJZW12mMUpXcu0pIUwmSPmmjfF40Py3TmWl5~nCHXyJYm4MzGhxXQq15Q~rylG7wcpSO38Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463543&amp;ar=2726856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw049f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships of NBS-LRR genes with &gt;80% bootstrap values. <strong>(A)</strong> Phylogenetic relationships of predicted NBS-LRR genes in <em>H. syriacus</em>. Red (TNL-A, TNL-B, CNL-5 and RPW-CNL subgroups) indicates expanded subgroups of the <em>H. syriacus</em> genome compared to other plant genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S8</a></span>). <strong>(B)</strong> Detailed phylogenetic relationships of expanded TNL subgroups are shown. Intact NB-ARC domains of <em>H. syriacus</em> (red), <em>G. raimondii</em> (green), <em>T. cacao</em> (blue), <em>S. lycopersicum</em> (orange), <em>A. thaliana</em> (light blue) and <em>V. vinifera</em> (purple) were used in the phylogenetic construction. </p></div></div><div content-id="dsw049-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/m_dsw049f4.png?Expires=1509239087&amp;Signature=gxjuKVnbMkTQYRGeeB43ePYXSMnSTKDGzDCUAcnATa0rcq~ZMNJ6zjVNRKq~aHSc1DSFTNS6rLqEvasseW-bfeanLzXhTd6ZfiPL8cBYjqPyCeCyt0MhbOA7WdTnU34diiIj6mhadPPC~NqAUJHCTheYmBn3zc5hrakh~GWDxy6t6EeE4JIMCc5WwNsiHRT4AZB2ZpxzEB2e24-H9cBvKLC6~-V~WX7DTI1mHjTF4QCeI0oxoVP2ez1VoNlJaxAPTR9HJIYh0has1cVYkNV7c9QE7ZvsbGOFstGs7vlaGnWXMV-pFcA5QR8fPdnGDKhq6MzMayJu1igRvyvZfObzVQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic relationships of NBS-LRR genes with &gt;80% bootstrap values. (A) Phylogenetic relationships of predicted NBS-LRR genes in H. syriacus. Red (TNL-A, TNL-B, CNL-5 and RPW-CNL subgroups) indicates expanded subgroups of the H. syriacus genome compared to other plant genomes (Supplementary Table S8). (B) Detailed phylogenetic relationships of expanded TNL subgroups are shown. Intact NB-ARC domains of H. syriacus (red), G. raimondii (green), T. cacao (blue), S. lycopersicum (orange), A. thaliana (light blue) and V. vinifera (purple) were used in the phylogenetic construction. " path-from-xml="dsw049f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463543/dsw049f4.tif" path-from-xml="dsw049f4.tif" target="_blank">View large</a><a section="61463543" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049f4.png?Expires=1509239087&amp;Signature=QsF~VYlkfljPmfQxP0i2v5ithZY2skRxM8yNPYL9eEVqCdM6UmpDIYssgK12FEnoWrHlfrjSdgEkDsKjEILX-KT8cGRdonQ4lZI9VU8q9UvjQoO4mOpW04czzxNDKXshbUN3gOCqb8w4~-w66msGXT9K6Eaj~tWMabyK77wdeNAVCnHnloeUHFH6UYOY2GqYxSBMdRHVbJ6jYQnUGOCoTNiW9fzGFdWqI3-3140dTYRqKZsNg~~ORNnwW5UHv8x5t2wEJJ1aGTbv0DlJZW12mMUpXcu0pIUwmSPmmjfF40Py3TmWl5~nCHXyJYm4MzGhxXQq15Q~rylG7wcpSO38Lw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463543&amp;ar=2726856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw049f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships of NBS-LRR genes with &gt;80% bootstrap values. <strong>(A)</strong> Phylogenetic relationships of predicted NBS-LRR genes in <em>H. syriacus</em>. Red (TNL-A, TNL-B, CNL-5 and RPW-CNL subgroups) indicates expanded subgroups of the <em>H. syriacus</em> genome compared to other plant genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S8</a></span>). <strong>(B)</strong> Detailed phylogenetic relationships of expanded TNL subgroups are shown. Intact NB-ARC domains of <em>H. syriacus</em> (red), <em>G. raimondii</em> (green), <em>T. cacao</em> (blue), <em>S. lycopersicum</em> (orange), <em>A. thaliana</em> (light blue) and <em>V. vinifera</em> (purple) were used in the phylogenetic construction. </p></div></div><div class="table-wrap"><div id="dsw049-T4" data-id="dsw049-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Comparative NBS-LRR gene family numbers</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Predicted domain </th><th>Class </th><th><em>H. syriacus</em> </th><th><em>G. raimondii</em> </th><th><em>T. cacao</em> </th><th><em>S. lycopersicum</em> </th><th><em>A. thaliana</em> </th><th><em>V. vinifera</em> </th><th><em>O. sativa</em> </th><th><em>A. trichopoda</em> </th></tr></thead><tbody><tr><td><strong>TIR group</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TIR-NBS-LRR </td><td>TNL </td><td>68 </td><td>26 </td><td>14 </td><td>19 </td><td>87 </td><td>19 </td><td>0 </td><td>9 </td></tr><tr><td> TIR-NBS </td><td>TN </td><td>9 </td><td>1 </td><td>3 </td><td>6 </td><td>17 </td><td>4 </td><td>2 </td><td>2 </td></tr><tr><td>% on NBS genes </td><td> </td><td>17 </td><td>9 </td><td>6 </td><td>9 </td><td>61 </td><td>7 </td><td>0.4 </td><td>10 </td></tr><tr><td><strong>Non-TIR group</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CC-NBS- LRR </td><td>CNL </td><td>183 </td><td>220 </td><td>202 </td><td>116 </td><td>52 </td><td>138 </td><td>337 </td><td>27 </td></tr><tr><td> CC-NBS </td><td>CN </td><td>77 </td><td>24 </td><td>25 </td><td>37 </td><td>3 </td><td>19 </td><td>104 </td><td>27 </td></tr><tr><td> NBS-LRR </td><td>NL </td><td>81 </td><td>28 </td><td>34 </td><td>39 </td><td>8 </td><td>110 </td><td>70 </td><td>18 </td></tr><tr><td> NBS </td><td>N </td><td>54 </td><td>4 </td><td>9 </td><td>50 </td><td>3 </td><td>32 </td><td>14 </td><td>29 </td></tr><tr><td>% on NBS genes </td><td> </td><td>84 </td><td>91 </td><td>94 </td><td>91 </td><td>39 </td><td>93 </td><td>99.6 </td><td>90 </td></tr><tr><td>Total NBS genes </td><td> </td><td>472 </td><td>303 </td><td>287 </td><td>267 </td><td>170 </td><td>322 </td><td>527 </td><td>112 </td></tr><tr><td>% on total genes </td><td> </td><td>0.53 </td><td>0.81 </td><td>0.97 </td><td>0.77 </td><td>0.63 </td><td>1.22 </td><td>1.35 </td><td>0.41 </td></tr><tr><td>Total no. of genes </td><td> </td><td>87,603 </td><td>37,505 </td><td>29,452 </td><td>34,727 </td><td>27,206 </td><td>26,346 </td><td>39,049 </td><td>26,846 </td></tr></tbody></table></div><div content-id="dsw049-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Predicted domain </th><th>Class </th><th><em>H. syriacus</em> </th><th><em>G. raimondii</em> </th><th><em>T. cacao</em> </th><th><em>S. lycopersicum</em> </th><th><em>A. thaliana</em> </th><th><em>V. vinifera</em> </th><th><em>O. sativa</em> </th><th><em>A. trichopoda</em> </th></tr></thead><tbody><tr><td><strong>TIR group</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TIR-NBS-LRR </td><td>TNL </td><td>68 </td><td>26 </td><td>14 </td><td>19 </td><td>87 </td><td>19 </td><td>0 </td><td>9 </td></tr><tr><td> TIR-NBS </td><td>TN </td><td>9 </td><td>1 </td><td>3 </td><td>6 </td><td>17 </td><td>4 </td><td>2 </td><td>2 </td></tr><tr><td>% on NBS genes </td><td> </td><td>17 </td><td>9 </td><td>6 </td><td>9 </td><td>61 </td><td>7 </td><td>0.4 </td><td>10 </td></tr><tr><td><strong>Non-TIR group</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CC-NBS- LRR </td><td>CNL </td><td>183 </td><td>220 </td><td>202 </td><td>116 </td><td>52 </td><td>138 </td><td>337 </td><td>27 </td></tr><tr><td> CC-NBS </td><td>CN </td><td>77 </td><td>24 </td><td>25 </td><td>37 </td><td>3 </td><td>19 </td><td>104 </td><td>27 </td></tr><tr><td> NBS-LRR </td><td>NL </td><td>81 </td><td>28 </td><td>34 </td><td>39 </td><td>8 </td><td>110 </td><td>70 </td><td>18 </td></tr><tr><td> NBS </td><td>N </td><td>54 </td><td>4 </td><td>9 </td><td>50 </td><td>3 </td><td>32 </td><td>14 </td><td>29 </td></tr><tr><td>% on NBS genes </td><td> </td><td>84 </td><td>91 </td><td>94 </td><td>91 </td><td>39 </td><td>93 </td><td>99.6 </td><td>90 </td></tr><tr><td>Total NBS genes </td><td> </td><td>472 </td><td>303 </td><td>287 </td><td>267 </td><td>170 </td><td>322 </td><td>527 </td><td>112 </td></tr><tr><td>% on total genes </td><td> </td><td>0.53 </td><td>0.81 </td><td>0.97 </td><td>0.77 </td><td>0.63 </td><td>1.22 </td><td>1.35 </td><td>0.41 </td></tr><tr><td>Total no. of genes </td><td> </td><td>87,603 </td><td>37,505 </td><td>29,452 </td><td>34,727 </td><td>27,206 </td><td>26,346 </td><td>39,049 </td><td>26,846 </td></tr></tbody></table></div></div></p>
                            <h2 scrollto-destination=61463544 id="61463544" class="section-title" >4. Discussion</h2>
<p>Polyploidy is an important mechanism of plant speciation that occurs in many angiosperms<a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and leads to increased genetic diversity compared to their diploid progenitors. Initial polyploidy events were followed by successive paleopolyploidy or diploidization events to stabilize the newly expanded genomes. Paleopolyploidy or diploidization returns a polyploidy genome to a diploid-like state and is characterized by loss of duplicated genes, chromosomes, and repetitive DNA, gene silencing, and altered chromosome pairings.<a reveal-id="dsw049-B4" data-open="dsw049-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The newly formed polyploids may experience rapid homeolog gene loss, genome reconstruction post-polyploidization, and altered patterns of gene expression.<a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Retained diploid genes are less often duplicated due to cumulative losses of the homeologous copy in a duplicated gene pair. Consequently, some genes are consistently returned to singleton status, while others, such as those encoding TFs, are retained in duplicate.<a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw049-B52" data-open="dsw049-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Duplicated TF genes were commonly found in the <em>H. syriacus</em> genome, although 25% of these genes showed evidence of convergent evolution, and their copy numbers varied greatly. Our phylogenetic analysis of individual genes in <em>H. syriacus</em> also indicated that homeolog gene loss events and diploidization occurred after WGD. Recent studies have suggested that one duplicate gene may be more susceptible to loss than others,<a reveal-id="dsw049-B4" data-open="dsw049-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> which could account for unequal gene dosage and corresponding phenotypic changes in <em>H. syriacus</em>.</p><p>The flowering phenotype of <em>H. syriacus</em> is characterized by multivoltinism, a long blooming period, and high blossom turnover. We found that the copy numbers of most flowering-related genes, such as <em>GIGANTEA</em>, <em>CONSTANS</em>, and <em>ELF4</em> (but not <em>FT</em>), were higher in <em>H. syriacus</em> than in the diploid genomes of <em>T. cacao, A. trichopoda, and A. thaliana</em>. In addition, <em>FAR1</em> genes, which modulate phytochrome A signaling by directly activating transcription of <em>FHY1</em> and <em>FHL</em> and lead to accumulation of nuclear phytochrome A, were significantly increased in <em>H. syriacus</em>. <em>FAR1</em> regulates the circadian clock, and its high copy number could directly affect the flowering phenotype of <em>H. syriacus</em> as seen in plants with spike inflorescence.</p><p>The investigation of duplication event timing in <em>H. syriacus</em> genome showed that two recent WGDs occurred after speciation. In the past 50 million years, the average global temperature has been &lt;20°C,<a reveal-id="dsw049-B53" data-open="dsw049-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> which is far below the optimal flowering temperature for <em>H. syriacus.</em> Lower temperatures, especially between 5 and 20 MYA, could have been an environmental suppressor of <em>H. syriacus</em> pollination that prompted polyploidization to overcome these unfavourable conditions and unreduced gamete formation.<a reveal-id="dsw049-B5" data-open="dsw049-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Furthermore, low temperatures could have exerted selective pressure on <em>H. syriacus</em> to extend its blooming period for increased chance of pollination.</p><p>Perennial plants are prone to invasion by pathogens before reproduction, and many fungal and bacterial diseases often threaten the life cycle of <em>H. syriacus</em>. Aside from primary defenses, such as thickened cell walls and secondary metabolites, plants have numerous disease resistance (<em>R</em>) genes that confer protection against various pathogens. In <em>H. syriacus</em>, NBS-containing <em>R</em> genes account for ∼0.53% of its total predicted genes, which is lower than other plant genomes studies, whose <em>R</em> gene proportions ranged from 0.63 to 1.35%. However, subsets of these genes, including those with TIR- and RPW8-encoding motifs, are markedly over-represented in the <em>H. syriacus</em> genome compared to those of other plants. Genes in the RPW8-NBS-LRR subclass provide broad-spectrum resistance against powdery mildew pathogens in <em>Arabidopsis</em>,<a reveal-id="dsw049-B54" data-open="dsw049-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and genes in TIR-NBS subclasses are conserved in basal angiosperms and eudicots, such as <em>A. trichopoda</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5</a></span>), but are absent in most monocots.<a reveal-id="dsw049-B55" data-open="dsw049-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsw049-B56" data-open="dsw049-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Their greater dominance in the <em>H. syriacus</em> genome indicates divergent evolution of TIR- and RPW-containing NBS genes from an ancestral origin may have led to more extensive expansion of this gene family. Moreover, the long life cycle of woody plants makes it difficult for them to adapt to pathogens undergoing more rapid evolution, thus favouring <em>R</em> gene maintenance and expansion for the plants’ survival.</p><p>Polyploidization in plants is a common mechanism for their adaptation to environmental change. After divergence from the <em>H. syriacus</em>-<em>G. raimondii</em> common ancestor, two WGDs and subsequent diploidization occurred in the <em>H. syriacus</em> genome to promote the plants’ survival in unfavourable environments. During the diploidization events, low temperatures may have selected for the maintenance of duplicate flowering-related genes whose high copy numbers led to the multivoltinism and long blooming period phenotypes expressed by <em>H. syriacus</em>. Further analyses <em>H. syriacus</em>, <em>T. cacao</em> and <em>G. raimondii</em> (DD) genomes with another diploid genome, <em>G. arboretum</em> (AA)<a reveal-id="dsw049-B11" data-open="dsw049-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and allotetraploid genome, <em>G. hirsutum</em> (A<sub>t</sub>D<sub>t</sub>)<a reveal-id="dsw049-B12" data-open="dsw049-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> will provide more information of evolution of Malvaceae plants.</p>                            <h2 scrollto-destination=61463550 id="61463550" class="section-title" >Availability</h2>
<p>This Whole Genome Shotgun project has been deposited at DDBJ/ENA/GenBank under the accession MBGJ00000000. The version described in this article is version MBGJ01000000. The raw sequence reads has been deposited at DDBJ/ENA/GenBank under accession SRP087036 (PRJNA341314). In addition, the genome data of <em>H. syriacus</em> are accessible at <a class="link link-uri" href="https://hibiscus.kobic.re.kr/hibiscus.en." target="_blank">https://hibiscus.kobic.re.kr/hibiscus.en</a>. </p>                            <h2 scrollto-destination=61463552 id="61463552" class="section-title" >Conflict of interest</h2>
<p>None declared. </p>                            <h2 scrollto-destination=61463554 id="61463554" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=61463556 id="61463556" class="section-title" >Funding</h2>
<p>This work was supported by the Korean Research Institute of Bioscience and Biotechnology initiative programme and by a grant from the Agricultural Genome Center of the Next Generation Biogreen 21 Programme (Project No. PJ011275, PJ011088 and PJ011100) of the Rural Development Administration, Republic of Korea. </p><p>Genome data including genome assembly, annotation and transcriptome data were also provided through webpage (<a class="link link-uri" href="https://hibiscus.kobic.re.kr/hibiscus.en" target="_blank">https://hibiscus.kobic.re.kr/hibiscus.en</a>).</p>                            <h2 scrollto-destination=61463559 id="61463559" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw049-b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paoletti</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Ferrara</div> <div class="given-names">A. M.</div></div><div class="name string-name"><div class="surname">Calatayud</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Deciduous shrubs for ozone bioindication: Hibiscus syriacus as an example</div>. <div class="source">Environ. Pollut</div> ., <div class="volume">157</div>, <div class="fpage">865</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciduous%20shrubs%20for%20ozone%20bioindication%3A%20Hibiscus%20syriacus%20as%20an%20example&amp;author=E.%20Paoletti&amp;author=A.%20M.%20Ferrara&amp;author=V.%20Calatayud&amp;publication_year=2009&amp;journal=Environ.%20Pollut&amp;volume=157&amp;pages=865-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.envpol.2008.11.009" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.envpol.2008.11.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.envpol.2008.11.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19081168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yun</div> <div class="given-names">B.-S.</div></div><div class="name string-name"><div class="surname">Ryoo</div> <div class="given-names">I.-J.</div></div><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">I.-K.</div></div><div class="name string-name"><div class="surname">Yoo</div> <div class="given-names">I.-D.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Hibispeptin A, a novel cyclic peptide from Hibiscus syriacus</div>. <div class="source">Tetrahedron Lett</div> ., <div class="volume">39</div>, <div class="fpage">993</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hibispeptin%20A%2C%20a%20novel%20cyclic%20peptide%20from%20Hibiscus%20syriacus&amp;author=B.-S.%20Yun&amp;author=I.-J.%20Ryoo&amp;author=I.-K.%20Lee&amp;author=I.-D.%20Yoo&amp;publication_year=1998&amp;journal=Tetrahedron%20Lett&amp;volume=39&amp;pages=993-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0040-4039(97)10797-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0040-4039(97)10797-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0040-4039(97)10797-3"> </span></div> </div></div></div></div></div><div content-id="dsw049-b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoo</div> <div class="given-names">I.-D.</div></div><div class="name string-name"><div class="surname">Yun</div> <div class="given-names">B.-S.</div></div><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">I.-K.</div></div><div class="name string-name"><div class="surname">Ryoo</div> <div class="given-names">I.-J.</div></div><div class="name string-name"><div class="surname">Choung</div> <div class="given-names">D.-H.</div></div><div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.-H.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Three naphthalenes from root bark of Hibiscus syriacus</div>. <div class="source">Phytochemistry</div> , <div class="volume">47</div>, <div class="fpage">799</div>–<div class="lpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20naphthalenes%20from%20root%20bark%20of%20Hibiscus%20syriacus&amp;author=I.-D.%20Yoo&amp;author=B.-S.%20Yun&amp;author=I.-K.%20Lee&amp;author=I.-J.%20Ryoo&amp;author=D.-H.%20Choung&amp;author=K.-H.%20Han&amp;publication_year=1998&amp;journal=Phytochemistry&amp;volume=47&amp;pages=799-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0031-9422(97)00674-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0031-9422(97)00674-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0031-9422(97)00674-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9542172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Semon</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Consequences of genome duplication</div>. <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">17</div>, <div class="fpage">505</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consequences%20of%20genome%20duplication&amp;author=M.%20Semon&amp;author=K.%20H.%20Wolfe&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=17&amp;pages=505-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2007.09.007" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2007.09.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2007.09.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18006297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">P. S.</div></div><div class="name string-name"><div class="surname">Marchant</div> <div class="given-names">D. B.</div></div><div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">D. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Polyploidy and genome evolution in plants</div>. <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">35</div>, <div class="fpage">119</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyploidy%20and%20genome%20evolution%20in%20plants&amp;author=P.%20S.%20Soltis&amp;author=D.%20B.%20Marchant&amp;author=Y.%20Van%20de%20Peer&amp;author=D.%20E.%20Soltis&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=35&amp;pages=119-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2015.11.003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2015.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2015.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26656231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Aury</div> <div class="given-names">J. M.</div></div><div class="name string-name"><div class="surname">Noel</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>. <div class="source">Nature</div> , <div class="volume">449</div>, <div class="fpage">463</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20genome%20sequence%20suggests%20ancestral%20hexaploidization%20in%20major%20angiosperm%20phyla&amp;author=O.%20Jaillon&amp;author=J.%20M.%20Aury&amp;author=B.%20Noel&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dehal</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Boore</div> <div class="given-names">J. L.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Two rounds of whole genome duplication in the ancestral vertebrate</div>. <div class="source">PLoS Biol</div> ., <div class="volume">3</div>, <div class="fpage">e314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20rounds%20of%20whole%20genome%20duplication%20in%20the%20ancestral%20vertebrate&amp;author=P.%20Dehal&amp;author=J.%20L.%20Boore&amp;publication_year=2005&amp;journal=PLoS%20Biol&amp;volume=3&amp;pages=e314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030314" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16128622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Otto</div> <div class="given-names">S. P.</div></div><div class="name string-name"><div class="surname">Whitton</div> <div class="given-names">J.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Polyploid incidence and evolution</div>. <div class="source">Annu. Rev. Genet</div> ., <div class="volume">34</div>, <div class="fpage">401</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyploid%20incidence%20and%20evolution&amp;author=S.%20P.%20Otto&amp;author=J.%20Whitton&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Genet&amp;volume=34&amp;pages=401-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.34.1.401" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.34.1.401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.34.1.401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11092833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fawcett</div> <div class="given-names">J. A.</div></div><div class="name string-name"><div class="surname">Maere</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Plants with double genomes might have had a better chance to survive the Cretaceous-Tertiary extinction event</div>. <div class="source">Proc. Natl. Acad. Sci. U S A</div> , <div class="volume">106</div>, <div class="fpage">5737</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plants%20with%20double%20genomes%20might%20have%20had%20a%20better%20chance%20to%20survive%20the%20Cretaceous-Tertiary%20extinction%20event&amp;author=J.%20A.%20Fawcett&amp;author=S.%20Maere&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=106&amp;pages=5737-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0900906106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0900906106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0900906106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19325131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">The draft genome of a diploid cotton Gossypium raimondii</div>. <div class="source">Nat. Genet</div> ., <div class="volume">44</div>, <div class="fpage">1098</div>–<div class="lpage">03</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20a%20diploid%20cotton%20Gossypium%20raimondii&amp;author=K.%20Wang&amp;author=Z.%20Wang&amp;author=F.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Genet&amp;volume=44&amp;pages=1098-03" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2371" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22922876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Fan</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Genome sequence of the cultivated cotton Gossypium arboreum</div>. <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">567</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20cultivated%20cotton%20Gossypium%20arboreum&amp;author=F.%20Li&amp;author=G.%20Fan&amp;author=K.%20Wang&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=567-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2987" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24836287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Fan</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Genome sequence of cultivated Upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">524</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20cultivated%20Upland%20cotton%20(Gossypium%20hirsutum%20TM-1)%20provides%20insights%20into%20genome%20evolution&amp;author=F.%20Li&amp;author=G.%20Fan&amp;author=C.%20Lu&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=524-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3208" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3208" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25893780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darlington</div> <div class="given-names">C. D.</div></div><div class="name string-name"><div class="surname">Wylie</div> <div class="given-names">A. P.</div></div></div> <div class="year">1955</div>, <div class="source">Chromosome Atlas of Flowering Plants</div> . 
<div class="publisher-name">Allen &amp; Unwin</div>: 
<div class="publisher-loc">London</div>.</div></div></div></div><div content-id="dsw049-b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C. V.</div></div><div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H. J.</div></div><div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.%20V.%20Henkel&amp;author=H.%20J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Magoc</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">FLASH: fast length adjustment of short reads to improve genome assemblies</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2957</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FLASH%3A%20fast%20length%20adjustment%20of%20short%20reads%20to%20improve%20genome%20assemblies&amp;author=T.%20Magoc&amp;author=S.%20L.%20Salzberg&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2957-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr507" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr507" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr507"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Park</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Yeom</div> <div class="given-names">S. I.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species</div>. <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">270</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20hot%20pepper%20provides%20insights%20into%20the%20evolution%20of%20pungency%20in%20Capsicum%20species&amp;author=S.%20Kim&amp;author=M.%20Park&amp;author=S.%20I.%20Yeom&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=270-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2877" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24441736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Keller</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Gunduz</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Hayes</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">AUGUSTUS: ab initio prediction of alternative transcripts</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W435</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;author=M.%20Stanke&amp;author=O.%20Keller&amp;author=I.%20Gunduz&amp;author=A.%20Hayes&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W435-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Argout</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Salse</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Aury</div> <div class="given-names">J. M.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The genome of Theobroma cacao</div>. <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Theobroma%20cacao&amp;author=X.%20Argout&amp;author=J.%20Salse&amp;author=J.%20M.%20Aury&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.736" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.736" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.736"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Clamp</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">GeneWise and Genomewise</div>. <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">988</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneWise%20and%20Genomewise&amp;author=E.%20Birney&amp;author=M.%20Clamp&amp;author=R.%20Durbin&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=988-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1865504" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1865504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1865504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</div>. <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Binns</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H. Y.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">InterProScan 5: genome-scale protein function classification</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1236</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.%20Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pruitt</div> <div class="given-names">K. D.</div></div><div class="name string-name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Brown</div> <div class="given-names">G. R.</div></div><div class="name string-name"><div class="surname">Maglott</div> <div class="given-names">D. R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D130</div>-<div class="lpage">135</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20Reference%20Sequences%20(RefSeq)%3A%20current%20status%2C%20new%20features%20and%20genome%20annotation%20policy&amp;author=K.%20D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=G.%20R.%20Brown&amp;author=D.%20R.%20Maglott&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D130-135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1079" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22121212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="collab">UniProt, C</div>. <div class="year">2013</div>, 
<div class="article-title">Update on activities at the Universal Protein Resource (UniProt) in 2013</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D43</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1068" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E. M.</div></div><div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">InterProScan–an integration platform for the signature-recognition methods in InterPro</div>. <div class="source">Bioinformatics</div> , <div class="volume">17</div>, <div class="fpage">847</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%E2%80%93an%20integration%20platform%20for%20the%20signature-recognition%20methods%20in%20InterPro&amp;author=E.%20M.%20Zdobnov&amp;author=R.%20Apweiler&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=847-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.847" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.9.847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.9.847"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11590104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cox</div> <div class="given-names">M. P.</div></div><div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D. A.</div></div><div class="name string-name"><div class="surname">Biggs</div> <div class="given-names">P. J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">SolexaQA: At-a-glance quality assessment of Illumina second-generation sequencing data</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">485</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SolexaQA%3A%20At-a-glance%20quality%20assessment%20of%20Illumina%20second-generation%20sequencing%20data&amp;author=M.%20P.%20Cox&amp;author=D.%20A.%20Peterson&amp;author=P.%20J.%20Biggs&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-485" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20875133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>. <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simao</div> <div class="given-names">F. A.</div></div><div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R. M.</div></div><div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E. V.</div></div><div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.%20A.%20Simao&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Stoeckert</div> <div class="given-names">C. J.</div>Jr.</div><div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D. S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>. <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.%20J.%20Stoeckert&amp;author=D.%20S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Debarry</div> <div class="given-names">J. D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20Debarry&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loytynoja</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Phylogeny-aware alignment with PRANK</div>. <div class="source">Methods Mol. Biol</div> ., <div class="volume">1079</div>, <div class="fpage">155</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny-aware%20alignment%20with%20PRANK&amp;author=A.%20Loytynoja&amp;publication_year=2014&amp;journal=Methods%20Mol.%20Biol&amp;volume=1079&amp;pages=155-70" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24170401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Drummond</div> <div class="given-names">A. J.</div></div><div class="name string-name"><div class="surname">Suchard</div> <div class="given-names">M. A.</div></div><div class="name string-name"><div class="surname">Xie</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Rambaut</div> <div class="given-names">A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">1969</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20phylogenetics%20with%20BEAUti%20and%20the%20BEAST%201.7&amp;author=A.%20J.%20Drummond&amp;author=M.%20A.%20Suchard&amp;author=D.%20Xie&amp;author=A.%20Rambaut&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=1969-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss075" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shapiro</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Rambaut</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Drummond</div> <div class="given-names">A. J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Choosing appropriate substitution models for the phylogenetic analysis of protein-coding sequences</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">23</div>, <div class="fpage">7</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Choosing%20appropriate%20substitution%20models%20for%20the%20phylogenetic%20analysis%20of%20protein-coding%20sequences&amp;author=B.%20Shapiro&amp;author=A.%20Rambaut&amp;author=A.%20J.%20Drummond&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol&amp;volume=23&amp;pages=7-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj021" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16177232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gernhard</div> <div class="given-names">T.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The conditioned reconstructed process</div>. <div class="source">J. Theor. Biol</div> ., <div class="volume">253</div>, <div class="fpage">769</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20conditioned%20reconstructed%20process&amp;author=T.%20Gernhard&amp;publication_year=2008&amp;journal=J.%20Theor.%20Biol&amp;volume=253&amp;pages=769-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2008.04.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2008.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2008.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18538793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Punta</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Coggill</div> <div class="given-names">P. C.</div></div><div class="name string-name"><div class="surname">Eberhardt</div> <div class="given-names">R. Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">The Pfam protein families database</div>. <div class="source">Nucleic Acids Res</div> , <div class="volume">40</div>, <div class="fpage">D290</div>–<div class="lpage">301</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=M.%20Punta&amp;author=P.%20C.%20Coggill&amp;author=R.%20Y.%20Eberhardt&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D290-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1065" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perez-Rodriguez</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Riano-Pachon</div> <div class="given-names">D. M.</div></div><div class="name string-name"><div class="surname">Correa</div> <div class="given-names">L. G.</div></div><div class="name string-name"><div class="surname">Rensing</div> <div class="given-names">S. A.</div></div><div class="name string-name"><div class="surname">Kersten</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Mueller-Roeber</div> <div class="given-names">B.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">PlnTFDB: updated content and new features of the plant transcription factor database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">38</div>, <div class="fpage">D822</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlnTFDB%3A%20updated%20content%20and%20new%20features%20of%20the%20plant%20transcription%20factor%20database&amp;author=P.%20Perez-Rodriguez&amp;author=D.%20M.%20Riano-Pachon&amp;author=L.%20G.%20Correa&amp;author=S.%20A.%20Rensing&amp;author=B.%20Kersten&amp;author=B.%20Mueller-Roeber&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=D822-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp805" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp805" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp805"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19858103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div><div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">HMMER web server: interactive sequence similarity searching</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W29</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HMMER%20web%20server%3A%20interactive%20sequence%20similarity%20searching&amp;author=R.%20D.%20Finn&amp;author=J.%20Clements&amp;author=S.%20R.%20Eddy&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W29-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr367" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marchin</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kelly</div> <div class="given-names">P. T.</div></div><div class="name string-name"><div class="surname">Fang</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Tracker: continuous HMMER and BLAST searching</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">388</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracker%3A%20continuous%20HMMER%20and%20BLAST%20searching&amp;author=M.%20Marchin&amp;author=P.%20T.%20Kelly&amp;author=J.%20Fang&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=388-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti012" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15347577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Milpetz</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C. P.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">SMART, a simple modular architecture research tool: identification of signaling domains</div>. <div class="source">Proc. Natl. Acad. Sci. U S A</div> , <div class="volume">95</div>, <div class="fpage">5857</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMART%2C%20a%20simple%20modular%20architecture%20research%20tool%3A%20identification%20of%20signaling%20domains&amp;author=J.%20Schultz&amp;author=F.%20Milpetz&amp;author=P.%20Bork&amp;author=C.%20P.%20Ponting&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=95&amp;pages=5857-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.11.5857" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.11.5857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.11.5857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9600884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div><div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Pfam: the protein families database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database&amp;author=R.%20D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lupas</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Van Dyke</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Stock</div> <div class="given-names">J.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Predicting coiled coils from protein sequences</div>. <div class="source">Science</div> , <div class="volume">252</div>, <div class="fpage">1162</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20coiled%20coils%20from%20protein%20sequences&amp;author=A.%20Lupas&amp;author=M.%20Van%20Dyke&amp;author=J.%20Stock&amp;publication_year=1991&amp;journal=Science&amp;volume=252&amp;pages=1162-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.252.5009.1162" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.252.5009.1162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.252.5009.1162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2031185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Potato Genome Sequencing</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Pan</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Genome sequence and analysis of the tuber crop potato</div>. <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">189</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20analysis%20of%20the%20tuber%20crop%20potato&amp;author=C.%20Potato%20Genome%20Sequencing&amp;author=X.%20Xu&amp;author=S.%20Pan&amp;publication_year=2011&amp;journal=Nature&amp;volume=475&amp;pages=189-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10158" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Muhlhausen</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Kollmar</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Whole genome duplication events in plant evolution reconstructed and predicted using myosin motor proteins</div>. <div class="source">BMC Evol. Biol</div> ., <div class="volume">13</div>, <div class="fpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20duplication%20events%20in%20plant%20evolution%20reconstructed%20and%20predicted%20using%20myosin%20motor%20proteins&amp;author=S.%20Muhlhausen&amp;author=M.%20Kollmar&amp;publication_year=2013&amp;journal=BMC%20Evol.%20Biol&amp;volume=13&amp;pages=202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-13-202" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-13-202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-13-202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24053117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Xia</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Shen</div> <div class="given-names">F.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genome-wide analysis of the RNA helicase gene family in Gossypium raimondii</div>. <div class="source">Int. J. Mol. Sci</div> ., <div class="volume">15</div>, <div class="fpage">4635</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20the%20RNA%20helicase%20gene%20family%20in%20Gossypium%20raimondii&amp;author=J.%20Chen&amp;author=Y.%20Zhang&amp;author=J.%20Liu&amp;author=M.%20Xia&amp;author=W.%20Wang&amp;author=F.%20Shen&amp;publication_year=2014&amp;journal=Int.%20J.%20Mol.%20Sci&amp;volume=15&amp;pages=4635-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms15034635" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms15034635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms15034635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24642883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schiessl</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Samans</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Hüttel</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Reinhardt</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Snowdon</div> <div class="given-names">R. J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Capturing sequence variation among flowering-time regulatory gene homologues in the allopolyploid crop species Brassica napus</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capturing%20sequence%20variation%20among%20flowering-time%20regulatory%20gene%20homologues%20in%20the%20allopolyploid%20crop%20species%20Brassica%20napus&amp;author=S.%20Schiessl&amp;author=B.%20Samans&amp;author=B.%20H%C3%BCttel&amp;author=R.%20Reinhardt&amp;author=R.%20J.%20Snowdon&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci&amp;volume=5&amp;pages=404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00404" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25202314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grover</div> <div class="given-names">C. E.</div></div><div class="name string-name"><div class="surname">Gallagher</div> <div class="given-names">J. P.</div></div><div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J. F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Candidate gene identification of flowering time genes in cotton</div>. <div class="source">Plant Genome</div> , <div class="volume">8</div>.</div></div></div></div><div content-id="dsw049-b47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dangl</div> <div class="given-names">J. L.</div></div><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">J. D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Plant pathogens and integrated defence responses to infection</div>. <div class="source">Nature</div> , <div class="volume">411</div>, <div class="fpage">826</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20pathogens%20and%20integrated%20defence%20responses%20to%20infection&amp;author=J.%20L.%20Dangl&amp;author=J.%20D.%20Jones&amp;publication_year=2001&amp;journal=Nature&amp;volume=411&amp;pages=826-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35081161" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35081161" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35081161"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11459065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McDowell</div> <div class="given-names">J. M.</div></div><div class="name string-name"><div class="surname">Woffenden</div> <div class="given-names">B. J.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Plant disease resistance genes: recent insights and potential applications</div>. <div class="source">Trends Biotechnol</div> ., <div class="volume">21</div>, <div class="fpage">178</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20disease%20resistance%20genes%3A%20recent%20insights%20and%20potential%20applications&amp;author=J.%20M.%20McDowell&amp;author=B.%20J.%20Woffenden&amp;publication_year=2003&amp;journal=Trends%20Biotechnol&amp;volume=21&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-7799(03)00053-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-7799(03)00053-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-7799(03)00053-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12679066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meyers</div> <div class="given-names">B. C.</div></div><div class="name string-name"><div class="surname">Kozik</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Griego</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Kuang</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Michelmore</div> <div class="given-names">R. W.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Genome-wide analysis of NBS-LRR-encoding genes in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">15</div>, <div class="fpage">809</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20NBS-LRR-encoding%20genes%20in%20Arabidopsis&amp;author=B.%20C.%20Meyers&amp;author=A.%20Kozik&amp;author=A.%20Griego&amp;author=H.%20Kuang&amp;author=R.%20W.%20Michelmore&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=809-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.009308" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.009308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.009308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">H. A.</div></div><div class="name string-name"><div class="surname">Yeom</div> <div class="given-names">S. I.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Plant NB-LRR proteins: tightly regulated sensors in a complex manner</div>. <div class="source">Brief. Funct. Genomics</div> , <div class="volume">14</div>, <div class="fpage">233</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20NB-LRR%20proteins%3A%20tightly%20regulated%20sensors%20in%20a%20complex%20manner&amp;author=H.%20A.%20Lee&amp;author=S.%20I.%20Yeom&amp;publication_year=2015&amp;journal=Brief.%20Funct.%20Genomics&amp;volume=14&amp;pages=233-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/elv012" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Felv012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Felv012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25825425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seo</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Yeom</div> <div class="given-names">S.-I.</div></div><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genome-wide comparative analyses reveal the dynamic evolution of nucleotide-binding leucine-rich repeat gene family among Solanaceae plants</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>.</div></div></div></div><div content-id="dsw049-b52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Freeling</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Scanlon</div> <div class="given-names">M. J.</div></div><div class="name string-name"><div class="surname">Fowler</div> <div class="given-names">J. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Fractionation and subfunctionalization following genome duplications: mechanisms that drive gene content and their consequences</div>. <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">35</div>, <div class="fpage">110</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fractionation%20and%20subfunctionalization%20following%20genome%20duplications%3A%20mechanisms%20that%20drive%20gene%20content%20and%20their%20consequences&amp;author=M.%20Freeling&amp;author=M.%20J.%20Scanlon&amp;author=J.%20E.%20Fowler&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=35&amp;pages=110-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2015.11.002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2015.11.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2015.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26657818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zachos</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Pagani</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Sloan</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Billups</div> <div class="given-names">K.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Trends, rhythms, and aberrations in global climate 65 Ma to present</div>. <div class="source">Science</div> , <div class="volume">292</div>, <div class="fpage">686</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trends%2C%20rhythms%2C%20and%20aberrations%20in%20global%20climate%2065%E2%80%89Ma%20to%20present&amp;author=J.%20Zachos&amp;author=M.%20Pagani&amp;author=L.%20Sloan&amp;author=E.%20Thomas&amp;author=K.%20Billups&amp;publication_year=2001&amp;journal=Science&amp;volume=292&amp;pages=686-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1059412" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1059412" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1059412"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11326091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ellwood</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Calis</div> <div class="given-names">O.</div></div></div>, et al.  . <div class="year">2001</div>, 
<div class="article-title">Broad-spectrum mildew resistance in Arabidopsis thaliana mediated by RPW8</div>. <div class="source">Science</div> , <div class="volume">291</div>, <div class="fpage">118</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Broad-spectrum%20mildew%20resistance%20in%20Arabidopsis%20thaliana%20mediated%20by%20RPW8&amp;author=S.%20Xiao&amp;author=S.%20Ellwood&amp;author=O.%20Calis&amp;publication_year=2001&amp;journal=Science&amp;volume=291&amp;pages=118-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.291.5501.118" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.291.5501.118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.291.5501.118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11141561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div><div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Fluhr</div> <div class="given-names">R.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Divergent evolution of plant NBS-LRR resistance gene homologues in dicot and cereal genomes</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">50</div>, <div class="fpage">203</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergent%20evolution%20of%20plant%20NBS-LRR%20resistance%20gene%20homologues%20in%20dicot%20and%20cereal%20genomes&amp;author=Q.%20Pan&amp;author=J.%20Wendel&amp;author=R.%20Fluhr&amp;publication_year=2000&amp;journal=J.%20Mol.%20Evol&amp;volume=50&amp;pages=203-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002399910023" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002399910023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002399910023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10754062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw049-b56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw049-b56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tarr</div> <div class="given-names">D. E.</div></div><div class="name string-name"><div class="surname">Alexander</div> <div class="given-names">H. M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TIR-NBS-LRR genes are rare in monocots: evidence from diverse monocot orders</div>. <div class="source">BMC Res. Notes</div> , <div class="volume">2</div>, <div class="fpage">197</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TIR-NBS-LRR%20genes%20are%20rare%20in%20monocots%3A%20evidence%20from%20diverse%20monocot%20orders&amp;author=D.%20E.%20Tarr&amp;author=H.%20M.%20Alexander&amp;publication_year=2009&amp;journal=BMC%20Res.%20Notes&amp;volume=2&amp;pages=197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-2-197" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-2-197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-2-197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19785756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute. </div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2726856&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw049/1/dsw049_Supp.zip?Expires=1509239087&amp;Signature=AAAzayVdrsz4RD5yPaQEHjPbhwHSylzcUpFpfacUT6yxmHjgfddzI0yFGKLsEx5T6JiRrY3iT59E2~368x3H2hzKojBfqCQp2e9ceLhAP~BYoiYjX3RZsoIwWgiIdiDyYYFQkpJS5M~kKS3oTVpyZwsMovvhm6WlETAIw80HNlmVYPTcH1y-xdGnhrShtf6a8eGoqLUlu9ep5Zoc5wT6uH28ghpy0K5QBJKgwE4cuhv1Aod1vz8qVnRAtpw~Zd94xYBcoRo35LcKbedFcSLurerpHdFRZcuv4Ny4rC7LNaCEWz6q2in5-gCzfOxzRC8uS4qAku9dunPka6yPLnRlFQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">670</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsw049"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2726856"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f71%2f2726856">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F71%2F2726856" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2726856'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome analysis of Hibiscus syriacus provides insights of polyploidization and indeterminate flowering in woody plants" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f71%2f2726856">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

