

<!DOCTYPE html>
<html lang="en">

<head><title>
	Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey (Lethenteron camtschaticum, Petromyzontidae) | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/93/2738680","@type":"ScholarlyArticle","name":"Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey ( Lethenteron camtschaticum , Petromyzontidae)","datePublished":"2016-12-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw053","keywords":["lamprey","chromosome evolution","repetitive sequence","satellite DNA","retrotransposon"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/93/2738680","author":[{"name":"Ishijima, Junko","affiliation":"1 Laboratory of Animal Cytogenetics, Department of Biological Sciences, Graduate School of Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan","@type":"Person"},{"name":"Uno, Yoshinobu","affiliation":"2 Laboratory of Animal Genetics, Department of Applied Molecular Biosciences, Nagoya University, Nagoya, Aichi 464-8601, Japan","@type":"Person"},{"name":"Nunome, Mitsuo","affiliation":"3 Avian Bioscience Research Center, Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Aichi 464-8601, Japan","@type":"Person"},{"name":"Nishida, Chizuko","affiliation":"4 Department of Natural History Sciences, Faculty of Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan","@type":"Person"},{"name":"Kuraku, Shigehiro","affiliation":"5 Phyloinformatics Unit, RIKEN Center for Life Science Technologies, Kobe 650-0047, Japan","@type":"Person"},{"name":"Matsuda, Yoichi","affiliation":"2 Laboratory of Animal Genetics, Department of Applied Molecular Biosciences, Nagoya University, Nagoya, Aichi 464-8601, Japan   3 Avian Bioscience Research Center, Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Aichi 464-8601, Japan","@type":"Person"}],"description":"All extant lamprey karyotypes are characterized by almost all dot-shaped microchromosomes. To understand the molecular basis of chromosome structure in lampreys, we performed chromosome C-banding and silver staining and chromosome mapping of the 18S–28S and 5S ribosomal RNA (rRNA) genes and telomeric TTAGGG repeats in the Arctic lamprey (Lethenteron camtschaticum). In addition, we cloned chromosome site-specific repetitive DNA sequences and characterized them by nucleotide sequencing, chromosome in situ hybridization, and filter hybridization. Three types of repetitive sequences were detected; a 200-bp AT-rich repetitive sequence, LCA-EcoRIa that co-localized with the 18S–28S rRNA gene clusters of 3 chromosomal pairs; a 364-bp AT-rich LCA-EcoRIb sequence that showed homology to the EcoRI sequence family from the sea lamprey (Petromyzon marinus), which contains short repeats as centromeric motifs; and a GC-rich 702-bp LCA-ApaI sequence that was distributed on nearly all chromosomes and showed significant homology with the integrase-coding region of a Ty3/Gypsy family long terminal repeat (LTR) retrotransposon. All three repetitive sequences are highly conserved within the Petromyzontidae or within Petromyzontidae and Mordaciidae. Molecular cytogenetic characterization of these site-specific repeats showed that they may be correlated with programed genome rearrangement (LCA-EcoRIa), centromere structure and function (LCA-EcoRIb), and site-specific amplification of LTR retroelements through homogenization between non-homologous chromosomes (LCA-ApaI).","pageStart":"93","pageEnd":"101","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217490&Signature=YSaTDXVtUkRPtDc-J1I-wCsCotiylxT5tT4ZO7yc11iYMnhg55mUsPKZb3SntYdsRU28VXHBiivsGEeyHQfI2jMed6tqQfFnfM~xvPxHl1YPtXBFNO7C9HVK6iNMNnDCOE4ZolUGBgKByk~YWpSjIaBfLao6KjV9J8z~W9B0p1aLdD-AmKXN8uiNOtlYYHfjXghVPTloGFnOi07DzQC3K8ZM1ab7sHGTfJr85oIEnz61eyGMYxfK~8YCtJ0UOAxzpf~~gZHBIDxnSDKuuL1uP~1b9dgmEBFdRAAJu7Vtg63yAimH3A25pjklER3Uwcu0Se1rNWNVKus6VhwVOx15Fg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey ( Lethenteron camtschaticum , Petromyzontidae)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1572217490&Signature=YSaTDXVtUkRPtDc-J1I-wCsCotiylxT5tT4ZO7yc11iYMnhg55mUsPKZb3SntYdsRU28VXHBiivsGEeyHQfI2jMed6tqQfFnfM~xvPxHl1YPtXBFNO7C9HVK6iNMNnDCOE4ZolUGBgKByk~YWpSjIaBfLao6KjV9J8z~W9B0p1aLdD-AmKXN8uiNOtlYYHfjXghVPTloGFnOi07DzQC3K8ZM1ab7sHGTfJr85oIEnz61eyGMYxfK~8YCtJ0UOAxzpf~~gZHBIDxnSDKuuL1uP~1b9dgmEBFdRAAJu7Vtg63yAimH3A25pjklER3Uwcu0Se1rNWNVKus6VhwVOx15Fg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey ( Lethenteron camtschaticum , Petromyzontidae) | DNA Research | Oxford Academic" />
    <meta property="og:description" content="All extant lamprey karyotypes are characterized by almost all dot-shaped microchromosomes. To understand the molecular basis of chromosome structure in lampreys, we performed chromosome C-banding and silver staining and chromosome mapping of the 18S–28S and 5S ribosomal RNA (rRNA) genes and telomeric TTAGGG repeats in the Arctic lamprey (Lethenteron camtschaticum). In addition, we cloned chromosome site-specific repetitive DNA sequences and characterized them by nucleotide sequencing, chromosome in situ hybridization, and filter hybridization. Three types of repetitive sequences were detected; a 200-bp AT-rich repetitive sequence, LCA-EcoRIa that co-localized with the 18S–28S rRNA gene clusters of 3 chromosomal pairs; a 364-bp AT-rich LCA-EcoRIb sequence that showed homology to the EcoRI sequence family from the sea lamprey (Petromyzon marinus), which contains short repeats as centromeric motifs; and a GC-rich 702-bp LCA-ApaI sequence that was distributed on nearly all chromosomes and showed significant homology with the integrase-coding region of a Ty3/Gypsy family long terminal repeat (LTR) retrotransposon. All three repetitive sequences are highly conserved within the Petromyzontidae or within Petromyzontidae and Mordaciidae. Molecular cytogenetic characterization of these site-specific repeats showed that they may be correlated with programed genome rearrangement (LCA-EcoRIa), centromere structure and function (LCA-EcoRIb), and site-specific amplification of LTR retroelements through homogenization between non-homologous chromosomes (LCA-ApaI)." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/93/2738680" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217490&amp;Signature=WGQyCuolH80cujBonS~1TZ4KUQMaNCuOLhJURiE4mWXP4ljUBPoLjjBZ1OaZA5TVPfWNwOmJ428u3CpatgoNvieemA9KfBYK-OKQZoSfrOcOt4BngDDY6Gjett~V5MY38x4dDpnBNZFpV88fYOG8tBNhB~cXfHK6wvLCshgLSsMMCWC-kuubKrv-iEAqetX~zKPSnRcTf6w4YLCXyzC5H5V0SKCb5v4ZMVs-j44DAbfU40cPcxf5CMrvQLJ86y8wYMgGZ8Ffz5j6x6SPxz9ZgnPZMutCiwqIsKpQtR~VomKUfj~mBr1nDGB~eKSGRF2-Z2TWbY-ApRCmZgXpBuGsKw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2016-12-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ishijima, Junko" /><meta name="citation_author_institution" content="Laboratory of Animal Cytogenetics, Department of Biological Sciences, Graduate School of Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan" /><meta name="citation_author" content="Uno, Yoshinobu" /><meta name="citation_author_institution" content="Laboratory of Animal Genetics, Department of Applied Molecular Biosciences, Nagoya University, Nagoya, Aichi 464-8601, Japan" /><meta name="citation_author" content="Nunome, Mitsuo" /><meta name="citation_author_institution" content="Avian Bioscience Research Center, Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Aichi 464-8601, Japan" /><meta name="citation_author" content="Nishida, Chizuko" /><meta name="citation_author_institution" content="Department of Natural History Sciences, Faculty of Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan" /><meta name="citation_author" content="Kuraku, Shigehiro" /><meta name="citation_author_institution" content="Phyloinformatics Unit, RIKEN Center for Life Science Technologies, Kobe 650-0047, Japan" /><meta name="citation_author" content="Matsuda, Yoichi" /><meta name="citation_author_institution" content="Laboratory of Animal Genetics, Department of Applied Molecular Biosciences, Nagoya University, Nagoya, Aichi 464-8601, Japan" /><meta name="citation_author_institution" content="Avian Bioscience Research Center, Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Aichi 464-8601, Japan" /><meta name="citation_title" content="Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey (Lethenteron camtschaticum, Petromyzontidae)" /><meta name="citation_firstpage" content="93" /><meta name="citation_lastpage" content="101" /><meta name="citation_doi" content="10.1093/dnares/dsw053" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Evidence from 18S ribosomal RNA sequences that lampreys and hagfishes form a natural group;  citation_journal_title=Science;  citation_year=1992;  citation_volume=257;  citation_pages=787-89; " /><meta name="citation_reference" content="citation_title=Time scale for cyclostome evolution inferred with a phylogenetic diagnosis of hagfish and lamprey cDNA sequences;  citation_journal_title=Zool. Sci;  citation_year=2006;  citation_volume=23;  citation_pages=1053-64; " /><meta name="citation_reference" content="citation_publisher=Oxford University Press, New York;  citation_title=Timetree of Life;  citation_year=2009;  citation_pages=317-9; " /><meta name="citation_reference" content="citation_publisher=Johns Hopkins University Press, Baltimore;  citation_title=North American Freshwater Fishes: Natural History, Ecology, Behavior, and Conservation;  citation_year=2014;  citation_pages=106-39; " /><meta name="citation_reference" content="citation_title=The taxonomy of the lampreys Geotria and Mordacia and their distribution in Australia;  citation_journal_title=Proc. Linn. Soc. Lond;  citation_year=1968;  citation_volume=179;  citation_pages=229-40; " /><meta name="citation_reference" content="citation_title=Phylogeny of living parasitic lampreys (Petromyzontiformes) based on morphological data;  citation_journal_title=Copeia;  citation_year=2003;  citation_volume=2003;  citation_pages=687-703; " /><meta name="citation_reference" content="citation_title=The mitotic chromosomes of the lamprey, Petromyzon marinus L;  citation_journal_title=Experientia;  citation_year=1970;  citation_volume=26;  citation_pages=429-30; " /><meta name="citation_reference" content="citation_title=Homogeneity of holarctic lamprey karyotypes;  citation_journal_title=Caryologia;  citation_year=1974;  citation_volume=27;  citation_pages=443-54; " /><meta name="citation_reference" content="citation_title=The chromosomes of the southern hemispheric lamprey, Geotria australis Gray;  citation_journal_title=Experientia;  citation_year=1981;  citation_volume=37;  citation_pages=239-40; " /><meta name="citation_reference" content="citation_title=Development and analysis of a germline BAC resource for the sea lamprey, a vertebrate that undergoes substantial chromatin diminution;  citation_journal_title=Chromosoma;  citation_year=2010;  citation_volume=119;  citation_pages=381-9; " /><meta name="citation_reference" content="citation_publisher=Springer, Heidelberg;  citation_title=Fish Karyotypes: A Check List;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Chromosomal study of a lamprey (Lampetra zanandreai Vladykov, 1955) (Petromyzonida: Petromyzontiformes): conventional and FISH analysis;  citation_journal_title=Chromosome Res;  citation_year=2011;  citation_volume=19;  citation_pages=481-91; " /><meta name="citation_reference" content="citation_title=Genome duplication in early vertebrates: insights from agnathan cytogenetics;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2013;  citation_volume=141;  citation_pages=80-9; " /><meta name="citation_reference" content="citation_title=Cytogenetic evidences of genome rearrangement and differential epigenetic chromatin modification in the sea lamprey (Petromyzon marinus);  citation_journal_title=Genetica;  citation_year=2014;  citation_volume=142;  citation_pages=545-54; " /><meta name="citation_reference" content="citation_title=Highly repeated sequences in mammalian genomes;  citation_journal_title=Int. Rev. Cytol;  citation_year=1982;  citation_volume=76;  citation_pages=67-112; " /><meta name="citation_reference" content="citation_title=Centromeres take flight: alpha satellite and the quest for the human centromere;  citation_journal_title=Cell;  citation_year=1998;  citation_volume=93;  citation_pages=317-20; " /><meta name="citation_reference" content="citation_title=The centromere paradox: stable inheritance with rapidly evolving DNA;  citation_journal_title=Science;  citation_year=2001;  citation_volume=293;  citation_pages=1098-102; " /><meta name="citation_reference" content="citation_title=A new EcoRI family of satellite DNA in lampreys;  citation_journal_title=FEBS Lett;  citation_year=1996;  citation_volume=394;  citation_pages=187-90; " /><meta name="citation_reference" content="citation_title=Somatic chromosomes of three species of Cyclostomata;  citation_journal_title=Chrom. Inform. Serv;  citation_year=1975;  citation_volume=18;  citation_pages=10-2; " /><meta name="citation_reference" content="citation_title=A simple technique for demonstrating centromeric heterochromatin;  citation_journal_title=Exp. Cell Res;  citation_year=1972;  citation_volume=75;  citation_pages=304-6; " /><meta name="citation_reference" content="citation_title=Controlled silver-staining of nucleolus organizer regions with a protective colloidal developer: a 1-step method;  citation_journal_title=Experientia;  citation_year=1980;  citation_volume=36;  citation_pages=1014-5; " /><meta name="citation_reference" content="citation_title=Divergence of repetitive DNA sequences in the heterochromatin of medaka fishes: molecular cytogenetic characterization of constitutive heterochromatin in two medaka species: Oryzias hubbsi and O. celebensis (Adrianichthyidae, Beloniformes);  citation_journal_title=Cytogenet. Genome Res;  citation_year=2013;  citation_volume=141;  citation_pages=212-26; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=Application of fluorescence in situ hybridization in genome analysis of the mouse;  citation_journal_title=Electrophoresis;  citation_year=1995;  citation_volume=16;  citation_pages=261-72; " /><meta name="citation_reference" content="citation_title=Visualization of centriole-bodies using silver stain;  citation_journal_title=Chromosoma;  citation_year=1977;  citation_volume=65;  citation_pages=9-20; " /><meta name="citation_reference" content="citation_title=Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=25;  citation_pages=474" /><meta name="citation_reference" content="citation_title=Programmed loss of millions of base pairs from a vertebrate genome;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2009;  citation_volume=106;  citation_pages=11212-7; " /><meta name="citation_reference" content="citation_title=Sequence organization and cytological localization of the minor satellite of mouse;  citation_journal_title=Nucleic Acids Res;  citation_year=1988;  citation_volume=16;  citation_pages=11645-61; " /><meta name="citation_reference" content="citation_title=A satellite III sequence shared by human chromosomes 13, 14, and 21 that is contiguous with alpha satellite DNA;  citation_journal_title=Cytogenet. Cell Genet;  citation_year=1992;  citation_volume=61;  citation_pages=81-6; " /><meta name="citation_reference" content="citation_title=The EcoRI centromeric satellite DNA of the Sparidae family (Pisces, Perciformes) contains a sequence motive common to other vertebrate centromeric satellite DNAs;  citation_journal_title=Cytogenet. Cell Genet;  citation_year=1995;  citation_volume=71;  citation_pages=345-51; " /><meta name="citation_reference" content="citation_title=Chromosomal study of native and hatchery trouts from Italy (Salmo trutta complex, Salmonidae): conventional and FISH analysis;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2009;  citation_volume=124;  citation_pages=51-62; " /><meta name="citation_reference" content="citation_title=Modular evolution of the integrase domain in the Ty3/Gypsy class of LTR retrotransposons;  citation_journal_title=J. Virol;  citation_year=1999;  citation_volume=73;  citation_pages=5186-90; " /><meta name="citation_reference" content="citation_publisher=Nova Science Publishers, Hauppauge;  citation_title=Focus on Genome Research;  citation_year=2004;  citation_pages=289-318; " /><meta name="citation_reference" content="citation_title=The nuclear DNA content and genetic diversity of Lampetra morii;  citation_journal_title=PLoS ONE;  citation_year=2016;  citation_volume=11;  citation_pages=e0157494" /><meta name="citation_reference" content="citation_title=Concerted evolution of repetitive DNA sequences in eukaryotes;  citation_journal_title=Q Rev Biol;  citation_year=1995;  citation_volume=70;  citation_pages=297-320; " /><meta name="citation_reference" content="citation_title=Satellite DNAs between selfishness and functionality: structure, genomics and evolution of tandem repeats in centromeric (hetero)chromatin;  citation_journal_title=Gene;  citation_year=2008;  citation_volume=409;  citation_pages=72-82; " /><meta name="citation_reference" content="citation_title=Identification of multiple Gypsy LTR-retrotransposon lineages in vertebrate genomes;  citation_journal_title=J. Mol. Evol;  citation_year=1999;  citation_volume=49;  citation_pages=358-66; " /><meta name="citation_reference" content="citation_title=Network dynamics of eukaryotic LTR retroelements beyond phylogenetic trees;  citation_journal_title=Biol. Direct;  citation_year=2009;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=The evolution of two partner LINE/SINE families and a full-length chromodomain-containing Ty3/Gypsy LTR element in the first reptilian genome of Anolis carolinensis;  citation_journal_title=Gene;  citation_year=2009;  citation_volume=441;  citation_pages=111-8; " /><meta name="citation_reference" content="citation_title=Evolutionary relationships among lamprey families: Amino acid composition analysis of lactate dehydrogenase;  citation_journal_title=Biochem. Syst. Ecol;  citation_year=1988;  citation_volume=16;  citation_issue=3;  citation_pages=351" /><meta name="citation_reference" content="citation_title=Cloning and analysis of the lamprey GnRH-III cDNA from eight species of lamprey representing the three families of Petromyzoniformes;  citation_journal_title=Gen. Comp. Endocrinol;  citation_year=2004;  citation_volume=139;  citation_pages=85-94; " /><meta name="citation_reference" content="citation_title=Occurrence of two functionally distinct proopiomelanocortin genes in all modern lampreys;  citation_journal_title=Gen. Comp. Endocrinol;  citation_year=2006;  citation_volume=148;  citation_pages=72-8; " /><meta name="citation_reference" content="citation_title=Impact of asymmetric gene repertoire between cyclostomes and gnathostomes;  citation_journal_title=Semin. Cell Dev. Biol;  citation_year=2013;  citation_volume=24;  citation_pages=119-27; " /><meta name="citation_reference" content="citation_title=Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=415-21; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/93/10731853/dsw053.pdf" /><meta name="description" content="Abstract.  All extant lamprey karyotypes are characterized by almost all dot-shaped microchromosomes. To understand the molecular basis of chromosome structure " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/93/2738680" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/93/2738680" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsw053';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2738680" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF4Gw8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklzaGlqaW1hLCBKdW5rbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQW5pbWFsIEN5dG9nZW5ldGljcywgRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgU2NpZW5jZSwgSG9ra2FpZG8gVW5pdmVyc2l0eSwgU2FwcG9ybywgSG9ra2FpZG8gMDYwLTA4MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVW5vLCBZb3NoaW5vYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEFuaW1hbCBHZW5ldGljcywgRGVwYXJ0bWVudCBvZiBBcHBsaWVkIE1vbGVjdWxhciBCaW9zY2llbmNlcywgTmFnb3lhIFVuaXZlcnNpdHksIE5hZ295YSwgQWljaGkgNDY0LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTnVub21lLCBNaXRzdW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBdmlhbiBCaW9zY2llbmNlIFJlc2VhcmNoIENlbnRlciwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb2FncmljdWx0dXJhbCBTY2llbmNlcywgTmFnb3lhIFVuaXZlcnNpdHksIE5hZ295YSwgQWljaGkgNDY0LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGlkYSwgQ2hpenVrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTmF0dXJhbCBIaXN0b3J5IFNjaWVuY2VzLCBGYWN1bHR5IG9mIFNjaWVuY2UsIEhva2thaWRvIFVuaXZlcnNpdHksIFNhcHBvcm8sIEhva2thaWRvIDA2MC0wODEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1cmFrdSwgU2hpZ2VoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGh5bG9pbmZvcm1hdGljcyBVbml0LCBSSUtFTiBDZW50ZXIgZm9yIExpZmUgU2NpZW5jZSBUZWNobm9sb2dpZXMsIEtvYmUgNjUwLTAwNDcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWF0c3VkYSwgWW9pY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBBbmltYWwgR2VuZXRpY3MsIERlcGFydG1lbnQgb2YgQXBwbGllZCBNb2xlY3VsYXIgQmlvc2NpZW5jZXMsIE5hZ295YSBVbml2ZXJzaXR5LCBOYWdveWEsIEFpY2hpIDQ2NC04NjAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkF2aWFuIEJpb3NjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvYWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBOYWdveWEgVW5pdmVyc2l0eSwgTmFnb3lhLCBBaWNoaSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik1vbGVjdWxhciBjeXRvZ2VuZXRpYyBjaGFyYWN0ZXJpemF0aW9uIG9mIGNocm9tb3NvbWUgc2l0ZS1zcGVjaWZpYyByZXBldGl0aXZlIHNlcXVlbmNlcyBpbiB0aGUgQXJjdGljIGxhbXByZXkgKExldGhlbnRlcm9uIGNhbXRzY2hhdGljdW0sIFBldHJvbXl6b250aWRhZSkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmcm9tIDE4UyByaWJvc29tYWwgUk5BIHNlcXVlbmNlcyB0aGF0IGxhbXByZXlzIGFuZCBoYWdmaXNoZXMgZm9ybSBhIG5hdHVyYWwgZ3JvdXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjU3OyAgY2l0YXRpb25fcGFnZXM9Nzg3LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaW1lIHNjYWxlIGZvciBjeWNsb3N0b21lIGV2b2x1dGlvbiBpbmZlcnJlZCB3aXRoIGEgcGh5bG9nZW5ldGljIGRpYWdub3NpcyBvZiBoYWdmaXNoIGFuZCBsYW1wcmV5IGNETkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ab29sLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMDUzLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9T3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MsIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9VGltZXRyZWUgb2YgTGlmZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3BhZ2VzPTMxNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Sm9obnMgSG9wa2lucyBVbml2ZXJzaXR5IFByZXNzLCBCYWx0aW1vcmU7ICBjaXRhdGlvbl90aXRsZT1Ob3J0aCBBbWVyaWNhbiBGcmVzaHdhdGVyIEZpc2hlczogTmF0dXJhbCBIaXN0b3J5LCBFY29sb2d5LCBCZWhhdmlvciwgYW5kIENvbnNlcnZhdGlvbjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTEwNi0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRheG9ub215IG9mIHRoZSBsYW1wcmV5cyBHZW90cmlhIGFuZCBNb3JkYWNpYSBhbmQgdGhlaXIgZGlzdHJpYnV0aW9uIGluIEF1c3RyYWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTGlubi4gU29jLiBMb25kOyAgY2l0YXRpb25feWVhcj0xOTY4OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTIyOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55IG9mIGxpdmluZyBwYXJhc2l0aWMgbGFtcHJleXMgKFBldHJvbXl6b250aWZvcm1lcykgYmFzZWQgb24gbW9ycGhvbG9naWNhbCBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db3BlaWE7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjAwMzsgIGNpdGF0aW9uX3BhZ2VzPTY4Ny03MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtaXRvdGljIGNocm9tb3NvbWVzIG9mIHRoZSBsYW1wcmV5LCBQZXRyb215em9uIG1hcmludXMgTDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXhwZXJpZW50aWE7ICBjaXRhdGlvbl95ZWFyPTE5NzA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MjktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbW9nZW5laXR5IG9mIGhvbGFyY3RpYyBsYW1wcmV5IGthcnlvdHlwZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhcnlvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTE5NzQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz00NDMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjaHJvbW9zb21lcyBvZiB0aGUgc291dGhlcm4gaGVtaXNwaGVyaWMgbGFtcHJleSwgR2VvdHJpYSBhdXN0cmFsaXMgR3JheTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXhwZXJpZW50aWE7ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0yMzktNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBhbmFseXNpcyBvZiBhIGdlcm1saW5lIEJBQyByZXNvdXJjZSBmb3IgdGhlIHNlYSBsYW1wcmV5LCBhIHZlcnRlYnJhdGUgdGhhdCB1bmRlcmdvZXMgc3Vic3RhbnRpYWwgY2hyb21hdGluIGRpbWludXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWE7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE5OyAgY2l0YXRpb25fcGFnZXM9MzgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgSGVpZGVsYmVyZzsgIGNpdGF0aW9uX3RpdGxlPUZpc2ggS2FyeW90eXBlczogQSBDaGVjayBMaXN0OyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBzdHVkeSBvZiBhIGxhbXByZXkgKExhbXBldHJhIHphbmFuZHJlYWkgVmxhZHlrb3YsIDE5NTUpIChQZXRyb215em9uaWRhOiBQZXRyb215em9udGlmb3JtZXMpOiBjb252ZW50aW9uYWwgYW5kIEZJU0ggYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NDgxLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZHVwbGljYXRpb24gaW4gZWFybHkgdmVydGVicmF0ZXM6IGluc2lnaHRzIGZyb20gYWduYXRoYW4gY3l0b2dlbmV0aWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9ODAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b2dlbmV0aWMgZXZpZGVuY2VzIG9mIGdlbm9tZSByZWFycmFuZ2VtZW50IGFuZCBkaWZmZXJlbnRpYWwgZXBpZ2VuZXRpYyBjaHJvbWF0aW4gbW9kaWZpY2F0aW9uIGluIHRoZSBzZWEgbGFtcHJleSAoUGV0cm9teXpvbiBtYXJpbnVzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQyOyAgY2l0YXRpb25fcGFnZXM9NTQ1LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgcmVwZWF0ZWQgc2VxdWVuY2VzIGluIG1hbW1hbGlhbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIFJldi4gQ3l0b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9NzY7ICBjaXRhdGlvbl9wYWdlcz02Ny0xMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbnRyb21lcmVzIHRha2UgZmxpZ2h0OiBhbHBoYSBzYXRlbGxpdGUgYW5kIHRoZSBxdWVzdCBmb3IgdGhlIGh1bWFuIGNlbnRyb21lcmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0zMTctMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjZW50cm9tZXJlIHBhcmFkb3g6IHN0YWJsZSBpbmhlcml0YW5jZSB3aXRoIHJhcGlkbHkgZXZvbHZpbmcgRE5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5MzsgIGNpdGF0aW9uX3BhZ2VzPTEwOTgtMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBFY29SSSBmYW1pbHkgb2Ygc2F0ZWxsaXRlIEROQSBpbiBsYW1wcmV5czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVCUyBMZXR0OyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM5NDsgIGNpdGF0aW9uX3BhZ2VzPTE4Ny05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29tYXRpYyBjaHJvbW9zb21lcyBvZiB0aHJlZSBzcGVjaWVzIG9mIEN5Y2xvc3RvbWF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb20uIEluZm9ybS4gU2VydjsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIHRlY2huaXF1ZSBmb3IgZGVtb25zdHJhdGluZyBjZW50cm9tZXJpYyBoZXRlcm9jaHJvbWF0aW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV4cC4gQ2VsbCBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz0zMDQtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbGxlZCBzaWx2ZXItc3RhaW5pbmcgb2YgbnVjbGVvbHVzIG9yZ2FuaXplciByZWdpb25zIHdpdGggYSBwcm90ZWN0aXZlIGNvbGxvaWRhbCBkZXZlbG9wZXI6IGEgMS1zdGVwIG1ldGhvZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXhwZXJpZW50aWE7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0xMDE0LTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyZ2VuY2Ugb2YgcmVwZXRpdGl2ZSBETkEgc2VxdWVuY2VzIGluIHRoZSBoZXRlcm9jaHJvbWF0aW4gb2YgbWVkYWthIGZpc2hlczogbW9sZWN1bGFyIGN5dG9nZW5ldGljIGNoYXJhY3Rlcml6YXRpb24gb2YgY29uc3RpdHV0aXZlIGhldGVyb2Nocm9tYXRpbiBpbiB0d28gbWVkYWthIHNwZWNpZXM6IE9yeXppYXMgaHViYnNpIGFuZCBPLiBjZWxlYmVuc2lzIChBZHJpYW5pY2h0aHlpZGFlLCBCZWxvbmlmb3JtZXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MjEyLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNjogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB2ZXJzaW9uIDYuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjcyNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbiBvZiBmbHVvcmVzY2VuY2UgaW4gc2l0dSBoeWJyaWRpemF0aW9uIGluIGdlbm9tZSBhbmFseXNpcyBvZiB0aGUgbW91c2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVsZWN0cm9waG9yZXNpczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI2MS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmlzdWFsaXphdGlvbiBvZiBjZW50cmlvbGUtYm9kaWVzIHVzaW5nIHNpbHZlciBzdGFpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tYTsgIGNpdGF0aW9uX3llYXI9MTk3NzsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTktMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFubm90YXRpb24sIHN1Ym1pc3Npb24gYW5kIHNjcmVlbmluZyBvZiByZXBldGl0aXZlIGVsZW1lbnRzIGluIFJlcGJhc2U6IFJlcGJhc2VTdWJtaXR0ZXIgYW5kIENlbnNvcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NDc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZ3JhbW1lZCBsb3NzIG9mIG1pbGxpb25zIG9mIGJhc2UgcGFpcnMgZnJvbSBhIHZlcnRlYnJhdGUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuUy5BOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTExMjEyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIG9yZ2FuaXphdGlvbiBhbmQgY3l0b2xvZ2ljYWwgbG9jYWxpemF0aW9uIG9mIHRoZSBtaW5vciBzYXRlbGxpdGUgb2YgbW91c2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE2NDUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2F0ZWxsaXRlIElJSSBzZXF1ZW5jZSBzaGFyZWQgYnkgaHVtYW4gY2hyb21vc29tZXMgMTMsIDE0LCBhbmQgMjEgdGhhdCBpcyBjb250aWd1b3VzIHdpdGggYWxwaGEgc2F0ZWxsaXRlIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b2dlbmV0LiBDZWxsIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9ODEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVjb1JJIGNlbnRyb21lcmljIHNhdGVsbGl0ZSBETkEgb2YgdGhlIFNwYXJpZGFlIGZhbWlseSAoUGlzY2VzLCBQZXJjaWZvcm1lcykgY29udGFpbnMgYSBzZXF1ZW5jZSBtb3RpdmUgY29tbW9uIHRvIG90aGVyIHZlcnRlYnJhdGUgY2VudHJvbWVyaWMgc2F0ZWxsaXRlIEROQXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN5dG9nZW5ldC4gQ2VsbCBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTM0NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgc3R1ZHkgb2YgbmF0aXZlIGFuZCBoYXRjaGVyeSB0cm91dHMgZnJvbSBJdGFseSAoU2FsbW8gdHJ1dHRhIGNvbXBsZXgsIFNhbG1vbmlkYWUpOiBjb252ZW50aW9uYWwgYW5kIEZJU0ggYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN5dG9nZW5ldC4gR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjQ7ICBjaXRhdGlvbl9wYWdlcz01MS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kdWxhciBldm9sdXRpb24gb2YgdGhlIGludGVncmFzZSBkb21haW4gaW4gdGhlIFR5My9HeXBzeSBjbGFzcyBvZiBMVFIgcmV0cm90cmFuc3Bvc29uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gVmlyb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz01MTg2LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Tm92YSBTY2llbmNlIFB1Ymxpc2hlcnMsIEhhdXBwYXVnZTsgIGNpdGF0aW9uX3RpdGxlPUZvY3VzIG9uIEdlbm9tZSBSZXNlYXJjaDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3BhZ2VzPTI4OS0zMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBudWNsZWFyIEROQSBjb250ZW50IGFuZCBnZW5ldGljIGRpdmVyc2l0eSBvZiBMYW1wZXRyYSBtb3JpaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE1NzQ5NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbmNlcnRlZCBldm9sdXRpb24gb2YgcmVwZXRpdGl2ZSBETkEgc2VxdWVuY2VzIGluIGV1a2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVEgUmV2IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0yOTctMzIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYXRlbGxpdGUgRE5BcyBiZXR3ZWVuIHNlbGZpc2huZXNzIGFuZCBmdW5jdGlvbmFsaXR5OiBzdHJ1Y3R1cmUsIGdlbm9taWNzIGFuZCBldm9sdXRpb24gb2YgdGFuZGVtIHJlcGVhdHMgaW4gY2VudHJvbWVyaWMgKGhldGVybyljaHJvbWF0aW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9NzItODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIG11bHRpcGxlIEd5cHN5IExUUi1yZXRyb3RyYW5zcG9zb24gbGluZWFnZXMgaW4gdmVydGVicmF0ZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz0zNTgtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldHdvcmsgZHluYW1pY3Mgb2YgZXVrYXJ5b3RpYyBMVFIgcmV0cm9lbGVtZW50cyBiZXlvbmQgcGh5bG9nZW5ldGljIHRyZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBEaXJlY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV2b2x1dGlvbiBvZiB0d28gcGFydG5lciBMSU5FL1NJTkUgZmFtaWxpZXMgYW5kIGEgZnVsbC1sZW5ndGggY2hyb21vZG9tYWluLWNvbnRhaW5pbmcgVHkzL0d5cHN5IExUUiBlbGVtZW50IGluIHRoZSBmaXJzdCByZXB0aWxpYW4gZ2Vub21lIG9mIEFub2xpcyBjYXJvbGluZW5zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDQxOyAgY2l0YXRpb25fcGFnZXM9MTExLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSByZWxhdGlvbnNoaXBzIGFtb25nIGxhbXByZXkgZmFtaWxpZXM6IEFtaW5vIGFjaWQgY29tcG9zaXRpb24gYW5hbHlzaXMgb2YgbGFjdGF0ZSBkZWh5ZHJvZ2VuYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBTeXN0LiBFY29sOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25faXNzdWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTM1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBsYW1wcmV5IEduUkgtSUlJIGNETkEgZnJvbSBlaWdodCBzcGVjaWVzIG9mIGxhbXByZXkgcmVwcmVzZW50aW5nIHRoZSB0aHJlZSBmYW1pbGllcyBvZiBQZXRyb215em9uaWZvcm1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuLiBDb21wLiBFbmRvY3Jpbm9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzOTsgIGNpdGF0aW9uX3BhZ2VzPTg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PY2N1cnJlbmNlIG9mIHR3byBmdW5jdGlvbmFsbHkgZGlzdGluY3QgcHJvb3Bpb21lbGFub2NvcnRpbiBnZW5lcyBpbiBhbGwgbW9kZXJuIGxhbXByZXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW4uIENvbXAuIEVuZG9jcmlub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTQ4OyAgY2l0YXRpb25fcGFnZXM9NzItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wYWN0IG9mIGFzeW1tZXRyaWMgZ2VuZSByZXBlcnRvaXJlIGJldHdlZW4gY3ljbG9zdG9tZXMgYW5kIGduYXRob3N0b21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2VtaW4uIENlbGwgRGV2LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTE5LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIG9mIHRoZSBzZWEgbGFtcHJleSAoUGV0cm9teXpvbiBtYXJpbnVzKSBnZW5vbWUgcHJvdmlkZXMgaW5zaWdodHMgaW50byB2ZXJ0ZWJyYXRlIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTQxNS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzEvOTMvMTA3MzE4NTMvZHN3MDUzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBbGwgZXh0YW50IGxhbXByZXkga2FyeW90eXBlcyBhcmUgY2hhcmFjdGVyaXplZCBieSBhbG1vc3QgYWxsIGRvdC1zaGFwZWQgbWljcm9jaHJvbW9zb21lcy4gVG8gdW5kZXJzdGFuZCB0aGUgbW9sZWN1bGFyIGJhc2lzIG9mIGNocm9tb3NvbWUgc3RydWN0dXJlICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjczODY4MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGQtuQ4lAbf1R6PHHP6IMbX7JdUDlw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f93%2f2738680" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f93%2f2738680" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=1509239084&amp;Signature=aLWDBixemHXKeu3p6McZmXZgXFz7xf7WLelwCCjdY~8udFvL8S~eId77UuZY5VBw0ysRWKx2B5waz613KnE0Z0gblzm28j5V8p0Rb5arIDU-Eg7sHtDEcIWGicykl4m08WcbPfF7JSQFpmh0HbcvUonRgVzC~0gC5kCp6~TugOdweXMyLwAHqyibS5VZcuMPcIrUQc4cDTH~8oG6PnCU-Mq~x~p00FnOgNgmaRDk-zmyhqKKpDJIyYQF4aHN5Y3HQxlNd3BcCwsGzk9VcEjR~sy2wigEIqZpfWD-1o0dCoy9PT8p-WiwywssxMqIeRDLdy~Qq6m8CeqNRKZYRvq4ug__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61463633">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463633">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463635">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463635">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463639">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463639">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463654">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463654">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463671">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463671">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463677">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463677">Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463679">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463679">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463681">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463681">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463683">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463683">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463685">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463685">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/81/2656160">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey (<em>Lethenteron camtschaticum</em>, Petromyzontidae)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Junko Ishijima</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junko Ishijima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratory of Animal Cytogenetics, Department of Biological Sciences, Graduate School of Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Junko+Ishijima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishijima J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Ishijima author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Yoshinobu Uno</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshinobu Uno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Laboratory of Animal Genetics, Department of Applied Molecular Biosciences, Nagoya University, Nagoya, Aichi 464-8601, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yoshinobu+Uno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uno Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Uno author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Mitsuo Nunome</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuo Nunome</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Avian Bioscience Research Center, Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Aichi 464-8601, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mitsuo+Nunome">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nunome M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nunome author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Chizuko Nishida</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chizuko Nishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Department of Natural History Sciences, Faculty of Science, Hokkaido University, Sapporo, Hokkaido 060-0810, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Chizuko+Nishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishida C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nishida author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Shigehiro Kuraku</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiro Kuraku</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Phyloinformatics Unit, RIKEN Center for Life Science Technologies, Kobe 650-0047, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shigehiro+Kuraku">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuraku S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kuraku author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Yoichi Matsuda</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoichi Matsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Laboratory of Animal Genetics, Department of Applied Molecular Biosciences, Nagoya University, Nagoya, Aichi 464-8601, Japan</div><div class="aff"><div class="label -label">3</div>Avian Bioscience Research Center, Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Aichi 464-8601, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw053-cor1" data-open="dsw053-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 52 789 4100. Fax. +81 52 789 4099. Email: <a href="mailto:yoimatsu@agr.nagoya-u.ac.jp" target="_blank">yoimatsu@agr.nagoya-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yoichi+Matsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsuda Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Matsuda author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw053-FM2"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw053-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 93–101, <a href='https://doi.org/10.1093/dnares/dsw053'>https://doi.org/10.1093/dnares/dsw053</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 December 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 May 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 November 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2738680" href="/dnaresearch/article-pdf/24/1/93/10731853/dsw053.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Junko Ishijima, Yoshinobu Uno, Mitsuo Nunome, Chizuko Nishida, Shigehiro Kuraku, Yoichi Matsuda;  Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey (<em>Lethenteron camtschaticum</em>, Petromyzontidae), <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 93–101, <a href='https://doi.org/10.1093/dnares/dsw053'>https://doi.org/10.1093/dnares/dsw053</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738680?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Molecular%20cytogenetic%20characterization%20of%20chromosome%20site-specific%20repetitive%20sequences%20in%20the%20Arctic%20lamprey%20(%26lt%3Bem%26gt%3BLethenteron%20camtschaticum%26lt%3B%2Fem%26gt%3B%2C%20Petromyzontidae)&amp;publicationDate=2016-12-24&amp;volumeNum=24&amp;issueNum=1&amp;author=Ishijima%2C%20Junko%3B%20Uno%2C%20Yoshinobu&amp;startPage=93&amp;endPage=101&amp;contentId=10.1093%2Fdnares%2Fdsw053&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f93%2f2738680" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61463633 id="61463633" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>All extant lamprey karyotypes are characterized by almost all dot-shaped microchromosomes. To understand the molecular basis of chromosome structure in lampreys, we performed chromosome C-banding and silver staining and chromosome mapping of the 18S–28S and 5S ribosomal RNA (rRNA) genes and telomeric TTAGGG repeats in the Arctic lamprey (<em>Lethenteron camtschaticum</em>). In addition, we cloned chromosome site-specific repetitive DNA sequences and characterized them by nucleotide sequencing, chromosome <em>in situ</em> hybridization, and filter hybridization. Three types of repetitive sequences were detected; a 200-bp AT-rich repetitive sequence, LCA-EcoRIa that co-localized with the 18S–28S rRNA gene clusters of 3 chromosomal pairs; a 364-bp AT-rich LCA-EcoRIb sequence that showed homology to the EcoRI sequence family from the sea lamprey (<em>Petromyzon marinus</em>), which contains short repeats as centromeric motifs; and a GC-rich 702-bp LCA-ApaI sequence that was distributed on nearly all chromosomes and showed significant homology with the integrase-coding region of a Ty3/Gypsy family long terminal repeat (LTR) retrotransposon. All three repetitive sequences are highly conserved within the Petromyzontidae or within Petromyzontidae and Mordaciidae. Molecular cytogenetic characterization of these site-specific repeats showed that they may be correlated with programed genome rearrangement (LCA-EcoRIa), centromere structure and function (LCA-EcoRIb), and site-specific amplification of LTR retroelements through homogenization between non-homologous chromosomes (LCA-ApaI).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="lamprey">lamprey</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromosome evolution&quot;">chromosome evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;repetitive sequence&quot;">repetitive sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;satellite DNA&quot;">satellite DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="retrotransposon">retrotransposon</a></div>                            <h2 scrollto-destination=61463635 id="61463635" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Lampreys (Petromyzontiformes) and hagfishes (Myxiniformes), which are known as cyclostomes or agnathans, occupy a basal position in the phylogeny of extant vertebrates and possess anatomical and developmental features that are quite different from those of gnathostomes (jawed vertebrates). Therefore, these animals are excellent models for understanding the evolution of vertebrates in a wide range of scientific ﬁelds, including genomics, immunology, endocrinology, neurobiology, and developmental biology. Molecular phylogenetic analyses have shown that extant lampreys and hagfishes diverged from the common ancestors of jawless vertebrates ∼480 million years ago (MYA).<a reveal-id="dsw053-B1 dsw053-B2 dsw053-B3" data-open="dsw053-B1 dsw053-B2 dsw053-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> All extant lamprey species are classified into a single order, Petromyzontiformes, which is composed of three families, Geotriidae, Mordaciidae, and Petromyzontidae.<a reveal-id="dsw053-B4" data-open="dsw053-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Geotriidae and Mordaciidae are distributed in the southern hemisphere, whereas Petromyzontidae are distributed in the Northern Hemisphere.<a reveal-id="dsw053-B5" data-open="dsw053-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw053-B6" data-open="dsw053-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Their karyotypes are very different from those of other vertebrates and are characterized by almost all dot-shaped chromosomes (regarded as microchromosomes). Species of Geotriidae and Petromyzontidae have 142–184 chromosomes, whereas Mordaciidae species have 76 chromosomes.<a reveal-id="dsw053-B7 dsw053-B8 dsw053-B9 dsw053-B10 dsw053-B11 dsw053-B12 dsw053-B13 dsw053-B14" data-open="dsw053-B7 dsw053-B8 dsw053-B9 dsw053-B10 dsw053-B11 dsw053-B12 dsw053-B13 dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>7–14</sup></a> However, the chromosome number in hagfishes is much smaller and variable (2<em>n</em> = 14, 28, 34, or 36).<a reveal-id="dsw053-B11" data-open="dsw053-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw053-B13" data-open="dsw053-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The GC-content of 4-fold degenerate sites (GC<sub>4</sub>) in the protein-coding regions also differ greatly between lampreys and hagfishes; GC<sub>4</sub> is much higher in lampreys than in hagfishes.<a reveal-id="dsw053-B2" data-open="dsw053-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> These results suggested that drastic chromosomal and genomic rearrangements occurred in either or both of the lineages after divergence from the common ancestor of cyclostomes.<a reveal-id="dsw053-B13" data-open="dsw053-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p><p>Highly repetitive DNA sequences are classified into two categories based on their genomic organization and chromosomal distribution.<a reveal-id="dsw053-B15" data-open="dsw053-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The first category consists of interspersed repetitive sequences such as short interspersed nuclear elements (SINEs) and long interspersed nuclear elements (LINEs), which are distributed throughout the genome. The second category is chromosome site-specific and highly repetitive sequences such as satellite DNA, which are associated with constitutive or centromeric heterochromatin. Chromosome site-specific repetitive sequences generally show a high rate of nucleotide substitution and are divergent between even closely related species.<a reveal-id="dsw053-B16" data-open="dsw053-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw053-B17" data-open="dsw053-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Thus, identification and comparison of site-specific repetitive sequences aids in tracing the evolutionary process of speciation. An EcoRI family of satellite DNA sequences identified in the sea lamprey <em>Petromyzon marinus</em> (Petromyzontidae) was the first reported in lampreys<a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and was shown to be distributed in the centromeric regions of 12 or 34 chromosomes.<a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> This repetitive sequence has high AT content and contains short repeat units. Sequences from the same family of EcoRI satellite DNA were later identified in two other Petromyzontidae species <em>Lampetra planeri</em> and <em>Lampetra zanandreai</em>, and these DNA sequences were localized to only two chromosomes in <em>La. zanandreai</em>.<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In lampreys, molecular and cytogenetic characterization of chromosome site-specific repetitive sequences is limited to this EcoRI satellite DNA.</p><p>The Arctic lamprey (<em>Lethenteron camtschaticum</em>, Petromyzontidae) is the most common and widespread lamprey in the arctic region from Lapland in the north to Kamchatka in the east and Japan and Korea in the south. To our knowledge, there has been only one study on the karyotype of this species, whose chromosome number has been reported to range from 144 to 162.<a reveal-id="dsw053-B19" data-open="dsw053-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> In this study, we aimed to characterize the molecular basis of its chromosome structure through conventional and molecular cytogenetic approaches and to clone chromosome site-specific repetitive sequences associated with the heterochromatin of <em>Le</em>. <em>camtschaticum</em>. We performed conventional Giemsa staining of chromosomes, chromosome banding (C-banding and silver staining), and chromosome mapping of the 18S–28S and 5S ribosomal RNA (rRNA) genes and telomeric TTAGGG repeats. Next, we isolated three different chromosome site-specific repetitive sequences, determined their nucleotide sequences and chromosomal distribution, and examined the conservation of these nucleotide sequences in other lamprey species. Based on the results, we discuss the process of chromosomal evolution in lampreys.</p>                            <h2 scrollto-destination=61463639 id="61463639" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=61463640 id="61463640" class="section-title" >2.1. Animals</h3>
<p>The Arctic lamprey (<em>Lethenteron camtschaticum</em>; synonym <em>Le. japonicum</em>, Petromyzontidae) was used for chromosome preparation and molecular cloning of repetitive sequences. After decapitation of two male and three female <em>Le. camtschaticum</em>, 5 ml of whole blood was collected from each individual in a test tube containing 20 µl of sodium heparin (final concentration: 10 IU/ml) (Mochida Pharmaceutical Co. Ltd., Tokyo, Japan) from each individual using a syringe and was used for cell culture and extraction of genomic DNA. All experimental procedures using animals were performed in accordance with the guidelines established by the Animal Care Committee of Hokkaido University.</p>                            <h3 scrollto-destination=61463642 id="61463642" class="section-title" >2.2. Lymphocyte culture</h3>
<p>Heparinized blood of <em>Le. camtschaticum</em> was added to Medium 199 (Thermo Fisher Scientific-Gibco, Carlsbad, CA, USA) containing 60 µg/ml kanamycin sulfate (Sigma-Aldrich, St. Louis, MO, USA) at a ratio of 1:5. After centrifugation at 300×<em>g</em> for 5 min, lymphocytes were obtained with a sterile Pasteur pipette from the buffy coat layer that is situated between the plasma and erythrocyte layers. The lymphocytes were suspended in culture medium [Medium 199 supplemented with 10% FBS, 100 µg/ml lipopolysaccharide (LPS, Sigma-Aldrich), 18 µg/ml phytohemagglutinin, M form (Thermo Fisher Scientific-Gibco), 60 µg/ml kanamycin sulfate, and 1% of Antibiotic–Antimycotic liquid (PSA, Thermo Fisher Scientific-Gibco)] and cultured in plastic bottles at 20 °C under CO<sub>2</sub>-free condition.</p>                            <h3 scrollto-destination=61463644 id="61463644" class="section-title" >2.3. Chromosome preparation and banding</h3>
<p>Lymphocytes were cultured for 6 days and exposed to 500 ng/ml colcemid for 2.5 h before harvest. The cells were collected by centrifugation (300×<em>g</em>, 7 min) and treated with 0.075 M KCl for 20 min at room temperature. The suspension was then fixed in methanol/acetic acid (3:1) and spread on clean glass slides using an air-drying method. For karyotyping, the slides were stained with 3% Giemsa solution (pH 6.8) for 10 min. The chromosomes were counted on printed pictures using a colony counter. C-bands were obtained by the barium hydroxide/saline/Giemsa (BSG) method.<a reveal-id="dsw053-B20" data-open="dsw053-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> C-banding and Ag-NOR staining<a reveal-id="dsw053-B21" data-open="dsw053-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> were performed on slides previously used for fluorescence <em>in situ</em> hybridization (FISH) analysis.</p>                            <h3 scrollto-destination=61463646 id="61463646" class="section-title" >2.4. Molecular cloning of repetitive sequence</h3>
<p>Extraction of genomic DNA and molecular cloning of repetitive sequences were performed as previously described.<a reveal-id="dsw053-B22" data-open="dsw053-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Genomic DNA was digested with restriction enzymes, electrophoresed on a 1.2% agarose gel, and DNA bands containing repetitive sequences were isolated from the gel and cloned into vectors. After the nucleotide sequences of the cloned repetitive sequences were obtained, their chromosome distribution, genomic organization, and sequence conservation in other lamprey species were examined. For detailed information, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span>.</p>                            <h3 scrollto-destination=61463648 id="61463648" class="section-title" >2.5. Molecular phylogeny</h3>
<p>Phylogenetic trees of the repetitive DNA sequences were constructed using the neighbor-joining (NJ) method as implemented in MEGA 6.0 software.<a reveal-id="dsw053-B23" data-open="dsw053-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> We used the Kimura-2-parameter model for nucleotide sequences and the Poisson model for amino acid sequences with Gamma distribution parameter <em>α</em> = 4 for rates and pattern settings. The bootstrap branch-supporting statistic was calculated for 1,000 replicates.</p>                            <h3 scrollto-destination=61463650 id="61463650" class="section-title" >2.6. FISH</h3>
<p>To examine the chromosomal distribution of the repetitive sequences isolated in this study, 18S–28S and 5S rRNA genes, and telomeric TTAGGG repeats, FISH analysis was performed as previously described.<a reveal-id="dsw053-B24" data-open="dsw053-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> For detailed information, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span>.</p>                            <h3 scrollto-destination=61463652 id="61463652" class="section-title" >2.7. Hybridization analysis of DNA blots</h3>
<p>The genomic organization of repetitive sequences and their sequence conservation were examined by Southern blot hybridization and slot-blot hybridization, respectively. For detailed methods, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span>.</p>                            <h2 scrollto-destination=61463654 id="61463654" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=61463655 id="61463655" class="section-title" >3.1. Karyotype of Le. camtschaticum and the chromosomal location of the major rRNA genes and telomeric repeat sequences</h3>

  <p>
    <a reveal-id="dsw053-F1" data-open="dsw053-F1" class="link link-reveal link-table xref-fig">Figure 1A</a> shows a Giemsa-stained metaphase spread of <em>Le. camtschaticum</em>, which consists of mostly small acrocentric or dot-shaped chromosomes and several pairs of small submetacentric chromosomes. The modal diploid chromosome number for 81 metaphase spreads was 168 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&amp;Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S1</a></span>). We collected chromosome samples from five individuals (two males and three females), and no differences in chromosome number and size were found between males and females. C-banding detected distinctive C-positive heterochromatin blocks on ∼30 pairs of chromosomes (<a reveal-id="dsw053-F1" data-open="dsw053-F1" class="link link-reveal link-table xref-fig">Fig. 1B and C</a>). Large C-bands were localized to the centromeric regions of the largest pair of submetacentric chromosomes and the centromeric regions of three other pairs of chromosomes. Distinctive C-bands were also located in other many small acrocentric and dot-shaped chromosomes. <div data-id="dsw053-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f1.png?Expires=1509239091&amp;Signature=P32A4ZF6-sDoF5rXboskBsJzlFk-jhbbgShoVCk~lkcBBnv3h9-hR-DjYJBMww~vmZTSFuzxI~-iiYwOvtjZbDT8ToRXZubjzU2HWW6LSYUTu4w0~ecb2a~eYdx5bMXTSgDmYDhRtPw18hVdzTV~d9jC6TT6gS1fIH2fMgJUa4o5BCSKuPd9q79DsRhNdaxtNduiY844YhEH4oyYaPehEvczerYpva9CQQocofvBYCXPe9DjVKHkXY5wGATms5C1axxTAbv06jo2XRo28ViBB-XBhD7059TLPZAqtoLB6KTaTAoqZQPqqrBWRwC31cuNIqwn85Vblltb2SxnlzKCgQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Giemsa-stained and C-banded chromosomes of Lethenteron camtschaticum. (A) Giemsa-stained metaphase spread. (B) C-banded metaphase spread. (C) C-banded karyotype. Scale bars represent 10 µm. The largest pair of submetacentric chromosomes and three pairs of chromosomes that have large C-bands are underlined." path-from-xml="dsw053f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463656/dsw053f1.tif" path-from-xml="dsw053f1.tif" target="_blank">View large</a><a section="61463656" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f1.png?Expires=1509239091&amp;Signature=hEukvO3ReOC1zikOq9P4dbjKbea0GIzn5cxi2517ha~9OPWg-tioHUyc6aFR8Z8Kg0VtUC6w7zmeE2lTzJ1FZmiZiYrIUJWki~yE3h4s04kuDgnBESxbsBmemdzBPt0ynYCALzuPnxsxOH98oTgMgtZ8qeWL0J8KndbCSEWKWS-iaHwO1iw1F4PpCfNhz6uwxhGD~lRL4NXR0zPkg-YuhaIPrkqVWnix3F02VTcafL~JgKYWQcvZxCKvgpH~lXTHmoQoCljaOhce1CptXnn2IoYp2jKXpPTKEaNJlOJ2e8RHA-eKnRwTGafoiwkyCIs9K8vCpQD~Q7sRM~jwOb6ufw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463656&amp;ar=2738680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw053f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Giemsa-stained and C-banded chromosomes of <em>Lethenteron camtschaticum</em>. (A) Giemsa-stained metaphase spread. (B) C-banded metaphase spread. (C) C-banded karyotype. Scale bars represent 10 µm. The largest pair of submetacentric chromosomes and three pairs of chromosomes that have large C-bands are underlined.</p></div></div><div content-id="dsw053-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f1.png?Expires=1509239091&amp;Signature=P32A4ZF6-sDoF5rXboskBsJzlFk-jhbbgShoVCk~lkcBBnv3h9-hR-DjYJBMww~vmZTSFuzxI~-iiYwOvtjZbDT8ToRXZubjzU2HWW6LSYUTu4w0~ecb2a~eYdx5bMXTSgDmYDhRtPw18hVdzTV~d9jC6TT6gS1fIH2fMgJUa4o5BCSKuPd9q79DsRhNdaxtNduiY844YhEH4oyYaPehEvczerYpva9CQQocofvBYCXPe9DjVKHkXY5wGATms5C1axxTAbv06jo2XRo28ViBB-XBhD7059TLPZAqtoLB6KTaTAoqZQPqqrBWRwC31cuNIqwn85Vblltb2SxnlzKCgQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Giemsa-stained and C-banded chromosomes of Lethenteron camtschaticum. (A) Giemsa-stained metaphase spread. (B) C-banded metaphase spread. (C) C-banded karyotype. Scale bars represent 10 µm. The largest pair of submetacentric chromosomes and three pairs of chromosomes that have large C-bands are underlined." path-from-xml="dsw053f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463656/dsw053f1.tif" path-from-xml="dsw053f1.tif" target="_blank">View large</a><a section="61463656" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f1.png?Expires=1509239091&amp;Signature=hEukvO3ReOC1zikOq9P4dbjKbea0GIzn5cxi2517ha~9OPWg-tioHUyc6aFR8Z8Kg0VtUC6w7zmeE2lTzJ1FZmiZiYrIUJWki~yE3h4s04kuDgnBESxbsBmemdzBPt0ynYCALzuPnxsxOH98oTgMgtZ8qeWL0J8KndbCSEWKWS-iaHwO1iw1F4PpCfNhz6uwxhGD~lRL4NXR0zPkg-YuhaIPrkqVWnix3F02VTcafL~JgKYWQcvZxCKvgpH~lXTHmoQoCljaOhce1CptXnn2IoYp2jKXpPTKEaNJlOJ2e8RHA-eKnRwTGafoiwkyCIs9K8vCpQD~Q7sRM~jwOb6ufw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463656&amp;ar=2738680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw053f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Giemsa-stained and C-banded chromosomes of <em>Lethenteron camtschaticum</em>. (A) Giemsa-stained metaphase spread. (B) C-banded metaphase spread. (C) C-banded karyotype. Scale bars represent 10 µm. The largest pair of submetacentric chromosomes and three pairs of chromosomes that have large C-bands are underlined.</p></div></div></p>
<p>FISH signals produced by the fluorescent probe for the 18S–28S rRNA genes were localized to six chromosomes; two signals were located in the pericentromeric regions of submetacentric chromosomes, and the others were in the terminal regions of four acrocentric chromosomes, which corresponded to C-band-positive heterochromatin regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1A</a></span>). To identify the active sites of the 18S–28S rRNA genes, we performed silver staining on the same chromosome slides used for FISH analysis of the 18S–28S rRNA genes.<a reveal-id="dsw053-B25" data-open="dsw053-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Ag-NORs were observed in the pericentromeric region of one submetacentric chromosome pair, but not on the other four acrocentric chromosomes bearing 18S–28S rRNA signals (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1B</a></span>). Only slight variations in the size of the Ag-NOR signals were observed among the individuals examined (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1C</a></span>). The 5S rDNA signals were detected on a single pair of small chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1D</a></span>), which was different from the chromosomes on which the 18S–28S rRNA signals and C-positive bands were localized (data not shown). FISH signals for TTAGGG repeat sequences were detected on the terminal ends of all chromosomes, and there were large differences in the size of signals between chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S1E</a></span>).</p>                            <h3 scrollto-destination=61463658 id="61463658" class="section-title" >3.2. Nucleotide sequences of the repetitive sequence families in the Le. camtschaticum genome</h3>
<p>We isolated repetitive sequences of the <em>Le. camtschaticum</em> genome by agarose gel electrophoresis of genomic DNA digested with restriction endonucleases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S2</a></span>). Nine clones obtained from a 200-bp EcoRI band were grouped into the same repetitive sequence family, named LCA-EcoRIa (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S3A</a></span>). The lengths of the nine fragments (LC149791–LC149799) were 200–202 bp, and the sequence similarity between the fragments ranged from 93.7 to 100.0%. The average GC content of the fragments was 47.5% (range 47.0–48.0%). Another EcoRI sequence family, LCA-EcoRIb, was isolated from a 360-bp EcoRI band (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S3B</a></span>). The length of all 11 isolated fragments (LC149800–LC149810) was 364 bp, and the sequence similarity between fragments ranged from 96.1 to 99.7%. The sequences were AT-rich, and the average GC content was 41.5% (range 40.4–42.0%). Seven clones of the LCA-ApaI sequence family were isolated from a 700-bp ApaI band. The lengths of the seven fragments (LC149811–LC149817) ranged from 700 to 702 bp, and the consensus length was 702 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S3C</a></span>). The sequence similarity between the fragments ranged from 91.3 to 99.4%. In contrast to the LCA-EcoRIa and LCA-EcoRIb sequences, the LCA-ApaI sequences were GC-rich, and the average GC content was 57.2% (range 56.7–57.8%).</p>                            <h3 scrollto-destination=61463660 id="61463660" class="section-title" >3.3. Similarity of the repetitive sequence families to other nucleotide sequences</h3>

  <p>Homology searches using the three repetitive sequence families were performed using the NCBI non-redundant sequence database (<a class="link link-uri" href="http://blast.ncbi.nlm.nih.gov" target="_blank">http://blast.ncbi.nlm.nih.gov</a>) and Repbase (<a class="link link-uri" href="http://www.girinst.org/repbase/" target="_blank">http://www.girinst.org/repbase/</a>).<a reveal-id="dsw053-B26" data-open="dsw053-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> The 200-bp LCA-EcoRIa consensus sequence showed high sequence identity with the nucleotide sequence of a BAC clone (KF318012) isolated from this species, with 94.5% identity at positions 6–205 and 97.6% identity from 206 to 371 in KF318012. The search also identified two partial sequences in a tandem repeat element, rpt200, in <em>Germ1</em> (GQ215662) of <em>P. marinus</em>, which is a genomic DNA sequence that is eliminated from germ cells by a programmed genome rearrangement.<a reveal-id="dsw053-B27" data-open="dsw053-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The sequences showed 76.8% identity at positions 5,133–5,283 (named as the rpt200_1 fragment) and 80.9% identity at positions 7,489–7,556 (rpt200_2) in the rpt200 element (<a reveal-id="dsw053-F2" data-open="dsw053-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The LCA-EcoRIb consensus sequence showed 84.8, 89.4, and 89.1% identity to the EcoRI family of satellite DNA sequences in <em>P. marinus</em> (X92515), <em>La. planeri</em>, and <em>La. zanandreai</em>, respectively (<a reveal-id="dsw053-F2" data-open="dsw053-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>).<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Short repeats, such as centromeric motifs, that appear in the centromeric sequences of several eukaryotic species,<a reveal-id="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" data-open="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" class="link link-ref link-reveal xref-bibr"><sup>28–31</sup></a> were detected in the consensus sequences of LCA-EcoRIb as well as the EcoRI satellite DNA sequences of the three lamprey species<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> (<a reveal-id="dsw053-F2" data-open="dsw053-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). A search using the LCA-ApaI sequence identified a partial sequence of the <em>P. marinus</em> long terminal repeat (LTR) retrotransposon Gypsy-26_PM-I in Repbase (<a reveal-id="dsw053-F2" data-open="dsw053-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). A search using the InterProScan tool (<a class="link link-uri" href="http://www.ebi.ac.uk/interpro/" target="_blank">http://www.ebi.ac.uk/interpro/</a>) revealed that LCA-ApaI shared 70.7% identity with a partial region of the integrase domain at position 4,252–4,954 of Gypsy-26_PM-I (<a reveal-id="dsw053-F3" data-open="dsw053-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). <div data-id="dsw053-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f2.png?Expires=1509239091&amp;Signature=GNqFK3Vtkh4bahxH-nsiaUYFitMQxXCTbBAZxcH-QOtw7JvAYJ3hCbdHJrgwTaiyRyvxh3OWW1HBWFni7rvu5OH-0LRtPe~AG1LkJhNyb3JttOk6rX3TGhtLecHbEtI08GHmw~vaO3seIfu4-V39g2g4FQxWr4WE3ccuJMygwbSbc2nV2gECD1c3FbhR-Bq45jMc588x4kDWbGrpWD~kwwVHQcHJYM5SJbnrfsud45DwMDWqSbl6dUW-axAIpFPaYJo7sijvgkrj0Ln7jiuiHgcWgaj68NQdrldUw8eTIzxOKlt3XOrNt8mUgwujB5WEjY-ruTogzorXdpDX2LC1Rg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparison of the nucleotide sequences of the LCA-EcoRIa, LCA-EcoRIb, and LCA-ApaI sequences with homologous sequences. (A) Alignment of the LCA-EcoRIa consensus sequence with two partial sequences of the tandem repeat sequence rpt200 (rpt200_1 at positions 5,133–5,283 and rpt200_2 at positions 7,489–7,556 of the Germ1 sequence (GQ215662) of P. marinus. (B) Alignment of the LCA-EcoRIb consensus sequence with the EcoRI satellite DNA sequence families from La. planeri (LPL),12La. zanandreai (LZA),12 and P. marinus (PMA; X92515).18 The 6-bp motif (A/T)(G/C)AAA(T/C) and its complementary sequence, which are homologous to centromeric motifs observed in other vertebrates,28–31 are shown in squares on the consensus sequence of LCA-EcoRIb. (C) Alignment of the LCA-ApaI consensus sequence with a partial sequence of the P. marinus Ty3/Gypsy family of LTR retrotransposons (Gypsy-26_PM-I at positions 4,252–4,954). Dots indicate identity with the LCA-EcoRIa, LCA-EcoRIb, or LCA-ApaI consensus sequence. The numbers at the end of lines are the sequence lengths." path-from-xml="dsw053f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463661/dsw053f2.tif" path-from-xml="dsw053f2.tif" target="_blank">View large</a><a section="61463661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f2.png?Expires=1509239091&amp;Signature=ZKcYi-MfjOrjg7E7kU66WsnyETI59f3-CTXOvzHupntOXMt3itgaI4KUAiKsy9VrCf50OD85PlmPnpbC05AeMPfRRPyMxm4NxFVlpi~FvoRvkppcfmYas-4QUuq4rZUP2lsHtOcXNUvxceN8SmC5Gd66rHY~ZzmG4KXK1Fl8mgkhZUjNCsUu83M7ZDg7JcbZY17TbeQy13PUT16lDcQpXD8HMlrRiwJCdRAQfx6vEfQYI6fOrcqXhcLM4xJQeXQNcw5oqXT5Dacj3BzieBMI-5ECFW5fp2gsmdDsculnHr-Niw01NpJ1v9AvcAkewM9VvVYq9ibKF~OCqZfHIvg52g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463661&amp;ar=2738680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw053f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the nucleotide sequences of the LCA-EcoRIa, LCA-EcoRIb, and LCA-ApaI sequences with homologous sequences. <strong>(A)</strong> Alignment of the LCA-EcoRIa consensus sequence with two partial sequences of the tandem repeat sequence rpt200 (rpt200_1 at positions 5,133–5,283 and rpt200_2 at positions 7,489–7,556 of the <em>Germ1</em> sequence (GQ215662) of <em>P. marinus</em>. <strong>(B)</strong> Alignment of the LCA-EcoRIb consensus sequence with the EcoRI satellite DNA sequence families from <em>La. planeri</em> (LPL),<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>La. zanandreai</em> (LZA),<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and <em>P. marinus</em> (PMA; X92515).<a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The 6-bp motif (A/T)(G/C)AAA(T/C) and its complementary sequence, which are homologous to centromeric motifs observed in other vertebrates,<a reveal-id="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" data-open="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" class="link link-ref link-reveal xref-bibr"><sup>28–31</sup></a> are shown in squares on the consensus sequence of LCA-EcoRIb. <strong>(C)</strong> Alignment of the LCA-ApaI consensus sequence with a partial sequence of the <em>P. marinus</em> Ty3/Gypsy family of LTR retrotransposons (Gypsy-26_PM-I at positions 4,252–4,954). Dots indicate identity with the LCA-EcoRIa, LCA-EcoRIb, or LCA-ApaI consensus sequence. The numbers at the end of lines are the sequence lengths.</p></div></div><div content-id="dsw053-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f2.png?Expires=1509239091&amp;Signature=GNqFK3Vtkh4bahxH-nsiaUYFitMQxXCTbBAZxcH-QOtw7JvAYJ3hCbdHJrgwTaiyRyvxh3OWW1HBWFni7rvu5OH-0LRtPe~AG1LkJhNyb3JttOk6rX3TGhtLecHbEtI08GHmw~vaO3seIfu4-V39g2g4FQxWr4WE3ccuJMygwbSbc2nV2gECD1c3FbhR-Bq45jMc588x4kDWbGrpWD~kwwVHQcHJYM5SJbnrfsud45DwMDWqSbl6dUW-axAIpFPaYJo7sijvgkrj0Ln7jiuiHgcWgaj68NQdrldUw8eTIzxOKlt3XOrNt8mUgwujB5WEjY-ruTogzorXdpDX2LC1Rg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparison of the nucleotide sequences of the LCA-EcoRIa, LCA-EcoRIb, and LCA-ApaI sequences with homologous sequences. (A) Alignment of the LCA-EcoRIa consensus sequence with two partial sequences of the tandem repeat sequence rpt200 (rpt200_1 at positions 5,133–5,283 and rpt200_2 at positions 7,489–7,556 of the Germ1 sequence (GQ215662) of P. marinus. (B) Alignment of the LCA-EcoRIb consensus sequence with the EcoRI satellite DNA sequence families from La. planeri (LPL),12La. zanandreai (LZA),12 and P. marinus (PMA; X92515).18 The 6-bp motif (A/T)(G/C)AAA(T/C) and its complementary sequence, which are homologous to centromeric motifs observed in other vertebrates,28–31 are shown in squares on the consensus sequence of LCA-EcoRIb. (C) Alignment of the LCA-ApaI consensus sequence with a partial sequence of the P. marinus Ty3/Gypsy family of LTR retrotransposons (Gypsy-26_PM-I at positions 4,252–4,954). Dots indicate identity with the LCA-EcoRIa, LCA-EcoRIb, or LCA-ApaI consensus sequence. The numbers at the end of lines are the sequence lengths." path-from-xml="dsw053f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463661/dsw053f2.tif" path-from-xml="dsw053f2.tif" target="_blank">View large</a><a section="61463661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f2.png?Expires=1509239091&amp;Signature=ZKcYi-MfjOrjg7E7kU66WsnyETI59f3-CTXOvzHupntOXMt3itgaI4KUAiKsy9VrCf50OD85PlmPnpbC05AeMPfRRPyMxm4NxFVlpi~FvoRvkppcfmYas-4QUuq4rZUP2lsHtOcXNUvxceN8SmC5Gd66rHY~ZzmG4KXK1Fl8mgkhZUjNCsUu83M7ZDg7JcbZY17TbeQy13PUT16lDcQpXD8HMlrRiwJCdRAQfx6vEfQYI6fOrcqXhcLM4xJQeXQNcw5oqXT5Dacj3BzieBMI-5ECFW5fp2gsmdDsculnHr-Niw01NpJ1v9AvcAkewM9VvVYq9ibKF~OCqZfHIvg52g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463661&amp;ar=2738680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw053f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the nucleotide sequences of the LCA-EcoRIa, LCA-EcoRIb, and LCA-ApaI sequences with homologous sequences. <strong>(A)</strong> Alignment of the LCA-EcoRIa consensus sequence with two partial sequences of the tandem repeat sequence rpt200 (rpt200_1 at positions 5,133–5,283 and rpt200_2 at positions 7,489–7,556 of the <em>Germ1</em> sequence (GQ215662) of <em>P. marinus</em>. <strong>(B)</strong> Alignment of the LCA-EcoRIb consensus sequence with the EcoRI satellite DNA sequence families from <em>La. planeri</em> (LPL),<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>La. zanandreai</em> (LZA),<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and <em>P. marinus</em> (PMA; X92515).<a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The 6-bp motif (A/T)(G/C)AAA(T/C) and its complementary sequence, which are homologous to centromeric motifs observed in other vertebrates,<a reveal-id="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" data-open="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" class="link link-ref link-reveal xref-bibr"><sup>28–31</sup></a> are shown in squares on the consensus sequence of LCA-EcoRIb. <strong>(C)</strong> Alignment of the LCA-ApaI consensus sequence with a partial sequence of the <em>P. marinus</em> Ty3/Gypsy family of LTR retrotransposons (Gypsy-26_PM-I at positions 4,252–4,954). Dots indicate identity with the LCA-EcoRIa, LCA-EcoRIb, or LCA-ApaI consensus sequence. The numbers at the end of lines are the sequence lengths.</p></div></div><div data-id="dsw053-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f3.png?Expires=1509239091&amp;Signature=bvsjuBiFam-TYs~ThUtDNB4C~4WQ0URnvOZ81DACu1E6YFMAPWTnogluM~P9ZRLOZjCMK1eLRBfEs6Wg6gYapsT86dmiGEIjnrA01cmu19CXJLCMMjbgkEUncxWHsFx7NHpkA95TgFwq~wdB6gtMvz1gNduxfPfpeprH~~L2MHwFhbYO9YX2GH9VNnHQVq5x6Iqr-dAjEqfiw9z6K1hHUDN26x0XGTjNBT5gmnYSd0lFSp4Rt5GLE-j5VZ6hPoTmNhj5cS7pCOxBZVAewieBf608DS7Ns9AalUniSDnwlvEdb1a7ijYIEUSL8SC~Z7Jd6LCTX57Woyh~WuzL0n5DkA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic representation of the Ty3/Gypsy family of LTR retrotransposons based on the Gypsy-26_PM-I sequence of P. marinus. Gypsy-26_PM-I contains sequences encoding a reverse transcriptase (RT), retroviral aspartyl protease (RVP), ribonuclease H (RH), and integrase (IN). The region at positions 4,252–4,954, which is homologous to the LCA-ApaI sequence, is shown with a bold underline. Bold lines above the ORF indicate the conserved domains." path-from-xml="dsw053f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463661/dsw053f3.tif" path-from-xml="dsw053f3.tif" target="_blank">View large</a><a section="61463661" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f3.png?Expires=1509239091&amp;Signature=LfyB0za5n70-wDLFWjFNmH3KCSTyecrsmz9cQcjCCei-FG1-vILA3r0u3xChwB7cYab87qMpxG3C-DdvO6sBnGG6A6rbdoR0sAvrliCbJZmDYNdgyQkMthYuBjHGDZZ1ZDIgP0fHMdzRvBJr0pYH3A74JpbEb0wCuJJoBWKGGWRfVwSYKX0tc~KldU9O5qdbVuani9jfRMhNgELZxAQcKKztvVNK8-DGXvRfZVnofjsiuSCMErD3wG6wy3GOushoO-kd4dUNsptuHgo1GUJWppyL2Lx6sltCr23Kx-oQR1RGi~XfkxOuxfEk2mbKzXGEhDq-KpXbv75T77lCBV25ZQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463661&amp;ar=2738680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw053f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the Ty3/Gypsy family of LTR retrotransposons based on the Gypsy-26_PM-I sequence of <em>P. marinus</em>. Gypsy-26_PM-I contains sequences encoding a reverse transcriptase (RT), retroviral aspartyl protease (RVP), ribonuclease H (RH), and integrase (IN). The region at positions 4,252–4,954, which is homologous to the LCA-ApaI sequence, is shown with a bold underline. Bold lines above the ORF indicate the conserved domains.</p></div></div><div content-id="dsw053-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f3.png?Expires=1509239091&amp;Signature=bvsjuBiFam-TYs~ThUtDNB4C~4WQ0URnvOZ81DACu1E6YFMAPWTnogluM~P9ZRLOZjCMK1eLRBfEs6Wg6gYapsT86dmiGEIjnrA01cmu19CXJLCMMjbgkEUncxWHsFx7NHpkA95TgFwq~wdB6gtMvz1gNduxfPfpeprH~~L2MHwFhbYO9YX2GH9VNnHQVq5x6Iqr-dAjEqfiw9z6K1hHUDN26x0XGTjNBT5gmnYSd0lFSp4Rt5GLE-j5VZ6hPoTmNhj5cS7pCOxBZVAewieBf608DS7Ns9AalUniSDnwlvEdb1a7ijYIEUSL8SC~Z7Jd6LCTX57Woyh~WuzL0n5DkA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic representation of the Ty3/Gypsy family of LTR retrotransposons based on the Gypsy-26_PM-I sequence of P. marinus. Gypsy-26_PM-I contains sequences encoding a reverse transcriptase (RT), retroviral aspartyl protease (RVP), ribonuclease H (RH), and integrase (IN). The region at positions 4,252–4,954, which is homologous to the LCA-ApaI sequence, is shown with a bold underline. Bold lines above the ORF indicate the conserved domains." path-from-xml="dsw053f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463661/dsw053f3.tif" path-from-xml="dsw053f3.tif" target="_blank">View large</a><a section="61463661" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f3.png?Expires=1509239091&amp;Signature=LfyB0za5n70-wDLFWjFNmH3KCSTyecrsmz9cQcjCCei-FG1-vILA3r0u3xChwB7cYab87qMpxG3C-DdvO6sBnGG6A6rbdoR0sAvrliCbJZmDYNdgyQkMthYuBjHGDZZ1ZDIgP0fHMdzRvBJr0pYH3A74JpbEb0wCuJJoBWKGGWRfVwSYKX0tc~KldU9O5qdbVuani9jfRMhNgELZxAQcKKztvVNK8-DGXvRfZVnofjsiuSCMErD3wG6wy3GOushoO-kd4dUNsptuHgo1GUJWppyL2Lx6sltCr23Kx-oQR1RGi~XfkxOuxfEk2mbKzXGEhDq-KpXbv75T77lCBV25ZQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=61463661&amp;ar=2738680&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw053f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the Ty3/Gypsy family of LTR retrotransposons based on the Gypsy-26_PM-I sequence of <em>P. marinus</em>. Gypsy-26_PM-I contains sequences encoding a reverse transcriptase (RT), retroviral aspartyl protease (RVP), ribonuclease H (RH), and integrase (IN). The region at positions 4,252–4,954, which is homologous to the LCA-ApaI sequence, is shown with a bold underline. Bold lines above the ORF indicate the conserved domains.</p></div></div></p>
                            <h3 scrollto-destination=61463662 id="61463662" class="section-title" >3.4. Chromosomal distribution of the repetitive sequences</h3>
<p>LCA-EcoRIa 2 (LC149792), LCA-EcoRIb 4 (LC149803), and LCA-ApaI 1 (LC149811) fragments were used for FISH analysis as representative probes of each repetitive sequence family. The results showed that the LCA-EcoRIa sequences were localized to the pericentromeric regions of six chromosomes (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), and the hybridization signals completely overlapped the signals for the 18S–28S rRNA genes (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4B and C</a>). Hybridization signals for the LCA-EcoRIb sequence were observed in the centromeric regions of six chromosomes (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>), which differed from the chromosomes bearing the 18S–28S rRNA genes (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4E and F</a>). The LCA-ApaI sequence was localized to the centromeric and/or pericentromeric regions of almost all chromosomes. C-band positive bands were also located in the centromeric regions of many small acrocentric and dot-shaped chromosomes (<a reveal-id="dsw053-F1" data-open="dsw053-F1" class="link link-reveal link-table xref-fig">Fig. 1B and C</a>), indicating that the C-positive heterochromatin regions overlapped or partially overlapped the signals of LCA-ApaI sequence. However, the size of signals differed among the chromosomes (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4G and H</a>). The signals were generally located in the centromeric regions on PI-positive blocks and/or in pericentromeric regions. Approximately 10 chromosomes showed large signals, and several large submetacentric chromosomes had more than 1 hybridization site. Two-color FISH analysis of LCA-ApaI and the 18S–28S rRNA genes showed that the LCA-ApaI sequences were located adjacent to the 18S–28S rRNA gene cluster but hardly overlapped it (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4I</a>). However, the LCA-ApaI signals partially overlapped the LCA-EcoRIb signals (<a reveal-id="dsw053-F4" data-open="dsw053-F4" class="link link-reveal link-table xref-fig">Fig. 4J</a>). <div data-id="dsw053-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f4.png?Expires=1509239091&Signature=IOYrm7ssZDRijxfZy82w9qlsrOSoIb8l2jJ1Tj2yGqcYcn-oSMD5QcIqNH2d64J0FByVt1XCHviYhxjWFB9NvV81vcfJv~MXqdyJrhHc38HDWWLPcoDDPY04L~OUgDQdwPxFnYOIc17PmYlWsUDFj07R~aygVvgXhV2DQQmrTgSnlPW0y5m9CRe5uEVW7sMTbR7kwguj0zUujs73vtyidH7xnxRpsMMNGd0Mr3nzdndxsu~idNEUpeCqSzNGJjdnTSuNQi1x3HH5q0gGmpahBzkeW1DYP3bKAvy6cNRGDEY9b0QdbwDh2f0UCq6Wj75D9AhpIXFpf1efuncf1bBA5g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Chromosomal distribution of the LCA-EcoRI and LCA-ApaI sequences. (A) DAPI-stained metaphase spread hybridized with biotin-labeled LCA-EcoRIa (green). (B) The same metaphase spread in (A) hybridized with a DIG-labeled 18S–28S rRNA gene probe (red). (C) Merged image of the signals for biotin-labeled LCA-EcoRIa and DIG-labeled 18S–28S rDNA. (D–F) Hybridization patterns of biotin-labeled LCA-EcoRIb (green) (D) and DIG-labeled 18S–28S rDNA (red) (E) on the same DAPI-stained metaphase spread and their merge (F). (G, H) Hybridization patterns of biotin-labeled LCA-ApaI (green) on a PI-stained metaphase spread (G) and the PI-stained metaphase spread (H). Enlarged photographs of chromosomes hybridized to the LCA-ApaI probe in two parts of the metaphase spread are shown. Squares with white outlines on the metaphase spreads indicate the magnified areas. (I) Comparison of the hybridization patterns of biotin-labeled 18S–28S rDNA (green) and DIG-labeled LCA-ApaI (red) on six DAPI-stained chromosomes. (J) Comparison of the chromosome distribution patterns of biotin-labeled LCA-EcoRIb (green) and DIG-labeled LCA-ApaI (red). In the merged images (I, J), the regions of overlap between the two probes are shown in yellow. Scale bars represent 10 µm. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes." path-from-xml="dsw053f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463663/dsw053f4.tif" path-from-xml="dsw053f4.tif" target="_blank">View large</a><a section="61463663" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f4.png?Expires=1509239091&Signature=IAee~a6oxmnpfvEfE4KS0VtinBMTOc-fDtqKMnUk0lrnqkDDtzMcxcQhZu~6tLRt7tP~3y7x4KrSi1LygzVW1TeOus-7pFsQ2tU9Ifh33XbHICtZuQPez-IKTUxAQg~1mqrmJd-2ysYNKMC2vdfAwF~JEAwsUWWQvbL4eYrFczUpY2vlzvYG6QWNOe-PT4xNvK8Kv-WyXCVktDWSGma-eyubJJ81UnhYJ7UmU-FczHHdPWMbeN9VicfSBKzausa13yyDX3pHKJ6pg4onrcAZ15v01oR7I~x8UpdlUpthdG5eRrKCLmwgCMjRbxTm0cIM3ugH4M2yV6Fnvqsgqu1DNA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463663&ar=2738680&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw053f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal distribution of the LCA-EcoRI and LCA-ApaI sequences. (A) DAPI-stained metaphase spread hybridized with biotin-labeled LCA-EcoRIa (green). (B) The same metaphase spread in (A) hybridized with a DIG-labeled 18S–28S rRNA gene probe (red). (C) Merged image of the signals for biotin-labeled LCA-EcoRIa and DIG-labeled 18S–28S rDNA. (D–F) Hybridization patterns of biotin-labeled LCA-EcoRIb (green) (D) and DIG-labeled 18S–28S rDNA (red) (E) on the same DAPI-stained metaphase spread and their merge (F). (G, H) Hybridization patterns of biotin-labeled LCA-ApaI (green) on a PI-stained metaphase spread (G) and the PI-stained metaphase spread (H). Enlarged photographs of chromosomes hybridized to the LCA-ApaI probe in two parts of the metaphase spread are shown. Squares with white outlines on the metaphase spreads indicate the magnified areas. (I) Comparison of the hybridization patterns of biotin-labeled 18S–28S rDNA (green) and DIG-labeled LCA-ApaI (red) on six DAPI-stained chromosomes. (J) Comparison of the chromosome distribution patterns of biotin-labeled LCA-EcoRIb (green) and DIG-labeled LCA-ApaI (red). In the merged images (I, J), the regions of overlap between the two probes are shown in yellow. Scale bars represent 10 µm. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes.</p></div></div><div content-id="dsw053-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f4.png?Expires=1509239091&Signature=IOYrm7ssZDRijxfZy82w9qlsrOSoIb8l2jJ1Tj2yGqcYcn-oSMD5QcIqNH2d64J0FByVt1XCHviYhxjWFB9NvV81vcfJv~MXqdyJrhHc38HDWWLPcoDDPY04L~OUgDQdwPxFnYOIc17PmYlWsUDFj07R~aygVvgXhV2DQQmrTgSnlPW0y5m9CRe5uEVW7sMTbR7kwguj0zUujs73vtyidH7xnxRpsMMNGd0Mr3nzdndxsu~idNEUpeCqSzNGJjdnTSuNQi1x3HH5q0gGmpahBzkeW1DYP3bKAvy6cNRGDEY9b0QdbwDh2f0UCq6Wj75D9AhpIXFpf1efuncf1bBA5g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Chromosomal distribution of the LCA-EcoRI and LCA-ApaI sequences. (A) DAPI-stained metaphase spread hybridized with biotin-labeled LCA-EcoRIa (green). (B) The same metaphase spread in (A) hybridized with a DIG-labeled 18S–28S rRNA gene probe (red). (C) Merged image of the signals for biotin-labeled LCA-EcoRIa and DIG-labeled 18S–28S rDNA. (D–F) Hybridization patterns of biotin-labeled LCA-EcoRIb (green) (D) and DIG-labeled 18S–28S rDNA (red) (E) on the same DAPI-stained metaphase spread and their merge (F). (G, H) Hybridization patterns of biotin-labeled LCA-ApaI (green) on a PI-stained metaphase spread (G) and the PI-stained metaphase spread (H). Enlarged photographs of chromosomes hybridized to the LCA-ApaI probe in two parts of the metaphase spread are shown. Squares with white outlines on the metaphase spreads indicate the magnified areas. (I) Comparison of the hybridization patterns of biotin-labeled 18S–28S rDNA (green) and DIG-labeled LCA-ApaI (red) on six DAPI-stained chromosomes. (J) Comparison of the chromosome distribution patterns of biotin-labeled LCA-EcoRIb (green) and DIG-labeled LCA-ApaI (red). In the merged images (I, J), the regions of overlap between the two probes are shown in yellow. Scale bars represent 10 µm. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes." path-from-xml="dsw053f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463663/dsw053f4.tif" path-from-xml="dsw053f4.tif" target="_blank">View large</a><a section="61463663" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f4.png?Expires=1509239091&Signature=IAee~a6oxmnpfvEfE4KS0VtinBMTOc-fDtqKMnUk0lrnqkDDtzMcxcQhZu~6tLRt7tP~3y7x4KrSi1LygzVW1TeOus-7pFsQ2tU9Ifh33XbHICtZuQPez-IKTUxAQg~1mqrmJd-2ysYNKMC2vdfAwF~JEAwsUWWQvbL4eYrFczUpY2vlzvYG6QWNOe-PT4xNvK8Kv-WyXCVktDWSGma-eyubJJ81UnhYJ7UmU-FczHHdPWMbeN9VicfSBKzausa13yyDX3pHKJ6pg4onrcAZ15v01oR7I~x8UpdlUpthdG5eRrKCLmwgCMjRbxTm0cIM3ugH4M2yV6Fnvqsgqu1DNA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463663&ar=2738680&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw053f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal distribution of the LCA-EcoRI and LCA-ApaI sequences. (A) DAPI-stained metaphase spread hybridized with biotin-labeled LCA-EcoRIa (green). (B) The same metaphase spread in (A) hybridized with a DIG-labeled 18S–28S rRNA gene probe (red). (C) Merged image of the signals for biotin-labeled LCA-EcoRIa and DIG-labeled 18S–28S rDNA. (D–F) Hybridization patterns of biotin-labeled LCA-EcoRIb (green) (D) and DIG-labeled 18S–28S rDNA (red) (E) on the same DAPI-stained metaphase spread and their merge (F). (G, H) Hybridization patterns of biotin-labeled LCA-ApaI (green) on a PI-stained metaphase spread (G) and the PI-stained metaphase spread (H). Enlarged photographs of chromosomes hybridized to the LCA-ApaI probe in two parts of the metaphase spread are shown. Squares with white outlines on the metaphase spreads indicate the magnified areas. (I) Comparison of the hybridization patterns of biotin-labeled 18S–28S rDNA (green) and DIG-labeled LCA-ApaI (red) on six DAPI-stained chromosomes. (J) Comparison of the chromosome distribution patterns of biotin-labeled LCA-EcoRIb (green) and DIG-labeled LCA-ApaI (red). In the merged images (I, J), the regions of overlap between the two probes are shown in yellow. Scale bars represent 10 µm. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes.</p></div></div></p>                            <h3 scrollto-destination=61463664 id="61463664" class="section-title" >3.5. Molecular phylogeny of the LCA-EcoRIb and LCA-ApaI sequences</h3>
<p>We performed phylogenetic analyses to examine the evolutionary relationships between the LCA-EcoRIb sequence and the EcoRI satellite sequences of other lamprey species as well as the LCA-ApaI sequence and Ty3/Gypsy LTR retrotransposons of other species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S4</a></span>). No sequences homologous to LCA-EcoRIa were found other than the rpt200 sequence of <em>P. marinus</em>; therefore, we did not perform molecular phylogenetic analysis of this sequence. NJ tree analysis of the EcoRI satellite repeats of lampreys revealed that all fragments of LCA-EcoRIb formed a clade supported by a high bootstrap value (100%), which was distinct from the clade containing the other three lamprey species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S4A</a></span>). Phylogenetic analysis of the LCA-ApaI sequences was performed using sequences encoding the integrase domain of the LTR retrotransposons obtained from GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov" target="_blank">http://www.ncbi.nlm.nih.gov</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Table S2</a></span>), most of which were used in a previous phylogenetic study.<a reveal-id="dsw053-B32" data-open="dsw053-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> A close phylogenetic relationship between the LCA-ApaI sequences and Gypsy-26_PM-I of <em>P. marinus</em> was supported by a high bootstrap value (100%; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S4B</a></span>).</p>                            <h3 scrollto-destination=61463666 id="61463666" class="section-title" >3.6. Genomic organization of the repetitive sequences</h3>
<p>FISH data revealed that LCA-EcoRIa is located in the spacer region of the 18S–28S rRNA gene cluster and tandem duplicated with the 18S–28S rDNA; therefore, we did not study the genomic organization of this repeated sequence. To examine the organization of the LCA-EcoRIb and LCA-ApaI sequences in the <em>Le</em>. <em>camtschaticum</em> genome, we performed Southern blot hybridizations using genomic DNA digested with AluI, ApaI, EcoRI, HpaII, MspI, and PstI (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5</a></span>). In a blot probed with LCA-EcoRIb, polymeric ladder bands of the 364-bp monomer units were detected in the EcoRI and AluI digests, which are characteristic of tandem repeat satellite DNA sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5A</a></span>). Additional hybridization bands were observed interstitially in the ladder bands of the 364-bp monomer units in the AluI digests, which were derived from internal AluI cleavage sites in the monomer unit (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5A</a></span>). In contrast to the EcoRI and AluI digests, intense hybridization signals were observed at higher molecular weights in the ApaI, PstI, and MspI (HpaII) digests, indicating that these restriction sites are not highly conserved. Genomic DNA digested with the methylation-sensitive HpaII and methylation-insensitive MspI enzymes produced the same hybridization patterns, which indicates that their cleavage sites are not methylated.</p><p>Hybridization with LCA-ApaI also resulted in a ladder of signals consisting of multiples of the 702-bp monomer in the ApaI digest (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. S5</a></span>B). The intensity of ladder bands increased for higher-order repeating structures, indicating that the ApaI site was less conserved in the tandem arrays. In contrast, AluI and MspI sites in the tandem arrays were highly conserved because no multimeric bands were detected in the AluI and MspI digests. The hybridization bands at lower molecular weight than the 702-bp band in the AluI and MspI digests were derived from the internal cleavage sites contained in the monomer unit (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Fig. 3C</a></span>). The interstitial band positioned between the monomeric and dimeric bands in the ApaI digest suggested that the ApaI site was highly conserved in another additional unit of the LCA-ApaI sequence. The hybridization bands smaller than the monomer unit were observed in the MspI digest, whereas higher molecular weight bands were observed in the HpaII digest blot, indicating that the LCA-ApaI sequence was highly methylated.</p>                            <h3 scrollto-destination=61463669 id="61463669" class="section-title" >3.7. Sequence conservation of the repetitive sequences</h3>
<p>Conservation of the LCA-EcoRIb and LCA-ApaI sequences was examined by slot-blot hybridization using genomic DNA from four Petromyzontiformes species [<em>Le. camtschaticum</em> and <em>P. marinus</em> (Petromyzontidae), <em>Geotria australis</em> (Geotriidae), and <em>Mordacia mordax</em> (Mordaciidae)], inshore hagfish (<em>Eptatretus burgeri</em>, Myxinidae, Myxiniformes), lesser spotted catshark (<em>Scyliorhinus canicula</em>, Scyliorhinidae, Carcharhiniformes), and medaka (<em>Oryzias latipes</em>, Adrianichthyidae, Beloniformes) (<a reveal-id="dsw053-F5" data-open="dsw053-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Using the LCA-EcoRIb probe, intense chemiluminescent hybridization signals were detected in <em>P. marinus</em>, which is in the same family (Petromyzontidae) as <em>Le. camtschaticum</em>. No signals were detected in the other species (<a reveal-id="dsw053-F5" data-open="dsw053-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). Using the LCA-ApaI probe, an intense hybridization signal was detected in <em>Le. camtschaticum</em>, and less intense signals were observed in <em>P. marinus</em> and <em>M. mordax</em> (<a reveal-id="dsw053-F5" data-open="dsw053-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). <div data-id="dsw053-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f5.png?Expires=1509239091&Signature=bhteshR-Njpv6Bul6PTQMWZhPyNP453eMdqcjDORN7n7gnQ6cNBJyl~gw63PZAm6NepE6mVqpYxdvh7z8Ney0q6nLco81g4yHn1~z53mns6nRioGmA2V9XBesbCPO9Sjmc0ljyDuZFBuVBDzT841JvX-Jc9Skty~CiPBZGXp9UyQv-mzkQjbHjeBAoNKHdwYivobjWk~N2GtTRz6kFIYf4HFR3lAimStRp9WiuI1p-NNAPG91v~ZDCXYOLfsrUstQOyCZfX~6-tvTZbJWOq4m4-zwwKICbfvOfj7n7qvBQB5zSeMB4enRKlZNccugIJcdmHP5VJutEXphfH0mbkoBw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Slot-blot hybridization of the LCA-EcoRIb (A) and LCA-ApaI (B) sequences in four lamprey species (Lethenteron camtschaticum, Petromyzon marinus, Geotria australis, and Mordacia mordax), the inshore hagfish (Eptatretus burgeri), the lesser spotted catshark (Scyliorhinus canicula), and the medaka (Oryzias latipes). (C) A schematic representation of the blotted genomic DNAs is shown. Each lane contained 100 ng of genomic DNA. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes." path-from-xml="dsw053f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463670/dsw053f5.tif" path-from-xml="dsw053f5.tif" target="_blank">View large</a><a section="61463670" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f5.png?Expires=1509239091&Signature=AWwZK8v4ejZzrOcyDcMCyOjNS9LSYz0J~CYhQCTaqRMhUCUY-QbcsTxGHnPIjXi7QEL459R1T0A34t~QZ8RsN6ii5V~mwq9VOmATNCfAdQ-NY3KFaZ~97Vqj6HH5RM0OLfMhbNx-ytjkmTCvYVZPohlNfWNJ0sB6KMZzMdBOprGHRT2LHqDAp9eKcn9gMWhnuYg1flG1dElWLiaSnAVwfTHJbnqpZDj7~rOdPbFoV3EN64rn2PzKfb8k40T-OkMw082iB8UlpHvqZahxnJm09FG6xprRlvTABRQS-WPUtSa8Kgx5V04Ltjh4fCdO5V4BByK1SqkqxH6AkphJCa4hOg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463670&ar=2738680&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw053f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Slot-blot hybridization of the LCA-EcoRIb (A) and LCA-ApaI (B) sequences in four lamprey species (<em>Lethenteron camtschaticum</em>, <em>Petromyzon marinus</em>, <em>Geotria australis</em>, and <em>Mordacia mordax</em>), the inshore hagfish (<em>Eptatretus burgeri</em>), the lesser spotted catshark (<em>Scyliorhinus canicula</em>), and the medaka (<em>Oryzias latipes</em>). (C) A schematic representation of the blotted genomic DNAs is shown. Each lane contained 100 ng of genomic DNA. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes.</p></div></div><div content-id="dsw053-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/m_dsw053f5.png?Expires=1509239091&Signature=bhteshR-Njpv6Bul6PTQMWZhPyNP453eMdqcjDORN7n7gnQ6cNBJyl~gw63PZAm6NepE6mVqpYxdvh7z8Ney0q6nLco81g4yHn1~z53mns6nRioGmA2V9XBesbCPO9Sjmc0ljyDuZFBuVBDzT841JvX-Jc9Skty~CiPBZGXp9UyQv-mzkQjbHjeBAoNKHdwYivobjWk~N2GtTRz6kFIYf4HFR3lAimStRp9WiuI1p-NNAPG91v~ZDCXYOLfsrUstQOyCZfX~6-tvTZbJWOq4m4-zwwKICbfvOfj7n7qvBQB5zSeMB4enRKlZNccugIJcdmHP5VJutEXphfH0mbkoBw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Slot-blot hybridization of the LCA-EcoRIb (A) and LCA-ApaI (B) sequences in four lamprey species (Lethenteron camtschaticum, Petromyzon marinus, Geotria australis, and Mordacia mordax), the inshore hagfish (Eptatretus burgeri), the lesser spotted catshark (Scyliorhinus canicula), and the medaka (Oryzias latipes). (C) A schematic representation of the blotted genomic DNAs is shown. Each lane contained 100 ng of genomic DNA. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes." path-from-xml="dsw053f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463670/dsw053f5.tif" path-from-xml="dsw053f5.tif" target="_blank">View large</a><a section="61463670" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053f5.png?Expires=1509239091&Signature=AWwZK8v4ejZzrOcyDcMCyOjNS9LSYz0J~CYhQCTaqRMhUCUY-QbcsTxGHnPIjXi7QEL459R1T0A34t~QZ8RsN6ii5V~mwq9VOmATNCfAdQ-NY3KFaZ~97Vqj6HH5RM0OLfMhbNx-ytjkmTCvYVZPohlNfWNJ0sB6KMZzMdBOprGHRT2LHqDAp9eKcn9gMWhnuYg1flG1dElWLiaSnAVwfTHJbnqpZDj7~rOdPbFoV3EN64rn2PzKfb8k40T-OkMw082iB8UlpHvqZahxnJm09FG6xprRlvTABRQS-WPUtSa8Kgx5V04Ltjh4fCdO5V4BByK1SqkqxH6AkphJCa4hOg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=61463670&ar=2738680&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw053f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Slot-blot hybridization of the LCA-EcoRIb (A) and LCA-ApaI (B) sequences in four lamprey species (<em>Lethenteron camtschaticum</em>, <em>Petromyzon marinus</em>, <em>Geotria australis</em>, and <em>Mordacia mordax</em>), the inshore hagfish (<em>Eptatretus burgeri</em>), the lesser spotted catshark (<em>Scyliorhinus canicula</em>), and the medaka (<em>Oryzias latipes</em>). (C) A schematic representation of the blotted genomic DNAs is shown. Each lane contained 100 ng of genomic DNA. LCA-EcoRIb 4 (LC149803) and LCA-ApaI 1 (LC149811) fragments were used as probes.</p></div></div></p>                            <h2 scrollto-destination=61463671 id="61463671" class="section-title" >4. Discussion</h2>
<p>Here, we determined the chromosome number and C-banded karyotype of <em>Le. camtschaticum</em> in somatic cells obtained from lymphocyte cultures. The diploid chromosome number was 168, which was not consistent with that reported in a previous study (2<em>n</em> = 144–162).<a reveal-id="dsw053-B19" data-open="dsw053-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> This discrepancy may be due to technical artifacts owing to the extremely small size and high number of tiny chromosomes in lampreys. The chromosome number of <em>Le. camtschaticum</em> determined in this study was similar to those reported in somatic cells of <em>Lampetra</em> (<em>La. fluviatilis</em> [2<em>n</em> = 164], <em>La. lamottenii</em> [2<em>n</em> = 166], <em>La. planeri</em> [2<em>n</em> = 164], <em>La. zanandreai</em> [2<em>n</em> = 164]),<a reveal-id="dsw053-B8" data-open="dsw053-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and <em>Petromyzon marinus</em> (2<em>n</em> = 164–168).<a reveal-id="dsw053-B7" data-open="dsw053-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw053-B10" data-open="dsw053-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> C-positive bands were distributed in the centromeric regions of ∼30 pairs of chromosomes, which was more than the 20 pairs reported in <em>La. zanandreai</em>.<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In this study, the 18S–28S rRNA genes were localized to six chromosomes; however, the gene copies were only active in a single pair of submetacentric chromosomes. In other species, the 18S–28S rRNA genes were localized to the pericentromeric regions of four and two chromosomes in somatic cells of <em>La. zanandreai</em><a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and <em>P marinus</em>,<a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> respectively. These results suggest that the chromosomal location of the 18S–28S rRNA genes in lampreys is variable. However, the chromosomal location of the 5S rRNA genes has not been reported for any other lampreys than <em>Le. camtschaticum</em>, in which, these genes are localized to two chromosomes. The 5S rRNA genes are generally localized to a single pair of chromosomes in fishes, amphibians, and mammals,<a reveal-id="dsw053-B33" data-open="dsw053-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> suggesting that chromosomal distribution of the 5S rRNA genes is conserved in vertebrates.</p><p>We isolated three families of repetitive sequences from <em>Le. camtschaticum</em>, two AT-rich sequences (LCA-EcoRIa and LCA-EcoRIb) and one GC-rich sequence (LCA-ApaI). The LCA-EcoRIa sequence, which co-localized with the 18S–28S rDNA, showed significant homology with the tandem rpt200 repeat of <em>P. marinus</em>. This sequence is located in the spacer regions between the intact 18S rDNA and fragmented 28S rDNA sequences in <em>P. marinus</em>, which are found in the <em>Germ1</em> sequence (GQ215662) that is eliminated from germ cells by a programmed genome rearrangement.<a reveal-id="dsw053-B27" data-open="dsw053-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> In the primary spermatocytes of <em>P. marinus</em> (2<em>n</em> = 198) at meiotic prophase I, FISH signals for rpt200 and 28S rDNA were distributed on about 20/99 and 10/99 chromosome pairs, respectively, and the signals co-localized on some chromosomes.<a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> However, in somatic cells of <em>P. marinus</em> (2<em>n</em> = 164–168), the rpt200 signals were localized to ∼14 pairs of chromosomes, and 28S rDNA signals were observed on only one pair of chromosomes, to which the rpt200 signal co-localized.<a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> This result indicates that most 18S–28S rDNA copies and several rpt200 copies are eliminated in somatic cells.<a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw053-B27" data-open="dsw053-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The germline DNA loss in somatic cells has also been reported in the Korean lamprey (<em>Lampetra morii</em>, Petromyzontidae) recently.<a reveal-id="dsw053-B34" data-open="dsw053-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Therefore, the presence and co-localization of the rpt200-related LCA-EcoRIa sequence and 18S–28S rDNA in a small number of chromosome pairs in somatic cells of <em>Le. camtschaticum</em> might be related to a programmed genome rearrangement in this species.</p><p>The LCA-EcoRIb sequence is a homolog of an EcoRI satellite DNA sequence that has been isolated from three lamprey species (<em>La. planeri</em>, <em>La. zanandreai</em>, and <em>P. marinus</em>).<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw053-B18" data-open="dsw053-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> These AT-rich EcoRI sequence families, including LCA-EcoRIb, range from 362 to 370 bp, and the sequence divergence between LCA-EcoRIb and the EcoRI satellite DNAs is relatively low (10.6–15.2%). These sequences contain short repeats, such as centromeric motifs, which are also observed in the centromeric sequences of several eukaryotic species such as bream, trout, mouse, and human,<a reveal-id="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" data-open="dsw053-B28 dsw053-B29 dsw053-B30 dsw053-B31" class="link link-ref link-reveal xref-bibr"><sup>28–31</sup></a> suggesting that these satellite DNAs and the LCA-EcoRIb sequence could be important for centromere structure and function in lampreys<em>.</em> A zoo-blot hybridization revealed that the EcoRIb sequence is conserved within Petromyzontidae, and it is distributed on six chromosomes in the somatic cells of <em>Le. camtschaticum</em>, on two chromosomes in <em>La. zanandreai</em>,<a reveal-id="dsw053-B12" data-open="dsw053-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and on 34 chromosomes in <em>P. marinus.</em><a reveal-id="dsw053-B14" data-open="dsw053-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> These results suggest that the EcoRI satellite sequence might have been acquired in the Petromyzontidae lineage and amplified on many more chromosomes in <em>Petromyzon</em> than in <em>Lampetra</em> and <em>Lethenteron</em> through homogenization between non-homologous chromosomes, called concerted evolution (reviewed in Elder and Turner<a reveal-id="dsw053-B35" data-open="dsw053-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and Plohl <em>et al.</em><a reveal-id="dsw053-B36" data-open="dsw053-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a>).</p><p>A search of RepBase revealed that the LCA-ApaI sequence showed significant homology with a partial integrase-coding region of the Ty3/Gypsy retrotransposon of <em>P. marinus</em> (Gypsy-26_PM-I), indicating that the LCA-ApaI sequence might be a LTR retrotransposon-derived repeat. LTR retrotransposons of the Ty3/Gypsy superfamily have been isolated from fungi, plants, and animals.<a reveal-id="dsw053-B37 dsw053-B38 dsw053-B39" data-open="dsw053-B37 dsw053-B38 dsw053-B39" class="link link-ref link-reveal xref-bibr"><sup>37–39</sup></a> The results of the slot-blot analysis indicate that the LCA-ApaI sequence is well conserved in Petromyzontidae and Mordaciidae, but not in Geotriidae. Mordaciidae, which is distributed in the southern hemisphere, has a lower chromosome number (2<em>n</em> = 76) than Petromyzontidae (2<em>n</em> = 142–174) in the northern hemisphere and Geotriidae (∼180 chromosomes) in the southern hemisphere, suggesting that extensive centric fusions occurred in Mordaciidae.<a reveal-id="dsw053-B9" data-open="dsw053-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw053-B13" data-open="dsw053-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Molecular phylogenetic analyses suggested that Mordaciidae is in a separate clade from Petromyzontidae and Geotriidae,<a reveal-id="dsw053-B40 dsw053-B41 dsw053-B42 dsw053-B43" data-open="dsw053-B40 dsw053-B41 dsw053-B42 dsw053-B43" class="link link-ref link-reveal xref-bibr"><sup>40–43</sup></a> which is reflected by the difference in chromosome numbers between Petromyzontidae/Geotriidae and Mordaciidae. The phylogenetic relationship among these three families strongly suggests that the LCA-ApaI sequence appeared in the common ancestor of Petromyzontiformes and amplified in Petromyzontidae and Mordaciidae but not in Geotriidae, in which no hybridization signal was detected by slot blot, although it is also possible that the LCA-ApaI sequence disappeared in the Geotriidae lineage.</p><p>A draft genome assembly of <em>P. marinus</em> has been reported, and 7,752 distinct families of repetitive sequences were identified, accounting for 34.7% of the genome.<a reveal-id="dsw053-B44" data-open="dsw053-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The genome sequence of <em>P. marinus</em> will facilitate cloning of repetitive sequences from another lamprey species and their comparison, which is needed to provide further insights into the molecular basis of chromosome structure and evolution in lampreys.</p>                            <h2 scrollto-destination=61463677 id="61463677" class="section-title" >Accession numbers</h2>
<p>LC149791–LC149817</p>                            <h2 scrollto-destination=61463679 id="61463679" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=61463681 id="61463681" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=61463683 id="61463683" class="section-title" >Funding</h2>
<p>This work was supported by a Grant-in-Aid for Scientific Research on Innovative Areas (No. 23113004) and a Grant-in-Aid for Scientific Research (B) (No. 22370081) from the Ministry of Education, Culture, Sports, Science and Technology of Japan. Our gratitude extends to suppliers of genomic DNA or tissues of cyclostome and chondrichthyan species, namely, H. Kawauchi, R. Kusakabe, S. Moriyama, M. Nozaki, K.G. Ota, J. Joss, S. Kuratani, and A. Takahashi.</p>                            <h2 scrollto-destination=61463685 id="61463685" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw053-b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stock</div> <div class="given-names">D. W.</div></div><div class="name string-name"><div class="surname">Whitt</div> <div class="given-names">G. S.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Evidence from 18S ribosomal RNA sequences that lampreys and hagfishes form a natural group</div>. <div class="source">Science</div> , <div class="volume">257</div>, <div class="fpage">787</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20from%2018S%20ribosomal%20RNA%20sequences%20that%20lampreys%20and%20hagfishes%20form%20a%20natural%20group&amp;author=D.%20W.%20Stock&amp;author=G.%20S.%20Whitt&amp;publication_year=1992&amp;journal=Science&amp;volume=257&amp;pages=787-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1496398" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1496398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1496398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1496398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Kuratani</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Time scale for cyclostome evolution inferred with a phylogenetic diagnosis of hagfish and lamprey cDNA sequences</div>. <div class="source">Zool. Sci</div> ., <div class="volume">23</div>, <div class="fpage">1053</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Time%20scale%20for%20cyclostome%20evolution%20inferred%20with%20a%20phylogenetic%20diagnosis%20of%20hagfish%20and%20lamprey%20cDNA%20sequences&amp;author=S.%20Kuraku&amp;author=S.%20Kuratani&amp;publication_year=2006&amp;journal=Zool.%20Sci&amp;volume=23&amp;pages=1053-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2108/zsj.23.1053" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2108%2Fzsj.23.1053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2108%2Fzsj.23.1053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17261918" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ota</div> <div class="given-names">K.G.</div></div><div class="name string-name"><div class="surname">Kuratani</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, Cyclostomata. In: <div class="person-group"><div class="name string-name"><div class="surname">Hedges</div> <div class="given-names">S.B.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> eds. <div class="source">Timetree of Life</div> .
<div class="publisher-loc">New York</div>: 
<div class="publisher-name">Oxford University Press</div>, pp. <div class="fpage">317</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsw053-b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">I.C.</div></div><div class="name string-name"><div class="surname">Gill</div> <div class="given-names">H.S.</div></div><div class="name string-name"><div class="surname">Renaud</div> <div class="given-names">C.B.</div></div></div> <div class="year">2014</div>, Petromyzontidae: lampreys. In: <div class="person-group"><div class="name string-name"><div class="surname">Burr</div> <div class="given-names">B.M.</div></div><div class="name string-name"><div class="surname">Warren</div> <div class="given-names">M.L.</div>Jr.</div></div>, eds.<div class="source">North American Freshwater Fishes: Natural History, Ecology, Behavior, and Conservation</div> . 
<div class="publisher-loc">Baltimore</div>: 
<div class="publisher-name">Johns Hopkins University Press</div>, pp. <div class="fpage">106</div>–<div class="lpage">39</div>.</div></div></div></div><div content-id="dsw053-b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">I.C.</div></div><div class="name string-name"><div class="surname">Strahan</div> <div class="given-names">R.</div></div></div> <div class="year">1968</div>, 
<div class="article-title">The taxonomy of the lampreys Geotria and Mordacia and their distribution in Australia</div>. <div class="source">Proc. Linn. Soc. Lond</div> ., <div class="volume">179</div>, <div class="fpage">229</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20taxonomy%20of%20the%20lampreys%20Geotria%20and%20Mordacia%20and%20their%20distribution%20in%20Australia&amp;author=I.C.%20Potter&amp;author=R.%20Strahan&amp;publication_year=1968&amp;journal=Proc.%20Linn.%20Soc.%20Lond&amp;volume=179&amp;pages=229-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1095-8312.1968.tb00980.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1095-8312.1968.tb00980.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1095-8312.1968.tb00980.x"> </span></div> </div></div></div></div></div><div content-id="dsw053-b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gill</div> <div class="given-names">H.S.</div></div><div class="name string-name"><div class="surname">Renaud</div> <div class="given-names">C.B.</div></div><div class="name string-name"><div class="surname">Chapleau</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Mayden</div> <div class="given-names">R.L.</div></div><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">I.C.</div></div><div class="name string-name"><div class="surname">Douglas</div> <div class="given-names">M.E.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Phylogeny of living parasitic lampreys (Petromyzontiformes) based on morphological data</div>. <div class="source">Copeia</div> , <div class="volume">2003</div>, <div class="fpage">687</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20living%20parasitic%20lampreys%20(Petromyzontiformes)%20based%20on%20morphological%20data&amp;author=H.S.%20Gill&amp;author=C.B.%20Renaud&amp;author=F.%20Chapleau&amp;author=R.L.%20Mayden&amp;author=I.C.%20Potter&amp;author=M.E.%20Douglas&amp;publication_year=2003&amp;journal=Copeia&amp;volume=2003&amp;pages=687-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1643/IA02-085.1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1643%2FIA02-085.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1643%2FIA02-085.1"> </span></div> </div></div></div></div></div><div content-id="dsw053-b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">I.C.</div></div><div class="name string-name"><div class="surname">Rothwell</div> <div class="given-names">B.</div></div></div> <div class="year">1970</div>, 
<div class="article-title">The mitotic chromosomes of the lamprey, <em>Petromyzon marinus</em> L</div>. <div class="source">Experientia</div> , <div class="volume">26</div>, <div class="fpage">429</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mitotic%20chromosomes%20of%20the%20lamprey%2C%20Petromyzon%20marinus%20L&amp;author=I.C.%20Potter&amp;author=B.%20Rothwell&amp;publication_year=1970&amp;journal=Experientia&amp;volume=26&amp;pages=429-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01896930" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01896930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01896930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5439626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">E.S.</div></div><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">I.C.</div></div><div class="name string-name"><div class="surname">Webb</div> <div class="given-names">C.J.</div></div></div> <div class="year">1974</div>, 
<div class="article-title">Homogeneity of holarctic lamprey karyotypes</div>. <div class="source">Caryologia</div> , <div class="volume">27</div>, <div class="fpage">443</div>–<div class="lpage">54</div>.</div></div></div></div><div content-id="dsw053-b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">E.S.</div></div><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">I.C.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">The chromosomes of the southern hemispheric lamprey, <em>Geotria australis</em> Gray</div>. <div class="source">Experientia</div> , <div class="volume">37</div>, <div class="fpage">239</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chromosomes%20of%20the%20southern%20hemispheric%20lamprey%2C%20Geotria%20australis%20Gray&amp;author=E.S.%20Robinson&amp;author=I.C.%20Potter&amp;publication_year=1981&amp;journal=Experientia&amp;volume=37&amp;pages=239-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01991630" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01991630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01991630"> </span></div> </div></div></div></div></div><div content-id="dsw053-b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">J.J.</div></div><div class="name string-name"><div class="surname">Stuart</div> <div class="given-names">A.B.</div></div><div class="name string-name"><div class="surname">Sauka-Spengler</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Clifton</div> <div class="given-names">S.W.</div></div><div class="name string-name"><div class="surname">Amemiya</div> <div class="given-names">C.T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Development and analysis of a germline BAC resource for the sea lamprey, a vertebrate that undergoes substantial chromatin diminution</div>. <div class="source">Chromosoma</div> , <div class="volume">119</div>, <div class="fpage">381</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20analysis%20of%20a%20germline%20BAC%20resource%20for%20the%20sea%20lamprey%2C%20a%20vertebrate%20that%20undergoes%20substantial%20chromatin%20diminution&amp;author=J.J.%20Smith&amp;author=A.B.%20Stuart&amp;author=T.%20Sauka-Spengler&amp;author=S.W.%20Clifton&amp;author=C.T.%20Amemiya&amp;publication_year=2010&amp;journal=Chromosoma&amp;volume=119&amp;pages=381-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00412-010-0263-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00412-010-0263-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00412-010-0263-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20195622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arai</div> <div class="given-names">R.</div></div></div> <div class="year">2011</div>, <div class="source">Fish Karyotypes: A Check List</div> . 
<div class="publisher-loc">Heidelberg</div>: 
<div class="publisher-name">Springer</div>.</div></div></div></div><div content-id="dsw053-b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Caputo</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Giovannotti</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Cerioni</div> <div class="given-names">P.N.</div></div><div class="name string-name"><div class="surname">Splendiani</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Tagliavini</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Olmo</div> <div class="given-names">E.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Chromosomal study of a lamprey (<em>Lampetra zanandreai</em> Vladykov, 1955) (Petromyzonida: Petromyzontiformes): conventional and FISH analysis</div>. <div class="source">Chromosome Res</div> ., <div class="volume">19</div>, <div class="fpage">481</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20study%20of%20a%20lamprey%20(Lampetra%20zanandreai%20Vladykov%2C%201955)%20(Petromyzonida%3A%20Petromyzontiformes)%3A%20conventional%20and%20FISH%20analysis&amp;author=V.%20Caputo&amp;author=M.%20Giovannotti&amp;author=P.N.%20Cerioni&amp;author=A.%20Splendiani&amp;author=J.%20Tagliavini&amp;author=E.%20Olmo&amp;publication_year=2011&amp;journal=Chromosome%20Res&amp;volume=19&amp;pages=481-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-011-9197-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-011-9197-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-011-9197-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21437736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Caputo Barucchi</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Giovannotti</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nisi Cerioni</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Splendiani</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genome duplication in early vertebrates: insights from agnathan cytogenetics</div>. <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">141</div>, <div class="fpage">80</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20duplication%20in%20early%20vertebrates%3A%20insights%20from%20agnathan%20cytogenetics&amp;author=V.%20Caputo%20Barucchi&amp;author=M.%20Giovannotti&amp;author=P.%20Nisi%20Cerioni&amp;author=A.%20Splendiani&amp;publication_year=2013&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=141&amp;pages=80-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000354098" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000354098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000354098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23949002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Covelo-Soto</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Morán</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Pasantes</div> <div class="given-names">J.J.</div></div><div class="name string-name"><div class="surname">Pérez-García</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Cytogenetic evidences of genome rearrangement and differential epigenetic chromatin modification in the sea lamprey (<em>Petromyzon marinus</em>)</div>. <div class="source">Genetica</div> , <div class="volume">142</div>, <div class="fpage">545</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytogenetic%20evidences%20of%20genome%20rearrangement%20and%20differential%20epigenetic%20chromatin%20modification%20in%20the%20sea%20lamprey%20(Petromyzon%20marinus)&amp;author=L.%20Covelo-Soto&amp;author=P.%20Mor%C3%A1n&amp;author=J.J.%20Pasantes&amp;author=C.%20P%C3%A9rez-Garc%C3%ADa&amp;publication_year=2014&amp;journal=Genetica&amp;volume=142&amp;pages=545-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-014-9802-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-014-9802-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-014-9802-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25432678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singer</div> <div class="given-names">M.F.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">Highly repeated sequences in mammalian genomes</div>. <div class="source">Int. Rev. Cytol</div> ., <div class="volume">76</div>, <div class="fpage">67</div>–<div class="lpage">112</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20repeated%20sequences%20in%20mammalian%20genomes&amp;author=M.F.%20Singer&amp;publication_year=1982&amp;journal=Int.%20Rev.%20Cytol&amp;volume=76&amp;pages=67-112" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6749748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murphy</div> <div class="given-names">T.D.</div></div><div class="name string-name"><div class="surname">Karpen</div> <div class="given-names">G.H.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Centromeres take flight: alpha satellite and the quest for the human centromere</div>. <div class="source">Cell</div> , <div class="volume">93</div>, <div class="fpage">317</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Centromeres%20take%20flight%3A%20alpha%20satellite%20and%20the%20quest%20for%20the%20human%20centromere&amp;author=T.D.%20Murphy&amp;author=G.H.%20Karpen&amp;publication_year=1998&amp;journal=Cell&amp;volume=93&amp;pages=317-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81158-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81158-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81158-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9590164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ahmad</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Malik</div> <div class="given-names">H.S.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">The centromere paradox: stable inheritance with rapidly evolving DNA</div>. <div class="source">Science</div> , <div class="volume">293</div>, <div class="fpage">1098</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20centromere%20paradox%3A%20stable%20inheritance%20with%20rapidly%20evolving%20DNA&amp;author=S.%20Henikoff&amp;author=K.%20Ahmad&amp;author=H.S.%20Malik&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=1098-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1062939" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1062939" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1062939"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11498581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boán</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Viñas</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Rodríguez</div> <div class="given-names">J.M.</div></div><div class="name string-name"><div class="surname">Sánchez</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Gómez-Márquez</div> <div class="given-names">J.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">A new <em>Eco</em>RI family of satellite DNA in lampreys</div>. <div class="source">FEBS Lett</div> ., <div class="volume">394</div>, <div class="fpage">187</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20EcoRI%20family%20of%20satellite%20DNA%20in%20lampreys&amp;author=F.%20Bo%C3%A1n&amp;author=A.%20Vi%C3%B1as&amp;author=J.M.%20Rodr%C3%ADguez&amp;author=L.%20S%C3%A1nchez&amp;author=J.%20G%C3%B3mez-M%C3%A1rquez&amp;publication_year=1996&amp;journal=FEBS%20Lett&amp;volume=394&amp;pages=187-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(96)00947-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(96)00947-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(96)00947-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8843161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kitada</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Tagawa</div> <div class="given-names">M.</div></div></div> <div class="year">1975</div>, 
<div class="article-title">Somatic chromosomes of three species of Cyclostomata</div>. <div class="source">Chrom. Inform. Serv</div> ., <div class="volume">18</div>, <div class="fpage">10</div>–<div class="lpage">2</div>.</div></div></div></div><div content-id="dsw053-b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sumner</div> <div class="given-names">A.T.</div></div></div> <div class="year">1972</div>, 
<div class="article-title">A simple technique for demonstrating centromeric heterochromatin</div>. <div class="source">Exp. Cell Res</div> ., <div class="volume">75</div>, <div class="fpage">304</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20technique%20for%20demonstrating%20centromeric%20heterochromatin&amp;author=A.T.%20Sumner&amp;publication_year=1972&amp;journal=Exp.%20Cell%20Res&amp;volume=75&amp;pages=304-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-4827(72)90558-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-4827(72)90558-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-4827(72)90558-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4117921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Howell</div> <div class="given-names">W.M.</div></div><div class="name string-name"><div class="surname">Black</div> <div class="given-names">D.A.</div></div></div> <div class="year">1980</div>, 
<div class="article-title">Controlled silver-staining of nucleolus organizer regions with a protective colloidal developer: a 1-step method</div>. <div class="source">Experientia</div> , <div class="volume">36</div>, <div class="fpage">1014</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Controlled%20silver-staining%20of%20nucleolus%20organizer%20regions%20with%20a%20protective%20colloidal%20developer%3A%20a%201-step%20method&amp;author=W.M.%20Howell&amp;author=D.A.%20Black&amp;publication_year=1980&amp;journal=Experientia&amp;volume=36&amp;pages=1014-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01953855" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01953855" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01953855"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6160049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Uno</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Asada</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Nishida</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Takehana</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Sakaizumi</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Matsuda</div> <div class="given-names">Y.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Divergence of repetitive DNA sequences in the heterochromatin of medaka fishes: molecular cytogenetic characterization of constitutive heterochromatin in two medaka species: <em>Oryzias hubbsi</em> and <em>O. celebensis</em> (Adrianichthyidae, Beloniformes)</div>. <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">141</div>, <div class="fpage">212</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20of%20repetitive%20DNA%20sequences%20in%20the%20heterochromatin%20of%20medaka%20fishes%3A%20molecular%20cytogenetic%20characterization%20of%20constitutive%20heterochromatin%20in%20two%20medaka%20species%3A%20Oryzias%20hubbsi%20and%20O.%20celebensis%20(Adrianichthyidae%2C%20Beloniformes)&amp;author=Y.%20Uno&amp;author=Y.%20Asada&amp;author=C.%20Nishida&amp;author=Y.%20Takehana&amp;author=M.%20Sakaizumi&amp;author=Y.%20Matsuda&amp;publication_year=2013&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=141&amp;pages=212-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000354668" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000354668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000354668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24028862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsuda</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Chapman</div> <div class="given-names">V.M.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Application of fluorescence in situ hybridization in genome analysis of the mouse</div>. <div class="source">Electrophoresis</div> , <div class="volume">16</div>, <div class="fpage">261</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20fluorescence%20in%20situ%20hybridization%20in%20genome%20analysis%20of%20the%20mouse&amp;author=Y.%20Matsuda&amp;author=V.M.%20Chapman&amp;publication_year=1995&amp;journal=Electrophoresis&amp;volume=16&amp;pages=261-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7774567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Howell</div> <div class="given-names">W.M.</div></div><div class="name string-name"><div class="surname">Hsu</div> <div class="given-names">T.C.</div></div><div class="name string-name"><div class="surname">Block</div> <div class="given-names">B.M.</div></div></div> <div class="year">1977</div>, 
<div class="article-title">Visualization of centriole-bodies using silver stain</div>. <div class="source">Chromosoma</div> , <div class="volume">65</div>, <div class="fpage">9</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Visualization%20of%20centriole-bodies%20using%20silver%20stain&amp;author=W.M.%20Howell&amp;author=T.C.%20Hsu&amp;author=B.M.%20Block&amp;publication_year=1977&amp;journal=Chromosoma&amp;volume=65&amp;pages=9-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00293127" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00293127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00293127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/75084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Gentles</div> <div class="given-names">A.J.</div></div><div class="name string-name"><div class="surname">Hankus</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">474</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotation%2C%20submission%20and%20screening%20of%20repetitive%20elements%20in%20Repbase%3A%20RepbaseSubmitter%20and%20Censor&amp;author=O.%20Kohany&amp;author=A.J.%20Gentles&amp;author=L.%20Hankus&amp;author=J.%20Jurka&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=25&amp;pages=474" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-474" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-474"> </span></div> </div></div></div></div></div><div content-id="dsw053-b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">J.J.</div></div><div class="name string-name"><div class="surname">Antonacci</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div><div class="name string-name"><div class="surname">Amemiya</div> <div class="given-names">C.T.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Programmed loss of millions of base pairs from a vertebrate genome</div>. <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">106</div>, <div class="fpage">11212</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Programmed%20loss%20of%20millions%20of%20base%20pairs%20from%20a%20vertebrate%20genome&amp;author=J.J.%20Smith&amp;author=F.%20Antonacci&amp;author=E.E.%20Eichler&amp;author=C.T.%20Amemiya&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=106&amp;pages=11212-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0902358106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0902358106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0902358106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19561299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wong</div> <div class="given-names">A. K.</div></div><div class="name string-name"><div class="surname">Rattner</div> <div class="given-names">J. B.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">Sequence organization and cytological localization of the minor satellite of mouse</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">16</div>, <div class="fpage">11645</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20organization%20and%20cytological%20localization%20of%20the%20minor%20satellite%20of%20mouse&amp;author=A.%20K.%20Wong&amp;author=J.%20B.%20Rattner&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res&amp;volume=16&amp;pages=11645-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.24.11645" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.24.11645" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.24.11645"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3211746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vissel</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Nagy</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Choo</div> <div class="given-names">K. H.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">A satellite III sequence shared by human chromosomes 13, 14, and 21 that is contiguous with alpha satellite DNA</div>. <div class="source">Cytogenet. Cell Genet</div> ., <div class="volume">61</div>, <div class="fpage">81</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20satellite%20III%20sequence%20shared%20by%20human%20chromosomes%2013%2C%2014%2C%20and%2021%20that%20is%20contiguous%20with%20alpha%20satellite%20DNA&amp;author=B.%20Vissel&amp;author=A.%20Nagy&amp;author=K.%20H.%20Choo&amp;publication_year=1992&amp;journal=Cytogenet.%20Cell%20Genet&amp;volume=61&amp;pages=81-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000133374" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000133374" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000133374"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1395730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garrido-Ramos</div> <div class="given-names">M. A.</div></div><div class="name string-name"><div class="surname">Jamilena</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Lozano</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Ruiz Rejón</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Ruiz Rejón</div> <div class="given-names">M.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">The <em>Eco</em>RI centromeric satellite DNA of the Sparidae family (Pisces, Perciformes) contains a sequence motive common to other vertebrate centromeric satellite DNAs</div>. <div class="source">Cytogenet. Cell Genet</div> ., <div class="volume">71</div>, <div class="fpage">345</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20EcoRI%20centromeric%20satellite%20DNA%20of%20the%20Sparidae%20family%20(Pisces%2C%20Perciformes)%20contains%20a%20sequence%20motive%20common%20to%20other%20vertebrate%20centromeric%20satellite%20DNAs&amp;author=M.%20A.%20Garrido-Ramos&amp;author=M.%20Jamilena&amp;author=R.%20Lozano&amp;author=C.%20Ruiz%20Rej%C3%B3n&amp;author=M.%20Ruiz%20Rej%C3%B3n&amp;publication_year=1995&amp;journal=Cytogenet.%20Cell%20Genet&amp;volume=71&amp;pages=345-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000134137" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000134137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000134137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8521722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Caputo</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Giovannotti</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nisi Cerioni</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Splendiani</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Olmo</div> <div class="given-names">E.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Chromosomal study of native and hatchery trouts from Italy (<em>Salmo trutta</em> complex, Salmonidae): conventional and FISH analysis</div>. <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">124</div>, <div class="fpage">51</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20study%20of%20native%20and%20hatchery%20trouts%20from%20Italy%20(Salmo%20trutta%20complex%2C%20Salmonidae)%3A%20conventional%20and%20FISH%20analysis&amp;author=V.%20Caputo&amp;author=M.%20Giovannotti&amp;author=P.%20Nisi%20Cerioni&amp;author=A.%20Splendiani&amp;author=E.%20Olmo&amp;publication_year=2009&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=124&amp;pages=51-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000200088" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000200088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000200088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19372669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Malik</div> <div class="given-names">H.S.</div></div><div class="name string-name"><div class="surname">Eickbush</div> <div class="given-names">T.H.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Modular evolution of the integrase domain in the Ty3/Gypsy class of LTR retrotransposons</div>. <div class="source">J. Virol</div> ., <div class="volume">73</div>, <div class="fpage">5186</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modular%20evolution%20of%20the%20integrase%20domain%20in%20the%20Ty3%2FGypsy%20class%20of%20LTR%20retrotransposons&amp;author=H.S.%20Malik&amp;author=T.H.%20Eickbush&amp;publication_year=1999&amp;journal=J.%20Virol&amp;volume=73&amp;pages=5186-90" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10233986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martins</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Wasko</div> <div class="given-names">A.P.</div></div></div> <div class="year">2004</div>, Organization and evolution of 5S ribosomal DNA in the fish genome. In: <div class="person-group"><div class="name string-name"><div class="surname">Williams</div> <div class="given-names">C.R.</div></div></div>, ed. <div class="source">Focus on Genome Research</div> . 
<div class="publisher-loc">Hauppauge</div>: 
<div class="publisher-name">Nova Science Publishers</div>. pp. <div class="fpage">289</div>–<div class="lpage">318</div>.</div></div></div></div><div content-id="dsw053-b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yan</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Meng</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The nuclear DNA content and genetic diversity of <em>Lampetra morii</em></div>. <div class="source">PLoS ONE</div> , <div class="volume">11</div>, <div class="fpage">e0157494</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nuclear%20DNA%20content%20and%20genetic%20diversity%20of%20Lampetra%20morii&amp;author=X.%20Yan&amp;author=W.%20Meng&amp;author=F.%20Wu&amp;author=A.%20Xu&amp;author=S.%20Chen&amp;author=S.%20Huang&amp;publication_year=2016&amp;journal=PLoS%20ONE&amp;volume=11&amp;pages=e0157494" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0157494" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0157494" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0157494"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27388621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elder</div> <div class="given-names">J. F.</div></div><div class="name string-name"><div class="surname">Turner</div> <div class="given-names">B. J.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Concerted evolution of repetitive DNA sequences in eukaryotes</div>. <div class="source">Q Rev Biol</div> , <div class="volume">70</div>, <div class="fpage">297</div>–<div class="lpage">320</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Concerted%20evolution%20of%20repetitive%20DNA%20sequences%20in%20eukaryotes&amp;author=J.%20F.%20Elder&amp;author=B.%20J.%20Turner&amp;publication_year=1995&amp;journal=Q%20Rev%20Biol&amp;volume=70&amp;pages=297-320" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/419073" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F419073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F419073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7568673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Plohl</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Luchetti</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Meštrović</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Mantovani</div> <div class="given-names">B.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Satellite DNAs between selfishness and functionality: structure, genomics and evolution of tandem repeats in centromeric (hetero)chromatin</div>. <div class="source">Gene</div> , <div class="volume">409</div>, <div class="fpage">72</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Satellite%20DNAs%20between%20selfishness%20and%20functionality%3A%20structure%2C%20genomics%20and%20evolution%20of%20tandem%20repeats%20in%20centromeric%20(hetero)chromatin&amp;author=M.%20Plohl&amp;author=A.%20Luchetti&amp;author=N.%20Me%C5%A1trovi%C4%87&amp;author=B.%20Mantovani&amp;publication_year=2008&amp;journal=Gene&amp;volume=409&amp;pages=72-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.11.013" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.11.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.11.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18182173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Herniou</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Tristem</div> <div class="given-names">M.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Identification of multiple Gypsy LTR-retrotransposon lineages in vertebrate genomes</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">49</div>, <div class="fpage">358</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20multiple%20Gypsy%20LTR-retrotransposon%20lineages%20in%20vertebrate%20genomes&amp;author=K.%20Miller&amp;author=C.%20Lynch&amp;author=J.%20Martin&amp;author=E.%20Herniou&amp;author=M.%20Tristem&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol&amp;volume=49&amp;pages=358-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006559" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006559" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006559"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10473777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Llorens</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Muñoz-Pomer</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Bernad</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Botella</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Moya</div> <div class="given-names">A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Network dynamics of eukaryotic LTR retroelements beyond phylogenetic trees</div>. <div class="source">Biol. Direct</div> , <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Network%20dynamics%20of%20eukaryotic%20LTR%20retroelements%20beyond%20phylogenetic%20trees&amp;author=C.%20Llorens&amp;author=A.%20Mu%C3%B1oz-Pomer&amp;author=L.%20Bernad&amp;author=H.%20Botella&amp;author=A.%20Moya&amp;publication_year=2009&amp;journal=Biol.%20Direct&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-4-41" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19883502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Piskurek</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Nishihara</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Okada</div> <div class="given-names">N.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">The evolution of two partner LINE/SINE families and a full-length chromodomain-containing Ty3/Gypsy LTR element in the first reptilian genome of <em>Anolis carolinensis</em></div>. <div class="source">Gene</div> , <div class="volume">441</div>, <div class="fpage">111</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20two%20partner%20LINE%2FSINE%20families%20and%20a%20full-length%20chromodomain-containing%20Ty3%2FGypsy%20LTR%20element%20in%20the%20first%20reptilian%20genome%20of%20Anolis%20carolinensis&amp;author=O.%20Piskurek&amp;author=H.%20Nishihara&amp;author=N.%20Okada&amp;publication_year=2009&amp;journal=Gene&amp;volume=441&amp;pages=111-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.11.030" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.11.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.11.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19118606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baldwin</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Mortimer</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Patak</div> <div class="given-names">A.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">Evolutionary relationships among lamprey families: Amino acid composition analysis of lactate dehydrogenase</div>. <div class="source">Biochem. Syst. Ecol</div> ., <div class="volume">16</div>, <div class="fpage">351</div>–<div class="issue">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20relationships%20among%20lamprey%20families%3A%20Amino%20acid%20composition%20analysis%20of%20lactate%20dehydrogenase&amp;author=J.%20Baldwin&amp;author=K.%20Mortimer&amp;author=A.%20Patak&amp;publication_year=1988&amp;journal=Biochem.%20Syst.%20Ecol&amp;volume=16&amp;pages=351" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0305-1978(88)90022-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0305-1978(88)90022-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0305-1978(88)90022-1"> </span></div> </div></div></div></div></div><div content-id="dsw053-b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Silver</div> <div class="given-names">M.R.</div></div><div class="name string-name"><div class="surname">Kawauchi</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Nozaki</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Sower</div> <div class="given-names">S.A.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Cloning and analysis of the lamprey GnRH-III cDNA from eight species of lamprey representing the three families of Petromyzoniformes</div>. <div class="source">Gen. Comp. Endocrinol</div> ., <div class="volume">139</div>, <div class="fpage">85</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20analysis%20of%20the%20lamprey%20GnRH-III%20cDNA%20from%20eight%20species%20of%20lamprey%20representing%20the%20three%20families%20of%20Petromyzoniformes&amp;author=M.R.%20Silver&amp;author=H.%20Kawauchi&amp;author=M.%20Nozaki&amp;author=S.A.%20Sower&amp;publication_year=2004&amp;journal=Gen.%20Comp.%20Endocrinol&amp;volume=139&amp;pages=85-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygcen.2004.07.011" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygcen.2004.07.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygcen.2004.07.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15474539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Nakata</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Moriyama</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Occurrence of two functionally distinct proopiomelanocortin genes in all modern lampreys</div>. <div class="source">Gen. Comp. Endocrinol</div> ., <div class="volume">148</div>, <div class="fpage">72</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occurrence%20of%20two%20functionally%20distinct%20proopiomelanocortin%20genes%20in%20all%20modern%20lampreys&amp;author=A.%20Takahashi&amp;author=O.%20Nakata&amp;author=S.%20Moriyama&amp;publication_year=2006&amp;journal=Gen.%20Comp.%20Endocrinol&amp;volume=148&amp;pages=72-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygcen.2005.09.003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygcen.2005.09.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygcen.2005.09.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16243327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Impact of asymmetric gene repertoire between cyclostomes and gnathostomes</div>. <div class="source">Semin. Cell Dev. Biol</div> ., <div class="volume">24</div>, <div class="fpage">119</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20asymmetric%20gene%20repertoire%20between%20cyclostomes%20and%20gnathostomes&amp;author=S.%20Kuraku&amp;publication_year=2013&amp;journal=Semin.%20Cell%20Dev.%20Biol&amp;volume=24&amp;pages=119-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.semcdb.2012.12.009" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.semcdb.2012.12.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.semcdb.2012.12.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23291292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw053-b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw053-b44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">J.J.</div></div><div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Holt</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Sequencing of the sea lamprey (<em>Petromyzon marinus</em>) genome provides insights into vertebrate evolution</div>. <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">415</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20the%20sea%20lamprey%20(Petromyzon%20marinus)%20genome%20provides%20insights%20into%20vertebrate%20evolution&amp;author=J.J.%20Smith&amp;author=S.%20Kuraku&amp;author=C.%20Holt&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=415-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2568" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2568" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2568"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23435085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute. </div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2738680&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw053/1/dsw053_Supp.pdf?Expires=1509239091&amp;Signature=cus7vxb2~pK6lSJxXsfYGs1TSrNJ8ahLxhCtsn3F27HNHJMzjXVgGjZqwMmYXmstMbNAfAax13CBN6Usja~yCdhkckdXRwcz6tXpK4bYZdQQ9ZGy000-SZd0nyDwz~7-cAPy5dxtEHCWE~j3w1QsJqY-s4AZVVB4BsQkAqwlrK9BaZ29oNV1Lw0E6B7UxChIP5NOJI6bekaSZGFZav8TOEa9bDiwPvgFDm65olQMGXwJUVqPaUsGOaEyTS~Rnn26iFN5w18m5NgWNVt-NnBrYH6rDZcVQedEjeiZxXS0UTpm4r~MoPeswbxoRrgUCEeUSutV3FZBdFRsZr9ZoEV-kw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">224</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsw053"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2738680"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f93%2f2738680">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F93%2F2738680" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2738680'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Molecular cytogenetic characterization of chromosome site-specific repetitive sequences in the Arctic lamprey (Lethenteron camtschaticum, Petromyzontidae)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f93%2f2738680">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

