

<!DOCTYPE html>
<html lang="en">

<head><title>
	Complete Genomic Structure of the Cultivated Rice Endophyte Azospirillum sp. B510 | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/1/37/347240","@type":"ScholarlyArticle","name":"Complete Genomic Structure of the Cultivated Rice Endophyte  Azospirillum  sp. B510","datePublished":"2010-01-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2010-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp026","keywords":["<italic>Azospirillum</italic>","endophyte","rice plant","N<sub>2</sub> fixation","plant hormone"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/1/37/347240","author":[{"name":"Kaneko, Takakazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan    2  Faculty of Engineering ,  Kyoto Sangyo University ,  Motoyama, Kamigamo, Kita-Ku, Kyoto 603-8555 ,  Japan","@type":"Person"},{"name":"Minamisawa, Kiwamu","affiliation":"3  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577 ,  Japan","@type":"Person"},{"name":"Isawa, Tsuyoshi","affiliation":"3  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577 ,  Japan","@type":"Person"},{"name":"Nakatsukasa, Hiroki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Mitsui, Hisayuki","affiliation":"3  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577 ,  Japan","@type":"Person"},{"name":"Kawaharada, Yasuyuki","affiliation":"3  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577 ,  Japan","@type":"Person"},{"name":"Nakamura, Yasukazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kawashima, Kumiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ono, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shimizu, Yoshimi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Takahashi, Chika","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Minami, Chiharu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujishiro, Tsunakazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Katoh, Midori","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakazaki, Naomi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamada, Manabu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"We determined the nucleotide sequence of the entire genome of a diazotrophic endophyte, Azospirillum sp. B510. Strain B510 is an endophytic bacterium isolated from stems of rice plants (Oryza sativa cv. Nipponbare). The genome of B510 consisted of a single chromosome (3 311 395 bp) and six plasmids, designated as pAB510a (1 455 109 bp), pAB510b (723 779 bp), pAB510c (681 723 bp), pAB510d (628 837 bp), pAB510e (537 299 bp), and pAB510f (261 596 bp). The chromosome bears 2893 potential protein-encoding genes, two sets of rRNA gene clusters (rrns), and 45 tRNA genes representing 37 tRNA species. The genomes of the six plasmids contained a total of 3416 protein-encoding genes, seven sets of rrns, and 34 tRNAs representing 19 tRNA species. Eight genes for plasmid-specific tRNA species are located on either pAB510a or pAB510d. Two out of eight genomic islands are inserted in the plasmids, pAB510b and pAB510e, and one of the islands is inserted into trnfM-CAU in the rrn located on pAB510e. Genes other than the nif gene cluster that are involved in N2 fixation and are homologues of Bradyrhizobium japonicum USDA110 include fixABCX, fixNOQP, fixHIS, fixG, and fixLJK. Three putative plant hormone-related genes encoding tryptophan 2-monooxytenase (iaaM) and indole-3-acetaldehyde hydrolase (iaaH), which are involved in IAA biosynthesis, and ACC deaminase (acdS), which reduces ethylene levels, were identified. Multiple gene-clusters for tripartite ATP-independent periplasmic-transport systems and a diverse set of malic enzymes were identified, suggesting that B510 utilizes C4-dicarboxylate during its symbiotic relationship with the host plant.","pageStart":"37","pageEnd":"50","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/1/1/m_cover.gif?Expires=1572217099&Signature=TaTkDZaKJzqGpGOdWbAyiwSSkz0pN6qmYGSHez3HRrjhjPr93soEfYYvoANdcQuxJN1i0beericRCOwYpTwyWqNEBcq1NYfaFvDQ2Xt4MfgoBeHh-qsLQsyW8EdMpUCruTGrfT4femhtCqPW0VmIs1ZihRKyirmswu0Wi1JomhS1xZpM5ZTyONs3wV-2f1xCGBp99bLKUGeNiYLX~VfJ4jeZ4J4n7w84oi-U7ArB5IgLsupk3c88ESC~Ezrf2TDzm63x8MUBqWC4PHhL3YAccW2aJilo6gfMuYkFYMDMosStCa64JGzRQKUOi3okXhoOaYxf3dwqzd0gmGx2AwtSIw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Complete Genomic Structure of the Cultivated Rice Endophyte  Azospirillum  sp. B510","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/1/1/m_cover.gif?Expires=1572217099&Signature=TaTkDZaKJzqGpGOdWbAyiwSSkz0pN6qmYGSHez3HRrjhjPr93soEfYYvoANdcQuxJN1i0beericRCOwYpTwyWqNEBcq1NYfaFvDQ2Xt4MfgoBeHh-qsLQsyW8EdMpUCruTGrfT4femhtCqPW0VmIs1ZihRKyirmswu0Wi1JomhS1xZpM5ZTyONs3wV-2f1xCGBp99bLKUGeNiYLX~VfJ4jeZ4J4n7w84oi-U7ArB5IgLsupk3c88ESC~Ezrf2TDzm63x8MUBqWC4PHhL3YAccW2aJilo6gfMuYkFYMDMosStCa64JGzRQKUOi3okXhoOaYxf3dwqzd0gmGx2AwtSIw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complete Genomic Structure of the Cultivated Rice Endophyte  Azospirillum  sp. B510 | DNA Research | Oxford Academic" />
    <meta property="og:description" content="We determined the nucleotide sequence of the entire genome of a diazotrophic endophyte, Azospirillum sp. B510. Strain B510 is an endophytic bacterium isolated from stems of rice plants (Oryza sativa cv. Nipponbare). The genome of B510 consisted of a single chromosome (3 311 395 bp) and six plasmids, designated as pAB510a (1 455 109 bp), pAB510b (723 779 bp), pAB510c (681 723 bp), pAB510d (628 837 bp), pAB510e (537 299 bp), and pAB510f (261 596 bp). The chromosome bears 2893 potential protein-encoding genes, two sets of rRNA gene clusters (rrns), and 45 tRNA genes representing 37 tRNA species. The genomes of the six plasmids contained a total of 3416 protein-encoding genes, seven sets of rrns, and 34 tRNAs representing 19 tRNA species. Eight genes for plasmid-specific tRNA species are located on either pAB510a or pAB510d. Two out of eight genomic islands are inserted in the plasmids, pAB510b and pAB510e, and one of the islands is inserted into trnfM-CAU in the rrn located on pAB510e. Genes other than the nif gene cluster that are involved in N2 fixation and are homologues of Bradyrhizobium japonicum USDA110 include fixABCX, fixNOQP, fixHIS, fixG, and fixLJK. Three putative plant hormone-related genes encoding tryptophan 2-monooxytenase (iaaM) and indole-3-acetaldehyde hydrolase (iaaH), which are involved in IAA biosynthesis, and ACC deaminase (acdS), which reduces ethylene levels, were identified. Multiple gene-clusters for tripartite ATP-independent periplasmic-transport systems and a diverse set of malic enzymes were identified, suggesting that B510 utilizes C4-dicarboxylate during its symbiotic relationship with the host plant." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/1/37/347240" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217099&amp;Signature=JidLb5~kLHiC7N7JChprHwMClahTzArgMnwzl9WxSeizbhZjaQb1uhClwG55puN--DEO9TBafSwVrgTX8znU30Iw2LPQBiq0172CXbzNe6FE~8-glRsDKvpumLfFeCMN1QTeEgwVl37hodBA4HYb6XSqNaw61hGVlPqGd0uCOn0TOJYi-3EWf3frcIPlxVbF~t8aWCV3HGJmvTzW2JrpaH6rCaAOWFrmKTgHH91QDI4SqCsQ8sT5kb1php-fIZwxWPD~SAcSfLZz3N4KJGsxGDbU7J9gcuT9064q575~ptBOY1iD5HBvNOto1YY-1NqKVykVw16eNzo6NNwPIdMB-Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2010-01-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kaneko, Takakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="Faculty of Engineering, Kyoto Sangyo University, Motoyama, Kamigamo, Kita-Ku, Kyoto 603-8555, Japan" /><meta name="citation_author" content="Minamisawa, Kiwamu" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577, Japan" /><meta name="citation_author" content="Isawa, Tsuyoshi" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577, Japan" /><meta name="citation_author" content="Nakatsukasa, Hiroki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Mitsui, Hisayuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577, Japan" /><meta name="citation_author" content="Kawaharada, Yasuyuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577, Japan" /><meta name="citation_author" content="Nakamura, Yasukazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kawashima, Kumiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ono, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shimizu, Yoshimi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Takahashi, Chika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Minami, Chiharu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujishiro, Tsunakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Katoh, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakazaki, Naomi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamada, Manabu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Complete Genomic Structure of the Cultivated Rice Endophyte Azospirillum sp. B510" /><meta name="citation_firstpage" content="37" /><meta name="citation_lastpage" content="50" /><meta name="citation_doi" content="10.1093/dnares/dsp026" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2010/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Bacterial endophytes in agricultural crops; Can. J. Microbiol.;  citation_year=1997;  citation_volume=43;  citation_pages=895-914; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of endophytic bacteria from wild and traditionally cultivated rice varieties; Soil Sci. Plant Nutr.;  citation_year=2000;  citation_volume=46;  citation_pages=617-29; " /><meta name="citation_reference" content="citation_title=Bacterial endophytes: potential role in developing sustainable systems of crop production; CRC Crit. Rev. Plant Sci.;  citation_year=2000;  citation_volume=19;  citation_pages=1-30; " /><meta name="citation_reference" content="citation_title=Endophytic bacteria in the rice plant; Microbes Environ.;  citation_year=2008;  citation_volume=23;  citation_pages=109-17; " /><meta name="citation_reference" content="citation_title=Microbial community analysis of the phytosphere using culture-independent methodologies; Microbes Environ.;  citation_year=2007;  citation_volume=22;  citation_pages=93-105; " /><meta name="citation_reference" content="citation_title=Life in grasses: diazotrophic endophytes; Trends Microbiol.;  citation_year=1998;  citation_volume=4;  citation_pages=139-43; " /><meta name="citation_reference" content="citation_title=Azoarcus sp. BH72 as a model for nitrogen-fixing grass endophytes; J. Biotechnol.;  citation_year=2003;  citation_volume=106;  citation_pages=169-78; " /><meta name="citation_reference" content="citation_title=Complete genome of the mutualistic N2-fixing grass endophyte Azoarcus sp. strain BH72; Nat. Biotechnol.;  citation_year=2006;  citation_volume=24;  citation_pages=1385-91; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the N2-foxong broad host range endophyte Klebsiella pneumoniae 342 and virulence predictions verified in mice; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e10000141" /><meta name="citation_reference" content="citation_title=A milestone for endophyte biotechnology; Nat. Biotechnol.;  citation_year=2006;  citation_volume=24;  citation_pages=1357-8; " /><meta name="citation_reference" content="citation_title=The genus Azospirillum;  citation_edition=3rd edition; citation_publisher=Springer, New York; The Prokaryotes, vol. 5;  citation_year=2006;  citation_pages=115-40; " /><meta name="citation_reference" content="citation_title=Agronomic applications of Azospirillum: an evaluation of 20 years worldwide field inoculation; Soil Biol. Biochem.;  citation_year=1994;  citation_volume=26;  citation_pages=1591-601; " /><meta name="citation_reference" content="citation_title=Endophytic colonization and in planta nitrogen fixation by a Herbaspirillum sp. isolated from wild rice species; Appl. Environ. Microbiol.;  citation_year=2001;  citation_volume=67;  citation_pages=5285-93; " /><meta name="citation_reference" content="citation_title=Azospirillum oryzae sp. nov., a nitrogen-fixing bacterium isolated from the roots of the rice plant Oryza sativa; Int. J. Syst. Evol. Microbiol.;  citation_year=2005;  citation_volume=55;  citation_pages=1435-8; " /><meta name="citation_reference" content="citation_title=Effects of colonization of a bacterial endophyte, Azospirillum sp. B510 on disease resistance in rice; citation_text=doi:10.1271/bbb.90402; Biosci. Biotechnol. Biochem.;  citation_year=2009;  citation_volume=73; " /><meta name="citation_reference" content="citation_title=Novel endophytic nitrogen-fixing clostridia from the grass Miscanthus sinensis as revealed by terminal restriction fragment length polymorphism analysis; Appl. Environ. Microbiol.;  citation_year=2004;  citation_volume=70;  citation_pages=6580-6; " /><meta name="citation_reference" content="citation_title=Effect of inoculation with anaerobic nitrogen-fixing consortium on salt tolerance of Miscanthus sinensis; Soil Sci. Plant Nutr.;  citation_year=2005;  citation_volume=51;  citation_pages=243-9; " /><meta name="citation_reference" content="citation_title=Expression of the nifH gene of a Herbaspirillum endophytes in wild rice species: daily rhythm during the light-dark cycle; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=71;  citation_pages=8183-90; " /><meta name="citation_reference" content="citation_title=Complete genomic sequence of the filamentous nitrogen-fixing cyanobacterium Anabaena sp. strain PCC 7120; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=205-13; " /><meta name="citation_reference" content="citation_title=ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=3100-8; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4636-41; " /><meta name="citation_reference" content="citation_title=The Road Map to the Manual;  citation_edition=2nd edition; citation_publisher=Springer, New York; Bergey's Manual of Systematic Bacteriology, vol. 1;  citation_year=2001;  citation_pages=119-66; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=KAAS: an automatic genome annotation and pathway reconstruction server; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W182-5; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D224-8; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=376" /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria; Mol. Biol. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=660-5; " /><meta name="citation_reference" content="citation_title=Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=5839-51; " /><meta name="citation_reference" content="citation_title=FtsK, a literate chromosome segregation machine; Mol. Microbiol.;  citation_year=2007;  citation_volume=64;  citation_pages=1434-41; " /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1269-76; " /><meta name="citation_reference" content="citation_title=Colonization and nitrogen-fixing ability of Herbaspirillum sp. Strain B501 gfp1 and assessment of its growth-promoting ability in cultivated rice; Microbes Environ.;  citation_year=2007;  citation_volume=22;  citation_pages=197-206; " /><meta name="citation_reference" content="citation_title=Genome structure of the genus Azospirillum; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=4113-6; " /><meta name="citation_reference" content="citation_title=Conserved gene cluster at replication origins of the alpha-proteobacteria Caulobacter crescentus and Rickettsia prowazekii; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=1824-9; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the facultative anaerobic magnetotactic bacterium Magnetospirillum sp. strain AMB-1; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=157-66; " /><meta name="citation_reference" content="citation_title=Insertion sequences revisited; citation_publisher=Washington, DC; Mobile DNA II, American Society for Microbiology;  citation_year=2002;  citation_pages=305-66; " /><meta name="citation_reference" content="citation_title=Bacteriophage prevalence in the genus Azospirillum and analysis of the first genome sequence of an Azospirillum brasilense integrative phage; Appl. Environ. Microbiol.;  citation_year=2008;  citation_volume=74;  citation_pages=861-74; " /><meta name="citation_reference" content="citation_title=CRISPR-a widespread system that provides acquired resistance against phages in bacteria and archaea; Nat. Rev. Microbiol.;  citation_year=2008;  citation_volume=6;  citation_pages=181-6; " /><meta name="citation_reference" content="citation_title=Small CRISPR RNAs guide antiviral defense in prokaryotes; Science;  citation_year=2008;  citation_volume=321;  citation_pages=960-4; " /><meta name="citation_reference" content="citation_title=Genomic islands: tools of bacterial horizontal gene transfer and evolution; FEMS Microbiol. Rev.;  citation_year=2009;  citation_volume=33;  citation_pages=376-93; " /><meta name="citation_reference" content="citation_title=rrnDB: documenting the number of rRNA and tRNA genes in bacteria and archaea; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D489-93; " /><meta name="citation_reference" content="citation_title=The fixABCX genes in Rhodospirillum rubrum encode a putative membrane complex participating in electron transfer to nitrogenase; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=2052-60; " /><meta name="citation_reference" content="citation_title=The electron transfer flavoprotein fixABCX gene products from Azospirillum brasilense show a NifA-dependent promoter regulation; Curr. Microbiol.;  citation_year=2004;  citation_volume=49;  citation_pages=267-73; " /><meta name="citation_reference" content="citation_title=Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia; Science;  citation_year=2007;  citation_volume=316;  citation_pages=1307-12; " /><meta name="citation_reference" content="citation_title=Genes for a microaerobically induced oxidase complex in Bradyrhizobium japonicum are essential for a nitrogen-fixing endosymbiosis; Proc. Natl Acad. Sci. USA.;  citation_year=1993;  citation_volume=90;  citation_pages=3309-13; " /><meta name="citation_reference" content="citation_title=A cytochrome cbb3 (cytochrome c) terminal oxidase in Azospirillum brasilense Sp7 supports microaerobic growth; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=5689-96; " /><meta name="citation_reference" content="citation_title=Genetic regulation of nitrogen fixation in rhizobia; Microbiol. Rev.;  citation_year=1994;  citation_volume=58;  citation_pages=352-86; " /><meta name="citation_reference" content="citation_title=Signal transduction by heme-containing PAS-domain proteins; J. Appl. Physiol.;  citation_year=2004;  citation_volume=96;  citation_pages=774-83; " /><meta name="citation_reference" content="citation_title=Properties of bacterial endophytes and their proposed role in plant growth; Trends Microbiol.;  citation_year=2008;  citation_volume=16;  citation_pages=463-71; " /><meta name="citation_reference" content="citation_title=Rhizobial strategies to enhance symbiotic interactions: rhizobitoxine and 1-aminocyclopropane-1-carboxylate deaminase; Microbes Environ.;  citation_year=2004;  citation_volume=19;  citation_pages=99-111; " /><meta name="citation_reference" content="citation_title=Identification of DNA sequences that regulate the expression of the Enterobacter cloacae UV4 1-aminocyclopropane-1-carboxylic acid deaminase gene; Can. J. Microbiol.;  citation_year=2000;  citation_volume=46;  citation_pages=1159-65; " /><meta name="citation_reference" content="citation_title=Transcriptiona; regulation of the Enterobacter cloacae UW4 1-aminocyclopropane-1-carboxylate (ACC) deaminase gene (acdS); Can. J. Microbiol.;  citation_year=2001;  citation_volume=47;  citation_pages=356-67; " /><meta name="citation_reference" content="citation_title=Bacterial ACC deaminase and the alleviation of plant stress; Adv. Appl. Micrbiol.;  citation_year=2004;  citation_volume=56;  citation_pages=291-312; " /><meta name="citation_reference" content="citation_title=Synthesis of phytohormones by plant-associated bacteria; CRC Crit. Rev. Microbiol.;  citation_year=1995;  citation_volume=21;  citation_pages=1-18; " /><meta name="citation_reference" content="citation_title=Transcriptional analysis of the Azospirillum brasilense indole-3-pyruvate decarboxylase gene and identification of a cis-acting sequence involved in auxin responsive expression; Plant Mol. Microbe Interact.;  citation_year=2005;  citation_volume=18;  citation_pages=311-23; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=TCDB: the transporter classification database for membrane transport protein analyses and information; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D181-6; " /><meta name="citation_reference" content="citation_title=TonB-dependent trans-envelope signaling: the exception or the rule?; Trends Microbiol.;  citation_year=2005;  citation_volume=13;  citation_pages=343-7; " /><meta name="citation_reference" content="citation_title=NAD-malic enzyme affects nitrogen fixing activity of Bradyrhizobium japonicum USDA 110 bacteroids in soybean nodules; Microbes Environ.;  citation_year=2008;  citation_volume=23;  citation_pages=215-20; " /><meta name="citation_reference" content="citation_title=Type IV secretion systems: tools of bacterial horizontal gene transfer and virulence; Cell. Microbiol.;  citation_year=2008;  citation_volume=10;  citation_pages=2377-86; " /><meta name="citation_reference" content="citation_title=Photosensing in chemotrophic, non-phototrophic bacteria: let there be light sensing too; Trends Microbiol.;  citation_year=2007;  citation_volume=15;  citation_pages=554-62; " /><meta name="citation_reference" content="citation_title=The pair of bacteriophytochromes from Agrobacterium tumefaciens are histidine kinases with opposing photobiological properties; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=2807-12; " /><meta name="citation_reference" content="citation_title=A singular bacteriophytochrome acquired by lateral gene transfer; J. Biol. Chem.;  citation_year=2007;  citation_volume=282;  citation_pages=7320-8; " /><meta name="citation_reference" content="citation_title=A quorum-quenching approach to identify quorum-sensing-regulated functions in Azospirillum lipoferum; Res. Microbiol.;  citation_year=2008;  citation_volume=159;  citation_pages=699-708; " /><meta name="citation_reference" content="citation_title=N-acyl-homoserine lactone-mediated quorum-sensing in Azospirillum: an exception rather than a rule; FEMS Microbiol. Ecol.;  citation_year=2006;  citation_volume=58;  citation_pages=155-68; " /><meta name="citation_reference" content="citation_title=Annotation of the pRhico plasmid of Azospirillum brasilense reveals its role in determining the outer surface composition; FEMS Microbiol. Lett.;  citation_year=2004;  citation_volume=232;  citation_pages=165-72; " /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/1/37/994067/dsp026.pdf" /><meta name="description" content="Abstract.  We determined the nucleotide sequence of the entire genome of a diazotrophic endophyte, Azospirillum sp. B510. Strain B510 is an endophytic bacterium" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/1/37/347240" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/1/37/347240" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsp026';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347240" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFhKgBPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5la28sIFRha2FrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgRW5naW5lZXJpbmcsIEt5b3RvIFNhbmd5byBVbml2ZXJzaXR5LCBNb3RveWFtYSwgS2FtaWdhbW8sIEtpdGEtS3UsIEt5b3RvIDYwMy04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pbmFtaXNhd2EsIEtpd2FtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSwgTWl5YWdpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklzYXdhLCBUc3V5b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSwgTWl5YWdpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2F0c3VrYXNhLCBIaXJva2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWl0c3VpLCBIaXNheXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSwgTWl5YWdpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthd2FoYXJhZGEsIFlhc3V5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgQW9iYS1rdSwgU2VuZGFpLCBNaXlhZ2kgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYW11cmEsIFlhc3VrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhdGFuYWJlLCBBa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhc2hpbWEsIEt1bWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPbm8sIEFraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaW1penUsIFlvc2hpbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWhhc2hpLCBDaGlrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaW5hbWksIENoaWhhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVqaXNoaXJvLCBUc3VuYWthenUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBNaXRzdXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthdG9oLCBNaWRvcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXpha2ksIE5hb21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2F5YW1hLCBTaGlub2J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFkYSwgTWFuYWJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb21wbGV0ZSBHZW5vbWljIFN0cnVjdHVyZSBvZiB0aGUgQ3VsdGl2YXRlZCBSaWNlIEVuZG9waHl0ZSBBem9zcGlyaWxsdW0gc3AuIEI1MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3AwMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCBlbmRvcGh5dGVzIGluIGFncmljdWx0dXJhbCBjcm9wczsgQ2FuLiBKLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9ODk1LTkxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGVuZG9waHl0aWMgYmFjdGVyaWEgZnJvbSB3aWxkIGFuZCB0cmFkaXRpb25hbGx5IGN1bHRpdmF0ZWQgcmljZSB2YXJpZXRpZXM7IFNvaWwgU2NpLiBQbGFudCBOdXRyLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTYxNy0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGVuZG9waHl0ZXM6IHBvdGVudGlhbCByb2xlIGluIGRldmVsb3Bpbmcgc3VzdGFpbmFibGUgc3lzdGVtcyBvZiBjcm9wIHByb2R1Y3Rpb247IENSQyBDcml0LiBSZXYuIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvcGh5dGljIGJhY3RlcmlhIGluIHRoZSByaWNlIHBsYW50OyBNaWNyb2JlcyBFbnZpcm9uLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEwOS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGNvbW11bml0eSBhbmFseXNpcyBvZiB0aGUgcGh5dG9zcGhlcmUgdXNpbmcgY3VsdHVyZS1pbmRlcGVuZGVudCBtZXRob2RvbG9naWVzOyBNaWNyb2JlcyBFbnZpcm9uLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTkzLTEwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlmZSBpbiBncmFzc2VzOiBkaWF6b3Ryb3BoaWMgZW5kb3BoeXRlczsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEzOS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXpvYXJjdXMgc3AuIEJINzIgYXMgYSBtb2RlbCBmb3Igbml0cm9nZW4tZml4aW5nIGdyYXNzIGVuZG9waHl0ZXM7IEouIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTE2OS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIG9mIHRoZSBtdXR1YWxpc3RpYyBOMi1maXhpbmcgZ3Jhc3MgZW5kb3BoeXRlIEF6b2FyY3VzIHNwLiBzdHJhaW4gQkg3MjsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODUtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgTjItZm94b25nIGJyb2FkIGhvc3QgcmFuZ2UgZW5kb3BoeXRlIEtsZWJzaWVsbGEgcG5ldW1vbmlhZSAzNDIgYW5kIHZpcnVsZW5jZSBwcmVkaWN0aW9ucyB2ZXJpZmllZCBpbiBtaWNlOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAwMTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtaWxlc3RvbmUgZm9yIGVuZG9waHl0ZSBiaW90ZWNobm9sb2d5OyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTM1Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2VudXMgQXpvc3BpcmlsbHVtOyAgY2l0YXRpb25fZWRpdGlvbj0zcmQgZWRpdGlvbjsgY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBOZXcgWW9yazsgVGhlIFByb2thcnlvdGVzLCB2b2wuIDU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl9wYWdlcz0xMTUtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFncm9ub21pYyBhcHBsaWNhdGlvbnMgb2YgQXpvc3BpcmlsbHVtOiBhbiBldmFsdWF0aW9uIG9mIDIwIHllYXJzIHdvcmxkd2lkZSBmaWVsZCBpbm9jdWxhdGlvbjsgU29pbCBCaW9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTE1OTEtNjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvcGh5dGljIGNvbG9uaXphdGlvbiBhbmQgaW4gcGxhbnRhIG5pdHJvZ2VuIGZpeGF0aW9uIGJ5IGEgSGVyYmFzcGlyaWxsdW0gc3AuIGlzb2xhdGVkIGZyb20gd2lsZCByaWNlIHNwZWNpZXM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz01Mjg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bem9zcGlyaWxsdW0gb3J5emFlIHNwLiBub3YuLCBhIG5pdHJvZ2VuLWZpeGluZyBiYWN0ZXJpdW0gaXNvbGF0ZWQgZnJvbSB0aGUgcm9vdHMgb2YgdGhlIHJpY2UgcGxhbnQgT3J5emEgc2F0aXZhOyBJbnQuIEouIFN5c3QuIEV2b2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0xNDM1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgY29sb25pemF0aW9uIG9mIGEgYmFjdGVyaWFsIGVuZG9waHl0ZSwgQXpvc3BpcmlsbHVtIHNwLiBCNTEwIG9uIGRpc2Vhc2UgcmVzaXN0YW5jZSBpbiByaWNlOyBjaXRhdGlvbl90ZXh0PWRvaToxMC4xMjcxL2JiYi45MDQwMjsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgZW5kb3BoeXRpYyBuaXRyb2dlbi1maXhpbmcgY2xvc3RyaWRpYSBmcm9tIHRoZSBncmFzcyBNaXNjYW50aHVzIHNpbmVuc2lzIGFzIHJldmVhbGVkIGJ5IHRlcm1pbmFsIHJlc3RyaWN0aW9uIGZyYWdtZW50IGxlbmd0aCBwb2x5bW9ycGhpc20gYW5hbHlzaXM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz02NTgwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdCBvZiBpbm9jdWxhdGlvbiB3aXRoIGFuYWVyb2JpYyBuaXRyb2dlbi1maXhpbmcgY29uc29ydGl1bSBvbiBzYWx0IHRvbGVyYW5jZSBvZiBNaXNjYW50aHVzIHNpbmVuc2lzOyBTb2lsIFNjaS4gUGxhbnQgTnV0ci47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0yNDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiB0aGUgbmlmSCBnZW5lIG9mIGEgSGVyYmFzcGlyaWxsdW0gZW5kb3BoeXRlcyBpbiB3aWxkIHJpY2Ugc3BlY2llczogZGFpbHkgcmh5dGhtIGR1cmluZyB0aGUgbGlnaHQtZGFyayBjeWNsZTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTgxODMtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9taWMgc2VxdWVuY2Ugb2YgdGhlIGZpbGFtZW50b3VzIG5pdHJvZ2VuLWZpeGluZyBjeWFub2JhY3Rlcml1bSBBbmFiYWVuYSBzcC4gc3RyYWluIFBDQyA3MTIwOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjA1LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUkFHT1JOLCBhIHByb2dyYW0gdG8gZGV0ZWN0IHRSTkEgZ2VuZXMgYW5kIHRtUk5BIGdlbmVzIGluIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BbW1lcjogY29uc2lzdGVudCBhbmQgcmFwaWQgYW5ub3RhdGlvbiBvZiByaWJvc29tYWwgUk5BIGdlbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0zMTAwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIG1pY3JvYmlhbCBnZW5lIGlkZW50aWZpY2F0aW9uIHdpdGggR0xJTU1FUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDYzNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJvYWQgTWFwIHRvIHRoZSBNYW51YWw7ICBjaXRhdGlvbl9lZGl0aW9uPTJuZCBlZGl0aW9uOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrOyBCZXJnZXkncyBNYW51YWwgb2YgU3lzdGVtYXRpYyBCYWN0ZXJpb2xvZ3ksIHZvbC4gMTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3BhZ2VzPTExOS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LQUFTOiBhbiBhdXRvbWF0aWMgZ2Vub21lIGFubm90YXRpb24gYW5kIHBhdGh3YXkgcmVjb25zdHJ1Y3Rpb24gc2VydmVyOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XMTgyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBkZXZlbG9wbWVudHMgaW4gdGhlIEludGVyUHJvIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EMjI0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDT0cgZGF0YWJhc2U6IGFuIHVwZGF0ZWQgdmVyc2lvbiBpbmNsdWRlcyBldWthcnlvdGVzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lTWF0Y2hlcjogYSBncmFwaGljYWwgdXNlciBpbnRlcmZhY2UgZm9yIEROQSBzZXF1ZW5jZSBjb21wYXJpc29uOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzeW1tZXRyaWMgc3Vic3RpdHV0aW9uIHBhdHRlcm5zIGluIHRoZSB0d28gRE5BIHN0cmFuZHMgb2YgYmFjdGVyaWE7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhhZ2VfRmluZGVyOiBhdXRvbWF0ZWQgaWRlbnRpZmljYXRpb24gYW5kIGNsYXNzaWZpY2F0aW9uIG9mIHByb3BoYWdlIHJlZ2lvbnMgaW4gY29tcGxldGUgYmFjdGVyaWFsIGdlbm9tZSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTU4MzktNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ0c0ssIGEgbGl0ZXJhdGUgY2hyb21vc29tZSBzZWdyZWdhdGlvbiBtYWNoaW5lOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz0xNDM0LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgZGUgbm92byBpZGVudGlmaWNhdGlvbiBvZiByZXBlYXQgc2VxdWVuY2UgZmFtaWxpZXMgaW4gc2VxdWVuY2VkIGdlbm9tZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTI2OS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29sb25pemF0aW9uIGFuZCBuaXRyb2dlbi1maXhpbmcgYWJpbGl0eSBvZiBIZXJiYXNwaXJpbGx1bSBzcC4gU3RyYWluIEI1MDEgZ2ZwMSBhbmQgYXNzZXNzbWVudCBvZiBpdHMgZ3Jvd3RoLXByb21vdGluZyBhYmlsaXR5IGluIGN1bHRpdmF0ZWQgcmljZTsgTWljcm9iZXMgRW52aXJvbi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBnZW51cyBBem9zcGlyaWxsdW07IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9NDExMy02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2ZWQgZ2VuZSBjbHVzdGVyIGF0IHJlcGxpY2F0aW9uIG9yaWdpbnMgb2YgdGhlIGFscGhhLXByb3Rlb2JhY3RlcmlhIENhdWxvYmFjdGVyIGNyZXNjZW50dXMgYW5kIFJpY2tldHRzaWEgcHJvd2F6ZWtpaTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz0xODI0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgZmFjdWx0YXRpdmUgYW5hZXJvYmljIG1hZ25ldG90YWN0aWMgYmFjdGVyaXVtIE1hZ25ldG9zcGlyaWxsdW0gc3AuIHN0cmFpbiBBTUItMTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNTctNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2VydGlvbiBzZXF1ZW5jZXMgcmV2aXNpdGVkOyBjaXRhdGlvbl9wdWJsaXNoZXI9V2FzaGluZ3RvbiwgREM7IE1vYmlsZSBETkEgSUksIEFtZXJpY2FuIFNvY2lldHkgZm9yIE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3BhZ2VzPTMwNS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaW9waGFnZSBwcmV2YWxlbmNlIGluIHRoZSBnZW51cyBBem9zcGlyaWxsdW0gYW5kIGFuYWx5c2lzIG9mIHRoZSBmaXJzdCBnZW5vbWUgc2VxdWVuY2Ugb2YgYW4gQXpvc3BpcmlsbHVtIGJyYXNpbGVuc2UgaW50ZWdyYXRpdmUgcGhhZ2U7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz04NjEtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUi1hIHdpZGVzcHJlYWQgc3lzdGVtIHRoYXQgcHJvdmlkZXMgYWNxdWlyZWQgcmVzaXN0YW5jZSBhZ2FpbnN0IHBoYWdlcyBpbiBiYWN0ZXJpYSBhbmQgYXJjaGFlYTsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE4MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbWFsbCBDUklTUFIgUk5BcyBndWlkZSBhbnRpdmlyYWwgZGVmZW5zZSBpbiBwcm9rYXJ5b3RlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjE7ICBjaXRhdGlvbl9wYWdlcz05NjAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBpc2xhbmRzOiB0b29scyBvZiBiYWN0ZXJpYWwgaG9yaXpvbnRhbCBnZW5lIHRyYW5zZmVyIGFuZCBldm9sdXRpb247IEZFTVMgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9Mzc2LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ycm5EQjogZG9jdW1lbnRpbmcgdGhlIG51bWJlciBvZiByUk5BIGFuZCB0Uk5BIGdlbmVzIGluIGJhY3RlcmlhIGFuZCBhcmNoYWVhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1ENDg5LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZml4QUJDWCBnZW5lcyBpbiBSaG9kb3NwaXJpbGx1bSBydWJydW0gZW5jb2RlIGEgcHV0YXRpdmUgbWVtYnJhbmUgY29tcGxleCBwYXJ0aWNpcGF0aW5nIGluIGVsZWN0cm9uIHRyYW5zZmVyIHRvIG5pdHJvZ2VuYXNlOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTIwNTItNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlbGVjdHJvbiB0cmFuc2ZlciBmbGF2b3Byb3RlaW4gZml4QUJDWCBnZW5lIHByb2R1Y3RzIGZyb20gQXpvc3BpcmlsbHVtIGJyYXNpbGVuc2Ugc2hvdyBhIE5pZkEtZGVwZW5kZW50IHByb21vdGVyIHJlZ3VsYXRpb247IEN1cnIuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz0yNjctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZXMgc3ltYmlvc2VzOiBhYnNlbmNlIG9mIE5vZCBnZW5lcyBpbiBwaG90b3N5bnRoZXRpYyBicmFkeXJoaXpvYmlhOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNjsgIGNpdGF0aW9uX3BhZ2VzPTEzMDctMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVzIGZvciBhIG1pY3JvYWVyb2JpY2FsbHkgaW5kdWNlZCBveGlkYXNlIGNvbXBsZXggaW4gQnJhZHlyaGl6b2JpdW0gamFwb25pY3VtIGFyZSBlc3NlbnRpYWwgZm9yIGEgbml0cm9nZW4tZml4aW5nIGVuZG9zeW1iaW9zaXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0EuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MzMwOS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjeXRvY2hyb21lIGNiYjMgKGN5dG9jaHJvbWUgYykgdGVybWluYWwgb3hpZGFzZSBpbiBBem9zcGlyaWxsdW0gYnJhc2lsZW5zZSBTcDcgc3VwcG9ydHMgbWljcm9hZXJvYmljIGdyb3d0aDsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODA7ICBjaXRhdGlvbl9wYWdlcz01Njg5LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHJlZ3VsYXRpb24gb2Ygbml0cm9nZW4gZml4YXRpb24gaW4gcmhpem9iaWE7IE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTM1Mi04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lnbmFsIHRyYW5zZHVjdGlvbiBieSBoZW1lLWNvbnRhaW5pbmcgUEFTLWRvbWFpbiBwcm90ZWluczsgSi4gQXBwbC4gUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz03NzQtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3BlcnRpZXMgb2YgYmFjdGVyaWFsIGVuZG9waHl0ZXMgYW5kIHRoZWlyIHByb3Bvc2VkIHJvbGUgaW4gcGxhbnQgZ3Jvd3RoOyBUcmVuZHMgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQ2My03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umhpem9iaWFsIHN0cmF0ZWdpZXMgdG8gZW5oYW5jZSBzeW1iaW90aWMgaW50ZXJhY3Rpb25zOiByaGl6b2JpdG94aW5lIGFuZCAxLWFtaW5vY3ljbG9wcm9wYW5lLTEtY2FyYm94eWxhdGUgZGVhbWluYXNlOyBNaWNyb2JlcyBFbnZpcm9uLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTk5LTExMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgRE5BIHNlcXVlbmNlcyB0aGF0IHJlZ3VsYXRlIHRoZSBleHByZXNzaW9uIG9mIHRoZSBFbnRlcm9iYWN0ZXIgY2xvYWNhZSBVVjQgMS1hbWlub2N5Y2xvcHJvcGFuZS0xLWNhcmJveHlsaWMgYWNpZCBkZWFtaW5hc2UgZ2VuZTsgQ2FuLiBKLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTE1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmE7IHJlZ3VsYXRpb24gb2YgdGhlIEVudGVyb2JhY3RlciBjbG9hY2FlIFVXNCAxLWFtaW5vY3ljbG9wcm9wYW5lLTEtY2FyYm94eWxhdGUgKEFDQykgZGVhbWluYXNlIGdlbmUgKGFjZFMpOyBDYW4uIEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0zNTYtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCBBQ0MgZGVhbWluYXNlIGFuZCB0aGUgYWxsZXZpYXRpb24gb2YgcGxhbnQgc3RyZXNzOyBBZHYuIEFwcGwuIE1pY3JiaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0zMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bnRoZXNpcyBvZiBwaHl0b2hvcm1vbmVzIGJ5IHBsYW50LWFzc29jaWF0ZWQgYmFjdGVyaWE7IENSQyBDcml0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgYW5hbHlzaXMgb2YgdGhlIEF6b3NwaXJpbGx1bSBicmFzaWxlbnNlIGluZG9sZS0zLXB5cnV2YXRlIGRlY2FyYm94eWxhc2UgZ2VuZSBhbmQgaWRlbnRpZmljYXRpb24gb2YgYSBjaXMtYWN0aW5nIHNlcXVlbmNlIGludm9sdmVkIGluIGF1eGluIHJlc3BvbnNpdmUgZXhwcmVzc2lvbjsgUGxhbnQgTW9sLiBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTMxMS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm9TY2FuOiBwcm90ZWluIGRvbWFpbnMgaWRlbnRpZmllcjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzExNi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VENEQjogdGhlIHRyYW5zcG9ydGVyIGNsYXNzaWZpY2F0aW9uIGRhdGFiYXNlIGZvciBtZW1icmFuZSB0cmFuc3BvcnQgcHJvdGVpbiBhbmFseXNlcyBhbmQgaW5mb3JtYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPUQxODEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9uQi1kZXBlbmRlbnQgdHJhbnMtZW52ZWxvcGUgc2lnbmFsaW5nOiB0aGUgZXhjZXB0aW9uIG9yIHRoZSBydWxlPzsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zNDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkFELW1hbGljIGVuenltZSBhZmZlY3RzIG5pdHJvZ2VuIGZpeGluZyBhY3Rpdml0eSBvZiBCcmFkeXJoaXpvYml1bSBqYXBvbmljdW0gVVNEQSAxMTAgYmFjdGVyb2lkcyBpbiBzb3liZWFuIG5vZHVsZXM7IE1pY3JvYmVzIEVudmlyb24uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjE1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UeXBlIElWIHNlY3JldGlvbiBzeXN0ZW1zOiB0b29scyBvZiBiYWN0ZXJpYWwgaG9yaXpvbnRhbCBnZW5lIHRyYW5zZmVyIGFuZCB2aXJ1bGVuY2U7IENlbGwuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yMzc3LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaG90b3NlbnNpbmcgaW4gY2hlbW90cm9waGljLCBub24tcGhvdG90cm9waGljIGJhY3RlcmlhOiBsZXQgdGhlcmUgYmUgbGlnaHQgc2Vuc2luZyB0b287IFRyZW5kcyBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTU0LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGFpciBvZiBiYWN0ZXJpb3BoeXRvY2hyb21lcyBmcm9tIEFncm9iYWN0ZXJpdW0gdHVtZWZhY2llbnMgYXJlIGhpc3RpZGluZSBraW5hc2VzIHdpdGggb3Bwb3NpbmcgcGhvdG9iaW9sb2dpY2FsIHByb3BlcnRpZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MjgwNy0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW5ndWxhciBiYWN0ZXJpb3BoeXRvY2hyb21lIGFjcXVpcmVkIGJ5IGxhdGVyYWwgZ2VuZSB0cmFuc2ZlcjsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjgyOyAgY2l0YXRpb25fcGFnZXM9NzMyMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHF1b3J1bS1xdWVuY2hpbmcgYXBwcm9hY2ggdG8gaWRlbnRpZnkgcXVvcnVtLXNlbnNpbmctcmVndWxhdGVkIGZ1bmN0aW9ucyBpbiBBem9zcGlyaWxsdW0gbGlwb2ZlcnVtOyBSZXMuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU5OyAgY2l0YXRpb25fcGFnZXM9Njk5LTcwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Ti1hY3lsLWhvbW9zZXJpbmUgbGFjdG9uZS1tZWRpYXRlZCBxdW9ydW0tc2Vuc2luZyBpbiBBem9zcGlyaWxsdW06IGFuIGV4Y2VwdGlvbiByYXRoZXIgdGhhbiBhIHJ1bGU7IEZFTVMgTWljcm9iaW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTE1NS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5ub3RhdGlvbiBvZiB0aGUgcFJoaWNvIHBsYXNtaWQgb2YgQXpvc3BpcmlsbHVtIGJyYXNpbGVuc2UgcmV2ZWFscyBpdHMgcm9sZSBpbiBkZXRlcm1pbmluZyB0aGUgb3V0ZXIgc3VyZmFjZSBjb21wb3NpdGlvbjsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIzMjsgIGNpdGF0aW9uX3BhZ2VzPTE2NS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy8xLzM3Lzk5NDA2Ny9kc3AwMjYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIGRldGVybWluZWQgdGhlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgdGhlIGVudGlyZSBnZW5vbWUgb2YgYSBkaWF6b3Ryb3BoaWMgZW5kb3BoeXRlLCBBem9zcGlyaWxsdW0gc3AuIEI1MTAuIFN0cmFpbiBCNTEwIGlzIGFuIGVuZG9waHl0aWMgYmFjdGVyaXVtIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDcyNDAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2Rk6sJvNeG+ltGXU+fd4ouBbIR9x24=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f37%2f347240" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f37%2f347240" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/1/1/m_cover.gif?Expires=1509238699&amp;Signature=JvFEswwd982sbhJ1P6Lv5ZhVudZbWjiqhvdc2WiswJy~UaerrJlKJqC1szS9yIOkXgrbqsNEdSr~iMaK5OJYLhPgu88WpQz5nciGvdkf9JxmosfiH37dFZpVJ6qIt~bN6jJk3sq9IgjU7MX2hnDqDPl~E2xKYmwCvLpLWDYs81KDYbvn9SBFgRjuAZK-1w-8VCpSdslz9-SQv0NZq5iZp55UJ0zQmeATeEkKAKmCy2QeBxgh4ahFTxB3AZvkvBctC6MoXbaJ2HbdMeHSPo2mW7L2yHg80RmbpAKDcTgmUDX3AZ~KgeBqD0UWtt83qlSemumG9piMfnJdj5eORYoJMg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/1">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4911370">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911370">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911372">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911372">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911377">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911377">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911392">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911392">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911522">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911522">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911524">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911524">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911526">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911526">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4911528">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4911528">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/1/23/347368">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Complete Genomic Structure of the Cultivated Rice Endophyte <em>Azospirillum</em> sp. B510
<i class='icon-availability_free' title='Free' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Takakazu Kaneko</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takakazu Kaneko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Faculty of Engineering</div>, <div class="insititution">Kyoto Sangyo University</div>, <div class="addr-line">Motoyama, Kamigamo, Kita-Ku, Kyoto 603-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Takakazu+Kaneko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kaneko T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kaneko author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Kiwamu Minamisawa</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kiwamu Minamisawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kiwamu+Minamisawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minamisawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Minamisawa author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Tsuyoshi Isawa</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyoshi Isawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Tsuyoshi+Isawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isawa T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Isawa author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hiroki Nakatsukasa</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroki Nakatsukasa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hiroki+Nakatsukasa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakatsukasa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nakatsukasa author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hisayuki Mitsui</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisayuki Mitsui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hisayuki+Mitsui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mitsui H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Mitsui author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Yasuyuki Kawaharada</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuyuki Kawaharada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yasuyuki+Kawaharada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawaharada Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kawaharada author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Yasukazu Nakamura</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasukazu Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yasukazu+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nakamura author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Watanabe author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Kumiko Kawashima</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kumiko Kawashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kumiko+Kawashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawashima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kawashima author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Akiko Ono</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Ono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Akiko+Ono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ono A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Ono author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Yoshimi Shimizu</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshimi Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yoshimi+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Shimizu author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Chika Takahashi</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chika Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Chika+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Takahashi author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Chiharu Minami</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiharu Minami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Chiharu+Minami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minami C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Minami author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Tsunakazu Fujishiro</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsunakazu Fujishiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Tsunakazu+Fujishiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Fujishiro author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kohara author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Midori Katoh</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Katoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Midori+Katoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katoh M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Katoh author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Naomi Nakazaki</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naomi Nakazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Naomi+Nakazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakazaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nakazaki author:N.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nakayama author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Manabu Yamada</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Manabu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yamada author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Tabata author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-438-52-3923. Fax. <div class="fax">+81-438-52-3924</div>. E-mail: <a href="mailto:ssato@kazusa.or.jp" target="_blank">ssato@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Sato author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 1, 1 February 2010, Pages 37–50, <a href='https://doi.org/10.1093/dnares/dsp026'>https://doi.org/10.1093/dnares/dsp026</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 January 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 September 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 December 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347240" href="/dnaresearch/article-pdf/17/1/37/994067/dsp026.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takakazu Kaneko, Kiwamu Minamisawa, Tsuyoshi Isawa, Hiroki Nakatsukasa, Hisayuki Mitsui, Yasuyuki Kawaharada, Yasukazu Nakamura, Akiko Watanabe, Kumiko Kawashima, Akiko Ono, Yoshimi Shimizu, Chika Takahashi, Chiharu Minami, Tsunakazu Fujishiro, Mitsuyo Kohara, Midori Katoh, Naomi Nakazaki, Shinobu Nakayama, Manabu Yamada, Satoshi Tabata, Shusei Sato;  Complete Genomic Structure of the Cultivated Rice Endophyte <em>Azospirillum</em> sp. B510, <em>DNA Research</em>, Volume 17, Issue 1, 1 February 2010, Pages 37–50, <a href='https://doi.org/10.1093/dnares/dsp026'>https://doi.org/10.1093/dnares/dsp026</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347240?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347240?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347240?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347240?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347240?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347240?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347240?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Complete%20Genomic%20Structure%20of%20the%20Cultivated%20Rice%20Endophyte%20%26lt%3Bem%26gt%3BAzospirillum%26lt%3B%2Fem%26gt%3B%20sp.%20B510&amp;publicationDate=2010-01-04&amp;volumeNum=17&amp;issueNum=1&amp;author=Kaneko%2C%20Takakazu%3B%20Minamisawa%2C%20Kiwamu&amp;startPage=37&amp;endPage=50&amp;contentId=10.1093%2Fdnares%2Fdsp026&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f37%2f347240" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4911370 id="4911370" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We determined the nucleotide sequence of the entire genome of a diazotrophic endophyte, <em>Azospirillum</em> sp. B510. Strain B510 is an endophytic bacterium isolated from stems of rice plants (<em>Oryza sativa</em> cv. Nipponbare). The genome of B510 consisted of a single chromosome (3 311 395 bp) and six plasmids, designated as pAB510a (1 455 109 bp), pAB510b (723 779 bp), pAB510c (681 723 bp), pAB510d (628 837 bp), pAB510e (537 299 bp), and pAB510f (261 596 bp). The chromosome bears 2893 potential protein-encoding genes, two sets of rRNA gene clusters (<em>rrns</em>), and 45 tRNA genes representing 37 tRNA species. The genomes of the six plasmids contained a total of 3416 protein-encoding genes, seven sets of <em>rrns</em>, and 34 tRNAs representing 19 tRNA species. Eight genes for plasmid-specific tRNA species are located on either pAB510a or pAB510d. Two out of eight genomic islands are inserted in the plasmids, pAB510b and pAB510e, and one of the islands is inserted into <em>trnfM</em>-CAU in the <em>rrn</em> located on pAB510e. Genes other than the <em>nif</em> gene cluster that are involved in N<sub>2</sub> fixation and are homologues of <em>Bradyrhizobium japonicum</em> USDA110 include <em>fixABCX</em>, <em>fixNOQP</em>, <em>fixHIS</em>, <em>fixG</em>, and <em>fixLJK</em>. Three putative plant hormone-related genes encoding tryptophan 2-monooxytenase (<em>iaaM</em>) and indole-3-acetaldehyde hydrolase (<em>iaaH</em>), which are involved in IAA biosynthesis, and ACC deaminase (<em>acdS</em>), which reduces ethylene levels, were identified. Multiple gene-clusters for tripartite ATP-independent periplasmic-transport systems and a diverse set of malic enzymes were identified, suggesting that B510 utilizes C<sub>4</sub>-dicarboxylate during its symbiotic relationship with the host plant.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Azospirillum"><em>Azospirillum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="endophyte">endophyte</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;rice plant&quot;">rice plant</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;N2 fixation&quot;">N<sub>2</sub> fixation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;plant hormone&quot;">plant hormone</a></div>                            <h2 scrollto-destination=4911372 id="4911372" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Endophytes are microorganisms that are able to colonize the intercellular, and sometimes also intracellular, spaces of plant tissues, without causing apparent damage to the host plant. Gram-positive and Gram-negative bacterial endophytes have been isolated from several tissues in numerous plant species.<sup><a reveal-id="DSP026C1" data-open="DSP026C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP026C2" data-open="DSP026C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Many endophytes have beneficial effects on plant growth and health.<sup><a reveal-id="DSP026C3  DSP026C5" data-open="DSP026C3  DSP026C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> N<sub>2</sub>-fixing bacterial endophytes, such as <em>Herbaspirillum seropedicae</em>, <em>Gluconacetobacter diazotrophicus</em>, and <em>Azoarcus</em> sp., have been found within the tissues of some crops and grasses, and partially contribute to the nitrogen requirement of the host plants.<sup><a reveal-id="DSP026C6" data-open="DSP026C6" class="link link-ref link-reveal xref-bibr">6</a></sup><em>Azoarcus</em> sp. strain BH72, isolated from the salt marsh plant kallar grass, is best studied in terms of the molecular mechanisms of establishment inside plants and endophyte functions.<sup><a reveal-id="DSP026C7" data-open="DSP026C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Krause et al.<sup><a reveal-id="DSP026C8" data-open="DSP026C8" class="link link-ref link-reveal xref-bibr">8</a></sup> reported the first full genome sequence of an endophyte, strain BH72 of <em>Azoarcus</em> species (4.38 Mb), and this sequence provided valuable insights into the life of bacterial endophytes, including information about interactions with host plants. Fouts et al.<sup><a reveal-id="DSP026C9" data-open="DSP026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> also reported the whole genome sequence of a N<sub>2</sub>-fixing endophyte, <em>Klebsiella pneumoniae</em> 342. Comparative genomics of naturally occurring bacterial endophytes provides information that can be used to develop enhanced bacterial endophytes.<sup><a reveal-id="DSP026C10" data-open="DSP026C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>The genus <em>Azospirillum</em> consists of spirillum-shaped, N<sub>2</sub>-fixing, Gram-negative alpha-proteobacteria that often live in the plant rhizosphere.<sup><a reveal-id="DSP026C11" data-open="DSP026C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Since <em>Azospirillum</em> inoculation promotes plant growth, agronomic applications of this genus have been developed.<sup><a reveal-id="DSP026C12" data-open="DSP026C12" class="link link-ref link-reveal xref-bibr">12</a></sup><em>Azospirillum</em> sp. B510 was isolated on 23 August 1999 from the surface-sterilized stems of rice plants (<em>Oryza sativa</em> cv. Nipponbare) that were cultivated in the Kashimadai experimental paddy field of Tohoku University (Miyagi, Japan).<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The B510 strain is closely related to <em>A. oryzae</em> COC8, which was reported as a paddy soil bacterium (with 97.7% identity in their 16S rRNA gene sequences),<sup><a reveal-id="DSP026C14" data-open="DSP026C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and B510 is classified in the same cluster of the phylogenetic tree as <em>A. oryzae</em> COC8 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In addition to being a diazotroph under free-living conditions, B510 was found to have positive motility, and to be capable of degrading plant cell walls.<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Inoculation with <em>Azospirillum</em> sp. B510 was shown to promote plant growth under both laboratory and field conditions (Isawa et al., unpublished results). Specifically, the field experiment in a field in Hokkaido, Japan, indicated that B510 inoculation increases stem number resulting in an increase in seed yield (Isawa et al., unpublished results). Moreover, B510 inoculation enhanced disease resistance to virulent rice blast fungus and the bacterial pathogen <em>Xanthomonas oryzae</em>.<sup><a reveal-id="DSP026C15" data-open="DSP026C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Thus, <em>Azospirillum</em> sp. B510 is likely a beneficial bacterium with agronomic applications.</p><p>In this study, we demonstrated the endophytic characteristics of <em>Azospirillum</em> sp. B510 and its ability to fix N<sub>2</sub><em>in planta</em>. Then, we determined the complete nucleotide sequence of the <em>Azospirillum</em> sp. B510 genome and deduced the gene repertoire in the genome. This is the first report of the genome structure of the genus <em>Azospirillum</em>.</p>                            <h2 scrollto-destination=4911377 id="4911377" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4911378 id="4911378" class="section-title" data-legacy-id=s2a>2.1. A bacterial strain, inoculation of rice plants, and estimation of N<sub>2</sub> fixation ability and of the internal Azospirillum sp. B510 population</h3>
<p><em>Azospirillum</em> sp. B510 is a diazotrophic endophyte that was isolated from the stem of cultivated rice, <em>O. sativa</em> cv. Nipponbare.<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Bacteria were cultured in Nutrient Broth (Difco, Detroit, MI, USA), collected by centrifugation at 5000<em>g</em> for 3 min, and washed twice with sterile saline (0.85% w/v NaCl). The bacterial cell suspension was adjusted to 2 × 10<sup>7</sup> cells ml<sup>−1</sup> in saline solution just before inoculation.<sup><a reveal-id="DSP026C16" data-open="DSP026C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>The hulls of rice seeds were carefully removed using forceps. After the hulled seeds were shaken in 10% (w/v) Ca(OCl)<sub>2</sub> for 30 min at 28°C, they were washed more than three times with sterile distilled water. A surface-sterilized seed was placed in a sterilized test tube (16.5 mm in diameter, 150 mm in height) containing 9 ml of 0.325% (w/v) semi-solid agar solution with the sterilized inorganic nutrients,<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP026C17" data-open="DSP026C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and the tube was covered with an aluminium cap. Each seed was inoculated with a bacterial cell suspension of 1 × 10<sup>7</sup> cells. The rice plants were cultivated at 25°C under long-day conditions (16-h light and 8-h dark) for 10 days in a plant growth cabinet (LH300; NK Systems Co. Ltd, Osaka, Japan) that provided 65 mmol photons m<sup>−2</sup> s<sup>−1</sup> of photosynthetically active radiation.<sup><a reveal-id="DSP026C18" data-open="DSP026C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>To estimate N<sub>2</sub>-fixing activity, acetylene was introduced into test tubes, each containing a 10-day-old rice seedling, and the tubes were enclosed with a sterilized rubber stopper. After a 24-h incubation period, the ethylene concentration was determined by gas chromatography as described previously.<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Internal populations of inoculated bacteria inside rice tissues were estimated as follows. The 10-day-old rice seedlings were sampled from the test tubes. After the seed parts of the seedlings were removed using forceps, the remaining parts of the seedlings were weighed. The parts of seedlings were dipped in 70% (v/v) ethanol and then immersed in 1% NaOCl solution for 30 s. They were then quickly washed three times with sterilized distilled water and then once with sterilized saline solution. After the surface-sterilized plants were aseptically macerated in 1 ml of saline solution using a mortar and pestle, the macerates were serially diluted with saline solution and plated on Nutrient Agar (Difco) plates. After incubation at 30°C for 7 days, colony forming units (CFUs) were determined based on the fresh weight of the rice plants. Simultaneously, uninoculated plants were grown and subjected to CFU determination, as a negative control.</p>                            <h3 scrollto-destination=4911382 id="4911382" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing</h3>
<p>Total cellular DNA was purified according to standard procedures, and three genomic libraries, based on two types of cloning vectors, were constructed for sequencing. The IB5100/1 library contained inserts of ∼3.0 kb cloned into pUC118 (Takara Bio Inc., Japan), the IB5102/3 library contained inserts of ∼4.5 kb cloned into pUC118, and the IB510b library contained inserts with an average size of 58 kb cloned into a BAC vector, pCC1BAC (Epicentre Bio., USA).</p><p>Genome sequencing was performed using the whole-genome shotgun method in combination with BAC end-sequencing. The nucleotide sequences of both ends of the clones from the IB5100/1, IB5102/3, and IB510b libraries were analysed using a Dye-terminator Cycle Sequencing Kit and the 3730XL Sequencer (Applied Biosystems, USA). The end-sequence data from the BAC clones facilitated the gap-closure process and provided the scaffolding for reconstruction of the sequence of the entire genome. We filled the remaining gaps in the sequence by primer walking, using the plasmids or the BAC clones as templates. The integrity of the reconstructed genome sequence was assessed by chromosome walking using the end sequences of the BAC clones.</p>                            <h3 scrollto-destination=4911385 id="4911385" class="section-title" data-legacy-id=s2c>2.3. Gene assignment, annotation, and information analyses</h3>
<p>RNA- and protein-encoding regions were assigned by a combination of computer prediction and similarity searches, as described previously.<sup><a reveal-id="DSP026C19" data-open="DSP026C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>Genes for structural RNAs were identified by similarity searches against an in-house structural RNA database that had been constructed based on data available in GenBank (rel.167). tmRNA-, tRNA-, and rRNA-encoding regions were predicted using the ARAGORN 1.2.20 program,<sup><a reveal-id="DSP026C20" data-open="DSP026C20" class="link link-ref link-reveal xref-bibr">20</a></sup> the tRNA scan-SE 1.23 program,<sup><a reveal-id="DSP026C21" data-open="DSP026C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and the RNAmmer ver.1.2S program,<sup><a reveal-id="DSP026C22" data-open="DSP026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> respectively, in combination with similarity searches.</p><p>The prediction of protein-encoding regions was carried out with the Glimmer 2.13 prediction program.<sup><a reveal-id="DSP026C23" data-open="DSP026C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Prior to prediction, a matrix was generated for the B510 genome by training with a data set of 610 open-reading frames that showed a high degree of sequence similarity to a translated gene set registered as the genomic data for both <em>Magnetospirillum magneticum</em> AMB-1 (accession number AP007255) and <em>Rhodospirillum rubrum</em> ATCC 11170 (CP000230), which are bacteria closely related to <em>Azospirillum</em> species.<sup><a reveal-id="DSP026C24" data-open="DSP026C24" class="link link-ref link-reveal xref-bibr">24</a></sup> All the predicted protein-encoding regions of 150 bp or more were translated into amino acid sequences, which were then subjected to similarity searches against the non-redundant (nr) protein database from NCBI (GenBank database rel. 167.0) using the BLASTP program.<sup><a reveal-id="DSP026C25" data-open="DSP026C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In parallel, all the predicted intergenic sequences were compared with sequences in the nr database using the BLASTX program to identify genes that were not detected by the prediction process. For predicted genes that did not show sequence similarity to known genes, only those equal to or longer than 150 bp were considered candidates.</p><p>To annotate the functions of the assigned genes, the KAAS system, which is based on bi-directional best-hit information from sequence similarity against the KEEG GENES database and on heuristics, was first applied to all predicted protein-encoding genes.<sup><a reveal-id="DSP026C26" data-open="DSP026C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Next, the group escaped from KAAS was deduced based on the sequence similarity of their translated protein products to those of genes of known function and to the protein motifs in the InterPro database (ver. 17.0).<sup><a reveal-id="DSP026C27" data-open="DSP026C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A BLAST score of 10<sup>−5</sup> was considered significant. Assignment of Clusters of Orthologous Groups of proteins (COGs) of predicted gene products was carried out by BLASTP analysis against the COG reference data set (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/COG/">http://www.ncbi.nlm.nih.gov/COG/</a>).<sup><a reveal-id="DSP026C28" data-open="DSP026C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A BLAST <em>E</em>-value of less than 10<sup>−10</sup> was considered significant. After filtering, COG assignments of the putative gene products were generated according to COG identification, using the best-hit pair in the reference data set.</p><p>Comparison between two genomic nucleotide sequences was performed using GenomeMatcher V.1.270.<sup><a reveal-id="DSP026C29" data-open="DSP026C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The GC-skew analysis was performed as described by Lobry.<sup><a reveal-id="DSP026C30" data-open="DSP026C30" class="link link-ref link-reveal xref-bibr">30</a></sup><em>Phage_Finder</em> ver. 4.6 was used to detect the prophage region inserted into the B510 genome.<sup><a reveal-id="DSP026C31" data-open="DSP026C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>The FtsK Orienting Polar Sequences (KOPS) motif is specifically oriented toward the replication terminus of the genomic sequences in alpha-proteobacteria.<sup><a reveal-id="DSP026C32" data-open="DSP026C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The cumulative distribution of the KOPS sequence patterns (GGGNAGGG) was calculated along each replicon of B510, and the distribution of these patterns in the genome was plotted. Multicopy DNA elements of longer than 500 bp that have the capacity to encode a putative transposase were identified as insertion sequences (ISs), using the BLAST2 program, and then classified using RECON1.05<sup><a reveal-id="DSP026C33" data-open="DSP026C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and the IS finder (<a class="link link-uri" href="http://www-is.biotoul.fr">www-is.biotoul.fr</a>).</p>                            <h2 scrollto-destination=4911392 id="4911392" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4911393 id="4911393" class="section-title" data-legacy-id=s3a>3.1. Colonization and N<sub>2</sub> fixation in rice plants</h3>
<p>The internal population of B510 was evaluated using surface-sterilized rice seedlings and the plate counting method. We calculated that there were 1.5–5.7 × 10<sup>4</sup> CFU g<sup>−1</sup> fresh weight of inoculated seedlings. In contrast, no colonies were detected in uninoculated rice plants. These data indicate that B510 cells colonized internal rice tissues, although the colonization level was lower than that reported for other endophytes, including <em>Herbaspirillum</em> sp. B501 (∼10<sup>6</sup> CFU g<sup>−1</sup> fresh weight).<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP026C34" data-open="DSP026C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Indeed, Yasuda et al.<sup><a reveal-id="DSP026C15" data-open="DSP026C15" class="link link-ref link-reveal xref-bibr">15</a></sup> also observed colonization of <em>Azospirillum</em> sp. B510 around the basal parts of shoots of cv. Nipponbare using <em>gusA</em>-tagged B510.</p><p>To evaluate the <em>in planta</em> N<sub>2</sub>-fixing activity of <em>Azospirillum</em> sp. B510, acetylene reduction activity was assayed using rice seedlings inoculated or not with the bacterium. In the presence of acetylene, the seedlings inoculated with <em>Azospirillum</em> sp. B510 showed marked acetylene reduction activity compared with the activity in uninoculated plants and in plants inoculated in air (control) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Acetylene reduction activity of <em>Azospirillum</em> sp. B510 <em>in planta</em> (43 nmol h<sup>−1</sup> g<sup>−1</sup> fresh weight) was similar to that of <em>Herbaspirillum</em> sp. B501 (67 nmol h<sup>−1</sup> g<sup>−1</sup> fresh weight), a well-characterized N<sub>2</sub>-fixing endophyte isolated from rice plants.<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP026C17" data-open="DSP026C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP026C34" data-open="DSP026C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=4911396 id="4911396" class="section-title" data-legacy-id=s3b>3.2. Sequencing and structural features of the genome</h3>
<p>The nucleotide sequence of the entire B510 genome was deduced initially by assembling a total of 66 554 sequence files, which corresponded to approximately six genome equivalents, according to the method described in the Materials and methods section. To ensure that the nucleotide sequence was sufficiently accurate for further analysis of gene structure and function, finishing was carried out by visually editing the draft sequences and by additional sequencing to close the gaps. The genome of B510 consists of a single chromosome and six circular plasmids designated as pAB510a, pAB510b, pAB510c, pAB510d, pAB510e, and pAB510f. The total size of the genome is 7 599 738 bp, and the average GC content is 67.6%. The size and the percentage of GC content of each replicon are summarized in Table <a reveal-id="DSP026TB1" data-open="DSP026TB1" class="link link-reveal link-table xref-fig">1</a>. The integrity of 99.9% of the final genome sequence was assessed by comparing the insert length of anchored BAC clones with the computed distance between the end sequences of the clones. The integrity of the remaining region (334 299–341 799 nt on the chromosome) where no BAC clone was anchored was confirmed using the sequence and insert length information of the plasmid clones.</p>                        <a id="4911398" scrollto-destination="4911398"></a>
<div class="table-wrap"><div id="DSP026TB1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Features of replicons in <em>Azospirillum</em> sp. B510 genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Chromosome </th><th>pAB510a </th><th>pAB510b </th><th>pAB510c </th><th>pAB510d </th><th>pAB510e </th><th>pAB510f </th></tr></thead><tbody><tr><td>Size (bp) </td><td>3 311 395 </td><td>1 455 109 </td><td>723 779 </td><td>681 723 </td><td>628 837 </td><td>537 299 </td><td>261 596 </td></tr><tr><td>G + C content (%) </td><td>67.8 </td><td>67.6 </td><td>67.5 </td><td>67.4 </td><td>68.0 </td><td>67.5 </td><td>65.9 </td></tr><tr><td>Prophage </td><td>2 </td><td>ND </td><td>ND </td><td>ND </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td>Genomic island </td><td>6 </td><td>ND </td><td>1 </td><td>ND </td><td>ND </td><td>1 </td><td>ND </td></tr><tr><td>tRNA genes </td><td>45 </td><td>14 </td><td>2 </td><td>3 </td><td>6 </td><td>9 </td><td>ND </td></tr><tr><td>rRNA genes<sup>a</sup> </td><td>2 (<em>rrn1,2</em>) </td><td>4 (<em>rrn4,5,6,7</em>) </td><td>1 (<em>rrn8</em>) </td><td>1 (<em>rrn9</em>) </td><td>ND </td><td>1 (<em>rrn3</em>) </td><td>ND </td></tr><tr><td>Protein genes </td><td>2893 </td><td>1131 </td><td>631 </td><td>533 </td><td>519 </td><td>415 </td><td>187 </td></tr><tr><td>COG assignment<sup>b</sup> </td><td>2020 </td><td>896 </td><td>525 </td><td>441 </td><td>389 </td><td>309 </td><td>138 </td></tr><tr><td>Not in COGs </td><td>873 </td><td>235 </td><td>106 </td><td>92 </td><td>130 </td><td>106 </td><td>49 </td></tr></tbody></table></div><div content-id="DSP026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Chromosome </th><th>pAB510a </th><th>pAB510b </th><th>pAB510c </th><th>pAB510d </th><th>pAB510e </th><th>pAB510f </th></tr></thead><tbody><tr><td>Size (bp) </td><td>3 311 395 </td><td>1 455 109 </td><td>723 779 </td><td>681 723 </td><td>628 837 </td><td>537 299 </td><td>261 596 </td></tr><tr><td>G + C content (%) </td><td>67.8 </td><td>67.6 </td><td>67.5 </td><td>67.4 </td><td>68.0 </td><td>67.5 </td><td>65.9 </td></tr><tr><td>Prophage </td><td>2 </td><td>ND </td><td>ND </td><td>ND </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td>Genomic island </td><td>6 </td><td>ND </td><td>1 </td><td>ND </td><td>ND </td><td>1 </td><td>ND </td></tr><tr><td>tRNA genes </td><td>45 </td><td>14 </td><td>2 </td><td>3 </td><td>6 </td><td>9 </td><td>ND </td></tr><tr><td>rRNA genes<sup>a</sup> </td><td>2 (<em>rrn1,2</em>) </td><td>4 (<em>rrn4,5,6,7</em>) </td><td>1 (<em>rrn8</em>) </td><td>1 (<em>rrn9</em>) </td><td>ND </td><td>1 (<em>rrn3</em>) </td><td>ND </td></tr><tr><td>Protein genes </td><td>2893 </td><td>1131 </td><td>631 </td><td>533 </td><td>519 </td><td>415 </td><td>187 </td></tr><tr><td>COG assignment<sup>b</sup> </td><td>2020 </td><td>896 </td><td>525 </td><td>441 </td><td>389 </td><td>309 </td><td>138 </td></tr><tr><td>Not in COGs </td><td>873 </td><td>235 </td><td>106 </td><td>92 </td><td>130 </td><td>106 </td><td>49 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>ND means ‘not identified’.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The parenthetic references show IDs for the rRNA gene cluster (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The numbers of genes classified into 19 COG categories except for ones in ‘function unknown’ are shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4911398" target="_blank">View Large</a></div></div><p>The nucleotide position was numbered from one nucleotide upstream of the predicted ATG start codon, based on the predicted translational initiation site of the homologue of <em>hemE</em> (AZL028930) in the chromosome. Nucleotide positions for the plasmids were assigned based on the predicted translational initiation site of AZLa11310 in pAB510a, AZLb06310 in pAB510b, AZLc05330 in pAB510c, AZLe04150 in pAB510e, AZLf01870 in pAB510f, and the termination of AZLd05190 in pAB510d, respectively.</p><p>Ten <em>Azospirillum</em> species have been examined for their genome composition, using pulsed-field gel electrophoresis.<sup><a reveal-id="DSP026C35" data-open="DSP026C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Multiple replicons were identified in 10 <em>Azospirillum</em> species as with B510. However, the chromosome size of each <em>Azospirillum</em> strain (&lt;2.7 Mb) was smaller than one of B510 (3.3 Mb). Linear plasmids were detected in several <em>Azospirillum</em> strains, such as <em>A. brasilense</em> and <em>A. lipoferum</em>,<sup><a reveal-id="DSP026C35" data-open="DSP026C35" class="link link-ref link-reveal xref-bibr">35</a></sup> but similar structural units were not found in the B510 genome.</p>                            <h3 scrollto-destination=4911401 id="4911401" class="section-title" data-legacy-id=s3c>3.3. Structural features of the genome</h3>
                            <h4 scrollto-destination=4911402 id="4911402" class="section-title" data-legacy-id=s3c1>3.3.1. Putative replication origin</h4>
<p>A GC-skew analysis was performed to locate the probable origin of DNA replication. We established that the shift of GC-skew values occurred in two regions of the chromosome, at coordinates 35 and 1710 kb, as shown in Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a> (the innermost circle). The <em>hemE</em> locus, which is known to associate with the origins of replication in alpha-proteobacteria,<sup><a reveal-id="DSP026C36" data-open="DSP026C36" class="link link-ref link-reveal xref-bibr">36</a></sup> was found to be adjacent to the shift point of the GC skew. A cluster of nine genes, <em>rho</em>–hypothetical–<em>hemH</em>–<em>hemE</em>–hypothetical–<em>maf</em>–<em>aroE</em>–<em>coaE</em>–<em>dnaQ</em> (AZL028900–AZL028930–AZL000010– AZL000050: these codes are hereinafter defined in the Protein-encoding genes section), occurring at ∼0 kb on the B510 chromosome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>) was commonly found in the <em>Magnetospirillum</em> sp. AMB-1 genome.<sup><a reveal-id="DSP026C37" data-open="DSP026C37" class="link link-ref link-reveal xref-bibr">37</a></sup><em>parA</em> (AZL000140) and <em>parB</em> (AZL000150) were found downstream of <em>dnaQ</em> (AZL000050; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). These findings strongly suggest that the <em>ori</em> region of the chromosome is located between AZL028930 and AZL000010. This location of the <em>ori</em> is also supported by the KOPS motif distribution analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>).</p>                        <a id="4911404" scrollto-destination="4911404"></a>
<div id="DSP026F1" data-id="DSP026F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/m_dsp02601.jpeg?Expires=1509238699&Signature=R0D~vdBugDo5gQj8X5j5KUrQCFF5Q6c75dlR6KxdecHbf~2AcKn9~5r4bSTgPybROX5MH7QPW7cwOP2-ufi4WZab6APHaoPv8hxndO83qVFcDA9OIcude4ZZkP0ZwFd1QVRivYcIvYMRXQQXS5m6hrgfEM8ebLf2rWRfWchsvbgVr19T9LF6Yv6zw~ftfyGIbx3fBUM91tcppo~NDtPw1v0RiwZ~~6ui1MbEZwBqxq6aAW0ZOivrNhauutRJ9cG2srfHmIWimehJZmTJiEwqb92f8UhtI2mPdUf06hmy2DvEio4cY-Tnx0StCZAfgf5WXbdOlfhKfJGWIqvuArvZUg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic representation of seven circular replicons in the Azospirillum sp. B510 genome. The scale for all plasmids is the same, and the scale for the chromosome is one-half that of the plasmids. The scale indicates the location (in kb) outside the map. The bars in the two outer circles, the outermost circle and the second circle, show the positions of the putative protein-encoding genes in clockwise and counter-clockwise directions, respectively. The putative genes are represented by 25 colours, based on COG assignments (Supplementary Fig. S2). In the third circles from the outside, positions of structural RNA genes are indicated by black (tRNAs) and red (rRNAs) bars. In the fourth circles from the outside, the red bars indicate the positions of ISs, and the pale-green and blue areas show the insertion of prophages and genomic islands, respectively. The innermost and second circles from the centre show the GC-skew values (yellow and purple) and the average GC percent (blue and red), respectively, calculated using a window-size of 10 kb. The scales for GC percent are presented on the second circles. Top and bottom of each scale are shown, as follows: 73.9/58.9% in the chromosome, 74.6/57.0% in pAB510a, 72.5/60.7% in pAB510b, 72.0/60.2% in pAB510c, 74.9/48.3% in pAB510d, 71.9/59.0% in pAB510e, and 71.1/52.7% in pAB510f." path-from-xml="dsp02601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4911404/dsp02601.jpeg" path-from-xml="dsp02601.jpeg" target="_blank">View large</a><a section="4911404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp02601.jpeg?Expires=1509238699&Signature=MDSeUuh5mFMsGxGd6HarY83nMN-CV-1LOrhGdP-rUEt0ENQO9dir5wTVe9E8Kojcm-vQZGa5GvwYyvpEam6LENiDi~rAX1HY7KG0SFeN-8DQ45bmajXCnFPnoZxQhIiD8w~HDVJAyMA0osYQ-PxPYQv-W~Rv9Acl5AfQ7FuiQYGvlOoYttaQ3of2tu8dzVPVkwoEfUiyYCUwVqgTTdcgu7FZZAnsGKAvHWeTKNuKUckrX63tKYXUJ0T6g64aIvvC9mRNgTC0KQqEVvxNbE~kuzLdWe2k5TITK3iEuxjHSeuxsVkVvrOxrNmXXna6IbCDxfucEigS~PH~5jIS~h7v8w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4911404&ar=347240&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of seven circular replicons in the <em>Azospirillum</em> sp. B510 genome. The scale for all plasmids is the same, and the scale for the chromosome is one-half that of the plasmids. The scale indicates the location (in kb) outside the map. The bars in the two outer circles, the outermost circle and the second circle, show the positions of the putative protein-encoding genes in clockwise and counter-clockwise directions, respectively. The putative genes are represented by 25 colours, based on COG assignments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). In the third circles from the outside, positions of structural RNA genes are indicated by black (tRNAs) and red (rRNAs) bars. In the fourth circles from the outside, the red bars indicate the positions of ISs, and the pale-green and blue areas show the insertion of prophages and genomic islands, respectively. The innermost and second circles from the centre show the GC-skew values (yellow and purple) and the average GC percent (blue and red), respectively, calculated using a window-size of 10 kb. The scales for GC percent are presented on the second circles. Top and bottom of each scale are shown, as follows: 73.9/58.9% in the chromosome, 74.6/57.0% in pAB510a, 72.5/60.7% in pAB510b, 72.0/60.2% in pAB510c, 74.9/48.3% in pAB510d, 71.9/59.0% in pAB510e, and 71.1/52.7% in pAB510f.</p></div></div><div content-id="DSP026F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/m_dsp02601.jpeg?Expires=1509238699&Signature=R0D~vdBugDo5gQj8X5j5KUrQCFF5Q6c75dlR6KxdecHbf~2AcKn9~5r4bSTgPybROX5MH7QPW7cwOP2-ufi4WZab6APHaoPv8hxndO83qVFcDA9OIcude4ZZkP0ZwFd1QVRivYcIvYMRXQQXS5m6hrgfEM8ebLf2rWRfWchsvbgVr19T9LF6Yv6zw~ftfyGIbx3fBUM91tcppo~NDtPw1v0RiwZ~~6ui1MbEZwBqxq6aAW0ZOivrNhauutRJ9cG2srfHmIWimehJZmTJiEwqb92f8UhtI2mPdUf06hmy2DvEio4cY-Tnx0StCZAfgf5WXbdOlfhKfJGWIqvuArvZUg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic representation of seven circular replicons in the Azospirillum sp. B510 genome. The scale for all plasmids is the same, and the scale for the chromosome is one-half that of the plasmids. The scale indicates the location (in kb) outside the map. The bars in the two outer circles, the outermost circle and the second circle, show the positions of the putative protein-encoding genes in clockwise and counter-clockwise directions, respectively. The putative genes are represented by 25 colours, based on COG assignments (Supplementary Fig. S2). In the third circles from the outside, positions of structural RNA genes are indicated by black (tRNAs) and red (rRNAs) bars. In the fourth circles from the outside, the red bars indicate the positions of ISs, and the pale-green and blue areas show the insertion of prophages and genomic islands, respectively. The innermost and second circles from the centre show the GC-skew values (yellow and purple) and the average GC percent (blue and red), respectively, calculated using a window-size of 10 kb. The scales for GC percent are presented on the second circles. Top and bottom of each scale are shown, as follows: 73.9/58.9% in the chromosome, 74.6/57.0% in pAB510a, 72.5/60.7% in pAB510b, 72.0/60.2% in pAB510c, 74.9/48.3% in pAB510d, 71.9/59.0% in pAB510e, and 71.1/52.7% in pAB510f." path-from-xml="dsp02601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4911404/dsp02601.jpeg" path-from-xml="dsp02601.jpeg" target="_blank">View large</a><a section="4911404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp02601.jpeg?Expires=1509238699&Signature=MDSeUuh5mFMsGxGd6HarY83nMN-CV-1LOrhGdP-rUEt0ENQO9dir5wTVe9E8Kojcm-vQZGa5GvwYyvpEam6LENiDi~rAX1HY7KG0SFeN-8DQ45bmajXCnFPnoZxQhIiD8w~HDVJAyMA0osYQ-PxPYQv-W~Rv9Acl5AfQ7FuiQYGvlOoYttaQ3of2tu8dzVPVkwoEfUiyYCUwVqgTTdcgu7FZZAnsGKAvHWeTKNuKUckrX63tKYXUJ0T6g64aIvvC9mRNgTC0KQqEVvxNbE~kuzLdWe2k5TITK3iEuxjHSeuxsVkVvrOxrNmXXna6IbCDxfucEigS~PH~5jIS~h7v8w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4911404&ar=347240&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of seven circular replicons in the <em>Azospirillum</em> sp. B510 genome. The scale for all plasmids is the same, and the scale for the chromosome is one-half that of the plasmids. The scale indicates the location (in kb) outside the map. The bars in the two outer circles, the outermost circle and the second circle, show the positions of the putative protein-encoding genes in clockwise and counter-clockwise directions, respectively. The putative genes are represented by 25 colours, based on COG assignments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). In the third circles from the outside, positions of structural RNA genes are indicated by black (tRNAs) and red (rRNAs) bars. In the fourth circles from the outside, the red bars indicate the positions of ISs, and the pale-green and blue areas show the insertion of prophages and genomic islands, respectively. The innermost and second circles from the centre show the GC-skew values (yellow and purple) and the average GC percent (blue and red), respectively, calculated using a window-size of 10 kb. The scales for GC percent are presented on the second circles. Top and bottom of each scale are shown, as follows: 73.9/58.9% in the chromosome, 74.6/57.0% in pAB510a, 72.5/60.7% in pAB510b, 72.0/60.2% in pAB510c, 74.9/48.3% in pAB510d, 71.9/59.0% in pAB510e, and 71.1/52.7% in pAB510f.</p></div></div><p>The shift of the GC skew was clearly observed for two of the six plasmids, pAB510a and pAB510d (Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>). The predicted origins of both of these plasmids were located at around 0 kb. The KOPS motif distribution analysis also indicates that these regions are the origins of replication (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). These regions accommodate <em>repA-parAB</em> genes (AZLa00020, AZLa00030, AZLa00040) in pAB510a and <em>repB</em> (AZLd00010) in pAB510d. However, no <em>repC</em> candidate was found in the B510 genome, suggesting that the two B510 plasmids, which contain typical origin of replication regions, may have distinct, unknown initiators of replication.</p>                            <h4 scrollto-destination=4911406 id="4911406" class="section-title" data-legacy-id=s3c2>3.3.2. Mobile DNA elements</h4>
<p>ISs are small mobile DNA elements capable of transposition via a self-encoded transposase and can be classified into various families based on their structure.<sup><a reveal-id="DSP026C38" data-open="DSP026C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Two-hundred and eighty IS copies were assigned in all seven replication units of the B510 genome. On the basis of the type of transposase present, these ISs could be classified into 29 groups of 12 families (<a class="link link-uri" href="http://www-is.biotoul.fr">www-is.biotoul.fr</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). As shown in Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>, they are rather evenly distributed among the replicons, although there are several regions where a disproportionate number of ISs are located within the replicon. The remarkable frequency of ISs was occurred in pAB510f, in which a 33-kb region corresponding to 12.6% of the replicon is occupied by 30 ISs classified into 19 groups.</p><p>It is remarkable that 27 out of 280 ISs were located from 3164 to 3208 kb, between CRISPER-like sequences (described later) and one of the rRNA gene clusters on the chromosome (Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>). Thirty-one ISs classified into 18 groups were also found between 80 and 157 kb on pAB510b (Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>). The partial segments (AZL027610 and AZLb01330) of a gene similar to recombinase were identified at both ends of the above-mentioned high-density IS regions. This feature suggests that these regions may behave as a single large mobile genetic element.</p>                            <h4 scrollto-destination=4911409 id="4911409" class="section-title" data-legacy-id=s3c3>3.3.3. ‘Phages’ and ‘a defense system’</h4>
<p>The <em>Phage_Finder</em> program detected two independent putative prophages with their putative <em>att</em> sites at coordinates 915901–982641 (B510PP01) and 2490727–2522517 (B510PP02) on the <em>Azospirillum</em> sp. B510 chromosome (Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>). The predicted sequences of <em>att</em> sites were assigned as 11 bases long (CAAGGCCGCCG) at both termini of B510PP01 and 12 bases (GCTGGGCGGCGGC) in B510PP02. The <em>attL</em> sites of these putative prophages (B510PP01 and B510PP02) were located near the potential genes encoding integrases AZL008670 and AZL022160, respectively. Moreover, a recombinase homologue (AZL008660) was found upstream of the AZL008670 integrase of B510PP01. Both B510PP01 and B510PP02 contained terminases (AZL008420 and AZL022110), capsid proteins (AZL008390, AZL008410, and AZL022100), and tail proteins (AZL008340 and AZL022050). These genetic traits found in two regions, B510PP01 and B510PP02, are characteristic of a horizontally transferred element.</p><p>A putative duplicated sequence of the B510PP01 prophage (at coordinates 915901–982641) was identified at coordinates 2296582–2356862 on the chromosome (Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>). A detailed comparison between these two regions showed that this phage-like element lacked regions corresponding to both termini, namely a 4.8-kb region on the <em>attL</em> side that includes the integrase gene (AZL008670), and a 1.7-kb region on the <em>attR</em> side. The presence of this putative cryptic phage with deletions may indicate that the B510 cell was subjected to multiple infection events during the course of evolution.</p><p>The release of phage particles from B510 cells is induced by mitomycin C treatment.<sup><a reveal-id="DSP026C39" data-open="DSP026C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The DNA in the released phage was estimated to be ∼10 kb in size. In the present study, however, we could not find a prophage-like region of this size in the entire B510 genome. A possible explanation for this is that diverse prophages in the genome were overlooked during the computer identification.</p><p>A search of the genome sequence by GenomeMatcher (ver.1.282) detected a large number of copies of short sequences at the coordinates 3158606–3163849 on the chromosome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4A and B</a>). This region exhibited a structure typical of CRISPR (Clustered Regularly Interspaced Short Palindromic Repeat) systems and is composed of 70 different spacers (36–42 bp), an identical repeat element (37 bp) containing an 8-bp palindrome (5′-CCTGGGCG), and <em>cas</em> genes encoding CRISPR-associated proteins (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S4C and S5</a>).<sup><a reveal-id="DSP026C40" data-open="DSP026C40" class="link link-ref link-reveal xref-bibr">40</a></sup> When the whole genome sequence of B510 was subjected to CRISPRFinder (<a class="link link-uri" href="http://crispr.u-psud.fr/crispr">http://crispr.u-psud.fr/crispr</a>), four additional CRISPR-like regions without complete <em>cas</em> genes were found exclusively at the coordinates 1016560–1018180, 1178059–1178606, 3035919–3038413, and 3199247–3200867 on the chromosome. Recently, it was reported that CRISPRs act as an antiphage defense system; bacteria integrate new spacers that are derived from phage genomic sequences during CRISPR-mediated phage resistance.<sup><a reveal-id="DSP026C40" data-open="DSP026C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSP026C41" data-open="DSP026C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p>                            <h4 scrollto-destination=4911414 id="4911414" class="section-title" data-legacy-id=s3c4>3.3.4. Genomic islands</h4>
<p>Genomic islands are regions that form syntenic groups consisting of multiple accessory genes, and they are inserted into the genome through horizontal transfer.<sup><a reveal-id="DSP026C42" data-open="DSP026C42" class="link link-ref link-reveal xref-bibr">42</a></sup> They are often inserted into tRNA genes in various bacterial genomes.<sup><a reveal-id="DSP026C42" data-open="DSP026C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Duplicated portions of tRNA genes, which are a typical feature of genomic islands, were found at eight locations in the B510 genome (Table <a reveal-id="DSP026TB1" data-open="DSP026TB1" class="link link-reveal link-table xref-fig">1</a>). These duplicated portions were separated from the corresponding genes by DNA segments ranging in size from 6.7 to 71 kb (B510GI01-06, d7, e8; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). This suggests that genomic islands were inserted into the tRNA genes of the B510 genome. ‘<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>’ summarizes the species of tRNA genes, their positions, the length of duplications, and the DNA regions that separate the perfect or near-perfect duplicated tRNA gene segments, which are 20–48 bp long. The presumptive genomic islands varied in size and contained putative genes for an integrase or a site-specific recombinase at one of their termini (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The GC content of these elements (53–67%) was lower than the average GC content of the chromosome (67.8%). As this type of nucleotide bias is a general feature of genomic islands,<sup><a reveal-id="DSP026C42" data-open="DSP026C42" class="link link-ref link-reveal xref-bibr">42</a></sup> the lower GC content of the presumptive genomic islands strongly suggests the horizontal origin of these elements.</p>                            <h3 scrollto-destination=4911416 id="4911416" class="section-title" data-legacy-id=s3d>3.4. Aspects of predicted genes and their organization</h3>
                            <h4 scrollto-destination=4911417 id="4911417" class="section-title" data-legacy-id=s3d1>3.4.1. RNA-encoding genes</h4>
<p>A total of 79 tRNA genes, representing 45 tRNA species, were identified (Fig. <a reveal-id="DSP026F1" data-open="DSP026F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsp026/DC1" target="_blank">Table S5</a>). These tRNA genes were dispersed throughout the genome, but did not occur in pAB510f (Table <a reveal-id="DSP026TB1" data-open="DSP026TB1" class="link link-reveal link-table xref-fig">1</a>). Among them, 37 tRNA genes are likely to be transcribed as a single unit, whereas 20 (representing <em>trnA</em>-UGC, <em>trnI</em>-GAU, and <em>trnfM</em>-CAU) are likely to form a common transcriptional unit consisting of eight rRNA gene clusters, and the remaining 22 are tandemly arranged into nine clusters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Duplicated gene sets for <em>trnK</em>, <em>trnT</em>, <em>trnP</em>, and <em>trnE</em> were arrayed in pairs as complete copies. Thirty-four of the 79 tRNA genes are encoded by plasmids (Table <a reveal-id="DSP026TB1" data-open="DSP026TB1" class="link link-reveal link-table xref-fig">1</a>). They were classified into 19 tRNA species, eight of which are specific to pAB510a or pAB510d (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>), indicating that these plasmids are essential.</p><p>Nine <em>rrns</em> with a form of 16S-<em>trnA</em>-<em>trnI</em>-23S-5S were assigned in the genome, two of which are on the chromosome (<em>rrn1</em> and <em>rrn2</em>) and seven on the plasmids (<em>rrn3</em>, <em>rrn4</em>, <em>rrn5</em>, <em>rrn6</em>, <em>rrn7</em>, <em>rrn8</em>, and <em>rrn9</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The presence of nine <em>rrns</em> shows not to be typical copy number as alpha-proteobacteria genome, because alpha-proteobacteria generally possess five or less copies of <em>rrn</em>, whereas gamma-proteobacteria and firmicutes possess nine or more copies (<a class="link link-uri" href="http://ribosome.mmg.msu.edu/rrndb/">http://ribosome.mmg.msu.edu/rrndb/</a>).<sup><a reveal-id="DSP026C43" data-open="DSP026C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>Of the five <em>rrn</em>-gene clusters, four (<em>rrn2</em>, <em>rrn3</em>, <em>rrn6</em>, and <em>rrn9</em>) were followed by <em>trnfM</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). A genome island (B510GIe8) was inserted in <em>trnfM</em> in <em>rrn3</em> located on pAB510e. One of the clusters lacking <em>trnfM</em> (<em>rrn5</em>) had a deletion of 522 nucleotides long, containing <em>trnA</em> and <em>trnI</em>, as well as a truncation of the 3′ region that encoded the 5S rRNA in addition to <em>trnfM</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>).</p><p>A gene-encoding transfer-messenger RNA (<em>ssrA</em>: AZLr028), which is known to be involved in the degradation of aberrantly synthesized proteins, was found in the B510 genome. Putative genes for two types of small RNAs, the B subunit of RNase P (<em>rnpB</em>: AZLr029), and the signal recognition particle (SRP) RNA (<em>ffs</em>: AZLr040) were also identified in the B510 genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h4 scrollto-destination=4911424 id="4911424" class="section-title" data-legacy-id=s3d2>3.4.2. Protein-encoding genes</h4>
<p>The potential protein-encoding regions were assigned using a combination of computer prediction using the Glimmer program and a similarity search, as described in the Materials and methods section. By taking into account sequence similarity to known genes and relative position of predicted encoding region, to avoid overlaps, the total number of putative protein-encoding genes assigned to the genome was 6309 (Table <a reveal-id="DSP026TB1" data-open="DSP026TB1" class="link link-reveal link-table xref-fig">1</a>). The average gene density was estimated as being one gene in every 1205 bp. The putative protein-encoding genes that start with ATG, GTG, TTG, or ATT codons were denoted by a serial number with the prefix ‘AZL’, representing the previous species name of this bacterium, <em>Azospirillum lipoferum</em>. It should be noted that the putative gene assignment used in this paper represents coding potential based on a defined set of assumptions.</p><p>We assigned functions to the 6309 potential protein-encoding genes by performing similarity searches, as described in the Materials and methods section. Seventy-five percent (4750) of the genes exhibited sequence similarity to genes of known function, and 25% (1559) showed sequence similarity to hypothetical genes.</p><p>We generated COG assignments of the translated gene products by conducting a BLASTP search against the COG reference data set. A total of 4718 putative gene products were classified into 23 COG categories, excluding gene products of unknown function, as shown in Table <a reveal-id="DSP026TB1" data-open="DSP026TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>.</p>                            <h3 scrollto-destination=4911433 id="4911433" class="section-title" data-legacy-id=s3e>3.5. Characteristic features of the protein-encoding genes</h3>
                            <h4 scrollto-destination=4911436 id="4911436" class="section-title" data-legacy-id=s3e1>3.5.1. Genes involved in N<sub>2</sub> fixation</h4>
<p>B510 is a bacterium capable of performing N<sub>2</sub> fixation in a modified Rennie semi-solid medium.<sup><a reveal-id="DSP026C13" data-open="DSP026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Furthermore, we demonstrated its ability to fix N<sub>2</sub><em>in planta</em> in this study (see the Colonization and N<sub>2</sub> fixation in rice plants section). The genes encoding the nitrogenase core and assembly proteins were separately clustered in three loci on the B510 chromosome. These three gene clusters were arranged in <em>nifA</em> [1 gene]–<em>nifB</em> [3 genes]–<em>nifZ</em> [1 gene]–<em>nifST</em> (AZL022440–AZL022530), <em>nifWV</em> [1 gene]–<em>nifSU</em> (AZL006520–AZL006560), and <em>nifHDK</em> [2 genes]–<em>nifENX</em> (AZL007710–AZL007640; Fig. <a reveal-id="DSP026F2" data-open="DSP026F2" class="link link-reveal link-table xref-fig">2</a>A). <em>nifQ</em> (AZL010780) was located at a distance from above other <em>nif</em> gene clusters. The <em>fixABCX</em> (AZL006470–AZL006500) cluster, which encoded components participating in the transport of electrons to nitrogenase,<sup><a reveal-id="DSP026C44" data-open="DSP026C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSP026C45" data-open="DSP026C45" class="link link-ref link-reveal xref-bibr">45</a></sup> was adjacent to the <em>nifWVSU</em> cluster and in an opposite orientation (Fig. <a reveal-id="DSP026F2" data-open="DSP026F2" class="link link-reveal link-table xref-fig">2</a>A). Not all of the major components of N<sub>2</sub> fixation were encoded by genes on the chromosome; AZLc04520, which is a homologue of <em>nifJ</em> encoding pyruvate–flavodoxin oxidoreductase, was solely found on the pAB510c plasmid. However, considering that a homologue of <em>nifF</em>, which encodes flavodoxin, an electron donor of NifJ, was not found in the B510 genome, AZLc04520 may not be involved in the major pathway for electron transfer to nitrogenase in B510.</p>                        <a id="4911441" scrollto-destination="4911441"></a>
<div id="DSP026F2" data-id="DSP026F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/m_dsp02602.jpeg?Expires=1509238700&Signature=NWcoAMIrXo8CnyFVumLmaiFQ14ujgSiOPZl5v8hqCN~lybnDbptYbbporWUF~BwG0IPVufMIafy5Wt8LspQ8NQOrlCEUvA5otfQyfBTecVW0vFqZX38yZQ9A~elhzz626RKxF-5OtO2g0SM0lXMMAqDDglchEekroOfo7kbhUFXH2~KN7j6q7kQk6Kv66qiSzAq1Cg0sW2PNtS-njejYttbWw6T0g3mfMi3Z0lqQLfNCoWaWI6596Gm8Vcj3BzU7BCmmA~4lNwA0fOCR3N9eKoelMz~5WRN~JkswR1j~KfO5LB-wqNeOA28aRJf~Ok6p4ht2vLIQo4WYZAghvVWrzA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic diagram of the N2-fixation gene clusters from Azospirillum sp. B510 and two Bradyrhizobium species. (A) nif gene cluster of Azospirillum sp. B510, Bradyrhizobium sp. ORS278, and Bradyrhizobium japonicum USDA110. The conservation of the gene arrangement of nif/fix is represented as boxes, with arrowheads indicating the direction of transcription. nif genes are surrounded by red boxes and are in capital letters. fix genes are in blue, lowercase font. Ferredoxin genes are shaded yellow. Black horizontal bars indicate the scale. The position on the chromosomal sequence is indicated in kb below both termini of the bars. The sequences were analysed using the tblastx function of GenomeMatcher V.1.270. The BLAST E-value of 10−10 was considered significant for the assignment of relationship. The bars connecting the maps represent close matches, with colour indicating the degree of amino acid identity determined in the BLAST alignment. The colour code is displayed at the bottom of the figure. (B) fix clusters of three bacteria. A box with a yellow border indicates that the gene cluster is located on the plasmid." path-from-xml="dsp02602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4911441/dsp02602.jpeg" path-from-xml="dsp02602.jpeg" target="_blank">View large</a><a section="4911441" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp02602.jpeg?Expires=1509238700&Signature=MdBmZSAFD6WzURZjW7XEbKkXC6qjhxFad97mfT1sU~CnkoaBHqym7mpLvbCnoSCsf8EVt4x9SG5wtG3h98A-WzT~iuwuNRgKE5WH~qUfXLGcs6O1soYb28vfHk6jFOZMqCC8915-NfDo91doBNhJMWRUiL85ebfoQxIczU4xneJEShsESCFyNOBEw4j9VZO7WcDq21HUAQRFRVmcWBfj94j48N6PiaWoXBOPnWxhtEatVnYDmRNlATkXiNI6hP2l2amgQPDHmAylVjs49p7EFwid-a~--zRFI-xvKvmXuEYWGwQe0hUWU8BZARftWVNv27mmo3BhViNLlOgktMlpZg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4911441&ar=347240&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the N<sub>2</sub>-fixation gene clusters from <em>Azospirillum</em> sp. B510 and two <em>Bradyrhizobium</em> species. (A) <em>nif</em> gene cluster of <em>Azospirillum</em> sp. B510, <em>Bradyrhizobium</em> sp. ORS278, and <em>Bradyrhizobium japonicum</em> USDA110. The conservation of the gene arrangement of <em>nif/fix</em> is represented as boxes, with arrowheads indicating the direction of transcription. <em>nif</em> genes are surrounded by red boxes and are in capital letters. <em>fix</em> genes are in blue, lowercase font. Ferredoxin genes are shaded yellow. Black horizontal bars indicate the scale. The position on the chromosomal sequence is indicated in kb below both termini of the bars. The sequences were analysed using the tblastx function of GenomeMatcher V.1.270. The BLAST <em>E</em>-value of 10<sup>−10</sup> was considered significant for the assignment of relationship. The bars connecting the maps represent close matches, with colour indicating the degree of amino acid identity determined in the BLAST alignment. The colour code is displayed at the bottom of the figure. (B) <em>fix</em> clusters of three bacteria. A box with a yellow border indicates that the gene cluster is located on the plasmid.</p></div></div><div content-id="DSP026F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/m_dsp02602.jpeg?Expires=1509238700&Signature=NWcoAMIrXo8CnyFVumLmaiFQ14ujgSiOPZl5v8hqCN~lybnDbptYbbporWUF~BwG0IPVufMIafy5Wt8LspQ8NQOrlCEUvA5otfQyfBTecVW0vFqZX38yZQ9A~elhzz626RKxF-5OtO2g0SM0lXMMAqDDglchEekroOfo7kbhUFXH2~KN7j6q7kQk6Kv66qiSzAq1Cg0sW2PNtS-njejYttbWw6T0g3mfMi3Z0lqQLfNCoWaWI6596Gm8Vcj3BzU7BCmmA~4lNwA0fOCR3N9eKoelMz~5WRN~JkswR1j~KfO5LB-wqNeOA28aRJf~Ok6p4ht2vLIQo4WYZAghvVWrzA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic diagram of the N2-fixation gene clusters from Azospirillum sp. B510 and two Bradyrhizobium species. (A) nif gene cluster of Azospirillum sp. B510, Bradyrhizobium sp. ORS278, and Bradyrhizobium japonicum USDA110. The conservation of the gene arrangement of nif/fix is represented as boxes, with arrowheads indicating the direction of transcription. nif genes are surrounded by red boxes and are in capital letters. fix genes are in blue, lowercase font. Ferredoxin genes are shaded yellow. Black horizontal bars indicate the scale. The position on the chromosomal sequence is indicated in kb below both termini of the bars. The sequences were analysed using the tblastx function of GenomeMatcher V.1.270. The BLAST E-value of 10−10 was considered significant for the assignment of relationship. The bars connecting the maps represent close matches, with colour indicating the degree of amino acid identity determined in the BLAST alignment. The colour code is displayed at the bottom of the figure. (B) fix clusters of three bacteria. A box with a yellow border indicates that the gene cluster is located on the plasmid." path-from-xml="dsp02602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4911441/dsp02602.jpeg" path-from-xml="dsp02602.jpeg" target="_blank">View large</a><a section="4911441" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp02602.jpeg?Expires=1509238700&Signature=MdBmZSAFD6WzURZjW7XEbKkXC6qjhxFad97mfT1sU~CnkoaBHqym7mpLvbCnoSCsf8EVt4x9SG5wtG3h98A-WzT~iuwuNRgKE5WH~qUfXLGcs6O1soYb28vfHk6jFOZMqCC8915-NfDo91doBNhJMWRUiL85ebfoQxIczU4xneJEShsESCFyNOBEw4j9VZO7WcDq21HUAQRFRVmcWBfj94j48N6PiaWoXBOPnWxhtEatVnYDmRNlATkXiNI6hP2l2amgQPDHmAylVjs49p7EFwid-a~--zRFI-xvKvmXuEYWGwQe0hUWU8BZARftWVNv27mmo3BhViNLlOgktMlpZg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4911441&ar=347240&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the N<sub>2</sub>-fixation gene clusters from <em>Azospirillum</em> sp. B510 and two <em>Bradyrhizobium</em> species. (A) <em>nif</em> gene cluster of <em>Azospirillum</em> sp. B510, <em>Bradyrhizobium</em> sp. ORS278, and <em>Bradyrhizobium japonicum</em> USDA110. The conservation of the gene arrangement of <em>nif/fix</em> is represented as boxes, with arrowheads indicating the direction of transcription. <em>nif</em> genes are surrounded by red boxes and are in capital letters. <em>fix</em> genes are in blue, lowercase font. Ferredoxin genes are shaded yellow. Black horizontal bars indicate the scale. The position on the chromosomal sequence is indicated in kb below both termini of the bars. The sequences were analysed using the tblastx function of GenomeMatcher V.1.270. The BLAST <em>E</em>-value of 10<sup>−10</sup> was considered significant for the assignment of relationship. The bars connecting the maps represent close matches, with colour indicating the degree of amino acid identity determined in the BLAST alignment. The colour code is displayed at the bottom of the figure. (B) <em>fix</em> clusters of three bacteria. A box with a yellow border indicates that the gene cluster is located on the plasmid.</p></div></div><p>The structures of the three <em>nif/fix</em> gene clusters in B510 were mostly conserved with those of <em>Bradyrhizobium</em> sp. ORS278, which forms nodules on stems of aquatic legumes (Fig. <a reveal-id="DSP026F2" data-open="DSP026F2" class="link link-reveal link-table xref-fig">2</a>A).<sup><a reveal-id="DSP026C46" data-open="DSP026C46" class="link link-ref link-reveal xref-bibr">46</a></sup> However, B510 has the following distinctive features: the inversion of <em>fixABCX</em>, the insertion of two other predicted genes (AZL007670 and AZL007680) into a region between <em>nifHDK</em> and <em>nifENX</em>, and the transposition of <em>nifQ</em> from these clusters (Fig. <a reveal-id="DSP026F2" data-open="DSP026F2" class="link link-reveal link-table xref-fig">2</a>A). These features may reflect the high frequency of genome rearrangements in B510.</p><p>Cytochrome <em>c</em> terminal oxidase is an enzyme essential for N<sub>2</sub> fixation in rhizobial species.<sup><a reveal-id="DSP026C47" data-open="DSP026C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The oxidase components and the cation pump that is involved in oxidase activity are encoded in rhizobia by <em>fixNOQP</em> and <em>fixGHIS</em>, respectively. These gene clusters were located in tandem on the chromosome of B510 (AZL003350–AZL003410), although <em>fixG</em> was missing from the gene cluster (Fig. <a reveal-id="DSP026F2" data-open="DSP026F2" class="link link-reveal link-table xref-fig">2</a>B). These gene clusters were located upstream of the <em>fixK</em> homologue (AZL003420: a probable transcriptional regulator). The <em>fixG</em> genes were present in triplicate and located at distinct positions on the chromosome (AZL016980), pAB510b (AZLb04870), and pAB510e (AZLe02910). Cytochrome <em>c</em> terminal oxidase is partially responsible for the N<sub>2</sub>-fixing capacity during microaerobic respiration in <em>A. brasilense</em>.<sup><a reveal-id="DSP026C48" data-open="DSP026C48" class="link link-ref link-reveal xref-bibr">48</a></sup> It is possible that cytochrome <em>c</em> terminal oxidase plays a similar role in B510, although the N<sub>2</sub>-fixing capacity of B510 during microaerobic respiration has not yet been investigated.</p><p>A two-component regulatory system, <em>fixLJ</em>, is required to activate the expression of the <em>nifA</em> and <em>fixK</em> regulators of N<sub>2</sub> fixation in rhizobia.<sup><a reveal-id="DSP026C49" data-open="DSP026C49" class="link link-ref link-reveal xref-bibr">49</a></sup> We identified a gene cluster in pAB510b that included <em>fixLJK</em> (AZLb06310, AZLb06300, and AZLb06280; Fig. <a reveal-id="DSP026F2" data-open="DSP026F2" class="link link-reveal link-table xref-fig">2</a>B). The domain organization of FixL (AZLb06310) in B510 was more similar to that of <em>Bradyrhizobium japonicum</em> (bll2760) than to that of <em>Sinorhizobium meliloti</em> (SMa1229).<sup><a reveal-id="DSP026C50" data-open="DSP026C50" class="link link-ref link-reveal xref-bibr">50</a></sup> The genome structures of two diazotrophic endophytes, <em>Azoarcus</em> sp. BH72 and <em>K. pneumoniae</em> Kp342, have been reported,<sup><a reveal-id="DSP026C8" data-open="DSP026C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSP026C9" data-open="DSP026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and the gene clusters encoding nitrogenase components in these genomes did not contain <em>fixL</em>. These findings suggest that the mechanism of N<sub>2</sub> fixation in B510 is similar to that in rhizobia and differs from that in two diverse endophytes, <em>Azoarcus</em> sp. BH72 and <em>K. pneumoniae</em> Kp342.</p>                            <h4 scrollto-destination=4911455 id="4911455" class="section-title" data-legacy-id=s3e2>3.5.2. Plant hormone-related genes</h4>
<p>Plant-associated bacteria, such as endophytes and rhizosphere bacteria, often produce plant hormones or modulate their functions by producing inhibitors for phytohormone biosynthesis.<sup><a reveal-id="DSP026C51" data-open="DSP026C51" class="link link-ref link-reveal xref-bibr">51</a></sup> In particular, the modulation of plant ethylene levels by bacterial 1-aminocyclopropane-1-carboxylate (ACC) deaminase interferes with the physiological functions of the host plant.<sup><a reveal-id="DSP026C51" data-open="DSP026C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSP026C52" data-open="DSP026C52" class="link link-ref link-reveal xref-bibr">52</a></sup> B510 possesses the <em>acdS</em> gene (AZLb04170), which encodes ACC deaminase, and the <em>acdR</em> gene (AZLb04180: leucine-responsible regulatory protein; LRP), which regulates the expression of <em>acdS</em> via its putative LRP binding site, on the pAB510b plasmid (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>).<sup><a reveal-id="DSP026C53" data-open="DSP026C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSP026C54" data-open="DSP026C54" class="link link-ref link-reveal xref-bibr">54</a></sup> It is possible that B510 produces ACC deaminase in response to plants, which then reduces the level of ethylene in plants, and which might promote plant growth and alleviate signs of environmental stress.<sup><a reveal-id="DSP026C51" data-open="DSP026C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSP026C55" data-open="DSP026C55" class="link link-ref link-reveal xref-bibr">55</a></sup> The <em>acdS</em> gene was not present in the previously reported genomes of <em>Azoarcus</em> sp. BH72 and <em>K. pneumoniae</em> Kp342.<sup><a reveal-id="DSP026C8" data-open="DSP026C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSP026C9" data-open="DSP026C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>Biosynthesis of indole-3-actic acid (IAA) is widespread among plant-associated bacteria and has a beneficial impact on plant growth.<sup><a reveal-id="DSP026C56" data-open="DSP026C56" class="link link-ref link-reveal xref-bibr">56</a></sup> In azospirilla and enterobacteria, IAA is generally synthesized from tryptophan via indole-3-pyruvic acid (IPyA). A key gene in this process is <em>ipdC</em>, which encodes indole-3-pyruvate decarboxylase (IPDC). No <em>ipdC</em> homologue was found in the B510 genome, irrespective of the presence of <em>ipdC</em> in the genome of the endophyte <em>K. pneumoniae</em> Kp342 (GKROPF_B2055)<sup><a reveal-id="DSP026C9" data-open="DSP026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and an IAA-inducible <em>ipdC</em> gene in rhizosphere bacterium <em>A. brasilense</em>.<sup><a reveal-id="DSP026C57" data-open="DSP026C57" class="link link-ref link-reveal xref-bibr">57</a></sup> We could not find an <em>iaaC</em> homologue in B510 either, even though the <em>iaaC</em> gene is co-transcribed with the <em>ipdC</em> gene in <em>A. brasilense</em>.<sup><a reveal-id="DSP026C57" data-open="DSP026C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Nevertheless, another pathway for IAA biosynthesis could exist in B510. The IAA biosynthetic pathway catalyses the decarboxylation of tryptophan into indole-3-actamide (IAM) and the hydrolysis of IAM to produce IAA. In <em>Agrobacterium tumefaciens</em> and <em>Pseudomonas syringae</em>, the <em>iaaM</em> and <em>iaaH</em> genes are known to be involved in these steps.<sup><a reveal-id="DSP026C56" data-open="DSP026C56" class="link link-ref link-reveal xref-bibr">56</a></sup> In B510, AZLb03560 and AZLb03580 were assigned as candidates for these genes that participate in the IAM pathway.</p>                            <h4 scrollto-destination=4911473 id="4911473" class="section-title" data-legacy-id=s3e3>3.5.3. Genes encoding transporters and iron-transport-related proteins</h4>
<p>Transporters play indispensable roles in various cellular processes including the delivery of nutrients, elimination of waste products, and adaptation to environmental conditions. A variety of transporters are also thought to be involved in the symbiotic exchange of nutrients between host plant cells and intracellular rhizobia in symbiosis.</p><p>We identified a total of 354 possible transporters based on sequence similarity to known transporters and an InterPro search.<sup><a reveal-id="DSP026C58" data-open="DSP026C58" class="link link-ref link-reveal xref-bibr">58</a></sup> These genes were subsequently classified into 62 families according to the transporter classification system used in Transport Classification Database (<a class="link link-uri" href="http://www.tcdb.org/">http://www.tcdb.org/</a>).<sup><a reveal-id="DSP026C59" data-open="DSP026C59" class="link link-ref link-reveal xref-bibr">59</a></sup> The gene composition of genes in these gene families in B510 is more similar to those in rhizobial species than in two other N<sub>2</sub>-fixing endophytes, <em>Azoarcus</em> sp. BH72 and <em>K. pneumoniae</em> Kp342 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Two of the families identified, the ATP-binding cassette (ABC) superfamily and the major facilitator superfamily (MFS), were composed of a large number of paralogs (141 and 57, respectively) that accounted for a significant portion of the transporter genes in the B510 genome. Although this feature is common among the rhizobia and the endophyte, there are significantly more ABC and MFS genes in B510 genome than in <em>Azoarcus</em> sp. BH72 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Moreover, members of the TC families, which include the Monovalent Cation (K<sup>+</sup> or Na<sup>+</sup>):Proton Antiporter-3 (CPA3) family and the Tricarboxylate Transporter (TTT) Family (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>), were found in B510 and rhizobia, but not in the <em>Azoarcus</em> sp. BH72 genome.</p><p>TonB-dependent outer-membrane proteins, which are responsible for the specific uptake of ferric-siderophore complexes, are crucial for the perception of environmental signals and are associated with plant pathogens.<sup><a reveal-id="DSP026C60" data-open="DSP026C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Strain B510 possesses one set of <em>tonB</em>/<em>exbB</em>/<em>exbD</em> accompanied with anti-sigma and ECF sigma factor genes on pAB510d (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>).<sup><a reveal-id="DSP026C60" data-open="DSP026C60" class="link link-ref link-reveal xref-bibr">60</a></sup> In contrast, B510 carried nine genes encoding TonB-dependent outer-membrane proteins on three replicons, pAB510d, the chromosome, and pAB510a (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The number of iron transport proteins in the B510 genome was not as high as that in other N<sub>2</sub>-fixing endophytes, such as <em>Azoarcus</em> sp. BH72 (22 genes), or in the rhizosphere bacterium, <em>Pseudomonas fluorescens</em> pf5 (45 genes). A homology search suggested that putative receptors for ferrioxamine, vitamin B12, hemin, and heme exist in B510. Because strain B510 produces siderophore (Kawaharada et al., unpublished results), the iron transport systems of B510 may be active in the bacterium in free-living and endophytic environments.</p><p>A C<sub>4</sub>-dicarboxylate, such as malate, is considered to be the major energy and carbon source of <em>Rhizobium</em> during symbiosis with host plants. The transport systems and metabolic enzymes of C<sub>4</sub>-dicarboxylates in <em>Rhizobium</em> are essential for their symbioses. With regard to the transport of carbon from the source to B510 during symbiosis, we identified five complete sets of gene cluster composed of three genes encoding the tripartite ATP-independent periplasmic-transport (TRAP-T) system (TC# 2.A.56) that transports C<sub>4</sub>-dicarboxylate (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). One of the gene cluster for TRAP-T system, AZLa08570, AZLa08580, and AZLa08600, is located upstream of a gene cluster containing citrate lyase subunits (AZLa08510, AZLa08500, AZLa08490, and AZLa08480) and a gene for malic enzyme (AZLa08520). Since citrate lyase works in citrate metabolic pathway under anoxic conditions in <em>K. pneumoniae</em> and <em>Escherichia coli</em>, it might allow B510 growing utilizing citrate as the energy source <em>in planta</em>. Malic enzymes are responsible for the respiratory metabolism in the N<sub>2</sub> fixation as occurs during rhizobial symbiotic relationships with plants.<sup><a reveal-id="DSP026C61" data-open="DSP026C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The genome sequence of B510 revealed the presence of three genes encoding malic enzymes (AZL000200, AZLa05660, and AZLa08520). In a phylogeny, including genes from rhizobia and other endophytes, plasmid-encoding AZLa05660 and AZLa08520 are classified in a group that includes DME (NAD<sup>+</sup>-dependent malic enzyme), and AZL000200 is in a cluster harbouring TME (NADP<sup>+</sup>-dependent malic enzyme; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). Consequently, these findings suggest that a system for uptake and metabolism of C<sub>4</sub> dicarboxylic acids exists in B510, as is the case in rhizobia, and this system may contribute a part of the carbon utilizations during the symbiosis.</p>                            <h4 scrollto-destination=4911504 id="4911504" class="section-title" data-legacy-id=s3e4>3.5.4. Genes for the type IV secretion system</h4>
<p>The B510 genome harboured a gene cluster with significant sequence similarity to that encoding the type IV secretion system (T4SS). Within this cluster was a gene encoding a putative coupling protein TraG/VirD4 (AZL011550), followed by the genes encoding components of secretion machinery TrbB/VirB11, TrbE/ViB4, TrbL/VirB6, TrbG/VirB9, and TrbI/VirB10 (AZL011560, AZL011580, AZL011600, AZL011620, and AZL011630, respectively). It is likely that this region is part of a genomic island, since an integrase gene (AZL011700) is coded in a region adjacent to this cluster; however, tRNA gene sequences were not detected in the border regions. The absence of genes encoding the other necessary components of T4SS might suggest that the genes in the cluster are not functional, but are remnants of genomic rearrangements. Alternatively, it is possible that there are functional homologues with less sequence similarity to prototypical T4SS proteins in the B510 genome.<sup><a reveal-id="DSP026C62" data-open="DSP026C62" class="link link-ref link-reveal xref-bibr">62</a></sup></p>                            <h4 scrollto-destination=4911512 id="4911512" class="section-title" data-legacy-id=s3e5>3.5.5. Genes involved in the photoresponse</h4>
<p>Plant phytochromes are one of the photoreceptors that enable plants to adapt their growth and development to the light environment. The phytochromes are also widely distributed in diverse prokaryotes, including cyanobacteria and proteobacteria.<sup><a reveal-id="DSP026C63" data-open="DSP026C63" class="link link-ref link-reveal xref-bibr">63</a></sup> The B510 genome has two genes, AZL019550 and AZLa05830, which encode distinct types of bacterial phytochromes, such as those found in the plant-tumour inducing bacterium <em>A. tumefaciens</em> and in the photosynthetic bradyrhizobium, <em>Bradyrhizobium</em> sp. ORS278.<sup><a reveal-id="DSP026C64" data-open="DSP026C64" class="link link-ref link-reveal xref-bibr">64</a>,<a reveal-id="DSP026C65" data-open="DSP026C65" class="link link-ref link-reveal xref-bibr">65</a></sup> The translated amino acid sequence of the plasmid-encoded AZLa05830 protein, which shows significant similarity (47% amino acid identity) with AtBphP1 (Atu1990) in <em>A. tumefaciens</em>, is composed of 751 amino acid residues and contains domains that are typical of bacterial phytochromes (PAS-GAF-phytochrome-HisK). A gene encoding a response regulator (AZLa05820) in a CheY superfamily is located downstream of AZLa05830, suggesting that this gene is involved in signal transduction. In contrast, AZL019550 encodes a polypeptide of 853 amino acid residues that contains ‘PAS-GAF-phytochrome-HisK-RR’ domains. The translated gene product had 52% amino acid identity with that of BrBphP2 (BRADO2008) of <em>Bradyrhizobium</em> sp. ORS278. These two candidates for the light-sensing proteins may involve in the signalling pathway, although the actual function of these phytochromes remains to be analysed in B510.</p>                            <h4 scrollto-destination=4911514 id="4911514" class="section-title" data-legacy-id=s3e6>3.5.6. Quorum-sensing</h4>
<p>Quorum-sensing regulation in several strains of <em>A. lipoferum</em> modulates functions related to rhizosphere competence and adaptation, such as pectinase activity, siderophore synthesis, and IAA production.<sup><a reveal-id="DSP026C66" data-open="DSP026C66" class="link link-ref link-reveal xref-bibr">66</a></sup> B510 produces acyl homoserine lactone (AHL) molecules in culture<sup><a reveal-id="DSP026C67" data-open="DSP026C67" class="link link-ref link-reveal xref-bibr">67</a></sup> and carries genes related to AHL synthase (AZLa05890), AHL acylase (AZL013430), and AHL efflux protein (AZLd00800). A LuxR family transcriptional regulator was not found in the vicinity of AHL synthase gene (AZLa05890), although there were 22 <em>luxR</em> family transcriptional regulators on the genome.</p>                            <h4 scrollto-destination=4911516 id="4911516" class="section-title" data-legacy-id=s3e7>3.5.7. Motility</h4>
<p>Most endophytes are motile, because the endophytes systemically spread into the plant from infection sites.<sup><a reveal-id="DSP026C2" data-open="DSP026C2" class="link link-ref link-reveal xref-bibr">2</a></sup> We found that B510 carried at least two sets of genes that encode proteins involved in flagella assembly (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). Thirty one flagella-related genes were clustered on the pAB510e plasmid (AZLe00750–AZLe01230), whereas 25 and 15 of flagella-related genes were present on the chromosome and pAB510a, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). Thirty redundant genes with roles in chemotaxis (e.g. <em>cheAWR</em>) or that encoded methyl-accepting chemotaxis proteins were found on the chromosome, pAB510a, pAB510c, pAB510d, and pAB510e.</p>                            <h4 scrollto-destination=4911518 id="4911518" class="section-title" data-legacy-id=s3e8>3.5.8. Comparison with the pRhico plasmid</h4>
<p><em>Azospirillum brasilense</em> strain Sp7 has a pRhico plasmid that is responsible for the interaction with plant roots. pRhico has been incompletely sequenced with five gaps, and the presence of 32 genes encoding enzymes involved in surface polysaccharide biosynthesis has been reported.<sup><a reveal-id="DSP026C68" data-open="DSP026C68" class="link link-ref link-reveal xref-bibr">68</a></sup> To examine the structural similarity between the pRhico plasmid and the B510 genome, a similarity search with the pRhico sequences as a query was conducted against the entire B510 genome sequence, using the BLAST program, with a threshold <em>E</em>-value of 10<sup>−10</sup>.</p><p>The search detected significant alignments between the B510 genome and 43 regions of pRhico, 23 of which were mapped on pAB510f. One of these sequences incompletely matched 881 nucleotides inside the putative replication protein gene (<em>repA</em>: AZLf00010) of pAB510f and one matched 138 nucleotides upstream of <em>repA</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>), and 17 mapped to regions on pAB510f that coincided with 11 genes that were presumably involved in cell envelope biosynthesis and outer membrane constitution (AZLf00040, AZLf00050, AZLf00060, AZLf00080, AZLf01050, AZLf01060, AZLf01640, AZLf01790, AZLf01050, and AZLf01810; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). The results suggest an evolutionary link and some functional relationship between pRhico and pAB510f. However, remaining portions of pRhico, such as those containing <em>exoC</em> and tRNA genes, were scattered throughout the B510 genome and most of the sequences were poorly conserved, indicating that drastic genome rearrangements had taken place since the two species diverged (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>).</p><p>The sequence information is available in public DNA databases (DDBJ/GenBank/EMBL), under the following accession numbers: AP010946 for the B510 chromosome, AP010947 for pAB510a, AP010948 for pAB510b, AP010949 for pAB510c, AP010950 for pAB510d, AP010951 for pAB510e, and AP010952 for pAB510f. The nucleotide sequences of the genome and annotations describing the predicted genes are available online at RhizoBase (<a class="link link-uri" href="http://genome.kazusa.or.jp/rhizobase">http://genome.kazusa.or.jp/rhizobase</a>).</p>                            <h2 scrollto-destination=4911522 id="4911522" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4911524 id="4911524" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation. This work was supported in part by Special Coordination Funds for Promoting Science and Technology and in part by a Grant-in-Aid for Scientific Research on Priority Areas ‘Comparative Genomics’ from the Ministry of Education, Culture, Sports, Science, and Technology of Japan, and in part by a grant from the Ministry of Agriculture, Forestry and Fisheries of Japan (Genomics for Agricultural Innovation, PMI-0002). Funding for open access charge: The Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=4911526 id="4911526" class="backacknowledgements-title" >Acknowledgement</h2>
<p>We are grateful to F. Wisniewski-Dyé (University Claude Bernard Lyon 1) for valuable comments concerning genes for quorum-sensing.</p>                            <h2 scrollto-destination=4911528 id="4911528" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsp026c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hallmann</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Quadt-Hallmann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mahaffee</div> <div class="given-names">W.F.</div></div>,  <div class="name"><div class="surname">Kloepper</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Bacterial endophytes in agricultural crops</div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">895</div>-<div class="lpage">914</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20endophytes%20in%20agricultural%20crops&amp;author=J.%20Hallmann&amp;author=A.%20Quadt-Hallmann&amp;author=W.F.%20Mahaffee&amp;author=J.W.%20Kloepper&amp;publication_year=1997&amp;journal=Can.%20J.%20Microbiol.&amp;volume=43&amp;pages=895-914" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/m97-131" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fm97-131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fm97-131"> </span></div> </div></div></div></div></div><div content-id="dsp026c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elbeltagy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nishioka</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Isolation and characterization of endophytic bacteria from wild and traditionally cultivated rice varieties</div>, <div class="source">Soil Sci. Plant Nutr.</div> , <div class="year">2000</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">617</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20endophytic%20bacteria%20from%20wild%20and%20traditionally%20cultivated%20rice%20varieties&amp;author=A.%20Elbeltagy&amp;author=K.%20Nishioka&amp;author=H.%20Suzuki&amp;publication_year=2000&amp;journal=Soil%20Sci.%20Plant%20Nutr.&amp;volume=46&amp;pages=617-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/00380768.2000.10409127" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00380768.2000.10409127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F00380768.2000.10409127"> </span></div> </div></div></div></div></div><div content-id="dsp026c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sturz</div> <div class="given-names">A.V.</div></div>,  <div class="name"><div class="surname">Christie</div> <div class="given-names">B.R.</div></div>,  <div class="name"><div class="surname">Nowak</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Bacterial endophytes: potential role in developing sustainable systems of crop production</div>, <div class="source">CRC Crit. Rev. Plant Sci.</div> , <div class="year">2000</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20endophytes%3A%20potential%20role%20in%20developing%20sustainable%20systems%20of%20crop%20production&amp;author=A.V.%20Sturz&amp;author=B.R.%20Christie&amp;author=J.%20Nowak&amp;publication_year=2000&amp;journal=CRC%20Crit.%20Rev.%20Plant%20Sci.&amp;volume=19&amp;pages=1-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0735-2689(01)80001-0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0735-2689(01)80001-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0735-2689(01)80001-0"> </span></div> </div></div></div></div></div><div content-id="dsp026c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mano</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Morisaki</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Endophytic bacteria in the rice plant</div>, <div class="source">Microbes Environ.</div> , <div class="year">2008</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">109</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endophytic%20bacteria%20in%20the%20rice%20plant&amp;author=H.%20Mano&amp;author=H.%20Morisaki&amp;publication_year=2008&amp;journal=Microbes%20Environ.&amp;volume=23&amp;pages=109-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.23.109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.23.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.23.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21558696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ezura</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Microbial community analysis of the phytosphere using culture-independent methodologies</div>, <div class="source">Microbes Environ.</div> , <div class="year">2007</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">93</div>-<div class="lpage">105</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20community%20analysis%20of%20the%20phytosphere%20using%20culture-independent%20methodologies&amp;author=A.%20Saito&amp;author=S.%20Ikeda&amp;author=H.%20Ezura&amp;author=K.%20Minamisawa&amp;publication_year=2007&amp;journal=Microbes%20Environ.&amp;volume=22&amp;pages=93-105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.22.93" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.22.93" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.22.93"> </span></div> </div></div></div></div></div><div content-id="dsp026c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reinhold-Hurek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hurek</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Life in grasses: diazotrophic endophytes</div>, <div class="source">Trends Microbiol.</div> , <div class="year">1998</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">139</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Life%20in%20grasses%3A%20diazotrophic%20endophytes&amp;author=B.%20Reinhold-Hurek&amp;author=T.%20Hurek&amp;publication_year=1998&amp;journal=Trends%20Microbiol.&amp;volume=4&amp;pages=139-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(98)01229-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(98)01229-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(98)01229-3"> </span></div> </div></div></div></div></div><div content-id="dsp026c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hurek</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Reinhold-Hurek</div> <div class="given-names">B.</div></div>. </div><div class="article-title"><em>Azoarcus</em> sp. BH72 as a model for nitrogen-fixing grass endophytes</div>, <div class="source">J. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">169</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Azoarcus%20sp.%20BH72%20as%20a%20model%20for%20nitrogen-fixing%20grass%20endophytes&amp;author=T.%20Hurek&amp;author=B.%20Reinhold-Hurek&amp;publication_year=2003&amp;journal=J.%20Biotechnol.&amp;volume=106&amp;pages=169-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jbiotec.2003.07.010" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jbiotec.2003.07.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jbiotec.2003.07.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14651859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krause</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ramakumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bartels</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Complete genome of the mutualistic N<sub>2</sub>-fixing grass endophyte <em>Azoarcus</em> sp. strain BH72</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1385</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20of%20the%20mutualistic%20N2-fixing%20grass%20endophyte%20Azoarcus%20sp.%20strain%20BH72&amp;author=A.%20Krause&amp;author=A.%20Ramakumar&amp;author=D.%20Bartels&amp;publication_year=2006&amp;journal=Nat.%20Biotechnol.&amp;volume=24&amp;pages=1385-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1243" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17057704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fouts</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Tyler</div> <div class="given-names">H.L.</div></div>,  <div class="name"><div class="surname">DeBoy</div> <div class="given-names">R.T.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the N<sub>2</sub>-foxong broad host range endophyte <em>Klebsiella pneumoniae</em> 342 and virulence predictions verified in mice</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e10000141</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20N2-foxong%20broad%20host%20range%20endophyte%20Klebsiella%20pneumoniae%20342%20and%20virulence%20predictions%20verified%20in%20mice&amp;author=D.E.%20Fouts&amp;author=H.L.%20Tyler&amp;author=R.T.%20DeBoy&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e10000141" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000141" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000141"> </span></div> </div></div></div></div></div><div content-id="dsp026c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A milestone for endophyte biotechnology</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1357</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20milestone%20for%20endophyte%20biotechnology&amp;author=K.%20Minamisawa&amp;publication_year=2006&amp;journal=Nat.%20Biotechnol.&amp;volume=24&amp;pages=1357-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1106-1357" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1106-1357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1106-1357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17093479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hartman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Baldani</div> <div class="given-names">J.I.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Dworkin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Flaknow</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rosenberg</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Schleifer</div> <div class="given-names">K-H.</div></div>,  <div class="name"><div class="surname">Stackerbrandt</div> <div class="given-names">E.</div></div>. </div><div class="article-title">The genus <em>Azospirillum</em></div>, <div class="source">The Prokaryotes, vol. 5</div> , <div class="year">2006</div><div class="edition">3rd edition</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">115</div>-<div class="lpage">40</div>)</div></div></div></div><div content-id="dsp026c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okon</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Labandera-Gonzalez</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Agronomic applications of <em>Azospirillum</em>: an evaluation of 20 years worldwide field inoculation</div>, <div class="source">Soil Biol. Biochem.</div> , <div class="year">1994</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1591</div>-<div class="lpage">601</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Agronomic%20applications%20of%20Azospirillum%3A%20an%20evaluation%20of%2020%20years%20worldwide%20field%20inoculation&amp;author=Y.%20Okon&amp;author=C.A.%20Labandera-Gonzalez&amp;publication_year=1994&amp;journal=Soil%20Biol.%20Biochem.&amp;volume=26&amp;pages=1591-601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0038-0717(94)90311-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0038-0717(94)90311-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0038-0717(94)90311-5"> </span></div> </div></div></div></div></div><div content-id="dsp026c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elbeltagy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nishioka</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Endophytic colonization and <em>in planta</em> nitrogen fixation by a <em>Herbaspirillum</em> sp. isolated from wild rice species</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">5285</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endophytic%20colonization%20and%20in%20planta%20nitrogen%20fixation%20by%20a%20Herbaspirillum%20sp.%20isolated%20from%20wild%20rice%20species&amp;author=A.%20Elbeltagy&amp;author=K.%20Nishioka&amp;author=T.%20Sato&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=67&amp;pages=5285-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.11.5285-5293.2001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.11.5285-5293.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.67.11.5285-5293.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11679357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Yokota</div> <div class="given-names">A.</div></div>. </div><div class="article-title"><em>Azospirillum oryzae</em> sp. nov., a nitrogen-fixing bacterium isolated from the roots of the rice plant <em>Oryza sativa</em></div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">1435</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Azospirillum%20oryzae%20sp.%20nov.%2C%20a%20nitrogen-fixing%20bacterium%20isolated%20from%20the%20roots%20of%20the%20rice%20plant%20Oryza%20sativa&amp;author=C.H.%20Xie&amp;author=A.%20Yokota&amp;publication_year=2005&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=55&amp;pages=1435-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.63503-0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.63503-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.63503-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16014463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yasuda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Isawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakashita</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Effects of colonization of a bacterial endophyte, <em>Azospirillum</em> sp. B510 on disease resistance in rice</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2009</div>, vol. <div class="volume">73</div>  <div class="comment">doi:10.1271/bbb.90402</div></div></div></div></div><div content-id="dsp026c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Novel endophytic nitrogen-fixing clostridia from the grass <em>Miscanthus sinensis</em> as revealed by terminal restriction fragment length polymorphism analysis</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">6580</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20endophytic%20nitrogen-fixing%20clostridia%20from%20the%20grass%20Miscanthus%20sinensis%20as%20revealed%20by%20terminal%20restriction%20fragment%20length%20polymorphism%20analysis&amp;author=T.%20Miyamoto&amp;author=M.%20Kawahara&amp;author=K.%20Minamisawa&amp;publication_year=2004&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=70&amp;pages=6580-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.70.11.6580-6586.2004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.70.11.6580-6586.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.70.11.6580-6586.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15528521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ye</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Effect of inoculation with anaerobic nitrogen-fixing consortium on salt tolerance of <em>Miscanthus sinensis</em></div>, <div class="source">Soil Sci. Plant Nutr.</div> , <div class="year">2005</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">243</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20inoculation%20with%20anaerobic%20nitrogen-fixing%20consortium%20on%20salt%20tolerance%20of%20Miscanthus%20sinensis&amp;author=B.%20Ye&amp;author=A.%20Saito&amp;author=K.%20Minamisawa&amp;publication_year=2005&amp;journal=Soil%20Sci.%20Plant%20Nutr.&amp;volume=51&amp;pages=243-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1747-0765.2005.tb00028.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1747-0765.2005.tb00028.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1747-0765.2005.tb00028.x"> </span></div> </div></div></div></div></div><div content-id="dsp026c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">You</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishiguchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Isawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mitsui</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Expression of the <em>nifH</em> gene of a <em>Herbaspirillum</em> endophytes in wild rice species: daily rhythm during the light-dark cycle</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">8183</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20nifH%20gene%20of%20a%20Herbaspirillum%20endophytes%20in%20wild%20rice%20species%3A%20daily%20rhythm%20during%20the%20light-dark%20cycle&amp;author=M.%20You&amp;author=T.%20Nishiguchi&amp;author=A.%20Saito&amp;author=T.%20Isawa&amp;author=H.%20Mitsui&amp;author=K.%20Minamisawa&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=8183-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.12.8183-8190.2005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.12.8183-8190.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.12.8183-8190.2005"> </span></div> </div></div></div></div></div><div content-id="dsp026c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wolk</div> <div class="given-names">C.P.</div></div>, et al. </div><div class="article-title">Complete genomic sequence of the filamentous nitrogen-fixing cyanobacterium <em>Anabaena</em> sp. strain PCC 7120</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">205</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genomic%20sequence%20of%20the%20filamentous%20nitrogen-fixing%20cyanobacterium%20Anabaena%20sp.%20strain%20PCC%207120&amp;author=T.%20Kaneko&amp;author=Y.%20Nakamura&amp;author=C.P.%20Wolk&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=205-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.5.205" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.5.205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.5.205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11759840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laslett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Canback</div> <div class="given-names">B.</div></div>. </div><div class="article-title">ARAGORN, a program to detect <em>tRNA</em> genes and <em>tmRNA</em> genes in nucleotide sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">11</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARAGORN%2C%20a%20program%20to%20detect%20tRNA%20genes%20and%20tmRNA%20genes%20in%20nucleotide%20sequences&amp;author=D.%20Laslett&amp;author=B.%20Canback&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh152" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagesen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hallin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rødland</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Rognes</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">3100</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAmmer%3A%20consistent%20and%20rapid%20annotation%20of%20ribosomal%20RNA%20genes&amp;author=K.%20Lagesen&amp;author=P.%20Hallin&amp;author=E.A.%20R%C3%B8dland&amp;author=H.H.%20Staerfeldt&amp;author=T.%20Rognes&amp;author=D.W.%20Ussery&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=3100-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm160" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Improved microbial gene identification with GLIMMER</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4636</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4636-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garrity</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Holt</div> <div class="given-names">J.G.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Garrity</div> <div class="given-names">G.M.</div></div>. </div><div class="article-title">The Road Map to the Manual</div>, <div class="source">Bergey's Manual of Systematic Bacteriology, vol. 1</div> , <div class="year">2001</div><div class="edition">2nd edition</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">119</div>-<div class="lpage">66</div>)</div></div></div></div><div content-id="dsp026c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okuda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yoshizawa</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">KAAS: an automatic genome annotation and pathway reconstruction server</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W182</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KAAS%3A%20an%20automatic%20genome%20annotation%20and%20pathway%20reconstruction%20server&amp;author=Y.%20Moriya&amp;author=M.%20Itoh&amp;author=S.%20Okuda&amp;author=A.C.%20Yoshizawa&amp;author=M.%20Kanehisa&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W182-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm321" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mulder</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div>, et al. </div><div class="article-title">New developments in the InterPro database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D224</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database&amp;author=N.J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>, et al. </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">41</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Asymmetric substitution patterns in the two DNA strands of bacteria</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">660</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%20in%20the%20two%20DNA%20strands%20of%20bacteria&amp;author=J.R.%20Lobry&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=13&amp;pages=660-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025626" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fouts</div> <div class="given-names">D.E.</div></div>. </div><div class="article-title"><em>Phage_Finder</em>: automated identification and classification of prophage regions in complete bacterial genome sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">5839</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phage_Finder%3A%20automated%20identification%20and%20classification%20of%20prophage%20regions%20in%20complete%20bacterial%20genome%20sequences&amp;author=D.E.%20Fouts&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=5839-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl732" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17062630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bigot</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sivanathan</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Possoz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Barre</div> <div class="given-names">F.X.</div></div>,  <div class="name"><div class="surname">Cornet</div> <div class="given-names">F.</div></div>. </div><div class="article-title">FtsK, a literate chromosome segregation machine</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">1434</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FtsK%2C%20a%20literate%20chromosome%20segregation%20machine&amp;author=S.%20Bigot&amp;author=V.%20Sivanathan&amp;author=C.%20Possoz&amp;author=F.X.%20Barre&amp;author=F.%20Cornet&amp;publication_year=2007&amp;journal=Mol.%20Microbiol.&amp;volume=64&amp;pages=1434-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2007.05755.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2007.05755.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2007.05755.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17511809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Automated de novo identification of repeat sequence families in sequenced genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1269</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes&amp;author=Z.%20Bao&amp;author=S.R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zakria</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Njoloma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Akao</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Colonization and nitrogen-fixing ability of <em>Herbaspirillum</em> sp. Strain B501 <em>gfp</em>1 and assessment of its growth-promoting ability in cultivated rice</div>, <div class="source">Microbes Environ.</div> , <div class="year">2007</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">197</div>-<div class="lpage">206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Colonization%20and%20nitrogen-fixing%20ability%20of%20Herbaspirillum%20sp.%20Strain%20B501%20gfp1%20and%20assessment%20of%20its%20growth-promoting%20ability%20in%20cultivated%20rice&amp;author=M.%20Zakria&amp;author=J.%20Njoloma&amp;author=Y.%20Saeki&amp;author=S.%20Akao&amp;publication_year=2007&amp;journal=Microbes%20Environ.&amp;volume=22&amp;pages=197-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.22.197" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.22.197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.22.197"> </span></div> </div></div></div></div></div><div content-id="dsp026c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin-Didonet</div> <div class="given-names">C.C.</div></div>,  <div class="name"><div class="surname">Chubatsu</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Souza</div> <div class="given-names">E.M.</div></div>, et al. </div><div class="article-title">Genome structure of the genus <em>Azospirillum</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">4113</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20genus%20Azospirillum&amp;author=C.C.%20Martin-Didonet&amp;author=L.S.%20Chubatsu&amp;author=E.M.%20Souza&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=4113-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.14.4113-4116.2000" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.14.4113-4116.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.14.4113-4116.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10869094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brassinga</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Siam</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Marczynski</div> <div class="given-names">G.T.</div></div>. </div><div class="article-title">Conserved gene cluster at replication origins of the alpha-proteobacteria <em>Caulobacter crescentus</em> and <em>Rickettsia prowazekii</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">1824</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20gene%20cluster%20at%20replication%20origins%20of%20the%20alpha-proteobacteria%20Caulobacter%20crescentus%20and%20Rickettsia%20prowazekii&amp;author=A.K.%20Brassinga&amp;author=R.%20Siam&amp;author=G.T.%20Marczynski&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=1824-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.5.1824-1829.2001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.5.1824-1829.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.5.1824-1829.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11160121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsunaga</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fukuda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wahyudi</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Murase</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takeyama</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Complete genome sequence of the facultative anaerobic magnetotactic bacterium <em>Magnetospirillum</em> sp. strain AMB-1</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">157</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20facultative%20anaerobic%20magnetotactic%20bacterium%20Magnetospirillum%20sp.%20strain%20AMB-1&amp;author=T.%20Matsunaga&amp;author=Y.%20Okamura&amp;author=Y.%20Fukuda&amp;author=A.T.%20Wahyudi&amp;author=Y.%20Murase&amp;author=H.%20Takeyama&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=157-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mahillon</div> <div class="given-names">J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Craig</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Craigie</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gellert</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lambowitz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Insertion sequences revisited</div>, <div class="source">Mobile DNA II, American Society for Microbiology</div> , <div class="year">2002</div><div class="publisher-loc">Washington, DC</div>(pg. <div class="fpage">305</div>-<div class="lpage">66</div>)</div></div></div></div><div content-id="dsp026c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boyer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Haurat</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Samain</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Bacteriophage prevalence in the genus <em>Azospirillum</em> and analysis of the first genome sequence of an <em>Azospirillum brasilense</em> integrative phage</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">861</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteriophage%20prevalence%20in%20the%20genus%20Azospirillum%20and%20analysis%20of%20the%20first%20genome%20sequence%20of%20an%20Azospirillum%20brasilense%20integrative%20phage&amp;author=M.%20Boyer&amp;author=J.%20Haurat&amp;author=S.%20Samain&amp;publication_year=2008&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=74&amp;pages=861-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02099-07" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02099-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02099-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18065619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kunin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>. </div><div class="article-title">CRISPR-a widespread system that provides acquired resistance against phages in bacteria and archaea</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">181</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR-a%20widespread%20system%20that%20provides%20acquired%20resistance%20against%20phages%20in%20bacteria%20and%20archaea&amp;author=R.%20Sorek&amp;author=V.%20Kunin&amp;author=P.%20Hugenholtz&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=6&amp;pages=181-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1793" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18157154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brougns</div> <div class="given-names">S.J.J.</div></div>,  <div class="name"><div class="surname">Jore</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Lundgre</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Small CRISPR RNAs guide antiviral defense in prokaryotes</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">321</div> (pg. <div class="fpage">960</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20CRISPR%20RNAs%20guide%20antiviral%20defense%20in%20prokaryotes&amp;author=S.J.J.%20Brougns&amp;author=M.M.%20Jore&amp;author=M.%20Lundgre&amp;publication_year=2008&amp;journal=Science&amp;volume=321&amp;pages=960-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1159689" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1159689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1159689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18703739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Juhas</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">van der Meer</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Gaillard</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Harding</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Hood</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Crook</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Genomic islands: tools of bacterial horizontal gene transfer and evolution</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2009</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">376</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20islands%3A%20tools%20of%20bacterial%20horizontal%20gene%20transfer%20and%20evolution&amp;author=M.%20Juhas&amp;author=J.R.%20van%20der%20Meer&amp;author=M.%20Gaillard&amp;author=R.M.%20Harding&amp;author=D.W.%20Hood&amp;author=D.W.%20Crook&amp;publication_year=2009&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=33&amp;pages=376-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2008.00136.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2008.00136.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2008.00136.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19178566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">Z.M.</div></div>,  <div class="name"><div class="surname">Bussema</div> <div class="given-names">C.</div>3rd</div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title"><em>rrn</em>DB: documenting the number of rRNA and tRNA genes in bacteria and archaea</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D489</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=rrnDB%3A%20documenting%20the%20number%20of%20rRNA%20and%20tRNA%20genes%20in%20bacteria%20and%20archaea&amp;author=Z.M.%20Lee&amp;author=C.%20Bussema&amp;author=T.M.%20Schmidt&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D489-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn689" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgren</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nordlund</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The <em>fixABCX</em> genes in <em>Rhodospirillum rubrum</em> encode a putative membrane complex participating in electron transfer to nitrogenase</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">2052</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fixABCX%20genes%20in%20Rhodospirillum%20rubrum%20encode%20a%20putative%20membrane%20complex%20participating%20in%20electron%20transfer%20to%20nitrogenase&amp;author=T.%20Edgren&amp;author=S.%20Nordlund&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=2052-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.7.2052-2060.2004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.7.2052-2060.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.7.2052-2060.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15028689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sperotto</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Gross</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vedoy</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Passaglia</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Schrank</div> <div class="given-names">I.S.</div></div>. </div><div class="article-title">The electron transfer flavoprotein <em>fixABCX</em> gene products from <em>Azospirillum brasilense</em> show a NifA-dependent promoter regulation</div>, <div class="source">Curr. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">267</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20electron%20transfer%20flavoprotein%20fixABCX%20gene%20products%20from%20Azospirillum%20brasilense%20show%20a%20NifA-dependent%20promoter%20regulation&amp;author=R.A.%20Sperotto&amp;author=J.%20Gross&amp;author=C.%20Vedoy&amp;author=L.M.%20Passaglia&amp;author=I.S.%20Schrank&amp;publication_year=2004&amp;journal=Curr.%20Microbiol.&amp;volume=49&amp;pages=267-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00284-004-4318-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00284-004-4318-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00284-004-4318-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15386115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giraud</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moulin</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Vallenet</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">316</div> (pg. <div class="fpage">1307</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%20symbioses%3A%20absence%20of%20Nod%20genes%20in%20photosynthetic%20bradyrhizobia&amp;author=E.%20Giraud&amp;author=L.%20Moulin&amp;author=D.%20Vallenet&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1307-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1139548" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1139548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1139548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Preisig</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Anthamatten</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hennecke</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genes for a microaerobically induced oxidase complex in <em>Bradyrhizobium japonicum</em> are essential for a nitrogen-fixing endosymbiosis</div>, <div class="source">Proc. Natl Acad. Sci. USA.</div> , <div class="year">1993</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">3309</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20for%20a%20microaerobically%20induced%20oxidase%20complex%20in%20Bradyrhizobium%20japonicum%20are%20essential%20for%20a%20nitrogen-fixing%20endosymbiosis&amp;author=O.%20Preisig&amp;author=D.%20Anthamatten&amp;author=H.%20Hennecke&amp;publication_year=1993&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA.&amp;volume=90&amp;pages=3309-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.8.3309" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.8.3309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.8.3309"> </span></div> </div></div></div></div></div><div content-id="dsp026c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marchal</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Keijers</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Haaker</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Vanderleyden</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A cytochrome <em>cbb</em>3 (cytochrome <em>c</em>) terminal oxidase in <em>Azospirillum brasilense</em> Sp7 supports microaerobic growth</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">5689</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20cytochrome%20cbb3%20(cytochrome%20c)%20terminal%20oxidase%20in%20Azospirillum%20brasilense%20Sp7%20supports%20microaerobic%20growth&amp;author=K.%20Marchal&amp;author=J.%20Sun&amp;author=V.%20Keijers&amp;author=H.%20Haaker&amp;author=J.%20Vanderleyden&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=5689-96" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9791120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fischer</div> <div class="given-names">H.M.</div></div>. </div><div class="article-title">Genetic regulation of nitrogen fixation in rhizobia</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1994</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">352</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20regulation%20of%20nitrogen%20fixation%20in%20rhizobia&amp;author=H.M.%20Fischer&amp;publication_year=1994&amp;journal=Microbiol.%20Rev.&amp;volume=58&amp;pages=352-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7968919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gilles-Gonzalez</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Gonzalez</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Signal transduction by heme-containing PAS-domain proteins</div>, <div class="source">J. Appl. Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">774</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signal%20transduction%20by%20heme-containing%20PAS-domain%20proteins&amp;author=M.A.%20Gilles-Gonzalez&amp;author=G.%20Gonzalez&amp;publication_year=2004&amp;journal=J.%20Appl.%20Physiol.&amp;volume=96&amp;pages=774-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/japplphysiol.00941.2003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fjapplphysiol.00941.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fjapplphysiol.00941.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14715687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hardoin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">van Overbeek</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">van Elsas</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Properties of bacterial endophytes and their proposed role in plant growth</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">463</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20of%20bacterial%20endophytes%20and%20their%20proposed%20role%20in%20plant%20growth&amp;author=P.%20Hardoin&amp;author=L.S.%20van%20Overbeek&amp;author=J.D.%20van%20Elsas&amp;publication_year=2008&amp;journal=Trends%20Microbiol.&amp;volume=16&amp;pages=463-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2008.07.008" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2008.07.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2008.07.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18789693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nukui</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Rhizobial strategies to enhance symbiotic interactions: rhizobitoxine and 1-aminocyclopropane-1-carboxylate deaminase</div>, <div class="source">Microbes Environ.</div> , <div class="year">2004</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">99</div>-<div class="lpage">111</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhizobial%20strategies%20to%20enhance%20symbiotic%20interactions%3A%20rhizobitoxine%20and%201-aminocyclopropane-1-carboxylate%20deaminase&amp;author=S.%20Okazaki&amp;author=N.%20Nukui&amp;author=M.%20Sugawara&amp;author=K.%20Minamisawa&amp;publication_year=2004&amp;journal=Microbes%20Environ.&amp;volume=19&amp;pages=99-111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.19.99" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.19.99" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.19.99"> </span></div> </div></div></div></div></div><div content-id="dsp026c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grichko</div> <div class="given-names">V.P.</div></div>,  <div class="name"><div class="surname">Glick</div> <div class="given-names">B.R.</div></div>. </div><div class="article-title">Identification of DNA sequences that regulate the expression of the <em>Enterobacter cloacae</em> UV4 1-aminocyclopropane-1-carboxylic acid deaminase gene</div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">1159</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20DNA%20sequences%20that%20regulate%20the%20expression%20of%20the%20Enterobacter%20cloacae%20UV4%201-aminocyclopropane-1-carboxylic%20acid%20deaminase%20gene&amp;author=V.P.%20Grichko&amp;author=B.R.%20Glick&amp;publication_year=2000&amp;journal=Can.%20J.%20Microbiol.&amp;volume=46&amp;pages=1159-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11142408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Glick</div> <div class="given-names">B.R.</div></div>. </div><div class="article-title">Transcriptiona; regulation of the <em>Enterobacter cloacae</em> UW4 1-aminocyclopropane-1-carboxylate (ACC) deaminase gene (<em>acdS</em>)</div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">356</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptiona%3B%20regulation%20of%20the%20Enterobacter%20cloacae%20UW4%201-aminocyclopropane-1-carboxylate%20(ACC)%20deaminase%20gene%20(acdS)&amp;author=J.%20Li&amp;author=B.R.%20Glick&amp;publication_year=2001&amp;journal=Can.%20J.%20Microbiol.&amp;volume=47&amp;pages=356-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/cjm-47-4-359" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fcjm-47-4-359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fcjm-47-4-359"> </span></div> </div></div></div></div></div><div content-id="dsp026c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glick</div> <div class="given-names">B.R.</div></div>. </div><div class="article-title">Bacterial ACC deaminase and the alleviation of plant stress</div>, <div class="source">Adv. Appl. Micrbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">291</div>-<div class="lpage">312</div>)</div></div></div></div><div content-id="dsp026c56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Costacurta</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vanderleyden</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Synthesis of phytohormones by plant-associated bacteria</div>, <div class="source">CRC Crit. Rev. Microbiol.</div> , <div class="year">1995</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthesis%20of%20phytohormones%20by%20plant-associated%20bacteria&amp;author=A.%20Costacurta&amp;author=J.%20Vanderleyden&amp;publication_year=1995&amp;journal=CRC%20Crit.%20Rev.%20Microbiol.&amp;volume=21&amp;pages=1-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3109/10408419509113531" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3109%2F10408419509113531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3109%2F10408419509113531"> </span></div> </div></div></div></div></div><div content-id="dsp026c57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Broek</div> <div class="given-names">A.V.</div></div>,  <div class="name"><div class="surname">Gysegom</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ona</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Transcriptional analysis of the <em>Azospirillum brasilense</em> indole-3-pyruvate decarboxylase gene and identification of a <em>cis</em>-acting sequence involved in auxin responsive expression</div>, <div class="source">Plant Mol. Microbe Interact.</div> , <div class="year">2005</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">311</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20analysis%20of%20the%20Azospirillum%20brasilense%20indole-3-pyruvate%20decarboxylase%20gene%20and%20identification%20of%20a%20cis-acting%20sequence%20involved%20in%20auxin%20responsive%20expression&amp;author=A.V.%20Broek&amp;author=P.%20Gysegom&amp;author=O.%20Ona&amp;publication_year=2005&amp;journal=Plant%20Mol.%20Microbe%20Interact.&amp;volume=18&amp;pages=311-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-18-0311" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-18-0311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-18-0311"> </span></div> </div></div></div></div></div><div content-id="dsp026c58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W116</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saier</div> <div class="given-names">M.H.</div>Jr</div>,  <div class="name"><div class="surname">Tran</div> <div class="given-names">C.V.</div></div>,  <div class="name"><div class="surname">Barabote</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">TCDB: the transporter classification database for membrane transport protein analyses and information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D181</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TCDB%3A%20the%20transporter%20classification%20database%20for%20membrane%20transport%20protein%20analyses%20and%20information&amp;author=M.H.%20Saier&amp;author=C.V.%20Tran&amp;author=R.D.%20Barabote&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D181-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koebnik</div> <div class="given-names">R.</div></div>. </div><div class="article-title">TonB-dependent trans-envelope signaling: the exception or the rule?</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">343</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TonB-dependent%20trans-envelope%20signaling%3A%20the%20exception%20or%20the%20rule%3F&amp;author=R.%20Koebnik&amp;publication_year=2005&amp;journal=Trends%20Microbiol.&amp;volume=13&amp;pages=343-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2005.06.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2005.06.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2005.06.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15993072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dao</div> <div class="given-names">T.V.</div></div>,  <div class="name"><div class="surname">Nomura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hamaguchi</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">NAD-malic enzyme affects nitrogen fixing activity of <em>Bradyrhizobium japonicum</em> USDA 110 bacteroids in soybean nodules</div>, <div class="source">Microbes Environ.</div> , <div class="year">2008</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">215</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NAD-malic%20enzyme%20affects%20nitrogen%20fixing%20activity%20of%20Bradyrhizobium%20japonicum%20USDA%20110%20bacteroids%20in%20soybean%20nodules&amp;author=T.V.%20Dao&amp;author=M.%20Nomura&amp;author=R.%20Hamaguchi&amp;publication_year=2008&amp;journal=Microbes%20Environ.&amp;volume=23&amp;pages=215-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.23.215" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.23.215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.23.215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21558711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Juhas</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Crook</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Hood</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Type IV secretion systems: tools of bacterial horizontal gene transfer and virulence</div>, <div class="source">Cell. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">2377</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Type%20IV%20secretion%20systems%3A%20tools%20of%20bacterial%20horizontal%20gene%20transfer%20and%20virulence&amp;author=M.%20Juhas&amp;author=D.W.%20Crook&amp;author=D.W.%20Hood&amp;publication_year=2008&amp;journal=Cell.%20Microbiol.&amp;volume=10&amp;pages=2377-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cmi.2008.10.issue-12" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcmi.2008.10.issue-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcmi.2008.10.issue-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18549454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van der Horst</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Key</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hellingwerf</div> <div class="given-names">K.J.</div></div>. </div><div class="article-title">Photosensing in chemotrophic, non-phototrophic bacteria: let there be light sensing too</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">554</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Photosensing%20in%20chemotrophic%2C%20non-phototrophic%20bacteria%3A%20let%20there%20be%20light%20sensing%20too&amp;author=M.A.%20van%20der%20Horst&amp;author=J.%20Key&amp;author=K.J.%20Hellingwerf&amp;publication_year=2007&amp;journal=Trends%20Microbiol.&amp;volume=15&amp;pages=554-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2007.09.009" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2007.09.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2007.09.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18024131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karniol</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Vierstra</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">The pair of bacteriophytochromes from <em>Agrobacterium tumefaciens</em> are histidine kinases with opposing photobiological properties</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">2807</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pair%20of%20bacteriophytochromes%20from%20Agrobacterium%20tumefaciens%20are%20histidine%20kinases%20with%20opposing%20photobiological%20properties&amp;author=B.%20Karniol&amp;author=R.D.%20Vierstra&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=2807-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0437914100" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0437914100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0437914100"> </span></div> </div></div></div></div></div><div content-id="dsp026c65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaubert</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lavergne</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fardoux</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A singular bacteriophytochrome acquired by lateral gene transfer</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2007</div>, vol. <div class="volume">282</div> (pg. <div class="fpage">7320</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20singular%20bacteriophytochrome%20acquired%20by%20lateral%20gene%20transfer&amp;author=M.%20Jaubert&amp;author=J.%20Lavergne&amp;author=J.%20Fardoux&amp;publication_year=2007&amp;journal=J.%20Biol.%20Chem.&amp;volume=282&amp;pages=7320-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M611173200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M611173200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M611173200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17218312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boyer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bally</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Perrotto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chaintreuil</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wisniewski-Dyé</div> <div class="given-names">F.</div></div>. </div><div class="article-title">A quorum-quenching approach to identify quorum-sensing-regulated functions in <em>Azospirillum lipoferum</em></div>, <div class="source">Res. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">699</div>-<div class="lpage">708</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20quorum-quenching%20approach%20to%20identify%20quorum-sensing-regulated%20functions%20in%20Azospirillum%20lipoferum&amp;author=M.%20Boyer&amp;author=R.%20Bally&amp;author=S.%20Perrotto&amp;author=C.%20Chaintreuil&amp;author=F.%20Wisniewski-Dy%C3%A9&amp;publication_year=2008&amp;journal=Res.%20Microbiol.&amp;volume=159&amp;pages=699-708" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.resmic.2008.08.003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.resmic.2008.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.resmic.2008.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18790051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vial</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cuny</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gluchoff-Fiasson</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">N-acyl-homoserine lactone-mediated quorum-sensing in <em>Azospirillum</em>: an exception rather than a rule</div>, <div class="source">FEMS Microbiol. Ecol.</div> , <div class="year">2006</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">155</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=N-acyl-homoserine%20lactone-mediated%20quorum-sensing%20in%20Azospirillum%3A%20an%20exception%20rather%20than%20a%20rule&amp;author=L.%20Vial&amp;author=C.%20Cuny&amp;author=K.%20Gluchoff-Fiasson&amp;publication_year=2006&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=58&amp;pages=155-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2006.58.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2006.58.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffem.2006.58.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17064258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsp026c68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsp026c68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vanbleu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Marchal</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lambrecht</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mathys</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vanderleyden</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Annotation of the pRhico plasmid of <em>Azospirillum brasilense</em> reveals its role in determining the outer surface composition</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2004</div>, vol. <div class="volume">232</div> (pg. <div class="fpage">165</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotation%20of%20the%20pRhico%20plasmid%20of%20Azospirillum%20brasilense%20reveals%20its%20role%20in%20determining%20the%20outer%20surface%20composition&amp;author=E.%20Vanbleu&amp;author=K.%20Marchal&amp;author=M.%20Lambrecht&amp;author=J.%20Mathys&amp;author=J.%20Vanderleyden&amp;publication_year=2004&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=232&amp;pages=165-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1097(04)00046-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1097(04)00046-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1097(04)00046-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15033235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347240&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_9.xls?Expires=1509238700&amp;Signature=Dm-WiH0lyNNYRI~yoLl9g29t8LAc-N6igh7WX6mbUG7xtcCeEHKr~6BBdDJl9bhU5N4~I1Nkw5OoLAUwnA0wK6xPQFOlki84hWd9OZ-CgCjspQ3sN~amDmEPOsLf7kL~9HJz2U63yW3c1RXOK43TyjmAlvZe~JByBJrSGRvP5s4C0zsyoMldRNFT8YqgzfzXkbIA1Avnu~U-AOORGdsVsI0PXYC1hXm3YPJpPWcThHRn4nhJURWv9SbLeNxfbJqjEkPKmm9CN8QFbaDOBw8o-lH37VmBYlHBBNEI84e4DPb8p07HEq-0r8GiiXFKrPfVE6Qis-4FIGsFotqUorcFPg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_8.xls?Expires=1509238700&amp;Signature=LmQFqsd3OHNSFy54muvgTdYEXsC2jin7UfmVgd3VKzWWf6W9cWfm9wjrxC7MOwIfL51P8UnaowZdh87s5JJFwWjYFDrNiKlXvZE8Yv66MCpEAWZHJLda60-FqP6AswHTKRJgy4A0bOq9KlaO-7ACLnpmeeHskwAJC3Lzg~RMQKfHZ8j8~BXsaN9VNP4W3v4YAbBtB2WkHDSBecP47t3tFA3wE1nULwRXAqiU4gxO0A7MjzXzQ80jB1s9ebKBzsR2ihQkdORoifHGnDrxwmvYvTgzNnmqZFrv1wEbTVmPW19f2JzMxbmg3xuqsWR50tSUvwTBmwd6ZMP-mcuSCu1BCQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_7.xls?Expires=1509238700&amp;Signature=a67y~jpfsq4MyRAcbnunP-RCMvuzVIEL~xm85if28v~-UAt9V-e2M2OrZAebSNi8zGyz5ML7REC5KYGTnhobD2JW9zMgRDXYFIe5VgfiP4XGb9Ist1jLVwdofW5k0~xzhCKFa0pEcHKALGcMyvhizYDAsZuEQqr0GT~nqZsVYtn-U~--9so0O-ysp-cn0dNdsQqvahSviU8dhyY7M-PU8w8CJNVuz7mIR4PcfmB261xgTDYeT4rbRLggmLKoyGmg0Uw3gTcKcjHCNHxMO3UsHJYA2y53NHmZCgqwtd~-yKwibYRrH9DEBHtW~dsrpyqARIKw8pB3cmM02iTCsQkppw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_6.pdf?Expires=1509238700&amp;Signature=BxHYeaMPKM~8NtqKZSmPTL2cAQjzfAOmyId9PPTfvbHnab~B6~4BGpntKDQPg-SYe1TpV2i7c8p78nLGCkiSh7rtoDqVZp-2Xve-yN6Kz6Q84yLpWou96dGYAdctjRlXoEShZHh9NTgVilQ21GhIVfZrXld~aMTIefSy5-isHanPmFD2~wTkfepsfI5TsNzHBfNKmGavcY8IYqcfLH03AKkkYk2KVRAIlMAbVJN7jRqyXVd0876PEuxNSuTgtHD~X7ARsMZ1~Igx5Kt9wtrqvaRzJyZFZlQGSRymryWW9qkhBY8I1cX8GXISDDA7pKU6X9fLLVvLhBiREr4NdJ~PwQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-8</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_5.pdf?Expires=1509238700&amp;Signature=JCRNN0cfjBNgFNDQ7DS~3ZStZjPB405xvUtH01v9wTw1CwBrNU-DyUnJ1-npPHODdkaDrfrHuUF7R7tnstXrUZFF0sW7mYIfORHEBwKXD7AKX1DilmWe~IVyhAVuhLwjzBc0kqqm9jFfoYGbJkJNJ6j-koziVH~xh9rDsNxxqSR4ElWbZDea-c35uQxMa5ddPF~Li1SuzoRpLxoIQ28BGPXKGYbZIGOGMIMaoShafF9MHobCigzeY~cIJUiQww4lN3hSCUZRUqfbdmNCq4XUabTmmWxys6lr3~wua313hjiqxYqkgQZzN7sWHff6f06PstGz3Kya8sVJU6JHKKL8bg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_4.pdf?Expires=1509238700&amp;Signature=VSAOalFdLb0igTiZw58w8xA0AUjiTcH6Qa8Nmr-0nFwo0-IsZ9BqSsUgcbqMNlbKpda3f38U2MZ9nW7nr00DRsh-KlezQENCgWVMKf3~nxsnZH4S0yaupTSJWF01WXLupwuWVjVpHxmjsBX2aFI5iSMpn4Gz7CQj7D32G3AzDaAQMutVryqgOnaOA7XpXq9hU5wV-Xd~P2Ohu2jVhBuG~NJxutUbX4r0rPrV~G3u7K4cGk~39EEptMDXt3UYHIeOzIpC6KCa8Se96EZjjJz~CSgVtxSg6DeSuyJCCmKAnSG6V78v0rebmzeqBtlDL~QU0FmneXfKhcG6I11Vrlb38g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_3.xls?Expires=1509238700&amp;Signature=MT65uGOYgbF-NvtIZSHK5eTqUrLqcvzuaVsE85mx1C9jrN9eZtfcQbQSJmpHorztMkqr2H38sjwol2Ng9fvXc~k1gPUqo1LS5IfuYVrWFD6Y6Bi1YpNVmKGysFs~10Eg3DZgkLux0EuB7Z4ABn1m7pYAjzwyN~JVFWkAZJuIuARFJsRg41f3RDySTXV-7HFxt7W-YIfIe6p-OldUV3pJGOCLop5oKXJOeeAQqHqJV-0Ck8DaFHghDzPz3ykZTUIBDpWGZZ0bVXQS51vCwzN4~xA8pcqOWSBeAKmjfs-PatqudhqnIE77Mcq-tmbwT1al19HldRu5tfz38LUWprJn~w__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_2.pdf?Expires=1509238700&amp;Signature=Z6Sj~xY0qbk-NqWEocPtt3x3et7pXCIMSPqVJbhLwHa-C0S6bGZCa4maUjw3UVj9x37twqYjfX3GphxGSTEvHZXkMQqFUnmZsc14WULV1vE0XqbFnmt6nGB-sGXwVGvKc~R3INcFp24211qFPkHikDqwswtnDNxsQfTBGT7PJdZFNbNoVS~Kei50rth7W7mNIBBvCdU5Nl3rQulWVJalKNbCN2Lnz35TVyDwcAAWPKhiKmAF6iv5vzL787F9ufmInI1gFnhAh8e85ldRU2byBxspCZJ-tF4WEWw-yQrz5Sn1wSWCefh64PRXJkF0SqgIlq5o-6Lsd2RX25E4946IgA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_10.xls?Expires=1509238700&amp;Signature=E9M5il5-GWQAJb2AQIDki0UWAiUJ0Aeh6dIB-i~CuWsgaSWN7RYGTReJLYoSUaZQksWhi0mnnGR4wJd3-UaMJ5NakjajbtP2E4sVmXLgKIO9BUcqYdfRt~lednbeeQDrYSp~K5ALbGkRGpHSGqjvwKFoFqLHIV78V5b4Cw2sfTWuoyVc-rW9pTjoA1bOvKZOC4VduQKUkkbmjPEq4LnV0lB8WoqulyH6Tkl44QM4uTaNwiW1HadP-LdIn-eATjZG5QsLRs7SFsUdIQ7Tq6TZFBXC2ll0yyLIM1ymvyH87FZ~hQ8RnBx5lc6L3ny32qN41xKz341~LmekLRRT1N8wPw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_table_1.xls?Expires=1509238700&amp;Signature=E2TLmtSmECF6-c7n~mhOBOxdlGZY67zoxbBHKdR5p87LV9G--OWpJxnE4hCey08UDFQ3SxDUy54t8UKH8tjChb-nQ9~k6WlM2X2fhrvh48pMfAusy5F7idUz5mDYnm2umO-PRygkUV52VJSKRM67poiC7rEUFwP88qPvJE5CoCfD90lD-uxZperSZmujw7aDN7q90tDQwIP4TuVbIljas0lFuSbVwm-S8txxQSeqoFCcHX-uDyADYttiHhdUqmefL6k6~X0p4f9bBtBTVWYGlNzebpuRInl-LqGK1-~vLauDcHkCtVW4gcHpi98nYr6OvnlT3NHWifhuYOloiJhkHw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/1/10.1093/dnares/dsp026/2/dsp026-dsp010_supp_fig.pdf?Expires=1509238700&amp;Signature=NAxVNQ-Z~4SGNT96-Nr9ZyxPzJBmTnpekSNXrSElGdsTTzj9J--e-z5QQosUhlaTT8M44V1t8MbRCrLwgHvaSXr0h0uNa4XkVHcPHy5gRwNEGN-RBaf6gx78ysNulqkbP0Q029K4ciDITgmrTc6ySvsttfk-mReQibLn0JADsEi-iPZesvJ6~EM62E-B5IY5Yg0BUKho8lGZRjce2ASJqrI-5DvgjLGdiBEJ-QDVH1okR7jChQZpjw6u8CmEykT1~5tkE7PHnKtbxL4qBX7IC~eFKY2EsNa0epF8QXSSSQ9JhmI8sa8tEmQ5g3QsRcf5RxSdLA9RSu7fx4qF6S4NPQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,125</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000275264300004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">57</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsp026"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347240"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000275264300004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f1%2f37%2f347240">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000275264300004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (57)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F1%2F37%2F347240" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347240'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complete Genomic Structure of the Cultivated Rice Endophyte Azospirillum sp. B510" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f1%2f37%2f347240">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

