

<!DOCTYPE html>
<html lang="en">

<head><title>
	Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced Brassica Crop Species | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/1/53/347430","@type":"ScholarlyArticle","name":"Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced  Brassica  Crop Species","datePublished":"2013-10-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2014-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst040","keywords":["brassica","microsatellite","distribution","marker","database"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/1/53/347430","author":[{"name":"Shi, Jiaqin","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Huang, Shunmou","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Zhan, Jiepeng","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Yu, Jingyin","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Wang, Xinfa","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Hua, Wei","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Liu, Shengyi","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Liu, Guihua","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"},{"name":"Wang, Hanzhong","affiliation":"Key Laboratory of Oil Crop Biology of the Ministry of Agriculture ,  Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences ,  Wuhan 430062 ,  China","@type":"Person"}],"description":"Although much research has been conducted, the pattern of microsatellite distribution has remained ambiguous, and the development/utilization of microsatellite markers has still been limited/inefficient in Brassica, due to the lack of genome sequences. In view of this, we conducted genome-wide microsatellite characterization and marker development in three recently sequenced Brassica crops: Brassica rapa, Brassica oleracea and Brassica napus. The analysed microsatellite characteristics of these Brassica species were highly similar or almost identical, which suggests that the pattern of microsatellite distribution is likely conservative in Brassica. The genomic distribution of microsatellites was highly non-uniform and positively or negatively correlated with genes or transposable elements, respectively. Of the total of 115 869, 185 662 and 356 522 simple sequence repeat (SSR) markers developed with high frequencies (408.2, 343.8 and 356.2 per Mb or one every 2.45, 2.91 and 2.81 kb, respectively), most represented new SSR markers, the majority had determined physical positions, and a large number were genic or putative single-locus SSR markers. We also constructed a comprehensive database for the newly developed SSR markers, which was integrated with public Brassica SSR markers and annotated genome components. The genome-wide SSR markers developed in this study provide a useful tool to extend the annotated genome resources of sequenced Brassica species to genetic study/breeding in different Brassica species.","pageStart":"53","pageEnd":"68","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1572217320&Signature=aun9IKyXasmjhx3YS3K~bSNfIqUZQWpE882AqojbnCZEuQ7oW1sEzG7hgFX9g7dN~vx1Zrwuj8BRT8tG~Gu2x2aRdFwPMUaB-ITSLVWdzgndTjRyXuQBkgeGIZk-7tWzOKyLiep4-N33UtD~8bTrkG5v7RQKP10stn3vYUtvYLGnSPUDESgLkItW4V8pEU4NfzGTfcM2GqZ7C2oXNkyJgsZs0T1xA04NmO1rXdJcU8yrRiXzpnm8nOZh3jUAZ1Qt0QeLEwkZ5s9LGubKq2vB1sRJDgbwozQdNScMNKGf5SvViiDlpHmkCJzr1meOI3f7gbBpmUUokk-sC2U0uhJfew__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced  Brassica  Crop Species","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1572217320&Signature=aun9IKyXasmjhx3YS3K~bSNfIqUZQWpE882AqojbnCZEuQ7oW1sEzG7hgFX9g7dN~vx1Zrwuj8BRT8tG~Gu2x2aRdFwPMUaB-ITSLVWdzgndTjRyXuQBkgeGIZk-7tWzOKyLiep4-N33UtD~8bTrkG5v7RQKP10stn3vYUtvYLGnSPUDESgLkItW4V8pEU4NfzGTfcM2GqZ7C2oXNkyJgsZs0T1xA04NmO1rXdJcU8yrRiXzpnm8nOZh3jUAZ1Qt0QeLEwkZ5s9LGubKq2vB1sRJDgbwozQdNScMNKGf5SvViiDlpHmkCJzr1meOI3f7gbBpmUUokk-sC2U0uhJfew__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced  Brassica  Crop Species | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Although much research has been conducted, the pattern of microsatellite distribution has remained ambiguous, and the development/utilization of microsatellite markers has still been limited/inefficient in Brassica, due to the lack of genome sequences. In view of this, we conducted genome-wide microsatellite characterization and marker development in three recently sequenced Brassica crops: Brassica rapa, Brassica oleracea and Brassica napus. The analysed microsatellite characteristics of these Brassica species were highly similar or almost identical, which suggests that the pattern of microsatellite distribution is likely conservative in Brassica. The genomic distribution of microsatellites was highly non-uniform and positively or negatively correlated with genes or transposable elements, respectively. Of the total of 115 869, 185 662 and 356 522 simple sequence repeat (SSR) markers developed with high frequencies (408.2, 343.8 and 356.2 per Mb or one every 2.45, 2.91 and 2.81 kb, respectively), most represented new SSR markers, the majority had determined physical positions, and a large number were genic or putative single-locus SSR markers. We also constructed a comprehensive database for the newly developed SSR markers, which was integrated with public Brassica SSR markers and annotated genome components. The genome-wide SSR markers developed in this study provide a useful tool to extend the annotated genome resources of sequenced Brassica species to genetic study/breeding in different Brassica species." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/1/53/347430" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217320&amp;Signature=JhDbKlTzOTP8BlroeaAMC1ABWRDmb9tXpaF-k6drDmJApIOMStS4RCkhD2w2yeSiUwncVf6iDFoOFUU8vs9zp3jAHQUaQA08qMLjcdHqmBMQdgW3SM0pCzw-Eo4QiiSMlyXPTAPDy9zSfCtwudNC2A1I90wiMbHANIv286nbi4XYsBdyMB-piAHs2d~nbT7ikr~0yt-WyMiSD1yJLoaoIo-cWlFpmi5dOrP1ir~vOQoaZ8gpIsr~4krhJlL6jnyMMdvSLiENZoKhGv5q1Vn6qVSdOr-uQpdRtUYCD8D0cUJe5CkJeHqHhkKdM3~-eKQ1rbHo7EmfENOMkF9rjqNKuw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2013-10-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shi, Jiaqin" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Huang, Shunmou" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Zhan, Jiepeng" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Yu, Jingyin" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Wang, Xinfa" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Hua, Wei" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Liu, Shengyi" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Liu, Guihua" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_author" content="Wang, Hanzhong" /><meta name="citation_author_institution" content="Key Laboratory of Oil Crop Biology of the Ministry of Agriculture, Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences, Wuhan 430062, China" /><meta name="citation_title" content="Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced Brassica Crop Species" /><meta name="citation_firstpage" content="53" /><meta name="citation_lastpage" content="68" /><meta name="citation_doi" content="10.1093/dnares/dst040" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2014/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Simple sequences are ubiquitous repetitive components of eukaryotic genomes; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=4127-38; " /><meta name="citation_reference" content="citation_title=Microsatellites: simple sequences with complex evolution; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=435-45; " /><meta name="citation_reference" content="citation_title=Microsatellites: genomic distribution, putative functions and mutational mechanisms: a review; Mol. Ecol.;  citation_year=2002;  citation_volume=11;  citation_pages=2453-65; " /><meta name="citation_reference" content="citation_title=Microsatellites within genes: structure, function, and evolution; Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=991-1007; " /><meta name="citation_reference" content="citation_title=Variable tandem repeats accelerate evolution of coding and regulatory sequences; Annu. Rev. Genet.;  citation_year=2010;  citation_volume=44;  citation_pages=445-77; " /><meta name="citation_reference" content="citation_title=PCR-amplified microsatellites as markers in plant genetics; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=175-82; " /><meta name="citation_reference" content="citation_title=The development and use of microsatellite markers for genetic analysis and plant breeding with emphasis on bread wheat; Euphytica;  citation_year=2000;  citation_volume=113;  citation_pages=163-85; " /><meta name="citation_reference" content="citation_title=Toward a unified approach to genetic mapping of eukaryotes based on sequence tagged microsatellite sites; Biotechnology;  citation_year=1990;  citation_volume=8;  citation_pages=930-2; " /><meta name="citation_reference" content="citation_title=Deciphering the diploid ancestral genome of the Mesohexaploid Brassica rapa.; Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=1541-54; " /><meta name="citation_reference" content="citation_publisher=Springer-Verlag, Berlin, Germany; Breeding Oilseed Brassicas;  citation_year=1993; " /><meta name="citation_reference" content="citation_title=Genome analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization; citation_author=U, N.; Japan J. Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=Efficient large-scale development of microsatellites for marker and mapping applications in Brassica crop species; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1103-12; " /><meta name="citation_reference" content="citation_title=Construction of an oilseed rape (Brassica napus L.) genetic map with SSR markers; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=1514-23; " /><meta name="citation_reference" content="citation_title=Microsatellite markers for genome analysis in Brassica. I. Development in Brassica napus and abundance in Brassicaceae species; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=689-94; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of microsatellites in Brassica rapa L; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=1092-8; " /><meta name="citation_reference" content="citation_title=Characteristics of microsatellites in Brassica rapa genome and their potential utilization for comparative genomics in Cruciferae; Breeding Sci.;  citation_year=2004;  citation_volume=54;  citation_pages=85-90; " /><meta name="citation_reference" content="citation_title=Abundance and characterization of simple-sequence repeats (SSRs) isolated from a size-fractionated genomic library of Brassica napus L (rapeseed); Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=206-11; " /><meta name="citation_reference" content="citation_title=Identification of polymorphic, conserved simple sequence repeats (SSRs) in cultivated Brassica species; Theor. Appl. Genet.;  citation_year=1996;  citation_volume=93;  citation_pages=534-8; " /><meta name="citation_reference" content="citation_title=Abundance, polymorphism and genetic mapping of microsatellites in oilseed rape (Brassica napus L.); Plant Breeding;  citation_year=1999;  citation_volume=118;  citation_pages=323-6; " /><meta name="citation_reference" content="citation_title=Construction of an integrated genetic linkage map for the A genome of Brassica napus using SSR markers derived from sequenced BACs in B. rapa; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=594" /><meta name="citation_reference" content="citation_title=Genomic distribution of simple sequence repeats in Brassica rapa.; Mol. Cells;  citation_year=2007;  citation_volume=23;  citation_pages=349-56; " /><meta name="citation_reference" content="citation_title=Characterization of simple sequence repeat markers derived in silico from Brassica rapa bacterial artical chromosome sequences and their application in Brassica napus; Mol. Ecol. Notes;  citation_year=2007;  citation_volume=7;  citation_pages=273-7; " /><meta name="citation_reference" content="citation_title=The reference genetic linkage map for the multinational Brassica rapa genome sequencing project; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=777-92; " /><meta name="citation_reference" content="citation_title=Development and genetic mapping of microsatellite markers from genome survey sequences in Brassica napus; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=1121-31; " /><meta name="citation_reference" content="citation_title=Identification and characterization of simple sequence repeat (SSR) markers derived in silico from Brassica oleracea genome shotgun sequences; Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=1191-4; " /><meta name="citation_reference" content="citation_title=Development and genetic mapping of microsatellite markers from whole genome shotgun sequences in Brassica oleracea; Mol. Breeding;  citation_year=2011;  citation_volume=28;  citation_pages=585-96; " /><meta name="citation_reference" content="citation_title=Identification and characterization of simple sequence repeat markers from Brassica napus expressed sequences; Mol. Ecol. Notes;  citation_year=2007;  citation_volume=7;  citation_pages=886-9; " /><meta name="citation_reference" content="citation_title=Sixteen new simple sequence repeat markers from Brassica juncea expressed sequences and their cross-species amplication; Mol. Ecol. Notes;  citation_year=2007;  citation_volume=7;  citation_pages=697-700; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in Brassica rapa: development, characterization, mapping, and their utility in other cultivated and wild Brassica relatives; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=305-20; " /><meta name="citation_reference" content="citation_title=Abundance, marker development and genetic mapping of microsatellites from unigenes in Brassica napus.; Mol. Breeding;  citation_year=2012;  citation_volume=30;  citation_pages=731-44; " /><meta name="citation_reference" content="citation_title=Large-scale development of functional markers in Brassica species; Genome;  citation_year=2011;  citation_volume=54;  citation_pages=763-70; " /><meta name="citation_reference" content="citation_title=Microsatellites in Brassica unigenes: relative abundance, marker design, and use in comparative physical mapping and genome analysis; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=55-67; " /><meta name="citation_reference" content="citation_title=Development of a core set of single-locus SSR markers for allotetraploid rapeseed (Brassica napus L.); Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=937-47; " /><meta name="citation_reference" content="citation_title=The genome of the mesopolyploid crop species Brassica rapa.; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1035-9; " /><meta name="citation_reference" content="citation_title=A genome-wide microsatellite polymorphism database for the indica and japonica rice; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=37-45; " /><meta name="citation_reference" content="citation_title=Microsatellite primer resource for Populus developed from the mapped sequence scaffolds of the Nisqually-1 genome; New Phytol.;  citation_year=2009;  citation_volume=181;  citation_pages=498-503; " /><meta name="citation_reference" content="citation_title=Genome-wide characterization of simple sequence repeats in cucumber (Cucumis sativus L.); BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=569" /><meta name="citation_reference" content="citation_title=Genome-wide distribution and organization of microsatellites in plants: an insight into marker development in Brachypodium; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e21298" /><meta name="citation_reference" content="citation_title=Genome-wide development and use of microsatellite markers for large-scale genotyping applications in foxtail millet [Setaria italica (L.)]; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=197-207; " /><meta name="citation_reference" content="citation_title=Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (Hordeum vulgare L.); Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=411-22; " /><meta name="citation_reference" content="citation_title=A web server for performing electronic PCR; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=W108-112; " /><meta name="citation_reference" content="citation_title=Development of a set of public SSR markers derived from genomic sequence of a rapid cycling Brassica oleracea L. genotype; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=117;  citation_pages=977-85; " /><meta name="citation_reference" content="citation_title=Using microsatellites to understand the physical distribution of recombination on soybean chromosomes; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e22306" /><meta name="citation_reference" content="citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes; Nat. Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=194-200; " /><meta name="citation_reference" content="citation_title=Chromosome triplication found across the tribe Brassiceae.; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=516-25; " /><meta name="citation_reference" content="citation_title=Genome-wide comparative analysis of the Brassica rapa gene space reveals genome shrinkage and differential loss of duplicated genes after whole genome triplication; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R111" /><meta name="citation_reference" content="citation_title=Comparative analysis between homoeologous genome segments of Brassica napus and its progenitor species reveals extensive sequence-level divergence; Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=1912-28; " /><meta name="citation_reference" content="citation_title=Microsatellite isolation and marker development in carrotâ€”genomic distribution, linkage mapping, genetic diversity analysis and marker transferability across Apiaceae.; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=386" /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of microsatellite distribution in plants: insight from the comparison of sequenced brassica, Arabidopsis and other angiosperm species; PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e59988" /><meta name="citation_reference" content="citation_title=Rates of nucleotide substitution in angiosperm mitochondrial DNA sequences and dates of divergence between Brassica and other angiosperm lineages; J. Mol. Evol.;  citation_year=1999;  citation_volume=48;  citation_pages=597-604; " /><meta name="citation_reference" content="citation_title=Microsatellite markers for genome analysis in Brassica. II. Assignment of rapeseed microsatellites to the A and C genomes and genetic mapping in Brassica oleracea L; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=695-9; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in plants: features and applications; Trends Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Development and application of microsatellites in candidate genes related to wood properties in the Chinese white poplar (Populus tomentosa Carr.); DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=31-44; " /><meta name="citation_reference" content="citation_title=Development and linkage mapping of unigene-derived microsatellite markers in Brassica rapa L.; Breeding Sci.;  citation_year=2011;  citation_volume=61;  citation_pages=160-7; " /><meta name="citation_reference" content="citation_title=Transferability, amplification quality, and genome specificity of microsatellites in Brassica carinata and related species; J. Appl. Genet;  citation_year=2010;  citation_volume=51;  citation_pages=123-31; " /><meta name="citation_reference" content="citation_title=Development and mapping of 2240 new SSR markers for rice (Oryza sativa L.); DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=199-207; " /><meta name="citation_reference" content="citation_title=Origin, evolution and genome distribution of microsatellites; Genet. Mol. Biol.;  citation_year=2006;  citation_volume=29;  citation_pages=294-307; " /><meta name="citation_reference" content="citation_title=Differential distribution of simple sequence repeats in eukaryotic genome sequences; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=1161-7; " /><meta name="citation_reference" content="citation_title=The mutation rates of di-, tri- and tetranucleotide repeats in Drosophila melanogaster; Mol. Biol. Evol.;  citation_year=1998;  citation_volume=15;  citation_pages=1751-60; " /><meta name="citation_reference" content="citation_title=Simple sequence repeat marker development from bacterial artificial chromosome end sequences and expressed sequence tags of flax (Linum usitatissimum L.); Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=685-94; " /><meta name="citation_reference" content="citation_title=Chromosome organization and genic expression; Cold Spring Harb. Symp. Quant. Biol.;  citation_year=1951;  citation_volume=16;  citation_pages=13-47; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/1/53/994825/dst040.pdf" /><meta name="description" content="Abstract.  Although much research has been conducted, the pattern of microsatellite distribution has remained ambiguous, and the development/utilization of micr" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/1/53/347430" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/1/53/347430" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dst040';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347430" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFt5EBPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGksIEppYXFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE9pbCBDcm9wIEJpb2xvZ3kgb2YgdGhlIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBPaWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIHRoZSBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVhbmcsIFNodW5tb3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBPaWwgQ3JvcCBCaW9sb2d5IG9mIHRoZSBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgT2lsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiB0aGUgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW4sIEppZXBlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBPaWwgQ3JvcCBCaW9sb2d5IG9mIHRoZSBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgT2lsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiB0aGUgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1LCBKaW5neWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgT2lsIENyb3AgQmlvbG9neSBvZiB0aGUgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE9pbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBYaW5mYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE9pbCBDcm9wIEJpb2xvZ3kgb2YgdGhlIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBPaWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIHRoZSBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVhLCBXZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBPaWwgQ3JvcCBCaW9sb2d5IG9mIHRoZSBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgT2lsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiB0aGUgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgU2hlbmd5aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE9pbCBDcm9wIEJpb2xvZ3kgb2YgdGhlIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBPaWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIHRoZSBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBHdWlodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBPaWwgQ3JvcCBCaW9sb2d5IG9mIHRoZSBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgT2lsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiB0aGUgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIEhhbnpob25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgT2lsIENyb3AgQmlvbG9neSBvZiB0aGUgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE9pbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS1XaWRlIE1pY3Jvc2F0ZWxsaXRlIENoYXJhY3Rlcml6YXRpb24gYW5kIE1hcmtlciBEZXZlbG9wbWVudCBpbiB0aGUgU2VxdWVuY2VkIEJyYXNzaWNhIENyb3AgU3BlY2llcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI2OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDA0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIHNlcXVlbmNlcyBhcmUgdWJpcXVpdG91cyByZXBldGl0aXZlIGNvbXBvbmVudHMgb2YgZXVrYXJ5b3RpYyBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00MTI3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXM6IHNpbXBsZSBzZXF1ZW5jZXMgd2l0aCBjb21wbGV4IGV2b2x1dGlvbjsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXM6IGdlbm9taWMgZGlzdHJpYnV0aW9uLCBwdXRhdGl2ZSBmdW5jdGlvbnMgYW5kIG11dGF0aW9uYWwgbWVjaGFuaXNtczogYSByZXZpZXc7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNDUzLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXMgd2l0aGluIGdlbmVzOiBzdHJ1Y3R1cmUsIGZ1bmN0aW9uLCBhbmQgZXZvbHV0aW9uOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTkxLTEwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhYmxlIHRhbmRlbSByZXBlYXRzIGFjY2VsZXJhdGUgZXZvbHV0aW9uIG9mIGNvZGluZyBhbmQgcmVndWxhdG9yeSBzZXF1ZW5jZXM7IEFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9NDQ1LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQ1ItYW1wbGlmaWVkIG1pY3Jvc2F0ZWxsaXRlcyBhcyBtYXJrZXJzIGluIHBsYW50IGdlbmV0aWNzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTc1LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGV2ZWxvcG1lbnQgYW5kIHVzZSBvZiBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZvciBnZW5ldGljIGFuYWx5c2lzIGFuZCBwbGFudCBicmVlZGluZyB3aXRoIGVtcGhhc2lzIG9uIGJyZWFkIHdoZWF0OyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9MTYzLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmQgYSB1bmlmaWVkIGFwcHJvYWNoIHRvIGdlbmV0aWMgbWFwcGluZyBvZiBldWthcnlvdGVzIGJhc2VkIG9uIHNlcXVlbmNlIHRhZ2dlZCBtaWNyb3NhdGVsbGl0ZSBzaXRlczsgQmlvdGVjaG5vbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTMwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlY2lwaGVyaW5nIHRoZSBkaXBsb2lkIGFuY2VzdHJhbCBnZW5vbWUgb2YgdGhlIE1lc29oZXhhcGxvaWQgQnJhc3NpY2EgcmFwYS47IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNTQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBCZXJsaW4sIEdlcm1hbnk7IEJyZWVkaW5nIE9pbHNlZWQgQnJhc3NpY2FzOyAgY2l0YXRpb25feWVhcj0xOTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5hbHlzaXMgaW4gQnJhc3NpY2Egd2l0aCBzcGVjaWFsIHJlZmVyZW5jZSB0byB0aGUgZXhwZXJpbWVudGFsIGZvcm1hdGlvbiBvZiBCLiBuYXB1cyBhbmQgcGVjdWxpYXIgbW9kZSBvZiBmZXJ0aWxpemF0aW9uOyBjaXRhdGlvbl9hdXRob3I9VSwgTi47IEphcGFuIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTE5MzU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00NTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBsYXJnZS1zY2FsZSBkZXZlbG9wbWVudCBvZiBtaWNyb3NhdGVsbGl0ZXMgZm9yIG1hcmtlciBhbmQgbWFwcGluZyBhcHBsaWNhdGlvbnMgaW4gQnJhc3NpY2EgY3JvcCBzcGVjaWVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTExMDMtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKSBnZW5ldGljIG1hcCB3aXRoIFNTUiBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTE1MTQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIGdlbm9tZSBhbmFseXNpcyBpbiBCcmFzc2ljYS4gSS4gRGV2ZWxvcG1lbnQgaW4gQnJhc3NpY2EgbmFwdXMgYW5kIGFidW5kYW5jZSBpbiBCcmFzc2ljYWNlYWUgc3BlY2llczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz02ODktOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gQnJhc3NpY2EgcmFwYSBMOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTEwOTItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXN0aWNzIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiBCcmFzc2ljYSByYXBhIGdlbm9tZSBhbmQgdGhlaXIgcG90ZW50aWFsIHV0aWxpemF0aW9uIGZvciBjb21wYXJhdGl2ZSBnZW5vbWljcyBpbiBDcnVjaWZlcmFlOyBCcmVlZGluZyBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9ODUtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFidW5kYW5jZSBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBzaW1wbGUtc2VxdWVuY2UgcmVwZWF0cyAoU1NScykgaXNvbGF0ZWQgZnJvbSBhIHNpemUtZnJhY3Rpb25hdGVkIGdlbm9taWMgbGlicmFyeSBvZiBCcmFzc2ljYSBuYXB1cyBMIChyYXBlc2VlZCk7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0yMDYtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHBvbHltb3JwaGljLCBjb25zZXJ2ZWQgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgKFNTUnMpIGluIGN1bHRpdmF0ZWQgQnJhc3NpY2Egc3BlY2llczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTUzNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnVuZGFuY2UsIHBvbHltb3JwaGlzbSBhbmQgZ2VuZXRpYyBtYXBwaW5nIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKTsgUGxhbnQgQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MzIzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhbiBpbnRlZ3JhdGVkIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIHRoZSBBIGdlbm9tZSBvZiBCcmFzc2ljYSBuYXB1cyB1c2luZyBTU1IgbWFya2VycyBkZXJpdmVkIGZyb20gc2VxdWVuY2VkIEJBQ3MgaW4gQi4gcmFwYTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTk0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBkaXN0cmlidXRpb24gb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gQnJhc3NpY2EgcmFwYS47IE1vbC4gQ2VsbHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0zNDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBtYXJrZXJzIGRlcml2ZWQgaW4gc2lsaWNvIGZyb20gQnJhc3NpY2EgcmFwYSBiYWN0ZXJpYWwgYXJ0aWNhbCBjaHJvbW9zb21lIHNlcXVlbmNlcyBhbmQgdGhlaXIgYXBwbGljYXRpb24gaW4gQnJhc3NpY2EgbmFwdXM7IE1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTI3My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVmZXJlbmNlIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIHRoZSBtdWx0aW5hdGlvbmFsIEJyYXNzaWNhIHJhcGEgZ2Vub21lIHNlcXVlbmNpbmcgcHJvamVjdDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz03NzctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBnZW5ldGljIG1hcHBpbmcgb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmcm9tIGdlbm9tZSBzdXJ2ZXkgc2VxdWVuY2VzIGluIEJyYXNzaWNhIG5hcHVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTExMjEtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgKFNTUikgbWFya2VycyBkZXJpdmVkIGluIHNpbGljbyBmcm9tIEJyYXNzaWNhIG9sZXJhY2VhIGdlbm9tZSBzaG90Z3VuIHNlcXVlbmNlczsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTE5MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgZ2VuZXRpYyBtYXBwaW5nIG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZnJvbSB3aG9sZSBnZW5vbWUgc2hvdGd1biBzZXF1ZW5jZXMgaW4gQnJhc3NpY2Egb2xlcmFjZWE7IE1vbC4gQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01ODUtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyBmcm9tIEJyYXNzaWNhIG5hcHVzIGV4cHJlc3NlZCBzZXF1ZW5jZXM7IE1vbC4gRWNvbC4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTg4Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaXh0ZWVuIG5ldyBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IG1hcmtlcnMgZnJvbSBCcmFzc2ljYSBqdW5jZWEgZXhwcmVzc2VkIHNlcXVlbmNlcyBhbmQgdGhlaXIgY3Jvc3Mtc3BlY2llcyBhbXBsaWNhdGlvbjsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9Njk3LTcwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuaWMgbWljcm9zYXRlbGxpdGUgbWFya2VycyBpbiBCcmFzc2ljYSByYXBhOiBkZXZlbG9wbWVudCwgY2hhcmFjdGVyaXphdGlvbiwgbWFwcGluZywgYW5kIHRoZWlyIHV0aWxpdHkgaW4gb3RoZXIgY3VsdGl2YXRlZCBhbmQgd2lsZCBCcmFzc2ljYSByZWxhdGl2ZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzA1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnVuZGFuY2UsIG1hcmtlciBkZXZlbG9wbWVudCBhbmQgZ2VuZXRpYyBtYXBwaW5nIG9mIG1pY3Jvc2F0ZWxsaXRlcyBmcm9tIHVuaWdlbmVzIGluIEJyYXNzaWNhIG5hcHVzLjsgTW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTczMS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgZGV2ZWxvcG1lbnQgb2YgZnVuY3Rpb25hbCBtYXJrZXJzIGluIEJyYXNzaWNhIHNwZWNpZXM7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTc2My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzIGluIEJyYXNzaWNhIHVuaWdlbmVzOiByZWxhdGl2ZSBhYnVuZGFuY2UsIG1hcmtlciBkZXNpZ24sIGFuZCB1c2UgaW4gY29tcGFyYXRpdmUgcGh5c2ljYWwgbWFwcGluZyBhbmQgZ2Vub21lIGFuYWx5c2lzOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz01NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYSBjb3JlIHNldCBvZiBzaW5nbGUtbG9jdXMgU1NSIG1hcmtlcnMgZm9yIGFsbG90ZXRyYXBsb2lkIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9OTM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBtZXNvcG9seXBsb2lkIGNyb3Agc3BlY2llcyBCcmFzc2ljYSByYXBhLjsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21lLXdpZGUgbWljcm9zYXRlbGxpdGUgcG9seW1vcnBoaXNtIGRhdGFiYXNlIGZvciB0aGUgaW5kaWNhIGFuZCBqYXBvbmljYSByaWNlOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBwcmltZXIgcmVzb3VyY2UgZm9yIFBvcHVsdXMgZGV2ZWxvcGVkIGZyb20gdGhlIG1hcHBlZCBzZXF1ZW5jZSBzY2FmZm9sZHMgb2YgdGhlIE5pc3F1YWxseS0xIGdlbm9tZTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTgxOyAgY2l0YXRpb25fcGFnZXM9NDk4LTUwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgY2hhcmFjdGVyaXphdGlvbiBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiBjdWN1bWJlciAoQ3VjdW1pcyBzYXRpdnVzIEwuKTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTY5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZGlzdHJpYnV0aW9uIGFuZCBvcmdhbml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGVzIGluIHBsYW50czogYW4gaW5zaWdodCBpbnRvIG1hcmtlciBkZXZlbG9wbWVudCBpbiBCcmFjaHlwb2RpdW07IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjEyOTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkZXZlbG9wbWVudCBhbmQgdXNlIG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIGxhcmdlLXNjYWxlIGdlbm90eXBpbmcgYXBwbGljYXRpb25zIGluIGZveHRhaWwgbWlsbGV0IFtTZXRhcmlhIGl0YWxpY2EgKEwuKV07IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTk3LTIwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9pdGluZyBFU1QgZGF0YWJhc2VzIGZvciB0aGUgZGV2ZWxvcG1lbnQgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgZ2VuZS1kZXJpdmVkIFNTUi1tYXJrZXJzIGluIGJhcmxleSAoSG9yZGV1bSB2dWxnYXJlIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz00MTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgd2ViIHNlcnZlciBmb3IgcGVyZm9ybWluZyBlbGVjdHJvbmljIFBDUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9VzEwOC0xMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgc2V0IG9mIHB1YmxpYyBTU1IgbWFya2VycyBkZXJpdmVkIGZyb20gZ2Vub21pYyBzZXF1ZW5jZSBvZiBhIHJhcGlkIGN5Y2xpbmcgQnJhc3NpY2Egb2xlcmFjZWEgTC4gZ2Vub3R5cGU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9OTc3LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBtaWNyb3NhdGVsbGl0ZXMgdG8gdW5kZXJzdGFuZCB0aGUgcGh5c2ljYWwgZGlzdHJpYnV0aW9uIG9mIHJlY29tYmluYXRpb24gb24gc295YmVhbiBjaHJvbW9zb21lczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMjMwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyBhcmUgcHJlZmVyZW50aWFsbHkgYXNzb2NpYXRlZCB3aXRoIG5vbnJlcGV0aXRpdmUgRE5BIGluIHBsYW50IGdlbm9tZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTk0LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZSB0cmlwbGljYXRpb24gZm91bmQgYWNyb3NzIHRoZSB0cmliZSBCcmFzc2ljZWFlLjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz01MTYtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBCcmFzc2ljYSByYXBhIGdlbmUgc3BhY2UgcmV2ZWFscyBnZW5vbWUgc2hyaW5rYWdlIGFuZCBkaWZmZXJlbnRpYWwgbG9zcyBvZiBkdXBsaWNhdGVkIGdlbmVzIGFmdGVyIHdob2xlIGdlbm9tZSB0cmlwbGljYXRpb247IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIxMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBiZXR3ZWVuIGhvbW9lb2xvZ291cyBnZW5vbWUgc2VnbWVudHMgb2YgQnJhc3NpY2EgbmFwdXMgYW5kIGl0cyBwcm9nZW5pdG9yIHNwZWNpZXMgcmV2ZWFscyBleHRlbnNpdmUgc2VxdWVuY2UtbGV2ZWwgZGl2ZXJnZW5jZTsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE5MTItMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIGlzb2xhdGlvbiBhbmQgbWFya2VyIGRldmVsb3BtZW50IGluIGNhcnJvdOKAlGdlbm9taWMgZGlzdHJpYnV0aW9uLCBsaW5rYWdlIG1hcHBpbmcsIGdlbmV0aWMgZGl2ZXJzaXR5IGFuYWx5c2lzIGFuZCBtYXJrZXIgdHJhbnNmZXJhYmlsaXR5IGFjcm9zcyBBcGlhY2VhZS47IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTM4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBkeW5hbWljcyBvZiBtaWNyb3NhdGVsbGl0ZSBkaXN0cmlidXRpb24gaW4gcGxhbnRzOiBpbnNpZ2h0IGZyb20gdGhlIGNvbXBhcmlzb24gb2Ygc2VxdWVuY2VkIGJyYXNzaWNhLCBBcmFiaWRvcHNpcyBhbmQgb3RoZXIgYW5naW9zcGVybSBzcGVjaWVzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTU5OTg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF0ZXMgb2YgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb24gaW4gYW5naW9zcGVybSBtaXRvY2hvbmRyaWFsIEROQSBzZXF1ZW5jZXMgYW5kIGRhdGVzIG9mIGRpdmVyZ2VuY2UgYmV0d2VlbiBCcmFzc2ljYSBhbmQgb3RoZXIgYW5naW9zcGVybSBsaW5lYWdlczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTU5Ny02MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIGdlbm9tZSBhbmFseXNpcyBpbiBCcmFzc2ljYS4gSUkuIEFzc2lnbm1lbnQgb2YgcmFwZXNlZWQgbWljcm9zYXRlbGxpdGVzIHRvIHRoZSBBIGFuZCBDIGdlbm9tZXMgYW5kIGdlbmV0aWMgbWFwcGluZyBpbiBCcmFzc2ljYSBvbGVyYWNlYSBMOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTY5NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBsYW50czogZmVhdHVyZXMgYW5kIGFwcGxpY2F0aW9uczsgVHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBhcHBsaWNhdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gY2FuZGlkYXRlIGdlbmVzIHJlbGF0ZWQgdG8gd29vZCBwcm9wZXJ0aWVzIGluIHRoZSBDaGluZXNlIHdoaXRlIHBvcGxhciAoUG9wdWx1cyB0b21lbnRvc2EgQ2Fyci4pOyBETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzEtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBsaW5rYWdlIG1hcHBpbmcgb2YgdW5pZ2VuZS1kZXJpdmVkIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgaW4gQnJhc3NpY2EgcmFwYSBMLjsgQnJlZWRpbmcgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTE2MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2ZlcmFiaWxpdHksIGFtcGxpZmljYXRpb24gcXVhbGl0eSwgYW5kIGdlbm9tZSBzcGVjaWZpY2l0eSBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gQnJhc3NpY2EgY2FyaW5hdGEgYW5kIHJlbGF0ZWQgc3BlY2llczsgSi4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0xMjMtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBtYXBwaW5nIG9mIDIyNDAgbmV3IFNTUiBtYXJrZXJzIGZvciByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTk5LTIwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luLCBldm9sdXRpb24gYW5kIGdlbm9tZSBkaXN0cmlidXRpb24gb2YgbWljcm9zYXRlbGxpdGVzOyBHZW5ldC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI5NC0zMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBkaXN0cmlidXRpb24gb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gZXVrYXJ5b3RpYyBnZW5vbWUgc2VxdWVuY2VzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTE2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbXV0YXRpb24gcmF0ZXMgb2YgZGktLCB0cmktIGFuZCB0ZXRyYW51Y2xlb3RpZGUgcmVwZWF0cyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VyIGRldmVsb3BtZW50IGZyb20gYmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZSBlbmQgc2VxdWVuY2VzIGFuZCBleHByZXNzZWQgc2VxdWVuY2UgdGFncyBvZiBmbGF4IChMaW51bSB1c2l0YXRpc3NpbXVtIEwuKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz02ODUtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWUgb3JnYW5pemF0aW9uIGFuZCBnZW5pYyBleHByZXNzaW9uOyBDb2xkIFNwcmluZyBIYXJiLiBTeW1wLiBRdWFudC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5NTE7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzEvNTMvOTk0ODI1L2RzdDA0MC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWx0aG91Z2ggbXVjaCByZXNlYXJjaCBoYXMgYmVlbiBjb25kdWN0ZWQsIHRoZSBwYXR0ZXJuIG9mIG1pY3Jvc2F0ZWxsaXRlIGRpc3RyaWJ1dGlvbiBoYXMgcmVtYWluZWQgYW1iaWd1b3VzLCBhbmQgdGhlIGRldmVsb3BtZW50L3V0aWxpemF0aW9uIG9mIG1pY3IiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0NzQzMCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGS3pYJ3Rq/c+8FxFIi4WcD/QiOblQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f53%2f347430" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f53%2f347430" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1509238921&amp;Signature=IMKeALqTlsSZee8zSWyElxMZ24su6PXBeAFCnAN~JpI~fanlONPgxIgL~Qx5SbWzWWiwpdQtKF0l6AcCKhCcoUhrscPsn0niVRpvCg81VB5CM54T9x9H8Pune-Wimysq4lPDokxJ7tyaj7qXHGrX9NPGpxhIzSyXAlvVP~-I6a~~5tsUB7fYwRT-gKzby9LEc4-6dOmnEIAbsOGZKE0UEnStyuiGZ-VpykivzCxhZFEiu~frZPVMowu07U5ZuFW4Mkny4~uMcOzDyRvP7GPZnt7DwaGyUlXnHObjzT8AloE5yTTLXs2WJAzl7zeoVtJ-b5y~VxDJqZT6OfNLE9A-pA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/1">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4916788">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916788">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916790">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916790">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916795">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916795">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916809">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916809">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916837">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916837">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916853">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916853">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916855">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916855">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4916857">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4916857">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/1/41/347328">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/1/69/347526">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced <em>Brassica</em> Crop Species
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jiaqin Shi</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiaqin Shi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jiaqin+Shi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shi J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Shi author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Shunmou Huang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shunmou Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shunmou+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Huang author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jiepeng Zhan</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiepeng Zhan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jiepeng+Zhan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhan J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Zhan author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jingyin Yu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jingyin Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jingyin+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yu author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Xinfa Wang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xinfa Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 027-86836265. Fax. +86 027-86836125. Email: <a href="mailto:wangxf12@sina.com" target="_blank">wangxf12@sina.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xinfa+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wang author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Wei Hua</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Hua</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Wei+Hua">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hua W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hua author:W.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Shengyi Liu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shengyi Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shengyi+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Liu author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Guihua Liu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guihua Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Guihua+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Liu author:G.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Hanzhong Wang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hanzhong Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Oil Crop Biology of the Ministry of Agriculture</div>, <div class="insititution">Oil Crops Research Institute of the Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hanzhong+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wang author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">â€ </div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 53â€“68, <a href='https://doi.org/10.1093/dnares/dst040'>https://doi.org/10.1093/dnares/dst040</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 October 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 June 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 September 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347430" href="/dnaresearch/article-pdf/21/1/53/994825/dst040.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jiaqin Shi, Shunmou Huang, Jiepeng Zhan, Jingyin Yu, Xinfa Wang, Wei Hua, Shengyi Liu, Guihua Liu, Hanzhong Wang;  Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced <em>Brassica</em> Crop Species, <em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 53â€“68, <a href='https://doi.org/10.1093/dnares/dst040'>https://doi.org/10.1093/dnares/dst040</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347430?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347430?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347430?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347430?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347430?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347430?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347430?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Microsatellite%20Characterization%20and%20Marker%20Development%20in%20the%20Sequenced%20%26lt%3Bem%26gt%3BBrassica%26lt%3B%2Fem%26gt%3B%20Crop%20Species&amp;publicationDate=2013-10-14&amp;volumeNum=21&amp;issueNum=1&amp;author=Shi%2C%20Jiaqin%3B%20Huang%2C%20Shunmou&amp;startPage=53&amp;endPage=68&amp;contentId=10.1093%2Fdnares%2Fdst040&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f53%2f347430" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4916788 id="4916788" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Although much research has been conducted, the pattern of microsatellite distribution has remained ambiguous, and the development/utilization of microsatellite markers has still been limited/inefficient in <em>Brassica</em>, due to the lack of genome sequences. In view of this, we conducted genome-wide microsatellite characterization and marker development in three recently sequenced <em>Brassica</em> crops: <em>Brassica rapa</em>, <em>Brassica oleracea</em> and <em>Brassica napus</em>. The analysed microsatellite characteristics of these <em>Brassica</em> species were highly similar or almost identical, which suggests that the pattern of microsatellite distribution is likely conservative in <em>Brassica</em>. The genomic distribution of microsatellites was highly non-uniform and positively or negatively correlated with genes or transposable elements, respectively. Of the total of 115 869, 185 662 and 356 522 simple sequence repeat (SSR) markers developed with high frequencies (408.2, 343.8 and 356.2 per Mb or one every 2.45, 2.91 and 2.81 kb, respectively), most represented new SSR markers, the majority had determined physical positions, and a large number were genic or putative single-locus SSR markers. We also constructed a comprehensive database for the newly developed SSR markers, which was integrated with public <em>Brassica</em> SSR markers and annotated genome components. The genome-wide SSR markers developed in this study provide a useful tool to extend the annotated genome resources of sequenced <em>Brassica</em> species to genetic study/breeding in different <em>Brassica</em> species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="brassica">brassica</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microsatellite">microsatellite</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="distribution">distribution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="marker">marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="database">database</a></div>                            <h2 scrollto-destination=4916790 id="4916790" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microsatellites, which are also known as simple sequence repeats (SSRs, often defined as 1â€“6 bp), variable numbers of tandem repeats (VNTRs) and short tandem repeats (STRs), have been found in all genomic regions of all examined organisms.<sup><a reveal-id="DST040C1" data-open="DST040C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Microsatellites have been traditionally regarded as â€˜junkâ€™ DNA and are mainly used as â€˜neutralâ€™ genetic markers.<sup><a reveal-id="DST040C2" data-open="DST040C2" class="link link-ref link-reveal xref-bibr">2</a></sup> In recent years, microsatellites have been demonstrated to have many important biological functions (e.g. the regulation of chromatin organization, DNA metabolic processes, gene activity and RNA structure)<sup><a reveal-id="DST040C3" data-open="DST040C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST040C4" data-open="DST040C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and have therefore emerged as the third major class of genetic variations, alongside single nucleotide polymorphisms (SNPs) and copy number variations (CNVs).<sup><a reveal-id="DST040C5" data-open="DST040C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Microsatellite markers are co-dominant, multi-allelic, easily detected, hyper-variable, highly reproducible and abundant in the genome.<sup><a reveal-id="DST040C6" data-open="DST040C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Therefore, among the available genetic marker systems (e.g. RFLP, RAPD, SSR, AFLP, SRAP and SNP), the SSR marker has been the preferential choice for various applications, such as variety identification, genetic diversity evaluation, phylogenetic relationship analysis, genetic map construction, linkage/association mapping of gene/QTL, marker-assisted selection and comparative mapping.<sup><a reveal-id="DST040C7" data-open="DST040C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST040C8" data-open="DST040C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>Of the 47 genera in the <em>Brassiceae</em> tribe within the <em>Brassicaceae</em> (<em>Cruciferae</em>) family, the genus <em>Brassica</em> currently comprises 38 species,<sup><a reveal-id="DST040C9" data-open="DST040C9" class="link link-ref link-reveal xref-bibr">9</a></sup> which include economically important crops that provide many vegetables, condiments, fodders and oil products.<sup><a reveal-id="DST040C10" data-open="DST040C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The main cultivated <em>Brassica</em> species include three diploid species, <em>Brassica rapa</em> (AA, <em>n</em> = 10), <em>Brassica nigra</em> (BB, <em>n</em> = 8) and <em>Brassica oleracea</em> (CC, <em>n</em> = 9) and three allotetraploid species, <em>Brassica juncea</em> (AABB, <em>n</em> = 18), <em>Brassica napus</em> (AACC, <em>n</em> = 19) and <em>Brassica carinata</em> (BBCC, <em>n</em> = 17). The genetic relationship of the six widely cultivated <em>Brassica</em> species are described as U's triangle<sup><a reveal-id="DST040C11" data-open="DST040C11" class="link link-ref link-reveal xref-bibr">11</a></sup> in which the three allotetraploid species originated from the chromosome doubling after the natural hybridization between the three diploid species.</p><p>Much research has been conducted to identify/characterize genomic/genic microsatellites and/or to develop markers in the <em>Brassica</em> species through probe (containing a repeated motif) hybridization against genomic/cDNA clones<sup><a reveal-id="DST040C12  DST040C19" data-open="DST040C12  DST040C19" class="link link-ref link-reveal xref-bibr">12â€“19</a></sup> or through <em>in silico</em> analysis of publicly available bacterial artificial chromosome (BAC) sequences,<sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> BAC-end sequences (BESs),<sup><a reveal-id="DST040C21  DST040C23" data-open="DST040C21  DST040C23" class="link link-ref link-reveal xref-bibr">21â€“23</a></sup> genome survey sequences (GSSs),<sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a></sup> whole genome shotgun sequences (WGSs),<sup><a reveal-id="DST040C25" data-open="DST040C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> expressed sequence tag sequences<sup><a reveal-id="DST040C27" data-open="DST040C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST040C28" data-open="DST040C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and unique transcript sequences.<sup><a reveal-id="DST040C29  DST040C32" data-open="DST040C29  DST040C32" class="link link-ref link-reveal xref-bibr">29â€“32</a></sup> However, the pattern of microsatellite distribution has remained ambiguous, and the development/utilization of SSR marker has still been limited/inefficient in <em>Brassica</em>, which is mostly due to the lack of genome sequences. First, the sequences, programmes, criteria and parameters that are used for mining microsatellites usually have differed across these previous studies, which have made it difficult to compare and integrate these results to obtain the definitive conclusions on the pattern of microsatellite distribution. Secondly, only a small part of the genomic sequences of usually one species have been analysed in each of these previous studies. Therefore, it has been impossible to obtain general conclusions on the pattern of microsatellite distribution. In addition, the total number (â‰ˆ10 000) of previously developed publicly available SSR markers is still limited<sup><a reveal-id="DST040C33" data-open="DST040C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and not sufficient for many studies, which require a large number and/or high density of genetic markers, such as high-density linkage map construction, gene/QTL fine-mapping and genome-wide/regional association mapping. Thirdly, due to the lack of genome sequences, the genomic distribution of microsatellites and the physical position(s)/product(s) number of the previously developed publicly available <em>Brassica</em> SSR markers have been all or mostly unclear, which has hindered their exact and/or effective utilization.</p><p>Thanks to the rapid development of genome sequencing technology, the genome sequences are currently available for tens of plant species (<a class="link link-uri" href="http://www.phytozome.net">http://www.phytozome.net</a>), including three recently sequenced <em>Brassica</em> crop species, namely <em>B. rapa</em>,<sup><a reveal-id="DST040C34" data-open="DST040C34" class="link link-ref link-reveal xref-bibr">34</a></sup><em>B. oleracea</em> (<a class="link link-uri" href="http://www.ocri-genomics.org/bolbase/index.html">http://www.ocri-genomics.org/bolbase/index.html</a>) and <em>B. napus</em> (our unpublished data). These sequences provide a powerful tool for genome-wide microsatellite characterization and/or marker development, which has been conducted in several model and crop plants, such as <em>Arabidopsis</em> (<a class="link link-uri" href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>), rice,<sup><a reveal-id="DST040C35" data-open="DST040C35" class="link link-ref link-reveal xref-bibr">35</a></sup> maize (mips.helmholtz-muenchen.de/plant/maize/), sorghum (genome.jgi-psf.org/Sorbi1/Sorbi1.home.htm), black cottonwood,<sup><a reveal-id="DST040C36" data-open="DST040C36" class="link link-ref link-reveal xref-bibr">36</a></sup> cucumber,<sup><a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a></sup><em>Brachypodim distachyon</em><sup><a reveal-id="DST040C38" data-open="DST040C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and foxtail millet<sup><a reveal-id="DST040C39" data-open="DST040C39" class="link link-ref link-reveal xref-bibr">39</a></sup> but not <em>Brassica</em>. In view of this circumstance, we conducted genome-wide microsatellite characterization and marker development in the three sequenced <em>Brassica</em> crop species. The main objectives of this study were as follows: (i) to characterize and compare the frequency and distribution with respect to the motif length, type and repeat number of microsatellites in the assembled genomic sequences of these <em>Brassica</em> species; (ii) to characterize and compare the genomic distribution of microsatellites in the assembled pseudochromosomes of these <em>Brassica</em> species; (iii) to develop SSR markers from the assembled genomic sequences of these <em>Brassica</em> species and determine their copy number and positional relationship with the previously developed publicly available <em>Brassica</em> SSR markers and the annotated genome components; (iv) to construct a user-friendly comprehensive SSR marker database of <em>Brassica</em> and (v) to evaluate the newly developed genome-wide SSR markers by PCR (polymerase chain reaction) amplification in representative <em>B. napus</em> inbred lines.</p>                            <h2 scrollto-destination=4916795 id="4916795" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4916796 id="4916796" class="section-title" data-legacy-id=s2a>2.1. Sources of genome sequences</h3>
<p>The three inbred/pure lines, namely Chiifu-401 (<em>B. rapa</em>), O212 (<em>B. oleracea</em>) and Zhongshuang11 (<em>B. napus</em>), were sequenced by our own and several other institutes using Illumina GA II technology, and high-quality sequence reads were assembled using stringent parameters. Finally, a total of 40 549 (283.8 Mb), 120 061 (540.0 Mb) and 5098 (1000.9 Mb) sequence scaffolds were obtained for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively, which represents 58.5, 77.6 and 81.7% of the nuclear genome and covers &gt;98% of the gene space.</p>                            <h3 scrollto-destination=4916798 id="4916798" class="section-title" data-legacy-id=s2b>2.2. Identification of microsatellites</h3>
<p>PERL5 script MIcroSAtellite (<a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>)<sup><a reveal-id="DST040C40" data-open="DST040C40" class="link link-ref link-reveal xref-bibr">40</a></sup> was used to identify and localize perfect microsatellites as well as compound microsatellites that are interrupted by a certain number of bases. The repeat unit length was defined as the default mono- to hexanucleotide because microsatellites of longer repeat units are very scarce. The minimum repeat unit was defined as 12, 6, 4, 3, 3 and 3, respectively, for the mono- to hexanucleotide. Compound microsatellites were defined as â‰¥2 repeats interrupted by â‰¤100 bp.</p>                            <h3 scrollto-destination=4916800 id="4916800" class="section-title" data-legacy-id=s2c>2.3. Development of SSR primers</h3>
<p>Primer pairs were designed from the flanking sequences of identified microsatellites using the primer3_core program (<a class="link link-uri" href="http://www-genome.wi.mit.edu/cgi-bin/primer/primer3_">http://www-genome.wi.mit.edu/cgi-bin/primer/primer3_</a><a class="link link-uri" href="http://www.cgi">www.cgi</a>) in batch mode. Two perl scripts, p3_in.pl and p3_out.pl, serve as interface modules for the programme-to-programme data interchange between MISA and the primer modeling software Primer3. The primer-designing parameters were 18â€“27 bp primer length, 57â€“63Â°C melting temperature, 30â€“70% GC content and 100â€“300 bp product size. The designed SSR primer pairs were denominated as the names of sequence scaffolds followed by a serial number of microsatellites (such as BrScaffold000001_1).</p>                            <h3 scrollto-destination=4916802 id="4916802" class="section-title" data-legacy-id=s2d>2.4. Localization/mapping of SSR markers by <em>in silico</em> PCR</h3>
<p>The primer-pair sequences of previously developed publicly available <em>Brassica</em> SSR markers were downloaded from the brassica.info website (<a class="link link-uri" href="http://www.brassica.info/resource/markers/ssr-exchange.php">http://www.brassica.info/resource/markers/ssr-exchange.php</a>) and additional files in the recent literature.<sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DST040C29" data-open="DST040C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DST040C32" data-open="DST040C32" class="link link-ref link-reveal xref-bibr">32</a></sup> To determine their physical positions and copy numbers, the previously and newly developed <em>Brassica</em> SSR markers were aligned to the assembled genomic sequences of the studied <em>Brassica</em> species. This alignment was conducted using the <em>in silico</em> PCR method<sup><a reveal-id="DST040C41" data-open="DST040C41" class="link link-ref link-reveal xref-bibr">41</a></sup> with the following default parameters: 2 bp mismatch, 1 bp gap, 50 bp margin and 50â€“1000 bp product size.</p>                            <h3 scrollto-destination=4916804 id="4916804" class="section-title" data-legacy-id=s2e>2.5. Validation of SSR markers by PCR amplification</h3>
<p>A total of 3974 SSR primer pairs were synthesized to test for PCR amplification in six representative <em>B. napus</em> cultivars/inbred lines (Tapidor, Westar, Zhongshuang11, No. 07197, No. 73290 and No. 91032), which were chosen from the core collections of a natural population and the parents of several segregating populations in our laboratory, for their large genetic distance and extreme trait(s) performance (our unpublished data).</p><p>Genomic DNA of the six accessions was isolated from young leaves. PCR was performed in 20-Âµl volume that contained 0.2 mM dNTP, 0.5 U of Taq DNA polymerase, 75 ng of template DNA, 0.5 ÂµM each primer and 1Ã— PCR buffer (10 mM Tris pH 9.0, 50 mM KCl and 1.5 mM MgCl<sub>2</sub>). DNA amplification was conducted by the â€˜touchdownâ€™ method, with the following thermal profile: initial denaturation at 94Â°C for 5 min; six cycles of 30 s at 94Â°C, 45 s at 63Â°C with a 1Â°C decrease in annealing temperature per cycle and 1 min at 72Â°C; 26 cycles of 30 s at 94Â°C, 45 s at 57Â°C and 1 min at 72Â°C and a final extension at 72Â°C for 10 min. The PCR products were separated on 6% denaturing polyacrylamide gels and were visualized by silver staining.</p>                            <h3 scrollto-destination=4916807 id="4916807" class="section-title" data-legacy-id=s2f>2.6. Statistical analysis</h3>
<p>The correlation analysis was performed using the SAS PROC CORR procedure incorporated into SAS version 8.0. The Excel statistical function CHISQ.TEST was used to obtain the significance level (<span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>) of the degree of fit for the practical and hypothetical distributions of microsatellites as well as genes and TEs in the assembled pseudochromosomes.</p>                            <h2 scrollto-destination=4916809 id="4916809" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4916810 id="4916810" class="section-title" data-legacy-id=s3a>3.1. Frequency and distribution with respect to the motif length, type and number</h3>
<p>A total of 140 998, 229 389 and 420 991 perfect mono- to hexanucleotide repeat microsatellites were identified from 283.8, 540.0 and 1000.9 Mb of assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>), with an overall frequency of 496.8, 424.8 and 420.6 per Mb or one every 2.01, 2.35 and 2.38 kb. </p>                        <a id="4916812" scrollto-destination="4916812"></a>
<div class="table-wrap"><div id="DST040TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Number, repeat number and total repeat length of the mono- to hexanucleotide repeats or motifs of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">MotifÂ </th><th colspan="3"><em>B. rapa</em><hr />Â </th><th colspan="3"><em>B. oleracea</em><hr />Â </th><th colspan="3"><em>B. napus</em><hr />Â </th></tr><tr><th>Number (%)Â </th><th>Repeat numberÂ </th><th>Total length (%)Â </th><th>Number (%)Â </th><th>Repeat numberÂ </th><th>Total length (%)Â </th><th>Number (%)Â </th><th>Repeat numberÂ </th><th>Total length (%)Â </th></tr></thead><tbody><tr><td>MonoÂ </td><td>31 258 (22.2)Â </td><td>12â€“307 (14.7)Â </td><td>458 968 (20.1)Â </td><td>55 433 (24.2)Â </td><td>3â€“65 (15.1)Â </td><td>838 104 (24.1)Â </td><td>97 128 (23.1)Â </td><td>12â€“2545 (15.2)Â </td><td>147 5939 (22.9)Â </td></tr><tr><td>AÂ </td><td>29 536 (20.9)Â </td><td>12â€“50 (14.5)Â </td><td>428 733 (18.7)Â </td><td>52 021 (22.7)Â </td><td>3â€“65 (15.0)Â </td><td>780 171 (22.5)Â </td><td>94 281 (22.4)Â </td><td>12â€“2545 (15.2)Â </td><td>1 432 867 (22.2)Â </td></tr><tr><td>CÂ </td><td>1722 (1.2)Â </td><td>12â€“307 (17.6)Â </td><td>30 235 (1.3)Â </td><td>3412 (1.5)Â </td><td>3â€“63 (17.0)Â </td><td>57 933 (1.7)Â </td><td>2847 (0.7)Â </td><td>12â€“83 (15.1)Â </td><td>43 072 (0.7)Â </td></tr><tr><td>DiÂ </td><td>33 885 (24.0)Â </td><td>6â€“3644 (11.1)Â </td><td>751 910 (32.9)Â </td><td>55 336 (24.1)Â </td><td>6â€“82 (8.8)Â </td><td>968 946 (27.9)Â </td><td>98 816 (23.5)Â </td><td>6â€“5556 (9.1)Â </td><td>1 789 752 (27.8)Â </td></tr><tr><td>ATÂ </td><td>19 697 (14.0)Â </td><td>6â€“419 (8.9)Â </td><td>350 590 (15.3)Â </td><td>33 315 (14.5)Â </td><td>6â€“63 (8.9)Â </td><td>596 070 (17.2)Â </td><td>57 070 (13.6)Â </td><td>6â€“5556 (8.7)Â </td><td>996 518 (15.5)Â </td></tr><tr><td>AGÂ </td><td>11 683 (8.3)Â </td><td>6â€“3644 (15.6)Â </td><td>364 004 (15.9)Â </td><td>18 593 (8.1)Â </td><td>6â€“44 (8.7)Â </td><td>322 438 (9.3)Â </td><td>34 638 (8.2)Â </td><td>6â€“1732 (9.9)Â </td><td>688 124 (10.7)Â </td></tr><tr><td>ACÂ </td><td>2490 (1.8)Â </td><td>6â€“131 (7.5)Â </td><td>37 124 (1.6)Â </td><td>3411 (1.5)Â </td><td>6â€“82 (7.4)Â </td><td>50 220 (1.4)Â </td><td>7072 (1.7)Â </td><td>6â€“92 (7.4)Â </td><td>104 648 (1.6)Â </td></tr><tr><td>CGÂ </td><td>15 (0.0)Â </td><td>6â€“8 (6.4)Â </td><td>192 (0.0)Â </td><td>17 (0.0)Â </td><td>6â€“9 (6.4)Â </td><td>218 (0.0)Â </td><td>36 (0.0)Â </td><td>6â€“8 (6.4)Â </td><td>462 (0.0)Â </td></tr><tr><td>TriÂ </td><td>32 387 (23.0)Â </td><td>4â€“812 (4.7)Â </td><td>459 039 (20.1)Â </td><td>47 716 (20.8)Â </td><td>4â€“1710 (4.7)Â </td><td>670 164 (19.3)Â </td><td>91 448 (21.7)Â </td><td>4â€“1794 (4.7)Â </td><td>1 277 172 (19.8)Â </td></tr><tr><td>AAGÂ </td><td>9796 (6.9)Â </td><td>4â€“812 (4.7)Â </td><td>139 239 (6.1)Â </td><td>15 322 (6.7)Â </td><td>4â€“81 (4.6)Â </td><td>212 973 (6.1)Â </td><td>29 395 (7.0)Â </td><td>4â€“162 (4.6)Â </td><td>408 264 (6.3)Â </td></tr><tr><td>AATÂ </td><td>6334 (4.5)Â </td><td>4â€“573 (4.9)Â </td><td>93 240 (4.1)Â </td><td>9355 (4.1)Â </td><td>4â€“1710 (5.2)Â </td><td>145 056 (4.2)Â </td><td>17 722 (4.2)Â </td><td>4â€“1794 (5.1)Â </td><td>268 956 (4.2)Â </td></tr><tr><td>ATCÂ </td><td>4211 (3.0)Â </td><td>4â€“190 (5.0)Â </td><td>63 213 (2.8)Â </td><td>6093 (2.7)Â </td><td>4â€“81 (4.7)Â </td><td>85 047 (2.4)Â </td><td>12 314 (2.9)Â </td><td>4â€“119 (4.6)Â </td><td>170 151 (2.6)Â </td></tr><tr><td>AACÂ </td><td>3637 (2.6)Â </td><td>4â€“11 (4.5)Â </td><td>49 146 (2.1)Â </td><td>5036 (2.2)Â </td><td>4â€“41 (4.4)Â </td><td>66 705 (1.9)Â </td><td>10 046 (2.4)Â </td><td>4â€“22 (4.5)Â </td><td>134 592 (2.1)Â </td></tr><tr><td>AGGÂ </td><td>3243 (2.3)Â </td><td>4â€“179 (4.6)Â </td><td>45 084 (2.0)Â </td><td>5425 (2.4)Â </td><td>4â€“13 (4.6)Â </td><td>74 763 (2.2)Â </td><td>8349 (2.0)Â </td><td>4â€“14 (4.6)Â </td><td>115 494 (1.8)Â </td></tr><tr><td>ACCÂ </td><td>2144 (1.5)Â </td><td>4â€“9 (4.5)Â </td><td>28 908 (1.3)Â </td><td>2788 (1.2)Â </td><td>4â€“43 (4.4)Â </td><td>37 155 (1.1)Â </td><td>5969 (1.4)Â </td><td>4â€“14 (4.4)Â </td><td>79 137 (1.2)Â </td></tr><tr><td>AGCÂ </td><td>1127 (0.8)Â </td><td>4â€“11 (4.4)Â </td><td>15 039 (0.7)Â </td><td>1390 (0.6)Â </td><td>4â€“22 (4.4)Â </td><td>18 507 (0.5)Â </td><td>2878 (0.7)Â </td><td>4â€“18 (4.4)Â </td><td>38 379 (0.6)Â </td></tr><tr><td>ACTÂ </td><td>674 (0.5)Â </td><td>4â€“10 (4.4)Â </td><td>8874 (0.4)Â </td><td>818 (0.4)Â </td><td>4â€“28 (4.5)Â </td><td>10 968 (0.3)Â </td><td>1754 (0.4)Â </td><td>4â€“17 (4.4)Â </td><td>23 265 (0.4)Â </td></tr><tr><td>CCGÂ </td><td>626 (0.4)Â </td><td>4â€“8 (4.4)Â </td><td>8220 (0.4)Â </td><td>799 (0.3)Â </td><td>4â€“7 (4.3)Â </td><td>10 203 (0.3)Â </td><td>1566 (0.4)Â </td><td>4â€“8 (4.3)Â </td><td>20 139 (0.3)Â </td></tr><tr><td>ACGÂ </td><td>595 (0.4)Â </td><td>4â€“9 (4.5)Â </td><td>8076 (0.4)Â </td><td>690 (0.3)Â </td><td>4â€“13 (4.2)Â </td><td>8787 (0.3)Â </td><td>1455 (0.3)Â </td><td>4â€“10 (4.3)Â </td><td>18 795 (0.3)Â </td></tr><tr><td>TetraÂ </td><td>29 433 (20.9)Â </td><td>3â€“264 (3.2)Â </td><td>376 668 (16.5)Â </td><td>48 394 (21.1)Â </td><td>3â€“54 (3.1)Â </td><td>608 452 (17.5)Â </td><td>91 268 (21.7)Â </td><td>3â€“631 (3.1)Â </td><td>1 148 496 (17.8)Â </td></tr><tr><td>AAATÂ </td><td>11 870 (8.4)Â </td><td>3â€“8 (3.2)Â </td><td>152 776 (6.7)Â </td><td>18 856 (8.2)Â </td><td>3â€“28 (3.1)Â </td><td>236 560 (6.8)Â </td><td>33 903 (8.1)Â </td><td>3â€“10 (3.1)Â </td><td>425 080 (6.6)Â </td></tr><tr><td>AAAGÂ </td><td>3496 (2.5)Â </td><td>3â€“60 (3.2)Â </td><td>44 608 (1.9)Â </td><td>5697 (2.5)Â </td><td>3â€“36 (3.2)Â </td><td>72 224 (2.1)Â </td><td>10 795 (2.6)Â </td><td>3â€“13 (3.2)Â </td><td>136 204 (2.1)Â </td></tr><tr><td>AAACÂ </td><td>3333 (2.4)Â </td><td>3â€“8 (3.1)Â </td><td>41 960 (1.8)Â </td><td>4723 (2.1)Â </td><td>3â€“26 (3.1)Â </td><td>59 076 (1.7)Â </td><td>9717 (2.3)Â </td><td>3â€“8 (3.1)Â </td><td>121 768 (1.9)Â </td></tr><tr><td>AATTÂ </td><td>2534 (1.8)Â </td><td>3â€“6 (3.1)Â </td><td>31 108 (1.4)Â </td><td>4088 (1.8)Â </td><td>3â€“10 (3.1)Â </td><td>51 416 (1.5)Â </td><td>7863 (1.9)Â </td><td>3â€“41 (3.1)Â </td><td>98 224 (1.5)Â </td></tr><tr><td>AATCÂ </td><td>1137 (0.8)Â </td><td>3â€“8 (3.2)Â </td><td>14 368 (0.6)Â </td><td>2956 (1.3)Â </td><td>3â€“27 (3.1)Â </td><td>36 908 (1.1)Â </td><td>5112 (1.2)Â </td><td>3â€“8 (3.1)Â </td><td>63 884 (1.0)Â </td></tr><tr><td>othersÂ </td><td>7063 (5.0)Â </td><td>3â€“264 (3.3)Â </td><td>91 848 (4.0)Â </td><td>12 074 (5.3)Â </td><td>3â€“54 (3.2)Â </td><td>152 268 (4.4)Â </td><td>23 878 (5.7)Â </td><td>3â€“631 (3.2)Â </td><td>303 336 (4.7)Â </td></tr><tr><td>PentaÂ </td><td>9856 (7.0)Â </td><td>3â€“114 (3.2)Â </td><td>156 510 (6.8)Â </td><td>15 012 (6.5)Â </td><td>3â€“40 (3.2)Â </td><td>241 090 (6.9)Â </td><td>29 058 (6.9)Â </td><td>3â€“28 (3.1)Â </td><td>457 065 (7.1)Â </td></tr><tr><td>AAAATÂ </td><td>2758 (2.0)Â </td><td>3â€“63 (3.1)Â </td><td>42 920 (1.9)Â </td><td>4051 (1.8)Â </td><td>3â€“18 (3.4)Â </td><td>67 905 (2.0)Â </td><td>7617 (1.8)Â </td><td>3â€“9 (3.1)Â </td><td>118 030 (1.8)Â </td></tr><tr><td>AACCGÂ </td><td>1000 (0.7)Â </td><td>3â€“68 (3.3)Â </td><td>16 510 (0.7)Â </td><td>2169 (0.9)Â </td><td>3â€“40 (3.2)Â </td><td>34 645 (1.0)Â </td><td>3541 (0.8)Â </td><td>3â€“7 (3.2)Â </td><td>56 435 (0.9)Â </td></tr><tr><td>AAAACÂ </td><td>878 (0.6)Â </td><td>3â€“7 (3.2)Â </td><td>13 905 (0.6)Â </td><td>1186 (0.5)Â </td><td>3â€“8 (3.2)Â </td><td>18 715 (0.5)Â </td><td>2424 (0.6)Â </td><td>3â€“10 (3.2)Â </td><td>38 280 (0.6)Â </td></tr><tr><td>AAAAGÂ </td><td>683 (0.5)Â </td><td>3â€“7 (3.1)Â </td><td>10 675 (0.5)Â </td><td>1164 (0.5)Â </td><td>3â€“14 (3.1)Â </td><td>18 260 (0.5)Â </td><td>2351 (0.6)Â </td><td>3â€“7 (3.1)Â </td><td>36 820 (0.6)Â </td></tr><tr><td>AAATTÂ </td><td>568 (0.4)Â </td><td>3â€“6 (3.1)Â </td><td>8790 (0.4)Â </td><td>784 (0.3)Â </td><td>3â€“5 (3.1)Â </td><td>12 030 (0.3)Â </td><td>1488 (0.4)Â </td><td>3â€“7 (3.1)Â </td><td>23 065 (0.4)Â </td></tr><tr><td>AAACCÂ </td><td>490 (0.3)Â </td><td>3â€“7 (3.2)Â </td><td>7850 (0.3)Â </td><td>713 (0.3)Â </td><td>3â€“6 (3.2)Â </td><td>11 325 (0.3)Â </td><td>1273 (0.3)Â </td><td>3â€“9 (3.2)Â </td><td>20 290 (0.3)Â </td></tr><tr><td>AATATÂ </td><td>394 (0.3)Â </td><td>3â€“6 (3.1)Â </td><td>6085 (0.3)Â </td><td>573 (0.2)Â </td><td>3â€“6 (3.1)Â </td><td>8890 (0.3)Â </td><td>1083 (0.3)Â </td><td>3â€“13 (3.1)Â </td><td>16 905 (0.3)Â </td></tr><tr><td>OthersÂ </td><td>3085 (2.2)Â </td><td>3â€“114 (3.2)Â </td><td>49 775 (2.2)Â </td><td>4372 (1.9)Â </td><td>3â€“16 (3.2)Â </td><td>69 320 (2.0)Â </td><td>9281 (2.2)Â </td><td>3â€“28 (3.2)Â </td><td>147 240 (2.3)Â </td></tr><tr><td>HexaÂ </td><td>4179 (3.0)Â </td><td>3â€“102 (3.4)Â </td><td>85 176 (3.7)Â </td><td>7498 (3.3)Â </td><td>3â€“33 (3.3)Â </td><td>146 370 (4.2)Â </td><td>13 273 (3.2)Â </td><td>3â€“1579 (3.7)Â </td><td>295 236 (4.6)Â </td></tr><tr><td>AAAAATÂ </td><td>628 (0.4)Â </td><td>3â€“52 (3.2)Â </td><td>12 000 (0.5)Â </td><td>987 (0.4)Â </td><td>3â€“22 (3.1)Â </td><td>18 618 (0.5)Â </td><td>1803 (0.4)Â </td><td>3â€“35 (3.1)Â </td><td>33 972 (0.5)Â </td></tr><tr><td>AAAATTÂ </td><td>267 (0.2)Â </td><td>3â€“6 (3.0)Â </td><td>4866 (0.2)Â </td><td>776 (0.3)Â </td><td>3â€“25 (3.1)Â </td><td>14 376 (0.4)Â </td><td>1341 (0.3)Â </td><td>3â€“154 (3.2)Â </td><td>25 614 (0.4)Â </td></tr><tr><td>AAAAACÂ </td><td>269 (0.2)Â </td><td>3â€“23 (3.3)Â </td><td>5298 (0.2)Â </td><td>395 (0.2)Â </td><td>3â€“7 (3.2)Â </td><td>7596 (0.2)Â </td><td>746 (0.2)Â </td><td>3â€“22 (3.3)Â </td><td>14 562 (0.2)Â </td></tr><tr><td>AAAAAGÂ </td><td>175 (0.1)Â </td><td>3â€“6 (3.2)Â </td><td>3330 (0.1)Â </td><td>357 (0.2)Â </td><td>3â€“14 (3.3)Â </td><td>6966 (0.2)Â </td><td>644 (0.2)Â </td><td>3â€“37 (3.3)Â </td><td>12 612 (0.2)Â </td></tr><tr><td>AAATATÂ </td><td>150 (0.1)Â </td><td>3â€“52 (3.8)Â </td><td>3390 (0.1)Â </td><td>246 (0.1)Â </td><td>3â€“16 (3.2)Â </td><td>4704 (0.1)Â </td><td>436 (0.1)Â </td><td>3â€“29 (3.3)Â </td><td>8562 (0.1)Â </td></tr><tr><td>OthersÂ </td><td>2690 (1.9)Â </td><td>3â€“102 (3.5)Â </td><td>56 292 (2.5)Â </td><td>4737 (2.1)Â </td><td>3â€“33 (3.3)Â </td><td>94 110 (2.7)Â </td><td>8303 (2.0)Â </td><td>3â€“1579 (4.0)Â </td><td>199 914 (3.1)Â </td></tr><tr><td>TotalÂ </td><td>140 998 (100)Â </td><td>3â€“3644 (8.0)Â </td><td>228 8271 (100)Â </td><td>229 389 (100)Â </td><td>3â€“1710 (7.7)Â </td><td>347 3126 (100)Â </td><td>420 991 (100)Â </td><td>3â€“5556 (7.7)Â </td><td>6 443 660 (100)Â </td></tr></tbody></table></div><div content-id="DST040TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">MotifÂ </th><th colspan="3"><em>B. rapa</em><hr />Â </th><th colspan="3"><em>B. oleracea</em><hr />Â </th><th colspan="3"><em>B. napus</em><hr />Â </th></tr><tr><th>Number (%)Â </th><th>Repeat numberÂ </th><th>Total length (%)Â </th><th>Number (%)Â </th><th>Repeat numberÂ </th><th>Total length (%)Â </th><th>Number (%)Â </th><th>Repeat numberÂ </th><th>Total length (%)Â </th></tr></thead><tbody><tr><td>MonoÂ </td><td>31 258 (22.2)Â </td><td>12â€“307 (14.7)Â </td><td>458 968 (20.1)Â </td><td>55 433 (24.2)Â </td><td>3â€“65 (15.1)Â </td><td>838 104 (24.1)Â </td><td>97 128 (23.1)Â </td><td>12â€“2545 (15.2)Â </td><td>147 5939 (22.9)Â </td></tr><tr><td>AÂ </td><td>29 536 (20.9)Â </td><td>12â€“50 (14.5)Â </td><td>428 733 (18.7)Â </td><td>52 021 (22.7)Â </td><td>3â€“65 (15.0)Â </td><td>780 171 (22.5)Â </td><td>94 281 (22.4)Â </td><td>12â€“2545 (15.2)Â </td><td>1 432 867 (22.2)Â </td></tr><tr><td>CÂ </td><td>1722 (1.2)Â </td><td>12â€“307 (17.6)Â </td><td>30 235 (1.3)Â </td><td>3412 (1.5)Â </td><td>3â€“63 (17.0)Â </td><td>57 933 (1.7)Â </td><td>2847 (0.7)Â </td><td>12â€“83 (15.1)Â </td><td>43 072 (0.7)Â </td></tr><tr><td>DiÂ </td><td>33 885 (24.0)Â </td><td>6â€“3644 (11.1)Â </td><td>751 910 (32.9)Â </td><td>55 336 (24.1)Â </td><td>6â€“82 (8.8)Â </td><td>968 946 (27.9)Â </td><td>98 816 (23.5)Â </td><td>6â€“5556 (9.1)Â </td><td>1 789 752 (27.8)Â </td></tr><tr><td>ATÂ </td><td>19 697 (14.0)Â </td><td>6â€“419 (8.9)Â </td><td>350 590 (15.3)Â </td><td>33 315 (14.5)Â </td><td>6â€“63 (8.9)Â </td><td>596 070 (17.2)Â </td><td>57 070 (13.6)Â </td><td>6â€“5556 (8.7)Â </td><td>996 518 (15.5)Â </td></tr><tr><td>AGÂ </td><td>11 683 (8.3)Â </td><td>6â€“3644 (15.6)Â </td><td>364 004 (15.9)Â </td><td>18 593 (8.1)Â </td><td>6â€“44 (8.7)Â </td><td>322 438 (9.3)Â </td><td>34 638 (8.2)Â </td><td>6â€“1732 (9.9)Â </td><td>688 124 (10.7)Â </td></tr><tr><td>ACÂ </td><td>2490 (1.8)Â </td><td>6â€“131 (7.5)Â </td><td>37 124 (1.6)Â </td><td>3411 (1.5)Â </td><td>6â€“82 (7.4)Â </td><td>50 220 (1.4)Â </td><td>7072 (1.7)Â </td><td>6â€“92 (7.4)Â </td><td>104 648 (1.6)Â </td></tr><tr><td>CGÂ </td><td>15 (0.0)Â </td><td>6â€“8 (6.4)Â </td><td>192 (0.0)Â </td><td>17 (0.0)Â </td><td>6â€“9 (6.4)Â </td><td>218 (0.0)Â </td><td>36 (0.0)Â </td><td>6â€“8 (6.4)Â </td><td>462 (0.0)Â </td></tr><tr><td>TriÂ </td><td>32 387 (23.0)Â </td><td>4â€“812 (4.7)Â </td><td>459 039 (20.1)Â </td><td>47 716 (20.8)Â </td><td>4â€“1710 (4.7)Â </td><td>670 164 (19.3)Â </td><td>91 448 (21.7)Â </td><td>4â€“1794 (4.7)Â </td><td>1 277 172 (19.8)Â </td></tr><tr><td>AAGÂ </td><td>9796 (6.9)Â </td><td>4â€“812 (4.7)Â </td><td>139 239 (6.1)Â </td><td>15 322 (6.7)Â </td><td>4â€“81 (4.6)Â </td><td>212 973 (6.1)Â </td><td>29 395 (7.0)Â </td><td>4â€“162 (4.6)Â </td><td>408 264 (6.3)Â </td></tr><tr><td>AATÂ </td><td>6334 (4.5)Â </td><td>4â€“573 (4.9)Â </td><td>93 240 (4.1)Â </td><td>9355 (4.1)Â </td><td>4â€“1710 (5.2)Â </td><td>145 056 (4.2)Â </td><td>17 722 (4.2)Â </td><td>4â€“1794 (5.1)Â </td><td>268 956 (4.2)Â </td></tr><tr><td>ATCÂ </td><td>4211 (3.0)Â </td><td>4â€“190 (5.0)Â </td><td>63 213 (2.8)Â </td><td>6093 (2.7)Â </td><td>4â€“81 (4.7)Â </td><td>85 047 (2.4)Â </td><td>12 314 (2.9)Â </td><td>4â€“119 (4.6)Â </td><td>170 151 (2.6)Â </td></tr><tr><td>AACÂ </td><td>3637 (2.6)Â </td><td>4â€“11 (4.5)Â </td><td>49 146 (2.1)Â </td><td>5036 (2.2)Â </td><td>4â€“41 (4.4)Â </td><td>66 705 (1.9)Â </td><td>10 046 (2.4)Â </td><td>4â€“22 (4.5)Â </td><td>134 592 (2.1)Â </td></tr><tr><td>AGGÂ </td><td>3243 (2.3)Â </td><td>4â€“179 (4.6)Â </td><td>45 084 (2.0)Â </td><td>5425 (2.4)Â </td><td>4â€“13 (4.6)Â </td><td>74 763 (2.2)Â </td><td>8349 (2.0)Â </td><td>4â€“14 (4.6)Â </td><td>115 494 (1.8)Â </td></tr><tr><td>ACCÂ </td><td>2144 (1.5)Â </td><td>4â€“9 (4.5)Â </td><td>28 908 (1.3)Â </td><td>2788 (1.2)Â </td><td>4â€“43 (4.4)Â </td><td>37 155 (1.1)Â </td><td>5969 (1.4)Â </td><td>4â€“14 (4.4)Â </td><td>79 137 (1.2)Â </td></tr><tr><td>AGCÂ </td><td>1127 (0.8)Â </td><td>4â€“11 (4.4)Â </td><td>15 039 (0.7)Â </td><td>1390 (0.6)Â </td><td>4â€“22 (4.4)Â </td><td>18 507 (0.5)Â </td><td>2878 (0.7)Â </td><td>4â€“18 (4.4)Â </td><td>38 379 (0.6)Â </td></tr><tr><td>ACTÂ </td><td>674 (0.5)Â </td><td>4â€“10 (4.4)Â </td><td>8874 (0.4)Â </td><td>818 (0.4)Â </td><td>4â€“28 (4.5)Â </td><td>10 968 (0.3)Â </td><td>1754 (0.4)Â </td><td>4â€“17 (4.4)Â </td><td>23 265 (0.4)Â </td></tr><tr><td>CCGÂ </td><td>626 (0.4)Â </td><td>4â€“8 (4.4)Â </td><td>8220 (0.4)Â </td><td>799 (0.3)Â </td><td>4â€“7 (4.3)Â </td><td>10 203 (0.3)Â </td><td>1566 (0.4)Â </td><td>4â€“8 (4.3)Â </td><td>20 139 (0.3)Â </td></tr><tr><td>ACGÂ </td><td>595 (0.4)Â </td><td>4â€“9 (4.5)Â </td><td>8076 (0.4)Â </td><td>690 (0.3)Â </td><td>4â€“13 (4.2)Â </td><td>8787 (0.3)Â </td><td>1455 (0.3)Â </td><td>4â€“10 (4.3)Â </td><td>18 795 (0.3)Â </td></tr><tr><td>TetraÂ </td><td>29 433 (20.9)Â </td><td>3â€“264 (3.2)Â </td><td>376 668 (16.5)Â </td><td>48 394 (21.1)Â </td><td>3â€“54 (3.1)Â </td><td>608 452 (17.5)Â </td><td>91 268 (21.7)Â </td><td>3â€“631 (3.1)Â </td><td>1 148 496 (17.8)Â </td></tr><tr><td>AAATÂ </td><td>11 870 (8.4)Â </td><td>3â€“8 (3.2)Â </td><td>152 776 (6.7)Â </td><td>18 856 (8.2)Â </td><td>3â€“28 (3.1)Â </td><td>236 560 (6.8)Â </td><td>33 903 (8.1)Â </td><td>3â€“10 (3.1)Â </td><td>425 080 (6.6)Â </td></tr><tr><td>AAAGÂ </td><td>3496 (2.5)Â </td><td>3â€“60 (3.2)Â </td><td>44 608 (1.9)Â </td><td>5697 (2.5)Â </td><td>3â€“36 (3.2)Â </td><td>72 224 (2.1)Â </td><td>10 795 (2.6)Â </td><td>3â€“13 (3.2)Â </td><td>136 204 (2.1)Â </td></tr><tr><td>AAACÂ </td><td>3333 (2.4)Â </td><td>3â€“8 (3.1)Â </td><td>41 960 (1.8)Â </td><td>4723 (2.1)Â </td><td>3â€“26 (3.1)Â </td><td>59 076 (1.7)Â </td><td>9717 (2.3)Â </td><td>3â€“8 (3.1)Â </td><td>121 768 (1.9)Â </td></tr><tr><td>AATTÂ </td><td>2534 (1.8)Â </td><td>3â€“6 (3.1)Â </td><td>31 108 (1.4)Â </td><td>4088 (1.8)Â </td><td>3â€“10 (3.1)Â </td><td>51 416 (1.5)Â </td><td>7863 (1.9)Â </td><td>3â€“41 (3.1)Â </td><td>98 224 (1.5)Â </td></tr><tr><td>AATCÂ </td><td>1137 (0.8)Â </td><td>3â€“8 (3.2)Â </td><td>14 368 (0.6)Â </td><td>2956 (1.3)Â </td><td>3â€“27 (3.1)Â </td><td>36 908 (1.1)Â </td><td>5112 (1.2)Â </td><td>3â€“8 (3.1)Â </td><td>63 884 (1.0)Â </td></tr><tr><td>othersÂ </td><td>7063 (5.0)Â </td><td>3â€“264 (3.3)Â </td><td>91 848 (4.0)Â </td><td>12 074 (5.3)Â </td><td>3â€“54 (3.2)Â </td><td>152 268 (4.4)Â </td><td>23 878 (5.7)Â </td><td>3â€“631 (3.2)Â </td><td>303 336 (4.7)Â </td></tr><tr><td>PentaÂ </td><td>9856 (7.0)Â </td><td>3â€“114 (3.2)Â </td><td>156 510 (6.8)Â </td><td>15 012 (6.5)Â </td><td>3â€“40 (3.2)Â </td><td>241 090 (6.9)Â </td><td>29 058 (6.9)Â </td><td>3â€“28 (3.1)Â </td><td>457 065 (7.1)Â </td></tr><tr><td>AAAATÂ </td><td>2758 (2.0)Â </td><td>3â€“63 (3.1)Â </td><td>42 920 (1.9)Â </td><td>4051 (1.8)Â </td><td>3â€“18 (3.4)Â </td><td>67 905 (2.0)Â </td><td>7617 (1.8)Â </td><td>3â€“9 (3.1)Â </td><td>118 030 (1.8)Â </td></tr><tr><td>AACCGÂ </td><td>1000 (0.7)Â </td><td>3â€“68 (3.3)Â </td><td>16 510 (0.7)Â </td><td>2169 (0.9)Â </td><td>3â€“40 (3.2)Â </td><td>34 645 (1.0)Â </td><td>3541 (0.8)Â </td><td>3â€“7 (3.2)Â </td><td>56 435 (0.9)Â </td></tr><tr><td>AAAACÂ </td><td>878 (0.6)Â </td><td>3â€“7 (3.2)Â </td><td>13 905 (0.6)Â </td><td>1186 (0.5)Â </td><td>3â€“8 (3.2)Â </td><td>18 715 (0.5)Â </td><td>2424 (0.6)Â </td><td>3â€“10 (3.2)Â </td><td>38 280 (0.6)Â </td></tr><tr><td>AAAAGÂ </td><td>683 (0.5)Â </td><td>3â€“7 (3.1)Â </td><td>10 675 (0.5)Â </td><td>1164 (0.5)Â </td><td>3â€“14 (3.1)Â </td><td>18 260 (0.5)Â </td><td>2351 (0.6)Â </td><td>3â€“7 (3.1)Â </td><td>36 820 (0.6)Â </td></tr><tr><td>AAATTÂ </td><td>568 (0.4)Â </td><td>3â€“6 (3.1)Â </td><td>8790 (0.4)Â </td><td>784 (0.3)Â </td><td>3â€“5 (3.1)Â </td><td>12 030 (0.3)Â </td><td>1488 (0.4)Â </td><td>3â€“7 (3.1)Â </td><td>23 065 (0.4)Â </td></tr><tr><td>AAACCÂ </td><td>490 (0.3)Â </td><td>3â€“7 (3.2)Â </td><td>7850 (0.3)Â </td><td>713 (0.3)Â </td><td>3â€“6 (3.2)Â </td><td>11 325 (0.3)Â </td><td>1273 (0.3)Â </td><td>3â€“9 (3.2)Â </td><td>20 290 (0.3)Â </td></tr><tr><td>AATATÂ </td><td>394 (0.3)Â </td><td>3â€“6 (3.1)Â </td><td>6085 (0.3)Â </td><td>573 (0.2)Â </td><td>3â€“6 (3.1)Â </td><td>8890 (0.3)Â </td><td>1083 (0.3)Â </td><td>3â€“13 (3.1)Â </td><td>16 905 (0.3)Â </td></tr><tr><td>OthersÂ </td><td>3085 (2.2)Â </td><td>3â€“114 (3.2)Â </td><td>49 775 (2.2)Â </td><td>4372 (1.9)Â </td><td>3â€“16 (3.2)Â </td><td>69 320 (2.0)Â </td><td>9281 (2.2)Â </td><td>3â€“28 (3.2)Â </td><td>147 240 (2.3)Â </td></tr><tr><td>HexaÂ </td><td>4179 (3.0)Â </td><td>3â€“102 (3.4)Â </td><td>85 176 (3.7)Â </td><td>7498 (3.3)Â </td><td>3â€“33 (3.3)Â </td><td>146 370 (4.2)Â </td><td>13 273 (3.2)Â </td><td>3â€“1579 (3.7)Â </td><td>295 236 (4.6)Â </td></tr><tr><td>AAAAATÂ </td><td>628 (0.4)Â </td><td>3â€“52 (3.2)Â </td><td>12 000 (0.5)Â </td><td>987 (0.4)Â </td><td>3â€“22 (3.1)Â </td><td>18 618 (0.5)Â </td><td>1803 (0.4)Â </td><td>3â€“35 (3.1)Â </td><td>33 972 (0.5)Â </td></tr><tr><td>AAAATTÂ </td><td>267 (0.2)Â </td><td>3â€“6 (3.0)Â </td><td>4866 (0.2)Â </td><td>776 (0.3)Â </td><td>3â€“25 (3.1)Â </td><td>14 376 (0.4)Â </td><td>1341 (0.3)Â </td><td>3â€“154 (3.2)Â </td><td>25 614 (0.4)Â </td></tr><tr><td>AAAAACÂ </td><td>269 (0.2)Â </td><td>3â€“23 (3.3)Â </td><td>5298 (0.2)Â </td><td>395 (0.2)Â </td><td>3â€“7 (3.2)Â </td><td>7596 (0.2)Â </td><td>746 (0.2)Â </td><td>3â€“22 (3.3)Â </td><td>14 562 (0.2)Â </td></tr><tr><td>AAAAAGÂ </td><td>175 (0.1)Â </td><td>3â€“6 (3.2)Â </td><td>3330 (0.1)Â </td><td>357 (0.2)Â </td><td>3â€“14 (3.3)Â </td><td>6966 (0.2)Â </td><td>644 (0.2)Â </td><td>3â€“37 (3.3)Â </td><td>12 612 (0.2)Â </td></tr><tr><td>AAATATÂ </td><td>150 (0.1)Â </td><td>3â€“52 (3.8)Â </td><td>3390 (0.1)Â </td><td>246 (0.1)Â </td><td>3â€“16 (3.2)Â </td><td>4704 (0.1)Â </td><td>436 (0.1)Â </td><td>3â€“29 (3.3)Â </td><td>8562 (0.1)Â </td></tr><tr><td>OthersÂ </td><td>2690 (1.9)Â </td><td>3â€“102 (3.5)Â </td><td>56 292 (2.5)Â </td><td>4737 (2.1)Â </td><td>3â€“33 (3.3)Â </td><td>94 110 (2.7)Â </td><td>8303 (2.0)Â </td><td>3â€“1579 (4.0)Â </td><td>199 914 (3.1)Â </td></tr><tr><td>TotalÂ </td><td>140 998 (100)Â </td><td>3â€“3644 (8.0)Â </td><td>228 8271 (100)Â </td><td>229 389 (100)Â </td><td>3â€“1710 (7.7)Â </td><td>347 3126 (100)Â </td><td>420 991 (100)Â </td><td>3â€“5556 (7.7)Â </td><td>6 443 660 (100)Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4916812" target="_blank">View Large</a></div></div><p>In accordance with their high correlation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), the distributions with respect to the motif length of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em> were almost identical: mono-, di-, tri- and tetranucleotide repeats accounted for very similar and relatively high proportions, whereas penta- and hexanucleotide repeats were relatively uncommon (Fig.Â <a reveal-id="DST040F1" data-open="DST040F1" class="link link-reveal link-table xref-fig">1</a>A). </p>                        <a id="4916814" scrollto-destination="4916814"></a>
<div id="DST040F1" data-id="DST040F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04001.png?Expires=1509238921&Signature=baQ8bI26gj~maQ7gPdz~FweLxaNIi355sX-kx4Op7b6sfjQmYAE0y92h3CrvuGHkUcypTnPhTF9~YKLNgC4P-tTncqYOd5t2~f5bi1YJYDiNii-Z6FZmvaMbaMv2VLaIo~Yf1v0tey0dEGQZHcioqq~luKqsAF36deX~ToOFK9G5z13Br7kSdfHMCCDMpUvdHW5V9wWrXGlFt6VowFGpiow~itZwgq2W4-82vFW0X1vlzTOpyA5XH0FUdYgMdpOz19qLtveOynAk5K2~J7ErrCYUpreoN9HJ~LDZV45aqzwXDecSCHwSc5JGmg0YY0GF7EXsmD6z1QBhqoCN3ZQUMg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution with respect to the motif length (A), type (B) and repeat number (C) of microsatellites in the assembled genomic sequences of B. rapa, B. oleracea and B. napus. The vertical axis shows the abundances (%) of microsatellites with different motif lengths, types or repeat numbers that are discriminated by the legends of different colours. For FigureÂ 1B, because there is a limitation in the number of items (&lt;256) in Excel, the abundances of the mono- to pentanucleotide motifs (a total of 151 types) are shown, while those for the hexanucleotide motifs (a total of 350 types) are displayed in Supplementary Table S2." path-from-xml="dst04001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916814/dst04001.tiff" path-from-xml="dst04001.tiff" target="_blank">View large</a><a section="4916814" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04001.png?Expires=1509238921&Signature=CsKptMccreN1FbXcZ0sVC8aVMYaGSeCPbNtWRR5nDxo1m6W2lHUx5YdNd30oVvTZXCT2ISQd8r6EvhrOgI0Zv986rLtGyqN7GSEvL7RLC2sfOg~WynvfTxikGDD39Lh5Fkq9ArSk~RV7Kp-Wp6QGW1XaStvhZbQ0s1VIxB7Ygzj3ausXxnA04qFx-2wjvRegHBouo4P0fhDhgmX~iWivZZ-L~fux1xZdsFpozo4eid2meehTuPxeU3faeImOC7-i8T42ELtXOgplM1O7HRRz12Zx4pHPTSTcOxnqQDZFlHfFcl-V21lMKPfwPar7OkIOc0lsW4uzuYUSyaU-HoVxNQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916814&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution with respect to the motif length (A), type (B) and repeat number (C) of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>. The vertical axis shows the abundances (%) of microsatellites with different motif lengths, types or repeat numbers that are discriminated by the legends of different colours. For FigureÂ <a reveal-id="DST040F1" data-open="DST040F1" class="link link-reveal link-table xref-fig">1</a>B, because there is a limitation in the number of items (&lt;256) in Excel, the abundances of the mono- to pentanucleotide motifs (a total of 151 types) are shown, while those for the hexanucleotide motifs (a total of 350 types) are displayed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p></div></div><div content-id="DST040F1" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04001.png?Expires=1509238921&Signature=baQ8bI26gj~maQ7gPdz~FweLxaNIi355sX-kx4Op7b6sfjQmYAE0y92h3CrvuGHkUcypTnPhTF9~YKLNgC4P-tTncqYOd5t2~f5bi1YJYDiNii-Z6FZmvaMbaMv2VLaIo~Yf1v0tey0dEGQZHcioqq~luKqsAF36deX~ToOFK9G5z13Br7kSdfHMCCDMpUvdHW5V9wWrXGlFt6VowFGpiow~itZwgq2W4-82vFW0X1vlzTOpyA5XH0FUdYgMdpOz19qLtveOynAk5K2~J7ErrCYUpreoN9HJ~LDZV45aqzwXDecSCHwSc5JGmg0YY0GF7EXsmD6z1QBhqoCN3ZQUMg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution with respect to the motif length (A), type (B) and repeat number (C) of microsatellites in the assembled genomic sequences of B. rapa, B. oleracea and B. napus. The vertical axis shows the abundances (%) of microsatellites with different motif lengths, types or repeat numbers that are discriminated by the legends of different colours. For FigureÂ 1B, because there is a limitation in the number of items (&lt;256) in Excel, the abundances of the mono- to pentanucleotide motifs (a total of 151 types) are shown, while those for the hexanucleotide motifs (a total of 350 types) are displayed in Supplementary Table S2." path-from-xml="dst04001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916814/dst04001.tiff" path-from-xml="dst04001.tiff" target="_blank">View large</a><a section="4916814" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04001.png?Expires=1509238921&Signature=CsKptMccreN1FbXcZ0sVC8aVMYaGSeCPbNtWRR5nDxo1m6W2lHUx5YdNd30oVvTZXCT2ISQd8r6EvhrOgI0Zv986rLtGyqN7GSEvL7RLC2sfOg~WynvfTxikGDD39Lh5Fkq9ArSk~RV7Kp-Wp6QGW1XaStvhZbQ0s1VIxB7Ygzj3ausXxnA04qFx-2wjvRegHBouo4P0fhDhgmX~iWivZZ-L~fux1xZdsFpozo4eid2meehTuPxeU3faeImOC7-i8T42ELtXOgplM1O7HRRz12Zx4pHPTSTcOxnqQDZFlHfFcl-V21lMKPfwPar7OkIOc0lsW4uzuYUSyaU-HoVxNQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916814&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution with respect to the motif length (A), type (B) and repeat number (C) of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>. The vertical axis shows the abundances (%) of microsatellites with different motif lengths, types or repeat numbers that are discriminated by the legends of different colours. For FigureÂ <a reveal-id="DST040F1" data-open="DST040F1" class="link link-reveal link-table xref-fig">1</a>B, because there is a limitation in the number of items (&lt;256) in Excel, the abundances of the mono- to pentanucleotide motifs (a total of 151 types) are shown, while those for the hexanucleotide motifs (a total of 350 types) are displayed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p></div></div><p>In accordance with their high correlation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), the distributions with respect to the motif type of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em> were almost identical (Fig.Â <a reveal-id="DST040F1" data-open="DST040F1" class="link link-reveal link-table xref-fig">1</a>B; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). More specifically, both the dominant/major and absent/scarce mono- to hexanucleotide motifs in the assembled genomic sequences of the three <em>Brassica</em> species were mostly identical (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Interestingly, the dominant/major motifs (A, AT, AAG/AAT, AAAT, AAAAT and AAAAAT) were all A/T rich (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>), whereas the absent/scarce motifs were mostly C/G rich (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>), which were highly consistent with the previous reports on microsatellites identified from 536 seed BACs of <em>B. rapa</em>,<sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> 3500 genomic clones<sup><a reveal-id="DST040C42" data-open="DST040C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and 595 577 WGSs<sup><a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> of <em>B. oleracea</em> and 13 794 GSSs (mainly BESs) of <em>B. napus.</em><sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a></sup> It should be noted that the nucleotide composition characteristics of both the dominant/major and absent/scarce motifs in the assembled genomic sequences of the three <em>Brassica</em> species corresponded well to their much higher A/T (mean = 63.8%) than C/G (mean = 36.2%) content.</p><p>In accordance with their high correlation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), the distributions with respect to the motif repeat number of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em> were also almost identical (Fig.Â <a reveal-id="DST040F1" data-open="DST040F1" class="link link-reveal link-table xref-fig">1</a>C). Obviously, the microsatellite abundances decreased significantly as the motif repeat number increased, and the rate of this change was the slowest for dinucleotide repeat, followed by mono- and trinucleotide repeats, and was faster for other long repeats (Fig.Â <a reveal-id="DST040F2" data-open="DST040F2" class="link link-reveal link-table xref-fig">2</a>). As a consequence, the difference between the average and minimum motif repeat numbers was the largest for dinucleotide repeat, followed by mono- and trinucleotide repeats, and was relatively small for tetra- to hexanucleotide repeats (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4916817" scrollto-destination="4916817"></a>
<div id="DST040F2" data-id="DST040F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04002.png?Expires=1509238921&Signature=POeyOHQ7m4WuVoylHIpEtrLVPF3rLXjqaYrueEKCsxI-4s~OtZ2WNhvmPE7s5aKFzKAq-o5aej4ERR55SN9TTglR-cvGDnRUjabfrUoITKIoXVtCTj~MTQ5Me4dcPCQG2~BMkEGjP34goXEZLuzRQbZFz5ejdA5Bldub5P2bUuJY~jMGGemB0E9zoOi40BmMMifOO2rOh5Y8ocWQJHpPyVT32MI~IKeLuBtXDBEHfU2K42qNpQZcNIppdkEmNZGc~oa4dJmnhB6ZLCMpt218oPz7Z5ZsStmeXT9qTHYoxClX0uOjvauKSDI8cRbFk9e5CASCd6Pszfsnqojw-Ln0Dw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution with respect to the motif repeat number of the individual mono- to hexanucleotide repeat microsatellites in the assembled genomic sequences of B. rapa, B. oleracea and B. napus. The vertical axis shows the abundances of microsatellites that have different motif repeat numbers (from 3 to &gt;20), which are discriminated by legends of different colours." path-from-xml="dst04002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916817/dst04002.tiff" path-from-xml="dst04002.tiff" target="_blank">View large</a><a section="4916817" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04002.png?Expires=1509238921&Signature=UyDN-ThuHSst9dwCgotC9uTHzd-yMnP-OOon5xMNjU28GTgqnGiUKacSRfhW7z1d-0lqaykKCuCI04ZlhNPJqf-2ga7plVyMn1X6i91kGZ-y4gSAdcgv2ozCXqYdfbElxIO5-W5PheVyopEvzbXlnMiGgFuTS-YctwoGbqozLVg~hcsHx5xYsPJv8lhtJgaCJ2zz6~ShUAnhcg2jCCPkTI-GbusAIGUDVeb86sSQoSnS4t6bMCyF-JiNbQoRChcDD73eVvNpwNex6of3PSvf0NsV4M9pkjX3ZhclIvLNfB~~pFdpvt-qrj4dn6t~nuM5A9qTLeDJjHcqTC-~7Kg80A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916817&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution with respect to the motif repeat number of the individual mono- to hexanucleotide repeat microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>. The vertical axis shows the abundances of microsatellites that have different motif repeat numbers (from 3 to &gt;20), which are discriminated by legends of different colours.</p></div></div><div content-id="DST040F2" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04002.png?Expires=1509238921&Signature=POeyOHQ7m4WuVoylHIpEtrLVPF3rLXjqaYrueEKCsxI-4s~OtZ2WNhvmPE7s5aKFzKAq-o5aej4ERR55SN9TTglR-cvGDnRUjabfrUoITKIoXVtCTj~MTQ5Me4dcPCQG2~BMkEGjP34goXEZLuzRQbZFz5ejdA5Bldub5P2bUuJY~jMGGemB0E9zoOi40BmMMifOO2rOh5Y8ocWQJHpPyVT32MI~IKeLuBtXDBEHfU2K42qNpQZcNIppdkEmNZGc~oa4dJmnhB6ZLCMpt218oPz7Z5ZsStmeXT9qTHYoxClX0uOjvauKSDI8cRbFk9e5CASCd6Pszfsnqojw-Ln0Dw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution with respect to the motif repeat number of the individual mono- to hexanucleotide repeat microsatellites in the assembled genomic sequences of B. rapa, B. oleracea and B. napus. The vertical axis shows the abundances of microsatellites that have different motif repeat numbers (from 3 to &gt;20), which are discriminated by legends of different colours." path-from-xml="dst04002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916817/dst04002.tiff" path-from-xml="dst04002.tiff" target="_blank">View large</a><a section="4916817" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04002.png?Expires=1509238921&Signature=UyDN-ThuHSst9dwCgotC9uTHzd-yMnP-OOon5xMNjU28GTgqnGiUKacSRfhW7z1d-0lqaykKCuCI04ZlhNPJqf-2ga7plVyMn1X6i91kGZ-y4gSAdcgv2ozCXqYdfbElxIO5-W5PheVyopEvzbXlnMiGgFuTS-YctwoGbqozLVg~hcsHx5xYsPJv8lhtJgaCJ2zz6~ShUAnhcg2jCCPkTI-GbusAIGUDVeb86sSQoSnS4t6bMCyF-JiNbQoRChcDD73eVvNpwNex6of3PSvf0NsV4M9pkjX3ZhclIvLNfB~~pFdpvt-qrj4dn6t~nuM5A9qTLeDJjHcqTC-~7Kg80A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916817&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution with respect to the motif repeat number of the individual mono- to hexanucleotide repeat microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>. The vertical axis shows the abundances of microsatellites that have different motif repeat numbers (from 3 to &gt;20), which are discriminated by legends of different colours.</p></div></div><p>In addition, the motif repeat number of the corresponding mono- to hexanucleotide repeats or motifs of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em> were highly similar (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). As a consequence, the total repeat length (=microsatellite number Ã— motif length Ã— motif repeat number) proportions of the corresponding mono- to hexanucleotide repeats or motifs of microsatellites in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em> were mostly similar (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p>                            <h3 scrollto-destination=4916819 id="4916819" class="section-title" data-legacy-id=s3b>3.2. Genomic distribution</h3>
<p>The genomic distributions of microsatellites and their relation with the annotated genome components (mainly as genes and TEs) were investigated (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>; TableÂ <a reveal-id="DST040TB2" data-open="DST040TB2" class="link link-reveal link-table xref-fig">2</a>), based on the assembled pseudochromosomes of the sequenced <em>Brassica</em> species (currently available for <em>B. rapa</em> and <em>B. oleracea</em>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). </p>                        <a id="4916821" scrollto-destination="4916821"></a>
<div class="table-wrap"><div id="DST040TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>Ï‡/italic&gt;<sup>2</sup> test between the practical and hypothetical/average distribution of microsatellites and their correlation with genes and TEs, for all pseudochromosomes of <em>B. rapa</em> and <em>B. oleracea</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SpeciesÂ </th><th rowspan="2">ChromosomeÂ </th><th colspan="4">Microsatellites<hr />Â </th><th colspan="2">Genes<hr />Â </th><th colspan="2">TEs<hr />Â </th></tr><tr><th>FrequencyÂ </th><th><span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>Â </th><th><em>r</em><sub>gene</sub>Â </th><th><em>r</em><sub>TE</sub>Â </th><th>FrequencyÂ </th><th><span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>Â </th><th>FrequencyÂ </th><th><span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>Â </th></tr></thead><tbody><tr><td>Â </td><td>A01Â </td><td>488Â </td><td>1.9Eâˆ’42Â </td><td>0.76Â </td><td>âˆ’0.69Â </td><td>146Â </td><td>8.7Eâˆ’60Â </td><td>841Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>A02Â </td><td>522Â </td><td>1.6Eâˆ’16Â </td><td>0.78Â </td><td>âˆ’0.65Â </td><td>150Â </td><td>8.3Eâˆ’41Â </td><td>869Â </td><td>4.0Eâˆ’220Â </td></tr><tr><td>Â </td><td>A03Â </td><td>539Â </td><td>1.2Eâˆ’13Â </td><td>0.46Â </td><td>âˆ’0.29Â </td><td>179Â </td><td>8.9Eâˆ’15Â </td><td>725Â </td><td>9.9Eâˆ’163Â </td></tr><tr><td>Â </td><td>A04Â </td><td>515Â </td><td>6.5Eâˆ’14Â </td><td>0.53Â </td><td>âˆ’0.49Â </td><td>149Â </td><td>1.2Eâˆ’17Â </td><td>887Â </td><td>1.9Eâˆ’134Â </td></tr><tr><td>Â </td><td>A05Â </td><td>500Â </td><td>9.8Eâˆ’37Â </td><td>0.77Â </td><td>âˆ’0.69Â </td><td>153Â </td><td>1.8Eâˆ’63Â </td><td>828Â </td><td>2.9Eâˆ’249Â </td></tr><tr><td><em>B. rapa</em>Â </td><td>A06Â </td><td>509Â </td><td>6.1Eâˆ’49Â </td><td>0.81Â </td><td>âˆ’0.58Â </td><td>153Â </td><td>7.6Eâˆ’45Â </td><td>828Â </td><td>7.6Eâˆ’122Â </td></tr><tr><td>Â </td><td>A07Â </td><td>524Â </td><td>5.2Eâˆ’32Â </td><td>0.79Â </td><td>âˆ’0.65Â </td><td>155Â </td><td>2.3Eâˆ’38Â </td><td>810Â </td><td>1.6Eâˆ’173Â </td></tr><tr><td>Â </td><td>A08Â </td><td>502Â </td><td>1.8Eâˆ’34Â </td><td>0.74Â </td><td>âˆ’0.66Â </td><td>150Â </td><td>1.8Eâˆ’56Â </td><td>867Â </td><td>5.0Eâˆ’206Â </td></tr><tr><td>Â </td><td>A09Â </td><td>505Â </td><td>4.3Eâˆ’58Â </td><td>0.78Â </td><td>âˆ’0.63Â </td><td>150Â </td><td>9.2Eâˆ’77Â </td><td>884Â </td><td>3.1Eâˆ’255Â </td></tr><tr><td>Â </td><td>A10Â </td><td>522Â </td><td>6.1Eâˆ’20Â </td><td>0.75Â </td><td>âˆ’0.57Â </td><td>166Â </td><td>6.5Eâˆ’36Â </td><td>767Â </td><td>3.0Eâˆ’210Â </td></tr><tr><td>Â </td><td>TotalÂ </td><td>512Â </td><td>2.4Eâˆ’287Â </td><td>0.75Â </td><td>âˆ’0.61Â </td><td>155Â </td><td>0.0E+00Â </td><td>831Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C01Â </td><td>442Â </td><td>1.5Eâˆ’81Â </td><td>0.90Â </td><td>âˆ’0.82Â </td><td>93Â </td><td>1.7Eâˆ’112Â </td><td>1570Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C02Â </td><td>429Â </td><td>8.2Eâˆ’108Â </td><td>0.81Â </td><td>âˆ’0.74Â </td><td>78Â </td><td>2.4Eâˆ’103Â </td><td>1673Â </td><td>6.2Eâˆ’248Â </td></tr><tr><td>Â </td><td>C03Â </td><td>459Â </td><td>1.4Eâˆ’190Â </td><td>0.85Â </td><td>âˆ’0.71Â </td><td>96Â </td><td>1.0Eâˆ’174Â </td><td>1606Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C04Â </td><td>446Â </td><td>6.2Eâˆ’59Â </td><td>0.86Â </td><td>âˆ’0.62Â </td><td>95Â </td><td>1.8Eâˆ’48Â </td><td>1590Â </td><td>2.2Eâˆ’258Â </td></tr><tr><td><em>B. oleracea</em>Â </td><td>C05Â </td><td>439Â </td><td>1.6Eâˆ’93Â </td><td>0.94Â </td><td>âˆ’0.84Â </td><td>97Â </td><td>4.2Eâˆ’100Â </td><td>1535Â </td><td>8.2Eâˆ’288Â </td></tr><tr><td>Â </td><td>C06Â </td><td>430Â </td><td>2.3Eâˆ’139Â </td><td>0.90Â </td><td>âˆ’0.79Â </td><td>89Â </td><td>4.0Eâˆ’150Â </td><td>1606Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C07Â </td><td>435Â </td><td>4.7Eâˆ’83Â </td><td>0.88Â </td><td>âˆ’0.74Â </td><td>86Â </td><td>6.6Eâˆ’92Â </td><td>1630Â </td><td>3.5Eâˆ’240Â </td></tr><tr><td>Â </td><td>C08Â </td><td>448Â </td><td>4.0Eâˆ’163Â </td><td>0.87Â </td><td>âˆ’0.72Â </td><td>99Â </td><td>2.1Eâˆ’173Â </td><td>1566Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C09Â </td><td>445Â </td><td>3.8Eâˆ’124Â </td><td>0.86Â </td><td>âˆ’0.73Â </td><td>96Â </td><td>3.3Eâˆ’127Â </td><td>1566Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>TotalÂ </td><td>442Â </td><td>0.0E+00Â </td><td>0.87Â </td><td>âˆ’0.73Â </td><td>92Â </td><td>0.0E+00Â </td><td>1596Â </td><td>0.0E+00Â </td></tr></tbody></table></div><div content-id="DST040TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SpeciesÂ </th><th rowspan="2">ChromosomeÂ </th><th colspan="4">Microsatellites<hr />Â </th><th colspan="2">Genes<hr />Â </th><th colspan="2">TEs<hr />Â </th></tr><tr><th>FrequencyÂ </th><th><span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>Â </th><th><em>r</em><sub>gene</sub>Â </th><th><em>r</em><sub>TE</sub>Â </th><th>FrequencyÂ </th><th><span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>Â </th><th>FrequencyÂ </th><th><span class="inline-formula no-formula-id">$$P_{\chi ^2 \,{\rm test}} $$</span>Â </th></tr></thead><tbody><tr><td>Â </td><td>A01Â </td><td>488Â </td><td>1.9Eâˆ’42Â </td><td>0.76Â </td><td>âˆ’0.69Â </td><td>146Â </td><td>8.7Eâˆ’60Â </td><td>841Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>A02Â </td><td>522Â </td><td>1.6Eâˆ’16Â </td><td>0.78Â </td><td>âˆ’0.65Â </td><td>150Â </td><td>8.3Eâˆ’41Â </td><td>869Â </td><td>4.0Eâˆ’220Â </td></tr><tr><td>Â </td><td>A03Â </td><td>539Â </td><td>1.2Eâˆ’13Â </td><td>0.46Â </td><td>âˆ’0.29Â </td><td>179Â </td><td>8.9Eâˆ’15Â </td><td>725Â </td><td>9.9Eâˆ’163Â </td></tr><tr><td>Â </td><td>A04Â </td><td>515Â </td><td>6.5Eâˆ’14Â </td><td>0.53Â </td><td>âˆ’0.49Â </td><td>149Â </td><td>1.2Eâˆ’17Â </td><td>887Â </td><td>1.9Eâˆ’134Â </td></tr><tr><td>Â </td><td>A05Â </td><td>500Â </td><td>9.8Eâˆ’37Â </td><td>0.77Â </td><td>âˆ’0.69Â </td><td>153Â </td><td>1.8Eâˆ’63Â </td><td>828Â </td><td>2.9Eâˆ’249Â </td></tr><tr><td><em>B. rapa</em>Â </td><td>A06Â </td><td>509Â </td><td>6.1Eâˆ’49Â </td><td>0.81Â </td><td>âˆ’0.58Â </td><td>153Â </td><td>7.6Eâˆ’45Â </td><td>828Â </td><td>7.6Eâˆ’122Â </td></tr><tr><td>Â </td><td>A07Â </td><td>524Â </td><td>5.2Eâˆ’32Â </td><td>0.79Â </td><td>âˆ’0.65Â </td><td>155Â </td><td>2.3Eâˆ’38Â </td><td>810Â </td><td>1.6Eâˆ’173Â </td></tr><tr><td>Â </td><td>A08Â </td><td>502Â </td><td>1.8Eâˆ’34Â </td><td>0.74Â </td><td>âˆ’0.66Â </td><td>150Â </td><td>1.8Eâˆ’56Â </td><td>867Â </td><td>5.0Eâˆ’206Â </td></tr><tr><td>Â </td><td>A09Â </td><td>505Â </td><td>4.3Eâˆ’58Â </td><td>0.78Â </td><td>âˆ’0.63Â </td><td>150Â </td><td>9.2Eâˆ’77Â </td><td>884Â </td><td>3.1Eâˆ’255Â </td></tr><tr><td>Â </td><td>A10Â </td><td>522Â </td><td>6.1Eâˆ’20Â </td><td>0.75Â </td><td>âˆ’0.57Â </td><td>166Â </td><td>6.5Eâˆ’36Â </td><td>767Â </td><td>3.0Eâˆ’210Â </td></tr><tr><td>Â </td><td>TotalÂ </td><td>512Â </td><td>2.4Eâˆ’287Â </td><td>0.75Â </td><td>âˆ’0.61Â </td><td>155Â </td><td>0.0E+00Â </td><td>831Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C01Â </td><td>442Â </td><td>1.5Eâˆ’81Â </td><td>0.90Â </td><td>âˆ’0.82Â </td><td>93Â </td><td>1.7Eâˆ’112Â </td><td>1570Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C02Â </td><td>429Â </td><td>8.2Eâˆ’108Â </td><td>0.81Â </td><td>âˆ’0.74Â </td><td>78Â </td><td>2.4Eâˆ’103Â </td><td>1673Â </td><td>6.2Eâˆ’248Â </td></tr><tr><td>Â </td><td>C03Â </td><td>459Â </td><td>1.4Eâˆ’190Â </td><td>0.85Â </td><td>âˆ’0.71Â </td><td>96Â </td><td>1.0Eâˆ’174Â </td><td>1606Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C04Â </td><td>446Â </td><td>6.2Eâˆ’59Â </td><td>0.86Â </td><td>âˆ’0.62Â </td><td>95Â </td><td>1.8Eâˆ’48Â </td><td>1590Â </td><td>2.2Eâˆ’258Â </td></tr><tr><td><em>B. oleracea</em>Â </td><td>C05Â </td><td>439Â </td><td>1.6Eâˆ’93Â </td><td>0.94Â </td><td>âˆ’0.84Â </td><td>97Â </td><td>4.2Eâˆ’100Â </td><td>1535Â </td><td>8.2Eâˆ’288Â </td></tr><tr><td>Â </td><td>C06Â </td><td>430Â </td><td>2.3Eâˆ’139Â </td><td>0.90Â </td><td>âˆ’0.79Â </td><td>89Â </td><td>4.0Eâˆ’150Â </td><td>1606Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C07Â </td><td>435Â </td><td>4.7Eâˆ’83Â </td><td>0.88Â </td><td>âˆ’0.74Â </td><td>86Â </td><td>6.6Eâˆ’92Â </td><td>1630Â </td><td>3.5Eâˆ’240Â </td></tr><tr><td>Â </td><td>C08Â </td><td>448Â </td><td>4.0Eâˆ’163Â </td><td>0.87Â </td><td>âˆ’0.72Â </td><td>99Â </td><td>2.1Eâˆ’173Â </td><td>1566Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>C09Â </td><td>445Â </td><td>3.8Eâˆ’124Â </td><td>0.86Â </td><td>âˆ’0.73Â </td><td>96Â </td><td>3.3Eâˆ’127Â </td><td>1566Â </td><td>0.0E+00Â </td></tr><tr><td>Â </td><td>TotalÂ </td><td>442Â </td><td>0.0E+00Â </td><td>0.87Â </td><td>âˆ’0.73Â </td><td>92Â </td><td>0.0E+00Â </td><td>1596Â </td><td>0.0E+00Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4916821" target="_blank">View Large</a></div></div>                        <a id="4916822" scrollto-destination="4916822"></a>
<div id="DST040F3" data-id="DST040F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04003.png?Expires=1509238921&Signature=X-Bav~BQ3Si-z0795eUYs7BK9rXk6u3kYDIaspGz-VLxubrdrzK--5YdqXfL~WovQZbYEX3~iN1-shtRUPgwanBYbXAZGzmhjJ6aQigMFZ4f2qbFcw3oT5w-BUmL8Vt9TtSvZGTWBXgJndcrOGodcKEieHtdaYKnbYwPWB80OYn3YjksUQADCfv9pj70OlOILyGqLjOd1BcMZrWypJ1~LWShA9RZSrfLuypV-2YvjP9InmLHaCbgl2N8~f7p~RVz2ziPArF-ZAJhEvE212pBMTjObLWlJCw8IBv91Fn87VICk07hg3sfhZzieYepwY8gkxhzO6gfv1VrQHijy1-jrg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide distributions of microsatellites as well as genes and TEs in the assembled pseudochromosomes of B. rapa (A) and B. oleracea (B). The horizontal axis shows the pseudochromosomes (B. rapa: A1â€“A10; B. oleracea: C1â€“C9), which are divided into 1-Mb intervals. The left and right vertical axes show the frequencies of the microsatellites/genes and TEs, respectively. On the figure, the curves/lines of different styles represent the practical/hypothetical(average) frequencies of microsatellites, genes and TEs, respectively." path-from-xml="dst04003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916822/dst04003.tiff" path-from-xml="dst04003.tiff" target="_blank">View large</a><a section="4916822" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04003.png?Expires=1509238921&Signature=PO1keXup9mrrP7VEH4q7CEXggP1HM6fRiPO9s6D33TMAx5VjcOpA7dg5zZNbrMWiPDspxnQ4a06vNcMTo0zLpnFae5ALgK43xBGhXyMVnYNdjBTiDcTCYJ97R9L2izs2T1jfuCCS4llsz0esDeJPQ55CHilsuSteSYoxJHRlpsITK3Zz54CONaGFRW1N9mtXTrF16KS1rjeUAo2snaC1iXcrsnGxZZTYV1trSpUR~v8wXleya7-rMId7vIyI02l7QiIois4fV85j7cHp8pf8h3jNaBEc8SkcsGTGnkSWt~dQ5pbRSrnJWcZWL1txDsBH~P2puA~uI9Kj7l3820Lc6A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916822&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide distributions of microsatellites as well as genes and TEs in the assembled pseudochromosomes of <em>B. rapa</em> (A) and <em>B. oleracea</em> (B). The horizontal axis shows the pseudochromosomes (<em>B. rapa</em>: A1â€“A10; <em>B. oleracea</em>: C1â€“C9), which are divided into 1-Mb intervals. The left and right vertical axes show the frequencies of the microsatellites/genes and TEs, respectively. On the figure, the curves/lines of different styles represent the practical/hypothetical(average) frequencies of microsatellites, genes and TEs, respectively.</p></div></div><div content-id="DST040F3" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04003.png?Expires=1509238921&Signature=X-Bav~BQ3Si-z0795eUYs7BK9rXk6u3kYDIaspGz-VLxubrdrzK--5YdqXfL~WovQZbYEX3~iN1-shtRUPgwanBYbXAZGzmhjJ6aQigMFZ4f2qbFcw3oT5w-BUmL8Vt9TtSvZGTWBXgJndcrOGodcKEieHtdaYKnbYwPWB80OYn3YjksUQADCfv9pj70OlOILyGqLjOd1BcMZrWypJ1~LWShA9RZSrfLuypV-2YvjP9InmLHaCbgl2N8~f7p~RVz2ziPArF-ZAJhEvE212pBMTjObLWlJCw8IBv91Fn87VICk07hg3sfhZzieYepwY8gkxhzO6gfv1VrQHijy1-jrg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genome-wide distributions of microsatellites as well as genes and TEs in the assembled pseudochromosomes of B. rapa (A) and B. oleracea (B). The horizontal axis shows the pseudochromosomes (B. rapa: A1â€“A10; B. oleracea: C1â€“C9), which are divided into 1-Mb intervals. The left and right vertical axes show the frequencies of the microsatellites/genes and TEs, respectively. On the figure, the curves/lines of different styles represent the practical/hypothetical(average) frequencies of microsatellites, genes and TEs, respectively." path-from-xml="dst04003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916822/dst04003.tiff" path-from-xml="dst04003.tiff" target="_blank">View large</a><a section="4916822" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04003.png?Expires=1509238921&Signature=PO1keXup9mrrP7VEH4q7CEXggP1HM6fRiPO9s6D33TMAx5VjcOpA7dg5zZNbrMWiPDspxnQ4a06vNcMTo0zLpnFae5ALgK43xBGhXyMVnYNdjBTiDcTCYJ97R9L2izs2T1jfuCCS4llsz0esDeJPQ55CHilsuSteSYoxJHRlpsITK3Zz54CONaGFRW1N9mtXTrF16KS1rjeUAo2snaC1iXcrsnGxZZTYV1trSpUR~v8wXleya7-rMId7vIyI02l7QiIois4fV85j7cHp8pf8h3jNaBEc8SkcsGTGnkSWt~dQ5pbRSrnJWcZWL1txDsBH~P2puA~uI9Kj7l3820Lc6A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916822&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide distributions of microsatellites as well as genes and TEs in the assembled pseudochromosomes of <em>B. rapa</em> (A) and <em>B. oleracea</em> (B). The horizontal axis shows the pseudochromosomes (<em>B. rapa</em>: A1â€“A10; <em>B. oleracea</em>: C1â€“C9), which are divided into 1-Mb intervals. The left and right vertical axes show the frequencies of the microsatellites/genes and TEs, respectively. On the figure, the curves/lines of different styles represent the practical/hypothetical(average) frequencies of microsatellites, genes and TEs, respectively.</p></div></div><p>For both <em>B. rapa</em> and <em>B. oleracea</em>, the frequency of microsatellites was high at/near both ends but low in/near the middle of all the pseudochromosomes (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>), which likely corresponded to the peri- telomere and centromere, respectively.<sup><a reveal-id="DST040C43" data-open="DST040C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The frequencies of microsatellites for the different pseudochromosomes of <em>B. rapa</em> or <em>B. oleracea</em> were generally comparable, which was in accordance with the similar frequencies of genes/TEs for these chromosomes (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>; TableÂ <a reveal-id="DST040TB2" data-open="DST040TB2" class="link link-reveal link-table xref-fig">2</a>). Interestingly, the homoeologous chromosomes A3 and C3 both exhibited the highest microsatellite frequency among all the pseudochromosomes of <em>B. rapa</em> or <em>B. oleracea</em>, respectively, which was in accordance with their highest gene frequency among these chromosomes (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>; TableÂ <a reveal-id="DST040TB2" data-open="DST040TB2" class="link link-reveal link-table xref-fig">2</a>). In accordance with the high significance of the <em>P</em>-values of the <em>Ï‡</em><sup>2</sup> test between the practical and hypothetical/average frequencies of microsatellites in the 1-Mb genomic intervals (TableÂ <a reveal-id="DST040TB2" data-open="DST040TB2" class="link link-reveal link-table xref-fig">2</a>), the physical distribution of microsatellites on all the pseudochromosomes of both <em>B. rapa</em> and <em>B. oleracea</em> were highly non-uniform (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>), which suggests the non-random occurrence of microsatellites. In accordance with the usually higher <em>P</em>-values of the <em>Ï‡</em><sup>2</sup> test between the practical and hypothetical/average frequencies of microsatellites for the 9 pseudochromosomes of <em>B. oleracea</em> than for the 10 pseudochromosomes of <em>B. rapa</em> (TableÂ <a reveal-id="DST040TB2" data-open="DST040TB2" class="link link-reveal link-table xref-fig">2</a>), the distribution of microsatellites was more uneven in <em>B. oleracea</em> than in <em>B. rapa</em> (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>), which was likely attributable to the more concentrated distribution of genes/TEs in <em>B. rapa</em> than in <em>B. oleracea</em>. For both <em>B. rapa</em> and <em>B. oleracea</em>, the frequencies of microsatellites in the 1-Mb genomic intervals studied were significantly positively or negatively correlated with those of genes (total <em>r</em> = 0.75 and 0.87) or TEs (total <em>r</em> = âˆ’0.61 and âˆ’0.73), respectively (TableÂ <a reveal-id="DST040TB2" data-open="DST040TB2" class="link link-reveal link-table xref-fig">2</a>), which was accordant with one of the interesting findings in this study, that the genomic distribution of microsatellites was generally in accordance with that of genes but opposite to that of TEs (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>). These results were in agreement with the previous findings, which showed that microsatellites are preferentially associated with non-repetitive DNA/gene sequences in the plant genome.<sup><a reveal-id="DST040C5" data-open="DST040C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST040C44" data-open="DST040C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The high agreement of microsatellites and genes strongly suggests the putative role of microsatellites in regulating gene function<sup><a reveal-id="DST040C3" data-open="DST040C3" class="link link-ref link-reveal xref-bibr">3</a>â€“<a reveal-id="DST040C5" data-open="DST040C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and the use of SSR markers for tagging/cloning genes.</p><p>In conclusion, the genomic distributions of microsatellites in the assembled pseudochromosomes of <em>B. rapa</em> and <em>B. oleracea</em> were generally similar.</p>                            <h3 scrollto-destination=4916825 id="4916825" class="section-title" data-legacy-id=s3c>3.3. Development and database of genome-wide SSR markers</h3>
<p>A total of 115 869 (92.1%), 185 662 (91.4%) and 356 522 (95.0%) primer pairs were successfully designed from the flanking sequences of 125 856, 203 161 and 375 214 mono- to hexanucleotide and compound microsatellites identified from the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively (TableÂ <a reveal-id="DST040TB3" data-open="DST040TB3" class="link link-reveal link-table xref-fig">3</a>). The primer pairs could not be designed for the remaining microsatellites, mostly due to the constraint of obtaining sufficient flanking sequences from either side of the identified microsatellites. Similar observations have also been observed in other genome-wide microsatellite marker development studies in plants, such as rice,<sup><a reveal-id="DST040C35" data-open="DST040C35" class="link link-ref link-reveal xref-bibr">35</a></sup> black cottonwood,<sup><a reveal-id="DST040C36" data-open="DST040C36" class="link link-ref link-reveal xref-bibr">36</a></sup> cucumber,<sup><a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a></sup><em>Brachypodim distachyon</em><sup><a reveal-id="DST040C38" data-open="DST040C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and foxtail millet.<sup><a reveal-id="DST040C39" data-open="DST040C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The frequencies of newly developed genome-wide SSR markers of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em> were 408.2, 343.8 and 356.2 per Mb or one every 2.45, 2.91 and 2.81 kb, respectively. Most of the genome-wide SSR markers of <em>B. rapa</em> (91.9%) and <em>B. oleracea</em> (75.4%) were developed from the mapped sequence scaffolds and thus have determined physical positions. The physical positions of the newly developed genome-wide SSR markers of <em>B. napus</em> will be determined soon because the anchoring of its sequence scaffolds will be completed after several months (our unpublished data). </p>                        <a id="4916827" scrollto-destination="4916827"></a>
<div class="table-wrap"><div id="DST040TB3" class="table-wrap-title"><div class="label -label">TableÂ 3.</div><div class="caption"><p>Number (%) of newly developed genome-wide SSR markers that generated certain numbers (from 0 to &gt;3) of <em>in silico</em> PCR products in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Markers fromÂ </th><th><em>In silico</em> PCR inÂ </th><th>ZeroÂ </th><th>OneÂ </th><th>TwoÂ </th><th>ThreeÂ </th><th>&gt;ThreeÂ </th><th>TotalÂ </th></tr></thead><tbody><tr><td>Â </td><td><em>B. rapa</em>Â </td><td>47 (0.0)Â </td><td>92 517 (79.8)Â </td><td>10 977 (9.5)Â </td><td>3309 (2.9)Â </td><td>9019 (7.8)Â </td><td>Â </td></tr><tr><td><em>B. rapa</em>Â </td><td><em>B. oleracea</em>Â </td><td>59 201 (51.1)Â </td><td>40 493 (34.9)Â </td><td>6370 (5.5)Â </td><td>2100 (1.8)Â </td><td>7705 (6.6)Â </td><td>115 869 (100)Â </td></tr><tr><td>Â </td><td><em>B. napus</em>Â </td><td>12 765 (11.0)Â </td><td>36 030 (31.1)Â </td><td>38 162 (32.9)Â </td><td>10 666 (9.2)Â </td><td>18 246 (15.7)Â </td><td>Â </td></tr><tr><td>Â </td><td><em>B. rapa</em>Â </td><td>110 784 (59.7)Â </td><td>48 423 (26.1)Â </td><td>8486 (4.6)Â </td><td>3349 (1.8)Â </td><td>14 620 (7.9)Â </td><td>Â </td></tr><tr><td><em>B. oleracea</em>Â </td><td><em>B. oleracea</em>Â </td><td>20 (0.0)Â </td><td>121 169 (65.3)Â </td><td>18 299 (9.9)Â </td><td>7140 (3.8)Â </td><td>39 034 (21.0)Â </td><td>185 662 (100)Â </td></tr><tr><td>Â </td><td><em>B. napus</em>Â </td><td>13 192 (7.1)Â </td><td>54 749 (29.5)Â </td><td>55 160 (29.7)Â </td><td>14 156 (7.6)Â </td><td>48 405 (26.1)Â </td><td>Â </td></tr><tr><td>Â </td><td><em>B. rapa</em>Â </td><td>157 473 (44.2)Â </td><td>138 202 (38.8)Â </td><td>20 604 (5.8)Â </td><td>7758 (2.2)Â </td><td>32 485 (9.1)Â </td><td>Â </td></tr><tr><td><em>B. napus</em>Â </td><td><em>B. oleracea</em>Â </td><td>106 706 (29.9)Â </td><td>153 690 (43.1)Â </td><td>24 601 (6.9)Â </td><td>10 264 (2.9)Â </td><td>61 261 (17.2)Â </td><td>356 522 (100)Â </td></tr><tr><td>Â </td><td><em>B. napus</em>Â </td><td>0 (0.0)Â </td><td>93 084 (26.1)Â </td><td>110 106 (30.9)Â </td><td>44 138 (12.4)Â </td><td>109 194 (30.6)Â </td><td>Â </td></tr></tbody></table></div><div content-id="DST040TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Markers fromÂ </th><th><em>In silico</em> PCR inÂ </th><th>ZeroÂ </th><th>OneÂ </th><th>TwoÂ </th><th>ThreeÂ </th><th>&gt;ThreeÂ </th><th>TotalÂ </th></tr></thead><tbody><tr><td>Â </td><td><em>B. rapa</em>Â </td><td>47 (0.0)Â </td><td>92 517 (79.8)Â </td><td>10 977 (9.5)Â </td><td>3309 (2.9)Â </td><td>9019 (7.8)Â </td><td>Â </td></tr><tr><td><em>B. rapa</em>Â </td><td><em>B. oleracea</em>Â </td><td>59 201 (51.1)Â </td><td>40 493 (34.9)Â </td><td>6370 (5.5)Â </td><td>2100 (1.8)Â </td><td>7705 (6.6)Â </td><td>115 869 (100)Â </td></tr><tr><td>Â </td><td><em>B. napus</em>Â </td><td>12 765 (11.0)Â </td><td>36 030 (31.1)Â </td><td>38 162 (32.9)Â </td><td>10 666 (9.2)Â </td><td>18 246 (15.7)Â </td><td>Â </td></tr><tr><td>Â </td><td><em>B. rapa</em>Â </td><td>110 784 (59.7)Â </td><td>48 423 (26.1)Â </td><td>8486 (4.6)Â </td><td>3349 (1.8)Â </td><td>14 620 (7.9)Â </td><td>Â </td></tr><tr><td><em>B. oleracea</em>Â </td><td><em>B. oleracea</em>Â </td><td>20 (0.0)Â </td><td>121 169 (65.3)Â </td><td>18 299 (9.9)Â </td><td>7140 (3.8)Â </td><td>39 034 (21.0)Â </td><td>185 662 (100)Â </td></tr><tr><td>Â </td><td><em>B. napus</em>Â </td><td>13 192 (7.1)Â </td><td>54 749 (29.5)Â </td><td>55 160 (29.7)Â </td><td>14 156 (7.6)Â </td><td>48 405 (26.1)Â </td><td>Â </td></tr><tr><td>Â </td><td><em>B. rapa</em>Â </td><td>157 473 (44.2)Â </td><td>138 202 (38.8)Â </td><td>20 604 (5.8)Â </td><td>7758 (2.2)Â </td><td>32 485 (9.1)Â </td><td>Â </td></tr><tr><td><em>B. napus</em>Â </td><td><em>B. oleracea</em>Â </td><td>106 706 (29.9)Â </td><td>153 690 (43.1)Â </td><td>24 601 (6.9)Â </td><td>10 264 (2.9)Â </td><td>61 261 (17.2)Â </td><td>356 522 (100)Â </td></tr><tr><td>Â </td><td><em>B. napus</em>Â </td><td>0 (0.0)Â </td><td>93 084 (26.1)Â </td><td>110 106 (30.9)Â </td><td>44 138 (12.4)Â </td><td>109 194 (30.6)Â </td><td>Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4916827" target="_blank">View Large</a></div></div><p>Because of the polyploidy nature of <em>Brassica</em>,<sup><a reveal-id="DST040C45" data-open="DST040C45" class="link link-ref link-reveal xref-bibr">45</a></sup> SSR markers usually amplify multiple fragments from homologous DNA sequences, which could complicate or cause errors in the genotype scoring. Therefore, all of the newly developed genome-wide SSR markers were subjected to <em>in silico</em> PCR analysis in the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, and the numbers of <em>in silico</em> PCR product(s) were recorded and summarized (TableÂ <a reveal-id="DST040TB3" data-open="DST040TB3" class="link link-reveal link-table xref-fig">3</a>). For the 115 869 SSR markers developed from <em>B. rapa</em>, 47 (0.0%), 92 517 (79.8%), 10 977 (9.5%), 3309 (2.9%) and 9019 (7.8%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. rapa</em>; 59 201 (51.1%), 40 493 (34.9%), 6370 (5.5%), 2100 (1.8%) and 7705 (6.6%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. oleracea</em> and 12 765 (11.0%), 36 030 (31.1%), 38 162 (32.9%), 10 666 (9.2%) and 18 246 (15.7%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. napus.</em> For the 185 662 SSR markers developed from <em>B. oleracea</em>, 20 (0.0%), 121 169 (65.3%), 18 299 (9.9%), 7140 (3.8%) and 39 034 (21.0%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. oleracea</em>; 110 784 (59.7%), 48 423 (26.1%), 8486 (4.6%), 3349 (1.8%) and 14 620 (7.9%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. rapa</em> and 13 192 (7.1%), 54 749 (29.5%), 55 160 (29.7%), 14 156 (7.6%) and 48 405 (26.1%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. napus.</em> For the 356 522 SSR markers developed from <em>B. napus</em>, 0 (0.0%), 93 084 (26.1%), 110 106 (30.9%), 44 138 (12.4%) and 109 194 (30.6%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. napus</em>; 157 473 (44.2%), 138 202 (38.8%), 20 604 (5.8%), 7758 (2.2%) and 32 485 (9.1%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. rapa</em> and 106 706 (29.9%), 153 690 (43.1%), 24 601 (6.9%), 10 264 (2.9%) and 61 261 (17.2%) markers generated 0, 1, 2, 3 and &gt;3 <em>in silico</em> PCR product(s), respectively, from the assembled genomic sequences of <em>B. oleracea</em>. Interestingly, the SSR markers that generated tens to thousands of <em>in silico</em> PCR products were mostly associated with the annotated TEs, especially the retrotransposons.</p><p>We also determined the relationship between the physical positions of the newly developed genome-wide SSR markers and the previously developed publicly available <em>Brassica</em> SSR markers as well as the annotated genome components (mainly as genes and TEs) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Of the 115 869 SSR markers developed from <em>B. rapa</em>, 5991 (5.2%), 22 596 (19.5%) and 32 648 (28.2%) were involved in public <em>Brassica</em> SSR markers, genes and TEs, respectively. Of the 185 662 SSR markers developed from <em>B. oleracea</em>, 12 322 (6.6%), 33 228 (17.9%) and 73 487 (39.6%) were involved in public <em>Brassica</em> SSR markers, genes and TEs, respectively. Of the 356 522 SSR markers developed from <em>B. napus</em>, 23 928 (6.7%), 58 952 (16.5%) and 161 090 (45.2%) were involved in public <em>Brassica</em> SSR markers, genes and TEs, respectively. Interestingly, the TE-associated SSR markers were rarely involved in the annotated genes and mostly generated tens to thousands of <em>in silico</em> PCR products.</p><p>To facilitate the access and effective utilization of the <em>Brassica</em> SSR markers, we constructed an integrative database (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>), which has search tools to obtain much useful information for the newly developed genome-wide SSR markers from the sequenced <em>Brassica</em> species and the previously developed publicly available <em>Brassica</em> SSR markers (Fig.Â <a reveal-id="DST040F4" data-open="DST040F4" class="link link-reveal link-table xref-fig">4</a>). For the previously developed publicly available <em>Brassica</em> SSR markers, this information includes the primer-pair sequences, microsatellite repeat, source, reference and number of <em>in silico</em> PCR product(s) in the assembled genomic sequences of the sequenced <em>Brassica</em> species (currently only for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>). For the newly developed genome-wide SSR markers from the sequenced <em>Brassica</em> species, this information includes the following: (i) the sequence, type, length and physical position of microsatellite repeat; (ii) the serial number, sequences, annealing temperatures, lengths and expected product size of primer pair; (iii) the number of <em>in silico</em> PCR product(s) in the assembled genomic sequences of the sequenced <em>Brassica</em> species (currently only for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>) and (iv) the positional relationship with the previously developed publicly available <em>Brassica</em> SSR markers and the annotated genome components (mainly genes and TEs). In addition, this database also provides useful analysis tools (such as BLAST, e-PCR, Primer3 and ExtractSeq) and web links to other databases (e.g. <a class="link link-uri" href="http://brassicadb.org/brad/">http://brassicadb.org/brad/</a>) and websites (e.g. <a class="link link-uri" href="http://www.brassica.info/">http://www.brassica.info/</a>) related to <em>Brassica</em> research. More importantly, this SSR marker database for <em>Brassica</em> will update as the number of sequenced <em>Brassica</em> species increases. </p>                        <a id="4916831" scrollto-destination="4916831"></a>
<div id="DST040F4" data-id="DST040F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04004.png?Expires=1509238921&Signature=Rs7~oTfHCLVsuU3fkg8kkGa1bS2JeIU~Kv~YuyuR5WTE2NAwZBqbUTAmsjfmcdoX3ETcySCQF1jsb-2vkyyU3c31aAFAMzq2gWyG-ALCWzJTbBxLUklCrMeBVuri7e59xeAREBf9JdJfV73NLjG1A-iFn1bXoIxZI2pIjFzhilqTfifoPFM-9RW3DKgzZ5ULBuw~g-HmA5ct-NYoHkuPZ4L8EJZBuXCTuLBueO3sixShcFuEMJoSeCxFQHYkm4iQ8Kq-i5HkF7vR1OtXUgnCwUySaNr77QRwrT~cc99fPPh57qzHyTtq2CTfk1cdfy85ODKrF1myStlA9FNxeTViWQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Brassica SSR marker database homepage (A) and search results pages (Bâ€“D). This database provides three search approaches: by the name of the previously developed publicly available Brassica SSR markers (e.g. BN12A), by the serial number of the SSR markers (e.g. BnScaffold000001_12) and by the start to end positions on the chromosome/scaffold (e.g. BnScaffold000001:20â€“20 000 bp). This database also provides many useful tools, such as e-PCR, BLAST and primer3. Figure Bâ€“D show the results of examples of the three search approaches that are provided in the homepage." path-from-xml="dst04004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916831/dst04004.tiff" path-from-xml="dst04004.tiff" target="_blank">View large</a><a section="4916831" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04004.png?Expires=1509238921&Signature=RONfHbgFXzBuMQx3n~zuFLSlBvYfvlO2ecP1OcPQ60b8ccxUvcRjRl5kHDijugrhF9uAIoD4cFvhEsHD1pSqz6QjUZ1dgEiZ8ZxifTf5n7v8DN7P0JizOTLHlzPKEHEgKVtRcZ-C75MBSnDQijbPo0q3hLr~buQnId6gXzjAlmk7dXU1Gd6KIppcxdLSjyVaPRI5~z8MwvJTsWsVNprlmxhPRr3B2bglpye84SQyMlTLhj3mT9ge0a619ETXN6PpMhCrvpHCf3aSm2e36uyre-LE8G5iGHHUEra372liwmxzGUZu8Ace2ZYRWs6B~h1Jbv2jReCJk2fI4ttpMuSUmQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916831&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Brassica</em> SSR marker database homepage (A) and search results pages (Bâ€“D). This database provides three search approaches: by the name of the previously developed publicly available <em>Brassica</em> SSR markers (e.g. BN12A), by the serial number of the SSR markers (e.g. BnScaffold000001_12) and by the start to end positions on the chromosome/scaffold (e.g. BnScaffold000001:20â€“20 000 bp). This database also provides many useful tools, such as e-PCR, BLAST and primer3. Figure Bâ€“D show the results of examples of the three search approaches that are provided in the homepage.</p></div></div><div content-id="DST040F4" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/m_dst04004.png?Expires=1509238921&Signature=Rs7~oTfHCLVsuU3fkg8kkGa1bS2JeIU~Kv~YuyuR5WTE2NAwZBqbUTAmsjfmcdoX3ETcySCQF1jsb-2vkyyU3c31aAFAMzq2gWyG-ALCWzJTbBxLUklCrMeBVuri7e59xeAREBf9JdJfV73NLjG1A-iFn1bXoIxZI2pIjFzhilqTfifoPFM-9RW3DKgzZ5ULBuw~g-HmA5ct-NYoHkuPZ4L8EJZBuXCTuLBueO3sixShcFuEMJoSeCxFQHYkm4iQ8Kq-i5HkF7vR1OtXUgnCwUySaNr77QRwrT~cc99fPPh57qzHyTtq2CTfk1cdfy85ODKrF1myStlA9FNxeTViWQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Brassica SSR marker database homepage (A) and search results pages (Bâ€“D). This database provides three search approaches: by the name of the previously developed publicly available Brassica SSR markers (e.g. BN12A), by the serial number of the SSR markers (e.g. BnScaffold000001_12) and by the start to end positions on the chromosome/scaffold (e.g. BnScaffold000001:20â€“20 000 bp). This database also provides many useful tools, such as e-PCR, BLAST and primer3. Figure Bâ€“D show the results of examples of the three search approaches that are provided in the homepage." path-from-xml="dst04004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4916831/dst04004.tiff" path-from-xml="dst04004.tiff" target="_blank">View large</a><a section="4916831" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst04004.png?Expires=1509238921&Signature=RONfHbgFXzBuMQx3n~zuFLSlBvYfvlO2ecP1OcPQ60b8ccxUvcRjRl5kHDijugrhF9uAIoD4cFvhEsHD1pSqz6QjUZ1dgEiZ8ZxifTf5n7v8DN7P0JizOTLHlzPKEHEgKVtRcZ-C75MBSnDQijbPo0q3hLr~buQnId6gXzjAlmk7dXU1Gd6KIppcxdLSjyVaPRI5~z8MwvJTsWsVNprlmxhPRr3B2bglpye84SQyMlTLhj3mT9ge0a619ETXN6PpMhCrvpHCf3aSm2e36uyre-LE8G5iGHHUEra372liwmxzGUZu8Ace2ZYRWs6B~h1Jbv2jReCJk2fI4ttpMuSUmQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4916831&ar=347430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Brassica</em> SSR marker database homepage (A) and search results pages (Bâ€“D). This database provides three search approaches: by the name of the previously developed publicly available <em>Brassica</em> SSR markers (e.g. BN12A), by the serial number of the SSR markers (e.g. BnScaffold000001_12) and by the start to end positions on the chromosome/scaffold (e.g. BnScaffold000001:20â€“20 000 bp). This database also provides many useful tools, such as e-PCR, BLAST and primer3. Figure Bâ€“D show the results of examples of the three search approaches that are provided in the homepage.</p></div></div>                            <h3 scrollto-destination=4916832 id="4916832" class="section-title" data-legacy-id=s3d>3.4. Experimental evaluation of the newly developed genome-wide SSR markers</h3>
<p>A total of 3974 newly developed SSR markers from <em>B. rapa</em> and <em>B. oleracea</em> were tested for amplification in the six representative <em>B. napus</em> cultivars/inbred lines (TableÂ <a reveal-id="DST040TB4" data-open="DST040TB4" class="link link-reveal link-table xref-fig">4</a>). Of these, 3880 SSR markers (97.6%) successfully amplified at least one clear fragment, while the remaining 94 (2.4%) failed to amplify, which could be due to the differences between the genome sequences of <em>B. napus</em> and its two progenitors, <em>B. rapa</em> and <em>B. oleracea</em>.<sup><a reveal-id="DST040C46" data-open="DST040C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DST040C47" data-open="DST040C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The amplification rate (97.6%) of the tested SSR markers in the six <em>B. napus</em> cultivars/inbred lines was slightly or much higher than the corresponding rates (94.3/82.9, 89.2 and 77.4%, respectively) for the previously developed SSR markers from GSSs (mainly BESs)/unique transcripts of <em>B. napus</em>,<sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a></sup> BACs of <em>B. rapa</em><sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and WGSs of <em>B. oleracea</em>,<sup><a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> which suggests that there is a high quality in the SSR markers that were developed from the assembled genomic sequences. The amplification rate of the tested SSR markers showed small variations for different motif lengths, motif repeat numbers and repeat lengths (i.e. motif length Ã— motif repeat number), which was consistent with the previous reports in <em>Brassica</em><sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and rice.<sup><a reveal-id="DST040C35" data-open="DST040C35" class="link link-ref link-reveal xref-bibr">35</a></sup> For the majority of the tested SSR markers, the numbers of fragment(s) amplified from the six representative <em>B. napus</em> cultivars/inbred lines were equal or very close to those of <em>in silico</em> PCR product(s) in the assembled genomic sequences of <em>B. napus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). In particular, most (1602 of 1813; 88.4%) of the tested SSR markers that generated one <em>in silico</em> PCR product in the assembled genomic sequences of <em>B. napus</em> also amplified only a single clear fragment from the six representative <em>B. napus</em> cultivars/inbred lines. A considerable proportion (1099 of 3880; 28.3%) of the successfully amplified SSR markers also produced weak fragment(s), which could correspond to non-specific amplification(s) from homologous DNA sequences. </p>                        <a id="4916834" scrollto-destination="4916834"></a>
<div class="table-wrap"><div id="DST040TB4" class="table-wrap-title"><div class="label -label">TableÂ 4.</div><div class="caption"><p>Amplification and polymorphism rate of the tested SSR markers and their association with the number of amplified fragment(s), the motif length, the motif repeat number and the repeat length</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>Â </th><th colspan="2">Total markers<hr />Â </th><th colspan="2">Amplified markers<hr />Â </th><th colspan="2">Polymorphic markers<hr />Â </th></tr><tr><th>Â </th><th>Â </th><th>NumberÂ </th><th>%Â </th><th>NumberÂ </th><th>%Â </th><th>NumberÂ </th><th>%Â </th></tr></thead><tbody><tr><td>Â </td><td>0Â </td><td>94Â </td><td>2.4Â </td><td>/Â </td><td>/Â </td><td>/Â </td><td>/Â </td></tr><tr><td>Â </td><td>1Â </td><td>1859Â </td><td>46.8Â </td><td>/Â </td><td>/Â </td><td>1255Â </td><td>67.5Â </td></tr><tr><td>Fragment (s) numberÂ </td><td>2Â </td><td>1541Â </td><td>38.8Â </td><td>/Â </td><td>/Â </td><td>1124Â </td><td>72.9Â </td></tr><tr><td>Â </td><td>3Â </td><td>352Â </td><td>8.9Â </td><td>/Â </td><td>/Â </td><td>276Â </td><td>78.4Â </td></tr><tr><td>Â </td><td>&gt;3Â </td><td>128Â </td><td>3.2Â </td><td>/Â </td><td>/Â </td><td>110Â </td><td>85.9Â </td></tr><tr><td>Â </td><td>MonoÂ </td><td>35Â </td><td>0.9Â </td><td>34Â </td><td>97.1Â </td><td>26Â </td><td>76.5Â </td></tr><tr><td>Â </td><td>DiÂ </td><td>3156Â </td><td>79.4Â </td><td>3077Â </td><td>97.5Â </td><td>2195Â </td><td>71.3Â </td></tr><tr><td>Â </td><td>TriÂ </td><td>594Â </td><td>14.9Â </td><td>584Â </td><td>98.3Â </td><td>397Â </td><td>68.0Â </td></tr><tr><td>Motif lengthÂ </td><td>TetraÂ </td><td>74Â </td><td>1.9Â </td><td>73Â </td><td>98.6Â </td><td>41Â </td><td>56.2Â </td></tr><tr><td>Â </td><td>PentaÂ </td><td>21Â </td><td>0.5Â </td><td>20Â </td><td>95.2Â </td><td>15Â </td><td>75.0Â </td></tr><tr><td>Â </td><td>HexaÂ </td><td>12Â </td><td>0.3Â </td><td>12Â </td><td>100.0Â </td><td>11Â </td><td>91.7Â </td></tr><tr><td>Â </td><td>CompoundÂ </td><td>82Â </td><td>2.1Â </td><td>80Â </td><td>97.6Â </td><td>80Â </td><td>100.0Â </td></tr><tr><td>Â </td><td>5Â </td><td>496Â </td><td>12.5Â </td><td>488Â </td><td>98.4Â </td><td>316Â </td><td>64.8Â </td></tr><tr><td>Â </td><td>6Â </td><td>98Â </td><td>2.5Â </td><td>96Â </td><td>98.0Â </td><td>67Â </td><td>69.8Â </td></tr><tr><td>Â </td><td>7Â </td><td>59Â </td><td>1.5Â </td><td>57Â </td><td>96.6Â </td><td>41Â </td><td>71.9Â </td></tr><tr><td>Â </td><td>10Â </td><td>791Â </td><td>19.9Â </td><td>765Â </td><td>96.7Â </td><td>539Â </td><td>70.5Â </td></tr><tr><td>Â </td><td>11Â </td><td>528Â </td><td>13.3Â </td><td>513Â </td><td>97.2Â </td><td>347Â </td><td>67.6Â </td></tr><tr><td>Motif repeat numberÂ </td><td>12Â </td><td>388Â </td><td>9.8Â </td><td>375Â </td><td>96.6Â </td><td>252Â </td><td>67.2Â </td></tr><tr><td>Â </td><td>13Â </td><td>576Â </td><td>14.5Â </td><td>567Â </td><td>98.4Â </td><td>386Â </td><td>68.1Â </td></tr><tr><td>Â </td><td>14Â </td><td>295Â </td><td>7.4Â </td><td>288Â </td><td>97.6Â </td><td>209Â </td><td>72.6Â </td></tr><tr><td>Â </td><td>15Â </td><td>98Â </td><td>2.5Â </td><td>95Â </td><td>96.9Â </td><td>76Â </td><td>80.0Â </td></tr><tr><td>Â </td><td>16Â </td><td>140Â </td><td>3.5Â </td><td>139Â </td><td>99.3Â </td><td>114Â </td><td>82.0Â </td></tr><tr><td>Â </td><td>17Â </td><td>105Â </td><td>2.6Â </td><td>105Â </td><td>100.0Â </td><td>88Â </td><td>83.8Â </td></tr><tr><td>Â </td><td>OthersÂ </td><td>400Â </td><td>10.1Â </td><td>392Â </td><td>98.0Â </td><td>330Â </td><td>84.2Â </td></tr><tr><td>Â </td><td>15Â </td><td>412Â </td><td>10.4Â </td><td>405Â </td><td>98.3Â </td><td>266Â </td><td>65.7Â </td></tr><tr><td>Â </td><td>18Â </td><td>82Â </td><td>2.1Â </td><td>80Â </td><td>97.6Â </td><td>55Â </td><td>68.8Â </td></tr><tr><td>Â </td><td>20Â </td><td>840Â </td><td>21.1Â </td><td>814Â </td><td>96.9Â </td><td>561Â </td><td>68.9Â </td></tr><tr><td>Â </td><td>21Â </td><td>58Â </td><td>1.5Â </td><td>56Â </td><td>96.6Â </td><td>40Â </td><td>71.4Â </td></tr><tr><td>Â </td><td>22Â </td><td>526Â </td><td>13.2Â </td><td>511Â </td><td>97.1Â </td><td>344Â </td><td>67.3Â </td></tr><tr><td>Repeat lengthÂ </td><td>24Â </td><td>418Â </td><td>10.5Â </td><td>405Â </td><td>96.9Â </td><td>275Â </td><td>67.9Â </td></tr><tr><td>Â </td><td>26Â </td><td>577Â </td><td>14.5Â </td><td>568Â </td><td>98.4Â </td><td>387Â </td><td>68.1Â </td></tr><tr><td>Â </td><td>28Â </td><td>303Â </td><td>7.6Â </td><td>295Â </td><td>97.4Â </td><td>213Â </td><td>72.2Â </td></tr><tr><td>Â </td><td>30Â </td><td>122Â </td><td>3.1Â </td><td>119Â </td><td>97.5Â </td><td>96Â </td><td>80.7Â </td></tr><tr><td>Â </td><td>32Â </td><td>133Â </td><td>3.3Â </td><td>132Â </td><td>99.2Â </td><td>107Â </td><td>81.1Â </td></tr><tr><td>Â </td><td>34Â </td><td>97Â </td><td>2.4Â </td><td>97Â </td><td>100.0Â </td><td>80Â </td><td>82.5Â </td></tr><tr><td>Â </td><td>OthersÂ </td><td>406Â </td><td>10.2Â </td><td>398Â </td><td>98.0Â </td><td>341Â </td><td>85.7Â </td></tr><tr><td>TotalÂ </td><td>Â </td><td>3974Â </td><td>100Â </td><td>3880Â </td><td>97.6Â </td><td>2765Â </td><td>71.3Â </td></tr></tbody></table></div><div content-id="DST040TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>Â </th><th colspan="2">Total markers<hr />Â </th><th colspan="2">Amplified markers<hr />Â </th><th colspan="2">Polymorphic markers<hr />Â </th></tr><tr><th>Â </th><th>Â </th><th>NumberÂ </th><th>%Â </th><th>NumberÂ </th><th>%Â </th><th>NumberÂ </th><th>%Â </th></tr></thead><tbody><tr><td>Â </td><td>0Â </td><td>94Â </td><td>2.4Â </td><td>/Â </td><td>/Â </td><td>/Â </td><td>/Â </td></tr><tr><td>Â </td><td>1Â </td><td>1859Â </td><td>46.8Â </td><td>/Â </td><td>/Â </td><td>1255Â </td><td>67.5Â </td></tr><tr><td>Fragment (s) numberÂ </td><td>2Â </td><td>1541Â </td><td>38.8Â </td><td>/Â </td><td>/Â </td><td>1124Â </td><td>72.9Â </td></tr><tr><td>Â </td><td>3Â </td><td>352Â </td><td>8.9Â </td><td>/Â </td><td>/Â </td><td>276Â </td><td>78.4Â </td></tr><tr><td>Â </td><td>&gt;3Â </td><td>128Â </td><td>3.2Â </td><td>/Â </td><td>/Â </td><td>110Â </td><td>85.9Â </td></tr><tr><td>Â </td><td>MonoÂ </td><td>35Â </td><td>0.9Â </td><td>34Â </td><td>97.1Â </td><td>26Â </td><td>76.5Â </td></tr><tr><td>Â </td><td>DiÂ </td><td>3156Â </td><td>79.4Â </td><td>3077Â </td><td>97.5Â </td><td>2195Â </td><td>71.3Â </td></tr><tr><td>Â </td><td>TriÂ </td><td>594Â </td><td>14.9Â </td><td>584Â </td><td>98.3Â </td><td>397Â </td><td>68.0Â </td></tr><tr><td>Motif lengthÂ </td><td>TetraÂ </td><td>74Â </td><td>1.9Â </td><td>73Â </td><td>98.6Â </td><td>41Â </td><td>56.2Â </td></tr><tr><td>Â </td><td>PentaÂ </td><td>21Â </td><td>0.5Â </td><td>20Â </td><td>95.2Â </td><td>15Â </td><td>75.0Â </td></tr><tr><td>Â </td><td>HexaÂ </td><td>12Â </td><td>0.3Â </td><td>12Â </td><td>100.0Â </td><td>11Â </td><td>91.7Â </td></tr><tr><td>Â </td><td>CompoundÂ </td><td>82Â </td><td>2.1Â </td><td>80Â </td><td>97.6Â </td><td>80Â </td><td>100.0Â </td></tr><tr><td>Â </td><td>5Â </td><td>496Â </td><td>12.5Â </td><td>488Â </td><td>98.4Â </td><td>316Â </td><td>64.8Â </td></tr><tr><td>Â </td><td>6Â </td><td>98Â </td><td>2.5Â </td><td>96Â </td><td>98.0Â </td><td>67Â </td><td>69.8Â </td></tr><tr><td>Â </td><td>7Â </td><td>59Â </td><td>1.5Â </td><td>57Â </td><td>96.6Â </td><td>41Â </td><td>71.9Â </td></tr><tr><td>Â </td><td>10Â </td><td>791Â </td><td>19.9Â </td><td>765Â </td><td>96.7Â </td><td>539Â </td><td>70.5Â </td></tr><tr><td>Â </td><td>11Â </td><td>528Â </td><td>13.3Â </td><td>513Â </td><td>97.2Â </td><td>347Â </td><td>67.6Â </td></tr><tr><td>Motif repeat numberÂ </td><td>12Â </td><td>388Â </td><td>9.8Â </td><td>375Â </td><td>96.6Â </td><td>252Â </td><td>67.2Â </td></tr><tr><td>Â </td><td>13Â </td><td>576Â </td><td>14.5Â </td><td>567Â </td><td>98.4Â </td><td>386Â </td><td>68.1Â </td></tr><tr><td>Â </td><td>14Â </td><td>295Â </td><td>7.4Â </td><td>288Â </td><td>97.6Â </td><td>209Â </td><td>72.6Â </td></tr><tr><td>Â </td><td>15Â </td><td>98Â </td><td>2.5Â </td><td>95Â </td><td>96.9Â </td><td>76Â </td><td>80.0Â </td></tr><tr><td>Â </td><td>16Â </td><td>140Â </td><td>3.5Â </td><td>139Â </td><td>99.3Â </td><td>114Â </td><td>82.0Â </td></tr><tr><td>Â </td><td>17Â </td><td>105Â </td><td>2.6Â </td><td>105Â </td><td>100.0Â </td><td>88Â </td><td>83.8Â </td></tr><tr><td>Â </td><td>OthersÂ </td><td>400Â </td><td>10.1Â </td><td>392Â </td><td>98.0Â </td><td>330Â </td><td>84.2Â </td></tr><tr><td>Â </td><td>15Â </td><td>412Â </td><td>10.4Â </td><td>405Â </td><td>98.3Â </td><td>266Â </td><td>65.7Â </td></tr><tr><td>Â </td><td>18Â </td><td>82Â </td><td>2.1Â </td><td>80Â </td><td>97.6Â </td><td>55Â </td><td>68.8Â </td></tr><tr><td>Â </td><td>20Â </td><td>840Â </td><td>21.1Â </td><td>814Â </td><td>96.9Â </td><td>561Â </td><td>68.9Â </td></tr><tr><td>Â </td><td>21Â </td><td>58Â </td><td>1.5Â </td><td>56Â </td><td>96.6Â </td><td>40Â </td><td>71.4Â </td></tr><tr><td>Â </td><td>22Â </td><td>526Â </td><td>13.2Â </td><td>511Â </td><td>97.1Â </td><td>344Â </td><td>67.3Â </td></tr><tr><td>Repeat lengthÂ </td><td>24Â </td><td>418Â </td><td>10.5Â </td><td>405Â </td><td>96.9Â </td><td>275Â </td><td>67.9Â </td></tr><tr><td>Â </td><td>26Â </td><td>577Â </td><td>14.5Â </td><td>568Â </td><td>98.4Â </td><td>387Â </td><td>68.1Â </td></tr><tr><td>Â </td><td>28Â </td><td>303Â </td><td>7.6Â </td><td>295Â </td><td>97.4Â </td><td>213Â </td><td>72.2Â </td></tr><tr><td>Â </td><td>30Â </td><td>122Â </td><td>3.1Â </td><td>119Â </td><td>97.5Â </td><td>96Â </td><td>80.7Â </td></tr><tr><td>Â </td><td>32Â </td><td>133Â </td><td>3.3Â </td><td>132Â </td><td>99.2Â </td><td>107Â </td><td>81.1Â </td></tr><tr><td>Â </td><td>34Â </td><td>97Â </td><td>2.4Â </td><td>97Â </td><td>100.0Â </td><td>80Â </td><td>82.5Â </td></tr><tr><td>Â </td><td>OthersÂ </td><td>406Â </td><td>10.2Â </td><td>398Â </td><td>98.0Â </td><td>341Â </td><td>85.7Â </td></tr><tr><td>TotalÂ </td><td>Â </td><td>3974Â </td><td>100Â </td><td>3880Â </td><td>97.6Â </td><td>2765Â </td><td>71.3Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4916834" target="_blank">View Large</a></div></div><p>The majority (2765 of 3880; 71.3%) of the successfully amplified SSR markers was polymorphic across the six representative <em>B. napus</em> cultivars/inbred lines (TableÂ <a reveal-id="DST040TB4" data-open="DST040TB4" class="link link-reveal link-table xref-fig">4</a>). The polymorphism rate of the tested SSR markers was almost equal to or slightly higher than the corresponding rates (73.4/65.9, 57.9, 69.5%) for the previously developed SSR markers from GSSs (mainly BESs)/unique transcripts of <em>B. napus</em>,<sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a></sup> BACs of <em>B. rapa</em><sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and WGSs of <em>B. oleracea</em>.<sup><a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Obviously, the polymorphism rate of the tested SSR markers increased (from 67.5% to 85.9%) as the number of amplified bands increased (from 1 to &gt;3). The polymorphism rate of the tested SSR markers decreased slightly from the mono- to tetranucleotide repeats, while it increased quickly from the penta- to hexanucleotide repeats. This inconsistency of the relationship between the SSR marker polymorphism level and the motif length was also observed frequently in the previous SSR marker evaluation experiments, such as in the tests of the 627 and 1000 SSR markers from the GSSs (mainly BESs) and unique transcripts, respectively, of <em>B. napus</em>,<sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a></sup> the 890 SSR markers from the BACs of <em>B. rapa</em>,<sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> the 1398 SSR markers from the WGSs of <em>B. oleracea</em><sup><a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and the 1009 SSR markers from the assembled genomic sequences of cucumber.<sup><a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a></sup> This type of inconsistency could be attributable to the observation that only a small number of SSR markers of the specific (usually long) motif length(s) have been used to investigate this relationship in all of the above-mentioned studies (e.g. only 21 and 12 penta- and hexanucleotide repeat SSR markers were tested in the current investigation), which worthwhile to develop more SSR markers with long motifs to further investigate the relationship between the SSR marker polymorphism level and the motif length. The polymorphism rate of the tested SSR markers was highly positively correlated with both the motif repeat number and the repeat length (<em>r</em> = 0.74 and 0.86, respectively), which was basically consistent with the previous reports in <em>Brassica</em><sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and other plant species, including cucumber<sup><a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and carrot.<sup><a reveal-id="DST040C48" data-open="DST040C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Both correlation coefficients in the current investigation were much higher than or equal to the corresponding values (0.21 and 0.41; 0.74 and _) that were estimated with the 627 SSR markers from the GSSs (mainly BESs) of <em>B. napus</em><sup><a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a></sup> or the 1009 SSR markers from the assembled genomic sequences of cucumber,<sup><a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a></sup> respectively. Strikingly, the tested SSR markers that were designed from compound repeats were almost all (80 of 82; 97.6%) polymorphic across the six representative <em>B. napus</em> cultivars/inbred lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>).</p><p>Because the 1055 and 2919 tested SSR markers were developed from the sequence scaffolds of <em>B. rapa</em> and <em>B. oleracea</em>, respectively, they were thus designated as â€˜BrSFâ€™ and â€˜BoSFâ€™. To facilitate the effective utilization of these tested newly developed BrSF and BoSF SSR markers, the following useful information was provided (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>): (i) the type, length, position and sequence of the microsatellite repeat; (ii) the name, sequences, annealing temperatures and expected product size of the primer pair; (iii) the number of <em>in silico</em> PCR product(s) in the assembled genomic sequences of the sequenced <em>Brassica</em> species (currently for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>) and (iv) the polymorphism survey and number of fragment(s) amplified in six representative <em>B. napus</em> cultivars/inbred lines.</p>                            <h2 scrollto-destination=4916837 id="4916837" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=4916838 id="4916838" class="section-title" data-legacy-id=s4a>4.1. The pattern of microsatellite distribution is likely conservative in <em>Brassica</em></h3>
<p>In the current study, almost all of the important characteristics of microsatellite distribution in the assembled genomic sequences of the three recently sequenced <em>Brassica</em> crop species have been analysed and compared. To the best of our knowledge, this study is the first report on the genome-wide analysis and comparison of the pattern of microsatellite distribution across the different species within the same genus in plants.</p><p>First, the frequencies of microsatellites in the assembled genomic sequences of <em>B. rapa</em> (496.8 per Mb), <em>B. oleracea</em> (424.8 per Mb) and <em>B. napus</em> (420.6 per Mb) were similar, and all were higher than almost all of the previous estimations.<sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST040C21" data-open="DST040C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DST040C42" data-open="DST040C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The slightly higher frequency of microsatellites in <em>B. rapa</em> than in both <em>B. oleracea</em> and <em>B. napus</em> is likely attributable to the more concentrated distribution and lower content of TEs in the assembled genomic sequences of <em>B. rapa</em> than in <em>B. oleracea</em> and <em>B. napus</em> (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>) because the frequencies (285.5, 272.0 and 285.4 per Mb) of microsatellites in the coding DNA sequences of the three species are almost equal.<sup><a reveal-id="DST040C49" data-open="DST040C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Secondly, in accordance with the high correlation between these variables (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), the distributions with respect to the motif length, type and repeat number of microsatellites in the assembled genomic sequences of the three <em>Brassica</em> species were almost identical (Fig.Â <a reveal-id="DST040F1" data-open="DST040F1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). More specifically, both the dominant/major and absent/scarce mono- to hexanucleotide motifs in the assembled genomic sequences of the three <em>Brassica</em> species were mostly identical (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Interestingly, the dominant/major motifs were all A/T rich, while the absent/scarce motifs were mostly C/G rich, which corresponded well to the much higher A/T than C/G content in the analysed sequences. Thirdly, the repeat numbers of the corresponding repeats or motifs for the three <em>Brassica</em> species were mostly similar (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Fourthly, the total repeat length (=microsatellite number Ã— motif length Ã— motif repeat number) proportions of the corresponding repeats or motifs of microsatellites in the assembled genomic sequences of the three <em>Brassica</em> species were also mostly similar (TableÂ <a reveal-id="DST040TB1" data-open="DST040TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). In addition, the genomic distributions of microsatellites in the assembled pseudochromosomes of <em>B. rapa</em> and <em>B. oleracea</em> were generally similar (Fig.Â <a reveal-id="DST040F3" data-open="DST040F3" class="link link-reveal link-table xref-fig">3</a>).</p><p>In conclusion, almost all of the analysed important characteristics of microsatellite distribution in the assembled genomic sequences of the three sequenced <em>Brassica</em> crop species were highly similar or almost identical, which suggests that the pattern of microsatellite distribution is likely conservative in <em>Brassica</em>. This circumstance is understandable because <em>B. napus</em> (AACC, 2n = 38) originated from the chromosome doubling after the very recent (â‰ˆ0.01 MYA) natural hybridization between <em>B. rapa</em> (AA, 2n = 20) and <em>B. oleracea</em> (CC, 2n = 18),<sup><a reveal-id="DST040C11" data-open="DST040C11" class="link link-ref link-reveal xref-bibr">11</a></sup> which diverged from a common ancestor only â‰ˆ5 MYA.<sup><a reveal-id="DST040C50" data-open="DST040C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p>                            <h3 scrollto-destination=4916842 id="4916842" class="section-title" data-legacy-id=s4b>4.2. Usefulness of the newly developed genome-wide <em>Brassica</em> SSR markers</h3>
<p>In the current study, a total of 115 869, 185 662 and 356 522 SSR markers were successfully developed from the assembled genomic sequences of <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively (TableÂ <a reveal-id="DST040TB3" data-open="DST040TB3" class="link link-reveal link-table xref-fig">3</a>), with the frequencies of 408.2, 343.8 and 356.2 per Mb or one every 2.45, 2.91 and 2.81 kb. To the best of our knowledge, this study is the first report on genome-wide SSR marker development in <em>Brassica</em>. Only a small proportion of the newly developed genome-wide SSR markers (5.2, 6.6 and 6.7% for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively) were involved in the previously developed publicly available <em>Brassica</em> SSR markers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>), which suggests that most of the newly developed genome-wide SSR markers should represent the new SSR markers. The huge-number and high-frequency genome-wide SSR markers developed from the sequenced <em>Brassica</em> species in this study could be useful for many studies that require large-number and/or high-density molecular markers, such as high-density linkage map construction, gene/QTL fine mapping and genome-wide/regional association mapping.</p><p>The acute physical positions of the majority of the newly developed genome-wide SSR markers of the sequenced <em>Brassica</em> species have been determined (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>) based on the mapped sequence scaffolds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>) from which they are designed. In fact, the physical positions of most of the previously developed publicly available <em>Brassica</em> SSR markers have also been determined by <em>in silico</em> mapping against the pseudochromosomes of these sequenced <em>Brassica</em> species (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>). The high-density SSR marker-based physical maps constructed in this study could be useful for the rapid selection of genome-wide SSR markers that are well distributed over these chromosomes for various genotyping applications.</p><p>Because of the polyploidy nature of <em>Brassica</em>,<sup><a reveal-id="DST040C45" data-open="DST040C45" class="link link-ref link-reveal xref-bibr">45</a></sup> the developed SSR markers usually amplify multiple fragments from the homologous DNA sequences, as revealed in the current (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>) and previous<sup><a reveal-id="DST040C12" data-open="DST040C12" class="link link-ref link-reveal xref-bibr">12</a>â€“<a reveal-id="DST040C14" data-open="DST040C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST040C22" data-open="DST040C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DST040C27" data-open="DST040C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST040C29" data-open="DST040C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DST040C42" data-open="DST040C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DST040C51" data-open="DST040C51" class="link link-ref link-reveal xref-bibr">51</a></sup> studies in <em>Brassica</em>. This could complicate or cause errors in the genotype scoring due to the reciprocal overlapping and uncertain allelism of these fragments.<sup><a reveal-id="DST040C33" data-open="DST040C33" class="link link-ref link-reveal xref-bibr">33</a></sup> However, only a small proportion of the previously developed publicly available <em>Brassica</em> SSR markers have been alleged to be single locus.<sup><a reveal-id="DST040C33" data-open="DST040C33" class="link link-ref link-reveal xref-bibr">33</a></sup>. Therefore, there is an urgent need to develop more single-locus SSR markers to facilitate their application in <em>Brassica</em>. Previously, the single-locus SSR markers were developed by practical PCR amplification in a panel of inbred lines,<sup><a reveal-id="DST040C33" data-open="DST040C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which was time consuming, labour intensive, high cost and, thus, inefficient. In the current study, through the highly efficient <em>in silico</em> PCR analysis, a large number of newly developed genome-wide SSR markers (92 517, 121 169 and 93 084 for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively) were found to generate one <em>in silico</em> PCR product in the assembled genomic sequences of the three sequenced <em>Brassica</em> species (TableÂ <a reveal-id="DST040TB3" data-open="DST040TB3" class="link link-reveal link-table xref-fig">3</a>). In addition, thousands of previously developed publicly available <em>Brassica</em> SSR markers were also found to generate one <em>in silico</em> PCR product in the assembled genomic sequences of these <em>Brassica</em> species (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>). More importantly, most (88.4%) of the tested SSR markers, that generated one <em>in silico</em> PCR product in the assembled genomic sequences of <em>B. napus</em>, also amplified a single clear fragment in the six representative <em>B. napus</em> cultivars/inbred lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). These results suggest that SSR markers that generate one <em>in silico</em> PCR product should be the putative single-locus markers and could be especially useful. Interestingly, the proportion (27.9%) of the newly developed genome-wide <em>Brassica</em> SSR markers (TableÂ <a reveal-id="DST040TB3" data-open="DST040TB3" class="link link-reveal link-table xref-fig">3</a>), which generated one <em>in silico</em> PCR product in the assembled genomic sequences of <em>B. napus</em>, was close to the corresponding proportion (33.8%) of the previously developed 9858 SSR marker from the GSSs/unique transcripts of <em>B. napus</em>, the BACs of <em>B. rapa</em> and the GSSs of <em>B. oleracea</em>,<sup><a reveal-id="DST040C33" data-open="DST040C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which amplified a single clear fragment in six <em>B. napus</em> inbred lines.</p><p>Also known as â€˜functionalâ€™ markers,<sup><a reveal-id="DST040C52" data-open="DST040C52" class="link link-ref link-reveal xref-bibr">52</a></sup> genic SSR markers are developed from genes and have a high transferability across related species.<sup><a reveal-id="DST040C52" data-open="DST040C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Although several studies have been conducted to develop genic SSR markers from the ESTs/unique transcripts of <em>B. rapa</em>,<sup><a reveal-id="DST040C29" data-open="DST040C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST040C31" data-open="DST040C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DST040C32" data-open="DST040C32" class="link link-ref link-reveal xref-bibr">32</a></sup><em>B. oleracea</em><sup><a reveal-id="DST040C31" data-open="DST040C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and <em>B. napus</em>,<sup><a reveal-id="DST040C30  DST040C32" data-open="DST040C30  DST040C32" class="link link-ref link-reveal xref-bibr">30â€“32</a></sup> the total number (&lt;5000) of publicly available genic SSR markers has remained limited in <em>Brassica</em> (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>). In the current study, a large number of newly developed genome-wide SSR markers (32 648, 33 228 and 58 952 for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively) were involved in the annotated genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>) and thus belonged to the genic SSR markers. Of these, only a small proportion (7.2, 6.1, 6.7% for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>, respectively) was involved in the previously developed publicly available <em>Brassica</em> SSR markers (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>). This finding suggests that most of these newly developed <em>Brassica</em> genic SSR markers could represent the new â€˜functionalâ€™ markers, which should be highly useful in evolutionary studies,<sup><a reveal-id="DST040C29" data-open="DST040C29" class="link link-ref link-reveal xref-bibr">29</a></sup> comparative mapping,<sup><a reveal-id="DST040C32" data-open="DST040C32" class="link link-ref link-reveal xref-bibr">32</a></sup> candidate gene association mapping<sup><a reveal-id="DST040C53" data-open="DST040C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and molecular breeding.</p><p>For the high transferability of SSR markers across the cultivated and wild <em>Brassica</em> species,<sup><a reveal-id="DST040C27" data-open="DST040C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST040C33" data-open="DST040C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DST040C54" data-open="DST040C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DST040C55" data-open="DST040C55" class="link link-ref link-reveal xref-bibr">55</a></sup> the developed genome-wide SSR markers from <em>B. rapa</em> (AA, 2n = 20), <em>B. oleracea</em> (CC, 2n = 18) and <em>B. napus</em> (AACC, 2n = 38) should also be useful for <em>B. nigra</em> (BB, 2n = 16), <em>B. juncea</em> (AABB, 2n = 36), <em>B. carinata</em> (BBCC, 2n = 34) and other <em>Brassica</em> species. In addition, according to the previous marker transferability research,<sup><a reveal-id="DST040C15" data-open="DST040C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST040C16" data-open="DST040C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST040C22" data-open="DST040C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST040C25" data-open="DST040C25" class="link link-ref link-reveal xref-bibr">25</a></sup> a considerable proportion of the newly developed genome-wide <em>Brassica</em> SSR markers (especially the genic SSR markers) should also be useful for the species that belong to other genera and tribes within the <em>Brassicaceae</em> family.</p><p>More importantly, we also constructed an integrative SSR marker database for <em>Brassica</em> (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>), which not only provides useful information on the newly developed genome-wide SSR markers from the sequenced <em>Brassica</em> species (currently only for <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>) but is also integrated with the previously developed publicly available <em>Brassica</em> SSR markers and the annotated genome components (mainly as genes and TEs). To the best of our knowledge, this is the first comprehensive SSR marker database for <em>Brassica</em> until now, and it should be a significant contribution to the <em>Brassic</em>a research community.</p>                            <h3 scrollto-destination=4916849 id="4916849" class="section-title" data-legacy-id=s4c>4.3. Implications for SSR marker development</h3>
<p>The numbers of clear fragment(s) amplified in the six representative <em>B. napus</em> cultivars/inbred lines for the 3974 tested SSR markers were usually equal or close to the numbers of <em>in silico</em> PCR product(s) in the assembled genomic sequences of <em>B. napus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). This finding suggests that the number of products amplified by SSR markers can be relatively accurately estimated by <em>in silico</em> PCR, which was in accordance with the previous reports in plants such as rice<sup><a reveal-id="DST040C56" data-open="DST040C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and <em>Brachypodium</em>.<sup><a reveal-id="DST040C38" data-open="DST040C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Therefore, the target microsatellite should be subjected to BLAST/<em>in silico</em> PCR analysis to estimate its copy number before SSR marker development, especially for the polyploidy species. In addition, most (88.4%) of the tested SSR markers that generate one <em>in silico</em> PCR product were also confirmed by practical PCR analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Therefore, the <em>in silico</em> identified single/low copy microsatellites should be preferential for marker development.</p><p>Replication slippage and recombination are currently two major mechanisms that are responsible for microsatellite expansion or contraction.<sup><a reveal-id="DST040C2" data-open="DST040C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST040C3" data-open="DST040C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST040C5" data-open="DST040C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST040C57" data-open="DST040C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Because of the small numbers of the tested SSR markers of specific motif length(s), the relationship between the SSR marker polymorphism level and the motif length was usually inconsistent in both the current (TableÂ <a reveal-id="DST040TB4" data-open="DST040TB4" class="link link-reveal link-table xref-fig">4</a>) and previous<sup><a reveal-id="DST040C20" data-open="DST040C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST040C24" data-open="DST040C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST040C26" data-open="DST040C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DST040C30" data-open="DST040C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a></sup> studies. However, the general trend was similar: the SSR marker polymorphism level tended to decrease as the motif length increased. This relationship is understandable because shorter motifs allow more possible replication slippage events per unit length of DNA.<sup><a reveal-id="DST040C58" data-open="DST040C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DST040C59" data-open="DST040C59" class="link link-ref link-reveal xref-bibr">59</a></sup> In addition, the SSR marker polymorphism level was positively correlated with both the motif repeat number and the repeat length in both the current (<em>r</em> = 0.74 and 0.86, respectively) and previous<sup><a reveal-id="DST040C37" data-open="DST040C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST040C47" data-open="DST040C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DST040C48" data-open="DST040C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DST040C60" data-open="DST040C60" class="link link-ref link-reveal xref-bibr">60</a></sup> studies. More importantly, the tested compound SSR markers were almost all (97.6%) polymorphic. These relationships are also understandable because more motifs, larger motif repeat number and longer repeat length give more opportunity for replication slippage.<sup><a reveal-id="DST040C2" data-open="DST040C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Therefore, microsatellites with a shorter motif length, larger motif repeat number, longer repeat length and especially the compound repeat should be preferential for marker development.</p><p>It should be noted that a considerable proportion (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>) of the newly developed genome-wide SSR markers from the sequenced <em>Brassica</em> species were involved in the so-called â€˜mobile DNA sequencesâ€™ TEs<sup><a reveal-id="DST040C61" data-open="DST040C61" class="link link-ref link-reveal xref-bibr">61</a></sup> and should thus be unstable. In addition, the SSR markers that are associated with TEs (especially retrotransposons) mostly generated tens to thousands of <em>in silico</em> PCR products (<a class="link link-uri" href="http://oilcrops.info/SSRdb">http://oilcrops.info/SSRdb</a>). Therefore, caution should be observed with respect to marker development based on microsatellites that are associated with TEs (especially retrotransposons).</p>                            <h2 scrollto-destination=4916853 id="4916853" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4916855 id="4916855" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the National Science and Technology Supporting Program (2010BAD01B02), the National Rapeseed Industry Technology System (CARS-13) and the Hubei Agricultural Science and Technology Innovation Center of China.</p>                            <h2 scrollto-destination=4916857 id="4916857" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dst040c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Renz</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">Simple sequences are ubiquitous repetitive components of eukaryotic genomes</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1984</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">4127</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequences%20are%20ubiquitous%20repetitive%20components%20of%20eukaryotic%20genomes&amp;author=D.%20Tautz&amp;author=M.%20Renz&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=4127-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.10.4127" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.10.4127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.10.4127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6328411" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>.Â </div><div class="article-title">Microsatellites: simple sequences with complex evolution</div>,Â <div class="source">Nat. Rev. Genet.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">5</div>Â (pg.Â <div class="fpage">435</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20simple%20sequences%20with%20complex%20evolution&amp;author=H.%20Ellegren&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=435-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1348" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153996" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.C.</div></div>,Â  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.B.</div></div>,Â  <div class="name"><div class="surname">Fahima</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Beiles</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>.Â </div><div class="article-title">Microsatellites: genomic distribution, putative functions and mutational mechanisms: a review</div>,Â <div class="source">Mol. Ecol.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">11</div>Â (pg.Â <div class="fpage">2453</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20genomic%20distribution%2C%20putative%20functions%20and%20mutational%20mechanisms%3A%20a%20review&amp;author=Y.C.%20Li&amp;author=A.B.%20Korol&amp;author=T.%20Fahima&amp;author=A.%20Beiles&amp;author=E.%20Nevo&amp;publication_year=2002&amp;journal=Mol.%20Ecol.&amp;volume=11&amp;pages=2453-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-294X.2002.01643.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294X.2002.01643.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-294X.2002.01643.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12453231" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.C.</div></div>,Â  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.B.</div></div>,Â  <div class="name"><div class="surname">Fahima</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>.Â </div><div class="article-title">Microsatellites within genes: structure, function, and evolution</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">21</div>Â (pg.Â <div class="fpage">991</div>-<div class="lpage">1007</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20within%20genes%3A%20structure%2C%20function%2C%20and%20evolution&amp;author=Y.C.%20Li&amp;author=A.B.%20Korol&amp;author=T.%20Fahima&amp;author=E.%20Nevo&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=991-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh073" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963101" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gemayel</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Vinces</div> <div class="given-names">M.D.</div></div>,Â  <div class="name"><div class="surname">Legendre</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Verstrepen</div> <div class="given-names">K.J.</div></div>.Â </div><div class="article-title">Variable tandem repeats accelerate evolution of coding and regulatory sequences</div>,Â <div class="source">Annu. Rev. Genet.</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">44</div>Â (pg.Â <div class="fpage">445</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variable%20tandem%20repeats%20accelerate%20evolution%20of%20coding%20and%20regulatory%20sequences&amp;author=R.%20Gemayel&amp;author=M.D.%20Vinces&amp;author=M.%20Legendre&amp;author=K.J.%20Verstrepen&amp;publication_year=2010&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=44&amp;pages=445-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-072610-155046" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-072610-155046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-072610-155046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20809801" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Olivieri</div> <div class="given-names">A.M.</div></div>.Â </div><div class="article-title">PCR-amplified microsatellites as markers in plant genetics</div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">3</div>Â (pg.Â <div class="fpage">175</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR-amplified%20microsatellites%20as%20markers%20in%20plant%20genetics&amp;author=M.%20Morgante&amp;author=A.M.%20Olivieri&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=175-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.1993.tb00020.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.1993.tb00020.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.1993.tb00020.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8401603" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">P.K.</div></div>,Â  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>.Â </div><div class="article-title">The development and use of microsatellite markers for genetic analysis and plant breeding with emphasis on bread wheat</div>,Â <div class="source">Euphytica</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">113</div>Â (pg.Â <div class="fpage">163</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20and%20use%20of%20microsatellite%20markers%20for%20genetic%20analysis%20and%20plant%20breeding%20with%20emphasis%20on%20bread%20wheat&amp;author=P.K.%20Gupta&amp;author=R.K.%20Varshney&amp;publication_year=2000&amp;journal=Euphytica&amp;volume=113&amp;pages=163-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1003910819967" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1003910819967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1003910819967"> </span></div>Â </div></div></div></div></div><div content-id="dst040c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beckmann</div> <div class="given-names">J.S.</div></div>,Â  <div class="name"><div class="surname">Soller</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">Toward a unified approach to genetic mapping of eukaryotes based on sequence tagged microsatellite sites</div>,Â <div class="source">Biotechnology</div>Â ,Â <div class="year">1990</div>,Â vol.Â <div class="volume">8</div>Â (pg.Â <div class="fpage">930</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20unified%20approach%20to%20genetic%20mapping%20of%20eukaryotes%20based%20on%20sequence%20tagged%20microsatellite%20sites&amp;author=J.S.%20Beckmann&amp;author=M.%20Soller&amp;publication_year=1990&amp;journal=Biotechnology&amp;volume=8&amp;pages=930-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1090-930" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1090-930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1090-930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1366775" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Mandakova</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Xie</div> <div class="given-names">Q.</div></div>,Â  <div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>.Â </div><div class="article-title">Deciphering the diploid ancestral genome of the Mesohexaploid <em>Brassica rapa</em>.</div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">25</div>Â (pg.Â <div class="fpage">1541</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20the%20diploid%20ancestral%20genome%20of%20the%20Mesohexaploid%20Brassica%20rapa.&amp;author=F.%20Cheng&amp;author=T.%20Mandakova&amp;author=J.%20Wu&amp;author=Q.%20Xie&amp;author=M.A.%20Lysak&amp;author=X.%20Wang&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=1541-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.110486" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.110486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.110486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23653472" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labana</div> <div class="given-names">K.S.</div></div>,Â  <div class="name"><div class="surname">Gupta</div> <div class="given-names">M.L.</div></div>.Â </div>,Â <div class="source">Breeding Oilseed Brassicas</div>Â ,Â <div class="year">1993</div><div class="publisher-loc">Berlin, Germany</div><div class="publisher-name">Springer-Verlag</div></div></div></div></div><div content-id="dst040c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="collab">U, N.</div><div class="article-title">Genome analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization</div>,Â <div class="source">Japan J. Bot.</div>Â ,Â <div class="year">1935</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">389</div>-<div class="lpage">452</div>)</div></div></div></div><div content-id="dst040c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">A.J.</div></div>,Â  <div class="name"><div class="surname">Moule</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.J.</div></div>.Â </div><div class="article-title">Efficient large-scale development of microsatellites for marker and mapping applications in <em>Brassica</em> crop species</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">108</div>Â (pg.Â <div class="fpage">1103</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20large-scale%20development%20of%20microsatellites%20for%20marker%20and%20mapping%20applications%20in%20Brassica%20crop%20species&amp;author=A.J.%20Lowe&amp;author=C.%20Moule&amp;author=M.%20Trick&amp;author=K.J.%20Edwards&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1103-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1522-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1522-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1522-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067397" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piquemal</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Cinquin</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Couton</div> <div class="given-names">F.</div></div>,Â et al.Â </div><div class="article-title">Construction of an oilseed rape (<em>Brassica napus</em> L.) genetic map with SSR markers</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">111</div>Â (pg.Â <div class="fpage">1514</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20oilseed%20rape%20(Brassica%20napus%20L.)%20genetic%20map%20with%20SSR%20markers&amp;author=J.%20Piquemal&amp;author=E.%20Cinquin&amp;author=F.%20Couton&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=1514-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0080-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0080-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0080-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16187118" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Struss</div> <div class="given-names">D.</div></div>.Â </div><div class="article-title">Microsatellite markers for genome analysis in <em>Brassica</em>. I. Development in <em>Brassica napus</em> and abundance in <em>Brassicaceae</em> species</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">102</div>Â (pg.Â <div class="fpage">689</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20for%20genome%20analysis%20in%20Brassica.%20I.%20Development%20in%20Brassica%20napus%20and%20abundance%20in%20Brassicaceae%20species&amp;author=J.%20Plieske&amp;author=D.%20Struss&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=689-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051698" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051698"> </span></div>Â </div></div></div></div></div><div content-id="dst040c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Nunome</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Kage</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">Isolation and characterization of microsatellites in <em>Brassica rapa</em> L</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">104</div>Â (pg.Â <div class="fpage">1092</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20microsatellites%20in%20Brassica%20rapa%20L&amp;author=K.%20Suwabe&amp;author=H.%20Iketani&amp;author=T.%20Nunome&amp;author=T.%20Kage&amp;author=M.%20Hirai&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=1092-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0875-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0875-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0875-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582617" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Nunome</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>.Â </div><div class="article-title">Characteristics of microsatellites in <em>Brassica rapa</em> genome and their potential utilization for comparative genomics in <em>Cruciferae</em></div>,Â <div class="source">Breeding Sci.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">54</div>Â (pg.Â <div class="fpage">85</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characteristics%20of%20microsatellites%20in%20Brassica%20rapa%20genome%20and%20their%20potential%20utilization%20for%20comparative%20genomics%20in%20Cruciferae&amp;author=K.%20Suwabe&amp;author=H.%20Iketani&amp;author=T.%20Nunome&amp;author=A.%20Ohyama&amp;author=M.%20Hirai&amp;author=H.%20Fukuoka&amp;publication_year=2004&amp;journal=Breeding%20Sci.&amp;volume=54&amp;pages=85-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.54.85" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.54.85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.54.85"> </span></div>Â </div></div></div></div></div><div content-id="dst040c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kresovich</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Szewcmcfadden</div> <div class="given-names">A.K.</div></div>,Â  <div class="name"><div class="surname">Bliek</div> <div class="given-names">S.M.</div></div>,Â  <div class="name"><div class="surname">Mcferson</div> <div class="given-names">J.R.</div></div>.Â </div><div class="article-title">Abundance and characterization of simple-sequence repeats (SSRs) isolated from a size-fractionated genomic library of <em>Brassica napus</em> L (rapeseed)</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">91</div>Â (pg.Â <div class="fpage">206</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%20and%20characterization%20of%20simple-sequence%20repeats%20(SSRs)%20isolated%20from%20a%20size-fractionated%20genomic%20library%20of%20Brassica%20napus%20L%20(rapeseed)&amp;author=S.%20Kresovich&amp;author=A.K.%20Szewcmcfadden&amp;author=S.M.%20Bliek&amp;author=J.R.%20Mcferson&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=206-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00220879" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00220879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00220879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169765" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">SzewcMcFadden</div> <div class="given-names">A.K.</div></div>,Â  <div class="name"><div class="surname">Kresovich</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Bliek</div> <div class="given-names">S.M.</div></div>,Â  <div class="name"><div class="surname">Mitchell</div> <div class="given-names">S.E.</div></div>,Â  <div class="name"><div class="surname">McFerson</div> <div class="given-names">J.R.</div></div>.Â </div><div class="article-title">Identification of polymorphic, conserved simple sequence repeats (SSRs) in cultivated <em>Brassica</em> species</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">93</div>Â (pg.Â <div class="fpage">534</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20polymorphic%2C%20conserved%20simple%20sequence%20repeats%20(SSRs)%20in%20cultivated%20Brassica%20species&amp;author=A.K.%20SzewcMcFadden&amp;author=S.%20Kresovich&amp;author=S.M.%20Bliek&amp;author=S.E.%20Mitchell&amp;author=J.R.%20McFerson&amp;publication_year=1996&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=93&amp;pages=534-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00417944" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00417944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00417944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24162344" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uzunova</div> <div class="given-names">M.I.</div></div>,Â  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>.Â </div><div class="article-title">Abundance, polymorphism and genetic mapping of microsatellites in oilseed rape (<em>Brassica napus</em> L.)</div>,Â <div class="source">Plant Breeding</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">118</div>Â (pg.Â <div class="fpage">323</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%2C%20polymorphism%20and%20genetic%20mapping%20of%20microsatellites%20in%20oilseed%20rape%20(Brassica%20napus%20L.)&amp;author=M.I.%20Uzunova&amp;author=W.%20Ecke&amp;publication_year=1999&amp;journal=Plant%20Breeding&amp;volume=118&amp;pages=323-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1439-0523.1999.00371.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1439-0523.1999.00371.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1439-0523.1999.00371.x"> </span></div>Â </div></div></div></div></div><div content-id="dst040c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Qian</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,Â et al.Â </div><div class="article-title">Construction of an integrated genetic linkage map for the A genome of <em>Brassica napus</em> using SSR markers derived from sequenced BACs in B. rapa</div>,Â <div class="source">BMC Genomics</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">11</div>Â pg.Â <div class="fpage">594</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20integrated%20genetic%20linkage%20map%20for%20the%20A%20genome%20of%20Brassica%20napus%20using%20SSR%20markers%20derived%20from%20sequenced%20BACs%20in%20B.%20rapa&amp;author=J.%20Xu&amp;author=X.%20Qian&amp;author=X.%20Wang&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=594" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-594" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20969760" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">C.P.</div></div>,Â  <div class="name"><div class="surname">Piao</div> <div class="given-names">Z.Y.</div></div>,Â  <div class="name"><div class="surname">Kang</div> <div class="given-names">T.W.</div></div>,Â et al.Â </div><div class="article-title">Genomic distribution of simple sequence repeats in <em>Brassica rapa</em>.</div>,Â <div class="source">Mol. Cells</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">349</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20distribution%20of%20simple%20sequence%20repeats%20in%20Brassica%20rapa.&amp;author=C.P.%20Hong&amp;author=Z.Y.%20Piao&amp;author=T.W.%20Kang&amp;publication_year=2007&amp;journal=Mol.%20Cells&amp;volume=23&amp;pages=349-56" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646709" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ling</div> <div class="given-names">A.E.</div></div>,Â  <div class="name"><div class="surname">Kaur</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Burgess</div> <div class="given-names">B.</div></div>,Â et al.Â </div><div class="article-title">Characterization of simple sequence repeat markers derived in silico from <em>Brassica rapa</em> bacterial artical chromosome sequences and their application in <em>Brassica napus</em></div>,Â <div class="source">Mol. Ecol. Notes</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">273</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20simple%20sequence%20repeat%20markers%20derived%20in%20silico%20from%20Brassica%20rapa%20bacterial%20artical%20chromosome%20sequences%20and%20their%20application%20in%20Brassica%20napus&amp;author=A.E.%20Ling&amp;author=J.%20Kaur&amp;author=B.%20Burgess&amp;publication_year=2007&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=7&amp;pages=273-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1471-8286.2006.01578.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1471-8286.2006.01578.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1471-8286.2006.01578.x"> </span></div>Â </div></div></div></div></div><div content-id="dst040c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">S.R.</div></div>,Â  <div class="name"><div class="surname">Teakle</div> <div class="given-names">G.R.</div></div>,Â  <div class="name"><div class="surname">Plaha</div> <div class="given-names">P.</div></div>,Â et al.Â </div><div class="article-title">The reference genetic linkage map for the multinational <em>Brassica rapa</em> genome sequencing project</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">115</div>Â (pg.Â <div class="fpage">777</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genetic%20linkage%20map%20for%20the%20multinational%20Brassica%20rapa%20genome%20sequencing%20project&amp;author=S.R.%20Choi&amp;author=G.R.%20Teakle&amp;author=P.%20Plaha&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=777-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0608-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0608-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0608-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646962" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Xia</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">Development and genetic mapping of microsatellite markers from genome survey sequences in <em>Brassica napus</em></div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">118</div>Â (pg.Â <div class="fpage">1121</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20genetic%20mapping%20of%20microsatellite%20markers%20from%20genome%20survey%20sequences%20in%20Brassica%20napus&amp;author=X.%20Cheng&amp;author=J.%20Xu&amp;author=S.%20Xia&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=1121-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-0967-8" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-0967-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-0967-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19190889" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burgess</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Mountford</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Hopkins</div> <div class="given-names">C.J.</div></div>,Â et al.Â </div><div class="article-title">Identification and characterization of simple sequence repeat (SSR) markers derived in silico from <em>Brassica oleracea</em> genome shotgun sequences</div>,Â <div class="source">Mol. Ecol. Notes</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">6</div>Â (pg.Â <div class="fpage">1191</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20simple%20sequence%20repeat%20(SSR)%20markers%20derived%20in%20silico%20from%20Brassica%20oleracea%20genome%20shotgun%20sequences&amp;author=B.%20Burgess&amp;author=H.%20Mountford&amp;author=C.J.%20Hopkins&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=1191-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2006.6.issue-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2006.6.issue-4"> </span></div>Â </div></div></div></div></div><div content-id="dst040c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.T.</div></div>,Â  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div>,Â et al.Â </div><div class="article-title">Development and genetic mapping of microsatellite markers from whole genome shotgun sequences in <em>Brassica oleracea</em></div>,Â <div class="source">Mol. Breeding</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">28</div>Â (pg.Â <div class="fpage">585</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20genetic%20mapping%20of%20microsatellite%20markers%20from%20whole%20genome%20shotgun%20sequences%20in%20Brassica%20oleracea&amp;author=H.T.%20Li&amp;author=X.%20Chen&amp;author=Y.%20Yang&amp;publication_year=2011&amp;journal=Mol.%20Breeding&amp;volume=28&amp;pages=585-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9509-y" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9509-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9509-y"> </span></div>Â </div></div></div></div></div><div content-id="dst040c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Batley</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Hopkins</div> <div class="given-names">C.J.</div></div>,Â  <div class="name"><div class="surname">Cogan</div> <div class="given-names">N.O.I.</div></div>,Â et al.Â </div><div class="article-title">Identification and characterization of simple sequence repeat markers from <em>Brassica napus</em> expressed sequences</div>,Â <div class="source">Mol. Ecol. Notes</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">886</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20simple%20sequence%20repeat%20markers%20from%20Brassica%20napus%20expressed%20sequences&amp;author=J.%20Batley&amp;author=C.J.%20Hopkins&amp;author=N.O.I.%20Cogan&amp;publication_year=2007&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=7&amp;pages=886-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2007.7.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2007.7.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2007.7.issue-5"> </span></div>Â </div></div></div></div></div><div content-id="dst040c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hopkins</div> <div class="given-names">C.J.</div></div>,Â  <div class="name"><div class="surname">Cogan</div> <div class="given-names">N.O.I.</div></div>,Â  <div class="name"><div class="surname">Hand</div> <div class="given-names">M.</div></div>,Â et al.Â </div><div class="article-title">Sixteen new simple sequence repeat markers from <em>Brassica juncea</em> expressed sequences and their cross-species amplication</div>,Â <div class="source">Mol. Ecol. Notes</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">697</div>-<div class="lpage">700</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sixteen%20new%20simple%20sequence%20repeat%20markers%20from%20Brassica%20juncea%20expressed%20sequences%20and%20their%20cross-species%20amplication&amp;author=C.J.%20Hopkins&amp;author=N.O.I.%20Cogan&amp;author=M.%20Hand&amp;publication_year=2007&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=7&amp;pages=697-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1471-8286.2007.01681.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1471-8286.2007.01681.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1471-8286.2007.01681.x"> </span></div>Â </div></div></div></div></div><div content-id="dst040c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">V.D.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,Â et al.Â </div><div class="article-title">Genic microsatellite markers in <em>Brassica rapa</em>: development, characterization, mapping, and their utility in other cultivated and wild <em>Brassica</em> relatives</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">18</div>Â (pg.Â <div class="fpage">305</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20Brassica%20rapa%3A%20development%2C%20characterization%2C%20mapping%2C%20and%20their%20utility%20in%20other%20cultivated%20and%20wild%20Brassica%20relatives&amp;author=N.%20Ramchiary&amp;author=V.D.%20Nguyen&amp;author=X.%20Li&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=305-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr017" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21768136" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.F.</div></div>,Â  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,Â et al.Â </div><div class="article-title">Abundance, marker development and genetic mapping of microsatellites from unigenes in <em>Brassica napus</em>.</div>,Â <div class="source">Mol. Breeding</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">731</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%2C%20marker%20development%20and%20genetic%20mapping%20of%20microsatellites%20from%20unigenes%20in%20Brassica%20napus.&amp;author=F.%20Wang&amp;author=X.F.%20Wang&amp;author=X.%20Chen&amp;publication_year=2012&amp;journal=Mol.%20Breeding&amp;volume=30&amp;pages=731-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9658-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9658-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9658-7"> </span></div>Â </div></div></div></div></div><div content-id="dst040c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">An</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Gao</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Fu</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Tang</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Ortegon</div> <div class="given-names">O.</div></div>.Â </div><div class="article-title">Large-scale development of functional markers in <em>Brassica</em> species</div>,Â <div class="source">Genome</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">54</div>Â (pg.Â <div class="fpage">763</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20development%20of%20functional%20markers%20in%20Brassica%20species&amp;author=Z.%20An&amp;author=C.%20Gao&amp;author=J.%20Li&amp;author=D.%20Fu&amp;author=Z.%20Tang&amp;author=O.%20Ortegon&amp;publication_year=2011&amp;journal=Genome&amp;volume=54&amp;pages=763-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g11-042" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg11-042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg11-042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21899400" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,Â  <div class="name"><div class="surname">Yadava</div> <div class="given-names">D.K.</div></div>,Â  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>.Â </div><div class="article-title">Microsatellites in <em>Brassica</em> unigenes: relative abundance, marker design, and use in comparative physical mapping and genome analysis</div>,Â <div class="source">Genome</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">53</div>Â (pg.Â <div class="fpage">55</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20Brassica%20unigenes%3A%20relative%20abundance%2C%20marker%20design%2C%20and%20use%20in%20comparative%20physical%20mapping%20and%20genome%20analysis&amp;author=S.K.%20Parida&amp;author=D.K.%20Yadava&amp;author=T.%20Mohapatra&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=55-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-084" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20130749" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Younas</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,Â et al.Â </div><div class="article-title">Development of a core set of single-locus SSR markers for allotetraploid rapeseed (<em>Brassica napus</em> L.)</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">126</div>Â (pg.Â <div class="fpage">937</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20core%20set%20of%20single-locus%20SSR%20markers%20for%20allotetraploid%20rapeseed%20(Brassica%20napus%20L.)&amp;author=H.%20Li&amp;author=M.%20Younas&amp;author=X.%20Wang&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=937-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-2027-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-2027-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-2027-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23238763" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em>.</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">43</div>Â (pg.Â <div class="fpage">1035</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20mesopolyploid%20crop%20species%20Brassica%20rapa.&amp;author=X.%20Wang&amp;author=H.%20Wang&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1035-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.919" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Deng</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Tan</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Xue</div> <div class="given-names">Q.</div></div>.Â </div><div class="article-title">A genome-wide microsatellite polymorphism database for the indica and japonica rice</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">37</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20microsatellite%20polymorphism%20database%20for%20the%20indica%20and%20japonica%20rice&amp;author=Z.%20Zhang&amp;author=Y.%20Deng&amp;author=J.%20Tan&amp;author=S.%20Hu&amp;author=J.%20Yu&amp;author=Q.%20Xue&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=37-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452422" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yin</div> <div class="given-names">T.M.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.Y.</div></div>,Â  <div class="name"><div class="surname">Gunter</div> <div class="given-names">L.E.</div></div>,Â et al.Â </div><div class="article-title">Microsatellite primer resource for <em>Populus</em> developed from the mapped sequence scaffolds of the Nisqually-1 genome</div>,Â <div class="source">New Phytol.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">181</div>Â (pg.Â <div class="fpage">498</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20primer%20resource%20for%20Populus%20developed%20from%20the%20mapped%20sequence%20scaffolds%20of%20the%20Nisqually-1%20genome&amp;author=T.M.%20Yin&amp;author=X.Y.%20Zhang&amp;author=L.E.%20Gunter&amp;publication_year=2009&amp;journal=New%20Phytol.&amp;volume=181&amp;pages=498-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2009.181.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2009.181.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2009.181.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121044" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cavagnaro</div> <div class="given-names">P.F.</div></div>,Â  <div class="name"><div class="surname">Senalik</div> <div class="given-names">D.A.</div></div>,Â  <div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide characterization of simple sequence repeats in cucumber (<em>Cucumis sativus</em> L.)</div>,Â <div class="source">BMC Genomics</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">11</div>Â pg.Â <div class="fpage">569</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20characterization%20of%20simple%20sequence%20repeats%20in%20cucumber%20(Cucumis%20sativus%20L.)&amp;author=P.F.%20Cavagnaro&amp;author=D.A.%20Senalik&amp;author=L.%20Yang&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=569" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-569" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20950470" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sonah</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Deshmukh</div> <div class="given-names">R.K.</div></div>,Â  <div class="name"><div class="surname">Sharma</div> <div class="given-names">A.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide distribution and organization of microsatellites in plants: an insight into marker development in <em>Brachypodium</em></div>,Â <div class="source">PLoS One</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">6</div>Â pg.Â <div class="fpage">e21298</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20distribution%20and%20organization%20of%20microsatellites%20in%20plants%3A%20an%20insight%20into%20marker%20development%20in%20Brachypodium&amp;author=H.%20Sonah&amp;author=R.K.%20Deshmukh&amp;author=A.%20Sharma&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e21298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021298" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21713003" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pandey</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Misra</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide development and use of microsatellite markers for large-scale genotyping applications in foxtail millet [<em>Setaria italica</em> (L.)]</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">20</div>Â (pg.Â <div class="fpage">197</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20development%20and%20use%20of%20microsatellite%20markers%20for%20large-scale%20genotyping%20applications%20in%20foxtail%20millet%20%5BSetaria%20italica%20(L.)%5D&amp;author=G.%20Pandey&amp;author=G.%20Misra&amp;author=K.%20Kumari&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=197-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382459" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thiel</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Michalek</div> <div class="given-names">W.</div></div>,Â  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,Â  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>.Â </div><div class="article-title">Exploiting EST databases for the development and characterization of gene-derived SSR-markers in barley (<em>Hordeum vulgare</em> L.)</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">106</div>Â (pg.Â <div class="fpage">411</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20EST%20databases%20for%20the%20development%20and%20characterization%20of%20gene-derived%20SSR-markers%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=T.%20Thiel&amp;author=W.%20Michalek&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=411-22" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12589540" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rotmistrovsky</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Jang</div> <div class="given-names">W.</div></div>,Â  <div class="name"><div class="surname">Schuler</div> <div class="given-names">G.D.</div></div>.Â </div><div class="article-title">A web server for performing electronic PCR</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">32</div>Â (pg.Â <div class="fpage">W108</div>-<div class="lpage">112</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20web%20server%20for%20performing%20electronic%20PCR&amp;author=K.%20Rotmistrovsky&amp;author=W.%20Jang&amp;author=G.D.%20Schuler&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=W108-112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh450" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh450" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh450"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215361" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iniguez-Luy</div> <div class="given-names">F.L.</div></div>,Â  <div class="name"><div class="surname">Voort</div> <div class="given-names">A.V.</div></div>,Â  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>.Â </div><div class="article-title">Development of a set of public SSR markers derived from genomic sequence of a rapid cycling <em>Brassica oleracea</em> L. genotype</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2008</div>,Â vol.Â <div class="volume">117</div>Â (pg.Â <div class="fpage">977</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20set%20of%20public%20SSR%20markers%20derived%20from%20genomic%20sequence%20of%20a%20rapid%20cycling%20Brassica%20oleracea%20L.%20genotype&amp;author=F.L.%20Iniguez-Luy&amp;author=A.V.%20Voort&amp;author=T.C.%20Osborn&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=117&amp;pages=977-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0837-9" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0837-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0837-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18651126" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ott</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Trautschold</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Sandhu</div> <div class="given-names">D.</div></div>.Â </div><div class="article-title">Using microsatellites to understand the physical distribution of recombination on soybean chromosomes</div>,Â <div class="source">PLoS One</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">6</div>Â pg.Â <div class="fpage">e22306</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20microsatellites%20to%20understand%20the%20physical%20distribution%20of%20recombination%20on%20soybean%20chromosomes&amp;author=A.%20Ott&amp;author=B.%20Trautschold&amp;author=D.%20Sandhu&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e22306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0022306" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0022306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0022306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21799819" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Hanafey</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Powell</div> <div class="given-names">W.</div></div>.Â </div><div class="article-title">Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">194</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20are%20preferentially%20associated%20with%20nonrepetitive%20DNA%20in%20plant%20genomes&amp;author=M.%20Morgante&amp;author=M.%20Hanafey&amp;author=W.%20Powell&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=30&amp;pages=194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng822" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11799393" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>,Â  <div class="name"><div class="surname">Koch</div> <div class="given-names">M.A.</div></div>,Â  <div class="name"><div class="surname">Pecinka</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Schubert</div> <div class="given-names">I.</div></div>.Â </div><div class="article-title">Chromosome triplication found across the tribe <em>Brassiceae</em>.</div>,Â <div class="source">Genome Res.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">15</div>Â (pg.Â <div class="fpage">516</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20triplication%20found%20across%20the%20tribe%20Brassiceae.&amp;author=M.A.%20Lysak&amp;author=M.A.%20Koch&amp;author=A.%20Pecinka&amp;author=I.%20Schubert&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=516-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3531105" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3531105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3531105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15781573" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mun</div> <div class="given-names">J.H.</div></div>,Â  <div class="name"><div class="surname">Kwon</div> <div class="given-names">S.J.</div></div>,Â  <div class="name"><div class="surname">Yang</div> <div class="given-names">T.J.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide comparative analysis of the <em>Brassica rapa</em> gene space reveals genome shrinkage and differential loss of duplicated genes after whole genome triplication</div>,Â <div class="source">Genome Biol.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">10</div>Â pg.Â <div class="fpage">R111</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20analysis%20of%20the%20Brassica%20rapa%20gene%20space%20reveals%20genome%20shrinkage%20and%20differential%20loss%20of%20duplicated%20genes%20after%20whole%20genome%20triplication&amp;author=J.H.%20Mun&amp;author=S.J.%20Kwon&amp;author=T.J.%20Yang&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-10-r111" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-10-r111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-10-r111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19821981" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Drou</div> <div class="given-names">N.</div></div>,Â et al.Â </div><div class="article-title">Comparative analysis between homoeologous genome segments of <em>Brassica napus</em> and its progenitor species reveals extensive sequence-level divergence</div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">21</div>Â (pg.Â <div class="fpage">1912</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20between%20homoeologous%20genome%20segments%20of%20Brassica%20napus%20and%20its%20progenitor%20species%20reveals%20extensive%20sequence-level%20divergence&amp;author=F.%20Cheung&amp;author=M.%20Trick&amp;author=N.%20Drou&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=1912-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.060376" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.060376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.060376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19602626" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cavagnaro</div> <div class="given-names">P.F.</div></div>,Â  <div class="name"><div class="surname">Chung</div> <div class="given-names">S.M.</div></div>,Â  <div class="name"><div class="surname">Manin</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">Microsatellite isolation and marker development in carrotâ€”genomic distribution, linkage mapping, genetic diversity analysis and marker transferability across <em>Apiaceae</em>.</div>,Â <div class="source">BMC Genomics</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">12</div>Â pg.Â <div class="fpage">386</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20isolation%20and%20marker%20development%20in%20carrot%E2%80%94genomic%20distribution%2C%20linkage%20mapping%2C%20genetic%20diversity%20analysis%20and%20marker%20transferability%20across%20Apiaceae.&amp;author=P.F.%20Cavagnaro&amp;author=S.M.%20Chung&amp;author=S.%20Manin&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-386" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-386" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-386"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21806822" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Fu</div> <div class="given-names">D.</div></div>,Â et al.Â </div><div class="article-title">Evolutionary dynamics of microsatellite distribution in plants: insight from the comparison of sequenced brassica, Arabidopsis and other angiosperm species</div>,Â <div class="source">PLoS One</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">8</div>Â pg.Â <div class="fpage">e59988</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20microsatellite%20distribution%20in%20plants%3A%20insight%20from%20the%20comparison%20of%20sequenced%20brassica%2C%20Arabidopsis%20and%20other%20angiosperm%20species&amp;author=J.%20Shi&amp;author=S.%20Huang&amp;author=D.%20Fu&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e59988" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0059988" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0059988" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0059988"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555856" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Y.W.</div></div>,Â  <div class="name"><div class="surname">Lai</div> <div class="given-names">K.N.</div></div>,Â  <div class="name"><div class="surname">Tai</div> <div class="given-names">P.Y.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>.Â </div><div class="article-title">Rates of nucleotide substitution in angiosperm mitochondrial DNA sequences and dates of divergence between <em>Brassica</em> and other angiosperm lineages</div>,Â <div class="source">J. Mol. Evol.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">48</div>Â (pg.Â <div class="fpage">597</div>-<div class="lpage">604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rates%20of%20nucleotide%20substitution%20in%20angiosperm%20mitochondrial%20DNA%20sequences%20and%20dates%20of%20divergence%20between%20Brassica%20and%20other%20angiosperm%20lineages&amp;author=Y.W.%20Yang&amp;author=K.N.%20Lai&amp;author=P.Y.%20Tai&amp;author=W.H.%20Li&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol.&amp;volume=48&amp;pages=597-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006502" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10198125" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saal</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Quiros</div> <div class="given-names">C.F.</div></div>,Â  <div class="name"><div class="surname">Struss</div> <div class="given-names">D.</div></div>.Â </div><div class="article-title">Microsatellite markers for genome analysis in <em>Brassica</em>. II. Assignment of rapeseed microsatellites to the A and C genomes and genetic mapping in <em>Brassica oleracea</em> L</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">102</div>Â (pg.Â <div class="fpage">695</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20for%20genome%20analysis%20in%20Brassica.%20II.%20Assignment%20of%20rapeseed%20microsatellites%20to%20the%20A%20and%20C%20genomes%20and%20genetic%20mapping%20in%20Brassica%20oleracea%20L&amp;author=B.%20Saal&amp;author=J.%20Plieske&amp;author=J.%20Hu&amp;author=C.F.%20Quiros&amp;author=D.%20Struss&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=695-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051699" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051699"> </span></div>Â </div></div></div></div></div><div content-id="dst040c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,Â  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>.Â </div><div class="article-title">Genic microsatellite markers in plants: features and applications</div>,Â <div class="source">Trends Biotechnol.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">48</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotechnol.&amp;volume=23&amp;pages=48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629858" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,Â  <div class="name"><div class="surname">Gong</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Pan</div> <div class="given-names">W.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>.Â </div><div class="article-title">Development and application of microsatellites in candidate genes related to wood properties in the Chinese white poplar (<em>Populus tomentosa</em> Carr.)</div>,Â <div class="source">DNA Res</div>Â ,Â <div class="year">2013</div>,Â vol.Â <div class="volume">20</div>Â (pg.Â <div class="fpage">31</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20application%20of%20microsatellites%20in%20candidate%20genes%20related%20to%20wood%20properties%20in%20the%20Chinese%20white%20poplar%20(Populus%20tomentosa%20Carr.)&amp;author=Q.%20Du&amp;author=C.%20Gong&amp;author=W.%20Pan&amp;author=D.%20Zhang&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=31-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss031" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23213110" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ge</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">T.</div></div>,Â et al.Â </div><div class="article-title">Development and linkage mapping of unigene-derived microsatellite markers in <em>Brassica rapa</em> L.</div>,Â <div class="source">Breeding Sci.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">61</div>Â (pg.Â <div class="fpage">160</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20linkage%20mapping%20of%20unigene-derived%20microsatellite%20markers%20in%20Brassica%20rapa%20L.&amp;author=Y.%20Ge&amp;author=N.%20Ramchiary&amp;author=T.%20Wang&amp;publication_year=2011&amp;journal=Breeding%20Sci.&amp;volume=61&amp;pages=160-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.61.160" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.61.160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.61.160"> </span></div>Â </div></div></div></div></div><div content-id="dst040c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marquez-Lema</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Velasco</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Perez-Vich</div> <div class="given-names">B.</div></div>.Â </div><div class="article-title">Transferability, amplification quality, and genome specificity of microsatellites in <em>Brassica carinata</em> and related species</div>,Â <div class="source">J. Appl. Genet</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">51</div>Â (pg.Â <div class="fpage">123</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transferability%2C%20amplification%20quality%2C%20and%20genome%20specificity%20of%20microsatellites%20in%20Brassica%20carinata%20and%20related%20species&amp;author=A.%20Marquez-Lema&amp;author=L.%20Velasco&amp;author=B.%20Perez-Vich&amp;publication_year=2010&amp;journal=J.%20Appl.%20Genet&amp;volume=51&amp;pages=123-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF03195720" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF03195720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF03195720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20453299" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCouch</div> <div class="given-names">S.R.</div></div>,Â  <div class="name"><div class="surname">Teytelman</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>,Â et al.Â </div><div class="article-title">Development and mapping of 2240 new SSR markers for rice (<em>Oryza sativa</em> L.)</div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">9</div>Â (pg.Â <div class="fpage">199</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20mapping%20of%202240%20new%20SSR%20markers%20for%20rice%20(Oryza%20sativa%20L.)&amp;author=S.R.%20McCouch&amp;author=L.%20Teytelman&amp;author=Y.%20Xu&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=199-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.6.199" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.6.199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.6.199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12597276" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliveira</div> <div class="given-names">E.J.</div></div>,Â  <div class="name"><div class="surname">Padua</div> <div class="given-names">J.G.</div></div>,Â  <div class="name"><div class="surname">Zucchi</div> <div class="given-names">M.I.</div></div>,Â  <div class="name"><div class="surname">Vencovsky</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Vieira</div> <div class="given-names">M.L.C.</div></div>.Â </div><div class="article-title">Origin, evolution and genome distribution of microsatellites</div>,Â <div class="source">Genet. Mol. Biol.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">29</div>Â (pg.Â <div class="fpage">294</div>-<div class="lpage">307</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%2C%20evolution%20and%20genome%20distribution%20of%20microsatellites&amp;author=E.J.%20Oliveira&amp;author=J.G.%20Padua&amp;author=M.I.%20Zucchi&amp;author=R.%20Vencovsky&amp;author=M.L.C.%20Vieira&amp;publication_year=2006&amp;journal=Genet.%20Mol.%20Biol.&amp;volume=29&amp;pages=294-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1415-47572006000200018" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1415-47572006000200018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS1415-47572006000200018"> </span></div>Â </div></div></div></div></div><div content-id="dst040c58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katti</div> <div class="given-names">M.V.</div></div>,Â  <div class="name"><div class="surname">Ranjekar</div> <div class="given-names">P.K.</div></div>,Â  <div class="name"><div class="surname">Gupta</div> <div class="given-names">V.S.</div></div>.Â </div><div class="article-title">Differential distribution of simple sequence repeats in eukaryotic genome sequences</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">18</div>Â (pg.Â <div class="fpage">1161</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20distribution%20of%20simple%20sequence%20repeats%20in%20eukaryotic%20genome%20sequences&amp;author=M.V.%20Katti&amp;author=P.K.%20Ranjekar&amp;author=V.S.%20Gupta&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=1161-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003903" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003903" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003903"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11420357" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schug</div> <div class="given-names">M.D.</div></div>,Â  <div class="name"><div class="surname">Hutter</div> <div class="given-names">C.M.</div></div>,Â  <div class="name"><div class="surname">Wetterstrand</div> <div class="given-names">K.A.</div></div>,Â  <div class="name"><div class="surname">Gaudette</div> <div class="given-names">M.S.</div></div>,Â  <div class="name"><div class="surname">Mackay</div> <div class="given-names">T.F.</div></div>,Â  <div class="name"><div class="surname">Aquadro</div> <div class="given-names">C.F.</div></div>.Â </div><div class="article-title">The mutation rates of di-, tri- and tetranucleotide repeats in <em>Drosophila melanogaster</em></div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">1998</div>,Â vol.Â <div class="volume">15</div>Â (pg.Â <div class="fpage">1751</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mutation%20rates%20of%20di-%2C%20tri-%20and%20tetranucleotide%20repeats%20in%20Drosophila%20melanogaster&amp;author=M.D.%20Schug&amp;author=C.M.%20Hutter&amp;author=K.A.%20Wetterstrand&amp;author=M.S.%20Gaudette&amp;author=T.F.%20Mackay&amp;author=C.F.%20Aquadro&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=15&amp;pages=1751-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025901" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025901" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025901"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9866209" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cloutier</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Miranda</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Ward</div> <div class="given-names">K.</div></div>,Â et al.Â </div><div class="article-title">Simple sequence repeat marker development from bacterial artificial chromosome end sequences and expressed sequence tags of flax (<em>Linum usitatissimum</em> L.)</div>,Â <div class="source">Theor. Appl. Genet.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">125</div>Â (pg.Â <div class="fpage">685</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat%20marker%20development%20from%20bacterial%20artificial%20chromosome%20end%20sequences%20and%20expressed%20sequence%20tags%20of%20flax%20(Linum%20usitatissimum%20L.)&amp;author=S.%20Cloutier&amp;author=E.%20Miranda&amp;author=K.%20Ward&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=685-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1860-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1860-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1860-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22484296" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dst040c61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst040c61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McClintock</div> <div class="given-names">B.</div></div>.Â </div><div class="article-title">Chromosome organization and genic expression</div>,Â <div class="source">Cold Spring Harb. Symp. Quant. Biol.</div>Â ,Â <div class="year">1951</div>,Â vol.Â <div class="volume">16</div>Â (pg.Â <div class="fpage">13</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20organization%20and%20genic%20expression&amp;author=B.%20McClintock&amp;publication_year=1951&amp;journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol.&amp;volume=16&amp;pages=13-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/SQB.1951.016.01.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2FSQB.1951.016.01.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2FSQB.1951.016.01.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14942727" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347430&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_8.xls?Expires=1509238921&amp;Signature=UsM4je214KGK~xnoIXn5S0GmVTtB8D5KQHazoyUqvlFYz5rTQa4ZkBnFYZhhOx1rIMa~WzMYbieRrAeQ0DNW2n5ovsJKoQv-KJEhvLKgxhSbPdDaGeAGhqS7ABYHi6Bj6TCg673Y6DHE-DbShLydHbgg~AoJk-JZR61VyMTkp~7S~s7MKKaRorO0ldI93QR7po0FOAFhg9Xiyvbc9IBzaqEv-yzvrIFKI0wG56icuTTfYmJkYYo~vdIRKB-~8EP2FAUDi-7~O1LPKJ-W3ULrywclgyEvMjJfUBesyZU9wW3x-tyLZoU-sfhFkOpVKiRs~c9QC3Nqaz~~JKlHXfQxSw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_7.xls?Expires=1509238921&amp;Signature=Ld3U5ONMtcbW0~fx9vvg20ND3VglMN4Sk~U9gQpAq0~kub~lRO2c8BZekt8ayByegJUf5fa0aB6kVfrtNgiQLlhq9azddgTz2I3TWZDxLXU908F93EHNSchO2UnZUnoNWOIaRJiZNA-WqJ-sCkccPuOeiFEJpiH-8ab6cMxoULLY8zwHCRR1em9jF7D5~buYRvDHmxdjhbip0xj3Hk3y5UNlsyWyNpXCUA7nMPhBxdsQj9OWM8aihfwgghWMf1l3J8vgO3RC8DVMjO-NDvU4Lc0GMGGS5qkW0aioIQJpMEbmEalSY4FwpRhBMv6QG2AjV~7xkZdb3-tA2B1-OidThA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_6.xls?Expires=1509238921&amp;Signature=Jq6hEg59Ae6tF~qMXf~BwtqQSl93LT66-Exc032zfdTfvd8Q7MiUFFeKZfNNAhKE2TLkdTADtld8-Mx6fGnehp529y5chV3maNmYTYJYGcJDXr4571Gf6YzUlZWIxnVIMjlb~v7S9tVAgAqqchMfBKWxOAeE-ug2MFQITniJWp1RHLmuBUmoIUAZgXAnoXnVwaZ7nqGTS3aRXsWJaUwvDa-N1qTjGsLb6pBsKDhIezWLb~EgnIzc0EpI7as8lIrLBrmKs9DHq4Z4G4x1uMZtLRrh2DqA4BatdPYPrp9Mt0Y97Kwi1JXYPXiGMwEAqsLSieDkjcm~fPydsKm6bBrVhw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_5.xls?Expires=1509238921&amp;Signature=MHDGahBVmqUbQ9jzHIQLvCXl1tC4VS132eGSUEJgbH5-7hRs9ZBducOcffluvlDuy5Fne9rED6-q5QB6l2RHz9-NPr54gGgdynCVLI3gqfgxeolzQgwQBRakOywmmq9mNqObE18XQEZ9spcZ4Lyu9HuY1sFMooBn5hyHePU~E69dXtzbBhO9QdbXEgMHDzpj3XzRnfKXHL5b3TskNTDhubW78tpsj4TpS1pJTYkzle5Zs1hQXOx51nfNlIQTkWB0Ou7tPcwRQCnTF9usrrLFgpQV3ChKvAVe-tXo8jNAbMHwS8NRO6RG-Wu1nP~8ghz4pO5mFAX1rToN4ZrGn2MyfQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_4.xls?Expires=1509238921&amp;Signature=GLsEXt7~OzIX-Y019sy4ZMnluUD0nNGe6LAAKB3FgRUiRogckmsf2Rz~I2~sNlpWlPLSyirlHu3LCscd-vktQ1dmcWR81C14jSEZxhNI-eTtaVrEzhRkicDzGPEpwRWVhKfe-r~7s-n5dGprYgtbfu~1FPMX0GQojqljEVk7IeECbuuQT3MFmkHPxA-~ofNspz8hIqD0LPQFhphQyCORR-5QSw-GCrAeFZvspD~iVXHWTxa6VzbTzKfXYQ05GtGmiQ2UNnp67w1odgINSizU0pXP7QC-kuOshuUyvILwTQj4UTuf5AUsZ1XiNhJb4~51EzXuTLlBT3pDG0M-gGJuRw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_3.xls?Expires=1509238921&amp;Signature=cG~M0Jkl4Qt1FqKwJylejOuDTKK7z5iQ5w9iN1R00V4RKbWRtThioeUQo4DMPDY7Jt~UWaIhs~DaU8BNuaLk5GTY6enZNInI7v-QT~Wt3PMstnQvlappmwUeV9Tb~qcnq99DF9sb-bD5bhS2d1xOnpxEhLS0REH~M-DO5EoD3S8KsbXXnpt31wMQZYTtXBnXOT2rAelw~z1MDRlG0LJ-StY1GSz4y2~nFnUbmusU7f~yORE0u6IkW3h~InZP0K~xxnom-Enm8kk23NgMDjiiKwMp-UZ425~STE9tzFEEkU3CBaVoDTpRN~NDqBwUgw5vIVt9XGJe5P~Bf80s8YyTKQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_2.xls?Expires=1509238921&amp;Signature=YMLJ08NNTccmm~1a8gNkJ8JcarckYGEenhkKjDzH4Y-E7K2Qw7XZSV7jLA-xN7KW0G4taEAyaPOSP299M3pzwKpMKhPiFQQtB5eE8ozlMoS6gjRqbNOVx43bB87k-wRrMKeKCAnVxJvdCYFri0Jfg3BBCI~N6CGU07SQRKUxafrTjaMUuBEYQDSUKRZaylHhLZhRSSWb6t9Pax4Y-HwX0HNjJ2BRwL3t73V-H0yPTjTU59prAy-2p51jRQvLKAqB24a5uVzGYV4fKyv9Kq-0riqTzrQD~2LIKuVmatayZeF1MwJngufmqS0Ps-EB2g-eKvqa8hKQ7Z5f0KM~SiXrIw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst040/2/dst040-dst040supp_table_1.xls?Expires=1509238921&amp;Signature=GA6eEUQ1AejoZkq4u~z04vyNfukMikQPHowg9s3kcSLXhghkkjY7NQjxVTLI7Alv-DUtD1bNKAFNClbNKmgjGo2TiW8bjTn-0OVxpvrYVM8UG-UIIZNCjgXghcEQM3Nun1m08JBpuWMfY38QJiCFK5AeFIUNLygVP96Tj1YR671ohTuZ~lNto-7UT-aLFWSBAgiIJ7jzKLxvUd50m9Opq5dPu-1cVHM8zs6i-FYA8iFTuojQ3WgCJHSOGl71C9Y~ov5DiqqS9e8Zi7sA0NZudPAkZ55BDcoGMEp4YZgK3k2embISzpJA8U--Lz5fl86wLT1S4LlazdwK1LS5ulWmjQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">102</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">36</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dst040"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347430"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f1%2f53%2f347430">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897000005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (36)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F1%2F53%2F347430" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347430'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Microsatellite Characterization and Marker Development in the Sequenced Brassica Crop Species" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f53%2f347430">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

