

<!DOCTYPE html>
<html lang="en">

<head><title>
	Deep Transcriptome Sequencing of Wild Halophyte Rice, Porteresia coarctata, Provides Novel Insights into the Salinity and Submergence Tolerance Factors | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/1/69/347526","@type":"ScholarlyArticle","name":"Deep Transcriptome Sequencing of Wild Halophyte Rice,  Porteresia coarctata , Provides Novel Insights into the Salinity and Submergence Tolerance Factors","datePublished":"2013-10-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2014-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst042","keywords":["metabolic pathways","<italic>Porteresia</italic>","salinity tolerance","submergence","transcriptome analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/1/69/347526","author":[{"name":"Garg, Rohini","affiliation":"Functional and Applied Genomics Laboratory ,  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Verma, Mohit","affiliation":"Functional and Applied Genomics Laboratory ,  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Agrawal, Shashank","affiliation":"Functional and Applied Genomics Laboratory ,  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Shankar, Rama","affiliation":"Functional and Applied Genomics Laboratory ,  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Majee, Manoj","affiliation":"Functional and Applied Genomics Laboratory ,  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Jain, Mukesh","affiliation":"Functional and Applied Genomics Laboratory ,  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"}],"description":"Porteresia coarctata is a wild relative of rice with capability of high salinity and submergence tolerance. The transcriptome analyses of Porteresia can lead to the identification of candidate genes involved in salinity and submergence tolerance. We sequenced the transcriptome of Porteresia under different conditions using Illumina platform and generated about 375 million high-quality reads. After optimized assembly, a total of 152 367 unique transcript sequences with average length of 794 bp were obtained. Many of these sequences might represent fragmented transcripts. Functional annotation revealed the presence of genes involved in diverse cellular processes and 2749 transcription factor (TF)-encoding genes in Porteresia. The differential gene expression analyses identified a total of 15 158 genes involved in salinity and/or submergence response(s). The stress-responsive members of different TF families, including MYB, bHLH, AP2-EREBP, WRKY, bZIP and NAC, were identified. We also revealed key metabolic pathways, including amino acid biosynthesis, hormone biosynthesis, secondary metabolite biosynthesis, carbohydrate metabolism and cell wall structures, involved in stress tolerance in Porteresia. The transcriptome analyses of Porteresia are expected to highlight genes/pathways involved in salinity and submergence tolerance of this halophyte species. The data can serve as a resource for unravelling the underlying mechanism and devising strategies to engineer salinity and submergence tolerance in rice.","pageStart":"69","pageEnd":"84","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1572217323&Signature=IrVDiTp6yEpob4g2dvpCJdBVJXogRDqb8D7InFMVVEO1mGowxV0JTQj3pYkWYVBCLoahVFnPD-oingzELijpxL1BW8XWPtDVoqyCKjzGYjlanp4QXr30dgK9MOfeMGKW~RRp4US7m2pyLs6yqbf0e~0kN9kYyFpH0muSXmUR-gAaSdG2G0jBpddo8g4F5JfMa4Ds2hAoQcsgkOBotXjlrYWJ6FdPm0p261NZBFFRORd7xZLWtkgIE6AfmYdjO3IkMCLJd6bDNl5TNixOzDXWmBNeBAJXoiPI5yJl9TrKwLJsQz3414mwgB92A0sgNbWbymTesgmVfHiwCoC9lGbfkg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Deep Transcriptome Sequencing of Wild Halophyte Rice,  Porteresia coarctata , Provides Novel Insights into the Salinity and Submergence Tolerance Factors","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1572217323&Signature=IrVDiTp6yEpob4g2dvpCJdBVJXogRDqb8D7InFMVVEO1mGowxV0JTQj3pYkWYVBCLoahVFnPD-oingzELijpxL1BW8XWPtDVoqyCKjzGYjlanp4QXr30dgK9MOfeMGKW~RRp4US7m2pyLs6yqbf0e~0kN9kYyFpH0muSXmUR-gAaSdG2G0jBpddo8g4F5JfMa4Ds2hAoQcsgkOBotXjlrYWJ6FdPm0p261NZBFFRORd7xZLWtkgIE6AfmYdjO3IkMCLJd6bDNl5TNixOzDXWmBNeBAJXoiPI5yJl9TrKwLJsQz3414mwgB92A0sgNbWbymTesgmVfHiwCoC9lGbfkg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Deep Transcriptome Sequencing of Wild Halophyte Rice,  Porteresia coarctata , Provides Novel Insights into the Salinity and Submergence Tolerance Factors | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Porteresia coarctata is a wild relative of rice with capability of high salinity and submergence tolerance. The transcriptome analyses of Porteresia can lead to the identification of candidate genes involved in salinity and submergence tolerance. We sequenced the transcriptome of Porteresia under different conditions using Illumina platform and generated about 375 million high-quality reads. After optimized assembly, a total of 152 367 unique transcript sequences with average length of 794 bp were obtained. Many of these sequences might represent fragmented transcripts. Functional annotation revealed the presence of genes involved in diverse cellular processes and 2749 transcription factor (TF)-encoding genes in Porteresia. The differential gene expression analyses identified a total of 15 158 genes involved in salinity and/or submergence response(s). The stress-responsive members of different TF families, including MYB, bHLH, AP2-EREBP, WRKY, bZIP and NAC, were identified. We also revealed key metabolic pathways, including amino acid biosynthesis, hormone biosynthesis, secondary metabolite biosynthesis, carbohydrate metabolism and cell wall structures, involved in stress tolerance in Porteresia. The transcriptome analyses of Porteresia are expected to highlight genes/pathways involved in salinity and submergence tolerance of this halophyte species. The data can serve as a resource for unravelling the underlying mechanism and devising strategies to engineer salinity and submergence tolerance in rice." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/1/69/347526" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217323&amp;Signature=Xel9HuMEdzSEQ7ea1i7GvglRrEUmxbTQR91YUjiZ5SP~fUs-ttKKT44unEw5uKBQs8ZvuDgiq5SUfANJKwo1qSAx9YSaHDEGugz6MzCc9GCiSETV~9sX5EfvGfHUWP9VplwAMvj62qGgfwUwTqNE-xGuy1mQcrDa7DUtPDzD~BJfjT5sgghGUlIznotMdBbT59KNjrxVS1H7w5OpzojjI4vqA7ivVm1LJqMKT6p81YSGmb7wXe6UBjc0ddo56wLqKKGXKs25hVsZf~dwSRsIT7LehADaImx-KhnsdBuv4wCbtib8YDxwl7trbFpJ208V4qAipOHDW4YKED7cOT21bg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2013-10-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Garg, Rohini" /><meta name="citation_author_institution" content="Functional and Applied Genomics Laboratory, National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Verma, Mohit" /><meta name="citation_author_institution" content="Functional and Applied Genomics Laboratory, National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Agrawal, Shashank" /><meta name="citation_author_institution" content="Functional and Applied Genomics Laboratory, National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Shankar, Rama" /><meta name="citation_author_institution" content="Functional and Applied Genomics Laboratory, National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Majee, Manoj" /><meta name="citation_author_institution" content="Functional and Applied Genomics Laboratory, National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Jain, Mukesh" /><meta name="citation_author_institution" content="Functional and Applied Genomics Laboratory, National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_title" content="Deep Transcriptome Sequencing of Wild Halophyte Rice, Porteresia coarctata, Provides Novel Insights into the Salinity and Submergence Tolerance Factors" /><meta name="citation_firstpage" content="69" /><meta name="citation_lastpage" content="84" /><meta name="citation_doi" content="10.1093/dnares/dst042" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2014/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome-wide transcriptional analysis of salinity stressed japonica and indica rice genotypes during panicle initiation stage; Plant Mol. Biol.;  citation_year=2007;  citation_volume=63;  citation_pages=609-23; " /><meta name="citation_reference" content="citation_title=Transcriptome map for seedling stage specific salinity stress response indicates a specific set of genes as candidate for saline tolerance in Oryza sativa L.; Funct. Integr. Genomics;  citation_year=2009;  citation_volume=9;  citation_pages=109-23; " /><meta name="citation_reference" content="citation_title=Root-specific transcript profiling of contrasting rice genotypes in response to salinity stress; Mol. Plant;  citation_year=2011;  citation_volume=4;  citation_pages=25-41; " /><meta name="citation_reference" content="citation_title=Massive parallel sequencing of mRNA in identification of unannotated salinity stress-inducible transcripts in rice (Oryza sativa L.); BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=683" /><meta name="citation_reference" content="citation_title=Flooding tolerance in halophytes; New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=964-74; " /><meta name="citation_reference" content="citation_title=Salinity tolerance in halophytes; New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=945-63; " /><meta name="citation_reference" content="citation_title=Abiotic stress and plant genome evolution. Search for new models; Plant Physiol.;  citation_year=2005;  citation_volume=138;  citation_pages=127-30; " /><meta name="citation_reference" content="citation_title=Porteresia coarctata (Roxb.) Tateoka: a wild rice a potential model for studying salt-stress biology in rice; Plant Cell Environ.;  citation_year=2010;  citation_volume=33;  citation_pages=526-42; " /><meta name="citation_reference" content="citation_title=Mechanisms of salt tolerance in wild rice (Oryza coarctata Roxb.); Plant Soil;  citation_year=1986;  citation_volume=92;  citation_pages=399-404; " /><meta name="citation_reference" content="citation_title=Molecular and physiological responses to abscisic acid and salts in roots of salt-sensitive and salt-tolerant Indica rice varieties; Plant Physiol.;  citation_year=1995;  citation_volume=107;  citation_pages=177-86; " /><meta name="citation_reference" content="citation_title=Salinity tolerance, Na+ exclusion and allele mining of HKT1;5 in Oryza sativa and O. glaberrima: many sources, many genes, one mechanism?; BMC Plant Biol.;  citation_year=2013;  citation_volume=13;  citation_pages=32" /><meta name="citation_reference" content="citation_title=Tolerance of anaerobic conditions caused by flooding during germination and early growth in rice (Oryza sativa L.); Front. Plant Sci.;  citation_year=2013;  citation_volume=4;  citation_pages=269" /><meta name="citation_reference" content="citation_title=Allele mining for stress tolerance genes in Oryza species and related germplasm; Mol. Biotechnol.;  citation_year=2004;  citation_volume=27;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=Salt cress. A halophyte and cryophyte Arabidopsis relative model system and its applicability to molecular genetic analyses of growth and development of extremophiles; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1718-37; " /><meta name="citation_reference" content="citation_title=Genome structures and halophyte-specific gene expression of the extremophile Thellungiella parvula in comparison with Thellungiella salsuginea (Thellungiella halophila) and Arabidopsis; Plant Physiol.;  citation_year=2010;  citation_volume=154;  citation_pages=1040-52; " /><meta name="citation_reference" content="citation_title=Insights into salt tolerance from the genome of Thellungiella salsuginea; Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=12219-24; " /><meta name="citation_reference" content="citation_title=A novel salt-tolerant L-myo-inositol-1-phosphate synthase from Porteresia coarctata (Roxb.) Tateoka, a halophytic wild rice molecular cloning, bacterial overexpression, characterization, and functional introgression into tobacco-conferring salt tolerance phenotype; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=28539-52; " /><meta name="citation_reference" content="citation_title=Inositol methyl transferase from a halophytic wild rice, Porteresia coarctata Roxb. (Tateoka) regulation of pinitol synthesis under abiotic stress; Plant Cell Environ.;  citation_year=2008;  citation_volume=31;  citation_pages=1442-59; " /><meta name="citation_reference" content="citation_title=Applications of new sequencing technologies for transcriptome analysis; Annu. Rev. Genomics Human Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=135-51; " /><meta name="citation_reference" content="citation_title=A next generation approach to the characterization of a non-model plant transcriptome; Curr. Sci.;  citation_year=2011;  citation_volume=101;  citation_pages=1435-9; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of wild chickpea as a rich resource for marker development; Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=690-702; " /><meta name="citation_reference" content="citation_title=NGS QC Toolkit: a toolkit for quality control of next generation sequencing data; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30619" /><meta name="citation_reference" content="citation_title=Gene discovery and tissue-specific transcriptome analysis in chickpea with massively parallel pyrosequencing and web resource development; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=1661-78; " /><meta name="citation_reference" content="citation_title=Improvement of the Oryza sativa Nipponbare reference genome using next generation sequence and optical map data; Rice;  citation_year=2013;  citation_volume=6;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Differential expression analysis for sequence count data; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R106" /><meta name="citation_reference" content="citation_title=BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3448-9; " /><meta name="citation_reference" content="citation_title=Gramene a bird's eye view of cereal genomes; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D717-723; " /><meta name="citation_reference" content="citation_title=Validation of internal control genes for quantitative gene expression studies in chickpea (Cicer arietinum L.); Biochem. Biophys. Res. Commun.;  citation_year=2010;  citation_volume=396;  citation_pages=283-8; " /><meta name="citation_reference" content="citation_title=Validation of housekeeping genes as internal control for studying gene expression in rice by quantitative real-time PCR; Biochem. Biophys. Res. Commun.;  citation_year=2006;  citation_volume=345;  citation_pages=646-51; " /><meta name="citation_reference" content="citation_title=Comparative analysis of kabuli chickpea transcriptome with desi and wild chickpea provides a rich resource for development of functional markers; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e52443" /><meta name="citation_reference" content="citation_title=RNA sequencing advances, challenges and opportunities; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=87-98; " /><meta name="citation_reference" content="citation_title=Next-generation sequencing technologies for gene expression profiling in plants; Brief Funct. Genomics;  citation_year=2012;  citation_volume=11;  citation_pages=63-70; " /><meta name="citation_reference" content="citation_title=Salinity stress adaptation competence in the extremophile Thellungiella halophila in comparison with its relative Arabidopsis thaliana; Plant J.;  citation_year=2005;  citation_volume=44;  citation_pages=826-39; " /><meta name="citation_reference" content="citation_title=Mechanisms of salinity tolerance; Annu. Rev. Plant Biol.;  citation_year=2008;  citation_volume=59;  citation_pages=651-81; " /><meta name="citation_reference" content="citation_title=Molecular characterization of the submergence response of the Arabidopsis thaliana ecotype Columbia; New Phytol.;  citation_year=2011;  citation_volume=190;  citation_pages=457-71; " /><meta name="citation_reference" content="citation_title=How plants cope with complete submergence; New Phytol.;  citation_year=2006;  citation_volume=170;  citation_pages=213-26; " /><meta name="citation_reference" content="citation_title=Natural variation of submergence tolerance among Arabidopsis thaliana accessions; New Phytol.;  citation_year=2011;  citation_volume=190;  citation_pages=299-310; " /><meta name="citation_reference" content="citation_title=Transcriptional profiling implicates novel interactions between abiotic stress and hormonal responses in Thellungiella, a close relative of Arabidopsis; Plant Physiol.;  citation_year=2006;  citation_volume=140;  citation_pages=1437-50; " /><meta name="citation_reference" content="citation_title=Cross-kingdom comparison of transcriptomic adjustments to low-oxygen stress highlights conserved and plant-specific responses; Plant Physiol.;  citation_year=2010;  citation_volume=152;  citation_pages=1484-500; " /><meta name="citation_reference" content="citation_title=The submergence tolerance regulator Sub1A mediates stress-responsive expression of AP2/ERF transcription factors; Plant Physiol.;  citation_year=2010;  citation_volume=152;  citation_pages=1674-92; " /><meta name="citation_reference" content="citation_title=A variable cluster of ethylene response factor-like genes regulates metabolic and developmental acclimation responses to submergence in rice; Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=2021-34; " /><meta name="citation_reference" content="citation_title=Transcript profiling of the anoxic rice coleoptile; Plant Physiol.;  citation_year=2007;  citation_volume=144;  citation_pages=218-31; " /><meta name="citation_reference" content="citation_title=Expression of alpha-expansin genes in young seedlings of rice (Oryza sativa L.); Planta;  citation_year=2000;  citation_volume=211;  citation_pages=467-73; " /><meta name="citation_reference" content="citation_title=Comparative transcriptomic profiling of a salt-tolerant wild tomato species and a salt-sensitive tomato cultivar; Plant Cell Physiol.;  citation_year=2010;  citation_volume=51;  citation_pages=997-1006; " /><meta name="citation_reference" content="citation_title=Distinct evolutionary patterns of Oryza glaberrima deciphered by genome sequencing and comparative analysis; Plant J.;  citation_year=2011;  citation_volume=66;  citation_pages=796-805; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of Oryza brachyantha reveals mechanisms underlying Oryza genome evolution; Nat Commun.;  citation_year=2013;  citation_volume=4;  citation_pages=1595" /><meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in cellular responses and tolerance to dehydration and cold stresses; Annu. Rev. Plant Biol.;  citation_year=2006;  citation_volume=57;  citation_pages=781-803; " /><meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in response to abiotic stresses in Arabidopsis and grasses; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=88-95; " /><meta name="citation_reference" content="citation_title=Transcription factor mediated abiotic stress signaling in rice; Plant Stress;  citation_year=2013;  citation_volume=7;  citation_pages=16-25; " /><meta name="citation_reference" content="citation_title=Evolutionary analysis of the Sub1 gene cluster that confers submergence tolerance to domesticated rice; Ann. Bot.;  citation_year=2009;  citation_volume=103;  citation_pages=143-50; " /><meta name="citation_reference" content="citation_title=Sub1A is an ethylene-response-factor-like gene that confers submergence tolerance to rice; Nature;  citation_year=2006;  citation_volume=442;  citation_pages=705-8; " /><meta name="citation_reference" content="citation_title=SUB1A-dependent and -independent mechanisms are involved in the flooding tolerance of wild rice species; Plant J.;  citation_year=2012;  citation_volume=72;  citation_pages=282-93; " /><meta name="citation_reference" content="citation_title=Production of coumaroylserotonin and feruloylserotonin in transgenic rice expressing pepper hydroxycinnamoyl-coenzyme aserotonin N-(hydroxycinnamoyl) transferase; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=346-56; " /><meta name="citation_reference" content="citation_title=Jasmonates an update on biosynthesis, signal transduction and action in plant stress response, growth and development; Ann. Bot.;  citation_year=2007;  citation_volume=100;  citation_pages=681-97; " /><meta name="citation_reference" content="citation_title=A functional genomics approach toward the understanding of secondary metabolism in plant cells; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=8595-600; " /><meta name="citation_reference" content="citation_title=ORCA3, a jasmonate-responsive transcriptional regulator of plant primary and secondary metabolism; Science;  citation_year=2000;  citation_volume=289;  citation_pages=295-7; " /><meta name="citation_reference" content="citation_title=Coordinated activation of metabolic pathways for antioxidants and defence compounds by jasmonates and their roles in stress tolerance in Arabidopsis; Plant J.;  citation_year=2005;  citation_volume=44;  citation_pages=653-68; " /><meta name="citation_reference" content="citation_title=The role of methionine recycling for ethylene synthesis in Arabidopsis; Plant J.;  citation_year=2007;  citation_volume=49;  citation_pages=238-49; " /><meta name="citation_reference" content="citation_title=The submergence tolerance gene SUB1A delays leaf senescence under prolonged darkness through hormonal regulation in rice; Plant Physiol.;  citation_year=2012;  citation_volume=160;  citation_pages=1795-807; " /><meta name="citation_reference" content="citation_title=Regulation of abscisic acid biosynthesis; Plant Physiol.;  citation_year=2003;  citation_volume=133;  citation_pages=29-36; " /><meta name="citation_reference" content="citation_title=Before and beyond ABA upstream sensing and internal signals that determine ABA accumulation and response under abiotic stress; Biochem. Soc. Trans.;  citation_year=2005;  citation_volume=33;  citation_pages=375-9; " /><meta name="citation_reference" content="citation_title=Mechanisms of high salinity tolerance in plants; Methods Enzymol.;  citation_year=2007;  citation_volume=428;  citation_pages=419-38; " /><meta name="citation_reference" content="citation_title=Abscisic acid emergence of a core signaling network; Annu. Rev. Plant Biol.;  citation_year=2010;  citation_volume=61;  citation_pages=651-79; " /><meta name="citation_reference" content="citation_title=Ethylene promotes submergence-induced expression of OsABA8ox1, a gene that encodes ABA 8′-hydroxylase in rice; Plant Cell Physiol.;  citation_year=2007;  citation_volume=4;  citation_pages=287-98; " /><meta name="citation_reference" content="citation_title=Salinity induces accumulation of soluble sugars and alters the activity of sugar metabolizing enzymes in rice plants; Biol. Plant.;  citation_year=1999;  citation_volume=42;  citation_pages=233-9; " /><meta name="citation_reference" content="citation_title=Effect of various abiotic stresses on the growth, soluble sugars and water relations of sorghum seedlings grown in light and darkness; Bulg. J. Plant Physiol.;  citation_year=2001;  citation_volume=27;  citation_pages=72-84; " /><meta name="citation_reference" content="citation_title=Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray; Plant J.;  citation_year=2002;  citation_volume=31;  citation_pages=279-92; " /><meta name="citation_reference" content="citation_title=Sugar signalling and gene expression in relation to carbohydrate metabolism under abiotic stresses in plants; J. Biosci.;  citation_year=2005;  citation_volume=30;  citation_pages=761-76; " /><meta name="citation_reference" content="citation_title=Soluble sugars—metabolism, sensing and abiotic stress a complex network in the life of plants; Plant Signal. Behav.;  citation_year=2009;  citation_volume=4;  citation_pages=388-93; " /><meta name="citation_reference" content="citation_title=Plant biology genetics of high-rise rice; Nature;  citation_year=2009;  citation_volume=460;  citation_pages=959-60; " /><meta name="citation_reference" content="citation_title=Trehalose accumulation in rice plants confers high tolerance levels to different abiotic stresses; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=15898-903; " /><meta name="citation_reference" content="citation_title=Accumulation of trehalose increases soluble sugar contents in rice plants conferring tolerance to drought and salt stress; Plant Biotechnol. Rep.;  citation_year=2012;  citation_volume=6;  citation_pages=89-96; " /><meta name="citation_reference" content="citation_title=Pathways and genetic determinants for cell wall-based osmotic stress tolerance in the Arabidopsis thaliana root system; citation_text=Ch. 2; citation_publisher=Wiley-Blackwell, Oxford, UK; Genes for plant abiotic stress;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Disruption of the cellulose synthase gene, AtCesA8/IRX1, enhances drought and osmotic stress tolerance in Arabidopsis; Plant J.;  citation_year=2005;  citation_volume=43;  citation_pages=273-83; " /><meta name="citation_reference" content="citation_title=A cellulose synthase-like protein is required for osmotic stress tolerance in Arabidopsis; Plant J.;  citation_year=2010;  citation_volume=63;  citation_pages=128-40; " /><meta name="citation_reference" content="citation_title=The plant cuticle is required for osmotic stress regulation of abscisic acid biosynthesis and osmotic stress tolerance in Arabidopsis; Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=1971-84; " /><meta name="citation_reference" content="citation_title=Root suberin forms an extracellular barrier that affects water relations and mineral nutrition in Arabidopsis; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000492" /><meta name="citation_reference" content="citation_title=Suberin goes genomics use of a short living plant to investigate a long lasting polymer; Front. Plant Sci.;  citation_year=2012;  citation_volume=3;  citation_pages=4" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/1/69/995224/dst042.pdf" /><meta name="description" content="Abstract.  Porteresia coarctata is a wild relative of rice with capability of high salinity and submergence tolerance. The transcriptome analyses of Porteresia " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/1/69/347526" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/1/69/347526" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dst042';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347526" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF9qoBPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYXJnLCBSb2hpbmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbmFsIGFuZCBBcHBsaWVkIEdlbm9taWNzIExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWZXJtYSwgTW9oaXQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbmFsIGFuZCBBcHBsaWVkIEdlbm9taWNzIExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBZ3Jhd2FsLCBTaGFzaGFuayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZ1bmN0aW9uYWwgYW5kIEFwcGxpZWQgR2Vub21pY3MgTGFib3JhdG9yeSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoYW5rYXIsIFJhbWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbmFsIGFuZCBBcHBsaWVkIEdlbm9taWNzIExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWplZSwgTWFub2oiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbmFsIGFuZCBBcHBsaWVkIEdlbm9taWNzIExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKYWluLCBNdWtlc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbmFsIGFuZCBBcHBsaWVkIEdlbm9taWNzIExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRlZXAgVHJhbnNjcmlwdG9tZSBTZXF1ZW5jaW5nIG9mIFdpbGQgSGFsb3BoeXRlIFJpY2UsIFBvcnRlcmVzaWEgY29hcmN0YXRhLCBQcm92aWRlcyBOb3ZlbCBJbnNpZ2h0cyBpbnRvIHRoZSBTYWxpbml0eSBhbmQgU3VibWVyZ2VuY2UgVG9sZXJhbmNlIEZhY3RvcnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBvZiBzYWxpbml0eSBzdHJlc3NlZCBqYXBvbmljYSBhbmQgaW5kaWNhIHJpY2UgZ2Vub3R5cGVzIGR1cmluZyBwYW5pY2xlIGluaXRpYXRpb24gc3RhZ2U7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz02MDktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgbWFwIGZvciBzZWVkbGluZyBzdGFnZSBzcGVjaWZpYyBzYWxpbml0eSBzdHJlc3MgcmVzcG9uc2UgaW5kaWNhdGVzIGEgc3BlY2lmaWMgc2V0IG9mIGdlbmVzIGFzIGNhbmRpZGF0ZSBmb3Igc2FsaW5lIHRvbGVyYW5jZSBpbiBPcnl6YSBzYXRpdmEgTC47IEZ1bmN0LiBJbnRlZ3IuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMDktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvb3Qtc3BlY2lmaWMgdHJhbnNjcmlwdCBwcm9maWxpbmcgb2YgY29udHJhc3RpbmcgcmljZSBnZW5vdHlwZXMgaW4gcmVzcG9uc2UgdG8gc2FsaW5pdHkgc3RyZXNzOyBNb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yNS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFzc2l2ZSBwYXJhbGxlbCBzZXF1ZW5jaW5nIG9mIG1STkEgaW4gaWRlbnRpZmljYXRpb24gb2YgdW5hbm5vdGF0ZWQgc2FsaW5pdHkgc3RyZXNzLWluZHVjaWJsZSB0cmFuc2NyaXB0cyBpbiByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz02ODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG9vZGluZyB0b2xlcmFuY2UgaW4gaGFsb3BoeXRlczsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9OTY0LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWxpbml0eSB0b2xlcmFuY2UgaW4gaGFsb3BoeXRlczsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYmlvdGljIHN0cmVzcyBhbmQgcGxhbnQgZ2Vub21lIGV2b2x1dGlvbi4gU2VhcmNoIGZvciBuZXcgbW9kZWxzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzg7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcnRlcmVzaWEgY29hcmN0YXRhIChSb3hiLikgVGF0ZW9rYTogYSB3aWxkIHJpY2UgYSBwb3RlbnRpYWwgbW9kZWwgZm9yIHN0dWR5aW5nIHNhbHQtc3RyZXNzIGJpb2xvZ3kgaW4gcmljZTsgUGxhbnQgQ2VsbCBFbnZpcm9uLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTUyNi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBzYWx0IHRvbGVyYW5jZSBpbiB3aWxkIHJpY2UgKE9yeXphIGNvYXJjdGF0YSBSb3hiLik7IFBsYW50IFNvaWw7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz0zOTktNDA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYW5kIHBoeXNpb2xvZ2ljYWwgcmVzcG9uc2VzIHRvIGFic2Npc2ljIGFjaWQgYW5kIHNhbHRzIGluIHJvb3RzIG9mIHNhbHQtc2Vuc2l0aXZlIGFuZCBzYWx0LXRvbGVyYW50IEluZGljYSByaWNlIHZhcmlldGllczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTc3LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWxpbml0eSB0b2xlcmFuY2UsIE5hKyBleGNsdXNpb24gYW5kIGFsbGVsZSBtaW5pbmcgb2YgSEtUMTs1IGluIE9yeXphIHNhdGl2YSBhbmQgTy4gZ2xhYmVycmltYTogbWFueSBzb3VyY2VzLCBtYW55IGdlbmVzLCBvbmUgbWVjaGFuaXNtPzsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub2xlcmFuY2Ugb2YgYW5hZXJvYmljIGNvbmRpdGlvbnMgY2F1c2VkIGJ5IGZsb29kaW5nIGR1cmluZyBnZXJtaW5hdGlvbiBhbmQgZWFybHkgZ3Jvd3RoIGluIHJpY2UgKE9yeXphIHNhdGl2YSBMLik7IEZyb250LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbGxlbGUgbWluaW5nIGZvciBzdHJlc3MgdG9sZXJhbmNlIGdlbmVzIGluIE9yeXphIHNwZWNpZXMgYW5kIHJlbGF0ZWQgZ2VybXBsYXNtOyBNb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTAxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbHQgY3Jlc3MuIEEgaGFsb3BoeXRlIGFuZCBjcnlvcGh5dGUgQXJhYmlkb3BzaXMgcmVsYXRpdmUgbW9kZWwgc3lzdGVtIGFuZCBpdHMgYXBwbGljYWJpbGl0eSB0byBtb2xlY3VsYXIgZ2VuZXRpYyBhbmFseXNlcyBvZiBncm93dGggYW5kIGRldmVsb3BtZW50IG9mIGV4dHJlbW9waGlsZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTE3MTgtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzdHJ1Y3R1cmVzIGFuZCBoYWxvcGh5dGUtc3BlY2lmaWMgZ2VuZSBleHByZXNzaW9uIG9mIHRoZSBleHRyZW1vcGhpbGUgVGhlbGx1bmdpZWxsYSBwYXJ2dWxhIGluIGNvbXBhcmlzb24gd2l0aCBUaGVsbHVuZ2llbGxhIHNhbHN1Z2luZWEgKFRoZWxsdW5naWVsbGEgaGFsb3BoaWxhKSBhbmQgQXJhYmlkb3BzaXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NDsgIGNpdGF0aW9uX3BhZ2VzPTEwNDAtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGludG8gc2FsdCB0b2xlcmFuY2UgZnJvbSB0aGUgZ2Vub21lIG9mIFRoZWxsdW5naWVsbGEgc2Fsc3VnaW5lYTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTIyMTktMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgc2FsdC10b2xlcmFudCBMLW15by1pbm9zaXRvbC0xLXBob3NwaGF0ZSBzeW50aGFzZSBmcm9tIFBvcnRlcmVzaWEgY29hcmN0YXRhIChSb3hiLikgVGF0ZW9rYSwgYSBoYWxvcGh5dGljIHdpbGQgcmljZSBtb2xlY3VsYXIgY2xvbmluZywgYmFjdGVyaWFsIG92ZXJleHByZXNzaW9uLCBjaGFyYWN0ZXJpemF0aW9uLCBhbmQgZnVuY3Rpb25hbCBpbnRyb2dyZXNzaW9uIGludG8gdG9iYWNjby1jb25mZXJyaW5nIHNhbHQgdG9sZXJhbmNlIHBoZW5vdHlwZTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9Mjg1MzktNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlub3NpdG9sIG1ldGh5bCB0cmFuc2ZlcmFzZSBmcm9tIGEgaGFsb3BoeXRpYyB3aWxkIHJpY2UsIFBvcnRlcmVzaWEgY29hcmN0YXRhIFJveGIuIChUYXRlb2thKSByZWd1bGF0aW9uIG9mIHBpbml0b2wgc3ludGhlc2lzIHVuZGVyIGFiaW90aWMgc3RyZXNzOyBQbGFudCBDZWxsIEVudmlyb24uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTQ0Mi01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwbGljYXRpb25zIG9mIG5ldyBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llcyBmb3IgdHJhbnNjcmlwdG9tZSBhbmFseXNpczsgQW5udS4gUmV2LiBHZW5vbWljcyBIdW1hbiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMzUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV4dCBnZW5lcmF0aW9uIGFwcHJvYWNoIHRvIHRoZSBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgbm9uLW1vZGVsIHBsYW50IHRyYW5zY3JpcHRvbWU7IEN1cnIuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTQzNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgb2Ygd2lsZCBjaGlja3BlYSBhcyBhIHJpY2ggcmVzb3VyY2UgZm9yIG1hcmtlciBkZXZlbG9wbWVudDsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02OTAtNzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OR1MgUUMgVG9vbGtpdDogYSB0b29sa2l0IGZvciBxdWFsaXR5IGNvbnRyb2wgb2YgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgZGF0YTsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMDYxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZGlzY292ZXJ5IGFuZCB0aXNzdWUtc3BlY2lmaWMgdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBpbiBjaGlja3BlYSB3aXRoIG1hc3NpdmVseSBwYXJhbGxlbCBweXJvc2VxdWVuY2luZyBhbmQgd2ViIHJlc291cmNlIGRldmVsb3BtZW50OyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0xNjYxLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlbWVudCBvZiB0aGUgT3J5emEgc2F0aXZhIE5pcHBvbmJhcmUgcmVmZXJlbmNlIGdlbm9tZSB1c2luZyBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2UgYW5kIG9wdGljYWwgbWFwIGRhdGE7IFJpY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBhbmFseXNpcyBmb3Igc2VxdWVuY2UgY291bnQgZGF0YTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9UjEwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpTkdPOiBhIEN5dG9zY2FwZSBwbHVnaW4gdG8gYXNzZXNzIG92ZXJyZXByZXNlbnRhdGlvbiBvZiBnZW5lIG9udG9sb2d5IGNhdGVnb3JpZXMgaW4gYmlvbG9naWNhbCBuZXR3b3JrczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNDQ4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyYW1lbmUgYSBiaXJkJ3MgZXllIHZpZXcgb2YgY2VyZWFsIGdlbm9tZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPUQ3MTctNzIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYWxpZGF0aW9uIG9mIGludGVybmFsIGNvbnRyb2wgZ2VuZXMgZm9yIHF1YW50aXRhdGl2ZSBnZW5lIGV4cHJlc3Npb24gc3R1ZGllcyBpbiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM5NjsgIGNpdGF0aW9uX3BhZ2VzPTI4My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYWxpZGF0aW9uIG9mIGhvdXNla2VlcGluZyBnZW5lcyBhcyBpbnRlcm5hbCBjb250cm9sIGZvciBzdHVkeWluZyBnZW5lIGV4cHJlc3Npb24gaW4gcmljZSBieSBxdWFudGl0YXRpdmUgcmVhbC10aW1lIFBDUjsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0NTsgIGNpdGF0aW9uX3BhZ2VzPTY0Ni01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2Yga2FidWxpIGNoaWNrcGVhIHRyYW5zY3JpcHRvbWUgd2l0aCBkZXNpIGFuZCB3aWxkIGNoaWNrcGVhIHByb3ZpZGVzIGEgcmljaCByZXNvdXJjZSBmb3IgZGV2ZWxvcG1lbnQgb2YgZnVuY3Rpb25hbCBtYXJrZXJzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTUyNDQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIHNlcXVlbmNpbmcgYWR2YW5jZXMsIGNoYWxsZW5nZXMgYW5kIG9wcG9ydHVuaXRpZXM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz04Ny05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzIGZvciBnZW5lIGV4cHJlc3Npb24gcHJvZmlsaW5nIGluIHBsYW50czsgQnJpZWYgRnVuY3QuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbGluaXR5IHN0cmVzcyBhZGFwdGF0aW9uIGNvbXBldGVuY2UgaW4gdGhlIGV4dHJlbW9waGlsZSBUaGVsbHVuZ2llbGxhIGhhbG9waGlsYSBpbiBjb21wYXJpc29uIHdpdGggaXRzIHJlbGF0aXZlIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTgyNi0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBzYWxpbml0eSB0b2xlcmFuY2U7IEFubnUuIFJldi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz02NTEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBzdWJtZXJnZW5jZSByZXNwb25zZSBvZiB0aGUgQXJhYmlkb3BzaXMgdGhhbGlhbmEgZWNvdHlwZSBDb2x1bWJpYTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTkwOyAgY2l0YXRpb25fcGFnZXM9NDU3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgcGxhbnRzIGNvcGUgd2l0aCBjb21wbGV0ZSBzdWJtZXJnZW5jZTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTcwOyAgY2l0YXRpb25fcGFnZXM9MjEzLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIHZhcmlhdGlvbiBvZiBzdWJtZXJnZW5jZSB0b2xlcmFuY2UgYW1vbmcgQXJhYmlkb3BzaXMgdGhhbGlhbmEgYWNjZXNzaW9uczsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTkwOyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIHByb2ZpbGluZyBpbXBsaWNhdGVzIG5vdmVsIGludGVyYWN0aW9ucyBiZXR3ZWVuIGFiaW90aWMgc3RyZXNzIGFuZCBob3Jtb25hbCByZXNwb25zZXMgaW4gVGhlbGx1bmdpZWxsYSwgYSBjbG9zZSByZWxhdGl2ZSBvZiBBcmFiaWRvcHNpczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTQwOyAgY2l0YXRpb25fcGFnZXM9MTQzNy01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3Jvc3Mta2luZ2RvbSBjb21wYXJpc29uIG9mIHRyYW5zY3JpcHRvbWljIGFkanVzdG1lbnRzIHRvIGxvdy1veHlnZW4gc3RyZXNzIGhpZ2hsaWdodHMgY29uc2VydmVkIGFuZCBwbGFudC1zcGVjaWZpYyByZXNwb25zZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1MjsgIGNpdGF0aW9uX3BhZ2VzPTE0ODQtNTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3VibWVyZ2VuY2UgdG9sZXJhbmNlIHJlZ3VsYXRvciBTdWIxQSBtZWRpYXRlcyBzdHJlc3MtcmVzcG9uc2l2ZSBleHByZXNzaW9uIG9mIEFQMi9FUkYgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTI7ICBjaXRhdGlvbl9wYWdlcz0xNjc0LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHZhcmlhYmxlIGNsdXN0ZXIgb2YgZXRoeWxlbmUgcmVzcG9uc2UgZmFjdG9yLWxpa2UgZ2VuZXMgcmVndWxhdGVzIG1ldGFib2xpYyBhbmQgZGV2ZWxvcG1lbnRhbCBhY2NsaW1hdGlvbiByZXNwb25zZXMgdG8gc3VibWVyZ2VuY2UgaW4gcmljZTsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTIwMjEtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQgcHJvZmlsaW5nIG9mIHRoZSBhbm94aWMgcmljZSBjb2xlb3B0aWxlOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz0yMTgtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2YgYWxwaGEtZXhwYW5zaW4gZ2VuZXMgaW4geW91bmcgc2VlZGxpbmdzIG9mIHJpY2UgKE9yeXphIHNhdGl2YSBMLik7IFBsYW50YTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTE7ICBjaXRhdGlvbl9wYWdlcz00NjctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljIHByb2ZpbGluZyBvZiBhIHNhbHQtdG9sZXJhbnQgd2lsZCB0b21hdG8gc3BlY2llcyBhbmQgYSBzYWx0LXNlbnNpdGl2ZSB0b21hdG8gY3VsdGl2YXI7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz05OTctMTAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluY3QgZXZvbHV0aW9uYXJ5IHBhdHRlcm5zIG9mIE9yeXphIGdsYWJlcnJpbWEgZGVjaXBoZXJlZCBieSBnZW5vbWUgc2VxdWVuY2luZyBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2YgT3J5emEgYnJhY2h5YW50aGEgcmV2ZWFscyBtZWNoYW5pc21zIHVuZGVybHlpbmcgT3J5emEgZ2Vub21lIGV2b2x1dGlvbjsgTmF0IENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE1OTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBuZXR3b3JrcyBpbiBjZWxsdWxhciByZXNwb25zZXMgYW5kIHRvbGVyYW5jZSB0byBkZWh5ZHJhdGlvbiBhbmQgY29sZCBzdHJlc3NlczsgQW5udS4gUmV2LiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTc4MS04MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCByZWd1bGF0b3J5IG5ldHdvcmtzIGluIHJlc3BvbnNlIHRvIGFiaW90aWMgc3RyZXNzZXMgaW4gQXJhYmlkb3BzaXMgYW5kIGdyYXNzZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTg4LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIGZhY3RvciBtZWRpYXRlZCBhYmlvdGljIHN0cmVzcyBzaWduYWxpbmcgaW4gcmljZTsgUGxhbnQgU3RyZXNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIHRoZSBTdWIxIGdlbmUgY2x1c3RlciB0aGF0IGNvbmZlcnMgc3VibWVyZ2VuY2UgdG9sZXJhbmNlIHRvIGRvbWVzdGljYXRlZCByaWNlOyBBbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTQzLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWIxQSBpcyBhbiBldGh5bGVuZS1yZXNwb25zZS1mYWN0b3ItbGlrZSBnZW5lIHRoYXQgY29uZmVycyBzdWJtZXJnZW5jZSB0b2xlcmFuY2UgdG8gcmljZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0MjsgIGNpdGF0aW9uX3BhZ2VzPTcwNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVUIxQS1kZXBlbmRlbnQgYW5kIC1pbmRlcGVuZGVudCBtZWNoYW5pc21zIGFyZSBpbnZvbHZlZCBpbiB0aGUgZmxvb2RpbmcgdG9sZXJhbmNlIG9mIHdpbGQgcmljZSBzcGVjaWVzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTI4Mi05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZHVjdGlvbiBvZiBjb3VtYXJveWxzZXJvdG9uaW4gYW5kIGZlcnVsb3lsc2Vyb3RvbmluIGluIHRyYW5zZ2VuaWMgcmljZSBleHByZXNzaW5nIHBlcHBlciBoeWRyb3h5Y2lubmFtb3lsLWNvZW56eW1lIGFzZXJvdG9uaW4gTi0oaHlkcm94eWNpbm5hbW95bCkgdHJhbnNmZXJhc2U7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTM0Ni01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SmFzbW9uYXRlcyBhbiB1cGRhdGUgb24gYmlvc3ludGhlc2lzLCBzaWduYWwgdHJhbnNkdWN0aW9uIGFuZCBhY3Rpb24gaW4gcGxhbnQgc3RyZXNzIHJlc3BvbnNlLCBncm93dGggYW5kIGRldmVsb3BtZW50OyBBbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9NjgxLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZ1bmN0aW9uYWwgZ2Vub21pY3MgYXBwcm9hY2ggdG93YXJkIHRoZSB1bmRlcnN0YW5kaW5nIG9mIHNlY29uZGFyeSBtZXRhYm9saXNtIGluIHBsYW50IGNlbGxzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTg1OTUtNjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PUkNBMywgYSBqYXNtb25hdGUtcmVzcG9uc2l2ZSB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yIG9mIHBsYW50IHByaW1hcnkgYW5kIHNlY29uZGFyeSBtZXRhYm9saXNtOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4OTsgIGNpdGF0aW9uX3BhZ2VzPTI5NS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db29yZGluYXRlZCBhY3RpdmF0aW9uIG9mIG1ldGFib2xpYyBwYXRod2F5cyBmb3IgYW50aW94aWRhbnRzIGFuZCBkZWZlbmNlIGNvbXBvdW5kcyBieSBqYXNtb25hdGVzIGFuZCB0aGVpciByb2xlcyBpbiBzdHJlc3MgdG9sZXJhbmNlIGluIEFyYWJpZG9wc2lzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTY1My02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgbWV0aGlvbmluZSByZWN5Y2xpbmcgZm9yIGV0aHlsZW5lIHN5bnRoZXNpcyBpbiBBcmFiaWRvcHNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz0yMzgtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdWJtZXJnZW5jZSB0b2xlcmFuY2UgZ2VuZSBTVUIxQSBkZWxheXMgbGVhZiBzZW5lc2NlbmNlIHVuZGVyIHByb2xvbmdlZCBkYXJrbmVzcyB0aHJvdWdoIGhvcm1vbmFsIHJlZ3VsYXRpb24gaW4gcmljZTsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTYwOyAgY2l0YXRpb25fcGFnZXM9MTc5NS04MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgYWJzY2lzaWMgYWNpZCBiaW9zeW50aGVzaXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CZWZvcmUgYW5kIGJleW9uZCBBQkEgdXBzdHJlYW0gc2Vuc2luZyBhbmQgaW50ZXJuYWwgc2lnbmFscyB0aGF0IGRldGVybWluZSBBQkEgYWNjdW11bGF0aW9uIGFuZCByZXNwb25zZSB1bmRlciBhYmlvdGljIHN0cmVzczsgQmlvY2hlbS4gU29jLiBUcmFucy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zNzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBoaWdoIHNhbGluaXR5IHRvbGVyYW5jZSBpbiBwbGFudHM7IE1ldGhvZHMgRW56eW1vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDI4OyAgY2l0YXRpb25fcGFnZXM9NDE5LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnNjaXNpYyBhY2lkIGVtZXJnZW5jZSBvZiBhIGNvcmUgc2lnbmFsaW5nIG5ldHdvcms7IEFubnUuIFJldi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz02NTEtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV0aHlsZW5lIHByb21vdGVzIHN1Ym1lcmdlbmNlLWluZHVjZWQgZXhwcmVzc2lvbiBvZiBPc0FCQThveDEsIGEgZ2VuZSB0aGF0IGVuY29kZXMgQUJBIDjigLItaHlkcm94eWxhc2UgaW4gcmljZTsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWxpbml0eSBpbmR1Y2VzIGFjY3VtdWxhdGlvbiBvZiBzb2x1YmxlIHN1Z2FycyBhbmQgYWx0ZXJzIHRoZSBhY3Rpdml0eSBvZiBzdWdhciBtZXRhYm9saXppbmcgZW56eW1lcyBpbiByaWNlIHBsYW50czsgQmlvbC4gUGxhbnQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MjMzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdCBvZiB2YXJpb3VzIGFiaW90aWMgc3RyZXNzZXMgb24gdGhlIGdyb3d0aCwgc29sdWJsZSBzdWdhcnMgYW5kIHdhdGVyIHJlbGF0aW9ucyBvZiBzb3JnaHVtIHNlZWRsaW5ncyBncm93biBpbiBsaWdodCBhbmQgZGFya25lc3M7IEJ1bGcuIEouIFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NzItODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbml0b3JpbmcgdGhlIGV4cHJlc3Npb24gcHJvZmlsZXMgb2YgNzAwMCBBcmFiaWRvcHNpcyBnZW5lcyB1bmRlciBkcm91Z2h0LCBjb2xkIGFuZCBoaWdoLXNhbGluaXR5IHN0cmVzc2VzIHVzaW5nIGEgZnVsbC1sZW5ndGggY0ROQSBtaWNyb2FycmF5OyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI3OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VnYXIgc2lnbmFsbGluZyBhbmQgZ2VuZSBleHByZXNzaW9uIGluIHJlbGF0aW9uIHRvIGNhcmJvaHlkcmF0ZSBtZXRhYm9saXNtIHVuZGVyIGFiaW90aWMgc3RyZXNzZXMgaW4gcGxhbnRzOyBKLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NzYxLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb2x1YmxlIHN1Z2Fyc+KAlG1ldGFib2xpc20sIHNlbnNpbmcgYW5kIGFiaW90aWMgc3RyZXNzIGEgY29tcGxleCBuZXR3b3JrIGluIHRoZSBsaWZlIG9mIHBsYW50czsgUGxhbnQgU2lnbmFsLiBCZWhhdi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTM4OC05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgYmlvbG9neSBnZW5ldGljcyBvZiBoaWdoLXJpc2UgcmljZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ2MDsgIGNpdGF0aW9uX3BhZ2VzPTk1OS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlaGFsb3NlIGFjY3VtdWxhdGlvbiBpbiByaWNlIHBsYW50cyBjb25mZXJzIGhpZ2ggdG9sZXJhbmNlIGxldmVscyB0byBkaWZmZXJlbnQgYWJpb3RpYyBzdHJlc3NlczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xNTg5OC05MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VtdWxhdGlvbiBvZiB0cmVoYWxvc2UgaW5jcmVhc2VzIHNvbHVibGUgc3VnYXIgY29udGVudHMgaW4gcmljZSBwbGFudHMgY29uZmVycmluZyB0b2xlcmFuY2UgdG8gZHJvdWdodCBhbmQgc2FsdCBzdHJlc3M7IFBsYW50IEJpb3RlY2hub2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTg5LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXRod2F5cyBhbmQgZ2VuZXRpYyBkZXRlcm1pbmFudHMgZm9yIGNlbGwgd2FsbC1iYXNlZCBvc21vdGljIHN0cmVzcyB0b2xlcmFuY2UgaW4gdGhlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHJvb3Qgc3lzdGVtOyBjaXRhdGlvbl90ZXh0PUNoLiAyOyBjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXktQmxhY2t3ZWxsLCBPeGZvcmQsIFVLOyBHZW5lcyBmb3IgcGxhbnQgYWJpb3RpYyBzdHJlc3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3J1cHRpb24gb2YgdGhlIGNlbGx1bG9zZSBzeW50aGFzZSBnZW5lLCBBdENlc0E4L0lSWDEsIGVuaGFuY2VzIGRyb3VnaHQgYW5kIG9zbW90aWMgc3RyZXNzIHRvbGVyYW5jZSBpbiBBcmFiaWRvcHNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0yNzMtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2VsbHVsb3NlIHN5bnRoYXNlLWxpa2UgcHJvdGVpbiBpcyByZXF1aXJlZCBmb3Igb3Ntb3RpYyBzdHJlc3MgdG9sZXJhbmNlIGluIEFyYWJpZG9wc2lzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTEyOC00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBsYW50IGN1dGljbGUgaXMgcmVxdWlyZWQgZm9yIG9zbW90aWMgc3RyZXNzIHJlZ3VsYXRpb24gb2YgYWJzY2lzaWMgYWNpZCBiaW9zeW50aGVzaXMgYW5kIG9zbW90aWMgc3RyZXNzIHRvbGVyYW5jZSBpbiBBcmFiaWRvcHNpczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE5NzEtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvb3Qgc3ViZXJpbiBmb3JtcyBhbiBleHRyYWNlbGx1bGFyIGJhcnJpZXIgdGhhdCBhZmZlY3RzIHdhdGVyIHJlbGF0aW9ucyBhbmQgbWluZXJhbCBudXRyaXRpb24gaW4gQXJhYmlkb3BzaXM7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDQ5MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YmVyaW4gZ29lcyBnZW5vbWljcyB1c2Ugb2YgYSBzaG9ydCBsaXZpbmcgcGxhbnQgdG8gaW52ZXN0aWdhdGUgYSBsb25nIGxhc3RpbmcgcG9seW1lcjsgRnJvbnQuIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvMS82OS85OTUyMjQvZHN0MDQyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQb3J0ZXJlc2lhIGNvYXJjdGF0YSBpcyBhIHdpbGQgcmVsYXRpdmUgb2YgcmljZSB3aXRoIGNhcGFiaWxpdHkgb2YgaGlnaCBzYWxpbml0eSBhbmQgc3VibWVyZ2VuY2UgdG9sZXJhbmNlLiBUaGUgdHJhbnNjcmlwdG9tZSBhbmFseXNlcyBvZiBQb3J0ZXJlc2lhICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ3NTI2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgQWxsIEZpZ3VyZXNkZAYzsAKP7yfm7gINppWwaLlfWIL5" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f69%2f347526" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f69%2f347526" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1509238921&amp;Signature=IMKeALqTlsSZee8zSWyElxMZ24su6PXBeAFCnAN~JpI~fanlONPgxIgL~Qx5SbWzWWiwpdQtKF0l6AcCKhCcoUhrscPsn0niVRpvCg81VB5CM54T9x9H8Pune-Wimysq4lPDokxJ7tyaj7qXHGrX9NPGpxhIzSyXAlvVP~-I6a~~5tsUB7fYwRT-gKzby9LEc4-6dOmnEIAbsOGZKE0UEnStyuiGZ-VpykivzCxhZFEiu~frZPVMowu07U5ZuFW4Mkny4~uMcOzDyRvP7GPZnt7DwaGyUlXnHObjzT8AloE5yTTLXs2WJAzl7zeoVtJ-b5y~VxDJqZT6OfNLE9A-pA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/1">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4921249">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921249">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921252">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921252">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921258">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921258">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921290">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921290">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921337">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921337">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921346">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921346">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921348">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921348">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921350">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921350">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4921352">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4921352">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/1/53/347430">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/1/85/347657">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Deep Transcriptome Sequencing of Wild Halophyte Rice, <em>Porteresia coarctata</em>, Provides Novel Insights into the Salinity and Submergence Tolerance Factors
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Rohini Garg</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rohini Garg</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Functional and Applied Genomics Laboratory</div>, <div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Rohini+Garg">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Garg R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Garg author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Mohit Verma</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mohit Verma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Functional and Applied Genomics Laboratory</div>, <div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mohit+Verma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Verma M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Verma author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Shashank Agrawal</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shashank Agrawal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Functional and Applied Genomics Laboratory</div>, <div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shashank+Agrawal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Agrawal S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Agrawal author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Rama Shankar</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rama Shankar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Functional and Applied Genomics Laboratory</div>, <div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Rama+Shankar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shankar R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Shankar author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Manoj Majee</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manoj Majee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Functional and Applied Genomics Laboratory</div>, <div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Manoj+Majee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Majee M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Majee author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Mukesh Jain</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mukesh Jain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Functional and Applied Genomics Laboratory</div>, <div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 11-26735182. Fax. +91 11-26741658. Email: <a href="mailto:mjain@nipgr.ac.in" target="_blank">mjain@nipgr.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mukesh+Jain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jain M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Jain author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 69–84, <a href='https://doi.org/10.1093/dnares/dst042'>https://doi.org/10.1093/dnares/dst042</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 October 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 March 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 September 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347526" href="/dnaresearch/article-pdf/21/1/69/995224/dst042.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rohini Garg, Mohit Verma, Shashank Agrawal, Rama Shankar, Manoj Majee, Mukesh Jain;  Deep Transcriptome Sequencing of Wild Halophyte Rice, <em>Porteresia coarctata</em>, Provides Novel Insights into the Salinity and Submergence Tolerance Factors, <em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 69–84, <a href='https://doi.org/10.1093/dnares/dst042'>https://doi.org/10.1093/dnares/dst042</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347526?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347526?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347526?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347526?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347526?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347526?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347526?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Deep%20Transcriptome%20Sequencing%20of%20Wild%20Halophyte%20Rice%2C%20%26lt%3Bem%26gt%3BPorteresia%20coarctata%26lt%3B%2Fem%26gt%3B%2C%20Provides%20Novel%20Insights%20into%20the%20Salinity%20and%20Submergence%20Tolerance%20Factors&amp;publicationDate=2013-10-08&amp;volumeNum=21&amp;issueNum=1&amp;author=Garg%2C%20Rohini%3B%20Verma%2C%20Mohit&amp;startPage=69&amp;endPage=84&amp;contentId=10.1093%2Fdnares%2Fdst042&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f69%2f347526" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4921249 id="4921249" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Porteresia coarctata</em> is a wild relative of rice with capability of high salinity and submergence tolerance. The transcriptome analyses of <em>Porteresia</em> can lead to the identification of candidate genes involved in salinity and submergence tolerance. We sequenced the transcriptome of <em>Porteresia</em> under different conditions using Illumina platform and generated about 375 million high-quality reads. After optimized assembly, a total of 152 367 unique transcript sequences with average length of 794 bp were obtained. Many of these sequences might represent fragmented transcripts. Functional annotation revealed the presence of genes involved in diverse cellular processes and 2749 transcription factor (TF)-encoding genes in <em>Porteresia</em>. The differential gene expression analyses identified a total of 15 158 genes involved in salinity and/or submergence response(s). The stress-responsive members of different TF families, including MYB, bHLH, AP2-EREBP, WRKY, bZIP and NAC, were identified. We also revealed key metabolic pathways, including amino acid biosynthesis, hormone biosynthesis, secondary metabolite biosynthesis, carbohydrate metabolism and cell wall structures, involved in stress tolerance in <em>Porteresia</em>. The transcriptome analyses of <em>Porteresia</em> are expected to highlight genes/pathways involved in salinity and submergence tolerance of this halophyte species. The data can serve as a resource for unravelling the underlying mechanism and devising strategies to engineer salinity and submergence tolerance in rice.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;metabolic pathways&quot;">metabolic pathways</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Porteresia"><em>Porteresia</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;salinity tolerance&quot;">salinity tolerance</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="submergence">submergence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptome analysis&quot;">transcriptome analysis</a></div>                            <h2 scrollto-destination=4921252 id="4921252" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>High salinity is a severe and increasing constraint on the global crop production. Most of our major crops are highly susceptible to salinity, and there is an urgent need to address this problem. Salinity tolerance is a complex multigenic trait, which depends on a range of physiological processes. The salinity stress response has been studied in model crop plant rice,<sup><a reveal-id="DST042C1  DST042C4" data-open="DST042C1  DST042C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> which represents a glycophyte plant and does not possess the capability to tolerate high salinity. The existence of wild halophytic species, which can grow not only in saline but also in submergence prone habitats, has been reported.<sup><a reveal-id="DST042C5" data-open="DST042C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST042C6" data-open="DST042C6" class="link link-ref link-reveal xref-bibr">6</a></sup> These wild species may prove to be an important gene reservoir for improving salinity tolerance in crop plants via genetic engineering approaches.<sup><a reveal-id="DST042C5  DST042C8" data-open="DST042C5  DST042C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup></p><p><em>Porteresia coarctata</em> (<em>Oryza coarctata</em>) is a wild halophyte, which shows considerable adaptation to high salinity (20–40 dS m<sup>−1</sup>) and can also tolerate complete submergence in saline water for a long period.<sup><a reveal-id="DST042C9" data-open="DST042C9" class="link link-ref link-reveal xref-bibr">9</a></sup><em>Porteresia</em> grows in abundance near coastal areas, where the vegetation is inundated twice a day with highly saline water. Although many salinity- and submergence-tolerant rice cultivars have also been identified,<sup><a reveal-id="DST042C10  DST042C12" data-open="DST042C10  DST042C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> they seem to be much less tolerant than <em>Porteresia</em>. It is a close wild relative of rice and can serve as an important source of genes related to salinity and submergence tolerance.<sup><a reveal-id="DST042C8" data-open="DST042C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST042C13" data-open="DST042C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Some novel insights into the molecular mechanisms underlying salinity tolerance have been obtained from transcriptome and genome analyses of close halophyte relatives of Arabidopsis, <em>Thellungiella halophila</em> and <em>T. parvula</em>.<sup><a reveal-id="DST042C14  DST042C16" data-open="DST042C14  DST042C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> A few studies have demonstrated that engineering of some genes from <em>Porteresia</em>, such as those involved in inositol metabolic pathway, can confer salinity tolerance in cultivated rice.<sup><a reveal-id="DST042C17" data-open="DST042C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST042C18" data-open="DST042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> However, only a few genes have been isolated from <em>Porteresia</em> so far. Furthermore, the effect of submergence stress and its combined effect with salinity have hardly been analysed.<sup><a reveal-id="DST042C5" data-open="DST042C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The discovery of more genes and metabolic pathways in wild species like <em>Porteresia</em> can provide a better strategy to impart salinity tolerance in cultivated rice.</p><p>The availability of next-generation sequencing technologies provides a high-throughput, rapid and cost-effective means to sequence and characterize the transcriptome of non-model and wild species as well.<sup><a reveal-id="DST042C19  DST042C21" data-open="DST042C19  DST042C21" class="link link-ref link-reveal xref-bibr">19–21</a></sup> In this study, we sequenced the transcriptome of <em>Porteresia</em> under different conditions using Illumina technology. The transcriptome assembly was generated to reveal the first picture of its gene content. We also analysed gene expression under salinity and submergence conditions and identified the genes and metabolic pathways involved in these stress responses. The data presented here will provide a very useful resource to engineer salinity and submergence tolerance stress in crop plant rice.</p>                            <h2 scrollto-destination=4921258 id="4921258" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4921261 id="4921261" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p><em>Porteresia coarctata</em> (Roxb) Tateoka plants were collected from saline river banks (Matla) of the Sunderbans in the coastal region of Bay of Bengal, India. The plants were brought to the laboratory along with soil and water. The plants were washed carefully to remove soil from the leaves and roots. The plants were given different treatments, including control (RO water, control), low salinity (450 mM NaCl, salt450), high salinity (700 mM NaCl solution, salt700), submergence (fully submerged in RO water, submergence) and submergence in low salinity (fully submerged in 450 mM NaCl, salt + submergence). The plants were harvested in three biological replicates 12 h after the treatment.</p>                            <h3 scrollto-destination=4921265 id="4921265" class="section-title" data-legacy-id=s2b>2.2. RNA isolation and sequencing</h3>
<p>Total RNA was isolated from each tissue sample (control, salt450, salt700, submergence and salt + submergence) using TRI reagent (Sigma Life Science, USA). The quantity and quality of RNA samples were checked using Nanodrop (Thermo Fisher Scientific) and Agilent Bioanalyzer (Agilent Technologies, Singapore). Equal amounts of total RNA from three biological replicates were pooled for all the five treated samples before library preparation. Sequencing of all the five samples was performed by the commercial service provider, Genotypic Technology Pvt. Ltd, Bangalore, India, each in one lane using an Illumina Genome Analyzer II to generate 100 base long paired-end reads. The raw data obtained were filtered through the standard Illumina pipeline. The filtered Fastq files were further subjected to more stringent quality control using the NGS QC Toolkit (v2.3) to remove the low-quality reads and reads containing adaptor/primer contamination.<sup><a reveal-id="DST042C22" data-open="DST042C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The high-quality filtered reads were used for further downstream processing. The entire sequence data generated in this study are available in the Gene Expression Omnibus database under the accession number GSE44913. The non-redundant Illumina reads were filtered using the CDHIT program (<a class="link link-uri" href="http://weizhong-lab.ucsd.edu/cd-hit/">http://weizhong-lab.ucsd.edu/cd-hit/</a>).</p>                            <h3 scrollto-destination=4921269 id="4921269" class="section-title" data-legacy-id=s2c>2.3. Transcriptome assembly</h3>
<p><em>De novo</em> transcriptome assembly was performed using various tools, including Velvet (v1.2.01), Oases (v0.2.04), ABySS (v1.2.6), Trinity (vr2012-05-18) and CLC Genomics Workbench (v4.7.2). Various parameters, including <em>k</em>-mer length, insert length and expected coverage, were optimized to obtain best assembly as described previously.<sup><a reveal-id="DST042C23" data-open="DST042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Reference-based assembly was performed using the CLC Genomics Workbench using default parameters, and assembled consensus sequences were exported into a fasta file. The annotated cDNA sequences of <em>japonica</em> rice (Os-Nipponbare-Reference-IRGSP-1.0)<sup><a reveal-id="DST042C24" data-open="DST042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> available at Rice Genome Annotation Project (MSUv7.0; <a class="link link-uri" href="http://rice.plantbiology.msu.edu/">http://rice.plantbiology.msu.edu/</a>) were used as a reference. Merged assembly was performed using the TGICL program (v2.0), with minimum overlap length of 40 and minimum percentage identity of 90 for overlaps. The quality of assemblies generated was assessed using the perl scripts given in the NGS QC Toolkit.</p>                            <h3 scrollto-destination=4921274 id="4921274" class="section-title" data-legacy-id=s2d>2.4. Functional annotation</h3>
<p>BLASTX searches against rice proteome, UniRef90, UniRef100 and NCBI non-redundant nucleotide databases, were performed to assign a putative function to each <em>Porteresia</em> transcript. GOSlim terms to each transcript were assigned on the basis of their best significant match (<em>E</em>-value ≤ 1<em>e</em>−10) rice protein to provide a broad overview of their function. An <em>E</em>-value cut-off of 1<em>e</em>−10 was used to identify orthologs between <em>Porteresia</em> and rice. The transcription factor (TF) encoding <em>Porteresia</em> transcripts were identified based on the hidden Markov model profile search of conserved domain present in each TF family as described previously.<sup><a reveal-id="DST042C23" data-open="DST042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The statistical significance of the difference in the number of members between <em>Porteresia</em> and rice in each TF family was calculated via Fisher's exact test.</p>                            <h3 scrollto-destination=4921279 id="4921279" class="section-title" data-legacy-id=s2e>2.5. Read mapping and gene expression analysis</h3>
<p>To estimate the gene expression, all the high-quality reads from each condition were aligned on the transcriptome assembly using the RNA-Seq Analysis utility of CLC Genomics Workbench. A maximum of two mismatches were allowed for the alignments. The read counts for each tissue sample were normalized by calculating the read per kilobase per million for each transcript. The DESeq software (v1.10.1)<sup><a reveal-id="DST042C25" data-open="DST042C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was used for differential gene expression analysis. DESeq provides a simple and computationally faster method for quantitative measurement of gene expression based on the negative binomial distribution with variance and mean linked by local regression.<sup><a reveal-id="DST042C25" data-open="DST042C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We calculated the size factor for each sample for normalization of read count data using DESeq. A <em>P</em>-value cut-off of ≤0.05 and at least two-fold change was used to identify differentially expressed genes. Heatmaps showing expression profiles (log<sub>2</sub> fold change) were generated using the MultiExperiment Viewer (MeV, v4.8). <em>K</em>-means clustering was performed using Euclidean distance matrix with 1000 iterations using MeV. Hierarchical clustering was performed using Euclidean distance matrix with a complete linkage rule using MeV.</p>                            <h3 scrollto-destination=4921283 id="4921283" class="section-title" data-legacy-id=s2f>2.6. Gene ontology and pathway enrichment analysis</h3>
<p>The best Arabidopsis hit corresponding to each <em>Porteresia</em> transcript was identified using BLAST search to study gene ontology (GO) enrichment. GO enrichment of different sets of differentially expressed genes was performed using the BiNGO tool.<sup><a reveal-id="DST042C26" data-open="DST042C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The metabolic pathway data available in the RiceCyc database of Gramene<sup><a reveal-id="DST042C27" data-open="DST042C27" class="link link-ref link-reveal xref-bibr">27</a></sup> were analysed to identify the enriched metabolic pathways in various gene sets. Both the GO and pathway enrichment analyses were performed at <em>P</em>-value cut-off of ≤0.05 after applying Benjamini-Hochberg correction.</p>                            <h3 scrollto-destination=4921286 id="4921286" class="section-title" data-legacy-id=s2g>2.7. Real-time PCR analysis</h3>
<p>For real-time PCR analysis, gene-specific primers were designed using the Primer Express (v3.0) software (Applied Biosystems, Santa Clara, CA, USA). The primer sequences used in this study are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Real-time PCRs were performed as described.<sup><a reveal-id="DST042C28" data-open="DST042C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The expression of most suitable internal control gene for rice, <em>UBQ5</em>,<sup><a reveal-id="DST042C29" data-open="DST042C29" class="link link-ref link-reveal xref-bibr">29</a></sup> in each sample was used for normalization. At least three independent biological replicates with three technical replicates of each biological replicates for each tissue sample were used for the analysis. The correlation between expression profiles of selected genes obtained from real-time PCR and RNA-seq data analyses was determined using MS Excel.</p>                            <h2 scrollto-destination=4921290 id="4921290" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4921293 id="4921293" class="section-title" data-legacy-id=s3a>3.1. Transcriptome sequencing</h3>
<p>To generate the transcriptome of <em>Porteresia</em>, we collected tissue samples from plants kept in water (control) and those subjected to salinity and submerged conditions. For salinity stress, we kept the plants in 450 mM (salt450) and 700 mM (salt700) sodium chloride solutions and for submergence stress, plants were kept completely submerged in water (submergence) and 450 mM sodium chloride solution (salt + submergence) and collected the tissues after 12 h of treatment. <em>Porteresia</em> is a halophyte, which grows naturally in coastal region, where plants experience lunar tide and are exposed to submergence in saline sea water every 12 h. Therefore, to obtain maximum representation of genes involved in signalling and adaptation response under salinity and submergence stress, we selected time point of 12 h for the analysis in our study. The RNA-seq libraries were constructed and sequenced using an Illumina sequencing platform. We generated a total of more than 422 million reads from above five samples. After quality filtering, about 375 million high-quality reads (ranging from about 65 to 90 million reads for each sample) were obtained (Table <a reveal-id="DST042TB1" data-open="DST042TB1" class="link link-reveal link-table xref-fig">1</a>). The average Phred quality score was at least 30 at each base position for filtered reads, indicating high quality of filtered reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="4921298" scrollto-destination="4921298"></a>
<div class="table-wrap"><div id="DST042TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of sequencing data generated and mapping to <em>Porteresia</em> transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Total reads </th><th>High-quality reads </th><th>Mapped reads<sup>a</sup> </th><th>Mapped reads (%) </th></tr></thead><tbody><tr><td>Control </td><td>73 200 160 </td><td>64 799 692 </td><td>56 638 113 </td><td>87.40 </td></tr><tr><td>Salt450 </td><td>78 717 162 </td><td>68 275 254 </td><td>59 818 916 </td><td>87.61 </td></tr><tr><td>Salt700 </td><td>73 664 082 </td><td>66 857 102 </td><td>58 431 827 </td><td>87.40 </td></tr><tr><td>Submerged </td><td>101 517 546 </td><td>89 800 570 </td><td>78 997 383 </td><td>87.97 </td></tr><tr><td>Salt450 + submerged </td><td>95 241 740 </td><td>85 027 006 </td><td>74 867 541 </td><td>88.05 </td></tr><tr><td>Total </td><td>422 340 690 </td><td>374 759 624 </td><td>328 753 780 </td><td>87.72 </td></tr></tbody></table></div><div content-id="DST042TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Total reads </th><th>High-quality reads </th><th>Mapped reads<sup>a</sup> </th><th>Mapped reads (%) </th></tr></thead><tbody><tr><td>Control </td><td>73 200 160 </td><td>64 799 692 </td><td>56 638 113 </td><td>87.40 </td></tr><tr><td>Salt450 </td><td>78 717 162 </td><td>68 275 254 </td><td>59 818 916 </td><td>87.61 </td></tr><tr><td>Salt700 </td><td>73 664 082 </td><td>66 857 102 </td><td>58 431 827 </td><td>87.40 </td></tr><tr><td>Submerged </td><td>101 517 546 </td><td>89 800 570 </td><td>78 997 383 </td><td>87.97 </td></tr><tr><td>Salt450 + submerged </td><td>95 241 740 </td><td>85 027 006 </td><td>74 867 541 </td><td>88.05 </td></tr><tr><td>Total </td><td>422 340 690 </td><td>374 759 624 </td><td>328 753 780 </td><td>87.72 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of high-quality reads mapped using CLC Genomics workbench.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4921298" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4921300 id="4921300" class="section-title" data-legacy-id=s3b>3.2. Transcriptome assembly optimization</h3>
<p>To generate the optimal assembly of the <em>Porteresia</em> transcriptome, first, we performed <em>de novo</em> assembly using all the high-quality reads (374 759 624) and non-redundant reads (194 316 505) after removing duplicate reads from all the five samples. We used Velvet, Oases, ABySS, Trinity and CLC Genomics Workbench softwares to perform <em>de novo</em> assembly. The <em>de novo</em> assembly was generated at different <em>k</em>-mer lengths using these softwares. From each software, the best assembly output was selected based on the statistics (average and N50 contig lengths) and similarity with rice proteome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Among the various softwares, Oases generated the best assembly with largest average and N50 lengths at <em>k</em>-mer length of 59 for both datasets (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). However, only 34% and 29% rice proteins showed a significant hit with the assemblies generated from whole and non-redundant datasets, respectively. The assembly output from other softwares also exhibited very low similarity to rice proteins. As <em>Porteresia</em> is a wild relative of rice, higher similarity to rice proteins was expected. These results suggested that the output of <em>de novo</em> assembly was not optimal.</p><p>Therefore, we followed a combined approach of both reference-based and <em>de novo</em> assemblies to generate the transcriptome of <em>Porteresia</em>. In the first step, we performed reference-based assembly using the CLC Genomics Workbench. We used all the reads from each sample to generate a consensus assembly (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The average and N50 lengths of reference assembly were much higher when compared with <em>de novo</em> assemblies. The consensus assembly from all the five samples generated a total of 36 966 contigs with the average length of 1265 bp and N50 length of 1761 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). This assembly utilized ∼81% of the total reads. To identify the novel genes in <em>Porteresia</em>, as a second step, we performed <em>de novo</em> assembly of unused reads in the first step of reference-based assembly. We used different softwares (CLC, Trinity, ABySS, Velvet and Oases) and parameters to select the best assembly. Oases generated the best assembly with largest average (544 bp) and N50 (719 bp) contig lengths. However, the number of consensus sequences (161 160) generated was quite large. As a last step, we generated a merged assembly of the contigs generated in primary assemblies of two steps via TGICL program. Our earlier studies have shown that TGICL generates optimal merged assembly from the contigs of primary assemblies.<sup><a reveal-id="DST042C23" data-open="DST042C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DST042C30" data-open="DST042C30" class="link link-ref link-reveal xref-bibr">30</a></sup> This assembly resulted in a total of 152 367 (39 203 contigs and 113 164 singletons) unique sequences (referred as contigs) of at least 200 bp in length (Table <a reveal-id="DST042TB2" data-open="DST042TB2" class="link link-reveal link-table xref-fig">2</a>). The average and N50 lengths of contigs were 794 and 1249 bp, respectively. The mapping of all the high-quality reads suggested that &gt;95% of reads were used in the final merged assembly and average read depth of each contig was 2352. About 55% of rice proteins were represented in this assembly and &gt;23% of rice proteins exhibited ≥70% coverage (Table <a reveal-id="DST042TB2" data-open="DST042TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="4921308" scrollto-destination="4921308"></a>
<div class="table-wrap"><div id="DST042TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Assembly statistics of <em>Porteresia</em> transcriptome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Total reads </td><td>374 759 624 </td></tr><tr><td>Total transcripts </td><td>152 367 </td></tr><tr><td>Size (Mb) </td><td>120.99 </td></tr><tr><td>Minimum transcript length (bp) </td><td>200 </td></tr><tr><td>Maximum transcript length (bp) </td><td>14 945 </td></tr><tr><td>Average transcript length (bp) </td><td>794.07 </td></tr><tr><td>N50 length (bp) </td><td>1249 </td></tr><tr><td>Total reads used for assembly </td><td>95.62% </td></tr><tr><td>Average read depth </td><td>2352 </td></tr><tr><td>Transcripts with significant hits<sup><sup>a</sup></sup> </td><td>48 048 (31.5%) </td></tr><tr><td>Rice proteins with significant hit<sup>b</sup> </td><td>21 499 (55.06%) </td></tr><tr><td>Rice proteins with ≥70% coverage<sup>c</sup> </td><td>9114 (23.34%) </td></tr></tbody></table></div><div content-id="DST042TB2" class="table-modal"><table><tbody><tr><td>Total reads </td><td>374 759 624 </td></tr><tr><td>Total transcripts </td><td>152 367 </td></tr><tr><td>Size (Mb) </td><td>120.99 </td></tr><tr><td>Minimum transcript length (bp) </td><td>200 </td></tr><tr><td>Maximum transcript length (bp) </td><td>14 945 </td></tr><tr><td>Average transcript length (bp) </td><td>794.07 </td></tr><tr><td>N50 length (bp) </td><td>1249 </td></tr><tr><td>Total reads used for assembly </td><td>95.62% </td></tr><tr><td>Average read depth </td><td>2352 </td></tr><tr><td>Transcripts with significant hits<sup><sup>a</sup></sup> </td><td>48 048 (31.5%) </td></tr><tr><td>Rice proteins with significant hit<sup>b</sup> </td><td>21 499 (55.06%) </td></tr><tr><td>Rice proteins with ≥70% coverage<sup>c</sup> </td><td>9114 (23.34%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of transcripts showing a significant hit (<em>E</em>-value ≤ 1<em>e</em>−5) with rice proteins.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of rice proteins showing a significant hit (<em>E</em>-value ≤ 1<em>e</em>−5) with <em>Porteresia</em> transcripts.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Number of rice proteins that showed ≥70% coverage.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4921308" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4921310 id="4921310" class="section-title" data-legacy-id=s3c>3.3. Functional annotation of <em>Porteresia</em> transcriptome</h3>
<p>The unique set of transcripts generated above were designated as <em>P. coarctata</em> tentative consensus (PcTC) transcripts and assigned unique identifier number from PcTC000001 to PcTC152367. The whole transcriptome sequence is available at <em>Porteresia</em> Transcriptome Sequence web page (<a class="link link-uri" href="http://nipgr.res.in/mjain.html?Page=porteresia">http://nipgr.res.in/mjain.html?Page=porteresia</a>). The total transcriptome assembly size was ∼121 Mb. About 48% of all the transcripts were longer than 500 bp and &gt;24% transcripts were larger than 1000 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Most of the smaller transcripts were represented by the singletons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Many of these are expected to merge into larger transcripts, once more sequence data of larger read length become available. The average GC content and distribution of <em>Porteresia</em> transcripts were significantly different than that of rice (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>), indicating substantial differences between rice and <em>Porteresia</em> genomes. Furthermore, to explore the putative function, we assigned gene description to each <em>Porteresia</em> transcript corresponding to the ortholog rice proteins and proteins in other databases. A gene description term could be assigned to ∼36% of <em>Porteresia</em> transcripts. The lower percentage of annotated transcripts was because most of the smaller transcripts generated in the <em>de novo</em> assembly did not show significant similarity with known proteins in various databases and presumably represent species-specific genes.</p><p>Based on their similarity with rice genes, <em>Porteresia</em> transcripts were assigned GOSlim terms under biological process, molecular function and cellular component categories. Among the biological process terms, biosynthetic process (20%) was most represented followed by nucleic acid metabolic process (15.3%) and response to stress (14.5%) (Fig. <a reveal-id="DST042F1" data-open="DST042F1" class="link link-reveal link-table xref-fig">1</a>A). A high representation of response to stress category might contribute to the high salinity and submergence tolerance phenotype of <em>Porteresia</em>. The largest number of transcripts belonged to catalytic activity (16.1%), protein binding (15.1%) and hydrolase activity (12.3%) among the molecular function GOSlim terms. Among the cellular component terms, the transcripts related to membrane (18.5%) and plastid (17%) were the largest in number (Fig. <a reveal-id="DST042F1" data-open="DST042F1" class="link link-reveal link-table xref-fig">1</a>A). We further analysed the <em>Porteresia</em> transcriptome to identify the TF-encoding genes. A total of 2749 transcripts encoding for TFs were identified. Among the 85 families represented in these transcripts, the MYB-domain (197) family TFs were most abundant followed by bHLH (150)- and NAC (147)-domain TFs (Fig. <a reveal-id="DST042F1" data-open="DST042F1" class="link link-reveal link-table xref-fig">1</a>B). Although the total number of TFs was slightly higher in <em>Porteresia</em> (2749) than in rice (2452), we found significant differences in their distribution in certain families. Notably, the number of TFs predicted in <em>Porteresia</em> for bromodomain, zinc-finger CCHC and SNF2 families were significantly higher than rice. Likewise, the number of predicted TFs in AP2-EREBP, GRAS, MADS, TCP, TRAF and ZF-HD families were significantly lesser in <em>Porteresia</em> than in rice (Fig. <a reveal-id="DST042F1" data-open="DST042F1" class="link link-reveal link-table xref-fig">1</a>B). These results suggest significant differences in the transcriptional machinery of the two species and might contribute to stress adaptation phenotype in <em>Porteresia</em>. </p>                        <a id="4921316" scrollto-destination="4921316"></a>
<div id="DST042F1" data-id="DST042F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04201.png?Expires=1509238923&Signature=AVqFXs0hYMgpQnHZzrcyfZ~3rBjEbDdPUqCgME2lTo3zu0LD28jnHmEhWdIRFhE5-jxLAYUyWQHWUJcLAhn9dcG~lLHNWiW2tNNuzbFQBLIoCKiOl17z51gDUL~7vyghnA2GA1sj2g~GtuOIoDzgFyO8tK~Giyi4ct2k9HhxuNflE6R-hvVQwWLgdxBRWigNCzwu5otcevEp2fwFS1G9yMRYcyYJ2SmRK3DdB1AwuhDTX6KZOm0rzZyFV7CpE~4F7xgfVyTQtNscP3wDkA0u6viA91CPZv07VHfsr~4yhLckeKN7zbd~-XuSQz246L9m25w1ghduy40tkueyA5J6Cg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Functional annotation of Porteresia transcripts. (A) GOSlim term assignment to the Porteresia transcripts in different categories of biological process, molecular function and cellular component. (B) The number of transcripts representing different TF families in Porteresia and rice transcriptomes are shown. Fisher's exact test was used to evaluate the significant difference in the number of members representing TF families in the two species. The TF families with statistically significant difference in the number of members are marked with single (P ≤ 0.05) or double (P ≤ 0.01) asterisks." path-from-xml="dst04201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921316/dst04201.tiff" path-from-xml="dst04201.tiff" target="_blank">View large</a><a section="4921316" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04201.png?Expires=1509238923&Signature=NFhOTgkrE2WY0YyHhqAMZhdBwiFwbgsDRjdPChl0JE3fDhRxF0Uk712rJML7-zNYC33OAJH~5Wem5TzFIDu5vR4s23AUxCZbnoNOmcfLb1aFHmdFsSJR3--OjIZKBEfhreV69Dl0vkYLQtglB4xOt4u0upXT68fmQ5-TdfAYGgS~qaT6fOl4p-4W~SKxiSjZPRAlpa3Cl-5B0hu2Sl~5gM1bs0soJB~nAnO0CyZvXZyEBpzk3HlSjOqizDAcg3Nk8XGlfirp6ytup0KHn7gVH5y7ckbF6xytUntqm-dMo5FUBxx3oHpjtCzXUteo3~VK1wG2p~GpmhPqTmHY8rz1Sg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921316&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional annotation of <em>Porteresia</em> transcripts. (A) GOSlim term assignment to the <em>Porteresia</em> transcripts in different categories of biological process, molecular function and cellular component. (B) The number of transcripts representing different TF families in <em>Porteresia</em> and rice transcriptomes are shown. Fisher's exact test was used to evaluate the significant difference in the number of members representing TF families in the two species. The TF families with statistically significant difference in the number of members are marked with single (<em>P</em> ≤ 0.05) or double (<em>P</em> ≤ 0.01) asterisks.</p></div></div><div content-id="DST042F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04201.png?Expires=1509238923&Signature=AVqFXs0hYMgpQnHZzrcyfZ~3rBjEbDdPUqCgME2lTo3zu0LD28jnHmEhWdIRFhE5-jxLAYUyWQHWUJcLAhn9dcG~lLHNWiW2tNNuzbFQBLIoCKiOl17z51gDUL~7vyghnA2GA1sj2g~GtuOIoDzgFyO8tK~Giyi4ct2k9HhxuNflE6R-hvVQwWLgdxBRWigNCzwu5otcevEp2fwFS1G9yMRYcyYJ2SmRK3DdB1AwuhDTX6KZOm0rzZyFV7CpE~4F7xgfVyTQtNscP3wDkA0u6viA91CPZv07VHfsr~4yhLckeKN7zbd~-XuSQz246L9m25w1ghduy40tkueyA5J6Cg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Functional annotation of Porteresia transcripts. (A) GOSlim term assignment to the Porteresia transcripts in different categories of biological process, molecular function and cellular component. (B) The number of transcripts representing different TF families in Porteresia and rice transcriptomes are shown. Fisher's exact test was used to evaluate the significant difference in the number of members representing TF families in the two species. The TF families with statistically significant difference in the number of members are marked with single (P ≤ 0.05) or double (P ≤ 0.01) asterisks." path-from-xml="dst04201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921316/dst04201.tiff" path-from-xml="dst04201.tiff" target="_blank">View large</a><a section="4921316" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04201.png?Expires=1509238923&Signature=NFhOTgkrE2WY0YyHhqAMZhdBwiFwbgsDRjdPChl0JE3fDhRxF0Uk712rJML7-zNYC33OAJH~5Wem5TzFIDu5vR4s23AUxCZbnoNOmcfLb1aFHmdFsSJR3--OjIZKBEfhreV69Dl0vkYLQtglB4xOt4u0upXT68fmQ5-TdfAYGgS~qaT6fOl4p-4W~SKxiSjZPRAlpa3Cl-5B0hu2Sl~5gM1bs0soJB~nAnO0CyZvXZyEBpzk3HlSjOqizDAcg3Nk8XGlfirp6ytup0KHn7gVH5y7ckbF6xytUntqm-dMo5FUBxx3oHpjtCzXUteo3~VK1wG2p~GpmhPqTmHY8rz1Sg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921316&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional annotation of <em>Porteresia</em> transcripts. (A) GOSlim term assignment to the <em>Porteresia</em> transcripts in different categories of biological process, molecular function and cellular component. (B) The number of transcripts representing different TF families in <em>Porteresia</em> and rice transcriptomes are shown. Fisher's exact test was used to evaluate the significant difference in the number of members representing TF families in the two species. The TF families with statistically significant difference in the number of members are marked with single (<em>P</em> ≤ 0.05) or double (<em>P</em> ≤ 0.01) asterisks.</p></div></div>                            <h3 scrollto-destination=4921318 id="4921318" class="section-title" data-legacy-id=s3d>3.4. Differential gene expression analysis</h3>
<p>In addition to the transcriptome construction, RNA-seq provides an opportunity to measure gene expression at the whole-genome level.<sup><a reveal-id="DST042C31" data-open="DST042C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DST042C32" data-open="DST042C32" class="link link-ref link-reveal xref-bibr">32</a></sup> To investigate the differential gene expression under different conditions, we mapped the reads from each sample on the <em>Porteresia</em> transcriptome. A total of 87–88% of the high-quality Illumina reads were mapped to the <em>Porteresia</em> transcripts (referred as genes hereafter) from each sample (Table <a reveal-id="DST042TB1" data-open="DST042TB1" class="link link-reveal link-table xref-fig">1</a>). Differential gene expression analysis was performed using the DESeq software to identify the genes with altered expression under different conditions. A total of 15 158 genes exhibited differential expression under at least one stress condition analysed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The largest number of genes was differentially expressed in the plants subjected to submergence stress (5864) followed by both salinity and submergence stresses together (5787) (Fig. <a reveal-id="DST042F2" data-open="DST042F2" class="link link-reveal link-table xref-fig">2</a>A). A total of 4841 and 4243 genes exhibited differential expression under low salinity (salt450) and high salinity (salt700) stress conditions, respectively. Under salinity stress, a larger number of genes were up-regulated (3020 under salt450 and 2553 under salt700) when compared with down-regulated genes (1821 under salt450 and 1690 under salt700). However, in contrast, the larger number of genes was down-regulated under submergence stress alone (3712) and salinity and submergence stresses together (3952) when compared with up-regulated genes (2152 for submergence and 1835 for salt + submergence) (Fig. <a reveal-id="DST042F2" data-open="DST042F2" class="link link-reveal link-table xref-fig">2</a>A). </p>                        <a id="4921322" scrollto-destination="4921322"></a>
<div id="DST042F2" data-id="DST042F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04202.png?Expires=1509238923&Signature=Ur9ajGyCSyHgPFQAvB1kInfYQjVDVzpVdKxmN9h9yL6EzaipNE3FnWqxN~tNOqnq095wYtIJxHztObZZORZkZTLeG8Tu5At7XEW8nqrdZTyGsfBcFuAeQ4pAEU6pUgUMKzCRzt4nq8In-lm0Zm7~t0G7J76xg6fl7vrd-rE5Oh62rFoFgSbxpi78RaV0XHYUdRVIw1T2bkwjFHh-wm-td41U7bFE3y5Mxtja7a3IYGRfEIxKKf0F6OB83vm7Hdgiz2udckoip5JAcB2WkzM1PdSgr3izHC7BPl0Zsbmbe3V40J8CShKiZ5irjZaQ0t0ifVufHTpQv8~zQIuNY~6qKg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Differential gene expression under different stress conditions. (A) The number of up- and down-regulated genes under different stress conditions (salt450, salt700, submergence and salt + submergence) is shown in the bar graph. The total number of genes differentially expressed under each condition is given on the top of each bar. (B) Venn diagram showing the number of genes differentially expressed under specific or different stress conditions." path-from-xml="dst04202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921322/dst04202.tiff" path-from-xml="dst04202.tiff" target="_blank">View large</a><a section="4921322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04202.png?Expires=1509238923&Signature=RWNaOsotQvDuO8qV-scF0oa98Mok7kf0lNxsk1uP053p9Sk939ZVxWmqMNvMzE4UolgRaYkqmxcvZDF8yY3Go-lCzs3McVa4DeyYooiCQajo4aS4bPNE5Ypz1fyjKWs4NTz~cdt-AeZ2YR~hgW6CcJq7BcHvY6dIIZeBdaR98~sZLfxpgS6FzTlNyHxke7aY6N2VYi-FoOoIGLj8x5SPej8ZpVTVA7G2w7nyNDXW8G8PL5YhgyUnc-WG3VaWag4UI3xUovQG~JGg85T0-5JT-lBX2V2JEWo2KCqUhgKN9pMKNyDZvPiAJunmG3Ro858r7ePY5VTY5HTjgBSyGv3KiA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921322&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differential gene expression under different stress conditions. (A) The number of up- and down-regulated genes under different stress conditions (salt450, salt700, submergence and salt + submergence) is shown in the bar graph. The total number of genes differentially expressed under each condition is given on the top of each bar. (B) Venn diagram showing the number of genes differentially expressed under specific or different stress conditions.</p></div></div><div content-id="DST042F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04202.png?Expires=1509238923&Signature=Ur9ajGyCSyHgPFQAvB1kInfYQjVDVzpVdKxmN9h9yL6EzaipNE3FnWqxN~tNOqnq095wYtIJxHztObZZORZkZTLeG8Tu5At7XEW8nqrdZTyGsfBcFuAeQ4pAEU6pUgUMKzCRzt4nq8In-lm0Zm7~t0G7J76xg6fl7vrd-rE5Oh62rFoFgSbxpi78RaV0XHYUdRVIw1T2bkwjFHh-wm-td41U7bFE3y5Mxtja7a3IYGRfEIxKKf0F6OB83vm7Hdgiz2udckoip5JAcB2WkzM1PdSgr3izHC7BPl0Zsbmbe3V40J8CShKiZ5irjZaQ0t0ifVufHTpQv8~zQIuNY~6qKg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Differential gene expression under different stress conditions. (A) The number of up- and down-regulated genes under different stress conditions (salt450, salt700, submergence and salt + submergence) is shown in the bar graph. The total number of genes differentially expressed under each condition is given on the top of each bar. (B) Venn diagram showing the number of genes differentially expressed under specific or different stress conditions." path-from-xml="dst04202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921322/dst04202.tiff" path-from-xml="dst04202.tiff" target="_blank">View large</a><a section="4921322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04202.png?Expires=1509238923&Signature=RWNaOsotQvDuO8qV-scF0oa98Mok7kf0lNxsk1uP053p9Sk939ZVxWmqMNvMzE4UolgRaYkqmxcvZDF8yY3Go-lCzs3McVa4DeyYooiCQajo4aS4bPNE5Ypz1fyjKWs4NTz~cdt-AeZ2YR~hgW6CcJq7BcHvY6dIIZeBdaR98~sZLfxpgS6FzTlNyHxke7aY6N2VYi-FoOoIGLj8x5SPej8ZpVTVA7G2w7nyNDXW8G8PL5YhgyUnc-WG3VaWag4UI3xUovQG~JGg85T0-5JT-lBX2V2JEWo2KCqUhgKN9pMKNyDZvPiAJunmG3Ro858r7ePY5VTY5HTjgBSyGv3KiA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921322&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differential gene expression under different stress conditions. (A) The number of up- and down-regulated genes under different stress conditions (salt450, salt700, submergence and salt + submergence) is shown in the bar graph. The total number of genes differentially expressed under each condition is given on the top of each bar. (B) Venn diagram showing the number of genes differentially expressed under specific or different stress conditions.</p></div></div><p>Furthermore, we identified genes showing overlapping or specific response to different stress conditions as depicted in Fig. <a reveal-id="DST042F2" data-open="DST042F2" class="link link-reveal link-table xref-fig">2</a>B. Interestingly, a larger fraction of genes (69.5%) exhibited response to a specific stress condition. Other 4625 (30.5%) genes were responsive to more than one stress conditions. The largest number of genes (2555) was commonly regulated by submergence and salt + submergence stresses. Notably, most of them (99.6%) exhibited similar response (681 up-regulated and 1865 down-regulated) under both conditions. However, among the 790 genes commonly regulated by both high salinity and submergence stresses, most (78.2%) exhibited opposite responses (473 up-regulated under high salinity but down-regulated under submergence stress and 145 down-regulated under high salinity but up-regulated under submergence stress). Interestingly, however, 84% of the 757 genes commonly regulated by low salinity and submergence stresses exhibited similar response. Only 957 genes exhibited differential expression under both low and high salinity conditions, but most of them (94.6%) were regulated in the same direction. A very small fraction of genes (116) were differentially regulated under all the stress conditions analysed. Furthermore, we performed <em>K</em>-means clustering analysis and generated 10 clusters with different gene expression patterns across different stress conditions analysed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Among the clusters generated, we identified four groups of genes with distinct gene expression patterns (Fig. <a reveal-id="DST042F3" data-open="DST042F3" class="link link-reveal link-table xref-fig">3</a>). Group I genes were up-regulated under salinity and/or salt + submergence stress. Group II genes were down-regulated under submergence and salt + submergence stress. Group III genes were up-regulated under submergence and salt + submergence stress. Group IV genes were up-regulated under salinity stress. </p>                        <a id="4921324" scrollto-destination="4921324"></a>
<div id="DST042F3" data-id="DST042F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04203.png?Expires=1509238923&Signature=C452XlUD8xd7w01uthCW5IZ6BR0L64Rexl1pFeGfjbPQzsl7wvdgVQtL7nUDTpUx000YsdCsspBPjcKdN4G-VDA6pSPj6axTlhNSsghfCMZESDNZRgkm8-NzIOzTp1kXHObhNdQmUcTyGVzRolkB0swyNlsKWm4Xl-S02MfuzXMLuLTh6Cuy2MvQGF55C2m3Q7fEptIeCWAQEfICU~X4mIsasd4L3nZprIjTcfBZGVkrcCCGio7dB-MoxVvJ8cG-u0meekn6i2Gu7FqNcwBfAyAsjytvJhVJrrn4FQK5gF6k5sRcKoQPjZWwTU9OiTTRN2QmmLCiQgVGAwixe-MG8Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="K-means clustering of expression profiles of genes differentially expressed under different stress conditions. The clustering was performed on log2 fold change for each gene under different stress conditions when compared with under control condition. The genes exhibiting a similar pattern of expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] have been grouped together in four distinct groups. The scale at the bottom shows log2 fold change. A colour version is available online." path-from-xml="dst04203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921324/dst04203.tiff" path-from-xml="dst04203.tiff" target="_blank">View large</a><a section="4921324" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04203.png?Expires=1509238923&Signature=deq5xUdY6YWxFRHp43z89mJc87YMVHjA0EI5YMGqLY3Wvzk7KcGJqsm3OetVOdQmJPe9-2Msh5v5-VK7NgB-EH~bPJu17I35A5AxAmJ3zxNsC62HaHENxezD4xSjzTmwDcnLOsWUTK9S9LKbpPzI2xWj0D6XhO4xi~VcRdt9xnC3FQWMDCRQdKaPywdMQE4ara9aTd1r7svlcd0XWuhvdy0q2ArMBDlvabivvArbrOU8MqA6mTcfsDY8CTLvZ5MZ7QkoEJyx5QqaLDXWym5pV89SatV8sUikNoaejL-lrAyezI6OfFCU2lsTBLhirXsUQyHUjf6P~52GbceTc5k99g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921324&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>K</em>-means clustering of expression profiles of genes differentially expressed under different stress conditions. The clustering was performed on log<sub>2</sub> fold change for each gene under different stress conditions when compared with under control condition. The genes exhibiting a similar pattern of expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] have been grouped together in four distinct groups. The scale at the bottom shows log<sub>2</sub> fold change. A colour version is available online.</p></div></div><div content-id="DST042F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04203.png?Expires=1509238923&Signature=C452XlUD8xd7w01uthCW5IZ6BR0L64Rexl1pFeGfjbPQzsl7wvdgVQtL7nUDTpUx000YsdCsspBPjcKdN4G-VDA6pSPj6axTlhNSsghfCMZESDNZRgkm8-NzIOzTp1kXHObhNdQmUcTyGVzRolkB0swyNlsKWm4Xl-S02MfuzXMLuLTh6Cuy2MvQGF55C2m3Q7fEptIeCWAQEfICU~X4mIsasd4L3nZprIjTcfBZGVkrcCCGio7dB-MoxVvJ8cG-u0meekn6i2Gu7FqNcwBfAyAsjytvJhVJrrn4FQK5gF6k5sRcKoQPjZWwTU9OiTTRN2QmmLCiQgVGAwixe-MG8Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="K-means clustering of expression profiles of genes differentially expressed under different stress conditions. The clustering was performed on log2 fold change for each gene under different stress conditions when compared with under control condition. The genes exhibiting a similar pattern of expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] have been grouped together in four distinct groups. The scale at the bottom shows log2 fold change. A colour version is available online." path-from-xml="dst04203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921324/dst04203.tiff" path-from-xml="dst04203.tiff" target="_blank">View large</a><a section="4921324" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04203.png?Expires=1509238923&Signature=deq5xUdY6YWxFRHp43z89mJc87YMVHjA0EI5YMGqLY3Wvzk7KcGJqsm3OetVOdQmJPe9-2Msh5v5-VK7NgB-EH~bPJu17I35A5AxAmJ3zxNsC62HaHENxezD4xSjzTmwDcnLOsWUTK9S9LKbpPzI2xWj0D6XhO4xi~VcRdt9xnC3FQWMDCRQdKaPywdMQE4ara9aTd1r7svlcd0XWuhvdy0q2ArMBDlvabivvArbrOU8MqA6mTcfsDY8CTLvZ5MZ7QkoEJyx5QqaLDXWym5pV89SatV8sUikNoaejL-lrAyezI6OfFCU2lsTBLhirXsUQyHUjf6P~52GbceTc5k99g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921324&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>K</em>-means clustering of expression profiles of genes differentially expressed under different stress conditions. The clustering was performed on log<sub>2</sub> fold change for each gene under different stress conditions when compared with under control condition. The genes exhibiting a similar pattern of expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] have been grouped together in four distinct groups. The scale at the bottom shows log<sub>2</sub> fold change. A colour version is available online.</p></div></div>                            <h3 scrollto-destination=4921325 id="4921325" class="section-title" data-legacy-id=s3e>3.5. Stress response of transcription factor genes</h3>
<p>TFs are the regulatory proteins, which play a key role in stress responses. Remarkably, we detected the differential expression of at least 329 (∼12%) TF-encoding genes under different stress conditions in <em>Porteresia</em>. A heatmap showing the differential expression of genes belonging to different TF families is shown in Fig. <a reveal-id="DST042F4" data-open="DST042F4" class="link link-reveal link-table xref-fig">4</a>. At least one member of 55 TF families exhibited differential expression. Among the various families, the largest number (32) of NAC TF family genes was differentially expressed. The members of other TF families, including MYB/MYB-related (25), bZIP (25), bHLH (23), HSF (18) and WRKY (17), were also significantly represented in the differentially expressed genes. A total of 96, 118, 132 and 127 TF genes exhibited differential expression under low salinity, high salinity, submergence and salt + submergence stress conditions, respectively. Among the 118 TF genes differentially expressed under high salinity stress, 88 were up-regulated and 30 were down-regulated. The members of NAC, MYB/MYB-related and WRKY families were the most abundant among these genes. In contrast, among the 132 TF genes differentially expressed under submergence stress, 44 were up-regulated and 88 were down-regulated. The members of bHLH, bZIP, HSF, NAC and MYB/MYB-related families were most abundant among these genes. In addition, several zinc-finger TFs of different classes were also regulated under different stress conditions. The members of these TF families have been found to be regulated under salinity and submergence stresses in Arabidopsis, rice and halophytes.<sup><a reveal-id="DST042C33  DST042C35" data-open="DST042C33  DST042C35" class="link link-ref link-reveal xref-bibr">33–35</a></sup> Altogether, these data suggest the extensive regulation of transcription under salinity and submergence stress. </p>                        <a id="4921327" scrollto-destination="4921327"></a>
<div id="DST042F4" data-id="DST042F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04204.png?Expires=1509238923&Signature=VD0g8ZIDhhrKInx1OYGt4a-jfpk6CGlTktQQy9obaEYu6ODvl5c8Dxq-pDBcle8rwqezBFRcPfxVkoRhhXDWsLq6YPpsVHQ8KwgTWi9HWpjYg03jYOIHrjMXCnqKjjVa3XB7yQWvFrE~PsyajeJdJPBZp5dVTWVT9LvRIunVg9RaAg-nYKJ6DNrwds84kG9xmdG9PfdAU2U9ZhZYtqMfI3EvC4HIB2SGj8eRQNnFZG80ZTGHcEbh8EWoAoBXQZcdEZNkvR8-nmngi79hMAxsXSYYn6eHiopMvFw-yKeVJAPKSpkbgtgg7KE1juIQIYwGUsjDB9nc~yNA3qom3Ea1ag__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Differential expression of TF-encoding genes under different conditions. Various TF families showing differential expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are given on the right side. The scale at the bottom represents log2 fold change. A colour version is available online." path-from-xml="dst04204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921327/dst04204.tiff" path-from-xml="dst04204.tiff" target="_blank">View large</a><a section="4921327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04204.png?Expires=1509238923&Signature=Axj370ZkY7Gu9Mzx92qHDB24Fz4tJNrx6pMpNR-UxuyCHOwYe6M56mWFfhVP6hm9vFlsFh5zSeTEmlrqfCEg~4ayO8tmtXD-SjbfreSnDWSmdOVv4lhtksCfwPm9GunQTOIH3dCBhLSO40B8cHKJddr0tRvHXoZ7xuOCU7wpJ32h47OGu1r4exk~jWgo6Amm4cn8PJl9cVY~Y1RP3vqHqDoFy6m70hDbOOqmsqc24LuPXbUSoZEUyQimvruFMhc~3wH6sPK-Qy1sLLpkjQEKBmitKB~6KURn1IOhk2YJbNekzLpPUXglXfdqq4fyH-XWF6ytAhGxkAsk2kt14EMMqQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921327&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differential expression of TF-encoding genes under different conditions. Various TF families showing differential expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are given on the right side. The scale at the bottom represents log<sub>2</sub> fold change. A colour version is available online.</p></div></div><div content-id="DST042F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04204.png?Expires=1509238923&Signature=VD0g8ZIDhhrKInx1OYGt4a-jfpk6CGlTktQQy9obaEYu6ODvl5c8Dxq-pDBcle8rwqezBFRcPfxVkoRhhXDWsLq6YPpsVHQ8KwgTWi9HWpjYg03jYOIHrjMXCnqKjjVa3XB7yQWvFrE~PsyajeJdJPBZp5dVTWVT9LvRIunVg9RaAg-nYKJ6DNrwds84kG9xmdG9PfdAU2U9ZhZYtqMfI3EvC4HIB2SGj8eRQNnFZG80ZTGHcEbh8EWoAoBXQZcdEZNkvR8-nmngi79hMAxsXSYYn6eHiopMvFw-yKeVJAPKSpkbgtgg7KE1juIQIYwGUsjDB9nc~yNA3qom3Ea1ag__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Differential expression of TF-encoding genes under different conditions. Various TF families showing differential expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are given on the right side. The scale at the bottom represents log2 fold change. A colour version is available online." path-from-xml="dst04204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921327/dst04204.tiff" path-from-xml="dst04204.tiff" target="_blank">View large</a><a section="4921327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04204.png?Expires=1509238923&Signature=Axj370ZkY7Gu9Mzx92qHDB24Fz4tJNrx6pMpNR-UxuyCHOwYe6M56mWFfhVP6hm9vFlsFh5zSeTEmlrqfCEg~4ayO8tmtXD-SjbfreSnDWSmdOVv4lhtksCfwPm9GunQTOIH3dCBhLSO40B8cHKJddr0tRvHXoZ7xuOCU7wpJ32h47OGu1r4exk~jWgo6Amm4cn8PJl9cVY~Y1RP3vqHqDoFy6m70hDbOOqmsqc24LuPXbUSoZEUyQimvruFMhc~3wH6sPK-Qy1sLLpkjQEKBmitKB~6KURn1IOhk2YJbNekzLpPUXglXfdqq4fyH-XWF6ytAhGxkAsk2kt14EMMqQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921327&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Differential expression of TF-encoding genes under different conditions. Various TF families showing differential expression under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are given on the right side. The scale at the bottom represents log<sub>2</sub> fold change. A colour version is available online.</p></div></div>                            <h3 scrollto-destination=4921328 id="4921328" class="section-title" data-legacy-id=s3f>3.6. Gene ontology and metabolic pathway enrichment analysis of stress-responsive genes</h3>
<p>To investigate the major functional categories represented in the stress-responsive genes identified above, we performed GO enrichment analysis. Most evidently, besides response to stress, we found the GO terms associated with various metabolic processes, such as secondary metabolite (oxoacid, ketone, amino acid, oxylipin, flavonoid, phenylpropanoid and chorismate) biosynthesis and hormone (jasmonic acid, ethylene and abscisic acid, ABA) biosynthesis, were most enriched in the genes up-regulated under different stress conditions analysed (Fig. <a reveal-id="DST042F5" data-open="DST042F5" class="link link-reveal link-table xref-fig">5</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). In addition, the GO terms, photosynthesis, photorespiration, rRNA processing and transport were also found enriched in up-regulated genes. However, in the down-regulated genes under salinity stress, the GO terms associated with polysaccharide (glucan, carbohydrate and starch) metabolism were most significantly represented (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S5 and S6</a>). The genes with function in protein turn over (proteolysis, peptidase activity and proteasome assembly) were highly enriched under low salinity. Such genes have been found responsible for salinity stress adaptation in halophyte, <em>T. halophila</em>.<sup><a reveal-id="DST042C33" data-open="DST042C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Under submergence alone or along with salinity stress, the GO terms response to various stresses, response to ABA, generation of precursor metabolites and energy as well as rRNA metabolism were most enriched. The finding of up-regulation of genes associated with energy conservation via metabolism rearrangement to enhance ATP production and/or nullifying deleterious effects of oxygen radicals, reactive oxygen species and other toxic compounds under submergence has been reported earlier also.<sup><a reveal-id="DST042C36" data-open="DST042C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DST042C37" data-open="DST042C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In addition, the GO terms, such as response to gibberellin stimulus, response to hydrogen per oxide and transport, were represented in the down-regulated genes under submergence stress (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S5 and S6</a>). The GO term, regulation of transcription, was also found enriched in the stress-responsive genes, which is consistent with differential expression of the large number of TF genes as described in previous section. Notably, we noticed the significant enrichment of cellular component GO term, cell wall, in the differentially expressed genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). The changes in transcript levels of a number of genes encoding cell wall-related proteins have been observed in halophyte, <em>T. halophila</em>, in response to abiotic stress conditions.<sup><a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST042C38" data-open="DST042C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Overall, the different groups of genes with distinct expression patterns were also found to be associated with similar GO terms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). For instance, the GO terms, response to stress and hormones were enriched in Group II genes, whereas photosynthesis and secondary metabolite biosynthesis terms were most significant in Group III genes. Group IV genes were found enriched in tryptophan biosynthesis, cellular amino acid derivatives biosynthesis, response to jasmonic acid and cell death-related GO terms. </p>                        <a id="4921330" scrollto-destination="4921330"></a>
<div id="DST042F5" data-id="DST042F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04205.png?Expires=1509238923&Signature=IE~b4~tX-CwoK8wwWsgqLborY7hBlgaSEC5urzEdGVk7rHn83abZieXwH0cQ209aiDl0ROscrl-FLQUg7nPLVYLDgNDAwKPNG7S-RzoOKx4waYRuQfQtAEj3M0YNVeh-VkfDgp6LpuUg2H-3PrdXMGR-Dg-5573IJ7Dh2Fu7-wWYANUA89E1-ZcHitWYP6dRUy4yM6DxJrvv~1V~epwJfQNou5ymcM63V8LoCyZqEISCGL5SLqaaOnHHJDsXRD6YehFpQdjnZewufEz3W5WlOnTZp9a2EDZUa~155nRvij189PL5hM8kmP72CqOu2H5El5r2tOcJQKYfAlH7d~81cA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="GO enrichment under different stress conditions. Significantly, enriched GO categories in the up-regulated genes under high salinity (A) and submergence (B) stress are shown. The genes were analysed using BiNGO and the biological process terms showing significant enrichment are shown. Node size is proportional to the number of genes in each category and shades represent the significance level (white—no significant difference; scale, P = 0.05 to P &lt; 0.0000005). A colour version is available online." path-from-xml="dst04205.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921330/dst04205.tiff" path-from-xml="dst04205.tiff" target="_blank">View large</a><a section="4921330" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04205.png?Expires=1509238923&Signature=S6uP~yV8M~MsWGMdYRZ8BG7uiqAYnNkGjEZUacEtz3il6h6LeIiS9EJt1rLo3NMN5bC7GRx8XrJcdm5Q7iTcTmyPfAJ3JoXetJElcbINAPPwFAgCLghPFxV0j-mvDeVDW4vhy7POKbYzxL33PQqtzOvFZEXgsoSP0RwlQndfEofdTkoHJycUFmOYxe5UvifYqp0tPf1kFZIa0KttzHB0Kv0Cfyx3x2C3JVwUj9hQF6nepKYoAAkn1CHVmDjSNQhvm6NCY2X7WLgrXuGJ-t3w9HUSQraAXbs0z1UKvjOAYwSjSjT6sOe18uSf6IuWhdph~YDTuHrQKdVVPzgfSm-etQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921330&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04205.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO enrichment under different stress conditions. Significantly, enriched GO categories in the up-regulated genes under high salinity (A) and submergence (B) stress are shown. The genes were analysed using BiNGO and the biological process terms showing significant enrichment are shown. Node size is proportional to the number of genes in each category and shades represent the significance level (white—no significant difference; scale, <em>P</em> = 0.05 to <em>P</em> &lt; 0.0000005). A colour version is available online.</p></div></div><div content-id="DST042F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04205.png?Expires=1509238923&Signature=IE~b4~tX-CwoK8wwWsgqLborY7hBlgaSEC5urzEdGVk7rHn83abZieXwH0cQ209aiDl0ROscrl-FLQUg7nPLVYLDgNDAwKPNG7S-RzoOKx4waYRuQfQtAEj3M0YNVeh-VkfDgp6LpuUg2H-3PrdXMGR-Dg-5573IJ7Dh2Fu7-wWYANUA89E1-ZcHitWYP6dRUy4yM6DxJrvv~1V~epwJfQNou5ymcM63V8LoCyZqEISCGL5SLqaaOnHHJDsXRD6YehFpQdjnZewufEz3W5WlOnTZp9a2EDZUa~155nRvij189PL5hM8kmP72CqOu2H5El5r2tOcJQKYfAlH7d~81cA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="GO enrichment under different stress conditions. Significantly, enriched GO categories in the up-regulated genes under high salinity (A) and submergence (B) stress are shown. The genes were analysed using BiNGO and the biological process terms showing significant enrichment are shown. Node size is proportional to the number of genes in each category and shades represent the significance level (white—no significant difference; scale, P = 0.05 to P &lt; 0.0000005). A colour version is available online." path-from-xml="dst04205.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921330/dst04205.tiff" path-from-xml="dst04205.tiff" target="_blank">View large</a><a section="4921330" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04205.png?Expires=1509238923&Signature=S6uP~yV8M~MsWGMdYRZ8BG7uiqAYnNkGjEZUacEtz3il6h6LeIiS9EJt1rLo3NMN5bC7GRx8XrJcdm5Q7iTcTmyPfAJ3JoXetJElcbINAPPwFAgCLghPFxV0j-mvDeVDW4vhy7POKbYzxL33PQqtzOvFZEXgsoSP0RwlQndfEofdTkoHJycUFmOYxe5UvifYqp0tPf1kFZIa0KttzHB0Kv0Cfyx3x2C3JVwUj9hQF6nepKYoAAkn1CHVmDjSNQhvm6NCY2X7WLgrXuGJ-t3w9HUSQraAXbs0z1UKvjOAYwSjSjT6sOe18uSf6IuWhdph~YDTuHrQKdVVPzgfSm-etQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921330&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04205.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO enrichment under different stress conditions. Significantly, enriched GO categories in the up-regulated genes under high salinity (A) and submergence (B) stress are shown. The genes were analysed using BiNGO and the biological process terms showing significant enrichment are shown. Node size is proportional to the number of genes in each category and shades represent the significance level (white—no significant difference; scale, <em>P</em> = 0.05 to <em>P</em> &lt; 0.0000005). A colour version is available online.</p></div></div><p>Many of the stress-responsive genes identified above encode for the enzymes involved in several metabolic pathways, which regulate diverse cellular processes. Therefore, we investigated the metabolic pathways significantly enriched in stress-responsive genes by integrating the stress-responsive gene set with metabolic pathway data available in the RiceCyc database. The pathways involved in amino acids biosynthesis, secondary metabolite biosynthesis, hormone biosynthesis, carbohydrate metabolism and cell wall structures were significantly overrepresented among stress-responsive genes (Fig. <a reveal-id="DST042F6" data-open="DST042F6" class="link link-reveal link-table xref-fig">6</a>), which are in well agreement with GO enrichment analysis. A list of pathways affected under stress conditions are given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>. Most evidently, the genes involved in the biosynthesis of various amino acids, including tryptophan, valine, isoleucine, glycine, glutamate and serine, were significantly represented in stress-responsive genes. The pathways for various stress-related hormone biosynthesis, such as jasmonic acid, ethylene and ABA biosynthesis, were also enriched. The genes involved in jasmonic acid biosynthesis were up-regulated under salinity stress, whereas genes for ABA biosynthesis were up-regulated under salinity stress, but down-regulated under submergence stress (Fig. <a reveal-id="DST042F6" data-open="DST042F6" class="link link-reveal link-table xref-fig">6</a>). The pathways involved in carbohydrate metabolism, such as starch degradation, trehalose biosynthesis and anaerobic respiration, were also overrepresented among the stress-responsive genes. The genes involved in anaerobic respiration were significantly up-regulated under submergence stress alone and along with salinity stress. The genes involved in anaerobic metabolism have been found regulated in response to submergence, flooding and O<sub>2</sub> deprivation across the plant species too.<sup><a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST042C39" data-open="DST042C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Notably, the genes involved in cell wall structure biosynthesis, such as cellulose and suberin biosynthesis, were also significantly enriched. The pathways involved in serotonin amides biosynthesis, phenylpropanoid biosynthesis, phospholipase activity and fatty acid derivative biosynthesis were also found enriched. A similar set of metabolic pathways were significantly represented in the different groups of genes with distinct expression patterns (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). </p>                        <a id="4921332" scrollto-destination="4921332"></a>
<div id="DST042F6" data-id="DST042F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04206.png?Expires=1509238923&Signature=fgd8de3HX0O2gAjSlisSD2XpCMs5lHSl25gPilph9T3MNsbbXD6jCYADYTK--qg0XV~UL2XOP~Bv7eO~tisGLvSBKrOlnq~hy9mCdwhhKddM8mhIGYtkDJkxR~FIU7HIz65aeEDLl0crpcFJj9iRa90WGGmh~2Lf9UowsRYjbyxJBsMi2jLsUccicHbW3W8d7lu1OueyzF-jGlem8kn7EqCmkc8tf4Z3me62ft0K63FIqHOb7XsoC~tVr5fmOUOJrkuOGu5tkCBlzPXwwUMnAQbML7VgQTqfsyhP8skiN4hPY-WeIQ01zhqetwtDbJT-gcdfE-wllUmJfuJ1kMQbhw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Regulation of metabolic pathways during salinity and submergence stress conditions. The metabolic pathways enriched in differentially expressed genes are shown. Heatmaps showing the expression profiles of genes involved in these pathways under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are also shown. The scale at the bottom represents log2 fold change. A colour version is available online." path-from-xml="dst04206.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921332/dst04206.tiff" path-from-xml="dst04206.tiff" target="_blank">View large</a><a section="4921332" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04206.png?Expires=1509238923&Signature=TZKpXXJLSHu2UsIVUi3YLfqFKAiR54p2wnnD-JqBMlXrx2mn~djYXvV91CMuFCQBwTvBx8X4XIzskDDsdgB9TBIi8h1cr6~C1iSRXgFbUjqU0dBzX8s8ovRMETcTygqVbvOB5XBW2tCANkQKaPni4rAJD0ZYTDbGMEW~RTP2YWMlEm8cv4CLDtLR6mn4QGXbR6y-fT0Mz1xxe0PN3buSmD-1oBG0tzErKln155TSCV040JKOIErE3zQUJJyKjAjXGSvq8F6ZPIUQSp79tHgSZcxjq3la3-umq4LUndqEldqEqZwWxdIbDZpcs9KdOh6VW7mVbvL-CXNwTjiKnyvzkQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921332&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04206.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Regulation of metabolic pathways during salinity and submergence stress conditions. The metabolic pathways enriched in differentially expressed genes are shown. Heatmaps showing the expression profiles of genes involved in these pathways under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are also shown. The scale at the bottom represents log<sub>2</sub> fold change. A colour version is available online.</p></div></div><div content-id="DST042F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04206.png?Expires=1509238923&Signature=fgd8de3HX0O2gAjSlisSD2XpCMs5lHSl25gPilph9T3MNsbbXD6jCYADYTK--qg0XV~UL2XOP~Bv7eO~tisGLvSBKrOlnq~hy9mCdwhhKddM8mhIGYtkDJkxR~FIU7HIz65aeEDLl0crpcFJj9iRa90WGGmh~2Lf9UowsRYjbyxJBsMi2jLsUccicHbW3W8d7lu1OueyzF-jGlem8kn7EqCmkc8tf4Z3me62ft0K63FIqHOb7XsoC~tVr5fmOUOJrkuOGu5tkCBlzPXwwUMnAQbML7VgQTqfsyhP8skiN4hPY-WeIQ01zhqetwtDbJT-gcdfE-wllUmJfuJ1kMQbhw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Regulation of metabolic pathways during salinity and submergence stress conditions. The metabolic pathways enriched in differentially expressed genes are shown. Heatmaps showing the expression profiles of genes involved in these pathways under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are also shown. The scale at the bottom represents log2 fold change. A colour version is available online." path-from-xml="dst04206.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921332/dst04206.tiff" path-from-xml="dst04206.tiff" target="_blank">View large</a><a section="4921332" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04206.png?Expires=1509238923&Signature=TZKpXXJLSHu2UsIVUi3YLfqFKAiR54p2wnnD-JqBMlXrx2mn~djYXvV91CMuFCQBwTvBx8X4XIzskDDsdgB9TBIi8h1cr6~C1iSRXgFbUjqU0dBzX8s8ovRMETcTygqVbvOB5XBW2tCANkQKaPni4rAJD0ZYTDbGMEW~RTP2YWMlEm8cv4CLDtLR6mn4QGXbR6y-fT0Mz1xxe0PN3buSmD-1oBG0tzErKln155TSCV040JKOIErE3zQUJJyKjAjXGSvq8F6ZPIUQSp79tHgSZcxjq3la3-umq4LUndqEldqEqZwWxdIbDZpcs9KdOh6VW7mVbvL-CXNwTjiKnyvzkQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921332&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04206.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Regulation of metabolic pathways during salinity and submergence stress conditions. The metabolic pathways enriched in differentially expressed genes are shown. Heatmaps showing the expression profiles of genes involved in these pathways under different stress conditions [salt450, salt700, submergence (sub) and salt + submergence (salt + sub)] are also shown. The scale at the bottom represents log<sub>2</sub> fold change. A colour version is available online.</p></div></div><p>Interestingly, we detected the differential expression of several genes in <em>Porteresia</em> implicated in <em>SUB1A-</em>mediated submergence tolerance response.<sup><a reveal-id="DST042C40" data-open="DST042C40" class="link link-ref link-reveal xref-bibr">40</a></sup> In addition to the genes involved in ethylene response, carbohydrate metabolism, photorespiration and anaerobic respiration, we found putative gibberellin biosynthesis genes regulated under submergence stress, which mediates shoot elongation. The expression of <em>Alcohol Dehydrogenase</em>, a marker gene for anoxia stress, was also up-regulated under submergence stress, consistent with previous findings.<sup><a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST042C40  DST042C42" data-open="DST042C40  DST042C42" class="link link-ref link-reveal xref-bibr">40–42</a></sup> We detected differential expression of several putative expansin genes as well under submergence and/or salt + submergence stress. The expansins have been shown to play a major role in cell expansion under anoxic condition, prevalent under submergence stress.<sup><a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST042C42" data-open="DST042C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DST042C43" data-open="DST042C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p>                            <h3 scrollto-destination=4921334 id="4921334" class="section-title" data-legacy-id=s3g>3.7. Real-time PCR validation of differential gene expression</h3>
<p>We validated the results of differential gene expression analysis obtained from RNA-seq data by real-time RT–PCR analysis. The expression of at least 10 randomly selected genes was analysed in the three biological replicates of five tissue samples representing different conditions. The analysis revealed similar expression pattern of all the selected genes in real-time PCR analysis as observed from RNA-seq data. The statistical analysis also showed very good correspondence (correlation coefficient of 0.792) among the results of real-time PCR and RNA-seq data analyses as shown in Fig. <a reveal-id="DST042F7" data-open="DST042F7" class="link link-reveal link-table xref-fig">7</a>. </p>                        <a id="4921336" scrollto-destination="4921336"></a>
<div id="DST042F7" data-id="DST042F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04207.png?Expires=1509238923&Signature=Ez2YdVRUNAA~iaqUYapSkDdNsjW3bJ2Lrq37VqTyvRKbJ-y2Rx3vvtx7jYpc5R7QMUU74Til4--T944i2Fz5fDrbtEvtWuCTBCqjrc8WeAsOeOnbZWau9-qMloMj0FpdHQpVtD2e7-wwrZiHbaPWm3YBw2etnmTpGaR7wDZMDMY2iJSI4dof9tXV-Q2aVz2EhJ04Mwvrj6dHXd8Uca8~eCx32aGu2yAPyLxX3GhblRH5MwKoPvC4qFLInMPU-wx8Aw0TdAw8lWKwUDLycywCIEzz0wVcyFXUEZP~GyiVGFKZKqPaXaBX8668Ie82ZNlfGjG~it~0XtWiXGHlGekorw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Validation of differential gene expression results obtained by RNA-seq. The correlation of gene expression results obtained from real-time PCR analysis and RNA-seq for 10 selected genes is shown." path-from-xml="dst04207.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921336/dst04207.tiff" path-from-xml="dst04207.tiff" target="_blank">View large</a><a section="4921336" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04207.png?Expires=1509238923&Signature=ZpB-qToj5V7hkhahRjOQ2-U7UcHjWmjYyYgqKTaUHtHV5Cy4VbonH2pJusBhtauoBH9kGvGyT559btoSZ2wfFCq7RylpgfSBI9qIRJ~H-gI68DoZJwI0upEh1s93f92qg7jrKeDfGdPUACNv9TIjY286~q9pY~TfoBdxp6zkmdN0CmjAzoMCykJD1EpegonmxDPkSY67HJ9bLhj2GcF0RFN-4wwjaahGAg2EJR~H~sLiCu9rRqZlOcjQ-empVI5M3ypMc6nOwlxDewf5S8zfbJJJdf-ylr4L9gBGoUFR4YthyJjxgHdAZ5X~EbmcwmpXKH5VBmyHzjQTN9u7jiQtzw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921336&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04207.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of differential gene expression results obtained by RNA-seq. The correlation of gene expression results obtained from real-time PCR analysis and RNA-seq for 10 selected genes is shown.</p></div></div><div content-id="DST042F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/m_dst04207.png?Expires=1509238923&Signature=Ez2YdVRUNAA~iaqUYapSkDdNsjW3bJ2Lrq37VqTyvRKbJ-y2Rx3vvtx7jYpc5R7QMUU74Til4--T944i2Fz5fDrbtEvtWuCTBCqjrc8WeAsOeOnbZWau9-qMloMj0FpdHQpVtD2e7-wwrZiHbaPWm3YBw2etnmTpGaR7wDZMDMY2iJSI4dof9tXV-Q2aVz2EhJ04Mwvrj6dHXd8Uca8~eCx32aGu2yAPyLxX3GhblRH5MwKoPvC4qFLInMPU-wx8Aw0TdAw8lWKwUDLycywCIEzz0wVcyFXUEZP~GyiVGFKZKqPaXaBX8668Ie82ZNlfGjG~it~0XtWiXGHlGekorw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Validation of differential gene expression results obtained by RNA-seq. The correlation of gene expression results obtained from real-time PCR analysis and RNA-seq for 10 selected genes is shown." path-from-xml="dst04207.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4921336/dst04207.tiff" path-from-xml="dst04207.tiff" target="_blank">View large</a><a section="4921336" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst04207.png?Expires=1509238923&Signature=ZpB-qToj5V7hkhahRjOQ2-U7UcHjWmjYyYgqKTaUHtHV5Cy4VbonH2pJusBhtauoBH9kGvGyT559btoSZ2wfFCq7RylpgfSBI9qIRJ~H-gI68DoZJwI0upEh1s93f92qg7jrKeDfGdPUACNv9TIjY286~q9pY~TfoBdxp6zkmdN0CmjAzoMCykJD1EpegonmxDPkSY67HJ9bLhj2GcF0RFN-4wwjaahGAg2EJR~H~sLiCu9rRqZlOcjQ-empVI5M3ypMc6nOwlxDewf5S8zfbJJJdf-ylr4L9gBGoUFR4YthyJjxgHdAZ5X~EbmcwmpXKH5VBmyHzjQTN9u7jiQtzw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4921336&ar=347526&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04207.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of differential gene expression results obtained by RNA-seq. The correlation of gene expression results obtained from real-time PCR analysis and RNA-seq for 10 selected genes is shown.</p></div></div>                            <h2 scrollto-destination=4921337 id="4921337" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Over the past many years, considerable efforts have been made to understand the molecular mechanism of salinity tolerance/adaptation in plants. Several studies have reported the molecular responses of plants to salinity stress.<sup><a reveal-id="DST042C1  DST042C4" data-open="DST042C1  DST042C4" class="link link-ref link-reveal xref-bibr">1–4</a>,<a reveal-id="DST042C34" data-open="DST042C34" class="link link-ref link-reveal xref-bibr">34</a></sup> However, only a few studies have been performed to analyse the submergence response.<sup><a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST042C36" data-open="DST042C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DST042C40" data-open="DST042C40" class="link link-ref link-reveal xref-bibr">40</a></sup> In the recent years, study of wild halophyte relatives has gained attention as models to investigate the mechanism of salinity and submergence tolerance due to their extreme tolerance/adaptability to these conditions.<sup><a reveal-id="DST042C5" data-open="DST042C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST042C6" data-open="DST042C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, most of such studies are confined to close relatives of Arabidopsis, <em>T. halophila</em> and <em>T. parvula</em>.<sup><a reveal-id="DST042C14  DST042C16" data-open="DST042C14  DST042C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> Recently, the transcriptome of a salinity-tolerant wild tomato species has also been reported.<sup><a reveal-id="DST042C44" data-open="DST042C44" class="link link-ref link-reveal xref-bibr">44</a></sup> However, considering the complexity of salinity response and fundamental differences in the physiology of monocots and dicots, these dicot halophytes are not ideal models to understand the mechanism of salinity tolerance in monocot crop plants. Recently, the importance of <em>P. coarctata</em>, wild halophyte relative rice, has been recognized as a model to study stress tolerance in monocots and important source of genes for the improvement of rice via biotechnological approaches.<sup><a reveal-id="DST042C8" data-open="DST042C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>In the present study, we sequenced the transcriptome of <em>Porteresia</em> under different stress conditions using Illumina technology to explore its gene complement. Unexpectedly, <em>Porteresia</em> transcripts exhibited low similarity with rice proteins, suggesting significant differences between <em>Porteresia</em> and rice genomes. The similarity of <em>Porteresia</em> genes with rice was lower than that of other sequenced wild rice species.<sup><a reveal-id="DST042C45" data-open="DST042C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DST042C46" data-open="DST042C46" class="link link-ref link-reveal xref-bibr">46</a></sup> A substantial number of genes appeared specific to <em>Porteresia</em> because they do not have an ortholog in rice and other organisms as predicted based on the sequence similarity. Furthermore, we observed expansion or contraction of certain TF families in <em>Porteresia</em> vis-a-vis rice. This unique gene complement might provide means of stress adaptation in <em>Porteresia</em>. We identified a total of 15 158 genes responsive to salinity and/or submergence stress, which have been implicated in diverse biological processes and metabolic pathways. The genes involved in similar biological processes, such as photosynthesis, secondary metabolite biosynthesis, hormone biosynthesis, carbohydrate metabolism, cell wall-related enzymes and transport etc., have been implicated in salinity and/or submergence stress in Arabidopsis, rice and halophytes as well.<sup><a reveal-id="DST042C33" data-open="DST042C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST042C38" data-open="DST042C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DST042C40" data-open="DST042C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST042C41" data-open="DST042C41" class="link link-ref link-reveal xref-bibr">41</a></sup> A significant number of genes of unknown function were also differentially expressed under various stress conditions analysed, and many of these genes were presumably specific to <em>Porteresia</em>, which might also play a crucial role in stress adaptation.</p><p>Several studies have demonstrated that TFs act as key regulators of plant response to environmental stimuli.<sup><a reveal-id="DST042C47  DST042C49" data-open="DST042C47  DST042C49" class="link link-ref link-reveal xref-bibr">47–49</a></sup> The role of certain TF families, such as AP2-EREBP, MYB and NAC, in stress responses has been well established. We also observed the differential expression of members of these TF families under salinity and submergence stresses in <em>Porteresia</em>. We detected that most of TF genes differentially expressed under salinity stress are up-regulated, whereas most of differentially expressed TF genes under submergence stress are down-regulated. This is in accordance with the total number of genes differentially expressed showing up- and down-regulation under salinity and submergence stresses. These results suggest that overall transcriptional activity is induced under salinity stress, but repressed under submergence stress. Based on the analyses, we found that role of NAC, MYB and WRKY family TFs may be more crucial for salinity tolerance, whereas bZIP, bHLH, HSF and AP2-EREBP families play a major role during submergence stress. The role of <em>SUBMERGENCE1</em> (<em>SUB1</em>) QTL in submergence tolerance is well known, which harbour a tandem cluster of AP2/ERF family genes.<sup><a reveal-id="DST042C40" data-open="DST042C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST042C41" data-open="DST042C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST042C50" data-open="DST042C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DST042C51" data-open="DST042C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Recently, Niroula <em>et al.</em><sup><a reveal-id="DST042C52" data-open="DST042C52" class="link link-ref link-reveal xref-bibr">52</a></sup> proposed the existence of SUB1-dependent and -independent mechanisms involved in submergence tolerance in wild rice plants. However, further investigations are required to find the exact mechanism of submergence tolerance in <em>Porteresia</em>.</p><p>Metabolic pathways maintain cellular homeostasis and are very important for imparting stress tolerance/adaptation in plants. We found significant enrichment of genes encoding key enzymes involved in important metabolic pathways under stress conditions in <em>Porteresia</em>. The production of secondary metabolites under stress condition has been well known as adaptive feature of plants. We also detected the enrichment of biosynthetic genes of certain secondary metabolites, such as hydroxycinnamic acid serotonin amides and phenylpropanoids etc., under stress conditions in <em>Porteresia</em>. Hydroxycinnamic acid serotonin amides have only been detected in rice, pepper and a few other plants so far and are supposed to act as antioxidants.<sup><a reveal-id="DST042C53" data-open="DST042C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Although their exact function is unclear, their antioxidant property may help encounter/adapt stress condition in <em>Porteresia</em>. Phenylpropanoid biosynthesis pathway of initial reactions (cinnamates biosynthesis) produces coumarate, which gives rise to a variety of secondary compounds such as flavonoids, stilbenes and lignin, which can help in protection against stress conditions.</p><p>The plant hormones, ethylene and ABA are well known to be involved in adaptive response to environmental stresses. Jasmonic acid has primarily been implicated in defence response, but is involved in other stress responses as well.<sup><a reveal-id="DST042C54" data-open="DST042C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Furthermore, the biosynthesis of several secondary metabolites by jasmonates has been well documented,<sup><a reveal-id="DST042C55  DST042C57" data-open="DST042C55  DST042C57" class="link link-ref link-reveal xref-bibr">55–57</a></sup> suggesting that biosynthesis of jasmonates may impart stress tolerance in plants via transcriptional activation of biosynthetic pathway of important secondary metabolites. The enzymes involved in cyclic methionine salvage pathway, which ensures ethylene biosynthesis via methionine recycling, were enriched under salinity stress.<sup><a reveal-id="DST042C58" data-open="DST042C58" class="link link-ref link-reveal xref-bibr">58</a></sup> We observed up-regulation of ethylene biosynthesis genes under salinity and submergence stresses. Earlier reports have also shown the altered ethylene levels underwater, which activates genes associated with stress acclimation.<sup><a reveal-id="DST042C41" data-open="DST042C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST042C50" data-open="DST042C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DST042C59" data-open="DST042C59" class="link link-ref link-reveal xref-bibr">59</a></sup> ABA is a well-known stress implicated in plant stress responses, including salinity stress.<sup><a reveal-id="DST042C60  DST042C63" data-open="DST042C60  DST042C63" class="link link-ref link-reveal xref-bibr">60–63</a></sup> The increased accumulation of ABA under salinity stress has been reported, which, in turn, activates several other genes to provide tolerance. However, under submergence stress, the levels of ABA are reduced.<sup><a reveal-id="DST042C64" data-open="DST042C64" class="link link-ref link-reveal xref-bibr">64</a></sup> We also observed the down-regulation of ABA biosynthesis genes under submergence stress. Recently, it has been proposed that <em>SUB1A</em> gene delays leaf senescence via hormonal regulation, which can contribute to enhanced stress tolerance in rice.<sup><a reveal-id="DST042C59" data-open="DST042C59" class="link link-ref link-reveal xref-bibr">59</a></sup></p><p>The enrichment of various pathways related to carbohydrate metabolism, such as starch degradation, anaerobic respiration and trehalose biosynthesis, is in concordance with earlier reports, demonstrating that concentrations of soluble sugars increases under various abiotic stress conditions.<sup><a reveal-id="DST042C65" data-open="DST042C65" class="link link-ref link-reveal xref-bibr">65</a>,<a reveal-id="DST042C66" data-open="DST042C66" class="link link-ref link-reveal xref-bibr">66</a></sup> The differential expression of genes related to carbohydrate metabolism in response to various abiotic stresses has been reported previously also.<sup><a reveal-id="DST042C67" data-open="DST042C67" class="link link-ref link-reveal xref-bibr">67</a></sup> The sugars, sucrose and glucose either act as substrates for cellular respiration or act as osmolytes to maintain cell homeostasis.<sup><a reveal-id="DST042C68" data-open="DST042C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DST042C69" data-open="DST042C69" class="link link-ref link-reveal xref-bibr">69</a></sup> In fact, one of the present hypotheses for survival of rice under submergence stress is based on carbohydrate metabolism only.<sup><a reveal-id="DST042C41" data-open="DST042C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST042C51" data-open="DST042C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DST042C70" data-open="DST042C70" class="link link-ref link-reveal xref-bibr">70</a></sup> Trehalose has been proposed as osmoprotectant against various stresses. It has been demonstrated that accumulation of trehalose in rice plants results in increased soluble sugar content, which confers tolerance to different abiotic stresses.<sup><a reveal-id="DST042C71" data-open="DST042C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DST042C72" data-open="DST042C72" class="link link-ref link-reveal xref-bibr">72</a></sup> Furthermore, the elevated transcript levels of the genes associated with anaerobic respiration can allow plant cells to invoke the energy conserving mode of ATP production under submergence stress in <em>Porteresia</em> as speculated earlier.<sup><a reveal-id="DST042C12" data-open="DST042C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST042C35" data-open="DST042C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Recently, Miro and Ismail<sup><a reveal-id="DST042C12" data-open="DST042C12" class="link link-ref link-reveal xref-bibr">12</a></sup> reviewed anaerobic respiration and starch degradation as the main traits associated with flooding tolerance in rice genotypes.</p><p>The biosynthetic pathways for suberin and cellulose, major constituents of cell wall, were also found enriched in stress-responsive genes in <em>Porteresia</em>. Although the exact mechanism is not known, several studies have suggested a crucial role of cell wall in stress tolerance.<sup><a reveal-id="DST042C73" data-open="DST042C73" class="link link-ref link-reveal xref-bibr">73</a></sup> The mutations in cellulose synthase genes have also been shown to alter stress tolerance in plants.<sup><a reveal-id="DST042C74" data-open="DST042C74" class="link link-ref link-reveal xref-bibr">74</a>,<a reveal-id="DST042C75" data-open="DST042C75" class="link link-ref link-reveal xref-bibr">75</a></sup> Suberin is a primary constituent of primary cell wall of underground tissues and is analogous to cutin, which forms a layer on plant aerial parts to prevent the direct contact of epidermal cell with environment and thus, acts as a physical barrier to enhance plant resistance to abiotic stress. Based on the analysis of cutin biosynthesis mutants, it has been demonstrated that plant cuticle is required for the regulation of ABA biosynthesis and osmotic stress tolerance.<sup><a reveal-id="DST042C76" data-open="DST042C76" class="link link-ref link-reveal xref-bibr">76</a></sup> Considering the similar biochemical nature of suberin and cutin, the increased biosynthesis of suberin may be crucial for stress tolerance in <em>Porteresia</em>. Furthermore, the characterization of <em>enhanced submergence 1</em> (<em>esb1</em>) mutant provided evidence for the role of suberin in water balance, ion translocation and stress tolerance.<sup><a reveal-id="DST042C77" data-open="DST042C77" class="link link-ref link-reveal xref-bibr">77</a>,<a reveal-id="DST042C78" data-open="DST042C78" class="link link-ref link-reveal xref-bibr">78</a></sup> Thus, alteration in the degree of suberization may result in improved stress tolerance in plants. Overall, it can be concluded that a well-organized transcriptional regulation is responsible for salinity and submergence tolerance in <em>Porteresia</em>. It will be very tempting to investigate the precise role of genes involved in the metabolic pathways enriched during salinity and submergence stresses in <em>Porteresia</em>. The genetic engineering of such candidate genes/pathways may confer stress tolerance in crop plants.</p><p>In conclusion, we report the transcriptome sequence of wild halophyte rice, <em>Porteresia</em>. The results suggest that extensive transcriptional reprogramming under salinity and submergence stresses is responsible for tolerance to these stresses in <em>Porteresia</em>. The transcriptome analyses presented in this study provide several candidate genes involved in salinity and submergence tolerance, which can be used as potential targets for genetic engineering and opens new avenues for future investigations. The functional analyses of genes/pathways identified here along with investigations on non-coding regulatory RNAs, epigenetic modifications, biochemical complexity and metabolite profiling would be very useful to help understand the exact mechanism of salinity and submergence tolerance in <em>Porteresia</em>.</p>                            <h2 scrollto-destination=4921346 id="4921346" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a> www.dnaresearch.oxfordjournals.org.</p>                            <h2 scrollto-destination=4921348 id="4921348" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was financially supported by the core grant from NIPGR. R.G. acknowledges INSPIRE Faculty Award from the Department of Science and Technology and Innovative Young Biotechnologists Award from the Department of Biotechnology, Government of India. R.S. acknowledges the Department of Biotechnology, Government of India, for research fellowship.</p>                            <h2 scrollto-destination=4921350 id="4921350" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are thankful to the anonymous reviewers for their invaluable suggestions.</p>                            <h2 scrollto-destination=4921352 id="4921352" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dst042c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walia</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ismail</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Condamine</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">Genome-wide transcriptional analysis of salinity stressed <em>japonica</em> and <em>indica</em> rice genotypes during panicle initiation stage</div>, <div class="source">Plant Mol. Biol<em>.</em></div> , <div class="year">2007</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">609</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-006-9112-0" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/s11103-006-9112-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20transcriptional%20analysis%20of%20salinity%20stressed%20japonica%20and%20indica%20rice%20genotypes%20during%20panicle%20initiation%20stage&amp;author=H.%20Walia&amp;author=C.%20Wilson&amp;author=L.%20Zeng&amp;author=A.M.%20Ismail&amp;author=P.%20Condamine&amp;author=T.J.%20Close&amp;publication_year=2007&amp;journal=Plant%20Mol.%20Biol.&amp;volume=63&amp;pages=609-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-006-9112-0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-006-9112-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-006-9112-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17160619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumari</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sabharwal</div> <div class="given-names">V.P.</div></div>,  <div class="name"><div class="surname">Kushwaha</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Sopory</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Singla-Pareek</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Pareek</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Transcriptome map for seedling stage specific salinity stress response indicates a specific set of genes as candidate for saline tolerance in <em>Oryza sativa</em> L.</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">109</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10142-008-0088-5" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/s10142-008-0088-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20map%20for%20seedling%20stage%20specific%20salinity%20stress%20response%20indicates%20a%20specific%20set%20of%20genes%20as%20candidate%20for%20saline%20tolerance%20in%20Oryza%20sativa%20L.&amp;author=S.%20Kumari&amp;author=V.P.%20Sabharwal&amp;author=H.R.%20Kushwaha&amp;author=S.K.%20Sopory&amp;author=S.L.%20Singla-Pareek&amp;author=A.%20Pareek&amp;publication_year=2009&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=9&amp;pages=109-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-008-0088-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-008-0088-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-008-0088-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18594887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cotsaftis</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Plett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Root-specific transcript profiling of contrasting rice genotypes in response to salinity stress</div>, <div class="source">Mol. Plant</div> , <div class="year">2011</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">25</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/mp/ssq056" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/mp/ssq056</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root-specific%20transcript%20profiling%20of%20contrasting%20rice%20genotypes%20in%20response%20to%20salinity%20stress&amp;author=O.%20Cotsaftis&amp;author=D.%20Plett&amp;author=A.A.%20Johnson&amp;publication_year=2011&amp;journal=Mol.%20Plant&amp;volume=4&amp;pages=25-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/ssq056" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fssq056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fssq056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20924028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizuno</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Massive parallel sequencing of mRNA in identification of unannotated salinity stress-inducible transcripts in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">683</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-683" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1186/1471-2164-11-683</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20parallel%20sequencing%20of%20mRNA%20in%20identification%20of%20unannotated%20salinity%20stress-inducible%20transcripts%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=H.%20Mizuno&amp;author=Y.%20Kawahara&amp;author=H.%20Sakai&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=683" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-683" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21122150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Colmer</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Flowers</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">Flooding tolerance in halophytes</div>, <div class="source">New Phytol.</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">964</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2008.02483.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1469-8137.2008.02483.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flooding%20tolerance%20in%20halophytes&amp;author=T.D.%20Colmer&amp;author=T.J.%20Flowers&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=964-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18482227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flowers</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Colmer</div> <div class="given-names">T.D.</div></div>. </div><div class="article-title">Salinity tolerance in halophytes</div>, <div class="source">New Phytol<em>.</em></div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">945</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2008.02531.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1469-8137.2008.02531.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20tolerance%20in%20halophytes&amp;author=T.J.%20Flowers&amp;author=T.D.%20Colmer&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=945-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18565144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amtmann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Bressan</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Abiotic stress and plant genome evolution. Search for new models</div>, <div class="source">Plant Physiol<em>.</em></div> , <div class="year">2005</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">127</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.105.059972" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.105.059972</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abiotic%20stress%20and%20plant%20genome%20evolution.%20Search%20for%20new%20models&amp;author=A.%20Amtmann&amp;author=H.J.%20Bohnert&amp;author=R.A.%20Bressan&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=138&amp;pages=127-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.059972" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.059972" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.059972"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15888685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sengupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Majumder</div> <div class="given-names">A.L.</div></div>. </div><div class="article-title"><em>Porteresia coarctata</em> (Roxb.) Tateoka: a wild rice a potential model for studying salt-stress biology in rice</div>, <div class="source">Plant Cell Environ<em>.</em></div> , <div class="year">2010</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">526</div>-<div class="lpage">42</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-3040.2009.02054.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-3040.2009.02054.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Porteresia%20coarctata%20(Roxb.)%20Tateoka%3A%20a%20wild%20rice%20a%20potential%20model%20for%20studying%20salt-stress%20biology%20in%20rice&amp;author=S.%20Sengupta&amp;author=A.L.%20Majumder&amp;publication_year=2010&amp;journal=Plant%20Cell%20Environ.&amp;volume=33&amp;pages=526-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2010.33.issue-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2010.33.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2010.33.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19843254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bal</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Dutt</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title">Mechanisms of salt tolerance in wild rice (<em>Oryza coarctata Roxb</em>.)</div>, <div class="source">Plant Soil</div> , <div class="year">1986</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">399</div>-<div class="lpage">404</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF02372487" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/BF02372487</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20salt%20tolerance%20in%20wild%20rice%20(Oryza%20coarctata%20Roxb.)&amp;author=A.R.%20Bal&amp;author=S.K.%20Dutt&amp;publication_year=1986&amp;journal=Plant%20Soil&amp;volume=92&amp;pages=399-404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02372487" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02372487" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02372487"> </span></div> </div></div></div></div></div><div content-id="dst042c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moons</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bauw</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Prinsen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Van Montagu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Van der Straeten</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Molecular and physiological responses to abscisic acid and salts in roots of salt-sensitive and salt-tolerant Indica rice varieties</div>, <div class="source">Plant Physiol.</div> , <div class="year">1995</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">177</div>-<div class="lpage">86</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.107.1.177" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.107.1.177</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20physiological%20responses%20to%20abscisic%20acid%20and%20salts%20in%20roots%20of%20salt-sensitive%20and%20salt-tolerant%20Indica%20rice%20varieties&amp;author=A.%20Moons&amp;author=G.%20Bauw&amp;author=E.%20Prinsen&amp;author=M.%20Van%20Montagu&amp;author=D.%20Van%20der%20Straeten&amp;publication_year=1995&amp;journal=Plant%20Physiol.&amp;volume=107&amp;pages=177-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.1.177" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.1.177" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.1.177"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7870812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Platten</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Egdane</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Ismail</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Salinity tolerance, Na<sup>+</sup> exclusion and allele mining of HKT1;5 in <em>Oryza sativa</em> and <em>O. glaberrima</em>: many sources, many genes, one mechanism?</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2013</div>, vol. <div class="volume">13</div> pg. <div class="fpage">32</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-13-32" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1186/1471-2229-13-32</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20tolerance%2C%20Na%2B%20exclusion%20and%20allele%20mining%20of%20HKT1%3B5%20in%20Oryza%20sativa%20and%20O.%20glaberrima%3A%20many%20sources%2C%20many%20genes%2C%20one%20mechanism%3F&amp;author=J.D.%20Platten&amp;author=J.A.%20Egdane&amp;author=A.M.%20Ismail&amp;publication_year=2013&amp;journal=BMC%20Plant%20Biol.&amp;volume=13&amp;pages=32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-13-32" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-13-32" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-13-32"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23445750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miro</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ismail</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Tolerance of anaerobic conditions caused by flooding during germination and early growth in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Front. Plant Sci.</div> , <div class="year">2013</div>, vol. <div class="volume">4</div> pg. <div class="fpage">269</div> <div class="pub-id-doi"><a href="https://doi.org/10.3389/fpls.2013.00269" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.3389/fpls.2013.00269</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tolerance%20of%20anaerobic%20conditions%20caused%20by%20flooding%20during%20germination%20and%20early%20growth%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=B.%20Miro&amp;author=A.M.%20Ismail&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci.&amp;volume=4&amp;pages=269" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2013.00269" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2013.00269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2013.00269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23888162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Latha</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rubia</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bennett</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Swaminathan</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Allele mining for stress tolerance genes in <em>Oryza</em> species and related germplasm</div>, <div class="source">Mol. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">101</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1385/MB:27:2:101" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1385/MB:27:2:101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allele%20mining%20for%20stress%20tolerance%20genes%20in%20Oryza%20species%20and%20related%20germplasm&amp;author=R.%20Latha&amp;author=L.%20Rubia&amp;author=J.%20Bennett&amp;author=M.S.%20Swaminathan&amp;publication_year=2004&amp;journal=Mol.%20Biotechnol.&amp;volume=27&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1385/MB:27:2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1385%2FMB:27:2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1385%2FMB:27:2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Inan</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Salt cress. A halophyte and cryophyte Arabidopsis relative model system and its applicability to molecular genetic analyses of growth and development of extremophiles</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">1718</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.041723" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.104.041723</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salt%20cress.%20A%20halophyte%20and%20cryophyte%20Arabidopsis%20relative%20model%20system%20and%20its%20applicability%20to%20molecular%20genetic%20analyses%20of%20growth%20and%20development%20of%20extremophiles&amp;author=G.%20Inan&amp;author=Q.%20Zhang&amp;author=P.%20Li&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=1718-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.041723" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.041723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.041723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oh</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Dassanayake</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">Genome structures and halophyte-specific gene expression of the extremophile <em>Thellungiella parvula</em> in comparison with <em>Thellungiella salsuginea</em> (<em>Thellungiella halophila</em>) and Arabidopsis</div>, <div class="source">Plant Physiol<em>.</em></div> , <div class="year">2010</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">1040</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.110.163923" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.110.163923</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structures%20and%20halophyte-specific%20gene%20expression%20of%20the%20extremophile%20Thellungiella%20parvula%20in%20comparison%20with%20Thellungiella%20salsuginea%20(Thellungiella%20halophila)%20and%20Arabidopsis&amp;author=D.H.%20Oh&amp;author=M.%20Dassanayake&amp;author=J.S.%20Haas&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=154&amp;pages=1040-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.163923" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.163923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.163923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20833729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.Y.</div></div>, et al. </div><div class="article-title">Insights into salt tolerance from the genome of <em>Thellungiella salsuginea</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2012</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">12219</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.1209954109" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1073/pnas.1209954109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20salt%20tolerance%20from%20the%20genome%20of%20Thellungiella%20salsuginea&amp;author=H.J.%20Wu&amp;author=Z.%20Zhang&amp;author=J.Y.%20Wang&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=12219-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1209954109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1209954109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1209954109"> </span></div> </div></div></div></div></div><div content-id="dst042c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Majee</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maitra</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dastidar</div> <div class="given-names">K.G.</div></div>, et al. </div><div class="article-title">A novel salt-tolerant L-myo-inositol-1-phosphate synthase from <em>Porteresia coarctata</em> (Roxb.) Tateoka, a halophytic wild rice molecular cloning, bacterial overexpression, characterization, and functional introgression into tobacco-conferring salt tolerance phenotype</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">28539</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M310138200" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1074/jbc.M310138200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20salt-tolerant%20L-myo-inositol-1-phosphate%20synthase%20from%20Porteresia%20coarctata%20(Roxb.)%20Tateoka%2C%20a%20halophytic%20wild%20rice%20molecular%20cloning%2C%20bacterial%20overexpression%2C%20characterization%2C%20and%20functional%20introgression%20into%20tobacco-conferring%20salt%20tolerance%20phenotype&amp;author=M.%20Majee&amp;author=S.%20Maitra&amp;author=K.G.%20Dastidar&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=28539-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M310138200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M310138200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M310138200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15016817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sengupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Patra</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ray</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Majumder</div> <div class="given-names">A.L.</div></div>. </div><div class="article-title">Inositol methyl transferase from a halophytic wild rice, <em>Porteresia coarctata</em> Roxb. (Tateoka) regulation of pinitol synthesis under abiotic stress</div>, <div class="source">Plant Cell Environ<em>.</em></div> , <div class="year">2008</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1442</div>-<div class="lpage">59</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-3040.2008.01850.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-3040.2008.01850.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inositol%20methyl%20transferase%20from%20a%20halophytic%20wild%20rice%2C%20Porteresia%20coarctata%20Roxb.%20(Tateoka)%20regulation%20of%20pinitol%20synthesis%20under%20abiotic%20stress&amp;author=S.%20Sengupta&amp;author=B.%20Patra&amp;author=S.%20Ray&amp;author=A.L.%20Majumder&amp;publication_year=2008&amp;journal=Plant%20Cell%20Environ.&amp;volume=31&amp;pages=1442-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2008.31.issue-10" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2008.31.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2008.31.issue-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18643954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morozova</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Hirst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Marra</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Applications of new sequencing technologies for transcriptome analysis</div>, <div class="source">Annu. Rev. Genomics Human Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">135</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev-genom-082908-145957" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1146/annurev-genom-082908-145957</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20new%20sequencing%20technologies%20for%20transcriptome%20analysis&amp;author=O.%20Morozova&amp;author=M.%20Hirst&amp;author=M.A.%20Marra&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Genomics%20Human%20Genet.&amp;volume=10&amp;pages=135-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genom-082908-145957" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genom-082908-145957" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genom-082908-145957"> </span></div> </div></div></div></div></div><div content-id="dst042c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A next generation approach to the characterization of a non-model plant transcriptome</div>, <div class="source">Curr. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">1435</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="dst042c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Priya</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transcriptome sequencing of wild chickpea as a rich resource for marker development</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">690</div>-<div class="lpage">702</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1467-7652.2012.00712.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1467-7652.2012.00712.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20wild%20chickpea%20as%20a%20rich%20resource%20for%20marker%20development&amp;author=S.%20Jhanwar&amp;author=P.%20Priya&amp;author=R.%20Garg&amp;author=S.K.%20Parida&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=690-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00712.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00712.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00712.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22672127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">NGS QC Toolkit: a toolkit for quality control of next generation sequencing data</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e30619</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0030619" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1371/journal.pone.0030619</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NGS%20QC%20Toolkit%3A%20a%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data&amp;author=R.K.%20Patel&amp;author=M.%20Jain&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30619" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030619" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22312429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Gene discovery and tissue-specific transcriptome analysis in chickpea with massively parallel pyrosequencing and web resource development</div>, <div class="source">Plant Physiol<em>.</em></div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1661</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.111.178616" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.111.178616</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20and%20tissue-specific%20transcriptome%20analysis%20in%20chickpea%20with%20massively%20parallel%20pyrosequencing%20and%20web%20resource%20development&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=S.%20Jhanwar&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1661-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.178616" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.178616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.178616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawahara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">de la Bastide</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">Improvement of the <em>Oryza sativa</em> Nipponbare reference genome using next generation sequence and optical map data</div>, <div class="source">Rice</div> , <div class="year">2013</div>, vol. <div class="volume">6</div> pg. <div class="fpage">4</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1939-8433-6-4" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1186/1939-8433-6-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20the%20Oryza%20sativa%20Nipponbare%20reference%20genome%20using%20next%20generation%20sequence%20and%20optical%20map%20data&amp;author=Y.%20Kawahara&amp;author=M.%20de%20la%20Bastide&amp;author=J.P.%20Hamilton&amp;publication_year=2013&amp;journal=Rice&amp;volume=6&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1939-8433-6-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1939-8433-6-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1939-8433-6-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24280374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Differential expression analysis for sequence count data</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">R106</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2010-11-10-r106" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1186/gb-2010-11-10-r106</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-10-r106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-10-r106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-10-r106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maere</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Heymans</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kuiper</div> <div class="given-names">M.</div></div>. </div><div class="article-title">BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3448</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti551" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/bioinformatics/bti551</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BiNGO%3A%20a%20Cytoscape%20plugin%20to%20assess%20overrepresentation%20of%20gene%20ontology%20categories%20in%20biological%20networks&amp;author=S.%20Maere&amp;author=K.%20Heymans&amp;author=M.%20Kuiper&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3448-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti551" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti551" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti551"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15972284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaiswal</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ni</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yap</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Gramene a bird's eye view of cereal genomes</div>, <div class="source">Nucleic Acids Res<em>.</em></div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D717</div>-<div class="lpage">723</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkj154" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/nar/gkj154</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gramene%20a%20bird's%20eye%20view%20of%20cereal%20genomes&amp;author=P.%20Jaiswal&amp;author=J.%20Ni&amp;author=I.%20Yap&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D717-723" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj154" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj154"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Validation of internal control genes for quantitative gene expression studies in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2010</div>, vol. <div class="volume">396</div> (pg. <div class="fpage">283</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2010.04.079" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.bbrc.2010.04.079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20internal%20control%20genes%20for%20quantitative%20gene%20expression%20studies%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Garg&amp;author=A.%20Sahoo&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2010&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=396&amp;pages=283-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2010.04.079" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2010.04.079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2010.04.079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20399753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nijhawan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Khurana</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Validation of housekeeping genes as internal control for studying gene expression in rice by quantitative real-time PCR</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2006</div>, vol. <div class="volume">345</div> (pg. <div class="fpage">646</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2006.04.140" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.bbrc.2006.04.140</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20housekeeping%20genes%20as%20internal%20control%20for%20studying%20gene%20expression%20in%20rice%20by%20quantitative%20real-time%20PCR&amp;author=M.%20Jain&amp;author=A.%20Nijhawan&amp;author=A.K.%20Tyagi&amp;author=J.P.%20Khurana&amp;publication_year=2006&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=345&amp;pages=646-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2006.04.140" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2006.04.140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2006.04.140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16690022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Agarwal</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Jhanwar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Priya</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Comparative analysis of kabuli chickpea transcriptome with desi and wild chickpea provides a rich resource for development of functional markers</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e52443</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0052443" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1371/journal.pone.0052443</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20kabuli%20chickpea%20transcriptome%20with%20desi%20and%20wild%20chickpea%20provides%20a%20rich%20resource%20for%20development%20of%20functional%20markers&amp;author=G.%20Agarwal&amp;author=S.%20Jhanwar&amp;author=P.%20Priya&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e52443" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0052443" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0052443"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ozsolak</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Milos</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">RNA sequencing advances, challenges and opportunities</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">87</div>-<div class="lpage">98</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg2934" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1038/nrg2934</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20sequencing%20advances%2C%20challenges%20and%20opportunities&amp;author=F.%20Ozsolak&amp;author=P.M.%20Milos&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=87-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2934" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2934"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21191423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Next-generation sequencing technologies for gene expression profiling in plants</div>, <div class="source">Brief Funct. Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">63</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bfgp/elr038" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/bfgp/elr038</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20technologies%20for%20gene%20expression%20profiling%20in%20plants&amp;author=M.%20Jain&amp;publication_year=2012&amp;journal=Brief%20Funct.%20Genomics&amp;volume=11&amp;pages=63-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/elr038" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Felr038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Felr038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22155524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gong</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Indu Rupassara</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Salinity stress adaptation competence in the extremophile <em>Thellungiella halophila</em> in comparison with its relative <em>Arabidopsis thaliana</em></div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">826</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2005.02587.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2005.02587.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20stress%20adaptation%20competence%20in%20the%20extremophile%20Thellungiella%20halophila%20in%20comparison%20with%20its%20relative%20Arabidopsis%20thaliana&amp;author=Q.%20Gong&amp;author=P.%20Li&amp;author=S.%20Ma&amp;author=S.%20Indu%20Rupassara&amp;author=H.J.%20Bohnert&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=44&amp;pages=826-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.44.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.44.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.44.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16297073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Munns</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tester</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mechanisms of salinity tolerance</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">651</div>-<div class="lpage">81</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.59.032607.092911" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1146/annurev.arplant.59.032607.092911</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20salinity%20tolerance&amp;author=R.%20Munns&amp;author=M.%20Tester&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=59&amp;pages=651-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.59.032607.092911" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.59.032607.092911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.59.032607.092911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18444910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Mustroph</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sasidharan</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Molecular characterization of the submergence response of the <em>Arabidopsis thaliana</em> ecotype Columbia</div>, <div class="source">New Phytol<em>.</em></div> , <div class="year">2011</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">457</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2010.03590.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1469-8137.2010.03590.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20the%20submergence%20response%20of%20the%20Arabidopsis%20thaliana%20ecotype%20Columbia&amp;author=S.C.%20Lee&amp;author=A.%20Mustroph&amp;author=R.%20Sasidharan&amp;publication_year=2011&amp;journal=New%20Phytol.&amp;volume=190&amp;pages=457-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03590.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03590.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03590.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21231933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voesenek</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Colmer</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Pierik</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Millenaar</div> <div class="given-names">F.F.</div></div>,  <div class="name"><div class="surname">Peeters</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">How plants cope with complete submergence</div>, <div class="source">New Phytol.</div> , <div class="year">2006</div>, vol. <div class="volume">170</div> (pg. <div class="fpage">213</div>-<div class="lpage">26</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2006.01692.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1469-8137.2006.01692.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20plants%20cope%20with%20complete%20submergence&amp;author=L.A.%20Voesenek&amp;author=T.D.%20Colmer&amp;author=R.%20Pierik&amp;author=F.F.%20Millenaar&amp;author=A.J.%20Peeters&amp;publication_year=2006&amp;journal=New%20Phytol.&amp;volume=170&amp;pages=213-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2006.170.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2006.170.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2006.170.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16608449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vashisht</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hesselink</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pierik</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Natural variation of submergence tolerance among <em>Arabidopsis thaliana</em> accessions</div>, <div class="source">New Phytol.</div> , <div class="year">2011</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">299</div>-<div class="lpage">310</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2010.03552.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1469-8137.2010.03552.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20of%20submergence%20tolerance%20among%20Arabidopsis%20thaliana%20accessions&amp;author=D.%20Vashisht&amp;author=A.%20Hesselink&amp;author=R.%20Pierik&amp;publication_year=2011&amp;journal=New%20Phytol.&amp;volume=190&amp;pages=299-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03552.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03552.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03552.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21108648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wong</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Labbe</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Transcriptional profiling implicates novel interactions between abiotic stress and hormonal responses in <em>Thellungiella</em>, a close relative of Arabidopsis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">140</div> (pg. <div class="fpage">1437</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.105.070508" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.105.070508</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20profiling%20implicates%20novel%20interactions%20between%20abiotic%20stress%20and%20hormonal%20responses%20in%20Thellungiella%2C%20a%20close%20relative%20of%20Arabidopsis&amp;author=C.E.%20Wong&amp;author=Y.%20Li&amp;author=A.%20Labbe&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=140&amp;pages=1437-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.070508" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.070508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.070508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16500996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mustroph</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Oosumi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Cross-kingdom comparison of transcriptomic adjustments to low-oxygen stress highlights conserved and plant-specific responses</div>, <div class="source">Plant Physiol.</div> , <div class="year">2010</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">1484</div>-<div class="lpage">500</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.151845" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.109.151845</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-kingdom%20comparison%20of%20transcriptomic%20adjustments%20to%20low-oxygen%20stress%20highlights%20conserved%20and%20plant-specific%20responses&amp;author=A.%20Mustroph&amp;author=S.C.%20Lee&amp;author=T.%20Oosumi&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=152&amp;pages=1484-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.151845" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.151845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.151845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20097791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jung</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Seo</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Walia</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The submergence tolerance regulator Sub1A mediates stress-responsive expression of AP2/ERF transcription factors</div>, <div class="source">Plant Physiol<em>.</em></div> , <div class="year">2010</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">1674</div>-<div class="lpage">92</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.152157" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.109.152157</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20submergence%20tolerance%20regulator%20Sub1A%20mediates%20stress-responsive%20expression%20of%20AP2%2FERF%20transcription%20factors&amp;author=K.H.%20Jung&amp;author=Y.S.%20Seo&amp;author=H.%20Walia&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=152&amp;pages=1674-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.152157" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.152157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.152157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20107022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ronald</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Bailey-Serres</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A variable cluster of ethylene response factor-like genes regulates metabolic and developmental acclimation responses to submergence in rice</div>, <div class="source">Plant Cell</div> , <div class="year">2006</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">2021</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.106.043000" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1105/tpc.106.043000</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20variable%20cluster%20of%20ethylene%20response%20factor-like%20genes%20regulates%20metabolic%20and%20developmental%20acclimation%20responses%20to%20submergence%20in%20rice&amp;author=T.%20Fukao&amp;author=K.%20Xu&amp;author=P.C.%20Ronald&amp;author=J.%20Bailey-Serres&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=2021-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.043000" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.043000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.043000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16816135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lasanthi-Kudahettige</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Magneschi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Loreti</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Transcript profiling of the anoxic rice coleoptile</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">218</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.106.093997" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.106.093997</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20profiling%20of%20the%20anoxic%20rice%20coleoptile&amp;author=R.%20Lasanthi-Kudahettige&amp;author=L.%20Magneschi&amp;author=E.%20Loreti&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=144&amp;pages=218-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.093997" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.093997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.093997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17369434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Takano</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Akita</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Expression of alpha-expansin genes in young seedlings of rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Planta</div> , <div class="year">2000</div>, vol. <div class="volume">211</div> (pg. <div class="fpage">467</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s004250000311" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/s004250000311</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20alpha-expansin%20genes%20in%20young%20seedlings%20of%20rice%20(Oryza%20sativa%20L.)&amp;author=J.%20Huang&amp;author=T.%20Takano&amp;author=S.%20Akita&amp;publication_year=2000&amp;journal=Planta&amp;volume=211&amp;pages=467-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004250000311" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004250000311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004250000311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11030545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Comparative transcriptomic profiling of a salt-tolerant wild tomato species and a salt-sensitive tomato cultivar</div>, <div class="source">Plant Cell Physiol<em>.</em></div> , <div class="year">2010</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">997</div>-<div class="lpage">1006</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcq056" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pcq056</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomic%20profiling%20of%20a%20salt-tolerant%20wild%20tomato%20species%20and%20a%20salt-sensitive%20tomato%20cultivar&amp;author=W.%20Sun&amp;author=X.%20Xu&amp;author=H.%20Zhu&amp;publication_year=2010&amp;journal=Plant%20Cell%20Physiol.&amp;volume=51&amp;pages=997-1006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcq056" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcq056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcq056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20410049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ikawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Distinct evolutionary patterns of <em>Oryza glaberrima</em> deciphered by genome sequencing and comparative analysis</div>, <div class="source">Plant J.</div> , <div class="year">2011</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">796</div>-<div class="lpage">805</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2011.04539.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2011.04539.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20evolutionary%20patterns%20of%20Oryza%20glaberrima%20deciphered%20by%20genome%20sequencing%20and%20comparative%20analysis&amp;author=H.%20Sakai&amp;author=H.%20Ikawa&amp;author=T.%20Tanaka&amp;publication_year=2011&amp;journal=Plant%20J.&amp;volume=66&amp;pages=796-805" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2011.04539.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2011.04539.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2011.04539.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21323774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Whole-genome sequencing of <em>Oryza brachyantha</em> reveals mechanisms underlying <em>Oryza</em> genome evolution</div>, <div class="source">Nat Commun.</div> , <div class="year">2013</div>, vol. <div class="volume">4</div> pg. <div class="fpage">1595</div> <div class="pub-id-doi"><a href="https://doi.org/10.1038/ncomms2596" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1038/ncomms2596</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20Oryza%20brachyantha%20reveals%20mechanisms%20underlying%20Oryza%20genome%20evolution&amp;author=J.%20Chen&amp;author=Q.%20Huang&amp;author=D.%20Gao&amp;publication_year=2013&amp;journal=Nat%20Commun.&amp;volume=4&amp;pages=1595" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2596" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23481403" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamaguchi-Shinozaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Transcriptional regulatory networks in cellular responses and tolerance to dehydration and cold stresses</div>, <div class="source">Annu. Rev. Plant Biol<em>.</em></div> , <div class="year">2006</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">781</div>-<div class="lpage">803</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.57.032905.105444" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1146/annurev.arplant.57.032905.105444</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulatory%20networks%20in%20cellular%20responses%20and%20tolerance%20to%20dehydration%20and%20cold%20stresses&amp;author=K.%20Yamaguchi-Shinozaki&amp;author=K.%20Shinozaki&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=57&amp;pages=781-803" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105444" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.57.032905.105444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.57.032905.105444"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669782" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakashima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamaguchi-Shinozaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Transcriptional regulatory networks in response to abiotic stresses in Arabidopsis and grasses</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">88</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.108.129791" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.108.129791</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulatory%20networks%20in%20response%20to%20abiotic%20stresses%20in%20Arabidopsis%20and%20grasses&amp;author=K.%20Nakashima&amp;author=Y.%20Ito&amp;author=K.%20Yamaguchi-Shinozaki&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=88-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.129791" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.129791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.129791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhattacharjee</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transcription factor mediated abiotic stress signaling in rice</div>, <div class="source">Plant Stress</div> , <div class="year">2013</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">16</div>-<div class="lpage">25</div>)</div></div></div></div><div content-id="dst042c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bailey-Serres</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Evolutionary analysis of the <em>Sub1</em> gene cluster that confers submergence tolerance to domesticated rice</div>, <div class="source">Ann. Bot.</div> , <div class="year">2009</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">143</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/aob/mcn172" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/aob/mcn172</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20analysis%20of%20the%20Sub1%20gene%20cluster%20that%20confers%20submergence%20tolerance%20to%20domesticated%20rice&amp;author=T.%20Fukao&amp;author=T.%20Harris&amp;author=J.%20Bailey-Serres&amp;publication_year=2009&amp;journal=Ann.%20Bot.&amp;volume=103&amp;pages=143-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcn172" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcn172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcn172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18824474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Fukao</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title"><em>Sub1A</em> is an ethylene-response-factor-like gene that confers submergence tolerance to rice</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">442</div> (pg. <div class="fpage">705</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature04920" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1038/nature04920</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sub1A%20is%20an%20ethylene-response-factor-like%20gene%20that%20confers%20submergence%20tolerance%20to%20rice&amp;author=K.%20Xu&amp;author=X.%20Xu&amp;author=T.%20Fukao&amp;publication_year=2006&amp;journal=Nature&amp;volume=442&amp;pages=705-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04920" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04920" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04920"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16900200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Niroula</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Pucciariello</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ho</div> <div class="given-names">V.T.</div></div>,  <div class="name"><div class="surname">Novi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fukao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Perata</div> <div class="given-names">P.</div></div>. </div><div class="article-title">SUB1A-dependent and -independent mechanisms are involved in the flooding tolerance of wild rice species</div>, <div class="source">Plant J.</div> , <div class="year">2012</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">282</div>-<div class="lpage">93</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2012.05078.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2012.05078.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SUB1A-dependent%20and%20-independent%20mechanisms%20are%20involved%20in%20the%20flooding%20tolerance%20of%20wild%20rice%20species&amp;author=R.K.%20Niroula&amp;author=C.%20Pucciariello&amp;author=V.T.%20Ho&amp;author=G.%20Novi&amp;author=T.%20Fukao&amp;author=P.%20Perata&amp;publication_year=2012&amp;journal=Plant%20J.&amp;volume=72&amp;pages=282-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2012.05078.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2012.05078.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2012.05078.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22709342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jang</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Ishihara</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Back</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Production of coumaroylserotonin and feruloylserotonin in transgenic rice expressing pepper hydroxycinnamoyl-coenzyme aserotonin <em>N</em>-(hydroxycinnamoyl) transferase</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">346</div>-<div class="lpage">56</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.103.038372" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.103.038372</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Production%20of%20coumaroylserotonin%20and%20feruloylserotonin%20in%20transgenic%20rice%20expressing%20pepper%20hydroxycinnamoyl-coenzyme%20aserotonin%20N-(hydroxycinnamoyl)%20transferase&amp;author=S.M.%20Jang&amp;author=A.%20Ishihara&amp;author=K.%20Back&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=346-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.038372" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.038372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.038372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15122017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wasternack</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Jasmonates an update on biosynthesis, signal transduction and action in plant stress response, growth and development</div>, <div class="source">Ann. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">681</div>-<div class="lpage">97</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/aob/mcm079" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/aob/mcm079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Jasmonates%20an%20update%20on%20biosynthesis%2C%20signal%20transduction%20and%20action%20in%20plant%20stress%20response%2C%20growth%20and%20development&amp;author=C.%20Wasternack&amp;publication_year=2007&amp;journal=Ann.%20Bot.&amp;volume=100&amp;pages=681-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcm079" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcm079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcm079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17513307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goossens</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Häkkinen</div> <div class="given-names">S.T.</div></div>,  <div class="name"><div class="surname">Laakso</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">A functional genomics approach toward the understanding of secondary metabolism in plant cells</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">8595</div>-<div class="lpage">600</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.1032967100" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1073/pnas.1032967100</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20functional%20genomics%20approach%20toward%20the%20understanding%20of%20secondary%20metabolism%20in%20plant%20cells&amp;author=A.%20Goossens&amp;author=S.T.%20H%C3%A4kkinen&amp;author=I.%20Laakso&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=8595-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1032967100" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1032967100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1032967100"> </span></div> </div></div></div></div></div><div content-id="dst042c56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van der Fits</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Memelink</div> <div class="given-names">J.</div></div>. </div><div class="article-title">ORCA3, a jasmonate-responsive transcriptional regulator of plant primary and secondary metabolism</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">289</div> (pg. <div class="fpage">295</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.289.5477.295" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1126/science.289.5477.295</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ORCA3%2C%20a%20jasmonate-responsive%20transcriptional%20regulator%20of%20plant%20primary%20and%20secondary%20metabolism&amp;author=L.%20van%20der%20Fits&amp;author=J.%20Memelink&amp;publication_year=2000&amp;journal=Science&amp;volume=289&amp;pages=295-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.289.5477.295" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.289.5477.295" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.289.5477.295"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10894776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki-Sekimoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Taki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Coordinated activation of metabolic pathways for antioxidants and defence compounds by jasmonates and their roles in stress tolerance in Arabidopsis</div>, <div class="source">Plant J<em>.</em></div> , <div class="year">2005</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">653</div>-<div class="lpage">68</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2005.02560.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2005.02560.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coordinated%20activation%20of%20metabolic%20pathways%20for%20antioxidants%20and%20defence%20compounds%20by%20jasmonates%20and%20their%20roles%20in%20stress%20tolerance%20in%20Arabidopsis&amp;author=Y.%20Sasaki-Sekimoto&amp;author=N.%20Taki&amp;author=T.%20Obayashi&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=44&amp;pages=653-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.44.issue-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.44.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.44.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16262714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bürstenbinder</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Rzewuski</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wirtz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hell</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sauter</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The role of methionine recycling for ethylene synthesis in Arabidopsis</div>, <div class="source">Plant J<em>.</em></div> , <div class="year">2007</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">238</div>-<div class="lpage">49</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2006.02942.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2006.02942.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20methionine%20recycling%20for%20ethylene%20synthesis%20in%20Arabidopsis&amp;author=K.%20B%C3%BCrstenbinder&amp;author=G.%20Rzewuski&amp;author=M.%20Wirtz&amp;author=R.%20Hell&amp;author=M.%20Sauter&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=49&amp;pages=238-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2006.02942.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2006.02942.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2006.02942.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17144895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yeung</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bailey-Serres</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The submergence tolerance gene <em>SUB1A</em> delays leaf senescence under prolonged darkness through hormonal regulation in rice</div>, <div class="source">Plant Physiol<em>.</em></div> , <div class="year">2012</div>, vol. <div class="volume">160</div> (pg. <div class="fpage">1795</div>-<div class="lpage">807</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.112.207738" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.112.207738</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20submergence%20tolerance%20gene%20SUB1A%20delays%20leaf%20senescence%20under%20prolonged%20darkness%20through%20hormonal%20regulation%20in%20rice&amp;author=T.%20Fukao&amp;author=E.%20Yeung&amp;author=J.%20Bailey-Serres&amp;publication_year=2012&amp;journal=Plant%20Physiol.&amp;volume=160&amp;pages=1795-807" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.207738" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.207738" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.207738"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23073696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiong</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Regulation of abscisic acid biosynthesis</div>, <div class="source">Plant Physiol<em>.</em></div> , <div class="year">2003</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">29</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.103.025395" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.103.025395</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20abscisic%20acid%20biosynthesis&amp;author=L.%20Xiong&amp;author=J.K.%20Zhu&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=133&amp;pages=29-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.025395" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.025395" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.025395"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12970472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Verslues</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Before and beyond ABA upstream sensing and internal signals that determine ABA accumulation and response under abiotic stress</div>, <div class="source">Biochem. Soc. Trans<em>.</em></div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">375</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1042/BST0330375" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1042/BST0330375</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Before%20and%20beyond%20ABA%20upstream%20sensing%20and%20internal%20signals%20that%20determine%20ABA%20accumulation%20and%20response%20under%20abiotic%20stress&amp;author=P.E.%20Verslues&amp;author=J.K.%20Zhu&amp;publication_year=2005&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=33&amp;pages=375-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0330375" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0330375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0330375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15787610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuteja</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Mechanisms of high salinity tolerance in plants</div>, <div class="source">Methods Enzymol.</div> , <div class="year">2007</div>, vol. <div class="volume">428</div> (pg. <div class="fpage">419</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20high%20salinity%20tolerance%20in%20plants&amp;author=N.%20Tuteja&amp;publication_year=2007&amp;journal=Methods%20Enzymol.&amp;volume=428&amp;pages=419-38" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17875432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cutler</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Rodriguez</div> <div class="given-names">P.L.</div></div>,  <div class="name"><div class="surname">Finkelstein</div> <div class="given-names">R.R.</div></div>,  <div class="name"><div class="surname">Abrams</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Abscisic acid emergence of a core signaling network</div>, <div class="source">Annu. Rev. Plant Biol<em>.</em></div> , <div class="year">2010</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">651</div>-<div class="lpage">79</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev-arplant-042809-112122" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1146/annurev-arplant-042809-112122</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abscisic%20acid%20emergence%20of%20a%20core%20signaling%20network&amp;author=S.R.%20Cutler&amp;author=P.L.%20Rodriguez&amp;author=R.R.%20Finkelstein&amp;author=S.R.%20Abrams&amp;publication_year=2010&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=61&amp;pages=651-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-042809-112122" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-042809-112122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-042809-112122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20192755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saika</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miyoshi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Ethylene promotes submergence-induced expression of <em>OsABA8ox1</em>, a gene that encodes ABA 8′-hydroxylase in rice</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">287</div>-<div class="lpage">98</div>)</div></div></div></div><div content-id="dst042c65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dubey</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Salinity induces accumulation of soluble sugars and alters the activity of sugar metabolizing enzymes in rice plants</div>, <div class="source">Biol. Plant<em>.</em></div> , <div class="year">1999</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">233</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1002160618700" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1023/A:1002160618700</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20induces%20accumulation%20of%20soluble%20sugars%20and%20alters%20the%20activity%20of%20sugar%20metabolizing%20enzymes%20in%20rice%20plants&amp;author=R.S.%20Dubey&amp;author=A.K.%20Singh&amp;publication_year=1999&amp;journal=Biol.%20Plant.&amp;volume=42&amp;pages=233-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1002160618700" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1002160618700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1002160618700"> </span></div> </div></div></div></div></div><div content-id="dst042c66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gill</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bhullar</div> <div class="given-names">S.S.</div></div>. </div><div class="article-title">Effect of various abiotic stresses on the growth, soluble sugars and water relations of sorghum seedlings grown in light and darkness</div>, <div class="source">Bulg. J. Plant Physiol<em>.</em></div> , <div class="year">2001</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">72</div>-<div class="lpage">84</div>)</div></div></div></div><div content-id="dst042c67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Narusaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ishida</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray</div>, <div class="source">Plant J.</div> , <div class="year">2002</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">279</div>-<div class="lpage">92</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.2002.01359.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1046/j.1365-313X.2002.01359.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monitoring%20the%20expression%20profiles%20of%207000%20Arabidopsis%20genes%20under%20drought%2C%20cold%20and%20high-salinity%20stresses%20using%20a%20full-length%20cDNA%20microarray&amp;author=M.%20Seki&amp;author=M.%20Narusaka&amp;author=J.%20Ishida&amp;publication_year=2002&amp;journal=Plant%20J.&amp;volume=31&amp;pages=279-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2002.01359.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2002.01359.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2002.01359.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12164808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Kaur</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Sugar signalling and gene expression in relation to carbohydrate metabolism under abiotic stresses in plants</div>, <div class="source">J. Biosci.</div> , <div class="year">2005</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">761</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF02703574" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/BF02703574</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sugar%20signalling%20and%20gene%20expression%20in%20relation%20to%20carbohydrate%20metabolism%20under%20abiotic%20stresses%20in%20plants&amp;author=A.K.%20Gupta&amp;author=N.%20Kaur&amp;publication_year=2005&amp;journal=J.%20Biosci.&amp;volume=30&amp;pages=761-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02703574" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02703574" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02703574"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16388148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Prado</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Podazza</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Soluble sugars—metabolism, sensing and abiotic stress a complex network in the life of plants</div>, <div class="source">Plant Signal. Behav.</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">388</div>-<div class="lpage">93</div>)<div class="pub-id-doi"><a href="https://doi.org/10.4161/psb.4.5.8294" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.4161/psb.4.5.8294</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soluble%20sugars%E2%80%94metabolism%2C%20sensing%20and%20abiotic%20stress%20a%20complex%20network%20in%20the%20life%20of%20plants&amp;author=M.%20Rosa&amp;author=C.%20Prado&amp;author=G.%20Podazza&amp;publication_year=2009&amp;journal=Plant%20Signal.%20Behav.&amp;volume=4&amp;pages=388-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.4.5.8294" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.4.5.8294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.4.5.8294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19816104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voesenek</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Bailey-Serres</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Plant biology genetics of high-rise rice</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">460</div> (pg. <div class="fpage">959</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/460959a" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1038/460959a</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20biology%20genetics%20of%20high-rise%20rice&amp;author=L.A.%20Voesenek&amp;author=J.%20Bailey-Serres&amp;publication_year=2009&amp;journal=Nature&amp;volume=460&amp;pages=959-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/460959a" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F460959a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F460959a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19693073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Owens</div> <div class="given-names">T.G.</div></div>, et al. </div><div class="article-title">Trehalose accumulation in rice plants confers high tolerance levels to different abiotic stresses</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">15898</div>-<div class="lpage">903</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.252637799" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1073/pnas.252637799</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trehalose%20accumulation%20in%20rice%20plants%20confers%20high%20tolerance%20levels%20to%20different%20abiotic%20stresses&amp;author=A.K.%20Garg&amp;author=J.K.%20Kim&amp;author=T.G.%20Owens&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=15898-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.252637799" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.252637799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.252637799"> </span></div> </div></div></div></div></div><div content-id="dst042c72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Redillas</div> <div class="given-names">M.C.F.R.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">S-H.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J.W.</div></div>, et al. </div><div class="article-title">Accumulation of trehalose increases soluble sugar contents in rice plants conferring tolerance to drought and salt stress</div>, <div class="source">Plant Biotechnol. Rep<em>.</em></div> , <div class="year">2012</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">89</div>-<div class="lpage">96</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11816-011-0210-3" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/s11816-011-0210-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accumulation%20of%20trehalose%20increases%20soluble%20sugar%20contents%20in%20rice%20plants%20conferring%20tolerance%20to%20drought%20and%20salt%20stress&amp;author=M.C.F.R.%20Redillas&amp;author=S-H.%20Park&amp;author=J.W.%20Lee&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20Rep.&amp;volume=6&amp;pages=89-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11816-011-0210-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11816-011-0210-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11816-011-0210-3"> </span></div> </div></div></div></div></div><div content-id="dst042c73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koiwa</div> <div class="given-names">H.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Jenks</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Pathways and genetic determinants for cell wall-based osmotic stress tolerance in the <em>Arabidopsis thaliana</em> root system</div>, <div class="source">Genes for plant abiotic stress</div> , <div class="year">2009</div><div class="publisher-loc">Oxford, UK</div><div class="publisher-name">Wiley-Blackwell</div> <div class="comment">Ch. 2</div></div></div></div></div><div content-id="dst042c74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Disruption of the cellulose synthase gene, <em>AtCesA8</em>/<em>IRX1</em>, enhances drought and osmotic stress tolerance in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">273</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2005.02452.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2005.02452.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disruption%20of%20the%20cellulose%20synthase%20gene%2C%20AtCesA8%2FIRX1%2C%20enhances%20drought%20and%20osmotic%20stress%20tolerance%20in%20Arabidopsis&amp;author=Z.%20Chen&amp;author=X.%20Hong&amp;author=H.%20Zhang&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=43&amp;pages=273-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.43.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.43.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.43.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15998313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Dellinger</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A cellulose synthase-like protein is required for osmotic stress tolerance in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2010</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">128</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20cellulose%20synthase-like%20protein%20is%20required%20for%20osmotic%20stress%20tolerance%20in%20Arabidopsis&amp;author=J.%20Zhu&amp;author=B.H.%20Lee&amp;author=M.%20Dellinger&amp;publication_year=2010&amp;journal=Plant%20J.&amp;volume=63&amp;pages=128-40" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20409003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Xiong</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The plant cuticle is required for osmotic stress regulation of abscisic acid biosynthesis and osmotic stress tolerance in Arabidopsis</div>, <div class="source">Plant Cell</div> , <div class="year">2011</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1971</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.110.081943" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1105/tpc.110.081943</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20plant%20cuticle%20is%20required%20for%20osmotic%20stress%20regulation%20of%20abscisic%20acid%20biosynthesis%20and%20osmotic%20stress%20tolerance%20in%20Arabidopsis&amp;author=Z.Y.%20Wang&amp;author=L.%20Xiong&amp;author=W.%20Li&amp;author=J.K.%20Zhu&amp;author=J.%20Zhu&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=1971-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.081943" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.081943" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.081943"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21610183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baxter</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Hosmani</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Rus</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Root suberin forms an extracellular barrier that affects water relations and mineral nutrition in Arabidopsis</div>, <div class="source">PLoS Genet<em>.</em></div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000492</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.1000492" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1371/journal.pgen.1000492</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%20suberin%20forms%20an%20extracellular%20barrier%20that%20affects%20water%20relations%20and%20mineral%20nutrition%20in%20Arabidopsis&amp;author=I.%20Baxter&amp;author=P.S.%20Hosmani&amp;author=A.%20Rus&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000492" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000492" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000492" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000492"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19461889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst042c78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst042c78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Franke</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Dombrink</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Schreiber</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Suberin goes genomics use of a short living plant to investigate a long lasting polymer</div>, <div class="source">Front. Plant Sci<em>.</em></div> , <div class="year">2012</div>, vol. <div class="volume">3</div> pg. <div class="fpage">4</div> <div class="pub-id-doi"><a href="https://doi.org/10.3389/fpls.2012.00004" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.3389/fpls.2012.00004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Suberin%20goes%20genomics%20use%20of%20a%20short%20living%20plant%20to%20investigate%20a%20long%20lasting%20polymer&amp;author=R.B.%20Franke&amp;author=I.%20Dombrink&amp;author=L.%20Schreiber&amp;publication_year=2012&amp;journal=Front.%20Plant%20Sci.&amp;volume=3&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2012.00004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2012.00004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2012.00004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22639633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347526&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst042-dst042supp_table4.pdf?Expires=1509238924&amp;Signature=gopzDcdy5YsnKE9VTruZ603DVv22ZtM2xKfJQffnEUqZw316K5W-XNZtfacICO727iOafUkYl24l2EVl-ffGHqddERpHIdy7i19nVyTip7HsLecQw6Q-QYQZy6gfwdsUNbRAl93yTwaRRpNu~eTXtYGmZXi4tw5UJIQtIzdDrDWMBBXEKeFOqhByBY5GqpLcsqiBeQ3GPl7liswbQvpxyEnbWY~N4TL09mEFEQLSumvMlhZUzv-Oi4v2PQsMmtCydh4Eb~FLioPD1n9DBqZKCiUVW66tcnqqe4zUASebSD9DyWYL8TGvxU5gCwdyEPdY4RtfzpPnjYLYibQJgzFyzQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst042/2/dst042-dst042supp.doc?Expires=1509238924&amp;Signature=fiCGFebiWHqk7VRcnfrST0lewyEy17ObN2u4uxlGH4~EzEYxq6q2AwtZnS5GdEmyVWmxE28iCR1Wh82bp-ci39E8QTBJr2BkZF4wnEbxywLSYSn8kpgQow-Ufz8822lVtWfQqZK5LzkDUXBwRR7c2T50vByHSeZt9on7HzexpsiWqRL7wD5ovZ4pJgY8AbPDSKf9LuOasv-rLn3jBXmvwJmD1Tr4EpEfCW6gff~nZhHYrO99kfknHtdOYhPKCjUOFyo0qakTO6~5Bcg5nWggyFLsKflROjQFryFqb6RTFgddtKaggwfh8Rq1UKo5Juo59H0qZ5eYzQoEAWHo2Q0wxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">207</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dst042"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347526"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f1%2f69%2f347526">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897000006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F1%2F69%2F347526" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347526'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Deep Transcriptome Sequencing of Wild Halophyte Rice, Porteresia coarctata, Provides Novel Insights into the Salinity and Submergence Tolerance Factors" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f69%2f347526">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

