

<!DOCTYPE html>
<html lang="en">

<head><title>
	High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/1/85/347657","@type":"ScholarlyArticle","name":"High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc","datePublished":"2013-10-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2014-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst043","keywords":["bivalve","genome sequencing","2b-RAD genotyping","linkage mapping","quantitative trait locus mapping"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/1/85/347657","author":[{"name":"Jiao, Wenqian","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Fu, Xiaoteng","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Dou, Jinzhuang","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Li, Hengde","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China    2  The Center for Applied Aquatic Genomics ,  Chinese Academy of Fishery Sciences ,  150 Qingta West Road, Beijing 100141 ,  China","@type":"Person"},{"name":"Su, Hailin","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Mao, Junxia","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Yu, Qian","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Zhang, Lingling","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Hu, Xiaoli","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Huang, Xiaoting","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Wang, Yangfan","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Wang, Shi","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"},{"name":"Bao, Zhenmin","affiliation":"1  Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences ,  Ocean University of China ,  5 Yushan Road, Qingdao 266003 ,  China","@type":"Person"}],"description":"Genetic linkage maps are indispensable tools in genetic and genomic studies. Recent development of genotyping-by-sequencing (GBS) methods holds great promise for constructing high-resolution linkage maps in organisms lacking extensive genomic resources. In the present study, linkage mapping was conducted for a bivalve mollusc (Chlamys farreri) using a newly developed GBS method—2b-restriction site-associated DNA (2b-RAD). Genome survey sequencing was performed to generate a preliminary reference genome that was utilized to facilitate linkage and quantitative trait locus (QTL) mapping in C. farreri. A high-resolution linkage map was constructed with a marker density (3806) that has, to our knowledge, never been achieved in any other molluscs. The linkage map covered nearly the whole genome (99.5%) with a resolution of 0.41 cM. QTL mapping and association analysis congruously revealed two growth-related QTLs and one potential sex-determination region. An important candidate QTL gene named PROP1, which functions in the regulation of growth hormone production in vertebrates, was identified from the growth-related QTL region detected on the linkage group LG3. We demonstrate that this linkage map can serve as an important platform for improving genome assembly and unifying multiple genomic resources. Our study, therefore, exemplifies how to build up an integrative genomic framework in a non-model organism.","pageStart":"85","pageEnd":"101","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1572217325&Signature=VD~saBvAa7T395tlgrqf2rjcc2I6uQ9Aw1FAj7or74JXkp23F4TV54n3lFVShbJggFMdcR3jVCRWo~PXA~bMsk03xbaKwMy1gfF2E~RoodY8Rbp0IWmB5lmGCqdSG67th1Kol097547-0f0mbi6FTw0BujtTFhzFph0VytTssTv9KlcfFXCkMkPaY7iQIrHYh3wShn6wn4GflhN0DuBnJIkvjdlI3~N2Jsh4IffopkvEQ64IemsiYPOapj-Bs6Qy83Dn7t6ERx4ieyCgjotQdygvmz2XgLm6JThvcjPP38RGnZxQhvsKvSw8~xYcPU9W-z-f7g5Wl0GzMYkRt0Eniw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1572217325&Signature=VD~saBvAa7T395tlgrqf2rjcc2I6uQ9Aw1FAj7or74JXkp23F4TV54n3lFVShbJggFMdcR3jVCRWo~PXA~bMsk03xbaKwMy1gfF2E~RoodY8Rbp0IWmB5lmGCqdSG67th1Kol097547-0f0mbi6FTw0BujtTFhzFph0VytTssTv9KlcfFXCkMkPaY7iQIrHYh3wShn6wn4GflhN0DuBnJIkvjdlI3~N2Jsh4IffopkvEQ64IemsiYPOapj-Bs6Qy83Dn7t6ERx4ieyCgjotQdygvmz2XgLm6JThvcjPP38RGnZxQhvsKvSw8~xYcPU9W-z-f7g5Wl0GzMYkRt0Eniw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Genetic linkage maps are indispensable tools in genetic and genomic studies. Recent development of genotyping-by-sequencing (GBS) methods holds great promise for constructing high-resolution linkage maps in organisms lacking extensive genomic resources. In the present study, linkage mapping was conducted for a bivalve mollusc (Chlamys farreri) using a newly developed GBS method—2b-restriction site-associated DNA (2b-RAD). Genome survey sequencing was performed to generate a preliminary reference genome that was utilized to facilitate linkage and quantitative trait locus (QTL) mapping in C. farreri. A high-resolution linkage map was constructed with a marker density (3806) that has, to our knowledge, never been achieved in any other molluscs. The linkage map covered nearly the whole genome (99.5%) with a resolution of 0.41 cM. QTL mapping and association analysis congruously revealed two growth-related QTLs and one potential sex-determination region. An important candidate QTL gene named PROP1, which functions in the regulation of growth hormone production in vertebrates, was identified from the growth-related QTL region detected on the linkage group LG3. We demonstrate that this linkage map can serve as an important platform for improving genome assembly and unifying multiple genomic resources. Our study, therefore, exemplifies how to build up an integrative genomic framework in a non-model organism." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/1/85/347657" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217325&amp;Signature=NuzTnNtG~07rPBlNqGe5nWiTD2a3vFh3qYbwklOdco~215ZPvUwhMbJk1J9HK~fQtjxHQwt42iYhISAtLUIkKZevAhCikQdBQyFRlUtI6zrB~dxy4TY-CeR9ENdCtdXLMIA3BqiI0WuyJSpEAr0FZNHSYDd-wy5HRg2cvQ61qLYFyXUlP3-jVHmANGyxHsoO2CTLS~0I90G9apSCwxJq~TKFeR3YZ1JB3Kl0QRbyMSHK5gUqICrN7hnNo94yuKR~DZvxKFKwyv2wB77qN26nOmUYCYkzZcm2LZlmbGSrhgJvuGMPYQ7gCk9d5JcWcA7cKrygepZPX5EW-gEQy2AmVw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2013-10-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Jiao, Wenqian" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Fu, Xiaoteng" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Dou, Jinzhuang" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Li, Hengde" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author_institution" content="The Center for Applied Aquatic Genomics, Chinese Academy of Fishery Sciences, 150 Qingta West Road, Beijing 100141, China" /><meta name="citation_author" content="Su, Hailin" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Mao, Junxia" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Yu, Qian" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Zhang, Lingling" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Hu, Xiaoli" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Huang, Xiaoting" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Wang, Yangfan" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Wang, Shi" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_author" content="Bao, Zhenmin" /><meta name="citation_author_institution" content="Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences, Ocean University of China, 5 Yushan Road, Qingdao 266003, China" /><meta name="citation_title" content="High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc" /><meta name="citation_firstpage" content="85" /><meta name="citation_lastpage" content="101" /><meta name="citation_doi" content="10.1093/dnares/dst043" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2014/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=RADSeq: next-generation population genetics; Brief Funct. Genomics;  citation_year=2011;  citation_volume=9;  citation_pages=416-23; " /><meta name="citation_reference" content="citation_title=2b-RAD: a simple and flexible method for genome-wide genotyping; Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=808-10; " /><meta name="citation_reference" content="citation_title=Double digest RADseq: an inexpensive method for De Novo SNP discovery and genotyping in model and non-model species; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e37135" /><meta name="citation_reference" content="citation_title=The magnitude of global marine species diversity; Curr. Biol.;  citation_year=2012;  citation_volume=22;  citation_pages=2189-202; " /><meta name="citation_reference" content="citation_author=FAO; citation_publisher=FAO Fisheries and Aquaculture Department, FAO, Rome; The State of World Fisheries and Aquaculture 2010;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing and de novo analysis for Yesso scallop (Patinopecten yessoensis) using 454 GS FLX; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e21560" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of Zhikong scallop (Chlamys farreri) and comparative transcriptomic analysis with Yesso scallop (Patinopecten yessoensis); PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e63927" /><meta name="citation_reference" content="citation_title=Insights into shell deposition in the Antarctic bivalve Laternula elliptica: gene discovery in the mantle transcriptome using 454 pyrosequencing; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=362" /><meta name="citation_reference" content="citation_title=Gene expression profiling of genetically determined growth variation in bivalve larvae (Crassostrea gigas); J. Exp. Biol.;  citation_year=2010;  citation_volume=213;  citation_pages=749-58; " /><meta name="citation_reference" content="citation_title=De novo assembly of the Manila clam Ruditapes philippinarum transcriptome provides new insights into expression bias, mitochondrial doubly uniparental inheritance and sex determination; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=29;  citation_pages=771-86; " /><meta name="citation_reference" content="citation_title=Molecular characterization of TGF-β type I receptor gene (Tgfbr1) in Chlamys farreri, and the association of allelic variants with growth traits; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e51005" /><meta name="citation_reference" content="citation_title=Genetic diversity of the sulfotransferase-like gene and one nonsynonymous SNP associated with growth traits of clam, Meretrix meretrix; Mol. Biol. Rep.;  citation_year=2012;  citation_volume=39;  citation_pages=1323-31; " /><meta name="citation_reference" content="citation_title=Novel protein genes in animal mtDNA: a new sex determination system in freshwater mussels (Bivalvia: Unionoida)?; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=1645-59; " /><meta name="citation_reference" content="citation_title=AFLP linkage map of an intraspecific cross in Chlamys farreri; J. Shellfish Res.;  citation_year=2004;  citation_volume=23;  citation_pages=491-9; " /><meta name="citation_reference" content="citation_title=Construction of AFLP-based genetic linkage map for Zhikong scallop, Chlamys farreri Jones et Preston and mapping of sex-linked markers; Aquaculture;  citation_year=2005;  citation_volume=245;  citation_pages=63-73; " /><meta name="citation_reference" content="citation_title=A preliminary genetic map of Zhikong scallop (Chlamys farreri Jones et Preston 1904); Aquaculture Res.;  citation_year=2005;  citation_volume=36;  citation_pages=643-53; " /><meta name="citation_reference" content="citation_title=Construction of microsatellite-based linkage maps and identification of size-related quantitative trait loci for Zhikong scallop (Chlamys farreri); Anim. Genet.;  citation_year=2009;  citation_volume=40;  citation_pages=821-31; " /><meta name="citation_reference" content="citation_title=A BAC-based physical map of Zhikong scallop (Chlamys farreri Jones et Preston); PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e27612" /><meta name="citation_reference" content="citation_title=FISH mapping and identification of Zhikong scallop (Chlamys farreri) chromosomes; Mar. Biotechnol.;  citation_year=2008;  citation_volume=10;  citation_pages=151-7; " /><meta name="citation_reference" content="citation_title=Fosmid library construction and initial analysis of end sequences in Zhikong scallop (Chlamys farreri); Mar. Biotechnol.;  citation_year=2007;  citation_volume=9;  citation_pages=606-12; " /><meta name="citation_reference" content="citation_title=Construction and characterization of two bacterial artificial chromosome libraries of Zhikong scallop, Chlamys farreri Jones et Preston, and identification of BAC clones containing the genes involved in its innate immune system; Mar. Biotechnol.;  citation_year=2008;  citation_volume=10;  citation_pages=358-65; " /><meta name="citation_reference" content="citation_title=Expressed sequence tags from the Zhikong scallop (Chlamys farreri): discovery and annotation of host-defense genes; Fish Shellfish Immunol.;  citation_year=2009;  citation_volume=26;  citation_pages=744-50; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY; Molecular Cloning: A Laboratory Manual;  citation_year=1989; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences; Bioinformatics;  citation_year=2005;  citation_volume=14;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Joinmap; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=739-44; " /><meta name="citation_reference" content="citation_title=Accurate construction of consensus genetic maps via integer linear programming; IEEE/ACM Trans. Comput. Biol. Bioinform.;  citation_year=2011;  citation_volume=8;  citation_pages=381-94; " /><meta name="citation_reference" content="citation_publisher=Aarhus University, Foulum, Denmark; Using DMU for QTL mapping;  citation_year=2003; " /><meta name="citation_reference" content="citation_title=A quick method for computing approximate thresholds for quantitative trait loci detection; Genetics;  citation_year=2001;  citation_volume=157;  citation_pages=425-32; " /><meta name="citation_reference" content="citation_title=A quick method to calculate QTL confidence interval; J. Genet.;  citation_year=2011;  citation_volume=90;  citation_pages=355-60; " /><meta name="citation_reference" content="citation_title=GWAF: an R package for genome-wide association analyses with family data; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=580-1; " /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=The karyotyping of C. farreri; J. Ocean Univ. Qingdao;  citation_year=1990;  citation_volume=20;  citation_pages=81-5; " /><meta name="citation_reference" content="citation_title=A maximum likelihood method for estimating genome length using genetic linkage data; Genetics;  citation_year=1991;  citation_volume=128;  citation_pages=175-82; " /><meta name="citation_reference" content="citation_title=A genetic map in the Mimulus guttatus species complex reveals transmission ratio distortion due to heterospecific interactions; Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=1701-16; " /><meta name="citation_reference" content="citation_title=Mutations in PROP1 cause familial combined pituitary hormone deficiency; Nat. Genet.;  citation_year=1998;  citation_volume=18;  citation_pages=147-9; " /><meta name="citation_reference" content="citation_title=A trans-species missense SNP in Amhr2 is associated with sex determination in the tiger pufferfish, Takifugu rubripes (fugu); PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002798" /><meta name="citation_reference" content="citation_title=Draft genome of the pearl oyster Pinctada fucata: a platform for understanding bivalve biology; DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=117-30; " /><meta name="citation_reference" content="citation_title=The oyster genome reveals stress adaptation and complexity of shell formation; Nature;  citation_year=2012;  citation_volume=490;  citation_pages=49-54; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms and their relationship to codon usage bias in the Pacific oyster Crassostrea gigas; Gene;  citation_year=2007;  citation_volume=406;  citation_pages=13-22; " /><meta name="citation_reference" content="citation_title=Associations of DNA polymorphisms in growth hormone and its transcriptional regulators with growth and carcass traits in two populations of Brangus bulls; Genet. Mol. Res.;  citation_year=2007;  citation_volume=6;  citation_pages=222-37; " /><meta name="citation_reference" content="citation_title=A novel missense (A79V) mutation of goat PROP1 gene and its association with production traits; Mol. Biol. Rep.;  citation_year=2009;  citation_volume=36;  citation_pages=2069-73; " /><meta name="citation_reference" content="citation_title=The prophet of PIT1 gene variation and its effect on growth traits in Chinese indigeous goat; J. Anim. Vet. Adv.;  citation_year=2010;  citation_volume=9;  citation_pages=2940-6; " /><meta name="citation_reference" content="citation_title=Identification of SNPs within the sheep PROP1 gene and their effects on wool traits; Mol. Biol. Rep.;  citation_year=2011;  citation_volume=38;  citation_pages=2723-8; " /><meta name="citation_reference" content="citation_title=Signaling and transcriptional mechanisms in pituitary development; Annu. Rev. Neurosci.;  citation_year=2001;  citation_volume=24;  citation_pages=327-55; " /><meta name="citation_reference" content="citation_title=Karyotypes on the scallops Euvola ziczac and Nodipecten nodosus from the Gulf of Cariaco, Sucre State, Venezuela; Bol. Inst. Oceanogr. Venez.;  citation_year=2000;  citation_volume=39;  citation_pages=49-54; " /><meta name="citation_reference" content="citation_title=Insights into bilaterian evolution from three spiralian genomes; Nature;  citation_year=2013;  citation_volume=493;  citation_pages=526-31; " /><meta name="citation_reference" content="citation_title=Butterfly genome reveals promiscuous exchange of mimicry adaptations among species; citation_author=Heliconius Genome Consortium; Nature;  citation_year=2012;  citation_volume=487;  citation_pages=94-8; " /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping loci de novo from short-read sequences; G3 (Bethesda);  citation_year=2011;  citation_volume=1;  citation_pages=171-82; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/1/85/995787/dst043.pdf" /><meta name="description" content="Abstract.  Genetic linkage maps are indispensable tools in genetic and genomic studies. Recent development of genotyping-by-sequencing (GBS) methods holds great" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/1/85/347657" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/1/85/347657" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dst043';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347657" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF+3w8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkppYW8sIFdlbnFpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdSwgWGlhb3RlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEb3UsIEppbnpodWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBIZW5nZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgQ2VudGVyIGZvciBBcHBsaWVkIEFxdWF0aWMgR2Vub21pY3MsIENoaW5lc2UgQWNhZGVteSBvZiBGaXNoZXJ5IFNjaWVuY2VzLCAxNTAgUWluZ3RhIFdlc3QgUm9hZCwgQmVpamluZyAxMDAxNDEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3UsIEhhaWxpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hbywgSnVueGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWXUsIFFpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgTGluZ2xpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBNYXJpbmUgR2VuZXRpY3MgYW5kIEJyZWVkaW5nLCBDb2xsZWdlIG9mIE1hcmluZSBMaWZlIFNjaWVuY2VzLCBPY2VhbiBVbml2ZXJzaXR5IG9mIENoaW5hLCA1IFl1c2hhbiBSb2FkLCBRaW5nZGFvIDI2NjAwMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdSwgWGlhb2xpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHVhbmcsIFhpYW90aW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgTWFyaW5lIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBNYXJpbmUgTGlmZSBTY2llbmNlcywgT2NlYW4gVW5pdmVyc2l0eSBvZiBDaGluYSwgNSBZdXNoYW4gUm9hZCwgUWluZ2RhbyAyNjYwMDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgWWFuZ2ZhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhbywgWmhlbm1pbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIE1hcmluZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgTWFyaW5lIExpZmUgU2NpZW5jZXMsIE9jZWFuIFVuaXZlcnNpdHkgb2YgQ2hpbmEsIDUgWXVzaGFuIFJvYWQsIFFpbmdkYW8gMjY2MDAzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSGlnaC1SZXNvbHV0aW9uIExpbmthZ2UgYW5kIFF1YW50aXRhdGl2ZSBUcmFpdCBMb2N1cyBNYXBwaW5nIEFpZGVkIGJ5IEdlbm9tZSBTdXJ2ZXkgU2VxdWVuY2luZzogQnVpbGRpbmcgVXAgQW4gSW50ZWdyYXRpdmUgR2Vub21pYyBGcmFtZXdvcmsgZm9yIGEgQml2YWx2ZSBNb2xsdXNjIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDA0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkFEU2VxOiBuZXh0LWdlbmVyYXRpb24gcG9wdWxhdGlvbiBnZW5ldGljczsgQnJpZWYgRnVuY3QuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00MTYtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPTJiLVJBRDogYSBzaW1wbGUgYW5kIGZsZXhpYmxlIG1ldGhvZCBmb3IgZ2Vub21lLXdpZGUgZ2Vub3R5cGluZzsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz04MDgtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvdWJsZSBkaWdlc3QgUkFEc2VxOiBhbiBpbmV4cGVuc2l2ZSBtZXRob2QgZm9yIERlIE5vdm8gU05QIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyBpbiBtb2RlbCBhbmQgbm9uLW1vZGVsIHNwZWNpZXM7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzcxMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWFnbml0dWRlIG9mIGdsb2JhbCBtYXJpbmUgc3BlY2llcyBkaXZlcnNpdHk7IEN1cnIuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjE4OS0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1GQU87IGNpdGF0aW9uX3B1Ymxpc2hlcj1GQU8gRmlzaGVyaWVzIGFuZCBBcXVhY3VsdHVyZSBEZXBhcnRtZW50LCBGQU8sIFJvbWU7IFRoZSBTdGF0ZSBvZiBXb3JsZCBGaXNoZXJpZXMgYW5kIEFxdWFjdWx0dXJlIDIwMTA7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBhbmQgZGUgbm92byBhbmFseXNpcyBmb3IgWWVzc28gc2NhbGxvcCAoUGF0aW5vcGVjdGVuIHllc3NvZW5zaXMpIHVzaW5nIDQ1NCBHUyBGTFg7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjE1NjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgb2YgWmhpa29uZyBzY2FsbG9wIChDaGxhbXlzIGZhcnJlcmkpIGFuZCBjb21wYXJhdGl2ZSB0cmFuc2NyaXB0b21pYyBhbmFseXNpcyB3aXRoIFllc3NvIHNjYWxsb3AgKFBhdGlub3BlY3RlbiB5ZXNzb2Vuc2lzKTsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2MzkyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGludG8gc2hlbGwgZGVwb3NpdGlvbiBpbiB0aGUgQW50YXJjdGljIGJpdmFsdmUgTGF0ZXJudWxhIGVsbGlwdGljYTogZ2VuZSBkaXNjb3ZlcnkgaW4gdGhlIG1hbnRsZSB0cmFuc2NyaXB0b21lIHVzaW5nIDQ1NCBweXJvc2VxdWVuY2luZzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MzYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBleHByZXNzaW9uIHByb2ZpbGluZyBvZiBnZW5ldGljYWxseSBkZXRlcm1pbmVkIGdyb3d0aCB2YXJpYXRpb24gaW4gYml2YWx2ZSBsYXJ2YWUgKENyYXNzb3N0cmVhIGdpZ2FzKTsgSi4gRXhwLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTM7ICBjaXRhdGlvbl9wYWdlcz03NDktNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIE1hbmlsYSBjbGFtIFJ1ZGl0YXBlcyBwaGlsaXBwaW5hcnVtIHRyYW5zY3JpcHRvbWUgcHJvdmlkZXMgbmV3IGluc2lnaHRzIGludG8gZXhwcmVzc2lvbiBiaWFzLCBtaXRvY2hvbmRyaWFsIGRvdWJseSB1bmlwYXJlbnRhbCBpbmhlcml0YW5jZSBhbmQgc2V4IGRldGVybWluYXRpb247IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz03NzEtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIFRHRi3OsiB0eXBlIEkgcmVjZXB0b3IgZ2VuZSAoVGdmYnIxKSBpbiBDaGxhbXlzIGZhcnJlcmksIGFuZCB0aGUgYXNzb2NpYXRpb24gb2YgYWxsZWxpYyB2YXJpYW50cyB3aXRoIGdyb3d0aCB0cmFpdHM7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNTEwMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBvZiB0aGUgc3VsZm90cmFuc2ZlcmFzZS1saWtlIGdlbmUgYW5kIG9uZSBub25zeW5vbnltb3VzIFNOUCBhc3NvY2lhdGVkIHdpdGggZ3Jvd3RoIHRyYWl0cyBvZiBjbGFtLCBNZXJldHJpeCBtZXJldHJpeDsgTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTMyMy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgcHJvdGVpbiBnZW5lcyBpbiBhbmltYWwgbXRETkE6IGEgbmV3IHNleCBkZXRlcm1pbmF0aW9uIHN5c3RlbSBpbiBmcmVzaHdhdGVyIG11c3NlbHMgKEJpdmFsdmlhOiBVbmlvbm9pZGEpPzsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE2NDUtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFGTFAgbGlua2FnZSBtYXAgb2YgYW4gaW50cmFzcGVjaWZpYyBjcm9zcyBpbiBDaGxhbXlzIGZhcnJlcmk7IEouIFNoZWxsZmlzaCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDkxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBBRkxQLWJhc2VkIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIFpoaWtvbmcgc2NhbGxvcCwgQ2hsYW15cyBmYXJyZXJpIEpvbmVzIGV0IFByZXN0b24gYW5kIG1hcHBpbmcgb2Ygc2V4LWxpbmtlZCBtYXJrZXJzOyBBcXVhY3VsdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDU7ICBjaXRhdGlvbl9wYWdlcz02My03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcmVsaW1pbmFyeSBnZW5ldGljIG1hcCBvZiBaaGlrb25nIHNjYWxsb3AgKENobGFteXMgZmFycmVyaSBKb25lcyBldCBQcmVzdG9uIDE5MDQpOyBBcXVhY3VsdHVyZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NjQzLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgbWljcm9zYXRlbGxpdGUtYmFzZWQgbGlua2FnZSBtYXBzIGFuZCBpZGVudGlmaWNhdGlvbiBvZiBzaXplLXJlbGF0ZWQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIFpoaWtvbmcgc2NhbGxvcCAoQ2hsYW15cyBmYXJyZXJpKTsgQW5pbS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ODIxLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEJBQy1iYXNlZCBwaHlzaWNhbCBtYXAgb2YgWmhpa29uZyBzY2FsbG9wIChDaGxhbXlzIGZhcnJlcmkgSm9uZXMgZXQgUHJlc3Rvbik7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjc2MTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GSVNIIG1hcHBpbmcgYW5kIGlkZW50aWZpY2F0aW9uIG9mIFpoaWtvbmcgc2NhbGxvcCAoQ2hsYW15cyBmYXJyZXJpKSBjaHJvbW9zb21lczsgTWFyLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE1MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3NtaWQgbGlicmFyeSBjb25zdHJ1Y3Rpb24gYW5kIGluaXRpYWwgYW5hbHlzaXMgb2YgZW5kIHNlcXVlbmNlcyBpbiBaaGlrb25nIHNjYWxsb3AgKENobGFteXMgZmFycmVyaSk7IE1hci4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTYwNi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHR3byBiYWN0ZXJpYWwgYXJ0aWZpY2lhbCBjaHJvbW9zb21lIGxpYnJhcmllcyBvZiBaaGlrb25nIHNjYWxsb3AsIENobGFteXMgZmFycmVyaSBKb25lcyBldCBQcmVzdG9uLCBhbmQgaWRlbnRpZmljYXRpb24gb2YgQkFDIGNsb25lcyBjb250YWluaW5nIHRoZSBnZW5lcyBpbnZvbHZlZCBpbiBpdHMgaW5uYXRlIGltbXVuZSBzeXN0ZW07IE1hci4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0zNTgtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzIGZyb20gdGhlIFpoaWtvbmcgc2NhbGxvcCAoQ2hsYW15cyBmYXJyZXJpKTogZGlzY292ZXJ5IGFuZCBhbm5vdGF0aW9uIG9mIGhvc3QtZGVmZW5zZSBnZW5lczsgRmlzaCBTaGVsbGZpc2ggSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz03NDQtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSBQcmVzcywgQ29sZCBTcHJpbmcgSGFyYm9yLCBOWTsgTW9sZWN1bGFyIENsb25pbmc6IEEgTGFib3JhdG9yeSBNYW51YWw7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdNQVA6IGEgZ2Vub21pYyBtYXBwaW5nIGFuZCBhbGlnbm1lbnQgcHJvZ3JhbSBmb3IgbVJOQSBhbmQgRVNUIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xODU5LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgaW50ZWdyYXRlZCBnZW5ldGljIGxpbmthZ2UgbWFwcyBieSBtZWFucyBvZiBhIG5ldyBjb21wdXRlciBwYWNrYWdlOiBKb2lubWFwOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NzM5LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSBjb25zdHJ1Y3Rpb24gb2YgY29uc2Vuc3VzIGdlbmV0aWMgbWFwcyB2aWEgaW50ZWdlciBsaW5lYXIgcHJvZ3JhbW1pbmc7IElFRUUvQUNNIFRyYW5zLiBDb21wdXQuIEJpb2wuIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFhcmh1cyBVbml2ZXJzaXR5LCBGb3VsdW0sIERlbm1hcms7IFVzaW5nIERNVSBmb3IgUVRMIG1hcHBpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcXVpY2sgbWV0aG9kIGZvciBjb21wdXRpbmcgYXBwcm94aW1hdGUgdGhyZXNob2xkcyBmb3IgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZGV0ZWN0aW9uOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTc7ICBjaXRhdGlvbl9wYWdlcz00MjUtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcXVpY2sgbWV0aG9kIHRvIGNhbGN1bGF0ZSBRVEwgY29uZmlkZW5jZSBpbnRlcnZhbDsgSi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MzU1LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HV0FGOiBhbiBSIHBhY2thZ2UgZm9yIGdlbm9tZS13aWRlIGFzc29jaWF0aW9uIGFuYWx5c2VzIHdpdGggZmFtaWx5IGRhdGE7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTgwLTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNPQVAyOiBhbiBpbXByb3ZlZCB1bHRyYWZhc3QgdG9vbCBmb3Igc2hvcnQgcmVhZCBhbGlnbm1lbnQ7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTk2Ni03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUga2FyeW90eXBpbmcgb2YgQy4gZmFycmVyaTsgSi4gT2NlYW4gVW5pdi4gUWluZ2RhbzsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTgxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWF4aW11bSBsaWtlbGlob29kIG1ldGhvZCBmb3IgZXN0aW1hdGluZyBnZW5vbWUgbGVuZ3RoIHVzaW5nIGdlbmV0aWMgbGlua2FnZSBkYXRhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBtYXAgaW4gdGhlIE1pbXVsdXMgZ3V0dGF0dXMgc3BlY2llcyBjb21wbGV4IHJldmVhbHMgdHJhbnNtaXNzaW9uIHJhdGlvIGRpc3RvcnRpb24gZHVlIHRvIGhldGVyb3NwZWNpZmljIGludGVyYWN0aW9uczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTU5OyAgY2l0YXRpb25fcGFnZXM9MTcwMS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25zIGluIFBST1AxIGNhdXNlIGZhbWlsaWFsIGNvbWJpbmVkIHBpdHVpdGFyeSBob3Jtb25lIGRlZmljaWVuY3k7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTQ3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdHJhbnMtc3BlY2llcyBtaXNzZW5zZSBTTlAgaW4gQW1ocjIgaXMgYXNzb2NpYXRlZCB3aXRoIHNleCBkZXRlcm1pbmF0aW9uIGluIHRoZSB0aWdlciBwdWZmZXJmaXNoLCBUYWtpZnVndSBydWJyaXBlcyAoZnVndSk7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjc5OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBvZiB0aGUgcGVhcmwgb3lzdGVyIFBpbmN0YWRhIGZ1Y2F0YTogYSBwbGF0Zm9ybSBmb3IgdW5kZXJzdGFuZGluZyBiaXZhbHZlIGJpb2xvZ3k7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTE3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3lzdGVyIGdlbm9tZSByZXZlYWxzIHN0cmVzcyBhZGFwdGF0aW9uIGFuZCBjb21wbGV4aXR5IG9mIHNoZWxsIGZvcm1hdGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MDsgIGNpdGF0aW9uX3BhZ2VzPTQ5LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGFuZCB0aGVpciByZWxhdGlvbnNoaXAgdG8gY29kb24gdXNhZ2UgYmlhcyBpbiB0aGUgUGFjaWZpYyBveXN0ZXIgQ3Jhc3Nvc3RyZWEgZ2lnYXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDA2OyAgY2l0YXRpb25fcGFnZXM9MTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9ucyBvZiBETkEgcG9seW1vcnBoaXNtcyBpbiBncm93dGggaG9ybW9uZSBhbmQgaXRzIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0b3JzIHdpdGggZ3Jvd3RoIGFuZCBjYXJjYXNzIHRyYWl0cyBpbiB0d28gcG9wdWxhdGlvbnMgb2YgQnJhbmd1cyBidWxsczsgR2VuZXQuIE1vbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjIyLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIG1pc3NlbnNlIChBNzlWKSBtdXRhdGlvbiBvZiBnb2F0IFBST1AxIGdlbmUgYW5kIGl0cyBhc3NvY2lhdGlvbiB3aXRoIHByb2R1Y3Rpb24gdHJhaXRzOyBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0yMDY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcHJvcGhldCBvZiBQSVQxIGdlbmUgdmFyaWF0aW9uIGFuZCBpdHMgZWZmZWN0IG9uIGdyb3d0aCB0cmFpdHMgaW4gQ2hpbmVzZSBpbmRpZ2VvdXMgZ29hdDsgSi4gQW5pbS4gVmV0LiBBZHYuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yOTQwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFNOUHMgd2l0aGluIHRoZSBzaGVlcCBQUk9QMSBnZW5lIGFuZCB0aGVpciBlZmZlY3RzIG9uIHdvb2wgdHJhaXRzOyBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yNzIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ25hbGluZyBhbmQgdHJhbnNjcmlwdGlvbmFsIG1lY2hhbmlzbXMgaW4gcGl0dWl0YXJ5IGRldmVsb3BtZW50OyBBbm51LiBSZXYuIE5ldXJvc2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTMyNy01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2FyeW90eXBlcyBvbiB0aGUgc2NhbGxvcHMgRXV2b2xhIHppY3phYyBhbmQgTm9kaXBlY3RlbiBub2Rvc3VzIGZyb20gdGhlIEd1bGYgb2YgQ2FyaWFjbywgU3VjcmUgU3RhdGUsIFZlbmV6dWVsYTsgQm9sLiBJbnN0LiBPY2Vhbm9nci4gVmVuZXouOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NDktNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGludG8gYmlsYXRlcmlhbiBldm9sdXRpb24gZnJvbSB0aHJlZSBzcGlyYWxpYW4gZ2Vub21lczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5MzsgIGNpdGF0aW9uX3BhZ2VzPTUyNi0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnV0dGVyZmx5IGdlbm9tZSByZXZlYWxzIHByb21pc2N1b3VzIGV4Y2hhbmdlIG9mIG1pbWljcnkgYWRhcHRhdGlvbnMgYW1vbmcgc3BlY2llczsgY2l0YXRpb25fYXV0aG9yPUhlbGljb25pdXMgR2Vub21lIENvbnNvcnRpdW07IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODc7ICBjaXRhdGlvbl9wYWdlcz05NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFja3M6IGJ1aWxkaW5nIGFuZCBnZW5vdHlwaW5nIGxvY2kgZGUgbm92byBmcm9tIHNob3J0LXJlYWQgc2VxdWVuY2VzOyBHMyAoQmV0aGVzZGEpOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xNzEtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8xLzg1Lzk5NTc4Ny9kc3QwNDMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEdlbmV0aWMgbGlua2FnZSBtYXBzIGFyZSBpbmRpc3BlbnNhYmxlIHRvb2xzIGluIGdlbmV0aWMgYW5kIGdlbm9taWMgc3R1ZGllcy4gUmVjZW50IGRldmVsb3BtZW50IG9mIGdlbm90eXBpbmctYnktc2VxdWVuY2luZyAoR0JTKSBtZXRob2RzIGhvbGRzIGdyZWF0IiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDc2NTcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2RkU2CAxNCfm8+m9Vf7F07P7iKOHH8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f85%2f347657" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f85%2f347657" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=1509238914&amp;Signature=RJjHjupra9V5Y0rAuU0oALQAodj0pkOIZ50h2-Y-SrsBItj6ZYTRGmWM2MBdGoy4UZdmu214sGnQFX20odafL7p2r~dDFkTWfPsOcSZCrEYut8c3m10knVMcKhuMIvi7Uhv3GYXuKGLwwZ9vVr2uFpsnAJWJC4~sqNzh3Ce0QqZ~l--ZLvlzGq4MqxdJuVofEalw405hZixpoUMUe7iIH8F6X4qrPj3Gf6cEMr~PLKxuOItSWMEtefm4RypDyqhQVBPnbvLXon1chD5tCtUCFziPEZdX9mQ2eGeLrlbnLObkF6eufw0SPWArtJqU9isAakTljtaGwrsU5s~NsEMLbw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/1">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4926059">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926059">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926061">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926061">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926065">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926065">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926086">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926086">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926115">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926115">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926133">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926133">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926137">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926137">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926142">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926142">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4926147">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4926147">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/1/69/347526">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Wenqian Jiao</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wenqian Jiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Wenqian+Jiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiao W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Jiao author:W.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Xiaoteng Fu</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoteng Fu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xiaoteng+Fu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Fu author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Jinzhuang Dou</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jinzhuang Dou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jinzhuang+Dou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dou J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Dou author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hengde Li</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hengde Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">The Center for Applied Aquatic Genomics</div>, <div class="insititution">Chinese Academy of Fishery Sciences</div>, <div class="addr-line">150 Qingta West Road, Beijing 100141</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hengde+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Li author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hailin Su</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hailin Su</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hailin+Su">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Su H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Su author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Junxia Mao</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junxia Mao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Junxia+Mao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mao J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Mao author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Qian Yu</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qian Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Qian+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yu author:Q.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Lingling Zhang</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lingling Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Lingling+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Zhang author:L.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Xiaoli Hu</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoli Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xiaoli+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hu author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Xiaoting Huang</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoting Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xiaoting+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Huang author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Yangfan Wang</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yangfan Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yangfan+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wang author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Shi Wang</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shi Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 532-82031969. Fax. +86 532-82031969. Email: <a href="mailto:swang@ouc.edu.cn" target="_blank">swang@ouc.edu.cn</a> (S.W.); Tel. +86 532-82031960. Fax. +86 532-82031960. Email: <a href="mailto:zmbao@ouc.edu.cn" target="_blank">zmbao@ouc.edu.cn</a> (Z.B.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shi+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wang author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Zhenmin Bao</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhenmin Bao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Marine Genetics and Breeding, College of Marine Life Sciences</div>, <div class="insititution">Ocean University of China</div>, <div class="addr-line">5 Yushan Road, Qingdao 266003</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 532-82031969. Fax. +86 532-82031969. Email: <a href="mailto:swang@ouc.edu.cn" target="_blank">swang@ouc.edu.cn</a> (S.W.); Tel. +86 532-82031960. Fax. +86 532-82031960. Email: <a href="mailto:zmbao@ouc.edu.cn" target="_blank">zmbao@ouc.edu.cn</a> (Z.B.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Zhenmin+Bao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bao Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Bao author:Z.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 85–101, <a href='https://doi.org/10.1093/dnares/dst043'>https://doi.org/10.1093/dnares/dst043</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 October 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 May 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 September 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347657" href="/dnaresearch/article-pdf/21/1/85/995787/dst043.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Wenqian Jiao, Xiaoteng Fu, Jinzhuang Dou, Hengde Li, Hailin Su, Junxia Mao, Qian Yu, Lingling Zhang, Xiaoli Hu, Xiaoting Huang, Yangfan Wang, Shi Wang, Zhenmin Bao;  High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc, <em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 85–101, <a href='https://doi.org/10.1093/dnares/dst043'>https://doi.org/10.1093/dnares/dst043</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347657?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347657?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347657?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347657?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347657?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347657?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347657?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-Resolution%20Linkage%20and%20Quantitative%20Trait%20Locus%20Mapping%20Aided%20by%20Genome%20Survey%20Sequencing%3A%20Building%20Up%20An%20Integrative%20Genomic%20Framework%20for%20a%20Bivalve%20Mollusc&amp;publicationDate=2013-10-09&amp;volumeNum=21&amp;issueNum=1&amp;author=Jiao%2C%20Wenqian%3B%20Fu%2C%20Xiaoteng&amp;startPage=85&amp;endPage=101&amp;contentId=10.1093%2Fdnares%2Fdst043&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f85%2f347657" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4926059 id="4926059" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genetic linkage maps are indispensable tools in genetic and genomic studies. Recent development of genotyping-by-sequencing (GBS) methods holds great promise for constructing high-resolution linkage maps in organisms lacking extensive genomic resources. In the present study, linkage mapping was conducted for a bivalve mollusc (<em>Chlamys farreri</em>) using a newly developed GBS method—2b-restriction site-associated DNA (2b-RAD). Genome survey sequencing was performed to generate a preliminary reference genome that was utilized to facilitate linkage and quantitative trait locus (QTL) mapping in <em>C. farreri</em>. A high-resolution linkage map was constructed with a marker density (3806) that has, to our knowledge, never been achieved in any other molluscs. The linkage map covered nearly the whole genome (99.5%) with a resolution of 0.41 cM. QTL mapping and association analysis congruously revealed two growth-related QTLs and one potential sex-determination region. An important candidate QTL gene named <em>PROP1</em>, which functions in the regulation of growth hormone production in vertebrates, was identified from the growth-related QTL region detected on the linkage group LG3. We demonstrate that this linkage map can serve as an important platform for improving genome assembly and unifying multiple genomic resources. Our study, therefore, exemplifies how to build up an integrative genomic framework in a non-model organism.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="bivalve">bivalve</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;2b-RAD genotyping&quot;">2b-RAD genotyping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage mapping&quot;">linkage mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;quantitative trait locus mapping&quot;">quantitative trait locus mapping</a></div>                            <h2 scrollto-destination=4926061 id="4926061" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>High-resolution linkage maps are exceptionally valuable tools in many genetic and genomic applications, such as fine-scale quantitative trait locus (QTL) mapping, characterization of recombination hotspots, comparative genome analysis and genome scaffolding. A key prerequisite for constructing a high-resolution linkage map is the availability of a large number of genetic markers (preferably, single-nucleotide polymorphisms—SNPs), which has been unfortunately out of reach for most non-model organisms until just a few years ago due to the huge investments in large-scale marker discovery and array-based genotyping. Recent advances in the next-generation sequencing technologies have revolutionized our ability to obtain massive genomic resources in a rapid and cost-effective manner and, therefore, greatly stimulate the development of a variety of genotyping-by-sequencing (GBS) methods.<sup><a reveal-id="DST043C1" data-open="DST043C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Among available GBS methods, restriction site-associated DNA (RAD) has gained popularity for the construction of high-density linkage maps,<sup><a reveal-id="DST043C1" data-open="DST043C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and several methods with simpler RAD library preparation protocols have been developed.<sup><a reveal-id="DST043C2" data-open="DST043C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST043C3" data-open="DST043C3" class="link link-ref link-reveal xref-bibr">3</a></sup> For example, Wang <em>et al.</em><sup><a reveal-id="DST043C2" data-open="DST043C2" class="link link-ref link-reveal xref-bibr">2</a></sup> have recently developed the 2b-RAD method by adopting Type IIB restriction enzymes to simplify RAD library preparation. The streamlined 2b-RAD method features even and tunable genome coverage, providing a flexible genotyping platform to diminish the extensive sequencing effort required for species with large genomes.</p><p>Bivalves comprise ∼14 000 species worldwide,<sup><a reveal-id="DST043C4" data-open="DST043C4" class="link link-ref link-reveal xref-bibr">4</a></sup> constituting the second largest group of molluscs. Many bivalves are of economic importance and support both commercial fisheries and mariculture efforts. They make up an important source of food all over the world, with a production of over 11.7 million metric tons in 2008, corresponding to 22% of the global aquaculture production.<sup><a reveal-id="DST043C5" data-open="DST043C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Elucidation of genetic bases underlying economically important traits, such as growth and reproduction, has been a central task in bivalve genetic and breeding studies for the purpose of genetic improvement. Linkage maps have been constructed for many bivalve species with the aim for QTL mapping of important traits. However, the resolution of these maps is generally low (mostly 10–20 cM), thus limiting their usefulness in determining quantitative trait genes. Great progresses have been recently achieved by transcriptome sequencing for large-scale identification of candidate genes involved in bivalve growth and reproduction.<sup><a reveal-id="DST043C6  DST043C10" data-open="DST043C6  DST043C10" class="link link-ref link-reveal xref-bibr">6–10</a></sup> Polymorphisms in some candidate genes have also been associated with the variation of bivalve growth traits.<sup><a reveal-id="DST043C11" data-open="DST043C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST043C12" data-open="DST043C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Bivalves exhibit fascinatingly diverse sexual patterning. Some species are dioecious, whereas some are simultaneous hermaphrodites with a few as protandrous hermaphrodites (male when young, then later become female) or proterogynous hermaphrodites (female when young, then later become male). Currently, the mechanism of sex determination remains poorly understood in bivalves. Some recent studies have associated the sex determination of bivalves with doubly uniparental inheritance (DUI) of mitochondria<sup><a reveal-id="DST043C10" data-open="DST043C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST043C13" data-open="DST043C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and a hypothetical model has been proposed.<sup><a reveal-id="DST043C10" data-open="DST043C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, the generality of the proposed model is constrained because DUI is a peculiar mtDNA inheritance system that exists only in a few bivalve species.</p><p>The scallop <em>Chlamys farreri</em> (Jones et Preston 1904) naturally distributes along the seacoasts of China, Japan and Korea, and is a commercially important bivalve species in China. With increasing acreage devoted to artificial farming, several issues such as declining production and frequent disease outbreaks have raised great concern in modern scallop aquaculture. Currently, genetic studies focusing on scallop growth, reproduction and immunity represent active research directions. Genetic and genomic resources for <em>C. farreri</em> have rapidly expanded, including low-density linkage,<sup><a reveal-id="DST043C14  DST043C17" data-open="DST043C14  DST043C17" class="link link-ref link-reveal xref-bibr">14–17</a></sup> physical<sup><a reveal-id="DST043C18" data-open="DST043C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and cytogenetic maps,<sup><a reveal-id="DST043C19" data-open="DST043C19" class="link link-ref link-reveal xref-bibr">19</a></sup> fosmid and bacterial artificial chromosome (BAC) libraries,<sup><a reveal-id="DST043C20" data-open="DST043C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST043C21" data-open="DST043C21" class="link link-ref link-reveal xref-bibr">21</a></sup> a large number of expressed sequence tags.<sup><a reveal-id="DST043C7" data-open="DST043C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST043C22" data-open="DST043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The direction of the present study was also shaped by our immediate need for building up an integrative genomic platform that unifies a variety of genomic resources, such as genome map, linkage map and physical map, and facilitates future genetic, genomic and breeding studies on <em>C. farreri</em>.</p>                            <h2 scrollto-destination=4926065 id="4926065" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4926066 id="4926066" class="section-title" data-legacy-id=s2a>2.1. Genome survey sequencing and gene annotation</h3>
<p>A two-year-old <em>C. farreri</em> individual was chosen for genome survey sequencing. Genomic DNA was extracted from its adductor muscle using the standard phenol/chloroform extraction method.<sup><a reveal-id="DST043C23" data-open="DST043C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Three paired-end DNA libraries with insert sizes of 175, 500 and 800 bp were constructed by following the Illumina's standard DNA library preparation protocol and were then sequenced using the Illumina HiSeq2000 platform.</p><p>Raw reads were first filtered to remove low-quality reads resulting from base-calling duplications or adapter contamination. Clean reads were assembled using the SOAPdenovo software, which applies the <em>de Bruijn</em> graph structure to construct contigs. Subsequently, all reads were realigned to the contigs and the paired-end information was used to join unique contigs into scaffolds. Finally, intra-scaffold gaps were filled using paired-end extracted reads that had one read uniquely aligned on a contig and another read located in a gap region. Genomic sequences were archived in the Sequence Read Archive (SRA) database (accession no. SRP018107).</p><p>Our group has recently reported the most comprehensive transcriptome resource for <em>C. farreri</em> by 454 sequencing of cDNA libraries generated from diverse developmental stages and adult tissues,<sup><a reveal-id="DST043C7" data-open="DST043C7" class="link link-ref link-reveal xref-bibr">7</a></sup> which provide an excellent basis for gene annotation of the obtained scaffolds. Genomic scaffolds were first annotated by aligning the 20 056 transcriptomic isogroup sequences (i.e. the longest isotig in each isogroup) onto these scaffolds using the GMAP program.<sup><a reveal-id="DST043C24" data-open="DST043C24" class="link link-ref link-reveal xref-bibr">24</a></sup> To increase the annotation rate, the unannotated scaffolds were compared against the Swiss-Prot and non-redundant (NR) protein databases using BlastX, with an <em>E</em>-value threshold of 1<em>E</em>−4. For scaffolds without significant matches, tBlastX search with an <em>E</em>-value threshold of 1<em>E</em>−4 was conducted against the NR nucleotide database for further annotation. To increase computational speed, all Blast searches were limited to the top 10 significant hits for each query.</p>                            <h3 scrollto-destination=4926070 id="4926070" class="section-title" data-legacy-id=s2b>2.2. Mapping families</h3>
<p>Twenty full-sib families were established in May 2009 by single-pair mating of ‘Penglai-Red’ scallops, an elite <em>C. farreri</em> strain developed by our group with features of red shell, fast growth and disease resistance. Growth-related traits including shell length, shell width, shell height and body weight were measured for all families at the age of 15 months. One of these families exhibiting high within-family variation of growth traits was chosen for linkage and QTL analysis.</p>                            <h3 scrollto-destination=4926072 id="4926072" class="section-title" data-legacy-id=s2c>2.3. 2b-RAD sequencing</h3>
<p>2b-RAD libraries were prepared for two parents and 96 progenies by following the protocol developed by Wang <em>et al.</em><sup><a reveal-id="DST043C2" data-open="DST043C2" class="link link-ref link-reveal xref-bibr">2</a></sup> For the parents, standard BsaXI libraries were constructed, whereas for the progenies, reduced representation (RR) libraries were constructed using adaptors with 5′-NNT-3′ overhangs to target a subset of all BsaXI fragments in the <em>C. farreri</em> genome. A unique barcode was incorporated into each library during library preparation, and then all libraries were pooled for single-end sequencing (1 × 50 bp) using an Illumina GA-II sequencer. All the 2b-RAD sequences were archived in the SRA database (accession no. SRA065207).</p>                            <h3 scrollto-destination=4926074 id="4926074" class="section-title" data-legacy-id=s2d>2.4. Sequence data preprocessing and <em>de novo</em> genotyping</h3>
<p>Raw reads were first trimmed to remove adaptor sequences. The terminal 3-bp positions were excluded from each read to eliminate artefacts that might have arisen at ligation sites. Reads with no restriction sites or containing ambiguous base calls (N), long homopolymer regions (&gt;10 bp), excessive numbers of low-quality positions (&gt;5 positions with quality of &lt;10) or mitochondrial origins were removed. The remaining trimmed, high-quality reads formed the basis for subsequent analysis.</p><p><em>De novo</em> 2b-RAD genotyping was performed using the RADtyping program v1.0 (<a class="link link-uri" href="http://www2.ouc.edu.cn/mollusk/detailen.asp?Id=727">http://www2.ouc.edu.cn/mollusk/detailen.asp?Id=727</a>) under default parameters. The RADtyping program has recently been developed by our group, and it is an integrated pipeline that enables both accurate <em>de novo</em> codominant and dominant genotyping in mapping populations.</p>                            <h3 scrollto-destination=4926077 id="4926077" class="section-title" data-legacy-id=s2e>2.5. Linkage map construction</h3>
<p>Segregating markers that could be genotyped in at least 80% of the individuals were considered as high-quality markers and were retained for further analysis. For each segregating marker, goodness of fit of the observed with the expected Mendelian ratio was assessed using the <em>χ</em><sup>2</sup> test, and those markers conforming to the expected Mendelian ratios (<em>P</em> ≥ 0.05) were included in map construction. Linkage analysis was performed using the JoinMap 4.0 software.<sup><a reveal-id="DST043C25" data-open="DST043C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Sex-specific maps were first constructed for each parent. Maternal and paternal datasets were created using the function ‘Create Maternal and Paternal Population Nodes’ in the JoinMap program, which was also used to partition 1:2:1-type data into 1:1 female-type and 1:1 male-type data. A logarithm of odds (LOD) threshold of at least 6.0 was used to assign markers into linkage groups. The regression mapping algorithm was selected for map construction. The Kosambi mapping function was used to convert the recombination frequencies into map distances (centiMorgans). A consensus map was established by integrating the sex-specific maps through shared markers using the MergeMap software.<sup><a reveal-id="DST043C26" data-open="DST043C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=4926079 id="4926079" class="section-title" data-legacy-id=s2f>2.6. QTL mapping and association analyses of growth traits and sex</h3>
<p>The distributions of growth traits were first assessed using the univariate procedure of MATLAB software to check whether they followed normal distributions. Pearson correlations among four growth traits were calculated using pairwise comparison.</p><div class="block-child-p">The regression method was used for QTL mapping of both growth traits and sex. Here, sex was used as a binary trait, supposing the haplotypes of the male and female parents are H<sub>pp</sub>, H<sub>pm</sub>, H<sub>mp</sub> and H<sub>mm</sub>, that <em>a</em><sub>f</sub>, <em>−a</em><sub>f</sub>, <em>a</em><sub>m</sub> and <em>−a</em><sub>m</sub> are their respective effects on the trait of interest, and <em>P</em><sub>pp</sub>, <em>P</em><sub>pm</sub>, <em>P</em><sub>mp</sub> and <em>P</em><sub>mm</sub> are the respective individual inheritance probabilities at putative positions in the corresponding parental haplotypes. Following the above assumption, <em>P</em><sub>pp</sub> + <em>P</em><sub>pm</sub> = <em>P</em><sub>mp</sub> + <em>P</em><sub>mm</sub> = 1. Therefore, the hypothesis to detect each QTL is: <a id="DST043M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST043M1"><div class="tex-math display-math">$$y = \mu + s + (P_{{\rm pp}} - P_{{\rm pm}} )a_{\rm f} + (P_{{\rm mp}} + P_{{\rm mm}} )a_{\rm m} + e$$</div></div><div class="label">(1)</div></div> and its null hypothesis is: <a id="DST043M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST043M2"><div class="tex-math display-math">$$y = \mu + s + e$$</div></div><div class="label">(2)</div></div> where <em>y</em> is the observed value of the trait of interest; <em>μ</em>, the population mean; <em>s</em>, the fixed effect of sex and <em>e</em>, the random error.</div><div class="block-child-p">For each chromosome, midpoints of all marker brackets along the chromosome were considered to be putative QTL positions, and the inheritance probabilities for individuals at each position were calculated using the IBD program<sup><a reveal-id="DST043C27" data-open="DST043C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and the corresponding LOD value was determined by: <a id="DST043M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST043M3"><div class="tex-math display-math">$$\hbox{LOD} = \displaystyle{1 \over 2}(\log _{10} e)\hbox{LRT} = 0.217\hbox{LRT}$$</div></div><div class="label">(3)</div></div> where the LRT statistic was calculated through the comparison of Equations (5) and (6). Chromosome-wide and genome-wide critical threshold values for QTL detection were approximated using Piepho's method.<sup><a reveal-id="DST043C28" data-open="DST043C28" class="link link-ref link-reveal xref-bibr">28</a></sup> If a QTL was detected, the confidence interval was calculated using Li's method.<sup><a reveal-id="DST043C29" data-open="DST043C29" class="link link-ref link-reveal xref-bibr">29</a></sup></div><p>Association analysis was performed as a complement approach to QTL mapping, which was implemented using an R package called ‘GWAF’<sup><a reveal-id="DST043C30" data-open="DST043C30" class="link link-ref link-reveal xref-bibr">30</a></sup> that tests genetic association between genotypes and traits by fitting a linear model accounting for within full-sib pedigree correlation. The additive model in GWAF was selected by use of the kinship coefficient matrix in the analysis to avoid false positives owing to unexpected familial correlation. The functions ‘lme.batch’ and ‘gee.lgst.batch’ were used to perform a global test (i.e. Wald <em>χ</em><sup>2</sup> test) of genotypic effects with growth traits and sex, respectively. Bonferroni correction was carried out to account for multiple comparisons.</p>                            <h3 scrollto-destination=4926084 id="4926084" class="section-title" data-legacy-id=s2g>2.7. Integration of the linkage map, genomic scaffolds and a physical map</h3>
<p>To associate genomic scaffolds with the linkage map, all markers in the linkage map were mapped onto the scaffolds using the SOAP2 software<sup><a reveal-id="DST043C31" data-open="DST043C31" class="link link-ref link-reveal xref-bibr">31</a></sup> (parameters –M 4, -v 2). Marker-associated scaffolds were further used as linkers to integrate the linkage map with a BAC-based physical map previously constructed for <em>C. farreri.</em><sup><a reveal-id="DST043C18" data-open="DST043C18" class="link link-ref link-reveal xref-bibr">18</a></sup> These scaffold sequences were compared against available BAC-end sequences (BESs) using BlastN with an <em>E</em>-value cut-off of 1<em>E</em>−10. A reciprocal comparison was performed to ensure one-to-one matches between the scaffolds and BAC contigs.</p>                            <h2 scrollto-destination=4926086 id="4926086" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4926087 id="4926087" class="section-title" data-legacy-id=s3a>3.1. Genome survey sequencing of <em>C. farreri</em></h3>
<p>The genome size of <em>C. farreri</em> is relatively large and was previously estimated as ∼1.2 Gb.<sup><a reveal-id="DST043C20" data-open="DST043C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Genome survey sequencing was performed for a single <em>C. farreri</em> individual by sequencing three paired-end DNA libraries with insert sizes of 175, 500 and 800 bp based on the Illumina HiSeq2000 platform. In total, 62.4 Gb of sequencing data, equivalent to ∼52× genome coverage, were produced (Table <a reveal-id="DST043TB1" data-open="DST043TB1" class="link link-reveal link-table xref-fig">1</a>). <em>K</em>-mer analysis revealed remarkably high heterozygosity (∼1.4%) in the <em>C. farreri</em> genome (Fig. <a reveal-id="DST043F1" data-open="DST043F1" class="link link-reveal link-table xref-fig">1</a>). Owing to the high genome heterozygosity, assembly of these data produced a large number of small scaffolds (N50 = 1.5 kb), with the longest one only reached to 46 kb (Table <a reveal-id="DST043TB2" data-open="DST043TB2" class="link link-reveal link-table xref-fig">2</a>). All the scaffold sequences are accessible at <a class="link link-uri" href="http://ipl.ouc.edu.cn/fuxiaoteng/cf_SRA_data">http://ipl.ouc.edu.cn/fuxiaoteng/cf_SRA_data</a>. Of 20 056 isogroups in the <em>C. farreri</em> transcriptome,<sup><a reveal-id="DST043C7" data-open="DST043C7" class="link link-ref link-reveal xref-bibr">7</a></sup> 18 316 (91.3%) could match with 16 489 genomic scaffolds, suggesting that the majority of genes were present in our reference genome dataset. Of the mapped transcriptomic isogroups, 88.4% had been functionally annotated through the homology-based comparison against public databases. Summing up all scaffolds resulted in a total length of ∼870 Mb, corresponding to ∼70% of genome coverage; though not complete, it constitutes the most informative reference genome currently available for <em>C. farreri</em>. </p>                        <a id="4926089" scrollto-destination="4926089"></a>
<div class="table-wrap"><div id="DST043TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of genome survey sequencing data for <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Paired-end libraries </th><th>Read length (bp) </th><th>Insert size (bp) </th><th>Total data (G) </th><th>Sequencing depth (X) </th></tr></thead><tbody><tr><td>L1 </td><td>100 </td><td>180 </td><td>32.7 </td><td>27.24 </td></tr><tr><td>L2 </td><td>100 </td><td>500 </td><td>21.4 </td><td>17.82 </td></tr><tr><td>L3 </td><td>100 </td><td>800 </td><td>8.4 </td><td>6.98 </td></tr><tr><td>Total </td><td>– </td><td>– </td><td>62.5 </td><td>52.04 </td></tr></tbody></table></div><div content-id="DST043TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Paired-end libraries </th><th>Read length (bp) </th><th>Insert size (bp) </th><th>Total data (G) </th><th>Sequencing depth (X) </th></tr></thead><tbody><tr><td>L1 </td><td>100 </td><td>180 </td><td>32.7 </td><td>27.24 </td></tr><tr><td>L2 </td><td>100 </td><td>500 </td><td>21.4 </td><td>17.82 </td></tr><tr><td>L3 </td><td>100 </td><td>800 </td><td>8.4 </td><td>6.98 </td></tr><tr><td>Total </td><td>– </td><td>– </td><td>62.5 </td><td>52.04 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4926089" target="_blank">View Large</a></div></div>                        <a id="4926090" scrollto-destination="4926090"></a>
<div class="table-wrap"><div id="DST043TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Statistics of genome assembly for <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"><em>K</em>-mer = 37 </th><th colspan="2">Contig<hr /> </th><th colspan="2">Scaffold<hr /> </th></tr><tr><th>Size (bp) </th><th>Number </th><th>Size (bp) </th><th>Number </th></tr></thead><tbody><tr><td>N90 </td><td>330 </td><td>708,971 </td><td>395 </td><td>595,809 </td></tr><tr><td>N80 </td><td>503 </td><td>497,805 </td><td>603 </td><td>420,096 </td></tr><tr><td>N70 </td><td>713 </td><td>353,715 </td><td>851 </td><td>298,664 </td></tr><tr><td>N60 </td><td>960 </td><td>249,175 </td><td>1142 </td><td>210,001 </td></tr><tr><td>N50 </td><td>1267 </td><td>170,707 </td><td>1517 </td><td>143,665 </td></tr><tr><td>Average length </td><td>807 </td><td>– </td><td>958 </td><td>– </td></tr><tr><td>Longest </td><td>36,486 </td><td>– </td><td>46,176 </td><td>– </td></tr><tr><td>Total length </td><td>863,450,239 </td><td>– </td><td>870,802,763 </td><td>– </td></tr><tr><td>Total number </td><td>1,069,270 </td><td>– </td><td>908,703 </td><td>– </td></tr><tr><td>Length &gt;100 bp </td><td>1,069,270 </td><td>– </td><td>908,703 </td><td>– </td></tr><tr><td>Length &gt;2 kb </td><td>82,556 </td><td>– </td><td>94,956 </td><td>– </td></tr></tbody></table></div><div content-id="DST043TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"><em>K</em>-mer = 37 </th><th colspan="2">Contig<hr /> </th><th colspan="2">Scaffold<hr /> </th></tr><tr><th>Size (bp) </th><th>Number </th><th>Size (bp) </th><th>Number </th></tr></thead><tbody><tr><td>N90 </td><td>330 </td><td>708,971 </td><td>395 </td><td>595,809 </td></tr><tr><td>N80 </td><td>503 </td><td>497,805 </td><td>603 </td><td>420,096 </td></tr><tr><td>N70 </td><td>713 </td><td>353,715 </td><td>851 </td><td>298,664 </td></tr><tr><td>N60 </td><td>960 </td><td>249,175 </td><td>1142 </td><td>210,001 </td></tr><tr><td>N50 </td><td>1267 </td><td>170,707 </td><td>1517 </td><td>143,665 </td></tr><tr><td>Average length </td><td>807 </td><td>– </td><td>958 </td><td>– </td></tr><tr><td>Longest </td><td>36,486 </td><td>– </td><td>46,176 </td><td>– </td></tr><tr><td>Total length </td><td>863,450,239 </td><td>– </td><td>870,802,763 </td><td>– </td></tr><tr><td>Total number </td><td>1,069,270 </td><td>– </td><td>908,703 </td><td>– </td></tr><tr><td>Length &gt;100 bp </td><td>1,069,270 </td><td>– </td><td>908,703 </td><td>– </td></tr><tr><td>Length &gt;2 kb </td><td>82,556 </td><td>– </td><td>94,956 </td><td>– </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4926090" target="_blank">View Large</a></div></div>                        <a id="4926091" scrollto-destination="4926091"></a>
<div id="DST043F1" data-id="DST043F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04301.png?Expires=1509238925&Signature=RD2-p28tUgSXk26VimRV9A8S6do5cjsgfFk8lWYoIxg~mMctJVqrttrLCCp61kEEOEM9NgzhYARWRYsV7ZvlwRUvz40q~CAf2Iunlkg-7znUr97BXPPR3w7XTVEzFx979Me-tThpslLPyxNJlOJNdxEWUqZqt9jCCaQKSYXCDNJTgmLQPBqmWA9ZIq5wIc7JCXoCEk2eX6qGuckkeMOIjxGIFhQ9JUE-yx7kOqES5TXuE8vsz1q7YOo8YVarEolpkd3qvRxQYZNcBl4OlWEF5CyUoaoZWCBWnKTOU8Us79HoDbhemQwS-xPty6wX0RKyMs99FzVtYtPoZ14tFQbCXA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="17-mer frequency distribution of sequencing reads. All 17-mer sequences were extracted from high-quality, paired-end reads generated by sequencing three genomic libraries with different insert sizes. The frequency of each 17-mer was calculated and plotted. Two peaks rather than one were observed, indicating high genome heterozygosity." path-from-xml="dst04301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926091/dst04301.tiff" path-from-xml="dst04301.tiff" target="_blank">View large</a><a section="4926091" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04301.png?Expires=1509238925&Signature=ZWZPvoT2YHDpi247wHmyCOwSi5yLwOHglz-BNC39irS7hFeRiXyB4gXB8XAIGp7lnk91hY08mr3DxxMnb4Cx9t~VH9uoJsx5yGqTYnb46oevlOiPyvCF0yMl1tdgTCTUmImB03bKsoZF7CjHIlu1ILIoebGSqGj64ygDTgcY4GdB9IwaNIN0gFlK1BlFdd0QA13tfv3Wrf1pob7nlYZv1XLrudrDEe4QosewFqcKzP0qVU8t4Z2PcPUxeQ2XVoYcVNV~HRMBvtnwWzvoAGb0~iVL47S2I8kO~znNolgNuEyZORJz5409ccdiBsbI~UH-A87kSMRn3Gxc-O4Ax4O3Mg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926091&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>17-mer frequency distribution of sequencing reads. All 17-mer sequences were extracted from high-quality, paired-end reads generated by sequencing three genomic libraries with different insert sizes. The frequency of each 17-mer was calculated and plotted. Two peaks rather than one were observed, indicating high genome heterozygosity.</p></div></div><div content-id="DST043F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04301.png?Expires=1509238925&Signature=RD2-p28tUgSXk26VimRV9A8S6do5cjsgfFk8lWYoIxg~mMctJVqrttrLCCp61kEEOEM9NgzhYARWRYsV7ZvlwRUvz40q~CAf2Iunlkg-7znUr97BXPPR3w7XTVEzFx979Me-tThpslLPyxNJlOJNdxEWUqZqt9jCCaQKSYXCDNJTgmLQPBqmWA9ZIq5wIc7JCXoCEk2eX6qGuckkeMOIjxGIFhQ9JUE-yx7kOqES5TXuE8vsz1q7YOo8YVarEolpkd3qvRxQYZNcBl4OlWEF5CyUoaoZWCBWnKTOU8Us79HoDbhemQwS-xPty6wX0RKyMs99FzVtYtPoZ14tFQbCXA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="17-mer frequency distribution of sequencing reads. All 17-mer sequences were extracted from high-quality, paired-end reads generated by sequencing three genomic libraries with different insert sizes. The frequency of each 17-mer was calculated and plotted. Two peaks rather than one were observed, indicating high genome heterozygosity." path-from-xml="dst04301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926091/dst04301.tiff" path-from-xml="dst04301.tiff" target="_blank">View large</a><a section="4926091" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04301.png?Expires=1509238925&Signature=ZWZPvoT2YHDpi247wHmyCOwSi5yLwOHglz-BNC39irS7hFeRiXyB4gXB8XAIGp7lnk91hY08mr3DxxMnb4Cx9t~VH9uoJsx5yGqTYnb46oevlOiPyvCF0yMl1tdgTCTUmImB03bKsoZF7CjHIlu1ILIoebGSqGj64ygDTgcY4GdB9IwaNIN0gFlK1BlFdd0QA13tfv3Wrf1pob7nlYZv1XLrudrDEe4QosewFqcKzP0qVU8t4Z2PcPUxeQ2XVoYcVNV~HRMBvtnwWzvoAGb0~iVL47S2I8kO~znNolgNuEyZORJz5409ccdiBsbI~UH-A87kSMRn3Gxc-O4Ax4O3Mg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926091&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>17-mer frequency distribution of sequencing reads. All 17-mer sequences were extracted from high-quality, paired-end reads generated by sequencing three genomic libraries with different insert sizes. The frequency of each 17-mer was calculated and plotted. Two peaks rather than one were observed, indicating high genome heterozygosity.</p></div></div>                            <h3 scrollto-destination=4926092 id="4926092" class="section-title" data-legacy-id=s3b>3.2. Optimization of a 2b-RAD sequencing plan for linkage mapping</h3>
<p>Because of the large genome size (∼1.2 Gb) of <em>C. farreri</em>, it is necessary to devise a cost-effective sequencing plan that can balance the total sequencing cost and genotyping accuracy. In total, 210 570 BsaXI sites were identified from the reference genome dataset. Based on the genome coverage (∼70%) of this dataset, the total number of BsaXI sites in the <em>C. farreri</em> genome was estimated as ∼0.3 million. Sequencing all of the BsaXI sites to 20× for a large mapping population (e.g. hundreds of individuals) would require a large sequencing investment. A notable feature of the 2b-RAD method is its ability to fine tune marker density as needed by constructing RR libraries using less-degenerate adaptors.<sup><a reveal-id="DST043C2" data-open="DST043C2" class="link link-ref link-reveal xref-bibr">2</a></sup> For example, an RR library constructed using adaptors with 5′-NNT-3′ overhangs only targets ∼1/10th of all the BsaXI sites in the <em>C. farreri</em> genome, and sequencing of such a library to 20× would require only 0.6 million reads per individual (in contrast to 6 million reads per individual for a standard BsaXI library), thus dramatically reducing the total sequencing cost.</p>                            <h3 scrollto-destination=4926094 id="4926094" class="section-title" data-legacy-id=s3c>3.3. 2b-RAD sequencing of a <em>C. farreri</em> mapping population</h3>
<p>Based on the calculations above, RR libraries were constructed for a <em>C. farreri</em> mapping population, sequencing of which produced 0.6–1.2 million high-quality reads per progeny. For the parents, standard libraries were constructed and sequenced to a sufficient depth (70–80×). As shown in Fig. <a reveal-id="DST043F2" data-open="DST043F2" class="link link-reveal link-table xref-fig">2</a>, target sites of the RR libraries could be reproducibly detected among different progenies and their average sequencing coverage was ∼6-fold higher than that of standard libraries. The sequencing depth for progenies ranged from 13.4 to 23.8 with an average of 16.75. Clustering parental reads resulted in 181 625 representative reference sites consisting of 138 141 parent-shared sites and 43 484 parent-specific sites. After filtering low-quality sites, 117 113 parent-shared and 35 799 parent-specific sites remained. These reference sites contained 92% of the unique BsaXI sites inferred from the scaffolds, suggesting that unique sites were well represented in the high-quality reference sites. </p>                        <a id="4926096" scrollto-destination="4926096"></a>
<div id="DST043F2" data-id="DST043F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04302.png?Expires=1509238925&Signature=SUREWl-mZbsMVce1nIPPZMvAYrficknfhEfarquepsUmPMBcc8w0pYa~xpyNKlnKHVNyjclbevEsDhGbFA5GyjU2w0EhOwUt8BoEY2i1znqb-E~q2H9QadXgHGV0a6XqKOfpxgE2ldbUHo4uRxlMaXHYkSHyDKQbITGJnrfAuGFdfIgfPE0iNKJplYc9zx~PvX8ANubfzdCq1477qX~tE2nyxava9SMxv05d3bENGo0mNYGCQPLLmteXIzEQ~xFieQ~WfnI7aWJ~kmZ6cfKF7WLVvII4fdXguX6j~wyRuPh9D~lLqr4e7B8Dh4MjiwiAgE-OxxPhaHXMPHKUXwz78g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of sequencing coverage for target sites derived from 365 randomly chosen C. farreri scaffolds (concatenated). For parents, sequencing data were generated from standard BsaXI libraries, whereas for progenies, RR libraries were sequenced. Target sites of the RR libraries could be reproducibly sequenced among progenies and their sequencing coverage was ∼6-fold higher than for standard libraries." path-from-xml="dst04302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926096/dst04302.tiff" path-from-xml="dst04302.tiff" target="_blank">View large</a><a section="4926096" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04302.png?Expires=1509238925&Signature=RM-09x~ucu1mnNA0okV7j5H8-X~08d0cMZ7vdx0S5jY3paySOYYQ3k~0fgVbsENzW-8sx5Pcn4QQXK6DjQAneGygy5Nv5MKbYKzj3FmRfVdOsxF6gWdupMM-A6Low1P0vX-b60J7-TacyRE0ObJeknyHt00IgkXZTo10tdTqHrbGgsGIYzubM5yovwZEr-VdDi5GxY1TP2Q1o6o7pSAvXAQrwstA6pAJjE3xS6HjvLZaS3P25wbNVyBcSJSYSXROZOcd~lAEaDJB3f1ywBgBrpB-VAfG-VDbmGOGRyrY9b6uXwClR2~FRmURxTL7r6ecNWSo-xLYXk23JQeo-Wvcdw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926096&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of sequencing coverage for target sites derived from 365 randomly chosen <em>C. farreri</em> scaffolds (concatenated). For parents, sequencing data were generated from standard BsaXI libraries, whereas for progenies, RR libraries were sequenced. Target sites of the RR libraries could be reproducibly sequenced among progenies and their sequencing coverage was ∼6-fold higher than for standard libraries.</p></div></div><div content-id="DST043F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04302.png?Expires=1509238925&Signature=SUREWl-mZbsMVce1nIPPZMvAYrficknfhEfarquepsUmPMBcc8w0pYa~xpyNKlnKHVNyjclbevEsDhGbFA5GyjU2w0EhOwUt8BoEY2i1znqb-E~q2H9QadXgHGV0a6XqKOfpxgE2ldbUHo4uRxlMaXHYkSHyDKQbITGJnrfAuGFdfIgfPE0iNKJplYc9zx~PvX8ANubfzdCq1477qX~tE2nyxava9SMxv05d3bENGo0mNYGCQPLLmteXIzEQ~xFieQ~WfnI7aWJ~kmZ6cfKF7WLVvII4fdXguX6j~wyRuPh9D~lLqr4e7B8Dh4MjiwiAgE-OxxPhaHXMPHKUXwz78g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of sequencing coverage for target sites derived from 365 randomly chosen C. farreri scaffolds (concatenated). For parents, sequencing data were generated from standard BsaXI libraries, whereas for progenies, RR libraries were sequenced. Target sites of the RR libraries could be reproducibly sequenced among progenies and their sequencing coverage was ∼6-fold higher than for standard libraries." path-from-xml="dst04302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926096/dst04302.tiff" path-from-xml="dst04302.tiff" target="_blank">View large</a><a section="4926096" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04302.png?Expires=1509238925&Signature=RM-09x~ucu1mnNA0okV7j5H8-X~08d0cMZ7vdx0S5jY3paySOYYQ3k~0fgVbsENzW-8sx5Pcn4QQXK6DjQAneGygy5Nv5MKbYKzj3FmRfVdOsxF6gWdupMM-A6Low1P0vX-b60J7-TacyRE0ObJeknyHt00IgkXZTo10tdTqHrbGgsGIYzubM5yovwZEr-VdDi5GxY1TP2Q1o6o7pSAvXAQrwstA6pAJjE3xS6HjvLZaS3P25wbNVyBcSJSYSXROZOcd~lAEaDJB3f1ywBgBrpB-VAfG-VDbmGOGRyrY9b6uXwClR2~FRmURxTL7r6ecNWSo-xLYXk23JQeo-Wvcdw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926096&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of sequencing coverage for target sites derived from 365 randomly chosen <em>C. farreri</em> scaffolds (concatenated). For parents, sequencing data were generated from standard BsaXI libraries, whereas for progenies, RR libraries were sequenced. Target sites of the RR libraries could be reproducibly sequenced among progenies and their sequencing coverage was ∼6-fold higher than for standard libraries.</p></div></div>                            <h3 scrollto-destination=4926097 id="4926097" class="section-title" data-legacy-id=s3d>3.4. High-resolution linkage mapping</h3>
<p>In total, 7458 polymorphic markers were identified, of which 6842 were heterozygous in at least one parent, including 2196 co-dominant and 4646 dominant markers. Dominant markers showing 3 : 1 segregation pattern in progenies were not included in subsequent analysis due to their inability to construct sex-specific maps. Totally, 4881 markers that conformed to the expected Mendelian ratios (<em>P</em> ≥ 0.05) were included in the linkage analysis. These markers were partitioned into maternal (1 : 1 female type; 2473 markers) and paternal (1 : 1 male type; 2175 markers) datasets for constructing sex-specific linkage maps. At the LOD threshold of 6.0, the markers were grouped into 19 linkage groups, corresponding to the haploid chromosome number of <em>C. farreri</em>.<sup><a reveal-id="DST043C32" data-open="DST043C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The female map contained 2025 markers and spanned 1175.4 cM with an average marker interval of 0.59 cM, whereas the male map contained 1861 markers and spanned 1154.9 cM with an average marker interval of 0.62 cM. Significant differences in recombination rates between the female and male maps were observed for linkage groups LG5, LG8, LG10 and LG15 (Table <a reveal-id="DST043TB3" data-open="DST043TB3" class="link link-reveal link-table xref-fig">3</a>). LG5 and LG10 showed higher recombination rates in females than in males, whereas the contrary was found for LG8 and LG15. </p>                        <a id="4926099" scrollto-destination="4926099"></a>
<div class="table-wrap"><div id="DST043TB3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of the consensus linkage map in <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Number of markers </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>Female/male recombination rate </th><th>Number of annotated markers </th></tr></thead><tbody><tr><td>1 </td><td>274 </td><td>96.09 </td><td>0.35 </td><td>0.97 </td><td>41 </td></tr><tr><td>2 </td><td>272 </td><td>88.62 </td><td>0.33 </td><td>0.81 </td><td>43 </td></tr><tr><td>3 </td><td>271 </td><td>95.36 </td><td>0.35 </td><td>1.04 </td><td>37 </td></tr><tr><td>4 </td><td>241 </td><td>75.95 </td><td>0.32 </td><td>1.00 </td><td>39 </td></tr><tr><td>5 </td><td>235 </td><td>71.48 </td><td>0.31 </td><td>1.37 </td><td>44 </td></tr><tr><td>6 </td><td>230 </td><td>99.62 </td><td>0.44 </td><td>1.24 </td><td>53 </td></tr><tr><td>7 </td><td>223 </td><td>77.77 </td><td>0.35 </td><td>1.00 </td><td>42 </td></tr><tr><td>8 </td><td>207 </td><td>68.57 </td><td>0.33 </td><td>0.70 </td><td>35 </td></tr><tr><td>9 </td><td>203 </td><td>91.94 </td><td>0.46 </td><td>1.30 </td><td>31 </td></tr><tr><td>10 </td><td>199 </td><td>54.99 </td><td>0.28 </td><td>2.25 </td><td>30 </td></tr><tr><td>11 </td><td>189 </td><td>96.59 </td><td>0.51 </td><td>0.97 </td><td>28 </td></tr><tr><td>12 </td><td>183 </td><td>92.82 </td><td>0.51 </td><td>0.87 </td><td>26 </td></tr><tr><td>13 </td><td>172 </td><td>81.04 </td><td>0.47 </td><td>1.14 </td><td>32 </td></tr><tr><td>14 </td><td>167 </td><td>65.84 </td><td>0.40 </td><td>1.34 </td><td>18 </td></tr><tr><td>15 </td><td>166 </td><td>78.2 </td><td>0.47 </td><td>0.62 </td><td>33 </td></tr><tr><td>16 </td><td>162 </td><td>84.01 </td><td>0.52 </td><td>0.94 </td><td>27 </td></tr><tr><td>17 </td><td>147 </td><td>81 </td><td>0.55 </td><td>0.82 </td><td>23 </td></tr><tr><td>18 </td><td>142 </td><td>64.36 </td><td>0.46 </td><td>1.07 </td><td>21 </td></tr><tr><td>19 </td><td>123 </td><td>79.11 </td><td>0.65 </td><td>1.00 </td><td>20 </td></tr><tr><td>All </td><td>3,806 </td><td>1543.36 </td><td>0.41 </td><td>1.01 </td><td>623 </td></tr></tbody></table></div><div content-id="DST043TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Number of markers </th><th>Length (cM) </th><th>Average marker interval (cM) </th><th>Female/male recombination rate </th><th>Number of annotated markers </th></tr></thead><tbody><tr><td>1 </td><td>274 </td><td>96.09 </td><td>0.35 </td><td>0.97 </td><td>41 </td></tr><tr><td>2 </td><td>272 </td><td>88.62 </td><td>0.33 </td><td>0.81 </td><td>43 </td></tr><tr><td>3 </td><td>271 </td><td>95.36 </td><td>0.35 </td><td>1.04 </td><td>37 </td></tr><tr><td>4 </td><td>241 </td><td>75.95 </td><td>0.32 </td><td>1.00 </td><td>39 </td></tr><tr><td>5 </td><td>235 </td><td>71.48 </td><td>0.31 </td><td>1.37 </td><td>44 </td></tr><tr><td>6 </td><td>230 </td><td>99.62 </td><td>0.44 </td><td>1.24 </td><td>53 </td></tr><tr><td>7 </td><td>223 </td><td>77.77 </td><td>0.35 </td><td>1.00 </td><td>42 </td></tr><tr><td>8 </td><td>207 </td><td>68.57 </td><td>0.33 </td><td>0.70 </td><td>35 </td></tr><tr><td>9 </td><td>203 </td><td>91.94 </td><td>0.46 </td><td>1.30 </td><td>31 </td></tr><tr><td>10 </td><td>199 </td><td>54.99 </td><td>0.28 </td><td>2.25 </td><td>30 </td></tr><tr><td>11 </td><td>189 </td><td>96.59 </td><td>0.51 </td><td>0.97 </td><td>28 </td></tr><tr><td>12 </td><td>183 </td><td>92.82 </td><td>0.51 </td><td>0.87 </td><td>26 </td></tr><tr><td>13 </td><td>172 </td><td>81.04 </td><td>0.47 </td><td>1.14 </td><td>32 </td></tr><tr><td>14 </td><td>167 </td><td>65.84 </td><td>0.40 </td><td>1.34 </td><td>18 </td></tr><tr><td>15 </td><td>166 </td><td>78.2 </td><td>0.47 </td><td>0.62 </td><td>33 </td></tr><tr><td>16 </td><td>162 </td><td>84.01 </td><td>0.52 </td><td>0.94 </td><td>27 </td></tr><tr><td>17 </td><td>147 </td><td>81 </td><td>0.55 </td><td>0.82 </td><td>23 </td></tr><tr><td>18 </td><td>142 </td><td>64.36 </td><td>0.46 </td><td>1.07 </td><td>21 </td></tr><tr><td>19 </td><td>123 </td><td>79.11 </td><td>0.65 </td><td>1.00 </td><td>20 </td></tr><tr><td>All </td><td>3,806 </td><td>1543.36 </td><td>0.41 </td><td>1.01 </td><td>623 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4926099" target="_blank">View Large</a></div></div><p>The sex-specific maps were further integrated using anchor markers that were heterozygous in both parents. The consensus map contained 3806 markers (Fig. <a reveal-id="DST043F3" data-open="DST043F3" class="link link-reveal link-table xref-fig">3</a>, Table <a reveal-id="DST043TB3" data-open="DST043TB3" class="link link-reveal link-table xref-fig">3</a>) and spanned 1543.4 cM with an average marker interval of 0.41 cM. The length of each linkage group ranged from 54.9 to 99.6 cM, and the marker density varied from 123 to 274 across the linkage groups. Mapping all the markers to the annotated genomic scaffolds revealed 623 markers (18–53 per linkage group; Table <a reveal-id="DST043TB3" data-open="DST043TB3" class="link link-reveal link-table xref-fig">3</a>) that resided in or close to genes. These markers represent a valuable resource for future comparative genome analysis. </p>                        <a id="4926101" scrollto-destination="4926101"></a>

  <div id="DST043F3" data-id="DST043F3" class="fig fig-section">
    <div class="label fig-label">Figure 3.</div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303A.png?Expires=1509238925&amp;Signature=CjX6KMQWbvgbgaghzaWZ8PQs3f~7m7XMLM61dPWapThxBM5F0ldZ~NVnqMgv0mgNG9Cm8hUA06VVr5HHcPWSKFzN5s7KcWba5M28LxgK6AQQcB3pTPHF-EIGgC966DGLVjTUs-x-K3zAhEFLwBiSPM9t5DTW1cXvHRCuV9dly5cdiPd0DCY4-JrAXdjPzV~~-9Y399ZEeAGGCUMx2ly0s3VN4FW1WHlUoYw7F7KD1XzcSGNhhOI4voA0uLHer~UASUNkkXcI-RYnK0cgqJ1GinjrpVKcJ3XDUVB03BZRcNQ5dEwLd8FYxc6zeK15C0yQu-uhmaLfff13buisY53-bQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303A.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303A.tiff" path-from-xml="dst04303A.tiff" target="_blank">View large</a>
        <a section="4926101" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303A.png?Expires=1509238925&amp;Signature=GtN4k53sdS~I11hmBUpwfljLnH7h1JP9-KOHRXLT8kv5KggMvC29PlMrh5ps089osHB2wDeYp9iWNyMY3J8q3sJQzqlg8wcIZL6jqB9L1-qjpgSaEfoAE8Cf168D4-EdpcGdinGVr2kHqHQvwt9V00pYqfpAKTsdvbsls1fA50gShY4bHZQyzHvGdUFbIEX~-OwsL8CMlKNVhYtxWwDXYRqpyz9kSApJtKfvp1WqqYlNfA7tv5nW4GDolxrf4wxtvMh0XVFS6X6uF59Y1DN1CBN3BVdhznqVr1QwYfa4e5tvjuMKBkQoXAcPbwl~vHd1uuiHVtaQfxgjsYBuY7mzXg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303A.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303B.png?Expires=1509238925&amp;Signature=YOpxz1KeNzqJn0J6JXjyKp9eHE3E3Un1aaFd4Ieh-x2QFqAQ4kmwmkef5ty5GpbJv8Yaq~8LF~669yZxNaPJMd5ijOVA0hejmj4KNPMmwwDF1iYbhmlcqg2Bh4iYdv7zbKKtv33xZm4P0N1~qHBSKKnqJ5dzLSJI3Zbhbc60h-RJSx2J01ZOEXD~6kn~e12MYgeGPoW5M6DYaihH0c4jATAyQSga~2qQQ21G9vKpK5haXdjGZGg7-RmmKuUaLpYu7ARoaFlXpealnlh2YM9i7NfzjMWOhSlBurK0yS2UzT1l2RYuBnpk0dGO4SyF2JSFqU2J7xbiYOboD4ARDMBJ6w__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303B.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303B.tiff" path-from-xml="dst04303B.tiff" target="_blank">View large</a>
        <a section="4926101" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303B.png?Expires=1509238925&amp;Signature=DdZXj6UDtOrYPosBEWBAUIRF6zxtsS2yOCOi6a9y798LXbC~ogDeUWc02k7N6Hd-AzdOXrCsNOHPSh3jaBJZhVrIYC1syEJIVhXkA2C9JY2d19sYiOQxDPjlXhMAT2EV5SClXxLZeX2DWqOGIgvLtlAhw77EPi2sciiJPectIXWC4DdFImtfGb~cnYFx8dZSwAxYqPsxMnnoZnsohqy44Dr3cOJp67ULUvKOL-4oLR-N4gM5tlyc48bxdCSaCuOy4ETQSAIrV-XSeUKj1EfVRXdREZSizu3Azsy3vgT1I0cW2FQDNfcZ67cgXtb5rCY24RpSGPgQQqymq8PLmfRydg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303B.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303C.png?Expires=1509238925&amp;Signature=K~Wf-NR0vS2wYC4Z~3KqysrIannHLCjUDXaiUqisHjIyORaw-0iWeB5djV-a5hpX1mtBPZFr0c0o2SYFKxwJvIBlwfmia9wiEVkYnjcTb0y9hRW0zPEGI6uaXr7uOLA1cQfB8Bxwp3ewIfyedyuGWA4lt~~exud6jPdmpWQ9n6qGdqWUZUGXZLZplv-r5BywvxWSbhTUWrL1KJp0FEp05vDO0ZdluPSNW-BR3FWOI~VbapBCwnYNJGu~55tgEXmUqp234g87el46KuJ4Ff5KdH~ExfSSHWO1fpnDk18pJ2jqR4w7wUv5QH50wRQjWzcONS1fALq~Mel3X9AmX-oTIA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303C.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303C.tiff" path-from-xml="dst04303C.tiff" target="_blank">View large</a>
        <a section="4926101" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303C.png?Expires=1509238925&amp;Signature=ENNvvJjiSQ2QvsDWYHk4gXFsCKkzaieqr4QqoQiyvOktk89M79GTdG6W~s3emszspzq~WBBSiC6lSvYzUbNJ5-EFWCLKrqqVvQuY1aw2fpoW4J4faO7TJQm48iJabM5OpD80hPC3eRzoDTVynAO817LsbbFgRSCSTxj6RtPTvwXtWyNGDCBVZqT4PmXnKH~76CXDuHFJHG4bJF11NqnSSiVi86kipX5XLeQEKh~4hguFRMaNwdhpU~mXdwOTA2STmtxy0s5A7YwjIUr8fkfRaYU5xP0nSt70YW0ibRj36ilun8bf4fxRi4Ni-z91UgGrdJAYAmOIPY28NCngN-PI2g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303C.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303D.png?Expires=1509238925&amp;Signature=aodHQLOCL6zpS3nw7T4YD6REAF6xxsXzbS3RPRIeKEFdKZo6KeGF9-67gZbV73QGRQEg0wXrHhcZX7UGIltyK7nqQntJkLJVlsmeqJCLrzDz4DuctLVGQSOlMTa8uX19BWBYEdipfCPjZnUDEuBgWNSEHceyKeHpzxlFCxocqyrmJwwzE35GSholzWuMsEtXXLpaBFplWufCB6bwwuZflM5Vgq6aQLfrxWu-Gff0tAGoT7Go5qE-yuJ2IJrft9xLQJkmf3ASWusqMsLgl9XANCdxVnCt9VCYE8tA4LiTdt9Tq5alkq15xCdJ8Kf1u27hf~Va-gdUQHzq0J7YGQV-yw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303D.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303D.tiff" path-from-xml="dst04303D.tiff" target="_blank">View large</a>
        <a section="4926101" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303D.png?Expires=1509238925&amp;Signature=K8toWHNGx~Y7TorgyPxpYwZF1eUWrQpn8gBuIbnFIbbtAGbEBKrdP9k83kAwdRQe8NtRe4SqM9v8kIYaeM8GR8Q2JMKHEB55cfSNk84eEPSj8B8cIMv~80eZxHjsoZHU9Xvf61kDLfWEqEpJGcIlCtcxP1HhDsyN0ommsaG2JvHv4FsNER6X7UBdegytUuZVysBQR8o15muTC2u7x~kmWJ0UiuRW~jVQ3nZT1xrnKN~DiyFySJLaFgqCw0U7ByHWuXPZ0pdQCEV2LshleMCS~DPtjbCe~p5N-kmG7ZSJUMpicqmONjA8HC6aQ7hZnfaWCReS32ZSyhDHEl1Bk08qfA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303D.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="caption fig-caption">
      <p>The high-density consensus linkage map of <em>C. farreri</em>, which contained 3806 markers in 19 linkage groups.</p>
    </div>
  </div>
  <div content-id="DST043F3" class="fig fig-modal reveal-modal">
    <div class="label fig-label">Figure 3.</div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303A.png?Expires=1509238925&amp;Signature=CjX6KMQWbvgbgaghzaWZ8PQs3f~7m7XMLM61dPWapThxBM5F0ldZ~NVnqMgv0mgNG9Cm8hUA06VVr5HHcPWSKFzN5s7KcWba5M28LxgK6AQQcB3pTPHF-EIGgC966DGLVjTUs-x-K3zAhEFLwBiSPM9t5DTW1cXvHRCuV9dly5cdiPd0DCY4-JrAXdjPzV~~-9Y399ZEeAGGCUMx2ly0s3VN4FW1WHlUoYw7F7KD1XzcSGNhhOI4voA0uLHer~UASUNkkXcI-RYnK0cgqJ1GinjrpVKcJ3XDUVB03BZRcNQ5dEwLd8FYxc6zeK15C0yQu-uhmaLfff13buisY53-bQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303A.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303A.tiff" path-from-xml="dst04303A.tiff" target="_blank">View large</a>
        <a section="4926101" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303A.png?Expires=1509238925&amp;Signature=GtN4k53sdS~I11hmBUpwfljLnH7h1JP9-KOHRXLT8kv5KggMvC29PlMrh5ps089osHB2wDeYp9iWNyMY3J8q3sJQzqlg8wcIZL6jqB9L1-qjpgSaEfoAE8Cf168D4-EdpcGdinGVr2kHqHQvwt9V00pYqfpAKTsdvbsls1fA50gShY4bHZQyzHvGdUFbIEX~-OwsL8CMlKNVhYtxWwDXYRqpyz9kSApJtKfvp1WqqYlNfA7tv5nW4GDolxrf4wxtvMh0XVFS6X6uF59Y1DN1CBN3BVdhznqVr1QwYfa4e5tvjuMKBkQoXAcPbwl~vHd1uuiHVtaQfxgjsYBuY7mzXg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303A.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303B.png?Expires=1509238925&amp;Signature=YOpxz1KeNzqJn0J6JXjyKp9eHE3E3Un1aaFd4Ieh-x2QFqAQ4kmwmkef5ty5GpbJv8Yaq~8LF~669yZxNaPJMd5ijOVA0hejmj4KNPMmwwDF1iYbhmlcqg2Bh4iYdv7zbKKtv33xZm4P0N1~qHBSKKnqJ5dzLSJI3Zbhbc60h-RJSx2J01ZOEXD~6kn~e12MYgeGPoW5M6DYaihH0c4jATAyQSga~2qQQ21G9vKpK5haXdjGZGg7-RmmKuUaLpYu7ARoaFlXpealnlh2YM9i7NfzjMWOhSlBurK0yS2UzT1l2RYuBnpk0dGO4SyF2JSFqU2J7xbiYOboD4ARDMBJ6w__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303B.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303B.tiff" path-from-xml="dst04303B.tiff" target="_blank">View large</a>
        <a section="4926101" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303B.png?Expires=1509238925&amp;Signature=DdZXj6UDtOrYPosBEWBAUIRF6zxtsS2yOCOi6a9y798LXbC~ogDeUWc02k7N6Hd-AzdOXrCsNOHPSh3jaBJZhVrIYC1syEJIVhXkA2C9JY2d19sYiOQxDPjlXhMAT2EV5SClXxLZeX2DWqOGIgvLtlAhw77EPi2sciiJPectIXWC4DdFImtfGb~cnYFx8dZSwAxYqPsxMnnoZnsohqy44Dr3cOJp67ULUvKOL-4oLR-N4gM5tlyc48bxdCSaCuOy4ETQSAIrV-XSeUKj1EfVRXdREZSizu3Azsy3vgT1I0cW2FQDNfcZ67cgXtb5rCY24RpSGPgQQqymq8PLmfRydg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303B.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303C.png?Expires=1509238925&amp;Signature=K~Wf-NR0vS2wYC4Z~3KqysrIannHLCjUDXaiUqisHjIyORaw-0iWeB5djV-a5hpX1mtBPZFr0c0o2SYFKxwJvIBlwfmia9wiEVkYnjcTb0y9hRW0zPEGI6uaXr7uOLA1cQfB8Bxwp3ewIfyedyuGWA4lt~~exud6jPdmpWQ9n6qGdqWUZUGXZLZplv-r5BywvxWSbhTUWrL1KJp0FEp05vDO0ZdluPSNW-BR3FWOI~VbapBCwnYNJGu~55tgEXmUqp234g87el46KuJ4Ff5KdH~ExfSSHWO1fpnDk18pJ2jqR4w7wUv5QH50wRQjWzcONS1fALq~Mel3X9AmX-oTIA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303C.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303C.tiff" path-from-xml="dst04303C.tiff" target="_blank">View large</a>
        <a section="4926101" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303C.png?Expires=1509238925&amp;Signature=ENNvvJjiSQ2QvsDWYHk4gXFsCKkzaieqr4QqoQiyvOktk89M79GTdG6W~s3emszspzq~WBBSiC6lSvYzUbNJ5-EFWCLKrqqVvQuY1aw2fpoW4J4faO7TJQm48iJabM5OpD80hPC3eRzoDTVynAO817LsbbFgRSCSTxj6RtPTvwXtWyNGDCBVZqT4PmXnKH~76CXDuHFJHG4bJF11NqnSSiVi86kipX5XLeQEKh~4hguFRMaNwdhpU~mXdwOTA2STmtxy0s5A7YwjIUr8fkfRaYU5xP0nSt70YW0ibRj36ilun8bf4fxRi4Ni-z91UgGrdJAYAmOIPY28NCngN-PI2g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303C.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="graphic-wrap">
      <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04303D.png?Expires=1509238925&amp;Signature=aodHQLOCL6zpS3nw7T4YD6REAF6xxsXzbS3RPRIeKEFdKZo6KeGF9-67gZbV73QGRQEg0wXrHhcZX7UGIltyK7nqQntJkLJVlsmeqJCLrzDz4DuctLVGQSOlMTa8uX19BWBYEdipfCPjZnUDEuBgWNSEHceyKeHpzxlFCxocqyrmJwwzE35GSholzWuMsEtXXLpaBFplWufCB6bwwuZflM5Vgq6aQLfrxWu-Gff0tAGoT7Go5qE-yuJ2IJrft9xLQJkmf3ASWusqMsLgl9XANCdxVnCt9VCYE8tA4LiTdt9Tq5alkq15xCdJ8Kf1u27hf~Va-gdUQHzq0J7YGQV-yw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The high-density consensus linkage map of C. farreri, which contained 3806 markers in 19 linkage groups." path-from-xml="dst04303D.tiff" />
      <div class="fig-orig original-slide">
        <a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926101/dst04303D.tiff" path-from-xml="dst04303D.tiff" target="_blank">View large</a>
        <a section="4926101" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04303D.png?Expires=1509238925&amp;Signature=K8toWHNGx~Y7TorgyPxpYwZF1eUWrQpn8gBuIbnFIbbtAGbEBKrdP9k83kAwdRQe8NtRe4SqM9v8kIYaeM8GR8Q2JMKHEB55cfSNk84eEPSj8B8cIMv~80eZxHjsoZHU9Xvf61kDLfWEqEpJGcIlCtcxP1HhDsyN0ommsaG2JvHv4FsNER6X7UBdegytUuZVysBQR8o15muTC2u7x~kmWJ0UiuRW~jVQ3nZT1xrnKN~DiyFySJLaFgqCw0U7ByHWuXPZ0pdQCEV2LshleMCS~DPtjbCe~p5N-kmG7ZSJUMpicqmONjA8HC6aQ7hZnfaWCReS32ZSyhDHEl1Bk08qfA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=4926101&amp;ar=347657&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst04303D.tiff" target="_blank">Download slide</a>
      </div>
    </div>
    <div class="caption fig-caption">
      <p>The high-density consensus linkage map of <em>C. farreri</em>, which contained 3806 markers in 19 linkage groups.</p>
    </div>
  </div>
<p>Genome length was estimated to be 1544.2 cM (<em>G</em><sub>e1</sub>) and 1559.5 cM (<em>G</em><sub>e2</sub>) using two different methods.<sup><a reveal-id="DST043C33" data-open="DST043C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DST043C34" data-open="DST043C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The average of the two estimates (1551.9 cM) was used as the expected genome length. Genome coverage of the consensus map was nearly complete and reached 99.5%. Given the estimated genome size of ∼1.2 Gb for <em>C. farreri</em>,<sup><a reveal-id="DST043C20" data-open="DST043C20" class="link link-ref link-reveal xref-bibr">20</a></sup> the average recombination rate across all the linkage groups was ∼1.3 cM/Mb.</p>                            <h3 scrollto-destination=4926103 id="4926103" class="section-title" data-legacy-id=s3e>3.5. QTL mapping and association analyses of growth traits and sex</h3>
<p>Pairwise comparisons among five growth traits using Pearson's correlation revealed that all of them were highly correlated with correlation coefficients ranging from 0.77 to 0.96 (Table <a reveal-id="DST043TB4" data-open="DST043TB4" class="link link-reveal link-table xref-fig">4</a>). The strongest correlation was observed between shell length and shell height (<em>r</em> = 0.96, <em>P</em> = 1<em>E</em>−10). A genome scan for these growth traits showed that they exhibited quite similar LOD profiles (Fig. <a reveal-id="DST043F4" data-open="DST043F4" class="link link-reveal link-table xref-fig">4</a>), indicating that these traits were possibly regulated by the same set of genes. Therefore, a principle component analysis was conducted, which revealed that the first principal component (PC1) explained ∼90% of the total phenotypic variation (Fig. <a reveal-id="DST043F5" data-open="DST043F5" class="link link-reveal link-table xref-fig">5</a>). To increase the QTL detection power, PC1 was utilized as a composite trait for QTL mapping. Two significant QTLs were detected on LG1 and LG3 (Fig. <a reveal-id="DST043F6" data-open="DST043F6" class="link link-reveal link-table xref-fig">6</a>a and Table <a reveal-id="DST043TB5" data-open="DST043TB5" class="link link-reveal link-table xref-fig">5</a>) and the one on LG3 that passed the genome-wide significance threshold showed a stronger QTL signal. The two QTLs explained 11.4 and 16.9% of the total phenotypic variation, respectively. Given the high marker density in the linkage map, association analysis was also performed to serve as a complementary approach to evaluate the QTL mapping results. Association analysis revealed a similar distribution pattern across all linkage groups as in QTL mapping analysis (Fig. <a reveal-id="DST043F6" data-open="DST043F6" class="link link-reveal link-table xref-fig">6</a>a) and supported QTL mapping results in general. Within the confidence interval of the QTL detected on LG3, one marker named f68558 was found to be located in the intron region of a transcription factor gene, <em>PROP1</em> (Homeobox protein prophet of Pit-1) (Fig. <a reveal-id="DST043F7" data-open="DST043F7" class="link link-reveal link-table xref-fig">7</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). It has been shown that <em>PROP1</em> can regulate the production of growth hormone,<sup><a reveal-id="DST043C35" data-open="DST043C35" class="link link-ref link-reveal xref-bibr">35</a></sup> a peptide that simulates growth, cell reproduction and regeneration in animals. Identification of <em>PROP1</em> within the QTL region suggests that this gene may represent an important candidate QTL gene that is worthy of further investigation. </p>                        <a id="4926105" scrollto-destination="4926105"></a>
<div class="table-wrap"><div id="DST043TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Pearson's correlation coefficients among five growth traits in <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Shell height </th><th>Shell length </th><th>Shell width </th><th>Body weight </th><th>Muscle weight </th></tr></thead><tbody><tr><td>Shell height </td><td>1.00 </td><td>0.96 </td><td>0.85 </td><td>0.94 </td><td>0.84 </td></tr><tr><td>Shell length </td><td>– </td><td>1.00 </td><td>0.83 </td><td>0.93 </td><td>0.81 </td></tr><tr><td>Shell width </td><td>– </td><td>– </td><td>1.00 </td><td>0.82 </td><td>0.77 </td></tr><tr><td>Body weight </td><td>– </td><td>– </td><td>– </td><td>1.00 </td><td>0.93 </td></tr><tr><td>Muscle weight </td><td>– </td><td>– </td><td>– </td><td>– </td><td>1.00 </td></tr></tbody></table></div><div content-id="DST043TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Shell height </th><th>Shell length </th><th>Shell width </th><th>Body weight </th><th>Muscle weight </th></tr></thead><tbody><tr><td>Shell height </td><td>1.00 </td><td>0.96 </td><td>0.85 </td><td>0.94 </td><td>0.84 </td></tr><tr><td>Shell length </td><td>– </td><td>1.00 </td><td>0.83 </td><td>0.93 </td><td>0.81 </td></tr><tr><td>Shell width </td><td>– </td><td>– </td><td>1.00 </td><td>0.82 </td><td>0.77 </td></tr><tr><td>Body weight </td><td>– </td><td>– </td><td>– </td><td>1.00 </td><td>0.93 </td></tr><tr><td>Muscle weight </td><td>– </td><td>– </td><td>– </td><td>– </td><td>1.00 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4926105" target="_blank">View Large</a></div></div>                        <a id="4926106" scrollto-destination="4926106"></a>
<div class="table-wrap"><div id="DST043TB5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>QTL mapping of the PC1 of five growth traits in <em>C. farreri</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>LG </th><th>Confidence interval (cM) </th><th>Closest marker (position) </th><th>PV (%)<sup>a</sup> </th><th>LOD </th></tr></thead><tbody><tr><td>PC1 </td><td>1 </td><td>55.13–70.68 </td><td>dm28438 (60.34), df12539 (61.10) </td><td>11.38 </td><td>2.52 </td></tr><tr><td> </td><td>3 </td><td>34.14–61.82 </td><td>m7430 (48.70), f17843 (48.72) </td><td>16.89 </td><td>3.76 </td></tr></tbody></table></div><div content-id="DST043TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>LG </th><th>Confidence interval (cM) </th><th>Closest marker (position) </th><th>PV (%)<sup>a</sup> </th><th>LOD </th></tr></thead><tbody><tr><td>PC1 </td><td>1 </td><td>55.13–70.68 </td><td>dm28438 (60.34), df12539 (61.10) </td><td>11.38 </td><td>2.52 </td></tr><tr><td> </td><td>3 </td><td>34.14–61.82 </td><td>m7430 (48.70), f17843 (48.72) </td><td>16.89 </td><td>3.76 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Phenotypic variance (PV) explained by a QTL.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4926106" target="_blank">View Large</a></div></div>                        <a id="4926107" scrollto-destination="4926107"></a>
<div id="DST043F4" data-id="DST043F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04304.png?Expires=1509238925&Signature=M~lnsHXfpiJJWOBCAUhTv6Q8JjbQk016ZNl9N~h~xy9kfrKngWAhnXZC9dgLFPhWLxTnRxDuw6kGgIqbLcfeiPADb4YNn~SaB34b6Z283SdpPgHcrc8qaHhwwYxOTYn~7PrdN51BfaobSSQQV1QdP9u2~m1Ek6bxDQmkKkqsOhZ8C6MpLO6P1a8I-BbKJNRXaUs2dPNrvtkjvxVm9axaq5TkOTzy2bj5AcxWnpRoh1aSq8VSTlhJADc9vWflwItX4iGBvcc5eWIjB58bpPfL8SepHZOp6WouqGTlfBt9pP8NR49SH261tgCju49yY3IOIlPdCJL9JwtLyTsbN-rqkw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="A genome scan of LOD profiles for five growth traits in C. farreri. All traits exhibited quite similar LOD distributions. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight. The dark and light dashed lines indicated the genome-wide and chromosome-wide significance thresholds." path-from-xml="dst04304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926107/dst04304.tiff" path-from-xml="dst04304.tiff" target="_blank">View large</a><a section="4926107" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04304.png?Expires=1509238925&Signature=SkRAhNr0SBgFQXXiZrEnW47YjCxQ2SD53eebzK7VvBbvnHaRBODXmQkWRCW3vSBrPV0RSN8UXpDxMS2ETGeV2OVqRqc7wetO1Q~t-jQeCiPHGiLb2eyDpFontLfkaT3kBtHPOGb~9QANaqyNTgt6PQQ2JWsDY1CjYA4KKr3UUaSo55zxshZMXqjsyXSGM39Ap7d-7tub-PuM9qMyLvM8ZVcSr1RysdOQ75MsagKuDQ2gi4TJmJmPKp~I3Oa4gqOzUA3C6~vtO7K0MTCZmiY4GEhDH-gpREIJOIjsqBjqV3cIOUSdiUigbkXrUBAhZrhK~0NvhbsORikrtauQ5lrODQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926107&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A genome scan of LOD profiles for five growth traits in <em>C. farreri</em>. All traits exhibited quite similar LOD distributions. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight. The dark and light dashed lines indicated the genome-wide and chromosome-wide significance thresholds.</p></div></div><div content-id="DST043F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04304.png?Expires=1509238925&Signature=M~lnsHXfpiJJWOBCAUhTv6Q8JjbQk016ZNl9N~h~xy9kfrKngWAhnXZC9dgLFPhWLxTnRxDuw6kGgIqbLcfeiPADb4YNn~SaB34b6Z283SdpPgHcrc8qaHhwwYxOTYn~7PrdN51BfaobSSQQV1QdP9u2~m1Ek6bxDQmkKkqsOhZ8C6MpLO6P1a8I-BbKJNRXaUs2dPNrvtkjvxVm9axaq5TkOTzy2bj5AcxWnpRoh1aSq8VSTlhJADc9vWflwItX4iGBvcc5eWIjB58bpPfL8SepHZOp6WouqGTlfBt9pP8NR49SH261tgCju49yY3IOIlPdCJL9JwtLyTsbN-rqkw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="A genome scan of LOD profiles for five growth traits in C. farreri. All traits exhibited quite similar LOD distributions. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight. The dark and light dashed lines indicated the genome-wide and chromosome-wide significance thresholds." path-from-xml="dst04304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926107/dst04304.tiff" path-from-xml="dst04304.tiff" target="_blank">View large</a><a section="4926107" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04304.png?Expires=1509238925&Signature=SkRAhNr0SBgFQXXiZrEnW47YjCxQ2SD53eebzK7VvBbvnHaRBODXmQkWRCW3vSBrPV0RSN8UXpDxMS2ETGeV2OVqRqc7wetO1Q~t-jQeCiPHGiLb2eyDpFontLfkaT3kBtHPOGb~9QANaqyNTgt6PQQ2JWsDY1CjYA4KKr3UUaSo55zxshZMXqjsyXSGM39Ap7d-7tub-PuM9qMyLvM8ZVcSr1RysdOQ75MsagKuDQ2gi4TJmJmPKp~I3Oa4gqOzUA3C6~vtO7K0MTCZmiY4GEhDH-gpREIJOIjsqBjqV3cIOUSdiUigbkXrUBAhZrhK~0NvhbsORikrtauQ5lrODQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926107&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A genome scan of LOD profiles for five growth traits in <em>C. farreri</em>. All traits exhibited quite similar LOD distributions. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight. The dark and light dashed lines indicated the genome-wide and chromosome-wide significance thresholds.</p></div></div>                        <a id="4926108" scrollto-destination="4926108"></a>
<div id="DST043F5" data-id="DST043F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04305.png?Expires=1509238925&Signature=IJUFADfKlUWzPT7uU~Fb5qLCx37P9k2WGubol4hxfwoYSuMvB~MgyOZhijML20kdHGxNjUkWGteKknI~omVjGR7W1yLyv6rD-pSBtcGtamz5llfGPDvuJhv6yI6YDM9UBSeaLEymDUzAHHJMXotOQ9gwc7ipmXXh9fYjuZpAiH15uNH1supD2diFUZmrsFmufE-o1zSahmujTron05L~dM~c0YWML7rtBeKyytE2luqhcHDZGdBUHd~ap1SjNN1gmQnnyjztg~OEWDevks6HolIRruO3VSC4g~bjCsZXT5rTSIlAOGQMQ1e60x9TV6LlXueYFbxd8sxIz6btnVO3pQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Principal component analysis of five growth traits in C. farreri. The PC1 explained ∼90% of the total variance. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight." path-from-xml="dst04305.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926108/dst04305.tiff" path-from-xml="dst04305.tiff" target="_blank">View large</a><a section="4926108" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04305.png?Expires=1509238925&Signature=cUTXcsyr-AJ20qddHH6fdqs2yyxMnBtPvuUfGOpt091WI7105CkgwtDiM2DMh2dmFAjAvzPlDnUu~mUByBOtX1BwSSt5g3G~I2lLXvqHdvtma2npAHVPV0ovrYt7IMRJHV3WojmEb37Rxz8M-SDfv-GoN~QdGiui4oe3l85om8iS5AR~NMYzFyzNcuZn0~SUiraq6ATZNlzW0NbrMLu8pi4PRUbIRhx~b05ji~lFRebwKU7gSimu3gJI9yaIxDt3BdMecFMfxtDV1tg6BIACcTVUUoBqhAhVnTiT5obCjlYfCeRKrr~2wFGAd04FuhiTXfdi5w5NOze7mw~yBIfXdQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926108&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04305.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Principal component analysis of five growth traits in <em>C. farreri</em>. The PC1 explained ∼90% of the total variance. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight.</p></div></div><div content-id="DST043F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04305.png?Expires=1509238925&Signature=IJUFADfKlUWzPT7uU~Fb5qLCx37P9k2WGubol4hxfwoYSuMvB~MgyOZhijML20kdHGxNjUkWGteKknI~omVjGR7W1yLyv6rD-pSBtcGtamz5llfGPDvuJhv6yI6YDM9UBSeaLEymDUzAHHJMXotOQ9gwc7ipmXXh9fYjuZpAiH15uNH1supD2diFUZmrsFmufE-o1zSahmujTron05L~dM~c0YWML7rtBeKyytE2luqhcHDZGdBUHd~ap1SjNN1gmQnnyjztg~OEWDevks6HolIRruO3VSC4g~bjCsZXT5rTSIlAOGQMQ1e60x9TV6LlXueYFbxd8sxIz6btnVO3pQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Principal component analysis of five growth traits in C. farreri. The PC1 explained ∼90% of the total variance. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight." path-from-xml="dst04305.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926108/dst04305.tiff" path-from-xml="dst04305.tiff" target="_blank">View large</a><a section="4926108" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04305.png?Expires=1509238925&Signature=cUTXcsyr-AJ20qddHH6fdqs2yyxMnBtPvuUfGOpt091WI7105CkgwtDiM2DMh2dmFAjAvzPlDnUu~mUByBOtX1BwSSt5g3G~I2lLXvqHdvtma2npAHVPV0ovrYt7IMRJHV3WojmEb37Rxz8M-SDfv-GoN~QdGiui4oe3l85om8iS5AR~NMYzFyzNcuZn0~SUiraq6ATZNlzW0NbrMLu8pi4PRUbIRhx~b05ji~lFRebwKU7gSimu3gJI9yaIxDt3BdMecFMfxtDV1tg6BIACcTVUUoBqhAhVnTiT5obCjlYfCeRKrr~2wFGAd04FuhiTXfdi5w5NOze7mw~yBIfXdQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926108&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04305.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Principal component analysis of five growth traits in <em>C. farreri</em>. The PC1 explained ∼90% of the total variance. SH, shell height; SL, shell length; SW, shell width; BW, body weight; AMW, adductor muscle weight.</p></div></div>                        <a id="4926109" scrollto-destination="4926109"></a>
<div id="DST043F6" data-id="DST043F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04306.png?Expires=1509238925&Signature=EN33EULVUkNubmt7njyhrHSyS-GOX4tPcQaIcGcZPl-TqhZ8rhX5yeyCZ62pnjea2Q9MQl5pqneRvcvnntnBD~nTs~tlOSKdIkqOGpmTF5y5WnmqkaJidQnT~4OWFlVfKeJ-Tku-p8cDkL7Chax283WhLq6bNN0n4A6z8WfyDqtNk4W0QXDtsrCxKEMl39Fn204vncQTeZ86GRtr~scnH64qbTPGrhGhdlrpWj5XI3RoDyPx8a2HN0sIBgeDydJtKrCCX-z8lSc8OTLxyOzndTtP1ZIzmCxhHrAr7ZqVxL0fl-VTd6Xbq8NAI6WtOdhC29YlbZBngDeTm47ueOWskA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="QTL mapping and association analysis of growth traits (a) and sex (b) in C. farreri. The PC1 for five growth traits was used in QTL mapping and association analyses. The solid and dashed lines indicate the genome-wide and chromosome-wide significance thresholds, respectively." path-from-xml="dst04306.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926109/dst04306.tiff" path-from-xml="dst04306.tiff" target="_blank">View large</a><a section="4926109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04306.png?Expires=1509238925&Signature=V~d-nh8Ztxq-aNiwaN7KsNYayi9VpVlTcKHH9nXBn~8J2FlyI2rz4i7kN6YOHtQL-oGImMsPzgl5YfnISc9q6Y0ZGyd7U76DRkPjC-1TayyQXBe4xib9gWqi~uCAIt84PxS03pcJm-sj6vH0ovlkj3lYReN2Jt5IsTrwyKSRYOn1DMyjvdENewDMXZUfv8dYrc09k4hqYJH09qfUNr0F~oFQ3UkBE4H8pSyCNoe6OUiA9wL4CrM4eELs4xF-Dqb~sns0Tg2Uz3hkDAosvtWYYDOefy5y335AuWIM~wjGCQJdbbP4eCCRqgaewLNwDsxyx-4gV3qImadVeUTtkyuE6A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926109&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04306.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>QTL mapping and association analysis of growth traits (a) and sex (b) in <em>C. farreri</em>. The PC1 for five growth traits was used in QTL mapping and association analyses. The solid and dashed lines indicate the genome-wide and chromosome-wide significance thresholds, respectively.</p></div></div><div content-id="DST043F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04306.png?Expires=1509238925&Signature=EN33EULVUkNubmt7njyhrHSyS-GOX4tPcQaIcGcZPl-TqhZ8rhX5yeyCZ62pnjea2Q9MQl5pqneRvcvnntnBD~nTs~tlOSKdIkqOGpmTF5y5WnmqkaJidQnT~4OWFlVfKeJ-Tku-p8cDkL7Chax283WhLq6bNN0n4A6z8WfyDqtNk4W0QXDtsrCxKEMl39Fn204vncQTeZ86GRtr~scnH64qbTPGrhGhdlrpWj5XI3RoDyPx8a2HN0sIBgeDydJtKrCCX-z8lSc8OTLxyOzndTtP1ZIzmCxhHrAr7ZqVxL0fl-VTd6Xbq8NAI6WtOdhC29YlbZBngDeTm47ueOWskA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="QTL mapping and association analysis of growth traits (a) and sex (b) in C. farreri. The PC1 for five growth traits was used in QTL mapping and association analyses. The solid and dashed lines indicate the genome-wide and chromosome-wide significance thresholds, respectively." path-from-xml="dst04306.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926109/dst04306.tiff" path-from-xml="dst04306.tiff" target="_blank">View large</a><a section="4926109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04306.png?Expires=1509238925&Signature=V~d-nh8Ztxq-aNiwaN7KsNYayi9VpVlTcKHH9nXBn~8J2FlyI2rz4i7kN6YOHtQL-oGImMsPzgl5YfnISc9q6Y0ZGyd7U76DRkPjC-1TayyQXBe4xib9gWqi~uCAIt84PxS03pcJm-sj6vH0ovlkj3lYReN2Jt5IsTrwyKSRYOn1DMyjvdENewDMXZUfv8dYrc09k4hqYJH09qfUNr0F~oFQ3UkBE4H8pSyCNoe6OUiA9wL4CrM4eELs4xF-Dqb~sns0Tg2Uz3hkDAosvtWYYDOefy5y335AuWIM~wjGCQJdbbP4eCCRqgaewLNwDsxyx-4gV3qImadVeUTtkyuE6A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926109&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04306.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>QTL mapping and association analysis of growth traits (a) and sex (b) in <em>C. farreri</em>. The PC1 for five growth traits was used in QTL mapping and association analyses. The solid and dashed lines indicate the genome-wide and chromosome-wide significance thresholds, respectively.</p></div></div>                        <a id="4926110" scrollto-destination="4926110"></a>
<div id="DST043F7" data-id="DST043F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04307.png?Expires=1509238925&Signature=YXQ3aBcSKLlgCpHyWfk8Aw22IVTkmoMLiz0KGDYRk6jU9sMyep6SYa5VCQuCax2bhvJlSUmCy~NgEr6N73QLZsWJ0XYni-fv9VB5i~HS0zqwAl4DwL9AuEY96bbyyC7lNZ8wmKiOXHgtvc41dZrXzxm5wQNUppCs5xlmBjT9gdkEbMZ7zn-6hMRA1ZqlGWyI3Tx89E3hTurPqoHJE4ox6aE-X2EmYJjvek~BS~wrt1dQs3DfvVtxx9IM9xVSqYzCvRy722bq9ZYxjSxTo7sam2E0u~52LAbsE3gPErwn~MAh3QNbFgNvN6xLMWEpV28iQe6ikwY-nKMPPjfdYo1~eg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The gene structures of PROP1 (Homeobox protein prophet of Pit-1) in two scallop species. For P. yessoensis, the full length of PROP1 was obtained from an ongoing genome sequencing project. For C. farreri, homologous genomic scaffolds were shown. The marker f68558 found in the confidence interval of a growth-related QTL detected on LG3 was located at the intron region of this transcription factor gene. The marker sequences were shown for the two species with the polymorphic locus in C. farreri indicated." path-from-xml="dst04307.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926110/dst04307.tiff" path-from-xml="dst04307.tiff" target="_blank">View large</a><a section="4926110" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04307.png?Expires=1509238925&Signature=NJ783J3HGoyRdI-vQ~fhARNbAls5CmsMCZcjS1ATMVDxyVxKfOF2Cqgdls~aiCbbOgeHiAEkYQPp4NDXLSHjEzDWLkVo3Q9Hyrx2eYQ50ThZBBz95VKHIyLmA6Zw3eWhapSU33TykPjGm2HONk19b5YrrwahhxmWP-oiC42pCuoRSXDMj9FxSzY50d5g64asvWD8Tc2j~Hdf33~6IbPu1EB6sJSALuCAtdXf9zwWLZnRZ3aw44q4bGsfYRYiL0DHmURfxwNiP3FOZSnKk1XD8B~L2vPv9ossKRcqCV6KX-U5DRe-fY~EWtn1ipWYoQBibdngMK24Wv~4NCG5smNjiA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926110&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04307.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The gene structures of <em>PROP1</em> (Homeobox protein prophet of Pit-1) in two scallop species. For <em>P. yessoensis</em>, the full length of <em>PROP1</em> was obtained from an ongoing genome sequencing project<em>.</em> For <em>C. farreri</em>, homologous genomic scaffolds were shown<em>.</em> The marker f68558 found in the confidence interval of a growth-related QTL detected on LG3 was located at the intron region of this transcription factor gene. The marker sequences were shown for the two species with the polymorphic locus in <em>C. farreri</em> indicated.</p></div></div><div content-id="DST043F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04307.png?Expires=1509238925&Signature=YXQ3aBcSKLlgCpHyWfk8Aw22IVTkmoMLiz0KGDYRk6jU9sMyep6SYa5VCQuCax2bhvJlSUmCy~NgEr6N73QLZsWJ0XYni-fv9VB5i~HS0zqwAl4DwL9AuEY96bbyyC7lNZ8wmKiOXHgtvc41dZrXzxm5wQNUppCs5xlmBjT9gdkEbMZ7zn-6hMRA1ZqlGWyI3Tx89E3hTurPqoHJE4ox6aE-X2EmYJjvek~BS~wrt1dQs3DfvVtxx9IM9xVSqYzCvRy722bq9ZYxjSxTo7sam2E0u~52LAbsE3gPErwn~MAh3QNbFgNvN6xLMWEpV28iQe6ikwY-nKMPPjfdYo1~eg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The gene structures of PROP1 (Homeobox protein prophet of Pit-1) in two scallop species. For P. yessoensis, the full length of PROP1 was obtained from an ongoing genome sequencing project. For C. farreri, homologous genomic scaffolds were shown. The marker f68558 found in the confidence interval of a growth-related QTL detected on LG3 was located at the intron region of this transcription factor gene. The marker sequences were shown for the two species with the polymorphic locus in C. farreri indicated." path-from-xml="dst04307.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926110/dst04307.tiff" path-from-xml="dst04307.tiff" target="_blank">View large</a><a section="4926110" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04307.png?Expires=1509238925&Signature=NJ783J3HGoyRdI-vQ~fhARNbAls5CmsMCZcjS1ATMVDxyVxKfOF2Cqgdls~aiCbbOgeHiAEkYQPp4NDXLSHjEzDWLkVo3Q9Hyrx2eYQ50ThZBBz95VKHIyLmA6Zw3eWhapSU33TykPjGm2HONk19b5YrrwahhxmWP-oiC42pCuoRSXDMj9FxSzY50d5g64asvWD8Tc2j~Hdf33~6IbPu1EB6sJSALuCAtdXf9zwWLZnRZ3aw44q4bGsfYRYiL0DHmURfxwNiP3FOZSnKk1XD8B~L2vPv9ossKRcqCV6KX-U5DRe-fY~EWtn1ipWYoQBibdngMK24Wv~4NCG5smNjiA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926110&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04307.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The gene structures of <em>PROP1</em> (Homeobox protein prophet of Pit-1) in two scallop species. For <em>P. yessoensis</em>, the full length of <em>PROP1</em> was obtained from an ongoing genome sequencing project<em>.</em> For <em>C. farreri</em>, homologous genomic scaffolds were shown<em>.</em> The marker f68558 found in the confidence interval of a growth-related QTL detected on LG3 was located at the intron region of this transcription factor gene. The marker sequences were shown for the two species with the polymorphic locus in <em>C. farreri</em> indicated.</p></div></div><p>While for sex, a highly significant QTL was finely mapped on LG11 with the confidence interval of 0.37 cM (34.61–34.98 cM; Fig. <a reveal-id="DST043F6" data-open="DST043F6" class="link link-reveal link-table xref-fig">6</a>b). Moreover, no global difference (female : male = 1.01) in the recombination rate was observed for this linkage group between the two sex-specific maps (Table <a reveal-id="DST043TB3" data-open="DST043TB3" class="link link-reveal link-table xref-fig">3</a>). Association analysis revealed 27 sex-related markers with high statistical significance (<em>P</em> &lt; 1<em>E</em>−6; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) and all of them fell into the narrow QTL region identified by the QTL mapping analysis. Interestingly, one sex-related marker named f93422 was located in the coding region of a transcription factor gene, <em>ZNFX1</em> (NFX1-type zinc finger-containing 1), which has been shown to be tightly linked with <em>AMHR2</em> (anti-Mullerian hormone receptor type II), the sex-determination gene in the tiger pufferfish, <em>Takifugu rubripes.</em><sup><a reveal-id="DST043C36" data-open="DST043C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                            <h3 scrollto-destination=4926112 id="4926112" class="section-title" data-legacy-id=s3f>3.6. Integration of the linkage map, genomic scaffolds and a physical map</h3>
<p>A high-resolution linkage map can serve as an important platform for integrating multiple genomic resources. One such application is to improve genome assembly by orienting genomic scaffolds. For <em>C. farreri</em>, a majority of larger genomic scaffolds contained at least one BsaXI site (77% for scaffolds of &gt; 5kb; 92% for scaffolds of &gt;10 kb) and therefore, it could be potentially oriented using a high-density linkage map. Of the 3806 markers in the consensus linkage map, 2174 could be unambiguously anchored to scaffolds. These scaffolds were not only useful for directing a higher-level genome assembly, but also could serve as ‘long’ surrogates for 2b-RAD tags to enhance their utility in unifying genomic resources. To demonstrate this potential, we attempted to integrate the linkage map with a BAC-based physical map previously constructed for <em>C. farreri.</em><sup><a reveal-id="DST043C18" data-open="DST043C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The physical map contained 2514 BAC contigs that had BESs; however, a large fraction of them had only a few BESs (e.g. 73% with less than three sequenced BAC clones), thus limiting the integration efficiency. Even so, 681 BAC contigs could still be linked with the linkage map through sequence comparison between BESs and marker-associated scaffolds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary dataset</a>). An example showing the integration of LG1 and BAC contigs is present in Fig. <a reveal-id="DST043F8" data-open="DST043F8" class="link link-reveal link-table xref-fig">8</a>. </p>                        <a id="4926114" scrollto-destination="4926114"></a>
<div id="DST043F8" data-id="DST043F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04308.png?Expires=1509238925&Signature=bmTU0QlW8aShIN93bbd6WXSK4v7-SJETelq-rukSUDsMd~-5QyZXooN8XMc9JhNg6pr795c8xGEqoAG2VgvKzNWDVhHNHgkJFoNMSVidb7ss-6wkg4tBXIccSbHLQR-T~JqqaroHvzB6E2ExKUs8v~BUjt~OHHzDn~DVrFPiIdh-GRVRDQUSN~v2gqvNNPwhrfSfMdLNCUicLFGdK3DJuFO1mg66Gezpcn0rdY36UHWxT0QfNgebruvIVjbVO6OEt3uAcxG0lpBbFRtdh6En0UFJZtEsZsQQa19vSQO7vs2oa1pYPPG2QHgOSXYFiNDXmdyUg7c4DJ-5BhaGE6LByg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic demonstration of the integration of linkage group LG1, genomic scaffolds and a BAC-based physical map." path-from-xml="dst04308.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926114/dst04308.tiff" path-from-xml="dst04308.tiff" target="_blank">View large</a><a section="4926114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04308.png?Expires=1509238925&Signature=NZSdczKJsevntUZYvuJW8ss4DOweeNK7aoaQ64WJU0kb7p0ZWwyW8BnsCJz34hZLptc0GRr4FvKDyWFT1BLQJ4kGPiITY5u3zgME93imbjN8yLarMZS8ilUfAXvEaZADq1nNPSuCgmWgSNSyDtp10VeD3n4oWc7r5Sg2o1BjrlC5fmt9EkxxZ-V3XPBRmfK~NsGcWH7v~GXnUNg4x8QO0HCMus47B3orvgDcnCYcJmlpOrKT6nhuebaBpWOy0HGpvarRYoo8XAoXUlp9XeipeOOM-ydxv3XcVnxN65kI9VJcDFIEbj3mSg3-oiM0qTJEGqopacXyH2CzdRsoQ8Pc7A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926114&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04308.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic demonstration of the integration of linkage group LG1, genomic scaffolds and a BAC-based physical map.</p></div></div><div content-id="DST043F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/m_dst04308.png?Expires=1509238925&Signature=bmTU0QlW8aShIN93bbd6WXSK4v7-SJETelq-rukSUDsMd~-5QyZXooN8XMc9JhNg6pr795c8xGEqoAG2VgvKzNWDVhHNHgkJFoNMSVidb7ss-6wkg4tBXIccSbHLQR-T~JqqaroHvzB6E2ExKUs8v~BUjt~OHHzDn~DVrFPiIdh-GRVRDQUSN~v2gqvNNPwhrfSfMdLNCUicLFGdK3DJuFO1mg66Gezpcn0rdY36UHWxT0QfNgebruvIVjbVO6OEt3uAcxG0lpBbFRtdh6En0UFJZtEsZsQQa19vSQO7vs2oa1pYPPG2QHgOSXYFiNDXmdyUg7c4DJ-5BhaGE6LByg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Schematic demonstration of the integration of linkage group LG1, genomic scaffolds and a BAC-based physical map." path-from-xml="dst04308.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4926114/dst04308.tiff" path-from-xml="dst04308.tiff" target="_blank">View large</a><a section="4926114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst04308.png?Expires=1509238925&Signature=NZSdczKJsevntUZYvuJW8ss4DOweeNK7aoaQ64WJU0kb7p0ZWwyW8BnsCJz34hZLptc0GRr4FvKDyWFT1BLQJ4kGPiITY5u3zgME93imbjN8yLarMZS8ilUfAXvEaZADq1nNPSuCgmWgSNSyDtp10VeD3n4oWc7r5Sg2o1BjrlC5fmt9EkxxZ-V3XPBRmfK~NsGcWH7v~GXnUNg4x8QO0HCMus47B3orvgDcnCYcJmlpOrKT6nhuebaBpWOy0HGpvarRYoo8XAoXUlp9XeipeOOM-ydxv3XcVnxN65kI9VJcDFIEbj3mSg3-oiM0qTJEGqopacXyH2CzdRsoQ8Pc7A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=4926114&ar=347657&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04308.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic demonstration of the integration of linkage group LG1, genomic scaffolds and a BAC-based physical map.</p></div></div>                            <h2 scrollto-destination=4926115 id="4926115" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=4926116 id="4926116" class="section-title" data-legacy-id=s4a>4.1. Genome sequencing in bivalves</h3>
<p>In spite of species abundance and diverse geographical distribution, little effort has been devoted to decoding bivalve genomes. To date, whole-genome sequencing has been performed for only two bivalve species, i.e. Pearl oyster (<em>Pinctada fucata</em>)<sup><a reveal-id="DST043C37" data-open="DST043C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and Pacific oyster (<em>Crassostrea gigas</em>),<sup><a reveal-id="DST043C38" data-open="DST043C38" class="link link-ref link-reveal xref-bibr">38</a></sup> providing the first glimpse into bivalve genome architecture. Apparently, genome sequencing of a broader range of bivalves would enable a better understanding of their phylogeny, speciation and diversification. Our study represents the first step towards fully decoding a scallop genome. Genome analysis revealed that <em>C. farreri</em> possesses a highly heterozygous genome. This finding is not unusual and has been reported in other bivalve species.<sup><a reveal-id="DST043C37" data-open="DST043C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST043C38" data-open="DST043C38" class="link link-ref link-reveal xref-bibr">38</a></sup> For example, it has been shown that, for the Pacific oyster, average density of SNPs is one SNP per 60 bp in coding regions and one per 40 bp in non-coding regions.<sup><a reveal-id="DST043C39" data-open="DST043C39" class="link link-ref link-reveal xref-bibr">39</a></sup> High genome heterozygosity poses the challenge of efficient genome assembly. In such instance, a high-resolution linkage map would be much valuable in that they can orientate the small scaffolds to improve genome assembly.</p>                            <h3 scrollto-destination=4926118 id="4926118" class="section-title" data-legacy-id=s4b>4.2. Devising an optimal 2b-RAD sequencing plan for linkage mapping</h3>
<p>Large-scale linkage mapping studies generally call for a cost-effective sequencing plan that reasonably balances the sequencing cost and genotyping accuracy. Through simulation analysis, a sequencing depth of 20× for both parents and progenies can provide sufficiently high genotyping accuracy (&gt;96%; unpublished data). Determining the amount of sequencing necessary for a desired sequencing depth would require the knowledge of a total number of restriction sites in a given genome. Without a reference genome, the total number of restriction sites can be simply predicted based on genome size and GC content by assuming sites that are Poisson distributed. However, it has been shown that such estimation departed considerably from the actual numbers in some instances.<sup><a reveal-id="DST043C1" data-open="DST043C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Our study demonstrates that genome survey sequencing can facilitate devising an optimal 2b-RAD sequencing plan by providing sufficient genome information to determine the minimal amount of sequencing required for a given sequencing depth. Meanwhile, the obtained genome information can also be used to enhance <em>de novo</em> 2b-RAD analysis in many other aspects, such as evaluation of the efficiency and reliability of <em>de novo</em> reference site reconstruction, identification of genes surrounding 2b-RAD tags and association of 2b-RAD tags with public genomic resources.</p><p>For species with very large genomes, sequencing all BsaXI sites at a depth of 20× for all individuals remains a substantial investment. For example, sequencing 200 human individuals with the genome size of ∼3 Gb would require 5.2 billion reads, which are approximately equivalent to the number of reads produced from &gt;4 full sequencing runs using the HiSeq2000 platform. One advantage of 2b-RAD methodology is the option to adjust marker density to the desired level using selective adaptors. Sequencing cost can be further reduced by sequencing a subset of BsaXI sites deriving from RR libraries. We have demonstrated that sequencing a <em>C. farreri</em> RR library only required about one-sixth of the sequencing cost for a standard library at a given sequencing coverage, thus reducing the total sequencing cost dramatically. Therefore, sequencing RR libraries represents an effective option for large-scale linkage mapping studies dealing with species with large genomes.</p>                            <h3 scrollto-destination=4926121 id="4926121" class="section-title" data-legacy-id=s4c>4.3. Linkage map construction and QTL analysis</h3>
<p>Thus far, linkage maps have been constructed for many bivalve species including <em>C. farreri.</em><sup><a reveal-id="DST043C14  DST043C17" data-open="DST043C14  DST043C17" class="link link-ref link-reveal xref-bibr">14–17</a></sup> However, these maps were usually built using hundreds of markers and the resolution was generally low (mostly 10–20 cM), limiting their use in fine-scale QTL mapping and many other applications. In this study, the linkage map constructed for <em>C. farreri</em> contained 3806 markers, a marker density that has, to our knowledge, never been reached for any other bivalve species. This linkage map covered nearly the whole genome (99.5%) with a resolution of 0.41 cM. In addition, ∼10% of the markers in the linkage map residing in or close to genes as revealed by mapping them to the annotated genomic scaffolds. Our map, therefore, provides sufficient resolution for fine-scale QTL mapping and facilitates the discovery of quantitative trait genes.</p><p>Growth traits are of particular interest to scallop researchers due to their high commercial significance in scallop aquaculture. QTL mapping represents an efficient approach to identify genetic loci underlying these traits for marker-assisted selection in genetic breeding. Two growth-related QTLs were detected on the linkage groups LG1 and LG3, which are also supported by the association analysis, a complementary approach to evaluate the QTL mapping results. The markers located at the confidence intervals of these QTLs constitute a valuable marker set for further evaluation of their utility in marker-assisted selection. In particular, the homeotic gene <em>PROP1</em>, which was identified from the QTL region on LG3, drew our great interest. This gene encodes a paired-like homeodomain protein that is necessary for <em>PIT1</em> (growth hormone factor 1) expression. In mammals, mutations in this gene can result in impaired production of growth hormone and other pituitary hormones.<sup><a reveal-id="DST043C35" data-open="DST043C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Polymorphisms in this gene have been associated with production traits in livestock animals, such as cattle,<sup><a reveal-id="DST043C40" data-open="DST043C40" class="link link-ref link-reveal xref-bibr">40</a></sup> goats <sup><a reveal-id="DST043C41" data-open="DST043C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST043C42" data-open="DST043C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and sheep.<sup><a reveal-id="DST043C43" data-open="DST043C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The marker f68558 is probably not the causal locus that is responsible for growth trait variation in <em>C. farreri</em>. Instead, we think it is more likely that f68558 is in tight linkage disequilibrium with the causal mutation that is possibly located in the regulatory region (e.g. promoter) of <em>PROP1</em>. The expression level of <em>PROP1</em> has been checked in major adult tissues (e.g. adductor muscle, digestive gland, gill, mantle, male and female gonads and kidney) of <em>C. farreri</em>. It turned out that <em>PROP1</em> was not expressed in neither of these tissues (data not shown), suggesting that scallop <em>PROP1</em> is probably expressed and functioning during early embryonic development just like its homologues in vertebrates. In vertebrates, <em>PROP1</em> is only temporarily expressed during early pituitary development.<sup><a reveal-id="DST043C44" data-open="DST043C44" class="link link-ref link-reveal xref-bibr">44</a></sup> However, determining the temporal and spatial expression of <em>PROP1</em> in bivalves is not straightforward, since the counterpart of pituitary has not been established in bivalves. Further work would be needed to overcome these barriers to evaluate the role(s) of <em>PROP1</em> in scallop growth regulation.</p><p>Unlike many other animals, scallops do not possess recognizable heteromorphic sex chromosomes,<sup><a reveal-id="DST043C45" data-open="DST043C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and little is currently known about their sex-determination mechanism. The use of molecular markers in mapping experiments to identify regions involved in sex determination represents an efficient approach to study sex-determination systems in species without heteromorphic sex chromosomes. In this study, both QTL mapping and association analysis revealed a highly significant sex-related QTL on LG11, suggesting the presence of a narrow sex-determination region (34.61–34.98 cM) in <em>C. farreri</em>. No global differences in recombination rates were detected for LG11 between the two sexes, which may account for the indistinguishable morphologies of sex chromosomes in <em>C. farreri</em>. Interestingly, we found that one sex-related marker (i.e. f93422) was situated at the transcription factor gene, <em>ZNFX1</em>, which has been shown to be tightly linked with the sex-determination gene <em>AMHR2</em> in the tiger pufferfish.<sup><a reveal-id="DST043C36" data-open="DST043C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Unfortunately, we do not have direct evidence to support the physical linkage between <em>ZNFX1</em> and <em>AMHR2</em> based on the available <em>C. farreri</em> genome information. Furthermore, searching another two molluscan genomes, i.e. the Pacific oyster <em>C. gigas</em> (<a class="link link-uri" href="http://gigadb.org/pacific_oyster">http://gigadb.org/pacific_oyster</a>) and the owl limpet <em>Lottia gigantea</em> (<a class="link link-uri" href="http://metazoa.ensembl.org/Lottia_gigantea">http://metazoa.ensembl.org/Lottia_gigantea</a>), did not support the two genes' linkage either, though this does not necessarily exclude the possibility of their physical linkage because these genome assemblies are still largely fragmented (scaffold N50 is 401 kb for <em>C. gigas</em><sup><a reveal-id="DST043C38" data-open="DST043C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and 1.87 Mb for <em>L. gigantean</em><sup><a reveal-id="DST043C46" data-open="DST043C46" class="link link-ref link-reveal xref-bibr">46</a></sup>). Our group has initiated the whole-genome sequencing project for <em>C. farreri</em>; therefore, this issue will be settled in the near future.</p>                            <h3 scrollto-destination=4926125 id="4926125" class="section-title" data-legacy-id=s4d>4.4. Integration of the linkage map with multiple genomic resources</h3>
<p>Recent studies have shown that RAD-based linkage maps can greatly improve large-scale genome assembly.<sup><a reveal-id="DST043C47" data-open="DST043C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DST043C48" data-open="DST043C48" class="link link-ref link-reveal xref-bibr">48</a></sup> However, RAD tags are usually short in length (35–100 bp), thus limiting their direct use for integration with other important genomic resources generated from RNA-Seq, ChIP-Seq, exome sequencing or BAC–fosmid-end sequencing. Here, we propose that RAD tags can be effectively extended using the scaffolds obtained from genome survey sequencing. For <em>C. farreri</em>, although a majority of the scaffolds were not very long (scaffold N50 = 1.5 kb), they could already serve as ‘long’ surrogates for 2b-RAD tags to enhance their utility. We have demonstrated that this approach could facilitate the integration of the scallop linkage map with the BAC-based physical map, even though a large fraction of BAC contigs contained only a few BESs (e.g. 73% with less than three sequenced BAC clones). Besides, such approach can also facilitate the identification of genes surrounding a given tag, which can be crucial for the identification of quantitative trait genes in QTL mapping and association studies. Therefore, it can be envisioned that this scallop linkage map would serve as an important platform for unifying genomic resources that have been accumulated for <em>C. farreri</em>.</p>                            <h3 scrollto-destination=4926128 id="4926128" class="section-title" data-legacy-id=s4e>4.5. Conclusions</h3>
<p>We generated a preliminary reference genome for a bivalve mollusc, expanding the genome resources currently available for bivalves—a group of animals that remain largely unexplored in terms of genome sequencing. A high-resolution linkage map was constructed for <em>C. farreri</em> with a marker density that has, to our knowledge, never been achieved in any other molluscs. Two growth-related QTLs and one putative sex-determination region were detected, and candidate genes identified from these QTL regions represent important targets for further evaluation. Integration of the genome map, linkage map and physical map exemplifies how to build up a comprehensive genomic framework in a non-model organism.</p>                            <h2 scrollto-destination=4926133 id="4926133" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4926137 id="4926137" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the National Natural Science Foundation of China (31130054), National Basic Research Program of China (2010CB126402), National High Technology Research and Development Program of China (2012AA10A402, 2012AA10A405 and 2012AA092204) and Natural Science Foundation for Distinguished Young Scholars of Shandong Province (2013JQE29001).</p>                            <h2 scrollto-destination=4926142 id="4926142" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr Xiaojun Zhang (Institute of Oceanology, Chinese Academy of Sciences) for kindly providing the physical map data.</p>                            <h2 scrollto-destination=4926147 id="4926147" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dst043c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davey</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Blaxter</div> <div class="given-names">M.L.</div></div>. </div><div class="article-title">RADSeq: next-generation population genetics</div>, <div class="source">Brief Funct. Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">416</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RADSeq%3A%20next-generation%20population%20genetics&amp;author=J.W.%20Davey&amp;author=M.L.%20Blaxter&amp;publication_year=2011&amp;journal=Brief%20Funct.%20Genomics&amp;volume=9&amp;pages=416-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/elq031" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Felq031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Felq031"> </span></div> </div></div></div></div></div><div content-id="dst043c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Meyer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">McKay</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Matz</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">2b-RAD: a simple and flexible method for genome-wide genotyping</div>, <div class="source"><em>Nat. Method</em>s</div> , <div class="year">2012</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">808</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=2b-RAD%3A%20a%20simple%20and%20flexible%20method%20for%20genome-wide%20genotyping&amp;author=S.%20Wang&amp;author=E.%20Meyer&amp;author=J.%20McKay&amp;author=M.V.%20Matz&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=808-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2023" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22609625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peterson</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Weber</div> <div class="given-names">J.N.</div></div>,  <div class="name"><div class="surname">Kay</div> <div class="given-names">E.H.</div></div>,  <div class="name"><div class="surname">Fisher</div> <div class="given-names">H.S.</div></div>,  <div class="name"><div class="surname">Hoekstra</div> <div class="given-names">H.E.</div></div>. </div><div class="article-title">Double digest RADseq: an inexpensive method for <em>De Novo</em> SNP discovery and genotyping in model and non-model species</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e37135</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double%20digest%20RADseq%3A%20an%20inexpensive%20method%20for%20De%20Novo%20SNP%20discovery%20and%20genotyping%20in%20model%20and%20non-model%20species&amp;author=B.K.%20Peterson&amp;author=J.N.%20Weber&amp;author=E.H.%20Kay&amp;author=H.S.%20Fisher&amp;author=H.E.%20Hoekstra&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e37135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0037135" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0037135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0037135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22675423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Appeltans</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ahyong</div> <div class="given-names">S.T.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">The magnitude of global marine species diversity</div>, <div class="source">Curr. Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">2189</div>-<div class="lpage">202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20magnitude%20of%20global%20marine%20species%20diversity&amp;author=W.%20Appeltans&amp;author=S.T.%20Ahyong&amp;author=G.%20Anderson&amp;publication_year=2012&amp;journal=Curr.%20Biol.&amp;volume=22&amp;pages=2189-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2012.09.036" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2012.09.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2012.09.036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23159596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="collab">FAO</div>, <div class="source">The State of World Fisheries and Aquaculture 2010</div> , <div class="year">2010</div><div class="publisher-loc">Rome</div><div class="publisher-name">FAO Fisheries and Aquaculture Department, FAO</div></div></div></div></div><div content-id="dst043c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Transcriptome sequencing and de novo analysis for Yesso scallop (<em>Patinopecten yessoensis</em>) using 454 GS FLX</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e21560</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20and%20de%20novo%20analysis%20for%20Yesso%20scallop%20(Patinopecten%20yessoensis)%20using%20454%20GS%20FLX&amp;author=R.%20Hou&amp;author=Z.%20Bao&amp;author=S.%20Wang&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e21560" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021560" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21720557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Transcriptome sequencing of Zhikong scallop (<em>Chlamys farreri</em>) and comparative transcriptomic analysis with Yesso scallop (<em>Patinopecten yessoensis</em>)</div>, <div class="source">PLoS One</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e63927</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20Zhikong%20scallop%20(Chlamys%20farreri)%20and%20comparative%20transcriptomic%20analysis%20with%20Yesso%20scallop%20(Patinopecten%20yessoensis)&amp;author=S.%20Wang&amp;author=R.%20Hou&amp;author=Z.%20Bao&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e63927" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0063927" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0063927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0063927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23667690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Thorne</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Vieira</div> <div class="given-names">F.A.</div></div>,  <div class="name"><div class="surname">Cardoso</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Power</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Peck</div> <div class="given-names">L.S.</div></div>. </div><div class="article-title">Insights into shell deposition in the Antarctic bivalve <em>Laternula elliptica</em>: gene discovery in the mantle transcriptome using 454 pyrosequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">362</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20shell%20deposition%20in%20the%20Antarctic%20bivalve%20Laternula%20elliptica%3A%20gene%20discovery%20in%20the%20mantle%20transcriptome%20using%20454%20pyrosequencing&amp;author=M.S.%20Clark&amp;author=M.A.%20Thorne&amp;author=F.A.%20Vieira&amp;author=J.C.%20Cardoso&amp;author=D.M.%20Power&amp;author=L.S.%20Peck&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=362" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-362" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20529341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Manahan</div> <div class="given-names">D.T.</div></div>. </div><div class="article-title">Gene expression profiling of genetically determined growth variation in bivalve larvae (<em>Crassostrea gigas</em>)</div>, <div class="source">J. Exp. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">213</div> (pg. <div class="fpage">749</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profiling%20of%20genetically%20determined%20growth%20variation%20in%20bivalve%20larvae%20(Crassostrea%20gigas)&amp;author=E.%20Meyer&amp;author=D.T.%20Manahan&amp;publication_year=2010&amp;journal=J.%20Exp.%20Biol.&amp;volume=213&amp;pages=749-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jeb.037242" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjeb.037242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjeb.037242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20154190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ghiselli</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Milani</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">P.L.</div></div>, et al. </div><div class="article-title"><em>De novo</em> assembly of the Manila clam <em>Ruditapes philippinarum</em> transcriptome provides new insights into expression bias, mitochondrial doubly uniparental inheritance and sex determination</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">771</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20the%20Manila%20clam%20Ruditapes%20philippinarum%20transcriptome%20provides%20new%20insights%20into%20expression%20bias%2C%20mitochondrial%20doubly%20uniparental%20inheritance%20and%20sex%20determination&amp;author=F.%20Ghiselli&amp;author=L.%20Milani&amp;author=P.L.%20Chang&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=29&amp;pages=771-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr248" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21976711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Molecular characterization of TGF-β type I receptor gene (Tgfbr1) in <em>Chlamys farreri</em>, and the association of allelic variants with growth traits</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e51005</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20TGF-%CE%B2%20type%20I%20receptor%20gene%20(Tgfbr1)%20in%20Chlamys%20farreri%2C%20and%20the%20association%20of%20allelic%20variants%20with%20growth%20traits&amp;author=H.%20Guo&amp;author=Z.%20Bao&amp;author=J.%20Li&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e51005" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0051005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0051005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0051005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23209843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">You</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genetic diversity of the sulfotransferase-like gene and one nonsynonymous SNP associated with growth traits of clam, <em>Meretrix meretrix</em></div>, <div class="source">Mol. Biol. Rep.</div> , <div class="year">2012</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">1323</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20the%20sulfotransferase-like%20gene%20and%20one%20nonsynonymous%20SNP%20associated%20with%20growth%20traits%20of%20clam%2C%20Meretrix%20meretrix&amp;author=C.%20Wang&amp;author=Y.%20You&amp;author=H.%20Wang&amp;author=B.%20Liu&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Rep.&amp;volume=39&amp;pages=1323-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11033-011-0865-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11033-011-0865-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11033-011-0865-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21598107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Breton</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">D.T.</div></div>,  <div class="name"><div class="surname">Shepardson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Novel protein genes in animal mtDNA: a new sex determination system in freshwater mussels (Bivalvia: Unionoida)?</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1645</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20protein%20genes%20in%20animal%20mtDNA%3A%20a%20new%20sex%20determination%20system%20in%20freshwater%20mussels%20(Bivalvia%3A%20Unionoida)%3F&amp;author=S.%20Breton&amp;author=D.T.%20Stewart&amp;author=S.%20Shepardson&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=1645-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq345" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21172831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">AFLP linkage map of an intraspecific cross in <em>Chlamys farreri</em></div>, <div class="source">J. Shellfish Res.</div> , <div class="year">2004</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">491</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="dst043c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xiang</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.J.</div></div>. </div><div class="article-title">Construction of AFLP-based genetic linkage map for Zhikong scallop, <em>Chlamys farreri</em> Jones et Preston and mapping of sex-linked markers</div>, <div class="source">Aquaculture</div> , <div class="year">2005</div>, vol. <div class="volume">245</div> (pg. <div class="fpage">63</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20AFLP-based%20genetic%20linkage%20map%20for%20Zhikong%20scallop%2C%20Chlamys%20farreri%20Jones%20et%20Preston%20and%20mapping%20of%20sex-linked%20markers&amp;author=L.%20Li&amp;author=J.H.%20Xiang&amp;author=X.%20Liu&amp;author=Y.%20Zhang&amp;author=B.%20Dong&amp;author=X.J.%20Zhang&amp;publication_year=2005&amp;journal=Aquaculture&amp;volume=245&amp;pages=63-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2004.12.015" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2004.12.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2004.12.015"> </span></div> </div></div></div></div></div><div content-id="dst043c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ni</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>. </div><div class="article-title">A preliminary genetic map of Zhikong scallop (<em>Chlamys farreri</em> Jones et Preston 1904)</div>, <div class="source">Aquaculture Res.</div> , <div class="year">2005</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">643</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20preliminary%20genetic%20map%20of%20Zhikong%20scallop%20(Chlamys%20farreri%20Jones%20et%20Preston%201904)&amp;author=L.%20Wang&amp;author=L.%20Song&amp;author=Y.%20Chang&amp;author=W.%20Xu&amp;author=D.%20Ni&amp;author=X.%20Guo&amp;publication_year=2005&amp;journal=Aquaculture%20Res.&amp;volume=36&amp;pages=643-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/are.2005.36.issue-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fare.2005.36.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fare.2005.36.issue-7"> </span></div> </div></div></div></div></div><div content-id="dst043c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Construction of microsatellite-based linkage maps and identification of size-related quantitative trait loci for Zhikong scallop (<em>Chlamys farreri</em>)</div>, <div class="source">Anim. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">821</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20microsatellite-based%20linkage%20maps%20and%20identification%20of%20size-related%20quantitative%20trait%20loci%20for%20Zhikong%20scallop%20(Chlamys%20farreri)&amp;author=A.%20Zhan&amp;author=J.%20Hu&amp;author=X.%20Hu&amp;publication_year=2009&amp;journal=Anim.%20Genet.&amp;volume=40&amp;pages=821-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2009.40.issue-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2009.40.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2009.40.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19515089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A BAC-based physical map of Zhikong scallop (<em>Chlamys farreri</em> Jones et Preston)</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e27612</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20BAC-based%20physical%20map%20of%20Zhikong%20scallop%20(Chlamys%20farreri%20Jones%20et%20Preston)&amp;author=X.%20Zhang&amp;author=C.%20Zhao&amp;author=C.%20Huang&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e27612" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0027612" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0027612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0027612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>. </div><div class="article-title">FISH mapping and identification of Zhikong scallop (<em>Chlamys farreri</em>) chromosomes</div>, <div class="source">Mar. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">151</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FISH%20mapping%20and%20identification%20of%20Zhikong%20scallop%20(Chlamys%20farreri)%20chromosomes&amp;author=L.%20Zhang&amp;author=Z.%20Bao&amp;author=S.%20Wang&amp;author=X.%20Hu&amp;author=J.%20Hu&amp;publication_year=2008&amp;journal=Mar.%20Biotechnol.&amp;volume=10&amp;pages=151-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-007-9045-x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-007-9045-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-007-9045-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17955291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Fosmid library construction and initial analysis of end sequences in Zhikong scallop (<em>Chlamys farreri</em>)</div>, <div class="source">Mar. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">606</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fosmid%20library%20construction%20and%20initial%20analysis%20of%20end%20sequences%20in%20Zhikong%20scallop%20(Chlamys%20farreri)&amp;author=L.%20Zhang&amp;author=Z.%20Bao&amp;author=J.%20Cheng&amp;publication_year=2007&amp;journal=Mar.%20Biotechnol.&amp;volume=9&amp;pages=606-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-007-9014-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-007-9014-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-007-9014-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17605073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Scheuring</div> <div class="given-names">C.F.</div></div>, et al. </div><div class="article-title">Construction and characterization of two bacterial artificial chromosome libraries of Zhikong scallop, <em>Chlamys farreri</em> Jones et Preston, and identification of BAC clones containing the genes involved in its innate immune system</div>, <div class="source">Mar. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">358</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20two%20bacterial%20artificial%20chromosome%20libraries%20of%20Zhikong%20scallop%2C%20Chlamys%20farreri%20Jones%20et%20Preston%2C%20and%20identification%20of%20BAC%20clones%20containing%20the%20genes%20involved%20in%20its%20innate%20immune%20system&amp;author=Y.%20Zhang&amp;author=X.%20Zhang&amp;author=C.F.%20Scheuring&amp;publication_year=2008&amp;journal=Mar.%20Biotechnol.&amp;volume=10&amp;pages=358-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-007-9071-8" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-007-9071-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-007-9071-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18283517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Expressed sequence tags from the Zhikong scallop (<em>Chlamys farreri</em>): discovery and annotation of host-defense genes</div>, <div class="source">Fish Shellfish Immunol.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">744</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expressed%20sequence%20tags%20from%20the%20Zhikong%20scallop%20(Chlamys%20farreri)%3A%20discovery%20and%20annotation%20of%20host-defense%20genes&amp;author=L.%20Wang&amp;author=L.%20Song&amp;author=J.%20Zhao&amp;publication_year=2009&amp;journal=Fish%20Shellfish%20Immunol.&amp;volume=26&amp;pages=744-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fsi.2009.03.002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fsi.2009.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fsi.2009.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19328855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maniatis</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fritsch</div> <div class="given-names">E.F.</div></div>,  <div class="name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div>. </div>, <div class="source">Molecular Cloning: A Laboratory Manual</div> , <div class="year">1989</div><div class="publisher-loc">Cold Spring Harbor, NY</div><div class="publisher-name">Cold Spring Harbor Laboratory Press</div></div></div></div></div><div content-id="dst043c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">C.K.</div></div>. </div><div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1859</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=14&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div> </div></div></div></div></div><div content-id="dst043c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Construction of integrated genetic linkage maps by means of a new computer package: Joinmap</div>, <div class="source">Plant J.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">739</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20integrated%20genetic%20linkage%20maps%20by%20means%20of%20a%20new%20computer%20package%3A%20Joinmap&amp;author=P.%20Stam&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=739-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1993.3.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1993.3.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1993.3.issue-5"> </span></div> </div></div></div></div></div><div content-id="dst043c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Accurate construction of consensus genetic maps via integer linear programming</div>, <div class="source">IEEE/ACM Trans. Comput. Biol. Bioinform.</div> , <div class="year">2011</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">381</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20construction%20of%20consensus%20genetic%20maps%20via%20integer%20linear%20programming&amp;author=Y.%20Wu&amp;author=T.J.%20Close&amp;author=S.%20Lonardi&amp;publication_year=2011&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol.%20Bioinform.&amp;volume=8&amp;pages=381-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TCBB.2010.35" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTCBB.2010.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTCBB.2010.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20479505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorensen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Thomsen</div> <div class="given-names">H.</div></div>. </div>, <div class="source">Using DMU for QTL mapping</div> , <div class="year">2003</div><div class="publisher-loc">Foulum, Denmark</div><div class="publisher-name">Aarhus University</div></div></div></div></div><div content-id="dst043c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piepho</div> <div class="given-names">H.P.</div></div>. </div><div class="article-title">A quick method for computing approximate thresholds for quantitative trait loci detection</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">425</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20quick%20method%20for%20computing%20approximate%20thresholds%20for%20quantitative%20trait%20loci%20detection&amp;author=H.P.%20Piepho&amp;publication_year=2001&amp;journal=Genetics&amp;volume=157&amp;pages=425-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11139522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A quick method to calculate QTL confidence interval</div>, <div class="source">J. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">355</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20quick%20method%20to%20calculate%20QTL%20confidence%20interval&amp;author=H.%20Li&amp;publication_year=2011&amp;journal=J.%20Genet.&amp;volume=90&amp;pages=355-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12041-011-0077-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12041-011-0077-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12041-011-0077-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21869489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">GWAF: an R package for genome-wide association analyses with family data</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">580</div>-<div class="lpage">1</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GWAF%3A%20an%20R%20package%20for%20genome-wide%20association%20analyses%20with%20family%20data&amp;author=M.H.%20Chen&amp;author=Q.%20Yang&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=580-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp710" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp710" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp710"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20040588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">SOAP2: an improved ultrafast tool for short read alignment</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1966</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The karyotyping of <em>C. farreri</em></div>, <div class="source">J. Ocean Univ. Qingdao</div> , <div class="year">1990</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">81</div>-<div class="lpage">5</div>)</div></div></div></div><div content-id="dst043c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chakravarti</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lasher</div> <div class="given-names">L.K.</div></div>,  <div class="name"><div class="surname">Reefer</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">A maximum likelihood method for estimating genome length using genetic linkage data</div>, <div class="source">Genetics</div> , <div class="year">1991</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">175</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20maximum%20likelihood%20method%20for%20estimating%20genome%20length%20using%20genetic%20linkage%20data&amp;author=A.%20Chakravarti&amp;author=L.K.%20Lasher&amp;author=J.E.%20Reefer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=128&amp;pages=175-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2060775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fishman</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kelly</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Willis</div> <div class="given-names">J.H.</div></div>. </div><div class="article-title">A genetic map in the <em>Mimulus guttatus</em> species complex reveals transmission ratio distortion due to heterospecific interactions</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">1701</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20map%20in%20the%20Mimulus%20guttatus%20species%20complex%20reveals%20transmission%20ratio%20distortion%20due%20to%20heterospecific%20interactions&amp;author=L.%20Fishman&amp;author=A.J.%20Kelly&amp;author=E.%20Morgan&amp;author=J.H.%20Willis&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=1701-16" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11779808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Cogan</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Pfaffle</div> <div class="given-names">R.W.</div></div>, et al. </div><div class="article-title">Mutations in PROP1 cause familial combined pituitary hormone deficiency</div>, <div class="source">Nat. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">147</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20in%20PROP1%20cause%20familial%20combined%20pituitary%20hormone%20deficiency&amp;author=W.%20Wu&amp;author=J.D.%20Cogan&amp;author=R.W.%20Pfaffle&amp;publication_year=1998&amp;journal=Nat.%20Genet.&amp;volume=18&amp;pages=147-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0298-147" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0298-147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0298-147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9462743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kamiya</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kai</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Tasumi</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A trans-species missense SNP in <em>Amhr2</em> is associated with sex determination in the tiger pufferfish, <em>Takifugu rubripes</em> (fugu)</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002798</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20trans-species%20missense%20SNP%20in%20Amhr2%20is%20associated%20with%20sex%20determination%20in%20the%20tiger%20pufferfish%2C%20Takifugu%20rubripes%20(fugu)&amp;author=T.%20Kamiya&amp;author=W.%20Kai&amp;author=S.%20Tasumi&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002798" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002798" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002798"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22807687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takeuchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawashima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koyanagi</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Draft genome of the pearl oyster <em>Pinctada fucata</em>: a platform for understanding bivalve biology</div>, <div class="source">DNA Res.</div> , <div class="year">2012</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">117</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20pearl%20oyster%20Pinctada%20fucata%3A%20a%20platform%20for%20understanding%20bivalve%20biology&amp;author=T.%20Takeuchi&amp;author=T.%20Kawashima&amp;author=R.%20Koyanagi&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=117-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22315334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">The oyster genome reveals stress adaptation and complexity of shell formation</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">490</div> (pg. <div class="fpage">49</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20oyster%20genome%20reveals%20stress%20adaptation%20and%20complexity%20of%20shell%20formation&amp;author=G.%20Zhang&amp;author=X.%20Fang&amp;author=X.%20Guo&amp;publication_year=2012&amp;journal=Nature&amp;volume=490&amp;pages=49-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11413" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11413"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22992520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sauvage</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bierne</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lapègue</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Boudry</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Single nucleotide polymorphisms and their relationship to codon usage bias in the Pacific oyster <em>Crassostrea gigas</em></div>, <div class="source">Gene</div> , <div class="year">2007</div>, vol. <div class="volume">406</div> (pg. <div class="fpage">13</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphisms%20and%20their%20relationship%20to%20codon%20usage%20bias%20in%20the%20Pacific%20oyster%20Crassostrea%20gigas&amp;author=C.%20Sauvage&amp;author=N.%20Bierne&amp;author=S.%20Lap%C3%A8gue&amp;author=P.%20Boudry&amp;publication_year=2007&amp;journal=Gene&amp;volume=406&amp;pages=13-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.05.011" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.05.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.05.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17616269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Enns</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Shirley</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Garcia</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Garrett</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Silver</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">Associations of DNA polymorphisms in growth hormone and its transcriptional regulators with growth and carcass traits in two populations of <em>Brangus bulls</em></div>, <div class="source">Genet. Mol. Res.</div> , <div class="year">2007</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">222</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Associations%20of%20DNA%20polymorphisms%20in%20growth%20hormone%20and%20its%20transcriptional%20regulators%20with%20growth%20and%20carcass%20traits%20in%20two%20populations%20of%20Brangus%20bulls&amp;author=M.G.%20Thomas&amp;author=R.M.%20Enns&amp;author=K.L.%20Shirley&amp;author=M.D.%20Garcia&amp;author=A.J.%20Garrett&amp;author=G.A.%20Silver&amp;publication_year=2007&amp;journal=Genet.%20Mol.%20Res.&amp;volume=6&amp;pages=222-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17469072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lei</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A novel missense (A79V) mutation of goat <em>PROP1</em> gene and its association with production traits</div>, <div class="source">Mol. Biol. Rep.</div> , <div class="year">2009</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">2069</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20missense%20(A79V)%20mutation%20of%20goat%20PROP1%20gene%20and%20its%20association%20with%20production%20traits&amp;author=X.%20Lan&amp;author=C.%20Pan&amp;author=L.%20Zhang&amp;author=M.%20Zhao&amp;author=C.%20Zhang&amp;author=C.%20Lei&amp;author=H.%20Chen&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Rep.&amp;volume=36&amp;pages=2069-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11033-008-9418-x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11033-008-9418-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11033-008-9418-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19031010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">H.F.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">H.L.</div></div>, et al. </div><div class="article-title">The prophet of PIT1 gene variation and its effect on growth traits in Chinese indigeous goat</div>, <div class="source">J. Anim. Vet. Adv.</div> , <div class="year">2010</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">2940</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20prophet%20of%20PIT1%20gene%20variation%20and%20its%20effect%20on%20growth%20traits%20in%20Chinese%20indigeous%20goat&amp;author=T.S.%20Xu&amp;author=J.B.%20Liu&amp;author=D.W.%20Yao&amp;author=H.F.%20Cai&amp;author=H.%20Chen&amp;author=H.L.%20Zhou&amp;publication_year=2010&amp;journal=J.%20Anim.%20Vet.%20Adv.&amp;volume=9&amp;pages=2940-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3923/javaa.2010.1143.1145" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3923%2Fjavaa.2010.1143.1145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3923%2Fjavaa.2010.1143.1145"> </span></div> </div></div></div></div></div><div content-id="dst043c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">X.C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.Y.</div></div>,  <div class="name"><div class="surname">Jia</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Z.S.</div></div>,  <div class="name"><div class="surname">Hui</div> <div class="given-names">W.Q.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.B.</div></div>, et al. </div><div class="article-title">Identification of SNPs within the sheep <em>PROP1</em> gene and their effects on wool traits</div>, <div class="source">Mol. Biol. Rep.</div> , <div class="year">2011</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">2723</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20SNPs%20within%20the%20sheep%20PROP1%20gene%20and%20their%20effects%20on%20wool%20traits&amp;author=X.C.%20Zeng&amp;author=H.Y.%20Chen&amp;author=B.%20Jia&amp;author=Z.S.%20Zhao&amp;author=W.Q.%20Hui&amp;author=Z.B.%20Wang&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Rep.&amp;volume=38&amp;pages=2723-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11033-010-0416-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11033-010-0416-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11033-010-0416-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21104025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dasen</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Rosenfeld</div> <div class="given-names">M.G.</div></div>. </div><div class="article-title">Signaling and transcriptional mechanisms in pituitary development</div>, <div class="source">Annu. Rev. Neurosci.</div> , <div class="year">2001</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">327</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signaling%20and%20transcriptional%20mechanisms%20in%20pituitary%20development&amp;author=J.S.%20Dasen&amp;author=M.G.%20Rosenfeld&amp;publication_year=2001&amp;journal=Annu.%20Rev.%20Neurosci.&amp;volume=24&amp;pages=327-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.neuro.24.1.327" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.neuro.24.1.327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.neuro.24.1.327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11283314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Basoa</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Alfonsi</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Perez</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Cequea</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Karyotypes on the scallops <em>Euvola ziczac</em> and <em>Nodipecten nodosus</em> from the Gulf of Cariaco, Sucre State, Venezuela</div>, <div class="source">Bol. Inst. Oceanogr. Venez.</div> , <div class="year">2000</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">49</div>-<div class="lpage">54</div>)</div></div></div></div><div content-id="dst043c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simakov</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Marletaz</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">S.J.</div></div>, et al. </div><div class="article-title">Insights into bilaterian evolution from three spiralian genomes</div>, <div class="source">Nature</div> , <div class="year">2013</div>, vol. <div class="volume">493</div> (pg. <div class="fpage">526</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20bilaterian%20evolution%20from%20three%20spiralian%20genomes&amp;author=O.%20Simakov&amp;author=F.%20Marletaz&amp;author=S.J.%20Cho&amp;publication_year=2013&amp;journal=Nature&amp;volume=493&amp;pages=526-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11696" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23254933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="collab">Heliconius Genome Consortium</div><div class="article-title">Butterfly genome reveals promiscuous exchange of mimicry adaptations among species</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">487</div> (pg. <div class="fpage">94</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22722851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dst043c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dst043c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Postlethwait</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Stacks: building and genotyping loci <em>de novo</em> from short-read sequences</div>, <div class="source">G3 (Bethesda)</div> , <div class="year">2011</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">171</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20building%20and%20genotyping%20loci%20de%20novo%20from%20short-read%20sequences&amp;author=J.M.%20Catchen&amp;author=A.%20Amores&amp;author=P.%20Hohenlohe&amp;author=W.%20Cresko&amp;author=J.%20Postlethwait&amp;publication_year=2011&amp;journal=G3%20(Bethesda)&amp;volume=1&amp;pages=171-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.000240" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.000240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.000240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Yuji Kohara</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347657&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst043-dst043supp_tables.pdf?Expires=1509238926&amp;Signature=SeI0M6o06NbbsnFxxhafsDBRoLlNecY~nv5gblCGRWVl5JxsQuJ63544PNFKXWRt7gTdIWhp0aP11DuHSRjCLZPp6Bihq2pNw0dNSlXK3A9fhw7gMMd0kAPMOLKxJKDixQ2KeegtJeQEDaaVCo7xjU3dN3YEd8RIGGSQZQJpWg5PUWpNEKHlpiI5~IvlME~vzAi9m~M3BtNEbzLfY30Zo9z6-KgHGoJQipBw2rDWOFJUV32FqvkRJN8ySn8Cg-JDXPsEx5hviSw20tSpv-SoJ0KIQPSiYjtR2QZX65gS4~65o-6NjIaSgyQFS07c51t2K-5L9bV-X-ywUfZrh3DKGg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst043/2/dst043-dst043supp.xls?Expires=1509238926&amp;Signature=PHLKEVWMTsDlW8JVizvCnX5M8t7nIAyMFQLJbCABojbhp0~ZK3-EUCkE9cCMuV6CJnAZaX5AtS4ownFgS7nWf-ySdhx0KrOs9tDG6FmSjbmqOtn7Y25jlKX7hdw1Z910n65uLWUNLpGgBuHe2-0jvc1X9IkJ2tlNIwqovz~PHG0TkObnIcCifpkV4Gj39Fu87X9mgNshuwBRai82CmWbpxcQxlMx0vlKudna2ZSSCM8UZYp~VLtZnL7Nri1c5e-vLD~qFmX5NjzSHonMe2vjKv8rpA4~ZdGAl3KmIScPx83CGlTn~b7tolmOI7LgFY3RP8n71h0a~eJ5brDKBbQaMw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">137</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">33</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dst043"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347657"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f1%2f85%2f347657">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897000007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (33)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F1%2F85%2F347657" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347657'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-Resolution Linkage and Quantitative Trait Locus Mapping Aided by Genome Survey Sequencing: Building Up An Integrative Genomic Framework for a Bivalve Mollusc" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f85%2f347657">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

