

<!DOCTYPE html>
<html lang="en">

<head><title>
	Complete Genome Sequence of Finegoldia magna, an Anaerobic Opportunistic Pathogen | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/1/39/357582","@type":"ScholarlyArticle","name":"Complete Genome Sequence of  Finegoldia magna , an Anaerobic Opportunistic Pathogen","datePublished":"2008-02-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2008-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm030","keywords":["whole genome sequence","Gram-positive anaerobic cocci","<italic>Peptostreptococcus magnus</italic>","albumin-binding protein","sortase"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/1/39/357582","author":[{"name":"Goto, Takatsugu","affiliation":"1  Department of Microbiology ,  Wakayama Medical University ,  811-1 Kimiidera, Wakayama, Wakayama 641-0012 ,  Japan","@type":"Person"},{"name":"Yamashita, Atsushi","affiliation":"2  Laboratory of Genomic Information ,  Kitasato Institute for Life Sciences, Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"3  Graduate School of Systems Life Sciences, Kyushu University ,  Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Matsutani, Minenosuke","affiliation":"4  Graduate School of Genetic Resource Technology, Kyushu University ,  Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Todo, Kozo","affiliation":"1  Department of Microbiology ,  Wakayama Medical University ,  811-1 Kimiidera, Wakayama, Wakayama 641-0012 ,  Japan    5  Department of Oral and Maxillofacial Surgery ,  Wakayama Medical University ,  811-1 Kimiidera, Wakayama, Wakayama 641-0012 ,  Japan","@type":"Person"},{"name":"Ohshima, Kenshiro","affiliation":"2  Laboratory of Genomic Information ,  Kitasato Institute for Life Sciences, Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan    6  Department of Computational Biology ,  Graduate School of Frontier Sciences, University of Tokyo ,  5-1-5 Kashinoha, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Toh, Hidehiro","affiliation":"2  Laboratory of Genomic Information ,  Kitasato Institute for Life Sciences, Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan    7  Computational and Experimental Systems Biology Group ,  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Miyamoto, Kazuaki","affiliation":"1  Department of Microbiology ,  Wakayama Medical University ,  811-1 Kimiidera, Wakayama, Wakayama 641-0012 ,  Japan","@type":"Person"},{"name":"Kuhara, Satoru","affiliation":"4  Graduate School of Genetic Resource Technology, Kyushu University ,  Hakozaki, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"2  Laboratory of Genomic Information ,  Kitasato Institute for Life Sciences, Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan    6  Department of Computational Biology ,  Graduate School of Frontier Sciences, University of Tokyo ,  5-1-5 Kashinoha, Kashiwa, Chiba 277-8561 ,  Japan    8  Genome Core Technology Facility, RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Shimizu, Tohru","affiliation":"9  Department of Bacteriology ,  Graduate School of Medical Science, Kanazawa University ,  13-1 Takaramachi, Kanazawa, Ishikawa 920-8640 ,  Japan","@type":"Person"},{"name":"Akimoto, Shigeru","affiliation":"1  Department of Microbiology ,  Wakayama Medical University ,  811-1 Kimiidera, Wakayama, Wakayama 641-0012 ,  Japan","@type":"Person"}],"description":"Finegoldia magna (formerly Peptostreptococcus magnus), a member of the Gram-positive anaerobic cocci (GPAC), is a commensal bacterium colonizing human skin and mucous membranes. Moreover, it is also recognized as an opportunistic pathogen responsible for various infectious diseases. Here, we report the complete genome sequence of F. magna ATCC 29328. The genome consists of a 1 797 577 bp circular chromosome and an 189 163 bp plasmid (pPEP1). The metabolic maps constructed based on the genome information confirmed that most F. magna strains cannot ferment most sugars, except fructose, and have various aminopeptidase activities. Three homologs of albumin-binding protein, a known virulence factor useful for antiphagocytosis, are encoded on the chromosome, and one albumin-binding protein homolog is encoded on the plasmid. A unique feature of the genome is that F. magna encodes many sortase genes, of which substrates may be involved in bacterial pathogenesis, such as antiphagocytosis and adherence to the host cell. The plasmid pPEP1 encodes seven sortase and seven substrate genes, whereas the chromosome encodes four sortase and 19 substrate genes. These plasmid-encoded sortases may play important roles in the pathogenesis of F. magna by enriching the variety of cell wall anchored surface proteins.","pageStart":"39","pageEnd":"47","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/1/1/m_cover.gif?Expires=1572217011&Signature=Tj84MKYUV3XZF4tYX8Xqeh640qB2RN5-qhBajCKrDlUO0es41ZJIiFPN4gJLNy4Isef9pPPc20ozOYgogs~qMkuKZlkbw5ez0RLSrRkMloYZwqKx0eATSNCOSI8I-KghmpgIQomSuPJ3LOmJIf-ZZUNgdq1OKOXfaGK6SNhG9JAuJikCl662xW9g82J-PHCGYOFLej25Ful5pGwAC8RkEZtu-ffxtI52UFYPd4IiUnyVliAoRi3sUEcaCpaYiYUOPD899wEp-qx-0Jma-t1U9SET6Mt38AiHbl3WR0vfSFmp-DxuiCChn9qWvTokNQRcJiAN1dXH47VTiQR77bspwQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Complete Genome Sequence of  Finegoldia magna , an Anaerobic Opportunistic Pathogen","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/1/1/m_cover.gif?Expires=1572217011&Signature=Tj84MKYUV3XZF4tYX8Xqeh640qB2RN5-qhBajCKrDlUO0es41ZJIiFPN4gJLNy4Isef9pPPc20ozOYgogs~qMkuKZlkbw5ez0RLSrRkMloYZwqKx0eATSNCOSI8I-KghmpgIQomSuPJ3LOmJIf-ZZUNgdq1OKOXfaGK6SNhG9JAuJikCl662xW9g82J-PHCGYOFLej25Ful5pGwAC8RkEZtu-ffxtI52UFYPd4IiUnyVliAoRi3sUEcaCpaYiYUOPD899wEp-qx-0Jma-t1U9SET6Mt38AiHbl3WR0vfSFmp-DxuiCChn9qWvTokNQRcJiAN1dXH47VTiQR77bspwQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complete Genome Sequence of  Finegoldia magna , an Anaerobic Opportunistic Pathogen | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Finegoldia magna (formerly Peptostreptococcus magnus), a member of the Gram-positive anaerobic cocci (GPAC), is a commensal bacterium colonizing human skin and mucous membranes. Moreover, it is also recognized as an opportunistic pathogen responsible for various infectious diseases. Here, we report the complete genome sequence of F. magna ATCC 29328. The genome consists of a 1&#160;797&#160;577 bp circular chromosome and an 189 163 bp plasmid (pPEP1). The metabolic maps constructed based on the genome information confirmed that most F. magna strains cannot ferment most sugars, except fructose, and have various aminopeptidase activities. Three homologs of albumin-binding protein, a known virulence factor useful for antiphagocytosis, are encoded on the chromosome, and one albumin-binding protein homolog is encoded on the plasmid. A unique feature of the genome is that F. magna encodes many sortase genes, of which substrates may be involved in bacterial pathogenesis, such as antiphagocytosis and adherence to the host cell. The plasmid pPEP1 encodes seven sortase and seven substrate genes, whereas the chromosome encodes four sortase and 19 substrate genes. These plasmid-encoded sortases may play important roles in the pathogenesis of F. magna by enriching the variety of cell wall anchored surface proteins." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/1/39/357582" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217011&amp;Signature=Y5ROOCgEJ4UapVhUlvHc~i~TxWVi8h-V4~MZCR0Zafx8d9dNkCu7Ugy9IyI-2DG8lOgu85u79XnyD7lq6205zgS6CGVuaG55OxR09c~pKTfCvO4TH1NLPHBJvkTBAI0uVFf7RCI5Cjt5spPnzy-FXifGmjLsWnmS83-eOSwHdzgeJcE9B7--r65zLrcVpfyYRj4SuVgz0Ifwkt5xQKOst140uxCMqAsL8zfuV93qE6mDVcZ9rjWhQtc234IUdTBCB1VDlARtv2bURyB2WXrMXJuOkeSXrz00UDqVuRn0LTGna3dcohShOJCl6C0KvUasYnNe~O7bCMBA9-wAWsXCgg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2008-02-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Goto, Takatsugu" /><meta name="citation_author_institution" content="Department of Microbiology, Wakayama Medical University, 811-1 Kimiidera, Wakayama, Wakayama 641-0012, Japan" /><meta name="citation_author" content="Yamashita, Atsushi" /><meta name="citation_author_institution" content="Laboratory of Genomic Information, Kitasato Institute for Life Sciences, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Graduate School of Systems Life Sciences, Kyushu University, Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Matsutani, Minenosuke" /><meta name="citation_author_institution" content="Graduate School of Genetic Resource Technology, Kyushu University, Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Todo, Kozo" /><meta name="citation_author_institution" content="Department of Microbiology, Wakayama Medical University, 811-1 Kimiidera, Wakayama, Wakayama 641-0012, Japan" /><meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery, Wakayama Medical University, 811-1 Kimiidera, Wakayama, Wakayama 641-0012, Japan" /><meta name="citation_author" content="Ohshima, Kenshiro" /><meta name="citation_author_institution" content="Laboratory of Genomic Information, Kitasato Institute for Life Sciences, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, 5-1-5 Kashinoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Toh, Hidehiro" /><meta name="citation_author_institution" content="Laboratory of Genomic Information, Kitasato Institute for Life Sciences, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Miyamoto, Kazuaki" /><meta name="citation_author_institution" content="Department of Microbiology, Wakayama Medical University, 811-1 Kimiidera, Wakayama, Wakayama 641-0012, Japan" /><meta name="citation_author" content="Kuhara, Satoru" /><meta name="citation_author_institution" content="Graduate School of Genetic Resource Technology, Kyushu University, Hakozaki, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Laboratory of Genomic Information, Kitasato Institute for Life Sciences, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, 5-1-5 Kashinoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author_institution" content="Genome Core Technology Facility, RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Shimizu, Tohru" /><meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Science, Kanazawa University, 13-1 Takaramachi, Kanazawa, Ishikawa 920-8640, Japan" /><meta name="citation_author" content="Akimoto, Shigeru" /><meta name="citation_author_institution" content="Department of Microbiology, Wakayama Medical University, 811-1 Kimiidera, Wakayama, Wakayama 641-0012, Japan" /><meta name="citation_title" content="Complete Genome Sequence of Finegoldia magna, an Anaerobic Opportunistic Pathogen" /><meta name="citation_firstpage" content="39" /><meta name="citation_lastpage" content="47" /><meta name="citation_doi" content="10.1093/dnares/dsm030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2008/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Gram-positive anaerobic cocci; Clin. Microbiol. Rev;  citation_year=1998;  citation_volume=11;  citation_pages=81-120; " /><meta name="citation_reference" content=" citation_edition=9th; citation_publisher=Arnold, London; Topley &amp; Wilson's Microbiology and Microbial Infections;  citation_year=1998;  citation_pages=783-797; " /><meta name="citation_reference" content=" citation_edition=5th; citation_publisher=Lippincott, Philadelphia; Color Atlas and Textbook of Diagnostic Microbiology;  citation_year=1997;  citation_pages=709-784; " /><meta name="citation_reference" content="citation_title=Peptococcus magnus: a significant human pathogen; Ann. Intern. Med;  citation_year=1980;  citation_volume=93;  citation_pages=244-248; " /><meta name="citation_reference" content="citation_title=Identification of interdomain sequences promoting the intronless evolution of a bacterial protein family; citation_publisher=USA; Proc. Natl. Acad. Sci;  citation_year=1996;  citation_volume=93;  citation_pages=8490-8495; " /><meta name="citation_reference" content="citation_title=Crystal structure and biological implications of a bacterial albumin binding module in complex with human serum albumin; J. Biol. Chem;  citation_year=2004;  citation_volume=279;  citation_pages=42924-42928; " /><meta name="citation_reference" content="citation_title=Whole-body autoradiography reveals that the Peptostreptococcus magnus immunoglobulin-binding domains of protein L preferentially target B lymphocytes in the spleen and lymph nodes in vivo; Cell. Microbiol;  citation_year=2004;  citation_volume=6;  citation_pages=609-623; " /><meta name="citation_reference" content="citation_title=Physical and genetic map of the Finegoldia magna (formerly Peptostreptococcus magnus) ATCC 29328 genome; FEMS Microbiol. Lett;  citation_year=2002;  citation_volume=210;  citation_pages=33-37; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the four rRNA operons in Finegoldia magna ATCC 29328; Syst. Appl. Microbiol;  citation_year=2004;  citation_volume=27;  citation_pages=18-26; " /><meta name="citation_reference" content="citation_title=Bacterial artificial chromosome library of Finegoldia magna ATCC 29328 for genetic mapping and comparative genomics; Microbiol. Immunol;  citation_year=2003;  citation_volume=47;  citation_pages=1005-1016; " /><meta name="citation_reference" content="citation_title=Automated finishing with autofinish; Genome Res;  citation_year=2001;  citation_volume=11;  citation_pages=614-625; " /><meta name="citation_reference" content="citation_title=An automated system for genome analysis to support microbial whole-genome shotgun sequencing; Biosci. Biotechnol. Biochem;  citation_year=2000;  citation_volume=64;  citation_pages=670-673; " /><meta name="citation_reference" content="citation_title=CRITICA: Coding region identification tool invoking comparative analysis; Mol. Biol. Evol;  citation_year=1999;  citation_volume=16;  citation_pages=512-524; " /><meta name="citation_reference" content="citation_title=Modeling and predicting transcriptional units of Escherichia coli genes using hidden Markov models; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=987-993; " /><meta name="citation_reference" content="citation_title=Combining diverse evidence for gene recognition in completely sequenced bacterial genomes; Nucleic Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=2941-2947; " /><meta name="citation_reference" content="citation_title=The COG database: new developments in phylogenetic classification of proteins from complete genomes; Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=22-28; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-964; " /><meta name="citation_reference" content="citation_title=The KEGG databases at GenomeNet; Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=42-46; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Clostridium perfringens, an anaerobic flesh-eater; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=996-1001; " /><meta name="citation_reference" content="citation_title=The genome sequence of Clostridium tetani, the causative agent of tetanus disease; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=1316-1321; " /><meta name="citation_reference" content="citation_title=Genome sequence and comparative analysis of the solvent-producing bacterium Clostridium acetobutylicum; J. Bacteriol;  citation_year=2001;  citation_volume=183;  citation_pages=4823-4838; " /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the oral bacterium Fusobacterium nucleatum strain ATCC 25586; J. Bacteriol;  citation_year=2002;  citation_volume=184;  citation_pages=2005-2018; " /><meta name="citation_reference" content="citation_title=PSORT: a program for detecting sorting signals in proteins and predicting their subcellular localization; Trends. Biochem. Sci;  citation_year=1999;  citation_volume=24;  citation_pages=34-36; " /><meta name="citation_reference" content="citation_title=Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites; Protein Eng;  citation_year=1997;  citation_volume=10;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=A comparative genome analysis identifies distinct sorting pathways in Gram-positive bacteria; Infect. Immun;  citation_year=2004;  citation_volume=72;  citation_pages=2710-2722; " /><meta name="citation_reference" content="citation_title=Organization of the origins of replication of the chromosomes of Mycobacterium smegmatis, Mycobacterium leprae and Mycobacterium tuberculosis and isolation of a functional origin from M. semgmatis; Mol. Microbiol;  citation_year=1996;  citation_volume=20;  citation_pages=283-293; " /><meta name="citation_reference" content="citation_title=The minimal gene complement of Mycoplasma genitalium; Science;  citation_year=1995;  citation_volume=270;  citation_pages=397-403; " /><meta name="citation_reference" content="citation_title=Bacterial DNA strand compositional asymmetry; Trends Microbiol;  citation_year=1999;  citation_volume=7;  citation_pages=305-308; " /><meta name="citation_reference" content="citation_title=Aminopeptidase activities in Peptostreptococcus spp. are statistically correlated to gelatin hydrolysis; Can. J. Microbiol;  citation_year=1998;  citation_volume=44;  citation_pages=303-306; " /><meta name="citation_reference" content="citation_title=Transfer of Peptococcus indolicus, Peptococcus asaccharolyticus, Peptococcus prevotii, and Peptococcus magnus to the Genus Peptostreptococcus and proposal of Peptostreptococcus tetradius sp. nov; Int. J. Syst. Bacteriol;  citation_year=1983;  citation_volume=33;  citation_pages=683-698; " /><meta name="citation_reference" content="citation_title=The Escherichia coli ATP-binding cassette (ABC) proteins; Mol. Microbiol;  citation_year=1998;  citation_volume=28;  citation_pages=5-13; " /><meta name="citation_reference" content="citation_title=Inventory, assembly and analysis of Bacillus subtilis ABC transport systems; J. Mol. Biol;  citation_year=1999;  citation_volume=287;  citation_pages=467-484; " /><meta name="citation_reference" content="citation_title=The ATP binding cassette (ABC) transport systems of Mycobacterium tuberculosis; FEMS Microbiol. Rev;  citation_year=2000;  citation_volume=24;  citation_pages=449-467; " /><meta name="citation_reference" content="citation_title=Analysis of the Streptomyces coelicolor sigE gene reveals the existence of a subfamily of eubacterial σ factors involved in the regulation of extracytoplasmic functions; citation_publisher=USA; Proc. Natl. Acad. Sci;  citation_year=1994;  citation_volume=91;  citation_pages=7573-7577; " /><meta name="citation_reference" content="citation_title=The extracytoplasmic function (ECF) sigma factors; Adv. Microb. Physiol;  citation_year=2002;  citation_volume=46;  citation_pages=47-110; " /><meta name="citation_reference" content="citation_title=Solution structure of the albumin-binding GA module: a versatile bacterial protein domain; J. Mol. Biol;  citation_year=1997;  citation_volume=266;  citation_pages=859-865; " /><meta name="citation_reference" content="citation_title=Structure, specificity, and mode of interaction for bacterial albumin-binding modules; J. Biol. Chem;  citation_year=2002;  citation_volume=277;  citation_pages=8114-8120; " /><meta name="citation_reference" content="citation_title=Structure of the IgG-binding regions of streptococcal protein G; EMBO J;  citation_year=1986;  citation_volume=5;  citation_pages=1567-1575; " /><meta name="citation_reference" content="citation_title=The functional units of a peptostreptococcal protein L; Mol. Microbiol;  citation_year=1994;  citation_volume=12;  citation_pages=911-920; " /><meta name="citation_reference" content="citation_title=Molecular characterization and expression of a gene encoding a Staphylococcus aureus collagen adhesion; J. Biol. Chem;  citation_year=1992;  citation_volume=267;  citation_pages=4766-4772; " /><meta name="citation_reference" content="citation_title=Characterization of a cell surface protein of Clostridium difficile with adhesive properties; Infect. Immun;  citation_year=2001;  citation_volume=69;  citation_pages=2144-2153; " /><meta name="citation_reference" content="citation_title=Sortase, a universal target for therapeutic agents against Gram-positive bacteria?; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=5013-5015; " /><meta name="citation_reference" content="citation_title=Sortase-catalyzed anchoring of surface proteins to the cell wall of Staphylococcus aureus; Mol. Microbiol;  citation_year=2001;  citation_volume=40;  citation_pages=1049-1057; " /><meta name="citation_reference" content="citation_title=Surface proteins of Gram-positive bacteria and mechanisms of their targeting to the cell wall envelope; Microbiol. Mol. Biol. Rev;  citation_year=1999;  citation_volume=63;  citation_pages=174-229; " /><meta name="citation_reference" content="citation_title=Sortase: the surface protein anchoring transpeptidase and the LPXTG motif; Trends. Microbiol;  citation_year=2000;  citation_volume=8;  citation_pages=148-151; " /><meta name="citation_reference" content="citation_title=Crystal structures of Staphylococcus aureus sortase A and its substrate complex; J. Biol. Chem;  citation_year=2004;  citation_volume=279;  citation_pages=31383-31389; " /><meta name="citation_reference" content="citation_title=An iron-regulated sortase anchors a class of surface protein during Staphylococcus aureus pathogenesis; citation_publisher=USA; Proc. Natl. Acad. Sci;  citation_year=2002;  citation_volume=99;  citation_pages=2293-2298; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/1/39/1023739/dsm030.pdf" /><meta name="description" content="Abstract.  Finegoldia magna (formerly Peptostreptococcus magnus), a member of the Gram-positive anaerobic cocci (GPAC), is a commensal bacterium colonizing huma" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/1/39/357582" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/1/39/357582" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsm030';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./357582" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFlH88bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdvdG8sIFRha2F0c3VndSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBXYWtheWFtYSBNZWRpY2FsIFVuaXZlcnNpdHksIDgxMS0xIEtpbWlpZGVyYSwgV2FrYXlhbWEsIFdha2F5YW1hIDY0MS0wMDEyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFzaGl0YSwgQXRzdXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgR2Vub21pYyBJbmZvcm1hdGlvbiwgS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2VzLCBLaXRhc2F0byBVbml2ZXJzaXR5LCAxLTE1LTEgS2l0YXNhdG8sIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOC04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpcmFrYXdhLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgU3lzdGVtcyBMaWZlIFNjaWVuY2VzLCBLeXVzaHUgVW5pdmVyc2l0eSwgSGFrb3pha2ksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWF0c3V0YW5pLCBNaW5lbm9zdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEdlbmV0aWMgUmVzb3VyY2UgVGVjaG5vbG9neSwgS3l1c2h1IFVuaXZlcnNpdHksIEhha296YWtpLCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvZG8sIEtvem8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgV2FrYXlhbWEgTWVkaWNhbCBVbml2ZXJzaXR5LCA4MTEtMSBLaW1paWRlcmEsIFdha2F5YW1hLCBXYWtheWFtYSA2NDEtMDAxMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE9yYWwgYW5kIE1heGlsbG9mYWNpYWwgU3VyZ2VyeSwgV2FrYXlhbWEgTWVkaWNhbCBVbml2ZXJzaXR5LCA4MTEtMSBLaW1paWRlcmEsIFdha2F5YW1hLCBXYWtheWFtYSA2NDEtMDAxMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaHNoaW1hLCBLZW5zaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgR2Vub21pYyBJbmZvcm1hdGlvbiwgS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2VzLCBLaXRhc2F0byBVbml2ZXJzaXR5LCAxLTE1LTEgS2l0YXNhdG8sIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOC04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01IEthc2hpbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9oLCBIaWRlaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgR2Vub21pYyBJbmZvcm1hdGlvbiwgS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2VzLCBLaXRhc2F0byBVbml2ZXJzaXR5LCAxLTE1LTEgS2l0YXNhdG8sIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOC04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvbXB1dGF0aW9uYWwgYW5kIEV4cGVyaW1lbnRhbCBTeXN0ZW1zIEJpb2xvZ3kgR3JvdXAsIFJJS0VOIEdlbm9taWMgU2NpZW5jZXMgQ2VudGVyLCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXlhbW90bywgS2F6dWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBXYWtheWFtYSBNZWRpY2FsIFVuaXZlcnNpdHksIDgxMS0xIEtpbWlpZGVyYSwgV2FrYXlhbWEsIFdha2F5YW1hIDY0MS0wMDEyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1aGFyYSwgU2F0b3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEdlbmV0aWMgUmVzb3VyY2UgVGVjaG5vbG9neSwgS3l1c2h1IFVuaXZlcnNpdHksIEhha296YWtpLCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhdHRvcmksIE1hc2FoaXJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBHZW5vbWljIEluZm9ybWF0aW9uLCBLaXRhc2F0byBJbnN0aXR1dGUgZm9yIExpZmUgU2NpZW5jZXMsIEtpdGFzYXRvIFVuaXZlcnNpdHksIDEtMTUtMSBLaXRhc2F0bywgU2FnYW1paGFyYSwgS2FuYWdhd2EgMjI4LTg1NTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGlub2hhLCBLYXNoaXdhLCBDaGliYSAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHZW5vbWUgQ29yZSBUZWNobm9sb2d5IEZhY2lsaXR5LCBSSUtFTiBHZW5vbWljIFNjaWVuY2VzIENlbnRlciwgMS03LTIyIFN1ZWhpcm8tY2hvLCBUc3VydW1pLWt1LCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbWl6dSwgVG9ocnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJhY3RlcmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljYWwgU2NpZW5jZSwgS2FuYXphd2EgVW5pdmVyc2l0eSwgMTMtMSBUYWthcmFtYWNoaSwgS2FuYXphd2EsIElzaGlrYXdhIDkyMC04NjQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFraW1vdG8sIFNoaWdlcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgV2FrYXlhbWEgTWVkaWNhbCBVbml2ZXJzaXR5LCA4MTEtMSBLaW1paWRlcmEsIFdha2F5YW1hLCBXYWtheWFtYSA2NDEtMDAxMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvbXBsZXRlIEdlbm9tZSBTZXF1ZW5jZSBvZiBGaW5lZ29sZGlhIG1hZ25hLCBhbiBBbmFlcm9iaWMgT3Bwb3J0dW5pc3RpYyBQYXRob2dlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA4LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JhbS1wb3NpdGl2ZSBhbmFlcm9iaWMgY29jY2k7IENsaW4uIE1pY3JvYmlvbC4gUmV2OyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODEtMTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj05dGg7IGNpdGF0aW9uX3B1Ymxpc2hlcj1Bcm5vbGQsIExvbmRvbjsgVG9wbGV5ICZhbXA7IFdpbHNvbidzIE1pY3JvYmlvbG9neSBhbmQgTWljcm9iaWFsIEluZmVjdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl9wYWdlcz03ODMtNzk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj01dGg7IGNpdGF0aW9uX3B1Ymxpc2hlcj1MaXBwaW5jb3R0LCBQaGlsYWRlbHBoaWE7IENvbG9yIEF0bGFzIGFuZCBUZXh0Ym9vayBvZiBEaWFnbm9zdGljIE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3BhZ2VzPTcwOS03ODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcHRvY29jY3VzIG1hZ251czogYSBzaWduaWZpY2FudCBodW1hbiBwYXRob2dlbjsgQW5uLiBJbnRlcm4uIE1lZDsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTI0NC0yNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGludGVyZG9tYWluIHNlcXVlbmNlcyBwcm9tb3RpbmcgdGhlIGludHJvbmxlc3MgZXZvbHV0aW9uIG9mIGEgYmFjdGVyaWFsIHByb3RlaW4gZmFtaWx5OyBjaXRhdGlvbl9wdWJsaXNoZXI9VVNBOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz04NDkwLTg0OTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyeXN0YWwgc3RydWN0dXJlIGFuZCBiaW9sb2dpY2FsIGltcGxpY2F0aW9ucyBvZiBhIGJhY3RlcmlhbCBhbGJ1bWluIGJpbmRpbmcgbW9kdWxlIGluIGNvbXBsZXggd2l0aCBodW1hbiBzZXJ1bSBhbGJ1bWluOyBKLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTQyOTI0LTQyOTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1ib2R5IGF1dG9yYWRpb2dyYXBoeSByZXZlYWxzIHRoYXQgdGhlIFBlcHRvc3RyZXB0b2NvY2N1cyBtYWdudXMgaW1tdW5vZ2xvYnVsaW4tYmluZGluZyBkb21haW5zIG9mIHByb3RlaW4gTCBwcmVmZXJlbnRpYWxseSB0YXJnZXQgQiBseW1waG9jeXRlcyBpbiB0aGUgc3BsZWVuIGFuZCBseW1waCBub2RlcyBpbiB2aXZvOyBDZWxsLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTYwOS02MjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeXNpY2FsIGFuZCBnZW5ldGljIG1hcCBvZiB0aGUgRmluZWdvbGRpYSBtYWduYSAoZm9ybWVybHkgUGVwdG9zdHJlcHRvY29jY3VzIG1hZ251cykgQVRDQyAyOTMyOCBnZW5vbWU7IEZFTVMgTWljcm9iaW9sLiBMZXR0OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIxMDsgIGNpdGF0aW9uX3BhZ2VzPTMzLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgZm91ciByUk5BIG9wZXJvbnMgaW4gRmluZWdvbGRpYSBtYWduYSBBVENDIDI5MzI4OyBTeXN0LiBBcHBsLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xOC0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZSBsaWJyYXJ5IG9mIEZpbmVnb2xkaWEgbWFnbmEgQVRDQyAyOTMyOCBmb3IgZ2VuZXRpYyBtYXBwaW5nIGFuZCBjb21wYXJhdGl2ZSBnZW5vbWljczsgTWljcm9iaW9sLiBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MTAwNS0xMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgZmluaXNoaW5nIHdpdGggYXV0b2ZpbmlzaDsgR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTYxNC02MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGF1dG9tYXRlZCBzeXN0ZW0gZm9yIGdlbm9tZSBhbmFseXNpcyB0byBzdXBwb3J0IG1pY3JvYmlhbCB3aG9sZS1nZW5vbWUgc2hvdGd1biBzZXF1ZW5jaW5nOyBCaW9zY2kuIEJpb3RlY2hub2wuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz02NzAtNjczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklUSUNBOiBDb2RpbmcgcmVnaW9uIGlkZW50aWZpY2F0aW9uIHRvb2wgaW52b2tpbmcgY29tcGFyYXRpdmUgYW5hbHlzaXM7IE1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTUxMi01MjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGVsaW5nIGFuZCBwcmVkaWN0aW5nIHRyYW5zY3JpcHRpb25hbCB1bml0cyBvZiBFc2NoZXJpY2hpYSBjb2xpIGdlbmVzIHVzaW5nIGhpZGRlbiBNYXJrb3YgbW9kZWxzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTk4Ny05OTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbWJpbmluZyBkaXZlcnNlIGV2aWRlbmNlIGZvciBnZW5lIHJlY29nbml0aW9uIGluIGNvbXBsZXRlbHkgc2VxdWVuY2VkIGJhY3RlcmlhbCBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTI5NDEtMjk0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENPRyBkYXRhYmFzZTogbmV3IGRldmVsb3BtZW50cyBpbiBwaHlsb2dlbmV0aWMgY2xhc3NpZmljYXRpb24gb2YgcHJvdGVpbnMgZnJvbSBjb21wbGV0ZSBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTIyLTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtOTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgS0VHRyBkYXRhYmFzZXMgYXQgR2Vub21lTmV0OyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTQyLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgQ2xvc3RyaWRpdW0gcGVyZnJpbmdlbnMsIGFuIGFuYWVyb2JpYyBmbGVzaC1lYXRlcjsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz05OTYtMTAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBDbG9zdHJpZGl1bSB0ZXRhbmksIHRoZSBjYXVzYXRpdmUgYWdlbnQgb2YgdGV0YW51cyBkaXNlYXNlOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0xMzE2LTEzMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIHNvbHZlbnQtcHJvZHVjaW5nIGJhY3Rlcml1bSBDbG9zdHJpZGl1bSBhY2V0b2J1dHlsaWN1bTsgSi4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE4MzsgIGNpdGF0aW9uX3BhZ2VzPTQ4MjMtNDgzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFuZCBhbmFseXNpcyBvZiB0aGUgb3JhbCBiYWN0ZXJpdW0gRnVzb2JhY3Rlcml1bSBudWNsZWF0dW0gc3RyYWluIEFUQ0MgMjU1ODY7IEouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODQ7ICBjaXRhdGlvbl9wYWdlcz0yMDA1LTIwMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBTT1JUOiBhIHByb2dyYW0gZm9yIGRldGVjdGluZyBzb3J0aW5nIHNpZ25hbHMgaW4gcHJvdGVpbnMgYW5kIHByZWRpY3RpbmcgdGhlaXIgc3ViY2VsbHVsYXIgbG9jYWxpemF0aW9uOyBUcmVuZHMuIEJpb2NoZW0uIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM0LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBwcm9rYXJ5b3RpYyBhbmQgZXVrYXJ5b3RpYyBzaWduYWwgcGVwdGlkZXMgYW5kIHByZWRpY3Rpb24gb2YgdGhlaXIgY2xlYXZhZ2Ugc2l0ZXM7IFByb3RlaW4gRW5nOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmF0aXZlIGdlbm9tZSBhbmFseXNpcyBpZGVudGlmaWVzIGRpc3RpbmN0IHNvcnRpbmcgcGF0aHdheXMgaW4gR3JhbS1wb3NpdGl2ZSBiYWN0ZXJpYTsgSW5mZWN0LiBJbW11bjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTI3MTAtMjcyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JnYW5pemF0aW9uIG9mIHRoZSBvcmlnaW5zIG9mIHJlcGxpY2F0aW9uIG9mIHRoZSBjaHJvbW9zb21lcyBvZiBNeWNvYmFjdGVyaXVtIHNtZWdtYXRpcywgTXljb2JhY3Rlcml1bSBsZXByYWUgYW5kIE15Y29iYWN0ZXJpdW0gdHViZXJjdWxvc2lzIGFuZCBpc29sYXRpb24gb2YgYSBmdW5jdGlvbmFsIG9yaWdpbiBmcm9tIE0uIHNlbWdtYXRpczsgTW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yODMtMjkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWluaW1hbCBnZW5lIGNvbXBsZW1lbnQgb2YgTXljb3BsYXNtYSBnZW5pdGFsaXVtOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTM5Ny00MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCBETkEgc3RyYW5kIGNvbXBvc2l0aW9uYWwgYXN5bW1ldHJ5OyBUcmVuZHMgTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zMDUtMzA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbWlub3BlcHRpZGFzZSBhY3Rpdml0aWVzIGluIFBlcHRvc3RyZXB0b2NvY2N1cyBzcHAuIGFyZSBzdGF0aXN0aWNhbGx5IGNvcnJlbGF0ZWQgdG8gZ2VsYXRpbiBoeWRyb2x5c2lzOyBDYW4uIEouIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTMwMy0zMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZmVyIG9mIFBlcHRvY29jY3VzIGluZG9saWN1cywgUGVwdG9jb2NjdXMgYXNhY2NoYXJvbHl0aWN1cywgUGVwdG9jb2NjdXMgcHJldm90aWksIGFuZCBQZXB0b2NvY2N1cyBtYWdudXMgdG8gdGhlIEdlbnVzIFBlcHRvc3RyZXB0b2NvY2N1cyBhbmQgcHJvcG9zYWwgb2YgUGVwdG9zdHJlcHRvY29jY3VzIHRldHJhZGl1cyBzcC4gbm92OyBJbnQuIEouIFN5c3QuIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTY4My02OTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBFc2NoZXJpY2hpYSBjb2xpIEFUUC1iaW5kaW5nIGNhc3NldHRlIChBQkMpIHByb3RlaW5zOyBNb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTUtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludmVudG9yeSwgYXNzZW1ibHkgYW5kIGFuYWx5c2lzIG9mIEJhY2lsbHVzIHN1YnRpbGlzIEFCQyB0cmFuc3BvcnQgc3lzdGVtczsgSi4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI4NzsgIGNpdGF0aW9uX3BhZ2VzPTQ2Ny00ODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBVFAgYmluZGluZyBjYXNzZXR0ZSAoQUJDKSB0cmFuc3BvcnQgc3lzdGVtcyBvZiBNeWNvYmFjdGVyaXVtIHR1YmVyY3Vsb3NpczsgRkVNUyBNaWNyb2Jpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OS00Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBTdHJlcHRvbXljZXMgY29lbGljb2xvciBzaWdFIGdlbmUgcmV2ZWFscyB0aGUgZXhpc3RlbmNlIG9mIGEgc3ViZmFtaWx5IG9mIGV1YmFjdGVyaWFsIM+DIGZhY3RvcnMgaW52b2x2ZWQgaW4gdGhlIHJlZ3VsYXRpb24gb2YgZXh0cmFjeXRvcGxhc21pYyBmdW5jdGlvbnM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1VU0E7IFByb2MuIE5hdGwuIEFjYWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTc1NzMtNzU3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV4dHJhY3l0b3BsYXNtaWMgZnVuY3Rpb24gKEVDRikgc2lnbWEgZmFjdG9yczsgQWR2LiBNaWNyb2IuIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz00Ny0xMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHV0aW9uIHN0cnVjdHVyZSBvZiB0aGUgYWxidW1pbi1iaW5kaW5nIEdBIG1vZHVsZTogYSB2ZXJzYXRpbGUgYmFjdGVyaWFsIHByb3RlaW4gZG9tYWluOyBKLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9ODU5LTg2NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlLCBzcGVjaWZpY2l0eSwgYW5kIG1vZGUgb2YgaW50ZXJhY3Rpb24gZm9yIGJhY3RlcmlhbCBhbGJ1bWluLWJpbmRpbmcgbW9kdWxlczsgSi4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzc7ICBjaXRhdGlvbl9wYWdlcz04MTE0LTgxMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZSBvZiB0aGUgSWdHLWJpbmRpbmcgcmVnaW9ucyBvZiBzdHJlcHRvY29jY2FsIHByb3RlaW4gRzsgRU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNTY3LTE1NzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmdW5jdGlvbmFsIHVuaXRzIG9mIGEgcGVwdG9zdHJlcHRvY29jY2FsIHByb3RlaW4gTDsgTW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz05MTEtOTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2hhcmFjdGVyaXphdGlvbiBhbmQgZXhwcmVzc2lvbiBvZiBhIGdlbmUgZW5jb2RpbmcgYSBTdGFwaHlsb2NvY2N1cyBhdXJldXMgY29sbGFnZW4gYWRoZXNpb247IEouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjY3OyAgY2l0YXRpb25fcGFnZXM9NDc2Ni00NzcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIGEgY2VsbCBzdXJmYWNlIHByb3RlaW4gb2YgQ2xvc3RyaWRpdW0gZGlmZmljaWxlIHdpdGggYWRoZXNpdmUgcHJvcGVydGllczsgSW5mZWN0LiBJbW11bjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTIxNDQtMjE1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29ydGFzZSwgYSB1bml2ZXJzYWwgdGFyZ2V0IGZvciB0aGVyYXBldXRpYyBhZ2VudHMgYWdhaW5zdCBHcmFtLXBvc2l0aXZlIGJhY3RlcmlhPzsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz01MDEzLTUwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvcnRhc2UtY2F0YWx5emVkIGFuY2hvcmluZyBvZiBzdXJmYWNlIHByb3RlaW5zIHRvIHRoZSBjZWxsIHdhbGwgb2YgU3RhcGh5bG9jb2NjdXMgYXVyZXVzOyBNb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTEwNDktMTA1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VyZmFjZSBwcm90ZWlucyBvZiBHcmFtLXBvc2l0aXZlIGJhY3RlcmlhIGFuZCBtZWNoYW5pc21zIG9mIHRoZWlyIHRhcmdldGluZyB0byB0aGUgY2VsbCB3YWxsIGVudmVsb3BlOyBNaWNyb2Jpb2wuIE1vbC4gQmlvbC4gUmV2OyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MTc0LTIyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29ydGFzZTogdGhlIHN1cmZhY2UgcHJvdGVpbiBhbmNob3JpbmcgdHJhbnNwZXB0aWRhc2UgYW5kIHRoZSBMUFhURyBtb3RpZjsgVHJlbmRzLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE0OC0xNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyeXN0YWwgc3RydWN0dXJlcyBvZiBTdGFwaHlsb2NvY2N1cyBhdXJldXMgc29ydGFzZSBBIGFuZCBpdHMgc3Vic3RyYXRlIGNvbXBsZXg7IEouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9MzEzODMtMzEzODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGlyb24tcmVndWxhdGVkIHNvcnRhc2UgYW5jaG9ycyBhIGNsYXNzIG9mIHN1cmZhY2UgcHJvdGVpbiBkdXJpbmcgU3RhcGh5bG9jb2NjdXMgYXVyZXVzIHBhdGhvZ2VuZXNpczsgY2l0YXRpb25fcHVibGlzaGVyPVVTQTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MjI5My0yMjk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvMS8zOS8xMDIzNzM5L2RzbTAzMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRmluZWdvbGRpYSBtYWduYSAoZm9ybWVybHkgUGVwdG9zdHJlcHRvY29jY3VzIG1hZ251cyksIGEgbWVtYmVyIG9mIHRoZSBHcmFtLXBvc2l0aXZlIGFuYWVyb2JpYyBjb2NjaSAoR1BBQyksIGlzIGEgY29tbWVuc2FsIGJhY3Rlcml1bSBjb2xvbml6aW5nIGh1bWEiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1NzU4MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGTDqndocC+syYtf9s8EjOMR8nbp+A==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f1%2f39%2f357582" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f1%2f39%2f357582" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/1/1/m_cover.gif?Expires=1509238611&amp;Signature=Pfv5ZydFeTzVot-9MTG5I2JIypcQua4WeW-ZaPEKbPC8ZvaVfRLx3VO2zHECPSqdHtQQp~K-5GThFalfV0ux1MqAkZpSy0xN2-NFRIJfmoxfBOdZpbvR6cYwKV13GSBvQ9~C~peE9x7amfV8PBXqV8vegf2V~NOD4hFqIrN4Zn-8kdgU2hZz17Eg8jQTZbI7QKf9Dbq37kaIMguFvnTc-pc1DFyQIdedkdywbmqqFFY3ygKCgy7W7OYOqJ5WBF2zPqJ5xuoL2YycaCxqCh9JUFJHtfOvXnqnsz4AqnSMRLGPfq0egE0fGCWJxgYTD-S7Y~GhjlnNxxqqAcu2XrptPA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/1">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5139876">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139876">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139878">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139878">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139881">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139881">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139886">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139910">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139910">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139913">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139913">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139915">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139915">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5139917">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5139917">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/1/25/357714">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/1/49/357306">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Complete Genome Sequence of <em>Finegoldia magna</em>, an Anaerobic Opportunistic Pathogen
<i class='icon-availability_free' title='Free' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Takatsugu Goto</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takatsugu Goto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Wakayama Medical University</div>, <div class="addr-line">811-1 Kimiidera, Wakayama, Wakayama 641-0012</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 73-441-0640. Fax. <div class="fax">+81 73-448-1026</div>. E-mail: <a href="mailto:t-goto@wakayama-med.ac.jp" target="_blank">t-goto@wakayama-med.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Takatsugu+Goto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Goto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Goto author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Atsushi Yamashita</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Genomic Information</div>, <div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Atsushi+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yamashita author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Graduate School of Systems Life Sciences, Kyushu University</div>, <div class="addr-line">Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hirakawa author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Minenosuke Matsutani</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Minenosuke Matsutani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Graduate School of Genetic Resource Technology, Kyushu University</div>, <div class="addr-line">Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Minenosuke+Matsutani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsutani M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Matsutani author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Kozo Todo</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kozo Todo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Wakayama Medical University</div>, <div class="addr-line">811-1 Kimiidera, Wakayama, Wakayama 641-0012</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Oral and Maxillofacial Surgery</div>, <div class="insititution">Wakayama Medical University</div>, <div class="addr-line">811-1 Kimiidera, Wakayama, Wakayama 641-0012</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kozo+Todo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Todo K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Todo author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Kenshiro Ohshima</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Ohshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Genomic Information</div>, <div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Computational Biology</div>, <div class="insititution">Graduate School of Frontier Sciences, University of Tokyo</div>, <div class="addr-line">5-1-5 Kashinoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kenshiro+Ohshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Ohshima author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hidehiro Toh</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hidehiro Toh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Genomic Information</div>, <div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Computational and Experimental Systems Biology Group</div>, <div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hidehiro+Toh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toh H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Toh author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Kazuaki Miyamoto</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuaki Miyamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Wakayama Medical University</div>, <div class="addr-line">811-1 Kimiidera, Wakayama, Wakayama 641-0012</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Kazuaki+Miyamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyamoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Miyamoto author:K.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Satoru Kuhara</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Kuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Graduate School of Genetic Resource Technology, Kyushu University</div>, <div class="addr-line">Hakozaki, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Satoru+Kuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuhara S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kuhara author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Genomic Information</div>, <div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Computational Biology</div>, <div class="insititution">Graduate School of Frontier Sciences, University of Tokyo</div>, <div class="addr-line">5-1-5 Kashinoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">8</div><div class="insititution">Genome Core Technology Facility, RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hattori author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Tohru Shimizu</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tohru Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="addr-line">Department of Bacteriology</div>, <div class="insititution">Graduate School of Medical Science, Kanazawa University</div>, <div class="addr-line">13-1 Takaramachi, Kanazawa, Ishikawa 920-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Tohru+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Shimizu author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Shigeru Akimoto</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeru Akimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Wakayama Medical University</div>, <div class="addr-line">811-1 Kimiidera, Wakayama, Wakayama 641-0012</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shigeru+Akimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Akimoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Akimoto author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 1, 1 February 2008, Pages 39–47, <a href='https://doi.org/10.1093/dnares/dsm030'>https://doi.org/10.1093/dnares/dsm030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 February 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 October 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 December 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="357582" href="/dnaresearch/article-pdf/15/1/39/1023739/dsm030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takatsugu Goto, Atsushi Yamashita, Hideki Hirakawa, Minenosuke Matsutani, Kozo Todo, Kenshiro Ohshima, Hidehiro Toh, Kazuaki Miyamoto, Satoru Kuhara, Masahira Hattori, Tohru Shimizu, Shigeru Akimoto;  Complete Genome Sequence of <em>Finegoldia magna</em>, an Anaerobic Opportunistic Pathogen, <em>DNA Research</em>, Volume 15, Issue 1, 1 February 2008, Pages 39–47, <a href='https://doi.org/10.1093/dnares/dsm030'>https://doi.org/10.1093/dnares/dsm030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/357582?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/357582?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/357582?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/357582?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/357582?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/357582?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/357582?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Complete%20Genome%20Sequence%20of%20%26lt%3Bem%26gt%3BFinegoldia%20magna%26lt%3B%2Fem%26gt%3B%2C%20an%20Anaerobic%20Opportunistic%20Pathogen&amp;publicationDate=2008-02-07&amp;volumeNum=15&amp;issueNum=1&amp;author=Goto%2C%20Takatsugu%3B%20Yamashita%2C%20Atsushi&amp;startPage=39&amp;endPage=47&amp;contentId=10.1093%2Fdnares%2Fdsm030&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f1%2f39%2f357582" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5139876 id="5139876" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Finegoldia magna</em> (formerly <em>Peptostreptococcus magnus</em>), a member of the Gram-positive anaerobic cocci (GPAC), is a commensal bacterium colonizing human skin and mucous membranes. Moreover, it is also recognized as an opportunistic pathogen responsible for various infectious diseases. Here, we report the complete genome sequence of <em>F. magna</em> ATCC 29328. The genome consists of a 1 797 577 bp circular chromosome and an 189 163 bp plasmid (pPEP1). The metabolic maps constructed based on the genome information confirmed that most <em>F. magna</em> strains cannot ferment most sugars, except fructose, and have various aminopeptidase activities. Three homologs of albumin-binding protein, a known virulence factor useful for antiphagocytosis, are encoded on the chromosome, and one albumin-binding protein homolog is encoded on the plasmid. A unique feature of the genome is that <em>F. magna</em> encodes many sortase genes, of which substrates may be involved in bacterial pathogenesis, such as antiphagocytosis and adherence to the host cell. The plasmid pPEP1 encodes seven sortase and seven substrate genes, whereas the chromosome encodes four sortase and 19 substrate genes. These plasmid-encoded sortases may play important roles in the pathogenesis of <em>F. magna</em> by enriching the variety of cell wall anchored surface proteins.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome sequence&quot;">whole genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Gram-positive anaerobic cocci&quot;">Gram-positive anaerobic cocci</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Peptostreptococcus magnus&quot;"><em>Peptostreptococcus magnus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;albumin-binding protein&quot;">albumin-binding protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="sortase">sortase</a></div>                            <h2 scrollto-destination=5139878 id="5139878" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Gram-positive anaerobic cocci (GPAC) are a major part of the normal human flora colonizing skin and mucous membranes of the mouth and gastrointestinal tracts. GPAC account for 20–40% of all anaerobes recovered from human clinical specimens.<sup><a reveal-id="DSM030C1  DSM030C3" data-open="DSM030C1  DSM030C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> In GPAC, <em>Finegoldia magna</em> (formerly <em>Peptostreptococcus magnus</em>) has the highest pathogenicity and is isolated most frequently from various infection sites, including soft tissue, bone and joint, and diabetic foot infections.<sup><a reveal-id="DSM030C1" data-open="DSM030C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Approximately 70% of <em>F. magna</em> strains recovered from human clinical materials coexist with other bacterial species such as group D streptococci, <em>Staphylococcus</em>, <em>Bacteroides</em>, and <em>Fusobacterium</em>.<sup><a reveal-id="DSM030C4" data-open="DSM030C4" class="link link-ref link-reveal xref-bibr">4</a></sup><em>Finegoldia magna</em> has well-known cell surface proteins such as albumin-binding protein and protein L, which may contribute to antiphagocytosis by binding the cell surface to human serum albumin and immunoglobulin light chain, respectively.<sup><a reveal-id="DSM030C5  DSM030C7" data-open="DSM030C5  DSM030C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> Despite this clinical importance of <em>F. magna</em>, other virulence factors and overall genomic information of <em>F. magna</em> have not been extensively studied. Therefore, we have so far constructed the physical map<sup><a reveal-id="DSM030C8" data-open="DSM030C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and the genetic map<sup><a reveal-id="DSM030C9" data-open="DSM030C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM030C10" data-open="DSM030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> of <em>F. magna</em> strain ATCC 29328, originally isolated from an abdominal wound.</p><p>Here, we report the complete genome sequence of <em>F. magna</em> ATCC 29328 to characterize its genomic structure in detail and its nature as an opportunistic pathogen. This is the first complete genome sequence among GPAC, and we believe our data will be of great use for genetic studies of other GPAC as well as <em>F. magna</em>.</p>                            <h2 scrollto-destination=5139881 id="5139881" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5139882 id="5139882" class="section-title" data-legacy-id=s2a>2.1. Genome sequencing</h3>
<p>The nucleotide sequence of the <em>Finegoldia magna</em> ATCC 29328 genome was determined by a whole-genome shotgun strategy. Briefly, <em>F. magna</em> ATCC 29328 was obtained from the American Type Culture Collection and was grown anaerobically at 37°C in liquid using Gifu Anaerobic Medium (GAM) (Nissui, Japan) for 39 h. The genomic DNA was isolated according to the conventional method using 0.4 mg/mL achromopeptidase (Wako, Japan), 1.3% sodium dodecyl sulfate and 2 mg Proteinase K (Roche, Germany). The isolated DNA was fragmented using the HydroShear process (GeneMachines Inc., USA). The fragmented DNA was fractionated by agarose gel electrophoresis to isolate the short (1–2 kb) and the long (4–5 kb) fractions of fragments. The 1–2 kb and 4-5 kb DNA were ligated into a <em>Hinc</em>II-digested pTS1 plasmid vector (Nippon genetech, Japan). The short-insert and the long-insert plasmids were introduced by electroporation into <em>Escherichia coli</em> DH5α and SURE2 (Stratagene, USA), respectively. A total of 38 776 reads (providing a 10-fold coverage) from both ends of inserts was sequenced using dye-terminator chemistry with MegaBACE1000, MegaBACE4000 (GE Healthcare, USA) and ABI 3037xl automated sequencers (ABI, USA). Sequence assembly was carried out using the Phred/Phrap/Consed package.<sup><a reveal-id="DSM030C11" data-open="DSM030C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Gaps were closed by PCR direct sequencing with oligonucleotide primers designed to anneal to each end of the neighboring contigs and by primer walking with bridge clones. Sequences of four rRNA operons were independently determined by genomic PCR and with BAC library.<sup><a reveal-id="DSM030C9" data-open="DSM030C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM030C10" data-open="DSM030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Sequencing of one long repeated region on the chromosome was performed by nested-deletion methods using Kilo-sequence deletion kit (Takara, Japan). Finally, sequences of one complete circular chromosome and one complete plasmid were obtained. Overall accuracy of the finished sequence was estimated to have an error rate of &lt;1 per 10 000 bases (Phrap score ≥40).</p>                            <h3 scrollto-destination=5139884 id="5139884" class="section-title" data-legacy-id=s2b>2.2. Bioinformatics</h3>
<p>Protein coding regions (ORFs &gt;150 bp) were identified using GenomeGambler 1.51,<sup><a reveal-id="DSM030C12" data-open="DSM030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> CRITICA,<sup><a reveal-id="DSM030C13" data-open="DSM030C13" class="link link-ref link-reveal xref-bibr">13</a></sup> GeneHacker,<sup><a reveal-id="DSM030C14" data-open="DSM030C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and Glimmer 2.0<sup>15</sup> programs. In the initial step of ORF prediction, 205 ORFs showing high similarity to the housekeeping proteins in the Clusters of Orthologous Groups of Proteins (COGs) database<sup><a reveal-id="DSM030C16" data-open="DSM030C16" class="link link-ref link-reveal xref-bibr">16</a></sup> were selected and used to create the training data set for CRITICA, GeneHacker, and Glimmer programs. Individual predicted ORFs were reviewed manually for the presence of start codons (ATG, TTG, and GTG) and ribosome-binding sequences. Protein functional annotation was based on homology searches against NCBI's non-redundant protein database by BLASTP.<sup><a reveal-id="DSM030C17" data-open="DSM030C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Transfer RNA genes were predicted by tRNAscan-SE.<sup><a reveal-id="DSM030C18" data-open="DSM030C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Functional classification of ORFs was made by homology search against COGs using BLASTP. The metabolic pathway was constructed based on the Kyoto Encyclopedia of Genes and Genomes (KEGG) database<sup><a reveal-id="DSM030C19" data-open="DSM030C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and was complemented manually according to pathways reported in several anaerobes.<sup><a reveal-id="DSM030C20  DSM030C23" data-open="DSM030C20  DSM030C23" class="link link-ref link-reveal xref-bibr">20–23</a></sup> Putative localization of predicted proteins was evaluated by the combination of PSORT<sup><a reveal-id="DSM030C24" data-open="DSM030C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and SignalP program.<sup><a reveal-id="DSM030C25" data-open="DSM030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The characteristic motifs in all protein-coding sequence of ATCC 29328 genome were detected by the Pfam analyses (<a class="link link-uri" href="http://www.sanger.ac.uk/">http://www.sanger.ac.uk/</a>) and COGs database. Sortase homologs and their substrates from the sequences of 15 chromosomes and 30 plasmids derived from 15 Gram-positive bacteria species (include <em>F. magna</em> ATCC 29328), whose both chromosomes and plasmids were sequenced to date, were identified <em>in silico</em> according to methods described by Comfort and Clubb.<sup><a reveal-id="DSM030C26" data-open="DSM030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The genomic sequences and the protein tables of bacteria were obtained through the NCBI (National Center of Biological Information, <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>). The complete genome sequence and annotation data of <em>F. magna</em> ATCC 29328 is available from the DDBJ/EMBL/GenBank database (accession nos. AP008971 and AP008972) and our web site (<a class="link link-uri" href="http://w5.grt.kyushu-u.ac.jp/Fmagna/">http://w4.grt.kyushu-u.ac.jp/Fmagna/</a>).</p>                            <h2 scrollto-destination=5139886 id="5139886" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5139887 id="5139887" class="section-title" data-legacy-id=s3a>3.1. General genome features</h3>
<p>The genome of <em>F. magna</em> ATCC 29328 consists of a circular chromosome (1 797 577 bp) and a plasmid pPEP1 (189 163 bp), with an average G + C content of 32.3 and 29.7%, respectively (Fig. <a reveal-id="DSM030F1" data-open="DSM030F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSM030TB1" data-open="DSM030TB1" class="link link-reveal link-table xref-fig">1</a>). The chromosome size was in good agreement with the size (1886 ± 62 kb) estimated in the physical map which we constructed previously.<sup><a reveal-id="DSM030C8" data-open="DSM030C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The recognition sites of <em>Apa</em>I, <em>Pme</em>I, <em>Sgr</em>AI, and I-<em>Ceu</em>I in our physical map<sup><a reveal-id="DSM030C8" data-open="DSM030C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and the locations of ORFs in our genetic map<sup><a reveal-id="DSM030C9" data-open="DSM030C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM030C10" data-open="DSM030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> were in nearly agreement with those in the complete genome sequence, except missing of an <em>Apa</em>I-fragment (94 bp) in the chromosomal physical map and a inversion of <em>Pme</em>I fragments in the plasmid physical map. We could not find any regions showing significantly higher G + C content on the chromosome, except for four rRNA operon regions (Fig. <a reveal-id="DSM030F1" data-open="DSM030F1" class="link link-reveal link-table xref-fig">1</a>). In the chromosome, we identified 1631 ORFs (FMG_0001–FMG_1631), covering 90.8% of the chromosome. Of these, 1177 (72.1%) could be assigned a putative function, 329 (20.2%) are conserved but the function unknown, and 125 (7.7%) are hypothetical genes unique to <em>F. magna</em> (Table <a reveal-id="DSM030TB1" data-open="DSM030TB1" class="link link-reveal link-table xref-fig">1</a>). The replication origin of the chromosome was deduced on the basis of the transition point in GC-skew analysis and the vicinal presence of <em>dnaA</em> box.<sup><a reveal-id="DSM030C27" data-open="DSM030C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A strong bias in gene orientation was observed, as 83.8% of ORFs on the chromosome are transcribed in the same orientation as DNA replication. Similarly, high bias in the gene orientation was observed in the genomes of <em>Clostridium perfringens</em> strain 13<sup>20</sup> and <em>Clostridium tetani</em> E88,<sup><a reveal-id="DSM030C21" data-open="DSM030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> respectively, and appears to be a common feature of low-GC Gram-positive bacteria.<sup><a reveal-id="DSM030C20" data-open="DSM030C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSM030C28" data-open="DSM030C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSM030C29" data-open="DSM030C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Plasmid pPEP1 encodes 182 ORFs (FMG_P0001–FMG_P0182). Of these, 45 (24.7%) could be assigned a putative function, 29 (15.9%) are conserved but the function unknown, and 108 (59.4%) are hypothetical genes (Table <a reveal-id="DSM030TB1" data-open="DSM030TB1" class="link link-reveal link-table xref-fig">1</a>). The replication origin of the plasmid could not be deduced by GC-skew analysis. Of the total ORFs on the plasmid, 87.4% were encoded on one strand DNA (Fig. <a reveal-id="DSM030F1" data-open="DSM030F1" class="link link-reveal link-table xref-fig">1</a>). The <em>F. magna</em> genome encodes only a few mobile genetic elements. The density of transposase genes per 1 Mb of chromosome in low-GC Gram-positive bacteria ranges from high (10.5 per Mb) in <em>Streptococcus pyogenes</em> MGAS8283 to low (3.63 per Mb) in <em>C. perfringens</em> strain 13. <em>F. magna</em> ATCC 29328 has only one transposase gene (FMG_0928) (0.56 per Mb) on the chromosome. In the plasmid, 14 putative transposase genes were encoded, but four out of 14 were truncated. <em>F. magna</em> has nine putative phage-related ORFs on the chromosome; however, these genes do not constitute a prophage region. Assuming that <em>F. magna</em> is closely related to clostridia<sup><a reveal-id="DSM030C10" data-open="DSM030C10" class="link link-ref link-reveal xref-bibr">10</a></sup>, <em>F. magna</em> has a stable chromosome, like clostridia, and recent events of gene acquisition by horizontal gene transfer are less likely.<sup><a reveal-id="DSM030C20" data-open="DSM030C20" class="link link-ref link-reveal xref-bibr">20</a>–<a reveal-id="DSM030C22" data-open="DSM030C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                        <a id="5139889" scrollto-destination="5139889"></a>
<div id="DSM030F1" data-id="DSM030F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/m_dsm03001.jpeg?Expires=1509238611&Signature=d08iZ0c4CwJK2GLM0-q9vf~w3SYPzz1UerYU7PxRkTe-FjPJtuYiwu1CiKycYMvvasfH1If~3PsGLftZzG3Wy3-i~~8b1diVDm1kTMmVlodRrn4KggHJQJMNszlvjLkVFVtb5q7s8YAFMRmhul27Yf-ELFD62qbxfufQ6BnAlREwCNejnKmvDhUKc2UpdpJRRTR6kedGNVDxsN9wTkW2WZVHlLmL6EX9E5fGr6Yj1p2MBZ-S7s3u~Pn1B918U61HEvv16i9zNPeE-s4AX3cEM4Lx3x6poOrtCBy6ULFzmzZMlpKLAAtlhq~M2w7YetehrWVHBea7wu5jZxz-pELj6g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Circular maps of the chromosome and the pPEP1 plasmid of F. magna ATCC 29328. Each circle of represents (from inside out): G + C content, GC skew (G + C/G  −  C), tRNA genes, rRNA operons (all consisted of 16S–23S–5S rRNA genes), predicted genes transcribed into the counterclockwise direction, and those into clockwise direction. Green- and red-arrow heads show sortase homologs (srf1 to srf11) and their substrates, respectively, identified in silico from the ATCC 29328 genome. The identified substrates contain three albumin-binding protein homologs (abp2, abp3, and abp4). The pPEP1 has two clusters (cluster-I and -II), containing several surface proteins identified by PSORT analyses24 in one region. All of the predicted genes are colored according to the Clusters of Orthologous Groups of Proteins functional classification.16" path-from-xml="dsm03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5139889/dsm03001.jpeg" path-from-xml="dsm03001.jpeg" target="_blank">View large</a><a section="5139889" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/dsm03001.jpeg?Expires=1509238611&Signature=G6gMbG3xgr7a9cylPoa948FSHlVCfZY7vT2o0ZbZwND0hyo8okDFzQJeuwebB3ZGTkVulmDnlhvV0m51f9NRHmLAUwor-8aDIUluJf0BWvCaDFyy2KOhN1gl7daniaPIJ~EXmpddwjetagXw-0Lg0OEUB39i6vrYDC~500d4v5JpQyF~xhg16592CUnZ8DKDaslvmWL5MMO-7xHgTaEopZDCv8gWmPg32v7~O6YrLtaZ~qPGlZ3knVPD~k3ZR8tEyeYF7afxVcfTu4~D-knb0zwWa6785XH3WaCN0u5Q2139Jyx0BQJd8ksalXFSWtf0HZR3i4yEFuqEqiNzawbppw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5139889&ar=357582&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular maps of the chromosome and the pPEP1 plasmid of <em>F. magna</em> ATCC 29328. Each circle of represents (from inside out): G + C content, GC skew (G + C/G  −  C), tRNA genes, rRNA operons (all consisted of 16S–23S–5S rRNA genes), predicted genes transcribed into the counterclockwise direction, and those into clockwise direction. Green- and red-arrow heads show sortase homologs (<em>srf1</em> to <em>srf11</em>) and their substrates, respectively, identified <em>in silico</em> from the ATCC 29328 genome. The identified substrates contain three albumin-binding protein homologs (<em>abp2</em>, <em>abp3</em>, and <em>abp4</em>). The pPEP1 has two clusters (cluster-I and -II), containing several surface proteins identified by PSORT analyses<sup>24</sup> in one region. All of the predicted genes are colored according to the Clusters of Orthologous Groups of Proteins functional classification.<sup>16</sup></p></div></div><div content-id="DSM030F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/m_dsm03001.jpeg?Expires=1509238611&Signature=d08iZ0c4CwJK2GLM0-q9vf~w3SYPzz1UerYU7PxRkTe-FjPJtuYiwu1CiKycYMvvasfH1If~3PsGLftZzG3Wy3-i~~8b1diVDm1kTMmVlodRrn4KggHJQJMNszlvjLkVFVtb5q7s8YAFMRmhul27Yf-ELFD62qbxfufQ6BnAlREwCNejnKmvDhUKc2UpdpJRRTR6kedGNVDxsN9wTkW2WZVHlLmL6EX9E5fGr6Yj1p2MBZ-S7s3u~Pn1B918U61HEvv16i9zNPeE-s4AX3cEM4Lx3x6poOrtCBy6ULFzmzZMlpKLAAtlhq~M2w7YetehrWVHBea7wu5jZxz-pELj6g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Circular maps of the chromosome and the pPEP1 plasmid of F. magna ATCC 29328. Each circle of represents (from inside out): G + C content, GC skew (G + C/G  −  C), tRNA genes, rRNA operons (all consisted of 16S–23S–5S rRNA genes), predicted genes transcribed into the counterclockwise direction, and those into clockwise direction. Green- and red-arrow heads show sortase homologs (srf1 to srf11) and their substrates, respectively, identified in silico from the ATCC 29328 genome. The identified substrates contain three albumin-binding protein homologs (abp2, abp3, and abp4). The pPEP1 has two clusters (cluster-I and -II), containing several surface proteins identified by PSORT analyses24 in one region. All of the predicted genes are colored according to the Clusters of Orthologous Groups of Proteins functional classification.16" path-from-xml="dsm03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5139889/dsm03001.jpeg" path-from-xml="dsm03001.jpeg" target="_blank">View large</a><a section="5139889" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/dsm03001.jpeg?Expires=1509238611&Signature=G6gMbG3xgr7a9cylPoa948FSHlVCfZY7vT2o0ZbZwND0hyo8okDFzQJeuwebB3ZGTkVulmDnlhvV0m51f9NRHmLAUwor-8aDIUluJf0BWvCaDFyy2KOhN1gl7daniaPIJ~EXmpddwjetagXw-0Lg0OEUB39i6vrYDC~500d4v5JpQyF~xhg16592CUnZ8DKDaslvmWL5MMO-7xHgTaEopZDCv8gWmPg32v7~O6YrLtaZ~qPGlZ3knVPD~k3ZR8tEyeYF7afxVcfTu4~D-knb0zwWa6785XH3WaCN0u5Q2139Jyx0BQJd8ksalXFSWtf0HZR3i4yEFuqEqiNzawbppw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5139889&ar=357582&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular maps of the chromosome and the pPEP1 plasmid of <em>F. magna</em> ATCC 29328. Each circle of represents (from inside out): G + C content, GC skew (G + C/G  −  C), tRNA genes, rRNA operons (all consisted of 16S–23S–5S rRNA genes), predicted genes transcribed into the counterclockwise direction, and those into clockwise direction. Green- and red-arrow heads show sortase homologs (<em>srf1</em> to <em>srf11</em>) and their substrates, respectively, identified <em>in silico</em> from the ATCC 29328 genome. The identified substrates contain three albumin-binding protein homologs (<em>abp2</em>, <em>abp3</em>, and <em>abp4</em>). The pPEP1 has two clusters (cluster-I and -II), containing several surface proteins identified by PSORT analyses<sup>24</sup> in one region. All of the predicted genes are colored according to the Clusters of Orthologous Groups of Proteins functional classification.<sup>16</sup></p></div></div>                        <a id="5139890" scrollto-destination="5139890"></a>
<div class="table-wrap"><div id="DSM030TB1" class="table-wrap-title"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Genome features of <em>F. magna</em> ATCC 29328</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Chromosome </th><th>Plasmid pPEP1 </th></tr></thead><tbody><tr><td>Length of sequence, bp </td><td>1,797,577 </td><td>189,163 </td></tr><tr><td>G + C content, % </td><td>32.3 </td><td>29.7 </td></tr><tr><td>Total number of ORFs </td><td>1,631 </td><td>182 </td></tr><tr><td> No. with assigned functions </td><td>1,177 </td><td>45 </td></tr><tr><td> No. of conserved hypotheticals* </td><td>329 </td><td>29 </td></tr><tr><td> No. of hypothetical proteins<sup>†</sup> </td><td>125 </td><td>108 </td></tr><tr><td>Average ORF length, bp </td><td>1,005 </td><td>888 </td></tr><tr><td>Coding density, % </td><td>90.8 </td><td>85.4 </td></tr><tr><td>rRNA operon </td><td>4 </td><td>0 </td></tr><tr><td>tRNA gene </td><td>48 </td><td>0 </td></tr></tbody></table></div><div content-id="DSM030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Chromosome </th><th>Plasmid pPEP1 </th></tr></thead><tbody><tr><td>Length of sequence, bp </td><td>1,797,577 </td><td>189,163 </td></tr><tr><td>G + C content, % </td><td>32.3 </td><td>29.7 </td></tr><tr><td>Total number of ORFs </td><td>1,631 </td><td>182 </td></tr><tr><td> No. with assigned functions </td><td>1,177 </td><td>45 </td></tr><tr><td> No. of conserved hypotheticals* </td><td>329 </td><td>29 </td></tr><tr><td> No. of hypothetical proteins<sup>†</sup> </td><td>125 </td><td>108 </td></tr><tr><td>Average ORF length, bp </td><td>1,005 </td><td>888 </td></tr><tr><td>Coding density, % </td><td>90.8 </td><td>85.4 </td></tr><tr><td>rRNA operon </td><td>4 </td><td>0 </td></tr><tr><td>tRNA gene </td><td>48 </td><td>0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Conserved hypothetical protein, sequence similarity to a translation of another ORF; however, there is currently no known function.</p></div></div><div content-id=""><div class="fn"><p><sup>†</sup>Hypothetical protein, no significant similarity to another protein.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5139890" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5139891 id="5139891" class="section-title" data-legacy-id=s3b>3.2. Sugar and amino acid metabolisms</h3>
<p>Most <em>F. magna</em> have a variety of aminopeptidase activities<sup><a reveal-id="DSM030C30" data-open="DSM030C30" class="link link-ref link-reveal xref-bibr">30</a></sup>; however, they have very limited saccharolytic activities.<sup><a reveal-id="DSM030C1" data-open="DSM030C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM030C31" data-open="DSM030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> It is known that most <em>F. magna</em> strains produce weak acid from the fermentation of fructose, and only a few strains produce weak acid from glucose, and all strains tested were unable to produce acid from maltose, mannose, and sucrose.<sup><a reveal-id="DSM030C1" data-open="DSM030C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM030C31" data-open="DSM030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We found that <em>F. magna</em> ATCC 29328 has a complete glycolysis pathway only for fructose (Supplementary Fig. S1). Fructose could be taken in by a phosphotransferase system (PTS) for fructose and fed into glycolysis followed by production of acetate as a major end product. However, PTS genes for glucose, maltose, mannose, glucitol, cellobiose, and lactose were absent in the <em>F. magna</em> genome. PTS for mannitol, galactitol, and sucrose were incomplete.</p><p>The tricarboxylic acid (TCA) cycle of <em>F. magna</em> ATCC 29328 was incomplete; only the step from pyruvate to fumarate was present (Supplementary Fig. S1). Similarly, incompleteness of the TCA cycle in clostridia such as <em>C. tetani, C. acetobutylicum</em>, and <em>C. perfringens</em> also has been reported.<sup><a reveal-id="DSM030C20  DSM030C22" data-open="DSM030C20  DSM030C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup><em>F. magna</em> ATCC 29328 has only gene sets for the production of acetate (and ethanol); although clostridia have complete gene sets for production of butyrate, acetate, lactate, and ethanol.<sup><a reveal-id="DSM030C20  DSM030C22" data-open="DSM030C20  DSM030C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup></p><p>We also constructed a metabolic map of amino acid degradation to investigate which amino acids are used for energy acquisition in <em>F. magna</em> (Supplementary Fig. S1). The metabolic map showed that <em>F. magna</em> ATCC 29328 converts at least five amino acids (glycine, serine, threonine, aspartate, and asparagine) into pyruvate, and produces acetate and ATP. The metabolic map also showed that three (glycine, serine, and threonine) out of the five amino acids lead to NH<sub>3</sub> production. As to amino acid biosynthesis pathways, <em>F. magna</em> ATCC 29328 has the complete gene sets required for biosynthesis of only five amino acids (glycine, serine, threonine, aspartate, and asparagine), suggesting that many of amino acids must be taken in from the environment (Supplementary Fig. S1). <em>F. magna</em> ATCC 29328 has many amino acid/oligopeptide transporters, enabling it to utilize amino acids as a major energy source by taking amino acids from the environment. As reported previously, <em>F. magna</em> has more aminopeptidase activities than other GPAC species.<sup><a reveal-id="DSM030C30" data-open="DSM030C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Bacterial aminopeptidases are generally regarded as a tissue-degrading enzyme and bacteria can utilize the degradation products as a nutrient source. In <em>F. magna</em> ATCC 29328 genome, four carboxypeptidase genes and six aminopeptidase genes including a Xaa-Pro aminopeptidase gene were found. These peptidase genes may contribute to bacterial growth and could account for higher pathogenicity of <em>F. magna</em> in GPAC.</p><p><em>Finegoldia magna</em> ATCC 29328 has at least 249 transport-related genes, accounting for 15.3% of the total ORFs. Of them, 49 ATP-binding cassette transporter systems (ABC transporters) could be identified on the basis of the similarities with ABC transporters from <em>E. coli</em>, <em>Bacillus subtilis</em>, and <em>Mycobacterium tuberculosis</em>.<sup><a reveal-id="DSM030C32  DSM030C34" data-open="DSM030C32  DSM030C34" class="link link-ref link-reveal xref-bibr">32–34</a></sup> These ABC transporters were divided into 14 importers (e.g., amino acid/oligopeptide importers) and 35 exporters (e.g., putative multidrug exporters) (Supplementary Fig. S1). We also found other types of transporters such as electron, ion, a fructose-specific PTS, amino acid/oligopeptide, and multidrug-efflux transporters. <em>Finegoldia magna</em> has two vacuolar (V)-type ATPases (one is similar to an ATPase of <em>C. tetani</em> and another is similar to that of <em>Fusobacterium nucleatum</em>), but it has no F<sub>0</sub>F<sub>1</sub>-type ATPase.</p>                            <h3 scrollto-destination=5139896 id="5139896" class="section-title" data-legacy-id=s3c>3.3. Regulation and stress responses</h3>
<p>Transcriptional and translational regulators, and stress responses of <em>F. magna</em> are poorly understood. <em>Finegoldia magna</em> ATCC 29328 has an RNA polymerase core enzyme (composed of RpoA, RpoB, RpoC, and RpoZ subunits), a major sigma factor, and two general stress sigma factors, but it has no an ECF-type sigma factor,<sup><a reveal-id="DSM030C35" data-open="DSM030C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSM030C36" data-open="DSM030C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which has been found in several Gram-positive bacteria. <em>Finegoldia magna</em> encodes 12 two-component regulator systems consisting of 13 sensor histidine kinases and 12 response regulators. An autoinducer-2 production protein LuxS related to a quorum-sensing system was encoded in the chromosome. The <em>F. magna</em> genome encodes no superoxide dismutase gene, but encodes genes for superoxide reductase, NADH oxidase, and putative NADH dehydrogenase. These genes may play a major role in enabling the organism to survive in intermediate aerobic conditions, such as mucosa and skin.</p>                            <h3 scrollto-destination=5139898 id="5139898" class="section-title" data-legacy-id=s3d>3.4. Virulence factors</h3>
<p>Putative virulence factors were screened from all ORFs of <em>F. magna</em> ATCC 29328 genome by BLAST, SignalP, PSORT, and Pfam analyses. We found some putative virulence-related proteins with signal sequences and characteristic motifs (Supplementary Table S1). Interestingly, the <em>F. magna</em> ATCC 29328 genome encodes the following four albumin-binding protein homologs: three (FMG_1501, FMG_1517, and FMG_1523) in the chromosome and one (FMGP_0118) in the plasmid. These four proteins have regions similar to the albumin-binding modules (GA module)<sup><a reveal-id="DSM030C6" data-open="DSM030C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSM030C37" data-open="DSM030C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSM030C38" data-open="DSM030C38" class="link link-ref link-reveal xref-bibr">38</a></sup> in PAB and urPAB proteins derived from <em>F. magna</em> ALB8 and ALB1B strains, respectively (Supplementary Fig. S2). These GA modules were also found in protein G from group C and G streptococci and in the protein L-like molecule from <em>F. magna</em> strain 3316.<sup><a reveal-id="DSM030C39" data-open="DSM030C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSM030C40" data-open="DSM030C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>Other virulence factors, except albumin-binding protein and protein L, have not been well studied in <em>F. magna</em>. In the ATCC 29328 genome, we found ten collagen adhesion protein homologs (Supplementary Table S1). These proteins possess several domains similar to the B-type domain (Pfam accession PF05738) in a collagen adhesion protein (Cna) derived from <em>Staphylococcus aureus</em>.<sup><a reveal-id="DSM030C41" data-open="DSM030C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Furthermore, the ATCC 29328 genome encodes 20 <em>N</em>-acetylmuramoyl-<span class="small-caps">l</span>-alanine amidase homologs (Cwp66 homologs) (Supplementary Table S1). These homologs have several putative cell wall binding repeat 2 (Pfam accession PF04122) motifs. The PF04122 motif is identified in autolysin and Cwp66, an adhesin of <em>Clostridium difficile</em>.<sup><a reveal-id="DSM030C42" data-open="DSM030C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The Cwp66 has been suggested as a multifunctional protein<sup><a reveal-id="DSM030C42" data-open="DSM030C42" class="link link-ref link-reveal xref-bibr">42</a></sup>; the C-terminal domain of Cwp66 gave partial adherence ability of <em>C. difficile</em> to host cells, the N-terminal domain has homology to some autolysins and could contribute to autolysis. We are interested in whether Cwp66 homologs of <em>F. magna</em> can act as adhesins to host cells, since the Cwp66 of <em>C. difficile</em> has partial adherence ability and are regarded as a virulence factor. Moreover, the ATCC 29328 genome encodes putative virulence factors such as a cell wall-associated serine proteinase precursor (FMG_0035) and a putative biofilm-associated surface protein (FMG_1333) (Supplementary Table S1).</p><p>Taken together, the pathogenic <em>F. magna</em> ATCC 29328 has several antiphagocytotic proteins such as albumin-binding proteins, and multiple putative host cell adhesins such as collagen adhesion proteins and Cwp66 homologs. Although the functions of these adhesins are not yet understood, future studies should disclose the mechanisms of virulence and the significance of <em>F. magna</em> as a member of the normal human flora.</p><p>An extracellular transpeptidase, sortase, catalyzes cell wall anchoring of surface proteins by cleaving the threonine and glycine residues of conserved LPXTG-like motif.<sup><a reveal-id="DSM030C43  DSM030C46" data-open="DSM030C43  DSM030C46" class="link link-ref link-reveal xref-bibr">43–46</a></sup> Sortases (SrtA, SrtB) from <em>Staphylococcus aureus</em> were identified first and have been well studied recently.<sup><a reveal-id="DSM030C47" data-open="DSM030C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSM030C48" data-open="DSM030C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Since some pathogens deficient in sortases exhibit reduced virulence, much interest has been focussed on sortase as a target for new antimicrobial agents.<sup><a reveal-id="DSM030C43  DSM030C46" data-open="DSM030C43  DSM030C46" class="link link-ref link-reveal xref-bibr">43–46</a></sup> Recently, Comfort and Clubb identified sortases and their substrates (surface proteins with the LPXTG-like motif) <em>in silico</em> from 241 bacterial chromosomes (96 species) sequenced completely, and they predicted some functional sortase–substrate linkages.<sup><a reveal-id="DSM030C26" data-open="DSM030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> By the same method with Comfort, we identified sortases and their substrates from the ATCC 29328 genome and several Gram-positive bacterial genomes.</p><p>The ATCC 29328 chromosome encoded four sortase homologs and 19 putative substrates (Figs. <a reveal-id="DSM030F1" data-open="DSM030F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSM030F2" data-open="DSM030F2" class="link link-reveal link-table xref-fig">2</a>A and Table <a reveal-id="DSM030TB2" data-open="DSM030TB2" class="link link-reveal link-table xref-fig">2</a>; Supplementary Tables S2 and S3). These 19 putative substrates include two albumin-binding protein homologs (FMG_1517 and FMG_1523) and putative biofilm-associated surface protein (FMG_1333), which may act as virulence factor (Table <a reveal-id="DSM030TB2" data-open="DSM030TB2" class="link link-reveal link-table xref-fig">2</a>; Supplementary Tables S1 and S3). Most interestingly, seven sortase homologs and seven substrates were identified from the plasmid pPEP1 (Figs. <a reveal-id="DSM030F1" data-open="DSM030F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSM030F2" data-open="DSM030F2" class="link link-reveal link-table xref-fig">2</a>B and Table <a reveal-id="DSM030TB2" data-open="DSM030TB2" class="link link-reveal link-table xref-fig">2</a>; Supplementary Tables S2 and S3). These seven substrates include an albumin-binding protein homolog (FMG_P0118) and four putative collagen adhesion proteins (FMG_P0123, FMG_P0125, FMG_P0146, and FMG_P0149) (Table <a reveal-id="DSM030TB2" data-open="DSM030TB2" class="link link-reveal link-table xref-fig">2</a>; Supplementary Tables S1 and S3).</p>                        <a id="5139904" scrollto-destination="5139904"></a>
<div id="DSM030F2" data-id="DSM030F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/m_dsm03002.jpeg?Expires=1509238611&Signature=DzWIyuBVGLm~1N3oo9iDXZIQSh2n0Y9UjduvY7oxS-KhlScAsY5DkIfyx3DwKZ4~k6uZbKi5YSd4oLlfreCowDYH1O390SLVwqzP8oliEOnvtkHmRkDWjf2jJTJ2wZpJzDBbZcB6ojQamDZ9ZHP77lYUHgbBQ4nxQ~TJzcYbn4M4i5fnzgvu56wFrpn~-R7ScCQirPJZCJkeepykGgSrLVt5XNA3KOC2mz6CaZHyHSjO7WySqB5-3qrqhor8qErv3g2bL2H9eTR3Y9DaxqVBrvKgJFnkQ8mDyzTwCCzZ72CrPbC6vxoPEsufcyRvZBFCbb0oHJa7uZ~r~IpAYsm2rw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Sortase homologs and substrates identified in silico from Gram-positive bacterial genomes. (A) Total number of ORFs in chromosome (15 species). (B) Total number of ORFs in plasmid (15 species, 30 plasmids). We identified ORFs from 15 Gram-positive bacterial species, whose both chromosomes and plasmids were sequenced to date. Abbreviations of bacterial species and detailed ORF data are showed in Supplementary Tables S2 and S3." path-from-xml="dsm03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5139904/dsm03002.jpeg" path-from-xml="dsm03002.jpeg" target="_blank">View large</a><a section="5139904" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/dsm03002.jpeg?Expires=1509238611&Signature=Kbr7bZh~JHBKTwc~-artm2LujlSsbyb~hDJDvGsPehExEfBA5HU8SEqv-Ua1WRgg5aiuHFTOejTNis7-YuigjJ8ON2vSw5KvtfXG6FvM5~vrxbmHQyF3DwNTs3VkOSOrJbxZogvcWVctrk6~lv3oXpMYuvyxrUz-zCSYWwEXaNN47Qi-cf00j4NSDoa0MIvTmq~wHjcVAsOnP5Df7J3migBXzcwDsm2-573FYZzGHa5Z49umh3~9q8m59k4GrCIT8n4VbPAarXTA5xyDM-4kJqzin8wPWwrRdQcYUpSxD9v9tgsVM7Hw61CR6-njrc1uUnwiTxmhItq-S2yVqIfuGQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5139904&ar=357582&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sortase homologs and substrates identified <em>in silico</em> from Gram-positive bacterial genomes. (<strong>A</strong>) Total number of ORFs in chromosome (15 species). (<strong>B</strong>) Total number of ORFs in plasmid (15 species, 30 plasmids). We identified ORFs from 15 Gram-positive bacterial species, whose both chromosomes and plasmids were sequenced to date. Abbreviations of bacterial species and detailed ORF data are showed in Supplementary Tables S2 and S3.</p></div></div><div content-id="DSM030F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/m_dsm03002.jpeg?Expires=1509238611&Signature=DzWIyuBVGLm~1N3oo9iDXZIQSh2n0Y9UjduvY7oxS-KhlScAsY5DkIfyx3DwKZ4~k6uZbKi5YSd4oLlfreCowDYH1O390SLVwqzP8oliEOnvtkHmRkDWjf2jJTJ2wZpJzDBbZcB6ojQamDZ9ZHP77lYUHgbBQ4nxQ~TJzcYbn4M4i5fnzgvu56wFrpn~-R7ScCQirPJZCJkeepykGgSrLVt5XNA3KOC2mz6CaZHyHSjO7WySqB5-3qrqhor8qErv3g2bL2H9eTR3Y9DaxqVBrvKgJFnkQ8mDyzTwCCzZ72CrPbC6vxoPEsufcyRvZBFCbb0oHJa7uZ~r~IpAYsm2rw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Sortase homologs and substrates identified in silico from Gram-positive bacterial genomes. (A) Total number of ORFs in chromosome (15 species). (B) Total number of ORFs in plasmid (15 species, 30 plasmids). We identified ORFs from 15 Gram-positive bacterial species, whose both chromosomes and plasmids were sequenced to date. Abbreviations of bacterial species and detailed ORF data are showed in Supplementary Tables S2 and S3." path-from-xml="dsm03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5139904/dsm03002.jpeg" path-from-xml="dsm03002.jpeg" target="_blank">View large</a><a section="5139904" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/dsm03002.jpeg?Expires=1509238611&Signature=Kbr7bZh~JHBKTwc~-artm2LujlSsbyb~hDJDvGsPehExEfBA5HU8SEqv-Ua1WRgg5aiuHFTOejTNis7-YuigjJ8ON2vSw5KvtfXG6FvM5~vrxbmHQyF3DwNTs3VkOSOrJbxZogvcWVctrk6~lv3oXpMYuvyxrUz-zCSYWwEXaNN47Qi-cf00j4NSDoa0MIvTmq~wHjcVAsOnP5Df7J3migBXzcwDsm2-573FYZzGHa5Z49umh3~9q8m59k4GrCIT8n4VbPAarXTA5xyDM-4kJqzin8wPWwrRdQcYUpSxD9v9tgsVM7Hw61CR6-njrc1uUnwiTxmhItq-S2yVqIfuGQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5139904&ar=357582&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sortase homologs and substrates identified <em>in silico</em> from Gram-positive bacterial genomes. (<strong>A</strong>) Total number of ORFs in chromosome (15 species). (<strong>B</strong>) Total number of ORFs in plasmid (15 species, 30 plasmids). We identified ORFs from 15 Gram-positive bacterial species, whose both chromosomes and plasmids were sequenced to date. Abbreviations of bacterial species and detailed ORF data are showed in Supplementary Tables S2 and S3.</p></div></div>                        <a id="5139905" scrollto-destination="5139905"></a>
<div class="table-wrap"><div id="DSM030TB2" class="table-wrap-title"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>The substrates of sortases identified in <em>F. magna</em> ATCC 29328 genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ORF no. </th><th>Product name </th><th>Motif* </th><th>Subfamily<sup>†</sup> </th><th>Cluster<sup>‡</sup> </th></tr></thead><tbody><tr><td>Chromosome </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> FMG_0037 </td><td>Putative collagen adhesion protein </td><td>LPESGS </td><td>— </td><td>No </td></tr><tr><td> FMG_0095 </td><td>Conserved hypothetical protein </td><td>PAAASF </td><td>— </td><td>No </td></tr><tr><td> FMG_0147 </td><td>Putative membrane protein </td><td>IALTAV </td><td>— </td><td>No </td></tr><tr><td> FMG_0186 </td><td>Hypothetical protein </td><td>YPLTGA </td><td>— </td><td>Yes </td></tr><tr><td> FMG_0187 </td><td>Cell wall surface anchor family protein </td><td>IPQTGG </td><td>3 </td><td>Yes </td></tr><tr><td> FMG_0188 </td><td>Putative cell wall surface anchor family protein </td><td>IPKTGD </td><td>3 </td><td>Yes </td></tr><tr><td> FMG_0229 </td><td>Putative surface protein precursor </td><td>LPKAGT </td><td>— </td><td>No </td></tr><tr><td> FMG_0238 </td><td>Branched-chain amino acid transporter </td><td>LPLSSV </td><td>— </td><td>No </td></tr><tr><td> FMG_0261 </td><td>Conserved hypothetical protein </td><td>YAASKS </td><td>— </td><td>No </td></tr><tr><td> FMG_0595 </td><td>Phosphate ABC transporter permease protein </td><td>MAYASG </td><td>— </td><td>No </td></tr><tr><td> FMG_0616 </td><td>Cell division protein homolog </td><td>IPPTGL </td><td>— </td><td>No </td></tr><tr><td> FMG_0717 </td><td>Cell division protein RodA homolog </td><td>MPITGI </td><td>— </td><td>No </td></tr><tr><td> FMG_1103 </td><td>Conserved hypothetical protein </td><td>VPFAGL </td><td>— </td><td>No </td></tr><tr><td> FMG_1333 </td><td>Putative biofilm-associated surface protein </td><td>LPKAGI </td><td>— </td><td>No </td></tr><tr><td> FMG_1335 </td><td>Hypothetical protein </td><td>LPKAGA </td><td>— </td><td>No </td></tr><tr><td> FMG_1352 </td><td>Hypothetical protein </td><td>LPKAGY </td><td>— </td><td>No </td></tr><tr><td> FMG_1517 </td><td>Albumin-binding protein homolog-2 </td><td>LPKAGS </td><td>— </td><td>No </td></tr><tr><td> FMG_1523 </td><td>Putative albumin-binding protein homolog-3 </td><td>LPKAGI </td><td>— </td><td>No </td></tr><tr><td> FMG_1550 </td><td>Conserved hypothetical protein </td><td>LPKAGS </td><td>— </td><td>No </td></tr><tr><td>Plasmid </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> FMG_P0118 </td><td>Albumin-binding protein homolog-4 like protein L </td><td>LPKAGS </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0119 </td><td>Hypothetical protein </td><td>LPKAGV </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0121 </td><td>Hypothetical protein </td><td>YAKTNI </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0123 </td><td>Putative collagen adhesion protein </td><td>VAIAGG </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0125 </td><td>Putative collagen adhesion protein </td><td>LPVTGL </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0146 </td><td>Putative collagen adhesion protein </td><td>IPSTGT </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0149 </td><td>Putative collagen adhesion protein </td><td>VPGTGT </td><td>— </td><td>Yes </td></tr></tbody></table></div><div content-id="DSM030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>ORF no. </th><th>Product name </th><th>Motif* </th><th>Subfamily<sup>†</sup> </th><th>Cluster<sup>‡</sup> </th></tr></thead><tbody><tr><td>Chromosome </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> FMG_0037 </td><td>Putative collagen adhesion protein </td><td>LPESGS </td><td>— </td><td>No </td></tr><tr><td> FMG_0095 </td><td>Conserved hypothetical protein </td><td>PAAASF </td><td>— </td><td>No </td></tr><tr><td> FMG_0147 </td><td>Putative membrane protein </td><td>IALTAV </td><td>— </td><td>No </td></tr><tr><td> FMG_0186 </td><td>Hypothetical protein </td><td>YPLTGA </td><td>— </td><td>Yes </td></tr><tr><td> FMG_0187 </td><td>Cell wall surface anchor family protein </td><td>IPQTGG </td><td>3 </td><td>Yes </td></tr><tr><td> FMG_0188 </td><td>Putative cell wall surface anchor family protein </td><td>IPKTGD </td><td>3 </td><td>Yes </td></tr><tr><td> FMG_0229 </td><td>Putative surface protein precursor </td><td>LPKAGT </td><td>— </td><td>No </td></tr><tr><td> FMG_0238 </td><td>Branched-chain amino acid transporter </td><td>LPLSSV </td><td>— </td><td>No </td></tr><tr><td> FMG_0261 </td><td>Conserved hypothetical protein </td><td>YAASKS </td><td>— </td><td>No </td></tr><tr><td> FMG_0595 </td><td>Phosphate ABC transporter permease protein </td><td>MAYASG </td><td>— </td><td>No </td></tr><tr><td> FMG_0616 </td><td>Cell division protein homolog </td><td>IPPTGL </td><td>— </td><td>No </td></tr><tr><td> FMG_0717 </td><td>Cell division protein RodA homolog </td><td>MPITGI </td><td>— </td><td>No </td></tr><tr><td> FMG_1103 </td><td>Conserved hypothetical protein </td><td>VPFAGL </td><td>— </td><td>No </td></tr><tr><td> FMG_1333 </td><td>Putative biofilm-associated surface protein </td><td>LPKAGI </td><td>— </td><td>No </td></tr><tr><td> FMG_1335 </td><td>Hypothetical protein </td><td>LPKAGA </td><td>— </td><td>No </td></tr><tr><td> FMG_1352 </td><td>Hypothetical protein </td><td>LPKAGY </td><td>— </td><td>No </td></tr><tr><td> FMG_1517 </td><td>Albumin-binding protein homolog-2 </td><td>LPKAGS </td><td>— </td><td>No </td></tr><tr><td> FMG_1523 </td><td>Putative albumin-binding protein homolog-3 </td><td>LPKAGI </td><td>— </td><td>No </td></tr><tr><td> FMG_1550 </td><td>Conserved hypothetical protein </td><td>LPKAGS </td><td>— </td><td>No </td></tr><tr><td>Plasmid </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> FMG_P0118 </td><td>Albumin-binding protein homolog-4 like protein L </td><td>LPKAGS </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0119 </td><td>Hypothetical protein </td><td>LPKAGV </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0121 </td><td>Hypothetical protein </td><td>YAKTNI </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0123 </td><td>Putative collagen adhesion protein </td><td>VAIAGG </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0125 </td><td>Putative collagen adhesion protein </td><td>LPVTGL </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0146 </td><td>Putative collagen adhesion protein </td><td>IPSTGT </td><td>— </td><td>Yes </td></tr><tr><td> FMG_P0149 </td><td>Putative collagen adhesion protein </td><td>VPGTGT </td><td>— </td><td>Yes </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*LPXTG-like motifs in substrates. These motifs were identified according to the methods of Comfort and Clubb.<sup>26</sup></p></div></div><div content-id=""><div class="fn"><p><sup>†</sup>Categorization into subfamilies according to sequence homology using PSI-BLAST profiles and a hidden Markov model.‘3’ and ‘—’ show the subfamily-3 and the unclassified subfamily, respectively, which were defined by Comfort.<sup>26</sup></p></div></div><div content-id=""><div class="fn"><p><sup>‡</sup>‘Yes’ shows that a sortase is genomically adjacent to its substrate.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5139905" target="_blank">View Large</a></div></div><p>By the same method with Comfort, we predicted functional sortase–substrate linkages for all sortases and substrates identified from the tested genomes. From the ATCC 29328 genome, we predicted some linkage among two sortases (FMG_0189 and FMG_0190) and two substrates (FMG_0187 and FMG_0188), because these four ORFs belong to a common family (Family 3 defined by Comfort) and are located on one cluster in the chromosome (Table <a reveal-id="DSM030TB2" data-open="DSM030TB2" class="link link-reveal link-table xref-fig">2</a>; Supplementary Tables S2 and S3). The remaining sortases and substrates of <em>F. magna</em> were assigned to an unclassified family (defined by Comfort), and their sortase–substrate linkages could not be predicted. Assuming that an unclassified substrate and its sortase in a common genomic cluster may have some functional linkage,<sup><a reveal-id="DSM030C26" data-open="DSM030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> eight out of the 24 unclassified substrates can be linked to their specific sortases (Table <a reveal-id="DSM030TB2" data-open="DSM030TB2" class="link link-reveal link-table xref-fig">2</a>; Supplementary Tables S2 and S3). These predicted sortase–substrate linkages on the genome should be further elucidated experimentally. For example, if we can identify a sortase that specifically recognize the LPKAG motif in albumin-binding proteins, it will lead to the development of sortase inhibitors as new antimicrobial agents.<sup><a reveal-id="DSM030C43" data-open="DSM030C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>To our knowledge, the pPEP1 plasmid may have the largest number of sortase genes among Gram-positive bacterial plasmids analysed so far, because we searched sortases and their substrates in 29 plasmids of 14 Gram-positive bacterial species, except for <em>F. magna</em>, and found that no plasmids have a sortase gene, except for <em>C. perfringens</em> str.13 plasmid (pCP13) with a sortase gene (PCP56) (Fig. <a reveal-id="DSM030F2" data-open="DSM030F2" class="link link-reveal link-table xref-fig">2</a>B; Supplementary Tables S2 and S3). We could not find a remarkable feature about the number of sortases and substrates among Gram-positive bacterial chromosomes (Fig. <a reveal-id="DSM030F2" data-open="DSM030F2" class="link link-reveal link-table xref-fig">2</a>A). The presence of as many as seven sortase genes in a plasmid may be unique to <em>F. magna</em>, since there are no sortase genes even on the large plasmids of the following Gram-positive species: <em>Bacillus anthracis</em> pXO1 (181 677 bp), <em>Bacillus cereus</em> pBc10987 (208 369 bp), and <em>C. acetobutylicum</em> pSOL1 (192 000 bp).</p>                            <h3 scrollto-destination=5139908 id="5139908" class="section-title" data-legacy-id=s3e>3.5. Conclusions</h3>
<p>This is the first report of a whole genome sequence of a member of any known GPAC. The sequence information presented and analysed here will contribute to an understanding of the taxonomy and pathogenicity of GPAC. The reason why <em>F. magna</em> is isolated most frequently and has the highest pathogenicity in GPAC may be due to the fact that <em>F. magna</em> genome encodes abundant aminopeptidases, multiple transporters and some virulence factors (e.g., albumin-binding proteins, Cwp66 homologs, and collagen adhesion proteins), beside with the limited capabilities of <em>F. magna</em> for sugar utilization and amino acid biosynthesis. It is of particular interest that four sortase genes were identified on the chromosome and that seven sortase genes were identified on the plasmid. Seven plasmid-encoded sortase genes unique to <em>F. magna</em> are of special interest, considering that plasmids are commonly considered to be of foreign origin. These mobile element-encoded sortases may play important roles in the pathogenesis of <em>F. magna</em> by enriching the variety of cell wall anchored surface proteins, which may enhance the interaction with host tissues and with other bacteria in mixed infection sites. Future studies, such as sortase–substrate linkage analyses and animal model experiments using sortase-negative mutant strains, will be required to clarify the roles of sortase on pathogenesis in <em>F. magna</em>.</p>                            <h2 scrollto-destination=5139910 id="5139910" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank T. Hayashi, N. Ogasawara, Y. Ohnishi, H. Hayashi, and H. Yoshikawa for their advice, support, and encouragement. We thank M. Yoshida, K. Furuya, C. Yoshino, A. Nakazawa, Y. Yamashita, and N. Ito for technical assistance.</p><p>The sequences reported in this paper have been deposited in the DDBJ/EMBL/GenBank database (accession chromosome, AP008971; plasmid, AP008972).</p>                            <h2 scrollto-destination=5139913 id="5139913" class="backsection-title" data-legacy-id=s4>Supplementary Data</h2>
<p>Supplementary data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5139915 id="5139915" class="backsection-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Research for the Future Program from the Japan Society for the Promotion of Science (JSPS-RFTF00L01411), in part by the Grant of the 21st Century COE Program, Ministry of Education, Culture, Sports, Science and Technology, Japan to Kitasato University, in part by Grant-in-Aid for Young Scientists (B) from the Ministry of Education, Culture, Sports, Science and Technology, Japan (MEXT-15790226), and in part by 2005 Wakayama Medical Award for Young Researchers.</p>                            <h2 scrollto-destination=5139917 id="5139917" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsm030c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murdoch</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Gram-positive anaerobic cocci</div>, <div class="source">Clin. Microbiol. Rev</div> , <div class="year">1998</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">81</div>-<div class="lpage">120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gram-positive%20anaerobic%20cocci&amp;author=D.%20A.%20Murdoch&amp;publication_year=1998&amp;journal=Clin.%20Microbiol.%20Rev&amp;volume=11&amp;pages=81-120" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9457430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murdoch</div> <div class="given-names">D. A.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Collier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Balows</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sussman</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Topley &amp; Wilson's Microbiology and Microbial Infections</div> , <div class="year">1998</div><div class="edition">9th</div><div class="publisher-loc">London</div><div class="publisher-name">Arnold</div>(pg. <div class="fpage">783</div>-<div class="lpage">797</div>)</div></div></div></div><div content-id="dsm030c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koneman</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Allen</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Janda</div> <div class="given-names">W. M.</div></div>,  <div class="name"><div class="surname">Schreckenberger</div> <div class="given-names">P. C.,</div></div>,  <div class="name"><div class="surname">Winn</div> <div class="given-names">W. C.</div>Jr.</div>. </div>, <div class="source">Color Atlas and Textbook of Diagnostic Microbiology</div> , <div class="year">1997</div><div class="edition">5th</div><div class="publisher-loc">Philadelphia</div><div class="publisher-name">Lippincott</div>(pg. <div class="fpage">709</div>-<div class="lpage">784</div>)</div></div></div></div><div content-id="dsm030c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bourgault</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Rosenblatt</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Fitzgerald</div> <div class="given-names">R. H</div></div>. </div><div class="article-title"><em>Peptococcus magnus</em>: a significant human pathogen</div>, <div class="source">Ann. Intern. Med</div> , <div class="year">1980</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">244</div>-<div class="lpage">248</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Peptococcus%20magnus%3A%20a%20significant%20human%20pathogen&amp;author=A.%20M.%20Bourgault&amp;author=J.%20E.%20Rosenblatt&amp;author=R.%20H%20Fitzgerald&amp;publication_year=1980&amp;journal=Ann.%20Intern.%20Med&amp;volume=93&amp;pages=244-248" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7326/0003-4819-93-2-244" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7326%2F0003-4819-93-2-244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7326%2F0003-4819-93-2-244"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7406374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Château</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Björck</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Identification of interdomain sequences promoting the intronless evolution of a bacterial protein family</div>, <div class="source">Proc. Natl. Acad. Sci</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> <div class="publisher-loc">USA</div>(pg. <div class="fpage">8490</div>-<div class="lpage">8495</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20interdomain%20sequences%20promoting%20the%20intronless%20evolution%20of%20a%20bacterial%20protein%20family&amp;author=M.%20de%20Ch%C3%A2teau&amp;author=L.%20Bj%C3%B6rck&amp;publication_year=1996&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=93&amp;pages=8490-8495" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.16.8490" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.16.8490" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.16.8490"> </span></div> </div></div></div></div></div><div content-id="dsm030c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lejon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Frick</div> <div class="given-names">I. M.</div></div>,  <div class="name"><div class="surname">Björck</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wikström</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Svensson</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Crystal structure and biological implications of a bacterial albumin binding module in complex with human serum albumin</div>, <div class="source">J. Biol. Chem</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">42924</div>-<div class="lpage">42928</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crystal%20structure%20and%20biological%20implications%20of%20a%20bacterial%20albumin%20binding%20module%20in%20complex%20with%20human%20serum%20albumin&amp;author=S.%20Lejon&amp;author=I.%20M.%20Frick&amp;author=L.%20Bj%C3%B6rck&amp;author=M.%20Wikstr%C3%B6m&amp;author=S.%20Svensson&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem&amp;volume=279&amp;pages=42924-42928" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M406957200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M406957200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M406957200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15269208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">D'Argy</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Whole-body autoradiography reveals that the <em>Peptostreptococcus magnus</em> immunoglobulin-binding domains of protein L preferentially target B lymphocytes in the spleen and lymph nodes <em>in vivo</em></div>, <div class="source">Cell. Microbiol</div> , <div class="year">2004</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">609</div>-<div class="lpage">623</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-body%20autoradiography%20reveals%20that%20the%20Peptostreptococcus%20magnus%20immunoglobulin-binding%20domains%20of%20protein%20L%20preferentially%20target%20B%20lymphocytes%20in%20the%20spleen%20and%20lymph%20nodes%20in%20vivo&amp;author=D.%20Smith&amp;author=R.%20D'Argy&amp;author=M.%20Nilsson&amp;publication_year=2004&amp;journal=Cell.%20Microbiol&amp;volume=6&amp;pages=609-623" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cmi.2004.6.issue-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcmi.2004.6.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcmi.2004.6.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15186398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Todo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miyamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Akimoto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Physical and genetic map of the <em>Finegoldia magna</em> (formerly <em>Peptostreptococcus magnus</em>) ATCC 29328 genome</div>, <div class="source">FEMS Microbiol. Lett</div> , <div class="year">2002</div>, vol. <div class="volume">210</div> (pg. <div class="fpage">33</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20and%20genetic%20map%20of%20the%20Finegoldia%20magna%20(formerly%20Peptostreptococcus%20magnus)%20ATCC%2029328%20genome&amp;author=K.%20Todo&amp;author=T.%20Goto&amp;author=K.%20Miyamoto&amp;author=S.%20Akimoto&amp;publication_year=2002&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=210&amp;pages=33-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2002.210.issue-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2002.210.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2002.210.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12023074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Todo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Honda</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Comparative analysis of the four rRNA operons in <em>Finegoldia magna</em> ATCC 29328</div>, <div class="source">Syst. Appl. Microbiol</div> , <div class="year">2004</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">18</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20four%20rRNA%20operons%20in%20Finegoldia%20magna%20ATCC%2029328&amp;author=K.%20Todo&amp;author=T.%20Goto&amp;author=A.%20Honda&amp;publication_year=2004&amp;journal=Syst.%20Appl.%20Microbiol&amp;volume=27&amp;pages=18-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1078/0723-2020-00247" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1078%2F0723-2020-00247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1078%2F0723-2020-00247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15053317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Todo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Miyamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Akimoto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Bacterial artificial chromosome library of <em>Finegoldia magna</em> ATCC 29328 for genetic mapping and comparative genomics</div>, <div class="source">Microbiol. Immunol</div> , <div class="year">2003</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">1005</div>-<div class="lpage">1016</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20artificial%20chromosome%20library%20of%20Finegoldia%20magna%20ATCC%2029328%20for%20genetic%20mapping%20and%20comparative%20genomics&amp;author=T.%20Goto&amp;author=K.%20Todo&amp;author=K.%20Miyamoto&amp;author=S.%20Akimoto&amp;publication_year=2003&amp;journal=Microbiol.%20Immunol&amp;volume=47&amp;pages=1005-1016" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.2003.47.issue-12" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.2003.47.issue-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.2003.47.issue-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14695451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gordon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Desmarais</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P</div></div>. </div><div class="article-title">Automated finishing with autofinish</div>, <div class="source">Genome Res</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">614</div>-<div class="lpage">625</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20finishing%20with%20autofinish&amp;author=D.%20Gordon&amp;author=C.%20Desmarais&amp;author=P%20Green&amp;publication_year=2001&amp;journal=Genome%20Res&amp;volume=11&amp;pages=614-625" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.171401" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.171401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.171401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakiyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">An automated system for genome analysis to support microbial whole-genome shotgun sequencing</div>, <div class="source">Biosci. Biotechnol. Biochem</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">670</div>-<div class="lpage">673</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20automated%20system%20for%20genome%20analysis%20to%20support%20microbial%20whole-genome%20shotgun%20sequencing&amp;author=T.%20Sakiyama&amp;author=H.%20Takami&amp;author=N.%20Ogasawara&amp;publication_year=2000&amp;journal=Biosci.%20Biotechnol.%20Biochem&amp;volume=64&amp;pages=670-673" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.64.670" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.64.670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.64.670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10803979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Badger</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">G. J.</div></div>. </div><div class="article-title">CRITICA: Coding region identification tool invoking comparative analysis</div>, <div class="source">Mol. Biol. Evol</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">512</div>-<div class="lpage">524</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRITICA%3A%20Coding%20region%20identification%20tool%20invoking%20comparative%20analysis&amp;author=J.%20H.%20Badger&amp;author=G.%20J.%20Olsen&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol&amp;volume=16&amp;pages=512-524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026133" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10331277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Modeling and predicting transcriptional units of <em>Escherichia coli</em> genes using hidden Markov models</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">987</div>-<div class="lpage">993</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20and%20predicting%20transcriptional%20units%20of%20Escherichia%20coli%20genes%20using%20hidden%20Markov%20models&amp;author=T.%20Yada&amp;author=M.%20Nakao&amp;author=Y.%20Totoki&amp;author=K.%20Nakai&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=987-993" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.12.987" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.12.987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.12.987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10745988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frishman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mironov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mewes</div> <div class="given-names">H. W.</div></div>,  <div class="name"><div class="surname">Gelfand</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Combining diverse evidence for gene recognition in completely sequenced bacterial genomes</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">2941</div>-<div class="lpage">2947</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combining%20diverse%20evidence%20for%20gene%20recognition%20in%20completely%20sequenced%20bacterial%20genomes&amp;author=D.%20Frishman&amp;author=A.%20Mironov&amp;author=H.%20W.%20Mewes&amp;author=M.%20Gelfand&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=2941-2947" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.12.2941" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.12.2941" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.12.2941"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9611239" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Natale</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Garkavtsev</div> <div class="given-names">I. V.</div></div>, et al. </div><div class="article-title">The COG database: new developments in phylogenetic classification of proteins from complete genomes</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">22</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20new%20developments%20in%20phylogenetic%20classification%20of%20proteins%20from%20complete%20genomes&amp;author=R.%20L.%20Tatusov&amp;author=D.%20A.%20Natale&amp;author=I.%20V.%20Garkavtsev&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=22-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.1.22" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.22" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.1.22"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-964" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawashima</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakaya</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The KEGG databases at GenomeNet</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">42</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20KEGG%20databases%20at%20GenomeNet&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;author=S.%20Kawashima&amp;author=A.%20Nakaya&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=42-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.42" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.42" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.42"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimizu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ohtani</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Clostridium perfringens</em>, an anaerobic flesh-eater</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">996</div>-<div class="lpage">1001</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Clostridium%20perfringens%2C%20an%20anaerobic%20flesh-eater&amp;author=T.%20Shimizu&amp;author=K.%20Ohtani&amp;author=H.%20Hirakawa&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=996-1001" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.022493799" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.022493799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.022493799"> </span></div> </div></div></div></div></div><div content-id="dsm030c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brüggemann</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bäumer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fricke</div> <div class="given-names">W. F.</div></div>, et al. </div><div class="article-title">The genome sequence of <em>Clostridium tetani</em>, the causative agent of tetanus disease</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">1316</div>-<div class="lpage">1321</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Clostridium%20tetani%2C%20the%20causative%20agent%20of%20tetanus%20disease&amp;author=H.%20Br%C3%BCggemann&amp;author=S.%20B%C3%A4umer&amp;author=W.%20F.%20Fricke&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=1316-1321" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0335853100" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0335853100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0335853100"> </span></div> </div></div></div></div></div><div content-id="dsm030c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nolling</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Breton</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Omelchenko</div> <div class="given-names">M. V.</div></div>, et al. </div><div class="article-title">Genome sequence and comparative analysis of the solvent-producing bacterium <em>Clostridium acetobutylicum</em></div>, <div class="source">J. Bacteriol</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">4823</div>-<div class="lpage">4838</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20comparative%20analysis%20of%20the%20solvent-producing%20bacterium%20Clostridium%20acetobutylicum&amp;author=J.%20Nolling&amp;author=G.%20Breton&amp;author=M.%20V.%20Omelchenko&amp;publication_year=2001&amp;journal=J.%20Bacteriol&amp;volume=183&amp;pages=4823-4838" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.16.4823-4838.2001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.16.4823-4838.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.16.4823-4838.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11466286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapatral</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ivanova</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Genome sequence and analysis of the oral bacterium <em>Fusobacterium nucleatum</em> strain ATCC 25586</div>, <div class="source">J. Bacteriol</div> , <div class="year">2002</div>, vol. <div class="volume">184</div> (pg. <div class="fpage">2005</div>-<div class="lpage">2018</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20analysis%20of%20the%20oral%20bacterium%20Fusobacterium%20nucleatum%20strain%20ATCC%2025586&amp;author=V.%20Kapatral&amp;author=I.%20Anderson&amp;author=N.%20Ivanova&amp;publication_year=2002&amp;journal=J.%20Bacteriol&amp;volume=184&amp;pages=2005-2018" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.184.7.2005-2018.2002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.184.7.2005-2018.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.184.7.2005-2018.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11889109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Horton</div> <div class="given-names">P.</div></div>. </div><div class="article-title">PSORT: a program for detecting sorting signals in proteins and predicting their subcellular localization</div>, <div class="source">Trends. Biochem. Sci</div> , <div class="year">1999</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">34</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PSORT%3A%20a%20program%20for%20detecting%20sorting%20signals%20in%20proteins%20and%20predicting%20their%20subcellular%20localization&amp;author=K.%20Nakai&amp;author=P.%20Horton&amp;publication_year=1999&amp;journal=Trends.%20Biochem.%20Sci&amp;volume=24&amp;pages=34-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(98)01336-X" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(98)01336-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(98)01336-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10087920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Engelbrecht</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites</div>, <div class="source">Protein Eng</div> , <div class="year">1997</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20prokaryotic%20and%20eukaryotic%20signal%20peptides%20and%20prediction%20of%20their%20cleavage%20sites&amp;author=H.%20Nielsen&amp;author=J.%20Engelbrecht&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;publication_year=1997&amp;journal=Protein%20Eng&amp;volume=10&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/protein/10.1.1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fprotein%2F10.1.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fprotein%2F10.1.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9051728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Comfort</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Clubb</div> <div class="given-names">R. T.</div></div>. </div><div class="article-title">A comparative genome analysis identifies distinct sorting pathways in Gram-positive bacteria</div>, <div class="source">Infect. Immun</div> , <div class="year">2004</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">2710</div>-<div class="lpage">2722</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20genome%20analysis%20identifies%20distinct%20sorting%20pathways%20in%20Gram-positive%20bacteria&amp;author=D.%20Comfort&amp;author=R.%20T.%20Clubb&amp;publication_year=2004&amp;journal=Infect.%20Immun&amp;volume=72&amp;pages=2710-2722" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.72.5.2710-2722.2004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.72.5.2710-2722.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.72.5.2710-2722.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15102780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salazar</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Fsihi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">de Rossi</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Organization of the origins of replication of the chromosomes of <em>Mycobacterium smegmatis</em>, <em>Mycobacterium leprae</em> and <em>Mycobacterium tuberculosis</em> and isolation of a functional origin from <em>M. semgmatis</em></div>, <div class="source">Mol. Microbiol</div> , <div class="year">1996</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">283</div>-<div class="lpage">293</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organization%20of%20the%20origins%20of%20replication%20of%20the%20chromosomes%20of%20Mycobacterium%20smegmatis%2C%20Mycobacterium%20leprae%20and%20Mycobacterium%20tuberculosis%20and%20isolation%20of%20a%20functional%20origin%20from%20M.%20semgmatis&amp;author=L.%20Salazar&amp;author=H.%20Fsihi&amp;author=E.%20de%20Rossi&amp;publication_year=1996&amp;journal=Mol.%20Microbiol&amp;volume=20&amp;pages=283-293" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1996.20.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1996.20.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1996.20.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8733228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fraser</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Gocayne</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">The minimal gene complement of <em>Mycoplasma genitalium</em></div>, <div class="source">Science</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">397</div>-<div class="lpage">403</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20minimal%20gene%20complement%20of%20Mycoplasma%20genitalium&amp;author=C.%20M.%20Fraser&amp;author=J.%20D.%20Gocayne&amp;author=O.%20White&amp;publication_year=1995&amp;journal=Science&amp;volume=270&amp;pages=397-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.270.5235.397" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.270.5235.397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.270.5235.397"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7569993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Bacterial DNA strand compositional asymmetry</div>, <div class="source">Trends Microbiol</div> , <div class="year">1999</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">305</div>-<div class="lpage">308</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20DNA%20strand%20compositional%20asymmetry&amp;author=S.%20Karlin&amp;publication_year=1999&amp;journal=Trends%20Microbiol&amp;volume=7&amp;pages=305-308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(99)01541-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(99)01541-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(99)01541-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10431198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">L. K.</div></div>,  <div class="name"><div class="surname">Mayrand</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Dillon</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title">Aminopeptidase activities in <em>Peptostreptococcus</em> spp. are statistically correlated to gelatin hydrolysis</div>, <div class="source">Can. J. Microbiol</div> , <div class="year">1998</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">303</div>-<div class="lpage">306</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminopeptidase%20activities%20in%20Peptostreptococcus%20spp.%20are%20statistically%20correlated%20to%20gelatin%20hydrolysis&amp;author=J.%20Ng&amp;author=L.%20K.%20Ng&amp;author=D.%20Mayrand&amp;author=J.%20A.%20Dillon&amp;publication_year=1998&amp;journal=Can.%20J.%20Microbiol&amp;volume=44&amp;pages=303-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/w98-016" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fw98-016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fw98-016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9606913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ezaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ninomiya</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yabuuchi</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Transfer of <em>Peptococcus indolicus</em>, <em>Peptococcus asaccharolyticus</em>, <em>Peptococcus prevotii</em>, and <em>Peptococcus magnus</em> to the Genus <em>Peptostreptococcus</em> and proposal of <em>Peptostreptococcus tetradius</em> sp. nov</div>, <div class="source">Int. J. Syst. Bacteriol</div> , <div class="year">1983</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">683</div>-<div class="lpage">698</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20of%20Peptococcus%20indolicus%2C%20Peptococcus%20asaccharolyticus%2C%20Peptococcus%20prevotii%2C%20and%20Peptococcus%20magnus%20to%20the%20Genus%20Peptostreptococcus%20and%20proposal%20of%20Peptostreptococcus%20tetradius%20sp.%20nov&amp;author=T.%20Ezaki&amp;author=N.%20Yamamoto&amp;author=K.%20Ninomiya&amp;author=S.%20Suzuki&amp;author=E.%20Yabuuchi&amp;publication_year=1983&amp;journal=Int.%20J.%20Syst.%20Bacteriol&amp;volume=33&amp;pages=683-698" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-33-4-683" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-33-4-683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-33-4-683"> </span></div> </div></div></div></div></div><div content-id="dsm030c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Linton</div> <div class="given-names">K. J.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">C. F.</div></div>. </div><div class="article-title">The <em>Escherichia coli</em> ATP-binding cassette (ABC) proteins</div>, <div class="source">Mol. Microbiol</div> , <div class="year">1998</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">5</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Escherichia%20coli%20ATP-binding%20cassette%20(ABC)%20proteins&amp;author=K.%20J.%20Linton&amp;author=C.%20F.%20Higgins&amp;publication_year=1998&amp;journal=Mol.%20Microbiol&amp;volume=28&amp;pages=5-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00764.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1998.00764.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1998.00764.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9593292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quentin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fichant</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Denizot</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Inventory, assembly and analysis of <em>Bacillus subtilis</em> ABC transport systems</div>, <div class="source">J. Mol. Biol</div> , <div class="year">1999</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">467</div>-<div class="lpage">484</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inventory%2C%20assembly%20and%20analysis%20of%20Bacillus%20subtilis%20ABC%20transport%20systems&amp;author=Y.%20Quentin&amp;author=G.%20Fichant&amp;author=F.%20Denizot&amp;publication_year=1999&amp;journal=J.%20Mol.%20Biol&amp;volume=287&amp;pages=467-484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1999.2624" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1999.2624" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1999.2624"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10092453" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Braibant</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gilot</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Content</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The ATP binding cassette (ABC) transport systems of <em>Mycobacterium tuberculosis</em></div>, <div class="source">FEMS Microbiol. Rev</div> , <div class="year">2000</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">449</div>-<div class="lpage">467</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ATP%20binding%20cassette%20(ABC)%20transport%20systems%20of%20Mycobacterium%20tuberculosis&amp;author=M.%20Braibant&amp;author=P.%20Gilot&amp;author=J.%20Content&amp;publication_year=2000&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=24&amp;pages=449-467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2000.tb00550.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2000.tb00550.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2000.tb00550.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10978546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lonetto</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">K. L.</div></div>,  <div class="name"><div class="surname">Rudd</div> <div class="given-names">K. E.</div></div>,  <div class="name"><div class="surname">Buttner</div> <div class="given-names">M. J.</div></div>. </div><div class="article-title">Analysis of the <em>Streptomyces coelicolor sigE</em> gene reveals the existence of a subfamily of eubacterial σ factors involved in the regulation of extracytoplasmic functions</div>, <div class="source">Proc. Natl. Acad. Sci</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> <div class="publisher-loc">USA</div>(pg. <div class="fpage">7573</div>-<div class="lpage">7577</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20Streptomyces%20coelicolor%20sigE%20gene%20reveals%20the%20existence%20of%20a%20subfamily%20of%20eubacterial%20%CF%83%20factors%20involved%20in%20the%20regulation%20of%20extracytoplasmic%20functions&amp;author=M.%20A.%20Lonetto&amp;author=K.%20L.%20Brown&amp;author=K.%20E.%20Rudd&amp;author=M.%20J.%20Buttner&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=91&amp;pages=7573-7577" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.16.7573" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.16.7573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.16.7573"> </span></div> </div></div></div></div></div><div content-id="dsm030c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Helmann</div> <div class="given-names">J. D.</div></div>. </div><div class="article-title">The extracytoplasmic function (ECF) sigma factors</div>, <div class="source">Adv. Microb. Physiol</div> , <div class="year">2002</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">47</div>-<div class="lpage">110</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20extracytoplasmic%20function%20(ECF)%20sigma%20factors&amp;author=J.%20D.%20Helmann&amp;publication_year=2002&amp;journal=Adv.%20Microb.%20Physiol&amp;volume=46&amp;pages=47-110" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12073657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johansson</div> <div class="given-names">M. U.</div></div>,  <div class="name"><div class="surname">de Château</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wikström</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Forsén</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Drakenberg</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Björck</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Solution structure of the albumin-binding GA module: a versatile bacterial protein domain</div>, <div class="source">J. Mol. Biol</div> , <div class="year">1997</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">859</div>-<div class="lpage">865</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solution%20structure%20of%20the%20albumin-binding%20GA%20module%3A%20a%20versatile%20bacterial%20protein%20domain&amp;author=M.%20U.%20Johansson&amp;author=M.%20de%20Ch%C3%A2teau&amp;author=M.%20Wikstr%C3%B6m&amp;author=S.%20Fors%C3%A9n&amp;author=T.%20Drakenberg&amp;author=L.%20Bj%C3%B6rck&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol&amp;volume=266&amp;pages=859-865" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0856" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0856" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0856"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9086265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johansson</div> <div class="given-names">M. U.</div></div>,  <div class="name"><div class="surname">Frick</div> <div class="given-names">I. M.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Structure, specificity, and mode of interaction for bacterial albumin-binding modules</div>, <div class="source">J. Biol. Chem</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">8114</div>-<div class="lpage">8120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%2C%20specificity%2C%20and%20mode%20of%20interaction%20for%20bacterial%20albumin-binding%20modules&amp;author=M.%20U.%20Johansson&amp;author=I.%20M.%20Frick&amp;author=H.%20Nilsson&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem&amp;volume=277&amp;pages=8114-8120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M109943200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M109943200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M109943200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11751858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guss</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Eliasson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Olsson</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Structure of the IgG-binding regions of streptococcal protein G</div>, <div class="source">EMBO J</div> , <div class="year">1986</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1567</div>-<div class="lpage">1575</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20IgG-binding%20regions%20of%20streptococcal%20protein%20G&amp;author=B.%20Guss&amp;author=M.%20Eliasson&amp;author=A.%20Olsson&amp;publication_year=1986&amp;journal=EMBO%20J&amp;volume=5&amp;pages=1567-1575" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3017704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murphy</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Duggleby</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Atkinson</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Trowern</div> <div class="given-names">A. R.</div></div>,  <div class="name"><div class="surname">Atkinson</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Goward</div> <div class="given-names">C. R.</div></div>. </div><div class="article-title">The functional units of a peptostreptococcal protein L</div>, <div class="source">Mol. Microbiol</div> , <div class="year">1994</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">911</div>-<div class="lpage">920</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functional%20units%20of%20a%20peptostreptococcal%20protein%20L&amp;author=J.%20P.%20Murphy&amp;author=C.%20J.%20Duggleby&amp;author=M.%20A.%20Atkinson&amp;author=A.%20R.%20Trowern&amp;author=T.%20Atkinson&amp;author=C.%20R.%20Goward&amp;publication_year=1994&amp;journal=Mol.%20Microbiol&amp;volume=12&amp;pages=911-920" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1994.12.issue-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1994.12.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1994.12.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7934898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Patti</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Jonsson</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Guss</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Molecular characterization and expression of a gene encoding a <em>Staphylococcus aureus</em> collagen adhesion</div>, <div class="source">J. Biol. Chem</div> , <div class="year">1992</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">4766</div>-<div class="lpage">4772</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20and%20expression%20of%20a%20gene%20encoding%20a%20Staphylococcus%20aureus%20collagen%20adhesion&amp;author=J.%20M.%20Patti&amp;author=H.%20Jonsson&amp;author=B.%20Guss&amp;publication_year=1992&amp;journal=J.%20Biol.%20Chem&amp;volume=267&amp;pages=4766-4772" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1311320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waligora</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Hennequin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Mullany</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bourlioux</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Collignon</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Karjalainen</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Characterization of a cell surface protein of <em>Clostridium difficile</em> with adhesive properties</div>, <div class="source">Infect. Immun</div> , <div class="year">2001</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">2144</div>-<div class="lpage">2153</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20cell%20surface%20protein%20of%20Clostridium%20difficile%20with%20adhesive%20properties&amp;author=A.%20J.%20Waligora&amp;author=C.%20Hennequin&amp;author=P.%20Mullany&amp;author=P.%20Bourlioux&amp;author=A.%20Collignon&amp;author=T.%20Karjalainen&amp;publication_year=2001&amp;journal=Infect.%20Immun&amp;volume=69&amp;pages=2144-2153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.69.4.2144-2153.2001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.69.4.2144-2153.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.69.4.2144-2153.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11254569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cossart</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jonquières</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Sortase, a universal target for therapeutic agents against Gram-positive bacteria?</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">5013</div>-<div class="lpage">5015</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sortase%2C%20a%20universal%20target%20for%20therapeutic%20agents%20against%20Gram-positive%20bacteria%3F&amp;author=P.%20Cossart&amp;author=R.%20Jonqui%C3%A8res&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=5013-5015" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.10.5013" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.10.5013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.10.5013"> </span></div> </div></div></div></div></div><div content-id="dsm030c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mazmanian</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Ton-That</div> <div class="given-names">H.,</div></div>,  <div class="name"><div class="surname">Schneewind</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Sortase-catalyzed anchoring of surface proteins to the cell wall of <em>Staphylococcus aureus</em></div>, <div class="source">Mol. Microbiol</div> , <div class="year">2001</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">1049</div>-<div class="lpage">1057</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sortase-catalyzed%20anchoring%20of%20surface%20proteins%20to%20the%20cell%20wall%20of%20Staphylococcus%20aureus&amp;author=S.%20K.%20Mazmanian&amp;author=H.%2C%20Ton-That&amp;author=O.%20Schneewind&amp;publication_year=2001&amp;journal=Mol.%20Microbiol&amp;volume=40&amp;pages=1049-1057" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02411.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2001.02411.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2001.02411.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11401711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Navarre</div> <div class="given-names">W. W.</div></div>,  <div class="name"><div class="surname">Schneewind</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Surface proteins of Gram-positive bacteria and mechanisms of their targeting to the cell wall envelope</div>, <div class="source">Microbiol. Mol. Biol. Rev</div> , <div class="year">1999</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">174</div>-<div class="lpage">229</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Surface%20proteins%20of%20Gram-positive%20bacteria%20and%20mechanisms%20of%20their%20targeting%20to%20the%20cell%20wall%20envelope&amp;author=W.%20W.%20Navarre&amp;author=O.%20Schneewind&amp;publication_year=1999&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=63&amp;pages=174-229" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10066836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Novick</div> <div class="given-names">R. P.</div></div>. </div><div class="article-title">Sortase: the surface protein anchoring transpeptidase and the LPXTG motif</div>, <div class="source">Trends. Microbiol</div> , <div class="year">2000</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">148</div>-<div class="lpage">151</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sortase%3A%20the%20surface%20protein%20anchoring%20transpeptidase%20and%20the%20LPXTG%20motif&amp;author=R.%20P.%20Novick&amp;publication_year=2000&amp;journal=Trends.%20Microbiol&amp;volume=8&amp;pages=148-151" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(00)01741-8" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(00)01741-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(00)01741-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10754567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bice</div> <div class="given-names">T. W.</div></div>,  <div class="name"><div class="surname">Ton-That</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Schneewind</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Narayana</div> <div class="given-names">S. V.</div></div>. </div><div class="article-title">Crystal structures of <em>Staphylococcus aureus</em> sortase A and its substrate complex</div>, <div class="source">J. Biol. Chem</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">31383</div>-<div class="lpage">31389</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crystal%20structures%20of%20Staphylococcus%20aureus%20sortase%20A%20and%20its%20substrate%20complex&amp;author=Y.%20Zong&amp;author=T.%20W.%20Bice&amp;author=H.%20Ton-That&amp;author=O.%20Schneewind&amp;author=S.%20V.%20Narayana&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem&amp;volume=279&amp;pages=31383-31389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M401374200" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M401374200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M401374200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15117963" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm030c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm030c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mazmanian</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Ton-That</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Schneewind</div> <div class="given-names">O.</div></div>. </div><div class="article-title">An iron-regulated sortase anchors a class of surface protein during <em>Staphylococcus aureus</em> pathogenesis</div>, <div class="source">Proc. Natl. Acad. Sci</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> <div class="publisher-loc">USA</div>(pg. <div class="fpage">2293</div>-<div class="lpage">2298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20iron-regulated%20sortase%20anchors%20a%20class%20of%20surface%20protein%20during%20Staphylococcus%20aureus%20pathogenesis&amp;author=S.%20K.%20Mazmanian&amp;author=H.%20Ton-That&amp;author=K.%20Su&amp;author=O.%20Schneewind&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=99&amp;pages=2293-2298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.032523999" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.032523999" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.032523999"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=357582&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/1/10.1093/dnares/dsm030/2/dsm030-dsm030supp.pdf?Expires=1509238612&amp;Signature=AMWIpYc-fBjaYRWsGX~qrD6wF8QQoGud2FKA7thy6d18bYLkjrr6blZRHgp6AyAryCUBxSzbXaDxLj3FYtiuO2mUdoHScxeYJ81cDMhdgAPT2noYfhSQxSRDwnrnZJZbLiW89y7TEJnjOJZe4jJV~zB3Rhf-i-8OxAEHn2~lp5ZjiPZv9YjitolH-wTZ8DKqBuJSdx1vccfsY4sdLva0rDqHc3Alj5JgfYR4QxvYZsWxIbKI~TntteIZGep6jbyRmW07lMqySPKE~AhtO96VHxdqc5IbPOrMdb-JQlIgPDELqla9TDZBFdUHd8ZyT~OpVssPVMVyyflw~RbEp7olFg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">17</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000254419400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">23</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsm030"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="357582"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000254419400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f1%2f39%2f357582">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000254419400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (23)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F1%2F39%2F357582" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/357582'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complete Genome Sequence of Finegoldia magna, an Anaerobic Opportunistic Pathogen" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f1%2f39%2f357582">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

