

<!DOCTYPE html>
<html lang="en">

<head><title>
	Pseudo-Response Regulator (PRR) Homologues of the Moss Physcomitrella patens: Insights into the Evolution of the PRR Family in Land Plants | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/1/39/378044","@type":"ScholarlyArticle","name":"Pseudo-Response Regulator (PRR) Homologues of the Moss  Physcomitrella patens : Insights into the Evolution of the PRR Family in Land Plants","datePublished":"2010-12-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2011-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq033","keywords":["circadian clock","His–Asp phosphorelay system","response regulator","pseudo-response regulator","<italic>Physcomitrella patens</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/1/39/378044","author":[{"name":"Satbhai, Santosh B.","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Yamashino, Takafumi","affiliation":"2  School of Agriculture ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Okada, Ryo","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Nomoto, Yuji","affiliation":"2  School of Agriculture ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Mizuno, Takeshi","affiliation":"2  School of Agriculture ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Tezuka, Yuki","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Itoh, Tomonori","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Tomita, Mitsuru","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Otsuki, Susumu","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"},{"name":"Aoki, Setsuyuki","affiliation":"1  Graduate School of Information Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya 464-8601 ,  Japan","@type":"Person"}],"description":"The pseudo-response regulators (PRRs) are the circadian clock component proteins in the model dicot Arabidopsis thaliana. They contain a receiver-like domain (RLD) similar to the receiver domains of the RRs in the His–Asp phosphorelay system, but the RLDs lack the phosphoacceptor aspartic acid residue invariably conserved in the receiver domains. To study the evolution of PRR genes in plants, here we characterize their homologue genes, PpPRR1, PpPRR2, PpPRR3 and PpPRR4, from the moss Physcomitrella patens. In the phylogenetic analysis, PpPRRs cluster together, sister to an angiosperm PRR gene subfamily, illustrating their close relationships with the angiosperm PRRs. However, distinct from the angiosperm sequences, the RLDs of PpPRR2/3/4 exhibit a potential phosphoacceptor aspartic acid–aspartic acid–lysine (DDK) motif. Consistently, the PpPRR2 RLD had phosphotransfer ability in vitro, suggesting that PpPRR2 functions as an RR. The PpPRR1 RLD, on the other hand, shows a partially diverged DDK motif, and it did not show phosphotransfer ability. All PpPRRs were expressed in a circadian and light-dependent manner, with differential regulation between PpPRR2/4 and PpPRR1/3. Altogether, our results illustrate that PRRs originated from an RR(s) and that there are intraspecific divergences among PpPRRs. Finally, we offer scenarios for the evolution of the PRR family in land plants.","pageStart":"39","pageEnd":"52","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=1572217140&Signature=bw4v9gG9OWvtWkvs74xNvCQfl06PTlFbdvoTzMMbATklM7hhKi2hnFJdZAcv5Acs~o0AHhnbxJVcECl4zwgKJqBIbLijFgShqUkPOw91iwwgTvnSid2aiRDXC7-I0quNVEUTXjE1NloYzErsPZ-B2xApzzp5oLFj0e4E3lM2eodhlN7nR5gNXf5rqA~7pQ-JznlRiffekBrqKfAtVy8txGHkjK5pTk9YsnZB-ZfbWlIWioPpJVP-n-QutRGgQmpCtO3JvdlQ2Tf6CElvh8HTqxvDjTxcYdJ0t8QLmIX4BkEyP6fRWiKr9Br~LURyIFFte113xMOi273SvLOx7uATNw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Pseudo-Response Regulator (PRR) Homologues of the Moss  Physcomitrella patens : Insights into the Evolution of the PRR Family in Land Plants","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=1572217140&Signature=bw4v9gG9OWvtWkvs74xNvCQfl06PTlFbdvoTzMMbATklM7hhKi2hnFJdZAcv5Acs~o0AHhnbxJVcECl4zwgKJqBIbLijFgShqUkPOw91iwwgTvnSid2aiRDXC7-I0quNVEUTXjE1NloYzErsPZ-B2xApzzp5oLFj0e4E3lM2eodhlN7nR5gNXf5rqA~7pQ-JznlRiffekBrqKfAtVy8txGHkjK5pTk9YsnZB-ZfbWlIWioPpJVP-n-QutRGgQmpCtO3JvdlQ2Tf6CElvh8HTqxvDjTxcYdJ0t8QLmIX4BkEyP6fRWiKr9Br~LURyIFFte113xMOi273SvLOx7uATNw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Pseudo-Response Regulator (PRR) Homologues of the Moss  Physcomitrella patens : Insights into the Evolution of the PRR Family in Land Plants | DNA Research | Oxford Academic" />
    <meta property="og:description" content="The pseudo-response regulators (PRRs) are the circadian clock component proteins in the model dicot Arabidopsis thaliana. They contain a receiver-like domain (RLD) similar to the receiver domains of the RRs in the His–Asp phosphorelay system, but the RLDs lack the phosphoacceptor aspartic acid residue invariably conserved in the receiver domains. To study the evolution of PRR genes in plants, here we characterize their homologue genes, PpPRR1, PpPRR2, PpPRR3 and PpPRR4, from the moss Physcomitrella patens. In the phylogenetic analysis, PpPRRs cluster together, sister to an angiosperm PRR gene subfamily, illustrating their close relationships with the angiosperm PRRs. However, distinct from the angiosperm sequences, the RLDs of PpPRR2/3/4 exhibit a potential phosphoacceptor aspartic acid–aspartic acid–lysine (DDK) motif. Consistently, the PpPRR2 RLD had phosphotransfer ability in vitro, suggesting that PpPRR2 functions as an RR. The PpPRR1 RLD, on the other hand, shows a partially diverged DDK motif, and it did not show phosphotransfer ability. All PpPRRs were expressed in a circadian and light-dependent manner, with differential regulation between PpPRR2/4 and PpPRR1/3. Altogether, our results illustrate that PRRs originated from an RR(s) and that there are intraspecific divergences among PpPRRs. Finally, we offer scenarios for the evolution of the PRR family in land plants." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/1/39/378044" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217140&amp;Signature=X5nnChVxVBrLB90RgnNHSkye2WHUP5FJHTVhDGiMGC9EWfUfsW1h~zlJqg5h6xKpFVsT0VM7mURqCwcBL1EOCQwRi9nZcTc77~8yjhibukw9YAMANw1wjHeBPIa-1r~rdHoqWigunNrfsDO0VQ353xcQTGjTAIgBVOsI4ldJ-0NNXGE4dYXtR80RKmgH2~aYHJeTTJHMTcPFuVqeMZC32IhEV4APmWYQIklTYCvQhy-r1LabfNTOQczcmt9bn0VovbZw79NoVIA6Z9MgPstwAAl9ACOXg7zjx1baYnYQ2vKjEwRqjCrdpK8-854~b9E1V1lJE4ssaXDERyWt~Jp7UA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2010-12-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Satbhai, Santosh B." /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Yamashino, Takafumi" /><meta name="citation_author_institution" content="School of Agriculture, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Okada, Ryo" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Nomoto, Yuji" /><meta name="citation_author_institution" content="School of Agriculture, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Mizuno, Takeshi" /><meta name="citation_author_institution" content="School of Agriculture, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Tezuka, Yuki" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Itoh, Tomonori" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Tomita, Mitsuru" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Otsuki, Susumu" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_author" content="Aoki, Setsuyuki" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya 464-8601, Japan" /><meta name="citation_title" content="Pseudo-Response Regulator (PRR) Homologues of the Moss Physcomitrella patens: Insights into the Evolution of the PRR Family in Land Plants" /><meta name="citation_firstpage" content="39" /><meta name="citation_lastpage" content="52" /><meta name="citation_doi" content="10.1093/dnares/dsq033" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2011/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Springer, New York; Cellular and Molecular Bases of Biological Clocks;  citation_year=1988; " /><meta name="citation_reference" content="citation_title=Temporal organization: reflections of a Darwinian clock-watcher; Annu. Rev. Physiol.;  citation_year=1993;  citation_volume=55;  citation_pages=16-54; " /><meta name="citation_reference" content="citation_title=SnapShot: circadian clock proteins; Cell;  citation_year=2008;  citation_volume=135;  citation_pages=368-8; " /><meta name="citation_reference" content="citation_title=Pseudo-response regulators (PRRs) or true oscillator components (TOCs); Plant Cell Physiol.;  citation_year=2005;  citation_volume=46;  citation_pages=677-85; " /><meta name="citation_reference" content="citation_title=The circadian system in higher plants; Annu. Rev. Plant Biol.;  citation_year=2009;  citation_volume=60;  citation_pages=357-77; " /><meta name="citation_reference" content="citation_title=Two-component phosphorelay signal transduction systems in plants: from hormone responses to circadian rhythms; Biosci. Biotechnol. Biochem.;  citation_year=2005;  citation_volume=69;  citation_pages=2263-76; " /><meta name="citation_reference" content="citation_title=Genes encoding pseudo-response regulators: insight into His-to-Asp phosphorelay and circadian rhythm in Arabidopsis thaliana; Plant Cell Physiol.;  citation_year=2000;  citation_volume=41;  citation_pages=791-803; " /><meta name="citation_reference" content="citation_title=The evolutionarily conserved OsPRR quintet: rice pseudo-response regulators implicated in circadian rhythm; Plant Cell Physiol.;  citation_year=2003;  citation_volume=44;  citation_pages=1229-36; " /><meta name="citation_reference" content="citation_title=Conservation of Arabidopsis flowering genes in model legumes; Plant Physiol.;  citation_year=2005;  citation_volume=137;  citation_pages=1420-34; " /><meta name="citation_reference" content="citation_title=Pea LATE BLOOMER1 is a GIGANTEA ortholog with roles in photoperiodic flowering, deetiolation, and transcriptional regulation of circadian clock gene homologs; Plant Physiol.;  citation_year=2007;  citation_volume=144;  citation_pages=648-61; " /><meta name="citation_reference" content="citation_title=Winter disruption of the circadian clock in chestnut; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=7037-42; " /><meta name="citation_reference" content="citation_title=Conserved expression profiles of circadian clock-related genes in two Lemna species showing long-day and short-day photoperiodic flowering responses; Plant Cell Physiol.;  citation_year=2005;  citation_volume=47;  citation_pages=601-12; " /><meta name="citation_reference" content="citation_title=Three type-B response regulators, ARR1, ARR10 and ARR12, play essential but redundant roles in cytokinin signal transduction throughout the life cycle of Arabidopsis thaliana; Plant Cell Physiol.;  citation_year=2008;  citation_volume=49;  citation_pages=47-57; " /><meta name="citation_reference" content="citation_title=Analysis of clock gene homologs using unifoliolates as target organs in soybean (Glycine max); J. Plant Physiol.;  citation_year=2009;  citation_volume=166;  citation_pages=278-89; " /><meta name="citation_reference" content="citation_title=Phylogenetic footprint of the plant clock system in angiosperms: evolutionary processes of pseudo-response regulators; BMC Evol. Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=126" /><meta name="citation_reference" content="citation_title=Clocks in the green lineage: comparative functional analysis of the circadian architecture of the picoeukaryote Ostreococcus; Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=3436-49; " /><meta name="citation_reference" content="citation_title=Green genes-comparative genomics of the green branch of life; Cell;  citation_year=2007;  citation_volume=129;  citation_pages=229-34; " /><meta name="citation_reference" content="citation_title=Exploring plant biodiversity: the Physcomitrella genome and beyond; Trends Plant Sci.;  citation_year=2008;  citation_volume=13;  citation_pages=542-9; " /><meta name="citation_reference" content="citation_title=Efficient gene targeting in the moss Physcomitrella patens; Plant J.;  citation_year=1997;  citation_volume=11;  citation_pages=1195-206; " /><meta name="citation_reference" content="citation_title=Functional characterization of CCA1/LHY homolog genes, PpCCA1a and PpCCA1b, in the moss Physcomitrella patens; Plant J.;  citation_year=2009;  citation_volume=60;  citation_pages=551-63; " /><meta name="citation_reference" content="DOE Joint Genome Institute P. patens subsp patens V1.1 http://genome.jgi-psf.org//Phypa1_1/Phypa1_1.home.html " /><meta name="citation_reference" content="citation_title=The Physcomitrella genome reveals evolutionary insights into the conquest of land by plants; Science;  citation_year=2010;  citation_volume=319;  citation_pages=64-9; " /><meta name="citation_reference" content="citation_title=Does the core circadian clock in the moss Physcomitrella patens (Bryophyta) comprise a single loop?; BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=109" /><meta name="citation_reference" content="citation_title=The isolation and preliminary characterization of auxotrophic and analogue resistant mutants in the moss Physcomitrella patens; Mol. Gen. Genet.;  citation_year=1977;  citation_volume=154;  citation_pages=87-95; " /><meta name="citation_reference" content="citation_title=Tagged mutagenesis and gene-trap in the moss, Physcomitrella patens by shuttle mutagenesis; DNA Res.;  citation_year=2000;  citation_volume=7;  citation_pages=9-17; " /><meta name="citation_reference" content="citation_title=CLUSTAL: a package for performing multiple sequence alignment on a microcomputer; Gene;  citation_year=1988;  citation_volume=73;  citation_pages=237-44; " /><meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences; Comput. Appl. Biosci.;  citation_year=1992;  citation_volume=8;  citation_pages=275-82; " /><meta name="citation_reference" content="citation_title=A simple method for estimating and testing minimum evolution trees; Mol. Biol. Evol.;  citation_year=1992;  citation_volume=9;  citation_pages=945-67; " /><meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using the bootstrap; Evolution;  citation_year=1985;  citation_volume=39;  citation_pages=783-91; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=PHYML Online–a web server for fast maximum likelihood-based phylogenetic inference; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W557-9; " /><meta name="citation_reference" content="citation_title=Arabidopsis clock-associated pseudo-response regulators PRR9, PRR7 and PRR5 coordinately and positively regulate flowering time through the canonical CONSTANS-dependent photoperiodic pathway; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=822-32; " /><meta name="citation_reference" content="citation_title=Protein measurement with the Folin phenol reagent; J. Biol. Chem.;  citation_year=1951;  citation_volume=193;  citation_pages=265-75; " /><meta name="citation_reference" content="citation_title=Phosphotransfer circuitry of the putative multi-signal transducer, ArcB, of Escherichia coli: in vitro studies with mutants; Mol. Microbiol.;  citation_year=1995;  citation_volume=18;  citation_pages=953-62; " /><meta name="citation_reference" content="citation_title=In vitro analysis of His-Asp phosphorelays in Aspergillus nidulans: the first direct biochemical evidence for the existence of His-Asp phosphotransfer systems in filamentous fungi; Biosci. Biotechnol. Biochem.;  citation_year=2007;  citation_volume=71;  citation_pages=2493-502; " /><meta name="citation_reference" content="citation_title=Communication modules in bacterial signaling proteins; Ann. Rev. Genet.;  citation_year=1992;  citation_volume=26;  citation_pages=71-112; " /><meta name="citation_reference" content="citation_title=Novel family of sensor histidine kinase genes in Arabidopsis thaliana; Plant Cell Physiol.;  citation_year=2001;  citation_volume=42;  citation_pages=231-5; " /><meta name="citation_reference" content="citation_title=Circadian expression of the PpLhcb2 gene encoding a major light-harvesting chlorophyll a/b-binding protein in the moss Physcomitrella patens; Plant Cell Physiol.;  citation_year=2004;  citation_volume=45;  citation_pages=68-76; " /><meta name="citation_reference" content="citation_title=Differential expression on a daily basis of plastid sigma factor PpSig genes from the moss Physcomitrella patens: regulatory interactions among PpSig5, the circadian clock and blue light signaling mediated by cryptochromes; Plant Physiol.;  citation_year=2004;  citation_volume=136;  citation_pages=4285-98; " /><meta name="citation_reference" content="citation_title=Photoperiod-regulated expression of the PpCOL1 gene encoding a homolog of CO/COL proteins in the moss Physcomitrella patens; Biochem. Biophys. Res. Commun.;  citation_year=2004;  citation_volume=324;  citation_pages=1296-301; " /><meta name="citation_reference" content="citation_title=Two-component signal transduction pathways in Arabidopsis; Plant Physiol.;  citation_year=2002;  citation_volume=129;  citation_pages=500-15; " /><meta name="citation_reference" content="citation_title=A genome-wide compilation of the two-component systems in Lotus japonicas; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=237-47; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/1/39/1074576/dsq033.pdf" /><meta name="description" content="Abstract.  The pseudo-response regulators (PRRs) are the circadian clock component proteins in the model dicot Arabidopsis thaliana. They contain a receiver-lik" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/1/39/378044" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/1/39/378044" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsq033';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./378044" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFjGc8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdGJoYWksIFNhbnRvc2ggQi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFnb3lhIFVuaXZlcnNpdHksIEZ1cm8tY2hvLCBDaGlrdXNhLWt1LCBOYWdveWEgNDY0LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYXNoaW5vLCBUYWthZnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgTmFnb3lhIFVuaXZlcnNpdHksIEZ1cm8tY2hvLCBDaGlrdXNhLWt1LCBOYWdveWEgNDY0LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2thZGEsIFJ5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYWdveWEgVW5pdmVyc2l0eSwgRnVyby1jaG8sIENoaWt1c2Eta3UsIE5hZ295YSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb21vdG8sIFl1amkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIE5hZ295YSBVbml2ZXJzaXR5LCBGdXJvLWNobywgQ2hpa3VzYS1rdSwgTmFnb3lhIDQ2NC04NjAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1penVubywgVGFrZXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgTmFnb3lhIFVuaXZlcnNpdHksIEZ1cm8tY2hvLCBDaGlrdXNhLWt1LCBOYWdveWEgNDY0LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGV6dWthLCBZdWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hZ295YSBVbml2ZXJzaXR5LCBGdXJvLWNobywgQ2hpa3VzYS1rdSwgTmFnb3lhIDQ2NC04NjAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl0b2gsIFRvbW9ub3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hZ295YSBVbml2ZXJzaXR5LCBGdXJvLWNobywgQ2hpa3VzYS1rdSwgTmFnb3lhIDQ2NC04NjAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvbWl0YSwgTWl0c3VydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYWdveWEgVW5pdmVyc2l0eSwgRnVyby1jaG8sIENoaWt1c2Eta3UsIE5hZ295YSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPdHN1a2ksIFN1c3VtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYWdveWEgVW5pdmVyc2l0eSwgRnVyby1jaG8sIENoaWt1c2Eta3UsIE5hZ295YSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBb2tpLCBTZXRzdXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFnb3lhIFVuaXZlcnNpdHksIEZ1cm8tY2hvLCBDaGlrdXNhLWt1LCBOYWdveWEgNDY0LTg2MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJQc2V1ZG8tUmVzcG9uc2UgUmVndWxhdG9yIChQUlIpIEhvbW9sb2d1ZXMgb2YgdGhlIE1vc3MgUGh5c2NvbWl0cmVsbGEgcGF0ZW5zOiBJbnNpZ2h0cyBpbnRvIHRoZSBFdm9sdXRpb24gb2YgdGhlIFBSUiBGYW1pbHkgaW4gTGFuZCBQbGFudHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3EwMzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgTmV3IFlvcms7IENlbGx1bGFyIGFuZCBNb2xlY3VsYXIgQmFzZXMgb2YgQmlvbG9naWNhbCBDbG9ja3M7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbXBvcmFsIG9yZ2FuaXphdGlvbjogcmVmbGVjdGlvbnMgb2YgYSBEYXJ3aW5pYW4gY2xvY2std2F0Y2hlcjsgQW5udS4gUmV2LiBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTE2LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbmFwU2hvdDogY2lyY2FkaWFuIGNsb2NrIHByb3RlaW5zOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTM2OC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qc2V1ZG8tcmVzcG9uc2UgcmVndWxhdG9ycyAoUFJScykgb3IgdHJ1ZSBvc2NpbGxhdG9yIGNvbXBvbmVudHMgKFRPQ3MpOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9Njc3LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY2lyY2FkaWFuIHN5c3RlbSBpbiBoaWdoZXIgcGxhbnRzOyBBbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MzU3LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28tY29tcG9uZW50IHBob3NwaG9yZWxheSBzaWduYWwgdHJhbnNkdWN0aW9uIHN5c3RlbXMgaW4gcGxhbnRzOiBmcm9tIGhvcm1vbmUgcmVzcG9uc2VzIHRvIGNpcmNhZGlhbiByaHl0aG1zOyBCaW9zY2kuIEJpb3RlY2hub2wuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MjI2My03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgZW5jb2RpbmcgcHNldWRvLXJlc3BvbnNlIHJlZ3VsYXRvcnM6IGluc2lnaHQgaW50byBIaXMtdG8tQXNwIHBob3NwaG9yZWxheSBhbmQgY2lyY2FkaWFuIHJoeXRobSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTc5MS04MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcmlseSBjb25zZXJ2ZWQgT3NQUlIgcXVpbnRldDogcmljZSBwc2V1ZG8tcmVzcG9uc2UgcmVndWxhdG9ycyBpbXBsaWNhdGVkIGluIGNpcmNhZGlhbiByaHl0aG07IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMjI5LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gb2YgQXJhYmlkb3BzaXMgZmxvd2VyaW5nIGdlbmVzIGluIG1vZGVsIGxlZ3VtZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEzNzsgIGNpdGF0aW9uX3BhZ2VzPTE0MjAtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlYSBMQVRFIEJMT09NRVIxIGlzIGEgR0lHQU5URUEgb3J0aG9sb2cgd2l0aCByb2xlcyBpbiBwaG90b3BlcmlvZGljIGZsb3dlcmluZywgZGVldGlvbGF0aW9uLCBhbmQgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gb2YgY2lyY2FkaWFuIGNsb2NrIGdlbmUgaG9tb2xvZ3M7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0NDsgIGNpdGF0aW9uX3BhZ2VzPTY0OC02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2ludGVyIGRpc3J1cHRpb24gb2YgdGhlIGNpcmNhZGlhbiBjbG9jayBpbiBjaGVzdG51dDsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz03MDM3LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2ZWQgZXhwcmVzc2lvbiBwcm9maWxlcyBvZiBjaXJjYWRpYW4gY2xvY2stcmVsYXRlZCBnZW5lcyBpbiB0d28gTGVtbmEgc3BlY2llcyBzaG93aW5nIGxvbmctZGF5IGFuZCBzaG9ydC1kYXkgcGhvdG9wZXJpb2RpYyBmbG93ZXJpbmcgcmVzcG9uc2VzOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NjAxLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaHJlZSB0eXBlLUIgcmVzcG9uc2UgcmVndWxhdG9ycywgQVJSMSwgQVJSMTAgYW5kIEFSUjEyLCBwbGF5IGVzc2VudGlhbCBidXQgcmVkdW5kYW50IHJvbGVzIGluIGN5dG9raW5pbiBzaWduYWwgdHJhbnNkdWN0aW9uIHRocm91Z2hvdXQgdGhlIGxpZmUgY3ljbGUgb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz00Ny01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY2xvY2sgZ2VuZSBob21vbG9ncyB1c2luZyB1bmlmb2xpb2xhdGVzIGFzIHRhcmdldCBvcmdhbnMgaW4gc295YmVhbiAoR2x5Y2luZSBtYXgpOyBKLiBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0yNzgtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBmb290cHJpbnQgb2YgdGhlIHBsYW50IGNsb2NrIHN5c3RlbSBpbiBhbmdpb3NwZXJtczogZXZvbHV0aW9uYXJ5IHByb2Nlc3NlcyBvZiBwc2V1ZG8tcmVzcG9uc2UgcmVndWxhdG9yczsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvY2tzIGluIHRoZSBncmVlbiBsaW5lYWdlOiBjb21wYXJhdGl2ZSBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIHRoZSBjaXJjYWRpYW4gYXJjaGl0ZWN0dXJlIG9mIHRoZSBwaWNvZXVrYXJ5b3RlIE9zdHJlb2NvY2N1czsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM0MzYtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyZWVuIGdlbmVzLWNvbXBhcmF0aXZlIGdlbm9taWNzIG9mIHRoZSBncmVlbiBicmFuY2ggb2YgbGlmZTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz0yMjktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyBwbGFudCBiaW9kaXZlcnNpdHk6IHRoZSBQaHlzY29taXRyZWxsYSBnZW5vbWUgYW5kIGJleW9uZDsgVHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01NDItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGdlbmUgdGFyZ2V0aW5nIGluIHRoZSBtb3NzIFBoeXNjb21pdHJlbGxhIHBhdGVuczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMTk1LTIwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBjaGFyYWN0ZXJpemF0aW9uIG9mIENDQTEvTEhZIGhvbW9sb2cgZ2VuZXMsIFBwQ0NBMWEgYW5kIFBwQ0NBMWIsIGluIHRoZSBtb3NzIFBoeXNjb21pdHJlbGxhIHBhdGVuczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz01NTEtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkRPRSBKb2ludCBHZW5vbWUgSW5zdGl0dXRlIFAuIHBhdGVucyBzdWJzcCBwYXRlbnMgVjEuMSBodHRwOi8vZ2Vub21lLmpnaS1wc2Yub3JnLy9QaHlwYTFfMS9QaHlwYTFfMS5ob21lLmh0bWwgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFBoeXNjb21pdHJlbGxhIGdlbm9tZSByZXZlYWxzIGV2b2x1dGlvbmFyeSBpbnNpZ2h0cyBpbnRvIHRoZSBjb25xdWVzdCBvZiBsYW5kIGJ5IHBsYW50czsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTk7ICBjaXRhdGlvbl9wYWdlcz02NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb2VzIHRoZSBjb3JlIGNpcmNhZGlhbiBjbG9jayBpbiB0aGUgbW9zcyBQaHlzY29taXRyZWxsYSBwYXRlbnMgKEJyeW9waHl0YSkgY29tcHJpc2UgYSBzaW5nbGUgbG9vcD87IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpc29sYXRpb24gYW5kIHByZWxpbWluYXJ5IGNoYXJhY3Rlcml6YXRpb24gb2YgYXV4b3Ryb3BoaWMgYW5kIGFuYWxvZ3VlIHJlc2lzdGFudCBtdXRhbnRzIGluIHRoZSBtb3NzIFBoeXNjb21pdHJlbGxhIHBhdGVuczsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk3NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTQ7ICBjaXRhdGlvbl9wYWdlcz04Ny05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFnZ2VkIG11dGFnZW5lc2lzIGFuZCBnZW5lLXRyYXAgaW4gdGhlIG1vc3MsIFBoeXNjb21pdHJlbGxhIHBhdGVucyBieSBzaHV0dGxlIG11dGFnZW5lc2lzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9OS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0xVU1RBTDogYSBwYWNrYWdlIGZvciBwZXJmb3JtaW5nIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBvbiBhIG1pY3JvY29tcHV0ZXI7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz0yMzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByYXBpZCBnZW5lcmF0aW9uIG9mIG11dGF0aW9uIGRhdGEgbWF0cmljZXMgZnJvbSBwcm90ZWluIHNlcXVlbmNlczsgQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIG1ldGhvZCBmb3IgZXN0aW1hdGluZyBhbmQgdGVzdGluZyBtaW5pbXVtIGV2b2x1dGlvbiB0cmVlczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25maWRlbmNlIGxpbWl0cyBvbiBwaHlsb2dlbmllczogYW4gYXBwcm9hY2ggdXNpbmcgdGhlIGJvb3RzdHJhcDsgRXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NzgzLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNDogTW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljcyBBbmFseXNpcyAoTUVHQSkgc29mdHdhcmUgdmVyc2lvbiA0LjA7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBIWU1MIE9ubGluZeKAk2Egd2ViIHNlcnZlciBmb3IgZmFzdCBtYXhpbXVtIGxpa2VsaWhvb2QtYmFzZWQgcGh5bG9nZW5ldGljIGluZmVyZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzU1Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBjbG9jay1hc3NvY2lhdGVkIHBzZXVkby1yZXNwb25zZSByZWd1bGF0b3JzIFBSUjksIFBSUjcgYW5kIFBSUjUgY29vcmRpbmF0ZWx5IGFuZCBwb3NpdGl2ZWx5IHJlZ3VsYXRlIGZsb3dlcmluZyB0aW1lIHRocm91Z2ggdGhlIGNhbm9uaWNhbCBDT05TVEFOUy1kZXBlbmRlbnQgcGhvdG9wZXJpb2RpYyBwYXRod2F5OyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9ODIyLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIG1lYXN1cmVtZW50IHdpdGggdGhlIEZvbGluIHBoZW5vbCByZWFnZW50OyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk1MTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz0yNjUtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBob3NwaG90cmFuc2ZlciBjaXJjdWl0cnkgb2YgdGhlIHB1dGF0aXZlIG11bHRpLXNpZ25hbCB0cmFuc2R1Y2VyLCBBcmNCLCBvZiBFc2NoZXJpY2hpYSBjb2xpOiBpbiB2aXRybyBzdHVkaWVzIHdpdGggbXV0YW50czsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9OTUzLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXRybyBhbmFseXNpcyBvZiBIaXMtQXNwIHBob3NwaG9yZWxheXMgaW4gQXNwZXJnaWxsdXMgbmlkdWxhbnM6IHRoZSBmaXJzdCBkaXJlY3QgYmlvY2hlbWljYWwgZXZpZGVuY2UgZm9yIHRoZSBleGlzdGVuY2Ugb2YgSGlzLUFzcCBwaG9zcGhvdHJhbnNmZXIgc3lzdGVtcyBpbiBmaWxhbWVudG91cyBmdW5naTsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTI0OTMtNTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21tdW5pY2F0aW9uIG1vZHVsZXMgaW4gYmFjdGVyaWFsIHNpZ25hbGluZyBwcm90ZWluczsgQW5uLiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTcxLTExMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgZmFtaWx5IG9mIHNlbnNvciBoaXN0aWRpbmUga2luYXNlIGdlbmVzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MjMxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmNhZGlhbiBleHByZXNzaW9uIG9mIHRoZSBQcExoY2IyIGdlbmUgZW5jb2RpbmcgYSBtYWpvciBsaWdodC1oYXJ2ZXN0aW5nIGNobG9yb3BoeWxsIGEvYi1iaW5kaW5nIHByb3RlaW4gaW4gdGhlIG1vc3MgUGh5c2NvbWl0cmVsbGEgcGF0ZW5zOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NjgtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBleHByZXNzaW9uIG9uIGEgZGFpbHkgYmFzaXMgb2YgcGxhc3RpZCBzaWdtYSBmYWN0b3IgUHBTaWcgZ2VuZXMgZnJvbSB0aGUgbW9zcyBQaHlzY29taXRyZWxsYSBwYXRlbnM6IHJlZ3VsYXRvcnkgaW50ZXJhY3Rpb25zIGFtb25nIFBwU2lnNSwgdGhlIGNpcmNhZGlhbiBjbG9jayBhbmQgYmx1ZSBsaWdodCBzaWduYWxpbmcgbWVkaWF0ZWQgYnkgY3J5cHRvY2hyb21lczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9NDI4NS05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvdG9wZXJpb2QtcmVndWxhdGVkIGV4cHJlc3Npb24gb2YgdGhlIFBwQ09MMSBnZW5lIGVuY29kaW5nIGEgaG9tb2xvZyBvZiBDTy9DT0wgcHJvdGVpbnMgaW4gdGhlIG1vc3MgUGh5c2NvbWl0cmVsbGEgcGF0ZW5zOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9MTI5Ni0zMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3by1jb21wb25lbnQgc2lnbmFsIHRyYW5zZHVjdGlvbiBwYXRod2F5cyBpbiBBcmFiaWRvcHNpczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI5OyAgY2l0YXRpb25fcGFnZXM9NTAwLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIGNvbXBpbGF0aW9uIG9mIHRoZSB0d28tY29tcG9uZW50IHN5c3RlbXMgaW4gTG90dXMgamFwb25pY2FzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIzNy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzEvMzkvMTA3NDU3Ni9kc3EwMzMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBwc2V1ZG8tcmVzcG9uc2UgcmVndWxhdG9ycyAoUFJScykgYXJlIHRoZSBjaXJjYWRpYW4gY2xvY2sgY29tcG9uZW50IHByb3RlaW5zIGluIHRoZSBtb2RlbCBkaWNvdCBBcmFiaWRvcHNpcyB0aGFsaWFuYS4gVGhleSBjb250YWluIGEgcmVjZWl2ZXItbGlrIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNzgwNDQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2Rk48fWcCi1b/SAZZR8y5mIcdXd5Pg=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f39%2f378044" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f39%2f378044" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=1509238734&amp;Signature=AoO0tuj2tAFPJEhEbdU1aNrZ2KIHI5pU5pg5G2b0C0gYJv2lzuWudGD9JAYU8gBJsP82Yy3SzG1qEnRlOQtv9A~RqFQ67CSjDvfwruKheZQu7XO~yrwTNygpQSRq~nSs2UfPUQVsarYiFcq85-4JoM2Q0LuNEEFW3J0xDL48K92kw7aSoDupSnbEO06u7juCgglbo0ZMvBVSK6t~vTa4OBWSGqxOZAHA0LHsGufhT5HXiQJcckXNSb2KSNHNFNFy5u~fokx7nuo46D1LZKLYsd1zkwxYStPhyZJwCMAUn1WsXJFCfCy0lue2AfbD7z8Ibea~V6whJU0wTnnRSxpBhA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/1">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5519668">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519668">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519670">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519670">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519676">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519676">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519695">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519695">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519742">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519742">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519769">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519769">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519773">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519773">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519776">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519776">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5519778">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5519778">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/1/31/377154">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/1/53/377009">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Pseudo-Response Regulator (PRR) Homologues of the Moss <em>Physcomitrella patens</em>: Insights into the Evolution of the PRR Family in Land Plants
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Santosh B. Satbhai</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Santosh B. Satbhai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Santosh+B.+Satbhai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satbhai S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Satbhai author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Takafumi Yamashino</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takafumi Yamashino</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Agriculture</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-52-789-4090. Fax. +81-52-789-4091. E-mail: <a href="mailto:yamasino@agr.nagoya-u.ac.jp" target="_blank">yamasino@agr.nagoya-u.ac.jp</a> (T.Y.); Tel/Fax. +81-52-789-5376. E-mail: <a href="mailto:aoki@is.nagoya-u.ac.jp" target="_blank">aoki@is.nagoya-u.ac.jp</a> (S.A.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Takafumi+Yamashino">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashino T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yamashino author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Ryo Okada</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryo Okada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ryo+Okada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okada R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Okada author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Yuji Nomoto</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Nomoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Agriculture</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yuji+Nomoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nomoto Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Nomoto author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Takeshi Mizuno</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Mizuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Agriculture</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Takeshi+Mizuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mizuno T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Mizuno author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Yuki Tezuka</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Tezuka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yuki+Tezuka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tezuka Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Tezuka author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Tomonori Itoh</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomonori Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Tomonori+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Itoh author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Mitsuru Tomita</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuru Tomita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mitsuru+Tomita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tomita M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Tomita author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Susumu Otsuki</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Susumu Otsuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Susumu+Otsuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Otsuki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Otsuki author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Setsuyuki Aoki</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Setsuyuki Aoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya 464-8601</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-52-789-4090. Fax. +81-52-789-4091. E-mail: <a href="mailto:yamasino@agr.nagoya-u.ac.jp" target="_blank">yamasino@agr.nagoya-u.ac.jp</a> (T.Y.); Tel/Fax. +81-52-789-5376. E-mail: <a href="mailto:aoki@is.nagoya-u.ac.jp" target="_blank">aoki@is.nagoya-u.ac.jp</a> (S.A.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Setsuyuki+Aoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aoki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Aoki author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 39–52, <a href='https://doi.org/10.1093/dnares/dsq033'>https://doi.org/10.1093/dnares/dsq033</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 December 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 September 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 November 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="378044" href="/dnaresearch/article-pdf/18/1/39/1074576/dsq033.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Santosh B. Satbhai, Takafumi Yamashino, Ryo Okada, Yuji Nomoto, Takeshi Mizuno, Yuki Tezuka, Tomonori Itoh, Mitsuru Tomita, Susumu Otsuki, Setsuyuki Aoki;  Pseudo-Response Regulator (PRR) Homologues of the Moss <em>Physcomitrella patens</em>: Insights into the Evolution of the PRR Family in Land Plants, <em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 39–52, <a href='https://doi.org/10.1093/dnares/dsq033'>https://doi.org/10.1093/dnares/dsq033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/378044?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/378044?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/378044?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/378044?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/378044?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/378044?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/378044?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Pseudo-Response%20Regulator%20(PRR)%20Homologues%20of%20the%20Moss%20%26lt%3Bem%26gt%3BPhyscomitrella%20patens%26lt%3B%2Fem%26gt%3B%3A%20Insights%20into%20the%20Evolution%20of%20the%20PRR%20Family%20in%20Land%20Plants&amp;publicationDate=2010-12-24&amp;volumeNum=18&amp;issueNum=1&amp;author=Satbhai%2C%20Santosh%20B.%3B%20Yamashino%2C%20Takafumi&amp;startPage=39&amp;endPage=52&amp;contentId=10.1093%2Fdnares%2Fdsq033&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f39%2f378044" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5519668 id="5519668" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The pseudo-response regulators (PRRs) are the circadian clock component proteins in the model dicot <em>Arabidopsis thaliana</em>. They contain a receiver-like domain (RLD) similar to the receiver domains of the RRs in the His–Asp phosphorelay system, but the RLDs lack the phosphoacceptor aspartic acid residue invariably conserved in the receiver domains. To study the evolution of <em>PRR</em> genes in plants, here we characterize their homologue genes, <em>PpPRR1</em>, <em>PpPRR2</em>, <em>PpPRR3</em> and <em>PpPRR4</em>, from the moss <em>Physcomitrella patens</em>. In the phylogenetic analysis, <em>PpPRRs</em> cluster together, sister to an angiosperm <em>PRR</em> gene subfamily, illustrating their close relationships with the angiosperm <em>PRRs</em>. However, distinct from the angiosperm sequences, the RLDs of PpPRR2/3/4 exhibit a potential phosphoacceptor aspartic acid–aspartic acid–lysine (DDK) motif. Consistently, the PpPRR2 RLD had phosphotransfer ability <em>in vitro</em>, suggesting that PpPRR2 functions as an RR. The PpPRR1 RLD, on the other hand, shows a partially diverged DDK motif, and it did not show phosphotransfer ability. All <em>PpPRRs</em> were expressed in a circadian and light-dependent manner, with differential regulation between <em>PpPRR2</em>/<em>4</em> and <em>PpPRR1</em>/<em>3</em>. Altogether, our results illustrate that PRRs originated from an RR(s) and that there are intraspecific divergences among PpPRRs. Finally, we offer scenarios for the evolution of the <em>PRR</em> family in land plants.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;circadian clock&quot;">circadian clock</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;His–Asp phosphorelay system&quot;">His–Asp phosphorelay system</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;response regulator&quot;">response regulator</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;pseudo-response regulator&quot;">pseudo-response regulator</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Physcomitrella patens&quot;"><em>Physcomitrella patens</em></a></div>                            <h2 scrollto-destination=5519670 id="5519670" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Circadian rhythms are endogenous biological oscillations with a period of 1 day, and they are controlled by an autonomous oscillator, the circadian clock.<sup><a reveal-id="DSQ033C1" data-open="DSQ033C1" class="link link-ref link-reveal xref-bibr">1</a></sup> This clock regulates the timing of metabolism, physiology and behaviour of organisms, coordinating them with environmental factors that cycle with the rotation of the earth.<sup><a reveal-id="DSQ033C1" data-open="DSQ033C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ033C2" data-open="DSQ033C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The mechanisms of the eukaryotic clocks are proposed to be founded on interlocked autoregulatory loops between genes that function as components of the clock machinery, whereas the identities of these genes are largely different between animals, plants and fungi.<sup><a reveal-id="DSQ033C3" data-open="DSQ033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In the model dicot <em>Arabidopsis thaliana</em>, a representative set of such component genes is the <em>pseudo-response regulator</em> (<em>PRR</em>) gene family, which comprises five member genes, <em>TOC1</em> (also called <em>PRR1</em>)<em>/PRR3/PRR5/PRR7/PRR9</em>, which play regulatory roles at multiple nodes in the interlocked loops of the <em>A. thaliana</em> circadian network.<sup><a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a></sup> All the <em>PRR</em> genes are, largely based on phenotypic analyses of mutants, supposed to be functionally important in the <em>A. thaliana</em> circadian system.<sup><a reveal-id="DSQ033C5" data-open="DSQ033C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Although phenotypic changes in a single mutation of each <em>PRR</em> gene are not large, combinations of mutations of different <em>PRR</em>s often result in stronger phenotypes, e.g. essentially arrhythmic in an extreme case.<sup><a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>PRRs share a conserved domain, the receiver-like domain (RLD), along with another domain CONSTANS/CONSTANS-LIKE/TOC1 (CCT). The RLD is similar to the receiver domain of the RRs in the histidine to aspartic acid (His–Asp) phosphorelay, a versatile signal transduction system in organisms from bacteria to eukaryotes other than animals.<sup><a reveal-id="DSQ033C5" data-open="DSQ033C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In the His–Asp phosphorelay, a phosphate group is transferred from a histidine kinase (HK), via an intermediate signal transducer histidine-containing phosphotransmitter (HPt), down to a counterpart RR, thereby transducing various environmental and endogenous signals intracellularly.<sup><a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> An aspartic acid residue in the receiver domain of RRs is conserved as the phosphoacceptor site,<sup><a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> whereas the RLDs of all the <em>A. thaliana</em> PRRs lack this aspartic acid residue, and they carry a glutamic acid instead.<sup><a reveal-id="DSQ033C5" data-open="DSQ033C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Consistent with this, Makino <em>et al</em>.<sup><a reveal-id="DSQ033C7" data-open="DSQ033C7" class="link link-ref link-reveal xref-bibr">7</a></sup> showed that the RLD of TOC1 did not undergo phosphotransfer <em>in vitro</em>.</p><p>Here, we hypothesize that the ancestors of PRRs were authentic RRs and they have lost the phosphorelay function through the course of evolution. If so, it is of particular interest to know in what evolutionary scenario PRRs lost the phosphorelay function, in order to understand the evolution of plant clock machineries. Homologue sequences of the <em>A. thaliana PRR</em> genes have recently been characterized in several other angiosperms.<sup><a reveal-id="DSQ033C8  DSQ033C15" data-open="DSQ033C8  DSQ033C15" class="link link-ref link-reveal xref-bibr">8–15</a></sup> Importantly, Corellou <em>et al</em>.<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> recently showed that the green alga <em>Ostreococcus tauri</em> has a <em>PRR</em> homologue sequence (<em>OtTOC1</em>) that functions as a master clock gene. The OtTOC1 protein carries a potential phosphoacceptor aspartic acid in its RLD,<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> indicating that the substitution of the aspartic acid to the glutamic acid is likely to date back to the period between the emergence of the green alga and the divergence of angiosperms. It would be informative, therefore, to characterize their homologues from non-angiosperm land plants, which cover a wide spectrum of phylogenetic groups.<sup><a reveal-id="DSQ033C17" data-open="DSQ033C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSQ033C18" data-open="DSQ033C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p><em>Physcomitrella patens</em>, a species of Bryopsida (moss), one of the basal land plants,<sup><a reveal-id="DSQ033C17" data-open="DSQ033C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSQ033C18" data-open="DSQ033C18" class="link link-ref link-reveal xref-bibr">18</a></sup> diverged from vascular plant lineages at least 450 million years ago.<sup><a reveal-id="DSQ033C18" data-open="DSQ033C18" class="link link-ref link-reveal xref-bibr">18</a></sup> This moss is an attractive model plant because various molecular biology techniques such as targeted gene disruption have been well established.<sup><a reveal-id="DSQ033C19" data-open="DSQ033C19" class="link link-ref link-reveal xref-bibr">19</a></sup> In a recent study, we isolated and characterized two <em>P. patens</em> cDNAs <em>PpCCA1a</em> and <em>PpCCA1b</em> encoding moss homologues of <em>A. thaliana</em> CCA1/LHY. CCA1/LHY is a pair of paralogous single myb proteins, which function as another type of important component proteins in the <em>A. thaliana</em> circadian network.<sup><a reveal-id="DSQ033C3" data-open="DSQ033C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Disruption experiments on <em>PpCCA1a</em> and <em>PpCCA1b</em> genes indicate that these two genes are functional counterparts of <em>CCA1/LHY</em>.<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We also identified four candidate genes that encode PRR homologues in the <em>P. patens</em> genome database.<sup><a reveal-id="DSQ033C20  DSQ033C22" data-open="DSQ033C20  DSQ033C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup> We isolated the full-length cDNA for one of these four moss <em>PRR</em> genes and showed that this gene is expressed in a circadian manner.<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Very recently, Holm <em>et al</em>.<sup><a reveal-id="DSQ033C23" data-open="DSQ033C23" class="link link-ref link-reveal xref-bibr">23</a></sup> reported a survey of clock-associated genes on the <em>P. patens</em> genome database, including the four moss <em>PRRs</em>. They constructed an unrooted tree of plant PRR homologues, in which the four moss PRRs clustered independently from angiosperm PRR subfamilies.<sup><a reveal-id="DSQ033C23" data-open="DSQ033C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>In this study, we isolated and characterized cDNAs for the remaining three moss <em>PRR</em> genes, for which gene structures were so far predicted only from the genomic sequences. Using experimentally validated sequences of the moss <em>PRR</em> cDNAs, we constructed a rooted phylogenetic tree, thereby clearly defining the evolution of plant PRRs. We also conducted biochemical characterization and expression analyses of the moss <em>PRRs</em>; the results of these experiments suggest that at least one of the moss PRRs functions as an authentic RR and that there are intraspecific divergences among the moss PRRs. Finally, we will discuss scenarios of the evolution and divergence of <em>PRR</em> homologue genes in land plants.</p>                            <h2 scrollto-destination=5519676 id="5519676" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5519677 id="5519677" class="section-title" data-legacy-id=s2a>2.1. Plant materials, growth conditions and light treatment</h3>
<p><em>Physcomitrella patens</em> ssp. <em>patens</em><sup><a reveal-id="DSQ033C24" data-open="DSQ033C24" class="link link-ref link-reveal xref-bibr">24</a></sup> was maintained in 12-h light and 12-h dark cycles (12:12LD) under white fluorescent light (light intensity ∼40 μmol m<sup>−2</sup> s<sup>−1</sup>) at 25°C. Protonemal and gametophore tissues were grown on BCDAT/G medium and BCD medium, respectively, both supplemented with 1 mM CaCl<sub>2</sub>.<sup><a reveal-id="DSQ033C25" data-open="DSQ033C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Protonemal cells were collected every 5–7 days and were ground with a homogenizer before application to new BCDAT agar plates. For light sources of the light induction experiments, the light-emitting diodes (STICK LED, Tokyo Rikakikai) were used for blue light (<em>λ</em><sub>max</sub> = 470 nm) and the red-emitting fluorescent tubes (FL20S-Re-66, Toshiba Lighting &amp; Technology) filtered through a red plastic sheet (Acrylite102, Mitsubishi Rayon) for red light (<em>λ</em><sub>max</sub> = 660 nm). White light was provided by fluorescence lumps (FL20SS-W/18, Toshiba Lighting &amp; Technology).</p>                            <h3 scrollto-destination=5519679 id="5519679" class="section-title" data-legacy-id=s2b>2.2. Identification and isolation of cDNAs covering the entire coding regions of PpPRRs</h3>
<p>The 5′- and 3′-terminal portions of the <em>PpPRR2</em> and <em>PpPRR4</em> cDNAs were RACE-amplified using GeneRacer (Invitrogen) with primers based on the JGI database sequences. The amplified cDNA fragments were cloned into the pGEM-T Easy vector (Promega) and sequenced with DYEenamic ET terminator [GE Healthcare (Former Amersham Biosciences)]. The middle region of each gene was amplified by RT-PCR with primers based on the RACE-amplified sequences. The entire regions of both cDNAs were amplified with KOD plus polymerase Ver.2 (TOYOBO), subjected to A-tailing by <em>Taq</em> polymerase (TAKARA Bio), cloned into the pGEM-T Easy vector and sequenced using a primer walking method. The <em>PpPRR3</em> cDNA that spans its entire coding region was amplified with KOD plus polymerase Ver.2 using primers based on the JGI database sequences. Nucleotide sequences were assembled by DNASIS software (Hitachi software engineering). All primer sets used are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 1</a>.</p>                            <h3 scrollto-destination=5519681 id="5519681" class="section-title" data-legacy-id=s2c>2.3. Phylogenetic analyses</h3>
<p>Amino acid sequences of PRR homologues were aligned using the ClustalW program<sup><a reveal-id="DSQ033C26" data-open="DSQ033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and the numbers of amino acid substitutions between each pair of PRR proteins were estimated by the Jones–Taylor–Thornton (JTT) model<sup><a reveal-id="DSQ033C27" data-open="DSQ033C27" class="link link-ref link-reveal xref-bibr">27</a></sup> with the complete-deletion option. From the estimated numbers of amino acid substitutions, a phylogenetic tree was reconstructed using the minimum evolution (ME) method.<sup><a reveal-id="DSQ033C28" data-open="DSQ033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The bootstrap values were calculated with 1000 replications.<sup><a reveal-id="DSQ033C29" data-open="DSQ033C29" class="link link-ref link-reveal xref-bibr">29</a></sup> These procedures were all performed using MEGA4.1 software (<a class="link link-uri" href="http://www.megasoftware.net/index.html">http://www.megasoftware.net/index.html</a><sup><a reveal-id="DSQ033C30" data-open="DSQ033C30" class="link link-ref link-reveal xref-bibr">30</a></sup>). We also reconstructed a phylogenetic tree by the maximum-likelihood method using PhyML (<a class="link link-uri" href="http://atgc.lirmm.fr/phyml/">http://atgc.lirmm.fr/phyml/</a><sup><a reveal-id="DSQ033C31" data-open="DSQ033C31" class="link link-ref link-reveal xref-bibr">31</a></sup>) applying the JTT model for amino acid substitutions. We obtained a similar phylogeny pattern with both the methods.</p>                            <h3 scrollto-destination=5519683 id="5519683" class="section-title" data-legacy-id=s2d>2.4. Semi-quantitative and quantitative RT-PCR analyses</h3>
<p>Semi-quantitative RT-PCR (sqRT-PCR) analysis was performed as described previously.<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We conducted preliminary experiments to improve its quantitativeness as follows. First, thermal cycle numbers were optimized so that signals from PCR products did not reach a plateau. Next, we confirmed that when various known relative amounts of cDNA for each tested gene were used as PCR templates, the amount of the PCR product of each reaction showed a linear relationship with that of the input cDNA. The primers and optimal cycle numbers for PCR are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 1</a>. Quantitative real-time PCR (qRT-PCR) analysis was performed as described previously.<sup><a reveal-id="DSQ033C32" data-open="DSQ033C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The primers are same as those used in the qRT-PCR analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 1</a>).</p>                            <h3 scrollto-destination=5519685 id="5519685" class="section-title" data-legacy-id=s2e>2.5. Construction of plasmids</h3>
<p>To express the PpPRR1 and PpPRR2 RLD peptides in <em>E. coli</em> cells, a cold shock expression system (pCold-II vector; TAKARA Bio) was used as follows. The coding region for the RLD of each PpPRR protein was PCR-amplified with primers described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 1</a>. The resultant fragment was cloned into the pGEM-T Easy vector, digested with <em>Kpn</em>I and <em>Xba</em>I and inserted into <em>Kpn</em>I–<em>Xba</em>I-cleaved pCold-II vector (pCold-II-His-PRR1 and pCold-II-His-PRR2 for PpPRR1 RLD and PpPRR2 RLD, respectively). The nucleotide sequence was determined by DYEenamic ET terminator. The RLD peptides are fused with a 6×His-tag, by expression from pCold-II vector, allowing their purification with a TALON™ metal affinity resin column as described below.</p>                            <h3 scrollto-destination=5519687 id="5519687" class="section-title" data-legacy-id=s2f>2.6. Expression and purification of the PpPRR1 and PpPRR2 RLD peptides</h3>
<p>The <em>E. coli</em> strain BL21 (DE3) harbouring pG-KJE8 (TAKARA Bio), which encodes chaperone proteins DnaK, DnaJ, GrpE, GroEL and GroES, was transformed with pCold-II-His-PRR1 or pCold-II-His-PRR2. To overproduce the RLD peptides of PpPRR1 and PpPRR2, each transformant was cultivated in 1 l of LB medium containing ampicillin (50 μg ml<sup>−1</sup>), Chloramphenicol (25 μg ml<sup>−1</sup>), tetracycline (20 ng ml<sup>−1</sup>) and <span class="small-caps">l</span>-arabinose (10 mg ml<sup>−1</sup>) at 37°C until the logarithmic phase of growth in a rotary shaker (at 110 r.p.m.). They were cold-shocked by standing at 15°C for 30 min after the addition of IPTG at the final concentration of 1 mM and cultured in a shaker (at 110 r.p.m.) at 15°C for 5 h. Cells were collected by centrifugation and suspended in 20 ml of a standard buffer [50 mM Tris–HCl (pH 8.0), 100 mM NaCl and 10% glycerol]. The cell suspension was mixed with DNase I (25 µg ml<sup>−1</sup>), EDTA (1.25 mM) and lysozyme (250 µg ml<sup>−1</sup>) at 4°C for 15 min, and then passed through a French Press at 100 MPa. The resultant cell lysate was centrifuged and separated from cell debris and then applied onto a TALON™ metal affinity resin (Clontech Laboratories) column. After washing the TALON™ column with 50 mM Tris–HCl (pH 8.0), 10% glycerol and 100 mM NaCl, each of 6×His-tagged PpPRR1 and PpPRR2 RLD peptides was eluted with the same buffer including 250 mM imidazole and finally dialysed against 50 mM Tris–HCl (pH 8.0) and 10% glycerol. Dialysed semi-purified protein was concentrated by loading on Amicon Ultra Filter (Ultracel-10K, Millipore) and centrifuged for 15 min at 4700 r.p.m. at 4°C. The sample was quantified by the Lowry method<sup><a reveal-id="DSQ033C33" data-open="DSQ033C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and subjected to SDS–PAGE and detected with Coomassie Brilliant Blue. It was also analysed by western blotting with anti-6×His antibodies (Cat. No. A190-114A, Bethyl Laboratories).</p>                            <h3 scrollto-destination=5519689 id="5519689" class="section-title" data-legacy-id=s2g>2.7. Preparation of the <em>E. coli</em> ArcB-enriched cytoplasmic membrane</h3>
<p><em>Escherichia coli</em> DAC903/pIA001-ArcB and DAC903/pIN-III were used as an ArcB overproducer and a negative control, respectively.<sup><a reveal-id="DSQ033C34" data-open="DSQ033C34" class="link link-ref link-reveal xref-bibr">34</a></sup> From each strain, membrane vesicles were isolated and pelleted as an insoluble fraction of the cell lysate according to the method of Azuma <em>et al</em>.<sup><a reveal-id="DSQ033C35" data-open="DSQ033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> They were suspended in a small volume of a buffer comprising 50 mM Tris-acetate (pH 8.0), 1 mM DTT, 2% glycerol and 250 mM sucrose in order to isolate cytoplasmic membrane as follows. The suspended membrane vesicles were layered over a 21-ml five steps gradient of sucrose dissolved in the same buffer (3 ml of 50%, 9 ml of 45%, 3 ml of 40%, 3 ml of 35% and 3 ml of 30% from bottom to top) in a bottle assembly polycarbonate tube (part no. 355618, Beckman Coulter) and centrifuged at 47 000 r.p.m. for 2 h at 4̊C with the type 50.2 Ti rotor (Beckman Coulter). The cytoplasmic membrane was collected by taking the fraction between the layers of 35 and 40% sucrose and diluted with a large volume of a buffer (T<sub>A</sub>GS-buffer) comprising 50 mM Tris-acetate (pH 7.8), 1 mM DTT, 10% glycerol, 100 mM sucrose. The cytoplasmic membrane was collected by centrifugation at 127 000<em>g</em> for 2 h and then resuspended in a small volume of T<sub>A</sub>GS-buffer. Finally, F<sub>1</sub>-ATPase was stripped from the cytoplasmic membrane as follows. The isolated cytoplasmic membrane (2 mg of protein) was treated with 1 ml of T<sub>A</sub>GS-buffer containing 4 M urea for 30 min on ice, and then recovered by centrifugation at 50 000 r.p.m. for 30 min. Then, the pellet was washed once with 1 ml of TGS buffer and finally with 1 ml of a buffer comprising 50 mM Tris–HCl (pH 8.0), 2 mM DTT and 10% glycerol. The urea-treated cytoplasmic membrane was suspended in 0.5 ml of the same buffer and stored at −80°C.</p>                            <h3 scrollto-destination=5519691 id="5519691" class="section-title" data-legacy-id=s2h>2.8. <em>In vitro</em> autophosphorylation assay</h3>
<p><em>In vitro</em> autophosphorylation reaction of the ArcB protein was carried out according to the method of Azuma <em>et al</em>.,<sup><a reveal-id="DSQ033C35" data-open="DSQ033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> 4.0 µg of ArcB-enriched <em>Escherichia coli</em> cytoplasmic membrane was incubated with 0.05 mM γ-[<sup>32</sup>P]-ATP (37 kBq) in TEG buffer (50 mM Tris–HCl, pH 8.0, 0.35 mM EDTA, 10% glycerol) containing 5 mM MgCl<sub>2</sub>, 200 mM KCl and 2 mM DTT at 25°C for 1, 5 and 10 min. Four micrograms of <em>E. coli</em> cytoplasmic membrane, which was prepared from the strain which does not overproduce ArcB, was also used as a negative control experiment. The reaction was terminated by the addition of SDS–PAGE sample buffer [finally 20 mM Tris–HCl (pH 8.0), 1% β-mercaptoethanol, 1% SDS, 6% glycerol and 0.02% Bromophenol Blue]. The samples were subjected to SDS–PAGE. The gel was dried and analysed with an imaging scanner (BAS-2500; Fuji Film).</p>                            <h3 scrollto-destination=5519693 id="5519693" class="section-title" data-legacy-id=s2i>2.9. <em>In vitro</em> His–Asp phosphotransfer assay</h3>
<p><em>In vitro</em> His–Asp phosphotransfer reaction from ArcB to the RLD peptides of PpPRR1 and PpPRR2 was examined according to the method of Azuma <em>et al</em>.<sup><a reveal-id="DSQ033C35" data-open="DSQ033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Four micrograms of <em>E. coli</em> ArcB-enriched cytoplasmic membrane was incubated with 0.05 mM γ-[<sup>32</sup>P]-ATP (37 kBq) and 4.5 µg of semi-purified PpPRR1 or PpPRR2 RLD peptide in TEG buffer (50 mM Tris–HCl, pH 8.0, 0.35 mM EDTA, 10% glycerol) containing 5 mM MgCl<sub>2</sub>, 200 mM KCl and 2 mM DTT at 25°C for 1, 5 and 10 min. The reaction was terminated by the addition of SDS–PAGE sample buffer. The samples were subjected to SDS–PAGE. The gel was dried and analysed with BAS-2500.</p>                            <h2 scrollto-destination=5519695 id="5519695" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5519696 id="5519696" class="section-title" data-legacy-id=s3a>3.1. Phylogenetic analyses of the moss PpPRR genes</h3>
<p>Previously, we identified four <em>PRR</em> homologue genes, <em>PpPRR1</em> (protein ID in the JGI <em>P. patens</em> database: 154145), <em>PpPRR2</em> (165025), <em>PpPRR3</em> (173125) and <em>PpPRR4</em> (165029) in the JGI genome database.<sup><a reveal-id="DSQ033C21" data-open="DSQ033C21" class="link link-ref link-reveal xref-bibr">21</a></sup> We characterized the <em>PpPRR1</em> gene (and its corresponding cDNA) as ‘<em>PpPRRa</em>' before,<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> but in this paper, we use the name ‘<em>PpPRR1</em>', not <em>PpPRRa</em>, to avoid confusion. Holm <em>et al</em>. also followed this nomenclature (<em>PpPRR1–PpPRR4</em>) deposited in the JGI genome database.<sup><a reveal-id="DSQ033C21" data-open="DSQ033C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Other than these four sequences, we found no gene models that are significantly similar to the angiosperm <em>PRR</em> genes in the moss genome. We isolated cDNAs that cover the entire coding regions for the <em>PpPRR2</em>, <em>PpPRR3</em> and <em>PpPRR4</em> genes and confirmed that all of their predicted proteins share, as do <em>PpPRR1</em>,<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> an N-terminal RLD and a C-terminal CCT domain (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a and b; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 2</a>). The receiver domains of authentic RRs share the aspartic acid–aspartic acid–lysine (DDK) motif, essential for the phosphotransfer activity,<sup><a reveal-id="DSQ033C36" data-open="DSQ033C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSQ033C37" data-open="DSQ033C37" class="link link-ref link-reveal xref-bibr">37</a></sup> whereas the first two residues of this motif are diverged in the angiosperm PRR proteins (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a; for review, see Mizuno and Nakamichi<sup><a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and Mizuno<sup><a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup>). In all the available angiosperm PRR sequences, the second aspartic acid, conserved as the phosphoacceptor residue in the His–Asp phosphotransfer, is replaced by a glutamic acid (E) (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a). In contrast, RLDs of moss PpPRR proteins are all predicted to retain the second aspartic acid; furthermore, PpPRR2, PpPRR3 and PpPRR4 also retain the first aspartic acid, hence exhibiting the complete DDK motif (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a). PpPRR1 replaces the first aspartic acid residue with a tyrosine (Y) (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a). These observations suggest that at least PpPRR2, PpPRR3 and PpPRR4 are potentially phosphorylated by a HK(s) as authentic RRs. The CCT domain is well conserved in all PpPRR proteins (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>b). </p>                        <a id="5519699" scrollto-destination="5519699"></a>
<div id="DSQ033F1" data-id="DSQ033F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03301.jpeg?Expires=1509238741&Signature=V-TLmXMhP5074URD4lF5A7LXsF7z8evXXivlDsIweOr~3mxUG9Z0BLVBGIZ45tnnd6Fpe9uGOTcDb7hvFBgkhiwwuuUCOYcJ6Nuun296Mp1MS0-ksQOz5GDHYQ6Sh7CLk59tM-a1p--vu~eLWF9c9fqEDggUi~0Fs3d3YrK8osv4mzvgpOr2velymyBJx-Z3M86I7mTrUnC2PELdp3A1vb0UK9soQlS7cddS3Ir3RQHGKhA~S1X3nyIPYl3FUtrN94qKpFpEhFItUM9IlcW9u8rYuGCJUamb7b1DZObRQo~h8QoyrjTN~4GLtB4t2jvDxGttr4Bn-eu8v1NDaKU0gg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment of conserved domains of PRR homologues from various species. RLDs (a) and CCT domains (b) of PRR homologue sequences from various plant species are aligned using ClustalW program.26 Shading was performed using BOXSHADE ver 3.21 (http://www.ch.embnet.org/software/BOX_form.html). Amino acids identical or similar in more than 70% of the sequences are shaded by black or grey background, respectively. The number at the end of each line indicates the rightmost amino acid. Arrowheads show amino acids corresponding to the DDK motif. The PRR homologue sequences are as follows: PpPRR1, PpPRR2, PpPRR3 and PpPRR4 from P. patens; SmPRR7a (450934, protein ID in the JGI Selaginella moellendorffii database), SmPRR7b (450936), SmTOC1 (438647) from S. moellendorffii; OtTOC1 (AY740079 in DDBJ/EMBL/GenBank) from O. tauri; AtPRR3 (BAB13744), AtPRR5 (BAB13743), AtPRR7 (BAB13742) and AtPRR9 (BAB13741) from A. thaliana." path-from-xml="dsq03301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519699/dsq03301.jpeg" path-from-xml="dsq03301.jpeg" target="_blank">View large</a><a section="5519699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03301.jpeg?Expires=1509238741&Signature=Femb7py66nVxv-a9D5ZEzuq5lh3GIkGspPSINtdGoMGUyTDrqxy-hIyH4tjqT-j6CNHewEBHTfMLEJUocNQg7aEv32iS--jGn-nIHxnSXidTuoO0yyagN0eBBcOvPvJmu6wdb1kEcXi19hgZ9vKtwPhMEFWmoU7yw~HgZIblLMS5XhNynLEA5tm9yAliouJlLN0eTfhhUQAzIpHo0o865emZqEoeiqz~wbo0Jn-k~roBpBuV~p-ZWMmuBXIX6Zs9FBu0RKLuorBjWtvMt-71WzffTFNBdhiAfnKCIJCoFCyfleI2ffrLsEWGG-bVObXsYnX5a0ElV4zhL~yYLPhsrw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519699&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of conserved domains of PRR homologues from various species. RLDs (a) and CCT domains (b) of PRR homologue sequences from various plant species are aligned using ClustalW program.<sup><a reveal-id="DSQ033C26" data-open="DSQ033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Shading was performed using BOXSHADE ver 3.21 (<a class="link link-uri" href="http://www.ch.embnet.org/software/BOX_form.html">http://www.ch.embnet.org/software/BOX_form.html</a>). Amino acids identical or similar in more than 70% of the sequences are shaded by black or grey background, respectively. The number at the end of each line indicates the rightmost amino acid. Arrowheads show amino acids corresponding to the DDK motif. The PRR homologue sequences are as follows: PpPRR1, PpPRR2, PpPRR3 and PpPRR4 from <em>P. patens</em>; SmPRR7a (450934, protein ID in the JGI <em>Selaginella moellendorffii</em> database), SmPRR7b (450936), SmTOC1 (438647) from <em>S. moellendorffii</em>; OtTOC1 (AY740079 in DDBJ/EMBL/GenBank) from <em>O. tauri</em>; AtPRR3 (BAB13744), AtPRR5 (BAB13743), AtPRR7 (BAB13742) and AtPRR9 (BAB13741) from <em>A. thaliana</em>.</p></div></div><div content-id="DSQ033F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03301.jpeg?Expires=1509238741&Signature=V-TLmXMhP5074URD4lF5A7LXsF7z8evXXivlDsIweOr~3mxUG9Z0BLVBGIZ45tnnd6Fpe9uGOTcDb7hvFBgkhiwwuuUCOYcJ6Nuun296Mp1MS0-ksQOz5GDHYQ6Sh7CLk59tM-a1p--vu~eLWF9c9fqEDggUi~0Fs3d3YrK8osv4mzvgpOr2velymyBJx-Z3M86I7mTrUnC2PELdp3A1vb0UK9soQlS7cddS3Ir3RQHGKhA~S1X3nyIPYl3FUtrN94qKpFpEhFItUM9IlcW9u8rYuGCJUamb7b1DZObRQo~h8QoyrjTN~4GLtB4t2jvDxGttr4Bn-eu8v1NDaKU0gg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment of conserved domains of PRR homologues from various species. RLDs (a) and CCT domains (b) of PRR homologue sequences from various plant species are aligned using ClustalW program.26 Shading was performed using BOXSHADE ver 3.21 (http://www.ch.embnet.org/software/BOX_form.html). Amino acids identical or similar in more than 70% of the sequences are shaded by black or grey background, respectively. The number at the end of each line indicates the rightmost amino acid. Arrowheads show amino acids corresponding to the DDK motif. The PRR homologue sequences are as follows: PpPRR1, PpPRR2, PpPRR3 and PpPRR4 from P. patens; SmPRR7a (450934, protein ID in the JGI Selaginella moellendorffii database), SmPRR7b (450936), SmTOC1 (438647) from S. moellendorffii; OtTOC1 (AY740079 in DDBJ/EMBL/GenBank) from O. tauri; AtPRR3 (BAB13744), AtPRR5 (BAB13743), AtPRR7 (BAB13742) and AtPRR9 (BAB13741) from A. thaliana." path-from-xml="dsq03301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519699/dsq03301.jpeg" path-from-xml="dsq03301.jpeg" target="_blank">View large</a><a section="5519699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03301.jpeg?Expires=1509238741&Signature=Femb7py66nVxv-a9D5ZEzuq5lh3GIkGspPSINtdGoMGUyTDrqxy-hIyH4tjqT-j6CNHewEBHTfMLEJUocNQg7aEv32iS--jGn-nIHxnSXidTuoO0yyagN0eBBcOvPvJmu6wdb1kEcXi19hgZ9vKtwPhMEFWmoU7yw~HgZIblLMS5XhNynLEA5tm9yAliouJlLN0eTfhhUQAzIpHo0o865emZqEoeiqz~wbo0Jn-k~roBpBuV~p-ZWMmuBXIX6Zs9FBu0RKLuorBjWtvMt-71WzffTFNBdhiAfnKCIJCoFCyfleI2ffrLsEWGG-bVObXsYnX5a0ElV4zhL~yYLPhsrw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519699&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of conserved domains of PRR homologues from various species. RLDs (a) and CCT domains (b) of PRR homologue sequences from various plant species are aligned using ClustalW program.<sup><a reveal-id="DSQ033C26" data-open="DSQ033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Shading was performed using BOXSHADE ver 3.21 (<a class="link link-uri" href="http://www.ch.embnet.org/software/BOX_form.html">http://www.ch.embnet.org/software/BOX_form.html</a>). Amino acids identical or similar in more than 70% of the sequences are shaded by black or grey background, respectively. The number at the end of each line indicates the rightmost amino acid. Arrowheads show amino acids corresponding to the DDK motif. The PRR homologue sequences are as follows: PpPRR1, PpPRR2, PpPRR3 and PpPRR4 from <em>P. patens</em>; SmPRR7a (450934, protein ID in the JGI <em>Selaginella moellendorffii</em> database), SmPRR7b (450936), SmTOC1 (438647) from <em>S. moellendorffii</em>; OtTOC1 (AY740079 in DDBJ/EMBL/GenBank) from <em>O. tauri</em>; AtPRR3 (BAB13744), AtPRR5 (BAB13743), AtPRR7 (BAB13742) and AtPRR9 (BAB13741) from <em>A. thaliana</em>.</p></div></div><p>To examine phylogenetic relationships among <em>PpPRRs</em> and other land plant <em>PRRs</em>, we constructed a rooted phylogenetic tree using PRR homologue sequences from various plants (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>). We used the <em>PRR</em> homologue of the green alga <em>O. tauri</em> (<em>OtTOC1</em>)<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> as the outgroup because: (i) OtTOC1 is positioned relatively distant from any other PRR homologue sequences in an unrooted tree of PRRs<sup><a reveal-id="DSQ033C23" data-open="DSQ033C23" class="link link-ref link-reveal xref-bibr">23</a></sup>; and (ii) OtTOC1 is sister to all the other PRR homologue sequences when an authentic RR sequence is used as the outgroup (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 3</a>). Also included were three algal sequences: two from <em>C. reinhardtii</em> and one from <em>Chlorella variabilis</em>. In the phylogenetic tree, the angiosperm sequences are divided, as previously reported,<sup><a reveal-id="DSQ033C8" data-open="DSQ033C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ033C12" data-open="DSQ033C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSQ033C15" data-open="DSQ033C15" class="link link-ref link-reveal xref-bibr">15</a></sup> into three groups: TOC1, PRR7/3 and PRR9/5 (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>). The TOC1 group is basal to all the other sequences that include both PRR7/3 and PRR9/5 groups, indicating a more ancient origin of the TOC1 group than the other two groups. The four moss PpPRR sequences are grouped with one another, further forming a cluster with two lycophyte sequences, and this cluster is sister to the PRR7/3 group (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>). This indicates that the moss <em>PpPRR</em> genes are more closely related to the PRR7/3 group than to the two other groups. The four moss sequences are divided into two groups PpPRR1/PpPRR3 and PpPRR2/PpPRR4 (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="5519705" scrollto-destination="5519705"></a>
<div id="DSQ033F2" data-id="DSQ033F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03302.jpeg?Expires=1509238741&Signature=C4h2TbD0S9he2A2z0hwMGGFJm3FxYmb9wc5hEPb8PKk2WtvKTVMRL6BTlXvEmBZrM~Fvbu6Hr8hKtEgS0V5Ua8d9DUB4iZcmSxFO5OpGbuJPJvxb1UO4og4AMRiRoejLSdYgaqOqMN7WDw4ymrYdvFxzgMXtj3UR37lcLn70TcL85N4S~p4rJiUJ6K-xivRl48wdO50WPd91e4hsGvnFQjgYOMlUr1OMg~G-ReYf7bv-WKWRd9rDcaQclvl8p~Z3L3-BiWXRTGkjrSn1g7EQ6-F2jQi4vRxpc8271D0vDmzUIHytRbzfJDe2SaZCGygXvlAf87qYsP5mJ8fzkbGeQQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree of PRR homologues in various plants. Amino acid sequences of PRR homologues including entire RLD and CCT domain regions were aligned using ClustalW program and the phylogenetic tree was reconstructed by the ME method28 by using MEGA4.1.30 The O. tauri PRR homologue (OtTOC116) was used as the outgroup. The numbers at each node represent the bootstrap values calculated based on 1000 bootstrap sampling and those that are higher than 50% are shown. The three PRR subfamilies are indicated. PpPRRs from P. patens are indicated in bold. Sequences used are as follows: OsTOC1 (BAD38854 in DDBJ/EMBL/GenBank), OsPRR37 (BAD38855), OsPRR73 (BAD38856), OsPRR59 (AK120059) and OsPRR95 (BAD38857) from Oryza sativa; PtTOC1 (XP_002330130.1 in NCBI protein database), PtPRR37 (XP_002311123.1), PtPRR73 (XP_002316333.1), PtPRR5a (XP_002321349.1), PtPRR5b (XP_002301442.1), PtPRR9a (XP_002320232.1) and PtPRR9b (XP_002301443.1) from Populus trichocarpa; VvTOC1 (XP_002281757.1), VvPRR37 (XP_002281776), VvPRR73 (XP_002275645), VvPRR5 (XP_002270811) and VvPRR9 (XP_002266192.1) from Vitis vinifera; LjTOC1 (chr4.CM0087.600.nc in miyakogusa.jp; http://www.kazusa.or.jp/lotus/index.html), LjPRR3 (LjT08O17.180/130/120), LjPRR5 (chr1.CM0105.560), LjPRR7 (chr3.LjT05P05.60) and LjPRR9 (chr3.CM0208.230) from Lotus japonicas;42 CrPRR1 (XP_001695777.1), CrPRR2 (XP_001701808.1) from Chlamydomonas reinhardtii; CvPRR1 (EFN58892.1) from C. variabilis. For other sequences, refer to the legend for Fig. 1." path-from-xml="dsq03302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519705/dsq03302.jpeg" path-from-xml="dsq03302.jpeg" target="_blank">View large</a><a section="5519705" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03302.jpeg?Expires=1509238741&Signature=A1FzDtyYt0JKLjQ~061gUuO2RivAkjuvL3-btJXGVHhuD4pqtmrZo4kwz-Jw6UU8wDmoeJ-dPWkhEBYHD3GRYgwnqIOZR-7yqgPk1pdds24iU08d3xBfezxVKaRb6H2V4gaAGu7VS3h~76KLw6j5YpivSJ-YfNlwGm8n2JxU7mxoxkY-gssHb5FEJqeQ93eJ1NuFRgCzWzkCMXb~WsRIj4pG-L2oN9xJ9BLhjvXPtlZMrRkSswATk7n8SGChI0f7FdGsdZp0Ujr3CXwK1dHra23bGpx85Q7cHNsdxYm5FnVUfwOwUduglUjwH2uVjdadtem1AGThTPX8J3F4AKcLCQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519705&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of PRR homologues in various plants. Amino acid sequences of PRR homologues including entire RLD and CCT domain regions were aligned using ClustalW program and the phylogenetic tree was reconstructed by the ME method<sup><a reveal-id="DSQ033C28" data-open="DSQ033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> by using MEGA4.1.<sup><a reveal-id="DSQ033C30" data-open="DSQ033C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The <em>O. tauri</em> PRR homologue (OtTOC1<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup>) was used as the outgroup. The numbers at each node represent the bootstrap values calculated based on 1000 bootstrap sampling and those that are higher than 50% are shown. The three PRR subfamilies are indicated. PpPRRs from <em>P. patens</em> are indicated in bold. Sequences used are as follows: OsTOC1 (BAD38854 in DDBJ/EMBL/GenBank), OsPRR37 (BAD38855), OsPRR73 (BAD38856), OsPRR59 (AK120059) and OsPRR95 (BAD38857) from <em>Oryza sativa</em>; PtTOC1 (XP_002330130.1 in NCBI protein database), PtPRR37 (XP_002311123.1), PtPRR73 (XP_002316333.1), PtPRR5a (XP_002321349.1), PtPRR5b (XP_002301442.1), PtPRR9a (XP_002320232.1) and PtPRR9b (XP_002301443.1) from <em>Populus trichocarpa</em>; VvTOC1 (XP_002281757.1), VvPRR37 (XP_002281776), VvPRR73 (XP_002275645), VvPRR5 (XP_002270811) and VvPRR9 (XP_002266192.1) from <em>Vitis vinifera</em>; LjTOC1 (chr4.CM0087.600.nc in miyakogusa.jp; <a class="link link-uri" href="http://www.kazusa.or.jp/lotus/index.html">http://www.kazusa.or.jp/lotus/index.html</a>), LjPRR3 (LjT08O17.180/130/120), LjPRR5 (chr1.CM0105.560), LjPRR7 (chr3.LjT05P05.60) and LjPRR9 (chr3.CM0208.230) from <em>Lotus japonicas</em>;<sup><a reveal-id="DSQ033C42" data-open="DSQ033C42" class="link link-ref link-reveal xref-bibr">42</a></sup> CrPRR1 (XP_001695777.1), CrPRR2 (XP_001701808.1) from <em>Chlamydomonas reinhardtii</em>; CvPRR1 (EFN58892.1) from <em>C. variabilis</em>. For other sequences, refer to the legend for Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div><div content-id="DSQ033F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03302.jpeg?Expires=1509238741&Signature=C4h2TbD0S9he2A2z0hwMGGFJm3FxYmb9wc5hEPb8PKk2WtvKTVMRL6BTlXvEmBZrM~Fvbu6Hr8hKtEgS0V5Ua8d9DUB4iZcmSxFO5OpGbuJPJvxb1UO4og4AMRiRoejLSdYgaqOqMN7WDw4ymrYdvFxzgMXtj3UR37lcLn70TcL85N4S~p4rJiUJ6K-xivRl48wdO50WPd91e4hsGvnFQjgYOMlUr1OMg~G-ReYf7bv-WKWRd9rDcaQclvl8p~Z3L3-BiWXRTGkjrSn1g7EQ6-F2jQi4vRxpc8271D0vDmzUIHytRbzfJDe2SaZCGygXvlAf87qYsP5mJ8fzkbGeQQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree of PRR homologues in various plants. Amino acid sequences of PRR homologues including entire RLD and CCT domain regions were aligned using ClustalW program and the phylogenetic tree was reconstructed by the ME method28 by using MEGA4.1.30 The O. tauri PRR homologue (OtTOC116) was used as the outgroup. The numbers at each node represent the bootstrap values calculated based on 1000 bootstrap sampling and those that are higher than 50% are shown. The three PRR subfamilies are indicated. PpPRRs from P. patens are indicated in bold. Sequences used are as follows: OsTOC1 (BAD38854 in DDBJ/EMBL/GenBank), OsPRR37 (BAD38855), OsPRR73 (BAD38856), OsPRR59 (AK120059) and OsPRR95 (BAD38857) from Oryza sativa; PtTOC1 (XP_002330130.1 in NCBI protein database), PtPRR37 (XP_002311123.1), PtPRR73 (XP_002316333.1), PtPRR5a (XP_002321349.1), PtPRR5b (XP_002301442.1), PtPRR9a (XP_002320232.1) and PtPRR9b (XP_002301443.1) from Populus trichocarpa; VvTOC1 (XP_002281757.1), VvPRR37 (XP_002281776), VvPRR73 (XP_002275645), VvPRR5 (XP_002270811) and VvPRR9 (XP_002266192.1) from Vitis vinifera; LjTOC1 (chr4.CM0087.600.nc in miyakogusa.jp; http://www.kazusa.or.jp/lotus/index.html), LjPRR3 (LjT08O17.180/130/120), LjPRR5 (chr1.CM0105.560), LjPRR7 (chr3.LjT05P05.60) and LjPRR9 (chr3.CM0208.230) from Lotus japonicas;42 CrPRR1 (XP_001695777.1), CrPRR2 (XP_001701808.1) from Chlamydomonas reinhardtii; CvPRR1 (EFN58892.1) from C. variabilis. For other sequences, refer to the legend for Fig. 1." path-from-xml="dsq03302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519705/dsq03302.jpeg" path-from-xml="dsq03302.jpeg" target="_blank">View large</a><a section="5519705" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03302.jpeg?Expires=1509238741&Signature=A1FzDtyYt0JKLjQ~061gUuO2RivAkjuvL3-btJXGVHhuD4pqtmrZo4kwz-Jw6UU8wDmoeJ-dPWkhEBYHD3GRYgwnqIOZR-7yqgPk1pdds24iU08d3xBfezxVKaRb6H2V4gaAGu7VS3h~76KLw6j5YpivSJ-YfNlwGm8n2JxU7mxoxkY-gssHb5FEJqeQ93eJ1NuFRgCzWzkCMXb~WsRIj4pG-L2oN9xJ9BLhjvXPtlZMrRkSswATk7n8SGChI0f7FdGsdZp0Ujr3CXwK1dHra23bGpx85Q7cHNsdxYm5FnVUfwOwUduglUjwH2uVjdadtem1AGThTPX8J3F4AKcLCQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519705&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of PRR homologues in various plants. Amino acid sequences of PRR homologues including entire RLD and CCT domain regions were aligned using ClustalW program and the phylogenetic tree was reconstructed by the ME method<sup><a reveal-id="DSQ033C28" data-open="DSQ033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> by using MEGA4.1.<sup><a reveal-id="DSQ033C30" data-open="DSQ033C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The <em>O. tauri</em> PRR homologue (OtTOC1<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup>) was used as the outgroup. The numbers at each node represent the bootstrap values calculated based on 1000 bootstrap sampling and those that are higher than 50% are shown. The three PRR subfamilies are indicated. PpPRRs from <em>P. patens</em> are indicated in bold. Sequences used are as follows: OsTOC1 (BAD38854 in DDBJ/EMBL/GenBank), OsPRR37 (BAD38855), OsPRR73 (BAD38856), OsPRR59 (AK120059) and OsPRR95 (BAD38857) from <em>Oryza sativa</em>; PtTOC1 (XP_002330130.1 in NCBI protein database), PtPRR37 (XP_002311123.1), PtPRR73 (XP_002316333.1), PtPRR5a (XP_002321349.1), PtPRR5b (XP_002301442.1), PtPRR9a (XP_002320232.1) and PtPRR9b (XP_002301443.1) from <em>Populus trichocarpa</em>; VvTOC1 (XP_002281757.1), VvPRR37 (XP_002281776), VvPRR73 (XP_002275645), VvPRR5 (XP_002270811) and VvPRR9 (XP_002266192.1) from <em>Vitis vinifera</em>; LjTOC1 (chr4.CM0087.600.nc in miyakogusa.jp; <a class="link link-uri" href="http://www.kazusa.or.jp/lotus/index.html">http://www.kazusa.or.jp/lotus/index.html</a>), LjPRR3 (LjT08O17.180/130/120), LjPRR5 (chr1.CM0105.560), LjPRR7 (chr3.LjT05P05.60) and LjPRR9 (chr3.CM0208.230) from <em>Lotus japonicas</em>;<sup><a reveal-id="DSQ033C42" data-open="DSQ033C42" class="link link-ref link-reveal xref-bibr">42</a></sup> CrPRR1 (XP_001695777.1), CrPRR2 (XP_001701808.1) from <em>Chlamydomonas reinhardtii</em>; CvPRR1 (EFN58892.1) from <em>C. variabilis</em>. For other sequences, refer to the legend for Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div><p>Next, we compared the distribution of intron insertion sites on the RLD and CCT coding regions between <em>PRR</em> genes (Fig. <a reveal-id="DSQ033F3" data-open="DSQ033F3" class="link link-reveal link-table xref-fig">3</a>). In the RLD region, two insertion sites are conserved among all the <em>PRR</em> sequences examined (Fig. <a reveal-id="DSQ033F3" data-open="DSQ033F3" class="link link-reveal link-table xref-fig">3</a>a; see Takata <em>et al.</em><sup><a reveal-id="DSQ033C15" data-open="DSQ033C15" class="link link-ref link-reveal xref-bibr">15</a></sup>). This distribution pattern of intron insertion sites is clearly different from those seen in the receiver domain of authentic RRs,<sup><a reveal-id="DSQ033C13" data-open="DSQ033C13" class="link link-ref link-reveal xref-bibr">13</a></sup> confirming the idea that <em>PRR</em> genes diverged distinctly from authentic <em>RR</em> genes. In the CCT domain region, the TOC1 group sequences are unique in that they show no intron, whereas all sequences in the other two groups have a conserved single insertion site in the middle of the domain (Fig. <a reveal-id="DSQ033F3" data-open="DSQ033F3" class="link link-reveal link-table xref-fig">3</a>b; see Takata <em>et al.</em><sup><a reveal-id="DSQ033C15" data-open="DSQ033C15" class="link link-ref link-reveal xref-bibr">15</a></sup>). <em>PpPRR</em> sequences are divided into two groups, <em>PpPRR1</em>/<em>PpPRR3</em> with one intron (like the PRR7/3 and PRR9/5 groups) and <em>PpPRR2</em>/<em>PpPRR4</em> with no intron (like the TOC1 group; Fig. <a reveal-id="DSQ033F3" data-open="DSQ033F3" class="link link-reveal link-table xref-fig">3</a>). This divergence is consistent with the results of the phylogenetic tree (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>), whereas the loss of an intron in <em>PpPRR2</em>/<em>PpPRR4</em> should have occurred in the moss lineage independently from the TOC1 group (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="5519711" scrollto-destination="5519711"></a>
<div id="DSQ033F3" data-id="DSQ033F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03303.jpeg?Expires=1509238741&Signature=LVd6BY4ghqbvzHtKtWteDOeWJfY7tOlyxAZ3WEH5CdJQ~MgyvdjAvq8gzCTaZ6J6k06~eJRuTR7VM7lbg9iO6lnB6~u5jEJ1FFDZhfs0-G9zcAFJyMjzjQWNlLVK003-U-6G~LeDSA2wPg3gUTLh3UC5rC8euZx7Y322XR31kuatgp6sFz2X6RD5v8PgFQlnNSmsd72tckhhvjChfEeCz7avEHO-EwsUWyLr~WNJxc03Ppd-Sq4Y1IUYmrhrEQUGdDn0MWHG7rDEVrKI~V3BF1VuyqFg1QOU55RILluFdZIxonVnh9Wxmy8KH~SbHq94AVvNLR~ydsNMzsjTFbI4ZA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of intron insertion sites on the conserved domains of PRR sequences. From the aligned amino acid sequences of PRR proteins, only three stretches [two from RLD (a) and one from CCT (b)] are shown, each consisting of nine amino acids with the amino acid corresponding to an intron insertion site (shaded in grey) being centred. The number of the rightmost amino acid of each stretch is indicated in parentheses. For the sequences used, refer to the legend of Fig. 2." path-from-xml="dsq03303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519711/dsq03303.jpeg" path-from-xml="dsq03303.jpeg" target="_blank">View large</a><a section="5519711" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03303.jpeg?Expires=1509238741&Signature=LcuSn1z-9kWhg2ij3o17cWk0uCVowj0WFaizMpm-3uvYul~jHEdpctp0ue1lAF1qwDqq~4pBtB2xC18t6FyjXTNwENXYquQcXl0couAGY7R-i6Hwhm2Kco4f2-8-o60q6igoQGkyrn7qwqaR8dGJZ-AGMFKG5f1-r2LziPi1nYD6fQ489JWSgYDmBUdBR8uApZMOAgnUHWa2l8csPYvQIswKrbHLU-saubZOFTVxZi6LO5we-UEzxI5dOWUw1uA056tUhnbi1VGfDkN9y~688HqkdKbzbDDgUeaUFx4u6iwj6ENyzLzLCrlO4BPK0rpoaI-Zc7TXpfn3ZO4fLFmHxA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519711&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of intron insertion sites on the conserved domains of PRR sequences. From the aligned amino acid sequences of PRR proteins, only three stretches [two from RLD (a) and one from CCT (b)] are shown, each consisting of nine amino acids with the amino acid corresponding to an intron insertion site (shaded in grey) being centred. The number of the rightmost amino acid of each stretch is indicated in parentheses. For the sequences used, refer to the legend of Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>.</p></div></div><div content-id="DSQ033F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03303.jpeg?Expires=1509238741&Signature=LVd6BY4ghqbvzHtKtWteDOeWJfY7tOlyxAZ3WEH5CdJQ~MgyvdjAvq8gzCTaZ6J6k06~eJRuTR7VM7lbg9iO6lnB6~u5jEJ1FFDZhfs0-G9zcAFJyMjzjQWNlLVK003-U-6G~LeDSA2wPg3gUTLh3UC5rC8euZx7Y322XR31kuatgp6sFz2X6RD5v8PgFQlnNSmsd72tckhhvjChfEeCz7avEHO-EwsUWyLr~WNJxc03Ppd-Sq4Y1IUYmrhrEQUGdDn0MWHG7rDEVrKI~V3BF1VuyqFg1QOU55RILluFdZIxonVnh9Wxmy8KH~SbHq94AVvNLR~ydsNMzsjTFbI4ZA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of intron insertion sites on the conserved domains of PRR sequences. From the aligned amino acid sequences of PRR proteins, only three stretches [two from RLD (a) and one from CCT (b)] are shown, each consisting of nine amino acids with the amino acid corresponding to an intron insertion site (shaded in grey) being centred. The number of the rightmost amino acid of each stretch is indicated in parentheses. For the sequences used, refer to the legend of Fig. 2." path-from-xml="dsq03303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519711/dsq03303.jpeg" path-from-xml="dsq03303.jpeg" target="_blank">View large</a><a section="5519711" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03303.jpeg?Expires=1509238741&Signature=LcuSn1z-9kWhg2ij3o17cWk0uCVowj0WFaizMpm-3uvYul~jHEdpctp0ue1lAF1qwDqq~4pBtB2xC18t6FyjXTNwENXYquQcXl0couAGY7R-i6Hwhm2Kco4f2-8-o60q6igoQGkyrn7qwqaR8dGJZ-AGMFKG5f1-r2LziPi1nYD6fQ489JWSgYDmBUdBR8uApZMOAgnUHWa2l8csPYvQIswKrbHLU-saubZOFTVxZi6LO5we-UEzxI5dOWUw1uA056tUhnbi1VGfDkN9y~688HqkdKbzbDDgUeaUFx4u6iwj6ENyzLzLCrlO4BPK0rpoaI-Zc7TXpfn3ZO4fLFmHxA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519711&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of intron insertion sites on the conserved domains of PRR sequences. From the aligned amino acid sequences of PRR proteins, only three stretches [two from RLD (a) and one from CCT (b)] are shown, each consisting of nine amino acids with the amino acid corresponding to an intron insertion site (shaded in grey) being centred. The number of the rightmost amino acid of each stretch is indicated in parentheses. For the sequences used, refer to the legend of Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>.</p></div></div>                            <h3 scrollto-destination=5519715 id="5519715" class="section-title" data-legacy-id=s3b>3.2. <em>In vitro</em> phosphotransfer from a HK to the RLD of PpPRR2</h3>
<p>We examined whether RLDs of PpPRR1 and PpPRR2, both of which retain the potential phosphoacceptor residue while showing mutually diverged features (Figs <a reveal-id="DSQ033F1  DSQ033F3" data-open="DSQ033F1  DSQ033F3" class="link link-reveal link-table xref-fig">1–3</a>), are phosphorylated by a His–Asp phosphorelay process in an <em>in vitro</em> phosphotransfer assay (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>). In this assay, ArcB, an <em>E. coli</em> HK, added in excess and hence lacking substrate specificity, transfers its phosphate to the phosphoacceptor site in a receiver domain. We purified the RLD peptides of PpPRR1 and PpPRR2 overexpressed in <em>E. coli</em> cells (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>b and c) and tested each of them to assess whether or not they undergo phosphotransfer. The PpPRR2 RLD peptide was phosphorylated within 5 min in the presence of ArcB (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>g). The phosphorylation levels of ArcB decreased concomitantly (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>g), indicating that the phosphorylation of PpPRR2 RLD is due to phosphotransfer from ArcB. On the other hand, we could not detect any phosphorylation signal with PpPRR1 RLD (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>f), consistent with its relatively diverged RLD sequence (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a). When the PpPRR2 RLD peptide was incubated with the membrane fraction with no overexpressed ArcB, we could not detect the phosphorylation signal (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>h). This supports the interpretation that the increased levels of phosphorylation seen with the PpPRR2 RLD peptide (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>g) is due to phosphotransfer from ArcB, but not due to other types of kinases. These observations indicate that PpPRR2 presumably functions as an RR in an unknown His–Asp phosphorelay signal transduction pathway in <em>P. patens</em>. Moreover, PpPRRs likely diverged from one another based not only on their RLD sequences but also on their phosphotransfer ability. </p>                        <a id="5519719" scrollto-destination="5519719"></a>
<div id="DSQ033F4" data-id="DSQ033F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03304.jpeg?Expires=1509238741&Signature=YmIshCQgdF4LEjlj8jti77YEYWoYdYPjCWS-JdN26jKDgWIhgJGXVF3PJc7ZlNZXhT1sNibIA~8271H0w5f2aD-IBLnYo-Ek0tBVa8gsvDP8kFtt~8I9-cQ5IQ7yLWEMLl0ZDq2FicH1YM-f5-ERgTq-AHav~wTTDOEZkBuqBwzvoUjveJ4uO-xZ25TZt6qTeBYYe5N5s7RhWe7M0JKXeSWDa5vZrc~nGW6CztTbISCpOwS0uAEq7kxHi3ftuyiTvxv2CFJkHA~tEAXpy5xPEwBAaHk1HXxa7gbGN24Vd064Zp0RWiu0Unq-U6pT268ggxymqtLFdCqSeQcytGKeMQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="In vitro His–Asp phosphotransfer to the PpPRR2 RLD peptide. (a) Overproduction of ArcB in E. coli cells. The cytoplasmic membrane proteins were extracted from an ArcB overproducer (right lane) and a negative control strain of E. coli (left lane), subjected to SDS–PAGE and detected with Coomassie Brilliant Blue (C.B.B.) (see 2.7. Preparation of the E. coli ArcB-enriched cytoplasmic membrane). The ArcB band of around 88 kDa is indicated with an open triangle. Purification of PpPRR1 (b) and PpPRR2 RLD (c) peptides is shown. PpPRR1 and PpPRR2 RLD peptides were, respectively, affinity purified with TALON™ metal affinity resin from the soluble protein fraction of cell lysate of each overproducer strain, subjected to SDS–PAGE and detected with C.B.B. Existence of PpPRR1 and PpPRR2 RLD peptides were further confirmed by western blotting with anti-6×His antibodies; PpPRR1 and PpPRR2 RLD peptides (∼17 kDa for each) are indicated with closed triangles. (d and e) Results of autophosphorylation assay. Escherichia coli ArcB-enriched membranes (e) or control membranes (d) were incubated with γ-[32P]-ATP for indicated times. (f–h) Results of in vitro His–Asp phosphotransfer assay. Escherichia coli ArcB-enriched membranes were incubated with γ-[32P]-ATP for indicated times in the presence of PpPRR1 (f) or PpPRR2 (g) RLD peptide. A reference reaction was conducted using control membranes incubated with γ-[32P]-ATP for indicated times in the presence of PpPRR2 RLD peptide (h). Signals from the gels were analysed by BAS-2500 for their phosphorylation status." path-from-xml="dsq03304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519719/dsq03304.jpeg" path-from-xml="dsq03304.jpeg" target="_blank">View large</a><a section="5519719" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03304.jpeg?Expires=1509238741&Signature=cTO0drwNqvNogcRlQzUhFj3EVw227vTKAEonFIdPg11p~KzUVoOnZuk1C3IPHn~B-Lz-22nqyhR-KxU2XUgczk5nymsl0E9FmDEutsFMHPj7CEWV7cBGjjz4VPWl7j~nHvxWNPuV-Piqfp48BdFnavbldQuq4qE~XO7Xt1MobCESOvjHEOc6DB7OuElJqBRN6OXPzBUOtMzGYqd5X0xT6hYZiBhT9qTZQ4JqZ-jnDiNFLUHyuxTwX7QSS-qwRNmb9Cdn6DIIKAizL9ram9mb86VnbWr2i4fk~LGhMVuRKfrtmOgPXQ1RgsJ73TMBheXKZARasvedD-gqshoCDXYy8A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519719&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> His–Asp phosphotransfer to the PpPRR2 RLD peptide. (a) Overproduction of ArcB in <em>E. coli</em> cells. The cytoplasmic membrane proteins were extracted from an ArcB overproducer (right lane) and a negative control strain of <em>E. coli</em> (left lane), subjected to SDS–PAGE and detected with Coomassie Brilliant Blue (C.B.B.) (see 2.7. Preparation of the E. coli ArcB-enriched cytoplasmic membrane). The ArcB band of around 88 kDa is indicated with an open triangle. Purification of PpPRR1 (b) and PpPRR2 RLD (c) peptides is shown. PpPRR1 and PpPRR2 RLD peptides were, respectively, affinity purified with TALON™ metal affinity resin from the soluble protein fraction of cell lysate of each overproducer strain, subjected to SDS–PAGE and detected with C.B.B. Existence of PpPRR1 and PpPRR2 RLD peptides were further confirmed by western blotting with anti-6×His antibodies; PpPRR1 and PpPRR2 RLD peptides (∼17 kDa for each) are indicated with closed triangles. (d and e) Results of autophosphorylation assay. <em>Escherichia coli</em> ArcB-enriched membranes (e) or control membranes (d) were incubated with γ-[<sup>32</sup>P]-ATP for indicated times. (f–h) Results of <em>in vitro</em> His–Asp phosphotransfer assay. <em>Escherichia coli</em> ArcB-enriched membranes were incubated with γ-[<sup>32</sup>P]-ATP for indicated times in the presence of PpPRR1 (f) or PpPRR2 (g) RLD peptide. A reference reaction was conducted using control membranes incubated with γ-[<sup>32</sup>P]-ATP for indicated times in the presence of PpPRR2 RLD peptide (h). Signals from the gels were analysed by BAS-2500 for their phosphorylation status.</p></div></div><div content-id="DSQ033F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03304.jpeg?Expires=1509238741&Signature=YmIshCQgdF4LEjlj8jti77YEYWoYdYPjCWS-JdN26jKDgWIhgJGXVF3PJc7ZlNZXhT1sNibIA~8271H0w5f2aD-IBLnYo-Ek0tBVa8gsvDP8kFtt~8I9-cQ5IQ7yLWEMLl0ZDq2FicH1YM-f5-ERgTq-AHav~wTTDOEZkBuqBwzvoUjveJ4uO-xZ25TZt6qTeBYYe5N5s7RhWe7M0JKXeSWDa5vZrc~nGW6CztTbISCpOwS0uAEq7kxHi3ftuyiTvxv2CFJkHA~tEAXpy5xPEwBAaHk1HXxa7gbGN24Vd064Zp0RWiu0Unq-U6pT268ggxymqtLFdCqSeQcytGKeMQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="In vitro His–Asp phosphotransfer to the PpPRR2 RLD peptide. (a) Overproduction of ArcB in E. coli cells. The cytoplasmic membrane proteins were extracted from an ArcB overproducer (right lane) and a negative control strain of E. coli (left lane), subjected to SDS–PAGE and detected with Coomassie Brilliant Blue (C.B.B.) (see 2.7. Preparation of the E. coli ArcB-enriched cytoplasmic membrane). The ArcB band of around 88 kDa is indicated with an open triangle. Purification of PpPRR1 (b) and PpPRR2 RLD (c) peptides is shown. PpPRR1 and PpPRR2 RLD peptides were, respectively, affinity purified with TALON™ metal affinity resin from the soluble protein fraction of cell lysate of each overproducer strain, subjected to SDS–PAGE and detected with C.B.B. Existence of PpPRR1 and PpPRR2 RLD peptides were further confirmed by western blotting with anti-6×His antibodies; PpPRR1 and PpPRR2 RLD peptides (∼17 kDa for each) are indicated with closed triangles. (d and e) Results of autophosphorylation assay. Escherichia coli ArcB-enriched membranes (e) or control membranes (d) were incubated with γ-[32P]-ATP for indicated times. (f–h) Results of in vitro His–Asp phosphotransfer assay. Escherichia coli ArcB-enriched membranes were incubated with γ-[32P]-ATP for indicated times in the presence of PpPRR1 (f) or PpPRR2 (g) RLD peptide. A reference reaction was conducted using control membranes incubated with γ-[32P]-ATP for indicated times in the presence of PpPRR2 RLD peptide (h). Signals from the gels were analysed by BAS-2500 for their phosphorylation status." path-from-xml="dsq03304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519719/dsq03304.jpeg" path-from-xml="dsq03304.jpeg" target="_blank">View large</a><a section="5519719" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03304.jpeg?Expires=1509238741&Signature=cTO0drwNqvNogcRlQzUhFj3EVw227vTKAEonFIdPg11p~KzUVoOnZuk1C3IPHn~B-Lz-22nqyhR-KxU2XUgczk5nymsl0E9FmDEutsFMHPj7CEWV7cBGjjz4VPWl7j~nHvxWNPuV-Piqfp48BdFnavbldQuq4qE~XO7Xt1MobCESOvjHEOc6DB7OuElJqBRN6OXPzBUOtMzGYqd5X0xT6hYZiBhT9qTZQ4JqZ-jnDiNFLUHyuxTwX7QSS-qwRNmb9Cdn6DIIKAizL9ram9mb86VnbWr2i4fk~LGhMVuRKfrtmOgPXQ1RgsJ73TMBheXKZARasvedD-gqshoCDXYy8A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519719&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>In vitro</em> His–Asp phosphotransfer to the PpPRR2 RLD peptide. (a) Overproduction of ArcB in <em>E. coli</em> cells. The cytoplasmic membrane proteins were extracted from an ArcB overproducer (right lane) and a negative control strain of <em>E. coli</em> (left lane), subjected to SDS–PAGE and detected with Coomassie Brilliant Blue (C.B.B.) (see 2.7. Preparation of the E. coli ArcB-enriched cytoplasmic membrane). The ArcB band of around 88 kDa is indicated with an open triangle. Purification of PpPRR1 (b) and PpPRR2 RLD (c) peptides is shown. PpPRR1 and PpPRR2 RLD peptides were, respectively, affinity purified with TALON™ metal affinity resin from the soluble protein fraction of cell lysate of each overproducer strain, subjected to SDS–PAGE and detected with C.B.B. Existence of PpPRR1 and PpPRR2 RLD peptides were further confirmed by western blotting with anti-6×His antibodies; PpPRR1 and PpPRR2 RLD peptides (∼17 kDa for each) are indicated with closed triangles. (d and e) Results of autophosphorylation assay. <em>Escherichia coli</em> ArcB-enriched membranes (e) or control membranes (d) were incubated with γ-[<sup>32</sup>P]-ATP for indicated times. (f–h) Results of <em>in vitro</em> His–Asp phosphotransfer assay. <em>Escherichia coli</em> ArcB-enriched membranes were incubated with γ-[<sup>32</sup>P]-ATP for indicated times in the presence of PpPRR1 (f) or PpPRR2 (g) RLD peptide. A reference reaction was conducted using control membranes incubated with γ-[<sup>32</sup>P]-ATP for indicated times in the presence of PpPRR2 RLD peptide (h). Signals from the gels were analysed by BAS-2500 for their phosphorylation status.</p></div></div>                            <h3 scrollto-destination=5519723 id="5519723" class="section-title" data-legacy-id=s3c>3.3. Circadian regulation of PpPRR expression profiles</h3>
<p>In a previous study,<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> we reported that the <em>PpPRR1</em> mRNA accumulation showed circadian variation in 12-h light and 12-h dark cycles (12:12LD) and in continuous darkness (DD), but not in continuous light (LL), by conducting sqRT-PCR analysis. Here, we studied mRNA accumulation profiles for all four <em>PpPRR</em> genes with the same method (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>). In 12:12LD, all genes showed high-amplitude mRNA rhythms with a period of ∼1 day, which peaked in the latter half of the light phase (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>a). In DD, all genes showed endogenous rhythms with damping (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>b). These rhythms in 12:12LD or DD showed phase relationships roughly similar to <em>A. thaliana PRR3</em>, <em>PRR5</em> or <em>PRR7</em> genes.<sup><a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In LL, in contrast, all the genes exhibited no hint of circadian regulation and were arrhythmic as demonstrated for <em>PpPRR1</em> (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>c; see Okada <em>et al.</em><sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup>). The arrhythmic profiles in LL are consistent with our observations that the moss genes so far tested are, if clock gene homologues or clock-controlled genes, all arrhythmic in LL,<sup><a reveal-id="DSQ033C38" data-open="DSQ033C38" class="link link-ref link-reveal xref-bibr">38</a>–<a reveal-id="DSQ033C40" data-open="DSQ033C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and this is in contrast to angiosperm <em>PRR</em> genes, all of which show robust circadian rhythms in LL.<sup><a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p>                        <a id="5519727" scrollto-destination="5519727"></a>
<div id="DSQ033F5" data-id="DSQ033F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03305.jpeg?Expires=1509238741&Signature=g0gtMIXpJ1qFNjzvnWoijM-UVNcC7vUai-geGlCeU1f3iy9kKYMA0t-jSCEBWm0iy10~FmRc6piU30C661Xq14TBSkrbnyKijv6egWwvHjpR1WsZ4ZlT5klMI9WOwkxhBl9kFInNYKhQtGk9XWhXDDt~Ic6gHguGfLyc3hDi5dxB~~U2g4tFcUruIso6DGWCtURL9Yrj1mCnSHgaDO3Wx8s3laIdBdmkQ3cwCklL1JQM8lNGfYqAcKegAfrmKJeWRG6efpyYDHP1SHunxMNdJGRsnnU9OFDqnAeC3fHg5cqFp4y71GP04XaQxaKZTnQeS05GThsm9sXom8L3Ba3Wpg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Changes in mRNA abundance for the PpPRR genes under 12:12LD, DD and LL conditions examined by the sqRT-PCR analysis. Physcomitrella patens protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in 12:12LD (a), DD (b) or LL (c) conditions at indicated times. From the top, changes in mRNA abundance for PpPRR1, PpPRR2, PpPRR3 and PpPRR4 are shown. Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively; regions shaded with light grey represent subjective light phases in (b) or subjective dark phases in (c). Graphs show the results of quantification of the mRNA levels for each gene after normalization to those for actin as a control.39 The maximum levels are set to 1.0. The photo below each graph shows the hybridized bands for each test gene or the control actin gene, detected as chemiluminescence signals. We obtained similar results in two independent experiments." path-from-xml="dsq03305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519727/dsq03305.jpeg" path-from-xml="dsq03305.jpeg" target="_blank">View large</a><a section="5519727" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03305.jpeg?Expires=1509238741&Signature=Jcicp1IoWt~xORE5-r0prOuBh4KZYZtIH0Y4DlePOJod0cDzuzOu0m39qy4zoyLnLpqJtGby0jU464N57hpnOA1NTLbBxU85cyMOUkEkhF~Wm1uUQyZ1RRdG3f5ya3h6f1V5H9Qmx4a3BUvVkrbPoZn7g09aWEc5kWwXscVkokiundyuq67jeORnsx-Fzal5wVO~IIY2ZqkB6H1VSKvF3vs09PApx~2X7zHGsQMWgioJU9dwPcVXfP6kA~EucseTD3FOq6wEvvVOmWFCRui~WPwxcJjcAc1wREUfOmc0qdh537W~yTWfX9XxTVTMk7ji7t0u6i9vK-HHXTi8pMz5qw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519727&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Changes in mRNA abundance for the <em>PpPRR</em> genes under 12:12LD, DD and LL conditions examined by the sqRT-PCR analysis. <em>Physcomitrella patens</em> protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in 12:12LD (a), DD (b) or LL (c) conditions at indicated times. From the top, changes in mRNA abundance for <em>PpPRR1</em>, <em>PpPRR2</em>, <em>PpPRR3</em> and <em>PpPRR4</em> are shown. Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively; regions shaded with light grey represent subjective light phases in (b) or subjective dark phases in (c). Graphs show the results of quantification of the mRNA levels for each gene after normalization to those for actin as a control.<sup><a reveal-id="DSQ033C39" data-open="DSQ033C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The maximum levels are set to 1.0. The photo below each graph shows the hybridized bands for each test gene or the control <em>actin</em> gene, detected as chemiluminescence signals. We obtained similar results in two independent experiments.</p></div></div><div content-id="DSQ033F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03305.jpeg?Expires=1509238741&Signature=g0gtMIXpJ1qFNjzvnWoijM-UVNcC7vUai-geGlCeU1f3iy9kKYMA0t-jSCEBWm0iy10~FmRc6piU30C661Xq14TBSkrbnyKijv6egWwvHjpR1WsZ4ZlT5klMI9WOwkxhBl9kFInNYKhQtGk9XWhXDDt~Ic6gHguGfLyc3hDi5dxB~~U2g4tFcUruIso6DGWCtURL9Yrj1mCnSHgaDO3Wx8s3laIdBdmkQ3cwCklL1JQM8lNGfYqAcKegAfrmKJeWRG6efpyYDHP1SHunxMNdJGRsnnU9OFDqnAeC3fHg5cqFp4y71GP04XaQxaKZTnQeS05GThsm9sXom8L3Ba3Wpg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Changes in mRNA abundance for the PpPRR genes under 12:12LD, DD and LL conditions examined by the sqRT-PCR analysis. Physcomitrella patens protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in 12:12LD (a), DD (b) or LL (c) conditions at indicated times. From the top, changes in mRNA abundance for PpPRR1, PpPRR2, PpPRR3 and PpPRR4 are shown. Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively; regions shaded with light grey represent subjective light phases in (b) or subjective dark phases in (c). Graphs show the results of quantification of the mRNA levels for each gene after normalization to those for actin as a control.39 The maximum levels are set to 1.0. The photo below each graph shows the hybridized bands for each test gene or the control actin gene, detected as chemiluminescence signals. We obtained similar results in two independent experiments." path-from-xml="dsq03305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519727/dsq03305.jpeg" path-from-xml="dsq03305.jpeg" target="_blank">View large</a><a section="5519727" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03305.jpeg?Expires=1509238741&Signature=Jcicp1IoWt~xORE5-r0prOuBh4KZYZtIH0Y4DlePOJod0cDzuzOu0m39qy4zoyLnLpqJtGby0jU464N57hpnOA1NTLbBxU85cyMOUkEkhF~Wm1uUQyZ1RRdG3f5ya3h6f1V5H9Qmx4a3BUvVkrbPoZn7g09aWEc5kWwXscVkokiundyuq67jeORnsx-Fzal5wVO~IIY2ZqkB6H1VSKvF3vs09PApx~2X7zHGsQMWgioJU9dwPcVXfP6kA~EucseTD3FOq6wEvvVOmWFCRui~WPwxcJjcAc1wREUfOmc0qdh537W~yTWfX9XxTVTMk7ji7t0u6i9vK-HHXTi8pMz5qw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519727&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Changes in mRNA abundance for the <em>PpPRR</em> genes under 12:12LD, DD and LL conditions examined by the sqRT-PCR analysis. <em>Physcomitrella patens</em> protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in 12:12LD (a), DD (b) or LL (c) conditions at indicated times. From the top, changes in mRNA abundance for <em>PpPRR1</em>, <em>PpPRR2</em>, <em>PpPRR3</em> and <em>PpPRR4</em> are shown. Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively; regions shaded with light grey represent subjective light phases in (b) or subjective dark phases in (c). Graphs show the results of quantification of the mRNA levels for each gene after normalization to those for actin as a control.<sup><a reveal-id="DSQ033C39" data-open="DSQ033C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The maximum levels are set to 1.0. The photo below each graph shows the hybridized bands for each test gene or the control <em>actin</em> gene, detected as chemiluminescence signals. We obtained similar results in two independent experiments.</p></div></div><p>The results of the sqRT-PCR experiments suggest that phases of the four genes seem to be differentially fine-tuned into two types: in 12:12LD, troughs of <em>PpPRR1</em>/<em>PpPRR3</em> occurred 4 h before dawn, whereas those of <em>PpPRR2</em>/<em>PpPRR4</em> just at dawn (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>a). We also measured mRNA accumulation for <em>PpPRRs</em> in one cycle of 12:12LD by the qRT-PCR analysis with a shorter sampling interval (Fig. <a reveal-id="DSQ033F6" data-open="DSQ033F6" class="link link-reveal link-table xref-fig">6</a>). At ZT01 (ZT, zeitgeber time: time in a light–dark cycle, putting the light onset as ZT0), the levels of <em>PpPRR1</em>/<em>PpPRR3</em> showed certain increase (30–40% of the maximum levels), whereas those of <em>PpPRR2</em>/<em>PpPRR4</em> were still very close to zero (Fig. <a reveal-id="DSQ033F6" data-open="DSQ033F6" class="link link-reveal link-table xref-fig">6</a>), confirming the distinction of expression profiles between <em>PpPRR1</em>/<em>PpPRR3</em> and <em>PpPRR2</em>/<em>PpPRR4</em>. This differential fine-tuning appears to be conferred by the endogenous circadian clock, because the sampling of cells at dawn (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>a, hours 0 and 24) was performed in the absence of light. This idea is also supported by the rhythms of <em>PpPRRs</em> in DD, where endogenous clock regulation is more clearly seen: the first peaks of <em>PpPRR2</em>/<em>PpPRR4</em> lagged behind those of <em>PpPRR1</em>/<em>PpPRR3</em> by around 4 h (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>b; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 4</a>). </p>                        <a id="5519732" scrollto-destination="5519732"></a>
<div id="DSQ033F6" data-id="DSQ033F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03306.jpeg?Expires=1509238741&Signature=ZB8P9FBU~esh7AFrMZX1mvzqhIa6yYmDRwW6j46FYt0kQPCvKpb7ZD6ywl4QUg2~4T4KNfwKw8n0BGsdqslCwkUvdCzVP~bBDeeJGwyBqo07MdsoAII0y2~D6HGwxct-ghfcyrl0-8Y1pisF4dpysjfwbRlWRcr1JqsFy5jec5n7KfrU-QfMoLkYl13IU9b9xbOqoDb7DGj8dCdE~g5GTmp68n9CROsY70Y7Z4WyFPKUij4F8yT~g9TIswP7tAiaY-Y90PHRIQprl4WDXvq4R339GRaqEUYa94sSBh1QYwrW~-i-flT9VF4EvCv6BMMGLZInpcYamYWQ9NDLxzCcHg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Changes in mRNA abundance for the PpPRR genes for one 12:12LD cycle examined by the qRT-PCR analysis. Physcomitrella patens protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in one full cycle of 12:12LD at indicated times. Shown are the changes in mRNA abundance for PpPRR1 (a), PpPRR2 (b), PpPRR3 (c) and PpPRR4 (d). Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively. The graphs show the averages and the standard deviations of four independent experiments, in which the results of quantification of mRNA levels for each gene were normalized to those of actin as a control.39" path-from-xml="dsq03306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519732/dsq03306.jpeg" path-from-xml="dsq03306.jpeg" target="_blank">View large</a><a section="5519732" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03306.jpeg?Expires=1509238741&Signature=ayPu79T9fjb6QoO60dCfJ9MUB2gMiy6BsYeJ8ytmnL25OQxTsp2x71rseGslHNskekXo0fBrtFDLMHWtV668SoMsmcJAhefJJPQ3oNIy2GMsZiDLMTdT5vIvsaA9C24ZprLD5bWkln-37jVE5DlOdETj-1MKaMSjX0dfgIrlAHUvqNpAPp1E3i~o7127oiJKxIwAer91ikRbMsGtdFqGIzMPJOYdTOpW~lr4YhBCiWod53Cznfmw~k1OML~QyQVe73GoLd9AQQdmijVFx2OKwA8yc54cJKhKyUQHNBKy7NG8913WKiXKUVqNCTg9wKpg5y9cQlfk3uZyoIBI2DyOdQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519732&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Changes in mRNA abundance for the <em>PpPRR</em> genes for one 12:12LD cycle examined by the qRT-PCR analysis. <em>Physcomitrella patens</em> protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in one full cycle of 12:12LD at indicated times. Shown are the changes in mRNA abundance for <em>PpPRR1</em> (a), <em>PpPRR2</em> (b), <em>PpPRR3</em> (c) and <em>PpPRR4</em> (d). Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively. The graphs show the averages and the standard deviations of four independent experiments, in which the results of quantification of mRNA levels for each gene were normalized to those of <em>actin</em> as a control.<sup><a reveal-id="DSQ033C39" data-open="DSQ033C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p></div></div><div content-id="DSQ033F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03306.jpeg?Expires=1509238741&Signature=ZB8P9FBU~esh7AFrMZX1mvzqhIa6yYmDRwW6j46FYt0kQPCvKpb7ZD6ywl4QUg2~4T4KNfwKw8n0BGsdqslCwkUvdCzVP~bBDeeJGwyBqo07MdsoAII0y2~D6HGwxct-ghfcyrl0-8Y1pisF4dpysjfwbRlWRcr1JqsFy5jec5n7KfrU-QfMoLkYl13IU9b9xbOqoDb7DGj8dCdE~g5GTmp68n9CROsY70Y7Z4WyFPKUij4F8yT~g9TIswP7tAiaY-Y90PHRIQprl4WDXvq4R339GRaqEUYa94sSBh1QYwrW~-i-flT9VF4EvCv6BMMGLZInpcYamYWQ9NDLxzCcHg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Changes in mRNA abundance for the PpPRR genes for one 12:12LD cycle examined by the qRT-PCR analysis. Physcomitrella patens protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in one full cycle of 12:12LD at indicated times. Shown are the changes in mRNA abundance for PpPRR1 (a), PpPRR2 (b), PpPRR3 (c) and PpPRR4 (d). Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively. The graphs show the averages and the standard deviations of four independent experiments, in which the results of quantification of mRNA levels for each gene were normalized to those of actin as a control.39" path-from-xml="dsq03306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519732/dsq03306.jpeg" path-from-xml="dsq03306.jpeg" target="_blank">View large</a><a section="5519732" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03306.jpeg?Expires=1509238741&Signature=ayPu79T9fjb6QoO60dCfJ9MUB2gMiy6BsYeJ8ytmnL25OQxTsp2x71rseGslHNskekXo0fBrtFDLMHWtV668SoMsmcJAhefJJPQ3oNIy2GMsZiDLMTdT5vIvsaA9C24ZprLD5bWkln-37jVE5DlOdETj-1MKaMSjX0dfgIrlAHUvqNpAPp1E3i~o7127oiJKxIwAer91ikRbMsGtdFqGIzMPJOYdTOpW~lr4YhBCiWod53Cznfmw~k1OML~QyQVe73GoLd9AQQdmijVFx2OKwA8yc54cJKhKyUQHNBKy7NG8913WKiXKUVqNCTg9wKpg5y9cQlfk3uZyoIBI2DyOdQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519732&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Changes in mRNA abundance for the <em>PpPRR</em> genes for one 12:12LD cycle examined by the qRT-PCR analysis. <em>Physcomitrella patens</em> protonemal cells were maintained in 12:12LD for more than 2 weeks after which cells were harvested in one full cycle of 12:12LD at indicated times. Shown are the changes in mRNA abundance for <em>PpPRR1</em> (a), <em>PpPRR2</em> (b), <em>PpPRR3</em> (c) and <em>PpPRR4</em> (d). Light conditions are overlaid with each graph: regions with no shade and those shaded with dark grey represent light and dark phases, respectively. The graphs show the averages and the standard deviations of four independent experiments, in which the results of quantification of mRNA levels for each gene were normalized to those of <em>actin</em> as a control.<sup><a reveal-id="DSQ033C39" data-open="DSQ033C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p></div></div>                            <h3 scrollto-destination=5519734 id="5519734" class="section-title" data-legacy-id=s3d>3.4. Induction of PpPRR by light</h3>
<p>We examined the effect of light on the accumulation of the <em>PpPRR</em> mRNAs by sqRT-PCR analyses. The mRNAs of all four <em>PpPRRs</em> were induced by a 2-h pulse of white, blue or red light (Fig. <a reveal-id="DSQ033F7" data-open="DSQ033F7" class="link link-reveal link-table xref-fig">7</a>). The rates of induction by white light are 8.7 for <em>PpPRR1</em>, 2.6 for <em>PpPRR2</em>, 6.2 for <em>PpPRR3</em> and 2.5 for <em>PpPRR4</em>. The rates of induction for <em>PpPRR1</em> and <em>PpPRR3</em> are the highest and second highest, respectively, for any colour of light; this observation does not contradict the idea that there is an intraspecific divergence between <em>PpPRR1</em>/<em>PpPRR3</em> and <em>PpPRR2</em>/<em>PpPRR4</em>. The rates of induction by blue light and red light are lower than those of white light for any <em>PpPRR</em> gene. </p>                        <a id="5519738" scrollto-destination="5519738"></a>
<div id="DSQ033F7" data-id="DSQ033F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03307.jpeg?Expires=1509238741&Signature=We17o-sv-T61PY5XmkUk3kYw1KQIMicIggJP90eaPXvYzV0X6MtxEZETuoAG0QCA0LPq5SaZgLGPg82BlaUYotL6sszcEX9iB6atgKjHbLBF3Rqk4jJekpFSzPKPlc4leyerGvq4SgyquTOYc5Gq13b6lkw8ypb3REcC-tlLgLG9C2yhn-uqDOJIdD~Mk-Bw9zhAgCXl~7ipa4hgDLyDCpJRCr3otB0F6cctT9Vie1bDLPv6q4ycsy5VwjUVV5EWkL7cX-7EpEUD1nfEvz1ZB7ZV4HWfwxIGBe2bH9~Pn7FnM6aDESskQPy8rkchAL-yE0L5StmMrlYqeBlvpW3AUw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Light-induced expression of the PpPRR genes examined by the sqRT-PCR analysis. Physcomitrella patens protonemal cells were maintained under LL for 1 week, exposed to darkness for 24 h, and 2 h of white light, blue light, red light or dark period was administered before the cells were sampled. The fluence rate of each colour or light was 40 µmol m−2 s−1. Total RNAs were extracted, and the abundances of PpPRR mRNAs were measured and normalized to those for actin as a control. The graph shows the mRNA levels of PpPRRs induced by different colour of light (white light, open bars; blue light, bars with horizontal hatches; red light, bars with vertical hatches) relative to those from samples maintained in the prolonged darkness, the latter of which are set to 1.0. Values are the mean ± SD of three replications. The bottom photos show the hybridized bands for PpPRRs (upper four slips) or the control actin gene for each PpPRR (lower four slips), detected as chemiluminescence signals. We obtained similar results in two independent experiments." path-from-xml="dsq03307.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519738/dsq03307.jpeg" path-from-xml="dsq03307.jpeg" target="_blank">View large</a><a section="5519738" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03307.jpeg?Expires=1509238741&Signature=B1GHFPMQDXnthZpnbnVR58jrVrNpmjc4dbZKaKG~tMx3atLcpDcZj-22LSnXYFSrVefijrscdvb4cpKr36-anZBLaoUubSAx3iL0OYmhF2tn4r46D09i8gKM2jXumj60czMAgvHsoP7xkc-R6r82I9jEKgBVsMXTNMFZazp9MqKFnnUM4UhwrEFLhxZDRbRmZuClLllamQMLQteujTG4IjFbWfhx34dnzmD~I-5uQrBsBM5BB5XiGFjXjhgh6N0GpaQzDgFhNeCuJCJ4WKM8dD3HMKALbIAfFfXpnSQjimlSM-eoSERYJxhEq78brHaCTmwMyLY~b5NLf-YkyPSk8g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519738&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03307.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Light-induced expression of the <em>PpPRR</em> genes examined by the sqRT-PCR analysis. <em>Physcomitrella patens</em> protonemal cells were maintained under LL for 1 week, exposed to darkness for 24 h, and 2 h of white light, blue light, red light or dark period was administered before the cells were sampled. The fluence rate of each colour or light was 40 µmol m<sup>−2</sup> s<sup>−1</sup>. Total RNAs were extracted, and the abundances of <em>PpPRR</em> mRNAs were measured and normalized to those for <em>actin</em> as a control. The graph shows the mRNA levels of <em>PpPRRs</em> induced by different colour of light (white light, open bars; blue light, bars with horizontal hatches; red light, bars with vertical hatches) relative to those from samples maintained in the prolonged darkness, the latter of which are set to 1.0. Values are the mean ± SD of three replications. The bottom photos show the hybridized bands for <em>PpPRRs</em> (upper four slips) or the control <em>actin</em> gene for each <em>PpPRR</em> (lower four slips), detected as chemiluminescence signals. We obtained similar results in two independent experiments.</p></div></div><div content-id="DSQ033F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03307.jpeg?Expires=1509238741&Signature=We17o-sv-T61PY5XmkUk3kYw1KQIMicIggJP90eaPXvYzV0X6MtxEZETuoAG0QCA0LPq5SaZgLGPg82BlaUYotL6sszcEX9iB6atgKjHbLBF3Rqk4jJekpFSzPKPlc4leyerGvq4SgyquTOYc5Gq13b6lkw8ypb3REcC-tlLgLG9C2yhn-uqDOJIdD~Mk-Bw9zhAgCXl~7ipa4hgDLyDCpJRCr3otB0F6cctT9Vie1bDLPv6q4ycsy5VwjUVV5EWkL7cX-7EpEUD1nfEvz1ZB7ZV4HWfwxIGBe2bH9~Pn7FnM6aDESskQPy8rkchAL-yE0L5StmMrlYqeBlvpW3AUw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Light-induced expression of the PpPRR genes examined by the sqRT-PCR analysis. Physcomitrella patens protonemal cells were maintained under LL for 1 week, exposed to darkness for 24 h, and 2 h of white light, blue light, red light or dark period was administered before the cells were sampled. The fluence rate of each colour or light was 40 µmol m−2 s−1. Total RNAs were extracted, and the abundances of PpPRR mRNAs were measured and normalized to those for actin as a control. The graph shows the mRNA levels of PpPRRs induced by different colour of light (white light, open bars; blue light, bars with horizontal hatches; red light, bars with vertical hatches) relative to those from samples maintained in the prolonged darkness, the latter of which are set to 1.0. Values are the mean ± SD of three replications. The bottom photos show the hybridized bands for PpPRRs (upper four slips) or the control actin gene for each PpPRR (lower four slips), detected as chemiluminescence signals. We obtained similar results in two independent experiments." path-from-xml="dsq03307.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519738/dsq03307.jpeg" path-from-xml="dsq03307.jpeg" target="_blank">View large</a><a section="5519738" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03307.jpeg?Expires=1509238741&Signature=B1GHFPMQDXnthZpnbnVR58jrVrNpmjc4dbZKaKG~tMx3atLcpDcZj-22LSnXYFSrVefijrscdvb4cpKr36-anZBLaoUubSAx3iL0OYmhF2tn4r46D09i8gKM2jXumj60czMAgvHsoP7xkc-R6r82I9jEKgBVsMXTNMFZazp9MqKFnnUM4UhwrEFLhxZDRbRmZuClLllamQMLQteujTG4IjFbWfhx34dnzmD~I-5uQrBsBM5BB5XiGFjXjhgh6N0GpaQzDgFhNeCuJCJ4WKM8dD3HMKALbIAfFfXpnSQjimlSM-eoSERYJxhEq78brHaCTmwMyLY~b5NLf-YkyPSk8g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519738&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03307.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Light-induced expression of the <em>PpPRR</em> genes examined by the sqRT-PCR analysis. <em>Physcomitrella patens</em> protonemal cells were maintained under LL for 1 week, exposed to darkness for 24 h, and 2 h of white light, blue light, red light or dark period was administered before the cells were sampled. The fluence rate of each colour or light was 40 µmol m<sup>−2</sup> s<sup>−1</sup>. Total RNAs were extracted, and the abundances of <em>PpPRR</em> mRNAs were measured and normalized to those for <em>actin</em> as a control. The graph shows the mRNA levels of <em>PpPRRs</em> induced by different colour of light (white light, open bars; blue light, bars with horizontal hatches; red light, bars with vertical hatches) relative to those from samples maintained in the prolonged darkness, the latter of which are set to 1.0. Values are the mean ± SD of three replications. The bottom photos show the hybridized bands for <em>PpPRRs</em> (upper four slips) or the control <em>actin</em> gene for each <em>PpPRR</em> (lower four slips), detected as chemiluminescence signals. We obtained similar results in two independent experiments.</p></div></div>                            <h2 scrollto-destination=5519742 id="5519742" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The results of the current study help to understand the origins of the angiosperm <em>PRR</em> genes. In our <em>in vitro</em> assay, the PpPRR2 RLD peptide underwent phosphotransfer (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>), consistent with its complete DDK motif (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a), strongly suggesting that PpPRR2 functions as an RR. Besides, according to our phylogenetic tree (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>), PpPRRs are phylogenetically closely related to angiosperm PRRs. Therefore, it is presumed that the similarity of RLDs of PRRs to the authentic receiver domain is not only superficial but that PRRs were certainly derived from an authentic RR. Most probably, angiosperm PRRs have lost their phosphorelay function through the course of evolution, largely due to the substitution of the phosphoacceptor aspartic acid to a glutamic acid in their RLDs, and other critical residues such as the first aspartic acid of the DDK motif should have concomitantly diverged. Importantly, the <em>O. tauri</em> OtTOC1 and the <em>C. reinhardtii</em> CrPRR2 also share not only the potential phosphoacceptor aspartic acid but also the entire DDK motif (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a; see Corellou <em>et al</em>.<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup>). Although there has so far been no report about whether or not these algal proteins undergo phosphotransfer, they are also assumed to function as RRs, representing prototypic proteins of the land plant PRRs.</p><p>Our results also offer evolutionary explanations not only for the origin but also for the diversity of <em>PRRs</em> in land plants (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>). The pattern of our tree indicates that the TOC1 group first diverged, and then split the PRR9/5 group and the other branch, the latter containing the PRR7/3 group. Since PpPRRs are positioned inside the branch containing the PRR7/3 group, the origins of the TOC1 and PRR9/5 groups date back before the divergence of moss from higher plant lineages. The ancient origin of the TOC1 group is supported by the observation that <em>C. reinhardtii</em> and <em>C. variabilis</em> seem to have a TOC1 ortholog (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>). Furthermore, since the cluster of PpPRRs and lycophyte SmPRR7a/7b is sister to all the angiosperm PRR7/3 sequences (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>), the divergence between the PRR7/3 group and the ancestor of PpPRRs and SmPRR7a/7b also predate the divergence of moss from higher plants. Therefore, the common ancestor of moss and higher plants possessed <em>TOC1</em>, <em>PRR7/3</em> and <em>PRR9/5</em> orthologues in its genome, but these genes appear to have been lost later within the moss lineage (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>). In the lycophyte lineage, <em>PRR7/3</em> and <em>PRR9/5</em> orthologues have been lost and the unique combination of the <em>TOC1</em> (<em>SmTOC1</em>) and <em>PpPRR</em> (<em>SmPRR7a/7b</em>) orthologues still now remain (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>). In the angiosperm lineage, only <em>PpPRR</em> ortholog(s) has been lost, resulting in the current three <em>PRR</em> groups (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>). Concerning the timings of substitution of the phosphoacceptor residue, there are two alternative explanations. In one scenario (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>a), the phosphoacceptor aspartic acid residue of the PpPRRs and SmPRR7a/7b can be traced back to that found in the algal sequences; the phosphoacceptor residue in these sequences has never been substituted. This explanation might seem simple and more likely; however, if this were the case, the aspartic acid residue must have been substituted, according to the branching patterns of our tree, independently within each lineage of the TOC1, PRR7/3 and PRR9/5 groups (Figs <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>a). In another scenario (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>b), simpler at least in terms of the frequency of substitutions, the aspartic acid was substituted only once to a glutamic acid before the divergence of all the land plant PRR subfamilies. In this case, an aspartic acid was regained by a second substitution in the ancestral lineage of <em>PpPRRs</em> and two lycophyte genes (Fig. <a reveal-id="DSQ033F8" data-open="DSQ033F8" class="link link-reveal link-table xref-fig">8</a>b). If this were the case, <em>PpPRR1</em> might be, again, in the process of divergence from authentic RR-type sequences. In order to know which hypothesis, or yet another scenario, is more plausible, we need more sequence data; in particular, <em>PRR</em> sequences should be characterized from other primitive plants, i.e. liverworts, hornworts, ferns and gymnosperms. </p>                        <a id="5519748" scrollto-destination="5519748"></a>
<div id="DSQ033F8" data-id="DSQ033F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03308.jpeg?Expires=1509238741&Signature=A8av5E2ifsJYqN8FVkyo1TP4bTX4zAoQXcWMMiH1~08kRjl1Sl1A2eHDoxflLpkJPQLBxIqxEWcr6cnHbXakFqKDrZtvB0hoxpJtzD~Y5~aU9pIK-VHu5n~kMIsnY3hMmLNIvqi1TLlJhMU4tJESi2ZQss-aZN4CGlfr8Xi8pibliqYR14UU4HVMfuMGgzsdZREbQdHpQjGr1D24cYWIT3XyImNdJMHt8r3WgmXlJt3-yfDXl12GJuxO28ShQdK~7in1QNE4zhLoNBqptzlH9QdcmOM5D-7RqQO7Vh-5mD5pprHDddoRwrLn8nulfURJnpawWpbdbaK~1vNjzOvgQw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Evolutionary scenarios for divergence of PpPRR genes in land plants. Two alternative explanations are presented: (a) scenario in which the phosphoacceptor aspartic acid residue in PpPRR and SmPRR7a/7b sequences has never been substituted and (b) scenario in which the aspartic acid was substituted to glutamic acid only once and it was regained by a second substitution in the ancestral lineage of PpPRRs and two lycophyte genes. Closed circles represent amino acid substitution events at the potential phosphoacceptor residue. Grey circles represent alternative timings of amino acid substitutions indicated by the closed circles. Crosses represent gene loss events (see the main text for detailed description)." path-from-xml="dsq03308.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519748/dsq03308.jpeg" path-from-xml="dsq03308.jpeg" target="_blank">View large</a><a section="5519748" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03308.jpeg?Expires=1509238741&Signature=IHRkTh47UV-O99fOv2p4p8X309ysF36aMap0P6iA6imbNKpltmmsgODWoBQekD~s-SluTVFYcARdjHKlP0CQ7nfrANIT1K6pztkHi9gVob8gwDhi2xt5nFe~36-lt6JAiu09CYuLXAfxuu-bzxTHk42OVHorOHoHIbPSkpfWOVtUczGt7ujpOd4s8WIpPzO0HSR-KJOEUOp~VviuQztpeA1DiCNVr0zlwmxpxGJNGRKzsEyJQ8g8aNwSrQMLF~-3RYSlKV9r-Hb74~YO1vIgdxe2G5IyorckhBcjHpBSBLYEaIXkfETaYVFVcMHB-6yiLI9cLMOFk5yrLE~EOtIYsw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519748&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03308.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary scenarios for divergence of <em>PpPRR</em> genes in land plants. Two alternative explanations are presented: (a) scenario in which the phosphoacceptor aspartic acid residue in PpPRR and SmPRR7a/7b sequences has never been substituted and (b) scenario in which the aspartic acid was substituted to glutamic acid only once and it was regained by a second substitution in the ancestral lineage of <em>PpPRRs</em> and two lycophyte genes. Closed circles represent amino acid substitution events at the potential phosphoacceptor residue. Grey circles represent alternative timings of amino acid substitutions indicated by the closed circles. Crosses represent gene loss events (see the main text for detailed description).</p></div></div><div content-id="DSQ033F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/m_dsq03308.jpeg?Expires=1509238741&Signature=A8av5E2ifsJYqN8FVkyo1TP4bTX4zAoQXcWMMiH1~08kRjl1Sl1A2eHDoxflLpkJPQLBxIqxEWcr6cnHbXakFqKDrZtvB0hoxpJtzD~Y5~aU9pIK-VHu5n~kMIsnY3hMmLNIvqi1TLlJhMU4tJESi2ZQss-aZN4CGlfr8Xi8pibliqYR14UU4HVMfuMGgzsdZREbQdHpQjGr1D24cYWIT3XyImNdJMHt8r3WgmXlJt3-yfDXl12GJuxO28ShQdK~7in1QNE4zhLoNBqptzlH9QdcmOM5D-7RqQO7Vh-5mD5pprHDddoRwrLn8nulfURJnpawWpbdbaK~1vNjzOvgQw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Evolutionary scenarios for divergence of PpPRR genes in land plants. Two alternative explanations are presented: (a) scenario in which the phosphoacceptor aspartic acid residue in PpPRR and SmPRR7a/7b sequences has never been substituted and (b) scenario in which the aspartic acid was substituted to glutamic acid only once and it was regained by a second substitution in the ancestral lineage of PpPRRs and two lycophyte genes. Closed circles represent amino acid substitution events at the potential phosphoacceptor residue. Grey circles represent alternative timings of amino acid substitutions indicated by the closed circles. Crosses represent gene loss events (see the main text for detailed description)." path-from-xml="dsq03308.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5519748/dsq03308.jpeg" path-from-xml="dsq03308.jpeg" target="_blank">View large</a><a section="5519748" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq03308.jpeg?Expires=1509238741&Signature=IHRkTh47UV-O99fOv2p4p8X309ysF36aMap0P6iA6imbNKpltmmsgODWoBQekD~s-SluTVFYcARdjHKlP0CQ7nfrANIT1K6pztkHi9gVob8gwDhi2xt5nFe~36-lt6JAiu09CYuLXAfxuu-bzxTHk42OVHorOHoHIbPSkpfWOVtUczGt7ujpOd4s8WIpPzO0HSR-KJOEUOp~VviuQztpeA1DiCNVr0zlwmxpxGJNGRKzsEyJQ8g8aNwSrQMLF~-3RYSlKV9r-Hb74~YO1vIgdxe2G5IyorckhBcjHpBSBLYEaIXkfETaYVFVcMHB-6yiLI9cLMOFk5yrLE~EOtIYsw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5519748&ar=378044&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03308.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary scenarios for divergence of <em>PpPRR</em> genes in land plants. Two alternative explanations are presented: (a) scenario in which the phosphoacceptor aspartic acid residue in PpPRR and SmPRR7a/7b sequences has never been substituted and (b) scenario in which the aspartic acid was substituted to glutamic acid only once and it was regained by a second substitution in the ancestral lineage of <em>PpPRRs</em> and two lycophyte genes. Closed circles represent amino acid substitution events at the potential phosphoacceptor residue. Grey circles represent alternative timings of amino acid substitutions indicated by the closed circles. Crosses represent gene loss events (see the main text for detailed description).</p></div></div><p>Our results demonstrate intraspecific divergences among the <em>PpPRR</em> genes. The phylogenetic tree (Fig. <a reveal-id="DSQ033F2" data-open="DSQ033F2" class="link link-reveal link-table xref-fig">2</a>), intron insertion sites (Fig. <a reveal-id="DSQ033F3" data-open="DSQ033F3" class="link link-reveal link-table xref-fig">3</a>) and expression profiles (Figs <a reveal-id="DSQ033F5  DSQ033F7" data-open="DSQ033F5  DSQ033F7" class="link link-reveal link-table xref-fig">5–7</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file 4</a>) suggest the divergence between <em>PpPRR1/PpPRR3</em> and <em>PpPRR2/PpPRR4</em>. The <em>A. thaliana PRR</em> family members show differentially regulated expression profiles, reflecting the fact that they act at different nodes in the circadian network and functionally diverged from one another.<sup><a reveal-id="DSQ033C4" data-open="DSQ033C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSQ033C6" data-open="DSQ033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Therefore, <em>PpPRR1/PpPRR3</em> and <em>PpPRR2/PpPRR4</em> are also predicted to be functionally diverged. It should be noted that Holm <em>et al</em>.<sup><a reveal-id="DSQ033C23" data-open="DSQ033C23" class="link link-ref link-reveal xref-bibr">23</a></sup> did not detect any differential expression among the four genes in a light–dark cycle followed by DD. This may be because their light–dark regime is different from ours: they used a long day regime, a 16-h light and 8-h dark cycle, whereas ours is a 12-h light and 12-h dark cycle(s) (12:12LD; Figs <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a> and <a reveal-id="DSQ033F6" data-open="DSQ033F6" class="link link-reveal link-table xref-fig">6</a>). All <em>PpPRRs</em> are induced by light (Fig. <a reveal-id="DSQ033F7" data-open="DSQ033F7" class="link link-reveal link-table xref-fig">7</a>), and this light responsiveness may have apparently synchronized the trough phases of <em>PpPRRs</em> with the earlier dawn in the short night of their long day regime, whereas difference in trough phases between <em>PpPRR1/PpPRR3</em> and <em>PpPRR2/PpPRR4</em> are obvious in our longer nights (Fig. <a reveal-id="DSQ033F5" data-open="DSQ033F5" class="link link-reveal link-table xref-fig">5</a>a).</p><p>On the other hand, the results of the <em>in vitro</em> assay (Fig. <a reveal-id="DSQ033F4" data-open="DSQ033F4" class="link link-reveal link-table xref-fig">4</a>) and detailed sequence comparison (Fig. <a reveal-id="DSQ033F1" data-open="DSQ033F1" class="link link-reveal link-table xref-fig">1</a>a) suggest that PpPRR2/PpPRR3/PpPRR4 function as RRs, whereas PpPRR1 does not or its phosphotransfer ability is very weak. The moss PpPRRs, if not all of them, are anticipated to have some clock-associated functions as do <em>A. thaliana</em> PRRs, from the following facts: (i) the circadian networks of <em>A. thaliana</em> and <em>P. patens</em> are predicted to be at least partially conserved<sup><a reveal-id="DSQ033C20" data-open="DSQ033C20" class="link link-ref link-reveal xref-bibr">20</a></sup>; and (ii) <em>O. tauri</em>, which belongs to green algae, the closest relative of land plants, has a <em>PRR</em> homologue that functions as a clock gene.<sup><a reveal-id="DSQ033C16" data-open="DSQ033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Since RRs are generally involved in signalling cascades responsive to environmental/endogenous signals, PpPRR2/PpPRR3/PpPRR4 might function, rather than in the core clock circuitry, in an input pathway(s) that must be responsive to environmental cues such as light and temperature.</p><p>In a future study, in parallel with gene knock out experiments, it should also be addressed whether PpPRRs are involved in phosphotransfer functions <em>in planta</em>. Moreover, a HK(s) and a HPt(s) that are partners to (at least some of) PpPRR proteins should be characterized. It might not be easy to identify a partner HK(s) because the <em>P. patens</em> genome contains as many as ∼50 HK sequences,<sup><a reveal-id="DSQ033C22" data-open="DSQ033C22" class="link link-ref link-reveal xref-bibr">22</a></sup> unlike <em>A. thaliana</em>, which contains only 8 HK genes.<sup><a reveal-id="DSQ033C41" data-open="DSQ033C41" class="link link-ref link-reveal xref-bibr">41</a></sup> However, the availability of the entire genome sequence, many full-length cDNA clones and the tractability of gene functional analysis based on gene targeting techniques will support the identification of such HK and HPt genes in <em>P. patens</em>, possibly as novel clock genes.</p>                            <h3 scrollto-destination=5519759 id="5519759" class="section-title" data-legacy-id=s4a>4.1. Conclusion</h3>
<p>Here, we demonstrated that the moss PpPRRs have close relationships with angiosperm PRRs and at least PpPRR2 can function as an RR, indicating that the plant clock-associated <em>PRR</em> families are derived from an authentic RR(s). Moreover, here, we offered evolutionary explanations for divergence of <em>PRR</em> genes in land plants. This study should be a foundation for understanding how a particular set of genes involved in regulation of the clock evolved their functions, in coordination with changes (or maintenance) of the clock mechanisms.</p>                            <h3 scrollto-destination=5519764 id="5519764" class="section-title" data-legacy-id=s4b>4.2. Accession numbers for the sequence data</h3>
<p>The sequences reported in this paper have been submitted to the public databases [DDBJ/EMBL/GenBank: AB558266 (<em>PpPRR1</em>), AB558268 (<em>PpPRR2</em>), AB558267 (<em>PpPRR3</em>) and AB558269 (<em>PpPRR4</em>)].</p>                            <h2 scrollto-destination=5519769 id="5519769" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5519773 id="5519773" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by grants from the Ministry of Education, Culture, Sports, Science and Technology Japan (MEXT) to S.B.S. and from the Japan Society for the Promotion of Science (JSPS) (21570005 to S.A. and 21780092 to T.Y.).</p>                            <h2 scrollto-destination=5519776 id="5519776" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Yuichi Oba (Nagoya University) and Kyoko Kanamaru (Nagoya University) for valuable advice and helpful discussion. We also thank Mayu Nakagawa (Nagoya University) and K. Sreenath (University of Massachusetts Medical School) for critical reading of the manuscript and Andrew Millar (Department of Biological Science, University of Edinburgh) for the BRASS software package.</p>                            <h2 scrollto-destination=5519778 id="5519778" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsq033c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edmunds</div> <div class="given-names">L.N.</div></div>. </div>, <div class="source">Cellular and Molecular Bases of Biological Clocks</div> , <div class="year">1988</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div></div></div></div></div><div content-id="dsq033c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pittendrigh</div> <div class="given-names">C.S.</div></div>. </div><div class="article-title">Temporal organization: reflections of a Darwinian clock-watcher</div>, <div class="source">Annu. Rev. Physiol.</div> , <div class="year">1993</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">16</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporal%20organization%3A%20reflections%20of%20a%20Darwinian%20clock-watcher&amp;author=C.S.%20Pittendrigh&amp;publication_year=1993&amp;journal=Annu.%20Rev.%20Physiol.&amp;volume=55&amp;pages=16-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ph.55.030193.000313" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ph.55.030193.000313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ph.55.030193.000313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8466172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">E.E.</div></div>,  <div class="name"><div class="surname">Kay</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">SnapShot: circadian clock proteins</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">368</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2008.09.042" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.cell.2008.09.042</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SnapShot%3A%20circadian%20clock%20proteins&amp;author=E.E.%20Hamilton&amp;author=S.A.%20Kay&amp;publication_year=2008&amp;journal=Cell&amp;volume=135&amp;pages=368-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.09.042" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.09.042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.09.042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18957209" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamichi</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Pseudo-response regulators (PRRs) or true oscillator components (TOCs)</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">677</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pci087" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pci087</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pseudo-response%20regulators%20(PRRs)%20or%20true%20oscillator%20components%20(TOCs)&amp;author=T.%20Mizuno&amp;author=N.%20Nakamichi&amp;publication_year=2005&amp;journal=Plant%20Cell%20Physiol.&amp;volume=46&amp;pages=677-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pci087" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpci087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpci087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15767264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harmer</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">The circadian system in higher plants</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">357</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.043008.092054" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1146/annurev.arplant.043008.092054</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20circadian%20system%20in%20higher%20plants&amp;author=S.L.%20Harmer&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=60&amp;pages=357-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.043008.092054" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.043008.092054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.043008.092054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19575587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Two-component phosphorelay signal transduction systems in plants: from hormone responses to circadian rhythms</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">2263</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1271/bbb.69.2263" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1271/bbb.69.2263</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two-component%20phosphorelay%20signal%20transduction%20systems%20in%20plants%3A%20from%20hormone%20responses%20to%20circadian%20rhythms&amp;author=T.%20Mizuno&amp;publication_year=2005&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=69&amp;pages=2263-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.69.2263" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.69.2263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.69.2263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16377883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makino</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kiba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Imamura</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genes encoding pseudo-response regulators: insight into His-to-Asp phosphorelay and circadian rhythm in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2000</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">791</div>-<div class="lpage">803</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20encoding%20pseudo-response%20regulators%3A%20insight%20into%20His-to-Asp%20phosphorelay%20and%20circadian%20rhythm%20in%20Arabidopsis%20thaliana&amp;author=S.%20Makino&amp;author=T.%20Kiba&amp;author=A.%20Imamura&amp;publication_year=2000&amp;journal=Plant%20Cell%20Physiol.&amp;volume=41&amp;pages=791-803" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/41.6.791" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2F41.6.791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2F41.6.791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10945350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murakami</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ashikari</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamashino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The evolutionarily conserved <em>OsPRR</em> quintet: rice pseudo-response regulators implicated in circadian rhythm</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">1229</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcg135" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pcg135</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionarily%20conserved%20OsPRR%20quintet%3A%20rice%20pseudo-response%20regulators%20implicated%20in%20circadian%20rhythm&amp;author=M.%20Murakami&amp;author=M.%20Ashikari&amp;author=K.%20Miura&amp;author=T.%20Yamashino&amp;author=T.%20Mizuno&amp;publication_year=2003&amp;journal=Plant%20Cell%20Physiol.&amp;volume=44&amp;pages=1229-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcg135" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcg135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcg135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14634161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hecht</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Foucher</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ferrándiz</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Conservation of <em>Arabidopsis</em> flowering genes in model legumes</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">1420</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.057018" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.104.057018</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20Arabidopsis%20flowering%20genes%20in%20model%20legumes&amp;author=V.%20Hecht&amp;author=F.%20Foucher&amp;author=C.%20Ferr%C3%A1ndiz&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=137&amp;pages=1420-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.057018" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.057018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.057018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15778459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hecht</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Knowles</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Vander Schoor</div> <div class="given-names">J.K.</div></div>, et al. </div><div class="article-title">Pea <em>LATE BLOOMER1</em> is a <em>GIGANTEA</em> ortholog with roles in photoperiodic flowering, deetiolation, and transcriptional regulation of circadian clock gene homologs</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">648</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.107.096818" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.107.096818</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pea%20LATE%20BLOOMER1%20is%20a%20GIGANTEA%20ortholog%20with%20roles%20in%20photoperiodic%20flowering%2C%20deetiolation%2C%20and%20transcriptional%20regulation%20of%20circadian%20clock%20gene%20homologs&amp;author=V.%20Hecht&amp;author=C.L.%20Knowles&amp;author=J.K.%20Vander%20Schoor&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=144&amp;pages=648-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.096818" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.096818" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.096818"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17468223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramos</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pérez-Solís</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ibáñez</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Winter disruption of the circadian clock in chestnut</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">7037</div>-<div class="lpage">42</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0408549102" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1073/pnas.0408549102</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Winter%20disruption%20of%20the%20circadian%20clock%20in%20chestnut&amp;author=A.%20Ramos&amp;author=E.%20P%C3%A9rez-Sol%C3%ADs&amp;author=C.%20Ib%C3%A1%C3%B1ez&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=7037-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0408549102" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0408549102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0408549102"> </span></div> </div></div></div></div></div><div content-id="dsq033c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miwa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Serikawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kondo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Oyama</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Conserved expression profiles of circadian clock-related genes in two <em>Lemna</em> species showing long-day and short-day photoperiodic flowering responses</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">601</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcj027" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pcj027</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20expression%20profiles%20of%20circadian%20clock-related%20genes%20in%20two%20Lemna%20species%20showing%20long-day%20and%20short-day%20photoperiodic%20flowering%20responses&amp;author=K.%20Miwa&amp;author=M.%20Serikawa&amp;author=S.%20Suzuki&amp;author=T.%20Kondo&amp;author=T.%20Oyama&amp;publication_year=2005&amp;journal=Plant%20Cell%20Physiol.&amp;volume=47&amp;pages=601-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcj027" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcj027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcj027"> </span></div> </div></div></div></div></div><div content-id="dsq033c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamashino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yokoyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Three type-B response regulators, ARR1, ARR10 and ARR12, play essential but redundant roles in cytokinin signal transduction throughout the life cycle of <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">47</div>-<div class="lpage">57</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcm165" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pcm165</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20type-B%20response%20regulators%2C%20ARR1%2C%20ARR10%20and%20ARR12%2C%20play%20essential%20but%20redundant%20roles%20in%20cytokinin%20signal%20transduction%20throughout%20the%20life%20cycle%20of%20Arabidopsis%20thaliana&amp;author=K.%20Ishida&amp;author=T.%20Yamashino&amp;author=A.%20Yokoyama&amp;author=T.%20Mizuno&amp;publication_year=2008&amp;journal=Plant%20Cell%20Physiol.&amp;volume=49&amp;pages=47-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm165" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18037673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Analysis of clock gene homologs using unifoliolates as target organs in soybean (<em>Glycine max</em>)</div>, <div class="source">J. Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">166</div> (pg. <div class="fpage">278</div>-<div class="lpage">89</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jplph.2008.06.003" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.jplph.2008.06.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20clock%20gene%20homologs%20using%20unifoliolates%20as%20target%20organs%20in%20soybean%20(Glycine%20max)&amp;author=H.%20Liu&amp;author=H.%20Wang&amp;author=P.%20Gao&amp;publication_year=2009&amp;journal=J.%20Plant%20Physiol.&amp;volume=166&amp;pages=278-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2008.06.003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2008.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2008.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18707796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takata</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Satio</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Uemura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Phylogenetic footprint of the plant clock system in angiosperms: evolutionary processes of pseudo-response regulators</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">126</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2148-10-126" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1186/1471-2148-10-126</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20footprint%20of%20the%20plant%20clock%20system%20in%20angiosperms%3A%20evolutionary%20processes%20of%20pseudo-response%20regulators&amp;author=N.%20Takata&amp;author=S.%20Saito&amp;author=C.T.%20Satio&amp;author=M.%20Uemura&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol.&amp;volume=10&amp;pages=126" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-126" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20433765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Corellou</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Schwartz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Motta</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Djouani-Tahri el</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sanchez</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bouget</div> <div class="given-names">F.Y.</div></div>. </div><div class="article-title">Clocks in the green lineage: comparative functional analysis of the circadian architecture of the picoeukaryote <em>Ostreococcus</em></div>, <div class="source">Plant Cell</div> , <div class="year">2009</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3436</div>-<div class="lpage">49</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.109.068825" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1105/tpc.109.068825</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clocks%20in%20the%20green%20lineage%3A%20comparative%20functional%20analysis%20of%20the%20circadian%20architecture%20of%20the%20picoeukaryote%20Ostreococcus&amp;author=F.%20Corellou&amp;author=C.%20Schwartz&amp;author=J.P.%20Motta&amp;author=B.%20Djouani-Tahri%20el&amp;author=F.%20Sanchez&amp;author=F.Y.%20Bouget&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=3436-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.068825" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.068825" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.068825"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19948792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bowman</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Floyd</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Sakakibara</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Green genes-comparative genomics of the green branch of life</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">229</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2007.04.004" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.cell.2007.04.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Green%20genes-comparative%20genomics%20of%20the%20green%20branch%20of%20life&amp;author=J.L.%20Bowman&amp;author=S.K.%20Floyd&amp;author=K.%20Sakakibara&amp;publication_year=2007&amp;journal=Cell&amp;volume=129&amp;pages=229-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.04.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.04.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17448980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zimmer</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Rensing</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Reski</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Exploring plant biodiversity: the <em>Physcomitrella</em> genome and beyond</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">542</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tplants.2008.07.002" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.tplants.2008.07.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20plant%20biodiversity%3A%20the%20Physcomitrella%20genome%20and%20beyond&amp;author=D.%20Lang&amp;author=A.D.%20Zimmer&amp;author=S.A.%20Rensing&amp;author=R.%20Reski&amp;publication_year=2008&amp;journal=Trends%20Plant%20Sci.&amp;volume=13&amp;pages=542-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2008.07.002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2008.07.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2008.07.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18762443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schaefer</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Zrÿd</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Efficient gene targeting in the moss <em>Physcomitrella patens</em></div>, <div class="source">Plant J.</div> , <div class="year">1997</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1195</div>-<div class="lpage">206</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.1997.11061195.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1046/j.1365-313X.1997.11061195.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20gene%20targeting%20in%20the%20moss%20Physcomitrella%20patens&amp;author=D.G.%20Schaefer&amp;author=J.P.%20Zr%C3%BFd&amp;publication_year=1997&amp;journal=Plant%20J.&amp;volume=11&amp;pages=1195-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1997.11061195.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1997.11061195.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1997.11061195.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9225463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okada</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kondo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Satbhai</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tsukuda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Functional characterization of <em>CCA1</em>/<em>LHY</em> homolog genes, <em>PpCCA1a</em> and <em>PpCCA1b</em>, in the moss <em>Physcomitrella patens</em></div>, <div class="source">Plant J.</div> , <div class="year">2009</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">551</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2009.03979.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-313X.2009.03979.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20characterization%20of%20CCA1%2FLHY%20homolog%20genes%2C%20PpCCA1a%20and%20PpCCA1b%2C%20in%20the%20moss%20Physcomitrella%20patens&amp;author=R.%20Okada&amp;author=S.%20Kondo&amp;author=S.B.%20Satbhai&amp;author=N.%20Yamaguchi&amp;author=M.%20Tsukuda&amp;author=S.%20Aoki&amp;publication_year=2009&amp;journal=Plant%20J.&amp;volume=60&amp;pages=551-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.60.issue-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.60.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.60.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19624471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="collab">DOE Joint Genome Institute</div><div class="article-title"><em>P. patens</em> subsp <em>patens</em> V1.1</div> <div class="comment"><a class="link link-uri" href="http://genome.jgi-psf.org//Phypa1_1/Phypa1_1.home.html">http://genome.jgi-psf.org//Phypa1_1/Phypa1_1.home.html</a></div></div></div></div></div><div content-id="dsq033c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rensing</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Lang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zimmer</div> <div class="given-names">A.D.</div></div>, et al. </div><div class="article-title">The <em>Physcomitrella</em> genome reveals evolutionary insights into the conquest of land by plants</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">319</div> (pg. <div class="fpage">64</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1150646" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1126/science.1150646</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Physcomitrella%20genome%20reveals%20evolutionary%20insights%20into%20the%20conquest%20of%20land%20by%20plants&amp;author=S.A.%20Rensing&amp;author=D.%20Lang&amp;author=A.D.%20Zimmer&amp;publication_year=2010&amp;journal=Science&amp;volume=319&amp;pages=64-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1150646" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1150646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1150646"> </span></div> </div></div></div></div></div><div content-id="dsq033c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holm</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Källman</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gyllenstrand</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hedman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lagercrantz</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Does the core circadian clock in the moss <em>Physcomitrella patens</em> (Bryophyta) comprise a single loop?</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">109</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-109" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1186/1471-2229-10-109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Does%20the%20core%20circadian%20clock%20in%20the%20moss%20Physcomitrella%20patens%20(Bryophyta)%20comprise%20a%20single%20loop%3F&amp;author=K.%20Holm&amp;author=T.%20K%C3%A4llman&amp;author=N.%20Gyllenstrand&amp;author=H.%20Hedman&amp;author=U.%20Lagercrantz&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20550695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashton</div> <div class="given-names">N.W.</div></div>,  <div class="name"><div class="surname">Cove</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">The isolation and preliminary characterization of auxotrophic and analogue resistant mutants in the moss <em>Physcomitrella patens</em></div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1977</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">87</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00265581" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/BF00265581</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20isolation%20and%20preliminary%20characterization%20of%20auxotrophic%20and%20analogue%20resistant%20mutants%20in%20the%20moss%20Physcomitrella%20patens&amp;author=N.W.%20Ashton&amp;author=D.J.%20Cove&amp;publication_year=1977&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=154&amp;pages=87-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00265581" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00265581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00265581"> </span></div> </div></div></div></div></div><div content-id="dsq033c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nishiyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hiwatashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sakakibara</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hasebe</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Tagged mutagenesis and gene-trap in the moss, <em>Physcomitrella patens</em> by shuttle mutagenesis</div>, <div class="source">DNA Res.</div> , <div class="year">2000</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">9</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/7.1.9" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/dnares/7.1.9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tagged%20mutagenesis%20and%20gene-trap%20in%20the%20moss%2C%20Physcomitrella%20patens%20by%20shuttle%20mutagenesis&amp;author=T.%20Nishiyama&amp;author=Y.%20Hiwatashi&amp;author=I.%20Sakakibara&amp;author=M.%20Kato&amp;author=M.%20Hasebe&amp;publication_year=2000&amp;journal=DNA%20Res.&amp;volume=7&amp;pages=9-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/7.1.9" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F7.1.9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F7.1.9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10718194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">CLUSTAL: a package for performing multiple sequence alignment on a microcomputer</div>, <div class="source">Gene</div> , <div class="year">1988</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">237</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0378-1119(88)90330-7" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/0378-1119(88)90330-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%3A%20a%20package%20for%20performing%20multiple%20sequence%20alignment%20on%20a%20microcomputer&amp;author=D.G.%20Higgins&amp;author=P.M.%20Sharp&amp;publication_year=1988&amp;journal=Gene&amp;volume=73&amp;pages=237-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(88)90330-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(88)90330-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(88)90330-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3243435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">D.T.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">The rapid generation of mutation data matrices from protein sequences</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1992</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">275</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=D.T.%20Jones&amp;author=W.R.%20Taylor&amp;author=J.M.%20Thornton&amp;publication_year=1992&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=8&amp;pages=275-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1633570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rzhetsky</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A simple method for estimating and testing minimum evolution trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1992</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">945</div>-<div class="lpage">67</div>)</div></div></div></div><div content-id="dsq033c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Confidence limits on phylogenies: an approach using the bootstrap</div>, <div class="source">Evolution</div> , <div class="year">1985</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">783</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2408678" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.2307/2408678</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;author=J.%20Felsenstein&amp;publication_year=1985&amp;journal=Evolution&amp;volume=39&amp;pages=783-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2408678" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2408678" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2408678"> </span></div> </div></div></div></div></div><div content-id="dsq033c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1596</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msm092" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/molbev/msm092</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lethiec</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Duroux</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div>. </div><div class="article-title">PHYML Online–a web server for fast maximum likelihood-based phylogenetic inference</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W557</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gki352" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/nar/gki352</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHYML%20Online%E2%80%93a%20web%20server%20for%20fast%20maximum%20likelihood-based%20phylogenetic%20inference&amp;author=S.%20Guindon&amp;author=F.%20Lethiec&amp;author=P.%20Duroux&amp;author=O.%20Gascuel&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W557-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki352" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamichi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Niinuma</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamashino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title"><em>Arabidopsis</em> clock-associated pseudo-response regulators PRR9, PRR7 and PRR5 coordinately and positively regulate flowering time through the canonical CONSTANS-dependent photoperiodic pathway</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">822</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcm056" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pcm056</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20clock-associated%20pseudo-response%20regulators%20PRR9%2C%20PRR7%20and%20PRR5%20coordinately%20and%20positively%20regulate%20flowering%20time%20through%20the%20canonical%20CONSTANS-dependent%20photoperiodic%20pathway&amp;author=N.%20Nakamichi&amp;author=M.%20Kita&amp;author=K.%20Niinuma&amp;author=S.%20Ito&amp;author=T.%20Yamashino&amp;author=T.%20Mizoguchi&amp;author=T.%20Mizuno&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=822-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm056" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17504813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowry</div> <div class="given-names">O.H.</div></div>,  <div class="name"><div class="surname">Rosebrough</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Farr</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Randall</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Protein measurement with the Folin phenol reagent</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1951</div>, vol. <div class="volume">193</div> (pg. <div class="fpage">265</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20measurement%20with%20the%20Folin%20phenol%20reagent&amp;author=O.H.%20Lowry&amp;author=N.J.%20Rosebrough&amp;author=A.L.%20Farr&amp;author=R.J.%20Randall&amp;publication_year=1951&amp;journal=J.%20Biol.%20Chem.&amp;volume=193&amp;pages=265-75" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14907713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuzuki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ishige</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Phosphotransfer circuitry of the putative multi-signal transducer, ArcB, of <em>Escherichia coli</em>: <em>in vitro</em> studies with mutants</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1995</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">953</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-2958.1995.18050953.x" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/j.1365-2958.1995.18050953.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphotransfer%20circuitry%20of%20the%20putative%20multi-signal%20transducer%2C%20ArcB%2C%20of%20Escherichia%20coli%3A%20in%20vitro%20studies%20with%20mutants&amp;author=M.%20Tsuzuki&amp;author=K.%20Ishige&amp;author=T.%20Mizuno&amp;publication_year=1995&amp;journal=Mol.%20Microbiol.&amp;volume=18&amp;pages=953-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1995.18.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1995.18.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1995.18.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8825099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Azuma</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kanamaru</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Matsushika</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamashino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">T.</div></div>. </div><div class="article-title"><em>In vitro</em> analysis of His-Asp phosphorelays in <em>Aspergillus nidulans</em>: the first direct biochemical evidence for the existence of His-Asp phosphotransfer systems in filamentous fungi</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">2493</div>-<div class="lpage">502</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1271/bbb.70292" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1271/bbb.70292</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vitro%20analysis%20of%20His-Asp%20phosphorelays%20in%20Aspergillus%20nidulans%3A%20the%20first%20direct%20biochemical%20evidence%20for%20the%20existence%20of%20His-Asp%20phosphotransfer%20systems%20in%20filamentous%20fungi&amp;author=N.%20Azuma&amp;author=K.%20Kanamaru&amp;author=A.%20Matsushika&amp;author=T.%20Yamashino&amp;author=T.%20Mizuno&amp;author=M.%20Kato&amp;author=T.%20Kobayashi&amp;publication_year=2007&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=71&amp;pages=2493-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.70292" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.70292" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.70292"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17928704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkinson</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Kofoid</div> <div class="given-names">E.C.</div></div>. </div><div class="article-title">Communication modules in bacterial signaling proteins</div>, <div class="source">Ann. Rev. Genet.</div> , <div class="year">1992</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">71</div>-<div class="lpage">112</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.ge.26.120192.000443" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1146/annurev.ge.26.120192.000443</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Communication%20modules%20in%20bacterial%20signaling%20proteins&amp;author=J.S.%20Parkinson&amp;author=E.C.%20Kofoid&amp;publication_year=1992&amp;journal=Ann.%20Rev.%20Genet.&amp;volume=26&amp;pages=71-112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.26.120192.000443" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.26.120192.000443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.26.120192.000443"> </span></div> </div></div></div></div></div><div content-id="dsq033c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ueguchi</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Koizumi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Novel family of sensor histidine kinase genes in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">231</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pce015" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pce015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20family%20of%20sensor%20histidine%20kinase%20genes%20in%20Arabidopsis%20thaliana&amp;author=C.%20Ueguchi&amp;author=H.%20Koizumi&amp;author=T.%20Suzuki&amp;author=T.%20Mizuno&amp;publication_year=2001&amp;journal=Plant%20Cell%20Physiol.&amp;volume=42&amp;pages=231-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pce015" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpce015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpce015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aoki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ichikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Circadian expression of the <em>PpLhcb2</em> gene encoding a major light-harvesting chlorophyll a/b-binding protein in the moss <em>Physcomitrella patens</em></div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">68</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pch006" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/pcp/pch006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circadian%20expression%20of%20the%20PpLhcb2%20gene%20encoding%20a%20major%20light-harvesting%20chlorophyll%20a%2Fb-binding%20protein%20in%20the%20moss%20Physcomitrella%20patens&amp;author=S.%20Aoki&amp;author=S.%20Kato&amp;author=K.%20Ichikawa&amp;author=M.%20Shimizu&amp;publication_year=2004&amp;journal=Plant%20Cell%20Physiol.&amp;volume=45&amp;pages=68-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pch006" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpch006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpch006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14749487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ichikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Imaizumi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Differential expression on a daily basis of plastid sigma factor <em>PpSig</em> genes from the moss <em>Physcomitrella patens</em>: regulatory interactions among <em>PpSig5</em>, the circadian clock and blue light signaling mediated by cryptochromes</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">4285</div>-<div class="lpage">98</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.053033" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.104.053033</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20on%20a%20daily%20basis%20of%20plastid%20sigma%20factor%20PpSig%20genes%20from%20the%20moss%20Physcomitrella%20patens%3A%20regulatory%20interactions%20among%20PpSig5%2C%20the%20circadian%20clock%20and%20blue%20light%20signaling%20mediated%20by%20cryptochromes&amp;author=K.%20Ichikawa&amp;author=M.%20Sugita&amp;author=T.%20Imaizumi&amp;author=M.%20Wada&amp;author=S.%20Aoki&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=136&amp;pages=4285-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.053033" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.053033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.053033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15563615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ichikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Photoperiod-regulated expression of the PpCOL1 gene encoding a homolog of CO/COL proteins in the moss <em>Physcomitrella patens</em></div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2004</div>, vol. <div class="volume">324</div> (pg. <div class="fpage">1296</div>-<div class="lpage">301</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2004.09.194" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/j.bbrc.2004.09.194</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Photoperiod-regulated%20expression%20of%20the%20PpCOL1%20gene%20encoding%20a%20homolog%20of%20CO%2FCOL%20proteins%20in%20the%20moss%20Physcomitrella%20patens&amp;author=M.%20Shimizu&amp;author=K.%20Ichikawa&amp;author=S.%20Aoki&amp;publication_year=2004&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=324&amp;pages=1296-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2004.09.194" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2004.09.194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2004.09.194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15504355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.C.</div></div>,  <div class="name"><div class="surname">Sheen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Two-component signal transduction pathways in <em>Arabidopsis</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2002</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">500</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.005504" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1104/pp.005504</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two-component%20signal%20transduction%20pathways%20in%20Arabidopsis&amp;author=I.%20Hwang&amp;author=H.C.%20Chen&amp;author=J.%20Sheen&amp;publication_year=2002&amp;journal=Plant%20Physiol.&amp;volume=129&amp;pages=500-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.005504" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.005504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.005504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12068096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsq033c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsq033c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Niwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamashino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A genome-wide compilation of the two-component systems in <em>Lotus japonicas</em></div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">237</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsp012" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/dnares/dsp012</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20compilation%20of%20the%20two-component%20systems%20in%20Lotus%20japonicas&amp;author=K.%20Ishida&amp;author=Y.%20Niwa&amp;author=T.%20Yamashino&amp;author=T.%20Mizuno&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=237-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp012" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19675111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=378044&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq033-dsq033supp4.png?Expires=1509238741&amp;Signature=M~pk3UsVIwKbXVWRzS2BZFHH5dt6oZqNUlQfZlBBi5Q02AUzxYTFNaSCN4lJBab0TWAGzRKz3ul4aSpO1-162KrBw3F4JJJFtpMOzcmZx3M5kbuH9OpLcjyj15jPstU1THCMsOVAJqA47NBEJJO9~3D6t7HSMD0kHI7AAHzzxm70HB6JvIKqB22heVx8DRGVVNn-33IWz~E6MAdqBFXOmmu7PHxpA52l-m0mzeZW6F5GEQmPdBdV9ejrdn1y5agTDtnZusuiC70ejYI0Z60XB5L1wg3GONj7H04dfOkwQohvlTnudm-KTC9phYWxmry6yB-REX1KnlALf079NA8I-Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq033-dsq033supp3.doc?Expires=1509238741&amp;Signature=SpaMFhGN58Feks6uuCqO8uoVL61Mc3k-rGhNSJVIwAgEemErMETBNapy3mf83l9F7RXUYTqxkSYUg6fb1NWegu~UAP8bRdOKgYN~Dn2eTram9LzW5Qf1p2ahACS5WEDpq3XlWjngcp18zGaSZcOUVqH5zG2Vn2EWX4Ou-USV16UYTUSxbMC2VuL2G4SdftEDfq7lrW1ZB0wMK3ZKU8WIHbMWQ7jthFCzDblssPNesRd32c5u2mJIE8yJOPwCUpKe0Aw2pvs4jSa54liFaKdlZ-HUTyUDg-zs65~kVI4lKC4Mo9C2r-kC1jVlRATfQIx2pxdYWe2~chW01Uzgm8-fwA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq033-dsq033supp2.doc?Expires=1509238741&amp;Signature=aFPhe95eHFV01Wf3556yJw1w2-mN~o3hDT-OvbdGj03Ttpfz751092JRv88mMy5~V4RZUSgF4a~Y~SFLcCpaUQ1C4AIx3XqDRvrtkJ--osIAkLvekuB1rHe7BQKiSRBjWbNHp6chB5xyls~rjMARxTwd7NXx97glOYlvDVMBOH1ade4kMOivf-q~s3tdTlj5FLXMw2bRj-2ko7JzDsei2QhLGGZYmp-pnzeGIiLumLT-UUCBBXFJwqimo033QijwiRmbXOQ9vmmfgbVksaJyKxgVqPml6FWg5CecV6ogPckD2x-7BNVD5HvOxSOuTor0aIyPPlr1LIwbZAur7TRvCA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq033/2/dsq033-dsq033supp1.doc?Expires=1509238741&amp;Signature=Eoh2m~t2lKqGXlyB6H2nwKCOHYesdBj44tZHaalD6U2a4sTfdZCwaFUu268K9daVJ09kMw8ds4YBtN0ig7Unhy-aoW7RQyF2CbgdCnYAczMQnV9eA9o2Ufwxhpw0J3~~KXtRRGgvh~cb1QYPncZS4VxXv76Jt~sCgwnf4IdN7rKG3T8VEtPaueOvpwt5OPKKatvRnWxSBpzlCpsD3f6u8jLxVZI3cl4d78H80gkuC56sqeWHYcyVWxa9ISSx46Eyvv1iC9HAGS-CLl~ZgGvktEErXbgk6cxNMwcgUaWJwgLOPvlerA0YnG0N096uGYTr7a87KxO6NA4yPAc5dHREsQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">34</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsq033"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="378044"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f1%2f39%2f378044">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000288188700004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F1%2F39%2F378044" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/378044'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Pseudo-Response Regulator (PRR) Homologues of the Moss Physcomitrella patens: Insights into the Evolution of the PRR Family in Land Plants" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f39%2f378044">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

