

<!DOCTYPE html>
<html lang="en">

<head><title>
	Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar (Populus tomentosa Carr.) | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/31/382624","@type":"ScholarlyArticle","name":"Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar ( Populus tomentosa  Carr.)","datePublished":"2012-12-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss031","keywords":["candidate gene-derived SSRs","cross-species transferability","<italic>in silico</italic> analysis of SSR variations","<italic>Populus tomentosa</italic>","single marker–trait association mapping"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/31/382624","author":[{"name":"Du, Qingzhang","affiliation":"National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology ,  Beijing Forestry University ,  Beijing 100083 ,  P. R. China  and  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants ,  Ministry of Education ,  College of Biological Sciences and Technology ,  Beijing Forestry University,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Gong, Chenrui","affiliation":"National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology ,  Beijing Forestry University ,  Beijing 100083 ,  P. R. China  and  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants ,  Ministry of Education ,  College of Biological Sciences and Technology ,  Beijing Forestry University,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Pan, Wei","affiliation":"National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology ,  Beijing Forestry University ,  Beijing 100083 ,  P. R. China  and  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants ,  Ministry of Education ,  College of Biological Sciences and Technology ,  Beijing Forestry University,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Zhang, Deqiang","affiliation":"National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology ,  Beijing Forestry University ,  Beijing 100083 ,  P. R. China  and  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants ,  Ministry of Education ,  College of Biological Sciences and Technology ,  Beijing Forestry University,  Beijing 100083 ,  P. R. China","@type":"Person"}],"description":"Gene-derived simple sequence repeats (genic SSRs), also known as functional markers, are often preferred over random genomic markers because they represent variation in gene coding and/or regulatory regions. We characterized 544 genic SSR loci derived from 138 candidate genes involved in wood formation, distributed throughout the genome of Populus tomentosa, a key ecological and cultivated wood production species. Of these SSRs, three-quarters were located in the promoter or intron regions, and dinucleotide (59.7%) and trinucleotide repeat motifs (26.5%) predominated. By screening 15 wild P. tomentosa ecotypes, we identified 188 polymorphic genic SSRs with 861 alleles, 2–7 alleles for each marker. Transferability analysis of 30 random genic SSRs, testing whether these SSRs work in 26 genotypes of five genus Populus sections (outgroup, Salix matsudana), showed that 72% of the SSRs could be amplified in Turanga and 100% could be amplified in Leuce. Based on genotyping of these 26 genotypes, a neighbour-joining analysis showed the expected six phylogenetic groupings. In silico analysis of SSR variation in 220 sequences that are homologous between P. tomentosa and Populus trichocarpa suggested that genic SSR variations between relatives were predominantly affected by repeat motif variations or flanking sequence mutations. Inheritance tests and single-marker associations demonstrated the power of genic SSRs in family-based linkage mapping and candidate gene-based association studies, as well as marker-assisted selection and comparative genomic studies of P. tomentosa and related species.","pageStart":"31","pageEnd":"44","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1572217253&Signature=fQPLqK6WZuyJpQJD0J5DjR7aJXCMMn66pwm5MdE0QKDaH4jSL2zh9aYvDu-VUYFrQRApm3QjUlWlqHdM8lZM3ZGU9vlwn~3JWOhk2u0dQPQ72q-WhIsMACJjTLqEFnkFPpmVkITlBLAUGA2YmBcqh-It0YdeHie-Zfhg7Cc9XPHMNgKMX28ElNmIjVqCTFCSExAjIEghhVGcg-Zpf4aYcn6XwQYCcaeCAVDCc3OqgaOyHHHGHTgc4gT4DWbjcAny6Fq-pGbebMev8NV6usBrs0wREVZMj3QCxNTi3ry3puU1BEa7~5tMzK4Z-cAJywUjay6Gxv~q7Tm3ykc-j290SQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar ( Populus tomentosa  Carr.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1572217253&Signature=fQPLqK6WZuyJpQJD0J5DjR7aJXCMMn66pwm5MdE0QKDaH4jSL2zh9aYvDu-VUYFrQRApm3QjUlWlqHdM8lZM3ZGU9vlwn~3JWOhk2u0dQPQ72q-WhIsMACJjTLqEFnkFPpmVkITlBLAUGA2YmBcqh-It0YdeHie-Zfhg7Cc9XPHMNgKMX28ElNmIjVqCTFCSExAjIEghhVGcg-Zpf4aYcn6XwQYCcaeCAVDCc3OqgaOyHHHGHTgc4gT4DWbjcAny6Fq-pGbebMev8NV6usBrs0wREVZMj3QCxNTi3ry3puU1BEa7~5tMzK4Z-cAJywUjay6Gxv~q7Tm3ykc-j290SQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar ( Populus tomentosa  Carr.) | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Gene-derived simple sequence repeats (genic SSRs), also known as functional markers, are often preferred over random genomic markers because they represent variation in gene coding and/or regulatory regions. We characterized 544 genic SSR loci derived from 138 candidate genes involved in wood formation, distributed throughout the genome of Populus tomentosa, a key ecological and cultivated wood production species. Of these SSRs, three-quarters were located in the promoter or intron regions, and dinucleotide (59.7%) and trinucleotide repeat motifs (26.5%) predominated. By screening 15 wild P. tomentosa ecotypes, we identified 188 polymorphic genic SSRs with 861 alleles, 2–7 alleles for each marker. Transferability analysis of 30 random genic SSRs, testing whether these SSRs work in 26 genotypes of five genus Populus sections (outgroup, Salix matsudana), showed that 72% of the SSRs could be amplified in Turanga and 100% could be amplified in Leuce. Based on genotyping of these 26 genotypes, a neighbour-joining analysis showed the expected six phylogenetic groupings. In silico analysis of SSR variation in 220 sequences that are homologous between P. tomentosa and Populus trichocarpa suggested that genic SSR variations between relatives were predominantly affected by repeat motif variations or flanking sequence mutations. Inheritance tests and single-marker associations demonstrated the power of genic SSRs in family-based linkage mapping and candidate gene-based association studies, as well as marker-assisted selection and comparative genomic studies of P. tomentosa and related species." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/31/382624" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217253&amp;Signature=ODhuvq-rAjIUD34ZkLV9BopAE1ecBeXGR9ge1~dlnzjVJhpcEwGupFcE1ldXWKB6uY0CbpOi5ZnMsp-F3HVzN3ETxqIVTeEPRv~erFDOT~7PJsr7sGrDf~b3o3vEUsllhgZhKikH9Bb0DN52Un0HPxPuu6rBkitCs6lfdcirP6wVg2TBGV4-9fpoa0swCXkUkxTNbiuY~HmLFPFdho4gio~13d5tDre8Ismsnp~izUV8jMmWsG2GN-vvhjP6mqePiadCGVwWF6Ux3jZi7c3bJ18nQeYgqFG9rCTctplndnFLK3AL1yi5BvMPYqb3Bvj~NpeASmybdDwDdVS3XEetBw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2012-12-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Du, Qingzhang" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China and Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University,Beijing 100083, P. R. China" /><meta name="citation_author" content="Gong, Chenrui" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China and Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University,Beijing 100083, P. R. China" /><meta name="citation_author" content="Pan, Wei" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China and Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University,Beijing 100083, P. R. China" /><meta name="citation_author" content="Zhang, Deqiang" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China and Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University,Beijing 100083, P. R. China" /><meta name="citation_title" content="Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar (Populus tomentosa Carr.)" /><meta name="citation_firstpage" content="31" /><meta name="citation_lastpage" content="44" /><meta name="citation_doi" content="10.1093/dnares/dss031" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray); Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; " /><meta name="citation_reference" content="citation_title=Poplar genome sequence: functional genomics in an ecologically dominant plant species; Trends Plant Sci.;  citation_year=2004;  citation_volume=9;  citation_pages=49-56; " /><meta name="citation_reference" content="citation_title=A genomic and molecular view of wood formation; Crit. Rev. Plant Sci.;  citation_year=2006;  citation_volume=25;  citation_pages=213-33; " /><meta name="citation_reference" content="citation_title=The actin multigene family in Populus: organization, expression and phylogenetic analysis; Mol. Genet. Genomics;  citation_year=2010;  citation_volume=284;  citation_pages=105-19; " /><meta name="citation_reference" content="citation_title=The status and advances of genetic improvement of Populus tomentosa Carr.; J. Beijing Forestry Univ.;  citation_year=1997;  citation_volume=6;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=Development of 15 novel microsatellite markers from cellulose synthase genes in Populus tomentosa (Salicaceae); Am. J. Bot.;  citation_year=2012;  citation_volume=99;  citation_pages=e46-8; " /><meta name="citation_reference" content="citation_title=Genetic diversity, inter-gene pool introgression and nutritional quality of common beans (Phaseolus vulgaris L.) from Central Africa; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=237-48; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in plants: features and applications; Trends Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes; Nat. Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=194-200; " /><meta name="citation_reference" content="citation_title=Microsatellite primers resource developed from the mapped sequence scaffolds of Nisqually-1 genome; New Phytol.;  citation_year=2009;  citation_volume=181;  citation_pages=498-503; " /><meta name="citation_reference" content="citation_title=Characterization of microsatellites in the coding regions of the Populus genome; Mol. Breeding;  citation_year=2011;  citation_volume=27;  citation_pages=59-66; " /><meta name="citation_reference" content="citation_title=Genetic diversity and population structure of Chinese white poplar (Populus tomentosa) revealed by SSR markers; citation_text=In press; J. Hered;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=, Genetic mapping in (Populus tomentosa × P. bolleana) and P. tomentosa Carr. using AFLP markers; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=657-62; " /><meta name="citation_reference" content="citation_title=Polymorphisms in cinnamoyl coa reductase (CCR) are associated with variation in microﬁbril angle in Eucalyptus spp.; Genetics;  citation_year=2005;  citation_volume=171;  citation_pages=1257-65; " /><meta name="citation_reference" content="citation_title=Expression and nucleotide diversity of the poplar COBL gene; Tree Genet. Genomes;  citation_year=2010;  citation_volume=6;  citation_pages=331-44; " /><meta name="citation_reference" content="citation_title=The sucrose synthase gene family in Populus: structure, expression, and evolution; Tree Genet. Genomes;  citation_year=2011;  citation_volume=7;  citation_pages=443-51; " /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, length variation, transposon associations, and genetic marker potential; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1441-52; " /><meta name="citation_reference" content="Yeh F.C. Yang R.C. Boyle T. POPGENE version 1.32: Microsoft Windows-based freeware for population genetic analysis, quick user guide 1999 Canada University of Alberta Center for International Forestry Research " /><meta name="citation_reference" content="citation_title=PowerMarker: an integrated analysis environment for genetic markers analysis; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=22128-9; " /><meta name="citation_reference" content="Du Q.Z. Pan W. Xu B.H. Li B.L. Zhang D.Q. Polymorphic SSR loci within PtoCesA genes are associated with growth and wood properties in Populus tomentosa New Phytol In press " /><meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=203-8; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-5; " /><meta name="citation_reference" content="citation_title=Statistical signiﬁcance for genomewide studies; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=9440-5; " /><meta name="citation_reference" content="citation_title=Systematics and evolution of Populus; citation_publisher=NRC Research Press, Ottawa, Canada; Biology of Populus and its implications for management and conservation;  citation_year=1996;  citation_pages=7-32; " /><meta name="citation_reference" content="citation_title=Characterization of microsatellites revealed by genomic sequencing of Populus trichocarpa; Can. J. Forest Res.;  citation_year=2004;  citation_volume=34;  citation_pages=5-93; " /><meta name="citation_reference" content="citation_title=Exploiting an oil palm EST database for the development of gene-derived SSR markers and their exploitation for assessment of genetic diversity; Biologia;  citation_year=2008;  citation_volume=63;  citation_pages=227-35; " /><meta name="citation_reference" content="citation_title=Development, characterization and cross-species/genera transferability of EST-SSR markers for rubber tree (Hevea brasiliensis); Mol. Breed.;  citation_year=2009;  citation_volume=23;  citation_pages=85-97; " /><meta name="citation_reference" content="citation_title=Cross-species transferability and mapping of genomic and cDNA SSRs in pines; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=1204-14; " /><meta name="citation_reference" content="citation_title=Characterization of the soybean genome using EST-derived microsatellite markers; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=271-81; " /><meta name="citation_reference" content="citation_title=Computational and experimental characterization of physically clustered simple sequence repeats in plants; Genetics;  citation_year=2000;  citation_volume=156;  citation_pages=847-54; " /><meta name="citation_reference" content="citation_title=Rapid turnover of long noncoding RNAs and the evolution of gene expression; PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002841" /><meta name="citation_reference" content="citation_title=Microsatellites within genes: structure, function, and evolution; Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=991-1007; " /><meta name="citation_reference" content="citation_title=Development and characterization of BAC-end sequence derived SSRs, and their incorporation into a new higher density genetic map for cultivated peanut (Arachis hypogaea L.); BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=10" /><meta name="citation_reference" content="citation_title=Microsatellites in different eukaryotic genomes: survey and analysis; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=967-81; " /><meta name="citation_reference" content="citation_title=Salient biological features, systematics, and genetic variation of Populus; citation_publisher=Springer, New York; Genetics and genomics of Populus;  citation_year=2010;  citation_pages=15-38; " /><meta name="citation_reference" content="citation_title=Genetic variation within and between populations of a desert poplar (Populus euphratica) revealed by SSR markers; Ann. Forest Sci.;  citation_year=2011;  citation_volume=68;  citation_pages=1143-9; " /><meta name="citation_reference" content="citation_title=Quantitative trait locus mapping can benefit from segregation distortion; Genetics;  citation_year=2008;  citation_volume=180;  citation_pages=2201-8; " /><meta name="citation_reference" content="citation_title=Common features of segregation distortion in plants and animals; Genetica;  citation_year=2003;  citation_volume=117;  citation_pages=27-35; " /><meta name="citation_reference" content="citation_title=Sex-ratio segregation distortion associated with reproductive isolation in Drosophila; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=13183-8; " /><meta name="citation_reference" content="citation_title=Conservation of noncoding microsatellites in plants: implication for gene regulation; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=323" /><meta name="citation_reference" content="citation_title=Differential expression of two distinct phenylalanine ammonia-lyase genes in condensed tannin-accumulating and lignifying cells of quaking aspen; Plant Physiol.;  citation_year=2002;  citation_volume=130;  citation_pages=796-807; " /><meta name="citation_reference" content="citation_title=Distinct roles of cinnamate 4-hydroxylase genes in Populus; Plant Cell Physiol.;  citation_year=2006;  citation_volume=47;  citation_pages=905-14; " /><meta name="citation_reference" content="citation_title=Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (Populus trichocarpa, Salicaceae) secondary xylem; New Phytol.;  citation_year=2010;  citation_volume=188;  citation_pages=515-32; " /><meta name="citation_reference" content="citation_title=Lignin biosynthesis; Annu. Rev. Plant Biol.;  citation_year=2003;  citation_volume=54;  citation_pages=519-46; " /><meta name="citation_reference" content="citation_title=Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=13118-23; " /><meta name="citation_reference" content="citation_title=EST data suggest that poplar is an ancient polyploidy; New Phytol.;  citation_year=2005;  citation_volume=167;  citation_pages=165-70; " /><meta name="citation_reference" content="citation_title=Molecular phylogeny of Salicaceae and closely related Flacourtiaceae: evidence from 5.8 S, ITS 1 and ITS 2 of the rDNA; Plant Syst. Evol.;  citation_year=1999;  citation_volume=215;  citation_pages=209-27; " /><meta name="citation_reference" content="citation_title=Survey and analysis of microsatellites in the silkworm, Bombyx mori: frequency, distribution, mutations, marker potential and their conservation in heterologous species; Genetics;  citation_year=2004;  citation_volume=169;  citation_pages=197-214; " /><meta name="citation_reference" content="citation_title=Microsatellites: simple sequences with complex evolution; Nature Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=435-45; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/31/1085089/dss031.pdf" /><meta name="description" content="Abstract.  Gene-derived simple sequence repeats (genic SSRs), also known as functional markers, are often preferred over random genomic markers because they rep" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/31/382624" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/31/382624" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dss031';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./382624" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFn3I8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkR1LCBRaW5nemhhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBFbmdpbmVlcmluZyBMYWJvcmF0b3J5IGZvciBUcmVlIEJyZWVkaW5nLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIGFuZCBLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljcyBhbmQgQnJlZWRpbmcgaW4gRm9yZXN0IFRyZWVzIGFuZCBPcm5hbWVudGFsIFBsYW50cywgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSxCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb25nLCBDaGVucnVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgRW5naW5lZXJpbmcgTGFib3JhdG9yeSBmb3IgVHJlZSBCcmVlZGluZywgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSBhbmQgS2V5IExhYm9yYXRvcnkgb2YgR2VuZXRpY3MgYW5kIEJyZWVkaW5nIGluIEZvcmVzdCBUcmVlcyBhbmQgT3JuYW1lbnRhbCBQbGFudHMsIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFuLCBXZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBFbmdpbmVlcmluZyBMYWJvcmF0b3J5IGZvciBUcmVlIEJyZWVkaW5nLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIGFuZCBLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljcyBhbmQgQnJlZWRpbmcgaW4gRm9yZXN0IFRyZWVzIGFuZCBPcm5hbWVudGFsIFBsYW50cywgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSxCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgRGVxaWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEVuZ2luZWVyaW5nIExhYm9yYXRvcnkgZm9yIFRyZWUgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEgYW5kIEtleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGV2ZWxvcG1lbnQgYW5kIEFwcGxpY2F0aW9uIG9mIE1pY3Jvc2F0ZWxsaXRlcyBpbiBDYW5kaWRhdGUgR2VuZXMgUmVsYXRlZCB0byBXb29kIFByb3BlcnRpZXMgaW4gdGhlIENoaW5lc2UgV2hpdGUgUG9wbGFyIChQb3B1bHVzIHRvbWVudG9zYSBDYXJyLikiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgYmxhY2sgY290dG9ud29vZCwgUG9wdWx1cyB0cmljaG9jYXJwYSAoVG9yci4gJmFtcDsgR3JheSk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzEzOyAgY2l0YXRpb25fcGFnZXM9MTU5Ni02MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcGxhciBnZW5vbWUgc2VxdWVuY2U6IGZ1bmN0aW9uYWwgZ2Vub21pY3MgaW4gYW4gZWNvbG9naWNhbGx5IGRvbWluYW50IHBsYW50IHNwZWNpZXM7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00OS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWljIGFuZCBtb2xlY3VsYXIgdmlldyBvZiB3b29kIGZvcm1hdGlvbjsgQ3JpdC4gUmV2LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjEzLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYWN0aW4gbXVsdGlnZW5lIGZhbWlseSBpbiBQb3B1bHVzOiBvcmdhbml6YXRpb24sIGV4cHJlc3Npb24gYW5kIHBoeWxvZ2VuZXRpYyBhbmFseXNpczsgTW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg0OyAgY2l0YXRpb25fcGFnZXM9MTA1LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3RhdHVzIGFuZCBhZHZhbmNlcyBvZiBnZW5ldGljIGltcHJvdmVtZW50IG9mIFBvcHVsdXMgdG9tZW50b3NhIENhcnIuOyBKLiBCZWlqaW5nIEZvcmVzdHJ5IFVuaXYuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIDE1IG5vdmVsIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZnJvbSBjZWxsdWxvc2Ugc3ludGhhc2UgZ2VuZXMgaW4gUG9wdWx1cyB0b21lbnRvc2EgKFNhbGljYWNlYWUpOyBBbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPWU0Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSwgaW50ZXItZ2VuZSBwb29sIGludHJvZ3Jlc3Npb24gYW5kIG51dHJpdGlvbmFsIHF1YWxpdHkgb2YgY29tbW9uIGJlYW5zIChQaGFzZW9sdXMgdnVsZ2FyaXMgTC4pIGZyb20gQ2VudHJhbCBBZnJpY2E7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIxOyAgY2l0YXRpb25fcGFnZXM9MjM3LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBsYW50czogZmVhdHVyZXMgYW5kIGFwcGxpY2F0aW9uczsgVHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyBhcmUgcHJlZmVyZW50aWFsbHkgYXNzb2NpYXRlZCB3aXRoIG5vbnJlcGV0aXRpdmUgRE5BIGluIHBsYW50IGdlbm9tZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTk0LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGUgcHJpbWVycyByZXNvdXJjZSBkZXZlbG9wZWQgZnJvbSB0aGUgbWFwcGVkIHNlcXVlbmNlIHNjYWZmb2xkcyBvZiBOaXNxdWFsbHktMSBnZW5vbWU7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MTsgIGNpdGF0aW9uX3BhZ2VzPTQ5OC01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGVzIGluIHRoZSBjb2RpbmcgcmVnaW9ucyBvZiB0aGUgUG9wdWx1cyBnZW5vbWU7IE1vbC4gQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01OS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgYW5kIHBvcHVsYXRpb24gc3RydWN0dXJlIG9mIENoaW5lc2Ugd2hpdGUgcG9wbGFyIChQb3B1bHVzIHRvbWVudG9zYSkgcmV2ZWFsZWQgYnkgU1NSIG1hcmtlcnM7IGNpdGF0aW9uX3RleHQ9SW4gcHJlc3M7IEouIEhlcmVkOyAgY2l0YXRpb25feWVhcj0yMDEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT0sIEdlbmV0aWMgbWFwcGluZyBpbiAoUG9wdWx1cyB0b21lbnRvc2Egw5cgUC4gYm9sbGVhbmEpIGFuZCBQLiB0b21lbnRvc2EgQ2Fyci4gdXNpbmcgQUZMUCBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTY1Ny02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seW1vcnBoaXNtcyBpbiBjaW5uYW1veWwgY29hIHJlZHVjdGFzZSAoQ0NSKSBhcmUgYXNzb2NpYXRlZCB3aXRoIHZhcmlhdGlvbiBpbiBtaWNyb++sgWJyaWwgYW5nbGUgaW4gRXVjYWx5cHR1cyBzcHAuOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE7ICBjaXRhdGlvbl9wYWdlcz0xMjU3LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIGFuZCBudWNsZW90aWRlIGRpdmVyc2l0eSBvZiB0aGUgcG9wbGFyIENPQkwgZ2VuZTsgVHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzMxLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3Vjcm9zZSBzeW50aGFzZSBnZW5lIGZhbWlseSBpbiBQb3B1bHVzOiBzdHJ1Y3R1cmUsIGV4cHJlc3Npb24sIGFuZCBldm9sdXRpb247IFRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQ0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHV0YXRpb25hbCBhbmQgZXhwZXJpbWVudGFsIGFuYWx5c2lzIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiByaWNlIChPcnl6YSBzYXRpdmEgTC4pOiBmcmVxdWVuY3ksIGxlbmd0aCB2YXJpYXRpb24sIHRyYW5zcG9zb24gYXNzb2NpYXRpb25zLCBhbmQgZ2VuZXRpYyBtYXJrZXIgcG90ZW50aWFsOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE0NDEtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlllaCBGLkMuIFlhbmcgUi5DLiBCb3lsZSBULiBQT1BHRU5FIHZlcnNpb24gMS4zMjogTWljcm9zb2Z0IFdpbmRvd3MtYmFzZWQgZnJlZXdhcmUgZm9yIHBvcHVsYXRpb24gZ2VuZXRpYyBhbmFseXNpcywgcXVpY2sgdXNlciBndWlkZSAxOTk5IENhbmFkYSBVbml2ZXJzaXR5IG9mIEFsYmVydGEgQ2VudGVyIGZvciBJbnRlcm5hdGlvbmFsIEZvcmVzdHJ5IFJlc2VhcmNoICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvd2VyTWFya2VyOiBhbiBpbnRlZ3JhdGVkIGFuYWx5c2lzIGVudmlyb25tZW50IGZvciBnZW5ldGljIG1hcmtlcnMgYW5hbHlzaXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjIxMjgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iRHUgUS5aLiBQYW4gVy4gWHUgQi5ILiBMaSBCLkwuIFpoYW5nIEQuUS4gUG9seW1vcnBoaWMgU1NSIGxvY2kgd2l0aGluIFB0b0Nlc0EgZ2VuZXMgYXJlIGFzc29jaWF0ZWQgd2l0aCBncm93dGggYW5kIHdvb2QgcHJvcGVydGllcyBpbiBQb3B1bHVzIHRvbWVudG9zYSBOZXcgUGh5dG9sIEluIHByZXNzICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdW5pZmllZCBtaXhlZC1tb2RlbCBtZXRob2QgZm9yIGFzc29jaWF0aW9uIG1hcHBpbmcgdGhhdCBhY2NvdW50cyBmb3IgbXVsdGlwbGUgbGV2ZWxzIG9mIHJlbGF0ZWRuZXNzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTIwMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UQVNTRUw6IHNvZnR3YXJlIGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nIG9mIGNvbXBsZXggdHJhaXRzIGluIGRpdmVyc2Ugc2FtcGxlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yNjMzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIHNpZ25p76yBY2FuY2UgZm9yIGdlbm9tZXdpZGUgc3R1ZGllczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9OTQ0MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljcyBhbmQgZXZvbHV0aW9uIG9mIFBvcHVsdXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1OUkMgUmVzZWFyY2ggUHJlc3MsIE90dGF3YSwgQ2FuYWRhOyBCaW9sb2d5IG9mIFBvcHVsdXMgYW5kIGl0cyBpbXBsaWNhdGlvbnMgZm9yIG1hbmFnZW1lbnQgYW5kIGNvbnNlcnZhdGlvbjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3BhZ2VzPTctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgbWljcm9zYXRlbGxpdGVzIHJldmVhbGVkIGJ5IGdlbm9taWMgc2VxdWVuY2luZyBvZiBQb3B1bHVzIHRyaWNob2NhcnBhOyBDYW4uIEouIEZvcmVzdCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9pdGluZyBhbiBvaWwgcGFsbSBFU1QgZGF0YWJhc2UgZm9yIHRoZSBkZXZlbG9wbWVudCBvZiBnZW5lLWRlcml2ZWQgU1NSIG1hcmtlcnMgYW5kIHRoZWlyIGV4cGxvaXRhdGlvbiBmb3IgYXNzZXNzbWVudCBvZiBnZW5ldGljIGRpdmVyc2l0eTsgQmlvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50LCBjaGFyYWN0ZXJpemF0aW9uIGFuZCBjcm9zcy1zcGVjaWVzL2dlbmVyYSB0cmFuc2ZlcmFiaWxpdHkgb2YgRVNULVNTUiBtYXJrZXJzIGZvciBydWJiZXIgdHJlZSAoSGV2ZWEgYnJhc2lsaWVuc2lzKTsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz04NS05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3Jvc3Mtc3BlY2llcyB0cmFuc2ZlcmFiaWxpdHkgYW5kIG1hcHBpbmcgb2YgZ2Vub21pYyBhbmQgY0ROQSBTU1JzIGluIHBpbmVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTEyMDQtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIHNveWJlYW4gZ2Vub21lIHVzaW5nIEVTVC1kZXJpdmVkIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjcxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGFuZCBleHBlcmltZW50YWwgY2hhcmFjdGVyaXphdGlvbiBvZiBwaHlzaWNhbGx5IGNsdXN0ZXJlZCBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiBwbGFudHM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTg0Ny01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgdHVybm92ZXIgb2YgbG9uZyBub25jb2RpbmcgUk5BcyBhbmQgdGhlIGV2b2x1dGlvbiBvZiBnZW5lIGV4cHJlc3Npb247IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjg0MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyB3aXRoaW4gZ2VuZXM6IHN0cnVjdHVyZSwgZnVuY3Rpb24sIGFuZCBldm9sdXRpb247IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz05OTEtMTAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgQkFDLWVuZCBzZXF1ZW5jZSBkZXJpdmVkIFNTUnMsIGFuZCB0aGVpciBpbmNvcnBvcmF0aW9uIGludG8gYSBuZXcgaGlnaGVyIGRlbnNpdHkgZ2VuZXRpYyBtYXAgZm9yIGN1bHRpdmF0ZWQgcGVhbnV0IChBcmFjaGlzIGh5cG9nYWVhIEwuKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXMgaW4gZGlmZmVyZW50IGV1a2FyeW90aWMgZ2Vub21lczogc3VydmV5IGFuZCBhbmFseXNpczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz05NjctODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbGllbnQgYmlvbG9naWNhbCBmZWF0dXJlcywgc3lzdGVtYXRpY3MsIGFuZCBnZW5ldGljIHZhcmlhdGlvbiBvZiBQb3B1bHVzOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrOyBHZW5ldGljcyBhbmQgZ2Vub21pY3Mgb2YgUG9wdWx1czsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3BhZ2VzPTE1LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhdGlvbiB3aXRoaW4gYW5kIGJldHdlZW4gcG9wdWxhdGlvbnMgb2YgYSBkZXNlcnQgcG9wbGFyIChQb3B1bHVzIGV1cGhyYXRpY2EpIHJldmVhbGVkIGJ5IFNTUiBtYXJrZXJzOyBBbm4uIEZvcmVzdCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MTE0My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgbWFwcGluZyBjYW4gYmVuZWZpdCBmcm9tIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTIyMDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIGZlYXR1cmVzIG9mIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gaW4gcGxhbnRzIGFuZCBhbmltYWxzOyBHZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0yNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2V4LXJhdGlvIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gYXNzb2NpYXRlZCB3aXRoIHJlcHJvZHVjdGl2ZSBpc29sYXRpb24gaW4gRHJvc29waGlsYTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0xMzE4My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gb2Ygbm9uY29kaW5nIG1pY3Jvc2F0ZWxsaXRlcyBpbiBwbGFudHM6IGltcGxpY2F0aW9uIGZvciBnZW5lIHJlZ3VsYXRpb247IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gb2YgdHdvIGRpc3RpbmN0IHBoZW55bGFsYW5pbmUgYW1tb25pYS1seWFzZSBnZW5lcyBpbiBjb25kZW5zZWQgdGFubmluLWFjY3VtdWxhdGluZyBhbmQgbGlnbmlmeWluZyBjZWxscyBvZiBxdWFraW5nIGFzcGVuOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzA7ICBjaXRhdGlvbl9wYWdlcz03OTYtODA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0aW5jdCByb2xlcyBvZiBjaW5uYW1hdGUgNC1oeWRyb3h5bGFzZSBnZW5lcyBpbiBQb3B1bHVzOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9OTA1LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBnZW5ldGljcyBvZiB0cmFpdHMgY29udHJvbGxpbmcgbGlnbmluIGFuZCBjZWxsdWxvc2UgYmlvc3ludGhlc2lzIGluIGJsYWNrIGNvdHRvbndvb2QgKFBvcHVsdXMgdHJpY2hvY2FycGEsIFNhbGljYWNlYWUpIHNlY29uZGFyeSB4eWxlbTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9NTE1LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaWduaW4gYmlvc3ludGhlc2lzOyBBbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NTE5LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWNyb3NlIHN5bnRoYXNlIGFmZmVjdHMgY2FyYm9uIHBhcnRpdGlvbmluZyB0byBpbmNyZWFzZSBjZWxsdWxvc2UgcHJvZHVjdGlvbiBhbmQgYWx0ZXJlZCBjZWxsIHdhbGwgdWx0cmFzdHJ1Y3R1cmU7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTEzMTE4LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1QgZGF0YSBzdWdnZXN0IHRoYXQgcG9wbGFyIGlzIGFuIGFuY2llbnQgcG9seXBsb2lkeTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9MTY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgcGh5bG9nZW55IG9mIFNhbGljYWNlYWUgYW5kIGNsb3NlbHkgcmVsYXRlZCBGbGFjb3VydGlhY2VhZTogZXZpZGVuY2UgZnJvbSA1LjggUywgSVRTIDEgYW5kIElUUyAyIG9mIHRoZSByRE5BOyBQbGFudCBTeXN0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz0yMDktMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cnZleSBhbmQgYW5hbHlzaXMgb2YgbWljcm9zYXRlbGxpdGVzIGluIHRoZSBzaWxrd29ybSwgQm9tYnl4IG1vcmk6IGZyZXF1ZW5jeSwgZGlzdHJpYnV0aW9uLCBtdXRhdGlvbnMsIG1hcmtlciBwb3RlbnRpYWwgYW5kIHRoZWlyIGNvbnNlcnZhdGlvbiBpbiBoZXRlcm9sb2dvdXMgc3BlY2llczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY5OyAgY2l0YXRpb25fcGFnZXM9MTk3LTIxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzOiBzaW1wbGUgc2VxdWVuY2VzIHdpdGggY29tcGxleCBldm9sdXRpb247IE5hdHVyZSBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvMS8zMS8xMDg1MDg5L2RzczAzMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2VuZS1kZXJpdmVkIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXRzIChnZW5pYyBTU1JzKSwgYWxzbyBrbm93biBhcyBmdW5jdGlvbmFsIG1hcmtlcnMsIGFyZSBvZnRlbiBwcmVmZXJyZWQgb3ZlciByYW5kb20gZ2Vub21pYyBtYXJrZXJzIGJlY2F1c2UgdGhleSByZXAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM4MjYyNCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGQQx1v9iqJFVAw5TCZ+HqL0nA1sJA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f31%2f382624" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f31%2f382624" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1509238849&amp;Signature=PSAuPLfsuhNKNMQR1FaKGKeXnYxa9VWWxVwVj8QgWwiWyxXnuqrnLzZcJc2CBUJltLVrinKxSB5nYhbXjUM9-je4LPvkdGahFY~o-WdQJK7WT08AL~8WvaZoAYAn96Q79t1-TpryREPtJCpPrqK7WAJ4VgdOJhwTB6p3hzdwKtg4L9c5bC-56h6Cf-rVzA9WfP1pApnAfqLDfc4Hxr8AkFr9iuyQDmP0~HPWyaQt67XCS7L1KewivcvyYV4gCPFA8PQEj~5vMJtfUjMZVsPsxoOYWtrZUgaCpQPDDd88T8mjJZgtwXU6D7a5ZHr1hjQj4sdG24HsvRLOqQWKflk3Jg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5605072">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605072">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605074">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605074">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605079">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605079">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605099">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605099">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605177">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605177">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605199">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605199">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605201">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605201">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5605203">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5605203">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/1/17/382150">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/1/45/382928">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar (<em>Populus tomentosa</em> Carr.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Qingzhang Du</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qingzhang Du</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div> and <div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants</div>, <div class="insititution">Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University,</div><div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Qingzhang+Du">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Du Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Du author:Q.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Chenrui Gong</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chenrui Gong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div> and <div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants</div>, <div class="insititution">Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University,</div><div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Chenrui+Gong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gong C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Gong author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Wei Pan</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Pan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div> and <div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants</div>, <div class="insititution">Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University,</div><div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Wei+Pan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pan W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Pan author:W.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Deqiang Zhang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deqiang Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div> and <div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants</div>, <div class="insititution">Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology</div>, <div class="insititution">Beijing Forestry University,</div><div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86-10-62336007. Fax. +86-10-62336164. Email: <a href="mailto:deqiangzhang@bjfu.edu.cn" target="_blank">deqiangzhang@bjfu.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Deqiang+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Zhang author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 31–44, <a href='https://doi.org/10.1093/dnares/dss031'>https://doi.org/10.1093/dnares/dss031</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 December 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 August 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="382624" href="/dnaresearch/article-pdf/20/1/31/1085089/dss031.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Qingzhang Du, Chenrui Gong, Wei Pan, Deqiang Zhang;  Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar (<em>Populus tomentosa</em> Carr.), <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 31–44, <a href='https://doi.org/10.1093/dnares/dss031'>https://doi.org/10.1093/dnares/dss031</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/382624?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/382624?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/382624?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/382624?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/382624?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/382624?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/382624?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20and%20Application%20of%20Microsatellites%20in%20Candidate%20Genes%20Related%20to%20Wood%20Properties%20in%20the%20Chinese%20White%20Poplar%20(%26lt%3Bem%26gt%3BPopulus%20tomentosa%26lt%3B%2Fem%26gt%3B%20Carr.)&amp;publicationDate=2012-12-04&amp;volumeNum=20&amp;issueNum=1&amp;author=Du%2C%20Qingzhang%3B%20Gong%2C%20Chenrui&amp;startPage=31&amp;endPage=44&amp;contentId=10.1093%2Fdnares%2Fdss031&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f31%2f382624" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5605072 id="5605072" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Gene-derived simple sequence repeats (genic SSRs), also known as functional markers, are often preferred over random genomic markers because they represent variation in gene coding and/or regulatory regions. We characterized 544 genic SSR loci derived from 138 candidate genes involved in wood formation, distributed throughout the genome of <em>Populus tomentosa</em>, a key ecological and cultivated wood production species. Of these SSRs, three-quarters were located in the promoter or intron regions, and dinucleotide (59.7%) and trinucleotide repeat motifs (26.5%) predominated. By screening 15 wild <em>P. tomentosa</em> ecotypes, we identified 188 polymorphic genic SSRs with 861 alleles, 2–7 alleles for each marker. Transferability analysis of 30 random genic SSRs, testing whether these SSRs work in 26 genotypes of five genus <em>Populus</em> sections (outgroup, <em>Salix matsudana</em>), showed that 72% of the SSRs could be amplified in <em>Turanga</em> and 100% could be amplified in <em>Leuce</em>. Based on genotyping of these 26 genotypes, a neighbour-joining analysis showed the expected six phylogenetic groupings. <em>In silico</em> analysis of SSR variation in 220 sequences that are homologous between <em>P. tomentosa</em> and <em>Populus trichocarpa</em> suggested that genic SSR variations between relatives were predominantly affected by repeat motif variations or flanking sequence mutations. Inheritance tests and single-marker associations demonstrated the power of genic SSRs in family-based linkage mapping and candidate gene-based association studies, as well as marker-assisted selection and comparative genomic studies of <em>P. tomentosa</em> and related species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;candidate gene-derived SSRs&quot;">candidate gene-derived SSRs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cross-species transferability&quot;">cross-species transferability</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;in silico analysis of SSR variations&quot;"><em>in silico</em> analysis of SSR variations</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Populus tomentosa&quot;"><em>Populus tomentosa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single marker–trait association mapping&quot;">single marker–trait association mapping</a></div>                            <h2 scrollto-destination=5605074 id="5605074" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Poplars (<em>Populus</em> spp.) are widely distributed all over the world and are an important commercial tree species for timber production. In addition to their important economic value, in environmental protection, poplars also play key pioneer roles in the stability and sustainability of forest ecosystems.<sup><a reveal-id="DSS031C1" data-open="DSS031C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS031C2" data-open="DSS031C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Many members of the genus <em>Populus</em> have been physiologically and genetically characterized based on their desirable biological characteristics, such as rapid growth, easy transformation, modest genome size, and ability to make interspecific crosses, propagate vegetatively.<sup><a reveal-id="DSS031C1" data-open="DSS031C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS031C2" data-open="DSS031C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Thus, poplars have become a model species for studies of angiosperm trees, particularly because the whole genome of <em>Populus trichocarpa</em> has been sequenced and annotated (<a class="link link-uri" href="http://genome.jgi-psf.org/Poptr1/Poptr1.home.html">http://genome.jgi-psf.org/Poptr1/Poptr1.home.html</a>).<sup><a reveal-id="DSS031C1" data-open="DSS031C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Additional genomics resources include databases of expressed sequence tags (ESTs) (<a class="link link-uri" href="http://www.populus.db.umu.se/index.html">http://www.populus.db.umu.se/index.html</a>) and simple sequence repeats (SSRs) (<a class="link link-uri" href="http://www.ornl.gov/sci/ipgc/ssr_resource.htm">http://www.ornl.gov/sci/ipgc/ssr_resource.htm</a>), and these resources not only provide data for comparison of a long-lived perennial to short-lived model plants (e.g. <em>Arabidopsis</em> and rice), but also offer new opportunities to explore the genetic basis of wood formation, perenniality, and dormancy.<sup><a reveal-id="DSS031C2" data-open="DSS031C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS031C3" data-open="DSS031C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>The Chinese white poplar (<em>Populus tomentosa</em>) belongs to the section <em>Leuce</em> within the <em>Populus</em> genus and is native to northern China with a distribution zone of 1 million km<sup>2</sup>. <em>P. tomentosa</em> is of major commercial importance in timber and pulp production and also plays an important role in ecological and environmental protection.<sup><a reveal-id="DSS031C4" data-open="DSS031C4" class="link link-ref link-reveal xref-bibr">4</a></sup> A vast amount of genetic variation has arisen during the evolution of the species, as is evident in the natural populations.<sup><a reveal-id="DSS031C5" data-open="DSS031C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a></sup> This accumulated genetic variation provides an important resource for the exploration of the molecular mechanisms of wood formation and is also a source of alleles for the potential improvement of wood products. However, conventional breeding programmes may not be sufficient to improve this long-growing species.<sup><a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Modern molecular breeding tools, such as molecular marker-assisted selection (MAS) breeding, could enhance important <em>Populus</em> agronomic traits, including growth rate, wood quality, and disease resistance. Hence, development of suitable genetic marker resources is an important foundation for MAS breeding.</p><p>Among molecular genetic markers, single nucleotide polymorphisms are often used for genetic analysis. However, DNA microsatellites, or SSR markers, are excellent genetic markers because they are hypervariable, co-dominant, and therefore highly informative.<sup><a reveal-id="DSS031C7" data-open="DSS031C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Moreover, compared with SSR markers derived from random genomic locations, SSR markers derived from genes will likely provide a much greater degree of resolution in association mapping because they occur within the gene and thus may affect gene expression or function.<sup><a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In addition, genic SSRs exhibit relatively high transferability to closely related species and can be used as anchor markers for comparative mapping and evolutionary studies.<sup><a reveal-id="DSS031C6  DSS031C9" data-open="DSS031C6  DSS031C9" class="link link-ref link-reveal xref-bibr">6–9</a></sup> Gene-based microsatellites have now been developed for a limited number of <em>Populus</em> species based on the <em>P. trichocarpa</em> genome sequence.<sup><a reveal-id="DSS031C10" data-open="DSS031C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS031C11" data-open="DSS031C11" class="link link-ref link-reveal xref-bibr">11</a></sup> However, very limited genomic information is available for <em>P. tomentosa</em>,<sup><a reveal-id="DSS031C4" data-open="DSS031C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS031C12" data-open="DSS031C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and as the <em>P. tomentosa</em> linkage map was constructed using amplified fragment length polymorphisms (AFLPs),<sup><a reveal-id="DSS031C13" data-open="DSS031C13" class="link link-ref link-reveal xref-bibr">13</a></sup> functional genomics studies of this economically important species are in their infancy. Furthermore, another important approach, the use of SSRs from fully characterized genes or full-length cDNA clones has not yet been utilized in <em>Populus</em>.</p><p>Wood quality traits are considered to be quantitative, controlled by multiple genes, with moderate-to-high heritability.<sup><a reveal-id="DSS031C14" data-open="DSS031C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Currently, large numbers of candidate genes involved in wood formation have been isolated from <em>P. tomentosa</em> using direct sequencing methods, although many have not been published.<sup><a reveal-id="DSS031C4" data-open="DSS031C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS031C15" data-open="DSS031C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS031C16" data-open="DSS031C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Therefore, to improve the properties of wood using a MAS approach, identification and characterization of species-specific SSR loci from wood formation-related genes is a highly promising approach. Here, we make use of the large dataset of available gene sequences to identify a large number of gene-based SSR markers for <em>P. tomentosa</em>. The specific aims of our study were to: (i) characterize the genic SSR loci in <em>P. tomentosa</em> and evaluate SSR primers and polymorphisms in different wild-type varieties, (ii) test cross-species transferability within the genus <em>Populus</em> and conduct <em>in silico</em> analysis of SSR variation between <em>P. tomentosa</em> and <em>P. trichocarpa</em>, and (iii) examine inheritance segregation in a mapping population and analyze single SSR marker–trait association in a natural population. This study provides a valuable SSR resource for comparative genomic studies of the genus <em>Populus</em>, and the genic microsatellites also serve as ‘framework’ markers for construction of a physical map for alignment of the ongoing sequencing of the complete <em>P. tomentosa</em> genome.</p>                            <h2 scrollto-destination=5605079 id="5605079" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5605080 id="5605080" class="section-title" data-legacy-id=s2a>2.1. Microsatellite identification, primer design, and SSR polymorphism screening</h3>
<p>Total genomic DNA was extracted from young leaves using the DNeasy Plant Mini kit (Qiagen China, Shanghai), following the manufacturer's protocol. The reference gene models of 150 candidate genes involved in wood formation were obtained by BLASTX analyses against the NCBI database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) or from the JGI database (<a class="link link-uri" href="http://genome.jgi-psf.org/Poptr1_1/Poptr1_1.home.html">http://genome.jgi-psf.org/Poptr1_1/Poptr1_1.home.html</a>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Subsequently, a set of specific primers was designed for polymerase chain reaction (PCR) amplification of all 150 genes, and total genomic DNA (20 ng per reaction) from the <em>P. tomentosa</em> LM50 clone was used for amplification. All the PCR amplification products from LM50 were sequenced (both strands) using conserved primers, the BigDye Terminator Cycle Sequencing kit, version 3.1 (Applied Biosystems, Beijing, China), and a Li-Cor 4300 genetic analyzer (Li-Cor Biosciences, Lincoln, Nebraska, USA). In all, a total of 696 792 bp of genomic DNA sequences from these 150 unique candidate genes, with an average of 4645 bp per gene, were obtained by direct sequencing, and the sequence data of the 150 candidate genes generated in this study were deposited in the GenBank Data Library under the accession numbers JX986583–JX986732 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These sequences were mined for genic SSR markers using the SSRIT software (<a class="link link-uri" href="http://www.gramene.org/db/searches/ssrtool)">http://www.gramene.org/db/searches/ssrtool)</a>.<sup><a reveal-id="DSS031C17" data-open="DSS031C17" class="link link-ref link-reveal xref-bibr">17</a></sup> All microsatellite loci were located exclusively in the coding and/or regulatory regions of candidate genes, i.e. exons, promoters, 5′ untranslated regions (UTRs), 3′ UTRs, or introns. The ideal marker contained a minimum SSR repeat of five for dinucleotide repeats, four for trinucleotide repeats, and three for tetranucleotide or longer (more than pentanucleotide) repeats. A set of compound perfect repeat units was also identified. Mononucleotide repeats and complex sequence repeats were excluded.</p><p>Primer pairs specific for the SSR flanking sequences were designed using the Primer 3 program (<a class="link link-uri" href="http://frodo.wi.mit.edu/primer3/primer3.FAQ.html">http://frodo.wi.mit.edu/primer3/primer3.FAQ.html</a>), according to the parameters reported by Du et al.<sup><a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a></sup> All SSR primers were initially screened using genomic DNA from the <em>P. tomentosa</em> LM50 clone (three replications). Amplification was carried out using standard PCR conditions with annealing temperatures (Ta) set according to the primer sequences. PCR products were separated by electrophoresis in 2% agarose gels. Electrophoresis products were visualized and photographed using the Fluorchem™ 5500 (Alfa Innotech Corp., USA) gel documentation system. Finally, a subset of optimal SSR primers was identified and designated as ‘validated genic SSR markers’.</p><p>All validated genic SSR markers were scored for amplicon size polymorphisms among 15 wild <em>P. tomentosa</em> ecotypes that represented nearly the entire <em>P. tomentosa</em> geographic distribution (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).<sup><a reveal-id="DSS031C4" data-open="DSS031C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS031C12" data-open="DSS031C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Observed product sizes and numbers of alleles per locus (<em>N</em><sub>A</sub>) were calculated for each marker using POPGENE, version 1.32.<sup><a reveal-id="DSS031C18" data-open="DSS031C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=5605084 id="5605084" class="section-title" data-legacy-id=s2b>2.2. PCR amplification and SSR genotyping</h3>
<p>The SSR amplification reaction system and PCR amplifications were conducted following the procedure of Du et al.<sup><a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a></sup> PCR product amplification was confirmed on a 2% agarose gel, and products were then separated by capillary electrophoresis using an ABI3730xl DNA Analyzer (Applied Biosystems). The polymorphic loci were analyzed using GeneMapper, version 4.0, with the LIZ 600 size standard (Applied Biosystems).</p>                            <h3 scrollto-destination=5605086 id="5605086" class="section-title" data-legacy-id=s2c>2.3. Cross-species transferability</h3>
<p>To assess cross-species transferability and allele length polymorphisms, 30 randomly selected genic SSRs were genotyped in 26 ecotypes (24 species) belonging to 5 sections of the genus <em>Populus</em>, using <em>Salix matsudana</em> as the outgroup (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>The summary statistical parameters reflected intra- and interpopulation genetic diversity levels, including the observed allele sizes, polymorphism information content (PIC), expected heterozygosity (<em>H</em><sub>E</sub>), and number of alleles (<em>N</em><sub>A</sub>), that were calculated for each marker using POPGENE, version 1.32. The discriminatory abilities of genic SSRs were estimated using cluster analyses to assess phylogenic relationships among related <em>Populus</em> species. A neighbour-joining (NJ) dendrogram was constructed using the proportion of shared alleles coefficient from PowerMarker, version 3.25<sup><a reveal-id="DSS031C19" data-open="DSS031C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and was drawn using TreeView version 1.66 (<a class="link link-uri" href="http://taxonomy.zoology.gla.ac.uk/rod/treeview.html">http://taxonomy.zoology.gla.ac.uk/rod/treeview.html</a>).</p>                            <h3 scrollto-destination=5605089 id="5605089" class="section-title" data-legacy-id=s2d>2.4. In silico analysis of SSR variations</h3>
<p>We used <em>in silico</em> identification of genic SSR variations between <em>P. tomentosa</em> and <em>P. trichocarpa</em> to validate the true SSR cross-species transferability within the genus <em>Populus</em>. All 220 available homologous sequences of <em>P. trichocarpa</em> were identified in the JGI Database, based on 220 specific amplicons with SSR markers selected from <em>P. tomentosa</em> candidate gene sequences. MEGA, version 5.1 (<a class="link link-uri" href="http://www.megasoftware.net/">http://www.megasoftware.net/</a>), was used to compare the amplified SSR alleles with the SSR-containing sequences between the two species (i.e. repeat length, repeat motif, and mutations in flanking sequences).</p>                            <h3 scrollto-destination=5605091 id="5605091" class="section-title" data-legacy-id=s2e>2.5. Testing inheritance in a linkage-mapping population</h3>
<p>To test the power of these novel genic SSR markers for constructing a family-based linkage map, 30 random genic SSRs selected from the 188 polymorphic markers (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>) were used to examine the observed segregation of markers, using 1000 F<sub>1</sub> progeny from a controlled hybridization between a female YX01 clone (<em>Populus alba</em> × <em>Populus glandulosa</em>) and a male LM 50 clone (<em>P. tomentosa</em>).<sup><a reveal-id="DSS031C12" data-open="DSS031C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Mendelian inheritance of microsatellite variants (alleles) was determined from the observed distribution of progeny genotypes when compared with the expected segregation ratios (1:1, 1:2:1, and 1:1:1:1), based on the hypothesized genotypes of the parents by performing a chi-squared (<em>χ</em><sup>2</sup>) test at the 0.01 probability level in SAS, version 8.2, (SAS Institute, Cary, NC, USA). </p>                        <a id="5605093" scrollto-destination="5605093"></a>
<div id="DSS031F1" data-id="DSS031F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03101.jpeg?Expires=1509238854&Signature=ByvhjYKseBrGoYPtxvjnnwNiJzuqp~wLKoxnSPlsWaj6smfNchtIvViPNa6pZTcgC7TpDTXYWgHhOy1hPPkrC6LOQSaE22d3xfalOj6WLpdn~7wIMVxpIqhqMLbt1ZQiAS8ehofosRmMze0pE8-ShjhhGMhCqabfAf7NJp9EfTx8awJk2C9tlHKQRhOjGSBR9Md-WeTSkdjP7PWZhIoFv1mXlIwJbGRt~gWPTMCWNFqqnfHTKiBrPF4u4F1bGkrbGn4XLs8hs~FDS7aJZj~0LpFmf4ZHjI0GxRIe06u~fPBq5djlOIJWEQU4xvtj1ppHYo3vXfXWy3qNOnBIcWz~Ng__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Flow diagram of P. tomentosa genic SSR marker development and applications in this study." path-from-xml="dss03101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605093/dss03101.jpeg" path-from-xml="dss03101.jpeg" target="_blank">View large</a><a section="5605093" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03101.jpeg?Expires=1509238854&Signature=OXLhHpYcMwTs4HqaKFMvEjK2ROIUyv2NwmejCZ~cw~3SxoCILkTJo6mOIHt27WTdP7iC4kQbhchl5zrj9IiJXThLvdSFDfvExHSu4xR-5c8il9yhWuIsQadOsyXviMV-gaNzmXNOJJ8FMm6BqvrUo4PcmdVAjkjsf0wBjdqK5hMmKoDICsmpI2l-j6R~2zVCfFkyp3Vb9RzAi2SV24ZYhYlu9Il92xTDaFf1-y3bDtFhKvcqOSx-w4TnBIlbmMUp7nmpZ6lhfHgta7TGJqsl8jg9TobO88ZlqtQ4cFICOAHv9ZnSj9wJwFnP38SoyBvAF-DiQ~EtRpIGCE70TeBCOg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605093&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow diagram of <em>P. tomentosa</em> genic SSR marker development and applications in this study.</p></div></div><div content-id="DSS031F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03101.jpeg?Expires=1509238854&Signature=ByvhjYKseBrGoYPtxvjnnwNiJzuqp~wLKoxnSPlsWaj6smfNchtIvViPNa6pZTcgC7TpDTXYWgHhOy1hPPkrC6LOQSaE22d3xfalOj6WLpdn~7wIMVxpIqhqMLbt1ZQiAS8ehofosRmMze0pE8-ShjhhGMhCqabfAf7NJp9EfTx8awJk2C9tlHKQRhOjGSBR9Md-WeTSkdjP7PWZhIoFv1mXlIwJbGRt~gWPTMCWNFqqnfHTKiBrPF4u4F1bGkrbGn4XLs8hs~FDS7aJZj~0LpFmf4ZHjI0GxRIe06u~fPBq5djlOIJWEQU4xvtj1ppHYo3vXfXWy3qNOnBIcWz~Ng__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Flow diagram of P. tomentosa genic SSR marker development and applications in this study." path-from-xml="dss03101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605093/dss03101.jpeg" path-from-xml="dss03101.jpeg" target="_blank">View large</a><a section="5605093" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03101.jpeg?Expires=1509238854&Signature=OXLhHpYcMwTs4HqaKFMvEjK2ROIUyv2NwmejCZ~cw~3SxoCILkTJo6mOIHt27WTdP7iC4kQbhchl5zrj9IiJXThLvdSFDfvExHSu4xR-5c8il9yhWuIsQadOsyXviMV-gaNzmXNOJJ8FMm6BqvrUo4PcmdVAjkjsf0wBjdqK5hMmKoDICsmpI2l-j6R~2zVCfFkyp3Vb9RzAi2SV24ZYhYlu9Il92xTDaFf1-y3bDtFhKvcqOSx-w4TnBIlbmMUp7nmpZ6lhfHgta7TGJqsl8jg9TobO88ZlqtQ4cFICOAHv9ZnSj9wJwFnP38SoyBvAF-DiQ~EtRpIGCE70TeBCOg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605093&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow diagram of <em>P. tomentosa</em> genic SSR marker development and applications in this study.</p></div></div>                            <h3 scrollto-destination=5605094 id="5605094" class="section-title" data-legacy-id=s2f>2.6. SSR marker–trait association studies in a natural population</h3>
<p>For association mapping, we used 40 genic SSRs randomly selected from the set of 188 polymorphic markers (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>), to associate with several wood quality traits in a natural population of 460 unrelated wild-type ecotypes from the <em>P. tomentosa</em> clonal arboretum, established in the national nursery of Guan Xian County, Shandong Province, China (36°23′′N, 115°47′′E), which represent all original provenances of this species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).<sup><a reveal-id="DSS031C4" data-open="DSS031C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS031C12" data-open="DSS031C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>In the natural population, wood property traits, including microfibre angle (MFA), fibre lengths, fibre widths, and holocellulose, α-cellulose, and lignin contents were measured according to the methods described by Du et al.<sup><a reveal-id="DSS031C20" data-open="DSS031C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Analysis of variance of these six phenotypic traits is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. These 40 random genic SSRs were applied to test the degree of resolution of genic SSRs in association with these 6 wood property traits, using the unified mixed model method (MLM) with 10<sup>4</sup> permutations in TASSEL, version 2.0.1.<sup><a reveal-id="DSS031C21" data-open="DSS031C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS031C22" data-open="DSS031C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In this Q + K model, the relative kinship matrix (K) was obtained using TASSEL, and the population structure matrix (Q) of the covariates was identified by Du et al.<sup><a reveal-id="DSS031C12" data-open="DSS031C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Corrections for multiple testing were performed using the positive false discovery rate (FDR) method with 10<sup>4</sup> permutations.<sup><a reveal-id="DSS031C23" data-open="DSS031C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h2 scrollto-destination=5605099 id="5605099" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5605101 id="5605101" class="section-title" data-legacy-id=s3a>3.1. Frequency and distribution of genic SSR markers</h3>
<p>By sequencing 150 candidate genes related to wood properties, we identified 544 SSR loci located in 138 (92%) unique genes from 696 792 bp total sequence, for an average of 1 SSR per 1.3 kb. The perfect SSRs were not evenly distributed, ranging from zero to seven per gene (average, 3.6), and seven loci were compound SSRs containing at least one repeat motif (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p><p>Analysis of all 544 gene-derived SSR loci revealed that dinucleotide (325, 59.7%) and trinucleotide repeat motifs (144, 26.5%) predominated, followed by tetranucleotide (33, 6.1%), hexanucleotide (13, 2.4%), and pentanucleotide (12, 2.2%) repeat motifs (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>A). Of the identified SSRs, slightly fewer than half (46.1%) were located in promoter regions, including dinucleotides (57.4%) and trinucleotides (28.7%) (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5A</a>). Conversely, in exons, 37.5% were dinucleotides and 62.5% were trinucleotides (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5A</a>). The dinucleotide repeat AT/TA was the most abundant motif detected in all genic SSRs (177, 32.6%), followed by ATT/TAA (53, 9.8%), AG/TC (49, 9.0%), CT/GA (48, 8.7%), and AAT/TTA (16, 3.0%) (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5B</a>). SSR length was most commonly 10–20 bp (70.5% of total SSRs), followed by 21–30 bp (20.9%) (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>C). The largest SSR found was a 68-bp dinucleotide repeat (AT/TA). </p>                        <a id="5605104" scrollto-destination="5605104"></a>
<div id="DSS031F2" data-id="DSS031F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03102.jpeg?Expires=1509238854&Signature=BCTdAibjqpENASnHV0vZvrLbLI0A02C4dmfmXrgOOTbQgkkL0LXZVfsjK~3o~b70zQLwuQmDSPAd1WlsdTM1ElhWsrKbGQ9wZqljAIPnbr2NOVnzGBc2ZfA88d5LlZ1gnzEcRJLr282FeKUPRYCkXF1q5eOoDc7IHcxqmoN4NmBEaEC1jCsmzd-Sg-hhcyVlfesxoEqMjHOTPHkZ030bDUYmybGZqASnJPU4uRoJU-bEBYry2aW3Lctm6VY7MNJVPGCwSxgSE42vPSZGF1n1aWRr-Q5yWEJYHOXmfm5L~XtQoIyU7OGmNl2iMjhq3r0uRqupxsfyVuQazm38QZC5qQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(A) Distribution of repeat motifs in 544 genic SSRs identified in 138 P. tomentosa genes; (B) Distribution of dinucleotide repeat motifs detected in 138 P. tomentosa genes; (C) Distribution of repeat numbers in di- and trinucleotide SSRs. Bars of different colours show the number of genic SSRs from dinucleotide and trinucleotide." path-from-xml="dss03102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605104/dss03102.jpeg" path-from-xml="dss03102.jpeg" target="_blank">View large</a><a section="5605104" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03102.jpeg?Expires=1509238854&Signature=H9nHcBXbA7ki7Upcx6J3~HpaYvovbNNy0AQO8mhpbyfrO8IaLzUFE1REhUH94Zl~iRwej-m5ilXQJK0f96tVyma2MAq~r9UKtx0lCGoWIrlJzf4UG74J94AjXDf-6kr46CDjqP9jHMjlsb5euzzBt61hn22U5MGm-A5RJVuUXFsaQKEVLbqIVx0ye53piHvbpqRC-aREXBj2gYRyqU6MbNO5gSYKs3IJTY4fS7sxX39h1ecpqLw5~6WEVWsazrx2R0pnPBkjnrkQAcJuxWM3eX~085LsjFwSMt3IGKWLfs2jjaD~~pL9TMKhVvBxxUNFIzAgMgRBCtGnqx2tAkQHdg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605104&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Distribution of repeat motifs in 544 genic SSRs identified in 138 <em>P. tomentosa</em> genes; (B) Distribution of dinucleotide repeat motifs detected in 138 <em>P. tomentosa</em> genes; (C) Distribution of repeat numbers in di- and trinucleotide SSRs. Bars of different colours show the number of genic SSRs from dinucleotide and trinucleotide.</p></div></div><div content-id="DSS031F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03102.jpeg?Expires=1509238854&Signature=BCTdAibjqpENASnHV0vZvrLbLI0A02C4dmfmXrgOOTbQgkkL0LXZVfsjK~3o~b70zQLwuQmDSPAd1WlsdTM1ElhWsrKbGQ9wZqljAIPnbr2NOVnzGBc2ZfA88d5LlZ1gnzEcRJLr282FeKUPRYCkXF1q5eOoDc7IHcxqmoN4NmBEaEC1jCsmzd-Sg-hhcyVlfesxoEqMjHOTPHkZ030bDUYmybGZqASnJPU4uRoJU-bEBYry2aW3Lctm6VY7MNJVPGCwSxgSE42vPSZGF1n1aWRr-Q5yWEJYHOXmfm5L~XtQoIyU7OGmNl2iMjhq3r0uRqupxsfyVuQazm38QZC5qQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(A) Distribution of repeat motifs in 544 genic SSRs identified in 138 P. tomentosa genes; (B) Distribution of dinucleotide repeat motifs detected in 138 P. tomentosa genes; (C) Distribution of repeat numbers in di- and trinucleotide SSRs. Bars of different colours show the number of genic SSRs from dinucleotide and trinucleotide." path-from-xml="dss03102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605104/dss03102.jpeg" path-from-xml="dss03102.jpeg" target="_blank">View large</a><a section="5605104" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03102.jpeg?Expires=1509238854&Signature=H9nHcBXbA7ki7Upcx6J3~HpaYvovbNNy0AQO8mhpbyfrO8IaLzUFE1REhUH94Zl~iRwej-m5ilXQJK0f96tVyma2MAq~r9UKtx0lCGoWIrlJzf4UG74J94AjXDf-6kr46CDjqP9jHMjlsb5euzzBt61hn22U5MGm-A5RJVuUXFsaQKEVLbqIVx0ye53piHvbpqRC-aREXBj2gYRyqU6MbNO5gSYKs3IJTY4fS7sxX39h1ecpqLw5~6WEVWsazrx2R0pnPBkjnrkQAcJuxWM3eX~085LsjFwSMt3IGKWLfs2jjaD~~pL9TMKhVvBxxUNFIzAgMgRBCtGnqx2tAkQHdg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605104&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Distribution of repeat motifs in 544 genic SSRs identified in 138 <em>P. tomentosa</em> genes; (B) Distribution of dinucleotide repeat motifs detected in 138 <em>P. tomentosa</em> genes; (C) Distribution of repeat numbers in di- and trinucleotide SSRs. Bars of different colours show the number of genic SSRs from dinucleotide and trinucleotide.</p></div></div>                            <h3 scrollto-destination=5605105 id="5605105" class="section-title" data-legacy-id=s3b>3.2. SSR primer design, screening, and polymorphism testing</h3>
<p>To determine whether the SSRs varied in length, and would therefore provide a useful marker, we designed flanking primers and determined the length of each SSR by capillary electrophoresis. Primer pairs specific for flanking sequences were designed for 481 of the SSR sequences (88.5%). Using the <em>P. tomentosa</em> LM50 clone, 413 primer pairs (85.9%) gave successful amplification, and the remaining 68 failed to generate PCR products at any annealing temperature used (Ta), and so were excluded from further analysis (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a>). Of the 413 working primer pairs, 292 (70.7%) amplified PCR products of the expected sizes; however, 65 and 20 PCR products were larger or smaller, respectively, than expected, and the remaining 36 primer pairs generated multiple PCR products (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a>). Details of the 292 optimal primers, including locus names, primer sequences, SSR repeat motifs, Ta, expected sizes, SSR sources (locations in genes), and GenBank accession numbers are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>. Distribution summaries, frequencies, and repeat motifs of these 292 genic SSR markers are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7A</a>.</p><p>To identify whether the identified SSRs varied in length in <em>P. tomentosa</em>, these 292 primers were subjected to further screening for polymorphisms in 15 wild <em>P. tomentosa</em> ecotypes. Twenty four primers amplified monomorphic products (1 band), 80 markers have identical heterozygotes among all 15 individuals (2 bands), and 188 primers (64.4%) generated clean and reproducible polymorphic bands (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The observed product sizes and numbers of alleles per locus (<em>N</em><sub>A</sub>) were calculated for each marker. A total of 861 unique alleles were identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). For the 188 polymorphic markers from 107 genes, the <em>N</em><sub>A</sub> ranged from 2 to 7 with an average of 3.6 observed alleles per polymorphic locus; in other regions, the mean <em>N</em><sub>A</sub> values varied between 2.4 for exons and 4.0 for promoter regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7B</a>).</p>                            <h3 scrollto-destination=5605108 id="5605108" class="section-title" data-legacy-id=s3c>3.3. SSR cross-species transferability within the genus Populus</h3>
<p>To determine the utility of this SSR marker set beyond <em>P. tomentosa</em>, we screened a subset of the markers to determine whether they could be amplified from other related species, including <em>Populus</em> and the closely related <em>Salix</em> species. The capacity of 30 representative <em>P. tomentosa</em> genic SSR primers to screen for polymorphisms was tested in 27 ecotypes of related <em>Populus</em> and <em>Salix</em> species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 3</a>). All tested SSR primer pairs displayed a high amplification frequency within and across sections at the subgenus level within <em>Populus</em>. Of the 30 examined SSR markers, 12 (40%) successfully amplified in all species. The transferability of the 30 primers tested in <em>Populus</em> sections was <em>Leuce</em> (100%), <em>Tacamahaca</em> (90%), <em>Aigeros</em> (83%), <em>Leucoides</em> (80%), and <em>Turanga</em> (72%) (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>). In <em>S. matsudana</em>, the frequency of amplification was lower, and only 50% yielded product (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>). Thus, the developed SSR markers could be applied across the genus <em>Populus</em> and provided data on polymorphisms among related species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="5605110" scrollto-destination="5605110"></a>
<div class="table-wrap"><div id="DSS031TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Diversity of 30 polymorphic <em>P. tomentosa</em> genic SSR markers for 26 genotypes within the genus <em>Populus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Locus </th><th rowspan="2">Total <em>N</em><sub>A</sub> </th><th rowspan="2">PIC </th><th colspan="3"><em>Tacamahaca</em><hr /> </th><th colspan="3"><em>Leuce</em><hr /> </th><th colspan="3"><em>Aigeiros</em><hr /> </th><th colspan="3"><em>Turanga</em><hr /> </th><th colspan="3"><em>Leucoides</em><hr /> </th><th colspan="3"><em>S. matsudana</em> (outgroup)<hr /> </th></tr><tr><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th>H<sub>E</sub> </th></tr></thead><tbody><tr><td>SSR12 </td><td>7 </td><td>0.663 </td><td>230–242 </td><td>2 </td><td>0.802 </td><td>230–248 </td><td>5 </td><td>0.363 </td><td>230–242 </td><td>3 </td><td>0.607 </td><td>246–248 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>/ </td><td>252–256 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR34 </td><td>6 </td><td>0.601 </td><td>98–102 </td><td>3 </td><td>0.675 </td><td>96–100 </td><td>2 </td><td>0.539 </td><td>0 </td><td>0 </td><td>/ </td><td>98 </td><td>1 </td><td>0.000 </td><td>96–98 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR39 </td><td>5 </td><td>0.457 </td><td>119–134 </td><td>3 </td><td>0.607 </td><td>119–128 </td><td>3 </td><td>0.750 </td><td>119–125 </td><td>2 </td><td>0.508 </td><td>0 </td><td>0 </td><td>/ </td><td>119–125 </td><td>2 </td><td>1.000 </td><td>134 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR43 </td><td>7 </td><td>0.783 </td><td>211–217 </td><td>4 </td><td>0.825 </td><td>211–227 </td><td>5 </td><td>0.615 </td><td>211–217 </td><td>4 </td><td>0.644 </td><td>189 </td><td>1 </td><td>0.000 </td><td>211–213 </td><td>2 </td><td>1.000 </td><td>195–198 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR47 </td><td>4 </td><td>0.548 </td><td>0 </td><td>0 </td><td>/ </td><td>192–198 </td><td>3 </td><td>0.458 </td><td>0 </td><td>0 </td><td>/ </td><td>201 </td><td>1 </td><td>0.000 </td><td>195–198 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR53 </td><td>10 </td><td>0.820 </td><td>117–131 </td><td>6 </td><td>0.864 </td><td>115–123 </td><td>5 </td><td>0.750 </td><td>111–117 </td><td>3 </td><td>0.714 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR54 </td><td>11 </td><td>0.841 </td><td>146–156 </td><td>4 </td><td>0.661 </td><td>136–142 </td><td>3 </td><td>0.538 </td><td>134–152 </td><td>4 </td><td>0.607 </td><td>138–144 </td><td>2 </td><td>0.667 </td><td>132–136 </td><td>2 </td><td>1.000 </td><td>136 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR57 </td><td>10 </td><td>0.818 </td><td>245–253 </td><td>4 </td><td>0.778 </td><td>227–243 </td><td>4 </td><td>0.561 </td><td>221–227 </td><td>2 </td><td>0.533 </td><td>253 </td><td>1 </td><td>0.000 </td><td>229 </td><td>1 </td><td>0.000 </td><td>261–265 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR58 </td><td>6 </td><td>0.539 </td><td>107–119 </td><td>4 </td><td>0.632 </td><td>110–121 </td><td>4 </td><td>0.503 </td><td>110 </td><td>1 </td><td>0.000 </td><td>110 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR62 </td><td>4 </td><td>0.529 </td><td>0 </td><td>0 </td><td>/ </td><td>399–403 </td><td>3 </td><td>0.250 </td><td>401–403 </td><td>2 </td><td>0.250 </td><td>403–405 </td><td>2 </td><td>0.667 </td><td>399 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR64 </td><td>6 </td><td>0.632 </td><td>377 </td><td>1 </td><td>0.000 </td><td>373–379 </td><td>4 </td><td>0.821 </td><td>375–377 </td><td>2 </td><td>0.429 </td><td>371 </td><td>1 </td><td>0.000 </td><td>375 </td><td>1 </td><td>0.000 </td><td>385 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR65 </td><td>6 </td><td>0.411 </td><td>263–273 </td><td>4 </td><td>0.725 </td><td>257–263 </td><td>3 </td><td>0.264 </td><td>0 </td><td>0 </td><td>/ </td><td>265 </td><td>1 </td><td>0.000 </td><td>271 </td><td>1 </td><td>0.000 </td><td>269–273 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR66 </td><td>5 </td><td>0.463 </td><td>0 </td><td>0 </td><td>/ </td><td>215–217 </td><td>2 </td><td>0.440 </td><td>199–219 </td><td>3 </td><td>0.644 </td><td>221 </td><td>1 </td><td>0.000 </td><td>217–221 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR67 </td><td>7 </td><td>0.789 </td><td>240–242 </td><td>2 </td><td>0.636 </td><td>242–246 </td><td>3 </td><td>0.350 </td><td>242–250 </td><td>3 </td><td>0.607 </td><td>246–250 </td><td>2 </td><td>0.667 </td><td>246 </td><td>1 </td><td>1.000 </td><td>260 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR69 </td><td>9 </td><td>0.834 </td><td>213–219 </td><td>4 </td><td>0.928 </td><td>221–235 </td><td>4 </td><td>0.333 </td><td>215–227 </td><td>4 </td><td>0.786 </td><td>219 </td><td>1 </td><td>0.000 </td><td>219 </td><td>1 </td><td>0.000 </td><td>209–213 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR70 </td><td>8 </td><td>0.727 </td><td>310–322 </td><td>4 </td><td>0.643 </td><td>312–318 </td><td>4 </td><td>0.733 </td><td>314–324 </td><td>3 </td><td>0.607 </td><td>0 </td><td>0 </td><td>/ </td><td>314–318 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR71 </td><td>10 </td><td>0.771 </td><td>192–228 </td><td>6 </td><td>0.849 </td><td>201–222 </td><td>4 </td><td>0.636 </td><td>228–246 </td><td>4 </td><td>0.821 </td><td>234 </td><td>1 </td><td>0.000 </td><td>204–234 </td><td>2 </td><td>1.000 </td><td>237 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR73 </td><td>10 </td><td>0.828 </td><td>169–183 </td><td>4 </td><td>0.588 </td><td>177–191 </td><td>5 </td><td>0.835 </td><td>173–191 </td><td>6 </td><td>0.703 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR74 </td><td>7 </td><td>0.750 </td><td>114–120 </td><td>3 </td><td>0.439 </td><td>114–124 </td><td>4 </td><td>0.846 </td><td>118–126 </td><td>4 </td><td>0.711 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR75 </td><td>8 </td><td>0.766 </td><td>202–220 </td><td>4 </td><td>0.867 </td><td>202–210 </td><td>4 </td><td>0.679 </td><td>204–210 </td><td>4 </td><td>0.750 </td><td>206–208 </td><td>2 </td><td>0.667 </td><td>206–208 </td><td>2 </td><td>1.000 </td><td>220. </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR77 </td><td>8 </td><td>0.690 </td><td>165–179 </td><td>5 </td><td>0.933 </td><td>163–169 </td><td>3 </td><td>0.604 </td><td>165–175 </td><td>4 </td><td>0.679 </td><td>165–169 </td><td>2 </td><td>0.667 </td><td>165–169 </td><td>2 </td><td>1.000 </td><td>181 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR91 </td><td>5 </td><td>0.640 </td><td>139–151 </td><td>4 </td><td>0.636 </td><td>135–147 </td><td>3 </td><td>0.590 </td><td>0 </td><td>0 </td><td>/ </td><td>147–155 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR96 </td><td>6 </td><td>0.739 </td><td>111–123 </td><td>3 </td><td>0.725 </td><td>108–123 </td><td>4 </td><td>0.632 </td><td>114–117 </td><td>2 </td><td>0.596 </td><td>0 </td><td>0 </td><td>/ </td><td>114 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR98 </td><td>8 </td><td>0.798 </td><td>224–234 </td><td>4 </td><td>0.642 </td><td>224–238 </td><td>5 </td><td>0.846 </td><td>226–236 </td><td>3 </td><td>0.607 </td><td>226–228 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR113 </td><td>6 </td><td>0.591 </td><td>193–199 </td><td>3 </td><td>0.408 </td><td>184–196 </td><td>4 </td><td>0.630 </td><td>196–202 </td><td>3 </td><td>0.593 </td><td>187–193 </td><td>2 </td><td>0.667 </td><td>193 </td><td>1 </td><td>0.000 </td><td>184–187 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR165 </td><td>8 </td><td>0.637 </td><td>310–322 </td><td>4 </td><td>0.636 </td><td>307–322 </td><td>4 </td><td>0.532 </td><td>304–316 </td><td>5 </td><td>0.720 </td><td>319–325 </td><td>3 </td><td>0.833 </td><td>304–310 </td><td>2 </td><td>1.000 </td><td>322 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR169 </td><td>7 </td><td>0.664 </td><td>209–219 </td><td>4 </td><td>0.650 </td><td>205–217 </td><td>4 </td><td>0.633 </td><td>0 </td><td>0 </td><td>/ </td><td>205–209 </td><td>2 </td><td>0.667 </td><td>211 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR170 </td><td>6 </td><td>0.479 </td><td>214–220 </td><td>4 </td><td>0.569 </td><td>202–220 </td><td>4 </td><td>0.705 </td><td>202–214 </td><td>2 </td><td>0.511 </td><td>212–220 </td><td>3 </td><td>0.833 </td><td>202–208 </td><td>2 </td><td>1.000 </td><td>222 </td><td>0 </td><td>/ </td></tr><tr><td>SSR176 </td><td>5 </td><td>0.732 </td><td>343–355 </td><td>4 </td><td>0.701 </td><td>346–355 </td><td>2 </td><td>0.360 </td><td>343–355 </td><td>3 </td><td>0.531 </td><td>0 </td><td>0 </td><td>/ </td><td>358 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR179 </td><td>8 </td><td>0.679 </td><td>245–261 </td><td>3 </td><td>0.558 </td><td>249–263 </td><td>4 </td><td>0.687 </td><td>253–261 </td><td>3 </td><td>0.607 </td><td>263–265 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>Mean </td><td>7.0 </td><td>0.675 </td><td>/ </td><td>3.4 </td><td>0.599 </td><td>/ </td><td>3.7 </td><td>0.582 </td><td>/ </td><td>2.6 </td><td>0.492 </td><td>/ </td><td>1.3 </td><td>0.300 </td><td>/ </td><td>1.1 </td><td>0.467 </td><td>/ </td><td>0.7 </td><td>0.233 </td></tr></tbody></table></div><div content-id="DSS031TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Locus </th><th rowspan="2">Total <em>N</em><sub>A</sub> </th><th rowspan="2">PIC </th><th colspan="3"><em>Tacamahaca</em><hr /> </th><th colspan="3"><em>Leuce</em><hr /> </th><th colspan="3"><em>Aigeiros</em><hr /> </th><th colspan="3"><em>Turanga</em><hr /> </th><th colspan="3"><em>Leucoides</em><hr /> </th><th colspan="3"><em>S. matsudana</em> (outgroup)<hr /> </th></tr><tr><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th><em>H</em><sub>E</sub> </th><th>Size (bp) </th><th><em>N</em><sub>A</sub> </th><th>H<sub>E</sub> </th></tr></thead><tbody><tr><td>SSR12 </td><td>7 </td><td>0.663 </td><td>230–242 </td><td>2 </td><td>0.802 </td><td>230–248 </td><td>5 </td><td>0.363 </td><td>230–242 </td><td>3 </td><td>0.607 </td><td>246–248 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>/ </td><td>252–256 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR34 </td><td>6 </td><td>0.601 </td><td>98–102 </td><td>3 </td><td>0.675 </td><td>96–100 </td><td>2 </td><td>0.539 </td><td>0 </td><td>0 </td><td>/ </td><td>98 </td><td>1 </td><td>0.000 </td><td>96–98 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR39 </td><td>5 </td><td>0.457 </td><td>119–134 </td><td>3 </td><td>0.607 </td><td>119–128 </td><td>3 </td><td>0.750 </td><td>119–125 </td><td>2 </td><td>0.508 </td><td>0 </td><td>0 </td><td>/ </td><td>119–125 </td><td>2 </td><td>1.000 </td><td>134 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR43 </td><td>7 </td><td>0.783 </td><td>211–217 </td><td>4 </td><td>0.825 </td><td>211–227 </td><td>5 </td><td>0.615 </td><td>211–217 </td><td>4 </td><td>0.644 </td><td>189 </td><td>1 </td><td>0.000 </td><td>211–213 </td><td>2 </td><td>1.000 </td><td>195–198 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR47 </td><td>4 </td><td>0.548 </td><td>0 </td><td>0 </td><td>/ </td><td>192–198 </td><td>3 </td><td>0.458 </td><td>0 </td><td>0 </td><td>/ </td><td>201 </td><td>1 </td><td>0.000 </td><td>195–198 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR53 </td><td>10 </td><td>0.820 </td><td>117–131 </td><td>6 </td><td>0.864 </td><td>115–123 </td><td>5 </td><td>0.750 </td><td>111–117 </td><td>3 </td><td>0.714 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR54 </td><td>11 </td><td>0.841 </td><td>146–156 </td><td>4 </td><td>0.661 </td><td>136–142 </td><td>3 </td><td>0.538 </td><td>134–152 </td><td>4 </td><td>0.607 </td><td>138–144 </td><td>2 </td><td>0.667 </td><td>132–136 </td><td>2 </td><td>1.000 </td><td>136 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR57 </td><td>10 </td><td>0.818 </td><td>245–253 </td><td>4 </td><td>0.778 </td><td>227–243 </td><td>4 </td><td>0.561 </td><td>221–227 </td><td>2 </td><td>0.533 </td><td>253 </td><td>1 </td><td>0.000 </td><td>229 </td><td>1 </td><td>0.000 </td><td>261–265 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR58 </td><td>6 </td><td>0.539 </td><td>107–119 </td><td>4 </td><td>0.632 </td><td>110–121 </td><td>4 </td><td>0.503 </td><td>110 </td><td>1 </td><td>0.000 </td><td>110 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR62 </td><td>4 </td><td>0.529 </td><td>0 </td><td>0 </td><td>/ </td><td>399–403 </td><td>3 </td><td>0.250 </td><td>401–403 </td><td>2 </td><td>0.250 </td><td>403–405 </td><td>2 </td><td>0.667 </td><td>399 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR64 </td><td>6 </td><td>0.632 </td><td>377 </td><td>1 </td><td>0.000 </td><td>373–379 </td><td>4 </td><td>0.821 </td><td>375–377 </td><td>2 </td><td>0.429 </td><td>371 </td><td>1 </td><td>0.000 </td><td>375 </td><td>1 </td><td>0.000 </td><td>385 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR65 </td><td>6 </td><td>0.411 </td><td>263–273 </td><td>4 </td><td>0.725 </td><td>257–263 </td><td>3 </td><td>0.264 </td><td>0 </td><td>0 </td><td>/ </td><td>265 </td><td>1 </td><td>0.000 </td><td>271 </td><td>1 </td><td>0.000 </td><td>269–273 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR66 </td><td>5 </td><td>0.463 </td><td>0 </td><td>0 </td><td>/ </td><td>215–217 </td><td>2 </td><td>0.440 </td><td>199–219 </td><td>3 </td><td>0.644 </td><td>221 </td><td>1 </td><td>0.000 </td><td>217–221 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR67 </td><td>7 </td><td>0.789 </td><td>240–242 </td><td>2 </td><td>0.636 </td><td>242–246 </td><td>3 </td><td>0.350 </td><td>242–250 </td><td>3 </td><td>0.607 </td><td>246–250 </td><td>2 </td><td>0.667 </td><td>246 </td><td>1 </td><td>1.000 </td><td>260 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR69 </td><td>9 </td><td>0.834 </td><td>213–219 </td><td>4 </td><td>0.928 </td><td>221–235 </td><td>4 </td><td>0.333 </td><td>215–227 </td><td>4 </td><td>0.786 </td><td>219 </td><td>1 </td><td>0.000 </td><td>219 </td><td>1 </td><td>0.000 </td><td>209–213 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR70 </td><td>8 </td><td>0.727 </td><td>310–322 </td><td>4 </td><td>0.643 </td><td>312–318 </td><td>4 </td><td>0.733 </td><td>314–324 </td><td>3 </td><td>0.607 </td><td>0 </td><td>0 </td><td>/ </td><td>314–318 </td><td>2 </td><td>1.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR71 </td><td>10 </td><td>0.771 </td><td>192–228 </td><td>6 </td><td>0.849 </td><td>201–222 </td><td>4 </td><td>0.636 </td><td>228–246 </td><td>4 </td><td>0.821 </td><td>234 </td><td>1 </td><td>0.000 </td><td>204–234 </td><td>2 </td><td>1.000 </td><td>237 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR73 </td><td>10 </td><td>0.828 </td><td>169–183 </td><td>4 </td><td>0.588 </td><td>177–191 </td><td>5 </td><td>0.835 </td><td>173–191 </td><td>6 </td><td>0.703 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR74 </td><td>7 </td><td>0.750 </td><td>114–120 </td><td>3 </td><td>0.439 </td><td>114–124 </td><td>4 </td><td>0.846 </td><td>118–126 </td><td>4 </td><td>0.711 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR75 </td><td>8 </td><td>0.766 </td><td>202–220 </td><td>4 </td><td>0.867 </td><td>202–210 </td><td>4 </td><td>0.679 </td><td>204–210 </td><td>4 </td><td>0.750 </td><td>206–208 </td><td>2 </td><td>0.667 </td><td>206–208 </td><td>2 </td><td>1.000 </td><td>220. </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR77 </td><td>8 </td><td>0.690 </td><td>165–179 </td><td>5 </td><td>0.933 </td><td>163–169 </td><td>3 </td><td>0.604 </td><td>165–175 </td><td>4 </td><td>0.679 </td><td>165–169 </td><td>2 </td><td>0.667 </td><td>165–169 </td><td>2 </td><td>1.000 </td><td>181 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR91 </td><td>5 </td><td>0.640 </td><td>139–151 </td><td>4 </td><td>0.636 </td><td>135–147 </td><td>3 </td><td>0.590 </td><td>0 </td><td>0 </td><td>/ </td><td>147–155 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR96 </td><td>6 </td><td>0.739 </td><td>111–123 </td><td>3 </td><td>0.725 </td><td>108–123 </td><td>4 </td><td>0.632 </td><td>114–117 </td><td>2 </td><td>0.596 </td><td>0 </td><td>0 </td><td>/ </td><td>114 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR98 </td><td>8 </td><td>0.798 </td><td>224–234 </td><td>4 </td><td>0.642 </td><td>224–238 </td><td>5 </td><td>0.846 </td><td>226–236 </td><td>3 </td><td>0.607 </td><td>226–228 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR113 </td><td>6 </td><td>0.591 </td><td>193–199 </td><td>3 </td><td>0.408 </td><td>184–196 </td><td>4 </td><td>0.630 </td><td>196–202 </td><td>3 </td><td>0.593 </td><td>187–193 </td><td>2 </td><td>0.667 </td><td>193 </td><td>1 </td><td>0.000 </td><td>184–187 </td><td>2 </td><td>1.000 </td></tr><tr><td>SSR165 </td><td>8 </td><td>0.637 </td><td>310–322 </td><td>4 </td><td>0.636 </td><td>307–322 </td><td>4 </td><td>0.532 </td><td>304–316 </td><td>5 </td><td>0.720 </td><td>319–325 </td><td>3 </td><td>0.833 </td><td>304–310 </td><td>2 </td><td>1.000 </td><td>322 </td><td>1 </td><td>0.000 </td></tr><tr><td>SSR169 </td><td>7 </td><td>0.664 </td><td>209–219 </td><td>4 </td><td>0.650 </td><td>205–217 </td><td>4 </td><td>0.633 </td><td>0 </td><td>0 </td><td>/ </td><td>205–209 </td><td>2 </td><td>0.667 </td><td>211 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR170 </td><td>6 </td><td>0.479 </td><td>214–220 </td><td>4 </td><td>0.569 </td><td>202–220 </td><td>4 </td><td>0.705 </td><td>202–214 </td><td>2 </td><td>0.511 </td><td>212–220 </td><td>3 </td><td>0.833 </td><td>202–208 </td><td>2 </td><td>1.000 </td><td>222 </td><td>0 </td><td>/ </td></tr><tr><td>SSR176 </td><td>5 </td><td>0.732 </td><td>343–355 </td><td>4 </td><td>0.701 </td><td>346–355 </td><td>2 </td><td>0.360 </td><td>343–355 </td><td>3 </td><td>0.531 </td><td>0 </td><td>0 </td><td>/ </td><td>358 </td><td>1 </td><td>0.000 </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>SSR179 </td><td>8 </td><td>0.679 </td><td>245–261 </td><td>3 </td><td>0.558 </td><td>249–263 </td><td>4 </td><td>0.687 </td><td>253–261 </td><td>3 </td><td>0.607 </td><td>263–265 </td><td>2 </td><td>0.667 </td><td>0 </td><td>0 </td><td>/ </td><td>0 </td><td>0 </td><td>/ </td></tr><tr><td>Mean </td><td>7.0 </td><td>0.675 </td><td>/ </td><td>3.4 </td><td>0.599 </td><td>/ </td><td>3.7 </td><td>0.582 </td><td>/ </td><td>2.6 </td><td>0.492 </td><td>/ </td><td>1.3 </td><td>0.300 </td><td>/ </td><td>1.1 </td><td>0.467 </td><td>/ </td><td>0.7 </td><td>0.233 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a> for further details of the 30 genic SSR markers.</p></div></div><div content-id=""><div class="fn"><p>The observed number of alleles per locus (<em>N</em><sub>A</sub>), PIC, expected heterozygosity (<em>H</em><sub>E</sub>), and not applied (/).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5605110" target="_blank">View Large</a></div></div><p>We also examined the allelic variation in these ecotypes, to determine whether these SSR markers could prove useful for genetic studies such as association mapping. Of the 27 ecotypes, 213 alleles were detected using the 30 SSR primers, with an average of 7.0 alleles per locus, ranging from 4 at SSR62 to 11 at SSR54 (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>). The PIC mean polymorphism level of the loci was 0.675, and it ranged from 0.411 for SSR65 to 0.841 for SSR54 (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>). In the five <em>Populus</em> sections, the mean <em>N</em><sub>A</sub> values were from 1.1 (<em>Leucoides</em>) to 3.7 (<em>Leuce</em>). The other diversity parameters (observed lengths and <em>H</em><sub>E</sub>) are shown in Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>.</p><p>We also tested whether these markers were informative for genetic analysis by determining whether the marker genotypes could be used to recapitulate the known phylogenetic relationship among the tested ecotypes. The NJ tree (Fig. <a reveal-id="DSS031F3" data-open="DSS031F3" class="link link-reveal link-table xref-fig">3</a>) based on the shared allele coefficients from these 30 SSRs revealed the expected genetic relationships among all 27 ecotypes and clustered them into 6 main groups, in agreement with 5 putative sections, and the outgroup was derived from the basic botanical classification of these tree species.<sup><a reveal-id="DSS031C10" data-open="DSS031C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS031C24" data-open="DSS031C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The relationships among species in each section are represented by the smaller branches within groups. In addition, we constructed trees based on the other functions in PowerMarker, and these were very similar to the NJ tree (Fig. <a reveal-id="DSS031F3" data-open="DSS031F3" class="link link-reveal link-table xref-fig">3</a>) in that the six distinct clusters were again present, although there were slight differences in branch lengths within clusters (data not shown). </p>                        <a id="5605116" scrollto-destination="5605116"></a>
<div id="DSS031F3" data-id="DSS031F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03103.jpeg?Expires=1509238854&Signature=F8vtSRAhC-NI-R3VB3lraTWXzvl6GXNSAuAXIepbLNNx9ojmvrmNQkRt~EOQWir7kGQXBinGDoGIN7zsnlgnzo6ZAKI1HX4faeSP7yJtXEuodPGkeXcvSxwMjXy1V-uWv-1mK3nqOWopwBOcbVsZPw3CAmexzdewRfCa2I8awUmrZDl-U3Y2Bv9J2lLKUxCIp7saWGdyi8Qt8kNm9AG92u7REkdls3W3UlOZ1v09TinWt16w0L8mEb8vOYhXmGlYwqTjZTLBTOW3p4lD1yBGgwCVpDZ~gz4HUXqnKCfnuNul699tky7Lp1IBp1XOax7PvqALB90GuMJpUb7r2ON9GQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenic relationship among 26 genotypes belonging to 5 sections of the genus Populus (S. matsudana as outgroup) based on 30 polymorphic genic SSR markers. The different colour branches denote the divergent clusters." path-from-xml="dss03103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605116/dss03103.jpeg" path-from-xml="dss03103.jpeg" target="_blank">View large</a><a section="5605116" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03103.jpeg?Expires=1509238854&Signature=Xu~Mx7T03kzMOtHUCYzzGUtzZzr-IxgyfGFshKH8bGNzDFFnnv--0CgN9jKwl3ukvjthlIFDcghsF6txfLUuFYxPL1LuRqMZ59inGBFmaZATUDJw~pMDSTR0n3h~XTrqz4Z6QDdFkC-Gdxh0Bwf0zKB8lMeJDLpob7VO~9mFqUNcMa1E41s76IGos~nRpc6OBNCKz4ghzZL1iiQ7VOguGNS-w7pim5ADLIhvblRRUrXw1bawi9VOoQ~hBQ4rW7F~b9yDpz~av7BnE~Uhsc7Lrj7tYXPTTF3BlkKKbj0ra76BJPDV6n~lIMinNgyxgU4UExHgPHcaeC0u0EIhk5P6aA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605116&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenic relationship among 26 genotypes belonging to 5 sections of the genus <em>Populus</em> (<em>S. matsudana</em> as outgroup) based on 30 polymorphic genic SSR markers. The different colour branches denote the divergent clusters.</p></div></div><div content-id="DSS031F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03103.jpeg?Expires=1509238854&Signature=F8vtSRAhC-NI-R3VB3lraTWXzvl6GXNSAuAXIepbLNNx9ojmvrmNQkRt~EOQWir7kGQXBinGDoGIN7zsnlgnzo6ZAKI1HX4faeSP7yJtXEuodPGkeXcvSxwMjXy1V-uWv-1mK3nqOWopwBOcbVsZPw3CAmexzdewRfCa2I8awUmrZDl-U3Y2Bv9J2lLKUxCIp7saWGdyi8Qt8kNm9AG92u7REkdls3W3UlOZ1v09TinWt16w0L8mEb8vOYhXmGlYwqTjZTLBTOW3p4lD1yBGgwCVpDZ~gz4HUXqnKCfnuNul699tky7Lp1IBp1XOax7PvqALB90GuMJpUb7r2ON9GQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenic relationship among 26 genotypes belonging to 5 sections of the genus Populus (S. matsudana as outgroup) based on 30 polymorphic genic SSR markers. The different colour branches denote the divergent clusters." path-from-xml="dss03103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605116/dss03103.jpeg" path-from-xml="dss03103.jpeg" target="_blank">View large</a><a section="5605116" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03103.jpeg?Expires=1509238854&Signature=Xu~Mx7T03kzMOtHUCYzzGUtzZzr-IxgyfGFshKH8bGNzDFFnnv--0CgN9jKwl3ukvjthlIFDcghsF6txfLUuFYxPL1LuRqMZ59inGBFmaZATUDJw~pMDSTR0n3h~XTrqz4Z6QDdFkC-Gdxh0Bwf0zKB8lMeJDLpob7VO~9mFqUNcMa1E41s76IGos~nRpc6OBNCKz4ghzZL1iiQ7VOguGNS-w7pim5ADLIhvblRRUrXw1bawi9VOoQ~hBQ4rW7F~b9yDpz~av7BnE~Uhsc7Lrj7tYXPTTF3BlkKKbj0ra76BJPDV6n~lIMinNgyxgU4UExHgPHcaeC0u0EIhk5P6aA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605116&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenic relationship among 26 genotypes belonging to 5 sections of the genus <em>Populus</em> (<em>S. matsudana</em> as outgroup) based on 30 polymorphic genic SSR markers. The different colour branches denote the divergent clusters.</p></div></div>                            <h3 scrollto-destination=5605125 id="5605125" class="section-title" data-legacy-id=s3d>3.4. Comparison of SSR variation between P. tomentosa and P. trichocarpa</h3>
<p>In addition to testing a subset of markers in multiple <em>Populus</em> and <em>Salix</em> species, we also examined these markers more extensively in the sequenced genome of <em>P. trichocarpa.</em> For all 292 genic SSR-containing sequences, only 75% (220) of homologous <em>P. trichocarpa</em> sequences (length, 90–593 bp) were identified (Fig. <a reveal-id="DSS031F1" data-open="DSS031F1" class="link link-reveal link-table xref-fig">1</a>). Alignment and comparison of the homologous sequences between these two species for each marker revealed 10 types of mutations and variations in SSR loci and their flanking regions (Table <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSS031F4" data-open="DSS031F4" class="link link-reveal link-table xref-fig">4</a>). For example, the SSR53 locus, which contains an imperfect (CT) motif, showed complex mutations characterized by variation in the repeat-motif length and point mutations in both the repeat motif and the flanking sequences (data not shown). Of all 220 genic SSRs, 39.1% (86) were present in <em>P. trichocarpa</em>, with 57 showing only variation in repeat number in both species (conserved flanking sequences) and 29 that were monomorphic and had conserved sequences between the 2 species (Table <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">2</a>). However, 41.8% (92) had polymorphisms of the SSR flanking sequences between the two species, suggesting that these SSR markers from <em>P. tomentosa</em> are not directly transferable to <em>P. trichocarpa</em> (Table <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">2</a>). A comparison of the SSR markers with mutations in flanking sequences showed the highest proportion in 3′UTRs (60.0%) and the lowest in 5′UTRs (25.8%). A total of 25 new SSR markers were identified in the corresponding sequences of <em>P. trichocarpa</em>, of which 17 (68%) were derived from promoter regions (Table <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">2</a>). Additionally, 100% of SSR markers from exon regions could be directly employed in genetic studies of <em>P. trichocarpa</em> (Table <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="5605138" scrollto-destination="5605138"></a>
<div class="table-wrap"><div id="DSS031TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Comparison of variations and mutations of 220 genic SSR loci and their flanking sequences between <em>P. tomentosa</em> and <em>P. trichocarpa</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types of variations and mutations </th><th>Promoters </th><th>Introns </th><th>5′UTRs </th><th>3′UTRs </th><th>Exons </th><th>Total </th></tr></thead><tbody><tr><td>Mutation in the flanking sequence only </td><td>5 </td><td>3 </td><td>2 </td><td>1 </td><td>0 </td><td>11 </td></tr><tr><td>Mutation in the flanking sequence and repeat number </td><td>24 </td><td>24 </td><td>6 </td><td>6 </td><td>0 </td><td>60 </td></tr><tr><td>Mutation in the flanking sequence and no SSR marker </td><td>7 </td><td>6 </td><td>0 </td><td>2 </td><td>0 </td><td>15 </td></tr><tr><td>Mutation in the flanking sequence and new SSR marker </td><td>4 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>6 </td></tr><tr><td>Variation of repeat number only </td><td>18 </td><td>24 </td><td>10 </td><td>3 </td><td>2 </td><td>57 </td></tr><tr><td>Mutation in SSR repeat motif only </td><td>1 </td><td>4 </td><td>3 </td><td>1 </td><td>0 </td><td>9 </td></tr><tr><td>Mutation in SSR repeat motif and repeat number </td><td>3 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>4 </td></tr><tr><td>No SSR marker only </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>4 </td></tr><tr><td>New SSR marker only </td><td>17 </td><td>5 </td><td>3 </td><td>0 </td><td>0 </td><td>25 </td></tr><tr><td>Identical sequence </td><td>9 </td><td>7 </td><td>6 </td><td>1 </td><td>6 </td><td>29 </td></tr><tr><td>Total </td><td>89 </td><td>77 </td><td>31 </td><td>15 </td><td>8 </td><td>220 </td></tr></tbody></table></div><div content-id="DSS031TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Types of variations and mutations </th><th>Promoters </th><th>Introns </th><th>5′UTRs </th><th>3′UTRs </th><th>Exons </th><th>Total </th></tr></thead><tbody><tr><td>Mutation in the flanking sequence only </td><td>5 </td><td>3 </td><td>2 </td><td>1 </td><td>0 </td><td>11 </td></tr><tr><td>Mutation in the flanking sequence and repeat number </td><td>24 </td><td>24 </td><td>6 </td><td>6 </td><td>0 </td><td>60 </td></tr><tr><td>Mutation in the flanking sequence and no SSR marker </td><td>7 </td><td>6 </td><td>0 </td><td>2 </td><td>0 </td><td>15 </td></tr><tr><td>Mutation in the flanking sequence and new SSR marker </td><td>4 </td><td>2 </td><td>0 </td><td>0 </td><td>0 </td><td>6 </td></tr><tr><td>Variation of repeat number only </td><td>18 </td><td>24 </td><td>10 </td><td>3 </td><td>2 </td><td>57 </td></tr><tr><td>Mutation in SSR repeat motif only </td><td>1 </td><td>4 </td><td>3 </td><td>1 </td><td>0 </td><td>9 </td></tr><tr><td>Mutation in SSR repeat motif and repeat number </td><td>3 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>4 </td></tr><tr><td>No SSR marker only </td><td>1 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>4 </td></tr><tr><td>New SSR marker only </td><td>17 </td><td>5 </td><td>3 </td><td>0 </td><td>0 </td><td>25 </td></tr><tr><td>Identical sequence </td><td>9 </td><td>7 </td><td>6 </td><td>1 </td><td>6 </td><td>29 </td></tr><tr><td>Total </td><td>89 </td><td>77 </td><td>31 </td><td>15 </td><td>8 </td><td>220 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5605138" target="_blank">View Large</a></div></div>                        <a id="5605148" scrollto-destination="5605148"></a>
<div id="DSS031F4" data-id="DSS031F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03104.jpeg?Expires=1509238854&Signature=BwF0PUEwma7uoMvcHdWA7fVd9DSWOmcielQpxzTadXkAjZLZivgs5F~as1ia1vOu~gVNPiBP4Ym7y1F4ymXh78n1gTqm4FPaiqmj24doOequVTzOsXm~UXxiT-2BlJqcSxDhUj43UwhZlohDt8PBi-NCz2ETal7T2UdrCKWGltKOElpwi4qn650u2dIaopCrtXwM9V7L1-efwk0M9BC6185EWNnFMdxAyJyTYpif~7uuIXLgXfAhG9v0-R5Jtt9srGWba-f4TRYVo6UoQ-n0gcXyoHlbV~FrZf08kk5OT9upWIc3lVfqyt4jpOufRclHRbuDP7gBrn2BQhK3JB09~w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment and comparison of variations and mutations in genic SSRs with sequences homologous between P. tomentosa (01) and P. trichocarpa (02), based on a number of genic SSRs. Ten types of mutations and variations for genic SSR loci with their flanking regions were identified (See Table 2 for details)." path-from-xml="dss03104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605148/dss03104.jpeg" path-from-xml="dss03104.jpeg" target="_blank">View large</a><a section="5605148" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03104.jpeg?Expires=1509238854&Signature=EvCiMFDhr12UKyrAQ1wOyNhhQCHp9lR6h2WRsfg3A1~MbCOdV9TgNAaxahcZfuHOxUI4cMttWteKQM5YXtkN6ZXJQWpM5rAktDuWuzovC7Em9IYzyAJuKLFY1KwVG7X3aojlx~VjHM3wWU4V-YW7MOMWbTCU50NyaoEbWu3L0xuGqKZuvwkFhvMjAy3mNh9JjR~a8IvQKoAgadSWayj0CbwLGmgVThJfdgoWJ3QfJml8pBU2b3NnPyUWbNPXNl5OWgfeHpQqlsovaG9INL0AYgpmdTBnyeDGMvB0AQJtYh6FNONXc8J1NawojFTe6LxCsqXo0~kdhwzvg9E5o1-mPQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605148&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment and comparison of variations and mutations in genic SSRs with sequences homologous between <em>P. tomentosa</em> (01) and <em>P. trichocarpa</em> (02), based on a number of genic SSRs. Ten types of mutations and variations for genic SSR loci with their flanking regions were identified (See <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">Table 2</a> for details).</p></div></div><div content-id="DSS031F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss03104.jpeg?Expires=1509238854&Signature=BwF0PUEwma7uoMvcHdWA7fVd9DSWOmcielQpxzTadXkAjZLZivgs5F~as1ia1vOu~gVNPiBP4Ym7y1F4ymXh78n1gTqm4FPaiqmj24doOequVTzOsXm~UXxiT-2BlJqcSxDhUj43UwhZlohDt8PBi-NCz2ETal7T2UdrCKWGltKOElpwi4qn650u2dIaopCrtXwM9V7L1-efwk0M9BC6185EWNnFMdxAyJyTYpif~7uuIXLgXfAhG9v0-R5Jtt9srGWba-f4TRYVo6UoQ-n0gcXyoHlbV~FrZf08kk5OT9upWIc3lVfqyt4jpOufRclHRbuDP7gBrn2BQhK3JB09~w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment and comparison of variations and mutations in genic SSRs with sequences homologous between P. tomentosa (01) and P. trichocarpa (02), based on a number of genic SSRs. Ten types of mutations and variations for genic SSR loci with their flanking regions were identified (See Table 2 for details)." path-from-xml="dss03104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5605148/dss03104.jpeg" path-from-xml="dss03104.jpeg" target="_blank">View large</a><a section="5605148" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss03104.jpeg?Expires=1509238854&Signature=EvCiMFDhr12UKyrAQ1wOyNhhQCHp9lR6h2WRsfg3A1~MbCOdV9TgNAaxahcZfuHOxUI4cMttWteKQM5YXtkN6ZXJQWpM5rAktDuWuzovC7Em9IYzyAJuKLFY1KwVG7X3aojlx~VjHM3wWU4V-YW7MOMWbTCU50NyaoEbWu3L0xuGqKZuvwkFhvMjAy3mNh9JjR~a8IvQKoAgadSWayj0CbwLGmgVThJfdgoWJ3QfJml8pBU2b3NnPyUWbNPXNl5OWgfeHpQqlsovaG9INL0AYgpmdTBnyeDGMvB0AQJtYh6FNONXc8J1NawojFTe6LxCsqXo0~kdhwzvg9E5o1-mPQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5605148&ar=382624&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment and comparison of variations and mutations in genic SSRs with sequences homologous between <em>P. tomentosa</em> (01) and <em>P. trichocarpa</em> (02), based on a number of genic SSRs. Ten types of mutations and variations for genic SSR loci with their flanking regions were identified (See <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">Table 2</a> for details).</p></div></div><p>We determined whether the mutations in SSR repeat motifs were transitions, transversions, insertions, or deletions related to the <em>P. tomentosa</em>–<em>P. trichocarpa</em> comparison. An insertion of even a single base in the repeat motif, e.g. (AT)<sub>5</sub> T (AT)<sub>3,</sub> and a deletion within the repeat tract, e.g. (AGG)<sub>3</sub> AG_ (AGG)<sub>2</sub>, disrupted the continuity of the perfect repeat units. Transversions (38/63) were the most abundant mutations in the sequences of these two species, whereas insertions (4/63) were the least abundant (Table <a reveal-id="DSS031TB3" data-open="DSS031TB3" class="link link-reveal link-table xref-fig">3</a>). Transitions and/or transversions accounted for 86% of the total mutations in the perfect repeats, whereas insertions and/or deletions were found in only 14%. T/C transversions were the most common substitutions that disrupted repeat continuity (Table <a reveal-id="DSS031TB3" data-open="DSS031TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="5605163" scrollto-destination="5605163"></a>

  <div class="table-wrap">
    <div id="DSS031TB3" class="table-wrap-title">
      <div class="label -label">Table 3.</div>
      <div class="caption">
        <p>Number and type of mutations found in perfect microsatellite motifs in <em>P. tomentosa</em> and <em>P. trichocarpa</em></p>
      </div>
    </div>
    <div class="table-overflow">
      <table>
        <thead name="thead" align="">
          <tr>
            <th colspan="2">Transition (16)<hr /> </th>
            <th colspan="4">Transversion (38) </th>
            <th>Deletion (5) </th>
            <th>Insertion (4) </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq1.gif?Expires=1509238854&amp;Signature=GYxnpVuWBOEytDqeB3~Y17xJi4-rWEbnmoC5iM5fyPiUAGGm3vbHiAE~j9TkNx2jPX3SbB7qsoMp3y4DnEslz-KVJGYbn-pw7er6C4CpFVygYrHXay~DmseDDe-Dh9w11vW3t~cx8JkjGzh8o0c6hRNVo6BSWNH5y-xNvk4BTLXDdzJC5ZwbxO5KrG0M4sHs2DbtnAe0uYPNQiqptKw0mqQJ01pXwpBO6VhFMpctDbNNK4-kPJgb1eAEKPGrATVLelmKL-OV0Oc1f0w2xEqpRT5Dxe3slwHrEbPXKSWby1gZRL0V~7WkGcizdqCSZNqOPFuQX69jZV5NyuuKMuHjdg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq1.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq2.gif?Expires=1509238854&amp;Signature=YzFdD~Dsoz9yHHPqRFHgErESg2PaX8DNjOV2sdezrmKO2xUsSifkCrgPPKjEfQ7bdLe0d08FthpEh1BueU3kOIB7oE2P2OVuM8a6YWQRrXf8CG1OVT7MdgM-POiyN1CZpqgt~BM6iDdenCBQn3CAkbyzdRRQbawZqQwRV9TRpncOPt9qJU8klkIreo~l~y-iifUCQWPUmLTohHDsD5ZWDsrpTkjg8TxfZ1xTWdvdwK2nl4BL9H8g3igu4CQNc4vP8zPdzQCD1z-pJOpwtYc7AutzauUOfpc~ysIPc7J3-3qrs8sNkBpVNerPRLIMJN2xFquHvIAMgV3KkQuKlsGgBg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq2.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq3.gif?Expires=1509238854&amp;Signature=Q4PR8y-HMLhoGTywSWIhL0vWSVIj2Nr68Kfe6ajbdl7MbQz0hwI9qTDWS9OYtABfsJIiMNajqbv4sR9uCjeF5EYKBl8mwxrJg4Mb9UjSGf5K0DcvrC5oVUD13VzLT53qpM3fpuhhsvIlg3huWdAOCF0vaAT3MlKmta3NC7qvCX94SP8G8~iPy~Q9OqkpgISykZHyQuM1TEBPBeHDH3zkP0eazvOP0qzoOoz8pjf40fjuWzsq27Y7XlWlug-1xO0x6pZryU4x5mAZWQ-K~fHiqF17JKrA3LB-iUwgcKYQvp3pPFYmQL2ICVscMRwJ-2it7gGVXL9TBVSmzgINXkTCTA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq3.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq4.gif?Expires=1509238854&amp;Signature=SQ~sf4R70kVF3P-FsIoi59cbpYCLOFx7HKa66atRKTyORa~unes9YkyWlQkWWSZOabRWcc8WIX9gnOdIdFz~Z7BrRWqjBjUKKHuk6-b1fo9ZhaEHkMaNU0z59qvTGuFXOPHsZ1j0NS-UGjV1L1-cmNSMO1cNzAy7oy74SsMotuubTLqHYpKDscInTfZcl58cCSKOmDH-y6knRwo-LaOC1GzI8obSNx1~K7Yxc0ZwIV6XjboCOeDB0d92V7SSKyVjKJV2JDceh1UCTVh2kOgU8THL~egk4KpHLc42zs0fZkUJKH3PxR2iC3c0Q8XQDLREBTqAiOjj9i7rkVRQ3H46Sw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq4.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq5.gif?Expires=1509238854&amp;Signature=P6UxgDO2IYEeqzbgkMMbIFa9k1CPUj68euW59RzLcR5QkrCtkJ08H3mcYzJr9bEZhFo7ECW6RXwZeAjdzGBVUAMDAoSTUx89bDxr2w08wQBWiPzXFsrRUtyaaf3UZYZwaiFhybcXuO1qOUMVrzNE4Ez-FcR~O4XeEhT96Lhv66P4kKlCPyDybKbD5Lmn2lC9Fn108R4EBxswIWBZoahgB7OY0HmyKqdeaiS1Ed4I-bxOYlTjehiA3ujNyTrvI0t9-x1xIt5L7ps0EvR8CnHhAAWhUa26SCYm5QB93P10gg9MYXn5fxqlsmr~yufVN6n3r6TYbYIM~BfZlDX2TOoEww__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq5.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq6.gif?Expires=1509238854&amp;Signature=QzQR-uQECbiUu3BnfrjmxdzcsqYi4bNQzT5BaH9eGJ8TjOlifPVbrRo9rP8I72y1uac6ToX43SILTVVOXPrtC8hJR5ek1okMUyGU1Qi9aqoDNiJtqEEayQzpxq4ILtDsIgwqE2Kios14ygg5e5SByMsjeU9XoO6r3xjw1kFvnBoiPBapvE22--BrDwqeDbZ7LC7boqacIOVDfgzFLPiKy206H6nMqq6vrSwcVan5hjzAQxZkNK1WRc5clDP2GqkUt17tI3UzMbFF~uliKgXZL-77nMHnY8n2uYGvERMsvEbifEBiflwJ4UTSRX8WCWUXNok3fxr1mU2Z3Mjc6sCDYQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq6.gif" />
              </span> </td>
            <td>A (1), T (2), C (1), G (1) </td>
            <td>A (2), T (2), C (0), G (0) </td>
          </tr>
          <tr>
            <td>13 (24%) </td>
            <td>3 (6%) </td>
            <td>11 (20%) </td>
            <td>5 (9%) </td>
            <td>17 (31%) </td>
            <td>5 (9%) </td>
            <td> </td>
            <td> </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div content-id="DSS031TB3" class="table-modal">
      <table>
        <thead name="thead" align="">
          <tr>
            <th colspan="2">Transition (16)<hr /> </th>
            <th colspan="4">Transversion (38) </th>
            <th>Deletion (5) </th>
            <th>Insertion (4) </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq1.gif?Expires=1509238854&amp;Signature=GYxnpVuWBOEytDqeB3~Y17xJi4-rWEbnmoC5iM5fyPiUAGGm3vbHiAE~j9TkNx2jPX3SbB7qsoMp3y4DnEslz-KVJGYbn-pw7er6C4CpFVygYrHXay~DmseDDe-Dh9w11vW3t~cx8JkjGzh8o0c6hRNVo6BSWNH5y-xNvk4BTLXDdzJC5ZwbxO5KrG0M4sHs2DbtnAe0uYPNQiqptKw0mqQJ01pXwpBO6VhFMpctDbNNK4-kPJgb1eAEKPGrATVLelmKL-OV0Oc1f0w2xEqpRT5Dxe3slwHrEbPXKSWby1gZRL0V~7WkGcizdqCSZNqOPFuQX69jZV5NyuuKMuHjdg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq1.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq2.gif?Expires=1509238854&amp;Signature=YzFdD~Dsoz9yHHPqRFHgErESg2PaX8DNjOV2sdezrmKO2xUsSifkCrgPPKjEfQ7bdLe0d08FthpEh1BueU3kOIB7oE2P2OVuM8a6YWQRrXf8CG1OVT7MdgM-POiyN1CZpqgt~BM6iDdenCBQn3CAkbyzdRRQbawZqQwRV9TRpncOPt9qJU8klkIreo~l~y-iifUCQWPUmLTohHDsD5ZWDsrpTkjg8TxfZ1xTWdvdwK2nl4BL9H8g3igu4CQNc4vP8zPdzQCD1z-pJOpwtYc7AutzauUOfpc~ysIPc7J3-3qrs8sNkBpVNerPRLIMJN2xFquHvIAMgV3KkQuKlsGgBg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq2.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq3.gif?Expires=1509238854&amp;Signature=Q4PR8y-HMLhoGTywSWIhL0vWSVIj2Nr68Kfe6ajbdl7MbQz0hwI9qTDWS9OYtABfsJIiMNajqbv4sR9uCjeF5EYKBl8mwxrJg4Mb9UjSGf5K0DcvrC5oVUD13VzLT53qpM3fpuhhsvIlg3huWdAOCF0vaAT3MlKmta3NC7qvCX94SP8G8~iPy~Q9OqkpgISykZHyQuM1TEBPBeHDH3zkP0eazvOP0qzoOoz8pjf40fjuWzsq27Y7XlWlug-1xO0x6pZryU4x5mAZWQ-K~fHiqF17JKrA3LB-iUwgcKYQvp3pPFYmQL2ICVscMRwJ-2it7gGVXL9TBVSmzgINXkTCTA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq3.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq4.gif?Expires=1509238854&amp;Signature=SQ~sf4R70kVF3P-FsIoi59cbpYCLOFx7HKa66atRKTyORa~unes9YkyWlQkWWSZOabRWcc8WIX9gnOdIdFz~Z7BrRWqjBjUKKHuk6-b1fo9ZhaEHkMaNU0z59qvTGuFXOPHsZ1j0NS-UGjV1L1-cmNSMO1cNzAy7oy74SsMotuubTLqHYpKDscInTfZcl58cCSKOmDH-y6knRwo-LaOC1GzI8obSNx1~K7Yxc0ZwIV6XjboCOeDB0d92V7SSKyVjKJV2JDceh1UCTVh2kOgU8THL~egk4KpHLc42zs0fZkUJKH3PxR2iC3c0Q8XQDLREBTqAiOjj9i7rkVRQ3H46Sw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq4.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq5.gif?Expires=1509238854&amp;Signature=P6UxgDO2IYEeqzbgkMMbIFa9k1CPUj68euW59RzLcR5QkrCtkJ08H3mcYzJr9bEZhFo7ECW6RXwZeAjdzGBVUAMDAoSTUx89bDxr2w08wQBWiPzXFsrRUtyaaf3UZYZwaiFhybcXuO1qOUMVrzNE4Ez-FcR~O4XeEhT96Lhv66P4kKlCPyDybKbD5Lmn2lC9Fn108R4EBxswIWBZoahgB7OY0HmyKqdeaiS1Ed4I-bxOYlTjehiA3ujNyTrvI0t9-x1xIt5L7ps0EvR8CnHhAAWhUa26SCYm5QB93P10gg9MYXn5fxqlsmr~yufVN6n3r6TYbYIM~BfZlDX2TOoEww__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq5.gif" />
              </span> </td>
            <td>
              <span class="inline-formula no-formula-id">
                <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/m_dss031ileq6.gif?Expires=1509238854&amp;Signature=QzQR-uQECbiUu3BnfrjmxdzcsqYi4bNQzT5BaH9eGJ8TjOlifPVbrRo9rP8I72y1uac6ToX43SILTVVOXPrtC8hJR5ek1okMUyGU1Qi9aqoDNiJtqEEayQzpxq4ILtDsIgwqE2Kios14ygg5e5SByMsjeU9XoO6r3xjw1kFvnBoiPBapvE22--BrDwqeDbZ7LC7boqacIOVDfgzFLPiKy206H6nMqq6vrSwcVan5hjzAQxZkNK1WRc5clDP2GqkUt17tI3UzMbFF~uliKgXZL-77nMHnY8n2uYGvERMsvEbifEBiflwJ4UTSRX8WCWUXNok3fxr1mU2Z3Mjc6sCDYQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss031ileq6.gif" />
              </span> </td>
            <td>A (1), T (2), C (1), G (1) </td>
            <td>A (2), T (2), C (0), G (0) </td>
          </tr>
          <tr>
            <td>13 (24%) </td>
            <td>3 (6%) </td>
            <td>11 (20%) </td>
            <td>5 (9%) </td>
            <td>17 (31%) </td>
            <td>5 (9%) </td>
            <td> </td>
            <td> </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div class="graphic-wrap">
      <a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5605163" target="_blank">View Large</a>
    </div>
  </div>
                            <h3 scrollto-destination=5605168 id="5605168" class="section-title" data-legacy-id=s3e>3.5. Inheritance test of genic SSR markers</h3>
<p>To further validate the usefulness of these genic SSR markers in genetic studies, we genotyped a randomly selected subset of 30 of the markers in a linkage mapping population of 1000 progeny from a controlled hybridization and tested for Mendelian segregation. Of the 30 genic SSRs, 5 loci (17%) were not used for analysis due to the presence of null alleles or unexpected lengths in the female parent. The remaining 25 loci segregated in the population (Table <a reveal-id="DSS031TB4" data-open="DSS031TB4" class="link link-reveal link-table xref-fig">4</a>). Of the segregating loci, 2 were heterozygous in the female parent only, 4 were heterozygous in the male parent only, 17 were heterozygous in both parents, and 2 (SSR211 and SSR249) were homozygous in both parents, and thus resulted in offspring identical to the parents and with the expected heterozygote genotype (Table <a reveal-id="DSS031TB4" data-open="DSS031TB4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="5605170" scrollto-destination="5605170"></a>
<div class="table-wrap"><div id="DSS031TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Mendelian inheritance patterns of 30 genic SSR markers based on segregation analyses of a cross between a female YX01 clone (<em>P. alba</em> × <em>P. glandulosa</em>) and a male LM 50 clone (<em>P. tomentosa</em>)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Locus </th><th>The allele genotype of P1(♀) </th><th>The allele genotype of P2(♂) </th><th>The segregation genotypes of F<sub>1</sub> progeny </th><th>The expected segregation ratio </th><th><em>P-</em>value </th></tr></thead><tbody><tr><td>SSR2 </td><td>331/341 (+) </td><td>329/335 (+) </td><td>329/331, 329/341, 331/335, 335/341 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR7 </td><td>156/162 (+) </td><td>159/165 (+) </td><td>156/159, 156/165, 159/162, 162/165 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR9 </td><td>250/256 (+) </td><td>250/256 (+) </td><td>250, 250/256, 256 </td><td>1:2:1 </td><td>NS </td></tr><tr><td>SSR13 </td><td>155/164 (+) </td><td>155/158 (+) </td><td>155, 155/158, 155/164, 158/164 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR16 </td><td>341 (−) Unexpected size </td><td>363 (−) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR21 </td><td>167/179(+) </td><td>169/175(+) </td><td>167/169, 167/175, 169/179, 175/179 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR33 </td><td>92 (−) </td><td>92/98 (+) </td><td>92, 92/98 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR44 </td><td>151/156 (+) </td><td>156 (−) </td><td>151/156, 156 </td><td>1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR45 </td><td>188/192 (+) </td><td>192/196 (+) </td><td>188/192, 188/196, 192/196, 192 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR47 </td><td>186/195 (+) </td><td>192/195 (+) </td><td>186/192, 186/195, 192/195, 195 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR50 </td><td>233/240 (+) </td><td>233/240 (+) </td><td>233, 233/240, 240 </td><td>1:2:1 </td><td>NS </td></tr><tr><td>SSR52 </td><td>145 (−) </td><td>145/148 (+) </td><td>145, 145/148 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR56 </td><td>228 (−) </td><td>221/228 (+) </td><td>221/228, 228 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR61 </td><td>No amplification </td><td>237 (−) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR63 </td><td>202 (−) </td><td>202/206 (+) </td><td>202, 202/206 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR71 </td><td>204/210 (+) </td><td>204/213 (+) </td><td>204, 204/210, 204/213, 210/213 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR77 </td><td>No amplification </td><td>171/175 (+) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR107 </td><td>169/172 (+) </td><td>169 (−) </td><td>169, 169/172 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR108 </td><td>177/183 (+) </td><td>171/179 (+) </td><td>171/177, 177/179, 171/183, 179/183 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR114 </td><td>102/110 (+) </td><td>104/108(+) </td><td>102/104, 102/108, 104/110, 108/110 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR118 </td><td>191/194 (+) </td><td>191/197 (+) </td><td>191/197, 191/194, 194/197, 191 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR124 </td><td>192/196 (+) </td><td>188/192 (+) </td><td>188/192, 188/196, 192/196, 192 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR142 </td><td>227/231 (+) </td><td>229/237 (+) </td><td>227/229, 227/237, 229/231, 231/237 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR150 </td><td>No amplification </td><td>240/244 (+) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR188 </td><td>207/211 (+) </td><td>207/213 (+) </td><td>207, 207/213, 207/213, 211/213 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR197 </td><td>430 (−) Unexpected size </td><td>417/419 (+) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR211 </td><td>157 (−) </td><td>160 (−) </td><td>157/160 </td><td>/ </td><td>/ </td></tr><tr><td>SSR233 </td><td>291/295 (+) </td><td>289/299 (+) </td><td>289/291, 289/295, 291/299, 295/299 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR249 </td><td>179 (−) </td><td>179 (−) </td><td>179 </td><td>/ </td><td>/ </td></tr><tr><td>SSR254 </td><td>283/289 (+) </td><td>283/289 (+) </td><td>283, 283/289, 289 </td><td>1:2:1 </td><td>NS </td></tr></tbody></table></div><div content-id="DSS031TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Locus </th><th>The allele genotype of P1(♀) </th><th>The allele genotype of P2(♂) </th><th>The segregation genotypes of F<sub>1</sub> progeny </th><th>The expected segregation ratio </th><th><em>P-</em>value </th></tr></thead><tbody><tr><td>SSR2 </td><td>331/341 (+) </td><td>329/335 (+) </td><td>329/331, 329/341, 331/335, 335/341 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR7 </td><td>156/162 (+) </td><td>159/165 (+) </td><td>156/159, 156/165, 159/162, 162/165 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR9 </td><td>250/256 (+) </td><td>250/256 (+) </td><td>250, 250/256, 256 </td><td>1:2:1 </td><td>NS </td></tr><tr><td>SSR13 </td><td>155/164 (+) </td><td>155/158 (+) </td><td>155, 155/158, 155/164, 158/164 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR16 </td><td>341 (−) Unexpected size </td><td>363 (−) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR21 </td><td>167/179(+) </td><td>169/175(+) </td><td>167/169, 167/175, 169/179, 175/179 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR33 </td><td>92 (−) </td><td>92/98 (+) </td><td>92, 92/98 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR44 </td><td>151/156 (+) </td><td>156 (−) </td><td>151/156, 156 </td><td>1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR45 </td><td>188/192 (+) </td><td>192/196 (+) </td><td>188/192, 188/196, 192/196, 192 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR47 </td><td>186/195 (+) </td><td>192/195 (+) </td><td>186/192, 186/195, 192/195, 195 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR50 </td><td>233/240 (+) </td><td>233/240 (+) </td><td>233, 233/240, 240 </td><td>1:2:1 </td><td>NS </td></tr><tr><td>SSR52 </td><td>145 (−) </td><td>145/148 (+) </td><td>145, 145/148 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR56 </td><td>228 (−) </td><td>221/228 (+) </td><td>221/228, 228 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR61 </td><td>No amplification </td><td>237 (−) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR63 </td><td>202 (−) </td><td>202/206 (+) </td><td>202, 202/206 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR71 </td><td>204/210 (+) </td><td>204/213 (+) </td><td>204, 204/210, 204/213, 210/213 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR77 </td><td>No amplification </td><td>171/175 (+) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR107 </td><td>169/172 (+) </td><td>169 (−) </td><td>169, 169/172 </td><td>1:1 </td><td>NS </td></tr><tr><td>SSR108 </td><td>177/183 (+) </td><td>171/179 (+) </td><td>171/177, 177/179, 171/183, 179/183 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR114 </td><td>102/110 (+) </td><td>104/108(+) </td><td>102/104, 102/108, 104/110, 108/110 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR118 </td><td>191/194 (+) </td><td>191/197 (+) </td><td>191/197, 191/194, 194/197, 191 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR124 </td><td>192/196 (+) </td><td>188/192 (+) </td><td>188/192, 188/196, 192/196, 192 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR142 </td><td>227/231 (+) </td><td>229/237 (+) </td><td>227/229, 227/237, 229/231, 231/237 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR150 </td><td>No amplification </td><td>240/244 (+) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR188 </td><td>207/211 (+) </td><td>207/213 (+) </td><td>207, 207/213, 207/213, 211/213 </td><td>1:1:1:1 </td><td><em>P</em> &lt; 0.01 </td></tr><tr><td>SSR197 </td><td>430 (−) Unexpected size </td><td>417/419 (+) </td><td>/ </td><td>/ </td><td>/ </td></tr><tr><td>SSR211 </td><td>157 (−) </td><td>160 (−) </td><td>157/160 </td><td>/ </td><td>/ </td></tr><tr><td>SSR233 </td><td>291/295 (+) </td><td>289/299 (+) </td><td>289/291, 289/295, 291/299, 295/299 </td><td>1:1:1:1 </td><td>NS </td></tr><tr><td>SSR249 </td><td>179 (−) </td><td>179 (−) </td><td>179 </td><td>/ </td><td>/ </td></tr><tr><td>SSR254 </td><td>283/289 (+) </td><td>283/289 (+) </td><td>283, 283/289, 289 </td><td>1:2:1 </td><td>NS </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a> for further details of these 30 genic SSR markers.</p></div></div><div content-id=""><div class="fn"><p>‘P1’ represents the female clone ‘YX01’ (<em>P. alba</em> × <em>P. glandulosa</em>), ‘P2’ represents the male clone ‘LM 50’ (<em>P. tomentosa</em>), ‘+’ represents heterozygote, and ‘–’ represents homozygote; the <em>χ</em><sup>2</sup> significance level was <em>P</em> &lt; 0.01, Ns, not significant; /, Not applied.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5605170" target="_blank">View Large</a></div></div><p>A chi-squared test was used to compare the segregation ratios of the 23 informative markers in these 1000 offspring. Eighteen were in accordance with Mendelian expectations (<em>P</em> ≥ 0.01), with a segregation ratio close to 1:2:1 for 3 SSR loci, 1:1 for 5 loci, and 1:1:1:1 for the remaining 10 markers (Table <a reveal-id="DSS031TB4" data-open="DSS031TB4" class="link link-reveal link-table xref-fig">4</a>). Thus, these novel genic SSR markers may represent a useful resource for linkage mapping in <em>P. tomentosa</em>.</p>                            <h3 scrollto-destination=5605172 id="5605172" class="section-title" data-legacy-id=s3f>3.6. Genic SSR polymorphisms associated with wood property traits</h3>
<p>Finally, to directly show that these markers could also prove useful for association mapping, we conducted a trial association mapping study using a subset of the genic SSRs to associate with traits affecting wood properties in <em>Populus</em>. For a random selection of 40 genic SSRs, single-marker association tests (240; 40 SSRs × 6 traits) were conducted using MLM. Twenty associations were found to be significant at a threshold of <em>P</em> &lt; 0.05 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Multiple test corrections using the FDR method reduced this number to 12 at a significance threshold of Q &lt; 0.10 (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). These loci accounted for the phenotypic variance, with individual effects ranging from 4.0 to 8.9% (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). Of these, α-cellulose and fibre length had three significant associations each, and holocellulose, MFA, and lignin had two significant associations each. However, no association with fibre width was detected (Q &lt; 0.10, Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). The 12 associations represent 9 SSR loci from different regions of 9 candidate genes (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). For example, SSR205 was located in the coding region (<em>PtoC4H1</em> exon 1) that was associated exclusively with lignin (<em>R</em><sup>2</sup> = 8.9%, Q = 0.0211) (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). For the holocellulose trait, SSR47 and SSR163, located in the non-coding regions of two candidate genes (<em>PtoKorB</em> and <em>PtoCslA4</em>), accounted for 4.3–8.7% of the phenotypic variance, and the SSR47 marker was similarly associated with α-cellulose content (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="5605176" scrollto-destination="5605176"></a>
<div class="table-wrap"><div id="DSS031TB5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Significant SSR marker–trait pair associations in the natural <em>P. tomentosa</em> population (<em>n</em> = 460), after correction for multiple testing [FDR (Q) ≤ 0.10]</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Gene symbol </th><th>Locus </th><th><em>P</em> </th><th><em>R</em><sup>2</sup> (%) </th><th>Q </th></tr></thead><tbody><tr><td rowspan="2">Lignin </td><td><em>PtoPAL 2</em> Intron1 </td><td>SSR198 </td><td>0.0021 </td><td>4.6 </td><td>0.0250 </td></tr><tr><td><em>PtoC4H1</em> Exon 1 </td><td>SSR205 </td><td>0.0016 </td><td>8.9 </td><td>0.0211 </td></tr><tr><td rowspan="2">Holocellulose </td><td><em>PtoKorB</em> Intron 3 </td><td>SSR47 </td><td>0.0031 </td><td>7.0 </td><td>0.0371 </td></tr><tr><td><em>PtoCslA4</em> Promoter </td><td>SSR163 </td><td>0.0106 </td><td>6.4 </td><td>0.0475 </td></tr><tr><td rowspan="3">α-Cellulose </td><td><em>PtoKorB</em> Intron 3 </td><td>SSR47 </td><td>0.0095 </td><td>4.0 </td><td>0.0475 </td></tr><tr><td><em>PtoWuA</em> Promoter </td><td>SSR53 </td><td>0.0200 </td><td>4.5 </td><td>0.0702 </td></tr><tr><td><em>PtoSuSy6</em> Promoter </td><td>SSR71 </td><td>0.0001 </td><td>7.4 </td><td>0.0172 </td></tr><tr><td rowspan="2">MFA </td><td><em>PtoPAL2</em> Intron 1 </td><td>SSR198 </td><td>0.0205 </td><td>8.0 </td><td>0.0750 </td></tr><tr><td><em>PtoCCR2</em> 5′UTR </td><td>SSR224 </td><td>4.33E–04 </td><td>7.5 </td><td>0.0105 </td></tr><tr><td rowspan="3">Fibre length </td><td><em>PtoExp10</em> Promoter </td><td>SSR79 </td><td>0.0022 </td><td>5.3 </td><td>0.0278 </td></tr><tr><td><em>PtoCslA4</em> Promoter </td><td>SSR163 </td><td>0.0175 </td><td>5.5 </td><td>0.0669 </td></tr><tr><td><em>PtoCslD9</em> Exon 4 </td><td>SSR187 </td><td>0.0027 </td><td>6.4 </td><td>0.0346 </td></tr></tbody></table></div><div content-id="DSS031TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Gene symbol </th><th>Locus </th><th><em>P</em> </th><th><em>R</em><sup>2</sup> (%) </th><th>Q </th></tr></thead><tbody><tr><td rowspan="2">Lignin </td><td><em>PtoPAL 2</em> Intron1 </td><td>SSR198 </td><td>0.0021 </td><td>4.6 </td><td>0.0250 </td></tr><tr><td><em>PtoC4H1</em> Exon 1 </td><td>SSR205 </td><td>0.0016 </td><td>8.9 </td><td>0.0211 </td></tr><tr><td rowspan="2">Holocellulose </td><td><em>PtoKorB</em> Intron 3 </td><td>SSR47 </td><td>0.0031 </td><td>7.0 </td><td>0.0371 </td></tr><tr><td><em>PtoCslA4</em> Promoter </td><td>SSR163 </td><td>0.0106 </td><td>6.4 </td><td>0.0475 </td></tr><tr><td rowspan="3">α-Cellulose </td><td><em>PtoKorB</em> Intron 3 </td><td>SSR47 </td><td>0.0095 </td><td>4.0 </td><td>0.0475 </td></tr><tr><td><em>PtoWuA</em> Promoter </td><td>SSR53 </td><td>0.0200 </td><td>4.5 </td><td>0.0702 </td></tr><tr><td><em>PtoSuSy6</em> Promoter </td><td>SSR71 </td><td>0.0001 </td><td>7.4 </td><td>0.0172 </td></tr><tr><td rowspan="2">MFA </td><td><em>PtoPAL2</em> Intron 1 </td><td>SSR198 </td><td>0.0205 </td><td>8.0 </td><td>0.0750 </td></tr><tr><td><em>PtoCCR2</em> 5′UTR </td><td>SSR224 </td><td>4.33E–04 </td><td>7.5 </td><td>0.0105 </td></tr><tr><td rowspan="3">Fibre length </td><td><em>PtoExp10</em> Promoter </td><td>SSR79 </td><td>0.0022 </td><td>5.3 </td><td>0.0278 </td></tr><tr><td><em>PtoCslA4</em> Promoter </td><td>SSR163 </td><td>0.0175 </td><td>5.5 </td><td>0.0669 </td></tr><tr><td><em>PtoCslD9</em> Exon 4 </td><td>SSR187 </td><td>0.0027 </td><td>6.4 </td><td>0.0346 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>P</em> = the significant level for association (the significance is <em>P</em> ≤ 0.05); <em>R</em><sup>2</sup> = percentage of the phenotypic variance explained; Lignin = lignin content; Holocellulose = holocellulose content; and α-cellulose = α-cellulose content.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5605176" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=5605177 id="5605177" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5605178 id="5605178" class="section-title" data-legacy-id=s4a>4.1. Development and characterization of genic SSR markers</h3>
<p>Here, we used a candidate gene approach to identify a set of SSR markers in <em>P. tomentosa</em> genes for wood properties, showing that this approach can provide useful genomics resources for linkage or association mapping and eventually for marker-assisted breeding to improve wood quality in this important timber crop. We successfully mined for genic SSRs in the sequences of 150 candidate genes associated with wood property traits. Our analysis of these SSR markers from <em>P. tomentosa</em> demonstrated that this approach may be useful for characterization of other pathways in <em>P. tomentosa</em> and for development of molecular markers for related model species with similar genomic resources. In our study, the frequency of genic SSRs was ∼1 SSR/1.3 kb. Higher or lower frequencies of genic SSRs have been reported elsewhere.<sup><a reveal-id="DSS031C11" data-open="DSS031C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS031C26" data-open="DSS031C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS031C27" data-open="DSS031C27" class="link link-ref link-reveal xref-bibr">27</a></sup> However, these frequency variations may be the result of the application of different SSR search criteria, methods, or the abundances of the sources of DNA sequences searched.<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>Dinucleotide motifs were the most abundant genic SSR markers (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5A</a>). This result is in contrast to previous findings identifying trinucleotide motifs as the most frequent genic repeats in most plant species,<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS031C28" data-open="DSS031C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSS031C29" data-open="DSS031C29" class="link link-ref link-reveal xref-bibr">29</a></sup> which may be attributable to the sources of the DNA sequences used (i.e. EST, cDNA, or gene sequences). In general, previous genic SSR makers were located in EST databases, and information about promoters and introns was not considered. Genic microsatellites are located mainly in promoters, introns, and UTRs of sequenced genes and are found at a lower frequency in conserved exons.<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a></sup> This agrees with the distribution pattern we report here (Fig. <a reveal-id="DSS031F2" data-open="DSS031F2" class="link link-reveal link-table xref-fig">2</a>A), suggesting that genic microsatellites may have a role in regulation of gene expression.<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS031C11" data-open="DSS031C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS031C28" data-open="DSS031C28" class="link link-ref link-reveal xref-bibr">28</a>–<a reveal-id="DSS031C30" data-open="DSS031C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Furthermore, polymorphism tests of 15 wild <em>P. tomentosa</em> ecotypes indicated that exonic SSRs contained less allelic variability than did non-coding SSRs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7B</a>), which reflects the higher selection pressure on the coding portion of the genome.<sup><a reveal-id="DSS031C31" data-open="DSS031C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSS031C32" data-open="DSS031C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>Our analysis of gene-derived SSRs found that SSRs with AT/TA motifs (32.6%) were the most frequent. This is similar to the cases of <em>P. trichocarpa</em> and <em>Arachis hypogaea</em>.<sup><a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSS031C33" data-open="DSS031C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Previous studies have indicated that (AT)<em>n</em> is the most common dinucleotide motif in plant genomes.<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS031C33" data-open="DSS031C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Thus, the genic SSR pattern identified in this study is likely to be a good reflection of genome-wide SSR frequencies. In trinucleotide SSRs, the polymorphism rate of the ATT/TAA SSRs is three times higher than that of AAT/TTA SSRs. This distribution suggests that AT-rich SSR loci may have a relatively high variability in <em>P. tomentosa</em> and also confirms the finding that AT-rich repeats (those repeats containing two or more A and/or T nucleotides per motif) are more common in non-coding regions.<sup><a reveal-id="DSS031C9" data-open="DSS031C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS031C17" data-open="DSS031C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSS031C34" data-open="DSS031C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p><p>In this study, 91.8% of the 292 optimal genic SSR primers produced at least 2 clean amplified bands, possibly due to the naturally occurring excess of heterozygotes in the <em>P. tomentosa</em> genome.<sup><a reveal-id="DSS031C12" data-open="DSS031C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Allelic diversity estimated for these polymorphic markers was an average of 3.6 alleles per locus, (range 2–7 alleles), and the value is lower than the <em>N</em><sub>A</sub> (4.3) in a population of 460 <em>P. tomentosa</em> and some other related species, such as <em>Populus nigra</em>, <em>P. trichocarpa</em>, <em>Populus tremuloides</em>, and <em>Populus euphratica</em>.<sup><a reveal-id="DSS031C6" data-open="DSS031C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS031C35" data-open="DSS031C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSS031C36" data-open="DSS031C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The level of allelic diversity reported in this study may be due to the limited sample size and/or the relatively conserved character of genic SSR markers. For generation of genetic maps, genic SSRs can determine the relative positions of transferable markers and directly compare candidate gene-containing SSRs and quantitative trait locus (QTL) locations across a broad variety of genetic backgrounds.<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSS031C29" data-open="DSS031C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSS031C32" data-open="DSS031C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The result of inheritance tests for 30 genic SSR markers suggests that many genic SSRs can be used for linkage mapping in <em>P. tomentosa</em>, and they are also useful for QTL and marker-aided selection of important traits. In addition, segregation distortion is increasingly recognized as a potentially powerful evolutionary force that may be beneficial for QTL mapping.<sup><a reveal-id="DSS031C37" data-open="DSS031C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSS031C38" data-open="DSS031C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The actual causes of the observed segregation distortions for markers are genes subjected to gametic or zygotic selection.<sup><a reveal-id="DSS031C37" data-open="DSS031C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Studies have suggested that epistasis contributes to segregation distortion, and segregation distortion may also be important for the evolution of many fundamental aspects of sexual reproduction.<sup><a reveal-id="DSS031C37  DSS031C39" data-open="DSS031C37  DSS031C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup></p><p>The presence of SSRs in transcribed regions can result in changes in function, transcription, or translation. For example, SSRs in coding regions that result in amino acid changes can cause either loss or gain of function, SSRs in the 5′UTR can affect gene transcription or translation, SSRs in the 3′UTR can be responsible for gene silencing or transcription slippage, and SSRs in introns might act as transcriptional enhancers of gene expression.<sup><a reveal-id="DSS031C8" data-open="DSS031C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS031C32" data-open="DSS031C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS031C40" data-open="DSS031C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Previous study that phenylalanine ammonia-lyase (PAL) transcripts have been localized to develop xylem cells in aspen (<em>P. tremuloides</em>) stem, was consistent with its involvement in lignin biosynthesis.<sup><a reveal-id="DSS031C41" data-open="DSS031C41" class="link link-ref link-reveal xref-bibr">41</a></sup> This report supported the identification of a single-marker non-coding association in <em>PtoPAL</em>2 that explained 4.6% of the phenotypic variation in lignin content (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). Cinnamate 4-hydroxylase (C4H1) is proposed to be associated with G lignin deposition,<sup><a reveal-id="DSS031C42" data-open="DSS031C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSS031C43" data-open="DSS031C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and a marker with significant association was located in the coding region (<em>PtoC4H1</em> exon 1) that was associated exclusively with lignin (<em>R</em><sup>2</sup> = 8.9%, Q = 0.0211) (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>). Physiological studies of <em>C4H</em> genes describe unique functions for the isoforms within the lignin biosynthetic pathway.<sup><a reveal-id="DSS031C43" data-open="DSS031C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Similarly, other significant associations located in different candidate genes, such as sucrose synthase (<em>SUSY</em>), Cellulase (<em>KOR</em>), and Cinnamoyl-CoA reductase (<em>CCR</em>) (Table <a reveal-id="DSS031TB5" data-open="DSS031TB5" class="link link-reveal link-table xref-fig">5</a>), were also supported by studies finding that they were involved in lignocellulosic cell wall development.<sup><a reveal-id="DSS031C3" data-open="DSS031C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS031C14" data-open="DSS031C14" class="link link-ref link-reveal xref-bibr">14</a>–<a reveal-id="DSS031C16" data-open="DSS031C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSS031C43" data-open="DSS031C43" class="link link-ref link-reveal xref-bibr">43</a>–<a reveal-id="DSS031C45" data-open="DSS031C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Association analysis in this study suggests that genic SSRs have considerable power in candidate gene-based association studies to identify allelic variation in genes associated with important wood quality traits,<sup><a reveal-id="DSS031C20" data-open="DSS031C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and these markers have the potential to be useful in genetic improvement of lignin and cellulose biosynthesis in poplar.</p>                            <h3 scrollto-destination=5605194 id="5605194" class="section-title" data-legacy-id=s4b>4.2. SSR variation among related species within the genus Populus</h3>
<p>The genus <em>Populus</em> contains six subgenera: <em>Abaso</em>, <em>Leuce</em>, <em>Leucoides</em>, <em>Aigeiros</em>, <em>Turanga</em>, and <em>Tacamahaca</em>, and <em>Turanga</em> is the most distant from <em>Leuce</em>.<sup><a reveal-id="DSS031C10" data-open="DSS031C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS031C24" data-open="DSS031C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSS031C46" data-open="DSS031C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS031C47" data-open="DSS031C47" class="link link-ref link-reveal xref-bibr">47</a></sup> We found that genic SSR markers in <em>P. tomentosa</em> have relatively high amplification rates among closely related taxa, including species of the <em>Leuce</em>, <em>Tacamahaca</em>, <em>Aigeiros</em>, and <em>Leucoides</em> sections, but a lower amplification rate in <em>Turanga</em> (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>). This successful amplification rate in different sections was positively correlated with mean <em>N<sub>A</sub></em> and PIC values (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>) that are due mainly to the phylogenetic divergence from <em>P. tomentosa</em> and is also related to the sampling species and their genetic backgrounds in each section. Furthermore, the amplification success rates were higher for members of the other subgenera, with the exception of <em>Turanga</em> (Table <a reveal-id="DSS031TB1" data-open="DSS031TB1" class="link link-reveal link-table xref-fig">1</a>), suggesting that the amplification frequency varies in tandem with evolutionary distance, and this may be because the flanking regions of the microsatellite, where the PCR primers bind to the DNA, are more similar in phylogenetically close species than phylogenetically distant species.<sup><a reveal-id="DSS031C10" data-open="DSS031C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS031C17" data-open="DSS031C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSS031C32" data-open="DSS031C32" class="link link-ref link-reveal xref-bibr">32</a></sup> It should, therefore, be possible to predict the utility of these primers according to the genetic distance from <em>P. tomentosa</em>.</p><p>Six distinct groups were identified within 27 samples, based on 30 genic SSR markers (Fig. <a reveal-id="DSS031F3" data-open="DSS031F3" class="link link-reveal link-table xref-fig">3</a>). The dendrogram grouping showed the expected segregation of the different sections within the genus <em>Populus</em>, which agreed with the subgenus botanical classification level of <em>Salicaceae</em>,<sup><a reveal-id="DSS031C24" data-open="DSS031C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS031C25" data-open="DSS031C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSS031C46" data-open="DSS031C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS031C47" data-open="DSS031C47" class="link link-ref link-reveal xref-bibr">47</a></sup> although a few species were not identified based on their putative relationships or as reported in other diversity studies.<sup><a reveal-id="DSS031C46" data-open="DSS031C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSS031C47" data-open="DSS031C47" class="link link-ref link-reveal xref-bibr">47</a></sup> For example, <em>Populus bolleana</em> is a variety of <em>Populus</em><em>alba</em>; however, these species did not group together in the closest branches, despite their shared ancestry (Fig. <a reveal-id="DSS031F3" data-open="DSS031F3" class="link link-reveal link-table xref-fig">3</a>). Overall, this analysis roughly supports the botanical classification of the germplasm surveyed and hints at the usefulness of the genic SSR markers for genetic diversity studies and other <em>Populus</em> genotyping applications.</p><p>In a comparison of genic SSR variations using homologous <em>P. tomentosa</em> and <em>P. trichocarpa</em> sequences (Table <a reveal-id="DSS031TB2" data-open="DSS031TB2" class="link link-reveal link-table xref-fig">2</a>), we identified both repeat motif variations and mutations in the flanking sequences in most markers, and this shows that microsatellite mutation patterns are often complex in cross-species amplification. However, our findings indicate that the effects of mutations accumulated over evolutionary time can readily be studied. Point mutations disrupted the repeat pattern, and in addition to a new class of repeat motifs, size variations at microsatellite loci caused by indels (insertions or deletions) in flanking sequences have also been reported.<sup><a reveal-id="DSS031C32" data-open="DSS031C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS031C40" data-open="DSS031C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSS031C48" data-open="DSS031C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS031C49" data-open="DSS031C49" class="link link-ref link-reveal xref-bibr">49</a></sup> We found the highest proportion of SSRs with mutations in flanking sequences in the 3′UTR (60.0%) and the lowest in the 5′UTR (25.8%). This suggests that variations and mutations in microsatellites may be influenced by the nature and functional composition of the flanking sequences.<sup><a reveal-id="DSS031C48" data-open="DSS031C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSS031C49" data-open="DSS031C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Transitions and/or transversions accounted for 86% of the total mutations of perfect repeat motifs in the two species (Table <a reveal-id="DSS031TB3" data-open="DSS031TB3" class="link link-reveal link-table xref-fig">3</a>), indicating that base substitution of microsatellites is more common than length variation of homologous loci between closely related species.<sup><a reveal-id="DSS031C49" data-open="DSS031C49" class="link link-ref link-reveal xref-bibr">49</a></sup> It also suggests that base substitution data are vital because genotyping using SSR alleles among related species is often erroneous, if based only on microsatellites of identical allele lengths obtained in electrophoresis. <em>In silico</em> identification of genic SSR variations in <em>Populus</em> is particularly useful for evolutionary studies and for increasing our understanding of the origin, mutational processes, and structure of microsatellites. Further reciprocal studies of microsatellite allelic variation in different species would shed additional light on directional evolution and genetic toxicology.<sup><a reveal-id="DSS031C40" data-open="DSS031C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSS031C48" data-open="DSS031C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p><p>Overall, the reliability of comparison of microsatellite variation data among related species is higher than cross-species amplification success rates or polymorphisms. Comparison of microsatellite variation in these two species suggests that use of cross-species SSR primers to investigate functionally important allelic polymorphisms related to the traits of interest may be inefficient. Therefore, the direct development of species-specific primers is necessary for association mapping, due to the variation in repeat number only at each SSR locus (conserved flanking sequences) among all individuals of the same species.</p>                            <h2 scrollto-destination=5605199 id="5605199" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5605201 id="5605201" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by the State Key Basic Research Program of China (No. 2012CB114506), and the Project of the National Natural Science Foundation of China (No. 31170622, 30872042).</p>                            <h2 scrollto-destination=5605203 id="5605203" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dss031c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Difazio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">1596</div>-<div class="lpage">604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brunner</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Busov</div> <div class="given-names">V.B.</div></div>,  <div class="name"><div class="surname">Strauss</div> <div class="given-names">S.H.</div></div>. </div><div class="article-title">Poplar genome sequence: functional genomics in an ecologically dominant plant species</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">49</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poplar%20genome%20sequence%3A%20functional%20genomics%20in%20an%20ecologically%20dominant%20plant%20species&amp;author=A.M.%20Brunner&amp;author=V.B.%20Busov&amp;author=S.H.%20Strauss&amp;publication_year=2004&amp;journal=Trends%20Plant%20Sci.&amp;volume=9&amp;pages=49-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2003.11.006" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2003.11.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2003.11.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14729219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chiang</div> <div class="given-names">V.L.</div></div>. </div><div class="article-title">A genomic and molecular view of wood formation</div>, <div class="source">Crit. Rev. Plant Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">213</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20and%20molecular%20view%20of%20wood%20formation&amp;author=L.%20Li&amp;author=S.%20Lu&amp;author=V.L.%20Chiang&amp;publication_year=2006&amp;journal=Crit.%20Rev.%20Plant%20Sci.&amp;volume=25&amp;pages=213-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07352680600611519" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07352680600611519" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07352680600611519"> </span></div> </div></div></div></div></div><div content-id="dss031c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Du</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">The actin multigene family in <em>Populus</em>: organization, expression and phylogenetic analysis</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">105</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20actin%20multigene%20family%20in%20Populus%3A%20organization%2C%20expression%20and%20phylogenetic%20analysis&amp;author=D.Q.%20Zhang&amp;author=Q.Z.%20Du&amp;author=B.H.%20Xu&amp;author=Z.Y.%20Zhang&amp;author=B.L.%20Li&amp;publication_year=2010&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=284&amp;pages=105-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-010-0552-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-010-0552-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-010-0552-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20577761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">The status and advances of genetic improvement of <em>Populus tomentosa</em> Carr.</div>, <div class="source">J. Beijing Forestry Univ.</div> , <div class="year">1997</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">1</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="dss031c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">Development of 15 novel microsatellite markers from cellulose synthase genes in <em>Populus tomentosa</em> (Salicaceae)</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2012</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">e46</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%2015%20novel%20microsatellite%20markers%20from%20cellulose%20synthase%20genes%20in%20Populus%20tomentosa%20(Salicaceae)&amp;author=Q.Z.%20Du&amp;author=D.Q.%20Zhang&amp;author=B.L.%20Li&amp;publication_year=2012&amp;journal=Am.%20J.%20Bot.&amp;volume=99&amp;pages=e46-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1100308" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1100308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1100308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22268219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blair</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">González</div> <div class="given-names">L.F.</div></div>,  <div class="name"><div class="surname">Kimani</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Butare</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genetic diversity, inter-gene pool introgression and nutritional quality of common beans (<em>Phaseolus vulgaris</em> L.) from Central Africa</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">237</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%2C%20inter-gene%20pool%20introgression%20and%20nutritional%20quality%20of%20common%20beans%20(Phaseolus%20vulgaris%20L.)%20from%20Central%20Africa&amp;author=M.W.%20Blair&amp;author=L.F.%20Gonz%C3%A1lez&amp;author=P.M.%20Kimani&amp;author=L.%20Butare&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=237-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1305-x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1305-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1305-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genic microsatellite markers in plants: features and applications</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">48</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotechnol.&amp;volume=23&amp;pages=48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanafey</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">194</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20are%20preferentially%20associated%20with%20nonrepetitive%20DNA%20in%20plant%20genomes&amp;author=M.%20Morgante&amp;author=M.%20Hanafey&amp;author=W.%20Powell&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=30&amp;pages=194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng822" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11799393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yin</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.Y.</div></div>,  <div class="name"><div class="surname">Gunter</div> <div class="given-names">L.E.</div></div>, et al. </div><div class="article-title">Microsatellite primers resource developed from the mapped sequence scaffolds of Nisqually-1 genome</div>, <div class="source">New Phytol.</div> , <div class="year">2009</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">498</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20primers%20resource%20developed%20from%20the%20mapped%20sequence%20scaffolds%20of%20Nisqually-1%20genome&amp;author=T.M.%20Yin&amp;author=X.Y.%20Zhang&amp;author=L.E.%20Gunter&amp;publication_year=2009&amp;journal=New%20Phytol.&amp;volume=181&amp;pages=498-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2009.181.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2009.181.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2009.181.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">S.X.</div></div>,  <div class="name"><div class="surname">Yin</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">M.X.</div></div>,  <div class="name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">Characterization of microsatellites in the coding regions of the <em>Populus</em> genome</div>, <div class="source">Mol. Breeding</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">59</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20microsatellites%20in%20the%20coding%20regions%20of%20the%20Populus%20genome&amp;author=S.X.%20Li&amp;author=T.M.%20Yin&amp;author=M.X.%20Wang&amp;author=G.A.%20Tuskan&amp;publication_year=2011&amp;journal=Mol.%20Breeding&amp;volume=27&amp;pages=59-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9413-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9413-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9413-5"> </span></div> </div></div></div></div></div><div content-id="dss031c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">B.W.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">Z.Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">Genetic diversity and population structure of Chinese white poplar (<em>Populus tomentosa</em>) revealed by SSR markers</div>, <div class="source">J. Hered</div> , <div class="year">2012</div> <div class="comment">In press</div></div></div></div></div><div content-id="dss031c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">, Genetic mapping in (Populus tomentosa × P. bolleana) and P. tomentosa Carr. using AFLP markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">657</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%2C%20Genetic%20mapping%20in%20(Populus%20tomentosa%20%C3%97%20P.%20bolleana)%20and%20P.%20tomentosa%20Carr.%20using%20AFLP%20markers&amp;author=D.Q.%20Zhang&amp;author=Z.Y.%20Zhang&amp;author=K.%20Yang&amp;author=B.L.%20Li&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=657-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1478-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1478-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1478-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14564399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thumma</div> <div class="given-names">B.R.</div></div>,  <div class="name"><div class="surname">Nolan</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Evans</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Moran</div> <div class="given-names">G.F.</div></div>. </div><div class="article-title">Polymorphisms in cinnamoyl coa reductase (CCR) are associated with variation in microﬁbril angle in <em>Eucalyptus</em> spp.</div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1257</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymorphisms%20in%20cinnamoyl%20coa%20reductase%20(CCR)%20are%20associated%20with%20variation%20in%20micro%EF%AC%81bril%20angle%20in%20Eucalyptus%20spp.&amp;author=B.R.%20Thumma&amp;author=M.F.%20Nolan&amp;author=R.%20Evans&amp;author=G.F.%20Moran&amp;publication_year=2005&amp;journal=Genetics&amp;volume=171&amp;pages=1257-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.042028" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.042028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.042028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">Expression and nucleotide diversity of the poplar <em>COBL</em> gene</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">331</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20and%20nucleotide%20diversity%20of%20the%20poplar%20COBL%20gene&amp;author=D.Q.%20Zhang&amp;author=X.H.%20Yang&amp;author=Z.Y.%20Zhang&amp;author=B.L.%20Li&amp;publication_year=2010&amp;journal=Tree%20Genet.%20Genomes&amp;volume=6&amp;pages=331-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-009-0252-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-009-0252-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-009-0252-7"> </span></div> </div></div></div></div></div><div content-id="dss031c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">The sucrose synthase gene family in <em>Populus</em>: structure, expression, and evolution</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2011</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">443</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sucrose%20synthase%20gene%20family%20in%20Populus%3A%20structure%2C%20expression%2C%20and%20evolution&amp;author=D.Q.%20Zhang&amp;author=B.H.%20Xu&amp;author=X.H.%20Yang&amp;author=Z.Y.%20Zhang&amp;author=B.L.%20Li&amp;publication_year=2011&amp;journal=Tree%20Genet.%20Genomes&amp;volume=7&amp;pages=443-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-010-0346-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-010-0346-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-010-0346-2"> </span></div> </div></div></div></div></div><div content-id="dss031c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Declerck</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhpur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mccouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length variation, transposon associations, and genetic marker potential</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20length%20variation%2C%20transposon%20associations%2C%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20Declerck&amp;author=A.%20Lukashova&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhpur&amp;author=S.%20Mccouch&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1441-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yeh</div> <div class="given-names">F.C.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Boyle</div> <div class="given-names">T.</div></div>. </div><div class="article-title">POPGENE version 1.32: Microsoft Windows-based freeware for population genetic analysis, quick user guide</div>, <div class="year">1999</div><div class="publisher-loc">Canada</div><div class="publisher-name">University of Alberta</div> <div class="comment">Center for International Forestry Research</div></div></div></div></div><div content-id="dss031c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muse</div> <div class="given-names">S.V.</div></div>. </div><div class="article-title">PowerMarker: an integrated analysis environment for genetic markers analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">22128</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="dss031c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>. </div> <div class="comment">Polymorphic SSR loci within PtoCesA genes are associated with growth and wood properties in <em>Populus tomentosa New Phytol</em> In press</div></div></div></div></div><div content-id="dss031c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Briggs</div> <div class="given-names">W.H.</div></div>, et al. </div><div class="article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">203</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed-model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=203-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Kroon</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Casstevens</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2633</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.E.%20Kroon&amp;author=T.M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Storey</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Tibshirani</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Statistical signiﬁcance for genomewide studies</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">9440</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20signi%EF%AC%81cance%20for%20genomewide%20studies&amp;author=J.D.%20Storey&amp;author=R.%20Tibshirani&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=9440-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1530509100" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1530509100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1530509100"> </span></div> </div></div></div></div></div><div content-id="dss031c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eckenwalder</div> <div class="given-names">J.E.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Stettler</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Bradshaw</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Heilman</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Hinckley</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Systematics and evolution of <em>Populus</em></div>, <div class="source">Biology of <em>Populus</em> and its implications for management and conservation</div> , <div class="year">1996</div><div class="publisher-loc">Ottawa, Canada</div><div class="publisher-name">NRC Research Press</div>(pg. <div class="fpage">7</div>-<div class="lpage">32</div>)</div></div></div></div><div content-id="dss031c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Gunter</div> <div class="given-names">L.E.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.M.</div></div>,  <div class="name"><div class="surname">Yin</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Sewell</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">DiFazio</div> <div class="given-names">S.P.</div></div>. </div><div class="article-title">Characterization of microsatellites revealed by genomic sequencing of <em>Populus trichocarpa</em></div>, <div class="source">Can. J. Forest Res.</div> , <div class="year">2004</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">5</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20microsatellites%20revealed%20by%20genomic%20sequencing%20of%20Populus%20trichocarpa&amp;author=G.A.%20Tuskan&amp;author=L.E.%20Gunter&amp;author=Z.M.%20Yang&amp;author=T.M.%20Yin&amp;author=M.M.%20Sewell&amp;author=S.P.%20DiFazio&amp;publication_year=2004&amp;journal=Can.%20J.%20Forest%20Res.&amp;volume=34&amp;pages=5-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/x03-283" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fx03-283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fx03-283"> </span></div> </div></div></div></div></div><div content-id="dss031c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zaki</div> <div class="given-names">N.M.</div></div>,  <div class="name"><div class="surname">Ting</div> <div class="given-names">N.-C.</div></div>, et al. </div><div class="article-title">Exploiting an oil palm EST database for the development of gene-derived SSR markers and their exploitation for assessment of genetic diversity</div>, <div class="source">Biologia</div> , <div class="year">2008</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">227</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20an%20oil%20palm%20EST%20database%20for%20the%20development%20of%20gene-derived%20SSR%20markers%20and%20their%20exploitation%20for%20assessment%20of%20genetic%20diversity&amp;author=R.%20Singh&amp;author=N.M.%20Zaki&amp;author=N.-C.%20Ting&amp;publication_year=2008&amp;journal=Biologia&amp;volume=63&amp;pages=227-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2478/s11756-008-0041-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2478%2Fs11756-008-0041-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2478%2Fs11756-008-0041-z"> </span></div> </div></div></div></div></div><div content-id="dss031c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feng</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">H.S.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Y.T.</div></div>. </div><div class="article-title">Development, characterization and cross-species/genera transferability of EST-SSR markers for rubber tree (<em>Hevea brasiliensis</em>)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">85</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%2C%20characterization%20and%20cross-species%2Fgenera%20transferability%20of%20EST-SSR%20markers%20for%20rubber%20tree%20(Hevea%20brasiliensis)&amp;author=S.P.%20Feng&amp;author=W.G.%20Li&amp;author=H.S.%20Huang&amp;author=J.Y.%20Wang&amp;author=Y.T.%20Wu&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=23&amp;pages=85-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-008-9216-0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-008-9216-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-008-9216-0"> </span></div> </div></div></div></div></div><div content-id="dss031c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chagné</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chaumeil</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ramboer</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Cross-species transferability and mapping of genomic and cDNA SSRs in pines</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">1204</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-species%20transferability%20and%20mapping%20of%20genomic%20and%20cDNA%20SSRs%20in%20pines&amp;author=D.%20Chagn%C3%A9&amp;author=P.%20Chaumeil&amp;author=A.%20Ramboer&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=1204-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1683-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1683-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1683-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hisano</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Characterization of the soybean genome using EST-derived microsatellite markers</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">271</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20soybean%20genome%20using%20EST-derived%20microsatellite%20markers&amp;author=H.%20Hisano&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=271-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm025" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cardle</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ramsay</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Milbourne</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Macaulay</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Marshall</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Waugh</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Computational and experimental characterization of physically clustered simple sequence repeats in plants</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">847</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20characterization%20of%20physically%20clustered%20simple%20sequence%20repeats%20in%20plants&amp;author=L.%20Cardle&amp;author=L.%20Ramsay&amp;author=D.%20Milbourne&amp;author=M.%20Macaulay&amp;author=D.%20Marshall&amp;author=R.%20Waugh&amp;publication_year=2000&amp;journal=Genetics&amp;volume=156&amp;pages=847-54" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11014830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kutter</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Watt</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Stefflova</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Rapid turnover of long noncoding RNAs and the evolution of gene expression</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002841</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.1002841" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1371/journal.pgen.1002841</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20turnover%20of%20long%20noncoding%20RNAs%20and%20the%20evolution%20of%20gene%20expression&amp;author=C.%20Kutter&amp;author=S.%20Watt&amp;author=K.%20Stefflova&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002841" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002841" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22844254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.C.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Fahima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Microsatellites within genes: structure, function, and evolution</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">991</div>-<div class="lpage">1007</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20within%20genes%3A%20structure%2C%20function%2C%20and%20evolution&amp;author=Y.C.%20Li&amp;author=A.B.%20Korol&amp;author=T.%20Fahima&amp;author=E.%20Nevo&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=991-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh073" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Penmetsa</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Development and characterization of BAC-end sequence derived SSRs, and their incorporation into a new higher density genetic map for cultivated peanut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> pg. <div class="fpage">10</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20BAC-end%20sequence%20derived%20SSRs%2C%20and%20their%20incorporation%20into%20a%20new%20higher%20density%20genetic%20map%20for%20cultivated%20peanut%20(Arachis%20hypogaea%20L.)&amp;author=H.%20Wang&amp;author=R.V.%20Penmetsa&amp;author=M.%20Yuan&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-10" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22260238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toth</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gaspari</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Jurke</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Microsatellites in different eukaryotic genomes: survey and analysis</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">967</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20different%20eukaryotic%20genomes%3A%20survey%20and%20analysis&amp;author=G.%20Toth&amp;author=Z.%20Gaspari&amp;author=J.%20Jurke&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=967-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.7.967" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.7.967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.7.967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slavov</div> <div class="given-names">G.T.</div></div>,  <div class="name"><div class="surname">Zhelev</div> <div class="given-names">P.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bhalerao</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Groover</div> <div class="given-names">A.T.</div></div>. </div><div class="article-title">Salient biological features, systematics, and genetic variation of <em>Populus</em></div>, <div class="source">Genetics and genomics of <em>Populu</em>s</div> , <div class="year">2010</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">15</div>-<div class="lpage">38</div>)</div></div></div></div><div content-id="dss031c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Genetic variation within and between populations of a desert poplar (<em>Populus euphratica</em>) revealed by SSR markers</div>, <div class="source">Ann. Forest Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">1143</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20within%20and%20between%20populations%20of%20a%20desert%20poplar%20(Populus%20euphratica)%20revealed%20by%20SSR%20markers&amp;author=J.%20Wang&amp;author=Z.%20Li&amp;author=Q.%20Guo&amp;author=G.%20Ren&amp;author=Y.%20Wu&amp;publication_year=2011&amp;journal=Ann.%20Forest%20Sci.&amp;volume=68&amp;pages=1143-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13595-011-0119-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13595-011-0119-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13595-011-0119-6"> </span></div> </div></div></div></div></div><div content-id="dss031c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">S.Z.</div></div>. </div><div class="article-title">Quantitative trait locus mapping can benefit from segregation distortion</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">2201</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20locus%20mapping%20can%20benefit%20from%20segregation%20distortion&amp;author=S.Z.%20Xu&amp;publication_year=2008&amp;journal=Genetics&amp;volume=180&amp;pages=2201-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.090688" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.090688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.090688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18957707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Taylor</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Ingvarsson</div> <div class="given-names">P.K.</div></div>. </div><div class="article-title">Common features of segregation distortion in plants and animals</div>, <div class="source">Genetica</div> , <div class="year">2003</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">27</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20features%20of%20segregation%20distortion%20in%20plants%20and%20animals&amp;author=D.R.%20Taylor&amp;author=P.K.%20Ingvarsson&amp;publication_year=2003&amp;journal=Genetica&amp;volume=117&amp;pages=27-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022308414864" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022308414864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022308414864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12656570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hartl</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Laurie</div> <div class="given-names">C.C.</div></div>. </div><div class="article-title">Sex-ratio segregation distortion associated with reproductive isolation in <em>Drosophila</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">13183</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex-ratio%20segregation%20distortion%20associated%20with%20reproductive%20isolation%20in%20Drosophila&amp;author=Y.%20Tao&amp;author=D.L.%20Hartl&amp;author=C.C.%20Laurie&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=13183-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.231478798" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.231478798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.231478798"> </span></div> </div></div></div></div></div><div content-id="dss031c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zuo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Conservation of noncoding microsatellites in plants: implication for gene regulation</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">323</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20noncoding%20microsatellites%20in%20plants%3A%20implication%20for%20gene%20regulation&amp;author=L.%20Zhang&amp;author=K.%20Zuo&amp;author=F.%20Zhang&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-323" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17187690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kao</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Harding</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Tsai</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">Differential expression of two distinct phenylalanine ammonia-lyase genes in condensed tannin-accumulating and lignifying cells of quaking aspen</div>, <div class="source">Plant Physiol.</div> , <div class="year">2002</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">796</div>-<div class="lpage">807</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20two%20distinct%20phenylalanine%20ammonia-lyase%20genes%20in%20condensed%20tannin-accumulating%20and%20lignifying%20cells%20of%20quaking%20aspen&amp;author=Y.Y.%20Kao&amp;author=S.A.%20Harding&amp;author=C.J.%20Tsai&amp;publication_year=2002&amp;journal=Plant%20Physiol.&amp;volume=130&amp;pages=796-807" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.006262" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.006262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.006262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12376645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.G.</div></div>,  <div class="name"><div class="surname">Chiang</div> <div class="given-names">V.L.</div></div>. </div><div class="article-title">Distinct roles of cinnamate 4-hydroxylase genes in <em>Populus</em></div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">905</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20roles%20of%20cinnamate%204-hydroxylase%20genes%20in%20Populus&amp;author=S.F.%20Lu&amp;author=Y.H.%20Zhou&amp;author=L.G.%20Li&amp;author=V.L.%20Chiang&amp;publication_year=2006&amp;journal=Plant%20Cell%20Physiol.&amp;volume=47&amp;pages=905-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcj063" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcj063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcj063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16720648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wegrzyn</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Eckert</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (<em>Populus trichocarpa</em>, Salicaceae) secondary xylem</div>, <div class="source">New Phytol.</div> , <div class="year">2010</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">515</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20genetics%20of%20traits%20controlling%20lignin%20and%20cellulose%20biosynthesis%20in%20black%20cottonwood%20(Populus%20trichocarpa%2C%20Salicaceae)%20secondary%20xylem&amp;author=J.L.%20Wegrzyn&amp;author=A.J.%20Eckert&amp;author=M.%20Choi&amp;publication_year=2010&amp;journal=New%20Phytol.&amp;volume=188&amp;pages=515-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03415.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03415.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03415.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20831625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boerjan</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ralph</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Baucher</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Lignin biosynthesis</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">519</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lignin%20biosynthesis&amp;author=W.%20Boerjan&amp;author=J.%20Ralph&amp;author=M.%20Baucher&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=54&amp;pages=519-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.54.031902.134938" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.54.031902.134938" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.54.031902.134938"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14503002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coleman</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mansﬁeld</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">13118</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sucrose%20synthase%20affects%20carbon%20partitioning%20to%20increase%20cellulose%20production%20and%20altered%20cell%20wall%20ultrastructure&amp;author=H.D.%20Coleman&amp;author=J.%20Yan&amp;author=S.D.%20Mans%EF%AC%81eld&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=13118-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0900188106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0900188106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0900188106"> </span></div> </div></div></div></div></div><div content-id="dss031c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Rombauts</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sterky</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Rouzé</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Vande Peer</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">EST data suggest that poplar is an ancient polyploidy</div>, <div class="source">New Phytol.</div> , <div class="year">2005</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">165</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%20data%20suggest%20that%20poplar%20is%20an%20ancient%20polyploidy&amp;author=L.%20Sterck&amp;author=S.%20Rombauts&amp;author=S.%20Jansson&amp;author=F.%20Sterky&amp;author=P.%20Rouz%C3%A9&amp;author=Y.%20Vande%20Peer&amp;publication_year=2005&amp;journal=New%20Phytol.&amp;volume=167&amp;pages=165-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2005.01378.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2005.01378.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2005.01378.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15948839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leskinen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Alström-Rapaport</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Molecular phylogeny of Salicaceae and closely related Flacourtiaceae: evidence from 5.8 S, ITS 1 and ITS 2 of the rDNA</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">209</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogeny%20of%20Salicaceae%20and%20closely%20related%20Flacourtiaceae%3A%20evidence%20from%205.8%20S%2C%20ITS%201%20and%20ITS%202%20of%20the%20rDNA&amp;author=E.%20Leskinen&amp;author=C.%20Alstr%C3%B6m-Rapaport&amp;publication_year=1999&amp;journal=Plant%20Syst.%20Evol.&amp;volume=215&amp;pages=209-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00984656" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00984656" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00984656"> </span></div> </div></div></div></div></div><div content-id="dss031c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prasad</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Muthulakshmi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Madhu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Archak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nagaraju</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Survey and analysis of microsatellites in the silkworm, <em>Bombyx mori</em>: frequency, distribution, mutations, marker potential and their conservation in heterologous species</div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">197</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20and%20analysis%20of%20microsatellites%20in%20the%20silkworm%2C%20Bombyx%20mori%3A%20frequency%2C%20distribution%2C%20mutations%2C%20marker%20potential%20and%20their%20conservation%20in%20heterologous%20species&amp;author=M.D.%20Prasad&amp;author=M.%20Muthulakshmi&amp;author=M.%20Madhu&amp;author=S.%20Archak&amp;author=K.%20Mita&amp;author=J.%20Nagaraju&amp;publication_year=2004&amp;journal=Genetics&amp;volume=169&amp;pages=197-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.031005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.031005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.031005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15371363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss031c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss031c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Microsatellites: simple sequences with complex evolution</div>, <div class="source">Nature Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">435</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20simple%20sequences%20with%20complex%20evolution&amp;author=H.%20Ellegren&amp;publication_year=2004&amp;journal=Nature%20Rev.%20Genet.&amp;volume=5&amp;pages=435-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1348" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1348"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=382624&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table8.doc?Expires=1509238854&amp;Signature=DwopIa2iClr5HoeqwUCI7~~yhLc3v8y1uP3tTHqDLy~s8L6ekb4ha~84vzeJYN1jbu2idUMlLIgxls1tLmWT5BSKahiczd2bQbR6sVtGz4WOtyDNQe9BQx0KrxDsZhUeQ7vRzvlrTG7ZL9L7y3k78JtPw72FbNUt0pvDKwfSx1gfgFDcfiogrf7J9MQngp1NzBlbtYselGC5lAIBzLRLmHi7DsW73IkxymPWb5NcroYfDecT3~LqY9SyhM0XUh6GWp4S9zjkBuPhpLg3xFBSVSKzwAQGt2LGu9MOSBpNGy-9gHo6YNuwIyNnW~Z-On~5eV~07GI125JUZvdv2QAQOA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-9</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table7.doc?Expires=1509238854&amp;Signature=HpUFPSW99vNojF0s6UetpNBpzt3dtWaQ7iP6qDL~03SdAAuzShNUHrik5VDSxjygo3U9mlfH-0iJPTUIlq7mGTou59zHu7IozHldttpWLii~Jwu4H-3hhcOcMy29-h35AFPyMErXtQmnrY4nx5wEFT9BqCLR-6kqwlwy4daQtAsyyvKDSnkJwfZPWd6ijV8eCYjoIYdTzHiWDZXcnGfGfkhHvevA4SZvjU6ySZGL7dKJ5669Uk9ty1Rnt4bU6LA~JNubwvYrIu6C8JjPeUyNWeUKUraLTjA3R3TXUj589krUuhMClnNibmssMQ7i4ZUrzj6k5459gdiUw9LHoAd3fA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-8</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table6.doc?Expires=1509238854&amp;Signature=DToaYesW5x-aoWZ-7aYCuCfXszJQaoReXLU7lBviKYSjcgFnmjQo9qCByqpLoK5CvvzOEoNqncYzwBoVuf5cn~pCilJCL1MKhQo-r8c3vtuIzANM1xCbrGWVKdcRlHy91m67EQ6Op1P7ruqF0-LVTfcP-APGtH21qm4p4VaqfVz3aRqDN-ASv55DiuoG0CrJX4wW3zWi-EZdQNCNMRapWge2pnNRfPfqSQlbCBvWnb4zN82AbIyE1p27yZ2T5abw72efywYeaopIBNT8j2SupMEy2vhZDtzq6Dr~toM6DsMHYptsYru3sBNOeoYRJSf2wNOV1pX2akyld-tZNgFbhw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table5.doc?Expires=1509238854&amp;Signature=Be8It0yZADeT9DVSFSfu9KYYz11ebo65~ez0MWfKJt52erH6s002FVhagYwnPjQE6lujQSioMj2GcDTDT7q5RI977ETg2nRwKBqz~Usp99w1uJnmm5kfQ2leyEvM5s601P9VHDIwLiJwoX6zpvd0EoA30X8detMktgUxc0a6BpH7CB8E2ROKftwP1U1tvUP-pm5PgweuCplW9bGYu2jmlizJtKk5no3lDOIuuTUdcKN-DY1IB8-AheDjUQwbAAeNiGyIhQXk7lVbyXux6sasjrdo8X6C1xczeGOJbrtx77cp7tygLLGVk3WVCw1lnxm5flTWVRR7AqFvqNgLJ4cc3Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table4.doc?Expires=1509238854&amp;Signature=BIioY1cvUI6tTo6Bq17UpiL-~ebLI8tcuPqmne5KofDh2SFWbqs7zn8FviH7mr-gD4zq8KeXqEJ~MtI7ZYAavbN55BA56Y8~UZxjZWM~1r2tDYbo~0KzVwLsJ6d8JxXjCkguFiufJn7LVr3NHnSH7vlUKh3Mlk5bih0FbVaONNiHtx6q9x8pwg4LTbb~OucAsCQeubfM8QmJPyEycWrbe8DU7ybVVDQqluWnFbQvl2DlWOp95XNDvONA9Wnra5FJlQoM~lYiRTxUhuDIHPIvsZt~bezbWp24EPMNyPLXNcQcecbslo88uH2ih3zdOBUAe2tJJNUiqJMkfVFxQ340Wg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table3.doc?Expires=1509238854&amp;Signature=a6Unxh~WazMJfEx7yscg2tcJ6d2l6OAuLO1zko~hl-300OBdywfVIWh9nWjsd0cKzObmDUnHlxURdE9gQeEf5s0w5uL6i813S~A2PplJu4s1DuF~YyirPVfOLXzmbBqgu~e7RdO6SHlxr49bt0dlAeTNKi4AjNe~JoXAAhGoMXs2NEZ-cCmJbyUMxFNImc-BA9zVI8ya0QFd502D48RTcy06YzhX37TyvAN6IO5SqDiKo9eXvkODVe77znLeOyTl2VnBMPj96yZgSU2yy6jbuWgP0MnChZURfgztawDVsLqIuqLnOL2903g3Py-rEoomnQ69DnQvoIpsATFpXdS2HQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table2.doc?Expires=1509238854&amp;Signature=G6EpUifMreWsR61T2aH~5BM426aSQpdCeV5jHCRLuq85M~Y87Ta6-mtkZlIu1tZQmF94hWD1d3kzU6eCDwSeLdvYb9qNfRnKz4h~HowW67PLRhdFjUSptXNf~dc694WVkNFHeL6-D0hnHBIZ2TvzKuNuAkc2a8IfhYieFiVsfZOghN3MpcGV5auEiuNm51DgW8x5TyTM61tH-kuBpdrbW6T4o4qnk2AONvpOS7c~rMX58pDzf-aQylylnMofQ~mi0zSxVzMiCRVXH6C4kCPo1vYVtFFjqzE0TbMuDvs0k1yWuqMP~uP4meZ5hk860ZR46tkifCYJWoLlBZgTtY9hMg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_table1.doc?Expires=1509238854&amp;Signature=gwrThdRfRX13eQhTBb5eikz4KMF0AbW33Opvj9aDjLUINRiJcrwI9o-TH3ACEqrmuZRv-JI0eUu50PtGK-hRmUhGXe5p3km2OHCauUwjisJ8US2NmoroE8Xvpn4hFP7FF~P8CD97KBSLQfGRbMPi1uaMa~RM6A1lgqI8-4VclJqPbMCGrg9a0FebO99u9Viiqr-xwhFoGi2kyDh5OMbvZYbm7tBw-Ip7uCSyEqbHDA7Env~8zntXMzCHoZYmGs6vCWQofoDdgCR~tOJ84mpUHYWIguGi5IclmPXBgDdmJXGIvXDA~GttrK66Eueq-~ZccJkivhAOWwN4haNn~oPM2A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss031/2/dss031-dss031supp_fig.png?Expires=1509238854&amp;Signature=EKQ-HV7lbzdlXeXl0gXv0KEMyJhuzVqmoiSddna1Cuf6P7hnUeWf4eszyt9bRU2-ECqICZqO4r6EGsqf6Il0gGunccEHdgS7X5tDeEvtceJwqT7W7cUfIbmpr0Z0vhH0sBxZhjLeTe~~nRjXFDcrbA9L7ev108nNXvvE-YU3V8Mwg8Z05a~0WvcmtnvX0QDIt0O6BWPWRZz5vO2YMschhgJsuTorkjQ0YqOr3aAI07ztYVaSqHlKpEi9Cs-yVTG0952ZUbKtTUU7idFmiKwDtpx5a5d8DJYozn7-Oxby~R2USDrDkUevFqVF9U~q28fWM~rRKa9yRY5rxkcJTlkuEw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">32</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dss031"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="382624"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f31%2f382624">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F31%2F382624" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/382624'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Development and Application of Microsatellites in Candidate Genes Related to Wood Properties in the Chinese White Poplar (Populus tomentosa Carr.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f31%2f382624">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

