

<!DOCTYPE html>
<html lang="en">

<head><title>
	Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/45/382928","@type":"ScholarlyArticle","name":"Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts","datePublished":"2012-12-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss032","keywords":["chimaeric expression sequence tag (EST)","transcription factory","inter- and intrachromosomal interaction","chromatin","chromatin loop"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/45/382928","author":[{"name":"Li, Songling","affiliation":"1  Theoretical Biophysics Group, Institute for Theoretical Physics ,  University of Heidelberg ,  Heidelberg ,  Germany    4  Shanghai Center for Bioinformation Technology (SCBIT) ,  Shanghai ,  P.R. China","@type":"Person"},{"name":"Heermann, Dieter W.","affiliation":"1  Theoretical Biophysics Group, Institute for Theoretical Physics ,  University of Heidelberg ,  Heidelberg ,  Germany    2  The Jackson Laboratory ,  Bar Harbor, Maine ,  USA    3  Shanghai Institutes for Biological Sciences (SIBS) ,  Shanghai ,  P.R. China","@type":"Person"}],"description":"Transcription-induced chimaeric transcripts, the potential post-transcriptional processing products, might reflect the spatial proximity of actively transcribed genes co-localized in transcription factories. A growing number of expression data deposited in databases provide us with the raw material for screening such chimaeric transcripts and using them as the probes to identify interactions between genes in cis or in trans. Based on the high-quality chimaeric transcripts gleaned from human expression sequence tag data with selection criteria, we identified the patterns of inter- and intrachromosomal gene–gene interactions. On top the contact pattern from interchromosomal interactions, we also observed an exponential behaviour of the intrachromosomal interactions within a certain length scale, which is consistent with the independent experimental results from Hi-C screening and with the Random Loop Model. A compatible result is found for mouse. Transcription-induced chimaeric transcripts, most of which might be accidental products with trivial functions, shed light on the spatial organization of chromosomes. These inter- and intrachromosomal interactions might contribute to the compaction of chromosomes, their segregation and formation of the chromosome territories, and their spatial distribution within the nucleus.","pageStart":"45","pageEnd":"53","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1572217255&Signature=GjvTRR~uR9tqEmRy-7STAvoKslsXgS86OMrfB0wpPomtKZd1BL~qs6dYe41fTtJjXed1YuZA5UCvan~3iaFGKYNb0T7qP99O~kKPT0Wo2sCXmzHsdo3qwdqZpbCH000-YpfyH~hYHB5kMOroBJ2Zb4W6HWYddOFYKysueSJVRaKpwJXA79sPAq2UUGjwMo4OW9Q2B9WzAeX2SQsNAPlEfohNWoYgD0tRyqR72wImT0bBUNg6tle2z6WXaAwqZ445xkmoGohe2jOYgPlC20WpSmBZNemkZwB6L-lOAdOxiTv2h5xIQV2gYkpru3Z4tvvxedGMYUGXeWHOi2TKgOeTSg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1572217255&Signature=GjvTRR~uR9tqEmRy-7STAvoKslsXgS86OMrfB0wpPomtKZd1BL~qs6dYe41fTtJjXed1YuZA5UCvan~3iaFGKYNb0T7qP99O~kKPT0Wo2sCXmzHsdo3qwdqZpbCH000-YpfyH~hYHB5kMOroBJ2Zb4W6HWYddOFYKysueSJVRaKpwJXA79sPAq2UUGjwMo4OW9Q2B9WzAeX2SQsNAPlEfohNWoYgD0tRyqR72wImT0bBUNg6tle2z6WXaAwqZ445xkmoGohe2jOYgPlC20WpSmBZNemkZwB6L-lOAdOxiTv2h5xIQV2gYkpru3Z4tvvxedGMYUGXeWHOi2TKgOeTSg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Transcription-induced chimaeric transcripts, the potential post-transcriptional processing products, might reflect the spatial proximity of actively transcribed genes co-localized in transcription factories. A growing number of expression data deposited in databases provide us with the raw material for screening such chimaeric transcripts and using them as the probes to identify interactions between genes in cis or in trans. Based on the high-quality chimaeric transcripts gleaned from human expression sequence tag data with selection criteria, we identified the patterns of inter- and intrachromosomal gene–gene interactions. On top the contact pattern from interchromosomal interactions, we also observed an exponential behaviour of the intrachromosomal interactions within a certain length scale, which is consistent with the independent experimental results from Hi-C screening and with the Random Loop Model. A compatible result is found for mouse. Transcription-induced chimaeric transcripts, most of which might be accidental products with trivial functions, shed light on the spatial organization of chromosomes. These inter- and intrachromosomal interactions might contribute to the compaction of chromosomes, their segregation and formation of the chromosome territories, and their spatial distribution within the nucleus." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/45/382928" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217255&amp;Signature=GAMdLt1xOfW7MjitqXhQwrTMlcpYiQxReSTjQtcxck44vgrIwBkkw67~tL2j7WuKEirSxzg0O74cmy5da0kipyfRrQpzhzglftaInUD4t9Nysz6MgepXt~zqtPY-F0Trcw73InAj99xxbmHBwdPZIUkUWC2GCyS5cMQLopHwL-9ylLZjXe3kWeGYgvQ1sSzsP3-n4fveEE31-v5NKBrKozO3p9C66d~atb75bBriiO9gKgjpZfjS0ZvKFmMV5wQx70juKfDJpwa~DxEyrR-DuD5tBfHEk1KleOBMAsE~UGy8CHZ0weorMcMLyjdaAtklpRkDdBjPicTWYOC~S2c~4A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2012-12-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Songling" /><meta name="citation_author_institution" content="Theoretical Biophysics Group, Institute for Theoretical Physics, University of Heidelberg, Heidelberg, Germany" /><meta name="citation_author_institution" content="Shanghai Center for Bioinformation Technology (SCBIT), Shanghai, P.R. China" /><meta name="citation_author" content="Heermann, Dieter W." /><meta name="citation_author_institution" content="Theoretical Biophysics Group, Institute for Theoretical Physics, University of Heidelberg, Heidelberg, Germany" /><meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine, USA" /><meta name="citation_author_institution" content="Shanghai Institutes for Biological Sciences (SIBS), Shanghai, P.R. China" /><meta name="citation_title" content="Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts" /><meta name="citation_firstpage" content="45" /><meta name="citation_lastpage" content="53" /><meta name="citation_doi" content="10.1093/dnares/dss032" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Transcription-mediated gene fusion in the human genome; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=30-6; " /><meta name="citation_reference" content="citation_title=Pre-mRNA trans-splicing: from kinetoplastids to mammals, an easy language for life diversity; Mem. Inst. Oswaldo Cruz;  citation_year=2005;  citation_volume=100;  citation_pages=501-13; " /><meta name="citation_reference" content="citation_title=Trans-splicing in protozoa and helminths; Infect. Agents Dis.;  citation_year=1992;  citation_volume=1;  citation_pages=212-8; " /><meta name="citation_reference" content="citation_title=A trans-spliced leader sequence on actin mRNA in C. elegans; Cell;  citation_year=1987;  citation_volume=49;  citation_pages=753-61; " /><meta name="citation_reference" content="citation_title=Genomic overview of mRNA 5′-leader trans-splicing in the ascidian Ciona intestinalis; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=3378-88; " /><meta name="citation_reference" content="citation_title=Trans-splicing as a novel mechanism to explain interallelic complementation in Drosophila; Genetics;  citation_year=2002;  citation_volume=160;  citation_pages=1481-7; " /><meta name="citation_reference" content="citation_title=A candidate chimeric mammalian mRNA transcript is derived from distinct chromosomes and is associated with nonconsensus splice junction motifs; DNA Cell Biol.;  citation_year=2003;  citation_volume=22;  citation_pages=303-15; " /><meta name="citation_reference" content="citation_title=Short homologous sequences are strongly associated with the generation of chimeric RNAs in eukaryotes; J. Mol. Evol.;  citation_year=2009;  citation_volume=68;  citation_pages=56-65; " /><meta name="citation_reference" content="citation_title=Novel domain combinations in proteins encoded by chimeric transcripts; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=i67-74; " /><meta name="citation_reference" content="citation_title=Tumor-specific retargeting of an oncogenic transcription factor chimera results in dysregulation of chromatin and transcription; Genome Res.;  citation_year=2011;  citation_volume=22;  citation_pages=259-70; " /><meta name="citation_reference" content="citation_title=A neoplastic gene fusion mimics trans-splicing of RNAs in normal human cells; Science;  citation_year=2008;  citation_volume=321;  citation_pages=1357-61; " /><meta name="citation_reference" content="citation_title=Transcription factories; Biochem. Soc. Trans.;  citation_year=2008;  citation_volume=36;  citation_pages=585-9; " /><meta name="citation_reference" content="citation_title=Close encounters between active genes in the nucleus; Genome Biol.;  citation_year=2005;  citation_volume=6;  citation_pages=237" /><meta name="citation_reference" content="citation_title=Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations; PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e138" /><meta name="citation_reference" content="citation_title=Fractal and statistical properties of large compact polymers: a computational study; Polymer;  citation_year=2004;  citation_volume=45;  citation_pages=717-31; " /><meta name="citation_reference" content="citation_title=Comprehensive mapping of long-range interactions reveals folding principles of the human genome; Science;  citation_year=2009;  citation_volume=326;  citation_pages=289-93; " /><meta name="citation_reference" content="citation_title=Random loop model for long polymers; Phys. Rev. E Stat. Nonlin. Soft Matter Phys.;  citation_year=2007;  citation_volume=76;  citation_pages=051805" /><meta name="citation_reference" content="citation_title=Spatially confined folding of chromatin in the interphase nucleus; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=3812-7; " /><meta name="citation_reference" content="citation_title=Topological interactions between ring polymers: implications for chromatin loops; J. Chem. Phys.;  citation_year=2010;  citation_volume=132;  citation_pages=044904" /><meta name="citation_reference" content="citation_title=Tentative mapping of transcription-induced interchromosomal interaction using chimeric EST and mRNA data; PLoS One;  citation_year=2007;  citation_volume=2;  citation_pages=e254" /><meta name="citation_reference" content="citation_title=Probabilistic modeling of Hi-C contact maps eliminates systematic biases to characterize global chromosomal architecture; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1059-65; " /><meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=COXPRESdb: a database of coexpressed gene networks in mammals; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D77-82; " /><meta name="citation_reference" content="citation_title=A novel algorithm for computational identification of contaminated EST libraries; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=1067-74; " /><meta name="citation_reference" content="citation_title=Review: proximity effects in the production of chromosome aberrations by ionizing radiation; Int. J. Radiat. Biol.;  citation_year=1997;  citation_volume=71;  citation_pages=1-19; " /><meta name="citation_reference" content="citation_title=Finding fusion genes resulting from chromosome rearrangement by analyzing the expressed sequence databases; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=13257-61; " /><meta name="citation_reference" content="citation_title=Fusion transcripts and transcribed retrotransposed loci discovered through comprehensive transcriptome analysis using paired-end ditags (PETs); Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=828-38; " /><meta name="citation_reference" content="citation_title=A probabilistic model for the arrangement of a subset of human chromosome territories in WI38 human fibroblasts; J. Cell Physiol.;  citation_year=2009;  citation_volume=221;  citation_pages=120-9; " /><meta name="citation_reference" content="citation_title=Cell type specific chromosome territory organization in the interphase nuclus of normal and cancer cells; J. Cell Physiol.;  citation_year=2009;  citation_volume=221;  citation_pages=130-8; " /><meta name="citation_reference" content="citation_title=Diffusion-driven looping provides a consistent framework for chromatin organization; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e12218" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/45/1085813/dss032.pdf" /><meta name="description" content="Abstract.  Transcription-induced chimaeric transcripts, the potential post-transcriptional processing products, might reflect the spatial proximity of actively " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/45/382928" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/45/382928" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dss032';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./382928" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFjEQ8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBTb25nbGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZW9yZXRpY2FsIEJpb3BoeXNpY3MgR3JvdXAsIEluc3RpdHV0ZSBmb3IgVGhlb3JldGljYWwgUGh5c2ljcywgVW5pdmVyc2l0eSBvZiBIZWlkZWxiZXJnLCBIZWlkZWxiZXJnLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2hhbmdoYWkgQ2VudGVyIGZvciBCaW9pbmZvcm1hdGlvbiBUZWNobm9sb2d5IChTQ0JJVCksIFNoYW5naGFpLCBQLlIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGVlcm1hbm4sIERpZXRlciBXLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZW9yZXRpY2FsIEJpb3BoeXNpY3MgR3JvdXAsIEluc3RpdHV0ZSBmb3IgVGhlb3JldGljYWwgUGh5c2ljcywgVW5pdmVyc2l0eSBvZiBIZWlkZWxiZXJnLCBIZWlkZWxiZXJnLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIEphY2tzb24gTGFib3JhdG9yeSwgQmFyIEhhcmJvciwgTWFpbmUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNoYW5naGFpIEluc3RpdHV0ZXMgZm9yIEJpb2xvZ2ljYWwgU2NpZW5jZXMgKFNJQlMpLCBTaGFuZ2hhaSwgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVXNpbmcgQ2hpbWFlcmljIEV4cHJlc3Npb24gU2VxdWVuY2UgVGFnIGFzIHRoZSBSZWZlcmVuY2UgdG8gSWRlbnRpZnkgVGhyZWUtRGltZW5zaW9uYWwgQ2hyb21vc29tZSBDb250YWN0cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbi1tZWRpYXRlZCBnZW5lIGZ1c2lvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTMwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZS1tUk5BIHRyYW5zLXNwbGljaW5nOiBmcm9tIGtpbmV0b3BsYXN0aWRzIHRvIG1hbW1hbHMsIGFuIGVhc3kgbGFuZ3VhZ2UgZm9yIGxpZmUgZGl2ZXJzaXR5OyBNZW0uIEluc3QuIE9zd2FsZG8gQ3J1ejsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz01MDEtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zLXNwbGljaW5nIGluIHByb3Rvem9hIGFuZCBoZWxtaW50aHM7IEluZmVjdC4gQWdlbnRzIERpcy47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTIxMi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRyYW5zLXNwbGljZWQgbGVhZGVyIHNlcXVlbmNlIG9uIGFjdGluIG1STkEgaW4gQy4gZWxlZ2FuczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTc1My02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBvdmVydmlldyBvZiBtUk5BIDXigLItbGVhZGVyIHRyYW5zLXNwbGljaW5nIGluIHRoZSBhc2NpZGlhbiBDaW9uYSBpbnRlc3RpbmFsaXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTMzNzgtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zLXNwbGljaW5nIGFzIGEgbm92ZWwgbWVjaGFuaXNtIHRvIGV4cGxhaW4gaW50ZXJhbGxlbGljIGNvbXBsZW1lbnRhdGlvbiBpbiBEcm9zb3BoaWxhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjA7ICBjaXRhdGlvbl9wYWdlcz0xNDgxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2FuZGlkYXRlIGNoaW1lcmljIG1hbW1hbGlhbiBtUk5BIHRyYW5zY3JpcHQgaXMgZGVyaXZlZCBmcm9tIGRpc3RpbmN0IGNocm9tb3NvbWVzIGFuZCBpcyBhc3NvY2lhdGVkIHdpdGggbm9uY29uc2Vuc3VzIHNwbGljZSBqdW5jdGlvbiBtb3RpZnM7IEROQSBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzAzLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG9ydCBob21vbG9nb3VzIHNlcXVlbmNlcyBhcmUgc3Ryb25nbHkgYXNzb2NpYXRlZCB3aXRoIHRoZSBnZW5lcmF0aW9uIG9mIGNoaW1lcmljIFJOQXMgaW4gZXVrYXJ5b3RlczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTU2LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBkb21haW4gY29tYmluYXRpb25zIGluIHByb3RlaW5zIGVuY29kZWQgYnkgY2hpbWVyaWMgdHJhbnNjcmlwdHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9aTY3LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UdW1vci1zcGVjaWZpYyByZXRhcmdldGluZyBvZiBhbiBvbmNvZ2VuaWMgdHJhbnNjcmlwdGlvbiBmYWN0b3IgY2hpbWVyYSByZXN1bHRzIGluIGR5c3JlZ3VsYXRpb24gb2YgY2hyb21hdGluIGFuZCB0cmFuc2NyaXB0aW9uOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI1OS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZW9wbGFzdGljIGdlbmUgZnVzaW9uIG1pbWljcyB0cmFucy1zcGxpY2luZyBvZiBSTkFzIGluIG5vcm1hbCBodW1hbiBjZWxsczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjE7ICBjaXRhdGlvbl9wYWdlcz0xMzU3LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIGZhY3RvcmllczsgQmlvY2hlbS4gU29jLiBUcmFucy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz01ODUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvc2UgZW5jb3VudGVycyBiZXR3ZWVuIGFjdGl2ZSBnZW5lcyBpbiB0aGUgbnVjbGV1czsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcm1pbmdsaW5nIG9mIGNocm9tb3NvbWUgdGVycml0b3JpZXMgaW4gaW50ZXJwaGFzZSBzdWdnZXN0cyByb2xlIGluIHRyYW5zbG9jYXRpb25zIGFuZCB0cmFuc2NyaXB0aW9uLWRlcGVuZGVudCBhc3NvY2lhdGlvbnM7IFBMb1MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWUxMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GcmFjdGFsIGFuZCBzdGF0aXN0aWNhbCBwcm9wZXJ0aWVzIG9mIGxhcmdlIGNvbXBhY3QgcG9seW1lcnM6IGEgY29tcHV0YXRpb25hbCBzdHVkeTsgUG9seW1lcjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTcxNy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBtYXBwaW5nIG9mIGxvbmctcmFuZ2UgaW50ZXJhY3Rpb25zIHJldmVhbHMgZm9sZGluZyBwcmluY2lwbGVzIG9mIHRoZSBodW1hbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI2OyAgY2l0YXRpb25fcGFnZXM9Mjg5LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYW5kb20gbG9vcCBtb2RlbCBmb3IgbG9uZyBwb2x5bWVyczsgUGh5cy4gUmV2LiBFIFN0YXQuIE5vbmxpbi4gU29mdCBNYXR0ZXIgUGh5cy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzY7ICBjaXRhdGlvbl9wYWdlcz0wNTE4MDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGF0aWFsbHkgY29uZmluZWQgZm9sZGluZyBvZiBjaHJvbWF0aW4gaW4gdGhlIGludGVycGhhc2UgbnVjbGV1czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MzgxMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3BvbG9naWNhbCBpbnRlcmFjdGlvbnMgYmV0d2VlbiByaW5nIHBvbHltZXJzOiBpbXBsaWNhdGlvbnMgZm9yIGNocm9tYXRpbiBsb29wczsgSi4gQ2hlbS4gUGh5cy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MDQ0OTA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVudGF0aXZlIG1hcHBpbmcgb2YgdHJhbnNjcmlwdGlvbi1pbmR1Y2VkIGludGVyY2hyb21vc29tYWwgaW50ZXJhY3Rpb24gdXNpbmcgY2hpbWVyaWMgRVNUIGFuZCBtUk5BIGRhdGE7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMjU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvYmFiaWxpc3RpYyBtb2RlbGluZyBvZiBIaS1DIGNvbnRhY3QgbWFwcyBlbGltaW5hdGVzIHN5c3RlbWF0aWMgYmlhc2VzIHRvIGNoYXJhY3Rlcml6ZSBnbG9iYWwgY2hyb21vc29tYWwgYXJjaGl0ZWN0dXJlOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEwNTktNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJN0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNPWFBSRVNkYjogYSBkYXRhYmFzZSBvZiBjb2V4cHJlc3NlZCBnZW5lIG5ldHdvcmtzIGluIG1hbW1hbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQ3Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBhbGdvcml0aG0gZm9yIGNvbXB1dGF0aW9uYWwgaWRlbnRpZmljYXRpb24gb2YgY29udGFtaW5hdGVkIEVTVCBsaWJyYXJpZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTEwNjctNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldmlldzogcHJveGltaXR5IGVmZmVjdHMgaW4gdGhlIHByb2R1Y3Rpb24gb2YgY2hyb21vc29tZSBhYmVycmF0aW9ucyBieSBpb25pemluZyByYWRpYXRpb247IEludC4gSi4gUmFkaWF0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmRpbmcgZnVzaW9uIGdlbmVzIHJlc3VsdGluZyBmcm9tIGNocm9tb3NvbWUgcmVhcnJhbmdlbWVudCBieSBhbmFseXppbmcgdGhlIGV4cHJlc3NlZCBzZXF1ZW5jZSBkYXRhYmFzZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTEzMjU3LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdXNpb24gdHJhbnNjcmlwdHMgYW5kIHRyYW5zY3JpYmVkIHJldHJvdHJhbnNwb3NlZCBsb2NpIGRpc2NvdmVyZWQgdGhyb3VnaCBjb21wcmVoZW5zaXZlIHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgdXNpbmcgcGFpcmVkLWVuZCBkaXRhZ3MgKFBFVHMpOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTgyOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9iYWJpbGlzdGljIG1vZGVsIGZvciB0aGUgYXJyYW5nZW1lbnQgb2YgYSBzdWJzZXQgb2YgaHVtYW4gY2hyb21vc29tZSB0ZXJyaXRvcmllcyBpbiBXSTM4IGh1bWFuIGZpYnJvYmxhc3RzOyBKLiBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIyMTsgIGNpdGF0aW9uX3BhZ2VzPTEyMC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZWxsIHR5cGUgc3BlY2lmaWMgY2hyb21vc29tZSB0ZXJyaXRvcnkgb3JnYW5pemF0aW9uIGluIHRoZSBpbnRlcnBoYXNlIG51Y2x1cyBvZiBub3JtYWwgYW5kIGNhbmNlciBjZWxsczsgSi4gQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjE7ICBjaXRhdGlvbl9wYWdlcz0xMzAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZnVzaW9uLWRyaXZlbiBsb29waW5nIHByb3ZpZGVzIGEgY29uc2lzdGVudCBmcmFtZXdvcmsgZm9yIGNocm9tYXRpbiBvcmdhbml6YXRpb247IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTIyMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvMS80NS8xMDg1ODEzL2RzczAzMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVHJhbnNjcmlwdGlvbi1pbmR1Y2VkIGNoaW1hZXJpYyB0cmFuc2NyaXB0cywgdGhlIHBvdGVudGlhbCBwb3N0LXRyYW5zY3JpcHRpb25hbCBwcm9jZXNzaW5nIHByb2R1Y3RzLCBtaWdodCByZWZsZWN0IHRoZSBzcGF0aWFsIHByb3hpbWl0eSBvZiBhY3RpdmVseSAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM4MjkyOCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGQli50UorbOn70N2chDzFd9EVJ1QA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f45%2f382928" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f45%2f382928" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1509238849&amp;Signature=PSAuPLfsuhNKNMQR1FaKGKeXnYxa9VWWxVwVj8QgWwiWyxXnuqrnLzZcJc2CBUJltLVrinKxSB5nYhbXjUM9-je4LPvkdGahFY~o-WdQJK7WT08AL~8WvaZoAYAn96Q79t1-TpryREPtJCpPrqK7WAJ4VgdOJhwTB6p3hzdwKtg4L9c5bC-56h6Cf-rVzA9WfP1pApnAfqLDfc4Hxr8AkFr9iuyQDmP0~HPWyaQt67XCS7L1KewivcvyYV4gCPFA8PQEj~5vMJtfUjMZVsPsxoOYWtrZUgaCpQPDDd88T8mjJZgtwXU6D7a5ZHr1hjQj4sdG24HsvRLOqQWKflk3Jg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5618468">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618468">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618470">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618470">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618475">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618475">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618499">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618499">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618544">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618544">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618546">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618546">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618549">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618549">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5618553">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5618553">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/1/31/382624">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/1/55/383179">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Songling Li</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Songling Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Theoretical Biophysics Group, Institute for Theoretical Physics</div>, <div class="insititution">University of Heidelberg</div>, <div class="addr-line">Heidelberg</div>, <div class="country">Germany</div></div><div class="aff"><div class="label -label">4</div><div class="insititution">Shanghai Center for Bioinformation Technology (SCBIT)</div>, <div class="addr-line">Shanghai</div>, <div class="country">P.R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Songling+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Li author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Dieter W. Heermann</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dieter W. Heermann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Theoretical Biophysics Group, Institute for Theoretical Physics</div>, <div class="insititution">University of Heidelberg</div>, <div class="addr-line">Heidelberg</div>, <div class="country">Germany</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">The Jackson Laboratory</div>, <div class="addr-line">Bar Harbor, Maine</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Shanghai Institutes for Biological Sciences (SIBS)</div>, <div class="addr-line">Shanghai</div>, <div class="country">P.R. China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +49-6221-549426, Email: <a href="mailto:heermann@tphys.uni-heidelberg.de" target="_blank">heermann@tphys.uni-heidelberg.de</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Dieter+W.+Heermann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Heermann D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Heermann author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 45–53, <a href='https://doi.org/10.1093/dnares/dss032'>https://doi.org/10.1093/dnares/dss032</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 December 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 March 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="382928" href="/dnaresearch/article-pdf/20/1/45/1085813/dss032.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Songling Li, Dieter W. Heermann;  Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts, <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 45–53, <a href='https://doi.org/10.1093/dnares/dss032'>https://doi.org/10.1093/dnares/dss032</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/382928?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/382928?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/382928?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/382928?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/382928?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/382928?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/382928?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Using%20Chimaeric%20Expression%20Sequence%20Tag%20as%20the%20Reference%20to%20Identify%20Three-Dimensional%20Chromosome%20Contacts&amp;publicationDate=2012-12-04&amp;volumeNum=20&amp;issueNum=1&amp;author=Li%2C%20Songling%3B%20Heermann%2C%20Dieter%20W.&amp;startPage=45&amp;endPage=53&amp;contentId=10.1093%2Fdnares%2Fdss032&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f45%2f382928" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5618468 id="5618468" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Transcription-induced chimaeric transcripts, the potential post-transcriptional processing products, might reflect the spatial proximity of actively transcribed genes co-localized in transcription factories. A growing number of expression data deposited in databases provide us with the raw material for screening such chimaeric transcripts and using them as the probes to identify interactions between genes in <em>cis</em> or in <em>trans</em>. Based on the high-quality chimaeric transcripts gleaned from human expression sequence tag data with selection criteria, we identified the patterns of inter- and intrachromosomal gene–gene interactions. On top the contact pattern from interchromosomal interactions, we also observed an exponential behaviour of the intrachromosomal interactions within a certain length scale, which is consistent with the independent experimental results from Hi-C screening and with the Random Loop Model. A compatible result is found for mouse. Transcription-induced chimaeric transcripts, most of which might be accidental products with trivial functions, shed light on the spatial organization of chromosomes. These inter- and intrachromosomal interactions might contribute to the compaction of chromosomes, their segregation and formation of the chromosome territories, and their spatial distribution within the nucleus.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chimaeric expression sequence tag (EST)&quot;">chimaeric expression sequence tag (EST)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factory&quot;">transcription factory</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;inter- and intrachromosomal interaction&quot;">inter- and intrachromosomal interaction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chromatin">chromatin</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromatin loop&quot;">chromatin loop</a></div>                            <h2 scrollto-destination=5618470 id="5618470" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Transcription-induced chimerism (TIC)<sup><a reveal-id="DSS032C1" data-open="DSS032C1" class="link link-ref link-reveal xref-bibr">1</a></sup> gives rise to chimaeric transcripts consisting of heterogeneous sequence segments originating from distant regions on the same chromosome or from two or more distinct chromosomes. Chimaeric transcripts from large-scale transcription databases were previously considered as contaminants. However, the increasing number of chimaeric transcripts has indeed been verified <em>in vivo</em>.<sup><a reveal-id="DSS032C2" data-open="DSS032C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Thus, there is good reason to take chimaeric transcripts seriously. Trans-splicing-mediated chimaeric transcripts have been widely observed in a variety of organisms. Trans-splicing is more frequently observed in lower unicellular eukaryotes and <em>Caenorhabditis elegans</em> and shows physiological significances.<sup><a reveal-id="DSS032C3" data-open="DSS032C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS032C4" data-open="DSS032C4" class="link link-ref link-reveal xref-bibr">4</a></sup> One of the most prominent examples involves <em>Ciona intestinalis</em>, in which the number of non-trans-spliced and trans-spliced genes is approximately equivalent.<sup><a reveal-id="DSS032C5" data-open="DSS032C5" class="link link-ref link-reveal xref-bibr">5</a></sup> In <em>Drosophila</em> and mammalian cells, however, trans-splicing is less common, and the functional importance of most trans-spliced products is unclear.<sup><a reveal-id="DSS032C6" data-open="DSS032C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS032C7" data-open="DSS032C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Later, Li et al. proposed a transcriptional-slippage model in light of short homologous sequences (SHSs) at junction sites in between fusion sequences.<sup><a reveal-id="DSS032C8" data-open="DSS032C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>Although most chimaera from higher eukaryotes are non-functional, those potentially encoded fusion genes could exert negative effects by competing with their parental genes,<sup><a reveal-id="DSS032C9" data-open="DSS032C9" class="link link-ref link-reveal xref-bibr">9</a></sup> or serving as an oncogenic transcription factor via mistargeting on the chromosome, and altering chromatin structure.<sup><a reveal-id="DSS032C10" data-open="DSS032C10" class="link link-ref link-reveal xref-bibr">10</a></sup> It is assumed that chimaeric transcripts expressed in normal and transformed cell might be mediated by two distinct mechanisms (transcription induced or chromosomal translocation induced),<sup><a reveal-id="DSS032C11" data-open="DSS032C11" class="link link-ref link-reveal xref-bibr">11</a></sup> either of which might involve physical proximity between gene loci. It is reasonable to assume that because different sequence modules of transcription-induced chimaeric transcripts need to be conjugated, their expression should be concomitant, and that distinct pre-RNA products should be confined in a specific cell subcompartment with close three-dimensional (3D) proximity. The candidate of such structure might be the transcription factory.<sup><a reveal-id="DSS032C12" data-open="DSS032C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The fact that fewer transcription factories exist than the number of expressed genes indicates that multiple actively expressed genes need to share the same transcriptional machinery and that genes separated by a long genomic distance can migrate to each other closely.<sup><a reveal-id="DSS032C13" data-open="DSS032C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Thus, spatial proximity of distinct transcripts in the transcription factories might facilitate the production of chimaeric transcripts that in turn can be used as the signal of chromosome loci contact.</p><p>Various polymer models have been postulated to describe the folding of compact but flexible chromatin fibre, which is crucial to gene expression activity.<sup><a reveal-id="DSS032C14" data-open="DSS032C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Among these models, the fractal globule model<sup><a reveal-id="DSS032C15" data-open="DSS032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> proposes that chromatin is folded initially into crumples and repeatedly into crumples-of-crumples. Some experimental support for this model comes from the <em>in vivo</em> screening on the chromatin contact with the Hi-C method developed by Lieberman-Aiden et al.<sup><a reveal-id="DSS032C16" data-open="DSS032C16" class="link link-ref link-reveal xref-bibr">16</a></sup> An alternative view comes from studies,<sup><a reveal-id="DSS032C17  DSS032C19" data-open="DSS032C17  DSS032C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> where a chromosome is viewed as an all scales-looped polymer.</p>
  <p>From the evidence presently available, a fusion process of heterogeneous transcripts is closely related to the spatial proximity of actively transcribed genes. In 2007, Unneberg et al. published the mapping results of transcription-induced interchromosomal interactions using the chimaeric expression sequence tag (EST) as the probe.<sup><a reveal-id="DSS032C20" data-open="DSS032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> After that, the increased number of data deposited in the databases has made possible a comprehensive study of chimaera and its relation to the 3D organization of chromosomes. Here, with chimaeric transcripts gleaned from human and mouse EST databases, we test the idea of TIC in terms of gene pairs' co-expression pattern. Comparing with normalized Hi-C contact maps<sup><a reveal-id="DSS032C21" data-open="DSS032C21" class="link link-ref link-reveal xref-bibr">21</a></sup> also confirms that these chimaeric genes are more likely to be physically proximal. Unlike the previous study result,<sup><a reveal-id="DSS032C20" data-open="DSS032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> we show that the arrangement of chromosomes within the cell nucleus is non-random. No evident changes, however, can be observed from the interchromosomal interaction patterns of normal and the tumour cells. Based on the gene pairs located at every single chromosome, we obtain the relationship between the genomic distance and the gene contact frequencies. A functional behaviour (<span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss032ileq1.gif?Expires=1509238856&amp;Signature=UHqRyK3h-NngV4HHFharQTH4OvET206hsYLzt-ZMoLJlwh54nz-plFT8e3lz2XZmS6cOl8p4LcZ6NKNAvNfsDudlaMQF9~TrhQjQSzxHQX4n15uU6FIePwBK~tcnffIIjPXtgzQwPlRrYoDNjd-Jlf71tAHIRVPv8-D6YpayxaS8rpYU5BZ41mDs-JnuJ3PcDzLCwpJ3ewl9FCwFjXdcQ~Dw5N60rkT-Hn1am56CF3TMvihegWuJStxhH1rcW9gsFIf5ORYTCPfIiaohFLX4q1NLwzyfvXoSEj0OuyvxTVI-KVoBOnt0LX~xXlM8607R7loJuSxNpmr367D3uA6uUg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss032ileq1.gif" /></span>, where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss032ileq2.gif?Expires=1509238856&amp;Signature=NlHh7F1GQmBvtZS1a2s~wErM7yXsvdKLbrMrX5Gjj4aaTfVy~hvBrGq~t0OZo37zC8yJFEYuaCKBq2OMZ2TMs-SqOy9AGQjPolN89yrCvnbCkB5nSE4Es7LkUq1aqObsXANG8uCP2nJSbaxgzccEURa7Kb90BX2BqxOvvrqbSxLpc7OPUxn~H1akI48PzYWCNVxJbin6gJur-nJzHcFojE5zl7JsYif83CB1rATjwMuMH32Mq7U8ojIASq0QA2MZKNt7z2FMIuksbb~2wGuMYuzJySoLJxS9d26OnSBnnmzDm8NbtmIBejhkbHaOy3NMoJH6FBaqvA-gKKtqOsWA1w__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss032ileq2.gif" /></span> is the probability of contact between two loci that are genomically separated by <em>x</em> base pairs) is observed from our data with the genomic distance ranging from 500 to 7–8 Mb and an exponent <em>α</em> = −1 on a double logarithmic plot that agrees with the experimental results from Hi-C screening.<sup><a reveal-id="DSS032C16" data-open="DSS032C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>
                            <h2 scrollto-destination=5618475 id="5618475" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5618476 id="5618476" class="section-title" data-legacy-id=s2a>2.1. Chimaeric transcripts identification</h3>
<p>mRNA data were gathered from gbEST, gbHTC, and gbPRI divisions of the National Center for Biotechnology Information (NCBI) Genbank database (Release 186). Genome sequences of human and mouse were downloaded from NCBI Genome database (Release 37). EST library annotation was obtained from NCBI ftp as well. Gene annotation was downloaded from Ensembl genome database (Version 64).</p><p>We used BLAT<sup><a reveal-id="DSS032C22" data-open="DSS032C22" class="link link-ref link-reveal xref-bibr">22</a></sup> for sequence alignment, with the minimal identity threshold of 95% and the rest of the parameters at their default values. After aligning, alignments in each query were sorted according to the score and then the identity.</p><p>Similar to previous studies,<sup><a reveal-id="DSS032C8" data-open="DSS032C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS032C20" data-open="DSS032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> we picked out the queries with top two ranked alignments having the sequences identity not ˂95% and minimal alignment length not ˂50 bp for gbEST sequences. For much longer gbHTC and gbPRI sequences, a threshold of 100 bp was used. A 10 bp overlapping at the binding site of the query was allowed because of the uncertainty in aligning. To make sure that the two best alignments in a query were unambiguously aligned, a uniqueness criterion was imposed.<sup><a reveal-id="DSS032C20" data-open="DSS032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Chimaeric transcripts with their boundary sites containing recognition sequences of the restriction enzymes used for library construction were removed. Similar to a previous study,<sup><a reveal-id="DSS032C8" data-open="DSS032C8" class="link link-ref link-reveal xref-bibr">8</a></sup> SHSs on the boundary sites of these retained transcripts were checked.</p>                            <h3 scrollto-destination=5618485 id="5618485" class="section-title" data-legacy-id=s2b>2.2. Gene mapping</h3>
<p>We mapped the fusion sequences onto the chromosomes according to the Ensembl Genome database and selected transcripts whose partners were both mapped onto the gene regions with the same directions as the annotation. For sequences with both partners mapped with the opposite directions of the corresponding genes, we re-evaluated their directions according to the GT/AG rule. Considering the possibility of tandem transcription or intergenic splicing, we removed the chimaeric transcripts in <em>cis</em> with genomic distance between gene loci smaller than 500 kb.</p>                            <h3 scrollto-destination=5618489 id="5618489" class="section-title" data-legacy-id=s2c>2.3. Co-expression pattern of the chimaeric gene partners</h3>
<p>Co-expression pattern of human and mouse genes was obtained from COXPRESdb (Version 4.1),<sup><a reveal-id="DSS032C23" data-open="DSS032C23" class="link link-ref link-reveal xref-bibr">23</a></sup> in which genes were indexed by their Entrez Gene IDs, and the Mutual Rank (MR) score was used as a measure for gene co-expression correlation. R package ‘biomaRt’ (Version 2.10.0) was used to convert the Ensembl Gene IDs of those mapped genes to the Entrez Gene IDs. To check whether the co-expression correlation of gene pairs in the chimaeric transcripts is non-preferential, MR scores of all possible gene pairs were used as a background.</p>                            <h3 scrollto-destination=5618494 id="5618494" class="section-title" data-legacy-id=s2d>2.4. Comparison with normalized Hi-C contact pattern</h3>
<p>We used normalized Hi-C contact pattern of human lymphoblast generated by Yaffe et al.<sup><a reveal-id="DSS032C21" data-open="DSS032C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The ratio of the observed to the expected contact number in each 1 Mb bin was transferred by the logarithm to base 2 as a measurement of contact enhancement. The larger the enhancement value for a bin pair is, the more the contacts between them can be observed than by chance. Consequently, an enhanced contact probability indicates a closer spatial relationship. Gene pairs of the chimaeric transcripts were mapped onto the corresponding bins of the chromosomes, and the enhancement was screened. For genes spanning over more than one bin, the contact enhancement was calculated by averaging weighted contact enhancement for each 1 Mb bin pair covered. The enhancement distributions of all possible inter- and intracontacts were used as backgrounds. The Kolmogorov–Smirnov test was applied to check the difference of the contact enhancement between chimaeric gene pairs and the background.</p>                            <h3 scrollto-destination=5618496 id="5618496" class="section-title" data-legacy-id=s2e>2.5. Inter- and intrachromosomal interactions pattern analysis</h3>
<p>Chimaeric transcripts extracted from the gbEST database were used for interaction pattern analysis. Chimaeras involving the same gene partners were grouped together so as to remove the overcounts. We picked out transcripts composed of two genes labelled by ‘protein coding’ biotype in the human Ensembl database, which were far more abundant than others, as the observed interchromosomal interaction pattern. We collected the frequencies of contact between two given chromosomes to obtain the interaction matrix and plotted the data as a heat map. From the human Ensembl database, we obtained the frequencies of protein coding genes on 23 chromosomes except chromosome Y. With these expected gene frequencies, we constructed a random matrix of chromosome pair-wise interaction by multiplying the gene frequencies from any two given chromosomes. For this symmetric matrix, each element in the upper half of the matrix was multiplied by a random number, and the corresponding element in the lower half was assigned the same value. This finished a shuffle step on the interaction pattern, and the eigenvalue of the matrix was calculated. Totally, 10<sup>5</sup> steps were carried out. After that, the mean eigenvalue was obtained and plotted against the eigenvalue from the observation. Furthermore, based on the cell source annotation from the human EST library report, we separated the interactions from normal cells from those from tumour cells and plotted the eigenvalue of interaction for comparison.</p><p>For intrachromosomal interactions, we made statistics on the frequencies of the gene–gene interaction under given genomic distance within every chromosome and plotted the frequencies of the interactions versus the genomic distance in a double logarithmic plot. For human, the contact probabilities were divided by their corresponding genomic distance for normalization and replotted against the genomic distance.</p>                            <h2 scrollto-destination=5618499 id="5618499" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5618500 id="5618500" class="section-title" data-legacy-id=s3a>3.1. Chimaeric transcripts validation and genome mapping</h3>
<p>The number of chimaera selected from gbEST, gbPRI, and gbHTC databases is given in Table <a reveal-id="DSS032TB1" data-open="DSS032TB1" class="link link-reveal link-table xref-fig">1</a>. Detailed information about the validated chimaeric transcripts can be found in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data</a>. We analysed the distribution of human chimaeric transcripts among the EST libraries. Of 8995 human libraries screened, 2617 (29%) were involved in the production of chimaera, and none of them overlapped with those highly contaminated libraries.<sup><a reveal-id="DSS032C24" data-open="DSS032C24" class="link link-ref link-reveal xref-bibr">24</a></sup> As shown in Fig. <a reveal-id="DSS032F1" data-open="DSS032F1" class="link link-reveal link-table xref-fig">1</a>A, most libraries contributed very few chimaera. One thousand one hundred and eleven and 518 libraries from a total of 2617 human libraries provided only one and two chimaera, respectively. A compatible result is found for mouse (Fig. <a reveal-id="DSS032F1" data-open="DSS032F1" class="link link-reveal link-table xref-fig">1</a>B). Such a result indicates that chimaera formation is an ubiquitous event with low frequency. </p>                        <a id="5618502" scrollto-destination="5618502"></a>
<div class="table-wrap"><div id="DSS032TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of the chimaeric transcripts identified from different databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Source </th><th colspan="2">GG<hr /> </th><th colspan="2">UG<hr /> </th><th colspan="2">Genes<hr /> </th><th>Size </th><th>Overlap </th></tr></thead><tbody><tr><td>Human gbEST </td><td>11 262 </td><td>773 </td><td>8261 </td><td>673 </td><td>7871 </td><td>1165 </td><td>518 </td><td> </td></tr><tr><td>Human gbHTC </td><td>282 </td><td>31 </td><td>266 </td><td>29 </td><td>508 </td><td>58 </td><td>2634 </td><td>51 </td></tr><tr><td>Human gbPRI </td><td>1094 </td><td>163 </td><td>657 </td><td>109 </td><td>1141 </td><td>208 </td><td>15 079 </td><td>72 </td></tr><tr><td>Mouse gbEST </td><td>2493 </td><td>269 </td><td>2230 </td><td>223 </td><td>3316 </td><td>426 </td><td>582 </td><td> </td></tr><tr><td>Mouse gbHTC </td><td>328 </td><td>32 </td><td>322 </td><td>27 </td><td>599 </td><td>54 </td><td>1846 </td><td>71 </td></tr></tbody></table></div><div content-id="DSS032TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Source </th><th colspan="2">GG<hr /> </th><th colspan="2">UG<hr /> </th><th colspan="2">Genes<hr /> </th><th>Size </th><th>Overlap </th></tr></thead><tbody><tr><td>Human gbEST </td><td>11 262 </td><td>773 </td><td>8261 </td><td>673 </td><td>7871 </td><td>1165 </td><td>518 </td><td> </td></tr><tr><td>Human gbHTC </td><td>282 </td><td>31 </td><td>266 </td><td>29 </td><td>508 </td><td>58 </td><td>2634 </td><td>51 </td></tr><tr><td>Human gbPRI </td><td>1094 </td><td>163 </td><td>657 </td><td>109 </td><td>1141 </td><td>208 </td><td>15 079 </td><td>72 </td></tr><tr><td>Mouse gbEST </td><td>2493 </td><td>269 </td><td>2230 </td><td>223 </td><td>3316 </td><td>426 </td><td>582 </td><td> </td></tr><tr><td>Mouse gbHTC </td><td>328 </td><td>32 </td><td>322 </td><td>27 </td><td>599 </td><td>54 </td><td>1846 </td><td>71 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The GG column represents the number of chimaeric transcripts whose partners were both mapped onto the gene regions with the same directions as the annotation. The UG column indicates the unique gene pairs identified from chimaeric transcripts of gene–gene interaction. The genes column corresponds to the number of genes participating in the chimaera production. Under GG, UG, and genes columns, left subcolumns indicate the number for interchromosomal interaction and the right ones denote intrachromosomal interaction. The size column shows the average length (unit in base pairs) of validated inter- and intrachromosomal chimaeric transcripts. And, the overlap column indicates the number of chimaeric gene pairs identified from the gbHTC or gbPRI database, which can also be found at the gbEST database.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5618502" target="_blank">View Large</a></div></div>                        <a id="5618504" scrollto-destination="5618504"></a>
<div id="DSS032F1" data-id="DSS032F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03201.jpeg?Expires=1509238856&Signature=V75p0fqi6QWHr6TfC3mrFh3FJKoVvS0dpgjCYxG4NIXbZSNcAZDNhRD9SP17buN5wl~~94OSHfmburSjZckjrggI5AQjz8HAuF3emlP8hmblIM1FwHpJXgt-0VQoMbvCPZdiTSYtrv3gbarYKQ0YwA~WI1EH6RI24HjddvYY~-cMbZZuMh95Llbatb3yvvA27vkSjQceOa5rW9Rwgwt50jmil1GB7c-ZIwSH9piBIbp~ejdmFEwKu4hu4oc2lG2hsOdIHIuKwp9Gtnc6eqdW-QSgKv6LCoLdz2UyTtUaST9mc6cr23sCkw867kFTmQqaE8IqJauC8yiR-KGPamoGrw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The distribution of human and mouse EST libraries according to the number of chimaera they produced. (A) for human and (B) for mouse. The x-axis represents the number of chimaera that can be identified from single library. The y-axis indicates the number of libraries producing the corresponding amount of chimaeric ESTs." path-from-xml="dss03201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618504/dss03201.jpeg" path-from-xml="dss03201.jpeg" target="_blank">View large</a><a section="5618504" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03201.jpeg?Expires=1509238856&Signature=Mj71fzAdYn8ywSut1Dmlqs-GaKNMx0aiELOnzYhSm9rj2VKY6BsfQTclQ1zcHFdS1vc2GRBDJUiWaOQ~z6qzBQFmsDcV87N7rj4oYsKcXS~XiS~u82Ex-dZuxuNgjy0Tt3ua~Qj77wibimKWW2vMNZEu4nc0ov1w50l7Ig4SziAAw5~NZh6BuQL-Jo-z3GpuPvYQfKnLmouPEm-Mq5MLTa1k-UoW9YN-6qKDYI40~pD3SjjQ5sSVtbZrwaE6W27ejn4P-b-rn9-H0b9bZEzt~hiTAyrUgJwYIMqKEaJrlk8aBORSDryhQhY3nnf-geXlZQd5DVTnNJxOZl4M5QosLQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618504&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of human and mouse EST libraries according to the number of chimaera they produced. (A) for human and (B) for mouse. The <em>x</em>-axis represents the number of chimaera that can be identified from single library. The <em>y</em>-axis indicates the number of libraries producing the corresponding amount of chimaeric ESTs.</p></div></div><div content-id="DSS032F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03201.jpeg?Expires=1509238856&Signature=V75p0fqi6QWHr6TfC3mrFh3FJKoVvS0dpgjCYxG4NIXbZSNcAZDNhRD9SP17buN5wl~~94OSHfmburSjZckjrggI5AQjz8HAuF3emlP8hmblIM1FwHpJXgt-0VQoMbvCPZdiTSYtrv3gbarYKQ0YwA~WI1EH6RI24HjddvYY~-cMbZZuMh95Llbatb3yvvA27vkSjQceOa5rW9Rwgwt50jmil1GB7c-ZIwSH9piBIbp~ejdmFEwKu4hu4oc2lG2hsOdIHIuKwp9Gtnc6eqdW-QSgKv6LCoLdz2UyTtUaST9mc6cr23sCkw867kFTmQqaE8IqJauC8yiR-KGPamoGrw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The distribution of human and mouse EST libraries according to the number of chimaera they produced. (A) for human and (B) for mouse. The x-axis represents the number of chimaera that can be identified from single library. The y-axis indicates the number of libraries producing the corresponding amount of chimaeric ESTs." path-from-xml="dss03201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618504/dss03201.jpeg" path-from-xml="dss03201.jpeg" target="_blank">View large</a><a section="5618504" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03201.jpeg?Expires=1509238856&Signature=Mj71fzAdYn8ywSut1Dmlqs-GaKNMx0aiELOnzYhSm9rj2VKY6BsfQTclQ1zcHFdS1vc2GRBDJUiWaOQ~z6qzBQFmsDcV87N7rj4oYsKcXS~XiS~u82Ex-dZuxuNgjy0Tt3ua~Qj77wibimKWW2vMNZEu4nc0ov1w50l7Ig4SziAAw5~NZh6BuQL-Jo-z3GpuPvYQfKnLmouPEm-Mq5MLTa1k-UoW9YN-6qKDYI40~pD3SjjQ5sSVtbZrwaE6W27ejn4P-b-rn9-H0b9bZEzt~hiTAyrUgJwYIMqKEaJrlk8aBORSDryhQhY3nnf-geXlZQd5DVTnNJxOZl4M5QosLQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618504&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of human and mouse EST libraries according to the number of chimaera they produced. (A) for human and (B) for mouse. The <em>x</em>-axis represents the number of chimaera that can be identified from single library. The <em>y</em>-axis indicates the number of libraries producing the corresponding amount of chimaeric ESTs.</p></div></div><p>From validated chimaera, most gene interactions were observed only once, whereas some gene partners associated with each other more frequently. In addition, some of these reoccurring chimaera can be identified from multiple different EST libraries, whereas other repeats originated only from a single library. Detailed mapping results are shown in Table <a reveal-id="DSS032TB1" data-open="DSS032TB1" class="link link-reveal link-table xref-fig">1</a>. The abundance of SHSs with different size is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>.</p>                            <h3 scrollto-destination=5618510 id="5618510" class="section-title" data-legacy-id=s3b>3.2. Co-expression pattern of the chimaeric gene partners</h3>
<p>Of the total of 8934 validated human chimaeric transcripts, 7412 gene partners with both genes Entrez Gene IDs known were collected. For mouse, 2229 gene pairs were obtained from 2453 chimaeras. As shown in Fig. <a reveal-id="DSS032F2" data-open="DSS032F2" class="link link-reveal link-table xref-fig">2</a>, gene pairs from the chimaeric transcripts associated more frequently with small MR score than with a large one, when compared with the co-expression pattern of all possible gene pairs. This observation revealed that gene pairs producing chimaeric transcripts showed stronger positive correlation in terms of their expression activities. This observation further supported the assumption that lots of identified chimaeric transcripts, if not all were transcription induced.<sup><a reveal-id="DSS032C2" data-open="DSS032C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p>                        <a id="5618515" scrollto-destination="5618515"></a>
<div id="DSS032F2" data-id="DSS032F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03202.jpeg?Expires=1509238856&Signature=L7MdEyrT8u0qdTwaHj6Ob3uc5QbFGWgZ~nhWxRvCPrkeQ88h6skoPDNpzkgRrrJrZpuF628r6b9dn~hliqpELKnBZM9K~JI8PhX33KsnHYay8NbMgn4ULfQawmhoxAK~IHMHvnsONsCZDxqgfjrCz88n2Q9c-EU616lDTrDxbpyzZCup-aaV-kVhSxRw-Wd0XldcgVuptn9k1yZXdpKFlpaKLl50qm48ZbgkqwACAGo8UuJi1dTpyIFqnK5DWFXkJzEtqCxy3jZuJ-xFyGZDxrzI11trCDeS8wbD~8styGQQ8Wda5XBnReSpPUsxew~H54gXLJ-JZk8yllUTh2a8bw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Co-expression pattern of genes from the chimaeric transcripts. (A) for human and (B) for mouse. The smaller the MR score is, the higher is the correlation between gene pairs' expression." path-from-xml="dss03202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618515/dss03202.jpeg" path-from-xml="dss03202.jpeg" target="_blank">View large</a><a section="5618515" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03202.jpeg?Expires=1509238856&Signature=JtD~hLQrZtv5vHSjoansAkQKEg3MW~jsqg7iD1J7AWDO3-TOfVY28Vf9zDcH5wOpnR5b1oV-XXbojqBrBkiEDz3gdxxMw1kzjAIcqZJSoO0IxtPvLyX0LPj1uOCTr3gxOddyk-5UL7jMncy78ByQM6bhDrC6qbepYm5G3YzwnRpla~XDW3KSi4LmNihHnXIpeWr9gHyjlO97fmH0oDoNE2giOUIDIk1kLUKZ66p8QH3F-4tsdtoExMNcwd2sQ9d8AddSoIp7SbHvhqseGuPjpye6n74BRo743tKuVo1VsyuEYaB48l3c-YXeNp-joKZgJIjb~RN6MyRUbL4wRscpSA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618515&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Co-expression pattern of genes from the chimaeric transcripts. (A) for human and (B) for mouse. The smaller the MR score is, the higher is the correlation between gene pairs' expression.</p></div></div><div content-id="DSS032F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03202.jpeg?Expires=1509238856&Signature=L7MdEyrT8u0qdTwaHj6Ob3uc5QbFGWgZ~nhWxRvCPrkeQ88h6skoPDNpzkgRrrJrZpuF628r6b9dn~hliqpELKnBZM9K~JI8PhX33KsnHYay8NbMgn4ULfQawmhoxAK~IHMHvnsONsCZDxqgfjrCz88n2Q9c-EU616lDTrDxbpyzZCup-aaV-kVhSxRw-Wd0XldcgVuptn9k1yZXdpKFlpaKLl50qm48ZbgkqwACAGo8UuJi1dTpyIFqnK5DWFXkJzEtqCxy3jZuJ-xFyGZDxrzI11trCDeS8wbD~8styGQQ8Wda5XBnReSpPUsxew~H54gXLJ-JZk8yllUTh2a8bw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Co-expression pattern of genes from the chimaeric transcripts. (A) for human and (B) for mouse. The smaller the MR score is, the higher is the correlation between gene pairs' expression." path-from-xml="dss03202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618515/dss03202.jpeg" path-from-xml="dss03202.jpeg" target="_blank">View large</a><a section="5618515" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03202.jpeg?Expires=1509238856&Signature=JtD~hLQrZtv5vHSjoansAkQKEg3MW~jsqg7iD1J7AWDO3-TOfVY28Vf9zDcH5wOpnR5b1oV-XXbojqBrBkiEDz3gdxxMw1kzjAIcqZJSoO0IxtPvLyX0LPj1uOCTr3gxOddyk-5UL7jMncy78ByQM6bhDrC6qbepYm5G3YzwnRpla~XDW3KSi4LmNihHnXIpeWr9gHyjlO97fmH0oDoNE2giOUIDIk1kLUKZ66p8QH3F-4tsdtoExMNcwd2sQ9d8AddSoIp7SbHvhqseGuPjpye6n74BRo743tKuVo1VsyuEYaB48l3c-YXeNp-joKZgJIjb~RN6MyRUbL4wRscpSA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618515&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Co-expression pattern of genes from the chimaeric transcripts. (A) for human and (B) for mouse. The smaller the MR score is, the higher is the correlation between gene pairs' expression.</p></div></div>                            <h3 scrollto-destination=5618517 id="5618517" class="section-title" data-legacy-id=s3c>3.3. Comparison with normalized Hi-C contact pattern</h3>
<p>From the enhancement distributions shown in Fig. <a reveal-id="DSS032F3" data-open="DSS032F3" class="link link-reveal link-table xref-fig">3</a>, both for inter- and intrachromosomal chimaeric transcripts, a shift towards larger enhancement can be seen with either kind of restriction enzyme. For intrachromosomal contacts, a large enhancement can be see for the tail, i.e. for gene pairs that are several mega base far away. Although the occurrence of long-range interactions is relatively low, their contact probabilities are much higher than expectation, which suggests a closer spatial distance between genome regions involved. From the cumulative distribution function (CDF) curves shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>, one can clearly see that the CDF of observation lies below that of background, which indicates that a higher contact probability between chimaeric genes is given over that of the random case. </p>                        <a id="5618520" scrollto-destination="5618520"></a>
<div id="DSS032F3" data-id="DSS032F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03203.jpeg?Expires=1509238856&Signature=UvRkPgNZviGYyPcSTCmwb3qzcb0n5GLfbZ8nXvx55mwRggMLUTiCeCi7qFCYjQT11UJjf9gwZeTF9UDMGdAi1RAGAtcAW84VuSXwL5pl2XOfec76fRKWlXiL388cq8hBBJaPdm77CCMog-p7VSIaAh8g2u8cBIVwmSGf-isRf2fdtzUmD6~YJpNuELlg70Fx5iAUAeZbZ5cEd8y28zCIDg4tc--qybthMn9HuIVJmsEkBgbsZ9dUuMAwx~O9snt32QxEpSY8iLOqjsQ80FLo2ActcB2pDv5NM9bJa-hzMrUOw8Msp2LCUDi9xw~MPkXavRUPZzZtvfZkq9yzkCpixg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparison with normalized Hi-C contact pattern. Patterns (A) and (B) were extracted from the contact map restricted by the enzyme HindIII, (A) for intra- and (B) for interchromosomal contact; Patterns (C) and (D) with restriction enzyme NcoI, (C) for intra- and (D) for interchromosomal contact." path-from-xml="dss03203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618520/dss03203.jpeg" path-from-xml="dss03203.jpeg" target="_blank">View large</a><a section="5618520" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03203.jpeg?Expires=1509238856&Signature=Ubu~c7-rw-ZqfhpUa9A7GolmvWszjpG3xlJELGQJIFP8vwi~wfIsyULjigl3grsh6dycfA4mKAUo3-WVo7lzo~lgHQazx2-dvQPjewwcmWkdZ94ne5BxV2-45MqXJwPwXdOGnf59HKX2Aj5drRvZ7ZcGeTsbHMBaMvT5oEcrRIGPKikYU2xQNtCFNhMkkrRvO~N0Jvla2XenxRRc2lnRLT9rnW8p77Qwm7UInMeU91rjWGcewN9HqwLZUTg6L9Lu68CzcE7eDcnGr1MPw0wcCN3hyWZHykviuYBQWs9KVZNXxetFUVoMpNY~JkcXNsuIl3zO7awzHoYkz3yIS~qKNg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618520&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison with normalized Hi-C contact pattern. Patterns (A) and (B) were extracted from the contact map restricted by the enzyme <em>Hin</em>dIII, (A) for intra- and (B) for interchromosomal contact; Patterns (C) and (D) with restriction enzyme <em>Nco</em>I, (C) for intra- and (D) for interchromosomal contact.</p></div></div><div content-id="DSS032F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03203.jpeg?Expires=1509238856&Signature=UvRkPgNZviGYyPcSTCmwb3qzcb0n5GLfbZ8nXvx55mwRggMLUTiCeCi7qFCYjQT11UJjf9gwZeTF9UDMGdAi1RAGAtcAW84VuSXwL5pl2XOfec76fRKWlXiL388cq8hBBJaPdm77CCMog-p7VSIaAh8g2u8cBIVwmSGf-isRf2fdtzUmD6~YJpNuELlg70Fx5iAUAeZbZ5cEd8y28zCIDg4tc--qybthMn9HuIVJmsEkBgbsZ9dUuMAwx~O9snt32QxEpSY8iLOqjsQ80FLo2ActcB2pDv5NM9bJa-hzMrUOw8Msp2LCUDi9xw~MPkXavRUPZzZtvfZkq9yzkCpixg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparison with normalized Hi-C contact pattern. Patterns (A) and (B) were extracted from the contact map restricted by the enzyme HindIII, (A) for intra- and (B) for interchromosomal contact; Patterns (C) and (D) with restriction enzyme NcoI, (C) for intra- and (D) for interchromosomal contact." path-from-xml="dss03203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618520/dss03203.jpeg" path-from-xml="dss03203.jpeg" target="_blank">View large</a><a section="5618520" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03203.jpeg?Expires=1509238856&Signature=Ubu~c7-rw-ZqfhpUa9A7GolmvWszjpG3xlJELGQJIFP8vwi~wfIsyULjigl3grsh6dycfA4mKAUo3-WVo7lzo~lgHQazx2-dvQPjewwcmWkdZ94ne5BxV2-45MqXJwPwXdOGnf59HKX2Aj5drRvZ7ZcGeTsbHMBaMvT5oEcrRIGPKikYU2xQNtCFNhMkkrRvO~N0Jvla2XenxRRc2lnRLT9rnW8p77Qwm7UInMeU91rjWGcewN9HqwLZUTg6L9Lu68CzcE7eDcnGr1MPw0wcCN3hyWZHykviuYBQWs9KVZNXxetFUVoMpNY~JkcXNsuIl3zO7awzHoYkz3yIS~qKNg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618520&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison with normalized Hi-C contact pattern. Patterns (A) and (B) were extracted from the contact map restricted by the enzyme <em>Hin</em>dIII, (A) for intra- and (B) for interchromosomal contact; Patterns (C) and (D) with restriction enzyme <em>Nco</em>I, (C) for intra- and (D) for interchromosomal contact.</p></div></div>                            <h3 scrollto-destination=5618521 id="5618521" class="section-title" data-legacy-id=s3d>3.4. Interchromosomal interaction pattern</h3>
<p>We picked out the unique ‘protein coding’ cases of interchromosomal gene interactions (for human, 6999 out of 8261 transcripts) and plotted these as a heat map (Fig. <a reveal-id="DSS032F4" data-open="DSS032F4" class="link link-reveal link-table xref-fig">4</a>A), according to where the chromosomes' gene partner was localized. As illustrated by the ‘protein coding’ gene frequencies on human chromosomes (as shown in Fig. <a reveal-id="DSS032F4" data-open="DSS032F4" class="link link-reveal link-table xref-fig">4</a>B), the interactions between chromosomes with higher gene content show a correlation as well. Because the expression data were collected from a variety of cells with different types and developmental stages, it cannot easily be determined whether or not there is a preference on the interaction of given chromosome pairs. But, what is evident is that the observed pattern is different from the pattern achieved from random interaction. This comparison can mathematically be done by comparing the eigenvalues of the heat maps viewed as matrices (Fig. <a reveal-id="DSS032F5" data-open="DSS032F5" class="link link-reveal link-table xref-fig">5</a>A). </p>                        <a id="5618523" scrollto-destination="5618523"></a>
<div id="DSS032F4" data-id="DSS032F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03204.jpeg?Expires=1509238856&Signature=hB3Bhx~~iym1O526~MIGuew1ICnuyKCEA3OiFkaf~hOUgVna~wXNDRFktmBOe5OQMw0t-WhYSsPtcTQgx8NsQmBTf1fteNptOE0FEOXU670JmxlEEG1OdfXmSHLlavMLp6Yorf926UGhunVzVNaXfccUbO1IyPib~8rQzlP3P4ln2miJNoanuw78xZ0vJAXrd9smxNJraGNiv9Oddfx~1VIt3dcBwT8QHHJsYyK9jIvPAQKGNl2sS2rpux02Kzx30AVqK4-MTYNYqL9UE04XHwkkHibzfv6v2QaGb27ekS7Zxn5sArzV0QXurWEOsIVgXhPwwqcz4j73zJm-M~lYMA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Interchromosomal interaction pattern for human chromosomes. (A) Contacts as a heat map of human interchromosomal gene interactions. Protein-coding genes were used as a measure of interactions. The density of the pink colour in each square of the heat map is proportional to the frequencies of chimaera composed of gene partners derived from two given chromosomes and (B) Protein-coding gene content of human 24 chromosomes." path-from-xml="dss03204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618523/dss03204.jpeg" path-from-xml="dss03204.jpeg" target="_blank">View large</a><a section="5618523" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03204.jpeg?Expires=1509238856&Signature=e00tmMMoWmi~zpdj0~OPuP92kXY~8UfrdMoLiHq1Bpb~mu7SMy8GeFcjPHLi5ugepsPlMPOq8ropmRxv6wvG7UYNb292PPlDPxiLCiO1wmsUiUugsjAhNxal7K8PqX~n4bevCfd8zXmzbp5IX9mCpYkSzk4KrqBSDprWKz6HiBvf1fVv-Ub3Ds1q4a0n-ywX2P-5CP5hthMFOoATQEF3RcNfR1Bu4fg64LQ9exDD4SB6nhLpFaX8DB5WgeU5WCd~5JNofPR1JhwX2TVNidhFuwFynnlx4D8E7BF3tDd7skHSpNCP1zIA3Gos8lSsCI4r54z~1AsXpo4nJWbLuZwl5Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618523&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interchromosomal interaction pattern for human chromosomes. (A) Contacts as a heat map of human interchromosomal gene interactions. Protein-coding genes were used as a measure of interactions. The density of the pink colour in each square of the heat map is proportional to the frequencies of chimaera composed of gene partners derived from two given chromosomes and (B) Protein-coding gene content of human 24 chromosomes.</p></div></div><div content-id="DSS032F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03204.jpeg?Expires=1509238856&Signature=hB3Bhx~~iym1O526~MIGuew1ICnuyKCEA3OiFkaf~hOUgVna~wXNDRFktmBOe5OQMw0t-WhYSsPtcTQgx8NsQmBTf1fteNptOE0FEOXU670JmxlEEG1OdfXmSHLlavMLp6Yorf926UGhunVzVNaXfccUbO1IyPib~8rQzlP3P4ln2miJNoanuw78xZ0vJAXrd9smxNJraGNiv9Oddfx~1VIt3dcBwT8QHHJsYyK9jIvPAQKGNl2sS2rpux02Kzx30AVqK4-MTYNYqL9UE04XHwkkHibzfv6v2QaGb27ekS7Zxn5sArzV0QXurWEOsIVgXhPwwqcz4j73zJm-M~lYMA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Interchromosomal interaction pattern for human chromosomes. (A) Contacts as a heat map of human interchromosomal gene interactions. Protein-coding genes were used as a measure of interactions. The density of the pink colour in each square of the heat map is proportional to the frequencies of chimaera composed of gene partners derived from two given chromosomes and (B) Protein-coding gene content of human 24 chromosomes." path-from-xml="dss03204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618523/dss03204.jpeg" path-from-xml="dss03204.jpeg" target="_blank">View large</a><a section="5618523" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03204.jpeg?Expires=1509238856&Signature=e00tmMMoWmi~zpdj0~OPuP92kXY~8UfrdMoLiHq1Bpb~mu7SMy8GeFcjPHLi5ugepsPlMPOq8ropmRxv6wvG7UYNb292PPlDPxiLCiO1wmsUiUugsjAhNxal7K8PqX~n4bevCfd8zXmzbp5IX9mCpYkSzk4KrqBSDprWKz6HiBvf1fVv-Ub3Ds1q4a0n-ywX2P-5CP5hthMFOoATQEF3RcNfR1Bu4fg64LQ9exDD4SB6nhLpFaX8DB5WgeU5WCd~5JNofPR1JhwX2TVNidhFuwFynnlx4D8E7BF3tDd7skHSpNCP1zIA3Gos8lSsCI4r54z~1AsXpo4nJWbLuZwl5Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618523&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interchromosomal interaction pattern for human chromosomes. (A) Contacts as a heat map of human interchromosomal gene interactions. Protein-coding genes were used as a measure of interactions. The density of the pink colour in each square of the heat map is proportional to the frequencies of chimaera composed of gene partners derived from two given chromosomes and (B) Protein-coding gene content of human 24 chromosomes.</p></div></div>                        <a id="5618524" scrollto-destination="5618524"></a>
<div id="DSS032F5" data-id="DSS032F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03205.jpeg?Expires=1509238856&Signature=JHmkYQSqXobyrBo9C75N9nbO2DdapZqeGqG-H8yvhKMAngHtvzPWGgzMfyQ~SxYISOaYeteY6skB6B~NVyS05W0N3Ox3smnWrTP09FbV-oTCrkcI7C6FJzGyPDsOfEhZ~i5KbYmCOKH6ob~oZj91h43MCflAl9HJHxnFZYkjdrZGrEgrl0x3F-3o5ft2VMOQ3tiTsUBpKpXdLEs4VXhntOB0EckDr24dJItvV9aKkwp8yVGzxK-398T-~SNZk2HV8CyzvAXN6oGKFaK-Tpw6l6oJAJV8L7Vkbjk9qLJIrKk2VRK2LS~ojZCtwkKpcVfjU3b-zerxLYSxsfhe5EudOw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparison of interchromosomal interaction patterns via eigenvalue. (A) Patterns from human EST chimaeric transcripts and random distribution simulation (excluding the chromosome Y). The correlation coefficient between the eigenvalues of the observed and the simulated contact matrices is calculated as −0.543. A 95% confidence interval for the eigenvalues based on the simulation pattern was calculated, which however is too small to be shown. None of the eigenvalues from the observed pattern fell into the corresponding confidence interval. (B) Patterns from human normal cells and tumour cells (excluding the chromosome Y)." path-from-xml="dss03205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618524/dss03205.jpeg" path-from-xml="dss03205.jpeg" target="_blank">View large</a><a section="5618524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03205.jpeg?Expires=1509238856&Signature=BhbqFa5Y1MtMbwLUhrHoDziLbvdfvtFTCCF08p6y6DEBGWIDZTDPh69tKFfnG-MlwC57jtWO7MfIA9kyhFCZI3J2ted47ScL7106E5CtdvvwkuA3ea67YrpFM38ALRFi6aLflqKnLdKb9afrBIy9dKE1b4aJnaHTJ7h0tyy3r-iVD4f~gF2J08GiQeTVqpl8qvhwgSyfDmTcf0~IHLuzxiWatF4sowZMQ2FhV0YO6~luw2ZmxgSdEQcLulWBWmq0fvVEJUQOMsreas5qNFPuOjrv73p0muHumCrC-d1zMk-0Fdr3Crwt8i6YBgqGEODcU6MkOj1iM87OyB2oIrJS2A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618524&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of interchromosomal interaction patterns via eigenvalue. (A) Patterns from human EST chimaeric transcripts and random distribution simulation (excluding the chromosome Y). The correlation coefficient between the eigenvalues of the observed and the simulated contact matrices is calculated as −0.543. A 95% confidence interval for the eigenvalues based on the simulation pattern was calculated, which however is too small to be shown. None of the eigenvalues from the observed pattern fell into the corresponding confidence interval. (B) Patterns from human normal cells and tumour cells (excluding the chromosome Y).</p></div></div><div content-id="DSS032F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03205.jpeg?Expires=1509238856&Signature=JHmkYQSqXobyrBo9C75N9nbO2DdapZqeGqG-H8yvhKMAngHtvzPWGgzMfyQ~SxYISOaYeteY6skB6B~NVyS05W0N3Ox3smnWrTP09FbV-oTCrkcI7C6FJzGyPDsOfEhZ~i5KbYmCOKH6ob~oZj91h43MCflAl9HJHxnFZYkjdrZGrEgrl0x3F-3o5ft2VMOQ3tiTsUBpKpXdLEs4VXhntOB0EckDr24dJItvV9aKkwp8yVGzxK-398T-~SNZk2HV8CyzvAXN6oGKFaK-Tpw6l6oJAJV8L7Vkbjk9qLJIrKk2VRK2LS~ojZCtwkKpcVfjU3b-zerxLYSxsfhe5EudOw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparison of interchromosomal interaction patterns via eigenvalue. (A) Patterns from human EST chimaeric transcripts and random distribution simulation (excluding the chromosome Y). The correlation coefficient between the eigenvalues of the observed and the simulated contact matrices is calculated as −0.543. A 95% confidence interval for the eigenvalues based on the simulation pattern was calculated, which however is too small to be shown. None of the eigenvalues from the observed pattern fell into the corresponding confidence interval. (B) Patterns from human normal cells and tumour cells (excluding the chromosome Y)." path-from-xml="dss03205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618524/dss03205.jpeg" path-from-xml="dss03205.jpeg" target="_blank">View large</a><a section="5618524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03205.jpeg?Expires=1509238856&Signature=BhbqFa5Y1MtMbwLUhrHoDziLbvdfvtFTCCF08p6y6DEBGWIDZTDPh69tKFfnG-MlwC57jtWO7MfIA9kyhFCZI3J2ted47ScL7106E5CtdvvwkuA3ea67YrpFM38ALRFi6aLflqKnLdKb9afrBIy9dKE1b4aJnaHTJ7h0tyy3r-iVD4f~gF2J08GiQeTVqpl8qvhwgSyfDmTcf0~IHLuzxiWatF4sowZMQ2FhV0YO6~luw2ZmxgSdEQcLulWBWmq0fvVEJUQOMsreas5qNFPuOjrv73p0muHumCrC-d1zMk-0Fdr3Crwt8i6YBgqGEODcU6MkOj1iM87OyB2oIrJS2A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618524&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of interchromosomal interaction patterns via eigenvalue. (A) Patterns from human EST chimaeric transcripts and random distribution simulation (excluding the chromosome Y). The correlation coefficient between the eigenvalues of the observed and the simulated contact matrices is calculated as −0.543. A 95% confidence interval for the eigenvalues based on the simulation pattern was calculated, which however is too small to be shown. None of the eigenvalues from the observed pattern fell into the corresponding confidence interval. (B) Patterns from human normal cells and tumour cells (excluding the chromosome Y).</p></div></div><p>Furthermore, we separated the chimaera according to the cell sources from which they originate (normal or tumour cell lines). Then, we constructed the interaction matrix and tried to compare the patterns from normal and tumour cell lines. For both cases, we acquired ∼3500 chimaera. A slight difference is observed for normal and tumour cells, but the correlation coefficient between the eigenvalues of two contact matrices is computed as 0.998, which indicates that two contact patterns are by and large the same (Fig. <a reveal-id="DSS032F5" data-open="DSS032F5" class="link link-reveal link-table xref-fig">5</a>B). Especially to the chromosomal translocation-mediated fusion transcripts in tumour cells, the ‘contact-first’ model and the ‘breakage-first’ model have been postulated.<sup><a reveal-id="DSS032C25" data-open="DSS032C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Under either scenario, a spatial proximity effect is obvious.<sup><a reveal-id="DSS032C25" data-open="DSS032C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Consequently, both chromosomal translocation-mediated and transcription-mediated fusion transcripts can be used to indicate the physical proximities between genes. Through a literature curation, we found that two gene pairs (<em>RGS17</em>/<em>TBL1XR1</em> and <em>BCAS3</em>/<em>BCAS4</em>) identified from the human ESTs libraries have been verified to be spatially proximate by fluorescence <em>in situ</em> hybridization experiments.<sup><a reveal-id="DSS032C26" data-open="DSS032C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS032C27" data-open="DSS032C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Chimaeric transcripts involved are listed in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data</a>.</p>                            <h3 scrollto-destination=5618526 id="5618526" class="section-title" data-legacy-id=s3e>3.5. Intrachromosomal interaction pattern</h3>
<p>For the intrachromosomal gene loci interaction data, we calculated the frequencies of gene–gene interactions for given genomic distances. The genomic distances between gene partners ranged from 509 Kb to 244.4 Mb. We plotted the probability of contact versus the genomic distance between gene partners on a double logarithmic plot. Two different types of behaviour can be observed. Below the distance of 7–8 Mb, a functional <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss032ileq3.gif?Expires=1509238856&Signature=Pnz-cY6RmbeAcSlqaqywB9iVfoUCSqGth2XuorNhVhibi~lMhREAJuMgjjknJqibqWVeoKx~Fxt9yzub-glcr6AeTuvEZSa53-M67~2TAUQ6RXdvAyuXJhen3nCfLlAWg5PMU5t~0MaFXH9xyxtxa9V0rhJYFyHP6BC84X5bUOZXzcl8tgIHj9mkTiSq5fZjl8zvSRHT8hhid~sGzkiffjPHEOvN-qZGsquerLUh9n5yVeAGwiHrdcdG984vVgr4Rlf3OgsQsPKqZ9elWOK-iyBAxGsM6HwR5vnY2uPyW7hE5GoIpRxDRRwab1IQ7yLcbQOMpT09POD6He~UbPY~wA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss032ileq3.gif" /></span> exists with an exponent α = −1 (see the double logarithmic plot shown in Fig. <a reveal-id="DSS032F6" data-open="DSS032F6" class="link link-reveal link-table xref-fig">6</a>). Above this distance, a quite different pattern is observed. The rapid drop for large genomic distances might result from the limitation of interaction data when the genomic distance is increased. To our surprise, a comparable pattern was observed for mouse as well, even though its EST coverage is not as high as that of human. With caution, we speculate that a general strategy for chromatin compaction might be utilized in mammalian cells. On account of the difference in the contact probability of chromatin with different length, we normalized the contact frequencies by the genomic distance and replotted the pattern of human. After normalization, as shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>, under relative short genomic distance, the contact probability remains stable. When the genomic distance is increased, the contact probability rises until a plateau is reached. The dropping at the end might be due to the data depletion. </p>                        <a id="5618535" scrollto-destination="5618535"></a>
<div id="DSS032F6" data-id="DSS032F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03206.jpeg?Expires=1509238856&Signature=ZU2qYogY~Fj7uitK9cipSvwX10UkodeNQ3yLaynqAXG5wBECZQAZmyXE05ZyNffAEOOIwRPrysEBzZ-e63x9XMRNBKy7ez-4vBDpZqhG9lP19SDauTnzSAXGR-r0F24dZzqMKylC7G3JH5M64cv4BHTRWXEwTH~6YhJwq9sMJF9vupCJ2V6biV125aRElZ7Y62Ov4Jad6MS-odadN4gCwtbAH9VjPpfbaJhYYPsR7xPt1DNdwapxTH4xqhiYAUMCc9BmN-HHCk~lEsV7KZUCShI2jYjS-HffNamdVEZduwHoh-FVH~PCpgjBPRzYROVuWmAzdMopqKuLTyPXZzz2iw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Fit of intrachromosomal interactions from human and mouse. Intrachromosomal gene pairs with genomic distance larger than 500 kb were considered. (A) Interaction patterns of human and (B) Interaction patterns of mouse. Confidence intervals(95%) of the slopes are shown." path-from-xml="dss03206.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618535/dss03206.jpeg" path-from-xml="dss03206.jpeg" target="_blank">View large</a><a section="5618535" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03206.jpeg?Expires=1509238856&Signature=aa9RXndTWe3d2GlTnrlJPzlNBOevvlMMrdCAlfUS4kM9oK94yrZnUvo9lMba~eAIzJpHrhR7~S7-VXx2ncywMmBbH6Qw2f4O~upPFk39GAZcyNKfBMVUz5jUShbFitm1qr2nCuLR1zyEs6rPjL4ypwJEDZMGTVIWR-7tvk7rGvbqVN1P897XSd~UkFKupe6kMGy47cdO70QvRuTjZDdfJLy7cX1cfY2~-VD8B8193wT097kH~W8uYZIdnNT5zr5iZzSEctxWR~TetI2jCE4cYlFBd7IKEk01F79R4~xnIGHPl9-O5SziVP31D~O9feid8oLlVY6bO~WCs2RhyS63Tw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618535&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03206.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fit of intrachromosomal interactions from human and mouse. Intrachromosomal gene pairs with genomic distance larger than 500 kb were considered. (A) Interaction patterns of human and (B) Interaction patterns of mouse. Confidence intervals(95%) of the slopes are shown.</p></div></div><div content-id="DSS032F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/m_dss03206.jpeg?Expires=1509238856&Signature=ZU2qYogY~Fj7uitK9cipSvwX10UkodeNQ3yLaynqAXG5wBECZQAZmyXE05ZyNffAEOOIwRPrysEBzZ-e63x9XMRNBKy7ez-4vBDpZqhG9lP19SDauTnzSAXGR-r0F24dZzqMKylC7G3JH5M64cv4BHTRWXEwTH~6YhJwq9sMJF9vupCJ2V6biV125aRElZ7Y62Ov4Jad6MS-odadN4gCwtbAH9VjPpfbaJhYYPsR7xPt1DNdwapxTH4xqhiYAUMCc9BmN-HHCk~lEsV7KZUCShI2jYjS-HffNamdVEZduwHoh-FVH~PCpgjBPRzYROVuWmAzdMopqKuLTyPXZzz2iw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Fit of intrachromosomal interactions from human and mouse. Intrachromosomal gene pairs with genomic distance larger than 500 kb were considered. (A) Interaction patterns of human and (B) Interaction patterns of mouse. Confidence intervals(95%) of the slopes are shown." path-from-xml="dss03206.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5618535/dss03206.jpeg" path-from-xml="dss03206.jpeg" target="_blank">View large</a><a section="5618535" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss03206.jpeg?Expires=1509238856&Signature=aa9RXndTWe3d2GlTnrlJPzlNBOevvlMMrdCAlfUS4kM9oK94yrZnUvo9lMba~eAIzJpHrhR7~S7-VXx2ncywMmBbH6Qw2f4O~upPFk39GAZcyNKfBMVUz5jUShbFitm1qr2nCuLR1zyEs6rPjL4ypwJEDZMGTVIWR-7tvk7rGvbqVN1P897XSd~UkFKupe6kMGy47cdO70QvRuTjZDdfJLy7cX1cfY2~-VD8B8193wT097kH~W8uYZIdnNT5zr5iZzSEctxWR~TetI2jCE4cYlFBd7IKEk01F79R4~xnIGHPl9-O5SziVP31D~O9feid8oLlVY6bO~WCs2RhyS63Tw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5618535&ar=382928&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03206.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fit of intrachromosomal interactions from human and mouse. Intrachromosomal gene pairs with genomic distance larger than 500 kb were considered. (A) Interaction patterns of human and (B) Interaction patterns of mouse. Confidence intervals(95%) of the slopes are shown.</p></div></div>                            <h3 scrollto-destination=5618538 id="5618538" class="section-title" data-legacy-id=s3f>3.6. Discussion</h3>
<p>From the human and mouse EST databases, we identified chimaeric transcripts with verification. Based on what we currently know about TIC and transcription factories, ‘genuine’ chimaera of this kind, functional or not, might originate from product interactions while being transcribed. A further analysis on the co-expression pattern of gene pairs in the examined chimaeric products revealed a strong correlation in their expression. To make the expression of chimaeric gene partners synchronous, two gene loci might be recruited to the same transcription factory, which, in turn, would decrease their spatial distance and increase the possibility of their aberrant ligation. This observation further supports the assumption of TIC<sup><a reveal-id="DSS032C1" data-open="DSS032C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and the role of the transcription factories in gene transcription events,<sup><a reveal-id="DSS032C12" data-open="DSS032C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS032C13" data-open="DSS032C13" class="link link-ref link-reveal xref-bibr">13</a></sup> as well.</p><p>Some chimaeric gene pairs in <em>cis</em> or in <em>trans</em> have been experimentally confirmed to be spatially proximate.<sup><a reveal-id="DSS032C26" data-open="DSS032C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS032C27" data-open="DSS032C27" class="link link-ref link-reveal xref-bibr">27</a></sup> To further validate the spatial association of these chimaeric gene pairs, we checked the contact probability of chromosomal regions, where chimaeric genes localize. Although only a normalized contact map of human lymphoblast is available so far, we still can see contact enhancement between some chimaeric genes. Considering the difference in expression profile of different cell lines, chimaeric gene pairs collected from other cell lines might not be actively expressed in the lymphoblast, which meets no requirement of the TIC. Thus, more chromosome contact maps from diverse cell lines would be helpful to confirm the spatial proximity of these chimaeric gene pairs. On the other hand, however, it is not necessary for each chimaeric gene pair to be functionally related. Considering the fact that a limited amount of the transcriptional factories exists in the cell nucleus, genes sharing the same transcriptional apparatus might just be temporal related.</p><p>For the time being, it is still too early to draw definitive conclusions about interchromosomal interaction pattern, but a probabilistic non-random arrangement of chromosomes seems plausible in mammalian cell nucleus.<sup><a reveal-id="DSS032C28" data-open="DSS032C28" class="link link-ref link-reveal xref-bibr">28</a></sup> From our observation, what can be confirmed at present is that the chromosomes pair-wise association pattern is not random, when compared with the pattern with chromosomes distributed arbitrarily. Some researches have shown that the relative arrangements of some chromosomes are conserved and independent of cell types, whereas others seem to be cell-type specific.<sup><a reveal-id="DSS032C29" data-open="DSS032C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The same is true with regard to the chromosomal rearrangement in tumour cell, a cell line in which disease-related fashion was observed.<sup><a reveal-id="DSS032C29" data-open="DSS032C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The observation that some chimaeric transcripts can be expressed in genetically rearranged cells and in normal cells raises the possibility that such physical proximity between gene loci might be preserved after the chromosomal translocation. This might be an explanation of the slight difference detected from the interchromosomal interaction patterns of normal and tumour cell lines.</p><p>The pattern reflected by the intrachromosomal loci interaction is very informative. From 500 to 7–8 Mb, the interaction probability shows a specific mathematical form (i.e. an exponential form) that is consistent with the data from chromosome conformation capture experiments. Thus, at this scale, the architecture of chromatin folding fits well with the experimental results<sup><a reveal-id="DSS032C16" data-open="DSS032C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and with a model recently proposed.<sup><a reveal-id="DSS032C17" data-open="DSS032C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS032C30" data-open="DSS032C30" class="link link-ref link-reveal xref-bibr">30</a></sup> In mouse, although it has fewer data, a coarse pattern compatible with that from human is seen as well. For the complexity of higher eukaryotic nucleus, the genomes must be well organized, covering several different length scales to keep an ordered but flexible architecture and to exert their functions properly. The strategy used by mammals for chromosome organization might be conserved across the species. Furthermore, the pattern shown after distance normalization indicated that a basal folding schema might exist. This basal form of condensation might organize the chromatin fibre into a series of units for subsequent folding. Due to the increase in the stiffness and the bulk size of the chromatin fibre after folding, the contact probability rises along with the extended genomic distance.</p>                            <h2 scrollto-destination=5618544 id="5618544" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a> www.dnaresearch.oxfordjournals.org.</p>                            <h2 scrollto-destination=5618546 id="5618546" class="section-title" data-legacy-id=s5>Funding</h2>
<p>S.L. gratefully acknowledges funding from the Heinz-Goetze-Foundation and the Heidelberg Graduate School of Mathematical and Computational Methods for the Sciences.</p>                            <h2 scrollto-destination=5618549 id="5618549" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors appreciate here the assistance from M. Bohn and the resources from the Heidelberg Graduate School of Mathematical and Computational Methods for the Sciences (HGSMathComp).</p>                            <h2 scrollto-destination=5618553 id="5618553" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dss032c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akiva</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Toporik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Edelheit</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Transcription-mediated gene fusion in the human genome</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">30</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-mediated%20gene%20fusion%20in%20the%20human%20genome&amp;author=P.%20Akiva&amp;author=A.%20Toporik&amp;author=S.%20Edelheit&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=30-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4137606" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4137606" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4137606"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mayer</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Floeter-Winter</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">Pre-mRNA trans-splicing: from kinetoplastids to mammals, an easy language for life diversity</div>, <div class="source">Mem. Inst. Oswaldo Cruz</div> , <div class="year">2005</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">501</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pre-mRNA%20trans-splicing%3A%20from%20kinetoplastids%20to%20mammals%2C%20an%20easy%20language%20for%20life%20diversity&amp;author=M.G.%20Mayer&amp;author=L.M.%20Floeter-Winter&amp;publication_year=2005&amp;journal=Mem.%20Inst.%20Oswaldo%20Cruz&amp;volume=100&amp;pages=501-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S0074-02762005000500010" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS0074-02762005000500010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS0074-02762005000500010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16184228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nilsen</div> <div class="given-names">T.W.</div></div>. </div><div class="article-title">Trans-splicing in protozoa and helminths</div>, <div class="source">Infect. Agents Dis.</div> , <div class="year">1992</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">212</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trans-splicing%20in%20protozoa%20and%20helminths&amp;author=T.W.%20Nilsen&amp;publication_year=1992&amp;journal=Infect.%20Agents%20Dis.&amp;volume=1&amp;pages=212-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1365548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krause</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hirsh</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A trans-spliced leader sequence on actin mRNA in C. elegans</div>, <div class="source">Cell</div> , <div class="year">1987</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">753</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20trans-spliced%20leader%20sequence%20on%20actin%20mRNA%20in%20C.%20elegans&amp;author=M.%20Krause&amp;author=D.%20Hirsh&amp;publication_year=1987&amp;journal=Cell&amp;volume=49&amp;pages=753-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(87)90613-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(87)90613-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(87)90613-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3581169" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Satou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hamaguchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Takeuchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hastings</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Satoh</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Genomic overview of mRNA 5′-leader trans-splicing in the ascidian <em>Ciona intestinalis</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">3378</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20overview%20of%20mRNA%205%E2%80%B2-leader%20trans-splicing%20in%20the%20ascidian%20Ciona%20intestinalis&amp;author=Y.%20Satou&amp;author=M.%20Hamaguchi&amp;author=K.%20Takeuchi&amp;author=K.E.%20Hastings&amp;author=N.%20Satoh&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=3378-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl418" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl418" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl418"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16822859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mongelard</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Labrador</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Baxter</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Gerasimova</div> <div class="given-names">T.I.</div></div>,  <div class="name"><div class="surname">Corces</div> <div class="given-names">V.G.</div></div>. </div><div class="article-title">Trans-splicing as a novel mechanism to explain interallelic complementation in Drosophila</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">160</div> (pg. <div class="fpage">1481</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trans-splicing%20as%20a%20novel%20mechanism%20to%20explain%20interallelic%20complementation%20in%20Drosophila&amp;author=F.%20Mongelard&amp;author=M.%20Labrador&amp;author=E.M.%20Baxter&amp;author=T.I.%20Gerasimova&amp;author=V.G.%20Corces&amp;publication_year=2002&amp;journal=Genetics&amp;volume=160&amp;pages=1481-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11973303" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Martignetti</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Yeo</div> <div class="given-names">T.T.</div></div>,  <div class="name"><div class="surname">Massa</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Longo</div> <div class="given-names">F.M.</div></div>. </div><div class="article-title">A candidate chimeric mammalian mRNA transcript is derived from distinct chromosomes and is associated with nonconsensus splice junction motifs</div>, <div class="source">DNA Cell Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">303</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20candidate%20chimeric%20mammalian%20mRNA%20transcript%20is%20derived%20from%20distinct%20chromosomes%20and%20is%20associated%20with%20nonconsensus%20splice%20junction%20motifs&amp;author=C.%20Zhang&amp;author=Y.%20Xie&amp;author=J.A.%20Martignetti&amp;author=T.T.%20Yeo&amp;author=S.M.%20Massa&amp;author=F.M.%20Longo&amp;publication_year=2003&amp;journal=DNA%20Cell%20Biol.&amp;volume=22&amp;pages=303-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/104454903322216653" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F104454903322216653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F104454903322216653"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12941158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Short homologous sequences are strongly associated with the generation of chimeric RNAs in eukaryotes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">56</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20homologous%20sequences%20are%20strongly%20associated%20with%20the%20generation%20of%20chimeric%20RNAs%20in%20eukaryotes&amp;author=X.%20Li&amp;author=L.%20Zhao&amp;author=H.%20Jiang&amp;author=W.%20Wang&amp;publication_year=2009&amp;journal=J.%20Mol.%20Evol.&amp;volume=68&amp;pages=56-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-008-9187-0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-008-9187-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-008-9187-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19089307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frenkel-Morgenstern</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Valencia</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Novel domain combinations in proteins encoded by chimeric transcripts</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">i67</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20domain%20combinations%20in%20proteins%20encoded%20by%20chimeric%20transcripts&amp;author=M.%20Frenkel-Morgenstern&amp;author=A.%20Valencia&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=i67-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts216" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22689780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Patel</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Simon</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Iglesia</div> <div class="given-names">M.D.</div></div>, et al. </div><div class="article-title">Tumor-specific retargeting of an oncogenic transcription factor chimera results in dysregulation of chromatin and transcription</div>, <div class="source">Genome Res.</div> , <div class="year">2011</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">259</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tumor-specific%20retargeting%20of%20an%20oncogenic%20transcription%20factor%20chimera%20results%20in%20dysregulation%20of%20chromatin%20and%20transcription&amp;author=M.%20Patel&amp;author=J.M.%20Simon&amp;author=M.D.%20Iglesia&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=259-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.125666.111" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.125666.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.125666.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mor</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Sklar</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A neoplastic gene fusion mimics trans-splicing of RNAs in normal human cells</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">321</div> (pg. <div class="fpage">1357</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20neoplastic%20gene%20fusion%20mimics%20trans-splicing%20of%20RNAs%20in%20normal%20human%20cells&amp;author=H.%20Li&amp;author=J.%20Wang&amp;author=G.%20Mor&amp;author=J.%20Sklar&amp;publication_year=2008&amp;journal=Science&amp;volume=321&amp;pages=1357-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1156725" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1156725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1156725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18772439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carter</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Eskiw</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">P.R.</div></div>. </div><div class="article-title">Transcription factories</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">585</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factories&amp;author=D.R.%20Carter&amp;author=C.%20Eskiw&amp;author=P.R.%20Cook&amp;publication_year=2008&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=36&amp;pages=585-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0360585" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0360585" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0360585"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18631121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chuang</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Belmont</div> <div class="given-names">A.S.</div></div>. </div><div class="article-title">Close encounters between active genes in the nucleus</div>, <div class="source">Genome Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">237</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Close%20encounters%20between%20active%20genes%20in%20the%20nucleus&amp;author=C.H.%20Chuang&amp;author=A.S.%20Belmont&amp;publication_year=2005&amp;journal=Genome%20Biol.&amp;volume=6&amp;pages=237" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2005-6-11-237" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-11-237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2005-6-11-237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16277755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Branco</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Pombo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations</div>, <div class="source">PLoS Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e138</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intermingling%20of%20chromosome%20territories%20in%20interphase%20suggests%20role%20in%20translocations%20and%20transcription-dependent%20associations&amp;author=M.R.%20Branco&amp;author=A.%20Pombo&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=e138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040138" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040138" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040138"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16623600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lua</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Borovinskiy</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Grosberg</div> <div class="given-names">A.Y.</div></div>. </div><div class="article-title">Fractal and statistical properties of large compact polymers: a computational study</div>, <div class="source">Polymer</div> , <div class="year">2004</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">717</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fractal%20and%20statistical%20properties%20of%20large%20compact%20polymers%3A%20a%20computational%20study&amp;author=R.%20Lua&amp;author=A.L.%20Borovinskiy&amp;author=A.Y.%20Grosberg&amp;publication_year=2004&amp;journal=Polymer&amp;volume=45&amp;pages=717-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.polymer.2003.10.073" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.polymer.2003.10.073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.polymer.2003.10.073"> </span></div> </div></div></div></div></div><div content-id="dss032c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lieberman-Aiden</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">van Berkum</div> <div class="given-names">L.N.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Comprehensive mapping of long-range interactions reveals folding principles of the human genome</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">289</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&amp;author=E.%20Lieberman-Aiden&amp;author=L.N.%20van%20Berkum&amp;author=L.%20Williams&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=289-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181369" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181369" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181369"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19815776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Heermann</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">van Driel</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Random loop model for long polymers</div>, <div class="source">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</div> , <div class="year">2007</div>, vol. <div class="volume">76</div> pg. <div class="fpage">051805</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Random%20loop%20model%20for%20long%20polymers&amp;author=M.%20Bohn&amp;author=D.W.%20Heermann&amp;author=R.%20van%20Driel&amp;publication_year=2007&amp;journal=Phys.%20Rev.%20E%20Stat.%20Nonlin.%20Soft%20Matter%20Phys.&amp;volume=76&amp;pages=051805" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1103/PhysRevE.76.051805" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1103%2FPhysRevE.76.051805" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1103%2FPhysRevE.76.051805"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18233679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mateos-Langerak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bohn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">de Leeuw</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Spatially confined folding of chromatin in the interphase nucleus</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">3812</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatially%20confined%20folding%20of%20chromatin%20in%20the%20interphase%20nucleus&amp;author=J.%20Mateos-Langerak&amp;author=M.%20Bohn&amp;author=W.%20de%20Leeuw&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=3812-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0809501106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0809501106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0809501106"> </span></div> </div></div></div></div></div><div content-id="dss032c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Heermann</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Topological interactions between ring polymers: implications for chromatin loops</div>, <div class="source">J. Chem. Phys.</div> , <div class="year">2010</div>, vol. <div class="volume">132</div> pg. <div class="fpage">044904</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Topological%20interactions%20between%20ring%20polymers%3A%20implications%20for%20chromatin%20loops&amp;author=M.%20Bohn&amp;author=D.W.%20Heermann&amp;publication_year=2010&amp;journal=J.%20Chem.%20Phys.&amp;volume=132&amp;pages=044904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1063/1.3302812" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1063%2F1.3302812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1063%2F1.3302812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20113063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Unneberg</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Tentative mapping of transcription-induced interchromosomal interaction using chimeric EST and mRNA data</div>, <div class="source">PLoS One</div> , <div class="year">2007</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e254</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tentative%20mapping%20of%20transcription-induced%20interchromosomal%20interaction%20using%20chimeric%20EST%20and%20mRNA%20data&amp;author=P.%20Unneberg&amp;author=M.J.%20Claverie&amp;publication_year=2007&amp;journal=PLoS%20One&amp;volume=2&amp;pages=e254" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0000254" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0000254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0000254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17330142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yaffe</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tanay</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Probabilistic modeling of Hi-C contact maps eliminates systematic biases to characterize global chromosomal architecture</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">1059</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probabilistic%20modeling%20of%20Hi-C%20contact%20maps%20eliminates%20systematic%20biases%20to%20characterize%20global%20chromosomal%20architecture&amp;author=E.%20Yaffe&amp;author=A.%20Tanay&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1059-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.947" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.947" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.947"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22001755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT–the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%93the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shibaoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohta</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div>. </div><div class="article-title">COXPRESdb: a database of coexpressed gene networks in mammals</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D77</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=COXPRESdb%3A%20a%20database%20of%20coexpressed%20gene%20networks%20in%20mammals&amp;author=T.%20Obayashi&amp;author=S.%20Hayashi&amp;author=M.%20Shibaoka&amp;author=M.%20Saeki&amp;author=H.%20Ohta&amp;author=K.%20Kinoshita&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D77-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm840" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Safer</div> <div class="given-names">H.M.</div></div>. </div><div class="article-title">A novel algorithm for computational identification of contaminated EST libraries</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1067</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20algorithm%20for%20computational%20identification%20of%20contaminated%20EST%20libraries&amp;author=R.%20Sorek&amp;author=H.M.%20Safer&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=1067-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg170" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12560505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sachs</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Review: proximity effects in the production of chromosome aberrations by ionizing radiation</div>, <div class="source">Int. J. Radiat. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">1</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Review%3A%20proximity%20effects%20in%20the%20production%20of%20chromosome%20aberrations%20by%20ionizing%20radiation&amp;author=R.K.%20Sachs&amp;author=A.M.%20Chen&amp;author=D.J.%20Brenner&amp;publication_year=1997&amp;journal=Int.%20J.%20Radiat.%20Biol.&amp;volume=71&amp;pages=1-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/095530097144364" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F095530097144364" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F095530097144364"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9020958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hahn</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bera</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Gehlhaus</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kirsch</div> <div class="given-names">I.R.</div></div>,  <div class="name"><div class="surname">Pastan</div> <div class="given-names">I.H.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Finding fusion genes resulting from chromosome rearrangement by analyzing the expressed sequence databases</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">13257</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finding%20fusion%20genes%20resulting%20from%20chromosome%20rearrangement%20by%20analyzing%20the%20expressed%20sequence%20databases&amp;author=Y.%20Hahn&amp;author=T.K.%20Bera&amp;author=K.%20Gehlhaus&amp;author=I.R.%20Kirsch&amp;author=I.H.%20Pastan&amp;author=B.%20Lee&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=13257-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0405490101" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0405490101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0405490101"> </span></div> </div></div></div></div></div><div content-id="dss032c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ooi</div> <div class="given-names">H.S.</div></div>,  <div class="name"><div class="surname">Choo</div> <div class="given-names">S.W.</div></div>, et al. </div><div class="article-title">Fusion transcripts and transcribed retrotransposed loci discovered through comprehensive transcriptome analysis using paired-end ditags (PETs)</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">828</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fusion%20transcripts%20and%20transcribed%20retrotransposed%20loci%20discovered%20through%20comprehensive%20transcriptome%20analysis%20using%20paired-end%20ditags%20(PETs)&amp;author=Y.%20Ruan&amp;author=H.S.%20Ooi&amp;author=S.W.%20Choo&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=828-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6018607" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6018607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6018607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17568001" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeitz</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Mukherjee</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A probabilistic model for the arrangement of a subset of human chromosome territories in WI38 human fibroblasts</div>, <div class="source">J. Cell Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">221</div> (pg. <div class="fpage">120</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20probabilistic%20model%20for%20the%20arrangement%20of%20a%20subset%20of%20human%20chromosome%20territories%20in%20WI38%20human%20fibroblasts&amp;author=M.J.%20Zeitz&amp;author=L.%20Mukherjee&amp;author=S.%20Bhattacharya&amp;author=J.%20Xu&amp;author=R.%20Berezney&amp;publication_year=2009&amp;journal=J.%20Cell%20Physiol.&amp;volume=221&amp;pages=120-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jcp.v221:1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjcp.v221:1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjcp.v221:1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19507193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marella</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mukherjee</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Berezney</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Cell type specific chromosome territory organization in the interphase nuclus of normal and cancer cells</div>, <div class="source">J. Cell Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">221</div> (pg. <div class="fpage">130</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20type%20specific%20chromosome%20territory%20organization%20in%20the%20interphase%20nuclus%20of%20normal%20and%20cancer%20cells&amp;author=N.V.%20Marella&amp;author=S.%20Bhattacharya&amp;author=L.%20Mukherjee&amp;author=J.%20Xu&amp;author=R.%20Berezney&amp;publication_year=2009&amp;journal=J.%20Cell%20Physiol.&amp;volume=221&amp;pages=130-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jcp.v221:1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjcp.v221:1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjcp.v221:1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19496171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dss032c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss032c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Heermann</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Diffusion-driven looping provides a consistent framework for chromatin organization</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e12218</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diffusion-driven%20looping%20provides%20a%20consistent%20framework%20for%20chromatin%20organization&amp;author=M.%20Bohn&amp;author=D.W.%20Heermann&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e12218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0012218" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0012218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0012218"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20811620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Minoru Ko</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=382928&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp_fig3.png?Expires=1509238856&amp;Signature=VLStcqKAG4fE4-iLnRQkNFadJ9avjMzK0wBF2SPXqdyFZXSDDQfRVfl11a1oYHrvG87h4qaJ4ZCA0NVTi4Q2s5BwTsFOQqJsurwmvJ6aMafX1Fln5YEoJM5fSPdIvTMfQqRds8kiisgsnSckWkjspCBJnutNH0ZgZ7ZlS1eHkEIlBo07n0W~XOA0L2ULAPX20cWy94bR9AhBcpDLHpkY8stSBadUO9TzWTnCYSmNpVh7hG6gBJoiPSkIeZncsPqghNPlrOI3ApJ281778s2Bv6HSB6J6BtluAwlOULNxTPkYgrHTZEDwSiI6pYuEGHEPc1yJko27HDlOZEPNrIvIIQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp_fig2.png?Expires=1509238856&amp;Signature=U4JdhdM3LCj0gpPjw2O3tW3nR1mJ4mlTT15tw9jqWJS7dKkJgq8nly568Hon21juxHmPzesz3hLfDKiHqvmY8sxSdtx2FAw-HRA7R6J1lMRrDbQs2qxziTxZMJfNF19B7R~M-XM8ZxATqg099tA~d31pK7MVmZRRsLOHifRhn95bsSzT4MXbyGBB1vBvV2Zqn8O6jR8ndEO2lP~zedK31E4Jr7gBKXwcGwLA-7O15IkHPthYIQg2UE55mBk7n1D7ClX2da7Lf3PUQ47vrXUBRM735Xjg-0XeZGBwhzoorR7eXm7kL4mC5XB9UxbbfZS6HSAEA45QPc~Pp7ERnVY7sw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp_fig1.png?Expires=1509238856&amp;Signature=Hmjnt0ijstsDciV3m6kUBoMl2QWsEdJtNX2kSBwJ9D9NUp-qu0zOIHwKOLXkXLQWrm4VplD43kBP3rs5gKDEkfVRdjjR0HgParZ07HWk80RATYW9RKygBYb9kyxQE8NXbKykxh-bKdLOuypYCgXlYhJyjy2-ANTSl50yhxBOaUHcWUXS9vSArYJFd8Tf5MlhQkDhD-uMuYUi6dBUiPWHyCgYCCtuzwV3HBfHYF~ceem6YKkIYMqJpx9PqPcaymsg0xmsiEMnbc2lZjemg6rQG9f23b1RGIpmBt1AIjWzOSw2s4JlH01AW2k7pYHVFFAK3nS-65U-vtVCBTI34d24sQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp3.xls?Expires=1509238856&amp;Signature=GWV96cbVAkpv~XxEb4V18dtoBKTib9WjZ4MxM8DxydJCavAZh8UN43rABsdhozxwSiD9Xi~LB5H-esQ84zW-~LuNUQDCtyBLdMTpVwP9lO6WpLoCZwCFsPWOserZWH4s4WThLmvTdfLYwYw8Yv80uZvUqd9~Yq1KHRFTgxPUjEAwVtW0dQQNkGBML1MSIuvWPO90D3KeM~AM2yCeky3ua6QypxyRGg-HS0RdN9OcLW7QEA9VmnNn3zjz-uFbpBXyPjruq5plifgqX7zr-ctAJ-u9eIRtWJ~ItXmEFoV9UmmBosR4l6HUm3vdpZ4HSH1wN6-ARepey4clo9UmFvENbg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp2.xls?Expires=1509238856&amp;Signature=es~ajTGUDaQE-OnF~ARHg9wusqlQkvPw59eHwob5sMA1~P2P7YHSGUNCIXOmXlquzFBTeEus2uHcFZQvy6NDcN~OuSXknTB7wm3GcRUqFq4BHtWUJh99NPDFjIvVab3p-a1J~W6vk-sCuykUlGXuLdm~R6j92rQHnY2fK57fVhlQeuN~9TpeIv1fg2-NBKCbaHhOH6Q7lNtxN5J4Ei5YXRlMYYRkOeX91YaE65LQF~KF21vwHY3SQgExSaiEvkrJfdAUO3SYyh9bbBhytTFZTKtdUAQ9UMQvumisnKOTOAhcEBYAtcasKiylPJDAb3UZC~0UDGig0sfY8ZKtr8rRAA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp1.xls?Expires=1509238856&amp;Signature=eQ4tc~gw~jAyH~IgQIbiCdECJhNqCqUYCaN1vdzhWbVOQQgyZEdcQQb2nVT1SG7BCKHkE-2pXEokKQOP5NEtEySlxug7ARD3R~BaOtoVAhLVyGCQO5wpKGw8pCtMWxwlX3QDAZQSwv-~D50Zq8msMQGSjboZOsmHeJhXzGtE06cwQ9kQTRuGXFjuiW4XPBCdpVkn0dizopsEdYAhwUI2sG1Nmz4AyifTOBRYElL3xTeCAai7rN9MW3HWD8Sje9CpuyPXdAUaZzZ2vXMbTFWMj5zq-RdLHeQWwcQg9vwe-8SedPFsjDwlxXi67YL1He7rVTWMZ5b4qcI65ZwpXlning__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss032/2/dss032-dss032supp.doc?Expires=1509238856&amp;Signature=D55ztIVma1LQo4xaNuyPcmPRZOq89nljzwmOvDogexlcgbwrBPPONK7OPRkPHL7qMOHWo4pwJZht9Gpn7fLl5PK2G9Cj8xBzpXgAo1MMpxcmau83f2KoAZFGMb7Ih9kQkkT3fUgRVzs66ZpUF5R6htVJ0yGb8GzV9Bc2IahXmyf-cfvsLbxbsYGy6LRgn2aUNXttU4wXd5KBdNDRYpb8QCxaIj2Zm9d0-hlSNG0eR1Viq-wlNzj3ek7KJsy2qDoAEqwfPXPaf1T7uw6s8YfVMHTvDdQpYRm3IFaJOkfQse9j5KIQCPgsA2Qj0P9-~154fwCpt9c3kc~xLmrq3PSdFw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">13</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dss032"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="382928"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f45%2f382928">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F45%2F382928" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/382928'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Using Chimaeric Expression Sequence Tag as the Reference to Identify Three-Dimensional Chromosome Contacts" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f45%2f382928">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

