

<!DOCTYPE html>
<html lang="en">

<head><title>
	Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/55/383179","@type":"ScholarlyArticle","name":"Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast","datePublished":"2013-01-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss033","keywords":["cell aggregation","map-based cloning","QTL analysis","<italic>Saccharomyces cerevisiae</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/55/383179","author":[{"name":"Li, Jiarui","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Wang, Lin","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Wu, Xiaoping","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Fang, Ou","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Wang, Luwen","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Lu, Chenqi","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Yang, Shengjie","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Hu, Xiaohua","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China","@type":"Person"},{"name":"Luo, Zewei","affiliation":"1  Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences ,  Fudan University ,  Shanghai 200433 ,  China    2  School of Biosciences, University of Birmingham, Birmingham B152TT, UK","@type":"Person"}],"description":"Cell aggregation in unicellular organisms, induced by either cell non-sexual adhesion to yield flocs and biofilm, or pheromone-driving sexual conjugation is of great significance in cellular stress response, medicine, and brewing industries. Most current literatures have focused on one form of cell aggregation termed flocculation and its major molecular determinants, the flocculation (FLO) family genes. Here, we implemented a map-based approach for dissecting the molecular basis of non-sexual cell aggregation in Saccharomyces cerevisiae. Genome-wide mapping has identified four major quantitative trait loci (QTL) underlying nature variation in the cell aggregation phenotype. High-resolution mapping following up with knockout and allele replacement experiments resolved the QTL into the underlying genes (AMN1, RGA1, FLO1, and FLO8) or even into the causative nucleotide. Genetic variation in the QTL genes can explain up to 46% of phenotypic variation of this trait. Of these genes, AMN1 plays the leading role, differing from the FLO family members, in regulating expression of cell clumping phenotype through inducing cell segregation defect. These findings provide novel insights into the molecular mechanism of how cell aggregation is regulated in budding yeast, and the data will be directly implicated to understand the molecular basis and evolutionary implications of cell aggregation in other fungus species.","pageStart":"55","pageEnd":"66","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1572217258&Signature=gLIhJkPy-E-cN2viEW3LaWb1U47XD35Fl2t8y94HcAuea-4uy547dAPkGkrbX5tzIYTNHentAHarIhihsAb4ki2GMFk7dzatd3N6By0nuI5jTRiVgtGhVRtYLVsJSJ0uQ4wEeo~4c7UQbUrrsekYegL7GNMVHGeYKaL-CUMqFP5QJZG7YRLsBjy~-4QGLeMhQFY9YI7mMcrttwm7fa0YsXWzzEZgC3408LtgXE6tGUP2cmOVdYSzh~ZRuXC1j1PNYnaYIRoNJRjdT5xRajeXveopabckuRUtvKvOUxBAdo3kAo1UUUZFPS2j~YLK~DEFjCZbE59QeAUKFU1jVrGBMQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1572217258&Signature=gLIhJkPy-E-cN2viEW3LaWb1U47XD35Fl2t8y94HcAuea-4uy547dAPkGkrbX5tzIYTNHentAHarIhihsAb4ki2GMFk7dzatd3N6By0nuI5jTRiVgtGhVRtYLVsJSJ0uQ4wEeo~4c7UQbUrrsekYegL7GNMVHGeYKaL-CUMqFP5QJZG7YRLsBjy~-4QGLeMhQFY9YI7mMcrttwm7fa0YsXWzzEZgC3408LtgXE6tGUP2cmOVdYSzh~ZRuXC1j1PNYnaYIRoNJRjdT5xRajeXveopabckuRUtvKvOUxBAdo3kAo1UUUZFPS2j~YLK~DEFjCZbE59QeAUKFU1jVrGBMQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Cell aggregation in unicellular organisms, induced by either cell non-sexual adhesion to yield flocs and biofilm, or pheromone-driving sexual conjugation is of great significance in cellular stress response, medicine, and brewing industries. Most current literatures have focused on one form of cell aggregation termed flocculation and its major molecular determinants, the flocculation (FLO) family genes. Here, we implemented a map-based approach for dissecting the molecular basis of non-sexual cell aggregation in Saccharomyces cerevisiae. Genome-wide mapping has identified four major quantitative trait loci (QTL) underlying nature variation in the cell aggregation phenotype. High-resolution mapping following up with knockout and allele replacement experiments resolved the QTL into the underlying genes (AMN1, RGA1, FLO1, and FLO8) or even into the causative nucleotide. Genetic variation in the QTL genes can explain up to 46% of phenotypic variation of this trait. Of these genes, AMN1 plays the leading role, differing from the FLO family members, in regulating expression of cell clumping phenotype through inducing cell segregation defect. These findings provide novel insights into the molecular mechanism of how cell aggregation is regulated in budding yeast, and the data will be directly implicated to understand the molecular basis and evolutionary implications of cell aggregation in other fungus species." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/55/383179" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217258&amp;Signature=KQ14-XUneWUQ2UrA~GiXfHhW8Ej5b3-Uz3aolvL8Z-mVt4YmCqAma9x2uVlsA93pNUJNLuE5IdcHbKKWYcJBOdQ966XrpzNxhLQ-98q7y6Fyk3f5Z-1zvzjbo6PCX5p9CgAZcLI-~aDucMQ1tDaA1Lj62TRSHncjyGMxlnH-zbgBjNUaq4g~579Pogn7DuS1LJDmuMcwg2f-O6s~PfNEeK~n~ibX-Y3N545XqPFPljVrHxLzcLfWD1bqAXMaxxA9Atf6IW~MmStnw-CVZMti6Mr6Dbkx7PjwdE0VGOHW7g9NgJOthQDCtEb-fh8kLu-EpyWHB2bnPAIPsl3pO8hZeA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2013-01-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Jiarui" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Wang, Lin" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Wu, Xiaoping" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Fang, Ou" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Wang, Luwen" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Lu, Chenqi" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Yang, Shengjie" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Hu, Xiaohua" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author" content="Luo, Zewei" /><meta name="citation_author_institution" content="Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Shanghai 200433, China" /><meta name="citation_author_institution" content="School of Biosciences, University of Birmingham, Birmingham B152TT, UK" /><meta name="citation_title" content="Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast" /><meta name="citation_firstpage" content="55" /><meta name="citation_lastpage" content="66" /><meta name="citation_doi" content="10.1093/dnares/dss033" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=FLO1 is a variable green beard gene that drives biofilm-like cooperation in budding yeast; Cell;  citation_year=2008;  citation_volume=135;  citation_pages=726-37; " /><meta name="citation_reference" content="citation_title=Unipolar cell divisions in the yeast S. cerevisiae lead to flamentous growth: regulation by starvation and RAS; Cell;  citation_year=1992;  citation_volume=68;  citation_pages=1077-90; " /><meta name="citation_reference" content="citation_title=Glucose depletion causes haploid invasive growth in yeast; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=13461-3; " /><meta name="citation_reference" content="citation_title=Tailoring wine yeast for the new millennium: novel approaches to the ancient art of winemaking; Yeast;  citation_year=2000;  citation_volume=16;  citation_pages=675-729; " /><meta name="citation_reference" content="citation_title=Localization of the dominant flocculation genes FLO5 and FLO8 of Saccharomyces cerevisiae; Yeast;  citation_year=1995;  citation_volume=30;  citation_pages=735-45; " /><meta name="citation_reference" content="citation_title=A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=12158-63; " /><meta name="citation_reference" content="citation_title=Molecular cloning and analysis of the dominant flocculation gene FLO8 from Saccharomyces cerevisiae; Mol. Gen. Genet.;  citation_year=1996;  citation_volume=251;  citation_pages=707-15; " /><meta name="citation_reference" content="citation_title=MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast FLO11 gene; EMBO J.;  citation_year=1999;  citation_volume=18;  citation_pages=1257-69; " /><meta name="citation_reference" content="citation_title=Origins of variation in the fungal cell surface; Nat. Rev. Microbiol.;  citation_year=2004;  citation_volume=2;  citation_pages=533-40; " /><meta name="citation_reference" content="citation_title=Flocculation, adhesion and biofilm formation in yeasts; Mol. Microbiol.;  citation_year=2006;  citation_volume=60;  citation_pages=5-15; " /><meta name="citation_reference" content="citation_title=The genetic architecture of quantitative traits; Annu. Rev. Genet.;  citation_year=2001;  citation_volume=35;  citation_pages=303-39; " /><meta name="citation_reference" content="citation_title=Dissecting the architecture of a quantitative trait locus in yeast; Nature;  citation_year=2002;  citation_volume=416;  citation_pages=326-30; " /><meta name="citation_reference" content="citation_title=Candidate-gene approaches for studying complex genetic traits: practical considerations; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=391-7; " /><meta name="citation_reference" content="citation_title=Integrating genotypic and expression data in a segregating mouse population to identify 5-lipoxygenase as a susceptibility gene for obesity and bone traits; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=1224-33; " /><meta name="citation_reference" content="citation_title=Genetic dissection of ethanol tolerance in the budding yeast Saccharomyces cerevisiae; Genetics;  citation_year=2007;  citation_volume=175;  citation_pages=1479-87; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press:, New York; Methods in Yeast Genetics. A Cold Spring Harbor Laboratory Course Manual;  citation_year=1998; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=A simple and efficient method for direct gene deletion in Saccharomyces cerevisiae; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=3329-30; " /><meta name="citation_reference" content="citation_title=Gene disruption with PCR products in Saccharomyces cerevisiae; Gene;  citation_year=1995;  citation_volume=158;  citation_pages=113-7; " /><meta name="citation_reference" content="citation_title=A rapid and simple method for preparation of RNA from Saccharomyces cerevisiae; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=3091-2; " /><meta name="citation_reference" content="citation_title=Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines; Yeast;  citation_year=1999;  citation_volume=15;  citation_pages=963-72; " /><meta name="citation_reference" content="citation_title=Genomic Libraries and a host strain designed for highly efficient two-hybrid selection in yeast; Genetics;  citation_year=1996;  citation_volume=144;  citation_pages=1425-36; " /><meta name="citation_reference" content="citation_title=The p21 Cdk-interacting protein Cip1 is a potent inhibitor of G1 cyclin-dependent kinases; Cell;  citation_year=1993;  citation_volume=75;  citation_pages=805-16; " /><meta name="citation_reference" content="citation_title=Bayesian inference in threshold models using Gibbs sampling; Genet. Sel. Evol.;  citation_year=1995;  citation_volume=27;  citation_pages=229-49; " /><meta name="citation_reference" content="citation_title=Multiple-interval mapping for ordinal traits; Genetics;  citation_year=2006;  citation_volume=173;  citation_pages=1649-63; " /><meta name="citation_reference" content="citation_title=Saccharomyces cerevisiae S288C has a mutation in FLO8, a gene required for filamentous growth; Genetics;  citation_year=1996;  citation_volume=144;  citation_pages=967-78; " /><meta name="citation_reference" content="citation_title=Analysis of the genes activated by the FLO8 gene in Saccharomyces cerevisiae; Curr. Genet.;  citation_year=1999;  citation_volume=36;  citation_pages=256-61; " /><meta name="citation_reference" content="citation_title=Exit from exit: resetting the cell cycle through Amn1 inhibition of G protein signaling; Cell;  citation_year=2003;  citation_volume=112;  citation_pages=697-709; " /><meta name="citation_reference" content="citation_title=Robust detection and genotyping of single feature polymorphisms from gene expression data; PLoS Comput. Biol.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000317" /><meta name="citation_reference" content="citation_title=Forkhead proteins control the outcome of transcription factor binding by antiactivation; EMBO J.;  citation_year=2007;  citation_volume=26;  citation_pages=4324-34; " /><meta name="citation_reference" content="citation_title=Yeast Cbk1 and Mob2 activate daughter specific genetic programs to induce asymmetric cell fates; Cell;  citation_year=2001;  citation_volume=107;  citation_pages=739-50; " /><meta name="citation_reference" content="citation_title=The role of Cdc42p GTPase-activating proteins in assembly of the septin ring in yeast; Mol. Biol. Cell;  citation_year=2003;  citation_volume=14;  citation_pages=4051-66; " /><meta name="citation_reference" content="citation_title=Adjacent positioning of cellular structures enabled by a Cdc42 GTPase-activating proteinâ€“mediated zone of inhibition; J. Cell Biol.;  citation_year=2007;  citation_volume=179;  citation_pages=1375-84; " /><meta name="citation_reference" content="citation_title=GTPase-activating proteins for Cdc42; Eukaryot. Cell;  citation_year=2002;  citation_volume=1;  citation_pages=469-80; " /><meta name="citation_reference" content="citation_title=Intragenic tandem repeats generate functional variability; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=986-90; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland, MA, USA; Genetics and Analysis of Quantitative Traits;  citation_year=1998; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci mapped to single-nucleotide resolution in yeast; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=1333-40; " /><meta name="citation_reference" content="citation_title=Genetic dissection of transcriptional regulation in budding yeast; Science;  citation_year=2002;  citation_volume=296;  citation_pages=752-5; " /><meta name="citation_reference" content="citation_title=CDD: a conserved domain database for the functional annotation of proteins; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=225-9; " /><meta name="citation_reference" content="citation_title=Transbilayer phospholipid flipping regulates Cdc42p signaling during poparized cell growth via Rga GTPase-activating proteins; Dev. Cell;  citation_year=2007;  citation_volume=13;  citation_pages=743-51; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/55/1086259/dss033.pdf" /><meta name="description" content="Abstract.  Cell aggregation in unicellular organisms, induced by either cell non-sexual adhesion to yield flocs and biofilm, or pheromone-driving sexual conjuga" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/55/383179" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/55/383179" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dss033';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./383179" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFv2E8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBKaWFydWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIFBvcHVsYXRpb24gJmFtcDsgUXVhbnRpdGF0aXZlIEdlbmV0aWNzLCBTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljIEVuZ2luZWVyaW5nLCBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgRnVkYW4gVW5pdmVyc2l0eSwgU2hhbmdoYWkgMjAwNDMzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIExpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUG9wdWxhdGlvbiAmYW1wOyBRdWFudGl0YXRpdmUgR2VuZXRpY3MsIFN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWMgRW5naW5lZXJpbmcsIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBGdWRhbiBVbml2ZXJzaXR5LCBTaGFuZ2hhaSAyMDA0MzMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV3UsIFhpYW9waW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQb3B1bGF0aW9uICZhbXA7IFF1YW50aXRhdGl2ZSBHZW5ldGljcywgU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgR2VuZXRpYyBFbmdpbmVlcmluZywgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIEZ1ZGFuIFVuaXZlcnNpdHksIFNoYW5naGFpIDIwMDQzMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYW5nLCBPdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUG9wdWxhdGlvbiAmYW1wOyBRdWFudGl0YXRpdmUgR2VuZXRpY3MsIFN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWMgRW5naW5lZXJpbmcsIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBGdWRhbiBVbml2ZXJzaXR5LCBTaGFuZ2hhaSAyMDA0MzMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgTHV3ZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIFBvcHVsYXRpb24gJmFtcDsgUXVhbnRpdGF0aXZlIEdlbmV0aWNzLCBTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljIEVuZ2luZWVyaW5nLCBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgRnVkYW4gVW5pdmVyc2l0eSwgU2hhbmdoYWkgMjAwNDMzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikx1LCBDaGVucWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIFBvcHVsYXRpb24gJmFtcDsgUXVhbnRpdGF0aXZlIEdlbmV0aWNzLCBTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljIEVuZ2luZWVyaW5nLCBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgRnVkYW4gVW5pdmVyc2l0eSwgU2hhbmdoYWkgMjAwNDMzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbmcsIFNoZW5namllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQb3B1bGF0aW9uICZhbXA7IFF1YW50aXRhdGl2ZSBHZW5ldGljcywgU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgR2VuZXRpYyBFbmdpbmVlcmluZywgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIEZ1ZGFuIFVuaXZlcnNpdHksIFNoYW5naGFpIDIwMDQzMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdSwgWGlhb2h1YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUG9wdWxhdGlvbiAmYW1wOyBRdWFudGl0YXRpdmUgR2VuZXRpY3MsIFN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWMgRW5naW5lZXJpbmcsIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBGdWRhbiBVbml2ZXJzaXR5LCBTaGFuZ2hhaSAyMDA0MzMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHVvLCBaZXdlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUG9wdWxhdGlvbiAmYW1wOyBRdWFudGl0YXRpdmUgR2VuZXRpY3MsIFN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWMgRW5naW5lZXJpbmcsIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBGdWRhbiBVbml2ZXJzaXR5LCBTaGFuZ2hhaSAyMDA0MzMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIEJpb3NjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIEJpcm1pbmdoYW0sIEJpcm1pbmdoYW0gQjE1MlRULCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUG9seWdlbmljIE1vbGVjdWxhciBBcmNoaXRlY3R1cmUgVW5kZXJseWluZyBOb24tU2V4dWFsIENlbGwgQWdncmVnYXRpb24gaW4gQnVkZGluZyBZZWFzdCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RkxPMSBpcyBhIHZhcmlhYmxlIGdyZWVuIGJlYXJkIGdlbmUgdGhhdCBkcml2ZXMgYmlvZmlsbS1saWtlIGNvb3BlcmF0aW9uIGluIGJ1ZGRpbmcgeWVhc3Q7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTM1OyAgY2l0YXRpb25fcGFnZXM9NzI2LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vbmlwb2xhciBjZWxsIGRpdmlzaW9ucyBpbiB0aGUgeWVhc3QgUy4gY2VyZXZpc2lhZSBsZWFkIHRvIGZsYW1lbnRvdXMgZ3Jvd3RoOiByZWd1bGF0aW9uIGJ5IHN0YXJ2YXRpb24gYW5kIFJBUzsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTEwNzctOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsdWNvc2UgZGVwbGV0aW9uIGNhdXNlcyBoYXBsb2lkIGludmFzaXZlIGdyb3d0aCBpbiB5ZWFzdDsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0xMzQ2MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYWlsb3Jpbmcgd2luZSB5ZWFzdCBmb3IgdGhlIG5ldyBtaWxsZW5uaXVtOiBub3ZlbCBhcHByb2FjaGVzIHRvIHRoZSBhbmNpZW50IGFydCBvZiB3aW5lbWFraW5nOyBZZWFzdDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTY3NS03Mjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2FsaXphdGlvbiBvZiB0aGUgZG9taW5hbnQgZmxvY2N1bGF0aW9uIGdlbmVzIEZMTzUgYW5kIEZMTzggb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBZZWFzdDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTczNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBTYWNjaGFyb215Y2VzIGdlbmUgZmFtaWx5IGludm9sdmVkIGluIGludmFzaXZlIGdyb3d0aCwgY2VsbC1jZWxsIGFkaGVzaW9uLCBhbmQgbWF0aW5nOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTEyMTU4LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2xvbmluZyBhbmQgYW5hbHlzaXMgb2YgdGhlIGRvbWluYW50IGZsb2NjdWxhdGlvbiBnZW5lIEZMTzggZnJvbSBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjUxOyAgY2l0YXRpb25fcGFnZXM9NzA3LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQVAga2luYXNlIGFuZCBjQU1QIGZpbGFtZW50YXRpb24gc2lnbmFsaW5nIHBhdGh3YXlzIGNvbnZlcmdlIG9uIHRoZSB1bnVzdWFsbHkgbGFyZ2UgcHJvbW90ZXIgb2YgdGhlIHllYXN0IEZMTzExIGdlbmU7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMjU3LTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW5zIG9mIHZhcmlhdGlvbiBpbiB0aGUgZnVuZ2FsIGNlbGwgc3VyZmFjZTsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTUzMy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmxvY2N1bGF0aW9uLCBhZGhlc2lvbiBhbmQgYmlvZmlsbSBmb3JtYXRpb24gaW4geWVhc3RzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz01LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2VuZXRpYyBhcmNoaXRlY3R1cmUgb2YgcXVhbnRpdGF0aXZlIHRyYWl0czsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0zMDMtMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3RpbmcgdGhlIGFyY2hpdGVjdHVyZSBvZiBhIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBpbiB5ZWFzdDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQxNjsgIGNpdGF0aW9uX3BhZ2VzPTMyNi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FuZGlkYXRlLWdlbmUgYXBwcm9hY2hlcyBmb3Igc3R1ZHlpbmcgY29tcGxleCBnZW5ldGljIHRyYWl0czogcHJhY3RpY2FsIGNvbnNpZGVyYXRpb25zOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zOTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpbmcgZ2Vub3R5cGljIGFuZCBleHByZXNzaW9uIGRhdGEgaW4gYSBzZWdyZWdhdGluZyBtb3VzZSBwb3B1bGF0aW9uIHRvIGlkZW50aWZ5IDUtbGlwb3h5Z2VuYXNlIGFzIGEgc3VzY2VwdGliaWxpdHkgZ2VuZSBmb3Igb2Jlc2l0eSBhbmQgYm9uZSB0cmFpdHM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTIyNC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXNzZWN0aW9uIG9mIGV0aGFub2wgdG9sZXJhbmNlIGluIHRoZSBidWRkaW5nIHllYXN0IFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9MTQ3OS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciBMYWJvcmF0b3J5IFByZXNzOiwgTmV3IFlvcms7IE1ldGhvZHMgaW4gWWVhc3QgR2VuZXRpY3MuIEEgQ29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnkgQ291cnNlIE1hbnVhbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFuZGVtIHJlcGVhdHMgZmluZGVyOiBhIHByb2dyYW0gdG8gYW5hbHl6ZSBETkEgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIGFuZCBlZmZpY2llbnQgbWV0aG9kIGZvciBkaXJlY3QgZ2VuZSBkZWxldGlvbiBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTMzMjktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZGlzcnVwdGlvbiB3aXRoIFBDUiBwcm9kdWN0cyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTU4OyAgY2l0YXRpb25fcGFnZXM9MTEzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmFwaWQgYW5kIHNpbXBsZSBtZXRob2QgZm9yIHByZXBhcmF0aW9uIG9mIFJOQSBmcm9tIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzA5MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGl0b3BlIHRhZ2dpbmcgb2YgeWVhc3QgZ2VuZXMgdXNpbmcgYSBQQ1ItYmFzZWQgc3RyYXRlZ3k6IG1vcmUgdGFncyBhbmQgaW1wcm92ZWQgcHJhY3RpY2FsIHJvdXRpbmVzOyBZZWFzdDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTk2My03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBMaWJyYXJpZXMgYW5kIGEgaG9zdCBzdHJhaW4gZGVzaWduZWQgZm9yIGhpZ2hseSBlZmZpY2llbnQgdHdvLWh5YnJpZCBzZWxlY3Rpb24gaW4geWVhc3Q7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE0NDsgIGNpdGF0aW9uX3BhZ2VzPTE0MjUtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwMjEgQ2RrLWludGVyYWN0aW5nIHByb3RlaW4gQ2lwMSBpcyBhIHBvdGVudCBpbmhpYml0b3Igb2YgRzEgY3ljbGluLWRlcGVuZGVudCBraW5hc2VzOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9ODA1LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXllc2lhbiBpbmZlcmVuY2UgaW4gdGhyZXNob2xkIG1vZGVscyB1c2luZyBHaWJicyBzYW1wbGluZzsgR2VuZXQuIFNlbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMjktNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlLWludGVydmFsIG1hcHBpbmcgZm9yIG9yZGluYWwgdHJhaXRzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzM7ICBjaXRhdGlvbl9wYWdlcz0xNjQ5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgUzI4OEMgaGFzIGEgbXV0YXRpb24gaW4gRkxPOCwgYSBnZW5lIHJlcXVpcmVkIGZvciBmaWxhbWVudG91cyBncm93dGg7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE0NDsgIGNpdGF0aW9uX3BhZ2VzPTk2Ny03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGdlbmVzIGFjdGl2YXRlZCBieSB0aGUgRkxPOCBnZW5lIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgQ3Vyci4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MjU2LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeGl0IGZyb20gZXhpdDogcmVzZXR0aW5nIHRoZSBjZWxsIGN5Y2xlIHRocm91Z2ggQW1uMSBpbmhpYml0aW9uIG9mIEcgcHJvdGVpbiBzaWduYWxpbmc7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9Njk3LTcwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9idXN0IGRldGVjdGlvbiBhbmQgZ2Vub3R5cGluZyBvZiBzaW5nbGUgZmVhdHVyZSBwb2x5bW9ycGhpc21zIGZyb20gZ2VuZSBleHByZXNzaW9uIGRhdGE7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAzMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3JraGVhZCBwcm90ZWlucyBjb250cm9sIHRoZSBvdXRjb21lIG9mIHRyYW5zY3JpcHRpb24gZmFjdG9yIGJpbmRpbmcgYnkgYW50aWFjdGl2YXRpb247IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MzI0LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ZZWFzdCBDYmsxIGFuZCBNb2IyIGFjdGl2YXRlIGRhdWdodGVyIHNwZWNpZmljIGdlbmV0aWMgcHJvZ3JhbXMgdG8gaW5kdWNlIGFzeW1tZXRyaWMgY2VsbCBmYXRlczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz03MzktNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIENkYzQycCBHVFBhc2UtYWN0aXZhdGluZyBwcm90ZWlucyBpbiBhc3NlbWJseSBvZiB0aGUgc2VwdGluIHJpbmcgaW4geWVhc3Q7IE1vbC4gQmlvbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQwNTEtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkamFjZW50IHBvc2l0aW9uaW5nIG9mIGNlbGx1bGFyIHN0cnVjdHVyZXMgZW5hYmxlZCBieSBhIENkYzQyIEdUUGFzZS1hY3RpdmF0aW5nIHByb3RlaW7igJNtZWRpYXRlZCB6b25lIG9mIGluaGliaXRpb247IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9MTM3NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R1RQYXNlLWFjdGl2YXRpbmcgcHJvdGVpbnMgZm9yIENkYzQyOyBFdWthcnlvdC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NDY5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyYWdlbmljIHRhbmRlbSByZXBlYXRzIGdlbmVyYXRlIGZ1bmN0aW9uYWwgdmFyaWFiaWxpdHk7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9OTg2LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U2luYXVlciBBc3NvY2lhdGVzLCBTdW5kZXJsYW5kLCBNQSwgVVNBOyBHZW5ldGljcyBhbmQgQW5hbHlzaXMgb2YgUXVhbnRpdGF0aXZlIFRyYWl0czsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgbWFwcGVkIHRvIHNpbmdsZS1udWNsZW90aWRlIHJlc29sdXRpb24gaW4geWVhc3Q7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTMzMy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXNzZWN0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIGluIGJ1ZGRpbmcgeWVhc3Q7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk2OyAgY2l0YXRpb25fcGFnZXM9NzUyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNERDogYSBjb25zZXJ2ZWQgZG9tYWluIGRhdGFiYXNlIGZvciB0aGUgZnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0yMjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNiaWxheWVyIHBob3NwaG9saXBpZCBmbGlwcGluZyByZWd1bGF0ZXMgQ2RjNDJwIHNpZ25hbGluZyBkdXJpbmcgcG9wYXJpemVkIGNlbGwgZ3Jvd3RoIHZpYSBSZ2EgR1RQYXNlLWFjdGl2YXRpbmcgcHJvdGVpbnM7IERldi4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTc0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzEvNTUvMTA4NjI1OS9kc3MwMzMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIENlbGwgYWdncmVnYXRpb24gaW4gdW5pY2VsbHVsYXIgb3JnYW5pc21zLCBpbmR1Y2VkIGJ5IGVpdGhlciBjZWxsIG5vbi1zZXh1YWwgYWRoZXNpb24gdG8geWllbGQgZmxvY3MgYW5kIGJpb2ZpbG0sIG9yIHBoZXJvbW9uZS1kcml2aW5nIHNleHVhbCBjb25qdWdhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODMxNzkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2RkaeaerQMIgM4dTpK66bAWZfdyONE=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f55%2f383179" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f55%2f383179" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=1509238858&amp;Signature=BCG0AGqqAqudwDpnztE3cN2QqAVla7qRoqBb-jqz5LhK46Ng0ZMxF~WLMfPPLhHl9T9tNKuBEZwZCTzG5DmS8Tc~arFw28vql~uIYUSilkLaustWdZhrVkKLVknPoxXFXBh1tA-EDlJhNPri5~DTZ5PqHJaYJggrePEmp8O8BRucGe3yl20qBOxewDbp8zWe60jbnAMgTGHayIET9N~jeszU-vcB4PyDudbr6ZMq878T-wC3WowQbycdX0q3dR0baeIcX-KdZCtzZNYXPmmx1ywAAnrfZ4NYd5NwM3uNCTlHA8eVneXim4CVad4vI-QIUFmIkCaM27apU72GQz-Qiw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5625506">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625506">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625508">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625508">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625511">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625511">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625549">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625549">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625598">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625598">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625610">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625610">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625615">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625615">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625619">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625619">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5625621">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5625621">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/1/45/382928">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/1/67/383475">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jiarui Li</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiarui Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jiarui+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Li author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Lin Wang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lin Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Lin+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wang author:L.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Xiaoping Wu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoping Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xiaoping+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wu author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Ou Fang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ou Fang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ou+Fang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fang O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Fang author:O.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Luwen Wang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Luwen Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Luwen+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Wang author:L.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Chenqi Lu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chenqi Lu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Chenqi+Lu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Lu author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Shengjie Yang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shengjie Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Shengjie+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yang author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Xiaohua Hu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaohua Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 21-65643966. Fax. +86 21-65648376. Email: <a href="mailto:xhhu@fudan.edu.cn" target="_blank">xhhu@fudan.edu.cn</a> (X.H); Tel. +86 21-55665342. Fax. +86 21-55665342. Email: <a href="mailto:zwluo@fudan.edu.cn" target="_blank">zwluo@fudan.edu.cn</a> (Z.L)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xiaohua+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hu author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Zewei Luo</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zewei Luo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Population &amp; Quantitative Genetics, State Key Laboratory of Genetic Engineering, School of Life Sciences</div>, <div class="insititution">Fudan University</div>, <div class="addr-line">Shanghai 200433</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">School of Biosciences, University of Birmingham, Birmingham B152TT, UK</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 21-65643966. Fax. +86 21-65648376. Email: <a href="mailto:xhhu@fudan.edu.cn" target="_blank">xhhu@fudan.edu.cn</a> (X.H); Tel. +86 21-55665342. Fax. +86 21-55665342. Email: <a href="mailto:zwluo@fudan.edu.cn" target="_blank">zwluo@fudan.edu.cn</a> (Z.L)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Zewei+Luo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luo Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Luo author:Z.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">â€ </div><div class="footnote-content">These two authors contributed equally to the research.</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 55â€“66, <a href='https://doi.org/10.1093/dnares/dss033'>https://doi.org/10.1093/dnares/dss033</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 January 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 October 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="383179" href="/dnaresearch/article-pdf/20/1/55/1086259/dss033.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jiarui Li, Lin Wang, Xiaoping Wu, Ou Fang, Luwen Wang, Chenqi Lu, Shengjie Yang, Xiaohua Hu, Zewei Luo;  Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast, <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 55â€“66, <a href='https://doi.org/10.1093/dnares/dss033'>https://doi.org/10.1093/dnares/dss033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/383179?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/383179?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/383179?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/383179?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/383179?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/383179?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/383179?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Polygenic%20Molecular%20Architecture%20Underlying%20Non-Sexual%20Cell%20Aggregation%20in%20Budding%20Yeast&amp;publicationDate=2013-01-02&amp;volumeNum=20&amp;issueNum=1&amp;author=Li%2C%20Jiarui%3B%20Wang%2C%20Lin&amp;startPage=55&amp;endPage=66&amp;contentId=10.1093%2Fdnares%2Fdss033&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f55%2f383179" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5625506 id="5625506" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Cell aggregation in unicellular organisms, induced by either cell non-sexual adhesion to yield flocs and biofilm, or pheromone-driving sexual conjugation is of great significance in cellular stress response, medicine, and brewing industries. Most current literatures have focused on one form of cell aggregation termed flocculation and its major molecular determinants, the flocculation (FLO) family genes. Here, we implemented a map-based approach for dissecting the molecular basis of non-sexual cell aggregation in <em>Saccharomyces cerevisiae</em>. Genome-wide mapping has identified four major quantitative trait loci (QTL) underlying nature variation in the cell aggregation phenotype. High-resolution mapping following up with knockout and allele replacement experiments resolved the QTL into the underlying genes (<em>AMN1</em>, <em>RGA1</em>, <em>FLO1</em>, and <em>FLO8</em>) or even into the causative nucleotide. Genetic variation in the QTL genes can explain up to 46% of phenotypic variation of this trait. Of these genes, <em>AMN1</em> plays the leading role, differing from the FLO family members, in regulating expression of cell clumping phenotype through inducing cell segregation defect. These findings provide novel insights into the molecular mechanism of how cell aggregation is regulated in budding yeast, and the data will be directly implicated to understand the molecular basis and evolutionary implications of cell aggregation in other fungus species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cell aggregation&quot;">cell aggregation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;map-based cloning&quot;">map-based cloning</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;QTL analysis&quot;">QTL analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Saccharomyces cerevisiae&quot;"><em>Saccharomyces cerevisiae</em></a></div>                            <h2 scrollto-destination=5625508 id="5625508" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Eukaryotic cells possess a remarkable capacity to aggregate together and form into a clump, which is crucial for the assembly of tissues and nervous systems, embryonic development, and cellular communication. <em>Saccharomyces cerevisiae</em>, the single-cell eukaryote, is also able to agglomerate cells into organized structures such as flocs, biofilm, and pseudohyphae. These non-sexual clumps create protective shields for cells in inner parts to escape from abiotic/biotic attacks,<sup><a reveal-id="DSS033C1" data-open="DSS033C1" class="link link-ref link-reveal xref-bibr">1</a></sup> respond to nutrient starvation,<sup><a reveal-id="DSS033C2" data-open="DSS033C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS033C3" data-open="DSS033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and have been widely used in industrial productions.<sup><a reveal-id="DSS033C4" data-open="DSS033C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Thus, understanding the underlying mechanisms of the cellular behaviour has significant medical and evolutionary implications. However, most studies in the literature have focused on flocculation, a non-sexual type of cell clumping, induced by interaction between cell wall glycoproteins encoded by highly homologous flocculation (FLO) genes (<em>FLO1</em>, <em>FLO5</em>, <em>FLO9</em>, and <em>FLO10</em>).<sup><a reveal-id="DSS033C5" data-open="DSS033C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS033C6" data-open="DSS033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In particular, <em>FLO1</em> was reported to be a key gene whose expression led to strong flocculation<sup><a reveal-id="DSS033C1" data-open="DSS033C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and was transcriptionally activated by <em>FLO8</em>.<sup><a reveal-id="DSS033C7" data-open="DSS033C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSS033C8" data-open="DSS033C8" class="link link-ref link-reveal xref-bibr">8</a></sup> So far, these studies followed a reverse genetic approach to screen for the flocculins that bound specific sugar residues on the surface of other cells and led to formation of cell flocs.<sup><a reveal-id="DSS033C9" data-open="DSS033C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS033C10" data-open="DSS033C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, the reverse approach has been debated for its requirement of prior information on the underlying pathways leading to phenotypic expression of the traits,<sup><a reveal-id="DSS033C11" data-open="DSS033C11" class="link link-ref link-reveal xref-bibr">11</a></sup> ignorance of the interactions between a gene of interest and the genetic background, and its incapacity to detect crucial polygenic interactions on phenotype.<sup><a reveal-id="DSS033C12  DSS033C14" data-open="DSS033C12  DSS033C14" class="link link-ref link-reveal xref-bibr">12â€“14</a></sup></p><p>We present here a forward genetic approach to uncover the molecular basis of genetic variation in non-sexual cell aggregation in a natural population of budding yeast (<em>S. cerevisiae</em>). The approach integrates mapping of quantitative trait loci (QTL) with major effects on the cell aggregation trait, resolving the major QTL into the QTL genes, identifying causative nucleotides within the QTL genes, and characterizing molecular functions of these QTL genes and their interactive effects on the trait.</p>                            <h2 scrollto-destination=5625511 id="5625511" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5625512 id="5625512" class="section-title" data-legacy-id=s2a>2.1. Yeast strains and growth conditions</h3>
<p>All yeast strains used in the present study are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. YH1A, an isogenic haploid strain of the standard reference strain S288C, was crossed with YL1C, also a haploid strain described elsewhere.<sup><a reveal-id="DSS033C15" data-open="DSS033C15" class="link link-ref link-reveal xref-bibr">15</a></sup> F<sub>1</sub> hybrids from crossing the parental lines were sporulated to generate 292 segregants that were equivalent to gametes, forming an F<sub>2</sub> generation. We created two other isogenic haploid strains from the parental lines with changed mating types through modifying their <em>HO</em> genes. We used the standard yeast media and growth conditions in mating, sporulation, and tetrad dissection experiments in the present study.<sup><a reveal-id="DSS033C16" data-open="DSS033C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=5625514 id="5625514" class="section-title" data-legacy-id=s2b>2.2. Scoring phenotype of cell aggregation</h3>
<p>The method used to score phenotype of cell aggregation in this study was modified from that proposed by Guo et al.<sup><a reveal-id="DSS033C6" data-open="DSS033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In detail, any strain preserved under â€“70Â°C was recovered by plate streaking, and three single clones for the strain were obtained. Each clone was cultured in a 5-ml YPD liquid medium at 30Â°C at 280 rpm for 22â€“24 h to make the cells enter the stationary phase of growth. The cultures were swirled briefly with a vortex shaker before a sediment test. Phenotype of the tested strain was recorded as a sedimentary time in hours (<em>T</em>), i.e. the time taken for the tested cells to sink completely in the culture buffer in a test tube. On the basis of the sedimentary time records, we grouped phenotype of all tested strains into four categories when contrasted with the sedimentary records of the parental strains, <em>T</em><sub>YH1A</sub> and <em>T</em><sub>YL1C</sub>, which are illustrated in Fig.Â <a reveal-id="DSS033F1" data-open="DSS033F1" class="link link-reveal link-table xref-fig">1</a>a. Phenotype of a tested strain was scored as one if <em>T</em> â‰¥ <em>T</em><sub>YH1A</sub>, two if <em>T</em><sub>YL1C</sub> &lt; <em>T</em> &lt; <em>T</em><sub>YH1A</sub>, three if 0 &lt; <em>T</em> â‰¤ <em>T</em><sub>YL1C,</sub> or four if <em>T</em> = 0. In other words, phenotype of cell aggregation was quantified as a categorical character with multiple thresholds in the present study. </p>                        <a id="5625516" scrollto-destination="5625516"></a>
<div id="DSS033F1" data-id="DSS033F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03301.jpeg?Expires=1509238859&Signature=VGGCB2PjOSlMzIXNLbHJyMVfbN7Fyo0MUd2~vOeah0972Ucj0P1SlUHNGE-OhXT0AQSapX~aMiLXSc2mOtpsR1OhWnTPyoQrqhKSS07Xa5Jhm6OzS1w6x6cXc5ULod1G165~JswhRzhwR87MdA9dZKWc6OBydqWibGUAmQT-QyfGadr0VDiFVo6vhzapTK0jvMIftSaPacQGuHJ2OUe-MuG-rg8M7NzhAOJT4~2A7ge71yDSl5JjHfZb6rQwKYnIQrEoBTRAueNHb7mH-EcABtio7OLDtymTuvTiHHCl3amVN3ljM4HPE-qGJTUeI0mMrby1NjSeFh2VMavJ3X1azw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="QTL fine mapping. (a) Cell aggregation phenotype of two parental strains YH1A and YL1C and their representative offspring segregants labelled 1â€“4. The phenotype was assayed as sediment time in hours (left) and by microscope viewing (right) at stationary phase of cell growth. Red horizontal line in the left panel marks the top boundary of cell sediment. (b) Mapping of major cell aggregation QTLs on the yeast chromosome II and XV and candidate genes in the QTL regions. The physical map information of the candidate genes was from Saccharomyces Genome Database (www.yeastgenome.org)." path-from-xml="dss03301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625516/dss03301.jpeg" path-from-xml="dss03301.jpeg" target="_blank">View large</a><a section="5625516" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03301.jpeg?Expires=1509238859&Signature=LZLqOXS~XEHdATx6FdIrGDYRH8~zaN~OYj5SjRrvKbxPHhwAxyrCvorcj8yRN2duGdTWA1JMCSL0jVX5ko1eiZA91db3RZhDND5MmhTc39xm1gK~0NoJioW~7mXvZIf3EXJXl33wVCan6xB7jjZ2GpHYOuwUbAMRqQ1UJglmF9GuTY9DsFrlKTmdXo-iRrqR~5mfCCvx2T~OluXuwGk5x8cJvWoLa5YYBEtj~L2xMagk9cqMi1U9anQPJBQNSnSwwbKh2zjDAP21lWcB8G~eZpLmK5NW~f0VlNe8Ex~S1yAdzVdpZw~XO7r-vC8~YPPQbfLISWhk-4rYc6PJLbPvOQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625516&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>QTL fine mapping. (a) Cell aggregation phenotype of two parental strains YH1A and YL1C and their representative offspring segregants labelled 1â€“4. The phenotype was assayed as sediment time in hours (left) and by microscope viewing (right) at stationary phase of cell growth. Red horizontal line in the left panel marks the top boundary of cell sediment. (b) Mapping of major cell aggregation QTLs on the yeast chromosome II and XV and candidate genes in the QTL regions. The physical map information of the candidate genes was from <em>Saccharomyces</em> Genome Database (<a class="link link-uri" href="http://www.yeastgenome.org">www.yeastgenome.org</a>).</p></div></div><div content-id="DSS033F1" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03301.jpeg?Expires=1509238859&Signature=VGGCB2PjOSlMzIXNLbHJyMVfbN7Fyo0MUd2~vOeah0972Ucj0P1SlUHNGE-OhXT0AQSapX~aMiLXSc2mOtpsR1OhWnTPyoQrqhKSS07Xa5Jhm6OzS1w6x6cXc5ULod1G165~JswhRzhwR87MdA9dZKWc6OBydqWibGUAmQT-QyfGadr0VDiFVo6vhzapTK0jvMIftSaPacQGuHJ2OUe-MuG-rg8M7NzhAOJT4~2A7ge71yDSl5JjHfZb6rQwKYnIQrEoBTRAueNHb7mH-EcABtio7OLDtymTuvTiHHCl3amVN3ljM4HPE-qGJTUeI0mMrby1NjSeFh2VMavJ3X1azw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="QTL fine mapping. (a) Cell aggregation phenotype of two parental strains YH1A and YL1C and their representative offspring segregants labelled 1â€“4. The phenotype was assayed as sediment time in hours (left) and by microscope viewing (right) at stationary phase of cell growth. Red horizontal line in the left panel marks the top boundary of cell sediment. (b) Mapping of major cell aggregation QTLs on the yeast chromosome II and XV and candidate genes in the QTL regions. The physical map information of the candidate genes was from Saccharomyces Genome Database (www.yeastgenome.org)." path-from-xml="dss03301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625516/dss03301.jpeg" path-from-xml="dss03301.jpeg" target="_blank">View large</a><a section="5625516" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03301.jpeg?Expires=1509238859&Signature=LZLqOXS~XEHdATx6FdIrGDYRH8~zaN~OYj5SjRrvKbxPHhwAxyrCvorcj8yRN2duGdTWA1JMCSL0jVX5ko1eiZA91db3RZhDND5MmhTc39xm1gK~0NoJioW~7mXvZIf3EXJXl33wVCan6xB7jjZ2GpHYOuwUbAMRqQ1UJglmF9GuTY9DsFrlKTmdXo-iRrqR~5mfCCvx2T~OluXuwGk5x8cJvWoLa5YYBEtj~L2xMagk9cqMi1U9anQPJBQNSnSwwbKh2zjDAP21lWcB8G~eZpLmK5NW~f0VlNe8Ex~S1yAdzVdpZw~XO7r-vC8~YPPQbfLISWhk-4rYc6PJLbPvOQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625516&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>QTL fine mapping. (a) Cell aggregation phenotype of two parental strains YH1A and YL1C and their representative offspring segregants labelled 1â€“4. The phenotype was assayed as sediment time in hours (left) and by microscope viewing (right) at stationary phase of cell growth. Red horizontal line in the left panel marks the top boundary of cell sediment. (b) Mapping of major cell aggregation QTLs on the yeast chromosome II and XV and candidate genes in the QTL regions. The physical map information of the candidate genes was from <em>Saccharomyces</em> Genome Database (<a class="link link-uri" href="http://www.yeastgenome.org">www.yeastgenome.org</a>).</p></div></div><p>In order for clear visualization of the cell aggregation phenotype, we present here microscope images for each of the four categorical phenotypes. To generate these images, the tested cells were first stained with Gram's iodine and then observed and photomicrographed using a Leica fluorescence microscope. All images illustrated in this paper were printed at the same magnification.</p>                            <h3 scrollto-destination=5625518 id="5625518" class="section-title" data-legacy-id=s2c>2.3. Marker development and genotyping</h3>
<p>We first searched the genome sequence of yeast <em>S. cerevisiae</em> (<a class="link link-uri" href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>) for short tandem repeats (STR) within the genome by making use of the computer software Tandem Repeats Finder 3.21.<sup><a reveal-id="DSS033C17" data-open="DSS033C17" class="link link-ref link-reveal xref-bibr">17</a></sup> We tested 561 evenly distributed STR sequences from the search as candidate markers and confirmed 264 STR loci that exhibited polymorphism between the two parental strains. These polymorphic markers were used as the basic marker set in the primary mapping experiment. Primer sequences for amplifying these STR markers were detailed in our previous study together with the experimental protocols for collecting and analysing STR genotype data.<sup><a reveal-id="DSS033C15" data-open="DSS033C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Beyond these STR markers, we developed a second set of single nucleotide polymorphism (SNP) markers to be used in the second stage of fine scale mapping of cell aggregation QTL by directly sequencing the relevant DNA regions. This added other 25 SNP markers in the QTL regions inferred from the primary QTL mapping (see marker locations and primer sequences in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p>                            <h3 scrollto-destination=5625522 id="5625522" class="section-title" data-legacy-id=s2d>2.4. Gene knockout, gene replacement, and nucleotide-specific mutagenesis</h3>
<p>We implemented the standard polymerase chain reaction (PCR)-based gene disruption techniques to knock out the candidate genes.<sup><a reveal-id="DSS033C18" data-open="DSS033C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS033C19" data-open="DSS033C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Single- and dual-knockout strains were constructed by replacing the target genes with the anti-hygromycin and anti-nourseothricin genes. The triple knockout strain was selected from offspring segregants from crossing of the dual-and single-knockout parental strains. All knockout strains obtained in the present study were confirmed by PCR experiments. The primer sequences for these PCRs are available upon request from the corresponding authors.</p><p>To perform the allelic exchange experiments and nucleotide-directed mutagenesis, we replaced the allele <em>in situ</em> with the replacement modules comprising the counterpart alleles and the Zeocin resistance gene as a dominant marker (detailed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S7 and S8</a>).</p>                            <h3 scrollto-destination=5625528 id="5625528" class="section-title" data-legacy-id=s2e>2.5. Preparation of RNA samples and RT-qPCR analysis</h3>
<p>Cells from a tested strain were cultured in 50 ml YPD liquid medium until reaching OD<sub>600</sub> = 0.8. Total RNA was isolated using the hot phenol protocol,<sup><a reveal-id="DSS033C20" data-open="DSS033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> purified with RNase-free DNase (Promega), and subjected to first-strand cDNA synthesis with SuperScript<sup>TM</sup> III Reverse transcriptase (Invitrogen). One microlitre of the single-strand cDNA after 10-fold dilution was used as template for real-time quantitative PCR (RT-qPCR) with SYBR-green (Toyobo) and <em>ACT1</em> as the internal control. Every tested strain was independently cultured three times to gain three independent samples, and each of the samples was assayed for three repeated times of qPCR. The oligonucleotide primers for the three genes assayed were RGA1f: 5â€²-CGGTACAGTCACTGGAATCGAA-3â€², RGA1r: 5â€²-ACGAATCAAACCTGGAGCAAA-3â€²ï¼›AMN1f: 5â€²-AGGCAAATACACACAGGTGGAA-3â€², AMN1r: 5â€²-TTCCCAAATACCGGCATCA-3â€²; and ACTINf: 5â€²-TCGTTCCAATTTACGCTGGTT-3â€², ACTINr: 5â€²-CGGCCAAATCGATTCTCAA-3â€²) for <em>RGA1</em>, <em>AMN1</em>, and <em>ACT1</em>, respectively.</p><p>We first sequenced the <em>FLO1</em> gene of the two parental strains, YL1C and YH1A, and found that there were only 44 bp in the <em>FLO1</em> gene, which distinguish <em>FLO1</em> from the paralogous genes <em>FLO5</em> and <em>FLO9</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). The sense primer of qPCR was designed across the 44-bp specific region, whereas the antisense primer on a non-specific region in 87-bp downstream. The respective specific sequences were â€˜GCCTCATCGCTATATGTTTTTGGâ€™ and â€˜GCTGGTAAGCACGCCTCTGTâ€™. The PCR product was sequenced and its specificity was confirmed.</p>                            <h3 scrollto-destination=5625535 id="5625535" class="section-title" data-legacy-id=s2f>2.6. Preparation of protein samples and western blotting assay</h3>
<p>We followed the PCR-based epitope tagging method to insert 6His into the C-terminal of the target protein as an epitope.<sup><a reveal-id="DSS033C21" data-open="DSS033C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Proteins were extracted using the protocol described by Adams et al.<sup><a reveal-id="DSS033C16" data-open="DSS033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Western blotting was performed using the standard laboratory procedures with monoclonal antibody specific for 6His (Univ. Co, China).</p>                            <h3 scrollto-destination=5625538 id="5625538" class="section-title" data-legacy-id=s2g>2.7. Yeast two-hybrid analysis</h3>
<p>Yeast two-hybrid assay in the present study was carried out according to the manufacturer's manual (Clontech, CA USA). Three alleles of <em>AMN1</em> (i.e. <em>AMN1<sup>YH1A</sup></em>, <em>AMN1<sup>YL1C</sup></em>, and <em>AMN1<sup>V368D</sup></em>) were cloned independently into the vector pGADT7 (GAD), whereas <em>TEM1</em> was inserted into the vector pGBKT7 (GBK). These vector constructs were confirmed by sequencing before being introduced into the reporter strains AH109 and Y187. These strains were then screened for positive co-transformation of the paired plasmids through growing them on the SC-Trp-Leu plates, synthetic complete medium lacking tryptophan and leucine. Positive interactions between <em>AMN1</em> and <em>TEM1</em> were detected as successful growth of the co-transformed strains on high-stringency SC-Trp-Leu-His plates (synthetic complete medium lacking tryptophan, leucine, and histidine) in the AH109 system.<sup><a reveal-id="DSS033C22" data-open="DSS033C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The positive interaction was also confirmed independently through the quantitative Î²-galactosidase assay in the Y187 system.<sup><a reveal-id="DSS033C23" data-open="DSS033C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h3 scrollto-destination=5625540 id="5625540" class="section-title" data-legacy-id=s2h>2.8. Data analyses</h3>

  <p>We implemented a Bayesian approach based on Gibbs sampling as proposed in Sorenson et al.<sup><a reveal-id="DSS033C24" data-open="DSS033C24" class="link link-ref link-reveal xref-bibr">24</a></sup> to estimate additive genetic variance component and, in turn, heritability of the multiple categorical phenotype of cell aggregation. In the statistical analysis, <em>y<sub>i</sub></em>, phenotype of the <em>i</em><sup>th</sup> segregant, is determined by a simple liability model <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq1.gif?Expires=1509238859&amp;Signature=aE92mPy62WDWKAROHi4HEKP4dyNn2mWT22ony9ajCp8UwN6BaAQm1Rqam7pzvSLXbeNukgnVIAY~oFzb8F5jZRxy3hnMG0o68FuzQv6ftNMPdg-OV4IoXrlJsWe~LxOZh6KLjUcSqzY09ovtcwqvBoZGLIIEiLWqq4XWi0eD~jOchD~orwkLj2M~oiQBPMpXYHjeDfhBq1EmhJAnJxMAHljv~Jm3Vj6wyI-98IsrWDM59TuCURWvsIvaarctp4zU1wyKSRNK6vWsGVN8FDTRLY58NDqGIXF3YkJLvidhEf4AmQv0984GpdgSb7SsC-xeK4qM51qhIi9o52MnAduJzg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq1.gif" /></span>, i.e. <em>y<sub>i</sub></em> = <em>j</em> (1 â‰¤ <em>j</em> â‰¤ 4), if <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq2.gif?Expires=1509238859&amp;Signature=PhDGR48U0m-My17wO~yTmEuSXWu4gPJZHGl~zdFxItH~cQjJ3Xikx4aTnZ0MxVPsmQ35aJUEkNU8ppHfBExBLVjDYUlbnO5x6vXnoERLd4bgOZ0qHIq0zGdTFF3tl~yKFZXpG62KmCQrsHuxdJy7a9ABmP5GdsIXvylMOdsPRYsyOUdsz85YpVVgvy0Tmbgl8RksQ3B3fsD5-~aZpGkQp2VvGxQh~H7R1L9W1SEfJUV6QKWp1gLg2a6w~lCd1rTFyoYgfXht0e7VdatkCvoYpdY11ZrCWLd-UzK0rMXewvaF9ukKfWKh0yeeflA~p2c1QU8Ji4i5d8hw4JYxLY6OJg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq2.gif" /></span>. In the liability model, <em>Î¼</em> is the population mean, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq3.gif?Expires=1509238859&amp;Signature=Oj-oXfeXQKC6z6jrWMA~9H2aaTTsZKH5a6FGgokYg1LJJOcPlFFsWFR0bJcs4IQX8GknlifWrpF9rgdUzgjw8RFLxdyQv-O7lyIGgd5j2XkOJMQI-CTLaokgUs~~jvcl4p4dZHj-ilni5iYcY5vvIV9UHtdgCPPuZyQjflj87NS75e9pOKVAJerbgPXXEWY0Dp3v~bgicivvlsIc~U6ymw75nxLyWwPXAT-eP3Lzv3zIoIHj0IQ4eI9pipLJ0LcwSlufIjO8ZhwrSFAJJ0w-4SKOJDL~mFwo8L3ZF2sLe6ogmcysTLGFPK7Iys2lclxPJKb0aD9DP90ufZbkM3Pn2A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq3.gif" /></span> is a random variable of additive genetic effect of the segregant on its phenotype, and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq4.gif?Expires=1509238859&amp;Signature=DiHFOafW1FD2u4Hry3qTfjJl-D7ahBig0GL8MoHKBUlIZ-103AKEmt9INhRTmim1AdF-FJgB3qkqfwbfgkVvGxX9wzlK91T1CAHqzycBCNY8pfDgyykHNtcwBMzVBZChl7s-Pni5IDNHHCqMYE7l1DYmHS67beWHQsYO926EceQ-Rp2cVd25ulRdtPo6fuKI02JsFpKridM26Y2COqzZCQevvggU4zKQAa6oGoHjXvioDErZ1DclDZpmBrVJBPA3qb2iUBBzicALlWyFVF8YT3~oCSWD5yJ4T0RDleo7shUUYVTn0y~av3RfTSyhyN0Rj-pNkVrSrRLmVn~h4ymAAg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq4.gif" /></span> is the residual or environmental random variable. We assumed that the random variables, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq5.gif?Expires=1509238859&amp;Signature=MBP8F-hyiRCYqFPOD3MLgxxOZXBnUqyaXi-~PQjtuJy-wjk2mWeW5uiMFuML7wxLcUg1Ozfk7R0~htaoraPwtt0eDAE5i1y1pryjKCuiKUFQyrB7cLfVS8AgSwOryMk2tr1b6HFIN1rySc6EdpvZ80xAhsivYMpZOCZXBldIShiqmH6jDBMu9okRex1XLPVqzY8ZV~C~IOgXmoevvWETkFPNXgzKuiLueW61z1n5CFWO59k9RLNWm7PrW8SpPHjUuaOjssBEPQM1kTTz8hwH9un1QsP3BBcj1FJyjNG66ktSzmV7FFo9qSpXwmAvjE9UVdEIB6h2sQZG-Bs9zV~b1g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq5.gif" /></span> and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq6.gif?Expires=1509238859&amp;Signature=N3UvEgMOwe5xGjrzGFnNMFeopZ688HsQEN-phOxnvEaHGryJxYttGoGbE74JSM01evIE1mBooZIZkN3Yby1ahtksvQc8Uwr602xhfKxOAML7ApbU8Vb-flEyFjKzf2lbdxe6HiEr0mtSsN-QipuJjlmV2LxSRKkMZfIc8kdaNrmEZ3o68vX~TatMg1w3m2t2BD7XBaEyTeeGNcesEo-bWya~bNB~7tmI6AOwpBsNwiepYNvH52i5jhA72kuXXW9xcfm~80GSHHQlo2L-DIbLydvu3I6KfIQyM59UhFplu6Tofv1vy5X8a9SD89uMrwli7VRREjIz0gu0SGdNtxw2YQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq6.gif" /></span>, follow independently a normal distribution with mean zero and variance <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq7.gif?Expires=1509238859&amp;Signature=UIdVbMDM~UZkfUKaLmbpz03yax1jql1LvTskn9eTgrBcGh~yldboTDRBcTxmK7qSMyKYr2xMA0la1V8hgicUp2xzQ-780U-gzeu~M6njawW324s4s4wCCg5WFUrlSAXtdqqKsr8hcod8utR5jxQwJYhpMEXlJUnWTHa6QphTPjHANXvnY~GF8jDw57dZi1wyRzgaLt3ULSClKqCaYgLTiMdeztBeJcZ1WhQIZ~E7DpRNee8ccb6km6Q6Pcsek5wfYWZdxFuUaeQb4pTrQ6UDeg~YRjvTWcKoElwxvoVN26ES3rwlOIcuLO0GIUogR8hc7Xxa~XUHZmxBULDbdc34oA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq7.gif" /></span> and a standard normal distribution, respectively. The threshold parameters satisfy <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss033ileq8.gif?Expires=1509238859&amp;Signature=M3kuZdRmEv~eKEWMjoeG0vMnewM6rlU15CKxlP0mzK9ahmC536OObkhdPJGaXR4H5tQNKrgjq0IawiacS8R95IT3C138AacXL8f2UdsR6VwBjvkpgnpnvTjuOnblOr5TIHWioVlwGAfVVC1p2TVzBr0ZfKlkiw5nzKziHyoIhRwXJLcWoSFfbDjNwWJ8LxtiisfzMekMCngo-QsbZrbuc3ZigpETG-MftdE24ydoPNPCDGWgW-mqPIDUypem4LIxd76c90msiOKB0Mn19qoGU8IDoa9-zraQy9KNbXtCcCInzc8wH0VVd7Ogc1hWO63Neoh-sHp9YISLnp6FbKOGgQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dss033ileq8.gif" /></span>. We implemented the algorithm developed by Dr Zhaobang Zeng at North Carolina State University for carrying out multiple-interval mapping of QTL affecting the multiple categorical phenotype of cell aggregation.<sup><a reveal-id="DSS033C25" data-open="DSS033C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p>
                            <h2 scrollto-destination=5625549 id="5625549" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5625552 id="5625552" class="section-title" data-legacy-id=s3a>3.1. Characterizing cell aggregation of yeast</h3>
<p>We first established a reliable and feasible method to quantify the aggregation phenotype, which was essential for a quantitative genetic analysis and further functional analysis with the cellular behaviour character. The left panel in Fig.Â <a reveal-id="DSS033F1" data-open="DSS033F1" class="link link-reveal link-table xref-fig">1</a>a shows sediment status of cells from the clumpy and non-clumpy parental strains YL1C and YH1A and their four representative segregants at three sediment time points, 0, 1.0, and 2.5 h. Thus, cell aggregation was, here, scored as a categorical quantitative trait, with the two parental strains, YH1A and YL1C, having phenotype categories 1 and 3, respectively and their offspring being scored as one of the four possible categories. In particular, cells of the segregant four sunk immediately in the clumping buffer (0 h) and the segregant was scored as category 4, a phenotype outside of the range of its two parents, showing transgressive segregation of the complex cell behaviour character. This method of categorizing the clumping character coordinates very well with that of the microscope views of the cell flocs, indicating the feasibility and reliability of the phenotyping assay.</p>                            <h3 scrollto-destination=5625555 id="5625555" class="section-title" data-legacy-id=s3b>3.2. Mapping and dissection of cell aggregation QTL</h3>
<p>From crossing the parental strains, YH1A and YL1C, that are highly divergent in cell aggregation phenotype, we created a full-sibling population comprising 292 segregants. We scored the segregants for their clumping phenotype and genotyped them at 264 segregating simple sequence repeat (SSR) markers. Based on the phenotype and marker genotype data, we implemented the multiple-interval mapping method<sup><a reveal-id="DSS033C25" data-open="DSS033C25" class="link link-ref link-reveal xref-bibr">25</a></sup> for a preliminary genome-wide scan for QTL affecting the multiple categorical phenotype of cell aggregation. The analysis detected four significant QTLs that mapped on the yeast chromosomes I, II, V, and XV, respectively. To narrow down these QTL regions, we genotyped the segregants at 1, 9, 4, and 11 further SNP markers in the 3 QTL regions, respectively and reran the multiple-interval mapping analysis. Because the chromosome I and V QTLs contain <em>FLO1</em> and <em>FLO8</em>, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), two previously known genes involved in flocculation in <em>S. cerevisiae</em>,<sup><a reveal-id="DSS033C5" data-open="DSS033C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS033C26" data-open="DSS033C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS033C27" data-open="DSS033C27" class="link link-ref link-reveal xref-bibr">27</a></sup> our focus here was on the other two QTLs on the yeast chromosomes II and XV. FigureÂ <a reveal-id="DSS033F1" data-open="DSS033F1" class="link link-reveal link-table xref-fig">1</a>b shows the likelihood profiles for the two major QTLs together with physical maps of candidate genes within the QTL regions. It can be seen that the fine mapping enables to resolve the chromosome II QTL into six candidate genes and the chromosome XV QTL into seven candidates.</p>                            <h3 scrollto-destination=5625557 id="5625557" class="section-title" data-legacy-id=s3c>3.3. Major-effect genes for cell aggregation and causative nucleotide in chromosome II QTL</h3>
<p>Among the six candidate genes within the chromosome II QTL (Fig.Â <a reveal-id="DSS033F1" data-open="DSS033F1" class="link link-reveal link-table xref-fig">1</a>b), three (<em>RIB7</em>, <em>RPB5</em>, and <em>CNS1</em>) are essential genes. We first knocked out each of the remaining three (<em>SLI15</em>, <em>ICS2</em>, and <em>AMN1</em>) in both the parental strains, YH1A and YL1C, and then evaluated the knockout strains for their clumping phenotype. We found that phenotype of the clumpy parental strain, YL1C, was dramatically altered after its <em>AMN1</em> gene was knocked out (Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>a). To further consolidate the candidacy of <em>AMN1</em> as the QTL gene, we carried out an allelic exchange experiment, in which the <em>AMN1</em> allele of the YL1C strain was replaced with that of the YH1A strain and vice versa. The aggregation phenotype of YL1C was clearly reduced after its <em>AMN1</em> allele was replaced by that of YH1A (Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>b). Moreover, the non-clumpy strain, YH1A, showed cell aggregation when its <em>AMN1</em> allele was replaced by that from YL1C (Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>b). These findings provide the strong evidence supporting that <em>AMN1</em> is indeed the major clumping QTL gene. It should be noted that effects of replacing AMN1 alleles are not reciprocally constant between the two parental strains. </p>                        <a id="5625561" scrollto-destination="5625561"></a>
<div id="DSS033F2" data-id="DSS033F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03302.jpeg?Expires=1509238859&Signature=B0zhAMph0WTdUoxZqCjGfPDu2iUptx8vXCiy6YahMeDvlco-LU5M60-XtC4ts8GRxXPjCUl1OBQ7rqtKDMg7Q26kXMvgDHfEvXA3du~HN-EoWcmDtVXpiENResS7MGl~vjTlgOcdPcEZsk4hxLg~JwNNrGzl5NF8UbK7OPhihG06WpPiShG89UI~Mgp1ghIfI-uGsycqdViNN77kr21bxTnliznmr60nIFr5jLcH3vz~-SBSll6j6OEpwXQiBvM8wOaq8mglebo-w1EstxQV9PRDF3GEyuhzwISjOL6L1lIACqK5ycDFe5p0p55zH1D20ITGkTCNzDL3x~eN3pJEmw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Cell aggregation phenotype of engineered parent strains across QTL genes. Cell aggregation phenotype that was scored by the sedimentary assay (the left panel) and microscope view (the right panel) for the two parental strains and their genetically modified strains through knock out or allele replacement. Red horizontal line in the left panel marks the top boundary of cell sediment. (a) The three QTL genes (AMN1, FLO8, and RGA1) were knocked out in the clumpy parent YL1C. â€˜Î”â€™ represents deletion, i.g. â€˜amn1Î”â€™ for deletion AMN1. (b) The alleles of three QTL genes were replaced in situ by their counterpart alleles between the parents. The superscripts â€˜Aâ€™ and â€˜Câ€™ refer to the YH1A-and YL1C-derived allele, respectively, i.g. AMN1A represents the allele of AMN1 derived from YH1A." path-from-xml="dss03302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625561/dss03302.jpeg" path-from-xml="dss03302.jpeg" target="_blank">View large</a><a section="5625561" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03302.jpeg?Expires=1509238859&Signature=cqPRf9dvyqwZfwPu2eSXA9Y96IIBJU83zVsDEefv9sOCqyLJIDiOCBFDdIKkaJEr-9Bc~37xwcUhLZedSzA97LVHTsem40JTNMCnxHMrg1YA7LLPuA0a08AMxZ2tjB2G98IlfeuNOZgxgCqq5DYy08DE6MPZSY5UKh1I9AlLCOlk6xy5otHbk65BfT8e9nDMLTRlsUD0TyXQO8qEEMEVV4SpbqGJB3pC1ZscZ0OUXmnwvsqqofCLpvGO-bRNHRvNyH6ZBZwZYDWLwItpmeDsKb~JmTIJkWF5kEoZm8odup4-AYEPIswAhYcG4DdxL1XKD9IU3nrHvEP7-ZUdf4X1CQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625561&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cell aggregation phenotype of engineered parent strains across QTL genes. Cell aggregation phenotype that was scored by the sedimentary assay (the left panel) and microscope view (the right panel) for the two parental strains and their genetically modified strains through knock out or allele replacement. Red horizontal line in the left panel marks the top boundary of cell sediment. (a) The three QTL genes (<em>AMN1</em>, <em>FLO8</em>, and <em>RGA1</em>) were knocked out in the clumpy parent YL1C. â€˜Î”â€™ represents deletion, i.g. â€˜<em>amn1</em>Î”â€™ for deletion <em>AMN1</em>. (b) The alleles of three QTL genes were replaced <em>in situ</em> by their counterpart alleles between the parents. The superscripts â€˜Aâ€™ and â€˜Câ€™ refer to the YH1A-and YL1C-derived allele, respectively, i.g. <em>AMN1<sup>A</sup></em> represents the allele of <em>AMN1</em> derived from YH1A.</p></div></div><div content-id="DSS033F2" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03302.jpeg?Expires=1509238859&Signature=B0zhAMph0WTdUoxZqCjGfPDu2iUptx8vXCiy6YahMeDvlco-LU5M60-XtC4ts8GRxXPjCUl1OBQ7rqtKDMg7Q26kXMvgDHfEvXA3du~HN-EoWcmDtVXpiENResS7MGl~vjTlgOcdPcEZsk4hxLg~JwNNrGzl5NF8UbK7OPhihG06WpPiShG89UI~Mgp1ghIfI-uGsycqdViNN77kr21bxTnliznmr60nIFr5jLcH3vz~-SBSll6j6OEpwXQiBvM8wOaq8mglebo-w1EstxQV9PRDF3GEyuhzwISjOL6L1lIACqK5ycDFe5p0p55zH1D20ITGkTCNzDL3x~eN3pJEmw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Cell aggregation phenotype of engineered parent strains across QTL genes. Cell aggregation phenotype that was scored by the sedimentary assay (the left panel) and microscope view (the right panel) for the two parental strains and their genetically modified strains through knock out or allele replacement. Red horizontal line in the left panel marks the top boundary of cell sediment. (a) The three QTL genes (AMN1, FLO8, and RGA1) were knocked out in the clumpy parent YL1C. â€˜Î”â€™ represents deletion, i.g. â€˜amn1Î”â€™ for deletion AMN1. (b) The alleles of three QTL genes were replaced in situ by their counterpart alleles between the parents. The superscripts â€˜Aâ€™ and â€˜Câ€™ refer to the YH1A-and YL1C-derived allele, respectively, i.g. AMN1A represents the allele of AMN1 derived from YH1A." path-from-xml="dss03302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625561/dss03302.jpeg" path-from-xml="dss03302.jpeg" target="_blank">View large</a><a section="5625561" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03302.jpeg?Expires=1509238859&Signature=cqPRf9dvyqwZfwPu2eSXA9Y96IIBJU83zVsDEefv9sOCqyLJIDiOCBFDdIKkaJEr-9Bc~37xwcUhLZedSzA97LVHTsem40JTNMCnxHMrg1YA7LLPuA0a08AMxZ2tjB2G98IlfeuNOZgxgCqq5DYy08DE6MPZSY5UKh1I9AlLCOlk6xy5otHbk65BfT8e9nDMLTRlsUD0TyXQO8qEEMEVV4SpbqGJB3pC1ZscZ0OUXmnwvsqqofCLpvGO-bRNHRvNyH6ZBZwZYDWLwItpmeDsKb~JmTIJkWF5kEoZm8odup4-AYEPIswAhYcG4DdxL1XKD9IU3nrHvEP7-ZUdf4X1CQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625561&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cell aggregation phenotype of engineered parent strains across QTL genes. Cell aggregation phenotype that was scored by the sedimentary assay (the left panel) and microscope view (the right panel) for the two parental strains and their genetically modified strains through knock out or allele replacement. Red horizontal line in the left panel marks the top boundary of cell sediment. (a) The three QTL genes (<em>AMN1</em>, <em>FLO8</em>, and <em>RGA1</em>) were knocked out in the clumpy parent YL1C. â€˜Î”â€™ represents deletion, i.g. â€˜<em>amn1</em>Î”â€™ for deletion <em>AMN1</em>. (b) The alleles of three QTL genes were replaced <em>in situ</em> by their counterpart alleles between the parents. The superscripts â€˜Aâ€™ and â€˜Câ€™ refer to the YH1A-and YL1C-derived allele, respectively, i.g. <em>AMN1<sup>A</sup></em> represents the allele of <em>AMN1</em> derived from YH1A.</p></div></div><p>We sequenced the <em>AMN1</em> alleles from the parental and several other laboratory strains and found three polymorphic sites within the Amn1 (Fig.Â <a reveal-id="DSS033F3" data-open="DSS033F3" class="link link-reveal link-table xref-fig">3</a>a). Meanwhile, we assayed their expression at both transcriptional and translational levels through RT-qPCR and western blotting assays. Although the <em>AMN1</em> gene is expressed significantly differentially between the two parental strains at both the mRNA and protein levels (Fig.Â <a reveal-id="DSS033F3" data-open="DSS033F3" class="link link-reveal link-table xref-fig">3</a>c), the polymorphisms in the open reading frame (ORF) are sufficient to lead to phenotypic variation according to the results of allelic exchange (Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>b).</p><p>We further used site-directed mutagenesis to identify the causative quantitative trait nucleotide(s) in the QTL gene <em>AMN1</em> by replacing the <em>AMN1</em> gene of the YL1C strain with three different DNA segments of 2150 bp in length from the strain YH1A, accordingly. The segments covered the whole coding sequence of the <em>AMN1</em> gene with the YH1A origin, but differed only at each of the three mutant nucleotides. We assayed these genetically modified strains for their clumping phenotype and found that only the strain that carried the V368D substitution in the <em>AMN1</em> gene could recover the phenotype of YL1C (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>a), strongly supporting the mutation as the causative quantitative trait nucleotide.</p>                            <h3 scrollto-destination=5625569 id="5625569" class="section-title" data-legacy-id=s3d>3.4. V368D variation determined the cell aggregation phenotype through the MEN pathway</h3>
<p>It has been reported that <em>AMN1</em> plays a key role as a daughter-specific switch that helps cells exit from the mitotic exit and reset the yeast cell cycle through binding to Tem1, a Ras-related GTPase. The <em>TEM1</em> gene activates a signal transduction pathway, mitotic exit network (MEN) to allow mitotic exit.<sup><a reveal-id="DSS033C28" data-open="DSS033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Based on this observation, we performed the yeast two-hybrid experiment in a standard strain AH109 and demonstrated that only Amn1<sup>YL1C</sup> and Amn1<sup>V368D</sup> can bind to Tem1 effectively (Fig.Â <a reveal-id="DSS033F4" data-open="DSS033F4" class="link link-reveal link-table xref-fig">4</a>a). The quantitative Î²-galactosidase analysis yielded the results that were highly consistent with the two-hybrid analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Furthermore, we compared expression of the downstream genes <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em> between YL1C and YH1A, using the Affymetrix microarray hybridization data described previously in Wang et al.<sup><a reveal-id="DSS033C29" data-open="DSS033C29" class="link link-ref link-reveal xref-bibr">29</a></sup> These genes were reported to underlie the cell wall degradation and daughter cell separation,<sup><a reveal-id="DSS033C30" data-open="DSS033C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and the expressions could be activated by Ace2, the transcriptional factor regulated by the MEN pathway.<sup><a reveal-id="DSS033C31" data-open="DSS033C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Our study shows a significantly lowered level of expression of these genes in YL1C than in YH1A (Fig.Â <a reveal-id="DSS033F4" data-open="DSS033F4" class="link link-reveal link-table xref-fig">4</a>b), suggesting that the transcription of <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em> would be inhibited in the clumpy parent strain, YL1C. Then, we constructed two strains both derived from YL1C, which differed only at the 368th amino acid residue of Amn1. The expression of these downstream genes <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em> exhibited significant difference between the strains (Fig.Â <a reveal-id="DSS033F4" data-open="DSS033F4" class="link link-reveal link-table xref-fig">4</a>c). These observations support the hypothesis that the V368D substitution in the Amn1 of the clumpy parent YL1C enhanced its binding to Tem1 and, in turn, blocked the MEN pathway, reduced transcription of the downstream genes <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em>, and finally resulted in inhibition of daughter cells' separation and formation of clumps (Fig.Â <a reveal-id="DSS033F4" data-open="DSS033F4" class="link link-reveal link-table xref-fig">4</a>d). </p>                        <a id="5625574" scrollto-destination="5625574"></a>
<div id="DSS033F4" data-id="DSS033F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03304.jpeg?Expires=1509238859&Signature=dAH1zlKzHh83Lp1mmniK9yIHd3eseEKbFw~TYxkRsb~5uu6yDVezeIFbHoA7PNFkU82lHHufExz9tW7gUJ7IpaL-pVCZEJVLbeQaB2Ecf3jv0lFeVTtJMXSgk5r4q7k6ptwKZM~2KL-Re7IEQkJxD5rrjkGLx~nKJwMg89GNetPsKcTQT~rxEbDvdQxtC~DKdpftnjcVxCV2SOoZQoA56E~m~GZ5Ntew5wDUhadnPTMpM39BAZH4Im~4KclWoTrYFfX3LPjx2vIat7fOReGJeMAwQ0Za-q-O-iTwT7DVe6mAHwRuvrtXL1T5uPLbT42Fa6TAAayYrMXVTgjo34wDuA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Functional verification of AMN1 underlying yeast cell aggregation. (a) Yeast two-hybrid assay on interaction between the AMN1 and TEM1 genes. (b) Expressional assay of the three downstream genes, DSE1, DSE2, and SCW11, from the two parental strains. (c) The microscopic view and the expression assay of DSE1, DSE2, and SCW11 between two strains derived from YL1C, only differing in the 368th amino acid residue of Amn1. (d) A hypothetical model on how the single nucleotide substitution (V368D) of Amn1 affects the MEN signal transduction pathway, which, in turn, determines daughter cell separation and formation of cell clumps." path-from-xml="dss03304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625574/dss03304.jpeg" path-from-xml="dss03304.jpeg" target="_blank">View large</a><a section="5625574" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03304.jpeg?Expires=1509238859&Signature=bdcaGg5bE61UeTwFjYz91XqUwiuP7slmP6Tg53gvf31gNvlLl-1wGSkQrDK84xi7ysKNXdblffYElRYOuOmPAPsYOrwsrwG8tFamHkTVqhqlsPRXZnPNIxnCTbnoCQ4CkTUlcmJX9RZHJClp1Jxcn-pjuuk4lgdaBR42WJP2-~dx-b8OgFEfQcACWGVbGt-nXuGXE-rZRSu-QRHHKtF5A34pI61fjIkOGDRht8mdVTMs1xzfJRnAGZKPS9EUwFNQjly0t~ITGjuWtAdIcyQXSHEhJyhYNmA2Py0WCxJ8r1ciK5aOZad-hLRNZJLu3nsMJbfblRVPy4pYlgW3Jx6LQw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625574&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional verification of <em>AMN1</em> underlying yeast cell aggregation. (a) Yeast two-hybrid assay on interaction between the <em>AMN1</em> and <em>TEM1</em> genes. (b) Expressional assay of the three downstream genes, <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em>, from the two parental strains. (c) The microscopic view and the expression assay of <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em> between two strains derived from YL1C, only differing in the 368th amino acid residue of Amn1. (d) A hypothetical model on how the single nucleotide substitution (V368D) of Amn1 affects the MEN signal transduction pathway, which, in turn, determines daughter cell separation and formation of cell clumps.</p></div></div><div content-id="DSS033F4" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03304.jpeg?Expires=1509238859&Signature=dAH1zlKzHh83Lp1mmniK9yIHd3eseEKbFw~TYxkRsb~5uu6yDVezeIFbHoA7PNFkU82lHHufExz9tW7gUJ7IpaL-pVCZEJVLbeQaB2Ecf3jv0lFeVTtJMXSgk5r4q7k6ptwKZM~2KL-Re7IEQkJxD5rrjkGLx~nKJwMg89GNetPsKcTQT~rxEbDvdQxtC~DKdpftnjcVxCV2SOoZQoA56E~m~GZ5Ntew5wDUhadnPTMpM39BAZH4Im~4KclWoTrYFfX3LPjx2vIat7fOReGJeMAwQ0Za-q-O-iTwT7DVe6mAHwRuvrtXL1T5uPLbT42Fa6TAAayYrMXVTgjo34wDuA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Functional verification of AMN1 underlying yeast cell aggregation. (a) Yeast two-hybrid assay on interaction between the AMN1 and TEM1 genes. (b) Expressional assay of the three downstream genes, DSE1, DSE2, and SCW11, from the two parental strains. (c) The microscopic view and the expression assay of DSE1, DSE2, and SCW11 between two strains derived from YL1C, only differing in the 368th amino acid residue of Amn1. (d) A hypothetical model on how the single nucleotide substitution (V368D) of Amn1 affects the MEN signal transduction pathway, which, in turn, determines daughter cell separation and formation of cell clumps." path-from-xml="dss03304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625574/dss03304.jpeg" path-from-xml="dss03304.jpeg" target="_blank">View large</a><a section="5625574" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03304.jpeg?Expires=1509238859&Signature=bdcaGg5bE61UeTwFjYz91XqUwiuP7slmP6Tg53gvf31gNvlLl-1wGSkQrDK84xi7ysKNXdblffYElRYOuOmPAPsYOrwsrwG8tFamHkTVqhqlsPRXZnPNIxnCTbnoCQ4CkTUlcmJX9RZHJClp1Jxcn-pjuuk4lgdaBR42WJP2-~dx-b8OgFEfQcACWGVbGt-nXuGXE-rZRSu-QRHHKtF5A34pI61fjIkOGDRht8mdVTMs1xzfJRnAGZKPS9EUwFNQjly0t~ITGjuWtAdIcyQXSHEhJyhYNmA2Py0WCxJ8r1ciK5aOZad-hLRNZJLu3nsMJbfblRVPy4pYlgW3Jx6LQw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625574&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional verification of <em>AMN1</em> underlying yeast cell aggregation. (a) Yeast two-hybrid assay on interaction between the <em>AMN1</em> and <em>TEM1</em> genes. (b) Expressional assay of the three downstream genes, <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em>, from the two parental strains. (c) The microscopic view and the expression assay of <em>DSE1</em>, <em>DSE2</em>, and <em>SCW11</em> between two strains derived from YL1C, only differing in the 368th amino acid residue of Amn1. (d) A hypothetical model on how the single nucleotide substitution (V368D) of Amn1 affects the MEN signal transduction pathway, which, in turn, determines daughter cell separation and formation of cell clumps.</p></div></div>                            <h3 scrollto-destination=5625575 id="5625575" class="section-title" data-legacy-id=s3e>3.5. <em>RGA1</em> as the chromosome XV clumping QTL gene</h3>
<p>To dissect the chromosome XV QTL, we screened for the candidate(s) that responded to knockout and gene replacement treatments. Of the seven ORFs within the QTL, only when the <em>RGA1</em> gene of the clumpy strain YL1C was silenced through knockout, its clumping phenotype was visibly altered (Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>a). In addition, the phenotype was also changed significantly when its <em>RGA1</em> allele was replaced with that from YH1A (Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>b). These observations strongly support the candidacy of <em>RGA1</em> as another major QTL gene. We compared the Rga1 sequence between the two parental strains and detected four polymorphic sites (Fig.Â <a reveal-id="DSS033F3" data-open="DSS033F3" class="link link-reveal link-table xref-fig">3</a>b). We failed to find a single nucleotide substitution that could explain the observed change in the phenotype, although we found that the third polymorphic site in the gene had a significant causative effect on invasive growth of the yeast cells (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Furthermore, we compared, but did not detect any significant change in expression of the <em>RGA1</em> alleles between the two parental strains at either the transcriptional or translational level (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). </p>                        <a id="5625577" scrollto-destination="5625577"></a>
<div id="DSS033F3" data-id="DSS033F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03303.jpeg?Expires=1509238860&Signature=G7t2uUpbj1IXriHL22GAxrutNiuOj07q7ss-2ZK1tBZHz-axmtnIkCgx2TIsQj1ohJXZEaLPQEP8Etf81l2I6UFha1Dl-Kp~wx1E-GgnRsk37U4PLYnrbMQUti6rehvJZ~dXP~UtoCubhb8mtB3IfV5ihpnwrNP2e1Jedka5LE~MXNvcVcOS5I4lUhQpvntWnrjMQefPlualh500pagnNnZ0TVqJNkRk-bfPUxwAOgGck8cY0c8XlOtMkJbWF4aFsBHYa~b7fS4sK~Mg3MN-aMZT1NBGRTV3Atoi9ygZ6~CIQbsXi7shNwDK6gNpbaasABbpaY4m95hT6GtwASS6Mw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genetic variation of QTL genes between two parental strains. Partial amino acid sequences of the cell aggregation QTL genes AMN1 (a) and RGA1 (b) with amino acid substitutions highlighted. The amino acid sequence in other yeast species is shown under the polymorphic residues when available: Saccharomyces paradoxus, Saccharomyces mikatae, Saccharomyces bayanus, Saccharomyces kudriavzevii, Saccharomyces kluyveri, Candida glabrata, Pichia angusta, Yarrowia lipolytica, and Kluyveromyces lactis. (c) Comparison in mRNA and protein expression of AMN1 between the two parental strains, YH1A and YL1C." path-from-xml="dss03303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625577/dss03303.jpeg" path-from-xml="dss03303.jpeg" target="_blank">View large</a><a section="5625577" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03303.jpeg?Expires=1509238860&Signature=CPE0u0w7R99KFjU90M72Mn4K7OSKvKvWwJqk7yioFqS2OJJiHPMI5iKybkP~dP3nSKnaDLK~DhHs2zfprG22c~pQPzlYaTxi10uOaU1iA0cxO3vXqXxmHHhvxrEwkf4Ck3iudmbCwBdwOKrxccxxEC8vuZjffPABtm-zC2QLfFDUAVLGjFvM2z-ywiNy92teyv2jDHLjRQbDeO1Z~Ge0X0TxDB3jCjMVJVho4bAVmd8gSknpbADxENwMtOMgFEMU-eB8e2tuF29OX9VeBqYzZxnkdnyP9UFYT7rFQJl3cGsL7sGnCA7Hgqv7KDTb1pdkYR71OlZKiTUoapZ0oEvudQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625577&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic variation of QTL genes between two parental strains. Partial amino acid sequences of the cell aggregation QTL genes <em>AMN1</em> (a) and <em>RGA1</em> (b) with amino acid substitutions highlighted. The amino acid sequence in other yeast species is shown under the polymorphic residues when available: <em>Saccharomyces paradoxus</em>, <em>Saccharomyces mikatae</em>, <em>Saccharomyces bayanus</em>, <em>Saccharomyces kudriavzevii</em>, <em>Saccharomyces kluyveri, Candida glabrata</em>, <em>Pichia angusta</em>, <em>Yarrowia lipolytica</em>, and <em>Kluyveromyces lactis</em>. (c) Comparison in mRNA and protein expression of <em>AMN1</em> between the two parental strains, YH1A and YL1C.</p></div></div><div content-id="DSS033F3" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03303.jpeg?Expires=1509238860&Signature=G7t2uUpbj1IXriHL22GAxrutNiuOj07q7ss-2ZK1tBZHz-axmtnIkCgx2TIsQj1ohJXZEaLPQEP8Etf81l2I6UFha1Dl-Kp~wx1E-GgnRsk37U4PLYnrbMQUti6rehvJZ~dXP~UtoCubhb8mtB3IfV5ihpnwrNP2e1Jedka5LE~MXNvcVcOS5I4lUhQpvntWnrjMQefPlualh500pagnNnZ0TVqJNkRk-bfPUxwAOgGck8cY0c8XlOtMkJbWF4aFsBHYa~b7fS4sK~Mg3MN-aMZT1NBGRTV3Atoi9ygZ6~CIQbsXi7shNwDK6gNpbaasABbpaY4m95hT6GtwASS6Mw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genetic variation of QTL genes between two parental strains. Partial amino acid sequences of the cell aggregation QTL genes AMN1 (a) and RGA1 (b) with amino acid substitutions highlighted. The amino acid sequence in other yeast species is shown under the polymorphic residues when available: Saccharomyces paradoxus, Saccharomyces mikatae, Saccharomyces bayanus, Saccharomyces kudriavzevii, Saccharomyces kluyveri, Candida glabrata, Pichia angusta, Yarrowia lipolytica, and Kluyveromyces lactis. (c) Comparison in mRNA and protein expression of AMN1 between the two parental strains, YH1A and YL1C." path-from-xml="dss03303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625577/dss03303.jpeg" path-from-xml="dss03303.jpeg" target="_blank">View large</a><a section="5625577" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03303.jpeg?Expires=1509238860&Signature=CPE0u0w7R99KFjU90M72Mn4K7OSKvKvWwJqk7yioFqS2OJJiHPMI5iKybkP~dP3nSKnaDLK~DhHs2zfprG22c~pQPzlYaTxi10uOaU1iA0cxO3vXqXxmHHhvxrEwkf4Ck3iudmbCwBdwOKrxccxxEC8vuZjffPABtm-zC2QLfFDUAVLGjFvM2z-ywiNy92teyv2jDHLjRQbDeO1Z~Ge0X0TxDB3jCjMVJVho4bAVmd8gSknpbADxENwMtOMgFEMU-eB8e2tuF29OX9VeBqYzZxnkdnyP9UFYT7rFQJl3cGsL7sGnCA7Hgqv7KDTb1pdkYR71OlZKiTUoapZ0oEvudQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625577&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic variation of QTL genes between two parental strains. Partial amino acid sequences of the cell aggregation QTL genes <em>AMN1</em> (a) and <em>RGA1</em> (b) with amino acid substitutions highlighted. The amino acid sequence in other yeast species is shown under the polymorphic residues when available: <em>Saccharomyces paradoxus</em>, <em>Saccharomyces mikatae</em>, <em>Saccharomyces bayanus</em>, <em>Saccharomyces kudriavzevii</em>, <em>Saccharomyces kluyveri, Candida glabrata</em>, <em>Pichia angusta</em>, <em>Yarrowia lipolytica</em>, and <em>Kluyveromyces lactis</em>. (c) Comparison in mRNA and protein expression of <em>AMN1</em> between the two parental strains, YH1A and YL1C.</p></div></div><p>Rga1, a member of the Cdc42 GTPase-activating protein family, has been reported as an apical and isotropic switch in controlling the budding pattern of yeast cells by influencing hydrolytic status.<sup><a reveal-id="DSS033C32" data-open="DSS033C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS033C33" data-open="DSS033C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In the presence of normally functional <em>RGA1</em>, Cdc42-GTP hydrolyzes and becomes Cdc42-GDP, leading to axial budding of cells that shows round cell morphology. The cells show bipolar budding and elongated morphology when their <em>RGA1</em> is silenced. However, it should be stressed that effect of the <em>RGA1</em> on cell budding and morphology would be only partial because there exist other family members such as <em>RGA2</em> and <em>BEM3</em>. Their presence may compensate for defects of the silent <em>RGA1</em>.<sup><a reveal-id="DSS033C34" data-open="DSS033C34" class="link link-ref link-reveal xref-bibr">34</a></sup> In light of the observation, we examined joint effects of the two verified QTL genes, <em>RGA1</em> and <em>AMN1</em>, on cell aggregation. FigureÂ <a reveal-id="DSS033F5" data-open="DSS033F5" class="link link-reveal link-table xref-fig">5</a> illustrates sequential influences of the two major clumping QTL genes, <em>RGA1</em> and <em>AMN1</em>, on budding pattern, cellular morphology, and cell separation of the clumpy strain YL1C. When the <em>AMN1</em> gene alone was knocked out whereas <em>RGA1</em> remained normal, the strain showed cell separation and largely round shape. When only <em>RGA1</em> was silenced whereas <em>AMN1</em> functioned normally, daughter cells of the strain were inhibited to separate from mother cells. Some of the cells then displayed bipolar budding and apical attachment between motherâ€“daughter cells in elongated shape, resulting in smaller clumps, and the presence of strings comprises a few apically linked cells. Whenever both the genes were knocked out in the clumpy strain, cells were promoted to bud in bipolarization, and inhibition of cell separation was released, leading to formation of bar-like cell clumps. These explain why single-knockout strain of <em>AMN1</em> has a more defected phenotype than dual-knockout strain. </p>                        <a id="5625579" scrollto-destination="5625579"></a>
<div id="DSS033F5" data-id="DSS033F5" class="fig fig-section"><div class="label fig-label">FigureÂ 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03305.jpeg?Expires=1509238860&Signature=f~IihMEUyZBWgbnntFWu8oQTJRor2I9PSt821TuzW3AaLt2kx-ynZtQDGEXypwW3L2-ANMPN54YayeC4JwodluY6S0EuWDqQQ8eKu3RD2zsEeQrlwy7xZgFdC2dL6TbUy9J9u6F9T~9lfy8C5I0imEn94pTGlT-kpcOgMKqCYUYvAZGbSBYIKlPz4OaXOw4Lr7EIqmOPwKKc34SIpdEYQGTPVpahMP74JzmVe5WHRdU~kol3OeSBof-m5BOt6fj9-qYtKvL3ZTBm~cKCQt~D452dPE0~N~LTlM3VYB~xYdMXn1UgBIt2f8CzY3Jstjjk34GQkHgitmrCprKZHVF1yQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Diagrammatic illustration of sequential effects of the two underlying QTL genes, AMN1 and RGA1. The two genes affect yeast cell aggregation through controlling cell separation and budding pattern, respectively. The black line indicates the effect of AMN1 on the aggregation trait and the blue the effect of RGA1 on the trait. The solid (or dotted) lines specify the genetic effects of the QTL genes in a normal (or dysfunctional) form." path-from-xml="dss03305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625579/dss03305.jpeg" path-from-xml="dss03305.jpeg" target="_blank">View large</a><a section="5625579" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03305.jpeg?Expires=1509238860&Signature=cOUeG2hfptzT72gzfZcWOXCeAJiEdUOxtrEAFk3uV8hy3yEXkucr3OCZdAr6c8oMth404V9CZxhFRzLK5~9kfqJWPpO37Pit1mXFmEPKZOPzUT96IJ0olh93x4NT8cojDC~akL9Lad~fptIyzXTvRpZ6~Ne56BqKokf5tTocH4HZ7t9NLj1n57LKVn1C30sL7eLQX-jeFvf462xzVxG-JO7Mny-9GBYx5gykdB1XVNL95vbERC-Hf-1r22I54WoC7XsI2vy8maheIcD5YUVRB3CVA2cptmX-Asa3lJS80shvUi2mj3ASDjPeyiNBPnU6tonglUINCw1G9pRINkLsuw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625579&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Diagrammatic illustration of sequential effects of the two underlying QTL genes, <em>AMN1</em> and <em>RGA1</em>. The two genes affect yeast cell aggregation through controlling cell separation and budding pattern, respectively. The black line indicates the effect of AMN1 on the aggregation trait and the blue the effect of RGA1 on the trait. The solid (or dotted) lines specify the genetic effects of the QTL genes in a normal (or dysfunctional) form.</p></div></div><div content-id="DSS033F5" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03305.jpeg?Expires=1509238860&Signature=f~IihMEUyZBWgbnntFWu8oQTJRor2I9PSt821TuzW3AaLt2kx-ynZtQDGEXypwW3L2-ANMPN54YayeC4JwodluY6S0EuWDqQQ8eKu3RD2zsEeQrlwy7xZgFdC2dL6TbUy9J9u6F9T~9lfy8C5I0imEn94pTGlT-kpcOgMKqCYUYvAZGbSBYIKlPz4OaXOw4Lr7EIqmOPwKKc34SIpdEYQGTPVpahMP74JzmVe5WHRdU~kol3OeSBof-m5BOt6fj9-qYtKvL3ZTBm~cKCQt~D452dPE0~N~LTlM3VYB~xYdMXn1UgBIt2f8CzY3Jstjjk34GQkHgitmrCprKZHVF1yQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Diagrammatic illustration of sequential effects of the two underlying QTL genes, AMN1 and RGA1. The two genes affect yeast cell aggregation through controlling cell separation and budding pattern, respectively. The black line indicates the effect of AMN1 on the aggregation trait and the blue the effect of RGA1 on the trait. The solid (or dotted) lines specify the genetic effects of the QTL genes in a normal (or dysfunctional) form." path-from-xml="dss03305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625579/dss03305.jpeg" path-from-xml="dss03305.jpeg" target="_blank">View large</a><a section="5625579" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03305.jpeg?Expires=1509238860&Signature=cOUeG2hfptzT72gzfZcWOXCeAJiEdUOxtrEAFk3uV8hy3yEXkucr3OCZdAr6c8oMth404V9CZxhFRzLK5~9kfqJWPpO37Pit1mXFmEPKZOPzUT96IJ0olh93x4NT8cojDC~akL9Lad~fptIyzXTvRpZ6~Ne56BqKokf5tTocH4HZ7t9NLj1n57LKVn1C30sL7eLQX-jeFvf462xzVxG-JO7Mny-9GBYx5gykdB1XVNL95vbERC-Hf-1r22I54WoC7XsI2vy8maheIcD5YUVRB3CVA2cptmX-Asa3lJS80shvUi2mj3ASDjPeyiNBPnU6tonglUINCw1G9pRINkLsuw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625579&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Diagrammatic illustration of sequential effects of the two underlying QTL genes, <em>AMN1</em> and <em>RGA1</em>. The two genes affect yeast cell aggregation through controlling cell separation and budding pattern, respectively. The black line indicates the effect of AMN1 on the aggregation trait and the blue the effect of RGA1 on the trait. The solid (or dotted) lines specify the genetic effects of the QTL genes in a normal (or dysfunctional) form.</p></div></div>                            <h3 scrollto-destination=5625581 id="5625581" class="section-title" data-legacy-id=s3f>3.6. <em>FLO1</em>, <em>FLO8</em>, and transgressive segregation</h3>
<p>Our QTL analysis detected two significant QTL regions containing the two genes relevant to flocculation, <em>FLO1</em> and <em>FLO8</em>. To test their candidacy as major cell aggregation genes, we first compared sequence of the two genes between the two parental strains, YH1A and YL1C, and found that the <em>FLO1</em> allele in the clumpy strain YL1C had eight fewer tandem repeats of an intragenic domain than those in the non-clumpy parent YH1A. This disagrees with the previous observations that the degree of cell flocculation was found to be proportional to the number of tandem repeats of the intragenic domain structure.<sup><a reveal-id="DSS033C1" data-open="DSS033C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS033C35" data-open="DSS033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Comparison of genic sequence of <em>FLO8</em> between the two parental strains revealed a nonsense substitution at the 425th nucleotide in YH1A, consistent with the previous study.<sup><a reveal-id="DSS033C26" data-open="DSS033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The mutation creates a stop codon (TAG) and results in a truncated protein. The defective <em>FLO8</em> in YH1A cannot activate expression of its <em>FLO1</em>, thus explaining why the <em>FLO1</em> gene did not lead to stronger flocculation as previously expected, although the non-clumpy strain carried the allele enhancing flocculation phenotype.</p><p>We knocked out the <em>FLO1</em> and <em>FLO8</em> genes individually in YH1A and YL1C and observed that the clumping phenotype of YL1C was markedly altered after either <em>FLO1</em> or <em>FLO8</em> gene was silenced (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>b). Moreover, we replaced the <em>FLO8</em> allele in YL1C with the YH1A-derived counterpart and observed a similar phenotype to the <em>FLO1</em> or <em>FLO8</em> deletion strain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>b). The data showed that genetic variation of both the FLO genes contributed significantly to the clumpy phenotype. We attempted but failed to replace the <em>FLO1</em> gene in the same way as reported above. This can be attributed to the difficulties arising from the location of the <em>FLO1</em> gene at the chromosomal end with rich heterochromatin, thus keeping the gene in a protected status from genetic modification. Secondly, the <em>FLO1</em> gene has an extremely high level of homology with its family members <em>FLO5</em> and <em>FLO9</em>, making it extremely difficult to ensure specificity in replacing the target gene.</p><p>Although it was highly technically challenging to profile expression of the <em>FLO1</em> gene due to its extremely high level of sequence homology with other FLO family members <em>FLO5</em> and <em>FLO9</em>, we succeeded in assaying expression of the FLO gene in the two parental strains through an <em>FLO1</em>-specific RT-qPCR experiment (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a> for detail). We analysed the <em>FLO1</em> and <em>FLO8</em> expression levels of the parental strains and their eight offspring individuals with two different genotypes at the QTL genes and varying phenotypes. <em>FLO8</em> expression showed no difference in the tested strains. Moreover, a highly expressed <em>FLO1</em> gene was common for the strong cell clumping phenotype (the phenotype categories 3 and 4) but was certainly not an indicator of clumpiness because the gene was also observed to be highly expressed in the non-clumpy strains (those with phenotype category 1) (Fig.Â <a reveal-id="DSS033F6" data-open="DSS033F6" class="link link-reveal link-table xref-fig">6</a>a). </p>                        <a id="5625590" scrollto-destination="5625590"></a>
<div id="DSS033F6" data-id="DSS033F6" class="fig fig-section"><div class="label fig-label">FigureÂ 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03306.jpeg?Expires=1509238860&Signature=cG4pqr5QzhAD7MtuUK0DnD3uKbya5xmHjpgfty5Sk4j-rGDbVu~UWrJS6HUbjv~~qfGo0RTc5GjUIFdCmBbRkOUds2DAV-0hgDL0Qh-YfsMadjc~Zqgo3~vfghmUOSzM8iT6G1~KWhfWEc0c8TZM9IZ4aBhLc8sOjkVJA3VHcRWIPL9T8yHFa42jrggbnOUPJZf9UEQNCcc9HidV-epmYqa-OhmrsAVg32JlUwVRBp1152eI3qPgo3oidKhG0fZil7u-a9UoMVNBZ7G22CSDHIR8gpFel1molrfPjz0Yhzz~IZoHJKimxg8aGZV54xNEJ22BYN~bU~rJ~DWmLyz56g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genetic interactions of the QTL genes. â€˜Aâ€™ and â€˜Câ€™ in (a) and (b) refer to as the YH1A- and YL1C-derived alleles, respectively. (a) Expression of FLO1 and FLO8 in two parental strains and their two groups of offspring segregants. The genotype of each strain at the four QTL genes is shown below, accordingly. (b) Distribution of segregants with different cell aggregation phenotypes (1â€“4) within each of genotypes at the four QTL genes. Each column represents a group of individuals shared with the same genotype at the four QTL genes. (c) Cell aggregation phenotype of the two parental strains and their two selected offspring segregants (S2 and S5) after different treatments of ethylenediaminetetraacetic acid and CaCl2. Panel 1(control) corresponds to where cells of the tested strains were scored without being treated, panel 2 (0.1 M ethylenediaminetetraacetic acid) to where the cells were washed twice with 0.1 M ethylenediaminetetraacetic acid and other two times with sterile water before being tested, and panel 3 (0.1% CaCl2) to where the cells were treated in the same way as those in panel 2, but tested in the flocculation buffer with 0.1% CaCl2 added. Red horizontal line represents the top boundary of cell sediment." path-from-xml="dss03306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625590/dss03306.jpeg" path-from-xml="dss03306.jpeg" target="_blank">View large</a><a section="5625590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03306.jpeg?Expires=1509238860&Signature=ePCiDp6I5sFRDUIInOHSVx0xAXJ3Qa-imFyazGxyaqbalxfPv5jDS2GU9ay9yWazNfusFZ6Rjl2o2-GlRRIgxUpzspIG3amoRAQ5FqOtdWuasgrfmhtZTdIyxo7Ib~E0Cm07M4swjfvCuydHN7BhFpSJHJ-tN79Ylg0NokgzKWtZVFhZJIUVsr8aWgK2AsYq6HQSapu0YZdhZbH8FtyBJCe5gCrpuBmrPI07-J5tgvZv561r5QwpGde948OVY0gLxngsmbLoXf~KOEimaBRdnwytpOy6GZKcff46dz0sMXvgHMXlwGeqDzOjEyyQb-~SKOgi5VFtfesjRwh4cCXBvA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625590&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic interactions of the QTL genes. â€˜Aâ€™ and â€˜Câ€™ in (a) and (b) refer to as the YH1A- and YL1C-derived alleles, respectively. (a) Expression of <em>FLO1</em> and <em>FLO8</em> in two parental strains and their two groups of offspring segregants. The genotype of each strain at the four QTL genes is shown below, accordingly. (b) Distribution of segregants with different cell aggregation phenotypes (1â€“4) within each of genotypes at the four QTL genes. Each column represents a group of individuals shared with the same genotype at the four QTL genes. (c) Cell aggregation phenotype of the two parental strains and their two selected offspring segregants (S2 and S5) after different treatments of ethylenediaminetetraacetic acid and CaCl<sub>2</sub>. Panel 1(control) corresponds to where cells of the tested strains were scored without being treated, panel 2 (0.1 M ethylenediaminetetraacetic acid) to where the cells were washed twice with 0.1 M ethylenediaminetetraacetic acid and other two times with sterile water before being tested, and panel 3 (0.1% CaCl<sub>2</sub>) to where the cells were treated in the same way as those in panel 2, but tested in the flocculation buffer with 0.1% CaCl<sub>2</sub> added. Red horizontal line represents the top boundary of cell sediment.</p></div></div><div content-id="DSS033F6" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/m_dss03306.jpeg?Expires=1509238860&Signature=cG4pqr5QzhAD7MtuUK0DnD3uKbya5xmHjpgfty5Sk4j-rGDbVu~UWrJS6HUbjv~~qfGo0RTc5GjUIFdCmBbRkOUds2DAV-0hgDL0Qh-YfsMadjc~Zqgo3~vfghmUOSzM8iT6G1~KWhfWEc0c8TZM9IZ4aBhLc8sOjkVJA3VHcRWIPL9T8yHFa42jrggbnOUPJZf9UEQNCcc9HidV-epmYqa-OhmrsAVg32JlUwVRBp1152eI3qPgo3oidKhG0fZil7u-a9UoMVNBZ7G22CSDHIR8gpFel1molrfPjz0Yhzz~IZoHJKimxg8aGZV54xNEJ22BYN~bU~rJ~DWmLyz56g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Genetic interactions of the QTL genes. â€˜Aâ€™ and â€˜Câ€™ in (a) and (b) refer to as the YH1A- and YL1C-derived alleles, respectively. (a) Expression of FLO1 and FLO8 in two parental strains and their two groups of offspring segregants. The genotype of each strain at the four QTL genes is shown below, accordingly. (b) Distribution of segregants with different cell aggregation phenotypes (1â€“4) within each of genotypes at the four QTL genes. Each column represents a group of individuals shared with the same genotype at the four QTL genes. (c) Cell aggregation phenotype of the two parental strains and their two selected offspring segregants (S2 and S5) after different treatments of ethylenediaminetetraacetic acid and CaCl2. Panel 1(control) corresponds to where cells of the tested strains were scored without being treated, panel 2 (0.1 M ethylenediaminetetraacetic acid) to where the cells were washed twice with 0.1 M ethylenediaminetetraacetic acid and other two times with sterile water before being tested, and panel 3 (0.1% CaCl2) to where the cells were treated in the same way as those in panel 2, but tested in the flocculation buffer with 0.1% CaCl2 added. Red horizontal line represents the top boundary of cell sediment." path-from-xml="dss03306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5625590/dss03306.jpeg" path-from-xml="dss03306.jpeg" target="_blank">View large</a><a section="5625590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss03306.jpeg?Expires=1509238860&Signature=ePCiDp6I5sFRDUIInOHSVx0xAXJ3Qa-imFyazGxyaqbalxfPv5jDS2GU9ay9yWazNfusFZ6Rjl2o2-GlRRIgxUpzspIG3amoRAQ5FqOtdWuasgrfmhtZTdIyxo7Ib~E0Cm07M4swjfvCuydHN7BhFpSJHJ-tN79Ylg0NokgzKWtZVFhZJIUVsr8aWgK2AsYq6HQSapu0YZdhZbH8FtyBJCe5gCrpuBmrPI07-J5tgvZv561r5QwpGde948OVY0gLxngsmbLoXf~KOEimaBRdnwytpOy6GZKcff46dz0sMXvgHMXlwGeqDzOjEyyQb-~SKOgi5VFtfesjRwh4cCXBvA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=5625590&ar=383179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic interactions of the QTL genes. â€˜Aâ€™ and â€˜Câ€™ in (a) and (b) refer to as the YH1A- and YL1C-derived alleles, respectively. (a) Expression of <em>FLO1</em> and <em>FLO8</em> in two parental strains and their two groups of offspring segregants. The genotype of each strain at the four QTL genes is shown below, accordingly. (b) Distribution of segregants with different cell aggregation phenotypes (1â€“4) within each of genotypes at the four QTL genes. Each column represents a group of individuals shared with the same genotype at the four QTL genes. (c) Cell aggregation phenotype of the two parental strains and their two selected offspring segregants (S2 and S5) after different treatments of ethylenediaminetetraacetic acid and CaCl<sub>2</sub>. Panel 1(control) corresponds to where cells of the tested strains were scored without being treated, panel 2 (0.1 M ethylenediaminetetraacetic acid) to where the cells were washed twice with 0.1 M ethylenediaminetetraacetic acid and other two times with sterile water before being tested, and panel 3 (0.1% CaCl<sub>2</sub>) to where the cells were treated in the same way as those in panel 2, but tested in the flocculation buffer with 0.1% CaCl<sub>2</sub> added. Red horizontal line represents the top boundary of cell sediment.</p></div></div><p>Intriguingly, 30 extremely clumpy segregants (<em>T</em> = 0) exhibited more strongly clumpy than parental strains, which is termed as transgressive segregation. To explore how the QTL genes affect the transgressive segregation, we first grouped the 292 segregants into 4 panels of 16 groups according to their allelic origin or joint genotype at the 4 QTL genes. FigureÂ <a reveal-id="DSS033F6" data-open="DSS033F6" class="link link-reveal link-table xref-fig">6</a>b illustrates distributions of the segregants with different phenotypes (1â€“4) within each of the four-gene genotypes and shows that the segregants receiving the four QTL genes from the clumpy parent, YL1C, have a much lower probability of expressing extreme clumpy phenotype (phenotypic category 4) than those with a YH1A-derived allele of <em>FLO1</em>, indicating a dispersion distribution of cell aggregation causing alleles at the QTL genes between the two yeast parental strains and thus explaining the transgressive segregation observed in the segregant population. Moreover, the phenotypic effect of the <em>FLO1</em> allele was highly background dependent. This <em>FLO1</em> allele would not cause the clumping phenotype in the absence of the other three QTL genes from the clumpy parent YL1C (group 7 vs. group 3 in Fig.Â <a reveal-id="DSS033F6" data-open="DSS033F6" class="link link-reveal link-table xref-fig">6</a>b), but plays the role of clumping enhancer.</p><p>Given the fact that the flocculins leading to cell flocculation share a common three-domain structure and require Ca<sup>2+</sup> ions to stabilize the structure,<sup><a reveal-id="DSS033C9" data-open="DSS033C9" class="link link-ref link-reveal xref-bibr">9</a></sup> we explored the effect of Ca<sup>2+</sup> ions on clumping phenotypes of the two parental strains and their two segregants (S2 and S5) that showed the transgressive phenotype. Figure 6c shows that the cell aggregation phenotype of the strains S2 and S5 was reduced to the extent of YL1C when Ca<sup>2+</sup> ions were chelated with the ethylenediaminetetraacetic acid treatment (panel 2), but recovered when the ions were added as CaCl<sub>2</sub> in the clumping buffer, suggesting that the transgressive phenotype can be attributed to FLO gene family. This assumption was confirmed by the further knockout experiments in which the transgressive phenotype was disrupted when either <em>FLO1</em> or <em>FLO8</em> was silenced in S2 and S5 strains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>b). Strikingly, the treatment did not result in altered phenotype of the clumpy parental strain (YL1C). This together with the fact that this parent carried the non-clumpy <em>FLO1</em> allele indicates that the phenotypic response to the Ca<sup>2+</sup> treatment actually reflects response of the Ca<sup>2+</sup>-dependent cell aggregation induced by the cell wall protein, which differs from that mediated by <em>AMN1</em>.</p><p>We assessed relative contribution of the QTL genes to phenotypic variation in cell aggregation and found that genetic variation in <em>AMN1</em> explained 25% of the phenotypic variation, whereas the other three QTL genes <em>FLO8</em>, <em>RGA1</em>, and <em>FLO1</em>, respectively, explained 17, 5, and 11% of the variation. Variation between aggregative genotypes at all the four genes explained 46% of the phenotypic variation. To further explore the joint and interactive effects of the genes, we generated three dual-knockout strains (YL1C-<em>amn1</em>Î”&amp;<em>flo8</em>Î”, YL1C-<em>amn1</em>Î”&amp;<em>rga1</em>Î”, and YL1C-<em>flo8</em>Î”&amp;<em>rga1</em>Î”) and one triple-deletion strain (YL1C-<em>amn1</em>Î”&amp;<em>flo8</em>Î”&amp;<em>rga1</em>Î”). Phenotypes of these engineered strains are shown together with other single QTL gene knockout strains and the wild-type strain in Fig.Â <a reveal-id="DSS033F2" data-open="DSS033F2" class="link link-reveal link-table xref-fig">2</a>. Although the degree of cell aggregation of the single-gene knockout strains was further reduced when they were knocked out for additional one or two of the QTL genes, it is clear that different configurations of the knockout QTL genes have led to different degrees of clumpiness. This, together with the observation illustrated in Fig.Â <a reveal-id="DSS033F6" data-open="DSS033F6" class="link link-reveal link-table xref-fig">6</a>b, reveals that the QTL genes show clearly epistatic effects on cell aggregation.</p>                            <h2 scrollto-destination=5625598 id="5625598" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The genetic architecture underlying a quantitative trait is determined by the number of genes that contribute to their additive and interactive effects on variation of the trait phenotype.<sup><a reveal-id="DSS033C36" data-open="DSS033C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Although many genomes have been fully sequenced and the number of genes with annotated function increases rapidly in the era of functional genomics, it still remains a great challenge to reconstruct the architecture with a full insight of the genetic parameters. Budding yeast, the single-cell eukaryote, but with probably the richest structural and functional information of the genome, provides an ideal model for dissecting the molecular basis of genetic variation in typical quantitative traits such as high-temperature growth<sup><a reveal-id="DSS033C12" data-open="DSS033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and sporulation efficiency<sup><a reveal-id="DSS033C37" data-open="DSS033C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and in genetic control of transcriptome abundance.<sup><a reveal-id="DSS033C38" data-open="DSS033C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The present study extends these successful examples by dissecting multiple-threshold phenotypic variation of yeast cell aggregation into a genic level.</p><p>Several genes have been reported previously in the literature to control clumping phenotype variation in budding yeast. Of these, the most prominent are the FLO gene family responsible for many cellular adhesion phenotypes, including flocculation.<sup><a reveal-id="DSS033C10" data-open="DSS033C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In sharp contrast, the present study reveals the two novel genes, <em>AMN1</em> and <em>RGA1</em>, in addition to previously identified two genes, <em>FLO1</em> and <em>FLO8</em>, that are significant determinants for the phenotype. Success of the present study in resolving the mapped QTLs into the underlying QTL genes can be attributed to that the major QTLs have been mapped at the high resolution by which direct gene targeting becomes possible. We noted that the simulation study by Li et al. predicted a much larger confidence interval for mapping categorical traits from a simulated backcross population with 200 individuals genotyped at genetic markers of every 10 cM. When compared with the simulation study, the present analysis has employed a substantially larger sample size (292 segregants) and higher density of markers in the QTL regions. These together with the fact that the budding yeast has a highly recombiogenic and compact genome paved the basis for the mapping resolution we have achieved.</p><p>The combined effect of these QTL genes explains 46% of the overall phenotypic variation, leaving the remaining 54% of the variation to be residual. The residual variation can be explained as two parts as well as their interaction, the genetic component whose underlying genetic effects are too small to be detected by the experiment, and the component contributed by environmental modification to the quantitative trait. In addition, we demonstrated that the QTL genes exhibited clear epistatic effects on the aggregation trait. These show that the underlying mechanism of yeast cell aggregation is more complicated than expected in the previous literature.</p><p>Of these QTL genes, <em>AMN1</em> plays the leading role in regulating cell aggregation. We provide strong and direct experimental evidence that supports this is through the genes modulating the mitotic exit network in <em>S. cerevisiae</em>.<sup><a reveal-id="DSS033C28" data-open="DSS033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> We have identified the <em>AMN1</em> as a major QTL gene of cell aggregation through a genome-wide QTL scan in a natural population and provided here the direct experimental evidence for the V368D substitution as the causative genetic variant. Our results also strongly indicate the putative domain of <em>AMN1</em>, and the functional significance of V368D, which has been recently predicted as a superfamily being highly conserved (e-value of 6.38e-96) across species.<sup><a reveal-id="DSS033C39" data-open="DSS033C39" class="link link-ref link-reveal xref-bibr">39</a></sup> On the other hand, the <em>RGA1</em> gene, a member of the GTPase-activating family, is first identified in the present study as a major gene affecting cellular clumping in budding yeast. This gene is required for yeast cells to switch from the apical to isotropic growth phase.<sup><a reveal-id="DSS033C40" data-open="DSS033C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The present study creates an opportunity to explore the joint effect of this cell aggregation QTL gene, a budding pattern controller, and another QTL gene <em>AMN1</em>, also a mitotic exit regulator. Genetic modification to these genes through gene knockouts disperses the well-known grape-like strings of cells (Fig.Â <a reveal-id="DSS033F5" data-open="DSS033F5" class="link link-reveal link-table xref-fig">5</a>).</p><p>In addition to the above two major genetic factors, <em>FLO1</em> and <em>FLO8</em> were also detected as the QTL genes through the present forward genetic analysis. Highly expressed <em>FLO1</em>, a cell surface adhesion regulator, has been widely recognized as a biomarker for cell flocculation in many candidate-based analysis. However, we present here the role of the gene, as merely one of the building elements in the genetic architecture underlying the complex phenotype, and show that <em>FLO1</em> is highly background dependent in determining the phenotype of its carrier. Unlike what has been widely accepted in the literature of yeast cell flocculation, expressed <em>FLO1</em> does not necessarily confer a clumping phenotype to its carriers. We demonstrate that the <em>FLO1</em> allele from a non-clumpy parental strain, in the presence of the other three QTL genes from a clumpy parental strain, creates the genotype for the clumping phenotype that substantially surpasses performance of both the parents, i.e. transgressive segregation of the QTL genes. Taking all these together, the QTL genes unveiled in the present study have been shown to be involved in multiple pathways that govern the cellular development (<em>RGA1</em>), the behaviour of daughter cells (<em>AMN1</em>), and the cell-surface adherence property (<em>FLO1</em> and <em>FLO8</em>). In contrast to the studies in which biological functions and their phenotypic significance of these QTL genes were examined either independently or jointly, the present study unveils the QTL genes as major determinants of the clumping phenotype in a systematic way. This clearly indicates the power and appropriateness of the forward genetic approach employed in the present study in uncovering the genetic basis of an aggregate polygenic trait such as cellular clumping that involves multiple constituent components.</p><p>The evolutionary significance of cellular clumping has recently been highlighted in the single-cell species because the clumping phenotype can protect inner individuals in cell clumps from biotic or/and abiotic stresses such as alcohol, anti-microbials, and physical attacks. The <em>FLO1</em> gene is recognized as a â€˜green beard geneâ€™ because its induced expression in a non-flocculent laboratory strain, S288C, can restore its cells to flocculate, and the cellular flocs, thus, generated provide a cooperative shield to protect from stresses in the living environment.<sup><a reveal-id="DSS033C1" data-open="DSS033C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Our data demonstrate that expression of <em>FLO1</em> may not be the biomarker for a clumping phenotype, although an unexpressed <em>FLO1</em> allele from the non-clumpy strain, which is isogenic to S228C, may enhance clumping of its offspring strains carrying other major clumping genes. In the polygenic system uncovered here, <em>AMN1</em> is found to be a more important genetic determinant than <em>FLO1</em> in controlling expression of the clumping phenotype, and this endogenously expressed gene may make a more appropriate target for investigating the molecular mechanism underlying the â€˜green beardâ€™ effect.</p>                            <h2 scrollto-destination=5625610 id="5625610" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss033/-/DC1" target="_blank">.</a></p>                            <h2 scrollto-destination=5625615 id="5625615" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the National Basic Research Program of China (2012CB316505) and National Natural Science Foundation of China (91231114, 31271335).</p>                            <h2 scrollto-destination=5625619 id="5625619" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Prof. M.J. Kearsey and Dr Julia Lodge for their comments on an earlier version of this paper. In particular, we thank two anonymous reviewers for their constructively critical comments and suggestions that have helped improve clarity and presentation of this paper.</p>                            <h2 scrollto-destination=5625621 id="5625621" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dss033c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smukalla</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Caldara</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Pochet</div> <div class="given-names">N.</div></div>,Â et al.Â </div><div class="article-title"><em>FLO1</em> is a variable green beard gene that drives biofilm-like cooperation in budding yeast</div>,Â <div class="source">Cell</div>Â ,Â <div class="year">2008</div>,Â vol.Â <div class="volume">135</div>Â (pg.Â <div class="fpage">726</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0083-6729(03)01002-1" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1016/S0083-6729(03)01002-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FLO1%20is%20a%20variable%20green%20beard%20gene%20that%20drives%20biofilm-like%20cooperation%20in%20budding%20yeast&amp;author=S.%20Smukalla&amp;author=M.%20Caldara&amp;author=N.%20Pochet&amp;publication_year=2008&amp;journal=Cell&amp;volume=135&amp;pages=726-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.09.037" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.09.037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.09.037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19013280" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gimeno</div> <div class="given-names">C.J.</div></div>,Â  <div class="name"><div class="surname">Ljungdahl</div> <div class="given-names">P.O.</div></div>,Â  <div class="name"><div class="surname">Styles</div> <div class="given-names">C.A.</div></div>,Â  <div class="name"><div class="surname">Fink</div> <div class="given-names">G.R.</div></div>.Â </div><div class="article-title">Unipolar cell divisions in the yeast <em>S. cerevisiae</em> lead to flamentous growth: regulation by starvation and RAS</div>,Â <div class="source">Cell</div>Â ,Â <div class="year">1992</div>,Â vol.Â <div class="volume">68</div>Â (pg.Â <div class="fpage">1077</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1681/ASN.2006090975" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1681/ASN.2006090975</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unipolar%20cell%20divisions%20in%20the%20yeast%20S.%20cerevisiae%20lead%20to%20flamentous%20growth%3A%20regulation%20by%20starvation%20and%20RAS&amp;author=C.J.%20Gimeno&amp;author=P.O.%20Ljungdahl&amp;author=C.A.%20Styles&amp;author=G.R.%20Fink&amp;publication_year=1992&amp;journal=Cell&amp;volume=68&amp;pages=1077-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(92)90079-R" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(92)90079-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(92)90079-R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1547504" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cullen</div> <div class="given-names">P.J.</div></div>,Â  <div class="name"><div class="surname">Sprague</div> <div class="given-names">G.F.</div></div>.Â </div><div class="article-title">Glucose depletion causes haploid invasive growth in yeast</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">97</div>Â (pg.Â <div class="fpage">13461</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1210/jc.2003-032149" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1210/jc.2003-032149</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glucose%20depletion%20causes%20haploid%20invasive%20growth%20in%20yeast&amp;author=P.J.%20Cullen&amp;author=G.F.%20Sprague&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=13461-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.011511198" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.011511198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.011511198"> </span></div>Â </div></div></div></div></div><div content-id="dss033c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pretorius</div> <div class="given-names">I.S.</div></div>.Â </div><div class="article-title">Tailoring wine yeast for the new millennium: novel approaches to the ancient art of winemaking</div>,Â <div class="source">Yeast</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">16</div>Â (pg.Â <div class="fpage">675</div>-<div class="lpage">729</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tailoring%20wine%20yeast%20for%20the%20new%20millennium%3A%20novel%20approaches%20to%20the%20ancient%20art%20of%20winemaking&amp;author=I.S.%20Pretorius&amp;publication_year=2000&amp;journal=Yeast&amp;volume=16&amp;pages=675-729" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10861899" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Teunissen</div> <div class="given-names">A.W.</div></div>,Â  <div class="name"><div class="surname">van den Berg</div> <div class="given-names">J.A.</div></div>,Â  <div class="name"><div class="surname">Steensma</div> <div class="given-names">H.Y.</div></div>.Â </div><div class="article-title">Localization of the dominant flocculation genes <em>FLO5</em> and <em>FLO8</em> of <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Yeast</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">735</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1159/000318776" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1159/000318776</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Localization%20of%20the%20dominant%20flocculation%20genes%20FLO5%20and%20FLO8%20of%20Saccharomyces%20cerevisiae&amp;author=A.W.%20Teunissen&amp;author=J.A.%20van%20den%20Berg&amp;author=H.Y.%20Steensma&amp;publication_year=1995&amp;journal=Yeast&amp;volume=30&amp;pages=735-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div>Â </div></div></div></div></div><div content-id="dss033c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Styles</div> <div class="given-names">C.A.</div></div>,Â  <div class="name"><div class="surname">Feng</div> <div class="given-names">Q.H.</div></div>,Â  <div class="name"><div class="surname">Fink</div> <div class="given-names">G.R.</div></div>.Â </div><div class="article-title">A <em>Saccharomyces</em> gene family involved in invasive growth, cell-cell adhesion, and mating</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">97</div>Â (pg.Â <div class="fpage">12158</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Saccharomyces%20gene%20family%20involved%20in%20invasive%20growth%2C%20cell-cell%20adhesion%2C%20and%20mating&amp;author=B.%20Guo&amp;author=C.A.%20Styles&amp;author=Q.H.%20Feng&amp;author=G.R.%20Fink&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=12158-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.220420397" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.220420397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.220420397"> </span></div>Â </div></div></div></div></div><div content-id="dss033c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">O.</div></div>,Â  <div class="name"><div class="surname">Suda</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Ohtani</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Sone</div> <div class="given-names">H.</div></div>.Â </div><div class="article-title">Molecular cloning and analysis of the dominant flocculation gene <em>FLO8</em> from <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Mol. Gen. Genet.</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">251</div>Â (pg.Â <div class="fpage">707</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20analysis%20of%20the%20dominant%20flocculation%20gene%20FLO8%20from%20Saccharomyces%20cerevisiae&amp;author=O.%20Kobayashi&amp;author=H.%20Suda&amp;author=T.%20Ohtani&amp;author=H.%20Sone&amp;publication_year=1996&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=251&amp;pages=707-15" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8757402" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rupp</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Summers</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Lo</div> <div class="given-names">H.J.</div></div>,Â  <div class="name"><div class="surname">Madhani</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Fink</div> <div class="given-names">G.</div></div>.Â </div><div class="article-title">MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast <em>FLO11</em> gene</div>,Â <div class="source">EMBO J.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">18</div>Â (pg.Â <div class="fpage">1257</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAP%20kinase%20and%20cAMP%20filamentation%20signaling%20pathways%20converge%20on%20the%20unusually%20large%20promoter%20of%20the%20yeast%20FLO11%20gene&amp;author=S.%20Rupp&amp;author=E.%20Summers&amp;author=H.J.%20Lo&amp;author=H.%20Madhani&amp;author=G.%20Fink&amp;publication_year=1999&amp;journal=EMBO%20J.&amp;volume=18&amp;pages=1257-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/18.5.1257" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F18.5.1257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F18.5.1257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10064592" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Verstrepen</div> <div class="given-names">K.J.</div></div>,Â  <div class="name"><div class="surname">Reynolds</div> <div class="given-names">T.B.</div></div>,Â  <div class="name"><div class="surname">Fink</div> <div class="given-names">G.R.</div></div>.Â </div><div class="article-title">Origins of variation in the fungal cell surface</div>,Â <div class="source">Nat. Rev. Microbiol.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">2</div>Â (pg.Â <div class="fpage">533</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20of%20variation%20in%20the%20fungal%20cell%20surface&amp;author=K.J.%20Verstrepen&amp;author=T.B.%20Reynolds&amp;author=G.R.%20Fink&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=2&amp;pages=533-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro927" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15197389" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Verstrepen</div> <div class="given-names">K.J.</div></div>,Â  <div class="name"><div class="surname">Klis</div> <div class="given-names">F.M.</div></div>.Â </div><div class="article-title">Flocculation, adhesion and biofilm formation in yeasts</div>,Â <div class="source">Mol. Microbiol.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">60</div>Â (pg.Â <div class="fpage">5</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flocculation%2C%20adhesion%20and%20biofilm%20formation%20in%20yeasts&amp;author=K.J.%20Verstrepen&amp;author=F.M.%20Klis&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=60&amp;pages=5-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.60.issue-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.60.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.60.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16556216" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackay</div> <div class="given-names">T.F.</div></div>.Â </div><div class="article-title">The genetic architecture of quantitative traits</div>,Â <div class="source">Annu. Rev. Genet.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">35</div>Â (pg.Â <div class="fpage">303</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20architecture%20of%20quantitative%20traits&amp;author=T.F.%20Mackay&amp;publication_year=2001&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=35&amp;pages=303-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.35.102401.090633" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.35.102401.090633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.35.102401.090633"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11700286" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steinmetz</div> <div class="given-names">L.M.</div></div>,Â  <div class="name"><div class="surname">Sinha</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Richards</div> <div class="given-names">D.R.</div></div>,Â et al.Â </div><div class="article-title">Dissecting the architecture of a quantitative trait locus in yeast</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">416</div>Â (pg.Â <div class="fpage">326</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20the%20architecture%20of%20a%20quantitative%20trait%20locus%20in%20yeast&amp;author=L.M.%20Steinmetz&amp;author=H.%20Sinha&amp;author=D.R.%20Richards&amp;publication_year=2002&amp;journal=Nature&amp;volume=416&amp;pages=326-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/416326a" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F416326a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F416326a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11907579" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tabor</div> <div class="given-names">H.K.</div></div>,Â  <div class="name"><div class="surname">Risch</div> <div class="given-names">N.J.</div></div>,Â  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.M.</div></div>.Â </div><div class="article-title">Candidate-gene approaches for studying complex genetic traits: practical considerations</div>,Â <div class="source">Nat. Rev. Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">3</div>Â (pg.Â <div class="fpage">391</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Candidate-gene%20approaches%20for%20studying%20complex%20genetic%20traits%3A%20practical%20considerations&amp;author=H.K.%20Tabor&amp;author=N.J.%20Risch&amp;author=R.M.%20Myers&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=391-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg796" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg796" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg796"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988764" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mehrabian</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Allayee</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Stockton</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title">Integrating genotypic and expression data in a segregating mouse population to identify 5-lipoxygenase as a susceptibility gene for obesity and bone traits</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">37</div>Â (pg.Â <div class="fpage">1224</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrating%20genotypic%20and%20expression%20data%20in%20a%20segregating%20mouse%20population%20to%20identify%205-lipoxygenase%20as%20a%20susceptibility%20gene%20for%20obesity%20and%20bone%20traits&amp;author=M.%20Mehrabian&amp;author=H.%20Allayee&amp;author=J.%20Stockton&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=1224-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1619" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16200066" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">X.H.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">M.H.</div></div>,Â  <div class="name"><div class="surname">Tan</div> <div class="given-names">T.</div></div>,Â et al.Â </div><div class="article-title">Genetic dissection of ethanol tolerance in the budding yeast <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">175</div>Â (pg.Â <div class="fpage">1479</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20ethanol%20tolerance%20in%20the%20budding%20yeast%20Saccharomyces%20cerevisiae&amp;author=X.H.%20Hu&amp;author=M.H.%20Wang&amp;author=T.%20Tan&amp;publication_year=2007&amp;journal=Genetics&amp;volume=175&amp;pages=1479-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.065292" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.065292" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.065292"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17194785" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Gottschling</div> <div class="given-names">D.E.</div></div>,Â  <div class="name"><div class="surname">Kaiser</div> <div class="given-names">C.A.</div></div>,Â  <div class="name"><div class="surname">Stearns</div> <div class="given-names">T.</div></div>.Â </div>,Â <div class="source">Methods in Yeast Genetics. A Cold Spring Harbor Laboratory Course Manual</div>Â ,Â <div class="year">1998</div><div class="publisher-loc">New York</div><div class="publisher-name">Cold Spring Harbor Laboratory Press:</div></div></div></div></div><div content-id="dss033c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>.Â </div><div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">27</div>Â (pg.Â <div class="fpage">573</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baudin</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Ozier-Kalogeropoulos</div> <div class="given-names">O.</div></div>,Â  <div class="name"><div class="surname">Denouel</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Lacroute</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Cullin</div> <div class="given-names">C.</div></div>.Â </div><div class="article-title">A simple and efficient method for direct gene deletion in <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">21</div>Â (pg.Â <div class="fpage">3329</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20and%20efficient%20method%20for%20direct%20gene%20deletion%20in%20Saccharomyces%20cerevisiae&amp;author=A.%20Baudin&amp;author=O.%20Ozier-Kalogeropoulos&amp;author=A.%20Denouel&amp;author=F.%20Lacroute&amp;author=C.%20Cullin&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=3329-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.14.3329" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.14.3329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.14.3329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8341614" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lorenz</div> <div class="given-names">M.C.</div></div>,Â  <div class="name"><div class="surname">Muir</div> <div class="given-names">R.S.</div></div>,Â  <div class="name"><div class="surname">Lim</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">McElver</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Weber</div> <div class="given-names">S.C.</div></div>,Â  <div class="name"><div class="surname">Heitman</div> <div class="given-names">J.</div></div>.Â </div><div class="article-title">Gene disruption with PCR products in <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Gene</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">158</div>Â (pg.Â <div class="fpage">113</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20disruption%20with%20PCR%20products%20in%20Saccharomyces%20cerevisiae&amp;author=M.C.%20Lorenz&amp;author=R.S.%20Muir&amp;author=E.%20Lim&amp;author=J.%20McElver&amp;author=S.C.%20Weber&amp;author=J.%20Heitman&amp;publication_year=1995&amp;journal=Gene&amp;volume=158&amp;pages=113-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(95)00144-U" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(95)00144-U" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(95)00144-U"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7789793" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmitt</div> <div class="given-names">M.E.</div></div>,Â  <div class="name"><div class="surname">Brown</div> <div class="given-names">T.A.</div></div>,Â  <div class="name"><div class="surname">Trumpower</div> <div class="given-names">B.L.</div></div>.Â </div><div class="article-title">A rapid and simple method for preparation of RNA from <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1990</div>,Â vol.Â <div class="volume">18</div>Â (pg.Â <div class="fpage">3091</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20and%20simple%20method%20for%20preparation%20of%20RNA%20from%20Saccharomyces%20cerevisiae&amp;author=M.E.%20Schmitt&amp;author=T.A.%20Brown&amp;author=B.L.%20Trumpower&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=3091-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.10.3091" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.10.3091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.10.3091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2190191" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Knop</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Siegers</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Pereira</div> <div class="given-names">G.</div></div>,Â et al.Â </div><div class="article-title">Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines</div>,Â <div class="source">Yeast</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">15</div>Â (pg.Â <div class="fpage">963</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epitope%20tagging%20of%20yeast%20genes%20using%20a%20PCR-based%20strategy%3A%20more%20tags%20and%20improved%20practical%20routines&amp;author=M.%20Knop&amp;author=K.%20Siegers&amp;author=G.%20Pereira&amp;publication_year=1999&amp;journal=Yeast&amp;volume=15&amp;pages=963-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(SICI)1097-0061(199907)15:10B&lt;&gt;1.0.CO;2-M" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(SICI)1097-0061(199907)15:10B&lt;&gt;1.0.CO;2-M" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(SICI)1097-0061(199907)15:10B&lt;&gt;1.0.CO;2-M"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10407276" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">James</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Halladay</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Craig</div> <div class="given-names">E.A.</div></div>.Â </div><div class="article-title">Genomic Libraries and a host strain designed for highly efficient two-hybrid selection in yeast</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">144</div>Â (pg.Â <div class="fpage">1425</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20Libraries%20and%20a%20host%20strain%20designed%20for%20highly%20efficient%20two-hybrid%20selection%20in%20yeast&amp;author=P.%20James&amp;author=J.%20Halladay&amp;author=E.A.%20Craig&amp;publication_year=1996&amp;journal=Genetics&amp;volume=144&amp;pages=1425-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8978031" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harper</div> <div class="given-names">J.W.</div></div>,Â  <div class="name"><div class="surname">Adami</div> <div class="given-names">G.R.</div></div>,Â  <div class="name"><div class="surname">Wei</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Keyomarsi</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Elledge</div> <div class="given-names">S.J.</div></div>.Â </div><div class="article-title">The p21 Cdk-interacting protein Cip1 is a potent inhibitor of G1 cyclin-dependent kinases</div>,Â <div class="source">Cell</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">75</div>Â (pg.Â <div class="fpage">805</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20p21%20Cdk-interacting%20protein%20Cip1%20is%20a%20potent%20inhibitor%20of%20G1%20cyclin-dependent%20kinases&amp;author=J.W.%20Harper&amp;author=G.R.%20Adami&amp;author=N.%20Wei&amp;author=K.%20Keyomarsi&amp;author=S.J.%20Elledge&amp;publication_year=1993&amp;journal=Cell&amp;volume=75&amp;pages=805-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(93)90499-G" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(93)90499-G" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(93)90499-G"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8242751" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorensen</div> <div class="given-names">D.A.</div></div>,Â  <div class="name"><div class="surname">Anderson</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Gianola</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Korsgaard</div> <div class="given-names">I.</div></div>.Â </div><div class="article-title">Bayesian inference in threshold models using Gibbs sampling</div>,Â <div class="source">Genet. Sel. Evol.</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">27</div>Â (pg.Â <div class="fpage">229</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20inference%20in%20threshold%20models%20using%20Gibbs%20sampling&amp;author=D.A.%20Sorensen&amp;author=S.%20Anderson&amp;author=D.%20Gianola&amp;author=I.%20Korsgaard&amp;publication_year=1995&amp;journal=Genet.%20Sel.%20Evol.&amp;volume=27&amp;pages=229-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1297-9686-27-3-229" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1297-9686-27-3-229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1297-9686-27-3-229"> </span></div>Â </div></div></div></div></div><div content-id="dss033c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Zeng</div> <div class="given-names">Z.B.</div></div>.Â </div><div class="article-title">Multiple-interval mapping for ordinal traits</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">173</div>Â (pg.Â <div class="fpage">1649</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple-interval%20mapping%20for%20ordinal%20traits&amp;author=J.%20Li&amp;author=S.%20Wang&amp;author=Z.B.%20Zeng&amp;publication_year=2006&amp;journal=Genetics&amp;volume=173&amp;pages=1649-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.054619" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.054619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.054619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16585135" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Styles</div> <div class="given-names">C.A.</div></div>,Â  <div class="name"><div class="surname">Fink</div> <div class="given-names">G.R.</div></div>.Â </div><div class="article-title"><em>Saccharomyces cerevisiae</em> S288C has a mutation in <em>FLO8</em>, a gene required for filamentous growth</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">144</div>Â (pg.Â <div class="fpage">967</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharomyces%20cerevisiae%20S288C%20has%20a%20mutation%20in%20FLO8%2C%20a%20gene%20required%20for%20filamentous%20growth&amp;author=H.%20Liu&amp;author=C.A.%20Styles&amp;author=G.R.%20Fink&amp;publication_year=1996&amp;journal=Genetics&amp;volume=144&amp;pages=967-78" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8913742" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">O.</div></div>,Â  <div class="name"><div class="surname">Yoshimoto</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Sone</div> <div class="given-names">H.</div></div>.Â </div><div class="article-title">Analysis of the genes activated by the <em>FLO8</em> gene in <em>Saccharomyces cerevisiae</em></div>,Â <div class="source">Curr. Genet.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">36</div>Â (pg.Â <div class="fpage">256</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20genes%20activated%20by%20the%20FLO8%20gene%20in%20Saccharomyces%20cerevisiae&amp;author=O.%20Kobayashi&amp;author=H.%20Yoshimoto&amp;author=H.%20Sone&amp;publication_year=1999&amp;journal=Curr.%20Genet.&amp;volume=36&amp;pages=256-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002940050498" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002940050498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002940050498"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10591965" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Shirogane</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Liu</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Harper</div> <div class="given-names">J.W.</div></div>,Â  <div class="name"><div class="surname">Elledge</div> <div class="given-names">S.J.</div></div>.Â </div><div class="article-title">Exit from exit: resetting the cell cycle through Amn1 inhibition of G protein signaling</div>,Â <div class="source">Cell</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">112</div>Â (pg.Â <div class="fpage">697</div>-<div class="lpage">709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exit%20from%20exit%3A%20resetting%20the%20cell%20cycle%20through%20Amn1%20inhibition%20of%20G%20protein%20signaling&amp;author=Y.%20Wang&amp;author=T.%20Shirogane&amp;author=D.%20Liu&amp;author=J.W.%20Harper&amp;author=S.J.%20Elledge&amp;publication_year=2003&amp;journal=Cell&amp;volume=112&amp;pages=697-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00121-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(03)00121-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(03)00121-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12628189" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">G.</div></div>,Â et al.Â </div><div class="article-title">Robust detection and genotyping of single feature polymorphisms from gene expression data</div>,Â <div class="source">PLoS Comput. Biol.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">5</div>Â pg.Â <div class="fpage">e1000317</div>Â <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robust%20detection%20and%20genotyping%20of%20single%20feature%20polymorphisms%20from%20gene%20expression%20data&amp;author=M.%20Wang&amp;author=X.%20Hu&amp;author=G.%20Li&amp;publication_year=2009&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=5&amp;pages=e1000317" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000317" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000317" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000317"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19282978" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voth</div> <div class="given-names">W.P.</div></div>,Â  <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Takahata</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Kretschmann</div> <div class="given-names">K.L.</div></div>,Â  <div class="name"><div class="surname">Lieb</div> <div class="given-names">J.D.</div></div>.Â </div><div class="article-title">Forkhead proteins control the outcome of transcription factor binding by antiactivation</div>,Â <div class="source">EMBO J.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">26</div>Â (pg.Â <div class="fpage">4324</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forkhead%20proteins%20control%20the%20outcome%20of%20transcription%20factor%20binding%20by%20antiactivation&amp;author=W.P.%20Voth&amp;author=Y.%20Yu&amp;author=S.%20Takahata&amp;author=K.L.%20Kretschmann&amp;author=J.D.%20Lieb&amp;publication_year=2007&amp;journal=EMBO%20J.&amp;volume=26&amp;pages=4324-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.emboj.7601859" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.emboj.7601859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.emboj.7601859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17898805" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lerner</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Chin</div> <div class="given-names">T.E.</div></div>,Â  <div class="name"><div class="surname">Brent</div> <div class="given-names">R.</div></div>.Â </div><div class="article-title">Yeast Cbk1 and Mob2 activate daughter specific genetic programs to induce asymmetric cell fates</div>,Â <div class="source">Cell</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">107</div>Â (pg.Â <div class="fpage">739</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Yeast%20Cbk1%20and%20Mob2%20activate%20daughter%20specific%20genetic%20programs%20to%20induce%20asymmetric%20cell%20fates&amp;author=A.%20Lerner&amp;author=T.E.%20Chin&amp;author=R.%20Brent&amp;publication_year=2001&amp;journal=Cell&amp;volume=107&amp;pages=739-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(01)00596-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(01)00596-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(01)00596-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11747810" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Caviston</div> <div class="given-names">J.P.</div></div>,Â  <div class="name"><div class="surname">Longtine</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Pringle</div> <div class="given-names">J.R.</div></div>,Â  <div class="name"><div class="surname">Bi</div> <div class="given-names">E.</div></div>.Â </div><div class="article-title">The role of Cdc42p GTPase-activating proteins in assembly of the septin ring in yeast</div>,Â <div class="source">Mol. Biol. Cell</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">4051</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20Cdc42p%20GTPase-activating%20proteins%20in%20assembly%20of%20the%20septin%20ring%20in%20yeast&amp;author=J.P.%20Caviston&amp;author=M.%20Longtine&amp;author=J.R.%20Pringle&amp;author=E.%20Bi&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Cell&amp;volume=14&amp;pages=4051-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.E03-04-0247" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.E03-04-0247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.E03-04-0247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14517318" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tong</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Gao</div> <div class="given-names">X.D.</div></div>,Â  <div class="name"><div class="surname">Howell</div> <div class="given-names">A.S.</div></div>,Â  <div class="name"><div class="surname">Bose</div> <div class="given-names">I.</div></div>,Â  <div class="name"><div class="surname">Lew</div> <div class="given-names">D.J.</div></div>,Â  <div class="name"><div class="surname">Bi</div> <div class="given-names">E.</div></div>.Â </div><div class="article-title">Adjacent positioning of cellular structures enabled by a Cdc42 GTPase-activating proteinâ€“mediated zone of inhibition</div>,Â <div class="source">J. Cell Biol.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">179</div>Â (pg.Â <div class="fpage">1375</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adjacent%20positioning%20of%20cellular%20structures%20enabled%20by%20a%20Cdc42%20GTPase-activating%20protein%E2%80%93mediated%20zone%20of%20inhibition&amp;author=Z.%20Tong&amp;author=X.D.%20Gao&amp;author=A.S.%20Howell&amp;author=I.%20Bose&amp;author=D.J.%20Lew&amp;author=E.%20Bi&amp;publication_year=2007&amp;journal=J.%20Cell%20Biol.&amp;volume=179&amp;pages=1375-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200705160" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200705160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200705160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18166650" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">G.R.</div></div>,Â  <div class="name"><div class="surname">Givan</div> <div class="given-names">S.A.</div></div>,Â  <div class="name"><div class="surname">Cullen</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Sprague</div> <div class="given-names">G.F.</div>Jr</div>.Â </div><div class="article-title">GTPase-activating proteins for Cdc42</div>,Â <div class="source">Eukaryot. Cell</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">1</div>Â (pg.Â <div class="fpage">469</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GTPase-activating%20proteins%20for%20Cdc42&amp;author=G.R.%20Smith&amp;author=S.A.%20Givan&amp;author=P.%20Cullen&amp;author=G.F.%20Sprague&amp;publication_year=2002&amp;journal=Eukaryot.%20Cell&amp;volume=1&amp;pages=469-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.1.3.469-480.2002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.1.3.469-480.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.1.3.469-480.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12455995" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Verstrepen</div> <div class="given-names">K.J.</div></div>,Â  <div class="name"><div class="surname">Jansen</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Lewitter</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Fink</div> <div class="given-names">G.R.</div></div>.Â </div><div class="article-title">Intragenic tandem repeats generate functional variability</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">37</div>Â (pg.Â <div class="fpage">986</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intragenic%20tandem%20repeats%20generate%20functional%20variability&amp;author=K.J.%20Verstrepen&amp;author=A.%20Jansen&amp;author=F.%20Lewitter&amp;author=G.R.%20Fink&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=986-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1618" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1618" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1618"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16086015" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Walsh</div> <div class="given-names">B.</div></div>.Â </div>,Â <div class="source">Genetics and Analysis of Quantitative Traits</div>Â ,Â <div class="year">1998</div><div class="publisher-loc">Sunderland, MA, USA</div><div class="publisher-name">Sinauer Associates</div></div></div></div></div><div content-id="dss033c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deutschbauer</div> <div class="given-names">A.M.</div></div>,Â  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>.Â </div><div class="article-title">Quantitative trait loci mapped to single-nucleotide resolution in yeast</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">37</div>Â (pg.Â <div class="fpage">1333</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20mapped%20to%20single-nucleotide%20resolution%20in%20yeast&amp;author=A.M.%20Deutschbauer&amp;author=R.W.%20Davis&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=1333-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1674" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1674" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1674"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16273108" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brem</div> <div class="given-names">R.B.</div></div>,Â  <div class="name"><div class="surname">Yvert</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Clinton</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Kruglyak</div> <div class="given-names">L.</div></div>.Â </div><div class="article-title">Genetic dissection of transcriptional regulation in budding yeast</div>,Â <div class="source">Science</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">296</div>Â (pg.Â <div class="fpage">752</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20transcriptional%20regulation%20in%20budding%20yeast&amp;author=R.B.%20Brem&amp;author=G.%20Yvert&amp;author=R.%20Clinton&amp;author=L.%20Kruglyak&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=752-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1069516" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1069516" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1069516"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11923494" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marchler-Bauer</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Lu</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Anderson</div> <div class="given-names">J.B.</div></div>,Â et al.Â </div><div class="article-title">CDD: a conserved domain database for the functional annotation of proteins</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">39</div>Â (pg.Â <div class="fpage">225</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDD%3A%20a%20conserved%20domain%20database%20for%20the%20functional%20annotation%20of%20proteins&amp;author=A.%20Marchler-Bauer&amp;author=S.%20Lu&amp;author=J.B.%20Anderson&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=225-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1189" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20823090" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="dss033c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dss033c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Fujimura-Kamada</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Hanamatsu</div> <div class="given-names">H.</div></div>,Â et al.Â </div><div class="article-title">Transbilayer phospholipid flipping regulates Cdc42p signaling during poparized cell growth via Rga GTPase-activating proteins</div>,Â <div class="source">Dev. Cell</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">13</div>Â (pg.Â <div class="fpage">743</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transbilayer%20phospholipid%20flipping%20regulates%20Cdc42p%20signaling%20during%20poparized%20cell%20growth%20via%20Rga%20GTPase-activating%20proteins&amp;author=K.%20Saito&amp;author=K.%20Fujimura-Kamada&amp;author=H.%20Hanamatsu&amp;publication_year=2007&amp;journal=Dev.%20Cell&amp;volume=13&amp;pages=743-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.devcel.2007.09.014" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.devcel.2007.09.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.devcel.2007.09.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17981141" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=383179&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_table.doc?Expires=1509238860&amp;Signature=Tj4X~34WggEn1yKHQ9-OxiTb6TVKzdtuEFRvXURx7tzTRD0CuePJ~-SVYoSIekeA9Wd~8E4mCUeWaDrw-s~C95qBw-vo8uuLH4VBAneVWsiiubdNbLtq1bD1EESXxJsy3DsevppTtCq9GKoWEFL6RzquklhX78VUC6u4fvRDgjj~ewZqISJjrvC6HDF~2Nap3Aos3vxvy82DVGbiV9Og7Q0Y6o1YRYgbyLJ3nn1aouNdC4PPu5z33HWhc1I-nr9E4vgoOA3~hL5M0U2lbAHG12vFeFdRnFXbwa~IFM36KLLC71PhnRSu~U3REEtC1tlIgsd5WgcJXikIneF9Wgoaew__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-10</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig8.png?Expires=1509238860&amp;Signature=SbegKyPQpyrtdr6snVPcEGtdBSvEOTucoQEGXVR4GncnFwp1b6n7IwUtgM8QRgjqrf00jXaC4478cJJLwiUkz5a9nsfOhC4IZsy9fZx4Si79vIr4BkROKB44KxWXDhwVCisXSd-shMHXbFE9r~DckUJMx7R-GtVe4xrMKPHwf2J65jLv06Zj~fonGzkv441yyG4slYgZcKVYSOh4ENPiAMjnQfR3vnlN-v~UuS9tBMx7B87gn1OskUcqqxJed6pA-7Lma1WcxbGP9~jmQbYFRDtfO6oo6zSx-95UIYpXT-k3zwHvSva8zEY42P8p~CnU2gR26vgYJ2cL6DPM~zcgkA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-9</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig7.png?Expires=1509238860&amp;Signature=gK-ylyn0w8y3brscXU7GJucApiKO-tYF~bJ~0c9iasCDdf7Ksm8Iw7ls6rxP-nNYRpn~t7EaKedx9roEXGXrSAF4ktS9saMbnYnMZpdnd36yUKCRzctbiak3nj9yhnZOjzQthpU7GQuxyRGXHVZ2QwpgBmeHIw20TfhQU4~DTjVO-uSf4MVJAKoOlALBXyUBG2vyExzceFisc3YTc-wd6-fJKYn~hD4HOhcdEV05f2Pk0kYUpf3o~1EKStNgsayN1G0GmQGzgDcM7cLdPRque88wIjDeMDYmudBNHO8HNVppJJvckiLqJHfYj5yDul~DwkZyIHXipIWL8B3NgAaIvg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-8</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig6.pdf?Expires=1509238860&amp;Signature=Qt2uOtm4e3kviu2qFuyj3s3ufcivc9InEG4RbwdqBHZOCZXxlfnaKT2M5Stf9vAny7ZRSr~44V8XpWMYgBqkbl7WzKwxWnhvxnvoyNZA7QRCp7ENSxsjMv3nKItG6Re8fBk1mOqPsBqCiHF5deOAE4XNJbhjupgBQYDSG3-uMqSSL55LZitcnBtn1o0T7uXKDjmAox1R03ng0kSwPvZhKAQHHD0ycLMDZzaYGxZa6XH2kqjEXY6niCRCb3vbYezvksGAERCY2FfUvcxEfdXGqsyGyIhUUAXdnyutn25uYIgzqdQp8jzFy-nke3vKGLwD397tMZMD43NW6TNIYqgUrg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig5.png?Expires=1509238860&amp;Signature=LEplwTARy0SMJH6BnzQ16OGljfmRhqFcLAbRy7wLgS4Or4zHqVzqQ8I74IP63kfT4U9LEx1FGBiBXXks3Cz03ZBooksJMur01YTRjmLglL6j1Tdb-eNfSqWAVqg7jjmyLI8dQqf1-PZornLLG6Y91uSgx39WpIsU-aUkv1KZ1hV3KXlzQ7JNLk80hGvzpogvSZjQ3obnJdOhk6zf9oB-BzGvvDi0O7y8p6mdZlZ5c3hCtCOHvN0qowAqV3TpK-fupRFXEadUzIb1blP041oujY5emlo1ON2nFx61-C09rjrV6ELNwx2tIPtBWbcQ08PFIDkNPO6fd-11dyzIHCxWuw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig4.png?Expires=1509238860&amp;Signature=W5334RMXRZgroEc4t7EBKochH6aDfYoG4LHy1CZyy5zVypjlVaPK~WLHydd3xOIZvdSQYiwpET-NprKafyTCVcH3pHgBemq5CYNcDETbJ3qsunXxmx3TN5pN~46eVoFa3FhWrd76l7VLK-eMiozY1X6XLPzNEr9SgiQlZ6HCFUMqUa~ojG1Bh6X8PEzKaj7c~bvAj0RFpD3LurR2gEZ-3agvXBG8rHpknSjR7p5xZ3x0xrBTNhsy-q0auH8kiVWwihoKMjHRZa86ha8xZ6MLdeOUcQM-kg7M2FidjqqllvbRXTfcTLKc4szqI1kmCTUjefgGFosowxe48QLI9VlUSg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig3.png?Expires=1509238860&amp;Signature=fFtHL00wrTe~qgC~J-C4EyAJjek~4DxSasJl-ECOROeH-xX9xomPSL3ZJ1cNNpVsNMNJ4mLEPlEANIzU2dhLKbMpVbiH0OyTS4ELyLwA~GL-ZI3P7mTO-cFQAMxQagHaqsMGmhctMS~uvMOWtNlIybbrVkmXE~LgDEciqAe4xjiRyKCFWB3xqAnwAOtmoTxLC94UHRNczlD~bOKC4xqiA-SK3saT02D4YVdSK9Ls3VS5Iamqv19i1ko2rT4S4RXibPq0ovFQuZYk2loxksJbk~z0ARxP-un6FY-z5Wy-Q7H-fAU1LGMzk5X9WgJwoNJAbZ4vvdL0NPV7mslIis2IvQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig2.png?Expires=1509238860&amp;Signature=W4czjkDsx0aaU02~4XmxokyUDVM8WNmVp12IoDlxOh10xmDIz2OHqklNKhOYJPps4a5hBaKQxpHLTat~6ucekHFi69dIXj4KrW93s36Yr1BokVc4zwZYx~mQ~qLbCZpdwgUuteybdAsd8N~680nhr3x0deW7hyA8L2jB3BgwG6M2WgDJ-C8tTMucHZxn3V7zFYdw-ri1S1xW2S-rJC5MyU7pAcSXFqz6hATh4VIhRUrbRk5tLMMSRDZ5hUumybKbBcq-cVdHcPxdgKhPRrn5lupy2zuYHmbak~MNtxALq9sm7VHM5bZxUKhoO~rvg-joNXU0iCFhw-UoyYGZBadOZg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp_fig1.png?Expires=1509238860&amp;Signature=IiA76it7FLaj8HSrQdU4oqGc8NcMHtOnWTD-qEvdbUuBBWA-nOiFHkR8qsGoZPtt0geFagKMQSTRH6lZeRB~BcJQLXIhVD2blu1Q14B5tBRWWRZ8eR5jVYOLTQreBpSBWRdLB6K4SOFY5N7ZoXYSt~iVn6XWk3tqMnAFRgra4iQAVvO19JuYHXsWoLHKPB4TKrSO4aqVqQLdQtk9EqtLlT7cTbrWsX81iyqiAlS1TEleAn6UFImgzKbq6xAHdbCU6fs9JibCUM6QAeenZ84fCDZb~53S7jsSUUCkL94PjpgEjE1Try9c8pGDPm0PBUKE1kpqH-SW3ip7YxW5n3vExg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss033/2/dss033-dss033supp.doc?Expires=1509238860&amp;Signature=Os4xuhQOfq6Kq5D62qWbvs~e734U3S4g-dHrO1WLTkpjN67olfp7X8HbvGA6PpU8i3zOkrVTfH9ayzBCrspqkYOKGxcxQyEFyjo5KA~LSfLb6y~5ViujA~AgPfc~Fg0NnY4eF8ABJMihGYEf4srKO4dWN7F4o3ErdQ54WIyKCKvmzxX0WxQnvq7TrBzHB4RaaRqeAaQlwXH6dzms-rQ9X3tMYGboehHuGdPr3~Hp9nUhbdzgUy3UOc8413RaEu~7CbAJmqQeDKfgU5NuhD-0lWvm2bWwisWJJwK4TLz4D1BjeON-arkt9ksyUB1x0pH32QTchr8sjkom2s8SPz5cUg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">34</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">9</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dss033"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="383179"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f55%2f383179">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (9)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F55%2F383179" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/383179'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Polygenic Molecular Architecture Underlying Non-Sexual Cell Aggregation in Budding Yeast" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f55%2f383179">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

