

<!DOCTYPE html>
<html lang="en">

<head><title>
	Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/79/440782","@type":"ScholarlyArticle","name":"Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database","datePublished":"2014-11-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu039","keywords":["foxtail millet (<italic>Setaria italica</italic> L.)","transposable element-based markers","retrotransposons","DNA transposons","database"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/79/440782","author":[{"name":"Yadav, Chandra Bhan","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Bonthala, Venkata Suresh","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Muthamilarasan, Mehanathan","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Pandey, Garima","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Khan, Yusuf","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Prasad, Manoj","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"}],"description":"Transposable elements (TEs) are major components of plant genome and are reported to play significant roles in functional genome diversity and phenotypic variations. Several TEs are highly polymorphic for insert location in the genome and this facilitates development of TE-based markers for various genotyping purposes. Considering this, a genome-wide analysis was performed in the model plant foxtail millet. A total of 30,706 TEs were identified and classified as DNA transposons (24,386), full-length Copia type (1,038), partial or solo Copia type (10,118), full-length Gypsy type (1,570), partial or solo Gypsy type (23,293) and Long- and Short-Interspersed Nuclear Elements (3,659 and 53, respectively). Further, 20,278 TE-based markers were developed, namely Retrotransposon-Based Insertion Polymorphisms (4,801, ∼24%), Inter-Retrotransposon Amplified Polymorphisms (3,239, ∼16%), Repeat Junction Markers (4,451, ∼22%), Repeat Junction-Junction Markers (329, ∼2%), Insertion-Site-Based Polymorphisms (7,401, ∼36%) and Retrotransposon-Microsatellite Amplified Polymorphisms (57, 0.2%). A total of 134 Repeat Junction Markers were screened in 96 accessions of Setaria italica and 3 wild Setaria accessions of which 30 showed polymorphism. Moreover, an open access database for these developed resources was constructed (Foxtail millet Transposable Elements-based Marker Database; http://59.163.192.83/ltrdb/index.html). Taken together, this study would serve as a valuable resource for large-scale genotyping applications in foxtail millet and related grass species.","pageStart":"79","pageEnd":"90","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217382&Signature=DiV-WYNGYYId3HCUzJbf~zCBc0ha0L4X9l0v07v0s3VUWzMIdU5xke3tS0nsxThHjf1Fy64QpCqRfjJUpJw5Sy0xlrtalUDGPkfC8vW1aM0Yp4sGEos4e~rmpLH0jU6f~vCBPk7dPGFPaziMxoxAyZAGdzi0Fhc1XfI3BHbmeCho~B3ykJOA5xX-1aDUxyI6G58JQMQaWFvczoFQxAOVXEkVM~lDu317XBKk6aQE5A4dmcGktnnwkjczkKfm6BdVCarag6-u~4Fu~zBpijLe0HogRmLDBRCjm-fx7CtQx84tCwuIUcxeC0EMw6WtFtpemhri2~-DotYooRTP41NJlg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217382&Signature=DiV-WYNGYYId3HCUzJbf~zCBc0ha0L4X9l0v07v0s3VUWzMIdU5xke3tS0nsxThHjf1Fy64QpCqRfjJUpJw5Sy0xlrtalUDGPkfC8vW1aM0Yp4sGEos4e~rmpLH0jU6f~vCBPk7dPGFPaziMxoxAyZAGdzi0Fhc1XfI3BHbmeCho~B3ykJOA5xX-1aDUxyI6G58JQMQaWFvczoFQxAOVXEkVM~lDu317XBKk6aQE5A4dmcGktnnwkjczkKfm6BdVCarag6-u~4Fu~zBpijLe0HogRmLDBRCjm-fx7CtQx84tCwuIUcxeC0EMw6WtFtpemhri2~-DotYooRTP41NJlg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Transposable elements (TEs) are major components of plant genome and are reported to play significant roles in functional genome diversity and phenotypic variations. Several TEs are highly polymorphic for insert location in the genome and this facilitates development of TE-based markers for various genotyping purposes. Considering this, a genome-wide analysis was performed in the model plant foxtail millet. A total of 30,706 TEs were identified and classified as DNA transposons (24,386), full-length Copia type (1,038), partial or solo Copia type (10,118), full-length Gypsy type (1,570), partial or solo Gypsy type (23,293) and Long- and Short-Interspersed Nuclear Elements (3,659 and 53, respectively). Further, 20,278 TE-based markers were developed, namely Retrotransposon-Based Insertion Polymorphisms (4,801, ∼24%), Inter-Retrotransposon Amplified Polymorphisms (3,239, ∼16%), Repeat Junction Markers (4,451, ∼22%), Repeat Junction-Junction Markers (329, ∼2%), Insertion-Site-Based Polymorphisms (7,401, ∼36%) and Retrotransposon-Microsatellite Amplified Polymorphisms (57, 0.2%). A total of 134 Repeat Junction Markers were screened in 96 accessions of Setaria italica and 3 wild Setaria accessions of which 30 showed polymorphism. Moreover, an open access database for these developed resources was constructed (Foxtail millet Transposable Elements-based Marker Database; http://59.163.192.83/ltrdb/index.html). Taken together, this study would serve as a valuable resource for large-scale genotyping applications in foxtail millet and related grass species." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/79/440782" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217382&amp;Signature=ZuZicucSQ8P7i8xd7n4cQlkIrlBO8wqTUVkLMhv7fTu7xeymkBZq72P-j42SHP6yL1mu7nN4DOFnSJp9kbLsUetqbUBeNgyNHM0lHfvv-tGgG2~GNt7ZB1gQ1KdhDSOFC~GWhVFO16dfV4jYgRxsjfey4FZNySPFydMJtf2R61kHqzdJQRacEpTRcPe0JeV8uRXcEzkZ7Ixf0U-aC5gub0kdd4-o~LYgYV1lNIhsqfHjI8pYRwwD~1bYyhOpi4OZVy3cQGANa1KyjEia0PyeGe6lmPfZ9KLwsXhUdFBE-c7DAIRgtgCCKqhaIQqEQlI59RXFj95KEdzXx~uBgCebHA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2014-11-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yadav, Chandra Bhan" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Bonthala, Venkata Suresh" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Muthamilarasan, Mehanathan" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Pandey, Garima" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Khan, Yusuf" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Prasad, Manoj" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_title" content="Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database" /><meta name="citation_firstpage" content="79" /><meta name="citation_lastpage" content="90" /><meta name="citation_doi" content="10.1093/dnares/dsu039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Retrotransposon sequence variation in four asexual plant species; J. Mol. Evol.;  citation_year=2006;  citation_volume=62;  citation_pages=375-87; " /><meta name="citation_reference" content="citation_title=Plant transposable elements: where genetics meets genomics; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=329-41; " /><meta name="citation_reference" content="citation_title=Do plants have a one-way ticket to genomic obesity?; Plant Cell;  citation_year=1997;  citation_volume=9;  citation_pages=1509-14; " /><meta name="citation_reference" content="citation_title=The paleontology of intergene retrotransposons of maize; Nat. Genet.;  citation_year=1998;  citation_volume=20;  citation_pages=43-5; " /><meta name="citation_reference" content="citation_title=Abundance, distribution and transcriptional activity of repetitive elements in the maize genome; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1660-76; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=Analysis of the bread wheat genome using whole-genome shotgun sequencing; Nature;  citation_year=2012;  citation_volume=491;  citation_pages=705-10; " /><meta name="citation_reference" content="citation_title=RJPrimers: unique transposable element insertion junction discovery and PCR primer design for marker development; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=W313-20; " /><meta name="citation_reference" content="citation_title=Rapid development of PCR-based genome specific repetitive DNA junction markers in wheat; Genome;  citation_year=2009;  citation_volume=52;  citation_pages=576-87; " /><meta name="citation_reference" content="citation_title=Transposable element junctions in marker development and genomic characterization of barley; Plant Genome;  citation_year=2014;  citation_volume=7;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Foxtail millet: a sequence-driven grass model system; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=137-41; " /><meta name="citation_reference" content="citation_title=Setaria viridis and Setaria italica, model genetic systems for Panicoid grasses; J. Exp. Bot.;  citation_year=2011;  citation_volume=62;  citation_pages=3031-7; " /><meta name="citation_reference" content="citation_title=Foxtail millet: a model crop for genetic and genomic studies in bioenergy grasses; Crit. Rev. Biotechnol.;  citation_year=2013;  citation_volume=33;  citation_pages=328-43; " /><meta name="citation_reference" content="citation_title=Advances in Setaria genomics for genetic improvement of cereals and bioenergy grasses; Theor. Appl. Genet.;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Genome sequence of foxtail millet (Setaria italica) provides insights into grass evolution and biofuel potential; Nature Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=549-54; " /><meta name="citation_reference" content="citation_title=Reference genome sequence of the model plant Setaria; Nature Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=555-61; " /><meta name="citation_reference" content="citation_title=Genome-wide development and use of microsatellite markers for large-scale genotyping applications in foxtail millet [Setaria italica (L.)]; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=197-207; " /><meta name="citation_reference" content="citation_title=Development of highly polymorphic simple sequence repeat markers using genome-wide microsatellite variant analysis in Foxtail millet [Setaria italica (L.) P. Beauv.]; BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=78" /><meta name="citation_reference" content="citation_title=Development of eSSR-markers in Setaria italica and their applicability in studying genetic diversity, cross-transferability and comparative mapping in millet and non-millet species; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e67742" /><meta name="citation_reference" content="citation_title=Development of 5123 intron-length polymorphic markers for large-scale genotyping applications in foxtail millet; DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=41-52; " /><meta name="citation_reference" content="citation_title=LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W265-8; " /><meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3497-500; " /><meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions; Mol. Biol. Evol.;  citation_year=1986;  citation_volume=3;  citation_pages=418-26; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=WEGO: a web tool for plotting GO annotations; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W293-7; " /><meta name="citation_reference" content="citation_title=Facing growth in the European Nucleotide Archive; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D30-5; " /><meta name="citation_reference" content="citation_title=TARGeT: a web-based pipeline for retrieving and characterizing gene and transposable element families from genomic sequences; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=e78" /><meta name="citation_reference" content="citation_text=Data analysis methods; citation_publisher=Science Publishers, Montpellier, France; Genetic diversity of cultivated tropical plants;  citation_year=2003;  citation_pages=43-76; " /><meta name="citation_reference" content="citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data; Mol. Biol. Evol.;  citation_year=1997;  citation_volume=14;  citation_pages=685-95; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies; Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=1567-87; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; Mol. Ecol.;  citation_year=2005;  citation_volume=14;  citation_pages=2611-20; " /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=Eukaryotic transposable elements and genome evolution; Trends Genet.;  citation_year=1989;  citation_volume=5;  citation_pages=103-7; " /><meta name="citation_reference" content="citation_title=Evolutionary history of Oryza sativa LTR retrotransposons: a preliminary survey of the rice genome sequences; BMC Genomics;  citation_year=2004;  citation_volume=5;  citation_pages=18" /><meta name="citation_reference" content="citation_title=Rapid recent growth and divergence of rice nuclear genomes; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=12404-10; " /><meta name="citation_reference" content="citation_title=Natural selection on gene function drives the evolution of LTR retrotransposon families in the rice genome; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=243-54; " /><meta name="citation_reference" content="citation_title=New insights into the history of rice domestication; Trends Genet.;  citation_year=2007;  citation_volume=23;  citation_pages=578-87; " /><meta name="citation_reference" content="citation_title=Genetics and phylogenetics of rice domestication; Curr. Opin. Genet. Dev.;  citation_year=2007;  citation_volume=17;  citation_pages=533-8; " /><meta name="citation_reference" content="citation_title=The complex history of the domestication of rice; Ann. Bot.;  citation_year=2007;  citation_volume=100;  citation_pages=951-7; " /><meta name="citation_reference" content="citation_title=Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1157-69; " /><meta name="citation_reference" content="citation_title=Characterization and functional annotation of nested transposable elements in eukaryotic genomes; Genomics;  citation_year=2012;  citation_volume=100;  citation_pages=222-30; " /><meta name="citation_reference" content="citation_title=Retrotransposon-based insertion polymorphisms (RBIP) for high throughput marker analysis; Plant J.;  citation_year=1998;  citation_volume=16;  citation_pages=643-50; " /><meta name="citation_reference" content="citation_title=Advances in plant gene-targeted and functional markers: a review; Plant Methods;  citation_year=2013;  citation_volume=9;  citation_pages=6" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/79/1208942/dsu039.pdf" /><meta name="description" content="Abstract.  Transposable elements (TEs) are major components of plant genome and are reported to play significant roles in functional genome diversity and phenot" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/79/440782" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/79/440782" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsu039';
    </script>
</head>

<script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2215,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,n,e){function r(e){if(!n[e]){var o=n[e]={exports:{}};t[e][0].call(o.exports,function(n){var o=t[e][1][n];return r(o||n)},o,o.exports)}return n[e].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<e.length;o++)r(e[o]);return r}({1:[function(t,n,e){function r(t){try{s.console&&console.log(t)}catch(n){}}var o,i=t("ee"),a=t(15),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,n,e){r(e.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,n){return t}).join(", ")))},{}],2:[function(t,n,e){function r(t,n,e,r,o){try{d?d-=1:i("err",[o||new UncaughtException(t,n,e)])}catch(s){try{i("ierr",[s,c.now(),!0])}catch(u){}}return"function"==typeof f&&f.apply(this,a(arguments))}function UncaughtException(t,n,e){this.message=t||"Uncaught error with no additional information",this.sourceURL=n,this.line=e}function o(t){i("err",[t,c.now()])}var i=t("handle"),a=t(16),s=t("ee"),c=t("loader"),f=window.onerror,u=!1,d=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(l){"stack"in l&&(t(8),t(7),"addEventListener"in window&&t(5),c.xhrWrappable&&t(9),u=!0)}s.on("fn-start",function(t,n,e){u&&(d+=1)}),s.on("fn-err",function(t,n,e){u&&(this.thrown=!0,o(e))}),s.on("fn-end",function(){u&&!this.thrown&&d>0&&(d-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,n,e){t("loader").features.ins=!0},{}],4:[function(t,n,e){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(8),s=t(7),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",l="resource",p="-start",h="-end",m="fn"+p,w="fn"+h,v="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(6);var b=NREUM.o.EV;o.on(m,function(t,n){var e=t[0];e instanceof b&&(this.bstStart=g.now())}),o.on(w,function(t,n){var e=t[0];e instanceof b&&i("bst",[e,n,this.bstStart,g.now()])}),a.on(m,function(t,n,e){this.bstStart=g.now(),this.bstType=e}),a.on(w,function(t,n){i(v,[n,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(w,function(t,n){i(v,[n,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+p,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],5:[function(t,n,e){function r(t){for(var n=t;n&&!n.hasOwnProperty(u);)n=Object.getPrototypeOf(n);n&&o(n)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,n){return t[1]}var a=t("ee").get("events"),s=t(18)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";n.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,n){var e=t[1],r=c(e,"nr@wrapped",function(){function t(){if("function"==typeof e.handleEvent)return e.handleEvent.apply(e,arguments)}var n={object:t,"function":e}[typeof e];return n?s(n,"fn-",null,n.name||"anonymous"):e});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],6:[function(t,n,e){var r=t("ee").get("history"),o=t(18)(r);n.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],7:[function(t,n,e){var r=t("ee").get("raf"),o=t(18)(r),i="equestAnimationFrame";n.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],8:[function(t,n,e){function r(t,n,e){t[0]=a(t[0],"fn-",null,e)}function o(t,n,e){this.method=e,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,e)}var i=t("ee").get("timer"),a=t(18)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";n.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],9:[function(t,n,e){function r(t,n){d.inPlace(n,["onreadystatechange"],"fn-",s)}function o(){var t=this,n=u.context(t);t.readyState>3&&!n.resolved&&(n.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):w?w(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,n){return n}function c(t,n){for(var e in t)n[e]=t[e];return n}t(5);var f=t("ee"),u=f.get("xhr"),d=t(18)(u),l=NREUM.o,p=l.XHR,h=l.MO,m=l.PR,w=l.SI,v="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];n.exports=u;var b=window.XMLHttpRequest=function(t){var n=new p(t);try{u.emit("new-xhr",[n],n),n.addEventListener(v,o,!1)}catch(e){try{u.emit("internal-error",[e])}catch(r){}}return n};if(c(p,b),b.prototype=p.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,n){r(t,n),i(n)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!w&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===v||a()})},{}],10:[function(t,n,e){function r(t){var n=this.params,e=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!n.aborted){if(e.duration=a.now()-this.startTime,4===t.readyState){n.status=t.status;var i=o(t,this.lastSize);if(i&&(e.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(n.cat=c.split(", ").pop())}}else n.status=0;e.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[n,e,this.startTime])}}}function o(t,n){var e=t.responseType;if("json"===e&&null!==n)return n;var r="arraybuffer"===e||"blob"===e||"json"===e?t.response:t.responseText;return h(r)}function i(t,n){var e=c(n),r=t.params;r.host=e.hostname+":"+e.port,r.pathname=e.pathname,t.sameOrigin=e.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(11),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,l=t("id"),p=t(14),h=t(13),m=window.XMLHttpRequest;a.features.xhr=!0,t(9),f.on("new-xhr",function(t){var n=this;n.totalCbs=0,n.called=0,n.cbTime=0,n.end=r,n.ended=!1,n.xhrGuids={},n.lastSize=null,p&&(p>34||p<10)||window.opera||t.addEventListener("progress",function(t){n.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,n){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,n){var e=this.metrics,r=t[0],o=this;if(e&&r){var i=h(r);i&&(e.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof n.onload))&&o.end(n)}catch(e){try{f.emit("internal-error",[e])}catch(r){}}};for(var s=0;s<d;s++)n.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,n,e){this.cbTime+=t,n?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof e.onload||this.end(e)}),f.on("xhr-load-added",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&!this.xhrGuids[e]&&(this.xhrGuids[e]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&this.xhrGuids[e]&&(delete this.xhrGuids[e],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],n)}),f.on("removeEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],n)}),f.on("fn-start",function(t,n,e){n instanceof m&&("onload"===e&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,n){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,n],n)})}},{}],11:[function(t,n,e){n.exports=function(t){var n=document.createElement("a"),e=window.location,r={};n.href=t,r.port=n.port;var o=n.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=n.hostname||e.hostname,r.pathname=n.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!n.protocol||":"===n.protocol||n.protocol===e.protocol,a=n.hostname===document.domain&&n.port===e.port;return r.sameOrigin=i&&(!n.hostname||a),r}},{}],12:[function(t,n,e){function r(){}function o(t,n,e){return function(){return i(t,[f.now()].concat(s(arguments)),n?null:this,e),n?void 0:this}}var i=t("handle"),a=t(15),s=t(16),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",p=l+"ixn-";a(d,function(t,n){u[n]=o(l+n,!0,"api")}),u.addPageAction=o(l+"addPageAction",!0),u.setCurrentRouteName=o(l+"routeName",!0),n.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,n){var e={},r=this,o="function"==typeof n;return i(p+"tracer",[f.now(),t,e],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],e),o)try{return n.apply(this,arguments)}finally{c.emit("fn-end",[f.now()],e)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,n){h[n]=o(p+n)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],13:[function(t,n,e){n.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(n){return}}}},{}],14:[function(t,n,e){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),n.exports=r},{}],15:[function(t,n,e){function r(t,n){var e=[],r="",i=0;for(r in t)o.call(t,r)&&(e[i]=n(r,t[r]),i+=1);return e}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],16:[function(t,n,e){function r(t,n,e){n||(n=0),"undefined"==typeof e&&(e=t?t.length:0);for(var r=-1,o=e-n||0,i=Array(o<0?0:o);++r<o;)i[r]=t[n+r];return i}n.exports=r},{}],17:[function(t,n,e){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],18:[function(t,n,e){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(16),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;n.exports=function(t,n){function e(t,n,e,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof e?e(r,a):e||{}}catch(f){l([f,"",[r,a,o],s])}u(n+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(n+"err",[r,a,d],s),d}finally{u(n+"end",[r,a,c],s)}}return r(t)?t:(n||(n=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,n,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<n.length;c++)s=n[c],a=t[s],r(a)||(t[s]=e(a,f?s+o:o,i,s))}function u(e,r,o){if(!c||n){var i=c;c=!0;try{t.emit(e,r,o,n)}catch(a){l([a,e,r,o])}c=i}}function d(t,n){if(Object.defineProperty&&Object.keys)try{var e=Object.keys(t);return e.forEach(function(e){Object.defineProperty(n,e,{get:function(){return t[e]},set:function(n){return t[e]=n,n}})}),n}catch(r){l([r])}for(var o in t)s.call(t,o)&&(n[o]=t[o]);return n}function l(n){try{t.emit("internal-error",n)}catch(e){}}return t||(t=o),e.inPlace=f,e.flag=a,e}},{}],ee:[function(t,n,e){function r(){}function o(t){function n(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function e(e,r,o,i){if(!l.aborted||i){t&&t(e,r,o);for(var a=n(o),s=h(e),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[y[e]];return d&&d.push([g,e,r,a]),a}}function p(t,n){v[t]=h(t).concat(n)}function h(t){return v[t]||[]}function m(t){return d[t]=d[t]||o(e)}function w(t,n){f(t,function(t,e){n=n||"feature",y[e]=n,n in u||(u[n]=[])})}var v={},y={},g={on:p,emit:e,get:m,listeners:h,context:n,buffer:w,abort:a,aborted:!1};return g}function i(){return new r}function a(){(u.api||u.feature)&&(l.aborted=!0,u=l.backlog={})}var s="nr@context",c=t("gos"),f=t(15),u={},d={},l=n.exports=o();l.backlog=u},{}],gos:[function(t,n,e){function r(t,n,e){if(o.call(t,n))return t[n];var r=e();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(t,n,e){function r(t,n,e,r){o.buffer([t],r),o.emit(t,n,e)}var o=t("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(t,n,e){function r(t){var n=typeof t;return!t||"object"!==n&&"function"!==n?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");n.exports=r},{}],loader:[function(t,n,e){function r(){if(!x++){var t=b.info=NREUM.info,n=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&n))return u.abort();f(y,function(n,e){t[n]||(t[n]=e)}),c("mark",["onload",a()+b.offset],null,"api");var e=l.createElement("script");e.src="https://"+t.agent,n.parentNode.insertBefore(e,n)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+b.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-b.offset}var s=(new Date).getTime(),c=t("handle"),f=t(15),u=t("ee"),d=window,l=d.document,p="addEventListener",h="attachEvent",m=d.XMLHttpRequest,w=m&&m.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:m,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},g=m&&w&&w[p]&&!/CriOS/.test(navigator.userAgent),b=n.exports={offset:s,now:a,origin:v,features:{},xhrWrappable:g};t(12),l[p]?(l[p]("DOMContentLoaded",i,!1),d[p]("load",r,!1)):(l[h]("onreadystatechange",o),d[h]("onload",r)),c("mark",["firstbyte",s],null,"api");var x=0,E=t(17)},{}]},{},["loader",2,10,4,3]);</script><body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./440782" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFimI8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhZGF2LCBDaGFuZHJhIEJoYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJvbnRoYWxhLCBWZW5rYXRhIFN1cmVzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTXV0aGFtaWxhcmFzYW4sIE1laGFuYXRoYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhbmRleSwgR2FyaW1hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaGFuLCBZdXN1ZiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUHJhc2FkLCBNYW5vaiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtd2lkZSBkZXZlbG9wbWVudCBvZiB0cmFuc3Bvc2FibGUgZWxlbWVudHMtYmFzZWQgbWFya2VycyBpbiBmb3h0YWlsIG1pbGxldCBhbmQgY29uc3RydWN0aW9uIG9mIGFuIGludGVncmF0ZWQgZGF0YWJhc2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iOTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldHJvdHJhbnNwb3NvbiBzZXF1ZW5jZSB2YXJpYXRpb24gaW4gZm91ciBhc2V4dWFsIHBsYW50IHNwZWNpZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0zNzUtODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHRyYW5zcG9zYWJsZSBlbGVtZW50czogd2hlcmUgZ2VuZXRpY3MgbWVldHMgZ2Vub21pY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMyOS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG8gcGxhbnRzIGhhdmUgYSBvbmUtd2F5IHRpY2tldCB0byBnZW5vbWljIG9iZXNpdHk/OyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNTA5LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGFsZW9udG9sb2d5IG9mIGludGVyZ2VuZSByZXRyb3RyYW5zcG9zb25zIG9mIG1haXplOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTQzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFidW5kYW5jZSwgZGlzdHJpYnV0aW9uIGFuZCB0cmFuc2NyaXB0aW9uYWwgYWN0aXZpdHkgb2YgcmVwZXRpdGl2ZSBlbGVtZW50cyBpbiB0aGUgbWFpemUgZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE2NjAtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGZsb3dlcmluZyBwbGFudCBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgY2l0YXRpb25fYXV0aG9yPUFyYWJpZG9wc2lzIEdlbm9tZSBJbml0aWF0aXZlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGJyZWFkIHdoZWF0IGdlbm9tZSB1c2luZyB3aG9sZS1nZW5vbWUgc2hvdGd1biBzZXF1ZW5jaW5nOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkxOyAgY2l0YXRpb25fcGFnZXM9NzA1LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SSlByaW1lcnM6IHVuaXF1ZSB0cmFuc3Bvc2FibGUgZWxlbWVudCBpbnNlcnRpb24ganVuY3Rpb24gZGlzY292ZXJ5IGFuZCBQQ1IgcHJpbWVyIGRlc2lnbiBmb3IgbWFya2VyIGRldmVsb3BtZW50OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1XMzEzLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBkZXZlbG9wbWVudCBvZiBQQ1ItYmFzZWQgZ2Vub21lIHNwZWNpZmljIHJlcGV0aXRpdmUgRE5BIGp1bmN0aW9uIG1hcmtlcnMgaW4gd2hlYXQ7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTU3Ni04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NhYmxlIGVsZW1lbnQganVuY3Rpb25zIGluIG1hcmtlciBkZXZlbG9wbWVudCBhbmQgZ2Vub21pYyBjaGFyYWN0ZXJpemF0aW9uIG9mIGJhcmxleTsgUGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZveHRhaWwgbWlsbGV0OiBhIHNlcXVlbmNlLWRyaXZlbiBncmFzcyBtb2RlbCBzeXN0ZW07IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTEzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2V0YXJpYSB2aXJpZGlzIGFuZCBTZXRhcmlhIGl0YWxpY2EsIG1vZGVsIGdlbmV0aWMgc3lzdGVtcyBmb3IgUGFuaWNvaWQgZ3Jhc3NlczsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MzAzMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3h0YWlsIG1pbGxldDogYSBtb2RlbCBjcm9wIGZvciBnZW5ldGljIGFuZCBnZW5vbWljIHN0dWRpZXMgaW4gYmlvZW5lcmd5IGdyYXNzZXM7IENyaXQuIFJldi4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zMjgtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIFNldGFyaWEgZ2Vub21pY3MgZm9yIGdlbmV0aWMgaW1wcm92ZW1lbnQgb2YgY2VyZWFscyBhbmQgYmlvZW5lcmd5IGdyYXNzZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBmb3h0YWlsIG1pbGxldCAoU2V0YXJpYSBpdGFsaWNhKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGdyYXNzIGV2b2x1dGlvbiBhbmQgYmlvZnVlbCBwb3RlbnRpYWw7IE5hdHVyZSBCaW90ZWNoLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVmZXJlbmNlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbW9kZWwgcGxhbnQgU2V0YXJpYTsgTmF0dXJlIEJpb3RlY2guOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NTU1LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkZXZlbG9wbWVudCBhbmQgdXNlIG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIGxhcmdlLXNjYWxlIGdlbm90eXBpbmcgYXBwbGljYXRpb25zIGluIGZveHRhaWwgbWlsbGV0IFtTZXRhcmlhIGl0YWxpY2EgKEwuKV07IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTk3LTIwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgaGlnaGx5IHBvbHltb3JwaGljIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyB1c2luZyBnZW5vbWUtd2lkZSBtaWNyb3NhdGVsbGl0ZSB2YXJpYW50IGFuYWx5c2lzIGluIEZveHRhaWwgbWlsbGV0IFtTZXRhcmlhIGl0YWxpY2EgKEwuKSBQLiBCZWF1di5dOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz03OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGVTU1ItbWFya2VycyBpbiBTZXRhcmlhIGl0YWxpY2EgYW5kIHRoZWlyIGFwcGxpY2FiaWxpdHkgaW4gc3R1ZHlpbmcgZ2VuZXRpYyBkaXZlcnNpdHksIGNyb3NzLXRyYW5zZmVyYWJpbGl0eSBhbmQgY29tcGFyYXRpdmUgbWFwcGluZyBpbiBtaWxsZXQgYW5kIG5vbi1taWxsZXQgc3BlY2llczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2Nzc0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIDUxMjMgaW50cm9uLWxlbmd0aCBwb2x5bW9ycGhpYyBtYXJrZXJzIGZvciBsYXJnZS1zY2FsZSBnZW5vdHlwaW5nIGFwcGxpY2F0aW9ucyBpbiBmb3h0YWlsIG1pbGxldDsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz00MS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TFRSX0ZJTkRFUjogYW4gZWZmaWNpZW50IHRvb2wgZm9yIHRoZSBwcmVkaWN0aW9uIG9mIGZ1bGwtbGVuZ3RoIExUUiByZXRyb3RyYW5zcG9zb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XMjY1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIHRoZSBDbHVzdGFsIHNlcmllcyBvZiBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzQ5Ny01MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBtZXRob2RzIGZvciBlc3RpbWF0aW5nIHRoZSBudW1iZXJzIG9mIHN5bm9ueW1vdXMgYW5kIG5vbnN5bm9ueW1vdXMgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb25zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz00MTgtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E0OiBNb2xlY3VsYXIgRXZvbHV0aW9uYXJ5IEdlbmV0aWNzIEFuYWx5c2lzIChNRUdBKSBzb2Z0d2FyZSB2ZXJzaW9uIDQuMDsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmxhc3QyR086IGEgdW5pdmVyc2FsIHRvb2wgZm9yIGFubm90YXRpb24sIHZpc3VhbGl6YXRpb24gYW5kIGFuYWx5c2lzIGluIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcmVzZWFyY2g7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzY3NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XRUdPOiBhIHdlYiB0b29sIGZvciBwbG90dGluZyBHTyBhbm5vdGF0aW9uczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9VzI5My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYWNpbmcgZ3Jvd3RoIGluIHRoZSBFdXJvcGVhbiBOdWNsZW90aWRlIEFyY2hpdmU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UQVJHZVQ6IGEgd2ViLWJhc2VkIHBpcGVsaW5lIGZvciByZXRyaWV2aW5nIGFuZCBjaGFyYWN0ZXJpemluZyBnZW5lIGFuZCB0cmFuc3Bvc2FibGUgZWxlbWVudCBmYW1pbGllcyBmcm9tIGdlbm9taWMgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1lNzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90ZXh0PURhdGEgYW5hbHlzaXMgbWV0aG9kczsgY2l0YXRpb25fcHVibGlzaGVyPVNjaWVuY2UgUHVibGlzaGVycywgTW9udHBlbGxpZXIsIEZyYW5jZTsgR2VuZXRpYyBkaXZlcnNpdHkgb2YgY3VsdGl2YXRlZCB0cm9waWNhbCBwbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl9wYWdlcz00My03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QklPTko6IGFuIGltcHJvdmVkIHZlcnNpb24gb2YgdGhlIE5KIGFsZ29yaXRobSBiYXNlZCBvbiBhIHNpbXBsZSBtb2RlbCBvZiBzZXF1ZW5jZSBkYXRhOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9Njg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOiBsaW5rZWQgbG9jaSBhbmQgY29ycmVsYXRlZCBhbGxlbGUgZnJlcXVlbmNpZXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTE1NjctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVyZW5jZSBvZiBwb3B1bGF0aW9uIHN0cnVjdHVyZSB1c2luZyBtdWx0aWxvY3VzIGdlbm90eXBlIGRhdGE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTk0NS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHRoZSBudW1iZXIgb2YgY2x1c3RlcnMgb2YgaW5kaXZpZHVhbHMgdXNpbmcgdGhlIHNvZnR3YXJlIFNUUlVDVFVSRTogYSBzaW11bGF0aW9uIHN0dWR5OyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjYxMS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY29zOiBhbiBpbmZvcm1hdGlvbiBhZXN0aGV0aWMgZm9yIGNvbXBhcmF0aXZlIGdlbm9taWNzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE2MzktNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV1a2FyeW90aWMgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGFuZCBnZW5vbWUgZXZvbHV0aW9uOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgT3J5emEgc2F0aXZhIExUUiByZXRyb3RyYW5zcG9zb25zOiBhIHByZWxpbWluYXJ5IHN1cnZleSBvZiB0aGUgcmljZSBnZW5vbWUgc2VxdWVuY2VzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgcmVjZW50IGdyb3d0aCBhbmQgZGl2ZXJnZW5jZSBvZiByaWNlIG51Y2xlYXIgZ2Vub21lczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xMjQwNC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmF0dXJhbCBzZWxlY3Rpb24gb24gZ2VuZSBmdW5jdGlvbiBkcml2ZXMgdGhlIGV2b2x1dGlvbiBvZiBMVFIgcmV0cm90cmFuc3Bvc29uIGZhbWlsaWVzIGluIHRoZSByaWNlIGdlbm9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yNDMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBpbnNpZ2h0cyBpbnRvIHRoZSBoaXN0b3J5IG9mIHJpY2UgZG9tZXN0aWNhdGlvbjsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3MgYW5kIHBoeWxvZ2VuZXRpY3Mgb2YgcmljZSBkb21lc3RpY2F0aW9uOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTUzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxleCBoaXN0b3J5IG9mIHRoZSBkb21lc3RpY2F0aW9uIG9mIHJpY2U7IEFubi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz05NTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBzdHJ1Y3R1cmUgcHJlZGljdGlvbiBmcm9tIGNvbnNlbnN1cyBzcGxpY2VkIGFsaWdubWVudCBvZiBtdWx0aXBsZSBFU1RzIG1hdGNoaW5nIHRoZSBzYW1lIGdlbm9taWMgbG9jdXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTE1Ny02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBhbmQgZnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIG5lc3RlZCB0cmFuc3Bvc2FibGUgZWxlbWVudHMgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0yMjItMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldHJvdHJhbnNwb3Nvbi1iYXNlZCBpbnNlcnRpb24gcG9seW1vcnBoaXNtcyAoUkJJUCkgZm9yIGhpZ2ggdGhyb3VnaHB1dCBtYXJrZXIgYW5hbHlzaXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NjQzLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlcyBpbiBwbGFudCBnZW5lLXRhcmdldGVkIGFuZCBmdW5jdGlvbmFsIG1hcmtlcnM6IGEgcmV2aWV3OyBQbGFudCBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzEvNzkvMTIwODk0Mi9kc3UwMzkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRyYW5zcG9zYWJsZSBlbGVtZW50cyAoVEVzKSBhcmUgbWFqb3IgY29tcG9uZW50cyBvZiBwbGFudCBnZW5vbWUgYW5kIGFyZSByZXBvcnRlZCB0byBwbGF5IHNpZ25pZmljYW50IHJvbGVzIGluIGZ1bmN0aW9uYWwgZ2Vub21lIGRpdmVyc2l0eSBhbmQgcGhlbm90IiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NDA3ODImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2RkV+WaGWGcdq1H0HBZATW1630HeRA=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f79%2f440782" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f79%2f440782" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1509238968&amp;Signature=EKhDR45hE0JvCD7NNoA5tSJUkxjnj69koznjLkmJx9te8bPX36EV57k~dMqMhpf3UAbpL09FykjVTRUPi2x23uanqwItrhyn8sAZZUUv2dvBiH9l0WZ6cSlozq3SPquUUWmBRlnRTNgbm5fxFw0n91Aul7Ya3Vji8eKqL9EI7JcGCTEk-igrHqdfk7ijALyGRxwV4NYzfVaUgtvZktJK08~hfyTg-fajyf3XpnObLSqYcXdCqdrDxxVTZT5GCzwl3m4aQzhKLn~WfSIBiOsH7BtEqv~Worvo48Dql9DyPmT1BhyAaf69BFpFYf3wni-4Ob7kad7ees~OsADdgayfmw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6584123">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584123">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584125">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584125">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584129">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584129">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584149">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584149">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584193">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584193">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584196">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584196">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584198">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584198">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584200">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584200">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6584203">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6584203">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/69/441516">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/91/441732">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Chandra Bhan Yadav</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chandra Bhan Yadav</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Chandra+Bhan+Yadav">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yadav C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yadav author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Venkata Suresh Bonthala</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Venkata Suresh Bonthala</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Venkata+Suresh+Bonthala">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bonthala V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Bonthala author:V.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Mehanathan Muthamilarasan</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mehanathan Muthamilarasan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Mehanathan+Muthamilarasan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muthamilarasan M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Muthamilarasan author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Garima Pandey</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Garima Pandey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Garima+Pandey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pandey G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Pandey author:G.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Yusuf Khan</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yusuf Khan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yusuf+Khan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Khan Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Khan author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Manoj Prasad</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manoj Prasad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 11-26735160. Fax. +91 11-26741658. E-mail: <a href="mailto:manoj_prasad@nipgr.ac.in" target="_blank">manoj_prasad@nipgr.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Manoj+Prasad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Prasad M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Prasad author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 79–90, <a href='https://doi.org/10.1093/dnares/dsu039'>https://doi.org/10.1093/dnares/dsu039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 April 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="440782" href="/dnaresearch/article-pdf/22/1/79/1208942/dsu039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Chandra Bhan Yadav, Venkata Suresh Bonthala, Mehanathan Muthamilarasan, Garima Pandey, Yusuf Khan, Manoj Prasad;  Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 79–90, <a href='https://doi.org/10.1093/dnares/dsu039'>https://doi.org/10.1093/dnares/dsu039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/440782?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/440782?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/440782?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/440782?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/440782?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/440782?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/440782?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20development%20of%20transposable%20elements-based%20markers%20in%20foxtail%20millet%20and%20construction%20of%20an%20integrated%20database&amp;publicationDate=2014-11-26&amp;volumeNum=22&amp;issueNum=1&amp;author=Yadav%2C%20Chandra%20Bhan%3B%20Bonthala%2C%20Venkata%20Suresh&amp;startPage=79&amp;endPage=90&amp;contentId=10.1093%2Fdnares%2Fdsu039&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f79%2f440782" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6584123 id="6584123" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Transposable elements (TEs) are major components of plant genome and are reported to play significant roles in functional genome diversity and phenotypic variations. Several TEs are highly polymorphic for insert location in the genome and this facilitates development of TE-based markers for various genotyping purposes. Considering this, a genome-wide analysis was performed in the model plant foxtail millet. A total of 30,706 TEs were identified and classified as DNA transposons (24,386), full-length <em>Copia</em> type (1,038), partial or solo <em>Copia</em> type (10,118), full-length <em>Gypsy</em> type (1,570), partial or solo <em>Gypsy</em> type (23,293) and Long- and Short-Interspersed Nuclear Elements (3,659 and 53, respectively). Further, 20,278 TE-based markers were developed, namely Retrotransposon-Based Insertion Polymorphisms (4,801, ∼24%), Inter-Retrotransposon Amplified Polymorphisms (3,239, ∼16%), Repeat Junction Markers (4,451, ∼22%), Repeat Junction-Junction Markers (329, ∼2%), Insertion-Site-Based Polymorphisms (7,401, ∼36%) and Retrotransposon-Microsatellite Amplified Polymorphisms (57, 0.2%). A total of 134 Repeat Junction Markers were screened in 96 accessions of <em>Setaria italica</em> and 3 wild <em>Setaria</em> accessions of which 30 showed polymorphism. Moreover, an open access database for these developed resources was constructed (Foxtail millet Transposable Elements-based Marker Database; <a class="link link-uri" href="http://59.163.192.83/ltrdb/index.html">http://59.163.192.83/ltrdb/index.html</a>). Taken together, this study would serve as a valuable resource for large-scale genotyping applications in foxtail millet and related grass species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;foxtail millet (Setaria italica L.)&quot;">foxtail millet (<em>Setaria italica</em> L.)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transposable element-based markers&quot;">transposable element-based markers</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="retrotransposons">retrotransposons</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA transposons&quot;">DNA transposons</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="database">database</a></div>                            <h2 scrollto-destination=6584125 id="6584125" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Transposable elements (TEs) constitute a significant fraction of plant genomes and are considered to be one of the major forces driving genome evolution. Further, TEs are capable of changing its position in the genome through transposition and so they are called as ‘jumping genes’. Each transposition event generates new variability by creating mutations and altering the genome size of a cell. On the basis of their mode of replication and transposition, the TEs are categorized as Class I and Class II. The Class I includes retrotransposons that produce RNA intermediates which are copied into DNA and then inserted into new locations within the genome while Class II TEs are DNA transposons that move directly by a ‘cut and paste’ mechanism.<sup><a reveal-id="DSU039C1" data-open="DSU039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Class I elements are further categorized into two subclasses, namely (i) LTR retrotransposons, flanked by long terminal repeats (LTRs), and (2) non-LTR elements which comprise Long-Interspersed Nuclear Elements (LINEs) and Short-Interspersed Nuclear Elements (SINEs). Retrotransposons are the most abundant mobile elements found in plant genomes,<sup><a reveal-id="DSU039C2" data-open="DSU039C2" class="link link-ref link-reveal xref-bibr">2</a></sup> as the replicative mode of retroelement transposition enables the LTR retrotransposon to accrue high copy number. Indeed, in some grasses, LTR retrotransposons represent up to 90% of the genome.<sup><a reveal-id="DSU039C2" data-open="DSU039C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU039C3" data-open="DSU039C3" class="link link-ref link-reveal xref-bibr">3</a></sup> They constitute for &gt;50% of the maize genome,<sup><a reveal-id="DSU039C4" data-open="DSU039C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSU039C5" data-open="DSU039C5" class="link link-ref link-reveal xref-bibr">5</a></sup> 14% of the <em>Arabidopsis</em> genome<sup><a reveal-id="DSU039C6" data-open="DSU039C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and up to 90% of the wheat genome.<sup><a reveal-id="DSU039C7" data-open="DSU039C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Similar to other plants, grass genomes are also rich in repetitive elements derived from retrotransposons which get amplified themselves in the genome through an RNA-mediated retrotransposition process.</p><p>The wide distribution of TEs in the plant genome, abundance and their variable arrangement pattern among closely related species facilitates their use as informative marker to assess genetic diversity in plant breeding programmes. TEs-based marker system takes advantage of their transpositional activity by which they cause insertions and hence variations; and also, the presence of conserved domains facilitates designing of PCR primers. So far, five classes of TE junction-based markers have been developed which include Repeat Junction Markers (RJMs), Repeat Junction-Junction Markers (RJJMs), Insertion Site-Based Polymorphism (ISBP), Inter-Retrotransposon Amplified Polymorphism (IRAP) and Retrotransposon-Based Insertion Polymorphism (RBIP).<sup><a reveal-id="DSU039C8" data-open="DSU039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Among these, RJMs are unique in the sense that cover both TE and gene region and hence can be useful in functional genomic studies. Development and utilization of very few insertional polymorphism-based markers were demonstrated in grass species. Wanjugi <em>et al.</em><sup><a reveal-id="DSU039C9" data-open="DSU039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> exploited the unique and abundant TE insertion junction regions identified from diploid <em>Aegilops tauschii</em> to develop genome-specific repeat DNA junction markers (RJMs) for use in hexaploid wheat. Identification of repeat junctions and large-scale development of TE-based marker was also successfully performed in barley.<sup><a reveal-id="DSU039C10" data-open="DSU039C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>Foxtail millet (<em>Setaria italica</em> L.) is a C<sub>4</sub> Panicoid grass with smaller genome (∼515 Mb), in-breeding and short life cycle.<sup><a reveal-id="DSU039C11" data-open="DSU039C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU039C12" data-open="DSU039C12" class="link link-ref link-reveal xref-bibr">12</a></sup> These attributes along with its genetic close-relatedness to other millets, cereals and several biofuel crops have made foxtail millet a model crop.<sup><a reveal-id="DSU039C13" data-open="DSU039C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSU039C14" data-open="DSU039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The release of draft genome sequence by BGI (Beijing Genomics Institute), China,<sup><a reveal-id="DSU039C15" data-open="DSU039C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and Joint Genome Institute (JGI) of the Department of Energy, USA,<sup><a reveal-id="DSU039C16" data-open="DSU039C16" class="link link-ref link-reveal xref-bibr">16</a></sup> had expedited the high-throughput analysis of genome and large-scale development of genomic resources such as simple sequence repeats (SSRs),<sup><a reveal-id="DSU039C17" data-open="DSU039C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU039C18" data-open="DSU039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> EST-derived SSRs<sup><a reveal-id="DSU039C19" data-open="DSU039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and intron length polymorphic markers (ILPs).<sup><a reveal-id="DSU039C20" data-open="DSU039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Considering the importance of foxtail millet, functional significance of TEs and the necessity of TE-based markers in genotyping applications, this study was performed to identify the different classes of TEs and develop molecular markers by utilizing the sequence information of TEs. Further, the developed resources are made available to the global research community through open access, web-based Foxtail millet Transposable Element-based Marker Database (FmTEMDb; <a class="link link-uri" href="http://59.163.192.83/ltrdb/index.html">http://59.163.192.83/ltrdb/index.html</a>).</p>                            <h2 scrollto-destination=6584129 id="6584129" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6584130 id="6584130" class="section-title" data-legacy-id=s2a>2.1. Identification of full-length retrotransposons and estimation of insertion time</h3>
<p>Genomic sequence of foxtail millet was retrieved from Phytozome (<a class="link link-uri" href="http://ftp://ftp.jgi-psf.org/pub/compgen/phytozome/v9.0/Sitalica/">ftp://ftp.jgi-psf.org/pub/compgen/phytozome/v9.0/Sitalica/</a>), and the full-length retrotransposons were predicted using LTR_FINDER tool (<a class="link link-uri" href="http://tlife.fudan.edu.cn/ltr_finder/">http://tlife.fudan.edu.cn/ltr_finder/</a>).<sup><a reveal-id="DSU039C21" data-open="DSU039C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The same tool was used to identify the Target Site Repeat (TSR), Primer Binding Site (PBS) and Polypurine Tract (PPT), Integrase [IN (core) and IN (c-term)] and RNaseH (RH) region for each predicted retrotransposon. 3′ and 5′ LTRs were identified based on their start and end (TG and CA, respectively) using in-house Perl script with the following parameters; LTR sequence length is 100–3,500 bp, and maximum distance between LTRs is 10,000 bp.</p><p>The 3′ and 5′ LTR sequences of the same Copia- and Gypsy-type retrotransposons were aligned by ClustalW<sup><a reveal-id="DSU039C22" data-open="DSU039C22" class="link link-ref link-reveal xref-bibr">22</a></sup> using default parameters, and the pairwise sequence divergence was calculated using the Ka/Ks calculator (<a class="link link-uri" href="https://code.google.com/p/kaks-calculator/wiki/KaKs_Calculator">https://code.google.com/p/kaks-calculator/wiki/KaKs_Calculator</a>). Based on NG parameter model,<sup><a reveal-id="DSU039C23" data-open="DSU039C23" class="link link-ref link-reveal xref-bibr">23</a></sup> the Ka and Ks, the numbers of synonymous (<em>S</em>) and non-synonymous (<em>N</em>) sites (<em>S</em> + <em>N</em> = <em>n</em>), and the numbers of synonymous (<em>S</em><sub>d</sub>) and non-synonymous (<em>N</em><sub>d</sub>) substitutions (<em>S</em><sub>d</sub> + <em>N</em><sub>d</sub> = <em>m</em>) were estimated. The time of insertion was calculated as described by Tamura <em>et al.</em><sup><a reveal-id="DSU039C24" data-open="DSU039C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=6584133 id="6584133" class="section-title" data-legacy-id=s2b>2.2. Identification of DNA transposons</h3>
<p>DNA transposons were identified by RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker">http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker</a>) with reference to repeat databases including Maize Transposable Element Database (maize TEDB), TIGR Gramineae Repeats v2.0, TIGR <em>Triticum</em> Repeats v3.0, TIGR <em>Oryza</em> Repeats v3.3, TIGR <em>Hordeum</em> Repeats v3.0, TIGR Sorghum Repeats v3.0 and Triticeae repeat (TREP) sequence database.<sup><a reveal-id="DSU039C8" data-open="DSU039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> BLAST search was performed in RepeatMasker using default parameters with ‘do not mask simple repeats or low-complex DNA’ option to avoid the regions of low complexity, such as simple tandem repeats, polypurines and AT-rich regions that can lead to spurious matches in database searches.</p>                            <h3 scrollto-destination=6584135 id="6584135" class="section-title" data-legacy-id=s2c>2.3. Insertion of transposons into intronic regions and functional annotation of genes interrupted with TEs</h3>
<p>The data of intronic regions were obtained from the gff file of <em>S. italica</em> available in Phytozome. The intronic sequences were further annotated for different classes of transposons, and nested TEs were predicted using RepeatMasker.<sup><a reveal-id="DSU039C8" data-open="DSU039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Further, the genes interrupted with retrotransposons were annotated with the BLASTX algorithm using Blast2GO<sup><a reveal-id="DSU039C25" data-open="DSU039C25" class="link link-ref link-reveal xref-bibr">25</a></sup> under expected value (<em>e</em>-value) of 1.0e−10 and minimal length cut-off value of 33 to exclude hits with minor local alignments. The Blast2GO annotation tool was used to assign the most probable Gene ontology (GO) terms to the genes, and the results were visualized by WEGO tool (Web Gene Ontology Annotation Plot).<sup><a reveal-id="DSU039C26" data-open="DSU039C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=6584137 id="6584137" class="section-title" data-legacy-id=s2d>2.4. Transcriptional activation of TEs in various tissues of foxtail millet</h3>
<p>To confirm the transcriptional activities of TEs in foxtail millet, the Illumina RNA-HiSeq data of four tissues, namely spica, stem, leaf and root were retrieved from European Nucleotide Archive [SRX128226 (spica); SRX128225 (stem); SRX128224 (leaf); SRX128223 (root)].<sup><a reveal-id="DSU039C27" data-open="DSU039C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The RNA-seq data were then filtered and mapped onto various classes of TEs using Bowtie 1.0.0 (<a class="link link-uri" href="http://bowtie-bio.sourceforge.net/index.shtml">http://bowtie-bio.sourceforge.net/index.shtml</a>). The mapped reads were analysed in all the four tissues of foxtail millet.</p>                            <h3 scrollto-destination=6584139 id="6584139" class="section-title" data-legacy-id=s2e>2.5. Primer designing, PCR amplification and detection of polymorphisms</h3>
<p>Unique repeat junctions in the foxtail millet TEs were identified by executing both BLASTN search and repeat junction finding algorithm of RJPrimers pipeline v1.0 (<a class="link link-uri" href="http://probes.pw.usda.gov/RJPrimers/">http://probes.pw.usda.gov/RJPrimers/</a>). The fully annotated repeat databases chosen for executing BLASTN search and identification of repeat junctions were maize TEDB (1313 sequences), TIGR Gramineae Repeats v2.0 (2,942 sequences), TIGR <em>Triticum</em> Repeats v3.0 (452 sequences), TIGR <em>Oryza</em> Repeats v3.3 (21,807 sequences), TIGR <em>Hordeum</em> Repeats v3.0 (630 sequences), TIGR Sorghum Repeats v3.0 (120 sequences) and TREP sequence database. <em>E</em>-value cut-off was set to 1e−50 for the top hit and 1e−5 was used as maximum for all the hits to reduce the detection of false-positive repeat junctions. Primer3 tool was used for designing primer pairs for the repeat junctions (<a class="link link-uri" href="http://bioinfo.ut.ee/primer3-0.4.0/">http://bioinfo.ut.ee/primer3-0.4.0/</a>).</p><p>The primers were further validated in 96 <em>S. italica</em> accessions and 3 wild accessions (one each from <em>Setaria viridis</em>, <em>Setaria sphacelata</em> and <em>Setaria verticillata</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Genomic DNA was extracted from the lyophilized tissue of young leaves as described by Pandey <em>et al.</em><sup><a reveal-id="DSU039C17" data-open="DSU039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The DNA was quantified using 0.8% agarose gel by comparing with λ-<em>Hin</em>dIII DNA (Fermentas) as marker. The PCR amplification reactions were performed in a 25 µl reaction volume containing 100 ng of genomic DNA, 1× Taq buffer, 2 mM of MgCl<sub>2</sub>, 0.2 mM dNTP mix (Promega), 0.5 mM each of the forward and reverse primers and four units of Taq polymerase (Biotools). The PCR reactions were performed in iCycler thermal cycler (Bio-Rad) and with one cycle of 3 min at 94°C, 34 cycles of 60 s at 94°C, 60 s at 60°C, 1.30 min at 72°C and a final extension of 10 min at 72°C. The PCR products were resolved on 2% agarose gel. The DNA bands were eluted from the gel using Real Genomics Hi Yield Gel/PCR Fragments Extraction Kit (Real Biotech Corporation) and cloned into pGEM<sup>®</sup>-T Easy vector (Promega) following the manufacturer's instructions. The recombinant plasmids were then transformed into <em>Escherichia coli</em> DH5α competent cells, and the plasmids were isolated from positive clones using <em>AccuPrep</em> Plasmid MiniPrep DNA Extraction Kit (Bioneer) following the manufacturer's protocol. The plasmids were sequenced in automated sequencer (3730xI DNA Analyzer, Applied Biosystems) using M13 forward and reverse primers. The sequence information was used to construct multiple sequence alignment using TARGeT-based multiple sequence aligner.<sup><a reveal-id="DSU039C28" data-open="DSU039C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=6584142 id="6584142" class="section-title" data-legacy-id=s2f>2.6. Phylogenetic and Bayesian model-based population structure analysis</h3>
<p>The marker profiles of 99 accessions of <em>Setaria</em> species were scored for the presence (1) or absence (0) of the amplicon and a binary matrix was generated. Co-migrating bands were assumed to be originated from the same genetic locus. Binary matrix was analysed using the DARwin software v5.0.158.<sup><a reveal-id="DSU039C29" data-open="DSU039C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Using pairwise similarity matrix of Jaccard's coefficient, the level of genetic diversity among the 99 accessions was calculated and an unweighted neighbour-joining (UNJ) tree was constructed with a bootstrap analysis of 1,000 replicates.<sup><a reveal-id="DSU039C30" data-open="DSU039C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>The existence of a structure was assessed using STRUCTURE 2.3.3 software,<sup><a reveal-id="DSU039C31" data-open="DSU039C31" class="link link-ref link-reveal xref-bibr">31</a></sup> based on Bayesian model-based cluster analysis. The method used 99 accessions of <em>Setaria</em> to infer the fraction of an individual accession's genetic ancestry that belongs to a population, for a given number of populations (<em>K</em>). The genotype of each individual accession is a function of the allele frequencies in the <em>K</em> populations (clusters) and the proportion of its genotype drawn from each of the <em>K</em> populations (q<sub>k</sub>). The ‘no admixture model’ was tested, as recommended for dominant loci and a permutation test using a Markov Chain Monte Carlo (MCMC) method was applied to examine the population structure. For each run, the burn-in time was 2,00,000, and the number of replications was 2,00,000.<sup><a reveal-id="DSU039C32" data-open="DSU039C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The MCMC chain was run six times, using a correlated allele frequency model (prior mean is 0.01, prior SD = 0.05 and Lambda set at 1.0 in the advance option of the STRUCTURE program). Since it was difficult to choose the ‘correct’ <em>K</em> from the Ln probability of data [Ln <em>P</em>(D)], the Δ<em>K</em> values were estimated as per the procedure suggested by Evanno <em>et al.</em><sup><a reveal-id="DSU039C33" data-open="DSU039C33" class="link link-ref link-reveal xref-bibr">33</a></sup> All the calculations pertaining to assignment of optimum <em>K</em> according to Evanno <em>et al.</em><sup><a reveal-id="DSU039C33" data-open="DSU039C33" class="link link-ref link-reveal xref-bibr">33</a></sup> were performed using Structure Harvester v0.9.94 software (<a class="link link-uri" href="http://taylor0.biology.ucla.edu/structureHarvester/">http://taylor0.biology.ucla.edu/structureHarvester/</a>). Maximum peak of ΔK was considered as true cluster number.</p>                            <h3 scrollto-destination=6584145 id="6584145" class="section-title" data-legacy-id=s2g>2.7. Comparative mapping of TEs of foxtail millet with related grass species</h3>
<p>The foxtail millet TE sequences were BLASTN searched against the genomes of sorghum (<em>Sorghum bicolor</em>), maize (<em>Zea mays</em>), rice (<em>Oryza sativa</em>) and <em>Brachypodium distachyon</em> in Phytozome using default parameters. The hits with &gt;80% similarity were taken, and the orthologues were confirmed by BLAST searching against the respective repeat databases. The comparative physical map was visualized using Circos v0.55 (<a class="link link-uri" href="http://circos.ca">http://circos.ca</a>).<sup><a reveal-id="DSU039C34" data-open="DSU039C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=6584147 id="6584147" class="section-title" data-legacy-id=s2h>2.8. Database construction</h3>
<p>To facilitate wider usage of these annotated TEs and the respective markers, a web-based open access database was constructed using open source softwares (Apache, PHP and MySQL). The user friendly web interface allows easy access of the TEs and TE-based marker information such as the sequences of forward and reverse primers, its respective length, melting temperature (°C) and the status of wet-lab validation. Further, the CMap feature has been integrated in the database, which enables the user to visualize the physical map of the TEs and TE-based marker (either chromosome-wise or primer types-wise). The CMap interface also allows the user to visualize the comparative map of TEs between foxtail millet chromosomes and chromosomes of sorghum, maize, rice and <em>Brachypodium</em>.</p>                            <h2 scrollto-destination=6584149 id="6584149" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6584151 id="6584151" class="section-title" data-legacy-id=s3a>3.1. Identification of Class I TEs in foxtail millet</h3>
<p>Class I TEs include retrotransposons that transpose via an RNA intermediate.<sup><a reveal-id="DSU039C35" data-open="DSU039C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Retrotransposons are divided into two major subclasses, namely LTR retrotransposons and the non-LTR retrotransposons, which differ in their structure and transposition cycle. LTRs are further classified into <em>Copia</em>-like and <em>Gypsy-</em>like, whereas non-LTRs are categorized as LINEs and SINEs. In foxtail millet, 2,608 intact full-length LTRs were predicted using LTR_FINDER tool. These LTRs were further analysed for the presence of coding regions such as ‘gag’ that encodes capsid-like protein, ‘pol’ encoding for protease, integrase and reverse transcriptase enzymes, and ‘env’ coding for envelope protein. In addition, the sequences from coding region of some retrotransposons were extracted using in-house Perl script and were confirmed with BLASTX analysis against the non-redundant database of NCBI.</p><p>Of the 2,608 LTRs, 1,038 were found to be full-length <em>Copia</em> type and 1,570 were full-length <em>Gypsy</em> type (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). In addition, partial or solo <em>Copia</em>-type (10,118) and <em>Gypsy</em>-type (23,293) retrotransposons were also identified (Fig. <a reveal-id="DSU039F1" data-open="DSU039F1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S4 and S5</a>). The length of <em>Copia</em> elements ranged from 1.4 to 23.9 kb with a mean of 7,708.36 bp, whereas the length of <em>Gypsy</em>-type LTRs varied from 1.8 to 25.9 kb with a mean of 11,776.3 bp. The full-length LTRs were defined by the presence of two LTRs (5′ and 3′; both starts and ends with TG and CA, respectively) flanking the coding regions, and PBS and PPT that vary depending on the TE family, ranging between 20 and 15 bp in length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Both the 3′ and 5′ LTR sequences of all LTR-type retrotransposons were extracted using in-house Perl script and analysed. The length of 3′ LTRs for <em>Copia</em>-type retrotransposons ranged from 0.1 to 3.4 kb with a mean length of 0.99 kb, whereas the length of <em>Gypsy</em>-type retrotransposons varied from 0.1 to 3.46 kb with a mean of 1.14 kb. Similarly, variations in length were also observed in 5′ LTRs of both <em>Copia</em>- and <em>Gypsy</em>-type retrotransposons. The length ranged from 0.1 to 3.4 kb with a mean length of 0.99 kb for <em>Copia</em> and 0.1 to 3.48 kb with a mean of 1.14 kb for <em>Gypsy</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). </p>                        <a id="6584158" scrollto-destination="6584158"></a>
<div id="DSU039F1" data-id="DSU039F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03901.png?Expires=1509238983&Signature=SuszuumU-RTmRGXu1YGY7wZFTXxAz9A~QaC-92kmZT7ejrL-IshpCIWKxLKrycC2vTXUe9GV1GuwLVygfTgyoA8PAg8jUF198dbruW8olPlX7xx21Y5DXXIIJDJUuYuI6Jkrgw8qpqScSJOjCb9RUmsF9OYN3ImXfaOhB6pB9Cft-KpJtyH7day7MktG2yyt~i536rAy4TzhqNSPmd4RNwtHHm0Zp7pWNZckFC0pnsyoLiJOx2x6MaqjuXW1pzAUBF6Ir0TB6TuVIa2BsFOSn7Vic~C7BG95LWToswMFQOmvbPsjPPGYhdw0Hwnx2inFOw81nkGVczt~CTdrxWad9A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of different classes of TEs across the nine chromosomes of foxtail millet." path-from-xml="dsu03901.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584158/dsu03901.tif" path-from-xml="dsu03901.tif" target="_blank">View large</a><a section="6584158" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03901.png?Expires=1509238983&Signature=gbiEimBRb2Veev8nNFmFGdg8Tuqo1ooVLTJUl8q4VQxeaGrp-pAZGsSv-hisR8U0edMP98~FKOGC0GuwR9B-nea7rPmKnPMh74LKJv9T10xOLMuPKLM1EiYAUCJL9dmM6X9edLHXldD-kiFJzEsFZtl-JBDninG66b~99JHm8S5DcRioIVDvS32YDat1vQHzvryjOkh3TJhG1Z5tJKEeKj2kYswOaniSaj7WU3tBcs9Y5uSdt1SSVv79ySmOjQoRu7FDEJQnxIa90ANIxzkGu19Uwd5RLrspChEnd2rjVcZkA-AAaGoOCaiitsOLkGAzvQyCl926ffzwRYvd-IyNXw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584158&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03901.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of different classes of TEs across the nine chromosomes of foxtail millet.</p></div></div><div content-id="DSU039F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03901.png?Expires=1509238983&Signature=SuszuumU-RTmRGXu1YGY7wZFTXxAz9A~QaC-92kmZT7ejrL-IshpCIWKxLKrycC2vTXUe9GV1GuwLVygfTgyoA8PAg8jUF198dbruW8olPlX7xx21Y5DXXIIJDJUuYuI6Jkrgw8qpqScSJOjCb9RUmsF9OYN3ImXfaOhB6pB9Cft-KpJtyH7day7MktG2yyt~i536rAy4TzhqNSPmd4RNwtHHm0Zp7pWNZckFC0pnsyoLiJOx2x6MaqjuXW1pzAUBF6Ir0TB6TuVIa2BsFOSn7Vic~C7BG95LWToswMFQOmvbPsjPPGYhdw0Hwnx2inFOw81nkGVczt~CTdrxWad9A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of different classes of TEs across the nine chromosomes of foxtail millet." path-from-xml="dsu03901.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584158/dsu03901.tif" path-from-xml="dsu03901.tif" target="_blank">View large</a><a section="6584158" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03901.png?Expires=1509238983&Signature=gbiEimBRb2Veev8nNFmFGdg8Tuqo1ooVLTJUl8q4VQxeaGrp-pAZGsSv-hisR8U0edMP98~FKOGC0GuwR9B-nea7rPmKnPMh74LKJv9T10xOLMuPKLM1EiYAUCJL9dmM6X9edLHXldD-kiFJzEsFZtl-JBDninG66b~99JHm8S5DcRioIVDvS32YDat1vQHzvryjOkh3TJhG1Z5tJKEeKj2kYswOaniSaj7WU3tBcs9Y5uSdt1SSVv79ySmOjQoRu7FDEJQnxIa90ANIxzkGu19Uwd5RLrspChEnd2rjVcZkA-AAaGoOCaiitsOLkGAzvQyCl926ffzwRYvd-IyNXw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584158&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03901.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of different classes of TEs across the nine chromosomes of foxtail millet.</p></div></div><p>The non-LTRs, distinguished from LTRs by the respective absence of LTRs, were categorized as LINEs and SINEs. A total of 3,653 LINEs and 53 SINEs were identified in foxtail millet genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>). Length of LINEs ranged from 0.1 to 14.0 kb with a mean length of 1.3 kb, while the length of SINEs varied from 0.1 to 1.8 kb with a mean length of 1.5 kb.</p>                            <h3 scrollto-destination=6584160 id="6584160" class="section-title" data-legacy-id=s3b>3.2. Chromosomal distribution of Class I TEs in foxtail millet</h3>
<p>All the four subclasses of Class I TEs were evidenced to be distributed in all the nine chromosomes of foxtail millet (Fig. <a reveal-id="DSU039F1" data-open="DSU039F1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). A maximum of <em>Copia</em>-type TEs are present in chromosome 8 (∼14%; ∼3.6 TEs per Mb) and minimum in chromosome 7 (6.5%; ∼2 TEs per Mb). Average distribution of <em>Copia</em>-type TE in foxtail millet genome is 2.6 per Mb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S4</a>). In case of <em>Gypsy</em>, a maximum of 199 elements were present in chromosome 4 (∼13%) and minimum in chromosome 7 (∼8%). Maximum density of <em>Gypsy</em>-type TEs was found in chromosome 6 (∼5 per Mb) and minimum was observed in chromosome 9 (∼3 per Mb) (Fig. <a reveal-id="DSU039F1" data-open="DSU039F1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S5</a>). Foxtail millet chromosome 9 comprised the maximum number of LINEs (532; 14.5%) and chromosome 2 had the minimum (233; ∼5%). The density of LINEs was maximum in chromosome 6 (10.5 per Mb) and minimum in chromosome 2 (∼5 per Mb) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). In case of SINEs, maximum of these elements were observed in chromosome 11 (∼21%) and minimum in chromosome 7 (∼6%) (Fig. <a reveal-id="DSU039F1" data-open="DSU039F1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>).</p>                            <h3 scrollto-destination=6584162 id="6584162" class="section-title" data-legacy-id=s3c>3.3. Estimation of LTR insertion time</h3>
<p>Insertion time of LTRs in the foxtail millet genome was calculated to predict the time taken for the occurrence of each transposition events, which facilitates the movement of LTRs from one position to another in the genome. For <em>Copia</em>-type elements, the distribution of the divergence time ranged from 0.000767 to 3.93273 MYA (million years ago) with a mean value of 1.28 MYA (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The distribution of the divergence time for <em>Gypsy</em>-type retrotransposons was estimated to be in a range of 0.000371–6.7687 MYA with a mean value of 0.82 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). These results are in agreement with the previous studies where the insertion timings are reported in a range of 0.00–6.00 MYA.<sup><a reveal-id="DSU039C36  DSU039C38" data-open="DSU039C36  DSU039C38" class="link link-ref link-reveal xref-bibr">36–38</a></sup> In rice, it was reported that 263 LTR-RTs (5%) have insertion dates &lt;14,000 yrs old, which is approximately the time of rice domestication.<sup><a reveal-id="DSU039C39  DSU039C41" data-open="DSU039C39  DSU039C41" class="link link-ref link-reveal xref-bibr">39–41</a></sup> Further, the Ka/Ks ratios estimated as &gt;1 signified that LTRs have underwent positive selection. This positive selection at the DNA level could have resulted from the ability of TE sequences to replicate faster than the host genome.</p>                            <h3 scrollto-destination=6584164 id="6584164" class="section-title" data-legacy-id=s3d>3.4. Identification of Class II type TEs and its distribution in foxtail millet genome</h3>
<p>A total of 24,386 DNA transposons belonging to Class II type TEs were identified in foxtail millet (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). The lengths of these TEs varied from 0.1 to 14.7 kb with a mean length of 0.78 kb. The 22,860 DNA transposons were further classified into 10 subclasses, namely DNA/CMC-EnSpm, DNA/En-Spm, DNA/hAT-Ac, DNA/hAT-Tag1, DNA/hAT-Tip100, DNA/MULE-MuDR, DNA/PIF-Harbinger, DNA/TcMar-Stowaway, RC/Helitron and DNA/Tourist based on similarity search with known TEs reported in other plant species. Of these, DNA/PIF-Harbinger accounts for the highest of Class II type TEs (12,758; ∼52%), followed by DNA/CMC-EnSpm (4,979; ∼20%). DNA/hAT-Tag1 (31) and DNA/Tourist (28) were found to be least in number, amounting for 0.1% of the Class II TEs identified in foxtail millet (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>).</p><p>Chromosomal distribution data of these 24,386 DNA transposons revealed that a maximum of 3,901 were present in chromosome 9 (16%) and minimum in chromosome 2 (1,366; ∼28%) (Fig. <a reveal-id="DSU039F1" data-open="DSU039F1" class="link link-reveal link-table xref-fig">1</a>). Higher density of Class II TEs was evidenced in chromosome 5 (∼71 per Mb) and lower in chromosome 2 (∼28 per Mb) with an average density of 61 DNA transposons per Mb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>).</p>                            <h3 scrollto-destination=6584167 id="6584167" class="section-title" data-legacy-id=s3e>3.5. Transcriptional activation of TEs in various tissues of foxtail millet</h3>
<p>Retrotransposons were found to be transcriptionally active in all the four tissues, namely leaf, root, spica and stem (Fig. <a reveal-id="DSU039F2" data-open="DSU039F2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S11–S15</a>). A maximum of <em>Copia</em>-type TEs (∼80%) matched with RNA-HiSeq reads which revealed that <em>Copia</em>-type TEs were prevalently expressed in tissues of foxtail millet. Approximately 49% of <em>Gypsy</em>-type retrotransposons matched with RNA-HiSeq reads. Similarly, ∼10% LINEs, ∼15% SINEs and ∼16% DNA transposons matched with the expressed reads (Fig. <a reveal-id="DSU039F2" data-open="DSU039F2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S11–S15</a>). </p>                        <a id="6584169" scrollto-destination="6584169"></a>
<div id="DSU039F2" data-id="DSU039F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03902.png?Expires=1509238983&Signature=cBq-ZKWPFg2-rTrcwCbwa6dO~YGjNjhmfKthfXxp~gzXZevNP4Q0pi4LyEdYr5K-o2LmzHlizfUG8yOm5qplUSsnmYOamd1R-1buikfgsiMsYAAFkgWhHoRfvG4boAg4qDbzgKG9HRYgls2hlr6w1twT~HEPM~m3wL2kkO6nPTatOfMz623dpjpWBmxWS6aYH5XaSqs8jAAfULjoLsDWWprqrkrWRr4ISHY6nCISRRlexpJfZTsuSA8s7M6WoVvfINTrVG-hBQPrjU-awP6CzILhV2WJ7A4noqh6O3hBeo50E6AbyvyMgVFzs-38zQGVWjG7Du-f2CvrJQu46sDMHQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="A comparative view of different classes of transposable elements in the intronic regions of Brachypodium, Rice, foxtail millet, Sorghum and maize genomes." path-from-xml="dsu03902.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584169/dsu03902.tif" path-from-xml="dsu03902.tif" target="_blank">View large</a><a section="6584169" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03902.png?Expires=1509238983&Signature=SKezjiA~6pO1hzHubts1iPyPXfnIQXSybIfMQj7sDvBUfsfDiDEOVsh6mH5L5TpZkDKaP2KX59CnocWqCiJA4zBWhAC77fM~jvb3uCGAtnP514aSVWjIUgWp3kuXYggkGSjXp0AujZqXSGBuiiXFlUokDfGiCbVHFFpew7shACV6Bg-TXv1B75FW147UdLZv0W1--skDAMpFBM6XgMZp~2biQrBRiUXueSrgJ6l30731-mrx5ik64~XN~OwO5nZGKUaVR16dYhocqt8ioqeP17EqC788WrkfMouAc61Owld~9726ZnEvJowP~v9tAUQFeRV2biXo1SRevsVAK3G-dg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584169&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03902.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A comparative view of different classes of transposable elements in the intronic regions of <em>Brachypodium</em>, Rice, foxtail millet, Sorghum and maize genomes.</p></div></div><div content-id="DSU039F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03902.png?Expires=1509238983&Signature=cBq-ZKWPFg2-rTrcwCbwa6dO~YGjNjhmfKthfXxp~gzXZevNP4Q0pi4LyEdYr5K-o2LmzHlizfUG8yOm5qplUSsnmYOamd1R-1buikfgsiMsYAAFkgWhHoRfvG4boAg4qDbzgKG9HRYgls2hlr6w1twT~HEPM~m3wL2kkO6nPTatOfMz623dpjpWBmxWS6aYH5XaSqs8jAAfULjoLsDWWprqrkrWRr4ISHY6nCISRRlexpJfZTsuSA8s7M6WoVvfINTrVG-hBQPrjU-awP6CzILhV2WJ7A4noqh6O3hBeo50E6AbyvyMgVFzs-38zQGVWjG7Du-f2CvrJQu46sDMHQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="A comparative view of different classes of transposable elements in the intronic regions of Brachypodium, Rice, foxtail millet, Sorghum and maize genomes." path-from-xml="dsu03902.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584169/dsu03902.tif" path-from-xml="dsu03902.tif" target="_blank">View large</a><a section="6584169" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03902.png?Expires=1509238983&Signature=SKezjiA~6pO1hzHubts1iPyPXfnIQXSybIfMQj7sDvBUfsfDiDEOVsh6mH5L5TpZkDKaP2KX59CnocWqCiJA4zBWhAC77fM~jvb3uCGAtnP514aSVWjIUgWp3kuXYggkGSjXp0AujZqXSGBuiiXFlUokDfGiCbVHFFpew7shACV6Bg-TXv1B75FW147UdLZv0W1--skDAMpFBM6XgMZp~2biQrBRiUXueSrgJ6l30731-mrx5ik64~XN~OwO5nZGKUaVR16dYhocqt8ioqeP17EqC788WrkfMouAc61Owld~9726ZnEvJowP~v9tAUQFeRV2biXo1SRevsVAK3G-dg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584169&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03902.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A comparative view of different classes of transposable elements in the intronic regions of <em>Brachypodium</em>, Rice, foxtail millet, Sorghum and maize genomes.</p></div></div>                            <h3 scrollto-destination=6584170 id="6584170" class="section-title" data-legacy-id=s3f>3.6. Development of TE-based markers</h3>
<p>All the 30,706 identified foxtail millet TEs were searched for potential repeat junctions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S3 and S4</a>). Based on repeat junction, six types of primers were designed, namely retrotransposon-based insertion polymorphism (RBIP), IRAP, RJM, RJJM, ISBP and retrotransposon-microsatellite amplified polymorphism (REMAP) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S16–S21</a>). Hence, a total of 20,278 primers were successfully designed, of which maximum was the ISBP (7,401; ∼36%) followed by RBIP (4,801; ∼24%). Only 57 primers could be designed for REMAP (∼0.2%). The numbers of IRAP, RJM and RJJM primers were 3,239 (∼16%), 4,451 (∼22%) and 329 (∼2%), respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S16–S21</a>). Although these markers were expected to include the fragments between the TE sequence on one side and the TE-inserted unique gene sequence on the other, some of the primers were evidenced to show same conformation at different locations in foxtail millet genome. This generation of duplications is due to the RJPrimer tool, using which the TE-based markers were developed.</p><p>Although primers could not be designed for LINEs and SINEs using Primer3 because of the limitations in their lengths, three RJM primers were manually designed for LINEs by aligning foxtail millet CDS on genomic sequences using GeneSeqer tool.<sup><a reveal-id="DSU039C42" data-open="DSU039C42" class="link link-ref link-reveal xref-bibr">42</a></sup> In addition, the TE-based forward and reverse primers were BLAST searched with available draft foxtail millet chromosomal pseudomolecule sequences to know their uniqueness/specificity in the foxtail genome. The results indicated that 1,522 (∼21%) of ISBP, 1,012 (∼21%) of RBIP, 712 (∼16%) of RJM, 546 (∼17%) of IRAP, 91 (∼28%) of RJJM and 8 (∼14%) of REMAP were unique.</p>                            <h3 scrollto-destination=6584173 id="6584173" class="section-title" data-legacy-id=s3g>3.7. Amplification and polymorphic potential of TE-based markers</h3>
<p>A total of 134 RJM primers were selected representing the nine chromosomes of foxtail millet for validation. Initially, all the 134 primer pairs were amplified in 96 accessions of <em>S. italica</em> and 3 wild <em>Setaria</em> (<em>S. viridis</em>, <em>S. sphacelata</em> and <em>S. verticillata</em>) accessions to examine the insertional polymorphism among these accessions (Fig. <a reveal-id="DSU039F3" data-open="DSU039F3" class="link link-reveal link-table xref-fig">3</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S22</a>). Of these, 104 (∼78%) amplified unique single allele, whereas 30 primers (∼22%) amplified more than single allele and was evidenced to be polymorphic. The amplicons showing polymorphism were sequenced and compared with the reference genome available in Phytozome to confirm whether all the sequences of 30 polymorphic primer pairs show complete similarity with amplified sequences (Fig. <a reveal-id="DSU039F4" data-open="DSU039F4" class="link link-reveal link-table xref-fig">4</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S6 and S7</a>). </p>                        <a id="6584175" scrollto-destination="6584175"></a>
<div id="DSU039F3" data-id="DSU039F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03903.png?Expires=1509238983&Signature=SKomSqvvpzounv0VirJP8ELR5XQP3BdgPVz77~Vo6SiIctu5wmb0589kbx3WYTWgBy9bzL22e2kWGi1So2c6guudWrDfeqQtbcYMJB-k0PTn4iojH79CM1pDVtblVrxooGsRX74zvlj5RDqY9U3pTQgdAg3YW6JesayTMjrTRB09JixRDYxQzI9FQMfAbSnaOPHtezLTcqkIUaxzIJVnpHCWRJx1FVUcaF6sl5HxPJz9D1GHTO81gGbpWqUqUmWcJYVf7V84jBOUnhI~RNCRPF~6hV8SHFuEfgYQntySGKmWO01foWC2em1Dqz~T8670UQ7GQvK5-dGSGo3MfuDfAw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The relative frequencies of Gene Ontology (GO) hits assigned to the GO functional categories; Biological Process, Molecular Function, and Cellular Component for Setaria italica genes which were interrupted with transposable elements. ‘DNA TEs’ indicates DNA transposons; ‘LTR’ represents Copia and Gypsy-type retrotransposons and ‘NonLTR’ indicates LINEs and SINEs." path-from-xml="dsu03903.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584175/dsu03903.tif" path-from-xml="dsu03903.tif" target="_blank">View large</a><a section="6584175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03903.png?Expires=1509238983&Signature=NDsZs9qI-D3RJw3Cj3~u1koE8JKvnGMkMq0eLSWX7~xcwOsun2DA29UGyf0sq5V0iHL~zCdLcAAUCJrSwRjZ-2gXkxoQqbEvQTA6Z3IiQbab4TolpKP4uES10q2~2fLqAPsc0qU6QMrEI~EV5HT2WZaD84q-TlYGsvBIZSNhEF47E5FJAMFhP~u3-6sj8MAIseHcpv73ZROa-EUFZOCvzp3YfpYnkbzexN9u5Uw8-E2WAzzY3CH7Ds6rRlAwf6RkWcMDenhBUJmrlVHmOSjdIsLBUcwFPswSfVcZ453oZhOUulvrTZY81AbBq62dxz5jNp2kzVEZjixSBTS5aBHrhg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584175&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03903.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relative frequencies of Gene Ontology (GO) hits assigned to the GO functional categories; Biological Process, Molecular Function, and Cellular Component for <em>Setaria italica</em> genes which were interrupted with transposable elements. ‘DNA TEs’ indicates DNA transposons; ‘LTR’ represents Copia and <em>Gypsy</em>-type retrotransposons and ‘NonLTR’ indicates LINEs and SINEs.</p></div></div><div content-id="DSU039F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03903.png?Expires=1509238983&Signature=SKomSqvvpzounv0VirJP8ELR5XQP3BdgPVz77~Vo6SiIctu5wmb0589kbx3WYTWgBy9bzL22e2kWGi1So2c6guudWrDfeqQtbcYMJB-k0PTn4iojH79CM1pDVtblVrxooGsRX74zvlj5RDqY9U3pTQgdAg3YW6JesayTMjrTRB09JixRDYxQzI9FQMfAbSnaOPHtezLTcqkIUaxzIJVnpHCWRJx1FVUcaF6sl5HxPJz9D1GHTO81gGbpWqUqUmWcJYVf7V84jBOUnhI~RNCRPF~6hV8SHFuEfgYQntySGKmWO01foWC2em1Dqz~T8670UQ7GQvK5-dGSGo3MfuDfAw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The relative frequencies of Gene Ontology (GO) hits assigned to the GO functional categories; Biological Process, Molecular Function, and Cellular Component for Setaria italica genes which were interrupted with transposable elements. ‘DNA TEs’ indicates DNA transposons; ‘LTR’ represents Copia and Gypsy-type retrotransposons and ‘NonLTR’ indicates LINEs and SINEs." path-from-xml="dsu03903.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584175/dsu03903.tif" path-from-xml="dsu03903.tif" target="_blank">View large</a><a section="6584175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03903.png?Expires=1509238983&Signature=NDsZs9qI-D3RJw3Cj3~u1koE8JKvnGMkMq0eLSWX7~xcwOsun2DA29UGyf0sq5V0iHL~zCdLcAAUCJrSwRjZ-2gXkxoQqbEvQTA6Z3IiQbab4TolpKP4uES10q2~2fLqAPsc0qU6QMrEI~EV5HT2WZaD84q-TlYGsvBIZSNhEF47E5FJAMFhP~u3-6sj8MAIseHcpv73ZROa-EUFZOCvzp3YfpYnkbzexN9u5Uw8-E2WAzzY3CH7Ds6rRlAwf6RkWcMDenhBUJmrlVHmOSjdIsLBUcwFPswSfVcZ453oZhOUulvrTZY81AbBq62dxz5jNp2kzVEZjixSBTS5aBHrhg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584175&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03903.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relative frequencies of Gene Ontology (GO) hits assigned to the GO functional categories; Biological Process, Molecular Function, and Cellular Component for <em>Setaria italica</em> genes which were interrupted with transposable elements. ‘DNA TEs’ indicates DNA transposons; ‘LTR’ represents Copia and <em>Gypsy</em>-type retrotransposons and ‘NonLTR’ indicates LINEs and SINEs.</p></div></div>                        <a id="6584176" scrollto-destination="6584176"></a>
<div id="DSU039F4" data-id="DSU039F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03904.png?Expires=1509238983&Signature=fgEw5UMcqNWyADcyrPUYRbdo9y5JeN6UA0~DHG29qRiuJ-ET1UCn5TGFk9w4SKuH6M0aD~MfgicyfvDB4V119jSzT-lBeHx66oTbnbM5803ikJt4r-60mxADD2c3B2KIEXPOVKtNvjPLJMvAAlED5saQJGIJ3-SlZCzj~Lqw95ZwcyWd3N5Y6Ch81MbQv0ktYsFnm1NPeDruQVhgBuBI3CuowCxnNhVLLBjEwr~JYKHYPzKwCsVH5cMjxM3RbLt6z2lgchbDAcKLAKLy-j81PO7xGm4nTJ5S5HOuhpnmp5xUy-~yZHH-VHi-~heU1ysd3DJVUumSyoQkY90qC9rD1A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Frequency of transcriptionally active transposable elements present in four tissues of foxtail millet." path-from-xml="dsu03904.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584176/dsu03904.tif" path-from-xml="dsu03904.tif" target="_blank">View large</a><a section="6584176" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03904.png?Expires=1509238983&Signature=AB7O19To6L8IV4EqsKfu2ZD3unRnl8E09pwHgeIrUDKY7DNfuEB80kK5ijXoJZFlUAhtFh4jLtXJ2J8TpKEIkIZ~SEc1H84iN3IorOQDrQrD7bwCj14sEoh3OuWDaY-ywCKsiYjAtXXM~yJAhqkGAPT2TAU5N~CMzw4zqtfQ5PFcRLBRKr--WDBAF3D3xQ0gm~Ets0~CWZYSgSVwkgdCJYZ7YoGcFQeyKzuYlu8CHAGRiRBfIq9EmM9WUOyM4vYoBIT5O8ExN9dmR5wHpJ7GyUwepXhFrLVEgy4DAs7powk1Ow1f23V9Zxzmiz-fxFvdnyHzdDvnF26E6HMhnvfXTA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584176&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03904.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency of transcriptionally active transposable elements present in four tissues of foxtail millet.</p></div></div><div content-id="DSU039F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03904.png?Expires=1509238983&Signature=fgEw5UMcqNWyADcyrPUYRbdo9y5JeN6UA0~DHG29qRiuJ-ET1UCn5TGFk9w4SKuH6M0aD~MfgicyfvDB4V119jSzT-lBeHx66oTbnbM5803ikJt4r-60mxADD2c3B2KIEXPOVKtNvjPLJMvAAlED5saQJGIJ3-SlZCzj~Lqw95ZwcyWd3N5Y6Ch81MbQv0ktYsFnm1NPeDruQVhgBuBI3CuowCxnNhVLLBjEwr~JYKHYPzKwCsVH5cMjxM3RbLt6z2lgchbDAcKLAKLy-j81PO7xGm4nTJ5S5HOuhpnmp5xUy-~yZHH-VHi-~heU1ysd3DJVUumSyoQkY90qC9rD1A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Frequency of transcriptionally active transposable elements present in four tissues of foxtail millet." path-from-xml="dsu03904.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584176/dsu03904.tif" path-from-xml="dsu03904.tif" target="_blank">View large</a><a section="6584176" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03904.png?Expires=1509238983&Signature=AB7O19To6L8IV4EqsKfu2ZD3unRnl8E09pwHgeIrUDKY7DNfuEB80kK5ijXoJZFlUAhtFh4jLtXJ2J8TpKEIkIZ~SEc1H84iN3IorOQDrQrD7bwCj14sEoh3OuWDaY-ywCKsiYjAtXXM~yJAhqkGAPT2TAU5N~CMzw4zqtfQ5PFcRLBRKr--WDBAF3D3xQ0gm~Ets0~CWZYSgSVwkgdCJYZ7YoGcFQeyKzuYlu8CHAGRiRBfIq9EmM9WUOyM4vYoBIT5O8ExN9dmR5wHpJ7GyUwepXhFrLVEgy4DAs7powk1Ow1f23V9Zxzmiz-fxFvdnyHzdDvnF26E6HMhnvfXTA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584176&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03904.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency of transcriptionally active transposable elements present in four tissues of foxtail millet.</p></div></div>                            <h3 scrollto-destination=6584177 id="6584177" class="section-title" data-legacy-id=s3h>3.8. Identification of TEs present in intronic regions and functional annotation of genes interrupted with TEs</h3>
<p>TEs are reported to play a crucial role in gene evolution by disrupting the genes. Upon getting integrated into the genome, TEs mediate chromosomal rearrangements, leading to accumulation of mutations and ultimately become transpositionally inactive. The presence of TEs was searched in the introns of 45,735 protein-coding genes of foxtail millet. Approximately 12% genes of foxtail millet were found to be integrated with TEs, of which ∼0.75% genes had <em>Copia</em>-type retrotransposons, ∼1% genes had <em>Gypsy</em> type, ∼3% genes had LINEs and ∼6% genes were interrupted with DNA transposons (Table <a reveal-id="DSU039TB1" data-open="DSU039TB1" class="link link-reveal link-table xref-fig">1</a>). LINEs (1,497 interrupted genes) and DNA transposons (830 genes with PIF-Harbinger and 635 genes with TcMar-Stowaway) were observed to be predominantly integrated within genic region. Similar phenomenon of TE insertion in intronic region was also observed in sorghum, maize, rice and <em>Brachypodium</em> (Fig. <a reveal-id="DSU039F5" data-open="DSU039F5" class="link link-reveal link-table xref-fig">5</a>; Table <a reveal-id="DSU039TB1" data-open="DSU039TB1" class="link link-reveal link-table xref-fig">1</a>)<em>.</em> Further, the patterns of the nested TEs (the insertion of TEs into pre-existing TEs) for all the types of retrotransposons were analysed. A total of 4,927 inserted TEs were found to be inserted within 2,078 host TEs, thus revealing that &gt;1 TE inserted into a single host TE. Of the total inserted TEs, 6.3% (308) was found to be inserted in RC/Helitron, 13.66% (673) in DNA/CMC-EnSpm, 11.26% (555) in DNA/En-Spm, 0.37% (18) in DNA/hAT-Ac, 0.30% (15) in DNA/MULE-MuDR, 0.41% (20) in DNA/TcMar-Stowaway, 3.75% (185) in DNA/PIF-Harbinger, 0.26%, (13) in DNA/Tourist, 0.52% (26) in LINE/L1, 26.91% (1,326) in Copia and 36.28% (1,788) in Gypsy (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). It has been reported that the movement of TEs in genomes results in the occurrence of nested TEs.<sup><a reveal-id="DSU039C43" data-open="DSU039C43" class="link link-ref link-reveal xref-bibr">43</a></sup> These nested TEs in foxtail millet genome may negatively influence genome expansion and enrich the diversity of gene expression or regulation. </p>                        <a id="6584179" scrollto-destination="6584179"></a>
<div class="table-wrap"><div id="DSU039TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Transposable elements interrupt the genes with clear signatures of insertions in intronic regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TE types </th><th>ch01 </th><th>ch02 </th><th>ch03 </th><th>ch04 </th><th>ch05 </th><th>ch06 </th><th>ch07 </th><th>ch08 </th><th>ch09 </th></tr></thead><tbody><tr><td>Copia </td><td>25 </td><td>38 </td><td>34 </td><td>37 </td><td>33 </td><td>31 </td><td>22 </td><td>27 </td><td>42 </td></tr><tr><td>Gypsy </td><td>62 </td><td>54 </td><td>70 </td><td>36 </td><td>63 </td><td>33 </td><td>43 </td><td>36 </td><td>77 </td></tr><tr><td>LINE </td><td>154 </td><td>199 </td><td>168 </td><td>138 </td><td>212 </td><td>122 </td><td>144 </td><td>76 </td><td>284 </td></tr><tr><td>SINE </td><td>4 </td><td>3 </td><td>1 </td><td>4 </td><td>3 </td><td>0 </td><td>0 </td><td>2 </td><td>3 </td></tr><tr><td>CMC-EnSpm </td><td>29 </td><td>26 </td><td>36 </td><td>16 </td><td>21 </td><td>16 </td><td>24 </td><td>28 </td><td>41 </td></tr><tr><td>En-Spm </td><td>0 </td><td>2 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>3 </td></tr><tr><td>hAT-Ac </td><td>6 </td><td>9 </td><td>5 </td><td>8 </td><td>10 </td><td>6 </td><td>11 </td><td>8 </td><td>11 </td></tr><tr><td>hAT-Tag1 </td><td>2 </td><td>6 </td><td>4 </td><td>2 </td><td>4 </td><td>8 </td><td>2 </td><td>6 </td><td>6 </td></tr><tr><td>hAT-Tip100 </td><td>8 </td><td>4 </td><td>6 </td><td>7 </td><td>7 </td><td>4 </td><td>3 </td><td>1 </td><td>3 </td></tr><tr><td>MULE-MuDR </td><td>30 </td><td>34 </td><td>31 </td><td>23 </td><td>44 </td><td>28 </td><td>29 </td><td>12 </td><td>32 </td></tr><tr><td>PIF-Harbinger </td><td>86 </td><td>103 </td><td>107 </td><td>55 </td><td>127 </td><td>52 </td><td>95 </td><td>42 </td><td>163 </td></tr><tr><td>TcMar-Stowaway </td><td>82 </td><td>89 </td><td>80 </td><td>42 </td><td>83 </td><td>27 </td><td>66 </td><td>42 </td><td>124 </td></tr><tr><td>RC/Helitron </td><td>59 </td><td>54 </td><td>62 </td><td>49 </td><td>62 </td><td>27 </td><td>45 </td><td>37 </td><td>89 </td></tr><tr><td>Tourist </td><td>3 </td><td>0 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>Total </td><td>305 </td><td>327 </td><td>333 </td><td>203 </td><td>359 </td><td>168 </td><td>276 </td><td>176 </td><td>473 </td></tr></tbody></table></div><div content-id="DSU039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>TE types </th><th>ch01 </th><th>ch02 </th><th>ch03 </th><th>ch04 </th><th>ch05 </th><th>ch06 </th><th>ch07 </th><th>ch08 </th><th>ch09 </th></tr></thead><tbody><tr><td>Copia </td><td>25 </td><td>38 </td><td>34 </td><td>37 </td><td>33 </td><td>31 </td><td>22 </td><td>27 </td><td>42 </td></tr><tr><td>Gypsy </td><td>62 </td><td>54 </td><td>70 </td><td>36 </td><td>63 </td><td>33 </td><td>43 </td><td>36 </td><td>77 </td></tr><tr><td>LINE </td><td>154 </td><td>199 </td><td>168 </td><td>138 </td><td>212 </td><td>122 </td><td>144 </td><td>76 </td><td>284 </td></tr><tr><td>SINE </td><td>4 </td><td>3 </td><td>1 </td><td>4 </td><td>3 </td><td>0 </td><td>0 </td><td>2 </td><td>3 </td></tr><tr><td>CMC-EnSpm </td><td>29 </td><td>26 </td><td>36 </td><td>16 </td><td>21 </td><td>16 </td><td>24 </td><td>28 </td><td>41 </td></tr><tr><td>En-Spm </td><td>0 </td><td>2 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>3 </td></tr><tr><td>hAT-Ac </td><td>6 </td><td>9 </td><td>5 </td><td>8 </td><td>10 </td><td>6 </td><td>11 </td><td>8 </td><td>11 </td></tr><tr><td>hAT-Tag1 </td><td>2 </td><td>6 </td><td>4 </td><td>2 </td><td>4 </td><td>8 </td><td>2 </td><td>6 </td><td>6 </td></tr><tr><td>hAT-Tip100 </td><td>8 </td><td>4 </td><td>6 </td><td>7 </td><td>7 </td><td>4 </td><td>3 </td><td>1 </td><td>3 </td></tr><tr><td>MULE-MuDR </td><td>30 </td><td>34 </td><td>31 </td><td>23 </td><td>44 </td><td>28 </td><td>29 </td><td>12 </td><td>32 </td></tr><tr><td>PIF-Harbinger </td><td>86 </td><td>103 </td><td>107 </td><td>55 </td><td>127 </td><td>52 </td><td>95 </td><td>42 </td><td>163 </td></tr><tr><td>TcMar-Stowaway </td><td>82 </td><td>89 </td><td>80 </td><td>42 </td><td>83 </td><td>27 </td><td>66 </td><td>42 </td><td>124 </td></tr><tr><td>RC/Helitron </td><td>59 </td><td>54 </td><td>62 </td><td>49 </td><td>62 </td><td>27 </td><td>45 </td><td>37 </td><td>89 </td></tr><tr><td>Tourist </td><td>3 </td><td>0 </td><td>1 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>Total </td><td>305 </td><td>327 </td><td>333 </td><td>203 </td><td>359 </td><td>168 </td><td>276 </td><td>176 </td><td>473 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6584179" target="_blank">View Large</a></div></div>                        <a id="6584180" scrollto-destination="6584180"></a>
<div id="DSU039F5" data-id="DSU039F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03905.png?Expires=1509238983&Signature=G9IKRRIThDdJDkOYreB-futI658DnsYJyrix3gXQTp0sA4vgT7znO1HA0~tNNiYYcYEJ2nA121N~NXbdO8QrMH4e0NAIx2ZUeh0AzasdRT7OZIJVykCetvuqP2CSe9jHb8tq3V9pCz1OySbgupNr~6stEXB2SuuTc6vMDIrVqt8aPExPL99Bq3HV20gs4bkBfz~gwSjp41oLD0qjGT~WzsomeHKyaGBM6UWL3Igktt938jXxL9xEY5BMRdHKQAEr-qY56VaodJbK99UDUI8w1zv5iCQgj9Rw3XWnawWnN3e~LtVV3D8jFECt-EqkpPESyTzmXfE2gXUi4qhwhlwbBw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="PCR amplification profile of Repeat Junction Marker ‘Solo_Gypsy_17547’. Lane M Marker; Lane 1: Foxtail millet cv. Prasad; Lane 2: cv. Lepakshi; Lane 3: IC403476; Lane 4: GS464; Lane 5: IC404178; Lane 6: IC403579; Lane 7: IC403476; Lane 8: IC403521; Lane 9: EC539248; Lane 10: EC539291; Lane 11: EC539300." path-from-xml="dsu03905.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584180/dsu03905.tif" path-from-xml="dsu03905.tif" target="_blank">View large</a><a section="6584180" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03905.png?Expires=1509238983&Signature=N2rUt4cZwv2jIRFum5hdB5A3jMcPgRlaHdFtkoLRvmkCq4SbP5ZkFkVtMDtIa3CHYw6lK2xuxwwcPoncHSKych~smSq8DauR-H4GLJ1oLMBxBcTymxlBCIgUQiqKeTVOuvtp3hitizGCdV2NTXht~9FJlVmPdMaRxN~MJeTUsXutQULVkZWZPve9IlZIW5hbJCQVrroCW2dnOa365eEGG1s3WmISd7OL9IQGRglMwUjytX9FRG9poDjGq1VyTeqfNgxAxAmXWXm69VS71OLnKFBbWn72-kbnyOTyU7j~zFwzqCKeGbcDND1hQg0Jx5NFasR-1G2lTECTT3JMiD-How__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584180&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03905.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCR amplification profile of Repeat Junction Marker ‘Solo_Gypsy_17547’. Lane M Marker; Lane 1: Foxtail millet cv. Prasad; Lane 2: cv. Lepakshi; Lane 3: IC403476; Lane 4: GS464; Lane 5: IC404178; Lane 6: IC403579; Lane 7: IC403476; Lane 8: IC403521; Lane 9: EC539248; Lane 10: EC539291; Lane 11: EC539300.</p></div></div><div content-id="DSU039F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03905.png?Expires=1509238983&Signature=G9IKRRIThDdJDkOYreB-futI658DnsYJyrix3gXQTp0sA4vgT7znO1HA0~tNNiYYcYEJ2nA121N~NXbdO8QrMH4e0NAIx2ZUeh0AzasdRT7OZIJVykCetvuqP2CSe9jHb8tq3V9pCz1OySbgupNr~6stEXB2SuuTc6vMDIrVqt8aPExPL99Bq3HV20gs4bkBfz~gwSjp41oLD0qjGT~WzsomeHKyaGBM6UWL3Igktt938jXxL9xEY5BMRdHKQAEr-qY56VaodJbK99UDUI8w1zv5iCQgj9Rw3XWnawWnN3e~LtVV3D8jFECt-EqkpPESyTzmXfE2gXUi4qhwhlwbBw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="PCR amplification profile of Repeat Junction Marker ‘Solo_Gypsy_17547’. Lane M Marker; Lane 1: Foxtail millet cv. Prasad; Lane 2: cv. Lepakshi; Lane 3: IC403476; Lane 4: GS464; Lane 5: IC404178; Lane 6: IC403579; Lane 7: IC403476; Lane 8: IC403521; Lane 9: EC539248; Lane 10: EC539291; Lane 11: EC539300." path-from-xml="dsu03905.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584180/dsu03905.tif" path-from-xml="dsu03905.tif" target="_blank">View large</a><a section="6584180" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03905.png?Expires=1509238983&Signature=N2rUt4cZwv2jIRFum5hdB5A3jMcPgRlaHdFtkoLRvmkCq4SbP5ZkFkVtMDtIa3CHYw6lK2xuxwwcPoncHSKych~smSq8DauR-H4GLJ1oLMBxBcTymxlBCIgUQiqKeTVOuvtp3hitizGCdV2NTXht~9FJlVmPdMaRxN~MJeTUsXutQULVkZWZPve9IlZIW5hbJCQVrroCW2dnOa365eEGG1s3WmISd7OL9IQGRglMwUjytX9FRG9poDjGq1VyTeqfNgxAxAmXWXm69VS71OLnKFBbWn72-kbnyOTyU7j~zFwzqCKeGbcDND1hQg0Jx5NFasR-1G2lTECTT3JMiD-How__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584180&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03905.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCR amplification profile of Repeat Junction Marker ‘Solo_Gypsy_17547’. Lane M Marker; Lane 1: Foxtail millet cv. Prasad; Lane 2: cv. Lepakshi; Lane 3: IC403476; Lane 4: GS464; Lane 5: IC404178; Lane 6: IC403579; Lane 7: IC403476; Lane 8: IC403521; Lane 9: EC539248; Lane 10: EC539291; Lane 11: EC539300.</p></div></div><p>Hence, considering the role of TEs in regulation of gene expression, functional annotation of the genes interrupted with TEs was performed. The analysis revealed that predominant genes were involved in organ development (Fig. <a reveal-id="DSU039F6" data-open="DSU039F6" class="link link-reveal link-table xref-fig">6</a>). Further chromatin regulatory genes containing SET domain gene family and AGO gene family which are the components of RNAi machinery were identified as the genes interrupted with TEs. Three genes namely RJM3, RJM4 and RJM7 that are interrupted with TEs were validated in <em>Setaria</em> species. Of these, RJM7 showed insertion of LINE-type transposons in 21st intron of EIF2ALPHA KINASE gene in <em>S. italica</em> as it acts as Eukaryotic translation initiation factor 2-alpha kinase (eif2-alpha kinase) in plants (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). However, there was no insertional signature observed in <em>S. verticillata</em>. </p>                        <a id="6584182" scrollto-destination="6584182"></a>
<div id="DSU039F6" data-id="DSU039F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03906.png?Expires=1509238983&Signature=Mvg85Qc7uGIuDYHNWxbrO5LoaoQloBlYd22mgEP4xCSFMJnTbuJ85LfNXHGSPe8vlQqoPvjcKPp3yk6myd3eBM9-~kGqPmZSappE-zq~XMJFOgcuzDueJnIZIlaafTF-gp-B6sTkLxfJ1DqliHWuqu6w7OSxymn1U8IqgXSSB44BywsOTkc1lS3exHM~qn30l-4~kL9QC4T-RQCYuaiSSLVZk-aoobJnj54WgcexIOrOcWwj-SiI~tR987yrXxNPgl~7FVF9sAo6F-FjWEtUxE9dkqpxF2PIuNSD1wrkVclGe7NTYrLWO-Y4BNUMqCVuh-arKIt-CZmNd2K42kJ9EQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Multiple sequence alignment of different accessions of foxtail millet obtained from Repeat Junction Marker of ‘DNA13398’. ‘Scaffold_8’ denotes the transposable element sequence retrieved from the reference genome of foxtail millet available in Phytozome." path-from-xml="dsu03906.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584182/dsu03906.tif" path-from-xml="dsu03906.tif" target="_blank">View large</a><a section="6584182" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03906.png?Expires=1509238983&Signature=c0GNf9sLGWrxPJYzy2ralqUmVB5PaQMPhPjkSh1pkKT61Fo26~QwxXCS3l~kXzB91DEgDjkMdzaJasA2Ly0mJ3ajHpys4TjuTMgi5w6Wg~FLTiQfAOk~zEfcdL3jHXItmWemByhdxDU0vOu2IUjPGpNvfFsxwfZTIG5KO6Y19Hh2pMXyh44EbYv446DdSH5hhLHPhUyqhOBeLtpTmR-nfzeyrzncxAa1Qkn6lBM9l5-i5gnKuidfSzCNRj9inV8ISb0F2Rg1uzgPi6KKHpzyjgiZ2tbq~30QzaglTZBtHN9etm-Lx8WjQZQfQaEcQuI-PabuBcLupg9Nb7FkpM4N9Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584182&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03906.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple sequence alignment of different accessions of foxtail millet obtained from Repeat Junction Marker of ‘DNA13398’. ‘Scaffold_8’ denotes the transposable element sequence retrieved from the reference genome of foxtail millet available in Phytozome.</p></div></div><div content-id="DSU039F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03906.png?Expires=1509238983&Signature=Mvg85Qc7uGIuDYHNWxbrO5LoaoQloBlYd22mgEP4xCSFMJnTbuJ85LfNXHGSPe8vlQqoPvjcKPp3yk6myd3eBM9-~kGqPmZSappE-zq~XMJFOgcuzDueJnIZIlaafTF-gp-B6sTkLxfJ1DqliHWuqu6w7OSxymn1U8IqgXSSB44BywsOTkc1lS3exHM~qn30l-4~kL9QC4T-RQCYuaiSSLVZk-aoobJnj54WgcexIOrOcWwj-SiI~tR987yrXxNPgl~7FVF9sAo6F-FjWEtUxE9dkqpxF2PIuNSD1wrkVclGe7NTYrLWO-Y4BNUMqCVuh-arKIt-CZmNd2K42kJ9EQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Multiple sequence alignment of different accessions of foxtail millet obtained from Repeat Junction Marker of ‘DNA13398’. ‘Scaffold_8’ denotes the transposable element sequence retrieved from the reference genome of foxtail millet available in Phytozome." path-from-xml="dsu03906.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584182/dsu03906.tif" path-from-xml="dsu03906.tif" target="_blank">View large</a><a section="6584182" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03906.png?Expires=1509238983&Signature=c0GNf9sLGWrxPJYzy2ralqUmVB5PaQMPhPjkSh1pkKT61Fo26~QwxXCS3l~kXzB91DEgDjkMdzaJasA2Ly0mJ3ajHpys4TjuTMgi5w6Wg~FLTiQfAOk~zEfcdL3jHXItmWemByhdxDU0vOu2IUjPGpNvfFsxwfZTIG5KO6Y19Hh2pMXyh44EbYv446DdSH5hhLHPhUyqhOBeLtpTmR-nfzeyrzncxAa1Qkn6lBM9l5-i5gnKuidfSzCNRj9inV8ISb0F2Rg1uzgPi6KKHpzyjgiZ2tbq~30QzaglTZBtHN9etm-Lx8WjQZQfQaEcQuI-PabuBcLupg9Nb7FkpM4N9Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584182&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03906.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple sequence alignment of different accessions of foxtail millet obtained from Repeat Junction Marker of ‘DNA13398’. ‘Scaffold_8’ denotes the transposable element sequence retrieved from the reference genome of foxtail millet available in Phytozome.</p></div></div>                            <h3 scrollto-destination=6584183 id="6584183" class="section-title" data-legacy-id=s3i>3.9. Phylogenetic and Bayesian model-based population structure analysis</h3>
<p>To analyse the diversity among diverse germplasm of 99 <em>Setaria</em> accessions<em>,</em> a dendogram was generated from a similarity matrix using NJ clustering method. Unrooted NJ method resolved them into four major subgroups. Group 1 contains the largest number of individuals that were mostly originated from USA, whereas Group 2 comprises the accessions from India. Groups 3 and 4 predominantly comprise accessions from USA and China, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>).</p><p>The TE-based marker data of 99 accessions of <em>Setaria</em> were also analyzed for the genetic structure. The software was run for the number of fixed subgroups (<em>K</em>) from 2 to 12, and more than three runs were performed for each <em>K</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S23</a>) and Δ<em>K</em> was estimated. Structure Harvester software based on Evanno method has delineated the optimum number of <em>K</em> as 4 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S24</a>). Similar to Hierarchical analysis, Bayesian model-based cluster analysis revealed that the 99 individuals were clustered into four groups, A–D (<em>K</em> = 4) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S11</a>). These individuals were further classified into the ones with ‘pure’ ancestry (where &gt;80% of their inferred ancestry was derived from only one of the clusters) and ‘mixed ancestry’ or ‘admixtures’ (where &gt;20% of inferred ancestry was derived from more than one cluster). Majority of the accessions (76) belonged to the ‘pure’ ancestry (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S25</a>). The remaining accessions were of ‘mixed’ ancestry. A comparison of the results from Bayesian model-based STRUCTURE analysis with the NJ-based tree revealed considerable congruence. Two out of the four STRUCTURE-based clusters matched with the specific groups of the NJ-based tree. However, Clusters 2 and 3 were the exceptions, where the accessions from NJ-based group were not contributed to correspondence group. By understanding the ISBP due to RJM, analysing the evolutionary aspects is possible using phylogenetic approach. These markers were also proved successful in analysing genetic diversity analysis and construction of physical and genetic linkage maps in wheat.<sup><a reveal-id="DSU039C9" data-open="DSU039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The major advantage of RJM is that it indicates the insertion polymorphism where different allelic states (the presence and absence of the transposon insertion) at a locus are revealed.<sup><a reveal-id="DSU039C44" data-open="DSU039C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Because of this unique advantage, RJMs are used in genetic, physical and radiation mapping studies.<sup><a reveal-id="DSU039C9" data-open="DSU039C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p>                            <h3 scrollto-destination=6584186 id="6584186" class="section-title" data-legacy-id=s3j>3.10. TE-based comparative mapping between foxtail millet and related grass species</h3>
<p>TE-based comparative orthologous relationships between 30,706 foxtail millet TEs and TEs of sorghum, maize, rice and <em>Brachypodium</em> were analysed (Fig. <a reveal-id="DSU039F7" data-open="DSU039F7" class="link link-reveal link-table xref-fig">7</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S26–S29</a>). Of the 30,706 TEs, 14,008 (∼46%) showed maximum synteny with sorghum, 12,485 (∼40%) with maize, 9,634 (∼31%) with rice and 1,313 (∼4%) with <em>Brachypodium</em>. The data revealed a decrease in the degree of synteny with respect to increase in the phylogenetic distance. Interestingly, TEs mapped in foxtail millet chromosome 9 showed highest synteny with all the four grass species [2,419 (∼17%) with sorghum, 2,103 (∼17%) with maize, 1,666 (∼17%) with rice and 205 (∼16%) with <em>Brachypodium</em>] (Fig. <a reveal-id="DSU039F7" data-open="DSU039F7" class="link link-reveal link-table xref-fig">7</a>). Similarly, TEs mapped in chromosome 2 of foxtail millet showed minimum synteny with all the four grasses [777 (∼5%) with sorghum, 715 (∼6%) with maize, 510 (∼5%) with rice and 69 (∼5%) with <em>Brachypodium</em>] (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S26–S29</a>). The wider genetic distances and low syntenic relationships among foxtail millet and other monocot genomes based on TE-based markers could be explained either through low conservation of TEs and/or species-specific transpositions. The independent evolutionary and divergence patterns of TEs have led to evolve unique transposition patterns in diverse crop lineages for generation of species-specific TEs resulting in their low conservation and synteny. This TE-based comparative mapping provides insights on the TEs in sorghum, maize, rice and <em>Brachypodium</em> and would enable map-based isolation and analysis of TEs in these grass species. </p>                        <a id="6584188" scrollto-destination="6584188"></a>
<div id="DSU039F7" data-id="DSU039F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03907.png?Expires=1509238983&Signature=BYgP2v-lhpz7V7-m57NHmsQwdvPKnHhiHZTcO5wPNKaZ6LbiaJxhc5ezuvGs7tBlwv7f5ii7D4HonUfSyvLgxyWGOo5bAFPcFkw5PfoF6hxr8-V~KiOLN3ZA7BlI91OIyra2H3KdOT8K15JLQM0NuL-X8zsy9M7c1i1jeGVopgsXivMjaH1PjVKFi8OoVBAyGShTC51gCLanFhA16GIPtpRnhO1IKFrgz1MXsL3p9Izpeh80O3RiWxJo0QYPhVdpeSyCQfb2bGfqDhlMoNO4~R0b-OL0hp3X7My7E7s1QLgEboTGQTm4v6LOM93cHTaVbqiH-cJuETi7EkI~m2hm7A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparative physical map of foxtail millet TEs with (A) sorghum, (B) maize, (C) rice and (D) Brachypodium." path-from-xml="dsu03907.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584188/dsu03907.tif" path-from-xml="dsu03907.tif" target="_blank">View large</a><a section="6584188" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03907.png?Expires=1509238983&Signature=cbn3QbuVE5tGmQTZoS~tUs2kRNKM6952gfsSARsl07WORefdjHNca-HYR81neLGLn1YdP1Nat77r0NsSLJEkGYzQ72KkRRsq-0DgkR2YBrWY~WQ-GdjCxs1IV0Wiv8MgBlW~6HDfP5VdcNzv0ioh~HlHzVWDh4W93AxiKHUg7cFBu~7LerhppMMqc2gTMeK4RZbnbpeh4Gu5nEbcDCzmdps08z1d2wuQ3rM2X1ztHflMJCHceQWMa6qXu5hU8HiqVa-Ubml~s~rqEehrROTuYMKljbiN3xM2iptx47doNaOrWp9sdfJdkAhkMH~RLe8BwPDs~vEcFOBcmtRGdYHzyg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584188&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03907.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative physical map of foxtail millet TEs with (A) sorghum, (B) maize, (C) rice and (D) <em>Brachypodium</em>.</p></div></div><div content-id="DSU039F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03907.png?Expires=1509238983&Signature=BYgP2v-lhpz7V7-m57NHmsQwdvPKnHhiHZTcO5wPNKaZ6LbiaJxhc5ezuvGs7tBlwv7f5ii7D4HonUfSyvLgxyWGOo5bAFPcFkw5PfoF6hxr8-V~KiOLN3ZA7BlI91OIyra2H3KdOT8K15JLQM0NuL-X8zsy9M7c1i1jeGVopgsXivMjaH1PjVKFi8OoVBAyGShTC51gCLanFhA16GIPtpRnhO1IKFrgz1MXsL3p9Izpeh80O3RiWxJo0QYPhVdpeSyCQfb2bGfqDhlMoNO4~R0b-OL0hp3X7My7E7s1QLgEboTGQTm4v6LOM93cHTaVbqiH-cJuETi7EkI~m2hm7A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Comparative physical map of foxtail millet TEs with (A) sorghum, (B) maize, (C) rice and (D) Brachypodium." path-from-xml="dsu03907.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584188/dsu03907.tif" path-from-xml="dsu03907.tif" target="_blank">View large</a><a section="6584188" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03907.png?Expires=1509238983&Signature=cbn3QbuVE5tGmQTZoS~tUs2kRNKM6952gfsSARsl07WORefdjHNca-HYR81neLGLn1YdP1Nat77r0NsSLJEkGYzQ72KkRRsq-0DgkR2YBrWY~WQ-GdjCxs1IV0Wiv8MgBlW~6HDfP5VdcNzv0ioh~HlHzVWDh4W93AxiKHUg7cFBu~7LerhppMMqc2gTMeK4RZbnbpeh4Gu5nEbcDCzmdps08z1d2wuQ3rM2X1ztHflMJCHceQWMa6qXu5hU8HiqVa-Ubml~s~rqEehrROTuYMKljbiN3xM2iptx47doNaOrWp9sdfJdkAhkMH~RLe8BwPDs~vEcFOBcmtRGdYHzyg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584188&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03907.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative physical map of foxtail millet TEs with (A) sorghum, (B) maize, (C) rice and (D) <em>Brachypodium</em>.</p></div></div>                            <h3 scrollto-destination=6584189 id="6584189" class="section-title" data-legacy-id=s3k>3.11. Online web-resource implementation and user interface</h3>
<p>Using the three-level schema of Apache, PHP and MySQL, open access Foxtail millet Transposable Element-based Marker Database (FmTEMDb; <a class="link link-uri" href="http://59.163.192.83/ltrdb/index.html">http://59.163.192.83/ltrdb/index.html</a>) was constructed (Fig. <a reveal-id="DSU039F8" data-open="DSU039F8" class="link link-reveal link-table xref-fig">8</a>). The database is a repository of complete TE data along with respective marker information. The details of TEs can be searched using six different search criteria such as TE IDs, type of TE, chromosome, etc. For each TE, the database will provide preliminary information of TE ID, chromosomal location, orientation of the coding strand, subclass (if any) and hyperlinks to retrieve the primers and view the physical map. Under primer details, the database will display the primer type, junction, TE type, TE source, strand orientation, start and end position of primers, melting temperature and GC percentage. The CMap feature of FmTEMDb allows the interactive visualization of physical and comparative map of TEs (Fig. <a reveal-id="DSU039F9" data-open="DSU039F9" class="link link-reveal link-table xref-fig">9</a>). The map could be browsed either by type of TE or chromosome-wise. Further, all the data stored in the database are available for download. Although the database is user friendly, a tutorial is also provided (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S12</a>). </p>                        <a id="6584191" scrollto-destination="6584191"></a>
<div id="DSU039F8" data-id="DSU039F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03908.png?Expires=1509238983&Signature=ICzECaq~ObD6o5JvoRFHd0i3R1jGXaavsw9JPaTGLEDLV7fyjylpq~iDHVTsgM3nHnApLNXa33vdZwzr3vwWO0gIN8kz69WStGYoJ7eVsjgSXGGcUYKvjrn4uye4Ueff2UG8fBxt7ns9HqGBmNkcYzyFkMwlFg5b5Ks4pdiKOwnQbLuYOS~OI4UQhTg2aVJASr36y8sK7amOo5oSa2DdXMI4DKzxeLlzfBzvJSV-ebr5A218N97P1iLIV40rqjOn4TLvV2dtIO7ydZqlfQx0xZ-7x4wbzSE7XC~J~SObmSEnBbwtIYDibS9IAY8AmYCfN5gDaWJVM6uKqeLWuCfjSA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Screenshots of foxtail millet Transposable Elements-based Marker Database. (A) Home page, (B) the details of a Copia-type retrotransposon displayed and (C) details of primers present in a Copia-type retrotransposon." path-from-xml="dsu03908.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584191/dsu03908.tif" path-from-xml="dsu03908.tif" target="_blank">View large</a><a section="6584191" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03908.png?Expires=1509238983&Signature=H9OnEB5XJrX2F9IZNRp5LC9SSv54fgaVWo2r~nA4GvoCGHq3Akg2DiT2H7gAE8xU6WoxMeKK8KvAwBPRaJd7VYQ7CU4K4nkpkU3JBSZr6RuwvFPWuJj1U6rA4G9ExORIdy-ilmf5GeraEoo06BNfPo~Prjbh2~Vb-6~iXzekWsBwiQ5UB01HR7MhuDC~w5J3E-CwLdXVcxjsyMdkk3Ay5kW6O~jC5KiACPKzLI0mc9JDzMWXg424nNnuklvLHqqO75AGuOJkp0iz80nmp592jCQgf8AvMIef5JbY2hd8JJNMGgd03dx5W3aKotVqf9kNSpCwqR9U5uMX8P2sVaLJhQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584191&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03908.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshots of foxtail millet Transposable Elements-based Marker Database. (A) Home page, (B) the details of a <em>Copia</em>-type retrotransposon displayed and (C) details of primers present in a <em>Copia</em>-type retrotransposon.</p></div></div><div content-id="DSU039F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03908.png?Expires=1509238983&Signature=ICzECaq~ObD6o5JvoRFHd0i3R1jGXaavsw9JPaTGLEDLV7fyjylpq~iDHVTsgM3nHnApLNXa33vdZwzr3vwWO0gIN8kz69WStGYoJ7eVsjgSXGGcUYKvjrn4uye4Ueff2UG8fBxt7ns9HqGBmNkcYzyFkMwlFg5b5Ks4pdiKOwnQbLuYOS~OI4UQhTg2aVJASr36y8sK7amOo5oSa2DdXMI4DKzxeLlzfBzvJSV-ebr5A218N97P1iLIV40rqjOn4TLvV2dtIO7ydZqlfQx0xZ-7x4wbzSE7XC~J~SObmSEnBbwtIYDibS9IAY8AmYCfN5gDaWJVM6uKqeLWuCfjSA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Screenshots of foxtail millet Transposable Elements-based Marker Database. (A) Home page, (B) the details of a Copia-type retrotransposon displayed and (C) details of primers present in a Copia-type retrotransposon." path-from-xml="dsu03908.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584191/dsu03908.tif" path-from-xml="dsu03908.tif" target="_blank">View large</a><a section="6584191" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03908.png?Expires=1509238983&Signature=H9OnEB5XJrX2F9IZNRp5LC9SSv54fgaVWo2r~nA4GvoCGHq3Akg2DiT2H7gAE8xU6WoxMeKK8KvAwBPRaJd7VYQ7CU4K4nkpkU3JBSZr6RuwvFPWuJj1U6rA4G9ExORIdy-ilmf5GeraEoo06BNfPo~Prjbh2~Vb-6~iXzekWsBwiQ5UB01HR7MhuDC~w5J3E-CwLdXVcxjsyMdkk3Ay5kW6O~jC5KiACPKzLI0mc9JDzMWXg424nNnuklvLHqqO75AGuOJkp0iz80nmp592jCQgf8AvMIef5JbY2hd8JJNMGgd03dx5W3aKotVqf9kNSpCwqR9U5uMX8P2sVaLJhQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584191&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03908.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshots of foxtail millet Transposable Elements-based Marker Database. (A) Home page, (B) the details of a <em>Copia</em>-type retrotransposon displayed and (C) details of primers present in a <em>Copia</em>-type retrotransposon.</p></div></div>                        <a id="6584192" scrollto-destination="6584192"></a>
<div id="DSU039F9" data-id="DSU039F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03909.png?Expires=1509238983&Signature=dXjncNf~w80llGJ83WhTbfTPsgwrK7f16x56vUlmfwE9qoXRwdBP-UcYGi1vJ~9b7MDENGyaNn4plTs-jH9vr4jXGW0RtbrOKNNLe~KwEIx00Mge3ECsD47t3PJ77vaVRYr9KBWf6fvhk1kpmlSJs89Ie5J4ixx~BqWnPRnaAebAikrDOcZYOyK1RoZZBr0vjecfKExOh894pcblScSIH7bbEohavx~L5qQ7QRAE2lEDnURl1bpgTpqPYdHg2-WVGRx4znXPLlSEi92Pah-a1CaktmUo8xMxzUBqGVqL92nQ4FO0JAZMRqA7TGqxAKkezs-KJKI-PAD-AYRS6xjeew__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Screenshots of CMap interface of foxtail millet Transposable Elements-based Marker Database. (A) Physical map of foxtail millet chromosome 1 showing all the transposable elements; (B) comparative mapping of transposable elements between chromosome 1 of foxtail millet and all the 10 chromosomes of sorghum. The blue lines indicate the syntenic partners." path-from-xml="dsu03909.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584192/dsu03909.tif" path-from-xml="dsu03909.tif" target="_blank">View large</a><a section="6584192" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03909.png?Expires=1509238983&Signature=ZpknjUb-8YwNaBvOGJSLWFiB5xaQdIzwBqxsIKODchJ15jtPh5iAPIaci9yfzMnYRaj6h-mD5wpy5KEfjfEvxPNdJCVGtSxuIL5LzYQWEoxtKM~l8TnIfkaCbBQh9zOJDhW5HnModv7GIvTpm3A5HffGMVDXkM-9vuocxkw7W08vk80gn90XfNX3BZnzwDFhh~ZLb~qHhZnTXWEyHSFo2qZy2SHOIAYp63VTE3ZlUw7wBQiCZKdZMb9fPWbXGlFZysS0WA1WEMbo6cif2y1HwnzYkSGXKbDVFy36iyRxErx4CYHAhKhAtAqDcTBZfEMED2iKiU~SrZ6nZ~kwkmfcUQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584192&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03909.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshots of CMap interface of foxtail millet Transposable Elements-based Marker Database. (A) Physical map of foxtail millet chromosome 1 showing all the transposable elements; (B) comparative mapping of transposable elements between chromosome 1 of foxtail millet and all the 10 chromosomes of sorghum. The blue lines indicate the syntenic partners.</p></div></div><div content-id="DSU039F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/m_dsu03909.png?Expires=1509238983&Signature=dXjncNf~w80llGJ83WhTbfTPsgwrK7f16x56vUlmfwE9qoXRwdBP-UcYGi1vJ~9b7MDENGyaNn4plTs-jH9vr4jXGW0RtbrOKNNLe~KwEIx00Mge3ECsD47t3PJ77vaVRYr9KBWf6fvhk1kpmlSJs89Ie5J4ixx~BqWnPRnaAebAikrDOcZYOyK1RoZZBr0vjecfKExOh894pcblScSIH7bbEohavx~L5qQ7QRAE2lEDnURl1bpgTpqPYdHg2-WVGRx4znXPLlSEi92Pah-a1CaktmUo8xMxzUBqGVqL92nQ4FO0JAZMRqA7TGqxAKkezs-KJKI-PAD-AYRS6xjeew__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Screenshots of CMap interface of foxtail millet Transposable Elements-based Marker Database. (A) Physical map of foxtail millet chromosome 1 showing all the transposable elements; (B) comparative mapping of transposable elements between chromosome 1 of foxtail millet and all the 10 chromosomes of sorghum. The blue lines indicate the syntenic partners." path-from-xml="dsu03909.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6584192/dsu03909.tif" path-from-xml="dsu03909.tif" target="_blank">View large</a><a section="6584192" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu03909.png?Expires=1509238983&Signature=ZpknjUb-8YwNaBvOGJSLWFiB5xaQdIzwBqxsIKODchJ15jtPh5iAPIaci9yfzMnYRaj6h-mD5wpy5KEfjfEvxPNdJCVGtSxuIL5LzYQWEoxtKM~l8TnIfkaCbBQh9zOJDhW5HnModv7GIvTpm3A5HffGMVDXkM-9vuocxkw7W08vk80gn90XfNX3BZnzwDFhh~ZLb~qHhZnTXWEyHSFo2qZy2SHOIAYp63VTE3ZlUw7wBQiCZKdZMb9fPWbXGlFZysS0WA1WEMbo6cif2y1HwnzYkSGXKbDVFy36iyRxErx4CYHAhKhAtAqDcTBZfEMED2iKiU~SrZ6nZ~kwkmfcUQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6584192&ar=440782&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03909.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshots of CMap interface of foxtail millet Transposable Elements-based Marker Database. (A) Physical map of foxtail millet chromosome 1 showing all the transposable elements; (B) comparative mapping of transposable elements between chromosome 1 of foxtail millet and all the 10 chromosomes of sorghum. The blue lines indicate the syntenic partners.</p></div></div>                            <h2 scrollto-destination=6584193 id="6584193" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>TEs are one of the major components of the plant genome, and they are reported to play a crucial role in functional genome diversity and phenotypic variations. Hence, analysing the organization of TEs in the genome would enable the researchers in dissecting the interplay between TEs and nearby gene expression which would enhance understanding the role of TEs in shaping the crop phenotypic diversity. The advent of next-generation sequencing and high-throughput sequence analysis platforms had facilitated the whole-genome sequencing and analysis of important crop plants. With the availability of genome sequence information, it is possible to investigate the organization of TEs in the genome. Further, the polymorphic potential of several types of TEs such as retrotransposons had encouraged the development of TE-based molecular markers that are useful for high-throughput genotyping applications.<sup><a reveal-id="DSU039C45" data-open="DSU039C45" class="link link-ref link-reveal xref-bibr">45</a></sup> The major cause of genome evolution is because of the TEs, which has generated genetic diversity upon which selection can act. Further, TE transposition is also an important factor for nucleotide-base mutation rate, and thus, TEs serve as potential agents of evolutionary changes. TEs also induce phenotypic changes associated with domestication or diversification of cultivated plants. It also causes gene disruption by creating insertion or deletion in exon or intron region, which could be the major force towards the differential expression and regulation of gene.</p><p>Foxtail millet is a model crop for studying the genetics and genomics of several millets, cereals and bioenergy grasses.<sup><a reveal-id="DSU039C13" data-open="DSU039C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSU039C14" data-open="DSU039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Hence, identifying the TEs, classifying and analysing its organization, and developing TE-based molecular markers in foxtail millet would serve as an important resource for millets, cereals and bioenergy genomics. Considering this, the present study was performed to identify a total of 30,706 TEs in foxtail millet and is classified into respective classes and subclasses. Further, the TEs present in intronic regions were identified, and functional annotation of respective genes was performed. Using the RNA-sequence data of four tissues, the transcriptional activation of TEs was analysed, and comparative physical mapping of foxtail millet TEs with sorghum, maize, rice and <em>Brachypodium</em> was performed. From 30,706 TEs, 20,278 markers were developed which belonged to six types. Of these, 134 RJMs were screened in 96 accessions of <em>S. italica</em> and 3 wild <em>Setaria</em> accessions of which 30 showed polymorphism. To provide the developed TE information to the global science community, a web-based, open access database (FmTEMDb; <a class="link link-uri" href="http://59.163.192.83/ltrdb/index.html">http://59.163.192.83/ltrdb/index.html</a>) was constructed. Promisingly, the TE data of foxtail millet along with the large-scale marker information reported in this study will be a valuable resource for foxtail millet genomic studies including genomic selection, fine mapping and phylogenetic analysis. Further, this would also assist in gaining new insights on the genome structure of this model crop as well as the potential of TEs in genetic variation studies.</p>                            <h2 scrollto-destination=6584196 id="6584196" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6584198 id="6584198" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The authors' work in this area was supported by the core grant of National Institute of Plant Genome Research (NIPGR), New Delhi, India. Funding to pay the Open Access publication charges for this article was provided by the National Institute of Plant Genome Research, New Delhi, India.</p>                            <h2 scrollto-destination=6584200 id="6584200" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Mr Mehanathan Muthamilarasan and Ms Garima Pandey acknowledge the award of Research Fellowships from University Grants Commission, New Delhi, India. The authors also thank Dr Swarup K. Parida, National Institute of Plant Genome Research, for critically reading the manuscript.</p><p><strong>Conflict of interest statement:</strong> None declared.</p>                            <h2 scrollto-destination=6584203 id="6584203" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsu039c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Docking</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Saade</div> <div class="given-names">F.E.</div></div>,  <div class="name"><div class="surname">Elliott</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Schoen</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Retrotransposon sequence variation in four asexual plant species</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">375</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposon%20sequence%20variation%20in%20four%20asexual%20plant%20species&amp;author=T.R.%20Docking&amp;author=F.E.%20Saade&amp;author=M.C.%20Elliott&amp;author=D.J.%20Schoen&amp;publication_year=2006&amp;journal=J.%20Mol.%20Evol.&amp;volume=62&amp;pages=375-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-004-0350-y" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-004-0350-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-004-0350-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16547645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Plant transposable elements: where genetics meets genomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">329</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transposable%20elements%3A%20where%20genetics%20meets%20genomics&amp;author=C.%20Feschotte&amp;author=N.%20Jiang&amp;author=S.R.%20Wessler&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=329-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg793" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Kellogg</div> <div class="given-names">E.A.</div></div>. </div><div class="article-title">Do plants have a one-way ticket to genomic obesity?</div>, <div class="source">Plant Cell</div> , <div class="year">1997</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1509</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20plants%20have%20a%20one-way%20ticket%20to%20genomic%20obesity%3F&amp;author=J.L.%20Bennetzen&amp;author=E.A.%20Kellogg&amp;publication_year=1997&amp;journal=Plant%20Cell&amp;volume=9&amp;pages=1509-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.9.9.1509" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.9.9.1509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.9.9.1509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12237393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">SanMiguel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Tikhonov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nakajima</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">The paleontology of intergene retrotransposons of maize</div>, <div class="source">Nat. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">43</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20paleontology%20of%20intergene%20retrotransposons%20of%20maize&amp;author=P.%20SanMiguel&amp;author=B.S.%20Gaut&amp;author=A.%20Tikhonov&amp;author=Y.%20Nakajima&amp;author=J.L.%20Bennetzen&amp;publication_year=1998&amp;journal=Nat.%20Genet.&amp;volume=20&amp;pages=43-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/1695" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F1695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F1695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9731528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyers</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Tingey</div> <div class="given-names">S.V.</div></div>,  <div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Abundance, distribution and transcriptional activity of repetitive elements in the maize genome</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1660</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%2C%20distribution%20and%20transcriptional%20activity%20of%20repetitive%20elements%20in%20the%20maize%20genome&amp;author=B.C.%20Meyers&amp;author=S.V.%20Tingey&amp;author=M.%20Morgante&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1660-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.188201" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.188201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.188201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="collab">Arabidopsis Genome Initiative</div><div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brenchley</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Spannagl</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pfeifer</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Analysis of the bread wheat genome using whole-genome shotgun sequencing</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">491</div> (pg. <div class="fpage">705</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20bread%20wheat%20genome%20using%20whole-genome%20shotgun%20sequencing&amp;author=R.%20Brenchley&amp;author=M.%20Spannagl&amp;author=M.%20Pfeifer&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=705-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11650" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">You</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Wanjugi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Huo</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">RJPrimers: unique transposable element insertion junction discovery and PCR primer design for marker development</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">W313</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RJPrimers%3A%20unique%20transposable%20element%20insertion%20junction%20discovery%20and%20PCR%20primer%20design%20for%20marker%20development&amp;author=F.M.%20You&amp;author=H.%20Wanjugi&amp;author=N.%20Huo&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=W313-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq425" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20497996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wanjugi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Coleman-Derr</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Huo</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Rapid development of PCR-based genome specific repetitive DNA junction markers in wheat</div>, <div class="source">Genome</div> , <div class="year">2009</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">576</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20development%20of%20PCR-based%20genome%20specific%20repetitive%20DNA%20junction%20markers%20in%20wheat&amp;author=H.%20Wanjugi&amp;author=D.%20Coleman-Derr&amp;author=N.%20Huo&amp;publication_year=2009&amp;journal=Genome&amp;volume=52&amp;pages=576-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-033" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19483776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mazaheri</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Kianian</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Mergoum</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Transposable element junctions in marker development and genomic characterization of barley</div>, <div class="source">Plant Genome</div> , <div class="year">2014</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20element%20junctions%20in%20marker%20development%20and%20genomic%20characterization%20of%20barley&amp;author=M.P.%20Mazaheri&amp;author=M.A.%20Kianian&amp;author=M.%20Mergoum&amp;publication_year=2014&amp;journal=Plant%20Genome&amp;volume=7&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2013.10.0036" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2013.10.0036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2013.10.0036"> </span></div> </div></div></div></div></div><div content-id="dsu039c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doust</div> <div class="given-names">A.N.</div></div>,  <div class="name"><div class="surname">Kellogg</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Foxtail millet: a sequence-driven grass model system</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">137</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxtail%20millet%3A%20a%20sequence-driven%20grass%20model%20system&amp;author=A.N.%20Doust&amp;author=E.A.%20Kellogg&amp;author=K.M.%20Devos&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=137-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.129627" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.129627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.129627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brutnell</div> <div class="given-names">T.P.</div></div>. </div><div class="article-title"><em>Setaria viridis</em> and <em>Setaria italica</em>, model genetic systems for Panicoid grasses</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">3031</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Setaria%20viridis%20and%20Setaria%20italica%2C%20model%20genetic%20systems%20for%20Panicoid%20grasses&amp;author=P.%20Li&amp;author=T.P.%20Brutnell&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot.&amp;volume=62&amp;pages=3031-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err096" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21459768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lata</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Foxtail millet: a model crop for genetic and genomic studies in bioenergy grasses</div>, <div class="source">Crit. Rev. Biotechnol.</div> , <div class="year">2013</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">328</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxtail%20millet%3A%20a%20model%20crop%20for%20genetic%20and%20genomic%20studies%20in%20bioenergy%20grasses&amp;author=C.%20Lata&amp;author=S.%20Gupta&amp;author=M.%20Prasad&amp;publication_year=2013&amp;journal=Crit.%20Rev.%20Biotechnol.&amp;volume=33&amp;pages=328-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3109/07388551.2012.716809" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3109%2F07388551.2012.716809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3109%2F07388551.2012.716809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22985089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muthamilarasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Advances in <em>Setaria</em> genomics for genetic improvement of cereals and bioenergy grasses</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2014</div><div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-014-2399-3" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1007/s00122-014-2399-3</a></div></div></div></div></div><div content-id="dsu039c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Quan</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Genome sequence of foxtail millet (<em>Setaria italica</em>) provides insights into grass evolution and biofuel potential</div>, <div class="source">Nature Biotech.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">549</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20foxtail%20millet%20(Setaria%20italica)%20provides%20insights%20into%20grass%20evolution%20and%20biofuel%20potential&amp;author=G.%20Zhang&amp;author=X.%20Liu&amp;author=Z.%20Quan&amp;publication_year=2012&amp;journal=Nature%20Biotech.&amp;volume=30&amp;pages=549-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2195" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2195"> </span></div> </div></div></div></div></div><div content-id="dsu039c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.S.H.</div></div>,  <div class="name"><div class="surname">Percifield</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Reference genome sequence of the model plant <em>Setaria</em></div>, <div class="source">Nature Biotech.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">555</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reference%20genome%20sequence%20of%20the%20model%20plant%20Setaria&amp;author=J.L.%20Bennetzen&amp;author=J.S.H.%20Wang&amp;author=R.%20Percifield&amp;publication_year=2012&amp;journal=Nature%20Biotech.&amp;volume=30&amp;pages=555-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2196" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2196"> </span></div> </div></div></div></div></div><div content-id="dsu039c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pandey</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Misra</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genome-wide development and use of microsatellite markers for large-scale genotyping applications in foxtail millet [<em>Setaria italica</em> (L.)]</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">197</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20development%20and%20use%20of%20microsatellite%20markers%20for%20large-scale%20genotyping%20applications%20in%20foxtail%20millet%20%5BSetaria%20italica%20(L.)%5D&amp;author=G.%20Pandey&amp;author=G.%20Misra&amp;author=K.%20Kumari&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=197-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">Development of highly polymorphic simple sequence repeat markers using genome-wide microsatellite variant analysis in Foxtail millet [<em>Setaria italica</em> (L.) P. Beauv.]</div>, <div class="source">BMC Genomics</div> , <div class="year">2014</div>, vol. <div class="volume">15</div> pg. <div class="fpage">78</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20highly%20polymorphic%20simple%20sequence%20repeat%20markers%20using%20genome-wide%20microsatellite%20variant%20analysis%20in%20Foxtail%20millet%20%5BSetaria%20italica%20(L.)%20P.%20Beauv.%5D&amp;author=S.%20Zhang&amp;author=C.%20Tang&amp;author=Q.%20Zhao&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-78" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-78" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-78"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24472631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muthamilarasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Misra</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Development of eSSR-markers in <em>Setaria italica</em> and their applicability in studying genetic diversity, cross-transferability and comparative mapping in millet and non-millet species</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e67742</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20eSSR-markers%20in%20Setaria%20italica%20and%20their%20applicability%20in%20studying%20genetic%20diversity%2C%20cross-transferability%20and%20comparative%20mapping%20in%20millet%20and%20non-millet%20species&amp;author=K.%20Kumari&amp;author=M.%20Muthamilarasan&amp;author=G.%20Misra&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e67742" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0067742" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0067742" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0067742"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23805325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muthamilarasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Venkata Suresh</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pandey</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Development of 5123 intron-length polymorphic markers for large-scale genotyping applications in foxtail millet</div>, <div class="source">DNA Res.</div> , <div class="year">2014</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">41</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%205123%20intron-length%20polymorphic%20markers%20for%20large-scale%20genotyping%20applications%20in%20foxtail%20millet&amp;author=M.%20Muthamilarasan&amp;author=B.%20Venkata%20Suresh&amp;author=G.%20Pandey&amp;author=K.%20Kumari&amp;author=S.K.%20Parida&amp;author=M.%20Prasad&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=41-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst039" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W265</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LTR_FINDER%3A%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons&amp;author=X.%20Zhao&amp;author=H.%20Wang&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W265-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm456" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm456" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm456"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chenna</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Multiple sequence alignment with the Clustal series of programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3497</div>-<div class="lpage">500</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20sequence%20alignment%20with%20the%20Clustal%20series%20of%20programs&amp;author=R.%20Chenna&amp;author=H.%20Sugawara&amp;author=T.%20Koike&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3497-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1986</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">418</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20methods%20for%20estimating%20the%20numbers%20of%20synonymous%20and%20nonsynonymous%20nucleotide%20substitutions&amp;author=M.%20Nei&amp;author=T.%20Gojobori&amp;publication_year=1986&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=3&amp;pages=418-26" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3444411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1596</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">WEGO: a web tool for plotting GO annotations</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W293</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WEGO%3A%20a%20web%20tool%20for%20plotting%20GO%20annotations&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W293-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl031" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cochrane</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Alako</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Amid</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Facing growth in the European Nucleotide Archive</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">D30</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Facing%20growth%20in%20the%20European%20Nucleotide%20Archive&amp;author=G.%20Cochrane&amp;author=B.%20Alako&amp;author=C.%20Amid&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D30-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1175" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23203883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Burnette</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">TARGeT: a web-based pipeline for retrieving and characterizing gene and transposable element families from genomic sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> pg. <div class="fpage">e78</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TARGeT%3A%20a%20web-based%20pipeline%20for%20retrieving%20and%20characterizing%20gene%20and%20transposable%20element%20families%20from%20genomic%20sequences&amp;author=Y.%20Han&amp;author=J.M.%20Burnette&amp;author=S.R.%20Wessler&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=e78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp295" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp295" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp295"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19429695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perrier</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Flori</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bonnot</div> <div class="given-names">F.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Hamon</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Seguin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Perrier</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Glaszmann</div> <div class="given-names">J.C.</div></div>. </div>, <div class="source">Genetic diversity of cultivated tropical plants</div> , <div class="year">2003</div><div class="publisher-loc">Montpellier, France</div><div class="publisher-name">Science Publishers</div>(pg. <div class="fpage">43</div>-<div class="lpage">76</div>) <div class="comment">Data analysis methods</div></div></div></div></div><div content-id="dsu039c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div>. </div><div class="article-title">BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">685</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;author=O.%20Gascuel&amp;publication_year=1997&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=14&amp;pages=685-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025808" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Falush</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Prithchard</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">1567</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Prithchard&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=1567-87" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Inference of population structure using multilocus genotype data</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">945</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evanno</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Regnaut</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Goudet</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2611</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;publication_year=2005&amp;journal=Mol.%20Ecol.&amp;volume=14&amp;pages=2611-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2005.14.issue-8" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2005.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2005.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Circos: an information aesthetic for comparative genomics</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1639</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finnegan</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Eukaryotic transposable elements and genome evolution</div>, <div class="source">Trends Genet.</div> , <div class="year">1989</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">103</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Eukaryotic%20transposable%20elements%20and%20genome%20evolution&amp;author=D.J.%20Finnegan&amp;publication_year=1989&amp;journal=Trends%20Genet.&amp;volume=5&amp;pages=103-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(89)90039-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(89)90039-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(89)90039-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2543105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">McCarthy</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Ganko</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">McDonald</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Evolutionary history of <em>Oryza sativa</em> LTR retrotransposons: a preliminary survey of the rice genome sequences</div>, <div class="source">BMC Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">18</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20history%20of%20Oryza%20sativa%20LTR%20retrotransposons%3A%20a%20preliminary%20survey%20of%20the%20rice%20genome%20sequences&amp;author=L.%20Gao&amp;author=E.M.%20McCarthy&amp;author=E.W.%20Ganko&amp;author=J.F.%20McDonald&amp;publication_year=2004&amp;journal=BMC%20Genomics&amp;volume=5&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-5-18" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-5-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-5-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15040813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Rapid recent growth and divergence of rice nuclear genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">12404</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20recent%20growth%20and%20divergence%20of%20rice%20nuclear%20genomes&amp;author=J.%20Ma&amp;author=J.L.%20Bennetzen&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=12404-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0403715101" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0403715101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0403715101"> </span></div> </div></div></div></div></div><div content-id="dsu039c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baucom</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Estill</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Lessbens-Mack</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Natural selection on gene function drives the evolution of LTR retrotransposon families in the rice genome</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">243</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20selection%20on%20gene%20function%20drives%20the%20evolution%20of%20LTR%20retrotransposon%20families%20in%20the%20rice%20genome&amp;author=R.S.%20Baucom&amp;author=J.C.%20Estill&amp;author=J.%20Lessbens-Mack&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=243-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.083360.108" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.083360.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.083360.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kovach</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Sweeney</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">New insights into the history of rice domestication</div>, <div class="source">Trends Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">578</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20insights%20into%20the%20history%20of%20rice%20domestication&amp;author=M.J.%20Kovach&amp;author=M.T.%20Sweeney&amp;author=S.R.%20McCouch&amp;publication_year=2007&amp;journal=Trends%20Genet.&amp;volume=23&amp;pages=578-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.08.012" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.08.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.08.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17963977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sang</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genetics and phylogenetics of rice domestication</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">533</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20and%20phylogenetics%20of%20rice%20domestication&amp;author=T.%20Sang&amp;author=S.%20Ge&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=17&amp;pages=533-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2007.09.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2007.09.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2007.09.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17988855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sweeney</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The complex history of the domestication of rice</div>, <div class="source">Ann. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">951</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complex%20history%20of%20the%20domestication%20of%20rice&amp;author=M.%20Sweeney&amp;author=S.%20McCouch&amp;publication_year=2007&amp;journal=Ann.%20Bot.&amp;volume=100&amp;pages=951-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcm128" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcm128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcm128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17617555" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1157</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20structure%20prediction%20from%20consensus%20spliced%20alignment%20of%20multiple%20ESTs%20matching%20the%20same%20genomic%20locus&amp;author=V.%20Brendel&amp;author=L.%20Xing&amp;author=W.%20Zhu&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=1157-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth058" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Characterization and functional annotation of nested transposable elements in eukaryotic genomes</div>, <div class="source">Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">222</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20functional%20annotation%20of%20nested%20transposable%20elements%20in%20eukaryotic%20genomes&amp;author=C.%20Gao&amp;author=M.%20Xiao&amp;author=X.%20Ren&amp;publication_year=2012&amp;journal=Genomics&amp;volume=100&amp;pages=222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2012.07.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2012.07.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2012.07.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22800764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flavell</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Knox</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Pearce</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Ellis</div> <div class="given-names">T.H.</div></div>. </div><div class="article-title">Retrotransposon-based insertion polymorphisms (RBIP) for high throughput marker analysis</div>, <div class="source">Plant J.</div> , <div class="year">1998</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">643</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposon-based%20insertion%20polymorphisms%20(RBIP)%20for%20high%20throughput%20marker%20analysis&amp;author=A.J.%20Flavell&amp;author=M.R.%20Knox&amp;author=S.R.%20Pearce&amp;author=T.H.%20Ellis&amp;publication_year=1998&amp;journal=Plant%20J.&amp;volume=16&amp;pages=643-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00334.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.1998.00334.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.1998.00334.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10036780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu039c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu039c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poczai</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Varga</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Laos</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Advances in plant gene-targeted and functional markers: a review</div>, <div class="source">Plant Methods</div> , <div class="year">2013</div>, vol. <div class="volume">9</div> pg. <div class="fpage">6</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20plant%20gene-targeted%20and%20functional%20markers%3A%20a%20review&amp;author=P.%20Poczai&amp;author=I.%20Varga&amp;author=M.%20Laos&amp;publication_year=2013&amp;journal=Plant%20Methods&amp;volume=9&amp;pages=6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1746-4811-9-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1746-4811-9-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1746-4811-9-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23406322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=440782&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table9.xls?Expires=1509238983&amp;Signature=IGG-79gup0~RHKBn4Ketxo0HUdPj~~ksZ~b2jZqUAA7fUcSo28uSJYDJ41iMJmkQiEeuCFoYIJklyvNB2wWFtyU0PCE01pHol1~-g6BKy2mpE1qmduA~a9j7OfS2VNa2dR9txCX1i1Rn00boMgniWwnDvfFQPYsiL2Np7ITg4EqTBv7Ut6TD5TDcLnhu1tcV5Oo2SoJqy4VJeQ~lHN9xYtzFLQGdxlsJF5ukFcZF87PkItUhBHbgEXQ-~zAxHAv5D2WhIRegHKyjtDAsuZEpocy0AeR7ljpAVpUhQlmfZYKdMs67Ecosi9ZEzMHS~XI2W73cssOlj1~sq3xovwvaNQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-30</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table8.xls?Expires=1509238983&amp;Signature=Fp9cimgfjRAqGCAwBU5FN~AgpTxX411yLgpKhHzGzzPeQ3gNZeXHWLxw~sKw3sMbfaJkUwIysFGUPq79sIT~P6zAGbY~mePsxlG5jajMZCy5Rw9K8CDZNJ-i4jJpYI5Ve438-uktY38kbj~ZvHaQJ37eDWrNbsaALDEaigFidZehJN9FL9wB5KuuhGeL0fz1c~oboRoDJogKDjjKljudo~FWnIZGDS2ye0~D4gOE8EBIipwDd6pUVCg9USliAkfkTy67w4E6rO0REYf2Im5D4W9csru0Dzreg34s5JyjY6Zr~BMId4WIRuhoK1EDZ2uXDp5mN9mdiilUu1mNBw7UbQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-29</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table7.xls?Expires=1509238983&amp;Signature=aNPGCmeysSxXgSztFsYnGgrvxrXYthiwen2kpmjBobiieA-AB9JLT3sHwDHBGR87kSm~KZUkjUaryi1ZjKbpClIx-jVPrviXfc-qNyUCifchhjHrEnr9sWZYDq~MDEbw2yTW5hsXWex3ZB6tXWpk~P0jcMDEr6rhJMRqnmP5C67C3yyq1amuymRZJiQD0-IWnehMAgZo468RoWAuRZVTCZZ5Sss~24xHUWWNIuFO1VkGa0gbAF6OY27oIiQEpC9Jve14ZrXIuoPZTWmFLkICyp4HfVUYDDoASqocFypcU0jvIPLO02KJ0KiOqGZsu6Ed362pGxCbOzhexr-xLcVRDg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-28</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table6.xls?Expires=1509238983&amp;Signature=SvCAk2YnhL7MvAh9oRC4DhMA1g-MVZEQqTfEyel5YcnBcr1br8GwNitixj-x8uknzHN7BnQtLmuSTC8hTv3KyR699ieRtuayFtyFGexM5y7TTXjtwfLoThly9gEmSXLi3dXcUKhH0QENPLUwCGC~iAsXpFjlE1X2EcXNB-gYaq8CI3a5bEUvtgTTpnbrosdsXG4QFQiXJ2chj9yKPsG3z4IRZ4C2nTmeL7YhWel1sWT4a4yWoPwJlXHAa4l2~a2sPB~Eo9PY86dkdmblm78Chc4PI-GR~BtdzBaBOU6FjtknwgMMRD1EsjwVK4wAgE0ji-7iW44AiP5VskzKX7Parw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-27</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table5.xls?Expires=1509238983&amp;Signature=S6iGYE~1q70ABfc9AT7nT-O7hqu3X-Y~BTU7Ny5bATg1SRkVSZd7Ygr6nfejBdoaqNERplh8VCs6SnGaPbalSSCtsR4MjUH31ALztQL68~dqFvGUWH8xrZU0wD2U65bZJvic59WdJZEuRcFOMXl0EoJkaLVbqXoLKpy11icoHgl2jnVpAtgnG6uO56wJaylPIU7bPDLlsgtpxrDz5GVMvtQadKQ04lRYTnMQRClAIOCt9nwi88UGfPFcrt9BjLm-l~MdGbVNleKmu94NDcxmIbWDnPZzdmdtnEeRLG0WsAVfUPBsre~32BO7BDJPuLCiQ~VXZeOkhcuyNvlB8FiOAg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-26</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table4.xls?Expires=1509238983&amp;Signature=AGRVWt0J5SzmpzHfWSZTFIbtlQ1LfAwIxqLvfAlLpV6u2P5iVVpQBEMEi8yNGVrJ0-gBwwhLc4i~-shubVTu2MkdwGZhOnMDPefkHW1tJUc0tALQsLpFYu4E2K6u5V1wJ1DtxWY6DZfNXGTkHkyxJYV4EgpwT-K61jHpISrm-NyrEU8zW9LYxIdAAdIt19WdhN4oe~yafgWDknK2vquZTPiwmeim2PD2RGzbgnInIkD3v-hYBdUVkx64m4Rtq8cHJ2YDRpBAyVpl~QlfEUocgpr7bhNf~OxhkxHot4Vq9pvzEtYZAF~PczFH-Q76j8-PQnVrfi0FQXhyis4VHQxI2Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-25</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table3.xls?Expires=1509238983&amp;Signature=D0NhSRy-L2BS16wy9TwWJXnjxCy2aRz-qg4d588yAfJQo8Hb1gJyTFdbwMd8NjwUAWX4TZFfYjLQIKiz5jH6qyqSSsnQ4ooqaWj-ctL5U04-Zqvx75MwV8KNjVXKRmcjZgy0iwwigBk8X6~N1jkbnzC9U9G6nuR1xV1PQGH2gPN-vL1SheL1MhKLSgTVEeeVewu34Ut0Y6odcfRvw0LpPPnZB6TUM9zbcSQRmAhYjKJ31UX2OGIB~Elh5KJivdruGXb1UB8fX9OwDM9DkLJEjDi6RYBsRSLG64UZkYFNqbbUyS~vualiTGxrd1cbtEPqqZnFZwTMGCwSTTc-UOtA0g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-24</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table29.xls?Expires=1509238983&amp;Signature=f-Q6uCAZQZ7gu3N6VAzT9ZXXtmFy0ozPlURqbs3Lo-xsx8NsU68T7Lw383ojFxA65P6zm~aXPt~Jzb5Ktqnb9rP4LKX~dtl3kAb5W4SDfctdswWzweKdF9ZbaR61ccbEyQDj0k9OXVJd7h3W79AsvzN9-HzfQpS6~F3g7IxO19dpvqsYpmDK82Ezg4CApFpBkdNpQNZsdvxwHY8bvVdaz-PGEXl7ZFWtmiF4UV2WJg0nYeanfLGPtuyITF0YX4~27lWom-LQ9Lk~8ZxpFVWTpGMil7scO9dy4rNQAkhXPdU8jlVyX-lF-2QwRhY~t9IWAmNTFDS7eOf9sMbVr98xhg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-23</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table28.xls?Expires=1509238983&amp;Signature=d4qkSiM0kerp648BIT8jdeuMwudC5BXJFZTOPQ5VwPQlyPdsJJDdXK34gv-8nvP4fOHMHZ5JH8xux1Lu4xnsQCzota3rEJUcYUMlp9odQsvEbUYX66Br94ZubNwMHr808Qcj4bT~U2JO-c2XiQ6301GCvyEtRHqHrArichrIiMRRfg2kbaSFmBf4e2GTIfY91YwPv076UP3K5YpCRUl4LDX5pV-kLQaq9dS1UfiEFfUzeqQTVhrdZBJ5~xd-5T~MHsx~AeNZeFVvPOjEPFiCTwOLDWLc1MDLoY3taSYgEpBVQjxtzhogDBiY9DtIuXhDZ6CVg5EM3wd6~R7UVBKE0g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-22</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table27.xls?Expires=1509238983&amp;Signature=Nqkd~MJalKfFBRl1p66XdVBq3YHMUwmaOZDaXHUzDAEgcJDqqjY7SqE6KytcWEr9uOqdKVdK5I8fTkFocqbaxRDtF~Ve~44RdHEtiAsrhppaFrwnu31VdckAi5dXNnKyBoB3YSedFicj1sPmOYaioypp969jpeG0V7UPypKLiSzhaMNDa8OdXIf-ojMKOTARFylR4drIiGv3ZMHFAlVmi1Gf3irSTOQoGn1BcOcnJhzLMa44UH9yTKxsqtQWg5--tRNWJuVAOjs-kXKX2czmdYDo3BaKygik0wjG1NXsK9syc-QsxK51YeA3e~dEb8jWHkwPj-xgFwLZG3zqGzefTw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-21</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table26.xls?Expires=1509238983&amp;Signature=bgQCBJLXjrveHAIGYDhM1l9cs8qaEvPk7oy6~ogyjTTtISqWFCdyYErwUdyZc5kcEWgGKro3OmdqPN7q69pyNQNNN4CXwLw-aWXMLeByAdCPbSi8rrE9p9s2yiK5kAH~jGAtbX9Ky3nj-f-DUzuWTFaRGS1HGKkAb8ORnhML9Dty6yKeA7Wyf0jJPHDHqTAGO0gNtAD948rW0I5jaOCaXsxuyLukeheXe~kHL1jEp72Oqev3d87xGQCTb-e6RJ51T1YbMR7KunyrBuabwHdTFLHjBwhehQGm0bEmI-T9G8qx1cPOJr0IjaDU2iXrP5oHbbhaz-Y9YZdYuiF3468ylg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-20</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table25.xls?Expires=1509238983&amp;Signature=WipX7bDayzFDMEdmkoCuHla~mwY4tUvW7yQLG1dNTEmjHzQiX~R7FYatfQO~ofIZpUC4y7SdubgeY4d9Rf4GjhoaKnjaGLRadxa~5HTFITtcg2L~vXcvXFDq6ZVK8hHy1r3k6BSAGucvSvpBoUEMH0jNBmylp0pcJ3ns~eqxMIpXl0Iv8ZKxoFeESYVfM0IuI-GGPXJIylNlex6VH~azCOlXnF1AXUoW0-yq7j3SKX-ba7crYkCG3J-YF31dPITn~XWHoWGsAo27wfTFS2XbOsgerRrVfVsQuC5VNJcPewP7yBDKhBUtzlDAHSmyehSDv5oKaAXl4jnwQGzl9zQaxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-19</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table24.doc?Expires=1509238983&amp;Signature=Zl0wG-VrgpJCYjBPLXbyJ2yE5zFAyo-KlMWxcd~rKtyzo4lalKtn8WLk82pG9O8ZbvRWLjiCz7reFaEtlfsPbDyzA0uI4tNuEfXC9-DU3NxhWeY50dk-Dgl6iT3c9H17Sv~TMkXqFijFPmuN6V4-k3CQLONYIlkPpaUekPqQGOdMyZQP21Ow9x~a4yBRHEkB9OxgzkynO93r1y5M7xWVkyONpZAtNXd7cwXZuuHf7Oi8FN0o~1MQEFxWVot2U8pxy3gJuzi~T2vrGImu01oSzFO6X0yhBz3kXUlGKTrfABMG15s28LnBNR97PUnUF-Am~1DHllglXNE9YyINgxiINA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-18</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table23.xls?Expires=1509238983&amp;Signature=ZEy18KlcDhdfPxjTWdTVRCOh0f7WC4kljWGkAM4nOP4u8iXpsNK9HkgiOddQ-jt~6HuC~B18~WOMRb-3kUURIlp6AyYdUBX0xYovQjaMqHIWUhTwZb2wAflDpS1zJgQXYS6Vi7xo6sqq3PGFLOIbogBP9r8LzRtnEr5GnK9XLGVJjTnijgxY-8F3hx7MRZCXjYZ4DRNDs9OPtL0bVDNYUVjcOog0d2pU85Gn47f3jMKnJZIWwh9-n3E1YRPVaz9n2Up1alFXqtHLasG6Xdb3uVbfwj6xgdaYg3m3BvNBfQ96H98IEdq4tn1o7tRm--epr-e00tpmUK5sKYnPInkwkw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-17</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table22.xls?Expires=1509238983&amp;Signature=IIo9LDGodMtfNiZPw116fz~lcN4gklcA7aHC1g1idognONu-Bc8Rvin2rxsi2y9A8QKodcEAud6YID-7pdlTOII96HTrCaGsUUqWuKXw3VRs~7IsCiTgjxGgJ7xqtu1YIqKKAaVVoUaU35XjIzc9tVTnRAj7LOIS-tEf0xNdIdM~oMbskOU2g-G6C1Dgy-RTnLwH~V7XU6kbHKJ3~Z00eBAqpjl11fLlmS05FTwvjLHTnfAKBBbmIm8UARtCKeQk1~CdYvf1gzmkZBBrKZpwJX1CP7DjNlJMyFM4ES9PVrIyiC25jgPTE~6VftKyfbQTMJVW5XmGdKD1JXt4NKQXsw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-16</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table21.xls?Expires=1509238983&amp;Signature=IsjdglRorphSny-2XrMx1Rs9cvTtrtInwEXIzJsnNKyTV~Pa36PjIARaS6t4-osOVQlrRvHLYxHPtREfAsm1xthT6qMr~T~7bRyMIzK2anrs4eGtp~vZxJGIAgBJW5OJiGeG4iLoFmQMlZsd3BKljv~Ba2O9Ncsu5CF4h6qTnd64WQn0Ob2hP87jecZMxB39FHuzQ6CdLIlCKa7tieeiyme9Bo7w6Go5f-z~mvnk3EV~kPSeYVopZccuhrV4m6HISevJttSNZ87nSpBCGO7sAVGE-uuU7bu7XhJq5G1XFujq7TlNUbYlsXvGeamcI-Ve9zVtnTbCfy9yNtHrPGIoBA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-15</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table20.xls?Expires=1509238983&amp;Signature=RyhsS3mnMnUD12U2aIf33RAPjTSIQabdtf7Y1gW3IgYnrxODQ24OuG2hGhipl6sQGsASH8cIcKCbcGJHb3vp4KJXJaFw790~0LPrg17jp0D-bXcTVVRMzY58QL9hjWErnx5R3xZnuY2ve-hxODfPu~AxU96lUu-ZBXst9UmIJKNSTyLcIkmCfV-Ek0YlTy3THzLMwtjdgt1qeRJcOYCoxhvPaa0OXv7a5UF9XsKsrb3zBP54mL7I8HBTtiBGjTxcb7ZUOLkV3EtP584RTrV0wOjOUdrWut0bkY-WEgTUNiFWiiMunwphWhXvy88ZRX8RuKhuao9moCob4RAF~vWgjg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-14</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table2.xls?Expires=1509238983&amp;Signature=H-Q9KjLb5VzxLpapjhiiYZwXNw-gqwbytEO5KXhhJos7zckulL7qsULPIBZPiuiUMiBylAeAuHa8bVeC5btMs2H95~R3U5HBEYKNXrY1I8YyhfqPHncLUuQjbRgLUSd~JuJz9F2fjXbUYvZ687A9axI-D1cwVFOMv7qVq--LzvXKrE03RGtujipv9mUVNDlk3-0HVrf6N6QXiM6kkRRFmTX500VLEwP6zXLBw16eLdETYyjXiwepFu7u2t0lXN6L5jI4zjWf-mE6Jjns2gqIBLSYvcCilnpqrKXfTfMiNFqYHgJQM08GyBK-psopCrlrzuvt6ER3zMq8YysyYR8GWg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table19.xls?Expires=1509238983&amp;Signature=WUOjz1MXgz-XriX7k0yEpyqRKfyJjC-PAsUpHI61P8wQlKS-hFOpjUJho6vR5~M1LtS5Lw7xvGQoKqdIWu2~zt-2WOpgScpR3kMp~cXB2X3aVeKwjtjItxL2CQ-SDo3beW6AmEaaACJpumqOGayebou~uflJjNYMhdWNDJA3YKHpbbsAR1aZAwDUa7p9zsEkYI~Sct1jnvApe0iQUpBb9FJtI42wzvvm6gpXVH5cogoIDn2d0Z-xCeeqDyUqp4NgHssPKvJTQ-FOGI20EnB0dXvG2Pt8-NZmdy2Wp0Q2G8cr9A5xspGvEAqJ2G9t8SjDOLK-gf875~UoQChEw7MNIA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-12</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table18.xls?Expires=1509238983&amp;Signature=IRr2xB4KyHxhKWv4B2EZ2jdPLLX8IS7rLorrTopLCFA73Ko6n3YdfLY~OlKlCz2wuFDFQ3gFFwmKJ~V5ZVRgR1vth6IALIZPQYbGifYMOIdVucy~1d9-2vaJhND85~EQqXZ6-rMs7k0MkIfh8PGJRvJNoFq-RlSwsSeoDbZGA27UHe5iMwmg-WZfjzNt2mD6iFpxl~BZmlChgsQMmfUeOaVgrPjGtcq4XiFGFgKjndVtjwK5A9Maxt-tn4zDwDOI9trym2yC8rZv~99mSJhK9eWjzISTjmP4~uuGcOgRRRNz6Gg9zaTS1fT1r8UJHx9h9rrOG40F2Rl--GrekKTYuQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table17.xls?Expires=1509238983&amp;Signature=dnDeD4GLePahiRy5qFBsgKVW7CqMju6ukn594yNPsuKf6Ml0775nwU7hboTPiBAv7lRBpRR2zD311gWTe~q1hV5DCLgbrtHnpeR1P0QAhO0jReUBKYVFUkjryzXsDSBGpzc58H1Koz8O16GqQRZJD17kUAuuJ5ObEQHnzwYWYTsqv8n4yXkf9oPOZApr3Dmqs4qv5cqPd9~BlpZgWbbMgMl2OX~tPVVT1h1QtybGeWnxuU5wRd4wNZrdq8Rscxix2T3BKOLbhYE0VLSqfYPR44rC6re3LWt-HNNlLzMk7Gw7AJ6pATaJtwws75PEwY~2n3JyNOeYNeYzY4EGe~tfrw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table16.xls?Expires=1509238983&amp;Signature=KkcuB8MCTbZmJ9HQfxhJlBArTHiHMWwwVV1dzgsQTvyLP4ANRAfOeZwoyZq8DusrEhVm5qhJjHrlNrtV8sirSxu39ieEwejCyGeCwsOkVhrcXj1iJQ9Fva86uetKvkqKPnH2bqlGClbRM6DbCvX2B1VEjz6Fi8cwt00nR8fIEqKe4beQlWttIS4Ekx7IUwIQozJatgFt3WZH6k2lwzbHzn6XbTnQjhe8V~8Lgkd4I7tN0eYyP1gMO4sOq9TgLZXVWcfZYBuUDZA1NtLIrp1eHAW2MHEYYaE5pyZn97WUK8oK2mP~CcDH-6vRa4fphYXoT7mA9lg3m0GGGfJeTRjTTg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table15.xls?Expires=1509238983&amp;Signature=aDXLIW-1fxVaR1GB4MFpqqMhVnagZBr88rcm~3zb~fQoBpKkDz9jJvLHYUyJ6P42KIExrIumOnEUkA8jOWwT94wNThup00afiNS8p4tE0fmcDQ3mcs2sLUQgkZB4z1Kg5CAcIWa~6UrUXHh7zKwIIYzVGHaDW77OxAxy-e5~91FxtHtps12u4D11kVh3V74mAgI4QHlGgggnjPQjWGUKXN6yk5F0lWI6475WEzFJA3OX5u7P1fHT26BshrQbBmza1jUaRVfC98X0Pp6Yg2uNPe05ZSZzMqpgR591Luw~~BoyRwymkxiuv8T9ZWpvuqhuCNSnbJ~KH-FlAM6B~H~cyA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table14.xls?Expires=1509238983&amp;Signature=CjuZonuoI-Gpg6igbCsfh54XLkgoND-AC3SPTFsAA14t-9-egkFzGTwjJ5gjXwK3KoMgQIfV27~Ge4R-sYgnDXl6QY5x5ZHpWfgv~ybwfuCiRDh-fyOO5D2DUlPc85iCKm~xW0eTzjzzUKn4AQrCtuAAvp7b6yiRbTZ4EpGEtLCMzHtzeSyItNYuVVFMFlNcD6xCoXGeQAxGeSX6wYt4YHlEZFg4eiaEhbaexjwEUf9onM1hmfcgcFhvwk0NTPnGlCzh9LX96w7BYiM5i14~f~6bo2VdzKMbTGYxUN8ZRBjr4iktHnNwjeOF9BzHvcZerEOqboFejRobbDTwYw4NaQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table13.xls?Expires=1509238983&amp;Signature=bSwzmXbBjC-d4tV-BSgaIijFqRUsSeowdMLQrfytgu2DDc4IZoVB8-5ka0Oa8wogOSYakZ4aZEvhC8V7Ow-b5cZRoNMPNBvBKdf4KTQb0aNMjXOQ8mqQtM7pQQkogM8YHhg0Z4nYYthIjv9RJDDAoPWzqm7O1PIbPHBoxFvUOR0nSg9Yc3PJUF93a8KlWXlMuNZPmRSqzUlJtxcHwzKzaQqqvQamoZUhznM81vTpB6j7FX9YcMZu1tOjIc~0dRyvKXIhSUspCkLEPZUGBsbzz~P8Ho8ju8WU016G73Qi2PON-zL05YMRN-48q-bW-2OTTa9kMUiyK~bYPjdj5latww__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table12.xls?Expires=1509238983&amp;Signature=LF~sU2RlZP7J~i1SpEQxeBMjRLMFpjnYMX-Q8Kf5BGxqP75~~8wQI4w20uYQV8FZdgPZFZ1wHCDQOY-9P7Ef2bNBW-wSMLjkVsaarMBWIZgAib50FlvYo8MKpHpL~7-ydIUs1LpaAXqEPJC-7lKfTs1tvejllsSgVKl2rZooS6GY1Lt5OZLAHK~MSmRTxc5bjMfb1g7UoVUoLGb5addc0gllAD2l63qVaX13-0zpk6zTjgCR9KbQ3Tj0mcAHYcGdUCtstLTquBmxQbIKy28nvhPUB7KwkZ-yCW-LhUge~M9PMADI33azPWBRSht6GoxHKZW9qeSIOVaclQ3X~S0Lxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table11.xls?Expires=1509238983&amp;Signature=aR4khDy1dFZIpekO-0duKM4Gn3O6zc04R2JAk16oiV5f9wk2x~SpT2DVZ~HlqCthCu~soVg3G2hHUxs1vE7FPo4GFEq9E~aSzuRpLi4fTtoHU6WJUim4hb6HbyN9KDBRtbMIsS9h-cuWC8QygdCUj-O8Y8xlVH7RQx4cJrcFEGVh2JSHINR53N1I7Bdo1Fx47BxvOF4FMlGKnJcvaQNbVjfHBXfI-sOi962A4taFgH2mUsBmGUrPu2~8j4AqKz9PYJ5qtr1MGxu5S2ZeCOOIqPMAb9mqIbR~ziDqFP4S56Z~~bwm0y1q7sQCB43Qz~bqmG9Btee-OW0H2YA0bWiviQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table10.xls?Expires=1509238983&amp;Signature=gYy2a889TeuECQZNxN9oyk~tQfyiYXAHG~wgUrZ~9-RZ~~c0bWzWVx4dboyBsn0FKxyBAKKDPjdpbV1ukwnRBKCVkCujZZYlZ745pHcnkaXDZk44FpJOfeGQ2~wDXzAJLIr6d2tkZv83oKkJb0IAEJ68nTcfBiz1Ly4~CJv7w2iLntT-SfdAcHtI4GXGWZ5P~Uvv~gXrPAaOPXiuSvJgvtjUXoLN~7jkRnyqQaazmfv7Z32wJGUk0xBaWIcgBk1HGgCDDLR3Z~UzRjxx0wqWWJ1WJkgMHoWQEwtqBVoZJshFXdQn7SIpU1JiSjLQ3qANmTFqY0tNziJ6mrlGxAGO6g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp_table1.xls?Expires=1509238983&amp;Signature=e0s2nIkdzEoxQSJzW57MqWat7Vo-4x3tJJbDp0JdRltMAWlcONl0DUWZ1~l9k6CfRhV40N-4lKZp5kKQa8NHVyDKh78VyXP4gmrITPamtEAOjqN8O~kMBebt-aAbGbwZ0GmdprEsGCmEEzpfTISKJch1b9-sBz0UNvBtzvasnEK0VQ2gapS0spj8c9Jx5bp7IrAMuGnoIge63cgqwgFAK6k0tcn1B9HE0eWXUO-DtVU6QxZkVPW46sB9MPHj9yvwP0jTmoSBomfnC4BExUzODBeEpevnGLIfFgq6uweA1iNbpqmhGkrFG8qJviha0R11G7KCxgRRLK9UkGS9CY7ESw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu039/2/dsu039-dsu039supp.pdf?Expires=1509238983&amp;Signature=S123XtB3RUm8kWVXhKI6M6p8kz0M3QRke46OAPmANEEUEcTsCZJpYbwPoHtdrCToDZAOqNCNbFXj1HH4zOZ6I3I0Qi9oXkmhUI4amLlRgXOf5WyYzz2aUzILYbm75E9wIrU5uCduGAdKC32ICdV9waJ5ZEuvvEHrwIYNm2iGUZ~oXCrtGxYfVdElnC0kkqs8wX2Kkd4ur5tEONij3zgWLP7JVPO~f8Yh~ywezRvVKyIihxaEAhzDTfiZA7PW6G8g2v3Q3hciQUfSweY4w4alMNOarorZFyqO5tRFK-Kv33LlIn33NzLiYTvgZ8klmPs48ps7aBiiy3rOeD0l9L0JAA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">150</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsu039"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="440782"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f79%2f440782">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F79%2F440782" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/440782'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide development of transposable elements-based markers in foxtail millet and construction of an integrated database" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f79%2f440782">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

