

<!DOCTYPE html>
<html lang="en">

<head><title>
	Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in Populus tomentosa† | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/53/441064","@type":"ScholarlyArticle","name":"Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in  Populus tomentosa   †  ","datePublished":"2014-11-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu040","keywords":["Chinese white poplar (<italic>Populus tomentosa</italic>)","epistasis","pathway-based multiple gene association","transcript profiling","validation population"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/53/441064","author":[{"name":"Du, Qingzhang","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Tian, Jiaxing","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Yang, Xiaohui","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Pan, Wei","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Xu, Baohua","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China","@type":"Person"},{"name":"Li, Bailian","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    3  Department of Forestry ,  North Carolina State University ,  Raleigh, NC 27695-8203 ,  USA","@type":"Person"},{"name":"Ingvarsson, Pär K.","affiliation":"4  Department of Ecology and Environmental Science ,  Umeå Plant Science Centre, Umeå University ,  Umeå SE-901 87 ,  Sweden","@type":"Person"},{"name":"Zhang, Deqiang","affiliation":"1  National Engineering Laboratory for Tree Breeding ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China    2  Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education ,  College of Biological Sciences and Technology, Beijing Forestry University ,  Beijing 100083 ,  P. R. China","@type":"Person"}],"description":"Economically important traits in many species generally show polygenic, quantitative inheritance. The components of genetic variation (additive, dominant and epistatic effects) of these traits conferred by multiple genes in shared biological pathways remain to be defined. Here, we investigated 11 full-length genes in cellulose biosynthesis, on 10 growth and wood-property traits, within a population of 460 unrelated Populus tomentosa individuals, via multi-gene association. To validate positive associations, we conducted single-marker analysis in a linkage population of 1,200 individuals. We identified 118, 121, and 43 associations (P&lt; 0.01) corresponding to additive, dominant, and epistatic effects, respectively, with low to moderate proportions of phenotypic variance (R2). Epistatic interaction models uncovered a combination of three non-synonymous sites from three unique genes, representing a significant epistasis for diameter at breast height and stem volume. Single-marker analysis validated 61 associations (false discovery rate, Q ≤ 0.10), representing 38 SNPs from nine genes, and its average effect (R2 = 3.8%) nearly 2-fold higher than that identified with multi-gene association, suggesting that multi-gene association can capture smaller individual variants. Moreover, a structural gene–gene network based on tissue-specific transcript abundances provides a better understanding of the multi-gene pathway affecting tree growth and lignocellulose biosynthesis. Our study highlights the importance of pathway-based multiple gene associations to uncover the nature of genetic variance for quantitative traits and may drive novel progress in molecular breeding.","pageStart":"53","pageEnd":"67","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217377&Signature=KwVIa5PUOAzTP7PGVofk0Mv5TPcKA2LZdveyiWojA43kESCChodqwY0CsErw9XJng-bRlOZrbsDUlBD1el4i2TkxY5HeF3fcKhePgQLfxipxbKn112wzcnrESqvbCUiXaFJmksjjnDJ4JdjxlL-AWrXU0wtPMPwKNOKf8X-LTs~ve7JybF8WGsRNMq9ZaDw12z2vjXYDIrd7qVndeR9cC5Xe3qTsRR4l9CzthC5KRUaPWUM21dcKl-6FQFA4SstoEN5U7U3vH8LquxAcPZDsKVMoPz6Z1Sr8M4yR2scYBZq21I6uYKCEMh4WQzDxOkb1~Bn1BIZv6jzZpe1x0~BoMA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in  Populus tomentosa   †  ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217377&Signature=KwVIa5PUOAzTP7PGVofk0Mv5TPcKA2LZdveyiWojA43kESCChodqwY0CsErw9XJng-bRlOZrbsDUlBD1el4i2TkxY5HeF3fcKhePgQLfxipxbKn112wzcnrESqvbCUiXaFJmksjjnDJ4JdjxlL-AWrXU0wtPMPwKNOKf8X-LTs~ve7JybF8WGsRNMq9ZaDw12z2vjXYDIrd7qVndeR9cC5Xe3qTsRR4l9CzthC5KRUaPWUM21dcKl-6FQFA4SstoEN5U7U3vH8LquxAcPZDsKVMoPz6Z1Sr8M4yR2scYBZq21I6uYKCEMh4WQzDxOkb1~Bn1BIZv6jzZpe1x0~BoMA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in  Populus tomentosa   †   | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Economically important traits in many species generally show polygenic, quantitative inheritance. The components of genetic variation (additive, dominant and epistatic effects) of these traits conferred by multiple genes in shared biological pathways remain to be defined. Here, we investigated 11 full-length genes in cellulose biosynthesis, on 10 growth and wood-property traits, within a population of 460 unrelated Populus tomentosa individuals, via multi-gene association. To validate positive associations, we conducted single-marker analysis in a linkage population of 1,200 individuals. We identified 118, 121, and 43 associations (P&amp;lt; 0.01) corresponding to additive, dominant, and epistatic effects, respectively, with low to moderate proportions of phenotypic variance (R2). Epistatic interaction models uncovered a combination of three non-synonymous sites from three unique genes, representing a significant epistasis for diameter at breast height and stem volume. Single-marker analysis validated 61 associations (false discovery rate, Q ≤ 0.10), representing 38 SNPs from nine genes, and its average effect (R2 = 3.8%) nearly 2-fold higher than that identified with multi-gene association, suggesting that multi-gene association can capture smaller individual variants. Moreover, a structural gene–gene network based on tissue-specific transcript abundances provides a better understanding of the multi-gene pathway affecting tree growth and lignocellulose biosynthesis. Our study highlights the importance of pathway-based multiple gene associations to uncover the nature of genetic variance for quantitative traits and may drive novel progress in molecular breeding." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/53/441064" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217377&amp;Signature=Cu~17Ij8dR3fHwR3jN3M9BXt356b6-6AQVcGj-AadG3Eu7MMkhn9vNOHVPygN~qmPtESXJukczjYvvXUkMnTmbw5sLgLd~FXphpLSLk13G~8P2KjBlzT8eRrDxfmIhpH7~U9x7vcS~7H7-1bXFoa3d3bJWM0d5sNS2C1SBZkRPziGeXgTIlrkV3cRk2eK-S1Q9CenYr7Gn8HQ2lPptcLUICaPf8w7AwE-iLfHj8gCAjJrPBFaeAQXgmkm~9WfoJMcGcBv5h6cupErgZeZnn5arDt15M~d7LzLjrmlxi9NzGbdafShWzHBxeBMsInDQTRmkuiCsfuv-5OObaqlx0Lfw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2014-11-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Du, Qingzhang" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Tian, Jiaxing" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Yang, Xiaohui" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Pan, Wei" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Xu, Baohua" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Li, Bailian" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Department of Forestry, North Carolina State University, Raleigh, NC 27695-8203, USA" /><meta name="citation_author" content="Ingvarsson, Pär K." /><meta name="citation_author_institution" content="Department of Ecology and Environmental Science, Umeå Plant Science Centre, Umeå University, Umeå SE-901 87, Sweden" /><meta name="citation_author" content="Zhang, Deqiang" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_title" content="Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in Populus tomentosa" /><meta name="citation_firstpage" content="53" /><meta name="citation_lastpage" content="67" /><meta name="citation_doi" content="10.1093/dnares/dsu040" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Forest tree genomics: growing resources and applications; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=111-22; " /><meta name="citation_reference" content="citation_title=Toward a systems approach to understanding plant cell walls; Science;  citation_year=2004;  citation_volume=306;  citation_pages=2206-11; " /><meta name="citation_reference" content="citation_title=Identification of genes required for cellulose synthesis by regression analysis of public microarray data sets; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=8633-8; " /><meta name="citation_reference" content="citation_title=DNA sequence variation and selection of tag single-nucleotide polymorphisms at candidate genes for drought-stress response in Pinus taeda L; Genetics;  citation_year=2006;  citation_volume=172;  citation_pages=1915-26; " /><meta name="citation_reference" content="citation_title=Association genetics of the loblolly pine (Pinus taeda, Pinaceae) metabolome; New Phytol.;  citation_year=2012;  citation_volume=193;  citation_pages=890-902; " /><meta name="citation_reference" content="citation_title=Polymorphisms in cinnamoyl coa reductase (CCR) are associated with variation in microﬁbril angle in Eucalyptus spp.; Genetics;  citation_year=2005;  citation_volume=171;  citation_pages=1257-65; " /><meta name="citation_reference" content="citation_title=Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (Populus trichocarpa, Salicaceae) secondary xylem; New Phytol.;  citation_year=2010;  citation_volume=188;  citation_pages=515-32; " /><meta name="citation_reference" content="citation_title=Association genetics of wood physical traits in the conifer White Spruce and relationships with gene expression; Genetics;  citation_year=2011;  citation_volume=188;  citation_pages=197-214; " /><meta name="citation_reference" content="citation_title=Polymorphic SSR loci within PtoCesA genes are associated with growth and wood properties in Populus tomentosa; New Phytol.;  citation_year=2013;  citation_volume=197;  citation_pages=763-76; " /><meta name="citation_reference" content="citation_title=Association genetics of chemical wood properties in black poplar (Populus nigra); New Phytol.;  citation_year=2013;  citation_volume=197;  citation_pages=162-76; " /><meta name="citation_reference" content="citation_title=Analysing biological pathways in genome-wide association studies; Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=843-54; " /><meta name="citation_reference" content="citation_title=Simultaneous analysis of all SNPs in genome-wide and re-sequencing association studies; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000130" /><meta name="citation_reference" content="citation_title=Epistasis dominates the genetic architecture of Drosophila quantitative traits; Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=15553-59; " /><meta name="citation_reference" content="citation_title=Common SNPs explain a large proportion of the heritability for human height; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=565-9; " /><meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=203-8; " /><meta name="citation_reference" content="citation_title=The UDP-glucuronate decarboxylase gene family in Populus: structure, expression, and association genetics,; PLoS ONE;  citation_year=2013;  citation_volume=84;  citation_pages=e60880" /><meta name="citation_reference" content="citation_title=Allelic variation in a cellulose synthase gene PtoCesA4 associated with growth and wood properties in Populus tomentosa; G3;  citation_year=2013;  citation_volume=3;  citation_pages=2069-84; " /><meta name="citation_reference" content="citation_title=Allelic variation in PtGA20Ox associates with growth and wood properties in Populus spp; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e53116" /><meta name="citation_reference" content="citation_title=Single-nucleotide polymorphisms in PtoCesA7 and their association with growth and wood properties in Populus tomentosa; Mol. Genet. Genomics;  citation_year=2014;  citation_volume=289;  citation_pages=439-55; " /><meta name="citation_reference" content="citation_title=Variation in growth, leaf, and wood property traits of Chinese white poplar Populus tomentosa, a major industrial tree species in Northern China; Can. J. For. Res.;  citation_year=2014;  citation_volume=44;  citation_pages=326-39; " /><meta name="citation_reference" content="citation_title=Generation and analysis of expressed sequence tags from six developing xylem libraries in Pinus radiata D.Don; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=1-18; " /><meta name="citation_reference" content="citation_title=The actin multigene family in Populus: organization, expression and phylogenetic analysis; Mol. Genet. Genomics;  citation_year=2010;  citation_volume=284;  citation_pages=105-19; " /><meta name="citation_reference" content="citation_title=Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation; Am. J. Hum. Genet.;  citation_year=2005;  citation_volume=76;  citation_pages=449-62; " /><meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=2496-7; " /><meta name="citation_reference" content="citation_publisher=Columbia University Press, New York; Molecular evolutionary genetics;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=The Bayesian lasso; J Am Stat Assoc.;  citation_year=2008;  citation_volume=103;  citation_pages=681-6; " /><meta name="citation_reference" content="citation_title=On the number of segregating sites in genetical models without recombination; Theor. Popul. Boil.;  citation_year=1975;  citation_volume=7;  citation_pages=188-93; " /><meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism; Genetics;  citation_year=1989;  citation_volume=123;  citation_pages=585-95; " /><meta name="citation_reference" content="citation_title=Statistical tests of neutrality of mutations; Genetics;  citation_year=1993;  citation_volume=133;  citation_pages=693-709; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Linkage disequilibrium in finite populations; Theor. Appl. Genet.;  citation_year=1968;  citation_volume=38;  citation_pages=226-31; " /><meta name="citation_reference" content="citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=11479-84; " /><meta name="citation_reference" content="citation_title=The Bayesian lasso for genome-wide association studies; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=516-23; " /><meta name="citation_reference" content="citation_title=Molecular evolution of synonymous codon usage in Populus; BMC Evol. Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=307" /><meta name="citation_reference" content="citation_title=Genome-wide association implicates numerous genes underlying ecological trait variation in natural populations of Populus trichocarpa; New Phytol.;  citation_year=2014;  citation_volume=22; " /><meta name="citation_reference" content="citation_title=Multifactor dimensionality reduction software for detecting gene-gene and gene-environment interactions; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=376-82; " /><meta name="citation_reference" content="citation_title=A flexible computational framework for detecting, characterizing, and interpreting statistical patterns of epistasis in genetic studies of human disease susceptibility; J. Theor. Boil.;  citation_year=2006;  citation_volume=241;  citation_pages=252-61; " /><meta name="citation_reference" content="citation_title=An information-gain approach to detecting three-way epistatic interactions in genetic association studies; J. Am. Med. Inform. Assoc.;  citation_year=2013;  citation_volume=20;  citation_pages=630-6; " /><meta name="citation_reference" content="Sinnwell J. Schaid D. haplo.stats: Statistical Analysis of Haplotypes With Traits and Covariates When Linkage Phase Is Ambiguous 2009 R package version 1.4.4. http://CRAN.R-project.org/package=haplo.stats (1 August 2012, date last accessed) " /><meta name="citation_reference" content="citation_title=Population structure and eigen analysis; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=2074-93; " /><meta name="citation_reference" content="citation_title=Genetic diversity and population structure of Chinese white poplar (Populus tomentosa) revealed by SSR markers; J. Hered.;  citation_year=2012;  citation_volume=103;  citation_pages=853-62; " /><meta name="citation_reference" content="citation_title=Statistical signiﬁcance for genome wide studies; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=9440-5; " /><meta name="citation_reference" content="citation_title=Nucleotide polymorphism and phenotypic associations within and around the phytochrome B2 locus in European aspen Populus tremula, Salicaceae; Genetics;  citation_year=2008;  citation_volume=178;  citation_pages=2217-26; " /><meta name="citation_reference" content="citation_title=Data and theory point to mainly additive genetic variance for complex traits; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000008" /><meta name="citation_reference" content="citation_title=Genomic selection for growth and wood quality in Eucalyptus: capturing the missing heritability and accelerating breeding for complex traits in forest trees; New Phytol.;  citation_year=2012;  citation_volume=194;  citation_pages=116-28; " /><meta name="citation_reference" content="citation_title=Cellulose synthesis in higher plants; Annu. Rev. Cell Dev. Biol.;  citation_year=2006;  citation_volume=22;  citation_pages=53-78; " /><meta name="citation_reference" content="citation_title=Genetic evidence for three unique components in primary cell-wall cellulose synthase complexes in Arabidopsis; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=15566-71; " /><meta name="citation_reference" content="citation_title=The irregular xylem 2 mutant is an allele of korrigan that affects the secondary cell wall of Arabidopsis thaliana; Plant J.;  citation_year=2004;  citation_volume=37;  citation_pages=730-40; " /><meta name="citation_reference" content="citation_title=Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=13118-23; " /><meta name="citation_reference" content="citation_title=Genomics of cellulose biosynthesis in poplars; New Phytol.;  citation_year=2004;  citation_volume=164;  citation_pages=53-61; " /><meta name="citation_reference" content="citation_title=A highly expressed family 1 β-glucosidase with transglycosylation capacity from the anaerobic fungus Piromyces sp, E2; Biochim. Biophys. Acta;  citation_year=2002;  citation_volume=1574;  citation_pages=293-303; " /><meta name="citation_reference" content="citation_title=Structural organization and a standardized nomenclature for plant endo-1,4-β-glucanases cellulases of glycosyl hydrolase family 9; Plant Physiol.;  citation_year=2007;  citation_volume=144;  citation_pages=1693-6; " /><meta name="citation_reference" content="citation_title=Functional cloning and characterization of a UDP-glucuronic acid decarboxylase: the pathogenic fungus Cryptococcus neoformans elucidates UDP-xylose synthesis; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=12003-08; " /><meta name="citation_reference" content="citation_title=Properties and kinetic analysis of UDP-glucose dehydrogenase from group A streptococci. Irreversible inhibition by UDP-chloroacetol; J. Biol. Chem.;  citation_year=1997;  citation_volume=272;  citation_pages=3416-22; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping for wood characteristics in Populus identifies an array of candidate single nucleotide polymorphisms; New Phytol.;  citation_year=2013;  citation_volume=200;  citation_pages=710-26; " /><meta name="citation_reference" content="citation_title=The molecular basis of dominance; Genetics;  citation_year=1981;  citation_volume=97;  citation_pages=639-66; " /><meta name="citation_reference" content="citation_title=An epistatic ratchet constrains the direction of glucocorticoid receptor evolution; Nature;  citation_year=2009;  citation_volume=461;  citation_pages=515-9; " /><meta name="citation_reference" content="citation_title=Epistasis among adaptive mutations in deer mouse hemoglobin; Science;  citation_year=2013;  citation_volume=340;  citation_pages=1324-7; " /><meta name="citation_reference" content="citation_title=A first-generation haplotype map of maize; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1115-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/53/1209786/dsu040.pdf" /><meta name="description" content="Abstract.  Economically important traits in many species generally show polygenic, quantitative inheritance. The components of genetic variation (additive, domi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/53/441064" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/53/441064" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsu040';
    </script>
</head>

<script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2053,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,n,e){function r(e){if(!n[e]){var o=n[e]={exports:{}};t[e][0].call(o.exports,function(n){var o=t[e][1][n];return r(o||n)},o,o.exports)}return n[e].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<e.length;o++)r(e[o]);return r}({1:[function(t,n,e){function r(t){try{s.console&&console.log(t)}catch(n){}}var o,i=t("ee"),a=t(15),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,n,e){r(e.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,n){return t}).join(", ")))},{}],2:[function(t,n,e){function r(t,n,e,r,o){try{d?d-=1:i("err",[o||new UncaughtException(t,n,e)])}catch(s){try{i("ierr",[s,c.now(),!0])}catch(u){}}return"function"==typeof f&&f.apply(this,a(arguments))}function UncaughtException(t,n,e){this.message=t||"Uncaught error with no additional information",this.sourceURL=n,this.line=e}function o(t){i("err",[t,c.now()])}var i=t("handle"),a=t(16),s=t("ee"),c=t("loader"),f=window.onerror,u=!1,d=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(l){"stack"in l&&(t(8),t(7),"addEventListener"in window&&t(5),c.xhrWrappable&&t(9),u=!0)}s.on("fn-start",function(t,n,e){u&&(d+=1)}),s.on("fn-err",function(t,n,e){u&&(this.thrown=!0,o(e))}),s.on("fn-end",function(){u&&!this.thrown&&d>0&&(d-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,n,e){t("loader").features.ins=!0},{}],4:[function(t,n,e){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(8),s=t(7),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",l="resource",p="-start",h="-end",m="fn"+p,w="fn"+h,v="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(6);var b=NREUM.o.EV;o.on(m,function(t,n){var e=t[0];e instanceof b&&(this.bstStart=g.now())}),o.on(w,function(t,n){var e=t[0];e instanceof b&&i("bst",[e,n,this.bstStart,g.now()])}),a.on(m,function(t,n,e){this.bstStart=g.now(),this.bstType=e}),a.on(w,function(t,n){i(v,[n,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(w,function(t,n){i(v,[n,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+p,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],5:[function(t,n,e){function r(t){for(var n=t;n&&!n.hasOwnProperty(u);)n=Object.getPrototypeOf(n);n&&o(n)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,n){return t[1]}var a=t("ee").get("events"),s=t(18)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";n.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,n){var e=t[1],r=c(e,"nr@wrapped",function(){function t(){if("function"==typeof e.handleEvent)return e.handleEvent.apply(e,arguments)}var n={object:t,"function":e}[typeof e];return n?s(n,"fn-",null,n.name||"anonymous"):e});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],6:[function(t,n,e){var r=t("ee").get("history"),o=t(18)(r);n.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],7:[function(t,n,e){var r=t("ee").get("raf"),o=t(18)(r),i="equestAnimationFrame";n.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],8:[function(t,n,e){function r(t,n,e){t[0]=a(t[0],"fn-",null,e)}function o(t,n,e){this.method=e,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,e)}var i=t("ee").get("timer"),a=t(18)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";n.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],9:[function(t,n,e){function r(t,n){d.inPlace(n,["onreadystatechange"],"fn-",s)}function o(){var t=this,n=u.context(t);t.readyState>3&&!n.resolved&&(n.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):w?w(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,n){return n}function c(t,n){for(var e in t)n[e]=t[e];return n}t(5);var f=t("ee"),u=f.get("xhr"),d=t(18)(u),l=NREUM.o,p=l.XHR,h=l.MO,m=l.PR,w=l.SI,v="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];n.exports=u;var b=window.XMLHttpRequest=function(t){var n=new p(t);try{u.emit("new-xhr",[n],n),n.addEventListener(v,o,!1)}catch(e){try{u.emit("internal-error",[e])}catch(r){}}return n};if(c(p,b),b.prototype=p.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,n){r(t,n),i(n)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!w&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===v||a()})},{}],10:[function(t,n,e){function r(t){var n=this.params,e=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!n.aborted){if(e.duration=a.now()-this.startTime,4===t.readyState){n.status=t.status;var i=o(t,this.lastSize);if(i&&(e.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(n.cat=c.split(", ").pop())}}else n.status=0;e.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[n,e,this.startTime])}}}function o(t,n){var e=t.responseType;if("json"===e&&null!==n)return n;var r="arraybuffer"===e||"blob"===e||"json"===e?t.response:t.responseText;return h(r)}function i(t,n){var e=c(n),r=t.params;r.host=e.hostname+":"+e.port,r.pathname=e.pathname,t.sameOrigin=e.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(11),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,l=t("id"),p=t(14),h=t(13),m=window.XMLHttpRequest;a.features.xhr=!0,t(9),f.on("new-xhr",function(t){var n=this;n.totalCbs=0,n.called=0,n.cbTime=0,n.end=r,n.ended=!1,n.xhrGuids={},n.lastSize=null,p&&(p>34||p<10)||window.opera||t.addEventListener("progress",function(t){n.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,n){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,n){var e=this.metrics,r=t[0],o=this;if(e&&r){var i=h(r);i&&(e.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof n.onload))&&o.end(n)}catch(e){try{f.emit("internal-error",[e])}catch(r){}}};for(var s=0;s<d;s++)n.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,n,e){this.cbTime+=t,n?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof e.onload||this.end(e)}),f.on("xhr-load-added",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&!this.xhrGuids[e]&&(this.xhrGuids[e]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&this.xhrGuids[e]&&(delete this.xhrGuids[e],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],n)}),f.on("removeEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],n)}),f.on("fn-start",function(t,n,e){n instanceof m&&("onload"===e&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,n){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,n],n)})}},{}],11:[function(t,n,e){n.exports=function(t){var n=document.createElement("a"),e=window.location,r={};n.href=t,r.port=n.port;var o=n.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=n.hostname||e.hostname,r.pathname=n.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!n.protocol||":"===n.protocol||n.protocol===e.protocol,a=n.hostname===document.domain&&n.port===e.port;return r.sameOrigin=i&&(!n.hostname||a),r}},{}],12:[function(t,n,e){function r(){}function o(t,n,e){return function(){return i(t,[f.now()].concat(s(arguments)),n?null:this,e),n?void 0:this}}var i=t("handle"),a=t(15),s=t(16),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",p=l+"ixn-";a(d,function(t,n){u[n]=o(l+n,!0,"api")}),u.addPageAction=o(l+"addPageAction",!0),u.setCurrentRouteName=o(l+"routeName",!0),n.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,n){var e={},r=this,o="function"==typeof n;return i(p+"tracer",[f.now(),t,e],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],e),o)try{return n.apply(this,arguments)}finally{c.emit("fn-end",[f.now()],e)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,n){h[n]=o(p+n)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],13:[function(t,n,e){n.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(n){return}}}},{}],14:[function(t,n,e){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),n.exports=r},{}],15:[function(t,n,e){function r(t,n){var e=[],r="",i=0;for(r in t)o.call(t,r)&&(e[i]=n(r,t[r]),i+=1);return e}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],16:[function(t,n,e){function r(t,n,e){n||(n=0),"undefined"==typeof e&&(e=t?t.length:0);for(var r=-1,o=e-n||0,i=Array(o<0?0:o);++r<o;)i[r]=t[n+r];return i}n.exports=r},{}],17:[function(t,n,e){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],18:[function(t,n,e){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(16),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;n.exports=function(t,n){function e(t,n,e,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof e?e(r,a):e||{}}catch(f){l([f,"",[r,a,o],s])}u(n+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(n+"err",[r,a,d],s),d}finally{u(n+"end",[r,a,c],s)}}return r(t)?t:(n||(n=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,n,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<n.length;c++)s=n[c],a=t[s],r(a)||(t[s]=e(a,f?s+o:o,i,s))}function u(e,r,o){if(!c||n){var i=c;c=!0;try{t.emit(e,r,o,n)}catch(a){l([a,e,r,o])}c=i}}function d(t,n){if(Object.defineProperty&&Object.keys)try{var e=Object.keys(t);return e.forEach(function(e){Object.defineProperty(n,e,{get:function(){return t[e]},set:function(n){return t[e]=n,n}})}),n}catch(r){l([r])}for(var o in t)s.call(t,o)&&(n[o]=t[o]);return n}function l(n){try{t.emit("internal-error",n)}catch(e){}}return t||(t=o),e.inPlace=f,e.flag=a,e}},{}],ee:[function(t,n,e){function r(){}function o(t){function n(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function e(e,r,o,i){if(!l.aborted||i){t&&t(e,r,o);for(var a=n(o),s=h(e),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[y[e]];return d&&d.push([g,e,r,a]),a}}function p(t,n){v[t]=h(t).concat(n)}function h(t){return v[t]||[]}function m(t){return d[t]=d[t]||o(e)}function w(t,n){f(t,function(t,e){n=n||"feature",y[e]=n,n in u||(u[n]=[])})}var v={},y={},g={on:p,emit:e,get:m,listeners:h,context:n,buffer:w,abort:a,aborted:!1};return g}function i(){return new r}function a(){(u.api||u.feature)&&(l.aborted=!0,u=l.backlog={})}var s="nr@context",c=t("gos"),f=t(15),u={},d={},l=n.exports=o();l.backlog=u},{}],gos:[function(t,n,e){function r(t,n,e){if(o.call(t,n))return t[n];var r=e();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(t,n,e){function r(t,n,e,r){o.buffer([t],r),o.emit(t,n,e)}var o=t("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(t,n,e){function r(t){var n=typeof t;return!t||"object"!==n&&"function"!==n?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");n.exports=r},{}],loader:[function(t,n,e){function r(){if(!x++){var t=b.info=NREUM.info,n=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&n))return u.abort();f(y,function(n,e){t[n]||(t[n]=e)}),c("mark",["onload",a()+b.offset],null,"api");var e=l.createElement("script");e.src="https://"+t.agent,n.parentNode.insertBefore(e,n)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+b.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-b.offset}var s=(new Date).getTime(),c=t("handle"),f=t(15),u=t("ee"),d=window,l=d.document,p="addEventListener",h="attachEvent",m=d.XMLHttpRequest,w=m&&m.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:m,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},g=m&&w&&w[p]&&!/CriOS/.test(navigator.userAgent),b=n.exports={offset:s,now:a,origin:v,features:{},xhrWrappable:g};t(12),l[p]?(l[p]("DOMContentLoaded",i,!1),d[p]("load",r,!1)):(l[h]("onreadystatechange",o),d[h]("onload",r)),c("mark",["firstbyte",s],null,"api");var x=0,E=t(17)},{}]},{},["loader",2,10,4,3]);</script><body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./441064" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFwJQBPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEdSwgUWluZ3poYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgRW5naW5lZXJpbmcgTGFib3JhdG9yeSBmb3IgVHJlZSBCcmVlZGluZywgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUaWFuLCBKaWF4aW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgRW5naW5lZXJpbmcgTGFib3JhdG9yeSBmb3IgVHJlZSBCcmVlZGluZywgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW5nLCBYaWFvaHVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgRW5naW5lZXJpbmcgTGFib3JhdG9yeSBmb3IgVHJlZSBCcmVlZGluZywgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYW4sIFdlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEVuZ2luZWVyaW5nIExhYm9yYXRvcnkgZm9yIFRyZWUgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljcyBhbmQgQnJlZWRpbmcgaW4gRm9yZXN0IFRyZWVzIGFuZCBPcm5hbWVudGFsIFBsYW50cywgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWHUsIEJhb2h1YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEVuZ2luZWVyaW5nIExhYm9yYXRvcnkgZm9yIFRyZWUgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljcyBhbmQgQnJlZWRpbmcgaW4gRm9yZXN0IFRyZWVzIGFuZCBPcm5hbWVudGFsIFBsYW50cywgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIEJhaWxpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBFbmdpbmVlcmluZyBMYWJvcmF0b3J5IGZvciBUcmVlIEJyZWVkaW5nLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgR2VuZXRpY3MgYW5kIEJyZWVkaW5nIGluIEZvcmVzdCBUcmVlcyBhbmQgT3JuYW1lbnRhbCBQbGFudHMsIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRm9yZXN0cnksIE5vcnRoIENhcm9saW5hIFN0YXRlIFVuaXZlcnNpdHksIFJhbGVpZ2gsIE5DIDI3Njk1LTgyMDMsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkluZ3ZhcnNzb24sIFDDpHIgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVjb2xvZ3kgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZSwgVW1lw6UgUGxhbnQgU2NpZW5jZSBDZW50cmUsIFVtZcOlIFVuaXZlcnNpdHksIFVtZcOlIFNFLTkwMSA4NywgU3dlZGVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIERlcWlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBFbmdpbmVlcmluZyBMYWJvcmF0b3J5IGZvciBUcmVlIEJyZWVkaW5nLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgR2VuZXRpY3MgYW5kIEJyZWVkaW5nIGluIEZvcmVzdCBUcmVlcyBhbmQgT3JuYW1lbnRhbCBQbGFudHMsIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSWRlbnRpZmljYXRpb24gb2YgYWRkaXRpdmUsIGRvbWluYW50LCBhbmQgZXBpc3RhdGljIHZhcmlhdGlvbiBjb25mZXJyZWQgYnkga2V5IGdlbmVzIGluIGNlbGx1bG9zZSBiaW9zeW50aGVzaXMgcGF0aHdheSBpbiBQb3B1bHVzIHRvbWVudG9zYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTA0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9yZXN0IHRyZWUgZ2Vub21pY3M6IGdyb3dpbmcgcmVzb3VyY2VzIGFuZCBhcHBsaWNhdGlvbnM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZCBhIHN5c3RlbXMgYXBwcm9hY2ggdG8gdW5kZXJzdGFuZGluZyBwbGFudCBjZWxsIHdhbGxzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMwNjsgIGNpdGF0aW9uX3BhZ2VzPTIyMDYtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGdlbmVzIHJlcXVpcmVkIGZvciBjZWxsdWxvc2Ugc3ludGhlc2lzIGJ5IHJlZ3Jlc3Npb24gYW5hbHlzaXMgb2YgcHVibGljIG1pY3JvYXJyYXkgZGF0YSBzZXRzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTg2MzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHNlcXVlbmNlIHZhcmlhdGlvbiBhbmQgc2VsZWN0aW9uIG9mIHRhZyBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGF0IGNhbmRpZGF0ZSBnZW5lcyBmb3IgZHJvdWdodC1zdHJlc3MgcmVzcG9uc2UgaW4gUGludXMgdGFlZGEgTDsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTcyOyAgY2l0YXRpb25fcGFnZXM9MTkxNS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gZ2VuZXRpY3Mgb2YgdGhlIGxvYmxvbGx5IHBpbmUgKFBpbnVzIHRhZWRhLCBQaW5hY2VhZSkgbWV0YWJvbG9tZTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTkzOyAgY2l0YXRpb25fcGFnZXM9ODkwLTkwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seW1vcnBoaXNtcyBpbiBjaW5uYW1veWwgY29hIHJlZHVjdGFzZSAoQ0NSKSBhcmUgYXNzb2NpYXRlZCB3aXRoIHZhcmlhdGlvbiBpbiBtaWNyb++sgWJyaWwgYW5nbGUgaW4gRXVjYWx5cHR1cyBzcHAuOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE7ICBjaXRhdGlvbl9wYWdlcz0xMjU3LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBnZW5ldGljcyBvZiB0cmFpdHMgY29udHJvbGxpbmcgbGlnbmluIGFuZCBjZWxsdWxvc2UgYmlvc3ludGhlc2lzIGluIGJsYWNrIGNvdHRvbndvb2QgKFBvcHVsdXMgdHJpY2hvY2FycGEsIFNhbGljYWNlYWUpIHNlY29uZGFyeSB4eWxlbTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9NTE1LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBnZW5ldGljcyBvZiB3b29kIHBoeXNpY2FsIHRyYWl0cyBpbiB0aGUgY29uaWZlciBXaGl0ZSBTcHJ1Y2UgYW5kIHJlbGF0aW9uc2hpcHMgd2l0aCBnZW5lIGV4cHJlc3Npb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTE5Ny0yMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHltb3JwaGljIFNTUiBsb2NpIHdpdGhpbiBQdG9DZXNBIGdlbmVzIGFyZSBhc3NvY2lhdGVkIHdpdGggZ3Jvd3RoIGFuZCB3b29kIHByb3BlcnRpZXMgaW4gUG9wdWx1cyB0b21lbnRvc2E7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTc2My03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gZ2VuZXRpY3Mgb2YgY2hlbWljYWwgd29vZCBwcm9wZXJ0aWVzIGluIGJsYWNrIHBvcGxhciAoUG9wdWx1cyBuaWdyYSk7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTE2Mi03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaW5nIGJpb2xvZ2ljYWwgcGF0aHdheXMgaW4gZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTg0My01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltdWx0YW5lb3VzIGFuYWx5c2lzIG9mIGFsbCBTTlBzIGluIGdlbm9tZS13aWRlIGFuZCByZS1zZXF1ZW5jaW5nIGFzc29jaWF0aW9uIHN0dWRpZXM7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDEzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaXN0YXNpcyBkb21pbmF0ZXMgdGhlIGdlbmV0aWMgYXJjaGl0ZWN0dXJlIG9mIERyb3NvcGhpbGEgcXVhbnRpdGF0aXZlIHRyYWl0czsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xNTU1My01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIFNOUHMgZXhwbGFpbiBhIGxhcmdlIHByb3BvcnRpb24gb2YgdGhlIGhlcml0YWJpbGl0eSBmb3IgaHVtYW4gaGVpZ2h0OyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTU2NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHVuaWZpZWQgbWl4ZWQtbW9kZWwgbWV0aG9kIGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nIHRoYXQgYWNjb3VudHMgZm9yIG11bHRpcGxlIGxldmVscyBvZiByZWxhdGVkbmVzczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFVEUC1nbHVjdXJvbmF0ZSBkZWNhcmJveHlsYXNlIGdlbmUgZmFtaWx5IGluIFBvcHVsdXM6IHN0cnVjdHVyZSwgZXhwcmVzc2lvbiwgYW5kIGFzc29jaWF0aW9uIGdlbmV0aWNzLDsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz1lNjA4ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbGxlbGljIHZhcmlhdGlvbiBpbiBhIGNlbGx1bG9zZSBzeW50aGFzZSBnZW5lIFB0b0Nlc0E0IGFzc29jaWF0ZWQgd2l0aCBncm93dGggYW5kIHdvb2QgcHJvcGVydGllcyBpbiBQb3B1bHVzIHRvbWVudG9zYTsgRzM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTIwNjktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbGVsaWMgdmFyaWF0aW9uIGluIFB0R0EyME94IGFzc29jaWF0ZXMgd2l0aCBncm93dGggYW5kIHdvb2QgcHJvcGVydGllcyBpbiBQb3B1bHVzIHNwcDsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MzExNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXMgaW4gUHRvQ2VzQTcgYW5kIHRoZWlyIGFzc29jaWF0aW9uIHdpdGggZ3Jvd3RoIGFuZCB3b29kIHByb3BlcnRpZXMgaW4gUG9wdWx1cyB0b21lbnRvc2E7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI4OTsgIGNpdGF0aW9uX3BhZ2VzPTQzOS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWF0aW9uIGluIGdyb3d0aCwgbGVhZiwgYW5kIHdvb2QgcHJvcGVydHkgdHJhaXRzIG9mIENoaW5lc2Ugd2hpdGUgcG9wbGFyIFBvcHVsdXMgdG9tZW50b3NhLCBhIG1ham9yIGluZHVzdHJpYWwgdHJlZSBzcGVjaWVzIGluIE5vcnRoZXJuIENoaW5hOyBDYW4uIEouIEZvci4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTMyNi0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgZnJvbSBzaXggZGV2ZWxvcGluZyB4eWxlbSBsaWJyYXJpZXMgaW4gUGludXMgcmFkaWF0YSBELkRvbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MS0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFjdGluIG11bHRpZ2VuZSBmYW1pbHkgaW4gUG9wdWx1czogb3JnYW5pemF0aW9uLCBleHByZXNzaW9uIGFuZCBwaHlsb2dlbmV0aWMgYW5hbHlzaXM7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4NDsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjb3VudGluZyBmb3IgZGVjYXkgb2YgbGlua2FnZSBkaXNlcXVpbGlicml1bSBpbiBoYXBsb3R5cGUgaW5mZXJlbmNlIGFuZCBtaXNzaW5nLWRhdGEgaW1wdXRhdGlvbjsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc2OyAgY2l0YXRpb25fcGFnZXM9NDQ5LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbmFTUCwgRE5BIHBvbHltb3JwaGlzbSBhbmFseXNlcyBieSB0aGUgY29hbGVzY2VudCBhbmQgb3RoZXIgbWV0aG9kczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yNDk2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2x1bWJpYSBVbml2ZXJzaXR5IFByZXNzLCBOZXcgWW9yazsgTW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEJheWVzaWFuIGxhc3NvOyBKIEFtIFN0YXQgQXNzb2MuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTY4MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgbnVtYmVyIG9mIHNlZ3JlZ2F0aW5nIHNpdGVzIGluIGdlbmV0aWNhbCBtb2RlbHMgd2l0aG91dCByZWNvbWJpbmF0aW9uOyBUaGVvci4gUG9wdWwuIEJvaWwuOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xODgtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIG1ldGhvZCBmb3IgdGVzdGluZyB0aGUgbmV1dHJhbCBtdXRhdGlvbiBoeXBvdGhlc2lzIGJ5IEROQSBwb2x5bW9ycGhpc207IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgdGVzdHMgb2YgbmV1dHJhbGl0eSBvZiBtdXRhdGlvbnM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTY5My03MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCB0Uk5BIGNvbnRlbnQgaW4gdW5pY2VsbHVsYXIgYW5kIG11bHRpY2VsbHVsYXIgb3JnYW5pc21zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBkaXNlcXVpbGlicml1bSBpbiBmaW5pdGUgcG9wdWxhdGlvbnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5Njg7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yMjYtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZSBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGFuZCBwaGVub3R5cGljIGFzc29jaWF0aW9ucyBpbiB0aGUgbWFpemUgZ2Vub21lOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTE0NzktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCYXllc2lhbiBsYXNzbyBmb3IgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01MTYtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldm9sdXRpb24gb2Ygc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBQb3B1bHVzOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTMwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIGltcGxpY2F0ZXMgbnVtZXJvdXMgZ2VuZXMgdW5kZXJseWluZyBlY29sb2dpY2FsIHRyYWl0IHZhcmlhdGlvbiBpbiBuYXR1cmFsIHBvcHVsYXRpb25zIG9mIFBvcHVsdXMgdHJpY2hvY2FycGE7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aWZhY3RvciBkaW1lbnNpb25hbGl0eSByZWR1Y3Rpb24gc29mdHdhcmUgZm9yIGRldGVjdGluZyBnZW5lLWdlbmUgYW5kIGdlbmUtZW52aXJvbm1lbnQgaW50ZXJhY3Rpb25zOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmbGV4aWJsZSBjb21wdXRhdGlvbmFsIGZyYW1ld29yayBmb3IgZGV0ZWN0aW5nLCBjaGFyYWN0ZXJpemluZywgYW5kIGludGVycHJldGluZyBzdGF0aXN0aWNhbCBwYXR0ZXJucyBvZiBlcGlzdGFzaXMgaW4gZ2VuZXRpYyBzdHVkaWVzIG9mIGh1bWFuIGRpc2Vhc2Ugc3VzY2VwdGliaWxpdHk7IEouIFRoZW9yLiBCb2lsLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDE7ICBjaXRhdGlvbl9wYWdlcz0yNTItNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGluZm9ybWF0aW9uLWdhaW4gYXBwcm9hY2ggdG8gZGV0ZWN0aW5nIHRocmVlLXdheSBlcGlzdGF0aWMgaW50ZXJhY3Rpb25zIGluIGdlbmV0aWMgYXNzb2NpYXRpb24gc3R1ZGllczsgSi4gQW0uIE1lZC4gSW5mb3JtLiBBc3NvYy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz02MzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iU2lubndlbGwgSi4gU2NoYWlkIEQuIGhhcGxvLnN0YXRzOiBTdGF0aXN0aWNhbCBBbmFseXNpcyBvZiBIYXBsb3R5cGVzIFdpdGggVHJhaXRzIGFuZCBDb3ZhcmlhdGVzIFdoZW4gTGlua2FnZSBQaGFzZSBJcyBBbWJpZ3VvdXMgMjAwOSBSIHBhY2thZ2UgdmVyc2lvbiAxLjQuNC4gaHR0cDovL0NSQU4uUi1wcm9qZWN0Lm9yZy9wYWNrYWdlPWhhcGxvLnN0YXRzICgxIEF1Z3VzdCAyMDEyLCBkYXRlIGxhc3QgYWNjZXNzZWQpICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gc3RydWN0dXJlIGFuZCBlaWdlbiBhbmFseXNpczsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTIwNzQtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IGFuZCBwb3B1bGF0aW9uIHN0cnVjdHVyZSBvZiBDaGluZXNlIHdoaXRlIHBvcGxhciAoUG9wdWx1cyB0b21lbnRvc2EpIHJldmVhbGVkIGJ5IFNTUiBtYXJrZXJzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9ODUzLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbCBzaWduae+sgWNhbmNlIGZvciBnZW5vbWUgd2lkZSBzdHVkaWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTk0NDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gYW5kIHBoZW5vdHlwaWMgYXNzb2NpYXRpb25zIHdpdGhpbiBhbmQgYXJvdW5kIHRoZSBwaHl0b2Nocm9tZSBCMiBsb2N1cyBpbiBFdXJvcGVhbiBhc3BlbiBQb3B1bHVzIHRyZW11bGEsIFNhbGljYWNlYWU7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3ODsgIGNpdGF0aW9uX3BhZ2VzPTIyMTctMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGEgYW5kIHRoZW9yeSBwb2ludCB0byBtYWlubHkgYWRkaXRpdmUgZ2VuZXRpYyB2YXJpYW5jZSBmb3IgY29tcGxleCB0cmFpdHM7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgc2VsZWN0aW9uIGZvciBncm93dGggYW5kIHdvb2QgcXVhbGl0eSBpbiBFdWNhbHlwdHVzOiBjYXB0dXJpbmcgdGhlIG1pc3NpbmcgaGVyaXRhYmlsaXR5IGFuZCBhY2NlbGVyYXRpbmcgYnJlZWRpbmcgZm9yIGNvbXBsZXggdHJhaXRzIGluIGZvcmVzdCB0cmVlczsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9MTE2LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZWxsdWxvc2Ugc3ludGhlc2lzIGluIGhpZ2hlciBwbGFudHM7IEFubnUuIFJldi4gQ2VsbCBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NTMtNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZXZpZGVuY2UgZm9yIHRocmVlIHVuaXF1ZSBjb21wb25lbnRzIGluIHByaW1hcnkgY2VsbC13YWxsIGNlbGx1bG9zZSBzeW50aGFzZSBjb21wbGV4ZXMgaW4gQXJhYmlkb3BzaXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTU1NjYtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpcnJlZ3VsYXIgeHlsZW0gMiBtdXRhbnQgaXMgYW4gYWxsZWxlIG9mIGtvcnJpZ2FuIHRoYXQgYWZmZWN0cyB0aGUgc2Vjb25kYXJ5IGNlbGwgd2FsbCBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz03MzAtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1Y3Jvc2Ugc3ludGhhc2UgYWZmZWN0cyBjYXJib24gcGFydGl0aW9uaW5nIHRvIGluY3JlYXNlIGNlbGx1bG9zZSBwcm9kdWN0aW9uIGFuZCBhbHRlcmVkIGNlbGwgd2FsbCB1bHRyYXN0cnVjdHVyZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xMzExOC0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pY3Mgb2YgY2VsbHVsb3NlIGJpb3N5bnRoZXNpcyBpbiBwb3BsYXJzOyBOZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz01My02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdobHkgZXhwcmVzc2VkIGZhbWlseSAxIM6yLWdsdWNvc2lkYXNlIHdpdGggdHJhbnNnbHljb3N5bGF0aW9uIGNhcGFjaXR5IGZyb20gdGhlIGFuYWVyb2JpYyBmdW5ndXMgUGlyb215Y2VzIHNwLCBFMjsgQmlvY2hpbS4gQmlvcGh5cy4gQWN0YTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTc0OyAgY2l0YXRpb25fcGFnZXM9MjkzLTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBvcmdhbml6YXRpb24gYW5kIGEgc3RhbmRhcmRpemVkIG5vbWVuY2xhdHVyZSBmb3IgcGxhbnQgZW5kby0xLDQtzrItZ2x1Y2FuYXNlcyBjZWxsdWxhc2VzIG9mIGdseWNvc3lsIGh5ZHJvbGFzZSBmYW1pbHkgOTsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ0OyAgY2l0YXRpb25fcGFnZXM9MTY5My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGNsb25pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBVRFAtZ2x1Y3Vyb25pYyBhY2lkIGRlY2FyYm94eWxhc2U6IHRoZSBwYXRob2dlbmljIGZ1bmd1cyBDcnlwdG9jb2NjdXMgbmVvZm9ybWFucyBlbHVjaWRhdGVzIFVEUC14eWxvc2Ugc3ludGhlc2lzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTIwMDMtMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3BlcnRpZXMgYW5kIGtpbmV0aWMgYW5hbHlzaXMgb2YgVURQLWdsdWNvc2UgZGVoeWRyb2dlbmFzZSBmcm9tIGdyb3VwIEEgc3RyZXB0b2NvY2NpLiBJcnJldmVyc2libGUgaW5oaWJpdGlvbiBieSBVRFAtY2hsb3JvYWNldG9sOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0zNDE2LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBtYXBwaW5nIGZvciB3b29kIGNoYXJhY3RlcmlzdGljcyBpbiBQb3B1bHVzIGlkZW50aWZpZXMgYW4gYXJyYXkgb2YgY2FuZGlkYXRlIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXM7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwMDsgIGNpdGF0aW9uX3BhZ2VzPTcxMC0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vbGVjdWxhciBiYXNpcyBvZiBkb21pbmFuY2U7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9NjM5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlcGlzdGF0aWMgcmF0Y2hldCBjb25zdHJhaW5zIHRoZSBkaXJlY3Rpb24gb2YgZ2x1Y29jb3J0aWNvaWQgcmVjZXB0b3IgZXZvbHV0aW9uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDYxOyAgY2l0YXRpb25fcGFnZXM9NTE1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaXN0YXNpcyBhbW9uZyBhZGFwdGl2ZSBtdXRhdGlvbnMgaW4gZGVlciBtb3VzZSBoZW1vZ2xvYmluOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM0MDsgIGNpdGF0aW9uX3BhZ2VzPTEzMjQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmaXJzdC1nZW5lcmF0aW9uIGhhcGxvdHlwZSBtYXAgb2YgbWFpemU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI2OyAgY2l0YXRpb25fcGFnZXM9MTExNS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvMS81My8xMjA5Nzg2L2RzdTA0MC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRWNvbm9taWNhbGx5IGltcG9ydGFudCB0cmFpdHMgaW4gbWFueSBzcGVjaWVzIGdlbmVyYWxseSBzaG93IHBvbHlnZW5pYywgcXVhbnRpdGF0aXZlIGluaGVyaXRhbmNlLiBUaGUgY29tcG9uZW50cyBvZiBnZW5ldGljIHZhcmlhdGlvbiAoYWRkaXRpdmUsIGRvbWkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ0MTA2NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGQuYUssUKrYu/BBk5IHAqUzGOguaA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f53%2f441064" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f53%2f441064" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1509238968&amp;Signature=EKhDR45hE0JvCD7NNoA5tSJUkxjnj69koznjLkmJx9te8bPX36EV57k~dMqMhpf3UAbpL09FykjVTRUPi2x23uanqwItrhyn8sAZZUUv2dvBiH9l0WZ6cSlozq3SPquUUWmBRlnRTNgbm5fxFw0n91Aul7Ya3Vji8eKqL9EI7JcGCTEk-igrHqdfk7ijALyGRxwV4NYzfVaUgtvZktJK08~hfyTg-fajyf3XpnObLSqYcXdCqdrDxxVTZT5GCzwl3m4aQzhKLn~WfSIBiOsH7BtEqv~Worvo48Dql9DyPmT1BhyAaf69BFpFYf3wni-4Ob7kad7ees~OsADdgayfmw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6588585">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588585">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588587">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588587">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588591">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588591">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588617">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588667">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588667">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588681">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588681">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588683">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588683">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588685">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588685">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6588687">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6588687">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/39/440428">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/69/441516">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in <em>Populus tomentosa</em><sup><a reveal-id="AN1" data-open="AN1" class="link link-ref link-reveal xref-fn">†</a></sup>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Qingzhang Du</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qingzhang Du</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Qingzhang+Du">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Du Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Du author:Q.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jiaxing Tian</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiaxing Tian</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jiaxing+Tian">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tian J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Tian author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Xiaohui Yang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaohui Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Xiaohui+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Yang author:X.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Wei Pan</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Pan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Wei+Pan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pan W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Pan author:W.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Baohua Xu</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Baohua Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Baohua+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Xu author:B.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Bailian Li</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bailian Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Forestry</div>, <div class="insititution">North Carolina State University</div>, <div class="addr-line">Raleigh, NC 27695-8203</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Bailian+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Li author:B.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Pär K. Ingvarsson</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">P&#228;r K. Ingvarsson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Ecology and Environmental Science</div>, <div class="insititution">Umeå Plant Science Centre, Umeå University</div>, <div class="addr-line">Umeå SE-901 87</div>, <div class="country">Sweden</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=P%c3%a4r+K.+Ingvarsson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ingvarsson P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Ingvarsson author:P.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Deqiang Zhang</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deqiang Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">National Engineering Laboratory for Tree Breeding</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education</div>, <div class="insititution">College of Biological Sciences and Technology, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">P. R. China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10-62336007. Fax. +86 10-62336164. E-mail: <a href="mailto:DeqiangZhang@bjfu.edu.cn" target="_blank">DeqiangZhang@bjfu.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Deqiang+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Zhang author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Sequence data corresponding to all candidate genes have been deposited in the GenBank Data Library, and the accession numbers were shown in text or tables.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 53–67, <a href='https://doi.org/10.1093/dnares/dsu040'>https://doi.org/10.1093/dnares/dsu040</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 August 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="441064" href="/dnaresearch/article-pdf/22/1/53/1209786/dsu040.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Qingzhang Du, Jiaxing Tian, Xiaohui Yang, Wei Pan, Baohua Xu, Bailian Li, Pär K. Ingvarsson, Deqiang Zhang;  Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in <em>Populus tomentosa</em><sup />, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 53–67, <a href='https://doi.org/10.1093/dnares/dsu040'>https://doi.org/10.1093/dnares/dsu040</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/441064?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/441064?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/441064?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/441064?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/441064?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/441064?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/441064?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20of%20additive%2C%20dominant%2C%20and%20epistatic%20variation%20conferred%20by%20key%20genes%20in%20cellulose%20biosynthesis%20pathway%20in%20%26lt%3Bem%26gt%3BPopulus%20tomentosa%26lt%3B%2Fem%26gt%3B%26lt%3Bsup%26gt%3B%26lt%3Ba%20reveal-id%3D%26quot%3BAN1%26quot%3B%20data-open%3D%26quot%3BAN1%26quot%3B%20class%3D%26quot%3Blink%20link-ref%20link-reveal%20xref-fn%26quot%3B%26gt%3B%E2%80%A0%26lt%3B%2Fa%26gt%3B%26lt%3B%2Fsup%26gt%3B&amp;publicationDate=2014-11-26&amp;volumeNum=22&amp;issueNum=1&amp;author=Du%2C%20Qingzhang%3B%20Tian%2C%20Jiaxing&amp;startPage=53&amp;endPage=67&amp;contentId=10.1093%2Fdnares%2Fdsu040&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f53%2f441064" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6588585 id="6588585" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Economically important traits in many species generally show polygenic, quantitative inheritance. The components of genetic variation (additive, dominant and epistatic effects) of these traits conferred by multiple genes in shared biological pathways remain to be defined. Here, we investigated 11 full-length genes in cellulose biosynthesis, on 10 growth and wood-property traits, within a population of 460 unrelated <em>Populus tomentosa</em> individuals, via multi-gene association. To validate positive associations, we conducted single-marker analysis in a linkage population of 1,200 individuals. We identified 118, 121, and 43 associations (<em>P</em>&lt; 0.01) corresponding to additive, dominant, and epistatic effects, respectively, with low to moderate proportions of phenotypic variance (<em>R</em><sup>2</sup>). Epistatic interaction models uncovered a combination of three non-synonymous sites from three unique genes, representing a significant epistasis for diameter at breast height and stem volume. Single-marker analysis validated 61 associations (false discovery rate, <em>Q</em> ≤ 0.10), representing 38 SNPs from nine genes, and its average effect (<em>R</em><sup>2</sup> = 3.8%) nearly 2-fold higher than that identified with multi-gene association, suggesting that multi-gene association can capture smaller individual variants. Moreover, a structural gene–gene network based on tissue-specific transcript abundances provides a better understanding of the multi-gene pathway affecting tree growth and lignocellulose biosynthesis. Our study highlights the importance of pathway-based multiple gene associations to uncover the nature of genetic variance for quantitative traits and may drive novel progress in molecular breeding.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Chinese white poplar (Populus tomentosa)&quot;">Chinese white poplar (<em>Populus tomentosa</em>)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epistasis">epistasis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;pathway-based multiple gene association&quot;">pathway-based multiple gene association</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcript profiling&quot;">transcript profiling</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;validation population&quot;">validation population</a></div>                            <h2 scrollto-destination=6588587 id="6588587" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Trees have unique features that distinguish them from most herbaceous species, including large sizes, long lifespans and woody, perennial growth. Many tree species occur in widely distributed populations that harbour a wealth of diversity.<sup><a reveal-id="DSU040C1" data-open="DSU040C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Large, diverse tree populations can be exploited for breeding to improve economically important properties. In addition, deciphering the mechanisms by which forest trees have adapted to their changing environments is an intriguing research problem. Tree growth and wood formation are complex dynamic processes that require coordinate regulation of diverse metabolic pathways in the whole plant.<sup><a reveal-id="DSU040C2" data-open="DSU040C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU040C3" data-open="DSU040C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Improving economically important traits in trees by molecular marker-assisted selection (MAS) will require improving our understanding of the molecular mechanisms underlying phenotypic variation, particularly for traits with complex genetic architecture, such as wood properties or growth. Research has been hindered by trees' large size, long generation times, and the lack of mutants for reverse genetic approaches.<sup><a reveal-id="DSU040C1" data-open="DSU040C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU040C4" data-open="DSU040C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>As an alternative approach, recent work has examined complex traits in trees by forward genetic methods, such as quantitative trait loci (QTL) and association mapping of genes or alleles underlying quantitative traits.<sup><a reveal-id="DSU040C5" data-open="DSU040C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Single-marker association studies have identified candidate functional single-nucleotide polymorphisms (SNPs) that explain a small proportion of the additive variation (≤5% on average) in growth or wood properties, and many of these genes have not been cloned in full length yet.<sup><a reveal-id="DSU040C6  DSU040C10" data-open="DSU040C6  DSU040C10" class="link link-ref link-reveal xref-bibr">6–10</a></sup> It is well known that genes do not work in isolation; instead, complex cellular pathways and molecular networks are often involved in phenotypic variation.<sup><a reveal-id="DSU040C11" data-open="DSU040C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Variation in quantitative traits includes additive, dominant, and epistatic effects that are conferred by multiple variants within interactive genes in different biological pathways.<sup><a reveal-id="DSU040C11" data-open="DSU040C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU040C12" data-open="DSU040C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The complete effects of genetic variation for multiple full-length genes [including promoter, introns, untranslated regions (UTRs), and exons] in related biological pathways remain to be addressed in forest trees. Multi-gene association models take advantage of large numbers of SNPs in population-wide linkage disequilibrium (LD) with QTL regions harbouring functional loci affecting complex traits.<sup><a reveal-id="DSU040C13" data-open="DSU040C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Studies using multi-gene association may identify genetic variants that jointly have significant effects but individually make only a small contribution.<sup><a reveal-id="DSU040C13" data-open="DSU040C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSU040C14" data-open="DSU040C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>Our previous studies using single-marker association (i.e. mixed linear models<sup><a reveal-id="DSU040C15" data-open="DSU040C15" class="link link-ref link-reveal xref-bibr">15</a></sup>) have identified sites of SNPs in several candidate genes that located in close proximity to the causal polymorphisms or even the functional variant itself for growth and wood-property traits.<sup><a reveal-id="DSU040C16  DSU040C19" data-open="DSU040C16  DSU040C19" class="link link-ref link-reveal xref-bibr">16–19</a></sup> To extend these observations, this study investigates the nature of genetic variance (additive, dominant, and epistatic effects) for 11 full-length candidate genes related to cellulose biosynthesis in 10 quantitative traits, using multi-gene association, as well as haplotype-based association approaches, in a population of 460 unrelated individuals of <em>Populus tomentosa</em>. To verify the most significant associations, we conducted single-marker analysis in a family-based linkage population consisting of 1,200 individuals. Moreover, tissue-specific transcript abundances of these genes provide a better understanding of the multi-gene network, affecting tree growth and lignocellulose biosynthesis.</p>                            <h2 scrollto-destination=6588591 id="6588591" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6588592 id="6588592" class="section-title" data-legacy-id=s2a>2.1. Population materials and phenotypic data</h3>
<p>The discovery population (association population) consisted of 460 24-yr-old unrelated individuals, representing almost the entire natural distribution range of <em>P. tomentosa</em>, i.e. Southern region, Northwestern region, and Northeastern region (30–40°N, 105–125°E). Plants were randomly sampled in 2009 from a collection of 1,047 natural <em>P. tomentosa</em> in Guan Xian County, Shandong Province, China (36°23′N, 115°47′E) and used for the initial multi-gene association (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>, Du <em>et al.</em><sup><a reveal-id="DSU040C12" data-open="DSU040C12" class="link link-ref link-reveal xref-bibr">12</a></sup>). In addition, 40 unrelated individuals were randomly selected from this association population and used to identify SNPs.</p><p>To confirm the association results identified in the discovery population, a validation population (linkage population) consisting of 1,200 hybrid individuals were randomly selected from 5,000 F<sub>1</sub> progeny established by controlled crossing between clone ‘YX01’ (<em>Populus alba</em>× <em>Populus glandulosa</em>) as the female and clone ‘LM 50’ (<em>P. tomentosa</em>) as the male. The progeny were planted in 2008 in the Xiao Tangshan horticultural fields, Beijing, China (40°2′N, 115°50′E) using a randomized complete block design with three replications per genotype.</p><p>Ten growth and wood-property traits were scored for all individuals in the two populations, using at least three replications per genotype. The growth traits were tree height, diameter at breast height (DBH), and stem volume. The wood-property traits were holocellulose, α-cellulose, hemicellulose, and lignin content (chemical composition), and fibre length, fibre width, and microfibre angle (physical properties) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S1–S2</a>). The detailed sampling and measurement methods, and phenotypic variance for these 10 traits in the two populations were conducted as previously described.<sup><a reveal-id="DSU040C17" data-open="DSU040C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU040C20" data-open="DSU040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Pearson's correlations for these 10 phenotypic traits were shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S3</a>.</p>                            <h3 scrollto-destination=6588596 id="6588596" class="section-title" data-legacy-id=s2b>2.2. RNA extraction and cDNAs identification</h3>
<p>Thirty ramets of 1-yr-old <em>P. tomentosa</em> clone ‘LM50’ were selected and planted at Xiao Tangshan horticultural fields, Beijing, China (40°2′N, 115°50′E). A piece of bark was peeled off from the main stem at ∼1.0 m height of 1-yr-old <em>P. tomentosa</em> clone ‘LM50’ using a chisel. The newly formed xylem (∼3 mm thick) was first removed from the exposed surface, and then mature xylem was collected by further scraping into the stem at ∼2 mm thick; finally, the collected samples were placed into 50-ml Falcon tubes filled with liquid nitrogen and stored in a freezer at −80°C until RNA extraction.<sup><a reveal-id="DSU040C21" data-open="DSU040C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Using the Plant Qiagen RNeasy kit, total RNA from the stem xylem tissues was extracted and then reverse transcribed into cDNA with the SuperScript First-Strand Synthesis system (Life Technologies, Carlsbad, CA, USA). The stem xylem cDNA library was constructed using the Superscript λ System (Life Technologies, Rockville, MD, USA). The detailed procedures for constructing the cDNA library were previously described by Li <em>et al.</em><sup><a reveal-id="DSU040C21" data-open="DSU040C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In our study, the cDNA library consisted of 5.0 × 10<sup>6</sup> pfu with an insert size range of 1.0–4.0 kb. By random end-sequencing of 30,000 cDNA clones and comparison with all sequences in NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>), we chose 11 cDNAs with high similarities to <em>Arabidopsis</em> or <em>Populus trichocarpa</em> sequences to examine the relationship with lignocellulosic biosynthesis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). All cDNA sequences chosen were deposited in GenBank (accession numbers shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p>                            <h3 scrollto-destination=6588598 id="6588598" class="section-title" data-legacy-id=s2c>2.3. DNA extraction and full-length genomic DNA identification</h3>
<p>Using the DNeasy Plant Mini kit (Qiagen China, Shanghai), total genomic DNA was extracted from fresh young leaves. For sequencing the full-length genomic DNA sequences, specific primers were designed based on each cDNA sequence. PCR amplification was performed according to Zhang <em>et al.</em><sup><a reveal-id="DSU040C22" data-open="DSU040C22" class="link link-ref link-reveal xref-bibr">22</a></sup> All the genomic DNA of candidate genes were obtained by direct sequencing of both strands, using conserved T7 and SP6 primers in the BigDye Terminator Cycle Sequencing Kit (version 3.1; Applied Biosystems, Foster City, CA, USA) and a 4300 DNA Analyzer (Li-Cor Biosciences, Lincoln, NE, USA). Full-length genomic DNA sequences of all 11 candidate genes were isolated by PCR amplification from the <em>P. tomentosa</em> LM50 clone (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Of these, five of these genes were previously reported in single-marker association studies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Also, these 11 full-length gene sequences were deposited in GenBank (accession numbers shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p>                            <h3 scrollto-destination=6588600 id="6588600" class="section-title" data-legacy-id=s2d>2.4. Tissue-specific expression analysis</h3>
<p>Fresh tissues (root, stem phloem, stem cambium, developing xylem, mature xylem, expanding leaf, expanded leaf, and apex) were sampled from 1-yr-old plants of <em>P. tomentosa</em> (clone <em>LM50</em>). Developing xylem tissues were collected by scraping the thin (∼1.0 mm) and partially ligniﬁed layer on the exposed xylem surface of main stems at the 1.0 m height of plant, and the liquid stem cambium was collected from the exposed xylem surface as described previously.<sup><a reveal-id="DSU040C21" data-open="DSU040C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The main root (5 cm in length), expanding leaf (the 2–3rd leaf from the stem top), expanded leaf (the 5–6th leaf), and the final shoot apex (2–3 mm in length) were immediately collected from 1-yr-old <em>P. tomentosa</em> clone ‘LM50’. The extraction of RNA from various fresh tissues and the reverse transcription into cDNA was used for testing of tissues-specific transcript abundance for all 11 candidate genes. Real-time quantitative PCR (RT–qPCR) was performed on a 7500 Fast Real-Time PCR System (ABI) using the LightCycler-FastStart DNA master SYBR Green I kit (Roche). The qPCR program and the real-time amplification reaction were performed as described by Du <em>et al.</em><sup><a reveal-id="DSU040C16" data-open="DSU040C16" class="link link-ref link-reveal xref-bibr">16</a></sup> All reactions were performed in triplicate technical and triplicate biological repetitions, respectively. The melting curve was used to check the specificity of the amplified fragments, and all data were analysed using the Opticon Monitor Analysis software 3.1 tool (Bio-Rad, USA), following the manual protocol. The results obtained for the different tissues were standardized to the levels of <em>Actin</em> gene (the internal control). The specific primer pairs were individually designed to target the 3′ UTR of each gene, using Primer Express 3.0 software (Applied Biosystems) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Mean value and standard deviation (<span class="small-caps">s.d.</span>) for expression data of all repetitions in each set was determined using SPSS Statistics Version 19.0 (SPSS Inc., Chicago, IL, USA) (Details not shown).</p><p>To better understand the co-expression interactions among these candidate genes, Pearson's correlations between each pair of genes were determined for all 11 genes, according to their tissue-specific transcript abundances in SPSS Statistics, version 19.0. A structural network representing significant gene–gene correlations was visualized using the NodeXL Excel Template, version 1.0.1.238 (<a class="link link-uri" href="http://nodexl.codeplex.com/">http://nodexl.codeplex.com/</a>).</p>                            <h3 scrollto-destination=6588603 id="6588603" class="section-title" data-legacy-id=s2e>2.5. SNP discovery and genotyping</h3>
<p>To identify SNPs, full-length genomic DNA of all 11 candidate genes were sequenced and aligned among 40 unrelated individuals from the association population of <em>P. tomentosa</em>, without considering insertions/deletions (INDELs). All 440 sequences were deposited in GenBank (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Sequence alignments and manual editing were performed as described by Du <em>et al.</em><sup><a reveal-id="DSU040C17" data-open="DSU040C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Next, all common SNPs (minor allele frequency ≥0.05) identified were genotyped using the Beckman Coulter (Franklin Lakes, NJ, USA) sequencing system.</p>                            <h3 scrollto-destination=6588605 id="6588605" class="section-title" data-legacy-id=s2f>2.6. Nucleotide diversity and LD</h3>
<p>Diploid sequences were phased into haplotypes with the Phase v. 2.1, using 10,000 iterations of the Bayesian Markov Chain Monte Carlo (MCMC) Model.<sup><a reveal-id="DSU040C23" data-open="DSU040C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We then used alignments of these 40 phased haplotypes for each gene to estimate the number of segregating sites, nucleotide diversity, neutrality tests, and the pattern of codon usage with DnaSP version 5.10.<sup><a reveal-id="DSU040C24" data-open="DSU040C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Nucleotide diversity was estimated using the average number of pairwise differences per site (<em>π</em>)<sup><a reveal-id="DSU040C25" data-open="DSU040C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSU040C26" data-open="DSU040C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and the number of segregating sites (<em>θw</em>).<sup><a reveal-id="DSU040C27" data-open="DSU040C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Neutrality test statistics, Tajima's D,<sup><a reveal-id="DSU040C28" data-open="DSU040C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and Fu and Li's D,<sup><a reveal-id="DSU040C29" data-open="DSU040C29" class="link link-ref link-reveal xref-bibr">29</a></sup> were calculated using data for the whole population with 10,000 coalescent simulations. The pattern of codon usage was measured using relative synonymous codon usage (RSCU).<sup><a reveal-id="DSU040C30" data-open="DSU040C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>The squared correlation of allele frequencies (<em>r</em><sup>2</sup>)<sup><a reveal-id="DSU040C31" data-open="DSU040C31" class="link link-ref link-reveal xref-bibr">31</a></sup> between each pair of common SNPs (frequency ≥ 0.05) within candidate genes was calculated with 10<sup>5</sup> permutations, using TASSEL Ver. 2.0.1 (<a class="link link-uri" href="http://www.maizegenetics.net/">http://www.maizegenetics.net/</a>). To assess the extent of LD within the sequenced gene regions, the decay of LD with physical distance (base pairs) within each candidate locus and over all candidate genes was estimated in 10,000 permutations of the data by non-linear regression.<sup><a reveal-id="DSU040C32" data-open="DSU040C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Singletons were excluded in the LD analyses. This analysis was done both within the three climatic regions and for the complete data set.</p>                            <h3 scrollto-destination=6588608 id="6588608" class="section-title" data-legacy-id=s2g>2.7. Association analysis</h3>
                            <h4 scrollto-destination=6588609 id="6588609" class="section-title" data-legacy-id=s2g1>2.7.1. Multi-SNP additive and dominance models</h4>
<p>fGWAS, version 2.0 (<a class="link link-uri" href="http://statgen.psu.edu/software/fgwas-soft.html">http://statgen.psu.edu/software/fgwas-soft.html</a>, Li <em>et al.</em><sup><a reveal-id="DSU040C33" data-open="DSU040C33" class="link link-ref link-reveal xref-bibr">33</a></sup>), in R (<a class="link link-uri" href="http://cran.r-project.org/">http://cran.r-project.org/</a>) was used to conduct multi-SNP association analyses. For each trait, the genotyping data for all common SNPs and their genomic positional information were identified and standardized using PLINK version 1.07 (<a class="link link-uri" href="http://pngu.mgh.harvard.edu/~purcell/plink/res.shtml">http://pngu.mgh.harvard.edu/~purcell/plink/res.shtml</a>). This software is a Bayesian hierarchical model and use a MCMC algorithm to simultaneously ﬁt and estimate the possible additive and dominant effects associated with all SNPs. The proportion of the phenotypic variance (<em>R</em><sup>2</sup>) explained by each particular SNP was estimated in the models. Positive dominance represents that the phenotypic mean value observed within the heterozygous class is higher than the average phenotypic mean across both homozygous classes based on Least Squares Means, whereas negative dominance was opposite (not a recessive lethal). To address the total phenotypic variance accounted for by all trait-associated SNPs on a trait-by-trait basis, we calculated a ‘cumulative <em>R</em><sup>2</sup>’ metric. These values were obtained by the difference in <em>R</em><sup>2</sup> between full and reduced models.<sup><a reveal-id="DSU040C34" data-open="DSU040C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The details were referenced according to McKown <em>et al.</em><sup><a reveal-id="DSU040C35" data-open="DSU040C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h4 scrollto-destination=6588611 id="6588611" class="section-title" data-legacy-id=s2g2>2.7.2. Multi-SNP epistasis models</h4>
<p>A multifactorial dimensionality reduction (MDR) method<sup><a reveal-id="DSU040C36" data-open="DSU040C36" class="link link-ref link-reveal xref-bibr">36</a></sup> that was designed specifically for detecting and characterizing non-additive interactions (i.e<em>.</em> epistasis). The ReliefF algorithm was applied to filter all unlinked SNPs (<em>r</em><sup>2</sup> &lt; 0.1 or different genes) to the best five loci for each trait that provide the greatest signal with 100 nearest neighbours. MDR was applied to assess all two-way to five-way models for the five significant filtered SNPs. Finally, an independent assessment of epistasis was performed using entropy-based measures of information gain.<sup><a reveal-id="DSU040C37" data-open="DSU040C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Specifically, a new measure model of three-way epistasis that adjusts for lower order effects was used to examine high-order non-additive interactions.<sup><a reveal-id="DSU040C38" data-open="DSU040C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p>                            <h4 scrollto-destination=6588613 id="6588613" class="section-title" data-legacy-id=s2g3>2.7.3. Haplotype-based association models</h4>
<p>All high-LD haplotypes (<em>r<sup>2</sup></em>≥ 0.70, <em>P</em>≤ 0.001) were estimated for each gene, and their frequencies were determined using HaploView version 4.2 (<a class="link link-uri" href="http://www.broad.mit.edu/mpg/haploview.html">http://www.broad.mit.edu/mpg/haploview.html</a>). Singleton alleles and haplotypes with a frequency &lt;5% were discarded when constructing the haplotypes. Haplotype-based association tests with phenotypic traits were performed using the Haplo.stats package in R,<sup><a reveal-id="DSU040C39" data-open="DSU040C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and significances of the haplotype associations were identified based on 10<sup>4</sup> permutation tests. The input consisted of genotype matrices, phenotype matrices, and structure analysis matrices (Q) to correct for population structure.<sup><a reveal-id="DSU040C40" data-open="DSU040C40" class="link link-ref link-reveal xref-bibr">40</a></sup> In this analysis, the Q matrix was identified according to the optimal subpopulation structure in our association population.<sup><a reveal-id="DSU040C41" data-open="DSU040C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Corrections for multiple testing of smoothed <em>P</em>-values for all associations were performed using the false discovery rate (FDR) through QVALUE.<sup><a reveal-id="DSU040C42" data-open="DSU040C42" class="link link-ref link-reveal xref-bibr">42</a></sup> A <em>q</em>-value of 0.10 was considered as the significance threshold.</p>                            <h4 scrollto-destination=6588615 id="6588615" class="section-title" data-legacy-id=s2g4>2.7.4. Single-marker linkage analysis</h4>
<p>On the basis of significant SNPs identified above, gene sequences were compared between hybrid parents, and SNP markers that segregated in the 1,200 F<sub>1</sub> progeny were selected. Inheritance tests of these SNPs were examined in the linkage population by performing a <em>χ</em><sup>2</sup> test at 0.01 probability, and SNPs following Mendelian expectations were used in single-marker analysis using PLINK version 1.07. The same FDR method was used to correct for multiple tests.<sup><a reveal-id="DSU040C42" data-open="DSU040C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h2 scrollto-destination=6588617 id="6588617" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6588618 id="6588618" class="section-title" data-legacy-id=s3a>3.1. Tissue-specific transcript profiling</h3>
<p>To provide insights into gene interactions and functions in the biological pathway, we examined transcript accumulation for 11 candidate genes in various tissues and organs of <em>P. tomentosa</em>, including root, stem, and leaf (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>A). We found that the 11 genes exhibited distinct, but partially overlapping patterns of expression. Most genes were preferentially expressed in the developing xylem and mature xylem, with moderate expression in the cambium region and expanded leaves (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>A). For example, in the stem of <em>P. tomentosa</em>, <em>Cellulose synthase 7</em> (<em>PtoCesA7</em>), <em>UDP-glucose dehydrogenase 1</em> (<em>PtoUGDH1</em>), and <em>S-adenosine-<span class="small-caps">l</span>-homocysteine-hydrolase 1</em> (<em>PtoSAHH1</em>) transcripts were most abundant in the developing and mature xylems, with moderate abundance in the cambium<em>. PtoCesA7 and PtoUGDH1</em> were also preferentially expressed in roots (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>A), whereas <em>PtoCesA3</em> and <em>UDP-glucuronate decarboxylase 1</em> (<em>PtoUXS1</em>) were highly expressed in the apex (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>A). <em>PtoUXS1</em> were most abundant in expanded leaves. </p>                        <a id="6588620" scrollto-destination="6588620"></a>
<div id="DSU040F1" data-id="DSU040F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04001.png?Expires=1509238978&Signature=GfefY9WdpAo3QSYYE9DAajWP7uHhU7ER9IkSFnjEL8yqhgO1YjnwUMcbxmTJ-iSfkyIz1Fw8y0LV8bfYn9aJNlUAKhvXXzbUw0yIUm~aDlUk3bdK0O1VD9XbPqjYK5GPX7j609XQofh-ginEYWLH2FwPakqUq-PE5nzTE8NN0A-WXdN~TGFIQsibh1x8hXqMd437IqNChMrd~2pwhg2m0PYPb1mMA2d1PfWXLn~FM0sv3S5VFcg-aBL1-Voeq2b3TpAJarmyM-E~L-pnhHt-IsfljZUv4M5bf-ftKFL981EbamHG3Rm3y9Ps2uWYJLBINszY~-XnBh40cwbBKGvJzA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Tissue-specific transcript profiling and a correlation network of 11 candidate genes in Populus tomentosa. (A) Relative transcript levels of 11 candidate genes in different tissues and organs of P. tomentosa. Expression levels were normalized to the Actin gene; Apex, apical shoot meristem, the error bars represent ±standard deviation (SD). (B) A gene–gene correlation network was constructed, on the basis of comparison and correlation analyses of transcription abundances among these genes. Solid lines represent positive correlations and dashed lines represent negative correlations; the thickness of each line indicates the strength/significance of correlations, P &lt; 0.05 level of significance (thin lines), P &lt; 0.01 (thick lines). Data were log-transformed before Pearson's correlation." path-from-xml="dsu04001.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588620/dsu04001.tif" path-from-xml="dsu04001.tif" target="_blank">View large</a><a section="6588620" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04001.png?Expires=1509238978&Signature=eTrfXDlYygCwpRnNyhQw5dCV7dEmeppVHmcTEtLv0dBTAy72kkIjVkpCKCwOE8JrwKp-VF3dU2Ph16niUjF5F63k2w4BrcH6sC85dbI1jQ5HadEjYH-6rcpQaGPt4IDKWzPMKp~UtjjeHIarCW7MJuAc988qYnthlTBQz7D1d7095Y3cDhMidW5yDRJYtdztvmCPaGqaX21vfM8PwHgRhfJRmNqoAYYQBH3oqjXRRJyIiahMCuLZXixWYnVfSmYW2ujDE2NzsNEKnB4xpwqqoQzdZbSe18GX6rki4mtdFbU8lg-a6jSFA9AJdbj9s58cyO3J0eVEgkc8jLzGavD9Ng__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588620&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04001.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tissue-specific transcript profiling and a correlation network of 11 candidate genes in <em>Populus tomentosa</em>. (A) Relative transcript levels of 11 candidate genes in different tissues and organs of <em>P. tomentosa</em>. Expression levels were normalized to the <em>Actin</em> gene; Apex, apical shoot meristem, the error bars represent ±standard deviation (SD). (B) A gene–gene correlation network was constructed, on the basis of comparison and correlation analyses of transcription abundances among these genes. Solid lines represent positive correlations and dashed lines represent negative correlations; the thickness of each line indicates the strength/significance of correlations, <em>P</em> &lt; 0.05 level of significance (thin lines), <em>P</em> &lt; 0.01 (thick lines). Data were log-transformed before Pearson's correlation.</p></div></div><div content-id="DSU040F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04001.png?Expires=1509238978&Signature=GfefY9WdpAo3QSYYE9DAajWP7uHhU7ER9IkSFnjEL8yqhgO1YjnwUMcbxmTJ-iSfkyIz1Fw8y0LV8bfYn9aJNlUAKhvXXzbUw0yIUm~aDlUk3bdK0O1VD9XbPqjYK5GPX7j609XQofh-ginEYWLH2FwPakqUq-PE5nzTE8NN0A-WXdN~TGFIQsibh1x8hXqMd437IqNChMrd~2pwhg2m0PYPb1mMA2d1PfWXLn~FM0sv3S5VFcg-aBL1-Voeq2b3TpAJarmyM-E~L-pnhHt-IsfljZUv4M5bf-ftKFL981EbamHG3Rm3y9Ps2uWYJLBINszY~-XnBh40cwbBKGvJzA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Tissue-specific transcript profiling and a correlation network of 11 candidate genes in Populus tomentosa. (A) Relative transcript levels of 11 candidate genes in different tissues and organs of P. tomentosa. Expression levels were normalized to the Actin gene; Apex, apical shoot meristem, the error bars represent ±standard deviation (SD). (B) A gene–gene correlation network was constructed, on the basis of comparison and correlation analyses of transcription abundances among these genes. Solid lines represent positive correlations and dashed lines represent negative correlations; the thickness of each line indicates the strength/significance of correlations, P &lt; 0.05 level of significance (thin lines), P &lt; 0.01 (thick lines). Data were log-transformed before Pearson's correlation." path-from-xml="dsu04001.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588620/dsu04001.tif" path-from-xml="dsu04001.tif" target="_blank">View large</a><a section="6588620" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04001.png?Expires=1509238978&Signature=eTrfXDlYygCwpRnNyhQw5dCV7dEmeppVHmcTEtLv0dBTAy72kkIjVkpCKCwOE8JrwKp-VF3dU2Ph16niUjF5F63k2w4BrcH6sC85dbI1jQ5HadEjYH-6rcpQaGPt4IDKWzPMKp~UtjjeHIarCW7MJuAc988qYnthlTBQz7D1d7095Y3cDhMidW5yDRJYtdztvmCPaGqaX21vfM8PwHgRhfJRmNqoAYYQBH3oqjXRRJyIiahMCuLZXixWYnVfSmYW2ujDE2NzsNEKnB4xpwqqoQzdZbSe18GX6rki4mtdFbU8lg-a6jSFA9AJdbj9s58cyO3J0eVEgkc8jLzGavD9Ng__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588620&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04001.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Tissue-specific transcript profiling and a correlation network of 11 candidate genes in <em>Populus tomentosa</em>. (A) Relative transcript levels of 11 candidate genes in different tissues and organs of <em>P. tomentosa</em>. Expression levels were normalized to the <em>Actin</em> gene; Apex, apical shoot meristem, the error bars represent ±standard deviation (SD). (B) A gene–gene correlation network was constructed, on the basis of comparison and correlation analyses of transcription abundances among these genes. Solid lines represent positive correlations and dashed lines represent negative correlations; the thickness of each line indicates the strength/significance of correlations, <em>P</em> &lt; 0.05 level of significance (thin lines), <em>P</em> &lt; 0.01 (thick lines). Data were log-transformed before Pearson's correlation.</p></div></div><p>Next, we constructed gene–gene correlation networks using the patterns of tissue-specific expression of these 11 candidate genes and identified 20 positive or negative gene–gene correlations (<em>P</em> ≤ 0.05, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>) that make up a highly interrelated network (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>B). Of these gene pairs, <em>PtoCesA3</em> and <em>Sucrose synthase 1</em> (<em>PtoSUS1</em>) (<em>P</em> ≤ 0.01, <em>R</em> = −0.938), <em>PtoSAHH1</em> and <em>PtoSAHH2</em> (<em>P</em> ≤ 0.01, <em>R</em> = 0.959), and <em>PtoCesA4</em> and <em>Endo-1,4-β-glucanase 1</em> (<em>PtoGH9A1</em>) (<em>P</em> ≤ 0.01, <em>R</em> = 0.952) showed highly significant correlations (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p>                            <h3 scrollto-destination=6588622 id="6588622" class="section-title" data-legacy-id=s3b>3.2. SNP discovery and nucleotide diversity</h3>
<p>We first used a re-sequencing approach to identify SNPs in all 11 candidate genes. By sequencing the 11 full-length genes from 40 unrelated individuals encompassing nearly the entire natural range of <em>P. tomentosa</em>, we identified 2,114 SNPs in the <em>c.</em> 47,300 bp sequenced, with an average density of one SNP per 25 bp. Of these SNPs, <em>c.</em> 44.0% (930) were common sites (frequency ≥ 0.05) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The SNPs were not evenly distributed, with the number of SNPs per gene ranging from 59 to 388 (average, 192), consistent with varying levels of nucleotide diversity among genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Using sequences from the <em>P. trichocarpa</em> reference as an out-group, a survey of the single-base mutation types indicated that C : T changes were the most abundant (38.6%), followed by A : G changes (30.4%), and overall SNPs the ratio of transitions to transversions was 2.23.</p><p>Within the coding regions of all 11 genes (22,098 bp total sequence), the average non-synonymous nucleotide diversity (<em>d<sub>N</sub></em>) was ∼5-fold lower than the synonymous nucleotide diversity (<em>d<sub>S</sub></em>), and the <em>d<sub>N</sub>/d<sub>S</sub></em> ratio was &lt;1 for all exons, indicating strong purifying selection at non-synonymous sites at these <em>Populus</em> genes.<sup><a reveal-id="DSU040C22" data-open="DSU040C22" class="link link-ref link-reveal xref-bibr">22</a></sup> To investigate the extent of codon bias in <em>P. tomentosa</em>, we calculated RSCU values for all types of codons and found that the 11 candidate genes show similar patterns of RSCU (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Most of the preferred codons end with U, but A and G were also less commonly found in the third position in these genes. Previous studies indicated that codon bias (with C or G ending codons) is expected to be strongest for highly expressed genes.<sup><a reveal-id="DSU040C34" data-open="DSU040C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Thus, our observation may be consistent with low expressed values for all these genes (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>A). More work is needed to sort these issues out in <em>P. tomentosa</em>. The values of Fu and Li's <em>D</em> statistical tests were negative for all genes, with four significant departures observed in the whole population (<em>P</em>≤ 0.05, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>), revealing an excess of low-frequency polymorphisms in the species-wide samples.</p>                            <h3 scrollto-destination=6588625 id="6588625" class="section-title" data-legacy-id=s3c>3.3. SNP genotyping and LD</h3>
<p>We selected a set of 651 common SNPs that met the quality control thresholds and that had genotype frequencies consistent with Hardy–Weinberg equilibrium (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S4</a>). The percentage of successfully genotyped SNPs that were found in non-coding regions of the genes was 75.4%, distributed within introns (45.4%), 5′ UTRs (16.9%), and 3′ UTRs (13.1%). Remaining SNPs were located in the coding regions, either at synonymous sites (18.5%) or at non-synonymous sites (6.1%).</p><p>Estimates of <em>r</em><sup>2</sup> values for all pairwise combinations of SNPs were pooled to assess the overall pattern of LD with physical distance (Fig. <a reveal-id="DSU040F2" data-open="DSU040F2" class="link link-reveal link-table xref-fig">2</a>). The non-linear regression shows a clear and rapid decline of LD from 0.60 to 0.10 at a distance of ∼1,100 bp in the whole population (Fig. <a reveal-id="DSU040F2" data-open="DSU040F2" class="link link-reveal link-table xref-fig">2</a>). Nevertheless, LD analyses within each geographical climatic region showed a much higher level of LD with the <em>r</em><sup>2</sup> values declining to 0.1 within <em>c.</em> 2,300 bp (Southern and Northeastern regions) and <em>c.</em> 3,400 bp (Northwestern region) (Fig. <a reveal-id="DSU040F2" data-open="DSU040F2" class="link link-reveal link-table xref-fig">2</a>). We found a clear and rapid decline of LD with distance within each gene in the whole population, respectively (<em>r<sup>2</sup></em> ≥ 0.1, within <em>c.</em> 500–1,800 bp, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>), indicating that LD does not extend over entire gene in the species. </p>                        <a id="6588628" scrollto-destination="6588628"></a>
<div id="DSU040F2" data-id="DSU040F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04002.png?Expires=1509238978&Signature=E7-nFYedIn4GHJjIAGQDrdRCAlV4iyFiSogT5XDYP6SGtB7R04ovwhHCgOK~x8PdWPBQyiG6XNoL035kHDuYaiYEFRZcQz0svTe1lQ9gv10JWNdqj1qgOWokSITSzmWj4szOt6-Yh9ZJ~R6KR~ONqhXT7BmbqGKKYuVJ3sqy7BSu~o53EfzshCoEP0kVBeilRHkmePCEZ1hlUtQFVbJn8Qg~4~RYafwHh5I0pEKILeOsZuPYA3ZIXAGn7elX6jIqgWQm9Jj1xJiLd~con7unGuPu42BTZfpt5BqbKDKP5vaFvf300dDTk-SFHP-UBVupUKn63ac05hdmWDXuC-pawA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Decay pattern of LD in all Populus tomentosa samples and each climatic region. Decay of LD for all common SNP (minor allele frequency ≥ 5%) sites pooled across all analysed genes. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the non-linear regressions of r2 (Er2) onto the physical distance in base pairs. The details of three climatic regions are shown in Supplementary Table S1." path-from-xml="dsu04002.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588628/dsu04002.tif" path-from-xml="dsu04002.tif" target="_blank">View large</a><a section="6588628" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04002.png?Expires=1509238978&Signature=HKxe1lfa8X-lSXf4w2d~aivInyRn9H5rAzITaem6WeWnM3QJqhpgfZ9iUOje2LRDKGLqqHY2oDZ1lJXBUzi1ZgL5dbf2rWHm3CiJ89GAhqm6zsKfWDtjCsO0vqzTeiu6XGgVU5qCnj~K7ucOj0OWVp5P53XkgP9Vy7MWOKIcKHXDvfGouXbU2U~q685fkFWiVQObIiZCBJY-m75nT8SgO~pWAYr1TL2MFOSQ9L103hglyVQDCd~PH0rzP~mf7Db5~jyS-xxfKANzuZKwjH-GxJHf9jrnD0UQr1hTPrZrkCph1oyi4BcKZrWA6W3xwnYPP2CfrHl12lDoASQ-OKRGnw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588628&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04002.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Decay pattern of LD in all <em>Populus tomentosa</em> samples and each climatic region. Decay of LD for all common SNP (minor allele frequency ≥ 5%) sites pooled across all analysed genes. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the non-linear regressions of <em>r</em><sup>2</sup> (Er2) onto the physical distance in base pairs. The details of three climatic regions are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p></div></div><div content-id="DSU040F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04002.png?Expires=1509238978&Signature=E7-nFYedIn4GHJjIAGQDrdRCAlV4iyFiSogT5XDYP6SGtB7R04ovwhHCgOK~x8PdWPBQyiG6XNoL035kHDuYaiYEFRZcQz0svTe1lQ9gv10JWNdqj1qgOWokSITSzmWj4szOt6-Yh9ZJ~R6KR~ONqhXT7BmbqGKKYuVJ3sqy7BSu~o53EfzshCoEP0kVBeilRHkmePCEZ1hlUtQFVbJn8Qg~4~RYafwHh5I0pEKILeOsZuPYA3ZIXAGn7elX6jIqgWQm9Jj1xJiLd~con7unGuPu42BTZfpt5BqbKDKP5vaFvf300dDTk-SFHP-UBVupUKn63ac05hdmWDXuC-pawA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Decay pattern of LD in all Populus tomentosa samples and each climatic region. Decay of LD for all common SNP (minor allele frequency ≥ 5%) sites pooled across all analysed genes. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the non-linear regressions of r2 (Er2) onto the physical distance in base pairs. The details of three climatic regions are shown in Supplementary Table S1." path-from-xml="dsu04002.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588628/dsu04002.tif" path-from-xml="dsu04002.tif" target="_blank">View large</a><a section="6588628" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04002.png?Expires=1509238978&Signature=HKxe1lfa8X-lSXf4w2d~aivInyRn9H5rAzITaem6WeWnM3QJqhpgfZ9iUOje2LRDKGLqqHY2oDZ1lJXBUzi1ZgL5dbf2rWHm3CiJ89GAhqm6zsKfWDtjCsO0vqzTeiu6XGgVU5qCnj~K7ucOj0OWVp5P53XkgP9Vy7MWOKIcKHXDvfGouXbU2U~q685fkFWiVQObIiZCBJY-m75nT8SgO~pWAYr1TL2MFOSQ9L103hglyVQDCd~PH0rzP~mf7Db5~jyS-xxfKANzuZKwjH-GxJHf9jrnD0UQr1hTPrZrkCph1oyi4BcKZrWA6W3xwnYPP2CfrHl12lDoASQ-OKRGnw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588628&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04002.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Decay pattern of LD in all <em>Populus tomentosa</em> samples and each climatic region. Decay of LD for all common SNP (minor allele frequency ≥ 5%) sites pooled across all analysed genes. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the non-linear regressions of <em>r</em><sup>2</sup> (Er2) onto the physical distance in base pairs. The details of three climatic regions are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p></div></div>                            <h3 scrollto-destination=6588629 id="6588629" class="section-title" data-legacy-id=s3d>3.4. Multi-SNP association under additive, dominance, and epistasis models</h3>
<p>We first employed Bayesian hierarchical models, emphasizing multi-SNP additive and dominant effects for each quantitative trait, and uncovered a multitude of genetic associations, including some not previously found by single-marker methods (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Some particular SNP-trait associations detected previously were examined in this case; for example, a non-synonymous substitution in exon3 of <em>PtoCesA4</em> (PtoCesA4_1914) was shown to be in strong association with α-cellulose.<sup><a reveal-id="DSU040C17" data-open="DSU040C17" class="link link-ref link-reveal xref-bibr">17</a></sup> We then constructed a structural network to visually represent all 123 associations (<em>P</em> &lt; 0.01) between all 10 growth and wood-property traits and 93 unique SNPs (Fig. <a reveal-id="DSU040F3" data-open="DSU040F3" class="link link-reveal link-table xref-fig">3</a>). The 93 loci were not evenly distributed among all 11 genes, with a range from 1 (<em>PtoBGlu1, β-1,4-glucosidase</em>) to 19 (<em>PtoUXS1</em>). 37.6% of the 93 loci were distributed in coding regions, and among these, 94.3% of the associations contained a combination of additive and dominant effects (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). </p>                        <a id="6588632" scrollto-destination="6588632"></a>
<div id="DSU040F3" data-id="DSU040F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04003.png?Expires=1509238978&Signature=KNHInMDv2wHeVMM-Gt7ir-UeQlarXQIkZfSRriOVnMS1SNLCrlBAcGbz9GPNZHg3413ld2ktUYcvFsKpO~d4ZwYS2WfmLBP7aWNvfZH3vqCxj27JmM2JmOozNtOzIOzxouy5GjDmcplzNhnrzEh7FvUAznZGTU1VMuW94jaHszNCDTzWcZKba66SCggiErFPsZJ0sCf89vKnRyrvoLx~KIDUdfELP~hkDkH~DJvHPufwRK8345MuO6sZgvR3nh5Ts~1lvJ5zv6jFmVR-6HuelZL7xLZoZHVG09GN6htVx29mTiPSfWV1mHlJ-mWqb~6dqfjE9A1hDEFWVlD0uRAQ7A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="A structural network that represents all significant associations (P &lt; 0.01) in the association population of Populus tomentosa. All associations were identified between 10 growth and wood-property traits and 93 unique SNPs from 11 candidate genes using Bayesian hierarchical models. Different colours represent different genes with corresponding SNPs. The number of SNPs identified in each gene is shown in parentheses." path-from-xml="dsu04003.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588632/dsu04003.tif" path-from-xml="dsu04003.tif" target="_blank">View large</a><a section="6588632" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04003.png?Expires=1509238978&Signature=LUzpD4jxyIHqwXEBLVF4Noeo7ZL9FwJNk4P5Li8pOXNYmarw5SHNXMJijSGJWlssWw4ky43JBB~~eTLlYlZwl-5ZG0riuM3UixqwISb5Y8YgNXWCI-CEjOFDgQ1U1gB-p3B7PDUIjSoxpx~Zmo2kAWSSGHudUXn~cCZXOqQ1sPcGZyXhzOoN7s8aRNhfpKRo2Njc3WUAF6RwYXjfCvBE8aI-CmXM8FUwoeM2YWfBU42ZoxxOx14iCI9BPKYtDJOUSMa2V8lZac1~cf1RSYmZqIlswoYRkkgXmkroDCOv6ep2syjn5I3fONpz9MOhlsXQeO9WdWzaTZQV6JlVqgUTmQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588632&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04003.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A structural network that represents all significant associations (<em>P</em> &lt; 0.01) in the association population of <em>Populus tomentosa</em>. All associations were identified between 10 growth and wood-property traits and 93 unique SNPs from 11 candidate genes using Bayesian hierarchical models. Different colours represent different genes with corresponding SNPs. The number of SNPs identified in each gene is shown in parentheses.</p></div></div><div content-id="DSU040F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04003.png?Expires=1509238978&Signature=KNHInMDv2wHeVMM-Gt7ir-UeQlarXQIkZfSRriOVnMS1SNLCrlBAcGbz9GPNZHg3413ld2ktUYcvFsKpO~d4ZwYS2WfmLBP7aWNvfZH3vqCxj27JmM2JmOozNtOzIOzxouy5GjDmcplzNhnrzEh7FvUAznZGTU1VMuW94jaHszNCDTzWcZKba66SCggiErFPsZJ0sCf89vKnRyrvoLx~KIDUdfELP~hkDkH~DJvHPufwRK8345MuO6sZgvR3nh5Ts~1lvJ5zv6jFmVR-6HuelZL7xLZoZHVG09GN6htVx29mTiPSfWV1mHlJ-mWqb~6dqfjE9A1hDEFWVlD0uRAQ7A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="A structural network that represents all significant associations (P &lt; 0.01) in the association population of Populus tomentosa. All associations were identified between 10 growth and wood-property traits and 93 unique SNPs from 11 candidate genes using Bayesian hierarchical models. Different colours represent different genes with corresponding SNPs. The number of SNPs identified in each gene is shown in parentheses." path-from-xml="dsu04003.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588632/dsu04003.tif" path-from-xml="dsu04003.tif" target="_blank">View large</a><a section="6588632" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04003.png?Expires=1509238978&Signature=LUzpD4jxyIHqwXEBLVF4Noeo7ZL9FwJNk4P5Li8pOXNYmarw5SHNXMJijSGJWlssWw4ky43JBB~~eTLlYlZwl-5ZG0riuM3UixqwISb5Y8YgNXWCI-CEjOFDgQ1U1gB-p3B7PDUIjSoxpx~Zmo2kAWSSGHudUXn~cCZXOqQ1sPcGZyXhzOoN7s8aRNhfpKRo2Njc3WUAF6RwYXjfCvBE8aI-CmXM8FUwoeM2YWfBU42ZoxxOx14iCI9BPKYtDJOUSMa2V8lZac1~cf1RSYmZqIlswoYRkkgXmkroDCOv6ep2syjn5I3fONpz9MOhlsXQeO9WdWzaTZQV6JlVqgUTmQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588632&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04003.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A structural network that represents all significant associations (<em>P</em> &lt; 0.01) in the association population of <em>Populus tomentosa</em>. All associations were identified between 10 growth and wood-property traits and 93 unique SNPs from 11 candidate genes using Bayesian hierarchical models. Different colours represent different genes with corresponding SNPs. The number of SNPs identified in each gene is shown in parentheses.</p></div></div>                            <h4 scrollto-destination=6588633 id="6588633" class="section-title" data-legacy-id=s3d1>3.4.1. Multi-gene associations under the additive effect model</h4>
<p>We detected 118 significant associations for a total of 92 unique SNPs within 11 genes associated with all 10 traits (Table <a reveal-id="DSU040TB1" data-open="DSU040TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S6</a>). Total numbers of identified SNP-trait associations varied across trait categories, and the association numbers of associations for wood chemical compositions, wood physical properties, and growth traits were 40, 38, and 40, respectively. SNP markers explained between 0.5 and 6.6% of the phenotypic variation (average <em>R</em><sup>2</sup> = 2.3%; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Twenty-four of the 92 SNP markers exhibited significant associations with at least two traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Although the association result does not indicate which SNPs may be causal, 11 of these 92 unique SNPs represent amino acid replacement (non-synonymous) polymorphisms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). For each trait, the number of significant SNPs ranged from 8 to 16, which captured a small to moderate fraction (17.0–36.0%) of the additive variation when considered jointly (Table <a reveal-id="DSU040TB1" data-open="DSU040TB1" class="link link-reveal link-table xref-fig">1</a>, Fig. <a reveal-id="DSU040F4" data-open="DSU040F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="6588635" scrollto-destination="6588635"></a>
<div class="table-wrap"><div id="DSU040TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of the additive effect and phenotypic contribution rate (<em>R</em><sup>2</sup>) of all significant SNPs for each trait in the <em>Populus tomentosa</em> association population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Number of candidate genes </th><th>Number of SNPs </th><th>Range of additive effect (%) </th><th>Range of <em>R</em><sup>2</sup> (%) </th><th>Total <em>R</em><sup>2</sup> (%)<sup>a</sup> </th></tr></thead><tbody><tr><td>Lignin (%) </td><td>6 </td><td>8 </td><td>0.477–2.391 </td><td>0.8–5.5 </td><td>21.9 </td></tr><tr><td>Holocellulose (%) </td><td>6 </td><td>12 </td><td>0.472–3.876 </td><td>0.7–4.1 </td><td>28.1 </td></tr><tr><td>Hemicellulose (%) </td><td>5 </td><td>8 </td><td>0.263–1.876 </td><td>0.9–3.1 </td><td>17.0 </td></tr><tr><td>α-Cellulose (%) </td><td>7 </td><td>12 </td><td>0.833–3.228 </td><td>0.5–5.7 </td><td>31.7 </td></tr><tr><td>Fibre length (mm) </td><td>7 </td><td>13 </td><td>0.007–0.143 </td><td>0.9–6.6 </td><td>36.0 </td></tr><tr><td>Fibre width (µm) </td><td>5 </td><td>9 </td><td>0.150–1.458 </td><td>0.8–6.1 </td><td>24.9 </td></tr><tr><td>Microﬁbre angle (MFA, °) </td><td>7 </td><td>16 </td><td>0.073–2.535 </td><td>0.6–3.6 </td><td>30.0 </td></tr><tr><td>Diameter at breast height (DBH, cm) </td><td>5 </td><td>11 </td><td>0.372–4.788 </td><td>0.5–3.2 </td><td>24.0 </td></tr><tr><td>Tree height (m) </td><td>8 </td><td>16 </td><td>0.192–1.621 </td><td>0.8–3.5 </td><td>33.1 </td></tr><tr><td>Stem volume (m<sup>3</sup>) </td><td>5 </td><td>13 </td><td>0.074–0.902 </td><td>0.7–4.0 </td><td>25.0 </td></tr></tbody></table></div><div content-id="DSU040TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Number of candidate genes </th><th>Number of SNPs </th><th>Range of additive effect (%) </th><th>Range of <em>R</em><sup>2</sup> (%) </th><th>Total <em>R</em><sup>2</sup> (%)<sup>a</sup> </th></tr></thead><tbody><tr><td>Lignin (%) </td><td>6 </td><td>8 </td><td>0.477–2.391 </td><td>0.8–5.5 </td><td>21.9 </td></tr><tr><td>Holocellulose (%) </td><td>6 </td><td>12 </td><td>0.472–3.876 </td><td>0.7–4.1 </td><td>28.1 </td></tr><tr><td>Hemicellulose (%) </td><td>5 </td><td>8 </td><td>0.263–1.876 </td><td>0.9–3.1 </td><td>17.0 </td></tr><tr><td>α-Cellulose (%) </td><td>7 </td><td>12 </td><td>0.833–3.228 </td><td>0.5–5.7 </td><td>31.7 </td></tr><tr><td>Fibre length (mm) </td><td>7 </td><td>13 </td><td>0.007–0.143 </td><td>0.9–6.6 </td><td>36.0 </td></tr><tr><td>Fibre width (µm) </td><td>5 </td><td>9 </td><td>0.150–1.458 </td><td>0.8–6.1 </td><td>24.9 </td></tr><tr><td>Microﬁbre angle (MFA, °) </td><td>7 </td><td>16 </td><td>0.073–2.535 </td><td>0.6–3.6 </td><td>30.0 </td></tr><tr><td>Diameter at breast height (DBH, cm) </td><td>5 </td><td>11 </td><td>0.372–4.788 </td><td>0.5–3.2 </td><td>24.0 </td></tr><tr><td>Tree height (m) </td><td>8 </td><td>16 </td><td>0.192–1.621 </td><td>0.8–3.5 </td><td>33.1 </td></tr><tr><td>Stem volume (m<sup>3</sup>) </td><td>5 </td><td>13 </td><td>0.074–0.902 </td><td>0.7–4.0 </td><td>25.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a> for further details of these association results under additive models; significance level for association (significance is <em>P</em> ≤ 0.01).</p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">a</div><p>Total <em>R</em><sup>2</sup> was calculated according to McKown <em>et al.</em><sup><a reveal-id="DSU040C35" data-open="DSU040C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6588635" target="_blank">View Large</a></div></div>                        <a id="6588636" scrollto-destination="6588636"></a>
<div id="DSU040F4" data-id="DSU040F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04004.png?Expires=1509238978&Signature=YXX9hBq6AyF~amsuQVomma5jXXcGSxaVE1AicVVAET6XkSsK71ivYMPWpPZRC11ozE8~QJF6khjnosx8qligLaY3Wr1XJyKCBpAB2NPZccevP7HgUVbTKc~VSBvjcvIF6W2FHzKaR0QYC~P7Pqqay4gMZ4lSlsM7U1GSdHiMaFnKqFafnCO7SG4WC-hu1DFyiEFkz7Y0ARaAhIKzdUP8FsYkV~vDMenyOk0-0~F6Ken198eH3Bt0DSXPd8vnyEKzEHPjj1MG~KnhFc6LW121Xa29JlqgUODqSyBK~a2WTIOE5RsSrT4XVD5PjBx28I66YNUtsWvZjSK1Ib0sUeo52Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Low-to-moderate phenotypic contribution rate (R2) explained for growth and wood properties in Populus tomentosa. The line and bar denote the numbers and phenotypic contribution rate (R2) of the list of SNPs identified for each trait under additive and dominant models, respectively. All SNPs were identified using the Bayesian hierarchical models in fGWAS, version 1.0 (http://statgen.psu.edu/software/fgwas-soft.html) in R (http://cran.r-project.org/)." path-from-xml="dsu04004.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588636/dsu04004.tif" path-from-xml="dsu04004.tif" target="_blank">View large</a><a section="6588636" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04004.png?Expires=1509238978&Signature=hFG3F6t15iKLdflLtEkEJTaLCC2Ng-ef3pw0ushDRX1hIPkrk90LQK74CkkpEhpvZoy1AJGJGOxB53w2lws5z890F8JE-IuxaJ2k0ys1A8ZTDBv0VbikJ09p-~rvz4Yy1ejh8soCX5GzABwTYiDZFI6xyQIuivmvp9HZRkVH0P9XsWiQO6VQMtrz3HwZ2eXTe9Ad18Qnfv9yY47B2VPOuTEy6ubAcUmeeDZRCjMy17~aevtTzjqcBZi-qBdNvueoJIbPqYXVtP-14rzCfFWW2JAsvcgF-Ww040MfivcPsKmKxbotCgf3KYEz7K0Vxvblf-mAp4WdH48maFrtrqOjHQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588636&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04004.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Low-to-moderate phenotypic contribution rate (<em>R</em><sup>2</sup>) explained for growth and wood properties in <em>Populus tomentosa</em>. The line and bar denote the numbers and phenotypic contribution rate (<em>R</em><sup>2</sup>) of the list of SNPs identified for each trait under additive and dominant models, respectively. All SNPs were identified using the Bayesian hierarchical models in fGWAS, version 1.0 (<a class="link link-uri" href="http://statgen.psu.edu/software/fgwas-soft.html">http://statgen.psu.edu/software/fgwas-soft.html</a>) in R (<a class="link link-uri" href="http://cran.r-project.org/">http://cran.r-project.org/</a>).</p></div></div><div content-id="DSU040F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04004.png?Expires=1509238978&Signature=YXX9hBq6AyF~amsuQVomma5jXXcGSxaVE1AicVVAET6XkSsK71ivYMPWpPZRC11ozE8~QJF6khjnosx8qligLaY3Wr1XJyKCBpAB2NPZccevP7HgUVbTKc~VSBvjcvIF6W2FHzKaR0QYC~P7Pqqay4gMZ4lSlsM7U1GSdHiMaFnKqFafnCO7SG4WC-hu1DFyiEFkz7Y0ARaAhIKzdUP8FsYkV~vDMenyOk0-0~F6Ken198eH3Bt0DSXPd8vnyEKzEHPjj1MG~KnhFc6LW121Xa29JlqgUODqSyBK~a2WTIOE5RsSrT4XVD5PjBx28I66YNUtsWvZjSK1Ib0sUeo52Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Low-to-moderate phenotypic contribution rate (R2) explained for growth and wood properties in Populus tomentosa. The line and bar denote the numbers and phenotypic contribution rate (R2) of the list of SNPs identified for each trait under additive and dominant models, respectively. All SNPs were identified using the Bayesian hierarchical models in fGWAS, version 1.0 (http://statgen.psu.edu/software/fgwas-soft.html) in R (http://cran.r-project.org/)." path-from-xml="dsu04004.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588636/dsu04004.tif" path-from-xml="dsu04004.tif" target="_blank">View large</a><a section="6588636" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04004.png?Expires=1509238978&Signature=hFG3F6t15iKLdflLtEkEJTaLCC2Ng-ef3pw0ushDRX1hIPkrk90LQK74CkkpEhpvZoy1AJGJGOxB53w2lws5z890F8JE-IuxaJ2k0ys1A8ZTDBv0VbikJ09p-~rvz4Yy1ejh8soCX5GzABwTYiDZFI6xyQIuivmvp9HZRkVH0P9XsWiQO6VQMtrz3HwZ2eXTe9Ad18Qnfv9yY47B2VPOuTEy6ubAcUmeeDZRCjMy17~aevtTzjqcBZi-qBdNvueoJIbPqYXVtP-14rzCfFWW2JAsvcgF-Ww040MfivcPsKmKxbotCgf3KYEz7K0Vxvblf-mAp4WdH48maFrtrqOjHQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588636&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04004.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Low-to-moderate phenotypic contribution rate (<em>R</em><sup>2</sup>) explained for growth and wood properties in <em>Populus tomentosa</em>. The line and bar denote the numbers and phenotypic contribution rate (<em>R</em><sup>2</sup>) of the list of SNPs identified for each trait under additive and dominant models, respectively. All SNPs were identified using the Bayesian hierarchical models in fGWAS, version 1.0 (<a class="link link-uri" href="http://statgen.psu.edu/software/fgwas-soft.html">http://statgen.psu.edu/software/fgwas-soft.html</a>) in R (<a class="link link-uri" href="http://cran.r-project.org/">http://cran.r-project.org/</a>).</p></div></div><p>Correspondingly, each trait was associated with variation in at least five candidate genes (Table <a reveal-id="DSU040TB1" data-open="DSU040TB1" class="link link-reveal link-table xref-fig">1</a>). Furthermore, we identified three genes (<em>PtoCesA4</em>, <em>PtoCesA7</em>, and <em>PtoSAHH2</em>) that were associated to all four wood chemical compositions. (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). For wood physical properties, SNP-trait associations were identified for four genes (<em>PtoCesA3</em>, <em>PtoCesA4</em>, <em>PtoSUS1</em>, and <em>PtoUXS1</em>) that were associated with all three traits. In addition, <em>PtoCesA7</em> and <em>PtoSAHH2</em> were associated with fibre length and microfibre angle (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Additive models found nine genes with SNPs associated with variation in growth traits; of these, two genes (<em>PtoCesA3</em> and <em>PtoUXS1</em>) with SNPs associated across all three growth traits, as well as identified with all three wood physical properties (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6</a>). At least five SNPs within <em>PtoUXS1</em> had associations to each growth trait. No significant SNP from <em>PtoUGDH</em> and <em>PtoBGlu1</em> was associated with growth traits.</p><p>On the basis of these additive SNPs simultaneously associated to each trait (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>), we identified phenotypic variation among possible genotype combinations for the same trait and some genotype combinations that may be useful for selection breeding. An example of multi-genotype combinations for lignin is shown in Fig. <a reveal-id="DSU040F5" data-open="DSU040F5" class="link link-reveal link-table xref-fig">5</a>. Eight SNPs from six different genes were significantly associated with lignin content, with two associated SNPs from <em>PtoGH9A1</em> showing low LD and hence representing independent associations. </p>                        <a id="6588648" scrollto-destination="6588648"></a>
<div id="DSU040F5" data-id="DSU040F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04005.png?Expires=1509238978&Signature=LRmi1pQ~DZuteruNhFTK7fNgwI~oC4hzvOJw-Jzq47Q9LixrLAFDiLRxBCuC2JJwzOv0YL4li-jNcUhVSl2jfBD7RlNZkD7HpP2ZfGdv3xUisSyVeMnnNoChIFdQJmP4IMfFvzl34IVnDH7GGKA-qBTB3pgMd3SxV2R0SU1nyYOpFObKcad8zsg08CxVESr2X8gA-ineIdFkPSmoFs2T7rq1Z7IIZAYMUHEhI3~jmplEszPXQQk9KptP7Uh8lBYF~HAjh4NgBA-IMAlaC905qO8WjQOT41Nc9c3uCn1xrARGvbpXequPEl~f2ycb5DsI3lxE2MeUIYuekx4YGZ3A3g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phenotypic variations among the possible genotype combinations for lignin under multi-SNP additive models. (A) These eight allelic variants have significant additive effect with a small proportion of phenotypic variation (R2), ranging from 0.80 to 5.48%. Pairwise LD plots among multiple loci within each candidate genes were estimated using TASSEL Ver. 2.0.1(http://www.maizegenetics.net/). Genotype effect and the position/haplotype block of each allelic variant were shown for lignin content. (B) Twenty possible common combinations with a frequency ≥5% from the eight allelic variants were identified with various phenotypic variations on lignin content in the association population of Populus tomentosa. Some combinations were discarded, because the sample size was &lt;10 individuals." path-from-xml="dsu04005.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588648/dsu04005.tif" path-from-xml="dsu04005.tif" target="_blank">View large</a><a section="6588648" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04005.png?Expires=1509238978&Signature=UUr7Aoh4nyOJ64kY1yogTMcKv6TBsZ-LjZqVD0F~56KoLs4XkZikJBn0nbc9ynNOq0J3gZAJaNxwXxaq0pWFE71q0cKgjNUJ0Pd1vBlSw4NtdsFHI0gxiZmWnyemR7W9e-KK-bEKV4udc1leylJf0B1EuCSeFntSRdFgslj5GXOTs5K08RTZDW7R1o0gS9ejSe6dh-NuYhJjqOuFvdxczadfm1QYXakiK0HBs6wzEZRZgNdilrtnBlaIH4NaEh3w~dpLozM6V2I15no-~WPqWBAQP0cjvIvZLcgERKkofp9f0RpwS1iBK3Wk5Cx8TDJJjnqJ79wEs1uzc75Iw6H9Qg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588648&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04005.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotypic variations among the possible genotype combinations for lignin under multi-SNP additive models. (A) These eight allelic variants have significant additive effect with a small proportion of phenotypic variation (<em>R</em><sup>2</sup>), ranging from 0.80 to 5.48%. Pairwise LD plots among multiple loci within each candidate genes were estimated using TASSEL Ver. 2.0.1(<a class="link link-uri" href="http://www.maizegenetics.net/">http://www.maizegenetics.net/</a>). Genotype effect and the position/haplotype block of each allelic variant were shown for lignin content. (B) Twenty possible common combinations with a frequency ≥5% from the eight allelic variants were identified with various phenotypic variations on lignin content in the association population of <em>Populus tomentosa</em>. Some combinations were discarded, because the sample size was &lt;10 individuals.</p></div></div><div content-id="DSU040F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04005.png?Expires=1509238978&Signature=LRmi1pQ~DZuteruNhFTK7fNgwI~oC4hzvOJw-Jzq47Q9LixrLAFDiLRxBCuC2JJwzOv0YL4li-jNcUhVSl2jfBD7RlNZkD7HpP2ZfGdv3xUisSyVeMnnNoChIFdQJmP4IMfFvzl34IVnDH7GGKA-qBTB3pgMd3SxV2R0SU1nyYOpFObKcad8zsg08CxVESr2X8gA-ineIdFkPSmoFs2T7rq1Z7IIZAYMUHEhI3~jmplEszPXQQk9KptP7Uh8lBYF~HAjh4NgBA-IMAlaC905qO8WjQOT41Nc9c3uCn1xrARGvbpXequPEl~f2ycb5DsI3lxE2MeUIYuekx4YGZ3A3g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phenotypic variations among the possible genotype combinations for lignin under multi-SNP additive models. (A) These eight allelic variants have significant additive effect with a small proportion of phenotypic variation (R2), ranging from 0.80 to 5.48%. Pairwise LD plots among multiple loci within each candidate genes were estimated using TASSEL Ver. 2.0.1(http://www.maizegenetics.net/). Genotype effect and the position/haplotype block of each allelic variant were shown for lignin content. (B) Twenty possible common combinations with a frequency ≥5% from the eight allelic variants were identified with various phenotypic variations on lignin content in the association population of Populus tomentosa. Some combinations were discarded, because the sample size was &lt;10 individuals." path-from-xml="dsu04005.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588648/dsu04005.tif" path-from-xml="dsu04005.tif" target="_blank">View large</a><a section="6588648" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04005.png?Expires=1509238978&Signature=UUr7Aoh4nyOJ64kY1yogTMcKv6TBsZ-LjZqVD0F~56KoLs4XkZikJBn0nbc9ynNOq0J3gZAJaNxwXxaq0pWFE71q0cKgjNUJ0Pd1vBlSw4NtdsFHI0gxiZmWnyemR7W9e-KK-bEKV4udc1leylJf0B1EuCSeFntSRdFgslj5GXOTs5K08RTZDW7R1o0gS9ejSe6dh-NuYhJjqOuFvdxczadfm1QYXakiK0HBs6wzEZRZgNdilrtnBlaIH4NaEh3w~dpLozM6V2I15no-~WPqWBAQP0cjvIvZLcgERKkofp9f0RpwS1iBK3Wk5Cx8TDJJjnqJ79wEs1uzc75Iw6H9Qg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588648&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04005.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotypic variations among the possible genotype combinations for lignin under multi-SNP additive models. (A) These eight allelic variants have significant additive effect with a small proportion of phenotypic variation (<em>R</em><sup>2</sup>), ranging from 0.80 to 5.48%. Pairwise LD plots among multiple loci within each candidate genes were estimated using TASSEL Ver. 2.0.1(<a class="link link-uri" href="http://www.maizegenetics.net/">http://www.maizegenetics.net/</a>). Genotype effect and the position/haplotype block of each allelic variant were shown for lignin content. (B) Twenty possible common combinations with a frequency ≥5% from the eight allelic variants were identified with various phenotypic variations on lignin content in the association population of <em>Populus tomentosa</em>. Some combinations were discarded, because the sample size was &lt;10 individuals.</p></div></div>                            <h4 scrollto-destination=6588649 id="6588649" class="section-title" data-legacy-id=s3d2>3.4.2. Multi-gene associations under the dominance model</h4>
<p>Under the dominance model, we detected 121 significant associations, including 72 associations with positive dominance values and 49 with negative values, representing a total of 92 unique SNPs within the 11 genes across all three trait categories (10 traits) (Table <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S6</a>). Of the 92 SNPs, ∼95% of the significant SNPs had both additive and dominant effects on all traits within each category (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Many genes were repeatedly associated with multiple traits within/across trait categories, and different SNPs with different effects from the same gene were identified. A number of SNP associations with positive versus negative effect across the three trait categories were 25/16 for wood chemical compositions, 25/12 for wood physical properties, and 22/21 for growth traits (Table <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a>). When we considered dominant variation per trait by combining the positive and negative types, the total number of significant SNPs varied from 8 to 18, with the total <em>R</em><sup>2</sup> explained ranging from 1.8 to 13.0% (Fig. <a reveal-id="DSU040F4" data-open="DSU040F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="6588651" scrollto-destination="6588651"></a>
<div class="table-wrap"><div id="DSU040TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of the dominant effect and phenotypic contribution rate (<em>R</em><sup>2</sup>) of all significant SNPs for each trait in the <em>Populus tomentosa</em> association population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>Number of candidate genes </th><th>Number of SNPs </th><th>Range of dominance effect (%) </th><th>Range of dominant effect </th><th>Range of <em>R</em><sup>2</sup> (%) </th><th>Total <em>R</em><sup>2</sup> (%)<sup>a</sup> </th><th>Negative/positive<sup>b</sup> </th></tr></thead><tbody><tr><td rowspan="2">Lignin (%) </td><td>6 </td><td>6 </td><td>−1.347 to −0.125 </td><td>−1.347 to −0.125 </td><td>0.6 to 2.0 </td><td>7.2 </td><td>N </td></tr><tr><td>3 </td><td>3 </td><td>1.579 to 2.192 </td><td>1.579 to 2.192 </td><td>2.4 to 3.5 </td><td>9.0 </td><td>P </td></tr><tr><td rowspan="2">Holocellulose (%) </td><td>4 </td><td>4 </td><td>−2.482 to −0.08 </td><td>−2.482 to −0.08 </td><td>0.8 to 3.6 </td><td>10.2 </td><td>N </td></tr><tr><td>3 </td><td>8 </td><td>0.941 to 2.371 </td><td>0.941 to 2.371 </td><td>1.7 to 4.0 </td><td>22.6 </td><td>P </td></tr><tr><td rowspan="2">Hemicellulose (%) </td><td>2 </td><td>2 </td><td>−2.110 to −1.527 </td><td>−2.110 to −1.527 </td><td>3.3 to 4.9 </td><td>8.1 </td><td>N </td></tr><tr><td>3 </td><td>6 </td><td>0.623 to 2.221 </td><td>0.623 to 2.221 </td><td>0.6 to 5.1 </td><td>17.6 </td><td>P </td></tr><tr><td rowspan="2">α-Cellulose (%) </td><td>4 </td><td>4 </td><td>−0.973 to −0.170 </td><td>−0.973 to −0.170 </td><td>0.7 to 2.0 </td><td>4.4 </td><td>N </td></tr><tr><td>5 </td><td>8 </td><td>0.073 to 3.347 </td><td>0.073 to 3.347 </td><td>0.6 to 3.9 </td><td>16.6 </td><td>P </td></tr><tr><td rowspan="2">Fibre length (mm) </td><td>4 </td><td>4 </td><td>−0.043 to −0.009 </td><td>−0.043 to −0.009 </td><td>0.9 to 2.5 </td><td>6.6 </td><td>N </td></tr><tr><td>4 </td><td>8 </td><td>0.008 to 0.071 </td><td>0.008 to 0.071 </td><td>0.6 to 4.5 </td><td>17.1 </td><td>P </td></tr><tr><td rowspan="2">Fibre width (µm) </td><td>2 </td><td>2 </td><td>−1.420 to −0.515 </td><td>−1.420 to −0.515 </td><td>1.1 to 3.5 </td><td>4.6 </td><td>N </td></tr><tr><td>5 </td><td>7 </td><td>0.179 to 1.301 </td><td>0.179 to 1.301 </td><td>0.6 to 3.1 </td><td>15.9 </td><td>P </td></tr><tr><td rowspan="2">Microﬁbre angle (MFA, °) </td><td>4 </td><td>6 </td><td>−3.882 to −0.433 </td><td>−3.882 to −0.433 </td><td>1.2 to 5.4 </td><td>17.3 </td><td>N </td></tr><tr><td>6 </td><td>10 </td><td>0.180 to 3.697 </td><td>0.180 to 3.697 </td><td>0.6 to 5.3 </td><td>27.9 </td><td>P </td></tr><tr><td rowspan="2">Diameter at breast height (DBH, cm) </td><td>5 </td><td>6 </td><td>−3.028 to −0.452 </td><td>−3.028 to −0.452 </td><td>0.9 to 3.0 </td><td>12.0 </td><td>N </td></tr><tr><td>4 </td><td>6 </td><td>0.694 to 2.785 </td><td>0.694 to 2.785 </td><td>1.2 to 3.2 </td><td>14.4 </td><td>P </td></tr><tr><td rowspan="2">Tree height (m) </td><td>6 </td><td>8 </td><td>−2.341 to −0.126 </td><td>−2.341 to −0.126 </td><td>0.7 to 4.4 </td><td>17.0 </td><td>N </td></tr><tr><td>7 </td><td>10 </td><td>0.202 to 2.303 </td><td>0.202 to 2.303 </td><td>1.0 to 4.9 </td><td>30.0 </td><td>P </td></tr><tr><td rowspan="2">Stem volume (m<sup>3</sup>) </td><td>4 </td><td>7 </td><td>−0.350 to −0.085 </td><td>−0.350 to −0.085 </td><td>1.7 to 3.9 </td><td>17.0 </td><td>N </td></tr><tr><td>3 </td><td>6 </td><td>0.044 to 0.389 </td><td>0.044 to 0.389 </td><td>1.0 to 4.1 </td><td>12.6 </td><td>P </td></tr></tbody></table></div><div content-id="DSU040TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>Number of candidate genes </th><th>Number of SNPs </th><th>Range of dominance effect (%) </th><th>Range of dominant effect </th><th>Range of <em>R</em><sup>2</sup> (%) </th><th>Total <em>R</em><sup>2</sup> (%)<sup>a</sup> </th><th>Negative/positive<sup>b</sup> </th></tr></thead><tbody><tr><td rowspan="2">Lignin (%) </td><td>6 </td><td>6 </td><td>−1.347 to −0.125 </td><td>−1.347 to −0.125 </td><td>0.6 to 2.0 </td><td>7.2 </td><td>N </td></tr><tr><td>3 </td><td>3 </td><td>1.579 to 2.192 </td><td>1.579 to 2.192 </td><td>2.4 to 3.5 </td><td>9.0 </td><td>P </td></tr><tr><td rowspan="2">Holocellulose (%) </td><td>4 </td><td>4 </td><td>−2.482 to −0.08 </td><td>−2.482 to −0.08 </td><td>0.8 to 3.6 </td><td>10.2 </td><td>N </td></tr><tr><td>3 </td><td>8 </td><td>0.941 to 2.371 </td><td>0.941 to 2.371 </td><td>1.7 to 4.0 </td><td>22.6 </td><td>P </td></tr><tr><td rowspan="2">Hemicellulose (%) </td><td>2 </td><td>2 </td><td>−2.110 to −1.527 </td><td>−2.110 to −1.527 </td><td>3.3 to 4.9 </td><td>8.1 </td><td>N </td></tr><tr><td>3 </td><td>6 </td><td>0.623 to 2.221 </td><td>0.623 to 2.221 </td><td>0.6 to 5.1 </td><td>17.6 </td><td>P </td></tr><tr><td rowspan="2">α-Cellulose (%) </td><td>4 </td><td>4 </td><td>−0.973 to −0.170 </td><td>−0.973 to −0.170 </td><td>0.7 to 2.0 </td><td>4.4 </td><td>N </td></tr><tr><td>5 </td><td>8 </td><td>0.073 to 3.347 </td><td>0.073 to 3.347 </td><td>0.6 to 3.9 </td><td>16.6 </td><td>P </td></tr><tr><td rowspan="2">Fibre length (mm) </td><td>4 </td><td>4 </td><td>−0.043 to −0.009 </td><td>−0.043 to −0.009 </td><td>0.9 to 2.5 </td><td>6.6 </td><td>N </td></tr><tr><td>4 </td><td>8 </td><td>0.008 to 0.071 </td><td>0.008 to 0.071 </td><td>0.6 to 4.5 </td><td>17.1 </td><td>P </td></tr><tr><td rowspan="2">Fibre width (µm) </td><td>2 </td><td>2 </td><td>−1.420 to −0.515 </td><td>−1.420 to −0.515 </td><td>1.1 to 3.5 </td><td>4.6 </td><td>N </td></tr><tr><td>5 </td><td>7 </td><td>0.179 to 1.301 </td><td>0.179 to 1.301 </td><td>0.6 to 3.1 </td><td>15.9 </td><td>P </td></tr><tr><td rowspan="2">Microﬁbre angle (MFA, °) </td><td>4 </td><td>6 </td><td>−3.882 to −0.433 </td><td>−3.882 to −0.433 </td><td>1.2 to 5.4 </td><td>17.3 </td><td>N </td></tr><tr><td>6 </td><td>10 </td><td>0.180 to 3.697 </td><td>0.180 to 3.697 </td><td>0.6 to 5.3 </td><td>27.9 </td><td>P </td></tr><tr><td rowspan="2">Diameter at breast height (DBH, cm) </td><td>5 </td><td>6 </td><td>−3.028 to −0.452 </td><td>−3.028 to −0.452 </td><td>0.9 to 3.0 </td><td>12.0 </td><td>N </td></tr><tr><td>4 </td><td>6 </td><td>0.694 to 2.785 </td><td>0.694 to 2.785 </td><td>1.2 to 3.2 </td><td>14.4 </td><td>P </td></tr><tr><td rowspan="2">Tree height (m) </td><td>6 </td><td>8 </td><td>−2.341 to −0.126 </td><td>−2.341 to −0.126 </td><td>0.7 to 4.4 </td><td>17.0 </td><td>N </td></tr><tr><td>7 </td><td>10 </td><td>0.202 to 2.303 </td><td>0.202 to 2.303 </td><td>1.0 to 4.9 </td><td>30.0 </td><td>P </td></tr><tr><td rowspan="2">Stem volume (m<sup>3</sup>) </td><td>4 </td><td>7 </td><td>−0.350 to −0.085 </td><td>−0.350 to −0.085 </td><td>1.7 to 3.9 </td><td>17.0 </td><td>N </td></tr><tr><td>3 </td><td>6 </td><td>0.044 to 0.389 </td><td>0.044 to 0.389 </td><td>1.0 to 4.1 </td><td>12.6 </td><td>P </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Significance level for association (significance is <em>P</em>≤ 0.01).</p></div></div><div content-id="TN4"><div class="fn"><div class="label -label">a</div><p>Total <em>R</em><sup>2</sup> was calculated according to McKown <em>et al</em>.<sup><a reveal-id="DSU040C35" data-open="DSU040C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p></div></div><div content-id="TN5"><div class="fn"><div class="label -label">b</div><p>N: negative dominant effect; P: positive dominant effect; see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a> for further details of these association results under dominance models.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6588651" target="_blank">View Large</a></div></div><p>The 72 associations showing positive dominant effects represented 59 SNPs from 10 genes associating with all 10 traits. Of these, 22 SNPs were located in the coding regions, including 7 non-synonymous and 15 synonymous changes. A number of significant SNPs per gene varied from 1 to 11. For each trait, the number of SNPs ranged from 3 to 10, with total <em>R</em><sup>2</sup> explained ranging from 9.0 to 30.0% (Table <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a>). For the 49 associations showing negative dominant effects, we identified 44 loci from 10 genes that were significantly associated with 10 traits, with a small <em>R</em><sup>2</sup> explained per locus ranging from 0.6 to 5.4%, respectively. The number of significant SNP within each gene varied from 1 (<em>PtoBGlu1</em>) to 11 (<em>PtoUXS1</em>). A range of 2–8 SNPs with the negative dominant effects was found for each trait, with total variation explained ranging from 4.4% (α-cellulose) to 17.3% (microfibre angle) (Table <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a>). Ten SNPs from six genes that simultaneously have positive and negative dominant effects for different traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). For example, the synonymous SNP <em>PtoSAHH2</em>_285 had a negative dominant effect for lignin, while having a positive effect for holocellulose. Table <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a> shows detailed descriptions of the genetic parameters for all traits under positive and negative dominant effects.</p>                            <h4 scrollto-destination=6588653 id="6588653" class="section-title" data-legacy-id=s3d3>3.4.3. Multi-gene associations under the epistasis model</h4>
<p>We used the MDR method (<a class="link link-uri" href="http://phg.mc.vanderbilt.edu/Software/MDR">http://phg.mc.vanderbilt.edu/Software/MDR</a>) to detect and characterize SNP–SNP epistasis associated with these traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S7 and S8</a>). We first applied the ReliefF algorithm to filter all unlinked SNPs (<em>r</em><sup>2</sup> &lt; 0.1 or located in different genes) to identify significant 5 loci per trait. After correcting for multiple testing, we found 42 associations (<em>Q</em> ≤ 0.10) across nine traits. These associations represent 34 unique SNPs from seven genes with main effects varying from 0.1 to 7.5% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). We found that 9 of the 34 unique SNPs observed in the epistasis models also showed significant additive and dominant effects. Using multi-way SNP–SNP interaction models for these filtered SNPs, we identified 58 significant SNP–SNP pairs where the epistatic interactions ranged from −7.0 to 0.9% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The majority of epistatic effects were found between unlinked loci located in different genes. We also detected epistatic effects among different SNPs within the same gene. For instance, two epistasis pairs consisting of three SNPs within <em>PtoCesA4</em> were identified for fibre length, all with negative interaction values (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>).</p><p>Three-way interaction model provided a combination of three non-synonymous loci (PtoCesA4_1914, PtoGH9A1_1936, and PtoGA20ox_632) from three unique genes, representing a significant epistatic interaction for DBH and stem volume, with three-way epistatic effects of −11.7 and −12.1% (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>A). Negative epistatic effects suggest redundancy between loci, meaning that these loci provide, in part, the same information for the traits.<sup><a reveal-id="DSU040C36" data-open="DSU040C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Figure <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>B shows the mean values of phenotypic variances among all three genotype combinations. We investigated the nature of epistatic interactions among non-synonymous mutations, contributing to the phenotypic variation, in the three genes, and discovered that most of the variation is captured by combinatorial permutations of allelic variants at three loci (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>C). We identified the specific combinatorial patterns of amino acid replacements at these three loci, by comparing the expected values of eight combinations with high (H) or low (L) phenotypic values with the observed values of the specific combinatorial patterns (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>). </p>                        <a id="6588656" scrollto-destination="6588656"></a>
<div id="DSU040F6" data-id="DSU040F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04006.png?Expires=1509238978&Signature=f1dZfC8gGXxA2uSelIiE10G~0zUheh4fPx1~pdyp4StDlaU7N7J~-ZL2-Lly4FBusCLNBxVt9btzEMyg71SnwplZnmxUmyO9Hc~tlOFyVBoBtbR3Q9XUl57Dfa508-BeCW8~hvFA0ZnvwVkwivjzxrpzRqVsXOj8gQmnMXZoHtlKL1AnSxqJ8ojY0tj8Pr-M24r4cU97t9Xwu6YjErKsPtcdnpfFBwh3FOvJNNqiqJLYjCJaBy24KDKn55fjvr2I33QaVwKQhPEIkVKsdKs9A7AQ3EcMVEgx-8F6OnZx3CciUzNgr-Njpu83PD7CYohKyV4NMus2wuwAa5ENyyJDCg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Three-way epistatic interaction of a combination of three non-synonymous loci for DBH and stem volume. (A) Summary of information gain by main effects, pairwise effects, and the three-way effect for each of three non-synonymous loci (PtoCesA4_1914, PtoGH9A1_1936, and PtoGA20ox_632). (B) Phenotypic variation for each genotype combination from the three non-synonymous loci. High-phenotype values for genotype combinations are shaded in dark grey, and low values are shaded in light grey; the vertical lines/boxes represent the higher or lower phenotypic values of different genotype combinations than the average values of sum of three separate genotypes. (C) All combinatorial permutations of allelic variants of three non-synonymous loci. Two high-epistatic combinatorial patterns of amino acid replacements at these three loci were identified (shaded regions)." path-from-xml="dsu04006.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588656/dsu04006.tif" path-from-xml="dsu04006.tif" target="_blank">View large</a><a section="6588656" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04006.png?Expires=1509238978&Signature=aYtThAfyOZ~saSUSEV53rc~hFGQySmPoc~cC5JinFC9QOFk7epVbIc4vI8JANPSCKavdg8SHIKTu8iU6betYU4cOzhkVfwCohG5PuZFUyFsljgjiXv6d9Io38NZ84xok5sWc7RgyO4LjdVX1r~peTAsdM9rmBRdFFt1Sk~QmIwmSyzVX4sPHNmC8KeGI1LnYankZxICaA0zaGBRC~I3lRSC3VPVvf4XO~ATQj5yJSpbk4IhuSdzyhL5EJ3y7ozHTa8UVoPrEIGXevb~IzqM9hBOJcJXqSjaw5UZV0BAOgg--lxPZRPxi8crCBqZ1UEr3vbCSlKQxkel4MTrq-RC2KQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588656&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04006.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three-way epistatic interaction of a combination of three non-synonymous loci for DBH and stem volume. (A) Summary of information gain by main effects, pairwise effects, and the three-way effect for each of three non-synonymous loci (PtoCesA4_1914, PtoGH9A1_1936, and PtoGA20ox_632). (B) Phenotypic variation for each genotype combination from the three non-synonymous loci. High-phenotype values for genotype combinations are shaded in dark grey, and low values are shaded in light grey; the vertical lines/boxes represent the higher or lower phenotypic values of different genotype combinations than the average values of sum of three separate genotypes. (C) All combinatorial permutations of allelic variants of three non-synonymous loci. Two high-epistatic combinatorial patterns of amino acid replacements at these three loci were identified (shaded regions).</p></div></div><div content-id="DSU040F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/m_dsu04006.png?Expires=1509238978&Signature=f1dZfC8gGXxA2uSelIiE10G~0zUheh4fPx1~pdyp4StDlaU7N7J~-ZL2-Lly4FBusCLNBxVt9btzEMyg71SnwplZnmxUmyO9Hc~tlOFyVBoBtbR3Q9XUl57Dfa508-BeCW8~hvFA0ZnvwVkwivjzxrpzRqVsXOj8gQmnMXZoHtlKL1AnSxqJ8ojY0tj8Pr-M24r4cU97t9Xwu6YjErKsPtcdnpfFBwh3FOvJNNqiqJLYjCJaBy24KDKn55fjvr2I33QaVwKQhPEIkVKsdKs9A7AQ3EcMVEgx-8F6OnZx3CciUzNgr-Njpu83PD7CYohKyV4NMus2wuwAa5ENyyJDCg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Three-way epistatic interaction of a combination of three non-synonymous loci for DBH and stem volume. (A) Summary of information gain by main effects, pairwise effects, and the three-way effect for each of three non-synonymous loci (PtoCesA4_1914, PtoGH9A1_1936, and PtoGA20ox_632). (B) Phenotypic variation for each genotype combination from the three non-synonymous loci. High-phenotype values for genotype combinations are shaded in dark grey, and low values are shaded in light grey; the vertical lines/boxes represent the higher or lower phenotypic values of different genotype combinations than the average values of sum of three separate genotypes. (C) All combinatorial permutations of allelic variants of three non-synonymous loci. Two high-epistatic combinatorial patterns of amino acid replacements at these three loci were identified (shaded regions)." path-from-xml="dsu04006.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6588656/dsu04006.tif" path-from-xml="dsu04006.tif" target="_blank">View large</a><a section="6588656" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu04006.png?Expires=1509238978&Signature=aYtThAfyOZ~saSUSEV53rc~hFGQySmPoc~cC5JinFC9QOFk7epVbIc4vI8JANPSCKavdg8SHIKTu8iU6betYU4cOzhkVfwCohG5PuZFUyFsljgjiXv6d9Io38NZ84xok5sWc7RgyO4LjdVX1r~peTAsdM9rmBRdFFt1Sk~QmIwmSyzVX4sPHNmC8KeGI1LnYankZxICaA0zaGBRC~I3lRSC3VPVvf4XO~ATQj5yJSpbk4IhuSdzyhL5EJ3y7ozHTa8UVoPrEIGXevb~IzqM9hBOJcJXqSjaw5UZV0BAOgg--lxPZRPxi8crCBqZ1UEr3vbCSlKQxkel4MTrq-RC2KQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6588656&ar=441064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04006.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three-way epistatic interaction of a combination of three non-synonymous loci for DBH and stem volume. (A) Summary of information gain by main effects, pairwise effects, and the three-way effect for each of three non-synonymous loci (PtoCesA4_1914, PtoGH9A1_1936, and PtoGA20ox_632). (B) Phenotypic variation for each genotype combination from the three non-synonymous loci. High-phenotype values for genotype combinations are shaded in dark grey, and low values are shaded in light grey; the vertical lines/boxes represent the higher or lower phenotypic values of different genotype combinations than the average values of sum of three separate genotypes. (C) All combinatorial permutations of allelic variants of three non-synonymous loci. Two high-epistatic combinatorial patterns of amino acid replacements at these three loci were identified (shaded regions).</p></div></div><p>Overall, using multi-gene association models, we captured all three components of genetic variation conferred by multiple variants within candidate genes in cellulose biosyntheses pathway both within and across these three trait categories. In total, we identified 118, 121, and 43 associations consistent with additive, dominant, and epistatic effects, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6–S8</a>). Also, we observed no differences in the magnitudes of additive and dominant effects between three trait categories (Tables <a reveal-id="DSU040TB1" data-open="DSU040TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a>). Some genes have extensive complexity in both genetic variation and the resulting phenotype, and numerous SNPs from different genes were associated across these three trait categories, suggesting that tree growth and wood biosynthesis are complex, dynamic processes that require the coordinate regulation of diverse sets of genes.<sup><a reveal-id="DSU040C2" data-open="DSU040C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU040C3" data-open="DSU040C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Notably, our present investigation suggests that <em>CesA</em> members may have additional functional roles in tree growth and development, beyond the direct effects of the cellulose synthase subunits in biosynthesis of the cell wall. <em>CesA</em> members represent an interesting set of candidates for further study in trees.</p>                            <h3 scrollto-destination=6588658 id="6588658" class="section-title" data-legacy-id=s3e>3.5. Validation of association in the linkage population</h3>
<p>To validate the associations identified above, we examined associations in a linkage population. From the 118 unique significant SNPs obtained by multi-SNP and MDR models (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>), we identified 109 common SNP markers that segregated in the 1,200 F<sub>1</sub> progeny following Mendelian expectations (<em>P</em> ≥ 0.01, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S5</a>). We conducted 1,090 tests for association of these SNP with growth and wood-quality traits (109 SNPs × 10 traits) in the linkage population. After correction for multiple testing errors, we found 61 associations at the threshold of <em>Q</em> ≤ 0.10, representing 38 SNPs from nine genes that show associations with all 10 traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). These 38 unique SNPs included 5 non-synonymous, 15 synonymous, and 18 non-coding SNPs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). Thirty SNPs were similarly detected by multi-SNP additive and dominance models in the discovery population, and nine loci were shared between the epistasis models and the single-marker analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>). Most of the significant markers were identified for multiple traits with different genotypic effects; for example, PtoCesA4_1914 is associated with seven traits, and PtoGA20ox_632 with six (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). In addition, 28 (46.0%) associations identified in the validation population were similarly detected in the multi-SNP models (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S9</a>). For each trait, the number of SNPs ranged from 3 to 9, with marker effects that varied from 2.2 to 13.5% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). Effect sizes for the SNPs analysed using single-marker linkage test were nearly 2-fold higher than effects identified using multi-SNP models (average <em>R<sup>2</sup></em> = 3.8%).</p><p>Correspondingly, each trait is associated with at least three candidate genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>) using single-marker linkage analysis. With wood chemical composition, we identified two genes (<em>PtoCesA4</em> and <em>PtoCesA7</em>) that were simultaneously associated with the content of all four polysaccharide traits. In addition, we identified that <em>PtoGH9A1</em> was significantly associated with hemicellulose and lignin contents, whereas <em>PtoCesA3</em> was associated with hemicellulose and α-cellulose content (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). Among wood physical properties and growth traits, SNP-trait associations identified two genes (<em>PtoCesA4</em> and <em>PtoGA20Ox</em>) that were associated with all six traits. Furthermore, <em>PtoUXS1</em> has SNPs associated with both DBH and stem volume traits, as well as two wood physical properties (fibre width and microfibre angle), consistent with the associations under multi-SNP models in our discovery population (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S9</a>). No SNPs from <em>PtoUGDH1</em> and <em>PtoBGlu1</em> were associated with any of the 10 phenotypic traits.</p>                            <h3 scrollto-destination=6588661 id="6588661" class="section-title" data-legacy-id=s3f>3.6 Haplotype-based association analysis</h3>
<p>We identified 138 high-LD blocks (<em>r<sup>2</sup></em> ≥ 0.70, <em>P</em> ≤ 0.001), including 429 common haplotypes (frequency ≥ 5%) from the 11 candidate genes, with block sizes from 2 to 564 bp. The number of common haplotypes per block varied from 2 to 7, with an average of 3.0 (Table <a reveal-id="DSU040TB3" data-open="DSU040TB3" class="link link-reveal link-table xref-fig">3</a>). The number of LD blocks and common haplotypes per gene varied from 4 to 20 and 15 to 60, respectively (Table <a reveal-id="DSU040TB3" data-open="DSU040TB3" class="link link-reveal link-table xref-fig">3</a>). Haplotype-based association tests identified a total of 162 common haplotypes from 66 blocks at <em>P</em> ≤ 0.05 (Table <a reveal-id="DSU040TB3" data-open="DSU040TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S10</a>). Using multiple test corrections, we found 39 significant blocks from 10 candidate genes, including 90 haplotypes, that associated with nine traits excluding stem volume (139 associations, <em>Q</em> ≤ 0.10); no significant haplotypes were found for <em>PtoSAHH2</em>. The number of blocks and haplotypes for each trait ranged from 2 to 12 and 5 to 33, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). </p>                        <a id="6588663" scrollto-destination="6588663"></a>
<div class="table-wrap"><div id="DSU040TB3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of haplotype-based association analysis within 11 candidate genes for each trait in the <em>Populus tomentosa</em> association population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Abbreviation of gene </th><th>Gene name </th><th>Number of LD block </th><th>Number of common haplotypes<sup>a</sup> </th><th>Length range of haplotype (bp) </th><th>Number of haplotypes (<em>P</em> ≤ 0.05)<sup>b</sup> </th><th>Number of haplotypes, FDR corrected [(<em>Q</em>) ≤ 0.10]<sup>c</sup> </th><th>Number of associated traits </th></tr></thead><tbody><tr><td><em>PtoBGlu1</em> </td><td><em>β-1,4-glucosidase</em> </td><td>4 </td><td>15 </td><td>35–564 </td><td>4 </td><td>4 </td><td>4 </td></tr><tr><td><em>PtoCesA3</em> </td><td><em>Cellulose synthase</em> </td><td>20 </td><td>58 </td><td>31–431 </td><td>20 </td><td>12 </td><td>3 </td></tr><tr><td><em>PtoCesA4</em> </td><td><em>Cellulose synthase</em> </td><td>19 </td><td>57 </td><td>2–320 </td><td>16 </td><td>11 </td><td>4 </td></tr><tr><td><em>PtoCesA7</em> </td><td><em>Cellulose synthase</em> </td><td>16 </td><td>60 </td><td>12–484 </td><td>23 </td><td>9 </td><td>5 </td></tr><tr><td><em>PtoGA20Ox</em> </td><td><em>GA20-oxidase</em> </td><td>9 </td><td>30 </td><td>52–246 </td><td>10 </td><td>7 </td><td>2 </td></tr><tr><td><em>PtoGH9A1</em> </td><td><em>Endo-1,4-β-glucanase</em> </td><td>14 </td><td>41 </td><td>27–274 </td><td>19 </td><td>9 </td><td>5 </td></tr><tr><td><em>PtoSAHH1</em> </td><td><em>S-adenosine-<span class="small-caps">l</span>-homocysteine-hydrolase</em> </td><td>8 </td><td>30 </td><td>60–202 </td><td>11 </td><td>6 </td><td>1 </td></tr><tr><td><em>PtoSAHH2</em> </td><td><em>S-adenosine-<span class="small-caps">l</span>-homocysteine-hydrolase</em> </td><td>5 </td><td>21 </td><td>77–355 </td><td>6 </td><td>0 </td><td>0 </td></tr><tr><td><em>PtoSUS1</em> </td><td><em>Sucrose synthase</em> </td><td>20 </td><td>46 </td><td>8–395 </td><td>25 </td><td>14 </td><td>6 </td></tr><tr><td><em>PtoUGDH1</em> </td><td><em>UDP-glucose dehydrogenase</em> </td><td>7 </td><td>28 </td><td>47–192 </td><td>8 </td><td>8 </td><td>3 </td></tr><tr><td><em>PtoUXS1</em> </td><td><em>UDP-glucuronate decarboxylase</em> </td><td>16 </td><td>43 </td><td>8–203 </td><td>20 </td><td>10 </td><td>7 </td></tr><tr><td>Total </td><td><em>/</em> </td><td>138 </td><td>429 </td><td>2–564 </td><td>162 </td><td>90 </td><td>/ </td></tr></tbody></table></div><div content-id="DSU040TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Abbreviation of gene </th><th>Gene name </th><th>Number of LD block </th><th>Number of common haplotypes<sup>a</sup> </th><th>Length range of haplotype (bp) </th><th>Number of haplotypes (<em>P</em> ≤ 0.05)<sup>b</sup> </th><th>Number of haplotypes, FDR corrected [(<em>Q</em>) ≤ 0.10]<sup>c</sup> </th><th>Number of associated traits </th></tr></thead><tbody><tr><td><em>PtoBGlu1</em> </td><td><em>β-1,4-glucosidase</em> </td><td>4 </td><td>15 </td><td>35–564 </td><td>4 </td><td>4 </td><td>4 </td></tr><tr><td><em>PtoCesA3</em> </td><td><em>Cellulose synthase</em> </td><td>20 </td><td>58 </td><td>31–431 </td><td>20 </td><td>12 </td><td>3 </td></tr><tr><td><em>PtoCesA4</em> </td><td><em>Cellulose synthase</em> </td><td>19 </td><td>57 </td><td>2–320 </td><td>16 </td><td>11 </td><td>4 </td></tr><tr><td><em>PtoCesA7</em> </td><td><em>Cellulose synthase</em> </td><td>16 </td><td>60 </td><td>12–484 </td><td>23 </td><td>9 </td><td>5 </td></tr><tr><td><em>PtoGA20Ox</em> </td><td><em>GA20-oxidase</em> </td><td>9 </td><td>30 </td><td>52–246 </td><td>10 </td><td>7 </td><td>2 </td></tr><tr><td><em>PtoGH9A1</em> </td><td><em>Endo-1,4-β-glucanase</em> </td><td>14 </td><td>41 </td><td>27–274 </td><td>19 </td><td>9 </td><td>5 </td></tr><tr><td><em>PtoSAHH1</em> </td><td><em>S-adenosine-<span class="small-caps">l</span>-homocysteine-hydrolase</em> </td><td>8 </td><td>30 </td><td>60–202 </td><td>11 </td><td>6 </td><td>1 </td></tr><tr><td><em>PtoSAHH2</em> </td><td><em>S-adenosine-<span class="small-caps">l</span>-homocysteine-hydrolase</em> </td><td>5 </td><td>21 </td><td>77–355 </td><td>6 </td><td>0 </td><td>0 </td></tr><tr><td><em>PtoSUS1</em> </td><td><em>Sucrose synthase</em> </td><td>20 </td><td>46 </td><td>8–395 </td><td>25 </td><td>14 </td><td>6 </td></tr><tr><td><em>PtoUGDH1</em> </td><td><em>UDP-glucose dehydrogenase</em> </td><td>7 </td><td>28 </td><td>47–192 </td><td>8 </td><td>8 </td><td>3 </td></tr><tr><td><em>PtoUXS1</em> </td><td><em>UDP-glucuronate decarboxylase</em> </td><td>16 </td><td>43 </td><td>8–203 </td><td>20 </td><td>10 </td><td>7 </td></tr><tr><td>Total </td><td><em>/</em> </td><td>138 </td><td>429 </td><td>2–564 </td><td>162 </td><td>90 </td><td>/ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN6"><div class="fn"><p>/: not applied.</p></div></div><div content-id="TN7"><div class="fn"><div class="label -label">a</div><p>Common haplotype (frequency ≥ 0.05).</p></div></div><div content-id="TN8"><div class="fn"><div class="label -label">b</div><p><em>P</em>-value = significance level for association (significance is <em>P</em> ≤ 0.05).</p></div></div><div content-id="TN9"><div class="fn"><div class="label -label">c</div><p><em>Q</em>-value = a correction for multiple testing [FDR (<em>Q</em>) ≤ 0.10]; see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S10 and S11</a> for further details of these haplotype-based association results.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6588663" target="_blank">View Large</a></div></div><p>Most of the significantly associated genes/blocks were shared among traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). For example, haplotypes belonging to <em>PtoUXS1</em> were associated with seven traits. <em>PtoCesA7</em> had haplotypes that were simultaneously associated with all four chemical composition traits. In addition, <em>PtoUGDH1</em> with haplotypes were significantly associated with three chemical compositions, excluding α-cellulose content (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). <em>PtoUXS1</em> had haplotypes that were associated with all three wood physical properties. Within the growth trait category, several significant haplotypes were identified within <em>PtoUXS1</em> and <em>PtoSUS1</em> that were associated with tree height and DBH. Examination of haplotypes may provide support for the SNP-based association.</p><p>On the basis of the significant blocks we identified within several genes for each trait (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>), we identified four exclusive multi-haplotype combinations with numerous genetic variants for three traits—holocellulose, lignin, and tree height (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Taking holocellulose content as example, we identified several combinations of multiple haplotypes from two significant blocks in <em>PtoCesA4</em> and <em>PtoGH9A1</em>, with mean values from 66.725 to 75.321%, consistent with the two genes having significant correlation (<em>P</em> ≤ 0.01, <em>R</em> = 0.952) (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>B). In addition, eight unique haplotypes from four genes, showing specific (presence/absence) for different climatic regions of the natural population of <em>P. tomentosa</em>, were associated with five traits, respectively (Table <a reveal-id="DSU040TB4" data-open="DSU040TB4" class="link link-reveal link-table xref-fig">4</a>). For fibre width, two haplotypes (C-A-G and T-A-G) from <em>PtoSUS1</em> and <em>PtoBGlu1</em> were exclusively found in the Northwestern region, and were associated with higher fibre width (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). </p>                        <a id="6588666" scrollto-destination="6588666"></a>
<div class="table-wrap"><div id="DSU040TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>List of climatic region-specific haplotypes significantly associated with wood quality and growth traits in <em>Populus tomentosa</em> after correction for multiple testing [FDR (<em>Q</em>) ≤ 0.10]</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th><em>P-</em>value </th><th><em>Q</em>-value </th><th>Amplicon </th><th>Significant haplotypes </th><th>NE </th><th>S </th><th>NW </th><th>Haplotype frequency </th><th>Phenotypic variation (mean ± SE) </th></tr></thead><tbody><tr><td>Holocellulose </td><td>0.0010 </td><td>0.0273 </td><td>PtoBGlu1_4972-4990-5014-5164 </td><td>C-G-G-C </td><td>- </td><td>+ </td><td>- </td><td>0.050 </td><td>68.200 ± 0.090 </td></tr><tr><td rowspan="2">Hemicellulose </td><td rowspan="2">0.0010 </td><td rowspan="2">0.0273 </td><td rowspan="2">PtoCesA3_555-571 </td><td>T-C </td><td>+ </td><td>+ </td><td>- </td><td>0.050 </td><td>36.905 ± 1.605 </td></tr><tr><td>A-C </td><td>+ </td><td>+ </td><td>- </td><td>0.325 </td><td>29.232 ± 1.431 </td></tr><tr><td rowspan="2">Fibre length </td><td>0.0001 </td><td>0.0160 </td><td>PtoBGlu1_5408-5417-5443 </td><td>T-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.050 </td><td>1.2920 ± 0.047 </td></tr><tr><td>0.0001 </td><td>0.0160 </td><td>PtoUXS1_1678-1685-1707-1848-1862-1884-1899-
1913-1914-2125-2131-2198-2221-2303 </td><td>G-G-T-G-C-C-T-T-G-A-T-C-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.111 </td><td>1.3077 ± 0.031 </td></tr><tr><td rowspan="2">Fibre width </td><td>0.0010 </td><td>0.0273 </td><td>PtoSUS1_3199-3242-3301 </td><td>C-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.050 </td><td>25.607 ± 1.243 </td></tr><tr><td>0.0001 </td><td>0.0160 </td><td>PtoBGlu1_5408-5417-5443 </td><td>T-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.050 </td><td>27.606 ± 1.245 </td></tr><tr><td>Tree height </td><td>0.0050 </td><td>0.0532 </td><td>PtoUXS1_3000-3005-3043-3054-3059 </td><td>G-T-T-T-C </td><td>+ </td><td>+ </td><td>- </td><td>0.111 </td><td>13.500 ± 0.991 </td></tr></tbody></table></div><div content-id="DSU040TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th><em>P-</em>value </th><th><em>Q</em>-value </th><th>Amplicon </th><th>Significant haplotypes </th><th>NE </th><th>S </th><th>NW </th><th>Haplotype frequency </th><th>Phenotypic variation (mean ± SE) </th></tr></thead><tbody><tr><td>Holocellulose </td><td>0.0010 </td><td>0.0273 </td><td>PtoBGlu1_4972-4990-5014-5164 </td><td>C-G-G-C </td><td>- </td><td>+ </td><td>- </td><td>0.050 </td><td>68.200 ± 0.090 </td></tr><tr><td rowspan="2">Hemicellulose </td><td rowspan="2">0.0010 </td><td rowspan="2">0.0273 </td><td rowspan="2">PtoCesA3_555-571 </td><td>T-C </td><td>+ </td><td>+ </td><td>- </td><td>0.050 </td><td>36.905 ± 1.605 </td></tr><tr><td>A-C </td><td>+ </td><td>+ </td><td>- </td><td>0.325 </td><td>29.232 ± 1.431 </td></tr><tr><td rowspan="2">Fibre length </td><td>0.0001 </td><td>0.0160 </td><td>PtoBGlu1_5408-5417-5443 </td><td>T-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.050 </td><td>1.2920 ± 0.047 </td></tr><tr><td>0.0001 </td><td>0.0160 </td><td>PtoUXS1_1678-1685-1707-1848-1862-1884-1899-
1913-1914-2125-2131-2198-2221-2303 </td><td>G-G-T-G-C-C-T-T-G-A-T-C-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.111 </td><td>1.3077 ± 0.031 </td></tr><tr><td rowspan="2">Fibre width </td><td>0.0010 </td><td>0.0273 </td><td>PtoSUS1_3199-3242-3301 </td><td>C-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.050 </td><td>25.607 ± 1.243 </td></tr><tr><td>0.0001 </td><td>0.0160 </td><td>PtoBGlu1_5408-5417-5443 </td><td>T-A-G </td><td>+ </td><td>- </td><td>- </td><td>0.050 </td><td>27.606 ± 1.245 </td></tr><tr><td>Tree height </td><td>0.0050 </td><td>0.0532 </td><td>PtoUXS1_3000-3005-3043-3054-3059 </td><td>G-T-T-T-C </td><td>+ </td><td>+ </td><td>- </td><td>0.111 </td><td>13.500 ± 0.991 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN10"><div class="fn"><p>These amplicons were selected based on the significant squared correlation of allele frequencies <em>r</em><sup>2</sup> (<em>r</em><sup>2</sup> ≥ 0.70, <em>P</em> ≤ 0.001); +: the haplotype was found in this climatic region; -: not found; NE: northeastern region; S: southern region; NW: northwestern region; SE: standard error.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6588666" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=6588667 id="6588667" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=6588668 id="6588668" class="section-title" data-legacy-id=s4a>4.1. Functional interpretations of genetic associations</h3>
<p>Here we have used multi-gene association approaches, combining extensive full-length candidate genes from a shared biological pathway and with detailed, replicated phenotypic information from natural populations of <em>P. tomemtosa</em> to uncover numerous loci underlying variation in wood chemical composition, physical properties, and growth traits. We further validated mostly these associations in an independent linkage population. Significant additive, dominant, and epistatic effects were observed for tree growth and lignocellulosic composition. The number of associations we detected is larger, while the average <em>R</em><sup>2</sup> explained individually is lower, than in previous single-marker association for similar traits in the association population.<sup><a reveal-id="DSU040C16" data-open="DSU040C16" class="link link-ref link-reveal xref-bibr">16</a>–<a reveal-id="DSU040C19" data-open="DSU040C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Supportively, effect sizes for the SNPs analysed in a large size of linkage population (average <em>R<sup>2</sup></em> = 3.8%) were nearly 2-fold higher than effects identified using multi-gene association models, suggesting that pathway-based association analyses across multiple genes may provide higher power to uncover the relatively small individual effects.<sup><a reveal-id="DSU040C14" data-open="DSU040C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU040C43" data-open="DSU040C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Also, it is possible that linkage drag in regions around the candidate genes causes larger <em>R</em><sup>2</sup> values of allelic variants in the linkage population than in the association population. More statistics models for estimating the genetic effect in multiple population backgrounds are needed. Markers found to be associated with traits span both coding and non-coding portions of genes, with a predominance for additive effects and small effect sizes (Tables <a reveal-id="DSU040TB1" data-open="DSU040TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSU040TB2" data-open="DSU040TB2" class="link link-reveal link-table xref-fig">2</a>), suggesting that this best describe the genetic architecture of complex traits in trees.<sup><a reveal-id="DSU040C5" data-open="DSU040C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSU040C44" data-open="DSU040C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Low to moderate <em>R</em><sup>2</sup> were captured by fitting all SNPs simultaneously from all 11 candidate genes (Fig. <a reveal-id="DSU040F4" data-open="DSU040F4" class="link link-reveal link-table xref-fig">4</a>), suggesting that we thus far lack sufficient genomic coverage to capture the majority genetic variation. Many genes with significant SNPs for three trait categories were detected, consistent with our expectations of modelling all combined markers from a set of interrelated genes concurrently, instead of using candidate gene-based ‘one-marker-at-a-time’ approach in tree species.<sup><a reveal-id="DSU040C45" data-open="DSU040C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p><p>All candidate genes were originally isolated from a xylem cDNA library of <em>P. tomentosa</em>, and a part of the genes was highly up-regulated in xylem tissue (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>A), indicating that these genes are likely main candidates for participating in secondary cell wall formation. Previous studies have indicated that wood biosynthesis and tree growth require the coordinate regulation of diverse metabolic pathways, and that the genes in these shared pathways are often functional homologs.<sup><a reveal-id="DSU040C3" data-open="DSU040C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU040C8" data-open="DSU040C8" class="link link-ref link-reveal xref-bibr">8</a></sup> This viewpoint agrees with our observation that the number of common genes (i.e. genes that were identified as being significantly associated with a trait at least twice using different association models, including additive effect, dominance, epistasis, and haplotype) affecting each trait varied from 2 to 6 (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>). We identified <em>PtoCesA4</em> and <em>PtoCesA7</em> as being associated to four wood chemical compositions (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S6</a>), consistent with the central role played by CesA7 and CesA4, as subunits of the CesA complex required for proper secondary wall cellulose synthesis as well as other possible functions related to wall function.<sup><a reveal-id="DSU040C46" data-open="DSU040C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSU040C47" data-open="DSU040C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The result supports previous association studies by Wegrzyn <em>et al.</em><sup><a reveal-id="DSU040C7" data-open="DSU040C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and Du <em>et al.</em><sup><a reveal-id="DSU040C16" data-open="DSU040C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU040C17" data-open="DSU040C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In addition, we identified two common genes directly related to polysaccharides (<em>PtoCesA4</em> and <em>PtoUXS1</em>) that were associated simultaneously with all three wood ultrastructure traits (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>). UXS is likely ubiquitous among plants and is a target for regulatory control during cell wall biosynthesis,<sup><a reveal-id="DSU040C17" data-open="DSU040C17" class="link link-ref link-reveal xref-bibr">17</a></sup> which further supports our findings that <em>PtoUXS1</em> explains a significant amount of <em>R</em><sup>2</sup> of all growth traits (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>). However, the mechanism by which specific allelic variation in <em>PtoUXS1</em> could affect tree cell elongation is not yet apparent. </p>                        <a id="6588671" scrollto-destination="6588671"></a>
<div class="table-wrap"><div id="DSU040TB5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Identification of the common genes significantly related to growth and wood properties in <em>Populus tomentosa</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>PtoBGlu1</em> </th><th><em>PtoCesA3</em> </th><th><em>PtoCesA4</em> </th><th><em>PtoCesA7</em> </th><th><em>PtoGA20Ox</em> </th><th><em>PtoGH9A1</em> </th><th><em>PtoSAHH1</em> </th><th><em>PtoSAHH2</em> </th><th><em>PtoSUS1</em> </th><th><em>PtoUGDH1</em> </th><th><em>PtoUXS1</em> </th></tr></thead><tbody><tr><td>Lignin </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>Holocellulose </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td></tr><tr><td>Hemicellulose </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td>+ </td></tr><tr><td>α-Cellulose </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td><td> </td><td> </td></tr><tr><td>Fibre length </td><td> </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>Fibre width </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td></tr><tr><td>Microfibre angle </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>DBH </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>Tree height </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td>+ </td></tr><tr><td>Stem volume </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td></tr></tbody></table></div><div content-id="DSU040TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>PtoBGlu1</em> </th><th><em>PtoCesA3</em> </th><th><em>PtoCesA4</em> </th><th><em>PtoCesA7</em> </th><th><em>PtoGA20Ox</em> </th><th><em>PtoGH9A1</em> </th><th><em>PtoSAHH1</em> </th><th><em>PtoSAHH2</em> </th><th><em>PtoSUS1</em> </th><th><em>PtoUGDH1</em> </th><th><em>PtoUXS1</em> </th></tr></thead><tbody><tr><td>Lignin </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>Holocellulose </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td></tr><tr><td>Hemicellulose </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td>+ </td></tr><tr><td>α-Cellulose </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td><td> </td><td> </td></tr><tr><td>Fibre length </td><td> </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>Fibre width </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td></tr><tr><td>Microfibre angle </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>DBH </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td></tr><tr><td>Tree height </td><td> </td><td> </td><td>+ </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td><td>+ </td><td> </td><td>+ </td></tr><tr><td>Stem volume </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>+ </td><td> </td><td> </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN11"><div class="fn"><p>+: genes that were identified as being significantly associated with a trait at least two times using different models, including additive effect, dominance, epistasis, and haplotype.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6588671" target="_blank">View Large</a></div></div><p>Taking α-cellulose content as an example, we identified six common genes significantly related to this trait, specifically <em>PtoBGlu1</em>, <em>PtoCesA3</em>, <em>PtoCesA4</em>, <em>PtoCesA7</em>, <em>PtoGH9A1</em>, and <em>PtoSUS1</em> (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>)<em>.</em> When taken together, multi-SNP models explained a small to moderate portions of the variation from both additive and dominant effects (31.7 and 12.2%, respectively). Previous evidence suggest that these six genes may play key roles in cellulose biosynthesis,<sup><a reveal-id="DSU040C47" data-open="DSU040C47" class="link link-ref link-reveal xref-bibr">47</a>–<a reveal-id="DSU040C49" data-open="DSU040C49" class="link link-ref link-reveal xref-bibr">49</a></sup> supporting these observations. SNPs in <em>PtoSUS1</em> showed high additive and dominant effects (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>), and two LD blocks, including five significant haplotypes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>), were associated with α-cellulose. In plants, SUS catalyzes a reversible reaction that preferentially converts sucrose into fructose and UDP-glucose. UDP-glucose is the immediate precursor for the synthesis of cellulose.<sup><a reveal-id="DSU040C50" data-open="DSU040C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Another enzyme involved in cellulose formation during xylem development is β-glucosidase (BGlu), which hydrolyzes cellobiose to glucose, and is a rate-limiting factor during enzymatic hydrolysis of cellulose.<sup><a reveal-id="DSU040C51" data-open="DSU040C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Here, we identified two haplotypes and a non-coding SNP in <em>PtoBGlu1</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S11</a>). Furthermore, korrigan cellulase (KOR), an endo-β-1,4-glucanase, belonging to the glycosyl hydrolase family 9 (GH9), can hydrolyze the β-1,4 linkage of the cellulose chain.<sup><a reveal-id="DSU040C52" data-open="DSU040C52" class="link link-ref link-reveal xref-bibr">52</a></sup><em>PtoGH9A1</em>, an orthologue <em>of KORRIGAN1</em>, contains three haplotypes associated with α-cellulose (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>), and we also found epistatic interactions for the trait between a non-synonymous SNP (PtoGH9A1_1936) and SNPs in <em>PtoCesA3</em> and <em>PtoCesA4</em>. These findings were supported by significant correlations in gene expression among these six genes (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>B).</p><p>A similar situation was also observed for hemicellulose, the second most abundant polysaccharide after cellulose, where we identified several haplotypes and eight SNPs from five of the candidate genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S11</a>). Of these, <em>PtoUGDP1</em> and <em>PtoUXS1</em> are key enzymes in hemicellulose biosynthesis.<sup><a reveal-id="DSU040C53" data-open="DSU040C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSU040C54" data-open="DSU040C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Five common candidate genes were shared between wood-property and growth traits (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>). <em>PtoSAHH2</em> is associated with hemicellulose, tree height, and stem volume. The observation is consistent with SAHH being a cytokinin-binding protein participating in xylogenesis.<sup><a reveal-id="DSU040C46" data-open="DSU040C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Interestingly, associations with lignin were found within genes directly or indirectly related to cellulose or xylan biosynthesis, such as <em>PtoUXS1</em>, <em>PtoCesA4</em>, and <em>PtoCesA7</em> (Table <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>). This finding was supported by the strong genetic interrelations between the lignin biosynthetic pathway and polysaccharide biosynthesis.<sup><a reveal-id="DSU040C3" data-open="DSU040C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU040C55" data-open="DSU040C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Applying biological pathway-based association<sup><a reveal-id="DSU040C11" data-open="DSU040C11" class="link link-ref link-reveal xref-bibr">11</a></sup> to examine many more candidate genes involved in the diverse metabolic pathways of tree is needed in future investigation.</p><p>The numerous genes that we detected have multiple significant associations across all three trait categories (Tables <a reveal-id="DSU040TB1  DSU040TB3" data-open="DSU040TB1  DSU040TB3" class="link link-reveal link-table xref-fig">1–3</a>, and <a reveal-id="DSU040TB5" data-open="DSU040TB5" class="link link-reveal link-table xref-fig">5</a>), and the finding was possible to be explained by two hypothesis. On one hand, significant genetic correlations were frequently estimated among multiple related trait fractions or percentages of the whole tree body, especially within each trait category in our present study (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S3</a>). On the other hand, this observation may be considered to be an indication of pleiotropy in the broad sense, and the potential pleiotropic loci identified in this study provide novel candidates for explaining phenotypic variation in <em>Populus</em>. Pleiotropic loci span multiple functions and may have effects by acting upstream in signalling pathways that affect many traits or by directly targeting multiple genes for regulation,<sup><a reveal-id="DSU040C35" data-open="DSU040C35" class="link link-ref link-reveal xref-bibr">35</a></sup> which is consistent with our discoveries that <em>PtoGA20Ox</em>, <em>PtoSAHH1</em>, and <em>PtoSAHH2</em> are all associated with multiple traits (Tables <a reveal-id="DSU040TB1  DSU040TB5" data-open="DSU040TB1  DSU040TB5" class="link link-reveal link-table xref-fig">1–5</a>). More work should be focused on transcription factors/regulators in future.</p>                            <h3 scrollto-destination=6588675 id="6588675" class="section-title" data-legacy-id=s4b>4.2. Epistasis for growth and wood-property traits</h3>
<p>Non-additive interactions between separate mutations (epistasis) can significantly influence the rate and direction of evolutionary change.<sup><a reveal-id="DSU040C13" data-open="DSU040C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Also, epistasis is an important but often-ignored component of genetic variation in breeding for quantitative traits, due to limited statistical power to detect and estimate epistasis. In our study, we detected and characterized SNP–SNP epistasis associated with the phenotypic traits by using a MDR method (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S7 and S8</a>). The detection of epistasis hence provides necessary information complementary to that gained through single-locus analysis. We observed that 11 of the 34 unique SNPs identified through the MDR analyses were located in coding regions, with the remaining 23 being located in non-coding regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>), supporting the notion that most associations exhibited pervasive epistatic effects and that it involves both non-coding and coding polymorphisms.<sup><a reveal-id="DSU040C56" data-open="DSU040C56" class="link link-ref link-reveal xref-bibr">56</a></sup></p><p>We identified three non-synonymous loci from three unique genes with significant epistatic effects for DBH and stem volume (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a> A–D). The similar epistatic pattern for these two traits may be due to a significant positive phenotypic correlation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S3</a>).<sup><a reveal-id="DSU040C20" data-open="DSU040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Also, the observation may be supported by previous studies reporting that examinations of interactions between amino acid mutations in the same/shared proteins underlying variation in a measurable phenotype may best reveal the mechanisms of epistasis.<sup><a reveal-id="DSU040C57" data-open="DSU040C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Without the expectations of an additive null model, the phenotypic effects of allelic substitutions (L → H and H → L) at specific combinatorial patterns of amino acid replacements at these three non-synonymous mutations were highly dependent on genetic background.<sup><a reveal-id="DSU040C58" data-open="DSU040C58" class="link link-ref link-reveal xref-bibr">58</a></sup> We observed two specific combinatorial patterns: first, in the presence of each amino acid type (H for Ser and L for Thr) in PtoGH9A1_1936, the Asn (L) in PtoGA20Ox_632 conferred an increased phenotype value on the Asn (L) background in PtoCesA4_1914, and a decrease on the His (H) background in PtoCesA4_1914 (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>B and C). Similarly, the L-type amino acid type in PtoCesA4_1914 conferred a decrease on the Ser (H) background in PtoGA20Ox_632 and an increase on the Asn (L) background in PtoGA20Ox_632 (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>B and C). Second, without consideration of amino acid types present in PtoCesA4_1914, the L-type in PtoGH9A1_1936 conferred an increased phenotype value in the L background in PtoGA20Ox_632 and a decrease on the H background in PtoGA20Ox_632 (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>B and C). Similarly, the L-type amino acid in PtoGA20Ox_632 conferred a decrease on the H background in PtoGH9A1_1936 and an increase on the L background in PtoGH9A1_1936 (Fig. <a reveal-id="DSU040F6" data-open="DSU040F6" class="link link-reveal link-table xref-fig">6</a>B and C). These findings are examples of sign epistasis<sup><a reveal-id="DSU040C58" data-open="DSU040C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and suggest the co-selection of epistatic alleles or protein–protein interactions, although the mechanisms for their generation and maintenance are not clear. These observations were supported by the significant positive correlations in gene expression for each pair of these three genes in our tissue-specific transcript profiling (Fig. <a reveal-id="DSU040F1" data-open="DSU040F1" class="link link-reveal link-table xref-fig">1</a>B).</p>                            <h3 scrollto-destination=6588678 id="6588678" class="section-title" data-legacy-id=s4c>4.3. Future perspectives for MAS breeding</h3>
<p>The future of tree improvement will depend on the co-selection of favourable alleles for traits of interest and on the application of combination of haplotypes/genotypes in a manner that effectively overcomes the low levels of LD observed in forest trees.<sup><a reveal-id="DSU040C59" data-open="DSU040C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Also, genomic selection (GS) is expected to cause a paradigm shift in tree breeding by improving the efficiency and speed of the breeding process.<sup><a reveal-id="DSU040C45" data-open="DSU040C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Therefore, examining the phenotypic differences among combinations of multiple haplotypes/genotypes for a specific trait (Fig. <a reveal-id="DSU040F5" data-open="DSU040F5" class="link link-reveal link-table xref-fig">5</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S3</a>) provides a foundation for dissecting complex traits through multi-gene or GS in trees. Only 63% of SNPs in haplotypes were identified in the multi-SNP models (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S11</a>), consistent with previous observations that haplotype-based tests can detect previously unknown QTL and are thus well supported for unraveling the genetic basis of complex traits.<sup><a reveal-id="DSU040C59" data-open="DSU040C59" class="link link-ref link-reveal xref-bibr">59</a></sup><em>Populus tomentosa</em> is distributed across a wide geographical area of northern China<sup><a reveal-id="DSU040C20" data-open="DSU040C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and thus can provide a potential source of region-specific haplotypes (Table <a reveal-id="DSU040TB4" data-open="DSU040TB4" class="link link-reveal link-table xref-fig">4</a>). These region-specific signatures may reflect, as well as affect, strong regional differentiation in these traits. Probably these fixed loci are likely key to understanding geographic adaptation and may also be targeted in region-based breeding.</p><p>The present work has identified a number of candidate allelic variants that could be exploited to alter key economic traits for industrial applications. However, the direct use of QTL or association information for MAS breeding has not been successful, chiefly because of the difficulty in transferring associations across populations and species of forest trees.<sup><a reveal-id="DSU040C45" data-open="DSU040C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Future work should focus on replicated sampling for a larger number of broadly representative ecotypes especially across different environments, as well as considering genetic background or genotype × environment interactions.<sup><a reveal-id="DSU040C1" data-open="DSU040C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Also, joint mapping with association populations and multiple specific biparental crosses is likely to be extremely powerful for improving map resolution and to improve identification of new alleles. The incorporation of more genes (such as gene family members, transcription factors/regulators, transferases, transporters, kinases, and hydrolases) in shared biosynthetic pathways<sup><a reveal-id="DSU040C11" data-open="DSU040C11" class="link link-ref link-reveal xref-bibr">11</a></sup> would provide a more complete dissection of the effects of genetic variation on growth and lignocellulosic traits. We believe these studies will gradually drive the application of MAS to selection breeding of trees at the early seedling stage.</p>                            <h2 scrollto-destination=6588681 id="6588681" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6588683 id="6588683" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the State Key Basic Research Program of China (No. 2012CB114506), the National ‘863’ Plan Project (No. 2013AA102702), and the 111 Project (No. B13007). Funding to pay the Open Access publication charges for this article was provided by the State Key Basic Research Program of China (No. 2012CB114506), the National ‘863’ Plan Project (No. 2013AA102702), and the 111 Project (No. B13007).</p>                            <h2 scrollto-destination=6588685 id="6588685" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Prof. Ronald R. Sederoff (NC State University, Raleigh, NC, USA) for detailed comments and specific suggestions for improving the manuscript, and also Prof. Rongling Wu and Dr Zhong Wang (Center for Statistical Genetics, Pennsylvania State University, Hershey, PA, USA) for advice on statistical analysis.</p>                            <h2 scrollto-destination=6588687 id="6588687" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsu040c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neale</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Forest tree genomics: growing resources and applications</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">111</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forest%20tree%20genomics%3A%20growing%20resources%20and%20applications&amp;author=D.B.%20Neale&amp;author=A.%20Kremer&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=111-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2931" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2931" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2931"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21245829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Somerville</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bauer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Brininstool</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Toward a systems approach to understanding plant cell walls</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">306</div> (pg. <div class="fpage">2206</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20systems%20approach%20to%20understanding%20plant%20cell%20walls&amp;author=C.%20Somerville&amp;author=S.%20Bauer&amp;author=G.%20Brininstool&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=2206-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1102765" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1102765" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1102765"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15618507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Persson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Milne</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Page</div> <div class="given-names">G.P.</div></div>,  <div class="name"><div class="surname">Somervill</div> <div class="given-names">C.R.</div></div>. </div><div class="article-title">Identification of genes required for cellulose synthesis by regression analysis of public microarray data sets</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">8633</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20required%20for%20cellulose%20synthesis%20by%20regression%20analysis%20of%20public%20microarray%20data%20sets&amp;author=S.%20Persson&amp;author=H.%20Wei&amp;author=J.%20Milne&amp;author=G.P.%20Page&amp;author=C.R.%20Somervill&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=8633-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0503392102" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0503392102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0503392102"> </span></div> </div></div></div></div></div><div content-id="dsu040c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">González-Martīnez</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Ersoz</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">G.R.</div></div>,  <div class="name"><div class="surname">Wheeler</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Neale</div> <div class="given-names">D.B.</div></div>. </div><div class="article-title">DNA sequence variation and selection of tag single-nucleotide polymorphisms at candidate genes for drought-stress response in <em>Pinus taeda</em> L</div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">172</div> (pg. <div class="fpage">1915</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20sequence%20variation%20and%20selection%20of%20tag%20single-nucleotide%20polymorphisms%20at%20candidate%20genes%20for%20drought-stress%20response%20in%20Pinus%20taeda%20L&amp;author=S.C.%20Gonz%C3%A1lez-Mart%C4%ABnez&amp;author=E.%20Ersoz&amp;author=G.R.%20Brown&amp;author=N.C.%20Wheeler&amp;author=D.B.%20Neale&amp;publication_year=2006&amp;journal=Genetics&amp;volume=172&amp;pages=1915-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.047126" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.047126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.047126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16387885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eckert</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Wegrzyn</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Cumbie</div> <div class="given-names">W.P.</div></div>, et al. </div><div class="article-title">Association genetics of the loblolly pine (<em>Pinus taeda</em>, Pinaceae) metabolome</div>, <div class="source">New Phytol.</div> , <div class="year">2012</div>, vol. <div class="volume">193</div> (pg. <div class="fpage">890</div>-<div class="lpage">902</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20genetics%20of%20the%20loblolly%20pine%20(Pinus%20taeda%2C%20Pinaceae)%20metabolome&amp;author=A.J.%20Eckert&amp;author=J.L.%20Wegrzyn&amp;author=W.P.%20Cumbie&amp;publication_year=2012&amp;journal=New%20Phytol.&amp;volume=193&amp;pages=890-902" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.03976.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03976.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.03976.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22129444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thumma</div> <div class="given-names">B.R.</div></div>,  <div class="name"><div class="surname">Nolan</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Evans</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Moran</div> <div class="given-names">G.F.</div></div>. </div><div class="article-title">Polymorphisms in cinnamoyl coa reductase (CCR) are associated with variation in microﬁbril angle in <em>Eucalyptus</em> spp.</div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1257</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymorphisms%20in%20cinnamoyl%20coa%20reductase%20(CCR)%20are%20associated%20with%20variation%20in%20micro%EF%AC%81bril%20angle%20in%20Eucalyptus%20spp.&amp;author=B.R.%20Thumma&amp;author=M.F.%20Nolan&amp;author=R.%20Evans&amp;author=G.F.%20Moran&amp;publication_year=2005&amp;journal=Genetics&amp;volume=171&amp;pages=1257-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.042028" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.042028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.042028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wegrzyn</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Eckert</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (<em>Populus trichocarpa</em>, Salicaceae) secondary xylem</div>, <div class="source">New Phytol.</div> , <div class="year">2010</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">515</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20genetics%20of%20traits%20controlling%20lignin%20and%20cellulose%20biosynthesis%20in%20black%20cottonwood%20(Populus%20trichocarpa%2C%20Salicaceae)%20secondary%20xylem&amp;author=J.L.%20Wegrzyn&amp;author=A.J.%20Eckert&amp;author=M.%20Choi&amp;publication_year=2010&amp;journal=New%20Phytol.&amp;volume=188&amp;pages=515-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03415.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03415.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03415.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20831625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beaulieu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Doerksen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Boyle</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Association genetics of wood physical traits in the conifer White Spruce and relationships with gene expression</div>, <div class="source">Genetics</div> , <div class="year">2011</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">197</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20genetics%20of%20wood%20physical%20traits%20in%20the%20conifer%20White%20Spruce%20and%20relationships%20with%20gene%20expression&amp;author=J.%20Beaulieu&amp;author=T.%20Doerksen&amp;author=B.%20Boyle&amp;publication_year=2011&amp;journal=Genetics&amp;volume=188&amp;pages=197-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.125781" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.125781" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.125781"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21385726" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>. </div><div class="article-title">Polymorphic SSR loci within <em>PtoCesA</em> genes are associated with growth and wood properties in <em>Populus tomentosa</em></div>, <div class="source">New Phytol.</div> , <div class="year">2013</div>, vol. <div class="volume">197</div> (pg. <div class="fpage">763</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymorphic%20SSR%20loci%20within%20PtoCesA%20genes%20are%20associated%20with%20growth%20and%20wood%20properties%20in%20Populus%20tomentosa&amp;author=Q.Z.%20Du&amp;author=W.%20Pan&amp;author=B.H.%20Xu&amp;author=B.L.%20Li&amp;author=D.Q.%20Zhang&amp;publication_year=2013&amp;journal=New%20Phytol.&amp;volume=197&amp;pages=763-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12072" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23278184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guerra</div> <div class="given-names">F.P.</div></div>,  <div class="name"><div class="surname">Wegrzyn</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Sykes</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Association genetics of chemical wood properties in black poplar (<em>Populus nigra</em>)</div>, <div class="source">New Phytol.</div> , <div class="year">2013</div>, vol. <div class="volume">197</div> (pg. <div class="fpage">162</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20genetics%20of%20chemical%20wood%20properties%20in%20black%20poplar%20(Populus%20nigra)&amp;author=F.P.%20Guerra&amp;author=J.L.%20Wegrzyn&amp;author=R.%20Sykes&amp;publication_year=2013&amp;journal=New%20Phytol.&amp;volume=197&amp;pages=162-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23157484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hakonarson</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Analysing biological pathways in genome-wide association studies</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">843</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysing%20biological%20pathways%20in%20genome-wide%20association%20studies&amp;author=K.%20Wang&amp;author=M.%20Li&amp;author=H.%20Hakonarson&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=843-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2884" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2884" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2884"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21085203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoggart</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Whittaker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">De Iorio</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Balding</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Simultaneous analysis of all SNPs in genome-wide and re-sequencing association studies</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000130</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simultaneous%20analysis%20of%20all%20SNPs%20in%20genome-wide%20and%20re-sequencing%20association%20studies&amp;author=C.J.%20Hoggart&amp;author=J.%20Whittaker&amp;author=M.%20De%20Iorio&amp;author=D.%20Balding&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000130" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18654633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Richards</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Carbone</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Epistasis dominates the genetic architecture of Drosophila quantitative traits</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2012</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">15553</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epistasis%20dominates%20the%20genetic%20architecture%20of%20Drosophila%20quantitative%20traits&amp;author=W.%20Huang&amp;author=S.%20Richards&amp;author=M.%20Carbone&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=15553-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1213423109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1213423109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1213423109"> </span></div> </div></div></div></div></div><div content-id="dsu040c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Benyamin</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">McEvoy</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Common SNPs explain a large proportion of the heritability for human height</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">565</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20SNPs%20explain%20a%20large%20proportion%20of%20the%20heritability%20for%20human%20height&amp;author=J.%20Yang&amp;author=B.%20Benyamin&amp;author=B.%20McEvoy&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=565-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.608" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.608" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.608"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20562875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Briggs</div> <div class="given-names">W.H.</div></div>, et al. </div><div class="article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">203</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed-model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=203-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>. </div><div class="article-title">The UDP-glucuronate decarboxylase gene family in <em>Populus</em>: structure, expression, and association genetics,</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">84</div> pg. <div class="fpage">e60880</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20UDP-glucuronate%20decarboxylase%20gene%20family%20in%20Populus%3A%20structure%2C%20expression%2C%20and%20association%20genetics%2C&amp;author=Q.%20Du&amp;author=W.%20Pan&amp;author=J.%20Tian&amp;author=B.%20Li&amp;author=D.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=84&amp;pages=e60880" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0060880" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0060880" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0060880"> </span></div> </div></div></div></div></div><div content-id="dsu040c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Allelic variation in a cellulose synthase gene <em>PtoCesA4</em> associated with growth and wood properties in <em>Populus tomentosa</em></div>, <div class="source">G3</div> , <div class="year">2013</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">2069</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20variation%20in%20a%20cellulose%20synthase%20gene%20PtoCesA4%20associated%20with%20growth%20and%20wood%20properties%20in%20Populus%20tomentosa&amp;author=Q.%20Du&amp;author=B.%20Xu&amp;author=W.%20Pan&amp;publication_year=2013&amp;journal=G3&amp;volume=3&amp;pages=2069-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.113.007724" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.113.007724" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.113.007724"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24048648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Allelic variation in <em>PtGA20Ox</em> associates with growth and wood properties in <em>Populus</em> spp</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e53116</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20variation%20in%20PtGA20Ox%20associates%20with%20growth%20and%20wood%20properties%20in%20Populus%20spp&amp;author=J.%20Tian&amp;author=Q.%20Du&amp;author=M.%20Chang&amp;author=D.%20Zhang&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e53116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0053116" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0053116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0053116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Single-nucleotide polymorphisms in <em>PtoCesA7</em> and their association with growth and wood properties in <em>Populus tomentosa</em></div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2014</div>, vol. <div class="volume">289</div> (pg. <div class="fpage">439</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-nucleotide%20polymorphisms%20in%20PtoCesA7%20and%20their%20association%20with%20growth%20and%20wood%20properties%20in%20Populus%20tomentosa&amp;author=J.%20Tian&amp;author=M.%20Chang&amp;author=Q.%20Du&amp;author=B.%20Xu&amp;author=D.%20Zhang&amp;publication_year=2014&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=289&amp;pages=439-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-014-0824-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-014-0824-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-014-0824-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24549852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gong</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Variation in growth, leaf, and wood property traits of Chinese white poplar <em>Populus tomentosa</em>, a major industrial tree species in Northern China</div>, <div class="source">Can. J. For. Res.</div> , <div class="year">2014</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">326</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20growth%2C%20leaf%2C%20and%20wood%20property%20traits%20of%20Chinese%20white%20poplar%20Populus%20tomentosa%2C%20a%20major%20industrial%20tree%20species%20in%20Northern%20China&amp;author=Q.%20Du&amp;author=B.%20Xu&amp;author=C.%20Gong&amp;publication_year=2014&amp;journal=Can.%20J.%20For.%20Res.&amp;volume=44&amp;pages=326-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/cjfr-2013-0416" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fcjfr-2013-0416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fcjfr-2013-0416"> </span></div> </div></div></div></div></div><div content-id="dsu040c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dillon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Southerton</div> <div class="given-names">S.G.</div></div>. </div><div class="article-title">Generation and analysis of expressed sequence tags from six developing xylem libraries in <em>Pinus radiata</em> D.Don</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20and%20analysis%20of%20expressed%20sequence%20tags%20from%20six%20developing%20xylem%20libraries%20in%20Pinus%20radiata%20D.Don&amp;author=X.%20Li&amp;author=H.%20Wu&amp;author=S.%20Dillon&amp;author=S.G.%20Southerton&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=1-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2009.02.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2009.02.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2009.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.Q.</div></div>,  <div class="name"><div class="surname">Du</div> <div class="given-names">Q.Z.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>. </div><div class="article-title">The actin multigene family in <em>Populus</em>: organization, expression and phylogenetic analysis</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">105</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20actin%20multigene%20family%20in%20Populus%3A%20organization%2C%20expression%20and%20phylogenetic%20analysis&amp;author=D.Q.%20Zhang&amp;author=Q.Z.%20Du&amp;author=B.H.%20Xu&amp;author=Z.Y.%20Zhang&amp;author=B.%20Li&amp;publication_year=2010&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=284&amp;pages=105-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-010-0552-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-010-0552-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-010-0552-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20577761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Scheet</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">449</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accounting%20for%20decay%20of%20linkage%20disequilibrium%20in%20haplotype%20inference%20and%20missing-data%20imputation&amp;author=M.%20Stephens&amp;author=P.%20Scheet&amp;publication_year=2005&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=76&amp;pages=449-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/428594" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F428594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F428594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15700229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sanchez-Delbarrio</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Messeguer</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Rozas</div> <div class="given-names">R.</div></div>. </div><div class="article-title">DnaSP, DNA polymorphism analyses by the coalescent and other methods</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">2496</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%2C%20DNA%20polymorphism%20analyses%20by%20the%20coalescent%20and%20other%20methods&amp;author=J.%20Rozas&amp;author=J.%20Sanchez-Delbarrio&amp;author=X.%20Messeguer&amp;author=R.%20Rozas&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=2496-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14668244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Molecular evolutionary genetics</div> , <div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Columbia University Press</div></div></div></div></div><div content-id="dsu040c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Park</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Casella</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The Bayesian lasso</div>, <div class="source">J Am Stat Assoc.</div> , <div class="year">2008</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">681</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Bayesian%20lasso&amp;author=T.%20Park&amp;author=G.%20Casella&amp;publication_year=2008&amp;journal=J%20Am%20Stat%20Assoc.&amp;volume=103&amp;pages=681-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1198/016214508000000337" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1198%2F016214508000000337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1198%2F016214508000000337"> </span></div> </div></div></div></div></div><div content-id="dsu040c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watterson</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">On the number of segregating sites in genetical models without recombination</div>, <div class="source">Theor. Popul. Boil.</div> , <div class="year">1975</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">188</div>-<div class="lpage">93</div>)</div></div></div></div><div content-id="dsu040c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tajima</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</div>, <div class="source">Genetics</div> , <div class="year">1989</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">585</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;author=F.%20Tajima&amp;publication_year=1989&amp;journal=Genetics&amp;volume=123&amp;pages=585-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Statistical tests of neutrality of mutations</div>, <div class="source">Genetics</div> , <div class="year">1993</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">693</div>-<div class="lpage">709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20tests%20of%20neutrality%20of%20mutations&amp;author=Y.%20Fu&amp;author=W.%20Li&amp;publication_year=1993&amp;journal=Genetics&amp;volume=133&amp;pages=693-709" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8454210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hill</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Linkage disequilibrium in finite populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1968</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">226</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%20in%20finite%20populations&amp;author=W.G.%20Hill&amp;author=A.%20Robertson&amp;publication_year=1968&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=38&amp;pages=226-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01245622" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01245622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01245622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24442307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Remington</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Thornsberry</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">11479</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20and%20phenotypic%20associations%20in%20the%20maize%20genome&amp;author=D.L.%20Remington&amp;author=J.M.%20Thornsberry&amp;author=Y.%20Matsuoka&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=11479-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.201394398" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.201394398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.201394398"> </span></div> </div></div></div></div></div><div content-id="dsu040c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Das</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The Bayesian lasso for genome-wide association studies</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">516</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Bayesian%20lasso%20for%20genome-wide%20association%20studies&amp;author=J.%20Li&amp;author=K.%20Das&amp;author=G.%20Fu&amp;author=R.%20Li&amp;author=R.%20Wu&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=516-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq688" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21156729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ingvarsson</div> <div class="given-names">P.K.</div></div>. </div><div class="article-title">Molecular evolution of synonymous codon usage in <em>Populus</em></div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">307</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%20synonymous%20codon%20usage%20in%20Populus&amp;author=P.K.%20Ingvarsson&amp;publication_year=2008&amp;journal=BMC%20Evol.%20Biol.&amp;volume=8&amp;pages=307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-8-307" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-8-307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-8-307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18983655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McKown</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Klápště</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Guy</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Genome-wide association implicates numerous genes underlying ecological trait variation in natural populations of <em>Populus trichocarpa</em></div>, <div class="source">New Phytol.</div> , <div class="year">2014</div>, vol. <div class="volume">22</div> <div class="pub-id-doi"><a href="https://doi.org/10.1111/nph.12815" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1111/nph.12815</a></div></div></div></div></div><div content-id="dsu040c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hahn</div> <div class="given-names">L.W.</div></div>,  <div class="name"><div class="surname">Ritchie</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Multifactor dimensionality reduction software for detecting gene-gene and gene-environment interactions</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">376</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multifactor%20dimensionality%20reduction%20software%20for%20detecting%20gene-gene%20and%20gene-environment%20interactions&amp;author=L.W.%20Hahn&amp;author=M.%20Ritchie&amp;author=J.%20Moore&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=376-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btf869" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtf869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtf869"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12584123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moore</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Gilbert</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tsai</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A flexible computational framework for detecting, characterizing, and interpreting statistical patterns of epistasis in genetic studies of human disease susceptibility</div>, <div class="source">J. Theor. Boil.</div> , <div class="year">2006</div>, vol. <div class="volume">241</div> (pg. <div class="fpage">252</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20flexible%20computational%20framework%20for%20detecting%2C%20characterizing%2C%20and%20interpreting%20statistical%20patterns%20of%20epistasis%20in%20genetic%20studies%20of%20human%20disease%20susceptibility&amp;author=J.H.%20Moore&amp;author=J.%20Gilbert&amp;author=C.%20Tsai&amp;publication_year=2006&amp;journal=J.%20Theor.%20Boil.&amp;volume=241&amp;pages=252-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2005.11.036" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2005.11.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2005.11.036"> </span></div> </div></div></div></div></div><div content-id="dsu040c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kiralis</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">An information-gain approach to detecting three-way epistatic interactions in genetic association studies</div>, <div class="source">J. Am. Med. Inform. Assoc.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">630</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20information-gain%20approach%20to%20detecting%20three-way%20epistatic%20interactions%20in%20genetic%20association%20studies&amp;author=T.%20Hu&amp;author=Y.%20Chen&amp;author=J.%20Kiralis&amp;publication_year=2013&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;volume=20&amp;pages=630-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/amiajnl-2012-001525" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Famiajnl-2012-001525" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Famiajnl-2012-001525"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23396514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sinnwell</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schaid</div> <div class="given-names">D.</div></div>. </div><div class="article-title">haplo.stats: Statistical Analysis of Haplotypes With Traits and Covariates When Linkage Phase Is Ambiguous</div>, <div class="year">2009</div> <div class="comment">R package version 1.4.4. <a class="link link-uri" href="http://CRAN.R-project.org/package=haplo.stats">http://CRAN.R-project.org/package=haplo.stats</a> (1 August 2012, date last accessed)</div></div></div></div></div><div content-id="dsu040c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Patterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Price</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Reich</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Population structure and eigen analysis</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">2074</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20eigen%20analysis&amp;author=N.%20Patterson&amp;author=A.%20Price&amp;author=D.%20Reich&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=2074-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020190" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020190"> </span></div> </div></div></div></div></div><div content-id="dsu040c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genetic diversity and population structure of Chinese white poplar (<em>Populus tomentosa</em>) revealed by SSR markers</div>, <div class="source">J. Hered.</div> , <div class="year">2012</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">853</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20population%20structure%20of%20Chinese%20white%20poplar%20(Populus%20tomentosa)%20revealed%20by%20SSR%20markers&amp;author=Q.%20Du&amp;author=B.%20Wang&amp;author=Z.%20Wei&amp;author=D.%20Zhang&amp;author=B.%20Li&amp;publication_year=2012&amp;journal=J.%20Hered.&amp;volume=103&amp;pages=853-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/ess061" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fess061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fess061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23008443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Storey</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Tibshirani</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Statistical signiﬁcance for genome wide studies</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">9440</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20signi%EF%AC%81cance%20for%20genome%20wide%20studies&amp;author=J.D.%20Storey&amp;author=R.%20Tibshirani&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=9440-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1530509100" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1530509100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1530509100"> </span></div> </div></div></div></div></div><div content-id="dsu040c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ingvarsson</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Garcia</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Luquez</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Hall</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Nucleotide polymorphism and phenotypic associations within and around the phytochrome B2 locus in European aspen <em>Populus tremula</em>, Salicaceae</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">178</div> (pg. <div class="fpage">2217</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20polymorphism%20and%20phenotypic%20associations%20within%20and%20around%20the%20phytochrome%20B2%20locus%20in%20European%20aspen%20Populus%20tremula%2C%20Salicaceae&amp;author=P.K.%20Ingvarsson&amp;author=M.V.%20Garcia&amp;author=V.%20Luquez&amp;author=D.%20Hall&amp;author=S.%20Jansson&amp;publication_year=2008&amp;journal=Genetics&amp;volume=178&amp;pages=2217-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.082354" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.082354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.082354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18245834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hill</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Goddard</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Visscher</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Data and theory point to mainly additive genetic variance for complex traits</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000008</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Data%20and%20theory%20point%20to%20mainly%20additive%20genetic%20variance%20for%20complex%20traits&amp;author=W.G.%20Hill&amp;author=M.%20Goddard&amp;author=P.%20Visscher&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000008" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18454194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Resende</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Resende</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sansaloni</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Genomic selection for growth and wood quality in <em>Eucalyptus</em>: capturing the missing heritability and accelerating breeding for complex traits in forest trees</div>, <div class="source">New Phytol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">116</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20selection%20for%20growth%20and%20wood%20quality%20in%20Eucalyptus%3A%20capturing%20the%20missing%20heritability%20and%20accelerating%20breeding%20for%20complex%20traits%20in%20forest%20trees&amp;author=M.D.%20Resende&amp;author=M.%20Resende&amp;author=C.%20Sansaloni&amp;publication_year=2012&amp;journal=New%20Phytol.&amp;volume=194&amp;pages=116-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.04038.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.04038.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.04038.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22309312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Somerville</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Cellulose synthesis in higher plants</div>, <div class="source">Annu. Rev. Cell Dev. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">53</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cellulose%20synthesis%20in%20higher%20plants&amp;author=C.%20Somerville&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Cell%20Dev.%20Biol.&amp;volume=22&amp;pages=53-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.cellbio.22.022206.160206" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.cellbio.22.022206.160206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.cellbio.22.022206.160206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16824006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Persson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paredez</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Carroll</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genetic evidence for three unique components in primary cell-wall cellulose synthase complexes in <em>Arabidopsis</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">15566</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20evidence%20for%20three%20unique%20components%20in%20primary%20cell-wall%20cellulose%20synthase%20complexes%20in%20Arabidopsis&amp;author=S.%20Persson&amp;author=A.%20Paredez&amp;author=A.%20Carroll&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=15566-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0706592104" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0706592104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0706592104"> </span></div> </div></div></div></div></div><div content-id="dsu040c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szyjanowicz</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">McKinnon</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">The irregular xylem 2 mutant is an allele of korrigan that affects the secondary cell wall of <em>Arabidopsis thaliana</em></div>, <div class="source">Plant J.</div> , <div class="year">2004</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">730</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20irregular%20xylem%202%20mutant%20is%20an%20allele%20of%20korrigan%20that%20affects%20the%20secondary%20cell%20wall%20of%20Arabidopsis%20thaliana&amp;author=P.M.%20Szyjanowicz&amp;author=I.%20McKinnon&amp;author=N.%20Taylor&amp;publication_year=2004&amp;journal=Plant%20J.&amp;volume=37&amp;pages=730-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.37.issue-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.37.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.37.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14871312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coleman</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mansﬁeld</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">13118</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sucrose%20synthase%20affects%20carbon%20partitioning%20to%20increase%20cellulose%20production%20and%20altered%20cell%20wall%20ultrastructure&amp;author=H.D.%20Coleman&amp;author=J.%20Yan&amp;author=S.%20Mans%EF%AC%81eld&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=13118-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0900188106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0900188106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0900188106"> </span></div> </div></div></div></div></div><div content-id="dsu040c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Joshi</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Bhandari</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ranjan</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Genomics of cellulose biosynthesis in poplars</div>, <div class="source">New Phytol.</div> , <div class="year">2004</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">53</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20cellulose%20biosynthesis%20in%20poplars&amp;author=C.P.%20Joshi&amp;author=S.%20Bhandari&amp;author=P.%20Ranjan&amp;publication_year=2004&amp;journal=New%20Phytol.&amp;volume=164&amp;pages=53-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2004.01155.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2004.01155.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2004.01155.x"> </span></div> </div></div></div></div></div><div content-id="dsu040c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harhangi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Steenbakkers</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Akhmanova</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">A highly expressed family 1 β-glucosidase with transglycosylation capacity from the anaerobic fungus <em>Piromyces</em> sp, <em>E2</em></div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="year">2002</div>, vol. <div class="volume">1574</div> (pg. <div class="fpage">293</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20highly%20expressed%20family%201%20%CE%B2-glucosidase%20with%20transglycosylation%20capacity%20from%20the%20anaerobic%20fungus%20Piromyces%20sp%2C%20E2&amp;author=H.%20Harhangi&amp;author=P.%20Steenbakkers&amp;author=A.%20Akhmanova&amp;publication_year=2002&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1574&amp;pages=293-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-4781(01)00380-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-4781(01)00380-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-4781(01)00380-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11997095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Urbanowicz</div> <div class="given-names">B.R.</div></div>,  <div class="name"><div class="surname">Bennett</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">del Campillo</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Structural organization and a standardized nomenclature for plant endo-1,4-<em>β</em>-glucanases cellulases of glycosyl hydrolase family 9</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">1693</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20organization%20and%20a%20standardized%20nomenclature%20for%20plant%20endo-1%2C4-%CE%B2-glucanases%20cellulases%20of%20glycosyl%20hydrolase%20family%209&amp;author=B.R.%20Urbanowicz&amp;author=A.%20Bennett&amp;author=E.%20del%20Campillo&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=144&amp;pages=1693-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.102574" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.102574" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.102574"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17687051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bar-Peled</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Griffith</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Doering</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Functional cloning and characterization of a UDP-glucuronic acid decarboxylase: the pathogenic fungus <em>Cryptococcus neoformans</em> elucidates UDP-xylose synthesis</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">12003</div>-<div class="lpage">08</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20cloning%20and%20characterization%20of%20a%20UDP-glucuronic%20acid%20decarboxylase%3A%20the%20pathogenic%20fungus%20Cryptococcus%20neoformans%20elucidates%20UDP-xylose%20synthesis&amp;author=M.%20Bar-Peled&amp;author=C.%20Griffith&amp;author=T.%20Doering&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=12003-08" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.211229198" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.211229198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.211229198"> </span></div> </div></div></div></div></div><div content-id="dsu040c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campbell</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sala</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">van de Rijn</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Tanner</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Properties and kinetic analysis of UDP-glucose dehydrogenase from group A streptococci. Irreversible inhibition by UDP-chloroacetol</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1997</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">3416</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20and%20kinetic%20analysis%20of%20UDP-glucose%20dehydrogenase%20from%20group%20A%20streptococci.%20Irreversible%20inhibition%20by%20UDP-chloroacetol&amp;author=R.%20Campbell&amp;author=R.%20Sala&amp;author=I.%20van%20de%20Rijn&amp;author=M.%20Tanner&amp;publication_year=1997&amp;journal=J.%20Biol.%20Chem.&amp;volume=272&amp;pages=3416-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.272.6.3416" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.272.6.3416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.272.6.3416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9013585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Porth</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Klápště</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Skyba</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Genome-wide association mapping for wood characteristics in <em>Populus</em> identifies an array of candidate single nucleotide polymorphisms</div>, <div class="source">New Phytol.</div> , <div class="year">2013</div>, vol. <div class="volume">200</div> (pg. <div class="fpage">710</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20for%20wood%20characteristics%20in%20Populus%20identifies%20an%20array%20of%20candidate%20single%20nucleotide%20polymorphisms&amp;author=I.%20Porth&amp;author=J.%20Kl%C3%A1p%C5%A1t%C4%9B&amp;author=O.%20Skyba&amp;publication_year=2013&amp;journal=New%20Phytol.&amp;volume=200&amp;pages=710-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12422" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23889164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kacser</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Burns</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">The molecular basis of dominance</div>, <div class="source">Genetics</div> , <div class="year">1981</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">639</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20basis%20of%20dominance&amp;author=H.%20Kacser&amp;author=J.A.%20Burns&amp;publication_year=1981&amp;journal=Genetics&amp;volume=97&amp;pages=639-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7297851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bridgham</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Ortlund</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">An epistatic ratchet constrains the direction of glucocorticoid receptor evolution</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">461</div> (pg. <div class="fpage">515</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20epistatic%20ratchet%20constrains%20the%20direction%20of%20glucocorticoid%20receptor%20evolution&amp;author=J.T.%20Bridgham&amp;author=E.A.%20Ortlund&amp;author=J.W.%20Thornton&amp;publication_year=2009&amp;journal=Nature&amp;volume=461&amp;pages=515-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08249" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19779450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Natarajan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Inoguchi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Weber</div> <div class="given-names">R.E.</div></div>, et al. </div><div class="article-title">Epistasis among adaptive mutations in deer mouse hemoglobin</div>, <div class="source">Science</div> , <div class="year">2013</div>, vol. <div class="volume">340</div> (pg. <div class="fpage">1324</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epistasis%20among%20adaptive%20mutations%20in%20deer%20mouse%20hemoglobin&amp;author=C.%20Natarajan&amp;author=N.%20Inoguchi&amp;author=R.E.%20Weber&amp;publication_year=2013&amp;journal=Science&amp;volume=340&amp;pages=1324-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1236862" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1236862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1236862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23766324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu040c59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu040c59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gore</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Chia</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Elshire</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A first-generation haplotype map of maize</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1115</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20first-generation%20haplotype%20map%20of%20maize&amp;author=M.A.%20Gore&amp;author=J.M.%20Chia&amp;author=R.%20Elshire&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1115-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1177837" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1177837" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1177837"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=441064&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_tables1.doc?Expires=1509238978&amp;Signature=eQbJCdZZGvJlxYldcMuqMz0V3dXMCtPMwkWNZe8Ck8~ekfzrrlN7CtZspz7E6CQLPAcTmf9nyWrSgxvbQSkHwYnPPLSNRgHYWEdPNkUw5AcTJkJ4VspdsPgD7zza8YSLla43MdypJunNXyvicuJP4LQeqk-bV1fXXdTWrnHV7Mw7nM9B1W8U27djFwkUJK4kR0q51ZtOUCRDVsYpeTXHURyiQ-gVwL87Agdj-znb0keziKxzcQaq8zXsg9XxFRErxMVUGgTtg1YyfhxbYizflpUncYbCyLalAkILgLYIR~5NKlfrWEPgcpJQR3M157e-J6E0S7lKuCajGxS1rVAcHg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-15</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table9.doc?Expires=1509238978&amp;Signature=dGXgBV1ERIMvzrJFJ8Ltncn6pV8PsghyBA7ViPZiGEQHtVFMu60fxKwIkzd3ovmNdrU~ySnNqBj2Jqz2Sh3k0exgPus0k-87D5WV3Kt1SQuQpJJHMDhF2VkRyoEWXKCI7FDLMWYejPreMiYyL4bYrOVJbrD7-cz9us7NUFraTcTa~6ETDZmSj5eRNP80zcL3g~9v3pqIfFgvN3484QilMASeF5jZKP86wJ~x9HW~UtY6mrvmYwL6G6OlYqAunOa0xiyMpUkd7sFETyBSOOaPIYVY8TA4NzalC7IuINwrCJ0IrZidqYeG5nb0R7l7v3tKpajtiNtR~gcpH-0tJfTsXw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-14</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table8.doc?Expires=1509238978&amp;Signature=cvL0WAncAL7KaMHb2~wUjFtezcNgZbZLbzk0b19Cm-~5n6KyuYErcECC8d91u58eSVBMfrkfGSxV29wOeTZcdf9jVd3FY7kG~PGIfE9v3sjBGy3ItL~JX37lgaS1IrZIqZvxIETMBMep8nqFIMW334CgNEhm1Hh8-iSUPbqY8gsGsCnVVRoVUA~OVFkU4GHI1H-JznVks3JFeIg1b~LqCbwG8vEAZBiJuulaWWg68~NVt-c7URz7qZ95rYlnsoDWgF4DBSqe~LeNtD04nYnKnhDAOynvpwWUYE6~9Az8hgYp4ouyPkv4pDX4nVGWpujT6cCF0GQUvPVzJegT5Ly4Kw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-13</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table7.doc?Expires=1509238978&amp;Signature=YgAsKsAnPDj0Pt1ohT9BFamINiiJE5jFgbhd7ibTGB7cy6uvgPrm1vcO96WW4lJ3ZqOIiGHyI4KEdvxTMtn8D4JYEM4kpNTBP4XltskibPU6BEy765xygFX8MiyLFfVVl-oJzQatOyqtcFtBUsWnTpixAeM6wiwE8~mdDYye5Na5CODkwxZVvplAv2qRRCNETfxeyJejdVRYTPXYi2GpX9-3MINJ6cC5qMyUUXtEoetfrtGytJ8YQEdhLHtL8-v~qEwUUtcg7479TspwT1Jm3dEOYQTNSLBfnPmWFR8wQreFc6ZomkWcG4NAcsLDczh2TB8PwZC8iWezklmHcMLIUw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-12</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table6.doc?Expires=1509238978&amp;Signature=SXFHo4597Dhe5LGL9EWCu0iqgaXU0bTkahinUf8AmYuFdF0rbWD2ZMCc8oHSlUgo0-VcqMiP9gkIluPeHzfxBlMje7WGGXqMTz7yq9QnEB6eLzVQpl2m14DtvllelI5X97r3rfQTcE1LGNAOIoqi7uMQqW3bUc-IESMVScRfEU-b0cs85Cu9Em2ue~sb30ce1vpiIswLx8JqseU~HG58ftRlaV6lpi6RI9YJvIOOtQ9SAsY6kKhsb8U1wLJ6lmdl6rz~6YvMyBHwmYFsFUgeCIZrIx83L5DJJ8vomQxhYe0HVu172ymUbOFiLGjVEj5Tsm8~i9OAOpBnroypP9yg7g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-11</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table5.doc?Expires=1509238978&amp;Signature=Sl87hRGnIKcR9AOll4fktrMu6Jh8J-nK8pA7lvXO3rm3oE~svpAZR6j6JU7cLc6MGAHiSdYObxBM5lQ5mJQmlbh2hMC7nwBocP29cNAnNWnLiw1NMTXiIr9uIpAS~BEd7GGj6YsFkgc~bllTpzHzVeGiPZ50pR2-cWWCAdOTcCbdbVwf0aGkBavBrV1KT1CgWUqw0i5SYGFrnl4g7PesrhilWu7aI-MP5D6tvkfsR0lqueTmWElH699Y8acEPGD0~87JpAnKEw1dmb9OBUgR1h7bGjGAcqZxsmy7Bmg-iPCwPd~WW2cfhYUJLf9yaN~4hHeLtjAXDHY4b5sE7gXzAw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-10</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table4.doc?Expires=1509238978&amp;Signature=arskKqEjhzVsauyuYXzoW4khQR07EM-MM~fX8Do7t2iJftlcE3DNXvEFSMbS8lfB4TI~B4brsO2T3h4bLX43NEDGKH9konK7bSKsSLRbCHpfutY1WUwhR2lMIw4U4-BqwehXWODFoqAwHyDbebFlr0~BDr7fQHR8kesfYrvbpevk81~0shkcJZo1KU33n3~7C~vf-RBng~WL~EYwQxipEHK7k4mtxkfg-VApitQi7nym9ZAnTYiuaGMKIHWt1s55uOuBcz82hHdI6vSNOnxtwMwS8mlrIEj7FoEcPO9Y9W8jtXzltNe55GFAn2nFwgc0eq66tl7tIF9dHABUTTNLNQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-9</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table3.doc?Expires=1509238978&amp;Signature=bdOIIPb4OhEG7T8TC2B9cTiXr-yjnvL5W-vpWwgC-GgKsOXXAArW2QbicpVE~d0muvDSZZZiOpDBA9ixkOKsJG3Q-kaABnkXEG3aYdLVn5WwtIzg7M6V9tMC1J9DQnNRhmYrg5Vg3o8Y6dSYEbNfC0F5r1TzvsRyjd1~bRHMBkgn-wT-k2zWG0kGPHq7AV36ZkydR47m3PHJwN-4xN~VpzA2eraHnozrOXOjGfKzzJ44zi30Gv8KQSWNk9RTWHU-lXwRb8Z-RGatvS5nKHxHQEVGXm3Z0JZewHyX6Wfod7zOU22MV9HKtoUctBwcPIfH3KCyfJqQchm8R2vQ3je5uA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-8</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table2.doc?Expires=1509238978&amp;Signature=KdflnP9PCpYrrUucyoF5i2tcMOc-dI9IquHDlpys6QRal~1gVXGaPpT9CB5hWoNcatDqOR8U1lpDy1qjHoUJl5zD45glOgMRYdSiG8-FI~R~gBkxP~RUHLyiXo4SmB6fiWmL5qMlTzFDIbg1w2pALpLeX-Sv46M~u1i7PgjjcMUoUpE9Q8BDIIsFXvK~hiFgUM6NPmD5fvtlu-vE1IY-ngLfKz8GkrnTM8b7KqETSmgHfVKPFOywjtbkj4y4ULcBMMy8FK20~oDmAF50r1YgE-GqGaHn~Z-MR6arrwlyF-K7GiuRQr6xDSnf~1wJI08~aJqFsOUPSmgVA8Maupofdw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table11.doc?Expires=1509238978&amp;Signature=f9aFbaRSALKGsaKHTnUu~Nl~jcBtC5MBJm4kLzw1rjXZb~6AijuOCBgtz6uZ7Al4x2iNs4qL7X314f4fd~AMYRehhZGBhyYJPkEZ3sRwqTkqAUy4kzy84hrOrTdyMmZ7ARuVfYgZtIyOzuu6c0PEj4Jf3tTZtTAPWdpmhpCcV4lRBBQuel3QuAmdbeKBR2m9ZVz6l~Z7WB1WmvU0qJ~cwK~CjTb683MQ~SW8M--RwS48ouuwcvT3hlpFeORJhCteTu4dEaOetunQoOM4~5ipA7w5sw8VnzvLZYNWWnOEuCapKlOXfpxdGSdZLPpIQnpjj-f52LGPHOzQvyGX2DtVXQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_table10.doc?Expires=1509238978&amp;Signature=Dno6Fas6DZVnZ63s-ViTdqS396icG8FRd~WbnZmNqCIXecQumrXDakj9zPYaevnJn1ZDHfkQ~SPcHl9N-zsdjY8DXbJYdxXpUABdlyv8~TVHDtuZnROJGKMXkS57fM6GHDEXbBK18I7zq3C9Z0S9Wt9ekuAJjQMdlzsAA~LdcL2pp-yhUzh7LiI6N-9jkA~-EMgpi4dKEUsBXbBcX98tiT1R5Uwrt91X6UlDxgnTJN7hQNYfFjLIme0NJhsYWHHnwsvw5s9eAv5xQoN6BD7IggT4xQyKmPlAJLSIT-1HEL16qwHRjfnPuAiiYF6fLldDlhhp7HF6fXIC2yHI~nQVvg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_fig3.doc?Expires=1509238978&amp;Signature=ZS75pFWQ0LOPIMAz7ag43GbKjJ35f6aZZ7HWvMrJBaPONrmng40HwD-dDrgwcgUJ8VZ0E3mp1GnRfD5kBODKSKvxbnjkf-gBVcexRso7hGdBThB~Ixh8UfMuqlRytYaHB4ip~HisBqbQLKcUe-Yafh0DW2LByAJD9~n7fY8pVt5V078WRih~PZKr7ofVXF8newkSGEge~7SPXoiLcYA8YO-Bi8AghxV9nVknBnud7bnAolen~4aQAuOvrC9ZyPRLTCkI3YRf1JjogN85R5TcaKT3ay81tJarG8u-zUIQzJI4Rrd9F3orihYQZmgpT8gD7m799OD1~jmDCciNdiIk1w__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_fig2.doc?Expires=1509238978&amp;Signature=dxUX1KYTv-0NWGCxBjJdSdxNrzge2N5XKt1NN~DJ32NrSXL6twZs8J3x6F3SLLi1ueKBYmYhP-KqT45My6RcEz8OfvKCuhj96MpEgq9qN1gpvaqMJLlxQnx7f1-h2ZmfouCbJPsO-39elND9uAmSeLgCfCTMbrDwhhitQ~p8mqPpf4INoZ5BmKH5VDFiITrQcZDQIbE0ScSzl6TGEo2MIG2zB4YJXH183VPwB1~rV1dHJ-pS8DPOnElicW5CcX~PdPKltnJCqiNKXC2uF-XY8L1-3v-R45SEMbUU3o4VD5zWIyqMIfE1vHrYEcykcytjfmXFmKPihjxZ-mGF0BvSQQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp_fig1.doc?Expires=1509238978&amp;Signature=QqmeN3JCjHk1vY~GN63MKSNstX~LwhQZOPQRZWApBuHhkFZEzGyV64qmlMOifS3Ugectk3aDt1cI-6TBW3tQm41abSp-ixuqlC-4NGu1B-VrmHnX6qubchrM~iMXZT-dO-Hp6AqS6WS~5AOI3YrdoamJxWbiUAWNA-U3JbQn-t0V6~SreFKeSD5Qckt0BtfZzdM3UeBReOao7s9l6Jfn4Uo6ilMOYE3vxZhv-REc2fFn8bNCEgvzd2WtUSVbtYuCo8SUBebfz4kutlQ5QSn2WHAfJhDMe29TXSWGtm8KDcc~fPSnj1VDVYJZ3a6WEJhw9EWAbproeYkMIVr4~KpXfg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu040/2/dsu040-dsu040supp.xlsx?Expires=1509238978&amp;Signature=KxUX2ZRMI4R9RmuJcjoluMpO1scccmryKDjSqGtSkvJ-jFfZiRUuD7PNADMroJfAVTzFZ0VlPGGuMO4zegUo2hTugNRQ48nbUhRUOWen2C4NoixKYk8zrDqvV2NUJIWVKIhyih-b2fOhcAGsGflw~FBDMs9OBrhlLlMRv3B~ExXkqQzVQ3IRrR2rSFOEWL7LIj1Pf2ku0xWXvhVp~2Px9rWColiCxXLwVi1qP0cc~Y7RjnjvjdtbAlEHZLfufVQo6zwMmGsTjmaS-IbAj6D09U0AvBmhh3Lo4t2PSbew35FLrTusmvDLH42hqRDWrtPBhtyuHmZBWKwPxx4B4He9tg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">105</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsu040"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="441064"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f53%2f441064">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F53%2F441064" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/441064'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification of additive, dominant, and epistatic variation conferred by key genes in cellulose biosynthesis pathway in Populus tomentosa†" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f53%2f441064">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

