

<!DOCTYPE html>
<html lang="en">

<head><title>
	LTR retrotransposon dynamics in the evolution of the olive (Olea europaea) genome | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/91/441732","@type":"ScholarlyArticle","name":"LTR retrotransposon dynamics in the evolution of the olive ( Olea europaea ) genome","datePublished":"2014-11-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu042","keywords":["LTR retrotransposons","next-generation sequencing","olive","insertion age","BAC sequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/91/441732","author":[{"name":"Barghini, Elena","affiliation":"1  Department of Agricultural, Food, and Environmental Sciences ,  University of Pisa ,  Pisa I-56124 ,  Italy","@type":"Person"},{"name":"Natali, Lucia","affiliation":"1  Department of Agricultural, Food, and Environmental Sciences ,  University of Pisa ,  Pisa I-56124 ,  Italy","@type":"Person"},{"name":"Giordani, Tommaso","affiliation":"1  Department of Agricultural, Food, and Environmental Sciences ,  University of Pisa ,  Pisa I-56124 ,  Italy","@type":"Person"},{"name":"Cossu, Rosa Maria","affiliation":"1  Department of Agricultural, Food, and Environmental Sciences ,  University of Pisa ,  Pisa I-56124 ,  Italy    2  Institute of Life Sciences ,  Scuola Superiore Sant'Anna ,  Pisa ,  Italy","@type":"Person"},{"name":"Scalabrin, Simone","affiliation":"3  IGA Technology Service ,  Udine ,  Italy","@type":"Person"},{"name":"Cattonaro, Federica","affiliation":"3  IGA Technology Service ,  Udine ,  Italy","@type":"Person"},{"name":"Šimková, Hana","affiliation":"4  Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research ,  Olomouc ,  Czech Republic","@type":"Person"},{"name":"Vrána, Jan","affiliation":"4  Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research ,  Olomouc ,  Czech Republic","@type":"Person"},{"name":"Doležel, Jaroslav","affiliation":"4  Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research ,  Olomouc ,  Czech Republic","@type":"Person"},{"name":"Morgante, Michele","affiliation":"5  Department of Crop and Environmental Sciences ,  University of Udine ,  Udine ,  Italy    6  Institute of Applied Genomics ,  Udine ,  Italy","@type":"Person"},{"name":"Cavallini, Andrea","affiliation":"1  Department of Agricultural, Food, and Environmental Sciences ,  University of Pisa ,  Pisa I-56124 ,  Italy","@type":"Person"}],"description":"Improved knowledge of genome composition, especially of its repetitive component, generates important information for both theoretical and applied research. The olive repetitive component is made up of two main classes of sequences: tandem repeats and retrotransposons (REs). In this study, we provide characterization of a sample of 254 unique full-length long terminal repeat (LTR) REs. In the sample, Ty1-Copia elements were more numerous than Ty3-Gypsy elements. Mapping a large set of Illumina whole-genome shotgun reads onto the identified retroelement set revealed that Gypsy elements are more redundant than Copia elements. The insertion time of intact retroelements was estimated based on sister LTR’s divergence. Although some elements inserted relatively recently, the mean insertion age of the isolated retroelements is around 18 million yrs. Gypsy and Copia retroelements showed different waves of transposition, with Gypsy elements especially active between 10 and 25 million yrs ago and nearly inactive in the last 7 million yrs. The occurrence of numerous solo-LTRs related to isolated full-length retroelements was ascertained for two Gypsy elements and one Copia element. Overall, the results reported in this study show that RE activity (both retrotransposition and DNA loss) has impacted the olive genome structure in more ancient times than in other angiosperms.","pageStart":"91","pageEnd":"100","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217384&Signature=GMmPtJNw8AvOcJvccPXGhQIA~gUZCAgvFWhCXk3NIo-3S96rAwrRaifgHgDvJ~HmNhy0ju9i8ayDbkjknZOKDHFrkmHAu4cHFuTrr0fiVb98NCx0VLu6CQFWm8QbiEibXxDHgVbo4iDOP-f3AFhmv9~KtgvjTrhrdsnkl5YqWGIy0cFxDtcNyWvMojZPtcK7ZSFA8HG-flOcdrzApQIluE2lr9h5raZ3rSXIu~EX-IFvLbW1aVDQWl1p0CYn-M~a2aPdoQAXlb6VkcCGUzHFZpWeeIhBw7aW36uKvYD7pGRxVbC12AvOKRq-KlFr3VQF2AiB~UVg7X~uT7WnsoJaRA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"LTR retrotransposon dynamics in the evolution of the olive ( Olea europaea ) genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217384&Signature=GMmPtJNw8AvOcJvccPXGhQIA~gUZCAgvFWhCXk3NIo-3S96rAwrRaifgHgDvJ~HmNhy0ju9i8ayDbkjknZOKDHFrkmHAu4cHFuTrr0fiVb98NCx0VLu6CQFWm8QbiEibXxDHgVbo4iDOP-f3AFhmv9~KtgvjTrhrdsnkl5YqWGIy0cFxDtcNyWvMojZPtcK7ZSFA8HG-flOcdrzApQIluE2lr9h5raZ3rSXIu~EX-IFvLbW1aVDQWl1p0CYn-M~a2aPdoQAXlb6VkcCGUzHFZpWeeIhBw7aW36uKvYD7pGRxVbC12AvOKRq-KlFr3VQF2AiB~UVg7X~uT7WnsoJaRA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="LTR retrotransposon dynamics in the evolution of the olive ( Olea europaea ) genome | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Improved knowledge of genome composition, especially of its repetitive component, generates important information for both theoretical and applied research. The olive repetitive component is made up of two main classes of sequences: tandem repeats and retrotransposons (REs). In this study, we provide characterization of a sample of 254 unique full-length long terminal repeat (LTR) REs. In the sample, Ty1-Copia elements were more numerous than Ty3-Gypsy elements. Mapping a large set of Illumina whole-genome shotgun reads onto the identified retroelement set revealed that Gypsy elements are more redundant than Copia elements. The insertion time of intact retroelements was estimated based on sister LTR’s divergence. Although some elements inserted relatively recently, the mean insertion age of the isolated retroelements is around 18 million yrs. Gypsy and Copia retroelements showed different waves of transposition, with Gypsy elements especially active between 10 and 25 million yrs ago and nearly inactive in the last 7 million yrs. The occurrence of numerous solo-LTRs related to isolated full-length retroelements was ascertained for two Gypsy&#160;elements and one Copia element. Overall, the results reported in this study show that RE activity (both retrotransposition and DNA loss) has impacted the olive genome structure in more ancient times than in other angiosperms." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/91/441732" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217384&amp;Signature=SuVnA0KibiI5CRduGZ2Dnf6gZdy78xHWoik5ByiFfhcdv5rvNELUTDJxfAHLV3kLoQJ6CRTGDEWzl3RvtzwlsrTxZpBbdTZee~QhOMkdQtTl2RVEahbsXAxOlRtQuVgc6nvBjXJt762i6cQkiZAAjmRcIZhUfNVdE1wO~UPqohD2KaqmdNgOo~nvV~HgK~gue~HvzxwHmjnk7XKpGEH1-QOUFI59E~UVkfZcBeISFJT2FM-X9ds~XDSzmCp7LEhJmyba1WcjOHLLM06iEKrdBqbwVtNyRCMgj~cVUBAS~6WU6gpJaaYNjGhBSfZ6aIvQeXCTi4jobZgGWAdBBL8EMg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2014-11-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Barghini, Elena" /><meta name="citation_author_institution" content="Department of Agricultural, Food, and Environmental Sciences, University of Pisa, Pisa I-56124, Italy" /><meta name="citation_author" content="Natali, Lucia" /><meta name="citation_author_institution" content="Department of Agricultural, Food, and Environmental Sciences, University of Pisa, Pisa I-56124, Italy" /><meta name="citation_author" content="Giordani, Tommaso" /><meta name="citation_author_institution" content="Department of Agricultural, Food, and Environmental Sciences, University of Pisa, Pisa I-56124, Italy" /><meta name="citation_author" content="Cossu, Rosa Maria" /><meta name="citation_author_institution" content="Department of Agricultural, Food, and Environmental Sciences, University of Pisa, Pisa I-56124, Italy" /><meta name="citation_author_institution" content="Institute of Life Sciences, Scuola Superiore Sant'Anna, Pisa, Italy" /><meta name="citation_author" content="Scalabrin, Simone" /><meta name="citation_author_institution" content="IGA Technology Service, Udine, Italy" /><meta name="citation_author" content="Cattonaro, Federica" /><meta name="citation_author_institution" content="IGA Technology Service, Udine, Italy" /><meta name="citation_author" content="Šimková, Hana" /><meta name="citation_author_institution" content="Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research, Olomouc, Czech Republic" /><meta name="citation_author" content="Vrána, Jan" /><meta name="citation_author_institution" content="Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research, Olomouc, Czech Republic" /><meta name="citation_author" content="Doležel, Jaroslav" /><meta name="citation_author_institution" content="Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research, Olomouc, Czech Republic" /><meta name="citation_author" content="Morgante, Michele" /><meta name="citation_author_institution" content="Department of Crop and Environmental Sciences, University of Udine, Udine, Italy" /><meta name="citation_author_institution" content="Institute of Applied Genomics, Udine, Italy" /><meta name="citation_author" content="Cavallini, Andrea" /><meta name="citation_author_institution" content="Department of Agricultural, Food, and Environmental Sciences, University of Pisa, Pisa I-56124, Italy" /><meta name="citation_title" content="LTR retrotransposon dynamics in the evolution of the olive (Olea europaea) genome" /><meta name="citation_firstpage" content="91" /><meta name="citation_lastpage" content="100" /><meta name="citation_doi" content="10.1093/dnares/dsu042" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Olive oil as a functional food: epidemiology and nutritional approaches; Nutrition Rev.;  citation_year=2002;  citation_volume=60;  citation_pages=170-6; " /><meta name="citation_reference" content="citation_title=Microsatellite markers are powerful tools for discriminating among olive cultivars and assigning them to geographically defined populations; Genome;  citation_year=2006;  citation_volume=49;  citation_pages=1606-15; " /><meta name="citation_reference" content="citation_title=Genetic relationships in the olive (Olea europaea L.) reflect multilocal selection of cultivars; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=251-8; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content estimations in wild olive (Olea europaea L. ssp. europaea var. sylvestris Brot.) and Portuguese cultivars of O. europaea using flow cytometry; Genet. Res. Crop Evol.;  citation_year=2007;  citation_volume=54;  citation_pages=21-5; " /><meta name="citation_reference" content="citation_title=Genomic organization, sequence interrelationship, and physical localization using in situ hybridization of two tandemly repeated DNA sequences in the genus Olea; Genome;  citation_year=1998;  citation_volume=41;  citation_pages=527-34; " /><meta name="citation_reference" content="citation_title=Amount and organization of the heterochromatin in Olea europaea and related species; Heredity;  citation_year=1999;  citation_volume=83;  citation_pages=188-95; " /><meta name="citation_reference" content="citation_title=A new repetitive DNA sequence family in the olive (Olea europaea L); Hereditas;  citation_year=2001;  citation_volume=134;  citation_pages=73-8; " /><meta name="citation_reference" content="citation_title=Genomic and chromosomal organization of Ty1-Copia-like sequences in Olea europaea and evolutionary relationships of Olea retroelements; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=926-33; " /><meta name="citation_reference" content="citation_title=Isolation of Ty1-Copia putative LTR sequences and their use as a tool to analyse genetic diversity in Olea europaea; Mol. Breed.;  citation_year=2007;  citation_volume=19;  citation_pages=255-65; " /><meta name="citation_reference" content="citation_title=A genome triplication associated with early diversification of the core eudicots; Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=R3" /><meta name="citation_reference" content="citation_title=Differential lineage-specific amplification of transposable elements is responsible for genome size variation in Gossypium; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1252-61; " /><meta name="citation_reference" content="citation_title=Plant retrotransposons; Ann. Rev. Genet.;  citation_year=1999;  citation_volume=33;  citation_pages=479-532; " /><meta name="citation_reference" content="citation_title=Three Tnt1 subfamilies show different stress-associated patterns of expression in tobacco. Consequences for retrotransposon control and evolution in plants; Plant Physiol.;  citation_year=2001;  citation_volume=127;  citation_pages=212-21; " /><meta name="citation_reference" content="citation_title=Evolution of DNA sequence nonhomologies among maize inbreds; Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=343-60; " /><meta name="citation_reference" content="citation_title=The Sorghum bicolor genome and the diversification of grasses; Nature;  citation_year=2009;  citation_volume=457;  citation_pages=551-6; " /><meta name="citation_reference" content="citation_title=Temporal dynamics in the evolution of the sunflower genome as revealed by sequencing and annotation of three large genomic regions; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=779-91; " /><meta name="citation_reference" content="citation_title=A computational study of the dynamics of LTR retrotransposons in the Populus trichocarpa genome; Tree Genet. Genomes;  citation_year=2012;  citation_volume=8;  citation_pages=61-75; " /><meta name="citation_reference" content="citation_title=The peculiar landscape of repetitive sequences in the olive (Olea europaea L.) genome; Genome Biol. Evol.;  citation_year=2014;  citation_volume=6;  citation_pages=776-91; " /><meta name="citation_reference" content="citation_title=The paleontology of intergene retrotransposons of maize; Nat. Genet.;  citation_year=1998;  citation_volume=20;  citation_pages=43-5; " /><meta name="citation_reference" content="citation_title=A high-yield procedure for isolation of metaphase chromosomes from root tips of Vicia faba L.; Planta;  citation_year=1992;  citation_volume=188;  citation_pages=93-8; " /><meta name="citation_reference" content="citation_title=Preparation of HMW DNA from plant nuclei and chromosomes isolated from root tips; Biol. Plant;  citation_year=2003;  citation_volume=46;  citation_pages=369-73; " /><meta name="citation_reference" content="citation_title=BAC libraries from wheat chromosome 7D: efficient tool for positional cloning of aphid resistance genes; J. Biomed. Biotechnol.;  citation_year=2011;  citation_volume=20;  citation_issue=11;  citation_pages=302543" /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; EMBnet. J.;  citation_year=2011;  citation_volume=17;  citation_pages=10-12; " /><meta name="citation_reference" content="citation_title=ABySS: a parallel assembler for short read sequence data; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1117-23; " /><meta name="citation_reference" content="citation_title=LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons; Nucl. Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W265-8; " /><meta name="citation_reference" content="citation_title=A dot-matriprogram with dynamic threshold control suited for genomic DNA and protein sequence analysis; Gene;  citation_year=1995;  citation_volume=167;  citation_pages=GC1-GC10; " /><meta name="citation_reference" content="citation_title=CLUSTALW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucl. Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=De novo assembly and functional annotation of the olive (Olea europaea) transcriptome; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=93-108; " /><meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions; Mol. Biol. Evol.;  citation_year=1986;  citation_volume=3;  citation_pages=418-26; " /><meta name="citation_reference" content="citation_title=DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=174-5; " /><meta name="citation_reference" content="citation_title=Phylogenetics of Olea (Oleaceae) based on plastid and nuclear ribosomal DNA sequences: tertiary climatic shifts and lineage differentiation times; Ann. Bot.;  citation_year=2009;  citation_volume=104;  citation_pages=143-60; " /><meta name="citation_reference" content="citation_title=Rapid recent growth and divergence of rice nuclear genomes; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=12404-10; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of eukaryotic repetitive elements; Cytogenet. Genome Res.;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=Evolutionary genomics of chromoviruses in eukaryotes; Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=781-98; " /><meta name="citation_reference" content="citation_title=The Norway spruce genome sequence and conifer genome evolution; Nature;  citation_year=2013;  citation_volume=497;  citation_pages=579-84; " /><meta name="citation_reference" content="citation_title=Molecular clocks and nucleotide substitution rates in higher plants; citation_publisher=Plenum Press, New York; Evolutionary Biology;  citation_year=1998;  citation_volume=30;  citation_pages=93-120; " /><meta name="citation_reference" content="citation_title=The genome of the cucumber, Cucumis sativus L.; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=1275-81; " /><meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; " /><meta name="citation_reference" content="citation_title=The bright side of transposons in crop evolution; citation_text=.; Brief. Funct. Genomics;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=The draft genome of the transgenic tropical fruit tree papaya (Carica papaya Linnaeus); Nature;  citation_year=2008;  citation_volume=452;  citation_pages=991-7; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla; citation_author=The French-Italian Public Consortium for Grape Genome Characterization; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=Plant centromeric retrotransposons: a structural and cytogenetic perspective; Mobile DNA;  citation_year=2011;  citation_volume=2;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Analyses of LTR-retrotransposon structures reveal recent and rapid genomic DNA loss in rice; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=860-9; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of an ancient retrotransposon family provides insights into evolution of genome size in the genus Oryza; Plant J.;  citation_year=2007;  citation_volume=52;  citation_pages=342-51; " /><meta name="citation_reference" content="citation_title=Evolution of genome size and complexity in Pinus; PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e4332" /><meta name="citation_reference" content="citation_title=Analysis of transposons and repeat composition of the sunflower (Helianthus annuus L.) genome; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=491-508; " /><meta name="citation_reference" content="citation_title=The repetitive component of the sunflower genome as revealed by different procedures for assembling next generation sequencing reads; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=686" /><meta name="citation_reference" content="citation_title=Assessing the extent of substitution rate variation of retrotransposon long terminal repeat sequences in Oryza sativa and Oryza glaberrima; Rice;  citation_year=2010;  citation_volume=3;  citation_pages=242-50; " /><meta name="citation_reference" content="citation_title=Proliferation of Ty3/gypsy-like retrotransposons in hybrid sunflower taxa inferred from phylogenetic data; BMC Biol.;  citation_year=2009;  citation_volume=7;  citation_pages=40" /><meta name="citation_reference" content="citation_title=Genetic variability in sunﬂower (Helianthus annuus L.) and in the Helianthus genus as assessed by retrotransposon-based molecular markers; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=119;  citation_pages=1027-38; " /><meta name="citation_reference" content="citation_title=Natural selection on gene function drives the evolution of LTR retrotransposon families in the rice genome; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=243-54; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of transposable elements in plants; Ann. Rev. Plant Biol.;  citation_year=2009;  citation_volume=60;  citation_pages=43-66; " /><meta name="citation_reference" content="citation_title=Exceptional diversity, non-random distribution, and rapid evolution of retroelements in the B73 maize genome; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000732" /><meta name="citation_reference" content="citation_title=Genome ecosystem and transposable elements species; Gene;  citation_year=2007;  citation_volume=390;  citation_pages=214-20; " /><meta name="citation_reference" content="citation_title=Dynamics of transposable elements: towards a community ecology of the genome; Trends Genet.;  citation_year=2009;  citation_volume=25;  citation_pages=317-23; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/91/1210761/dsu042.pdf" /><meta name="description" content="Abstract.  Improved knowledge of genome composition, especially of its repetitive component, generates important information for both theoretical and applied re" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/91/441732" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/91/441732" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsu042';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./441732" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFjIEBPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYXJnaGluaSwgRWxlbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCwgRm9vZCwgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgUGlzYSwgUGlzYSBJLTU2MTI0LCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5hdGFsaSwgTHVjaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCwgRm9vZCwgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgUGlzYSwgUGlzYSBJLTU2MTI0LCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikdpb3JkYW5pLCBUb21tYXNvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyYWwsIEZvb2QsIGFuZCBFbnZpcm9ubWVudGFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFBpc2EsIFBpc2EgSS01NjEyNCwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb3NzdSwgUm9zYSBNYXJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsLCBGb29kLCBhbmQgRW52aXJvbm1lbnRhbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBQaXNhLCBQaXNhIEktNTYxMjQsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIExpZmUgU2NpZW5jZXMsIFNjdW9sYSBTdXBlcmlvcmUgU2FudCdBbm5hLCBQaXNhLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjYWxhYnJpbiwgU2ltb25lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSUdBIFRlY2hub2xvZ3kgU2VydmljZSwgVWRpbmUsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2F0dG9uYXJvLCBGZWRlcmljYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklHQSBUZWNobm9sb2d5IFNlcnZpY2UsIFVkaW5lLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IsWgaW1rb3bDoSwgSGFuYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFeHBlcmltZW50YWwgQm90YW55LCBDZW50cmUgb2YgdGhlIFJlZ2lvbiBIYW7DoSBmb3IgQmlvdGVjaG5vbG9naWNhbCBhbmQgQWdyaWN1bHR1cmFsIFJlc2VhcmNoLCBPbG9tb3VjLCBDemVjaCBSZXB1YmxpYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZyw6FuYSwgSmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEV4cGVyaW1lbnRhbCBCb3RhbnksIENlbnRyZSBvZiB0aGUgUmVnaW9uIEhhbsOhIGZvciBCaW90ZWNobm9sb2dpY2FsIGFuZCBBZ3JpY3VsdHVyYWwgUmVzZWFyY2gsIE9sb21vdWMsIEN6ZWNoIFJlcHVibGljIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRG9sZcW+ZWwsIEphcm9zbGF2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEV4cGVyaW1lbnRhbCBCb3RhbnksIENlbnRyZSBvZiB0aGUgUmVnaW9uIEhhbsOhIGZvciBCaW90ZWNobm9sb2dpY2FsIGFuZCBBZ3JpY3VsdHVyYWwgUmVzZWFyY2gsIE9sb21vdWMsIEN6ZWNoIFJlcHVibGljIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yZ2FudGUsIE1pY2hlbGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENyb3AgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgVWRpbmUsIFVkaW5lLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBBcHBsaWVkIEdlbm9taWNzLCBVZGluZSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDYXZhbGxpbmksIEFuZHJlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsLCBGb29kLCBhbmQgRW52aXJvbm1lbnRhbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBQaXNhLCBQaXNhIEktNTYxMjQsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJMVFIgcmV0cm90cmFuc3Bvc29uIGR5bmFtaWNzIGluIHRoZSBldm9sdXRpb24gb2YgdGhlIG9saXZlIChPbGVhIGV1cm9wYWVhKSBnZW5vbWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI5MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbGl2ZSBvaWwgYXMgYSBmdW5jdGlvbmFsIGZvb2Q6IGVwaWRlbWlvbG9neSBhbmQgbnV0cml0aW9uYWwgYXBwcm9hY2hlczsgTnV0cml0aW9uIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0xNzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGUgbWFya2VycyBhcmUgcG93ZXJmdWwgdG9vbHMgZm9yIGRpc2NyaW1pbmF0aW5nIGFtb25nIG9saXZlIGN1bHRpdmFycyBhbmQgYXNzaWduaW5nIHRoZW0gdG8gZ2VvZ3JhcGhpY2FsbHkgZGVmaW5lZCBwb3B1bGF0aW9uczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MTYwNi0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyByZWxhdGlvbnNoaXBzIGluIHRoZSBvbGl2ZSAoT2xlYSBldXJvcGFlYSBMLikgcmVmbGVjdCBtdWx0aWxvY2FsIHNlbGVjdGlvbiBvZiBjdWx0aXZhcnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MjUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgRE5BIGNvbnRlbnQgZXN0aW1hdGlvbnMgaW4gd2lsZCBvbGl2ZSAoT2xlYSBldXJvcGFlYSBMLiBzc3AuIGV1cm9wYWVhIHZhci4gc3lsdmVzdHJpcyBCcm90LikgYW5kIFBvcnR1Z3Vlc2UgY3VsdGl2YXJzIG9mIE8uIGV1cm9wYWVhIHVzaW5nIGZsb3cgY3l0b21ldHJ5OyBHZW5ldC4gUmVzLiBDcm9wIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MjEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBvcmdhbml6YXRpb24sIHNlcXVlbmNlIGludGVycmVsYXRpb25zaGlwLCBhbmQgcGh5c2ljYWwgbG9jYWxpemF0aW9uIHVzaW5nIGluIHNpdHUgaHlicmlkaXphdGlvbiBvZiB0d28gdGFuZGVtbHkgcmVwZWF0ZWQgRE5BIHNlcXVlbmNlcyBpbiB0aGUgZ2VudXMgT2xlYTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NTI3LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbW91bnQgYW5kIG9yZ2FuaXphdGlvbiBvZiB0aGUgaGV0ZXJvY2hyb21hdGluIGluIE9sZWEgZXVyb3BhZWEgYW5kIHJlbGF0ZWQgc3BlY2llczsgSGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9ODM7ICBjaXRhdGlvbl9wYWdlcz0xODgtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IHJlcGV0aXRpdmUgRE5BIHNlcXVlbmNlIGZhbWlseSBpbiB0aGUgb2xpdmUgKE9sZWEgZXVyb3BhZWEgTCk7IEhlcmVkaXRhczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz03My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGFuZCBjaHJvbW9zb21hbCBvcmdhbml6YXRpb24gb2YgVHkxLUNvcGlhLWxpa2Ugc2VxdWVuY2VzIGluIE9sZWEgZXVyb3BhZWEgYW5kIGV2b2x1dGlvbmFyeSByZWxhdGlvbnNoaXBzIG9mIE9sZWEgcmV0cm9lbGVtZW50czsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz05MjYtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiBUeTEtQ29waWEgcHV0YXRpdmUgTFRSIHNlcXVlbmNlcyBhbmQgdGhlaXIgdXNlIGFzIGEgdG9vbCB0byBhbmFseXNlIGdlbmV0aWMgZGl2ZXJzaXR5IGluIE9sZWEgZXVyb3BhZWE7IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MjU1LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZSB0cmlwbGljYXRpb24gYXNzb2NpYXRlZCB3aXRoIGVhcmx5IGRpdmVyc2lmaWNhdGlvbiBvZiB0aGUgY29yZSBldWRpY290czsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9UjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgbGluZWFnZS1zcGVjaWZpYyBhbXBsaWZpY2F0aW9uIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpcyByZXNwb25zaWJsZSBmb3IgZ2Vub21lIHNpemUgdmFyaWF0aW9uIGluIEdvc3N5cGl1bTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMjUyLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCByZXRyb3RyYW5zcG9zb25zOyBBbm4uIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NDc5LTUzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhyZWUgVG50MSBzdWJmYW1pbGllcyBzaG93IGRpZmZlcmVudCBzdHJlc3MtYXNzb2NpYXRlZCBwYXR0ZXJucyBvZiBleHByZXNzaW9uIGluIHRvYmFjY28uIENvbnNlcXVlbmNlcyBmb3IgcmV0cm90cmFuc3Bvc29uIGNvbnRyb2wgYW5kIGV2b2x1dGlvbiBpbiBwbGFudHM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyNzsgIGNpdGF0aW9uX3BhZ2VzPTIxMi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIEROQSBzZXF1ZW5jZSBub25ob21vbG9naWVzIGFtb25nIG1haXplIGluYnJlZHM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zNDMtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTb3JnaHVtIGJpY29sb3IgZ2Vub21lIGFuZCB0aGUgZGl2ZXJzaWZpY2F0aW9uIG9mIGdyYXNzZXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NTc7ICBjaXRhdGlvbl9wYWdlcz01NTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVtcG9yYWwgZHluYW1pY3MgaW4gdGhlIGV2b2x1dGlvbiBvZiB0aGUgc3VuZmxvd2VyIGdlbm9tZSBhcyByZXZlYWxlZCBieSBzZXF1ZW5jaW5nIGFuZCBhbm5vdGF0aW9uIG9mIHRocmVlIGxhcmdlIGdlbm9taWMgcmVnaW9uczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz03NzktOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHV0YXRpb25hbCBzdHVkeSBvZiB0aGUgZHluYW1pY3Mgb2YgTFRSIHJldHJvdHJhbnNwb3NvbnMgaW4gdGhlIFBvcHVsdXMgdHJpY2hvY2FycGEgZ2Vub21lOyBUcmVlIEdlbmV0LiBHZW5vbWVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz02MS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBlY3VsaWFyIGxhbmRzY2FwZSBvZiByZXBldGl0aXZlIHNlcXVlbmNlcyBpbiB0aGUgb2xpdmUgKE9sZWEgZXVyb3BhZWEgTC4pIGdlbm9tZTsgR2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03NzYtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwYWxlb250b2xvZ3kgb2YgaW50ZXJnZW5lIHJldHJvdHJhbnNwb3NvbnMgb2YgbWFpemU7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLXlpZWxkIHByb2NlZHVyZSBmb3IgaXNvbGF0aW9uIG9mIG1ldGFwaGFzZSBjaHJvbW9zb21lcyBmcm9tIHJvb3QgdGlwcyBvZiBWaWNpYSBmYWJhIEwuOyBQbGFudGE7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9OTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlcGFyYXRpb24gb2YgSE1XIEROQSBmcm9tIHBsYW50IG51Y2xlaSBhbmQgY2hyb21vc29tZXMgaXNvbGF0ZWQgZnJvbSByb290IHRpcHM7IEJpb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MzY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CQUMgbGlicmFyaWVzIGZyb20gd2hlYXQgY2hyb21vc29tZSA3RDogZWZmaWNpZW50IHRvb2wgZm9yIHBvc2l0aW9uYWwgY2xvbmluZyBvZiBhcGhpZCByZXNpc3RhbmNlIGdlbmVzOyBKLiBCaW9tZWQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25faXNzdWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zMDI1NDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXRhZGFwdCByZW1vdmVzIGFkYXB0ZXIgc2VxdWVuY2VzIGZyb20gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgcmVhZHM7IEVNQm5ldC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUJ5U1M6IGEgcGFyYWxsZWwgYXNzZW1ibGVyIGZvciBzaG9ydCByZWFkIHNlcXVlbmNlIGRhdGE7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTExNy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TFRSX0ZJTkRFUjogYW4gZWZmaWNpZW50IHRvb2wgZm9yIHRoZSBwcmVkaWN0aW9uIG9mIGZ1bGwtbGVuZ3RoIExUUiByZXRyb3RyYW5zcG9zb25zOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9VzI2NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRvdC1tYXRyaXByb2dyYW0gd2l0aCBkeW5hbWljIHRocmVzaG9sZCBjb250cm9sIHN1aXRlZCBmb3IgZ2Vub21pYyBETkEgYW5kIHByb3RlaW4gc2VxdWVuY2UgYW5hbHlzaXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9R0MxLUdDMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUxXOiBpbXByb3ZpbmcgdGhlIHNlbnNpdGl2aXR5IG9mIHByb2dyZXNzaXZlIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB0aHJvdWdoIHNlcXVlbmNlIHdlaWdodGluZywgcG9zaXRpb24tc3BlY2lmaWMgZ2FwIHBlbmFsdGllcyBhbmQgd2VpZ2h0IG1hdHJpeCBjaG9pY2U7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IGFuZCBmdW5jdGlvbmFsIGFubm90YXRpb24gb2YgdGhlIG9saXZlIChPbGVhIGV1cm9wYWVhKSB0cmFuc2NyaXB0b21lOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTkzLTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIG1ldGhvZHMgZm9yIGVzdGltYXRpbmcgdGhlIG51bWJlcnMgb2Ygc3lub255bW91cyBhbmQgbm9uc3lub255bW91cyBudWNsZW90aWRlIHN1YnN0aXR1dGlvbnM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQxOC0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG5hU1AgdmVyc2lvbiAzOiBhbiBpbnRlZ3JhdGVkIHByb2dyYW0gZm9yIG1vbGVjdWxhciBwb3B1bGF0aW9uIGdlbmV0aWNzIGFuZCBtb2xlY3VsYXIgZXZvbHV0aW9uIGFuYWx5c2lzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWNzIG9mIE9sZWEgKE9sZWFjZWFlKSBiYXNlZCBvbiBwbGFzdGlkIGFuZCBudWNsZWFyIHJpYm9zb21hbCBETkEgc2VxdWVuY2VzOiB0ZXJ0aWFyeSBjbGltYXRpYyBzaGlmdHMgYW5kIGxpbmVhZ2UgZGlmZmVyZW50aWF0aW9uIHRpbWVzOyBBbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTQzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCByZWNlbnQgZ3Jvd3RoIGFuZCBkaXZlcmdlbmNlIG9mIHJpY2UgbnVjbGVhciBnZW5vbWVzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xMjQwNC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwYmFzZSBVcGRhdGUsIGEgZGF0YWJhc2Ugb2YgZXVrYXJ5b3RpYyByZXBldGl0aXZlIGVsZW1lbnRzOyBDeXRvZ2VuZXQuIEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZ2Vub21pY3Mgb2YgY2hyb21vdmlydXNlcyBpbiBldWthcnlvdGVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NzgxLTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTm9yd2F5IHNwcnVjZSBnZW5vbWUgc2VxdWVuY2UgYW5kIGNvbmlmZXIgZ2Vub21lIGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NzsgIGNpdGF0aW9uX3BhZ2VzPTU3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb2NrcyBhbmQgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb24gcmF0ZXMgaW4gaGlnaGVyIHBsYW50czsgY2l0YXRpb25fcHVibGlzaGVyPVBsZW51bSBQcmVzcywgTmV3IFlvcms7IEV2b2x1dGlvbmFyeSBCaW9sb2d5OyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9OTMtMTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBjdWN1bWJlciwgQ3VjdW1pcyBzYXRpdnVzIEwuOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNzUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCNzMgbWFpemUgZ2Vub21lOiBjb21wbGV4aXR5LCBkaXZlcnNpdHksIGFuZCBkeW5hbWljczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMTEyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBicmlnaHQgc2lkZSBvZiB0cmFuc3Bvc29ucyBpbiBjcm9wIGV2b2x1dGlvbjsgY2l0YXRpb25fdGV4dD0uOyBCcmllZi4gRnVuY3QuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIHRoZSB0cmFuc2dlbmljIHRyb3BpY2FsIGZydWl0IHRyZWUgcGFwYXlhIChDYXJpY2EgcGFwYXlhIExpbm5hZXVzKTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MjsgIGNpdGF0aW9uX3BhZ2VzPTk5MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3JhcGV2aW5lIGdlbm9tZSBzZXF1ZW5jZSBzdWdnZXN0cyBhbmNlc3RyYWwgaGV4YXBsb2lkaXphdGlvbiBpbiBtYWpvciBhbmdpb3NwZXJtIHBoeWxhOyBjaXRhdGlvbl9hdXRob3I9VGhlIEZyZW5jaC1JdGFsaWFuIFB1YmxpYyBDb25zb3J0aXVtIGZvciBHcmFwZSBHZW5vbWUgQ2hhcmFjdGVyaXphdGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0OTsgIGNpdGF0aW9uX3BhZ2VzPTQ2My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBjZW50cm9tZXJpYyByZXRyb3RyYW5zcG9zb25zOiBhIHN0cnVjdHVyYWwgYW5kIGN5dG9nZW5ldGljIHBlcnNwZWN0aXZlOyBNb2JpbGUgRE5BOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzZXMgb2YgTFRSLXJldHJvdHJhbnNwb3NvbiBzdHJ1Y3R1cmVzIHJldmVhbCByZWNlbnQgYW5kIHJhcGlkIGdlbm9taWMgRE5BIGxvc3MgaW4gcmljZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGR5bmFtaWNzIG9mIGFuIGFuY2llbnQgcmV0cm90cmFuc3Bvc29uIGZhbWlseSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGV2b2x1dGlvbiBvZiBnZW5vbWUgc2l6ZSBpbiB0aGUgZ2VudXMgT3J5emE7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MzQyLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgZ2Vub21lIHNpemUgYW5kIGNvbXBsZXhpdHkgaW4gUGludXM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lNDMzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRyYW5zcG9zb25zIGFuZCByZXBlYXQgY29tcG9zaXRpb24gb2YgdGhlIHN1bmZsb3dlciAoSGVsaWFudGh1cyBhbm51dXMgTC4pIGdlbm9tZTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz00OTEtNTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVwZXRpdGl2ZSBjb21wb25lbnQgb2YgdGhlIHN1bmZsb3dlciBnZW5vbWUgYXMgcmV2ZWFsZWQgYnkgZGlmZmVyZW50IHByb2NlZHVyZXMgZm9yIGFzc2VtYmxpbmcgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgcmVhZHM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTY4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc2luZyB0aGUgZXh0ZW50IG9mIHN1YnN0aXR1dGlvbiByYXRlIHZhcmlhdGlvbiBvZiByZXRyb3RyYW5zcG9zb24gbG9uZyB0ZXJtaW5hbCByZXBlYXQgc2VxdWVuY2VzIGluIE9yeXphIHNhdGl2YSBhbmQgT3J5emEgZ2xhYmVycmltYTsgUmljZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjQyLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9saWZlcmF0aW9uIG9mIFR5My9neXBzeS1saWtlIHJldHJvdHJhbnNwb3NvbnMgaW4gaHlicmlkIHN1bmZsb3dlciB0YXhhIGluZmVycmVkIGZyb20gcGh5bG9nZW5ldGljIGRhdGE7IEJNQyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhYmlsaXR5IGluIHN1bu+sgm93ZXIgKEhlbGlhbnRodXMgYW5udXVzIEwuKSBhbmQgaW4gdGhlIEhlbGlhbnRodXMgZ2VudXMgYXMgYXNzZXNzZWQgYnkgcmV0cm90cmFuc3Bvc29uLWJhc2VkIG1vbGVjdWxhciBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExOTsgIGNpdGF0aW9uX3BhZ2VzPTEwMjctMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgc2VsZWN0aW9uIG9uIGdlbmUgZnVuY3Rpb24gZHJpdmVzIHRoZSBldm9sdXRpb24gb2YgTFRSIHJldHJvdHJhbnNwb3NvbiBmYW1pbGllcyBpbiB0aGUgcmljZSBnZW5vbWU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MjQzLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIHJlZ3VsYXRpb24gb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIHBsYW50czsgQW5uLiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9NDMtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4Y2VwdGlvbmFsIGRpdmVyc2l0eSwgbm9uLXJhbmRvbSBkaXN0cmlidXRpb24sIGFuZCByYXBpZCBldm9sdXRpb24gb2YgcmV0cm9lbGVtZW50cyBpbiB0aGUgQjczIG1haXplIGdlbm9tZTsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwNzMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGVjb3N5c3RlbSBhbmQgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIHNwZWNpZXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzkwOyAgY2l0YXRpb25fcGFnZXM9MjE0LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljcyBvZiB0cmFuc3Bvc2FibGUgZWxlbWVudHM6IHRvd2FyZHMgYSBjb21tdW5pdHkgZWNvbG9neSBvZiB0aGUgZ2Vub21lOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvMS85MS8xMjEwNzYxL2RzdTA0Mi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSW1wcm92ZWQga25vd2xlZGdlIG9mIGdlbm9tZSBjb21wb3NpdGlvbiwgZXNwZWNpYWxseSBvZiBpdHMgcmVwZXRpdGl2ZSBjb21wb25lbnQsIGdlbmVyYXRlcyBpbXBvcnRhbnQgaW5mb3JtYXRpb24gZm9yIGJvdGggdGhlb3JldGljYWwgYW5kIGFwcGxpZWQgcmUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ0MTczMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGQspZ303hBqiHwmaU+4jTgOJZSV1g==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f91%2f441732" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f91%2f441732" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1509238984&amp;Signature=d9axHzGQ2c6JH3VivPOEwf4o76NgTBYVw-GkY-dtGboPCmDjaDNrkVwKDCKuOx1X7H4aE9y6yK~ZPahj32RHSnIjCC4DSh2CRKcMnx5vflZgpRVF52vm0kb9YXcgKe6tu3uawJSxqzSYUX9Uk-snB~EpBtAg3pYlriBsHQQ6QNWTqYsbl0gdDynfppRm6istSW4sHia5wR3b8ZemojV-iKB4gu6v0XdAPg53c4hR1~uj6pOLCGzGkT~yDPgMPKFWVZN88oQUevZOF5ZKzQnbMPldszxJy65CbaerXFKTDNpHvmlS8K7zSGpsEjH2wmJqQPQoruOt940QdtLMvc3rVw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6596721">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6596721">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6596725">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6596725">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6596753">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6596753">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6596821">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6596821">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6596972">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6596972">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6597009">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6597009">5. Data retrieval</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6597011">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6597011">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6597013">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6597013">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6597015">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6597015">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6597017">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6597017">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/79/440782">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/101/442161">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">LTR retrotransposon dynamics in the evolution of the olive (<em>Olea europaea</em>) genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Elena Barghini</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elena Barghini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Agricultural, Food, and Environmental Sciences</div>, <div class="insititution">University of Pisa</div>, <div class="addr-line">Pisa I-56124</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Elena+Barghini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Barghini E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Barghini author:E.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Lucia Natali</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lucia Natali</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Agricultural, Food, and Environmental Sciences</div>, <div class="insititution">University of Pisa</div>, <div class="addr-line">Pisa I-56124</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Lucia+Natali">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Natali L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Natali author:L.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Tommaso Giordani</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tommaso Giordani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Agricultural, Food, and Environmental Sciences</div>, <div class="insititution">University of Pisa</div>, <div class="addr-line">Pisa I-56124</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Tommaso+Giordani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Giordani T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Giordani author:T.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Rosa Maria Cossu</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rosa Maria Cossu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Agricultural, Food, and Environmental Sciences</div>, <div class="insititution">University of Pisa</div>, <div class="addr-line">Pisa I-56124</div>, <div class="country">Italy</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Institute of Life Sciences</div>, <div class="insititution">Scuola Superiore Sant'Anna</div>, <div class="addr-line">Pisa</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Rosa+Maria+Cossu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cossu R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Cossu author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Simone Scalabrin</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Simone Scalabrin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">IGA Technology Service</div>, <div class="addr-line">Udine</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Simone+Scalabrin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Scalabrin S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Scalabrin author:S.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Federica Cattonaro</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Federica Cattonaro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">IGA Technology Service</div>, <div class="addr-line">Udine</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Federica+Cattonaro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cattonaro F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Cattonaro author:F.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Hana Šimková</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hana Šimkov&#225;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research</div>, <div class="addr-line">Olomouc</div>, <div class="country">Czech Republic</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Hana+%c5%a0imkov%c3%a1">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Šimkov&#225; H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Šimkov&#225; author:H.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Jan Vrána</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jan Vr&#225;na</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research</div>, <div class="addr-line">Olomouc</div>, <div class="country">Czech Republic</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jan+Vr%c3%a1na">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vr&#225;na J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Vr&#225;na author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Jaroslav Doležel</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jaroslav Doležel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Institute of Experimental Botany, Centre of the Region Haná for Biotechnological and Agricultural Research</div>, <div class="addr-line">Olomouc</div>, <div class="country">Czech Republic</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jaroslav+Dole%c5%beel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Doležel J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Doležel author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                    <span class="al-author-name">

                                        <a class="linked-name" href="javascript:;">Michele Morgante</a>
                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michele Morgante</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Crop and Environmental Sciences</div>, <div class="insititution">University of Udine</div>, <div class="addr-line">Udine</div>, <div class="country">Italy</div></div><div class="aff"><div class="label -label">6</div><div class="insititution">Institute of Applied Genomics</div>, <div class="addr-line">Udine</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Michele+Morgante">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morgante M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Morgante author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name">

                                            <a class="linked-name" href="javascript:;">Andrea Cavallini</a>
                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrea Cavallini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Agricultural, Food, and Environmental Sciences</div>, <div class="insititution">University of Pisa</div>, <div class="addr-line">Pisa I-56124</div>, <div class="country">Italy</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +39 050-2216-665. Fax. +39 050-2216-661. E-mail: <a href="mailto:andrea.cavallini@unipi.it" target="_blank">andrea.cavallini@unipi.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Andrea+Cavallini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cavallini A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Cavallini author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors equally contributed to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 91–100, <a href='https://doi.org/10.1093/dnares/dsu042'>https://doi.org/10.1093/dnares/dsu042</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 July 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="441732" href="/dnaresearch/article-pdf/22/1/91/1210761/dsu042.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Elena Barghini, Lucia Natali, Tommaso Giordani, Rosa Maria Cossu, Simone Scalabrin, Federica Cattonaro, Hana Šimková, Jan Vrána, Jaroslav Doležel, Michele Morgante, Andrea Cavallini;  LTR retrotransposon dynamics in the evolution of the olive (<em>Olea europaea</em>) genome, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 91–100, <a href='https://doi.org/10.1093/dnares/dsu042'>https://doi.org/10.1093/dnares/dsu042</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/441732?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/441732?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/441732?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/441732?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/441732?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/441732?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/441732?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=LTR%20retrotransposon%20dynamics%20in%20the%20evolution%20of%20the%20olive%20(%26lt%3Bem%26gt%3BOlea%20europaea%26lt%3B%2Fem%26gt%3B)%20genome&amp;publicationDate=2014-11-26&amp;volumeNum=22&amp;issueNum=1&amp;author=Barghini%2C%20Elena%3B%20Natali%2C%20Lucia&amp;startPage=91&amp;endPage=100&amp;contentId=10.1093%2Fdnares%2Fdsu042&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f91%2f441732" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6596721 id="6596721" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Improved knowledge of genome composition, especially of its repetitive component, generates important information for both theoretical and applied research. The olive repetitive component is made up of two main classes of sequences: tandem repeats and retrotransposons (REs). In this study, we provide characterization of a sample of 254 unique full-length long terminal repeat (LTR) REs. In the sample, Ty1-<em>Copia</em> elements were more numerous than Ty3-<em>Gypsy</em> elements. Mapping a large set of Illumina whole-genome shotgun reads onto the identified retroelement set revealed that <em>Gypsy</em> elements are more redundant than <em>Copia</em> elements. The insertion time of intact retroelements was estimated based on sister LTR’s divergence. Although some elements inserted relatively recently, the mean insertion age of the isolated retroelements is around 18 million yrs. <em>Gypsy</em> and <em>Copia</em> retroelements showed different waves of transposition, with <em>Gypsy</em> elements especially active between 10 and 25 million yrs ago and nearly inactive in the last 7 million yrs. The occurrence of numerous solo-LTRs related to isolated full-length retroelements was ascertained for two <em>Gypsy</em> elements and one <em>Copia</em> element. Overall, the results reported in this study show that RE activity (both retrotransposition and DNA loss) has impacted the olive genome structure in more ancient times than in other angiosperms.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;LTR retrotransposons&quot;">LTR retrotransposons</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next-generation sequencing&quot;">next-generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="olive">olive</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;insertion age&quot;">insertion age</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;BAC sequencing&quot;">BAC sequencing</a></div>                            <h2 scrollto-destination=6596725 id="6596725" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The cultivation of olive trees (<em>Olea europaea</em> L.) dates back to ancient times. Mythology ascribes the domestication of this species to a divine will: the cultivation of the tree and the treatment of the drupe were taught to the people of Athens by the goddess Athena. In recent years, olive cultivation has been subjected to a growing interest because of the economic, cultural, and ecological importance of olive trees in the Mediterranean area. This interest has grown even outside the Mediterranean area because of health properties of olive oil, related to its composition of fatty acids and secondary metabolites.<sup><a reveal-id="DSU042C1" data-open="DSU042C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p><p>From a genetic point of view, this renewed interest in olive tree cultivation has primarily resulted in data on the large genetic variability occurring within <em>O. europaea</em> and related species.<sup><a reveal-id="DSU042C2" data-open="DSU042C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Such a large genetic variability can be explained by the prevalent self-incompatibility of olive genotypes, which results in high levels of heterozygosis and DNA polymorphisms. Owing to spontaneous crossings and to the appearance of bud sports over the course of millennia, numerous new genotypes appeared, the best of which were fixed by agamic propagation.<sup><a reveal-id="DSU042C3" data-open="DSU042C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p><em>Olea europaea</em> L. has a medium-sized haploid genome of 1.4 Gb,<sup><a reveal-id="DSU042C4" data-open="DSU042C4" class="link link-ref link-reveal xref-bibr">4</a></sup> whose structure has been long uncharacterized. Concerning the repetitive component of the genome, before 2014 only four tandem repeat sequences had been characterized and localized by cytological hybridization on chromosomes.<sup><a reveal-id="DSU042C5  DSU042C7" data-open="DSU042C5  DSU042C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> In addition, a few putative retrotransposon (RE) fragments were isolated and sequenced.<sup><a reveal-id="DSU042C8" data-open="DSU042C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU042C9" data-open="DSU042C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>After sequencing the whole genome of some plant species, the knowledge of the structure and organization of plant genomes has been substantially improved, leading to the view that genome evolution of angiosperms has been accompanied—and possibly promoted—by polyploidization events and differential amplification of repetitive DNA.<sup><a reveal-id="DSU042C10" data-open="DSU042C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>The repetitive DNA in plants is mainly represented by Class I transposons (REs) that are capable of replicating through a ‘copy and paste’ mechanism and can potentially increase the genome size of their host species in a very short time.<sup><a reveal-id="DSU042C11" data-open="DSU042C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>Among REs, elements that contain direct long terminal repeats (LTRs) are predominant in plants. LTR-REs vary in size from a few hundred base pairs to over 10 kb, with LTRs that usually contain the promoter and RNA processing signals.<sup><a reveal-id="DSU042C12" data-open="DSU042C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Internal to the 5′ and 3′ LTRs, respectively, are the primer-binding site (PBS) and the polypurine tract (PPT), which provide the signals for reverse transcription of RE transcripts into the cDNA that will be reintegrated into the genome. In autonomous elements, these two sequence sites flank the region that contains ORFs for Gag, a structural protein of the virus-like particles, and for Pol. Pol encodes a polyprotein with protease, reverse transcriptase (RT), RNaseH, and integrase enzyme domains, which are required for the replication and the integration of the elements in the host chromosomes.</p><p>LTR-REs belong to two major superfamilies, called <em>Gypsy</em> and <em>Copia</em>, differing in the position of the integrase domain within the encoded polyprotein.</p><p>The occurrence of different RE families, characterized by sequence variability in both the coding, transcribed portion and the LTRs<sup><a reveal-id="DSU042C13" data-open="DSU042C13" class="link link-ref link-reveal xref-bibr">13</a></sup> has been reported in several genomes. These families were probably generated by the replicative mechanism of LTR-REs, coupled with the error-prone nature of transcription and reverse transcription. RE families have amplified differentially in different lineages within plant genera or even within a single species (for example, in maize) over a time-span of &lt;1 million yrs (MY).<sup><a reveal-id="DSU042C14" data-open="DSU042C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Similar events have taken place in several cereal species: for example, in the genome of <em>Sorghum bicolor</em>, the insertion of transposable elements and their removal by unequal recombination or by DNA loss resulted in an average RE insertion age of 0.8 MY, with 50% of the detected elements having inserted within the last 500,000 yrs.<sup><a reveal-id="DSU042C15" data-open="DSU042C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Such processes have also been detected in some dicots, although to a less dramatic extent.<sup><a reveal-id="DSU042C11" data-open="DSU042C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU042C16" data-open="DSU042C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU042C17" data-open="DSU042C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>Recently, next-generation sequencing technologies and different computational procedures have been used to gain a general insight into the composition of the olive genome and its repetitive fraction.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Illumina and 454 reads from genomic DNA were assembled following different procedures, obtaining &gt;200,000 differently redundant contigs, with a mean length of &gt;1,000 nt. By combining identification and mapping of repeated sequences, it was established that tandem repeats represent a very large portion of the olive genome (∼31%), consisting of six main families of different length. The other large redundant class in the olive genome is represented by transposable elements, especially LTR-REs.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>The identification and characterization of olive LTR-REs were, however, difficult because of the lack of large sequenced genomic regions. For example, an accurate dating of amplification events of the LTR-RE component requires a comparison of the two LTR sequences from individual, full-length elements.<sup><a reveal-id="DSU042C19" data-open="DSU042C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>In the frame of a project aimed to sequence the olive genome, a bacterial artificial chromosome (BAC) library was produced. A number of BAC clones were pooled and then sequenced using the Illumina procedure. These sequences were assembled and the resulting contigs analysed to identify full-length LTR-REs, allowing the first characterization of such elements in olive, especially in relation to their insertion age.</p>                            <h2 scrollto-destination=6596753 id="6596753" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6596759 id="6596759" class="section-title" data-legacy-id=s2a>2.1. Sequencing of BAC clones</h3>
<p>A BAC library from <em>O. europaea</em> cv. Leccino was produced as follows. Cell nuclei were isolated from the youngest leaves of the olive tree following the protocol of Doležel <em>et al.</em><sup><a reveal-id="DSU042C20" data-open="DSU042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Briefly, the leaves were fixed for 20 min at 5°C in 2% (v/v) formaldehyde and immediately afterwards chopped by a razor blade in ice-cold isolation buffer (15 mM Tris, 10 mM EDTA, 130 mM KCl, 20 mM NaCl, 1 mM spermine, 1 mM spermidine, 45 mM β-mercaptoethanol, and 0.1% Triton X-100, pH 9.4). The suspension of released nuclei was passed through a 50-µm-pore nylon mesh to remove large tissue and cellular fragments, and was then stained using DAPI (2 µg/ml). Intact nuclei were sorted using flow cytometry and used to prepare high-molecular-weight (HMW) DNA.<sup><a reveal-id="DSU042C21" data-open="DSU042C21" class="link link-ref link-reveal xref-bibr">21</a></sup> HMW DNA of 1.8 million nuclei (∼5.4 µg DNA) was used to construct a large insert library cloned in pIndigoBAC-5 vector (Epicentre, Madison, WI, USA) as previously described.<sup><a reveal-id="DSU042C22" data-open="DSU042C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>We shotgun-sequenced 12 pools each formed by 384 BACs (corresponding to 12,384-well library plates) for a total of 4,608 clones using the Illumina procedure. DNA from each plate pool was prepared using the Illustra TempliPhi Large Construct V2 kit (Resnova). Each BAC pool was then individually assembled as described below.</p><p>Paired-end libraries were prepared by using the Nextera DNA Sample Prep Kit (Illumina, Inc., San Diego, CA, USA) and sequenced in two lanes on an Illumina HiSeq2000 at the 12-plex level of multiplexing, producing from 30 to 50 million reads per pool (paired-end 100-nt set-up). The processing of fluorescent images into sequences, base-calling, and quality value calculations was performed using the Illumina data-processing pipeline (version 1.8.2).</p><p>Illumina reads were then processed to remove adapters using Cutadapt (<a class="link link-uri" href="https://pypi.python.org/pypi/cutadapt/1.4.2">https://pypi.python.org/pypi/cutadapt/1.4.2</a>, 12 November 2014, date last accessed),<sup><a reveal-id="DSU042C23" data-open="DSU042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> with default parameters except -O 10 -n 2 -m 50 and finally paired again using an internally developed Python script. To trim low-quality regions and to remove bacterial contaminants and olive chloroplast sequence, reads were further processed with ERNE-FILTER (<a class="link link-uri" href="http://erne.sourceforge.net">http://erne.sourceforge.net</a>, 12 November 2014, date last accessed) using default parameters except–min-size 50 and —errors-rate 25.</p><p>Illumina reads from each individual BAC pool were assembled using ABySS<sup><a reveal-id="DSU042C24" data-open="DSU042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> with the following parameters: <em>k</em> = 71, l = 1, aligner = map, <em>b</em> = 1,000,000, <em>P</em> = 0.95, <em>s</em> = 500, <em>n</em> = 10. An internally developed Perl script was used to remove scaffolds shorter than 500 nt.</p>                            <h3 scrollto-destination=6596780 id="6596780" class="section-title" data-legacy-id=s2b>2.2. Identification of full-length LTR-REs</h3>
<p>Assembled contigs longer than 10,000 nt were surveyed for the identification of full-length LTR-REs based on structural features and sequence similarity to the <em>Olea</em> RE database<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and to public sequence databases (non-redundant nucleotide and protein NCBI databases, RepBase database).</p><p>Structural features were identified using the LTR-FINDER<sup><a reveal-id="DSU042C25" data-open="DSU042C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and DOTTER<sup><a reveal-id="DSU042C26" data-open="DSU042C26" class="link link-ref link-reveal xref-bibr">26</a></sup> software. Alignment boundaries were obtained by adjusting the ends of LTR-pair candidates using the Smith–Waterman algorithm. These boundaries were re-adjusted based on the occurrence of typical LTR-RE features that include the following: being flanked by the dinucleotides TG and CA at 5′ and 3′ ends, respectively; the presence of a target-site duplication (TSD) of 4–6 nt; a putative 15- to 18-nt PBS, complementary to a tRNA at the end of the putative 5′-LTR; and a 20- to 25-nt PPT just upstream of the 5′ end of the 3′ LTR.</p><p>All putative LTR-REs were subsequently annotated by BLASTX and BLASTN against the public non-redundant databases at NCBI, and by RepeatMasker against the RepeatExplorer-based database of olive-repeated sequences.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> To limit false-positive detection, we used a fixed <em>E</em>-value threshold of <em>E</em> &lt; 10<sup>−5</sup> for BLASTN and <em>E</em> &lt; 10<sup>−10</sup> for BLASTX. The full-length REs that were identified as belonging to <em>Gypsy</em> or <em>Copia</em> superfamilies were then used as a reference database for a further BLASTN search, in order to classify previously unclassified elements.</p><p>In other analyses, 10,000 nt upstream and downstream of each LTR-RE was subjected, whenever possible, to a BLAST search to identify other sequences (coding and/or non-coding) occurring in proximity of each RE.</p>                            <h3 scrollto-destination=6596793 id="6596793" class="section-title" data-legacy-id=s2c>2.3. Estimation of LTR-RE abundance</h3>
<p>To estimate the redundancy of the LTR-RE set and of the <em>Gypsy</em> and <em>Copia</em> superfamilies, a large set of Illumina whole-genome shotgun reads (total coverage 8.1×)<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> was mapped onto all isolated elements, using CLC-BIO Genomic Workbench 6.5.1, with the following parameters: mismatch cost = 1, deletion cost = 1, insertion cost = 1, similarity = 0.9, and length fraction = 0.9. To obtain reads of constant length, all bases exceeding 75 nt were cut. In this analysis, multireads (i.e. those reads that matched multiple distinct sequences) were distributed randomly, and hence, the number of mapped reads to a single sequence would be only an indication of its redundancy. On the other hand, if all sequences of a sequence class are taken together, the total number of mapped reads (in respect to total genomic reads) reveals the effective redundancy of that class.</p><p>The redundancy level of each single sequence was estimated by mapping the same large set of Illumina whole-genome shotgun reads as above onto each isolated RE, one by one. Redundancy values are reported as the number of mapped reads per kb of sequence length.</p><p>In other analyses, in order to evaluate the occurrence of solo-LTRs, the same read set was mapped onto each isolated RE one by one, keeping the 5′-LTR and the inter-LTR region separated.</p>                            <h3 scrollto-destination=6596806 id="6596806" class="section-title" data-legacy-id=s2d>2.4. Phylogenetic analyses</h3>
<p>Full-length LTR-REs were scored for retrotranscriptase, RNAseH, and integrase domains of <em>Gypsy</em> and <em>Copia</em> elements—separately—using TBLASTN against an internally developed library of RE proteins. All sequences of at least 80 amino acids were collected and aligned using CLUSTALW,<sup><a reveal-id="DSU042C27" data-open="DSU042C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and a tree was generated. The tree was visualized using FigTree (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>, 12 November 2014, date last accessed).</p>                            <h3 scrollto-destination=6596812 id="6596812" class="section-title" data-legacy-id=s2e>2.5. Insertion age calculation of full-length LTR-REs</h3>
<p>RE insertion age was estimated comparing the 5′- and 3′-LTRs of each putative RE. The two LTRs of a single RE are identical at the time of insertion because they are copied from the same template.</p><p>A synonymous substitution rate was calculated comparing 20 protein-coding sequences (longer than 200 nt, putatively nuclear, and unique) of another species belonging to the <em>Oleaceae</em> family, <em>Fraxinus excelsior</em> (The British Ash Tree Genome Project, <a class="link link-uri" href="http://www.ashgenome.org/">http://www.ashgenome.org/</a>, 12 November 2014, date last accessed) to orthologous sequences of <em>O. europaea</em>, selected from an available olive transcriptome.<sup><a reveal-id="DSU042C28" data-open="DSU042C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Rates of synonymous nucleotide substitution for each gene sequence were calculated by the method of Nei and Gojobori<sup><a reveal-id="DSU042C29" data-open="DSU042C29" class="link link-ref link-reveal xref-bibr">29</a></sup> with the Jukes–Cantor correction as implemented in the DnaSP program.<sup><a reveal-id="DSU042C30" data-open="DSU042C30" class="link link-ref link-reveal xref-bibr">30</a></sup> As the estimated separation between <em>Olea</em> and <em>Fraxinus</em> is dated to between 40 and 45 MY ago,<sup><a reveal-id="DSU042C31" data-open="DSU042C31" class="link link-ref link-reveal xref-bibr">31</a></sup> 42.5 MY was used for estimating the synonymous nucleotide substitution rate.</p><p>The two LTRs of each full-length RE were aligned with the ClustalW software,<sup><a reveal-id="DSU042C26" data-open="DSU042C26" class="link link-ref link-reveal xref-bibr">26</a></sup> indels were eliminated, and the number of synonymous nucleotide substitutions per site was calculated using an internally developed pipeline. As proposed by Ma and Bennetzen,<sup><a reveal-id="DSU042C32" data-open="DSU042C32" class="link link-ref link-reveal xref-bibr">32</a></sup> we used twice the mean number of synonymous substitutions per site per year as the nucleotide substitution rate between LTRs. Based on this rate, the insertion time for each full-length RE was estimated.</p>                            <h2 scrollto-destination=6596821 id="6596821" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6596824 id="6596824" class="section-title" data-legacy-id=s3a>3.1. Isolation and annotation of full-length LTR-REs</h3>
<p>The BAC library consisted of 44,928 clones with an average insert size of 112 kb. Considering the 1C genome size of <em>O. europaea</em> cv. Leccino (1.49 Gb, determined by DNA flow cytometry; J. Čížková, personal communication), the library should represent 3.3 genome equivalents.</p><p>We sequenced a total of 4,608 clones in pools of 384 and, after assembly of each individual pool, we selected all contigs longer than 10 kb (275, for a total of 7,653,690 nt). These were scored for the occurrence of putative full-length LTR-REs, searching for structural features and sequence similarities, that is, the occurrence of two relatively intact LTRs, of identified PPT and PBS sites, and of flanking TSDs.</p><p>A set of 254 putative full-length REs was isolated from 245 over 275 contigs (1 RE from each of 236 contigs and 2 REs from each of 9 contigs). We defined full-length elements based on the occurrence of intact ends, irrespective of whether these elements were potentially functional or contained inactivating mutations in their internal sequence (Table <a reveal-id="DSU042TB1" data-open="DSU042TB1" class="link link-reveal link-table xref-fig">1</a>). Many sequences showed all distinctive structural features of REs, although in some cases one feature was missing. </p>                        <a id="6596839" scrollto-destination="6596839"></a>
<div class="table-wrap"><div id="DSU042TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Mean characteristics of <em>Copia</em>, <em>Gypsy</em>, and unknown putative full-length retroelements identified in the BAC clones</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Superfamily </th><th>No. of REs </th><th>RE length (nt) ± SE </th><th>5′ LTR length (nt) ± SE </th><th>3′ LTR length (nt) ± SE </th><th>Mean number of mapped reads per 1,000 nt ± SE </th><th>Putative insertion age (MY) ± SE </th></tr></thead><tbody><tr><td><em>Copia</em> </td><td>166 </td><td>5,605.0 ± 151.9 </td><td>564.7 ± 41.0 </td><td>540.1 ± 28.1 </td><td>6,322.3 ± 708.7 </td><td>17.13 ± 0.72 </td></tr><tr><td><em>Gypsy</em> </td><td>81 </td><td>7,632.0 ± 348.3 </td><td>643.2 ± 45.5 </td><td>646.1 ± 46.4 </td><td>8,749.6 ± 1,049.0 </td><td>19.34 ± 0.77 </td></tr><tr><td>Unknown </td><td>7 </td><td>4,705.0 ± 2,764.0 </td><td>464.4 ± 170.3 </td><td>458.4 ± 164.8 </td><td>1,477.0 ± 965.0 </td><td>21.10 ± 3.39 </td></tr></tbody></table></div><div content-id="DSU042TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Superfamily </th><th>No. of REs </th><th>RE length (nt) ± SE </th><th>5′ LTR length (nt) ± SE </th><th>3′ LTR length (nt) ± SE </th><th>Mean number of mapped reads per 1,000 nt ± SE </th><th>Putative insertion age (MY) ± SE </th></tr></thead><tbody><tr><td><em>Copia</em> </td><td>166 </td><td>5,605.0 ± 151.9 </td><td>564.7 ± 41.0 </td><td>540.1 ± 28.1 </td><td>6,322.3 ± 708.7 </td><td>17.13 ± 0.72 </td></tr><tr><td><em>Gypsy</em> </td><td>81 </td><td>7,632.0 ± 348.3 </td><td>643.2 ± 45.5 </td><td>646.1 ± 46.4 </td><td>8,749.6 ± 1,049.0 </td><td>19.34 ± 0.77 </td></tr><tr><td>Unknown </td><td>7 </td><td>4,705.0 ± 2,764.0 </td><td>464.4 ± 170.3 </td><td>458.4 ± 164.8 </td><td>1,477.0 ± 965.0 </td><td>21.10 ± 3.39 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6596839" target="_blank">View Large</a></div></div><p>The isolated REs covered a total of 1,584,566 nt over 7,653,690 nt, with a mean length of 6226.5 nt. They were classified as <em>Gypsy</em> or <em>Copia</em> according to BLAST searches against NCBI, RepBase,<sup><a reveal-id="DSU042C33" data-open="DSU042C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and OLEAREP<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> databases and to a subsequent BLAST search using the same olive REs as a reference database.</p><p>The majority of isolated full-length REs belonged to the <em>Copia</em> superfamily (166), followed by the <em>Gypsy</em> superfamily (81, of which 36 contained an integrase chromodomain<sup><a reveal-id="DSU042C34" data-open="DSU042C34" class="link link-ref link-reveal xref-bibr">34</a></sup>). Seven REs were classified as unknown because they lacked distinctive protein-coding sequences suitable for classifying the element. For 222, 171, and 253 elements, the putative TSD, PBS, and PPT were identified, respectively. Seventy-two elements showed all typical protein domains of LTR-REs. The main features of each isolated RE are reported in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data S1</a>.</p><p>Sequences adjacent to isolated REs were also scored for similarity to other genic or non-genic sequences. The occurrence of putative genes and transposon-related sequences is summarized in Table <a reveal-id="DSU042TB2" data-open="DSU042TB2" class="link link-reveal link-table xref-fig">2</a>. In many cases (99 REs, 38.93%), the isolated full-length REs lied in proximity of gene sequences. Ninety-five REs (37.40%) were close to REs or DNA transposon fragments. Only 16 REs (6.30%) were flanked at both sides by transposon fragments, possibly representing loci with nested elements. </p>                        <a id="6596853" scrollto-destination="6596853"></a>
<div class="table-wrap"><div id="DSU042TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Occurrence of sequences belonging to genes and/or transposable elements (REs and DNA transposons) in the upstream and/or downstream regions of isolated full-length REs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Upstream sequence </th><th>Downstream sequence </th><th>No. of full-length REs </th></tr></thead><tbody><tr><td>Gene </td><td>Gene </td><td>13 (5.12%) </td></tr><tr><td>Gene </td><td>Unclassified </td><td>41 (16.14%) </td></tr><tr><td>Unclassified </td><td>Gene </td><td>27 (10.63%) </td></tr><tr><td>Gene </td><td>Transposon </td><td>5 (1.97%) </td></tr><tr><td>Transposon </td><td>Gene </td><td>13 (5.12%) </td></tr><tr><td>Transposon </td><td>Unclassified </td><td>36 (14.17%) </td></tr><tr><td>Unclassified </td><td>Transposon </td><td>25 (9.84%) </td></tr><tr><td>Transposon </td><td>Transposon </td><td>16 (6.30%) </td></tr><tr><td>Unclassified </td><td>Unclassified </td><td>78 (30.71%) </td></tr></tbody></table></div><div content-id="DSU042TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Upstream sequence </th><th>Downstream sequence </th><th>No. of full-length REs </th></tr></thead><tbody><tr><td>Gene </td><td>Gene </td><td>13 (5.12%) </td></tr><tr><td>Gene </td><td>Unclassified </td><td>41 (16.14%) </td></tr><tr><td>Unclassified </td><td>Gene </td><td>27 (10.63%) </td></tr><tr><td>Gene </td><td>Transposon </td><td>5 (1.97%) </td></tr><tr><td>Transposon </td><td>Gene </td><td>13 (5.12%) </td></tr><tr><td>Transposon </td><td>Unclassified </td><td>36 (14.17%) </td></tr><tr><td>Unclassified </td><td>Transposon </td><td>25 (9.84%) </td></tr><tr><td>Transposon </td><td>Transposon </td><td>16 (6.30%) </td></tr><tr><td>Unclassified </td><td>Unclassified </td><td>78 (30.71%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6596853" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6596857 id="6596857" class="section-title" data-legacy-id=s3b>3.2. Genomic redundancy of isolated LTR-REs</h3>
<p>Considering the percentage of Illumina reads that matched to a class of sequences as an indicator of the proportion of that class in the olive genome, previous experiments<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> indicated that LTR-REs account for 38.8% of the olive genome. Using the same set of reads for mapping the 254 REs isolated from BAC clones, we found that they were mapped by 17,107,830 reads, corresponding to 12.3% of the genome, that is, these REs represent around one-third of the RE population in the genome.</p><p>Mapping results of the different RE superfamilies are summarized in Fig. <a reveal-id="DSU042F1" data-open="DSU042F1" class="link link-reveal link-table xref-fig">1</a>. In a previous work on olive genome,<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> the ratio between redundancy of <em>Gypsy</em> and <em>Copia</em> REs was 1.17. In the RE sample described here, the ratio is quite different, amounting to 0.92, indicating that the REs identified in this study do not represent the whole olive RE set. However, it is worth noting that, although <em>Gypsy</em> full-length elements are only 81 versus 166 <em>Copia</em> elements, they are mapped by a number of reads similar to that mapping to the <em>Copia</em> REs. This result confirms that, in the olive genome, the number of <em>Gypsy</em> families is lower than that of <em>Copia</em>, but <em>Gypsy</em> REs are more redundant than <em>Copia</em> REs.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                        <a id="6596868" scrollto-destination="6596868"></a>
<div id="DSU042F1" data-id="DSU042F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04201.png?Expires=1509238985&Signature=OM1X9SReQUVs4OMPAOfy9cxwwrJaob9EiaI3rLr1pZx3N9bUzSGEaEFlrY79Ry2tdTB~hqgmlyUMGC5JfztijXhMGxfG0z4Jnhg8Ru33dM~rS9p9xX7GImUIiYdkTL6W4WG8p00SRN86Ijs5k23vRHo1CBmNYAgVun4BhSmYqXM-quqQ6AScvJivHzZcFOQ6A7rmyJw2w7k4UPQWshEvcjvWDgcQA-cQonAF5oGIvB-q576VXTJjW1zoMmudZ3sGZXUWSevh528SC58KCiLt1dhttEux1M9dEvjXrANqMtF636WE-2-sojgrQ0L9ufzNRvJvI1aEHDrXWBnG~4TYsw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Number of full-length REs identified in this study, separated according to their superfamily. Each bar in the histogram shows the number of Illumina reads that matched to all REs (height) and the number of REs (width) of each superfamily." path-from-xml="dsu04201.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596868/dsu04201.tif" path-from-xml="dsu04201.tif" target="_blank">View large</a><a section="6596868" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04201.png?Expires=1509238985&Signature=Uo8vmNsoc4Q6Dp4lITjaHY8BQqFWC3sbVrdEkgFHI18-QJOupM78rS2WYWetpuljGjfkwbS6qqtIU0mEBRvffLzgUO5epwr5jDol2~0IR6ttBhX9WyTCHPkKoS3MNUW2DKTR46ZV1F2zXmf7VSHAWjCL-~BP6gHTu6WIcGotZufTphS79nLr6iY7Q5vAZe5MxW5bUnFgXyE4qqpk7~31W1RAGbYxNI1hWrX9vDJFaTii1KIBImN-9BV6LgcH3qZusFHK1PUSqETuqMNSp4-mqQXG4ctJFxibxxxEG7l~0qMcgUIF7AdjlpTRwqUpe6f-bS7MWyS~8GK3kkUZr89qwg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596868&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04201.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of full-length REs identified in this study, separated according to their superfamily. Each bar in the histogram shows the number of Illumina reads that matched to all REs (height) and the number of REs (width) of each superfamily.</p></div></div><div content-id="DSU042F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04201.png?Expires=1509238985&Signature=OM1X9SReQUVs4OMPAOfy9cxwwrJaob9EiaI3rLr1pZx3N9bUzSGEaEFlrY79Ry2tdTB~hqgmlyUMGC5JfztijXhMGxfG0z4Jnhg8Ru33dM~rS9p9xX7GImUIiYdkTL6W4WG8p00SRN86Ijs5k23vRHo1CBmNYAgVun4BhSmYqXM-quqQ6AScvJivHzZcFOQ6A7rmyJw2w7k4UPQWshEvcjvWDgcQA-cQonAF5oGIvB-q576VXTJjW1zoMmudZ3sGZXUWSevh528SC58KCiLt1dhttEux1M9dEvjXrANqMtF636WE-2-sojgrQ0L9ufzNRvJvI1aEHDrXWBnG~4TYsw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Number of full-length REs identified in this study, separated according to their superfamily. Each bar in the histogram shows the number of Illumina reads that matched to all REs (height) and the number of REs (width) of each superfamily." path-from-xml="dsu04201.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596868/dsu04201.tif" path-from-xml="dsu04201.tif" target="_blank">View large</a><a section="6596868" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04201.png?Expires=1509238985&Signature=Uo8vmNsoc4Q6Dp4lITjaHY8BQqFWC3sbVrdEkgFHI18-QJOupM78rS2WYWetpuljGjfkwbS6qqtIU0mEBRvffLzgUO5epwr5jDol2~0IR6ttBhX9WyTCHPkKoS3MNUW2DKTR46ZV1F2zXmf7VSHAWjCL-~BP6gHTu6WIcGotZufTphS79nLr6iY7Q5vAZe5MxW5bUnFgXyE4qqpk7~31W1RAGbYxNI1hWrX9vDJFaTii1KIBImN-9BV6LgcH3qZusFHK1PUSqETuqMNSp4-mqQXG4ctJFxibxxxEG7l~0qMcgUIF7AdjlpTRwqUpe6f-bS7MWyS~8GK3kkUZr89qwg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596868&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04201.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of full-length REs identified in this study, separated according to their superfamily. Each bar in the histogram shows the number of Illumina reads that matched to all REs (height) and the number of REs (width) of each superfamily.</p></div></div><p>The distribution of the number of mapped reads of isolated <em>Gypsy</em> and <em>Copia</em> REs is reported in Fig. <a reveal-id="DSU042F2" data-open="DSU042F2" class="link link-reveal link-table xref-fig">2</a>. <em>Gypsy</em> elements showed a larger mean and distribution around the mean compared with <em>Copia</em>. </p>                        <a id="6596879" scrollto-destination="6596879"></a>
<div id="DSU042F2" data-id="DSU042F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04202.png?Expires=1509238985&Signature=K0aBZJrpM3J-uR-e1eHSW4f07cspI~eFM~dvwQwP-zflU2~e0gBhpyzGWS-WlEeG~eeW38WpVl85NQFgQ24xQcsuO-wXBRJubViJMrVc2-0nkaelDF2SujqxprFrwmExTXWSN8suCQPDVR4W1QmkmkoWHCoggFHUjwYFQX0sHgJ2kEl~zNAW1Jt5jqDophLACN2tH2oxmgWEIzQ6dOm2oEpkn4ReY0KXe-Poq1lSOpZjaFcICWmztVZ9PA1fmLRRCAWriuQbakP0HojlKGRiwvBxZSNZ~h-Jo8sTkmLRUosdhzwL6IGHFamdJ6X3-zRuYkUw18SBZ~O4VctJR-47pA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Box and whiskers plot of RE redundancy (calculated as the number of mapped reads per kb) of olive Copia and Gypsy REs. The boxes represent the 25–75%, whiskers represent the whole range of values, and lines in the box represent the mean values of the distribution." path-from-xml="dsu04202.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596879/dsu04202.tif" path-from-xml="dsu04202.tif" target="_blank">View large</a><a section="6596879" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04202.png?Expires=1509238985&Signature=TkYTftwv~VccyCtcoZv-Q1VaPVD5qaLF2MkcbAV5nOQkV83UDdfVFxY35bM5QiajGPnEpd9DI6eieIvJZTsp3cAHzrLPAB261QpK5QNNOjrwdNYX2aJXmPq8bgTcaJ1sIzVpFLX1vyPSCn24nc~qs7OAspkpHzTjxdFnEiBSLe-9OotD861s8IyON29wU2HNrWcCfgxCtSd2yHT1P~Go2Qyfod7kCrXkFfJWxb7k6ZNWAleRKCTNiEk05OIMOiyej7-V4S82Bzn-aU7BHyGcMvLhnIWWOdiWUKEGN7Hdv~sgaMtpyarrSoVhaFdJNmtjncfWiB6vVTgiWSFC9Bqe8A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596879&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04202.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Box and whiskers plot of RE redundancy (calculated as the number of mapped reads per kb) of olive <em>Copia</em> and <em>Gypsy</em> REs. The boxes represent the 25–75%, whiskers represent the whole range of values, and lines in the box represent the mean values of the distribution.</p></div></div><div content-id="DSU042F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04202.png?Expires=1509238985&Signature=K0aBZJrpM3J-uR-e1eHSW4f07cspI~eFM~dvwQwP-zflU2~e0gBhpyzGWS-WlEeG~eeW38WpVl85NQFgQ24xQcsuO-wXBRJubViJMrVc2-0nkaelDF2SujqxprFrwmExTXWSN8suCQPDVR4W1QmkmkoWHCoggFHUjwYFQX0sHgJ2kEl~zNAW1Jt5jqDophLACN2tH2oxmgWEIzQ6dOm2oEpkn4ReY0KXe-Poq1lSOpZjaFcICWmztVZ9PA1fmLRRCAWriuQbakP0HojlKGRiwvBxZSNZ~h-Jo8sTkmLRUosdhzwL6IGHFamdJ6X3-zRuYkUw18SBZ~O4VctJR-47pA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Box and whiskers plot of RE redundancy (calculated as the number of mapped reads per kb) of olive Copia and Gypsy REs. The boxes represent the 25–75%, whiskers represent the whole range of values, and lines in the box represent the mean values of the distribution." path-from-xml="dsu04202.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596879/dsu04202.tif" path-from-xml="dsu04202.tif" target="_blank">View large</a><a section="6596879" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04202.png?Expires=1509238985&Signature=TkYTftwv~VccyCtcoZv-Q1VaPVD5qaLF2MkcbAV5nOQkV83UDdfVFxY35bM5QiajGPnEpd9DI6eieIvJZTsp3cAHzrLPAB261QpK5QNNOjrwdNYX2aJXmPq8bgTcaJ1sIzVpFLX1vyPSCn24nc~qs7OAspkpHzTjxdFnEiBSLe-9OotD861s8IyON29wU2HNrWcCfgxCtSd2yHT1P~Go2Qyfod7kCrXkFfJWxb7k6ZNWAleRKCTNiEk05OIMOiyej7-V4S82Bzn-aU7BHyGcMvLhnIWWOdiWUKEGN7Hdv~sgaMtpyarrSoVhaFdJNmtjncfWiB6vVTgiWSFC9Bqe8A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596879&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04202.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Box and whiskers plot of RE redundancy (calculated as the number of mapped reads per kb) of olive <em>Copia</em> and <em>Gypsy</em> REs. The boxes represent the 25–75%, whiskers represent the whole range of values, and lines in the box represent the mean values of the distribution.</p></div></div><p>To estimate the equilibrium between RE replication and RE loss, Illumina reads were mapped onto each full-length LTR-RE, keeping LTR sequences separate from the respective inter-LTR region. The results of this analysis are shown in Fig. <a reveal-id="DSU042F3" data-open="DSU042F3" class="link link-reveal link-table xref-fig">3</a>. </p>                        <a id="6596883" scrollto-destination="6596883"></a>
<div id="DSU042F3" data-id="DSU042F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04203.png?Expires=1509238985&Signature=DInCWzVs4CKCZEsYla1uSsHw~wFNZAtsjFvmv4V9LQWiQ-p0ljYXmGMj2Tl5rqBGB0aHo66Wh1wMevOKmNh1CBY6LejNU7FCNFVm7ny6-fov3e5J-qQCyZy0L~bZy9H6kvfDrhSbKyXW3b1b3orKxL3au~OuIuGGO5FEg~8ohdUmb0hrPpgCk7JCUaLrX3b3hu0No8CA4zRk0rRswOy4YDxu0g7dqkouGtn9A1hw1pofwUZrm8fgFwrI7tvP8YFc68Wficb4Azr8JWCaN6qxh8LDga7SuEEiYanGcH8-8d5u4sXLkR4ReXMZ-gcxKOorm~Q1mIg2vM5l8vZTleFMcA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of full-length olive LTR-REs according to the ratio between the number of mapped reads per kb measured separately on LTR and inter-LTR regions." path-from-xml="dsu04203.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596883/dsu04203.tif" path-from-xml="dsu04203.tif" target="_blank">View large</a><a section="6596883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04203.png?Expires=1509238985&Signature=DgtyxGy1s5OBSN4JJFCmLyLHjT4BSUWRJ9KHpI7zDXStx1avYsZUL2NNGnXq513rfDvlEy05pbpMsAdEfqaIF5yQbkRL6jo7pXXWtUXvREqmpxChBmFOw1GXSTtgH3ACa6VVggDnzjIkD5e4dqIuY80tTTUaLR3YTP3hnGChzSzQsAXs6B6iAQ3cguKf9qo5O7elgAwath853RwWzJm9PFw211BiYDFRrNuNl7FBXUt50I9KHcQSDSfK6Goq862eTL~9gL19n9tBkIoXJrjAcQdqZRSGbX~5D0hUVzvD3Fzwk78QXtv5VMRKkeBmVFSi~yAITf4hQw9DlWdpqX2PUQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596883&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04203.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of full-length olive LTR-REs according to the ratio between the number of mapped reads per kb measured separately on LTR and inter-LTR regions.</p></div></div><div content-id="DSU042F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04203.png?Expires=1509238985&Signature=DInCWzVs4CKCZEsYla1uSsHw~wFNZAtsjFvmv4V9LQWiQ-p0ljYXmGMj2Tl5rqBGB0aHo66Wh1wMevOKmNh1CBY6LejNU7FCNFVm7ny6-fov3e5J-qQCyZy0L~bZy9H6kvfDrhSbKyXW3b1b3orKxL3au~OuIuGGO5FEg~8ohdUmb0hrPpgCk7JCUaLrX3b3hu0No8CA4zRk0rRswOy4YDxu0g7dqkouGtn9A1hw1pofwUZrm8fgFwrI7tvP8YFc68Wficb4Azr8JWCaN6qxh8LDga7SuEEiYanGcH8-8d5u4sXLkR4ReXMZ-gcxKOorm~Q1mIg2vM5l8vZTleFMcA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distribution of full-length olive LTR-REs according to the ratio between the number of mapped reads per kb measured separately on LTR and inter-LTR regions." path-from-xml="dsu04203.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596883/dsu04203.tif" path-from-xml="dsu04203.tif" target="_blank">View large</a><a section="6596883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04203.png?Expires=1509238985&Signature=DgtyxGy1s5OBSN4JJFCmLyLHjT4BSUWRJ9KHpI7zDXStx1avYsZUL2NNGnXq513rfDvlEy05pbpMsAdEfqaIF5yQbkRL6jo7pXXWtUXvREqmpxChBmFOw1GXSTtgH3ACa6VVggDnzjIkD5e4dqIuY80tTTUaLR3YTP3hnGChzSzQsAXs6B6iAQ3cguKf9qo5O7elgAwath853RwWzJm9PFw211BiYDFRrNuNl7FBXUt50I9KHcQSDSfK6Goq862eTL~9gL19n9tBkIoXJrjAcQdqZRSGbX~5D0hUVzvD3Fzwk78QXtv5VMRKkeBmVFSi~yAITf4hQw9DlWdpqX2PUQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596883&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04203.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of full-length olive LTR-REs according to the ratio between the number of mapped reads per kb measured separately on LTR and inter-LTR regions.</p></div></div><p>The ratio between the numbers of mapped reads per kb between the 5′-LTR and the respective inter-LTR DNA sequence ranged from 0.003 to 20.52. If all REs belonging to the same family were intact, that is, composed of two LTRs and one inter-LTR region, the ratio should have been 2. Conservatively, we considered the occurrence of solo-LTRs only for those LTR-REs whose ratio was higher than 2.5. Only a small number of REs showed ratios higher than 2.5 (16 of 254—eight <em>Gypsy</em>, seven <em>Copia</em>, and one unknown element), and only three showed ratios higher than 10.</p><p>Many LTR-REs showed a ratio lower than 2 (Fig. <a reveal-id="DSU042F3" data-open="DSU042F3" class="link link-reveal link-table xref-fig">3</a>), that is, the inter-LTR region was more represented in the genome than in the LTR. This result suggests the presence of different families that share, at least in part, the inter-LTR region and show a higher level of sequence conservation of the <em>pol</em> protein-coding domains. Interestingly, Fig. <a reveal-id="DSU042F3" data-open="DSU042F3" class="link link-reveal link-table xref-fig">3</a> suggests that this aspect is especially true for <em>Copia</em> REs.</p><p>Finally, one <em>Copia</em> and two <em>Gypsy</em> elements showed the highest ratio between the numbers of mapped reads per kb of the LTR and inter-LTRs (Fig. <a reveal-id="DSU042F3" data-open="DSU042F3" class="link link-reveal link-table xref-fig">3</a>), indicating that unequal homologous recombination has particularly affected elements similar to these two REs. The <em>Copia</em> RE shows sequence similarity to <em>Ale</em> retroelements of other species and it is medium redundant. Both <em>Gypsy</em> REs show similarity to chromodomain-containing Res, and the estimated redundancy of the full-length forms of them is very high.</p>                            <h3 scrollto-destination=6596893 id="6596893" class="section-title" data-legacy-id=s3c>3.3. Phylogenetic relationships among isolated LTR-REs</h3>
<p>Phylogenetic analyses were performed by the neighbour-joining method to evaluate the relationship between isolated LTR-REs. Two phylogenetic trees were constructed based on the transcribed putative retrotranscriptase sequence of 93 <em>Copia</em> and 43 <em>Gypsy</em> REs (Figs <a reveal-id="DSU042F4" data-open="DSU042F4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSU042F5" data-open="DSU042F5" class="link link-reveal link-table xref-fig">5</a>). The bootstrap values suggest the occurrence of distinct families, indicated in the figures by different colours. </p>                        <a id="6596900" scrollto-destination="6596900"></a>
<div id="DSU042F4" data-id="DSU042F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04204.png?Expires=1509238985&Signature=aMRhTF6VdbuFYbcqj~leXCZVwFOt~GeoPbxyFirz1Tuuk3vXt3GZRBlZd11CX42aFvXTf2D4lcctH4-698mv8zePi2nrb4T2sR67tklRyfB4H~Ud0GN3dNXBdHuYN8HViGxGZCYNHZuN3w8AtHEuk8wShFYI8-AdczTvOknO7P~t7X3lobYJe9usIl9jr8kC71qqed768KajcMKJc3xjhhmJfCcG59pAIaYqtoNiI67hSTSACIDp1G2eM~jSGLCzJ9RzVQ7ZknhbM3imllbPlBj65Mzj1FUlSenHj0PlcUDQJFXCxGKPjESJ~y8IaO90K6uDSjG-8xeB9GmHEiaX6w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree obtained from the neighbour-joining analysis of 93 Copia retrotranscriptase sequences. Different Copia families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates a Copia RE with an LTR/inter-LTR ratio of &gt;10." path-from-xml="dsu04204.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596900/dsu04204.tif" path-from-xml="dsu04204.tif" target="_blank">View large</a><a section="6596900" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04204.png?Expires=1509238985&Signature=XP8K9dmG9FpQGF1epgiBIVNgQJdHRB~Rhi5xcVSfhceA2kCZk8TDlG3yQrDHxv8LUO5MPA2ipmdPeFClYjbXfURhsLvuAN4CKJsSMu83TJHl1pbCuYUU0P2rK8xEpTzV~FqozNo-uxr19VnMP2y5UUR93uZO0M58jQaMJ9A1Muf~Q2qV6fh6LoABUiuNyNbai7CiC3cKKetNmPKP0pLNmkaElNBZJglq3OBxvPmCnIa5zKj2oHF0aA7fyeKWS6hg97kEWaPXcwWQfihoNlLGs38umxgqUk1Vot-V3KrchEvkIVEb-G-qKvgMwXoSd7fK8wA73nVemTBkLq6j5DK9-g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596900&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04204.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree obtained from the neighbour-joining analysis of 93 <em>Copia</em> retrotranscriptase sequences. Different <em>Copia</em> families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates a <em>Copia</em> RE with an LTR/inter-LTR ratio of &gt;10.</p></div></div><div content-id="DSU042F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04204.png?Expires=1509238985&Signature=aMRhTF6VdbuFYbcqj~leXCZVwFOt~GeoPbxyFirz1Tuuk3vXt3GZRBlZd11CX42aFvXTf2D4lcctH4-698mv8zePi2nrb4T2sR67tklRyfB4H~Ud0GN3dNXBdHuYN8HViGxGZCYNHZuN3w8AtHEuk8wShFYI8-AdczTvOknO7P~t7X3lobYJe9usIl9jr8kC71qqed768KajcMKJc3xjhhmJfCcG59pAIaYqtoNiI67hSTSACIDp1G2eM~jSGLCzJ9RzVQ7ZknhbM3imllbPlBj65Mzj1FUlSenHj0PlcUDQJFXCxGKPjESJ~y8IaO90K6uDSjG-8xeB9GmHEiaX6w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree obtained from the neighbour-joining analysis of 93 Copia retrotranscriptase sequences. Different Copia families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates a Copia RE with an LTR/inter-LTR ratio of &gt;10." path-from-xml="dsu04204.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596900/dsu04204.tif" path-from-xml="dsu04204.tif" target="_blank">View large</a><a section="6596900" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04204.png?Expires=1509238985&Signature=XP8K9dmG9FpQGF1epgiBIVNgQJdHRB~Rhi5xcVSfhceA2kCZk8TDlG3yQrDHxv8LUO5MPA2ipmdPeFClYjbXfURhsLvuAN4CKJsSMu83TJHl1pbCuYUU0P2rK8xEpTzV~FqozNo-uxr19VnMP2y5UUR93uZO0M58jQaMJ9A1Muf~Q2qV6fh6LoABUiuNyNbai7CiC3cKKetNmPKP0pLNmkaElNBZJglq3OBxvPmCnIa5zKj2oHF0aA7fyeKWS6hg97kEWaPXcwWQfihoNlLGs38umxgqUk1Vot-V3KrchEvkIVEb-G-qKvgMwXoSd7fK8wA73nVemTBkLq6j5DK9-g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596900&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04204.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree obtained from the neighbour-joining analysis of 93 <em>Copia</em> retrotranscriptase sequences. Different <em>Copia</em> families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates a <em>Copia</em> RE with an LTR/inter-LTR ratio of &gt;10.</p></div></div>                        <a id="6596909" scrollto-destination="6596909"></a>
<div id="DSU042F5" data-id="DSU042F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04205.png?Expires=1509238985&Signature=ZJP4ArWyLCbc-TxpDfimaKVHS5MlV15MkcyXulGMcGTyrKsVY~HzxtD2A2nZny9L2ey15Wh7h2Ryo1sFHpniyakHvkN9j1Zmvb23GsGo-qU8ucIu6zAK-WMSJ8l6H7YIWFs2Jw9uzgzkT9LKrojPw3tNivcKWAaJhsrA0EYRyDv5VkQrBCTGIBPfwckOXEGgLBfgnho66ycYQ~SNY9p0anHxybnoCgoPSBvm9zZSICdodQgosDYlnmMO1Z6YiEnvMhQHLWHfUsf20hEsxZwzJ0XU51zihltshLrAqgVaVcBdXo0iFe5IhosARQCgqhgl99U3n6Nzp1qLsr~-MIh7PA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree obtained from the neighbour-joining analysis of 43 Gypsy retrotranscriptase sequences. Different Gypsy families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates two Gypsy REs with an LTR/inter-LTR ratio of &gt;10." path-from-xml="dsu04205.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596909/dsu04205.tif" path-from-xml="dsu04205.tif" target="_blank">View large</a><a section="6596909" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04205.png?Expires=1509238985&Signature=MpljImbhGng8C2OQQB2gCga0dBfcp7PZUs4NhFjfMEU3W-Jf~dZFhkH9cDz9ONBlgNsotBWSc~fqGBtoFqhqLAoJoaG~4A~Ux58k8Zl8iU5JSK7mrti4-epXOPaKvfi6y-q4Y~ydJY0RamlK~iUJNS9~5djHWmehehI6rpCnOHxu~K6mlgQuLGa004FtNUWRBexHsu6186hY8YwyvDEAXtmlyIk9GC4ntXrDCdYgPnwrzaRAJY2zjDSlFpbcFJpiAwukkVqHErolTh51Eh-O9ymqXGPNbLXEYImo2oEE-c-vR2LU-6xhjANTgcVXNwMKtbZ9o72WOOgvCG-hLjVqKA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596909&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04205.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree obtained from the neighbour-joining analysis of 43 <em>Gypsy</em> retrotranscriptase sequences. Different <em>Gypsy</em> families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates two <em>Gypsy</em> REs with an LTR/inter-LTR ratio of &gt;10.</p></div></div><div content-id="DSU042F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04205.png?Expires=1509238985&Signature=ZJP4ArWyLCbc-TxpDfimaKVHS5MlV15MkcyXulGMcGTyrKsVY~HzxtD2A2nZny9L2ey15Wh7h2Ryo1sFHpniyakHvkN9j1Zmvb23GsGo-qU8ucIu6zAK-WMSJ8l6H7YIWFs2Jw9uzgzkT9LKrojPw3tNivcKWAaJhsrA0EYRyDv5VkQrBCTGIBPfwckOXEGgLBfgnho66ycYQ~SNY9p0anHxybnoCgoPSBvm9zZSICdodQgosDYlnmMO1Z6YiEnvMhQHLWHfUsf20hEsxZwzJ0XU51zihltshLrAqgVaVcBdXo0iFe5IhosARQCgqhgl99U3n6Nzp1qLsr~-MIh7PA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Phylogenetic tree obtained from the neighbour-joining analysis of 43 Gypsy retrotranscriptase sequences. Different Gypsy families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates two Gypsy REs with an LTR/inter-LTR ratio of &gt;10." path-from-xml="dsu04205.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596909/dsu04205.tif" path-from-xml="dsu04205.tif" target="_blank">View large</a><a section="6596909" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04205.png?Expires=1509238985&Signature=MpljImbhGng8C2OQQB2gCga0dBfcp7PZUs4NhFjfMEU3W-Jf~dZFhkH9cDz9ONBlgNsotBWSc~fqGBtoFqhqLAoJoaG~4A~Ux58k8Zl8iU5JSK7mrti4-epXOPaKvfi6y-q4Y~ydJY0RamlK~iUJNS9~5djHWmehehI6rpCnOHxu~K6mlgQuLGa004FtNUWRBexHsu6186hY8YwyvDEAXtmlyIk9GC4ntXrDCdYgPnwrzaRAJY2zjDSlFpbcFJpiAwukkVqHErolTh51Eh-O9ymqXGPNbLXEYImo2oEE-c-vR2LU-6xhjANTgcVXNwMKtbZ9o72WOOgvCG-hLjVqKA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596909&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04205.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree obtained from the neighbour-joining analysis of 43 <em>Gypsy</em> retrotranscriptase sequences. Different <em>Gypsy</em> families are indicated by different grey tones (different colours in the online version of DNA Research). For each RE, the area of the symbol indicates the redundancy of that element in the olive genome. The bar represents the genetic distance. Asterisks indicate bootstrap values &gt;50%. The letter S indicates two <em>Gypsy</em> REs with an LTR/inter-LTR ratio of &gt;10.</p></div></div><p>The redundancy of each element is also reported in Figs <a reveal-id="DSU042F4" data-open="DSU042F4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSU042F5" data-open="DSU042F5" class="link link-reveal link-table xref-fig">5</a>. Within each family, especially for the <em>Gypsy</em> superfamily, different elements often showed similar redundancy. In many cases, the least redundant <em>Copia</em> elements did not cluster in the tree. In contrast, <em>Gypsy</em> unclustered elements were generally highly redundant. It should be noted that 9 of 20 <em>Copia</em> elements with mapped reads/kb &gt;15,000 are reported in the retrotranscriptase tree, all belonging to the same, large cluster. In the other cases, they did not carry the RT-encoding domain. Using other RE protein domains, seven more <em>Copia</em> elements with mapped reads/kb &gt;15,000 resulted in the same cluster (data not shown), indicating that the most redundant elements of this superfamily share some sequence similarity. Four of 20 most redundant REs did not carry protein domains, hence their relationship to the other sequences remained unknown.</p>                            <h3 scrollto-destination=6596920 id="6596920" class="section-title" data-legacy-id=s3d>3.4. Putative insertion dates of LTR-REs</h3>
<p>Intact retroelements have a built-in molecular clock that is useful for estimating their insertion times, based on sister LTR divergence. In fact, when an RE is inserted into the genome, its LTRs are usually 100% identical. Considering that the retroelement transcription starts from the R region in the 5′-LTR and terminates at the end of the R region in the 3′-LTR (thus including only one copy of each U5 and U3 regions), the combination of single copy U5 and U3 regions with a hybrid R region during reverse transcription into cDNA yields two identical LTRs at both ends, prior to integration.<sup><a reveal-id="DSU042C12" data-open="DSU042C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Mutations then occur within the two LTRs, and as more time passes since the insertion, the larger the genetic distance between LTRs becomes. Hence, the RE insertion time can be estimated using a nucleotide substitution rate suitable for such elements, which is assumed to be higher than that of gene regions.<sup><a reveal-id="DSU042C32" data-open="DSU042C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>We estimated the synonymous substitution rate of genes in <em>O. europaea</em> by comparing orthologous cDNA sequences of <em>O. europaea</em> and <em>F. excelsior</em> (a species belonging to the Oleaceae family, for which a large amount of sequence data are available; Buggs and Sollars, personal communication), that is, 20 coding sequences for a total of 7,282 nt in <em>O. europaea</em> and 6,552 nt in <em>Fraxinus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The mean number of synonymous substitutions per site (<em>K</em><sub>s</sub>) was 0.077.</p><p>Based on the separation between the olive and the ash tree, estimated to be ∼42.5 MY,<sup><a reveal-id="DSU042C31" data-open="DSU042C31" class="link link-ref link-reveal xref-bibr">31</a></sup> the resulting synonymous substitution rate is 1.8 × 10<sup>−9</sup>, which is lower than those reported for herbaceous species, and similar to those of other perennial tree species.<sup><a reveal-id="DSU042C17" data-open="DSU042C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU042C35" data-open="DSU042C35" class="link link-ref link-reveal xref-bibr">35</a></sup> In fact, it is known that the generation time of a species affects its nucleotide substitution rate<sup><a reveal-id="DSU042C36" data-open="DSU042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and trees have a much longer generation time than herbaceous species.</p><p>It was proposed that mutation rates for LTR-REs may be approximately 2-fold higher than synonymous substitution rates for protein-coding genes.<sup><a reveal-id="DSU042C32" data-open="DSU042C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Consequently, in our calculations of LTR-RE insertion dates, we used a substitution rate per year of 3.6 × 10<sup>−9</sup>.</p><p>The sequences of LTR pairs were compared and the putative insertion times were calculated for each full-length LTR-RE. Taking into account the whole set of full-length REs, the nucleotide distance (<em>K</em><sub>s</sub>) between sister LTRs showed a large variation between retroelements, representing a time-span of, at most, 46 MY. The putative mean age of analysed LTR-REs was 17.94 MY (standard error = 0.54 MY). The distribution of full-length LTR-REs according to their putative insertion date is reported in Fig. <a reveal-id="DSU042F6" data-open="DSU042F6" class="link link-reveal link-table xref-fig">6</a>. </p>                        <a id="6596945" scrollto-destination="6596945"></a>
<div id="DSU042F6" data-id="DSU042F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04206.png?Expires=1509238985&Signature=I~K2umHdAW4MjgAHNrRADsPmC2fTAUN~6c3wB1rJoWvTClQLDFDNBMn1pNM6Kgc-VwPAGnUJ4qKtwz2tI96~fIQr3vtgyGoxwe4kBU-jwbZuVZkpRhtGkOpJMytQ9zRUHVbHCwDoejn-1xpHOrLC0Ctkw-moTHtLaz1RDZYYLmhsfuezcCl54b3ODGjR8uxC17l8PvtkZIK9YyrNTFm3ZAefwvS~yptcTjBP3WW0UsqyAJhNVXcQIxG2N1a5J9Py1JNC4nJ~P95-F~-5Td1X7vSw9OT9wXBHn8EmnZVOoMXItSbuKqG7XyokKPhsYYtfQbn79fYz6NRkp1HoSHJ1JQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distributions of full-length REs identified in this study, according to their estimated insertion ages (MY). Mean insertion dates are reported in parentheses." path-from-xml="dsu04206.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596945/dsu04206.tif" path-from-xml="dsu04206.tif" target="_blank">View large</a><a section="6596945" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04206.png?Expires=1509238985&Signature=XLD3-S-nwbxgMmO28Z1YFY1BRBJqkjAWDJd5uRR-rIv~MMPc8iE-zaT6Uwuy1zsOk9pKkoqQkO1r87w-9vbPtBZJ~Kz61I-up6HhZvOjOS9iPExuXectBh2I8TuXtrRakWiP5KiPUKdZYCnglxu96YMDyq3QTSUeVAqYAKVu3NofsVDLjbmF0exwztwzBZraGSXACJaUBQpdrN0mrs-xfp7zf0HHIn0-JQubrXDJlvMo2SrsUSXOumU-4robuzFW4iKHdKDQFwzUPy5wOVyzLRgXvg8cPp71jY3LDMhPDyBKboiUoO6tREP4zwhln8atB88Bn6i57rGXuvh~kc9Suw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596945&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04206.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of full-length REs identified in this study, according to their estimated insertion ages (MY). Mean insertion dates are reported in parentheses.</p></div></div><div content-id="DSU042F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04206.png?Expires=1509238985&Signature=I~K2umHdAW4MjgAHNrRADsPmC2fTAUN~6c3wB1rJoWvTClQLDFDNBMn1pNM6Kgc-VwPAGnUJ4qKtwz2tI96~fIQr3vtgyGoxwe4kBU-jwbZuVZkpRhtGkOpJMytQ9zRUHVbHCwDoejn-1xpHOrLC0Ctkw-moTHtLaz1RDZYYLmhsfuezcCl54b3ODGjR8uxC17l8PvtkZIK9YyrNTFm3ZAefwvS~yptcTjBP3WW0UsqyAJhNVXcQIxG2N1a5J9Py1JNC4nJ~P95-F~-5Td1X7vSw9OT9wXBHn8EmnZVOoMXItSbuKqG7XyokKPhsYYtfQbn79fYz6NRkp1HoSHJ1JQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Distributions of full-length REs identified in this study, according to their estimated insertion ages (MY). Mean insertion dates are reported in parentheses." path-from-xml="dsu04206.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596945/dsu04206.tif" path-from-xml="dsu04206.tif" target="_blank">View large</a><a section="6596945" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04206.png?Expires=1509238985&Signature=XLD3-S-nwbxgMmO28Z1YFY1BRBJqkjAWDJd5uRR-rIv~MMPc8iE-zaT6Uwuy1zsOk9pKkoqQkO1r87w-9vbPtBZJ~Kz61I-up6HhZvOjOS9iPExuXectBh2I8TuXtrRakWiP5KiPUKdZYCnglxu96YMDyq3QTSUeVAqYAKVu3NofsVDLjbmF0exwztwzBZraGSXACJaUBQpdrN0mrs-xfp7zf0HHIn0-JQubrXDJlvMo2SrsUSXOumU-4robuzFW4iKHdKDQFwzUPy5wOVyzLRgXvg8cPp71jY3LDMhPDyBKboiUoO6tREP4zwhln8atB88Bn6i57rGXuvh~kc9Suw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596945&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04206.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributions of full-length REs identified in this study, according to their estimated insertion ages (MY). Mean insertion dates are reported in parentheses.</p></div></div><p>Analysis of the insertion date profiles provides evidence for a partial overlap among retrotransposition waves of <em>Gypsy</em> and <em>Copia</em> full-length LTR-REs. For example, taking into consideration the last 7 MY, nearly all retrotransposition events involve <em>Copia</em> elements. On the other hand, <em>Gypsy</em>-related elements show a large peak of retrotransposition between 10 and 25 MY ago, in contrast to <em>Copia</em> REs, whose retrotransposition activity is scattered over the last 40 MY.</p><p>Interestingly, two REs (one <em>Copia</em> and one <em>Gypsy</em>) did not show variations between their LTRs, suggesting that insertion should have occurred between 0 and 1.760 MY ago and between 0 and 0.387 MY ago, respectively, that is, the retrotransposition process could still be active.</p><p>The relationship between the insertion age of an RE and its redundancy in the genome is shown in Fig. <a reveal-id="DSU042F7" data-open="DSU042F7" class="link link-reveal link-table xref-fig">7</a>, and indicates the intensity of retrotransposition activity in the period in which retrotransposition has occurred, although the case that the insertion time of an element cannot correspond to the period of largest activity of its RE family cannot be ruled out. </p>                        <a id="6596966" scrollto-destination="6596966"></a>
<div id="DSU042F7" data-id="DSU042F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04207.png?Expires=1509238985&Signature=F8kPI6WeJYY6ATvV9kENfJhOdubiCc8ipv4evB3OLpIZPWwsHTGs73AeUpmrvsV9~tqxzvYX9Pj2M-xLxSMN6VBc~gT~9ioZnyR0qBb20bsMxgfhziLbmvv61o6ox-IJ8PMfsqGBrfncmZbs2bh4M2uOESHxG9tHKglzRUvNO8nRznxT1lwtDvStIpr-JlX9dJXOT4Wl3VRp5zVTnBmUPXAIJhJcjzlW34oYzUNlqqINdtbOjfxcVwy6KwtRzHYFfQbXBKxutYwCpwyEgc8L34wYIwNSGgrWmqxHJ9wZ5KfkxUxlRLZofhUSFhXPl0jd7QSWZoRDIl42ba2C6tUSbw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The relationship between estimated insertion ages (MY) and the redundancy of full-length REs identified in this study." path-from-xml="dsu04207.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596966/dsu04207.tif" path-from-xml="dsu04207.tif" target="_blank">View large</a><a section="6596966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04207.png?Expires=1509238985&Signature=HfRIMC~g7YWZ7lQMHXmC7gSN7FW2l5FhVj3VYhEc8VtMwDxrDiA1N-KKbhaoYZNS5JSTT8d2t3dsjgPUiJVaVq4HK-~h8qe0ujRUwGYgQ7cknq0Wo4kAQV7GN4PMIB~~Vo6-fCVXBDIVpDY7pJuI~GJc53a7sHvaZ0OCsG8OqDSxVDNZpvn6~AKWkNGiGoaQhjd6iJRUKTvCLtbunyMRP8L7DzRYdbDPhRNk65kAf8-ohN7leBGmPYenRBFtViMoiorvDb85k-BKt07a8fnHnvP99U0WO3GnW2uahGBATpUm7J2KpPQFyRqsik500gRlUa9niqwLLBlysLDv5m1doA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596966&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04207.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relationship between estimated insertion ages (MY) and the redundancy of full-length REs identified in this study.</p></div></div><div content-id="DSU042F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/m_dsu04207.png?Expires=1509238985&Signature=F8kPI6WeJYY6ATvV9kENfJhOdubiCc8ipv4evB3OLpIZPWwsHTGs73AeUpmrvsV9~tqxzvYX9Pj2M-xLxSMN6VBc~gT~9ioZnyR0qBb20bsMxgfhziLbmvv61o6ox-IJ8PMfsqGBrfncmZbs2bh4M2uOESHxG9tHKglzRUvNO8nRznxT1lwtDvStIpr-JlX9dJXOT4Wl3VRp5zVTnBmUPXAIJhJcjzlW34oYzUNlqqINdtbOjfxcVwy6KwtRzHYFfQbXBKxutYwCpwyEgc8L34wYIwNSGgrWmqxHJ9wZ5KfkxUxlRLZofhUSFhXPl0jd7QSWZoRDIl42ba2C6tUSbw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The relationship between estimated insertion ages (MY) and the redundancy of full-length REs identified in this study." path-from-xml="dsu04207.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6596966/dsu04207.tif" path-from-xml="dsu04207.tif" target="_blank">View large</a><a section="6596966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu04207.png?Expires=1509238985&Signature=HfRIMC~g7YWZ7lQMHXmC7gSN7FW2l5FhVj3VYhEc8VtMwDxrDiA1N-KKbhaoYZNS5JSTT8d2t3dsjgPUiJVaVq4HK-~h8qe0ujRUwGYgQ7cknq0Wo4kAQV7GN4PMIB~~Vo6-fCVXBDIVpDY7pJuI~GJc53a7sHvaZ0OCsG8OqDSxVDNZpvn6~AKWkNGiGoaQhjd6iJRUKTvCLtbunyMRP8L7DzRYdbDPhRNk65kAf8-ohN7leBGmPYenRBFtViMoiorvDb85k-BKt07a8fnHnvP99U0WO3GnW2uahGBATpUm7J2KpPQFyRqsik500gRlUa9niqwLLBlysLDv5m1doA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6596966&ar=441732&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04207.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relationship between estimated insertion ages (MY) and the redundancy of full-length REs identified in this study.</p></div></div><p>Figure <a reveal-id="DSU042F7" data-open="DSU042F7" class="link link-reveal link-table xref-fig">7</a> suggests that intense retrotransposition waves have occurred, for some RE families of both superfamilies, between 10 and 30 MY ago. One <em>Gypsy</em> and two <em>Copia</em> families may have been especially active between 23 and 29 MY ago.</p>                            <h2 scrollto-destination=6596972 id="6596972" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The repetitive component of the olive genome amounts to &gt;70%.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Around 30% of the genome is made of tandem repeat sequences and 40% of transposable elements. Such a genome composition appears to be peculiar to <em>O. europaea</em>. In other species, tandem repeats account for &lt;10% of the genome, with the exception of the genome of cucumber,<sup><a reveal-id="DSU042C37" data-open="DSU042C37" class="link link-ref link-reveal xref-bibr">37</a></sup> which is composed of ∼23% of such sequences, a percentage, however, lower than that measured in the olive genome. In plant species other than olive, transposable elements account for the vast majority of the repetitive component. The partial replacement of REs by tandem repeats in the olive genome prompted us to study the dynamics of LTR-REs in this species.</p><p>We analysed olive LTR-REs based on sister LTR identification in large contigs obtained assembling Illumina-sequenced BAC pools. By this approach, only putative full-length REs, that is, with two very similar LTRs, were scored. In total, we have isolated 254 full-length LTR-REs that can be added to the already available RE sequences.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>The isolated full-length REs and their remnants represent 12.33% of the olive genome and constitute a particular fraction of these sequences. First, the sample of REs may have been biased by the assembly process. Repetitive elements represent the most difficult parts to assemble in any shotgun assembly project and are often under-represented in whole-genome assemblies. We assembled pools of 384 BACs that should on average each comprise 42,000 nt of genomic sequence, corresponding to &lt;3% of the total genome size for olive. This should make the assembly of repetitive elements easier, even though difficulties may remain for high-copy sequences (that are represented in multiple copies within each pool) as well as for elements that have identical LTRs at their ends, where it may have been difficult to assemble a complete element with both LTRs. This may have affected not only the representation of different families and superfamilies, but also our estimation of insertion times of elements, because the REs with identical or nearly identical LTRs may not have been reconstructed as complete elements. Secondly, the method used to isolate full-length REs allowed us to select especially dispersed elements, whereas nested elements were probably under-represented in the sample, and it is known that nested REs constitute a large fraction of the repetitive component in medium- to large-sized genomes, such as those of maize<sup><a reveal-id="DSU042C38" data-open="DSU042C38" class="link link-ref link-reveal xref-bibr">38</a></sup> or sunflower.<sup><a reveal-id="DSU042C16" data-open="DSU042C16" class="link link-ref link-reveal xref-bibr">16</a></sup> It is possible that the REs isolated in this work are preferentially located in the gene-rich fraction of the olive genome, as suggested by the large number of protein-coding sequences that were found in the upstream or downstream region of REs. Nevertheless, the selected set of 254 full-length REs constitutes a valuable sample of these sequences, allowing an analysis of RE dynamics in this species.</p><p>In angiosperms, <em>Gypsy</em> and <em>Copia</em> superfamilies are differently represented in the genome. Different ratios between <em>Gypsy</em> and <em>Copia</em> RE frequencies were reported,<sup><a reveal-id="DSU042C39" data-open="DSU042C39" class="link link-ref link-reveal xref-bibr">39</a></sup> ranging from 5 : 1 in papaya<sup><a reveal-id="DSU042C40" data-open="DSU042C40" class="link link-ref link-reveal xref-bibr">40</a></sup> to 1 : 2 in grapevine.<sup><a reveal-id="DSU042C41" data-open="DSU042C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Analysis of the whole olive genome showed a ratio of 1.17.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The isolated olive full-length REs showed, on the contrary, a slight prevalence of <em>Copia</em> over <em>Gypsy</em> elements. This could be related to a different accumulation in genic and intergenic regions of the genome between <em>Copia</em> and <em>Gypsy</em> elements. In fact, <em>Copia</em> REs are often scattered on the chromosomes, whereas <em>Gypsy</em> elements preferentially accumulate—in a nested way—in specific chromosomal locations and structures, such as centromeric and pericentromeric regions.<sup><a reveal-id="DSU042C42" data-open="DSU042C42" class="link link-ref link-reveal xref-bibr">42</a></sup> In this sense, the larger number of isolated <em>Copia</em> REs would be a consequence of the procedure used for RE identification. On the other hand, analysis of RE redundancy clearly showed that isolated full-length <em>Gypsy</em> elements, although at a lower number than <em>Copia</em>, account for a larger fraction of the genome, suggesting that they are more redundant, and confirming the previously reported results.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>The occurrence of RE families in the olive genome was established according primarily to sequence similarity of their RT-coding sequences. Although, in many cases, bootstrap values were &lt;50%, the trees suggest the occurrence of a number of RE clusters. It is to be noted that the most redundant elements (both <em>Gypsy</em> and <em>Copia</em>) are clustered, indicating that the corresponding families have been highly active during olive genome evolution.</p><p>The relatively low frequency of REs in the olive genome could be related to a low rate of retrotransposition, but also to RE loss.<sup><a reveal-id="DSU042C43" data-open="DSU042C43" class="link link-ref link-reveal xref-bibr">43</a></sup> RE DNA removal is driven in plants by a number of mechanisms, including DNA rearrangements and unequal homologous recombination; solo-LTRs are the main products of such processes.<sup><a reveal-id="DSU042C32" data-open="DSU042C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSU042C44" data-open="DSU042C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSU042C45" data-open="DSU042C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p><p>Analysing the relative redundancy of LTRs and inter-LTR regions in the same full-length RE allowed us to evaluate the occurrence of solo-LTRs related (i.e. belonging to the same family) to that RE. Solo-LTRs related to the isolated full-length REs were rare: only 16 of 254 REs showed a ratio between the number of mapped reads per kb of LTR and inter-LTR &gt;2.5. The LTR/inter-LTR ratios of <em>Gypsy</em> elements were generally higher than those of <em>Copia</em>. These ratios were especially high for one <em>Copia</em> and two <em>Gypsy</em> elements, sharing sequence similarity with chromodomain-containing and <em>Ale</em> REs of other species, indicating the occurrence of a large number of solo-LTRs for RE families that are related to these full-length elements. However, the presence of REs sharing LTRs but not internal regions cannot be ruled out and could lead to an overestimation of solo-LTR frequencies.</p><p>If genome size derives from an equilibrium between enlargement (by polyploidization and RE amplification) and reduction (by DNA loss) in olive, the genome size was increased by massive amplification of REs and of tandem repeats.<sup><a reveal-id="DSU042C18" data-open="DSU042C18" class="link link-ref link-reveal xref-bibr">18</a></sup> This increase was partly counterbalanced by DNA loss, related to both <em>Gypsy</em> and <em>Copia</em> elements, in contrast to other species in which solo-LTRs have been found especially in <em>Copia</em> elements, as, for example, in sunflower.<sup><a reveal-id="DSU042C46" data-open="DSU042C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSU042C47" data-open="DSU042C47" class="link link-ref link-reveal xref-bibr">47</a></sup></p><p>Concerning the amplification of REs, the identification of sister LTRs allowed us, for the first time, to date the insertion of REs in the olive genome, using the method established by SanMiguel <em>et al.</em><sup><a reveal-id="DSU042C19" data-open="DSU042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> in maize. Obviously, the estimation of insertion time by the number of mutations in sister LTRs is subject to error, because it assumes that the same mutation rates operate in all retroelements and chromosome positions, while that was not proved to be true in, for example, the genus <em>Oryza</em>.<sup><a reveal-id="DSU042C48" data-open="DSU042C48" class="link link-ref link-reveal xref-bibr">48</a></sup> However, this method appears as the most suitable to study RE dynamics.</p><p>Analysis of sister LTR similarity indicates that, in olive, both <em>Gypsy</em> and <em>Copia</em> REs have been active in the same period. Nearly, all the identified full-length elements appear to be mobilized in a time-span of 40 MY (Fig. <a reveal-id="DSU042F4" data-open="DSU042F4" class="link link-reveal link-table xref-fig">4</a>), although it is conceivable that more ancient REs are not easily recognizable because of accumulation of variability between sister LTRs.</p><p>The mean insertion date of olive <em>Copia</em> full-length REs is lower than that of <em>Gypsy</em>. The insertion date profiles indicate that, during the last 40 MY, <em>Copia</em> and <em>Gypsy</em> REs have both been active, but with different time courses. For example, only one isolated <em>Gypsy</em> full-length RE inserted between 1 and 8 MY ago. Moreover, the percentage of <em>Gypsy</em> REs inserted between 10 and 25 MY ago, and hence, presumably, their retrotransposition activity is by far larger than that of <em>Copia</em> elements. Different amplification histories of these RE superfamilies during the evolution of the host species have been reported in many plant species. <em>De facto</em> analysis of crop genomes in a phylogenetic context reveals scarce congruence in RE content and highlights differences in the success of different RE types.<sup><a reveal-id="DSU042C39" data-open="DSU042C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>In contrast to other species, such as maize<sup><a reveal-id="DSU042C14" data-open="DSU042C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and sunflower,<sup><a reveal-id="DSU042C49" data-open="DSU042C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSU042C50" data-open="DSU042C50" class="link link-ref link-reveal xref-bibr">50</a></sup> in which the RE burst is very recent and probably still occurring, in the olive genome the insertion of new REs appears to be decreasing in frequency in the last 8 MY, for both <em>Gypsy</em> and <em>Copia</em> REs. A similar time course of the RE amplification wave was reported in the genome of a gymnosperm, the Norway spruce.<sup><a reveal-id="DSU042C35" data-open="DSU042C35" class="link link-ref link-reveal xref-bibr">35</a></sup> While our estimates of insertion ages may have been biased against the most recent elements by the assembly process as discussed previously, it is to be considered that all those REs interrupted by other elements (i.e. presumably older than inserted ones) are not included in the sample. The observation of a considerable number of elements inserted &gt;10 MY ago is still valid and represents a clear distinctive feature of the olive genome in comparison with other angiosperm genomes analysed so far.</p><p>The relationship between insertion time of an element and its redundancy offers further insights into olive RE dynamics. <em>Gypsy</em> and <em>Copia</em> full-length REs inserted between 10 and 30 MY ago are by far the most abundant, while recently inserted elements show low levels of redundancy. This is expected because it is known that lowly redundant elements are more prone to escape RE silencing. The age distribution of REs further suggests a progressive reduction in RE activity from 20 MY ago until now.</p><p>Dating the amplification process of another major repeat class in the olive genome, the tandem repeats, could clarify if a type of ‘competition’ in the genome colonization has occurred between repeat types during olive evolution.</p><p>Concerning olive RE transcription, no data are currently available. Ancient LTR-REs are generally inactive or less active than young ones, probably because of the accumulation of mutations determining premature stop codons in the coding portion of the LTR-RE, as observed in rice.<sup><a reveal-id="DSU042C51" data-open="DSU042C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Moreover, there is also a strong control of RE activity by the host species; it has been established that REs are especially silenced by siRNA.<sup><a reveal-id="DSU042C52" data-open="DSU042C52" class="link link-ref link-reveal xref-bibr">52</a></sup> It is plausible that the large number of LTR-RE fragments spread throughout the olive genome can produce siRNAs that silence related retroelements.</p><p>In conclusion, our analyses show many aspects of RE dynamics in the evolution of the olive genome. Some data are similar to those observed in other plant species, but some peculiarities of the olive genome also emerged for this repeat class, besides the extreme redundancy of tandem repeats. All these data support the theory that if RE dynamics are similar, including birth through transposition, silencing and then death by both random mutation and possibly deletion from the genome,<sup><a reveal-id="DSU042C53" data-open="DSU042C53" class="link link-ref link-reveal xref-bibr">53</a></sup> the factors inducing such dynamics might be different in different RE lineages and possibly related to the ‘ecosystem’ in which the REs interact and compete.<sup><a reveal-id="DSU042C54" data-open="DSU042C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Hence, according to Venner <em>et al.</em>,<sup><a reveal-id="DSU042C55" data-open="DSU042C55" class="link link-ref link-reveal xref-bibr">55</a></sup> olive REs can be considered a community of different organisms in the genome, with ‘species’ (corresponding to RE superfamilies) and ‘subspecies’, characterized by different LTR sequences, activity, and evolution history.</p>                            <h2 scrollto-destination=6597009 id="6597009" class="section-title" data-legacy-id=s5>5. Data retrieval</h2>
<p>Whole-genome shotgun sequences described are available on NCBI Sequence Read Archive under SRA Project number SRX465835. Assembled BAC sequences and RE sequences are available at the Sequence Repository Page of the Department of Agriculture, Food, and Environment of University of Pisa (<a class="link link-uri" href="http://www.agr.unipi.it/ricerca/plant-genetics-and-genomics-lab/sequence-repository.html">http://www.agr.unipi.it/ricerca/plant-genetics-and-genomics-lab/sequence-repository.html</a>, 12 November 2014, date last accessed) and on the NCBI website (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>, 12 November 2014, date last accessed) under the accession numbers KM577349–KM577602.</p>                            <h2 scrollto-destination=6597011 id="6597011" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6597013 id="6597013" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This research work was funded by Ministero delle Politiche Agricole, Italy, Progetto ‘OLEA: Genomica e miglioramento genetico dell'olivo’. BAC library construction was supported by the Czech Science Foundation (award no. P501/12/G090) and by grant award LO1204 from the National Program of Sustainability I. Funding to pay the Open Access publication charges for this article was provided by Department of Agriculture, Food and Environment, University of Pisa.</p>                            <h2 scrollto-destination=6597015 id="6597015" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to our colleagues Jarmila Číhalíková, Jana Dostálová, and Zdeňka Dubská for the assistance with nuclei sorting and BAC library construction, and to Dr Jana Čížková for the estimation of nuclear genome size. We thank Drs Richard Buggs and Elizabeth Sollars (The British Ash Tree Genome Project, School of Biological &amp; Chemical Sciences, Queen Mary University of London, UK) for kindly providing ash tree cDNA sequences.</p>                            <h2 scrollto-destination=6597017 id="6597017" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsu042c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stark</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Madar</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Olive oil as a functional food: epidemiology and nutritional approaches</div>, <div class="source">Nutrition Rev.</div> , <div class="year">2002</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">170</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Olive%20oil%20as%20a%20functional%20food%3A%20epidemiology%20and%20nutritional%20approaches&amp;author=A.H.%20Stark&amp;author=Z.%20Madar&amp;publication_year=2002&amp;journal=Nutrition%20Rev.&amp;volume=60&amp;pages=170-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1301/002966402320243250" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1301%2F002966402320243250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1301%2F002966402320243250"> </span></div> </div></div></div></div></div><div content-id="dsu042c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sarri</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Baldoni</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Porceddu</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Microsatellite markers are powerful tools for discriminating among olive cultivars and assigning them to geographically defined populations</div>, <div class="source">Genome</div> , <div class="year">2006</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">1606</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20are%20powerful%20tools%20for%20discriminating%20among%20olive%20cultivars%20and%20assigning%20them%20to%20geographically%20defined%20populations&amp;author=V.%20Sarri&amp;author=L.%20Baldoni&amp;author=A.%20Porceddu&amp;publication_year=2006&amp;journal=Genome&amp;volume=49&amp;pages=1606-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g06-126" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg06-126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg06-126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17426775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Besnard</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Baradat</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bervillé</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Genetic relationships in the olive (<em>Olea europaea</em> L.) reflect multilocal selection of cultivars</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">251</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20relationships%20in%20the%20olive%20(Olea%20europaea%20L.)%20reflect%20multilocal%20selection%20of%20cultivars&amp;author=G.%20Besnard&amp;author=P.%20Baradat&amp;author=A.%20Bervill%C3%A9&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=251-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051642" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051642"> </span></div> </div></div></div></div></div><div content-id="dsu042c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loureiro</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rodriguez</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Costa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Santos</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Nuclear DNA content estimations in wild olive (<em>Olea europaea</em> L. ssp. <em>europaea</em> var. <em>sylvestris</em> Brot.) and Portuguese cultivars of <em>O. europaea</em> using flow cytometry</div>, <div class="source">Genet. Res. Crop Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">21</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20estimations%20in%20wild%20olive%20(Olea%20europaea%20L.%20ssp.%20europaea%20var.%20sylvestris%20Brot.)%20and%20Portuguese%20cultivars%20of%20O.%20europaea%20using%20flow%20cytometry&amp;author=J.%20Loureiro&amp;author=E.%20Rodriguez&amp;author=A.%20Costa&amp;author=C.%20Santos&amp;publication_year=2007&amp;journal=Genet.%20Res.%20Crop%20Evol.&amp;volume=54&amp;pages=21-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-006-9115-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-006-9115-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-006-9115-3"> </span></div> </div></div></div></div></div><div content-id="dsu042c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katsiotis</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hagidimitriou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Douka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hatzopoulos</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Genomic organization, sequence interrelationship, and physical localization using in situ hybridization of two tandemly repeated DNA sequences in the genus <em>Olea</em></div>, <div class="source">Genome</div> , <div class="year">1998</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">527</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20organization%2C%20sequence%20interrelationship%2C%20and%20physical%20localization%20using%20in%20situ%20hybridization%20of%20two%20tandemly%20repeated%20DNA%20sequences%20in%20the%20genus%20Olea&amp;author=A.%20Katsiotis&amp;author=M.%20Hagidimitriou&amp;author=A.%20Douka&amp;author=P.%20Hatzopoulos&amp;publication_year=1998&amp;journal=Genome&amp;volume=41&amp;pages=527-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-41-4-527" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-41-4-527" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-41-4-527"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9796101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bitonti</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Cozza</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chiappetta</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Amount and organization of the heterochromatin in <em>Olea europaea</em> and related species</div>, <div class="source">Heredity</div> , <div class="year">1999</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">188</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amount%20and%20organization%20of%20the%20heterochromatin%20in%20Olea%20europaea%20and%20related%20species&amp;author=M.B.%20Bitonti&amp;author=R.%20Cozza&amp;author=A.%20Chiappetta&amp;publication_year=1999&amp;journal=Heredity&amp;volume=83&amp;pages=188-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2540.1999.00564.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2540.1999.00564.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2540.1999.00564.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10469207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lorite</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Garcia</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Carrillo</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Palomeque</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A new repetitive DNA sequence family in the olive (<em>Olea europaea</em> L)</div>, <div class="source">Hereditas</div> , <div class="year">2001</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">73</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20repetitive%20DNA%20sequence%20family%20in%20the%20olive%20(Olea%20europaea%20L)&amp;author=P.%20Lorite&amp;author=M.F.%20Garcia&amp;author=J.A.%20Carrillo&amp;author=T.%20Palomeque&amp;publication_year=2001&amp;journal=Hereditas&amp;volume=134&amp;pages=73-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1601-5223.2001.00073.x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1601-5223.2001.00073.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1601-5223.2001.00073.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11525067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stergiou</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Katsiotis</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hagidimitriou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Loukas</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genomic and chromosomal organization of Ty1-<em>Copia</em>-like sequences in <em>Olea europaea</em> and evolutionary relationships of <em>Olea</em> retroelements</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">926</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20and%20chromosomal%20organization%20of%20Ty1-Copia-like%20sequences%20in%20Olea%20europaea%20and%20evolutionary%20relationships%20of%20Olea%20retroelements&amp;author=G.%20Stergiou&amp;author=A.%20Katsiotis&amp;author=M.%20Hagidimitriou&amp;author=M.%20Loukas&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=926-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-001-0835-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-001-0835-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-001-0835-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Natali</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Giordani</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Buti</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cavallini</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Isolation of Ty1-<em>Copia</em> putative LTR sequences and their use as a tool to analyse genetic diversity in <em>Olea europaea</em></div>, <div class="source">Mol. Breed.</div> , <div class="year">2007</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">255</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20Ty1-Copia%20putative%20LTR%20sequences%20and%20their%20use%20as%20a%20tool%20to%20analyse%20genetic%20diversity%20in%20Olea%20europaea&amp;author=L.%20Natali&amp;author=T.%20Giordani&amp;author=M.%20Buti&amp;author=A.%20Cavallini&amp;publication_year=2007&amp;journal=Mol.%20Breed.&amp;volume=19&amp;pages=255-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-006-9060-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-006-9060-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-006-9060-z"> </span></div> </div></div></div></div></div><div content-id="dsu042c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Leebens-Mack</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ayyampalayam</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A genome triplication associated with early diversification of the core eudicots</div>, <div class="source">Genome Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">R3</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome%20triplication%20associated%20with%20early%20diversification%20of%20the%20core%20eudicots&amp;author=Y.%20Jiao&amp;author=J.%20Leebens-Mack&amp;author=S.%20Ayyampalayam&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=R3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-1-r3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-1-r3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-1-r3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22280555" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hawkins</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Nason</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Wing</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Differential lineage-specific amplification of transposable elements is responsible for genome size variation in <em>Gossypium</em></div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1252</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20lineage-specific%20amplification%20of%20transposable%20elements%20is%20responsible%20for%20genome%20size%20variation%20in%20Gossypium&amp;author=J.S.%20Hawkins&amp;author=H.R.%20Kim&amp;author=J.D.%20Nason&amp;author=R.A.%20Wing&amp;author=J.F.%20Wendel&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1252-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5282906" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5282906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5282906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16954538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Plant retrotransposons</div>, <div class="source">Ann. Rev. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">479</div>-<div class="lpage">532</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20retrotransposons&amp;author=A.%20Kumar&amp;author=J.B.%20Bennetzen&amp;publication_year=1999&amp;journal=Ann.%20Rev.%20Genet.&amp;volume=33&amp;pages=479-532" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.33.1.479" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.33.1.479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.33.1.479"> </span></div> </div></div></div></div></div><div content-id="dsu042c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beguiristain</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Grandbastien</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Puigdomenech</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Casacuberta</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Three Tnt1 subfamilies show different stress-associated patterns of expression in tobacco. Consequences for retrotransposon control and evolution in plants</div>, <div class="source">Plant Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">212</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20Tnt1%20subfamilies%20show%20different%20stress-associated%20patterns%20of%20expression%20in%20tobacco.%20Consequences%20for%20retrotransposon%20control%20and%20evolution%20in%20plants&amp;author=T.%20Beguiristain&amp;author=M.A.%20Grandbastien&amp;author=P.%20Puigdomenech&amp;author=J.M.%20Casacuberta&amp;publication_year=2001&amp;journal=Plant%20Physiol.&amp;volume=127&amp;pages=212-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.127.1.212" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.127.1.212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.127.1.212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11553749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fengler</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tingey</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evolution of DNA sequence nonhomologies among maize inbreds</div>, <div class="source">Plant Cell</div> , <div class="year">2005</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">343</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20DNA%20sequence%20nonhomologies%20among%20maize%20inbreds&amp;author=S.%20Brunner&amp;author=K.%20Fengler&amp;author=M.%20Morgante&amp;author=S.%20Tingey&amp;author=A.%20Rafalski&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=343-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.025627" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.025627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.025627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Bowers</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Bruggmann</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">457</div> (pg. <div class="fpage">551</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;author=A.H.%20Paterson&amp;author=J.E.%20Bowers&amp;author=R.%20Bruggmann&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=551-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07723" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buti</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Giordani</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cattonaro</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Temporal dynamics in the evolution of the sunflower genome as revealed by sequencing and annotation of three large genomic regions</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">779</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporal%20dynamics%20in%20the%20evolution%20of%20the%20sunflower%20genome%20as%20revealed%20by%20sequencing%20and%20annotation%20of%20three%20large%20genomic%20regions&amp;author=M.%20Buti&amp;author=T.%20Giordani&amp;author=F.%20Cattonaro&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=779-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1626-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1626-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1626-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21647740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cossu</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Buti</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Giordani</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Natali</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cavallini</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A computational study of the dynamics of LTR retrotransposons in the <em>Populus trichocarpa</em> genome</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">61</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computational%20study%20of%20the%20dynamics%20of%20LTR%20retrotransposons%20in%20the%20Populus%20trichocarpa%20genome&amp;author=R.M.%20Cossu&amp;author=M.%20Buti&amp;author=T.%20Giordani&amp;author=L.%20Natali&amp;author=A.%20Cavallini&amp;publication_year=2012&amp;journal=Tree%20Genet.%20Genomes&amp;volume=8&amp;pages=61-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-011-0421-3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-011-0421-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-011-0421-3"> </span></div> </div></div></div></div></div><div content-id="dsu042c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barghini</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Natali</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cossu</div> <div class="given-names">R.M.</div></div>, et al. </div><div class="article-title">The peculiar landscape of repetitive sequences in the olive (<em>Olea europaea</em> L.) genome</div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2014</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">776</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20peculiar%20landscape%20of%20repetitive%20sequences%20in%20the%20olive%20(Olea%20europaea%20L.)%20genome&amp;author=E.%20Barghini&amp;author=L.%20Natali&amp;author=R.M.%20Cossu&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol.&amp;volume=6&amp;pages=776-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu058" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24671744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">SanMiguel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Tikhonov</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The paleontology of intergene retrotransposons of maize</div>, <div class="source">Nat. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">43</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20paleontology%20of%20intergene%20retrotransposons%20of%20maize&amp;author=P.%20SanMiguel&amp;author=B.S.%20Gaut&amp;author=A.%20Tikhonov&amp;publication_year=1998&amp;journal=Nat.%20Genet.&amp;volume=20&amp;pages=43-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/1695" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F1695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F1695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9731528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doležel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Číhalíková</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lucretti</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A high-yield procedure for isolation of metaphase chromosomes from root tips of <em>Vicia faba</em> L.</div>, <div class="source">Planta</div> , <div class="year">1992</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">93</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-yield%20procedure%20for%20isolation%20of%20metaphase%20chromosomes%20from%20root%20tips%20of%20Vicia%20faba%20L.&amp;author=J.%20Dole%C5%BEel&amp;author=J.%20%C4%8C%C3%ADhal%C3%ADkov%C3%A1&amp;author=S.%20Lucretti&amp;publication_year=1992&amp;journal=Planta&amp;volume=188&amp;pages=93-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00198944" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00198944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00198944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24178204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Šimková</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Číhalíková</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vrána</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lysák</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Doležel</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Preparation of HMW DNA from plant nuclei and chromosomes isolated from root tips</div>, <div class="source">Biol. Plant</div> , <div class="year">2003</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">369</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preparation%20of%20HMW%20DNA%20from%20plant%20nuclei%20and%20chromosomes%20isolated%20from%20root%20tips&amp;author=H.%20%C5%A0imkov%C3%A1&amp;author=J.%20%C4%8C%C3%ADhal%C3%ADkov%C3%A1&amp;author=J.%20Vr%C3%A1na&amp;author=M.A.%20Lys%C3%A1k&amp;author=J.%20Dole%C5%BEel&amp;publication_year=2003&amp;journal=Biol.%20Plant&amp;volume=46&amp;pages=369-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1024322001786" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1024322001786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1024322001786"> </span></div> </div></div></div></div></div><div content-id="dsu042c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Šimková</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Šafář</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kubaláková</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">BAC libraries from wheat chromosome 7D: efficient tool for positional cloning of aphid resistance genes</div>, <div class="source">J. Biomed. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">20</div> <div class="issue">11</div>pg. <div class="fpage">302543</div> </div></div></div></div><div content-id="dsu042c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>, <div class="source">EMBnet. J.</div> , <div class="year">2011</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">10</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=EMBnet.%20J.&amp;volume=17&amp;pages=10-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="dsu042c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jackman</div> <div class="given-names">S.D.</div></div>, et al. </div><div class="article-title">ABySS: a parallel assembler for short read sequence data</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1117</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ABySS%3A%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data&amp;author=J.T.%20Simpson&amp;author=K.%20Wong&amp;author=S.D.%20Jackman&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1117-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089532.108" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089532.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089532.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W265</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LTR_FINDER%3A%20an%20efficient%20tool%20for%20the%20prediction%20of%20full-length%20LTR%20retrotransposons&amp;author=Z.%20Xu&amp;author=H.%20Wang&amp;publication_year=2007&amp;journal=Nucl.%20Acids%20Res.&amp;volume=35&amp;pages=W265-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm286" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm286"> </span></div> </div></div></div></div></div><div content-id="dsu042c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sonnhammer</div> <div class="given-names">E.L.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A dot-matriprogram with dynamic threshold control suited for genomic DNA and protein sequence analysis</div>, <div class="source">Gene</div> , <div class="year">1995</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">GC1</div>-<div class="lpage">GC10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dot-matriprogram%20with%20dynamic%20threshold%20control%20suited%20for%20genomic%20DNA%20and%20protein%20sequence%20analysis&amp;author=E.L.%20Sonnhammer&amp;author=R.%20Durbin&amp;publication_year=1995&amp;journal=Gene&amp;volume=167&amp;pages=GC1-GC10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(95)00714-8" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(95)00714-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(95)00714-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8566757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">CLUSTALW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTALW%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucl.%20Acids%20Res.&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div> </div></div></div></div></div><div content-id="dsu042c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Munoz-Merida</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">González-Plaza</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Cañada</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title"><em>De novo</em> assembly and functional annotation of the olive (<em>Olea europaea</em>) transcriptome</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">93</div>-<div class="lpage">108</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20functional%20annotation%20of%20the%20olive%20(Olea%20europaea)%20transcriptome&amp;author=A.%20Munoz-Merida&amp;author=J.J.%20Gonz%C3%A1lez-Plaza&amp;author=A.%20Ca%C3%B1ada&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=93-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss036" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23297299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1986</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">418</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20methods%20for%20estimating%20the%20numbers%20of%20synonymous%20and%20nonsynonymous%20nucleotide%20substitutions&amp;author=M.%20Nei&amp;author=T.%20Gojobori&amp;publication_year=1986&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=3&amp;pages=418-26" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3444411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rozas</div> <div class="given-names">R.</div></div>. </div><div class="article-title">DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">174</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%20version%203%3A%20an%20integrated%20program%20for%20molecular%20population%20genetics%20and%20molecular%20evolution%20analysis&amp;author=J.%20Rozas&amp;author=R.%20Rozas&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=174-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.2.174" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.2.174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.2.174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10089204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Besnard</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Rubio de Casas</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Christin</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Vargas</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Phylogenetics of <em>Olea</em> (Oleaceae) based on plastid and nuclear ribosomal DNA sequences: tertiary climatic shifts and lineage differentiation times</div>, <div class="source">Ann. Bot.</div> , <div class="year">2009</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">143</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetics%20of%20Olea%20(Oleaceae)%20based%20on%20plastid%20and%20nuclear%20ribosomal%20DNA%20sequences%3A%20tertiary%20climatic%20shifts%20and%20lineage%20differentiation%20times&amp;author=G.%20Besnard&amp;author=R.%20Rubio%20de%20Casas&amp;author=P.A.%20Christin&amp;author=P.%20Vargas&amp;publication_year=2009&amp;journal=Ann.%20Bot.&amp;volume=104&amp;pages=143-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcp105" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcp105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcp105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19465750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Rapid recent growth and divergence of rice nuclear genomes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">12404</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20recent%20growth%20and%20divergence%20of%20rice%20nuclear%20genomes&amp;author=J.%20Ma&amp;author=J.L.%20Bennetzen&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=12404-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0403715101" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0403715101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0403715101"> </span></div> </div></div></div></div></div><div content-id="dsu042c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Repbase Update, a database of eukaryotic repetitive elements</div>, <div class="source"><em>Cytogenet. Genome Res</em>.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">462</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res.&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gorinsek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gubensek</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kordis</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Evolutionary genomics of chromoviruses in eukaryotes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">781</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20genomics%20of%20chromoviruses%20in%20eukaryotes&amp;author=B.%20Gorinsek&amp;author=F.%20Gubensek&amp;author=D.%20Kordis&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=781-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh057" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14739248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nystedt</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Nathaniel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Street</div> <div class="given-names">N.R.</div></div>, et al. </div><div class="article-title">The Norway spruce genome sequence and conifer genome evolution</div>, <div class="source">Nature</div> , <div class="year">2013</div>, vol. <div class="volume">497</div> (pg. <div class="fpage">579</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Norway%20spruce%20genome%20sequence%20and%20conifer%20genome%20evolution&amp;author=B.%20Nystedt&amp;author=R.%20Nathaniel&amp;author=N.R.%20Street&amp;publication_year=2013&amp;journal=Nature&amp;volume=497&amp;pages=579-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12211" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23698360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Hecht</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Macintyre</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Clegg</div> <div class="given-names">M.T.</div></div>. </div><div class="article-title">Molecular clocks and nucleotide substitution rates in higher plants</div>, <div class="source">Evolutionary Biology</div> , <div class="year">1998</div>, vol. <div class="volume">30</div> <div class="publisher-loc">New York</div><div class="publisher-name">Plenum Press</div>(pg. <div class="fpage">93</div>-<div class="lpage">120</div>)</div></div></div></div><div content-id="dsu042c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">The genome of the cucumber, <em>Cucumis sativus</em> L.</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">1275</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20cucumber%2C%20Cucumis%20sativus%20L.&amp;author=S.%20Huang&amp;author=R.%20Li&amp;author=Z.%20Zhang&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=1275-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.475" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fulton</div> <div class="given-names">R.S.</div></div>, et al. </div><div class="article-title">The B73 maize genome: complexity, diversity, and dynamics</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1112</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1178534" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1178534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vitte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fustier</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Alix</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tenaillon</div> <div class="given-names">M.I.</div></div>. </div><div class="article-title">The bright side of transposons in crop evolution</div>, <div class="source">Brief. Funct. Genomics</div> , <div class="year">2014</div> <div class="comment">.</div><div class="pub-id-doi"><a href="https://doi.org/10.1093/bfgp/elu002" class="link link-pub-id-doi" target="_blank">
                    https://doi.org/10.1093/bfgp/elu002</a></div></div></div></div></div><div content-id="dsu042c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ming</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The draft genome of the transgenic tropical fruit tree papaya (<em>Carica papaya</em> Linnaeus)</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">452</div> (pg. <div class="fpage">991</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20the%20transgenic%20tropical%20fruit%20tree%20papaya%20(Carica%20papaya%20Linnaeus)&amp;author=R.%20Ming&amp;author=S.%20Hou&amp;author=Y.%20Feng&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=991-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06856" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06856" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06856"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18432245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="collab">The French-Italian Public Consortium for Grape Genome Characterization</div><div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">463</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neumann</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Navrátilová</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Koblížková</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Plant centromeric retrotransposons: a structural and cytogenetic perspective</div>, <div class="source">Mobile DNA</div> , <div class="year">2011</div>, vol. <div class="volume">2</div> pg. <div class="fpage">4</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20centromeric%20retrotransposons%3A%20a%20structural%20and%20cytogenetic%20perspective&amp;author=P.%20Neumann&amp;author=A.%20Navr%C3%A1tilov%C3%A1&amp;author=A.%20Kobl%C3%AD%C5%BEkov%C3%A1&amp;publication_year=2011&amp;journal=Mobile%20DNA&amp;volume=2&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1759-8753-2-4" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1759-8753-2-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1759-8753-2-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21371312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Analyses of LTR-retrotransposon structures reveal recent and rapid genomic DNA loss in rice</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">860</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyses%20of%20LTR-retrotransposon%20structures%20reveal%20recent%20and%20rapid%20genomic%20DNA%20loss%20in%20rice&amp;author=J.%20Ma&amp;author=K.M.%20Devos&amp;author=J.L.%20Bennetzen&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=860-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1466204" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1466204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1466204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15078861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ammiraju</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Zuccolo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Evolutionary dynamics of an ancient retrotransposon family provides insights into evolution of genome size in the genus <em>Oryza</em></div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">342</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20an%20ancient%20retrotransposon%20family%20provides%20insights%20into%20evolution%20of%20genome%20size%20in%20the%20genus%20Oryza&amp;author=J.S.%20Ammiraju&amp;author=A.%20Zuccolo&amp;author=Y.%20Yu&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=52&amp;pages=342-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2007.52.issue-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2007.52.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2007.52.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17764506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morse</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Islam-Faridi</div> <div class="given-names">M.N.</div></div>, et al. </div><div class="article-title">Evolution of genome size and complexity in <em>Pinus</em></div>, <div class="source">PLoS ONE</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e4332</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20genome%20size%20and%20complexity%20in%20Pinus&amp;author=A.M.%20Morse&amp;author=D.G.%20Peterson&amp;author=M.N.%20Islam-Faridi&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e4332" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0004332" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0004332" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0004332"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19194510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cavallini</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Natali</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zuccolo</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Analysis of transposons and repeat composition of the sunflower (<em>Helianthus annuus</em> L.) genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">491</div>-<div class="lpage">508</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20transposons%20and%20repeat%20composition%20of%20the%20sunflower%20(Helianthus%20annuus%20L.)%20genome&amp;author=A.%20Cavallini&amp;author=L.%20Natali&amp;author=A.%20Zuccolo&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=491-508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1170-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1170-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1170-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19826774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Natali</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cossu</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Barghini</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The repetitive component of the sunflower genome as revealed by different procedures for assembling next generation sequencing reads</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">686</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20repetitive%20component%20of%20the%20sunflower%20genome%20as%20revealed%20by%20different%20procedures%20for%20assembling%20next%20generation%20sequencing%20reads&amp;author=L.%20Natali&amp;author=R.M.%20Cossu&amp;author=E.%20Barghini&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=686" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-686" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-686" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-686"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24093210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zuccolo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sebastian</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Assessing the extent of substitution rate variation of retrotransposon long terminal repeat sequences in <em>Oryza sativa</em> and <em>Oryza glaberrima</em></div>, <div class="source">Rice</div> , <div class="year">2010</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">242</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20extent%20of%20substitution%20rate%20variation%20of%20retrotransposon%20long%20terminal%20repeat%20sequences%20in%20Oryza%20sativa%20and%20Oryza%20glaberrima&amp;author=A.%20Zuccolo&amp;author=A.%20Sebastian&amp;author=Y.%20Yu&amp;publication_year=2010&amp;journal=Rice&amp;volume=3&amp;pages=242-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12284-010-9046-7" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12284-010-9046-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12284-010-9046-7"> </span></div> </div></div></div></div></div><div content-id="dsu042c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ungerer</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Strakosh</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Stimpson</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Proliferation of Ty3/<em>gypsy</em>-like retrotransposons in hybrid sunflower taxa inferred from phylogenetic data</div>, <div class="source">BMC Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> pg. <div class="fpage">40</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proliferation%20of%20Ty3%2Fgypsy-like%20retrotransposons%20in%20hybrid%20sunflower%20taxa%20inferred%20from%20phylogenetic%20data&amp;author=M.C.%20Ungerer&amp;author=S.C.%20Strakosh&amp;author=K.M.%20Stimpson&amp;publication_year=2009&amp;journal=BMC%20Biol.&amp;volume=7&amp;pages=40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1741-7007-7-40" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1741-7007-7-40" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1741-7007-7-40"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19594956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vukich</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schulman</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Giordani</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genetic variability in sunﬂower (<em>Helianthus annuus</em> L.) and in the <em>Helianthus</em> genus as assessed by retrotransposon-based molecular markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">1027</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variability%20in%20sun%EF%AC%82ower%20(Helianthus%20annuus%20L.)%20and%20in%20the%20Helianthus%20genus%20as%20assessed%20by%20retrotransposon-based%20molecular%20markers&amp;author=M.%20Vukich&amp;author=A.H.%20Schulman&amp;author=T.%20Giordani&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=119&amp;pages=1027-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1106-2" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1106-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1106-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19618160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baucom</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Estill</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Leebens-Mack</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Natural selection on gene function drives the evolution of LTR retrotransposon families in the rice genome</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">243</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20selection%20on%20gene%20function%20drives%20the%20evolution%20of%20LTR%20retrotransposon%20families%20in%20the%20rice%20genome&amp;author=R.S.%20Baucom&amp;author=J.C.%20Estill&amp;author=J.%20Leebens-Mack&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=243-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.083360.108" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.083360.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.083360.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lisch</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Epigenetic regulation of transposable elements in plants</div>, <div class="source">Ann. Rev. Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">43</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20transposable%20elements%20in%20plants&amp;author=D.%20Lisch&amp;publication_year=2009&amp;journal=Ann.%20Rev.%20Plant%20Biol.&amp;volume=60&amp;pages=43-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.59.032607.092744" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.59.032607.092744" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.59.032607.092744"> </span></div> </div></div></div></div></div><div content-id="dsu042c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baucom</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Estill</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Chaparro</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Exceptional diversity, non-random distribution, and rapid evolution of retroelements in the B73 maize genome</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000732</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exceptional%20diversity%2C%20non-random%20distribution%2C%20and%20rapid%20evolution%20of%20retroelements%20in%20the%20B73%20maize%20genome&amp;author=R.S.%20Baucom&amp;author=J.C.%20Estill&amp;author=C.%20Chaparro&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000732" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000732" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19936065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le Rouzic</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dupas</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Capy</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Genome ecosystem and transposable elements species</div>, <div class="source">Gene</div> , <div class="year">2007</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">214</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20ecosystem%20and%20transposable%20elements%20species&amp;author=A.%20Le%20Rouzic&amp;author=S.%20Dupas&amp;author=P.%20Capy&amp;publication_year=2007&amp;journal=Gene&amp;volume=390&amp;pages=214-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.09.023" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.09.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.09.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17188821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu042c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu042c55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Biemont</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Dynamics of transposable elements: towards a community ecology of the genome</div>, <div class="source">Trends Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">317</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamics%20of%20transposable%20elements%3A%20towards%20a%20community%20ecology%20of%20the%20genome&amp;author=S.%20Venner&amp;author=C.%20Feschotte&amp;author=C.%20Biemont&amp;publication_year=2009&amp;journal=Trends%20Genet.&amp;volume=25&amp;pages=317-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2009.05.003" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2009.05.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2009.05.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19540613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=441732&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu042-dsu042supp_table1.xlsx?Expires=1509238985&amp;Signature=Kk7mA1rwbJaBIzki~N6aKgVyEcI5srIdn7CwCRZfYJRtukJyYF3a8smS6sJ-V8HpO3MJNWaP9Ah8KyUlCiS0KCCyfQhGbCXaGW73lZxr-Q~IXXgRVfl5TrfJviOWpV08SNWgb1xrrFp9KBLZfBaVWZW~4ecCtMk5zovytMY6pk~axxvHIKEu2n6N2MaQAYIUNYdOUXZDYDNNaB4K8fg30Z-Gc3oKljGHZ8XpdNtES74ye8RgFxAnWYn10hI751mdrjXJ1XfK86DzvwHlUeCxznuKLERlJAlYKpJLYLnhEMyofgpL0~6qdbV1TLBNav4aK67RCwhYIHXAzv2qUctvjQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu042/2/dsu042-dsu042supp.doc?Expires=1509238985&amp;Signature=Xq8TzYQ9CS7IUR1olz8m1JVe-55IGszDi9tFxb7cU5D-oVvCNdZR~I16XokeIwAzqec3~9priJ7Xyc2rU5EqCGE5yCbQbFaGrYVNKsjqY7EcgWR0WUpjh2jwU9Vb63-w5CdyzjUjU9OQ8Wqoz5IHy4JWkSum71KPRSemfhlnr4~con0BII2~dYOH1JdgPp6Bd4iIpQuWF2QdyKKCw-6hbBBjYdgVx5kTnOhKql4e5CNnm4UOoNyTbZMkYZRji3eKAK5vIE7UqcAf219xiEX02W~emrMaIODq44DNfuO4oq4GAjJLKXHxYPii10nH8CdnN0Vo4JUi85JQcVEFeL9dOw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">65</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsu042"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="441732"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f91%2f441732">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F91%2F441732" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/441732'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="LTR retrotransposon dynamics in the evolution of the olive (Olea europaea) genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f91%2f441732">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

