

<!DOCTYPE html>
<html lang="en">

<head><title>
	Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/109/442245","@type":"ScholarlyArticle","name":"Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression","datePublished":"2015-01-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu044","keywords":["epigenetics","genome tracks","gene expression","transcription start sites","bioinformatic tool"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/109/442245","author":[{"name":"Kravatsky, Yuri V.","affiliation":"Engelhardt Institute of Molecular Biology of Russian Academy of Sciences ,  Moscow 119991 ,  Russia","@type":"Person"},{"name":"Chechetkin, Vladimir R.","affiliation":"Engelhardt Institute of Molecular Biology of Russian Academy of Sciences ,  Moscow 119991 ,  Russia","@type":"Person"},{"name":"Tchurikov, Nikolai A.","affiliation":"Engelhardt Institute of Molecular Biology of Russian Academy of Sciences ,  Moscow 119991 ,  Russia","@type":"Person"},{"name":"Kravatskaya, Galina I.","affiliation":"Engelhardt Institute of Molecular Biology of Russian Academy of Sciences ,  Moscow 119991 ,  Russia","@type":"Person"}],"description":"The broad class of tasks in genetics and epigenetics can be reduced to the study of various features that are distributed over the genome (genome tracks). The rapid and efficient processing of the huge amount of data stored in the genome-scale databases cannot be achieved without the software packages based on the analytical criteria. However, strong inhomogeneity of genome tracks hampers the development of relevant statistics. We developed the criteria for the assessment of genome track inhomogeneity and correlations between two genome tracks. We also developed a software package, Genome Track Analyzer, based on this theory. The theory and software were tested on simulated data and were applied to the study of correlations between CpG islands and transcription start sites in the Homo sapiens genome, between profiles of protein-binding sites in chromosomes of Drosophila melanogaster, and between DNA double-strand breaks and histone marks in the H. sapiens genome. Significant correlations between transcription start sites on the forward and the reverse strands were observed in genomes of D. melanogaster, Caenorhabditis elegans, Mus musculus, H. sapiens, and Danio rerio. The observed correlations may be related to the regulation of gene expression in eukaryotes. Genome Track Analyzer is freely available at http://ancorr.eimb.ru/.","pageStart":"109","pageEnd":"119","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217388&Signature=bCczNNqcQrJIIjzBIAuaBSQf2AT2hhGXrP0PZSXtevhX09YrxunaK1aoSfXUhOmVqU3TynQeyCnEGddEXLAn0~k5jNa4wSIbO3NgS7Bvc7qbgwSCZd4HnrMtCR5R7rjvkxM5SxS-LvYdRQsgX~8UlGoy3DdQIPOgICoanm43XnUj~Thp8MrpVBj6dZpHJr0vyKeZbCbbGl29HAKzSy354fglLPyqHGf99gBQ5R1dYhvF3fuGc0gjLw3aQW2YyE1aMCGafHarbmS~YiaH9PhTxT54EX9X8ZlrAugmyRuUdL8TFSGnZzSD6BYH4IpUBteHoZiDFblor3byPm1v2zqr8g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1572217388&Signature=bCczNNqcQrJIIjzBIAuaBSQf2AT2hhGXrP0PZSXtevhX09YrxunaK1aoSfXUhOmVqU3TynQeyCnEGddEXLAn0~k5jNa4wSIbO3NgS7Bvc7qbgwSCZd4HnrMtCR5R7rjvkxM5SxS-LvYdRQsgX~8UlGoy3DdQIPOgICoanm43XnUj~Thp8MrpVBj6dZpHJr0vyKeZbCbbGl29HAKzSy354fglLPyqHGf99gBQ5R1dYhvF3fuGc0gjLw3aQW2YyE1aMCGafHarbmS~YiaH9PhTxT54EX9X8ZlrAugmyRuUdL8TFSGnZzSD6BYH4IpUBteHoZiDFblor3byPm1v2zqr8g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression | DNA Research | Oxford Academic" />
    <meta property="og:description" content="The broad class of tasks in genetics and epigenetics can be reduced to the study of various features that are distributed over the genome (genome tracks). The rapid and efficient processing of the huge amount of data stored in the genome-scale databases cannot be achieved without the software packages based on the analytical criteria. However, strong inhomogeneity of genome tracks hampers the development of relevant statistics. We developed the criteria for the assessment of genome track inhomogeneity and correlations between two genome tracks. We also developed a software package, Genome Track Analyzer, based on this theory. The theory and software were tested on simulated data and were applied to the study of correlations between CpG islands and transcription start sites in the Homo sapiens genome, between profiles of protein-binding sites in chromosomes of Drosophila melanogaster, and between DNA double-strand breaks and histone marks in the H. sapiens genome. Significant correlations between transcription start sites on the forward and the reverse strands were observed in genomes of D. melanogaster, Caenorhabditis elegans, Mus musculus, H. sapiens, and Danio rerio. The observed correlations may be related to the regulation of gene expression in eukaryotes. Genome Track Analyzer is freely available at http://ancorr.eimb.ru/." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/109/442245" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217388&amp;Signature=B-4OZYyhjvaS27fDuDyw7X9JjMY3Un4a-fNaf5Hz0k6nCywiIFjECteVCTAwjH3rwDkp-Xrhdv8JMR~juCtHI-~W3zJfGk6qucintc8fN6DkyWs~RpmqQTJYL~bguEKhZ0ZtG2jYPoR8GhgVmr34Xia4rJQfcWcF8beuQ0lbb417soVuTDphtSHbnp4S7E4s86JAOw114NFm6VVQl9YHyn0dbjCJc83ikmGwyXJTzlQ7XYAQp5uMBRPVYe6HFqiPaNvk4yJ~2PviCcUJ4bh49T0rNMBulbG7MVyKd~0SoZPce9D7l9dfuwTtpbcjXl4XxQM1dxHwkuA9o1ZYnxY5iA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2015-01-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kravatsky, Yuri V." /><meta name="citation_author_institution" content="Engelhardt Institute of Molecular Biology of Russian Academy of Sciences, Moscow 119991, Russia" /><meta name="citation_author" content="Chechetkin, Vladimir R." /><meta name="citation_author_institution" content="Engelhardt Institute of Molecular Biology of Russian Academy of Sciences, Moscow 119991, Russia" /><meta name="citation_author" content="Tchurikov, Nikolai A." /><meta name="citation_author_institution" content="Engelhardt Institute of Molecular Biology of Russian Academy of Sciences, Moscow 119991, Russia" /><meta name="citation_author" content="Kravatskaya, Galina I." /><meta name="citation_author_institution" content="Engelhardt Institute of Molecular Biology of Russian Academy of Sciences, Moscow 119991, Russia" /><meta name="citation_title" content="Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression" /><meta name="citation_firstpage" content="109" /><meta name="citation_lastpage" content="119" /><meta name="citation_doi" content="10.1093/dnares/dsu044" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The NIH Roadmap Epigenomics Mapping Consortium; Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=1045-8; " /><meta name="citation_reference" content="citation_title=The ENCODE project and perspectives on pathways; Genet. Epidemiol.;  citation_year=2014;  citation_volume=38;  citation_pages=275-80; " /><meta name="citation_reference" content="citation_title=EPD and EPDnew, high-quality promoter resources in the next-generation sequencing era; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D157-64; " /><meta name="citation_reference" content="citation_title=DBTSS: Database of Transcriptional Start Sites progress report in 2012; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D150-4; " /><meta name="citation_reference" content="citation_title=Redefining CpG islands using hidden Markov models; Biostatistics;  citation_year=2010;  citation_volume=11;  citation_pages=499-514; " /><meta name="citation_reference" content="citation_title=A promoter-level mammalian expression atlas; citation_author=FANTOM Consortium, R.P., CLST; Nature;  citation_year=2014;  citation_volume=507;  citation_pages=462-70; " /><meta name="citation_reference" content="citation_title=Chapter 11: genome-wide association studies; PLoS Comput. Biol.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002822" /><meta name="citation_reference" content="citation_publisher=Springer, New York; An introduction to the theory of point processes;  citation_year=2003; " /><meta name="citation_reference" content="citation_publisher=Birkhäuser, Boston, MA; Point process theory and applications: marked point and piecewise deterministic processes;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=An overview of recent developments in genomics and associated statistical methods; Philos. Trans. A Math. Phys. Eng. Sci.;  citation_year=2009;  citation_volume=367;  citation_pages=4313-37; " /><meta name="citation_reference" content="citation_title=Statistical analysis of the genomic distribution and correlation of regulatory elements in the ENCODE regions; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=787-97; " /><meta name="citation_reference" content="citation_title=The Genomic HyperBrowser: inferential genomics at the sequence level; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R121" /><meta name="citation_reference" content="citation_title=The Genomic HyperBrowser: an analysis web server for genome-scale data; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=W133-41; " /><meta name="citation_reference" content="citation_title=Exploring massive, genome scale datasets with the GenometriCorr package; PLoS Comput. Biol.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002529" /><meta name="citation_reference" content="citation_publisher=Wiley, New York; An introduction to probability theory and its applications;  citation_year=1967; " /><meta name="citation_reference" content="citation_title=Spectral sum rules and search for periodicities in DNA sequences; Phys. Lett. A;  citation_year=2011;  citation_volume=375;  citation_pages=1729-32; " /><meta name="citation_reference" content="citation_title=Statistics of genome architecture; Phys. Lett. A;  citation_year=2013;  citation_volume=377;  citation_pages=3312-6; " /><meta name="citation_reference" content="citation_title=Computation for ChIP-seq and RNA-seq studies; Nat. Meth.;  citation_year=2009;  citation_volume=6;  citation_pages=S22-32; " /><meta name="citation_reference" content="citation_title=Evaluation of algorithm performance in ChIP-seq peak detection; PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e11471" /><meta name="citation_reference" content="citation_title=Shape-based peak identification for ChIP-Seq; BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=15" /><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=R137" /><meta name="citation_reference" content="citation_title=CpG islands – ‘a rough guide; FEBS Lett.;  citation_year=2009;  citation_volume=583;  citation_pages=1713-20; " /><meta name="citation_reference" content="citation_title=Human-specific CpG ‘beacons’ identify loci associated with human-specific traits and disease; Epigenetics;  citation_year=2012;  citation_volume=7;  citation_pages=1188-99; " /><meta name="citation_reference" content="citation_title=CpG domains downstream of TSSs promote high levels of gene expression; Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=3551-64; " /><meta name="citation_reference" content="citation_title=Evidence for RNA synthesis in the intergenic region between enhancer and promoter and its inhibition by insulators in Drosophila melanogaster; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=111-22; " /><meta name="citation_reference" content="citation_title=Widespread transcription at neuronal activity-regulated enhancers; Nature;  citation_year=2010;  citation_volume=465;  citation_pages=182-7; " /><meta name="citation_reference" content="citation_title=How bidirectional becomes unidirectional; Nat. Struct. Mol. Biol.;  citation_year=2013;  citation_volume=20;  citation_pages=1022-4; " /><meta name="citation_reference" content="citation_title=The Integrated Genome Browser: free software for distribution and exploration of genome-scale datasets; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2730-1; " /><meta name="citation_reference" content="citation_title=The functions of E(Z)/EZH2-mediated methylation of lysine 27 in histone H3; Curr. Opin. Genet. Dev.;  citation_year=2004;  citation_volume=14;  citation_pages=155-64; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of polycomb targets in Drosophila melanogaster; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=700-5; " /><meta name="citation_reference" content="citation_title=Stability and dynamics of polycomb target sites in Drosophila development; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000178" /><meta name="citation_reference" content="citation_title=Architectural and functional diversity of polycomb group response elements in Drosophila; Genetics;  citation_year=2013;  citation_volume=195;  citation_pages=407-19; " /><meta name="citation_reference" content="citation_title=ArrayExpress update – trends in database growth and links to data analysis tools; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D987-90; " /><meta name="citation_reference" content="citation_title=Cooperativity, specificity, and evolutionary stability of polycomb targeting in Drosophila; Cell Rep.;  citation_year=2014;  citation_volume=9;  citation_pages=219-33; " /><meta name="citation_reference" content="citation_title=Genome-wide profiling of forum domains in Drosophila melanogaster; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=3667-85; " /><meta name="citation_reference" content="citation_title=DNA double-strand breaks coupled with PARP1 and HNRNPA2B1 binding sites flank coordinately expressed domains in human chromosomes; PLoS Genet.;  citation_year=2013;  citation_volume=9;  citation_pages=e1003429" /><meta name="citation_reference" content="citation_title=Nucleotide-resolution DNA double-strand break mapping by next-generation sequencing; Nat. Meth.;  citation_year=2013;  citation_volume=10;  citation_pages=361-5; " /><meta name="citation_reference" content="citation_title=H3K4me3 interactions with TAF3 regulate preinitiation complex assembly and selective gene activation; Cell;  citation_year=2013;  citation_volume=152;  citation_pages=1021-36; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/109/1212378/dsu044.pdf" /><meta name="description" content="Abstract.  The broad class of tasks in genetics and epigenetics can be reduced to the study of various features that are distributed over the genome (genome tra" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/109/442245" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/109/442245" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsu044';
    </script>
</head>

<script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1875,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,n,e){function r(e){if(!n[e]){var o=n[e]={exports:{}};t[e][0].call(o.exports,function(n){var o=t[e][1][n];return r(o||n)},o,o.exports)}return n[e].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<e.length;o++)r(e[o]);return r}({1:[function(t,n,e){function r(t){try{s.console&&console.log(t)}catch(n){}}var o,i=t("ee"),a=t(15),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,n,e){r(e.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,n){return t}).join(", ")))},{}],2:[function(t,n,e){function r(t,n,e,r,o){try{d?d-=1:i("err",[o||new UncaughtException(t,n,e)])}catch(s){try{i("ierr",[s,c.now(),!0])}catch(u){}}return"function"==typeof f&&f.apply(this,a(arguments))}function UncaughtException(t,n,e){this.message=t||"Uncaught error with no additional information",this.sourceURL=n,this.line=e}function o(t){i("err",[t,c.now()])}var i=t("handle"),a=t(16),s=t("ee"),c=t("loader"),f=window.onerror,u=!1,d=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(l){"stack"in l&&(t(8),t(7),"addEventListener"in window&&t(5),c.xhrWrappable&&t(9),u=!0)}s.on("fn-start",function(t,n,e){u&&(d+=1)}),s.on("fn-err",function(t,n,e){u&&(this.thrown=!0,o(e))}),s.on("fn-end",function(){u&&!this.thrown&&d>0&&(d-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,n,e){t("loader").features.ins=!0},{}],4:[function(t,n,e){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(8),s=t(7),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",l="resource",p="-start",h="-end",m="fn"+p,w="fn"+h,v="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(6);var b=NREUM.o.EV;o.on(m,function(t,n){var e=t[0];e instanceof b&&(this.bstStart=g.now())}),o.on(w,function(t,n){var e=t[0];e instanceof b&&i("bst",[e,n,this.bstStart,g.now()])}),a.on(m,function(t,n,e){this.bstStart=g.now(),this.bstType=e}),a.on(w,function(t,n){i(v,[n,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(w,function(t,n){i(v,[n,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+p,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],5:[function(t,n,e){function r(t){for(var n=t;n&&!n.hasOwnProperty(u);)n=Object.getPrototypeOf(n);n&&o(n)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,n){return t[1]}var a=t("ee").get("events"),s=t(18)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";n.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,n){var e=t[1],r=c(e,"nr@wrapped",function(){function t(){if("function"==typeof e.handleEvent)return e.handleEvent.apply(e,arguments)}var n={object:t,"function":e}[typeof e];return n?s(n,"fn-",null,n.name||"anonymous"):e});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],6:[function(t,n,e){var r=t("ee").get("history"),o=t(18)(r);n.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],7:[function(t,n,e){var r=t("ee").get("raf"),o=t(18)(r),i="equestAnimationFrame";n.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],8:[function(t,n,e){function r(t,n,e){t[0]=a(t[0],"fn-",null,e)}function o(t,n,e){this.method=e,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,e)}var i=t("ee").get("timer"),a=t(18)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";n.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],9:[function(t,n,e){function r(t,n){d.inPlace(n,["onreadystatechange"],"fn-",s)}function o(){var t=this,n=u.context(t);t.readyState>3&&!n.resolved&&(n.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):w?w(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,n){return n}function c(t,n){for(var e in t)n[e]=t[e];return n}t(5);var f=t("ee"),u=f.get("xhr"),d=t(18)(u),l=NREUM.o,p=l.XHR,h=l.MO,m=l.PR,w=l.SI,v="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];n.exports=u;var b=window.XMLHttpRequest=function(t){var n=new p(t);try{u.emit("new-xhr",[n],n),n.addEventListener(v,o,!1)}catch(e){try{u.emit("internal-error",[e])}catch(r){}}return n};if(c(p,b),b.prototype=p.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,n){r(t,n),i(n)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!w&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===v||a()})},{}],10:[function(t,n,e){function r(t){var n=this.params,e=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!n.aborted){if(e.duration=a.now()-this.startTime,4===t.readyState){n.status=t.status;var i=o(t,this.lastSize);if(i&&(e.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(n.cat=c.split(", ").pop())}}else n.status=0;e.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[n,e,this.startTime])}}}function o(t,n){var e=t.responseType;if("json"===e&&null!==n)return n;var r="arraybuffer"===e||"blob"===e||"json"===e?t.response:t.responseText;return h(r)}function i(t,n){var e=c(n),r=t.params;r.host=e.hostname+":"+e.port,r.pathname=e.pathname,t.sameOrigin=e.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(11),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,l=t("id"),p=t(14),h=t(13),m=window.XMLHttpRequest;a.features.xhr=!0,t(9),f.on("new-xhr",function(t){var n=this;n.totalCbs=0,n.called=0,n.cbTime=0,n.end=r,n.ended=!1,n.xhrGuids={},n.lastSize=null,p&&(p>34||p<10)||window.opera||t.addEventListener("progress",function(t){n.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,n){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,n){var e=this.metrics,r=t[0],o=this;if(e&&r){var i=h(r);i&&(e.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof n.onload))&&o.end(n)}catch(e){try{f.emit("internal-error",[e])}catch(r){}}};for(var s=0;s<d;s++)n.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,n,e){this.cbTime+=t,n?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof e.onload||this.end(e)}),f.on("xhr-load-added",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&!this.xhrGuids[e]&&(this.xhrGuids[e]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&this.xhrGuids[e]&&(delete this.xhrGuids[e],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],n)}),f.on("removeEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],n)}),f.on("fn-start",function(t,n,e){n instanceof m&&("onload"===e&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,n){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,n],n)})}},{}],11:[function(t,n,e){n.exports=function(t){var n=document.createElement("a"),e=window.location,r={};n.href=t,r.port=n.port;var o=n.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=n.hostname||e.hostname,r.pathname=n.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!n.protocol||":"===n.protocol||n.protocol===e.protocol,a=n.hostname===document.domain&&n.port===e.port;return r.sameOrigin=i&&(!n.hostname||a),r}},{}],12:[function(t,n,e){function r(){}function o(t,n,e){return function(){return i(t,[f.now()].concat(s(arguments)),n?null:this,e),n?void 0:this}}var i=t("handle"),a=t(15),s=t(16),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",p=l+"ixn-";a(d,function(t,n){u[n]=o(l+n,!0,"api")}),u.addPageAction=o(l+"addPageAction",!0),u.setCurrentRouteName=o(l+"routeName",!0),n.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,n){var e={},r=this,o="function"==typeof n;return i(p+"tracer",[f.now(),t,e],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],e),o)try{return n.apply(this,arguments)}finally{c.emit("fn-end",[f.now()],e)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,n){h[n]=o(p+n)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],13:[function(t,n,e){n.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(n){return}}}},{}],14:[function(t,n,e){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),n.exports=r},{}],15:[function(t,n,e){function r(t,n){var e=[],r="",i=0;for(r in t)o.call(t,r)&&(e[i]=n(r,t[r]),i+=1);return e}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],16:[function(t,n,e){function r(t,n,e){n||(n=0),"undefined"==typeof e&&(e=t?t.length:0);for(var r=-1,o=e-n||0,i=Array(o<0?0:o);++r<o;)i[r]=t[n+r];return i}n.exports=r},{}],17:[function(t,n,e){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],18:[function(t,n,e){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(16),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;n.exports=function(t,n){function e(t,n,e,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof e?e(r,a):e||{}}catch(f){l([f,"",[r,a,o],s])}u(n+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(n+"err",[r,a,d],s),d}finally{u(n+"end",[r,a,c],s)}}return r(t)?t:(n||(n=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,n,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<n.length;c++)s=n[c],a=t[s],r(a)||(t[s]=e(a,f?s+o:o,i,s))}function u(e,r,o){if(!c||n){var i=c;c=!0;try{t.emit(e,r,o,n)}catch(a){l([a,e,r,o])}c=i}}function d(t,n){if(Object.defineProperty&&Object.keys)try{var e=Object.keys(t);return e.forEach(function(e){Object.defineProperty(n,e,{get:function(){return t[e]},set:function(n){return t[e]=n,n}})}),n}catch(r){l([r])}for(var o in t)s.call(t,o)&&(n[o]=t[o]);return n}function l(n){try{t.emit("internal-error",n)}catch(e){}}return t||(t=o),e.inPlace=f,e.flag=a,e}},{}],ee:[function(t,n,e){function r(){}function o(t){function n(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function e(e,r,o,i){if(!l.aborted||i){t&&t(e,r,o);for(var a=n(o),s=h(e),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[y[e]];return d&&d.push([g,e,r,a]),a}}function p(t,n){v[t]=h(t).concat(n)}function h(t){return v[t]||[]}function m(t){return d[t]=d[t]||o(e)}function w(t,n){f(t,function(t,e){n=n||"feature",y[e]=n,n in u||(u[n]=[])})}var v={},y={},g={on:p,emit:e,get:m,listeners:h,context:n,buffer:w,abort:a,aborted:!1};return g}function i(){return new r}function a(){(u.api||u.feature)&&(l.aborted=!0,u=l.backlog={})}var s="nr@context",c=t("gos"),f=t(15),u={},d={},l=n.exports=o();l.backlog=u},{}],gos:[function(t,n,e){function r(t,n,e){if(o.call(t,n))return t[n];var r=e();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(t,n,e){function r(t,n,e,r){o.buffer([t],r),o.emit(t,n,e)}var o=t("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(t,n,e){function r(t){var n=typeof t;return!t||"object"!==n&&"function"!==n?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");n.exports=r},{}],loader:[function(t,n,e){function r(){if(!x++){var t=b.info=NREUM.info,n=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&n))return u.abort();f(y,function(n,e){t[n]||(t[n]=e)}),c("mark",["onload",a()+b.offset],null,"api");var e=l.createElement("script");e.src="https://"+t.agent,n.parentNode.insertBefore(e,n)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+b.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-b.offset}var s=(new Date).getTime(),c=t("handle"),f=t(15),u=t("ee"),d=window,l=d.document,p="addEventListener",h="attachEvent",m=d.XMLHttpRequest,w=m&&m.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:m,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},g=m&&w&&w[p]&&!/CriOS/.test(navigator.userAgent),b=n.exports={offset:s,now:a,origin:v,features:{},xhrWrappable:g};t(12),l[p]?(l[p]("DOMContentLoaded",i,!1),d[p]("load",r,!1)):(l[h]("onreadystatechange",o),d[h]("onload",r)),c("mark",["firstbyte",s],null,"api");var x=0,E=t(17)},{}]},{},["loader",2,10,4,3]);</script><body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./442245" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFiU48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktyYXZhdHNreSwgWXVyaSBWLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVuZ2VsaGFyZHQgSW5zdGl0dXRlIG9mIE1vbGVjdWxhciBCaW9sb2d5IG9mIFJ1c3NpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgTW9zY293IDExOTk5MSwgUnVzc2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlY2hldGtpbiwgVmxhZGltaXIgUi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFbmdlbGhhcmR0IEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgQmlvbG9neSBvZiBSdXNzaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIE1vc2NvdyAxMTk5OTEsIFJ1c3NpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRjaHVyaWtvdiwgTmlrb2xhaSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVuZ2VsaGFyZHQgSW5zdGl0dXRlIG9mIE1vbGVjdWxhciBCaW9sb2d5IG9mIFJ1c3NpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgTW9zY293IDExOTk5MSwgUnVzc2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3JhdmF0c2theWEsIEdhbGluYSBJLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVuZ2VsaGFyZHQgSW5zdGl0dXRlIG9mIE1vbGVjdWxhciBCaW9sb2d5IG9mIFJ1c3NpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgTW9zY293IDExOTk5MSwgUnVzc2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtd2lkZSBzdHVkeSBvZiBjb3JyZWxhdGlvbnMgYmV0d2VlbiBnZW5vbWljIGZlYXR1cmVzIGFuZCB0aGVpciByZWxhdGlvbnNoaXAgd2l0aCB0aGUgcmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjExOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTA0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE5JSCBSb2FkbWFwIEVwaWdlbm9taWNzIE1hcHBpbmcgQ29uc29ydGl1bTsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTEwNDUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVOQ09ERSBwcm9qZWN0IGFuZCBwZXJzcGVjdGl2ZXMgb24gcGF0aHdheXM7IEdlbmV0LiBFcGlkZW1pb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9Mjc1LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FUEQgYW5kIEVQRG5ldywgaGlnaC1xdWFsaXR5IHByb21vdGVyIHJlc291cmNlcyBpbiB0aGUgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgZXJhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1EMTU3LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EQlRTUzogRGF0YWJhc2Ugb2YgVHJhbnNjcmlwdGlvbmFsIFN0YXJ0IFNpdGVzIHByb2dyZXNzIHJlcG9ydCBpbiAyMDEyOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMTUwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZGVmaW5pbmcgQ3BHIGlzbGFuZHMgdXNpbmcgaGlkZGVuIE1hcmtvdiBtb2RlbHM7IEJpb3N0YXRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00OTktNTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb21vdGVyLWxldmVsIG1hbW1hbGlhbiBleHByZXNzaW9uIGF0bGFzOyBjaXRhdGlvbl9hdXRob3I9RkFOVE9NIENvbnNvcnRpdW0sIFIuUC4sIENMU1Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MDc7ICBjaXRhdGlvbl9wYWdlcz00NjItNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXB0ZXIgMTE6IGdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXM7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI4MjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrOyBBbiBpbnRyb2R1Y3Rpb24gdG8gdGhlIHRoZW9yeSBvZiBwb2ludCBwcm9jZXNzZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1CaXJraMOkdXNlciwgQm9zdG9uLCBNQTsgUG9pbnQgcHJvY2VzcyB0aGVvcnkgYW5kIGFwcGxpY2F0aW9uczogbWFya2VkIHBvaW50IGFuZCBwaWVjZXdpc2UgZGV0ZXJtaW5pc3RpYyBwcm9jZXNzZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIG92ZXJ2aWV3IG9mIHJlY2VudCBkZXZlbG9wbWVudHMgaW4gZ2Vub21pY3MgYW5kIGFzc29jaWF0ZWQgc3RhdGlzdGljYWwgbWV0aG9kczsgUGhpbG9zLiBUcmFucy4gQSBNYXRoLiBQaHlzLiBFbmcuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzY3OyAgY2l0YXRpb25fcGFnZXM9NDMxMy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgYW5hbHlzaXMgb2YgdGhlIGdlbm9taWMgZGlzdHJpYnV0aW9uIGFuZCBjb3JyZWxhdGlvbiBvZiByZWd1bGF0b3J5IGVsZW1lbnRzIGluIHRoZSBFTkNPREUgcmVnaW9uczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz03ODctOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHZW5vbWljIEh5cGVyQnJvd3NlcjogaW5mZXJlbnRpYWwgZ2Vub21pY3MgYXQgdGhlIHNlcXVlbmNlIGxldmVsOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SMTIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbm9taWMgSHlwZXJCcm93c2VyOiBhbiBhbmFseXNpcyB3ZWIgc2VydmVyIGZvciBnZW5vbWUtc2NhbGUgZGF0YTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9VzEzMy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9yaW5nIG1hc3NpdmUsIGdlbm9tZSBzY2FsZSBkYXRhc2V0cyB3aXRoIHRoZSBHZW5vbWV0cmlDb3JyIHBhY2thZ2U7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI1MjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXksIE5ldyBZb3JrOyBBbiBpbnRyb2R1Y3Rpb24gdG8gcHJvYmFiaWxpdHkgdGhlb3J5IGFuZCBpdHMgYXBwbGljYXRpb25zOyAgY2l0YXRpb25feWVhcj0xOTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjdHJhbCBzdW0gcnVsZXMgYW5kIHNlYXJjaCBmb3IgcGVyaW9kaWNpdGllcyBpbiBETkEgc2VxdWVuY2VzOyBQaHlzLiBMZXR0LiBBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM3NTsgIGNpdGF0aW9uX3BhZ2VzPTE3MjktMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY3Mgb2YgZ2Vub21lIGFyY2hpdGVjdHVyZTsgUGh5cy4gTGV0dC4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNzc7ICBjaXRhdGlvbl9wYWdlcz0zMzEyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uIGZvciBDaElQLXNlcSBhbmQgUk5BLXNlcSBzdHVkaWVzOyBOYXQuIE1ldGguOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1TMjItMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2YgYWxnb3JpdGhtIHBlcmZvcm1hbmNlIGluIENoSVAtc2VxIHBlYWsgZGV0ZWN0aW9uOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTExNDcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hhcGUtYmFzZWQgcGVhayBpZGVudGlmaWNhdGlvbiBmb3IgQ2hJUC1TZXE7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZWwtYmFzZWQgYW5hbHlzaXMgb2YgQ2hJUC1TZXEgKE1BQ1MpOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPVIxMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DcEcgaXNsYW5kcyDigJMg4oCYYSByb3VnaCBndWlkZTsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01ODM7ICBjaXRhdGlvbl9wYWdlcz0xNzEzLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbi1zcGVjaWZpYyBDcEcg4oCYYmVhY29uc+KAmSBpZGVudGlmeSBsb2NpIGFzc29jaWF0ZWQgd2l0aCBodW1hbi1zcGVjaWZpYyB0cmFpdHMgYW5kIGRpc2Vhc2U7IEVwaWdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMTg4LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DcEcgZG9tYWlucyBkb3duc3RyZWFtIG9mIFRTU3MgcHJvbW90ZSBoaWdoIGxldmVscyBvZiBnZW5lIGV4cHJlc3Npb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTM1NTEtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBSTkEgc3ludGhlc2lzIGluIHRoZSBpbnRlcmdlbmljIHJlZ2lvbiBiZXR3ZWVuIGVuaGFuY2VyIGFuZCBwcm9tb3RlciBhbmQgaXRzIGluaGliaXRpb24gYnkgaW5zdWxhdG9ycyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIHRyYW5zY3JpcHRpb24gYXQgbmV1cm9uYWwgYWN0aXZpdHktcmVndWxhdGVkIGVuaGFuY2VyczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NTsgIGNpdGF0aW9uX3BhZ2VzPTE4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgYmlkaXJlY3Rpb25hbCBiZWNvbWVzIHVuaWRpcmVjdGlvbmFsOyBOYXQuIFN0cnVjdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMjItNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEludGVncmF0ZWQgR2Vub21lIEJyb3dzZXI6IGZyZWUgc29mdHdhcmUgZm9yIGRpc3RyaWJ1dGlvbiBhbmQgZXhwbG9yYXRpb24gb2YgZ2Vub21lLXNjYWxlIGRhdGFzZXRzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI3MzAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZ1bmN0aW9ucyBvZiBFKFopL0VaSDItbWVkaWF0ZWQgbWV0aHlsYXRpb24gb2YgbHlzaW5lIDI3IGluIGhpc3RvbmUgSDM7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBwb2x5Y29tYiB0YXJnZXRzIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTcwMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFiaWxpdHkgYW5kIGR5bmFtaWNzIG9mIHBvbHljb21iIHRhcmdldCBzaXRlcyBpbiBEcm9zb3BoaWxhIGRldmVsb3BtZW50OyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAxNzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmNoaXRlY3R1cmFsIGFuZCBmdW5jdGlvbmFsIGRpdmVyc2l0eSBvZiBwb2x5Y29tYiBncm91cCByZXNwb25zZSBlbGVtZW50cyBpbiBEcm9zb3BoaWxhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTU7ICBjaXRhdGlvbl9wYWdlcz00MDctMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFycmF5RXhwcmVzcyB1cGRhdGUg4oCTIHRyZW5kcyBpbiBkYXRhYmFzZSBncm93dGggYW5kIGxpbmtzIHRvIGRhdGEgYW5hbHlzaXMgdG9vbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQ5ODctOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvb3BlcmF0aXZpdHksIHNwZWNpZmljaXR5LCBhbmQgZXZvbHV0aW9uYXJ5IHN0YWJpbGl0eSBvZiBwb2x5Y29tYiB0YXJnZXRpbmcgaW4gRHJvc29waGlsYTsgQ2VsbCBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMTktMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHByb2ZpbGluZyBvZiBmb3J1bSBkb21haW5zIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0zNjY3LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgZG91YmxlLXN0cmFuZCBicmVha3MgY291cGxlZCB3aXRoIFBBUlAxIGFuZCBITlJOUEEyQjEgYmluZGluZyBzaXRlcyBmbGFuayBjb29yZGluYXRlbHkgZXhwcmVzc2VkIGRvbWFpbnMgaW4gaHVtYW4gY2hyb21vc29tZXM7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzQyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3RpZGUtcmVzb2x1dGlvbiBETkEgZG91YmxlLXN0cmFuZCBicmVhayBtYXBwaW5nIGJ5IG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyBOYXQuIE1ldGguOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MzYxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUgzSzRtZTMgaW50ZXJhY3Rpb25zIHdpdGggVEFGMyByZWd1bGF0ZSBwcmVpbml0aWF0aW9uIGNvbXBsZXggYXNzZW1ibHkgYW5kIHNlbGVjdGl2ZSBnZW5lIGFjdGl2YXRpb247IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9MTAyMS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzEvMTA5LzEyMTIzNzgvZHN1MDQ0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgYnJvYWQgY2xhc3Mgb2YgdGFza3MgaW4gZ2VuZXRpY3MgYW5kIGVwaWdlbmV0aWNzIGNhbiBiZSByZWR1Y2VkIHRvIHRoZSBzdHVkeSBvZiB2YXJpb3VzIGZlYXR1cmVzIHRoYXQgYXJlIGRpc3RyaWJ1dGVkIG92ZXIgdGhlIGdlbm9tZSAoZ2Vub21lIHRyYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQyMjQ1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgQWxsIEZpZ3VyZXNkZPKzKaMatKnMxhrAKv6xzsMlobvp" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f109%2f442245" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f109%2f442245" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=1509238968&amp;Signature=EKhDR45hE0JvCD7NNoA5tSJUkxjnj69koznjLkmJx9te8bPX36EV57k~dMqMhpf3UAbpL09FykjVTRUPi2x23uanqwItrhyn8sAZZUUv2dvBiH9l0WZ6cSlozq3SPquUUWmBRlnRTNgbm5fxFw0n91Aul7Ya3Vji8eKqL9EI7JcGCTEk-igrHqdfk7ijALyGRxwV4NYzfVaUgtvZktJK08~hfyTg-fajyf3XpnObLSqYcXdCqdrDxxVTZT5GCzwl3m4aQzhKLn~WfSIBiOsH7BtEqv~Worvo48Dql9DyPmT1BhyAaf69BFpFYf3wni-4Ob7kad7ees~OsADdgayfmw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6615554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615554">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615556">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615556">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615562">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615562">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615585">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615585">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615606">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615606">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615611">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615611">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615613">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615613">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615615">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615615">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6615617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6615617">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/101/442161">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Yuri V. Kravatsky</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuri V. Kravatsky</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Engelhardt Institute of Molecular Biology of Russian Academy of Sciences</div>, <div class="addr-line">Moscow 119991</div>, <div class="country">Russia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +7 499-135-2311. Fax. +7 499-135-1405. E-mail: <a href="mailto:jiri@eimb.ru" target="_blank">jiri@eimb.ru</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Yuri+V.+Kravatsky">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kravatsky Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kravatsky author:Y.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Vladimir R. Chechetkin</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vladimir R. Chechetkin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Engelhardt Institute of Molecular Biology of Russian Academy of Sciences</div>, <div class="addr-line">Moscow 119991</div>, <div class="country">Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Vladimir+R.+Chechetkin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chechetkin V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Chechetkin author:V.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Nikolai A. Tchurikov</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nikolai A. Tchurikov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Engelhardt Institute of Molecular Biology of Russian Academy of Sciences</div>, <div class="addr-line">Moscow 119991</div>, <div class="country">Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Nikolai+A.+Tchurikov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tchurikov N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Tchurikov author:N.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Galina I. Kravatskaya</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Galina I. Kravatskaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Engelhardt Institute of Molecular Biology of Russian Academy of Sciences</div>, <div class="addr-line">Moscow 119991</div>, <div class="country">Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Galina+I.+Kravatskaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kravatskaya G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Kravatskaya author:G.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 109–119, <a href='https://doi.org/10.1093/dnares/dsu044'>https://doi.org/10.1093/dnares/dsu044</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 January 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 June 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 November 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="442245" href="/dnaresearch/article-pdf/22/1/109/1212378/dsu044.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuri V. Kravatsky, Vladimir R. Chechetkin, Nikolai A. Tchurikov, Galina I. Kravatskaya;  Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 109–119, <a href='https://doi.org/10.1093/dnares/dsu044'>https://doi.org/10.1093/dnares/dsu044</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/442245?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/442245?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/442245?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/442245?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/442245?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/442245?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/442245?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20study%20of%20correlations%20between%20genomic%20features%20and%20their%20relationship%20with%20the%20regulation%20of%20gene%20expression&amp;publicationDate=2015-01-27&amp;volumeNum=22&amp;issueNum=1&amp;author=Kravatsky%2C%20Yuri%20V.%3B%20Chechetkin%2C%20Vladimir%20R.&amp;startPage=109&amp;endPage=119&amp;contentId=10.1093%2Fdnares%2Fdsu044&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f109%2f442245" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6615554 id="6615554" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The broad class of tasks in genetics and epigenetics can be reduced to the study of various features that are distributed over the genome (genome tracks). The rapid and efficient processing of the huge amount of data stored in the genome-scale databases cannot be achieved without the software packages based on the analytical criteria. However, strong inhomogeneity of genome tracks hampers the development of relevant statistics. We developed the criteria for the assessment of genome track inhomogeneity and correlations between two genome tracks. We also developed a software package, Genome Track Analyzer, based on this theory. The theory and software were tested on simulated data and were applied to the study of correlations between CpG islands and transcription start sites in the <em>Homo sapiens</em> genome, between profiles of protein-binding sites in chromosomes of <em>Drosophila melanogaster</em>, and between DNA double-strand breaks and histone marks in the <em>H. sapiens</em> genome. Significant correlations between transcription start sites on the forward and the reverse strands were observed in genomes of <em>D. melanogaster</em>, <em>Caenorhabditis elegans, Mus musculus</em>, <em>H. sapiens</em>, and <em>Danio rerio</em>. The observed correlations may be related to the regulation of gene expression in eukaryotes. Genome Track Analyzer is freely available at <a class="link link-uri" href="http://ancorr.eimb.ru/">http://ancorr.eimb.ru/</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetics">epigenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome tracks&quot;">genome tracks</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene expression&quot;">gene expression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription start sites&quot;">transcription start sites</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bioinformatic tool&quot;">bioinformatic tool</a></div>                            <h2 scrollto-destination=6615556 id="6615556" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Data mining of extensive genome-scale databases, such as the Human Epigenome Atlas,<sup><a reveal-id="DSU044C1" data-open="DSU044C1" class="link link-ref link-reveal xref-bibr">1</a></sup> ENCODE,<sup><a reveal-id="DSU044C2" data-open="DSU044C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Eukaryotic Promoter Database,<sup><a reveal-id="DSU044C3" data-open="DSU044C3" class="link link-ref link-reveal xref-bibr">3</a></sup> DBTSS,<sup><a reveal-id="DSU044C4" data-open="DSU044C4" class="link link-ref link-reveal xref-bibr">4</a></sup> database on CpG islands,<sup><a reveal-id="DSU044C5" data-open="DSU044C5" class="link link-ref link-reveal xref-bibr">5</a></sup> FANTOM5,<sup><a reveal-id="DSU044C6" data-open="DSU044C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and many others, cannot be performed without efficient bioinformatic tools. Typically, the researcher is interested in finding correlations between various characteristics distributed over the genome (commonly termed genome tracks). The trends in the close or remote positioning between genomic features may indicate their coordinated action in chromatin packing, recombination, replication, or transcription.<sup><a reveal-id="DSU044C7" data-open="DSU044C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The features on the genome can be presented as points [e.g. transcription start sites (TSS), DNA double-strand breaks (DSBs), contacts between chromosomes], stretches (transposons, exons and introns, CpG islands), or profiles defined throughout the sites (expression profiles, protein-binding profiles). The stretches can be described in terms of their lengths and positions of some characteristic points (starts/centres/ends). The effects related to these two parameters (overlapping and positioning of stretches) should be studied separately. A part of correlational analysis related to the mutual positioning of starts/centres/ends of stretches can also be reduced to the study of point distributions. The processing of profiles is rather complicated. The processing of data from several experiments and data on the background noise results in identifying the maxima in profiles or the significant regions distributed over the genome. Such maxima and regions can further be treated as points and stretches. Therefore, the general assessment of correlations between features of two types always includes an estimation of the closeness between two point sets.</p><p>Despite the seemingly simple formulation, such analysis remains a challenge.<sup><a reveal-id="DSU044C8  DSU044C10" data-open="DSU044C8  DSU044C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> The distribution of distances between consecutive genome track elements proved to be significantly different from that in the reference random model, with the variations in lengths between consecutive genomic elements being much stronger than the approximately homogeneous distribution of randomly positioned elements (see Results). In this sense, the distribution of genome track elements may be called inhomogeneous. The strong inhomogeneity in the distribution of genomic features over the genome, which is inherent to most genetic data, hampers statistical analysis. The application of simple and standardized methods, such as correlation functions, needs the coarse graining of data over 10–100 kb bins<sup><a reveal-id="DSU044C11" data-open="DSU044C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and smears the resolution of correlations. The proper approach should be based on the relevant statistical criteria. The developed analytical criteria and respective packages ought to be robust against the spatial inhomogeneity of feature distribution.</p><p>Among the available packages, the Genomic HyperBrowser<sup><a reveal-id="DSU044C12" data-open="DSU044C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU044C13" data-open="DSU044C13" class="link link-ref link-reveal xref-bibr">13</a></sup> is based on Monte Carlo (MC) simulations for the assessment of the statistical significance of correlations between genome tracks of two types, whereas GenometriCorr<sup><a reveal-id="DSU044C14" data-open="DSU044C14" class="link link-ref link-reveal xref-bibr">14</a></sup> uses the combination of statistically grounded analytical criteria with a permutation test. The application of simulations generally needs at least two rounds of simulations with an enhanced number of realizations to prove the statistical convergence of a method and is rather time consuming. The simulations are difficult to implement for the study of large genome-scale data sets. In contrast to packages based on the simulations, packages based on the analytical criteria (i.e. on the mathematical expressions derived in advance) do not need a large number of trial realizations for assessment of statistical significance and, therefore, save a lot of computational time.</p><p>Primarily, we were interested in the applications of available packages to the particular correlations between DSBs and epigenetic features. We began by testing available packages. In a random independent set of points, the position of each point does not depend on the positions of other random points or on the positions of points of any other (random or not) set. Thus, if at least one of two sets is random and independent, correlations between the two sets should be absent. Testing of both packages, Genomic HyperBrowser and GenometriCorr, for the absence of correlations between a random independent set and a strongly inhomogeneous, non-random genomic set revealed that the packages may indicate artificial correlations in this case. Moreover, as both packages do not suggest a measure for assessment of intrinsic inhomogeneity, the attribution between a random set and a non-random inhomogeneous set is not known <em>a priori</em>. If the true attribution was permuted, the correct assessment of correlations actually failed in both packages. The failure is perhaps related to problems in statistical assessment. Note that our remarks are concerned only with the particular options in the Genomic HyperBrowser and GenometriCorr corresponding to the study of correlations between two point sets. Therefore, we were prompted to develop an original statistical algorithm and package, the Genome Track Analyzer, based on this algorithm. Our package is freely available at <a class="link link-uri" href="http://ancorr.eimb.ru/">http://ancorr.eimb.ru/</a>. The usage of the analytical criteria permits the rapid processing of large-scale data. We tried to formulate our resulting analytical criteria in terms of Gaussian <em>z</em>-statistics that is familiar to most researchers.</p><p>The testing of available packages and the study of particular examples proved that the assessment of genome track inhomogeneity and the robustness of the algorithm against inhomogeneity are crucial for the correct analysis of correlations between genome tracks. In this article, we present the quantitative measure for inhomogeneity of genome tracks, robust statistical criteria, and an algorithm for the analysis of correlations between two genome tracks, and we compare the different packages. Furthermore, we demonstrate the application of our package to the study of particular correlations between different genetic and epigenetic features related to the regulation of gene expression in eukaryotes. Among our particular applications, the correlations between CpG islands and TSS are well studied and serve in our work mainly as an additional test, whereas the correlations between TSS on the forward and reverse strands in eukaryotic genomes, as well as the correlations between DSBs and H3K4me3 marks in the <em>Homo sapiens</em> genome, can be considered as original. The correlations between protein-binding profiles for <em>Drosophila melanogaster</em> were likely not studied using the similar statistical methods. They reflect the mutual dependence between features that was established experimentally. The correlations between particular genomic tracks obtained with the Genomic HyperBrowser and GenometriCorr were compared with that obtained using our package.</p>                            <h2 scrollto-destination=6615562 id="6615562" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
<p>In this section, we describe the basic theory and the main algorithm. The genome tracks proved to be often inhomogeneous and strongly different from quasi-homogeneous random distributions. We present the quantitative criteria for assessment of genome track inhomogeneity. The algorithm developed in this article is robust against inhomogeneity effects. It is implied that, after preprocessing (Preprocessing of input genetic data), the genetic data will correspond to the set of points that divides the genome (or a genome region) under study into a set of fragments.</p>                            <h3 scrollto-destination=6615564 id="6615564" class="section-title" data-legacy-id=s2a>2.1. Reference model</h3>
<div class="block-child-p">The reference model used in this work for the assessment of correlation significance is based on the random division of an interval. Let the interval of length <em>M</em> be divided by <em>N</em> – 1 points into <em>N</em> fragments: <a id="DSU044M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">M</mi><mo xmlns="">=</mo><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></munderover><mrow xmlns=""><mrow><msub><mi>L</mi><mi>i</mi></msub></mrow></mrow></math></div></div><div class="label">(1)</div></div> Here, <em>L<sub>i</sub></em> is the length of fragment. The respective mean length is given by: <a id="DSU044M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo fence="false" stretchy="false">⟨</mo></mrow><mi xmlns="">L</mi><mrow xmlns=""><mo fence="false" stretchy="false">⟩</mo></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mi>M</mi><mi>N</mi></mfrac></mrow></mstyle></math></div></div><div class="label">(2)</div></div> We will use henceforth the normalized lengths: <a id="DSU044M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>l</mi><mi>i</mi></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>L</mi><mi>i</mi></msub></mrow></mrow><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mi>L</mi><mrow><mo fence="false" stretchy="false">⟩</mo></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(3)</div></div> The probability that the normalized lengths of fragments (<span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>l</mi><mi>i</mi><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow></math></span>) exceed some <em>a priori</em> chosen values (<span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>l</mi><mi>i</mi></msub></mrow></math></span>) is defined by De Finetti distribution:<sup><a reveal-id="DSU044C15" data-open="DSU044C15" class="link link-ref link-reveal xref-bibr">15</a></sup><a id="DSU044M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo movablelimits="true" xmlns="">Pr</mo><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msubsup><mi>l</mi><mn>1</mn><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow><mo xmlns="">≥</mo><mrow xmlns=""><msub><mi>l</mi><mn>1</mn></msub></mrow><mo xmlns="">,</mo><mo xmlns="">…</mo><mspace width="thinmathspace" xmlns="" /><mo xmlns="">,</mo><mrow xmlns=""><msubsup><mi>l</mi><mi>N</mi><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow><mo xmlns="">≥</mo><mrow xmlns=""><msub><mi>l</mi><mi>N</mi></msub></mrow><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><msubsup xmlns=""><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mstyle><mrow><mfrac><mrow><mrow><msub><mi>l</mi><mn>1</mn></msub></mrow></mrow><mi>N</mi></mfrac></mrow><mo>−</mo><mo>⋯</mo><mo>−</mo><mstyle><mrow><mfrac><mrow><mrow><msub><mi>l</mi><mi>N</mi></msub></mrow></mrow><mi>N</mi></mfrac></mrow></mstyle></mstyle></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mi>N</mi><mo>−</mo><mn>1</mn></mrow></msubsup></math></div></div><div class="label">(4)</div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mo>+</mo></msub></mrow><mo xmlns="">=</mo><mi xmlns="">x</mi></math></span> if <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">x</mi><mo xmlns="">&gt;</mo><mn xmlns="">0</mn></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mo>+</mo></msub></mrow><mo xmlns="">=</mo><mn xmlns="">0</mn></math></span> if <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">x</mi><mo xmlns="">≤</mo><mn xmlns="">0</mn></math></span>. The analytical derivation of De Finetti distribution and some relevant results can be found in previous publications.<sup><a reveal-id="DSU044C16" data-open="DSU044C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU044C17" data-open="DSU044C17" class="link link-ref link-reveal xref-bibr">17</a></sup> We present below the auxiliary statistical criteria needed for the main algorithm.</div><div class="block-child-p">The corresponding one-fragment probability is defined as: <a id="DSU044M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo movablelimits="true" xmlns="">Pr</mo><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msubsup><mi>l</mi><mn>1</mn><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow><mo xmlns="">≥</mo><mi xmlns="">l</mi><mo xmlns="">,</mo><mrow xmlns=""><msubsup><mi>l</mi><mn>2</mn><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow><mo xmlns="">≥</mo><mn xmlns="">0</mn><mo xmlns="">,</mo><mo xmlns="">…</mo><mspace width="thinmathspace" xmlns="" /><mo xmlns="">,</mo><mrow xmlns=""><msubsup><mi>l</mi><mi>N</mi><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow><mo xmlns="">≥</mo><mn xmlns="">0</mn><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><msubsup xmlns=""><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mstyle><mrow><mfrac><mi>l</mi><mi>N</mi></mfrac></mrow></mstyle></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mi>N</mi><mo>−</mo><mn>1</mn></mrow></msubsup></math></div></div><div class="label">(5)</div></div> and, at the limit of large <em>N</em>, can be approximated by the exponential distribution <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo movablelimits="true" xmlns="">Pr</mo><mo xmlns="">≈</mo><mrow xmlns=""><msup><mi>e</mi><mrow><mo>−</mo><mi>l</mi></mrow></msup></mrow></math></span>. The mean minimum length from <em>k</em> fragments and its dispersion are: <a id="DSU044M6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo fence="false" stretchy="false">⟨</mo></mrow><mrow xmlns=""><msub><mi>l</mi><mrow><mo movablelimits="true">min</mo><mspace width="thinmathspace" /><mrow><mo stretchy="false">|</mo></mrow><mspace width="thinmathspace" /><mi>k</mi></mrow></msub></mrow><mrow xmlns=""><mo fence="false" stretchy="false">⟩</mo></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mn>1</mn><mi>k</mi></mfrac></mrow><mo>;</mo><mspace width="1em" /><mrow><msup><mi>σ</mi><mn>2</mn></msup></mrow><mo stretchy="false">(</mo><mrow><msub><mi>l</mi><mrow><mo movablelimits="true">min</mo><mspace width="thinmathspace" /><mrow><mo stretchy="false">|</mo></mrow><mspace width="thinmathspace" /><mi>k</mi></mrow></msub></mrow><mo stretchy="false">)</mo><mo>=</mo><mstyle><mrow><mfrac><mrow><mi>N</mi><mo>−</mo><mn>1</mn></mrow><mrow><mrow><msup><mi>k</mi><mn>2</mn></msup></mrow><mo stretchy="false">(</mo><mi>N</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mstyle></math></div></div><div class="label">(6)</div></div> The characteristic maximum and minimum values in the complete set of normalized lengths corresponding to random division of an interval can be estimated as:<sup><a reveal-id="DSU044C17" data-open="DSU044C17" class="link link-ref link-reveal xref-bibr">17</a></sup><a id="DSU044M7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M7"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>l</mi><mrow><mo movablelimits="true">max</mo></mrow></msub></mrow><mo xmlns="">≈</mo><mi xmlns="">ln</mi><mo xmlns="">⁡</mo><mi xmlns="">N</mi><mo xmlns="">;</mo><mspace width="1em" xmlns="" /><mrow xmlns=""><msub><mi>l</mi><mrow><mo movablelimits="true">min</mo></mrow></msub></mrow><mo xmlns="">≈</mo><mstyle xmlns=""><mrow><mfrac><mn>1</mn><mi>N</mi></mfrac></mrow></mstyle></math></div></div><div class="label">(7)</div></div> If the input data are filtered out with respect to outliers, the potential outliers should be compared with extreme values in the reference model.</div>                            <h3 scrollto-destination=6615567 id="6615567" class="section-title" data-legacy-id=s2b>2.2. Assessment of genome track inhomogeneity</h3>
<div class="block-child-p">When studying correlations between genome tracks, it is useful to begin with the preliminary assessment of input data. In our package, the (in)homogeneity of length distribution related to the genome tracks is assessed by two methods. In the first method, the distribution of normalized fragment lengths is compared with one-fragment distribution (5) by the Kolmogorov–Smirnov criterion. In the second method, the homogeneity of length distribution can be assessed with an entropy-like function:<sup><a reveal-id="DSU044C17" data-open="DSU044C17" class="link link-ref link-reveal xref-bibr">17</a></sup><a id="DSU044M8" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M8"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">structural</mi></mrow></mrow></msub></mrow><mo xmlns="">=</mo><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></munderover><mrow xmlns=""><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow><mi>ln</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></mrow></math></div></div><div class="label">(8)</div></div> As can be proved, the function (8) at the restriction (1) attains its minimum equal to zero for <em>l</em><sub>1</sub> = <em>l</em><sub>2</sub> = … = <em>l<sub>N</sub></em> = 1. This distribution corresponds to the most homogeneous one. The higher the value of the entropy (8), the stronger the variations in fragment lengths or the higher the inhomogeneity of input data. The mean value and dispersion of structural entropy for the random division of an interval are: <a id="DSU044M9" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M9"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo fence="false" stretchy="false">⟨</mo></mrow><mrow xmlns=""><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mrow xmlns=""><mo fence="false" stretchy="false">⟩</mo></mrow><mo xmlns="">=</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mi xmlns="">C</mi><mo stretchy="false" xmlns="">)</mo><mi xmlns="">N</mi><mo xmlns="">=</mo><mn xmlns="">0.422785</mn><mspace width="thinmathspace" xmlns="" /><mo xmlns="">…</mo><mspace width="thinmathspace" xmlns="" /><mi xmlns="">N</mi><mo xmlns="">;</mo><mspace width="1em" xmlns="" /><mrow xmlns=""><msup><mi>σ</mi><mn>2</mn></msup></mrow><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mtext xmlns="">0</mtext><mrow xmlns=""><mn>.289868</mn></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">…</mo><mspace width="thinmathspace" xmlns="" /><mi xmlns="">N</mi></math></div></div><div class="label">(9)</div></div> where <em>C</em> = 0.577215… is the Euler constant. The (in)homogeneity of the input data can be estimated in terms of the <em>z</em>-ratio: <a id="DSU044M10" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M10"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>z</mi><mi>S</mi></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mrow><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mo>−</mo><mrow><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">data</mi></mrow></mrow></msub></mrow></mrow><mrow><mi>σ</mi><mo stretchy="false">(</mo><mrow><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(10)</div></div> The <em>z</em>-ratio (10) obeys approximately Gaussian statistics for the random deviations.</div>                            <h3 scrollto-destination=6615569 id="6615569" class="section-title" data-legacy-id=s2c>2.3. Ordering of genomic features</h3>
<div class="block-child-p">Many genetic features are ordered with respect to each other, e.g. 5′-ends of CpG islands often precede the TSS. Such ordering may be related to the coordinated regulation mechanisms. The level of ordering for the pairs of points can be assessed by the criterion: <a id="DSU044M11" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M11"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>z</mi><mi>P</mi></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>P</mi><mo>+</mo></msub></mrow><mrow><mrow><mo>/</mo></mrow></mrow><mi>P</mi><mo>−</mo><mn>0.5</mn></mrow><mrow><mn>0.5</mn><mrow><msup><mi>P</mi><mrow><mo>−</mo><mn>1</mn><mrow><mo>/</mo></mrow><mn>2</mn></mrow></msup></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(11)</div></div> where <em>P</em> is the total number of pairs and <em>P</em><sub>+</sub> is the number of pairs in which the points of one type precede the points of the other type. At the limit of large <em>P</em>, the deviations (11) also obey Gaussian statistics.</div>                            <h3 scrollto-destination=6615571 id="6615571" class="section-title" data-legacy-id=s2d>2.4. Preprocessing of input genetic data</h3>
                            <h4 scrollto-destination=6615572 id="6615572" class="section-title" data-legacy-id=s2d1>2.4.1. Points</h4>
<p>If the input genetic data correspond to the particular sites, e.g. sites of DSBs, TSS, etc., they are commonly used without additional preprocessing. If needed, preliminary clustering of points can be performed.</p>                            <h4 scrollto-destination=6615574 id="6615574" class="section-title" data-legacy-id=s2d2>2.4.2. Stretches</h4>
<p>The stretches, such as CpG islands, transposons, etc., are replaced by points in the starts/centres/ends of the stretches. The overlapping of stretches is permissible. The mean distance between consecutive reference points should be much longer than the characteristic length of stretches.</p>                            <h4 scrollto-destination=6615576 id="6615576" class="section-title" data-legacy-id=s2d3>2.4.3. Profiles</h4>
<div class="block-child-p">The profiles characterizing DNA–protein binding are assumed to be defined throughout the sites. The formats of profiles may depend on the database or vary even within a particular database. Commonly, the peaks in profiles should be determined from several experimental sets and from the set with background data. In this case, using the data from several experiments as well as the data on the background noise, the maxima in profiles or the significant regions can be identified over the genome by one of numerous available packages.<sup><a reveal-id="DSU044C18  DSU044C20" data-open="DSU044C18  DSU044C20" class="link link-ref link-reveal xref-bibr">18–20</a></sup> Then, such maxima and regions can be processed as points and stretches. We integrated into our server the popular peak-caller MACS2,<sup><a reveal-id="DSU044C21" data-open="DSU044C21" class="link link-ref link-reveal xref-bibr">21</a></sup> which is convenient for processing ChIP-Seq data. If needed, the researcher can use any other available caller for profile preprocessing. Some profiles are presented in databases in aggregated form comprising several data sets. After identifying peaks with an available caller or when using aggregated profiles, further clustering preprocessing of profiles can be performed. In our package, such clustering preprocessing first filters out the insignificant values lower than a given threshold (in terms of the mean + <em>n</em> SD). Then, the significant values exceeding a given threshold are clustered by the following rule: the consecutive points nearer than a given distance belong to the same cluster. The values of the threshold and the clustering distance are defined by the user. The site corresponding to a cluster is determined by the centre-of-mass rule: <a id="DSU044M12" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M12"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>m</mi><mi>c</mi></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><msub><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mspace width="thinmathspace" /><mspace width="thinmathspace" /><mo>∈</mo><mspace width="thinmathspace" /><mspace width="thinmathspace" /><mrow><mi mathvariant="normal">cluster</mi></mrow></mrow></msub><mrow><mrow><msub><mi>m</mi><mi>i</mi></msub></mrow><mrow><msub><mi>H</mi><mi>i</mi></msub></mrow></mrow></mrow><mrow><msub><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mspace width="thinmathspace" /><mspace width="thinmathspace" /><mo>∈</mo><mspace width="thinmathspace" /><mspace width="thinmathspace" /><mrow><mi mathvariant="normal">cluster</mi></mrow></mrow></msub><mrow><mrow><msub><mi>H</mi><mi>i</mi></msub></mrow></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(12)</div></div> where <em>H<sub>i</sub></em> is the height of profile at the site <em>m<sub>i</sub></em>. As the correlations may depend on the clustering distance (Correlations between protein-binding profiles), we recommend using a variable clustering distance that covers a range of characteristic lengths for the problem concerned. The format of input data for points is txt and for stretches is BED/BED6, whereas the profiles can be loaded in SGR, WIG, bigWIG, or bedGraph.</div>                            <h3 scrollto-destination=6615578 id="6615578" class="section-title" data-legacy-id=s2e>2.5. Algorithm</h3>
<p>The analysis shows that most genetic data are strongly inhomogeneous, i.e. the variations in lengths corresponding to genome tracks are stronger than that corresponding to the random division of the genome in the same number of fragments (Assessment of genome track inhomogeneity). Therefore, the criteria and algorithm for assessment of mutual correlations between two genome tracks should be robust against inhomogeneity of corresponding length distributions. The algorithm presented below satisfies this requirement.</p><div class="block-child-p">It is assumed that the genome strand under study is restricted by terminal points corresponding to one or two genome tracks. Such a definition excludes the possible edge effects. Within such a strand, let the number of points related to the first set be <em>N<sub>A</sub></em> (the related points will be denoted for brevity by <em>A</em>), whereas the corresponding number of points for the second set is <em>N<sub>B</sub></em> (the related points will be denoted by <em>B</em>). Then, the fractions of points are determined as <a id="DSU044M13" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M13"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>N</mi><mi>A</mi></msub></mrow></mrow><mrow><mrow><msub><mi>N</mi><mi>A</mi></msub></mrow><mo>+</mo><mrow><msub><mi>N</mi><mi>B</mi></msub></mrow></mrow></mfrac></mrow><mo>;</mo><mspace width="1em" /><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><mo>=</mo><mstyle><mrow><mfrac><mrow><mrow><msub><mi>N</mi><mi>B</mi></msub></mrow></mrow><mrow><mrow><msub><mi>N</mi><mi>A</mi></msub></mrow><mo>+</mo><mrow><msub><mi>N</mi><mi>B</mi></msub></mrow></mrow></mfrac></mrow></mstyle></mstyle></math></div></div><div class="label">(13)</div></div> The division of points on the same strand into two sets generates a string of points that may formally be presented as a sequence composed of <em>A</em> and <em>B</em>. The neighbouring points of two types can be encountered only within the following sequences: <em>ABA</em>, <em>BAB</em>, <em>AABB</em>, <em>BBAA</em>, and their combinations. Our algorithm relies on the following rules:
Consider, for example, the combination <em>ABA</em>. Let the distance for <em>AB</em> be <em>L</em><sub>1</sub> and the distance for <em>BA</em> be <em>L</em><sub>2</sub>. The mean local distance for this combination is: <a id="DSU044M14" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M14"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mi>L</mi><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>L</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>L</mi><mn>2</mn></msub></mrow></mrow><mn>2</mn></mfrac></mrow></mstyle></math></div></div><div class="label">(14)</div></div> whereas the respective normalized length between the nearest neighbours (NN) is determined as: <a id="DSU044M15" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M15"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mo movablelimits="true">min</mo><mo stretchy="false">(</mo><mrow><msub><mi>L</mi><mn>1</mn></msub></mrow><mo>,</mo><mrow><msub><mi>L</mi><mn>2</mn></msub></mrow><mo stretchy="false">)</mo></mrow><mrow><mrow><msub><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mi>L</mi><mrow><mo fence="false" stretchy="false">⟩</mo></mrow></mrow><mrow><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(15)</div></div> A similar normalization was used also by Favorov <em>et al.</em><sup><a reveal-id="DSU044C14" data-open="DSU044C14" class="link link-ref link-reveal xref-bibr">14</a></sup></div><ul class="bullet"><li><p>The subsequences (<em>A</em>)<em><sub>k</sub></em> or (<em>B</em>)<em><sub>k</sub></em> (<em>k</em> ≥ 3) are discarded, because they cannot be associated with the neighbouring points of the other type. The subsequences <em>AABB</em> or <em>BBAA</em> are also discarded.</p></li><li><p>This leaves us only two fundamental types of subsequences: <em>ABA</em> and <em>BAB</em>. All other possible sequences under consideration are their combinations. Our algorithm processes the entire sequence of points, splitting it into fundamental subsequences <em>ABA</em> and <em>BAB</em> from left to right. In the process of splitting, consequent fundamental subsequences can have only one common border letter, and they cannot overlap by two or more letters. For example, <em>ABABA</em> is split into <em>ABA</em> and <em>ABA</em>.</p></li><li><p>In the subsequence <em>BAB</em>, the pair of the nearest neighbours is determined relative to the <em>A</em> point and corresponds to the minimum of two lengths in the combinations <em>BA</em> and <em>AB</em>, whereas in the subsequence <em>ABA</em> the pair of the nearest neighbours is determined relative to the <em>B</em> point and corresponds to the minimum of two lengths in the combinations <em>AB</em> and <em>BA</em>.</p></li></ul><div class="block-child-p">The correlations between two sets of points are assumed to be absent if at least one of the sets <em>A</em> and <em>B</em> is random and independent. Consider the situation when one of the sets is random (<em>R</em>), whereas the other is a non-random inhomogeneous (<em>I</em>) set. This implies the correspondence <em>A</em>, <em>B</em> ↔ <em>R</em>, <em>I</em> or <em>ABA</em>, <em>BAB</em> ↔ <em>RIR</em>, <em>IRI</em>. It is very important that the statistics turns out to be different for the combinations <em>IRI</em> and <em>RIR</em>. The moments for the normalized lengths in the combinations <em>IRI</em> are defined by Equation (6) for <em>k</em> = <em>N</em> = 2: <a id="DSU044M16" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M16"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mrow><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mn>1</mn><mn>2</mn></mfrac></mrow><mo>;</mo><mspace width="1em" /><msubsup><mi>σ</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mn>2</mn></msubsup><mo>=</mo><mstyle><mrow><mfrac><mn>1</mn><mrow><mn>12</mn></mrow></mfrac></mrow></mstyle></mstyle></math></div></div><div class="label">(16)</div></div> The significance of divergence between <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mrow><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow></math></span> and the mean normalized length <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>l</mi><mo stretchy="false">¯</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow></math></span> obtained in a particular realization may be assessed by the <em>z</em>-ratio: <a id="DSU044M17" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M17"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>z</mi><mrow><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mrow><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><mrow><mo fence="false" stretchy="false">⟩</mo></mrow></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo>−</mo><mrow><msub><mrow><mrow><mover><mi>l</mi><mo stretchy="false">¯</mo></mover></mrow></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow></mrow><mrow><mrow><msup><mrow><mo stretchy="false">(</mo><msubsup><mi>σ</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mn>2</mn></msubsup><mrow><mo>/</mo></mrow><mrow><msub><mi>K</mi><mrow><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow><mrow><mn>1</mn><mrow><mo>/</mo></mrow><mn>2</mn></mrow></msup></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(17)</div></div> which obeys the standard Gaussian statistics <em>N</em>(0, 1) at the limit of large number of combinations <em>K<sub>IRI</sub></em>. Let the neighbouring points for a combination <em>RIR</em> be <em>P</em><sub>1</sub> and <em>P</em><sub>2</sub>, i.e. <em>RIR</em> can be considered as a part of combination <em>Р</em><sub>1</sub><em>RIRР</em><sub>2</sub>. The random points (<em>R</em>) in the combination <em>RIR</em> are homogeneously distributed within intervals <em>Р</em><sub>1</sub><em>I</em> and <em>IР</em><sub>2</sub>. The corresponding moments for the normalized lengths (15) in the combinations <em>RIR</em> are calculated as <a id="DSU044M18" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M18"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="right" columnspacing="thickmathspace" rowspacing=".5em" xmlns=""><mtr><mtd><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><msubsup><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow><mi>k</mi></msubsup><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo>=</mo></mtd><mtd><munderover><mo>∫</mo><mn>0</mn><mrow><mrow><msub><mi>L</mi><mrow><mrow><msub><mi>P</mi><mrow><mrow><mspace width="1pt" /></mrow><mn>1</mn></mrow></msub></mrow><mi>I</mi></mrow></msub></mrow></mrow></munderover><mrow><mstyle><mrow><mfrac><mrow><mi>d</mi><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mrow><msub><mi>P</mi><mn>1</mn></msub></mrow><mi>I</mi></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mrow><munderover><mo>∫</mo><mn>0</mn><mrow><mrow><msub><mi>L</mi><mrow><mi>I</mi><mrow><msub><mi>P</mi><mrow><mrow><mspace width="1pt" /></mrow><mn>2</mn></mrow></msub></mrow></mrow></msub></mrow></mrow></munderover><mrow><mstyle><mrow><mfrac><mrow><mi>d</mi><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mi>I</mi><mrow><msub><mi>P</mi><mrow><mrow><mspace width="1pt" /></mrow><mn>2</mn></mrow></msub></mrow></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mrow><mfenced close="⟩" open="⟨"><mrow><mrow><msup><mrow><mfenced close=")" open="("><mrow><mstyle><mrow><mfrac><mrow><mn>2</mn><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow><mo>+</mo><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mrow></mfenced></mrow><mi>k</mi></msup></mrow></mrow></mfenced><mi>θ</mi><mfenced close="]" open="["><mrow><mn>1</mn><mspace width="thinmathspace" /><mo>−</mo><mspace width="thinmathspace" /><mstyle><mrow><mfrac><mrow><mn>2</mn><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow></mrow><mrow><mo stretchy="false">(</mo><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow><mo>+</mo><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mrow></mfenced></mtd></mtr><mtr><mtd /><mtd><mo>+</mo><munderover><mo>∫</mo><mn>0</mn><mrow><mrow><msub><mi>L</mi><mrow><mrow><msub><mi>P</mi><mrow><mrow><mspace width="1pt" /></mrow><mn>1</mn></mrow></msub></mrow><mi>I</mi></mrow></msub></mrow></mrow></munderover><mrow><mstyle><mrow><mfrac><mrow><mi>d</mi><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mrow><msub><mi>P</mi><mn>1</mn></msub></mrow><mi>I</mi></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mrow><munderover><mo>∫</mo><mn>0</mn><mrow><mrow><msub><mi>L</mi><mrow><mi>I</mi><mrow><msub><mi>P</mi><mrow><mrow><mspace width="1pt" /></mrow><mn>2</mn></mrow></msub></mrow></mrow></msub></mrow></mrow></munderover><mrow><mstyle><mrow><mfrac><mrow><mi>d</mi><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mi>I</mi><mrow><msub><mi>P</mi><mrow><mrow><mspace width="1pt" /></mrow><mn>2</mn></mrow></msub></mrow></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mrow><mfenced close="⟩" open="⟨"><mrow><mrow><msup><mrow><mfenced close=")" open="("><mrow><mstyle><mrow><mfrac><mrow><mn>2</mn><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow><mo>+</mo><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mrow></mfenced></mrow><mi>k</mi></msup></mrow></mrow></mfenced><mi>θ</mi><mfenced close="]" open="["><mrow><mn>1</mn><mspace width="thinmathspace" /><mo>−</mo><mspace width="thinmathspace" /><mstyle><mrow><mfrac><mrow><mn>2</mn><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow></mrow><mrow><mo stretchy="false">(</mo><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">RI</mi></mrow></mrow></msub></mrow><mo>+</mo><mrow><msub><mi>L</mi><mrow><mrow><mi mathvariant="normal">IR</mi></mrow></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mrow></mfenced></mtd></mtr></mtable></math></div></div><div class="label">(18)</div></div> where <em>θ</em>(<em>x</em>) is the Heaviside step function, <em>θ</em>(<em>x</em>) = 1 if <em>x</em> &gt; 0 and <em>θ</em>(<em>x</em>) = 0 if <em>x</em> &lt; 0. The angular brackets denote the averaging over an ensemble of realizations. The bulky but straightforward computations provide the explicit expressions: <a id="DSU044M19" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M19"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" columnspacing="thickmathspace" rowspacing=".5em" xmlns=""><mtr><mtd /><mtd><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mrow><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" /><mo>=</mo><mspace width="thinmathspace" /><mo fence="false" stretchy="false">⟨</mo><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mspace width="thinmathspace" /><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mn>2</mn><mspace width="thinmathspace" /><mi>ln</mi><mspace width="thinmathspace" /><mn>2</mn><mo stretchy="false">)</mo><mo>−</mo><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mi>ln</mi><mspace width="thinmathspace" /><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" /><mo>+</mo><mspace width="thinmathspace" /><mn>1</mn></mtd></mtr><mtr><mtd /><mtd><mspace width="6pc" /><mo>−</mo><mo stretchy="false">(</mo><mn>1</mn><mspace width="thinmathspace" /><mo>−</mo><mspace width="thinmathspace" /><msubsup><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow><mrow><mn>2</mn></mrow></msubsup><mo stretchy="false">)</mo><mi>ln</mi><mspace width="thinmathspace" /><mo stretchy="false">(</mo><mn>1</mn><mspace width="thinmathspace" /><mo>+</mo><mspace width="thinmathspace" /><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo fence="false" stretchy="false">⟩</mo><mo>;</mo></mtd></mtr><mtr><mtd /><mtd><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><msubsup><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow><mn>2</mn></msubsup><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub><mspace width="thinmathspace" /><mo>=</mo><mspace width="thinmathspace" /><mo fence="false" stretchy="false">⟨</mo><mn>2</mn><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mn>2</mn><mspace width="thinmathspace" /><mi>ln</mi><mspace width="thinmathspace" /><mn>2</mn><mo stretchy="false">)</mo><mspace width="thinmathspace" /><mo>+</mo><mspace width="thinmathspace" /><mn>4</mn><mi>ln</mi><mspace width="thinmathspace" /><mo stretchy="false">(</mo><mn>1</mn><mo>+</mo><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mrow><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mo fence="false" stretchy="false">⟩</mo><mo>;</mo></mtd></mtr><mtr><mtd /><mtd><msubsup><mi>σ</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mn>2</mn></msubsup><mspace width="thinmathspace" /><mo>=</mo><mspace width="thinmathspace" /><mrow><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><msubsup><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow><mn>2</mn></msubsup><msub><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo>−</mo><mrow><mo fence="false" stretchy="false">⟨</mo></mrow><mrow><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><msubsup><mrow><mo fence="false" stretchy="false">⟩</mo></mrow><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mrow><mn>2</mn></mrow></msubsup></mtd></mtr></mtable></math></div></div><div class="label">(19)</div></div> where <a id="DSU044M20" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M20"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>r</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns="" /><mo movablelimits="true" xmlns="">min</mo><mrow xmlns=""><mo>(</mo><mrow><mstyle><mrow><mfrac><mrow><mrow><msub><mi>L</mi><mrow><mrow><msub><mi>P</mi><mn>1</mn></msub></mrow><mi>I</mi></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mi>I</mi><mrow><msub><mi>P</mi><mn>2</mn></msub></mrow></mrow></msub></mrow></mrow></mfrac></mrow><mo>,</mo><mstyle><mrow><mfrac><mrow><mrow><msub><mi>L</mi><mrow><mi>I</mi><mrow><msub><mi>P</mi><mn>2</mn></msub></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>L</mi><mrow><mrow><msub><mi>P</mi><mn>1</mn></msub></mrow><mi>I</mi></mrow></msub></mrow></mrow></mfrac></mrow></mstyle></mstyle></mrow><mo>)</mo></mrow></math></div></div><div class="label">(20)</div></div> If the averaging over the ensemble of realizations is replaced by the mean in a particular realization, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo fence="false" stretchy="false" xmlns="">⟨</mo><mo xmlns="">…</mo><mo fence="false" stretchy="false" xmlns="">⟩</mo><mo stretchy="false" xmlns="">→</mo><munder xmlns=""><mo movablelimits="false">∑</mo><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></munder><mrow xmlns=""><mo stretchy="false">(</mo><mo>…</mo><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mrow><msub><mi>K</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow></mrow></math></span> (here the dots correspond to the expressions in the right-hand side of Equation (19), and <em>K<sub>RIR</sub></em> is the total number of combinations <em>RIR</em>), the variable <a id="DSU044M21" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M21"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ς</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns="" /><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mrow><mrow><mover><mi>l</mi><mo stretchy="false">~</mo></mover></mrow></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" /><mo>−</mo><mspace width="thinmathspace" /><mrow><msub><mrow><mrow><mover><mi>l</mi><mo stretchy="false">¯</mo></mover></mrow></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow></mrow><mrow><mrow><msup><mrow><mo stretchy="false">(</mo><msubsup><mrow><mover><mi>σ</mi><mo stretchy="false">~</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mn>2</mn></msubsup><mrow><mo>/</mo></mrow><mrow><msub><mi>K</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow><mrow><mn>1</mn><mrow><mo>/</mo></mrow><mn>2</mn></mrow></msup></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(21)</div></div> obeys Gaussian statistics <em>N</em>(0, <em>c</em><sub>ς</sub>) with dispersion <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>c</mi><mi>ς</mi><mn>2</mn></msubsup><mo xmlns="">≠</mo><mn xmlns="">1</mn></math></span>. Here, the wave over the reference random values corresponds to the replacement <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo fence="false" stretchy="false" xmlns="">⟨</mo><mo xmlns="">…</mo><mo fence="false" stretchy="false" xmlns="">⟩</mo><mo stretchy="false" xmlns="">→</mo><munder xmlns=""><mo movablelimits="false">∑</mo><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></munder><mrow xmlns=""><mo stretchy="false">(</mo><mo>…</mo><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mrow><msub><mi>K</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow></mrow></math></span> in Equation (19) and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>l</mi><mo stretchy="false">¯</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow></math></span> is the mean normalized length for the combinations <em>RIR</em>. The variable corresponding to the standard Gaussian statistics <em>N</em>(0, 1) can be obtained by the normalization: <a id="DSU044M22" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M22"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>z</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns="" /><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>ς</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>c</mi><mi>ς</mi></msub></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(22)</div></div> The normalization parameter <em>c</em><sub>ς</sub> depends weakly on the ratio between the numbers of points belonging to random and inhomogeneous sets, <em>N<sub>R</sub></em>/<em>N<sub>I</sub></em>. The simulations yield the dependence, <em>N<sub>R</sub></em>/<em>N<sub>I</sub></em> = 0.25; 0.5; 1; 2; 4 and <em>c</em><sub>ς</sub> = 0.833; 0.837; 0.855; 0.862; 0.881. We used the linear interpolation for <em>c</em><sub>ς</sub> within this range of ratios <em>N<sub>R</sub></em>/<em>N<sub>I</sub></em>, while beyond this range the values of <em>c</em><sub>ς</sub> were taken to be equal to the boundary quantities.</div><div class="block-child-p">The united Gaussian criterion of proximity between two sets of points is formulated in terms of: <a id="DSU044M23" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M23"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>z</mi><mrow><mrow><mi mathvariant="normal">corr</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns="" /><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>S</mi><mrow><mi>K</mi><mo>,</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo>−</mo><mrow><msub><mi>S</mi><mi>K</mi></msub></mrow></mrow><mrow><mrow><msup><mrow><mo stretchy="false">(</mo><mrow><msub><mi>K</mi><mrow><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><msubsup><mi>σ</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mn>2</mn></msubsup><mo>+</mo><mrow><msub><mi>K</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><msubsup><mi>c</mi><mi>ς</mi><mn>2</mn></msubsup><msubsup><mrow><mover><mi>σ</mi><mo stretchy="false">~</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow><mrow><mn>1</mn><mrow><mo>/</mo></mrow><mn>2</mn></mrow></msup></mrow></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(23)</div></div> where <a id="DSU044M24" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M24"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>S</mi><mi>K</mi></msub></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns="" /><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>K</mi></munderover><mrow xmlns=""><mrow><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">local</mi><mo>;</mo></mrow><mi>k</mi></mrow></msub></mrow></mrow></math></div></div><div class="label">(24)</div></div><a id="DSU044M25" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M25"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>S</mi><mrow><mi>K</mi><mo>,</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" xmlns="" /><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns="" /><mrow xmlns=""><msub><mi>K</mi><mrow><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><mo fence="false" stretchy="false" xmlns="">⟨</mo><mrow xmlns=""><msub><mi>l</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>I</mi><mi>R</mi><mi>I</mi></mrow></msub></mrow><mrow xmlns=""><msub><mo fence="false" stretchy="false">⟩</mo><mrow><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow><mo xmlns="">+</mo><mrow xmlns=""><msub><mi>K</mi><mrow><mi>R</mi><mi>I</mi><mi>R</mi></mrow></msub></mrow><mrow xmlns=""><msub><mrow><mover><mi>l</mi><mo stretchy="false">~</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">NN</mi></mrow><mo>,</mo><mi>R</mi><mi>I</mi><mi>R</mi><mo>;</mo><mrow><mi mathvariant="normal">random</mi></mrow></mrow></msub></mrow></math></div></div><div class="label">(25)</div></div> and the other nomenclature is as defined in Equations (16)–(22). The correspondence <em>A</em>, <em>B</em> ↔ <em>R</em>, <em>I</em> or <em>ABA</em>, <em>BAB</em> ↔ <em>RIR</em>, <em>IRI</em> is defined by the criterion (10). The set with the higher |<em>z<sub>S</sub></em>| is associated with an inhomogeneous set, whereas the set with the lower |<em>z<sub>S</sub></em>| is associated with a random set. The criterion (23) remains valid when the inhomogeneous non-random set tends towards a random set. The positive values of <em>z</em><sub>corr</sub> reflect a trend towards shorter distances between point sets relative to the reference model (or correlations), whereas the negative values of <em>z</em><sub>corr</sub> reflect a trend towards longer distances between point sets (or anticorrelations). The ordering between the pairs of the nearest neighbours contributing to sum (24) can be additionally assessed by <em>z<sub>P</sub></em>-criterion (11).</div><div class="block-child-p">The mean fraction of the nearest neighbours between two random sets of points is <a id="DSU044M26" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU044M26"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" columnspacing="thickmathspace" rowspacing=".5em" xmlns=""><mtr><mtd><mrow><msub><mi>F</mi><mrow><mrow><mi mathvariant="normal">NN</mi></mrow></mrow></msub></mrow><mspace width="thinmathspace" /><mo>=</mo><mspace width="thinmathspace" /></mtd><mtd><mo stretchy="false">(</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><msubsup><mi>f</mi><mi>B</mi><mn>4</mn></msubsup><mo>+</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><msubsup><mi>f</mi><mi>A</mi><mn>4</mn></msubsup><mo stretchy="false">)</mo><munderover><mo movablelimits="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>0</mn></mrow><mi mathvariant="normal">∞</mi></munderover><mrow><mo stretchy="false">(</mo><mi>k</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo><mrow><msup><mrow><mo stretchy="false">(</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mi>k</mi></msup></mrow></mrow><mo>+</mo><mn>2</mn><msubsup><mi>f</mi><mi>A</mi><mn>2</mn></msubsup><msubsup><mi>f</mi><mi>B</mi><mn>2</mn></msubsup><munderover><mo movablelimits="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi mathvariant="normal">∞</mi></munderover><mrow><mi>k</mi><mrow><msup><mrow><mo stretchy="false">(</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mi>k</mi></msup></mrow></mrow></mtd></mtr><mtr><mtd><mo>=</mo></mtd><mtd><mstyle><mrow><mfrac><mrow><mo stretchy="false">(</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><msubsup><mi>f</mi><mi>B</mi><mn>4</mn></msubsup><mo>+</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><msubsup><mi>f</mi><mi>A</mi><mn>4</mn></msubsup><mo stretchy="false">)</mo></mrow><mrow><mrow><msup><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></mrow></mfrac></mrow><mo>+</mo><mstyle><mrow><mfrac><mrow><mn>2</mn><msubsup><mi>f</mi><mi>A</mi><mn>3</mn></msubsup><msubsup><mi>f</mi><mi>B</mi><mn>3</mn></msubsup></mrow><mrow><mrow><msup><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>A</mi></msub></mrow><mrow><mspace width="thinmathspace" /><msub><mi>f</mi><mi>B</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></mrow></mfrac></mrow></mstyle></mstyle></mtd></mtr></mtable></math></div></div><div class="label">(26)</div></div> In particular, if <em>f<sub>A</sub></em> = <em>f<sub>B</sub></em> = 0.5, the mean fraction is <em>F<sub>NN</sub></em> = 1/6. This fraction is assessed relative to the complete set of points <em>N</em> = <em>N<sub>A</sub></em> + <em>N<sub>B</sub></em>, i.e. the mean number of pairs of the nearest neighbors is determined as <em>N</em>·<em>F<sub>NN</sub></em>. The significant bias against frequency (26) is interesting for assessment of trends in natural selection and large-scale genome organization.</div>                            <h2 scrollto-destination=6615585 id="6615585" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6615586 id="6615586" class="section-title" data-legacy-id=s3a>3.1. Tests and comparison of different packages</h3>
<p>The distribution of genome tracks over the genome is often strongly inhomogeneous. As a typical example, we chose the exons on the forward and reverse strands of 23 human chromosomes (the Y-chromosome was discarded due to insufficient data). Both the Kolmogorov–Smirnov and entropy criteria revealed strong inhomogeneity of this set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S1</a>). The positions of points in any random set are independent of that of exons and should be uncorrelated with them. This test was used to verify the predictions in different packages. For a particular MC realization, the <em>P</em>-value characterizing the significance of correlations between exons and a random set was calculated for each chromosome separately. The correlations were assumed to be significant if <em>P</em> &lt; 0.05. Then, we calculated the observed fraction of events with predicted <em>P</em> &lt; 0.05 per 100 MC realizations (false discovery rate, FDR). As neither Genomic HyperBrowser<sup><a reveal-id="DSU044C12" data-open="DSU044C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU044C13" data-open="DSU044C13" class="link link-ref link-reveal xref-bibr">13</a></sup> nor GeometriCorr<sup><a reveal-id="DSU044C14" data-open="DSU044C14" class="link link-ref link-reveal xref-bibr">14</a></sup> suggests the criteria for the comparison of the inhomogeneity of two sets, we performed the blind computations in which the attribution of two sets was unknown.</p><p>Statistical analysis of correlations between genome tracks with Genomic HyperBrowser was performed to test the hypothesis: point–point located nearby? The correlations in this package were assessed via MC simulations, in which one of the sets was preserved, whereas the other was replaced by MC samples. The maximal number of MC samples was 1,000. To create single-line commands suitable for batch processing, we used Perl scripts written by ourselves. GenometriCorr used the combination of the Kolmogorov–Smirnov test for the normalized lengths together with a permutation test for distances between the points of two sets. In GenometriCorr, the user should manually set the extreme 5′- and 3′-ends of the data range to define the limits of the chromosome region under study for correct correlations. This option is called ‘mapping’ and is turned off by default. For setting mapping, we used an R script written by ourselves. The number of permutations for the assessment of absolute distance <em>P</em>-value was 1,000. For our package, we used the united Gaussian criterion (23) with attribution of sets by the entropy <em>z</em> criterion (10). The results are compared in Table <a reveal-id="DSU044TB1" data-open="DSU044TB1" class="link link-reveal link-table xref-fig">1</a>. </p>                        <a id="6615589" scrollto-destination="6615589"></a>
<div class="table-wrap"><div id="DSU044TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>FDR for the tests on the absence of correlations between exons in human chromosomes and random sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Number of points in random sets relative to that of exons </th><th colspan="6">Fraction of events with <em>P</em> &lt; 0.05 per 100 MC realizations<hr /> </th></tr><tr><th colspan="3">Exons on forward strand<hr /> </th><th colspan="3">Exons on reverse strand<hr /> </th></tr><tr><th>2-Fold less </th><th>Equal, ±10% </th><th>2-Fold more </th><th>2-Fold less </th><th>Equal, ±10% </th><th>2-Fold more </th></tr></thead><tbody><tr><td colspan="6">Genomic HyperBrowser<sup><a reveal-id="DSU044C12" data-open="DSU044C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU044C13" data-open="DSU044C13" class="link link-ref link-reveal xref-bibr">13</a></sup> </td><td> </td></tr><tr><td><em>P</em>-value, exon positions preserved, random set MC randomized </td><td>0.253 </td><td>0.253 </td><td>0.247 </td><td>0.249 </td><td>0.250 </td><td>0.248 </td></tr><tr><td>Adjusted <em>P</em>-value, exon positions preserved, random set MC randomized </td><td>0.227 </td><td>0.228 </td><td>0.227 </td><td>0.227 </td><td>0.231 </td><td>0.228 </td></tr><tr><td><em>P</em>-value, random set preserved, exon positions MC randomized </td><td>0.538 </td><td>0.597 </td><td>0.670 </td><td>0.588 </td><td>0.635 </td><td>0.773 </td></tr><tr><td>Adjusted <em>P</em>-value, random set preserved, exon positions MC randomized </td><td>0.523 </td><td>0.582 </td><td>0.656 </td><td>0.573 </td><td>0.628 </td><td>0.768 </td></tr><tr><td colspan="6">GenometriCorr<sup><a reveal-id="DSU044C14" data-open="DSU044C14" class="link link-ref link-reveal xref-bibr">14</a></sup> </td><td> </td></tr><tr><td>Relative distance KS <em>P</em>-value, exons—reference; random—query </td><td>0.047 </td><td>0.049 </td><td>0.051 </td><td>0.051 </td><td>0.048 </td><td>0.050 </td></tr><tr><td>Absolute distance <em>P</em>-value, exons—reference; random—query </td><td>0.109 </td><td>0.106 </td><td>0.100 </td><td>0.110 </td><td>0.107 </td><td>0.109 </td></tr><tr><td>Relative distance KS <em>P</em>-value, random—reference; exons—query </td><td>0.537 </td><td>0.437 </td><td>0.428 </td><td>0.579 </td><td>0.450 </td><td>0.215 </td></tr><tr><td>Absolute distance <em>P</em>-value, random—reference; exons—query </td><td>0.705 </td><td>0.697 </td><td>0.662 </td><td>0.725 </td><td>0.693 </td><td>0.599 </td></tr><tr><td colspan="6">Genome Track Analyzer </td><td> </td></tr><tr><td>United <em>z</em>-criterion, Equation (23) </td><td>0.056 </td><td>0.052 </td><td>0.051 </td><td>0.044 </td><td>0.046 </td><td>0.048 </td></tr></tbody></table></div><div content-id="DSU044TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Number of points in random sets relative to that of exons </th><th colspan="6">Fraction of events with <em>P</em> &lt; 0.05 per 100 MC realizations<hr /> </th></tr><tr><th colspan="3">Exons on forward strand<hr /> </th><th colspan="3">Exons on reverse strand<hr /> </th></tr><tr><th>2-Fold less </th><th>Equal, ±10% </th><th>2-Fold more </th><th>2-Fold less </th><th>Equal, ±10% </th><th>2-Fold more </th></tr></thead><tbody><tr><td colspan="6">Genomic HyperBrowser<sup><a reveal-id="DSU044C12" data-open="DSU044C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU044C13" data-open="DSU044C13" class="link link-ref link-reveal xref-bibr">13</a></sup> </td><td> </td></tr><tr><td><em>P</em>-value, exon positions preserved, random set MC randomized </td><td>0.253 </td><td>0.253 </td><td>0.247 </td><td>0.249 </td><td>0.250 </td><td>0.248 </td></tr><tr><td>Adjusted <em>P</em>-value, exon positions preserved, random set MC randomized </td><td>0.227 </td><td>0.228 </td><td>0.227 </td><td>0.227 </td><td>0.231 </td><td>0.228 </td></tr><tr><td><em>P</em>-value, random set preserved, exon positions MC randomized </td><td>0.538 </td><td>0.597 </td><td>0.670 </td><td>0.588 </td><td>0.635 </td><td>0.773 </td></tr><tr><td>Adjusted <em>P</em>-value, random set preserved, exon positions MC randomized </td><td>0.523 </td><td>0.582 </td><td>0.656 </td><td>0.573 </td><td>0.628 </td><td>0.768 </td></tr><tr><td colspan="6">GenometriCorr<sup><a reveal-id="DSU044C14" data-open="DSU044C14" class="link link-ref link-reveal xref-bibr">14</a></sup> </td><td> </td></tr><tr><td>Relative distance KS <em>P</em>-value, exons—reference; random—query </td><td>0.047 </td><td>0.049 </td><td>0.051 </td><td>0.051 </td><td>0.048 </td><td>0.050 </td></tr><tr><td>Absolute distance <em>P</em>-value, exons—reference; random—query </td><td>0.109 </td><td>0.106 </td><td>0.100 </td><td>0.110 </td><td>0.107 </td><td>0.109 </td></tr><tr><td>Relative distance KS <em>P</em>-value, random—reference; exons—query </td><td>0.537 </td><td>0.437 </td><td>0.428 </td><td>0.579 </td><td>0.450 </td><td>0.215 </td></tr><tr><td>Absolute distance <em>P</em>-value, random—reference; exons—query </td><td>0.705 </td><td>0.697 </td><td>0.662 </td><td>0.725 </td><td>0.693 </td><td>0.599 </td></tr><tr><td colspan="6">Genome Track Analyzer </td><td> </td></tr><tr><td>United <em>z</em>-criterion, Equation (23) </td><td>0.056 </td><td>0.052 </td><td>0.051 </td><td>0.044 </td><td>0.046 </td><td>0.048 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>All tests were performed with predicted <em>P</em>-values &lt;0.05. The expected mean value and standard deviation for FDR per 100 MC realizations should be 0.05 ± 0.005.</p></div></div><div content-id="TN2"><div class="fn"><p>FDR, false discovery rate; MC, Monte Carlo; KS, Kolmogorov–Smirnov test.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6615589" target="_blank">View Large</a></div></div><p>The results in Table <a reveal-id="DSU044TB1" data-open="DSU044TB1" class="link link-reveal link-table xref-fig">1</a> reveal strong dependence of the FDR on the attribution of sets. If exon positions are preserved, while the random set is MC randomized, the Genomic HyperBrowser provides the FDR about five times higher than the expected rate. GenometriCorr provides the correct rate for the relative distance KS <em>P</em>-value and the correspondence exons—reference, random—query, whereas the usage of the absolute distance <em>P</em>-value provides the FDR ∼2-fold higher than the expected rate at the same correspondence. The permutation in set attribution (i.e. the random set is preserved, while exon positions are MC randomized for the Genomic HyperBrowser and random—reference, exons—query for GenometriCorr) leads to the complete failure of the correlation prediction in both packages. This proves that the preliminary assessment of randomness/inhomogeneity is crucial for the correct statistical predictions.</p><p>The computation time needed for one random realization in Genomic HyperBrowser was 1 h 35 min, in GenometriCorr 52 min, and in Genome Track Analyzer 9 s. This time included the duration of loading and processing of input data, direct computations, and processing of output data. In particular, for our package, the time needed for the loading and processing of input data was 8 s, and the processing of output data needed 1 s, whereas the direct computations took only 0.01 s.</p>                            <h3 scrollto-destination=6615592 id="6615592" class="section-title" data-legacy-id=s3b>3.2. Correlations between CpG islands and TSS</h3>
<p>The software package was applied to the analysis of particular genetic problems. The first one concerned the correlations between CpG islands and transcription start sites.<sup><a reveal-id="DSU044C5" data-open="DSU044C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSU044C22  DSU044C24" data-open="DSU044C22  DSU044C24" class="link link-ref link-reveal xref-bibr">22–24</a></sup> The positions and overlapping of CpG islands with TSS affect gene expression in eukaryotes. The data on CpG islands on human chromosomes were taken from URL <a class="link link-uri" href="http://rafalab.jhsph.edu/CGI/">http://rafalab.jhsph.edu/CGI/</a>,<sup><a reveal-id="DSU044C5" data-open="DSU044C5" class="link link-ref link-reveal xref-bibr">5</a></sup> whereas the data on TSS were taken from the Eukaryotic Promoter Database (<a class="link link-uri" href="http://epd.vital-it.ch/">http://epd.vital-it.ch/</a><em>H. Sapiens</em> hg19 version EPDnew 003).<sup><a reveal-id="DSU044C3" data-open="DSU044C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Our method supported the strong correlations between CpG islands and TSS with clear preceding ordering between 5′-ends of CpG islands and TSS. The study also showed overlap between CpG island and the nearest TSS (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S2</a>).</p>                            <h3 scrollto-destination=6615594 id="6615594" class="section-title" data-legacy-id=s3c>3.3. Correlations between TSS on the forward and reverse strands</h3>
<p>Additionally, we found significant correlations between TSS on the forward and reverse strands of <em>D. melanogaster</em>, <em>Caenorhabditis elegans</em>, <em>Mus musculus</em>, <em>H. sapiens</em>, and <em>Danio rerio</em> (zebrafish) genomes. The positions of TSS on the reverse strand were projected to the forward strand. The detailed results are presented for <em>D. melanogaster</em> and <em>H. sapiens</em>. In Table <a reveal-id="DSU044TB2" data-open="DSU044TB2" class="link link-reveal link-table xref-fig">2</a>, we included only the chromosomes in which the number of nearest neighbours exceeded 50 to ensure the applicability of Gaussian statistics. These data show clear positional and ordering correlations for the significant fraction of the TSS on two strands. Implying that expression regulation factors are upstream of TSS, the bias in projected TSS on the reverse strand preceding neighbouring TSS on the forward strand indicates that some regulatory elements such as CpG islands, enhancers, silencers, etc., may be common. Such a mode of regulation expression has, in fact, been experimentally established.<sup><a reveal-id="DSU044C25" data-open="DSU044C25" class="link link-ref link-reveal xref-bibr">25</a>–<a reveal-id="DSU044C27" data-open="DSU044C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The statistical significance of these correlations versus the counterpart random model proves the positive selection of such a mode of expression regulation during evolution. The fraction of the nearest neighbours for TSS turned out to be less than the theoretical prediction for the random sets with the same fractions of points [Equation (26)] for the <em>H. sapiens</em> genome and rather close to the theoretical prediction for the <em>D. melanogaster</em> genome (Table <a reveal-id="DSU044TB2" data-open="DSU044TB2" class="link link-reveal link-table xref-fig">2</a>). The histograms for the relative and absolute lengths between neighbouring TSS, scattering plot, and the list of the top nearest TSS on the reverse and forward strands are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S3</a>. The histograms for both <em>H. sapiens</em> and <em>D. melanogaster</em> genomes show the length distributions peaked at the small distances between neighboring TSS in accordance with significant positional correlations. The histograms for the absolute lengths revealed a sharp drop at the distances exceeding the characteristic nucleosome length of ∼200 bp. Though the absolute and relative lengths between neighbouring TSS were significantly correlated (Spearman correlation coefficients exceeded 0.7, <em>P</em> &lt; 0.001), only the criteria formulated in terms of relative lengths proved to be robust against the inhomogeneity of genome tracks. The distribution of the nearest neighbouring TSS on the forward and reverse strands across particular chromosomes of the <em>H. sapiens</em> genome is shown in Fig. <a reveal-id="DSU044F1" data-open="DSU044F1" class="link link-reveal link-table xref-fig">1</a>a, which was drawn with the Integrated Genome Browser.<sup><a reveal-id="DSU044C28" data-open="DSU044C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The distributions of the TSS pairs across all chromosomes of the <em>H. sapiens</em> genome listed in Table <a reveal-id="DSU044TB2" data-open="DSU044TB2" class="link link-reveal link-table xref-fig">2</a> can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S4</a>. </p>                        <a id="6615596" scrollto-destination="6615596"></a>
<div class="table-wrap"><div id="DSU044TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Correlations between transcription start sites on the forward and reverse strands</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th><em>z</em> </th><th><em>P</em> </th><th><em>z<sub>P</sub></em> </th><th><em>P</em> </th><th>TSS forward </th><th>TSS reverse </th><th>NN </th></tr></thead><tbody><tr><td colspan="8">A. <em>Homo sapiens</em> genome </td></tr><tr><td>chr1 </td><td>2.59 </td><td>0.010 </td><td>5.93 </td><td>&lt;0.001 </td><td>1,226 </td><td>1,156 </td><td>230 </td></tr><tr><td>chr2 </td><td>3.77 </td><td>&lt;0.001 </td><td>4.28 </td><td>&lt;0.001 </td><td>817 </td><td>728 </td><td>177 </td></tr><tr><td>chr3 </td><td>3.01 </td><td>0.003 </td><td>4.60 </td><td>&lt;0.001 </td><td>643 </td><td>663 </td><td>133 </td></tr><tr><td>chr4 </td><td>0.28 </td><td>0.779 </td><td>2.07 </td><td>0.038 </td><td>446 </td><td>471 </td><td>103 </td></tr><tr><td>chr5 </td><td>4.39 </td><td>&lt;0.001 </td><td>4.42 </td><td>&lt;0.001 </td><td>570 </td><td>497 </td><td>113 </td></tr><tr><td>chr6 </td><td>4.34 </td><td>&lt;0.001 </td><td>5.22 </td><td>&lt;0.001 </td><td>609 </td><td>617 </td><td>132 </td></tr><tr><td>chr7 </td><td>0.65 </td><td>0.516 </td><td>2.97 </td><td>0.003 </td><td>562 </td><td>509 </td><td>109 </td></tr><tr><td>chr8 </td><td>2.76 </td><td>0.006 </td><td>4.48 </td><td>&lt;0.001 </td><td>397 </td><td>423 </td><td>72 </td></tr><tr><td>chr9 </td><td>2.97 </td><td>0.003 </td><td>3.39 </td><td>&lt;0.001 </td><td>443 </td><td>507 </td><td>95 </td></tr><tr><td>chr10 </td><td>1.97 </td><td>0.049 </td><td>4.27 </td><td>&lt;0.001 </td><td>471 </td><td>475 </td><td>106 </td></tr><tr><td>chr11 </td><td>3.52 </td><td>&lt;0.001 </td><td>5.02 </td><td>&lt;0.001 </td><td>696 </td><td>656 </td><td>129 </td></tr><tr><td>chr12 </td><td>4.21 </td><td>&lt;0.001 </td><td>5.36 </td><td>&lt;0.001 </td><td>623 </td><td>656 </td><td>147 </td></tr><tr><td>chr14 </td><td>4.28 </td><td>&lt;0.001 </td><td>4.64 </td><td>&lt;0.001 </td><td>382 </td><td>363 </td><td>90 </td></tr><tr><td>chr15 </td><td>1.94 </td><td>0.052 </td><td>5.74 </td><td>&lt;0.001 </td><td>338 </td><td>359 </td><td>82 </td></tr><tr><td>chr16 </td><td>3.73 </td><td>&lt;0.001 </td><td>6.30 </td><td>&lt;0.001 </td><td>571 </td><td>422 </td><td>113 </td></tr><tr><td>chr17 </td><td>5.08 </td><td>&lt;0.001 </td><td>6.37 </td><td>&lt;0.001 </td><td>633 </td><td>725 </td><td>150 </td></tr><tr><td>chr19 </td><td>4.28 </td><td>&lt;0.001 </td><td>7.65 </td><td>&lt;0.001 </td><td>823 </td><td>772 </td><td>178 </td></tr><tr><td>chr20 </td><td>2.57 </td><td>0.010 </td><td>4.06 </td><td>&lt;0.001 </td><td>320 </td><td>291 </td><td>70 </td></tr><tr><td>chr22 </td><td>1.77 </td><td>0.077 </td><td>4.16 </td><td>&lt;0.001 </td><td>268 </td><td>269 </td><td>63 </td></tr><tr><td>chrX </td><td>2.24 </td><td>0.025 </td><td>1.43 </td><td>0.153 </td><td>477 </td><td>453 </td><td>96 </td></tr><tr><td colspan="8">B. <em>Drosophila melanogaster</em> genome </td></tr><tr><td>chr2L </td><td>11.98 </td><td>&lt;0.001 </td><td>15.68 </td><td>&lt;0.001 </td><td>1,393 </td><td>1,379 </td><td>454 </td></tr><tr><td>chr2R </td><td>13.72 </td><td>&lt;0.001 </td><td>14.73 </td><td>&lt;0.001 </td><td>1,578 </td><td>1,529 </td><td>481 </td></tr><tr><td>chr3L </td><td>12.57 </td><td>&lt;0.001 </td><td>15.59 </td><td>&lt;0.001 </td><td>1,528 </td><td>1,570 </td><td>473 </td></tr><tr><td>chr3R </td><td>14.19 </td><td>&lt;0.001 </td><td>16.61 </td><td>&lt;0.001 </td><td>1,762 </td><td>1,852 </td><td>583 </td></tr><tr><td>chrX </td><td>12.42 </td><td>&lt;0.001 </td><td>14.08 </td><td>&lt;0.001 </td><td>1,176 </td><td>1,198 </td><td>387 </td></tr></tbody></table></div><div content-id="DSU044TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th><em>z</em> </th><th><em>P</em> </th><th><em>z<sub>P</sub></em> </th><th><em>P</em> </th><th>TSS forward </th><th>TSS reverse </th><th>NN </th></tr></thead><tbody><tr><td colspan="8">A. <em>Homo sapiens</em> genome </td></tr><tr><td>chr1 </td><td>2.59 </td><td>0.010 </td><td>5.93 </td><td>&lt;0.001 </td><td>1,226 </td><td>1,156 </td><td>230 </td></tr><tr><td>chr2 </td><td>3.77 </td><td>&lt;0.001 </td><td>4.28 </td><td>&lt;0.001 </td><td>817 </td><td>728 </td><td>177 </td></tr><tr><td>chr3 </td><td>3.01 </td><td>0.003 </td><td>4.60 </td><td>&lt;0.001 </td><td>643 </td><td>663 </td><td>133 </td></tr><tr><td>chr4 </td><td>0.28 </td><td>0.779 </td><td>2.07 </td><td>0.038 </td><td>446 </td><td>471 </td><td>103 </td></tr><tr><td>chr5 </td><td>4.39 </td><td>&lt;0.001 </td><td>4.42 </td><td>&lt;0.001 </td><td>570 </td><td>497 </td><td>113 </td></tr><tr><td>chr6 </td><td>4.34 </td><td>&lt;0.001 </td><td>5.22 </td><td>&lt;0.001 </td><td>609 </td><td>617 </td><td>132 </td></tr><tr><td>chr7 </td><td>0.65 </td><td>0.516 </td><td>2.97 </td><td>0.003 </td><td>562 </td><td>509 </td><td>109 </td></tr><tr><td>chr8 </td><td>2.76 </td><td>0.006 </td><td>4.48 </td><td>&lt;0.001 </td><td>397 </td><td>423 </td><td>72 </td></tr><tr><td>chr9 </td><td>2.97 </td><td>0.003 </td><td>3.39 </td><td>&lt;0.001 </td><td>443 </td><td>507 </td><td>95 </td></tr><tr><td>chr10 </td><td>1.97 </td><td>0.049 </td><td>4.27 </td><td>&lt;0.001 </td><td>471 </td><td>475 </td><td>106 </td></tr><tr><td>chr11 </td><td>3.52 </td><td>&lt;0.001 </td><td>5.02 </td><td>&lt;0.001 </td><td>696 </td><td>656 </td><td>129 </td></tr><tr><td>chr12 </td><td>4.21 </td><td>&lt;0.001 </td><td>5.36 </td><td>&lt;0.001 </td><td>623 </td><td>656 </td><td>147 </td></tr><tr><td>chr14 </td><td>4.28 </td><td>&lt;0.001 </td><td>4.64 </td><td>&lt;0.001 </td><td>382 </td><td>363 </td><td>90 </td></tr><tr><td>chr15 </td><td>1.94 </td><td>0.052 </td><td>5.74 </td><td>&lt;0.001 </td><td>338 </td><td>359 </td><td>82 </td></tr><tr><td>chr16 </td><td>3.73 </td><td>&lt;0.001 </td><td>6.30 </td><td>&lt;0.001 </td><td>571 </td><td>422 </td><td>113 </td></tr><tr><td>chr17 </td><td>5.08 </td><td>&lt;0.001 </td><td>6.37 </td><td>&lt;0.001 </td><td>633 </td><td>725 </td><td>150 </td></tr><tr><td>chr19 </td><td>4.28 </td><td>&lt;0.001 </td><td>7.65 </td><td>&lt;0.001 </td><td>823 </td><td>772 </td><td>178 </td></tr><tr><td>chr20 </td><td>2.57 </td><td>0.010 </td><td>4.06 </td><td>&lt;0.001 </td><td>320 </td><td>291 </td><td>70 </td></tr><tr><td>chr22 </td><td>1.77 </td><td>0.077 </td><td>4.16 </td><td>&lt;0.001 </td><td>268 </td><td>269 </td><td>63 </td></tr><tr><td>chrX </td><td>2.24 </td><td>0.025 </td><td>1.43 </td><td>0.153 </td><td>477 </td><td>453 </td><td>96 </td></tr><tr><td colspan="8">B. <em>Drosophila melanogaster</em> genome </td></tr><tr><td>chr2L </td><td>11.98 </td><td>&lt;0.001 </td><td>15.68 </td><td>&lt;0.001 </td><td>1,393 </td><td>1,379 </td><td>454 </td></tr><tr><td>chr2R </td><td>13.72 </td><td>&lt;0.001 </td><td>14.73 </td><td>&lt;0.001 </td><td>1,578 </td><td>1,529 </td><td>481 </td></tr><tr><td>chr3L </td><td>12.57 </td><td>&lt;0.001 </td><td>15.59 </td><td>&lt;0.001 </td><td>1,528 </td><td>1,570 </td><td>473 </td></tr><tr><td>chr3R </td><td>14.19 </td><td>&lt;0.001 </td><td>16.61 </td><td>&lt;0.001 </td><td>1,762 </td><td>1,852 </td><td>583 </td></tr><tr><td>chrX </td><td>12.42 </td><td>&lt;0.001 </td><td>14.08 </td><td>&lt;0.001 </td><td>1,176 </td><td>1,198 </td><td>387 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>The positions of TSS on the reverse strand were projected to the forward strand. <em>z</em> and <em>z<sub>P</sub></em> are calculated by Equations (23) and (11) and characterize the positional and ordering correlations between TSS, respectively. The 1% significance thresholds for |<em>z</em>| and |<em>z<sub>P</sub></em>| in the case of random correlations correspond to 2.58, while 5% significance thresholds correspond to 1.96. The positive values of <em>z<sub>P</sub></em> indicate that projected TSS on the reverse strand precede TSS on the forward strand. The corresponding <em>P</em>-values were calculated using Gaussian statistics. The data were filtered by the number of pairs of the nearest neighbours (NN) exceeding 50 to ensure the applicability of Gaussian statistics.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6615596" target="_blank">View Large</a></div></div>                        <a id="6615597" scrollto-destination="6615597"></a>
<div id="DSU044F1" data-id="DSU044F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/m_dsu04401.png?Expires=1509238989&Signature=J8zHbjub2DF3aoYGAnQP5if5llA7aCIFri69nyezBNIfebO4a-XCW3~h8v-Mty-6jO0ATpJwqb4FyxnsZM88B0yFo1QxqCZhWiCnbU81naGyNLJ9zeqyJFwnsepBJK-Qj~yvN4DqHiP5XNofcYV~gUXCOjQ99oLgnsZCKX0Qw78D3kADOiTKFna3SvOEacJ3d3OLauYbkoKjiTDJU2vex10C0AksmMC~fXdAgyRpqww1dSgYIZNC6qD9Qgg7VAk4qYY~2jSpyNLIJy5z0fgcu-bBaM4~yJTAaU3o2HW-Fbl~D1nGUI6WJmxmSN-~mxmBl3D-oxSsBBrXbj3aNOSvQg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(a) The distributions of the nearest neighbouring transcription start sites (NN TSS) on the forward and reverse strands across particular chromosomes of the Homo sapiens genome. The cytobands across corresponding chromosomes and relevant length scales are shown below the TSS. The length scale is in megabases. The blue vertical lines correspond to the pairs of NN TSS. The 15 closest pairs on each chromosome are marked by the red lines, and the names of the corresponding NN TSS are indicated. Names shown above the red lines correspond to the TSS on the forward strand, whereas names shown below the red line correspond to the TSS on the reverse strand (names are given according to EPD notation). (b) Particular examples of NN TSS pairs in the H. sapiens genome. The transcriptions factors (TF) participating in the regulation of expression of a particular gene are listed after the name of the gene. The TF that match genes on both strands are marked in red. The data on binding sites for TF associated with genes were taken from http://www.genecards.org." path-from-xml="dsu04401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6615597/dsu04401.tiff" path-from-xml="dsu04401.tiff" target="_blank">View large</a><a section="6615597" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu04401.png?Expires=1509238989&Signature=KSXCm5BaT3Kw12lydtyfRaWzYxF6BIy3T0Ir70W8oYXsfLoOmxiGhVSk2ljQJfEWHPtF6woouy8nXeJq6y1KyO0qBvS~iU5JmUKk7tK7EwcjLt1hfxTxmi0LfR~5rKk-ZXiPWHSo9tH-~aTa7YJz6JOh4S9ZPQK~PUFnGxS~jASQwtgYsLpEy5k2ICE-6VeY6ejflOyLPKgJeC03SmOgFjQWtafad8rVx3BO0Fq0tObOw7dQNHiWChz4h99u6-ie104CN6h543NG6R7ly7CMHkNioKy6naCbbiceW81mXBwCfk32qMDxl6YTmdS34Y1j5kTsQIVRGPM0coziLBEsAw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6615597&ar=442245&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The distributions of the nearest neighbouring transcription start sites (NN TSS) on the forward and reverse strands across particular chromosomes of the <em>Homo sapiens</em> genome. The cytobands across corresponding chromosomes and relevant length scales are shown below the TSS. The length scale is in megabases. The blue vertical lines correspond to the pairs of NN TSS. The 15 closest pairs on each chromosome are marked by the red lines, and the names of the corresponding NN TSS are indicated. Names shown above the red lines correspond to the TSS on the forward strand, whereas names shown below the red line correspond to the TSS on the reverse strand (names are given according to EPD notation). (b) Particular examples of NN TSS pairs in the <em>H. sapiens</em> genome. The transcriptions factors (TF) participating in the regulation of expression of a particular gene are listed after the name of the gene. The TF that match genes on both strands are marked in red. The data on binding sites for TF associated with genes were taken from <a class="link link-uri" href="http://www.genecards.org">http://www.genecards.org</a>.</p></div></div><div content-id="DSU044F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/m_dsu04401.png?Expires=1509238989&Signature=J8zHbjub2DF3aoYGAnQP5if5llA7aCIFri69nyezBNIfebO4a-XCW3~h8v-Mty-6jO0ATpJwqb4FyxnsZM88B0yFo1QxqCZhWiCnbU81naGyNLJ9zeqyJFwnsepBJK-Qj~yvN4DqHiP5XNofcYV~gUXCOjQ99oLgnsZCKX0Qw78D3kADOiTKFna3SvOEacJ3d3OLauYbkoKjiTDJU2vex10C0AksmMC~fXdAgyRpqww1dSgYIZNC6qD9Qgg7VAk4qYY~2jSpyNLIJy5z0fgcu-bBaM4~yJTAaU3o2HW-Fbl~D1nGUI6WJmxmSN-~mxmBl3D-oxSsBBrXbj3aNOSvQg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(a) The distributions of the nearest neighbouring transcription start sites (NN TSS) on the forward and reverse strands across particular chromosomes of the Homo sapiens genome. The cytobands across corresponding chromosomes and relevant length scales are shown below the TSS. The length scale is in megabases. The blue vertical lines correspond to the pairs of NN TSS. The 15 closest pairs on each chromosome are marked by the red lines, and the names of the corresponding NN TSS are indicated. Names shown above the red lines correspond to the TSS on the forward strand, whereas names shown below the red line correspond to the TSS on the reverse strand (names are given according to EPD notation). (b) Particular examples of NN TSS pairs in the H. sapiens genome. The transcriptions factors (TF) participating in the regulation of expression of a particular gene are listed after the name of the gene. The TF that match genes on both strands are marked in red. The data on binding sites for TF associated with genes were taken from http://www.genecards.org." path-from-xml="dsu04401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6615597/dsu04401.tiff" path-from-xml="dsu04401.tiff" target="_blank">View large</a><a section="6615597" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu04401.png?Expires=1509238989&Signature=KSXCm5BaT3Kw12lydtyfRaWzYxF6BIy3T0Ir70W8oYXsfLoOmxiGhVSk2ljQJfEWHPtF6woouy8nXeJq6y1KyO0qBvS~iU5JmUKk7tK7EwcjLt1hfxTxmi0LfR~5rKk-ZXiPWHSo9tH-~aTa7YJz6JOh4S9ZPQK~PUFnGxS~jASQwtgYsLpEy5k2ICE-6VeY6ejflOyLPKgJeC03SmOgFjQWtafad8rVx3BO0Fq0tObOw7dQNHiWChz4h99u6-ie104CN6h543NG6R7ly7CMHkNioKy6naCbbiceW81mXBwCfk32qMDxl6YTmdS34Y1j5kTsQIVRGPM0coziLBEsAw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6615597&ar=442245&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The distributions of the nearest neighbouring transcription start sites (NN TSS) on the forward and reverse strands across particular chromosomes of the <em>Homo sapiens</em> genome. The cytobands across corresponding chromosomes and relevant length scales are shown below the TSS. The length scale is in megabases. The blue vertical lines correspond to the pairs of NN TSS. The 15 closest pairs on each chromosome are marked by the red lines, and the names of the corresponding NN TSS are indicated. Names shown above the red lines correspond to the TSS on the forward strand, whereas names shown below the red line correspond to the TSS on the reverse strand (names are given according to EPD notation). (b) Particular examples of NN TSS pairs in the <em>H. sapiens</em> genome. The transcriptions factors (TF) participating in the regulation of expression of a particular gene are listed after the name of the gene. The TF that match genes on both strands are marked in red. The data on binding sites for TF associated with genes were taken from <a class="link link-uri" href="http://www.genecards.org">http://www.genecards.org</a>.</p></div></div><p>The analysis of some nearest TSS duplets revealed that they correspond to rather short intergenic regions between pairs of closely located genes that reside on different strands. They may have mutual enhancers or bidirectional promoters, and thus should correspond to genes with coordinated expression. The particular examples are shown in Fig. <a reveal-id="DSU044F1" data-open="DSU044F1" class="link link-reveal link-table xref-fig">1</a>b. Currently, we are analysing these gene pairs to determine the role of this type of genome organization for coordinated regulation of genes in the human genome.</p>                            <h3 scrollto-destination=6615599 id="6615599" class="section-title" data-legacy-id=s3d>3.4. Correlations between protein-binding profiles</h3>
<p>The second problem concerned the relationships between profiles characterizing DNA binding with proteins E(Z), Pc-S2, and Psc, and H3me3K27 marks in the chromosomes of <em>D. melanogaster</em>. E(Z), Pc-S2, and Psc belong to the polycomb group (PcG) of proteins, which are important for maintaining the transcriptional repression of homeotic genes.<sup><a reveal-id="DSU044C29" data-open="DSU044C29" class="link link-ref link-reveal xref-bibr">29</a>–<a reveal-id="DSU044C32" data-open="DSU044C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The corresponding processed and aggregated profiles were obtained by Schwartz <em>et al.</em><sup><a reveal-id="DSU044C29" data-open="DSU044C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and were taken from EMBL ArrayExpress accession E-MEXP-535.<sup><a reveal-id="DSU044C33" data-open="DSU044C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The profiles were preliminarily filtered by the cut-off threshold mean + 2 SD and clustered with a distance in the range 50–500 nt with steps of 50 nt (Preprocessing of input genetic data). The data before and after preprocessing with a clustering distance of 50 nt are shown in Fig. <a reveal-id="DSU044F2" data-open="DSU044F2" class="link link-reveal link-table xref-fig">2</a>a. The corresponding <em>z</em>-ratios [Equation (23)] indicate strong correlations between binding profiles for the PcG proteins and for H3me3K27 marks (Fig. <a reveal-id="DSU044F2" data-open="DSU044F2" class="link link-reveal link-table xref-fig">2</a>b). The correlations strongly depend on the clustering distance (Fig. <a reveal-id="DSU044F2" data-open="DSU044F2" class="link link-reveal link-table xref-fig">2</a>c and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S5</a>). As the characteristic binding region for the PcG proteins is ∼50 nt,<sup><a reveal-id="DSU044C34" data-open="DSU044C34" class="link link-ref link-reveal xref-bibr">34</a></sup> the clustering distance of 50 nt may be considered as optimal in this example. The cut-off threshold affects the number of nearest neighbours, but it retains the mode of correlations at a given clustering distance. These observations are in accordance with the data on the coordinate action of the E(Z), Pc-S2, and Psc proteins, and the H3me3K27 marks in the silencing mechanisms for <em>D. melanogaster.</em><sup><a reveal-id="DSU044C29" data-open="DSU044C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSU044C30" data-open="DSU044C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                        <a id="6615601" scrollto-destination="6615601"></a>
<div id="DSU044F2" data-id="DSU044F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/m_dsu04402.png?Expires=1509238989&Signature=fhF2oyqNIe-sZ~5nSg9G1nYpIt0b8tCf3yT31hbK~TdpPKzIK7kNTxs8qK33hYOejtmIl3qbro2Q8tEP~71dFGXz35Co15oWif1v5Zn7bEpNtZqs-y2CnW8HYFZDhZwSqT8U3ttFwmnJsOYcu8T-VSWD7x7RSVPJvatEcrN3RDvOcq0-w5n2-rAN5KVq6SSYXxjcc6TqUyyYC3hRaScWxp1uMJvoF54rURZ11v--K~jQuxUoLRBCIWKrArps1sErC6wreXPKTb2p6UpJfhR~0BFYZpFx8haSu3afXtUbUr-70NyIErQEvIoM0gwzDh5KnTXlWOWMievR4NPak5NrYQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(a) The binding profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks over chromosome 3R of Drosophila melanogaster. For the study of correlations, these profiles were preliminary filtered by the cut-off threshold mean + 2 SD and clustered with distance of 50 nt [Preprocessing of input genetic data and Equation (12)]. The input data after preprocessing are shown below initial profiles. (b) z-ratios [Equation (23)] characterizing pairwise positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for the H3me3K27 mark in the different chromosomes of D. melanogaster. The input data were preprocessed as described above. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for z-ratios correspond to 5% (|z| = 1.96) and 1% (|z| = 2.58) significance thresholds for random correlations. (c) Ratios characterizing positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks in the chromosome 2R of D. melanogaster at the different clustering lengths. The profiles were preliminary filtered by the cut-off threshold mean + 2 SD. The positive values of zcorr reflect a trend towards shorter distances between profiles relative to the reference model (or correlations), whereas the negative values of zcorr reflect a trend towards longer distances between profiles (or anticorrelations)." path-from-xml="dsu04402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6615601/dsu04402.tiff" path-from-xml="dsu04402.tiff" target="_blank">View large</a><a section="6615601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu04402.png?Expires=1509238989&Signature=JVGVFttcJPQ9tZdOMOzhM1PIMojtCnux1v8veib-ScBxX8rCAozOtedr4JwWg8cBvrl2IC41vsrx2N~KxInXM5EUL0z8LZikt0HxoBm-n0TOFwuxKMcP1bsBndkBjGbyUFbKEKxNsHaVjDKMUHZzC2BeLfPxabY9ZcMl34H4jPn22exbhgWusJW2lWrQTuFsqdOS7KuWhoTWkahVKnyFzGCQnQLC-ga~aODnMpg2H341P8LlQEea3r7hihFQou4S1OpukMy5KtPj8tqpYyzjeo0oi~WkvSx3FEb-Ni813kQ91sVbNwe26DDSa6M1cnNqEMA4nBpIJLGs4wtEK2mb7w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6615601&ar=442245&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The binding profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks over chromosome 3R of <em>Drosophila melanogaster</em>. For the study of correlations, these profiles were preliminary filtered by the cut-off threshold mean + 2 SD and clustered with distance of 50 nt [Preprocessing of input genetic data and Equation (12)]. The input data after preprocessing are shown below initial profiles. (b) <em>z</em>-ratios [Equation (23)] characterizing pairwise positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for the H3me3K27 mark in the different chromosomes of <em>D. melanogaster</em>. The input data were preprocessed as described above. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for <em>z</em>-ratios correspond to 5% (|<em>z</em>| = 1.96) and 1% (|<em>z</em>| = 2.58) significance thresholds for random correlations. (c) Ratios characterizing positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks in the chromosome 2R of <em>D. melanogaster</em> at the different clustering lengths. The profiles were preliminary filtered by the cut-off threshold mean + 2 SD. The positive values of <em>z</em><sub>corr</sub> reflect a trend towards shorter distances between profiles relative to the reference model (or correlations), whereas the negative values of <em>z</em><sub>corr</sub> reflect a trend towards longer distances between profiles (or anticorrelations).</p></div></div><div content-id="DSU044F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/m_dsu04402.png?Expires=1509238989&Signature=fhF2oyqNIe-sZ~5nSg9G1nYpIt0b8tCf3yT31hbK~TdpPKzIK7kNTxs8qK33hYOejtmIl3qbro2Q8tEP~71dFGXz35Co15oWif1v5Zn7bEpNtZqs-y2CnW8HYFZDhZwSqT8U3ttFwmnJsOYcu8T-VSWD7x7RSVPJvatEcrN3RDvOcq0-w5n2-rAN5KVq6SSYXxjcc6TqUyyYC3hRaScWxp1uMJvoF54rURZ11v--K~jQuxUoLRBCIWKrArps1sErC6wreXPKTb2p6UpJfhR~0BFYZpFx8haSu3afXtUbUr-70NyIErQEvIoM0gwzDh5KnTXlWOWMievR4NPak5NrYQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(a) The binding profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks over chromosome 3R of Drosophila melanogaster. For the study of correlations, these profiles were preliminary filtered by the cut-off threshold mean + 2 SD and clustered with distance of 50 nt [Preprocessing of input genetic data and Equation (12)]. The input data after preprocessing are shown below initial profiles. (b) z-ratios [Equation (23)] characterizing pairwise positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for the H3me3K27 mark in the different chromosomes of D. melanogaster. The input data were preprocessed as described above. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for z-ratios correspond to 5% (|z| = 1.96) and 1% (|z| = 2.58) significance thresholds for random correlations. (c) Ratios characterizing positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks in the chromosome 2R of D. melanogaster at the different clustering lengths. The profiles were preliminary filtered by the cut-off threshold mean + 2 SD. The positive values of zcorr reflect a trend towards shorter distances between profiles relative to the reference model (or correlations), whereas the negative values of zcorr reflect a trend towards longer distances between profiles (or anticorrelations)." path-from-xml="dsu04402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6615601/dsu04402.tiff" path-from-xml="dsu04402.tiff" target="_blank">View large</a><a section="6615601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu04402.png?Expires=1509238989&Signature=JVGVFttcJPQ9tZdOMOzhM1PIMojtCnux1v8veib-ScBxX8rCAozOtedr4JwWg8cBvrl2IC41vsrx2N~KxInXM5EUL0z8LZikt0HxoBm-n0TOFwuxKMcP1bsBndkBjGbyUFbKEKxNsHaVjDKMUHZzC2BeLfPxabY9ZcMl34H4jPn22exbhgWusJW2lWrQTuFsqdOS7KuWhoTWkahVKnyFzGCQnQLC-ga~aODnMpg2H341P8LlQEea3r7hihFQou4S1OpukMy5KtPj8tqpYyzjeo0oi~WkvSx3FEb-Ni813kQ91sVbNwe26DDSa6M1cnNqEMA4nBpIJLGs4wtEK2mb7w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6615601&ar=442245&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The binding profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks over chromosome 3R of <em>Drosophila melanogaster</em>. For the study of correlations, these profiles were preliminary filtered by the cut-off threshold mean + 2 SD and clustered with distance of 50 nt [Preprocessing of input genetic data and Equation (12)]. The input data after preprocessing are shown below initial profiles. (b) <em>z</em>-ratios [Equation (23)] characterizing pairwise positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for the H3me3K27 mark in the different chromosomes of <em>D. melanogaster</em>. The input data were preprocessed as described above. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for <em>z</em>-ratios correspond to 5% (|<em>z</em>| = 1.96) and 1% (|<em>z</em>| = 2.58) significance thresholds for random correlations. (c) Ratios characterizing positional correlations between profiles for proteins E(Z), Pc-S2, and Psc, and for H3me3K27 histone marks in the chromosome 2R of <em>D. melanogaster</em> at the different clustering lengths. The profiles were preliminary filtered by the cut-off threshold mean + 2 SD. The positive values of <em>z</em><sub>corr</sub> reflect a trend towards shorter distances between profiles relative to the reference model (or correlations), whereas the negative values of <em>z</em><sub>corr</sub> reflect a trend towards longer distances between profiles (or anticorrelations).</p></div></div>                            <h3 scrollto-destination=6615602 id="6615602" class="section-title" data-legacy-id=s3e>3.5. Correlations between DSBs and histone marks</h3>
<p>Finally, we studied the relationships between DSBs and histone mark H3K4me3 in human chromosomes. DSBs induced by physical, chemical, or genetic agents provide important information about the large-scale organization of chromatin and may potentially be related to genomic stability, including translocations, deletions, or amplifications.<sup><a reveal-id="DSU044C35  DSU044C37" data-open="DSU044C35  DSU044C37" class="link link-ref link-reveal xref-bibr">35–37</a></sup> H3K4me3 is a well-known, promoter-specific histone modification that is associated with transcription and active genes. Recently, it was demonstrated that this epigenetic mark selectively directs global TFIID recruitment to active genes, some of which are p53 targets.<sup><a reveal-id="DSU044C38" data-open="DSU044C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The data on nucleotide-resolved DSBs were submitted to GEO with accession number GSE53811. DSBs were preliminarily preprocessed: DSBs mapped below 5% and above 95% of coverage were cut-off; the remaining DSBs were clustered within a distance of 1 kb (Preprocessing of input genetic data). Such a clustering distance provides the distribution of lengths between consecutive DSBs, which is similar to that observed in gel electrophoresis.<sup><a reveal-id="DSU044C35" data-open="DSU044C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSU044C36" data-open="DSU044C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The profiles for histone marks were taken from ENCODE accession wgEncodeEH000953 in the pre-aligned format BAM and were then processed with MACS2.<sup><a reveal-id="DSU044C21" data-open="DSU044C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The MACS2 peak-caller was used with the option ‘callpeak’ for identifying significant regions (treated like stretches in our package). Then, the correlations of corresponding midpoints of stretches with DSBs were studied with our package. The summary of correlations is shown in Fig. <a reveal-id="DSU044F3" data-open="DSU044F3" class="link link-reveal link-table xref-fig">3</a> and reveals clear positional correlations between these features. Such correlations support the hypothesis regarding the relationships between DSBs and coordinated gene expression.<sup><a reveal-id="DSU044C35" data-open="DSU044C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSU044C36" data-open="DSU044C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                        <a id="6615604" scrollto-destination="6615604"></a>
<div id="DSU044F3" data-id="DSU044F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/m_dsu04403.png?Expires=1509238989&Signature=FZlmT~jQrWgkiDLnIu5QAEpQxrq58Fk~8TqxzVhxRbENf7A2DkQD9vzaYugz-Rah3is~5O-JPYif4wJ5Gs57HESaZEEX7j-EX7wDWm3P0qcreXgqAPJYl3YV8zTP0PArQaQqdl39cxovuOscZIsAofZZkv1R8twG-F0HghuMr8hZyQgyh86LdROrSrKcV-bBhoYWBum2yx0SoGdWc5rK2mjIngFCynvQ2fnvRe6ixDi79omugvT341ST-1y5ay6gnOAyZWkPET4f1H9DszXiZNxtoLkDH3wDrwI7JoQWC8N-QyPfx2v52h2Po4wmlQ45~eSSfRsqmYBMgEUQpcK4Zw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(a) The distributions of DNA double-strand breaks (DSBs) and H3K4me3 histone marks over human chromosome 7. The distributions of DSBs and histone marks were coarse-grained over bins of 100 kb, i.e. the heights in these distributions correspond to the number of points in the bins of 100 kb. Both sets were preprocessed as described in the main text. The distribution of cytobands across chromosome 7 is shown above the length scale. (b) z-ratios [Equation (23)] characterizing pairwise positional correlations between distributions of DSBs and H3K4me3 in the human chromosomes. The correlations for the Y-chromosome are not shown due to poor statistics. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for z-ratios correspond to 5% (z = 1.96) and 1% (z = 2.58) significance thresholds for random correlations." path-from-xml="dsu04403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6615604/dsu04403.tiff" path-from-xml="dsu04403.tiff" target="_blank">View large</a><a section="6615604" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu04403.png?Expires=1509238989&Signature=RpSIs~MfZr5sgf17UlEWZz7Bbt9dY6nMrrIkxP2Qlyf6j4IKoDUPAbY8CkqkQKMv9nHRobwsXhdloIFZ7uj1PHKl6by8JOC1kdH8p-tb3jwlRtEB-hHQhBMbuUHAhF~wIxKaWwYPXvD3ccI0kBw3vlAYGyKoN2UH6XbzcROSw4iZ6EUEK-sE5d2XTImCYrmkb3D4UAv1x0BIcjkA4-uYERX4TpxOPPpYDDEoOkixg~069WK9fxcTplwmy0Kol9BJgq1MrGtUcPRX5aJP0usE1E5qB0MoRUXfwO0x9ejLovztwaW1l73O8sfjmi9J8ooa2c8L4K6DgOTtbskbgiGVYg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6615604&ar=442245&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The distributions of DNA double-strand breaks (DSBs) and H3K4me3 histone marks over human chromosome 7. The distributions of DSBs and histone marks were coarse-grained over bins of 100 kb, i.e. the heights in these distributions correspond to the number of points in the bins of 100 kb. Both sets were preprocessed as described in the main text. The distribution of cytobands across chromosome 7 is shown above the length scale. (b) <em>z</em>-ratios [Equation (23)] characterizing pairwise positional correlations between distributions of DSBs and H3K4me3 in the human chromosomes. The correlations for the Y-chromosome are not shown due to poor statistics. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for <em>z</em>-ratios correspond to 5% (<em>z</em> = 1.96) and 1% (<em>z</em> = 2.58) significance thresholds for random correlations.</p></div></div><div content-id="DSU044F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/m_dsu04403.png?Expires=1509238989&Signature=FZlmT~jQrWgkiDLnIu5QAEpQxrq58Fk~8TqxzVhxRbENf7A2DkQD9vzaYugz-Rah3is~5O-JPYif4wJ5Gs57HESaZEEX7j-EX7wDWm3P0qcreXgqAPJYl3YV8zTP0PArQaQqdl39cxovuOscZIsAofZZkv1R8twG-F0HghuMr8hZyQgyh86LdROrSrKcV-bBhoYWBum2yx0SoGdWc5rK2mjIngFCynvQ2fnvRe6ixDi79omugvT341ST-1y5ay6gnOAyZWkPET4f1H9DszXiZNxtoLkDH3wDrwI7JoQWC8N-QyPfx2v52h2Po4wmlQ45~eSSfRsqmYBMgEUQpcK4Zw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="(a) The distributions of DNA double-strand breaks (DSBs) and H3K4me3 histone marks over human chromosome 7. The distributions of DSBs and histone marks were coarse-grained over bins of 100 kb, i.e. the heights in these distributions correspond to the number of points in the bins of 100 kb. Both sets were preprocessed as described in the main text. The distribution of cytobands across chromosome 7 is shown above the length scale. (b) z-ratios [Equation (23)] characterizing pairwise positional correlations between distributions of DSBs and H3K4me3 in the human chromosomes. The correlations for the Y-chromosome are not shown due to poor statistics. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for z-ratios correspond to 5% (z = 1.96) and 1% (z = 2.58) significance thresholds for random correlations." path-from-xml="dsu04403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6615604/dsu04403.tiff" path-from-xml="dsu04403.tiff" target="_blank">View large</a><a section="6615604" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu04403.png?Expires=1509238989&Signature=RpSIs~MfZr5sgf17UlEWZz7Bbt9dY6nMrrIkxP2Qlyf6j4IKoDUPAbY8CkqkQKMv9nHRobwsXhdloIFZ7uj1PHKl6by8JOC1kdH8p-tb3jwlRtEB-hHQhBMbuUHAhF~wIxKaWwYPXvD3ccI0kBw3vlAYGyKoN2UH6XbzcROSw4iZ6EUEK-sE5d2XTImCYrmkb3D4UAv1x0BIcjkA4-uYERX4TpxOPPpYDDEoOkixg~069WK9fxcTplwmy0Kol9BJgq1MrGtUcPRX5aJP0usE1E5qB0MoRUXfwO0x9ejLovztwaW1l73O8sfjmi9J8ooa2c8L4K6DgOTtbskbgiGVYg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=6615604&ar=442245&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) The distributions of DNA double-strand breaks (DSBs) and H3K4me3 histone marks over human chromosome 7. The distributions of DSBs and histone marks were coarse-grained over bins of 100 kb, i.e. the heights in these distributions correspond to the number of points in the bins of 100 kb. Both sets were preprocessed as described in the main text. The distribution of cytobands across chromosome 7 is shown above the length scale. (b) <em>z</em>-ratios [Equation (23)] characterizing pairwise positional correlations between distributions of DSBs and H3K4me3 in the human chromosomes. The correlations for the Y-chromosome are not shown due to poor statistics. The numbers below the chromosome nomenclature correspond to that of the nearest neighbours. The horizontal broken lines for <em>z</em>-ratios correspond to 5% (<em>z</em> = 1.96) and 1% (<em>z</em> = 2.58) significance thresholds for random correlations.</p></div></div><p>The correlations between particular genomic tracks in the following sections, Correlations between CpG islands and TSS, Correlations between TSS on the forward and reverse strands, Correlations between protein-binding profiles, and Correlations between DSBs and histone marks, obtained with Genomic HyperBrowser, GenometriCorr, and Genome Track Analyzer are mutually compared in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S6</a>. The comparison showed that the detected correlations appeared to be significant in all three packages (with minor divergence depending on the package and method). The significance of correlations in all packages indicates that they are real, non-artificial correlations. Generally, the coincidence of the correlation significance obtained with the different packages may depend on the particular chosen example (see Tests and comparison of different packages).</p>                            <h2 scrollto-destination=6615606 id="6615606" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Our study proves that the proper treatment of inhomogeneity inherent to genome tracks is essential for the correct assessment of correlations between genome tracks. We tried a series of algorithms, including our variants and those suggested by others,<sup><a reveal-id="DSU044C12" data-open="DSU044C12" class="link link-ref link-reveal xref-bibr">12</a>–<a reveal-id="DSU044C14" data-open="DSU044C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and found that none of the algorithms that used absolute length scales satisfied the test for the absence of correlations between a random independent set and a strongly inhomogeneous non-random set (Tests and comparison of different packages and Table <a reveal-id="DSU044TB1" data-open="DSU044TB1" class="link link-reveal link-table xref-fig">1</a>). The choice of the relative lengths with local normalization turned out to be crucial for the absence of correlations between a strongly inhomogeneous set and a random one. The significant inhomogeneity inherent to genome tracks and strong variations in characteristic lengths may be related to the large-scale chromatin organization (see Ref. <a reveal-id="DSU044C17" data-open="DSU044C17" class="link link-ref link-reveal xref-bibr">17</a> for discussion and further references).</p><p>All <em>z</em>-criteria (10), (11), and (23) used in our package are based on the central limit theorem. The variations in the vicinity |<em>z</em>| ≤ 2.7 proved to be approximately Gaussian for the random sets. The application of Gaussian criteria implies that the number of pairs of nearest neighbours, <em>K</em>, should be sufficiently large. The convergence can be assessed by the parameter <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">1</mn><mrow xmlns=""><mo>/</mo></mrow><msqrt xmlns=""><mi>K</mi></msqrt></math></span>. We determined that 50–100 representatives or more are sufficient for Gaussian approximation in the corresponding <em>z</em>-criteria. For the small sets, the statistical scattering becomes large and the deviations from Gaussian statistics may be significant. As the tails of distributions for <em>z</em>-ratios are non-Gaussian, we used the coarse-grained scale for the presentation of significance for output <em>z</em>-values: |<em>z</em>| &lt; 1.8, insignificant; 1.8&lt; |<em>z</em>| &lt;1.96, fuzzy; 1.96&lt; |<em>z</em>| &lt;2.58, significant; and |<em>z</em>| &gt; 2.58, highly significant. In the visual presentation of the software, all grades are marked by different colours.</p><p>Our analytical criteria proved to be robust against the inhomogeneous distribution of lengths corresponding to genome tracks, which is typical of genetic problems. The analytical criteria significantly shorten the time of computational analysis and permit the study of larger data sets in comparison with MC simulations (Tests and comparison of different packages). Note that the analysis of large output data arrays needs the application of extreme value statistics for the assessment of their statistical significance. For example, in the case of random correlations, ∼5% of output <em>z</em>-values can exceed the threshold |<em>z</em>| &gt;1.96. Generally, the distribution of the large output array of <em>z</em>-values can be compared with Gaussian distribution by the Kolmogorov–Smirnov criterion.</p><p>A similar approach can be applied to the study of distributions of nucleotide content or DNA physico-chemical characteristics over the genome. In the latter case, these parameters should, initially, be coarse-grained over a window of length <em>W</em>. The distribution of parameters over consecutive non-overlapping windows can further be treated in line with profile preprocessing (Preprocessing of input genetic data). If needed, the continuous criteria used in our package can be replaced by their corresponding discrete counterparts.<sup><a reveal-id="DSU044C17" data-open="DSU044C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In the next step, we will include the more detailed analysis of correlations between stretches that are characterized by both the positions of the stretch centres and the overlap of stretches. To conclude, the Genome Track Analyzer provides an efficient tool for the study of correlations between genome features with useful applications to many genetic problems. The study of correlations between genome tracks may shed light on the intricate regulation networks in the different organisms.</p>                            <h2 scrollto-destination=6615611 id="6615611" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6615613 id="6615613" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Russian Fund of Basic Research (grant number 14-04-01638) and a grant from the Molecular and Cellular Biology Program of the Russian Academy of Sciences.</p>                            <h2 scrollto-destination=6615615 id="6615615" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are grateful to A. Favorov for helpful communications and are also grateful to D. Fedoseeva and N. Kochanova for discussion on bidirectional transcription.</p>                            <h2 scrollto-destination=6615617 id="6615617" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsu044c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernstein</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Stamatoyannopoulos</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Costello</div> <div class="given-names">J.F.</div></div>, et al. </div><div class="article-title">The NIH Roadmap Epigenomics Mapping Consortium</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2010</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1045</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20NIH%20Roadmap%20Epigenomics%20Mapping%20Consortium&amp;author=B.E.%20Bernstein&amp;author=J.A.%20Stamatoyannopoulos&amp;author=J.F.%20Costello&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=1045-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1010-1045" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1010-1045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1010-1045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20944595" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tragante</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Asselbergs</div> <div class="given-names">F.W.</div></div>. </div><div class="article-title">The ENCODE project and perspectives on pathways</div>, <div class="source">Genet. Epidemiol.</div> , <div class="year">2014</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">275</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ENCODE%20project%20and%20perspectives%20on%20pathways&amp;author=V.%20Tragante&amp;author=J.H.%20Moore&amp;author=F.W.%20Asselbergs&amp;publication_year=2014&amp;journal=Genet.%20Epidemiol.&amp;volume=38&amp;pages=275-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/gepi.21802" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fgepi.21802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fgepi.21802"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24723339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dreos</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ambrosini</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Cavin Perier</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bucher</div> <div class="given-names">P.</div></div>. </div><div class="article-title">EPD and EPDnew, high-quality promoter resources in the next-generation sequencing era</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">D157</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EPD%20and%20EPDnew%2C%20high-quality%20promoter%20resources%20in%20the%20next-generation%20sequencing%20era&amp;author=R.%20Dreos&amp;author=G.%20Ambrosini&amp;author=R.%20Cavin%20Perier&amp;author=P.%20Bucher&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D157-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1233" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamashita</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">DBTSS: Database of Transcriptional Start Sites progress report in 2012</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">D150</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBTSS%3A%20Database%20of%20Transcriptional%20Start%20Sites%20progress%20report%20in%202012&amp;author=R.%20Yamashita&amp;author=S.%20Sugano&amp;author=Y.%20Suzuki&amp;author=K.%20Nakai&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D150-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Caffo</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Jaffee</div> <div class="given-names">H.A.</div></div>,  <div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Feinberg</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">Redefining CpG islands using hidden Markov models</div>, <div class="source">Biostatistics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">499</div>-<div class="lpage">514</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Redefining%20CpG%20islands%20using%20hidden%20Markov%20models&amp;author=H.%20Wu&amp;author=B.%20Caffo&amp;author=H.A.%20Jaffee&amp;author=R.A.%20Irizarry&amp;author=A.P.%20Feinberg&amp;publication_year=2010&amp;journal=Biostatistics&amp;volume=11&amp;pages=499-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/kxq005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2Fkxq005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2Fkxq005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20212320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="collab">FANTOM Consortium, R.P., CLST</div><div class="article-title">A promoter-level mammalian expression atlas</div>, <div class="source">Nature</div> , <div class="year">2014</div>, vol. <div class="volume">507</div> (pg. <div class="fpage">462</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13182" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13182" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13182"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24670764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bush</div> <div class="given-names">W.S.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">J.H.</div></div>. </div><div class="article-title">Chapter 11: genome-wide association studies</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002822</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chapter%2011%3A%20genome-wide%20association%20studies&amp;author=W.S.%20Bush&amp;author=J.H.%20Moore&amp;publication_year=2012&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=8&amp;pages=e1002822" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002822" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Daley</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Vere-Jones</div> <div class="given-names">D.</div></div>. </div>, <div class="source">An introduction to the theory of point processes</div> , <div class="year">2003</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div></div></div></div></div><div content-id="dsu044c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jacobsen</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Point process theory and applications: marked point and piecewise deterministic processes</div> , <div class="year">2006</div><div class="publisher-loc">Boston, MA</div><div class="publisher-name">Birkhäuser</div></div></div></div></div><div content-id="dsu044c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bickel</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">An overview of recent developments in genomics and associated statistical methods</div>, <div class="source">Philos. Trans. A Math. Phys. Eng. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">367</div> (pg. <div class="fpage">4313</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20overview%20of%20recent%20developments%20in%20genomics%20and%20associated%20statistical%20methods&amp;author=P.J.%20Bickel&amp;author=J.B.%20Brown&amp;author=H.%20Huang&amp;author=Q.%20Li&amp;publication_year=2009&amp;journal=Philos.%20Trans.%20A%20Math.%20Phys.%20Eng.%20Sci.&amp;volume=367&amp;pages=4313-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rsta.2009.0164" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frsta.2009.0164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frsta.2009.0164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19805447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.D.</div></div>,  <div class="name"><div class="surname">Paccanaro</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Statistical analysis of the genomic distribution and correlation of regulatory elements in the ENCODE regions</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">787</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20analysis%20of%20the%20genomic%20distribution%20and%20correlation%20of%20regulatory%20elements%20in%20the%20ENCODE%20regions&amp;author=Z.D.%20Zhang&amp;author=A.%20Paccanaro&amp;author=Y.%20Fu&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=787-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5573107" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5573107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5573107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17567997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandve</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Gundersen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rydbeck</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The Genomic HyperBrowser: inferential genomics at the sequence level</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">R121</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genomic%20HyperBrowser%3A%20inferential%20genomics%20at%20the%20sequence%20level&amp;author=G.K.%20Sandve&amp;author=S.%20Gundersen&amp;author=H.%20Rydbeck&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-12-r121" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-12-r121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-12-r121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21182759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandve</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Gundersen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Johansen</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">The Genomic HyperBrowser: an analysis web server for genome-scale data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">W133</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genomic%20HyperBrowser%3A%20an%20analysis%20web%20server%20for%20genome-scale%20data&amp;author=G.K.%20Sandve&amp;author=S.%20Gundersen&amp;author=M.%20Johansen&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=W133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt342" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt342" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt342"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23632163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Favorov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mularoni</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cope</div> <div class="given-names">L.M.</div></div>, et al. </div><div class="article-title">Exploring massive, genome scale datasets with the GenometriCorr package</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002529</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20massive%2C%20genome%20scale%20datasets%20with%20the%20GenometriCorr%20package&amp;author=A.%20Favorov&amp;author=L.%20Mularoni&amp;author=L.M.%20Cope&amp;publication_year=2012&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=8&amp;pages=e1002529" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002529" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22693437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feller</div> <div class="given-names">W.</div></div>. </div>, <div class="source">An introduction to probability theory and its applications</div> , <div class="year">1967</div><div class="publisher-loc">New York</div><div class="publisher-name">Wiley</div></div></div></div></div><div content-id="dsu044c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chechetkin</div> <div class="given-names">V.R.</div></div>. </div><div class="article-title">Spectral sum rules and search for periodicities in DNA sequences</div>, <div class="source">Phys. Lett. A</div> , <div class="year">2011</div>, vol. <div class="volume">375</div> (pg. <div class="fpage">1729</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spectral%20sum%20rules%20and%20search%20for%20periodicities%20in%20DNA%20sequences&amp;author=V.R.%20Chechetkin&amp;publication_year=2011&amp;journal=Phys.%20Lett.%20A&amp;volume=375&amp;pages=1729-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.physleta.2011.03.007" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.physleta.2011.03.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.physleta.2011.03.007"> </span></div> </div></div></div></div></div><div content-id="dsu044c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chechetkin</div> <div class="given-names">V.R.</div></div>. </div><div class="article-title">Statistics of genome architecture</div>, <div class="source">Phys. Lett. A</div> , <div class="year">2013</div>, vol. <div class="volume">377</div> (pg. <div class="fpage">3312</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistics%20of%20genome%20architecture&amp;author=V.R.%20Chechetkin&amp;publication_year=2013&amp;journal=Phys.%20Lett.%20A&amp;volume=377&amp;pages=3312-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.physleta.2013.10.021" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.physleta.2013.10.021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.physleta.2013.10.021"> </span></div> </div></div></div></div></div><div content-id="dsu044c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pepke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Computation for ChIP-seq and RNA-seq studies</div>, <div class="source">Nat. Meth.</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">S22</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computation%20for%20ChIP-seq%20and%20RNA-seq%20studies&amp;author=S.%20Pepke&amp;author=B.%20Wold&amp;author=A.%20Mortazavi&amp;publication_year=2009&amp;journal=Nat.%20Meth.&amp;volume=6&amp;pages=S22-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1371" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1371"> </span></div> </div></div></div></div></div><div content-id="dsu044c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilbanks</div> <div class="given-names">E.G.</div></div>,  <div class="name"><div class="surname">Facciotti</div> <div class="given-names">M.T.</div></div>. </div><div class="article-title">Evaluation of algorithm performance in ChIP-seq peak detection</div>, <div class="source">PLoS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e11471</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20algorithm%20performance%20in%20ChIP-seq%20peak%20detection&amp;author=E.G.%20Wilbanks&amp;author=M.T.%20Facciotti&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e11471" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0011471" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0011471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0011471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20628599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hower</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Evans</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Shape-based peak identification for ChIP-Seq</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">15</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shape-based%20peak%20identification%20for%20ChIP-Seq&amp;author=V.%20Hower&amp;author=S.N.%20Evans&amp;author=L.%20Pachter&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-15" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21226895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Meyer</div> <div class="given-names">C.A.</div></div>, et al. </div><div class="article-title">Model-based analysis of ChIP-Seq (MACS)</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">R137</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Model-based%20analysis%20of%20ChIP-Seq%20(MACS)&amp;author=Y.%20Zhang&amp;author=T.%20Liu&amp;author=C.A.%20Meyer&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=R137" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-9-r137" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-9-r137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-9-r137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18798982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Illingworth</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Bird</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">CpG islands – ‘a rough guide</div>, <div class="source">FEBS Lett.</div> , <div class="year">2009</div>, vol. <div class="volume">583</div> (pg. <div class="fpage">1713</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20islands%20%E2%80%93%20%E2%80%98a%20rough%20guide&amp;author=R.S.%20Illingworth&amp;author=A.P.%20Bird&amp;publication_year=2009&amp;journal=FEBS%20Lett.&amp;volume=583&amp;pages=1713-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2009.04.012" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2009.04.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2009.04.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19376112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bell</div> <div class="given-names">C.G.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Butcher</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Roos</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Walter</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Beck</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Human-specific CpG ‘beacons’ identify loci associated with human-specific traits and disease</div>, <div class="source">Epigenetics</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1188</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human-specific%20CpG%20%E2%80%98beacons%E2%80%99%20identify%20loci%20associated%20with%20human-specific%20traits%20and%20disease&amp;author=C.G.%20Bell&amp;author=G.A.%20Wilson&amp;author=L.M.%20Butcher&amp;author=C.%20Roos&amp;author=L.%20Walter&amp;author=S.%20Beck&amp;publication_year=2012&amp;journal=Epigenetics&amp;volume=7&amp;pages=1188-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.22127" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.22127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.22127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22968434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krinner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Heitzer</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Diermeier</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Obermeier</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Langst</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wagner</div> <div class="given-names">R.</div></div>. </div><div class="article-title">CpG domains downstream of TSSs promote high levels of gene expression</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2014</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">3551</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20domains%20downstream%20of%20TSSs%20promote%20high%20levels%20of%20gene%20expression&amp;author=S.%20Krinner&amp;author=A.P.%20Heitzer&amp;author=S.D.%20Diermeier&amp;author=I.%20Obermeier&amp;author=G.%20Langst&amp;author=R.%20Wagner&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=3551-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1358" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1358" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1358"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24413563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tchurikov</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Kretova</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Moiseeva</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Sosin</div> <div class="given-names">D.V.</div></div>. </div><div class="article-title">Evidence for RNA synthesis in the intergenic region between enhancer and promoter and its inhibition by insulators in <em>Drosophila melanogaster</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">111</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20RNA%20synthesis%20in%20the%20intergenic%20region%20between%20enhancer%20and%20promoter%20and%20its%20inhibition%20by%20insulators%20in%20Drosophila%20melanogaster&amp;author=N.A.%20Tchurikov&amp;author=O.V.%20Kretova&amp;author=E.D.%20Moiseeva&amp;author=D.V.%20Sosin&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=111-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn926" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn926"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19022852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Hemberg</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gray</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">Widespread transcription at neuronal activity-regulated enhancers</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">465</div> (pg. <div class="fpage">182</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20transcription%20at%20neuronal%20activity-regulated%20enhancers&amp;author=T.K.%20Kim&amp;author=M.%20Hemberg&amp;author=J.M.%20Gray&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=182-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09033" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20393465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richard</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Manley</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">How bidirectional becomes unidirectional</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1022</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20bidirectional%20becomes%20unidirectional&amp;author=P.%20Richard&amp;author=J.L.%20Manley&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=20&amp;pages=1022-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2657" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2657"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24008561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicol</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Helt</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Blanchard</div> <div class="given-names">S.G.</div>Jr</div>,  <div class="name"><div class="surname">Raja</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Loraine</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">The Integrated Genome Browser: free software for distribution and exploration of genome-scale datasets</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2730</div>-<div class="lpage">1</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Integrated%20Genome%20Browser%3A%20free%20software%20for%20distribution%20and%20exploration%20of%20genome-scale%20datasets&amp;author=J.W.%20Nicol&amp;author=G.A.%20Helt&amp;author=S.G.%20Blanchard&amp;author=A.%20Raja&amp;author=A.E.%20Loraine&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2730-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp472" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp472" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp472"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19654113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">The functions of E(Z)/EZH2-mediated methylation of lysine 27 in histone H3</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">155</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functions%20of%20E(Z)%2FEZH2-mediated%20methylation%20of%20lysine%2027%20in%20histone%20H3&amp;author=R.%20Cao&amp;author=Y.%20Zhang&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=14&amp;pages=155-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2004.02.001" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2004.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2004.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15196462" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schwartz</div> <div class="given-names">Y.B.</div></div>,  <div class="name"><div class="surname">Kahn</div> <div class="given-names">T.G.</div></div>,  <div class="name"><div class="surname">Nix</div> <div class="given-names">D.A.</div></div>, et al. </div><div class="article-title">Genome-wide analysis of polycomb targets in <em>Drosophila melanogaster</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">700</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20polycomb%20targets%20in%20Drosophila%20melanogaster&amp;author=Y.B.%20Schwartz&amp;author=T.G.%20Kahn&amp;author=D.A.%20Nix&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=700-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1817" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1817" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1817"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16732288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kwong</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Adryan</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bell</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Stability and dynamics of polycomb target sites in <em>Drosophila</em> development</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000178</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stability%20and%20dynamics%20of%20polycomb%20target%20sites%20in%20Drosophila%20development&amp;author=C.%20Kwong&amp;author=B.%20Adryan&amp;author=I.%20Bell&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000178" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000178" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000178"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18773083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Kassis</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Architectural and functional diversity of polycomb group response elements in <em>Drosophila</em></div>, <div class="source">Genetics</div> , <div class="year">2013</div>, vol. <div class="volume">195</div> (pg. <div class="fpage">407</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Architectural%20and%20functional%20diversity%20of%20polycomb%20group%20response%20elements%20in%20Drosophila&amp;author=J.L.%20Brown&amp;author=J.A.%20Kassis&amp;publication_year=2013&amp;journal=Genetics&amp;volume=195&amp;pages=407-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.113.153247" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.113.153247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.113.153247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23934890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rustici</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kolesnikov</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Brandizi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">ArrayExpress update – trends in database growth and links to data analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">D987</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ArrayExpress%20update%20%E2%80%93%20trends%20in%20database%20growth%20and%20links%20to%20data%20analysis%20tools&amp;author=G.%20Rustici&amp;author=N.%20Kolesnikov&amp;author=M.%20Brandizi&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D987-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1174" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuettengruber</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Oded Elkayam</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sexton</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Cooperativity, specificity, and evolutionary stability of polycomb targeting in <em>Drosophila</em></div>, <div class="source">Cell Rep.</div> , <div class="year">2014</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">219</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cooperativity%2C%20specificity%2C%20and%20evolutionary%20stability%20of%20polycomb%20targeting%20in%20Drosophila&amp;author=B.%20Schuettengruber&amp;author=N.%20Oded%20Elkayam&amp;author=T.%20Sexton&amp;publication_year=2014&amp;journal=Cell%20Rep.&amp;volume=9&amp;pages=219-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2014.08.072" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2014.08.072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2014.08.072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25284790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tchurikov</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Kretova</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Sosin</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Zykov</div> <div class="given-names">I.A.</div></div>,  <div class="name"><div class="surname">Zhimulev</div> <div class="given-names">I.F.</div></div>,  <div class="name"><div class="surname">Kravatsky</div> <div class="given-names">Y.V.</div></div>. </div><div class="article-title">Genome-wide profiling of forum domains in <em>Drosophila melanogaster</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">3667</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20profiling%20of%20forum%20domains%20in%20Drosophila%20melanogaster&amp;author=N.A.%20Tchurikov&amp;author=O.V.%20Kretova&amp;author=D.V.%20Sosin&amp;author=I.A.%20Zykov&amp;author=I.F.%20Zhimulev&amp;author=Y.V.%20Kravatsky&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=3667-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1353" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1353" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1353"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21247882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tchurikov</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Kretova</div> <div class="given-names">O.V.</div></div>,  <div class="name"><div class="surname">Fedoseeva</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">DNA double-strand breaks coupled with PARP1 and HNRNPA2B1 binding sites flank coordinately expressed domains in human chromosomes</div>, <div class="source">PLoS Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">9</div> pg. <div class="fpage">e1003429</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20double-strand%20breaks%20coupled%20with%20PARP1%20and%20HNRNPA2B1%20binding%20sites%20flank%20coordinately%20expressed%20domains%20in%20human%20chromosomes&amp;author=N.A.%20Tchurikov&amp;author=O.V.%20Kretova&amp;author=D.M.%20Fedoseeva&amp;publication_year=2013&amp;journal=PLoS%20Genet.&amp;volume=9&amp;pages=e1003429" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003429" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003429" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003429"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23593027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsu044c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Crosetto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mitra</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Silva</div> <div class="given-names">M.J.</div></div>, et al. </div><div class="article-title">Nucleotide-resolution DNA double-strand break mapping by next-generation sequencing</div>, <div class="source">Nat. Meth.</div> , <div class="year">2013</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">361</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide-resolution%20DNA%20double-strand%20break%20mapping%20by%20next-generation%20sequencing&amp;author=N.%20Crosetto&amp;author=A.%20Mitra&amp;author=M.J.%20Silva&amp;publication_year=2013&amp;journal=Nat.%20Meth.&amp;volume=10&amp;pages=361-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2408" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2408" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2408"> </span></div> </div></div></div></div></div><div content-id="dsu044c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsu044c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lauberth</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">H3K4me3 interactions with TAF3 regulate preinitiation complex assembly and selective gene activation</div>, <div class="source">Cell</div> , <div class="year">2013</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">1021</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H3K4me3%20interactions%20with%20TAF3%20regulate%20preinitiation%20complex%20assembly%20and%20selective%20gene%20activation&amp;author=S.M.%20Lauberth&amp;author=T.%20Nakayama&amp;author=X.%20Wu&amp;publication_year=2013&amp;journal=Cell&amp;volume=152&amp;pages=1021-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.01.052" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.01.052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.01.052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23452851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=442245&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu044-dsu044supp_data6.xls?Expires=1509238989&amp;Signature=fC74-7GVxvaX-c5YrDHms8HHRhEjvrqOy-oLqFuCbgHN~HhWVRNB96GJy9QXnedM2AIDdx8oZBiAsjfPzd7xoIqNZqPOm~sKNffE20Qk-B0Xk8nbPLmL65QssT2B-fVr1Y9kGyh~QYdjl2u~StLEN53Ask~smW2ceYaPzqhCXr9Ivw9QtWnC-qY2k0xay9OEtCvh5PcLxqvqa6g2haBMmM3KgiSCCuA~Hbgh5NIb-19mAKwFs0BfRk6mubVnbWvd9ttl-yPoBLRoWIFCHskLFbcn56Nhahk1o3bjzwfbz8b-eDyXTKRPIdRqdglgZG4Oqbic6ZXtREmI6zgWHcWXmg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu044-dsu044supp_data5.xls?Expires=1509238989&amp;Signature=VKSkAnCE29R~AkpgBFg~73nvcsym4sKbwbbKKY~Glwh3xo52KNu-lhIWbefaeTtM90G2aYapXk4PyYQJtWZa6kbXJUzG4r87p3L0uwi4jjjcEmWympvsw4lqrXNvU07fz1530QjXvEI3kdOg~rbWFWOscig5o1eVRabhmEQlBGN00p8TUhSyLsT1DielYusP6eqilGoiIM-GqeaJ9dy9tWsRMckAWbym~9hUWsMQQ8MDo7VTr9I9hMi3o9fviaE5BR6wRWBrasM9C~3G-grGQKvMqFwKp3bjdsN4aI-D~1wbLZfZYzD1Jp0hW1TpDX6ZMbm9Xg4kaLxWsk2rqDmJkw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu044-dsu044supp_data4.pdf?Expires=1509238989&amp;Signature=IBi7HduUCc3ERGpT-w3b5j9JG8O4Rum6HfREK9l0~rW03dfIvZCv4rRyaGzaDEDmm2npLyUKttjJ~J-3vK81a5SJ~bQQVUEEqPWiwfLtGnQkJCI3JQacxiRVnUCyEMYkntK~P162BGW9Hobwz86wyXLm8TqQsoFDV~et64cPWC~e-s7djOxMjdBPNTukZYQm7O5jayNIlt2h9cjEMo2JAJ19GI111Vv9ns8O~LhG0vm2KpWd8jE90-j7tM3FRRHqacH2J9w8O~GSjEoHSFcF-5037FfWs2XOpuw~vb1vZXwK6yvKjjfdpVkp9oxE9Qk216~vX8XHLtX0rVkA9DsDeA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu044-dsu044supp_data3.xls?Expires=1509238989&amp;Signature=T5JA-P0dWCtscZ~nr6N6-HIpRRDpN0N7XQeJxfeTzxr0IEj~8b8CpRFWICYLmFtT2mr43ftyJWZpayepnQGQ5TySyF0S5Ufmrw2aQqBfSSBWUR8lm--MNkDzTRgSJb2WvK8W5MDvWbk9bowhv~roIQETDDI5IXUaipC2vjuTlzZpY0Peakoo92wJSmG-qNllItRluTCvA4l7ihD6ENbjtUJfwCC8ObidRDTdFomi5IqiKPDYPPAO99x5hq~zdtFVVyUzsaJ7v5Rde68OxxP2spMN06HX3lf49GlkLtMi68sKdnMJC9CLEnTu9X9pSu3Ao00X9LHszobtZsKPlIit0g__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu044-dsu044supp_data2.xls?Expires=1509238989&amp;Signature=MJPiEDZF3VfX3B6nHFsea2UkKQT5QuvXCKMe1KvsZVl7lSCwiXox0ng3qmOypdlE9CyBCDpteRoIWcZD0baJ0bTqLIcwZLnYiaCZfIP0KLJi6Dh2~EQYO8xmxG14-~W98AC5YbUsyRnYkBBSVbAfxMASUZhnmZU-6k8NwMKnzU3LgMvDRRSiQ8gPyXdbgs6flwfeNlQe0-dwDjspSvITpMRjoooJwdXnUZxxQYZgBQBcu8jr6dHMSkhVCtByujBXShxVP7u8Pbvmw3a2514alahJnv2ApPhBdLquy24oNjQwYkKBupPYwMKoKnZzD8AgMz3pe13m3YhKrX~DqtSmlA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu044/2/dsu044-dsu044supp_data1.xls?Expires=1509238989&amp;Signature=IUctEV-Qvy1nwoMGhcq~tFUzcjip-kuWPfog3ysW~M1bcYWQzj09h0PVCsgwK2Dzxafa5bpgt7Rv7P3u190qyP-jFeFWxikRoL62fZzLjYVxs6Q87S0GhDgHPeRTWFX5Q7tt7o7JpLQmOlLxiXONdw5IFTn1gEhj2moBdGMhZX-Re3~nYSWmShrmTLn4tHVGpULOHX-7DDhYpvc0ZUTc8wKH~V06YYesHQBzCybQ9hds8TR1hlXu0uCv9Zv07dWO0WU0ZzO0tewZOe~DEQ8OQHp5UC8CdDMgAlL5kNQ~oZ9XihAwWCyprJijGlflpkq0tF1Lf-qelRoUdG1DS-n7GA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">49</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100011&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsu044"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="442245"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100011&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f109%2f442245">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100011&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F109%2F442245" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/442245'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide study of correlations between genomic features and their relationship with the regulation of gene expression" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f109%2f442245">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

