

<!DOCTYPE html>
<html lang="en">

<head><title>
	Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous Lep1 Helitrons | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/1/11/484721","@type":"ScholarlyArticle","name":"Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous  Lep 1  Helitrons ","datePublished":"2011-11-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr038","keywords":["<italic>Helitron</italic>","sequence gain","genome rearrangement"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/1/11/484721","author":[{"name":"Coates, Brad S.","affiliation":"1  USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory ,  Iowa State University ,  Ames, IA 50011 ,  USA","@type":"Person"},{"name":"Hellmich, Richard L.","affiliation":"1  USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory ,  Iowa State University ,  Ames, IA 50011 ,  USA    2  Department of Entomology ,  Iowa State University ,  Ames, IA 50011 ,  USA","@type":"Person"},{"name":"Grant, David M.","affiliation":"1  USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory ,  Iowa State University ,  Ames, IA 50011 ,  USA    3  Department of Agronomy ,  Iowa State University ,  Ames, IA 50011 ,  USA","@type":"Person"},{"name":"Abel, Craig A.","affiliation":"1  USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory ,  Iowa State University ,  Ames, IA 50011 ,  USA    2  Department of Entomology ,  Iowa State University ,  Ames, IA 50011 ,  USA","@type":"Person"}],"description":"Transposable elements (TEs) can affect the structure of genomes through their acquisition and transposition of novel DNA sequences. The 134-bp repetitive elements, Lep1, are conserved non-autonomous Helitrons in lepidopteran genomes that have characteristic 5′-CT and 3′-CTAY nucleotide termini, a 3′-terminal hairpin structure, a 5′- and 3′-subterminal inverted repeat (SIR), and integrations that occur between AT or TT nucleotides. Lep1 Helitrons have acquired and propagated sequences downstream of their 3′-CTAY termini that are 57–344-bp in length and have termini composed of a 3′-CTRR preceded by a 3′-hairpin structure and a region complementary to the 5′-SIR (3′-SIRb). Features of both the Lep1 Helitron and multiple acquired sequences indicate that secondary structures at the 3′-terminus may have a role in rolling circle replication or genome integration mechanisms, and are a prerequisite for novel end creation by Helitron-like TEs. The preferential integration of Lep1 Helitrons in proximity to gene-coding regions results in the creation of genetic novelty that is shown to impact gene structure and function through the introduction of novel exon sequence (exon shuffling). These findings are important in understanding the structural requirements of genomic DNA sequences that are acquired and transposed by Helitron-like TEs.","pageStart":"11","pageEnd":"21","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=1572217184&Signature=LUW2k~5Zvn13HVDYuNk2DkpjszqV~I9q9iCyS0ZCtDnUXKZ4cR2Nt3cEuvJeMDzkGRuQIBPQcGZ6kkc-xw2UHyhvBSlbatNMURadKV5MBsnF0o0Ogp5RQpH8ZYuIpmYIvYbZLs0faWYS4oZTjK~p4B4ZwPSv~bD79FdpkrMUF50D1MGiPCGDV7CBn5U8y9gjODezdzzF6LqUPRJU8w2jT2b7e7ONstwrv0muSGXhKLNWu588ZjKyRrfTEhS3ZWVfDK65oy7PDFq7nC5EONsSeNmcwPJDKe0Z9dB15D6OGscovVvxPdcUCKQPqPVSHakQuigI3VZ0zVmuRrjgPOeSrQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous  Lep 1  Helitrons ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=1572217184&Signature=LUW2k~5Zvn13HVDYuNk2DkpjszqV~I9q9iCyS0ZCtDnUXKZ4cR2Nt3cEuvJeMDzkGRuQIBPQcGZ6kkc-xw2UHyhvBSlbatNMURadKV5MBsnF0o0Ogp5RQpH8ZYuIpmYIvYbZLs0faWYS4oZTjK~p4B4ZwPSv~bD79FdpkrMUF50D1MGiPCGDV7CBn5U8y9gjODezdzzF6LqUPRJU8w2jT2b7e7ONstwrv0muSGXhKLNWu588ZjKyRrfTEhS3ZWVfDK65oy7PDFq7nC5EONsSeNmcwPJDKe0Z9dB15D6OGscovVvxPdcUCKQPqPVSHakQuigI3VZ0zVmuRrjgPOeSrQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous  Lep 1  Helitrons  | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Transposable elements (TEs) can affect the structure of genomes through their acquisition and transposition of novel DNA sequences. The 134-bp repetitive elements, Lep1, are conserved non-autonomous Helitrons in lepidopteran genomes that have characteristic 5′-CT and 3′-CTAY nucleotide termini, a 3′-terminal hairpin structure, a 5′- and 3′-subterminal inverted repeat (SIR), and integrations that occur between AT or TT nucleotides. Lep1 Helitrons have acquired and propagated sequences downstream of their 3′-CTAY termini that are 57–344-bp in length and have termini composed of a 3′-CTRR preceded by a 3′-hairpin structure and a region complementary to the 5′-SIR (3′-SIRb). Features of both the Lep1 Helitron and multiple acquired sequences indicate that secondary structures at the 3′-terminus may have a role in rolling circle replication or genome integration mechanisms, and are a prerequisite for novel end creation by Helitron-like TEs. The preferential integration of Lep1 Helitrons in proximity to gene-coding regions results in the creation of genetic novelty that is shown to impact gene structure and function through the introduction of novel exon sequence (exon shuffling). These findings are important in understanding the structural requirements of genomic DNA sequences that are acquired and transposed by Helitron-like TEs." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/1/11/484721" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217184&amp;Signature=EXyUwyNI1ai19cjUVOaZsIu3271H6LsiyKigUU4HPyopAPFUgBGLwUjI66F3tdryMC5rXL8noOJv1Ck2FmB3M33l4rALfQvYq9cU2kvh1F4lKQrlkUQQaQtbgWrTmQmnygUSZYYd8dfKYw4pcxa-9rb5LrfST-42pdImXGwzw6tOW9IT7~spRiAiAqY6SUIB2SjiEkZM6DgOUibO1ALDoAdAELlqSqxaqbnig1wpPnbNmtt~UYyGj7YlDqwMCccqWwhJgs1OafIZqMKxVMsz-cjkppvscihviJTgNLWGOU0NV2MneVzCv9CTyyYD0C4j~Nu4IZOvTeA4prnWZH6Oyg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2011-11-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Coates, Brad S." /><meta name="citation_author_institution" content="USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_author" content="Hellmich, Richard L." /><meta name="citation_author_institution" content="USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_author_institution" content="Department of Entomology, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_author" content="Grant, David M." /><meta name="citation_author_institution" content="USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_author_institution" content="Department of Agronomy, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_author" content="Abel, Craig A." /><meta name="citation_author_institution" content="USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_author_institution" content="Department of Entomology, Iowa State University, Ames, IA 50011, USA" /><meta name="citation_title" content="Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous Lep1 Helitrons" /><meta name="citation_firstpage" content="11" /><meta name="citation_lastpage" content="21" /><meta name="citation_doi" content="10.1093/dnares/dsr038" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2012/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Rolling-circle transposons in eukaryotes; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=8714-9; " /><meta name="citation_reference" content="citation_title=Recent amplification of miniature inverted-repeat transposable elements in the vector mosquito Culex pipiens: characterization of the Mimo family; Gene;  citation_year=2000;  citation_volume=250;  citation_pages=109-16; " /><meta name="citation_reference" content="citation_title=Helitrons on a roll: eukaryotic rolling-circle transposons; Trends Genet.;  citation_year=2007;  citation_volume=23;  citation_pages=521-9; " /><meta name="citation_reference" content="citation_title=Gene movement by Helitron transposons contributes to haplotype variability of maize; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=9068-73; " /><meta name="citation_reference" content="citation_title=Sequencing of Aspergillus nidulans and comparative analysis with A. fumigatus and A. oryzæ; Nature;  citation_year=2005;  citation_volume=438;  citation_pages=1105-15; " /><meta name="citation_reference" content="citation_title=Give-and-take: interactions between DNA transposons and their host plant genomes; Curr. Opin. Genet. Dev.;  citation_year=2009;  citation_volume=17;  citation_pages=486-92; " /><meta name="citation_reference" content="citation_title=Abundant and species specific DINE-1 transposable elements in 12 Drosophila genomes; Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=R39" /><meta name="citation_reference" content="citation_title=Structure-based discovery and description of plant and animal Helitrons; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=12832-7; " /><meta name="citation_reference" content="citation_title=Tourist: a large family of small inverted repeat elements frequently associated with maize genes; Plant Cell;  citation_year=1992;  citation_volume=4;  citation_pages=1283-94; " /><meta name="citation_reference" content="citation_title=Excision of Helitron transposons in maize; Genetics;  citation_year=2009;  citation_volume=182;  citation_pages=399-402; " /><meta name="citation_reference" content="citation_title=A transposon, Ping, is integrated into intron 4 of the DROOPING LEAF gene of rice, weakly reducing its expression and causing a mild drooping leaf phenotype; Plant Cell Physiol.;  citation_year=2008;  citation_volume=49;  citation_pages=1176-84; " /><meta name="citation_reference" content="citation_title=Alternative splicing induced by insertion of retrotransposons into the maize waxy gene; Plant Cell;  citation_year=1992;  citation_volume=4;  citation_pages=811-20; " /><meta name="citation_reference" content="citation_title=Recent amplification and impact of MITEs on the genome of grapevine (Vitis vinifera L.); Genome Biol. Evol.;  citation_year=2009;  citation_volume=1;  citation_pages=75-84; " /><meta name="citation_reference" content="citation_title=Treasures in the attic: rolling circle transposons discovered in eukaryotic genomes; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=8923-4; " /><meta name="citation_reference" content="citation_title=Intraspecific violation of genetic colinearity and its implications in maize; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=9573-8; " /><meta name="citation_reference" content="citation_title=Gene duplication and exon shuffling by Helitron-like transposons generate intraspecies diversity in maize; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=997-1002; " /><meta name="citation_reference" content="citation_title=Helitrons contribute to the lack of gene colinearity observed in modern maize inbreds; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=9993-4; " /><meta name="citation_reference" content="citation_title=Coding sequences of functioning human genes derived entirely from mobile element sequences; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=16825-30; " /><meta name="citation_reference" content="citation_title=Transposable elements, gene creation and genome rearrangement in flowering plants; Curr. Opin. Gene Devel.;  citation_year=2005;  citation_volume=15;  citation_pages=621-7; " /><meta name="citation_reference" content="citation_title=Distribution, diversity, evolution, and survival of Helitrons in the maize genome; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=19922-7; " /><meta name="citation_reference" content="citation_title=A novel class of Helitron-related transposable elements in maize contain portions of multiple pseudogenes; Plant Mol. Biol.;  citation_year=2005;  citation_volume=57;  citation_pages=115-27; " /><meta name="citation_reference" content="citation_title=Alternative Ac/Ds transposition induces major chromosomal rearrangements in maize; Genes Dev.;  citation_year=2009;  citation_volume=23;  citation_pages=755-65; " /><meta name="citation_reference" content="citation_title=The origin and behavior of mutable loci in maize; Proc. Natl. Acad. Sci. USA;  citation_year=1950;  citation_volume=36;  citation_pages=344-55; " /><meta name="citation_reference" content="citation_title=Retrotransposon-mediated genome evolution on a local ecological scale; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=6250-2; " /><meta name="citation_reference" content="citation_title=The molecular basis of PM hybrid dysgenesis: the role of the P element, a P-strain-specific transposon family; Cell;  citation_year=1982;  citation_volume=29;  citation_pages=995-1004; " /><meta name="citation_reference" content="citation_title=Evolutionary genetics: jumping into a new species; Curr. Biol.;  citation_year=2006;  citation_volume=16;  citation_pages=R890-2; " /><meta name="citation_reference" content="citation_title=Genome-wide patterns of adaptation to temperate environments associated with transposable elements in Drosophila; PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000905" /><meta name="citation_reference" content="citation_title=Identification of a gene associated with Bt resistance in Heliothis virescens; Science;  citation_year=2001;  citation_volume=293;  citation_pages=857-60; " /><meta name="citation_reference" content="citation_title=Extensive synteny conservation of holocentric chromosomes in Lepidoptera despite high rates of local genome rearrangements; Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=7600-5; " /><meta name="citation_reference" content="citation_title=The genome of a lepidopteran model insect, the silkworm Bombyx mori; citation_author=International Silkworm Genome Consortium; Insect Biochem. Mol. Biol.;  citation_year=2008;  citation_volume=38;  citation_pages=1036-45; " /><meta name="citation_reference" content="citation_title=The genome sequence of silkworm, Bombyx mori; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=27-36; " /><meta name="citation_reference" content="citation_title=A draft sequence for the genome of the domesticated silkworm (Bombyx mori); Science;  citation_year=2004;  citation_volume=306;  citation_pages=1937-40; " /><meta name="citation_reference" content="citation_title=Genome-wide screening and characterization of transposable elements and their distribution in the silkworm, Bombyx mori; Insect Biochem. Mol. Biol.;  citation_year=2009;  citation_volume=38;  citation_pages=1046-57; " /><meta name="citation_reference" content="citation_title=SINEs: short interspersed repeated elements of the eukaryotic genome; Trends Ecol. Evol.;  citation_year=1991;  citation_volume=6;  citation_pages=358-61; " /><meta name="citation_reference" content="citation_title=A Helitron-like transposon superfamily from Lepidoptera disrupts (GAAA)n microsatellites and is responsible for flanking sequence similarity within a microsatellite family; J. Mol. Evol.;  citation_year=2010;  citation_volume=70;  citation_pages=278-88; " /><meta name="citation_reference" content="citation_title=Characterization of the P25 silk gene and associated insertion elements in Galleria melonella; Gene;  citation_year=1998;  citation_volume=209;  citation_pages=157-65; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of multilocus microsatellite arrangements in the genome of the butterfly Bicyclus anynana, with implications for other Lepidoptera; Heredity;  citation_year=2007;  citation_volume=98;  citation_pages=320-8; " /><meta name="citation_reference" content="citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT; Nucl. Acids Symp. Ser.;  citation_year=1999;  citation_volume=41;  citation_pages=95-8; " /><meta name="citation_reference" content="citation_title=MEGA4: molecular evolutionary genetic analysis MEGA. software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=Mfold web server for nucleic acid folding and hybridization prediction; Nucl. Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3406-15; " /><meta name="citation_reference" content="citation_title=CMap 1,01: a comparative mapping application for the Internet; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=3042-2; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=Blast2go: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=High-throughput functional annotation and data mining with the Blast2GO suite; Nucl. Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=3421-35; " /><meta name="citation_reference" content="citation_title=A novel class of miniature inverted repeat transposable elements MITEs that contain hitchhiking GTCYn microsatellites; Insect Mol. Biol.;  citation_year=2011;  citation_volume=20;  citation_pages=15-27; " /><meta name="citation_reference" content="citation_title=Model-based identification of Helitrons results in a new classification of their families in Arabidopsis thaliana; Gene;  citation_year=2007;  citation_volume=403;  citation_pages=18-28; " /><meta name="citation_reference" content="citation_title=A comparative computational analysis of nonautonomous Helitron elements between maize and rice; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=467" /><meta name="citation_reference" content="citation_title=The replication of plastid microcircles involved rolling circle intermediates; Nucl. Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=1991-2002; " /><meta name="citation_reference" content="citation_title=Cytosine methylation and the ecology of intragenomic parasites; Trends Genet.;  citation_year=1997;  citation_volume=13;  citation_pages=335-40; " /><meta name="citation_reference" content="citation_title=Selfish DNA-the ultimate parasite; Nature;  citation_year=1980;  citation_volume=284;  citation_pages=604-7; " /><meta name="citation_reference" content="citation_title=Short interspersed elements SINEs are a major source of canine genome diversity; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1798-808; " /><meta name="citation_reference" content="citation_title=A novel SINE family occurs frequently in both genomic DNA and transcribed sequences in ixodid ticks from the arthropod sub-phylum Chelicerata; Gene;  citation_year=2008;  citation_volume=415;  citation_pages=13-22; " /><meta name="citation_reference" content="citation_title=Paucity of chimeric gene-transposable element transcripts in the Drosophila melanogaster genome; BMC Biol.;  citation_year=2005;  citation_volume=3;  citation_pages=24" /><meta name="citation_reference" content="citation_title=Transposable elements in mammals promote regulatory variation and diversification of genes with specialized functions; Trends Genet.;  citation_year=2003;  citation_volume=19;  citation_pages=530-6; " /><meta name="citation_reference" content="citation_title=De novo synthesis of an intron by the maize transposable element dissociation; Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=12150-4; " /><meta name="citation_reference" content="citation_title=Transposable element insertions respecify alternative exon splicing in three Drosophila myosin heavy chain mutants; Genetics;  citation_year=1988;  citation_volume=150;  citation_pages=1105-14; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/1/11/1310455/dsr038.pdf" /><meta name="description" content="Abstract.  Transposable elements (TEs) can affect the structure of genomes through their acquisition and transposition of novel DNA sequences. The 134-bp repeti" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/1/11/484721" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/1/11/484721" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsr038';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484721" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQFgHk8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNvYXRlcywgQnJhZCBTLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBDb3JuIEluc2VjdCBhbmQgQ3JvcCBHZW5ldGljcyBSZXNlYXJjaCBVbml0LCAxMTMgR2VuZXRpY3MgTGFib3JhdG9yeSwgSW93YSBTdGF0ZSBVbml2ZXJzaXR5LCBBbWVzLCBJQSA1MDAxMSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGVsbG1pY2gsIFJpY2hhcmQgTC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVU0RBLUFSUywgQ29ybiBJbnNlY3QgYW5kIENyb3AgR2VuZXRpY3MgUmVzZWFyY2ggVW5pdCwgMTEzIEdlbmV0aWNzIExhYm9yYXRvcnksIElvd2EgU3RhdGUgVW5pdmVyc2l0eSwgQW1lcywgSUEgNTAwMTEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRW50b21vbG9neSwgSW93YSBTdGF0ZSBVbml2ZXJzaXR5LCBBbWVzLCBJQSA1MDAxMSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3JhbnQsIERhdmlkIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVVNEQS1BUlMsIENvcm4gSW5zZWN0IGFuZCBDcm9wIEdlbmV0aWNzIFJlc2VhcmNoIFVuaXQsIDExMyBHZW5ldGljcyBMYWJvcmF0b3J5LCBJb3dhIFN0YXRlIFVuaXZlcnNpdHksIEFtZXMsIElBIDUwMDExLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncm9ub215LCBJb3dhIFN0YXRlIFVuaXZlcnNpdHksIEFtZXMsIElBIDUwMDExLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmVsLCBDcmFpZyBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBDb3JuIEluc2VjdCBhbmQgQ3JvcCBHZW5ldGljcyBSZXNlYXJjaCBVbml0LCAxMTMgR2VuZXRpY3MgTGFib3JhdG9yeSwgSW93YSBTdGF0ZSBVbml2ZXJzaXR5LCBBbWVzLCBJQSA1MDAxMSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFbnRvbW9sb2d5LCBJb3dhIFN0YXRlIFVuaXZlcnNpdHksIEFtZXMsIElBIDUwMDExLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik1vYmlsaXppbmcgdGhlIEdlbm9tZSBvZiBMZXBpZG9wdGVyYSB0aHJvdWdoIE5vdmVsIFNlcXVlbmNlIEdhaW5zIGFuZCBFbmQgQ3JlYXRpb24gYnkgTm9uLWF1dG9ub21vdXMgTGVwMSBIZWxpdHJvbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGxpbmctY2lyY2xlIHRyYW5zcG9zb25zIGluIGV1a2FyeW90ZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9ODcxNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQgYW1wbGlmaWNhdGlvbiBvZiBtaW5pYXR1cmUgaW52ZXJ0ZWQtcmVwZWF0IHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiB0aGUgdmVjdG9yIG1vc3F1aXRvIEN1bGV4IHBpcGllbnM6IGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIE1pbW8gZmFtaWx5OyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI1MDsgIGNpdGF0aW9uX3BhZ2VzPTEwOS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVsaXRyb25zIG9uIGEgcm9sbDogZXVrYXJ5b3RpYyByb2xsaW5nLWNpcmNsZSB0cmFuc3Bvc29uczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTUyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG1vdmVtZW50IGJ5IEhlbGl0cm9uIHRyYW5zcG9zb25zIGNvbnRyaWJ1dGVzIHRvIGhhcGxvdHlwZSB2YXJpYWJpbGl0eSBvZiBtYWl6ZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9OTA2OC03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBvZiBBc3BlcmdpbGx1cyBuaWR1bGFucyBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXMgd2l0aCBBLiBmdW1pZ2F0dXMgYW5kIEEuIG9yeXrDpjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzODsgIGNpdGF0aW9uX3BhZ2VzPTExMDUtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdpdmUtYW5kLXRha2U6IGludGVyYWN0aW9ucyBiZXR3ZWVuIEROQSB0cmFuc3Bvc29ucyBhbmQgdGhlaXIgaG9zdCBwbGFudCBnZW5vbWVzOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ni05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJ1bmRhbnQgYW5kIHNwZWNpZXMgc3BlY2lmaWMgRElORS0xIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiAxMiBEcm9zb3BoaWxhIGdlbm9tZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9UjM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlLWJhc2VkIGRpc2NvdmVyeSBhbmQgZGVzY3JpcHRpb24gb2YgcGxhbnQgYW5kIGFuaW1hbCBIZWxpdHJvbnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTEyODMyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvdXJpc3Q6IGEgbGFyZ2UgZmFtaWx5IG9mIHNtYWxsIGludmVydGVkIHJlcGVhdCBlbGVtZW50cyBmcmVxdWVudGx5IGFzc29jaWF0ZWQgd2l0aCBtYWl6ZSBnZW5lczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTI4My05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhjaXNpb24gb2YgSGVsaXRyb24gdHJhbnNwb3NvbnMgaW4gbWFpemU7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MjsgIGNpdGF0aW9uX3BhZ2VzPTM5OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdHJhbnNwb3NvbiwgUGluZywgaXMgaW50ZWdyYXRlZCBpbnRvIGludHJvbiA0IG9mIHRoZSBEUk9PUElORyBMRUFGIGdlbmUgb2YgcmljZSwgd2Vha2x5IHJlZHVjaW5nIGl0cyBleHByZXNzaW9uIGFuZCBjYXVzaW5nIGEgbWlsZCBkcm9vcGluZyBsZWFmIHBoZW5vdHlwZTsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTExNzYtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGluZHVjZWQgYnkgaW5zZXJ0aW9uIG9mIHJldHJvdHJhbnNwb3NvbnMgaW50byB0aGUgbWFpemUgd2F4eSBnZW5lOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz04MTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY2VudCBhbXBsaWZpY2F0aW9uIGFuZCBpbXBhY3Qgb2YgTUlURXMgb24gdGhlIGdlbm9tZSBvZiBncmFwZXZpbmUgKFZpdGlzIHZpbmlmZXJhIEwuKTsgR2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz03NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlYXN1cmVzIGluIHRoZSBhdHRpYzogcm9sbGluZyBjaXJjbGUgdHJhbnNwb3NvbnMgZGlzY292ZXJlZCBpbiBldWthcnlvdGljIGdlbm9tZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9ODkyMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyYXNwZWNpZmljIHZpb2xhdGlvbiBvZiBnZW5ldGljIGNvbGluZWFyaXR5IGFuZCBpdHMgaW1wbGljYXRpb25zIGluIG1haXplOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTk1NzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBkdXBsaWNhdGlvbiBhbmQgZXhvbiBzaHVmZmxpbmcgYnkgSGVsaXRyb24tbGlrZSB0cmFuc3Bvc29ucyBnZW5lcmF0ZSBpbnRyYXNwZWNpZXMgZGl2ZXJzaXR5IGluIG1haXplOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTk5Ny0xMDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZWxpdHJvbnMgY29udHJpYnV0ZSB0byB0aGUgbGFjayBvZiBnZW5lIGNvbGluZWFyaXR5IG9ic2VydmVkIGluIG1vZGVybiBtYWl6ZSBpbmJyZWRzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz05OTkzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZGluZyBzZXF1ZW5jZXMgb2YgZnVuY3Rpb25pbmcgaHVtYW4gZ2VuZXMgZGVyaXZlZCBlbnRpcmVseSBmcm9tIG1vYmlsZSBlbGVtZW50IHNlcXVlbmNlczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTY4MjUtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYWJsZSBlbGVtZW50cywgZ2VuZSBjcmVhdGlvbiBhbmQgZ2Vub21lIHJlYXJyYW5nZW1lbnQgaW4gZmxvd2VyaW5nIHBsYW50czsgQ3Vyci4gT3Bpbi4gR2VuZSBEZXZlbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz02MjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdHJpYnV0aW9uLCBkaXZlcnNpdHksIGV2b2x1dGlvbiwgYW5kIHN1cnZpdmFsIG9mIEhlbGl0cm9ucyBpbiB0aGUgbWFpemUgZ2Vub21lOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xOTkyMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGNsYXNzIG9mIEhlbGl0cm9uLXJlbGF0ZWQgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIG1haXplIGNvbnRhaW4gcG9ydGlvbnMgb2YgbXVsdGlwbGUgcHNldWRvZ2VuZXM7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0xMTUtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIEFjL0RzIHRyYW5zcG9zaXRpb24gaW5kdWNlcyBtYWpvciBjaHJvbW9zb21hbCByZWFycmFuZ2VtZW50cyBpbiBtYWl6ZTsgR2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9yaWdpbiBhbmQgYmVoYXZpb3Igb2YgbXV0YWJsZSBsb2NpIGluIG1haXplOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk1MDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTM0NC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV0cm90cmFuc3Bvc29uLW1lZGlhdGVkIGdlbm9tZSBldm9sdXRpb24gb24gYSBsb2NhbCBlY29sb2dpY2FsIHNjYWxlOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTYyNTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vbGVjdWxhciBiYXNpcyBvZiBQTSBoeWJyaWQgZHlzZ2VuZXNpczogdGhlIHJvbGUgb2YgdGhlIFAgZWxlbWVudCwgYSBQLXN0cmFpbi1zcGVjaWZpYyB0cmFuc3Bvc29uIGZhbWlseTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTk5NS0xMDA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZ2VuZXRpY3M6IGp1bXBpbmcgaW50byBhIG5ldyBzcGVjaWVzOyBDdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPVI4OTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgcGF0dGVybnMgb2YgYWRhcHRhdGlvbiB0byB0ZW1wZXJhdGUgZW52aXJvbm1lbnRzIGFzc29jaWF0ZWQgd2l0aCB0cmFuc3Bvc2FibGUgZWxlbWVudHMgaW4gRHJvc29waGlsYTsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwOTA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBnZW5lIGFzc29jaWF0ZWQgd2l0aCBCdCByZXNpc3RhbmNlIGluIEhlbGlvdGhpcyB2aXJlc2NlbnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjkzOyAgY2l0YXRpb25fcGFnZXM9ODU3LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgc3ludGVueSBjb25zZXJ2YXRpb24gb2YgaG9sb2NlbnRyaWMgY2hyb21vc29tZXMgaW4gTGVwaWRvcHRlcmEgZGVzcGl0ZSBoaWdoIHJhdGVzIG9mIGxvY2FsIGdlbm9tZSByZWFycmFuZ2VtZW50czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NzYwMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIGEgbGVwaWRvcHRlcmFuIG1vZGVsIGluc2VjdCwgdGhlIHNpbGt3b3JtIEJvbWJ5eCBtb3JpOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBTaWxrd29ybSBHZW5vbWUgQ29uc29ydGl1bTsgSW5zZWN0IEJpb2NoZW0uIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMDM2LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIHNpbGt3b3JtLCBCb21ieXggbW9yaTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkcmFmdCBzZXF1ZW5jZSBmb3IgdGhlIGdlbm9tZSBvZiB0aGUgZG9tZXN0aWNhdGVkIHNpbGt3b3JtIChCb21ieXggbW9yaSk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA2OyAgY2l0YXRpb25fcGFnZXM9MTkzNy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgc2NyZWVuaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBhbmQgdGhlaXIgZGlzdHJpYnV0aW9uIGluIHRoZSBzaWxrd29ybSwgQm9tYnl4IG1vcmk7IEluc2VjdCBCaW9jaGVtLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTA0Ni01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U0lORXM6IHNob3J0IGludGVyc3BlcnNlZCByZXBlYXRlZCBlbGVtZW50cyBvZiB0aGUgZXVrYXJ5b3RpYyBnZW5vbWU7IFRyZW5kcyBFY29sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzU4LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEhlbGl0cm9uLWxpa2UgdHJhbnNwb3NvbiBzdXBlcmZhbWlseSBmcm9tIExlcGlkb3B0ZXJhIGRpc3J1cHRzIChHQUFBKW4gbWljcm9zYXRlbGxpdGVzIGFuZCBpcyByZXNwb25zaWJsZSBmb3IgZmxhbmtpbmcgc2VxdWVuY2Ugc2ltaWxhcml0eSB3aXRoaW4gYSBtaWNyb3NhdGVsbGl0ZSBmYW1pbHk7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0yNzgtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIFAyNSBzaWxrIGdlbmUgYW5kIGFzc29jaWF0ZWQgaW5zZXJ0aW9uIGVsZW1lbnRzIGluIEdhbGxlcmlhIG1lbG9uZWxsYTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDk7ICBjaXRhdGlvbl9wYWdlcz0xNTctNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBkeW5hbWljcyBvZiBtdWx0aWxvY3VzIG1pY3Jvc2F0ZWxsaXRlIGFycmFuZ2VtZW50cyBpbiB0aGUgZ2Vub21lIG9mIHRoZSBidXR0ZXJmbHkgQmljeWNsdXMgYW55bmFuYSwgd2l0aCBpbXBsaWNhdGlvbnMgZm9yIG90aGVyIExlcGlkb3B0ZXJhOyBIZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTMyMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9FZGl0OiBhIHVzZXItZnJpZW5kbHkgYmlvbG9naWNhbCBzZXF1ZW5jZSBhbGlnbm1lbnQgZWRpdG9yIGFuZCBhbmFseXNpcyBwcm9ncmFtIGZvciBXaW5kb3dzIDk1Lzk4L05UOyBOdWNsLiBBY2lkcyBTeW1wLiBTZXIuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9OTUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTQ6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpYyBhbmFseXNpcyBNRUdBLiBzb2Z0d2FyZSB2ZXJzaW9uIDQuMDsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWZvbGQgd2ViIHNlcnZlciBmb3IgbnVjbGVpYyBhY2lkIGZvbGRpbmcgYW5kIGh5YnJpZGl6YXRpb24gcHJlZGljdGlvbjsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM0MDYtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNNYXAgMSwwMTogYSBjb21wYXJhdGl2ZSBtYXBwaW5nIGFwcGxpY2F0aW9uIGZvciB0aGUgSW50ZXJuZXQ7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzA0Mi0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DQVAzOiBhIEROQSBzZXF1ZW5jZSBhc3NlbWJseSBwcm9ncmFtOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ODY4LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJnbzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNjc0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBmdW5jdGlvbmFsIGFubm90YXRpb24gYW5kIGRhdGEgbWluaW5nIHdpdGggdGhlIEJsYXN0MkdPIHN1aXRlOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MzQyMS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBjbGFzcyBvZiBtaW5pYXR1cmUgaW52ZXJ0ZWQgcmVwZWF0IHRyYW5zcG9zYWJsZSBlbGVtZW50cyBNSVRFcyB0aGF0IGNvbnRhaW4gaGl0Y2hoaWtpbmcgR1RDWW4gbWljcm9zYXRlbGxpdGVzOyBJbnNlY3QgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE1LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlbC1iYXNlZCBpZGVudGlmaWNhdGlvbiBvZiBIZWxpdHJvbnMgcmVzdWx0cyBpbiBhIG5ldyBjbGFzc2lmaWNhdGlvbiBvZiB0aGVpciBmYW1pbGllcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00MDM7ICBjaXRhdGlvbl9wYWdlcz0xOC0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wYXJhdGl2ZSBjb21wdXRhdGlvbmFsIGFuYWx5c2lzIG9mIG5vbmF1dG9ub21vdXMgSGVsaXRyb24gZWxlbWVudHMgYmV0d2VlbiBtYWl6ZSBhbmQgcmljZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVwbGljYXRpb24gb2YgcGxhc3RpZCBtaWNyb2NpcmNsZXMgaW52b2x2ZWQgcm9sbGluZyBjaXJjbGUgaW50ZXJtZWRpYXRlczsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTE5OTEtMjAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b3NpbmUgbWV0aHlsYXRpb24gYW5kIHRoZSBlY29sb2d5IG9mIGludHJhZ2Vub21pYyBwYXJhc2l0ZXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMzUtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGZpc2ggRE5BLXRoZSB1bHRpbWF0ZSBwYXJhc2l0ZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTI4NDsgIGNpdGF0aW9uX3BhZ2VzPTYwNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG9ydCBpbnRlcnNwZXJzZWQgZWxlbWVudHMgU0lORXMgYXJlIGEgbWFqb3Igc291cmNlIG9mIGNhbmluZSBnZW5vbWUgZGl2ZXJzaXR5OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3OTgtODA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIFNJTkUgZmFtaWx5IG9jY3VycyBmcmVxdWVudGx5IGluIGJvdGggZ2Vub21pYyBETkEgYW5kIHRyYW5zY3JpYmVkIHNlcXVlbmNlcyBpbiBpeG9kaWQgdGlja3MgZnJvbSB0aGUgYXJ0aHJvcG9kIHN1Yi1waHlsdW0gQ2hlbGljZXJhdGE7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDE1OyAgY2l0YXRpb25fcGFnZXM9MTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdWNpdHkgb2YgY2hpbWVyaWMgZ2VuZS10cmFuc3Bvc2FibGUgZWxlbWVudCB0cmFuc2NyaXB0cyBpbiB0aGUgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgZ2Vub21lOyBCTUMgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIG1hbW1hbHMgcHJvbW90ZSByZWd1bGF0b3J5IHZhcmlhdGlvbiBhbmQgZGl2ZXJzaWZpY2F0aW9uIG9mIGdlbmVzIHdpdGggc3BlY2lhbGl6ZWQgZnVuY3Rpb25zOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTMwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gc3ludGhlc2lzIG9mIGFuIGludHJvbiBieSB0aGUgbWFpemUgdHJhbnNwb3NhYmxlIGVsZW1lbnQgZGlzc29jaWF0aW9uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTEyMTUwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYWJsZSBlbGVtZW50IGluc2VydGlvbnMgcmVzcGVjaWZ5IGFsdGVybmF0aXZlIGV4b24gc3BsaWNpbmcgaW4gdGhyZWUgRHJvc29waGlsYSBteW9zaW4gaGVhdnkgY2hhaW4gbXV0YW50czsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9MTEwNS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzEvMTEvMTMxMDQ1NS9kc3IwMzgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRyYW5zcG9zYWJsZSBlbGVtZW50cyAoVEVzKSBjYW4gYWZmZWN0IHRoZSBzdHJ1Y3R1cmUgb2YgZ2Vub21lcyB0aHJvdWdoIHRoZWlyIGFjcXVpc2l0aW9uIGFuZCB0cmFuc3Bvc2l0aW9uIG9mIG5vdmVsIEROQSBzZXF1ZW5jZXMuIFRoZSAxMzQtYnAgcmVwZXRpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODQ3MjEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2RkvdhcnkgCXB1kdfyGpinFUmEOPTw=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f11%2f484721" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f11%2f484721" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=1509238784&amp;Signature=Qha2YYpJxYxoFwATLQRwfzIPuY2er9j-C1ngBM34DGe4ntbmgNO~rdgEdQGnx04Z9~VAwu2EFGNQn-rUAv42BMIEIvx5PU5pl97hsGBwVbxEtC~y6eaqXlZNCS0aNScC4dO8lCBN2rhwjqz-G7gstz4xI7uiLVo4IX8uI~wjcynp-IdAkIGPOL-Y1t5J0bJ8R1AbonFtsdPyt4xeqwGcxKi37b-ooMe9w033EkQx9xHZXkf7g9wEF5zY2YtgTg8K1jvpRoq9gBBqI5rMlvq8jF3nTCH3~IdWt0OpBDmjROXUkvbd54TwrJvNGXBWioEbeXZxKK7yGR6g98YPT45WAQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/1">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7365617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365617">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365619">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365619">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365624">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365624">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365635">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365635">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365658">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365658">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365660">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365660">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365662">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365662">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7365664">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7365664">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/1/1/484514">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/1/23/484949">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous <em>Lep</em>1 <em>Helitrons</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Brad S. Coates</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Brad S. Coates</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +515 294-0668. Fax. +515 294-2265. Email: <a href="mailto:brad.coates@ars.udsa.gov" target="_blank">brad.coates@ars.udsa.gov</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Brad+S.+Coates">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Coates B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Coates author:B.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Richard L. Hellmich</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Richard L. Hellmich</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Entomology</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Richard+L.+Hellmich">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hellmich R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hellmich author:R.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">David M. Grant</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David M. Grant</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Agronomy</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=David+M.+Grant">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Grant D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Grant author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Craig A. Abel</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Craig A. Abel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Corn Insect and Crop Genetics Research Unit, 113 Genetics Laboratory</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Entomology</div>, <div class="insititution">Iowa State University</div>, <div class="addr-line">Ames, IA 50011</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Craig+A.+Abel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abel C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Abel author:C.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Yuji Kohara</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 11–21, <a href='https://doi.org/10.1093/dnares/dsr038'>https://doi.org/10.1093/dnares/dsr038</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 November 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 June 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 September 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484721" href="/dnaresearch/article-pdf/19/1/11/1310455/dsr038.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Brad S. Coates, Richard L. Hellmich, David M. Grant, Craig A. Abel;  Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous <em>Lep</em>1 <em>Helitrons</em>, <em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 11–21, <a href='https://doi.org/10.1093/dnares/dsr038'>https://doi.org/10.1093/dnares/dsr038</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484721?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484721?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484721?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484721?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484721?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484721?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484721?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Mobilizing%20the%20Genome%20of%20Lepidoptera%20through%20Novel%20Sequence%20Gains%20and%20End%20Creation%20by%20Non-autonomous%20%26lt%3Bem%26gt%3BLep%26lt%3B%2Fem%26gt%3B1%20%26lt%3Bem%26gt%3BHelitrons%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2011-11-15&amp;volumeNum=19&amp;issueNum=1&amp;author=Coates%2C%20Brad%20S.%3B%20Hellmich%2C%20Richard%20L.&amp;startPage=11&amp;endPage=21&amp;contentId=10.1093%2Fdnares%2Fdsr038&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f11%2f484721" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7365617 id="7365617" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Transposable elements (TEs) can affect the structure of genomes through their acquisition and transposition of novel DNA sequences. The 134-bp repetitive elements, <em>Lep</em>1, are conserved non-autonomous <em>Helitrons</em> in lepidopteran genomes that have characteristic 5′-CT and 3′-CTAY nucleotide termini, a 3′-terminal hairpin structure, a 5′- and 3′-subterminal inverted repeat (SIR), and integrations that occur between AT or TT nucleotides. <em>Lep</em>1 <em>Helitrons</em> have acquired and propagated sequences downstream of their 3′-CTAY termini that are 57–344-bp in length and have termini composed of a 3′-CTRR preceded by a 3′-hairpin structure and a region complementary to the 5′-SIR (3′-SIRb). Features of both the <em>Lep</em>1 <em>Helitron</em> and multiple acquired sequences indicate that secondary structures at the 3′-terminus may have a role in rolling circle replication or genome integration mechanisms, and are a prerequisite for novel end creation by <em>Helitron</em>-like TEs. The preferential integration of <em>Lep</em>1 <em>Helitrons</em> in proximity to gene-coding regions results in the creation of genetic novelty that is shown to impact gene structure and function through the introduction of novel exon sequence (exon shuffling). These findings are important in understanding the structural requirements of genomic DNA sequences that are acquired and transposed by <em>Helitron-</em>like TEs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Helitron"><em>Helitron</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence gain&quot;">sequence gain</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome rearrangement&quot;">genome rearrangement</a></div>                            <h2 scrollto-destination=7365619 id="7365619" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Helitrons</em> are class II transposable elements (TEs) that are proposed to propagate by rolling circle replication (RCR) at the DNA level<sup><a reveal-id="DSR038C1" data-open="DSR038C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and are dependent upon the function of Replicase/Helicase (RepHel) proteins for autonomous transposition.<sup><a reveal-id="DSR038C1" data-open="DSR038C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSR038C2" data-open="DSR038C2" class="link link-ref link-reveal xref-bibr">2</a></sup><em>Helitrons</em> show a high degree of sequence plasticity, such that computational predications mainly rely upon the identification of conserved 5′-CT and 3′-CTRR termini,<sup><a reveal-id="DSR038C3" data-open="DSR038C3" class="link link-ref link-reveal xref-bibr">3</a></sup> a 6–20-bp stem-loop structure near the 3′-terminus,<sup><a reveal-id="DSR038C4" data-open="DSR038C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR038C5" data-open="DSR038C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and predicted integration between AT<sup><a reveal-id="DSR038C1" data-open="DSR038C1" class="link link-ref link-reveal xref-bibr">1</a></sup> or TT nucleotides.<sup><a reveal-id="DSR038C3" data-open="DSR038C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Non-autonomous <em>Helitrons</em> are small in size due to lack of an internal protein-coding region, but often remain mobile through the retention of functional <em>trans</em>-acting RepHel proteins.<sup><a reveal-id="DSR038C1" data-open="DSR038C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSR038C2" data-open="DSR038C2" class="link link-ref link-reveal xref-bibr">2</a></sup> A high copy number of non-autonomous <em>Helitrons</em> within genomes likely results from evasion of host repression.<sup><a reveal-id="DSR038C6" data-open="DSR038C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The observed paucities in sequence variation among <em>Helitrons</em> appears indicative of recent bursts in transposition.<sup><a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR038C8" data-open="DSR038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Certain groups of class II DNA transposons and <em>Helitrons</em> integrate frequently in proximity to protein-coding regions<sup><a reveal-id="DSR038C9" data-open="DSR038C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and can affect the structure and function of genes and gene products.<sup><a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR038C10" data-open="DSR038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> These integrations can result in the modification of transcriptional efficiency,<sup><a reveal-id="DSR038C11" data-open="DSR038C11" class="link link-ref link-reveal xref-bibr">11</a></sup> as well as introduce transcript splice variation and polyadenylation sites, changes in transcription start and stop sites, and incorporation of novel exon sequence.<sup><a reveal-id="DSR038C12" data-open="DSR038C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR038C13" data-open="DSR038C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p><em>Helitrons</em> are potent modifiers of genome structure and function due to frequent acquisition and transposition of host genomic DNA, which oftentimes results in exon shuffling or the duplication of gene sequences.<sup><a reveal-id="DSR038C4" data-open="DSR038C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR038C14  DSR038C18" data-open="DSR038C14  DSR038C18" class="link link-ref link-reveal xref-bibr">14–18</a></sup> The mechanism by which <em>Helitrons</em> acquire novel sequence remains largely unknown,<sup><a reveal-id="DSR038C19" data-open="DSR038C19" class="link link-ref link-reveal xref-bibr">19</a></sup> but is hypothesized to occur at the DNA level due to transposition of both intron and exon sequences. Furthermore, <em>Helitrons</em> are often chimeric constructs that have acquired DNA from multiple independent loci,<sup><a reveal-id="DSR038C16" data-open="DSR038C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR038C17" data-open="DSR038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> which may occur by a step-wise addition of novel 5′- and 3′-ends that are compatible with the minimal requirements for functioning during RCR.<sup><a reveal-id="DSR038C6" data-open="DSR038C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR038C10" data-open="DSR038C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> These instances indicate that class II TEs and <em>Helitrons</em> participate in the rearrangement and duplication of genome regions and contribute to the evolution of novel eukaryotic genome functions.<sup><a reveal-id="DSR038C21" data-open="DSR038C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSR038C22" data-open="DSR038C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>TE integration and excision mutations cause phenotypic variation at the individual and population scale,<sup><a reveal-id="DSR038C23" data-open="DSR038C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSR038C24" data-open="DSR038C24" class="link link-ref link-reveal xref-bibr">24</a></sup> may be contributing factors to speciation events<sup><a reveal-id="DSR038C25" data-open="DSR038C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSR038C26" data-open="DSR038C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and provide the genetic novelties for local adaptation via natural selection.<sup><a reveal-id="DSR038C27" data-open="DSR038C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The insect order Lepidoptera contains the second largest number of species on earth, some of which cause widespread damage to crop plants. Short repetitive sequences and TEs are important players in the generation of genetic diversity and evolution of Lepidoptera due to integrations within genes,<sup><a reveal-id="DSR038C28" data-open="DSR038C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR038C29" data-open="DSR038C29" class="link link-ref link-reveal xref-bibr">29</a></sup> but the genome-wide affects of TE-derived mutations upon genetic and phenotypic variation remain relatively unknown. The silkmoth, <em>Bombyx mori</em>, is the lepidopteran model species whose ∼420 Mb genome sequence assembly is composed of ∼43.6% repetitive DNA,<sup><a reveal-id="DSR038C30" data-open="DSR038C30" class="link link-ref link-reveal xref-bibr">30</a></sup> with most being &lt;500 bp.<sup><a reveal-id="DSR038C31" data-open="DSR038C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR038C32" data-open="DSR038C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Nearly 13% of the <em>B. mori</em> genome comprise short interspersed nuclear elements (SINEs),<sup><a reveal-id="DSR038C33" data-open="DSR038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which are class I TEs derived from tRNA, 5S rRNA or 7SL RNA-like sequences that propagate by retrotransposition.<sup><a reveal-id="DSR038C34" data-open="DSR038C34" class="link link-ref link-reveal xref-bibr">34</a></sup> In contrast, DNA-based class II TEs occupy ∼3% of the <em>B. mori</em> genome (<em>Helitrons</em> 0.1%),<sup><a reveal-id="DSR038C33" data-open="DSR038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and are mostly located within introns and non-coding DNA of Lepidoptera.<sup><a reveal-id="DSR038C35" data-open="DSR038C35" class="link link-ref link-reveal xref-bibr">35</a></sup> This dearth of TE knowledge among lepidopteran species has resulted in difficulty in interpreting their role in the generation of structural and functional genome variance.</p><p>The <em>Lep</em>1 box is a conserved 99-bp repetitive DNA sequence originally described within intron and untranslated regions from eight lepidopteran species.<sup><a reveal-id="DSR038C36" data-open="DSR038C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The <em>Lep</em>1 box retains homology +10 to −50 of the core repeat, which was later described as the 134-bp lepidopteran-specific common sequence 3 (LSCS3).<sup><a reveal-id="DSR038C37" data-open="DSR038C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In the following, we annotate the <em>Lep</em>1 element as a <em>Helitron</em>-like TE and indicate that multiple novel DNA sequences have been acquired and propagated by the <em>Lep</em>1 <em>Helitron</em>. The conservation of predicted secondary structures between the ancestral <em>Helitron</em> and an acquired terminal region offers significant insight into the mode of <em>Helitron</em> propagation and features within the acquired sequence required for propagation. <em>Lep</em>1 <em>Helitrons</em> and acquired sequences co-localize with gene-coding regions in the <em>B. mori</em> genome, cause structural gene mutations, and are important mediators of genome-wide mutation.</p>                            <h2 scrollto-destination=7365624 id="7365624" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7365625 id="7365625" class="section-title" data-legacy-id=s2a>2.1. Annotation of the Lep1 Helitron</h3>
<p>All annotations are made with respect to the reverse complement of the LSCS3 sequence<sup><a reveal-id="DSR038C37" data-open="DSR038C37" class="link link-ref link-reveal xref-bibr">37</a></sup> that includes the 99-bp <em>Lep</em>1 box<sup><a reveal-id="DSR038C36" data-open="DSR038C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and from hereon is referred to as <em>Lep</em>1. Sequences showing homology to <em>Lep</em>1 were retrieved from the GenBank non-redundant (nr) nucleotide database via a BLASTn search that used <em>Lep</em>1 as the query (conducted 11-04-2010), with output filtered for ≥70% homology over ≥100 bp. Similarly, GenBank dbEST accessions for species of Lepidoptera were downloaded in FASTA format (10 November 2010), imported into a local databases using BioEdit,<sup><a reveal-id="DSR038C38" data-open="DSR038C38" class="link link-ref link-reveal xref-bibr">38</a></sup> queried using <em>Lep</em>1, and results filtered and aligned as described previously. All lepidopteran DNA sequence accessions that passed filter criteria were downloaded in FASTA format, imported into the MEGA 5.0 sequence alignment application,<sup><a reveal-id="DSR038C39" data-open="DSR038C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and a multiple sequence alignment was made using the ClustalW algorithm (default parameters: gap opening penalty, 15; gap extension penalty, 6.66; weight matrix IUB, and transition weight, 0.5) in the MEGA 5.0 alignment module.<sup><a reveal-id="DSR038C39" data-open="DSR038C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>Sequence homologies flanking the <em>Lep</em>1 sequence were identified by performing an ‘all vs. all’ search using the BLASTn algorithm using all nr and dbEST ‘hits’ to <em>Lep</em>1. The regions of intraspecific DNA sequence homology were extracted from accessions using a custom PERL script, and then used as input into the Mfold DNA secondary structure server<sup><a reveal-id="DSR038C40" data-open="DSR038C40" class="link link-ref link-reveal xref-bibr">40</a></sup> (<a class="link link-uri" href="http://mfold.rna.albany.edu/?q=mfold/DNA-Folding-Form">http://mfold.rna.albany.edu/?q=mfold/DNA-Folding-Form</a>) with the partial function and pair probabilities = 25°C.</p>                            <h3 scrollto-destination=7365628 id="7365628" class="section-title" data-legacy-id=s2b>2.2. Estimation of Lep1 genome copy number and distribution</h3>
<p>Scaffolds from the <em>B. mori</em> whole genome sequence build v. 2.3 were downloaded from Kaikobase (<a class="link link-uri" href="http://sgp.dna.affrc.go.jp/pubdata/genomicsequences.html">http://sgp.dna.affrc.go.jp/pubdata/genomicsequences.html</a>; file assembledset.txt.gz) and imported into a local database using BioEdit.<sup><a reveal-id="DSR038C38" data-open="DSR038C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Build v. 2.3 was searched with <em>Lep</em>1 as the query using the BLASTn algorithm and results filtered for ‘hits’ showing ≥80% similarity over ≥50 bp. The putative Bm<em>Lep</em>1 integration positions were called the Bm<em>Lep</em>1 model v. 2.3, which was then merged with positions of the <em>B. mori</em> assembly v. 2.3 gene models (file: glean_cds_on_chr.gff at <a class="link link-uri" href="http://sgp.dna.affrc.go.jp/pubdata/genomicsequences.html">http://sgp.dna.affrc.go.jp/pubdata/genomicsequences.html</a>). The combined features were displayed using CMap.<sup><a reveal-id="DSR038C41" data-open="DSR038C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Sequence intervals for Bm<em>Lep</em>1 elements were retrieved from the assembled <em>B. mori</em> scaffold for chromosome 1 (Z chromosome) using a custom PERL script, reverse complements generated using the Sequence Manipulation Suite (SMS) at <a class="link link-uri" href="http://www.bioinformatics.org/sms2/rev_comp.html">http://www.bioinformatics.org/sms2/rev_comp.html</a>, and FASTA formatted text imported into MEGA 5.0.<sup><a reveal-id="DSR038C39" data-open="DSR038C39" class="link link-ref link-reveal xref-bibr">39</a></sup> A multiple sequence alignment was made for all <em>B. mori Lep</em>1 elements using parameters described earlier, gamma parameter estimated, and a maximum likelihood-based estimation of <em>Lep</em>1 phylogenetic relationship made using the general time reversible model of sequence evolution. Nucleotide sites were chosen using a partial deletion of missing characters (cut-off = 0.05), and all possible trees were interrogated using the Close-Neighbor-Interchange heuristic. Node support was acquired using 1000 bootstrap pseudoreplicates reported within a strict consensus tree.</p><p>The frequency of <em>Lep</em>1 integrations within species of Lepidoptera was investigated using full bacterial artificial chromosome (BAC) insert sequences from <em>Bicyclus anynana</em>, <em>B. mori</em>, <em>Heliconius melpomene</em>, <em>H. numata</em>, <em>Helicoverpa armigera</em>, <em>Papilio dardanus</em>, and <em>Spodoptera frugiperda</em>. These sequences were downloaded from NCBI, imported into BioEdit,<sup><a reveal-id="DSR038C38" data-open="DSR038C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and a search for <em>Lep</em>1 positions performed as described previously. The mean frequency of <em>Lep</em>1 integrations were calculated manually and significance of frequency differences among species was assessed using <em>F</em>-statistics (significance threshold <em>α</em> = 0.05).</p>                            <h3 scrollto-destination=7365631 id="7365631" class="section-title" data-legacy-id=s2c>2.3. Predictions of haplotype variation caused by Lep1 elements</h3>
<p>Nucleotide accessions from the NCBI nr database were used to query derived protein sequences within the nr protein database using the blastx algorithm and results filtered for ‘hits’ showing ≥50% identity. Proteins derived from orthologous lepidopteran genes that were not identified within the initial <em>Lep</em>1 screen (see 2.1), but present within the blastx output were compared manually to predict instances of copy number variation within or between species. <em>Lep</em>1copy number variation at orthologous loci among species was further investigated using integration/excision variation among cadherin gene sequences from <em>B. mori</em> (gene model: BTGIBMGA013616), <em>H. armigera</em> (GenBank accession: AY714876.1), and <em>Ostrinia</em><em>nubilalis</em> (DQ000165.1). Sequences were imported into a local database in BioEdit,<sup><a reveal-id="DSR038C38" data-open="DSR038C38" class="link link-ref link-reveal xref-bibr">38</a></sup> searched using with <em>Lep</em>1 as the query and alignment of exon 1, intron 1, and exon 2 using the MEGA 5.0 sequence alignment application<sup><a reveal-id="DSR038C39" data-open="DSR038C39" class="link link-ref link-reveal xref-bibr">39</a></sup> was created as described previously.</p>                            <h3 scrollto-destination=7365633 id="7365633" class="section-title" data-legacy-id=s2d>2.4. Predictions of Lep1 modification to gene structure De novo</h3>
<p>Acquisition of transcribed <em>Lep</em>1 sequences was performed for <em>O. nubilalis</em>, where total RNA was isolated from whole larvae using RNAagents kit (Promega, Madison, WI) and cDNA was synthesized using the SMART cDNA Synthesis Kit (Clontech, Mountain View, CA) according to manufacturer's instructions, RACE reactions using the CDSIII primer (Clontech) with On<em>Lep</em>1-f2 (5′-TAC TRA TAT TAT AAA GCT GAA GAG TT-3′) and SMART V with On<em>Lep</em>1-r (5′-GAT AAA TGG GCT ATC TAA CAC TGA AAG-3′), and amplified according to manufacturer's instructions. PCR fragments were cloned, and sequenced using T7 and SP6 primers, and resulting sequence data were assembled into contigs using CAPs<sup><a reveal-id="DSR038C42" data-open="DSR038C42" class="link link-ref link-reveal xref-bibr">42</a></sup> as described previously. Contigs were annotated using the Blast2Go suite,<sup><a reveal-id="DSR038C43" data-open="DSR038C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSR038C44" data-open="DSR038C44" class="link link-ref link-reveal xref-bibr">44</a></sup> where the GenBank nr protein database was interrogated using the BLASTx algorithm.</p>                            <h2 scrollto-destination=7365635 id="7365635" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7365636 id="7365636" class="section-title" data-legacy-id=s3a>3.1. Annotation of the Lep1 Helitron</h3>
<p>A conserved 99-bp <em>Lep</em>1 box was previously described in eight lepidopteran species<sup><a reveal-id="DSR038C3" data-open="DSR038C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR038C6" data-open="DSR038C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and later described as portion of a 134-bp consensus LSCS3.<sup><a reveal-id="DSR038C3" data-open="DSR038C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Our homology-based searches of the GenBank nr/nt database resulted in the estimation of 618 regions within 210 nucleotide sequence accessions from Lepidoptera that show ≥70.0% similarity to <em>Lep</em>1 (32 species; mean similarity: 80.3 ± 5.8% over 120.5 ± 19.0 bp; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Eighteen of these <em>Lep</em>1-containing accessions were annotated as microsatellite loci and 51 <em>Lep</em>1s were within introns or untranslated regions of known genes. The remaining 526 <em>Lep</em>1s were within un-annotated sequences from BAC full inserts of the lepidopteran species <em>B. mori</em>, <em>B. anynana</em>, <em>S. frugiperda</em>, <em>H. armigera</em>, <em>H. melpomene</em>, <em>H. erato</em> and <em>P. dardanus</em>. An analogous search of the GenBank dbEST database identified 443 accessions from lepidopteran species that showed ≥71.1% interspecific similarity with 84 of these dbEST hits being ≥130 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Multiple sequence alignment of GenBank nr/nt and dbEST accessions that contained full-length <em>Lep</em>1 elements resulted in a 197-bp consensus that shared <em>Helitron</em>-like 5′-CT and 3′-CTRY termini,<sup><a reveal-id="DSR038C1" data-open="DSR038C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and were, respectively, designated as region H1 and region H2 of <em>Lep</em>1 (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A). The nucleotides directly adjacent to the 5′- and 3′-ends consisted of TA or AA in 96.2% of predicted <em>Lep</em>1s and showed not discernable target site duplications which are consistent with previously described <em>Helitron</em> genomic integration events.<sup><a reveal-id="DSR038C1" data-open="DSR038C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSR038C3" data-open="DSR038C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p>                        <a id="7365638" scrollto-destination="7365638"></a>
<div id="DSR038F1" data-id="DSR038F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03801.jpeg?Expires=1509238785&Signature=DCMa9YjwaGLr9D2P7LVec~R4LMIqnIZjqZXkLwBNaNYdtJsFiiSa6PQ~6n7Ht7SrULZTIgtELs-ODxAMkU4tVym5PCpwyiQ06guPOWzWjICcgjSDCAu5PSKe3ILxQ2dJfayoOHcZsEgOgy-MxFVAeVPGXxFjVxzYziU~Y-acXcSquEjYUdG3J1d3tuxkWyTudunOc3sIZ96jr~aBBhLUy6SK490Xzb3FzPGZrgK4wurClC2bCetl24iF1LKa50MVDXDS8sZlSzk4LZbT78PfJPnPWIs7~hltrxurOC16qJGmSpBVxTOVUtwz9KV96cOhV3f7Vk1ut977k8pVhUMYDA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment of Lep1 Helitron sequences from species of Lepidoptera. (A) Termini are defined by 5′-CT and 3′-CTRY motifs, and show secondary structures that are formed between a 5′-SIR that is complementary to a 3′-SIR (underline) and a 6–7-bp hairpin is present at the 3′-terminus (3′-stem-loop; double underlined). Additionally, Lep1s contain a hitchhiking (GTTT)n repeat microsatellite located between the SIRs. Flanking genomic sequence (small caps) that indicates integration occurs between TT or TA dinucleotides (enclosed in boxes). Corresponding secondary structures for Bombyx mori and Heliconius melpomene Lep1s are in Fig. 2. (B) Alignment of the same 5′-region of the Lep1 Helitron as in Fig. 1A, showing base secondary structure elements between an alternate 5′-SIR (5′-SIRa) and a 3′-SIR (3′-SIRa) within downstream acquired Helitron regions and a new 3′-stem-loop." path-from-xml="dsr03801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365638/dsr03801.jpeg" path-from-xml="dsr03801.jpeg" target="_blank">View large</a><a section="7365638" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03801.jpeg?Expires=1509238785&Signature=d5AWaSx~xk7qvxMKhPXxs--64Yx4e2UDzyFCvh3vaYOZFL46OM-MBGcChjrFsotljKf9y2-BZOmvgdd~taheSWjq52jkscBOFyVCRgNPTto~jWoji0tcwiYj0SVHHla7ZzlxrJskr4dHNOfc4ev7wB1c8oyHmR3dglJnzq4jmk~MIE4rsfkS~b5Rr4aeBFrLCXoXKPLuvwGZt5CvClsIM38iqTIWVNRwKvub~9lsg9TjJ93ZJek0Nruve1D5harmdQ~lcUj7nfFF1EZ7EPnJ46DnhF40jVJSRa8T-9WG6Hw4e5thaVSgOa3wEY-Jbkvy~FJIxRyguG9R~G82g7TYRw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365638&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of <em>Lep</em>1 <em>Helitron</em> sequences from species of Lepidoptera. (A) Termini are defined by 5′-CT and 3′-CTRY motifs, and show secondary structures that are formed between a 5′-SIR that is complementary to a 3′-SIR (<span class="underline">underline</span>) and a 6–7-bp hairpin is present at the 3′-terminus (3′-stem-loop; double underlined). Additionally, <em>Lep</em>1s contain a hitchhiking (GTTT)<sub>n</sub> repeat microsatellite located between the SIRs. Flanking genomic sequence (small caps) that indicates integration occurs between TT or TA dinucleotides (enclosed in boxes). Corresponding secondary structures for <em>Bombyx mori</em> and <em>Heliconius melpomene Lep</em>1s are in Fig. <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>. (B) Alignment of the same 5′-region of the <em>Lep</em>1 <em>Helitron</em> as in Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A, showing base secondary structure elements between an alternate 5′-SIR (5′-SIRa) and a 3′-SIR (3′-SIRa) within downstream acquired <em>Helitron</em> regions and a new 3′-stem-loop.</p></div></div><div content-id="DSR038F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03801.jpeg?Expires=1509238785&Signature=DCMa9YjwaGLr9D2P7LVec~R4LMIqnIZjqZXkLwBNaNYdtJsFiiSa6PQ~6n7Ht7SrULZTIgtELs-ODxAMkU4tVym5PCpwyiQ06guPOWzWjICcgjSDCAu5PSKe3ILxQ2dJfayoOHcZsEgOgy-MxFVAeVPGXxFjVxzYziU~Y-acXcSquEjYUdG3J1d3tuxkWyTudunOc3sIZ96jr~aBBhLUy6SK490Xzb3FzPGZrgK4wurClC2bCetl24iF1LKa50MVDXDS8sZlSzk4LZbT78PfJPnPWIs7~hltrxurOC16qJGmSpBVxTOVUtwz9KV96cOhV3f7Vk1ut977k8pVhUMYDA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment of Lep1 Helitron sequences from species of Lepidoptera. (A) Termini are defined by 5′-CT and 3′-CTRY motifs, and show secondary structures that are formed between a 5′-SIR that is complementary to a 3′-SIR (underline) and a 6–7-bp hairpin is present at the 3′-terminus (3′-stem-loop; double underlined). Additionally, Lep1s contain a hitchhiking (GTTT)n repeat microsatellite located between the SIRs. Flanking genomic sequence (small caps) that indicates integration occurs between TT or TA dinucleotides (enclosed in boxes). Corresponding secondary structures for Bombyx mori and Heliconius melpomene Lep1s are in Fig. 2. (B) Alignment of the same 5′-region of the Lep1 Helitron as in Fig. 1A, showing base secondary structure elements between an alternate 5′-SIR (5′-SIRa) and a 3′-SIR (3′-SIRa) within downstream acquired Helitron regions and a new 3′-stem-loop." path-from-xml="dsr03801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365638/dsr03801.jpeg" path-from-xml="dsr03801.jpeg" target="_blank">View large</a><a section="7365638" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03801.jpeg?Expires=1509238785&Signature=d5AWaSx~xk7qvxMKhPXxs--64Yx4e2UDzyFCvh3vaYOZFL46OM-MBGcChjrFsotljKf9y2-BZOmvgdd~taheSWjq52jkscBOFyVCRgNPTto~jWoji0tcwiYj0SVHHla7ZzlxrJskr4dHNOfc4ev7wB1c8oyHmR3dglJnzq4jmk~MIE4rsfkS~b5Rr4aeBFrLCXoXKPLuvwGZt5CvClsIM38iqTIWVNRwKvub~9lsg9TjJ93ZJek0Nruve1D5harmdQ~lcUj7nfFF1EZ7EPnJ46DnhF40jVJSRa8T-9WG6Hw4e5thaVSgOa3wEY-Jbkvy~FJIxRyguG9R~G82g7TYRw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365638&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of <em>Lep</em>1 <em>Helitron</em> sequences from species of Lepidoptera. (A) Termini are defined by 5′-CT and 3′-CTRY motifs, and show secondary structures that are formed between a 5′-SIR that is complementary to a 3′-SIR (<span class="underline">underline</span>) and a 6–7-bp hairpin is present at the 3′-terminus (3′-stem-loop; double underlined). Additionally, <em>Lep</em>1s contain a hitchhiking (GTTT)<sub>n</sub> repeat microsatellite located between the SIRs. Flanking genomic sequence (small caps) that indicates integration occurs between TT or TA dinucleotides (enclosed in boxes). Corresponding secondary structures for <em>Bombyx mori</em> and <em>Heliconius melpomene Lep</em>1s are in Fig. <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>. (B) Alignment of the same 5′-region of the <em>Lep</em>1 <em>Helitron</em> as in Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A, showing base secondary structure elements between an alternate 5′-SIR (5′-SIRa) and a 3′-SIR (3′-SIRa) within downstream acquired <em>Helitron</em> regions and a new 3′-stem-loop.</p></div></div><p>Structural homology among <em>Helitrons</em> is often used for prediction and characterization, and is based upon a conserved 3′-stem-loop (hairpin) formed upstream of the 3′-CTRR terminus.<sup><a reveal-id="DSR038C4" data-open="DSR038C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR038C5" data-open="DSR038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR038C8" data-open="DSR038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> We predicted that a 3′-stem-loop would form at the 3′ <em>Lep</em>1 terminus in <em>B. mori</em> accession D86623.1 [AATCTACATCATTCGCGAGTGACTTAGGCTA] (nucleotides involved in base pairing are double underlined) with a Gibbs free-energy change (Δ<em>G</em>) = −2. 82 kcal mol<sup>−1</sup> (Figs <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A and <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>A), as well as at the 3′-terminus other lepidopteran <em>Lep</em>1s (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A; not all data shown) including <em>H. melpomene</em> (Fig. <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>B; Δ<em>G</em> = –3.67 kcal mol<sup>−1</sup>). In addition to 5′-CT and 3′-CTRY termini, the formation of a 3′-stem-loop (hairpin) near the 3′-terminus are hallmarks of <em>Helitron</em> TEs and suggest important roles of these structures in RCR or genome integration.<sup><a reveal-id="DSR038C6" data-open="DSR038C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR038C10" data-open="DSR038C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> A second conserved stem-loop (hairpin) structure was predicted from <em>B. mori</em> and <em>H. melpomene Lep</em>1s that we designated the 5′-inverted repeat (5′-IR). This 5′-IR involves base pairing between portions of the 5′-subterminal inverted repeat (SIRa) and the microsatellite loop, and are analogous to the 5′-IR formed by the <em>Helitron</em>-like <em>Drosophila</em> interspersed nuclear element (<em>DINE</em>-1) family of TEs<sup><a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and the lepidopteran microsatellite associated interspersed nuclear element (<em>MINE</em>-1).<sup><a reveal-id="DSR038C35" data-open="DSR038C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Specifically, the <em>B. mori</em> GenBank accession D86623.1 is predicted to have a 7-bp 5′-IR formed by a portion of the 5′-SIRa sequence [<span class="underline">AC</span>T<span class="underline">AAT</span>A<span class="underline">TT</span>-7nt-GGA<span class="underline">AA</span>G<span class="underline">ATT</span>T<span class="underline">GT</span>TT] (nucleotides involved in intramolecular base pairing are underlined; Δ<em>G</em> = –0.83 kcal mol<sup>−1</sup>; Fig. <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>A), and the 6 bp 5′-IR in <em>H. melpomene Lep</em>1s are formed between later six nucleotides of the 5′-SIRa and the (GTTT)<sub>n</sub> microsatellite loop (Δ<em>G</em> = –0.99 kcal mol<sup>−1</sup>; Fig. <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>B). The conservation of hairpin structures adjacent to both the 5′- and 3′-termini among 460 of 618 (74.4%) of <em>Lep</em>1s predicted among nr database accessions suggests a role in the RepHel protein recognition, nascent strand cleavage, or other portion of the RCR mechanism,<sup><a reveal-id="DSR038C5" data-open="DSR038C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR038C10" data-open="DSR038C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR038C18" data-open="DSR038C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> but further investigation is required to elucidate their role. </p>                        <a id="7365640" scrollto-destination="7365640"></a>
<div id="DSR038F2" data-id="DSR038F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03802.jpeg?Expires=1509238785&Signature=c6pA9eZI53ZSglzb3SpUxXhBC68yTkzyW9sCOtZ11v-G9B3j0RZ1q~5qfCH~reFSTZa-xxRidzI5v4ZnYZsCEsujqINMjUuwbwi-5Wq4cBSNLARz6CbLeXR96HZzqLzt6eFcx3SM~Mv7atUzfJ-n-hzCAs5RzOfSdLsXMvcbKLqjxrcRxYiRS6R~5QeigmYzVmk1EE1pq1I23BObw4qi4r1bTXxXmwe2D-ldo3Yqe7qgmGYz~6eUCsC3BTshgdBe~OAvbRhazuOtzzot0iLP3Uarkg3~dfysW1vYyzYcrcfuoQFHoYEi65zvNW4i-tUZIgsj~c2K8nPQ5TZQiyyA7g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The secondary structure conserved among Lep1 elements. (A) Bombyx mori GenBank accession D86623.1 and (B) Heliconius melpomene GenBank accession CR974474.4 positions 69 560–69 971. Helitron-like features include 5′-CT and 3′-CTRY termini, a 3′-stem-loop is located two to three nucleotides upstream of the 3′-CTRY terminus (nucleotide with parallel grey bars), and a 5′-SIR that is complementary to a 3′-SIR (nucleotide with parallel black bars). Nucleotides involved in the formation of a 5′-IR are paralleled by yellow bars and the structure is shown within the boxed area for H. melpomene." path-from-xml="dsr03802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365640/dsr03802.jpeg" path-from-xml="dsr03802.jpeg" target="_blank">View large</a><a section="7365640" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03802.jpeg?Expires=1509238785&Signature=ND~v~Fnm66ULvbg4LvJ-xSEIIvE15qYqivlhQTC-W231hbJXj3JunE1vknz2tb7dNFHGovcgLJqBTr8e78QM~FiD58vBTksiAv4i1JjGjVzlopOUbKgb0MknYKbxEc1BgyjBJr3Sfass0k7SmyT3OxK9FfXa6tLUvBIV~kGufs74ParUDbBOG-f5DD6ehE90-YFzgtdOizosmNwS4TdyYwYE1cADw9y9E8P~4FgcY~b3V5iuE~cj5jOaYn2X2IoyXPSWytQEBe-Kuz~bCkDCNdWt5ObWNxyxU06GHYQENKxRnc3HOiV6s5ILNRAS1kbh3KCb17Dptja0OW7DzxO5Wg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365640&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The secondary structure conserved among <em>Lep</em>1 elements. (A) <em>Bombyx mori</em> GenBank accession D86623.1 and (B) <em>Heliconius melpomene</em> GenBank accession CR974474.4 positions 69 560–69 971. <em>Helitron</em>-like features include 5′-CT and 3′-CTRY termini, a 3′-stem-loop is located two to three nucleotides upstream of the 3′-CTRY terminus (nucleotide with parallel grey bars), and a 5′-SIR that is complementary to a 3′-SIR (nucleotide with parallel black bars). Nucleotides involved in the formation of a 5′-IR are paralleled by yellow bars and the structure is shown within the boxed area for <em>H. melpomene</em>.</p></div></div><div content-id="DSR038F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03802.jpeg?Expires=1509238785&Signature=c6pA9eZI53ZSglzb3SpUxXhBC68yTkzyW9sCOtZ11v-G9B3j0RZ1q~5qfCH~reFSTZa-xxRidzI5v4ZnYZsCEsujqINMjUuwbwi-5Wq4cBSNLARz6CbLeXR96HZzqLzt6eFcx3SM~Mv7atUzfJ-n-hzCAs5RzOfSdLsXMvcbKLqjxrcRxYiRS6R~5QeigmYzVmk1EE1pq1I23BObw4qi4r1bTXxXmwe2D-ldo3Yqe7qgmGYz~6eUCsC3BTshgdBe~OAvbRhazuOtzzot0iLP3Uarkg3~dfysW1vYyzYcrcfuoQFHoYEi65zvNW4i-tUZIgsj~c2K8nPQ5TZQiyyA7g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="The secondary structure conserved among Lep1 elements. (A) Bombyx mori GenBank accession D86623.1 and (B) Heliconius melpomene GenBank accession CR974474.4 positions 69 560–69 971. Helitron-like features include 5′-CT and 3′-CTRY termini, a 3′-stem-loop is located two to three nucleotides upstream of the 3′-CTRY terminus (nucleotide with parallel grey bars), and a 5′-SIR that is complementary to a 3′-SIR (nucleotide with parallel black bars). Nucleotides involved in the formation of a 5′-IR are paralleled by yellow bars and the structure is shown within the boxed area for H. melpomene." path-from-xml="dsr03802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365640/dsr03802.jpeg" path-from-xml="dsr03802.jpeg" target="_blank">View large</a><a section="7365640" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03802.jpeg?Expires=1509238785&Signature=ND~v~Fnm66ULvbg4LvJ-xSEIIvE15qYqivlhQTC-W231hbJXj3JunE1vknz2tb7dNFHGovcgLJqBTr8e78QM~FiD58vBTksiAv4i1JjGjVzlopOUbKgb0MknYKbxEc1BgyjBJr3Sfass0k7SmyT3OxK9FfXa6tLUvBIV~kGufs74ParUDbBOG-f5DD6ehE90-YFzgtdOizosmNwS4TdyYwYE1cADw9y9E8P~4FgcY~b3V5iuE~cj5jOaYn2X2IoyXPSWytQEBe-Kuz~bCkDCNdWt5ObWNxyxU06GHYQENKxRnc3HOiV6s5ILNRAS1kbh3KCb17Dptja0OW7DzxO5Wg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365640&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The secondary structure conserved among <em>Lep</em>1 elements. (A) <em>Bombyx mori</em> GenBank accession D86623.1 and (B) <em>Heliconius melpomene</em> GenBank accession CR974474.4 positions 69 560–69 971. <em>Helitron</em>-like features include 5′-CT and 3′-CTRY termini, a 3′-stem-loop is located two to three nucleotides upstream of the 3′-CTRY terminus (nucleotide with parallel grey bars), and a 5′-SIR that is complementary to a 3′-SIR (nucleotide with parallel black bars). Nucleotides involved in the formation of a 5′-IR are paralleled by yellow bars and the structure is shown within the boxed area for <em>H. melpomene</em>.</p></div></div><p>Structural predictions for <em>Lep</em>1 further indicated that intramolecular base pairing may occur between nucleotides immediately downstream of the 5′-CT and a region 26–37 nt upstream of the 3′-CTRY terminus and involve interaction of the SIRa. Specifically, nucleotides of the 5′-SIRa from <em>B. mori</em> accession D86623.1 [A<span class="underline">CTAATA</span>TT-7nt-ATA<span class="underline">AAGATTT</span>GTTT] are predicted to pair with the 3′-SIRa [A<span class="underline">AAA</span>T<span class="underline">TCTT</span>TTCC<span class="underline">ATTAG</span>A] located 49 bp downstream (nucleotides involved in base pairing are underlined in; Δ<em>G</em> = –5.92 kcal mol<sup>−1</sup>; Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A). Analogous structures were predicted from <em>H. melpomene</em> (Figs <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A and <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">2</a>B) as well as all other full-length <em>Lep</em>1s (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A; all data not shown). These molecular interactions of <em>Lep</em>1 contribute to an overall secondary structure that is stable at 25°C for <em>B. mori</em> (Δ<em>G</em> = –19.74 kcal mol<sup>−1</sup>) and <em>H. melpomene</em> (Δ<em>G</em> = –11.90 kcal mol<sup>−1</sup>; <a reveal-id="DSR038F2" data-open="DSR038F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), and is analogous to the structure formed by the <em>MINE</em>-1 <em>Helitron</em> from Lepidoptera.<sup><a reveal-id="DSR038C35" data-open="DSR038C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h3 scrollto-destination=7365642 id="7365642" class="section-title" data-legacy-id=s3b>3.2. Annotation of acquired end sequences</h3>
<p><em>Helitrons</em> modify the structure of genomes through the acquisition and transposition of novel DNA sequence that occurs by a largely unresolved mechanism.<sup><a reveal-id="DSR038C19" data-open="DSR038C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We described the consensus <em>Lep</em>1 element sequence from 31 accession for species of Lepidoptera that shared <em>Helitron-</em>like termini at the border of regions H1 and H2 (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A), but also identified 62–342-bp sequence regions downstream of the 3′-CTRY terminus at the boundary of region H2 that are shared within a species. The novel shared sequences downstream of the 3′-CTRY were referred to as region H3. Specifically, the region H3 showed ≤62.3% similarity between species and ≥95.7% similarity within a species or closely related species such as between the 55- and 59-bp region H3 from <em>Helicoverpa</em> species <em>H. armigera</em> (GenBank accession: FP340429.1) and <em>H. zea</em> (EU327673.1; Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>B). Among region H3 sequences from the same species or closely related species, sequence similarity terminated at a ubiquitous 3′-CTAG motif that was followed by a thymidine nucleotide (T; Figs <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>B and <a reveal-id="DSR038F3" data-open="DSR038F3" class="link link-reveal link-table xref-fig">3</a>), and only one variant of the H3 region was described within a species. The <em>Lep</em>1s predicted from the <em>B. mori</em> genome assembly shows two unique 87- or 335-bp sequences within region H3 that were, respectively, represented by GenBank accessions: DQ242656.1 and D86623.1 (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>B). The <em>B. mori Lep</em>1 <em>Helitrons</em> with an 87- or 335-bp region H3 were subsequently called Bm<em>Lep</em>1_87 and Bm<em>Lep</em>1_335 variants, respectively, and alignments showed no discernable homology (Fig. <a reveal-id="DSR038F3" data-open="DSR038F3" class="link link-reveal link-table xref-fig">3</a>). Phylogenetic reconstruction of <em>B. mori Lep</em>1_87 and _335-bp <em>Helitron</em> variants from chromosome 1 suggested that two weakly supported clades may exist, which indicated that <em>Lep</em>1s have evolved independently within the <em>B. mori</em> genome through the acquisition of two different downstream sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>; gamma parameter rate distribution <em>γ</em> = 3.9894; Log likelihood = <em>D</em> = –3729.34)<em>.</em> Independent gain of sequence mutations has previously been identified for <em>Helitrons</em> in the maize genome<sup><a reveal-id="DSR038C6" data-open="DSR038C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR038C10" data-open="DSR038C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and indicate that arthropod genomes are also modified by <em>Helitron</em> movements. </p>                        <a id="7365644" scrollto-destination="7365644"></a>
<div id="DSR038F3" data-id="DSR038F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03803.jpeg?Expires=1509238785&Signature=gd56yKr-vMGXOsbzgSgQKXyp-NVIzGh8oqebgbX6V4nD1G4E6xSKbK-oY5TzPXJrnrUOrmkmA7gOIPOIv1hummYmgFNqagIgnU-3t~nb1YXZSJMuIn1AkGdAIK~qa32zrnA-C6Me~nGufqAgSLJyq9quPE7fRz62XyIEFj5~~HiB0AyotLJ0eswymtu~R5I0Sy9GDdOnRecMqgvdeEZj~g2fkiSPCepTPKbBY2SvyrzOBu8KatWGAJ7IPLWzvXNFhAeXffaoIjscWabS7hkF4mazzTyAOAT1PGHG5TsEFjN4hSdrxzyyRlO72Gpx-BH70GIeXra7c-K5ZdqPtH3Jeg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment of nucleotides for secondary structure of Bombyx mori Lep1 Helitrons. The 5′-SIR form alternate hairpin structures with either a 5′-invered repeat (5′-IR; yellow arrows) or a 3′-SIR within the Lep1 consensus Helitron (black arrows), as well as a 3′-SIR within the acquired Helitron end sequence (red arrows) within both the 87- and 355-bp acquired regions. Two hairpins are formed, one at the 3′-terminus of the Lep1 Helitron and one within each of the 87- or 335-bp acquired Helitron end sequence (grey arrows). The 3′-termini of the ancestral Lep1 Helitron and the acquired end sequences are, respectively, composed of CTAY and CTAG motifs (yellow highlighted nucleotides)." path-from-xml="dsr03803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365644/dsr03803.jpeg" path-from-xml="dsr03803.jpeg" target="_blank">View large</a><a section="7365644" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03803.jpeg?Expires=1509238785&Signature=HqFMrTiXsyUkFAuw0luQ1ihl~hJdGW7ZYXhL4lo~a67uLdp3YTuJh74RZuMWq27SrBwBR-KbVlWWHdxwmEnoan5cwY2UPF7l44NtwNZpBlJA0nsSw-CzyeNXlBysTbYz6ugGH6ZRsU3q6CgCPA-4GCMXd8KbqH1FSNsl-Rx5ZkRAyR667PdufuNw-dmXGgmYmmUy1O7D38Ns5i3UTGpL-2Btk0tZ1iFb5DEijMWEC9OcdDmJVRdcd8zl2qopkY65rVhxHYS2Hx4mnSNaExmpWdZ~1jh8oMwct9LtdXZCdZeE3vym5lmyIXwSBwvc2OYuQPx76sHThKNRV8OD3gqFwQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365644&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of nucleotides for secondary structure of <em>Bombyx mori Lep</em>1 <em>Helitrons</em>. The 5′-SIR form alternate hairpin structures with either a 5′-invered repeat (5′-IR; yellow arrows) or a 3′-SIR within the <em>Lep</em>1 consensus <em>Helitron</em> (black arrows), as well as a 3′-SIR within the acquired <em>Helitron</em> end sequence (red arrows) within both the 87- and 355-bp acquired regions. Two hairpins are formed, one at the 3′-terminus of the <em>Lep</em>1 <em>Helitron</em> and one within each of the 87- or 335-bp acquired <em>Helitron</em> end sequence (grey arrows). The 3′-termini of the ancestral <em>Lep</em>1 <em>Helitron</em> and the acquired end sequences are, respectively, composed of CTAY and CTAG motifs (yellow highlighted nucleotides).</p></div></div><div content-id="DSR038F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03803.jpeg?Expires=1509238785&Signature=gd56yKr-vMGXOsbzgSgQKXyp-NVIzGh8oqebgbX6V4nD1G4E6xSKbK-oY5TzPXJrnrUOrmkmA7gOIPOIv1hummYmgFNqagIgnU-3t~nb1YXZSJMuIn1AkGdAIK~qa32zrnA-C6Me~nGufqAgSLJyq9quPE7fRz62XyIEFj5~~HiB0AyotLJ0eswymtu~R5I0Sy9GDdOnRecMqgvdeEZj~g2fkiSPCepTPKbBY2SvyrzOBu8KatWGAJ7IPLWzvXNFhAeXffaoIjscWabS7hkF4mazzTyAOAT1PGHG5TsEFjN4hSdrxzyyRlO72Gpx-BH70GIeXra7c-K5ZdqPtH3Jeg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Alignment of nucleotides for secondary structure of Bombyx mori Lep1 Helitrons. The 5′-SIR form alternate hairpin structures with either a 5′-invered repeat (5′-IR; yellow arrows) or a 3′-SIR within the Lep1 consensus Helitron (black arrows), as well as a 3′-SIR within the acquired Helitron end sequence (red arrows) within both the 87- and 355-bp acquired regions. Two hairpins are formed, one at the 3′-terminus of the Lep1 Helitron and one within each of the 87- or 335-bp acquired Helitron end sequence (grey arrows). The 3′-termini of the ancestral Lep1 Helitron and the acquired end sequences are, respectively, composed of CTAY and CTAG motifs (yellow highlighted nucleotides)." path-from-xml="dsr03803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365644/dsr03803.jpeg" path-from-xml="dsr03803.jpeg" target="_blank">View large</a><a section="7365644" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03803.jpeg?Expires=1509238785&Signature=HqFMrTiXsyUkFAuw0luQ1ihl~hJdGW7ZYXhL4lo~a67uLdp3YTuJh74RZuMWq27SrBwBR-KbVlWWHdxwmEnoan5cwY2UPF7l44NtwNZpBlJA0nsSw-CzyeNXlBysTbYz6ugGH6ZRsU3q6CgCPA-4GCMXd8KbqH1FSNsl-Rx5ZkRAyR667PdufuNw-dmXGgmYmmUy1O7D38Ns5i3UTGpL-2Btk0tZ1iFb5DEijMWEC9OcdDmJVRdcd8zl2qopkY65rVhxHYS2Hx4mnSNaExmpWdZ~1jh8oMwct9LtdXZCdZeE3vym5lmyIXwSBwvc2OYuQPx76sHThKNRV8OD3gqFwQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365644&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of nucleotides for secondary structure of <em>Bombyx mori Lep</em>1 <em>Helitrons</em>. The 5′-SIR form alternate hairpin structures with either a 5′-invered repeat (5′-IR; yellow arrows) or a 3′-SIR within the <em>Lep</em>1 consensus <em>Helitron</em> (black arrows), as well as a 3′-SIR within the acquired <em>Helitron</em> end sequence (red arrows) within both the 87- and 355-bp acquired regions. Two hairpins are formed, one at the 3′-terminus of the <em>Lep</em>1 <em>Helitron</em> and one within each of the 87- or 335-bp acquired <em>Helitron</em> end sequence (grey arrows). The 3′-termini of the ancestral <em>Lep</em>1 <em>Helitron</em> and the acquired end sequences are, respectively, composed of CTAY and CTAG motifs (yellow highlighted nucleotides).</p></div></div><p>Although the sequences in region H3 share little interspecific sequence similarity, the secondary structures predicted to form appear to be identical by state with those formed in region H2. Specifically, the 7–9-bp 3′-stem-loops (hairpins), respectively, formed in region H3 of Bm<em>Lep</em>1_87 and Bm<em>Lep</em>1_335 <em>Helitrons</em> (Δ<em>G</em> ≤ –7.90 kcal mol<sup>−1</sup>) are more highly stable compared with the analogous structure in the ancestral region H2 (Δ<em>G</em> ≤ –2.99 kcal mol<sup>−1</sup>; Fig. <a reveal-id="DSR038F3" data-open="DSR038F3" class="link link-reveal link-table xref-fig">3</a>). This evidence may suggest that <em>Helitrons</em> are dependent upon a 3′-stem-loop directly upstream of the 3′-CTAG terminus for RCR function,<sup><a reveal-id="DSR038C5" data-open="DSR038C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and that acquired sequences undergo selection for the capacity to support propagation by RCR.<sup><a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> A functional switch from use of ancestral to derived <em>Lep</em>1 <em>Helitron</em> ends may have been influenced by the comparatively higher stability we predicted for 3′-stem-loops within the <em>Lep</em>1 acquired sequence. Thereby, functional shifts could occur when equally or more efficient terminal structure are encountered by change within flanking DNA or swapped between other <em>Helitrons</em>. In contrast, reduction in 3′-stem-loop stability in region H2 also could have resulted from degradation following the relaxation of selective constraints, and would mirror the degradation of a <em>Helitron</em>-like 3′-CTAG terminus that followed accretion by maize <em>Helitrons</em>.<sup><a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p><p>Additionally, intramolecular base pairings we predicted between the 5′-SIRa (region H1) and 3′-SIRa (region H2; Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>A) are analogously formed through interaction of nucleotides within the 5′-SIRa of the ancestral <em>Lep</em>1 region H1 and a 3′-SIRb within the derived region H3 (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>B). These interactions between the 5′-SIRb and 3′-SIRb in <em>B. mori Lep</em>1_87 and _335-bp <em>Helitrons</em> (Δ<em>G</em> ≤ −2.56 kcal mol<sup>−1</sup>) is lower than between the 5′-SIRa and 3′-SIRa (Δ<em>G</em> ≤ −5.92 kcal mol<sup>−1</sup>; Fig. <a reveal-id="DSR038F3" data-open="DSR038F3" class="link link-reveal link-table xref-fig">3</a>), but remain consistent with the characteristic secondary structures described previously for insect <em>Helitrons</em>.<sup><a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR038C45" data-open="DSR038C45" class="link link-ref link-reveal xref-bibr">45</a></sup> The formation of a hairpin between the 5′-SIR and 3′-SIRs within both the ancestral and acquired regions indicates that, in addition to the 3′-stem-loop structure, the base pairing between SIRs at proximal and distal ends of <em>Lep</em>1 may potentially be required from RCR. Furthermore, the requirement for a 3′-SIR within the independently acquired DNA sequences in proximity to a 3′-CTAG novel terminal motif could suggest that gain of sequence mutations by <em>Lep</em>1 may be rare. This hypothesis could be supported by our description of two <em>Helitron</em> variations in the <em>B. mori</em> genome, which contrasts with the high number of cryptic <em>Helitrons</em> from the maize genome where functional constraints appear to be relaxed.<sup><a reveal-id="DSR038C17" data-open="DSR038C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>The described accretion by lepidopteran <em>Helitrons</em> involves sequences that are relatively small and added in a unidirectional fashion. DNA sequence upstream of the single 5′-CT terminal motif showed no homology among integrations in lepidopteran genomes or conserved secondary structures that would be indicative of a chimeric <em>Helitron.</em><sup><a reveal-id="DSR038C46" data-open="DSR038C46" class="link link-ref link-reveal xref-bibr">46</a></sup> These observations are analogous to those described by Yang and Bennetzen.<sup><a reveal-id="DSR038C8" data-open="DSR038C8" class="link link-ref link-reveal xref-bibr">8</a></sup><em>Lep</em>1 was not shown to capture entire genes, but may be due to inability to accurately describe haplotype variation from available data resources or the possible culling of the mutations from genomes due to negative affects on genome function.<sup><a reveal-id="DSR038C47" data-open="DSR038C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Smaller non-autonomous <em>Helitrons</em> are better able to evade host repression<sup><a reveal-id="DSR038C6" data-open="DSR038C6" class="link link-ref link-reveal xref-bibr">6</a></sup> or show greater replication efficiency within the RCR mechanism.<sup><a reveal-id="DSR038C48" data-open="DSR038C48" class="link link-ref link-reveal xref-bibr">48</a></sup><em>Lep</em>1s also appears to gain sequence only at the 3′ end, which contrasts with bidirectional end creation by maize <em>Helitrons</em>.<sup><a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> This directionality of <em>Lep</em>1s may result from the preferential capture that is known to occur in the same orientation as the RepHel-coding sequence of the autonomous <em>Helitron</em>, but this cannot be ascertained until further research is preformed to identify the parent TE of the non-autonomous <em>Lep</em>1 element.</p>                            <h3 scrollto-destination=7365648 id="7365648" class="section-title" data-legacy-id=s3c>3.3. Lep1 copy number and genome distribution</h3>
<p>The variation in non-autonomous <em>Helitrons</em> copy number among species of Lepidoptera may result from differential effects of replicative repression via DNA methylation,<sup><a reveal-id="DSR038C49" data-open="DSR038C49" class="link link-ref link-reveal xref-bibr">49</a></sup> mutation-selection balance within the overall genome architecture,<sup><a reveal-id="DSR038C50" data-open="DSR038C50" class="link link-ref link-reveal xref-bibr">50</a></sup> or random genetic drift at the population scale. We showed that <em>Lep</em>1 <em>Helitron</em> integration densities range from 1.04 × 10<sup>−5</sup> to 1.8 × 10<sup>−4</sup> based upon annotation of full BAC insert sequences. These estimates indicated that <em>Lep</em>1s have an ∼13-fold copy number difference across species' genomes (Table <a reveal-id="DSR038TB1" data-open="DSR038TB1" class="link link-reveal link-table xref-fig">1</a>), and is comparable to the ∼11-fold variance observed among <em>DINE</em>-1 insertions within <em>Drosophila</em> genomes.<sup><a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The highest <em>Lep</em>1 copy number density was estimated for the butterfly <em>P. dardanus</em>, wherein <em>Lep</em>1 abundance was significantly higher than for all other species except for <em>B. anynana</em> (<em>P</em>-values ≥0.0089; Student's-<em>t</em> values not shown). The mean <em>Lep</em>1 integration density among butterflies (6.9 × 10<sup>−5</sup> ± 6.8 × 10<sup>−5</sup>) is not significantly different from the densities estimated among moth species (mean: 2.5 × 10<sup>−5</sup> ± 2.0 × 10<sup>−5</sup>; <em>F</em>-statistic = 1.4; d.f.<sub>num</sub> = 1; d.f.<sub>den</sub> = 2; <em>P</em>-value = 0.3583). </p>                        <a id="7365650" scrollto-destination="7365650"></a>
<div class="table-wrap"><div id="DSR038TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Estimated frequency of <em>Lep</em>1 <em>Helitrons</em> per mega base (Mb) in species of Lepidoptera</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03806.gif?Expires=1509238785&Signature=GlxbKfQAFQ5Avpc54Zt7E7xVV6OS1~VcwL3P7I0IZ08WdErSwsPYWcDJJ19MtET8tOolVmM5zPt9svf6l~PMKMJ0UN2ZpvlmQ7-QnY71vug~eI7KdP6zAY~cBtgQ4lLBq8RSeeWwiNpt2rIc-OEw6EHPqFUpwuKVj2gkhPeg4s63bX6JuC88QV8zbhayP~vL3HUTw0I-2L2jE43hzgogwzcsMS4Z~o2~IYTHlpaQsvzESr~vONoKtv2GSxJNn3zMHJOKE8u0dBhyhQU97XJkXVNU-Ck3OaTN8H5DCj0n4TXPNlObKUg1aYkgGGUk5RZwBsULyex9x~YlFNe983QSPw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="graphic" path-from-xml="dsr03806.gif" /> </td></tr></tbody></table></div><div content-id="DSR038TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03806.gif?Expires=1509238785&Signature=GlxbKfQAFQ5Avpc54Zt7E7xVV6OS1~VcwL3P7I0IZ08WdErSwsPYWcDJJ19MtET8tOolVmM5zPt9svf6l~PMKMJ0UN2ZpvlmQ7-QnY71vug~eI7KdP6zAY~cBtgQ4lLBq8RSeeWwiNpt2rIc-OEw6EHPqFUpwuKVj2gkhPeg4s63bX6JuC88QV8zbhayP~vL3HUTw0I-2L2jE43hzgogwzcsMS4Z~o2~IYTHlpaQsvzESr~vONoKtv2GSxJNn3zMHJOKE8u0dBhyhQU97XJkXVNU-Ck3OaTN8H5DCj0n4TXPNlObKUg1aYkgGGUk5RZwBsULyex9x~YlFNe983QSPw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="graphic" path-from-xml="dsr03806.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Estimates from BAC full-insert sequences. Respective GenBank accessions and positional information is provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7365650" target="_blank">View Large</a></div></div><p>The only whole genome sequence available for use to directly estimate the TE frequencies in Lepidoptera is the 432 Mb <em>B. mori</em> assembly (build v.2.3),<sup><a reveal-id="DSR038C30" data-open="DSR038C30" class="link link-ref link-reveal xref-bibr">30</a></sup> from which we estimated 5541 putative Bm<em>Lep</em>1 integrations. Each putative Bm<em>Lep</em>1 <em>Helitron</em> was assigned a unique identifier (Bm<em>Lep</em>1_000001 to Bm<em>Lep</em>1_005541), and further categorized as containing either the Bm<em>Lep</em>1_87 or Bm<em>Lep</em>1_335 variant downstream sequence described previously (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The ancestral regions of the <em>Lep</em>1 <em>Helitron</em> (regions H1 thru H2;) within build v. 2.3 showed ≥94.3 ± 1.9% similarity with the <em>B. mori Lep</em>1 elements from accessions DQ242656.1 and D86623.1 (Fig. <a reveal-id="DSR038F1" data-open="DSR038F1" class="link link-reveal link-table xref-fig">1</a>B). This paucity of <em>Lep</em>1 sequence evolution within the <em>B. mori</em> genome may indicate a recent burst in transposition,<sup><a reveal-id="DSR038C7" data-open="DSR038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR038C8" data-open="DSR038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> or a high degree of functional conservation. The density of <em>B. mori Lep</em>1s estimated from the genome sequence (1.3 × 10<sup>−5</sup>) is ∼2-fold lower than that estimated from BAC full inserts (2.5 × 10<sup>−5</sup>; Table <a reveal-id="DSR038TB1" data-open="DSR038TB1" class="link link-reveal link-table xref-fig">1</a>), and highlights the error that may be associated with subsampling from BAC sequences. Mapping the positions of Bm<em>Lep</em>1 onto chromosome assemblies indicated that integrations are co-localized with protein-coding genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>; chromosome 1 shown in Fig. <a reveal-id="DSR038F4" data-open="DSR038F4" class="link link-reveal link-table xref-fig">4</a>), which agrees with seminal evidence of <em>Lep</em>1 being within gene intervals.<sup><a reveal-id="DSR038C36" data-open="DSR038C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Furthermore, <em>Lep</em>1 proximity to <em>B. mori</em> gene-coding regions suggests that <em>Lep</em>1s may affect gene structure and function on a genome-wide scale.<sup><a reveal-id="DSR038C9" data-open="DSR038C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR038C20" data-open="DSR038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The effects of <em>Lep</em>1 <em>Helitron</em> integrations upon gene structure and function are for the first time presented in Section 3.4. </p>                        <a id="7365652" scrollto-destination="7365652"></a>
<div id="DSR038F4" data-id="DSR038F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03804.jpeg?Expires=1509238785&Signature=FkDse~2VmzRJSmG5dLkp1VG8xtgsGxCGt55jWxutgKaJkdx4RHowkwMKGaCBrs0GcReJow5YExqaUMqc02nbSlxyIIQM8Txc0jlZpWkoMOZ0CIFuogoxuWufa2NbAerCIF32WLPww7r8pm9OhnIFsmIFWOyR2IoP8XBOyNZJ40jfUzp5t14O8brXgdHqcDh~7LB0uQlj~0PZJLFNyIVy2Qu~s-jGcdcTAXsNXEbmnOyyzYIs7l0YmtwpyvpWFlkWP1eZAeNmike6pBHZyq2IA-zBJnVhmkCXfcVa6RtV01OJjDg32X~rAjMgNyucnNEBKxul5p58x3-BCgwB~9~Gew__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Lep1Helitron integration positions on the Bombyx mori chromosome 1. Lep1 model v. 2.3 showing positions of the whole genome assembly via a CMap output. Integrations are classified into Lep1s that have acquired novel 87- or 335-bp end sequences (positional data for all 28 linkage groups present in Supplementary Table S3)." path-from-xml="dsr03804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365652/dsr03804.jpeg" path-from-xml="dsr03804.jpeg" target="_blank">View large</a><a section="7365652" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03804.jpeg?Expires=1509238785&Signature=DKEe35A4byf1HdCDjQ1LRFmv6JljQjgRIlj-Bq1XPDBo7OhNhcnXEq6B~1rTWbF99Pe6XjY9kFpdo6N5dnIQtHg8FpkCE1UFFGbdQW63w1ZAk-Cmd7oAJmz0B0YDExvy6wwVHg0aJhrEpkgbRBnoPTQOVcmx246c3WxsYZQyZCwYR-~qajvSJ5b77HwVWjt1LKzsD4HmYa7rXM-V02peu0UaCkL66zVt2NrMSxdXYUwT4OdCfVruuBaJWqIz~5-v2W0XEnNqoyrlJyZHLIEtqZ-NA4ibLuh9AO5Nf41Nzk4MDOryOXxgzHLt89aEIkkFMDEeqh-IKNRBTZjWxiuB8g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365652&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lep<em>1</em><em>Helitron</em> integration positions on the <em>Bombyx mori</em> chromosome 1. <em>Lep</em>1 model v. 2.3 showing positions of the whole genome assembly via a CMap output. Integrations are classified into <em>Lep</em>1s that have acquired novel 87- or 335-bp end sequences (positional data for all 28 linkage groups present in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p></div></div><div content-id="DSR038F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03804.jpeg?Expires=1509238785&Signature=FkDse~2VmzRJSmG5dLkp1VG8xtgsGxCGt55jWxutgKaJkdx4RHowkwMKGaCBrs0GcReJow5YExqaUMqc02nbSlxyIIQM8Txc0jlZpWkoMOZ0CIFuogoxuWufa2NbAerCIF32WLPww7r8pm9OhnIFsmIFWOyR2IoP8XBOyNZJ40jfUzp5t14O8brXgdHqcDh~7LB0uQlj~0PZJLFNyIVy2Qu~s-jGcdcTAXsNXEbmnOyyzYIs7l0YmtwpyvpWFlkWP1eZAeNmike6pBHZyq2IA-zBJnVhmkCXfcVa6RtV01OJjDg32X~rAjMgNyucnNEBKxul5p58x3-BCgwB~9~Gew__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Lep1Helitron integration positions on the Bombyx mori chromosome 1. Lep1 model v. 2.3 showing positions of the whole genome assembly via a CMap output. Integrations are classified into Lep1s that have acquired novel 87- or 335-bp end sequences (positional data for all 28 linkage groups present in Supplementary Table S3)." path-from-xml="dsr03804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365652/dsr03804.jpeg" path-from-xml="dsr03804.jpeg" target="_blank">View large</a><a section="7365652" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03804.jpeg?Expires=1509238785&Signature=DKEe35A4byf1HdCDjQ1LRFmv6JljQjgRIlj-Bq1XPDBo7OhNhcnXEq6B~1rTWbF99Pe6XjY9kFpdo6N5dnIQtHg8FpkCE1UFFGbdQW63w1ZAk-Cmd7oAJmz0B0YDExvy6wwVHg0aJhrEpkgbRBnoPTQOVcmx246c3WxsYZQyZCwYR-~qajvSJ5b77HwVWjt1LKzsD4HmYa7rXM-V02peu0UaCkL66zVt2NrMSxdXYUwT4OdCfVruuBaJWqIz~5-v2W0XEnNqoyrlJyZHLIEtqZ-NA4ibLuh9AO5Nf41Nzk4MDOryOXxgzHLt89aEIkkFMDEeqh-IKNRBTZjWxiuB8g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365652&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lep<em>1</em><em>Helitron</em> integration positions on the <em>Bombyx mori</em> chromosome 1. <em>Lep</em>1 model v. 2.3 showing positions of the whole genome assembly via a CMap output. Integrations are classified into <em>Lep</em>1s that have acquired novel 87- or 335-bp end sequences (positional data for all 28 linkage groups present in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p></div></div>                            <h3 scrollto-destination=7365653 id="7365653" class="section-title" data-legacy-id=s3d>3.4. Lep1 elements modify gene structure</h3>
<p>The movement and propagation of TEs introduce haplotype variation,<sup><a reveal-id="DSR038C51" data-open="DSR038C51" class="link link-ref link-reveal xref-bibr">51</a></sup> and alter gene functions when integrated within coding regions of a genome.<sup><a reveal-id="DSR038C28" data-open="DSR038C28" class="link link-ref link-reveal xref-bibr">28</a></sup> TEs are present within insect ESTs<sup><a reveal-id="DSR038C52" data-open="DSR038C52" class="link link-ref link-reveal xref-bibr">52</a></sup> and mature transcripts of <em>D. melanogaster</em>,<sup><a reveal-id="DSR038C53" data-open="DSR038C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and can cause alternative the modification of <em>cis</em>-regulatory function,<sup><a reveal-id="DSR038C54" data-open="DSR038C54" class="link link-ref link-reveal xref-bibr">54</a></sup> introduction of frameshift and premature stop codon mutations,<sup><a reveal-id="DSR038C28" data-open="DSR038C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and be involved in exon shuffling<sup><a reveal-id="DSR038C18" data-open="DSR038C18" class="link link-ref link-reveal xref-bibr">18</a></sup> or insertion of introns.<sup><a reveal-id="DSR038C55" data-open="DSR038C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Evidence that <em>Lep</em>1s co-localize with protein-coding genes in <em>B. mori</em> genome (section 3.3) suggests that integrations may affect the structure and function of lepidopteran genes, and that presence/absence mutation at orthologous loci may be a source of function genetic variation among species. Comparison among orthologs of the cadherin gene from <em>B. mori</em>, <em>H. armigera</em>, and <em>O. nubilalis</em> genomes showed that <em>Helitron</em> copy number variation is present. Specifically, a nucleotide sequence with 68.7% similarity to the <em>Lep</em>1 consensus was predicted within the <em>H. armigera</em> cadherin intron 1, whereas orthologous introns from <em>B. mori</em> or <em>O. nubilalis</em> show no <em>Lep</em>1-like sequence (Fig. <a reveal-id="DSR038F5" data-open="DSR038F5" class="link link-reveal link-table xref-fig">5</a>A). Although the effects of this integration upon gene function was not investigated, TE integrations within introns are known to affect splicing efficiencies<sup><a reveal-id="DSR038C56" data-open="DSR038C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and indicated that <em>Lep</em>1s are a source of genome copy number variation between lepidopteran species. </p>                        <a id="7365655" scrollto-destination="7365655"></a>
<div id="DSR038F5" data-id="DSR038F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03805.jpeg?Expires=1509238785&Signature=g5TpB2rhybewD8ykT2EvgT8qqhpKNXyMYHPApAtsM-Csn2UEFbRUFo4MtjzlWN4-uWru0LXxxa99AAid8YeI5ii-TNLcfE2RrHIBkQej4rouWvAlESkW~w1UaZlHrUtv~7Kh-AUFjBHMLWjORNKLun1mEC4N8XZAintvnXbH~luIO~87wiMh~IOXic2nQxTbRaUdMtpCYs2p13nd~4DUVZiwQ9-yoJh9vA~1Wga0zjT3PlL7~cjigVMh6S7R-fVf1QRmzw2HsI~a6z5GXvMcL0p6XTbJ60lUbNrc2OdakJ8nvJ-0NDwHsEVA4X-~mrpMsaLJYxaJHpNa-pYP~av~2A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Lep1Helitron copy number variation. (A) Integration variation among Bombyx mori BGIBMGA013616, O. nubilalis DQ000165.1, and Helicoverpa armigera AY714876.1 DNA sequence accessions for intron 1 at the cadherin locus. The annotated H. armigera Lep1 (HaLep1) is integrated into the minus strand (positions 2216–2085) and shows 68.7% nucleotide sequence similarity to the Lep1 consensus and retains 5′-CT and 3′-CTAY termini. (B) Integration of a Lep1 element within the O. nubilalis allatotropin neuropeptide precursor gene that is compared with the Spodoptera frugiperda homolog (AJ566903.1) (integration is underscored by &lt;). Gaps in the alignment and missing data are represented by - and *, respectively." path-from-xml="dsr03805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365655/dsr03805.jpeg" path-from-xml="dsr03805.jpeg" target="_blank">View large</a><a section="7365655" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03805.jpeg?Expires=1509238785&Signature=ILQqRY1iB9110ppj2U5vgZiCtvP7en1gc5txAeJPta-RRqRHNP8dfFofN59~VdQSZd0FYmhWwR5yVRi88C3kjBsdOUHaTA3GfrmXk~e7CN1Ubl8qew4lUZ6B4GW5zGFDMu-t~2-Dt3CgQDSLVOOA8gnrRCPK9me4MvzobfvpRTqtQbsx6JKDNmSlq-RiWrSK3dgxqii2h8PMUoiBLVPkT6NJo564BTFQKC8yWL-Jej1bzCZCvp90faTcJFv2lyHklLFeo~92F4dD3XH0yk9KWcI3zZW6oPBZ2AhxA5Hy33GihIf5~IT-c0Vz60jCIqRcju6GaYX6-cLBs1zZ6FdKWA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365655&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lep<em>1</em><em>Helitron</em> copy number variation. (A) Integration variation among <em>Bombyx mori</em> BGIBMGA013616, <em>O. nubilalis</em> DQ000165.1, and <em>Helicoverpa armigera</em> AY714876.1 DNA sequence accessions for intron 1 at the cadherin locus. The annotated <em>H. armigera Lep</em>1 (Ha<em>Lep</em>1) is integrated into the minus strand (positions 2216–2085) and shows 68.7% nucleotide sequence similarity to the <em>Lep</em>1 consensus and retains 5′-CT and 3′-CTAY termini. (B) Integration of a <em>Lep</em>1 element within the <em>O. nubilalis</em> allatotropin neuropeptide precursor gene that is compared with the <em>Spodoptera frugiperda</em> homolog (AJ566903.1) (integration is underscored by &lt;). Gaps in the alignment and missing data are represented by - and *, respectively.</p></div></div><div content-id="DSR038F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/m_dsr03805.jpeg?Expires=1509238785&Signature=g5TpB2rhybewD8ykT2EvgT8qqhpKNXyMYHPApAtsM-Csn2UEFbRUFo4MtjzlWN4-uWru0LXxxa99AAid8YeI5ii-TNLcfE2RrHIBkQej4rouWvAlESkW~w1UaZlHrUtv~7Kh-AUFjBHMLWjORNKLun1mEC4N8XZAintvnXbH~luIO~87wiMh~IOXic2nQxTbRaUdMtpCYs2p13nd~4DUVZiwQ9-yoJh9vA~1Wga0zjT3PlL7~cjigVMh6S7R-fVf1QRmzw2HsI~a6z5GXvMcL0p6XTbJ60lUbNrc2OdakJ8nvJ-0NDwHsEVA4X-~mrpMsaLJYxaJHpNa-pYP~av~2A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Lep1Helitron copy number variation. (A) Integration variation among Bombyx mori BGIBMGA013616, O. nubilalis DQ000165.1, and Helicoverpa armigera AY714876.1 DNA sequence accessions for intron 1 at the cadherin locus. The annotated H. armigera Lep1 (HaLep1) is integrated into the minus strand (positions 2216–2085) and shows 68.7% nucleotide sequence similarity to the Lep1 consensus and retains 5′-CT and 3′-CTAY termini. (B) Integration of a Lep1 element within the O. nubilalis allatotropin neuropeptide precursor gene that is compared with the Spodoptera frugiperda homolog (AJ566903.1) (integration is underscored by &lt;). Gaps in the alignment and missing data are represented by - and *, respectively." path-from-xml="dsr03805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7365655/dsr03805.jpeg" path-from-xml="dsr03805.jpeg" target="_blank">View large</a><a section="7365655" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr03805.jpeg?Expires=1509238785&Signature=ILQqRY1iB9110ppj2U5vgZiCtvP7en1gc5txAeJPta-RRqRHNP8dfFofN59~VdQSZd0FYmhWwR5yVRi88C3kjBsdOUHaTA3GfrmXk~e7CN1Ubl8qew4lUZ6B4GW5zGFDMu-t~2-Dt3CgQDSLVOOA8gnrRCPK9me4MvzobfvpRTqtQbsx6JKDNmSlq-RiWrSK3dgxqii2h8PMUoiBLVPkT6NJo564BTFQKC8yWL-Jej1bzCZCvp90faTcJFv2lyHklLFeo~92F4dD3XH0yk9KWcI3zZW6oPBZ2AhxA5Hy33GihIf5~IT-c0Vz60jCIqRcju6GaYX6-cLBs1zZ6FdKWA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7365655&ar=484721&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lep<em>1</em><em>Helitron</em> copy number variation. (A) Integration variation among <em>Bombyx mori</em> BGIBMGA013616, <em>O. nubilalis</em> DQ000165.1, and <em>Helicoverpa armigera</em> AY714876.1 DNA sequence accessions for intron 1 at the cadherin locus. The annotated <em>H. armigera Lep</em>1 (Ha<em>Lep</em>1) is integrated into the minus strand (positions 2216–2085) and shows 68.7% nucleotide sequence similarity to the <em>Lep</em>1 consensus and retains 5′-CT and 3′-CTAY termini. (B) Integration of a <em>Lep</em>1 element within the <em>O. nubilalis</em> allatotropin neuropeptide precursor gene that is compared with the <em>Spodoptera frugiperda</em> homolog (AJ566903.1) (integration is underscored by &lt;). Gaps in the alignment and missing data are represented by - and *, respectively.</p></div></div><p>Analogously, <em>Lep</em>1 <em>Helitron</em> integrations were described within cDNA-RACE products from 46 <em>O. nubilalis</em> clones (29.6 kb total; mean insert size: 616.1 ± 244.5 bp; GenBank accessions: JG732059–JG732089; JG744027–JG744041). Sequence from RACE products were assembled into 8 contigs and 14 singletons (3.56 ± 1.94 reads per contig), and 21 of these contigs were subsequently annotated as having a <em>Lep</em>1 integration (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Functional annotation of these contigs indicated that all transcript-derived <em>O. nubilalis Lep1 Helitrons</em> were within intron or untranslated regions (data not shown), with the exception of contig04. Contig04 was predicted to show 85% amino acid similarity to the <em>S. frugiperda</em> allatotropin neuropeptide (<em>at2</em>a; GenBank accession CAD98809.1) and that a <em>Lep</em>1 <em>Helitron</em> integration had occurred within the protein-coding regions in the <em>O. nubilalis</em> ortholog (Fig. <a reveal-id="DSR038F5" data-open="DSR038F5" class="link link-reveal link-table xref-fig">5</a>B). When compared to the 53 aa <em>S. frugiperda at2a</em> gene sequence, the C-terminal 37 aa of the 64 residue <em>O. nubilalis</em> ortholog was predicted to be encoded by regions H2 and H3 of an integrated <em>Lep</em>1 <em>Helitron</em> (Fig. <a reveal-id="DSR038F5" data-open="DSR038F5" class="link link-reveal link-table xref-fig">5</a>B). The integration inserted a novel protein-coding sequence that contains a TAA stop codon and changed the predicted molecular weight and isoelectric point of the <em>O. nubilalis at</em>2 protein (pI ∼10.87; 13.6 kDa) compared with that of <em>S. frugiperda</em> (pI = 11.4; 6.1 kDa). The affect of these changes on protein function was not investigated further, but indicated that the <em>Lep</em>1 <em>Helitron</em> can affect the structure and function of gene coding sequences in Lepidoptera.</p><p>In conclusion, a comparative genomics approach was used to identify novel sequences acquired by the highly conserved ancestral <em>Lep</em>1 <em>Helitron</em>. Although the primary sequence among gained sequences are variable, a conservation of secondary structures showed that sequence identity by state is an important factor in determining the success of acquired genomic regions for in subsequent transposition events within the genome. <em>Lep</em>1 provides insight into the structural requirements for RCR in animal <em>Helitrons</em>. Furthermore, the prevalence and preference of <em>Lep</em>1 integrations in proximity to gene-coding regions shows that this class of <em>Helitrons</em> impacts the structure and function of genomes in which they reside.</p>                            <h2 scrollto-destination=7365658 id="7365658" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7365660 id="7365660" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the USDA Current Research Information System (CRIS) project number 3625-22000-017-00D.</p>                            <h2 scrollto-destination=7365662 id="7365662" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This research was a joint contribution of the United States Department of Agriculture (USDA), Agricultural Research Service and the Iowa State University outreach station, Ames, IA (Project 3543). This article reports the results of research only. Mention of a proprietary product or service does not constitute an endorsement or recommendation by USDA or Iowa State University for its use.</p>                            <h2 scrollto-destination=7365664 id="7365664" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsr038c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Rolling-circle transposons in eukaryotes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">8714</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rolling-circle%20transposons%20in%20eukaryotes&amp;author=V.V.%20Kapitonov&amp;author=J.%20Jurka&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=8714-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.151269298" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.151269298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.151269298"> </span></div> </div></div></div></div></div><div content-id="dsr038c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Mouches</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Recent amplification of miniature inverted-repeat transposable elements in the vector mosquito <em>Culex pipiens</em>: characterization of the <em>Mimo</em> family</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">250</div> (pg. <div class="fpage">109</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20amplification%20of%20miniature%20inverted-repeat%20transposable%20elements%20in%20the%20vector%20mosquito%20Culex%20pipiens%3A%20characterization%20of%20the%20Mimo%20family&amp;author=C.%20Feschotte&amp;author=C.%20Mouches&amp;publication_year=2000&amp;journal=Gene&amp;volume=250&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00187-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00187-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00187-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10854784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title"><em>Helitrons</em> on a roll: eukaryotic rolling-circle transposons</div>, <div class="source">Trends Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">521</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helitrons%20on%20a%20roll%3A%20eukaryotic%20rolling-circle%20transposons&amp;author=V.V.%20Kapitonov&amp;author=J.%20Jurka&amp;publication_year=2007&amp;journal=Trends%20Genet.&amp;volume=23&amp;pages=521-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.08.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.08.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.08.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17850916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Messing</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dooner</div> <div class="given-names">H.K.</div></div>. </div><div class="article-title">Gene movement by <em>Helitron</em> transposons contributes to haplotype variability of maize</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">9068</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20movement%20by%20Helitron%20transposons%20contributes%20to%20haplotype%20variability%20of%20maize&amp;author=J.%20Lai&amp;author=Y.%20Li&amp;author=J.%20Messing&amp;author=H.K.%20Dooner&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=9068-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0502923102" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0502923102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0502923102"> </span></div> </div></div></div></div></div><div content-id="dsr038c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galagan</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Calvo</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Cuomo</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Sequencing of <em>Aspergillus nidulans</em> and comparative analysis with <em>A. fumigatus</em> and <em>A. oryzæ</em></div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">438</div> (pg. <div class="fpage">1105</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20Aspergillus%20nidulans%20and%20comparative%20analysis%20with%20A.%20fumigatus%20and%20A.%20oryz%C3%A6&amp;author=J.E.%20Galagan&amp;author=S.E.%20Calvo&amp;author=C.%20Cuomo&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=1105-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04341" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16372000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Donner</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Weil</div> <div class="given-names">C.F.</div></div>. </div><div class="article-title">Give-and-take: interactions between DNA transposons and their host plant genomes</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2009</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">486</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Give-and-take%3A%20interactions%20between%20DNA%20transposons%20and%20their%20host%20plant%20genomes&amp;author=H.K.%20Donner&amp;author=C.F.%20Weil&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=17&amp;pages=486-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2007.08.010" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2007.08.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2007.08.010"> </span></div> </div></div></div></div></div><div content-id="dsr038c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.P.</div></div>,  <div class="name"><div class="surname">Barbash</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Abundant and species specific <em>DINE</em>-1 transposable elements in 12 <em>Drosophila</em> genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">R39</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundant%20and%20species%20specific%20DINE-1%20transposable%20elements%20in%2012%20Drosophila%20genomes&amp;author=H.P.%20Yang&amp;author=D.A.%20Barbash&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=R39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-2-r39" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-2-r39" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-2-r39"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18291035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Structure-based discovery and description of plant and animal <em>Helitrons</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">12832</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure-based%20discovery%20and%20description%20of%20plant%20and%20animal%20Helitrons&amp;author=L.%20Yang&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=12832-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905563106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905563106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905563106"> </span></div> </div></div></div></div></div><div content-id="dsr038c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bureau</div> <div class="given-names">T.E.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title"><em>Tourist</em>: a large family of small inverted repeat elements frequently associated with maize genes</div>, <div class="source">Plant Cell</div> , <div class="year">1992</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1283</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tourist%3A%20a%20large%20family%20of%20small%20inverted%20repeat%20elements%20frequently%20associated%20with%20maize%20genes&amp;author=T.E.%20Bureau&amp;author=S.R.%20Wessler&amp;publication_year=1992&amp;journal=Plant%20Cell&amp;volume=4&amp;pages=1283-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.4.10.1283" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.4.10.1283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.4.10.1283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1332797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Donner</div> <div class="given-names">H.K.</div></div>. </div><div class="article-title">Excision of Helitron transposons in maize</div>, <div class="source">Genetics</div> , <div class="year">2009</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">399</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Excision%20of%20Helitron%20transposons%20in%20maize&amp;author=Y.%20Li&amp;author=H.K.%20Donner&amp;publication_year=2009&amp;journal=Genetics&amp;volume=182&amp;pages=399-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.101527" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.101527" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.101527"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19255366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohmori</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Abiko</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Horibata</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hirano</div> <div class="given-names">H.Y.</div></div>. </div><div class="article-title">A transposon, <em>Ping</em>, is integrated into intron 4 of the DROOPING LEAF gene of rice, weakly reducing its expression and causing a mild drooping leaf phenotype</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">1176</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20transposon%2C%20Ping%2C%20is%20integrated%20into%20intron%204%20of%20the%20DROOPING%20LEAF%20gene%20of%20rice%2C%20weakly%20reducing%20its%20expression%20and%20causing%20a%20mild%20drooping%20leaf%20phenotype&amp;author=Y.%20Ohmori&amp;author=M.%20Abiko&amp;author=A.%20Horibata&amp;author=H.Y.%20Hirano&amp;publication_year=2008&amp;journal=Plant%20Cell%20Physiol.&amp;volume=49&amp;pages=1176-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcn093" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcn093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcn093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varagona</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Purugganan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Alternative splicing induced by insertion of retrotransposons into the maize <em>waxy</em> gene</div>, <div class="source">Plant Cell</div> , <div class="year">1992</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">811</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20induced%20by%20insertion%20of%20retrotransposons%20into%20the%20maize%20waxy%20gene&amp;author=M.J.%20Varagona&amp;author=M.%20Purugganan&amp;author=S.R.%20Wessler&amp;publication_year=1992&amp;journal=Plant%20Cell&amp;volume=4&amp;pages=811-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.4.7.811" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.4.7.811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.4.7.811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1327340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Boue</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Forneck</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Casacuberta</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Recent amplification and impact of MITEs on the genome of grapevine (<em>Vitis vinifera</em> L.)</div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">75</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20amplification%20and%20impact%20of%20MITEs%20on%20the%20genome%20of%20grapevine%20(Vitis%20vinifera%20L.)&amp;author=A.%20Benjak&amp;author=S.%20Boue&amp;author=A.%20Forneck&amp;author=J.M.%20Casacuberta&amp;publication_year=2009&amp;journal=Genome%20Biol.%20Evol.&amp;volume=1&amp;pages=75-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evp009" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevp009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevp009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20333179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Treasures in the attic: rolling circle transposons discovered in eukaryotic genomes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">8923</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Treasures%20in%20the%20attic%3A%20rolling%20circle%20transposons%20discovered%20in%20eukaryotic%20genomes&amp;author=C.%20Feschotte&amp;author=S.R.%20Wessler&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=8923-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.171326198" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.171326198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.171326198"> </span></div> </div></div></div></div></div><div content-id="dsr038c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dooner</div> <div class="given-names">H.K.</div></div>. </div><div class="article-title">Intraspecific violation of genetic colinearity and its implications in maize</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">9573</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intraspecific%20violation%20of%20genetic%20colinearity%20and%20its%20implications%20in%20maize&amp;author=H.%20Fu&amp;author=H.K.%20Dooner&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=9573-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.132259199" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.132259199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.132259199"> </span></div> </div></div></div></div></div><div content-id="dsr038c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pea</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fengler</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Gene duplication and exon shuffling by <em>Helitron</em>-like transposons generate intraspecies diversity in maize</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">997</div>-<div class="lpage">1002</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20exon%20shuffling%20by%20Helitron-like%20transposons%20generate%20intraspecies%20diversity%20in%20maize&amp;author=M.%20Morgante&amp;author=S.%20Brunner&amp;author=G.%20Pea&amp;author=K.%20Fengler&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=997-1002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1615" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lal</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Hannah</div> <div class="given-names">L.C.</div></div>. </div><div class="article-title"><em>Helitrons</em> contribute to the lack of gene colinearity observed in modern maize inbreds</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">9993</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helitrons%20contribute%20to%20the%20lack%20of%20gene%20colinearity%20observed%20in%20modern%20maize%20inbreds&amp;author=S.K.%20Lal&amp;author=L.C.%20Hannah&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=9993-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0504713102" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0504713102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0504713102"> </span></div> </div></div></div></div></div><div content-id="dsr038c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Britten</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Coding sequences of functioning human genes derived entirely from mobile element sequences</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">16825</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding%20sequences%20of%20functioning%20human%20genes%20derived%20entirely%20from%20mobile%20element%20sequences&amp;author=R.J.%20Britten&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=16825-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0406985101" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0406985101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0406985101"> </span></div> </div></div></div></div></div><div content-id="dsr038c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Transposable elements, gene creation and genome rearrangement in flowering plants</div>, <div class="source">Curr. Opin. Gene Devel.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">621</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%2C%20gene%20creation%20and%20genome%20rearrangement%20in%20flowering%20plants&amp;author=J.L.%20Bennetzen&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Gene%20Devel.&amp;volume=15&amp;pages=621-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2005.09.010" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2005.09.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2005.09.010"> </span></div> </div></div></div></div></div><div content-id="dsr038c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Distribution, diversity, evolution, and survival of <em>Helitrons</em> in the maize genome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">19922</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%2C%20diversity%2C%20evolution%2C%20and%20survival%20of%20Helitrons%20in%20the%20maize%20genome&amp;author=L.%20Yang&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=19922-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0908008106" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0908008106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0908008106"> </span></div> </div></div></div></div></div><div content-id="dsr038c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gallavotti</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stryker</div> <div class="given-names">G.A.</div></div>, et al. </div><div class="article-title">A novel class of <em>Helitron</em>-related transposable elements in maize contain portions of multiple pseudogenes</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">115</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20Helitron-related%20transposable%20elements%20in%20maize%20contain%20portions%20of%20multiple%20pseudogenes&amp;author=S.%20Gupta&amp;author=A.%20Gallavotti&amp;author=G.A.%20Stryker&amp;publication_year=2005&amp;journal=Plant%20Mol.%20Biol.&amp;volume=57&amp;pages=115-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-004-6636-z" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-004-6636-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-004-6636-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15821872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pulletikurti</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Alternative Ac/Ds transposition induces major chromosomal rearrangements in maize</div>, <div class="source">Genes Dev.</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">755</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20Ac%2FDs%20transposition%20induces%20major%20chromosomal%20rearrangements%20in%20maize&amp;author=J.%20Zhang&amp;author=C.%20Yu&amp;author=V.%20Pulletikurti&amp;publication_year=2009&amp;journal=Genes%20Dev.&amp;volume=23&amp;pages=755-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1776909" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1776909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1776909"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19299561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McClintock</div> <div class="given-names">B.</div></div>. </div><div class="article-title">The origin and behavior of mutable loci in maize</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1950</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">344</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20and%20behavior%20of%20mutable%20loci%20in%20maize&amp;author=B.%20McClintock&amp;publication_year=1950&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=36&amp;pages=344-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.36.6.344" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.36.6.344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.36.6.344"> </span></div> </div></div></div></div></div><div content-id="dsr038c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Retrotransposon-mediated genome evolution on a local ecological scale</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">6250</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposon-mediated%20genome%20evolution%20on%20a%20local%20ecological%20scale&amp;author=J.F.%20Wendel&amp;author=S.R.%20Wessler&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=6250-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.12.6250" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.12.6250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.12.6250"> </span></div> </div></div></div></div></div><div content-id="dsr038c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Binghamp</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kidwell</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Rubin</div> <div class="given-names">G.M.</div></div>. </div><div class="article-title">The molecular basis of PM hybrid dysgenesis: the role of the <em>P</em> element, a <em>P</em>-strain-specific transposon family</div>, <div class="source">Cell</div> , <div class="year">1982</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">995</div>-<div class="lpage">1004</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20basis%20of%20PM%20hybrid%20dysgenesis%3A%20the%20role%20of%20the%20P%20element%2C%20a%20P-strain-specific%20transposon%20family&amp;author=M.%20Binghamp&amp;author=M.G.%20Kidwell&amp;author=G.M.%20Rubin&amp;publication_year=1982&amp;journal=Cell&amp;volume=29&amp;pages=995-1004" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(82)90463-9" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(82)90463-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(82)90463-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6295641" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noor</div> <div class="given-names">M.A.F.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">A.S.</div></div>. </div><div class="article-title">Evolutionary genetics: jumping into a new species</div>, <div class="source"><em>Curr. Biol</em>.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">R890</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20genetics%3A%20jumping%20into%20a%20new%20species&amp;author=M.A.F.%20Noor&amp;author=A.S.%20Chang&amp;publication_year=2006&amp;journal=Curr.%20Biol.&amp;volume=16&amp;pages=R890-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2006.09.022" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2006.09.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2006.09.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17055974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gonzalez</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Karasov</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Messer</div> <div class="given-names">P.W.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Genome-wide patterns of adaptation to temperate environments associated with transposable elements in <em>Drosophila</em></div>, <div class="source">PLoS Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1000905</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20patterns%20of%20adaptation%20to%20temperate%20environments%20associated%20with%20transposable%20elements%20in%20Drosophila&amp;author=J.%20Gonzalez&amp;author=T.%20Karasov&amp;author=P.W.%20Messer&amp;author=D.A.%20Petrov&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1000905" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000905" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000905"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20386746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gahan</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Gould</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Heckel</div> <div class="given-names">D.G.</div></div>. </div><div class="article-title">Identification of a gene associated with Bt resistance in <em>Heliothis virescens</em></div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">293</div> (pg. <div class="fpage">857</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20gene%20associated%20with%20Bt%20resistance%20in%20Heliothis%20virescens&amp;author=L.J.%20Gahan&amp;author=F.%20Gould&amp;author=D.G.%20Heckel&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=857-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1060949" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1060949" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1060949"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11486086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">d'Alençon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Sezutsu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Legeai</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Extensive synteny conservation of holocentric chromosomes in Lepidoptera despite high rates of local genome rearrangements</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">7600</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20synteny%20conservation%20of%20holocentric%20chromosomes%20in%20Lepidoptera%20despite%20high%20rates%20of%20local%20genome%20rearrangements&amp;author=E.%20d'Alen%C3%A7on&amp;author=H.%20Sezutsu&amp;author=F.%20Legeai&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=7600-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0910413107" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0910413107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0910413107"> </span></div> </div></div></div></div></div><div content-id="dsr038c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="collab">International Silkworm Genome Consortium</div><div class="article-title">The genome of a lepidopteran model insect, the silkworm <em>Bombyx mori</em></div>, <div class="source">Insect Biochem. Mol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1036</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ibmb.2008.11.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ibmb.2008.11.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ibmb.2008.11.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121390" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kasahara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome sequence of silkworm, <em>Bombyx mori</em></div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">27</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20silkworm%2C%20Bombyx%20mori&amp;author=K.%20Mita&amp;author=M.%20Kasahara&amp;author=S.%20Sasaki&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=27-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.1.27" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15141943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A draft sequence for the genome of the domesticated silkworm (<em>Bombyx mori</em>)</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">306</div> (pg. <div class="fpage">1937</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20for%20the%20genome%20of%20the%20domesticated%20silkworm%20(Bombyx%20mori)&amp;author=Q.%20Xia&amp;author=Z.%20Zhou&amp;author=C.%20Lu&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=1937-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1102210" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1102210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1102210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15591204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osanai-Futahashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suetsugu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujiwara</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genome-wide screening and characterization of transposable elements and their distribution in the silkworm, <em>Bombyx mori</em></div>, <div class="source">Insect Biochem. Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1046</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20screening%20and%20characterization%20of%20transposable%20elements%20and%20their%20distribution%20in%20the%20silkworm%2C%20Bombyx%20mori&amp;author=M.%20Osanai-Futahashi&amp;author=Y.%20Suetsugu&amp;author=K.%20Mita&amp;author=H.%20Fujiwara&amp;publication_year=2009&amp;journal=Insect%20Biochem.%20Mol.%20Biol.&amp;volume=38&amp;pages=1046-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ibmb.2008.05.012" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ibmb.2008.05.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ibmb.2008.05.012"> </span></div> </div></div></div></div></div><div content-id="dsr038c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okada</div> <div class="given-names">N.</div></div>. </div><div class="article-title">SINEs: short interspersed repeated elements of the eukaryotic genome</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">1991</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">358</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SINEs%3A%20short%20interspersed%20repeated%20elements%20of%20the%20eukaryotic%20genome&amp;author=N.%20Okada&amp;publication_year=1991&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=6&amp;pages=358-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0169-5347(91)90226-N" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0169-5347(91)90226-N" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0169-5347(91)90226-N"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21232509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coates</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Sumerford</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Hellmich</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Lewis</div> <div class="given-names">L.C.</div></div>. </div><div class="article-title">A <em>Helitron</em>-like transposon superfamily from Lepidoptera disrupts (GAAA)<sub>n</sub> microsatellites and is responsible for flanking sequence similarity within a microsatellite family</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">278</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Helitron-like%20transposon%20superfamily%20from%20Lepidoptera%20disrupts%20(GAAA)n%20microsatellites%20and%20is%20responsible%20for%20flanking%20sequence%20similarity%20within%20a%20microsatellite%20family&amp;author=B.S.%20Coates&amp;author=D.V.%20Sumerford&amp;author=R.L.%20Hellmich&amp;author=L.C.%20Lewis&amp;publication_year=2010&amp;journal=J.%20Mol.%20Evol.&amp;volume=70&amp;pages=278-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-010-9330-6" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-010-9330-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-010-9330-6"> </span></div> </div></div></div></div></div><div content-id="dsr038c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Teng</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zurovec</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Characterization of the <em>P25</em> silk gene and associated insertion elements in <em>Galleria melonella</em></div>, <div class="source">Gene</div> , <div class="year">1998</div>, vol. <div class="volume">209</div> (pg. <div class="fpage">157</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20P25%20silk%20gene%20and%20associated%20insertion%20elements%20in%20Galleria%20melonella&amp;author=C.%20Yang&amp;author=X.%20Teng&amp;author=M.%20Zurovec&amp;publication_year=1998&amp;journal=Gene&amp;volume=209&amp;pages=157-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(98)00029-8" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(98)00029-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(98)00029-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9583947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van't Hof</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Brakefield</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Saccheri</div> <div class="given-names">I.J.</div></div>,  <div class="name"><div class="surname">Zwaan</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Evolutionary dynamics of multilocus microsatellite arrangements in the genome of the butterfly <em>Bicyclus anynana</em>, with implications for other Lepidoptera</div>, <div class="source">Heredity</div> , <div class="year">2007</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">320</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20multilocus%20microsatellite%20arrangements%20in%20the%20genome%20of%20the%20butterfly%20Bicyclus%20anynana%2C%20with%20implications%20for%20other%20Lepidoptera&amp;author=A.E.%20Van't%20Hof&amp;author=P.M.%20Brakefield&amp;author=I.J.%20Saccheri&amp;author=B.J.%20Zwaan&amp;publication_year=2007&amp;journal=Heredity&amp;volume=98&amp;pages=320-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800944" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17327875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT</div>, <div class="source">Nucl. Acids Symp. Ser.</div> , <div class="year">1999</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">95</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="dsr038c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA4: molecular evolutionary genetic analysis MEGA. software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1596</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20molecular%20evolutionary%20genetic%20analysis%20MEGA.%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zuker</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mfold web server for nucleic acid folding and hybridization prediction</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3406</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mfold%20web%20server%20for%20nucleic%20acid%20folding%20and%20hybridization%20prediction&amp;author=M.%20Zuker&amp;publication_year=2003&amp;journal=Nucl.%20Acids%20Res.&amp;volume=31&amp;pages=3406-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg595" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg595"> </span></div> </div></div></div></div></div><div content-id="dsr038c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Youens-Clark</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Faga</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Yap</div> <div class="given-names">I.L.</div></div>,  <div class="name"><div class="surname">Stein</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>. </div><div class="article-title">CMap 1,01: a comparative mapping application for the Internet</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3042</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CMap%201%2C01%3A%20a%20comparative%20mapping%20application%20for%20the%20Internet&amp;author=K.%20Youens-Clark&amp;author=B.%20Faga&amp;author=I.L.%20Yap&amp;author=L.%20Stein&amp;author=D.%20Ware&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=3042-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp458" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp458" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp458"> </span></div> </div></div></div></div></div><div content-id="dsr038c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garca-Gomez</div> <div class="given-names">J. M.</div></div>, et al. </div><div class="article-title">Blast2go: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2go%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.%20M.%20Garca-Gomez&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">High-throughput functional annotation and data mining with the Blast2GO suite</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3421</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20functional%20annotation%20and%20data%20mining%20with%20the%20Blast2GO%20suite&amp;author=S.%20G%C3%B6tz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;publication_year=2008&amp;journal=Nucl.%20Acids%20Res.&amp;volume=36&amp;pages=3421-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn176" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn176"> </span></div> </div></div></div></div></div><div content-id="dsr038c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coates</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Kroemer</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Sumerford</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Hellmich</div> <div class="given-names">R.L.</div></div>. </div><div class="article-title">A novel class of miniature inverted repeat transposable elements MITEs that contain hitchhiking GTCYn microsatellites</div>, <div class="source">Insect Mol. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">15</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20miniature%20inverted%20repeat%20transposable%20elements%20MITEs%20that%20contain%20hitchhiking%20GTCYn%20microsatellites&amp;author=B.S.%20Coates&amp;author=J.A.%20Kroemer&amp;author=D.V.%20Sumerford&amp;author=R.L.%20Hellmich&amp;publication_year=2011&amp;journal=Insect%20Mol.%20Biol.&amp;volume=20&amp;pages=15-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imb.2011.20.issue-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimb.2011.20.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimb.2011.20.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20977507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tempel</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nicolas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Amrani</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Couee</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Model-based identification of <em>Helitrons</em> results in a new classification of their families in <em>Arabidopsis thaliana</em></div>, <div class="source">Gene</div> , <div class="year">2007</div>, vol. <div class="volume">403</div> (pg. <div class="fpage">18</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Model-based%20identification%20of%20Helitrons%20results%20in%20a%20new%20classification%20of%20their%20families%20in%20Arabidopsis%20thaliana&amp;author=S.%20Tempel&amp;author=J.%20Nicolas&amp;author=A.E.%20Amrani&amp;author=I.%20Couee&amp;publication_year=2007&amp;journal=Gene&amp;volume=403&amp;pages=18-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.06.030" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.06.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.06.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17889452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sweredowski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">L.D.R.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>. </div><div class="article-title">A comparative computational analysis of nonautonomous <em>Helitron</em> elements between maize and rice</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">467</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20computational%20analysis%20of%20nonautonomous%20Helitron%20elements%20between%20maize%20and%20rice&amp;author=M.%20Sweredowski&amp;author=L.D.R.%20Wilson&amp;author=B.S.%20Gaut&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-467" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18842139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leung</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">J.T.Y.</div></div>. </div><div class="article-title">The replication of plastid microcircles involved rolling circle intermediates</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">1991</div>-<div class="lpage">2002</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20replication%20of%20plastid%20microcircles%20involved%20rolling%20circle%20intermediates&amp;author=S.K.%20Leung&amp;author=J.T.Y.%20Wong&amp;publication_year=2009&amp;journal=Nucl.%20Acids%20Res.&amp;volume=37&amp;pages=1991-2002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp063" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp063"> </span></div> </div></div></div></div></div><div content-id="dsr038c49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoder</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Walsh</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Bestor</div> <div class="given-names">T.H.</div></div>. </div><div class="article-title">Cytosine methylation and the ecology of intragenomic parasites</div>, <div class="source">Trends Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">335</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytosine%20methylation%20and%20the%20ecology%20of%20intragenomic%20parasites&amp;author=J.A.%20Yoder&amp;author=C.P.%20Walsh&amp;author=T.H.%20Bestor&amp;publication_year=1997&amp;journal=Trends%20Genet.&amp;volume=13&amp;pages=335-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(97)01181-5" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(97)01181-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(97)01181-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9260521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orgel</div> <div class="given-names">L.E.</div></div>,  <div class="name"><div class="surname">Crick</div> <div class="given-names">F.H.C.</div></div>. </div><div class="article-title">Selfish DNA-the ultimate parasite</div>, <div class="source">Nature</div> , <div class="year">1980</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">604</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20DNA-the%20ultimate%20parasite&amp;author=L.E.%20Orgel&amp;author=F.H.C.%20Crick&amp;publication_year=1980&amp;journal=Nature&amp;volume=284&amp;pages=604-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/284604a0" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F284604a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F284604a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7366731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Kirkness</div> <div class="given-names">E.F.</div></div>. </div><div class="article-title">Short interspersed elements SINEs are a major source of canine genome diversity</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1798</div>-<div class="lpage">808</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20interspersed%20elements%20SINEs%20are%20a%20major%20source%20of%20canine%20genome%20diversity&amp;author=W.%20Wang&amp;author=E.F.%20Kirkness&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1798-808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3765505" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3765505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3765505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sunter</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Skilton</div> <div class="given-names">R.A.</div></div>, et al. </div><div class="article-title">A novel SINE family occurs frequently in both genomic DNA and transcribed sequences in ixodid ticks from the arthropod sub-phylum Chelicerata</div>, <div class="source">Gene</div> , <div class="year">2008</div>, vol. <div class="volume">415</div> (pg. <div class="fpage">13</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20SINE%20family%20occurs%20frequently%20in%20both%20genomic%20DNA%20and%20transcribed%20sequences%20in%20ixodid%20ticks%20from%20the%20arthropod%20sub-phylum%20Chelicerata&amp;author=J.D.%20Sunter&amp;author=S.P.%20Patel&amp;author=R.A.%20Skilton&amp;publication_year=2008&amp;journal=Gene&amp;volume=415&amp;pages=13-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.01.026" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.01.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.01.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18394826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lipatov</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lenkov</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Bergman</div> <div class="given-names">C.M.</div></div>. </div><div class="article-title">Paucity of chimeric gene-transposable element transcripts in the <em>Drosophila melanogaster</em> genome</div>, <div class="source">BMC Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> pg. <div class="fpage">24</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paucity%20of%20chimeric%20gene-transposable%20element%20transcripts%20in%20the%20Drosophila%20melanogaster%20genome&amp;author=M.%20Lipatov&amp;author=K.%20Lenkov&amp;author=D.A.%20Petrov&amp;author=C.M.%20Bergman&amp;publication_year=2005&amp;journal=BMC%20Biol.&amp;volume=3&amp;pages=24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1741-7007-3-24" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1741-7007-3-24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1741-7007-3-24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16283942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van de Legemaat</div> <div class="given-names">L.N.</div></div>,  <div class="name"><div class="surname">Landry</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Mager</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Medstrand</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Transposable elements in mammals promote regulatory variation and diversification of genes with specialized functions</div>, <div class="source">Trends Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">530</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%20in%20mammals%20promote%20regulatory%20variation%20and%20diversification%20of%20genes%20with%20specialized%20functions&amp;author=L.N.%20van%20de%20Legemaat&amp;author=J.R.%20Landry&amp;author=D.L.%20Mager&amp;author=P.%20Medstrand&amp;publication_year=2003&amp;journal=Trends%20Genet.&amp;volume=19&amp;pages=530-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.08.004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.08.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2003.08.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14550626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr038c55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giroux</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Clancy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Baier</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title"><em>De novo</em> synthesis of an intron by the maize transposable element dissociation</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">12150</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20synthesis%20of%20an%20intron%20by%20the%20maize%20transposable%20element%20dissociation&amp;author=M.J.%20Giroux&amp;author=M.%20Clancy&amp;author=J.%20Baier&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=12150-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.25.12150" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.25.12150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.25.12150"> </span></div> </div></div></div></div></div><div content-id="dsr038c56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr038c56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davis</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Dietz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Standiford</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Emerson</div> <div class="given-names">C.P.</div>Jr.</div>. </div><div class="article-title">Transposable element insertions respecify alternative exon splicing in three <em>Drosophila</em> myosin heavy chain mutants</div>, <div class="source">Genetics</div> , <div class="year">1988</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">1105</div>-<div class="lpage">14</div>)</div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Published by Oxford University Press on behalf of Kazusa DNA Research Institute 2011</div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484721&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr038-dsr038supp_table3.txt?Expires=1509238785&amp;Signature=ZnNODiO054Vnn1abaG~ErjynCt7eDHTjdUloxXkB2OdBY8pr5vLh-GhJEm6PZq8W-xGgSnnK-GC5S2dOkmrv3QxXufJUEufVh1bWxqEunbSA4YXgyZcmeEQ12NGcFPrhA1A9DlaR5n9S0B6Qea~l-5rM6zDBQqGm9qf9yIka7QhqCI6I80-egj2~MuRP4AUaJUr3H~HwGn3Y8uycCe05N8Xse2j-KGn89WAOxKJ72UmBBQTPFesngdVrsB36xaQ3OWMoECXEFVQeOPtv0SxRL~hCcgbELmFKiFcCD0RQw7V8CgKagWL-cvSOnX2-oEu2ShIJs8nLugh0rEzeyudaQw__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-7</a> - txt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr038-dsr038supp_table2.doc?Expires=1509238785&amp;Signature=XlqweHRwCumUCsLdqVbaeWvMT-Wr04BT21bDiy93d4VusSs03aJFJ-sSwhbM8AExI-CTyf~Z0lOMgRtTKq8B2ZXjA9w~8GVFj4tN~8GHjb3pOlL9Et9cN0OS7-NEqbwRMeOeGUIF71hFbZIVnO45g1O0oXa453RrVyjnKgyOrhzlfVbpnnuFfybt9m2I4tR5X47KHVVgS64lMGAsqUO3hP1r8zWyGZ7KhkFthUqKUfrD8Wn5mD2kZITMNYoGEylh~nyIdPRx2twYMyRyhba3HQJteNX4jHoXizNC27Ec6-56fnAEKDhi6z-HHIO2pLaDQx1VR7j93-aTMUOof3qOFg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr038-dsr038supp_table1.doc?Expires=1509238785&amp;Signature=gK7bKOBp92S3uQVjd6wSzxZv-3LNTBzNHbZpMz36sJcJHOC6CaccDSRnI5DzffxAvzgWHCnUvAaaRAd0XHAtq2VaL5qpuSWwOF~SnA90w8nHZJIbD-H04RpJkT8iEW3R~ktlsiybezGwLDVAoytbnLW8LHhiGS-wzTjBs-SOJyo-A8j8tHPcypZ58arZJIUtzTLgapxFrKxt3hfWqyPykzhYZoCYJiPbcM0NcC4y6DB9ThSnUo91JbG0o1rBUytaT7unFUOJw7sPaAtccPhWrOKdmECyZHJDJNWovRnnPW6S09Ycr6WYMOtS0zI8nux3ltlbpsqDQ0GKaOMcsjeVeg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr038-dsr038supp_fig2.png?Expires=1509238785&amp;Signature=SY8LMPqUwbnnIV1TayBzMnrazWq4l7es5zpJ8D~Zz0zHLdwPe9M42XEMDpLnD3DCcTz8vODc4Ftj3laQ1P9OCb1ehDPRlcyz38juk-UjANV5l2OsAkAZgriG56iHiAkiUI~8DiSGdqUA8f7jxmZ07R65Cd5GrU44YqUhKA5qq1Bs20KV1KxVOxOVthMK2Fcw-BPgIR2~s5jqJWKjqb7NVLMTXIhegNNmQuOoD1wViGfiuyuSyHuzrLyxDGOcukxJJZDyPEdNhPtXTlUAO~jMs6dtgrNq6hSgZ1v98nyyzHMoTq1j5l7QBd0NlAZetKO46fOwJXwxbtxMxtWD-lOKHQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr038-dsr038supp_fig1.png?Expires=1509238785&amp;Signature=YLmwOStsM4xbYWMKl2xeJpQ8i28xxMhmgftUggWAwMZsn0NKY2EpOlbg7~BAI8TzNqsogJBey-ZsTLV61pUsssxbyNvSajh-P0FaEQiHPL0oaR5pvS~wSFeziL1Vp8mfheCXEiPwuqQWoicpD0dyYCbH1i2SBN4LQGwUW0MtWXwRD9YghdYStxCl7jJGY6-Iq~lqUvUrOdHidiG3KntFr4nyYu16yrvYzGZqbQbx~KzXq0QpNLUVFn9BKjQEMTYvOOD-QWjdn1MB3S7T24E3IQxhUnwONIPyHLY9PkU7HytHQ0TbYU~1~4717UpdzVc66V7NhllZ-DpwfrEZ45tAfg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr038/2/dsr038-dsr038supp.doc?Expires=1509238785&amp;Signature=I3TEvDFrkJ9wF-vtvfSXWZMmKQW0ub2IFJROpVc-wVFNUPvquIig4sPHqv-helwPl9OlsFi3yVikHP79RvjVzt0LHgJ65LeuZuJbH5CitAVvQ~S7p0aU7eVHqVusN84GKBtL3DdqcSApg6Vx9lzfgfwEXbt2EsrYbc0AuVEzzSDHggk7H6124peebITevyNPeTvvp8pCe3EuxLlACpOkirD4L9d1lP5fDESBqWeeQoX4Rw~fvjGrqoj-NZXWcMTjAC37JhaXwRu-nn4ul0B3NNfKDaANYrW3ujFQ4oVu7DNsl2a4eAlL2KoWqZoIVbLZuxg-oPgJq-M9EYIZLIdwyQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">12</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsr038"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484721"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f1%2f11%2f484721">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000300760000002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F1%2F11%2F484721" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484721'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Mobilizing the Genome of Lepidoptera through Novel Sequence Gains and End Creation by Non-autonomous Lep1 Helitrons" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f11%2f484721">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

