

<!DOCTYPE html>
<html lang="en">

<head><title>
	Preferential Nucleosome Occupancy at High Values of DNA Helical Rise | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/1/81/486167","@type":"ScholarlyArticle","name":"Preferential Nucleosome Occupancy at High Values of DNA Helical Rise","datePublished":"2012-01-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr043","keywords":["nucleosome","tetranucleotide","helical rise"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/1/81/486167","author":[{"name":"Pedone, Francesco","affiliation":"1  Department of Biology and Biotechnology ,  ‘Sapienza’ University ,  P.le A. Moro 5, 00185 Rome ,  Italy","@type":"Person"},{"name":"Santoni, Daniele","affiliation":"2  Barcelona Institute for Research in Biomedicine (IRB) ,  Barcelona Science Park, C/Samitier 1-5, 08015 Barcelona ,  Spain    3  Istituto di Analisi dei Sistemi e Informatica Antonio Ruberti, Consiglio Nazionale delle Ricerche ,  Viale Manzoni 30, 00185 Rome ,  Italy","@type":"Person"}],"description":"Nucleosomes are the basic structural units of eukaryotic chromatin and play a key role in the regulation of gene expression. Nucleosome formation depends on several factors, including properties of the sequence itself, but also physical constraints and epigenetic factors such as chromatin-remodelling enzymes. In this view, a sequence-dependent approach is able to capture a general tendency of a region to bind a histone octamer. A reference data set of positioned nucleosomes of Saccharomyces cerevisiae was used to study the role of DNA helical rise in histone–DNA interaction. Genomic sequences were transformed into arrays of helical rise values by a tetranucleotide code and then turned into profiles of mean helical rise values. These profiles resemble maps of nucleosome occupancy, suggesting that intrinsic histone–DNA interactions are linked to helical rise. The obtained results show that preferential nucleosome occupancy occurs where the mean helical rise reaches its largest values. Mean helical rise profiles obtained by using maps of positioned nucleosomes of the Drosophila melanogaster and Plasmodium falciparum genomes, as well as Homo sapiens chromosome 20 confirm that nucleosomes are mainly located where the mean helical rise reaches its largest values.","pageStart":"81","pageEnd":"90","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=1572217195&Signature=GXQlZsJtWwSMjtnQ-J4B1rD~6iBlW-G1Jx76BNUd9gPa-NR~aGVuOKNUON4zfCRvzZm8FEjmGr0EUic6cXei39XkQq56ueOXpSXFQdBorKVKuLgNiGUHMp77JhdTsV6dX-dgMzV8Ox7gxyOT23sZhm16WiwFobtr6Rq2TXwP5XWIwoZg6uVQ8uW7yzcndQ9cXrxfl2sbg-UXK39xS9JAxO52fybnHb1KRCzatn3DDg4vDu9bZeNEOh~Q3voH274F-OZjOw-jS-E~16D~UMqfWbCiOdvEZ58o-I2Qe7GaPVLJ2nHQKsZLWKe3tAvlHZvN-OQZzAm-ZG5oOQfRr6206w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Preferential Nucleosome Occupancy at High Values of DNA Helical Rise","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=1572217195&Signature=GXQlZsJtWwSMjtnQ-J4B1rD~6iBlW-G1Jx76BNUd9gPa-NR~aGVuOKNUON4zfCRvzZm8FEjmGr0EUic6cXei39XkQq56ueOXpSXFQdBorKVKuLgNiGUHMp77JhdTsV6dX-dgMzV8Ox7gxyOT23sZhm16WiwFobtr6Rq2TXwP5XWIwoZg6uVQ8uW7yzcndQ9cXrxfl2sbg-UXK39xS9JAxO52fybnHb1KRCzatn3DDg4vDu9bZeNEOh~Q3voH274F-OZjOw-jS-E~16D~UMqfWbCiOdvEZ58o-I2Qe7GaPVLJ2nHQKsZLWKe3tAvlHZvN-OQZzAm-ZG5oOQfRr6206w__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Preferential Nucleosome Occupancy at High Values of DNA Helical Rise | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Nucleosomes are the basic structural units of eukaryotic chromatin and play a key role in the regulation of gene expression. Nucleosome formation depends on several factors, including properties of the sequence itself, but also physical constraints and epigenetic factors such as chromatin-remodelling enzymes. In this view, a sequence-dependent approach is able to capture a general tendency of a region to bind a histone octamer. A reference data set of positioned nucleosomes of Saccharomyces cerevisiae was used to study the role of DNA helical rise in histone–DNA interaction. Genomic sequences were transformed into arrays of helical rise values by a tetranucleotide code and then turned into profiles of mean helical rise values. These profiles resemble maps of nucleosome occupancy, suggesting that intrinsic histone–DNA interactions are linked to helical rise. The obtained results show that preferential nucleosome occupancy occurs where the mean helical rise reaches its largest values. Mean helical rise profiles obtained by using maps of positioned nucleosomes of the Drosophila melanogaster and Plasmodium falciparum genomes, as well as Homo sapiens chromosome 20 confirm that nucleosomes are mainly located where the mean helical rise reaches its largest values." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/1/81/486167" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572217195&amp;Signature=PpS1RGRRFIU6imMm76ZIZSswXl-oe9kKogHebSGaK1kFFjN2ialaEu3vkyQUXUo56bYi4hR9fgHHelJqIMVopXTUFkf1l6aqGiLtkKxUeOJ1ga0TihXeul8JPAc2SAEW5A08nd96dwMR4Cpt7rNhg9kD-MnlbApRjW1Rs9YybR~EXzX5GuqldSCX~j501ig9Bw4NxSfkCS4K0byQnquKJKzSrzW0~JaatKLK5BMQ9ijZpqeesfgNRViMakcnIGKdhZ4IYwZQb-MhuycZ29o1-OErlcQPL6K5G6q1qjp0bLTozQ1Dm-yeQoqleBrVDHCjAwBe~X~mh~ZEP4wRIvRBzA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2012-01-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Pedone, Francesco" /><meta name="citation_author_institution" content="Department of Biology and Biotechnology, ‘Sapienza’ University, P.le A. Moro 5, 00185 Rome, Italy" /><meta name="citation_author" content="Santoni, Daniele" /><meta name="citation_author_institution" content="Barcelona Institute for Research in Biomedicine (IRB), Barcelona Science Park, C/Samitier 1-5, 08015 Barcelona, Spain" /><meta name="citation_author_institution" content="Istituto di Analisi dei Sistemi e Informatica Antonio Ruberti, Consiglio Nazionale delle Ricerche, Viale Manzoni 30, 00185 Rome, Italy" /><meta name="citation_title" content="Preferential Nucleosome Occupancy at High Values of DNA Helical Rise" /><meta name="citation_firstpage" content="81" /><meta name="citation_lastpage" content="90" /><meta name="citation_doi" content="10.1093/dnares/dsr043" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2012/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=On the occurrence of nucleosome phasing in chromatin; Cell;  citation_year=1977;  citation_volume=12;  citation_pages=829-36; " /><meta name="citation_reference" content="citation_title=Structure of DNA in chromatin; Eur. J. Cell Biol.;  citation_year=1980;  citation_volume=22;  citation_pages=74" /><meta name="citation_reference" content="citation_title=The pitch of chromatin DNA is reflected in its nucleotide sequence; Proc. Natl Acad. Sci. USA;  citation_year=1980;  citation_volume=77;  citation_pages=3816-20; " /><meta name="citation_reference" content="citation_title=Nucleotide sequence-directed mapping of the nucleosome; Nucleic Acids Res.;  citation_year=1983;  citation_volume=11;  citation_pages=3833-51; " /><meta name="citation_reference" content="citation_title=The [(G/C)3NN]n motif: a common DNA repeat that excludes nucleosomes; Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=8863-7; " /><meta name="citation_reference" content="citation_title=New DNA sequence rules for high affinity binding to hystone octamer and sequence-directed nucleosome positioning; J. Mol. Biol.;  citation_year=1998;  citation_volume=276;  citation_pages=19-42; " /><meta name="citation_reference" content="citation_title=Nucleosomal locations of dominant DNA sequence motifs for histone–DNA interactions and nucleosome positioning; J. Mol. Biol.;  citation_year=2004;  citation_volume=338;  citation_pages=695-709; " /><meta name="citation_reference" content="citation_title=Gene expression divergence in yeast is coupled to evolution of DNA-encoded nucleosome organization; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=438-45; " /><meta name="citation_reference" content="citation_title=A genomic code for nucleosome positioning; Nature;  citation_year=2006;  citation_volume=442;  citation_pages=772-8; " /><meta name="citation_reference" content="citation_title=The DNA-encoded nucleosome organization of a eukaryotic genome; Nature;  citation_year=2009;  citation_volume=458;  citation_pages=362-6; " /><meta name="citation_reference" content="citation_title=Nucleosome positioning signals in genomic DNA; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1170-7; " /><meta name="citation_reference" content="citation_title=DNA physical properties determine nucleosome occupancy from yeast to fly; Nucleic Acid Res.;  citation_year=2008;  citation_volume=36;  citation_pages=3746-56; " /><meta name="citation_reference" content="citation_title=Using DNA mechanics to predict in vitro nucleosome positions and formation energies; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=4707-22; " /><meta name="citation_reference" content="citation_title=A compiled and systematic reference map of nucleosome positions across the Saccharomyces cerevisiae genome; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R109" /><meta name="citation_reference" content="citation_title=A barrier nucleosome model for statistical positioning of nucleosomes throughout the yeast genome; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1073-83; " /><meta name="citation_reference" content="citation_title=Intrinsic histone–DNA interactions are not the major determinant of nucleosome positions in vivo; Nat. Struct. Mol. Biol.;  citation_year=2009;  citation_volume=16;  citation_pages=847-52; " /><meta name="citation_reference" content="citation_title=Evidence against a genomic code for nucleosome positioning; Nat. Struct. Mol. Biol.;  citation_year=2010;  citation_volume=17;  citation_pages=920-3; " /><meta name="citation_reference" content="citation_title=Nucleosome sequence preferences influence in vivo nucleosome organization; Nat. Struct. Mol. Biol.;  citation_year=2010;  citation_volume=17;  citation_pages=918-20; " /><meta name="citation_reference" content="citation_title=Contribution of histone sequence preferences to nucleosome organization: proposed definitions and methodology; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=140" /><meta name="citation_reference" content="citation_title=Chromatin remodelling at promoters suppresses antisense transcription; Nature;  citation_year=2007;  citation_volume=450;  citation_pages=1031-5; " /><meta name="citation_reference" content="citation_title=Sequence-dependent DNA torsional rigidity: a tetranucleotide code; Biophys. Chem.;  citation_year=2004;  citation_volume=112;  citation_pages=77-88; " /><meta name="citation_reference" content="citation_title=Sequence-dependent DNA helical rise and nucleosome stability; BMC Mol. Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=105" /><meta name="citation_reference" content="citation_title=The structure of DNA in the nucleosome core; Nature;  citation_year=2003;  citation_volume=423;  citation_pages=145-50; " /><meta name="citation_reference" content="citation_title=Dynamic regulation of nucleosome positioning in the human genome; Cell;  citation_year=2008;  citation_volume=132;  citation_pages=887-98; " /><meta name="citation_reference" content="citation_title=Analysis of nucleosome positioning determined by DNA helix curvature in the human genome; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=72-83; " /><meta name="citation_reference" content="citation_title=DNA conformation and dynamics; Radiat, Environ. Biophys.;  citation_year=1999;  citation_volume=38;  citation_pages=23-9; " /><meta name="citation_reference" content="citation_title=Elastic origin of chiral selection in DNA wrapping; Phys. Rev. E;  citation_year=2008;  citation_volume=77;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Nucleosome organization in the Drosophila genome; Nature;  citation_year=2008;  citation_volume=453;  citation_pages=358-62; " /><meta name="citation_reference" content="citation_title=Nucleosome landscape and control of transcription in the human malaria parasite; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=228-38; " /><meta name="citation_reference" content="citation_title=G + C content dominates intrinsic nucleosome occupancy; BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=442" /><meta name="citation_reference" content="citation_title=High nucleosome occupancy is encoded at human regulatory sequences; PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e9129" /><meta name="citation_reference" content="citation_title=Sequence signature of nucleosome positioning in Caenorhabditis elegans; Genomics Proteomics Bioinformatics;  citation_year=2010;  citation_volume=8;  citation_pages=92-102; " /><meta name="citation_reference" content="citation_title=Determinants of nucleosome organization in primary human cells; Nature;  citation_year=2011;  citation_volume=474;  citation_pages=516-20; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/1/81/1314394/dsr043.pdf" /><meta name="description" content="Abstract.  Nucleosomes are the basic structural units of eukaryotic chromatin and play a key role in the regulation of gene expression. Nucleosome formation dep" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/1/81/486167" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/1/81/486167" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsr043';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./486167" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF10M8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBlZG9uZSwgRnJhbmNlc2NvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2d5IGFuZCBCaW90ZWNobm9sb2d5LCDigJhTYXBpZW56YeKAmSBVbml2ZXJzaXR5LCBQLmxlIEEuIE1vcm8gNSwgMDAxODUgUm9tZSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYW50b25pLCBEYW5pZWxlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmFyY2Vsb25hIEluc3RpdHV0ZSBmb3IgUmVzZWFyY2ggaW4gQmlvbWVkaWNpbmUgKElSQiksIEJhcmNlbG9uYSBTY2llbmNlIFBhcmssIEMvU2FtaXRpZXIgMS01LCAwODAxNSBCYXJjZWxvbmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSXN0aXR1dG8gZGkgQW5hbGlzaSBkZWkgU2lzdGVtaSBlIEluZm9ybWF0aWNhIEFudG9uaW8gUnViZXJ0aSwgQ29uc2lnbGlvIE5hemlvbmFsZSBkZWxsZSBSaWNlcmNoZSwgVmlhbGUgTWFuem9uaSAzMCwgMDAxODUgUm9tZSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlByZWZlcmVudGlhbCBOdWNsZW9zb21lIE9jY3VwYW5jeSBhdCBIaWdoIFZhbHVlcyBvZiBETkEgSGVsaWNhbCBSaXNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgb2NjdXJyZW5jZSBvZiBudWNsZW9zb21lIHBoYXNpbmcgaW4gY2hyb21hdGluOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ODI5LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgb2YgRE5BIGluIGNocm9tYXRpbjsgRXVyLiBKLiBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGl0Y2ggb2YgY2hyb21hdGluIEROQSBpcyByZWZsZWN0ZWQgaW4gaXRzIG51Y2xlb3RpZGUgc2VxdWVuY2U7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz0zODE2LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW90aWRlIHNlcXVlbmNlLWRpcmVjdGVkIG1hcHBpbmcgb2YgdGhlIG51Y2xlb3NvbWU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM4MzMtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBbKEcvQykzTk5dbiBtb3RpZjogYSBjb21tb24gRE5BIHJlcGVhdCB0aGF0IGV4Y2x1ZGVzIG51Y2xlb3NvbWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9ODg2My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgRE5BIHNlcXVlbmNlIHJ1bGVzIGZvciBoaWdoIGFmZmluaXR5IGJpbmRpbmcgdG8gaHlzdG9uZSBvY3RhbWVyIGFuZCBzZXF1ZW5jZS1kaXJlY3RlZCBudWNsZW9zb21lIHBvc2l0aW9uaW5nOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI3NjsgIGNpdGF0aW9uX3BhZ2VzPTE5LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW9zb21hbCBsb2NhdGlvbnMgb2YgZG9taW5hbnQgRE5BIHNlcXVlbmNlIG1vdGlmcyBmb3IgaGlzdG9uZeKAk0ROQSBpbnRlcmFjdGlvbnMgYW5kIG51Y2xlb3NvbWUgcG9zaXRpb25pbmc7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzM4OyAgY2l0YXRpb25fcGFnZXM9Njk1LTcwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBleHByZXNzaW9uIGRpdmVyZ2VuY2UgaW4geWVhc3QgaXMgY291cGxlZCB0byBldm9sdXRpb24gb2YgRE5BLWVuY29kZWQgbnVjbGVvc29tZSBvcmdhbml6YXRpb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NDM4LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9taWMgY29kZSBmb3IgbnVjbGVvc29tZSBwb3NpdGlvbmluZzsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0MjsgIGNpdGF0aW9uX3BhZ2VzPTc3Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRE5BLWVuY29kZWQgbnVjbGVvc29tZSBvcmdhbml6YXRpb24gb2YgYSBldWthcnlvdGljIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ1ODsgIGNpdGF0aW9uX3BhZ2VzPTM2Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW9zb21lIHBvc2l0aW9uaW5nIHNpZ25hbHMgaW4gZ2Vub21pYyBETkE7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTE3MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgcGh5c2ljYWwgcHJvcGVydGllcyBkZXRlcm1pbmUgbnVjbGVvc29tZSBvY2N1cGFuY3kgZnJvbSB5ZWFzdCB0byBmbHk7IE51Y2xlaWMgQWNpZCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9Mzc0Ni01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgRE5BIG1lY2hhbmljcyB0byBwcmVkaWN0IGluIHZpdHJvIG51Y2xlb3NvbWUgcG9zaXRpb25zIGFuZCBmb3JtYXRpb24gZW5lcmdpZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTQ3MDctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGlsZWQgYW5kIHN5c3RlbWF0aWMgcmVmZXJlbmNlIG1hcCBvZiBudWNsZW9zb21lIHBvc2l0aW9ucyBhY3Jvc3MgdGhlIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBnZW5vbWU7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIxMDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGJhcnJpZXIgbnVjbGVvc29tZSBtb2RlbCBmb3Igc3RhdGlzdGljYWwgcG9zaXRpb25pbmcgb2YgbnVjbGVvc29tZXMgdGhyb3VnaG91dCB0aGUgeWVhc3QgZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEwNzMtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJpbnNpYyBoaXN0b25l4oCTRE5BIGludGVyYWN0aW9ucyBhcmUgbm90IHRoZSBtYWpvciBkZXRlcm1pbmFudCBvZiBudWNsZW9zb21lIHBvc2l0aW9ucyBpbiB2aXZvOyBOYXQuIFN0cnVjdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTg0Ny01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgYWdhaW5zdCBhIGdlbm9taWMgY29kZSBmb3IgbnVjbGVvc29tZSBwb3NpdGlvbmluZzsgTmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz05MjAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvc29tZSBzZXF1ZW5jZSBwcmVmZXJlbmNlcyBpbmZsdWVuY2UgaW4gdml2byBudWNsZW9zb21lIG9yZ2FuaXphdGlvbjsgTmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz05MTgtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyaWJ1dGlvbiBvZiBoaXN0b25lIHNlcXVlbmNlIHByZWZlcmVuY2VzIHRvIG51Y2xlb3NvbWUgb3JnYW5pemF0aW9uOiBwcm9wb3NlZCBkZWZpbml0aW9ucyBhbmQgbWV0aG9kb2xvZ3k7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tYXRpbiByZW1vZGVsbGluZyBhdCBwcm9tb3RlcnMgc3VwcHJlc3NlcyBhbnRpc2Vuc2UgdHJhbnNjcmlwdGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ1MDsgIGNpdGF0aW9uX3BhZ2VzPTEwMzEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UtZGVwZW5kZW50IEROQSB0b3JzaW9uYWwgcmlnaWRpdHk6IGEgdGV0cmFudWNsZW90aWRlIGNvZGU7IEJpb3BoeXMuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1kZXBlbmRlbnQgRE5BIGhlbGljYWwgcmlzZSBhbmQgbnVjbGVvc29tZSBzdGFiaWxpdHk7IEJNQyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN0cnVjdHVyZSBvZiBETkEgaW4gdGhlIG51Y2xlb3NvbWUgY29yZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQyMzsgIGNpdGF0aW9uX3BhZ2VzPTE0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyByZWd1bGF0aW9uIG9mIG51Y2xlb3NvbWUgcG9zaXRpb25pbmcgaW4gdGhlIGh1bWFuIGdlbm9tZTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz04ODctOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIG51Y2xlb3NvbWUgcG9zaXRpb25pbmcgZGV0ZXJtaW5lZCBieSBETkEgaGVsaXggY3VydmF0dXJlIGluIHRoZSBodW1hbiBnZW5vbWU7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTcyLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgY29uZm9ybWF0aW9uIGFuZCBkeW5hbWljczsgUmFkaWF0LCBFbnZpcm9uLiBCaW9waHlzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTIzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVsYXN0aWMgb3JpZ2luIG9mIGNoaXJhbCBzZWxlY3Rpb24gaW4gRE5BIHdyYXBwaW5nOyBQaHlzLiBSZXYuIEU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz0xLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW9zb21lIG9yZ2FuaXphdGlvbiBpbiB0aGUgRHJvc29waGlsYSBnZW5vbWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTM7ICBjaXRhdGlvbl9wYWdlcz0zNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3NvbWUgbGFuZHNjYXBlIGFuZCBjb250cm9sIG9mIHRyYW5zY3JpcHRpb24gaW4gdGhlIGh1bWFuIG1hbGFyaWEgcGFyYXNpdGU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HICsgQyBjb250ZW50IGRvbWluYXRlcyBpbnRyaW5zaWMgbnVjbGVvc29tZSBvY2N1cGFuY3k7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggbnVjbGVvc29tZSBvY2N1cGFuY3kgaXMgZW5jb2RlZCBhdCBodW1hbiByZWd1bGF0b3J5IHNlcXVlbmNlczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWU5MTI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2Ugc2lnbmF0dXJlIG9mIG51Y2xlb3NvbWUgcG9zaXRpb25pbmcgaW4gQ2Flbm9yaGFiZGl0aXMgZWxlZ2FuczsgR2Vub21pY3MgUHJvdGVvbWljcyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTItMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlcm1pbmFudHMgb2YgbnVjbGVvc29tZSBvcmdhbml6YXRpb24gaW4gcHJpbWFyeSBodW1hbiBjZWxsczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ3NDsgIGNpdGF0aW9uX3BhZ2VzPTUxNi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzEvODEvMTMxNDM5NC9kc3IwNDMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE51Y2xlb3NvbWVzIGFyZSB0aGUgYmFzaWMgc3RydWN0dXJhbCB1bml0cyBvZiBldWthcnlvdGljIGNocm9tYXRpbiBhbmQgcGxheSBhIGtleSByb2xlIGluIHRoZSByZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbi4gTnVjbGVvc29tZSBmb3JtYXRpb24gZGVwIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODYxNjcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBBbGwgRmlndXJlc2RkpbpzSki+RO3JoeXTRx0WHg8hl3A=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f81%2f486167" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f81%2f486167" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=1509238795&amp;Signature=cqTLIIDsHRwJT9pOG1rgKbaiEeu~LgMTikVs9XLRoRzrrgK73Ldlzce9-dWB6ZLeBsVIG1-P4EvQ441k18HhaWTyPKW0OUpRlAeXi3duhA~NBpgvYpdn23suRvTp3Ilc-UUmdgSbbK90JfG2XZKBmAWQFgr1nn4T56d3tkxUOqEV37eLbGUYmKRQR3pqqV0dgCBdYhme2ur7F5T8udZrHLiT2ISBGEHSkhfQ7WZoJBO2Syk7BBJ3kRt~zd8WEGq9FNx662tzjooG6xwv6pf4lJHhXrITknJE-ly5cUBOUfs7FhebbZj0v6gmmYASisfihfBHKoj6YEoumBGD8HuuiQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/1">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7400584">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400584">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7400586">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400586">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7400589">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400589">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7400598">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400598">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7400619">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400619">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7400622">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400622">Authors’ contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7400624">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7400624">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/1/67/485988">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/1/91/486252">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Preferential Nucleosome Occupancy at High Values of DNA Helical Rise
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Francesco Pedone</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Francesco Pedone</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biology and Biotechnology</div>, <div class="insititution">‘Sapienza’ University</div>, <div class="addr-line">P.le A. Moro 5, 00185 Rome</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Francesco+Pedone">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pedone F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Pedone author:F.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Daniele Santoni</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daniele Santoni</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Barcelona Institute for Research in Biomedicine (IRB)</div>, <div class="addr-line">Barcelona Science Park, C/Samitier 1-5, 08015 Barcelona</div>, <div class="country">Spain</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Istituto di Analisi dei Sistemi e Informatica Antonio Ruberti, Consiglio Nazionale delle Ricerche</div>, <div class="addr-line">Viale Manzoni 30, 00185 Rome</div>, <div class="country">Italy</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. E-mail: <a href="mailto:francesco.pedone@uniroma1.it" target="_blank">francesco.pedone@uniroma1.it</a> (F. Pedone); <a href="mailto:santoni@caspur.it" target="_blank">santoni@caspur.it</a> (D. Santoni).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Daniele+Santoni">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Santoni D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Santoni author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 81–90, <a href='https://doi.org/10.1093/dnares/dsr043'>https://doi.org/10.1093/dnares/dsr043</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 January 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 July 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 November 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="486167" href="/dnaresearch/article-pdf/19/1/81/1314394/dsr043.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Francesco Pedone, Daniele Santoni;  Preferential Nucleosome Occupancy at High Values of DNA Helical Rise, <em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 81–90, <a href='https://doi.org/10.1093/dnares/dsr043'>https://doi.org/10.1093/dnares/dsr043</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/486167?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/486167?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/486167?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/486167?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/486167?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/486167?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/486167?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Preferential%20Nucleosome%20Occupancy%20at%20High%20Values%20of%20DNA%20Helical%20Rise&amp;publicationDate=2012-01-09&amp;volumeNum=19&amp;issueNum=1&amp;author=Pedone%2C%20Francesco%3B%20Santoni%2C%20Daniele&amp;startPage=81&amp;endPage=90&amp;contentId=10.1093%2Fdnares%2Fdsr043&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f81%2f486167" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7400584 id="7400584" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Nucleosomes are the basic structural units of eukaryotic chromatin and play a key role in the regulation of gene expression. Nucleosome formation depends on several factors, including properties of the sequence itself, but also physical constraints and epigenetic factors such as chromatin-remodelling enzymes. In this view, a sequence-dependent approach is able to capture a general tendency of a region to bind a histone octamer. A reference data set of positioned nucleosomes of <em>Saccharomyces cerevisiae</em> was used to study the role of DNA helical rise in histone–DNA interaction. Genomic sequences were transformed into arrays of helical rise values by a tetranucleotide code and then turned into profiles of mean helical rise values. These profiles resemble maps of nucleosome occupancy, suggesting that intrinsic histone–DNA interactions are linked to helical rise. The obtained results show that preferential nucleosome occupancy occurs where the mean helical rise reaches its largest values. Mean helical rise profiles obtained by using maps of positioned nucleosomes of the <em>Drosophila melanogaster</em> and <em>Plasmodium falciparum</em> genomes, as well as <em>Homo sapiens</em> chromosome 20 confirm that nucleosomes are mainly located where the mean helical rise reaches its largest values.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="nucleosome">nucleosome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="tetranucleotide">tetranucleotide</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;helical rise&quot;">helical rise</a></div>                            <h2 scrollto-destination=7400586 id="7400586" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The observation that specific DNA sequences favour the formation of nucleosomes<sup><a reveal-id="DSR043C1  DSR043C6" data-open="DSR043C1  DSR043C6" class="link link-ref link-reveal xref-bibr">1–6</a></sup> raises the possibility that sequence plays a significant role in organizing nucleosomal arrays.<sup><a reveal-id="DSR043C7" data-open="DSR043C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR043C8" data-open="DSR043C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Recently, computational models for sequence-based prediction of nucleosome positioning in <em>Saccharomyces cerevisiae</em> were proposed.<sup><a reveal-id="DSR043C9" data-open="DSR043C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR043C10" data-open="DSR043C10" class="link link-ref link-reveal xref-bibr">10</a></sup> These models are generative and used dinucleotide and pentanucleotide frequencies collected from a training set of aligned nucleosome-bound sequence to predict histone–DNA interactions. Peckham <em>et al</em>.<sup><a reveal-id="DSR043C11" data-open="DSR043C11" class="link link-ref link-reveal xref-bibr">11</a></sup> proposed a complementary computational model that is discriminative, rather than generative, and that focuses on only sequences that show the strongest signals of nucleosome occupancy or vacancy in a microarray-based assay of <em>S. cerevisiae</em>. Miele <em>et al</em>.<sup><a reveal-id="DSR043C12" data-open="DSR043C12" class="link link-ref link-reveal xref-bibr">12</a></sup> used sequence-dependent DNA flexibility and an intrinsic curvature to predict the nucleosome occupancy along the genomes of <em>S. cerevisiae</em> and <em>Drosophila melanogaster</em>. Morozov <em>et al</em>.<sup><a reveal-id="DSR043C13" data-open="DSR043C13" class="link link-ref link-reveal xref-bibr">13</a></sup> used bending data to design both strong and weak histone-binding sequences, and measured the corresponding free energies of nucleosome formation in order to provide a physical explanation for the intrinsic sequence dependence of histone–DNA interactions. A reference map of nucleosome positions in <em>S. cerevisiae</em> has recently been compiled through the analysis of several genome-wide data sets produced in various laboratories and reported at <a class="link link-uri" href="http://atlas.bx.psu.edu/">http://atlas.bx.psu.edu/</a>, where it has been made available as a Microsoft Excel worksheet named ‘additional data file 1’.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In yeast, non-random positioning of nucleosomes has been established mainly at promoters, where a nucleosome-free region (NFR) is followed by a highly positioned nucleosome at the +1 position. The +1 nucleosome forms a barrier against which subsequent nucleosomes with a decreasing stability are located.<sup><a reveal-id="DSR043C15" data-open="DSR043C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The role of intrinsic DNA sequences either in <em>in vitro</em> or <em>in vivo</em> nucleosome positioning has recently been at the centre of a debate<sup><a reveal-id="DSR043C16  DSR043C19" data-open="DSR043C16  DSR043C19" class="link link-ref link-reveal xref-bibr">16–19</a></sup> about the existence of a genomic code for nucleosome positioning. One central topic of the debate concerns the correct use of the terms ‘positioning’ and ‘occupancy’, which are both connected to the uncertainty on the location of the histone dyad axis on the DNA sequence. In this paper, we adopt the following simple nomenclature: ‘positioning’ locates the dyad axis with an uncertainty of a few base pairs, whereas ‘occupancy’ allows reaching a few tens of  base pairs, and this error in location is called fuzziness. It is generally accepted that the DNA sequence influences, to some extent, <em>in vivo</em> nucleosome positioning, which is regulated by many other additional epigenetic factors. In yeast, the role of chromatin-remodelling enzymes, such as isw2, has extensively been studied and a map of genome-wide nucleosome positions in an isw2 deletion strain has been reported in the ‘additional data file 1’.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The study of isw2 has implications for the predictions of nucleosome positions on the basis of the DNA sequence alone and it has been hypothesized that the loss of isw2 would allow nucleosomes to adopt their inherent positioning preference.<sup><a reveal-id="DSR043C20" data-open="DSR043C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In previous papers,<sup><a reveal-id="DSR043C21" data-open="DSR043C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> we reported a correlation between the stability of nucleosomes and the presence of a symmetrical distribution of distances from the nucleosomal dyad axis at specific points. These points represent the locations of phosphates along the DNA minor groove, where histone–DNA electrostatic bonds are formed. These symmetric distances were calculated by means of a tetranucleotide helical rise code<sup><a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> applied to known nucleosome-positioning sequences. In that paper, the presence of symmetric distances was mainly correlated to the probability of finding nucleosomes, and on the other hand, we succeeded in observing the absence of symmetric distances in NFRs, which are usually found <em>in vivo</em> at the transcription start sites (TSSs) of human promoters. We recently extended our analysis to yeast promoters and observed that helical rise profiles of NFR sequences exhibit lower mean values than those obtained for adjacent sequences where nucleosomes are usually found. In addition, we argued that the generation of mean helical rise curves and the presence of a symmetric shape in their profile could be related to the presence of nucleosomes. These preliminary results prompted us to analyse the whole yeast genome in a systematic way, so that we generated profiles of helical rise values averaged by various smoothing windows (see methods) and searched for possible correlations between helical rise profiles and known nucleosome positions. We hypothesized that the histone protein begins to interact with DNA exploring at least 26 bp, a length where six principal interactions between the DNA phosphates and the basic amino acids of H3 and H4 histones are formed.<sup><a reveal-id="DSR043C23" data-open="DSR043C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Once the optimal site of interaction has been found, DNA continues to wrap around the histone protein and 56 bp are involved in the formation of six additional interactions. When 90 bp are engaged in the complex, about a complete turn of DNA is wrapped, while the final form of the nucleosome is achieved when ∼150 bp make one and three quarters of a turn of DNA wrap around the histone protein. We obtained then our helical rise profiles by a set of smoothing windows of 26, 56, 90 and 150 bp, respectively, in order to account for the start and the end of histone–DNA interaction. We also compared our helical rise profiles with those obtained by the dinucleotide frequency method developed by Segal <em>et al</em>.<sup><a reveal-id="DSR043C9" data-open="DSR043C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR043C10" data-open="DSR043C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and the results are available at <a class="link link-uri" href="http://genie.weizmann.ac.il/software/nucleo_prediction.html">http://genie.weizmann.ac.il/software/nucleo_prediction.html</a>.</p>                            <h2 scrollto-destination=7400589 id="7400589" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7400590 id="7400590" class="section-title" data-legacy-id=s2a>2.1. Nucleosome positions</h3>
<p>Data referring to nucleosome positioning in <em>S. cerevisiae</em> and <em>D. melanogaster</em> were collected from reference maps of nucleosome positions reported at <a class="link link-uri" href="http://atlas.bx.psu.edu/">http://atlas.bx.psu.edu/</a>. Nucleosome positions of <em>Plasmodium falciparum</em> were downloaded from plasmoDB (<a class="link link-uri" href="http://www.plasmoDB.org">www.plasmoDB.org</a>) and those of human chromosome 20, derived from the experimental data of Schones <em>et al</em>.,<sup><a reveal-id="DSR043C24" data-open="DSR043C24" class="link link-ref link-reveal xref-bibr">24</a></sup> were downloaded from <a class="link link-uri" href="http://www.gri.seu.edu.cn/icons">http://www.gri.seu.edu.cn/icons</a>.<sup><a reveal-id="DSR043C25" data-open="DSR043C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>Nucleosome positions derived by computational analysis of human chromosome 20 were derived from <a class="link link-uri" href="http://www.gri.seu.edu.cn/icons">http://www.gri.seu.edu.cn/icons</a>.</p>                            <h3 scrollto-destination=7400594 id="7400594" class="section-title" data-legacy-id=s2b>2.2. DNA sequences</h3>
<p>The whole nucleotide sequences of <em>S. cerevisiae</em>, <em>D. melanogaster</em> and <em>P. falciparum</em> were downloaded from the NCBI database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a>). The DNA sequence of human chromosome 20 was downloaded from the UCSC genome database, built hg 18 (<a class="link link-uri" href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>). DNA sequences of 1-kb long (symmetric with respect to the potential dyad axis) were extracted (hg 18 built).</p>                            <h3 scrollto-destination=7400596 id="7400596" class="section-title" data-legacy-id=s2c>2.3. Computational analysis</h3>
<div class="block-child-p">We used a tetranucleotide helical rise code<sup><a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> to ‘measure’ sequence length in terms of helical rise. Each 1-kb sequence was analysed obtaining a local helical rise profile according to the tetranucleotide code. Four different window sizes of 26, 56, 90 and 150 bp, respectively, were used to get smoothed helical rise profiles, in the following way:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr043ueq1.gif?Expires=1509238795&Signature=b7-2zkH6wknwl-Ew6WrxvPlzSrr5djXzzPNu9sjNN-rT8OEJb2vjQJ697dibrQPbN-6Pw~~pWMjjXzbiDcsedDcP0n2zM9549Am5BiSsrdeTcd~yz8RLqmR~4d4lfR1TCoTj7-jqPQZUbTuq0coivZV-hlgrnA3vNHl6WE-hrMQ~P6DjbllpJ9wElpDugSgh09SCy1gv5vgeHSnG~eDFazECPve3a0z6bwopjrtzfA7Wr5fuAy2e0~oNl95cqeeCOVjVKn-DsSTXr7mYKeJ3KKvIqu5aEWnYFOyuVSgQ4eRneK5RuwI4uU~2f6yNu58GdkhsAuqMextwWMu9enznTw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" id="dsr043ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr043ueq1.gif?Expires=1509238795&Signature=gdtgrfVb8Bn4LHemAPl8TggfUvFSfn-Gbm6ec4heTq7If5yYVa7Wg9l6fEuWsfFHAMsbvM9IURvJKPrWQ0saJhxgCl8b-YvBKbHwayJFjaR2c2p-EZbdWzGwLpSrbImFXNqI0ddNnddX8fdxi1YG6f821VVYWEgy2CPV9bw-bZWfgWl3CJ9O-EFx73T2-MsvkD~KfPCRb8YX0LBkPTUaVrKBC6XZ4ROQNS8F-rHxkM3TO~G7bk278C43qkIM2R6-HvJ~Ty2--ED6I8B2Zm4GechkxX-YbvLnGnZbsvnjv4WOPsE-p6WqM8iYA-qHi19PI-M9a7QDju66TrEhwPac7g__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="formula" path-from-xml="dsr043ueq1.gif" /></a></div></div></div> where <em>p<sub>j</sub></em> is the plain helical rise profile.</div>                            <h2 scrollto-destination=7400598 id="7400598" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7400599 id="7400599" class="section-title" data-legacy-id=s3a>3.1. Symmetrical helical rise profiles</h3>
<p>The DNA sequences were downloaded from the NCBI database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a>) and transformed, by our tetranucleotide code, into an array of consecutive helical rise values. In yeast, this primary array was averaged by means of four different smoothing windows and, for each window, 61 110 mapped dyad nucleosome positions, belonging to the reference set included in the additional data file 1,<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> were used to obtain 61 110 profiles of 1-kb-long mean helical rise, each containing 500 bp located upstream of the central dyad axis and 500 downstream. For each window, the average of the 61 110 profiles was performed for the reference data set and reported in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A as a solid black line. Same approach was used for any of the six other data sets contained in the additional data file 1,<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> returning helical rise profiles (data not shown) closely similar to those generated for the reference set. An analogous procedure was executed for the 54 907 nucleosome positions of an isw2 deletion strain of <em>S. cerevisiae</em><sup><a reveal-id="DSR043C20" data-open="DSR043C20" class="link link-ref link-reveal xref-bibr">20</a></sup> reported in the same additional data file 1.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p>                        <a id="7400601" scrollto-destination="7400601"></a>
<div id="DSR043F1" data-id="DSR043F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04301.jpeg?Expires=1509238795&Signature=QoPVu2KzBZGJnqFzXwiCFyAX0AbetrCjhi51MLlZwngzejFg0T~fIMwGpmyxc7yQ4TIA37zY3LL1ove4SLAuw43oMyPF1rSfRBck0Y1J-XSQeJEJZQoKpSlFq4DlJ7eJc2ootpmT6XTaR-bAU-OvhEO4w0E8apPjkbWMD54kQA-0KjCnJtc2Rjd6eBD2Q5flbpJNAX0rNZQ~VUlRmdaeZ7YJY1ggE80FvMXvyhHfS9MyUuqgGXrCGEG0-I1yp-jZL-ys1XVYhboEkFAcXIBk9ZPVWywzS8u6NbMU5KvpoWS0pw46A0qiMS~rGqxL8wylQYwh6B15triulZcEsn7cGw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise profiles of nucleosome positions. (A) Average helical rise profiles of mapped nucleosome positions in yeast (average of 61 110 points, black solid line) and in an isw2 deletion strain (average of 54 907 points, pink solid line) for four different smoothing windows of 26, 56, 90 and 150 bp. (B) Average helical rise profiles of mapped nucleosome positions in chromosome I of yeast (average of 1183 points, black solid line), in an isw2 deletion strain (average of 1069 points, pink solid line) and in scrambled nucleosome positions in chromosome I of yeast (average of 1183 points, grey solid line) with a smoothing window of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper." path-from-xml="dsr04301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400601/dsr04301.jpeg" path-from-xml="dsr04301.jpeg" target="_blank">View large</a><a section="7400601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04301.jpeg?Expires=1509238795&Signature=Rz9BSLCCxZRFlVfqWfOP-QW-zPYHzJQFb1MG4bJ8GS60Sk-Pj7GNGNSywYLPWifie6W-Zv-8MlVXcd1E28MAzKqxUakCvj4glEpCxS9Lod~5Y-~nJ2ate1rWckYcwC5v1DlH42BxmHP0D5pAaGm-6v7xldqb3fKRZEoY0RxUjTuz2PRAG2IyhuCrYpxBlniVjsRMJOlBLXvr9CcJwBA17-sOCWUIerioptGKl7PZIcecQP4pZsj5FDWBe0cWS2wtEBLNPsoPn7PhOZHNObetLo5lbpXv7~Q2QTDp9VE5cqmliaAu8hJREKWlCc7sGXX-X8x2CXfEXaO3B0oo11a-Cg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400601&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise profiles of nucleosome positions. (A) Average helical rise profiles of mapped nucleosome positions in yeast (average of 61 110 points, black solid line) and in an isw2 deletion strain (average of 54 907 points, pink solid line) for four different smoothing windows of 26, 56, 90 and 150 bp. (B) Average helical rise profiles of mapped nucleosome positions in chromosome I of yeast (average of 1183 points, black solid line), in an isw2 deletion strain (average of 1069 points, pink solid line) and in scrambled nucleosome positions in chromosome I of yeast (average of 1183 points, grey solid line) with a smoothing window of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper.</p></div></div><div content-id="DSR043F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04301.jpeg?Expires=1509238795&Signature=QoPVu2KzBZGJnqFzXwiCFyAX0AbetrCjhi51MLlZwngzejFg0T~fIMwGpmyxc7yQ4TIA37zY3LL1ove4SLAuw43oMyPF1rSfRBck0Y1J-XSQeJEJZQoKpSlFq4DlJ7eJc2ootpmT6XTaR-bAU-OvhEO4w0E8apPjkbWMD54kQA-0KjCnJtc2Rjd6eBD2Q5flbpJNAX0rNZQ~VUlRmdaeZ7YJY1ggE80FvMXvyhHfS9MyUuqgGXrCGEG0-I1yp-jZL-ys1XVYhboEkFAcXIBk9ZPVWywzS8u6NbMU5KvpoWS0pw46A0qiMS~rGqxL8wylQYwh6B15triulZcEsn7cGw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise profiles of nucleosome positions. (A) Average helical rise profiles of mapped nucleosome positions in yeast (average of 61 110 points, black solid line) and in an isw2 deletion strain (average of 54 907 points, pink solid line) for four different smoothing windows of 26, 56, 90 and 150 bp. (B) Average helical rise profiles of mapped nucleosome positions in chromosome I of yeast (average of 1183 points, black solid line), in an isw2 deletion strain (average of 1069 points, pink solid line) and in scrambled nucleosome positions in chromosome I of yeast (average of 1183 points, grey solid line) with a smoothing window of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper." path-from-xml="dsr04301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400601/dsr04301.jpeg" path-from-xml="dsr04301.jpeg" target="_blank">View large</a><a section="7400601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04301.jpeg?Expires=1509238795&Signature=Rz9BSLCCxZRFlVfqWfOP-QW-zPYHzJQFb1MG4bJ8GS60Sk-Pj7GNGNSywYLPWifie6W-Zv-8MlVXcd1E28MAzKqxUakCvj4glEpCxS9Lod~5Y-~nJ2ate1rWckYcwC5v1DlH42BxmHP0D5pAaGm-6v7xldqb3fKRZEoY0RxUjTuz2PRAG2IyhuCrYpxBlniVjsRMJOlBLXvr9CcJwBA17-sOCWUIerioptGKl7PZIcecQP4pZsj5FDWBe0cWS2wtEBLNPsoPn7PhOZHNObetLo5lbpXv7~Q2QTDp9VE5cqmliaAu8hJREKWlCc7sGXX-X8x2CXfEXaO3B0oo11a-Cg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400601&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise profiles of nucleosome positions. (A) Average helical rise profiles of mapped nucleosome positions in yeast (average of 61 110 points, black solid line) and in an isw2 deletion strain (average of 54 907 points, pink solid line) for four different smoothing windows of 26, 56, 90 and 150 bp. (B) Average helical rise profiles of mapped nucleosome positions in chromosome I of yeast (average of 1183 points, black solid line), in an isw2 deletion strain (average of 1069 points, pink solid line) and in scrambled nucleosome positions in chromosome I of yeast (average of 1183 points, grey solid line) with a smoothing window of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper.</p></div></div><p>The results obtained for the isw2 deletion strain are shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A as a solid pink line. In the same figure are shown, as markers, the positions of five nucleosomes that cover 147 bp each and are mutually separated by 18 bp of linker DNA as reported for yeast.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In each of the four panels both the reference and the isw2 curves are symmetric with respect to the dyad axis located at bp number 0. This behavior is more pronounced for the 90-bp long smoothing window, which represents a length equivalent to a full turn of DNA wrapped around the histone core. When applied to the DNA sequence alone, the helical rise method shows that yeast nucleosomes are positioned at higher helical rise values than their linkers. The symmetry of the central peak supports our early results<sup><a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> on the correlation between nucleosome stability and symmetrical distribution of helical rise values around the nucleosomal dyad axis. In the four panels of Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A, the mean helical rise values of each isw2 profile are systematically larger than those of the reference set. Moreover, the reference profiles corresponding to the 26- and the 56-bp-long smoothing windows, unlike their isw2 counterparts, are both characterized by a central peak clearly split into two adjacent maxima. These maxima are separated by ∼60 bp and their disappearance is therefore evident in the helical rise profiles obtained with smoothing windows of 90 and 150 bp reported in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A. An explanation for the differences in the rise value and peak shape between the isw2 and reference samples may be found considering the role of the isw2-remodelling enzymes in <em>S. cerevisiae</em>. It has been reported that isw2 enzymes’ reposition nucleosomes near promoter regions in order to suppress antisense transcription; the repositioning can range from a few bp to ±73 bp and in an isw2 deletion strain, nucleosomes tend to place themselves at their preferred targets, selected on the basis of the DNA sequence alone.<sup><a reveal-id="DSR043C20" data-open="DSR043C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Our algorithm, based on a sequence-dependent approach, correctly shows preferential sites when the effect of isw2 is silenced. The presence of the double peaks, shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> for the 26- and the 56-bp-long smoothing windows in the wild-type profiles, is due to the action of isw2, which repositions nucleosomes on the DNA sequence. The helical rise can actually render nucleosome occupancy; indeed, in the reference sample, isw2 remodellers are active and the mapped nucleosome positions are split away from the central position; on the other hand, due to the convergence of the isw2 nucleosome positions towards a single maximum value in the central peak, the isw2 average profiles are always higher than those corresponding to the reference set.</p><p>Shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>B are the helical rise profiles obtained for the single chromosome I of <em>S. cerevisiae</em>. The shape of the black and pink curves (reference and isw2 data set, respectively) is identical to that reported for the whole yeast genome shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A at the smoothing window of 56 bp. Also shown in Fig <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>B with a grey continuous line is a control curve obtained for the DNA sequence of chromosome I with dyad positions scrambled with those of chromosome II. The loss of symmetry and the random shape of the scrambled curve are evident.</p>                            <h3 scrollto-destination=7400604 id="7400604" class="section-title" data-legacy-id=s3b>3.2. Peak height and nucleosome occupancy</h3>
<p>Another characteristic of the profiles shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> is represented by the lower values of the mean helical rise for nucleosomes adjacent to the central one. As to this feature, a preliminary attempt was made to understand how the peak height of the helical rise profiles is correlated to nucleosome stability. We reprocessed the reference data set in the additional data file 1,<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> where nucleosome occupancy is expressed in the range from 1 to 7 arbitrary units, classifying nucleosome positions as: very stable (peak height &gt;4, comprising about 24 600 samples), stable (2&lt; peak height &lt;<span class="underline">4</span>, comprising about 29 800 samples) and weak (peak height &lt;2, comprising about 5600 samples). The results obtained for a smoothing window of 90 bp are reported in Fig. <a reveal-id="DSR043F2" data-open="DSR043F2" class="link link-reveal link-table xref-fig">2</a>, where the corresponding overall profile from Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A is also shown. Data displayed in Fig. <a reveal-id="DSR043F2" data-open="DSR043F2" class="link link-reveal link-table xref-fig">2</a> state a clear direct relationship between the helical rise peak height and nucleosome occupancy; therefore, our method can be considered to unambiguously assess nucleosome occupancy in a quantitative manner. Furthermore, once again the peaks neighbouring the central maximum exhibit a lower height, except for those marked as weak and represented by a blue solid line. A statistical test (Wilcoxon–Mann–Whitney) was performed between the distribution of helical rise at dyad positions and the distribution of the helical rise at random positions. A <em>P</em>-value smaller than E-16 assures us that the helical rise is not randomly distributed with respect to nucleosome maps. </p>                        <a id="7400606" scrollto-destination="7400606"></a>
<div id="DSR043F2" data-id="DSR043F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04302.jpeg?Expires=1509238795&Signature=SQcVQ9urrbj6E1p33cBkDgYaBQe3IIwQ9~Bcw2j3qAk9YsxkfEOg2xoUHH9722Yxa4u1HfMZ7Fi3G3UmDiQV0KhuYILgkyeYgud1-p6cJ20SciWSv~vVTrhmlepBrHfcabHG2EQrEm5Izx3eRJGF7s9W3CcLe0MmgMBYcrbxvehgLfXDJYnEGLF9rKXHz-Jb0W7YbEbJzbLgdKSXf6Crv4Tglr89MwMr~hoOfdbkusPp~PbhqFVRbOyEFjeOFT6CyTiCqyPOchgqakYDCDMw1gar0Oc-n1NPnYrS5KAXPRhsIs-SnMVdukjdKo8uX0QcSG4Pt1dQY6y~WRYCu~axdQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Peak height analysis of nucleosome positions. Average helical rise profiles of mapped nucleosome positions in yeast for a 90-bp smoothing window. Average of 24 600 points with a peak height &gt;4 (red solid line); of 29 800 points with a peak height &lt;4 and &gt;2 (green solid line); of 5600 point with a peak height &lt;2 (blue solid line); of 61 110 points previously shown in Fig. 1A with a 90-bp smoothing window (black solid line). Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper." path-from-xml="dsr04302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400606/dsr04302.jpeg" path-from-xml="dsr04302.jpeg" target="_blank">View large</a><a section="7400606" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04302.jpeg?Expires=1509238795&Signature=HNYOhlOgJ2vwhFkX4hECvjFxxx~BsyPG1eQclWzVDzy6SiWbg7V~FCIB9agoXMZgfJ8ZDeQMjVW87WNyPQZbGa4pC4944-SzpkxRhdxilAR-k5dFiVBgK6JMM7RAbBToAN7j3lr7G5mORkOGcykDQWD3Ktabcd8GwrNFGGWSmZ-5IqkqQ1aDLMfXigkqXaOEi~nsVCX8OZYh2RNb9dmfRaxAwhVvqVfgnnkV~CPTp8CVVB3hTTQNepxJUvcPgMUiKF3SSt1CSa0L59KPftkcbsxYw7FQMQVcqgQOQoopdm2Fb4fsa88ILp5R7M2HbjEq3L6q832Zztlosqc~QQlPKA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400606&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Peak height analysis of nucleosome positions. Average helical rise profiles of mapped nucleosome positions in yeast for a 90-bp smoothing window. Average of 24 600 points with a peak height &gt;4 (red solid line); of 29 800 points with a peak height &lt;4 and &gt;2 (green solid line); of 5600 point with a peak height &lt;2 (blue solid line); of 61 110 points previously shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A with a 90-bp smoothing window (black solid line). Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper.</p></div></div><div content-id="DSR043F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04302.jpeg?Expires=1509238795&Signature=SQcVQ9urrbj6E1p33cBkDgYaBQe3IIwQ9~Bcw2j3qAk9YsxkfEOg2xoUHH9722Yxa4u1HfMZ7Fi3G3UmDiQV0KhuYILgkyeYgud1-p6cJ20SciWSv~vVTrhmlepBrHfcabHG2EQrEm5Izx3eRJGF7s9W3CcLe0MmgMBYcrbxvehgLfXDJYnEGLF9rKXHz-Jb0W7YbEbJzbLgdKSXf6Crv4Tglr89MwMr~hoOfdbkusPp~PbhqFVRbOyEFjeOFT6CyTiCqyPOchgqakYDCDMw1gar0Oc-n1NPnYrS5KAXPRhsIs-SnMVdukjdKo8uX0QcSG4Pt1dQY6y~WRYCu~axdQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Peak height analysis of nucleosome positions. Average helical rise profiles of mapped nucleosome positions in yeast for a 90-bp smoothing window. Average of 24 600 points with a peak height &gt;4 (red solid line); of 29 800 points with a peak height &lt;4 and &gt;2 (green solid line); of 5600 point with a peak height &lt;2 (blue solid line); of 61 110 points previously shown in Fig. 1A with a 90-bp smoothing window (black solid line). Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper." path-from-xml="dsr04302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400606/dsr04302.jpeg" path-from-xml="dsr04302.jpeg" target="_blank">View large</a><a section="7400606" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04302.jpeg?Expires=1509238795&Signature=HNYOhlOgJ2vwhFkX4hECvjFxxx~BsyPG1eQclWzVDzy6SiWbg7V~FCIB9agoXMZgfJ8ZDeQMjVW87WNyPQZbGa4pC4944-SzpkxRhdxilAR-k5dFiVBgK6JMM7RAbBToAN7j3lr7G5mORkOGcykDQWD3Ktabcd8GwrNFGGWSmZ-5IqkqQ1aDLMfXigkqXaOEi~nsVCX8OZYh2RNb9dmfRaxAwhVvqVfgnnkV~CPTp8CVVB3hTTQNepxJUvcPgMUiKF3SSt1CSa0L59KPftkcbsxYw7FQMQVcqgQOQoopdm2Fb4fsa88ILp5R7M2HbjEq3L6q832Zztlosqc~QQlPKA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400606&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Peak height analysis of nucleosome positions. Average helical rise profiles of mapped nucleosome positions in yeast for a 90-bp smoothing window. Average of 24 600 points with a peak height &gt;4 (red solid line); of 29 800 points with a peak height &lt;4 and &gt;2 (green solid line); of 5600 point with a peak height &lt;2 (blue solid line); of 61 110 points previously shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A with a 90-bp smoothing window (black solid line). Black dots with (146-bp long) horizontal bars mark positions of mapped nucleosomes. Colour figures can be found in the online version of this paper.</p></div></div>                            <h3 scrollto-destination=7400607 id="7400607" class="section-title" data-legacy-id=s3c>3.3. Helical rise profiles of promoters</h3>
<p>The above-mentioned features should reflect the general organization of yeast nucleosomes covering transcribed genes. Figure <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a> presents the helical rise profiles of YAL053W, an RNA-polymerase II transcribed gene lying on chromosome 1 whose mapped nucleosome positions are reported in the additional data file 1.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In panel A, the helical rise profile of YAL053W smoothed at 90 bp, is shown as a solid black line. Nucleosome positions and peak heights are reported as horizontal bars on the bottom and the top of the figure, respectively. Near the TSS and the transcription termination site (TTS), marked by a green and a red dot, respectively, the rise values reach ∼3.05 Å, while in the regions where the most stable nucleosomes named +2, +3, +4 etc. are located, the rise values exceed 3.25 Å. The difference of ∼0.2 Å between the helical rise values of NFRs and those of stable nucleosomes may appear a modest amount, but it must be remembered that the main force involved is of electrostatic nature<sup><a reveal-id="DSR043C26" data-open="DSR043C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and follows Coulomb's law, which establishes that the attraction between two charges is proportional to the inverse square of the distance. In addition, the damping down of the attraction depends on the dielectric constant of the medium as well, and free DNA and histone proteins share a hydration shell with a value of dielectric constant equal to 80 due to water. When nucleosomes are formed, the hydration shell gets partially lost and the value of the dielectric constant, although not easy to calculate, is considerably reduced, with a consequent increase in coulombic attraction. The balance between the distance of charges and the stiffness of the hydration shell is likely to be involved in nucleosome formation and mobility. Moreover, it is a known fact that the DNA double helix, when wrapping around a histone in a counterclockwise sense, unwinds and gets stretched,<sup><a reveal-id="DSR043C27" data-open="DSR043C27" class="link link-ref link-reveal xref-bibr">27</a></sup> with a consequent variation in the distance between charged phosphate groups along the DNA. In our opinion, the observed preference for higher helical rise values in nucleosome formation is correlated to the lower energetic cost required to stretch DNA in order to form stable interactions between negative DNA phosphates and positive histone groups. In other words, DNA wrapped around a histone protein is constrained and the release of the constraint is better achieved when the mean helical rise is &gt;3.2 Å. In panel A of fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>A, the positions of isw2 nucleosomes are marked by blue horizontal bars. R emodeling of nucleosomes is manifest near NFRs, as well as in regions where the helical rise profile exhibits lower values. This feature is highlighted by the pink profile as well, that shows the occupancy level of YAL053W calculated according to the dinucleotide frequency method by Segal <em>et al</em>.<sup><a reveal-id="DSR043C9" data-open="DSR043C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The black and pink profiles share a few similarities either in the regions from position +2 to +7, where a high value of nucleosome occupancy and of helical rise are found, and in the regions surrounding NFRs like TSS and TTS where to low occupancy values high helical rise ones correspond. At positions +8 and +11 a different shape of the helical rise profiles can be observed, characterized by a dyad position at the minimum values of a ‘V’-shaped curve instead of the more usual maximum values of a ‘peak’-shaped curve. A logical inference is that the central peaks reported in Figs. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSR043F2" data-open="DSR043F2" class="link link-reveal link-table xref-fig">2</a> are averaged by the sum of both types of helical rise profiles, ‘peak’- and ‘V’-shaped, so that the average central peaks are always symmetrical but their maximum values are somehow lowered by the contribution from the ‘V’-shaped profile. It must be considered that the major contribution to the mean helical rise profile is due to the peak-shaped curves. </p>                        <a id="7400609" scrollto-destination="7400609"></a>
<div id="DSR043F3" data-id="DSR043F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04303.jpeg?Expires=1509238796&Signature=PFrRu9wJd4LeyGlXojEog2wgR1XxCK0O7mGttuo7LpSQd1qHlyosgMzrcgA9xw5lgeWsa1UXRSvN-lGMBL4kUtyzzMNYQSci-hbur2kVyaTfyBjCde2vgr~GF~3UoXoXQlFDQUKyQcfYXO8tQ7qJPKP0udVJw~c1nc7DwtahQliYmRME2-Vz6cZw~L~cGUFRf14o8LGUxxcjAEQnmGrY00EoEo7ML359aOw2fOQJQ9WQm24JRIP~jx8vTMF14T06AKDat-veYdMBLc~E~MuTYNEVGkgd6umbNjjMMXZfTE7KE4-VY66iqZxL1xvMgvdjhujUKgSEnunKbBebZUnGWA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise profiles of nucleosome positions at promoters. (A) Helical rise profile (black solid line) of the YAL053W promoter sequence in chromosome 1 of S. cerevisiae, obtained by a smoothing window of 90 bp. Nucleosome occupancy profile (pink solid line) obtained by the dinucleotide frequency method by Segal et al. Upper black dots with horizontal bars return peak heights of the corresponding nucleosomes. Lower black dots with horizontal bars return nucleosome numbering. Blue dots with horizontal bars return nucleosome repositioning in the isw2 deletion strain. Green and red dots represent TSS and TTS, respectively. (B) Helical rise profile (black solid line) of YAL053W smoothed at 90 bp as in (A). Average helical rise profiles (red solid lines) for nucleosomes sharing the same position number. Upper red dots with horizontal bars return mean peak heights of the corresponding nucleosomes expressed in arbitrary units as reported in the ‘additional data file 1’.14 Lower black dots with horizontal bars return nucleosome numbering. Green and red dots represent TSSs and TTSs, respectively. (C) Scale enlargement of the mean helical rise profiles for nucleosomes sharing the same position number reported in (B, black solid line). The mirror image of the same profiles (pink solid line). The average between peak heights of the black and pink profiles (blue diamonds). Black dots with horizontal bars return nucleosome numbering. Colour figures can be found in the online version of this paper." path-from-xml="dsr04303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400609/dsr04303.jpeg" path-from-xml="dsr04303.jpeg" target="_blank">View large</a><a section="7400609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04303.jpeg?Expires=1509238796&Signature=LBtFsKG1QDgUgZX2WQbSlbsSjN126oR2W1jvkxw-mWpFkwxstNitmFrDng~-86pvNuqMmf2Qn9NjyRNtYwKTuv9lIF2o-YDHI48pAhwktZFYzEz4qao6Y6YUCfaH8dEUGp2raMImVPFC6a91JMVaMu79xWtZNWqbNPmeTZILlp7RMrNi1KW42Hfr~PtffIRchgqO0cK54n3sQNUchXfKLDIB0LblcgQEyqFeoOyUYqcXPi4XW9f0Ny7W4jH-7BxwilVZGOTAqq8cygSfGJYt-xr5nlwnuqb4ZeaCPCYJqjuzVACCqOiFtVpd8nTIxyv6rKFLvZtGZWxQ7l1jMKb~HA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400609&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise profiles of nucleosome positions at promoters. (A) Helical rise profile (black solid line) of the YAL053W promoter sequence in chromosome 1 of <em>S. cerevisiae,</em> obtained by a smoothing window of 90 bp. Nucleosome occupancy profile (pink solid line) obtained by the dinucleotide frequency method by Segal <em>et al</em>. Upper black dots with horizontal bars return peak heights of the corresponding nucleosomes. Lower black dots with horizontal bars return nucleosome numbering. Blue dots with horizontal bars return nucleosome repositioning in the isw2 deletion strain. Green and red dots represent TSS and TTS, respectively. (B) Helical rise profile (black solid line) of YAL053W smoothed at 90 bp as in (A). Average helical rise profiles (red solid lines) for nucleosomes sharing the same position number. Upper red dots with horizontal bars return mean peak heights of the corresponding nucleosomes expressed in arbitrary units as reported in the ‘additional data file 1’.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Lower black dots with horizontal bars return nucleosome numbering. Green and red dots represent TSSs and TTSs, respectively. (C) Scale enlargement of the mean helical rise profiles for nucleosomes sharing the same position number reported in (B, black solid line). The mirror image of the same profiles (pink solid line). The average between peak heights of the black and pink profiles (blue diamonds). Black dots with horizontal bars return nucleosome numbering. Colour figures can be found in the online version of this paper.</p></div></div><div content-id="DSR043F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04303.jpeg?Expires=1509238796&Signature=PFrRu9wJd4LeyGlXojEog2wgR1XxCK0O7mGttuo7LpSQd1qHlyosgMzrcgA9xw5lgeWsa1UXRSvN-lGMBL4kUtyzzMNYQSci-hbur2kVyaTfyBjCde2vgr~GF~3UoXoXQlFDQUKyQcfYXO8tQ7qJPKP0udVJw~c1nc7DwtahQliYmRME2-Vz6cZw~L~cGUFRf14o8LGUxxcjAEQnmGrY00EoEo7ML359aOw2fOQJQ9WQm24JRIP~jx8vTMF14T06AKDat-veYdMBLc~E~MuTYNEVGkgd6umbNjjMMXZfTE7KE4-VY66iqZxL1xvMgvdjhujUKgSEnunKbBebZUnGWA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise profiles of nucleosome positions at promoters. (A) Helical rise profile (black solid line) of the YAL053W promoter sequence in chromosome 1 of S. cerevisiae, obtained by a smoothing window of 90 bp. Nucleosome occupancy profile (pink solid line) obtained by the dinucleotide frequency method by Segal et al. Upper black dots with horizontal bars return peak heights of the corresponding nucleosomes. Lower black dots with horizontal bars return nucleosome numbering. Blue dots with horizontal bars return nucleosome repositioning in the isw2 deletion strain. Green and red dots represent TSS and TTS, respectively. (B) Helical rise profile (black solid line) of YAL053W smoothed at 90 bp as in (A). Average helical rise profiles (red solid lines) for nucleosomes sharing the same position number. Upper red dots with horizontal bars return mean peak heights of the corresponding nucleosomes expressed in arbitrary units as reported in the ‘additional data file 1’.14 Lower black dots with horizontal bars return nucleosome numbering. Green and red dots represent TSSs and TTSs, respectively. (C) Scale enlargement of the mean helical rise profiles for nucleosomes sharing the same position number reported in (B, black solid line). The mirror image of the same profiles (pink solid line). The average between peak heights of the black and pink profiles (blue diamonds). Black dots with horizontal bars return nucleosome numbering. Colour figures can be found in the online version of this paper." path-from-xml="dsr04303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400609/dsr04303.jpeg" path-from-xml="dsr04303.jpeg" target="_blank">View large</a><a section="7400609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04303.jpeg?Expires=1509238796&Signature=LBtFsKG1QDgUgZX2WQbSlbsSjN126oR2W1jvkxw-mWpFkwxstNitmFrDng~-86pvNuqMmf2Qn9NjyRNtYwKTuv9lIF2o-YDHI48pAhwktZFYzEz4qao6Y6YUCfaH8dEUGp2raMImVPFC6a91JMVaMu79xWtZNWqbNPmeTZILlp7RMrNi1KW42Hfr~PtffIRchgqO0cK54n3sQNUchXfKLDIB0LblcgQEyqFeoOyUYqcXPi4XW9f0Ny7W4jH-7BxwilVZGOTAqq8cygSfGJYt-xr5nlwnuqb4ZeaCPCYJqjuzVACCqOiFtVpd8nTIxyv6rKFLvZtGZWxQ7l1jMKb~HA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400609&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise profiles of nucleosome positions at promoters. (A) Helical rise profile (black solid line) of the YAL053W promoter sequence in chromosome 1 of <em>S. cerevisiae,</em> obtained by a smoothing window of 90 bp. Nucleosome occupancy profile (pink solid line) obtained by the dinucleotide frequency method by Segal <em>et al</em>. Upper black dots with horizontal bars return peak heights of the corresponding nucleosomes. Lower black dots with horizontal bars return nucleosome numbering. Blue dots with horizontal bars return nucleosome repositioning in the isw2 deletion strain. Green and red dots represent TSS and TTS, respectively. (B) Helical rise profile (black solid line) of YAL053W smoothed at 90 bp as in (A). Average helical rise profiles (red solid lines) for nucleosomes sharing the same position number. Upper red dots with horizontal bars return mean peak heights of the corresponding nucleosomes expressed in arbitrary units as reported in the ‘additional data file 1’.<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Lower black dots with horizontal bars return nucleosome numbering. Green and red dots represent TSSs and TTSs, respectively. (C) Scale enlargement of the mean helical rise profiles for nucleosomes sharing the same position number reported in (B, black solid line). The mirror image of the same profiles (pink solid line). The average between peak heights of the black and pink profiles (blue diamonds). Black dots with horizontal bars return nucleosome numbering. Colour figures can be found in the online version of this paper.</p></div></div><p>The similarities observed, in Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>, between our curves and those by Segal <em>et al</em>. can be explained, considering that nucleosome formation depends on a first step characterized by the bending of DNA and correlated to the dinucleotide distribution, as demonstrated by Segal <em>et al</em>.<sup><a reveal-id="DSR043C9" data-open="DSR043C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The following step is associated with the stability of bonds formed between DNA and histone proteins, and the second stage is correlated to DNA helical rise.<sup><a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>We analysed the helical rise profiles of all transcribed genes in yeast in order to compare their average peak heights with those reported in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>A for the reference data set. In Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>B, the helix rise profile of YAL053W, smoothed at 90 bp, is marked by a black solid line. In the same figure, average profiles for nucleosomes characterized by the same position number are shown. Black horizontal bars mark the nucleosome numbers, whereas red horizontal bars show the average peak height calculated for all the nucleosomes having the same number. It can be seen that the average peak height of the +1 nucleosome is larger (4.3 arbitrary units) than the heights of the following ones, in agreement with the <em>barrier</em> model.<sup><a reveal-id="DSR043C15" data-open="DSR043C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>In Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>, average helical rise profiles for nucleosomes sharing the same position number are marked by solid red lines. As apparent from the graph, average helical rise profiles for nucleosomes at positions 0 and +1 are characterized by the lowest values. Statistical tests (non-parametric Wilcoxon–Mann–Whitney) were performed in order to asses the difference between +1 and +2 (and +3, +4, …, +7) rise value distribution (dyad rise values mediated on a window of 90 bp). <em>P</em>-values smaller than E-16 were computed, assessing that the distributions do no't belong to the same population, strongly rejecting the null hypothesis. It has recently been reported<sup><a reveal-id="DSR043C16" data-open="DSR043C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR043C17" data-open="DSR043C17" class="link link-ref link-reveal xref-bibr">17</a></sup> that, in nucleosomes assembled <em>in vitro,</em> the usual <em>barrier-model</em> pattern, with the +1 nucleosome being the most stable, is not seen. This result supports our idea that the obtained helical rise profiles are mainly correlated to the DNA sequence and can mirror <em>in vitro</em> occupancy. In Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>C, average helical rise profiles for nucleosomes sharing the same position number (solid red line in Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>B) are shown on a magnified scale and marked by a solid black line. The presence of a <em>barrier</em> beginning at position +2 is now clearer. Note here that the number of nucleosomes used to get the average profiles varies from 7497 (for position +1) to ∼1000 (for position +15), reaching the final value of ∼48 000 positioned nucleosomes instead of the 61 110 reported in the additional data file 1<sup><a reveal-id="DSR043C14" data-open="DSR043C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and used to construct Figs <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSR043F2" data-open="DSR043F2" class="link link-reveal link-table xref-fig">2</a>. About half of the 7497 nucleosomes at position +1 reside on opposite strands (3758 on the Watson and 3739 on the Crick strand) and this trend is fulfilled by the following nucleosomes (at increasing position numbers) as well. As a consequence, the global nucleosome pattern must be represented as due to the presence of a mirror image of the original black profiles, as shown in Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>C (pink solid line). Here the blue diamonds represent the average between peak heights belonging to black and pink profiles. The mean helical rise of the blue diamonds ranging from position +2 to position +12 shows an almost constant value of 3.175 Å, similar to the mean rise of the central peak shown in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> (black continuous line with a smoothing window of 90 bp). The mean helical rise of the blue diamonds derived from positions 0, +1, +14 and +13 shows a value of about 3.168 Å similar to the peaks neighbouring the central one in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a>. The adoption of the mirror image in Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>C allowed us to interpret the shape of Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> mainly due to the contribution of the peaks that constitute the dynamic barrier as shown in Fig. <a reveal-id="DSR043F3" data-open="DSR043F3" class="link link-reveal link-table xref-fig">3</a>C.</p>                            <h3 scrollto-destination=7400613 id="7400613" class="section-title" data-legacy-id=s3d>3.4. Helical rise profiles of additional genomes</h3>
<p>Figure <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a> presents the helical rise profiles derived from experimentally mapped nucleosome positions in <em>D. melanogaster</em>,<sup><a reveal-id="DSR043C28" data-open="DSR043C28" class="link link-ref link-reveal xref-bibr">28</a></sup><em>P. falciparum</em><sup><a reveal-id="DSR043C29" data-open="DSR043C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and in chromosome 20 of human genome.<sup><a reveal-id="DSR043C25" data-open="DSR043C25" class="link link-ref link-reveal xref-bibr">25</a></sup> All the curves were obtained with smoothing windows of 56 bp in order to assess the presence of peaks split in two adjacent maxima as previously observed in yeast. This pattern is not evident in the helical rise profiles shown in Fig. <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a>, and only in the case of <em>D. melanogaster</em> (Fig. <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a>A) the presence of three peaks around the central zone of the reference dyad position may be related to the action of remodelling enzymes as previously shown and discussed in Fig. <a reveal-id="DSR043F1" data-open="DSR043F1" class="link link-reveal link-table xref-fig">1</a> in the case of yeast. The nucleosome positions mapped in yeast have been derived by six different experimental data sets and the best accuracy<sup><a reveal-id="DSR043C20" data-open="DSR043C20" class="link link-ref link-reveal xref-bibr">20</a></sup> of tiled microarrays covering the entire yeast genome was extended up to a 5-bp spacing. Only one data set of nucleosome positions is still available for <em>D. melanogaster</em>, <em>P. falciparum</em> and <em>Homo sapiens</em> while the spacing of their tiled microarrays comprises 36, 36 and 25 bp, respectively, therefore the results shown in Fig. <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a> must be considered as preliminary ones. A general conclusion is evident from the analysis of the three genomes, i.e. the preferential nucleosome occupancy occurs where the mean helical rise reaches its largest values. Probably the different shape of the three curves reflects the different organization of nucleosome near the +1 position at promoters. In yeast, the +1 nucleosome is usually found at +75 bp from TSS and in <em>D. melanogaster</em> this position is shifted at +135.<sup><a reveal-id="DSR043C28" data-open="DSR043C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In <em>Plasmodium</em>, near promoters are present NFRr that are devoid of nucleosomes;<sup><a reveal-id="DSR043C29" data-open="DSR043C29" class="link link-ref link-reveal xref-bibr">29</a></sup> in human expressed and inexpressed genes exhibit differential positioning of nucleosome +1.<sup><a reveal-id="DSR043C24" data-open="DSR043C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Figure <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a>D shows the helical rise profiles derived by nucleosome positions in chromosome 20 of the human genome computationally mapped from the DNA sequence. Dyad positions were elaborated by Liu <em>et al</em>.<sup><a reveal-id="DSR043C25" data-open="DSR043C25" class="link link-ref link-reveal xref-bibr">25</a></sup> either from their curvature profile method or from curves obtained according the dinucleotide frequency method of Segal <em>et al</em>.<sup><a reveal-id="DSR043C9" data-open="DSR043C9" class="link link-ref link-reveal xref-bibr">9</a></sup> A comparison of the helical rise profile reported in Fig. <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a>C for the experimentally determined dyad positions of human chromosome 20 with the two computationally derived curves of Fig. <a reveal-id="DSR043F4" data-open="DSR043F4" class="link link-reveal link-table xref-fig">4</a>D shows that the predictive power of the curvature profile method (black continuous line) is higher than that attainable with the dinucleotide frequency method (grey continuous line) although for the curve obtained with the curvature profile a remarkable shift of ∼40 bp in the position of the central peak is observed. We are not able at the moment to explain the reason for this shift. </p>                        <a id="7400615" scrollto-destination="7400615"></a>
<div id="DSR043F4" data-id="DSR043F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04304.jpeg?Expires=1509238796&Signature=XZEUpGq3ADM0j21d7UKjhSnGgUp8WQWMUw1yvSqAvNqPF-BIW~aWnbY9rYSp0ZYj83lxb3qm0L8HIWHzVnjBhBZijXwTiTIzYWa0w7ZyUTKFvta3uXP9dYEhWC9EFceu1Lrzw11Y2S80AufGMrTIP-lJ55dZT7Cq0Y-46DbN3-5IZYZOBM7GMwOSbfiQs48stoXizAL7a~ULbiVVzDp6jXFIlyOo9Dck-kxFV574ERQOaMcQqpAyINjsGKa-ULHHVta1ZtBEeYY1MIpDVNg0GRNl4Xsf8UnFC~Chk9jo-mpDYTVeQO94Mmaoa9eAV7MPvRWoPl2KVJLIVnGOJqTG1Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise profiles of nucleosome positions of various genomes. Average helical rise profiles of experimentally mapped nucleosome positions in D. melanogaster (A, average of 617 304 points, black solid line), P. falciparum (B, average of 89 115 points, black solid line), chromosome 20 of human genome (C, average of 65 379 points, black solid line) and in computationally mapped nucleosome positions in chromosome 20 of human genome (D, average of 65 537 points, dinucleotide frequency method, grey solid line and average of 65 537 points, curvature profile method, black solid line). All the curves were obtained with smoothing windows of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars return the position of the central nucleosome." path-from-xml="dsr04304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400615/dsr04304.jpeg" path-from-xml="dsr04304.jpeg" target="_blank">View large</a><a section="7400615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04304.jpeg?Expires=1509238796&Signature=WzLXGOM~H7O8z~a2rWG1rzjmc1TRCqLMuPpzu~1ONmP2ZHXBBA50zhm~knaLmIitgI5Kk3IGWcsXdTck8ZT8HtWtzBX2uvEtp6MdXeP4BQA-7hQB7dSj77CvtkOtSUFNNiEyyclTLR~ggf~POOR-KczxtYtSUs3xShz2dnI0BD2dAe9hxj2krOyjZQ9jsUEfg7MvFMpSTxZvDbkGz15OMuAxNoSmAPaiGufZ-tEWGGBim88QlipeSSd~H2bJxh2N~cuI1a~2QDwPd5UW5VUHObt8DKNc1j~jVuaFQQeFJKc~szWL2hlkCGAjcDcNHFlxAKDCApjGwwlWzwMkyFrUSg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400615&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise profiles of nucleosome positions of various genomes. Average helical rise profiles of experimentally mapped nucleosome positions in <em>D. melanogaster</em> (A, average of 617 304 points, black solid line), <em>P. falciparum</em> (B, average of 89 115 points, black solid line), chromosome 20 of human genome (C, average of 65 379 points, black solid line) and in computationally mapped nucleosome positions in chromosome 20 of human genome (D, average of 65 537 points, dinucleotide frequency method, grey solid line and average of 65 537 points, curvature profile method, black solid line). All the curves were obtained with smoothing windows of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars return the position of the central nucleosome.</p></div></div><div content-id="DSR043F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04304.jpeg?Expires=1509238796&Signature=XZEUpGq3ADM0j21d7UKjhSnGgUp8WQWMUw1yvSqAvNqPF-BIW~aWnbY9rYSp0ZYj83lxb3qm0L8HIWHzVnjBhBZijXwTiTIzYWa0w7ZyUTKFvta3uXP9dYEhWC9EFceu1Lrzw11Y2S80AufGMrTIP-lJ55dZT7Cq0Y-46DbN3-5IZYZOBM7GMwOSbfiQs48stoXizAL7a~ULbiVVzDp6jXFIlyOo9Dck-kxFV574ERQOaMcQqpAyINjsGKa-ULHHVta1ZtBEeYY1MIpDVNg0GRNl4Xsf8UnFC~Chk9jo-mpDYTVeQO94Mmaoa9eAV7MPvRWoPl2KVJLIVnGOJqTG1Q__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise profiles of nucleosome positions of various genomes. Average helical rise profiles of experimentally mapped nucleosome positions in D. melanogaster (A, average of 617 304 points, black solid line), P. falciparum (B, average of 89 115 points, black solid line), chromosome 20 of human genome (C, average of 65 379 points, black solid line) and in computationally mapped nucleosome positions in chromosome 20 of human genome (D, average of 65 537 points, dinucleotide frequency method, grey solid line and average of 65 537 points, curvature profile method, black solid line). All the curves were obtained with smoothing windows of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars return the position of the central nucleosome." path-from-xml="dsr04304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400615/dsr04304.jpeg" path-from-xml="dsr04304.jpeg" target="_blank">View large</a><a section="7400615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04304.jpeg?Expires=1509238796&Signature=WzLXGOM~H7O8z~a2rWG1rzjmc1TRCqLMuPpzu~1ONmP2ZHXBBA50zhm~knaLmIitgI5Kk3IGWcsXdTck8ZT8HtWtzBX2uvEtp6MdXeP4BQA-7hQB7dSj77CvtkOtSUFNNiEyyclTLR~ggf~POOR-KczxtYtSUs3xShz2dnI0BD2dAe9hxj2krOyjZQ9jsUEfg7MvFMpSTxZvDbkGz15OMuAxNoSmAPaiGufZ-tEWGGBim88QlipeSSd~H2bJxh2N~cuI1a~2QDwPd5UW5VUHObt8DKNc1j~jVuaFQQeFJKc~szWL2hlkCGAjcDcNHFlxAKDCApjGwwlWzwMkyFrUSg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400615&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise profiles of nucleosome positions of various genomes. Average helical rise profiles of experimentally mapped nucleosome positions in <em>D. melanogaster</em> (A, average of 617 304 points, black solid line), <em>P. falciparum</em> (B, average of 89 115 points, black solid line), chromosome 20 of human genome (C, average of 65 379 points, black solid line) and in computationally mapped nucleosome positions in chromosome 20 of human genome (D, average of 65 537 points, dinucleotide frequency method, grey solid line and average of 65 537 points, curvature profile method, black solid line). All the curves were obtained with smoothing windows of 56 bp. Nucleosome dyad positions are aligned at bp number 0; 500 bp upstream and downstream of this point are shown. Black dots with (146-bp long) horizontal bars return the position of the central nucleosome.</p></div></div>                            <h3 scrollto-destination=7400616 id="7400616" class="section-title" data-legacy-id=s3e>3.5. Helical rise and GC content</h3>
<p>The GC content of DNA sequences was found to be correlated to the intrinsic nucleosome occupancy in yeast and in <em>Caenorhabditis elegans</em>,<sup><a reveal-id="DSR043C30" data-open="DSR043C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and a high GC content was found at human regulator sequences enriched in well-positioned nucleosomes.<sup><a reveal-id="DSR043C31" data-open="DSR043C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Chen <em>et al</em>.<sup><a reveal-id="DSR043C32" data-open="DSR043C32" class="link link-ref link-reveal xref-bibr">32</a></sup> reported that in <em>C. elegans</em>, the nucleosome-enriched regions are GC-rich when compared with the nucleosome-free sequences and, recently, the GC content has played an important role in the determinants of nucleosome organization in human cells.<sup><a reveal-id="DSR043C33" data-open="DSR043C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We have reanalysed the helical rise values of the 136 possible tetranucleotides reported in our previous study<sup><a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and in Fig. <a reveal-id="DSR043F5" data-open="DSR043F5" class="link link-reveal link-table xref-fig">5</a> we plot the distributions of their values as a function of the five different GC contents. At 100 and 0% GC, the distribution of the helical rise is represented with only 10 samples, the profiles are almost flat and the mean helical rises are similar, 3.13 and 3.12 Å, respectively. The profiles of the 25, 50 and 75% GC show that the mean helical rise increases with the increasing of the GC content ranging from 3.07 to 3.3 Å. The correspondence of high helical rise and GC-rich sequences may constitute a very simple explanation of why our helical rise profiles resemble maps of nucleosome occupancy currently reported in the literature and why nucleosomes are mainly found at positions where helical rise reaches its largest averaged values. </p>                        <a id="7400618" scrollto-destination="7400618"></a>
<div id="DSR043F5" data-id="DSR043F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04305.jpeg?Expires=1509238796&Signature=EhcY3OFUUD02VWsIGxEJWX44A7qSaJnXzTzFIOZw31UlXtiqKhHeJE0FlmVSzCdTP-ct7JCzqhB7jmSht1eRqfkrNhbhPnypMaF1TzydxHneHWaoa36iZVXnTtFgQ4UPQi~NUXxZBVkmsFXyy3bKCoVPnNj5ixpxDhrtahGJdzKDGzpXhVyPNIvek5o3oiIz9LivwcjqCQYeIBx72Sq889sKzjuwjJw0Oqzd9fQ7G9IFcRguKg9cnkd6UUN9R0XtiJnsn1zeqOLW7Hk0-EBoxJ2GKeNzHfkVRIi9RzpMVgpA9neYUO2cuLlogR8Af4CIOj~B2LOi7FqZqU74LcDAqw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise and GC content. The helical rise values of the 136 unique tetranucleotides are shown divided according to their GC content." path-from-xml="dsr04305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400618/dsr04305.jpeg" path-from-xml="dsr04305.jpeg" target="_blank">View large</a><a section="7400618" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04305.jpeg?Expires=1509238796&Signature=Ic3k0Yv7208ilnXwRL0ZgHpCWtOJcX6tulxFEV4W8vPzVk54jalvrdEJ0x8Cz4Vx5w9MFYosAGnNISKn9hzbUzqAdEqJG33IakIj1S51B6TSf83p8PvGwfEUsF1SGqVog21D790BTPl4eF0ELy62gmGZdzYeuOVVdtXBf-f5CsPLkizS6ESFinpxJdGN-MAqW3QYSUn8ZsplN5BV0088XlTYwAOW80548jszwl7jBVUqBqejfLIZlSFcQ8zejkEJ1ZpnmNmQNPy8T8TO7miU-5FuKfs-FC2TT3oufrdqHFSfSozjliyhtIWMj-wrUnr98y62fvywNkl6HtGH98wLMw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400618&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise and GC content. The helical rise values of the 136 unique tetranucleotides are shown divided according to their GC content.</p></div></div><div content-id="DSR043F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/m_dsr04305.jpeg?Expires=1509238796&Signature=EhcY3OFUUD02VWsIGxEJWX44A7qSaJnXzTzFIOZw31UlXtiqKhHeJE0FlmVSzCdTP-ct7JCzqhB7jmSht1eRqfkrNhbhPnypMaF1TzydxHneHWaoa36iZVXnTtFgQ4UPQi~NUXxZBVkmsFXyy3bKCoVPnNj5ixpxDhrtahGJdzKDGzpXhVyPNIvek5o3oiIz9LivwcjqCQYeIBx72Sq889sKzjuwjJw0Oqzd9fQ7G9IFcRguKg9cnkd6UUN9R0XtiJnsn1zeqOLW7Hk0-EBoxJ2GKeNzHfkVRIi9RzpMVgpA9neYUO2cuLlogR8Af4CIOj~B2LOi7FqZqU74LcDAqw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Helical rise and GC content. The helical rise values of the 136 unique tetranucleotides are shown divided according to their GC content." path-from-xml="dsr04305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7400618/dsr04305.jpeg" path-from-xml="dsr04305.jpeg" target="_blank">View large</a><a section="7400618" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr043/2/dsr04305.jpeg?Expires=1509238796&Signature=Ic3k0Yv7208ilnXwRL0ZgHpCWtOJcX6tulxFEV4W8vPzVk54jalvrdEJ0x8Cz4Vx5w9MFYosAGnNISKn9hzbUzqAdEqJG33IakIj1S51B6TSf83p8PvGwfEUsF1SGqVog21D790BTPl4eF0ELy62gmGZdzYeuOVVdtXBf-f5CsPLkizS6ESFinpxJdGN-MAqW3QYSUn8ZsplN5BV0088XlTYwAOW80548jszwl7jBVUqBqejfLIZlSFcQ8zejkEJ1ZpnmNmQNPy8T8TO7miU-5FuKfs-FC2TT3oufrdqHFSfSozjliyhtIWMj-wrUnr98y62fvywNkl6HtGH98wLMw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=7400618&ar=486167&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Helical rise and GC content. The helical rise values of the 136 unique tetranucleotides are shown divided according to their GC content.</p></div></div>                            <h2 scrollto-destination=7400619 id="7400619" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>We used genomic maps of yeast nucleosome positions as primary standards to study the role of DNA helical rise in chromatin organization. Preferential positions for nucleosomes were found where the mean helical rise reaches its largest values at GC-rich DNA sequences. This result was confirmed by a preliminary survey of genomic maps of positioned nucleosomes of the <em>D. melanogaster</em> and <em>P. falciparum</em> genomes and of the human chromosome 20. We suggest that, in order to relieve the constraint imposed by the bending of the double helix, DNA regions characterized by high helical rise values are favoured when compared with shorter ones, i.e. the former do not need further stretch in order to reach their interaction points and, therefore, the stability of nucleosomes is directly correlated to the mean value of helical rise. In a previous paper,<sup><a reveal-id="DSR043C22" data-open="DSR043C22" class="link link-ref link-reveal xref-bibr">22</a></sup> we reported a correlation between the stability of nucleosomes and the presence of a symmetrical distribution of distances from the nucleosomal dyad axis at specific points. The same feature is now observed in yeast helical rise profiles as a series of ‘peak’- and ‘V’-shaped curves. The difference between the two shapes resides in the lower stability of the ‘V’-shaped profiles, due to their lower helical rise values. In descending order of preference in nucleosome formation, DNA sequences characterized by the smaller helical rise values (i.e. TSS) are placed last.</p><p>Preferential nucleosome occupancy was found downstream of NFRs, as observed in the barrier nucleosome model<sup><a reveal-id="DSR043C15" data-open="DSR043C15" class="link link-ref link-reveal xref-bibr">15</a></sup> for statistical positioning of nucleosomes in yeast.</p>                            <h2 scrollto-destination=7400622 id="7400622" class="section-title" data-legacy-id=s5>Authors’ contributions</h2>
<p>F.P. and D.S. equally contributed to the project, both in terms of computational analysis and drafting of the manuscript. All authors read and approved the final manuscript.</p>                            <h2 scrollto-destination=7400624 id="7400624" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsr043c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lohr</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fatchell</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Van-Holde</div> <div class="given-names">K.E.</div></div>. </div><div class="article-title">On the occurrence of nucleosome phasing in chromatin</div>, <div class="source">Cell</div> , <div class="year">1977</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">829</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20occurrence%20of%20nucleosome%20phasing%20in%20chromatin&amp;author=D.%20Lohr&amp;author=K.%20Fatchell&amp;author=K.E.%20Van-Holde&amp;publication_year=1977&amp;journal=Cell&amp;volume=12&amp;pages=829-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(77)90281-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(77)90281-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(77)90281-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/336221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trifonov</div> <div class="given-names">E.N.</div></div>. </div><div class="article-title">Structure of DNA in chromatin</div>, <div class="source">Eur. J. Cell Biol.</div> , <div class="year">1980</div>, vol. <div class="volume">22</div> pg. <div class="fpage">74</div> </div></div></div></div><div content-id="dsr043c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trifonov</div> <div class="given-names">E.N.</div></div>,  <div class="name"><div class="surname">Sussman</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">The pitch of chromatin DNA is reflected in its nucleotide sequence</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1980</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">3816</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pitch%20of%20chromatin%20DNA%20is%20reflected%20in%20its%20nucleotide%20sequence&amp;author=E.N.%20Trifonov&amp;author=J.L.%20Sussman&amp;publication_year=1980&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=77&amp;pages=3816-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.77.7.3816" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.77.7.3816" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.77.7.3816"> </span></div> </div></div></div></div></div><div content-id="dsr043c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mengeritsky</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Trifonov</div> <div class="given-names">E.N.</div></div>. </div><div class="article-title">Nucleotide sequence-directed mapping of the nucleosome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1983</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">3833</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequence-directed%20mapping%20of%20the%20nucleosome&amp;author=G.%20Mengeritsky&amp;author=E.N.%20Trifonov&amp;publication_year=1983&amp;journal=Nucleic%20Acids%20Res.&amp;volume=11&amp;pages=3833-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/11.11.3833" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F11.11.3833" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F11.11.3833"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6856466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Griffith</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">The [(G/C)3NN]n motif: a common DNA repeat that excludes nucleosomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">8863</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%5B(G%2FC)3NN%5Dn%20motif%3A%20a%20common%20DNA%20repeat%20that%20excludes%20nucleosomes&amp;author=Y.H.%20Wang&amp;author=J.D.%20Griffith&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=8863-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.17.8863" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.17.8863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.17.8863"> </span></div> </div></div></div></div></div><div content-id="dsr043c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowary</div> <div class="given-names">P.T.</div></div>,  <div class="name"><div class="surname">Widom</div> <div class="given-names">J.</div></div>. </div><div class="article-title">New DNA sequence rules for high affinity binding to hystone octamer and sequence-directed nucleosome positioning</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">19</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20DNA%20sequence%20rules%20for%20high%20affinity%20binding%20to%20hystone%20octamer%20and%20sequence-directed%20nucleosome%20positioning&amp;author=P.T.%20Lowary&amp;author=J.%20Widom&amp;publication_year=1998&amp;journal=J.%20Mol.%20Biol.&amp;volume=276&amp;pages=19-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.1494" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.1494" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.1494"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9514715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thåström</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bingham</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Widom</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Nucleosomal locations of dominant DNA sequence motifs for histone–DNA interactions and nucleosome positioning</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">338</div> (pg. <div class="fpage">695</div>-<div class="lpage">709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosomal%20locations%20of%20dominant%20DNA%20sequence%20motifs%20for%20histone%E2%80%93DNA%20interactions%20and%20nucleosome%20positioning&amp;author=A.%20Th%C3%A5str%C3%B6m&amp;author=L.M.%20Bingham&amp;author=J.%20Widom&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol.&amp;volume=338&amp;pages=695-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2004.03.032" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2004.03.032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2004.03.032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15099738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Field</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fondufe-Mittendorf</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">I.K.</div></div>, et al. </div><div class="article-title">Gene expression divergence in yeast is coupled to evolution of DNA-encoded nucleosome organization</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">438</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20divergence%20in%20yeast%20is%20coupled%20to%20evolution%20of%20DNA-encoded%20nucleosome%20organization&amp;author=Y.%20Field&amp;author=Y.%20Fondufe-Mittendorf&amp;author=I.K.%20Moore&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=438-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.324" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19252487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Segal</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fondufe-Mittendorf</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A genomic code for nucleosome positioning</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">442</div> (pg. <div class="fpage">772</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20code%20for%20nucleosome%20positioning&amp;author=E.%20Segal&amp;author=Y.%20Fondufe-Mittendorf&amp;author=L.%20Chen&amp;publication_year=2006&amp;journal=Nature&amp;volume=442&amp;pages=772-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04979" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16862119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaplan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">I.K.</div></div>,  <div class="name"><div class="surname">Fondufe-Mittendorf</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The DNA-encoded nucleosome organization of a eukaryotic genome</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">458</div> (pg. <div class="fpage">362</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20DNA-encoded%20nucleosome%20organization%20of%20a%20eukaryotic%20genome&amp;author=N.%20Kaplan&amp;author=I.K.%20Moore&amp;author=Y.%20Fondufe-Mittendorf&amp;publication_year=2009&amp;journal=Nature&amp;volume=458&amp;pages=362-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07667" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07667" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07667"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19092803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peckham</div> <div class="given-names">H.E.</div></div>,  <div class="name"><div class="surname">Thurman</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Nucleosome positioning signals in genomic DNA</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1170</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20positioning%20signals%20in%20genomic%20DNA&amp;author=H.E.%20Peckham&amp;author=R.E.%20Thurman&amp;author=Y.%20Fu&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1170-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6101007" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6101007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6101007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17620451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miele</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Vaillant</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">d'Aubenton-Carafa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Themes</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Grange</div> <div class="given-names">T.</div></div>. </div><div class="article-title">DNA physical properties determine nucleosome occupancy from yeast to fly</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3746</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20physical%20properties%20determine%20nucleosome%20occupancy%20from%20yeast%20to%20fly&amp;author=V.%20Miele&amp;author=C.%20Vaillant&amp;author=Y.%20d'Aubenton-Carafa&amp;author=C.%20Themes&amp;author=T.%20Grange&amp;publication_year=2008&amp;journal=Nucleic%20Acid%20Res.&amp;volume=36&amp;pages=3746-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn262" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn262"> </span></div> </div></div></div></div></div><div content-id="dsr043c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morozov</div> <div class="given-names">A.V.</div></div>,  <div class="name"><div class="surname">Fortney</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Gaykalova</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Studitsky</div> <div class="given-names">V.M.</div></div>,  <div class="name"><div class="surname">Widom</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Siggia</div> <div class="given-names">E.D.</div></div>. </div><div class="article-title">Using DNA mechanics to predict in vitro nucleosome positions and formation energies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">4707</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20DNA%20mechanics%20to%20predict%20in%20vitro%20nucleosome%20positions%20and%20formation%20energies&amp;author=A.V.%20Morozov&amp;author=K.%20Fortney&amp;author=D.A.%20Gaykalova&amp;author=V.M.%20Studitsky&amp;author=J.%20Widom&amp;author=E.D.%20Siggia&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=4707-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp475" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19509309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pugh</div> <div class="given-names">B.F.</div></div>. </div><div class="article-title">A compiled and systematic reference map of nucleosome positions across the <em>Saccharomyces cerevisiae</em> genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R109</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20compiled%20and%20systematic%20reference%20map%20of%20nucleosome%20positions%20across%20the%20Saccharomyces%20cerevisiae%20genome&amp;author=C.%20Jiang&amp;author=B.F.%20Pugh&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-10-r109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-10-r109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-10-r109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19814794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mavrich</div> <div class="given-names">T.N.</div></div>,  <div class="name"><div class="surname">Ioshikhes</div> <div class="given-names">I.P.</div></div>,  <div class="name"><div class="surname">Venters</div> <div class="given-names">B.J.</div></div>, et al. </div><div class="article-title">A barrier nucleosome model for statistical positioning of nucleosomes throughout the yeast genome</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1073</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20barrier%20nucleosome%20model%20for%20statistical%20positioning%20of%20nucleosomes%20throughout%20the%20yeast%20genome&amp;author=T.N.%20Mavrich&amp;author=I.P.%20Ioshikhes&amp;author=B.J.%20Venters&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1073-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.078261.108" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.078261.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.078261.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18550805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Moqtaderi</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Ratner</div> <div class="given-names">B.P.</div></div>, et al. </div><div class="article-title">Intrinsic histone–DNA interactions are not the major determinant of nucleosome positions in vivo</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">847</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intrinsic%20histone%E2%80%93DNA%20interactions%20are%20not%20the%20major%20determinant%20of%20nucleosome%20positions%20in%20vivo&amp;author=Y.%20Zhang&amp;author=Z.%20Moqtaderi&amp;author=B.P.%20Ratner&amp;publication_year=2009&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=16&amp;pages=847-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1636" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19620965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Moqtaderi</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Ratner</div> <div class="given-names">B.P.</div></div>, et al. </div><div class="article-title">Evidence against a genomic code for nucleosome positioning</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">920</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20against%20a%20genomic%20code%20for%20nucleosome%20positioning&amp;author=Y.%20Zhang&amp;author=Z.%20Moqtaderi&amp;author=B.P.%20Ratner&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=17&amp;pages=920-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb0810-920" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb0810-920" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb0810-920"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20683474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaplan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Fondufe-Mittendorf</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Nucleosome sequence preferences influence in vivo nucleosome organization</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">918</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20sequence%20preferences%20influence%20in%20vivo%20nucleosome%20organization&amp;author=N.%20Kaplan&amp;author=I.%20Moore&amp;author=Y.%20Fondufe-Mittendorf&amp;publication_year=2010&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=17&amp;pages=918-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb0810-918" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb0810-918" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb0810-918"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20683473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaplan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Lieb</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Widom</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Segal</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Contribution of histone sequence preferences to nucleosome organization: proposed definitions and methodology</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">140</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contribution%20of%20histone%20sequence%20preferences%20to%20nucleosome%20organization%3A%20proposed%20definitions%20and%20methodology&amp;author=N.%20Kaplan&amp;author=T.R.%20Hughes&amp;author=J.D.%20Lieb&amp;author=J.%20Widom&amp;author=E.%20Segal&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=140" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-11-140" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-11-140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-11-140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21118582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whitehouse</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Rando</div> <div class="given-names">O.J.</div></div>,  <div class="name"><div class="surname">Delrow</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tsukiyama</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chromatin remodelling at promoters suppresses antisense transcription</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">450</div> (pg. <div class="fpage">1031</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20remodelling%20at%20promoters%20suppresses%20antisense%20transcription&amp;author=I.%20Whitehouse&amp;author=O.J.%20Rando&amp;author=J.%20Delrow&amp;author=T.%20Tsukiyama&amp;publication_year=2007&amp;journal=Nature&amp;volume=450&amp;pages=1031-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06391" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18075583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pedone</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Mazzei</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Santoni</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Sequence-dependent DNA torsional rigidity: a tetranucleotide code</div>, <div class="source">Biophys. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">77</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-dependent%20DNA%20torsional%20rigidity%3A%20a%20tetranucleotide%20code&amp;author=F.%20Pedone&amp;author=F.%20Mazzei&amp;author=D.%20Santoni&amp;publication_year=2004&amp;journal=Biophys.%20Chem.&amp;volume=112&amp;pages=77-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bpc.2004.07.005" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bpc.2004.07.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bpc.2004.07.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15501578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pedone</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Santoni</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Sequence-dependent DNA helical rise and nucleosome stability</div>, <div class="source">BMC <em>Mol. Biol.</em></div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">105</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-dependent%20DNA%20helical%20rise%20and%20nucleosome%20stability&amp;author=F.%20Pedone&amp;author=D.%20Santoni&amp;publication_year=2009&amp;journal=BMC%20Mol.%20Biol.&amp;volume=10&amp;pages=105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2199-10-105" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2199-10-105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2199-10-105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19943916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richmond</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Davey</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">The structure of DNA in the nucleosome core</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">423</div> (pg. <div class="fpage">145</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20DNA%20in%20the%20nucleosome%20core&amp;author=T.J.%20Richmond&amp;author=C.A.%20Davey&amp;publication_year=2003&amp;journal=Nature&amp;volume=423&amp;pages=145-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01595" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12736678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schones</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Dynamic regulation of nucleosome positioning in the human genome</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">887</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20regulation%20of%20nucleosome%20positioning%20in%20the%20human%20genome&amp;author=D.E.%20Schones&amp;author=K.%20Cui&amp;author=S.%20Cuddapah&amp;publication_year=2008&amp;journal=Cell&amp;volume=132&amp;pages=887-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.02.022" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.02.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.02.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18329373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Duan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Analysis of nucleosome positioning determined by DNA helix curvature in the human genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">72</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20nucleosome%20positioning%20determined%20by%20DNA%20helix%20curvature%20in%20the%20human%20genome&amp;author=H.%20Liu&amp;author=X.%20Duan&amp;author=S.%20Yu&amp;author=X.%20Sun&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=72-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-72" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21269520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ninaber</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Goodfellow</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">DNA conformation and dynamics</div>, <div class="source">Radiat, Environ. Biophys.</div> , <div class="year">1999</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">23</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20conformation%20and%20dynamics&amp;author=A.%20Ninaber&amp;author=J.M.%20Goodfellow&amp;publication_year=1999&amp;journal=Radiat%2C%20Environ.%20Biophys.&amp;volume=38&amp;pages=23-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004110050134" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004110050134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004110050134"> </span></div> </div></div></div></div></div><div content-id="dsr043c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yanao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Oshikawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Elastic origin of chiral selection in DNA wrapping</div>, <div class="source">Phys. Rev. E</div> , <div class="year">2008</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">1</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elastic%20origin%20of%20chiral%20selection%20in%20DNA%20wrapping&amp;author=T.%20Yanao&amp;author=K.%20Oshikawa&amp;publication_year=2008&amp;journal=Phys.%20Rev.%20E&amp;volume=77&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1103/PhysRevE.77.021904" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1103%2FPhysRevE.77.021904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1103%2FPhysRevE.77.021904"> </span></div> </div></div></div></div></div><div content-id="dsr043c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mavrich</div> <div class="given-names">T.N.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ioshikes</div> <div class="given-names">I.P.</div></div>, et al. </div><div class="article-title">Nucleosome organization in the <em>Drosophila</em> genome</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">453</div> (pg. <div class="fpage">358</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20organization%20in%20the%20Drosophila%20genome&amp;author=T.N.%20Mavrich&amp;author=C.%20Jiang&amp;author=I.P.%20Ioshikes&amp;publication_year=2008&amp;journal=Nature&amp;volume=453&amp;pages=358-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06929" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18408708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ponts</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">E.Y.</div></div>,  <div class="name"><div class="surname">Prudhomme</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Nucleosome landscape and control of transcription in the human malaria parasite</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">228</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20landscape%20and%20control%20of%20transcription%20in%20the%20human%20malaria%20parasite&amp;author=N.%20Ponts&amp;author=E.Y.%20Harris&amp;author=J.%20Prudhomme&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=228-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.101063.109" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.101063.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.101063.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20054063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tillo</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">T.R.</div></div>. </div><div class="article-title">G + C content dominates intrinsic nucleosome occupancy</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">442</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=G%20%2B%20C%20content%20dominates%20intrinsic%20nucleosome%20occupancy&amp;author=D.%20Tillo&amp;author=T.R.%20Hughes&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-442" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20028554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tillo</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kaplan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">I.K.</div></div>, et al. </div><div class="article-title">High nucleosome occupancy is encoded at human regulatory sequences</div>, <div class="source">PLoS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e9129</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20nucleosome%20occupancy%20is%20encoded%20at%20human%20regulatory%20sequences&amp;author=D.%20Tillo&amp;author=N.%20Kaplan&amp;author=I.K.%20Moore&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e9129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009129" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0009129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20161746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Sequence signature of nucleosome positioning in <em>Caenorhabditis elegans</em></div>, <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">92</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20signature%20of%20nucleosome%20positioning%20in%20Caenorhabditis%20elegans&amp;author=K.%20Chen&amp;author=L.%20Wang&amp;author=M.%20Yang&amp;publication_year=2010&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=8&amp;pages=92-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1672-0229(10)60010-1" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1672-0229(10)60010-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1672-0229(10)60010-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20691394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsr043c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsr043c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Valouev</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Boyd</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Fire</div> <div class="given-names">A.Z.</div></div>,  <div class="name"><div class="surname">Sidow</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Determinants of nucleosome organization in primary human cells</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">474</div> (pg. <div class="fpage">516</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinants%20of%20nucleosome%20organization%20in%20primary%20human%20cells&amp;author=A.%20Valouev&amp;author=S.M.%20Johnson&amp;author=S.D.%20Boyd&amp;author=C.L.%20Smith&amp;author=A.Z.%20Fire&amp;author=A.%20Sidow&amp;publication_year=2011&amp;journal=Nature&amp;volume=474&amp;pages=516-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10002" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21602827" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=486167&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">8</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsr043"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="486167"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f1%2f81%2f486167">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F1%2F81%2F486167" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/486167'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Preferential Nucleosome Occupancy at High Values of DNA Helical Rise" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f81%2f486167">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

