

<!DOCTYPE html>
<html lang="en">

<head><title>
	Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis | DNA Research | Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/05a7cb21266cf8620ab7a7401900ccee49ae8478/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/7f205a0b34174418576911868d654b544c0687d4/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/8b6babff47b8a9793f37036fd1b1a3ad41d38423/UI/app/vendor/jquery-1.8.3.min.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/1/1/502776","@type":"ScholarlyArticle","name":"Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis","datePublished":"2007-03-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm002","keywords":["array CGH","molecular cytogenetics","microdeletion","copy-number variation","power analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/1/1/502776","author":[{"name":"Hehir-Kwa, Jayne Y.","affiliation":"Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences ,  Radboud University Nijmegen Medical Centre ,  Nijmegen ,  The Netherlands","@type":"Person"},{"name":"Egmont-Petersen, Michael","affiliation":"Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences ,  Radboud University Nijmegen Medical Centre ,  Nijmegen ,  The Netherlands","@type":"Person"},{"name":"Janssen, Irene M.","affiliation":"Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences ,  Radboud University Nijmegen Medical Centre ,  Nijmegen ,  The Netherlands","@type":"Person"},{"name":"Smeets, Dominique","affiliation":"Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences ,  Radboud University Nijmegen Medical Centre ,  Nijmegen ,  The Netherlands","@type":"Person"},{"name":"van Kessel, Ad Geurts","affiliation":"Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences ,  Radboud University Nijmegen Medical Centre ,  Nijmegen ,  The Netherlands","@type":"Person"},{"name":"Veltman, Joris A.","affiliation":"Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences ,  Radboud University Nijmegen Medical Centre ,  Nijmegen ,  The Netherlands","@type":"Person"}],"description":"Recently, comparative genomic hybridization onto bacterial artificial chromosome (BAC) arrays (array-based comparative genomic hybridization) has proved to be successful for the detection of submicroscopic DNA copy-number variations in health and disease. Technological improvements to achieve a higher resolution have resulted in the generation of additional microarray platforms encompassing larger numbers of shorter DNA targets (oligonucleotides). Here, we present a novel method to estimate the ability of a microarray to detect genomic copy-number variations of different sizes and types (i.e. deletions or duplications). We applied our method, which is based on statistical power analysis, to four widely used high-density genomic microarray platforms. By doing so, we found that the high-density oligonucleotide platforms are superior to the BAC platform for the genome-wide detection of copy-number variations smaller than 1 Mb. The capacity to reliably detect single copy-number variations below 100 kb, however, appeared to be limited for all platforms tested. In addition, our analysis revealed an unexpected platform-dependent difference in sensitivity to detect a single copy-number loss and a single copy-number gain. These analyses provide a first objective insight into the true capacities and limitations of different genomic microarrays to detect and define DNA copy-number variations.","pageStart":"1","pageEnd":"11","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/1/0/m_dnaresearch14_1.cover.gif?Expires=1572216962&Signature=RqFyLLn4c8MPXD6CTwFSrZ1eKiCslrK2uc90kRhgzeDFVwX~5iGhg6BRkogFP4rPxbsCxk1zwACSIFszOAJhs5fZ9FbP12VmE3aUMi1GprQOz8lbKvrMsKsp~QPKqVDz7~2aGbe9QAU3VTw7L0wAjp1Mn2ZkhmxPRiUxx9vJM7dTmNNmYqHCrOQ2BgeacffIg~vmIzt6kBfQHvgGytyCyi0jhNBaV4anNfQlD5bHJlG~vvn5tLLBKvpvqVRSP2-c8E1RHqjFNov~BPtsoh1gA7ND9iMqZBF96XpxIc~6pDKe4XtrihJdg9qfXZ5DuUa~rsKF~8Vvs22o44DNCmtAHw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/1/0/m_dnaresearch14_1.cover.gif?Expires=1572216962&Signature=RqFyLLn4c8MPXD6CTwFSrZ1eKiCslrK2uc90kRhgzeDFVwX~5iGhg6BRkogFP4rPxbsCxk1zwACSIFszOAJhs5fZ9FbP12VmE3aUMi1GprQOz8lbKvrMsKsp~QPKqVDz7~2aGbe9QAU3VTw7L0wAjp1Mn2ZkhmxPRiUxx9vJM7dTmNNmYqHCrOQ2BgeacffIg~vmIzt6kBfQHvgGytyCyi0jhNBaV4anNfQlD5bHJlG~vvn5tLLBKvpvqVRSP2-c8E1RHqjFNov~BPtsoh1gA7ND9iMqZBF96XpxIc~6pDKe4XtrihJdg9qfXZ5DuUa~rsKF~8Vvs22o44DNCmtAHw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis | DNA Research | Oxford Academic" />
    <meta property="og:description" content="Recently, comparative genomic hybridization onto bacterial artificial chromosome (BAC) arrays (array-based comparative genomic hybridization) has proved to be successful for the detection of submicroscopic DNA copy-number variations in health and disease. Technological improvements to achieve a higher resolution have resulted in the generation of additional microarray platforms encompassing larger numbers of shorter DNA targets (oligonucleotides). Here, we present a novel method to estimate the ability of a microarray to detect genomic copy-number variations of different sizes and types (i.e. deletions or duplications). We applied our method, which is based on statistical power analysis, to four widely used high-density genomic microarray platforms. By doing so, we found that the high-density oligonucleotide platforms are superior to the BAC platform for the genome-wide detection of copy-number variations smaller than 1 Mb. The capacity to reliably detect single copy-number variations below 100 kb, however, appeared to be limited for all platforms tested. In addition, our analysis revealed an unexpected platform-dependent difference in sensitivity to detect a single copy-number loss and a single copy-number gain. These analyses provide a first objective insight into the true capacities and limitations of different genomic microarrays to detect and define DNA copy-number variations." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/1/1/502776" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=1572216962&amp;Signature=X1DfbPxQS-SxhtxKTSl~Tmeq2IvID45rl4nauFfI659zNSrMIJm26RGBpZIFuB1XI-GAixqwCXtaDCovYyy3gDnXPYkEHpbqve02vLt~17NC4Di8uyTM8F2NzxRDiumx4PSdDpSq~npN2PniQHfkmny2KiyGR6HljsGLY-241alibp5v2icsP0CmdT0jujtH4HYWl~eHXSvY0lBMUC9b0RxGo-SCzuAYV7PkptofJxdjXL8Z9YfUaHygKeTg6~kqj53UVp01mTWB3Y4vGOlQfKaxybUEe8eMSU~udLhxsaTAtxa2hfdCaxQt-gH-ypbOydT3bB33tYcMmtfV8LzNSg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" />
    <meta property="og:updated_time" content="2007-03-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hehir-Kwa, Jayne Y." /><meta name="citation_author_institution" content="Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands" /><meta name="citation_author" content="Egmont-Petersen, Michael" /><meta name="citation_author_institution" content="Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands" /><meta name="citation_author" content="Janssen, Irene M." /><meta name="citation_author_institution" content="Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands" /><meta name="citation_author" content="Smeets, Dominique" /><meta name="citation_author_institution" content="Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands" /><meta name="citation_author" content="van Kessel, Ad Geurts" /><meta name="citation_author_institution" content="Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands" /><meta name="citation_author" content="Veltman, Joris A." /><meta name="citation_author_institution" content="Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands" /><meta name="citation_title" content="Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis" /><meta name="citation_firstpage" content="1" /><meta name="citation_lastpage" content="11" /><meta name="citation_doi" content="10.1093/dnares/dsm002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Comparative genomic hybridization; Annu. Rev. Genomics Hum. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=331-354; " /><meta name="citation_reference" content="citation_title=The new cytogenetics: blurring the boundaries with molecular biology; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=782-792; " /><meta name="citation_reference" content="citation_title=Assembly of microarrays for genome-wide measurement of DNA copy number; Nat. Genet.;  citation_year=2001;  citation_volume=29;  citation_pages=263-264; " /><meta name="citation_reference" content="citation_title=DNA microarrays for comparative genomic hybridization based on DOP-PCR amplification of BAC and PAC clones; Genes Chromosomes Cancer;  citation_year=2003;  citation_volume=36;  citation_pages=361-374; " /><meta name="citation_reference" content="citation_title=Array-based comparative genomic hybridization for the genome-wide detection of submicroscopic chromosomal abnormalities; Am. J. Hum. Genet.;  citation_year=2003;  citation_volume=73;  citation_pages=1261-1270; " /><meta name="citation_reference" content="citation_title=A tiling resolution DNA microarray with complete coverage of the human genome; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=299-303; " /><meta name="citation_reference" content="citation_title=A set of BAC clones spanning the human genome; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=3651-3660; " /><meta name="citation_reference" content="citation_title=Array CGH technologies and their applications to cancer genomes; Chromosome Res.;  citation_year=2005;  citation_volume=13;  citation_pages=237-248; " /><meta name="citation_reference" content="citation_title=Distinct genomic profiles in hereditary breast tumors identified by array-based comparative genomic hybridization; Cancer Res.;  citation_year=2005;  citation_volume=65;  citation_pages=7612-7621; " /><meta name="citation_reference" content="citation_title=Array comparative genomic hybridization and its applications in cancer; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=S11-S17; " /><meta name="citation_reference" content="citation_title=Mutations in a new member of the chromodomain gene family cause CHARGE syndrome; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=955-957; " /><meta name="citation_reference" content="citation_title=Identification of disease genes by whole genome CGH arrays; Hum. Mol. Genet.;  citation_year=2005;  citation_volume=14;  citation_pages=R215-R223; " /><meta name="citation_reference" content="citation_title=A new chromosome 17q21.31 microdeletion syndrome associated with a common inversion polymorphism; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=999-1001; " /><meta name="citation_reference" content="citation_title=Microdeletion encompassing MAPT at chromosome 17q21.3 is associated with developmental delay and learning disability; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=1032-1037; " /><meta name="citation_reference" content="citation_title=Discovery of previously unidentified genomic disorders from the duplication architecture of the human genome; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=1038-1042; " /><meta name="citation_reference" content="citation_title=Microarray based comparative genomic hybridisation (array-CGH) detects submicroscopic chromosomal deletions and duplications in patients with learning disability/mental retardation and dysmorphic features; J. Med. Genet.;  citation_year=2004;  citation_volume=41;  citation_pages=241-248; " /><meta name="citation_reference" content="citation_title=Diagnostic genome profiling in mental retardation; Am. J. Hum. Genet.;  citation_year=2005;  citation_volume=77;  citation_pages=606-616; " /><meta name="citation_reference" content="citation_title=Prenatal detection of unbalanced chromosomal rearrangements by array-CGH; J. Med. Genet.;  citation_year=2006;  citation_volume=43;  citation_pages=353-361; " /><meta name="citation_reference" content="citation_title=Evolution of prenatal genetics: from point mutation testing to chromosomal microarray analysis; Expert Rev. Mol. Diagn.;  citation_year=2005;  citation_volume=5;  citation_pages=883-892; " /><meta name="citation_reference" content="citation_title=Detection of large-scale variation in the human genome; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=949-951; " /><meta name="citation_reference" content="citation_title=Large-scale copy number polymorphism in the human genome; Science;  citation_year=2004;  citation_volume=305;  citation_pages=525-528; " /><meta name="citation_reference" content="citation_title=Segmental duplications and copy-number variation in the human genome; Am. J. Hum. Genet.;  citation_year=2005;  citation_volume=77;  citation_pages=78-88; " /><meta name="citation_reference" content="citation_title=Structural variation in the human genome; Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=85-97; " /><meta name="citation_reference" content="citation_title=Comparative genomic hybridization using oligonucleotide microarrays and total genomic DNA; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=17765-17770; " /><meta name="citation_reference" content="citation_title=Human and mouse oligonucleotide-based array CGH; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=e192" /><meta name="citation_reference" content="citation_title=Analysis of chromosome breakpoints in neuroblastoma at sub-kilobase resolution using fine-tiling oligonucleotide array CGH; Genes Chromosomes Cancer;  citation_year=2005;  citation_volume=44;  citation_pages=305-319; " /><meta name="citation_reference" content="citation_title=Whole genome DNA copy number changes identified by high-density oligonucleotide arrays; Hum. Genomics;  citation_year=2004;  citation_volume=1;  citation_pages=287-299; " /><meta name="citation_reference" content="citation_title=A robust algorithm for copy number detection using high-density oligonucleotide single-nucleotide polymorphism genotyping arrays; Cancer Res.;  citation_year=2005;  citation_volume=65;  citation_pages=6071-6079; " /><meta name="citation_reference" content="citation_title=High-resolution identification of chromosomal abnormalities using oligonucleotide arrays containing 116,204 SNPs; Am. J. Hum. Genet.;  citation_year=2005;  citation_volume=77;  citation_pages=709-726; " /><meta name="citation_reference" content="citation_title=Oligonucleotide microarray analysis of genomic imbalance in children with mental retardation; Am. J. Hum. Genet.;  citation_year=2006;  citation_volume=79;  citation_pages=500-513; " /><meta name="citation_reference" content="citation_title=High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1136-1148; " /><meta name="citation_reference" content="citation_title=Standardizing global gene expression analysis between laboratories and across platforms; Nat. Methods;  citation_year=2005;  citation_volume=2;  citation_pages=351-356; " /><meta name="citation_reference" content="citation_title=Experimental comparison and cross-validation of the Affymetrix and Illumina gene expression analysis platforms; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5914-5923; " /><meta name="citation_reference" content="citation_title=Multiple-laboratory comparison of microarray platforms; Nat. Methods;  citation_year=2005;  citation_volume=2;  citation_pages=345-350; " /><meta name="citation_reference" content="citation_title=Independence and reproducibility across microarray platforms; Nat. Methods;  citation_year=2005;  citation_volume=2;  citation_pages=337-344; " /><meta name="citation_reference" content=" citation_edition=2nd ed.; citation_publisher=McGraw-Hill, New York; Nonparametric Statistics for the Behavioral Sciences;  citation_year=1988; " /><meta name="citation_reference" content="citation_title=Array comparative genomic hybridization analysis of genomic alterations in breast cancer subtypes; Cancer Res.;  citation_year=2004;  citation_volume=64;  citation_pages=8541-8549; " /><meta name="citation_reference" content="citation_title=Errors of the second kind in testing “Student's” hypothesis; J. Am. Stat. Assoc.;  citation_year=1936;  citation_volume=31;  citation_pages=318-326; " /><meta name="citation_reference" content="citation_title=Applications of the non-central t-distribution; Biometrika;  citation_year=1940;  citation_volume=31;  citation_pages=362-389; " /><meta name="citation_reference" content="citation_title=Circular binary segmentation for the analysis of array-based DNA copy number data; Biostatistics;  citation_year=2004;  citation_volume=5;  citation_pages=557-572; " /><meta name="citation_reference" content="citation_title=Hidden Markov models approach to the analysis of array CGH data; J. Multivariate Anal.;  citation_year=2004;  citation_volume=90;  citation_issue=1;  citation_pages=132-153; " /><meta name="citation_reference" content="citation_title=STAC: a method for testing the significance of DNA copy number aberrations across multiple array-CGH experiments; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1149-1158; " /><meta name="citation_reference" content=" citation_edition=2nd ed.; citation_publisher=Lawrence Erlbaum Associates, London; Statistical Power Analysis: A Simple and General Model for Traditional and Modern Hypothesis Tests;  citation_year=2004; " /><meta name="citation_reference" content="citation_title=Identification of frequent chromosome copy-number polymorphisms by use of high-resolution single-nucleotide-polymorphism arrays; Am. J. Hum. Genet.;  citation_year=2006;  citation_volume=78;  citation_pages=520-522; " /><meta name="citation_reference" content="citation_title=Denoising array-based comparative genomic hybridization data using wavelets; Biostatistics;  citation_year=2005;  citation_volume=6;  citation_pages=211-226; " /><meta name="citation_reference" content="citation_publisher=Springer, New York; Fundamentals of Modern Statistical Methods: Substantially Improving Power and Accuracy;  citation_year=2001; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/1/1/17462480/dsm002.pdf" /><meta name="description" content="Abstract.  Recently, comparative genomic hybridization onto bacterial artificial chromosome (BAC) arrays (array-based comparative genomic hybridization) has pro" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/1/1/502776" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/1/1/502776" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2777753,2796125&authnips=128.235.8.16&doi=10.1093/dnares/dsm002';
    </script>
</head>

<body data-sitename="dnaresearch" class="off-canvas pg_article" theme-dnaresearch>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./502776" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjUzMDk3NzA2D2QWAmYPZBYCZg9kFgRmD2QWAgIPD2QWAgIBD2QWAmYPFgIeBFRleHQF1mc8bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhlaGlyLUt3YSwgSmF5bmUgWS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEh1bWFuIEdlbmV0aWNzLCBOaWptZWdlbiBDZW50cmUgZm9yIE1vbGVjdWxhciBMaWZlIFNjaWVuY2VzLCBSYWRib3VkIFVuaXZlcnNpdHkgTmlqbWVnZW4gTWVkaWNhbCBDZW50cmUsIE5pam1lZ2VuLCBUaGUgTmV0aGVybGFuZHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJFZ21vbnQtUGV0ZXJzZW4sIE1pY2hhZWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEh1bWFuIEdlbmV0aWNzLCBOaWptZWdlbiBDZW50cmUgZm9yIE1vbGVjdWxhciBMaWZlIFNjaWVuY2VzLCBSYWRib3VkIFVuaXZlcnNpdHkgTmlqbWVnZW4gTWVkaWNhbCBDZW50cmUsIE5pam1lZ2VuLCBUaGUgTmV0aGVybGFuZHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKYW5zc2VuLCBJcmVuZSBNLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgSHVtYW4gR2VuZXRpY3MsIE5pam1lZ2VuIENlbnRyZSBmb3IgTW9sZWN1bGFyIExpZmUgU2NpZW5jZXMsIFJhZGJvdWQgVW5pdmVyc2l0eSBOaWptZWdlbiBNZWRpY2FsIENlbnRyZSwgTmlqbWVnZW4sIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNtZWV0cywgRG9taW5pcXVlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5ldGljcywgTmlqbWVnZW4gQ2VudHJlIGZvciBNb2xlY3VsYXIgTGlmZSBTY2llbmNlcywgUmFkYm91ZCBVbml2ZXJzaXR5IE5pam1lZ2VuIE1lZGljYWwgQ2VudHJlLCBOaWptZWdlbiwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0idmFuIEtlc3NlbCwgQWQgR2V1cnRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5ldGljcywgTmlqbWVnZW4gQ2VudHJlIGZvciBNb2xlY3VsYXIgTGlmZSBTY2llbmNlcywgUmFkYm91ZCBVbml2ZXJzaXR5IE5pam1lZ2VuIE1lZGljYWwgQ2VudHJlLCBOaWptZWdlbiwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmVsdG1hbiwgSm9yaXMgQS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEh1bWFuIEdlbmV0aWNzLCBOaWptZWdlbiBDZW50cmUgZm9yIE1vbGVjdWxhciBMaWZlIFNjaWVuY2VzLCBSYWRib3VkIFVuaXZlcnNpdHkgTmlqbWVnZW4gTWVkaWNhbCBDZW50cmUsIE5pam1lZ2VuLCBUaGUgTmV0aGVybGFuZHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS13aWRlIENvcHkgTnVtYmVyIFByb2ZpbGluZyBvbiBIaWdoLWRlbnNpdHkgQmFjdGVyaWFsIEFydGlmaWNpYWwgQ2hyb21vc29tZXMsIFNpbmdsZS1udWNsZW90aWRlIFBvbHltb3JwaGlzbXMsIGFuZCBPbGlnb251Y2xlb3RpZGUgTWljcm9hcnJheXM6IEEgUGxhdGZvcm0gQ29tcGFyaXNvbiBiYXNlZCBvbiBTdGF0aXN0aWNhbCBQb3dlciBBbmFseXNpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNtMDAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDcvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljIGh5YnJpZGl6YXRpb247IEFubnUuIFJldi4gR2Vub21pY3MgSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMzMS0zNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZXcgY3l0b2dlbmV0aWNzOiBibHVycmluZyB0aGUgYm91bmRhcmllcyB3aXRoIG1vbGVjdWxhciBiaW9sb2d5OyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03ODItNzkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NlbWJseSBvZiBtaWNyb2FycmF5cyBmb3IgZ2Vub21lLXdpZGUgbWVhc3VyZW1lbnQgb2YgRE5BIGNvcHkgbnVtYmVyOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI2My0yNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtaWNyb2FycmF5cyBmb3IgY29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uIGJhc2VkIG9uIERPUC1QQ1IgYW1wbGlmaWNhdGlvbiBvZiBCQUMgYW5kIFBBQyBjbG9uZXM7IEdlbmVzIENocm9tb3NvbWVzIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTM2MS0zNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFycmF5LWJhc2VkIGNvbXBhcmF0aXZlIGdlbm9taWMgaHlicmlkaXphdGlvbiBmb3IgdGhlIGdlbm9tZS13aWRlIGRldGVjdGlvbiBvZiBzdWJtaWNyb3Njb3BpYyBjaHJvbW9zb21hbCBhYm5vcm1hbGl0aWVzOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz0xMjYxLTEyNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGlsaW5nIHJlc29sdXRpb24gRE5BIG1pY3JvYXJyYXkgd2l0aCBjb21wbGV0ZSBjb3ZlcmFnZSBvZiB0aGUgaHVtYW4gZ2Vub21lOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2V0IG9mIEJBQyBjbG9uZXMgc3Bhbm5pbmcgdGhlIGh1bWFuIGdlbm9tZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzY1MS0zNjYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcnJheSBDR0ggdGVjaG5vbG9naWVzIGFuZCB0aGVpciBhcHBsaWNhdGlvbnMgdG8gY2FuY2VyIGdlbm9tZXM7IENocm9tb3NvbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIzNy0yNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RpbmN0IGdlbm9taWMgcHJvZmlsZXMgaW4gaGVyZWRpdGFyeSBicmVhc3QgdHVtb3JzIGlkZW50aWZpZWQgYnkgYXJyYXktYmFzZWQgY29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uOyBDYW5jZXIgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTc2MTItNzYyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJyYXkgY29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uIGFuZCBpdHMgYXBwbGljYXRpb25zIGluIGNhbmNlcjsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1TMTEtUzE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbnMgaW4gYSBuZXcgbWVtYmVyIG9mIHRoZSBjaHJvbW9kb21haW4gZ2VuZSBmYW1pbHkgY2F1c2UgQ0hBUkdFIHN5bmRyb21lOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTk1NS05NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGRpc2Vhc2UgZ2VuZXMgYnkgd2hvbGUgZ2Vub21lIENHSCBhcnJheXM7IEh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMjE1LVIyMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IGNocm9tb3NvbWUgMTdxMjEuMzEgbWljcm9kZWxldGlvbiBzeW5kcm9tZSBhc3NvY2lhdGVkIHdpdGggYSBjb21tb24gaW52ZXJzaW9uIHBvbHltb3JwaGlzbTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz05OTktMTAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9kZWxldGlvbiBlbmNvbXBhc3NpbmcgTUFQVCBhdCBjaHJvbW9zb21lIDE3cTIxLjMgaXMgYXNzb2NpYXRlZCB3aXRoIGRldmVsb3BtZW50YWwgZGVsYXkgYW5kIGxlYXJuaW5nIGRpc2FiaWxpdHk7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTAzMi0xMDM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3Zlcnkgb2YgcHJldmlvdXNseSB1bmlkZW50aWZpZWQgZ2Vub21pYyBkaXNvcmRlcnMgZnJvbSB0aGUgZHVwbGljYXRpb24gYXJjaGl0ZWN0dXJlIG9mIHRoZSBodW1hbiBnZW5vbWU7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTAzOC0xMDQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2FycmF5IGJhc2VkIGNvbXBhcmF0aXZlIGdlbm9taWMgaHlicmlkaXNhdGlvbiAoYXJyYXktQ0dIKSBkZXRlY3RzIHN1Ym1pY3Jvc2NvcGljIGNocm9tb3NvbWFsIGRlbGV0aW9ucyBhbmQgZHVwbGljYXRpb25zIGluIHBhdGllbnRzIHdpdGggbGVhcm5pbmcgZGlzYWJpbGl0eS9tZW50YWwgcmV0YXJkYXRpb24gYW5kIGR5c21vcnBoaWMgZmVhdHVyZXM7IEouIE1lZC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MjQxLTI0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlhZ25vc3RpYyBnZW5vbWUgcHJvZmlsaW5nIGluIG1lbnRhbCByZXRhcmRhdGlvbjsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc3OyAgY2l0YXRpb25fcGFnZXM9NjA2LTYxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlbmF0YWwgZGV0ZWN0aW9uIG9mIHVuYmFsYW5jZWQgY2hyb21vc29tYWwgcmVhcnJhbmdlbWVudHMgYnkgYXJyYXktQ0dIOyBKLiBNZWQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTM1My0zNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBwcmVuYXRhbCBnZW5ldGljczogZnJvbSBwb2ludCBtdXRhdGlvbiB0ZXN0aW5nIHRvIGNocm9tb3NvbWFsIG1pY3JvYXJyYXkgYW5hbHlzaXM7IEV4cGVydCBSZXYuIE1vbC4gRGlhZ24uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz04ODMtODkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgbGFyZ2Utc2NhbGUgdmFyaWF0aW9uIGluIHRoZSBodW1hbiBnZW5vbWU7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9OTQ5LTk1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgY29weSBudW1iZXIgcG9seW1vcnBoaXNtIGluIHRoZSBodW1hbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA1OyAgY2l0YXRpb25fcGFnZXM9NTI1LTUyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VnbWVudGFsIGR1cGxpY2F0aW9ucyBhbmQgY29weS1udW1iZXIgdmFyaWF0aW9uIGluIHRoZSBodW1hbiBnZW5vbWU7IEFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTc4LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIHZhcmlhdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz04NS05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uIHVzaW5nIG9saWdvbnVjbGVvdGlkZSBtaWNyb2FycmF5cyBhbmQgdG90YWwgZ2Vub21pYyBETkE7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE3NzY1LTE3NzcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBhbmQgbW91c2Ugb2xpZ29udWNsZW90aWRlLWJhc2VkIGFycmF5IENHSDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9ZTE5MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNocm9tb3NvbWUgYnJlYWtwb2ludHMgaW4gbmV1cm9ibGFzdG9tYSBhdCBzdWIta2lsb2Jhc2UgcmVzb2x1dGlvbiB1c2luZyBmaW5lLXRpbGluZyBvbGlnb251Y2xlb3RpZGUgYXJyYXkgQ0dIOyBHZW5lcyBDaHJvbW9zb21lcyBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0zMDUtMzE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBnZW5vbWUgRE5BIGNvcHkgbnVtYmVyIGNoYW5nZXMgaWRlbnRpZmllZCBieSBoaWdoLWRlbnNpdHkgb2xpZ29udWNsZW90aWRlIGFycmF5czsgSHVtLiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9Mjg3LTI5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2J1c3QgYWxnb3JpdGhtIGZvciBjb3B5IG51bWJlciBkZXRlY3Rpb24gdXNpbmcgaGlnaC1kZW5zaXR5IG9saWdvbnVjbGVvdGlkZSBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZ2Vub3R5cGluZyBhcnJheXM7IENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9NjA3MS02MDc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gaWRlbnRpZmljYXRpb24gb2YgY2hyb21vc29tYWwgYWJub3JtYWxpdGllcyB1c2luZyBvbGlnb251Y2xlb3RpZGUgYXJyYXlzIGNvbnRhaW5pbmcgMTE2LDIwNCBTTlBzOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz03MDktNzI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbGlnb251Y2xlb3RpZGUgbWljcm9hcnJheSBhbmFseXNpcyBvZiBnZW5vbWljIGltYmFsYW5jZSBpbiBjaGlsZHJlbiB3aXRoIG1lbnRhbCByZXRhcmRhdGlvbjsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9NTAwLTUxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIGdlbm9taWMgcHJvZmlsaW5nIG9mIGNocm9tb3NvbWFsIGFiZXJyYXRpb25zIHVzaW5nIEluZmluaXVtIHdob2xlLWdlbm9tZSBnZW5vdHlwaW5nOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExMzYtMTE0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhbmRhcmRpemluZyBnbG9iYWwgZ2VuZSBleHByZXNzaW9uIGFuYWx5c2lzIGJldHdlZW4gbGFib3JhdG9yaWVzIGFuZCBhY3Jvc3MgcGxhdGZvcm1zOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTM1MS0zNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBjb21wYXJpc29uIGFuZCBjcm9zcy12YWxpZGF0aW9uIG9mIHRoZSBBZmZ5bWV0cml4IGFuZCBJbGx1bWluYSBnZW5lIGV4cHJlc3Npb24gYW5hbHlzaXMgcGxhdGZvcm1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz01OTE0LTU5MjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlLWxhYm9yYXRvcnkgY29tcGFyaXNvbiBvZiBtaWNyb2FycmF5IHBsYXRmb3JtczsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNDUtMzUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmRlcGVuZGVuY2UgYW5kIHJlcHJvZHVjaWJpbGl0eSBhY3Jvc3MgbWljcm9hcnJheSBwbGF0Zm9ybXM7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MzM3LTM0NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249Mm5kIGVkLjsgY2l0YXRpb25fcHVibGlzaGVyPU1jR3Jhdy1IaWxsLCBOZXcgWW9yazsgTm9ucGFyYW1ldHJpYyBTdGF0aXN0aWNzIGZvciB0aGUgQmVoYXZpb3JhbCBTY2llbmNlczsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJyYXkgY29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uIGFuYWx5c2lzIG9mIGdlbm9taWMgYWx0ZXJhdGlvbnMgaW4gYnJlYXN0IGNhbmNlciBzdWJ0eXBlczsgQ2FuY2VyIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz04NTQxLTg1NDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVycm9ycyBvZiB0aGUgc2Vjb25kIGtpbmQgaW4gdGVzdGluZyDigJxTdHVkZW50J3PigJ0gaHlwb3RoZXNpczsgSi4gQW0uIFN0YXQuIEFzc29jLjsgIGNpdGF0aW9uX3llYXI9MTkzNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTMxOC0zMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9ucyBvZiB0aGUgbm9uLWNlbnRyYWwgdC1kaXN0cmlidXRpb247IEJpb21ldHJpa2E7ICBjaXRhdGlvbl95ZWFyPTE5NDA7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNjItMzg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBiaW5hcnkgc2VnbWVudGF0aW9uIGZvciB0aGUgYW5hbHlzaXMgb2YgYXJyYXktYmFzZWQgRE5BIGNvcHkgbnVtYmVyIGRhdGE7IEJpb3N0YXRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTU1Ny01NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZGRlbiBNYXJrb3YgbW9kZWxzIGFwcHJvYWNoIHRvIHRoZSBhbmFseXNpcyBvZiBhcnJheSBDR0ggZGF0YTsgSi4gTXVsdGl2YXJpYXRlIEFuYWwuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25faXNzdWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTEzMi0xNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNUQUM6IGEgbWV0aG9kIGZvciB0ZXN0aW5nIHRoZSBzaWduaWZpY2FuY2Ugb2YgRE5BIGNvcHkgbnVtYmVyIGFiZXJyYXRpb25zIGFjcm9zcyBtdWx0aXBsZSBhcnJheS1DR0ggZXhwZXJpbWVudHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE0OS0xMTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj0ybmQgZWQuOyBjaXRhdGlvbl9wdWJsaXNoZXI9TGF3cmVuY2UgRXJsYmF1bSBBc3NvY2lhdGVzLCBMb25kb247IFN0YXRpc3RpY2FsIFBvd2VyIEFuYWx5c2lzOiBBIFNpbXBsZSBhbmQgR2VuZXJhbCBNb2RlbCBmb3IgVHJhZGl0aW9uYWwgYW5kIE1vZGVybiBIeXBvdGhlc2lzIFRlc3RzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBmcmVxdWVudCBjaHJvbW9zb21lIGNvcHktbnVtYmVyIHBvbHltb3JwaGlzbXMgYnkgdXNlIG9mIGhpZ2gtcmVzb2x1dGlvbiBzaW5nbGUtbnVjbGVvdGlkZS1wb2x5bW9ycGhpc20gYXJyYXlzOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz01MjAtNTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZW5vaXNpbmcgYXJyYXktYmFzZWQgY29tcGFyYXRpdmUgZ2Vub21pYyBoeWJyaWRpemF0aW9uIGRhdGEgdXNpbmcgd2F2ZWxldHM7IEJpb3N0YXRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0yMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgTmV3IFlvcms7IEZ1bmRhbWVudGFscyBvZiBNb2Rlcm4gU3RhdGlzdGljYWwgTWV0aG9kczogU3Vic3RhbnRpYWxseSBJbXByb3ZpbmcgUG93ZXIgYW5kIEFjY3VyYWN5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvMS8xLzE3NDYyNDgwL2RzbTAwMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUmVjZW50bHksIGNvbXBhcmF0aXZlIGdlbm9taWMgaHlicmlkaXphdGlvbiBvbnRvIGJhY3RlcmlhbCBhcnRpZmljaWFsIGNocm9tb3NvbWUgKEJBQykgYXJyYXlzIChhcnJheS1iYXNlZCBjb21wYXJhdGl2ZSBnZW5vbWljIGh5YnJpZGl6YXRpb24pIGhhcyBwcm8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwMjc3NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIEFsbCBGaWd1cmVzZGQtWsYx1gGKS4jkS09+046upNDmcA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="3000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="basic" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="basic" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f1%2f502776" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f1%2f502776" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">
            
                <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"/>
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"/>
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933326">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933323">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933327">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933328">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933324">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933329">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933330">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933331">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933332">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933333">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933334">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-933325">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933335">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933336">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933337">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933338">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933339">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933340">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-933341">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/1/0/m_dnaresearch14_1.cover.gif?Expires=1509238562&amp;Signature=abtm4q0~8vRL4oWjyiJuFLIFA01ddMIidjDbR67QWBhTpgYidVYHUYOIGMTena5eqb7GuXkGJ9w-w7gfrmWq0XgyWgZZVRSp6tRM13pGwcrvOMJzEPaY-LV1wTD1wiZaa8GQ2hBzvYEwQimGHlsozQJVHkc1qxB80FMwEbsjaUIE7D2x~47Xw9Yg~gPH0dYRar5uGSqf8D4xbIBFyKkjp~swXKVKFoZHu-LOnL5AYuwYk4NlOMbOaK4slOoXTcYDF2VRLZpYYU8BHzL21CFnxj2yZMcbagIfxKjtqp9fUOPvw0NY7TeuvmpZClTmMHim7rIFwEItesvsuiB66TbmAA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/1">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87194279">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194279">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194281">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194281">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194285">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194285">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194311">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194311">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194328">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194328">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194336">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194336">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87194338">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87194338">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/14/1/13/503019">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Jayne Y. Hehir-Kwa</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jayne Y. Hehir-Kwa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences</div>, <div class="insititution">Radboud University Nijmegen Medical Centre</div>, <div class="addr-line">Nijmegen</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Jayne+Y.+Hehir-Kwa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hehir-Kwa J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Hehir-Kwa author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Michael Egmont-Petersen</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michael Egmont-Petersen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences</div>, <div class="insititution">Radboud University Nijmegen Medical Centre</div>, <div class="addr-line">Nijmegen</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Michael+Egmont-Petersen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Egmont-Petersen M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Egmont-Petersen author:M.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Irene M. Janssen</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Irene M. Janssen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences</div>, <div class="insititution">Radboud University Nijmegen Medical Centre</div>, <div class="addr-line">Nijmegen</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Irene+M.+Janssen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Janssen I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Janssen author:I.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Dominique Smeets</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dominique Smeets</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences</div>, <div class="insititution">Radboud University Nijmegen Medical Centre</div>, <div class="addr-line">Nijmegen</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Dominique+Smeets">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Smeets D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Smeets author:D.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Ad Geurts van Kessel</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ad Geurts van Kessel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences</div>, <div class="insititution">Radboud University Nijmegen Medical Centre</div>, <div class="addr-line">Nijmegen</div>, <div class="country">The Netherlands</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Ad+Geurts+van+Kessel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=van Kessel A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:van Kessel author:A.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Joris A. Veltman</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Joris A. Veltman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Human Genetics, Nijmegen Centre for Molecular Life Sciences</div>, <div class="insititution">Radboud University Nijmegen Medical Centre</div>, <div class="addr-line">Nijmegen</div>, <div class="country">The Netherlands</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +31-24-3614941. Fax. <div class="fax">+31-24-3668752</div>, E-mail: <a href="mailto:j.veltman@antrg.umcn.nl" target="_blank">j.veltman@antrg.umcn.nl</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/search-results?f_Authors=Joris+A.+Veltman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Veltman J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=&quot;author:Veltman author:J.&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 1, 1 January 2007, Pages 1–11, <a href='https://doi.org/10.1093/dnares/dsm002'>https://doi.org/10.1093/dnares/dsm002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 March 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 September 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">16 January 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="502776" href="/dnaresearch/article-pdf/14/1/1/17462480/dsm002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jayne Y. Hehir-Kwa, Michael Egmont-Petersen, Irene M. Janssen, Dominique Smeets, Ad Geurts van Kessel, Joris A. Veltman;  Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis, <em>DNA Research</em>, Volume 14, Issue 1, 1 January 2007, Pages 1–11, <a href='https://doi.org/10.1093/dnares/dsm002'>https://doi.org/10.1093/dnares/dsm002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/502776?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/502776?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/502776?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/502776?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/502776?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/502776?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/502776?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20Copy%20Number%20Profiling%20on%20High-density%20Bacterial%20Artificial%20Chromosomes%2C%20Single-nucleotide%20Polymorphisms%2C%20and%20Oligonucleotide%20Microarrays%3A%20A%20Platform%20Comparison%20based%20on%20Statistical%20Power%20Analysis&amp;publicationDate=2007-03-15&amp;volumeNum=14&amp;issueNum=1&amp;author=Hehir-Kwa%2C%20Jayne%20Y.%3B%20Egmont-Petersen%2C%20Michael&amp;startPage=1&amp;endPage=11&amp;contentId=10.1093%2Fdnares%2Fdsm002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=0&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f1%2f502776" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87194279 id="87194279" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Recently, comparative genomic hybridization onto bacterial artificial chromosome (BAC) arrays (array-based comparative genomic hybridization) has proved to be successful for the detection of submicroscopic DNA copy-number variations in health and disease. Technological improvements to achieve a higher resolution have resulted in the generation of additional microarray platforms encompassing larger numbers of shorter DNA targets (oligonucleotides). Here, we present a novel method to estimate the ability of a microarray to detect genomic copy-number variations of different sizes and types (i.e. deletions or duplications). We applied our method, which is based on statistical power analysis, to four widely used high-density genomic microarray platforms. By doing so, we found that the high-density oligonucleotide platforms are superior to the BAC platform for the genome-wide detection of copy-number variations smaller than 1 Mb. The capacity to reliably detect single copy-number variations below 100 kb, however, appeared to be limited for all platforms tested. In addition, our analysis revealed an unexpected platform-dependent difference in sensitivity to detect a single copy-number loss and a single copy-number gain. These analyses provide a first objective insight into the true capacities and limitations of different genomic microarrays to detect and define DNA copy-number variations.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;array CGH&quot;">array CGH</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular cytogenetics&quot;">molecular cytogenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microdeletion">microdeletion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;copy-number variation&quot;">copy-number variation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;power analysis&quot;">power analysis</a></div>                            <h2 scrollto-destination=87194281 id="87194281" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Conceptual and technological developments in molecular cytogenetic techniques are now enhancing the resolution power of conventional chromosome analysis from the megabase to the kilobase level. Array-based comparative genomic hybridization (array CGH), i.e. the application of CGH to an array of genomic fragments such as bacterial artificial chromosomes (BACs), has been the method of choice for genome-wide copy-number analysis in the last few years.<sup><a reveal-id="DSM002C1" data-open="DSM002C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM002C2" data-open="DSM002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The density of the various ‘whole-genome’ BAC clone sets commonly used varies from one clone per Mb<sup><a reveal-id="DSM002C3  DSM002C5" data-open="DSM002C3  DSM002C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> to an overlapping clone set covering the entire human genome with one clone per 100 kb.<sup><a reveal-id="DSM002C6" data-open="DSM002C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSM002C7" data-open="DSM002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Array CGH has rapidly become an important genome analysis tool in cancer research,<sup><a reveal-id="DSM002C8  DSM002C10" data-open="DSM002C8  DSM002C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> in the identification of novel microdeletion syndromes and gene identification studies,<sup><a reveal-id="DSM002C11  DSM002C15" data-open="DSM002C11  DSM002C15" class="link link-ref link-reveal xref-bibr">11–15</a></sup> in the diagnosis of patients with congenital malformation syndromes and/or unexplained mental retardation,<sup><a reveal-id="DSM002C5" data-open="DSM002C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSM002C16" data-open="DSM002C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and in prenatal diagnosis.<sup><a reveal-id="DSM002C18" data-open="DSM002C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM002C19" data-open="DSM002C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Although disease-causing genomic alterations are thought to be rare, recent work using high-resolution microarray approaches has indicated that genomic copy-number variation without immediate phenotypic consequences is widespread throughout the entire human genome.<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSM002C20  DSM002C23" data-open="DSM002C20  DSM002C23" class="link link-ref link-reveal xref-bibr">20–23</a></sup></p><p>Most currently used genomic copy-number profiling microarrays are produced in academic settings, and the resolution of these microarrays varies depending on the type and number of genomic targets selected, the protocols used, and the data-analysis tools employed. Only recently, private enterprises embarked on this novel microarray application, and several companies are now offering microarrays for genomic copy-number profiling. Most of these microarrays encompass 25–85-mer oligonucleotides targeting random genomic sequences<sup><a reveal-id="DSM002C24  DSM002C26" data-open="DSM002C24  DSM002C26" class="link link-ref link-reveal xref-bibr">24–26</a></sup> or single-nucleotide polymorphisms (SNPs).<sup><a reveal-id="DSM002C27  DSM002C31" data-open="DSM002C27  DSM002C31" class="link link-ref link-reveal xref-bibr">27–31</a></sup> The theoretical advantages of using such commercial platforms are numerous: (1) they provide a higher genome coverage than most microarrays generated in academia, (2) they can be produced in large quantities according to industrial quality standards, (3) they are available to all researchers, also those without dedicated microarray facilities, and (4) their widespread use will generate large comparable data sets that facilitate data comparison and cooperation between research groups. At present, however, little is known about the actual performance of these platforms, and first-time users will find limited guidance on which platform is most appropriate for their applications and requirements. Although various platform comparison studies have been reported for microarray-based expression profiling,<sup><a reveal-id="DSM002C32  DSM002C35" data-open="DSM002C32  DSM002C35" class="link link-ref link-reveal xref-bibr">32–35</a></sup> as yet a comprehensive platform comparison for genomic profiling, including an adequate statistical power analysis, has not been reported.</p><p>A prerequisite for a performance comparison of genomic microarrays is the availability of a comprehensive method that accounts for specific requirements associated with genomic microarray data such as adjacency of probes and asymmetric <em>y</em>-axis measurements associated with deletions and/or duplications. Here, we introduce a method that adheres to these requirements, and that is based on statistical power calculations to compare the practical resolution of individual genomic microarray experiments obtained using different microarray platforms. The method is validated using simulated data sets as well as data sets obtained using our in-house tiling-resolution BAC arrays and commercially available 100k SNP, 250k SNP, and 385k oligonucleotide microarray platforms. From our results, we conclude that the increased probe density of the commercially available microarray platforms, although accompanied by a lower signal-to-noise ratio, results in a higher genome-wide copy-number detection resolution.</p>                            <h2 scrollto-destination=87194285 id="87194285" class="section-title" data-legacy-id=s2>2. Methods</h2>
                            <h3 scrollto-destination=87194286 id="87194286" class="section-title" data-legacy-id=s2a>2.1. Patients and healthy donors</h3>
<p>The platform comparison was performed using DNA from 13 patients harboring submicroscopic genomic copy-number variations previously identified by tiling-resolution array CGH.<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Genomic DNA was isolated from blood leukocytes by standard procedures. Male and female reference DNA pools previously used for tiling-resolution BAC array analysis were also used for hybridization to the NimbleGen oligonucleotide microarrays. These reference pools contain equal amounts of genomic DNA from 10 healthy donors (males or females). For the Affymetrix SNP oligonucleotide microarray experiments, using single color hybridizations, two male and two female reference pools were used for normalization purposes.</p>                            <h3 scrollto-destination=87194288 id="87194288" class="section-title" data-legacy-id=s2b>2.2. Tiling-resolution BAC array CGH</h3>
<p>Previously, we reported an array CGH study<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> using a tiling-resolution microarray encompassing 32,447 overlapping BAC clones selected to cover the entire human genome.<sup><a reveal-id="DSM002C6" data-open="DSM002C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSM002C7" data-open="DSM002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Hundred patients with unexplained mental retardation were hybridized in duplicate against a sex-mismatched reference pool to this microarray. On the basis of these hybridizations, we selected 13 patients with validated submicroscopic copy-number variations, both single copy-number gains and losses, for hybridization to the other platforms.</p>                            <h3 scrollto-destination=87194290 id="87194290" class="section-title" data-legacy-id=s2c>2.3. Affymetrix 100k SNP arrays</h3>
<p>The Affymetrix 100k SNP array contains 25-mer oligonucleotides representing a total of 116,204 SNPs, distributed over two microarrays. Array experiments were performed according to protocols provided by the manufacturer (Affymetrix, Inc., Santa Clara, CA) as described previously.<sup><a reveal-id="DSM002C27" data-open="DSM002C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Copy-number estimations were determined using the recently published software package CNAG (Copy Number Analyzer for Affymetrix GeneChip Mapping 100k arrays<sup><a reveal-id="DSM002C28" data-open="DSM002C28" class="link link-ref link-reveal xref-bibr">28</a></sup>). This algorithm strongly improves the signal-to-noise ratios of the copy-number data by (1) accounting for the length and GC content of the polymerase chain reaction products using quadratic regressions and by (2) normalizing the patient samples to reference samples run in parallel.</p>                            <h3 scrollto-destination=87194292 id="87194292" class="section-title" data-legacy-id=s2d>2.4. Affymetrix 250k SNP arrays</h3>
<p>Affymetrix provides two microarrays each containing approximately 250,000 SNPs, and together forming the 500k assay. For this study, we selected the Nsp 250k SNP array, which contains 262,264 25-mer oligonucleotides. For the 100k SNP array experiments, the 250k SNP array experiments were performed according to protocols provided by the manufacturer (Affymetrix, Inc., Santa Clara, CA). Copy-number estimates were determined using the CNAG software package,<sup><a reveal-id="DSM002C28" data-open="DSM002C28" class="link link-ref link-reveal xref-bibr">28</a></sup> which was recently updated for the analysis of these arrays (version 2.0).</p>                            <h3 scrollto-destination=87194294 id="87194294" class="section-title" data-legacy-id=s2e>2.5. NimbleGen 385k oligonucleotide arrays</h3>
<p>The NimbleGen whole genome oligonucleotide microarray contains 386,165 isothermal probes (45–75-mer), spanning the human genome at a mean probe spacing of 7 kb. Isothermal oligonucleotide design, array fabrication, DNA labeling, CGH experiments, data normalization, and log<sub>2</sub>(Cy3/Cy5) copy-number ratio calculations were performed by NimbleGen according to published procedures.<sup><a reveal-id="DSM002C26" data-open="DSM002C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=87194296 id="87194296" class="section-title" data-legacy-id=s2f>2.6. Hidden Markov analysis</h3>
<p>The normalized ratios were analyzed for loss and gain of regions by a standard Hidden Markov Model (HMM), which was optimized for each of the microarray platforms in order to maximize the detection of the known validated copy-number aberrations, while minimizing the false-positive rate, as described before.<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=87194298 id="87194298" class="section-title" data-legacy-id=s2g>2.7. Statistical power analysis</h3>
<p>For each of the four microarray platforms, we performed a statistical power analysis of adjacent targets surrounding a specific locus on a chromosome. This revealed the relationship between the genomic length of the copy-number variation, the noise contained in measurements, and, ultimately, the false-positive and false-negative detection rates for the microarray, and thus, provided a platform-independent discrimination statistic describing the ability of a microarray to detect single copy-number variations.</p><p>The statistical power analysis comprises the following steps: </p><ul class="list-simple"><li><p>(1) Determination of the distribution of the noise,</p></li><li><p>(2) Establishment of estimates for significant changes and the variance of noise within each experiment,</p></li><li><p>(3) Calculation of the number of data points required for detection of copy-number variations, and</p></li><li><p>(4) Determination of the resolution of a microarray platform.</p></li></ul>                            <h4 scrollto-destination=87194302 id="87194302" class="section-title" data-legacy-id=s2g1>2.7.1. Determination of the distribution of the noise</h4>
<p>The method assumes a normal distribution of noise within the copy-number data. We used a χ<sup>2</sup> goodness-of-fit test,<sup><a reveal-id="DSM002C36" data-open="DSM002C36" class="link link-ref link-reveal xref-bibr">36</a></sup> using a <em>p</em>-value of less than 0.05, and could not reject this hypothesis, thereby justifying the application of the method used for calculating the statistical power.</p>                            <h4 scrollto-destination=87194304 id="87194304" class="section-title" data-legacy-id=s2g2>2.7.2. Establishment of estimates for significant changes and variance of noise</h4>

  <div class="block-child-p">To provide an estimation of a single copy-number loss, the mean log<sub>2</sub> ratio is calculated over all targets on the X chromosome,<sup><a reveal-id="DSM002C37" data-open="DSM002C37" class="link link-ref link-reveal xref-bibr">37</a></sup> excluding those mapped to the pseudo-autosomal regions. This provides an estimate of a significant change to be used in the power calculations, and requires that experiments used for the comparison are performed on the basis of sex mismatch (either in silico or in vitro, depending on the microarray platform used). From the estimate of a single loss, an estimate of a single gain (μ̂<sub>Gain</sub>) is calculated via <a id="DSM002M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM002M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm00203.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00203.gif?Expires=1509238563&amp;Signature=JR4I6KI4y00Kdmuz0QltxtM2Sk5FBZ5YrOFZKJsGS40StVLXnlnSscrjkBF0Ufu4W0sAbQ2z0Q9mdf4kBD1Uqd2Bvzm7dtbs5NbjTRsUbrAXqrKPxv7OSKuZl72zDjzGvTER01nnNN-5Pkn2kupBhhkThPA7e212AvjwssvjOz8r8mJf1XePL7yW~mwhU3fAgIIYyoCGw5M3HLA6UywaBxvCt8QHyu9Edp9vnJh-L907dsHR5FBbrrcyPgKXTxgRorAAuuq6jdfKV9YDMyVoqWt~9AY0dhYMAfMJN3mStZOkbE4oZl9eELaygq8ZkJQSfG-ZBZOHXhshc07qcqmxLQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="formula" path-from-xml="dsm00203.gif" /></a></div></div><div class="label">1</div></div> where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00207.gif?Expires=1509238563&amp;Signature=NehNZTL9hvsuvOjnMZ5bOUxwPfo8rWlJE4cFMS2~HR-HfLPsMt5WMGMMYG3WXuEjkw7AdVz528ifUPhDzOQS0MaTqYlu7Hcbe1iDt1dcxiFj224UsEhHpCLnty9~CFDxrwtFfVAtH~i76RXYWe5gG5zfazuDNxOvOHwwGPQSoZpkn5htrMaxIyJA9WP~oBMFUY6VoxguzCSGhIZi2dyb49EU45JJSP2MiApLb-ePvHjykTMrjI9yuDk1uYJkD8ozj5N4ZfnsOwrzMX6HeYklw7mSTTXmk5slP61Sc36B6i~21J9iLubHphT520tbz2eNURSYdtDK2cAG~o3AURN-uQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="forumla" path-from-xml="dsm00207.gif" /></span> is the mean log<sub>2</sub> ratio of targets located on chromosome X and Chr X<sub>theoretical</sub> the theoretical ratio of a single loss (see Supplementary Data). The standard deviation of all log<sub>2</sub> ratios from autosomal targets, excluding those known to be involved in validated copy-number variations, is used as an estimate of the variance.</div>
                            <h4 scrollto-destination=87194306 id="87194306" class="section-title" data-legacy-id=s2g3>2.7.3. Calculation of the number of data points required for detection of genomic copy-number variations</h4>
<p>We calculate the number of data points required to detect a genuine single copy-number variation (as estimated by the mean chromosome X values) given the autosomal standard deviation, with a confidence factor determined by the desired statistical power. This is done by determining the number of data points required to lie in the outer regions of the distribution of the copy-number ratios for it to be deemed unusual in terms of the expected (normal) distribution. We use the non-central <em>T</em> cumulative distribution in order to determine the number of sample points required to satisfy the desired power given estimates of significant changes and an estimate of the variance.<sup><a reveal-id="DSM002C38" data-open="DSM002C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSM002C39" data-open="DSM002C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>
  <div class="block-child-p">In this study, we chose to use a power of 95% and a two-sided <em>t</em>-test, given the required significance level α. Note that the statistical power (1 − β) is the probability that a true aberration of <em>n</em> adjacent probes is detected (Type II error). The significance level α is the probability of observing a particular deviation between the mean of the <em>n</em> adjacent probes and the rest of the probes on the chromosome, when no actual copy-number variation is present (Type I error). Hence, we aim to solve the following series of equations for the desired power (1 − β) = 0.95. We first define the non-centrality parameter as <a id="DSM002M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM002M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm00204.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00204.gif?Expires=1509238563&amp;Signature=aK1HZDXy2GngT9Z-ndwcCwLk-X3jQ-eln0e3qtlq80GH7GDDdTAIEYD~tZJe9RGhjQXwJbc9Ou9gvBNgM62nWIgHtKLDEmb1qF80~IHLCXmukeoG2nfWFQEI4vgUy33CdzHg1E7h503wA0uPinG7YaM6VEO23EOXlqTocVHXLJWCDfksGChYW77-bqWIBciViDnKR0zSifQiEcEFQfGgD4a1HSWBYD4YwM8WKsOAPrJZGcIfumaZsO~J2REeFZs4K5cMj-Bx550Kg01LgOtY2FeCuu7X41NtaYaLF4~fmMIGZ3P9gXCigrvUyFqyEG~IFRajKe0rkfpQh7Kjt3BJ1A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="formula" path-from-xml="dsm00204.gif" /></a></div></div><div class="label">2</div></div> where μ̂<sub>1</sub> is the estimate of the ratio pertaining to the copy-number variation, μ̂<sub>0</sub> the mean of the autosomal ratios, σ̂ the standard deviation, estimated using the autosomal targets, and <em>n</em> the number of adjacent targets per aberration. We define two cut-offs, via the inverse of the Student's <em>T</em> cumulative (central) distribution function (<a class="link link-uri" href="http://mathworld.wolfram.com/NoncentralStudentst-Distribution.html">http://mathworld.wolfram.com/NoncentralStudentst-Distribution.html</a>), <em>T</em><sup>−1</sup>, using the desired power, and the inverse of the power. The cut-offs <em>C</em><sub>1</sub> and <em>C</em><sub>2</sub> are defined as <a id="DSM002M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM002M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm00205.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00205.gif?Expires=1509238563&amp;Signature=LPBNT8vh~OpRnHYajHo~9Dr32ODfIReoQDWlyKL-MhHPuJ2QdDhQpWKAj8Mp6rMsjunpX5rB7dzPHNNc0WHOWZcZFZqFzabl~gBbYhe0SlIsJoY8t-mvUn1kSU6AiH-3xapyU9vM-l9xCW6Y4TVLoPqd8psZfd7vXCO4sn2t8ZcBMtTmPYvD3Z6TE39aJZjhc2yc6ipkq8WaT80fJW0-iauxP8C8YgF0NPpviHtGkB1hB2HDPsAUMjlxgCTX8PfvAznqjw3u~wKlys0FF2Bsi0GVPN4Dms1SngF4N-ZnsRnQBcOMRVA7WH1CSaVoou-f6UYXmoj~1WwR80DGLc2sRg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="formula" path-from-xml="dsm00205.gif" /></a></div></div><div class="label">3</div></div> where the degrees of freedom df = <em>n</em> − 1 and α is the required significance level. The power is then calculated with the non-central cumulative <em>T</em> distribution function <em>T</em><sub>NC</sub> as follows: <a id="DSM002M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM002M4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm00206.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00206.gif?Expires=1509238563&amp;Signature=S-uaIxQ~neU1Xa2daVUR99IldAW0XGKYLVLL0e6t24RoQ6xcrFxgtVlzgf41eCvZUEj7DNIFXIucWS2sleTaajEzlGVFO26Mffp-2fKowqH0PY15G-5InfbRA5VE7kMYlxc2bvt6RT9UD73yyjSq~DN-D3MSCm1QZkL~ofcb1Z3idgD4XNLWk8bfiRTKiH-jRcAbt7XX~h5kMM1WLT6OPJVeWk-hmKet4jFB~5m29AE4R3X~ByFoBVNxkYsUgf8oK~b~~N47hP2gCcYkRcyUOGqMI~vhu6BwHC0hLjWbsd5ZiyenRXS3YwOJSoT7SHQ94HUgKdb8H-oIHIU0TYWIxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="formula" path-from-xml="dsm00206.gif" /></a></div></div><div class="label">4</div></div> We then find the number of adjacent probes <em>n</em> [in Equation (<a href="#DSM002M2" class="sectionLink xref-sec">2</a>)] required to solve the function <em>power</em> in order to achieve our desired power.</div>
                            <h4 scrollto-destination=87194309 id="87194309" class="section-title" data-legacy-id=s2g4>2.7.4. Determination of the resolution of a microarray platform</h4>
<p>To calculate the resolution of a microarray platform, the outcome of the power analysis is used in conjunction with the genomic coverage of the platform. The distribution of the microarray probes throughout the genome is determined by the size of the gaps between the microarray targets. For our calculation, we take into account the uneven genomic distribution of the microarray targets and assume that copy-number variations can occur randomly throughout the whole genome. Next, we create a window with size equal to the number of data points required to detect a copy-number variation, as given by the power calculation, and determine the number of instances within the genome where the window has a size less than the size of the variation to be detected. This is compared to the total possible number of windows that occur within the genome. By doing so, we create a genome-wide probability that a copy-number variation with a particular size independent of its genomic location will be detected by a microarray platform. We calculated the resolution for single copy-number variations with genomic sizes ranging from 10 kb to 1 Mb, separately for gains and losses.</p>                            <h2 scrollto-destination=87194311 id="87194311" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87194312 id="87194312" class="section-title" data-legacy-id=s3a>3.1. Study design</h3>
<p>In this study, we assessed the capacity of various genomic microarray platforms to detect submicroscopic single copy-number variations, including deletions and duplications. We selected samples from 13 patients in which we have previously identified and validated copy-number variations using our in-house produced tiling-resolution 32k BAC arrays.<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> These samples were hybridized onto 100k Affymetrix SNP arrays, 250k Affymetrix SNP arrays, and 385k NimbleGen oligonucleotide arrays. As an example, Fig. <a reveal-id="DSM002F1" data-open="DSM002F1" class="link link-reveal link-table xref-fig">1</a> shows the chromosome profile obtained for the various platforms in a patient with a 0.54-Mb sized deletion at 9q33.1. We applied a standard HMM algorithm to automatically detect copy-number variations onto the different platforms. Next, we developed and tested a novel method based on statistical power analysis for an objective comparison of the detection resolution of the different platforms.</p>                        <a id="87194314" scrollto-destination="87194314"></a>
<div id="DSM002F1" data-id="DSM002F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00201.jpeg?Expires=1509238563&Signature=XWTgMdjny-V3uVvbuMY3TQQWGbxvtwESRemeTmbvZ9kHYngZaohFrzCxW8XFMPdHCnOXdHdTbt~EoEmq73rrYJ5~giG6m116TM0iqdQOUKk4bSKq7GdEf7U7imfEFkwgVSEEm22tu-5NYix9VfWYtJR7dwPmyLGA1o3F1tlwftWcc-GeTjjgzNE71vNU2-9c3fNA43Shj6ZBAWxOEXPcU-wyf2IX7amqGNm-ApnVzXQM7v7vrVTUXfLgNQKhhRczvNs4vCFjiPkeitM2hp2lptxKHdr9ikh~99tcNeF~FpN2G3rOtBWmmweve8yx-A7KWRnFEwNVFqAv6aCGnMKJzw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Detection of known and validated submicroscopic copy-number variations by high-density BAC, SNP, and oligonucleotide arrays. Individual chromosome plots are shown for Patient 8 (chromosome 9), with the log2 T/R (test-over-reference) values plotted on the y-axis versus the genomic position on chromosome 9 on the x-axis. Results are shown for the tiling-resolution 32k BAC array (A), the 100k SNP array (B), the 250k SNP array (C), and the 385k oligonucleotide array (D). A known and validated microdeletion of 0.54 Mb on 9q33.1 is detected by all four genomic microarray platforms (see black arrow). In addition, a previously undetected microduplication is clearly visible on the chromosome profile obtained by the 250k SNP array (see grey arrow). This figure also shows the different levels of microarray noise present for the different microarray platforms." path-from-xml="dsm00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194314/dsm00201.jpeg" path-from-xml="dsm00201.jpeg" target="_blank">View large</a><a section="87194314" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/dsm00201.jpeg?Expires=1509238563&Signature=Oi0CED3CW4K73wWySLem6rTC5QN0kEpQIhQhp~BungaHduZKRhsSKxaP5UIS3iJY170Qa4iP3BpepfbRlbK0KQ~ZIcG~3N9njgq9LuYbNXf29oDL6qfhsiMSc3dbq4uoxABgLcwTDMroUcj4jbu4tqJS9jjHBR1-rfzcsAzOWA0E53l0P91GPshgpkFPBnoA7Ky43Ry0mOlB--m6nSXprB8M8-bwtXv~Gs28JHWSbb3YOgP12prkPJGC0S-b~5Puns954703GH9BZRuI05V7ITMfdG7uqQ2MB705iKFQB5FS13t3LdDje2y5L7pfFR1y5-8DPwWdI~Fh5S4uG-5DhA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87194314&ar=502776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of known and validated submicroscopic copy-number variations by high-density BAC, SNP, and oligonucleotide arrays. Individual chromosome plots are shown for Patient 8 (chromosome 9), with the log<sub>2</sub> T/R (test-over-reference) values plotted on the <em>y</em>-axis versus the genomic position on chromosome 9 on the <em>x</em>-axis. Results are shown for the tiling-resolution 32k BAC array (<strong>A</strong>), the 100k SNP array (<strong>B</strong>), the 250k SNP array (<strong>C</strong>), and the 385k oligonucleotide array (<strong>D</strong>). A known and validated microdeletion of 0.54 Mb on 9q33.1 is detected by all four genomic microarray platforms (see black arrow). In addition, a previously undetected microduplication is clearly visible on the chromosome profile obtained by the 250k SNP array (see grey arrow). This figure also shows the different levels of microarray noise present for the different microarray platforms.</p></div></div><div content-id="DSM002F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00201.jpeg?Expires=1509238563&Signature=XWTgMdjny-V3uVvbuMY3TQQWGbxvtwESRemeTmbvZ9kHYngZaohFrzCxW8XFMPdHCnOXdHdTbt~EoEmq73rrYJ5~giG6m116TM0iqdQOUKk4bSKq7GdEf7U7imfEFkwgVSEEm22tu-5NYix9VfWYtJR7dwPmyLGA1o3F1tlwftWcc-GeTjjgzNE71vNU2-9c3fNA43Shj6ZBAWxOEXPcU-wyf2IX7amqGNm-ApnVzXQM7v7vrVTUXfLgNQKhhRczvNs4vCFjiPkeitM2hp2lptxKHdr9ikh~99tcNeF~FpN2G3rOtBWmmweve8yx-A7KWRnFEwNVFqAv6aCGnMKJzw__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Detection of known and validated submicroscopic copy-number variations by high-density BAC, SNP, and oligonucleotide arrays. Individual chromosome plots are shown for Patient 8 (chromosome 9), with the log2 T/R (test-over-reference) values plotted on the y-axis versus the genomic position on chromosome 9 on the x-axis. Results are shown for the tiling-resolution 32k BAC array (A), the 100k SNP array (B), the 250k SNP array (C), and the 385k oligonucleotide array (D). A known and validated microdeletion of 0.54 Mb on 9q33.1 is detected by all four genomic microarray platforms (see black arrow). In addition, a previously undetected microduplication is clearly visible on the chromosome profile obtained by the 250k SNP array (see grey arrow). This figure also shows the different levels of microarray noise present for the different microarray platforms." path-from-xml="dsm00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194314/dsm00201.jpeg" path-from-xml="dsm00201.jpeg" target="_blank">View large</a><a section="87194314" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/dsm00201.jpeg?Expires=1509238563&Signature=Oi0CED3CW4K73wWySLem6rTC5QN0kEpQIhQhp~BungaHduZKRhsSKxaP5UIS3iJY170Qa4iP3BpepfbRlbK0KQ~ZIcG~3N9njgq9LuYbNXf29oDL6qfhsiMSc3dbq4uoxABgLcwTDMroUcj4jbu4tqJS9jjHBR1-rfzcsAzOWA0E53l0P91GPshgpkFPBnoA7Ky43Ry0mOlB--m6nSXprB8M8-bwtXv~Gs28JHWSbb3YOgP12prkPJGC0S-b~5Puns954703GH9BZRuI05V7ITMfdG7uqQ2MB705iKFQB5FS13t3LdDje2y5L7pfFR1y5-8DPwWdI~Fh5S4uG-5DhA__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87194314&ar=502776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of known and validated submicroscopic copy-number variations by high-density BAC, SNP, and oligonucleotide arrays. Individual chromosome plots are shown for Patient 8 (chromosome 9), with the log<sub>2</sub> T/R (test-over-reference) values plotted on the <em>y</em>-axis versus the genomic position on chromosome 9 on the <em>x</em>-axis. Results are shown for the tiling-resolution 32k BAC array (<strong>A</strong>), the 100k SNP array (<strong>B</strong>), the 250k SNP array (<strong>C</strong>), and the 385k oligonucleotide array (<strong>D</strong>). A known and validated microdeletion of 0.54 Mb on 9q33.1 is detected by all four genomic microarray platforms (see black arrow). In addition, a previously undetected microduplication is clearly visible on the chromosome profile obtained by the 250k SNP array (see grey arrow). This figure also shows the different levels of microarray noise present for the different microarray platforms.</p></div></div>                            <h3 scrollto-destination=87194315 id="87194315" class="section-title" data-legacy-id=s3b>3.2. Automatic detection of copy-number aberrations by HMM</h3>
<p>In order to obtain independent information on the performance of the different microarray platforms in identifying submicroscopic copy-number variations, we applied a single automated HMM algorithm to the experiments performed in this study (see Table <a reveal-id="DSM002TB1" data-open="DSM002TB1" class="link link-reveal link-table xref-fig">1</a>). The known and validated copy-number changes were previously identified on the 32k BAC microarray platform,<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and ranged in sizes from 230 kb to 8.9 Mb. Samples from 10 patients were tested on the 385k NimbleGen oligonucleotide microarray platform, and all of the previously identified and validated copy-number variations were detected by the automated HMM algorithm. In contrast, two of the previously identified and validated copy-number variations out of the 13 tested were not automatically detected on the Affymetrix 100k SNP array platform. One of these was a 350 kb deletion on chromosome 7q11.21 (Patient 5), and the other was a 1.65 Mb deletion on chromosome 15q24 (Patient 11). The HMM algorithm correctly detected 10 out of 11 previously identified and validated copy-number variations on the Affymetrix 250k SNP microarray. Again, the 350 kb deletion on 7q11.21 could not be detected automatically, whereas the 1.65 Mb deletion on 15q24 was readily detected on this platform. In addition to the known and validated copy-number variations, a large number of additional copy-number variations was detected but not validated.</p>                        <a id="87194317" scrollto-destination="87194317"></a>
<div class="table-wrap"><div id="DSM002TB1" class="table-wrap-title"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Detection of known and validated submicroscopic copy-number variations onto high-density BAC, SNP and oligonucleotide microarrays</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Patient </th><th rowspan="3">Copy number </th><th rowspan="3">Chromosome </th><th rowspan="3">Size (Mb) </th><th colspan="4">No. of targets in region </th><th colspan="4">Average ratio targets in region </th><th colspan="3">Detected by HMM<sup>a</sup> </th></tr><tr><th>32k </th><th>100k </th><th>250k </th><th>385k </th><th>32k </th><th>100k </th><th>250k </th><th>385k </th><th>100k </th><th>250k </th><th>385k </th></tr><tr><th>BACs </th><th>SNPs </th><th>SNPs </th><th>Oligos </th><th>BACs </th><th>SNPs </th><th>SNPs </th><th>Oligos </th><th>SNPs </th><th>SNPs </th><th>Oligos </th></tr></thead><tbody><tr><td>1 </td><td>Loss </td><td>1 </td><td>3.93 </td><td>42 </td><td>125 </td><td>230 </td><td>n.d. </td><td>−0.41 </td><td>−0.51 </td><td>−0.45 </td><td>n.d. </td><td>Yes </td><td>Yes </td><td>n.d. </td></tr><tr><td>2 </td><td>Gain </td><td>1 </td><td>2.12 </td><td>21 </td><td>50 </td><td>120 </td><td>176 </td><td>0.44 </td><td>0.49 </td><td>0.77 </td><td>0.59 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>3 </td><td>Loss </td><td>2 </td><td>0.92 </td><td>11 </td><td>47 </td><td>64 </td><td>127 </td><td>−0.59 </td><td>−0.43 </td><td>−0.52 </td><td>−0.45 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>4 </td><td>Gain </td><td>5 </td><td>1.24 </td><td>16 </td><td>40 </td><td>n.d. </td><td>n.d. </td><td>0.29 </td><td>0.30 </td><td>n.d. </td><td>n.d. </td><td>Yes </td><td>n.d. </td><td>n.d. </td></tr><tr><td>5 </td><td>Loss </td><td>7 </td><td>0.35 </td><td>18 </td><td>1 </td><td>13 </td><td>30 </td><td>−0.23 </td><td>−0.52 </td><td>−0.40 </td><td>−0.24 </td><td>No </td><td>No </td><td>Yes </td></tr><tr><td>6 </td><td>Gain </td><td>9 </td><td>0.23 </td><td>5 </td><td>23 </td><td>38 </td><td>40 </td><td>0.35 </td><td>0.30 </td><td>0.47 </td><td>0.27 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>7 </td><td>Loss </td><td>9 </td><td>2.85 </td><td>30 </td><td>145 </td><td>320 </td><td>n.d. </td><td>−0.39 </td><td>−0.45 </td><td>−0.44 </td><td>n.d. </td><td>Yes </td><td>Yes </td><td>n.d. </td></tr><tr><td>8 </td><td>Loss </td><td>9 </td><td>0.54 </td><td>6 </td><td>22 </td><td>70 </td><td>88 </td><td>−0.50 </td><td>−0.44 </td><td>−0.44 </td><td>−0.37 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>9 </td><td>Loss </td><td>11 </td><td>9.15 </td><td>80 </td><td>551 </td><td>923 </td><td>1299 </td><td>−0.35 </td><td>−0.47 </td><td>−0.47 </td><td>−0.50 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>10 </td><td>Gain </td><td>12 </td><td>2.30 </td><td>39 </td><td>69 </td><td>n.d. </td><td>353 </td><td>0.32 </td><td>0.26 </td><td>n.d. </td><td>0.23 </td><td>Yes </td><td>n.d. </td><td>Yes </td></tr><tr><td>11 </td><td>Loss </td><td>15 </td><td>1.65 </td><td>16 </td><td>4 </td><td>40 </td><td>204 </td><td>−0.33 </td><td>−0.36 </td><td>−0.50 </td><td>−0.35 </td><td>No </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>2.89 </td><td>28 </td><td>64 </td><td>151 </td><td>420 </td><td>0.37 </td><td>0.26 </td><td>0.46 </td><td>0.29 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>1.43 </td><td>14 </td><td>18 </td><td>91 </td><td>198 </td><td>0.36 </td><td>0.19 </td><td>0.44 </td><td>0.31 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>2.88 </td><td>30 </td><td>205 </td><td>279 </td><td>442 </td><td>0.41 </td><td>0.29 </td><td>0.45 </td><td>0.28 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>1.48 </td><td>24 </td><td>21 </td><td>64 </td><td>189 </td><td>0.33 </td><td>0.26 </td><td>0.52 </td><td>0.31 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>13 </td><td>Loss </td><td>22 </td><td>2.66 </td><td>35 </td><td>36 </td><td>130 </td><td>306 </td><td>−0.41 </td><td>−0.47 </td><td>−0.41 </td><td>−0.23 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr></tbody></table></div><div content-id="DSM002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Patient </th><th rowspan="3">Copy number </th><th rowspan="3">Chromosome </th><th rowspan="3">Size (Mb) </th><th colspan="4">No. of targets in region </th><th colspan="4">Average ratio targets in region </th><th colspan="3">Detected by HMM<sup>a</sup> </th></tr><tr><th>32k </th><th>100k </th><th>250k </th><th>385k </th><th>32k </th><th>100k </th><th>250k </th><th>385k </th><th>100k </th><th>250k </th><th>385k </th></tr><tr><th>BACs </th><th>SNPs </th><th>SNPs </th><th>Oligos </th><th>BACs </th><th>SNPs </th><th>SNPs </th><th>Oligos </th><th>SNPs </th><th>SNPs </th><th>Oligos </th></tr></thead><tbody><tr><td>1 </td><td>Loss </td><td>1 </td><td>3.93 </td><td>42 </td><td>125 </td><td>230 </td><td>n.d. </td><td>−0.41 </td><td>−0.51 </td><td>−0.45 </td><td>n.d. </td><td>Yes </td><td>Yes </td><td>n.d. </td></tr><tr><td>2 </td><td>Gain </td><td>1 </td><td>2.12 </td><td>21 </td><td>50 </td><td>120 </td><td>176 </td><td>0.44 </td><td>0.49 </td><td>0.77 </td><td>0.59 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>3 </td><td>Loss </td><td>2 </td><td>0.92 </td><td>11 </td><td>47 </td><td>64 </td><td>127 </td><td>−0.59 </td><td>−0.43 </td><td>−0.52 </td><td>−0.45 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>4 </td><td>Gain </td><td>5 </td><td>1.24 </td><td>16 </td><td>40 </td><td>n.d. </td><td>n.d. </td><td>0.29 </td><td>0.30 </td><td>n.d. </td><td>n.d. </td><td>Yes </td><td>n.d. </td><td>n.d. </td></tr><tr><td>5 </td><td>Loss </td><td>7 </td><td>0.35 </td><td>18 </td><td>1 </td><td>13 </td><td>30 </td><td>−0.23 </td><td>−0.52 </td><td>−0.40 </td><td>−0.24 </td><td>No </td><td>No </td><td>Yes </td></tr><tr><td>6 </td><td>Gain </td><td>9 </td><td>0.23 </td><td>5 </td><td>23 </td><td>38 </td><td>40 </td><td>0.35 </td><td>0.30 </td><td>0.47 </td><td>0.27 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>7 </td><td>Loss </td><td>9 </td><td>2.85 </td><td>30 </td><td>145 </td><td>320 </td><td>n.d. </td><td>−0.39 </td><td>−0.45 </td><td>−0.44 </td><td>n.d. </td><td>Yes </td><td>Yes </td><td>n.d. </td></tr><tr><td>8 </td><td>Loss </td><td>9 </td><td>0.54 </td><td>6 </td><td>22 </td><td>70 </td><td>88 </td><td>−0.50 </td><td>−0.44 </td><td>−0.44 </td><td>−0.37 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>9 </td><td>Loss </td><td>11 </td><td>9.15 </td><td>80 </td><td>551 </td><td>923 </td><td>1299 </td><td>−0.35 </td><td>−0.47 </td><td>−0.47 </td><td>−0.50 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>10 </td><td>Gain </td><td>12 </td><td>2.30 </td><td>39 </td><td>69 </td><td>n.d. </td><td>353 </td><td>0.32 </td><td>0.26 </td><td>n.d. </td><td>0.23 </td><td>Yes </td><td>n.d. </td><td>Yes </td></tr><tr><td>11 </td><td>Loss </td><td>15 </td><td>1.65 </td><td>16 </td><td>4 </td><td>40 </td><td>204 </td><td>−0.33 </td><td>−0.36 </td><td>−0.50 </td><td>−0.35 </td><td>No </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>2.89 </td><td>28 </td><td>64 </td><td>151 </td><td>420 </td><td>0.37 </td><td>0.26 </td><td>0.46 </td><td>0.29 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>1.43 </td><td>14 </td><td>18 </td><td>91 </td><td>198 </td><td>0.36 </td><td>0.19 </td><td>0.44 </td><td>0.31 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>2.88 </td><td>30 </td><td>205 </td><td>279 </td><td>442 </td><td>0.41 </td><td>0.29 </td><td>0.45 </td><td>0.28 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>12 </td><td>Gain </td><td>17 </td><td>1.48 </td><td>24 </td><td>21 </td><td>64 </td><td>189 </td><td>0.33 </td><td>0.26 </td><td>0.52 </td><td>0.31 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr><tr><td>13 </td><td>Loss </td><td>22 </td><td>2.66 </td><td>35 </td><td>36 </td><td>130 </td><td>306 </td><td>−0.41 </td><td>−0.47 </td><td>−0.41 </td><td>−0.23 </td><td>Yes </td><td>Yes </td><td>Yes </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>All copy-number variations were initially detected by an automated HMM on the 32k BAC array.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194317" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87194318 id="87194318" class="section-title" data-legacy-id=s3c>3.3. Verification of power calculation using simulated data</h3>
<p>In order to verify the power calculation, we created a step function of a single copy-number loss based on our model for an aberration and corrupted it with a noise signal that had a normal distribution to simulate a single copy-number loss (Supplementary Fig. 1A). The results of the power analysis on this data set are displayed in Supplementary Fig. 1B. This analysis shows that a minimum of four data points with log<sub>2</sub> ratios outside the normal distribution is required for a single copy-number loss to be detected with the desired power (95%). Subsequently, a Monte Carlo simulation was used to test the behavior of the power calculation. We artificially generated 400 samples of size 4 under the null hypothesis with a mean of 0, and another 400 with a mean resembling a loss, which represents the alternate hypothesis. The results of this analysis are illustrated in Supplementary Fig. 1C, where the null hypothesis converges to the expected 5% and the alternative hypothesis to 95%. This analysis shows that the power calculation is effective in determining the required data points for the successful detection of a copy-number variation.</p>                            <h3 scrollto-destination=87194320 id="87194320" class="section-title" data-legacy-id=s3d>3.4. Power calculation on experimental data</h3>
<p>After having verified the power calculation, and having confirmed that the distribution of the experimental noise was normal (Supplementary Fig. 2), we applied this method to the experimental data set described above. For each sex-mismatched experiment, we calculated the mean of all unique chromosome X log<sub>2</sub> ratios and the standard deviation of the autosomal log<sub>2</sub> ratios to provide an initial insight into the performance of each microarray platform and the values to be used in the power calculations (Table <a reveal-id="DSM002TB2" data-open="DSM002TB2" class="link link-reveal link-table xref-fig">2</a>). The average log<sub>2</sub> ratios of these chromosome X targets were similar for the BAC array platform and the Affymetrix SNP array platform (∼0.47), whereas the NimbleGen oligonucleotide platform exhibited a lower average of approximately 0.38. The average standard deviation of the log<sub>2</sub> ratios of the autosomal targets varied twofold between the different microarray platforms. The 32k BAC platform exhibited the lowest standard deviation, and the 385k NimbleGen oligonucleotide platform the highest. In addition, as all BAC array hybridizations were performed in duplicate, we were able to determine the influence of replicate analyses on the noise level. As can be seen in Table <a reveal-id="DSM002TB2" data-open="DSM002TB2" class="link link-reveal link-table xref-fig">2</a>, the autosomal standard deviation is reduced by almost 50% after averaging data from two experiments.</p>                        <a id="87194322" scrollto-destination="87194322"></a>
<div class="table-wrap"><div id="DSM002TB2" class="table-wrap-title"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Signal-to-noise parameters of the four genomic copy-number profiling platforms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Patient </th><th colspan="2">32k BAC array </th><th>Duplicate 32k BAC array </th><th colspan="2">Affymetrix 100k SNP array </th><th colspan="2">Affymetrix 250k SNP array </th><th colspan="2">NimbleGen 385k Oligonucleotide array </th></tr><tr><th>Mean X<sup>a</sup> </th><th>Auto STD<sup>b</sup> </th><th>Auto STD </th><th>Mean X </th><th>Auto STD </th><th>Mean X </th><th>Auto STD </th><th>Mean X </th><th>Auto STD </th></tr></thead><tbody><tr><td>1 </td><td>0.47 </td><td>0.10 </td><td>0.06 </td><td>0.48 </td><td>0.15 </td><td>0.49 </td><td>0.18 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>2 </td><td>0.49 </td><td>0.14 </td><td>0.08 </td><td>0.48 </td><td>0.13 </td><td>0.48 </td><td>0.14 </td><td>0.42 </td><td>0.20 </td></tr><tr><td>3 </td><td>0.49 </td><td>0.12 </td><td>0.07 </td><td>0.48 </td><td>0.16 </td><td>0.45 </td><td>0.16 </td><td>0.35 </td><td>0.25 </td></tr><tr><td>4 </td><td>0.42 </td><td>0.13 </td><td>0.07 </td><td>0.47 </td><td>0.14 </td><td>n.d. </td><td>n.d. </td><td>n.d. </td><td>n.d. </td></tr><tr><td>5 </td><td>0.40 </td><td>0.09 </td><td>0.05 </td><td>0.45 </td><td>0.16 </td><td>0.46 </td><td>0.18 </td><td>0.29 </td><td>0.24 </td></tr><tr><td>6 </td><td>0.45 </td><td>0.10 </td><td>0.05 </td><td>0.47 </td><td>0.17 </td><td>0.43 </td><td>0.13 </td><td>0.38 </td><td>0.28 </td></tr><tr><td>7 </td><td>0.46 </td><td>0.12 </td><td>0.06 </td><td>0.48 </td><td>0.14 </td><td>0.43 </td><td>0.13 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>8 </td><td>0.47 </td><td>0.11 </td><td>0.06 </td><td>0.46 </td><td>0.13 </td><td>0.42 </td><td>0.15 </td><td>0.35 </td><td>0.24 </td></tr><tr><td>9 </td><td>0.40 </td><td>0.12 </td><td>0.06 </td><td>0.48 </td><td>0.24 </td><td>0.47 </td><td>0.16 </td><td>0.51 </td><td>0.21 </td></tr><tr><td>10 </td><td>0.49 </td><td>0.11 </td><td>0.07 </td><td>0.48 </td><td>0.13 </td><td>n.d. </td><td>n.d. </td><td>0.39 </td><td>0.22 </td></tr><tr><td>11 </td><td>0.43 </td><td>0.09 </td><td>0.06 </td><td>0.45 </td><td>0.16 </td><td>0.49 </td><td>0.14 </td><td>0.41 </td><td>0.25 </td></tr><tr><td>12 </td><td>0.56 </td><td>0.13 </td><td>0.07 </td><td>0.48 </td><td>0.16 </td><td>0.44 </td><td>0.14 </td><td>0.43 </td><td>0.21 </td></tr><tr><td>13 </td><td>0.50 </td><td>0.10 </td><td>0.06 </td><td>0.48 </td><td>0.17 </td><td>0.48 </td><td>0.17 </td><td>0.32 </td><td>0.20 </td></tr><tr><td>Average </td><td>0.46 </td><td>0.11 </td><td>0.06 </td><td>0.47 </td><td>0.16 </td><td>0.46 </td><td>0.15 </td><td>0.38 </td><td>0.23 </td></tr></tbody></table></div><div content-id="DSM002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Patient </th><th colspan="2">32k BAC array </th><th>Duplicate 32k BAC array </th><th colspan="2">Affymetrix 100k SNP array </th><th colspan="2">Affymetrix 250k SNP array </th><th colspan="2">NimbleGen 385k Oligonucleotide array </th></tr><tr><th>Mean X<sup>a</sup> </th><th>Auto STD<sup>b</sup> </th><th>Auto STD </th><th>Mean X </th><th>Auto STD </th><th>Mean X </th><th>Auto STD </th><th>Mean X </th><th>Auto STD </th></tr></thead><tbody><tr><td>1 </td><td>0.47 </td><td>0.10 </td><td>0.06 </td><td>0.48 </td><td>0.15 </td><td>0.49 </td><td>0.18 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>2 </td><td>0.49 </td><td>0.14 </td><td>0.08 </td><td>0.48 </td><td>0.13 </td><td>0.48 </td><td>0.14 </td><td>0.42 </td><td>0.20 </td></tr><tr><td>3 </td><td>0.49 </td><td>0.12 </td><td>0.07 </td><td>0.48 </td><td>0.16 </td><td>0.45 </td><td>0.16 </td><td>0.35 </td><td>0.25 </td></tr><tr><td>4 </td><td>0.42 </td><td>0.13 </td><td>0.07 </td><td>0.47 </td><td>0.14 </td><td>n.d. </td><td>n.d. </td><td>n.d. </td><td>n.d. </td></tr><tr><td>5 </td><td>0.40 </td><td>0.09 </td><td>0.05 </td><td>0.45 </td><td>0.16 </td><td>0.46 </td><td>0.18 </td><td>0.29 </td><td>0.24 </td></tr><tr><td>6 </td><td>0.45 </td><td>0.10 </td><td>0.05 </td><td>0.47 </td><td>0.17 </td><td>0.43 </td><td>0.13 </td><td>0.38 </td><td>0.28 </td></tr><tr><td>7 </td><td>0.46 </td><td>0.12 </td><td>0.06 </td><td>0.48 </td><td>0.14 </td><td>0.43 </td><td>0.13 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>8 </td><td>0.47 </td><td>0.11 </td><td>0.06 </td><td>0.46 </td><td>0.13 </td><td>0.42 </td><td>0.15 </td><td>0.35 </td><td>0.24 </td></tr><tr><td>9 </td><td>0.40 </td><td>0.12 </td><td>0.06 </td><td>0.48 </td><td>0.24 </td><td>0.47 </td><td>0.16 </td><td>0.51 </td><td>0.21 </td></tr><tr><td>10 </td><td>0.49 </td><td>0.11 </td><td>0.07 </td><td>0.48 </td><td>0.13 </td><td>n.d. </td><td>n.d. </td><td>0.39 </td><td>0.22 </td></tr><tr><td>11 </td><td>0.43 </td><td>0.09 </td><td>0.06 </td><td>0.45 </td><td>0.16 </td><td>0.49 </td><td>0.14 </td><td>0.41 </td><td>0.25 </td></tr><tr><td>12 </td><td>0.56 </td><td>0.13 </td><td>0.07 </td><td>0.48 </td><td>0.16 </td><td>0.44 </td><td>0.14 </td><td>0.43 </td><td>0.21 </td></tr><tr><td>13 </td><td>0.50 </td><td>0.10 </td><td>0.06 </td><td>0.48 </td><td>0.17 </td><td>0.48 </td><td>0.17 </td><td>0.32 </td><td>0.20 </td></tr><tr><td>Average </td><td>0.46 </td><td>0.11 </td><td>0.06 </td><td>0.47 </td><td>0.16 </td><td>0.46 </td><td>0.15 </td><td>0.38 </td><td>0.23 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Mean log<sub>2</sub>-transformed test-over-reference ratio of the X chromosome, excluding the pseudo-autosomal regions, obtained from calculations in sex-mismatched hybridization experiments. For the BAC and the NimbleGen platforms, data were obtained within each two-color experiment, for the Affymetrix SNP platform, data were combined in silico from different one-color experiments. For this analysis, four reference pool samples (two of each sex) were processed in parallel with the patient samples.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Standard deviation calculated over the log<sub>2</sub>-transformed test-over-reference ratios for all autosomal targets, excluding the genomic regions known to harbor submicroscopic copy-number variations.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194322" target="_blank">View Large</a></div></div><p>Next, the statistical power analysis was used to determine the minimum number of adjacently located autosomal targets required for the reliable detection of a single copy-number loss or gain (Table <a reveal-id="DSM002TB3" data-open="DSM002TB3" class="link link-reveal link-table xref-fig">3</a>, Supplementary Table 1). An average of four adjacently located BAC clones showing a single copy-number loss provided 95% confidence of representing a true copy-number variation. A similar power for detection of a copy-number loss required, on average, five consecutive SNPs on the 100k platform, four SNPs on the 250k platform, and eight consecutive oligonucleotides on the 385k platform. The reliable detection of a single copy-number gain required more consecutive targets, as could be expected based on the theoretical log<sub>2</sub> ratio difference between a single copy-number loss (−1) and gain (0.66). For the 32k BAC and for the 100k and 250k SNP array platforms, this increase was moderate, with one to three additional targets being required, respectively. For the 385k oligonucleotide platform, this increase was considerable, i.e. at least twice as many targets were required for reliable detection of a single copy-number gain (Fig. <a reveal-id="DSM002F2" data-open="DSM002F2" class="link link-reveal link-table xref-fig">2</a>).</p>                        <a id="87194324" scrollto-destination="87194324"></a>
<div id="DSM002F2" data-id="DSM002F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00202.jpeg?Expires=1509238563&Signature=eHJoA6ksad5RkkKU8nYRhG5RAVJ6ezkM~9UdFHA3ACvJAkuxHE41zcaRR8VHvcNHP-ysXsnwmrFGry1h806dlYlwwY-qE~3qfqJOabIrA~ejJNMjzwWxTOlq5ac6NpGypJ9QDYgpgOeNKLR0qnexpEn3YVFKgwPCnRxmnQH83FVtJXzdX9h~pY~FIyL2BfcHI7uZe30gABu3qalZWPHzxeAOxypFiw-U3zZyn~edOva6XLFque7WeRZ2unhbub61PyHk-YyKgbEof8FJ~LujLOVFyRT1f9kFd3eoXeJhB2aoeCZ7W8zNOldKiVTr~XxLvpjIBzgruHU5Jn4UQTJY1A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Result of the power analysis of the four genomic microarray platforms for detection of a single copy-number gain or loss contained by different numbers of consecutive targets. The resulting power for a single copy-number gain (dotted) and a single copy-number loss (line) are displayed for the 32k BAC array platform (A), the 100k SNP array (B), the 250k SNP array (C), and 385k oligonucleotide array platform (D). The increase in number of targets has a varying impact on the resulting power across the four different microarray platforms. In addition, the number of consecutive targets required to detect single copy-number gains differs considerably from the number of targets needed to detect a single copy-number loss, and this difference appears to be platform-dependent." path-from-xml="dsm00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194324/dsm00202.jpeg" path-from-xml="dsm00202.jpeg" target="_blank">View large</a><a section="87194324" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/dsm00202.jpeg?Expires=1509238563&Signature=UbSA8JsAjRl6SDhIo4V~YeYYvQx5jx0ZSXn5OrPO5Y95UkU~ZQrjPKAKWc95rIxiD9Xbs3~AfG36lb2pMHPQ8~ihsN59AZK5AgCKpcLYujmprZ0LpazO-31sANSvzMUKtszJ~tXMNjVV61ywtEiG855oALKeM49~K5RtfM7clArzwpS0rZIIaUQYTYEpNbFT6WCrNts3aZBWSV3UBgOzCNPvCy~v3bYrD5R7qKwM-J8InJ0TnSFzvz4VTQ0rQEGhuUPjoXxTbqnEj2dYYgIf4b5UQXqFQ59qkUerGl9FoCzLBz7yVKj-iEzvnu6MOM0ygnFUenw3RRiH1oRwLeEkbQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87194324&ar=502776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Result of the power analysis of the four genomic microarray platforms for detection of a single copy-number gain or loss contained by different numbers of consecutive targets. The resulting power for a single copy-number gain (dotted) and a single copy-number loss (line) are displayed for the 32k BAC array platform (<strong>A</strong>), the 100k SNP array (<strong>B</strong>), the 250k SNP array (<strong>C</strong>), and 385k oligonucleotide array platform (<strong>D</strong>). The increase in number of targets has a varying impact on the resulting power across the four different microarray platforms. In addition, the number of consecutive targets required to detect single copy-number gains differs considerably from the number of targets needed to detect a single copy-number loss, and this difference appears to be platform-dependent.</p></div></div><div content-id="DSM002F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/m_dsm00202.jpeg?Expires=1509238563&Signature=eHJoA6ksad5RkkKU8nYRhG5RAVJ6ezkM~9UdFHA3ACvJAkuxHE41zcaRR8VHvcNHP-ysXsnwmrFGry1h806dlYlwwY-qE~3qfqJOabIrA~ejJNMjzwWxTOlq5ac6NpGypJ9QDYgpgOeNKLR0qnexpEn3YVFKgwPCnRxmnQH83FVtJXzdX9h~pY~FIyL2BfcHI7uZe30gABu3qalZWPHzxeAOxypFiw-U3zZyn~edOva6XLFque7WeRZ2unhbub61PyHk-YyKgbEof8FJ~LujLOVFyRT1f9kFd3eoXeJhB2aoeCZ7W8zNOldKiVTr~XxLvpjIBzgruHU5Jn4UQTJY1A__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q" alt="Result of the power analysis of the four genomic microarray platforms for detection of a single copy-number gain or loss contained by different numbers of consecutive targets. The resulting power for a single copy-number gain (dotted) and a single copy-number loss (line) are displayed for the 32k BAC array platform (A), the 100k SNP array (B), the 250k SNP array (C), and 385k oligonucleotide array platform (D). The increase in number of targets has a varying impact on the resulting power across the four different microarray platforms. In addition, the number of consecutive targets required to detect single copy-number gains differs considerably from the number of targets needed to detect a single copy-number loss, and this difference appears to be platform-dependent." path-from-xml="dsm00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87194324/dsm00202.jpeg" path-from-xml="dsm00202.jpeg" target="_blank">View large</a><a section="87194324" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/dsm00202.jpeg?Expires=1509238563&Signature=UbSA8JsAjRl6SDhIo4V~YeYYvQx5jx0ZSXn5OrPO5Y95UkU~ZQrjPKAKWc95rIxiD9Xbs3~AfG36lb2pMHPQ8~ihsN59AZK5AgCKpcLYujmprZ0LpazO-31sANSvzMUKtszJ~tXMNjVV61ywtEiG855oALKeM49~K5RtfM7clArzwpS0rZIIaUQYTYEpNbFT6WCrNts3aZBWSV3UBgOzCNPvCy~v3bYrD5R7qKwM-J8InJ0TnSFzvz4VTQ0rQEGhuUPjoXxTbqnEj2dYYgIf4b5UQXqFQ59qkUerGl9FoCzLBz7yVKj-iEzvnu6MOM0ygnFUenw3RRiH1oRwLeEkbQ__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&sec=87194324&ar=502776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Result of the power analysis of the four genomic microarray platforms for detection of a single copy-number gain or loss contained by different numbers of consecutive targets. The resulting power for a single copy-number gain (dotted) and a single copy-number loss (line) are displayed for the 32k BAC array platform (<strong>A</strong>), the 100k SNP array (<strong>B</strong>), the 250k SNP array (<strong>C</strong>), and 385k oligonucleotide array platform (<strong>D</strong>). The increase in number of targets has a varying impact on the resulting power across the four different microarray platforms. In addition, the number of consecutive targets required to detect single copy-number gains differs considerably from the number of targets needed to detect a single copy-number loss, and this difference appears to be platform-dependent.</p></div></div>                        <a id="87194325" scrollto-destination="87194325"></a>
<div class="table-wrap"><div id="DSM002TB3" class="table-wrap-title"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Result of the statistical power analysis: How many consecutive targets are required to detect a single copy-number loss or gain?</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Patient </th><th colspan="2">32k BAC array </th><th colspan="2">Duplicate 32k BAC array </th><th colspan="2">Affymetrix 100k SNP array </th><th colspan="2">Affymetrix 250k SNP array </th><th colspan="2">NimbleGen 385k Oligonucleotide array </th></tr><tr><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th></tr></thead><tbody><tr><td>1 </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>4 </td><td>6 </td><td>5 </td><td>7 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>2 </td><td>4 </td><td>11 </td><td>3 </td><td>4 </td><td>4 </td><td>6 </td><td>4 </td><td>6 </td><td>5 </td><td>13 </td></tr><tr><td>3 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>4 </td><td>7 </td><td>5 </td><td>7 </td><td>10 </td><td>19 </td></tr><tr><td>4 </td><td>4 </td><td>6 </td><td>3 </td><td>4 </td><td>4 </td><td>6 </td><td>n.d. </td><td>n.d. </td><td>n.d. </td><td>n.d. </td></tr><tr><td>5 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>5 </td><td>8 </td><td>5 </td><td>8 </td><td>11 </td><td>31 </td></tr><tr><td>6 </td><td>3 </td><td>5 </td><td>3 </td><td>3 </td><td>5 </td><td>8 </td><td>4 </td><td>6 </td><td>9 </td><td>27 </td></tr><tr><td>7 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>4 </td><td>6 </td><td>4 </td><td>6 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>8 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>4 </td><td>6 </td><td>5 </td><td>8 </td><td>8 </td><td>24 </td></tr><tr><td>9 </td><td>4 </td><td>5 </td><td>3 </td><td>4 </td><td>7 </td><td>9 </td><td>4 </td><td>7 </td><td>5 </td><td>8 </td></tr><tr><td>10 </td><td>3 </td><td>5 </td><td>3 </td><td>4 </td><td>4 </td><td>6 </td><td>n.d. </td><td>n.d. </td><td>7 </td><td>14 </td></tr><tr><td>11 </td><td>3 </td><td>5 </td><td>3 </td><td>4 </td><td>5 </td><td>7 </td><td>4 </td><td>6 </td><td>7 </td><td>18 </td></tr><tr><td>12 </td><td>3 </td><td>5 </td><td>3 </td><td>4 </td><td>5 </td><td>8 </td><td>4 </td><td>5 </td><td>5 </td><td>16 </td></tr><tr><td>13 </td><td>3 </td><td>4 </td><td>3 </td><td>4 </td><td>5 </td><td>7 </td><td>4 </td><td>7 </td><td>9 </td><td>14 </td></tr><tr><td>Average </td><td>4 </td><td>5 </td><td>3 </td><td>4 </td><td>5 </td><td>7 </td><td>4 </td><td>7 </td><td>8 </td><td>18 </td></tr></tbody></table></div><div content-id="DSM002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Patient </th><th colspan="2">32k BAC array </th><th colspan="2">Duplicate 32k BAC array </th><th colspan="2">Affymetrix 100k SNP array </th><th colspan="2">Affymetrix 250k SNP array </th><th colspan="2">NimbleGen 385k Oligonucleotide array </th></tr><tr><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th></tr></thead><tbody><tr><td>1 </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>4 </td><td>6 </td><td>5 </td><td>7 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>2 </td><td>4 </td><td>11 </td><td>3 </td><td>4 </td><td>4 </td><td>6 </td><td>4 </td><td>6 </td><td>5 </td><td>13 </td></tr><tr><td>3 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>4 </td><td>7 </td><td>5 </td><td>7 </td><td>10 </td><td>19 </td></tr><tr><td>4 </td><td>4 </td><td>6 </td><td>3 </td><td>4 </td><td>4 </td><td>6 </td><td>n.d. </td><td>n.d. </td><td>n.d. </td><td>n.d. </td></tr><tr><td>5 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>5 </td><td>8 </td><td>5 </td><td>8 </td><td>11 </td><td>31 </td></tr><tr><td>6 </td><td>3 </td><td>5 </td><td>3 </td><td>3 </td><td>5 </td><td>8 </td><td>4 </td><td>6 </td><td>9 </td><td>27 </td></tr><tr><td>7 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>4 </td><td>6 </td><td>4 </td><td>6 </td><td>n.d. </td><td>n.d. </td></tr><tr><td>8 </td><td>4 </td><td>5 </td><td>3 </td><td>3 </td><td>4 </td><td>6 </td><td>5 </td><td>8 </td><td>8 </td><td>24 </td></tr><tr><td>9 </td><td>4 </td><td>5 </td><td>3 </td><td>4 </td><td>7 </td><td>9 </td><td>4 </td><td>7 </td><td>5 </td><td>8 </td></tr><tr><td>10 </td><td>3 </td><td>5 </td><td>3 </td><td>4 </td><td>4 </td><td>6 </td><td>n.d. </td><td>n.d. </td><td>7 </td><td>14 </td></tr><tr><td>11 </td><td>3 </td><td>5 </td><td>3 </td><td>4 </td><td>5 </td><td>7 </td><td>4 </td><td>6 </td><td>7 </td><td>18 </td></tr><tr><td>12 </td><td>3 </td><td>5 </td><td>3 </td><td>4 </td><td>5 </td><td>8 </td><td>4 </td><td>5 </td><td>5 </td><td>16 </td></tr><tr><td>13 </td><td>3 </td><td>4 </td><td>3 </td><td>4 </td><td>5 </td><td>7 </td><td>4 </td><td>7 </td><td>9 </td><td>14 </td></tr><tr><td>Average </td><td>4 </td><td>5 </td><td>3 </td><td>4 </td><td>5 </td><td>7 </td><td>4 </td><td>7 </td><td>8 </td><td>18 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>For this analysis, we used a power of 95%.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194325" target="_blank">View Large</a></div></div><p>These power analysis results can be translated into genome-wide copy-number detection resolutions by combining these results with the genomic coverage of the different microarray platforms (Table <a reveal-id="DSM002TB4" data-open="DSM002TB4" class="link link-reveal link-table xref-fig">4</a>, Supplementary Table 2, Supplementary Fig. 3). This resulted for each platform in a probability to detect a single copy-number gain or a loss throughout the genome with a size range from 10 kb to 1 Mb and a desired power of 95%. From this analysis, it can be concluded that (1) high-density oligonucleotide/SNP-based platforms are significantly better in detecting copy-number variations below 1 Mb than the BAC array platform, (2) copy-number variations smaller than 100 kb remain difficult to detect even onto these high-density platforms, despite an average target spacing of 7, 12, or 30 kb, and (3) small-sized single copy-number gains are much more difficult to detect than single copy-number losses of the same size.</p>                        <a id="87194327" scrollto-destination="87194327"></a>
<div class="table-wrap"><div id="DSM002TB4" class="table-wrap-title"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Probability to detect a single copy-number gain or loss with different genomic sizes onto the four platforms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">32k BAC array </th><th colspan="2">Affymetrix 100k SNP array </th><th colspan="2">Affymetrix 250k SNP array </th><th colspan="2">NimbleGen 385k Oligonucleotide array </th></tr><tr><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th></tr></thead><tbody><tr><td>10 kb </td><td>0.00 </td><td>0.00 </td><td>0.03 </td><td>0.01 </td><td>0.12 </td><td>0.02 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>50 kb </td><td>0.01 </td><td>0.00 </td><td>0.28 </td><td>0.14 </td><td>0.68 </td><td>0.38 </td><td>0.28 </td><td>0.00 </td></tr><tr><td>100 kb </td><td>0.02 </td><td>0.01 </td><td>0.56 </td><td>0.39 </td><td>0.88 </td><td>0.75 </td><td>0.94 </td><td>0.01 </td></tr><tr><td>200 kb </td><td>0.11 </td><td>0.05 </td><td>0.81 </td><td>0.71 </td><td>0.94 </td><td>0.91 </td><td>0.95 </td><td>0.93 </td></tr><tr><td>300 kb </td><td>0.32 </td><td>0.16 </td><td>0.88 </td><td>0.83 </td><td>0.95 </td><td>0.94 </td><td>0.95 </td><td>0.94 </td></tr><tr><td>400 kb </td><td>0.60 </td><td>0.36 </td><td>0.91 </td><td>0.88 </td><td>0.95 </td><td>0.94 </td><td>0.95 </td><td>0.94 </td></tr><tr><td>500 kb </td><td>0.83 </td><td>0.60 </td><td>0.93 </td><td>0.91 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td></tr><tr><td>1 Mb </td><td>0.95 </td><td>0.95 </td><td>0.94 </td><td>0.94 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td></tr></tbody></table></div><div content-id="DSM002TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">32k BAC array </th><th colspan="2">Affymetrix 100k SNP array </th><th colspan="2">Affymetrix 250k SNP array </th><th colspan="2">NimbleGen 385k Oligonucleotide array </th></tr><tr><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th><th>Loss </th><th>Gain </th></tr></thead><tbody><tr><td>10 kb </td><td>0.00 </td><td>0.00 </td><td>0.03 </td><td>0.01 </td><td>0.12 </td><td>0.02 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>50 kb </td><td>0.01 </td><td>0.00 </td><td>0.28 </td><td>0.14 </td><td>0.68 </td><td>0.38 </td><td>0.28 </td><td>0.00 </td></tr><tr><td>100 kb </td><td>0.02 </td><td>0.01 </td><td>0.56 </td><td>0.39 </td><td>0.88 </td><td>0.75 </td><td>0.94 </td><td>0.01 </td></tr><tr><td>200 kb </td><td>0.11 </td><td>0.05 </td><td>0.81 </td><td>0.71 </td><td>0.94 </td><td>0.91 </td><td>0.95 </td><td>0.93 </td></tr><tr><td>300 kb </td><td>0.32 </td><td>0.16 </td><td>0.88 </td><td>0.83 </td><td>0.95 </td><td>0.94 </td><td>0.95 </td><td>0.94 </td></tr><tr><td>400 kb </td><td>0.60 </td><td>0.36 </td><td>0.91 </td><td>0.88 </td><td>0.95 </td><td>0.94 </td><td>0.95 </td><td>0.94 </td></tr><tr><td>500 kb </td><td>0.83 </td><td>0.60 </td><td>0.93 </td><td>0.91 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td></tr><tr><td>1 Mb </td><td>0.95 </td><td>0.95 </td><td>0.94 </td><td>0.94 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td><td>0.95 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>This table combines the results from Table <a reveal-id="DSM002TB2" data-open="DSM002TB2" class="link link-reveal link-table xref-fig">3</a> with those of Supplementary Table 2. For this analysis, we used a power of 95%.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87194327" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87194328 id="87194328" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We have developed a novel method for establishing the practical resolution of a genomic microarray to detect copy-number variation and applied this method, based on statistical power analysis, to three commercially available microarray platforms and to our in-house BAC microarray platform. For each platform, we calculated the number of adjacent targets required to reliably detect a single copy-number variation (gain or loss), given the required minimum rate of false-positives and false-negatives. On the basis of this calculation, we determined the probability of detecting copy-number variations of different sizes onto a genomic microarray, taking into account the number and genomic position of all targets on the microarray platform used. This unbiased resolution statistic is an important performance measure for genomic microarray platforms as well as for individual microarray experiments, which had not been established for genomic microarrays before. Previously, the resolution of a genomic microarray could only be judged by the mean spacing of targets, a measure that solely reflects the overall genomic coverage. The results of our power analysis, however, clearly demonstrate that the level of noise and the sensitivity of copy-number measurements co-determine the practical resolution. In addition, our analysis revealed an unexpected platform-dependent difference in sensitivity to detect a single copy-number loss and a single copy-number gain. Accurate performance measures are important for researchers to gage the sensitivity and specificity of individual experiments or different platforms. Also, in a diagnostic setting, where microarray-based genome profiling is rapidly being introduced,<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSM002C30" data-open="DSM002C30" class="link link-ref link-reveal xref-bibr">30</a></sup> it will be essential to have a robust estimate of the practical resolution of the genome-wide scan.</p><p>Several platform comparisons have been performed for gene expression microarrays.<sup><a reveal-id="DSM002C32  DSM002C35" data-open="DSM002C32  DSM002C35" class="link link-ref link-reveal xref-bibr">32–35</a></sup> The statistical methods described in these studies, however, cannot be used for genomic microarrays as they do not account for various intrinsic aspects of genomic microarrays such as the adjacency of targets and the difference between detecting a single copy-number loss and a single copy-number gain. Several statistical methods have been developed for different aspects of genomic microarray analysis, such as preprocessing (normalization<sup><a reveal-id="DSM002C17" data-open="DSM002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSM002C28" data-open="DSM002C28" class="link link-ref link-reveal xref-bibr">28</a></sup>), automatic detection of copy-number variations,<sup><a reveal-id="DSM002C40" data-open="DSM002C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSM002C41" data-open="DSM002C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and the analysis of Type I errors across genomic microarrays obtained from multiple experiments and samples.<sup><a reveal-id="DSM002C42" data-open="DSM002C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Here, we report on a resolution statistic for genomic microarrays that uses an approach based on hypothesis testing and statistical power calculations. The method is based on the following variables: (1) the genomic coverage of the platform, (2) an estimate of the noise in the microarray experiment (the standard deviation of the autosomal targets), (3) an estimate of a single copy-number loss (ratio of the chromosome X unique regions of sex-mismatch experiments),<sup><a reveal-id="DSM002C37" data-open="DSM002C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and (4) the desired statistical power. The method requires a normal distribution of the noise, which was confirmed by a χ<sup>2</sup>-test, thereby allowing the use of the <em>T</em> distributions. We validated our method using a Monte Carlo technique to simulate the Type I and Type II detection errors for a single copy-number loss, requiring a statistical power of 95%. These simulations were in agreement with the calculated Type I and Type II errors resulting from a two-sided <em>t</em>-test. The calculations yielded the minimal number of required adjacent targets at each locus in order to detect a single copy-number variation, taking into account the required statistical power.<sup><a reveal-id="DSM002C36" data-open="DSM002C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSM002C43" data-open="DSM002C43" class="link link-ref link-reveal xref-bibr">43</a></sup> By combining the genomic location of targets with the minimally required sample size, we obtained an objective genome-wide resolution statistic.</p><p>We used our method to characterize the detection performance of the four genomic microarray platforms using experimental data from 13 samples with submicroscopic copy-number variations hybridized to the different platforms. Automatic copy-number analyses detected the large majority of known submicroscopic copy-number variations on all genomic microarray platforms. Two genomic variations were not detected by the 100k SNP array platform, due to poor SNP coverage for these regions, a problem reported also by others<sup><a reveal-id="DSM002C44" data-open="DSM002C44" class="link link-ref link-reveal xref-bibr">44</a></sup> (see also Supplementary Fig. 3). This can be reduced by simply adding more targets for these regions. Indeed, one of the two variations was identified automatically by the 250k SNP array. This analysis also revealed considerable and reproducible differences in signal-to-noise ratios between the different platforms. Signal-to-noise ratios were highest for the BAC array platform, which may be due to a more robust hybridization performance of larger genomic fragments as compared to the smaller targets used for the other platforms. The Affymetrix SNP arrays containing only 25-mers, however, showed signal-to-noise ratios which were only slightly lower than the BAC array platform after data normalization using the CNAG package.<sup><a reveal-id="DSM002C28" data-open="DSM002C28" class="link link-ref link-reveal xref-bibr">28</a></sup> It should be noted that no such data preprocessing was performed for the NimbleGen oligonucleotide platform which displayed the lowest signal-to-noise ratios. This may indicate that preprocessing of the data can have a significant effect on the detection resolution of an individual genomic microarray experiment and argues for a significant effort to be made in this field of genomic microarray data analysis.<sup><a reveal-id="DSM002C45" data-open="DSM002C45" class="link link-ref link-reveal xref-bibr">45</a></sup> In addition, the noise in a genomic microarray experiment can be significantly reduced using replicate analyses, as was shown for the BAC array platform.</p><p>The statistical power analysis indicated that, on average, four consecutive BACs are required for the reliable detection of a single copy-number loss, five for the 100k SNP array platform, four for the 250k SNP array platform, and eight consecutive oligonucleotide probes for the NimbleGen 385k oligonucleotide platform. These numbers are markedly different for the detection of single copy-number gains (see Fig. <a reveal-id="DSM002F2" data-open="DSM002F2" class="link link-reveal link-table xref-fig">2</a>). This is caused by the fact that the theoretical ratio of a single copy-number gain (three vs. two copies) is much closer to the random noise level than a single copy-number loss (one vs. two copies). Therefore, it is relatively difficult to discriminate between a true copy-number gain and random experimental noise. This poses a serious problem for those platforms that display a high noise level. The estimate for a single copy-number loss on the NimbleGen oligonucleotide platform is − 0.38 and that for a single copy-number gain is 0.22, within one standard deviation of the mean (0.23 for this platform, see Table <a reveal-id="DSM002TB2" data-open="DSM002TB2" class="link link-reveal link-table xref-fig">2</a> and Supplementary Table 1). As a consequence, reliable detection of a single copy-number gain on this platform requires 10 consecutive oligonucleotides more (18) than detection of a single copy-number loss (8). In contrast, the detection of a single copy-number gain on the BAC array platform with the lowest noise level requires only one consecutive clone more (5) than that of a single copy-number loss (4). These results demonstrate the impact of the different detection limits regarding single losses and gains, resulting in more targets being required in the latter case.<sup><a reveal-id="DSM002C46" data-open="DSM002C46" class="link link-ref link-reveal xref-bibr">46</a></sup> It is important to account for this asymmetric detection limit caused by the different signal-to-noise ratios associated with gains and losses.</p><p>The commercially available microarrays contain 3 to 12 times as many targets as our tiling-resolution BAC microarray, and this can compensate for the lower signal-to-nose level obtained on these platforms. In addition, the targets on these microarrays are much smaller in size as compared to BAC clones with an average insert size of 170 kb, thereby theoretically allowing the detection of aberrations below 100 kb. Table <a reveal-id="DSM002TB4" data-open="DSM002TB4" class="link link-reveal link-table xref-fig">4</a> shows the probability of detecting a single copy-number gain or loss with different genomic sizes onto the four platforms. This table clearly shows that the commercial platforms outperform the BAC array platform for the detection of aberrations below 1 Mb in size. The Affymetrix 250k SNP array appeared most sensitive for the detection of copy-number variations below the 100-kb level, especially for copy-number gains. However, even on this platform, the probability of detecting a single copy-number gain with a genomic size of 50 kb was only 38% (68% for a single copy-number loss). A similar analysis was performed in silico for the 500k SNP array platform by assuming that the 250k Sty array shows a similar sensitivity and noise distribution as the 250k Nsp array used in this study. This calculation indicated for the 500k SNP array that the probability of detecting a single copy-number loss or gain with a genomic size of 50 kb was 87 and 72%, respectively (Supplementary Table 2). This shows that even these high-density platforms will have significant difficulties in detecting single copy-number variations smaller than 100 kb. As stated above, the use of replicate measurements and/or improvements in data preprocessing can significantly improve the sensitivity of the different genomic microarray platforms.</p><p>Next to performance, many other factors, including the availability and consistency in quality of microarrays over time, the amount and quality of input DNA required, the price, and the access to a microarray facility or service company, may affect the choice for a certain microarray platform. An important advantage of using a widely available commercial platform is that it facilitates data exchange between research groups. In addition, the production of arrays containing more than a hundred thousand targets is not practically achievable for academic groups, especially since most currently available microarray spotters have a practical limitation of ∼60,000 spots per slide. An important bonus of using SNP arrays is that it allows genotyping together with CGH. This provides additional information such as copy-number neutral loss-of-heterozygosity. Initial SNP selection against regions with a high frequency of copy-number variation in the population, however, has recently been shown to impact the detection of this specific form of copy-number variation on these platforms.<sup><a reveal-id="DSM002C44" data-open="DSM002C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Besides Affymetrix and NimbleGen, companies such as Agilent and Illumina have also developed high-density genomic microarrays that can be used for CGH applications.<sup><a reveal-id="DSM002C24" data-open="DSM002C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSM002C31" data-open="DSM002C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>In conclusion, we present a straightforward statistical method for establishing the practical resolution of an individual genomic microarray experiment. Application of this method to different genomic microarray platforms clearly shows that these platforms vary in their capacity to reliably detect copy-number variations of different sizes and different types. This should be taken into account for estimating the practical resolution of a platform to detect genomic copy-number variations.</p>                            <h2 scrollto-destination=87194336 id="87194336" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Rolph Pfundt, Bert de Vries, and Han Brunner for useful discussions and critical proofreading of the manuscript. This work was supported by grants from the Netherlands Organization for Health Research and Development (to J.A.V., ZonMW 912-04-047 and 917-66-363).</p>                            <h2 scrollto-destination=87194338 id="87194338" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsm002c1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pinkel</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Albertson</div> <div class="given-names">D. G.</div></div>. </div><div class="article-title">Comparative genomic hybridization</div>, <div class="source">Annu. Rev. Genomics Hum. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">331</div>-<div class="lpage">354</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20hybridization&amp;author=D.%20Pinkel&amp;author=D.%20G.%20Albertson&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Genomics%20Hum.%20Genet.&amp;volume=6&amp;pages=331-354" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genom.6.080604.162140" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genom.6.080604.162140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genom.6.080604.162140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16124865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Speicher</div> <div class="given-names">M. R.</div></div>,  <div class="name"><div class="surname">Carter</div> <div class="given-names">N. P.</div></div>. </div><div class="article-title">The new cytogenetics: blurring the boundaries with molecular biology</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">782</div>-<div class="lpage">792</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20new%20cytogenetics%3A%20blurring%20the%20boundaries%20with%20molecular%20biology&amp;author=M.%20R.%20Speicher&amp;author=N.%20P.%20Carter&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=782-792" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1692" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16145555" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snijders</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Nowak</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Segraves</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Assembly of microarrays for genome-wide measurement of DNA copy number</div>, <div class="source">Nat. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">263</div>-<div class="lpage">264</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20of%20microarrays%20for%20genome-wide%20measurement%20of%20DNA%20copy%20number&amp;author=A.%20M.%20Snijders&amp;author=N.%20Nowak&amp;author=R.%20Segraves&amp;publication_year=2001&amp;journal=Nat.%20Genet.&amp;volume=29&amp;pages=263-264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng754" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11687795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fiegler</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Carr</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Douglas</div> <div class="given-names">E. J.</div></div>, et al. </div><div class="article-title">DNA microarrays for comparative genomic hybridization based on DOP-PCR amplification of BAC and PAC clones</div>, <div class="source">Genes Chromosomes Cancer</div> , <div class="year">2003</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">361</div>-<div class="lpage">374</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20microarrays%20for%20comparative%20genomic%20hybridization%20based%20on%20DOP-PCR%20amplification%20of%20BAC%20and%20PAC%20clones&amp;author=H.%20Fiegler&amp;author=P.%20Carr&amp;author=E.%20J.%20Douglas&amp;publication_year=2003&amp;journal=Genes%20Chromosomes%20Cancer&amp;volume=36&amp;pages=361-374" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-2264" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-2264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-2264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12619160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vissers</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">de Vries</div> <div class="given-names">B. B.</div></div>,  <div class="name"><div class="surname">Osoegawa</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Array-based comparative genomic hybridization for the genome-wide detection of submicroscopic chromosomal abnormalities</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">1261</div>-<div class="lpage">1270</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array-based%20comparative%20genomic%20hybridization%20for%20the%20genome-wide%20detection%20of%20submicroscopic%20chromosomal%20abnormalities&amp;author=L.%20E.%20Vissers&amp;author=B.%20B.%20de%20Vries&amp;author=K.%20Osoegawa&amp;publication_year=2003&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=73&amp;pages=1261-1270" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/379977" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F379977" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F379977"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14628292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishkanian</div> <div class="given-names">A. S.</div></div>,  <div class="name"><div class="surname">Malloff</div> <div class="given-names">C. A.</div></div>,  <div class="name"><div class="surname">Watson</div> <div class="given-names">S. K.</div></div>, et al. </div><div class="article-title">A tiling resolution DNA microarray with complete coverage of the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">299</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tiling%20resolution%20DNA%20microarray%20with%20complete%20coverage%20of%20the%20human%20genome&amp;author=A.%20S.%20Ishkanian&amp;author=C.%20A.%20Malloff&amp;author=S.%20K.%20Watson&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=299-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1307" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14981516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bosdet</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Smailus</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">A set of BAC clones spanning the human genome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">3651</div>-<div class="lpage">3660</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20set%20of%20BAC%20clones%20spanning%20the%20human%20genome&amp;author=M.%20Krzywinski&amp;author=I.%20Bosdet&amp;author=D.%20Smailus&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=3651-3660" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh700" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh700"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davies</div> <div class="given-names">J. J.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">I. M.</div></div>,  <div class="name"><div class="surname">Lam</div> <div class="given-names">W. L.</div></div>. </div><div class="article-title">Array CGH technologies and their applications to cancer genomes</div>, <div class="source">Chromosome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">237</div>-<div class="lpage">248</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array%20CGH%20technologies%20and%20their%20applications%20to%20cancer%20genomes&amp;author=J.%20J.%20Davies&amp;author=I.%20M.%20Wilson&amp;author=W.%20L.%20Lam&amp;publication_year=2005&amp;journal=Chromosome%20Res.&amp;volume=13&amp;pages=237-248" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-005-2168-x" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-005-2168-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-005-2168-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15868418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jonsson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Naylor</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Vallon-Christersson</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Distinct genomic profiles in hereditary breast tumors identified by array-based comparative genomic hybridization</div>, <div class="source">Cancer Res.</div> , <div class="year">2005</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">7612</div>-<div class="lpage">7621</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20genomic%20profiles%20in%20hereditary%20breast%20tumors%20identified%20by%20array-based%20comparative%20genomic%20hybridization&amp;author=G.%20Jonsson&amp;author=T.%20L.%20Naylor&amp;author=J.%20Vallon-Christersson&amp;publication_year=2005&amp;journal=Cancer%20Res.&amp;volume=65&amp;pages=7612-7621" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16140926" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pinkel</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Albertson</div> <div class="given-names">D. G.</div></div>. </div><div class="article-title">Array comparative genomic hybridization and its applications in cancer</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> <div class="supplement">suppl.</div>(pg. <div class="fpage">S11</div>-<div class="lpage">S17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array%20comparative%20genomic%20hybridization%20and%20its%20applications%20in%20cancer&amp;author=D.%20Pinkel&amp;author=D.%20G.%20Albertson&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=S11-S17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1569" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15920524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vissers</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">van Ravenswaaij</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Admiraal</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Mutations in a new member of the chromodomain gene family cause CHARGE syndrome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">955</div>-<div class="lpage">957</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20in%20a%20new%20member%20of%20the%20chromodomain%20gene%20family%20cause%20CHARGE%20syndrome&amp;author=L.%20E.%20Vissers&amp;author=C.%20M.%20van%20Ravenswaaij&amp;author=R.%20Admiraal&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=955-957" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1407" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15300250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vissers</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">Veltman</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Geurts van Kessel</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Brunner</div> <div class="given-names">H. G.</div></div>. </div><div class="article-title">Identification of disease genes by whole genome CGH arrays</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">R215</div>-<div class="lpage">R223</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20disease%20genes%20by%20whole%20genome%20CGH%20arrays&amp;author=L.%20E.%20Vissers&amp;author=J.%20A.%20Veltman&amp;author=A.%20Geurts%20van%20Kessel&amp;author=H.%20G.%20Brunner&amp;publication_year=2005&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=14&amp;pages=R215-R223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddi268" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddi268" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddi268"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16244320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koolen</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Vissers</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">Pfundt</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A new chromosome 17q21.31 microdeletion syndrome associated with a common inversion polymorphism</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">999</div>-<div class="lpage">1001</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20chromosome%2017q21.31%20microdeletion%20syndrome%20associated%20with%20a%20common%20inversion%20polymorphism&amp;author=D.%20A.%20Koolen&amp;author=L.%20E.%20Vissers&amp;author=R.%20Pfundt&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=999-1001" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1853" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16906164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shaw-Smith</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pittman</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Willatt</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Microdeletion encompassing MAPT at chromosome 17q21.3 is associated with developmental delay and learning disability</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1032</div>-<div class="lpage">1037</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microdeletion%20encompassing%20MAPT%20at%20chromosome%2017q21.3%20is%20associated%20with%20developmental%20delay%20and%20learning%20disability&amp;author=C.%20Shaw-Smith&amp;author=A.%20M.%20Pittman&amp;author=L.%20Willatt&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=1032-1037" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1858" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1858" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1858"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16906163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Hansen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Selzer</div> <div class="given-names">R. R.</div></div>, et al. </div><div class="article-title">Discovery of previously unidentified genomic disorders from the duplication architecture of the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1038</div>-<div class="lpage">1042</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20previously%20unidentified%20genomic%20disorders%20from%20the%20duplication%20architecture%20of%20the%20human%20genome&amp;author=A.%20J.%20Sharp&amp;author=S.%20Hansen&amp;author=R.%20R.%20Selzer&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=1038-1042" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1862" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16906162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shaw-Smith</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Redon</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rickman</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Microarray based comparative genomic hybridisation (array-CGH) detects submicroscopic chromosomal deletions and duplications in patients with learning disability/mental retardation and dysmorphic features</div>, <div class="source">J. Med. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">241</div>-<div class="lpage">248</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20based%20comparative%20genomic%20hybridisation%20(array-CGH)%20detects%20submicroscopic%20chromosomal%20deletions%20and%20duplications%20in%20patients%20with%20learning%20disability%2Fmental%20retardation%20and%20dysmorphic%20features&amp;author=C.%20Shaw-Smith&amp;author=R.%20Redon&amp;author=L.%20Rickman&amp;publication_year=2004&amp;journal=J.%20Med.%20Genet.&amp;volume=41&amp;pages=241-248" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/jmg.2003.017731" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fjmg.2003.017731" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fjmg.2003.017731"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Vries</div> <div class="given-names">B. B. A.</div></div>,  <div class="name"><div class="surname">Pfundt</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Leisink</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Diagnostic genome profiling in mental retardation</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">606</div>-<div class="lpage">616</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diagnostic%20genome%20profiling%20in%20mental%20retardation&amp;author=B.%20B.%20A.%20de%20Vries&amp;author=R.%20Pfundt&amp;author=M.%20Leisink&amp;publication_year=2005&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=77&amp;pages=606-616" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/491719" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F491719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F491719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16175506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rickman</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Fiegler</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shaw-Smith</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Prenatal detection of unbalanced chromosomal rearrangements by array-CGH</div>, <div class="source">J. Med. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">353</div>-<div class="lpage">361</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prenatal%20detection%20of%20unbalanced%20chromosomal%20rearrangements%20by%20array-CGH&amp;author=L.%20Rickman&amp;author=H.%20Fiegler&amp;author=C.%20Shaw-Smith&amp;publication_year=2006&amp;journal=J.%20Med.%20Genet.&amp;volume=43&amp;pages=353-361" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/jmg.2005.037648" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fjmg.2005.037648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fjmg.2005.037648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16199537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roa</div> <div class="given-names">B. B.</div></div>,  <div class="name"><div class="surname">Pulliam</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Eng</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Cheung</div> <div class="given-names">S. W.</div></div>. </div><div class="article-title">Evolution of prenatal genetics: from point mutation testing to chromosomal microarray analysis</div>, <div class="source">Expert Rev. Mol. Diagn.</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">883</div>-<div class="lpage">892</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20prenatal%20genetics%3A%20from%20point%20mutation%20testing%20to%20chromosomal%20microarray%20analysis&amp;author=B.%20B.%20Roa&amp;author=J.%20Pulliam&amp;author=C.%20M.%20Eng&amp;author=S.%20W.%20Cheung&amp;publication_year=2005&amp;journal=Expert%20Rev.%20Mol.%20Diagn.&amp;volume=5&amp;pages=883-892" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1586/14737159.5.6.883" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1586%2F14737159.5.6.883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1586%2F14737159.5.6.883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16255630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iafrate</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Feuk</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Rivera</div> <div class="given-names">M. N.</div></div>, et al. </div><div class="article-title">Detection of large-scale variation in the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">949</div>-<div class="lpage">951</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20large-scale%20variation%20in%20the%20human%20genome&amp;author=A.%20J.%20Iafrate&amp;author=L.%20Feuk&amp;author=M.%20N.%20Rivera&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=949-951" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1416" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15286789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sebat</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lakshmi</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Troge</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Large-scale copy number polymorphism in the human genome</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">305</div> (pg. <div class="fpage">525</div>-<div class="lpage">528</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20copy%20number%20polymorphism%20in%20the%20human%20genome&amp;author=J.%20Sebat&amp;author=B.%20Lakshmi&amp;author=J.%20Troge&amp;publication_year=2004&amp;journal=Science&amp;volume=305&amp;pages=525-528" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1098918" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1098918" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1098918"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15273396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Locke</div> <div class="given-names">D. P.</div></div>,  <div class="name"><div class="surname">McGrath</div> <div class="given-names">S. D.</div></div>, et al. </div><div class="article-title">Segmental duplications and copy-number variation in the human genome</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">78</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmental%20duplications%20and%20copy-number%20variation%20in%20the%20human%20genome&amp;author=A.%20J.%20Sharp&amp;author=D.%20P.%20Locke&amp;author=S.%20D.%20McGrath&amp;publication_year=2005&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=77&amp;pages=78-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/431652" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F431652" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F431652"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15918152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feuk</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Carson</div> <div class="given-names">A. R.</div></div>,  <div class="name"><div class="surname">Scherer</div> <div class="given-names">S. W.</div></div>. </div><div class="article-title">Structural variation in the human genome</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">85</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20variation%20in%20the%20human%20genome&amp;author=L.%20Feuk&amp;author=A.%20R.%20Carson&amp;author=S.%20W.%20Scherer&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=85-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1767" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1767" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1767"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16418744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrett</div> <div class="given-names">M. T.</div></div>,  <div class="name"><div class="surname">Scheffer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ben-Dor</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Comparative genomic hybridization using oligonucleotide microarrays and total genomic DNA</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">17765</div>-<div class="lpage">17770</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20hybridization%20using%20oligonucleotide%20microarrays%20and%20total%20genomic%20DNA&amp;author=M.%20T.%20Barrett&amp;author=A.%20Scheffer&amp;author=A.%20Ben-Dor&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=17765-17770" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0407979101" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0407979101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0407979101"> </span></div> </div></div></div></div></div><div content-id="dsm002c25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van den Ijssel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Tijssen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chin</div> <div class="given-names">S. F.</div></div>, et al. </div><div class="article-title">Human and mouse oligonucleotide-based array CGH</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> pg. <div class="fpage">e192</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20and%20mouse%20oligonucleotide-based%20array%20CGH&amp;author=P.%20van%20den%20Ijssel&amp;author=M.%20Tijssen&amp;author=S.%20F.%20Chin&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=e192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gni191" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgni191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgni191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16361265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Selzer</div> <div class="given-names">R. R.</div></div>,  <div class="name"><div class="surname">Richmond</div> <div class="given-names">T. A.</div></div>,  <div class="name"><div class="surname">Pofahl</div> <div class="given-names">N. J.</div></div>, et al. </div><div class="article-title">Analysis of chromosome breakpoints in neuroblastoma at sub-kilobase resolution using fine-tiling oligonucleotide array CGH</div>, <div class="source">Genes Chromosomes Cancer</div> , <div class="year">2005</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">305</div>-<div class="lpage">319</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20chromosome%20breakpoints%20in%20neuroblastoma%20at%20sub-kilobase%20resolution%20using%20fine-tiling%20oligonucleotide%20array%20CGH&amp;author=R.%20R.%20Selzer&amp;author=T.%20A.%20Richmond&amp;author=N.%20J.%20Pofahl&amp;publication_year=2005&amp;journal=Genes%20Chromosomes%20Cancer&amp;volume=44&amp;pages=305-319" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/gcc.v44:3" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fgcc.v44:3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fgcc.v44:3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16075461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Whole genome DNA copy number changes identified by high-density oligonucleotide arrays</div>, <div class="source">Hum. Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">287</div>-<div class="lpage">299</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20DNA%20copy%20number%20changes%20identified%20by%20high-density%20oligonucleotide%20arrays&amp;author=J.%20Huang&amp;author=W.%20Wei&amp;author=J.%20Zhang&amp;publication_year=2004&amp;journal=Hum.%20Genomics&amp;volume=1&amp;pages=287-299" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1479-7364-1-4-287" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1479-7364-1-4-287" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1479-7364-1-4-287"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15588488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nannya</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sanada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakazaki</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A robust algorithm for copy number detection using high-density oligonucleotide single-nucleotide polymorphism genotyping arrays</div>, <div class="source">Cancer Res.</div> , <div class="year">2005</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">6071</div>-<div class="lpage">6079</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%20algorithm%20for%20copy%20number%20detection%20using%20high-density%20oligonucleotide%20single-nucleotide%20polymorphism%20genotyping%20arrays&amp;author=Y.%20Nannya&amp;author=M.%20Sanada&amp;author=K.%20Nakazaki&amp;publication_year=2005&amp;journal=Cancer%20Res.&amp;volume=65&amp;pages=6071-6079" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-05-0465" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-05-0465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-05-0465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16024607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slater</div> <div class="given-names">H. R.</div></div>,  <div class="name"><div class="surname">Bailey</div> <div class="given-names">D. K.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">High-resolution identification of chromosomal abnormalities using oligonucleotide arrays containing 116,204 SNPs</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">709</div>-<div class="lpage">726</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20identification%20of%20chromosomal%20abnormalities%20using%20oligonucleotide%20arrays%20containing%20116%2C204%20SNPs&amp;author=H.%20R.%20Slater&amp;author=D.%20K.%20Bailey&amp;author=H.%20Ren&amp;publication_year=2005&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=77&amp;pages=709-726" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/497343" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F497343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F497343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16252233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Friedman</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Baross</div> <div class="given-names">Á.</div></div>,  <div class="name"><div class="surname">Delaney</div> <div class="given-names">A. D.</div></div>, et al. </div><div class="article-title">Oligonucleotide microarray analysis of genomic imbalance in children with mental retardation</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">500</div>-<div class="lpage">513</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oligonucleotide%20microarray%20analysis%20of%20genomic%20imbalance%20in%20children%20with%20mental%20retardation&amp;author=J.%20M.%20Friedman&amp;author=%C3%81.%20Baross&amp;author=A.%20D.%20Delaney&amp;publication_year=2006&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=79&amp;pages=500-513" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/507471" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F507471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F507471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16909388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peiffer</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Le</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Steemers</div> <div class="given-names">F. J.</div></div>, et al. </div><div class="article-title">High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1136</div>-<div class="lpage">1148</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20genomic%20profiling%20of%20chromosomal%20aberrations%20using%20Infinium%20whole-genome%20genotyping&amp;author=D.%20A.%20Peiffer&amp;author=J.%20M.%20Le&amp;author=F.%20J.%20Steemers&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1136-1148" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5402306" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5402306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5402306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16899659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bammler</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Beyer</div> <div class="given-names">R. P.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Standardizing global gene expression analysis between laboratories and across platforms</div>, <div class="source">Nat. Methods</div> , <div class="year">2005</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">351</div>-<div class="lpage">356</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Standardizing%20global%20gene%20expression%20analysis%20between%20laboratories%20and%20across%20platforms&amp;author=T.%20Bammler&amp;author=R.%20P.%20Beyer&amp;author=S.%20Bhattacharya&amp;publication_year=2005&amp;journal=Nat.%20Methods&amp;volume=2&amp;pages=351-356" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth0605-477a" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth0605-477a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth0605-477a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15846362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barnes</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Freudenberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Aronow</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pavlidis</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Experimental comparison and cross-validation of the Affymetrix and Illumina gene expression analysis platforms</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">5914</div>-<div class="lpage">5923</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20comparison%20and%20cross-validation%20of%20the%20Affymetrix%20and%20Illumina%20gene%20expression%20analysis%20platforms&amp;author=M.%20Barnes&amp;author=J.%20Freudenberg&amp;author=S.%20Thompson&amp;author=B.%20Aronow&amp;author=P.%20Pavlidis&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5914-5923" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki890" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16237126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Warren</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Spencer</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Multiple-laboratory comparison of microarray platforms</div>, <div class="source">Nat. Methods</div> , <div class="year">2005</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">345</div>-<div class="lpage">350</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple-laboratory%20comparison%20of%20microarray%20platforms&amp;author=R.%20A.%20Irizarry&amp;author=D.%20Warren&amp;author=F.%20Spencer&amp;publication_year=2005&amp;journal=Nat.%20Methods&amp;volume=2&amp;pages=345-350" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth756" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15846361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larkin</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Frank</div> <div class="given-names">B. C.</div></div>,  <div class="name"><div class="surname">Gavras</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sultana</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Quackenbush</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Independence and reproducibility across microarray platforms</div>, <div class="source">Nat. Methods</div> , <div class="year">2005</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">337</div>-<div class="lpage">344</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Independence%20and%20reproducibility%20across%20microarray%20platforms&amp;author=J.%20E.%20Larkin&amp;author=B.%20C.%20Frank&amp;author=H.%20Gavras&amp;author=R.%20Sultana&amp;author=J.%20Quackenbush&amp;publication_year=2005&amp;journal=Nat.%20Methods&amp;volume=2&amp;pages=337-344" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth757" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth757"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15846360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Siegel</div> <div class="given-names">S.</div></div>. </div>, <div class="source">Nonparametric Statistics for the Behavioral Sciences</div> , <div class="year">1988</div><div class="edition">2nd ed.</div><div class="publisher-loc">New York</div><div class="publisher-name">McGraw-Hill</div></div></div></div></div><div content-id="dsm002c37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loo</div> <div class="given-names">L. W. M.</div></div>,  <div class="name"><div class="surname">Grove</div> <div class="given-names">D. I.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">E. M.</div></div>, et al. </div><div class="article-title">Array comparative genomic hybridization analysis of genomic alterations in breast cancer subtypes</div>, <div class="source">Cancer Res.</div> , <div class="year">2004</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">8541</div>-<div class="lpage">8549</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array%20comparative%20genomic%20hybridization%20analysis%20of%20genomic%20alterations%20in%20breast%20cancer%20subtypes&amp;author=L.%20W.%20M.%20Loo&amp;author=D.%20I.%20Grove&amp;author=E.%20M.%20Williams&amp;publication_year=2004&amp;journal=Cancer%20Res.&amp;volume=64&amp;pages=8541-8549" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-04-1992" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-04-1992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-04-1992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15574760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neyman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tokarska</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Errors of the second kind in testing “Student's” hypothesis</div>, <div class="source">J. Am. Stat. Assoc.</div> , <div class="year">1936</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">318</div>-<div class="lpage">326</div>)</div></div></div></div><div content-id="dsm002c39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">N. L.</div></div>,  <div class="name"><div class="surname">Welch</div> <div class="given-names">B. L.</div></div>. </div><div class="article-title">Applications of the non-central <em>t</em>-distribution</div>, <div class="source">Biometrika</div> , <div class="year">1940</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">362</div>-<div class="lpage">389</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20the%20non-central%20t-distribution&amp;author=N.%20L.%20Johnson&amp;author=B.%20L.%20Welch&amp;publication_year=1940&amp;journal=Biometrika&amp;volume=31&amp;pages=362-389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biomet/31.3-4.362" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiomet%2F31.3-4.362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiomet%2F31.3-4.362"> </span></div> </div></div></div></div></div><div content-id="dsm002c40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olshen</div> <div class="given-names">A. B.</div></div>,  <div class="name"><div class="surname">Venkatatraman</div> <div class="given-names">E. S.</div></div>,  <div class="name"><div class="surname">Lucito</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wigler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Circular binary segmentation for the analysis of array-based DNA copy number data</div>, <div class="source">Biostatistics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">557</div>-<div class="lpage">572</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20binary%20segmentation%20for%20the%20analysis%20of%20array-based%20DNA%20copy%20number%20data&amp;author=A.%20B.%20Olshen&amp;author=E.%20S.%20Venkatatraman&amp;author=R.%20Lucito&amp;author=M.%20Wigler&amp;publication_year=2004&amp;journal=Biostatistics&amp;volume=5&amp;pages=557-572" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/kxh008" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2Fkxh008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2Fkxh008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15475419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fridlyand</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Snijders</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Pinkel</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Albertson</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">A. N.</div></div>. </div><div class="article-title">Hidden Markov models approach to the analysis of array CGH data</div>, <div class="source">J. Multivariate Anal.</div> , <div class="year">2004</div>, vol. <div class="volume">90</div> <div class="issue">1</div>(pg. <div class="fpage">132</div>-<div class="lpage">153</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hidden%20Markov%20models%20approach%20to%20the%20analysis%20of%20array%20CGH%20data&amp;author=J.%20Fridlyand&amp;author=A.%20M.%20Snijders&amp;author=D.%20Pinkel&amp;author=D.%20G.%20Albertson&amp;author=A.%20N.%20Jain&amp;publication_year=2004&amp;journal=J.%20Multivariate%20Anal.&amp;volume=90&amp;pages=132-153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmva.2004.02.008" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmva.2004.02.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmva.2004.02.008"> </span></div> </div></div></div></div></div><div content-id="dsm002c42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diskin</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Eck</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Greshock</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">STAC: a method for testing the significance of DNA copy number aberrations across multiple array-CGH experiments</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1149</div>-<div class="lpage">1158</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAC%3A%20a%20method%20for%20testing%20the%20significance%20of%20DNA%20copy%20number%20aberrations%20across%20multiple%20array-CGH%20experiments&amp;author=S.%20J.%20Diskin&amp;author=T.%20Eck&amp;author=J.%20Greshock&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1149-1158" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5076506" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5076506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5076506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16899652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murphy</div> <div class="given-names">K. R.</div></div>. </div>, <div class="source">Statistical Power Analysis: A Simple and General Model for Traditional and Modern Hypothesis Tests</div> , <div class="year">2004</div><div class="edition">2nd ed.</div><div class="publisher-loc">London</div><div class="publisher-name">Lawrence Erlbaum Associates</div></div></div></div></div><div content-id="dsm002c44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wirtenberger</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hemminki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Burwinkel</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Identification of frequent chromosome copy-number polymorphisms by use of high-resolution single-nucleotide-polymorphism arrays</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">520</div>-<div class="lpage">522</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20frequent%20chromosome%20copy-number%20polymorphisms%20by%20use%20of%20high-resolution%20single-nucleotide-polymorphism%20arrays&amp;author=M.%20Wirtenberger&amp;author=K.%20Hemminki&amp;author=B.%20Burwinkel&amp;publication_year=2006&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=78&amp;pages=520-522" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/500793" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F500793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F500793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16572511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hsu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Self</div> <div class="given-names">S. G.</div></div>,  <div class="name"><div class="surname">Grove</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Denoising array-based comparative genomic hybridization data using wavelets</div>, <div class="source">Biostatistics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">211</div>-<div class="lpage">226</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Denoising%20array-based%20comparative%20genomic%20hybridization%20data%20using%20wavelets&amp;author=L.%20Hsu&amp;author=S.%20G.%20Self&amp;author=D.%20Grove&amp;publication_year=2005&amp;journal=Biostatistics&amp;volume=6&amp;pages=211-226" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/kxi004" target="_blank">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2Fkxi004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2Fkxi004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15772101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsm002c46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsm002c46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilcox</div> <div class="given-names">R. R.</div></div>. </div>, <div class="source">Fundamentals of Modern Statistical Methods: Substantially Improving Power and Accuracy</div> , <div class="year">2001</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div></div></div></div></div></div>        <!-- /foreach -->




        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=502776&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/1/10.1093_dnares_dsm002/1/dsm002_Supplementary_Data.zip?Expires=1509238563&amp;Signature=ceLI0CAqAJ4X-SPj7I4KOlT9ZrJCxlpHFxYnGiqNRRNY3tGqGP3flRb6bIDgaelpmd01KoUNb8ya9WQLSPR0rwXF8mP-hK1wLTdcPYUsVycDqMjY9g13T-rRJOpWBl3EV~LD1C8UqetvwuzSTwNhgINYjbstNlxovpoMV1YC1~GVQFTMYXMfbMVEuAzeNgzPTkpl3EVu7hgFU3~r0V59p~H-tc7U7~DTRL9RFoh6IXcFaBac9kwpE~pTYETY0095rbSv2SZOPhOeLaTIpKjYVaCA5WhLjfK2gjbkQ1Bzy6kUGymNZH5h7S7OVz6Tjtghd4vxWBKhVoQB6BkIEbERwA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">18</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248042500001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">73</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class='altmetric-embed' data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsm002"
     
        
        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="502776"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userLoggedIn="False" data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userLoggedIn="False" data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userLoggedIn="False" data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userLoggedIn="False" data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248042500001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f1%2f1%2f502776">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000248042500001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (73)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F1%2F1%2F502776" target="_blank">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/502776'>CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx041/4562529">Investigating transcription factor synergism in humans</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx042/4554776">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx040/4282306">Comparative genome and transcriptome analyses reveal adaptations to opportunistic infections in woody plant degrading pathogens of <em>Botryosphaeriaceae</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx039/4265514">Circular RNAs are abundant and dynamically expressed during embryonic muscle development in chickens</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="/dnaresearch/article/doi/10.1093/dnares/dsx038/4191345">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide Copy Number Profiling on High-density Bacterial Artificial Chromosomes, Single-nucleotide Polymorphisms, and Oligonucleotide Microarrays: A Platform Comparison based on Statistical Power Analysis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=' '>
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2017 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2017 Oxford University Press</li>
    <li><a href="/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f1%2f1%2f502776">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/aebc256a112dbaf84b128ee864a3ae7c70f3b601/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92f525bab295d0ffa6f942402d5f7034757b1440/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/c847e338b5a9aa3e333cffb5d518f1fff4a42520/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

